<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-11-11 06:30:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>26</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>60</td>
          <td>133</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>103</td>
          <td>143</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>298</td>
          <td>133</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>252</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is strongly associated with treatment resistance and shorter survival for patients with cancer, including patients with glioblastoma. The non-chromosomal inheritance of ecDNA during cell division is a major contributor to intratumoral genetic heterogeneity. At present, the spatial dynamics of ecDNA, and the impact on tumor evolutionary trajectories, are not well understood. Here, we investigate the spatial-temporal evolution of ecDNA and its clinical impact by analyzing tumor samples from 94 treatment-naive human IDH-wildtype glioblastoma patients. We developed a spatial-temporal computational model of ecDNA positive tumors (‘SPECIES’) that integrates whole-genome sequencing, multi-region DNA FISH, and nascent RNAscope, to provide unique insight into the spatial dynamics of ecDNA evolution. Random segregation in combination with positive selection of ecDNAs induce large, predictable spatial patterns of cell-to-cell ecDNA copy number variation that are highly dependent on the oncogene encoded on the circular DNA. EGFR ecDNAs often reach high mean copy number (mean of 50 copies per tumor cell), are under strong positive selection (mean selection coefficient, s > 2) and do not co-amplify other oncogenes on the same ecDNA particles. In contrast, PDGFRA ecDNAs have lower mean copy number (mean of 15 copies per cell), are under weaker positive selection and frequently co-amplify other oncogenes on the same ecDNA. Evolutionary modeling suggests that EGFR ecDNAs often accumulate prior to clonal expansion. EGFR structural variants, including vIII and c-terminal deletions are under strong positive selection, are found exclusively on ecDNA, and are intermixed with wild-type EGFR ecDNAs. Simulations show EGFRvIII ecDNA likely arises after ecDNA formation in a cell with high wild-type EGFR copy number (> 10) before the onset of the most recent clonal expansion. This remains true even in cases of co-selection and co-amplification of multiple oncogenic ecDNA species in a subset of patients. Overall, our results suggest a potential time window in which early ecDNA detection may provide an opportunity for more effective intervention. Highlights ecDNA is the most common mechanism of focal oncogene amplification in IDHwt glioblastoma. EGFR and its variants on ecDNA are particularly potent, likely arising early in tumor development, providing a strong oncogenic stimulus to drive tumorigenesis. Wild-type and variant EGFR ecDNA heteroplasmy (co-occurrence) is common with EGFRvIII or c-terminal deletions being derived from EGFR wild-type ecDNA prior to the most recent clonal expansion. Tumors with ecDNA amplified EGFR versus PDGFRA exhibit different evolutionary trajectories. SPECIES model can infer spatial evolutionary dynamics of ecDNA in cancer. A delay between ecDNA accumulation and subsequent oncogenic mutation may give a therapeutic window for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e8d31fa2dc1841943bfa95252e831edb576e77" target='_blank'>
              Extrachromosomal DNA driven oncogene spatial heterogeneity and evolution in glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, M. Haughey, J. Luebeck, A. Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Jeanette Kittel, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G Jones, E. Nye, M. Green, Lucy Meader, E. Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f686dc6f9303431a89daa1a7f74014cc18c394d" target='_blank'>
              Coordinated inheritance of extrachromosomal DNAs in cancer cells
              </a>
            </td>
          <td>
            King L. Hung, Matthew G Jones, I. Wong, Ellis J Curtis, Joshua T. Lange, Britney Jiayu He, J. Luebeck, R. Schmargon, Elisa Scanu, Lotte Brückner, Xiaowei Yan, Rui Li, Aditi Gnanasekar, R. Chamorro González, J. Belk, Zhonglin Liu, Bruno Melillo, V. Bafna, J. R. Dörr, Benjamin Werner, Weini Huang, Benjamin F. Cravatt, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, S. Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf0847e7afd3b17c26839196bb1a05ffa786834" target='_blank'>
              Aneuploidy as a driver of human cancer.
              </a>
            </td>
          <td>
            Eran Sdeor, Hajime Okada, Ron Saad, Tal Ben-Yishay, Uri Ben-David
          </td>
          <td>2024-10-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Translocations have largely been implicated in tumor development. However, beyond the consequences of aberrant gene expression near the breakpoint, their effects remain unexplored. In this work, we characterize the interplay between translocations, chromatin organization and gene expression using mantle cell lymphoma (MCL) as a model. We show that in vitro induced MCL-associated translocations can drive transcriptional changes at entire chromosome arms affecting multiple genes in a regulon-like fashion. Additionally, overexpressed genes in MCL patients are enriched in the exact same genomic regions, further underlining its potential relevance for lymphomagenesis. Moreover, we demonstrate a clear link between the translocation-induced transcriptional alterations and genome organization, with genes most susceptible to change expression residing in pre-existing long-range interacting loops spanning 50 megabases. The translocation places the strong immunoglobulin enhancer into this loop, allowing the spread of its regulatory potential over the entire affected chromosome arm. Finally, we show that translocation-induced effects mainly represent expression enhancement of genes already active prior to translocation formation, highlighting the importance of the epigenetic state of the cell in which this initial hit occurs. In summary, we show that translocations can induce ample, simultaneous gene expression changes affecting entire chromosome arms, representing an important new mechanism for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdbb3468215c566e38e0c9ba35afc86f4ee62c1" target='_blank'>
              Translocations can drive expression changes of multiple genes in regulons covering entire chromosome arms
              </a>
            </td>
          <td>
            Anna Oncins, Roser Zaurin, Houyem Toukabri, Kimberly Quililan, José R Hernández Mora, Magdalena A. Karpinska, Erik Wernersson, Alastair Smith, Agostina Bianchi, Leone Albinati, Andrea Rivero, François Serra, Raúl Gómez, Cristina López, S. Beà, Nadia Halidi, A. Valencia, Magda Bienko, A. M. Oudelaar, Renée Beekman
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09784c909701bec6584bd91b9c974eb6800207e8" target='_blank'>
              Enhancing transcription–replication conflict targets ecDNA-positive cancers
              </a>
            </td>
          <td>
            Jun Tang, Natasha E Weiser, Guiping Wang, Sudhir Chowdhry, Ellis J Curtis, Yanding Zhao, I. Wong, Georgi K. Marinov, Rui Li, Philip Hanoian, Edison Tse, Salvador Garcia Mojica, Ryan J. Hansen, Joshua Plum, Auzon Steffy, S. Milutinovic, S. T. Meyer, J. Luebeck, Yanbo Wang, Shu Zhang, Nicolas Altemose, Christina Curtis, W.J. Greenleaf, V. Bafna, S. J. Benkovic, Anthony B Pinkerton, S. Kasibhatla, Christian A Hassig, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4921d9b380eadcd80f19284f6396e453cfba83e" target='_blank'>
              Exploring the potential of extrachromosomal DNA as a novel oncogenic driver.
              </a>
            </td>
          <td>
            Huanbo Zhu, Longtao Huangfu, Junbing Chen, Jiafu Ji, Xiaofang Xing
          </td>
          <td>2024-09-27</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer Shelton, Minita J. Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) emerges as a crucial promoter of rapid intra-tumour heterogeneity and oncogene amplification, contributing to tumour progression and adaptability to resistance. Multiple types of ecDNA are often observed within the same cancer cell, promoting high levels of complexity. While the study of multiple ecDNA types is still in its early stages with exciting new clinical and experimental studies in human cancers, few theoretical models have been proposed to understand the mechanisms enabling the maintenance of multiple ecDNA types (mix cells). Here, based on our general framework of modeling multiple ecDNA types, we explore the conditions that allow for a stable coexistence of multiple ecDNA types within single cancer cells. We study how genetic mutations or phenotypic switching among different ecDNA types contribute to the maintenance of ecDNA diversity under different fitness regimes. We show that either positive epistasis or switching is required to stably maintain mix cell populations. Counter-intuitively, intermediate rather than high switching rates lead to the highest fraction of mix cells, which is further confirmed by our analysis of sub-population compositions, transition probabilities between pure and mix states as well as single-cell Shannon indices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398d1444c0fac5c6ddf4fe45cb3ef1584d2da2f5" target='_blank'>
              The maintenance of multiple ecDNA types in single cancer cells
              </a>
            </td>
          <td>
            Elisa Scanu, Benjamin Werner, Weini Huang
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd2a73c90ca9b2dfa75e988092f2839d133a616" target='_blank'>
              Origins and impact of extrachromosomal DNA
              </a>
            </td>
          <td>
            Chris Bailey, O. Pich, K. Thol, T. Watkins, J. Luebeck, A. Rowan, Georgia Stavrou, Natasha E Weiser, Bhargavi Dameracharla, R. Bentham, Wei-Ting Lu, Jeanette Kittel, S. Y. C. Yang, B. Howitt, Natasha Sharma, Maria Litovchenko, R. Salgado, King L. Hung, A. Cornish, David A Moore, R. Houlston, V. Bafna, Howard Y. Chang, S. Nik-Zainal, N. Kanu, N. Mcgranahan, Adrienne M. Flanagan, P. Mischel, M. Jamal-Hanjani, C. Swanton
          </td>
          <td>2024-11-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Ribosomal RNA (rRNA) genes exist in multiple copies arranged in tandem arrays known as ribosomal DNA (rDNA). The total number of gene copies is variable, and the mechanisms buffering this copy number variation remain unresolved. We surveyed the number, distribution, and activity of rDNA arrays at the level of individual chromosomes across multiple human and primate genomes. Each individual possessed a unique fingerprint of copy number distribution and activity of rDNA arrays. In some cases, entire rDNA arrays were transcriptionally silent. Silent rDNA arrays showed reduced association with the nucleolus and decreased interchromosomal interactions, indicating that the nucleolar organizer function of rDNA depends on transcriptional activity. Methyl-sequencing of flow-sorted chromosomes, combined with long read sequencing, showed epigenetic modification of rDNA promoter and coding region by DNA methylation. Silent arrays were in a closed chromatin state, as indicated by the accessibility profiles derived from Fiber-seq. Removing DNA methylation restored the transcriptional activity of silent arrays. Array activity status remained stable through the iPS cell re-programming. Family trio analysis demonstrated that the inactive rDNA haplotype can be traced to one of the parental genomes, suggesting that the epigenetic state of rDNA arrays may be heritable. We propose that the dosage of rRNA genes is epigenetically regulated by DNA methylation, and these methylation patterns specify nucleolar organizer function and can propagate transgenerationally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ccedcc94269c3f1af6cd47f297cceb334ba7506" target='_blank'>
              Epigenetic control and inheritance of rDNA arrays
              </a>
            </td>
          <td>
            T. Potapova, Paxton Kostos, S. McKinney, M. Borchers, Jeff Haug, A. Guarracino, Steven J. Solar, Madelaine M. Gogol, Graciela Monfort Anez, L. G. de Lima, Yan Wang, Kate Hall, Sophie Hoffman, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e1ab3261b0f7d101efd025d4b4ace17226e118a" target='_blank'>
              High level of aneuploidy and recurrent loss of chromosome 11 as relevant features of somatotroph pituitary tumors
              </a>
            </td>
          <td>
            J. Rymuza, P. Kober, M. Maksymowicz, Aleksandra Nyc, B. J. Mossakowska, R. Woroniecka, Natalia Maławska, B. Grygalewicz, Szymon Baluszek, Grzegorz Zieliński, J. Kunicki, M. Bujko
          </td>
          <td>2024-11-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Robertsonian chromosomes are a type of variant chromosome found commonly in nature. Present in one in 800 humans, these chromosomes can underlie infertility, trisomies, and increased cancer incidence. Recognized cytogenetically for more than a century, their origins have remained mysterious. Recent advances in genomics allowed us to assemble three human Robertsonian chromosomes completely. We identify a common breakpoint and epigenetic changes in centromeres that provide insight into the formation and propagation of common Robertsonian translocations. Further investigation of the assembled genomes of chimpanzee and bonobo highlights the structural features of the human genome that uniquely enable the specific crossover event that creates these chromosomes. Resolving the structure and epigenetic features of human Robertsonian chromosomes at a molecular level paves the way to understanding how chromosomal structural variation occurs more generally, and how chromosomes evolve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a20561e10d74149e6076e6c3bf2d3aed806c4" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes
              </a>
            </td>
          <td>
            Leonardo Gomes de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cancer genomes frequently carry APOBEC (apolipoprotein B mRNA editing catalytic polypeptide-like)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and/or evolution. However, the pure mutagenic impacts of the specific enzymes among this family that are responsible for APOBEC-associated mutagenesis remain unclear, particularly the comparative mutagenic activities of APOBEC3A and APOBEC3B. Here, we investigated the mutagenic contributions of these enzymes through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC3A or APOBEC3B cassettes. Our findings demonstrated that transcriptional APOBEC3A upregulation led to the acquisition of a massive number of genomic mutations in a few cell cycles. By contrast, APOBEC3B upregulation did not generate a substantial number of mutations in gastric epithelium. APOBEC3B-associated mutagenesis remained insignificant even after a combined inactivation of TP53. Based on the spectrum of acquired mutations after APOBEC3A upregulation, we further analyzed APOBEC3A-associated mutational signatures, encompassing indels mainly composed of 1bp deletions, characteristics of clustered mutations, and selective pressures operative on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4025cc28df4328626e6c695274ccd359914520" target='_blank'>
              APOBEC3A, not APOBEC3B, drives deaminase mutagenesis in human gastric epithelium
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction: Copy number variations (CNVs) are structural genomic alterations that involve changes in the number of copies of specific DNA regions. These variations can include deletions, duplications, and more complex rearrangements, and play a critical role in cancer progression by amplifying oncogenes, deleting tumor suppressor genes, or altering other key genomic regions. Despite the importance of CNVs in cancer biology, there is a lack of comprehensive resources that aggregate CNV data across multiple cancer types, impeding the exploration of their role in different malignancies. The PanCNV-Explorer was developed to fill this gap by providing a pan-cancer resource that integrates CNV data for over 21 cancer types. Methods: The PanCNV-Explorer was developed, utilizing data from repositories such as COSMIC, DepMap, DGV, and dbVAR. Amplifications and deletions can be visualized, with detailed chromosome-specific plots highlighting CNV patterns. In addition, a genome browser has been implemented to display regions of interest for searching for genes of potential relevance. Results: The database contains over 5 million CNVs from 15,809 samples, spanning 21 primary cancer types. Pathogenic CNVs from ClinVar and DepMap were compared to benign data from DGV and dbVAR, allowing a comprehensive analysis across tissues. CNV patterns were visualized for individual cancers. We demonstrate with use cases, focusing on common oncogenes such as MYC and tumor suppressor genes such as TP53, the ability of the resource to search for cancer driver genes. For example, chromosome 12 in pancreatic cancer showed frequent amplifications in the region containing the KRAS oncogene. In another example, we show how driver gene candidates already described in certain cancer entities can also be found with high frequency in other entities. Conclusion: The PanCNV-Explorer provides a valuable resource for pan-cancer CNV analysis, facilitating the exploration of CNV-driven tumor heterogeneity. The integrated visualization tools and genome browser enable detailed exploration of genomic regions, enhancing cancer research. As a valuable and easy-to-use platform that allows researchers to quantify and compare CNVs across cancer types, the PanCNV-Explorer is an essential tool for advancing cancer genomics research. The main page is available at https://mtb.bioinf.med.uni-goettingen.de/pancnv-explorer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0740ab5d12642c500ab7be4dd721a73d9e32d198" target='_blank'>
              PanCNV Explorer: A pan-cancer database for copy number variations.
              </a>
            </td>
          <td>
            Kevin Kornrumpf, Jürgen Dönitz
          </td>
          <td>2024-09-24</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Traditional drug discovery efforts have largely focused on targeting rapid, reversible protein-mediated adaptations to undermine cancer cells’ resistance to therapy. However, cancer cells also exploit DNA-based strategies, typically viewed as slow, irreversible, and unpredictable changes like point mutations or the selection of drug-resistant clones. Contrary to this perception, extrachromosomal DNA (ecDNA) represents a form of DNA alteration that is rapid, reversible, and predictable, playing a crucial role in cancer’s adaptive response. In this study, we present a novel post-processing pipeline for the automated detection and quantification of ecDNA in Fluorescence in situ Hybridization (FISH) images using the Microscopy Image Analyzer (MIA) tool. Our approach is particularly designed to monitor ecDNA dynamics during drug treatment, providing a quantitative framework to understand how ecDNA enables cancer cells to swiftly and reversibly adapt to therapeutic pressure. This pipeline not only offers a valuable resource for researchers aiming to automate ecDNA detection in FISH images but also sheds light on the adaptive mechanisms of ecDNA in response to epigenetic remodeling agents like JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a33656032abe4b14c89c4da5e33bf1e9ab825af" target='_blank'>
              Leveraging AI to Automate Detection and Quantification of Extrachromosomal DNA (ecDNA) to Decode Drug Responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcfc9fc5c7ce6b28eb00b12e973f760953fb4c62" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-09-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="It is over twenty years since the publication of the C57BL/6J mouse reference genome, which has been a key catalyst for understanding mammalian disease biology. However, the mouse reference genome still lacks telomeres and centromeres, contains 281 chromosomal sequence gaps, and only partially represents many biomedically relevant loci. We present the first T2T mouse genomes for two key inbred strains, C57BL/6J and CAST/EiJ. These T2T genomes reveal significant variability in telomere and centromere sizes and structural organisation. We add an additional 213 Mbp of novel sequence to the reference genome containing 517 protein-coding genes. We examined two important but incomplete loci in the mouse genome - the pseudoautosomal region (PAR) on the sex chromosomes and KRAB zinc finger proteins (KZFPs) loci. We identified distant locations of the PAR boundary, different copy number and sizes of segmental duplications, and a multitude of amino acid substitution mutations in PAR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30db2388994ae46d83e7501923b0604dde5725a" target='_blank'>
              The structural diversity of telomeres and centromeres across mouse subspecies revealed by complete assemblies
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Mohab Helmy, Tadafumi Kato, A. Czechanski, Michael A. Quail, Iraad Bonner, Emma Dawson, Anne Ferguson-Smith, L. Reinholdt, David J. Adams, Thomas M. Keane
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="As a major type of structural variants, tandem duplication plays a critical role in tumorigenesis by increasing oncogene dosage. Recent work has revealed that noncoding enhancers are also affected by duplications leading to the activation of oncogenes that are inside or outside of the duplicated regions. However, the prevalence of enhancer duplication and the identity of their target genes remains largely unknown in the cancer genome. Here, by analyzing whole-genome sequencing data in a non-gene-centric manner, we identify 881 duplication hotspots in 13 major cancer types, most of which do not contain protein-coding genes. We show that the hotspots are enriched with distal enhancer elements and are highly lineage-specific. We develop a HiChIP-based methodology that navigates enhancer-promoter contact maps to prioritize the target genes for the duplication hotspots harboring enhancer elements. The methodology identifies many novel enhancer duplication events activating oncogenes such as ESR1, FOXA1, GATA3, GATA6, TP63, and VEGFA, as well as potentially novel oncogenes such as GRHL2, IRF2BP2, and CREB3L1 In particular, we identify a duplication hotspot on Chromosome 10p15 harboring a cluster of enhancers, which skips over two genes, through a long-range chromatin interaction, to activate an oncogenic isoform of the NET1 gene to promote migration of gastric cancer cells. Focusing on tandem duplications, our study substantially extends the catalog of noncoding driver alterations in multiple cancer types, revealing attractive targets for functional characterization and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269520ff92f1d6c1a566c78a1f6e2344309fcdcb" target='_blank'>
              Chromatin interaction maps identify oncogenic targets of enhancer duplications in cancer.
              </a>
            </td>
          <td>
            Yueqiang Song, Fuyuan Li, Shangzi Wang, Yuntong Wang, Cong Lai, Lian Chen, Ning Jiang, Jin Li, Xingdong Chen, Swneke D. Bailey, Xiaoyang Zhang
          </td>
          <td>2024-10-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Technologies for precisely inserting large DNA sequences into the genome are critical for diverse research and therapeutic applications. Large serine recombinases (LSRs) can mediate direct, site-specific genomic integration of multi-kilobase DNA sequences without a pre-installed landing pad, but current approaches suffer from low insertion rates and high off-target activity. Here, we present a comprehensive engineering roadmap for the joint optimization of DNA recombination efficiency and specificity. We combined directed evolution, structural analysis, and computational models to rapidly identify additive mutational combinations. We further enhanced performance through donor DNA optimization and dCas9 fusions, enabling simultaneous target and donor recruitment. Top engineered LSR variants achieved up to 53% integration efficiency and 97% genome-wide specificity at an endogenous human locus, and effectively integrated large DNA cargoes (up to 12 kb tested) for stable expression in challenging cell types, including non-dividing cells, human embryonic stem cells, and primary human T cells. This blueprint for rational engineering of DNA recombinases enables precise genome engineering without the generation of double-stranded breaks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e612f09bf2ad777f926525b4211346f40770798" target='_blank'>
              Site-specific DNA insertion into the human genome with engineered recombinases
              </a>
            </td>
          <td>
            Alison Fanton, Liam J. Bartie, Juliana Q. Martins, Vincent Q. Tran, Laine Goudy, Matthew G. Durrant, Jingyi Wei, April Pawluk, Silvana Konermann, Alexander Marson, Luke A. Gilbert, Patrick D. Hsu
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA transactions introduce torsional constraints that pose an inherent risk to genome integrity. While topoisomerase 1 (TOP1) activity is essential for removing DNA supercoiling, aberrant stabilization of TOP1:DNA cleavage complexes (TOP1ccs) can result in cytotoxic DNA lesions. What protects genomic hot spots of topological stress from aberrant TOP1 activity remains unknown. Here, we identify chromatin context as an essential means to coordinate TOP1cc resolution. Through its ability to bind poly(ADP-ribose) (PAR), a protein modification required for TOP1cc repair, the histone variant macroH2A1.1 establishes a TOP1-permissive chromatin environment, while the alternatively spliced macroH2A1.2 isoform is unable to bind PAR or protect from TOP1ccs. By visualizing transcription-induced topological stress in single cells, we find that macroH2A1.1 facilitates PAR-dependent recruitment of the TOP1cc repair effector XRCC1 to protect from ssDNA damage. Impaired macroH2A1.1 splicing, a frequent cancer feature, was predictive of increased sensitivity to TOP1 poisons in a pharmaco-genomic screen in breast cancer cells, and macroH2A1.1 inactivation mirrored this effect. Consistent with this, low macroH2A1.1 expression correlated with improved survival in cancer patients treated with TOP1 inhibitors. We propose that macroH2A1 alternative splicing serves as an epigenetic modulator of TOP1-associated genome maintenance and a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3b4454ca7bb6b690498643fce3388d2e75cea12" target='_blank'>
              Epigenetic control of Topoisomerase 1 activity presents a cancer vulnerability
              </a>
            </td>
          <td>
            Tae-Hee Lee, Colina X Qiao, Vladislav Kuzin, Yuepeng Shi, Vijayalalitha Ramanaranayan, Tongyu Wu, Xianzhen Zhou, David Corujo, M. Buschbeck, Laura Baranello, Philipp Oberdoerffer
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a180cfc530d1d7fe2d98876dc6894d589798a6e8" target='_blank'>
              Mapping extrachromosomal DNA amplifications during cancer progression
              </a>
            </td>
          <td>
            Hoon Kim, Soyeon Kim, Taylor E. Wade, Eunchae Yeo, A. Lipsa, A. Gołȩbiewska, Kevin C Johnson, Sepil An, Junyong Ko, Yoonjoo Nam, Hwa Yeon Lee, Seunghyun Kang, Heesuk Chung, S. Niclou, H. Moon, S. Paek, V. Bafna, J. Luebeck, R.G.W. Verhaak
          </td>
          <td>2024-10-14</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>70</td>
        </tr>

        <tr id="It is unclear how cells counteract the potentially harmful effects of uncoordinated DNA replication in the context of oncogenic stress. Here, we identify the WRAD (WDR5/RBBP5/ASH2L/DPY30) core as a modulator of DNA replication in pancreatic ductal adenocarcinoma (PDAC) models. Molecular analyses demonstrated that the WRAD core interacts with the replisome complex, with disruption of DPY30 resulting in DNA re-replication, DNA damage, and chromosomal instability (CIN) without affecting cancer cell proliferation. Consequently, in immunocompetent models, DPY30 loss induced T cell infiltration and immune-mediated clearance of highly proliferating cancer cells with complex karyotypes, thus improving anti-tumor efficacy upon anti-PD-1 treatment. In PDAC patients, DPY30 expression was associated with high tumor grade, worse prognosis, and limited response to immune checkpoint blockade. Together, our findings indicate that the WRAD core sustains genome stability and suggest that low intratumor DPY30 levels may identify PDAC patients who will benefit from immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561fe3b30b264bde1623b0000d0a802b77d77a31" target='_blank'>
              WRAD core perturbation impairs DNA replication fidelity promoting immunoediting in pancreatic cancer
              </a>
            </td>
          <td>
            F. Citron, I. Ho, Chiara Balestrieri, Zhaoliang Liu, Er-Yen Yen, Luca Cecchetto, Luigi Perelli, Li Zhang, Luis Castillo Montañez, Nicholas Blazanin, Charles A. Dyke, Rutvi Shah, Sergio Attanasio, Sanjana Srinivasan, Ko-Chien Chen, Ziheng Chen, Iolanda Scognamiglio, Nhung Pham, Hania Khan, Shan Jiang, Jing Pan, B. Vanderkruk, Cecilia S. Leung, Mahinur Mattohti, Kunal Rai, Yanshuo Chu, Linghua Wang, Sisi Gao, Angela Deem, A. Carugo, Huamin Wang, W. Yao, Giovanni Tonon, Yun Xiong, Philip L. Lorenzi, C. Bonini, M. Anna Zal, Bradford G Hoffman, Timmothy Heffernan, Virginia Giuliani, Collene R. Jeter, Yonathan Lissanu, G. Genovese, M. D. Pilato, A. Viale, Giulio F. Draetta
          </td>
          <td>2024-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ffead5506fa79af05a88145b919b97ef5788bd8" target='_blank'>
              The genetic mechanism of B chromosome drive in rye illuminated by chromosome-scale assembly
              </a>
            </td>
          <td>
            Jianyong Chen, Jan Bartoš, A. Boudichevskaia, Anna Voigt, M. T. Rabanus-Wallace, Steven Dreissig, Zuzana Tulpová, H. Šimková, J. Macas, Gihwan Kim, Jonas Buhl, Katharina Bürstenbinder, F. Blattner, Jörg Fuchs, Thomas Schmutzer, A. Himmelbach, V. Schubert, Andreas Houben
          </td>
          <td>2024-11-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy nearly half of the human genome and play diverse biological roles. Despite their abundance, the extent to which TEs contribute to three-dimensional (3D) genome structure remains unclear. To investigate this, we developed te_hic, a computational pipeline that integrates genomic data with chromatin interactions. Our analysis reveals that TE sequences are responsible for 3D genome structure in interphase nuclei. This phenomenon is mediated by the recruitment of specific transcription factors to TEs, which enables both the formation and disruption of chromatin contacts. We computationally identified known contact-forming proteins (CTCF, RAD21, SMC3) and breaking proteins (RNF2), as well as novel candidate contact-formers (SMARCA4, MAFK). Knockdown of the predicted contact-formers SMARCA4 and MAFK decreased contacts and loops at and between TEs. Notably, SMARCA4 knockdown selectively reduced short-range contacts, highlighting its role in maintaining 3D genome structure mediated by TE binding. Overall, our findings demonstrate that TEs are crucial determinants of 3D genome organization in mammalian cells. Key findings TEs determine 48% of the 3D genome structure alone, and 70% of heterotypic contacts. A/B compartments, TADs, and loop signatures can be retrieved using TE-mapped reads only. TFs can be divided into contact formers and breakers at TEs. SMARCA4 and MAFK build chromatin contacts between TE sequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d79bb0690215dabbc840c511519efb306a3da77" target='_blank'>
              Dissecting the Contribution of Transposable Elements to Interphase Chromosome Structure
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhouqi Huang, Li Sun, Xuemeng Zhou, Jiangping He, Alexander Strunnikov, Andrew P. Hutchins
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="R-loops are triple-stranded nucleic acid structures that occur when newly synthesized single-stranded RNA anneals to duplex DNA upon the collision of replication forks with transcription complexes. These RNA–DNA hybrids facilitate several transcriptional processes in the cell and have been described extensively in the literature. Recently, evidence has emerged that R-loops are key regulators of DNA tumor virus transcription and the replication of their lifecycle. Studies have demonstrated that R-loops on the Human Papillomavirus (HPV) genome must be resolved to maintain genome maintenance and avoid viral integration, a hallmark of HPV cancers. For Epstein–Barr virus (EBV), R-loops are formed at the oriLyt to establish lytic replication. Structural maintenance of chromosome proteins 5/6 (SMC5/6) bind to these viral R-loops to repress EBV lytic replication. Most viruses in the herpesvirales order, such as KSHV, contain R-loop-forming sequences. In this perspective, we will describe the current, although limited, literature demonstrating the importance of RNA–DNA hybrids to regulate DNA virus transcription. We will also detail potential new areas of R-loop research and how these viruses can be used as tools to study the growing field of R-loops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5eca0edc04ce77a226ea03abe7a598c769c1488" target='_blank'>
              Regulation of R-Loops in DNA Tumor Viruses
              </a>
            </td>
          <td>
            Anaiya Crowner, Keely Smith, M. DeSmet
          </td>
          <td>2024-10-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA-dependent protein kinase (DNA-PK) is a key effector of non-homologous end joining (NHEJ)-mediated double-strand break (DSB) repair. Since its identification, a substantial body of evidence has demonstrated that DNA-PK is frequently overexpressed in cancer, plays a critical role in tumor development and progression, and is associated with poor prognosis in cancer patients. Recent studies have also uncovered novel functions of DNA-PK, shifting the paradigm of the role of DNA-PK in oncogenesis and renewing interest in targeting DNA-PK for cancer therapy. To gain genetic insight into the cellular pathways requiring DNA-PK activity, we used a CRISPR/Cas9 screen to identify genes in which defects cause hypersensitivity to DNA-PK inhibitors. We identified over one hundred genes involved in DNA replication, cell cycle regulation, and RNA processing that promoted cell survival when DNA-PK kinase activity was suppressed. This gene set will be useful for characterizing novel biological processes that require DNA-PK activity and identifying predictive biomarkers of response to DNA-PK inhibition in the clinic. We also validated several genes from this set and reported previously undescribed genes that modulate the response to DNA-PK inhibitors. In particular, we found that compromising the mRNA splicing pathway led to marked hypersensitivity to DNA-PK inhibition, providing a possible rationale for the combined use of splicing inhibitors and DNA-PK inhibitors for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b81329cc49dba729d43bb24bfcaa92943d47c86" target='_blank'>
              Disruption of RNA Splicing Increases Vulnerability of Cells to DNA-PK Inhibitors
              </a>
            </td>
          <td>
            A. P. Kovina, Artem V. Luzhin, V. Tatarskiy, Dmitry A. Deriglazov, Natalia V. Petrova, N. V. Petrova, L. Kondratyeva, O. Kantidze, S. V. Razin, A. K. Velichko
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0970058b2af0e5ea082fb819477f1708b4c4db9a" target='_blank'>
              DNA demethylation triggers cell free DNA release in colorectal cancer cells
              </a>
            </td>
          <td>
            Valeria Pessei, M. Macagno, Elisa Mariella, Noemi Congiusta, V. Battaglieri, Paolo Battuello, Marco Viviani, Giulia Gionfriddo, S. Lamba, Annalisa Lorenzato, D. Oddo, Fariha Idrees, Alessandro Cavaliere, A. Bartolini, S. Guarrera, M. Linnebacher, Laura Monteonofrio, Luca Cardone, Michele Milella, A. Bertotti, Silvia Soddu, Elena Grassi, G. Crisafulli, A. Bardelli, L. Barault, F. Di Nicolantonio
          </td>
          <td>2024-10-09</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d68a011913dc4e071fed1241ce9416cb60688a" target='_blank'>
              A novel ITGB8 transcript variant sustains ovarian cancer cell survival through genomic instability and altered ploidy on a mutant p53 background
              </a>
            </td>
          <td>
            Aravindan Narayanan, Ankita S. More, Muskan Talreja, Avinash M. Mali, Sannannagari Boya Vinay, S. Bapat
          </td>
          <td>2024-11-06</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Satellite DNA makes up ∼11% of the mouse genome, predominantly located in centromeric and pericentric regions, which are crucial for chromosome segregation. While comprehensive assemblies of these regions have been established in the human genome, they are still lacking in the mouse genome. In this study, we used PacBio long-read sequencing, CUT&RUN sequencing, DNA methylation analysis, and RNA sequencing to generate genomic and epigenomic maps of these regions. We find that centromeric regions are primarily occupied by 120-mer Minor satellites, with other Minor Satellite length variants, 112-mers and 112-64-dimers, localized at centromere-pericentric junctions. Pericentromeric regions are mainly composed of homogeneous Major satellites, while pericentric-chromosomal junctions contain a higher density of divergent satellites. Additionally, the density of non-satellite repeats increases progressively from centromeres to pericentromeres, and further toward chromosomal arm junctions. We found that 120-mer Minor satellites in the core centromere are highly enriched with CENP-A, while the 112-mers and 112-64-dimers show lower CENP-A levels. Homogeneous Major satellites are more enriched with H3K9me3 heterochromatin, whereas divergent Major satellites are preferentially associated with H3K27me3. Furthermore, DNA methylation levels are lower in centromeres compared to pericentric regions. We also observed that only a small subset of satellites is transcribed into RNA, particularly regions exhibiting lower DNA methylation density. Our comprehensive assembly and characterization of the genomic and epigenomic landscape of mouse centromeric and pericentric regions have major implications for satellite biology and ongoing mouse telomere-to-telomere (T2T) assembly efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951d9f9f9b8e262cc6e6811c22c27c69e6bb984f" target='_blank'>
              Genomic and epigenomic maps of mouse centromeres and pericentromeres
              </a>
            </td>
          <td>
            Gitika Chaudhry, Jinguye Chen, Lucy Snipes, Smriti Bahl, Jenika Packiaraj, Xuan Lin, Jitendra Thakur
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios, Lei Peng, Luojia Yang
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214031bb67ad0add97aa1077e44bc79d2bc07460" target='_blank'>
              Genomic landscape of adult testicular germ cell tumours in the 100,000 Genomes Project
              </a>
            </td>
          <td>
            Máire Ní Leathlobhair, A. Frangou, B. Kinnersley, A. Cornish, D. Chubb, E. Lakatos, Prabhu Arumugam, Andreas J. Gruber, P. Law, Avraam Tapinos, G. M. Jakobsdottir, I. Peneva, A. Sahli, Evie M Smyth, Richard Y Ball, R. Sylva, K. Benes, Dan Stark, Robin J Young, Alexander T J Lee, Vincent Wolverson, R. Houlston, A. Sosinsky, A. Protheroe, Matthew J. Murray, David C Wedge, C. Verrill, 
          </td>
          <td>2024-10-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits widespread epigenetic alterations, yet their impact on cis-regulatory elements (CREs) and retrotransposons remains poorly understood. Here, we present an integrated epigenomic and transcriptomic analysis of HCC tumors and matched tumor-adjacent normal tissues. We identified extensive DNA hypomethylation coupled with changes in histone modifications at partially methylated domains, CREs, and retrotransposons. These epigenetic aberrations were associated with dysregulated expression of genes involved in cell cycle regulation, immune response, and extracellular matrix organization. Notably, our findings revealed a novel mechanism for the transcriptional dysregulation of GPC3, a key HCC biomarker and immunotherapeutic target. We observed that GPC3 upregulation is driven by both the reactivation of a fetal liver super enhancer and hypomethylation of GPC3-associated CpG islands. Moreover, we found that DNA hypomethylation-driven aberrant expression of retrotransposons carries prognostic significance in HCC. Patients with high expression of a long non-coding RNA driven by a HERVE-int element exhibited more aggressive tumors, poorer clinical outcomes, and molecular features associated with favorable immunotherapy response. Together, our study provides a comprehensive resource for understanding the role of epigenetic dysregulation in HCC and identifies retrotransposon-associated transcripts as potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce7deec20d100e57e2cf628c181e276b8e32f748" target='_blank'>
              Epigenomic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potentials
              </a>
            </td>
          <td>
            Clooney C. Y. Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y. J. Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B. S. Lai, K. Yip, Alfred S. Cheng, Danny Leung
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Caspase activated DNase (CAD) induced DNA breaks promote cell differentiation and therapy-induced cancer cell resistance. CAD targeting activity is assumed to be unique to each condition, as differentiation and cancer genesis are divergent cell fates. Here, we made the surprising discovery that a subset of CAD-bound targets in differentiating muscle cells are the same genes involved in the genesis of cancer-causing translocations. In muscle cells, a prominent CAD-bound gene pair is Pax7 and Foxo1a, the mismatched reciprocal loci that give rise to alveolar rhabdomyosarcoma. We show that CAD-targeted breaks in the Pax7 gene are physiologic to reduce Pax7 expression, a prerequisite for muscle cell differentiation. A cohort of these CAD gene targets are also conserved in early differentiating T cells and include genes that spur leukemia/lymphoma translocations. Our results suggest the CAD targeting of translocation prone oncogenic genes is non-pathologic biology and aligns with initiation of cell fate transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95535f8b5c0fda938ec705d36c58cef6067d4ac1" target='_blank'>
              Caspase-Activated DNase localizes to cancer causing translocation breakpoints during cell differentiation
              </a>
            </td>
          <td>
            Dalal Alsowaida, Brian D. Larsen, Sarah Hachmer, M. Azimi, Eric Arezza, Steve Brunette, Steven Tur, C. Palii, Bassam Albraidy, Claus S. Sorensen, Marjorie Brand, F. J. Dilworth, L. Megeney, Senior Scientist
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract N6-Methyladenosine (m6A) is the most abundant internal modification in mRNAs. Despite accumulating evidence for the profound impact of m6A on cancer biology, there are conflicting reports that alterations in genes encoding the m6A machinery proteins can either promote or suppress cancer, even in the same tumor type. Using data from The Cancer Genome Atlas, we performed a pan-cancer investigation of 15 m6A core factors in nearly 10000 samples from 31 tumor types to reveal underlying cross-tumor patterns. Altered expression, largely driven by copy number variations at the chromosome arm level, results in the most common mode of dysregulation of these factors. YTHDF1, YTHDF2, YTHDF3 and VIRMA are the most frequently altered factors and the only ones to be uniquely altered when tumors are grouped according to the expression pattern of the m6A factors. These genes are also the only ones with coherent, pan-cancer predictive power for progression-free survival. On the contrary, METTL3, the most intensively studied m6A factor as a cancer target, shows much lower levels of alteration and no predictive power for patient survival. Therefore, we propose the non-enzymatic YTHDF and VIRMA genes as preferred subjects to dissect the role of m6A in cancer and as priority cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cb34b460873bf086e54bf45d997fe617e03818" target='_blank'>
              The three YTHDF paralogs and VIRMA are strong cross-histotype tumor driver candidates among m6A core genes
              </a>
            </td>
          <td>
            E. Destefanis, D. Sighel, Davide Dalfovo, Riccardo Gilmozzi, F. Broso, Andrea Cappannini, Janusz M. Bujnicki, Alessandro Romanel, E. Dassi, A. Quattrone
          </td>
          <td>2024-10-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Aberrant meiosis in plants often leads to aneuploidy, genetic instability, and sterility. This can occur due to several factors, including chromosome misalignment, defective synapsis or environmental factors that may result in unusual genetic combinations in the offsprings. Unusual chromosome behavior during male meiosis in Humulus lupulus is linked to irregular chromosome segregation and genome instability. However, the origin of meiotic instability remains unclear. We analyzed the centromeric landscape of H. lupulus to determine its role in aberrant chromosomal segregation during cell division. Using a combination of bioinformatic, molecular and cytogenetic approaches, we identified new centromeric repeats and revealed two types of centromeric organizations. Cytogenetic localization on metaphase chromosomes confirmed the genomic distribution of major repeat arrays and revealed unique features that contribute to aberrant segregation. Two centromeric types are composed of the major repeats SaazCEN and SaazCRM1 which are further accompanied by chromosome-specific centromeric satellites, Saaz40, Saaz293, Saaz85, and HuluTR120. Chromosome 2 displays unbalanced segregation during the cell division, implicating an important role for its centromere structure in segregation patterns. Moreover, Saaz293 is a new marker for studying aneuploidy in hop. Our findings provide new insights on chromosome segregation in hop and highlight the diversity and complexity of the centromere organization in H. lupulus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc46ea72029acc6577e0f7b5a251b7322d20a30" target='_blank'>
              Centromeric repeat diversity underlies non-Mendelian segregation pattern in hop (Humulus lupulus)
              </a>
            </td>
          <td>
            Lucie Horáková, R. Čegan, Pavel Jedlička, Pavla Navrátilová, Hiroyuki Tanaka, Atsushi Toyoda, Takehiko Itoh, Takashi Akagi, Eiichiro Ono, Vojtěch Hudzieczek, Josef Patzak, Jan Šafář, R. Hobza, Václav Bačovský
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). Using a functional genome-wide CRISPRi screen in PDAC cells, we identified activators of selective MYC translation through its 5’ untranslated region (5’UTR) and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. Among these RBPs, our top hit was RBM42, which is highly expressed in PDAC and predicts poor survival. Combining polysome sequencing and CLIP-seq analyses, we find that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing IP-mass spectrometry analysis, we find that RMB42 is a novel ribosome-associated protein (RAP). Using DMS-Seq and mutagenesis analysis, we show that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex. Importantly, RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/168afbf485caea55a52d0fa0de95551cbcbbc5ea" target='_blank'>
              Functional screen for mediators of onco-mRNA translation specificity
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="PURPOSE Nuclear protein in testis carcinoma (NC) is an underdiagnosed and aggressive squamous/poorly differentiated cancer characterized by rearrangement of the gene NUTM1 on chromosome 15q14. Co-occurring alternations have not been fully characterized. METHODS We analyzed the genomic and immune landscape of 54 cases of NC that underwent DNA- and RNA-based NGS sequencing (Caris). RESULTS While NC is driven by NUTM1 fusion oncoproteins, co-occurring DNA mutations in epigenetic or cell cycle pathways were observed in 26% of cases. There was no significant difference between the fusion partner of NUTM1 and co-occurring gene mutations. RNA sequencing analysis showed increased MYC pathway activity in NC compared with head and neck squamous cell carcinoma (HNSCC) and lung squamous cell carcinoma (LUSC), which is consistent with the known pathophysiology of NC. Characterization of the NC tumor microenvironment using RNA sequencing revealed significantly lower immune cell infiltration compared with HNSCC and LUSC. NC was 10× higher in patients with HNSCC and LUSC younger than 50 years than in those older than 70 years. CONCLUSION To our knowledge, this is the first series of NC profiled broadly at the DNA and RNA level. We observed fewer intratumoral immune cells by RNA sequencing, which may be associated with anecdotal data of lack of immunotherapy benefit in NC. High MYC pathway activity in NC supports ongoing trials targeting MYC suppression. The incidence of NC among patients younger than 50 years with LUSC/HNSCC supports testing for NC in these patients. The prognosis of NCs remains dismal, and future studies should focus on improving the response to immunotherapy and targeting MYC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c7a9669ff7523a745b866e303487bfb7d01d6d" target='_blank'>
              Multiomic Characterization and Molecular Profiling of Nuclear Protein in Testis Carcinoma
              </a>
            </td>
          <td>
            G. Kroening, Jia Luo, Mark G Evans, T. Adeyelu, S. Ou, Z. Arter, Trisha M Wise-Draper, A. Sukari, Asfar S Azmi, David R Braxton, A. Elliott, David A Bryant, M. Oberley, Chul Kim, Geoffrey I. Shapiro, Christopher A French, M. Nagasaka
          </td>
          <td>2024-10-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Mutations in genes encoding critical epigenetic regulators are frequently noted in bladder cancer, however, the impact of these mutations on therapeutic efficacy is unclear. One of the most common driver mutations in bladder cancer occurs in the KDM6A gene, which encodes a histone demethylase that promotes gene transcription. Retrospective analyses of patients with bladder cancer demonstrated that KDM6A mutations correlate with improved overall survival (OS) with immune checkpoint therapy (ICT), while they are associated with lower OS in patients undergoing cisplatin-based chemotherapy. Mechanistic studies utilizing CRISPR-Cas9 mediated deletion of Kdm6a showed reduced expression of DNA mismatch repair (MMR) and DNA double-stranded base repair (DSBR) genes in tumor cells with improved response to anti-PD-1 therapy and attenuated sensitivity to cisplatin-based chemotherapy in preclinical models of bladder cancer. Additionally, the loss of Kdm6a-mediated reduction in glycolysis and intratumoral lactate accumulation impaired histone 3 lysine 9 lactylation (H3K9la) and histone 3 lysine 18 lactylation (H3K18la) in Tregs with concurrent decrease in the expression of key genes including Foxp3, Tgfb and Pdcd1 and their immune-suppressive function. Further, reduced expansion of PD-1hi Tregs improved the ratio of cytotoxic T cells to Tregs and response to anti-PD-1 therapy in Kdm6a deficient tumor-bearing mice. Collectively, this study provided key insights into the role of KDM6A-mediated epigenetic regulation of DNA repair and metabolic reprogramming which potentially govern response to chemotherapy and ICT thus highlighting the utility of KDM6A mutation status for patient stratification and development of personalized treatment algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825bb07017841ff9cc12c29d05779f53d82d0047" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming differentially regulates responses to immune checkpoint therapy and chemotherapy in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, D. Raychaudhuri, Bidisha Chakraborty, Swadhin Meher, Aminah J. Tannir, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, P. Pilié, Sangeeta Goswami
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Optimizing prevention and early detection of cancer requires understanding the number, types and timing of driver mutations. To quantify this, we exploited the elevated cancer incidence and mutation rates in germline BRCA1 and BRCA2 (gBRCA1/2) carriers. Using novel statistical models, we identify genomic deletions as the likely rate-limiting mutational processes, with 1-3 deletions required to initiate breast and ovarian tumors. gBRCA1/2-driven hereditary and sporadic tumors undergo convergent evolution to develop a similar set of driver deletions, and deletions explain the elevated cancer risk of gBRCA1/2-carriers. Orthogonal mutation timing analysis identifies deletions of chromosome 17 and 13q as early, recurrent events. Single-cell analyses confirmed deletion rate differences in gBRCA1/2 vs. non-carrier tumors as well as cells engineered to harbor gBRCA1/2. The centrality of deletion-associated chromosomal instability to tumorigenesis shapes interpretation of the somatic evolution of non-malignant tissue and guides strategies for precision prevention and early detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/142f79a57f8e5624ca9c9770400c8298f772b241" target='_blank'>
              Deletions Rate-Limit Breast and Ovarian Cancer Initiation
              </a>
            </td>
          <td>
            Kathleen E. Houlahan, Mahad Bihie, Julián Grandvallet Contreras, Daniel J. Fulop, Gonzalo Lopez, Hsin-Hsiung Huang, P. Van Loo, Christina Curtis, P. Boutros, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric malignancy. Based on gene expression profiling, B-ALL can be classified into distinct transcriptional subtypes with differing disease outcomes. Many of these transcriptional subtypes are defined by mutations in transcription factors and chromatin-modifying enzymes, but how such diverse mutations lead to distinct transcriptional subtypes remains unclear. To illuminate the chromatin regulatory landscape in B-ALL, we analyzed 3D genome organization, open chromatin, and gene expression in 53 primary patient samples. At the level of 3D genome organization, we identified chromatin interactions that vary across transcriptional subtypes. These sites of variable 3D chromatin interactions correlate with local gene expression changes and are enriched for core drivers of B-ALL observed in genome-wide CRISPR knock-out screens. Sites of variable 3D genome interactions are frequently shared across multiple transcriptional subtypes and are enriched for open chromatin sites found in normal B-cell development but repressed in mature B-cells. Within an individual patient sample, the chromatin landscape can resemble progenitor chromatin states at some loci and mature B-cell chromatin at others, suggesting that the chromatin in B-ALL patient tumor cells is in a partially arrested immature state. By analyzing transcriptomic data from large cohorts of B-ALL patients, we identify gene expression programs that are shared across transcriptional subtypes, associated with B-cell developmental stages, and predictive of patient survival. In combination, these results show that the 3D genome organization of B-ALL reflects B-cell developmental stages and helps illustrate how B-cell developmental arrest interacts with transcriptional subtypes to drive B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1e7461bf425fea64b8e5097538780b0bffaf" target='_blank'>
              Heterogeneity in chromatin structure drives core regulatory pathways in B-cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Arko Sen, Zhichao Xu, Sélène T. Tyndale, Jean Yasis, Chae Yun Cho, Rosalind Bump, Sahaana Chandran, Linda Luo, Yi Fu, Lillian Kay Petersen, Max Shokhirev, D. J. Kuo, G. McVicker, Jesse R. Dixon
          </td>
          <td>2024-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9a6a91b5f3ac052c57405c773236aad0d8d4b4" target='_blank'>
              Recent insights into the causes and consequences of chromosome mis-segregation.
              </a>
            </td>
          <td>
            Romain Devillers, Alexsandro dos Santos, Quentin Destombes, Mathieu Laplante, S. Elowe
          </td>
          <td>2024-09-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Epigenetic mechanisms often fuel the quick evolution of cancer cells from normal cells. Mutations or aberrant expressions in the enzymes of DNA methylation, histone post-translational modifications, and chromatin remodellers have been extensively investigated in cancer pathogenesis; however, cancer-associated histone mutants have gained momentum in recent decades. Next-generation sequencing of cancer cells has identified somatic recurrent mutations in all the histones (H3, H4, H2A, H2B, and H1) with different frequencies for various tumour types. Importantly, the well-characterised H3K27M, H3G34R/V, and H3K36M mutations are termed as oncohistone mutants because of their wide roles, from defects in cellular differentiation, transcriptional dysregulation, and perturbed epigenomic profiles to genomic instabilities. Mechanistically, these histone mutants impart their effects on histone modifications and/or on irregular distributions of chromatin complexes. Recent studies have identified the crucial roles of the H3K27M and H3G34R/V mutants in the DNA damage response pathway, but their impacts on chemotherapy and tumour progression remain elusive. In this review, we summarise the recent developments in their functions toward genomic instabilities and tumour progression. Finally, we discuss how such a mechanistic understanding can be harnessed toward the potential treatment of tumours harbouring the H3K27M, H3G34R/V, and H3K36M mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d93aa8724101a0a6b04c672de1eafdda67a5e3" target='_blank'>
              Emerging roles of cancer-associated histone mutations in genomic instabilities
              </a>
            </td>
          <td>
            Priyanka Yadav, Ronit Jain, Rajesh Kumar Yadav
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="To achieve replicative immortality, cancer cells must activate telomere maintenance mechanisms. In 10-15% of cancers, this is enabled by recombination-based alternative lengthening of telomeres pathways (ALT). ALT cells display several hallmarks including heterogeneous telomere length, extrachromosomal telomeric repeats and ALT-associated PML bodies. ALT cells also have high telomeric replication stress (RS) enhanced by fork-stalling structures (R-loops, G4s) and altered chromatin states. In ALT cells, telomeric RS promotes telomere elongation but above a certain threshold becomes detrimental to cell survival. Manipulating RS at telomeres has thus been proposed as a therapeutic strategy against ALT cancers. Through analysis of genome-wide CRISPR fitness screens, we identified ALT-specific vulnerabilities and describe here our characterization of the roles of SUB1, a ssDNA-binding protein, as a novel regulator of telomere stability. SUB1 depletion further increases RS at ALT telomeres, profoundly impairing ALT cell growth without impacting telomerase-positive cancer cells. During RS, SUB1 is recruited to stalled forks and ALT telomeres via its ssDNA-binding domain. This recruitment is potentiated by RPA depletion, suggesting that these factors may compete for ssDNA. The viability of ALT cells and their resilience towards RS also requires ssDNA-binding by SUB1. SUB1 depletion accelerates cell death induced by FANCM depletion, triggering unsustainable levels of telomeric damage specifically in ALT cells. Finally, combining SUB1 depletion with RS-inducing drugs rapidly induces replication catastrophe in ALT cells. Altogether, our work identifies SUB1 as a new ALT susceptibility with important roles in the mitigation of RS at ALT telomeres and suggests new therapeutic strategies for a host of still poorly managed cancers. Significance Statement Currently, there are few treatment options for ALT cancers with chemotherapy still occupying center stage despite often limited efficacy. ALT cancer cells experience high levels of replication stress at telomeres and its enhancement (e.g. via ATR inhibition) is a promising therapeutic strategy. Sensitivity to ATR inhibition varies amongst ALT cell lines/tumors warranting the development of additional ways to modulate telomeric replication stress. Here we identify SUB1, a single-stranded DNA-binding protein, as a vulnerability of ALT cells. SUB1 localizes to ALT telomeres and mitigates deleterious replication stress. SUB1 depletion synergizes with ATR inhibition and FANCM downregulation suggesting that co-targeting SUB1 with other regulators of replication stress at telomeres may kill ALT cancer cells more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12dc8d86a154542f040802530b59736f3f3ac6dd" target='_blank'>
              The Single-Stranded DNA-Binding Factor SUB1/PC4 Alleviates Replication Stress at Telomeres and is a Vulnerability of ALT Cancer Cells
              </a>
            </td>
          <td>
            Jean-Christophe Dubois, Erin Bonnell, Julie Frion, Samuel Zimmer, Muhammad Riaz Khan, Gabriela M. Teplitz, Lisa Casimir, Élie Méthot, Amélie Filion, Mouhamed Idrissou, Pierre-Étienne Jacques, R. Wellinger, Alexandre Maréchal
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Understanding the role of repetitive elements (REs) in cancer development is crucial for identifying novel biomarkers and therapeutic targets. This study investigated the locus-specific dysregulation of REs, including differential expression and methylation of REs, across 12 TCGA cancer types stratified by their genomic context (i.e., genic and intergenic REs). We found uniquely dysregulated genic REs co-regulated with their corresponding transcripts and associated with distinct biological functions in different cancer types. Uniquely dysregulated intergenic REs were identified in each cancer type and used to cluster different sample types. Recurrently dysregulated REs were identified in several cancer types, with genes associated with up-regulated genic REs involved in cell cycle processes and those associated with down-regulated REs involved in the extracellular matrix. Interestingly, 4 out of 5 REs consistently down-regulated in all 12 cancer types were located in the intronic region of the TMEM252, a recently discovered tumor suppressor gene. TMEM252 expression was also down-regulated in 10 of 12 cancer types, suggesting its potential importance across a wide range of cancer types. With the corresponding DNA methylation array data, we found a higher prevalence of hypo-methylated REs in most cancer types (10 out of 12). Despite the slight overlaps between differentially expressed REs and differentially methylated REs, we showed that methylation of locus-specific REs negatively correlates with their expression in some cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58fdef1beb592eec1a6ab24f91cfe621cf7209ac" target='_blank'>
              Dysregulation of locus-specific repetitive elements in TCGA pan-cancers
              </a>
            </td>
          <td>
            Chao Wang, Chun Liang
          </td>
          <td>2024-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107c0d6e681a7d833da603e062a1adf5d877660c" target='_blank'>
              MLL oncoprotein levels influence leukemia lineage identities
              </a>
            </td>
          <td>
            D. Janssens, Melodie Duran, Dominik J. Otto, Weifang Wu, Yiling Xu, Danielle C Kirkey, C. Mullighan, Joanna S Yi, S. Meshinchi, Jay Sarthy, K. Ahmad, S. Henikoff
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Epimutations modify gene expression and lead to phenotypic variation while the encoding DNA sequence remains unchanged. Epimutations mediated by RNA interference (RNAi) and/or chromatin modifications can confer antifungal drug resistance and may impact virulence traits in fungi (1–5). However, whether these epigenetic modifications can be transmitted across generations following sexual reproduction was unclear. This study demonstrates that RNAi epimutations conferring antifungal drug resistance are transgenerationally inherited in the human fungal pathogen Mucor circinelloides. Our research revealed that RNAi-based antifungal resistance follows a DNA sequence-independent, non-Mendelian inheritance pattern. Small RNAs (sRNAs) are the exclusive determinants of inheritance, transmitting drug resistance independently of other known repressive epigenetic modifications. Unique sRNA signature patterns can be traced through inheritance from parent to progeny, further supporting RNA as an alternative molecule for transmitting information across generations. This study marks a significant advance in understanding epigenetic inheritance, highlighting RNAi-exclusive epimutations as a widespread phenomenon in the pathogenic M. circinelloides species complex. Understanding how epimutations occur, propagate, and confer resistance may enable their detection in other eukaryotic pathogens, provide solutions for challenges posed by rising antimicrobial drug resistance (AMR), and also advance research on phenotypic adaptability and its evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7aab0fd6f057abc3a0de9ddf4f9920c418d202" target='_blank'>
              RNAi epimutations conferring antifungal drug resistance are inheritable
              </a>
            </td>
          <td>
            Carlos Pérez-Arques, M. Navarro-Mendoza, Ziyan Xu, Grit Walther, Joseph Heitman
          </td>
          <td>2024-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Amongst them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8d12d939b21bc8eddc64b81e916d3c7019a271" target='_blank'>
              TopBP1 biomolecular condensates: a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA replication often encounters obstacles like the stalled transcription machinery and R-loops. While ribonucleases and DNA-RNA helicases can resolve these structures, the role of chromatin remodelers remains understudied. Through a series of in vitro and in vivo experiments, we show that the chromatin remodeler SMARCAD1, which associates with active replication forks, is crucial for resolving nearby R-loops to maintain fork stability. SMARCAD1 directly binds R-loops via its ATPase domain and associates with the replisome through its N-terminus region. Both interactions are critical for resolving R-loops within cells. Genome-wide assays reveal that cells expressing mutant SMARCAD1 accumulate significantly more R-loops than wild-type cells, particularly in regions distinct from known fork blockage-prone sites. These R-loop-enriched regions in SMARCAD1 mutants also exhibit increased mutagenesis in germline tumors, suggesting they are mutation hotspots in cancer. Therefore, SMARCAD1 acts as an R-loop sensor and resolvase at actively progressing forks, maintaining genome stability and preventing tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2647786c65a810f532e08e30214a54e76697fe" target='_blank'>
              SMARCAD1 Regulates R-Loops at Active Replication Forks Linked to Cancer Mutation Hotspots
              </a>
            </td>
          <td>
            Sidrit Uruci, N. M. Hoitsma, María E. Soler-Oliva, Aleix Bayona-Feliú, Vincent Gaggioli, María L. García-Rubio, Calvin Lo, Collin Bakker, Jessica Marinello, E. Manolika, Giovanni Capranico, M. S. Luijsterburg, Karolin Luger, A. Aguilera, Nitika Taneja
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Purine stretches, sequences of adenine (A) and guanine (G) in DNA, play critical roles in binding regulatory protein factors and influence gene expression by affecting DNA folding. This study investigates the relationship between purine stretches and cancer development, considering the aromaticity of purines, quantified by methods like Hückel’s rule and NICS calculations, and the importance of the flanking sequence context. A pronounced avoidance of long purine stretches by typical cancer mutations was observed in public data on the intergenic regions of cancer patients, suggesting a role of intergenic sequences in chromatin reorganization and gene regulation. A statistically significant shortening of purine stretches in cancerous tumors (p value < 0.0001) was found. The insights into the aromatic nature of purines and their stacking energies explain the role of purine stretches in DNA structure, contributing to their role in cancer progression. This research lays the groundwork for understanding the nature of purine stretches, emphasizing their importance in gene regulation and chromatin restructuring, and offers potential avenues for novel cancer therapies and insights into cancer etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06dba24a26acdb2675c0db386c20ae4052e4546a" target='_blank'>
              The Avoidance of Purine Stretches by Cancer Mutations
              </a>
            </td>
          <td>
            Aleksandr V. Vikhorev, Ivan V. Savelev, O. O. Polesskaya, Michael M. Rempel, Richard A. Miller, A. Vetcher, Max Myakishev-Rempel
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6d816344be553ab4e2f5399f5fa15a18a391b" target='_blank'>
              High prevalence of chromosome 17 in breast cancer micronuclei: a means to get rid of tumor suppressors?
              </a>
            </td>
          <td>
            Laxmi Kumari, S. Sreedharanunni, Divya Dahiya, Pranab Dey, Alka Bhatia
          </td>
          <td>2024-10-22</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3095632bcf16399c0c6ee358ffba9b30df1dc77" target='_blank'>
              Integrative spatial and genomic analysis of tumor heterogeneity with Tumoroscope
              </a>
            </td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Alicja Ra Czkowska, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, Johan Hartman, Dominika Nowis, A. Carbone, Jens Lagergren, E. Szczurek
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of CTC-IGC was significantly associated with poorer progression-free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13c77b6b25150646e97a913d6046b3f971cd07d" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            James B. Hicks, Michael J. Schmidt, Amin Nahgdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, Isla Garraway, Michael Lewis, Ana Aparicio, A. Zurita, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah Amend
          </td>
          <td>2024-10-14</td>
          <td>Research Square</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer, and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. Through analysis of genome sequencing from primary HGSOC, we observed an association between high levels of APOBEC3 mutagenesis and poor overall survival. We experimentally addressed this correlation by modeling A3A activity in HGSOC cell lines and mouse models which resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal-transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, and providing a mechanism for their increased metastatic potential. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious impact of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b59ee718cc58bb611f3f35bfae513ef8466daf1" target='_blank'>
              APOBEC3A drives metastasis of high-grade serous ovarian cancer by altering epithelial-to-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J. Bednarski, Ronny Drapkin, Mary Mullen, Abby M. Green
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70c12109946fe269f22601a33ca821cdb6d42676" target='_blank'>
              Copy number variation heterogeneity reveals biological inconsistency in hierarchical cancer classifications
              </a>
            </td>
          <td>
            Ziying Yang, Paula Carrio-Cordo, Michael Baudis
          </td>
          <td>2024-11-06</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f93c4dc38f47f9f9a8e9e5ea8975939c269a20" target='_blank'>
              GLTSCR1 deficiency promotes colorectal cancer development through regulating non-homologous end joining.
              </a>
            </td>
          <td>
            Fengyan Han, Xiaoxu Zhou, Lu Liu, Beibei Yang, Pengyuan Liu, Enping Xu, Zhe Tang, Honghe Zhang
          </td>
          <td>2024-10-11</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary When oncogenes, which are genes that can cause cancer, become active, they disrupt normal cell processes, especially DNA replication. This disruption is known as replication stress (RS), in which the DNA copying process is stalled or damaged. To prevent cancer, cells usually have a defense system called the DNA damage response (DDR), which can prevent damaged cells from turning into cancer. However, some cells manage to survive this stress by developing replication stress tolerance (RST). These cells can continue to grow, leading to genomic instability (GIN), which is a key feature of cancer. GIN causes various genetic changes that make cells more likely to become cancerous and more difficult to treat. This review explains how oncogenes cause RS and how cells cope with it, leading to the development of cancer. Understanding these processes can help in developing new cancer treatments. Abstract Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Nearly identical, repetitive elements in the genome contribute to the variability in genetic inheritance patterns, particularly in regions like the RCCX locus, where such repeats can lead to structural variations. In addition, during the formation of gametes as a result of meiosis, variants of loci with repetitive elements that do not code for the required proteins may occur. As a result, an individual with certain genetic rearrangements in this region may have an increased risk of developing a congenital disorder, particularly in cases where the non-functional allele is inherited dominantly. At the same time, there is still no routine or generally recognized diagnostic method to determine the sequence of the repetitive fragments. The functionally important RCCX locus consists of such repetitive fragments. The available knowledge about the genomic variants of the RCCX locus is fragmented, as there is no standardized method to determine its structure. It should be noted that in some structural variants of the RCCX locus, the sequence of protein-coding genes is disrupted, leading to the development of diseases such as congenital adrenal hyperplasia (CAH). Although genetic testing is generally accepted as a gold standard for CAH diagnosis, there are a myriad of strategies on which exact methods to use and in which order. The reason for this inconsistency lies in the complexity of the RCCX locus and the fact that each patient or carrier may have a highly individualized mutation or combination thereof. In this review, we have discussed all known methods that can be used to study the structure of the RCCX locus. As a result, optimal approaches are proposed for the diagnosis of the most common disease caused by lesions in the RCCX–CAH due to CYP21A2 deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7835d159d0ca81c4a93ae9b32e5d3579ffc2c9" target='_blank'>
              Genomic complexity and clinical significance of the RCCX locus
              </a>
            </td>
          <td>
            Vladimir V. Shiryagin, Andrey A. Devyatkin, Oleg D. Fateev, Ekaterina S. Petriaikina, Viktor P. Bogdanov, Zoia G. Antysheva, P. Volchkov, Sergey M. Yudin, M. Woroncow, V. I. Skvortsova
          </td>
          <td>2024-11-04</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome instability relies on preserving the chromatin structure, with any histone imbalances threating DNA integrity. Histone synthesis occurs in the cytoplasm, followed by a maturation process before their nuclear translocation. This maturation involves protein folding and the establishment of post-translational modifications. Disruptions in this pathway hinder chromatin assembly and contribute to genome instability. JMJD1B, a histone demethylase, not only regulates gene expression but also ensures a proper supply of histones H3 and H4 for the chromatin assembly. Reduced JMJD1B levels lead to the cytoplasmic accumulation of histones, causing defects in the chromatin assembly and resulting in DNA damage. To investigate the role of JMJD1B in regulating genome stability and the malignancy of melanoma tumors, we used a JMJD1B/KDM3B knockout in B16F10 mouse melanoma cells to perform tumorigenic and genome instability assays. Additionally, we analyzed the transcriptomic data of human cutaneous melanoma tumors. Our results show the enhanced tumorigenic properties of JMJD1B knockout melanoma cells both in vitro and in vivo. The γH2AX staining, Micrococcal Nuclease sensitivity, and comet assays demonstrated increased DNA damage and genome instability. The JMJD1B expression in human melanoma tumors correlates with a lower mutational burden and fewer oncogenic driver mutations. Our findings highlight JMJD1B’s role in maintaining genome integrity by ensuring a proper histone supply to the nucleus, expanding its function beyond gene expression regulation. JMJD1B emerges as a crucial player in preserving genome stability and the development of melanoma, with a potential role as a safeguard against oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1490db96e0fd741bd69913b1e9fe76e21ec91750" target='_blank'>
              Unraveling the Role of JMJD1B in Genome Stability and the Malignancy of Melanomas
              </a>
            </td>
          <td>
            Perla Cruz, Diego Peña-Lopez, Diego Figueroa, Isidora Riobó, Vincenzo Benedetti, Francisco Saavedra, Claudia Espinoza-Arratia, Thelma M. Escobar, A. Lladser, Alejandra Loyola
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Approximately 50% of patients diagnosed with ovarian cancer harbor tumors with mutations in BRCA1, BRCA2, or other genes involved in homologous recombination repair (HR). The presence of homologous recombination deficiency (HRD) is an approved biomarker for poly-ADP-ribose polymerase inhibitors (PARPis) as a maintenance treatment following a positive response to initial platinum-based chemotherapy. Despite this treatment option, the development of resistance to PARPis is common among recurrent disease patients, leading to a poor prognosis. In this study, we conducted a comprehensive analysis using publicly available datasets to elucidate the molecular mechanisms driving PARPi resistance in BRCA1-deficient ovarian cancer. Our findings reveal a central role for the interferon (IFN) pathway in mediating resistance in the context of BRCA1 deficiency. Through integrative bioinformatics approaches, we identified LY6E, an interferon-stimulated gene, as a key mediator of PARPi resistance, with its expression linked to an immunosuppressive tumor microenvironment (TME) encouraging tumor progression and invasion. LY6E amplification correlates with poor prognosis and increased expression of immune-related gene signatures, which is predictive of immunotherapy response. Interestingly, LY6E expression upon PARPi treatment resistance was found to be dependent on BRCA1 status. Gene expression analysis in the Orien/cBioPortal database revealed an association between LY6E and genes involved in DNA repair, such as Rad21 and PUF60, emphasizing the interplay between DNA repair pathways and immune modulation. Moreover, PUF60, Rad21, and LY6E are located on chromosome 8q24, a locus often amplified and associated with the progression of ovarian cancer. Overall, our study provides novel insights into the molecular determinants of PARPi resistance and highlights LY6E as a promising prognostic biomarker in the management of HRD ovarian cancer. Future studies are needed to fully elucidate the molecular mechanisms underlying the role of LY6E in PARPi resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4052305f8854485a3b85613abf68315aff09bfc" target='_blank'>
              Exploring Molecular Drivers of PARPi Resistance in BRCA1-Deficient Ovarian Cancer: The Role of LY6E and Immunomodulation
              </a>
            </td>
          <td>
            T. Petta, Joseph Carlson
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and show that STAG2 is uniquely tumor suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility and 3D genome conformation that impact cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relates STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin mediated mechanisms of lung tumor suppression. SUMMARY STAG2 is a frequently mutated cohesin subunit across several cancers and one of the most important functional suppressors of lung adenocarcinoma. Our findings underscore important roles of STAG2 in suppressing lung tumorigenesis and highlight a STAG2-PAXIP1/PAGR1 tumor-suppressive program that may transcend cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0708e85569c4f6a48ce19a8abe96d4d4b9af8050" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis
              </a>
            </td>
          <td>
            Emily L. Ashkin, Yuning J. Tang, Haiqing Xu, King L. Hung, J. Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to drive divergent regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under “cellular constraint”. We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and non-coding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share paradoxical evolutionary signals of positive and negative selection, suggesting a dynamic role in driving human adaptation. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0bc98afde8c329e9647a592102463c5d074b6c9" target='_blank'>
              Epigenetic constraint of cellular genomes evolutionarily links genetic variation to function
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, Fei-Fei Cheng, Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Histone cell cycle regulator A (HIRA) confers chromatin accessibility and regulates developmental hematopoiesis. But whether HIRA displays similar role in leukemia, a condition caused by abnormalities during hematopoiesis, remain elusive. Here we show that HIRA interact with heterochromatin cluster in Chronic Myeloid Leukemia, K562, cells. FRAP, FLIM-FRET and ATAC-sequencing analysis revealed increased chromatin compaction, altered spatial distribution of chromatin towards nuclear periphery and loss in chromatin accessibility at the promoter and gene bodies upon downregulation of HIRA in K562 cells. Enhanced chromatin compaction was attributed to increased histone H3K9me3 level mediated by histone methyltransferase SETDB1. Incorporation of histone H3.3 within the SETDB1 promoter in HIRA-knockdown cells induced SETDB1 expression. HIRA-SETDB1-H3K9me3 axis regulate the chromatin architecture in CML cells leading to inhibition of proliferation while induction in differentiation. We anticipate that the exploration of this axis would introduce new paradigm in understanding and targeting molecules that could influence CML disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34277340c832b2731531b1f079bf3c59dfc74d99" target='_blank'>
              HIRA-SETDB1-H3K9me3 axis regulate chromatin architecture in chronic myeloid leukemia cells
              </a>
            </td>
          <td>
            M. Shirude, Anjali Devarajan, Debasree Dutta
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7fcfc955df2c529779499d67e1343e62c090d8b" target='_blank'>
              Polyploidy mitigates the impact of DNA damage while simultaneously bearing its burden
              </a>
            </td>
          <td>
            Kazuki Hayashi, Kisara Horisaka, Yoshiyuki Harada, Yuta Ogawa, Takako Yamashita, Taku Kitano, Masahiro Wakita, T. Fukusumi, Hidenori Inohara, Eiji Hara, Tomonori Matsumoto
          </td>
          <td>2024-10-13</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Myeloid malignancies carrying somatic DNMT3A mutations (DNMT3Amut) are usually resistant to standard therapy. DNMT3Amut leukemia cells accumulate toxic DNA double strand breaks (DSBs) and collapsed replication forks, rendering them dependent on DNA damage response (DDR). DNA polymerase theta (Polθ), a key element in Polθ-mediated DNA end-joining (TMEJ), is essential for survival and proliferation of DNMT3Amut leukemia cells. Polθ is overexpressed in DNMT3Amut leukemia cells due to abrogation of PARP1 PARylation-dependent UBE2O E3 ligase-mediated ubiquitination and proteasomal degradation of Polθ. In addition, PARP1- mediated recruitment of the SMARCAD1-MSH2/MSH3 repressive complex to DSBs was diminished in DNMT3Amut leukemia cells which facilitated loading of Polθ on DNA damage and promoting TMEJ and replication fork restart. Polθ inhibitors enhanced the anti-leukemic effects of mainstream drugs such as FLT3 kinase inhibitor quizartinib, cytarabine and etoposide in vitro and in mice with FLT3(ITD);DNMT3Amut leukemia. Altogether, Polθ is an attractive target in DNMT3Amut hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a67e19e6f65c3b6d38c0743f352c92d85456a92" target='_blank'>
              DNA polymerase theta-mediated DNA repair is a functional dependency and therapeutic vulnerability in DNMT3A deficient leukemia cells
              </a>
            </td>
          <td>
            Bac Viet Le, Umeshkumar Vekariya, Monika M. Toma, M. Nieborowska-Skorska, Marie-christine Caron, M. Gozdecka, Zayd Haydar, Martin Walsh, Jayashri Ghosh, Elaine Vaughan-Williams, Paulina Podszywalow-Bartnicka, Anna-Mariya Kukuyan, Sylwia Ziolkowska, E. Hadzijusufovic, Gurushankar Chandramouly, K. Piwocka, R. Pomerantz, G. Vassiliou, B. Huntly, P. Valent, Alfonso Bellacosa, Jean-Yves Masson, Gaorav P. Gupta, Grant A. Challen, Tomasz Skorski
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The mRNA cap methyltransferase CMTR1 plays a crucial role in RNA metabolism and gene expression regulation, yet its significance in cancer remains largely unexplored. Here, we present a comprehensive multi-omics analysis of CMTR1 across various human cancers, revealing its widespread upregulation and potential as a therapeutic target. Integrating transcriptomic and proteomic data from a large set of cancer samples, we demonstrate that CMTR1 is upregulated at the mRNA, protein, and phosphoprotein levels across multiple cancer types. Functional studies using CRISPR-mediated knockout and siRNA knockdown in breast cancer models show that CMTR1 depletion significantly inhibits tumor growth both in vitro and in vivo. Transcriptomic analysis reveals that CMTR1 primarily regulates ribosomal protein genes and other transcripts containing 5’ Terminal Oligopyrimidine (TOP) motifs. Additionally, CMTR1 affects the expression of snoRNA host genes and snoRNAs, suggesting a broader role in RNA metabolism. Mechanistically, we propose that CMTR1’s target specificity is partly determined by mRNA structure, particularly the presence of 5’TOP motifs. Furthermore, we identify a novel CMTR1 inhibitor, N97911, through in silico screening and biochemical assays, which demonstrates significant anti-tumor activity in vitro. Our findings establish CMTR1 as a key player in cancer biology, regulating critical aspects of RNA metabolism and ribosome biogenesis, and highlight its potential as a therapeutic target across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba981edad8c7b621348b03d39c7c8a8dd0ce83be" target='_blank'>
              Multi-omics analysis reveals CMTR1 upregulation in cancer and roles in ribosomal protein gene expression and tumor growth
              </a>
            </td>
          <td>
            I. Campeanu, Yuanyuan Jiang, Hilda Afisllari, Sijana H Dzinic, Lisa A. Polin, Zeng‐Quan Yang
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Nuclear lipids play roles in regulatory processes such as signaling, transcriptional regulation, and DNA repair. In this report, we demonstrate that nuclear lipids may contribute to Ki-67-regulated chromosome integrity during mitosis. In COS-7 cells, nuclear lipids are enriched at the perichromosomal layer and excluded from intrachromosomal regions during early mitosis, but are then detected in intrachromosomal regions during late mitosis, as revealed by TT-ExM, an improved expansion microscopy technique that enables high-sensitivity, super-resolution imaging of proteins, lipids, and nuclear DNA. The nuclear non-histone protein Ki-67 acts as a surfactant to form a repulsive molecular brush around fully condensed sister chromatids in early mitosis, preventing the diffusion or penetration of nuclear lipids into intrachromosomal regions. Ki-67 is phosphorylated during mitosis by cyclin-dependent kinase 1 (CDK1), the best-known master regulator of the cell cycle. Both Ki-67 knockdown and reduced Ki-67 phosphorylation by CDK1 inhibitors allow nuclear lipids to penetrate chromosomal regions. Thus, both Ki-67 protein level and phosphorylation status during mitosis appear to influence the perichromosomal distribution of nuclear lipids. Ki-67 knockdown and CDK1 inhibition also lead to uneven chromosome disjunction between daughter cells, highlighting the critical role of this regulatory mechanism in ensuring accurate chromosome segregation. Given that Ki-67 has been proposed to promote chromosome individualization during open mitosis in vertebrates, our results reveal that nuclear lipid enrichment at the perichromosomal layer enhances Ki-67’s ability to form a protective chromosomal envelope, which is critical for correct chromosome segregation and maintenance of genome integrity during mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b2e24bce325ec96509845393530c4fd71c4335" target='_blank'>
              Ki-67 and CDK1 Control the Dynamic Association of Nuclear Lipids with Mitotic Chromosomes
              </a>
            </td>
          <td>
            Hsiao-Tang Hu, Ueh-Ting Tim Wang, Bi-Chang Chen, Y. Hsueh, Ting-Fang Wang
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Carcinogen-induced mutations are thought near-random, with rare cancer-driver mutations underlying clonal expansion. Using high-fidelity Duplex Sequencing to reach a mutation frequency sensitivity of 4×10-9 per nt, we report that sun exposure creates pervasive mutations at sites with ∼100-fold UV-sensitivity in RNA-processing gene promoters – cyclobutane pyrimidine dimer (CPD) hyperhotspots – and these mutations have a mini-driver clonal expansion phenotype. Numerically, human skin harbored 10-fold more genuine mutations than previously reported, with neonatal skin containing 90,000 per cell; UV signature mutations increased 8,000-fold in sun-exposed skin, averaging 3×10-5 per nt. Clonal expansion by neutral drift or passenger formation was nil. Tumor suppressor gene hotspots reached variant allele frequency 0.1-10% via 30-3,000 fold clonal expansion, in occasional biopsies. CPD hyperhotspots reached those frequencies in every biopsy, with modest clonal expansion. In vitro, tumor hotspot mutations arose occasionally over weeks of chronic low-dose exposure, whereas CPD hyperhotspot mutations arose in days at 1000-fold higher frequencies, growing exponentially. UV targeted mini-drivers in every skin cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccbc5cb5ec1678ecc35a610c5268f3895b55100" target='_blank'>
              Pervasive Induction of Regulatory Mutation Microclones in Sun-exposed Skin
              </a>
            </td>
          <td>
            Vijay Menon, Alejandro Garcia-Ruiz, Susan Neveu, Brenda Cartmel, Leah M. Ferrucci, Meg Palmatier, Christine Ko, Kenneth Y. Tsai, Mio Nakamura, Sa Rang Kim, Michael Girardi, Karl Kornacker, Douglas E. Brash
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Persistent DNA double-strand breaks (DSBs) are enigmatically implicated in neurodegenerative diseases including Huntington’s disease (HD), the inherited late-onset disorder caused by CAG repeat elongations in Huntingtin (HTT). Here we combine biochemistry, computation and molecular cell biology to unveil a mechanism whereby HTT coordinates a Transcription-Coupled Non-Homologous End-Joining (TC-NHEJ) complex. HTT joins TC-NHEJ proteins PNKP, Ku70/80, and XRCC4 with chromatin remodeler Brahma-related Gene 1 (BRG1) to resolve transcription-associated DSBs in brain. HTT recruitment to DSBs in transcriptionally active gene- rich regions is BRG1-dependent while efficient TC-NHEJ protein recruitment is HTT-dependent. Notably, mHTT compromises TC-NHEJ interactions and repair activity, promoting DSB accumulation in HD tissues. Importantly, HTT or PNKP overexpression restores TC-NHEJ in a Drosophila HD model dramatically improving genome integrity, motor defects, and lifespan. Collective results uncover HTT stimulation of DSB repair by organizing a TC-NHEJ complex that is impaired by mHTT thereby implicating dysregulation of transcription-coupled DSB repair in mHTT pathophysiology. Highlights • BRG1 recruits HTT and NHEJ components to transcriptionally active DSBs. • HTT joins BRG1 and PNKP to efficiently repair transcription related DSBs in brain. • Mutant HTT impairs the functional integrity of TC-NHEJ complex for DSB repair. • HTT expression improves DSB repair, genome integrity and phenotypes in HD flies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9eb03b67998e606f7a7795acb11a2068118359e" target='_blank'>
              Chromatin remodeler BRG1 recruits huntingtin to repair DNA double-strand breaks in neurons
              </a>
            </td>
          <td>
            Subrata Pradhan, Keegan M. Bush, Nan Zhang, R. Pandita, Chi-Lin Tsai, Charlene Smith, Devon F. Pandlebury, Sagar Gaikwad, Francis Leonard, Linghui Nie, Annie Tao, William Russell, Subo Yuan, Sanjeev Choudhary, Kenneth S. Ramos, Cornelis J. Elferink, Yogesh P. Wairkar, John A. Tainer, Leslie Michels Thompson, T. Pandita, P. Sarkar
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235fec0818688f0c80daa0063ad7c555bc1856b" target='_blank'>
              Hypoxia-dependent recruitment of error-prone DNA polymerases to genome replication.
              </a>
            </td>
          <td>
            Ran Yehuda, Ido Dromi, Y. Levin, Thomas Carell, Nicholas Geacintov, Z. Livneh
          </td>
          <td>2024-10-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Although generally unknown, the age of a newly diagnosed tumor encodes valuable etiologic and prognostic information. Here, we estimate the age of breast cancers, defined as the time from the start of growth to detection, using a measure of epigenetic entropy derived from genome-wide methylation arrays. Based on an ensemble of neutrally fluctuating CpG (fCpG) sites, this stochastic epigenetic clock differs from conventional clocks that measure age-related increases in methylation. We show that younger tumors exhibit hallmarks of aggressiveness, such as increased proliferation and genomic instability, whereas older tumors are characterized by elevated immune infiltration, indicative of enhanced immune surveillance. These findings suggest that the clock captures a tumor’s effective growth rate resulting from the evolutionary-ecological competition between intrinsic growth potential and external systemic pressures. Because of the clock’s ability to delineate old and stable from young and aggressive tumors, it has potential applications in risk stratification of early-stage breast cancers and guiding early detection efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dff77a79c0945793feb47e32141039b96a0e80" target='_blank'>
              Mapping the Temporal Landscape of Breast Cancer Using Epigenetic Entropy
              </a>
            </td>
          <td>
            Daniel L. Monyak, Shannon T. Holloway, Graham J. Gumbert, Lars J. Grimm, E. S. Hwang, Jeffrey R. Marks, Darryl Shibata, Marc D. Ryser
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metapolycentromeres consist of multiple sequential domains of centromeric chromatin associated with a centromere-specific variant of histone H3 (CENP-A), functioning collectively as a single centromere. To date, they have been revealed in nine flowering plant, five insect and six vertebrate species. In this paper, we focus on their structure and possible mechanisms of emergence and evolution. The metapolycentromeres may vary in the number of centromeric domains and in their genetic content and epigenetic modifications. However, these variations do not seem to affect their function. The emergence of metapolycentromeres has been attributed to multiple Robertsonian translocations and segmental duplications. Conditions of genomic instability, such as interspecific hybridization and malignant neoplasms, are suggested as triggers for the de novo emergence of metapolycentromeres. Addressing the “centromere paradox” – the rapid evolution of centromeric DNA and proteins despite their conserved cellular function – we explore the centromere drive hypothesis as a plausible explanation for the dynamic evolution of centromeres in general, and in particular the emergence of metapolycentromeres and holocentromeres. Apparently, metapolycentromeres are more common across different species than it was believed until recently. Indeed, a systematic review of the available cytogenetic publications allowed us to identify 27 candidate species with metapolycentromeres. Тhe list of the already established and newly revealed candidate species thus spans 27 species of flowering plants and eight species of gymnosperm plants, five species of insects, and seven species of vertebrates. This indicates an erratic phylogenetic distribution of the species with metapolycentromeres and may suggest an independent emergence of the metapolycentromeres in the course of evolution. However, the current catalog of species with identified and likely metapolycentromeres remains too short to draw reliable conclusions about their evolution, particularly in the absence of knowledge about related species without metapolycentromeres for comparative analysis. More studies are necessary to shed light on the mechanisms of metapolycentromere formation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a7898b3ea2f2d63e1ff7e6046657eb47951d50" target='_blank'>
              Structure and evolution of metapolycentromeres
              </a>
            </td>
          <td>
            E. Grishko, P. Borodin
          </td>
          <td>2024-10-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b108c069a149d86683e9a9aa9e66ae175bdebcb9" target='_blank'>
              MECOM Locus classical transcript isoforms affect tumor immune microenvironment and different targets in ovarian cancer
              </a>
            </td>
          <td>
            Ning Lan, Shuheng Bai, Min Chen, Xuan Wang, Zhaode Feng, Ying Gao, B. Hui, Wen Ma, Xiangxiang Zhang, Fengyuan Hu, Wanyi Liu, Wenyang Li, Fang Wu, Juan Ren
          </td>
          <td>2024-10-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pericentromeres are heterochromatic regions adjacent to centromeres that ensure accurate chromosome segregation. Despite their conserved function, they are composed of rapidly evolving A/T-rich satellite DNA. This paradoxical observation is partially resolved by epigenetic mechanisms that maintain heterochromatin, independent of specific DNA sequences. However, it is unclear how satellite DNA sequence variation impacts de novo formation of pericentric heterochromatin, which is initially absent from paternal chromosomes in the zygote. Here we show that satellite variation has functional consequences for zygotic heterochromatin formation, recruitment of the Chromosome Passenger Complex (CPC), and interactions with spindle microtubules. In M. musculus zygotes, Polycomb Repressive Complex 1 (PRC1) is recruited to pericentric satellites by its AT-hook domain, which binds runs of A/T nucleotides, to generate H2AK119ub1 (H2Aub) heterochromatin. By fertilizing M. musculus eggs with sperm from other mouse species, we show that species-specific satellite sequences differ in their ability to recruit PRC1 and form H2Aub. This satellite-DNA mediated increase in PRC1 heterochromatin leads to reduced CPC recruitment and increased microtubule forces on kinetochores. Our results provide a direct link between satellite DNA composition and pericentromere function in the zygote, when epigenetic pathways maintaining pericentromere heterochromatin are absent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6422169819169157447ac8bc6a3224b6f8ab89" target='_blank'>
              Species-specific satellite DNA composition dictates de novo formation of PRC1-mediated pericentric heterochromatin
              </a>
            </td>
          <td>
            Piero Lamelza, Malena Parrado, Emma Hamlin, Micheal A. Lampson
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chronically persistent viruses are integral components of the organismal ecosystem in humans and animals 1 2. Many of these viruses replicate and accumulate within the cell nucleus 3. The nuclear location allows viruses to evade cytoplasmic host viral sensors and promotes viral replication 4. One of the unexplored and puzzling aspects of the viral nuclear lifecycle involves the virus’s ability to deal with the physical constraints of nuclear architecture. To replicate and accumulate within the nucleus in large numbers sufficient for infection spreading, DNA viruses need to overcome the spatial limitations imposed by chromatin and the nuclear matrix. We found that one of the most widespread and potentially lethal human viruses, the JC polyomavirus 5, interferes with nuclear heterochromatin to create virus-occupied space. The JC virus’s impact on heterochromatin is mediated by the viral nonstructural protein, Agnoprotein (Agno). Agno’s interference with heterochromatin is governed by structurally diverse mimics of host epigenetic regulators that facilitate virus-induced chromatin reorganization and a dramatic decline in nuclear stiffness in the infected cells. The JCV epigenetic mimicry is critical for the virus infection, as evident from reduced replication of mimic-mutant viruses. Our data suggest that modulation of nuclear mechanical properties is a novel strategy enabling chronicity of the JC and possibly other nuclear virus infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab25f0e5f1471f31142457449e1e4104ce760805" target='_blank'>
              Chromatin mimicry by human JC virus
              </a>
            </td>
          <td>
            Uwe Schaefer, Yekaterina A. Miroshnikova, Wei Xie, Adam G. Larson, Ziyu Lu, Shoudeng Chen, Martina Bradic, Y. Goldgur, Kexin Chen, , Junyue Cao, Dinshaw J. Patel, G. Narlikar, Sara A. Wickström, Alexander Tarakhovsky
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7ee5fba3b3bfc377211f04436065c1bd9deb16" target='_blank'>
              Elucidating the genotoxicity of Fusobacterium nucleatum-secreted mutagens in colorectal cancer carcinogenesis
              </a>
            </td>
          <td>
            Wenye Xu, Yuchen Zhang, Dongjiao Chen, Dan Huang, Yang Zhao, Wei Hu, Ling Lin, Yingzhi Liu, Shilan Wang, Judeng Zeng, Chuan Xie, H. Chan, Qing Li, Huarong Chen, Xiaodong Liu, Sunny H Wong, Jun Yu, Francis K L Chan, Matthew T V Chan, Siew C Ng, William K. K. Wu, Lin Zhang
          </td>
          <td>2024-09-27</td>
          <td>Gut Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: It is widely accepted that the 3D chromatin organization in human cell nuclei is not random and recent investigations point towards an interactive relation of epigenetic functioning and chromatin (re-)organization. Although chromatin organization seems to be the result of self-organization of the entirety of all molecules available in the cell nucleus, a general question remains open as to what extent chromatin organization might additionally be predetermined by the DNA sequence and, if so, if there are characteristic differences that distinguish typical regions involved in dysfunction-related aberrations from normal ones, since typical DNA breakpoint regions involved in disease-related chromosome aberrations are not randomly distributed along the DNA sequence. Methods: Highly conserved k-mer patterns in intronic and intergenic regions have been reported in eukaryotic genomes. In this article, we search and analyze regions deviating from average spectra (ReDFAS) of k-mer word frequencies in the human genome. This includes all assembled regions, e.g., telomeric, centromeric, genic as well as intergenic regions. Results: A positive correlation between k-mer spectra and 3D contact frequencies, obtained exemplarily from given Hi-C datasets, has been found indicating a relation of ReDFAS to chromatin organization and interactions. We also searched and found correlations of known functional annotations, e.g., genes correlating with ReDFAS. Selected regions known to contain typical breakpoints on chromosomes 9 and 5 that are involved in cancer-related chromosomal aberrations appear to be enriched in ReDFAS. Since transposable elements like ALUs are often assigned as major players in 3D genome organization, we also studied their impact on our examples but could not find a correlation between ALU regions and breakpoints comparable to ReDFAS. Conclusions: Our findings might show that ReDFAS are associated with instable regions of the genome and regions with many chromatin contacts which is in line with current research indicating that chromatin loop anchor points lead to genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3f4228708cdafa938a5ddafca7f4b52c72767a" target='_blank'>
              Peculiar k-mer Spectra Are Correlated with 3D Contact Frequencies and Breakpoint Regions in the Human Genome
              </a>
            </td>
          <td>
            Wisam Mohammed Hikmat, Aaron Sievers, Michael Hausmann, G. Hildenbrand
          </td>
          <td>2024-09-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Mutations in genes involved in DNA damage repair (DDR) often lead to premature aging syndromes. While recent evidence suggests that inflammation, alongside mutation accumulation and cell death, may drive disease phenotypes, its precise contribution to in vivo pathophysiology remains unclear. Here, by modeling Ataxia Telangiectasia (A-T) and Bloom Syndrome in the African turquoise killifish (N. furzeri), we replicate key phenotypes of DDR syndromes, including infertility, cytoplasmic DNA fragments, and reduced lifespan. The link between DDR defects and inflammation is attributed to the activation of the cGAS-STING pathway and interferon signaling by cytoplasmic DNA. Accordingly, mutating cGAS partially rescues germline defects and senescence in A-T fish. Double mutants also display reversal of telomere abnormalities and suppression of transposable elements, underscoring cGAS’s non-canonical role as a DDR inhibitor. Our findings emphasize the role of interferon signaling in A-T pathology and identify the cGAS-STING pathway as a potential therapeutic target for genomic instability syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0762a0a7a477e0941f31815c1aa8f1b5b79a238" target='_blank'>
              The cGAS-STING pathway is an in vivo modifier of genomic instability syndromes
              </a>
            </td>
          <td>
            Marva Bergman, Uri Goshtchevsky, Tehila Atlan, Gwendoline Astre, Ryan Halabi, Hosniyah El, Eitan Moses, A. J. Lemus, BA Benayoun, Y. Tzfati, Ido Ben-Ami, I. Harel
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The zinc finger protein CTCF is ubiquitously expressed and is integral to the regulation of chromatin architecture through its interaction with cohesin. Conversely, CTCFL expression is predominantly restricted to the adult male testis but is aberrantly expressed in certain cancers. Despite their distinct expression patterns, the cooperative and competitive mechanisms by which CTCF and CTCFL regulate target gene expression in spermatocytes and cancer cells remain inadequately understood. In this review, we comprehensively examine the literature on the divergent amino acid sequences, target sites, expression profiles and functions of CTCF and CTCFL in normal tissues and cancers. We further elucidate the mechanisms by which CTCFL competitively or cooperatively binds to CTCF target sites during spermatogenesis and carcinogenesis to modulate chromatin architecture. We mainly focus on the role of CTCFL in testicular and cancer development, highlighting its interaction with CTCF at CTCF binding sites to regulate target genes. In the testis, CTCF and CTCFL cooperate to regulate the expression of testis-specific genes, essential for proper germ cell progression. In cancers, CTCFL overexpression competes with CTCF for DNA binding, leading to aberrant gene expression, a more relaxed chromatin state, and altered chromatin loops. By uncovering the roles of CTCF and CTCFL in spermatogenesis and carcinogenesis, we can better understand the implications of aberrant CTCFL expression in altering chromatin loops and its contribution to disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0158f6b03a46c193781d8f1661a7ed6e9e29" target='_blank'>
              Interaction of CTCF and CTCFL in genome regulation through chromatin architecture during the spermatogenesis and carcinogenesis
              </a>
            </td>
          <td>
            Xin Tong, Yang Gao, Zhongjing Su
          </td>
          <td>2024-10-15</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Previous evidence indicates that the NDC80 complex, a conserved spindle microtubule-binding component of the kinetochore, is overexpressed and associated with prognosis in certain cancer types. Herein, we assessed the expression and prognostic value of NDC80 complex components in pan-cancer and interrogated their potential functions in tumor context through multiple databases and software. Results: Our findings showed that the expression of NDC80 complex components was aberrant across almost all cancer types and correlated positively with poor prognosis at the pan-cancer level. Furthermore, the expression levels of NDC80 complex components were positively associated with Th2 cell infiltration in the majority of cancer types. Additionally, higher expression of the NDC80 complex components was associated with increased immune checkpoint gene expression and TP53 mutation in specific cancer types. We also discovered that NDC80 complex components play pivotal roles in cell division, and the cell cycle within the tumor context. Moreover, knockdown of NDC80 significantly suppressed cell growth and inhibited the G1-S phase transition in two breast cancer cell lines. Conclusions: Our study suggests that the NDC80 complex components could serve as reliable biomarkers for cancer detection and prognosis in pan-cancer, in addition to uncovering their role as cancer-promoting genes involved in Th2 cell infiltration, immune checkpoint, cell growth, and TP53 mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92eb99ee7693c0d7c984ba194cdb675e99d2dd73" target='_blank'>
              Implications of the NDC80 complex on the tumor immune microenvironment and cell growth in pan-cancer
              </a>
            </td>
          <td>
            Jia-Xing Wang, Teng-yue Diao, Xue-Ling Yang, Ke Li, Jun Yang, Xie-Qun Chen
          </td>
          <td>2024-10-14</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In this review article we will highlight the evidences that how oncogenes are formed due to the physical genetic variations in proto-oncogenes and tumor suppressor genes and various planned immunotherapies which will include- The immune checkpoint inhibitor-opposing antibodies, adoptive cell treatments, and biologic modifiers (cytokines and vaccines). We will make an effort to provide guidance and potential fixes for these issues, along with pertinent sources for foundational research. For suitable studies, a literature search was undertaken from various database sources such as PubMed, EMBASE, and Google Scholar. One type of gene known as an oncogene-a cellular gene that becomes dysfunctional owing to mutation and overexpression-is the cause of cancer. Certain oncogenes seem to inhibit the homeostatic mechanism by limiting the single cell lineage of leukemia stem cells. According to the clonal theory of oncogenes, tumors are thought to begin in a single cell, Moreover, the growth of tumors is closely linked to the prevention of apoptosis, or programmed cell death. These activities of oncogene can be minimized by some immunological therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77360e601c608aabfa1825689be8b174a0154240" target='_blank'>
              Oncogenetics: Unraveling the Genetic Underpinnings of Cancer for Improved Immunotherapeutic Outcomes.
              </a>
            </td>
          <td>
            Stuti Dwivedi, Praveencumar R, T. Sivakumar, M. K. Posa, R. Dhakar, Ruchi Tiwari
          </td>
          <td>2024-10-29</td>
          <td>Zhongguo ying yong sheng li xue za zhi = Zhongguo yingyong shenglixue zazhi = Chinese journal of applied physiology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background: Aberrant gene promoter methylation is one of the hallmarks of Acute Myeloid Leukemia (AML). RAD21 is an important gene, implicated in sister chromatids cohesion, DNA repair, the regulation of gene transcription, apoptosis and hematopoiesis. Methods: In this study, we investigate the possible implication of RAD21 promoter methylation in AML pathogenesis using a cohort of AML patients and a cohort of healthy individuals. Results: RAD21 promoter methylation was found in 24% of patients and in none of the controls (p = 0.023), indicating a possible contribution to AML development. Interestingly, a statistically higher frequency of RAD21 methylation was observed in patients with trisomy 8 (9/21, 42.9%, p = 0.021), while none of the patients with aberrations of chromosome 11 had RAD21 gene promoter methylation (0%, 0/11, p = 0.048). Patients with monosomal and complex karyotypes showed low frequencies of RAD21 methylation (7.7% and 15.4%, respectively) without reaching statistical significance. Moreover, ASXL1 mutations were not found to be associated with RAD21 methylation. Conclusions: This is the first study which provides evidence for a possible pathogenetic role of RAD21 promoter methylation in AML development and especially in AML with trisomy 8. Further studies of RAD21 promoter methylation in large series of different AML genetic subgroups may contribute to the elucidation of AML pathogenesis and to the identification of new epigenetic biomarkers with diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c902aea9aee61d29bf4e327d8db03e3f748b0fb1" target='_blank'>
              Cohesin RAD21 Gene Promoter Methylation in Patients with Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Manola, Sophia Zachaki, Katerina Kakosaiou, Agapi Ioannidou, Marina Kalomoiraki, T. Rampias
          </td>
          <td>2024-10-01</td>
          <td>Life</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD. Methods Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration. Results In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment. Conclusion ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c42616846e496900baa7e96476fa73f42799ad" target='_blank'>
              APC mutations dysregulate alternative polyadenylation in cancer
              </a>
            </td>
          <td>
            Austin M. Gabel, Andrea E. Belleville, James D Thomas, Jose Mario Bello Pineda, Robert K Bradley
          </td>
          <td>2024-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Here, we present a method for enrichment of double-stranded cfDNA with an average length of ∼40 bp from cfDNA for high-throughput DNA sequencing. This class of cfDNA is enriched at gene promoters and binding sites of transcription factors or structural DNA-binding proteins, so that a genome-wide DNA footprint is directly captured from liquid biopsies. In short double-stranded cfDNA from healthy individuals, we find significant enrichment of 203 transcription factor motifs. Additionally, short double-stranded cfDNA signals at specific genomic regions correlate negatively with DNA methylation, positively with H3K4me3 histone modifications and gene transcription. The diagnostic potential of short double-stranded cell-free DNA (cfDNA) in blood plasma has not yet been recognized. When comparing short double-stranded cfDNA from patient samples of pancreatic ductal adenocarcinoma with colorectal carcinoma or septic with postoperative controls, we identify 136 and 241 differentially enriched loci, respectively. Using these differentially enriched loci, the disease types can be clearly distinguished by principal component analysis, demonstrating the diagnostic potential of short double-stranded cfDNA signals as a new class of biomarkers for liquid biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/003bad913ef23b451e8afe9e333721e6d528290e" target='_blank'>
              A novel approach for in vivo DNA footprinting using short double-stranded cell-free DNA from plasma
              </a>
            </td>
          <td>
            Jan Müller, Christina Hartwig, Mirko Sonntag, Lisa Bitzer, Christopher Adelmann, Y. Vainshtein, Karolina Glanz, S. Decker, Thorsten Brenner, Georg F. Weber, A. von Haeseler, Kai Sohn
          </td>
          <td>2024-08-01</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Endogenous retroelements (EREs), which comprise half of the human genome, play a pivotal role in genome dynamics. Some EREs retained the ability to encode proteins, although most degenerated or served as a source for novel genes and regulatory elements during evolution. Despite ERE repression mechanisms developed to maintain genome stability, widespread pervasive ERE activation is observed in cancer including hematological malignancies. Challenging the perception of noncoding DNA as "junk," EREs are underestimated contributors to cancer driver mechanisms as well as antitumoral immunity by providing innate immune ligands and tumor antigens. This review highlights recent progress in understanding ERE co-option events in cancer and focuses on the controversial debate surrounding their causal role in shaping malignant phenotype. We provide insights into the rapidly evolving landscape of ERE research in hematological malignancies and their clinical implications in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70dce85bfb8495c63680f246f15584cd08410198" target='_blank'>
              Endogenous retroelements in hematological malignancies: From epigenetic dysregulation to therapeutic targeting.
              </a>
            </td>
          <td>
            Mohamed Chour, Françoise Porteu, S. Depil, V. Alcazer
          </td>
          <td>2024-10-10</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 High-grade gliomas are among the deadliest of all childhood cancers and the leading cause of cancer-related mortality in children. The standard treatment is resection followed by chemotherapy, most often temozolomide and radiation therapy. Unfortunately, there is no evidence that temozolomide influences survival in pediatric patients and only a limited number of new drugs have been identified for specific subgroups of tumors. The identification of fusion genes as possible drivers of the disease offers an opportunity for targeted therapies. We present a case of a six-year-old patient with a novel TRIM24::NTRK2 fusion positive epithelioid glioblastoma with subsequent relapses.



 We investigated the ramification of this fusion and evolution of the tumor through whole genome sequencing, RNA-sequencing and DNA methylation profiling, and studied the effect of the patient’s treatment using patient-derived cell cultures.



 Our studies show that the TRIM24::NTRK2 fusion has oncogenic abilities but loses its importance as the tumor evolves. In addition, we demonstrate an increased drug resistance over time, further indicating the diminished driver function of the fusion.



 The use of resequencing revealed parallel development of tumor subclones with different genomic alterations, highlighting the importance of genomic profiling from various parts of the tumor.



 This work was supported by the Swedish Research Council, the Swedish Childhood Cancer Foundation, and the Swedish Cancer Society
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6825c06ca5f91e7d7121415ac1f08b97ce06cc4e" target='_blank'>
              P16.10.A UNRAVELING THE CLINICAL IMPLICATIONS OF TRIM24::NTRK2 FUSION GENE IN PEDIATRIC HIGH-GRADE GLIOMA: A CASE STUDY
              </a>
            </td>
          <td>
            I. Karlsson, S. Keane, L. Deland, A. Lindlöf, T. Kling, T. Olsson Bontell, H. Fagman, J. Nilsson, D. Nilsson, M. Sabel, H. Carén, F. Abel
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eba36ecae0bb22585b48631895c433e55a25862" target='_blank'>
              Nucleolus and centromere Tyramide Signal Amplification-Seq reveals variable localization of heterochromatin in different cell types
              </a>
            </td>
          <td>
            Pradeep Kumar, Omid Gholamalamdari, Yang Zhang, Liguo Zhang, Anastassiia Vertii, Tom van Schaik, Daan Peric-Hupkes, Takayo Sasaki, David M. Gilbert, B. van Steensel, Jian Ma, Paul D. Kaufman, A. Belmont
          </td>
          <td>2024-09-13</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7405dbfe981797b8a9d71c0696e2eb74696ed4d" target='_blank'>
              Genomic and transcriptomic landscape of human gastrointestinal stromal tumors
              </a>
            </td>
          <td>
            Feifei Xie, Shuzhen Luo, Dongbing Liu, Xiaojing Lu, Ming Wang, Xiaoxiao Liu, Fujian Jia, Yuzhi Pang, Yanying Shen, Chunling Zeng, Xin-Li Ma, Daoqiang Tang, L. Tu, Lin-Xi Yang, Yumei Cheng, Yuxiang Luo, Fanfan Xie, Hao Hou, Tao Huang, B-L Ni, C. Zhuang, Wen-Yi Zhao, Ke Li, Xufen Zheng, Wenbo Bi, X. Jia, Yi He, Simin Wang, Hui Cao, Kui Wu, Yue-Qing Wang
          </td>
          <td>2024-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Chromatin, with its complex spatial and temporal organization, plays a crucial role in regulating gene expression. Recent advancements in super-resolution microscopy have revealed that nanoscale domains of heterochromatin (repressed segments) embedded within a euchromatin (active segments) background are fundamental units of 3D chromatin organization. In tissue-resident cells, the size of these heterochromatin domains varies with the microenvironment, particularly its stiffness, and chromatin organization is also influenced by pharmacological and epigenetic drugs. However, the mechanisms governing heterochromatin domain size under various conditions and their impact on gene expression remain unclear. To address this knowledge gap, we have developed a dynamic, next-generation sequencing informed chromatin copolymer model. Our model simulates the spatiotemporal evolution of chromatin, driven by passive diffusion and active epigenetic reactions, which interconvert euchromatin and heterochromatin. By integrating chromatin-chromatin interaction energetics and diffusion-reaction dynamics, we predict the formation of nanoscale heterochromatin-rich domains and establish a scaling relationship between their size and the modulation of epigenetic reaction rates. Additionally, our model predicts that epigenetic and chromatin compaction changes in response to changes in global reaction rates occur predominantly at domain boundaries. We validated these predictions via Hi-C contact map analysis and super-resolution imaging of hyperacetylated melanoma cells. Subsequent RNA-seq analysis suggested a pivotal role of these epigenetic shifts in influencing the metastatic potential of these cells. We further validated our mesoscale findings against chromatin rearrangement in hMSCs, which exhibit sensitivity of epigenetic reaction rates to changes in microenvironmental stiffness. Finally, we evaluated the effects of cycling of epigenetic reaction rates in silico, mimicking the cellular transition to different extracellular conditions, and back again. This finding reveals a cell-type invariant mechanism driven by domain boundaries, whereby chromatin organization guides epigenetic memory formation. Our findings show that chromatin reorganization in response to changes in epigenetic reaction rates resulting from alterations in the microenvironment, drug exposure and disease progression impacts both immediate cellular responses and long-term epigenetic memory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e55d9db8c8d7ff0628f2ac7f62c8baea4b57b" target='_blank'>
              Polymer Model Integrates Super-Resolution Imaging and Epigenomic Sequencing to Elucidate the Role of Epigenetic Reactions in Shaping 4D Chromatin Organization
              </a>
            </td>
          <td>
            Vinayak Vinayak, Ramin Basir, Rosela Golloshi, Joshua Toth, Lucas E. Sant’Anna, Melike Lakadamyali, R. McCord, Vivek B. Shenoy
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation (DNAm) suppresses gene expression and contributes to dosage compensation in mammals but whether DNAm plays a similar role in female ZW chromosome heterogametic species remains unresolved. We assessed chromosome-level DNAm using whole genome bisulphite sequencing in two avian species, zebra finches and jackdaws. Dosage compensation by DNAm would result in higher and more variable DNAm level in males relative to females on the Z chromosome. However, we found that the level of DNAm and its variance on the Z chromosome was lower in males. Moreover, male Z chromosome-based gene promoters were more frequently hypomethylated compared to females, indicating absence of upregulation on a gene-by-gene basis across the female Z chromosome. We suggest our findings reveal mitigation of an intra-genomic sexual conflict, with females suppressing expression of Z chromosome-based genes that benefit male but not female fitness. W was the most methylated chromosome, but hypermethylation on the W chromosome was mostly confined to intergenic regions, presumably resulting in the downregulation of transposable elements known to comprise a large part of the W chromosome. Thus, DNAm is involved in the development of sex-dependent phenotypes, but dosage compensation is achieved through other mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba474a1b02e8d6f31095725d41eef1d38d99559" target='_blank'>
              Dosage compensation and sexual conflict in female heterogametic methylomes
              </a>
            </td>
          <td>
            Marianthi Tangili, Joanna Sudyka, Fabrício Furni, P. Palsbøll, Simon Verhulst
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The elevated level of replication stress is an intrinsic characteristic of cancer cells. Targeting the mechanisms that maintain genome stability to further increase replication stress and thus induce severe genome instability has become a promising approach for cancer treatment. Here, we identify histone deacetylase 8 (HDAC8) as a drug target whose inactivation synergizes with the inhibition of checkpoint kinases to elicit substantial replication stress and compromise genome integrity selectively in cancer cells. We showed that simultaneous inhibition of HDAC8 and checkpoint kinases led to extensive replication fork collapse, irreversible cell-cycle arrest, and synergistic vulnerability in various cancer cells. The efficacy of the combination treatment was further validated in patient tumor-derived organoid (PDO) and xenograft mouse (PDX) models, providing important insights into patient-specific drug responses. Our data revealed that HDAC8 activity was essential for reducing the acetylation level of structural maintenance of chromosomes protein 3 (SMC3) ahead of replication forks and preventing R loop formation. HDAC8 inactivation resulted in slowed fork progression and checkpoint kinase activation. Our findings indicate that HDAC8 guards the integrity of the replicating genome, and the cancer-specific synthetic lethality between HDAC8 and checkpoint kinases provides a promising replication stress-targeting strategy for treating a broad range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ad8e7b369dc59096bea669bbbbe3db9dbfc35f1" target='_blank'>
              Combined HDAC8 and checkpoint kinase inhibition induces tumor-selective synthetic lethality in preclinical models.
              </a>
            </td>
          <td>
            Ting-Yu Chang, Yan Yan, Zih-Yao Yu, Moeez Rathore, Nian-Zhe Lee, Hui-Ju Tseng, Li-Hsin Cheng, Wei-Jan Huang, Wei Zhang, Ernest R Chan, Yulan Qing, Ming-Lun Kang, Rui Wang, Kelvin K Tsai, J. Pink, William E Harte, Stanton L. Gerson, Sung-Bau Lee
          </td>
          <td>2024-10-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The CHAMP1 complex, a little-known but highly conserved protein complex consisting of CHAMP1, POGZ, and HP1α, is enriched in heterochromatin though its cellular function in these regions of the genome remain unknown. Here we show that the CHAMP complex promotes heterochromatin assembly at multiple chromosomal sites, including centromeres and telomeres, and promotes homology-directed repair (HDR) of DNA double strand breaks (DSBs) in these regions. The CHAMP1 complex is also required for heterochromatin assembly and DSB repair in highly-specialized chromosomal regions, such as the highly-compacted telomeres of ALT (Alternative Lengthening of Telomeres) positive tumor cells. Moreover, the CHAMP1 complex binds and recruits the writer methyltransferase SETDB1 to heterochromatin regions of the genome and is required for efficient DSB repair at these sites. Importantly, peripheral blood lymphocytes from individuals with CHAMP1 syndrome, an inherited neurologic disorder resulting from heterozygous mutations in CHAMP1, also exhibit defective heterochromatin clustering and defective repair of local DSBs, suggesting that a defect in DNA repair underlies this syndrome. Taken together, the CHAMP1 complex has a novel role in heterochromatin assembly and the enhancement of HDR in heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454e4622c260f66b26c8cb18854b8ca092214e10" target='_blank'>
              CHAMP1 complex directs heterochromatin assembly and promotes homology-directed DNA repair
              </a>
            </td>
          <td>
            Feng Li, Tianpeng Zhang, Aleem Syed, Amira Elbakry, Noella Holmer, Huy Nguyen, Sirisha Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transcription factors (TFs) are pivotal in orchestrating lineage decisions and their binding is often accompanied by chromatin remodeling. However, a comprehensive understanding of how cellular and epigenetic contexts influence TF binding and the subsequent activation of lineage-specific programs remains lacking. Here, we dissected the molecular mechanisms underlying the binding of one such TF – Neurog2 and its functional consequences in two cell-types: mouse embryonic stem cells and neural progenitor cells. Our findings reveal that cell-type-specific chromatin accessibility and motif syntax are key determinants for TF binding. While Neurog2 binding primarily leads to chromatin opening, DNA demethylation and increased chromatin interactions, we also uncovered strong indirect, cell-type-specific effects, which ultimately result in vastly different epigenetic landscape. Furthermore, we identify shared and cell-type-specific Neurog2 interactors, including the SWI/SNF and NuRD complexes. Our study shed light on how cellular environment can modulate TF function to establish lineage identity during development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd3b13b3ffc7b199c0c0fc2ae483d8e468547c4" target='_blank'>
              Context-dependent epigenome rewiring during neuronal differentiation
              </a>
            </td>
          <td>
            Vera Manelli, Jeisimhan Diwakar, Sude Beşkardeş, Dácil Alonso-Gil, I. Forné, F. Chong, A. Imhof, Boyan Bonev
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Medulloblastoma and ependymoma are prevalent pediatric central nervous system tumors with significant molecular and clinical heterogeneity. We collected bulk RNA sequencing data from 888 medulloblastoma and 370 ependymoma tumors to establish a comprehensive reference landscape. Following rigorous batch effect correction, normalization, and dimensionality reduction, we constructed a unified landscape to explore gene expression, signaling pathways, gene fusions, and copy number variations. Our analysis revealed distinct clustering patterns, including two primary ependymoma compartments, EPN-E1 and EPN-E2, each with specific gene fusions and molecular signatures. In medulloblastoma, we achieved precise stratification of Group 3/4 tumors by subtype and in SHH tumors by patient age. Our landscape serves as a vital resource for identifying biomarkers, refining diagnoses, and enables the mapping of new patients’ bulk RNA-seq data onto the reference framework to facilitate accurate disease subtype identification. The landscape is accessible via Oncoscape, an interactive platform, empowering global exploration and application. One Sentence Summary A landscape built using only Transcriptomic analysis for medulloblastoma and ependymoma reveals novel insights about subtype specific biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7340d3bdec67105d60c89f8f44d73e7f797360" target='_blank'>
              Transcriptomic landscape identifies two unrecognized ependymoma subtypes and novel pathways in medulloblastoma
              </a>
            </td>
          <td>
            Sonali Arora, Nicholas Nuechterlein, Matt Jensen, Gregory Glatzer, P. Sievers, S. Varadharajan, A. Korshunov, Felix Sahm, Stephen C. Mack, Michael D Taylor, Eric C. Holland
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7def5ed43bd9c730c55fb611f1c5567d64e4d7" target='_blank'>
              Decoding the epigenetics and chromatin loop dynamics of androgen receptor-mediated transcription
              </a>
            </td>
          <td>
            Umut Berkay Altıntaş, Ji-Heui Seo, C. Giambartolomei, Dogancan Ozturan, Brad J Fortunato, Geoffrey M. Nelson, Seth R Goldman, Karen Adelman, Faraz Hach, M. Freedman, N. Lack
          </td>
          <td>2024-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Acanthamoeba is a cosmopolitan freshwater amoebae known for its association with Nucleocytoplasmic Large DNA Viruses (NCLDVs). Previous studies have shown that Acanthamoeba spp. undergo lateral gene transfer (LGT) with NCLDVs. Here we have leveraged chromosome-scale assemblies of two strains of Acanthamoeba castellanii, Neff and C3, to investigate the occurrence and genomic context of viral LGT in Acanthamoeba. We show that the viral ‘footprints’ in the C3 and Neff genomes are largely non-overlapping and that viral genes in Neff are found disproportionately in large sub-telomeric insertions. Multiple partial copies of these insertions are found throughout the Neff genome, but they are not expressed and do not encode functions for their own mobility. Viral regions are hyper-methylated and highly condensed, suggesting that the expression of recently acquired viral DNA is suppressed in heterochromatic regions. We propose a three-step model for the origin and fate of viral sequences in Acanthamoeba: (i) integration of DNA from giant viruses, (ii) epigenetic suppression of the viral DNAs, which allows them to persist in the genome, and (iii) deterioration of viral genomes by point mutation and intra- and inter-chromosomal recombination. Viral integrations in Acanthamoeba spp. are transient and may not have long-lasting effects on the fitness of the host amoeba. Our work strengthens a growing body of work showing widespread but transient integration of viral DNA in protists and extends the relevance of epigenetic silencing mechanisms to the evolution of Amoebozoa. We highlight the importance of host genome dynamics for understanding the evolution of endogenized viral elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677a6acc4759cb2f267a8b95a64864a690721e79" target='_blank'>
              Epigenetic silencing and host genome evolution determine fate of viral insertions in Acanthamoeba
              </a>
            </td>
          <td>
            Cédric Blais, Morgan J. Colp, Luke A. Sarre, Alex de Mendoza, John M. Archibald
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary DNA-dependent protein kinase (DNA-PK) is a key player in repairing DNA damage. Ku proteins detect DNA damage and activate DNA-PK. Blocking DNA-PK can make cancer treatments such as radiation more effective. We have developed Ku–DNA binding inhibitors (Ku-DBis) that stop DNA-PK activation, making cancer cells more sensitive to radiation. We recently discovered new Ku-DBis that enter cells better than predecessor compounds, allowing cellular and in vivo analyses. Some non-small cell lung cancer (NSCLC) cells, especially those lacking ATM, were very sensitive to these inhibitors and radiation. However, cells lacking BRCA1 were resistant, which reduced the effectiveness of radiation. In animal studies of NSCLC, Ku-DBi treatment inhibited DNA-PK and enhanced a radiation-dependent decrease in tumor cell proliferation. This is the first time a Ku-targeted inhibitor has been shown to work in living organisms, suggesting their potential application in cancer treatment. Abstract Background: DNA-dependent protein kinase (DNA-PK) is a validated cancer therapeutic target involved in DNA damage response (DDR) and non-homologous end-joining (NHEJ) repair of DNA double-strand breaks (DSBs). Ku serves as a sensor of DSBs by binding to DNA ends and activating DNA-PK. Inhibition of DNA-PK is a common strategy to block DSB repair and improve efficacy of ionizing radiation (IR) therapy and radiomimetic drug therapies. We have previously developed Ku–DNA binding inhibitors (Ku-DBis) that block in vitro and cellular NHEJ activity, abrogate DNA-PK autophosphorylation, and potentiate cellular sensitivity to IR. Results and Conclusions: Here we report the discovery of oxindole Ku-DBis with improved cellular uptake and retained potent Ku-inhibitory activity. Variable monotherapy activity was observed in a panel of non-small cell lung cancer (NSCLC) cell lines, with ATM-null cells being the most sensitive and showing synergy with IR. BRCA1-deficient cells were resistant to single-agent treatment and antagonistic when combined with DSB-generating therapies. In vivo studies in an NSCLC xenograft model demonstrated that the Ku-DBi treatment blocked IR-dependent DNA-PKcs autophosphorylation, modulated DDR, and reduced tumor cell proliferation. This represents the first in vivo demonstration of a Ku-targeted DNA-binding inhibitor impacting IR response and highlights the potential therapeutic utility of Ku-DBis for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3d24d7ccb0dfea0d53a93383e79bae41d2a8cf" target='_blank'>
              Impact of Optimized Ku–DNA Binding Inhibitors on the Cellular and In Vivo DNA Damage Response
              </a>
            </td>
          <td>
            P. Mendoza-Munoz, N. Kushwaha, Dineshsinha Chauhan, Karim Ben Ali Gacem, Joy E. Garrett, J. Dynlacht, J. Charbonnier, Navnath S. Gavande, J. Turchi
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Human stem cells rely on enhanced DNA repair mechanisms to safeguard their ability to replenish somatic tissues. Telomerase counteracts telomere shortening and is a component of the stem cell DNA repair system that is regulated by ATM and ATR kinases. Here, we report that the DNA repair enzyme APEX2, but not its close paralog APEX1, is required for efficient telomerase reverse transcriptase (TERT) gene expression in human embryonic stem cells (hESC) and a melanoma cell line. We also observed that APEX2 knockdown significantly diminished telomerase enzyme activity. While APEX1 is known to regulate certain transcription factors, APEX2 has not been reported to influence gene expression. To gain insight into how APEX2 influences gene expression, we conducted RNA-seq following APEX2 knockdown in hESC. These results indicated that a number of genes, in addition to TERT, relied on APEX2 for efficient expression. Genes affected by APEX2 knockdown were significantly enriched for specific repetitive DNA families. These include mammalian-wide interspersed repeats (MIRs) and Alu elements. Chromatin immunoprecipitation experiments demonstrated the highest APEX2 binding near MIR sequences in TERT intron 2. Surprisingly, binding was low in the TERT proximal promoter, a region known to control TERT transcription. MIR and other repetitive DNA regions are common sites of DNA damage, suggesting that APEX2 recruitment and repair of TERT MIR sequences may play a role in influencing TERT expression. This new role for APEX2 in promoting efficient gene expression deepens our understanding of an emerging cancer therapeutic target. Further, as the TERT gene plays critical roles in stem cell maintenance, organismal development and aging, as well as in short telomere disorders and cancer, our observations provide insight into new strategies to modulate the expression of this important enzyme.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c18f456e9f9ac926ada6d6d91f9b7e89f672dbdc" target='_blank'>
              Apurinic/apyrimidinic endodeoxyribonuclease 2 (APEX2/APE2) is required for efficient expression of TERT in human embryonic stem cells
              </a>
            </td>
          <td>
            Josh L. Stern, Lindsay F. Rizzardi, Natalie R. Gassman
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The BRCA2 tumor suppressor protein has roles in homologous recombination DNA repair and replication fork protection and carriers of mutation in the BRCA2 gene (BRCA2mut/+) are at high risk for breast cancers as a consequence of genomic instability. Mammary epithelial cells (MECs) are subjected to significant hormone-driven replication. BRCA2mut/+ MECs must therefore be able to mitigate RS-induced chronic DNA damage that could result in cell death or senescence. In response to acute RS in hTERT-MECs, we have found that the EGFR ligand, amphiregulin (AREG) rapidly undergoes retrograde trafficking to the nucleus and nuclear membrane (NM) in association with lamin A resulting in a transient increase in H3K9me3 heterochromatin most markedly in BRCA2mut/+ hTERT-MECs. RS promoted NM incorporation of prelamin A that was dependent on endogenous but not exogenous AREG. Prelamin A is reported to stabilize SUV39h1 and expression of a C-terminal truncated AREG that translocates to the NM increased SUV39h1, HP1α and H3K9me3. AREG knockdown reduced both HP1α and SUV39h1 accompanied by massive decompaction and reduction in global H3K9me3 heterochromatin. Loss of heterochromatin upon AREG depletion increased DNA damage, reduced replication fork speed and resulted in a significant increase in firing of new and multiple origins. Knockdown of endogenous AREG also impacted the NM resulting in activation of a type-1 interferon (IFN)-like response promoting senescence. Chronic RS also stimulated nuclear AREG along with FOXM1 and ESR1 mRNA and protein. Consistent with this, nuclear AREG, FOXM1 and ERα were detected in hyperplastic regions of human tissue. Overall, our work reveals a critical and unanticipated nuclear role for AREG in maintenance and transient induction of heterochromatin in response to RS preventing excessive DNA damage and senescence in BRCA2mut/+ MECs. Since nuclear AREG also promotes ERα expression, this may link RS with aberrant expansion of estrogen receptor-expressing MECs found in BRCA2mut/+ mammary tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7115be9e7c57e4cfb92a7b960c20389981e78db4" target='_blank'>
              Intracellular amphiregulin is critical for heterochromatin maintenance and genomic stability in response to replication stress in BRCA2mut/+mammary epithelial cells
              </a>
            </td>
          <td>
            Tianqi Jiang, A. Zein, M.A. Christine Pratt
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dff3afbe0101de03898d02a2c33635ef29e9865" target='_blank'>
              PRKACB is a novel imprinted gene in marsupials
              </a>
            </td>
          <td>
            Trent Newman, D. M. Bond, Teruhito Ishihara, Phoebe Rizzoli, Quentin Gouil, T. Hore, Geoff Shaw, Marilyn B. Renfree
          </td>
          <td>2024-09-28</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dd1ec7da9b646258b3f0f120b2ef1215f6414e" target='_blank'>
              EIF4A3-mediated oncogenic circRNA hsa_circ_0001165 advances esophageal squamous cell carcinoma progression through the miR-381-3p/TNS3 pathway
              </a>
            </td>
          <td>
            Xun Zhang, Yan Bian, Qiuxin Li, Chuting Yu, Ye Gao, Bo Tian, Wenqiang Xia, Wei Wang, Lei Xin, Han Lin, Luowei Wang
          </td>
          <td>2024-10-09</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) is a common malignant tumor with a poor prognosis. Long noncoding RNAs (lncRNAs) have recently gained attention for their pivotal role in regulating cancer progression, including CRC. This study aimed to investigate the biological mechanisms underlying the participation of long intergenic non-protein coding RNA 1134 (LINC01134) in the progression of CRC. Material and Methods: Quantitative Real-time-PCR (RT-qPCR) and western blot were applied to assess the expression levels of mRNA and protein. Functional experiments (CCK8 assay, colon formation assay, EdU assay and flow cytometry) were applied to assess cell viability and apoptosis. RNA-RNA interaction assays, subcellular fractionation analysis and dual luciferase reporter assays were employed to explore molecular interactions between LINC01134 and solute carrier family 1 member 5 (SLC1A5). The mRNA stability was analyzed using actinomycin D (ActD). Results: We found that LINC01134 expression was highly expressed in CRC tissues and positively correlated with advanced clinical stages and unfavorable prognosis, which is consistent with findings from CRC cell lines. Functional experiments showed that suppressing LINC01134 restrained the proliferation of CRC both in vitro and in vivo and induced apoptosis of CRC cells. Gene co-expression analysis revealed a positive relationship between LINC01134 and SLC1A5, which was also upregulated and associated with unfavorable prognosis in CRC. Further analysis of RNA interactions and mRNA stability revealed that LINC01134 directly binds to SLC1A5 mRNA, enhancing its stability. Remarkably, silencing SLC1A5 expression partially counteracted the promotion of CRC cell proliferation by LINC01134 overexpression and alleviated its inhibition of apoptosis. Conclusions: Our findings indicated that LINC01134 functioned as an oncogene in CRC by binding directly to SLC1A5 mRNA and increasing its stability. Therefore, targeting LINC01134 could be a potential therapeutic target for treating CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed35691af773ac6298dfcda195e4c6e969c81ece" target='_blank'>
              LINC01134 Directly Binds and Regulates SLC1A5 Stability to Promotes Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Li Yao, Jinxiu Wu, Xiaofeng Wang, Nailing Wang
          </td>
          <td>2024-10-07</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Transposable elements (TEs) are key contributors to genetic novelty. Despite increasing evidence of their importance, their roles in shaping the regulatory landscape of diverse immune cell populations remain largely unclear. Using single-cell multiome data from human peripheral blood mononuclear cells, we annotated the cell-specific cis-regulatory elements for major immune cell populations and identified a highly cell-specific signature of the overrepresented TE families. Focusing on monocytes that bear fast-evolving transcriptomes, we found that high proportions of their enhancers are TE-derived and bound by multiple pioneer transcription factors. Among them, we confirmed that the core myeloid regulator SPI1 can bind and regulate hundreds of TE-derived enhancers, which further affect the expression of adjacent immune genes. Additionally, interspecies comparison reveals that non-conserved monocyte enhancers are frequently generated by lineage-specific TE insertions, and correlate with the evolved gene expression between human and mouse. Overall, our study supports the importance of TEs in shaping the regulatory landscape of diverse immune cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec49246bea325aa9a378a750ef1fb9175032369" target='_blank'>
              Unraveling transposable element-mediated regulatory landscape in diverse human immune cells
              </a>
            </td>
          <td>
            Cui-Qiong Du, Hairui Fan, Jiayao Jiang, Juan Yang, Shuai Chen, Nadezhda E. Vorobyeva, Congrui Zhu, Liming Mao, Chenxi Li, Yanhua Li, Wen-hua Bao, Ming-an Sun
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CTCF-mediated chromatin folding plays a key role in gene regulation, however the mechanisms underlying cell type-specific control are not fully elucidated. Comprehensive analyses reveal that CTCF binding stability and cohesin overlap in mice and humans is regulated by species specific differences in accessibility, the presence of CTCF extended Up and Downstream binding site sequences and motifs corresponding to expressed TFs enriched at most bound sites. CTCF repositions nucleosomes asymmetrically, with linker spacing altered by accessibility, while cohesin-mediated nucleosome phasing is affected by surrounding motifs which also impact chromatin insulation. Importantly, cell type-specific transcriptional programs determine motif enrichment at CTCF bound sites, reflecting the stabilizing/destabilizing effect of individual TFs. These studies identify mechanisms underlying cell type-specific CTCF profiles, linked to local and long-range chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd7aecf488f24390e9186daea0b3c5eb50eab16" target='_blank'>
              A genome wide code to define cell-type specific CTCF binding and chromatin organization
              </a>
            </td>
          <td>
            Catherine Do, Guimei Jiang, Paul Zappile, Adriana Hegu, Jane A Skok
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The response to targeted therapies and immune checkpoint inhibitors for patients suffering from metastatic clear cell renal cell carcinoma (ccRCC) is heterogeneous and currently not predictable in clinic. In this work, a comprehensive integrated study of 700 ccRCCs profiled by DNA methylation and RNA sequencing showed that the hyper-methylated tumors exhibited a worse prognosis, a higher fraction of cycling tumor cells and a lower activity of homeobox transcription factors. To translate the use of DNA methylation information into a clinical setting, we developed a simple model accurately predicting the ccRCC methylation subtypes (AUC-ROCs of 0.91) from two gene expression ratios (IGF2BP3/PCCA, TNNT1/TMEM88). In addition, these methylation subtypes were significantly associated with the therapeutic outcome of patients to anti-PD-1, mTOR inhibitor or tyrosine kinase inhibitor therapies. Overall, our framework for predicting the ccRCC DNA methylation subtypes from targeted gene expression data is easy to translate in clinic and contributes to better personalization of ccRCC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0521226e2f11211160c8cebde1cb06c9530046ba" target='_blank'>
              DNA methylation is a key determinant of response to targeted and immune checkpoint therapies in metastatic renal cell carcinoma
              </a>
            </td>
          <td>
            Florian Jeanneret, Sarah Schoch, Pedro Ballester, Stefan N Symeonides, Alexander Laird, Håkan Axelson, Delphine Pflieger, Christophe Battail
          </td>
          <td>2024-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Topoisomerases alter DNA topology by making transient DNA strand breaks (DSBs) in DNA. The DNA cleavage reaction mechanism includes the formation of a reversible protein/DNA complex that allows rapid resealing of the transient break. This mechanism allows changes in DNA topology with minimal risks of persistent DNA damage. Nonetheless, small molecules, alternate DNA structures, or mutations in topoisomerase proteins can impede the resealing of the transient breaks, leading to genome instability and potentially cell death. The consequences of high levels of enzyme/DNA adducts differ for type I and type II topoisomerases. Top1 action on DNA containing ribonucleotides leads to 2–5 nucleotide deletions in repeated sequences, while mutant Top1 enzymes can generate large deletions. By contrast, small molecules that target Top2, or mutant Top2 enzymes with elevated levels of cleavage lead to small de novo duplications. Both Top1 and Top2 have the potential to generate large rearrangements and translocations. Thus, genome instability due to topoisomerase mis-function is a potential pathogenic mechanism especially leading to oncogenic progression. Recent studies support the potential roles of topoisomerases in genetic changes in cancer cells, highlighting the need to understand how cells limit genome instability induced by topoisomerases. This review highlights recent studies that bear on these questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc620e1f3abd707db6dd8102f3cc18d304a5ef7" target='_blank'>
              Genome Instability Induced by Topoisomerase Misfunction
              </a>
            </td>
          <td>
            K. C. Nitiss, Afif F. Bandak, James M. Berger, J. Nitiss
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Ovarian cancer is one of the most lethal gynecological malignancies worldwide, posing immense challenges in early detection, displaying grim prognoses, and having high rates of recurrence and mortality. This malignancy exhibits a range of ethnic diversity-mortality rates, each with distinct sub-categories characterized by variations in both molecular attributes and clinical behavior. The most prevalent and also most aggressive among the various ovarian cancer sub-types is high-grade serous ovarian carcinoma (HGSOC). This carcinoma is marked by genomic instability, resulting in the malfunction of tumor suppressor genes and oncogenes, caused by homologous recombination deficiency, copy number variations, chromosomal abnormalities, and epigenetic changes. The pursuit of understanding and targeting genomic instability has significantly guided approaches to screening, prognosis, and therapeutic strategies in ovarian cancer. Further insights from molecular investigations have unveiled the regulatory role of microRNAs (short non-coding RNAs) in modulating the expression of target tumor suppressors and oncogenes by interacting with messenger RNA (mRNA). Studies have shown that miRNAs are dysregulated in ovarian cancer, however, no data has shown significant relationships between these miRNAs to the patient demographics, including race/ethnicity. miRNAs can alter oncogenes, and tumor suppressor gene expression levels, serving as early biomarkers. We utilized the TCGA database from the University of Alabama at Birmingham (UALCAN) software to analyze the expressions of targeted genes and miRNAs. We determined various oncogenes and tumor suppressor genes with higher statistical significance on survival rates, race, cancer stages, tumor grades, and miRNAs linked to them. Additionally, genomic instability was assessed using aneuploidy score, fraction of genome altered, mutation counts, mutation burden, and prognosis (i.e., overall survival and progression-free survival), downloaded from cBioPortal, and the genomic instability parameters were calculated by Spearman correlation, and represented by a heatmap. Further, DNA and RNA from 22 HGSOC samples of paraffin-embedded tissues of ovarian cancer patients were collected for RT-qPCR assays for several miRNAs. Of the genes analyzed, BRCA1, BRAF, CCNE1, and miRNAs linked to these genes showed statistically significant data on survival rates. BRAF/CCNE1 is upregulated while BRCA 1 is downregulated in ovarian high-grade serous carcinoma. We also found quantitative differential expressions of some miRNAs analyzed in African Americans versus White Americans. Successful identification of early molecular markers in ovarian cancer will potentially allow for early detection and decreased mortality rates with HGSOC. Analysis of the appearance and abundance of miRNAs by qPCR assay in early-stage ovarian cancer can aid as an early diagnostic and prognostic tool. In a further study, we hope to verify our results by testing a larger cohort of samples from African Ancestry.
 Citation Format: Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga. miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f1e0052202e2129f599047f6e77f66413139b6" target='_blank'>
              Abstract C075: miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry
              </a>
            </td>
          <td>
            Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac0738a96662e313aea5fd9200de2f16be79bfc" target='_blank'>
              R-loops’ m6A modification and its roles in cancers
              </a>
            </td>
          <td>
            Yue Qiu, Changfeng Man, Luyu Zhu, Shiqi Zhang, Xiaoyan Wang, Dandan Gong, Yu Fan
          </td>
          <td>2024-10-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Fibroblast Growth Factor Receptor 2 inhibition presents a promising therapeutic approach for restraining the growth and survival of cancer cells, particularly in breast tumours. However, the emergence of resistance to FGFR2 inhibitors like PD173074 highlights the importance of understanding the molecular mechanisms driving resistance and identifying effective therapeutic strategies. In this study, we employed temporal quantitative proteomics and phosphoproteomics, complemented by computational clustering, PTM-SEA analysis, and kinase activity prediction, to monitor the response of MFM223 triple negative breast cancer cells to FGFR2 inhibition. Strikingly, we observed a marked enrichment of ribosome biogenesis function modules immediately following treatment, a phenomenon not observed with other inhibitors. Additionally, we discovered that CX-5461, an RNA polymerase I inhibitor, synergistically enhanced the growth inhibition induced by PD173074, mechanistically attributed to its significant suppression of rDNA transcription stimulated by PD173074. Moreover, our phosphoproteomics dynamic profiling identified the clustering of kinases within MAPK and ErbB signalling pathways, indicative of their reactivation in response to FGFR2 inhibition. Experimentally validating this finding, we observed a notable rebound in phosphorylation levels of key kinases such as ERK1/2 and ErbB3, and demonstrated a substantial synergistic effect of PD173074 in combination with Trametinib, a MEK inhibitor, in suppressing cancer cell growth. Collectively, our findings provide critical insights into the network rewiring triggered by FGFR2 inhibition and offer a foundation for the rational design of combinatorial therapeutic strategies to overcome resistance mechanisms associated with FGFR2 inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc089ba7e1dffd9ff2361b087cdde2575bba907" target='_blank'>
              Ribosomal Biogenesis Hyperactivation and ErbB signalling Mediated Network Rewiring Causes Adaptive Resistance to FGFR2 Inhibition
              </a>
            </td>
          <td>
            Tao Zang, Sung-young Shin, Callan McCrimmon, Mandy Theocharous, Ralf B. Schittenhelm, Thierry Jarde, Roger J. Daly, Lan K. Nguyen
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract The heterogeneity of tumor clones drives the selection and evolution of distinct tumor cell populations, resulting in an intricate and dynamic tumor evolution process. While tumor bulk DNA sequencing helps elucidate intratumor heterogeneity, challenges such as the misidentification of mutation multiplicity due to copy number variations and uncertainties in the reconstruction process hinder the accurate inference of tumor evolution. In this study, we introduce a novel approach, REconstructing Tumor Clonal Heterogeneity and Evolutionary Relationships (RETCHER), which characterizes more realistic cancer cell fractions by accurately identifying mutation multiplicity while considering uncertainty during the reconstruction process and the credibility and reasonableness of subclone clustering. This method comprehensively and accurately infers multiple forms of tumor clonal heterogeneity and phylogenetic relationships. RETCHER outperforms existing methods on simulated data and infers clearer subclone structures and evolutionary relationships in real multisample sequencing data from five tumor types. By precisely analysing the complex clonal heterogeneity within tumors, RETCHER provides a new approach to tumor evolution research and offers scientific evidence for developing precise and personalized treatment strategies. This approach is expected to play a significant role in tumor evolution research, clinical diagnosis, and treatment. RETCHER is available for free at https://github.com/zlsys3/RETCHER.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f822108aa29d6372cc1d56eba22e5f8f0a1fffc3" target='_blank'>
              Reconstructing tumor clonal heterogeneity and evolutionary relationships based on tumor DNA sequencing data
              </a>
            </td>
          <td>
            Zhen Wang, Yanhua Fang, Ruoyu Wang, Liwen Kong, Shanshan Liang, Shuai Tao
          </td>
          <td>2024-09-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Non-melanoma skin cancer (NMSC) mainly includes basal cell carcinoma, cutaneous squamous cell carcinoma, and Merkel cell carcinoma, showing a low mortality rate but the highest incidence worldwide. In recent decades, research has focused on understanding the pathogenesis and clinical treatments of NMSC, leading to significant advances in our knowledge of these diseases and the development of novel therapies, including immunotherapy. Nevertheless, the low to moderate objective response rate, high recurrence, and therapeutic resistance remain persistent challenges, which are partly attributable to the intratumoral heterogeneity. This heterogeneity indicates that tumor cells, immune cells, and stromal cells in the tumor microenvironment can be reshaped to a series of phenotypic and transcriptional cell states that vary in invasiveness and treatment responsiveness. The advent of single-cell RNA sequencing (scRNA-seq) has enabled the comprehensive profiling of gene expression heterogeneity at the single-cell level, which has been applied to NMSC to quantify cell compositions, define states, understand tumor evolution, and discern drug resistance. In this review, we highlight the key findings, with a focus on intratumoral heterogeneity and the mechanism of drug resistance in NMSC, as revealed by scRNA-seq. Furthermore, we propose potential avenues for future research in NMSC using scRNA-seq.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5efda06af7a4c8d9343615ef50b870bd5ba887" target='_blank'>
              Unraveling the landscape of non-melanoma skin cancer through single-cell RNA sequencing technology
              </a>
            </td>
          <td>
            G. Yan, Xiuli Wang, Guolong Zhang
          </td>
          <td>2024-11-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The p53 tumor suppressor is an indispensable regulator of DNA damage responses that accelerates carcinogenesis when mutated. In this report, we uncover a new mechanism by which p53 maintains genomic integrity in the absence of canonical DNA damage response activation. Specifically, loss of p53 dramatically alters chromatin structure at the nuclear periphery, allowing increased transmission of an environmental carcinogen, ultraviolet (UV) radiation, into the nucleus. Genome-wide mapping of UV-induced DNA lesions in p53-deficient primary cells reveals elevated lesion abundance in regions corresponding to locations of high mutation burden in malignant melanomas. These findings uncover a novel role of p53 in the suppression of mutations that contribute to cancer and highlight the critical influence of nuclear architecture in regulating sensitivity to carcinogens. One-Sentence Summary The p53 tumor suppressor reduces carcinogen sensitivity and mutagenic potential by maintaining nuclear architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce95350c5fb375408fd587e0eea1b652005b9ec" target='_blank'>
              p53 Regulates Nuclear Architecture to Reduce Carcinogen Sensitivity and Mutagenic Potential
              </a>
            </td>
          <td>
            Devin A. King, Dakota E. McCoy, Adrian Perdyan, Jakub Mieczkowski, Thierry Douki, Jennifer A. Dionne, Rafael E. Herrera, Ashby J. Morrison
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT Esophageal squamous cell carcinoma (ESCC) is an aggressive malignant neoplasm, and up to now, the role of long non-coding RNA (lncRNA) AP001885.4 in cancer, including ESCC, is absolutely unclear. The GEPIA database was applied to identify differentially expressed and prognosis-associated genes in esophageal cancer (ESCA). CCK-8, colony formation, Western blot, and qRT-PCR methods were harnessed to investigate the role and mechanism of AP001885.4 in esophageal carcinogenesis. By analyzing TCGA data in the GEPIA database, two lncRNAs were selected. AP001885.4 was overexpressed and positively associated with the unfavorable outcome of ESCC patients, and LINC001786 was under-expressed and negatively linked with the poor prognosis. Knockdown of AP001885.4 suppressed the proliferation and colony formation of ESCC cells. Importantly, the silence of AP001885.4 downregulated c-myc. Mechanically, the knockdown of AP001885.4 reduced METTL3 expression and m6A modification in c-myc mRNA, and METTL3 positively regulated c-myc. Furthermore, the knockdown of AP001885.4 diminished histone lactylation and NF-κB (p65) expression, and the protein lactylation inhibitors (2-DG, 2-deoxy-D-glucose and oxamate) and the NF-κB inhibitor (JSH-23) also lessened c-myc expression. Consequently, our findings suggested that AP001885.4 promoted the proliferation of esophageal squamous cell carcinoma cells by histone lactylation- and NF-κB (p65)-dependent transcription activation and METTL3-mediated mRNA stability of c-myc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc1a10205f412d6e9e461bcb67c0812d136bd4c" target='_blank'>
              AP001885.4 promotes the proliferation of esophageal squamous cell carcinoma cells by histone lactylation- and NF-κB (p65)-dependent transcription activation and METTL3-mediated mRNA stability of c-myc
              </a>
            </td>
          <td>
            C. Fu, Wen Jiang, Chong Wang, Sheng-Jie Song, Hao Tao, Xin-Guo Zhang, Wen-Ting Li, Xin Jin, Bin-Bing Yu, Jia-Jie Hao, Wen-Juan Sun, Jie Bai, Zhi-Zhou Shi
          </td>
          <td>2024-11-01</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b023a760b40f6707fa684ea79bd76ea869f1a" target='_blank'>
              PER3 promoter hypermethylation correlates to the progression of pan-cancer
              </a>
            </td>
          <td>
            Yaoxu Li, Wenjuan Li, Jinhai Deng, Mingzhu Yin
          </td>
          <td>2024-10-14</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved tool for the removal of DNA double-strand breaks (DSBs) and the preservation of stalled and damaged DNA replication forks. Successful completion of HR requires the tumor suppressor BRCA2. Germline mutations in BRCA2 lead to familial breast, ovarian, and other cancers, underscoring the importance of this protein for maintaining genome stability. BRCA2 harbors two distinct DNA binding domains, one that possesses three oligonucleotide/oligosaccharide binding (OB) folds (known as the OB-DBD), and with the other residing in the C-terminal recombinase binding domain (termed the CTRB-DBD) encoded by the last gene exon. Here, we employ a combination of genetic, biochemical, and cellular approaches to delineate contributions of these two DNA binding domains toward HR and the maintenance of stressed DNA replication forks. We show that OB-DBD and CTRB-DBD confer ssDNA and dsDNA binding capabilities to BRCA2, respectively, and that BRCA2 variants mutated in either DNA binding domain are impaired in the ability to load the recombinase RAD51 onto ssDNA pre-occupied by RPA. While the CTRB-DBD mutant is modestly affected for HR, it exhibits a strong defect in the protection of stressed replication forks. In contrast, the OB-DBD is indispensable for both BRCA2 functions. Our study thus defines the unique contributions of the two BRCA2 DNA binding domains in genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b867a23dce384adad0b091d20596685ac211bb92" target='_blank'>
              Distinct roles of the two BRCA2 DNA binding domains in DNA damage repair and replication fork preservation
              </a>
            </td>
          <td>
            Francisco Neal, Wenjing Li, Mollie E. Uhrig, Neelam Sharma, Shahrez Syed, Sandeep Burma, R. Hromas, Alexander V. Mazin, Eloise Dray, David S. Libich, Shaun K. Olsen, Elizabeth V. Wasmuth, Weixing Zhao, Claus S. Sørensen, C. Wiese, Youngho Kwon, Patrick Sung
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The determination of the cancer prognosis is paramount for patients and medical personnel so that they can devise treatment strategies. Transcriptional-based signatures and subtypes derived from cancer biopsy material have been used in clinical practice for several cancer types to aid in setting the patient prognosis and forming treatment strategies. Other genomic features in cancer biopsies, such as copy number alterations (CNAs), have been underused in clinical practice, and yet they represent a complementary source of molecular information that can add detail to the prognosis, which is supported by recent work in breast, ovarian, and lung cancers. Here, through a systematic strategy, we explored the prognostic power of CNAs in 37 cancer types. In this analysis, we defined two modes of informative features, deep and soft, depending on the number of alleles gained or lost. These informative modes were grouped by amplifications or deletions to form four single-data prognostic models. Finally, the single-data models were summed or combined to generate four additional multidata prognostic models. First, we show that the modes of features are cancer-type dependent, where deep alterations generate better models. Nevertheless, some cancers require soft alterations to generate a feasible model due to the lack of significant deep alterations. Then, we show that the models generated by summing coefficients from amplifications and deletions appear to be more practical for many but not all cancer types. We show that the CNA-derived risk group is independent of other clinical factors. Furthermore, overall, we show that CNA-derived models can define clinically relevant risk groups in 33 of the 37 (90%) cancer types analyzed. Our study highlights the use of CNAs as biomarkers that are potentially clinically relevant to survival in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46f0ca721f6c654c87b0f219cadce02b53e07fbf" target='_blank'>
              Systematic Modeling of Risk-Associated Copy Number Alterations in Cancer
              </a>
            </td>
          <td>
            Alejandra Guardado, Raúl Aguirre-Gamboa, Víctor Treviño
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA damage tolerance pathways that allow for the completion of replication following fork arrest are critical in maintaining genome stability during cell division. The main DNA damage tolerance pathways include strand switching, replication fork reversal and translesion synthesis (TLS). The TLS pathway is mediated by specialised DNA polymerases that can accommodate altered DNA structures during DNA synthesis, and are important in allowing replication to proceed after fork arrest, preventing fork collapse that can generate more deleterious double-strand breaks in the genome. TLS may occur directly at the fork, or at gaps remaining behind the fork, in the process of post-replication repair. Inactivating mutations in the human POLH gene encoding the Y-family DNA polymerase Pol η causes the skin cancer-prone genetic disease xeroderma pigmentosum variant (XPV). Pol η also contributes to chemoresistance during cancer treatment by bypassing DNA lesions induced by anti-cancer drugs including cisplatin. We review the current understanding of the canonical role of Pol η in translesion synthesis following replication arrest, as well as a number of emerging non-canonical roles of the protein in other aspects of DNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/247363b13f865e5ca1882a279d7c40f1f96c114f" target='_blank'>
              Canonical and Non-Canonical Roles of Human DNA Polymerase η
              </a>
            </td>
          <td>
            Salma Bedaiwi, Anam Usmani, Michael P. Carty
          </td>
          <td>2024-09-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gallbladder cancer (GBC) is a highly aggressive malignancy lacking clinically available targeted therapeutic agents. Super-enhancers (SEs) are crucial epigenetic cis-regulatory elements whose extensive reprogramming drives aberrant transcription in cancers. To study SE in GBC, the genomic distribution of H3K27ac is profiled in multiple GBC tissue and cell line samples to establish the SE landscape and its associated core regulatory circuitry (CRC). The biliary lineage factor SOX9 and Wnt pathway effector TCF7L2, two master transcription factor (TF) candidates identified by CRC analysis, are verified to co-occupy each other's SE region, forming a mutually autoregulatory loop to drive oncogenic SE reprogramming in a subset of GBC. The SOX9/TCF7L2 double-high GBC cells are highly dependent on the two TFs and enriched of SE-associated gene signatures related to stemness, ErbB and Wnt pathways. Patients with more such GBC cells exhibited significantly worse prognosis. Furthermore, SOX9/TCF7L2 double-high GBC preclinical models are found to be susceptible to SE-targeted CDK7 inhibition therapy in vitro and in vivo. Together, this study provides novel insights into the epigenetic mechanisms underlying the oncogenesis of a subset of GBCs with poorer prognosis and illustrates promising prognostic stratification and therapeutic strategies for treating those GBC patients in future clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ddd86e82fc0ab68dcd9c2f37791c7eb9588c417" target='_blank'>
              Super-Enhancer Reprograming Driven by SOX9 and TCF7L2 Represents Transcription-Targeted Therapeutic Vulnerability for Treating Gallbladder Cancer.
              </a>
            </td>
          <td>
            Siyuan Yan, Zhaonan Liu, Teng Wang, Yi Sui, Xiangsong Wu, Jiayi Shen, Peng Pu, Yang Yang, Sizhong Wu, Shimei Qiu, Ziyi Wang, Xiaoqing Jiang, Feiling Feng, Guoqiang Li, Fatao Liu, Chaoxian Zhao, Ke Liu, Jiayi Feng, Maolan Li, Kwan Man, Chaochen Wang, Yujie Tang, Yingbin Liu
          </td>
          <td>2024-11-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d92fbde3dae7404c9f30bd22e44407b07e5261a" target='_blank'>
              Self-sustaining long-term 3D epithelioid cultures reveal drivers of clonal expansion in esophageal epithelium
              </a>
            </td>
          <td>
            A. Herms, D. Fernandez-Antoran, M. Alcolea, Argyro Kalogeropoulou, Ujjwal Banerjee, G. Piedrafita, E. Abby, J. A. Valverde-Lopez, Inês S Ferreira, Irene Caseda, M. T. Bejar, S. Dentro, Sara Vidal-Notari, S. Ong, B. Colom, K. Murai, C. King, Krishnaa Mahbubani, K. Saeb‐Parsy, Alan R Lowe, M. Gerstung, P. Jones
          </td>
          <td>2024-09-23</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="
 Purpose: Desmoid tumors are bland fibroblastic tumors that do not metastasize but have a high rate of local recurrence. Previously published studies proposed two different transcriptomic signatures to predict relapse. Molecular heterogeneity has been well established in high-grade sarcomas but little is known about molecular variability within locally aggressive tumors such as desmoids. Experimental Design: We performed transcriptomic profiling of 31 specimens from 20 primary desmoid tumors to identify genes predictive of relapse. We also performed multi-omic analysis including DNA methylation, copy number alterations, point mutations and gene expression on 24 specimens from different regions of primary and recurrent desmoid tumors from 3 patients (7-9 specimens per patient). Results: We observed highly variable expression of transcriptomic prognostic signatures, both in patients who did or did not progress. Signatures associated with favorable and unfavorable outcome were detected in different regions within the same tumor. Further multi-omic studies showed remarkable intra- and inter-tumor heterogeneity of genomic, epigenomic and transcriptomic patterns. The transcriptomic profiles showed the highest degree of variability within tumors and between primary and recurrent tumors from the same patient. Conclusions: This study shows an unexpected degree of intra- and inter-tumor heterogeneity in desmoid tumors. Our analysis indicates that molecular analysis of a single tumor biopsy may underestimate the magnitude of molecular alterations in desmoid tumors. Our study also shows that recurrent desmoid tumors acquire multiple new molecular alterations. Thus, molecular heterogeneity is an important consideration in drug development and validation of prognostic and predictive biomarkers for desmoid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50e3a34897b4df8b167fa47ca15d74b0fa8a5419" target='_blank'>
              Genomic, epigenomic and transcriptomic inter- and intra-tumor heterogeneity in desmoid tumors
              </a>
            </td>
          <td>
            Chelsea De Bellis, S. Vennam, C. Eeles, Pegah Rahimizadeh, Justin Cates, Thomas Stricker, Jenny Hoffman, Kristen N. Ganjoo, Gregory W. Charville, Benjamin Haibe-Kains, Matt van de Rijn, Joanna Przybyl
          </td>
          <td>2024-11-08</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Transcription of actively expressed genes is dampened for kilobases around DNA lesions via chromatin modifications. This is believed to favour repair and prevent genome instability. Nonetheless, mounting evidence suggests that transcription may be induced by DNA breakage, resulting in the local de novo synthesis of non-coding RNAs (ncRNAs). Such transcripts have been proposed to play important functions in both DNA damage signalling and repair. Here, we review the recently identified mechanistic details of transcriptional silencing at damaged chromatin, highlighting how post-translational histone modifications can also be modulated by the local synthesis of DNA damage-induced ncRNAs. Finally, we envision that these entangled transcriptional events at DNA breakages can be targeted to modulate DNA repair, with potential implications for locus-specific therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8045eca3080065650673ed1f8657736a081879a" target='_blank'>
              Traffic light at DSB-transit regulation between gene transcription and DNA repair.
              </a>
            </td>
          <td>
            Stefania Modafferi, Francesca Esposito, Sara Tavella, Ubaldo Gioia, S. Francia
          </td>
          <td>2024-09-27</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4788ed4991cb05a8733cec69abe88f6738bf0a69" target='_blank'>
              Differential chromatin accessibility and transcriptional dynamics define breast cancer subtypes and their lineages.
              </a>
            </td>
          <td>
            Michael D. Iglesia, R. Jayasinghe, Siqi Chen, Nadezhda V. Terekhanova, John M Herndon, E. Storrs, A. Karpova, D. Zhou, Nataly Naser Al Deen, Andrew T Shinkle, Rita Jui-Hsien Lu, Wagma Caravan, Andrew Houston, Yanyan Zhao, Kazuhito Sato, Preet Lal, Cherease Street, Fernanda Martins Rodrigues, Austin N. Southard-Smith, A. L. N. Targino da Costa, Houxiang Zhu, Chia-Kuei Mo, Lisa Crowson, Robert S Fulton, Matthew A. Wyczalkowski, C. Fronick, L. Fulton, Hua Sun, Sherri R Davies, Elizabeth L. Appelbaum, S. Chasnoff, Madelyn Carmody, Candace Brooks, Ruiyang Liu, M. Wendl, C. Oh, Diane E Bender, C. Cruchaga, O. Harari, Andrea L Bredemeyer, K. Lavine, R. Bose, J. Margenthaler, Jason M. Held, Samuel Achilefu, F. Ademuyiwa, R. Aft, Cynthia X Ma, G. A. Colditz, Tao Ju, Stephen T. Oh, James A J Fitzpatrick, E. S. Hwang, K. Shoghi, M. Chheda, D. Veis, Feng Chen, Ryan C. Fields, William E Gillanders, Li Ding
          </td>
          <td>2024-10-30</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="The cohesin protein complex extrudes chromatin loops, stopping at CTCF-bound sites, to organize chromosomes into topologically associated domains, yet the biological implications of this process are poorly understood. We show that cohesin is required for the post-mitotic differentiation and function of antigen-presenting dendritic cells (DCs), particularly for antigen cross-presentation and IL-12 secretion by type 1 conventional DCs (cDC1s) in vivo. The chromatin organization of DCs was shaped by cohesin and the DC-specifying transcription factor IRF8, which controlled chromatin looping and chromosome compartmentalization, respectively. Notably, optimal expression of IRF8 itself required CTCF/cohesin-binding sites demarcating the Irf8 gene. During DC activation, cohesin was required for the induction of a subset of genes with distal enhancers. Accordingly, the deletion of CTCF sites flanking the Il12b gene reduced IL-12 production by cDC1s. Our data reveal an essential role of cohesin-mediated chromatin regulation in cell differentiation and function in vivo, and its bi-directional crosstalk with lineage-specifying transcription factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9ce66c6782a6994a188bf7a82871c7e4fe008d8" target='_blank'>
              Cohesin-mediated chromatin remodeling controls the differentiation and function of conventional dendritic cells
              </a>
            </td>
          <td>
            Nicholas M. Adams, Alexandra Galitsyna, I. Tiniakou, E. Esteva, Colleen M. Lau, Jojo Reyes, Nezar Abdennur, Alexey Shkolikov, George S. Yap, Alireza Khodadadi-Jamayran, L. Mirny, Boris Reizis
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="DNA double-strand break (DSB) repair pathways are crucial for maintaining genome stability and cell viability. However, these pathways can mistakenly recognize chromosome ends as DNA breaks, leading to adverse outcomes such as telomere fusions and malignant transformation. The shelterin complex protects telomeres from activation of DNA repair pathways by inhibiting nonhomologous end joining (NHEJ), homologous recombination (HR), and microhomology-mediated end joining (MMEJ). The focus of this paper is on MMEJ, an error-prone DSB repair pathway characterized by short insertions and deletions flanked by sequence homology. MMEJ is critical in mediating telomere fusions in cells lacking the shelterin complex and at critically short telomeres. Furthermore, studies suggest that MMEJ is the preferred pathway for repairing intratelomeric DSBs and facilitates escape from telomere crisis. Targeting MMEJ to prevent telomere fusions in hematologic malignancies is of potential therapeutic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ae75e964c3e7cfe4dbb822826b27c518fe45ac" target='_blank'>
              The Role of Microhomology-Mediated End Joining (MMEJ) at Dysfunctional Telomeres.
              </a>
            </td>
          <td>
            David Billing, Agnel Sfeir
          </td>
          <td>2024-11-05</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="DNA lies at the heart of the central dogma of life. Altering DNA can modify the flow of information in fundamental cellular processes such as transcription and translation. The ability to precisely manipulate DNA has led to remarkable advances in treating incurable human genetic ailments and has changed the landscape of biological research. Genome editors such as CRISPR-Cas nucleases and TALENs have become ubiquitous tools in basic and applied biological research and have been translated to the clinic to treat patients. The specificity and modularity of these genome editors have made it possible to efficiently engineer genomic DNA; however, underlying principles governing editing outcomes in eukaryotes are still being uncovered. Editing efficiency can vary from cell type to cell type for the same DNA target sequence, necessitating de novo design and validation efforts. Chromatin structure and epigenetic modifications have been shown to affect the activity of genome editors because of the role they play in hierarchical organization of the underlying DNA. Understanding the nuclear search mechanism of genome editors and their molecular interactions with higher order chromatin will lead to improved models for predicting precise genome editing outcomes. Insights from such studies will unlock the entire genome to be engineered for the creation of novel therapies to treat critical illnesses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de15652cba672686fe241fc8c1aaf364c6e3c7f6" target='_blank'>
              Impact of Chromatin Organization and Epigenetics on CRISPR-Cas and TALEN Genome Editing.
              </a>
            </td>
          <td>
            Surbhi Jain, Guanhua Xun, Huimin Zhao
          </td>
          <td>2024-09-24</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf19432cc15369ce28dc15aa8918b2d7b3bd99a" target='_blank'>
              TP53: the unluckiest of genes?
              </a>
            </td>
          <td>
            A. Joerger, Thorsten Stiewe, Thierry Soussi
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1dca550adace3299de2700d468a5c183e07ebc1" target='_blank'>
              Widespread mutagenesis and chromosomal instability shape somatic genomes in systemic sclerosis
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Thomas Blouin, James McCollum, Latarsha Porcher, François Virard, Jiri Zavadil, Carol Feghali-Bostwick, Natalie Saini
          </td>
          <td>2024-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1f558b1760a8e01aa262dc80b32d0f2f31634b" target='_blank'>
              CISD3 is a prognostic biomarker and therapeutic target in pan-cancer
              </a>
            </td>
          <td>
            Jie Li, Han Yang, Yixin Qi, Peng Yu, Xiahui Han, Z. Zhang, Kai Zhao, Xinbao Yin, Guanqun Zhu, Xuechuan Yan, Zaiqing Jiang, Xuezhen Ma, Tianzhen He, Ke Wang
          </td>
          <td>2024-10-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4229ea75fc458fb4ee692eac3c38175128262ee2" target='_blank'>
              Effect of fibroblast heterogeneity on prognosis and drug resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Tingjie Wang, Lingxi Tian, Bing Wei, Jun Li, Cuiyun Zhang, Ruitao Long, Xiaofei Zhu, Yougai Zhang, Bo Wang, Guangbo Tang, Jun Yang, Yongjun Guo
          </td>
          <td>2024-11-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1844470abca3c1fa6252bba317101f9730f2c0f4" target='_blank'>
              5-Formylcytosine: a new epigenetic player
              </a>
            </td>
          <td>
            Dharmendra Kumar, Iqbal Hyder, Wilfried A. Kues
          </td>
          <td>2024-10-23</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2b209080acf772da88d3de252f9c4248e66e84" target='_blank'>
              Hallmarks of a genomically distinct subclass of head and neck cancer
              </a>
            </td>
          <td>
            Tara Muijlwijk, I. Nauta, Anabel van der Lee, Kari J T Grünewald, A. Brink, Sonja H Ganzevles, Robert J. Baatenburg de Jong, L. Atanesyan, S. Savola, M. A. van de Wiel, Laura A N Peferoen, Elisabeth Bloemena, R. van de Ven, C. R. Leemans, J. Poell, R. Brakenhoff
          </td>
          <td>2024-10-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>3</td>
          <td>1</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The repair of DNA double-strand breaks (DSBs) through homologous recombination (HR) is vital for maintaining the stability and integrity of the genome. RNA binding proteins (RBPs) intricately regulate the DNA damage repair process, yet the precise molecular mechanisms underlying their function remain incompletely understood. In this study, we highlight the pivotal role of RPS15, a representative RBP, in homologous recombination repair. Specifically, we demonstrate that RPS15 promotes DNA end resection, a crucial step in homologous recombination. Notably, we identify an interaction between RPS15 and CtIP, a key factor in homologous recombination repair. This interaction is essential for CtIP recruitment to DSB sites, subsequent RPA coating, and RAD51 replacement, all critical steps in efficient homologous recombination repair and conferring resistance to genotoxic treatments. Functionally, suppressing RPS15 expression sensitizes cancer cells to X-ray radiation and enhances the therapeutic synergistic effect of PARP1 inhibitors in breast cancer cells. In summary, our findings reveal that RPS15 promotes DNA end resection to ensure effective homologous recombination repair, suggesting its potential as a therapeutic target in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2aedc66ec12ddcad5319b720ce461d7c9233af5" target='_blank'>
              RPS15 Coordinates with CtIP to Facilitate Homologous Recombination and Confer Therapeutic Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Baohang Lin, Guan Huang, Zishan Yuan, Xun Peng, Chunliang Yu, Jialu Zheng, Zequn Li, Juanyun Li, Jinan Liang, Bo Xu
          </td>
          <td>2024-10-03</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Elucidating the mechanisms underlying cancer development and proliferation is important for the development of therapeutic methods for the complete cure of cancer. In particular, the identification of diagnostic markers for early detection and new therapeutic strategies for refractory gastrointestinal cancers are needed. Various abnormal phenomena occur in cancer cells, such as functional changes of proteins, led by genomic mutations, and changes in gene expression due to dysregulation of epigenetic regulation. This is no exception for noncoding RNA (ncRNA), which do not encode proteins. Recent reports have revealed that microRNA (miRNA), long noncoding RNA (lncRNA), and circular RNA (circRNA) are deeply involved in cancer progression. These ncRNAs have attracted attention as gene expression regulatory molecules. Recent advances in technology have made it possible not only to read DNA and RNA sequences but also to study the modification state of each base. In particular, comprehensive analysis of N6-methyladenosine (m6A) has been performed by many research groups, with multiple studies reporting that m6A modifications of specific genes are associated with cancer progression. Based on the above, this review examines how ncRNA modifications are related to cancer progression in gastrointestinal cancers such as colorectal and pancreatic cancer. We also discuss enzyme inhibitors that have been reported to have drug discovery potential targeting m6A modifications. By utilizing the new perspective of ncRNA modification, we may be able to accumulate knowledge on the molecular biology of cancer and contribute to human health through diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24e2ef3283a2f1c095d3a859b796d319aaa73ec" target='_blank'>
              Recent advances in noncoding RNA modifications of gastrointestinal cancer.
              </a>
            </td>
          <td>
            T. Hara, Sikun Meng, Yasuko Arao, Yoshiko Saito, Kana Inoue, Sarah Rennie, Ken Ofusa, Y. Doki, H. Eguchi, Toru Kitagawa, Hideshi Ishii
          </td>
          <td>2024-11-01</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f8baa71bbf0cba21e18b948cf25d10831f4654" target='_blank'>
              Protein-RNA interaction dynamics reveal key regulators of oncogenic KRAS-driven cancers
              </a>
            </td>
          <td>
            Ka-Yun Ban, Yong-woo Na, Juhan Song, Jong-Seo Kim, Jimi Kim
          </td>
          <td>2024-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The switch/sucrose non-fermentable (SWI/SNF) subfamily are evolutionarily conserved, ATP-dependent chromatin-remodelling complexes that alter nucleosome position and regulate a spectrum of nuclear processes, including gene expression, DNA replication, DNA damage repair, genome stability and tumour suppression. These complexes, through their ATP-dependent chromatin remodelling, contribute to the dynamic regulation of genetic information and the maintenance of cellular processes essential for normal cellular function and overall genomic integrity. Mutations in SWI/SNF subunits are detected in 25% of human malignancies, indicating that efficient functioning of this complex is required to prevent tumourigenesis in diverse tissues. During development, SWI/SNF subunits help establish and maintain gene expression patterns essential for proper cellular identity and function, including maintenance of lineage-specific enhancers. Moreover, specific molecular signatures associated with SWI/SNF mutations, including disruption of SWI/SNF activity at enhancers, evasion of G0 cell cycle arrest, induction of cellular plasticity through pro-oncogene activation and Polycomb group (PcG) complex antagonism, are linked to the initiation and progression of carcinogenesis. Here, we review the molecular insights into the aetiology of human malignancies driven by disruption of the SWI/SNF complex and correlate these mechanisms to their developmental functions. Finally, we discuss the therapeutic potential of targeting SWI/SNF subunits in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8b74139f9c3a18f8d669933f43897d2c9b0b42" target='_blank'>
              Opening and changing: mammalian SWI/SNF complexes in organ development and carcinogenesis
              </a>
            </td>
          <td>
            Fadia Abu Sailik, B. Emerald, S. Ansari
          </td>
          <td>2024-10-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is an immensely complex ecosystem1,2. This complexity underlies difficulties in elucidating principles of spatial organization and using molecular profiling of the TME for clinical use3. Through statistical analysis of 96 spatial transcriptomic (ST-seq) datasets spanning twelve diverse tumor types, we found a conserved distribution of multicellular, transcriptionally covarying units termed ‘Spatial Groups’ (SGs). SGs were either dependent on a hierarchical local spatial context – enriched for cell-extrinsic processes such as immune regulation and signal transduction – or independent from local spatial context – enriched for cell-intrinsic processes such as protein and RNA metabolism, DNA repair, and cell cycle regulation. We used SGs to define a measure of gene spatial heterogeneity – ‘spatial lability’ – and categorized all 96 tumors by their TME spatial lability profiles. The resulting classification captured spatial variation in cell-extrinsic versus cell-intrinsic biology and motivated class-specific strategies for therapeutic intervention. Using this classification to characterize pre-treatment biopsy samples of 16 non-small cell lung cancer (NSCLC) patients outside our database distinguished responders and non-responders to immune checkpoint blockade while programmed death-ligand 1 (PD-L1) status and spatially unaware bulk transcriptional markers did not. Our findings show conserved principles of TME spatial biology that are both biologically and clinically significant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a400ff77ff3e3d5b4307dd66b458357a7b860383" target='_blank'>
              Conserved principles of spatial biology define tumor heterogeneity and response to immunotherapy
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, C. Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: MCM8 and MCM9 are newly proposed cancer predisposition genes, linked to polyposis and early-onset cancer, in addition to their association with hypogonadism. Given the uncertain range of phenotypic manifestations and unclear cancer risk estimates, this study aimed to delineate the molecular and clinical characteristics of individuals with biallelic germline MCM8/MCM9 variants. Methods: Population allele frequencies and biallelic variant carrier frequencies were calculated using data from gnomAD, and a variant enrichment analysis was conducted across multiple cancer and non-cancer phenotypes using data from the 100K Genomes Project and the 200K exome release of the UK Biobank. A case series was conducted, including previously reported variant carriers with and without updated clinical data and newly identified carriers through the European Reference Network (ERN) initiative for rare genetic tumor risk syndromes (GENTURIS). Additionally, mutational signature analysis was performed on tumor data from our case series and publicly available datasets from the Hartwig Medical Foundation and TCGA Pan-Cancer Atlas to identify mutational signatures potentially associated with MCM8/MCM9 deficiency. Results: Predicted loss of function and missense variants in MCM8 (1.4 per 100,000 individuals) and MCM9 (2.5 per 100,000 individuals) were found to be rare in gnomAD. However, biallelic MCM9 variants showed significant enrichment in cases from the 100K Genomes Project compared to controls for colonic polyps (odds ratio (OR) 6.51, 95% confidence interval (CI) 1.24 - 34.11; P = 0.03), rectal polyps (OR 8.40, 95% CI 1.28 - 55.35; P = 0.03), and gastric cancer (OR 27.03, 95% CI 2.93 - 248.5; P = 0.004). No significant enrichment was found for biallelic MCM8 variant carriers or in the 200K UK Biobank. In our case series, which included 26 biallelic MCM8 and 28 biallelic MCM9 variant carriers, we documented polyposis, gastric cancer, and early-onset colorectal cancer in 6, 1, and 6 biallelic MCM9 variant carriers, respectively, while these phenotypes were not observed in biallelic MCM8 variant carriers. Additionally, our case series indicates that, beyond hypogonadism--which was present in 23 and 26 of the carriers, respectively--biallelic MCM8 and MCM9 variants are associated with early-onset germ cell tumors (occurring before age 15) in 2 MCM8 and 1 MCM9 variant carriers. Tumors from MCM8/MCM9 variant carriers with available germline sequencing data predominantly displayed clock-like mutational processes (single base substitution signatures 1 and 5), with no evidence of signatures associated with DNA repair deficiencies. Discussion: Our data indicates that biallelic MCM9 variants are associated with polyposis, gastric cancer, and early-onset CRC, while both biallelic MCM8 and MCM9 variants are linked to hypogonadism and the early development of germ cell tumors. These findings underscore the importance of including MCM8/MCM9 in diagnostic gene panels for certain clinical contexts and suggest that biallelic carriers may benefit from cancer surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e202e9882506edb7c1bed18a46ebb4ac96aec6c" target='_blank'>
              Clinical syndromes linked to biallelic germline variants in MCM8 and MCM9
              </a>
            </td>
          <td>
            N. C. Helderman, T. Yang, C. Palles, D. Terlouw, H. Mei, R. Vorderman, D. Cats, M. Diaz Gay, M. Jongmans, A. Ramdien, I. van de Beek, T. F. Eleveld, A. Green, F. Hes, M. M. van den Heuvel-Eibrink, A. Van Der Kelen, S. Kliesch, R. P. Kuiper, I. M. M. Lakeman, L. E. E. L. O. Lashley, L. H. J. Looijenga, M. Oud, J. SteingroÌver, Y. Tenenbaum-Rakover, C. Tops, F. TuÌttelmann, R. D. de Voer, D. Westra, M. J. Wyrwoll, M. Golubicki, M. Antelo, L. Bonjoch, M. Terradas, L. Valle, L. B. Alexandrov, H. Morreau, T. van Wezel, S. Castellví-Bel, Gol
          </td>
          <td>2024-11-04</td>
          <td>None</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a36577362842a55eb6cd804fed42f385c47b74" target='_blank'>
              Tissue microarray analyses of the essential DNA repair factors ATM, DNA-PKcs and Ku80 in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            H. Zech, Clara Marie von Bargen, A. Oetting, N. Möckelmann, Christina Möller-Koop, M. Witt, N. Struve, Cordula Petersen, Christian Betz, K. Rothkamm, A. Münscher, T. Clauditz, T. Rieckmann
          </td>
          <td>2024-10-30</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Accurate monitoring of base excision repair (BER) activity in cancer cells is critical for advancing the comprehension of DNA repair processes, gaining insights into cancer development, and guiding treatment strategies. However, current assay techniques for assessing BER activity in cancer cells face challenges due to the heterogeneous origins and diversity of BER enzymes. In this work, we present a highly reliable triple loop-interlocked DNA codec (GATED) that enables precise assessment of BER activity in cancer cells through signal amplification mediated by multienzyme orthogonal activation. The GATED device features a dumbbell-shaped DNA probe to encode two BER enzymes for BER-related signal conversion as well as two bound circular DNA to decode the apurinic/apyrimidinic sites for apurinic/apyrimidinic endonuclease 1 (APE1)-mediated signal amplification. Importantly, GATED is orthogonally activated by multiple target BER enzymes (i.e., uracil DNA glycosylase, thymine DNA glycosylase, and APE1), resulting in a unified fluorescent signal that significantly improves the detection specificity and sensitivity to BER enzymes. Additionally, we demonstrate that the GATED has exceptional biostability within complex biological systems, where it was successfully employed to monitor BER activity in cancer cells with high specificity and enabled cell-based high-throughput screening for BER inhibitors. The GATED provides a much-needed tool for the real-time monitoring of BER activity and the screening of BER inhibitors in cancer cells, potentially advancing both the investigation and clinical application of BER biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4455a54c11e6664a89c2b008ca24acc5e5bb02" target='_blank'>
              Multienzymatic Orthogonal Activation of DNA Codec Enables Tumor-Specific Imaging of Base Excision Repair Activity.
              </a>
            </td>
          <td>
            Wei-Jia Zeng, Zhaopeng Chen, Yan-Mei Lei, W. Liang, Yaqin Chai, Ruo Yuan, Ying Zhuo
          </td>
          <td>2024-09-26</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Targeting DNA repair pathways is an important strategy in anticancer therapy. However, the unrevealed interactions between different DNA repair systems may interfere with the desired therapeutic effect. Among DNA repair systems, BER and NHEJ protect genome integrity through the entire cell cycle. BER is involved in the repair of DNA base lesions and DNA single-strand breaks (SSBs), while NHEJ is responsible for the repair of DNA double-strand breaks (DSBs). Previously, we showed that BER deficiency leads to downregulation of NHEJ gene expression. Here, we studied BER’s response to NHEJ deficiency induced by knockdown of NHEJ scaffold protein XRCC4 and compared the knockdown effects in normal (TIG-1) and hTERT-modified cells (NBE1). We investigated the expression of the XRCC1, LIG3, and APE1 genes of BER and LIG4; the Ku70/Ku80 genes of NHEJ at the mRNA and protein levels; as well as p53, Sp1 and PARP1. We found that, in both cell lines, XRCC4 knockdown leads to a decrease in the mRNA levels of both BER and NHEJ genes, though the effect on protein level is not uniform. XRCC4 knockdown caused an increase in p53 and Sp1 proteins, but caused G1/S delay only in normal cells. Despite the increased p53 protein, p21 did not significantly increase in NBE1 cells with overexpressed hTERT, and this correlated with the absence of G1/S delay in these cells. The data highlight the regulatory function of the XRCC4 scaffold protein and imply its connection to a transcriptional regulatory network or mRNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1cc5f4d901a34f86a0875c2ef98b93136d4591" target='_blank'>
              Crosstalk between BER and NHEJ in XRCC4-Deficient Cells Depending on hTERT Overexpression
              </a>
            </td>
          <td>
            Svetlana V. Sergeeva, P. S. Loshchenova, D. Oshchepkov, K. Orishchenko
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401dfb6b1e61a2b7931608ee7b632746b618204c" target='_blank'>
              Potential promising of synthetic lethality in cancer research and treatment.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Behnam Najafiyan, Reza Morovatshoar, Mahdieh Khorsandi, Adib Dashtizadeh, Arash Kiani, Farnoosh Farzam, Kimia Sadat Kazemi, Mohsen Nabi Afjadi
          </td>
          <td>2024-09-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="HAGLR Opposite Strand lncRNA (HAGLROS) is a long non-coding RNA (lncRNA) located on the long arm of human chromosome 2 at locus 2q31.1. Emerging evidence highlights HAGLROS as a pivotal player in human cancers, characterized by its significant upregulation across multiple malignancies where it functions as an oncogenic driver. Its aberrant expression is closely linked to the initiation and progression of 13 distinct cancer types, notably correlating with adverse clinical outcomes and reduced overall survival rates in 9 of these cancer types. Mechanistically, HAGLROS is under the regulatory influence of the transcription factor STAT3, exerts competitive binding to 9 miRNAs, activates 5 signaling pathways pivotal for cancer cell proliferation and metastasis, as well as intricately modulates gene expression profiles. Given its multifaceted roles, HAGLROS emerges as a promising candidate for cancer diagnostics and prognostics. Moreover, its potential as a therapeutic target holds considerable promise for novel treatment strategies in oncology. This review synthesizes current research on HAGLROS, covering its expression patterns, biological roles, and clinical significance in cancer. By shedding light on these aspects, this review aims to contribute new perspectives that advance our understanding of cancer biology, enhance diagnostic accuracy, refine prognostic assessments, and pave the way for targeted therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb39b9ed8d4ba49047e0b81dbd39b3093253cd68" target='_blank'>
              LncRNA HAGLROS: A Vital Oncogenic Propellant in Various Human Cancers.
              </a>
            </td>
          <td>
            Jingjie Yang, Haodong He, Lihan Chen, Yuzhang Wei, Yulong Liu, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2024-10-31</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In mammalian cells, repair centers for DNA double-strand breaks (DSBs) have been identified. However, previous researches predominantly rely on methods that induce specific DSBs by cutting particular DNA sequences. The clustering and its spatiotemporal properties of non-specifically DSBs, especially those induced by environmental stresses such as irradiation, remains unclear. In this study, we used Dragonfly microscopy to induce high-precision damage in cells and discovered that DSB clustering during the early stages of DNA damage response (DDR) and repair, but not during the repair plateau phase. Early in DDR, DSB clustered into existing 53BP1 foci. The DSB clustering at different stages has different implications for DNA repair. By controlling the distance between adjacent damage points, we found that the probability of DSB clustering remains constant at distances of 0.8 - 1.4 um, while clustering does not occur beyond 1.4 um. Within the 0.8 um range, the probability of clustering significantly increases due to the phase separation effect of 53BP1. Using a Monte Carlo approach, we developed a dynamic model of 53BP1 foci formation, fission, and fusion. This model accurately predicts experimental outcomes and further demonstrates the temporal and spatial influences on DSB clustering. These results showed that, similarly to specifically induced DSBs, non-specifically induced DSBs can also cluster. The extent of DSB clustering is influenced by both temporal and spatial factors, which provide new insights into the dynamics of DSB clustering and the role of 53BP1 in DNA repair processes. Such findings could enhance our understanding of DNA damage responses and help us improve DNA repair therapies in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a1ca8ef0d6b8c925946d87a4968b5bc72de6f8" target='_blank'>
              Double-Strand Break Clustering: An Economical and Effective Strategy for DNA Repair
              </a>
            </td>
          <td>
            Junyi Chen, Wenzong Ma, Yuqi Ma, Gen Yang
          </td>
          <td>2024-10-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genotoxic stress resulting from DNA damage is resolved through a signaling cascade known as the DNA Damage Response (DDR). The repair of damaged DNA is essential for cell survival, often requiring the DDR to attenuate other cellular processes such as the cell cycle, DNA replication, and transcription of genes not involved in DDR. The complex relationship between DDR and transcription has only recently been investigated. Transcription can facilitate the DDR in response to double-strand breaks (DSBs) and stimulate nucleotide excision repair (NER). However, transcription may need to be reduced to prevent potential interference with the repair machinery. In this review, we discuss various mechanisms that regulate transcription repression in response to different types of DNA damage, categorizing them by their range and duration of effect. Finally, we explore various models of transcription recovery following DNA damage-induced repression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f4075bc573ead401678be4700475fac14136de" target='_blank'>
              From silence to symphony: transcriptional repression and recovery in response to DNA damage.
              </a>
            </td>
          <td>
            Kamal Ajit, M. Gullerová
          </td>
          <td>2024-10-01</td>
          <td>Transcription</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="In recent years, significant advances have been made in understanding the intricate details of the mechanisms underlying alternative lengthening of telomeres (ALT). Studies of a specialized DNA strand break repair mechanism, known as break-induced replication, and the advent of telomere-specific DNA damaging strategies and proteomic methodologies to profile the ribonucleoprotein composition of telomeres enabled the discovery of networks of proteins that coordinate the stepwise homology-directed DNA repair and DNA synthesis processes of ALT. These networks couple mediators of homologous recombination, DNA template-switching, long-range template-directed DNA synthesis, and DNA strand resolution with SUMO-dependent liquid condensate formation to create discrete nuclear bodies where telomere extension occurs. This review will discuss the recent findings of how these networks may cooperate to mediate telomere extension by the ALT mechanism and their impact on telomere function and integrity in ALT cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83e5826a6f5452a3b8a693a0b412a4e75953bec" target='_blank'>
              Mechanisms of Alternative Lengthening of Telomeres.
              </a>
            </td>
          <td>
            Roderick J. O’Sullivan, Roger A. Greenberg
          </td>
          <td>2024-11-05</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Radiation is a mainstay of lung cancer treatment; however, resistance frequently develops. Identifying novel therapeutic targets to increase radiation sensitivity is crucial. S6K1 is a serine/threonine kinase known to regulate protein translation which is associated with radioresistance, but the mechanisms involved are unknown. We proposed to determine whether S6K1 promotes radioresistance by regulating DNA repair in lung cancer. Colony formation, protein expression and proliferation were assessed. S6K1 was modulated pharmacologically by either PF-4708671 or genetically by Crispr-Cas9. Higher radioresistance levels in lung cancer cells were associated with lower phosphoactivation of MRN complex members, a key activator of radiation-induced DNA repair signaling. We also found lower levels of p-ATM, a target of the MRN complex, in more radioresistant cells, which was associated with a lower expression of γ-H2AX cafter radiation. Further, genetic and pharmacological S6K1 targeting sensitized lung cancer cells to low doses of radiation (p ≤ 0.01). Additionally, S6K1−/− deletion increased the phosphoactivation of MRN complex members, indicating that S6K1 itself can shut down DNA damage regulated by MRN signaling. This is the first report showing that S6K1 inhibition radiosensitizes lung cancer cells by decreasing MRN complex-regulated DNA repair signaling. Future studies should evaluate the role of S6K1 as a target to overcome radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623356ed9108e3db105e7395f2b3d04852458dc3" target='_blank'>
              S6K1 Controls DNA Damage Signaling Modulated by the MRN Complex to Induce Radioresistance in Lung Cancer
              </a>
            </td>
          <td>
            Ali Calderon-Aparicio, Jun He, N. Simone
          </td>
          <td>2024-09-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Limited understanding of the biology predisposing certain human papillomavirus-related (HPV+) oropharyngeal squamous cell carcinomas (OPSCCs) to relapse impedes therapeutic personalization. We aimed to identify molecular traits that distinguish recurrence-prone tumors. Methods 50 HPV+ OPSCCs that later recurred (cases) and 50 non-recurrent controls matched for stage, therapy, and smoking history were RNA-sequenced. Groups were compared by gene set enrichment analysis, and select differences were validated by immunohistochemistry. Features discriminating groups were scored in each tumor using gene set variation analysis, and scores were evaluated for recurrence prediction ability. Results Cases downregulated pathways linked to anti-tumor immunity (FDR-adjusted p<.05) and contained fewer tumor-infiltrating lymphocytes (p<.001), including cytotoxic T-cells (p=.005). Cases also upregulated pathways related to cell division and other aspects of tumor progression. Upregulated and downregulated pathways were respectively used to define a tumor progression score (TPS) and immune suppression score (ISS) for each tumor. Correlation between TPS and ISS (r=.603, p<.001) was potentially explained by observed upregulation of DNA repair pathways in cases, which might enhance their progression directly and by limiting cytosolic DNA-induced inflammation. Accordingly, cases contained fewer double-strand breaks based on staining for phospho-RPA32 (p=.006) and γ-H2AX (p=.005) and downregulated pro-inflammatory components of the cytoplasmic DNA sensing pathway. A combined score derived from TPS and ISS optimized recurrence prediction and stratified survival in a manner generalizable to three external cohorts. Conclusions We provide novel evidence that limiting genomic instability makes tumor-intrinsic and immune-mediated contributions to HPV+ OPSCC recurrence risk, opening opportunities to detect and target this treatment-resistant biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec576946511005c671d94c46d7f168994a727f00" target='_blank'>
              Association of oropharyngeal cancer recurrence with tumor-intrinsic and immune-mediated sequelae of reduced genomic instability
              </a>
            </td>
          <td>
            M. Sannigrahi, Lovely Raghav, Dominick J. Rich, Travis P Schrank, Joseph A. Califano, John N. Lukens, Lova Sun, Iain M. Morgan, Roger B. Cohen, Alexander Lin, Xinyi Liu, , Jianxin You, L. Mirabello, Sambit Mishra, D. Shimunov, R. Brody, Alexander T. Pearson, Phyllis A. Gimotty, Ahmed Diab, Jalal B. Jalaly, D. Basu
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="In yeasts and higher eukaryotes, chromatin motions may be tuned to genomic functions, with transcriptional activation and the DNA damage response both leading to profound changes in chromatin dynamics. The RAD51 recombinase is a key mediator of chromatin mobility following DNA damage. As functions of RAD51 beyond DNA repair are being discovered, we asked if RAD51 modulates chromatin dynamics in the absence of DNA damage and found that inhibition or depletion of RAD51 alters chromatin motions in undamaged cells. Inhibition of RAD51 increased nucleosome clustering. Predictions from polymer models are that chromatin clusters reduce chain mobility and, indeed, we measured reduced motion of individual chromatin loci in cells treated with a RAD51 inhibitor. This effect was conserved in mammalian cells, yeasts, and plant cells. In contrast, RAD51 depletion or inhibition increased global chromatin motions at the microscale. The results uncover a role for RAD51 in regulating local and global chromatin dynamics independently from DNA damage and highlight the importance of considering different physical scales when studying chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/926c5111c1c77f4578f6316a9db621bcbc770d0e" target='_blank'>
              RAD51 regulates eukaryotic chromatin motions in the absence of DNA damage.
              </a>
            </td>
          <td>
            Amine Maarouf, Fadil Iqbal, Sarvath Sanaullah, Maëlle Locatelli, Andrew T Atanasiu, Daniel Kolbin, Chloé Hommais, Joëlle Mühlemann, Keith Bonin, Kerry Bloom, Jing Liu, Pierre Vidi
          </td>
          <td>2024-09-18</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Objectives: Cervical cancer (CC) remains a significant global health challenge, characterized by genetic heterogeneity and a complex molecular landscape, both of which contribute to its pathogenesis. This study aimed to investigate germline variants in proto-oncogenes and tumor suppressor genes in cervical cancer patients, with the objective of clarifying their potential role in disease development. Methods: We utilized a custom next-generation sequencing (NGS) panel targeting 48 genes implicated in oncogenesis. Germline DNA samples from cervical cancer patients were analyzed in order to identify nucleotide sequence alterations. Variants were classified according to pathogenicity and clinical relevance, based on established guidelines. Results: A total of 148 nucleotide variants were detected within the cohort. Of these, 35 variants (23.6%) were classified as benign. In contrast, 105 variants (70.9%) were identified as variants of uncertain significance (VUSs). Moreover, seven pathogenic or likely pathogenic mutations were discovered, along with the polymorphic variant rs1042522 in the TP53 gene, which has been associated with an increased risk of cervical cancer. Conclusions: Our findings contribute to expanding our understanding of the molecular genetic landscape of cervical cancer. They emphasize the potential contribution of rare germline mutations to its development and progression. These results highlight the importance of comprehensive genetic screening in order to improve diagnostic and therapeutic approaches for cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9744546f38ec2bc7fa36858f93c880cbede4f99" target='_blank'>
              Germline Variants in Proto-Oncogenes and Tumor Suppressor Genes in Women with Cervical Cancer
              </a>
            </td>
          <td>
            K. Lenkova, R. Khusainova, R. Minyazeva, A. Zaripova, I. Gilyazova, Natalia Mokrysheva, I. Minniakhmetov
          </td>
          <td>2024-10-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High expression of the long non-coding RNA (lncRNA) FAM30A has been previously associated with leukemic stem cell (LSC) activity and poor prognosis in both adult and paediatric acute myeloid leukaemia (AML) patients, yet it has not been functionally studied. This study provides the first cellular characterization of FAM30A focussing on an internal tandemly organised region, referred to as FAM30A repeats. FAM30A levels correlated with canonical AML LSC signatures and FAM30A depletion decreased cell viability as well as increased sensitivity to chemotherapeutics. It also inhibited colony formation, promoted granulocytic differentiation and abrogated leukemic engraftment in murine bone marrow in vivo. Overexpression of FAM30A repeats in this setting enhanced stemness, proliferation, chemoresistance, and engraftment thus highlighting the biological relevance of this region for LSC biology. On the molecular level, FAM30A repeats interact with the pro-LSC regulator Musashi-2 (MSI2), positively influencing expression of its targets including RUNX1 isoforms. We herein uncover that this FAM30A-MSI2-RUNX1 regulatory loop is of potential relevance for LSC maintenance in AML. These findings provide valuable insights into FAM30A’s cellular role and highlight its targeting potential for eliminating LSCs and improving treatment outcomes in AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e944155596f6ba606bda7e6372bd8e29a902812" target='_blank'>
              The long non-coding RNA FAM30A regulates the Musashi2-RUNX1 axis and is required for LSC function in AML cells
              </a>
            </td>
          <td>
            Jaime Calvo Sánchez, Mary T Scott, Athina Varnava, Tala Alakhras, Alice Wedler, Danny Misiak, N. Bley, C. Ihling, A. Sinz, Karen Keeshan, S. Hüttelmaier, David Vetrie, Marcel Köhn
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Extrachromosomal circular DNA (eccDNA) has emerged as a central focus in molecular biology, with various types being found across species through advanced techniques, including high-throughput sequencing. This dynamic molecule exerts significant influence on aging and immune function and plays pivotal roles in autoimmune diseases, type 2 diabetes mellitus, cancer, and genetic disorders.


SUMMARY
This comprehensive review investigates the classification, characteristics, formation processes, and multifaceted functions of eccDNA, providing an in-depth exploration of its mechanisms in diverse diseases.


KEY MESSAGES
The goal of this review is to establish a robust theoretical foundation for a more comprehensive understanding of eccDNA, offering valuable insights for the development of clinical diagnostics and innovative therapeutic strategies in the context of related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f915db54a9ed6fe76aa5229ced4103ebd9df09e" target='_blank'>
              Exploring the Role of Extrachromosomal Circular DNA in Human Diseases.
              </a>
            </td>
          <td>
            Yali Peng, Huihui Tao, Guoying Wang, Mengyao Wu, Tinatin Xu, Chunmei Wen, Xuejia Zheng, Yong Dai
          </td>
          <td>2024-09-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Previous study indicated that CHK1 was important for repairing DNA damage and cell cycle regulation. Aims: To investigate the association of Checkpoint kinase 1 (CHK1) expression with clinicopathological features, prognosis, and immune infiltration in cancer. Methods: Several databases were searched for relevant publications to conduct a meta-analysis to reveal the association between CHK1 and clinicopathological features of cancer. TIMER and GEPIA datasets were utilized to explore the differential expression of CHK1 of tumors. GEPIA and Kaplan-Meier Plotter databases were adopted to detect the prognostic significance of CHK1 in tumor. TIMER and cBioPortal databases were used for the analysis regarding immune infiltration and mutation respectively. Results: We found that CHK1 expression was significantly associated with low differentiation (OR=3.94, 95% CI: 2.73-5.67, P<0.05), advanced stage (OR=3.20, 95% CI: 2.30-4.44, P<0.05), vascular infiltration (OR=3.24, 95% CI: 2.18-4.82, P<0.05) and lymph node metastasis (OR=3.55, 95% CI: 2.62-4.82, P<0.05) of various cancers. CHK1 was highly expressed in multiple cancers and was related to clinical stage, survival, immune infiltration in pan-cancers. Conclusions: Our study found that CHK1 was significantly related to prognosis and immunological status in various cancers, suggesting that CHK1 may serve as a useful biomarker for prognosis and immune infiltration in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48bd08dda4568b1d8c35e755803a8c000bea8fc" target='_blank'>
              Exploration of the carcinogenetic and immune role of CHK1 in human cancer
              </a>
            </td>
          <td>
            Jian Zhou, Ziyi Wu, Dilihumaer Aili, Lu Wang, Tanghai Liu
          </td>
          <td>2024-09-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Objectives: Sarcomas are a heterogeneous group of cancers, many with high rates of recurrence and metastasis, leading to significant morbidity and mortality. Due to a lack of early diagnostic biomarkers, by the time recurrent disease can be clinically detected, it is often extensive and difficult to treat. Here, we sought to investigate methods of detecting ctDNA in sarcoma patient plasma to potentially monitor disease recurrence, progression, and response to treatment. Methods: Whole-exome sequencing of matched tumor and blood samples revealed patient-specific mutations, which were used to develop personalized assays to detect ctDNA in patient plasma. Since ctDNA is present in extremely low quantities, detection requires highly sensitive methodologies. Droplet digital PCR is highly sensitive; however, it is limited in that it can only be used to target one tumor variant at a time. Therefore, a protocol combining multiplex PCR and targeted amplicon sequencing was developed. Results: ddPCR was successfully able to detect tumor-specific mutations in plasma, confirming the presence of ctDNA in sarcoma patients. Multiplex PCR followed by amplicon sequencing was able to detect multiple tumor variants simultaneously, although it was not as sensitive as ddPCR. Additionally, ctDNA was detected in patient plasma collected at two different time points. Conclusions: This work demonstrates that although there is a lack of recurrent biomarkers, personalized assays detecting ctDNA have the potential to be used to monitor disease progression in sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9568c80338e8c3723c68b184905d017f0426c4" target='_blank'>
              Investigating the Use of Circulating Tumor DNA for Sarcoma Management
              </a>
            </td>
          <td>
            Paige Darville-O'Quinn, N. Gokgoz, Kim M. Tsoi, I L Andrulis, Jay S. Wunder
          </td>
          <td>2024-10-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chromatin organization is important for gene transcription in pig genome. However, its three-dimensional (3D) structure and dynamics are much less investigated than those in human. Here we applied the long-reads chromatin interaction analysis by paired-end tag sequencing (ChIA-PET) method to map the whole-genome chromatin interactions mediated by CCCTC-binding factor (CTCF) and RNA polymerase Ⅱ (RNAPⅡ or POLⅡ) in porcine macrophage cells before and after polyinosinic-polycytidylic acid [Poly(I:C)] induction. Our results revealed that Poly(I:C) induction impacts the 3D genome organization in the 3D4/21 cells at the fine-scale chromatin loop level rather than at the large-scale domain level. Furthermore, our findings underscored the pivotal role of CTCF anchored chromatin interactions in reshaping chromatin architecture during immune responses. Knock-out of the CTCF locus further confirmed that the CTCF anchored enhancers are associated with the activation of immune genes via long-range interactions. Notably, ChIA-PET data also supported the spatial relationship between single nucleotide polymorphisms (SNPs) and the related gene transcription in 3D genome aspect. Our findings in this study provide new clues and potential targets to explore key elements related to diseases in swine and are also likely to shed light on elucidating chromatin organization and dynamics underlying the process of mammalian infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97442a763cf352589738944e6bb5a4d860334082" target='_blank'>
              Virus Infection Induces Immune Gene Activation with CTCF Anchored Enhancers and Chromatin Interactions in Pig Genome.
              </a>
            </td>
          <td>
            Jianhua Cao, Ruimin Ren, Xiaolong Li, Xiaoqian Zhang, Yan Sun, Xiaohuan Tian, Ru Liu, Xiangdong Liu, Yijun Ruan, Guoliang Li, Shuhong Zhao
          </td>
          <td>2024-09-23</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48d863d20b27ee2206ff6d80ccbbfa69c29c0c2" target='_blank'>
              3D genome contributes to MHC-II neoantigen prediction
              </a>
            </td>
          <td>
            Mofan Feng, Liangjie Liu, K. Su, Xianbin Su, Luming Meng, Zehua Guo, Dan Cao, Jiayi Wang, Guang He, Yi Shi
          </td>
          <td>2024-09-26</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Purpose Upper tract urothelial carcinoma (UTUC) and urothelial carcinoma of the bladder (UCB) share histomorphological and therapeutic features but distinct epidemiologic and clinicopathologic characteristics. We examined alterations of chromatin regulatory genes in molecular subtypes, clonal relatedness, and T‐cell receptor (TCR) diversity in UTUC and UCB. Materials and Methods Targeted next‐generation sequencing or whole‐exome DNA sequencing and TCR sequencing were conducted with 34 UTUC and 49 UCB specimens from 63 patients. Tumors were subtyped based on the expression of CK5 and GATA3. Results of tissue microarray of 78 muscle‐invasive bladder cancer (MIBC) samples were used as prognostic factors of different subtypes of MIBC. Results Chromatin regulatory genes were frequently mutated in both UTUC and UCB. Rapid relapse and progression of non‐MIBC are correlated with alterations of KMT2C and EP300. Frequency of alterations in chromatin regulatory genes is higher in UTUC patients with SBS22 and SBS2 signatures and lower in UCB patients with SBS2 and SBS6 signatures. GATA3 and CK5 double‐positive patients with higher frequencies of SMARCA4, ARID1A, and EP300 mutations have better prognoses than patients with basal subtypes. Although UTUC and UCB in the same patient can be either clonally related or developed independently, mutated genes in chromatin pathway were enriched in the related clones. Compared to UTUC, UCB had more deleterious mutations in DNA damage repair (DDR) genes, higher levels of tumor mutation burden (TMB) and copy number variations (CNVs), as well as higher TCR clonality and lower TCR diversity. Conclusions Since genetic alterations of the chromatin pathway genes are important in both UTUC and UCB, they could serve as potential biomarkers for predicting disease progression and therapeutic targets. Differences in mutation frequencies of DDR pathway, TMB, CNV, and TCR might be the contributing factors for the distinct responses to immune checkpoint inhibitor (ICI) between UTUC and UCB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4b963d4cd7ebf8697e4f003e48f64b41e6ab93" target='_blank'>
              Genetic Alterations in Chromatin Regulatory Genes in Upper Tract Urothelial Carcinoma and Urothelial Bladder Cancer
              </a>
            </td>
          <td>
            Shuo Wang, Xuzhi Yan, W. Lan, Yapeng Wang, Ze Wang, D. Tong, Yao Zhang, Qiang Ran, Haoyang Li, Junhao Jin, Haiyang Xiao, Jing Xu, Qian Yan, Dianzheng Zhang, Qiang Ma, Hualiang Xiao, Jun Qin, Luofu Wang, Jun Jiang, Qiuli Liu
          </td>
          <td>2024-11-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background R3HDM1, an RNA binding protein with one R3H domain, remains uncharacterized in terms of its association with tumor progression, malignant cell regulation, and the tumor immune microenvironment. This paper aims to fill this gap by analyzing the potential of R3HDM1 in diagnosis, prognosis, chemotherapy, and immune function across various cancers. Methods Data was collected from the Firehost database (http://gdac.broadinstitute.org) to obtain the TCGA pan-cancer queue containing tumor and normal samples. Additional data on miRNA, TCPA, mutations, and clinical information were gathered from the UCSC Xena database (https://xenabrowser.net/datapages/). The mutation frequency and locus of R3HDM1 in the TCGA database were examined using the cBioPortal. External validation through GEO data was conducted to assess the differential expression of R3HDM1 in different cancers. Protein expression levels were evaluated using the Clinical Proteomics Tumor Analysis Alliance (CPTAC). The differential expression of R3HDM1 was verified in lung adenocarcinoma cell lines and normal lung glandular epithelial cells via RT-qPCR. Cell migration and proliferation experiments were conducted by knocking down the expression of R3HDM1 in two lung adenocarcinoma cell lines using small interfering RNA. The biological role of R3HDM1 in pan-cancer was explored using the GSEA method. Multiple immune infiltration algorithms from the TIMER2.0 database was employed to investigate the correlation between R3HDM1 expression and the tumor immune microenvironment. Validation of transcriptome immune infiltration was based on 140 single-cell datasets from the TISCH database. The study also characterized a pan-cancer survival profile and analyzed the differential expression of R3HDM1 in different molecular subtypes. The relationship between R3HDM1 and drug resistance was investigated using four chemotherapy data sources: CellMiner, GDSC, CTRP and PRISM. The impact of chemicals on the expression of R3HDM1 was explored through the CTD database. Result The study revealed differential expression of R3HDM1 in various tumors, indicating its potential as an early diagnostic marker. Changes in somatic copy number (SCNA) and DNA methylation were identified as factors contributing to abnormal expression levels. Additionally, the study found that R3HDM1 expression is associated with clinical features, metabolic pathways, and important pathways related to metastasis and the immune system. High expression of R3HDM1 was linked to poor prognosis across different tumors and altered drug sensitivity. Furthermore, the expression of R3HDM1 showed significant correlations with immune modulatory molecules and biomarkers of lymphocyte subpopulation infiltration. Finally, the study highlighted four chemicals that could influence the expression of R3HDM1. Conclusion Overall, this study proposes that R3HDM1 expression is a promising biomarker for predicting the prognosis of cancer, especially lung adenocarcinoma, and the efficacy of immunotherapy, demonstrating the rationale for further exploration in the development of anti-tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379d6a67141459985f554b1a56ebc613e9c22b34" target='_blank'>
              Comprehensive pan-cancer analysis and experiments revealed R3HDM1 as a novel predictive biomarker for prognosis and immune therapy response
              </a>
            </td>
          <td>
            Jiawei Liu, Zhitong Bing, Junling Wang
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract DNA damage response (DDR) pathway is the coordinated cellular network dealing with the identification, signaling, and repair of DNA damage. It tightly regulates cell cycle progression and promotes DNA repair to minimize DNA damage to daughter cells. Key proteins involved in DDR are frequently mutated/inactivated in human cancers and promote genomic instability, a recognized hallmark of cancer. Besides being an intrinsic property of tumors, DDR also represents a unique therapeutic opportunity. Indeed, inhibition of DDR is expected to delay repair, causing persistent unrepaired breaks, to interfere with cell cycle progression, and to sensitize cancer cells to several DNA‐damaging agents, such as radiotherapy and chemotherapy. In addition, DDR defects in cancer cells have been shown to render these cells more dependent on the remaining pathways, which could be targeted very specifically (synthetic lethal approach). Research over the past two decades has led to the synthesis and testing of hundreds of small inhibitors against key DDR proteins, some of which have shown antitumor activity in human cancers. In parallel, the search for synthetic lethality interaction is broadening the use of DDR inhibitors. In this review, we discuss the state‐of‐art of ataxia‐telangiectasia mutated, ataxia‐telangiectasia‐and‐Rad3‐related protein, checkpoint kinase 1, Wee1 and Polθ inhibitors, highlighting the results obtained in the ongoing clinical trials both in monotherapy and in combination with chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3effb948dd60b6e28e5a1e98f095e523b94a901" target='_blank'>
              Targeting the DNA damage response in cancer
              </a>
            </td>
          <td>
            Guffanti Federica, Chiappa Michela, Damia Giovanna
          </td>
          <td>2024-10-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A critical area of recent cancer research is the emergence of transition states between normal and cancer that exhibit increased cell plasticity which underlies tumor cell heterogeneity. Pancreatic ductal adenocarcinoma (PDAC) can arise from the combination of a transition state termed acinar-to-ductal metaplasia (ADM) and a gain-of-function mutation in the proto-oncogene KRAS. During ADM, digestive enzyme-producing acinar cells acquire a transient ductal epithelium-like phenotype while maintaining their geographical acinar organization. One route of ADM initiation is the overexpression of the Krüppel-like factor 4 gene (KLF4) in the absence of oncogenic driver mutations. Here, we asked to what extent cells acquire and retain an epigenetic memory of the ADM transition state in the absence of oncogene mutation. We identified differential DNA methylation at Kras-downstream PI3K and Rho/Rac/Cdc42 GTPase pathway genes during ADM, as well as a corresponding gene expression increase in these pathways. Importantly, differential methylation persisted after gene expression returned to normal. Caerulein exposure, which induces widespread digestive system changes in addition to ADM, showed similar changes in DNA methylation in ADM cells. Regions of differential methylation were enriched for motifs of KLF and AP-1 family transcription factors, as were those of human pancreatic intraepithelial neoplasia (PanIN) samples, demonstrating the relevance of this epigenetic transition state memory in human carcinogenesis. Finally, single-cell spatial transcriptomics revealed that these ADM transition cells were enriched for PI3K pathway and AP1 family members, linking epigenetic memory to cancer cell plasticity even in the absence of oncogene mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da20ab316bd30b6e134263f4015da1c852acb904" target='_blank'>
              DNA methylation memory of pancreatic acinar-ductal metaplasia transition state altering Kras-downstream PI3K and Rho GTPase signaling in the absence of Kras mutation
              </a>
            </td>
          <td>
            E. K. Lo, Adrian Idrizi, Rakel Tryggvadottir, Weiqiang Zhou, Wenpin Hou, Hongkai Ji, Patrick Cahan, Andrew P. Feinberg
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb94655719ab703b20f1ab3bcef7376bf12c183f" target='_blank'>
              Genome-wide methylation profiling reveals extracellular vesicle DNA as an ex vivo surrogate of cancer cell-derived DNA
              </a>
            </td>
          <td>
            Kyung-A Kim, Sunmin Kim, I. Wortzel, Suho Lee, Yoon Dae Han, Tae-Min Kim, Han Sang Kim
          </td>
          <td>2024-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Alternative polyadenylation (APA) is a major mechanism of post-transcriptional regulation that affects mRNA stability, localization and translation efficiency. Previous pan-cancer studies have revealed that APA is frequently disrupted in cancer and is associated with patient outcomes. Yet, little is known about cancer type-specific APA alterations. Here, we integrated RNA-sequencing data from a Korean cohort (GEO: GSE40419) and The Cancer Genome Atlas (TCGA) to comprehensively analyze APA alterations in lung adenocarcinomas (LUADs). Comparing expression levels of core genes involved in polyadenylation, we find that overall, the set of 28 of 31 genes are upregulated, with CSTF2 particularly upregulated. We observed broad and recurrent APA changes in LUAD growth-promoting genes. In addition, we find enrichment of APA events in genes associated with known LUAD pathways and an increased heterogeneity in polyadenylation (polyA) site usage of proliferation-associated genes. Upon further investigation, we report smoking-specific APA changes are also highly relevant to LUAD development. Overall, our in-depth analysis reveals APA as an important driver for the molecular and clinical features of lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8028b81c80ed86505ee18a83ac6ee8a99e30532" target='_blank'>
              Alternative polyadenylation shapes the molecular and clinical features of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yipeng Gao, Vikram R Shaw, Christopher I Amos
          </td>
          <td>2024-11-02</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a lethal disease with limited therapies. In our screening using patient-derived organoids, we identified cyclin-dependent kinase 12 (CDK12) as a promising target in a subset of PDAC. Both pharmacological and genetic inhibition of CDK12 exerted potent antitumor activity in vitro and in vivo, inducing double-strand DNA breaks and apoptosis. CDK12 is a member of cyclin-dependent kinases that is involved in gene expression regulations at multiple levels. Recently, CDK12 has gained attention as a therapeutic target in several types of cancers including triple-negative breast cancer and Ewing sarcoma. However, little is known about the impact and possibility of CDK12 inhibition in pancreatic cancer treatment. As the lack of specific inhibitors that selectively target CDK12, studying CDK12 has long been challenging. To overcome this, we used CRISPR-Cas9 mediated genome editing to generate FKBP12F36V (dTAG) knock-in PDAC cell lines, enabling rapid elimination of endogenous CDK12 using a degrader against dTAG (dTAG-13). Quantitative-image-based cytometry and time lapse imaging showed that CDK12 ablation induces mitotic arrest, DNA damage and apoptosis (mitotic catastrophe). Specifically, loss of CDK12 kinase activity in the S phase was sufficient to slow DNA replication speed, leading to spindle assembly checkpoint activation and mitotic catastrophe. Importantly, cells that escaped mitotic death accumulate micronuclei, triggering immune response signaling that is mediated by the cGAS-STING-type I interferon pathway. Our results suggest that CDK12 is a promising target in a subset of PDAC patients, where CDK12 inhibition exerts antitumor activity both in cell autonomous and non-cell autonomous ways through inducing mitotic catastrophe as well as triggering immune response.
 Citation Format: Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro. CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A084.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632d48b78e0c9131037bdfa10d93bd71182ca006" target='_blank'>
              Abstract A084: CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer
              </a>
            </td>
          <td>
            Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In many organisms, aging is a clear risk factor for increased rates of chromosome mis-segregation, the main source of aneuploidy. Here, we report that old yeast mother cells lose chromosomes by partitioning them asymmetrically to their daughter cells together with the pre-existing (old) Sindle Pole Body (centrosome equivalent in yeast). Strikingly, remodelling of the NPC and the displacement of its nuclear basket triggered these asymmetric segregation events. Concomitantly, basket displacement also caused unspliced pre-mRNAs to leak to the cytoplasm. We show that removing the introns of three genes involved in chromosome segregation was sufficient to fully suppress chromosome loss in old cells. Furthermore, promoting pre-mRNA leakage in young cells also caused asymmetric chromosome partition and loss through the same three introns. Therefore, we propose that basket displacement from NPCs and its ensuing effects on pre-mRNA quality control are a key trigger of aging phenotypes such as aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7e4a2437a944eea036632a9da1081015ec98c9" target='_blank'>
              Dissociation of the nuclear basket triggers chromosome loss in aging yeast
              </a>
            </td>
          <td>
            M. Mirkovic, Jordan McCarthy, A. C. Meinema, J. Parenteau, Sung Sik Lee, Sherif Abou Elela, Yves Barral
          </td>
          <td>2024-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="As we age, many tissues become colonised by microscopic clones carrying somatic driver mutations. Some of these clones represent a first step towards cancer whereas others may contribute to ageing and other diseases. However, our understanding of the clonal landscapes of human tissues, and their impact on cancer risk, ageing and disease, remains limited due to the challenge of detecting somatic mutations present in small numbers of cells. Here, we introduce a new version of nanorate sequencing (NanoSeq), a duplex sequencing method with error rates <5 errors per billion base pairs, which is compatible with whole-exome and targeted gene sequencing. Deep sequencing of polyclonal samples with single-molecule sensitivity enables the simultaneous detection of mutations in large numbers of clones, yielding accurate somatic mutation rates, mutational signatures and driver mutation frequencies in any tissue. Applying targeted NanoSeq to 1,042 non-invasive samples of oral epithelium and 371 samples of blood from a twin cohort, we found an unprecedentedly rich landscape of selection, with 49 genes under positive selection driving clonal expansions in the oral epithelium, over 62,000 driver mutations, and evidence of negative selection in some genes. The high number of positively selected mutations in multiple genes provides high-resolution maps of selection across coding and non-coding sites, a form of in vivo saturation mutagenesis. Multivariate regression models enable mutational epidemiology studies on how carcinogenic exposures and cancer risk factors, such as age, tobacco or alcohol, alter the acquisition and selection of somatic mutations. Accurate single-molecule sequencing has the potential to unveil the polyclonal landscape of any tissue, providing a powerful tool to study early carcinogenesis, cancer prevention and the role of somatic mutations in ageing and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb1ed2352282f2639f8bc59358f511e72d35ab26" target='_blank'>
              Somatic mutation and selection at epidemiological scale
              </a>
            </td>
          <td>
            A. R. Lawson, F. Abascal, P. A. Nicola, S. Lensing, Amy L Roberts, Georgios Kalantzis, A. Baez-Ortega, Natalia Brzozowska, Julia S. El-Sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, Ayrun Nessa, S. Wadge, Anna L. Paterson, Doris M. Rassl, R. E. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, M. Neville, M. Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, Nick Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D. Young, A. KrishnaaT., Mahbubani, K. Saeb-Parsy, Hilary C. Martin, M. R. Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2024-11-01</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1436facb9b04a543d668f4862e82c8d1f590873" target='_blank'>
              FBXO2 as a switch guides a special fate of tumor clones evolving into a highly malignant transcriptional subtype in oral squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jingyi Cheng, O. Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) have emerged as crucial regulators of cellular processes, overturning their previous classification as “junk” DNA. Their involvement in various cellular functions makes them potential therapeutic targets in a range of diseases, particularly in tumorigenesis and cancer. Among these lncRNAs, MALAT1 (Metastasis-Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear-Enriched Abundant Transcript 1) are prominent oncogenic lncRNAs with complex regulatory roles across multiple cancer types. Although the roles of these lncRNAs in cancer have been extensively studied, the majority of these investigations have been conducted in 2D culture systems. While 2D cell cultures are well-established and widely accepted models in cell biology, they lack the physiological complexity of 3D tumor architecture. In contrast, 3D cultures, where cells grow as three-dimensional clusters, better mimic in vivo conditions, making them essential for understanding tumor microenvironment (TME) dynamics. Despite the importance of 3D models, there is a lack of literature exploring lncRNA expression in 2D monolayers versus 3D cultures across different cancer types. This study examines the expression and function of the lncRNAs NEAT1 and MALAT1 in 3D tumoroids compared to 2D monolayer cultures, aiming to bridge the gap between in vitro models and the complex in vivo tumor microenvironment. We addressed this gap by quantifying the expression differences of NEAT1 and MALAT1 using qRT-PCR in breast cancer, liver cancer, cervical cancer, and glioblastoma (GBM) cells grown in both 2D and 3D cultures. Significant variations in NEAT1 and MALAT1 expression were observed between 2D and 3D cultures across these cancer types, signifying the need to study lncRNAs in 3D micro-environments. Furthermore, we established correlations between NEAT1 expression and cancer traits such as stemness, invasion, glucose transporter expression, and epithelial-mesenchymal transition (EMT) in GBM 3D tumoroids compared to 2D monolayers. Using siRNA to downregulate NEAT1 in GBM tumoroids, we demonstrated that reducing NEAT1 expression to the levels comparable to 2D cultures led to a decrease in the expression of mRNA markers associated with stemness, invasion, glucose transporters, and EMT. Additionally, siRNA-mediated downregulation of NEAT1 in 3D tumoroids directly impacted cancer properties, as validated by phenotypic assays, where reduced proliferation, migration, and invasion were observed in GBM (comparing 3D to 2D models). Therefore, our work provides new insights into the expression of two key oncogenic lncRNAs in 3D microenvironments of various cancers and lays the foundation for integrating lncRNAs as important molecular players within 3D culture systems, offering a better understanding of the in vivo complexity of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4574fbc2dd767027ac41776d25b2fefa2c78aa8" target='_blank'>
              Differential expression of oncogenic lncRNAs NEAT1 and MALAT1 in 2D monolayer vs. 3D tumoroid culture and its implications in cancer progression
              </a>
            </td>
          <td>
            Arpita Ghosh, R. Soundharya, M. Jolly, A. Majumder
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9108f268b2483ac30e3a4fd357c7d047dfce981" target='_blank'>
              Nuclear receptor E75/NR1D2 promotes tumor malignant transformation by integrating Hippo and Notch pathways
              </a>
            </td>
          <td>
            Xianping Wang, Yifan Guo, Peng Lin, Min Yu, Sha Song, Wenyan Xu, D. Kong, Yin Wang, Yanxiao Zhang, Fei Lu, Qi Xie, Xianjue Ma
          </td>
          <td>2024-11-08</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Early cancer detection has the potential to significantly improve treatment outcomes and survival rates. This study investigates the roles of 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC) as biomarkers for early-stage colorectal cancer (CRC) detection in cell-free DNA (cfDNA). Using whole genome sequencing, we analyzed cfDNA from 37 treatment-naive CRC patients and 32 healthy controls. Our findings indicate that combining measurements of 5mC and 5hmC significantly enhances diagnostic accuracy (AUC = 0.95) compared to traditional approaches that conflate these markers (modified C, AUC = 0.66). Notably, 71.7% of differentially methylated regions (DMRs) exhibiting an increase in 5hmC in stage I cfDNA also showed a corresponding decrease in 5mC in stage IV, suggesting that 5hmC can effectively track regions undergoing demethylation during tumor development. These results support the hypothesis that distinguishing between 5mC and 5hmC can improve the sensitivity of liquid biopsy tests for early cancer detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7787a16ebcea187a8cc9692dc78dcefaed5765df" target='_blank'>
              5-methylcytosine and 5-hydroxymethylcytosine are synergistic biomarkers for early detection of colorectal cancer
              </a>
            </td>
          <td>
            Fabio Puddu, Annelie Johansson, Aurélie Modat, Jamie Scotcher, Riccha Sethi, Shirong Yu, Nicholas Harding, Mark Hill, Ermira Lleshi, Casper Lumby, Jean Teyssandier, Michael Wilson, Robert Crawford, Tom Charlesworth, Robert J Osborne, Shankar Balasubramanian, Páidí Creed
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dermatomyositis is an autoimmune condition characterized by a high interferon signature of unknown etiology. Because genes constitute only <2% of our genomes, there is a need to explore the role of the non-coding genome in disease pathogenesis. Our genomes include roughly 1.2 million Alu elements occupying about 10% of the genome and can form double-stranded (ds)RNA capable of triggering MDA5 leading to interferon production. We aligned muscle biopsy RNA sequencing data to the Telomere-to-Telomere reference genome and quantified short interspersed elements including Alus. Dermatomyositis muscle (n=39) showed a global elevation in Alu expression as well as an increased expression of unique Alu elements (n=557, p<0.05) compared to healthy controls (n=34), in a pattern not seen in other myositis types (n=81). The majority (75.3%) of these Alus originated from genomic regions outside genes, with a hot spot of expression on chromosome 19. A subset of the uniquely overexpressed Alus (n=167) correlated strongly with interferon stimulated genes and markers of myositis activity. Since Alu transcripts have a propensity to form dsRNA and are the major targets of both ADAR and MDA5, we quantified the A-to-I RNA editomes inside Alus and found a uniquely expanded Alu editome in dermatomyositis compared to other myositis types, reflecting an increase in dsRNA. Edited Alus clustered on chromosome 19, which is known to have the highest concentration of dsRNA. We hypothesize that overexpressed Alus in dermatomyositis form endogenous dsRNA that exceed the capacity of RNA editing enzymes and trigger dsRNA sensors leading to interferon production.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba433b84cde6ca292fa0f83a61e9a34e6185a71" target='_blank'>
              Alu overexpression leads to increased double-stranded RNA in dermatomyositis
              </a>
            </td>
          <td>
            Rayan Najjar, Andrew Mammen Mph, Tomas Mustelin PhD
          </td>
          <td>2024-11-04</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The interplay between cancer and the immune system has captivated researchers for a long time. Recent developments in cancer immunotherapy have substantiated this interest with a significant benefit to cancer patients. Tumor and immune cells are regulated via a wide range of molecular mechanisms involving intricate transcriptional and epigenetic networks. Epigenetic processes influence chromatin structure and accessibility, thus governing gene expression, replication, and DNA damage repair. However, aberrations within epigenetic signatures are frequently observed in cancer. One of the key epigenetic marks is the trimethylation of histone 3 at lysine 9 (H3K9me3), confined mainly within constitutive heterochromatin to suppress DNA accessibility. It is deposited at repetitive elements, centromeric and telomeric loci, as well as at the promoters of various genes. Dysregulated H3K9me3 deposition disrupts multiple pathways, including immune signaling. Consequently, altered H3K9me3 dynamics may modify the efficacy of immunotherapy. Indeed, growing evidence highlights the pivotal roles of various proteins mediating H3K9me3 deposition (SETDB1/2, SUV39H1/2), erasure (KDM3, KDM4 families, KDM7B, LSD1) and interpretation (HP1 proteins, KAP1, CHD4, CDYL, UHRF1) in modulating immunotherapy effectiveness. Here, we review the existing literature to synthesize the available information on the influence of these H3K9me3 writers, erasers, and readers on the response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/145e5ffc384f02a28629c78f8d74cf791460fc6e" target='_blank'>
              The Roles of H3K9me3 Writers, Readers, and Erasers in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Urszula Oleksiewicz, Monika Kuciak, Anna Jaworska, Dominika Adamczak, Anna Bisok, Julia Mierzejewska, Justyna Sadowska, Patrycja Czerwińska, Andrzej Mackiewicz
          </td>
          <td>2024-10-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In response to DNA double-strand damage, ongoing transcription is inhibited to facilitate accurate DNA repair while transcriptional recovery occurs after DNA repair is complete. However, the mechanisms at play and the identity of the transcripts being regulated in this manner are unclear. In contrast to the situation following UV damage, we found that transcriptional recovery after ionizing radiation (IR) occurs in a manner independent of the HIRA histone chaperone. Sequencing of the nascent transcripts identified a programmed transcriptional response, where certain transcripts and pathways are rapidly downregulated after IR, while other transcripts and pathways are upregulated. Specifically, most of the loss of nascent transcripts occurring after IR is due to inhibition of transcriptional initiation of the highly transcribed histone genes and the rDNA. To identify factors responsible for transcriptional inhibition after IR in an unbiased manner, we performed a whole genome gRNA library CRISPR/Cas9 screen. Many of the top hits on our screen were factors required for protein neddylation. However, at short times after inhibition of neddylation, transcriptional inhibition still occurred after IR, even though neddylation was effectively inhibited. Persistent inhibition of neddylation blocked transcriptional inhibition after IR, and it also leads to cell cycle arrest. Indeed, we uncovered that many inhibitors and conditions that lead to cell cycle arrest in G1 or G2 phase also prevent transcriptional inhibition after IR. As such, it appears that transcriptional inhibition after IR occurs preferentially at highly expressed genes in cycling cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c249bb9c322b48ef03149a30028a53f5cc017969" target='_blank'>
              Transcriptional inhibition after irradiation occurs preferentially at highly expressed genes in a manner dependent on cell cycle progression
              </a>
            </td>
          <td>
            Zulong Chen, Xin Wang, Xinlei Gao, Nina Arslanovic, Kaifu Chen, Jessica K Tyler
          </td>
          <td>2024-10-11</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Eukaryotic chromosomes segregate faithfully prior to nuclear division to ensure genome stability. If segregation becomes defective, the chromosome copy number of the cell may alter leading to aneuploidy and/or polyploidy, both common hallmarks of cancers. In eukaryotes, aurora kinases regulate chromosome segregation during mitosis and meiosis, but their functions in the divergent, single-celled eukaryotic pathogen Trypanosoma brucei are less understood. Here, we focused on one of three aurora kinases in these parasites, TbAUK3, a homologue of the human aurora kinase AURKC, whose functions are primarily restricted to meiosis. We show that RNAi targeted depletion of TbAUK3 correlates with nuclear segregation defects, reduced proliferation, and decreased DNA synthesis, suggestive of a role for TbAUK3 during mitotic, not meiotic, chromosome segregation. Moreover, we uncover a putative role for TbAUK3 during the parasite’s response to DNA damage since we show that depletion of TbAUK3 enhances DNA instability and sensitivity to genotoxic agents. HIGHLIGHTS The C-terminus of TbAUK3 is disordered TbAUK3 depletion coincides with nuclear segregation defects Depletion of TbAUK3 enhances DNA instability">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125d96f157b6efd3c7ef3b8d6665537e416170cc" target='_blank'>
              AUK3 is required for faithful nuclear segregation in the bloodstream form of Trypanosoma brucei
              </a>
            </td>
          <td>
            J. A. Black, B. Poulton, B.S. Gonzaga, A. Iskantar, L. Tosi, R. McCulloch
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Despite recent advances in the treatment of pancreatic adenocarcinoma (PDAC), the median survival remains <12 months. Patients typically present with late-stage disease and are often unable to tolerate drug combination regimens. Large-scale genomic approaches categorise PDAC into two distinct molecular subtypes, based on transcriptomic signatures, and these are termed the classical and squamous (or basal) subtypes. Although it is well-described that PDAC harbours gene mutations (i.e. KRAS G12D, p53 R172H) that initiate the disease, little is known about the influence of lineage-defining Transcription Factors (TF) on PDAC subtypes and their contribution to tumor progression. We took an unbiased approach to discover driving TFs in PDAC, we performed ChIP-seq of the active enhancer mark H3K27Ac from surplus tissue following resection of primary PDAC tumors (n=6) and normal adjacent tissue (n=4). Analysis of the H3K27Ac sites showed that the most enriched motifs within all experimentally mapped enhancer elements were HNF motifs (HNF4G & HNF4A) and Forkhead (i.e., FOXA1), confirming that these two classes of TFs constitute the lineage-defining factors in these clinical samples. Investigation of publicly available clinical datasets revealed HNF4G to be amplified in 9% of PDAC patients. Using Rapid Immunoprecipitation and Mass Spectrometry of Endogenous protein complexes (RIME), our method for unbiased discovery of endogenous protein complexes, we investigated the fundamental differences in the FOXA1 interactome between models of classical and squamous subtypes. HNF4A and HNF4G were only seen as FOXA1-interactors in models of the classical subtype. HNF4G Knockout (KO) clones injected orthotopically into the pancreas of mice, showed a significant increase (p-value<0.0006) in median survival and slower tumor growth compared to Control (Cas-9) (p-value <0.0001). HNF4G-KO dependent genes were specifically expressed in primary tumors (with decreased EMT signalling) versus metastatic samples (PanCRux dataset), implying that the HNF4G target genes were mostly limited to primary tumor contexts. We found Protein Arginine Methylate Transferase (PRMT1) to be a significant and reproducible interactor of FOXA1 and HNF4G. We observed a significant global reduction in PRMT1 chromatin binding in both the HNF4G-KO and FOXA1 depleted cells compared to the control. This data suggests a dependency on both HNF4G and FOXA1 for recruitment of PRMT1 to chromatin. We find HNF4G to be the key driver of classical subtype primary disease, in part by recruiting the methyltransferase PRMT1 to regulatory regions revealing a therapeutically exploitable opportunity. A molecular switch occurs in advanced disease, whereby HNF4G expression decreases, unmasking FOXA1’s transcriptional activity. De-repressed FOXA1 drives late-stage disease, by orchestrating metastasis-specific enhancer-promoter loops to direct metastatic gene regulation. We suggest the existence of stage-dependent TF activity, triggered by molecular compartmentalisation that mediates multistage progression of PDAC.
 Citation Format: Shalini Vasudev Rao, Lisa V Young, Danya Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll. Transcription factor switching drives progression of the classical subtype of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e663bb2d35a48fcc777219947b4d5e9554b7ef19" target='_blank'>
              Abstract C071: Transcription factor switching drives progression of the classical subtype of pancreatic cancer
              </a>
            </td>
          <td>
            Shalini V Rao, Lisa Young, D. Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) describes the process where distinct epigenetic states may be transmitted between generations, resulting in stable gene expression and phenotypic differences between individuals that persist independently of DNA sequence variation. Chromatin modifications have been demonstrated as important in TEI, however, the extent to which they require other signals to establish and maintain epigenetic states is still unclear. Here we investigate whether small non-coding RNAs contribute to different epigenetic states of the Fab2L transgene in Drosophila triggered by transient chromatin contacts, which requires Polycomb complex activity to deposit the H3K27me3 modification for long-term TEI. Using mutants deficient in known small non-coding RNAs, high throughput sequencing, investigation of chromatin conformation and gene expression analysis we demonstrate that small non-coding RNAs do not contribute directly to initiation or maintenance of silencing. However, we uncover an indirect role for microRNA expression in transgene silencing through effects on Polycomb complex expression. Additionally, we show that a commonly used marker gene, Stubble (Sb), affects Polycomb complex expression, which may be important in interpreting experiments assaying Polycomb function in Drosophila development. By ruling out a plausible candidate for TEI at the Fab2L transgene our work highlights the variability in different modes of TEI across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be548511a3f566003319963d618b6d4697f22992" target='_blank'>
              Polycomb-mediated transgenerational epigenetic inheritance of Drosophila eye colour is independent of small RNAs
              </a>
            </td>
          <td>
            Maximilian H. Fitz-James, Poppy Sparrow, Christopher Paton, P. Sarkies
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="During meiosis, each chromosome pair experiences at least one crossover (CO), which directs their balanced segregation in addition to shuffling genetic information. COs tend to be away from each other, a phenomenon known as CO interference. The main biochemical pathway for CO formation, which is conserved in distant eukaryotes, involves the ZMM proteins together with the MLH1-MLH3 complex (MutLγ). Here, we aim to clarify the role of MutLγ in CO formation in Arabidopsis thaliana. We show that AtMutLγ is partially dispensable for ZMM-dependant CO formation. HEI10 large foci - that mark CO sites in wild-type-form at a normal level in mlh1 and mlh3 mutants, but are inefficiently maturated into COs. Mutating the MUS81 nuclease in either mlh1 or mlh3 leads to chromosome fragmentation, which is suppressed by further mutating the zmm msh5. This suggests that in the absence of MutLγ, recombination intermediates produced by ZMMs are resolved by MUS81, which does not ensure CO formation. Finally, CO interference is not affected in mlh1, which is compatible with a random sub-sampling of normally patterned CO sites. We conclude that AtMutLγ imposes designated recombination intermediates to be resolved exclusively as COs, supporting the view that MutLγ asymmetrically resolves double-Holliday junctions, yielding COs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/804b8cf2149074d9d9f514d468fba802009e7940" target='_blank'>
              MutLγ enforces meiotic crossovers in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Stéphanie Durand, Qichao Lian, Victor Solier, J. B. Fernandes, Raphael Mercier
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Survivin is known for its dual biological role in apoptosis inhibition and mitotic progression. In addition to its being part of the chromosomal passenger complex (CPC), recent findings suggest additional roles for Survivin in the DNA damage response, further contributing to therapy resistance. In this study, we investigated the role of Survivin and the CPC proteins in the cellular response to irradiation with a focus on DNA replication processes. As is known, ionizing radiation leads to an increased expression of Survivin and its accumulation in nuclear foci, which we now know to be specifically localized to centromeric heterochromatin. The depletion of Survivin and Aurora B increases the DNA damage marker γH2AX, indicative of an impaired repair capacity. The presence of Survivin and the CPC in nuclear foci that we already identified during the S phase co-localize with the proliferating cell nuclear antigen (PCNA), further implying a potential role during replication. Indeed, Survivin knockdown reduced replication fork speed as assessed via DNA fiber assays. Mechanistically, we identified a PIP-box motif in INCENP mediating the interaction with PCNA to assist in managing damage-induced replication stress. Survivin depletion forces cells to undergo unphysiological genome replication via mitotic DNA synthesis (MiDAS), resulting in chromosome breaks. Finally, we revealed that Aurora B kinase liberates Pol η by phosphorylating polymerase delta-interacting protein 2 (POLDIP2) to resume the replication of damaged sites via translesion synthesis. In this study, we assigned a direct function to the CPC in the transition from stalled replication forks to translesion synthesis, further emphasizing the ubiquitous overexpression of Survivin particularly in tumors. This study, for the first time, assigns a direct function to the chromosomal passenger complex, CPC, including Survivin, Aurora B kinase, Borealin, and INCENP, in the transition from stalled replication forks (involving PCNA binding) to translesion synthesis (liberating Pol η by phosphorylating POLDIP2), and thus in maintaining genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933e65144a080ac3e72c369a27df2e2dc4c884df" target='_blank'>
              Old Passengers as New Drivers: Chromosomal Passenger Proteins Engage in Translesion Synthesis
              </a>
            </td>
          <td>
            Katharina Falke, E. Schröder, S. Mosel, Cansu N. Yürük, Sophie Feldmann, Désirée Gül, Paul Stahl, Roland H. Stauber, S. Knauer
          </td>
          <td>2024-10-31</td>
          <td>Cells</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70ab09cd735f5a7dcc73837a413f17cd6dc2a165" target='_blank'>
              Impact of DNA Repair Deficiency in the Evolving Treatment Landscape of Bladder Cancer.
              </a>
            </td>
          <td>
            Vincent D D'Andrea, C.J. Magnani, John Ernandez, J. Bellmunt, M. Mossanen, T. Clinton, F. L. Carvalho, K. Mouw
          </td>
          <td>2024-10-09</td>
          <td>Current urology reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ed2057a00d451d2750eb4c6690ca3020a9e3a6" target='_blank'>
              The prominent pervasive oncogenic role and tissue specific permissiveness of RAS gene mutations
              </a>
            </td>
          <td>
            Ming Yi, Daniel Soppet, Frank McCormick, D. Nissley
          </td>
          <td>2024-10-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genomic instability is a core characteristic of cancer, often stemming from defects in DNA damage response (DDR) or increased replication stress. DDR defects can lead to significant genetic alterations, including changes in gene copy numbers, gene rearrangements, and mutations, which accumulate over time and drive the clonal evolution of cancer cells. However, these vulnerabilities also present opportunities for targeted therapies that exploit DDR deficiencies, potentially improving treatment efficacy and patient outcomes. The development of PARP inhibitors like Olaparib has significantly improved the treatment of cancers with DDR defects (e.g., BRCA1 or BRCA2 mutations) based on synthetic lethality. This achievement has spurred further research into identifying additional therapeutic targets within the DDR pathway. Recent progress includes the development of inhibitors targeting other key DDR components such as DNA-PK, ATM, ATR, Chk1, Chk2, and Wee1 kinases. Current research is focused on optimizing these therapies by developing predictive biomarkers for treatment response, analyzing mechanisms of resistance (both intrinsic and acquired), and exploring the potential for combining DDR-targeted therapies with chemotherapy, radiotherapy, and immunotherapy. This article provides an overview of the latest advancements in targeted anti-tumor therapies based on DDR and their implications for future cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdf72d2a9fa59533e1f8a82e5fc32e8e54f1de8" target='_blank'>
              Advancing cancer therapy: new frontiers in targeting DNA damage response
              </a>
            </td>
          <td>
            Jiekun Qian, Guoliang Liao, Maohui Chen, Ren-Wang Peng, Xin Yan, Jianting Du, Renjie Huang, Maojie Pan, Yuxing Lin, Xian Gong, Guo-bing Xu, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Determining a novel etiology and mechanism of human cancer requires extraction of characteristic mutational signatures derived from chemical substances. This study explored the mutational signatures of N-nitroso bile acid conjugates using Salmonella strains. Exposing S. typhimurium TA1535 to N-nitroso-glycine/taurine bile acid conjugates induced a predominance of C:G to T:A transitions. Two mutational signatures, B1 and B2, were extracted. Signature B1 is associated with N-nitroso-glycine bile acid conjugates, while Signature B2 is linked to N-nitroso-taurine bile acid conjugates. Signature B1 revealed a strong transcribed strand bias with GCC and GCT contexts, and the mutation pattern of N-nitroso-glycine bile acid conjugates in YG7108, which lacks O6-methylguanine DNA methyltransferases, matched that of the wild-type strain TA1535, suggesting that O6-methyl-deoxyguanosine contributes to mutations in the relevant regions. COSMIC database-based similarity analysis revealed that Signature B1 closely resembled SBS42, which is associated with occupational cholangiocarcinoma caused by overexposure to 1,2-dichlolopropane (1,2-DCP) and/or dichloromethane (DCM). Moreover, the inflammatory response pathway was induced by 1,2-DCP exposure in a human cholangiocyte cell line, and iNOS expression was positive in occupational cholangiocarcinomas. These results suggest that 1,2-DCP triggers an inflammatory response in biliary epithelial cells by upregulating iNOS and N-nitroso-glycine bile acid conjugate production, resulting in cholangiocarcinoma via DNA adduct formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16ffe28d485b3038d354ba43187bea62b1635ee" target='_blank'>
              Whole Genome Sequencing Analysis of Model Organisms Elucidates the Association Between Environmental Factors and Human Cancer Development
              </a>
            </td>
          <td>
            Shinya Hasegawa, Yutaka Shoji, Mamoru Kato, Asmaa Elzawahry, M. Nagai, Min Gi, Shugo Suzuki, Hideki Wanibuchi, S. Mimaki, K. Tsuchihara, Y. Totsuka
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Many current genome editing technologies rely on the action of large serine integrases (LSIs) to insert gene-sized DNA sequences into the genome. Bxb1 is the most commonly used LSI for therapeutic efforts, including PASTE, PASSIGE and I-PGI. While Bxb1 demonstrated good activity in vitro in cycling cells, the activity in non-dividing hepatocytes was significantly less efficient. Further, wild-type Bxb1 is known to have detectable off-target activity at cryptic genomic sites, which presents a potential safety risk for therapeutic development. To address these issues, we developed Bxb1 variants that demonstrate increased specificity and potency in vitro and have engineered stabilized Bxb1 variants that increase in vivo activity over 25-fold enabling targeted integration at therapeutically relevant levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eabaf707807c0a3a5fcf5be4fb9e92d429e828a2" target='_blank'>
              Engineered Bxb1 variants improve integrase activity and fidelity
              </a>
            </td>
          <td>
            Joshua Rose, Hyaeyeong Kim, Jessica von Stetina, Mahant Thangavelu, Hasahn L. Conway, Mollie S. O’Hara, Patrick Hanna, Thomas Biondi, Maike Thamsen Dunyak, Jonathan D. Finn, Sandeep Kumar, Jesse C. Cochrane
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7228635119ad497118a336ef6dd401a87d976c61" target='_blank'>
              LINC01320 facilitates cell proliferation and migration of ovarian cancer via regulating PURB/DDB2/NEDD4L/TGF-β axis
              </a>
            </td>
          <td>
            Gaigai Wang, Bingya Xu, Xiangling Yu, Meng Liu, Tiantian Wu, Wenxin Gao, Haoyue Hu, Bing Jiang, Yibo Wu, Tao Zhou, Xia Chen, Cong Shen
          </td>
          <td>2024-10-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Differential RNA expression is becoming increasingly valuable in evaluating tumor heterogeneity for a better understanding of malignant tumors and guiding personalized therapy. However, traditional techniques for analyzing cellular RNA are mainly focused on determining the absolute level of RNA, which may lead to inaccuracies in understanding tumor heterogeneity, primarily due to i) the subtle differences in certain RNA types that have similar total concentrations and ii) the existence of variations in RNA expression across different samples. Herein, a detachable DNA assembly module is proposed that is capable not only of quantifying the expression level of target RNA but also of innovatively evaluating its proportion within its RNA family population through a sequential assembly and disassembly route. Using the let-7 family as an experimental model, a significant difference is discovered in let-7a proportion between normal mammary epithelial cells and breast cancer cells, a characteristic that is often missed in bulk analysis of traditional techniques. By combining concentration and proportion information, the detachable DNA assembly module demonstrates markedly higher efficiency in discerning among various types of cells compared to traditional techniques. This innovative assembly module is expected to offer a new perspective to highlight tumor heterogeneity and guide personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b6572cf9fba5fd9ee657a20fd3934f1dd18a4" target='_blank'>
              Detachable DNA Assembly Module to Dissect Tumor Cells Heterogeneity via RNA Pinpoint Screening.
              </a>
            </td>
          <td>
            Wei Liu, Ni Liao, Yanmei Lei, Wenbin Liang, Xia Yang, Ruo Yuan, Chao Yang, Ying Zhuo
          </td>
          <td>2024-10-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for assessing chromatin stability in healthy and sick dogs (with squamous cell carcinoma). The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable a more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. Abstract Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Disclosure: A. Sanghi: None. S. Wu: None. T. Chou: None. L.A. Orloff: None. M. Kasowski: None. M. Snyder: None. Although genetic mutations are causal in human cancers, the contribution of chromatin changes to cancer onset and progression remains less well understood. Multi-omics profiling of human tumors can provide insight into how alterations in chromatin structure are propagated through the pathway of gene expression to result in cancer signaling. We applied multi-omics profiling of 36 human thyroid cancer primary tumors, metastases, and patient-matched normal tissue. Over 80% of the tumors harbored BRAFV600E mutations, and the remaining tumors mostly had activating mutations in the MAPK pathway. Through quantification of chromatin accessibility associated with active transcription units and global protein expression, we identify local enhancers that are highly correlated with coordinated RNA and protein expression. The enhancers are predicted to be actively bound by MAPK transcription factor, indicating they are under MAPK control. These cancer-specific enhancers associated with upregulation of lysosomal peptidase expression and dipeptide abundance in tumors. We show that upregulated dipeptides stimulate MAPK activity, and BRAF inhibition results in loss of lysosomal function in mutants. Perturbation of lysosomal peptidases and dipeptide abundance decreases MAPK activity only in mutant cells compared to wildtype. These analyses suggest that BRAF thyroid tumors integrate MAPK-driven epigenetic regulation with lysosomal metabolite signaling in a positive feedback circuit. Using local cancer-specific epigenetic features, we isolated key cancer signaling, making way for potential targets for cancer therapeutics. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/032e66998348d599db74b255c1962f11618e9fbe" target='_blank'>
              7948 Integration of Epigenomics and Metabolomics Reveals New Signaling in Aggressive Thyroid Cancer
              </a>
            </td>
          <td>
            A. Sanghi, S. Wu, T. Chou, L. A. Orloff, M. Kasowski, M. Snyder
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The integrity of the actively transcribed genome against helix-distorting DNA lesions relies on a multilayered cellular response that enhances Transcription-Coupled Nucleotide Excision Repair (TC-NER). When defective, TC-NER is causatively associated with Cockayne-Syndrome (CS), a rare severe human progeroid disorder. Although the presence of unresolved transcription-blocking lesions is considered a driver of the aging process, the molecular features of the transcription-driven response to genotoxic stress in CS-B cells remain largely unknown. Here, an in-depth view of the transcriptional and associated chromatin dynamics that occur in CS-B cells illuminates the role of CSB therein. By employing high-throughput genome-wide approaches, we observed that absence of a functional CSB protein results in a delay in transcription progression, more positioned +1 nucleosomes, and less dynamic chromatin structure, compared to normal cells. We found that early after exposure to UV, CS-B cells released RNA polymerase II (RNAPII) from promoter-proximal pause sites into elongation. However, the magnitude of this response and the progression of RNAPII were reduced compared to normal counterparts. Notably, we detected increased post-UV retainment of unprocessed nascent RNA transcripts and chromatin-associated elongating RNAPII molecules. Contrary to the prevailing models, we found that transcription initiation is operational in CS-B fibroblasts early after UV and that chromatin accessibility showed a marginal increase. Our study provides robust evidence for the role of CSB in shaping the transcription and chromatin landscape both in homeostasis and in response to genotoxic insults, which is independent of its known role in TC-NER, and which may underlie major aspects of the CS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966a15785f4d1763d145786faf0eda45c3c2df7f" target='_blank'>
              Cockayne syndrome B protein is implicated in transcription and associated chromatin dynamics in homeostatic and genotoxic conditions.
              </a>
            </td>
          <td>
            Anastasios Liakos, Katerina Z Ntakou-Zamplara, Nelina Angelova, Dimitris Konstantopoulos, Anna-Chloe Synacheri, Zoi Spyropoulou, Iason A Tsarmaklis, Despoina Korrou-Karava, Georgios Nikolopoulos, Matthieu D. Lavigne, Maria Fousteri
          </td>
          <td>2024-10-06</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Study of early pancreas neoplasia in humans had previously been limited by lack of available tissue. Recent work by our group on deceased donor pancreata identified pancreatic intraepithelial neoplasia (PanIN) lesions in non-diseased organs and defined a transcriptomic signature for the microenvironment of these pre-cancerous lesions. Prevalence of PanINs in healthy human tissue was higher than expected and established a novel model to expand understanding of the complex biology of these precursor lesions. Epigenetic changes in chromatin structure and the subsequent effects on gene expression are essential processes in neoplasia that have yet to be described at the single cell level in the human pancreas. We hypothesized that previously defined gene signatures could identify acinar, ductal, and PanIN cells in donor pancreata based on chromatin accessibility profiling by single nuclear transposase-accessible chromatin preparation followed by high-throughput sequencing (snATAC-seq). Comparison between the accessibility patterns, transcription factor motifs, and differences in accessibility versus expression profiles can then be used to expand prior knowledge of early pancreatic neoplasia and identify novel targets for further study and therapy selection. To test this hypothesis, single nuclei from seventeen donor pancreata (twenty total samples) were isolated for snATAC-seq using the 10x Genomics platform. Data analysis was conducted using CellRanger, Seurat, Signac, AUCell, and other programs to identify and label chromatin peaks in individual nuclei corresponding to acinar, ductal, and PanIN origin according to gene signatures identified by scRNA-seq and special transcriptomics on matched donor pancreata. Using this method, cells corresponding to these gene signatures, including PanIN- like cells, were indeed identifiable with differentially accessible regions of the genome then analyzed for transcription factor motif enrichment between these cell types. Motifs enriched in cells identified as PanINs compared to either ductal or acinar cells included several transcription factors implicated in tumorigenesis. These results are similar to previously demonstrated essential factors in pancreatic neoplasia, suggesting this strategy is a relevant method for data analysis and discovery of novel factors implicated in tumorigenesis. Ongoing studies include validation of these findings in vitro using human pancreatic normal epithelial cells, human tumor cell lines, and organoid models derived from human normal and tumor samples. Together, this study demonstrates that PanINs can be identified in human pancreas tissue by snATAC-seq and that differentially expressed motifs can be identified, providing opportunity for further integration with transcriptomic information to elucidate detailed understanding of early neoplasia in the pancreas for future drug development and targeted therapeutics studies.
 Citation Format: Jamie N Mills, Aaron Dendekker, Joyce K Thompson, Hannah Watkoske, Simone Benitz, Padma Kadiyala, Ahmed Elhossiny, Howard Crawford, Eileen Carpenter, Marina Pasca di Magliano, Filip Bednar. Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee7faf599c3d04827a3bdb6a5acb4e73be0ca2f" target='_blank'>
              Abstract C056: Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq
              </a>
            </td>
          <td>
            Jamie N. Mills, Aaron Dendekker, Joyce K Thompson, Hannah R Watkoske, S. Benitz, Padma Kadiyala, Ahmed M. Elhossiny, Howard C. Crawford, Eileen S Carpenter, M. Pasca di Magliano, Filip Bednar
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42bb491bf59a8058f50cf3417e3f4f9a441fc86" target='_blank'>
              Cancer treatment monitoring using cell-free DNA fragmentomes
              </a>
            </td>
          <td>
            Iris van 't Erve, B. Alipanahi, K. Lumbard, Z. Skidmore, L. Rinaldi, Laurel K Millberg, J. Carey, B. Chesnick, S. Cristiano, Carter Portwood, Tony Wu, E. Peters, K. Bolhuis, Cornelis J A Punt, Jennifer Tom, P.B. Bach, N. Dracopoli, Gerrit A Meijer, Robert B Scharpf, V. Velculescu, R. Fijneman, A. Leal
          </td>
          <td>2024-10-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="
 Women of African descent are at an increased risk of developing and dying from aggressive subtypes of breast cancer. A connection between aggressive disease and Western Sub-Saharan African ancestry has been postulated, but it remains largely unknown to what extent breast cancer in Africa is reminiscent of breast cancer in U.S. African American (AA) women who experience disproportionately high mortality rates. We performed ATAC- and RNA-sequencing on 9 human triple-negative breast cancer cell lines of U.S. origin and discovered that African ancestry influences the chromatin landscape, leading to disparate transcription factor (TF) activity and downstream gene expression patterns indicative of an aggressive tumor biology. Here, we describe an ambitious study that employs single-nucleus (sn) ATAC- and RNA-sequencing (snMultiome) of frozen breast tumors to characterize chromatin accessibility and gene expression patterns with single-cell resolution in AA (n=33), Kenyan (n=25), and European American (EA, n=24) women in relation to genetic ancestry, risk factor exposures, clinical characteristics, and 5-year survival. To achieve this, we successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue through an optimized combination of enzymatic digestion and automated tissue homogenization. We performed snMultiome sequencing of 82 tumors using the 10x Genomics platform. Following filtering, normalization (SCT for snRNA; LSI for snATAC), peak calling (MACS2), and integration (Harmony), our dataset includes a total of 296,557 nuclei. Cancerous (163,419 nuclei) and non-cancerous (133,138 nuclei) cells were distinguished based on DNA copy number (CopyKat). Within the microenvironment, 11 major immune, epithelial, and stromal cell types were successfully annotated, exhibiting distinct patterns by population group (e.g. AA tumors showed markedly increased abundance of myeloid and T-cells, while Kenyan tumors showed increased abundance of pericytes and fibroblasts, relative to EA). A large number of enriched TFs within each cell type varied significantly by population group, suggesting distinct chromatin accessibility patterns related to genetic ancestry. CD45+, EPCAM- cells were further extracted, clustered, and manually annotated for subpopulations based on known marker genes. We detected 26 distinct immune subpopulations across our samples, including 2 inflammatory macrophage, 6 immunosuppressive macrophage, 1 fibronectin+/thrombospondin+ monocyte, 6 T-cell, 1 natural killer cell, 2 B-cell, 1 mast cell, and 4 dendritic cell subpopulations. Differential gene expression analyses of each immune cell subpopulation, adjusted for age, BMI, and tumor subtype, revealed ancestry-specific gene expression patterns, which could provide novel insights into functional differences that may impact tumor immune response. Current efforts focus on in-depth molecular characterization of ancestry-related differences in the tumor immune environment and distinct signatures present in lethal disease.
 Citation Format: Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Tiffany H. Dorsey, Francis Makokha, Shahin Sayed, Gretchen Gierach, Stefan Ambs. Characterization of the breast tumor immune microenvironment in women of African descent using single-nucleus RNA- and ATAC-sequencing [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c691a244278d4d06415c78c2b87c77a2297b77d" target='_blank'>
              Abstract C089: Characterization of the breast tumor immune microenvironment in women of African descent using single-nucleus RNA- and ATAC-sequencing
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Tiffany H Dorsey, Francis W. Makokha, S. Sayed, Gretchen Gierach, S. Ambs
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c35f97c112d09b7e983874a9efe5c2fc2f69d6c" target='_blank'>
              Enhancers in T Cell development and malignant lesions
              </a>
            </td>
          <td>
            Tong Zhang, Lin Zou
          </td>
          <td>2024-09-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precise regulation of gene expression is essential for proper development and the maintenance of homeostasis in organisms. Studies have shown that some transcriptional regulatory proteins influence gene expression through the formation of dynamic, locally concentrated assemblies known as condensates, while dysregulation of transcriptional condensates was associated with several cancers, such as Ewing sarcoma and AML [Wang Y et al. (2023) Nat Chem Biol 19, 1223-1234; Chandra B et al. (2022) Cancer Discov 12, 1152-1169]. Mutations in the histone acetylation "reader" eleven-nineteen-leukemia (ENL) have been shown to form discrete condensates at endogenous genomic targets, but it remains unclear how ENL mutations drive tumorigenesis and whether it is correlated with their condensate formation property. Liu et al. now show, using a conditional knock-in mouse model, that ENL YEATS domain mutation is a bona fide oncogenic driver for AML. This mutant ENL forms condensates in hematopoietic stem/progenitor cells at the genomic loci of key leukemogenic genes, including Meis1 and Hoxa cluster genes, and disrupting condensate formation via mutagenesis impairs its chromatin and oncogenic function. Furthermore, they show that small-molecule inhibition of the acetyl-binding activity displaces ENL mutant condensates from oncogenic target loci, and this inhibitor significantly impairs the onset and progression of AML driven by mutant ENL in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9e0f2605685a3812a479056f634ccfed5efb36" target='_blank'>
              ENL mutation and AML: a new model that reveals oncogenic condensate's function in leukemogenesis.
              </a>
            </td>
          <td>
            Zhong Fan, Yanan Jiang, Xiaotian Zhang
          </td>
          <td>2024-09-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Efficient DNA replication requires highly coordinated programs for the timely recruitment of protein complexes to DNA replication forks. Defects in this process result in replication stress, which in turn activates cell cycle checkpoints, suppresses cell proliferation and induces apoptosis. In response to persistent cell growth signals that speed up DNA replication processes, cells accelerate the recruitment of DNA replication proteins to avoid DNA replication stress. The mechanisms by which cell growth signals induce processes to facilitate the recruitment of DNA replication proteins onto the replication sites remain unclear. Here, we report that the epidermal growth factor receptor (EGFR) phosphorylates heat shock protein 70 (HSP70) for DNA replication. Such a modification promotes nuclear localization and chromatin association of HSP70, which interacts with the DNA replication coordinator, proliferating cell nuclear antigen (PCNA). HSP70 subsequently facilitates the loading of PCNA onto chromatin. Knockdown or chemical inhibition of HSP70 suppresses PCNA association with chromatin and impairs DNA synthesis and Okazaki fragment maturation, leading to replicative DNA double-strand breaks and apoptosis. Furthermore, chemical inhibition of HSP70 potentiates EGFR-tyrosine kinase inhibitor-induced tumor reduction in vivo. This work expands our understanding of oncogenesis-induced DNA replication processes and provides a foundation for improved treatments for EGFR-mutated lung cancer by simultaneously targeting HSP70.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca2470519b5a8cc8cd3ca52af8ede3fd0298ef0" target='_blank'>
              EGFR-mediated HSP70 phosphorylation facilitates PCNA association with chromatin and DNA replication.
              </a>
            </td>
          <td>
            Yingying Wang, Anthony Fernandez, Xinyu Pei, Bing Liu, Lei Shen, Yao Yan, Hitendra S Solanki, Lin Yang, Mian Zhou, Yuming Guo, Jun Wu, K. Reckamp, Li Zheng, Binghui Shen
          </td>
          <td>2024-10-29</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb6a4f203d0ffda30fcc3f0dfa56bd3b405a484" target='_blank'>
              Amplified centrosomes—more than just a threat
              </a>
            </td>
          <td>
            Eva Kiermaier, Isabel Stötzel, Marina A Schapfl, A. Villunger
          </td>
          <td>2024-09-16</td>
          <td>EMBO Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f337210da04933a9ae37820fdc6d7b7fe43d5d5" target='_blank'>
              CALB2 drives pancreatic cancer metastasis through inflammatory reprogramming of the tumor microenvironment
              </a>
            </td>
          <td>
            Jinxin Tao, Yani Gu, Zeyu Zhang, Guihu Weng, Yueze Liu, Jie Ren, Yanan Shi, J. Qiu, Yuanyang Wang, Dan Su, Ruobing Wang, Yifan Fu, Tao Liu, Liyuan Ye, W. Luo, Hao Chen, Gang Yang, Zhe Cao, Hua Huang, Jianchun Xiao, Bo Ren, Lei You, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-10-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bb750f22d3a6ebea43d03722b56d7da477c099" target='_blank'>
              Causes and consequences of DNA double-stranded breaks in cardiovascular disease.
              </a>
            </td>
          <td>
            A. Marian
          </td>
          <td>2024-10-15</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Each cancer is the result of an individually unique evolutionary process. Yet, as demonstrated by extensive genetic analyses of human specimens, the genetic endpoints of cancers across different tissues are remarkably specific, with individual cancer driver genes being typically associated with only a limited set of tumor types. Tissue and cellular contexts thus impose strong and genetically predictable evolutionary constraints on carcinogenesis. Lineage-specific transcription factors, central regulators of organismal development, tissue homeostasis, and regeneration, often also support cancer cell fitness in a lineage-specific manner. In this review, we discuss recent results on the interactions between transcriptional lineage factor programs and oncogenic pathways and how such interactions may determine the oncogenic competence of cancer-associated genetic alterations. These developments are starting to shed light on the molecular basis of tissue specificity in carcinogenesis, with relevance for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f236b52bc63786a8f5d7d9e1f3c0d9457cd40be6" target='_blank'>
              Lineage-Specific Transcription Factors in Carcinogenesis
              </a>
            </td>
          <td>
            Charles J. David, Sakari Vanharanta
          </td>
          <td>2024-10-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) govern the specific patterns and dynamics of gene expression in cells during development, which are the fundamental mechanisms behind cell differentiation. However, the genomic characteristics of single-cell CREs (scCREs) closely linked to cell differentiation during development remain unclear. To explore this, we systematically analyzed ∼250,000 putative scCREs obtained from snATAC-seq analysis of the developing mouse cerebellum. We found that over 80% of these scCREs show pleiotropic effects, being active in two or more cell types. The pleiotropic degrees of proximal and distal scCREs are positively correlated with the density and diversity of transcription factor (TF) binding motifs and GC content. There is a negative correlation between the pleiotropic degrees of scCREs and their distances to the nearest TSSs, and proximal scCREs display higher relevance strengths than distal ones. Furthermore, both proximal and distal scCREs related to cell differentiation exhibit enhanced sequence-level evolutionary conservation, increased density and diversity of TF binding motifs, elevated GC content, and greater distances from their nearest genes. Together, our findings reveal the general genomic characteristics of putative scCREs and provide insights into the genomic and evolutionary mechanisms by which scCREs regulate cell differentiation during development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4ca574f55ca19f77da34f44bd0df942265e3268" target='_blank'>
              Characterization of single-cell cis-regulatory elements informs implications for cell differentiation.
              </a>
            </td>
          <td>
            , Zhen Liu
          </td>
          <td>2024-11-07</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39abde50129bd51799285e63a0661a0178d6716" target='_blank'>
              DNA damage response in breast cancer and its significant role in guiding novel precise therapies
              </a>
            </td>
          <td>
            Jiayi Li, Ziqi Jia, Lin Dong, Heng Cao, Yansong Huang, Hengyi Xu, Zhixuan Xie, Yiwen Jiang, Xiang Wang, Jiaqi Liu
          </td>
          <td>2024-09-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Breast cancer, despite significant advancements in treatment, remains a major cause of cancer-related deaths among women. Immunotherapy, an emerging therapeutic strategy, offers promise for better outcomes, particularly through the modulation of immune functions. Glioma-Associated Oncogene Homolog 1 (GLI1), a transcription factor implicated in cancer biology, has shown varying roles in different cancers. However, its immunoregulatory functions in breast invasive carcinoma (BRCA) remain elusive. The current study aimed to unravel the expression patterns and immune-regulatory roles of GLI1 in BRCA. Methods Utilizing multiple bioinformatic platforms (TIMER2.0, GEPIA2, and R packages) based on The Cancer Genome Atlas (TCGA) and/or Genotype-Tissue Expression (GTEx) databases, we analyzed the expression of GLI1 in BRCA and its pan-cancer expression profiles. We further validated these findings by conducting qPCR and immunohistochemical staining on clinical BRCA samples. Kaplan-Meier analysis and Cox proportional hazards regression were performed to assess the prognostic value of GLI1. Additionally, the association between GLI1 expression and immune infiltration within the tumor immune microenvironment (TMIE) was examined. Results The findings reveal dysregulated expression of GLI1 in numerous cancers, with a significant decrease observed in BRCA. High GLI1 expression indicated better survival outcomes and was correlated with the age and stage of BRCA patients. GLI1 was involved in immune status, as evidenced by its strong correlations with immune and stromal scores and the infiltration levels of multiple immune cells. Meanwhile, GLI1 was co-expressed with multiple immune-related genes, and high GLI1 expression was associated with the activation of immune-related pathways, such as binding to proteasome and mismatch repair and retinol metabolism signaling pathways. Additionally, the differential expression of GLI1 may be related to the effect of immunotherapy on CTLA-4, PD-1, and other signals, and can effectively predict the immune efficacy. Conclusion Our study underscores the critical role of GLI1 in BRCA, both as a potential tumor suppressor and an immune regulator. The association between GLI1 expression and favorable prognosis suggests its potential as a prognostic biomarker and immunotherapeutic target in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a81802d7ee86a4f3f182d6937952c4611208b17a" target='_blank'>
              Glioma-associated oncogene homolog 1 in breast invasive carcinoma: a comprehensive bioinformatic analysis and experimental validation
              </a>
            </td>
          <td>
            Teng Qi, Yujie Hu, Junhao Wan, Bo Zhao, Jinsuo Xiao, Jie Liu, Ye Cheng, He Wu, Yonggang Lv, Fuqing Ji
          </td>
          <td>2024-10-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA in eukaryotic genomes is under constant assault from both exogenous and endogenous sources, leading to DNA damage, which is considered a major molecular driver of ageing. Fortunately, the genome and the central exome are safeguarded against these attacks by abundant peripheral non-coding DNA. Non-coding DNA codes for small non-coding RNAs that inactivate foreign nucleic acids in the cytoplasm and physically blocks these attacks in the nucleus. Damage to non-coding DNA produced during such blockage is removed in the form of extrachromosomal circular DNA (eccDNA) through nucleic pore complexes. Consequently, non-coding DNA serves as a line of defence for the exome against DNA damage. The total amount of non-coding DNA/heterochromatin declines with age, resulting in a decrease in both physical blockage and eccDNA exclusion, and thus an increase in the accumulation of DNA damage in the nucleus during ageing and in age-related diseases. Here, we summarize recent evidence supporting a protective role of non-coding DNA in healthy and pathological states and argue that DNA damage is the proximate cause of ageing and age-related genetic diseases. Strategies aimed at strengthening the protective role of non-coding DNA/heterochromatin could potentially offer better systematic protection for the dynamic genome and the exome against diverse assaults, reduce the burden of DNA damage to the exome, and thus slow ageing, counteract age-related genetic diseases and promote a healthier life for individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7879deb4075aa031b3e715005da950caf3fffb2" target='_blank'>
              Protection of the genome and the central exome by peripheral non-coding DNA against DNA damage in health, ageing and age-related diseases.
              </a>
            </td>
          <td>
            Guo-Hua Qiu, Mingjun Fu, Xintian Zheng, Cuiqin Huang
          </td>
          <td>2024-09-26</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653d7fee74f8332187f2be2503fada0f457b09c5" target='_blank'>
              STAG2 loss in Ewing sarcoma alters enhancer-promoter contacts dependent and independent of EWS::FLI1.
              </a>
            </td>
          <td>
            D. Giménez-Llorente, Ana Cuadrado, María José Andreu, Inmaculada Sanclemente-Alamán, Maria Solé-Ferran, M. Rodríguez-Corsino, Ana Losada
          </td>
          <td>2024-11-01</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma is characterized by high recurrence rates and poor prognosis. Cancer stem cells contribute to tumor heterogeneity, treatment resistance, and recurrence. This study aims to identify key genes associated with stemness and immune cell infiltration in HCC. We analyzed RNA sequencing data from The Cancer Genome Atlas to calculate mRNA expression-based stemness index in HCC. A weighted gene co-expression network analysis was performed to identify stemness-related gene modules. A single-sample gene set enrichment analysis was used to evaluate immune cell infiltration. Key genes were validated using RT-qPCR. The mRNAsi was significantly higher in HCC tissues compared to adjacent normal tissues and correlated with poor overall survival. WGCNA and subsequent analyses identified 10 key genes, including minichromosome maintenance complex component 2, cell division cycle 6, forkhead box M1, NIMA-related kinase 2, Holliday junction recognition protein, DNA topoisomerase II alpha, denticleless E3 ubiquitin protein ligase homolog, maternal embryonic leucine zipper kinase, protein regulator of cytokinesis 1, and kinesin family member C1, associated with stemness and low immune cell infiltration. These genes were significantly upregulated in HCC tissues. A functional enrichment analysis revealed their involvement in cell cycle regulation. This study identified 10 key genes related to stemness and immune cell infiltration in HCC. These genes, primarily involved in cell cycle regulation, may serve as potential targets for developing more effective treatments to reduce HCC recurrence and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cdb0958bba2e0a1bf3933bb69de1a7e184f583f" target='_blank'>
              Identification of Crucial Cancer Stem Cell Genes Linked to Immune Cell Infiltration and Survival in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Lien-Hung Huang, Shao-Chun Wu, , , P. Chien, Hui-Ping Lin, Chia-Jung Wu, Ting-Min Hsieh, Ching-Hua Hsieh
          </td>
          <td>2024-11-07</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Humans are exposed to DNA alkylating agents through endogenous metabolism, environmental exposure and cancer chemotherapy. The resulting alkylated DNA adducts may elicit genome instability by perturbing DNA replication and transcription. R-loops regulate various cellular processes, including transcription, DNA repair, and telomere maintenance. However, unscheduled R-loops are also recognized as potential sources of DNA damage and genome instability. In this study, by employing fluorescence microscopy and R-loop sequencing approaches, we uncovered, for the first time, that minor-groove N2-alkyl-dG lesions elicit elevated R-loop accumulation in chromatin and in plasmid DNA in cells. We also demonstrated that the N2-alkyl-dG-induced R-loops impede transcription elongation and compromise genome integrity. Moreover, genetic depletion of DDX23, a R-loop helicase, renders cells more sensitive toward benzo[a]pyrene diolepoxide, a carcinogen that induces mainly the minor-groove N2-dG adduct. Together, our work unveiled that unrepaired minor-groove N2-alkyl-dG lesions may perturb genome integrity through augmenting R-loop levels in chromatin. Our findings suggest a potential therapeutic strategy involving the combination of R-loop helicase inhibitors with DNA alkylating drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bca69f7ad1277a7093d1fb762d63dd7f27acf5" target='_blank'>
              N 2-Alkyl-dG lesions elicit R-loop accumulation in the genome.
              </a>
            </td>
          <td>
            Yinan Wang, Feng Tang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-10-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast Cancer stands on the second position in the world in being common and women happen to have it with high rate of about five-folds around the world. The causes of occurrence can matter with different humans be it external factors or the internal genetic ones. Breast cancer is primarily driven by mutations in the BRCA1 and BRCA2 susceptibility genes. These BC susceptibility genes encode proteins critical for DNA homologous recombination repair (HRR). Poly (ADP ribose) polymerases (PARP) are the essential enzymes involved in the repairing of the damaged DNA. So the inhibition of these inhibitors can be considered as the promising strategy for targeting cancers with defective damage in the deoxyribonucleic acid. Olaparib and talazoparib are PARP inhibitors (PARPi) are being employed for the monotherapies in case of the deleterious germline HER2-negative and BRCA-mutated breast cancer. The potency of PARP for trapping on DNA and causes cytotoxicity may have difference in the safety and efficacy with the PARPi. The PARPi have been found its place in the all different types of Breast Cancers and have shown potential benefits. The purpose of this review is to provide an update on the oral poly(ADP-ribose) polymerase (PARP)inhibitors for the improvement in the treatment and management of Breast Cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45e72ba4b14c0918fa21b6e7ff5f0987ade38944" target='_blank'>
              PARP Pioneers: Using BRCA1/2 Mutation-targeted Inhibition to Revolutionize Breast Cancer Treatment.
              </a>
            </td>
          <td>
            Navneet Sharma, Akash Bhati, Shagun Aggarwal, Kamal Shah, H. K. Dewangan
          </td>
          <td>2024-10-17</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Telomeres are repetitive DNA sequences and associated protein complexes located at the end of chromatin. As a result of the DNA replication ending issue, telomeric DNA shortens during each cell cycle. The shelterin protein complex caps telomeric ends and forms a high-order protein-DNA structure to protect telomeric DNA. The stability of telomeres is critical for cellular function and is related to the progression of many human diseases. Telomeric repeat-containing RNA (TERRA) is a noncoding RNA transcribed from telomeric DNA regions. TERRA plays an essential role in regulating and maintaining the stability of telomeres. Heterogeneous nuclear ribonucleoproteins (hnRNPs) are RNA-binding proteins associated with complex and diverse biological processes. HnRNPA1 can recognize both TERRA and telomeric DNA. Previous research reported that hnRNPA1, TERRA, and POT1, a component of the shelterin complex, worked coordinately and displaced replication protein A from telomeric ssDNA after DNA replication, promoting telomere capping to preserve genomic integrity. However, the detailed molecular mechanism has remained unclear for over twenty years. Our study revealed the molecular structure through which the hnRNPA1 UP1 domain interacts with TERRA. Through structural analysis, we identified critical residues on the interacting surface between UP1 and TERRA. Furthermore, we proved that nucleic acids significantly increase the phase separation ability of hnRNPA1 and disrupting the UP1-TERRA interaction extraordinarily affects hnRNPA1 droplet formation both in vitro and in vivo. Taken together, these data revealed the molecular mechanism of the droplet formation of hnRNPA1 and TERRA and the possible function of the droplets for maintaining genomic stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25492a2b1a56e49b28564a2273617aaa54f72032" target='_blank'>
              Phase separation of hnRNPA1 and TERRA regulates telomeric stability.
              </a>
            </td>
          <td>
            Ziyan Xu, Yongrui Liu, Fudong Li, Yi Yang, Hong Zhang, Xing Liu, Xin Xie, Xianjun Chen, Yunyu Shi, Liang Zhang
          </td>
          <td>2024-09-23</td>
          <td>Journal of molecular cell biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281649fc3a01628d835005fce4ae40aa70c5e3f2" target='_blank'>
              Oncometabolites at the crossroads of genetic, epigenetic and ecological alterations in cancer.
              </a>
            </td>
          <td>
            Letizia Lanzetti
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alterations in deoxyribonucleoside triphosphate (dNTP) pools have been linked to increased mutation rates and genome instability in unicellular organisms and cell cultures. However, the role of dNTP pool changes in tumor development in mammals remains unclear. In this study, we present a mouse model with a point mutation at the allosteric specificity site of ribonucleotide reductase, RRM1-Y285A. This mutation reduced ribonucleotide reductase activity, impairing the synthesis of deoxyadenosine triphosphate (dATP) and deoxyguanosine triphosphate (dGTP). Heterozygous Rrm1+/Y285A mice exhibited distinct alterations in dNTP pools across various organs, shorter lifespans and earlier tumor onset compared with wild-type controls. Mutational spectrum analysis of tumors revealed two distinct signatures, one resembling a signature extracted from a human cancer harboring a mutation of the same amino acid residue in ribonucleotide reductase, RRM1Y285C. Our findings suggest that mutations in enzymes involved in dNTP metabolism can serve as drivers of cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe3b3a3839a433e41d5ae4062ca8fa74fde87a0" target='_blank'>
              Altered dNTP pools accelerate tumor formation in mice.
              </a>
            </td>
          <td>
            Phong Tran, Pradeep Mishra, Leonard G. Williams, Roman Moskalenko, Sushma Sharma, A. K. Nilsson, Danielle L. Watt, Pernilla Andersson, Anders Bergh, Z. Pursell, A. Chabes
          </td>
          <td>2024-10-03</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a101d4e936af34d1b466e5552558a397fc61b" target='_blank'>
              Carcinogenic Mechanisms of Hexavalent Chromium: From DNA Breaks to Chromosome Instability and Neoplastic Transformation.
              </a>
            </td>
          <td>
            Idoia Meaza, Aggie R Williams, Sandra S Wise, Haiyan Lu, John W Pierce
          </td>
          <td>2024-10-28</td>
          <td>Current environmental health reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b57b76c24e922e56ff6f3fa5ae5de6f644999f2" target='_blank'>
              Human papillomavirus-encoded circular RNA circE7 promotes immune evasion in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Junshang Ge, Yi Meng, Jiayue Guo, Pan Chen, Jie Wang, Lei Shi, Dan Wang, Hongke Qu, Pan Wu, Chunmei Fan, Shanshan Zhang, Q. Liao, Ming Zhou, Bo Xiang, Fuyan Wang, Ming-Liang Tan, Zhaojian Gong, Wei Xiong, Zhaoyang Zeng
          </td>
          <td>2024-10-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Endometrial carcinoma (EC) is a prevalent gynecological malignancy that poses a significant threat to women’s health worldwide. However, its pathogenesis and underlying mechanisms remains unclear. In this study, expression quantitative trait loci data, Mendelian randomization analysis, and differential expression analysis were performed to identify potential targets. A prognostic risk signature was subsequently constructed for EC patients based on the expression of these genes. Four bioinformatics algorithms, including generalized linear model, extreme gradient boosting, support vector machine, and random forest, were used to identify hub genes in EC. The expression of ring finger protein 144A (RNF144A) was validated using quantitative real-time polymerase chain reaction. Cellular proliferation and migration ability were evaluated using CCK-8 and Transwell assays, respectively. The genes RNF144A, ketohexokinase, and Rab interacting lysosomal protein like 2 were identified as potential targets for EC. Their differential expression was observed in EC patients, and Mendelian randomization analysis revealed a negative correlation between these genes and the development of EC. Mechanistic analyses suggested a strong association between these genes and the tumor immune microenvironment. The constructed risk signature was significantly associated with the prognosis, age, cancer stage, and grade of EC patients. Furthermore, based on interacted model algorithms, RNF144A was identified as a hub gene in EC. It was found to be significantly downregulated in EC samples, and its expression was positively correlated with the stage and grade of EC patients. In vitro experiments showed that overexpression of RNF144A significantly promoted cell growth and migration in EC cells. In conclusion, this study provides insights into the molecular mechanisms underlying EC progression and identifies preliminary candidate biomarkers for the development of EC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30ccab2586145d3cccf21eccbbc93cc5b295b360" target='_blank'>
              RNF144A as a potential risk gene for endometrial carcinoma: Insights from Mendelian randomization, bulk RNA sequencing, single-cell RNA, and experimental analysis
              </a>
            </td>
          <td>
            Xiqin Ruan, Ni Wang, Qingwen Xie, Yi Du
          </td>
          <td>2024-10-04</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Despite extensive research, the complex biology of pancreatic ductal adenocarcinoma (PDAC) continues to impede progress in improving its prognosis. Its resistance to conventional therapies, along with diverse mutations, necessitate improved molecular stratification to develop tailored approaches and to improve survival of PDAC patients. Our previous studies on ATM- deficient models revealed that serine/threonine kinase ATM loss induces homologous recombination deficiency (HRD), promoting invasiveness and desmoplastic reactions, while also sensitizing tumors to platinum-based therapies and PARP1 inhibition. Interestingly, our data also identified the histone-methyltransferase EZH2 as a direct ATM substrate to operate as a crucial regulator of ATM-specific oncogenic traits. In detail, murine and human ATM-deficient tumors accumulated EZH2, suggesting that ATM status contributes to rewiring the epigenetic landscape of pancreatic tumors, subsequently supporting oncogenic features. To investigate how EZH2 impacts ATM-deficient PDAC pathogenesis, we generated conditional double ATM and EZH2-knockout in the Kras G12D/+ ; Ptf1a Cre/+ mice (AKC, Atm fl/fl ; LSL-Kras G12D/+ ; Ptf1a Cre/+ , and EAKC, Ezh2 fl/fl ; Atm fl/fl ; LSL-Kras G12D/+ ; Ptf1a Cre/+ ). EZH2 depletion in AKC mice prolonged overall survival and reinstated a differentiated phenotype. Furthermore, PDAC patients with negative ATM and EZH2 expression from the TCGA-PAAD cohort (PanCancer Atlas) also showed significantly extended progression-free survival compared to ATM neg EZH2 pos patients. In line, CRISPR/Cas9-mediated EZH2 depletion in ATM-deficient pancreatic cancer cells affected their epigenetic landscape and significantly upregulated transcriptomic programs associated with cell differentiation, thereby restoring differentiated tumor features ex vivo. We showed that ATM status significantly impacts EZH2 activity through a post-translational pathway, promoting tumor supportive EZH2 activity in pancreatic cancer. ATM kinase facilitates EZH2 ubiquitination and degradation by recruiting the Cullin1 E3-ubiquitin ligase complex. Additionally, combining an EZH2 inhibitor with PARP inhibition enhances cytotoxicity in ATM- deficient HRD PDAC. Overall, our findings demonstrate that ATM deficiency drives a pro- oncogenic mesenchymal differentiation program via the histone-methyltransferase EZH2. We emphasize the importance of stratifying patients based on ATM status before considering pharmacological interference with EZH2, offering a promising approach for targeting ATM- deficient HRD tumors.
 Citation Format: Elodie Roger, Anna Härle, Lukas Perkhofer, Elisabeth Hessmann, Alexander Kleger, Johann Gout. Epigenetic reprograming in ATM-deficient pancreatic cancer: targeting EZH2- mediated oncogenic traits [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B088.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde281aef7a4cfc6dbd17009895cff9b452a2fd" target='_blank'>
              Abstract B088: Epigenetic reprograming in ATM-deficient pancreatic cancer: targeting EZH2- mediated oncogenic traits
              </a>
            </td>
          <td>
            E. Roger, A. Härle, L. Perkhofer, E. Hessmann, A. Kleger, J. Gout
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Torsional stress in chromatin plays a fundamental role in cellular functions, influencing key processes such as transcription, replication, and chromatin organization. Transcription and other processes may generate and be regulated by torsional stress. In the genome, the interplay of these processes creates complicated patterns of both positive (+) and negative (-) torsion. However, a challenge in generating an accurate torsion map is determining the zero-torsion baseline signal, which is conflated with chromatin accessibility. Here, we introduce a high-resolution method based on the intercalator trimethylpsoralen (TMP) to address this challenge. We describe a method to establish the zero-torsion baseline while preserving the chromatin state of the genome. This approach enables both high-resolution mapping of accessibility and torsional stress in chromatin in the cell. Our analysis reveals transcription-generated torsional domains consistent with the twin-supercoiled-domain model of transcription and suggests a role for torsional stress in recruiting topoisomerases and in regulating 3D genome architecture via cohesin. This new method provides a potential path forward for using TMP to measure torsional stress in the genome without the confounding contribution of accessibility in chromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36069fa62e02b5fd242a41fc1c6b2402d69824c5" target='_blank'>
              High-Resolution Genome-Wide Mapping of Chromatin Accessibility and Torsional Stress
              </a>
            </td>
          <td>
            Porter M. Hall, L. A. Mayse, Lu Bai, Marcus B. Smolka, B. F. Pugh, Michelle D. Wang
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prolyl-4-hydroxylase subunit alpha3 (P4HA3) is a triple helical procollagen synthesis protein. The role of P4HA3 in cancer development is not well known and lacks comprehensive analyses among human cancers. This study aimed to investigate the relationship between P4HA3 expression and anti-tumor immunity and its prognostic value in pan-cancer. P4HA3 expression was analyzed from TIMER2.0, GTEx, GEPIA2.0 and TCGA databases. Genetic and DNA methylation alterations, survival analysis and proteins co-expression analysis of P4HA3 in cBio Cancer Genomics Portal, TCGA, GSCA and TIMER2.0. The correlation between P4HA3 expression and immune infiltration was analyzed by TIDE, XCELL, MCPCOUNTER, and EPIC. We performed EdU and transwell experiments to evaluate the influence of P4HA3 on the proliferation, migration and invasion abilities of different tumors. Patients derived xenograft (PDX) and subcutaneous transplantation models were utilized to explore the correlation between P4HA3 and immunotherapy response in triple-negative breast cancer (TNBC). Among 33 types of cancers, P4HA3 had generally different expression between different tumors, further analysis showed that the expression of P4HA3 was correlated with the cells infiltration of the tumor microenvironment (TME). The expression of P4HA3 was positively with the cell proliferation markers and epithelial-mesenchymal transition (EMT) markers. Moreover, P4HA3 deficiency inhibited the proliferation, migration and invasion abilities of tumor cells, and promoted anti-tumor immunotherapy of PD-1/PD-L1 inhibitor. This pan-cancer analysis of P4HA3 provides a comprehensive understanding of its oncogenic and prognosis role in different cancers, P4HA3 abnormal expression could be a useful biomarker for predicting the effectiveness of immunotherapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e4a1f2b5a8ac672064bf194617e5237e9701626" target='_blank'>
              A computational analysis of the oncogenic and anti-tumor immunity role of P4HA3 in human cancers.
              </a>
            </td>
          <td>
            Hongyan Huang, Fu Wei Zhang, Jie Yu, Yan-Hong Xiao, Di Zhu, XiaoLin Yi, XiaoHua Lin, Ming Jin, Hai Yun Jin, Yong Sheng Huang, Shu Wei Ren
          </td>
          <td>2024-11-06</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Homologous recombination (HR) deficiency upon BRCA2 loss arises from defects in the formation of RAD51 nucleoprotein filaments. Here, we demonstrate that loss of the anti-recombinase FIGNL1 retains RAD51 loading at DNA double-stranded breaks (DSBs) in BRCA2-deficient cells, leading to genome stability, HR proficiency, and viability of BRCA2-deficient mouse embryonic stem cells. Mechanistically, we directly show that strand invasion and subsequent HR defects upon BRCA2 loss primarily arises from the unrestricted removal of RAD51 from DSB sites by FIGNL1, rather than from defective RAD51 loading. Furthermore, we identify that the MMS22L-TONSL complex interacts with FIGNL1 and is critical for HR in BRCA2/FIGNL1 double-deficient cells. These findings identify a pathway for tightly regulating RAD51 activity to promote efficient HR, offering insights into mechanisms of chemoresistance in BRCA2-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cebc2a7a30b95e95e9cd143f7fd6f4463ef94048" target='_blank'>
              FIGNL1 inhibits homologous recombination in BRCA2 deficient cells by dissociating RAD51 filaments
              </a>
            </td>
          <td>
            Raviprasad Kuthethur, A. Acharya, S. Sengodan, Carmen Fonseca, Nupur Nagar, Safa Nasrin VZ, Oluwakemi Ibini, E. Manolika, Kelly de Koning, Stefan Braunshier, Julien Dessapt, Amélie Fradet-Turcotte, Joyce H.G. Lebbink, Roland Kanaar, K. Poluri, Shyam K. Sharan, Petr Cejka, A. Chaudhuri
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842eba76f2be7eab5006f9657126f976601f2a" target='_blank'>
              GUCA2A dysregulation as a promising biomarker for accurate diagnosis and prognosis of colorectal cancer
              </a>
            </td>
          <td>
            Pooya Jalali, Shahram Aliyari, Marziyeh Etesami, Mahsa Saeedi Niasar, Sahar Taher, K. Kavousi, E. Nazemalhosseini Mojarad, Zahra Salehi
          </td>
          <td>2024-11-01</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Epigenetic alterations in gene expression have been implicated in cancer development and tumor immune escape, with posttranslational histone or non-histone modifications representing attractive targets for disease surveillance and therapy. SET domain bifurcated 1 (SETDB1) is a histone lysine methyltransferase that reversibly catalyzes the di- and tri-methylation of histone 3 lysine 9 (H3K9) on euchromatin, inhibiting gene transcription within these regions and facilitating the switch from euchromatic to heterochromatic states. Emerging evidence suggests that SETDB1 amplification and aberrant activation are significantly associated with poor prognosis in hepatocellular carcinoma (HCC), and contribute to HCC development, immune escape, and immune checkpoint blockade (ICB) resistance. Here, we provide an updated overview of the cellular and molecular effects of SETDB1 activity in hepatocarcinogenesis and progression and focus on studies linking its function to immunotherapy for HCC, and present current challenges and future perspectives for targeting SETDB1 in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775e9dd157e3b9dd836833865f11fe90ae06aaa7" target='_blank'>
              The function of histone methyltransferase SETDB1 and its roles in liver cancer
              </a>
            </td>
          <td>
            Enxiang Zhang, Pingping He
          </td>
          <td>2024-11-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Long non-coding RNAs (lncRNAs) are key regulators of cellular processes that underpin cancer development and progression. DRAIC is a migration inhibitor that has been linked with lung adenocarcinoma progression; however, its mechanisms remain to be studied. Methods: Several bioinformatics tools were used to explore the role of DRAIC in lung adenocarcinoma (LUAD) and lung squamous cell carcinoma (LUSC). Results: Our bioinformatics analysis illustrates that patients with low expression of DRAIC have poor overall survival outcomes. In addition, the mRNA of SH3 domain-binding kinase 1 (SBK1) was downregulated in this cohort of patients. Mechanistic analysis showed that SBK1 is under the DRAIC competing endogenous RNAs network, potentially through sponging of miRNA-92a. Conclusions: Consistent dysregulation of the DRAIC-SBK1 axis was linked to poor survival outcome in both LUAD and LUSC, suggesting a tumour inhibitor role and providing potential for new diagnostics and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cfe6b5bf620754eaa8939a6be9e1d03d66c123" target='_blank'>
              Dysregulation of the DRAIC/SBK1 Axis Promotes Lung Cancer Progression
              </a>
            </td>
          <td>
            Rashed Alhammad, Milicia Allison, Fares Alhammad, C. Anene
          </td>
          <td>2024-10-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="To better understand cervical cancer progression, we analyzed RNA from 262 biopsies from women referred for colposcopy We determined HPV type and analyzed the expression of 51 genes. HPV31 was significantly more prevalent in precancer than stage 1 cancer and invasive cancer (p < 0.0001) and HPV16 increased in invasive disease (p < 0.0001). CCNE1, MELTF, and ULBP2 were significantly increased in HPV16-positive compared to HPV31 precancers while NECTIN2 and HLA-E expression decreased. Markers of the innate immune system, DNA repair genes, and cell cycle genes are significantly increased during cancer progression (p = 0.0001). In contrast, the TP53 and RB1 tumor suppressor gene expression is significantly decreased in cancer cells. TheT cell markers CD28 and FLT3LG expression decreased in cancer while FOXP3, IDO1, and ULBP2 expression increased. There is a significantly higher survival rate in individuals with increased expression of CD28 (p = 0.0005), FOXP3 (p = 0.0002), IDO1 (p = 0.038), FLT3LG (p = 0.026), APOBEC3B (p = 0.0011), and RUNX3 (p = 0.019), and a significantly lower survival rate in individuals with increased expression of ULBP2 (p = 0.035). These results will help us understand the molecular factors influencing the progression of cervical precancer to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42c89296b981257a0aa9870c2f9a33229ea5420f" target='_blank'>
              Analysis of the progression of cervical cancer in Guatemala- from pre-malignancy to invasive disease
              </a>
            </td>
          <td>
            Emma Robinson, Isabel Rodriguez, V. Argueta, Yi Xie, Hong Lou, Rose Milano, Hyo Jung Lee, L. Burdett, Sambit Mishra, Meredith Yeager, L. Mirabello, Michael Dean, R. Orozco
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Deciphering shared features between patients through unsupervised analyses of tumor single-cell transcriptomes is hindered by the predominant clustering of malignant cells based on the patient’s tumor of origin. In contrast, cancer-associated non-malignant cells within tumors cluster according to their somatic cell type (e.g. macrophage, fibroblast, astrocyte), regardless of the individual patient. We investigated the origin of this contrasting clustering behavior using computational analyses and novel data sampling techniques across 160 tumors representing 9 distinct cancer types. This led to the unexpected disclosing of tumor-specific malignant cell types. We demonstrate that tumor-driven malignant cell clustering is independent from technical or computational biases, nor is it reducible to defined gene sets with tumor-specific expression across all examined cancers. On the opposite, we unveil redundant information dispersed across the transcriptome that encodes a unique identity shared by the malignant cells within each tumor. Additionally, we found a similar genomic encoding for the identity of normal cell types. Finally, we demonstrate that malignant cell identities are maintained across space and over time, as are the identities of normal somatic cells types. These findings suggest the establishment of a distinct type of malignant cells within each tumor, robustly and diffusively encoded across the entire transcribed genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb2c0940cbf36006f73fd3a210e9572d4abd3013" target='_blank'>
              A unique malignant cell type per patient tumor is encoded in each cancer cell transcriptome
              </a>
            </td>
          <td>
            M. Saurty-Seerunghen, Elias A. El-Habr, Yossi Eliaz, Léa Bellenger, Christophe Antoniewski, Hervé Chneiweiss, M. Junier
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="The incidence of colorectal cancer (CRC) among individuals under age 50, or early-onset CRC (EOCRC), has been rising over the past few decades for unclear reasons, and the etiology of the disease remains largely unknown. Known genetic risk factors do not explain this increase, pointing to possible environmental and as-yet unidentified genetic contributors and their interactions. Previous research linked genetic variation on chromosome 6 to increased CRC risk. This region harbors multiple immune genes, including the gene encoding Major Histocompatibility Complex (MHC) class I polypeptide-related sequence A (MICA). MICA is a polygenic ligand for the Natural Killer Group 2D receptor (NKG2D), a receptor expressed on Natural Killer (NK) cells and other lymphocytes. Given that intra-tumoral NK cell infiltration correlates with favorable CRC outcomes, we hypothesized that germline genetic variation in MICA could influence CRC risk. In a discovery set of 40,125 cases and controls, we show that the minor G allele at Chr6:31373718C>G (hg19) is associated with increased risk for CRC (odds ratio (OR) = 1.09, 95% confidence interval (CI) 1.04 - 1.15, p = 0.0009). The effect is stronger in EOCRC (OR = 1.26, 95% CI 1.08 - 1.44, p = 0.0023) than in those 50 and over (OR = 1.07, 95% CI 1.02 - 1.13; p = 0.012) (Ratio of ORs = 1.32, 95% CI 1.14 - 1.52, p = 0.0002). In an independent validation set of 77,983 cases and controls, the adjusted interaction by age -of-onset was significant at OR = 1.15 (95% CI 1.03 - 1.34, p = 0.0150) with a higher risk in EOCRC. Expression quantitative trait locus analysis in normal colonic epithelia showed that MICA RNA expression decreases linearly with each additional copy of the minor G allele (p = 3.345 x 10-18). Bulk RNA analysis of the tumor immune microenvironment revealed that tumors from patients with CG or GG genotypes have lower resting and activated NK cell infiltration as compared to tumors from patients with CC genotype. Multiplex immunofluorescence analysis demonstrated that patients with a G allele (i.e. CG or GG genotype, but not CC genotype) have a statistically significant decrease in the number of NK cells in tumor compared to adjacent normal colonic mucosa. Taken together, population-based epidemiologic, molecular, genetic, cellular and immunologic evidence demonstrate that MICA genotype is associated with increased risk of EOCRC and reduced number of NK cells in colorectal tumors, suggesting that patients with a G allele have altered NK cell-mediated immunosurveillance. These novel findings suggest that EOCRC may have a previously unrecognized innate immune-mediated etiology which merits further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5c740e57eda493d70527270b12cf7926d01624" target='_blank'>
              Genetic Variation and Regulation of MICA Alters Natural Killer Cell-Mediated Immunosurveillance in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            H. McGee, J. Bonner, Colt Egelston, Yubo Fu, Oscar Colunga Flores, S. Lindsey, Lawrence Shaktah, Ferran Moratalla-Navarro, Yasmin Kamal, Kevin K. Tsang, C. P. Walker, G. Idos, K. McDonnell, H. Rennert, Elizabeth L Barry, Hermann Brenner, Daniel D. Buchanan, P. Campbell, A. T. Chan, J. Chang-Claude, Jane C Figueiredo, M. Gago-Domínguez, M. Hoffmeister, Li Hsu, J. Huyghe, M. Jenkins, L. Le Marchand, Heinz-Josef Lenz, Li Li, A. Lindblom, Yun Ru Liu Ruby, B. Lynch, Christina C. Newton, K. Offit, Shuji Ogino, Rebeca Sanz Pamplona, Andrew J. Pellatt, Paul D. P. Pharoah, Amanda I. Phipps, Lorena Reynaga, Allyson S. Templeton, C. Um, Alicja Wolk, M. Woods, A. H. Wu, Yen Yun, Weishan Zheng, Terence M. Williams, David V Conti, Ulrike Peters, F. Lejbkowicz, J. Greenson, S. Schmit, W. Gauderman, Stanley R. Hamilton, Victor Moreno, Gadi Rennert, Stephen B. Gruber
          </td>
          <td>2024-09-26</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="High Grade Serous Ovarian Cancer (HG-SOC), the most prevalent and aggressive gynecological malignancy, is marked by ubiquitous loss of functional p53, largely due to point mutations that arise very early in carcinogenesis. These mutations often lead to p53 protein misfolding and subsequent aggregation, yet the alterations in intracellular p53 dynamics throughout ovarian cancer progression remain poorly understood. HG-SOC originates from the fallopian tube epithelium, with a well-documented stepwise progression beginning with early pre-malignant p53 signatures. These signatures represent largely normal cells that express and accumulate mutant p53, which then transform into benign serous tubal intraepithelial lesions (STIL), progress into late pre-malignant serous tubal intraepithelial carcinoma (STIC), and ultimately lead to HGSOC. Here, we show that the transition from folded, soluble to aggregated mutant p53 occurs during the malignant transformation of benign precursor lesions into HGSOC. We analyzed fallopian tube tissue collected from ten salpingo-oophorectomy cases and determined the proportion of cells carrying soluble versus mis-folded/mutant p53 through conformation-sensitive staining and quantification. Misfolded p53 protein, prone to aggregation, is present in STICs and HG-SOCs, but notably absent from preneoplastic lesions and surrounding healthy tissue. Overall, our results indicate that aggregation of mutant p53 is a structural defect that distinguishes preneoplastic early lesions from late premalignant and malignant ones, offering a potential treatment window for targeting p53 aggregation and halting ovarian cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e004df7ee189f194b9dc43a60b80d81862858bf" target='_blank'>
              Mutant p53 Misfolding and Aggregation Precedes Transformation into High-Grade Serous Ovarian Carcinoma
              </a>
            </td>
          <td>
            S. Sartini, Lexi Omholt, Neda A Moatamed, A. Soragni
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The spatial organization of chromatin within the nucleus plays a crucial role in gene expression and genome function. However, the quantitative relationship between this organization and nuclear biochemical processes remains under debate. In this study, we present a graph-based generative model, bioSBM, designed to capture long-range chromatin interaction patterns from Hi-C data and, importantly, simultaneously, link these patterns to biochemical features. Applying bioSBM to Hi-C maps of the GM12878 lymphoblastoid cell line, we identified a latent structure of chromatin interactions, revealing 12 distinct communities that strongly align with known biological annotations. Additionally, we infer a linear transformation that maps biochemical observables, such as histone marks, to the parameters of the generative graph model, enabling accurate genome-wide predictions of chromatin contact maps on out-of-sample data, both within the same cell line, and on the completely unseen HCT116 cell line under RAD21 depletion. These findings highlight bioSBM's potential as a powerful tool for elucidating the relationship between biochemistry and chromatin architecture and predicting long-range genome organization from independent biochemical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3b40ebcae85ab8524cde3cbb7cf148a03d0c300" target='_blank'>
              bioSBM: a random graph model to integrate epigenomic data in chromatin structure prediction
              </a>
            </td>
          <td>
            Alex Chen Yi Zhang, Angelo Rosa, Guido Sanguinetti
          </td>
          <td>2024-09-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. Previous studies have identified m6A as a key player in glioma pathogenesis, promoting cell stemness and tumor cell proliferation. However, there is also evidence of a potential anti-cancer role of m6A. Here, we present a comprehensive overview of the m6A landscape in a glioma cell line model using Nanopore direct RNA sequencing platform. The study investigates functional and biological impact of targeted knockdown (KD) of key m6A regulators (reader: IGF2BP2, writer: METTL3, eraser: ALKBH5). Transcriptome wide mapping of m6A demonstrated enrichment in CDS and 3’UTR regions. Differential patterns of non-DRACH motifs were observed across the KD conditions. Our analysis identified 716 common m6A sites across all conditions and clusters of KD specific modifications enriched for distinct functional pathways. IGF2BP2 KD was associated with pathways involving protein metabolism and apoptosis regulation. METTL3 KD was linked to cell cycle and stress response regulation and ALKBH5 KD impacted ribosome biogenesis and protein synthesis. Here, for the first time we also report detection of full-length RNA transcripts in extracellular vesicles (EVs) derived RNA from knockdown and control conditions. These findings underscore the unique contributions of each m6A regulator to glioma cell transcriptomics and suggest potential pathways for elucidating the overall role of m6A machinery in glioma pathogenesis. Important clinical applications include identification of novel diagnostic, prognostic, and therapeutic m6A RNA biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3136f121112542cac680afcce8f3495635fccbb" target='_blank'>
              Exploring N6-Methyladenosine and long transcripts in glioma cells and extracellular vesicles
              </a>
            </td>
          <td>
            S. Batool, Ana K Escobedo, Hanna Lee, Prerna Khanna, Kase D. Haas, Tiffaney Hsia, Bob S Carter, L. Balaj
          </td>
          <td>2024-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9291404ff76427f0917fd53c4b67029ebf03d548" target='_blank'>
              Role of non-coding RNA in lineage plasticity of prostate cancer.
              </a>
            </td>
          <td>
            Wenhui Tan, Changkai Xiao, Min Ma, Youhan Cao, Zhenguo Huang, Xiaolan Wang, Ran Kang, Zhenfa Li, Ermao Li
          </td>
          <td>2024-11-04</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d829dac90ae6e0bee2354e14ecf9cb6f0b45ddf8" target='_blank'>
              Polyclonal-to-monoclonal transition in colorectal precancerous evolution.
              </a>
            </td>
          <td>
            Zhaolian Lu, Shanlan Mo, Duo Xie, Xiangwei Zhai, Shanjun Deng, Kantian Zhou, Kun Wang, Xueling Kang, Hao Zhang, Juanzhen Tong, Liangzhen Hou, Huijuan Hu, Xuefei Li, Da Zhou, L. T. Lee, Li Liu, Yaxi Zhu, Jing Yu, Ping Lan, Jiguang Wang, Zhen He, Xionglei He, Zheng Hu
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Background: Gliomas, the most prevalent type of primary brain tumor, stand out as one of the most aggressive and lethal types of human cancer. Methods & Results: To uncover potential prognostic markers, we employed the weighted correlation network analysis (WGCNA) on the Chinese Glioma Genome Atlas (CGGA) 693 dataset to reveal four modules significantly associated with glioma clinical traits, primarily involved in immune function, cell cycle regulation, and ribosome biogenesis. Using the least absolute shrinkage and selection operator (LASSO) regression algorithm, we identified 11 key genes and developed a prognostic risk score model, which exhibits precise prognostic prediction in the CGGA 325 dataset. More importantly, we also validated the model in 12 glioma patients with overall survival (OS) ranging from 4 to 132 months using mRNA sequencing and immunohistochemical analysis. The analysis of immune infiltration revealed that patients with high-risk scores exhibit a heightened immune infiltration, particularly immune suppression cells, along with increased expression of immune checkpoints. Furthermore, we explored potentially effective drugs targeting 11 key genes for gliomas using the library of integrated network-based cellular signatures (LINCS) L1000 database, identifying that in vitro, both torin-1 and clofarabine exhibit promising anti-glioma activity and inhibitory effect on the cell cycle, a significant pathway enriched in the identified glioma modules. Conclusions: In conclusion, our study provides valuable insights into molecular mechanisms and identifying potential therapeutic targets for gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d22cc28dbea3a944dde294d9243d2a693f09d67" target='_blank'>
              Identification of Key Immune and Cell Cycle Modules and Prognostic Genes for Glioma Patients through Transcriptome Analysis
              </a>
            </td>
          <td>
            Kaimin Guo, Jinna Yang, Ruonan Jiang, Xiaxia Ren, Peng Liu, Wenjia Wang, Shuiping Zhou, Xiaoguang Wang, Li Ma, Yunhui Hu
          </td>
          <td>2024-09-28</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Centromeres are chromosomal regions historically understudied with sequencing technologies due to their repetitive nature and short-read mapping limitations. However, recent improvements in long-read sequencing allowed for the investigation of complex regions of the genome at the sequence and epigenetic levels. Here, we present Centromere Dip Region (CDR)-Finder: a tool to identify regions of hypomethylation within the centromeres of high-quality, contiguous genome assemblies. These regions are typically associated with a unique type of chromatin containing the histone H3 variant CENP-A, which marks the location of the kinetochore. CDR-Finder identifies the CDRs in large and short centromeres and generates a BED file indicating the location of the CDRs within the centromere. It also outputs a plot for visualization, validation, and downstream analysis. CDR-Finder is available at https://github.com/EichlerLab/CDR-Finder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ea2acfbe88409c0df7154515c53c47ebf174b6" target='_blank'>
              Identification and annotation of centromeric hypomethylated regions with Centromere Dip Region (CDR)-Finder
              </a>
            </td>
          <td>
            F. Mastrorosa, Keisuke K. Oshima, Allison N. Rozanski, William T. Harvey, Evan E. Eichler, Glennis A. Logsdon
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c75249f54c500f783739b3eb3c7b22b00e436d" target='_blank'>
              Azacytidine treatment affects the methylation pattern of genomic and cell-free DNA in uveal melanoma cell lines
              </a>
            </td>
          <td>
            Sarah Tadhg Ferrier, Mingyang Li, J. Burnier
          </td>
          <td>2024-10-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The role of tertiary lymphoid structures (TLS) in oncology is gaining interest, particularly in colorectal carcinoma, yet a thorough analysis remains elusive. This study pioneered a novel TLS quantification system for prognostic and therapeutic response prediction in colorectal carcinoma, alongside a comprehensive depiction of the TLS landscape. Utilizing single-cell sequencing, we established a TLS score within the Tumor Immune Microenvironment (TIME). Analysis of tertiary lymphoid structure-related genes (TLSRGs) in 1,184 patients with colon adenocarcinoma/rectum adenocarcinoma (COADREAD) from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases led to the identification of two distinct molecular subtypes. Differentially expressed genes (DEGs) further segregated these patients into gene subtypes. A TLS score was formulated using gene set variation analysis (GSVA) and its efficacy in predicting immunotherapy outcomes was validated in two independent cohorts. High-scoring patients exhibited a ‘hot’ immune phenotype, correlating with enhanced immunotherapy efficacy. Key genes in our model, including C5AR1, APOE, CYR1P1, and SPP1, were implicated in COADREAD cell proliferation, invasion, and PD-L1 expression. These insights offer a novel approach to colorectal carcinoma treatment, emphasizing TLS targeting as a potential anti-tumor strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254e5c3894e09ee3e8e46fe24980a38edb7262c2" target='_blank'>
              Prognostic and therapeutic potential of gene profiles related to tertiary lymphoid structures in colorectal cancer
              </a>
            </td>
          <td>
            Jinglu Yu, Yabin Gong, Xiaowei Huang, Yufang Bao
          </td>
          <td>2024-10-31</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd2329352934bc5310fa61340f1798aa2a300f6" target='_blank'>
              Super-silencer perturbation by EZH2 and REST inhibition leads to large loss of chromatin interactions and reduction in cancer growth.
              </a>
            </td>
          <td>
            Ying Zhang, Kaijing Chen, Seng Chuan Tang, Yichao Cai, A. Nambu, Y. See, Chaoyu Fu, Anandhkumar Raju, Benjamin Lebeau, Zixun Ling, J. J. Chan, Yvonne Tay, Marek Mutwil, Manikandan Lakshmanan, G. Tucker-Kellogg, W. Chng, D. Tenen, Motomi Osato, Vinay Tergaonkar, M. Fullwood
          </td>
          <td>2024-09-20</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Our understanding of neoadjuvant treatment with microtubule inhibitors (MTIs) for triple negative breast cancer (TNBC) remains limited. To advance our understanding of the role of breast cancer driver genes' mutational status with pathological complete response (pCR; ypT0/isypN0) prediction and to identify distinct gene sets for MTIs like eribulin and paclitaxel, we carried out targeted genomic (n = 50) and whole transcriptomic profiling (n = 64) of TNBC tumor samples from the Japan Breast Cancer Research Group 22 (JBCRG-22) clinical trial. Lower PIK3CA, PTEN, and HRAS mutations were found in homologous recombination deficiency (HRD)-high (HRD score ≥ 42) tumors with higher pCR rates. When HRD-high tumors were stratified by tumor BRCA mutation status, the pCR rates in BRCA2-mutated tumors were higher (83% vs. 36%). Transcriptomic profiling of TP53-positive tumors identified downregulation of FGFR2 (false discovery rate p value = 2.07e-7), which was also the only common gene between HRD-high and -low tumors with pCR/quasi-pCR treated with paclitaxel and eribulin combined with carboplatin, respectively. Differential enrichment analysis of the HRD-high group posttreatment tumors revealed significant correlation (p = 0.006) of the glycan degradation pathway. FGFR2 expression and the differentially enriched pathways play a role in the response and resistance to MTIs containing carboplatin treatment in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9036482597be2dd99c4d3d465b78b62d6cfa1a9" target='_blank'>
              Genomic and transcriptomic profiling of pre- and postneoadjuvant chemotherapy triple negative breast cancer tumors.
              </a>
            </td>
          <td>
            T. Nishimura, Ravindranath M Velaga, Norikazu Masuda, K. Kawaguchi, S. Kawaguchi, Masahiro Takada, Y. Maeshima, Sunao Tanaka, Y. Kikawa, Takayuki Kadoya, Hiroko Bando, Rikiya Nakamura, Y. Yamamoto, T. Ueno, H. Yasojima, Hiroshi Ishiguro, S. Morita, Shinji Ohno, Hironori Haga, Fumihiko Matsuda, Seishi Ogawa, M. Toi
          </td>
          <td>2024-10-07</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This study aimed to address new molecular events occurring in precancerous stages of colorectal cancer (CRC) by integrated analysis of gene expression data and DNA methylation profiles. Whole-transcriptome sequencing analysis was performed on 16 fresh frozen colorectal adenoma and matched mucosa specimens along with validation of candidates in an independent cohort of 20 fresh frozen paired adenoma and adjacent mucosa tissues as well as eight independent public datasets (335 cases). Genome-wide methylation profiles were determined for 5 adenoma pairs and confirmed by pyrosequencing on 20 tissue pairs used for validation. Functional analysis was performed in vitro and in vivo using the inflammation-associated azoxymethane/dextran sodium sulfate (AOM/DSS) and the sporadic colorectal carcinogenesis (six AOM injections) mouse models as well as ApcMin/+ mice. Candidates were investigated by immunohistochemical staining of human adenomas and early-stage (pT1) CRC tumors. A total of 1,917 differentially expressed genes and 148,191 differentially methylated CpG sites were detected in adenomas compared with adjacent mucosa samples. Based on the transcriptome data and relevance to CRC, TACSTD2, MMP7, MMP1, CLDN2, CLDN1, and ETV4, were selected for further validation. TACSTD2 promoter hypomethylation in adenoma tissues was validated in 20 additional tissue pairs and corresponded with increased TACSTD2 expression. TACSTD2 was also overexpressed in an in vitro transformation model of human colonic epithelial cells. The TACSTD2 protein TROP2 was elevated in human adenomas, pT1 tumors, and in murine adenomas, while it was absent in the unaffected adjacent mucosa. TROP2 overexpression might trigger the development of precancerous lesions and could help identify early transformation foci in colon biopsies. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf2c25521e6c5d07fbdce38e2d34466f87e5b71" target='_blank'>
              TACSTD2 expression marks the early transition to colon adenomas
              </a>
            </td>
          <td>
            Anna Siskova, K. Huebner, Giulio Ferrero, A. Gehring, Chuanpit Hampel, A. Naccarati, Barbara Pardini, S. Tarallo, Carol-Immanuel Geppert, Jan Prochazka, J. Tureckova, J. Jungwirth, T. Hucl, J. Kral, L. Vodickova, K. Erlenbach-Wuensch, A. Hartmann, Katerina Honkova, R. Matej, P. Vodicka, Veronika Vymetálková, R. Schneider-Stock
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187812e12825019ab421661fdd39125bff16e32a" target='_blank'>
              Potential role of heteroplasmic mitochondrial DNA mutations in modulating the subtype-specific adaptation of oral squamous cell carcinoma to cisplatin therapy
              </a>
            </td>
          <td>
            Amnani Aminuddin, Pei Yuen Ng, Chee Onn Leong, S. Makpol, E. Chua
          </td>
          <td>2024-10-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="INTRODUCTION
In chromatin nucleosomes, the presence-instead of canonical histone H3-of its variant, CENH3 (in plants), is considered the most reliable marker of the location of centromeres. In this study, we investigated effects of distant hybridization and maternal cytoplasm on centromere size in alloploid hybrids between wheat and rye as compared to their parental forms.


METHODS
Centromere sizes were measured using 2D images of CENH3 fluorescent signals on interphase nuclei obtained from parental forms and a triticale hybrid (genomic formula AABBBRR) in which the maternal form is wheat and secalotriticum hybrids (genomic formula RRAABBB) in which the maternal form is rye. For measurements, we selected the largest spherical nuclei with large nucleoli in the late G2 phase, in which most of the loading of CENH3 into centromeric chromatin takes place.


RESULTS
When processing the results of measurement of centromere sizes in the hybrids, the obtained values were compared with those expected for the case of no change in centromere sizes in any of the parental sets of chromosomes. We found no significant differences between expected and measured values.


CONCLUSION
We believe that in the case of allopolyploid hybrids between wheat and rye, centromeres of chromosomes from the parental species retain the sizes formed during evolution. This conservatism may be promoted by the high similarity in the structure of the CENH3 molecules between these species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13c11924647b4d33fd28b8eee261c47cb717952" target='_blank'>
              The parental centromere sizes remain unaltered in allopolyploid wheat-rye hybrids.
              </a>
            </td>
          <td>
            E. V. Evtushenko, S. S. Gatzkaya, Petr I Stepochkin, A. Vershinin
          </td>
          <td>2024-10-01</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 The dynamic interplay between tumor cells and γδT cells within the tumor microenvironment (TME) significantly influences disease progression and immunotherapy outcome. Here, we delved into the modulation of γδT-cell activation by tumor cell ligands CD112 and CD155, which interact with the activating receptor DNAM-1 on γδT cells. Spatial and single cell RNA sequencing (scRNA-seq), as well as spatial metabolome analysis, from neuroblastoma (NB) revealed that the expression levels and localization of CD112 and CD155 varied across and within tumors, correlating with differentiation status, metabolic pathways, and ultimately disease prognosis and patient survival. Both in vivo tumor xenograft experiments and in vitro co-culture experiments demonstrated that a high CD112/CD155 expression ratio in tumors enhanced γδT-cell-mediated cytotoxicity, while a low-ratio fostered tumor resistance. Mechanistically, CD112 sustained DNAM-1-mediated γδT-cell activation, whereas CD155 downregulated DNAM-1 expression via TRIM21-mediated ubiquitin proteasomal degradation. By interacting with tumor cells differentially expressing CD112 and CD155, intratumoral γδT cells exhibited varying degrees of activation and DNAM-1 expression, representing three major functional subsets. This study underscores the complexity of tumor-immune crosstalk, offering insights into how tumor heterogeneity shapes the immune landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0f304618ec33893cbfa169baffb49b70570809" target='_blank'>
              Spatial and Single Cell Analyses Reveal Heterogeneity of DNAM-1 Receptor-Ligand Interactions that Instructs Intratumoral γδT-Cell Activity
              </a>
            </td>
          <td>
            Xiaolin Wang, Hui Wang, Zhengjing Lu, Xiangjun Liu, Wenjia Chai, Wei Wang, Jun Feng, Shen Yang, Wei Yang, Haiyan Cheng, Chenghao Chen, Shihan Zhang, N. Sun, Qiaoyin Liu, Qiliang Li, Wenqi Song, Fang Jin, Qi Zeng, Shengcai Wang, Yan Su, Huanmin Wang, Xinxin Ni, Jingang Gui
          </td>
          <td>2024-11-08</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Radiotherapy remains a cornerstone in cancer treatment, leveraging ionizing radiation to eradicate malignant cells. Its efficacy, however, is frequently challenged by the heterogeneous sensitivity of tumors and surrounding tissues to radiation. Therefore, understanding the molecular mechanisms underlying radiosensitivity is crucial for improving treatment outcomes. Among the myriad of molecular players involved, the tumor suppressor protein p53 stands out as a central regulator with significant implications for radiosensitivity. Known as the “guardian of the genome”, p53 plays a pivotal role in maintaining genomic stability and orchestrating cellular responses such as cell cycle arrest, DNA repair, apoptosis, and senescence in response to various stress signals, including radiation-induced DNA damage. Activation of p53 triggers the transcription of target genes involved in DNA repair pathways, such as p21, MDM2, and GADD45, facilitating the repair of radiation-induced DNA damage or the elimination of irreparably damaged cells. This, in turn, influences the overall radiosensitivity of tissues. Mutations in the TP53 gene, which encodes p53, are among the most frequent genetic alterations in human cancers. Loss or dysfunction of p53 can compromise the cellular response to radiation, leading to increased resistance to therapy and poorer clinical outcomes. Conversely, intact p53 function is associated with enhanced radiosensitivity due to its ability to promote cell cycle arrest and apoptosis in response to radiation-induced DNA damage. In conclusion, elucidating the molecular mechanisms by which p53 influences radiosensitivity is essential for advancing our understanding of the radiation response in cancer cells and developing more effective therapeutic approaches to cancer treatment. This review provides a comprehensive overview of the multifaceted role of p53 in modulating cellular responses to radiation, emphasizing its influence on radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edeb6b0914ac30942be5b714c5e52dd1ba564b4d" target='_blank'>
              Exploring the Role of p53 in Radiosensitivity: A Key Player in Cancer Therapy
              </a>
            </td>
          <td>
            Tusher- Al-Arafat, Aihong Mao, T. Katsube, Bing Wang
          </td>
          <td>2024-10-24</td>
          <td>Radiation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0d10a131e1ec85b605aa6456ac30c2943dbeb4" target='_blank'>
              Spatial immunogenomic patterns associated with lymph node metastasis in lung adenocarcinoma
              </a>
            </td>
          <td>
            Fanjie Meng, Hao Li, Ruoyi Jin, Airong Yang, Hao Luo, Xiao Li, Peiyu Wang, Yaxing Zhao, O. Chervova, Kaicheng Tang, Sida Cheng, Bin Hu, Yun Li, Jianpeng Sheng, Fan Yang, David Carbone, Kezhong Chen, Jun Wang
          </td>
          <td>2024-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a liver cancer of adolescents and young adults characterized by fusions of the genes encoding the protein kinase A catalytic subunit, PRKACA, and heat shock protein, DNAJB1. The chimeric DNAJB1-PRKACA protein has increased kinase activity and is essential for FLC xenograft growth. Here, we explore the critical oncogenic pathways controlled by DNAJB1-PRKACA using patient-derived FLC models, engineered systems, and patient samples. We show that a core function of DNAJB1-PRKACA is the phosphorylation and inactivation of Salt-inducible kinases (SIKs). This leads to deregulation of the CRTC2 transcriptional co-activator and p300 acetyltransferase, resulting in transcriptional reprogramming and increased global histone acetylation, driving malignant growth. Our studies establish a central oncogenic mechanism of DNAJB1-PRKACA and suggest the potential of targeting CRTC2/p300 in FLC. Notably, these findings link this rare cancer's signature fusion oncoprotein to more common cancer gene alterations involving STK11 and GNAS, which also function via SIK suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77fde5ecc4bc9340e646816c229c18dfe2fdb5e" target='_blank'>
              DNAJB1-PRKACA fusion drives fibrolamellar liver cancer through impaired SIK signaling and CRTC2/p300-mediated transcriptional reprogramming.
              </a>
            </td>
          <td>
            Ilaria Gritti, Jinkai Wan, Vajira Weeresekara, Joel M Vaz, Giuseppe Tarantino, Tenna Holgersen Bryde, Vindhya Vijay, Ashwin V. Kammula, Prabhat Kattel, Songli Zhu, Phuong Vu, Marina Chan, Meng-Ju Wu, John D Gordan, K. Patra, Vanessa S Silveira, R. Manguso, Marc N. Wein, Christopher J Ott, Jun Qi, David Liu, Kei Sakamoto, Taranjit S. Gujral, N. Bardeesy
          </td>
          <td>2024-09-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Src-associated in mitosis 68 kDa protein (Sam68) is a protein encoded by the heteronuclear ribonucleoprotein particle K homology (KH) single domain-containing, RNA-binding, signal transduction-associated protein 1 (known as KHDRBS1) gene in humans. This protein contains binding sites for critical components in a variety of cellular processes, including the regulation of gene expression, RNA processing and cell signaling. Thus, Sam68 may play a role in a variety of diseases, including cancer. Sam68 has been widely demonstrated to participate in tumor cell proliferation, progression and metastasis to be involved in the regulation of cancer stem cell self-renewal. Based on the body of evidence available, Sam68 emerges as a promising target for this disease. The objectives of the present included summarizing the role of Sam68 in cancer murine models and cancer patients, unraveling the molecular mechanisms underlying its oncogenic potential and discussing the effectiveness of antitumor agents in reducing the malignant effects of Sam68 during tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca262590d67f00beb61f7cfdc140e58c3ea69281" target='_blank'>
              Role of Sam68 in different types of cancer (Review)
              </a>
            </td>
          <td>
            C. Jiménez-Cortegana, F. Sánchez-Jiménez, L. de la Cruz‑Merino, Víctor Sánchez-Margalet
          </td>
          <td>2024-10-22</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Asymmetric partition of fate determinants during cell division is a hallmark of cell differentiation. Recent works suggested that such a mechanism is hijacked by cancer cells to increase both their phenotypic heterogeneity and plasticity and in turn their fitness. To quantify fluctuations in the partitioning of cellular elements, imaging-based approaches are used, whose accuracy is limited by the difficulty of detecting cell divisions. Our work addresses this gap proposing a general method based on high-throughput flow cytometry measurements coupled with a theoretical frameworks. We applied our method to a panel of both normal and cancerous human colon cells, showing that different kinds of colon adenocarcinoma cells display very distinct extents of fluctuations in their cytoplasm partition, explained by an asymmetric division of their size. To test the accuracy of our population-level protocol, we directly measure the inherited fractions of cellular elements from extensive time-lapses of live-cell laser scanning microscopy, finding excellent agreement across the cell types. Ultimately, our flow cytometry-based method promise to be accurate and easily applicable to a wide range of biological systems where the quantification of partition fluctuations would help accounting for the observed phenotypic heterogeneity and plasticity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89534ca95bbe7d2c00c135f356aa0991c7b534e4" target='_blank'>
              Robust assessment of asymmetric division in colon cancer cells
              </a>
            </td>
          <td>
            Domenico Caudo, Chiara Giannattasio, Simone Scalise, V. Turris, Fabio Giavazzi, G. Ruocco, Giorgio Gosti, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-09-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Meiotic crossover, i.e., the reciprocal exchange of chromosome fragments during meiosis, is a key driver of genetic diversity. Crossover is initiated by the formation of programmed DNA double-strand breaks (DSBs). While the role of ATAXIA-TELANGIECTASIA AND RAD3-RELATED (ATR) kinase in DNA damage signaling is well-known, its impact on crossover formation remains understudied. Here, using measurements of recombination at chromosomal intervals and genome-wide crossover mapping, we showed that ATR inactivation in Arabidopsis (Arabidopsis thaliana) leads to dramatic crossover redistribution, with an increase in crossover frequency in chromosome arms and a decrease in pericentromeres. These global changes in crossover placement were not caused by alterations in DSB numbers, which we demonstrated by analyzing phosphorylated H2A.X foci in zygonema. Using the seed-typing technique, we found that hotspot usage remains mainly unchanged in atr mutants compared to wild-type individuals. Moreover, atr showed no change in the number of crossovers caused by two independent pathways, which implies no effect on crossover pathway choice. Analyses of genetic interaction indicate that while the effects of atr are independent of MMS AND UV SENSITIVE81 (MUS81), ZIPPER1 (ZYP1), FANCONI ANEMIA COMPLEMENTATION GROUP M (FANCM) and D2 (FANCD2), the underlying mechanism may be similar between ATR and FANCD2. This study extends our understanding of ATR's role in meiosis, uncovering functions in regulating crossover distribution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacde0c3932254585c19d65235879a141c6f4b4" target='_blank'>
              The kinase ATR controls meiotic crossover distribution at the genome scale in Arabidopsis.
              </a>
            </td>
          <td>
            Longfei Zhu, J. Dluzewska, N. Fernández-Jiménez, Rajeev Ranjan, Alexandre Pelé, Wojciech Dzięgielewski, Maja Szymańska-Lejman, Karolina Hus, Julia Górna, M. Pradillo, Piotr A. Ziolkowski
          </td>
          <td>2024-10-29</td>
          <td>The Plant cell</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Ewing sarcoma, an aggressive pediatric cancer, is driven by the EWS::FLI1 fusion protein, which disrupts gene expression by hijacking the BAF chromatin remodeling complex. Central to this mechanism is the formation of biomolecular condensates, mediated by the prion-like domains (PrLDs) of EWS and ARID1A, a core BAF subunit. ARID1A serves as a critical interface between EWS::FLI1 and the BAF complex, with its condensate-forming ability essential for the aberrant gene expression that drives tumor growth. The loss of condensate-competent ARID1A significantly impairs tumor progression, identifying it as a potential therapeutic target. However, targeting condensate formation is challenging due to the transient nature of the interactions involved, complicating the development of effective inhibitors. This work underscores the importance of further investigation into therapeutic strategies aimed at disrupting condensate formation in Ewing sarcoma and other related malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d97173e3a4e6f56ec219f7a894c5bb5f707ced" target='_blank'>
              Hijacking the BAF complex: the mechanistic interplay of ARID1A and EWS::FLI1 in Ewing sarcoma.
              </a>
            </td>
          <td>
            Erich J Sohn, David S. Libich
          </td>
          <td>2024-09-29</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer is associated with high morbidity and mortality, which are closely influenced by protein post-translational modifications (PTMs). Lysine crotonylation (Kcr) serves as a newly identified PTM type that plays a role in various biological processes; however, its involvement in breast cancer progression remains unclear. Minichromosome maintenance 6 (MCM6) is a critical component of DNA replication and has been previous confirmed to exhibit a significant role in tumorigenesis. Despite this, a comprehensive analysis of MCM6, particularly regarding its modifications in breast cancer is lacking. In this study, we found MCM6 is upregulated in breast invasive carcinoma (BRCA) and is associated with poorer overall survival by regulating the DNA damage repair mechanisms. Furthermore, MCM6-knockdown resulted in decreased cell proliferation and inhibited the DNA replication, leading to DNA replication stress and sustained DNA damage, thereby enhancing the chemotherapeutic sensitivity of breast cancer. Additionally, SIRT7-mediated crotonylation of MCM6 at K599 (MCM6-K599cr) was significantly upregulated in response to DNA replication stress, primarily due to the disassemebly of the MCM2-7 complex and regulated by RNF8-mediated ubiquitination. Concurrently, kaempferol, which acts as a regulator of SIRT7, was found to enhance the Kcr level of MCM6, reducing tumour weight, particular when combined with paclitaxel, highlighting its potential chemotherapeutic target for BRCA therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc331419a4f2d077914bda931ebc6dc0480b9e3e" target='_blank'>
              Crotonylation of MCM6 enhances chemotherapeutics sensitivity of breast cancer via inducing DNA replication stress.
              </a>
            </td>
          <td>
            Haoyun Song, Zhao Guo, Kun Xie, Xiangwen Liu, Xu Yang, Rong Shen, Degui Wang
          </td>
          <td>2024-10-30</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The kinetochore complex, an important protein assembly situated on the centromere, plays a pivotal role in chromosome segregation during cell division. Like in animals and fungi, the plant kinetochore complex is important for maintaining chromosome stability, regulating microtubule attachment, executing error correction mechanisms, and participating in signaling pathways to ensure accurate chromosome segregation. This review summarizes the composition, function, and regulation of the plant kinetochore complex, emphasizing the interactions of kinetochore proteins with centromeric DNAs (cenDNAs) and RNAs (cenRNAs). Additionally, the applications of the centromeric histone H3 variant (the core kinetochore protein CENH3, first identified as CENP-A in mammals) in the generation of ploidy-variable plants and synthesis of plant artificial chromosomes (PACs) are discussed. The review serves as a comprehensive roadmap for researchers delving into plant kinetochore exploration, highlighting the potential of kinetochore proteins in driving technological innovations in synthetic genomics and plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f840a6e5c8eb7572f7b8a202d1475f565b0be6f0" target='_blank'>
              Plant kinetochore complex: composition, function, and regulation
              </a>
            </td>
          <td>
            Yuqian Xie, Mingliang Wang, Beixin Mo, Chao Liang
          </td>
          <td>2024-10-10</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Hematologic abnormalities are the most common symptoms of Shwachman-Diamond syndrome (SDS). The causative gene for SDS is the Shwachman-Bodian-Diamond syndrome (SBDS) gene; however, the function of SBDS and pathogenesis of each condition in SDS are largely unknown. SBDS is known to be localized at mitotic spindles and stabilizes microtubules. Previously, we demonstrated that SBDS is ubiquitinated and subsequently degraded in the mitotic phase, thereby accelerating mitotic progression. In this study, we examined mitosis in a myeloid cell model of SDS (SDS cells). 4',6-Diamidino-2-phenylindole (DAPI)-stained chromosome observation and cell cycle analysis of nocodazole-synchronized cells revealed that the SDS cells have abnormally rapid mitosis. In addition, many lagging chromosomes and micronuclei were detected. Moreover, the phosphorylation of threonine tyrosine kinase, the crucial kinase of the spindle assembly checkpoint (SAC), was suppressed. Chromosomal instability caused by SAC dysfunction may cause a variety of clinical conditions, including hematologic tumors in patients with SDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21f843bea77ce16bbf79a7b0ddb3549e10b17a0" target='_blank'>
              Mitotic abnormalities and spindle assembly checkpoint inactivation in a cell model of Shwachman-Diamond syndrome with mutations in the Shwachman-Bodian-Diamond syndrome gene, 258+2T > C.
              </a>
            </td>
          <td>
            Yukihiro Sera, Tsuneo Imanaka, Yusuke Iguchi, Masafumi Yamaguchi
          </td>
          <td>2024-10-26</td>
          <td>Drug discoveries & therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The role of Epiplakin1 (EPPK1) in colon adenocarcinoma (COAD) was analyzed through a comprehensive evaluation of its expression, methylation, genetic mutations, and prognostic implications. A significant up-regulation of EPPK1 expression in COAD malignant cells compared to normal control samples was observed using data from the UALCAN database. EPPK1 expression was found to be elevated across different cancer development stages, racial groups, genders, and age groups, emphasizing its crucial role in cancer proliferation. Validation of EPPK1 expression through the GEPIA2.0 dataset further confirmed its overexpression in COAD when compared to normal samples. The analysis revealed dysregulation across all four stages of cancer development, with the highest expression in stage IV and the lowest in stage II. Additionally, promoter methylation analysis demonstrated a fundamental relationship between COAD samples and normal controls, revealing significant methylation patterns across different clinical parameters, including cancer stages, race, gender, and age. Overall survival (OS) and disease-free survival (DFS) analysis using the KM plotter showed a strong association between high EPPK1 expression and worse survival outcomes in COAD patients. Conversely, low EPPK1 expression was linked to better OS and DFS. Genetic mutation analysis using cBioPortal identified minimal EPPK1 mutations in COAD, predominantly truncating and missense mutations, highlighting their relevance to EPPK1 dysregulation in COAD. These findings underscore the important role of EPPK1 in COAD development and proliferation, suggesting its potential as a therapeutic target and prognostic marker. Further exploration of EPPK1’s molecular mechanisms and its involvement in the COAD microenvironment may reveal new pathways for targeted treatments and precision medicine strategies against this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1746eb2037e9f374ca2ed51f59d8d79d309f8805" target='_blank'>
              Deciphering the Role of EPPK1 in the Development of Colon Adenocarcinoma by Integrated Bioinformatics Approach
              </a>
            </td>
          <td>
            Muhammad Akram, Muhammad Umair Abid
          </td>
          <td>2024-10-24</td>
          <td>Journal of Clinical and Nursing Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Complete expression loss of S-methyl-5′-thioadenosine phosphorylase (MTAP) is caused by homozygous 9p21 deletion and results in a critical vulnerability of cancer cells towards drugs targeting multiple different pathways. MTAP deficiency is common in ductal adenocarcinoma of the pancreas, but data on its intratumoral heterogeneity - a potential obstacle for targeted therapies - are lacking. To learn more on the role and potential heterogeneity of MTAP deficiency in pancreatic adenocarcinoma, 236 primary tumors were analyzed in a tissue microarray (TMA) format by immunohistochemistry (IHC) and fluorescence in situ hybridization (FISH) for MTAP expression and 9p21 deletions. The cohort included 236 cancers from which 6 samples were taken from as many as possible different tumor containing primary tumor blocks and 3 samples were added from as many as possible lymph node metastases to construct a “heterogeneity TMA”. All available tumor containing blocks (average 4) from 6 selected cancers were also analyzed by IHC and FISH on whole sections. Complete absence of MTAP staining was seen in 350 (38.6%) of 907 interpretable TMA samples. 284 (98.6%) of 288 samples with a complete MTAP expression loss and FISH data had a homozygous 9p21 deletion while there were only two tumor spots with homozygous deletions within 557 samples with retained MTAP expression (99.6% concordance). On a patient level, an MTAP deficiency was found in 64 (41.0%) of 156 tumors for which at least 3 different samples were interpretable on the TMAs. 62 (96.9%) of these patients showed an MTAP loss in all interpretable samples (homogeneous MTAP deficiency). The TMA data were confirmed on whole sections in 5 of 6 patients. In the discrepant case a focal MTAP loss was identified in a tumor considered MTAP wild type based on the TMA results. It is concluded that MTAP expression loss is frequent and highly homogeneous in ductal adenocarcinomas of the pancreas. Given the high concordance rate between homozygous deletion and MTAP expression loss homozygous deletion may represent the only mechanism for MTAP deficiency in pancreatic cancer. MTAP IHC is an excellent tool for identification of MTPA deficient cases. MTAP IHC may also have considerable diagnostic value as even low-grade cancer glands can be easily identified in small biopsies from MTAP deficient cases. Considering also their aggressive clinical behavior, pancreatic adenocarcinomas may represent an ideal cancer type for studying new drugs targeting MTAP deficient cancer cells in clinical trials.
 Citation Format: Natalia Gorbokon, Katharina Teljuk, Frank Jacobsen, Till Krech, Till Clauditz, Stefan Steurer, Ronald Simon, Guido Sauter, Sarah Minner, Lukas Bubendorf, Thilo Hackert, Nina Schraps, Faik G Uzunoglu, Martina Kluth. MTAP deficiency is frequent and mostly homogeneous in ductal adenocarcinomas of the pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ece68516216c0dd87294424662694645b72a9ed" target='_blank'>
              Abstract A028: MTAP deficiency is frequent and mostly homogeneous in ductal adenocarcinomas of the pancreas
              </a>
            </td>
          <td>
            N. Gorbokon, Katharina Teljuk, Frank Jacobsen, T. Krech, T. Clauditz, S. Steurer, R. Simon, G. Sauter, S. Minner, Lukas Bubendorf, T. Hackert, Nina Schraps, F. G. Uzunoglu, M. Kluth
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="ABSTRACT Reactive oxygen species (ROS) pose a lethal risk for all life forms by causing damage to cell processes, genome-wide DNA damage-driving mutation, replicative instability, and death. Thus, the development of mechanisms to resist or repair ROS-induced DNA damage is critical for the reliable replication of nucleic acids. DNA repair and protection mechanisms have been discovered in all forms of life. However, the vast array of microbes that may harbor novel repair or protection mechanisms, especially bacterial viruses, have not been adequately assessed. Here, we screened a microbial gene library composed primarily of phage open reading frames (ORFs) to uncover elements that overcome a DNA damage blockade. We report the discovery of one such protein, termed F21, which promotes bacterial survival by possibly repairing or protecting DNA in the face of ROS-induced DNA damage. IMPORTANCE Discovery of proteins that promote DNA damage repair and protection in the face of reactive oxygen species (ROS) is of vital importance. Our group is in possession of a unique microbial DNA library with which we can screen for undiscovered genes that encode novel proteins with DNA damage repair and protective functions. This library is composed of diverse DNA from a variety of sources, namely bacteriophages, which must be assessed for their novel functions. This work focuses on the discovery of DNA damage repair and protection, but the possibilities for discovery are endless, thus highlighting the significance of this work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6c7c4deca0a79aaa3478f278cc3fed0733ed18e" target='_blank'>
              Gene discovery from microbial gene libraries I: protection against reactive oxygen species-driven DNA damage
              </a>
            </td>
          <td>
            John C Taylor, Carmen Gu Liu, James D Chang, Brianna E Thompson, A. Maresso
          </td>
          <td>2024-09-16</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Genetic or epigenetic variations in regulatory enhancer elements increase susceptibility to a range of pathologies, including pancreas cancer. Pinpointing genes affecting pancreas cancer risk holds promise for early detection, prevention, and effective therapies. Despite recent advances, linking enhancer elements to target genes and predicting the transcriptional outcome of enhancer dysfunction remain significant challenges. Using 3D chromatin assays, we generated an extensive enhancer interaction dataset for the human pancreas, spanning more than 20 donors and five major cell types, including both the exocrine and endocrine compartments. We employed a network approach to parse chromatin interactions into enhancer-promoter tree models, facilitating quantitative, genome-wide analysis of enhancer connectivity. Using the tree models, we developed a machine learning algorithm capable of estimating the impact of enhancer perturbations on cell-type specific gene expression in the human pancreas. Complementing this computational approach, we streamlined an experimental platform using CRISPR-interference and RNA-FISH coupled with high-throughput imaging to quantitatively measure the effect size of enhancer perturbations in primary human pancreas cells. Our enhancer tree models enabled functional annotation of genetic variants associated with pancreas diseases, revealing novel target genes in specific cell types. For pancreas cancer (PDAC), our data suggests a stronger association of disease susceptibility variants with acinar cells, even though ductal cells historically have been the focus of PDAC. This discovery linked three potentially functional variants associated with PDAC risk to acinar cell enhancers, revealing novel target genes to further investigation. By integrating genomics with mechanistic findings into enhancer function, our work offers a framework for understanding genetic basis of pancreas cancer disease risk.
 Citation Format: Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, Jason Hoskins, Laufey Amundadottir, Efsun Arda. A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f97224f03d537588713402ad00fc72aae371ba05" target='_blank'>
              Abstract C058: A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions
              </a>
            </td>
          <td>
            Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, Jason W. Hoskins, L. Amundadottir, Efsun Arda
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>