<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-13 09:26:49 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>27</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>64</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>307</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>263</td>
          <td>157</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) can drive oncogene amplification, gene expression and intratumor heterogeneity, representing a major force in cancer initiation and progression. The phenomenon becomes even more intricate as distinct types of ecDNA present within a single cancer cell. While exciting as a new and significant observation across various cancer types, there is a lack of a general framework capturing the dynamics of multiple ecDNA types theoretically. Here, we present novel mathematical models investigating the proliferation and expansion of multiple ecDNA types in a growing cell population. By switching on and off a single parameter, we model different scenarios including ecDNA species with different oncogenes, genotypes with same oncogenes but different point mutations and phenotypes with identical genetic compositions but different functions. We analyse the fraction of ecDNA-positive and free cells as well as how the mean and variance of the copy number of cells carrying one or more ecDNA types change over time. Our results showed that switching does not play a role in the fraction and copy number distribution of total ecDNA-positive cells, if selection is identical among different ecDNA types. In addition, while cells with multiple ecDNA cannot be maintained in the scenario of ecDNA species without extra fitness advantages, they can persist and even dominate the ecDNA-positive population if switching is possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f328238baa1a06ddbda4cdafb65854f0db985125" target='_blank'>
              Population dynamics of multiple ecDNA types
              </a>
            </td>
          <td>
            Elisa Scanu, Benjamin Werner, Weini Huang
          </td>
          <td>2024-11-21</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc1dada885aaee19cff1e9a95787aee6cf49a1c" target='_blank'>
              Comprehensive profiling of extrachromosomal circular DNAs in colorectal cancer progression
              </a>
            </td>
          <td>
            Lu Lu, Mingjie Chen, Guicheng Zhang, Yujing Liu, Xiangyuan Xu, Zenghua Jiang, Yangxian Xu, Tao Liu, Fan Yang, Guang Ji, Hanchen Xu
          </td>
          <td>2024-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Independently of the pathways or circuits deregulated in cancer cells, these present altered transcription patterns, often also direct consequence of deregulation of transcription factors. In this sense, also the RNA Polymerase complexes responsible for transcription can be affected in cancers. We find that upregulations of RNA Polymerase II subunits, especially the largest ones, correlates with poor cancer patients’ outcome across a range of tumor types, presenting increased genome instability. Overexpressing the subunits RPB1, RPB3 and RPB4 in cells we find that these induce DNA damage. However, the mechanisms behind this increased genome instability are specific for each subunit, linked to the unique transcription alterations generated by the subunit overexpression. Importantly, we find significant overlap between the genes with more DNA damage in our cell line models and those more affected in cancers with subunit upregulation, indicating that upregulations could be responsible for some of the phenotypes present in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1c9c056673cff0cd8a0e04f5d4022f59e547e9" target='_blank'>
              RNA Polymerase II subunits overexpressions induce genome instability and deregulate transcription
              </a>
            </td>
          <td>
            Martina Mustè Sadurnì, Megan Louise Jones, Lucas Pavlou, Mohamed Radi, Smaragda Kompocholi, Paolo Passaretti, Agnieszka Gambus, Lea H. Gregersen, Marco Saponaro
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) can drive tumor initiation, progression and resistance in some of the most aggressive cancers and is emerging as a promising anti-cancer target. However, detection currently requires costly whole-genome sequencing (WGS) or labor-intensive cytogenetic or FISH imaging, limiting its application in routine clinical diagnosis. To overcome this, we developed ecPath (ecDNA from histopathology), a computational method for predicting ecDNA status from routinely available hematoxylin and eosin (H&E) images. ecPath implements a deep-learning method we call transcriptomics-guided learning, which utilizes both transcriptomics and H&E images during the training phase to enable successful ecDNA prediction from H&E images alone, a task not achievable with models trained on H&E images only. It is trained on more than 6,000 tumor whole-slide images from the TCGA cohort with the best performance in predicting ecDNA status in brain and stomach tumors (average AUC=0.78). ecPath revealed that ecDNA-positive tumors are enriched with pleomorphic, larger and high-density nuclei. Testing in an independent cohort, ecPath predicted ecDNA status of 985 pediatric brain tumor patients with an AUC of 0.72. Finally, we applied ecPath to identify ecDNA-positive tumors in the TCGA cohort for which no WGS data were available. Like WGS-based ecDNA-positive labels, the predicted ecDNA-positive status also identify poor prognoses for low grade glioma patients. These results demonstrate that ecPath enables the detection of ecDNA from routinely available H&E imaging alone and help nominate aggressive tumors with ecDNA to study and target it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ffb85a2375bcb28d5b3da1f1dc6239edbfdf5f" target='_blank'>
              ecPath detects ecDNA in tumors from histopathology images
              </a>
            </td>
          <td>
            Mudra Choudhury, Lihe Liu, Anamika Yadav, Owen S Chapman, Zahra Ahmadi, Raneen Younis, Chinmay Sharma, Navansh Goel, S. Sridhar, Rishaan Kenkre, Aditi Dutta, Shanqing Wang, E. Shulman, S. R. Dhruba, Danh-Tai Hoang, Kevin M. Tharp, Megan R Paul, Denise Malicki, K. Yip, E. Ruppin, Lukas Chavez, Sanju Sinha
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromatin architecture governs DNA accessibility and gene expression. Thus, any perturbations to chromatin can significantly alter gene expression programs and promote disease. Prior studies demonstrate that every amino acid in a histone is functionally significant, and that even a single amino acid substitution can drive specific cancers. We previously observed that naturally occurring H2B variants are dysregulated during the epithelial to mesenchymal transition (EMT) in bronchial epithelial cells. Naturally occurring H2B variants differ from canonical H2B by only a few amino acids, yet single amino acid changes in other histone variants (e.g., H3.3) can drive cancer. We therefore hypothesized that H2B variants might function like oncohistones, and investigated how they modify chromatin architecture, dynamics, and function. We find that H2B variants are frequently dysregulated in many cancers, and correlate with patient prognosis. Despite high sequence similarity, mutations in each H2B variant tend to occur at specific “hotspots” in cancer. Some H2B variants cause tighter DNA wrapping around nucleosomes, leading to more compact chromatin structures and reduced transcription factor accessibility to nucleosomal DNA. They also altered genome-wide accessibility to oncogenic regulatory elements and genes, with concomitant changes in oncogenic gene expression programs. Although we did not observe changes in cell proliferation or migration in vitro, our Gene Ontology (GO) analyses of ATAC-seq peaks and RNA-seq data indicated significant changes in oncogenic pathways. These findings suggest that H2B variants may influence early-stage, cancer-associated regulatory mechanisms, potentially setting the stage for oncogenesis later on. Thus, H2B variant expression could serve as an early cancer biomarker, and H2B variants might be novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a5eb06aa259c3b0468d0c21d9ca27556de1d374" target='_blank'>
              Aberrant expression of histone H2B variants reshape chromatin and alter oncogenic gene expression programs
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Youssef A. Hegazy, Kristin Chesnutt, Meredith Eckstein, Richard N. Cassidy, Hejer Dhahri, Richard L. Bennett, Daniёl P. Melters, Elisson Lopes, Zhen Fu, Kin H. Lau, Darrell P Chandler, Michael G. Poirier, Yamini Dalal, Jonathan D. Licht, Yvonne N Fondufe-Mittendorf
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose of the study: to determine the role of retroelements in chromoanagenesis mechanisms in cancer etiopathogenesis.Material and Methods. The search for relevant sources was carried out in the Scopus, Web of Science, PubMed, Elibrary systems, including publications from February 2002 to December 2023. Of the 864 scientifc articles found, 60 were used to write a systematic review.Results. According to original works and meta-analyses results, the cause of complex chromosomal rearrangements during cancer development may be retroelement pathological activation. Chromoanagenesis involves LINE1, SVA, Alu, HERV, which cause double-stranded DNA breaks, insertions in tumor suppressor genes region, the formation of chimeric oncogenes due to retroelement use as new promoters, and function as molecular “band-aids” in non-homologous end junctions and form bridges of distal DNA fragments. Global structural rearrangements of chromosomes observed during chromoanagenesis may be consequences of retroelements activation, which participate in non-allelic homologous recombination and in microhomology-mediated joining of ends characteristic. Certain types of neoplasms, such as colon cancer, are characterized by both high levels of chromothripsis and retroelement activity. In head and neck squamous cell carcinoma, chromoplexy is specifc, the sources of sequences at the breakpoints of which are retroelements. During chromoanagenesis, activation of proto-oncogenes and inactivation of tumor suppressor genes are observed, which is also a consequence of retroelement activation. This is due to the presence of retroelement sequences in proto-oncogenes promoter regions and introns (which become the basis for chimeric oncogene formation) and hot spots of insertional mutagenesis in tumor suppressor genes (transpositions into these regions inactivate these genes).Conclusion. The results obtained on the driver effect of retroelements in chromothripsis, chromoplexy and chromoanasynthesis mechanisms, which are the basis for the formation of clonal evolution of tumors, indicate promise of targeted therapy aimed at silencing the activity of retroelements in cancer patients treatment. For this purpose, it is possible to use microRNAs complementary to retroelements, which are also involved in tumor development, as tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3dd701cb78af4fbe30529399084bdaa803a811" target='_blank'>
              Participation of retroelements in chromoanagenesis in cancer development
              </a>
            </td>
          <td>
            R. N. Mustafin
          </td>
          <td>2024-11-15</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cultured pluripotent stem cells are unique in being the only fully diploid immortal human cell lines. However, during continued culture they can acquire significant chromosome abnormalities. Chromosome 12 trisomy is the most common whole-chromosome abnormality found during culture of human induced pluripotent stem cells (iPSCs). The conventional paradigm is that trisomy 12 occurs very rarely but provides a proliferative advantage, enabling these cells to outcompete the diploid. Here, we challenge this prevailing model by demonstrating that trisomy 12 can arise simultaneously during mitosis in a high percentage (∼2%) of diploid iPSCs. Using a single cell line that reproducibly undergoes transition from diploid to trisomy 12, we found that proliferation differences alone could not account for the rapid dominance of trisomic cells. Through careful mapping by fluorescent in-situ hybridization (FISH), we identified critical transition passages where trisomic cells first appeared and swiftly gained dominance. Remarkably, single trisomic cells repeatedly emerged de novo from diploid parents. Delving deeper, we discovered an extremely high incidence of chromosome 12 anaphase bridging exclusively during transition passages, along with overrepresentation of chromosome 12 chromatids in micronuclei. These micronuclei failed to replicate during S phase, leading to M phase cells containing two normal replicated copies of chromosome 12 and one unreplicated single chromatid. Consequently, we observed 1.5% of metaphase cells with an unpaired chromosome 12 chromatid positioned outside the metaphase plate, and 2% anaphase cells showing three chromosome 12 signals oriented to one pole and two to the other. Further analysis revealed that nearly 20% of subtelomeric repeats were eroded in the p arms but not q arms of chromosome 12 during transition passages. We found that p arm fusions were exclusively responsible for the chromosome 12 bridging observed in anaphase cells of transition passages. Our findings unveil a novel mechanism of whole-chromosome instability in iPSCs, where chromosome 12p arm-specific segregation errors occur simultaneously in a high percentage of cells rather than originating from single, rare events. The slight yet significant growth advantage of trisomy 12 cells allows them to persist and eventually dominate the population. This discovery has profound implications for pinpointing origins of chromosome instability during culture of iPSCs and helping to promote their effective use in research and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ada4689ab4331e9882850f67ceae204b178e90" target='_blank'>
              Origin of Chromosome 12 Trisomy Surge in Human Induced Pluripotent Stem Cells (iPSCs)
              </a>
            </td>
          <td>
            Maria Narożna, Megan C. Latham, G. Gorbsky
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The 2-micron plasmid residing within the host budding yeast Saccharomyces cerevisiae nucleus serves as a model system for understanding the mechanism of segregation and stable maintenance of circular endogenously present extrachromosomal DNA in eukaryotic cells. The plasmid is maintained at a high average copy number (40-60 copies per yeast cell) through generations despite there is no apparent benefit to the host. Notably, the segregation mechanism of 2-micron plasmid shares significant similarities with those of bacterial low-copy-number plasmids and episomal forms of viral genomes in mammalian cells. These similarities include formation of a complex where the plasmid- or viral encoded proteins bind to a plasmid- or viral genome-borne locus, respectively and interaction of the complex with the host proteins. These together form a partitioning system that ensures stable symmetric inheritance of both these genomes from mother to daughter cells. Recent studies with substantial evidence showed that the 2-micron plasmid, like episomes of viruses such as Epstein-Barr virus, relies on tethering itself to the host chromosomes in a non-random fashion for equal segregation. This review delves into the probable chromosome hitchhiking mechanisms of 2-micron plasmid during its segregation, highlighting the roles of specific plasmid-encoded proteins and their interactions with host proteins and the chromosomes. Understanding these mechanisms provides broader insights into the genetic stability and inheritance of extrachromosomal genetic elements across diverse biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf1d5837f159323449ccdc148cbe5bd08f01794" target='_blank'>
              Chromosome hitchhiking: a potential strategy adopted by the selfish yeast plasmids to ensure symmetric inheritance during cell division.
              </a>
            </td>
          <td>
            Deepanshu Kumar, S. Ghosh
          </td>
          <td>2024-12-13</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acquisition of a telomere maintenance mechanism is essential for cancer cells. In a minority of tumors, telomeres are lengthened via Alternative Maintenance of Telomeres (ALT), a telomerase-independent pathway based on homologous recombination. ALT tumors have heavily rearranged genomes with many structural variants containing telomere repeats. To better understand the genetic evolution of these tumors, we seek to determine if certain genomic loci tend to spatially associate with telomeres in ALT and are especially liable to experience telomere recombination events as a result. Assays that reveal close spatial associations between genomic loci, such as SPRITE and Hi-C, have enabled extensive exploration of genomic spatial organization. However, as analysis pipelines for these next-generation sequencing-based assays typically discard reads aligning to repetitive elements, little is known about the spatial arrangement of telomeres and other repetitive loci in the nucleus. Here, we present TelSPRITE, a novel approach to extracting telomere contact frequencies from SPRITE data. We identify reads containing telomere repeats and sort them into a single bin, quantifying spatial contacts between the telomere bin and the rest of the genome. Our analysis reveals a strong dependency of telomere contact frequency on chromosomal distance from the telomere, consistent with the known effect of linear distance on 3-dimensional spatial contacts. Telomere contacts are also strongly enriched near centromeres, a phenomenon that may be reflective of spatial clustering of heterochromatic regions. ALT cell lines are globally enriched for telomere content and display distinctive intrachromosomal spikes in telomere contact frequency. Our customized analysis of long read sequencing data suggests that loci with high telomere contact frequencies represent structural variants containing telomere repeats in ALT cells. Collectively, our results demonstrate general principles of telomeric spatial organization while also profiling the spectrum of genomic rearrangements in ALT cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca078135018e0ad318b04cc28cb3eb2ce26683a8" target='_blank'>
              Telomere interactions and structural variants in ALT cells revealed with TelSPRITE
              </a>
            </td>
          <td>
            David G. Wilson, Sarah F. Clatterbuck Soper, M. Pineda, Robert L. Walker, Paul S. Meltzer
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin D. Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer. To resolve genome dysregulation associated with HPV integration, we performed Oxford Nanopore long-read sequencing on 72 cervical cancer genomes from an Ugandan dataset that was previously characterized using short-read sequencing. We found recurrent structural rearrangement patterns at HPV integration events, which we categorized as: del(etion)-like, dup(lication)-like, translocation, multibreakpoint, or repeat region integrations. Integrations involving amplified HPV-human concatemers, particularly multibreakpoint events, frequently harbored heterogeneous forms and copy numbers of the viral genome. Transcriptionally active integrants were characterized by unmethylated regions in both the viral and human genomes downstream from the viral transcription start site, resulting in HPV-human fusion transcripts. In contrast, integrants without evidence of expression lacked consistent methylation patterns. Furthermore, whereas transcriptional dysregulation was limited to genes within 200 kilobases of an HPV integrant, dysregulation of the human epigenome in the form of allelic differentially methylated regions affected megabase expanses of the genome, irrespective of the integrant's transcriptional status. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8457c68dc264c60760742af32b0cda2346039" target='_blank'>
              Rearrangements of viral and human genomes at human papillomavirus integration events and their allele-specific impacts on cancer genome regulation.
              </a>
            </td>
          <td>
            Vanessa L. Porter, Michelle Ng, Kieran O’Neill, Signe MacLennan, R. Corbett, Luka Culibrk, Zeid Hamadeh, Marissa Iden, Rachel Schmidt, Shirng-Wern Tsaih, C. Nakisige, Martin Origa, Jackson Orem, Glenn Chang, Jeremy Fan, K. Nip, Vahid Akbari, Simon K. Chan, James Hopkins, Richard A. Moore, E. Chuah, K. Mungall, A. Mungall, Inanҫ Birol, S. Jones, Janet S. Rader, M. Marra
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Ultraviolet (UV)-induced DNA mutations produce genetic drivers of cutaneous melanoma initiation and numerous neoantigens that can trigger anti-tumor immune responses in the host. Consequently, melanoma cells must rapidly evolve to evade immune detection by simultaneously modulating cell-autonomous epigenetic mechanisms and tumor-microenvironment interactions. Angiogenesis has been implicated in this process; although an increase of vasculature initiates the immune response in normal tissue, solid tumors manage to somehow enhance blood flow while preventing immune cell infiltration. By comparing the expression of transcription factors (TFs) across early-stage melanoma, naevi, and other cancer types, we found the homeodomain-containing TF HOXD13 drives a melanoblast-like developmental program, which is upregulated in melanoma and strongly correlated with angiogenesis and immune cell exclusion. Using transcriptomics, 3D chromatin profiling, and in vivo models, we demonstrate that HOXD13 upregulation promotes tumor growth in vivo by concomitantly enhancing angiogenesis and suppressing T-cell infiltration. HOXD13 orchestrates 3D chromatin contacts between distal enhancers and promoters, simultaneously activating VEGFA, SEMA3A, and CD73. VEGFA and SEMA3A remodel the tumor vasculature and CD73 elevates extracellular levels of adenosine, a vasodilator and immune suppressor that binds adenosine receptors (AdR) on endothelial and T cells. In line with these findings, HOXD13-induced growth advantage in vivo was significantly reversed by the concomitant administration of VEGFR and AdR inhibitors. By revealing a dual pro-angiogenic and immunosuppressive HOXD13-CD73/VEGF gene regulatory axis, we identify a subset of patients who might benefit from combinations of AdR and VEGFR inhibitors which are both currently being tested in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e09ae66ed07e79880f3fb8e2a1de2aa1f2113" target='_blank'>
              A targetable developmental program co-regulates angiogenesis and immune evasion
              </a>
            </td>
          <td>
            P. Berico, Amanda Flores Yanke, Fatemeh Van Rajabpour, Catherine Do, Irving Simonin Wilmer, Theodore Sakellaropoulos, Ines Delclaux, Robert Stagnitta, Estefania Vázquez-Cruz, Iman Osman, Jane A Skok, C. D. Robles-Espinoza, Amanda W. Lund, Markus Schober, Eva Hernando
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2ef81377921558c83081b12d35bab26833147b" target='_blank'>
              Extrachromosomal circular DNA containing DTX1 promotes cell growth in hydroquinone-induced malignantly transformed cells by regulating the transcription of DTX1
              </a>
            </td>
          <td>
            X. Ling, Qunfang Jiao, Daifan Lin, Jialong Chen, Yali Han, Jinxue Meng, Bohuan Zhong, He Zhang, Gongda Zhang, Fangling Zhu, Jiheng Qin, Yongdui Ruan, Linhua Liu
          </td>
          <td>2024-11-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a4f47822586a1fd8d669e38036044aa6da804e" target='_blank'>
              Chromothripsis in cancer.
              </a>
            </td>
          <td>
            Milena Simovic-Lorenz, Aurélie Ernst
          </td>
          <td>2024-11-15</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Wang Yuh-Hwa
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9f63e7b6f9db6db3b912732c476af942c10e05" target='_blank'>
              Decreased mitochondrial transcription factor A and mitochondrial DNA copy number promote cyclin-dependent kinase inhibitor 1A expression and reduce tumorigenic properties of colorectal cancer cells
              </a>
            </td>
          <td>
            Jessika Buchwaldt, Tania Fritsch, Monika Hartmann, H. Witzel, Michael Kloth, Wilfried Roth, K. Tagscherer, Nils Hartmann
          </td>
          <td>2024-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Chromosomal rearrangements on the short arm of chromosome 8 cause 8p syndrome, a rare developmental disorder characterized by neurodevelopmental delays, epilepsy, and cardiac abnormalities. While significant progress has been made in managing the symptoms of 8p syndrome and other conditions caused by large-scale chromosomal aneuploidies, no therapeutic approach has yet been demonstrated to target the underlying disease-causing chromosome. Here, we establish a two-step approach to eliminate the abnormal copy of chromosome 8 and restore euploidy in cells derived from an individual with a complex rearrangement of chromosome 8p. Transcriptomic analysis revealed 361 differentially expressed genes between the proband and the euploid revertant, highlighting genes both within and outside the 8p region that may contribute to 8p syndrome pathology. Our work demonstrates the feasibility of using chromosome engineering to correct complex aneuploidies in vitro and suggests a potential therapeutic avenue for disorders caused by chromosomal rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9c5433c28b14d1239c7306435b8438d42ccf6" target='_blank'>
              Chromosome engineering to restore euploidy in cells harboring a complex rearrangement of chromosome 8
              </a>
            </td>
          <td>
            Sophia N. Lee, Lu Qiao, Sarah L. Thompson, R. Hagenson, Teresa Davoli, J. Sheltzer
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ebab4a081c455b627a48f2ce35c264b01bf2033" target='_blank'>
              EZH2 inhibition sensitizes MYC-high medulloblastoma cancers to PARP inhibition by regulating NUPR1-mediated DNA repair.
              </a>
            </td>
          <td>
            Jianzhong Yu, Jichang Han, Meng Yu, Huanwen Rui, An Sun, Hao Li
          </td>
          <td>2024-11-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Alternative Lengthening of Telomeres (ALT) is a homologous recombination-dependent telomere elongation mechanism utilized by at least 10-15% of all cancers. Here we identified that the DNA topoisomerase, TOP3A is enriched at the telomeres of ALT cells but not at the telomeres of telomerase-positive (Tel) cancer cells. We demonstrate that TOP3A stabilizes the shelterin protein TERF2 in ALT cancer cell lines but not in Tel cells and that long non-coding telomere transcribed RNA (TERRA) enrichment at telomeres depends upon TOP3A. TOP3A also promotes the generation of single-stranded telomeric C-strand (ssTeloC) DNA, which is a recently discovered marker for ALT. Additionally, we found that inducing TOP3A-DNA-protein crosslinks in ALT cells suppresses TERRA enrichment as well as destabilizes TERF2. Taken together these observations uncover the unexplored functions of TOP3A at ALT telomeres and suggest the potential of developing an ALT-specific cancer therapeutic strategy targeting TOP3A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e2b4b91b59c932435955d5d27a82b4340d75f06" target='_blank'>
              Topoisomerase 3α (TOP3A) Dependent Alternative Lengthening of Telomeres (ALT)
              </a>
            </td>
          <td>
            Prashant Khandagale, Yilun Sun, Sourav Saha, L. Saha, Y. Pommier
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ (WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors, and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72eaa094c63444d04a8a1f5ac2cb2674cf4fc8fe" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Alexander Howard, Bertram Aschenbrenner, Emily Lay, Natalia Krajic, Emma Bailey, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis.
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcd0f4cc9961826189852b167ab5b2ef51c1fec" target='_blank'>
              Integrating Circle-Seq with transcriptomics reveals genome-wide characterization of extrachromosomal circular DNA for dilated cardiomyopathy
              </a>
            </td>
          <td>
            Zhenhao Lin, Fangjie Dai, Bo Li, Yongchao Zhao, Changqian Wang
          </td>
          <td>2024-11-29</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA double strand break (DSB) is a highly toxic lesion that can generate genome instability, a major source of tumorigenesis. DSBs are mainly repaired by non-homologous end joining (NHEJ) or homologous recombination (HR). The selection of the DSB repair pathway primarily depends on the DNA resection of the DSB ends. Indeed, HR is initiated by resection at the DSB generating 3’ single stranded extension. The shieldin complex prevents resection fostering DSB repair toward NHEJ. Here, we reveal that the inflammasome sensor NLRP3 facilitates DNA end resection to promote the HR pathway in an inflammasome-independent manner. Strikingly, NLRP3 silencing decreases HR efficiency, as evidenced by RAD51 foci and functional HR assays. Mechanistically, we describe that NLRP3 interacts with REV7, a subunit of the shieldin complex, and its depletion increases REV7 recruitment to IR-induced DSBs. Similar to cancer cells harboring HR mutated genes, we find that NLRP3 deficient cells are sensitive to PARP inhibitors (PARPi) and exhibit an epistatic relationship with BRCA1 deficiency. Remarkably, loss of REV7 in NLRP3-depleted cells induces PARPi resistance by restoring HR. This study unravels the crucial role of the innate immune receptor NLRP3 in regulating the selection of DSB repair pathways to maintain genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88ff36a1381788be79115bbf708323b6cc93cae" target='_blank'>
              The inflammasome sensor NLRP3 interacts with REV7 to maintain genome integrity through homologous recombination
              </a>
            </td>
          <td>
            Delphine Burlet, Md Muntaz Khan, Sabine Hacot, Hannes Buthmann, Léa Bardoulet, Anne-Laure Huber, Julie Gorry, Bernard S. Lopez, Yohann Couté, Matthias Geyer, Agnès Tissier, Virginie Petrilli
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="INTRODUCTION
The SEWA cell line, which is derived from a virus induced murine osteosarcoma (OS) ascites was established in the 1980s from a serially transplanted male derived tumor that first published in 1961. It has been applied in about 50 studies but was never genetically characterized in detail; this study fills that gap.


METHODS
The SEWA cell line was analyzed for its chromosomal constitution using molecular cytogenetic approaches. Array comparative genomic hybridization was performed to characterize copy number alterations.


RESULTS
SEWA has a near diploid karyotype without Y chromosome material. The complex karyotype includes neocentrics and simple and complex rearrangements. Amplification of MYC oncogene was detected in two homogeneously staining regions on two different derivative chromosomes.


CONCLUSION
An in silico translation of the obtained results to the human genome indicated that SEWA is suitable as a model for advanced human OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec328ac3891c2b11391d74ef5192bbc587c6a2" target='_blank'>
              Cytogenomic characterization of murine osteosarcoma cell line SEWA.
              </a>
            </td>
          <td>
            Thomas Liehr, Martina Rinčic
          </td>
          <td>2024-12-11</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma Multiforme (GBM) is the most prevalent and highly malignant form of adult brain cancer characterized by poor overall survival rates. Effective therapeutic modalities remain limited, necessitating the search for novel treatments. Neurodevelopmental pathways have been implicated in glioma formation, with key neurodevelopmental regulators being re- expressed or co-opted during glioma tumorigenesis. Here we identified a serine/threonine kinase, NUAK family kinase 2 (NUAK2), as a fetal oncogene in mouse and human brains. We found robust expression of NUAK2 in the embryonic brain that decreases throughout postnatal stages and then is re-expressed in malignant gliomas. However, the role of NUAK2 in GBM tumorigenesis remains unclear. We demonstrate that CRIPSR-Cas9 mediated NUAK2 deletion in GBM cells results in suppression of proliferation, while overexpression leads to enhanced cell growth in both in vitro and in vivo models. Further investigation of the downstream biological processes dysregulated in the absence of NUAK2 reveals that NUAK2 modulates extracellular matrix (ECM) components to facilitate migratory behavior. Lastly, we determined that pharmaceutical inhibition of NUAK2 is sufficient to impede the proliferation and migration of malignant glioma cells. Our results suggest that NUAK2 is an actionable therapeutic target for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c93062bc61d08f65a71be5b4b969d76224456d0" target='_blank'>
              A fetal oncogene NUAK2 is an emerging therapeutic target in glioblastoma
              </a>
            </td>
          <td>
            Hanhee Jo, Aneesh Dalvi, Wenqi Yang, Elizabeth Morozova, Sarah Munoz, Stacey M. Glasgow
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G. Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, , Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Large genomic rearrangements of the PALB2 gene, especially deletions and duplications, are associated with hereditary breast–ovarian cancer. This study investigates intronic breakpoints linked to rearrangements in PALB2 exon 11, which is crucial for understanding the mechanisms affecting patients with hereditary breast and ovarian syndrome. Through next-generation sequencing, one duplication and three deletions were identified, confirmed by Multiplex Ligation-Dependent Probe Amplification. Detailed characterization revealed a tandem duplication of 5134 base pairs mediated by AluY repeats and identical deletions in three unrelated patients promoted by AluSx elements, resulting in a truncated PALB2 protein. These findings highlight the instability of intronic regions flanking exon 11 and suggest directions for future research on the prevalence and functional implications of these genomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30610e3b0b4ee5300bddce9e6a181cf9f25f21c9" target='_blank'>
              Alu–Mediated Duplication and Deletion of Exon 11 Are Frequent Mechanisms of PALB2 Inactivation, Predisposing Individuals to Hereditary Breast–Ovarian Cancer Syndrome
              </a>
            </td>
          <td>
            Diletta Sidoti, Valeria Margotta, Diletta Calosci, Erika Fiorentini, C. Bacci, Francesca Gensini, Laura Papi, Marco Montini
          </td>
          <td>2024-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Myeloid leukemias are heterogeneous cancers with diverse mutations, sometimes in genes with unclear roles and unknown functional partners. PHF6 and PHIP are two poorly-understood chromatin-binding proteins recurrently mutated in acute myeloid leukemia (AML). PHF6 mutations are associated with poorer outcomes, while PHIP was recently identified as the most common selective mutation in Black patients in AML. Here, we show that PHF6 is a transcriptional repressor that suppresses a stemness gene network, and that PHF6 missense mutations, classified by current clinical algorithms as variants of unknown significance, produce unstable or non-functional protein. We present multiple lines of evidence converging on a critical mechanistic connection between PHF6 and PHIP. We show that PHIP loss phenocopies PHF6 loss, and that PHF6 requires PHIP to occupy chromatin and exert its downstream transcriptional program. Our work unifies PHF6 and PHIP, two disparate leukemia-mutated proteins, into a common functional complex that suppresses AML stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acc625ad647593a9481f172008906d4067da9a6" target='_blank'>
              Leukemia-mutated proteins PHF6 and PHIP form a chromatin complex that represses acute myeloid leukemia stemness
              </a>
            </td>
          <td>
            Aishwarya S Pawar, Patrick Somers, Aleena Alex, Subin S. George, Charles Antony, Roman Verner, Sanese K. White-Brown, Mohit Khera, M. S. Mendoza-Figueroa, K. F. Liu, Jennifer Morrissette, V. Paralkar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D. Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DNA viruses at once elicit and commandeer host pathways, including DNA repair pathways for virus replication. Despite encoding its own DNA polymerase and processivity factor, human cytomegalovirus (HCMV) recruits the cellular processivity factor, proliferating cell nuclear antigen (PCNA) and specialized host DNA polymerases involved in translesion synthesis (TLS) to replication compartments (RCs) where viral DNA (vDNA) is synthesized. While the recruitment of TLS polymerases is important for viral genome stability, the role of PCNA is poorly understood. PCNA function in DNA repair is regulated by monoubiquitination (mUb) or SUMOylation of PCNA at lysine 164 (K164). We find that mUb-PCNA increases over the course of infection, and modification of K164 is required for PCNA-mediated restriction of virus replication. mUb-PCNA plays important known roles in recruiting TLS polymerases to DNA, which we have shown are important for viral genome integrity and diversity, represented by novel junctions and single nucleotide variants (SNVs), respectively. We find that PCNA drives SNVs on vDNA similar to Y-family TLS polymerases, but that this did not require modification at K164. Unlike TLS polymerases, PCNA was dispensable for preventing large scale rearrangements on vDNA. These striking results suggest separable PCNA-dependent and - independent functions of TLS polymerases on vDNA. By extension, these results imply roles for TLS polymerase beyond their canonical function in TLS in host biology. These findings highlight PCNA as a complex restriction factor for HCMV infection, likely with multiple distinct roles, and provides new insights into the PCNA-mediated regulation of DNA synthesis and repair in viral infection. IMPORTANCE Genome synthesis is a critical step of virus life cycles and a major target of antiviral drugs. Human cytomegalovirus, like other herpesviruses, encodes machinery sufficient for viral DNA synthesis and relies on host factors for efficient replication. We have shown that host DNA repair factors play important roles in HCMV replication, but our understanding of this is incomplete. Building on previous findings that specialized host DNA polymerases contribute to HCMV genome integrity and diversity, we sought to determine the importance of PCNA, the central polymerase regulator. PCNA associates with nascent viral DNA and restricts HCMV replication. While PCNA is dispensable for genome integrity, it contributes to genome diversity. Our findings suggest that host polymerases function on viral genomes by separable PCNA-dependent and - independent mechanisms. Through revealing complex roles for PCNA in HCMV replication, this study expands the repertoire of host DNA synthesis and repair proteins hijacked by this ubiquitous herpesvirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe97c65ecc13c9b2841234ccb055156970288c9" target='_blank'>
              Complex roles for proliferating cell nuclear antigen in restricting human cytomegalovirus replication
              </a>
            </td>
          <td>
            P. Longmire, Olivia Daigle, S. Zeltzer, Matias Lee, Marek Svoboda, Marco Padilla-Rodriguez, Carly Bobak, Giovanni Bosco, F. Goodrum
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="To maintain genome stability, proliferating cells must enact a program of telomere maintenance. While most tumors maintain telomeres through the action of telomerase, a subset of tumors utilize a DNA-templated process termed Alternative Lengthening of Telomeres or ALT. ALT is associated with mutations in the ATRX/DAXX/H3.3 histone chaperone complex, which is responsible for deposition of non-replicative histone variant H3.3 at heterochromatic regions of the genome including telomeres. We wished to better understand the role DAXX plays in ALT suppression, and to determine which disease-associated DAXX mutations are unable to suppress ALT. To answer this question, we have leveraged the G292 cell line, in which ATRX is wild type but DAXX has undergone a fusion event with the non-canonical kinesin KIFC3. Restoration of wild-type DAXX in G292 localizes ATRX and abrogates ALT. Using this model system, we tested the ability of a panel of disease-associated DAXX missense variants to suppress ALT. Missense mutations in the ATRX binding domain, the histone binding domain, and the C-terminal SUMO interaction motif reduce the ability of DAXX to suppress ALT. Unexpectedly, we find that mutations in the DAXX histone binding domain lead to failure of ATRX localization. We conclude that a key function of DAXX in ALT suppression is the localization of ATRX to nuclear foci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecce9b8b3920bbf0f0b14800927fbabcdf8182c8" target='_blank'>
              Cancer-associated DAXX mutations reveal a critical role for ATRX localization in ALT suppression
              </a>
            </td>
          <td>
            Sarah F. Clatterbuck Soper, Robert L. Walker, M. Pineda, Y. Zhu, James Dalgleish, Jasmine Wang, Paul S. Meltzer
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Breast cancer is the most common cancer in women and the 2nd most common cancer worldwide, yearly impacting over 2 million females and causing 650 thousand deaths. It has been widely studied, but its epigenetic variation is not entirely unveiled. We aimed to identify epigenetic mechanisms impacting the expression of breast cancer related genes to detect new potential biomarkers and therapeutic targets. We considered The Cancer Genome Atlas database with over 800 samples and several omics datasets such as mRNA, miRNA, DNA methylation, which we used to select 2701 features that were statistically significant to differ between cancer and control samples using the Monte Carlo Feature Selection and Interdependency Discovery algorithm, from an initial total of 417,486. Their biological impact on cancerogenesis was confirmed using: statistical analysis, natural language processing, linear and machine learning models as well as: transcription factors identification, drugs and 3D chromatin structure analyses. Classification of cancer vs control samples on the selected features returned high classification weighted Accuracy from 0.91 to 0.98 depending on feature-type: mRNA, miRNA, DNA methylation, and classification algorithm. In general, cancer samples showed lower expression of differentially expressed genes and increased β-values of differentially methylated sites. We identified mRNAs whose expression is well explained by miRNA expression and differentially methylated sites β-values. We recognized differentially methylated sites possibly affecting NRF1 and MXI1 transcription factors binding, causing a disturbance in NKAPL and PITX1 expression, respectively. Our 3D models showed more loosely packed chromatin in cancer. This study successfully points out numerous possible regulatory dependencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471e2fdeae8d795259c36bd54b5931925e83e88a" target='_blank'>
              Unveiling epigenetic regulatory elements associated with breast cancer development
              </a>
            </td>
          <td>
            Marta Jardanowska-Kotuniak, Michał Dramiński, Michał Wlasnowolski, Marcin Łapiński, Kaustav Sengupta, Abhishek Agarwal, Adam Filip, Nimisha Ghosh, Vera Pancaldi, Marcin Grynberg, I. Saha, Dariusz Plewczyński, Michal J Dabrowski
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5–34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Telomeric repeat-containing RNA (TERRA), transcribed from subtelomeric regions towards telomeric ends, poses challenges in deciphering its complete sequences. Utilizing TERRA-capture RNA-seq and Oxford Nanopore direct RNA sequencing to acquire full-length TERRA, we annotate TERRA transcription regions in the human T2T-CHM13 reference genome. TERRA transcripts encompass hundreds to over a thousand nucleotides of telomeric repeats, predominantly originating from 61-29-37 bp repeat promoters enriched with H3K4me3, RNA pol II, CTCF, and R-loops. We develop a bioinformatics tool, TERRA-QUANT, for quantifying TERRA using RNA-seq datasets and find that TERRA increases with age in blood, brain, and fibroblasts. TERRA upregulation in aged leukocytes is confirmed by RT-qPCR. Single-cell RNA-seq analysis demonstrates TERRA expression across various cell types, with upregulation observed in neurons during human embryonic stem cell differentiation. Additionally, TERRA levels are elevated in brain cells in the early stage of Alzheimer’s disease. Our study provides evidence linking TERRA to human aging and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fedf4165a727cf3608fb4e1f9bd6a9359a39d7e" target='_blank'>
              Telomeric Repeat-Containing RNA Increases in Aged Human Cells
              </a>
            </td>
          <td>
            Yu-Hung Hsieh, Chin-Hua Tai, Meng-Ting Yeh, Yu-Chen Chen, Po-Cheng Yang, Chien-Ping Yen, Der-Sheng Han, Hsueh-Ping Catherine Chu
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The study explores the role of KIF18A, a gene linked to whole-genome doubling (WGD) and chromosomal instability (CIN), in the metastatic progression of hepatocellular carcinoma (HCC). We found that KIF18A is a critical link between CIN and anoikis resistance, key factors in HCC metastasis. Through the bulk-seq analysis, we identified the pressures of anoikis and CIN faced by tumor cells during HCC metastasis and identified hub genes central to this process. Our results reveal that E2F activation during HCC progression leads to the transcription of KIF18A, promoting the survival and metastasis of CIN tumors. Overexpression of KIF18A leads to longer tumor life, more micronuclei, and increased survival and metastasis through non-canonical NF-kB activation. Deletion of KIF18A stabilizes the nuclear membrane of the micronucleus, silences cGAS-STING and PI3K-AKT pathways, and inhibits classical NF-kB. The study’s limitations include the need for further animal studies to validate the findings and explore the transient activation of the cGAS-STING pathway. Additionally, future research could focus on the potential therapeutic implications of targeting KIF18A in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae22aedbaa5a2bdc1f487c6a0f986379d3bb08c" target='_blank'>
              KIF18A in HCC Metastasis: Dual Role in Anoikis Resistance and Chromosome Instability
              </a>
            </td>
          <td>
            Changhao Kan, Weiyu Bai, Xinru Yang, Hayam Hamdy, Xianghui Yang, Xin Liu, Yan Lv, Junling Shen, Jianwei Sun
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) liquid biopsies offer a powerful, non-invasive approach for cancer diagnostics, leveraging DNA fragments primarily released during apoptosis. Using droplet digital PCR on cfDNA samples from 88 breast and colorectal cancer patients, we uncovered highly consistent nucleosome breakpoints across individuals. Proximity to these cfDNA breakpoints diminishes the sensitivity of somatic mutation assays, including those targeting IDH1 and NRAS hotspot mutations, underscoring the importance of precise nucleosome positioning data. We introduce a probabilistic nucleosome protection scoring method that enables high-resolution mapping of nucleosome positions and cfDNA breakpoints. These maps reveal distinct nucleosome spacing patterns between heterochromatin and euchromatin, corresponding to different chromatin fiber topologies. Analysis of X-chromosome inactivation further supports the topoisomer model: heterochromatin’s 187 bp nucleosome repeat length differs from euchromatin’s 182 bp, generating wave interference patterns in nucleosome signals. Our findings suggest that loop-mediated transitions from T2 to T1 topoisomers shape chromatin accessibility and cfDNA fragmentation, advancing the diagnostic potential of liquid biopsy assays. Highlights Nucleosome breakpoint conservation impacts somatic mutation assay sensitivity Probabilistic protection scoring enhances cfDNA nucleosome peak resolution Chromatin states exhibit unique nucleosome spacing linked to distinct topologies, influencing cfDNA fragmentation Chromosome X-inactivation phasing analyses reveal waveform interference in nucleosome positioning signals Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72833584f0c36d4300251f882a1e6ae1ee535bda" target='_blank'>
              Unraveling cfDNA Fragmentation Patterns: Linking Chromatin Architecture to Cancer Diagnostic Sensitivity
              </a>
            </td>
          <td>
            Andrew D Johnston, Fiach Antaw, Jennifer Lu, C. Stirzaker, M. Trau, D. Korbie
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="DNA replication and RNA transcription processes compete for the same DNA template and, thus, frequently collide. These transcription-replication collisions are thought to lead to genomic instability, which places a selective pressure on organisms to avoid them. Here, we review the predisposing causes, molecular mechanisms, and downstream consequences of transcription-replication collisions (TRCs) with a strong emphasis on prokaryotic model systems, before contrasting prokaryotic findings with cases in eukaryotic systems. Current research points to genomic structure as the primary determinant of steady-state TRC levels and RNA polymerase regulation as the primary inducer of excess TRCs. We review the proposed mechanisms of TRC-induced DNA damage, attempting to clarify their mechanistic requirements. Finally, we discuss what drives genomes to select against TRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/640b1703040bd391f51873c4274b272148a62c32" target='_blank'>
              Clues to transcription/replication collision-induced DNA damage: it was RNAP, in the chromosome, with the fork.
              </a>
            </td>
          <td>
            Matthew B Cooke, Christophe Herman, P. Sivaramakrishnan
          </td>
          <td>2024-11-24</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma is recognized as the most aggressive type of primary brain tumor. Despite recent advances in understanding the molecular mechanisms involved in the biology of glioblastoma, patient survival rates remain disappointing, primarily due to the lack of effective treatment options. Tumor necrosis factor receptor-associated protein 1 (TRAP1), a member of the heat shock protein 90 (Hsp90) family, refers to a protein predominantly localized in the mitochondria that regulates both cellular metabolic reprogramming and mitochondrial apoptosis. This protein is highly expressed in several types of tumors, including colorectal cancer, breast cancer, prostate cancer, and lung cancer, and is often associated with drug resistance. However, TRAP1 is also downregulated in certain cancers such as ovarian cancer, bladder cancer, and renal cancer, where its lower expression correlates with poorer prognoses and chemoresistance. The role of TRAP1 lies in enhancing or suppressing oxidative phosphorylation, with the impact of such regulation on tumor development and progression being a matter of ongoing debate. These observations prompt further investigation into the mechanisms responsible for the dual role of TRAP1 as both an oncogene and a tumor suppressor in specific types of tumors, particularly glioblastoma. The present study reviews the role of TRAP1 in the development and progression of glioblastoma and discusses the potential of targeting TRAP1 as a novel therapeutic approach against tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/236ff07b8d8eaf07109bcd843d2d90948b45e7ad" target='_blank'>
              Role of TRAP1 Protein in the Development and Progression of Glioblastoma
              </a>
            </td>
          <td>
            I. Gareev, A. Yasinskaya, S. Roumiantsev
          </td>
          <td>2024-12-28</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4cf48abf7265148385bfea0bb2728f6fd6101" target='_blank'>
              SMYD3 drives cell cycle and epithelial-mesenchymal transition pathways through dual gene transcriptional repression and activation in HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Madhavi Murali, Abbas Saeed, Sohyoung Kim, K. Burkitt, Hui Cheng, Arfa Moshiri, Jawad Akhtar, D. Tsai, Marie Luff, Baktiar Karim, V. Saloura
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38564ca14081459f6f145ede37b42148eaa6f90c" target='_blank'>
              PinX1 plays multifaceted roles in human cancers: a review and perspectives
              </a>
            </td>
          <td>
            Dian You, Kaiwen Tong, Yuan Li, Ting Zhang, Yongqiang Wu, Ling Wang, Guangming Chen, Xiaoying Zhang
          </td>
          <td>2024-11-17</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fecaeaf832acff0ef3a7b992effb9b3cb46a2f35" target='_blank'>
              TORC2 inhibition triggers yeast chromosome fragmentation through misregulated Base Excision Repair of clustered oxidation events
              </a>
            </td>
          <td>
            Kenji Shimada, Cleo V D Tarashev, Stephanie Bregenhorn, C. Gerhold, Barbara van Loon, Gregory Roth, Verena Hurst, Josef Jiricny, S. B. Helliwell, Susan M. Gasser
          </td>
          <td>2024-11-15</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Alterations in DNA methylation profiles are typically found in cancer cells, combining genome-wide hypomethylation with hypermethylation of specific regions, such as CpG islands, which are normally unmethylated. Driving effects in cancer development have been associated with alteration of DNA methylation in certain regions, inducing, for example, the repression of tumor suppressor genes or the activation of oncogenes and retrotransposons. These alterations represent prime candidates for the development of specific markers for the detection, diagnosis and prognosis of cancer. In particular, these markers, distributed along the genome, provide a wealth of information that offers potential for innovation in the field of liquid biopsy, in particular thanks to the emergence of artificial intelligence for diagnostic purposes. This could overcome the limitations related to sensitivities and specificities, which remain too low for the most difficult applications in oncology: the detection of cancers at an early stage, the monitoring of residual disease and the analysis of brain tumors. In addition, targeting the enzymatic processes that control the epigenome offers new therapeutic strategies that could reverse the regulatory anomalies of these altered epigenomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6734e14b7462be1947bbbaa4edaec73c090d31d" target='_blank'>
              [Epigenetics and cancer: the role of DNA methylation].
              </a>
            </td>
          <td>
            Marine Gorse, Charline Bianchi, Charlotte Proudhon
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Structural variation (SV), defined as balanced and unbalanced chromosomal rearrangements >1 kb, is a major contributor to germline and neoplastic disease. Large variants have historically been evaluated by chromosome analysis and now are commonly recognized by chromosomal microarray analysis (CMA). The increasing application of genome sequencing (GS) in the clinic and the relatively high incidence of chromosomal abnormalities in sick newborns and children highlights the need for accurate SV interpretation and reporting. In this review, we describe SV patterns of common cytogenetic abnormalities for laboratorians who review GS data.


CONTENT
GS has the potential to detect diverse chromosomal abnormalities and sequence breakpoint junctions to clarify variant structure. No single GS analysis pipeline can detect all SV, and visualization of sequence data is crucial to recognize specific patterns. Here we describe genomic signatures of translocations, inverted duplications adjacent to terminal deletions, recombinant chromosomes, marker chromosomes, ring chromosomes, isodicentric and isochromosomes, and mosaic aneuploidy. Distinguishing these more complex abnormalities from simple deletions and duplications is critical for phenotypic interpretation and recurrence risk recommendations.


SUMMARY
Unlike single-nucleotide variant calling, identification of chromosome rearrangements by GS requires further processing and multiple callers. SV databases have caveats and limitations depending on the platform (CMA vs sequencing) and resolution (exome vs genome). In the rapidly evolving era of clinical genomics, where a single test can identify both sequence and structural variants, optimal patient care stems from the integration of molecular and cytogenetic expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a2a826f1810ca4839038af6a42ce3daef01b34" target='_blank'>
              Structural Variation Interpretation in the Genome Sequencing Era: Lessons from Cytogenetics.
              </a>
            </td>
          <td>
            Lucilla Pizzo, M. K. Rudd
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-stranded DNA secondary structures such as G-quadruplexes (G4s) can potentially disrupt transcription, replication, and repair. Using bio-informatic analysis, here we show that BMI1 is enriched at putative G4s flanked by heterochromatin domains, and that BMI1 knockdown in human dermal fibroblasts (HDFs) resulted in heterochromatin relaxation and G4 induction, followed by replication stress and genomic instability. In these cells, G4s co-localized with large 53BP1 and PCNA foci resembling replication catastrophes. Inhibiting transcription partly attenuated DNA damage, suggesting rescue of transcription-replication collisions at difficult-to-replicate sequences. In BMI1 knockdown or pyridostatin-exposed HDFs, the Werner helicase accumulated and co-localized with G4s. Acute WRN knockdown also resulted in G4 induction. In HDFs from Werner and Hutchinson-Gilford progeria syndromes, loss of heterochromatin and nuclear envelope anomalies were associated with G4 induction and DNA damage. Nuclear envelope anomalies were also prominent following BMI1 knockdown. These findings suggests that heterochromatin-mediated repression of G4s attenuates replication stress and genomic instability, and that this mechanism is shared across distinct progeroid models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731920cfa53eda1eb3b00a50ee2eec0629da79d1" target='_blank'>
              BMI1-mediated heterochromatinization represses G-quadruplex DNA formation to maintain genomic stability during replication
              </a>
            </td>
          <td>
            Roy Hanna, Gilbert Bernier
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge.


EXPERIMENTAL DESIGN
We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. The DIAMOND (Detection of Long Interspersed Nuclear Element Altered Methylation ON plasma DNA) method targets 30-40,000 young L1 scattered throughout the genome, covering about 100,000 CpG sites and is based on a reference-free analysis pipeline.


RESULTS
Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers (colorectal, breast, lung, ovarian, gastric cancers and uveal melanoma including localized stages) in two independent cohorts (AUC = 88% to 100%, N = 747). DIAMOND can also be used to perform copy number alterations (CNA) analysis which improves cancer detection.


CONCLUSIONS
This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321c21ecf5626d3880de9fbd35f995ec3b32a1f7" target='_blank'>
              Non-invasive multi-cancer detection using DNA hypomethylation of LINE-1 retrotransposons.
              </a>
            </td>
          <td>
            M. Michel, Maryam Heidary, Anissa Mechri, Kévin Da Silva, Marine Gorse, Victoria Dixon, Klaus von Grafenstein, Charline Bianchi, C. Hego, A. Rampanou, C. Lamy, M. Kamal, C. Le Tourneau, M. Séné, I. Bieche, Cécile Reyes, D. Gentien, Marc-Henri Stern, Olivier Lantz, L. Cabel, J-Y Pierga, F. Bidard, Chloé-Agathe Azencott, C. Proudhon
          </td>
          <td>2024-12-02</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f345edfe89fa16dab1e87b151db2db75ec0ac8dc" target='_blank'>
              Insulin signaling regulates R2 retrotransposon expression to orchestrate transgenerational rDNA copy number maintenance
              </a>
            </td>
          <td>
            Jonathan O. Nelson, Alyssa Slicko, Amelie A Raz, Yukiko M. Yamashita
          </td>
          <td>2025-01-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA, our findings rather suggest that protein-DNA co-condensates ‘sense’ the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d84d86b264f5c1ad8de5c81e2ae317173c400dd" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Thejas Satheesh, Rohit Kumar Singh, Prakshi Gaur, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroendocrine and tuft cells are rare, chemosensory epithelial lineages defined by expression of ASCL1 and POU2F3 transcription factors, respectively1,2. Neuroendocrine cancers, including small cell lung cancer (SCLC), frequently display tuft-like subsets, a feature linked to poor patient outcomes3–13. The mechanisms driving neuroendocrine–tuft tumour heterogeneity, and the origins of tuft-like cancers are unknown. Using multiple genetically-engineered animal models of SCLC, we demonstrate that a basal cell of origin (but not the accepted neuroendocrine origin) generates neuroendocrine–tuft-like tumours that highly recapitulate human SCLC. Single-cell clonal analyses of basal-derived SCLC further uncovers unexpected transcriptional states and lineage trajectories underlying neuroendocrine–tuft plasticity. Uniquely in basal cells, introduction of genetic alterations enriched in human tuft-like SCLC, including high MYC, PTEN loss, and ASCL1 suppression, cooperate to promote tuft-like tumours. Transcriptomics of 944 human SCLCs reveal a basal-like subset and a tuft-ionocyte-like state that altogether demonstrate remarkable conservation between cancer states and normal basal cell injury response mechanisms14–18. Together, these data suggest that the basal cell is a plausible origin for SCLC and other neuroendocrine-tuft cancers that can explain neuroendocrine–tuft heterogeneity—offering new insights for targeting lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a9f4695f47fd01e98ecf709150d28f3a1f7f97" target='_blank'>
              Basal cell of origin resolves neuroendocrine–tuft lineage plasticity in cancer
              </a>
            </td>
          <td>
            A. Ireland, Sarah B. Hawgood, Daniel A. Xie, Margaret W. Barbier, Scarlett Lucas-Randolph, Darren R Tyson, Lisa Y. Zuo, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Sonam Puri, Charles M. Rudin, J. Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e175d7f8eb691015e68d15e186cd0072f7b5db" target='_blank'>
              hnRNPU-mediated pathogenic alternative splicing drives gastric cancer progression
              </a>
            </td>
          <td>
            Guoguo Jin, Yanming Song, Shaobo Fang, Mingyang Yan, Zhaojie Yang, Yang Shao, Kexin Zhao, Meng Liu, Zhenwei Wang, Zhiping Guo, Zigang Dong
          </td>
          <td>2025-01-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia frequently harbor fusion oncogenes associated with poor prognosis, including KMT2A, NUP98 and GLIS2 rearrangements. While murine models have demonstrated their leukemogenic activities, the steps from a normal human cell to leukemic blasts remain unclear. Here, we precisely reproduced the inversion of chromosome 16 resulting in ETO2::GLIS2 fusion in human induced pluripotent stem cells (iPSC). IPSC-derived ETO2::GLIS2-expressing hematopoietic cells showed differentiation alterations in vitro and efficiently induced in vivo development of leukemia that closely phenocopied human acute megakaryoblastic leukemia (AMKL) reflected by flow cytometry and single cell transcriptomes. Comparison of iPS-derived cells with patient-derived cells revealed altered chromatin accessibility at early and later bona fide leukemia stages with aberrantly higher accessibility and expression of the osteogenic homeobox factor DLX3 that preceded increased accessibility to ETS factors. DLX3 overexpression in normal CD34+ cells increased accessibility to ETS motifs and reduced accessibility to GATA motifs. A DLX3 transcriptional module was globally enriched in both ETO2::GLIS2 AMKL and some aggressive pediatric osteosarcoma. Importantly, DLX3 knock-out abrogated leukemia initiation in this ETO2::GLIS2 iPSC model. Collectively, characterization of a novel human iPSC-derived AMKL model revealed hijacking of the osteogenic homeobox transcription factor DLX3 as an essential early step in chromatin changes and leukemogenesis driven by the ETO2::GLIS2 fusion oncogene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d639e01bc6f7f80f46142030b31449774684dd" target='_blank'>
              Progressive chromatin rewiring by ETO2::GLIS2 revealed in a human iPSC model of pediatric leukemia initiation.
              </a>
            </td>
          <td>
            Fabien Boudia, Marie-Laurence Baille, Loélia Babin, Zakia Aid, E. Robert, Julie Riviere, Klaudia Galant, Verónica Alonso-Pérez, Laura Anselmi, Brahim Arkoun, N. Abermil, Christophe Marzac, S. Bertuccio, Alexia Regnault de Premesnil, Cécile K. Lopez, Alexandre Eeckhoutte, Audrey Naimo, B. Leite, C. Catelain, Christophe Metereau, Patrick Gonin, Nathalie Gaspar, J. Schwaller, Olivier A Bernard, H. Raslova, Murielle Gaudry, A. Marchais, Hélène Lapillonne, Arnaud Petit, Françoise Pflumio, M. Arcangeli, Erika Brunet, T. Mercher
          </td>
          <td>2024-12-10</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Emerging evidence indicates that aberrations in the writing, reading, or erasure of chromatin modification codes are pivotal events in various human cancers. In this study, we conducted a systematic investigation of histone modification recognition proteins in a pair of colorectal cancer cell lines, SW480 and SW620. Using chromatin fractionation combined with data-independent acquisition mass spectrometry (DIA-MS), we developed a robust method to identify changes in histone modification recognition proteins during cancer progression. Our analysis revealed 22 proteins that were significantly upregulated and 22 proteins that were significantly downregulated in SW620 cells compared to SW480 cells. Notably, SMARCA1, a member of the ISWI family belonging to ATP-dependent chromatin remodeling complexes, was downregulated in SW620 cells compared to SW480 cells. Its high expression was strongly correlated with poor patient prognosis, aligning with the proposed role of SMARCA1 in promoting colorectal cancer (CRC) metastasis. Furthermore, reduced SMARCA1 expression altered the levels of metastasis-related matrix metalloproteinases (MMPs) in these cells. In conclusion, by systematically profiling histone modification recognition proteins in a matched pair of primary and metastatic CRC cell lines, we identified SMARCA1 as a potential driver of CRC metastasis and a promising therapeutic target for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43b84672f56bb75e7d699d7465b27140159f571" target='_blank'>
              Identification of SMARCA1 as a key regulator for Colorectal Cancer
              </a>
            </td>
          <td>
            Chunqing Fu, Shoufeng Duan, Chengwen Zhang, Xinyu Cui, Fengxian Wang, Lichen Wang, Jinglei Hu, Tengjiao Li, Lin Li
          </td>
          <td>2024-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557911aa9625d7bd95b52abd71f3273a6c915b9d" target='_blank'>
              Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in cancer.
              </a>
            </td>
          <td>
            Yang Yang, Michelle L Badura, Patrick C O'Leary, Henry Delavan, Troy M Robinson, Emily A Egusa, Xiaoming Zhong, Jason Swinderman, Haolong Li, Meng Zhang, Minkyu Kim, Alan Ashworth, Felix Y. Feng, Jonathan Chou, Lixing Yang
          </td>
          <td>2024-11-18</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12abdeefb14dd8084b12c06a67b90a6d3dae88f" target='_blank'>
              CPNE7 promotes colorectal tumorigenesis by interacting with NONO to initiate ZFP42 transcription
              </a>
            </td>
          <td>
            Liangbo Zhao, Xiao Sun, Chenying Hou, Yanmei Yang, Peiwen Wang, Zhaoyuan Xu, Zhenzhen Chen, Xiangrui Zhang, Guanghua Wu, Hong Chen, Hao Xing, Huimin Xie, Luyun He, Shuiling Jin, Benyu Liu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cells exhibit a remarkable resilience to cytotoxic stress, often adapting through transcriptional changes linked to alterations in chromatin structure. In several types of cancer, these adaptations involve epigenetic modifications and restructuring of topologically associating domains (TADs). However, the underlying principles by which chromatin architecture facilitates such adaptability across different cancers remain poorly understood. To investigate the role of chromatin in this process, we developed a physics-based mechanistic model that connects chromatin organization to cell fate decisions, specifically survival following chemotherapy. Our model builds on the observation that chromatin forms packing domains, which influence transcriptional efficiency through macromolecular crowding. The model accurately predicts chemoevasion in vitro, suggesting that changes in packing domains affect the likelihood of survival. Consistent results across diverse cancer types indicate that the model captures fundamental principles of chromatin-mediated adaptation, independent of the specific cancer or chemotherapy mechanisms involved. Based on these insights, we hypothesized that compounds capable of modulating packing domains, termed Transcriptional Plasticity Regulators (TPRs), could prevent cellular adaptation to chemotherapy. Using live-cell chromatin imaging, we conducted a compound screen that identified several TPRs which synergistically enhanced chemotherapyinduced cell death. The most effective TPR significantly improved therapeutic outcomes in a patient-derived xenograft (PDX) model of ovarian cancer. These findings underscore the central role of chromatin in cellular adaptation to cytotoxic stress and present a novel framework for enhancing cancer therapies, with broad potential across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35f61f79981a14cfb46f8ab43779aba13110922" target='_blank'>
              Leveraging chromatin packing domains to target chemoevasion in vivo
              </a>
            </td>
          <td>
            J. Frederick, R. Virk, I. C. Ye, L. Almassalha, Greta Wodarcyk, D. VanDerway, Paola Carrillo Gonzalez, Rikkert J. Nap, Vasundhara Agrawal, Nicholas M. Anthony, John Carinato, Wing Shun Li, Cody L Dunton, Karla I Medina, Rivaan Kakkaramadam, Surbhi Jain, Shohreh Shahabi, Guillermo Ameer, I. Szleifer, Vadim Backman
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="SUMMARY DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild‐type Arabidopsis thaliana somatic cells exposed either to UV‐B, to UV‐C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia‐telangiectasia‐mutated) and ATR (Ataxia‐telangiectasia‐mutated and Rad3‐related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations‐induced deleted regions allowed determining that exposure to UV‐B, UV‐C and protons induced the microhomology‐mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations‐induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract The p53 family of transcription factors (p53, p63 and p73) regulate diverse organismal processes including tumor suppression, maintenance of genome integrity and the development of skin and limbs. Crosstalk between transcription factors with highly similar DNA binding profiles, like those in the p53 family, can dramatically alter gene regulation. While p53 is primarily associated with transcriptional activation, p63 mediates both activation and repression. The specific mechanisms controlling p63-dependent gene regulatory activity are not well understood. Here, we use massively parallel reporter assays (MPRA) to investigate how local DNA sequence context influences p63-dependent transcriptional activity. Most regulatory elements with a p63 response element motif (p63RE) activate transcription, although binding of the p63 paralog, p53, drives a substantial proportion of that activity. p63RE sequence content and co-enrichment with other known activating and repressing transcription factors, including lineage-specific factors, correlates with differential p63RE-mediated activities. p63 isoforms dramatically alter transcriptional behavior, primarily shifting inactive regulatory elements towards high p63-dependent activity. Our analysis provides novel insight into how local sequence and cellular context influences p63-dependent behaviors and highlights the key, yet still understudied, role of transcription factor paralogs and isoforms in controlling gene regulatory element activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d1ad5efee3b5666ece1f29c9beb73579a9fd3a9" target='_blank'>
              Crosstalk between paralogs and isoforms influences p63-dependent regulatory element activity
              </a>
            </td>
          <td>
            Gabriele Baniulyte, Abby A. McCann, Dana L. Woodstock, Morgan A. Sammons
          </td>
          <td>2024-11-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff1090b04839914362d5c68932f9ab6b30614f" target='_blank'>
              Emerging roles of cohesin-STAG2 in cancer.
              </a>
            </td>
          <td>
            Julia S Scott, Loubna Al Ayadi, Emmanouela Epeslidou, Roan H van Scheppingen, Anna Mukha, Lucas J T Kaaij, C. Lutz, S. Prekovic
          </td>
          <td>2024-11-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), a highly aggressive brain tumor, frequently develops resistance to temozolomide (TMZ), the current standard chemotherapy. Our study investigates the potential of combining TMZ with the poly (ADP-ribose) polymerase inhibitor Olaparib (OLA) to overcome TMZ resistance. Using in vitro models, including U251 cell lines and patient-derived GBM primary cultures, we demonstrate that OLA enhances TMZ efficacy by disrupting base excision repair and potentiating DNA damage-induced cytotoxicity. A CRISPR knockout screen identified DNA mismatch repair (MMR) gene deficiencies as key drivers of TMZ resistance, which OLA effectively counteracts. Co-treatment with TMZ and OLA significantly reduced tumor cell viability, even in MMR-deficient and MGMT-expressing contexts, suggesting a synergistic mechanism. Gene expression analysis revealed that the combination therapy impacts cell cycle regulation, stress responses, and extracellular matrix integrity, leading to mitotic catastrophe and apoptosis. These findings propose the TMZ-OLA combination as a promising therapeutic strategy for overcoming chemoresistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edc2d11f92b839420cb397da78a6ec7ab2aa82c" target='_blank'>
              PARP inhibitor counteracts Temozolomide Resistance in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Edouard Samarut, Tabourel Gaston, Briand Joséphine, Landais Yuna, Gratas Catherine, Cartron Pierre-François, Vallette François, Oliver Lisa, Sérandour A. Aurélien
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A central goal of cancer genomics is to identify, in each patient, all the cancer-driving mutations. Among them, point mutations are referred to as cancer-driving nucleotides (CDNs), which recur in cancers. The companion study shows that the probability of i recurrent hits in n patients would decrease exponentially with i; hence, any mutation with i ≥ 3 hits in The Cancer Genome Atlas (TCGA) database is a high-probability CDN. This study characterizes the 50–150 CDNs identifiable for each cancer type of TCGA (while anticipating 10 times more undiscovered ones) as follows: (i) CDNs tend to code for amino acids of divergent chemical properties. (ii) At the genic level, far more CDNs (more than fivefold) fall on noncanonical than canonical cancer-driving genes (CDGs). Most undiscovered CDNs are expected to be on unknown CDGs. (iii) CDNs tend to be more widely shared among cancer types than canonical CDGs, mainly because of the higher resolution at the nucleotide than the whole-gene level. (iv) Most important, among the 50–100 coding region mutations carried by a cancer patient, 5–8 CDNs are expected but only 0–2 CDNs have been identified at present. This low level of identification has hampered functional test and gene-targeted therapy. We show that, by expanding the sample size to 105, most CDNs can be identified. Full CDN identification will then facilitate the design of patient-specific targeting against multiple CDN-harboring genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb003947f43738ae589b3a6538f7d27fb518829" target='_blank'>
              Characterization of cancer-driving nucleotides (CDNs) across genes, cancer types, and patients
              </a>
            </td>
          <td>
            Lingjie Zhang, Tong Deng, Zhongqi Liufu, Xiangnyu Chen, Shijie Wu, Xueyu Liu, Changhao Shi, Bingjie Chen, Zheng Hu, Qichun Cai, Chenli Liu, Mengfeng Li, Miles E. Tracy, Xuemei Lu, Chung-I Wu, Hai-Jun Wen
          </td>
          <td>2024-12-17</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="TNK2 is a ubiquitously expressed nonreceptor-type tyrosine kinase. TNK2 participates in tumorigenesis, and TNK2 activation has been found in various cancers; therefore, TNK2 is a promising target for cancer chemotherapy. While the TNK2 inhibitor XMD16-5 is highly selective, it inhibits cytokinesis at higher concentrations by targeting Aurora B kinase, a key enzyme for cell division. Cytokinesis failure frequently generates polyploid cells, and the surviving polyploid cells risk leading to cancer development and malignant progression via chromosome instability. In this study, to investigate the possibility that (R)-9bMS, a TNK2 inhibitor structurally related to XMD16-5, drives malignant progression by inducing abnormal cell division, we examined its effects on cell division, Aurora B autophosphorylation, and colony formation. Cell count results showed a reduction in the number of A431, HeLa S3, HCT116, and MCF7 cells upon TNK2 inhibitor treatment. Microscopic observation indicated the formation of multinucleated and nucleus-enlarged cells. An increase in DNA content was confirmed with flow cytometry, which was underpinned by an increased number of centrosomes. Time-lapse imaging revealed mitotic failure, such as mitotic slippage and cytokinesis failure, as a cause of polyploidization. Of note, TNK2 knockdown significantly increased multinucleated cells, but the effect was quite weak, suggesting that TNK2 inhibition may only partially contribute to mitotic failure and polyploidization. Expectedly, Aurora B phosphorylation was reduced by (R)-9bMS like XMD16-5, but not by TNK2 knockdown. Collectively, TNK2 inhibitors (R)-9bMS and XMD16-5 induce polyploidization via mitotic failure caused by the inhibition of Aurora B kinase rather than TNK2. Notably, (R)-9bMS treatment promoted anchorage-independent colony formation, a hallmark of cancer. Our findings suggest that (R)-9bMS at a high concentration risks promoting cancer development or malignant progression. Therefore, caution should be used when using TNK2 inhibitors for cancers where TNK2 activation is not the transforming mutation and higher concentrations of TNK2 inhibitors are required to slow proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83021c8768a54cea8919a992b5bb8d7150bf2fc1" target='_blank'>
              TNK2 Inhibitor (R)-9bMS Causes Polyploidization Through Mitotic Failure by Targeting Aurora B.
              </a>
            </td>
          <td>
            Mayu Murata, Hiroki Kuwajima, Junna Tanaka, Nanami Hasegawa, Ryuzaburo Yuki, Youhei Saito, Yuji Nakayama
          </td>
          <td>2024-12-01</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, Nour H Marzouk, Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-06</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks as the third most prevalent cancer globally and is the second leading cause of cancer mortality. FAM49B, a member of the FAM49 gene family, is a recently identified, evolutionarily conserved gene. Emerging studies indicate that FAM49B plays a role in various cancers, though its specific mechanism in CRC remains largely unexplored. In this study, we observed that FAM49B was abnormally expressed in CRC tissues and cell lines, with elevated expression correlating with poor patient prognosis. FAM49B knockdown markedly suppressed CRC cell proliferation by arresting the cell cycle and reducing cell migration and invasion. Single-cell RNA-seq (ScRNA-seq) analysis revealed that high FAM49B expression in malignant epithelial cell clusters was strongly linked to c-Myc oncogene activation. Further, FAM49B knockdown significantly reduced c-Myc expression by enhancing its K48 ubiquitination. We identified NEK9 as a direct interacting partner of FAM49B, with FAM49B knockdown inhibiting NEK9-Thr210 phosphorylation. Similarly, high NEK9 expression was linked to unfavorable prognosis in CRC. In FAM49B-overexpressing CRC cells, NEK9 knockdown significantly suppressed c-Myc expression, c-Myc-ser62 phosphorylation, and reduced cell proliferation, migration, and invasion. Thus, directly targeting the FAM49B/NEK9/c-Myc pathway presents a promising therapeutic approach for c-Myc positive CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aecd72da94ed813728ad7c7716f5f1747f110c" target='_blank'>
              FAM49B drives colorectal cancer progression by stabilizing c-Myc through NEK9 phosphorylation.
              </a>
            </td>
          <td>
            Chen Lu, Tianyu Liu, E. Yimin, Lin Miao, Chunzhao Yu, Jianping Zhang, Xiagang Luo
          </td>
          <td>2025-01-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA replication in humans requires precise regulation to ensure accurate genome duplication and maintain genome integrity. A key indicator of this regulation is replication timing, which reflects the interplay between origin firing and fork dynamics. We present a high-resolution (1-kilobase) mathematical model that maps firing rate distributions to replication timing profiles across various cell lines, validated using Repli-seq data. The model effectively captures genome-wide replication patterns while identifying local discrepancies. Notably, regions where the model and data diverge often overlap with fragile sites and long genes, highlighting the influence of genomic architecture on replication dynamics. Conversely, regions of high concordance are associated with open chromatin and active promoters, where elevated firing rates facilitate timely fork progression and reduce replication stress. By establishing these correlations, our model provides a valuable framework for exploring the structural interplay between replication timing, transcription, and chromatin organisation, offering new insights into mechanisms underlying replication stress and its implications for genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00336bf19aa7ef4b1f6f04b70f3ff986994f75fa" target='_blank'>
              DNA replication timing reveals genome-wide features of transcription and fragility
              </a>
            </td>
          <td>
            Francis X Berkemeier, Peter R. Cook, M. Boemo
          </td>
          <td>2024-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="STIL is a regulatory protein essential for centriole biogenesis, and its dysregulation has been implicated in various diseases, including malignancies. However, its role in non-small-cell lung carcinoma (NSCLC) remains unclear. In this study, we examined STIL expression and its potential association with chromosomal numerical abnormalities (CNAs) in NSCLC using The Cancer Genome Atlas (TCGA) dataset, immunohistochemical analysis, and in vitro experiments with NSCLC cell lines designed to overexpress STIL. TCGA data revealed upregulated STIL mRNA expression in lung adenocarcinoma (LUAD) and lung squamous cell carcinoma (LUSC), the two major subtypes of NSCLC. Immunohistochemical analysis of cases from our hospital (LUAD, n = 268; LUSC, n = 98) revealed STIL protein overexpression. To elucidate the functional role of STIL, an inducible STIL-overexpressing H1299 NSCLC cell line was generated. Overexpression of STIL in these cells promoted centrosome amplification, leading to chromosomal instability. Finally, analysis of arm-level chromosomal copy number alterations from the TCGA dataset revealed that elevated STIL mRNA expression was associated with CNAs in both LUAD and LUSC. These findings suggest that STIL overexpression is associated with CNAs in NSCLC, likely through centrosome amplification, which is linked to chromosomal instability and might represent a potential therapeutic target for NSCLC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/295e6446f450cae4badff7db3db7985be75e6bc3" target='_blank'>
              STIL Overexpression Is Associated with Chromosomal Numerical Abnormalities in Non-Small-Cell Lung Carcinoma Through Centrosome Amplification
              </a>
            </td>
          <td>
            Shunsuke Ohtsuka, Hisami Kato, Rei Ishikawa, Hirofumi Watanabe, Ryosuke Miyazaki, Shin-ya Katsuragi, Katsuhiro Yoshimura, Hidetaka Yamada, Yasuhiro Sakai, Y. Inoue, Yusuke Takanashi, Keigo Sekihara, K. Funai, H. Sugimura, Kazuya Shinmura
          </td>
          <td>2024-12-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053c90f2e024c025545d0a836542f7f28e9a61c7" target='_blank'>
              KLF5 loss sensitizes cells to ATR inhibition and is synthetic lethal with ARID1A deficiency
              </a>
            </td>
          <td>
            Samah W. Awwad, Colm Doyle, Josie Coulthard, Aldo S. Bader, Nadia Gueorguieva, Simon Lam, Vipul Gupta, R. Belotserkovskaya, Tuan-Anh Tran, Shankar Balasubramanian, S. P. Jackson
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Pseudouridine (Ψ) is a post-transcriptional modifier of RNA, often referred to as the ‘fifth nucleotide’ owing to its regulatory role in various biological functions as well as because of its significant involvement in the pathogenesis of human cancer. In recent years, research has revealed various Ψ modifications in different RNA types, including messenger RNA, transfer RNA, ribosomal RNA, small nuclear RNA, and long noncoding RNA. Pseudouridylation can significantly alter RNA structure and thermodynamic stability, as the Ψ-adenine (A) base pair is more stable than the typical uridine (U)-A base pair is due to its structural similarity to adenine. Studies have linked Ψ expression to the development and progression of several digestive system cancers, such as liver cancer and colorectal cancer, and nondigestive system cancers, such as breast cancer, non-small cell lung cancer, prostate cancer, glioblastoma, ovarian cancer, oral squamous cell carcinoma, and pituitary cancer. The present review briefly outlines the chemical structure, synthesis, and regulatory mechanisms of Ψ. This review summarizes the effects of pseudouridylation on various substrates of RNA and briefly discusses methods for detecting Ψ. Last, it focuses on how RNA pseudouridylation influences different cancers, emphasizing the search for novel approaches to cancer diagnosis, treatment, and prognosis through Ψ modification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9466c719e8cf6ff18a146061c808211210df53e3" target='_blank'>
              Advancements in pseudouridine modifying enzyme and cancer
              </a>
            </td>
          <td>
            Kaijie Liu, Shujun Zhang, Yafeng Liu, Xinjun Hu, Xinyu Gu
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/934d3abd576e91e9afbce02b03dd982277191339" target='_blank'>
              Mitochondria-localized MBD2c facilitates mtDNA transcription and drug resistance.
              </a>
            </td>
          <td>
            Yijie Hao, Zilong Zhou, Rui Liu, Shengqi Shen, Haiying Liu, Yingli Zhou, Yuchen Sun, Qiankun Mao, Tong Zhang, Shiting Li, Zhaoji Liu, Yiyang Chu, Linchong Sun, P. Gao, Huafeng Zhang
          </td>
          <td>2024-11-28</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8117cfd933320eeb9a62cf47bdb749f3f59bcf32" target='_blank'>
              Normal breast tissues harbour rare populations of aneuploid epithelial cells.
              </a>
            </td>
          <td>
            Yiyun Lin, Junke Wang, Kaile Wang, S. Bai, A. Thennavan, Runmin Wei, Yun Yan, Jianzhuo Li, Heba Elgamal, Emi Sei, Anna K. Casasent, Mitchell Rao, Chenling Tang, A. Multani, Jin Ma, Jessica Montalvan, C. Nagi, Sebastian Winocour, Bora Lim, Alastair Thompson, Nicholas Navin
          </td>
          <td>2024-11-20</td>
          <td>Nature</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be789fb687d3f78efa3b5343344eaabbca303d35" target='_blank'>
              Chromatin licensing and DNA replication factor 1 as a potential prognostic and diagnostic biomarker for gastric cancer
              </a>
            </td>
          <td>
            Sijia Wang, Min Cui, Na Zhou, Meichen Tong, Shuhua Wu, Hong Li, Zhang Cao
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Targeting DNA damage response (DDR) pathways represents one of the principal approaches in cancer therapy. However, defects in DDR mechanisms, exhibited by various tumors, can also promote tumor progression and resistance to therapy, negatively impacting patient survival. Therefore, identifying new molecules from natural extracts could provide a powerful source of novel compounds for cancer treatment strategies. In this context, we investigated the role of oleanolic acid (OA), identified in fermented Aglianico red grape pomace, in modulating the DDR in response to camptothecin (CPT), an inhibitor of topoisomerase I. Specifically, we found that OA can influence the choice of DNA repair pathway upon CPT treatment, shifting the repair process from homologous recombination gene conversion to single-strand annealing. Moreover, our data demonstrate that combining sub-lethal concentrations of OA with CPT enhances the efficacy of topoisomerase I inhibition compared to CPT alone. Overall, these findings highlight a new role for OA in the DDR, leading to a more mutagenic DNA repair pathway and increased sensitivity in the HeLa cancer cell line.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba28716fad27e1778281b97439ea3f957928277" target='_blank'>
              Oleanolic Acid Modulates DNA Damage Response to Camptothecin Increasing Cancer Cell Death
              </a>
            </td>
          <td>
            Giulio Mazzarotti, Maria Cuomo, Maria Carmen Ragosta, Andrea Russo, Margherita D’Angelo, Annamaria Medugno, Giuseppe Maria Napolitano, C. A. Iannuzzi, I. Forte, Rosa Camerlingo, Sharon Burk, F. Errichiello, Luigi Frusciante, M. Forino, M. R. Campitiello, M. de Laurentiis, A. Giordano, Luigi Alfano
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e87243dcc8a034ecc3e29fa0ee06621055ecda9" target='_blank'>
              Engrailed 2 facilitates progression of triple-negative and HER2-enriched breast cancer by binding to enhancer region of Tenascin-C
              </a>
            </td>
          <td>
            Dandan Chen, Rongping Yin
          </td>
          <td>2024-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The human genome is packaged within a three-dimensional (3D) nucleus and organized into structural units known as compartments, topologically associating domains (TADs), and loops. TAD boundaries, separating adjacent TADs, have been found to be well conserved across mammalian species and more evolutionarily constrained than TADs themselves. Recent studies show that structural variants (SVs) can modify 3D genomes through the disruption of TADs, which play an essential role in insulating genes from outside regulatory elements' aberrant regulation. However, how SV affects the 3D genome structure and their association among different aspects of gene regulation and candidate cis-regulatory elements (cCREs) have rarely been studied systematically. Here, we assess the impact of SVs intersecting with TAD boundaries by developing an integrative Hi-C analysis pipeline, which enables the generation of an in-depth catalog of TADs and TAD boundaries in human lymphoblastoid cell lines (LCLs) to fill the gap of limited resources. Our catalog contains 18,865 TADs, including 4596 sub-TADs, with 185 SVs (TAD-SVs) that alter chromatin architecture. By leveraging the ENCODE registry of cCREs in humans, we determine that 34 of 185 TAD-SVs intersect with cCREs and observe significant enrichment of TAD-SVs within cCREs. This study provides a database of TADs and TAD-SVs in the human genome that will facilitate future investigations of the impact of SVs on chromatin structure and gene regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe7b730dd8aa8992f72d3009d5b16854b631e1b" target='_blank'>
              An integrative TAD catalog in lymphoblastoid cell lines discloses the functional impact of deletions and insertions in human genomes.
              </a>
            </td>
          <td>
            Chong Li, M. Bonder, Sabriya Syed, Matthew Jensen, Mark B. Gerstein, Michael C Zody, Mark J. P. Chaisson, M. Talkowski, T. Marschall, Jan O. Korbel, E. Eichler, Charles Lee, Xinghua Shi
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Aneuploidy, characterized by imbalanced chromosome numbers, is a hallmark of cancer and is strongly correlated with lethal progression in prostate cancer. Our recent work identified that gains of chromosome 8q, the most frequently gained chromosome in aneuploidy prostate cancers, drive cancer progression, partially through the cohesin RAD21 gene located on the chr8q24 region. We demonstrated that increased RAD21 expression accelerates lethal progression and oncogenesis in both prostate cancer and Ewing sarcoma by alleviating oncogenic stress during early-stage oncogenesis in both cell line and organoid models. Therefore, we hypothesize that elevated RAD21 levels could enable cancer cells to evade immune surveillance by mitigating oncogenic stress and DNA damage. In this study, we developed isogenic prostate and breast cancer cell culture models with varying levels of RAD21. Using a natural killer (NK) cell co-culture assay, we observed that cells with elevated RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited increased survivorship when co-cultured with NK cells. These findings align with publicly available data showing that tumors with high RAD21 levels have reduced NK cell infiltration in prostate cancer. Our results suggest that increased RAD21 expression, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Understanding the role of RAD21 in immune evasion could reveal novel therapeutic strategies targeting immune surveillance mechanisms in prostate cancer.
 Citation Format: Elise G. DeArment, Ruoxi W. Wang, Kate Lu, Xiaofeng A. Su, Andrew Elliott, Nicholas A. Zorko, Justin H. Hwang, Xiaofeng A. Su. Aneuploidy-associated cohesin RAD21 gains promote immune evasion in prostate cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e04861d70a2d0fe00d7781f4f4827f407b579d8" target='_blank'>
              Abstract B021: Aneuploidy-associated cohesin RAD21 gains promote immune evasion in prostate cancers
              </a>
            </td>
          <td>
            Elise G. DeArment, Ruoxi W. Wang, Kate Lu, Xiaofeng A. Su, A. Elliott, Nicholas A Zorko, Justin H. Hwang
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The natural ends of chromosomes resemble double-strand breaks (DSBs), which would activate the DNA damage response (DDR) pathway without the protection provided by a specialized protein complex called shelterin. Over the past decades, extensive research has uncovered the mechanism of action and the high degree of specialization provided by the shelterin complex to prevent aberrant activation of DNA repair machinery at chromosome ends in somatic cells. However, recent findings have revealed striking differences in the mechanisms of end protection in stem cells compared to somatic cells. In this review, we discuss what is known about the differences between stem cells and somatic cells regarding chromosome end protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64f88771a224944498f40c132aea82d1a6ba4a" target='_blank'>
              Telomere Protection in Stem Cells.
              </a>
            </td>
          <td>
            Marta Markiewicz-Potoczny, E. L. Denchi
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Epitranscriptomics, the study of RNA modifications such as N6-methyladenosine (m6A), has emerged as a pivotal field in cancer research. These chemical modifications influence gene expression, protein translation and cellular behavior, driving critical processes like tumor initiation, progression and metastasis. Furthermore, RNA modifications contribute to cancer stem cell plasticity, promoting survival and therapy resistance. Treatment resistance, a major obstacle in cancer therapy, is often driven by aberrant RNA modifications that affect the stability of coding and non-coding RNAs, leading to enhanced DNA repair, drug efflux and immune evasion. As a result, targeting RNA-modifying enzymes has gained attention as a novel therapeutic strategy. Inhibitors of "writers," "erasers" and "readers" of these modifications are currently being explored to restore sensitivity to conventional therapies. This commentary discusses the emerging role of RNA modifications in cancer progression and treatment resistance, highlighting the potential for novel therapeutic interventions in combatting drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/275d0d235e86c5013c4a72496e49c927fb2f4445" target='_blank'>
              Epitranscriptomics in oncology: The double-edged role of RNA modifications in cancer and resistance
              </a>
            </td>
          <td>
            Harsh Nawal
          </td>
          <td>2024-11-15</td>
          <td>International Journal of Recent Innovations in Medicine and Clinical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA repair dysregulation is a key driver of cancer development. Understanding the molecular mechanisms underlying DNA repair dysregulation in cancer cells is crucial for cancer development and therapies. Here, we report that enhancer of zeste homolog 2 (EZH2) directly methylates poly(adenosine diphosphate–ribose) polymerase-1 (PARP-1), an essential enzyme involved in DNA repair, and regulates its activity. Functionally, EZH2-catalyzed methylation represses PARP1 catalytic activity, down-regulates the recruitment of x-ray repair cross-complementing group-1 to DNA lesions and its associated DNA damage repair; on the other hand, it protects the cells from nicotinamide adenine dinucleotide overconsumption upon DNA damage formation. Meanwhile, EZH2-mediated methylation regulates PARP1 transcriptional and oncogenic activity, at least in part, through impairing PARP1-E2F1 interaction and E2F1 transcription factor activity. EZH2 and PARP1 inhibitors synergistically suppress prostate cancer growth. Collectively, our findings uncover an insight of EZH2 functions in fine-tuning PARP1 activity during DNA damage repair and cancer progression, which provides a rationale for combinational targeting EZH2 and PARP1 in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e5d1675eeb186efe28f57213417662345e8e12" target='_blank'>
              EZH2 directly methylates PARP1 and regulates its activity in cancer
              </a>
            </td>
          <td>
            Qingshu Meng, Jiangchuan Shen, Yanan Ren, Qi Liu, Rui Wang, Qiaqia Li, Weihua Jiang, Quan Wang, Yixiang Zhang, Jonathan C Trinidad, Xiaotong Lu, Ting-You Wang, Yanqian Li, Chaehyun Yum, Yang Yi, Yongyong Yang, Dongyu Zhao, C. Harris, S. Kalantry, Kaifu Chen, Rendong Yang, Hengyao Niu, Qi Cao
          </td>
          <td>2024-11-27</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Malignant gliomas are heterogeneous tumors, mostly incurable, arising in the central nervous system (CNS) driven by genetic, epigenetic, and metabolic aberrations. Mutations in isocitrate dehydrogenase (IDH1/2mut) enzymes are predominantly found in low-grade gliomas and secondary high-grade gliomas, with IDH1 mutations being more prevalent. Mutant-IDH1/2 confers a gain-of-function activity that favors the conversion of a-ketoglutarate (α-KG) to the oncometabolite 2-hydroxyglutarate (2-HG), resulting in an aberrant hypermethylation phenotype. Yet, the complete depiction of the epigenetic alterations in IDHmut cells has not been thoroughly explored. Here, we applied an unbiased approach, leveraging epigenetic-focused cytometry by time-of-flight (CyTOF) analysis, to systematically profile the effect of mutant-IDH1 expression on a broad panel of histone modifications at single-cell resolution. This analysis revealed extensive remodeling of chromatin patterns by mutant-IDH1, with the most prominent being deregulation of histone acetylation marks. The loss of histone acetylation occurs rapidly following mutant-IDH1 induction and affects acetylation patterns over enhancers and intergenic regions. Notably, the changes in acetylation are not predominantly driven by 2-HG, can be rescued by pharmacological inhibition of mutant-IDH1, and reversed by acetate supplementations. Furthermore, cells expressing mutant-IDH1 show higher epigenetic and transcriptional heterogeneity and upregulation of oncogenes such as KRAS and MYC, highlighting its tumorigenic potential. Our study underscores the tight interaction between chromatin and metabolism dysregulation in glioma and highlights epigenetic and oncogenic pathways affected by mutant-IDH1-driven metabolic rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1395bf17f297913f9bbe2a4e1db052179fa9f3f" target='_blank'>
              Oncogenic IDH1mut drives robust loss of histone acetylation and increases chromatin heterogeneity.
              </a>
            </td>
          <td>
            N. Furth, Niv Cohen, Avishay Spitzer, T. Salame, B. Dassa, Tevie Mehlman, Alexander Brandis, Arieh Moussaieff, D. Friedmann-Morvinski, Maria G Castro, Jerome Fortin, M. Suvà, I. Tirosh, A. Erez, Guy Ron, E. Shema
          </td>
          <td>2024-12-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract H-DNA is an intramolecular DNA triplex formed by homopurine/homopyrimidine mirror repeats. Since its discovery, the field has advanced from characterizing the structure in vitro to discovering its existence and role in vivo. H-DNA interacts with cellular machinery in unique ways, stalling DNA and RNA polymerases and causing genome instability. The foundational S1 nuclease and chemical probing technologies originally used to show H-DNA formation have been updated and combined with genome-wide sequencing methods for large-scale mapping of secondary structures. There is evidence for triplex H-DNA’s role in polycystic kidney disease (PKD), cancer, and numerous repeat expansion diseases (REDs). In PKD, an H-DNA forming repeat region within the PKD1 gene stalls DNA replication and induces fragility. H-DNA-forming repeats in various genes have a role in cancer; the most well-studied examples involve H-DNA-mediated fragility causing translocations in multiple lymphomas. Lastly, H-DNA-forming repeats have been implicated in four REDs: Friedreich’s ataxia, GAA-FGF14-related ataxia, X-linked Dystonia Parkinsonism, and cerebellar ataxia, neuropathy and vestibular areflexia syndrome. In this review, we summarize H-DNA’s discovery and characterization, evidence for its existence and function in vivo, and the field’s current knowledge on its role in physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d258222649c5f8f00b18b553e105c9ceb00bafd" target='_blank'>
              Triplex H-DNA structure: the long and winding road from the discovery to its role in human disease
              </a>
            </td>
          <td>
            Julia A. Hisey, Chiara Masnovo, S. Mirkin
          </td>
          <td>2024-10-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In mammals, X-linked dosage compensation involves two processes: X-chromosome inactivation (XCI) to balance X chromosome dosage between males and females, and hyperactivation of the remaining X chromosome (Xa-hyperactivation) to achieve X-autosome balance in both sexes. Studies of both processes have largely focused on coding genes and have not accounted for transposable elements (TEs) which comprise 50% of the X-chromosome, despite TEs being suspected to have numerous epigenetic functions. This oversight is due in part to the technical challenge of capturing repeat RNAs, bioinformatically aligning them, and determining allelic origin. To overcome these challenges, here we develop a new bioinformatic pipeline tailored to repetitive elements with capability for allelic discrimination. We then apply the pipeline to our recent So-Smart-Seq analysis of single embryos to comprehensively interrogate whether X-linked TEs are subject to either XCI or Xa-hyperactivation. With regards to XCI, we observe significant differences in TE silencing in parentally driven “imprinted” XCI versus zygotically driven “random” XCI. Chromosomal positioning and genetic background impact TE silencing. We also find that SINEs may influence 3D organization during XCI. In contrast, TEs do not undergo Xa-hyperactivation. Thus, while coding genes are subject to both forms of dosage compensation, TEs participate only in Xi silencing. Evolutionary and functional implications are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b9ae3587e3b90516f521d0a832022aafe22d64b" target='_blank'>
              Dosage compensation of transposable elements in mammals
              </a>
            </td>
          <td>
            Chunyao Wei, Barry A. Kesner, Uri Weissbein, Peera Wasserzug-Pash, Priyojit Das, Jeannie T Lee
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The phosphorylation of histone H2A.X into γH2A.X is a crucial early event in the DNA damage response, marking DNA damage sites and initiating repair processes. While ATM kinase is traditionally recognized as the primary mediator of H2A.X phosphorylation, our study identifies serine/threonine kinase 39 (STK39) as a novel enhancer of this critical signaling pathway. We demonstrate that after DNA damage, STK39 undergoes phosphorylation by the ATM kinase, facilitating its interaction with the Mre11-Rad50-Nbs1 complex and subsequent recruitment to chromatin. This recruitment enables STK39 to further phosphorylate H2A.X, thus amplifying γH2A.X production and promoting homologous recombination repair. Notably, we observe a significant upregulation of STK39 in pancreatic adenocarcinoma (PAAD) tissues, correlating with heightened resistance to PARPi therapy. Furthermore, we demonstrate the synergistic efficacy of combining STK39 inhibition with PARP inhibitors in suppressing and reversing PAAD growth. This study not only provides new insights into the molecular dynamics of H2A.X phosphorylation but also highlights the therapeutic potential of targeting STK39 to enhance PARPi sensitivity in PAAD (created with BioRender).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b5527588649efe249bff697a10bc6cd9dfa473" target='_blank'>
              STK39-mediated amplification of γ-H2A.X promotes homologous recombination and contributes to PARP inhibitor resistance
              </a>
            </td>
          <td>
            Yi Xu, Changying Li, Huan Yin, S. Nowsheen, Xin Xu, Wenjuan Kang, Xin Liu, Lifeng Chen, Zhenkun Lou, Junlin Yi, Min Deng
          </td>
          <td>2024-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067cac427326fdea9459d93348affd2bc0e1346c" target='_blank'>
              Nucleus-translocated GCLM promotes chemoresistance in colorectal cancer through a moonlighting function
              </a>
            </td>
          <td>
            Jin-Fei Lin, Ze-xian Liu, Dong-Liang Chen, , Fen Cao, Kai Yu, Ting Li, Hai-Yu Mo, Hui Sheng, Zhi-Bing Liang, Kun Liao, Yi Han, Shan-Shan Li, Zhao-Lei Zeng, Song Gao, Huai-qiang Ju, Rui-Hua Xu
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Centromere-proximal crossovers are repressed during meiosis across all species. However, there is a strong correlation between aberrant centromeric crossovers (C-COs) and the occurrence of meiotic aneuploidy and disorders such as Down syndrome. Despite decades of work in understanding repression of C-COs, the molecular basis of how they cause chromosomal mis-segregation is unknown. Here, we show that increased C-COs result in improper separation of homologs leading to increased meiosis I errors and aneuploidy in Schizosaccharomyces pombe. C-COs cause nondisjunction events that cannot be explained by random segregation of homologs and additionally, may also disrupt mono-orientation of sister chromatids in meiosis I leading to their premature separation. We provide evidence that the protected population of cohesins at centromeres during meiosis I may interfere with timely resolution of C-COs causing homolog nondisjunction. These molecular insights will improve our understanding of infertility and aneuploidy-associated developmental disorders in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f2a30c0ac6592890cb4c754d0ce4842ca6f026" target='_blank'>
              Centromere-proximal crossovers disrupt proper homologous chromosome disjunction during meiosis
              </a>
            </td>
          <td>
            Sucharita Sen, Sneha S. Sarkar, G. Menon, Mridula Nambiar
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small non-coding RNAs (ncRNAs) that negatively regulate gene expression. MiRNAs regulate fundamental biological processes and have significant roles in several pathologies, including cancer. Cervical cancer is the best-known example of a widespread human malignancy with a demonstrated viral etiology. Infection with high-risk human papillomavirus (hrHPV) has been shown to be a causative factor for cervical carcinogenesis. Despite the occurrence of prophylactic vaccines, highly sensitive HPV diagnostics, and innovative new therapies, cervical cancer remains a main cause of death in developing countries. The relationship between hrHPV infection and cervical cancer depends on the integration of viral DNA to the host genome, disrupting the viral regulator E2 and the continuous production of the viral E6 and E7 proteins, which are necessary to acquire and maintain a transformed phenotype but insufficient for malignant cervical carcinogenesis. Lately, miRNAs, the tumor microenvironment, and immune evasion have been found to be major players in cervical carcinogenesis after hrHPV infection. Many miRNAs have been widely reported as deregulated in cervical cancer. Here, the relevance of miRNA in HPV-mediated transformation is critically reviewed in the context of the natural history of hrHPV infection and cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/803f13a1c54c6112e209a331f8578096768dd866" target='_blank'>
              The Natural History of Cervical Cancer and the Case for MicroRNAs: Is Human Papillomavirus Infection the Whole Story?
              </a>
            </td>
          <td>
            Giovanni Palomino-Vizcaino, Evelyn Gabriela Bañuelos-Villegas, L. M. Álvarez-Salas
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245a4598675a5f03862517e366728182b100458" target='_blank'>
              Molecular mechanism targeting condensin for chromosome condensation.
              </a>
            </td>
          <td>
            Menglu Wang, Daniel Robertson, Juan Zou, Christos Spanos, J. Rappsilber, A. Marston
          </td>
          <td>2024-12-17</td>
          <td>The EMBO journal</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="An ADP‐ribosylating enzyme called poly (ADP–ribose) polymerase 1 (PARP1) is necessary to start several types of DNA repair. PARP1 also contributes significantly to the regulation of gene expression through enzyme‐independent motif recognition and enzyme‐dependent chromatin remodeling. Thus, synthetic lethality is achieved in the therapy of tumors and cancers by blocking undesired DNA repair by the inhibition of its enzyme activity with small molecules. In addition to outlining the most recent research from 2019 to 2024 on the precise ways in which PARP1 regulates each process independently, we discussed how transcription and DNA repair are interdependent enough that perturbations caused by PARP1 enzymatic inhibition, disease‐related enzyme hyperactivation. Herein, this concept review systematically summarizes the different approach for designing selective PARP1 inhibitor that are i) PARP1–DNA trapping approach, ii) hybrid approach, iii) PROTAC approach, and iv) dual target inhibitor approach and updates on clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec80be41f9967e872ba7ea36021cfd284d0a854" target='_blank'>
              Recent Progress in Poly(ADP–Ribose) Polymerase‐1 Inhibitors for Application in Cancer
              </a>
            </td>
          <td>
            Anuradha Mehra, Shahnaz Islam, Rekha Sangwan
          </td>
          <td>2025-01-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In cancer research, the term epigenetics was used in the 1970s in its modern sense encompassing non-genetic events modifying the chromatin state, mainly to oppose the emerging oncogene paradigm. However, starting from the establishment of this prominent concept, the importance of these epigenetic phenomena in cancer rarely led to questioning the causal role of genetic alterations. Only in the last 10 years, the accumulation of problematic data, better experimental technologies, and some ambitious models pushed the idea that epigenetics could be at least as important as genetics in early oncogenesis. Until this year, a direct demonstration of epigenetic oncogenesis was still lacking. Now, Parreno, Cavalli and colleagues, using a refined experimental model in the fruit fly Drosophila melanogaster, enforced the initiation of tumors solely by imposing a transient loss of Polycomb repression, leading to a purely epigenetic oncogenesis phenomenon. Despite a few caveats that we discuss, this pioneering work represents a major breakpoint in cancer research. We are led to consider the theoretical and conceptual implications on oncogenesis and to search for links between this artificial experimental model and naturally occurring processes, while revisiting cancer theories that were previously proposed as alternatives to the oncogene-centered paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f2f2c2f1a7eb766c5cfd45527bf643538142c9e" target='_blank'>
              Evidence of Epigenetic Oncogenesis: A Turning Point in Cancer Research.
              </a>
            </td>
          <td>
            Jean-Pascal Capp, Benoît Aliaga, Vera Pancaldi
          </td>
          <td>2024-11-21</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA is frequently damaged by genotoxic stresses such as ionizing radiation, reactive oxygen species, and nitrogen species. DNA damage is a key contributor to cancer initiation and progression, and thus the precise and timely repair of these harmful lesions is required. Recent studies revealed transcription as a source of genome instability, and transcription-coupled DNA damage has been a focus in cancer research. Impaired mRNA export is closely related to DNA damage through R-loop formation. The molecular machineries of transcription-coupled DNA damage have been extensively analyzed in Saccharomyces cerevisiae. However, the molecular basis of these phenomena in higher eukaryotes remains elusive. In this review, we focus on the relationship between deregulated mRNA export through the transcription-export-2 (TREX-2) complex and cancer development. Particularly, the expression of germinal center-associated nuclear protein (GANP), a molecular scaffold in the TREX-2 complex, is highly associated with tumorigenesis in mice and humans. Although the deregulated expression of other components in the TREX-2 complex might affect cancer development, we have directly demonstrated the significance of GANP in tumorigenesis using genetically modified mice. Additionally, we describe recent evidence for medical applications demonstrating that the downregulation of the other components may be a good candidate for a chemotherapeutic target in terms of reducing the side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e429b3bb884f7cc3d479875de747738fafbdce" target='_blank'>
              Tumorigenesis Caused by Aberrant Expression of GANP, a Central Component in the Mammalian TREX-2 Complex-Lessons from Transcription-Coupled DNA Damages.
              </a>
            </td>
          <td>
            Andri Rezano, Naomi Gondo, Yasuhiro Sakai, Yuko Nakamura, S. Phimsen, Tokio Tani, Akihiko Ito, Seiji Okada, Kazuhiko Kuwahara
          </td>
          <td>2024-12-19</td>
          <td>International journal of molecular sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Endogenous DNA damage occurs throughout the cell cycle, with cells responding differently at various stages. The base excision repair (BER) pathway predominantly repairs damaged bases in the genome. While extensively studied in interphase cells, it is unknown if BER operates in mitosis and how apurinic/apyrimidinic (AP) sites, intermediates in the BER pathway that inhibit transcriptional elongation, are processed for post-mitotic gene reactivation. In this study, using an alkaline comet assay, we demonstrate that BER is inefficient in mitosis and that AP endonuclease 1 (APE1), a key BER enzyme, is required for the repair of damage post-mitosis. We previously demonstrated that APE1 is acetylated (AcAPE1) in the chromatin. Using high-resolution microscopy, we show that AcAPE1 remains associated with specific regions in the condensed chromatin in each of the phases of mitosis. This association presumably occurs via the binding of APE1 to the G-quadruplex structure, a non-canonical DNA structure predominantly present in the transcribed gene regions. Additionally, using a nascent RNA detection strategy, we demonstrate that the knockdown of APE1 delayed the rapid post-mitotic transcriptional reactivation of genes. Our findings highlight the functional importance of APE1 in the mitotic chromosomes to facilitate faster repair of endogenous damage and rapid post-mitotic gene reactivation in daughter cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff8deb0833af4e28b49141ab02f559a559938c63" target='_blank'>
              Base Excision Repair in Mitotic Cells and the Role of Apurinic/Apyrimidinic Endonuclease 1 (APE1) in Post-Mitotic Transcriptional Reactivation of Genes
              </a>
            </td>
          <td>
            Suravi Pramanik, Yingling Chen, K. Bhakat
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa78a29add9f8f7c58ef77a453eb2141f303497" target='_blank'>
              Differentiation signals induce APOBEC3A expression via GRHL3 in squamous epithelia and squamous cell carcinoma
              </a>
            </td>
          <td>
            Nicola J Smith, Ian Reddin, Paige Policelli, Sunwoo Oh, Nur Zainal, Emma Howes, Benjamin Jenkins, Ian Tracy, Mark Edmond, Benjamin Sharpe, Damian Amendra, Ke Zheng, Nagayasu Egawa, John Doorbar, Anjali Rao, Sangeetha Mahadevan, Michael A. Carpenter, R. Harris, Simak Ali, Christopher Hanley, Rémi Buisson, Emma King, Gareth J. Thomas, Tim R. Fenton
          </td>
          <td>2024-11-15</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated oncogenes in cancer and is involved in various tumorigenic processes. Among its diverse roles, KRAS mutations have been shown to upregulate macropinocytosis, a form of endocytosis that allows cells to engulf extracellular fluid and its contained solutes, subsequently supporting the increased need for cancer cells' growth and proliferation. The nutrient uptake function of macropinocytosis was initially described within the framework of KRAS-driven pancreatic tumors. Although constitutive macropinocytosis can be induced by activating mutations of proteins that are commonly found in cancer, KRAS-induced macropinocytosis constitutes a fundamental area of research due to the high occurrence of mutated KRAS in cancer (~1/3 of all cancers).  Furthermore, KRAS-mutated macropinocytosis not only contributes to tumorigenesis but also plays a critical role in developing resistance to treatments, as it was found to be implicated in Multi-Drug Resistance (MDR) in cancer cells. This mini-review aims to synthesize current knowledge of mechanisms of KRAS-mutated macropinocytosis briefly and examines the relationship between KRAS and macropinocytosis in the light of its role in cancer progression and drug resistance, highlighting therapeutic implications, targeting potential vulnerabilities and outlining clinical advancements in pertinent therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4b47f4761d206b1edf858bd1fc0d092897567" target='_blank'>
              The Role of KRAS in Macropinocytosis and its Implications of Multi-Drug Resistance in Cancer
              </a>
            </td>
          <td>
            F. Fenyvesi, Zeinab Ibrahim
          </td>
          <td>2024-12-23</td>
          <td>De Remediis</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Telomere maintenance is crucial for preventing the linear eukaryotic chromosome ends from being mistaken for DNA double-strand breaks, thereby avoiding chromosome fusions and the loss of genetic material. Unlike most eukaryotes that use telomerase for telomere maintenance, Drosophila relies on retrotransposable elements-specifically HeT-A, TAHRE, and TART (collectively referred to as HTT)-which are regulated and precisely targeted to chromosome ends. Drosophila telomere protection is mediated by a set of fast-evolving proteins, termed terminin, which bind to chromosome termini without sequence specificity, balancing DNA damage response factors to avoid erroneous repair mechanisms. This unique telomere capping mechanism highlights an alternative evolutionary strategy to compensate for telomerase loss. The modulation of recombination and transcription at Drosophila telomeres offers insights into the diverse mechanisms of telomere maintenance. Recent studies at the population level have begun to reveal the architecture of telomere arrays, the diversity among the HTT subfamilies, and their relative frequencies, aiming to understand whether and how these elements have evolved to reach an equilibrium with the host and to resolve genetic conflicts. Further studies may shed light on the complex relationships between telomere transcription, recombination, and maintenance, underscoring the adaptive plasticity of telomeric complexes across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc71fe25065e2464b3633178f88c0066a196f35c" target='_blank'>
              Maintaining Telomeres without Telomerase in Drosophila: Novel Mechanisms and Rapid Evolution to Save a Genus.
              </a>
            </td>
          <td>
            S. Cacchione, G. Cenci, Anne-Marie Dion-Côté, D. Barbash, G. Raffa
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The compaction of genomic DNA into nucleosomes with the help of histones had long been considered as a fundamental feature exclusive to eukaryotic cells. However, it was recently shown that archaea and bacteria also encode histones. A complex picture has emerged with more recent discoveries of eukaryotic-like histones within one phylum of double-stranded DNA (dsDNA) viruses. Nevertheless, the extent to which other dsDNA viruses encode histones remains largely unexplored. Here we conducted a metagenomic survey of viral histones that were further clustered based on sequence and predicted structural similarities. We identified over 1,500 viral histones and histone-fold proteins, including previously undescribed proteins found in the viral class Caudoviricetes. Structural predictions and in vitro assays demonstrated that histone triplets (where three histone folds are fused) and singlets co-occurring in the same viral genome are capable of forming nucleosome-like particles. Beyond nucleosomal histone functions, our analysis revealed six types of structurally and functionally diverse viral histone-fold proteins, some of which do not have known structural or functional homologs. Altogether, our findings reveal a previously unrecognized diversity of viral histones in dsDNA viruses, expanding the known repertoire, structural diversity, and functional versatility of viral histones beyond nucleocytoplasmic large DNA viruses. Highlights Histone triplets and singlets co-occurring in the same dsDNA viral genomes assemble into nucleosome-like particles Six novel types of structurally and functionally distinct viral histone-fold proteins are identified The presence of viral histones is significantly positively associated with proteins containing SNF2 chromatin remodeling domain found in viral genomes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c4e9a1d11d7eedf5c70bc4fc7cab1443b14667" target='_blank'>
              Metagenomic mining reveals novel viral histones in dsDNA viruses
              </a>
            </td>
          <td>
            Yang Liu, Zhuru Hou, Wanshan Hao, Shaoqing Cui, Haibo Wang, Yue Liu
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In many eukaryotes, meiotic recombination occurs preferentially at discrete sites, called recombination hotspots. In various lineages, recombination hotspots are located in regions with promoter-like features and are evolutionarily stable. Conversely, in some mammals, hotspots are driven by PRDM9 that targets recombination away from promoters. Paradoxically, PRDM9 induces the self-destruction of its targets and this triggers an ultra-fast evolution of mammalian hotspots. PRDM9 is ancestral to all animals, suggesting a critical importance for the meiotic program, but has been lost in many lineages with surprisingly little effect on meiosis success. However, it is unclear whether the function of PRDM9 described in mammals is shared by other species. To investigate this, we analyzed the recombination landscape of several salmonids, the genome of which harbors one full-length PRDM9 and several truncated paralogs. We identified recombination initiation sites in Oncorhynchus mykiss by mapping meiotic DNA double-strand breaks (DSBs). We found that DSBs clustered at hotspots positioned away from promoters, enriched for the H3K4me3 and H3K36me3 and the location of which depended on the genotype of full-length Prdm9. We observed a high level of polymorphism in the zinc finger domain of full-length Prdm9, indicating diversification driven by positive selection. Moreover, population-scaled recombination maps in O. mykiss, Oncorhynchus kisutch and Salmo salar revealed a rapid turnover of recombination hotspots caused by PRDM9 target motif erosion. Our results imply that PRDM9 function is conserved across vertebrates and that the peculiar evolutionary runaway caused by PRDM9 has been active for several hundred million years.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f09c603402d3b52a99c356601f8730c3715c9b0" target='_blank'>
              PRDM9 drives the location and rapid evolution of recombination hotspots in salmonid fish
              </a>
            </td>
          <td>
            Marie-Laure Raynaud, Paola Sanna, Julien Joseph, J. Clément, Yukiko Imai, J. Lareyre, Audrey Laurent, Nicolas Galtier, F. Baudat, Laurent Duret, P. Gagnaire, B. de Massy
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1113f4b65f7bb6543b590129c22b6e32ee76e8" target='_blank'>
              Transposable element 5mC methylation state of blood cells predicts age and disease.
              </a>
            </td>
          <td>
            Francesco Morandini, Jinlong Y Lu, Cheyenne Rechsteiner, A. Shadyab, Ramon Casanova, Beverly M Snively, A. Seluanov, Vera Gorbunova
          </td>
          <td>2024-11-27</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Introduction Cancer’s inherent heterogeneity, marked by diverse genetic and molecular alterations, presents significant challenges for developing effective treatments. One such alteration is the regulation of disulfidoptosis, a recently discovered programmed cell death pathway. RPN1, a key regulator associated with disulfidoptosis, may influence various aspects of tumor biology, including immune evasion and cellular senescence. This study aims to dissect the role of RPN1 in pan-cancer and its potential as a therapeutic target. Methods We employed a pan-cancer analysis to explore RPN1 expression and its association with clinical outcomes across multiple tumor types. Immune cell infiltration and expression of immune checkpoint genes were analyzed in relation to RPN1. Additionally, cellular senescence markers were assessed in RPN1 knockdown tumor cells. Gene regulatory mechanisms were studied through gene copy number variations, DNA methylation analysis, and transcriptional regulation by SP1. Results RPN1 is overexpressed in a wide range of tumor types and correlates with poor clinical outcomes, including overall survival, disease-specific survival, and progression-free intervals. Our analysis shows that RPN1 is involved in immune evasion, correlating with the presence of myeloid dendritic cells, macrophages, and tumor-associated fibroblasts, and influencing T-cell activity. RPN1 knockdown led to reduced tumor cell proliferation and induced cellular senescence, marked by increased senescence-associated biomarkers and β-galactosidase activity. RPN1 expression was found to be regulated by gene copy number variations, reduced DNA methylation, and transcriptional control via SP1. Discussion These findings highlight RPN1 as a key pan-cancer regulator, influencing immune microenvironment interactions and cellular senescence. The regulation of disulfidoptosis by RPN1 presents a promising avenue for therapeutic intervention. Targeting RPN1 could enhance immunotherapy efficacy and help mitigate tumor progression, offering a potential strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2952a313a9a344ecc3d40275b633ebc51cd3bfe9" target='_blank'>
              Dissecting the functions and regulatory mechanisms of disulfidoptosis-related RPN1 in pan-cancer: modulation of immune microenvironment and cellular senescence
              </a>
            </td>
          <td>
            Lexin Qin, Tingting Liang, Hailong Zhang, Xian Gong, Meidan Wei, Xiangrong Song, Yaoyu Hu, Xinyu Zhu, Wentao Hu, Jianxiang Li, Jin Wang
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Micronuclei are the extra-nuclear chromatin compartments separated from the primary nucleus and surrounded by their own nuclear envelope. For a long time it has been thought that micronuclei is the final stage of the pathological process in a cell. They have been used as biomarkers of the influence of genotoxic factors as well as of genome instability in various diseases. Nowadays, it is demonstrated that micronuclei could be involved in the cellular activities, affect the nuclear genome and lead to the changes in cell and tissue physiology. It is known that the formation of micronuclei is one of the steps in selective chromatin elimination in the ontogenesis of plant and animal species. The regions to be marked and eliminated from cell nucleus are recognized at the level of genome. This process is often accompanied by modifications with the heterochromatin formation, changes in the chromosome condensation and in the position of chromosomes in the nucleus. The processes observed in selective and non-selective chromatin elimination are similar to a great extent. The fact that the role of micronuclei in the cell functioning is not well-known yet, and the composition of the micronuclei and the ways of chromatin elimination could influence their role in the development of the pathogenesis, emphasizes the importance of additional studies for a more profound investigation of this phenomenon.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeddb8a010fe6c42592e816c3b4480c6ec74d17" target='_blank'>
              The role of micronuclei in chromatin elimination
              </a>
            </td>
          <td>
            Yu. R. Akhmadullina, P. O. Khomenko
          </td>
          <td>2024-11-18</td>
          <td>Žurnal obŝej biologii</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Herpes simplex virus 1 (HSV-1) latently infected neurons display diverse patterns in the distribution of the viral genomes within the nucleus. A key pattern involves quiescent HSV-1 genomes sequestered in promyelocytic leukemia nuclear bodies (PML NBs) forming viral DNA-containing PML-NBs (vDCP NBs). Using a cellular model that replicates vDCP NB formation, we previously demonstrated that these viral genomes are chromatinized with the H3.3 histone variant modified on its lysine 9 by trimethylation (H3.3K9me3), a mark associated with transcriptional repression. Here, we identify the HUSH complex and its effectors, SETDB1 and MORC2, as crucial for the acquisition of H3K9me3 on PML NB-associated HSV-1 and the maintenance of HSV-1 transcriptional repression. ChIP-seq analyses show H3K9me3 association with the entire viral genome. Inactivating the HUSH-SETDB1-MORC2 complex before infection significantly reduces H3K9me3 on the viral genome, with minimal impact on the cellular genome, aside from expected changes in LINE-1 retroelements. Depletion of HUSH, SETDB1, or MORC2 alleviates HSV-1 repression in infected primary human fibroblasts and human induced pluripotent stem cell-derived sensory neurons (hiPSDN). We found that the viral protein ICP0 induces MORC2 degradation via the proteasome machinery. This process is concurrent with ICP0 and MORC2 depletion capability to reactivate silenced HSV-1 in hiPSDN. Overall, our findings underscore the robust antiviral function of the HUSH-SETDB1-MORC2 repressor complex against a herpesvirus by modulating chromatin marks linked to repression, thus presenting promising avenues for anti-herpesvirus therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51f3591ca12cf64a45c7109b791fb25e851310" target='_blank'>
              The HUSH epigenetic repressor complex silences PML nuclear body-associated HSV-1 quiescent genomes.
              </a>
            </td>
          <td>
            Simon Roubille, Tristan Escure, F. Juillard, Armelle Corpet, Rémi Néplaz, Olivier Binda, Coline Seurre, M. Gonin, Stuart Bloor, Camille Cohen, Pascale Texier, Oscar Haigh, Olivier Pascual, Yonatan Ganor, Frédérique Magdinier, Marc Labetoulle, Paul J. Lehner, P. Lomonte
          </td>
          <td>2024-11-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer has become the leading cause of death worldwide, and it is the second most common cancer in women and the third most common cancer in men. Accumulating evidence suggests that genetic and epigenetic factors play a key role in the development of colorectal cancer. Cancer Stem Cells (CSC) play an important role in the suppression or development of cancer in various conditions. In recent years, non-coding RNAs (ncRNA) have been the focus, and the association of CSC and non-coding RNA has played a crucial role in the development of human cancers. These non-coding RNAs are known to be expressed in many cancers. Studies have suggested that ncRNAs are dysregulated in colorectal cancer cells, and different factors, like Wnt and Notch, are involved in this dysregulation. ncRNAs play a significant role in cancer initiation, migration, and resistance to therapies. Moreover, long noncoding RNAs are known to regulate tumor suppressor genes or oncogenes. Targeting different ncRNAs like miRNA, circular RNA, long noncoding RNAs, and small interfering RNA may provide efficient, targeted therapeutic strategies for colon cancer treatment. This review aims to briefly discuss the latest findings on the role of noncoding RNAs in the prognosis and diagnosis of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf75cafb36b85765499ce0fa8a4cfb98a12e479" target='_blank'>
              The Role of Noncoding RNAs in the Prognosis and Diagnosis of Colorectal Cancer: An Emerging Biomarker.
              </a>
            </td>
          <td>
            O. S. Vani, Kavitha R Thangaraj, Varshaa Ravichandran, Solomon F D Paul
          </td>
          <td>2024-11-26</td>
          <td>MicroRNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Targeted inhibition of mitochondrial oxidative phosphorylation (OXPHOS) complex generation is an emerging and promising cancer treatment strategy, but limited targets and specific inhibitors have been reported. Leucine-rich pentatricopeptide repeat-containing protein (LRPPRC) is an atypical RNA-binding protein that regulates the stability of all 13 mitochondrial DNA-encoded mRNA (mt-mRNA) and thus participates in the synthesis of the OXPHOS complex. LRPPRC is also a prospective therapeutic target for lung adenocarcinoma, serving as a promising target for OXPHOS inhibition. In this study, we identified Demethylzeylasteral (T-96), a small molecule extracted from the Chinese herb Tripterygium wilfordii Hook. f., as a novel inhibitor of LRPPRC. T-96 directly bound to the RNA-binding domain of LRPPRC, inhibiting its interaction with mt-mRNA. This led to instability in both mt-mRNA and LRPPRC protein. Treatment with T-96 significantly reduced the mRNA and protein levels of the OXPHOS complex. As a consequence of LRPPRC inhibition, T-96 treatment induced a defect in the synthesis of the OXPHOS complex, inhibiting mitochondrial aerobic respiration and ATP synthesis. Moreover, T-96 exhibited potent antitumor activity for lung adenocarcinoma in vitro and in vivo, and the antitumor effect of T-96 was dependent on LRPPRC expression. In conclusion, this study not only identified the first traditional Chinese medicine monomer inhibitor against OXPHOS complex biosynthesis as well as a novel target of Demethylzeylasteral, but also shed light on the unique antitumor mechanism of bioactive compounds derived from traditional Chinese medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4890b41147c58886a09a1ee65796e1fce270586" target='_blank'>
              Demethylzeylasteral inhibits oxidative phosphorylation complex biogenesis by targeting LRPPRC in lung cancer
              </a>
            </td>
          <td>
            Lina Wang, Wei Zhou, Wenxi Wang, Yuxin Liang, Qiqi Xue, Zhen Zhang, Jinghe Yuan, Xiaohong Fang
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089aac455387b5d1dba7549c993d3541b59537a0" target='_blank'>
              Circular RNA circPHLPP2 promotes tumor growth and anti-PD-1 resistance through binding ILF3 to regulate IL36γ transcription in colorectal cancer
              </a>
            </td>
          <td>
            Yan Hu, Ze-Rong Cai, Ren-Ze Huang, De-Shen Wang, Huai-qiang Ju, Dong-Liang Chen
          </td>
          <td>2024-12-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Cells release extracellular vesicles (EVs) loaded with functional biomolecules that facilitate intercellular communication in health and disease. These EVs contain DNA (EV-DNA), mirroring the mutational profile of parental cells, yet the mechanism of DNA packaging onto EVs and its role in diseases like cancer remain unclear. Using advanced imaging and biochemical techniques, we discovered that EV-DNA predominantly resides on the vesicle surface, bound to modified histones. Through genome-wide CRISPR knockout screens in cancer cell lines, we identified immune-related pathways and genes crucial for EV-DNA loading, such as APAF1 and NCF1. In several cancer models, loss of APAF1 reduced EV-DNA packaging, and enhancing metastasis, thus implicating EV-DNA in immune surveillance against cancer spread. We demonstrated that EV-DNA uptake by liver macrophages triggers DNA damage response, altering cytokine production and promoting immune activation in pre-metastatic sites. Moreover, quantifying EV-DNA in colorectal cancer biopsies revealed its potential as a predictive metastatic risk biomarker. Our findings shed light on EV-DNA packaging mechanisms and its significance, suggesting that elevated levels of tumor-derived EV-DNA bolster anti-tumor immunity and inhibit metastasis.
 Citation Format: Inbal Wortzel, Han-Sang Kim, David C. Lyden. DNA associated with EVs is uniquely chromatinized and prevents metastasis by enhancing anti-tumor immunity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edef13099e4837d7d8441f1fcd7734045284c7c9" target='_blank'>
              Abstract PR014: DNA associated with EVs is uniquely chromatinized and prevents metastasis by enhancing anti-tumor immunity
              </a>
            </td>
          <td>
            I. Wortzel, H. Kim, David C. Lyden
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Protein partitioning between the nucleus and the cytoplasm is a defining feature of eukaryotic cells, and it is thought to be primarily regulated by transport across the nuclear envelope1–3. However, during early vertebrate development, ensuring accurate nuclear composition is challenging due to dramatic changes in cell size and rapid cell division4–7. Here, we show that the embryonic linker histone H1.8 is rapidly partitioned into the nucleus through a nuclear import-independent mechanism. In the cytoplasm of Xenopus laevis egg extracts, prior to nuclear assembly, H1.8 condenses on the surface of chromatids as droplets that partially wet them. Simultaneously, H1.8 also forms condensates in the cytoplasm that buffer the condensate nucleation rate on the chromatid surface, resulting in a nuclear partitioning that is independent of the DNA-to-cytoplasm ratio. To show the generality of the surface condensation mechanism, we extended our study to Nucleophosmin 1 (NPM1) in human cell culture. Similar to H1.8, NPM1 forms a layer around chromatids that breaks into droplets that subsequently are incorporated into the nucleus. Our findings show that the properties of condensate wetting and nucleation on chromatid surfaces provide an alternative and robust biophysical mechanism to regulate nuclear composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38c88c21e5cee8fdb8ab6805a2d3aaf9112a170" target='_blank'>
              Nuclear partitioning by surface condensation on metaphase chromatids
              </a>
            </td>
          <td>
            V. Murugesan, Thomas Quail, Maria Elsner, Jan Brugués
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background Chromosomal instability (CIN), a hallmark of cancer, is commonly linked to poor prognosis in high‐grade prostate cancer (PCa). Paradoxically, excessively high levels of CIN may impair cancer cell viability. Consequently, understanding how tumours adapt to CIN is critical for identifying novel therapeutic targets. Methods Bioinformatic analyses were conducted to identify genes overexpressed in PCa tissues using The Cancer Genome Atlas (TCGA) and GEO datasets. Western blotting and immunohistochemistry assays were applied to determine the expression levels of euchromatic histone lysine methyltransferase 2 (EHMT2), pT232‐Aurora B and Cullin 3 (CUL3). The proliferation of cells was measured through CCK‐8 tests, clonogenesis and subcutaneous xenografts of human PCa cells in BALB/c nude mice. Live cell imaging, immunofluorescence (IF) and flow cytometry were used to confirm the role of EHMT2 in PCa cell mitosis. Co‐immunoprecipitation, Western blotting and IF assays further elucidated the underlying molecular mechanisms. Results EHMT2 was highly expressed in metastatic PCa tissues exhibiting elevated CIN and was strongly associated with adverse clinical outcomes in patients with PCa. Silencing EHMT2 impaired cell division, inducing G2/M‐phase arrest and mitotic catastrophe in PCa cells. Mechanistically, EHMT2 is indispensable to ensure the full activation of Aurora B through centromeric R‐loop‐driven ATR–CHK1 pathway, with EHMT2 protein expression peaking during the G2/M‐phase. Moreover, CUL3 was identified as a binding partner of EHMT2, mediating its polyubiquitination and destabilising its protein levels. Conclusions This study reveals a CUL3–EHMT2–Aurora B regulatory axis that safeguards accurate chromosome segregation in PCa cells, supporting the potential therapeutic application of EHMT2 inhibitors. Key points Euchromatic histone lysine methyltransferase 2 (EHMT2) is overexpressed in advanced prostate cancer, restraining catastrophic chromosomal instability (CIN) and enhancing cell fitness. EHMT2 functions via the centromeric R‐loop‐driven ATR–CHK1–Aurora B pathway to promote chromosomal stability. EHMT2 confers enzalutamide resistance via activating Aurora B. Cullin 3 (CUL3) promotes EHMT2 destabilisation via deubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc5ece350ce05d9ec1131335e290684b93468c8" target='_blank'>
              EHMT2‐mediated R‐loop formation promotes the malignant progression of prostate cancer via activating Aurora B
              </a>
            </td>
          <td>
            Yuyang Zhang, Mingqin Su, Yiming Chen, Li Cui, Wei Xia, Renfang Xu, Dong Xue, Xiansheng Zhang, Xingliang Feng
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aea14196d457e30b1e0c0858af5e94a5516dad2" target='_blank'>
              RBM47 promotes cell proliferation and immune evasion by upregulating PDIA6: a novel mechanism of pancreatic cancer progression
              </a>
            </td>
          <td>
            Yihui Ma, Enjie Liu, H. Fan, Chenfei Li, Pei Huang, Meiying Cui, Zhengyang Wang, Jing Zhou, Kuisheng Chen
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed9e928f87fb979a12e3b014048518d7d5d625c" target='_blank'>
              Engineered transcription-associated Cas9 targeting in eukaryotic cells
              </a>
            </td>
          <td>
            Gregory W. Goldberg, M. Kogenaru, Sarah Keegan, Max A B Haase, Larisa Kagermazova, Mauricio A. Arias, Kenenna Onyebeke, Samantha Adams, Daniel K. Beyer, D. Fenyö, M. Noyes, J. Boeke
          </td>
          <td>2024-11-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="
 The majority of the human genome (>70%) contains non-coding DNA consisting of retrotransposable elements (RTE), such as LINE-1, SINE/Alu elements, human endogenous retroviruses (HERV), and satellite repeat DNA (Hsat2 and Hsat3). The expression of these elements is suppressed in normal adult tissues but are abnormally expressed in many cancers. Another key mechanism in cancer initiation and progression is the release of tumor-derived extracellular vesicles (EV) that are long-distance communication vehicles propagating signals from tumors inducing metabolic changes, immune dysregulation and promoting metastases across the body. Our work has uncovered a link between RTE, HERV and Hsat expression in cancer and EV in driving tumor progression and immune dysregulation. W hole transcriptome sequencing of EV derived from several different cancer cell lines and patient samples, including Ewing sarcoma, osteosarcoma, prostate, pancreatic, and breast cancer found that RTE, HERV and Hsat2,3 transcripts are highly enriched in EV versus coding region transcripts and in much higher abundance than in the tumor cell themselves. Tumor-derived EV contained dsRNA, dsDNA and RNA:DNA hybrids from these non-coding regions. Plasma EV isolated from both metastatic and non-metastatic patients also showed an enrichment of these RTE, HERV and Hsat2,3 elements and clearly differentiated healthy, normal donor EV that had little or no detectable signal. Evidence of local dissemination of these non-coding elements in EV was also found in vivo in tumor xenografts that showed transfer of Hsat2,3 transcripts into the murine stroma that naturally lack these satellite repeats. Treatment of fibroblasts and myeloid cells with tumor-derived EV triggered innate immune responses via type I IFN and pro-inflammatory cytokines, including IL-1beta, IL-6, IL-8, and TNFalpha owing to the pathogen-like nucleic acid sequences (viral mimicry) of these elements. These EV-induced chronic inflammatory effects were mainly induced through the cGAS-STING nucleic acid sensing pathway. Evidence of DNA damage in stromal fibroblasts and induced senescence was also found as a result of EV treatment. A key element of RTE is the expression of LINE-1 and HERV derived reverse transcriptase (RT) that propagates the action of RTE via RNA:DNA hybrids, dsDNA and retrotransposition in the genome. In a spontaneous estradiol-driven breast cancer model in ACI rats, long-term treatment with an HIV RT inhibitor (lamivudine/3TC) inhibited breast tumor development associated with reduced RTE activation in the mammary gland and in circulating immune cells as well as reduced chronic systemic innate inflammation. Our results suggest that RTE, HERV and Hsat activation, together with their local and systemic dissemination in EV plays an important role in cancer initiation and progression associated with chronic inflammation. We also suggest that abnormal dissemination of these activated non-coding elements in EV may also play a role in "inflammaging" facilitating not only cancer but also other chronic diseases.
 Citation Format: Laszlo Radvanyi, Valentina Evdokimova, Peter Ruzanov, Zhenbo Zhang, Poul Sorenson, Hendrik Gassmann, Stefan Burdach, Lincoln D. Stein. Dissemination of retrotransposable elements from the non-coding genome in tumor-derived extracellular vesicles target stromal and immune cells to induce local and systemic inflammation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4545e23fd117647dd7aa5ce20199e63c3fd7396" target='_blank'>
              Abstract C012: Dissemination of retrotransposable elements from the non-coding genome in tumor-derived extracellular vesicles target stromal and immune cells to induce local and systemic inflammation
              </a>
            </td>
          <td>
            Laszlo Radvanyi, V. Evdokimova, P. Ruzanov, Zhenbo Zhang, Poul Sorenson, H. Gassmann, Stefan Burdach, Lincoln D. Stein
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Objective: Long non-coding RNAs (lncRNAs) participate in the formation, progression, and metastasis of cancer. This study aimed to explore the roles of the lncRNA ST8SIA6 antisense RNA 1 (ST8SIA6-AS1) in tumorigenesis and elucidate the underlying molecular mechanism of its upregulation in hepatocellular carcinoma (HCC). Material and Methods: A total of 56 in-house pairs of HCC tissues were examined, and ST8SIA6-AS1 levels were determined through real-time polymerase chain reaction (PCR). The biological behavior of ST8SIA6-AS1 by Crispr-Cas9-based gene repression and activation was determined in vitro and in vivo. The binding sites and biological behavior of Myc proto-oncogene and forkhead box A on chromatin were investigated through luciferase reporter assays, chromatin immunoprecipitation–quantitative PCR, and co-immunoprecipitation (co-IP) assays. The regulatory mechanisms of ST8SIA6-AS1 expression were analyzed with encyclopedia of DNA elements and gene expression profiling interactive analysis. Results: The expression of ST8SIA6-AS1 significantly increased in multiple HCC cell lines and the 56 in-house pairs of HCC tissues (P = 0.0018). Functionally, high-efficiency Crispr-Cas9-based knockdown of ST8SIA6-AS1 revealed that ST8SIA6-AS1 knockdown attenuated the proliferation, migration, and infiltration of HCC cells and considerably reduced the growth rate of subcutaneous and orthotopic HCC tumors. Conversely, ST8SIA6-AS1 overexpression considerably improved the oncogenic characteristics of the HCC cells. Furthermore, ST8SIA6-AS1 upregulation was regulated by the direct binding of transcription factor Myc to the −260 bp to+155 bp and +1003 bp to +1312 bp regions of the ST8SIA6-AS1 transcription start site, which is a segment with high level of H3K27 acetylation. Myc knockdown or treatment with the BET bromodomain inhibitor JQ-1 considerably reduced ST8SIA6-AS1 RNA expression in the HCC cells. Conclusion: Our study has established the oncogenic role of ST8SIA6-AS1 and elucidated the Myc-dependent upregulation mechanism of ST8SIA6-AS1 in HCC, providing a profound theoretical molecular basis for the carcinogenic function of ST8SIA6-AS1 in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33616059336d41e344c8cfbb1d3985982730aa02" target='_blank'>
              Crispr-Cas9-based long non-coding RNA interference and activation identified that the aberrant expression of Myc-regulated ST8SIA6 antisense RNA 1 promotes tumorigenesis and metastasis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xueqian Liu, Dong Jiang, Yang Liu, Kun Xie, Yijun Zhao, Fubao Liu
          </td>
          <td>2024-11-25</td>
          <td>CytoJournal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the precise mechanisms by which DDAs induce cell death are not fully understood and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with the sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and downregulating tRNALeu(TAA). Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), leading to ER stress and subsequent cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11 knockout or transfection of tRNALeu(TAA). Our proteomic analysis suggests that tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells, and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs, and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811a0f920ef5bd12afa937061ab8c997f4e1a5be" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, Atsuo T. Sasaki, Akio Kanai
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mismatch repair deficiency leads to high mutation rates and microsatellite instability (MSI-H), associated with immune infiltration and responsiveness to immunotherapies. In early stages, MSI-H tumors generally have a better prognosis and lower metastatic potential than microsatellite-stable (MSS) tumors, especially in colorectal cancer. However, in advanced stages, MSI-H tumors lose this survival advantage for reasons that remain unclear. We developed a syngeneic mouse model of MSI cancer by knocking out the MMR gene Msh2 in the metastatic 4T1 breast cancer cell line. This model mirrored genomic features of MSI-H cancers and showed reduction in metastatic incidence compared to their MSS counterparts. In MSI-H tumors, we observed an enrichment of immune gene-signatures that negatively correlated with metastasis incidence. Importantly, a hybrid epithelial-mesenchymal signature, related to aggressiveness was detected only in metastatic MSI-H tumors which may explain the worse outcomes after recurrence of MSI tumors compared to MSS. Interestingly, we identified immature myeloid cells at primary and metastatic sites in MSI-H tumor-bearing mice, suggesting that MMR deficiency elicits specific immune responses beyond T-cell activation. Significance A novel syngeneic mouse model of MSI cancer demonstrates that the immune system regulates MSI cancer cell dissemination, offering an important tool to model advanced stages of human MSI-driven disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4d06352a13d99fbaae03568dc6632e6da89c31" target='_blank'>
              Effect of MisMatch Repair Deficiency on metastasis occurrence modelized in a syngeneic mouse model
              </a>
            </td>
          <td>
            Pierre Laplante, Reginaldo Rosa, L. Nebot-Bral, Jordane Goulas, Sergey I. Nikolaev, Aymeric Silvin, P. Kannouche
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf2e1956ceee35cd1cab8cf25a742bffadc692d" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56327c0d899d960b104f68192383cb94fd68fc2e" target='_blank'>
              Circular RNAs in cancer: roles, mechanisms, and therapeutic potential across colorectal, gastric, liver, and lung carcinomas
              </a>
            </td>
          <td>
            Ayushi Malviya, Rajabrata Bhuyan
          </td>
          <td>2025-01-04</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zong-xiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc5c8e78609811ba357b3aeb0141c3d25fbb990" target='_blank'>
              Heterochromatin-dependent transcription links the PRC2 complex to small RNA-mediated DNA elimination
              </a>
            </td>
          <td>
            Therese Solberg, Chundi Wang, Ryuma Matsubara, Zhiwei Wen, M. Nowacki
          </td>
          <td>2024-11-29</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548909e3fc8f58fb6f1caf3e51e4cd2b9742adb3" target='_blank'>
              Unique structural configuration of EV-DNA primes Kupffer cell-mediated antitumor immunity to prevent metastatic progression.
              </a>
            </td>
          <td>
            I. Wortzel, Yura Seo, Ife Akano, Lee Shaashua, G. Tobias, Jakob Hebert, Kyung-A Kim, DooA Kim, Shani Dror, Yanshen Liu, Griffin Campbell Azrak, Michele Cioffi, Kofi Ennu Johnson, Tammy Hennika, Meshulam Zisha Twerski, Alexis Kushner, Robert Math, Yoon Dae Han, Dai Hoon Han, Minsun Jung, Juyeong Park, Soonmyung Paik, Jeon-Soo Shin, Min Goo Lee, Marco Vincenzo Russo, D. Zakheim, Jesse Barnes, Sunjoy Mehta, Katia Manova, R. Schwartz, B. K. Thakur, Nancy Boudreau, Irina R Matei, Haiying Zhang, Simone Sidoli, Jacqueline Bromberg, Yael David, H. Kim, D. Lyden
          </td>
          <td>2024-12-03</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most prevalent forms of cancer globally, and has recently become the leading cause of cancer-related mortality in women. BC is a heterogeneous disease comprising various histopathological and molecular subtypes with differing levels of malignancy, and each patient has an individual prognosis. Etiology and pathogenesis are complex and involve a considerable number of genetic alterations and dozens of alterations in non-coding RNA expression. Non-coding RNAs are part of an abundant family of single-stranded RNA molecules acting as key regulators in DNA replication, mRNA processing and translation, cell differentiation, growth, and overall genomic stability. In the context of breast cancer, non-coding RNAs are involved in cell cycle control and tumor cell migration and invasion, as well as treatment resistance. Alterations in non-coding RNA expression may contribute to the development and progression of breast cancer, making them promising biomarkers and targets for novel therapeutic approaches. Currently, the use of non-coding RNAs has not yet been applied to routine practice; however, their potential has been very well studied. The present review is a literature overview of current knowledge and its objective is to delineate the function of diverse classes of non-coding RNAs in breast cancer, with a particular emphasis on their potential utility as diagnostic and prognostic markers or as therapeutic targets and tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0045975d00155c439ba80ba1b588114b3af277" target='_blank'>
              Non-Coding RNAs in Breast Cancer: Diagnostic and Therapeutic Implications
              </a>
            </td>
          <td>
            Roman Beňačka, Daniela Szabóová, Z. Guľašová, Z. Hertelyová
          </td>
          <td>2024-12-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Identifying the roles of genes in cancer is critical in discovering potential genetic therapies for cancer care. Translocon-associated protein delta (TRAPδ), also known as signal sequence receptor 4 (SSR4), is a constituent unit in the TRAP/SSR complex that resides in the endoplasmic reticulum and plays a key role in transporting newly synthesized proteins into the endoplasmic reticulumn. However, its biological role in disease development remains unknown to date. This is the first study to identify the role of TRAPδ/SSR4 in colorectal cancer cells in vitro. Upon successful transient knockdown of TRAPδ/SSR4, we observed significant reduction of cell viability in all colorectal cancer cell lines tested. Both HCT 116 and SW480 cell lines were significantly arrested at S and G1 phases, while DLD-1 cells were significantly apoptotic. Moreover, TRAPδ/SSR4 stable knockdown HCT 116 and SW480 cells showed significantly lower viability, anchorage-independent growth, and increased S and G1 phase arrests. Overall, we conclude TRAPδ/SSR4 is a potential oncogene in human colorectal cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c566aa31384f1776417a4c1880816c68b25b7d5" target='_blank'>
              Identification of Translocon-associated Protein Delta as An Oncogene in Human Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Darshika Amarakoon, Wu-Joo Lee, Jing Peng, Seong-Ho Lee
          </td>
          <td>2024-12-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent type of liver cancer. Dosage suppressor of NNF1 (DSN1), a component of the MIS12 kinetochore complex, encodes a kinetochore protein crucial for proper mitotic assembly. The role of DSN1 in HCC remains to be elucidated. In this study, we utilized The Cancer Genome Atlas, the Hepatocellular carcinoma Cell Database, and other databases to analyze DSN1 expression and prognosis in samples from patients with HCC. We investigated the signaling pathways regulated by DSN1 and their implications in HCC. Additionally, we engineered siRNAsiRNA/shRNA/shRNA and overexpression vectors for DSN1 and assessed the specific mechanisms of regulatory pathways of DSN1 in hepatoma cell lines and subcutaneous tumor xenograft model. Our findings revealed that DSN1 expression was significantly upregulated in patients with HCC, correlating with decreased survival rates. Elevated DSN1 expression led to the overproduction of cell cycle-related proteins through direct interaction with Centromere Protein T. This interaction contributes to chromosomal instability in patients with HCC, resulting in an aberrant cell cycle and fostering the development and progression of HCC. Increased DSN1 expression is pivotal in HCC initiation and progression. Investigating DSN1 offers valuable insights into the pathogenesis, treatment, and prevention of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162c8061c9e0cfc53558ac673f085aa56ba63f7e" target='_blank'>
              DSN1 Interaction With Centromere-Associated Proteins Promotes Chromosomal Instability in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Hongrui Zhou, Mengxue Zhang, Jiabing Lian, Ruichang Wang, Zhen-Jun Yang, Jin Wang, Xiuli Bi
          </td>
          <td>2024-11-19</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Trinucleotide repeats in the human genome are implicated in various neurodegenerative diseases. The tendency of these repetitive DNA sequences to form non-B DNA structures can cause abnormal replication, leading to genomic instability. This instability contributes to disease progression, though the underlying mechanisms are not fully understood. We investigated the replication of DNA containing CAG and CTG trinucleotide repeats using individual components of the T7 bacteriophage replication machinery, as well as the complete replisome. Our results show that repeats in linear single-stranded DNA (ssDNA) inhibit the activity of T7 DNA polymerase and ssDNA-binding proteins, with a more pronounced effect observed in CTG repeats compared to CAG repeats. Minicircle templates containing CTG repeats exhibited robust DNA synthesis on both the leading and lagging strands, though synthesis was not enhanced by the T7 gene 2.5 ssDNA-binding protein. The lagging strand products generated from the CTG repeat minicircle were significantly longer than those from random sequence templates, and their lengths were not extended by the presence of T7 gene 2.5 protein. When the repeated sequences were incorporated into the T7 phage genome, heterogeneity was observed downstream of the repeats, depending on their length. We propose that aberrant extension occurs predominantly in the lagging strand, driven by dynamic interactions between the repeated sequences and the DNA replisome. This study may provide a foundation for understanding the mechanisms underlying the extension or deletion of repetitive genomic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5b02514d0537a2d3c7fa270216bb9c6028b5f7d" target='_blank'>
              Replication of DNA containing trinucleotide repeats by the bacteriophage T7 replisome
              </a>
            </td>
          <td>
            Seung-Joo Lee, Hitoshi Mitsunobu, Alfredo J. Hernandez
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Fumarate hydratase (FH) deficiency causes hereditary leiomyomatosis and renal cell carcinoma (RCC). FH-deficient tumors lack effective therapeutic options. Here, we utilized an epigenetic-focused single-guide RNA library to elucidate potential drug targets in FH-deficient tumors. The screen identified chromodomain helicase DNA binding protein 6 (CHD6) as an essential regulator of the growth of FH-mutated RCC. Mechanically, FH loss induced fumarate-mediated succinylation and inactivation of KEAP1, blocking subsequent ubiquitin-proteasome degradation of CHD6. Stabilized CHD6 formed a complex with p65 to establish pro-inflammatory enhancers and thereby regulate NF-κB-mediated transcription. Moreover, CHD6 recruited mSWI/SNF ATPases to maintain chromatin accessibility at CHD6-bound enhancers. The PROTAC degrader of SMARCA2/4 AU-15330 effectively abolished structures of cis-regulatory elements bound by CHD6 and suppressed the growth of FH-mutated, but not FH-intact, RCC in vivo. Collectively, these data indicate that CHD6 is a molecular bridge between FH deficiency and pro-inflammatory enhancers assembly that endows FH-deficient tumors with epigenetic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6839c985f22be7537f272a56a2547ccb209ecc66" target='_blank'>
              Chromatin Helicase CHD6 Establishes Pro-inflammatory Enhancers and is a Synthetic Lethal Target in FH-Deficient Renal Cell Carcinoma.
              </a>
            </td>
          <td>
            Juan Jin, Jun Luo, Xiaodong Jin, Kiat Shenq Lim, Yang He, Jiawei Ding, Yan Shen, Yuchen Hou, Hanqing Liu, Xiaoyu Zhu, Jing Zhao, Wenjie Zhou, Hai Huang, Yi Gao, Jun Xiao, Hongchao He, Qunyi Li, Lianxin Liu, Li Chen, Qiang He, Chuanjie Zhang
          </td>
          <td>2024-11-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef930bd56fde447e683b8e74f0950f751b9d7434" target='_blank'>
              Reassessing the roles of oxidative DNA base lesion 8-oxoGua and repair enzyme OGG1 in tumorigenesis
              </a>
            </td>
          <td>
            Jing Wang, Chunshuang Li, Jinling Han, Yaoyao Xue, Xu Zheng, Ruoxi Wang, Z. Radák, Y. Nakabeppu, Istvan Boldogh, Xueqing Ba
          </td>
          <td>2025-01-01</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8663834f0e1d3ecd98b4319fc6ee165b1d3d20b" target='_blank'>
              Analysis of mRNA Pentatricopeptide Repeat Domain 1 as a prospective oncogene in clear cell renal cell carcinoma that accelerates tumor cells proliferation and invasion via the Akt/GSK3β/β-catenin pathway
              </a>
            </td>
          <td>
            Z. Zhou, Yulong Li, Y. Chai, Yong Zhang, Pu Yan
          </td>
          <td>2025-01-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Constitutively active NOTCH2 signaling is a hallmark in chronic lymphocytic leukemia (CLL). The precise underlying defect remains obscure. Here we show that the mRNA sequence coding for the NOTCH2 negative regulatory region (NRR) is consistently deleted in CLL cells. The most common NOTCH2ΔNRR-DEL2 deletion is associated with two intronic single nucleotide variations (SNVs) which either create (CTTAT, G>A for rs2453058) or destroy (CTCGT, A>G for rs5025718) a putative splicing branch point sequence (BPS). Phylogenetic analysis demonstrates that rs2453058 is part of an ancient NOTCH2 gene variant (*1A01) which is associated with type 2 diabetes mellitus (T2DM) and is two times more frequent in Europeans than in East Asians, resembling the differences in CLL incidence. In contrast, rs5025718 belongs to a recent NOTCH2 variant (*1a4) that dominates the world outside Africa. Nanopore sequencing indicates that somatic reciprocal crossing over between rs2453058 (*1A01) and rs5025718 (*1a4) leads to recombined NOTCH2 alleles with altered BPS patterns in NOTCH2*1A01/*1a4 CLL cases. This would explain the loss of the NRR domain by aberrant pre-mRNA splicing and consequently the NOTCH2 gain-of-function phenotype. Together, our findings suggest that somatic recombination of inherited NOTCH2 variants might be relevant to CLL etiology and may at least partly explain its geographical clustering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d5b526c4b4d5a4555e25526b7ffec423997f32" target='_blank'>
              Somatic Recombination Between an Ancient and a Recent NOTCH2 Gene Variant Is Associated with the NOTCH2 Gain-of-Function Phenotype in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Rainer Hubmann, M. Hilgarth, Tamara Löwenstern, Andrea Lienhard, Filip Sima, Manuel Reisinger, Claudia Hobel-Kleisch, E. Porpaczy, Torsten Haferlach, Gregor Hoermann, Franco Laccone, C. Jungbauer, P. Valent, Philipp B. Staber, M. Shehata, Ulrich Jäger
          </td>
          <td>2024-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer.
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, Tamara Basta
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Poly (ADP-ribose) glycohydrolase (PARG) is a dePARylating enzyme which promotes DNA repair by removal of poly (ADP-ribose) (PAR) from PARylated proteins. Loss or inhibition of PARG results in replication stress and sensitizes cancer cells to DNA-damaging agents. PARG inhibitors are now undergoing clinical development for patients having tumors with homologous recombination deficiency (HRD), such as cancer patients with germline or somatic BRCA1/2-mutations. PARP inhibitors kill BRCA-deficient cancer cells by increasing single-stranded DNA gaps (ssGAPs) during replication. Here, we report that, like PARP inhibitor (PARPi), PARG inhibitor (PARGi) treatment also causes an accumulation of ssGAPs in sensitive cells. PARGi exposure increased accumulation of S-phase-specific PAR, a marker for Okazaki fragment processing (OFP) defects on lagging strands and induced ssGAPs, in sensitive cells but not in resistant cells. PARGi also caused accumulation of PAR at the replication forks and at the ssDNA sites in sensitive cells. Additionally, PARGi exhibited monotherapy activity in specific HR-deficient, as well as HR-proficient, patient-derived, or patient-derived xenograft (PDX)-derived organoids of ovarian cancer, and drug sensitivity directly correlated with the accumulation of ssGAPs. Taken together, PARGi treatment results in toxic accumulation of PAR at replication forks resulting in ssGAPs due to OFP defects during replication. Regardless of the BRCA/HRD-status, the induction of ssGAPs in preclinical models of ovarian cancer cells correlates with PARGi sensitivity. Patient-derived organoids (PDOs) may be a useful model system for testing PARGi sensitivity and functional biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e659026df6adca407f9a041f329b596cf911a69" target='_blank'>
              PARG inhibitor sensitivity correlates with accumulation of single-stranded DNA gaps in preclinical models of ovarian cancer.
              </a>
            </td>
          <td>
            Ramya Ravindranathan, Ozge Somuncu, A. A. B. A. da Costa, Sirisha Mukkavalli, Benjamin P Lamarre, Huy Nguyen, Carter Grochala, Yuqing Jiao, Joyce Liu, B. Kochupurakkal, K. Parmar, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2024-11-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="A retrovirus inserts its genome into the DNA of a cell, occasionally a germ-line cell that gives rise to descendants of the host organism: it is then called an endogenous retrovirus (ERV). The human genome contains relics from many kinds of ancient ERV. Some relics contributed new genes and regulatory elements. This study finds further kinds of ancient ERV, in the thoroughly-studied human genome version hg38: ERV-Hako, ERV-Saru, ERV-Hou, ERV-Han, and ERV-Goku. It also finds many relics of ERV-V, previously known from just two copies on chromosome 19 with placental genes. It finds a type of ERV flanked by MER41E long terminal repeats (LTRs), with surprisingly little similarity to the known MER41 ERV. ERV-Hako has subtypes that contain sequence from host genes SUSD6 and SPHKAP : the SUSD6 variant was transferred between catarrhine and platyrrhine primates. A retrovirus uses tRNA to prime reverse transcription: Hako is the only human ERV relic that used tRNA-Trp (tryptophan, symbol W), and HERV-W is misnamed because it used tRNA-Arg, based on the Genomic tRNA Database. One ERV-Saru LTR is the previously-described enhancer of AIM2 in innate immunity. This study contributes to understanding primate ERV history, but also shows that related ERVs can have drastic differences, challenging the goal of clearly annotating all ERV relics in genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362de87510ddffd6ce47f2c96a4e681ca10206a6" target='_blank'>
              Further varieties of ancient endogenous retrovirus in human DNA
              </a>
            </td>
          <td>
            Martin C. Frith
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b32d455915f14c9cee5327c8bd03503f5d7e8b33" target='_blank'>
              SNAI1 promotes epithelial-mesenchymal transition and maintains cancer stem cell-like properties in thymic epithelial tumors through the PIK3R2/p-EphA2 Axis
              </a>
            </td>
          <td>
            H. E, Lei Zhang, Zhenhua Yang, Long Xu, Tao Wang, Junhong Guo, Lang Xia, Juemin Yu, Heyong Wang, Y. She, Junqi Wu, Yue Zhao, Chang Chen, Deping Zhao
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d39afb3a607434c2c68d1fdc67b14d1c8df6aab" target='_blank'>
              Aberrant METTL1-mediated tRNA m7G modification alters B-cell responses in systemic autoimmunity in humans and mice
              </a>
            </td>
          <td>
            Shuyi Wang, Hui Han, Yichao Qian, Xinyuan Ruan, Zhangmei Lin, Jin Li, Binfeng Chen, Yimei Lai, Zhaoyu Wang, Mengyuan Li, Jingping Wen, Xiaoyu Yin, Niansheng Yang, Shuibin Lin, Hui Zhang
          </td>
          <td>2024-12-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA replication represents a series of precisely regulated events performed by a complex protein machinery that guarantees accurate duplication of the genetic information. Since DNA replication is permanently faced by a variety of exogenous and endogenous stressors, DNA damage response, repair and replication must be closely coordinated to maintain genomic integrity. HROB has been identified recently as a binding partner and activator of the Mcm8/9 helicase involved in DNA interstrand crosslink (ICL) repair. We identified HROB independently as a nuclear protein whose expression is co-regulated with various DNA replication factors. Accordingly, the HROB protein level showed a maximum in S phase and a downregulation in quiescence. Structural prediction and homology searches revealed that HROB is a largely intrinsically disordered protein bearing a helix-rich region and a canonical oligonucleotide/oligosaccharide-binding-fold motif that originated early in eukaryotic evolution. Employing a flow cytometry Förster resonance energy transfer (FRET) assay, we detected associations between HROB and proteins of the DNA replication machinery. Moreover, ectopic expression of HROB protein led to an almost complete shutdown of DNA replication. The available data imply a function for HROB during DNA replication across barriers such as ICLs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/063ed9d559a592d4694044ef441583042ee83050" target='_blank'>
              HROB Is Implicated in DNA Replication
              </a>
            </td>
          <td>
            Julia Kutz, Hannes Schmietendorf, Sheikh Anika Rahman, Franz Opel, Helmut Pospiech
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The strong size increase of the human neocortex is supported both by the amplification and the basal translocation of a neural stem cell population, the basal radial glial cells (or bRG cells). Using live imaging of second trimester human fetal tissue and cortical organoids, we identify two independent translocation modes for bRG cell colonization of the human neocortex. On top of an actomyosin-dependent movement called mitotic somal translocation (MST), we identify a microtubule-dependent motion occurring during interphase, that we call interphasic somal translocation (IST). We show that IST is driven by the LINC complex, through the nuclear envelope recruitment of the dynein motor and of its activator LIS1. Consequently, IST severely altered in LIS1 patient-derived cortical organoids. We also demonstrate that MST occurs during prometaphase and is a mitotic spindle translocation event. MST is controlled by the mitotic cell rounding molecular pathway, via Moesin and Vimentin, driving translocation. We report that 85% of bRG cell translocation is due to IST, for a total movement of 0,67 mm per month of human fetal gestation. Our work identifies how bRG cells colonize the human fetal cortex, and further shows that IST and MST are conserved in bRG-related migrating glioblastoma cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5162f39b5b35fd089916583d60a7b8a2e59a995" target='_blank'>
              Two independent translocation modes drive neural stem cell dissemination into the human fetal cortex
              </a>
            </td>
          <td>
            Ryszard Wimmer, Pauline Lestienne, Clarisse Brunet, L. Coquand, Amandine Di Cicco, Christophe Chehade, A. Artioli, Matthieu Cortes, Anne-Sophie Macé, Xiuyu Wang, Jean-Baptiste Manneville, Bettina Bessière, Ananya Roy, K. Forsberg-Nilsson, Julia Ladewig, Fabien Guimiot, Alexandre D. Baffet
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Prostate cancer is a malignant tumor caused by the malignant proliferation of epithelial cells, which is highly heterogeneous and drug-resistant, and neuroendocrine prostate cancer (NEPC) is an essential cause of drug resistance in its late stage. Elucidating the evolution of NEPC and the resistance process of enzalutamide, a novel antiandrogen, will be of great help in improving the prognosis of patients. As a research hotspot in the field of molecular biology in recent years, the wide range of biological functions of long noncoding RNAs (lncRNAs) has demonstrated their position in the therapeutic process of many diseases, and a large number of studies have revealed their critical roles in tumor progression and drug resistance. Therefore, elucidating the involvement of lncRNAs in the formation of NEPCs and their interrelationship with enzalutamide resistance may provide new ideas for a deeper understanding of the development of this disease and the occurrence of enzalutamide resistance and give a new direction for reversing the therapeutic dilemma of advanced prostate cancer. This article focuses on lncRNAs that regulate enzalutamide resistance and the neuroendocrine transition of prostate cancer through epigenetic, androgen receptor (AR) signaling, and non-AR pathways that act as “molecular sponges” interacting with miRNAs. Some insights into these mechanisms are used to provide some help for subsequent research in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50743b295e6011754fbae884a2dd3b94e2ad534d" target='_blank'>
              Long noncoding RNA mediates enzalutamide resistance and transformation in neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Zhe Zhu, Wenjing Xuan, Chaohui Wang, Chancan Li
          </td>
          <td>2024-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10-10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5'-TGT*-3' motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential.IMPORTANCEYarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica.
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and environmental microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite its relatively recent emergence, single-cell sequencing has cemented its place in scientific research. It has grown exponentially in less than two decades since its start, with broad impact in the biological sciences. The blood represents an attractive system for early development and application of single-cell technologies. As a result, single-cell analyses in blood and leukemia have led the way in describing how cellular heterogeneity affects cancer progression. In this review, we discuss the technological and conceptual advances brought by single-cell genomics, ranging from genetic evolution and differentiation states that mediate drug resistance to the complex interactions required for immunotherapy responses. These high-resolution insights are starting to enter clinical assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fa592467bc471ca2cb2ea46b2951c1d4aa2628" target='_blank'>
              Insights into Leukemia from Single-Cell Genomics
              </a>
            </td>
          <td>
            Nurefsan Sariipek, Peter van Galen
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The presence of specific genetic mutations in patients with glioblastoma multiforme (GBM) is associated with improved survival outcomes. Disruption of the DNA damage response (DDR) pathway in tumor cells enhances the effectiveness of radiotherapy drugs, while increased mutational burden following tumor cell damage also facilitates the efficacy of immunotherapy. The ATRX gene, located on chromosome X, plays a crucial role in DDR. The aim of this research is to elucidate the correlation between ATRX mutations and GBM. Dataset obtained from TCGA-GBM were conducted an analysis on the genomic features, biological characteristics, immunopathological markers, and clinical prognosis of patients carrying ATRX mutations. Our findings revealed a significantly elevated level of microsatellite instability in individuals with ATRX mutants, along with significant alterations in the receptor-tyrosine kinase (RTK)-ras pathway among patients exhibiting combined ATRX mutations. TCGA-GBM patients with concurrent ATRX mutations exhibited sensitivity to 26 chemotherapeutic and anticancer drugs, which exerted their effects by modulating the DDR of tumor cells through highly correlated mechanisms involving the RTK-ras pathway. Additionally, we observed an enrichment of ATRX mutations in specific pathways associated with DDR among TCGA-GBM patients. Our model also demonstrated prolonged overall survival in patients carrying ATRX mutations, particularly showing strong predictive value for 3- and 5-year survival rates. Furthermore, additional protective factors such as younger age, female gender, combined IDH mutations, and TP53 mutations were identified. The results underscore the protective role and prognostic significance of ATRX mutations in GBM as a potential therapeutic target and biomarker for patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89334fb95e9947311852d880d8c780526377835e" target='_blank'>
              ATRX mutation modifies the DNA damage response in glioblastoma multiforme tumor cells and enhances patient prognosis.
              </a>
            </td>
          <td>
            Rou Yu, Keru Huang, Xinyan He, Jingwen Zhang, Yushan Ma, Hui Liu
          </td>
          <td>2025-01-10</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Many organisms have sex chromosomes with large non-recombining regions that expand in a stepwise manner, although the underlying reasons remain poorly understood. Recently, we proposed that recombination suppression may evolve in sex chromosomes simply because of the presence of recessive deleterious mutations within genomes. Specifically, we demonstrated that chromosomal inversions suppressing recombination and carrying by chance fewer deleterious mutations than average have a selective advantage. In addition, we showed that the permanent heterozygosity of Y-like sex chromosomes facilitates the fixation of these less-loaded inversions by a sheltering effect, i.e., by preventing the expression of recessive deleterious mutations in a homozygous state when they increase in frequency. In contrast, similar less-loaded inversions in autosomes suffer from a disadvantage in the homozygous state as their frequency increases, preventing their fixation. However, the methodology and significance of our previous study have been questioned. Here, we present new analyses that further reinforce our original claims, demonstrating that the lower-load advantage and the sheltering effect can explain the fixation of inversions on sex chromosomes over a broad range of parameter values. We show that these mechanisms promote the fixation of inversions on Y-like chromosomes at rates exceeding those expected under drift alone. We used, as a control, autosomes with a similar population size as the Y chromosome, which, we argue, provides the appropriate neutral control for the sheltering effect. We also address criticisms regarding our focus on inversions surviving the first 20 generations in a figure of our previous study, and show that these criticisms stemmed from a misunderstanding of what this figure was intended to illustrate. Including all inversions, even those that went extinct within 20 generations, does not alter our conclusions. Overall, the present study offers new support for our theory based on the combination of lower-load advantage and sheltering effect, and addresses the questions about its significance and range of applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ec04a434911fce88a9e92ab2aad9f2d21d7ad1" target='_blank'>
              Deleterious mutations can contribute to the evolution of recombination suppression between sex chromosomes
              </a>
            </td>
          <td>
            Paul Jay, A. Véber, T. Giraud
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceda101cbd27e107d725418d6adeff717071b560" target='_blank'>
              RTEL1 is upregulated in gastric cancer and promotes tumor growth
              </a>
            </td>
          <td>
            Chunyu Yang, Suzeng Wang, Ge Gao, Peiwen Xu, Mengyuan Qian, Yuan Yin, Surui Yao, Zhaohui Huang, Zehua Bian
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Methyltransferase-like 3 (METTL3) is a primary RNA methyltransferase that catalyzes N6-methyladenosine (m6A) modification. The current study aims to further delineate the effect and mechanism of METTL3 in hepatocellular carcinoma (HCC). By using a murine model of hepatocellular cancer development induced via hydrodynamic tail vein injection, we showed that METTL3 enhanced HCC development. In cultured human HCC cell lines (Huh7 and PLC/PRF/5), we observed that stable knockdown of METTL3 by shRNA significantly decreased tumor cell proliferation, colony formation and invasion, in vitro. When Huh7 and PLC/PRF/5 cells with shRNA knockdown of METTL3 were inoculated into the livers of SCID mice, we found that METTL3 knockdown significantly inhibited the growth of HCC xenograft tumors. These findings establish METTL3 as an important oncogene in HCC. Through N6-methyladenosine-sequencing (m6A-Seq), RNA sequencing (RNA-Seq) and subsequent validation studies, we identified BMI1 and RNF2, two key components of the polycomb repressive complex 1 (PRC1), as direct downstream targets of METTL3-mediated m6A modification in HCC cells. Our data indicated that METTL3 catalyzed m6A modification of BMI1 and RNF2 mRNAs which led to increased mRNA stability via the m6A reader proteins IGF2BP1/2/3. Furthermore, we showed that the METTL3 inhibitor, STM2457, significantly inhibited HCC cell growth in vitro and in mice. Collectively, this study provides novel evidence that METTL3 promotes HCC development and progression through m6A modification of BMI1 and RNF2. Our findings suggest that the METTL3-m6A-BMI1/RNF2 signaling axis may represent a new therapeutic target for the treatment of HCC. Implications: The METTL3-m6A-BMI1/RNF2 signaling axis promotes HCC development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2828db6fb6b4ab3788a6525cfb5da2199636a58" target='_blank'>
              METTL3-mediated m6A modification regulates the polycomb repressive complex 1 (PRC1) components BMI1 and RNF2 in hepatocellular carcinoma cells.
              </a>
            </td>
          <td>
            Tong Wu, Weina Chen, Jinqiang Zhang, Wenbo Ma, Nianli Liu
          </td>
          <td>2024-12-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Poly(ADP‐ribose) polymerase (PARP) inhibitors (PARPis) are currently used to treat BRCA1/2 mutant cancers. Although PARPi sensitivity has been attributed to homologous recombination (HR) defects, other roles of HR factors have also been linked to response to PARPi, including replication fork protection. In this study, we investigated PARPi sensitivity in ovarian cancer patient-derived xenograft (PDX) models in relation to HR proficiency and replication fork protection. Analysis of BRCA1/2 status showed that in our cohort of 31 ovarian cancer PDX models 22.6% harbored a BRCA1/2 alteration (7/31), and 48.3% (15/31) were genomically unstable as measured by copy number alteration analysis. In vivo, PARPi olaparib response was measured in 15 selected PDX models. Functional assessment of HR using ex vivo irradiation-induced RAD51 foci formation identified all olaparib-sensitive PDX models, including four models without BRCA1/2 alterations. In contrast, replication fork protection or replication speed in ex vivo tumor tissue did not correlate with olaparib response. Targeted panel sequencing in olaparib-sensitive models lacking BRCA1/2 alterations revealed a MUS81 variant as a possible mechanism underlying PARPi sensitivity. Combined, we show that ex vivo RAD51 analysis effectively predicts in vivo olaparib response and revealed a subset of PARPi-sensitive, HR-deficient ovarian cancer PDX models, lacking a BRCA1/2 alteration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bb7e4412573dd728e35dd6cc514e3145be82d94" target='_blank'>
              RAD51 recruitment but not replication fork stability associates with PARP inhibitor response in ovarian cancer patient-derived xenograft models
              </a>
            </td>
          <td>
            Francien Talens, Vivian Oliviera Nunes Teixeira, Yannick P Kok, Mengting Chen, E. Rosenberg, Rashmie D. Debipersad, E. Duiker, Nathalie van den Tempel, M. Janatová, P. Zemánková, P. Nederlof, G. Wisman, Z. Kleibl, Steven de Jong, Marcel A.T.M. van Vugt
          </td>
          <td>2024-11-28</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and malignant type of brain tumor in adults and no effective therapies exist to combat this disease. Surgical resection, radio- and temozolomide-based-chemotherapy are insufficient mainly due to inter- and intra-tumoral heterogeneity of GBM tumors, therefore personalized approaches are required. Additionally, diverse tumor microenvironment with a leading role of glioblastoma stem cells (GSCs) in hypoxic core underlie the aggressiveness and recurrence of GBM. However, the molecules and pathways which directly promote tumor aggressiveness together with the driving process of epithelial-to-mesenchymal transition (EMT) and the components of extracellular matrix (ECM), remain only partially covered with prior studies focused mainly on coding oncogenes and tumor suppressor genes. Over the past years, a special attention has been paid to non-coding, circular RNAs (circRNAs) differentially expressed in various cancers, including GBM. Despite high abundance and stability, the exact biological roles of most individual circRNAs have not been fully revealed, so far. Besides anti-cancer targets, circRNAs have been considered as essential biomarkers and stratifying criterion of malignancy. Thus, studies on functions of differentially expressed circRNAs in GBM are an emerging field to be explored. In herein study, the biological role of circRNA CLIP2 (circCLIP2) was found highly overexpressed in primary tumor tissues. Upon the knock-down of circCLIP2 in 2D and 3D in vitro models, functional tests performed clearly indicate a significant contribution of circCLIP2 to aggressive potential of GBM as showed by decreased rates of proliferation, migration and invasion. Additional studies indicated predominant expression of circCLIP2 in GSC fractions additionally elevated by hypoxic conditions. The increased rates of proliferation and migration as well as molecular analyses were also confirmed in in vivo settings where xenograft model was applied. The expression of EMT marker Snail2 and ECM-degrading enzyme metalloproteinase 9 (MMP9) were significantly downregulated in the tumors formed from circCLIP2 silenced cells. Taken together, these results indicate a strong contribution of circCLIP2 in the aggressive phenotype of GBM which is due to a modulation of GSC proliferative potential as well as migration and invasion via EMT together with the modulator of ECM composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41e1172182008f639a17a4a3feb76b12f54727b" target='_blank'>
              Circular RNA - circCLIP2 is predominantly expressed in GSC niche and enhances glioblastoma aggressiveness via EMT and ECM signaling
              </a>
            </td>
          <td>
            J. Misiorek, Ż. Zarębska, K. Kuczyński, J. Latowska-Łysiak, A. Grabowska, P. Głodowicz, M. Sajek, Anna Karlik, Dorota Wronka, A. Barciszewska, Łukasz Przybył, K. Rolle
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7746fcc03c2930c5c3eb9fafbd71a061c1f28654" target='_blank'>
              KDM1A epigenetically enhances RAD51 expression to suppress the STING-associated anti-tumor immunity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Qingyuan Yang, Shiyin Wei, Cen Qiu, Chenjie Han, Zunguo Du, Ning Wu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Aneuploidy, arising from gain or loss of chromosomes due to nondisjunction, is a special class of mutation. It can create significant phenotypic changes by altering abundance of hundreds of genes in a single event, providing material for adaptive evolution. But it can also incur large fitness costs relative to other types of mutations. Understanding mutational dynamics of aneuploidy is important for modeling its impact in nature, but aneuploidy rates are difficult to measure accurately. One challenge is that aneuploid karyotypes may revert back to euploidy, biasing forward mutation rate estimates - yet the rate of aneuploidy reversion is largely uncharacterized. Furthermore, current rate estimates are confounded because fitness differences between euploids and aneuploids are typically not accounted for in rate calculations. We developed a unique fluctuation assay in a wild-yeast model to measure the rate of extra-chromosome loss across three aneuploid chromosomes, while accounting for fitness differences between aneuploid and euploid cells. We show that incorporating fitness effects is essential to obtain accurate estimates of aneuploidy rates. Furthermore, the rate of extra-chromosome loss, separate from karyotype fitness differences, varies across chromosomes. We also measured rates in a strain lacking RNA-binding protein Ssd1, important for aneuploidy tolerance and implicated in chromosome segregation. We found no role for Ssd1 in the loss of native aneuploid chromosomes, although it did impact an engineered chromosome XV with a perturbed centromeric sequence. We discuss the impacts and challenges of modeling aneuploidy dynamics in real world situations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f543ecd0458b30a0801433f2e7a5623276dbbac" target='_blank'>
              On the rate of aneuploidy reversion in a wild yeast model.
              </a>
            </td>
          <td>
            James Hose, Qi Zhang, Nathaniel P. Sharp, A. Gasch
          </td>
          <td>2024-11-25</td>
          <td>Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb33d6f2533f4e59a412dd13e19420718a10637" target='_blank'>
              The putative error prone polymerase REV1 mediates DNA damage and drug resistance in Candida albicans
              </a>
            </td>
          <td>
            Michelle R. Agyare-Tabbi, Deeva Uthayakumar, Desiree Francis, Laetitia Maroc, Chris Grant, Peter McQueen, Garret Westmacott, Hajer Shaker, Iwona Skulska, Isabelle Gagnon-Arsenault, Jonathan Boisvert, Christian R. Landry, Rebecca S. Shapiro
          </td>
          <td>2024-11-29</td>
          <td>Npj Antimicrobials and Resistance</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E. Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark D. Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oncogenic gamma herpesviruses, including Epstein–Barr Virus (EBV) and Kaposi’s Sarcoma-associated Herpesvirus (KSHV), are opportunistic cancer-causing viruses and induces oncogenesis through complex mechanisms, which involves manipulation of cellular physiology as well as epigenetic and epitranscriptomic reprogramming. In this review, we describe the intricate processes by which these viruses interact with the epigenetic machinery, leading to alterations in DNA methylation, histone modifications, and the involvement of non-coding RNAs. The key viral proteins such as EBNA1 and LMP1 encoded by EBV; LANA and vGPCR encoded by KSHV; play pivotal roles in these modifications by interacting with host factors, and dysregulating signaling pathways. The resultant reprogramming can lead to activation of oncogenes, silencing of tumor suppressor genes, and evasion of the immune response, which ultimately contributes to the oncogenic potential of these viruses. Furthermore, in this review, we explore current therapeutic strategies targeting these epigenetic alterations and discuss future directions for research and treatment. Through this comprehensive examination of the epigenetic and epitranscriptomic reprogramming mechanisms employed by oncogenic gamma herpesviruses, we aim to provide valuable insights into potential avenues for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d106bf10134bf9b8889820e2db155164b9012c0" target='_blank'>
              Epigenetic and epitranscriptomic regulation during oncogenic γ-herpesvirus infection
              </a>
            </td>
          <td>
            Rajnish Kumar Singh, Ramakrishna Vangala, Atharva S. Torne, Dipayan Bose, E. Robertson
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b03c34c1e229c6ce19e44e6aedadd94bcc58ad59" target='_blank'>
              CUL4A-DDB1-circRFWD2 E3 ligase complex mediates the ubiquitination of p27 to promote multiple myeloma proliferation
              </a>
            </td>
          <td>
            Jie Min, Jialei Mao, Hui Shi, Yumeng Peng, Xiaoning Xu, M. Guo, X. Tang, Ye Yang, C. Gu
          </td>
          <td>2024-11-22</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cytosine (DNA) methylation plays important roles in silencing transposable elements, plant development, genomic imprinting, stress responses, and maintenance of genome stability. To better understand the functions of this epigenetic modification, several tools have been developed to manipulate DNA methylation levels. These include mutants of DNA methylation writers and readers, targeted manipulation of locus-specific methylation, and the use of chemical inhibitors. Here, we summarize the effects of commonly used cytidine analog chemical inhibitors represented by zebularine, 5-azacytidine, and their related compounds on plants. These analogs are incorporated into the chromosomal DNA, where they block the activity of the replicative CG DNA methyltransferase 1 (MET1). This leads to manifold alterations in plant epigenome, modified developmental programs, or suppression of hybridization barriers. We also highlight the DNA-damaging effects of cytidine analogs, particularly the formation of stable DNA-protein crosslinks between DNA and MET1. This sheds new light on specific phenotypes observed upon cytidine analog treatments. In conclusion, cytidine analogs remain a vital tool for plant genome research and have the potential to open new promising avenues for applications in plant biotechnology and breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347eb5947bdb94f55760bce2a406c75b6df28aaf" target='_blank'>
              Cytidine analogs in plant epigenetic research and beyond.
              </a>
            </td>
          <td>
            Eva Dvořák Tomaštíková, A. Pečinka
          </td>
          <td>2024-12-28</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Abstract Background Lung adenocarcinoma is the most common type of lung cancer, accounting for approximately 40% of all lung cancer cases, and has the highest incidence among lung cancer subtypes. Recent studies have suggested that long non-coding RNAs (lncRNAs) play a crucial role in the initiation and progression of lung adenocarcinoma. Methods Based on integrative analysis through databases, we screened Long intergenic non-protein coding RNA 00839 (LINC00839) as one of the most highly upregulated lncRNAs in lung adenocarcinoma. In vitro and in vivo experiments demonstrated that LINC00839 promotes lung adenocarcinoma proliferation, migration, and invasion and that it is present in exosomes secreted by lung adenocarcinoma cells. Results In the cytoplasm, LINC00839 regulates the Toll-like receptor 4 (TLR4)/NF-κB signaling pathway by acting as a molecular sponge of miR-17-5p, thereby influencing the biological behavior of lung adenocarcinoma cells. LINC00839 binds to Polypyrimidine tract binding protein 1 (PTBP1) in the nucleus to regulate the nuclear translocation of NF-κB p65 molecules and, consequently, the transcription of downstream molecules. Conclusions Our study confirmed that LINC00839 promotes the biological progression of lung adenocarcinoma by performing dual roles in the cytoplasm and nucleus to co-regulate the NF-κB signaling pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46547bc3e4e21d3caf821f6cee47fe9308f4d352" target='_blank'>
              Exosomal long non-coding RNA-LINC00839 promotes lung adenocarcinoma progression by activating NF-κB signaling pathway
              </a>
            </td>
          <td>
            Yue Sun, Fang Zhen, Hongyi Wang, Xiao Liang, Yaru Wang, Feiran Wang, Jing Hu
          </td>
          <td>2024-11-25</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b3b6f34b71d4b5f3b8e63dfe568f53d1e1b13e" target='_blank'>
              Stabilization of expandable DNA repeats by the replication factor Mcm10 promotes cell viability
              </a>
            </td>
          <td>
            Chiara Masnovo, Zohar Paleiov, Daniel Dovrat, Laurel K Baxter, Sofia Movafaghi, A. Aharoni, S. Mirkin
          </td>
          <td>2024-12-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f105728d3e05cb655e57590a8ecd77f62627ac7b" target='_blank'>
              Mechanistic basis of atypical TERT promoter mutations
              </a>
            </td>
          <td>
            K. Elliott, Vinod Kumar Singh, Alan Bäckerholm, Linnea Ögren, M. Lindberg, K. Soczek, E. Hoberg, Tom Luijts, Jimmy Van den Eynden, Maria Falkenberg, Jennifer A. Doudna, Anders Ståhlberg, Erik Larsson
          </td>
          <td>2024-11-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="
 Cancer-associated fibroblasts (CAFs) are a heterogenous cell population that can promote esophageal cancer progression through multiple molecular mechanisms including increased cancer cell proliferation and migration, immunosuppression, and therapeutic resistance. Despite the divergence of CAFs from normal fibroblasts with regards to phenotype and functions, CAFs rarely exhibit widespread genetic alterations. Instead, it is proposed that CAFs are regulated at the epigenetic level. Our objective is to delineate this epigenetic regulation of CAFs in esophageal cancer. Ultimately, we aim to identify the epigenetic regulatory complexes that mediate CAF heterogeneity and pro-tumorigenic functions. To this end, we completed RNA-seq and ATAC-seq on fibroblasts isolated from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC) and non-cancerous esophageal tissue. Enrichment analysis identified the polycomb repressive complex 2 (PRC2) as a potential regulator of ESCC and EAC CAFs. PRC2 catalyzes the methylation (mono-, di- or tri-) of lysine 27 on histone 3 (H3K27me1/2/3) resulting in transcriptional repression. Inhibition of two key proteins of PRC2 (EZH2 and SUZ12) through either pharmacological or genetic modulation, altered expression of multiple CAF phenotype related proteins and inhibited CAF functions including CAF proliferation. Ongoing studies, using co-cultures of fibroblasts and tumor cells, seek to identify if inhibition of PRC2 can prevent the initial transition of normal esophageal fibroblasts into CAFs. Importantly, we are conducting in vivo studies to determine if selective PRC2 inhibition in CAFs can inhibit esophageal tumor progression with specific focus on primary tumor growth and metastatic spread. In summary, initial studies have identify that PRC2 complex mediates CAF phenotype and functions, and we aim to establish the larger impact of these CAF specific effects on esophageal cancer.
 Citation Format: Karen J. Dunbar, Raul Navaridas Fernandez de Bobadil, Katherine M. Cunningham, Emily Esquea, Gizem Efe, Anil K. Rustgi. Polycomb repressive complex 2 (PRC2) mediates the epigenetic regulation of cancer-associated fibroblasts in esophageal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d29e0ba845d63b544dd787fbf41b70e6dd633d82" target='_blank'>
              Abstract A030: Polycomb repressive complex 2 (PRC2) mediates the epigenetic regulation of cancer-associated fibroblasts in esophageal cancer
              </a>
            </td>
          <td>
            Karen J Dunbar, Raul Navaridas Fernandez de Bobadil, Katherine Cunningham, Emily Esquea, Gizem Efe, Anil K. Rustgi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A significant portion of human cancers utilize a recombination-based pathway, Alternative Lengthening of Telomeres (ALT), to extend telomeres. To gain further insights into this pathway, we developed a high-throughput imaging-based screen named TAILS (Telomeric ALT In situ Localization Screen), to identify genes that either promote or inhibit ALT activity. Screening over 1000 genes implicated in DNA transactions, TAILS revealed both well-established and novel ALT modulators. We have identified new factors that promote ALT, such as the nucleosome-remodeling factor CHD4 and the chromatin reader SGF29, as well as factors that suppress ALT, including the RNA helicases DDX39A/B, the replication factor TIMELESS, and components of the chromatin assembly factor CAF1. Our data indicate that defects in histone deposition significantly contribute to ALT-associated phenotypes. Based on these findings, we demonstrate that pharmacological treatments can be employed to either exacerbate or suppress ALT-associated phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b124646ba82a460ac6aa3c7cf35a789c6739919" target='_blank'>
              Identification of Novel Modulators of the ALT Pathway Through a Native FISH-Based Optical Screen
              </a>
            </td>
          <td>
            Benura Azeroglu, Simran Khurana, Shih-Chun Wang, Gianna M Tricola, Shalu Sharma, Camille Jubelin, Ylenia Cortolezzis, Gianluca Pegoraro, Kyle M. Miller, Travis H. Stracker, Eros Lazzerini Denchi
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The EML4–ALK oncogene drives tumour progression in approximately 5% of cases of non-small-cell lung cancers. At least 15 EML4–ALK variants have been identified, which elicit differential responses to conventional ALK inhibitors. Unfortunately, most, if not all, patients eventually acquire resistance to these inhibitors and succumb to the disease, which warrants the need for alternative targets to be identified. The most aggressive variant, EML4–ALK variant 3 (V3), assembles into a complex on interphase microtubules together with the NEK9 and NEK7 kinases, which leads to the downstream phosphorylation of NEK7 substrates. Overall, this promotes an elongated cell morphology and an enhanced migratory phenotype, which likely contributes to the increased metastasis often seen in V3 patients. Here, using two separate approaches to displace V3 from microtubules and a variety of in vitro assays, we show that microtubule association of EML4–ALK V3 is required for both V3 phenotypes, as removal of the oncogenic fusion protein from microtubules led to the dissociation of the V3–NEK9–NEK7 complex and the reversal of both phenotypic changes. Overall, we propose that targeting the interaction between EML4–ALK V3 and microtubules might offer a novel therapeutic option, independent of ALK activity, for V3+ NSCLC patients with acquired resistance to ALK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2861ca0c2468f59f9063b573d82cf9d569f6d7" target='_blank'>
              Microtubule Association of EML4–ALK V3 Is Key for the Elongated Cell Morphology and Enhanced Migration Observed in V3 Cells
              </a>
            </td>
          <td>
            S. Papageorgiou, Sarah L. Pashley, Laura O’Regan, K. Straatman, Andrew M. Fry
          </td>
          <td>2024-11-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma (GBM) is uniformly lethal due to profound treatment resistance. Altered cellular metabolism is a key mediator of GBM treatment resistance. Uptake of the essential sulfur-containing amino acid methionine is drastically elevated in GBMs compared to normal cells, however, it is not known how this methionine is utilized or whether it relates to GBM treatment resistance. Here, we find that radiation acutely increases the levels of methionine-related metabolites in a variety of treatment-resistant GBM models. Stable isotope tracing studies further revealed that radiation acutely activates methionine to S-adenosyl methionine (SAM) conversion through an active signaling event mediated by the kinases of the DNA damage response. In vivo tumor SAM synthesis increases after radiation, while normal brain SAM production remains unchanged, indicating a tumor- specific metabolic alteration to radiation. Pharmacological and dietary strategies to block methionine to SAM conversion slowed DNA damage response and increased cell death following radiation in vitro. Mechanistically, these effects are due to depletion of DNA repair proteins and are reversed by SAM supplementation. These effects are selective to GBMs lacking the methionine salvage enzyme methylthioadenosine phosphorylase. Pharmacological inhibition of SAM synthesis hindered tumor growth in flank and orthotopic in vivo GBM models when combined with radiation. By contrast, methionine depletion does not reduce tumor SAM levels and fails to radiosensitize intracranial models, indicating depleting SAM, as opposed to simply lowering methionine, is critical for hindering tumor growth in intracranial models of GBM. These results highlight a new signaling link between DNA damage and SAM synthesis and define the metabolic fates of methionine in GBM in vivo. Inhibiting radiation-induced SAM synthesis slows DNA repair and augments radiation efficacy in GBM. Using MAT2A inhibitors to deplete SAM may selectively overcome treatment resistance in GBMs with defective methionine salvage while sparing normal brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11aa91d406c5f4b75a9a6b9dbcbaac218f6d1df0" target='_blank'>
              Reciprocal links between methionine metabolism, DNA repair and therapy resistance in glioblastoma
              </a>
            </td>
          <td>
            Navyateja Korimerla, Baharan Meghdadi, Isra Haq, Kari Wilder-Romans, Jie Xu, Nicole Becker, Ziqing Zhu, Peter Kalev, Nathan Qi, Charles Evans, Maureen Kachman, Zitong Zhao, Angelica Lin, A. Scott, Alexandra M O'Brien, Ayesha U. Kothari, Peter Sajjakulnukit, Li Zhang, Sravya Palavalasa, Erik R Peterson, Marc L Hyer, Katya Marjon, Taryn Sleger, Meredith A. Morgan, C. Lyssiotis, Everett M. Stone, S. Ferris, Theodore S. Lawrence, Deepak Nagrath, Weihua Zhou, Daniel R Wahl
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Aberrant epigenetic alterations are known to lead to cancer arising from normal non-cancerous cells. Unlike genetic mutations, these dysregulations can be reversible and are potential targets for anticancer drugs. Curcumin, a natural plant-derived compound, has been shown to have an anticancer effect via its influence on epigenetic regulation. However, the exact identity of those epigenetic changes, as well as their mechanisms of action, remain largely unknown. The goal of this study is to use a 3D network model to identify what histone codes are being modified by curcumin and hypothesize the enzymatic pathways through which those changes occur in breast cancer. We analyzed multi-omic data from the NIH LINCS program and identified two novel histone marks associated with curcumin that are influenced by twenty-three phosphoproteins involved in cell signaling which are profiled in this paper. These histone and phosphoprotein signatures can be used as potential biomarkers for future chromatin-based drug therapies in breast cancer. GitHub URL Source code is available at https://github.com/mollahlab/ECHC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd80504373cad8ddc1f4f4900bd8a98fdd6b50ec" target='_blank'>
              Deciphering epigenetic influence of curcumin on histone signatures in breast cancer using 3D network
              </a>
            </td>
          <td>
            Tina Tang, Mikhail Y. Berezin, Shamim A. Mollah
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer, the most prevalent cancer in females, is a heterogeneous disease with various molecular subtypes, which presents challenges in diagnosis and treatment. Ubiquitination is one of the most critical post-translational protein modifications, that plays regulatory roles in numerous cellular processes including cell cycle progression, DNA replication & repair, apoptosis, transcription regulation, protein localization, trafficking and signal transduction. This modification can be reversed by deubiquitinases, or DUBs, a superfamily of cysteine proteases and metalloproteases that cleave ubiquitin-protein bonds. Dysregulation of DUBs has been associated to various diseases including cancer, making them promising targets for cancer therapy. We leveraged publicly available breast cancer datasets and employed various bioinformatics tools to identify differentially expressed DUBs in breast cancer. Our analysis identified six genes (COPS5, EIF3H, MINDY1, MINDY2, PSMD14 and USP26) with significant differential expression and survival implications. We further validated our findings experimentally and found upregulation of COPS5, EIF3H and MINDY 1 in MCF-7 and T47D breast cancer cell lines using qPCR analysis. To identify the role of these genes, EIF3H and COPS5, in disease progression, we constructed a protein-protein interaction (PPI) network with genes associated with metastasis and explored their correlation at the gene expression level in breast cancer patients. Together, this comprehensive study sheds light on DUB gene expression patterns in breast cancer with the potential to identify novel targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1182059f20169c0a82d0630262bfb5d46b482ee5" target='_blank'>
              Deubiquitinating enzymes in breast cancer: in silico analysis of gene expression and metastatic correlation.
              </a>
            </td>
          <td>
            Gaurav Sahoo, Shruti Bandyopadhyay, Ekta Tripathi, Prashanthi Karyala
          </td>
          <td>2024-12-13</td>
          <td>Journal of biomolecular structure & dynamics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer refers to malignant tumors occurring in the walls of the colon or rectum. The roles of WD Repeat Domain 12 (WDR12) and mitochondrial ribosome-associated tumor suppressor 4 (MRTO4) genes in colorectal cancer remain unclear. The colorectal cancer dataset GSE113513 configuration file was downloaded from the gene expression omnibus database generated from GPL15207. Differentially expressed genes screening, functional enrichment analysis, gene set enrichment analysis, Weighted Gene Co-expression Network Analysis, construction and analysis of protein–protein interaction networks, survival analysis, and gene expression heatmap plotting were conducted. Comparative toxicogenomics database analysis was performed to find diseases most relevant to core genes. TargetScan was used to screen miRNAs regulating core genes. A total of 3106 differentially expressed genes were identified. According to gene ontology analysis, they mainly enriched in organic acid metabolic processes, condensed chromosome kinetochore, oxidoreductase activity, and cell cycle. In Kyoto encyclopedia of genes and genomes analysis, they primarily concentrated in the cell cycle, TGF-β signaling pathway, Jak-STAT signaling pathway, PI3K-Akt signaling pathway, Ras signaling pathway, TNF signaling pathway, p53 signaling pathway, NF-kB signaling pathway, and WNT signaling pathway. Weighted Gene Co-expression Network Analysis with a soft thresholding power set to 12 generated 29 modules. The protein–protein interaction network identified 6 core genes (DDX27, NAT10, WDR12, DKC1, MRTO4, and NOP56). Survival analysis showed core genes (POSTN, MYH11, LUM, COL6A3, and COL4A1) as risk factors. Gene expression heatmap revealed high expression of core genes (WDR12 and MRTO4) in colorectal samples. Comparative toxicogenomics database analysis linked core genes (WDR12 and MRTO4) with local tumor infiltration, bowel obstruction, abdominal pain, and colorectal neoplasms. WDR12 and MRTO4 genes are highly expressed in colorectal cancer, potentially influencing its progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3eaef691d4ef44bd15d0a47a76d9dd081032185" target='_blank'>
              Roles of WDR12 and MRTO4 genes in colorectal cancer
              </a>
            </td>
          <td>
            Huanping Luo
          </td>
          <td>2024-12-27</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Increased telomerase activity has been considered as a conspicuous sign of human cancers. The catalytic cores of telomerase involve a reverse transcriptase and the human telomerase RNA (hTR). However, current detection of telomerase is largely limited to its activity at the tissue and single-cell levels. To reveal the precise distribution of subcellular hTR and telomerase activity, here a modular engineered DNA nanodevice (DNA-ND) is designed capable of imaging hTR and telomerase activity in cytoplasm and nucleus, enabling colocalization analysis. DNA-ND is a modular DNA complex comprising hTR and telomerase activity detection modules, which respectively sense intercellular hTR and telomerase activity via target-sensitive allosteric transition of DNA switches, actuating orthogonal activation of fluorescence signals to achieve in situ co-imaging of hTR and telomerase activity. By integrating DNA-ND with specific localized signals, the DNA-ND based precise profiling of colocalization of hTR and telomerase activity in different cell lines as well as their dynamic changes during pharmacological interventions is demonstrated. Notably, the results suggest that the locations of hTR and telomerase activity are not exactly overlapped, indicating the influence of intracellular environment on the binding of hTR to telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a34a17a7f6af1948fe0851efc3da900886ee2f" target='_blank'>
              A Modular Engineered DNA Nanodevice for Precise Profiling of Telomerase RNA Location and Activity.
              </a>
            </td>
          <td>
            Shi-Yi Zhang, Jian Lv, Ze-Rui Zhou, Peter X Geng, Dawei Li, Ruocan Qian, H. Ju
          </td>
          <td>2024-12-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Saccharomyces cerevisiae meiosis-specific Hop1, a structural constituent of the synaptonemal complex, also facilitates the formation of programmed DNA double-strand breaks and the pairing of homologous chromosomes. Here, we reveal a serendipitous discovery that Hop1 possesses robust DNA-independent ATPase activity, although it lacks recognizable sequence motifs required for ATP binding and hydrolysis. By leveraging molecular docking combined with molecular dynamics simulations and biochemical assays, we identified an ensemble of five amino acid residues in Hop1 that could potentially participate in ATP-binding and hydrolysis. Consistent with this premise, we found that Hop1 binds to ATP and that substitution of amino acid residues in the putative ATP-binding site significantly impaired its ATPase activity, suggesting that this activity is intrinsic to Hop1. Notably, K65A and N67Q substitutions in the Hop1 N-terminal HORMA domain synergistically abolished its ATPase activity, noticeably impaired its DNA-binding affinity and reduced its association with meiotic chromosomes, while enhancing the frequency of meiotic crossovers (COs). Overall, our study establishes Hop1 as a DNA-independent ATPase and reveals a potential biological function for its ATPase activity in the regulation of meiotic CO frequency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2297e5fc2e40c9dac9d79a6d0d22ef4ffd4fc1ea" target='_blank'>
              The ATPase activity of yeast chromosome axis protein Hop1 affects the frequency of meiotic crossovers.
              </a>
            </td>
          <td>
            Kshitiza M Dhyani, Suman Dash, Sameer Joshi, Aditi Garg, D. Pal, K. T. Nishant, K. Muniyappa
          </td>
          <td>2024-12-27</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The memory of cellular identity is crucial for the correct development of an individual and is maintained throughout life by the epigenome. Chromatin marks, such as DNA methylation and histone modifications, ensure the stability of gene expression programmes over time and through cell division. Loss of these marks can lead to severe pathologies, including cancer and developmental syndromes. However, reprogramming of cellular identity is also a natural phenomenon that occurs early in mammalian development, particularly in the germ line, which enables the production of mature and functional gametes. The germ line transmits genetic and epigenetic information to the next generation, contributing to the survival of the species. Primordial germ cells (PGCs) undergo extensive chromatin remodelling, including global DNA demethylation and erasure of the parental imprints. This review introduces the concept of epigenetic reprogramming, its discovery and key steps, as well as the transcriptional and chromatin changes that accompany germ cell formation in mice. Finally, we discuss the epigenetic mechanisms of genomic imprinting, its discovery, regulation and relevance to human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2611fcc1b3dfdfc6647da0d47d718b5970da542" target='_blank'>
              [Epigenetic reprogramming, germline and genomic imprinting].
              </a>
            </td>
          <td>
            Clara Roidor, Karim Chebli, Maud Borensztein
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) broadly impacts host genes, affecting the infected cell population and inducing the development of a disease with a poor prognosis, adult T-cell leukemia-lymphoma (ATL). This study aimed to provide a comprehensive epigenomic characterization of the infected cell population and evaluated the transcriptome and chromatin structures of peripheral blood cells in HTLV-1-infected individuals using RNA sequencing (RNA-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq). The infected cells showed significant changes in gene expression patterns from the polyclonal stage and before ATL onset while demonstrating similarities to tumor-forming ATL cells. These similarities were a result of large-scale open chromatin changes, supporting the independent early formation of epigenomic aberrations as an underlying mechanism for later clonal propagation. This study also demonstrated that HTLV-1 Tax directly affects the host chromatin structure, thereby developing fundamental epigenomic characteristics. Several Tax target genes, including the RASGRP3-ERK pathway, were recognized, indicating an impact on signaling pathways. This genome-wide variability in chromatin structural property is a novel feature of HTLV-1 infection and may contribute to pathogenic mechanisms. In addition, it has crucial implications for better understanding the impact of HTLV-1 on the host genome and identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f7c450fea8921cdae499b880a5d0560a4c49aa" target='_blank'>
              Rewired chromatin structure and epigenetic gene dysregulation during HTLV-1 infection to leukemogenesis.
              </a>
            </td>
          <td>
            Jun Mizuike, Kako Suzuki, Shu Tosaka, Yuta Kuze, Seiichiro Kobayashi, Makoto Nakashima, Koji Jimbo, Y. Nannya, Yutaka Suzuki, K. Uchimaru, Makoto Yamagishi
          </td>
          <td>2024-11-19</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b020bbdff578adee5b23263cfe22de8ce98c09" target='_blank'>
              EPLIN, a prospective oncogenic molecule with contribution to growth, migration and drug resistance in pancreatic cancer
              </a>
            </td>
          <td>
            J. Zeng, Cai Wang, F. Ruge, E. Ji, Tracy A. Martin, Andrew J Sanders, Shuqin Jia, Chunyi Hao, Wen G Jiang
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="To achieve replicative immortality, cancer cells must activate telomere maintenance mechanisms. In 10 to 15% of cancers, this is enabled by recombination-based alternative lengthening of telomeres pathways (ALT). ALT cells display several hallmarks including heterogeneous telomere length, extrachromosomal telomeric repeats, and ALT-associated PML bodies. ALT cells also have high telomeric replication stress (RS) enhanced by fork-stalling structures (R-loops and G4s) and altered chromatin states. In ALT cells, telomeric RS promotes telomere elongation but above a certain threshold becomes detrimental to cell survival. Manipulating RS at telomeres has thus been proposed as a therapeutic strategy against ALT cancers. Through analysis of genome-wide CRISPR fitness screens, we identified ALT-specific vulnerabilities and describe here our characterization of the roles of SUB1, a ssDNA-binding protein, in telomere stability. SUB1 depletion increases RS at ALT telomeres, profoundly impairing ALT cell growth without impacting telomerase-positive cells. During RS, SUB1 is recruited to stalled forks and ALT telomeres via its ssDNA-binding domain. This recruitment is potentiated by RPA depletion, suggesting that these factors may compete for ssDNA. The viability of ALT cells and their resilience toward RS also requires ssDNA binding by SUB1. SUB1 depletion accelerates cell death induced by FANCM depletion, triggering unsustainable levels of telomeric damage in ALT cells. Finally, combining SUB1 depletion with RS-inducing drugs rapidly induces replication catastrophe in ALT cells. Altogether, our work identifies SUB1 as an ALT susceptibility with roles in the mitigation of RS at ALT telomeres and suggests advanced therapeutic strategies for a host of still poorly managed cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6497c277f2efbf01c25fef9e39f419c5577c8b02" target='_blank'>
              The single-stranded DNA-binding factor SUB1/PC4 alleviates replication stress at telomeres and is a vulnerability of ALT cancer cells.
              </a>
            </td>
          <td>
            Jean-Christophe Dubois, Erin Bonnell, Amélie Filion, Julie Frion, Samuel Zimmer, Muhammad Riaz Khan, Gabriela M. Teplitz, Lisa Casimir, Élie Méthot, Isabelle Marois, Mouhamed Idrissou, Pierre-Étienne Jacques, R. Wellinger, Alexandre Maréchal
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8be129739336245b9c8e7af2246ffb0c95045f4e" target='_blank'>
              Ailanthone disturbs cross-talk between cancer cells and tumor-associated macrophages via HIF1-α/LINC01956/FUS/β-catenin signaling pathway in glioblastoma
              </a>
            </td>
          <td>
            Xubin Deng, Qianbing Zhang, Fa Jin, Fengfei Lu, Guosheng Duan, Luwei Han, Meiling Zhu, Zhengyan Yang, Gong Zhang
          </td>
          <td>2024-12-05</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Pituitary neuroendocrine tumors (PitNETs) are unusual among neoplasms in that sporadic tumors are not typically associated with genetic mutations. Instead, epigenetics, a non-mutational process by which gene expression is modified via a variety of mechanisms, may be a driving factor in PitNET growth and behavior. DNA methylation is one of the most well understood forms of epigenetic modification. Research on DNA methylation profiles of PitNETs has identified a large number of genes silenced or upregulated by DNA methylation, particularly when methylated at CpG islands of gene promoter regions. Global patterns of DNA methylation may provide valuable insight into origins of pituitary adenoma subtypes, assist with PitNet diagnostics and have been found to correspond to the current WHO classification of PitNETs based on transcription factor lineage. Analysis of differentially methylated regions of individual genes may have prognostic value as well as guide research toward nonsurgical therapeutic strategies. Pituitary epigenetics and DNA methylation analysis are rapidly growing areas of interest with the potential to shape the future of pituitary tumor diagnostics and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91afb11897061f0dd790a1feb3aca845da9bfbb" target='_blank'>
              Current understanding of the role of DNA methylation in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Racheal Peterson, David J Cote, G. Zada
          </td>
          <td>2025-01-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hypoxia-induced radioresistance limits therapeutic success in cancer. In addition, p53 mutations are widespread in tumors including non-small cell lung carcinomas (NSCLCs), and they might modify the radiation response of hypoxic tumor cells. We therefore analyzed the DNA damage and inflammatory response in chronically hypoxic (1% O2, 48 h) p53 null H358 NSCLC cells after X-ray exposure. We used the colony-forming ability assay to determine cell survival, γH2AX immunofluorescence microscopy to quantify DNA double-strand breaks (DSBs), flow cytometry of DAPI-stained cells to measure cell cycle distribution, ELISAs to quantify IL-6 and IL-8 secretion in cell culture supernatants, and RNA sequencing to determine gene expression. Chronic hypoxia increased the colony-forming ability and radioresistance of H358 cells. It did not affect the formation or resolution of X-ray-induced DSBs. It reduced the fraction of cells undergoing G2 arrest after X-ray exposure and delayed the onset of G2 arrest. Hypoxia led to an earlier enhancement in cytokines secretion rate after X-irradiation compared to normoxic controls. Gene expression changes were most pronounced after the combined exposure to hypoxia and X-rays and pertained to senescence and different cell death pathways. In conclusion, hypoxia-induced radioresistance is present despite the absence of functional p53. This resistance is related to differences in clonogenicity, cell cycle regulation, cytokine secretion, and gene expression under chronic hypoxia, but not to differences in DNA DSB repair kinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e9671201812e6ed77aea8e679f837c002dc2eb" target='_blank'>
              DNA Damage and Inflammatory Response of p53 Null H358 Non-Small Cell Lung Cancer Cells to X-Ray Exposure Under Chronic Hypoxia
              </a>
            </td>
          <td>
            H. Nisar, Melanie Brauny, Frederik M Labonté, Claudia Schmitz, Bikash Konda, C. Hellweg
          </td>
          <td>2024-11-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Long non-coding RNA (lncRNA) is a type of non-coding RNA distinguished by a length exceeding 200 nucleotides. Recent studies indicated that lncRNAs participate in various biological processes, such as chromatin remodeling, transcriptional and post-transcriptional regulation, and the modulation of cell proliferation, death, and differentiation, hence influencing gene expression and cellular function. ADAMTS9-AS1, an antisense long non-coding RNA situated on human chromosome 3p14.1, has garnered significant interest due to its pivotal involvement in the advancement and spread of diverse malignant tumors. ADAMTS9-AS1 functions as a competitive endogenous RNA (ceRNA) that interacts with multiple microRNAs (miRNAs) and plays a crucial role in regulating gene expression and cellular functions by modulating essential signaling pathways, including PI3K/AKT/mTOR, Wnt/β-catenin, and Ras/MAPK pathways. Dysregulation of this factor has been linked to tumor development, migration, invasion, and resistance to apoptotic mechanisms, including as iron-induced apoptosis, underscoring its intricate function in cancer pathology. While current research has clarified certain pathways involved in cancer formation, additional clinical and in vivo investigations are necessary to enhance comprehension of its specific involvement across various cancer types. This review encapsulates the recent discoveries on the correlation of ADAMTS9-AS1 with numerous malignancies, clarifying its molecular mechanisms and its prospective role as a therapeutic target in oncology. Furthermore, it identifies ADAMTS9-AS1 as a potential early diagnostic biomarker and therapeutic target, offering novel opportunities for targeted intervention in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd8babd07e146bd73db5bdfb569c7497c29a95ac" target='_blank'>
              The Dual Role of ADAMTS9-AS1 in Various Human Cancers: Molecular Pathogenesis and Clinical Implications.
              </a>
            </td>
          <td>
            Haodong He, Jingjie Yang, Yan Zhou, Xinyan Zheng, Lihan Chen, Zhujun Mao, Chuyuan Liao, Tongtong Li, Haoran Liu, Gang Zhou, Houdong Li, Chengfu Yuan
          </td>
          <td>2025-01-03</td>
          <td>Anti-cancer agents in medicinal chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Increased extracellular DNA (exDNA) levels in the blood are a hallmark of metastatic cancer, arising from tumor lysis, apoptosis, necrosis, and neutrophil extracellular traps (NETs) released by tumor-expanded neutrophils. NETs, web-like chromatin structures extruded by neutrophils to trap pathogens, are highly immunogenic due to their DNA and histone content and their persistent interaction with dendritic cells. Tumor cells can also release extracellular traps. In myeloid leukemia, blasts release nuclear content via traps to activate coagulation or sustain myeloproliferation. Extracellular traps have been found in NPM1 mutant AML patients, with mutant NPM co-localizing with histones along exDNA traps. Forcing trap extrusion in NPM mutant leukemia cells has been used to create dendritic cell-based vaccines that break tolerance against NPMc antigens. This suggests that NETs in the tumor microenvironment (TME) may promote tolerance, with breaking tolerance requiring consistent NET formation and dendritic cell interaction. NETs are also seen in solid tumors, such as triple-negative breast cancer (TNBC), where they aid in cancer cell migration, invasion, and awakening of circulating tumor cells. However, the capacity of tumor cells to directly extrude DNA traps is not fully understood. To investigate this, various human and murine breast cancer cell lines were seeded on poly-D-lysine coated slides, stained with DAPI and Sytoxgreen, and observed via confocal microscopy. Tumor cells extruded DNA traps within 4 hours without stimuli. Notably, trap extrusion correlated with cell line aggressiveness and was abolished by DNase, indicating dsDNA composition. The absence of traps in 24-hour cultures suggests DNA extrusion may help rare tumor cells survive and proliferate. In line, DNase treatment reduced tumor proliferation and increased apoptosis. To assess the significance of DNA traps in vivo, BALB/c mice were injected with the highly metastatic 4T1 clone 5 breast cancer model, with or without DNase treatment, and monitored starting 5 days post-injection and weekly until day 28. Histological analysis at the injection site 5 days post-injection showed that DNase treatment reduced both local trap formation and tumor cell proliferation, as indicated by BrdU incorporation. Moreover, immunohistochemistry revealed that in DNase-treated mice, initial tumor growth was followed by complete tumor regression, accompanied by local and systemic CD8 T cell activation. Correspondingly, although a few metastases were detectable at day 14 in DNase-treated mice, no metastases were observed by day 28. Although it remains unclear whether tumor-derived or immune cell-derived traps are most relevant in our model, the results demonstrate that their presence in the TME promotes tolerance despite the inherent immune adjuvant properties of the traps. This suggests that DNase treatment could be particularly effective in combating early tumor development or relapse when used as part of a combination therapy designed to sustain anti-tumor immune responses.
 Citation Format: Sabina Sangaletti, Paola Portararo, Laura Botti, Valeria Cancila, Claudio Tripodo, Mario P. Colombo, Claudia Chiodoni. Degradation of extracellular trap DNA sustains anti-tumor immune responses in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9102f696549c2a39f10ac4008739665e02303d60" target='_blank'>
              Abstract C014: Degradation of extracellular trap DNA sustains anti-tumor immune responses in breast cancer
              </a>
            </td>
          <td>
            S. Sangaletti, P. Portararo, Laura Botti, V. Cancila, Claudio Tripodo, M. P. Colombo, C. Chiodoni
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The de-ubiquitinase USP33 has been shown to possess either tumour-promoting or inhibitory effect on human cancer cells. However, all these findings are mainly based on in vitro cell culture models, and the in vivo evidence, which is more plausible to digest the functional role of USP33 in carcinogenic process, is still lacking. Here, we demonstrate that USP33 modulates DNA damage responses including cell cycle arrest and apoptosis induction through associating with p53. It directly interacts with p53 to mediate its de-ubiquitination and further  stabilisation under DNA damage condition. Depletion of USP33 induces an enhanced level of p53 ubiquitination, which de-stabilises p53 protein leading to impaired DNA damage responses. Furthermore, USP33 silencing shows either promoted or inhibited effect on cell proliferation in human cancer cells with p53 WT and mutant background, respectively. Consistently, mice with hepatocyte-specific USP33 knockout are more sensitive to nitrosodiethylamine (DEN)-induced hepatocarcinogenesis compared to wild type mice. Thus, our in vitro and in vivo evidences illustrate that USP33 possesses anti-tumour activity via regulating p53 stability and activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf79241c93d773eae434c4f3d8db032442a0c49" target='_blank'>
              USP33 Regulates DNA Damage Response and Carcinogenesis Through Deubiquitylating and Stabilising p53.
              </a>
            </td>
          <td>
            Yuqi Zhu, Zixiang Chen, Kaifeng Niu, Mengge Li, Yuchun Deng, Ji Zhang, D. Wei, Jiaqi Wang, YongLiang Zhao
          </td>
          <td>2024-12-18</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="lncRNAs (long non-coding RNAs) are heterogeneous RNA molecules that modulate various cellular processes, such as proliferation, differentiation, migration, invasion, and apoptosis, via different mechanisms. An increasing amount of research indicates that abnormal expression of lncRNA influences the development of drug resistance as well as the genesis and advancement of cancer, including melanoma. Furthermore, they are attractive biomarkers for non-invasive cancer diagnostics due to their strongly modulated expression and improved tissue and disease specificity. This review offers a succinct overview of the present understanding concerning the potential diagnostic biomarker potential of lncRNAs in melanoma. Cell death occurs frequently during growth and throughout life and is an active, organized, and genetically determined process. It is essential for the regulation of homeostasis. Controlled cell death and non-programmed cell death are both forms of cell death. The most prevalent forms of regulatory cell death are pyroptosis, ferroptosis, autophagy, necroptosis, necrosis, and apoptosis. Ferroptosis, pyroptosis, and autophagy are less common forms of cell death compared to necrosis, apoptosis, and necroptosis. ncRNAs are regulatory RNA molecules that are not involved in encoding proteins. They primarily consist of circular RNAs (circ RNAs), lncRNAs, and microRNAs (miRNAs). Moreover, non-coding RNAs have the ability to modulate tumor cell autophagy, pyroptosis, and ferroptosis at the transcriptional or post-transcriptional stage, as well as function as oncogenes and tumor suppressor genes, which can have considerable effects on the incidence and growth of tumors. This review concentrated on the recent advancements in the research of the diagnostic and therapeutic functions of ncRNAs in the regulation of programmed cell death in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c2d5632ba8374554ff55670c724793b6787a625" target='_blank'>
              Diagnostic and therapeutic role of non-coding RNAs regulating programmed cell death in melanoma
              </a>
            </td>
          <td>
            Zixu Wang, Cong Xie, Xiao Chen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="BACKGROUND
Diffuse hemispheric glioma, H3G34R/V-mutant (DHG-H3G34) is characterized by poor prognosis and lack of effective treatment options. DHG-H3G34R further harbor deactivation of Alpha-Thalassemia/Mental Retardation Syndrome X-linked protein (ATRX; DHG-H3G34R_ATRX) suggesting a unique interaction of these two oncogenic alterations. In this study, we dissect their cell biological interplay, investigate the impact on telomere stabilization and, consequently, validate a targeted therapy approach.


METHODS
We characterized patient-derived primary pediatric high-grade glioma (pHGG) models for telomere-maintenance mechanisms, DNA damage stress (including protein expression, pH2AX/Rad51 foci, cell-cycle arrest) and their sensitivity towards poly-ADP polymerase inhibitor (PARPi) combinations. Human induced pluripotent stem cells (iPSCs) were used for modelling the disease. The anticancer activity of PARPi combinations in vivo was studied in Chorioallantoic Membrane (CAM) and orthotopic in vivo experiments. Finally, we treated a DHG-H3G34R_ATRX patient with a PARPi combination therapy.


RESULTS
We elaborate that alternative lengthening of telomeres (ALT) is a key characteristic of DHG-H3G34R_ATRX. A dominant cooperative effect between H3G34R and ATRX loss in ALT activation also became apparent in iPSCs, which endogenously exert telomerase activity. In both, patient-derived DHG-H3G34R_ATRX models and H3G34R+/ATRX- iPSCs, the ALT phenotype was associated with increased basal DNA damage stress, mediating synergistic susceptibility towards PARPi (talazoparib, niraparib) combinations with topoisomerase-I inhibitors (topotecan, irinotecan). In a first-of-its-kind case, treatment of a DHG-H3G34R_ATRX patient with the brain-penetrant PARP inhibitor niraparib and topotecan resulted in a significant tumor reduction.


CONCLUSION
Our preclinical and clinical data strongly support the further development of PARPis together with DNA damage stress-inducing treatment regimens for DHG-H3G34R_ATRX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77b9cc12ad1d9d295c346bb0f9c67781ab01cbd" target='_blank'>
              Alternative lengthening of telomere-based immortalization renders H3G34R -mutant diffuse hemispheric glioma hypersensitive to PARP inhibitor combination regimens.
              </a>
            </td>
          <td>
            Anna Laemmerer, Christian Lehmann, L. Mayr, K. Bruckner, L. Gabler, D. Senfter, Philipp Meyer, Theresa Balber, C. Pirker, C. Jaunecker, D. Kirchhofer, P. Vician, Michelle Griesser, S. Spiegl-Kreinecker, M. Schmook, Tatjana Traub‐Weidinger, Peter Kuess, Franziska Eckert, A. Federico, S. Madlener, Natalia Stepien, Bernhard Robl, A. Baumgartner, Johannes Hainfellner, Karin Dieckmann, C. Dorfer, Karl Roessler, Nina S. Corsini, K. Holzmann, Wolfgang M. Schmidt, A. Peyrl, Amedeo A Azizi, C. Haberler, A. Beck, Stefan M Pfister, Julia Schueler, D. Loetsch-Gojo, Jürgen A Knoblich, W. Berger, J. Gojo
          </td>
          <td>2024-11-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase1 (PARP1) plays a vital role in DNA repair and its inhibition in cancer cells may cause cell apoptosis. In this study, we investigated the effects of a PARP1 variant, V762A, which is strongly associated with several cancers in humans, on the inhibition of PARP1 by three FDA approved inhibitors: niraparib, rucaparib and talazoparib. Our work suggests that these inhibitors bind to the V762A mutant more effectively than to the wild-type (WT), with similar binding free energies between them. Talazoparib inhibition uniquely lowers the average residue fluctuations in the mutant than the WT including lower fluctuations of mutant’s N- and C-terminal residues, conserved H-Y-E traid residues and donor loop (D-loop) residues which important for catalysis more effectively than other inhibitions. However, talazoparib also enhances destabilizing interactions between the mutation site in the HD domain in the mutant than WT. Further, talazoparib inhibition significantly disrupts the functional fluctuations of terminal regions in the mutant, which are otherwise present in the WT. Lastly, the mutation and inhibition do not significantly affect PARP1’s essential dynamics. GRAPHICAL ABSTRACT Figure. Impact of V762A mutation on inhibition of PARP1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0ac4c03972c2e7f8fd81eda26b14e0cd715d39" target='_blank'>
              Impact of a Cancer-Associated Mutation on Poly(ADP-ribose) Polymerase1 Inhibition
              </a>
            </td>
          <td>
            Neel Shanmugam, Shubham Chatterjee, G. A. Cisneros
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Circular RNA (circRNA) emerges as a significant sub-type of single-stranded non-coding RNA within colorectal cancer (CRC), boasting high abundance. Delving into research, numerous pivotal roles of circRNA in therapeutic contexts within CRC have come to light, encompassing areas such as metastasis, apoptosis, and proliferation. Moreover, circRNAs exhibit significant involvement in the advancement of therapeutic strategies, demonstrating unique correlations with tumor staging, size and overall survival rates in colorectal cancer. These associations position circRNAs as potential candidates for both anticancer interventions and prognostic biomarkers. Among all cancers, colorectal cancer is the second most prevalent cause of cancer-related death and the third most common disease to be diagnosed worldwide. To gain deeper insights into the impact of circRNA-based therapeutic developments on CRC and its progression, this comprehensive review aims to synthesize the roles of specific therapeutic applications targeting circRNAs in CRC. It also aims to evaluate circRNAs' potential as useful therapeutic targets and prognostic indicators in the context of colorectal cancer. The overarching goal of this review is to illuminate the landscape of therapeutic strategies and aid in clinical decision-making processes related to CRC. By elucidating the intricate interplay between circRNAs and therapeutic interventions, this review seeks to contribute to the advancement of therapeutic modalities and improve patient outcomes in the realm of colorectal cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e2a54ee5b48b0544173f0c3c5f63c22c950fe" target='_blank'>
              Circular RNAs: Therapeutic Uses in Colorectal Cancer
              </a>
            </td>
          <td>
            Muthusamy Thangavel, Chalini Vijayakumar, Deepalakshmi Balakrishnan
          </td>
          <td>2024-11-30</td>
          <td>International Journal of Experimental Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="During infection, the autonomous parvovirus minute virus of mice (MVM) induces cellular DNA breaks and localizes to such sites, which presumably affords an environment beneficial for genome replication. MVM replication also benefits from the DNA damage response (DDR) mediated by the ataxia-telangiectasia mutated (ATM) kinase, while the ataxia telangiectasia and Rad-3 related (ATR) arm of the DDR is disabled, which prevents activation of its primary target, checkpoint kinase 1 (Chk1). We find here that Chk1 inactivation strongly correlates with dephosphorylation of one of its targets, RAD51, known to play a pivotal role in homologous recombination repair (HRR), thus leading to substantial inhibition of DNA repair in infected cells. We demonstrate colocalization of replicating MVM DNA with cellular double-strand breaks (DSBs) during infection, and show that an agent that exogenously induces cellular DSBs significantly increases viral DNA replication levels, establishing a role for cellular genome damage in facilitating virus DNA replication. Additionally, overexpression of active Chk1 during MVM infection was found to re-establish the activating phosphorylation of RAD51 Thr 309, significantly suppress infection-induced reduction of HRR efficiency with a concomitant increase in cellular genome DSBs, and reduce viral DNA replication levels. Thus, we conclude that during infection, MVM inhibition of Chk1 activation enhances viral replication, at least in part, by inhibiting cellular HRR.IMPORTANCEThe autonomous parvovirus minute virus of mice (MVM) has a compact DNA genome encoding a minimum number of proteins. During infection, it induces cellular DNA damage and both utilizes and modifies the subsequent cellular DNA damage response (DDR) in various ways to facilitate its replication. One of MVM's activities in this regard is to inhibit one of the primary arms of the DDR, the ataxia telangiectasia and Rad-3 related (ATR) pathway, which prevents activation of checkpoint kinase 1 (Chk1), a key protein involved in controlling the cellular DDR and preserving genome integrity. We show that prevention by MVM of Chk1 activation leads to inhibition of homologous recombination repair (HRR) of cellular DNA, which helps sustain viral replication. This work illuminates another way in which autonomous parvoviruses adjust the cellular environment for their replicative advantage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab15a702412827cf231b7b73cc79a81325904c4" target='_blank'>
              Inactivation of checkpoint kinase 1 (Chk1) during parvovirus minute virus of mice (MVM) infection inhibits cellular homologous recombination repair and facilitates viral genome replication.
              </a>
            </td>
          <td>
            Igor Etingov, D. Pintel
          </td>
          <td>2024-11-20</td>
          <td>Journal of virology</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3b31b19a2854fcc9c5d78fda08a89b093f37f4" target='_blank'>
              DCAF13-mediated K63-linked ubiquitination of RNA polymerase I promotes uncontrolled proliferation in Breast Cancer
              </a>
            </td>
          <td>
            , Bing Yang, Haiyan Yan, Xingyu Ma, Bin Tian, Bingqi Zheng, Yong-Xian Chen, Yi-Ming Dong, Jinsi Deng, Ziling Zhan, Yanmei Shi, Jing Yuan Zhang, Daning Lu, Jie-Hua He, Yin Zhang, KaiShun Hu, Shuang Zhu, Keda Zhou, Yu-Chan Zhang, Yiqing Zheng, Dong Yin, 
          </td>
          <td>2025-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85d17441ac44a75b94aa59b4d0c571fba9dac0d" target='_blank'>
              RNA-binding motif protein RBM39 enhances the proliferation of gastric cancer cells by facilitating an oncogenic splicing switch in MRPL33.
              </a>
            </td>
          <td>
            Cheng-Piao Lu, Jiabao Li, Dong-Bao Li, Yuhan Wang, Xiao-Gang Jiang, Jing-Jing Ma, Guoqiang Xu
          </td>
          <td>2025-01-03</td>
          <td>Acta pharmacologica Sinica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic therapies facilitate transcription of immunogenic repetitive elements that cull cancer cells through 'viral mimicry' responses. Paradoxically, cancer-initiating events also facilitate transcription of repetitive elements. Contributions of repetitive element transcription towards cancer initiation, and the mechanisms by which cancer cells evade lethal viral mimicry responses during tumor initiation remain poorly understood. In this report, we characterize premalignant lesions of the fallopian tube along with syngeneic epithelial ovarian cancer models to explore the earliest events of tumorigenesis following loss of the p53 tumor suppressor protein. We report that p53 loss permits transcription of immunogenic repetitive elements and chronic viral mimicry activation that increases cellular tolerance of cytosolic nucleic acids and diminishes cellular immunogenicity. This selection process can be partially attenuated pharmacologically. Altogether, these results reveal that viral mimicry conditioning following p53 loss promotes immune evasion and may represent a pharmacological target for early cancer interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70afa0639e2729a2d6c69d21792968f408209d78" target='_blank'>
              Chronic viral mimicry induction following p53 loss promotes immune evasion.
              </a>
            </td>
          <td>
            Charles A. Ishak, Sajid A. Marhon, N. Tchrakian, Anjelica Hodgson, Helen Loo Yau, Isabela M. Gonzaga, Melanie Peralta, Illinca M. Lungu, Stephanie Gomez, , S. Y. Shen, Raymond Chen, Jocelyn Chen, Biji Chatterjee, Kevin Wanniarachchi, Junwoo Lee, Nicholas Zehrbach, Amir Hosseini, P. Mehdipour, Siyu Sun, Alexander Solovyov, Ilias Ettayebi, Kyle E. Francis, Aobo He, Taiyi Wu, Shengrui Feng, Tiago da Silva Medina, Felipe Campos de Almeida, Jane Bayani, Jason Li, Spencer MacDonald, Yadong Wang, Sarah S Garcia, E. Arthofer, Noor Diab, Aneil Srivastava, Paul Tran Austin, Peter J. B. Sabatini, Benjamin D. Greenbaum, Catherine A. O'Brien, Trevor G. Shepherd, Ming-Sound Tsao, Katherine B. Chiappinelli, Amit M. Oza, Blaise A. Clarke, Rob Rottapel, Stephanie Lheureux, Daniel D De Carvalho
          </td>
          <td>2025-01-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Background: NTHL1 is a DNA glycosylase, and mutations in this gene are linked to colorectal polyps and colorectal cancer (CRC). SBS30, a mutational signature associated with NTHL1 inactivation, correlates with c-alkylation (c-alkyl), a CRC mutational signature of alkylating damage. We aimed to investigate the potential for synthetic lethality between c-alkyl and NTHL1 inactivation as a novel therapeutic strategy for CRC. Methods: Using DepMap data, we categorized CRC cell lines into low and high c-alkyl groups. We performed CRISPR-mediated NTHL1 knockouts (KOs) in high vs low c-alkyl groups of cell lines and patient-derived CRC organoids and tested for synthetic lethality and its mechanisms. Results: High c-alkyl cells with NTHL1 KO showed decreased proliferation compared to low c-alkyl cells, with cell cycle arrest predominantly in the G2/S phase. Treatment with N-Nitroso-N-methylurea (MNU, alkylating reagent) converted low c-alkyl cells to a high c-alkyl state and conferred increased susceptibility to NTHL1 KO. High c-alkyl patient-derived organoids similarly showed lower proliferation with NTHL1 KO compared to low c-alkyl ones. High c-alkyl cells exhibited increased DNA damage, indicated by γH2AX and CHK1 phosphorylation. Low c-alkyl cells exhibited significant HDAC1 and HDAC2 upregulation, which was not seen in high c-alkyl cells. This could explain their protection against NTHL1 KO, whereas the lack of change in HDAC protein level may explain the failure of high c-alkyl cells to repair NTHL1-induced DNA damage. High c-alkyl cells with NTHL1 KO additionally showed increased DNA damage and cell death when treated with checkpoint inhibitors AZD-7762, LY2606368, and Rabusertib. Conclusions: NTHL1 KO presents a promising therapeutic target, especially for high c-alkyl CRCs that are more susceptible to DNA damage. Our findings also suggest that combining NTHL1 inhibition with CHK1 inhibition may offer new therapeutic opportunities in CRC.
 Citation Format: Hyeoncheol Kim, Jules Cazaubiel,, Tung-Lin Chiang, Carino Gurjao, Samantha Fitzgerald, Marios Giannakis. Increased vulnerability of colorectal cancer with high alkylation damage to NTHL1 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad8e781bd0deb99dc7d838c25939d2a03c2eb3f7" target='_blank'>
              Abstract A019: Increased vulnerability of colorectal cancer with high alkylation damage to NTHL1 inactivation
              </a>
            </td>
          <td>
            Hyeoncheol Kim, Jules Cazaubiel, Tung-Lin Chiang, Carino Gurjao, Samantha Fitzgerald, M. Giannakis
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Neurofibromatosis type 1 (NF1)-associated high-grade gliomas (HGGs) harboring ATRX mutations exhibit an aggressive clinical phenotype, driven by heightened genomic instability and metabolic reprogramming. Existing therapies, including chemotherapy and radiotherapy, are limited by resistance mechanisms and formation of secondary malignancy, underscoring the need for novel therapeutic strategies. Here, we report the results of a high-throughput screening of 10,000 small molecules aimed at identifying compounds selectively targeting vulnerabilities associated with concurrent ATRX and NF1 loss. Among the screened compounds, K784-6195 (ChemDiv ID) emerged as a promising candidate, exhibiting marked selective cytotoxicity in NF1-associated glioma cell lines with ATRX deficiency (IC50 = 4.84 µM). In comparison, wild-type ATRX sporadic glioma cell lines (U251) exhibited significantly reduced sensitivity to K784-6195 (IC50 = 37.03 µM). However, ATRX knockout U251 glioma cells recapitulating concurrent ATRX and NF1 loss exhibited heightened susceptibility to K784-6195 (IC50 = 20-23 µM) compared to their wild-type counterpart. Metabolomic analysis revealed that K784-6195 treatment impairs metabolic pathways, including the pentose phosphate pathway, glutamine metabolism, and redox homeostasis, leading to oxidative stress and impaired cell survival. These findings highlight K784-6195 as a promising candidate for therapeutic development, offering a targeted approach for the treatment of NF-1 associated HGGs with ATRX deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e7fa79cd403992a8afcaf330e468a9f47a26c7" target='_blank'>
              High-Throughput Targeted Drug Screening for NF1-associated High-Grade Gliomas with ATRX Deficiency
              </a>
            </td>
          <td>
            Swati Dubey, Simran Rai, Fabiola Guillen, Ajeeth Iyer, Su Aung, M. Yuan, Charles G. Eberhart, Fausto J Rodriguez
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND Hepatocellular carcinoma (HCC) is a prevalent and aggressive tumor. Sorafenib is the first-line treatment for patients with advanced HCC, but resistance to sorafenib has become a significant challenge in this therapy. Cancer stem cells play a crucial role in sorafenib resistance in HCC. Our previous study revealed that the long non-coding RNA (lncRNA) KIF9-AS1 is an oncogenic gene in HCC. However, the role of KIF9-AS1 in drug resistance and cancer stemness in HCC remains unclear. Herein, we aimed to investigate the function and mechanism of the lncRNA KIF9-AS1 in cancer stemness and drug resistance in HCC. AIM To describe the role of the lncRNA KIF9-AS1 in cancer stemness and drug resistance in HCC and elucidate the underlying mechanism. METHODS Tumor tissue and adjacent non-cancerous tissue samples were collected from HCC patients. Sphere formation was quantified via a tumor sphere assay. Cell viability, proliferation, and apoptosis were evaluated via Cell Counting Kit-8, flow cytometry, and colony formation assays, respectively. The interactions between the lncRNA KIF9-AS1 and its downstream targets were confirmed via RNA immunoprecipitation and coimmunoprecipitation. The tumorigenic role of KIF9-AS1 was validated in a mouse model. RESULTS Compared with that in normal controls, the expression of the lncRNA KIF9-AS1 was upregulated in HCC tissues. Knockdown of KIF9-AS1 inhibited stemness and attenuated sorafenib resistance in HCC cells. Mechanistically, N6-methyladenosine modification mediated by methyltransferase-like 3/insulin-like growth factor 2 mRNA-binding protein 1 stabilized and increased the expression of KIF9-AS1. Additionally, KIF9-AS1 increased the stability and expression of short stature homeobox 2 by promoting ubiquitin-specific peptidase 1-induced deubiquitination. Furthermore, depletion of KIF9-AS1 alleviated sorafenib resistance in a xenograft mouse model of HCC. CONCLUSION The N6-methyladenosine-modified lncRNA KIF9-AS1 promoted stemness and sorafenib resistance in HCC by upregulating short stature homeobox 2 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cf000dfd891b5e782e0e0fbfcfa7811492c167a" target='_blank'>
              N6-methyladenosine-modified long non-coding RNA KIF9-AS1 promotes stemness and sorafenib resistance in hepatocellular carcinoma by upregulating SHOX2 expression
              </a>
            </td>
          <td>
            Yong Yu, Xiang-Hong Lu, Jin-Song Mu, Jiangyun Meng, Jiang-Shan Sun, Hai-Xu Chen, Yang Yan, Ke Meng
          </td>
          <td>2024-12-28</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary In addition to being essential for DNA replication and repair, proliferating cell nuclear antigen (PCNA) has recently been linked to regulation of signalling, metabolism and apoptosis. As PCNA is a new and unexplored target for cancer treatment, we have here investigated the regulatory role of PCNA in 10 multiple myeloma (MM) cell lines using a multi-omics approach and a PCNA-targeting peptide, ATX-101, currently in clinical development. We have found that ATX-101’s efficacy is linked to PCNA’s role in regulation of proteasomal and ER stress in MM cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a84301979821512ede666f571de223ed396c3f64" target='_blank'>
              Multiple Myeloma Cells with Increased Proteasomal and ER Stress Are Hypersensitive to ATX-101, an Experimental Peptide Drug Targeting PCNA
              </a>
            </td>
          <td>
            Camilla Olaisen, L. M. Røst, Animesh Sharma, C. K. Søgaard, T. Khong, Sigrid Berg, Mi Jang, A. Nedal, Andrew Spencer, P. Bruheim, M. Otterlei
          </td>
          <td>2024-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cf43551cc8a193ed8c795f9a17b57aa6cc6bab" target='_blank'>
              Chk2 sustains PLK1 activity in mitosis to ensure proper chromosome segregation
              </a>
            </td>
          <td>
            Elizabeth M Black, Carlos Andrés Ramírez Parrado, Isabelle Trier, Wenxue Li, Yoon Ki Joo, Jennifer Pichurin, Yansheng Liu, Lilian Kabeche
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Pyrrolizidine alkaloids (PAs) are common phytotoxins that are found worldwide. Upon hepatic metabolic activation, the reactive PA metabolites covalently bind to DNAs and form DNA adducts, causing mutagenicity and tumorigenicity in the liver. However, the molecular basis of the formation and removal of PA-derived DNA adducts remains largely unexplored. In the present study, Sprague Dawley (SD) rats were exposed to retrorsine (RTS), a representative PA, at a human-relevant dose of 3.3 mg/kg/day for 28 days. The rats were divided into three groups: control, RTS-28 (sacrificed after continuous RTS exposure), and RTS-161 (sacrificed at 133 days post-RTS-exposure). The multi-omics analyses demonstrated the involvement of homologous recombination (HR) and non-homologous end joining (NHEJ) repair pathways as a response to PA-induced DNA damage. Additionally, the characteristic guanine adducts induced by RTS exposure were in accordance with the higher expression of XPA and XPC, indicating that nucleotide excision repair (NER) and base excision repair (BER) also contributed to repairing RTS-induced DNA damage. Furthermore, we also showed that DNA damage persisted after PA exposure, and mutagenically related repair errors might occur due to the prolonged genotoxic effects. The present study lays the foundation for bridging PA-derived DNA adducts, DNA damage, DNA repair, and the follow-up mutagenesis and carcinogenesis associated with PA exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1feb2f6457794419daf5cf5b6bd5c75cde0a14e8" target='_blank'>
              Repair of Retrorsine-Induced DNA Damage in Rat Livers: Insights Gained from Transcriptomic and Proteomic Studies
              </a>
            </td>
          <td>
            Yun Long, Yiwei Wang, Zijing Song, Xin He, Yisheng He, Ge Lin
          </td>
          <td>2024-12-01</td>
          <td>Toxins</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc7636cc7694bca6c468bc6e4c750272c2be1508" target='_blank'>
              SPRTN metalloprotease participates in repair of ROS-mediated DNA-protein crosslinks
              </a>
            </td>
          <td>
            Luke Erber, Arnold Scott Groehler IV, Cesar I. Cyuzuzo, Jahan Baker-Wainwright, Reeja S. Maskey, Lei Li, Yuichi J Machida, Natalia Tretyakova
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Taxanes are frequently used anticancer drugs known to kill tumor cells by inducing mitotic aberrations and segregation defects. A defining feature of specific cancers, notably triple-negative breast cancer (TNBC) and particularly those deficient in BRCA1, is chromosomal instability (CIN). Here, we focused on understanding the mechanisms of docetaxel-induced cytotoxicity, especially in the context of BRCA1-deficient TNBC. Using functional genetic screens in CIN+ cells, we identified genes that mediate docetaxel response and found an interaction between Huntingtin (HTT) and BRCA1-associated protein-1 (BAP1). We employed Brca1-/-;p53-/- mammary tumor cells, derived from genetically engineered mouse tumors that closely mimic the human disease, to investigate the role of these genes in CIN+ BRCA1-deficient cells. Specifically, we observed that loss of HTT sensitizes CIN+ BRCA1-deficient mammary tumor cells to docetaxel by shortening mitotic spindle poles and increasing spindle multipolarity. In contrast, BAP1 depletion protected cells against these spindle aberrations by restoring spindle length and enhancing mitotic clustering of the extra centrosomes. In conclusion, our findings shed light on the roles of HTT and BAP1 in controlling mitotic spindle multipolarity and centrosome clustering, specifically in the absence of BRCA1. This affects the response to microtubule-targeting agents and suggests that further studies of the interaction of these genes with the mitotic spindle may provide useful insights into how to target CIN+ cells, particularly in the challenging therapeutic landscape of BRCA1-deficient TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2088ba0887dd7b3e666579053f1f608773cc6a" target='_blank'>
              Docetaxel response in BRCA1,p53-deficient mammary tumor cells is affected by Huntingtin and BAP1.
              </a>
            </td>
          <td>
            Martín González-Fernández, Carmen Perry, Nora Merete Gerhards, P. Francica, Sven Rottenberg
          </td>
          <td>2024-12-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
 Esophageal squamous cell carcinoma (ESCC) is a malignant tumor with high morbidity and mortality, and easy to develop resistance to chemotherapeutic agents. Telomeres are DNA-protein complexes located at the termini of chromosomes in eukaryotic cells, which are unreplaceable in maintaining the stability and integrity of genome. Telomerase, an RNA-dependent DNA polymerase, play vital role in telomere length maintain, targeting telomerase is a promising therapeutic strategy for cancer.
 AIM
 To investigate the efficacy and underlying mechanisms of BIBR1532, a telomerase inhibitor, in ESCC.
 METHODS
 KYSE150 and KYSE410 cells were cultured and exposed to various concentrations of BIBR1532. Cell viability was assessed at 48 hours and 72 hours to determine the IC50 values. The effects of BIBR1532 on ESCC cell proliferation, migration, and cellular senescence were evaluated using the cell counting kit-8 assay, plate colony formation assay, scratch assay, transwell assay, and β-galactosidase staining, respectively. Western blotting was performed to detect the expression of proteins in BIBR1532-treated ESCC cells, such as human telomerase reverse transcriptase (hTERT), key molecules involved in DNA damage response (DDR) or cellular senescence, as well as telomere-binding proteins. Additionally, a tumor-bearing nude mouse model was established to evaluate the anti-cancer effect of BIBR1532 in vivo .
 RESULTS
 The IC50 values for KYSE150 and KYSE410 cells after 48 hours of BIBR1532 exposure were 48.53 μM and 39.59 μM, respectively. These values decreased to 37.22 μM and 22.71 μM, respectively, following a longer exposure of 72 hours. BIBR1532 exhibited dose-dependent effects on KYSE150 and KYSE410 cells, including decreased hTERT expression, inhibition of proliferation and metastasis, and induction of cellular senescence. Mechanistically, BIBR1532 upregulated the expression of the DDR protein, γ-H2AX, and activated the ataxia telangiectasia and Rad3-related protein (ATR)/ check point kinase 1 (CHK-1) and ataxia-telangiectasia mutated gene (ATM)/CHK2 pathways. BIBR1532 downregulated the expression of telomere-binding proteins, including telomeric-repeat binding factor 1 (TRF1), TRF2, protection of telomeres 1, and TIN2-interacting protein 1. In a nude mouse xenograft model, BIBR1532 significantly suppressed tumor growth, reduced hTERT expression, and increased γ-H2AX protein levels. Hematoxylin and eosin staining of various organs, including the heart, liver, spleen, lungs, and kidneys, revealed no apparent adverse effects.
 CONCLUSION
 BIBR1532 exerts anti-cancer effects on ESCC by inducing DDR through the ATR/CHK1 and ATM/CHK2 pathways and downregulating the expression of telomere-binding proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c9a7cb4627655be894378fe9cf7a95af0081d7" target='_blank'>
              BIBR1532 inhibits proliferation and metastasis of esophageal squamous cancer cells by inducing telomere dysregulation
              </a>
            </td>
          <td>
            Qin Wang, Qingrong Li, Lei Xu, Zichun Yuan, Xiao Liu, Mao-Ju Tang, Man Luo, Xiao-Wu Zhong, Qiang Ma, Xiao-Lan Guo
          </td>
          <td>2025-01-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) poses a substantial global health concern, exhibits inconspicuous early symptoms, and is typically diagnosed at advanced stages leading to unfavorable outcomes. The intricate tumor microenvironment plays a crucial role in CRC development and progression, where chemokines contribute significantly. These chemokines exhibit widespread expression within tumor cells, facilitating immune cell infiltration, angiogenesis, and the establishment of distant metastases. The dysregulation of various chemokines in the context of CRC has emerged as a pivotal factor in the disease's pathogenesis.To explore the relationship between chemokine gene expression and CRC patient survival, as well as to clarify their biological roles,We conducted RNA-sequencing (RNA-seq) analysis on a cohort of 88 CRC patients with tumor samples, thereby enabling a detailed exploration of chemokine involvement in CRC. This study was rigorously augmented using comprehensive datasets from The Cancer Genome Atlas (TCGA), ensuring a robust analysis of gene expression patterns associated with clinical outcomes.Through data analysis, we identified key genes from the chemokine family thought pertinent to CRC outcomes. Consequently, we constructed a novel prognostic model based on the risk score derived from these chemokine expressions. Validation against clinical metadata, executed through immunohistochemistry analysis, affirmed the relevance and accuracy of our model in predicting patient survival.Our findings illuminate the critical role of chemokines in shaping the immune microenvironment of CRC, thereby highlighting potential therapeutic targets for future treatment strategies. Our new prognostic model could provide important information for the development of targeted therapies for CRC, enhancing personalized treatment approaches andultimately improving survival for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45933796c7d55da0a6726e0ff10a2daf18918f4" target='_blank'>
              Chemokine family significance and prognostic value in colorectal cancer
              </a>
            </td>
          <td>
            Yi Ding, Yinnan Chen, Siyun Xie, Quanpeng Qiu, Xiaolong Guo, Yun Feng, Hongxia Li, Fang Zhu, Yaping Liu
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Long non-coding RNA Rhabdomyosarcoma 2-associated Transcript (RMST) is a crucial regulator in various biological processes, particularly in neurogenesis and cancer progression. This review summarizes current knowledge on structure, expression patterns, and functional roles across different organs and diseases of RMST. RMST exhibits tissue-specific expression, notably in brain tissues and vascular endothelial cells, and plays a significant role in neuronal differentiation through interaction with SRY-box 2 (SOX2). In cancer, RMST predominantly functions as a tumor suppressor, with context-dependent roles observed across different cancer types. RMST is also implicated in neurological disorders, cardiovascular diseases, and Hirschsprung's disease. Mechanistically, RMST acts as a competing endogenous RNA and a transcriptional regulator, interacting with various microRNAs and proteins to modulate gene expression. The potential of RMST as a biomarker and therapeutic target is increasingly recognized, particularly in atherosclerosis and cancer. While current findings are promising, further research is needed to fully elucidate the functions and translate these insights into clinical applications of RMST. This review underscores the significance of RMST in cellular processes and disease pathogenesis, highlighting its potential as a novel target for diagnostic and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274785be6ba4141ed74a6cfcddf7cc469ab67f46" target='_blank'>
              RMST: A long noncoding RNA involved in cancer and disease.
              </a>
            </td>
          <td>
            Hidenori Tani
          </td>
          <td>2024-11-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to “intrinsic resistance”, e.g., acquired by random mutations, cancer cells typically escape from certain treatments (“acquired resistance”) by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Epigenetics which examines the regulation of genes without modification of the DNA sequence, plays a crucial role in various biological processes and disease mechanisms. Among the different forms of epigenetic modifications, histone post-translational modifications (PTMs) are important for modulating chromatin structure and gene expression. Aberrant levels of histone PTMs are implicated in a wide range of diseases, including cancer, making them promising targets for biomarker discovery and therapeutic intervention. In this context, blood, tissues, or cells serve as valuable resources for epigenetic research and analysis. Traditional methods such as mass spectrometry and western blotting are widely used to study histone PTMs, providing qualitative and (semi)quantitative information. However, these techniques often face limitations that could include throughput and scalability, particularly when applied to clinical samples. To overcome these challenges, we developed and validated 13 Nu.Q® immunoassays to detect and quantify specific histone PTM-nucleosomes from K2EDTA plasma samples. Then, we tested these assays on other types of samples, including chromatin extracts from frozen tissues, as well as cell lines and white blood cells Our findings demonstrate that the Nu.Q® assays offer high specificity, sensitivity, precision and linearity, making them effective tools for epigenetic profiling. A comparative analysis of HeLa cells using mass spectrometry, Western blot, and Nu.Q® immunoassays revealed a consistent histone PTMs signature, further validating the effectiveness of these assays. Additionally, we successfully applied Nu.Q® assays across various biological samples, including human tissues from different organs and specific white blood cell subtypes, highlighting their versatility and applicability in diverse biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea54269010715f8d7295e217ed70c9663ed44c1" target='_blank'>
              High-Throughput Epigenetic Profiling Immunoassays for Accelerated Disease Research and Clinical Development
              </a>
            </td>
          <td>
            P. Van den Ackerveken, Clotilde Hannart, Dorian Pamart, Robin Varsebroucq, Marion Wargnies, Olivia Thiry, Marie Lurkin, Séverine Vincent, Muriel Chapelier, Guillaume Rommelaere, M. Herzog
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92e2f5cabd9e2a4c286349a54228e7bf25b75d61" target='_blank'>
              E2F1-driven CENPM expression promotes glycolytic reprogramming and tumorigenicity in glioblastoma
              </a>
            </td>
          <td>
            Zhiqiang Yi, Yanfei Jia, Runchun Lu, Chunwei Li, Long Wen, Xiangdong Yin, Junfei Yi, Liang Li
          </td>
          <td>2024-12-21</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bce9a209f6edbf8f76b448377a962fadd58edf" target='_blank'>
              The lives of cells, recorded.
              </a>
            </td>
          <td>
            Amjad Askary, Wei Chen, Junhong Choi, Lucia Y. Du, M. Elowitz, James A. Gagnon, Alexander F. Schier, S. Seidel, J. Shendure, Tanja Stadler, Martin Tran
          </td>
          <td>2024-11-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Reverse-transcribing viruses (RTVs) characterized by reverse transcription required for their replication infect nearly all the eukaryotes. After decades of extensive analyses and discoveries, the understanding of the diversity of RTVs has largely stagnated. Herein, we discover a previously neglected lineage of RTVs, designated Kuafuorterviruses, in animals. Through screening over 8000 eukaryote genomes, we identify the presence of endogenous Kuafuorterviruses in the genomes of 169 eumetazoans dispersed across 11 animal phyla. Phylogenetic analyses and sequence similarity networks indicate that Kuafuorterviruses constitute a novel major lineage of RTVs. The discovery of Kuafuorterviruses refines our understanding of the diversity, evolution, and classification of RTVs and has implications in annotating animal genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09fd4b26c457880f808962c9907b8488777fe4fe" target='_blank'>
              Kuafuorterviruses, a novel major lineage of reverse-transcribing viruses
              </a>
            </td>
          <td>
            Zhen Gong, Guan-Zhu Han
          </td>
          <td>2024-12-19</td>
          <td>Virus Evolution</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Cancer research has historically focused on the Somatic Mutation Theory (SMT), viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking Cancer Evolution: From Genetic Mutations to Complex Information Systems in Tumor Reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third leading cause of cancer mortality in the United States. Familial adenomatous polyposis (FAP) is a hereditary syndrome that raises the risk of developing CRC, with total colectomy as the only effective prevention. Even though FAP is rare (0.5% of all CRC cases), this disease model is well suited for studying the early stages of malignant transformation as patients form many polyps reflective of pre-cancer states. In order to spatially profile and analyze the pre-cancer and tumor microenvironment, we have performed single-cell multiplexed imaging for 52 samples: 12 normal mucosa,16 FAP mucosa,18 FAP polyps, 2 FAP adenocarcinoma, and 4 sporadic colorectal cancer (CRCs) using Co-detection by Indexing (CODEX) imaging platform. The data revealed significant changes in cell type composition occurring in early stage polyps and during the malignant transformation of polyps to CRC. We observe a decrease in CD4+/CD8+ T cell ratio and M1/M2 macrophage ratio along the FAP disease continuum. Advanced dysplastic polyps show a higher population of cancer associated fibroblasts (CAFs), which likely alter the pre-cancer microenvironment. Within polyps and CRCs, we observe strong nuclear expression of beta-catenin and higher number neo-angiogenesis events, unlike FAP mucosa and normal colon counterparts. We identify an increase in cancer stem cells (CSCs) within the glandular crypts of the FAP polyps and also detect Tregs, tumor associated macrophages (TAMs) and vascular endothelial cells supporting CSC survival and proliferation. We detect a potential immunosuppressive microenvironment within the tumor ‘nest’ of FAP adenocarcinoma samples, where tumor cells tend to segregate and remain distant from the invading immune cells. TAMs were found to infiltrate the tumor area, along with angiogenesis and tumor proliferation. CAFs were found to be enriched near the inflammatory region within polyps and CRCs and may have several roles in supporting tumor growth. Neighborhood analyses between adjacent FAP mucosa and FAP polyps show significant differences in spatial location of cells based on functionality. For example, in FAP mucosa, naive CD4+ T cells alone tend to localize near the fibroblast within the stromal compartment. However, in FAP polyp, CD4+T cells colocalize with the macrophages for T cell activation. Our data are expected to serve as a useful resource for understanding the early stages of neogenesis and the pre-cancer microenvironment, which may benefit early detection, therapeutic intervention and future prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f34b5e9edbfa348b570295378a92ebadba11703" target='_blank'>
              Single-cell spatial mapping reveals alteration of cell type composition and tissue microenvironment during early colorectal cancer formation
              </a>
            </td>
          <td>
            Tuhin K. Guha, Edward D. Esplin, Aaron M. Horning, R. Chiu, Kristina Paul, Annika K. Weimer, Winston R. Becker, Rozelle Laquindanum, M. Mills, D. G. Esplin, Jeanne Shen, Emma Monte, Shannon M. White, Thomas V. Karathanos, Daniel L. Cotter, Joanna Bi, U. Ladabaum, Teri A. Longacre, Christina Curtis, William J. Greenleaf, James M Ford, Michael P. Snyder
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Noncoding RNA 886 has emerged as a pivotal regulatory RNA with distinct functions across tissues, acting as a regulator of protein activity by directly binding to protein partners. While it is well recognized as a tumor suppressor in prostate cancer, the underlying molecular mechanisms remain elusive. To discover the principal pathways regulated by nc886 in prostate cancer, we used a transcriptomic and proteomic approach analyzing malignant DU145, LNCaP, PC3, and benign RWPE-1 prostate cell line models transiently transfected with in vitro transcribed nc886 or antisense oligonucleotides. Multiomics revelead a significant enrichment of immune system-related pathways across the cell lines, including cytokines and interferon signaling. The interferon response provoked by nc886 was validated by functional assays. The invariability of PKR phosphorylation and NF-κB activity in the gain/loss of nc886 function experiments and the positive regulation of innate immunity suggest a PKR-independent mechanism of nc886 action. Accordingly, the GSEA of the PRAD-TCGA data set revealed immune stimulation as the nc886 most associated node also in the clinical setting. Our study showed that the reduction of nc886 leads to a blunted immune response in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f4cce51821d8413b62bbd710ee2168d2423850" target='_blank'>
              Multi-Omics Study Reveals Nc886/vtRNA2-1 as a Positive Regulator of Prostate Cancer Cell Immunity.
              </a>
            </td>
          <td>
            Carolina Oliveira-Rizzo, Camilla L Colantuono, Ana J Fernández-Alvarez, Graciela L Boccaccio, Beatriz Garat, J. Sotelo-Silveira, Shahbaz Khan, V. Ignatchenko, Yong Sun Lee, T. Kislinger, Stanley K Liu, R. Fort, M. A. Duhagon
          </td>
          <td>2024-12-26</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract BACKGROUND Genome-wide DNA methylation analysis is currently available and is used to diagnose brain tumors. Although MGMT promoter hypermethylation is a strong biomarker in glioblastoma (GBM), there are few reports of DNA methylation profiles associated with prognosis or other DNA methylation biomarkers. In this study, we examined DNA methylation profiles correlated with prognosis and DNA methylation biomarkers. METHODS We selected 20 long-term survivors (LS) and 20 short-term survivors (SS) from 407 patients with newly diagnosed IDH-wild-type GBM treated at our institution, divided by MGMT methylation status (methylated; M/unmethylated; U), and adjusted for age. DNA methylation analysis was performed by EPIC array, and DKFZ classifier, tSNE analysis, and copy number analysis were performed. DNA methylation data and DNA methylation profiles were compared between LS and SS groups. RESULTS There were no significant differences in age and sex between the LS and SS groups in the M and U groups, respectively. Median OS (months) in the M group was 62.1/17.3 in LS/SS and 28.1/6.8 in the U group. Differentially methylated genes in the promoter region between LS/SS groups in the M group were HOXD13 and HAGH (methylated in LS), KCTD14 and ALPK3 (methylated in SS). Those in the U group were KLF14 and HOXB4 (methylated in LS), ZMYND10 and SLFN13 (methylated in SS). Gene Ontology analysis revealed that the genes with a hypermethylated promoter in the LS are enriched in G protein-coupled receptor signaling pathway and plasma membrane in the M group and Homeobox antennapedia and sequence-specific double-stranded DNA binding in the U group. CONCLUSION We identified DNA methylation biomarkers and key pathways involved in the prognosis of GBM with methylated and unmethylated MGMT promoter. Multi-omics analysis is necessary to validate the mechanism related to the prognosis of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0739f72de230cfe19a8e3c83b923df4e43528c3c" target='_blank'>
              10211- GEN-12 IDENTIFICATION OF PROGNOSIS-RELATED METHYLATION PROFILES AND METHYLATION BIOMARKERS IN IDH-WILD-TYPE GLIOBLASTOMA
              </a>
            </td>
          <td>
            K. Saito, K. Satomi, S. Shimizu, Kazuha Kugasawa, N. Sasaki, Keiichi Kobayashi, Junji Shibahara, Koichi Ichimura, Hirofumi Nakotmi, M. Nagane
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Myelodysplastic syndromes and other cancers are often associated with mutations in the U2 snRNP protein SF3B1. Common SF3B1 mutations, including K700E, disrupt SF3B1 interaction with the protein SUGP1 and induce aberrant activation of cryptic 3’ splice sites (ss), presumably resulting from aberrant U2/branch site (BS) recognition by the mutant spliceosome. Here, we apply the new method of U2 IP-seq to profile BS binding across the transcriptome of K562 leukemia cells carrying the SF3B1 K700E mutation. For cryptic 3’ ss activated by K700E, we identify their associated BSs and show that they are indeed shifted from the WT sites. Unexpectedly, we also identify thousands of additional changes in BS binding in the mutant cells that do not alter 3’ ss choice. These new BS are usually very close to the natural sites, occur upstream or downstream, and either exhibit stronger base-pairing potential with U2 snRNA or are adjacent to stronger polypyrimidine tracts than the WT sites. The widespread imprecision in BS recognition induced by K700E with limited changes in 3’ ss selection supports a positive role for SUGP1 in early BS choice and expands the physiological consequences of this oncogenic mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f4020459d06bbfad174a8dccbd619e96815f28" target='_blank'>
              Cancer-associated SF3B1 mutation K700E causes widespread changes in U2/branchpoint recognition without altering splicing
              </a>
            </td>
          <td>
            A. Damianov, Chia-Ho Lin, Jian Zhang, James L. Manley, Douglas L. Black
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a malignancy associated with high morbidity and mortality, yet treatment options are limited. In addition to genetic alterations, aberrant gene expression contributes to the pathology of malignant diseases. In the present study, we identified 629 genes consistently dysregulated between OSCC and normal oral mucosa across nine public gene expression datasets. Among them, mitosis-related genes were significantly enriched, including spindle and kinetochore-associated complex subunit 1 (SKA1), whose roles in OSCC had been studied only to a very limited extent. We show that SKA1 promoted proliferation and colony formation in 2D and 3D, shortened the duration of metaphase, and increased the migration of OSCC cell lines. In addition, high SKA1 expression enhanced radioresistance, a previously unknown effect of this gene, which was accompanied by a reduction of radiation-induced senescence. SKA1 was also upregulated in a subset of advanced oral premalignancies and promoted tumor-relevant properties in a corresponding cell line. Gene expression patterns evoked by SKA1 overexpression confirmed that this gene is able to advance properties required for both early and advanced stages of tumorigenesis. In summary, our data show that SKA1 contributes to malignant progression in OSCC and may be a useful marker of radioresistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec70f60faa1954d800f562931fb2bf10f070ffa" target='_blank'>
              SKA1 promotes oncogenic properties in oral dysplasia and oral squamous cell carcinoma, and augments resistance to radiotherapy.
              </a>
            </td>
          <td>
            A. Grandits, Barbara Andrea Reinoehl, Renate Wagner, Peter Kuess, Franziska Eckert, A. Berghoff, Thorsten Fuereder, Rotraud Wieser
          </td>
          <td>2024-12-10</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="ABSTRACT Missing an entire chromosome or chromosome arm in normal diploid cells has a deleterious impact on cell viability, which may contribute to the development of specific birth defects. Nevertheless, the effects of chromosome loss in human cells have remained unexplored due to the lack of suitable model systems. Here, we developed an efficient, selection‐free approach to generate partial monosomy in human induced pluripotent stem cells (iPSCs). The introduction of Cas9 proteins and a pair of gRNAs induces over megabase‐sized interstitial chromosomal deletions. Using human chromosome 21 (HSA21) as a model, partial monosomy 21q (PM21q) iPSC lines with deletions ranging from 4.5 to 27.9 Mb were isolated. A 33.6 Mb deletion, encompassing all protein‐coding genes on 21q, was also achieved, establishing the first 21q monosomy human iPSC line. Transcriptome and proteome analyses revealed that the abundances of mRNA and protein encoded by the majority of genes in the monosomic regions are half of the diploid expression level, indicating an absence of dosage compensation. The ability to generate customized partial monosomy cell lines on an isogenic, karyotypically normal background should facilitate the gain of novel insights into the impact of chromosome loss on cellular fitness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f457fb7efd63146141d89c41994690b979076d69" target='_blank'>
              Generation of Monosomy 21q Human iPS Cells by CRISPR/Cas9‐Mediated Interstitial Megabase Deletion
              </a>
            </td>
          <td>
            Masaya Egawa, N. Uno, Rina Komazaki, Yusuke Ohkame, Kyotaro Yamazaki, Chihiro Yoshimatsu, Yuki Ishizu, Yusaku Okano, Hitomaru Miyamoto, Mitsuhiko Osaki, Teruhiko Suzuki, Kazuyoshi Hosomichi, Yasunori Aizawa, Yasuhiro Kazuki, Kazuma Tomizuka
          </td>
          <td>2024-11-24</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mycoplasma bovis (M. bovis) is characterized by a reduced genomic size and limited synthetic capacity, including the inability to synthesize nucleotides de novo, relies on nucleases for nutrient acquisition and survival. A number of nucleases have been implicated in M. bovis pathogenicity, facilitating substrate degradation and contributing to DNA repair mechanisms that enhance bacterial persistence. The present study confirmed that the T5.808 mutant, in which a novel nuclease gene (Mbov_0701) was disrupted by the mini-Tn4001 transposon, exhibits a growth defect when co-cultured with EBL cells. However, the restoration of Mbov_0701 resulted in the resumption of growth in the mutant. The characterization of MbovP701 revealed that it had high activity in hydrolyzing dsDNA with 5′- to 3′- polarity. Furthermore, the substrates of MbovP701 were extended to include linear dsDNA, ssDNA, RNA, and plasmid DNA. The exonuclease activity is dependent on the presence of Mn2+ and/or Mg2+ ions, with an optimal pH and temperature of 8.3 and 43 °C, respectively. The truncation experiments of rMbovP701 revealed that YqaJ (41–185 aa) is the key functional domain of MbovP701 exonuclease. In conclusion, the present study identified a novel nuclease in M. bovis that plays an essential role in the proliferation of this minimal organism. This finding elucidates the survival strategy and pathogenesis of M. bovis, suggesting a potential therapeutic strategy for the treatment of M. bovis through targeting the inhibition of MbovP701. Moreover, it provides a foundation for future investigations into the interactions between MbovP701 and other nucleases involved in M. bovis biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edf88c44ad6d18a2167a54f0906a3be9810ae56b" target='_blank'>
              Novel Nuclease MbovP701 with a Yqaj Domain Is Interrelated with the Growth of Mycoplasma bovis
              </a>
            </td>
          <td>
            Zhiyu Hao, Doukun Lu, Xixi Li, Abdul Raheem, Gang Zhao, A. Dawood, Yingyu Chen, Xi Chen, Changmin Hu, Jianguo Chen, Lei Zhang, Xifang Zhu, A. Guo
          </td>
          <td>2024-12-01</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566de0581abca83bb852fa5fa0a344501c4488be" target='_blank'>
              A STT3A-dependent PD-L1 glycosylation modification mediated by GMPS drives tumor immune evasion in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xinyu Guo, Tianming Cui, Linmao Sun, Yumin Fu, Cheng Cheng, Chenghui Wu, Yitong Zhu, Shuhang Liang, Yufeng Liu, Shuo Zhou, Xianying Li, Changyong Ji, Kun Ma, Ning Zhang, Qi Chu, Changjian Xing, Shumin Deng, Jiabei Wang, Yao Liu, Lianxin Liu
          </td>
          <td>2024-12-17</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains challenging to treat, which in part relates to genetic heterogeneity of the disease, to the protective tumor microenvironment driving resistance to therapy, and also to immune evasion characteristics of leukemic cells. Targeting epigenetic programs in AML provides an attractive opportunity to impair long-term proliferation and induce differentiation. The novel inhibitor JNJ- 75276617 (bleximenib) targets the menin-KMT2A interaction and provides preclinical efficacy in AML (Kwon et al1). Here, we provide mechanistic insight in how JNJ- 75276617 impairs proliferation and drives differentiation of primary AML patient cells. A large-scale drug screen was set up in which genetic alterations and quantitative proteomics were compared with drug sensitivity in a preclinical setting, which revealed that granulocyte macrophage progenitor (GMP)-like AMLs display the greatest sensitivity. Furthermore, we identified that NPM1c/DNMT3Amut AMLs are sensitive, and some NPM1wt AML subtypes without KMT2A-MLLT3 rearrangements benefit from menin-KMT2A inhibition. Genome-wide ChIP-seq studies revealed patient-specific epigenetic alterations upon JNJ-75276617 treatment, uncovering a striking upregulation of MHC class I and class II expression as a consequence of epigenetic changes upon menin-KMT2A inhibition, independent of MEIS1 loss but involving CIITA activation. Functionally, this results in enhanced sensitivity of leukemic blasts to T cell-mediated cytotoxicity in allogeneic and autologous settings. Our data indicate that JNJ-75276617 provides a potential therapeutic approach whereby not only proliferation is impaired and differentiation is induced, but whereby therapeutic benefit might also be achieved by reactivating the antigen presentation machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eecdee97edcaeb1d7fdc720efba9913f9174c55" target='_blank'>
              Bleximenib, the novel menin-KMT2A inhibitor JNJ-75276617, impairs long-term proliferation and immune evasion in acute myeloid leukemia.
              </a>
            </td>
          <td>
            S. Hogeling, Duy Minh Lê, Nikita D. La Rose, Min Chul Kwon, Albertus T J Wierenga, Fiona A J Van den Heuvel, V. van den Boom, Anna Kuchnio, Ulrike Philippar, G. Huls, J. Schuringa
          </td>
          <td>2024-12-19</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="ABSTRACT DNA damage repair is a crucial cellular mechanism for rectifying DNA lesions arising during growth and development. Among the various repair pathways, postreplication repair (PRR) plays a pivotal role in resolving single-stranded gaps induced by DNA damage. However, the contribution of PRR to virulence remains elusive in the fungal pathogen Candida albicans (C. albicans). In this study, we investigated the role of Rad18, a critical component of PRR, in DNA damage response and virulence in C. albicans. We observed that deletion of RAD18 in C. albicans resulted in heightened sensitivity to DNA damage stress. Through deletion of specific internal domains coupled with spot assay analysis, we show that the internal RING and SAP domains play essential roles in DNA damage response, whereas the ZNF domain was less important. Surprisingly, the lack of Rad18 in C. albicans resulted in heightened intracellular survival within macrophages and elevated virulence in the Galleria mellonella model. RNAseq analysis revealed that loss of Rad18 upregulated the transcription of genes encoding transporters and oxidoreductases, as well as virulence genes, including HWP1 and ECE1. Suppression of the transcription of these virulence genes in the RAD18 deletion strain by a dCas9-mediated CRISPRi system reversed this increased virulence. Taken together, these data demonstrate that Rad18 plays a significant role in virulence partially through transcriptional suppression of virulence genes HWP1 and ECE1 in C. albicans. Our findings provide valuable insights into the intricate relationship between DNA damage response and virulence in C. albicans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85478461426a57f06cbb96c87887e10f784922c8" target='_blank'>
              DNA damage repair factor Rad18 controls virulence partially via transcriptional suppression of genes HWP1 and ECE1 in Candida albicans
              </a>
            </td>
          <td>
            Runlu Chen, Yuting Feng, Huaxin Cai, Shaling Yang, Xiaoyin She, Jinrong Feng
          </td>
          <td>2024-11-21</td>
          <td>Virulence</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent evidence suggests a crucial biological role for Circular RNAs (circRNAs) in keloid diseases, yet the underlying mechanisms remain unclear. This study explored the biological effects and molecular mechanisms of hsa_circ_0002198 in keloid formation. Methods Real-time quantitative PCR (qRT-PCR) was employed to assess the expression of circ_0002198 in keloid tissues, normal skin tissues, keloid fibroblasts (KFs), and normal skin fibroblasts (NFs) from nine patients. To investigate the role of circ_0002198 in keloid pathogenesis, cell transfection technology was utilized to knock down circ_0002198. Various experiments including Cell Counting Kit-8 (CCK-8), 5-Ethynyl-2′-deoxyuridine (EdU), Transwell, wound healing assay, flow cytometry, and others were conducted to explore the potential mechanisms associated with circ_0002198 expression. The RNA-binding protein Eukaryotic translation initiation factor 4A, isoform 3 (EIF4A3) binding to circ_0002198 was identified and confirmed through bioinformatics databases prediction and RNA immunoprecipitation (RIP) assay. Finally, the expression of EIF4A3 was assessed, and both silencing and overexpression were employed to verify its role in circ_0002198 regulation. Results The expression levels of circ_0002198 and EIF4A3 were notably elevated in keloid tissues and KFs compared to normal skin tissues and NFs. The reduction of circ_0002198 expression in KFs significantly impeded their proliferation, migration, and invasion. It also hindered the cell cycle process and the expression of associated proteins while concurrently promoting apoptosis in KFs. EIF4A3 was identified to bind to the flanks of circ_0002198, enhancing the occurrence of circ_0002198 and its role in regulating the progression of KFs. Conclusion Our study offers insights into how Circular RNA may contribute to the pathogenesis of keloid formation, highlighting Circ_0002198 as a potential novel biomarker for keloids in association with EIF4A3. Further research, involving larger study cohorts, is necessary to broaden our understanding of keloid mechanisms and potential treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bfb73631e54cdc07daef4fa2c27e715644f2cfb" target='_blank'>
              EIF4A3 Enhances the Proliferation and Cell Cycle Progression of Keloid Fibroblasts by Inducing the hsa_circ_0002198 Expression
              </a>
            </td>
          <td>
            Zidi Xu, Chang Li, Xueyi Liu, Yongting Zhou, Yingbo Zhang, Jie-Xin Wang, Hao Wu, Abdullah Al-danakh, Yixuan Peng, Zhibo Xiao
          </td>
          <td>2024-12-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the deadliest solid cancers; thus, identifying more effective therapies is a major unmet need. In this study, we characterized the super enhancer (SE) landscape of human PDAC to identify drivers of the disease that might be targetable. This analysis revealed MICAL2 as a super enhancer-associated gene in human PDAC, which encodes the flavin monooxygenase MICAL2 that induces actin depolymerization and indirectly promotes SRF transcription by modulating the availability of serum response factor coactivators myocardin-related transcription factors (MRTF-A and MRTF-B). MICAL2 was overexpressed in PDAC, and high MICAL2 expression correlated with poor patient prognosis. Transcriptional analysis revealed that MICAL2 upregulates KRAS and EMT signaling pathways, contributing to tumor growth and metastasis. In loss and gain of function experiments in human and mouse PDAC cells, MICAL2 promoted both ERK1/2 and AKT activation. Consistent with its role in actin depolymerization and KRAS signaling, loss of MICAL2 also inhibited macropinocytosis. MICAL2, MRTF-A, and MRTF-B influenced PDAC cell proliferation and migration and promoted cell cycle progression in vitro. Importantly, MICAL2 supported in vivo tumor growth and metastasis. Interestingly, MRTF-B, but not MRTF-A, phenocopied MICAL2-driven phenotypes in vivo. This study highlights the multiple ways in which MICAL2 impacts PDAC biology and provides a foundation for future investigations into the potential of targeting MICAL2 for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dad317044b8f46c0a8b2261932606b72978f45" target='_blank'>
              MICAL2 Promotes Pancreatic Cancer Growth and Metastasis.
              </a>
            </td>
          <td>
            Bharti Garg, Sohini Khan, Asimina Courelli, Ponmathi Panneerpandian, Deepa Sheik Pran Babu, Evangeline S Mose, K. Gulay, Shweta Sharma, Divya Sood, Alexander T Wenzel, Alexei Martsinkovskiy, Nirakar Rajbhandari, Jay Patel, Dawn Jaquish, Edgar Esparza, Katelin Jaque, Neetu Aggarwal, Guillem Lambies, Anthony D'Ippolito, Kathryn Austgen, Brian Johnston, David A Orlando, Gung Ho Jang, Steven Gallinger, Elliot Goodfellow, P. Brodt, C. Commisso, Pablo Tamayo, Jill P. Mesirov, Hervé Tiriac, Andrew M Lowy
          </td>
          <td>2025-01-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfa96c68ddc7260bde1a2ca148e053288fbd50cb" target='_blank'>
              Fundamentals of DNA methylation in development.
              </a>
            </td>
          <td>
            Caitlyn A Gillespie, Amrin Chowdhury, Katie A Quinn, Michael W. Jenkins, Andrew M Rollins, M. Watanabe, Stephanie M Ford
          </td>
          <td>2024-12-10</td>
          <td>Pediatric research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA polymerases frequently misincorporate ribonucleoside 5′-triphosphates into nascent DNA strands. This study examined the effects of an incorporated ribonucleoside on untargeted mutations in human cells. Riboguanosine (rG) was introduced into the downstream region of the supF gene to preferentially detect the untargeted mutations. The plasmid containing rG was transfected into U2OS cells and the replicated DNA was recovered after 48 h. The mutation analysis using the indicator Escherichia coli RF01 strain showed the frequent induction of untargeted base substitutions at the G bases of 5′-GpA-3′ dinucleotides, similar to action-at-a-distance mutations induced by an oxidatively damaged base, 8-oxo-7,8-dihydroguanine, and an apolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like 3 (APOBEC3) cytosine deaminase. APOBEC3B was then knocked down by RNA interference and the plasmid bearing rG was introduced into the knockdown cells. The untargeted mutations at 5′-GpA-3′ sites were reduced by ~80%. These results suggested that ribonucleosides embedded in DNA induce base substitution mutations at G bases in the same strand by an APOBEC3B-dependent mechanism, implying that ribonucleosides contribute to APOBEC3-dependent cancer initiation events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb0de9186686963f356e0e609ad78d3f431dea9b" target='_blank'>
              Untargeted Mutation Triggered by Ribonucleoside Embedded in DNA
              </a>
            </td>
          <td>
            Tetsuya Suzuki, Kiyoharu Yasui, Yasuo Komatsu, Hiroyuki Kamiya
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e371697d9c914794d0b48ee32bc9a4939073aa5" target='_blank'>
              Oncogene activated human breast luminal progenitors contribute basally located myoepithelial cells
              </a>
            </td>
          <td>
            K. T. Kohler, Jiyoung Kim, R. Villadsen, Lone Rønnov-Jessen, O. Petersen
          </td>
          <td>2024-12-18</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 Breast cancer metastases can go undetected for long periods of time, are highly resistant to therapy, and are the cause for more than 90% of breast cancer mortality. Dormancy is a critical step in the metastatic cascade, whereby disseminated cancer cells (DCCs) enter a non-proliferative state at distal sites until they reactivate and give rise to overt metastasis, often years after disease remission. As such, dormancy represents a major clinical obstacle in breast cancer patient care. Chromatin remodeling serves as a foundation for the cellular reprogramming required to drive cell fate transitions. However, little is known about the nature of this remodeling in dormancy. Here, we hypothesize that differential deposition of histone H3.3 determines whether DCCs enter and remain dormant in secondary organs or proliferate and form overt metastases. We used a combination of H3.3 deposition sensors and genetic tools to manipulate H3.3 and HIRA in vitro and in vivo to track H3.3 deposition through the metastatic cascade and evaluate its importance for dormancy. To evaluate relevance to patient disease we developed a dormancy score to implement from publicly available RNA seq. To define mechanism of regulation, we used a combination of whole genome sequencing (ChIP-seq, ATAC-seq and RNA-seq) followed by in-depth analysis and functional validation. Our data demonstrates that H3.3 incorporation at secondary sites is heterogenous, revealing a subset of DCCs with low H3.3 in chromatin. Blocking H3.3 deposition onto chromatin leads to a reversible arrest in G0/G1 which tracks with the activation of canonical dormancy signaling. Mechanistically, our data supports a model where blocking H3.3 incorporation into chromatin directly represses the expression of SKP2, the main regulator of progression through the restriction point of cell cycle via regulation of p27 degradation. SKP2 suppression in turn leads to the accumulation of p27 thereby causing cell cycle arrest in G0/G1 and dormancy establishment. We traced this H3.3-mediated regulation of dormancy through the dynamic regulation of H3.3 histone chaperone, HIRA, whose levels inversely correlate with dormancy in experimental models as well as in patient samples. We found that HIRA levels respond to the metastatic microenvironment where fibrotic conditions induce HIRA expression and trigger metastatic outgrowth in an H3.3-dependent manner. Together, our work unveils for the first time a patient-relevant epigenetic mechanism, impinging on H3.3 deposition, as a major regulatory point of DCCs’ cell fate at secondary organs controlling both entry of DCC’s into dormant state as well as their reactivation and thriving as overt metastasis.
 Citation Format: Stanislav Drapela, Cristina Megino-Luque, Rojan Chimeh Rad, Devesh Raizada, Nadir Sarigul, Didem Ilter, Brian J. Czerniecki, Jose Javier Bravo-Cordero, Ana P. Gomes. Microenvironment driven dynamic deposition of histone H3.3 controls entry and exit from dormancy in disseminated cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5879de8eccb4afc8b39e0e3158aebd1bfd3c6ac" target='_blank'>
              Abstract A013: Microenvironment driven dynamic deposition of histone H3.3 controls entry and exit from dormancy in disseminated cancer cells
              </a>
            </td>
          <td>
            S. Drápela, Cristina Megino-Luque, Rojan Chimeh Rad, Devesh Raizada, Nadir Sarigul, Didem Ilter, B. Czerniecki, J. Bravo-Cordero, Ana P. Gomes
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Most cancer cells show increased chromosome missegregation, known as chromosomal instability (CIN), which promotes cancer progression and drug resistance. The underlying causes of CIN in cancer cells are not fully understood. Here we found that breast cancer cell lines show a reduced kinetochore localization of ROD, ZW10, and Zwilch, components of the fibrous corona, compared with non-transformed breast epithelial cell lines. The fibrous corona is a structure formed on kinetochores before their end-on attachment to microtubules and plays a role in efficient kinetochore capture and the spindle assembly checkpoint. The reduction in the fibrous corona was not due to reduced expression levels of the fibrous corona components or to a reduction in outer kinetochore components. Kinetochore localization of Bub1 and CENP-E, which play a role in the recruitment of the fibrous corona to kinetochores, was reduced in cancer cell lines, presumably due to reduced activity of Mps1, which is required for their recruitment to kinetochores through phosphorylating KNL1. Increasing kinetochore localization of Bub1 and CENP-E in cancer cells restored the level of the fibrous corona. Cancer cell lines showed a reduced capacity to nucleate microtubules from kinetochores, which was recently shown to be dependent on the fibrous corona, and increasing kinetochore localization of Bub1 and CENP-E restored the microtubule nucleation capacity on kinetochores. Our study revealed a distinct feature of cancer cell lines that may be related to CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54368cadf3707166a1cb068412715ab9475ab2b6" target='_blank'>
              Fibrous corona is reduced in cancer cell lines that attenuate microtubule nucleation from kinetochores.
              </a>
            </td>
          <td>
            Yudai Ishikawa, Hirotaka Fukue, Runa Iwakami, Masanori Ikeda, K. Iemura, Kozo Tanaka
          </td>
          <td>2024-11-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fee039445989781ab8a7ddb4e737986ea83ee7ff" target='_blank'>
              Targeting mitochondrial RNAs enhances the efficacy of the DNA-demethylating agents
              </a>
            </td>
          <td>
            Stephanie Tan, Sujin Kim, Yoo-Ree Kim
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract We previously reported endogenous activation of the DNA damage response (DDR) in the epidermis surrounding basal cell carcinoma resected from Nagasaki atomic bomb survivors, suggesting the presence of genomic instability (GIN) in the survivors as a late effect of radiation. Dual-color immunofluorescence (IF) analysis of TP53-binding protein-1 (53BP1) and a proliferative indicator, Ki-67, to elucidate GIN in tumor tissues revealed that abnormal 53BP1 expression is closely associated with carcinogenesis in several organs. The present study aimed to confirm the presence of radiation-induced GIN in the non-neoplastic epidermis of patients with radiation-induced skin cancer. Formalin-fixed paraffin-embedded tissues were obtained from all participants between 2008 and 2019 at the Nagasaki University Hospital. 53BP1 nuclear expression was examined using dual-color IF analysis with Ki-67 expression to assess the extent and integrity of the DDR. Expressions of gamma-H2AX, p53 and p21 were also analyzed using the dual-color IF analysis for their association with 53BP1. The results of this study provide evidence for sporadic activation of the DDR in medically irradiated and ultraviolet-exposed epidermis as a long-lasting radiation effect, which is a predisposition to skin cancer. Furthermore, the incidence of abnormal 53BP1 expression in cancer cells was higher than in non-neoplastic epidermal cells surrounding cancer, suggesting a correlation between the type of 53BP1 and the malignant potential of skin tumors. This study highlights the usefulness of dual-color IF for 53BP1 (and Ki-67) as an indicator to estimate the level of GIN as a long-lasting health effect of radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab225f148a06d2534be09b0fe51d429a9fdf5fb" target='_blank'>
              Detection of genome instability by 53BP1 expression as a long-lasting health effect in human epidermis surrounding radiation-induced skin cancers
              </a>
            </td>
          <td>
            Katsuya Matsuda, Hirokazu Kurohama, Yutaka Kuwatsuka, Akira Iwanaga, Hiroyuki Murota, Masahiro Nakashima
          </td>
          <td>2024-12-01</td>
          <td>Journal of Radiation Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Inotuzumab Ozogamicin (InO) is an antibody-calicheamicin conjugate with striking efficacy in B-cell acute lymphoblastic leukemia (B-ALL). However, there is wide inter-patient variability in treatment response, and the genetic basis of this variation remains largely unknown. Using a genome-wide CRISPR screen, we discovered the loss of DNTT as a primary driver of InO resistance. Mechanistically, DNTT downregulation attenuated InO-induced DNA damage response, cell cycle arrest, and mitochondrial apoptotic priming, ultimately leading to leukemia resistance to InO. Ex vivo leukemia InO sensitivity was highly associated with DNTT expression in ALL blasts, with substantial intra-leukemia heterogeneity as revealed by scRNA sequencing. In B-ALL patients enrolled in the COG trial AALL1621, we observed consistent DNTT downregulation in residual blasts post-InO treatment. The selection of DNTT-low blasts by InO therapy was also recapitulated in vivo using patient-derived xenograft models. Collectively, our data indicate that DNTT is a key regulator of calicheamicin response in leukemia and thus a potential biomarker for individualizing InO therapy in B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0d43dca72c75e6c736c5bab87e5881541ce7f" target='_blank'>
              DNTT-mediated DNA Damage Response Drives Inotuzumab Ozogamicin Resistance in B-cell Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Carolin S Escherich, Takaya Moriyama, Zhenhua Li, Yu-Chih Hsiao, Wenjian Yang, Yizhen Li, Noemi Reyes, Megan Walker, Amit Budhraja, Sheetal Bhatara, E. Diaz-Flores, Wendy Stock, E. Paietta, M. Konopleva, S. Kornblau, M. Litzow, H. Inaba, Ching-Hon Pui, J. Opferman, M. Loh, Jiyang Yu, Maureen O'Brien, William E Evans, Jun J Yang
          </td>
          <td>2024-12-27</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="It is important to systematically identify tumor suppressor genes (TSGs) to improve our understanding of tumorigenesis and develop strategies for early diagnosis and mitigating disease progression. In the present study, we used an in vivo genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)-CRISPR-associated protein 9 (Cas9) screen and identified FPS/FES-related (FER) as a TSG. Single-cell RNA sequencing (scRNA-seq) revealed that normal cells with low FER expression exhibited elevated malignant transformation potential and stemness properties. FER knockout promoted the tumorigenic transformation, characterized by high colony-forming efficiency and suspension growth ability, acquired tumorigenicity in vivo, increased metabolic activity, dedifferentiation properties, and immune evasion. Moreover, analysis revealed that low FER expression tumors share molecular phenotypes with FER knockout cells, suggesting the consistent role of FER in tumor initiation and progression. Taken together, our findings not only provide insights into the essential role of FER as a tumor suppressor in tumor initiation and progression but also highlight its potential as a target for future clinical diagnosis. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0ef168e6b6a500c7fdbd949d7e67ca980d8052" target='_blank'>
              CRISPR-Cas9 screening identifies the role of FER as a tumor suppressor.
              </a>
            </td>
          <td>
            Jiaqi Wang, Ran Yang, Feng-Po Wang, Junlei Zhang, Yutong Dong, Jiangjun Wang, Meng Yu, Yixiao Xu, Lianlian Liu, Yuda Cheng, Chen Zhang, Yi Yang, Wubin Yang, Jiali Wang, Guangxing Chen, Yi Huang, Yanping Tian, Rui Jian, Bing Ni, Wei Wu, Y. Ruan
          </td>
          <td>2024-12-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="According to the principles of Evolutionary Cancer Cell Biology (ECCB), cancer stem cells (CSCs) do not derive from normal stem cells; rather, they originate from a distinct functional phenotype of germline cells characterized by asymmetric cell division (ACD). This phenotype proliferates through ACD, producing self-renewing cells alongside non-proliferating daughter cells with CSC qualities. ECCB posits that CSCs do not proliferate. Similar to protists, there exists a close reciprocal relationship between sister cells that collectively form a germline and stem cells. These sister cells perform complementary roles: proliferating cancer germline cells generate stem cells, whereas the non-proliferating CSCs give rise to progenitor cells for the formation of new germline clones. ECCB distinguishes between primary CSCs, which are associated with carcinogenesis and primary tumors, and secondary CSCs, which are linked to metastases. This unicellular stem cell system is homologous to that of parasitic protists, such as amebae. Both CSCs and amebae stem cells are produced by an oxygen-sensitive germline and are vulnerable to damage when oxygen levels exceed 6.0% (germline hyperoxia), as elevated oxygen concentrations can harm the germline genome. Germline cells that lose their stemness quality continue to cycle through defective symmetric cell divisions (DSCD). However, to restore functionality, the DSCD genome must be repaired through hyperpolyploidization. This process occurs in native polyploid giant cancer cells, which are homologous to the multinucleated genome repair structures found in protists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22fd9b25c7727535490357470b16f77d5c7c81c2" target='_blank'>
              Reevaluating cancer stem cells and polyploid giant cancer cells from the evolutionary cancer cell biology perspective
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2024-11-21</td>
          <td>Cancer Plus</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c2410d9b633d4c17e5953af5897b6315595eef9" target='_blank'>
              KDM1A-mediated ZFP64 demethylation activates CENPL to promote epithelial ovarian cancer progression.
              </a>
            </td>
          <td>
            Jie Wang, Xinjian Fang, Yajun Xing, M. Ding, Liangxue Zhu, Mingyun Wang
          </td>
          <td>2024-12-01</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) is the most lethal tumor arising from thyroid follicular epithelium. Lenvatinib is an off-label use option for ATC patients in many countries but an approved prescription in Japan. However, lenvatinib resistance is a substantial clinical challenge. Clinical ATC samples including lenvatinib-resistant tumors are used to build patient-derived cells and patient-derived xenografts. High-throughput drug screening and synergy analyses are performed to identify an effective combination partner for lenvatinib. Cellular functions are detected by cell senescence, apoptosis, cell cycle, cell viability and colony formation assays. CDK2 inhibition showed the significant synthetic lethality with lenvatinib via inhibiting G1/S transition and inducing cell senescence in ATC. High expression of CDK2 is associated with lenvatinib resistance and poor clinical outcomes of ATC patients. Lenvatinib increased protein expression of CDK2 in lenvatinib-resistant ATC cells. Mechanistically, lenvatinib inhibited protein degradation of CDK2 via reducing CDK2's interaction with the RACK1-FBW7 complex, which is involved in ubiquitination and subsequent proteasomal degradation of CDK2. Combination of CDK2 inhibitors in clinical trials (Dinaciclib or PF-07104091) and lenvatinib markedly suppressed growth of xenograft tumors from the lenvatinib-resistant patient. The findings support the combination therapy strategy of lenvatinib and CDK2 inhibitor for lenvatinib-resistant ATC patients with high CDK2 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e2b7e2d82a92cc7bf250d9214904a5c6b09e971" target='_blank'>
              Targeting CDK2 Confers Vulnerability to Lenvatinib Via Driving Senescence in Anaplastic Thyroid Cancer.
              </a>
            </td>
          <td>
            Ben Ma, Youzhou Sang, Xiaoxue Du, Yanzhi Zhang, Min Yin, Weibo Xu, Wanlin Liu, Jiayi Lu, Qing Guan, Yunjun Wang, Tian Liao, Yuting Wang, J. Xiang, Rongliang Shi, N. Qu, Qinghai Ji, Jiwei Zhang, Dongmei Ji, Yu Wang
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Telomeres protect the terminal ends of eukaryotic chromosomes, making them essential for genome stability, cellular senescence, and species evolution. These structures consist of tandem repeats of nucleotides, referred to as telomeric repeat sequences. The enzyme telomerase, along with various telomere-binding proteins, manages the synthesis of these repeat sequences. Telomerase functions by adding telomeric repeats to the ends of chromosomes, counteracting the shortening that occurs during DNA replication. In most somatic cells, telomerase activity is generally low or absent, leading to gradual telomere loss and, eventually, cellular senescence. Although the ancestral telomeric repeat sequence is often proposed to be TTAGGG, studies have shown that convergent evolution has produced similar telomeric motifs in various evolutionary lineages. This is evidenced by variations in telomeric repeat sequences among organisms, which may include single or double nucleotide substitutions, deletions, and even exceptionally large telomeric repeats in certain species. Investigating these telomeric repeat sequences across a wide range of organisms and identifying the evolutionary changes within each phylum offer valuable insights into telomeric sequences and the telomerases responsible for their synthesis. Such research not only deepens our understanding of telomere biology across different organisms but also holds potential for addressing age-related challenges and cancer, paving the way for innovative solutions to some of the most critical issues in contemporary medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c54b2377d4f8de003848b40b972b9328835" target='_blank'>
              A Comprehensive Review on the Telomeric Repeat Sequence in Different Organisms
              </a>
            </td>
          <td>
            Monish Prasanna, Kousik Varadan, Achsha Babu, Arun Arumugaperumal
          </td>
          <td>2024-11-18</td>
          <td>Medinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c804fe505d873573d79f164c86523ae09b15f4c" target='_blank'>
              PCNA and Rnh1 independently participate in the protection of mitochondrial genome against UV-induced mutagenesis in yeast cells
              </a>
            </td>
          <td>
            Martyna Latoszek, Katarzyna Bagińska-Drabiuk, E. Sledziewska-Gojska, Aneta Kaniak-Golik
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract To develop a novel therapeutic approach targeting DNA repair mechanisms for isocitrate dehydrogenase (IDH) mutant gliomas, we examined the combined effects and mechanisms of poly(ADP-ribose) glycohydrolase (PARG) inhibitors (PARGi) and radiation therapy (IR). Poly(ADP-ribose) polymers (PAR chains) are essential DNA repair enzymes synthesized by poly(ADP-ribose) polymerase (PARP) using nicotinamide adenine dinucleotide (NAD) as a substrate. PAR chains are degraded by PARG, and their length is precisely controlled by the balance of PARP and PARG activities, playing crucial roles in DNA damage response and cellular stress reactions. Previous studies showed that the combination of PARGi and temozolomide (TMZ) exhibited selective cytotoxicity in IDH mutant glioma cells, and our study confirmed cell death due to a novel DNA repair vulnerability independent of NAD depletion. Experiments using patient-derived IDH mutant glioma cell lines demonstrated that the combination of PARGi and IR exhibited strong cytotoxicity towards IDH mutant cells. Additionally, lentivirus-mediated PARG knockout (KO) experiments showed high radiation sensitivity in PARG KO cells, and partial recovery of cytotoxicity with IDH inhibitors. These results suggest that the combination of PARGi and IR has synergistic effects specifically in IDH mutant cells. Furthermore, we analyzed the expression of DNA damage markers PAR and γH2AX over time using western blotting and immunofluorescence, and confirmed tumor reduction in a mouse model with IDH mutant PARG-KO cell lines treated with IR. PARGi, which have high selectivity and bioavailability, are expected to have lower systemic toxicity compared to conventional alkylating agents, and the synergistic cytotoxic effects could potentially reduce postoperative radiation doses. Particularly, since recurrent tumors after TMZ treatment often acquire mutations in DNA repair enzymes, this therapeutic strategy offers a promising new approach based on novel mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6d19ee447efb25e764131bc8f12c5b2fdfef45" target='_blank'>
              10054- CBMS-1 TARGETING POLY(ADP-RIBOSE) TO INDUCE SELECTIVE CELL DEATH IN IDH-MUTANT GLIOMAS
              </a>
            </td>
          <td>
            Y. Kitagawa, Ali M Nasser, Ethan Wetzel, Julie Miller, Hiroaki Wakimoto, Cahill Daniel
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chemical modification of RNAs is important for posttranscriptional gene regulation. The METTL3-METTL14 complex generates most N6-methyladenosine (m6A) modifications in messenger RNAs (mRNAs), and dysregulated methyltransferase expression has been linked to cancers. Here we show that a changed sequence context for m6A can promote oncogenesis. A gain-of-function missense mutation from patients with cancer, METTL14R298P, increases malignant cell growth in culture and transgenic mice without increasing global m6A levels in mRNAs. The mutant methyltransferase preferentially modifies noncanonical sites containing a GGAU motif, in vitro and in vivo. The m6A in GGAU context is detected by the YTH family of readers similarly to the canonical sites but is demethylated less efficiently by an eraser, ALKBH5. Combining the biochemical and structural data, we provide a model for how the cognate RNA sequences are selected for methylation by METTL3-METTL14. Our work highlights that sequence-specific m6A deposition is important and that increased GGAU methylation can promote oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb873e5c13473ef69e6a927a44ac7bf8be076b4" target='_blank'>
              Cancer mutations rewire the RNA methylation specificity of METTL3-METTL14
              </a>
            </td>
          <td>
            Chi Zhang, Robyn L Scott, Luiza Tunes, Meng-Hsiung Hsieh, Ping Wang, Ashwani Kumar, Brijesh B Khadgi, Yen-Yu Yang, Katelyn A Doxtader Lacy, E. Herrell, Xunzhi Zhang, Bret M. Evers, Yinsheng Wang, Chao Xing, Hao Zhu, Yunsun Nam
          </td>
          <td>2024-12-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [10, 2],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>