<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-07 09:31:38 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>36</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>73</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>341</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>299</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of prostate cancer that strongly correlates with metastatic burden and appears prominently in both primary cancer and metastatic disease. Low Gleason score primary prostate tumors display pervasive centrosome loss, a known mechanistic driver of CIN, that disrupts normal spindle assembly and increases mitotic errors. Previously, we found that transient depletion of centrosomes in immortalized, non-tumorigenic prostate epithelial cells (PrEC) induced a burst of CIN, generating cell lines capable of forming xenograft tumors. Here, we use a multi-omics approach to identify the oncogenic alterations caused by transient centrosome loss. By integrating genomic and transcriptomic data of the prostate lines, we identified a consensus set of focal copy-number variations (CNVs) induced by centrosome loss in cultured cells that are also detectable within a subset of samples from a prostate cancer patient cohort. Using this CNV signature, we were able to derive a unique transcriptomic signature from prostate cancer patient samples that showed strong predictive value for adverse clinical outcomes. In summary, our experimental system uses centrosome loss to promote a punctuated burst of genomic crisis that is characteristic of genome evolution during prostate cancer progression. Consequently, this prostate cancer model produced recurrent structural variations that are detectable in patient samples and associate with worse outcomes. Graphic Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7263ae00a48684dacddf5d2cc713b37a7dad7c68" target='_blank'>
              Transient centrosome loss in cultured prostate epithelial cells induces chromosomal instability to produce an oncogenic genotype that correlates with poor clinical outcomes
              </a>
            </td>
          <td>
            Jiawen Yang, Diogo de Oliveira Pessoa, John M. Ryniawec, Matthew R. Coope, Daniel W. Buster, Emily Loertscher, Mengdie Wang, Chen Chen, A. Cress, Gregory C Rogers, Megha Padi
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46c73bfec534e9d57e00ca368fdb0c9b2c13fa38" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Bishakha Joyeeta Saha, Julia Thiel, Zhangshuai Dai, Meghana Manjunath, Nadine Schacherer, Yara Souto, A. Abdelrahman, Frank Essmann, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Platinum (Pt) drugs are widely utilized in cancer chemotherapy. Although cytotoxic and resistance mechanisms of Pt drugs have been thoroughly explored, it remains elusive what factors affect the receptiveness of DNA to drug-induced damage in nuclei. Here, we demonstrate that nuclear locations of chromatin play a key role in Pt drug-induced DNA damage susceptibility in vivo. By integrating data from damage-seq experiments with 3D genome structure information, we show that nuclear locations of chromatin relative to specific nuclear bodies and compartments explain patterns of cisplatin DNA damage susceptibility. This aligns with observations of cisplatin enrichment in biomolecular condensates at certain nuclear bodies. Finally, 3D structure mapping of DNA damage reveals characteristic differences between nuclear distributions of oxaliplatin-induced DNA damage in drug resistant versus sensitive cells. DNA damage increases in gene-poor chromatin at the nuclear periphery, while it decreases in gene-rich regions located at nuclear speckles. This suggests a strategic redistribution of Pt drug-induced damage in nuclei during chemoresistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad670ec02d61bb33260095b67444d0174970be28" target='_blank'>
              3D genome organization shapes DNA damage susceptibility to platinum-based drugs
              </a>
            </td>
          <td>
            Ye Wang, Aslı Yıldırım, L. Boninsegna, Valentina Christian, Sung-Hae L Kang, X. J. Zhou, Frank Alber
          </td>
          <td>2025-05-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aebba9c32cce818d2e97fcfa124c22eff76b4f3" target='_blank'>
              LSH-mediated resolution of R-loops mitigates transcription-replication conflicts to preserve genomic stability in prostate cancer cells
              </a>
            </td>
          <td>
            Kai Ni, Yu Zhang, Kun Zheng, Jianwen Huang, Nuorong Xiong, Rong Liang, Yingxiao Chen, Honglin Jin, Youlong Hai, Wang Li, Jiayi Ma, Ranxing Yang, Guanggai Xia, Yueqing Bai, Lujie Song, Xiaoyong Hu, Zijian Tang, Qiang Fu
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b41608370ae074e53fc257e2ae8f3d6f11f6ed" target='_blank'>
              The ubiquitin protease Ubp10 suppresses the formation of translocations at Cdc13 binding sites
              </a>
            </td>
          <td>
            David I. Gonzalez, Allison R. Westerbeek, Esther A. Epum, K. Friedman
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer, closely associated with tumor evolution, metastasis, immune evasion, and resistance to treatment. CIN is driven by persistent chromosome missegregation, resulting in abnormal chromosomal copy numbers and promoting tumor progression through mitotic errors, faulty chromosome segregation, and the formation of micronuclei or chromosome bridges. Previous studies have demonstrated that p62 localizes to micronuclei, where it interacts with mitochondria, enhancing ROS-mediated cysteine oxidation and promoting p62 homo-oligomerization. This disrupts repair by recruiting CHMP7 and other ESCRT-III components, impairing their function in restoring the micronuclear envelope. Notably, we observed a time-dependent increase in nuclear p62 levels following radiation therapy, which correlated with enhanced chromatin fragmentation and chromothripsis. While p62 inhibits homologous recombination repair and promotes non-homologous end joining, we hypothesize that radiation-induced nuclear p62 accumulation impairs nuclear envelope resealing by inhibiting ESCRT-III components, thereby contributing to chromosomal fragmentation and chromothripsis. These chromosomal alterations may play a significant role in tumor evolution and treatment resistance. Our findings suggest that irradiation-induced p62 accumulation is crucial for chromothripsis and may affect nuclear repair processes, impacting chromosomal stability. Future research will focus on elucidating how p62 contributes to these abnormalities and their implications for tumor progression and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a93ad5e20684542464f1d1902962e0915d7957" target='_blank'>
              Irradiation-induced increase in nuclear p62 levels contributes to chromosomal fragmentation and chromothripsis.
              </a>
            </td>
          <td>
            Xiaojuan Yang, Ying Zhou, Buzhe Zhang, Xue Tao, Weili Qi, Hong Wu
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How germ cells balance retrotransposon-induced genome innovation with the need for genomic integrity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR-retrotransposon nomad completes the full mobilization cascade—including mRNA export, protein translation, and reverse transcription—to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings suggest that ecDNA formation acts as a protective mechanism to sequester retrotransposon-derived DNA and prevent widespread genomic integration during spermatogenesis, thereby preserving genome stability while allowing limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e357c0e1274581f338acdf17e4b08fe74b5c828" target='_blank'>
              Safeguarding spermatogenesis from retrotransposon insertions by forming ecDNA
              </a>
            </td>
          <td>
            Lauren Tracy, Z. Zhang
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal 11q13.3 amplification is the most common gene copy-number variation event in head and neck squamous cell carcinoma (HNSCC) that corresponds with poor prognosis. Although cyclin D1, a G1/S phase cell-cycle regulatory protein at this locus, is considered as a key driver of malignant progression, further exploration is needed to develop more effective targets for cases with this amplification. Using CRISPR-based gene knockout screening of genes located in chr11q13.3, we found that loss of the gene encoding the Fas-associated death domain (FADD) protein, a well-recognized adapter to caspase-8 that induces cell apoptosis, significantly reduced cancer cell proliferation. FADD expression was elevated in chr11q13.3-amplified tumors and correlated with poor prognosis. RNA sequencing, mass spectrometry, and proteomics analyses revealed a direct relationship between FADD and the DNA helicase MCM5 in the S phase. FADD and cyclin D1 acted at different stages of the cell cycle to synergistically induce proliferation, and caspase-8 deficiency was required for the oncogenic activity of FADD. In a patient-derived xenograft model with chr11q13.3 amplification, combined administration of the DNA helicase complex inhibitor and CDK4/6 inhibitor effectively curtailed tumor growth. Overall, this study identified a nonclassic oncogenic role for FADD in mediating tumor progression in HNSCC and provided a feasible treatment option for patients with chr11q13.3 amplification. Significance: FADD promotes progression of tumors with chr11q13.3 amplification by binding to the DNA helicase complex, which can be targeted in combination with cyclin D1 as a viable therapeutic strategy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d12eb40d352b5dd3f41e425053f82e55565a55cc" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Emerging evidence suggests that dysregulated RNA-binding proteins (RBPs) are associated with a wide variety of cancers. However, the exact roles and pathways of RBPs in the tumorigenesis of hepatocellular carcinoma (HCC), the most common subtype of liver cancer, remain largely unknown. Here, we systematically searched for altered RBP candidates in HCC through multi-omics data integrative analyses and identified that GPATCH4 gene is amplified in >70% HCC patients and its high expression predicts poor prognosis. We mapped the in vivo RNA binding sites of GPATCH4 by iCLIP-seq and characterized that GPATCH4 primarily bound ribosomal RNA (rRNAs). GPATCH4 promoted HCC cell proliferation and transformation both in vitro and in vivo through increasing rRNA transcription and global protein synthesis. GPATCH4 is mainly localized in the nucleolus and helps to unwind RNA loops formed at the rDNA through interacting with DDX21 via its C-terminal intrinsically disordered region. Removal of accumulated R-loops induced by GPATCH4 depletion rescued decreased rRNA transcription and cell proliferation. Taken together, we characterized the understudied GPATCH4 as an RBP with oncogenic function in HCC and revealed a new mechanism by which GPATCH4 functions as a regulator of nucleolar R-loops to control rRNA transcription through interacting with DDX21.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76262c7c41bf119badd0461b097958f0ba08c39" target='_blank'>
              GPATCH4 functions as a regulator of nucleolar R-loops in hepatocellular carcinoma cells
              </a>
            </td>
          <td>
            Yi-Ming Zhao, Yan Jiang, Jin-Zhu Wang, Shang Cao, Hong Zhu, Wei-Kang Wang, Jian-Cheng Yu, Jiaquan Liu, Jingyi Hui
          </td>
          <td>2025-05-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles are unique, allowing sample stratification into ATRT subtypes. The TE activity signature in ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC shows broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC is also unique in having significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and-MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88b1746e8b34806ab31721a10a88339f9bbc0904" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c01f9fcd4978a181214c5cc7b38fe087c3cef1" target='_blank'>
              ZCCHC4 Promotes Translation of Replication-dependent Histone mRNAs by Recruiting Cytoplasmic eIF3 complex
              </a>
            </td>
          <td>
            Ruiqi Wang, Xiaoyan Shi, Yangyi Zhang, Yuci Wang, Yanlan Cao, Rui Feng, Chen Chen, Yingchun Zhang, Hao Chen, Honghui Ma
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription-replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription-induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription-Replication Conflicts: Unlocking New Frontiers in Cancer.
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc53004c4a4eae46c08e03815bc43c29345a6bec" target='_blank'>
              The landscape and consequences of transcription stress
              </a>
            </td>
          <td>
            Osama Hidmi, Sara Oster Flayshman, Diala Shatleh, Jonathan Monin, Rami I. Aqeilan
          </td>
          <td>2025-05-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ploidy determination across the genome has been challenging for low-pass-WGS tumor-only samples. We present BACDAC, a method that calculates tumor ploidy down to 1.2X effective tumor coverage. Allele fraction patterns displayed in the Constellation Plot verify tumor ploidy and reveal subclonal populations. BACDAC outputs a metric, 2N+LOH, that when combined with ploidy better distinguishes near-diploid from high-ploidy tumors. Validated using TCGA, BACDAC had good agreement with other methods and 88% agreement with experimental methods. Discrepancies occur mainly when BACDAC predicts diploidy with subclones rather than high-ploidy. Applied to 653 low-pass-WGS samples spanning 12 cancer subtypes, BACDAC calls 40% as high-ploidy. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03599-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a5801328a2687dfdef502362bf6b982a0712bd" target='_blank'>
              Tumor ploidy determination in low-pass whole genome sequencing and allelic copy number visualization using the Constellation Plot
              </a>
            </td>
          <td>
            Sarah H. Johnson, J. Smadbeck, Roman M. Ženka, Michael T. Barrett, Athanasios Gaitatzes, Arnav Solanki, Angela B. Florio, M. Borad, John C. Cheville, G. Vasmatzis
          </td>
          <td>2025-05-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Osimertinib, a third-generation EGFR tyrosine kinase inhibitor (EGFR-TKI), has dramatically transformed the treatment landscape for patients with EGFR-mutant NSCLC. However, the long-term success of this therapy is often compromised by the onset of acquired resistance, with non-genetic mechanisms increasingly recognized as pivotal contributors. Here, we exploit a multi-omics approach to profile genome-wide chromatin accessibility and transcriptional landscapes between drug sensitive and resistant EGFR-mutant cells. Our findings reveal a robust concordance between epigenetic regulome and transcriptomic changes that characterize the osimertinib resistant state. Through CRISPR-based functional genomics screen targeting epigenetic regulators and transcription factors, we uncover a critical regulatory network featuring key members of the NuRD and PRC2 complexes that mediate resistance. Most critically, our screen identifies FOSL1 and JUN, two subunits of the AP-1 transcription factor within this network, as the most significant hits. Mechanistically, we demonstrate that cis-regulatory elements exhibiting altered chromatin accessibility in the resistant state are enriched for cognate AP-1 binding motifs, enabling AP-1 to orchestrate a gene expression program that underpins the druggable MEK/ERK signaling axis, potentially enhancing cell viability and fitness of resistant cells. Importantly, genetic depletion or pharmacological inhibition of AP-1 reinstates cellular and molecular sensitivity, and reverts resistance-associated phenotypes, such as epithelial-to-mesenchymal transition, upon anti-EGFR rechallenge by repressing AKT and ERK signaling. These findings provide novel insights into the epigenetic and transcriptional control of osimertinib resistance in EGFR-mutant NSCLC, highlighting AP-1 as a targetable vulnerability of resistance-related hallmarks and offering a promising avenue for developing resistance reversal strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b98b05c64d117a31d26404e164fc0fbed4525adc" target='_blank'>
              Integrative multi-omics identifies AP-1 transcription factor as a targetable mediator of acquired osimertinib resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            B. Dayanc, Sude Eris, Nazife Ege Gulfirat, Gulden Ozden-Yilmaz, E. Cakiroglu, Ozlem Silan Coskun Deniz, G. Karakülah, Serap Erkek-Ozhan, Serif Senturk
          </td>
          <td>2025-05-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6786fefc42e4d77082eb7a0b7de5ec1cad7e96" target='_blank'>
              MITF maintains genome stability in non-melanocytic cell lineages and suppresses Hippo pathway signaling
              </a>
            </td>
          <td>
            Drifa H. Gudmundsdottir, Adrián López García de Lomana, Thejus B. Venkatesh, Kritika Kirty, S. Sigurdsson, Linda Vidarsdottir, R. Dilshat, Erla Sveinbjornsdottir, Snædís Ragnarsdóttir, Daniel H Magnusson, Maria R. Bustos, E. Steingrímsson, Thorkell Gudjónsson, Stefan Sigurdsson
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosome instability leading to aneuploidy and accumulation of copy number gains or losses is a hallmark of cancer. Copy number alteration (CNA) signatures are increasingly used for cancer risk stratification, but size thresholds for defining CNAs across cancers are variable and the biological and clinical implications of CNA size heterogeneity and co-occurrence are incompletely understood. Here we analyze CNA and clinical data from 691 meningiomas and 10,383 tumors from The Cancer Genome Atlas to develop cancer- and chromosome-specific size-dependent CNA and CNA co-occurrence models to predict tumor control and overall survival. Our results shed light on technical considerations for biomarker development and reveal prognostic CNAs with optimized size thresholds and co-occurrence patterns that refine risk stratification across a diversity of cancer types. These data suggest that consideration of CNA size, focality, number, and co-occurrence can be used to identify biomarkers of aggressive tumor behavior that may be useful for individualized risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa10dcbb2ebaf779a0622a82815ec7ac7b25cca" target='_blank'>
              Pan-cancer copy number analysis identifies optimized size thresholds and co-occurrence models for individualized risk stratification
              </a>
            </td>
          <td>
            Minh P Nguyen, William C Chen, K. Mirchia, A. Choudhury, Naomi Zakimi, Vijay Nitturi, Tiemo J. Klisch, Stephen T. Magill, Calixto-Hope G. Lucas, Akash J Patel, David R Raleigh
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer.
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, Ping Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Segmental duplications (SDs) are blocks of genomic DNA with high sequence homology that are hotspots for chromosomal rearrangements, coinciding with copy-number and single-nucleotide variations in the population. SDs could represent unstable genomic regions that are susceptible to somatic alterations in human cancers. Here, we aimed to elucidate the genomic locations of SDs in relation to cancer-related genes and their propensity for somatic alterations in cancer. The analysis showed that tumor suppressor genes (TSGs) were less associated with SDs compared to non-cancer genes in nearly all mammalian species. TSGs with SDs were larger in size in humans but only modestly conserved among mammals. In humans, the proportion of non-cancer genes with SDs decreased as the gene age increased. However, for TSGs, a loss of association with SDs was apparent among young genes. Pan-cancer analysis revealed that TSGs with SDs were more prone to deletions and structural variations independently of gene size. Re-analysis of publicly available experimental data further revealed that genes with SDs tended to replicate late and were more likely to undergo the error-prone mitotic DNA synthesis upon replication stress. In conclusion, the loss of SDs from TSGs during mammalian evolution protects against tumor formation by reducing somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7dee62f17cffda68dd501cc4ece25c8e474528a" target='_blank'>
              Tumor Suppressor Genes with Segmental Duplications are Prone to Somatic Deletions and Structural Variations.
              </a>
            </td>
          <td>
            Ziheng Huang, Lin Zhang, Huarong Chen, Xiaodong Liu, Likai Tan, Dan Huang, Yingzhi Liu, Yushan Wang, Xinyi Zhang, A. S. Cheng, Maggie Haitian Wang, Wei Kang, K. To, Jun Yu, Ho Ko, Le Yu, Sunny H Wong, Tony Gin, M. T. Chan, Xiansong Wang, W. K. K. Wu
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Imprinted genes are epigenetically regulated in normal tissues to follow monoallelic expression according to the parent of origin of each allele. Some of these patterns are dysregulated in cancer.


RESULTS
We developed a novel computational multi-omic pipeline to evaluate monoallelic and biallelic expression patterns based on matched RNA-seq expression data, whole-exome sequencing information, and copy number data. We analyzed allelic expression of the entire genes, individual isoforms, and each exon of 59,283 autosomal protein-coding and ncRNA genes, with a focus on 94 genes previously reported to be imprinted. We analyzed 108 cell lines from 9 different tumor histologies using molecular data from the DepMap Portal for the Cancer Cell Line Encyclopedia. Allelic expression patterns of imprinted genes and isoforms in tumor cells were variable. We also identified additional genes and isoforms with predominantly monoallelic expression due to a variety of potential mechanisms. We provide a novel public dataset of transcriptome-wide allelic expression patterns in cell lines from diverse tumor categories, which can serve as a resource for future cancer studies. We examined associations of in vitro cell line response to antitumor agents and repurposed drugs with allelic patterns and overall levels of isoform expression of imprinted genes and of additional genes with predominantly monoallelic expression. Drug response was associated with isoform expression patterns of multiple imprinted genes including CPA4, DGCR6, DNMT1, GNAS, GRB10, H19, NAA60, OSBPL5, PHACTR2, and ZFAT, predominantly monoallelically expressed MAP2K5 and BCLAF1, and additional predominantly monoallelically expressed genes. Multiple associations may be related to mechanisms of drug activity, including associations between the response to the DNA damaging agents and allelic expression of ZFAT, CDC27, and BCLAF1 isoforms, and the response to inhibitors of multiple signaling pathways with expression patterns of GNAS isoforms.


CONCLUSIONS
Tumor cells have a range of monoallelic and biallelic expression patterns in both imprinted and non-imprinted genes and are likely affected by the complex interplay among changes in allelic expression, sequence variants, copy number changes, and expression changes of biologically important genes. Multiple isoform-specific patterns of allelic expression were associated with drug response, indicating complex mechanisms of cancer chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e53122e0648c0b26f74137cf51cf8b0a5877a45c" target='_blank'>
              Allelic expression patterns of imprinted and non-imprinted genes in cancer cell lines from multiple histologies
              </a>
            </td>
          <td>
            Julia Krushkal, Travis L. Jensen, George Wright, Yingdong Zhao
          </td>
          <td>2025-05-25</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 8523


 Background:
 Driver-negative lung adenocarcinoma (DN-LUAD) without actionable genomic alterations (AGAs) has a poor prognosis. A deeper understanding of its molecular background, particularly across the entire genome, is crucial for improving risk assessment and therapeutic development.
 Methods:
 We performed deep whole-genome sequencing (WGS; tumor depth: 100–120x, normal depth: 30x, paired-end reads: 150 bp) and RNA sequencing using fresh-frozen tissues from four institutions. DN-LUAD was defined as LUAD without known actionable mutations or gene fusions. We analyzed somatic mutations, copy number alterations (CNAs), and structural variations (SVs). Tumors were classified as whole-genome doubling (WGD) if more than half of the autosomal tumor genome showed at least two copies in the major copy numbers of somatic cells. HRDetect score ≥0.7 was used to determine the presence of homologous recombination DNA repair deficiency (HRD) (Davies H, et al. Nat Med. 2017). Gene Set Enrichment Analysis (GSEA) was used for pathway analysis.
 Results:
 Among the 745 patients (pts), 517 were classified as having DN-LUAD. WGS identified AGAs undetected by whole-exome sequencing (WES) in 33 (4.4%) pts (
 EGFR
 13,
 KRAS
 7,
 BRAF
 5,
 ERBB2
 4,
 MET
 4, and
 HRAS
 1). The frequently observed genomic alterations in DN-LUAD are shown in the Table. WGD was observed in 62.1% (n = 321) of DN-LUADs and was significantly associated with higher TMB, CNA, and SV burden. Tumor suppressor gene (TSG) mutations in
 TP53
 ,
 STK11
 , and
 KEAP1
 , as well as
 CDKN2A
 copy number loss and SVs, were significantly more frequent in WGD pts. HRD was identified in 19 DN-LUAD pts (3.7%), with a significantly higher frequency in the WGD group (WGD vs. without WGD; 4.98% vs. 1.5%, p = 0.016). GSEA showed significant (q < 0.0001) upregulation of cell cycle pathways (E2F targets, G2M checkpoint, and MYC targets) and downregulation of immune pathways (allograft rejection and interferon-gamma response) in DN-LUADs with WGD.
 Conclusions:
 This largest-ever WGS study identified AGAs undetectable by WES and uncovered a subgroup of DN-LUAD characterized by increased genomic instability driven by multiple TSG alterations associated with WGD, which was also more likely to exhibit HRD. WGD was associated with the upregulation of cell cycle pathways and the downregulation of immune pathways. These findings highlight the critical role of WGS in elucidating the pathogenesis of DN-LUAD.





 AllN = 517
 WGDn = 321
 Without WGDn = 196

 p





 Median TMB (Mutations/Mb)
 6.6
 10.2
 3.0
 <0.0001


 Median No. of CNAs
 101
 117
 77
 <0.0001


 Median No. of SVs
 185
 244
 103
 <0.0001



 TP53

 381 (74)
 277 (86)
 104 (53)
 <0.0001



 SMARCA4

 59 (11)
 43 (13)
 16 (8)
 0.094



 STK11

 56 (11)
 42 (13)
 14 (7)
 0.049



 CDKN2A

 44 (9)
 33 (10)
 11 (6)
 0.092



 KEAP1

 40 (8)
 34 (11)
 6 (3)
 0.003



 CDKN2A
 loss

 106 (21)
 78 (24)
 28 (14)
 0.009



 MET
 amplification

 44 (9)
 34 (11)
 10 (5)
 0.045



 CDKN2A

 135 (26)
 95 (30)
 40 (20)
 0.028



 FHIT

 125 (24)
 99 (31)
 26 (13)
 <0.0001




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b5bbd4f2e96da01654006c7177d2c691759a6f" target='_blank'>
              Whole-genome and transcriptome landscape of actionable driver-negative lung adenocarcinoma.
              </a>
            </td>
          <td>
            Masahiro Torasawa, K. Shiraishi, Akifumi Mochizuki, Shingo Matsumoto, Junko Hamamoto, Hirokazu Matsushita, Issei Imoto, Hiroyuki Yasuda, Kazuhisa Takahashi, Yuichi Shiraishi, Masahiro Tsuboi, Koichi Goto, Shun-ichi Watanabe, R. Hamamoto, T. Kohno
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 e17121


 Background:
 Chromosome arm 8q is frequently amplified in prostate cancer, harboring genes that drive tumor progression, therapy resistance, and poor outcomes. Key loci include
 MYC
 , which regulates cell proliferation;
 AR
 , central to prostate cancer biology and resistance; and
 RAD21
 , essential for chromosomal stability. While the prognostic implications of 8q gains have been explored, their role in driving racial disparities in prostate cancer outcomes remains underexplored. This study investigates chromosome 8q amplification as a prognostic biomarker, examining its differential prevalence and impact across racial groups to elucidate its contribution to disparities in disease progression and patient survival.
 Methods:
 Using cBioPortal's clinical and genomic data from Memorial Sloan Kettering Cancer Center, this work examined two datasets from 2021 and 2024. Comprehensive genomic profiling included copy number alterations, mutational co-occurrence of important 8q genes, and mutational analysis. Multivariate Cox regression and Kaplan-Meier survival analysis were conducted to evaluate the impact of 8q amplifications on overall survival (OS) and disease outcomes, stratified by racial demographic groups and disease stages.
 Results:
 In this cohort of 1,577 prostate cancer patients, 652 (41%) exhibited chromosome 8q amplification, which correlated with aggressive tumor phenotypes. The most common amplifications were observed in PTEN (21%), FOXA1 (17%), AR (16%), and MYC (7%),, with notable co-occurrences observed between MYC-RAD21, MYC-NBN, and PRDM14-ELOC (
 P
 = <0.001), linking these alterations to genomic instability. Kaplan-Meier analysis showed patients with 8q gain had significantly inferior overall survival (HR = 2.31, 95% CI: 1.99 - 2.69,
 P
 < 0.001, mOS gain vs neutral: 35mo vs 55mo). Black patients exhibited a higher frequency of 8q gain (10% vs. 6.1%,
 P
 = 0.003) and distinct oncogenic profiles, including MYC (12%), RAD21 (7%), and LYN (4%) alterations. While Black patients experienced inferior overall outcomes, chromosome 8q amplification was a consistent predictor of poor prognosis across all racial groups.
 Conclusions:
 This study establishes chromosome 8q amplification as a robust prognostic biomarker in prostate cancer. The integration of clinical and genomic data highlights the potential of 8q amplification to guide personalized therapeutic strategies. These findings underscore the value of incorporating 8q analysis into clinical risk stratification frameworks to enhance treatment decision-making and address racial disparities in prostate cancer outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844ee49cccb708b6dfbb629567a7cde15db25ac6" target='_blank'>
              Prognostic impact and racial disparities of chromosome 8q gains in prostate cancer: Insights from integrated molecular profiling.
              </a>
            </td>
          <td>
            Chidiebube J. Ugwu, G. Laliotis, U. C. Okere, Nneoma Ubah, Elvis Obomanu, Colton Jones, Muluken Megiso, Nzubechukwu J. Ugochukwu, Gabor Varadi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05df4b99ec2d4ca5afa53e36017d7bbb5db0a668" target='_blank'>
              Functional Genomic Screens Reveal RBBP4 as a Key Regulator of Cell Cycle Progression in TMZ-Resistant Glioblastoma
              </a>
            </td>
          <td>
            Ezgi Yagmur Kala, Filiz Senbabaoglu Aksu, Erdem Ercan, Altar Ozbıyık, Ahmet Cingöz, Ozlem Yedier-Bayram, A. C. Aksu, Ayse Derya Cavga, E. Yilmaz, I. Kok, A. Humeyra, Dur Karasayar, Ibrahim Kulac, Hamzah Syed, M. Philpott, AP Cribbs, T. Bagci-Onder
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48b921317e55f0a7a7e7ce1f3c412ed182dc30b" target='_blank'>
              DNA2 and FANCM function in two distinctive pathways in disrupting TERRA R-loops and suppressing replication stress at ALT telomeres
              </a>
            </td>
          <td>
            Ashwin Ragupathi, Heba Z. Abid, Yun Chen, Rongwei Zhao, Settapong T. Kosiyatrakul, Derin I. Yetil, Julia Neiswander, Rachel Feltman, Shannon Thomas, Benjamin Yusupov, Manrose Singh, Li Zheng, Binghui Shen, Huaiying Zhang, H. C. Chu, Carl L. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulator often disrupted in cancer, yet how promoter methylation dynamics translate into transcriptional changes during cancer progression remains incompletely understood. Here, we employed targeted bisulfite sequencing and RNA-seq on paired tumor and non-tumor tissues from 80 Korean colorectal cancer (CRC) patients to map promoter methylation and gene expression dynamics. Promoters with high baseline methylation in non-tumor tissues tended to become hypomethylated in tumors, while those with low baseline methylation underwent partial hypermethylation. However, these changes did not consistently correlate with gene silencing or activation. Strikingly, promoters marked by Polycomb (PcG+) in non-tumor tissue were prone to hypermethylation yet often remained transcriptionally active in tumors, a paradox most prominent in transcription factor (TF) genes. In contrast, hypermethylation in PcG− promoters was more consistently associated with transcriptional repression. Our findings suggest that epigenetic plasticity at PcG+ TF gene promoters can override the typically repressive effects of DNA methylation, potentially enabling tumors to maintain or enhance the expression of key regulatory genes. This highlights the importance of PcG occupancy in shaping the functional consequences of methylation changes during colorectal tumorigenesis, warranting deeper investigation into how these epigenetic adaptations drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65ef4383977e8a75d030b12da77f7092d6182f23" target='_blank'>
              Transcription factors overcome the repressive impact of Polycomb-associated methylation in tumors
              </a>
            </td>
          <td>
            Min-Kyeong Kwon, Goeun Park, Dayoung Go, Donghyun Park, Sridhar Hannenhalli, Sun Shim Choi
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf2932e90bcc5681634229bcd158ed1d080d8b" target='_blank'>
              Single-cell Characterization of DNA Hydroxymethylation of the Mouse Brain During Aging
              </a>
            </td>
          <td>
            Yali Bai, Tianjiao Yuan, Liuhao Ren, Yuhang Huan, Fa Yang, Yutong Li, Aicui Zhang, Yueyang Liu, Tian Tian, Ningxia Kang, Danni Chen, Xuchao Xie, Qiang Wang, Wei Chen, Yinghui Zheng, Xiaoliang Sunney Xie, Yunlong Cao
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy. Although prognosis is favorable for patients with an early-stage disease, those with recurrent or more advanced disease have low response rates to chemotherapy and poor clinical outcomes. Previously, we have shown that DNA repair gene (NEIL3) is required for retaining replication fork integrity during replication stress. Here, we examined whether the overexpression of NEIL3 in endometrial cancer associated with altered genomic instability, tumor immunogenicity and anti-tumor immunity in endometrial tumor. In this study, we show that endometrial cancer patients with tumors that a have high NEIL3 expression associated with worse overall survival (OS) outcomes in patients. In addition, tumor with high NEIL3 expression is associated with high number of mutation and chromosomal instability. Furthermore, NEIL3 expression in EC tumors positively correlated with mutation of DNA polymerase eta (POLE) and TP53 as well as high expression of replicative polymerases genes (POLE, POLD1 and POLA1). In contrast, tumor with high NEIL3 expression exhibit low tumor immunogenicity and poor anti-tumor immune cell infiltration. Our findings may have important clinical implications for utilizing NEIL3 as a potential prognostic biomarker to stratify EC patients and as a target to enhance immunotherapy response in endometrial cancer. However, our NEIL3 overexpression associated observation still requires further experimental-based scientific validation studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00393-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229a73859c8660361d382a29f27f2f93cf0da384" target='_blank'>
              DNA glycosylase (NEIL3) overexpression associated with low tumor immune infiltration and poor overall patient survival in endometrial cancer
              </a>
            </td>
          <td>
            Cristofer Barry, Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2025-05-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Medulloblastoma is the most common malignant brain tumor of childhood. The highest-risk tumors are driven by recurrent Myc amplifications (Myc-MB) and experience poorer outcomes despite intensive multimodal therapy. The Myc transcription factor defines core regulatory circuitry for these tumors and acts to broadly amplify downstream pro-survival transcriptional programs. Therapeutic targeting of Myc directly has proven elusive, but inhibiting transcriptional cofactors may present an indirect means of drugging the oncogenic transcriptional circuitry sustaining Myc-MB.


METHODS
Independent CRISPR-Cas9 screens were pooled to identify conserved dependencies in Myc-MB. We performed chromatin conformation capture (Hi-C) from primary patient Myc-MB samples to map enhancer-promoter interactions. We then treated in vitro and xenograft models with CDK9/7 inhibitors to evaluate effect on Myc-driven programs and tumor growth.


RESULTS
Eight CRISPR-Cas9 screens performed across three independent labs identify CDK9 as a conserved dependency in Myc-MB. Myc-MB cells are susceptible to CDK9 inhibition, which is synergistic with concurrent inhibition of CDK7. Inhibition of transcriptional CDKs disrupts enhancer-promoter activity in Myc-MB and downregulates Myc-driven transcriptional programs, exerting potent anti-tumor effect.


CONCLUSIONS
Our findings identify CDK9 inhibition as a translationally promising strategy for the treatment of Myc-MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51ba77a6e26484b146e2f28eb319b79477cb1029" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma.
              </a>
            </td>
          <td>
            L. M. Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Etienne P. Danis, Pradeep Bompada, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-05-15</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4f9af4dcbefbfc2af9850cbacc2d2628f2e0e1e" target='_blank'>
              TEITbase: a database for transposable element (TE)-initiated transcripts in human cancers
              </a>
            </td>
          <td>
            Yun Zhang, Jianqi She, Xueyan Hu, Yueqi Jin, Changyu Tao, Minghao Du, Ence Yang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Mutation, deletion, or silencing of genes encoding cellular metabolism factors occurs frequently in human malignancies. Neomorphic mutations in isocitrate dehydrogenases 1 and 2 (IDH1/2) promoting the production of R-2-hydroxyglutarate (R-2HG) instead of α-ketoglutarate (αKG) are recurrent in human brain cancers and constitute an early event in low-grade gliomagenesis. Due to its structural similarity with αKG, R-2HG acts as an inhibitor of αKG-dependent enzymes. These include the JUMONJI family of lysine demethylases, among which KDM4A is particularly sensitive to R-2HG-mediated inhibition. However, the precise molecular mechanism through which inhibition of αKG-dependent enzymes by R-2HG promotes gliomagenesis remains poorly understood. Here, we show that treatment with R-2HG induces cellular senescence in a p53-dependent manner. Furthermore, expression of mutated IDH1R132H or exposure to R-2HG, which leads to KDM4A inhibition, causes telomeric dysfunction. We demonstrate that KDM4A localizes to telomeric repeats and regulates abundance of H3K9(me3) at telomeres. We show that R-2HG caused reduced replication fork progression, and that depletion of SMARCAL1, a helicase involved in replication fork reversal, rescues telomeric defects caused by R-2HG or KDM4A depletion. These results establish a model whereby IDH1/2 mutations cause R-2HG-mediated inhibition of KDM4A, leading to telomeric DNA replication defects, telomere dysfunction, and associated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728014013df3b457cbd616bc46c4a758e940b3f8" target='_blank'>
              R-2-hydroxyglutarate-mediated inhibition of KDM4A compromises telomere integrity
              </a>
            </td>
          <td>
            Florence Couteau, Laurence M Gagné, Karine Boulay, Philippe Rousseau, Mélissa Carbonneau, Mary McQuaid, Jyoti Sharma, C. Sawchyn, Erlinda Fernandez, Dagmar Glatz, Rana Rizk, Marie-Eve Lalonde, Yosra Mehrjoo, Tsz Wai Chu, G. Moquin-Beaudry, Christian Beauséjour, Mikhail Sergeev, Santiago Costantino, D. Avizonis, I. Topisirovic, Nada Jabado, Hugo Wurtele, Chantal Autexier, Frédérick A. Mallette
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The human genome is widely transcribed, with part of these transcribed regions producing stably expressed protein-coding or non-coding RNAs. Long intergenic non-coding RNAs (lincRNAs) are significantly differentially expressed in various cell lines and tissues. However, the influence of their transcription events remains unclear. In this study, we constructed a human genomic interaction network and found frequent interactions between lincRNA genes and protein-coding genes that are highly related to the occupancy of RNA polymerase II on the lincRNA gene. Interestingly, in the human genome interaction networks, the degree of lincRNA genes was significantly higher than that of protein-coding genes. The promoter regions of the protein-coding genes interacting with the lincRNA genes are enriched with R-loop structures, indicating that lincRNA may influence the target genes through R-loop structures. These promoters were enriched in more transcription factor binding sites. Furthermore, the whole network and sub-network could be utilized to explore potential biomarkers of leukemia. We found that zinc finger protein 668 (ZNF668), eosinophil granule ontogeny transcript (EGOT), and glutamate metabotropic receptor 7 (GRM7) could serve as novel biomarkers for acute myeloid leukemia (LMAL). Pasireotide acetate (CAS No. 396091-76-2) represents a potential drug for LMAL patients. These results suggested that potential biomarkers and corresponding drugs for cancer could be identified based on lincRNA–promoter network/sub-network topological parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fd4f873b33226dc62ac979d88627e1f00373ee" target='_blank'>
              Genome-Wide Detection of Leukemia Biomarkers from lincRNA–Protein-Coding Gene Interaction Networks in the Three-Dimensional Chromatin Structure
              </a>
            </td>
          <td>
            Yue Hou, Wei Ning, Muren Huhe, Chuanjun Shu
          </td>
          <td>2025-05-22</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Aim: The over-activation of oncogenes is a critical genetic event in the development and progression of solid malignancies. Gene amplification and specific mutations represent the prominent mechanisms that convert proto-oncogenes into their active, oncogenic forms. C-myc oncogene (gene locus: 8q24.21) regulates crucial cell and tissue functions, whereas its deregulation is implicated in carcinogenesis. Our research aimed to investigate the impact C-myc numerical imbalances on a series of laryngeal squamous cell carcinomas (LSCCs) characterized by different clinic-pathological features. Materials and Methods: We analyzed 55 LSCC archival tissue sections by implementing a dual-color gene/chromosome fluorescent in situ hybridization assay (C-myc/Chr 8 FISH). Results: According to the extracted C-myc gene/Chr 8 signal ratios, we identified 18 cases with numerical imbalances. The majority of them were gene amplifications (pure n=15, amplification and polysomy n=1), whereas the last two (n=2) cases presented multiple gene copies as a result of Chr 8 polysomy. C-myc amplification was correlated with advanced stage. Stage III/IV cases demonstrated the highest levels of gene amplification (p=0.024). Conclusion: C-myc amplification, combined or not with Chr 8 polysomy, is a relatively frequent event in LSCC. This numerical imbalance is associated with an aggressive phenotype, and it seems to be a significant key element among other oncogenes and suppressor genes that form specific genomic signatures in the corresponding patients with LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf169522cd061e72ca9d71b7495c4317a28f61a5" target='_blank'>
              C-myc Oncogene Numerical Imbalances Analysis in Laryngeal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Sotirios Papouliakos, V. Papanikolaou, A. Chrysovergis, E. Tsiambas, D. Roukas, Sofianiki Mastronikoli, G. Tsouvelas, G. Agrogiannis, Nikolaos Kavantzas, Vasileios Ragos, Andreas C. Lazaris, E. Kyrodimos
          </td>
          <td>2025-05-27</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="A characteristic of triple-negative breast cancer (TNBC) is the epigenetic regulation of tumor suppressor genes, leading to TNBC heterogeneity and treatment resistance in patients. TNBC exhibits high methylation rates, leading to the silencing of numerous tumor suppressor genes. DNA methyltransferase inhibitors (DNMTis) have shown limited clinical efficacy in TNBC treatment. This study aims to uncover a target that could be used to reverse the epigenetic silencing of tumor suppressor genes in TNBC. The Western blot analysis demonstrated that ROR1 knockdown, an oncofetal gene, reduced DNMT3A and DNMT3B protein expression in the TNBC cell lines MDA-MB-231 and HCC1806, as well as a non-malignant breast cell line, MCF10A. The reduced representation bisulfite sequencing (RRBS) analysis identified differential methylation of CREB3L1 when ROR1 is knocked down in TNBC cell lines. CREB3L1 is a transcription factor that plays tumor-suppressive roles in TNBC and is commonly epigenetically silenced in patients. This study shows that ROR1 requires pSTAT3 activation to upregulate DNMT3A and DNMT3B expression to induce CREB3L1 epigenetic silencing in TNBC. ROR1 knockdown resulted in the re-expression of CREB3L1 in TNBC cells. The data provide evidence that ROR1 inhibition, in combination with DNMTis, could enhance patient outcomes as a therapeutic approach for TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac855cfb53ff2a45c0c45fe36069b3895f3153ea" target='_blank'>
              Uncovering a Novel Role of ROR1 in the Epigenetic Regulation of Tumor Suppressor Gene CREB3L1 in Triple-Negative Breast Cancer Cells
              </a>
            </td>
          <td>
            Victoria L. Reed, Eric Lalu, Leena Yoon, Norman Fultang, B. Peethambaran
          </td>
          <td>2025-05-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Human development and health depend on the precise expression of relevant genes in specific cells, at precise times and in response to appropriate stimuli. This is known as context-dependent gene regulation and relies on the activities of a diverse 'zoo' of DNA elements within the genome that are collectively called context-dependent cis-regulatory elements (cdCREs). cdCREs may comprise as much as 10% of the genome and include better-known sequences such as enhancers, silencers and promoters that form the basis of complex multicellularity. Diverse vertebrate body plans not only share considerable phenotypic similarities but also cell types, the genes they express and, in a growing number of cases, in the function and nucleotide sequence of cdCREs. The current review will critically evaluate current methodologies to identify cdCREs and re-evaluate a place for comparative genomics amongst them. We will also explore the function of cdCREs and discuss methods of analysing their function in disease-associated physiologies and behaviours using in vivo models such as CRISPR-generated GA mouse lines. Finally, we will study the effects of epigenetic mechanisms such as DNA methylation on cdCRE activity and examine how genetics and epigenetics can interact to alter disease susceptibility. Given that genome-wide association studies (GWAS) suggest that 95% of disease-associated genomic variation reside in the 98% of the less understood noncoding genome, the need to understand the role of conserved vertebrate cdCREs in development and health in vivo has never been more pressing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382d555c7aff11257c9330d9a7647ebcfb9ae6bb" target='_blank'>
              Exploring the roles of conserved context-dependent cis-regulatory elements (cdCREs) in multicellularity, human health and disease.
              </a>
            </td>
          <td>
            Andrew McEwan, Alexander Rattray, Greg Hutchings, Elizabeth A. Hay, Chris Murgatroyd, A. MacKenzie
          </td>
          <td>2025-07-04</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1a766c9476f757f76140a52e67514cf195c7f" target='_blank'>
              Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination.
              </a>
            </td>
          <td>
            K. Zhao, J. Vos, S. Lam, Lillian A Boe, Daniel Muldoon, Catherine Y Han, Cristina Valero, Mark Lee, C. Fitzgerald, A. Lee, Manu Prasad, Swati Jain, Xinzhu Deng, Timothy A Chan, Michael F. Berger, C. Bandlamudi, Xi Kathy Zhou, Luc G. T. Morris
          </td>
          <td>2025-06-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Targeted inhibition of DNA damage response proteins has received significant clinical attention owing to the success of PARP inhibitors. Due to the loss of the G1/S checkpoint, cancer cells are reliant on the G2/M checkpoint to cope with elevated DNA replication stress. We previously demonstrated a single induction of 8-oxo-guanine at telomeres in cancer cells was sufficient to induce replication stress, but was well tolerated at the cellular level. Here, we found inhibition of ATR, Chk1, or Wee1 after induction of telomere oxidative stress significantly induced genome instability and reduced cell viability. This occurred at doses markedly less than those required to increase instability in non-cancer cells. We determined the mechanism of this instability is due to cells progressing through S-phase with telomere damage and exiting G2-phase prematurely, prolonging their mitosis. This study demonstrates targeted oxidative base damage at telomeres can enhance the therapeutic efficacy of ATR inhibition in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520fb2116ae4feadf96591801b8f58200ddc5cec" target='_blank'>
              Oxidative Base Damage to Telomeres Sensitizes Cancer Cells to ATR Inhibition
              </a>
            </td>
          <td>
            Alex Garbouchian, Natalia Cestari Moreno, A. Dey, Patricia Opresko, Ryan Barnes
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Exacerbating the DNA replication problems of cancer cells serves as a viable therapeutic approach. Nevertheless, the cytotoxicity of cancer drugs is often hampered by therapy-induced senescence, leading to unfavorable patient outcomes. Here, we employ acute replisome dysfunction in combination with Ataxia telangiectasia and Rad3-related (ATR) inhibition as a strategy to divert senescent cells toward death by triggering DNA replication catastrophe, a form of irreversible replication fork collapse caused by excessive single-stranded DNA (ssDNA) accumulation. RNA-sequencing revealed a distinct set of p53-responsive genes responsible for death. We identify c-Jun N-terminal kinase (JNK) to be essential for augmenting p53-dependent apoptotic programs and inducing pan-nuclear distribution of γH2AX, together constituting a feed-forward loop to drive cell death. Activation of DNA-PKcs initiates the signaling cascade of replication catastrophe, including CHK1-dependent JNK activation, which relies on MRE11 and PARP1 to expand and recognize ssDNA gaps, defining replication-associated gaps as an underlying basis for replication catastrophe. Our study elucidates the dynamic regulation of proximal and distal effectors along the DNA-PKcs/JNK/p53 axis that govern the cell fate decision between senescence and death. We propose that key determinants of replication catastrophe signaling are targetable vulnerabilities that can be exploited to limit senescent cell populations and increase the efficacy of anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c9be1cc99c822e2708983101c9fab63e3ac941" target='_blank'>
              The DNA-PKcs/JNK/p53 pathway underlies changes in cell fate decision toward death during DNA replication catastrophe
              </a>
            </td>
          <td>
            Jinal A. Patel, Julie Rageul, Natalie Lo, Auntara Nandi, Camryn Zezelic, Cynthia T Lee, Arafat Khan, Hyungjin Kim
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0065672d86f3f94440ecf420ec518fca8aa5a1ec" target='_blank'>
              Integrated Spatial Analysis of Ovarian Precancerous Lesions
              </a>
            </td>
          <td>
            Tu-Yung Chang, Yen-Wei Chien, Szu-Hua Chen, Annelise Sokolow, Yeh Wang, Brant G. Wang, Tricia A Numan, M. Herman Chui, Rebecca Stone, T. Pisanic, Nicholas Papadopoulos, Tian-Li Wang, Christopher Douville, Leslie Cope, I.-M. Shih
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41db77b7599b92a63937c7f3610506611aa97e8f" target='_blank'>
              A survey of human cancer-germline genes : linking X chromosome localization, DNA methylation and sex-biased expression in early embryos
              </a>
            </td>
          <td>
            A. Loriot, Julie Devis, Laurent Gatto, C. De Smet
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is a group of bone and soft tissue cancers in children and young adults. The EWS-FLI1 protein, the oncogenic driver in EwS, is a fusion of the FET protein, EWSR1, at its amino terminus, with the carboxy terminus of the transcription factor, FLI1. Since EwS cells have pronounced sensitivity to radiation and chemotherapy-induced DNA damage, the role of EWS-FLI1 in DNA repair is likely. Here, we demonstrate that EWS-FLI1 causes a defect in microhomology-mediated end-joining (MMEJ) repair. EWSR1 is a splicing factor that promotes the faithful splicing of the POLQ pre-mRNA, required for the expression of POLΘ, a critical protein in the MMEJ pathway. Expression of EWS-FLI1, or loss of EWSR1, causes exon 25 skipping of the POLQ transcript, decreased POLΘ protein expression, impaired MMEJ, and cellular sensitivity to inhibitors of NHEJ repair and HR repair, through the mechanism of synthetic lethality. Knockdown of EWS-FLI1 expression restores POLΘ mitotic foci and increases MMEJ activity. Based on this mechanism, inhibitors of NHEJ or HR may provide a targeted therapy for patients with Ewing sarcoma. Highlights - Ewing sarcoma tumors have a deficiency in POLθ expression and a corresponding loss of MMEJ activity - EWSR1 is a splicing factor that interacts with other splicing factors such as FUBP1 and KHSRP/FUBP2 to accurately splice the POLQ mRNA. - The EWS-FLI1 fusion oncoprotein, or loss of EWSR1, causes a splicing defect, leading to exon25 skipping of the POLQ pre-mRNA and loss of POLθ expression - The MMEJ deficiency of EwS cells results in cellular sensitivity to inhibitors of Homologous Recombination or Non-Homologous End-Joining - Exon 25 skipping of POLQ mRNA is a predictive biomarker for HR inhibitors in human cancers">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1eaf5b79d22ae4a1a31716b8447290969ad6a6" target='_blank'>
              Disruption of Microhomology-mediated End-joining in Ewing Sarcoma
              </a>
            </td>
          <td>
            S. Asada, G. Zhu, J. Abeykoon, Yutaro Tanaka, Huy Nguyen, Y. Hirohashi, Divya R Iyer, N. Ashton, S. Mukkavalli, Lige Jiang, Miles Del Busso, J. Thangaiah, Steven I. Robinson, K. Parmar, E. V. Van Allen, R. Gillani, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Inter-chromosomal interactions play a crucial role in 3D genome organization, yet the organizational principles and functional significances remain elusive. In general, lncRNA loci and transcripts are frequently associated with transcriptional programs modulated by long-range chromatin interactions. Here, we identified a novel lncRNA named Gm26793, which is abundantly distributed in the primitive streak and mesodermal cells of embryonic day 7.5 mouse gastrula. Through genetic ablation of Gm26793, we observed a preferential responsiveness to primitive endoderm lineage during stem cell differentiation, as well as enhanced occurrence of transient and degenerative state cells in early mouse embryos when the cell fate segregates between epiblast and primitive endoderm. Mechanistically, we revealed that the genomic locus of Gm26793, rather than the lncRNA transcript or adjacent gene, governs the cell fate preference towards primitive endoderm. Concretely, Gm26793 locus (Chromosome 7) forms an inter-chromosomal molecular lock with Cubn (Chromosome 2) via CTCF, restraining the expression of Cubn and maintaining a natural epigenetic landscape, thus ensuring the proper lineage specification in vitro and in vivo. Overall, our study provides a clear paradigm that inter-chromosomal interaction collaborates with architectural factors to stabilize nuclear conformation and guarantee faithful gene expression during stem cell differentiation and mammalian embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5737d88dd5a339188134898f4ba7c9851ba9807" target='_blank'>
              Genomic locus of lncRNA-Gm26793 forms an inter-chromosomal interaction with Cubn to ensure proper stem cell differentiation in vitro and in vivo
              </a>
            </td>
          <td>
            Zhiwen Liu, Xin Wan, Jiehui Chen, Yongjian Ma, Yonggao Fu, Yingying Chen, Mingzhu Wen, Yun Yang, Yun Qian, Yong Zhang, Dahai Zhu, Jinsong Li, Naihe Jing, Xianfa Yang
          </td>
          <td>2025-06-03</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Every generation, the human genome is shuffled during meiosis and a single fertilized egg gives rise to all of the cells of the body1. Meiotic errors leading to chromosomal abnormalities are known causes of pregnancy loss2,3, but genetic aetiologies of euploid pregnancy loss remain largely unexplained4. Here we characterize sequence diversity in early pregnancy loss through whole-genome sequencing of 1,007 fetal samples and 934 parental samples from 467 trios affected by pregnancy loss (fetus, mother and father). Sequenced parental genomes enabled us to determine both the parental and meiotic origins of chromosomal abnormalities, detected in half of our set. It further enabled us to assess de novo mutations on both homologous chromosomes from parents transmitting extra chromosomes, and date them, revealing that 6.6% of maternal mutations occurred before sister chromatid formation in fetal oocytes. We find a similar number of de novo mutations in the trios affected by pregnancy loss as in 9,651 adult trios, but three times the number of pathogenic small (<50 bp) sequence variant genotypes in the loss cases compared with adults. Overall, our findings indicate that around 1 in 136 pregnancies is lost due to a pathogenic small sequence variant genotype in the fetus. Our results highlight the vast sequence diversity that is lost in early pregnancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa3651219511c4ef3423241fcef77876aaae5715" target='_blank'>
              Sequence diversity lost in early pregnancy
              </a>
            </td>
          <td>
            G. Arnadottir, Hákon Jónsson, T. Hartwig, J. Gruhn, P. L. Møller, Arnaldur Gylfason, D. Westergaard, Andrew Chi-ho Chan, A. Oddsson, L. Stefánsdóttir, L. Roux, V. Steinthorsdottir, Kristjan H Swerford Moore, S. Olafsson, Pall I. Olason, H. Eggertsson, G. Halldorsson, G. Walters, H. Stefánsson, S. A. Gudjonsson, G. Pálsson, B. Jensson, Run Fridriksdottir, Jesper Friis Petersen, , H. Nielsen, S. Bliddal, K. Banasik, M. Krog, A. Kolte, N. la Cour Freiesleben, S. Ostrowski, Erik Sørensen, Mille Løhr, Markus J. Herregård, E. Hoffmann, J. Gruhn, Andrew Chi-ho Chan, U. Þorsteinsdóttir, Kári Stefánsson, Hákon Jónsson, O. Magnusson, L. Ambye, Lone Schmidt, P. Kamstrup, M. Nyegaard, Ellen Christine Leth Løkkegaard, H. E. Bredkjær, H. W. Hvidman, Agnar Helgason, G. Norddahl, P. D. Rohde, J. Saemundsdottir, O. Magnusson, B. Halldórsson, D. Gudbjartsson, P. Sulem, U. Thorsteinsdóttir, E. Hoffmann, H. Nielsen, Kári Stefánsson
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>2</td>
          <td>184</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="For chromosome abnormalities (CA), such as Down syndrome (DS), the influence of genomic variations to chromosome conformation and gene transcription remains elusive. Based on the complete genomic sequence from the parents of the DS trisomy patient, we systematically delineated an atlas of parental-specific haploid single nucleotide polymorphism (SNP), copy number variation (CNV), three-dimensional (3D) genome, and RNA expression profiles of the diencephalon in the DS patient. The integrated haploid multi-omics analysis demonstrated that one-dimensional genomic variations including SNPs and CNVs in the DS patient are highly correlated with the alterations of the 3D genome and the subsequent gene transcription. The correlation relationship remains valid in haploid-levels. Moreover, we revealed the 3D genome alteration associated mis-regulation of DS-related genes, which facilitates to understanding the pathogenesis of CA. Together, our study contributes to decipher the coding from one-dimensional genomic variations to 3D genomic architecture and the subsequent gene transcription in healthy and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9de44c4aa76967640e3414bc7acd3b2037c35" target='_blank'>
              Deciphering Haploid Chromosome Conformation Alternation in Down Syndrome by Multiple Haploid Omics Analysis.
              </a>
            </td>
          <td>
            Chengchao Wu, Tianshu Zhou, Wenfu Ke, Wei Xiong, Zhihui Zhang, Siheng Zhang, Jinyue Wang, Lulu Deng, Keji Yan, Man Wang, Shenglong He, Qi Gong, Chao Ma, Xiaping Chen, Yan Li, He Long, Chong Guo, Gang Cao, Zhijun Zhang
          </td>
          <td>2025-06-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2397ca3adf8845c9715ac4636c6b9da727b026" target='_blank'>
              Epigenetic dynamics and molecular mechanisms in oncogenesis, tumor progression, and therapy resistance.
              </a>
            </td>
          <td>
            Mohammed Kaleem, Lubna Azmi, N. Shahzad, Murtada Taha, Shiv Kumar, Md. Ali Mujtaba, Abdulaziz Ali H Hazazi, A. Kayali
          </td>
          <td>2025-05-13</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ed51bfd7d0538e3c3fdaf6bc3f9553670a3712" target='_blank'>
              Cancer-Associated USP28 Missense Mutations Disrupt 53BP1 Interaction and p53 Stabilization
              </a>
            </td>
          <td>
            Hazrat Belal, Esther Feng Ying Ng, Midori Ohta, Franz Meitinger
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef02803fbaf5cb1abbd79351dc6a4ca1c68ba79" target='_blank'>
              Constitutional copy number amplifications: rare or under-evaluated? Revisiting a 25-year-old cold case.
              </a>
            </td>
          <td>
            Eliana Salvo, Romano Tenconi, R. Giorda, Sara Bertuzzo, Luca Cesana, Roberta Murru, S. Giglio, Mana M Mehrjouy, Niels Tommerup, Orsetta Zuffardi, M. Bonaglia
          </td>
          <td>2025-06-04</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The human genome encodes 19 adenosine and cytidine deaminase genes, classified as A-to-I versus C-to-U editors. A-to-I editors have been widely identified as a promising therapeutic target in various cancers. Conversely, the investigation into C-to-U editors is relatively limited. This study evaluated RNA-editing genes in prostate cancer (PCa). Notably, the APOBEC3 genes are clustered in terms of their chromosomal locations, and their transcriptional changes exhibit significant positive correlations in both primary PCa and castration-resistant prostate cancer (CRPC). One member of this family, APOBEC3C, is demonstrated here as an androgen receptor (AR)-repressed gene. Consistently, APOBEC3 loci are epigenetically inhibited in PCa progression, with APOBEC3C level lower in PSA-high patients. APOBEC3C-low PCa cohorts exhibit increased resistance to Abiraterone and Enzalutamide. Clinicopathological profiling further confirmed APOBEC3C downregulation along PCa progression to advanced phases (grade IV/V, stage III-IV, and pathological stage T3-4), underscoring its prognostic value. Additionally, APOBEC3C expression inversely correlates with PCa relapse and mortality, and low APOBEC3C levels are linked to unfavorable survival. Notably, integrated analyses identified APOBEC3C as the sole RNA-editing gene with significance in both differential expression and PCa prognosis, and APOBEC3C had the best diagnostic performance among 19 genes. Our efforts provide a foundation for further RNA editors research in PCa diagnosis and therapy, and grant APOBEC3C as a candidate tumor suppressor. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00169-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0fae2048158500534064188ceb1148d343ad87" target='_blank'>
              Characterization of RNA editing gene APOBEC3C as a candidate tumor suppressor in prostate cancer
              </a>
            </td>
          <td>
            Li-Yang Wang, Ji Shi, Mo-Fei Wang, Yi-Meng Liu, Hong-Shan Guo, Jin-Cheng Wang, Shu Jiang, Jia-Qian Liang, Xinghua Liao, Shao-Yong Chen
          </td>
          <td>2025-05-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Endogenized giant viruses are emerging as major contributors to the genome evolution of microbial eukaryotes, with both degraded and fully functional latent viruses being found integrated in diverse lineages. The mechanisms that determine the fate of viral integrants are poorly understood, however. Acanthamoeba is a unicellular eukaryote known for undergoing lateral gene transfer (LGT) with viruses. Here we have leveraged chromosome-scale assemblies of two strains of Acanthamoeba, Neff and C3, to investigate the genomic mechanisms that mediate the fate of viral integrations in eukaryotic genomes. Results Viral integrations in the C3 and Neff genomes are largely non-overlapping and disproportionately found in sub-telomeric regions. Multiple partial copies of these insertions are found throughout the Neff genome, but they are not expressed, do not obviously encode functions associated with their own mobility, and are colonized by host mobile elements. Viral regions are hypermethylated and highly condensed, suggesting that the expression of recently acquired viral DNA is suppressed in heterochromatic regions. Conclusions We propose a model for the trajectory of viral sequences in Acanthamoeba: (i) integration of DNA from giant viruses, (ii) epigenetic suppression of the viral DNAs, allowing them to persist in the genome, and (iii) deterioration of viral genomes by point mutation, mobile element colonization, and intra- and inter-chromosomal recombination. Viral integrations in Acanthamoeba spp. are transient and may not have long-lasting effects on the fitness of the amoeba. Our work highlights the importance of host genome dynamics and epigenetic silencing for understanding the evolution of endogenized viral elements. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02280-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f54d3754da37f780fd27a13a0f10078e856d51" target='_blank'>
              Epigenetic silencing and genome dynamics determine the fate of giant virus endogenizations in Acanthamoeba
              </a>
            </td>
          <td>
            Cédric Blais, Morgan J. Colp, Luke A. Sarre, Alex de Mendoza, John M. Archibald
          </td>
          <td>2025-07-01</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Fluoroquinolones induce double-strand breaks in bacterial DNA, triggering the SOS response, a major DNA damage response that ensures the expression of repair proteins but also promotes the emergence and spread of antibiotic resistance. Fluoroquinolone resistance, particularly in Escherichia coli, is a growing global health concern. Understanding bacterial responses to these antibiotics is critical for developing preventive strategies and novel treatments to combat resistance development. This study investigates DNA morphology in E. coli following exposure to ciprofloxacin (CIP), a fluoroquinolone antibiotic. We show that CIP induces a stepwise DNA reorganization, culminating in a highly dense nucleoid structure at midcell-a process we term DNA supercompaction. This phenomenon occurred also with other genotoxic agents. Live-cell imaging revealed that RecN, a structural maintenance of chromosomes (SMC)-like protein, is required for DNA supercompaction, and that RecN's dynamics and activity in this response depend on RecA. Additionally, RecN and RecA frequently colocalized at nucleoid-associated positions. We suggest that RecN and RecA play active roles in DNA supercompaction following severe DNA damage, that their interplay is part of a prompt universal survival response to DNA double-strand breaks in E. coli, and that the extent of the compaction response depends on the DNA damage severity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506e392d68ea028b658f17b6f901be5aeef9ee4e" target='_blank'>
              RecN and RecA orchestrate an ordered DNA supercompaction response following ciprofloxacin-induced DNA damage in Escherichia coli.
              </a>
            </td>
          <td>
            Krister Vikedal, S. B. Ræder, Ida Mathilde Marstein Riisnæs, Magnar Bjørås, J. Booth, Kirsten Skarstad, Emily Helgesen
          </td>
          <td>2025-05-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background CTCF, a highly studied transcription factor, is essential for chromatin interaction maintenance. Several independent studies report that CTCF interacts with RNAs in vitro and in cells. Yet continuous debates about the authenticity of the RNA-binding affinity of CTCF and its biological role remain in large part due to limited research techniques available, such as CLIP-seq. Result Here, we investigate RNA’s role in CTCF’s transcription factor function through its chromatin occupancy. To systematically explore whether RNAs affect CTCF’s ability to bind DNA, we perturb CTCF-RNA interactions by three independent approaches and examine CTCF genome occupancy by ChIP-seq. Although RNase A and triptolide treatment each affect a certain number of CTCF-binding peaks, few peaks overlap between treatment groups indicating the effect of RNA in regulating CTCF’s DNA binding affinity is variable between loci. In addition, limited transcriptional or chromatin accessibility changes occur between cells expressing wild-type CTCF or CTCF lacking the RNA binding region. Conclusion Our data provide a complementary approach and in silico evidence to consider the significance of RNA affecting CTCF’s DNA binding affinity globally. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03582-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a181f3e3ab1f0c8b8efc73cd9bde644736cf84" target='_blank'>
              Deciphering the role of RNA in regulating CTCF’s DNA binding affinity in leukemia cells
              </a>
            </td>
          <td>
            Judith Hyle, Wenjie Qi, M. Djekidel, Wojciech Rosikiewicz, Beisi Xu, Chunliang Li
          </td>
          <td>2025-05-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, A. Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract MYC proteins are potent oncoproteins that drive tumorigenesis in a wide range of cancers, making it critical to understand their oncogenic functions and underlying mechanisms. Although MYC overexpression induces transcriptional and replication‐associated stress, recent studies have paradoxically identified MYC as a key resilience factor that protects cancer cells from these stressors. In this review, we explore the dual role of MYC in both driving and mitigating cellular stress to achieve its oncogenic function. We also examine how MYC‐induced transcriptional and replicative stress generates potentially immunogenic nucleic acid species while simultaneously helping cancer cells evade host immune recognition. We propose a model in which MYC plays a critical role in managing the stress it induces, thereby maintaining a balance that promotes tumor growth. Based on this model, we discuss potential therapeutic strategies targeting MYC‐dependent stress responses, offering new avenues for cancer treatment and highlighting the complexity of MYC‐driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ce286c658baf7f61e9c844f34e8e87797cdc79" target='_blank'>
              MYC: The Guardian of Its Own Chaos
              </a>
            </td>
          <td>
            A. Gaballa, Bastian Krenz, Leonie Uhl
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Nucleoporin 98 (NUP98) fusion oncoproteins are strong drivers of pediatric acute myeloid leukemia (AML) with poor prognosis. Here we show that NUP98 fusion-expressing AML harbors an epigenetic signature that is characterized by increased accessibility of hematopoietic stem cell genes and enrichment of activating histone marks. We employ an AML model for ligand-induced degradation of the NUP98::KDM5A fusion oncoprotein to identify epigenetic programs and transcriptional targets that are directly regulated by NUP98::KDM5A through CUT&Tag and nascent RNA-seq. Orthogonal genome-wide CRISPR/Cas9 screening identifies 12 direct NUP98::KDM5A target genes, which are essential for AML cell growth. Among these, we validate cyclin-dependent kinase 12 (CDK12) as a druggable vulnerability in NUP98::KDM5A-expressing AML. In line with its role in the transcription of DNA damage repair genes, small-molecule-mediated CDK12 inactivation causes increased DNA damage, leading to AML cell death. Altogether, we show that NUP98::KDM5A directly regulates a core set of essential target genes and reveal CDK12 as an actionable vulnerability in AML with oncogenic NUP98 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac2ff38eb74ff54c2c90e5931c96b450175dda5" target='_blank'>
              Transcriptional and epigenetic rewiring by the NUP98::KDM5A fusion oncoprotein directly activates CDK12
              </a>
            </td>
          <td>
            Selina Troester, Thomas Eder, Nadja Wukowits, M. Piontek, P. Fernández-Pernas, Johannes Schmoellerl, B. Haladik, G. Manhart, M. Allram, Margarita Maurer-Granofszky, Nastassja Scheidegger, K. Nebral, Giulio Superti-Furga, Roland Meisel, B. Bornhauser, P. Valent, Michael N Dworzak, Johannes Zuber, Kaan Boztug, Florian Grebien
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, M. Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37e8efeea3f80c12980365d094cd561d8a7d326" target='_blank'>
              LINE-1 ORF1p is a shared and immunogenic antigen in cancer
              </a>
            </td>
          <td>
            Wilson McKerrow, Megan Snyder, Chong Chu, Deena Kelly, Heike Keilhack, Liyang Diao
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, , M. Balaban, , W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, , E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>None</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Aging and cancer, though distinct biological processes, share overlapping molecular pathways, particularly in epigenetic regulation. Among these, DNA methylation is central to mediating gene expression, maintaining cellular identity, and regulating genome stability. This review explores how age-associated changes in DNA methylation, characterized by both global hypomethylation and focal hypermethylation, contribute to the emergence of cancer. We discuss mechanisms of DNA methylation drift, the development of epigenetic clocks, and the role of entropy and epigenetic mosaicism, in aging and tumorigenesis. Emphasis is placed on how stochastic methylation errors accumulate in aging cells and lead to epiallelic shifts and gene silencing, predisposing tissues to malignant transformation, even despite recently increased cancer incidences at younger ages. We also highlight the translational potential of DNA methylation-based biomarkers, and therapeutic targets, in age-related diseases. By framing cancer as a disease of accelerated epigenetic aging, this review offers a unifying perspective and calls for age-aware approaches to both basic research and clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80ee6ccb4c2cbc13d84b76cf48a4e779ceb4234" target='_blank'>
              DNA Methylation, Aging, and Cancer
              </a>
            </td>
          <td>
            Himani Vaidya, Jaroslav Jelinek, Jean-Pierre J. Issa
          </td>
          <td>2025-06-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The conceptional shifts on genome organization and hereditary variability occurred during transition from classical mendelian to current mobile or dynamic genetics. The main changed premises of this transition are firstly presented in detail. Mendelian genetics mainly conceived genome as the set of chromosomes with of all genes. Now genome semantics is changed. It comprises entire hereditary constitution of the cell, including both structural and dynamic aspects of coding, storage and transfer of species-specific information. There are three kinds of heritable changes: mutations, variations and epigenetic alterations. It is reasonable to discriminate in the genome two subsystems: Obligate genetic elements (OGE) and Facultative genetic elements (FGE). FGEs comprise various kinds of repeated DNA, mobile elements, amplicons, inserted viral and foreign DNA, B-chromosomes and cytobionts. FGEs are predominant genome content of many plants. The number and cell topography of FGEs are different in different cells/tissues and most eukaryote individuals. Changes in the structure or order of OGEs correspond to classical mutations. Various changes in FGEs it is reasonable to call variations. Facultative elements and their variations are the first genomic reaction on biotic and environmental challenges. Together with epigenetic alterations they implement the operational genomic memory. Three template genome processes Replication, Transcription, Translation and three basic genetic processes – Repair, Recombination and Segregation are capable to facultative expression according to principle: the unity of the whole and freedom of the parts. This is the essence of the presented generalized concept of the genome organization and hereditary variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08181d3ed9f40145f41c1bd8a5154dfef8c50437" target='_blank'>
              Beyond Mendel and Morgan to the dynamic genome
              </a>
            </td>
          <td>
            M. D. Golubovsky
          </td>
          <td>2025-05-11</td>
          <td>Vavilovia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b94228db7f216ff8d9ace96b3b303dbe02cf6e0" target='_blank'>
              Circular RNA rising star: new findings of circPRMT5 in cancer.
              </a>
            </td>
          <td>
            DuanZhong Chen, Caiyan Yin, Chao Li, Jun He, Peng Wu
          </td>
          <td>2025-06-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Historically, DNA sequence mutability has been considered relatively uniform and low in tumors with chromosomal instability (CIN), based on the assumption that high mutability would be detrimental in karyotypically aberrant contexts. Recent in silico analyses have challenged this view, suggesting some heterogeneity in mutation rates across CIN tumors; however, these predictions lack experimental validation. It also remains unclear how the intertumor variability of mutation rates compares to intratumor diversification and evolves along disease progression, whether mutation rates are functionally relevant in CIN cancers, and which mutational processes shape mutational accrual during CIN tumor onset and evolution. To address these gaps, we performed mutation accumulation experiments using clonal populations of patient-derived tumoroids from seven CIN, microsatellite-stable colorectal cancers (CRCs), and one microsatellite-unstable CRC. Each tumor exhibited a distinctive mutation rate footprint that was conserved among different clones from the same ancestor. In contrast, mutation rates diverged markedly across different tumors, with variations in magnitude within microsatellite-stable tumors as prominent as those distinguishing them from microsatellite-unstable tumors. New mutations reflected mutational processes associated with defective DNA replication and repair, which were not detected in normal tissues. Last, both mutation accumulation assays and high-depth whole-exome sequencing of subclonal variants showed higher mutation rates in metastatic lesions compared with matched primary tumors, suggesting positive selection for cells with increasing mutability during cancer dissemination. By providing an empirical assessment of mutation rates in human cancer, our data delineate heterogeneity, heritability, and progression-associated evolvability of DNA mutational instability as hallmarks of microsatellite-stable CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c57088e09b71e7ad2e34a58e69bb450d1b43d18" target='_blank'>
              Heterogeneity and evolution of DNA mutation rates in microsatellite stable colorectal cancer.
              </a>
            </td>
          <td>
            Elena Grassi, V. Vurchio, George D. Cresswell, Irene Catalano, B. Lupo, F. Sassi, Francesco Galimi, Sofia Borgato, Martina Ferri, Marco Viviani, Simone Pompei, Gianvito Urgese, Bingjie Chen, E. Zanella, F. Cottino, Mariangela Russo, G. Mauri, F. Pietrantonio, M. G. Zampino, L. Lazzari, S. Marsoni, A. Bardelli, M. Lagomarsino, A. Sottoriva, L. Trusolino, Andrea Bertotti
          </td>
          <td>2025-05-21</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Ageing is a gradual biological process marked by a decline in physiological function, increasing susceptibility to disease, and mortality. Transposable elements (TEs) are repetitive DNA sequences capable of moving within the genome and thus potentially inducing mutations and disrupting normal cellular functions. Their mobile nature contributes to genomic variation, as transposition events can alter gene expression, chromosome structure, and the epigenetic landscape. To mitigate TE-induced damage, cells rely on epigenetic mechanisms, such as DNA methylation, histone modifications, and small RNAs, to repress TE activity. However, these silencing mechanisms become less effective with age, leading to increased TE activation. This review explores the dual role of TEs as both a cause and consequence of ageing, suggesting a complex relationship between TEs and the ageing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b56d174d42a8f2b0813e61df3277c11215871" target='_blank'>
              Exploring the Relationship of Transposable Elements and Ageing: Causes and Consequences
              </a>
            </td>
          <td>
            Miriam Merenciano, Anaïs Larue, Chloé Garambois, William Vilas Boas Nunes, Cristina Vieira
          </td>
          <td>2025-05-15</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary A comprehensive study analyzed circulating cell-free DNA (cfDNA) from 30 oral squamous cell carcinoma (OSCC) patients, 199 public OSCC, and 192 normal samples to explore its potential in early-stage diagnostics and prognosis. The findings suggest that the cfDNA concentration correlates with the tumor stage, malignancy, and survival prognosis in OSCC patients. Deep genome sequencing of cfDNA revealed several genomic alterations: 1. Copy number variation analysis identified amplifications and deletions at loci 1q, 2q, 3p, 3q, and 8q22. Notably, amplifications of TP53 and PIK3CA, along with other genes related to keratinization, were identified in relapsed OSCC cases. Single nucleotide variations were observed in genes highly mutated in OSCC. 2. A novel fusion gene, TRMO-TRNT1, was detected in seven high-grade tumor samples. The parental genes, TRMO and TRNT1, are involved in tRNA modification and DNA repair, respectively. 3. Integrations of human papillomavirus, simian virus, and enterovirus were found in the OSCC samples, potentially indicating viral involvement in OSCC pathogenesis. These results underscore the utility of cfDNA as a non-invasive biomarker for OSCC, offering insights into tumor genomics and aiding in early detection and prognosis assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c52e96cadb3f868e649f16559709defe911b5c" target='_blank'>
              Applications for Circulating Cell-Free DNA in Oral Squamous Cell Carcinoma: A Non-Invasive Approach for Detecting Structural Variants, Fusions, and Oncoviruses
              </a>
            </td>
          <td>
            M. Bhattacharya, D. Yaniv, D. P. D’Souza, Eyal Yosefof, Sharon Tzelnick, R. Detroja, Tal Wax, A. Levy-Barda, G. Baum, A. Mizrachi, G. Bachar, Milana Frenkel Morgenstern
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00047d76a26a1c71865a20d3d10212996573c803" target='_blank'>
              A recurrent pathogenic BRCA2 truncating variant reveals a role for BRCA2-PCAF complex in modulating NF-κB-driven transcription
              </a>
            </td>
          <td>
            Anna Minello, Jesus Gomez-Escudero, S. C. Sridhara, Charlotte Martin, Elodie Girard, Juan C. Cañas, Yasin Memari, Maria R. Bustos, A. Galarreta, Virginie Boucherit, E. Imyanitov, S. K. Bodvarsdottir, Sylvain Baulande, A. Vincent-Salomon, Nicolas Servant, Matthias Altmeyer, Stefan Sigurdsson, D. Stoppa-Lyonnet, S. Nik-Zainal, A. Carreira
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593951b80dada0fabedb9d7b8680116321df128" target='_blank'>
              Development of Paclitaxel Resistance in Triple-Negative Breast Cancer Is Associated with Extensive DNA Methylation Changes That Are Partially Reversed by Decitabine
              </a>
            </td>
          <td>
            Lenka Trnkova, M. Burikova, A. Soltysova, A. Ficek, Jana Plava, Andrea Cumova, Lucia Rojikova, Kristina Jakic, Eva Sedlackova, Boris Tichý, V. Bystrý, F. Busato, Yimin Shen, M. Matúšková, L. Kucerova, J. Tost, S. Miklikova, M. Cihova, V. Buocikova, B. Smolková
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0631bffe60efc9e321a6ea4a06deb1f550391138" target='_blank'>
              Clonal effects of the Ras oncogene revealed by somatic mutagenesis in a Drosophila cancer model
              </a>
            </td>
          <td>
            Takuya Akiyama, M. Gibson
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61956d7589478355803ed0351b120e6a0e356110" target='_blank'>
              SLFN11 Enhances Cisplatin Sensitivity in Pediatric Cancer via Activation of Stress-Response and Suppression of Survival Pathways
              </a>
            </td>
          <td>
            A. Jabeen, Dina Awartani, S. Sherif, Eiman I. Ahmed, R. Alanany, Ayman Saleh, W. Hendrickx, C. Raynaud
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Retroviral and retrotransposon invasion pose a constant threat to genome integrity and have driven the evolution of host defense pathways able to counter these attacks. The human silencing hub (HUSH complex) is an epigenetic transcriptional repressor complex that recognizes and silences newly integrated retroelements through the establishment of ectopic heterochromatin and chromatin compaction. HUSH provides a genome-wide immunosurveillance system whose challenging task is to detect and silence any newly integrated retroelements, and it thus plays a key role in host defense. To distinguish self from nonself genomic DNA, HUSH recognizes long single-exon (intronless) DNA, the essential hallmark of reverse transcription. Retroelements, being RNA derived, lack classical, noncoding cellular introns, so a long, intronless sequence of DNA is the abnormal molecular pattern that allows HUSH to distinguish invading retroelements from intron-containing host genes. As a newly identified component of the innate immune system, HUSH protects the genome from the reverse flow of genetic information from RNA to DNA, revealing an unanticipated mechanism of postintegration genome immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b1d5426e615e5a445a45b06d1bcad8bd22f0c5" target='_blank'>
              Silencing by the HUSH Epigenetic Transcriptional Repressor Complex.
              </a>
            </td>
          <td>
            Paul J. Lehner
          </td>
          <td>2025-06-01</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy with a 5-year overall survival (OS) rate of approximately 12%. More than 90% of PDAC patients harbor oncogenic mutations in the Kirsten rat sarcoma viral homolog (KRAS) gene. MRTX1133 (MRTX), a novel inhibitor of KRASG12D (the most common KRAS mutation found in pancreatic and colon cancers) has shown promise as a therapeutic agent. To address reported resistance to MRTX, we adapted our anti-leukemia co-targeting strategy and evaluated a combination of MRTX and Bedaquiline (BED), an FDA-approved inhibitor of mitochondrial ATP production, in in vitro human PDAC models. The combination of MRTX and BED demonstrated enhanced cytotoxic effects by disrupting all 11 genes within the DNA helicase family (CMG complex: CDC45-MCM-GINS), which are essential for initiating DNA replication and regulating cell cycle progression. Notably, real-world data analysis from Caris Life Sciences and NCI-TCGA database revealed that low transcriptomic expression of the DNA helicase CMG complex was significantly associated with prolonged survival (e.g., low CDC45 expression and low GINS2 expression with greater than 8 months longer overall survival) in PDAC patients with KRASG12 mutations (N = 9,717; P < 0.00001). However, this combination therapy also triggered strong pro-survival nuclear reprogramming. This effect was mediated by significant genetic activation of an NFκB2-DDIT (DNA damage-induced transcript) axis, which supported tumor chromosomal integrity and DNA repair mechanisms. To overcome NFκB2-driven resistance mechanisms, we explored a triple-targeting strategy that addresses metabolic and genomic plasticity in addition to actively intercepting cell division. This approach combines MRTX1133, Bedaquiline, and the NFκB2 inhibitor SN52, offering a novel therapeutic avenue to treat aggressive pancreatic cancer and potentially improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00669-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecf14d3f8785e3a0d9d03809564ba14faf0a6c7" target='_blank'>
              Targeting DNA helicase CMG complex and NFκB2-driven drug-resistant transcriptional axis to effectively treat KRASG12D-mutated pancreatic cancer
              </a>
            </td>
          <td>
            Jeffrey Xiao, Joshua Kim, Brandon Park, David J. Baylink, Cedric Kwon, Victoria Tran, Scott Lee, Kevin Codorniz, Laren D Tan, Pamela Lobo Moreno, Amy Schill-Depew, S. Mirshahidi, David de Semir, Diana Hanna, Kiran Naqvi, H. Cao, Chien-Shing Chen, Joanne Xiu, H. Lenz, Hamid Mirshahidi, Mark E. Reeves, Yi Xu
          </td>
          <td>2025-05-26</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D. Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromodomain Helicase DNA-binding protein 1-Like (CHD1L) is a chromatin remodeling enzyme increasingly recognized as an oncogenic factor promoting tumor progression and metastatic potential by orchestrating transcriptional programs that drive epithelial-mesenchymal transition (EMT), cytoskeletal remodeling, and metastatic dissemination. In parallel, CHD1L has emerged as a master regulator of tumor cell survival by regulating DNA damage response and repair and enforcing G1 cell cycle progression. Furthermore, CHD1L plays a key role in immune evasion pathways by regulating signaling cascades and by suppressing both apoptotic and non-apoptotic cell death. In particular, CHD1L is a key suppressor of PARthanatos, a caspase-independent mechanism triggered by poly(ADP-ribose) (PAR) polymer fragmentation and apoptosis-inducing factor (AIF) activation. By regulating SPOCK1, MDM2, and TCTP, CHD1L further supports survival under cellular stress. Its overexpression correlates with metastasis, therapy resistance, and poor prognosis across many solid tumors. This review covers CHD1L’s structure, oncogenic functions, and developmental origins, and highlights emerging therapeutic strategies that target CHD1L as a druggable vulnerability in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6296c3dbcbe7b1d1919c8debad6aee4011adbed2" target='_blank'>
              CHD1L in cancer and beyond: structure, oncogenic functions, and therapeutic potential
              </a>
            </td>
          <td>
            Sophia Clune, Paul Awolade, Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-05-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) are transcripts over 200 nucleotides long that do not encode proteins. Although many lncRNAs remain uncharacterized, they are known to play diverse regulatory roles in gene expression. A group of lncRNAs called natural antisense transcripts can form double-stranded structures with their sense partners due to sequence complementarity. These duplexes can become substrates for A-to-I RNA editing, an epitranscriptomic modification mediated by ADAR enzymes. RNA editing is known to influence transcript splicing, affect the resulting gene expression product or alter RNA stability, all of which can impact cancer cell biology. Here, we show a novel natural antisense transcript, UGGT1-AS1, that we have identified and characterized in terms of its cellular localization and sense partner interactions. Furthermore, we demonstrate that UGGT1-AS1 affects cell proliferation and regulates the stability of the UGGT1 sense transcript. Finally, using publicly available RNA sequencing data, we identify A-to-I RNA editing events in the protein-coding gene UGGT1 and further confirm them by RT-PCR and Sanger sequencing in MCF7 cell lines. We hypothesize that UGGT1-AS1 may act as a triggering factor for the A-to-I RNA editing process in its sense partner. Our findings highlight the regulatory role of UGGT1-AS1 and suggest its involvement in RNA editing and cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/576ef5cd2e6364d6e6fb3e63af7cf639406c2d92" target='_blank'>
              Novel lncRNA UGGT1-AS1 Regulates UGGT1 Expression in Breast Cancer Cell Line
              </a>
            </td>
          <td>
            Klaudia Samorowska, Elżbieta Wanowska, M. Szcześniak
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Ewing sarcomas (ESs) are aggressive paediatric tumours of bone and soft tissues afflicting children and adolescents. Despite current therapies having improved the 5-year survival rate to 70% in patients with localized disease, 25% of patients relapse and most have metastasis at diagnosis. Resistance to chemotherapy, together with the high propensity to metastasize, remain the main causes of treatment failure. Thus, identifying novel targets for alternative therapeutic approaches is urgently needed. Methods Biochemical and functional analyses were carried out to elucidate the mechanism of regulation of YAP1 expression by pncRNA_YAP1-1 in ES cells. Results Here, we identified a novel promoter-associated noncoding RNA, pncRNA_YAP1-1, transcribed from the YAP1 promoter in ES cells. We found that pncRNA_YAP1-1 level exerts antitumour effects on ES by destabilizing YAP1 protein. The molecular mechanism relies on the interaction of pncRNA_YAP1-1 with the RNA binding protein FUS, which stabilizes the transcript. Furthermore, pncRNA_YAP1-1 binding to TEAD impairs its interaction with YAP1, thus determining YAP1 translocation into the cytoplasm, its phosphorylation and degradation. Conclusions Overall, our findings reveal a novel layer of regulation of YAP1 protein expression by pncRNA_YAP1-1 in Ewing sarcoma. Considering the role of YAP1 in therapy response and cell propensity to metastasize, our results indicate pncRNA_YAP1-1 as an actionable target that could be exploited to enhance chemotherapy efficacy in Ewing sarcoma. Significance PncRNA_YAP1-1 counteracts the YAP1 oncogenic transcriptional program in Ewing sarcoma cells by interfering with YAP1-TEAD interaction and impairing YAP1 protein stability. These findings uncover a novel treatment option for Ewing sarcoma. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00736-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945aa0f8515a96b425b539864e7b5d32769ff4ff" target='_blank'>
              YAP1 promoter-associated noncoding RNA affects Ewing sarcoma cell tumorigenicity by regulating YAP1 expression
              </a>
            </td>
          <td>
            L. Chellini, Arianna Del Verme, Veronica Riccioni, M. Paronetto
          </td>
          <td>2025-05-25</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The majority of cancer cells experience replication stress, which ultimately causes them to enter mitosis with underreplicated DNA. To alleviate the consequences of replication stress, cells utilize a mechanism known as MiDAS that functions to complete synthesis of underreplicated DNA in early mitosis. This process is considered an Achilles heel for highly replicative cancers. In this study, we show that human TopBP1 localizes to sites of underreplicated DNA marked by FANCD2 and promotes MiDAS through recruitment of the nuclease scaffold protein SLX4. Additionally, we demonstrate that the recruitment of SLX4 to TopBP1 foci in mitosis depends on TopBP1-K704, SLX4-T1260, and several SUMO-interaction motifs in SLX4. Lastly, we show that the recruitment of SLX4 to TopBP1 foci in mitosis is important to prevent transmission of DNA damage to daughter cells. Based on this, we hypothesize that targeting the TopBP1-SLX4 interaction in mitosis may be a potential strategy for anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d03ab5b01bb44008997c89ac819106d74fa906d" target='_blank'>
              TopBP1 coordinates DNA repair synthesis in mitosis via recruitment of the nuclease scaffold SLX4
              </a>
            </td>
          <td>
            Jonas Bagge, Kamilla Vandsø Petersen, Sinem N Karakus, Thorbjørn M Nielsen, Johanne Rask, Christian R Brøgger, Jonas Jensen, Meliti Skouteri, Antony M. Carr, I. Hendriks, V. Oestergaard, Michael Lisby
          </td>
          <td>2025-07-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ead10baece9545ae1e3801dde43ce3d766e518" target='_blank'>
              METTL5 Blockade Enhances Anti-Tumor Immune Response via Inducing Neoantigen Generation
              </a>
            </td>
          <td>
            Yangyi Zhang, Xiaoyan Shi, Yuci Wang, Ruiqi Wang, Yanlan Cao, Hao Chen
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Androgen receptor (AR), a member of the nuclear receptor superfamily controls prostate epithelial cell plasticity by repressing a panel of genes involved in epithelial-mesenchymal transition (EMT), including the human CDH2 gene encoding N-cadherin. At the opposite, pathological AR variants such as AR-V7 associated with prostate tumor progression upregulate those EMT genes. Here, focusing on the human CDH2 gene, we show that this duality between AR and AR-V7 relies on a potential human accelerated region present in the intron 1. This fastest-evolving region of the human genome is actually a variable number tandem repeat (VNTR) comprising 24 repetitions of a DNA sequence that englobes binding sites for steroid hormone receptors, recombination signal binding protein for immunoglobulin kappa j region (RBPJ) an effector of the Notch pathway, and zinc finger e-box binding homeobox 1 (ZEB1). Genomic DNA sequencing, multiple sequence alignment, data mining, as well as protein-DNA interaction and gene expression analyses indicate that this VNTR constitutes a potential transcriptional hub for different transcription factors to control human CDH2 expression. Also, our data suggest that prostate tumor cells may unlock an up to now unknown molecular mechanism associated with a fine-tuned control of human CDH2 gene expression. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05725-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b161c5324c0a97c9c54e04e62611e3f6b1ceb501" target='_blank'>
              Human-specific genomic evolution of a regulatory network enables fine-tuning of N-cadherin gene expression
              </a>
            </td>
          <td>
            É. Erdmann, Savera Agolli, Simon Fix, F. Cottard, Christine Keyser, V. Zvénigorosky, Angéla Gonzalez, Zakary Haili, Bruno Kieffer, J. Céraline
          </td>
          <td>2025-05-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) contributes to 3% of all childhood cancers with roughly 400-500 cases diagnosed each year in the United States. The World Health Organization classifies rhabdomyosarcoma into four histological subtypes which include alveolar, embryonal, spindle-cell and pleomorphic. The primary genetic drivers in a subset of alveolar and spindle-cell histological subtypes are gene fusions. This review explores the fusion oncogenes identified in RMS such as PAX-and NCOA2-based fusions, along with discussing studies defining fusion oncogene biology and tumorigenic mechanisms. Focus areas include data around transformation events and progression along with dysregulated biological processes. Furthermore, we summarize model systems, ranging from cell to animal models, that have been implemented to study fusion oncogenes identified in RMS. With the constant identification of novel fusion oncogenes, this review also emphasizes the need for genetically characterizing RMS tumors and rapidly developing new model systems. These models are critical to study fusion oncogene activity and to delineate key regulatory players and potential therapeutic targets that suppress tumorigenesis. The identification of RMS fusion oncogenes and integration with animal and cell culture models will help identify conserved molecular targets, optimize therapeutic approaches, and ultimately improve clinical outcomes for children with RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23d84102b5431b0bc6bd415779fd94bf54425c" target='_blank'>
              Fusion oncogenes in rhabdomyosarcoma: model systems, mechanisms of tumorigenesis, and therapeutic implications
              </a>
            </td>
          <td>
            Chinmay S. Sankhe, Lisa Hall, Genevieve C. Kendall
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abnormal nuclear morphology is a hallmark of human diseases, including cancers and age-related disorders. Previously, maintenance of nuclear morphology and integrity was thought to be solely dependent on a force balance between nuclear mechanical resistance and actin antagonism. However, our recent work revealed that inhibiting RNA polymerase II suppresses nuclear blebbing independent of altering force balance, but the mechanism remains unknown. Through removing cell culture media serum and then adding it back, we can decrease and then restore transcriptional activity. Decreasing transcriptional activity decreases nuclear bleb formation, stability, and rupture while returning transcriptional activity restores nuclear blebbing. These modulations of transcriptional activity did not alter nuclear or actin mechanics. The mean square displacement (MSD) of chromatin domains labeled via transfected Cy3-dNTPs revealed that transcription activity regulates chromatin motion. To determine if increasing chromatin motion is a mechanism to increase nuclear blebbing, we used an established RAD51 inhibitor BO2. We verified BO2 increases chromatin domain motion which resulted in increased nuclear blebbing. We reveal the mechanism by which transcriptional activity drives nuclear blebbing is through chromatin motion. Thus, two hallmarks of human disease are directly linked via transcriptional activity and abnormal nuclear shape. Statement of Significance Nuclear blebs are hallmarks of disease progression that cause dysfunction, but how they are formed remains unanswered. We find that chromatin motion generated by transcriptional activity is essential for both nuclear bleb formation and stability. This was independent of changes in nuclear stiffness or actin antagonism. This finding provides a key advancement in our understanding of nuclear bleb formation. Furthermore, it reveals transcriptional activity as a novel contributor to nuclear blebbing in addition to the paradigm of nuclear shape determined as a force balance between nuclear resistance and actin antagonism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a26f910c21dd15469983a962e8b18f8de0bd93e" target='_blank'>
              Transcriptional activity generates chromatin motion that drives nuclear blebbing
              </a>
            </td>
          <td>
            Kelsey Prince, Katie Lin, Andy Li, Nick Borowski, A. Stephens
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The relationship between TP53 and transposable elements (TEs) has been obscure. Given the important role of TEs in oncogenesis, a comprehensive profiling of TE expression dynamics under the regulation of TP53 provides valuable resources for more clarity in TP53's roles in cancer. In this study, we characterized the TE transcriptomic landscape using long-read RNA-seq and short-read RNA-seq in three cancer cell lines varying only in TP53 genetic status. To identify transcripts that use TEs as potential promoters, we developed a computational pipeline, TEProf3, and identified in total 1942 transcripts with high confidence. Among these TE-derived transcripts, 239 are activated by TP53 and 221 are repressed by TP53. These TP53-responsive TE-derived transcripts are mainly driven by members of the ERV and LINE families. Following knockdown of wild-type (WT) TP53 expression, rescuing WT TP53 expression allows for partial recovery of the TE expression profile observed in the context of chronic TP53 expression. TP53 mutations R175H and R273H manifest their oncogenic characteristic partially through activating TE promoters in a cell type–specific manner. Lastly, we identified important sequence motifs that help govern the interactions between TEs and TP53, where TP53 activates TEs with TP53 binding motifs through direct binding and represses TEs indirectly via other pathways. Overall, we present a comprehensive profiling of the impact of TP53 on the activity of TE-derived promoters in isogenic cancer cell lines and provide a high-confidence TE expression atlas of TE promoters that are direct and indirect targets of TP53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53c846a8196b5df2291595170d1ce035cd5084b" target='_blank'>
              Charting the regulatory landscape of TP53 on transposable elements in cancer
              </a>
            </td>
          <td>
            Xuan Qu, Yonghao Liang, Colin McCornack, Xiaoyun Xing, Heather Schmidt, Chad Tomlinson, C. Fronick, E. Belter, Juan F. Macias-Velasco, Ting Wang
          </td>
          <td>2025-05-13</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The tumor suppressor gene NKX3.1 and the LPL gene are located in close proximity on chromosome 8, and their deletion has been reported in multiple studies. However, the significance of LPL loss may be misinterpreted due to its co-deletion with NKX3.1, a well-established event in prostate carcinogenesis. This study investigates whether LPL deletion represents a biologically relevant event or occurs merely as a bystander to NKX3.1 loss. We analyzed 28 formalin-fixed paraffin-embedded prostate cancer samples with confirmed LPL deletion and 28 without. Immunohistochemical staining was performed, and previously published whole-genome sequencing data from 103 prostate cancer patients were reanalyzed. Deletion of the 8p21.3 region was associated with higher Gleason grade groups. While NKX3.1 expression was significantly reduced in prostate cancer compared to benign prostatic hyperplasia, LPL protein expression showed no significant difference between cancerous and benign tissue, nor was it affected by the 8p21.3 deletion status. Copy number analysis confirmed the co-deletion of NKX3.1 and LPL in 54 patients. Notably, NKX3.1 loss without accompanying LPL deletion was observed in eight additional cases. These findings suggest that LPL deletion is a passenger event secondary to NKX3.1 loss and underscore the importance of cautious interpretation of cytogenetic findings involving the LPL locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74796121b67d50bf9eb77e02037db96e6cc25807" target='_blank'>
              Deletions of LPL and NKX3.1 in Prostate Cancer Progression: Game Changers or By-Standers in Tumor Evolution
              </a>
            </td>
          <td>
            Tereza Vodičková, Mária Wozniaková, Vladimír Židlík, Jana Žmolíková, Jana Dvořáčková, Adéla Kondé, Jana Schwarzerová, Michal Grepl, J. Bouchal
          </td>
          <td>2025-05-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is an aggressive hematopoietic malignancy, necessitating the exploration of innovative therapeutic approaches. Targeting epigenetic mechanisms has emerged as a promising avenue for cancer treatment. EP300 belongs to the KAT3 family of histone/non-histone lysine acetyltransferases, regulating gene expression by acetylating H3K27. However, the role of EP300 and its potential as a targeted therapy in DLBCL remains unknown. Methods Public datasets were collected to evaluate the expression and clinical significance of epigenetic modification-related genes in patients with DLBCL. Flow cytometry, colony formation, and western blotting were conducted to investigate the function of EP300. CCK8, proliferation, cell cycle, and apoptosis assays, as well as experiments in tumor-bearing mouse models were conducted to determine the therapeutic effect of the EP300 inhibitor A485 alone or in combination with the XPO1 inhibitor KPT8602. RNA-seq was used to investigate the molecular mechanisms underlying the inhibition of DLBCL development by A485. Results EP300 is frequently overexpressed in DLBCL and is associated with poor prognosis, highlighting its potential role in lymphoma progression. In this study, we found that A485, a novel small-molecule inhibitor targeting the conserved histone acetyltransferase (HAT) domain of EP300, significantly reduced H3K27Ac levels and demonstrated potent antitumor effects in DLBCL cells, both in vitro and in vivo. Furthermore, we showed that A485 attenuated DLBCL progression by inhibiting the MYC and E2F1 pathways. Notably, the combination of A485 with the XPO1 inhibitor KPT8602 produced synergistic anti-lymphoma in vitro and in vivo effects in DLBCL cell lines. This combination therapy resulted in enhanced tumor suppression in a DLBCL xenograft model with minimal toxicity. These findings suggested that targeting EP300, particularly in conjunction with XPO1 inhibition, could represent a promising therapeutic strategy for DLBCL treatment. Conclusions Our study elucidated that EP300 inhibition, especially in combination with XPO1 blockade, could serve as a promising therapeutic strategy for the treatment of DLBCL. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14257-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920e8f7451cedecae320b60bb672cda4294ef282" target='_blank'>
              Targeting EP300 in diffuse large b-cell lymphoma: efficacy of A485 and synergistic effects with XPO1 inhibition
              </a>
            </td>
          <td>
            Yanan Jiang, Donghui Xing, Xiang He, Wenqi Wu, Hong Xu, H. Sun, Yixin Zhai, Kaiping Luo, Zhigang Zhao
          </td>
          <td>2025-05-28</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a013cf13f270b70a82c0a8355212488d6a1c6ef" target='_blank'>
              Microhomology-Mediated End Joining as a Novel Mechanism Underlying Androgen Insensitivity Syndrome
              </a>
            </td>
          <td>
            J. M. Marques, R. M. Ramos, N. D. R. D' Alessandre, G. D. A. Guardia, A. F. Afonso, B. L. Braga, M. F. D. A. Funari, M. Y. Nishi, P. Asprino, S. Domenice, P. A. F. Galante, B. B. Mendonca, R. L. Batista
          </td>
          <td>2025-06-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a deadly disease known for its genetic heterogeneity. LTR12C is an endogenous retrovirus-derived regulator of pro-apoptotic genes and is normally silenced by epigenetic regulation. In this study, we found that the treatment of two glioblastoma cell lines, T98-G and U87-MG, with DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibitors activated LTR12C expression. Combined treatment with these epigenetic drugs exerted a synergistic action on the LTR12C activation in both cell lines, while treatment with each drug as a single agent had a far weaker effect. A strong induction of the expression of the TP63 gene was seen in both cell lines, with the pro-apoptotic isoform GTA-p63 accounting for most of this increase. Coherently, downstream targets of p63, such as p21 and PUMA, were also induced by the combined treatment. Furthermore, we observed a significant reduction in the GBM cell growth and viability following the dual DNMT/HDAC inhibition. These findings reveal that the reactivation of LTR12C expression has the potential to modulate survival pathways in glioblastoma and provide information regarding possible epigenetic mechanisms that can be used to treat this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b329500c4414f15e1eee2caec6a0bfaed4d1e7f7" target='_blank'>
              Functional Characterization of LTR12C as Regulators of Germ-Cell-Associated TA-p63 in U87-MG and T98-G In Vitro Models
              </a>
            </td>
          <td>
            Lucia Meola, Sohum Rajesh Shetty, A. Peschiaroli, Claudio Sette, Camilla Bernardini
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background
 DNA 5-methylcytosine (5mC) and RNA N6-methyladenosine (m6A) methylation are prevalent modifications in eukaryotes, both playing crucial roles in gene regulation. Recent studies have explored their crosstalk and impact on transcription. However, the intricate relationships among 5mC, m6A, and gene expression remain incompletely elucidated. Results We collect data on 5mC, m6A, and gene expression from samples from three tissues from each of four pregnant cattle and sheep. We construct a comprehensive genome-wide self-interaction (same gene) and across-interaction (across genes) network of 5mC and m6A within gene-bodies or promoters and gene expression in both species. Qualitative analysis identifies uniquely expressed genes with specific m6A methylation in each tissue from both species. A quantitative comparison of gene expression ratio between methylated and unmethylated genes for m6A within gene body and promoter, and 5mC within gene body and promoter confirms the positive effect of RNA methylation on gene expression. Importantly, the influence of RNA methylation on gene expression is stronger than that of DNA methylation. The predominant self- and across-interactions are between RNA methylation within gene bodies and gene expression, as well as between RNA methylation within promoters and gene expression in both species. Conclusions RNA methylation has a stronger effect on gene expression than does DNA methylation within gene bodies and promoters. DNA and RNA methylation in gene-bodies has a greater impact on gene expression than those in promoters. These findings deepen comprehension of the dynamics and complex relationships among the epigenome, epitranscriptome, and transcriptome, offering fresh insights for advancing epigenetics research. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03617-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79390cad38f810c7589608ce6b54c5711ffacee" target='_blank'>
              Analyzing the relationship of RNA and DNA methylation with gene expression
              </a>
            </td>
          <td>
            Shangqian Xie, Darren Hagen, G. Becker, Kimberly M. Davenport, Katie A Shira, Morgan R Stegemiller, Jacob W. Thorne, Sarem F. Khilji, D. Konetchy, Patricia Villamediana, Brenda M Murdoch, S. McKay
          </td>
          <td>2025-05-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f4e542b3c6d086c581d428766c82b0185e38fd" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability status at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Pediatric brain tumors, the most devastating cancers affecting children, are believed to originate from neural stem/progenitor cells in developing brain. In precise timing and specific regions during the brain development, chromatin deregulation plays crucial roles in redirecting normal neuronal differentiation pathways toward tumorigenesis. Indeed, epigenomic abnormalities are thought to be more important for brain tumor formation especially in children than adults, as pediatric brain tumors generally exhibit fewer genetic mutations compared to adult brain tumors. Given the small number of mutations, targeting such limited alterations involved in cancer epigenomes is expected to be more effective in pediatric brain tumors. The mechanisms of cancer epigenomes include mutation or dysregulation of chromatin remodelers, histone modifiers, histones themselves, and DNA methylation enzymes. Furthermore, genomic rearrangements and/or higher-order chromatin topology also contribute to these epigenomic mechanisms. These mechanisms are commonly observed in various types of pediatric brain tumors. However, alterations in chromatin regulatory factors differ across tumor types, reflecting the unique epigenetic landscapes shaped by their tumor origins. Accordingly, clarifying their functional similarities and differences across tumor types could offer valuable insights for finding new therapeutic strategies. Thus, this review article focuses on elucidating how pediatric brain tumors arise from epigenomic deregulation and what epigenetic molecules or mechanisms could serve as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a334571a5c8d012851148672d620a490ae61353" target='_blank'>
              Epigenetic modifications and their roles in pediatric brain tumor formation: emerging insights from chromatin dysregulation
              </a>
            </td>
          <td>
            Kento Kawata, Owen S. Chapman, Satoshi Narumi, Daisuke Kawauchi
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="VIRMA (also known as KIAA1429), as a core regulatory subunit of the m6A methyltransferase complex, plays a key role in tumorigenesis and progression by dynamically regulating RNA methylation modifications. Studies have shown that VIRMA is aberrantly overexpressed in more than 20 types of malignant tumors, including liver cancer, breast cancer, and lung cancer, and is significantly associated with chromosome 8q amplification and poor prognosis. Its mechanism of action involves regulating the expression of tumor-associated genes through both m6A-dependent and m6A-independent pathways, thereby promoting tumor proliferation, metastasis, and drug resistance. These findings suggest that VIRMA has the potential to serve as a pan-cancer diagnostic and prognostic biomarker. This review summarizes the role of VIRMA in malignant tumors from multiple perspectives and explores its potential applications in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1f0fb758f358894db60bb1aa6efc139d365cc4" target='_blank'>
              The Multifaceted Role of VIRMA, a Core Component of the Methyltransferase Complex, in Cancer and Cancer Therapy
              </a>
            </td>
          <td>
            Jinmeng Lu, Chengyu Zhang, Mengshuang Yin, Huili You, Chao Xiong, Jing Wu, Ying Gong, Zhangang Xiao, Jing Shen
          </td>
          <td>2025-06-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d46f5896df3074e9079fd05c2ab487e0d0fe604" target='_blank'>
              Context-Dependent Regulation of Microhomology-Mediated End Joining in Normal Tissues: Insights into Tissue-Specific Activation of DNA Repair Pathways
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1683b65b8227a142ec0662e619dfd830c76c770" target='_blank'>
              Major effect mutations drive DNA methylation variation after colonization of a novel habitat
              </a>
            </td>
          <td>
            Johan Zicola, Emmanuel Tergemina, Mehmet Göktay, Célia Neto, Robert J. Schmitz, Angela M. Hancock
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) are a leading cause of death in patients with advanced disease. MBM treatment relay on targeted and immunotherapy and on stereotactic radiosurgery as gold standard. Life expectancy has improved significantly with these therapies however, targeted therapy is short lived and only about half of the patients respond to immunotherapy, while radiation is limited by melanoma cells intrinsic resistance to DNA damage. New therapeutic approaches are therefore needed to treat MBM. Here we investigate a new role of Notch1 in genomic instability and demonstrate that blockade of both Notch1 and the DNA repair factor ChK1 causes extensive DNA damage and tumor cell death increasing survival in MBM bearing mice. Methods Anti-Notch1 (anti-N1) was previously described. Prexaserib, a ChK1 inhibitor, is currently in clinical trials. K457 and A375 melanoma cells were used. RNA sequencing was performed in K457 cells treated with anti-N1 and Gene Set Enrichment Analysis performed. DNA damage was evaluated by a DNA fiber assay to assess replication fork speed; and γH2AX foci count and neutral comet assay to quantify double strand breaks. Cell survival was evaluated by trypan blue and a colony formation assay. Luciferase expressing A375 cells were orthotopically inoculated in the right cerebral cortex of athymic nude mice, for in vivo evaluation of a therapy with anti-N1 and prexasertib. Survival was assessed by Kaplan-Meyer survival curves and significance assessed by a Log-rank test. Results Notch1 blockade caused genomic instability by reducing histone availability, leading to DNA replication stress and DNA damage. This in turn, resulted in the activation of the DNA Damage Response pathway ATR/ChK1 to counter the damage. Co-inhibition of Notch1, via anti-N1, and ChK1, via prexasertib (prex), exacerbated DNA damage increasing melanoma cell death. Importantly, combination anti-N1/prex significantly improved survival of mice bearing MBMs. Conclusions A therapy with anti-N1/prexasertib could represent a novel treatment strategy, alone or in combination with current treatment regimens, for melanoma brain metastases. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03411-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a360dee32c6a48d005c164a15d89bdb2679317f" target='_blank'>
              Co-inhibition of Notch1 and ChK1 triggers genomic instability and melanoma cell death increasing the lifespan of mice bearing melanoma brain metastasis
              </a>
            </td>
          <td>
            Varsha Thakur, Vijay S Thakur, Dazhi Wang, Juliano Tiburcio de Freitas, Anna Bianchi, Luis Nivelo, Oliver Umland, S. Welford, Barbara Bedogni
          </td>
          <td>2025-05-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci.To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows).Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription.Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Enhancers are non-coding DNA regions that facilitate gene transcription, with a specialized subset, super-enhancers, known to exert exceptionally strong transcriptional activation effects. Super-enhancers have been implicated in oncogenesis, and their identification is achievable through histone mark chromatin immunoprecipitation followed by sequencing data using existing analytical tools. However, conventional super-enhancer detection methodologies often do not accurately reflect actual gene expression levels, and the large volume of identified super-enhancers complicates comprehensive analysis. To address these limitations, we developed the super-enhancer to gene links (SE-to-gene Links) analysis, a platform named “SEgene” which incorporates the peak-to-gene links approach—a statistical method designed to reveal correlations between genes and peak regions (https://github.com/hamamoto-lab/SEgene). This platform enables a targeted evaluation of super-enhancer regions in relation to gene expression, facilitating the identification of super-enhancers that are functionally linked to transcriptional activity. Here, we demonstrate the application of SE-to-gene Links analysis to public datasets, confirming its efficacy in accurately detecting super-enhancers and identifying functionally associated genes. Additionally, SE-to-gene Links analysis identified ERBB2 as a significant gene of interest in the lung adenocarcinoma dataset from the National Cancer Center Japan cohort, suggesting a potential impact across multiple patient samples. Thus, the SE-to-gene Links analysis provides an analytical tool for evaluating super-enhancers as potential therapeutic targets, supporting the identification of clinically significant super-enhancer regions and their functionally associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f21eaa7e9146967a377d180baeda8f0749c3fefc" target='_blank'>
              SEgene identifies links between super enhancers and gene expression across cell types
              </a>
            </td>
          <td>
            Norio Shinkai, Ken Asada, Hidenori Machino, Ken Takasawa, Satoshi Takahashi, Nobuji Kouno, M. Komatsu, R. Hamamoto, S. Kaneko
          </td>
          <td>2025-05-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Oncogene activation through DNA amplification or overexpression plays a critical role in cancer initiation and progression. Chromosome 17 harbors numerous cancer-associated genetic anomalies, some of which are strongly linked to poor prognosis in breast cancer patients. The FOXK2 gene, located on 17q25, encodes a transcription factor with a forkhead DNA binding domain. Integrative analyses of publicly available genomic datasets reveal that FOXK2 is frequently amplified and overexpressed in breast cancer, correlating with poor overall survival. FOXK2 knockdown via shRNA significantly inhibited the proliferation, migration, and anchorage-independent growth of breast cancer cells, causing G0/G1 cell cycle arrest. The C-terminal nuclear localization signal (NLS) is crucial for FOXK2's oncological function. Additionally, FOXK2 knockdown in MCF7 breast cancer cells delayed tumor growth in a xenograft mouse model. Inhibiting FOXK2 expression also sensitized breast cancer cells to frontline chemotherapy agents, including doxorubicin, 5-fluorouracil, and etoposide in vitro. Co-overexpression of FOXK2 and PI3KCA with oncogenic mutations (E545K or H1047R) induced the transformation of non-tumorigenic MCF-10A cells, suggesting that FOXK2 acts as an oncogene in breast cancer and contributes to PI3KCA-driven tumorigenesis. CCNE2, PDK1, and ESR1 were identified as direct transcriptional targets of FOXK2 in MCF-7 cells. Moreover, co-inhibition of FOXK2 function, either through knockdown or inhibition of its transcriptional targets (CCNE2/CDK2 by Dinaciclib or PDK1 by DCA) and PI3KCA by alpelisib, had synergistic anti-tumor effects in breast cancer cells with FOXK2 overexpression and PI3KCA oncogenic mutations. FOXK2 functional inhibition by the CDK2 inhibitor Dinaciclib enhanced the tumor-suppressive effect of alpelisib in a MCF7 breast cancer xenograft mouse model. These findings provide compelling evidence that FOXK2 plays an oncogenic role in breast tumorigenesis. The combination of PIK3CA and CCNE2/CDK2 inhibition by alpelisib and Dinaciclib offers a potential therapeutic strategy for breast cancer with FOXK2 overexpression and PIK3CA oncogenic mutation.
 Citation Format: Hong Zhang, Yang Yu, Weng-Ming Cao, Feng Cheng, Edaise da silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang. Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy? [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-24.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caaa74a368ec1f757b26670f47abf7954355f919" target='_blank'>
              Abstract P5-02-24: Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy?
              </a>
            </td>
          <td>
            Hong Zhang, Yang Yu, Wen-Ming Cao, Feng Cheng, E. D. da Silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61df03d16c11da50362eb27baf0281aee3eb21d5" target='_blank'>
              A novel role for UPF1 in safeguarding DNA replication in S phase and beyond
              </a>
            </td>
          <td>
            Thomas Walne, Laura Maple, Nan Li, Isabel Christie, Carl Smythe, Ruth Thompson
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved by specialized helicases. Previous in vitro studies showed that DEAH-box helicase 36 (DHX36/G4R1/RHAU) has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in the accumulation of cytoplasmic DNA fragments, an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression, and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 messenger RNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd74a217dd008b29b89824a440aee3aba255159" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, Shangming Tang, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Yuh-Hwa Wang
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758af464be2f5fe707681ad668cca7d960076b75" target='_blank'>
              Targetable BIRC5 dependency in therapy-resistant TP53 mutated acute myeloid leukemia
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, Fang Qi Lim, Yang Mi, E. Olesinski, Cheryl Gek Teng Chan, S. Jasdanwala, Xiao Xian Lin, Yichen Wang, Jordan Yong Ming Tan, K. Bhatia, Valeriia Sapozhnikova, Chuqi Wang, Aarthi Nivasini Mahesh, Daniel Tan En Liang, Nishtha Chitkara, P. Mertins, L. Hogdal, Brandon D Brown, T. Haferlach, Camille Lobry, Coleman Lindsley, Alexandre Puissant, Han Kiat Ho, Sanjukta Das, A. Letai, S. Kornblau, J. Krönke, Edward Ayoub, Koji Itahana, Michael Andreeff, Shruti Bhatt
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The nucleolus is a large membraneless subnuclear structure, the main function of which is ribosome biogenesis. However, there is growing evidence that the function of the nucleolus extends beyond this process. While the nucleolus is the most transcriptionally active site in the nucleus, it is also the compartment for the location and regulation of repressive genomic domains and, like the nuclear lamina, is the hub for the organization of inactive heterochromatin. Studies in human and Drosophila cells have shown that a decrease in some nucleolar proteins leads to changes in nucleolar morphology, heterochromatin organization and declustering of centromeres. This work is devoted to the study of the effects of Novel nucleolar protein 3 (Non3) gene mutations in D. melanogaster on the organization of chromatin in the nucleus. Previously, it was shown that partial deletion of the Non3 gene leads to embryonic lethality, and a decrease in NON3 causes an extension of ontogenesis and formation of a Minute-like phenotype in adult flies. In the present work, we have shown that mutations in the Non3 gene suppress the position effect variegation (PEV) and increase the frequency of meiotic recombination. We have analyzed the classical heterochromatin markers in Non3 mutants and shown that the amount of the HP1 protein as well as the modification of the histone H3K9me2 do not change significantly in larval brains and salivary glands compared to the control in Western blot analysis. Immunostaining with antibodies to HP1 and H3K9me2 did not reveal a significant reduction or change in the localization patterns of these proteins in the pericentromeric regions of salivary gland polytene chromosomes either. We analyzed the localization of the HP1 protein in Non3 mutants using DNA adenine methyltransferase identification (DamID) analysis and did not find substantial differences in protein distribution compared to the control. In hemocytes of Non3 mutants, we observed changes in the morphology of the nucleolus and in the size of the region detected by anti-centromere antibodies, but this was not accompanied by declustering of centromeres and their untethering from the nucleolar periphery. Thus, the NON3 protein is important for the formation/function of the nucleolus and is required for the correct chromatin packaging, but the exact mechanism of NON3 involvement in these processes requires further investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d67fd0595392b8b07d4241837bf9f0f2261d7f" target='_blank'>
              The effects of Non3 mutations on chromatin organization in Drosophila melanogaster
              </a>
            </td>
          <td>
            A. A. Yushkova, A. Ogienko, E. N. Andreyeva, A. Pindyurin, A. E. Letiagina, E. S. Omelina
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cf680837c223860a295c7d973297324c3a3da0" target='_blank'>
              From predictive biomarker to therapeutic target: the dual role of SLFN11 in chemotherapy sensitivity.
              </a>
            </td>
          <td>
            Yue Feng, Yingze Li, Zhenhao Zhang, Yuxuan Dai, Xingchun Gou, Kejing Lao, Running Zhang
          </td>
          <td>2025-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861ff3d8af6cc40f991267fd62388116259dc4df" target='_blank'>
              EWSR1::ETS-low cells promote metabolic reprogramming of the tryptophan-kynurenine-AHR axis, immunosuppression, and poor outcome in Ewing sarcoma
              </a>
            </td>
          <td>
            Martha J. Carreño-Gonzalez, Kimberley M. Hanssen, Ahmed Sadik, Alessa Henneberg, Anna C. Ehlers, A. K. Ceranski, Zuzanna Kolodynska, Malenka Zimmermann, R. Imle, Florian H. Geyer, Alina Ritter, A. Banito, M. Knott, S. Ohmura, F. Cidre-Aranaz, Christiane Opitz, T. G. Grünewald
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Aberrant function or overactivation of exonuclease 1 (EXO1) may be associated with cancer tumor development, drug resistance, and response to immunotherapy in female-related cancers. Methods By analyzing RNA-sequencing data from The Cancer Genome Atlas database, combined with validation through quantitative polymerase chain reaction experiments, we explored the expression levels of EXO1 in breast cancer (BRCA) cell lines and assessed its multidimensional roles in various female-related cancers. Results Our experiments revealed elevated expression of EXO1 in BRCA cell lines, consistent with the RNA-sequencing data. The high expression of EXO1 is associated with poor prognosis in various female-related cancers, especially in BRCA and UCEC. It significantly correlates with clinical and pathological characteristics. In specific cancer subtypes like the basal-like subtype of BRCA, high EXO1 expression is associated with a better prognosis. Genetic mutation analysis indicates a higher frequency of EXO1 gene mutations in uterine sarcoma and BRCA. DNA methylation levels may play a role in the regulation of EXO1 gene expression in some cancers. EXO1 expression is correlated with various factors within the tumor immune microenvironment and may be associated with the sensitivity to anticancer drugs. Conclusion EXO1 exhibits multidimensional roles in female-related cancers as a prognostic biomarker and potentially influences tumor immune therapy responses and drug sensitivities. Further studies are needed to fully understand the complex mechanisms underlying these associations and to explore potential therapeutic strategies targeting EXO1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00f61f934389fb7d273be0eaa350ec5d0b17ae1" target='_blank'>
              Deciphering the multifaceted role of EXO1 in female-related cancers: implications for prognosis and therapeutic responsiveness
              </a>
            </td>
          <td>
            Cong Yu, Guoying Wu
          </td>
          <td>2025-05-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this study, we established dozens of single provirus-infected cellular clones offering various transcriptional phenotypes of HIV. We proposed that stochastic fluctuations in HIV transcription can appear at, at least, two levels: (1) the chromosomal landscape and (2) the in situ HIV integration site. In the former case, proviruses integrating at different genomic locations demonstrated a variety of transcriptional bursting and can be classified in noise space constructed based on the parameters associated with the coefficient of variation and using a mathematical model fitting a curve of exponential decay. In the latter case, stochastic HIV gene expression tended to be a pure epigenetic phenomenon: the identical provirus demonstrated fluctuations in its transcription with an elevated frequency. We observed similar expression patterns between sense and antisense RNA transcripts. Notably, both HIV long terminal repeats reacted to drug stimulation and may reveal distinct behaviors. Overall, our data suggest that HIV antisense transcripts could be involved in the stochastic nature of HIV transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2887cf5883e567b88fc75ea922207e1c7b94ecf7" target='_blank'>
              The influence of HIV sense and antisense transcripts on stochastic HIV gene expression and reactivation
              </a>
            </td>
          <td>
            Kamil Więcek, Janusz Wiśniewski, Heng-Chang Chen
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 The combination of CDK4/6 inhibitors (CDK4/6i) with endocrine therapy prolongs survival of patients with ER+/HER2- metastatic breast cancer. Nearly all patients with advanced disease treated with this combination eventually progress, highlighting a need for therapeutic interventions that prevent the emergence of drug resistance. Drug tolerant persister (DTP) cells represent a reservoir of surviving cells from which drug-resistant clones eventually emerge. We hypothesize that targeting the DTP state which emerges under selective pressure from CDK4/6i could prevent the emergence of drug resistance. In vitro treatment of MCF7 and T47D ER+/HER2- breast cancer cells with the CDK4/6i Palbociclib and estrogen deprivation for 4 weeks suppressed cell viability with 2-5% of cells surviving treatment and achieving a DTP state. Cell cycle analysis by PI staining revealed that 95% of the DTP cells were arrested in G0/G1 after 4 weeks of continuous therapy. Upon drug washout, the DTP population resumed proliferation within ∼1 week and regained sensitivity to retreatment with CDK4/6i, suggesting epigenetic reprogramming is responsible for the DTP state. Whole exome sequencing did not show enrichment of known genomic alterations associated with CDK4/6i resistance (e.g., RB1, FAT1, PTEN). Bulk RNA sequencing of MCF7 DTP cells revealed transcriptome reprogramming compared to untreated controls, which reverted upon drug washout to that of the treatment naïve state. While cell cycle-related gene signatures such as E2F targets and G2M checkpoint were downregulated, interferon response signatures were upregulated in MCF7 DTP cells relative to untreated controls. MCF7 persisters were characterized by enrichment in senescence and chemotherapy-induced stressed gene signatures, while diapause and MYC target signatures were downregulated. These same signatures were similarly enriched in ER+/HER2- tumors from patients in the NeoPalAna study treated with neoadjuvant palbociclib and the aromatase inhibitor anastrozole compared to baseline biopsies. To study evolution of drug resistance, we transduced MCF7 and T47D cells with the LARRY (Lineage and RNA recovery) barcode library to uniquely label and track each persister cells long-term, and identify programs that are causal to the emergence of drug resistant clones. We performed single cell RNA-seq on barcoded cells treated with palbociclib plus estrogen deprivation for 0 h, 72 h, and 4 weeks. Several clones that arrested in G0/G1 initially eventually expanded at 4 weeks, while other clones were eliminated. We are currently investigating which genes/pathways are upregulated in the barcoded clones enriched at the 4-week time point. We will then validate results from these single cell approach in cells surviving neoadjuvant treatment in the NeoPalAna trial. In summary, a small subset of ER+/HER2- breast cancer cells survive long-term treatment with CDK4/6i plus estrogen deprivation and exhibits reversible transcriptomic features of DTPs. Our ongoing single cell-seq analyses may reveal transcriptional programs that enable the DTP state which, in turn, could be targeted to reduce survival of drug-tolerant cells and prevent the emergence of drug resistance.
 Citation Format: Yuki Matsunaga, Hima Milan Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, Dan Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, Ariella B. Hanker. Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-30.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a0b4864d21ce42a1a87671ea1a257909c574fe" target='_blank'>
              Abstract P3-04-30: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, H. M. Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromosomal instability in Alzheimer’s disease (AD) neurons has been previously reported. This pilot study aimed to establish a quantitative technique for assessing X chromosome centromere signals using fluorescence in situ hybridization (FISH). Hippocampal brain tissue was collected at autopsy from sporadic AD patients and age- and gender-matched controls. FISH was utilized to detect and measure the intensity of hybridization signals for X chromosome centromeres in the interphase nuclei of hippocampal brain cells. The premature centromere division (PCD) phenomenon, marked by a close bipartite signal appearing as two separated FISH spots, was examined to see if the hybridized DNA amount in each spot matched the expected centromere DNA amount. The technique effectively distinguished between PCD+ and PCD− signals. The average PCD frequency of the X chromosome in the AD group was 7 ± 1%, compared with 3.2 ± 0.84% in the controls. This quantitative approach supports qualitative analyses of FISH centromere spots, reinforcing findings of chromosomal instability in AD. The presence of a double signal at the centromere of a single X chromosome indicates re-entered cell cycles, DNA replication, and PCD in hippocampal neurons. This technique provides a reliable method for identifying PCD + signals and contributes to understanding chromosomal instability in AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682335163a6ea498c4fd17acd3e21f0876fd8d01" target='_blank'>
              Assessment of X Chromosome Centromere Instability in Alzheimer’s Disease: A Quantitative FISH Approach
              </a>
            </td>
          <td>
            B. Spremo-Potparević, Petar Popović, Dijana Topalović, Andrea Pirković, George Perry, L. Živković
          </td>
          <td>2025-06-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction As the most prevalent internal RNA modification in eukaryotic transcripts, N6-methyladenosine (m6A) which is catalyzed by methyltransferase-like 3 (METTL3), is widely involved in cancerous diseases. However, the role of METTL3 and small nucleolar RNA host gene 1 (SNHG1) playing in osteosarcoma (OS) remains largely unknown. Methods Bioinformatics analysis, RT-qPCR, western blotting assays were used to detect the expression of METTL3, SNHG1, RNA binding motif protein 15 (RBM15), WD repeat domain 74 (WDR74) and EWS RNA binding protein 1 (EWSR1) accordingly. Cell proliferation and motility ability changes were assessed by colony formation and transwell migration assays. RNA stability changes were evaluated by an actinomycin D assay. The level of SNHG1 m6A modification changes were addressed by an RNA immunoprecipitation (MeRIP)-qPCR assay. RNA pulldown assays and RNA immunoprecipitation assays were applied to detect the interactions between SNHG1 and proteins. A chromatin immunoprecipitation (ChIP)-qPCR assay was performed to verify the binding effect between WDR74 promoter region and EWSR1. Orthotopic xenograft mouse models were constructed to evaluate the role of METTL3 playing in OS tumorigenesis and lung metastasis in vivo. Results It was uncovered that METTL3 was significantly upregulated in OS tissues and cell lines. As an oncogenic regulator, METTL3 promoted proliferation and migration in OS cells by enhancing the stability of SNHG1. Mechanically, it was displayed that METTL3 catalyzed SNHG1 m6A modification with the assistance of RBM15. More deeply, it was found that SNHG1 promoted OS cells proliferation and migration via regulation of its neighboring gene WDR74. Meanwhile, it was discovered that SNHG1 affected WDR74 transcription by EWSR1 recruitment. Finally, it was displayed that overexpression of METTL3 promoted SNHG1 and WDR74 expression, and upregulation of METTL3 facilitated OS tumorigenesis and lung metastasis in vivo. Conclusion The present research illustrated that METTL3 enhanced the stability of SNHG1 with the assistance of RBM15 in an m6A dependent manner in OS cells. And SNHG1, promoted the transcription of WDR74 in cis, via recruitment of EWSR1, thereby facilitated WDR74-mediated proliferation and migration in OS cells. These findings provide new insights into the epigenetic regulation of OS and highlight potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2ca98a4d106c9a2af12fa851ebd022b78fd1b3b" target='_blank'>
              METTL3-mediated SNHG1 m6A modification promotes proliferation and migration through transcriptional regulation of WDR74 in osteosarcoma
              </a>
            </td>
          <td>
            Guanzhen Qiu, Yuxin Bao, Yuanzhuang Zhang, Yeqiu Xu, Tianhua Qiao, Chenghao Li, Hanjie Zhai, Zhenjun Chen, Fu Ren, Yong Wang
          </td>
          <td>2025-05-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4533ea1c7d9cb0be79c4a8caa2ae7f918764b84b" target='_blank'>
              PCSK9 expression and cancer survival: a prognostic biomarker at the intersection of oncology and geroscience.
              </a>
            </td>
          <td>
            Z. Ungvári, O. Menyhárt, Andrea Lehoczki, M. Fekete, G. Bianchini, Balázs Győrffy
          </td>
          <td>2025-06-27</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Introduction Lung adenocarcinoma (LUAD) is among the most prevalent and lethal forms of cancer worldwide, largely due to the lack of early symptoms and frequent late-stage diagnosis. G protein γ subunit 7 (GNG7) has been implicated in the regulation of cell proliferation, apoptosis, and migration across various cancers. However, its immunological role in LUAD progression remains poorly understood. Methods We analyzed The Cancer Genome Atlas (TCGA) database to assess the relationship between GNG7 expression and clinical outcomes in LUAD. Immune cell infiltration and immune-related gene expression were evaluated in association with GNG7 levels. In vitro functional assays, including proliferation, migration, invasion, and apoptosis assays, were performed following GNG7 overexpression. A prognostic model was constructed based on immune-related genes regulated by GNG7 and validated using the GSE31210 and IMvigor210 cohorts. Results Low GNG7 expression was associated with enhanced tumor growth and poor prognosis in LUAD patients. GNG7 expression correlated significantly with immune cell infiltration and key immune regulatory markers. In vitro, GNG7 overexpression suppressed LUAD cell proliferation, migration, and invasion, while promoting apoptosis. The developed GNG7-related immune gene prognostic model effectively predicted both patient prognosis and immunotherapy response. Discussion Our findings highlight the critical role of GNG7 in LUAD progression and its modulation of the tumor immune microenvironment. GNG7 shows promise as a prognostic biomarker and potential therapeutic target for immune-based LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f48d0336fee1a88fd069a56195eaee5ee00c526" target='_blank'>
              GNG7 as a tumor-suppressor gene in lung adenocarcinoma: implications for prognosis and immune-based therapies
              </a>
            </td>
          <td>
            Kexin Luo, Meihan Liu, Zhongqin Peng, Haiyang Zhao, Guoyi Li, Yuanze Cai, Yumeng Lei, Hongpan Zhang, Yongsheng Zhao
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, T. Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The overexpressed gene MYEOV in multiple myeloma, as an oncogene, has been widely recognized for its high expression levels in various malignant tumors. MYEOV plays a significant role in multiple malignancies, particularly in diseases such as multiple myeloma, breast cancer, lung cancer, pancreatic cancer, and esophageal cancer. The presence of the open reading frame of MYEOV in humans and other primates suggests its potential protein-coding capacity, although direct evidence of functional MYEOV protein is currently lacking. The role of MYEOV in various tumors is not limited to its direct effects as an oncogene; it also includes its complex role in tumor cell signaling pathways and its ability to participate in miRNA regulatory networks as a competing endogenous RNA (ceRNA). Studies have shown that MYEOV may affect the expression of cancer-related genes through enhancer activity. This article aims to provide a comprehensive review of the role of MYEOV, its involvement in signaling pathways in tumor cells, and the latest advancements in MYEOV-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e799fb69a5049d179cc77bd3716c162048abf1c" target='_blank'>
              The role of MYEOV gene: a review and future directions
              </a>
            </td>
          <td>
            Yidan Xi, Xuefei Liu, Yanan Ge, Mingzhe Jiang, Dong Zhao, Ershu Zhao, Fei Cai, Xinlong Wang, Jiatong Li, Tingting Li, Zhendong Zheng
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Embryogenesis and tumorigenesis share several key biological characteristics, such as rapid cell proliferation, high plasticity, and immune evasion. This similarity indicates that developmental pathways can be hijacked, leading to the formation of malignant cell states. With regard to this, cancer can be regarded as a stem cell disease. On the contrary, a fetus, in many ways, has similar characteristics to the “ideal tumor”, such as immune evasion and rapid growth. Therefore, deciphering the molecular mechanisms beneath these phenomena will help us to understand the embryonic origins of cancer. This review discusses the relationship between embryogenesis and tumorigenesis, highlighting the potential roles played by DUX4. DUX4 is involved in the activation of the zygote genome and then facilitates the establishment of totipotency in pre-implantation embryos, whereas the misexpression of DUX4 is associated with different types of cancer. Taken together, this indicates that DUX4 performs analogous functions in these two processes and connects embryogenesis and tumorigenesis. Through examining DUX4, this review underscores the importance of developmental mechanisms in cancer biology, suggesting that the insights gained from studying embryonic processes may provide novel therapeutic strategies. As we continue to explore the complex relationship between cancer and embryogenesis, elucidating the role of DUX4 in linking these two processes will be critical for developing targeted therapies that exploit developmental pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f5d78fbcd896e6b005b6f4da73702c30eda3a6" target='_blank'>
              Embryonic Origins of Cancer: Insights from Double Homeobox 4 Regulation
              </a>
            </td>
          <td>
            B. Fu, Hong Ma, Liang Wang, Zhenhua Guo, Fang Wang, Di Liu, Dongjie Zhang
          </td>
          <td>2025-05-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background/Aim: Tongue squamous cell carcinoma (TSCC), a highly aggressive subtype of head and neck cancers, is characterized by frequent lymphatic metastasis and poor prognosis. Recently, we showed that lymphatic vessel endothelial hyaluronan receptor 1 (LYVE-1) is involved in TSCC progression, yet the underlying molecular mechanisms remain unclear. Materials and Methods: CRISPR/Cas9 gene editing was employed to generate LYVE1 knockout (KO) TSCC cell lines. Single-cell clones were isolated, screened, and validated through sequencing and Inference of CRISPR Edits (ICE) analysis and qRT-PCR. RNA sequencing was performed on LYVE1 KO and wild-type (WT) cells to identify differentially expressed genes (DEGs). Bioinformatic analyses, including Gene Ontology (GO) enrichment and protein-protein interaction (PPI) network mapping, were conducted to explore affected pathways. Finally, network topology was examined using NetworkAnalyzer and cytoHubba plugins. Results: Transcriptomic analysis revealed significant down-regulation of pro-metastatic pathways, including epithelial-mesenchymal transition (EMT), extracellular matrix remodeling, and immune modulation. DEG analysis identified 263 genes, with key down-regulated targets such as WNT5A, TGFB2, and MMP2, and up-regulation of tumor-suppressive genes including PTGS2. GO and PPI analyses highlighted LYVE1’s pivotal role in regulating cell adhesion, migration, and immune response. Conclusion: LYVE1 KO reduces TSCC invasive potential by disrupting EMT and tumor-stroma interactions, aligning with previous experimental findings. These results suggest LYVE1 as a critical driver of metastasis, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff3c633bd83b5225f79493e53059643bbd3c2790" target='_blank'>
              CRISPR/Cas9-mediated Knockout of LYVE1 In Human Tongue Cancer Cells Reveals Transcriptomic Changes in Metastasis-associated Pathways
              </a>
            </td>
          <td>
            Sini Karinen, Johanna FORERO-RODRÍGUEZ, Annika Järvinen, Kari K Eklund, Gonçalo Barreto, A. Salem
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. 
Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nu- clear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syn- drome in the family, the maternal grandfather had a similar tumor requiring amputation. 
Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-06-20</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, Dariusz Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA) is present in food, water, and drugs and is considered a probable human carcinogen by the International Agency for Research on Cancer. The mechanism of action of NDMA involves the generation of carcinogenic methyl lesions such as 3-methyladenine (3MeA) on DNA bases. Alkyladenine DNA Glycosylase (AAG) removes 3MeA to initiate Base Excision Repair, leaving an intermediary lesion that is subsequently resolved by backbone cleavage, nucleotide insertion, and backbone ligation. The intermediate steps following lesion removal produce potentially toxic and mutagenic single-strand DNA breaks. Here, we explored differences between males and females regarding downstream DNA damage, toxicity, mutations and cancer arising from 3MeA in the livers of WT, Aag-/-, and Aag-overexpressing (AagTg) mice. We found that males were more susceptible to NDMA-induced mutations (WT and Aag-/-) and cancer (all genotypes). In contrast, AagTg females were more prone than males to micronucleus induction. As we showed in our prior analyses where data were pooled for males and females, Aag-/- mice were significantly more susceptible to NDMA-induced mutations and cancer, and AagTg mice displayed significantly greater toxicity. Building on these findings, our analyses of sex-related differences show that Aag deficiency and maleness are both susceptibility factors for NDMA-induced liver cancer, while Aag overexpression drives toxicity, potentially with a greater effect on females. By assessing differences between males and females, this study reveals a deeper mechanistic understanding of the underpinnings for a well-known increased risk of liver cancer in men versus women by demonstrating a higher susceptibility of male mice to both mutations and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2ad8f9213bd9ddfa98933a729d0b81e465af3f" target='_blank'>
              Sex and Alkyladenine DNA Glycosylase Expression are Key Susceptibility Factors for NDMA-induced Mutations, Toxicity, and Cancer
              </a>
            </td>
          <td>
            Jennifer E. Kay, J. Corrigan, Lindsay B Volk, Amanda L Armijo, Ilana S Nazari, D. Torous, Svetlana L. Avlasevich, R. G. Croy, D. Wadduwage, S. Dertinger, J. Essigmann, Leona D. Samson, B. Engelward
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66c374a597e5040e9fb64e91003c1321c506ad2b" target='_blank'>
              cIAP1 inhibitor of apoptosis is a tumor suppressor in Ewing sarcoma
              </a>
            </td>
          <td>
            F. Cidre-Aranaz, Florian H. Geyer, T. L. Hölting, Tobias Faehling, Alina Ritter, Malenka Zimmermann, Lianghao Mao, Javier Alonso, A. Sastre, Ashok Kumar Jayavelu, M. Knott, J. Musa, T. G. Grünewald
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA topoisomerases, particularly type II, are crucial for DNA processes, such as replication, transcription, and chromosome segregation, making them prime targets for cancer therapy. This review delves into the multifaceted mechanisms of action of type II topoisomerases, highlighting their essential roles beyond cancer progression. It explores recent advancements in screening and designing metallic complexes as inhibitors of topoisomerase II activity. Emphasizing the structural and functional diversity between alpha and beta isoforms, it elucidates their significance in DNA metabolism and genome integrity. Additionally, this review discusses the interplay of topoisomerase II with cellular components, underscoring its regulatory roles in gene expression. Insights into screening and design strategies for metallic complex inhibitors are provided, showcasing their therapeutic potential against cancer. Overall, this review highlights the importance of understanding topoisomerase II inhibition mechanisms and the versatility of metallic complexes in biomedical research, paving the way for novel therapeutic strategies and broader applications beyond cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a3a5eac5f306d411595288c7868532f4247f662" target='_blank'>
              Topoisomerase II Inhibition in Cancer: A Focus on Metal Complexes.
              </a>
            </td>
          <td>
            A. O. Akinyemi, Josias da Silveira Rocha, Gabriela Porto de Oliveira, Josenilton de Jesus Santos, B. Owoyemi, Fillipe V. Rocha
          </td>
          <td>2025-06-05</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer progression is a complex, multi-stage development process characterized by dynamic changes at the molecular level. Understanding these changes may provide new insights into tumorigenesis and potential therapeutic targets. This study focuses on the evolutionary transcriptomics of cancer, specifically analyzing the Transcriptome Age Index (TAI) across different pathological stages. By examining various cancers at four distinct pathological stages, we identify a significant «hourglass» pattern in TAI indices of ductal carcinoma of the breast, bladder carcinoma, and liver carcinoma, suggesting a conserved evolutionary trajectory during tumor development. The results reveal that early and late stages of these cancers exhibit higher TAI values, indicative of more novel gene expression, while intermediate stages show a dip in TAI, reflecting a more ancient evolutionary origin of expressed genes. This «hourglass» pattern underscores the evolutionary constraints and innovations at play during tumor progression. Our findings contribute to the growing body of evidence that evolutionary principles are deeply embedded in cancer biology, offering new perspectives on the dynamics of gene expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d899ba20c349660eb4cd72c9aa668578d579a2" target='_blank'>
              Evolutionary Transcriptomics of Cancer Development
              </a>
            </td>
          <td>
            Roman Ivanov, D. Afonnikov, Yury Matushkin, S. Lashin
          </td>
          <td>2025-05-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Esophageal cancer (ESCA) is a highly aggressive malignancy with poor prognosis. Small nuclear ribonucleoprotein polypeptide B2 (SNRPB2) is a core component of the spliceosome involved in pre-mRNA splicing. However, its role in tumor development and progression remains largely unclear. This study aimed to evaluate the clinical relevance and prognostic value of SNRPB2 in ESCA. Methods SNRPB2 mRNA expression and genetic alterations were analyzed using GEPIA2 and cBioPortal. Protein expression was assessed by immunohistochemistry in paraffin-embedded esophageal squamous cell carcinoma (ESCC) tissues. Functional assays in ESCC cell lines were conducted to determine the biological role of SNRPB2. Immune-related and functional analyses were performed using TIMER, TISIDB, TISCH, Gene Ontology (GO), and Gene Set Enrichment Analysis (GSEA). Cycloheximide (CHX) chase assays were used to assess protein stability. Results SNRPB2 mRNA was upregulated in ESCA and associated with tumor progression and poor prognosis. Immunohistochemistry confirmed high SNRPB2 protein expression in ESCC, correlating with vessel carcinoma embolus, lymph node metastasis, clinical stage, and tumor grade. SNRPB2 knockdown significantly inhibited ESCC cell proliferation, migration, and invasion in vitro and in vivo. GSEA indicated that SNRPB2 suppresses the Rb/E2F pathway. Mechanistically, SNRPB2 stabilized E2F4 protein by preventing its proteasomal degradation, and E2F4 overexpression reversed the tumor-suppressive effects of SNRPB2 silencing. Immune analyses showed that SNRPB2 expression correlated with increased infiltration of activated CD8+ T cells, γδ T cells, dendritic cells, and monocytes, as well as immune-related genes including PDCD1, CD274, CTLA4, HLA-DRA, and B2M. These findings suggest a dual role for SNRPB2 in promoting tumor progression and modulating the immune microenvironment in ESCA. Conclusion SNRPB2 promotes ESCC progression by stabilizing E2F4 and regulating cell cycle genes. It is also associated with immune infiltration and gene expression in ESCA. SNRPB2 may serve as a prognostic biomarker and potential therapeutic target in esophageal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be72f1ea4a0bed7498caf12a2cefb50221a4f455" target='_blank'>
              SNRPB2 facilitates esophageal squamous cell carcinoma oncogenesis and progression via E2F4 stabilization
              </a>
            </td>
          <td>
            Feng Xu, Chen-Cheng Zhu, Chen Lu, Guangyao Ning, Ren-Quan Zhang
          </td>
          <td>2025-06-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Previously, we found that an uncharacterized protein CXorf38 is significantly downregulated in human ZIP8-knockout (KO) cells. Given that ZIP8 regulates essential micronutrients linked to diseases including cancer, this study aims to characterize CXorf38 and evaluate its role in lung adenocarcinoma. Methods: iTRAQ-based proteomics was previously used to identify the abundance of proteins in ZIP8-KO cells. Cell proliferation and colony formation assays were used to examine the function of CXorf38 by overexpressing the gene in lung adenocarcinoma cell lines. Kaplan–Meier survival analysis was used to assess the prognostic value of CXorf38, while TCGA clinical database analysis was used to evaluate its expression in lung cancer tissues, particularly in smokers. Bioinformatics analyses (GO, KEGG, PPI, and ICI) were performed on CXorf38-coexpressed genes derived from patients with lung cancer. Results: CXorf38 overexpression suppressed lung cancer cell proliferation and colony formation, suggesting a tumor-suppressive role. Higher CXorf38 expression correlated with improved survival in patients with lung adenocarcinoma, but not in lung squamous cell carcinoma. Clinical data showed CXorf38 downregulation with lung cancer tissues of smokers, indicating a potential role in smoking-induced cancer progression and treatment. Functional analysis using bioinformatics linked CXorf38 to immune response regulation, suggesting involvement in the tumor immune microenvironment. Conclusions: Our study reveals for the first time that CXorf38 is a potential tumor suppressor, prognostic biomarker, and/or tumor immune regulator in lung adenocarcinoma—further research is warranted to explore its role in tumor immunity and its therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de769da2616c4ecb2cadcb3f426fcabdecfbd13" target='_blank'>
              Chromosome X Open Reading Frame 38 (CXorf38) Is a Tumor Suppressor and Potential Prognostic Biomarker in Lung Adenocarcinoma: The First Characterization
              </a>
            </td>
          <td>
            Rui Yan, H. Tan, Na-Li Cai, Le Yu, Yan Gao, Yan-Ming Xu, Andy T. Y. Lau
          </td>
          <td>2025-06-01</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f31d8e3aa41a4d4ed605eb1a2bff762b1dfaf76" target='_blank'>
              NUSAP1 regulates mitotic processes via KIF2C interaction and AURKA phosphorylation in primary microcephaly
              </a>
            </td>
          <td>
            Decheng Ren, Keyi Li, Zhen Liu, Yan Bi, Liangjie Liu, Lei Ji, Ke Yang, Yingying Luo, Le Luo, Yang Yan, Yang Li, Feng-ping Yang, Hua Wang, Guang He, Xiao Mao
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Women who harbor germline heterozygous mutations of BRCA1 or BRCA2 have a high risk of breast cancer. Our previous study showed that patient-derived, ostensibly normal BRCA2mut/+ luminal progenitor (LP) cells are more prone to exhibit sub-chromosomal copy number variations and associated DNA damage relative to non-carriers, potentially reflecting early breast tumorigenesis. Clinically assessable biomarkers for early pathological changes of BRCA1/2 mutation in LP cells remain unknown. Single-cell RNA sequencing (scRNAseq) of LP cells of BRCA1/2mut/+ carriers revealed enrichment of KIT expression (KIT+) and subsequent transcriptional factor activations were observed in LP cells of BRCA1/2mut/+ carriers relative to non-carriers. Moreover, pathway analysis uncovered that KIT+ BRCA-mutated LP cells were enriched in pathways involving DNA binding transcription activator activity and oxidative phosphorylation. These gene signature profiles were recapitulated in bulk RNA-seq of BRCA2 mut/+ carrier LP cells. Collectively, our preliminary data suggest such analyses may identify potential biomarkers for near-term risk prediction that reflect early pathogenesis of BRCA1/2-associated carcinogenesis. Following further validation via tissue microassay analysis and clinical studies, our findings may eventually assist patients and clinicians in decision making regarding the timing and necessity of preventive surgeries for BRCA1/2 mutation carriers.
 Citation Format: Zuen Ren, Siang Boon Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, Veerle Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W. Ellisen. Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a865906cd027bf501016cc53a811825bba9923" target='_blank'>
              Abstract P3-04-06: Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer
              </a>
            </td>
          <td>
            Zuen Ren, S. Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, V. Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W Ellisen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461cfd126a8923a87cd09690f854d762a0619664" target='_blank'>
              A DNA alteration and methylation co-detection method for clinical purpose.
              </a>
            </td>
          <td>
            Jiyan Yu, Chunhe Yang, Xintao Zhu, Zhankun Wang, Boping Xu, Ye Cai, Jingbo Zhao, Ruijian Guo, Wuzhou Yuan, Jianqing Wang, Bohao Dong, Frank Ron Zheng, Shuang Yang
          </td>
          <td>2025-06-04</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Oral squamous cell carcinoma (OSCC) is the most common malignant tumor in the oral cavity, with over 70% of cases detected in advanced stages and a 5-year survival rate of 50%. RAD51, a crucial DNA repair molecule, has also been implicated in the regulation of cell adhesion. However, its multi-platform expression patterns and functional role in cell adhesion within the context of OSCC have not yet been comprehensively validated. Methods Bulk RNA (RNA-seq and microarray), immunohistochemistry (IHC), and single-cell sequencing (scRNA-seq) were used to assess RAD51 expression and its clinical relevance in OSCC. CRISPR was used to knock out RAD51 and explore its effect on OSCC cell proliferation. Bulk RNA and scRNA-seq analyses were integrated to explore RAD51’s molecular mechanisms, cellular communication, and metabolic pathways. Drug sensitivity analysis aimed to explore the relationship between RAD51 and drugs, while the target molecules of RAD51 transcription factors were identified. Results In bulk RNA (1,006 OSCC samples), IHC (162 OSCC tissues), and scRNA-seq (9,693 malignant epithelial cells), RAD51 was comprehensively upregulated in OSCC. The proliferation of OSCC cells was inhibited due to knockout of RAD51. Bulk RNA and scRNA-seq indicated that RAD51 was associated with cell adhesion. High expression of RAD51 was associated with the MIF pathway in cell communication, with the TCA cycle metabolism being mainly regulated. The expression of RAD51 was associated with resistance to Erlotinib, Gefitinib, and Selumetinib. In the molecular network, RAD51 was demonstrated a targeting relationship with adhesion-related molecules like CD74, JUP and MIF. Conclusion High expression of RAD51 may promote the advancement of OSCC by regulating the related mechanisms of cell adhesion. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03085-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909ee3395b3a888687421947a7604b186d6873d8" target='_blank'>
              Molecular mechanism and biological pathway of high-expressed RAD51 in regulating cell adhesion and potentially affecting oral squamous cell carcinoma
              </a>
            </td>
          <td>
            , , Yan Cui, , Lin-Qing Huang, Chang-Zhan Zhou, Bin Li, Qi Li, Bang-Teng Chi, , Di-Yuan Qin, Jian-Di Li, Gang Chen, Dong-Ming Li, Y. Dang
          </td>
          <td>2025-07-04</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Pituitary adenomas (PAs) are benign tumors in the pituitary gland. However, 30–40% of these tumors are invasive, complicating diagnosis and treatment. Invasive pituitary adenomas (IPAs) often respond poorly to conventional therapies, emphasizing the need for better diagnostic and therapeutic strategies. Understanding DNA methylation patterns in IPAs may reveal new biomarkers and therapeutic targets, leading to more effective management of this challenging disease. Methods Reduced representation bisulfite sequencing (RRBS) and RNA sequencing (RNA-seq) were performed on 129 samples from the Second Affiliated Hospital of Zhejiang University, including 69 tissue samples from invasive and non-invasive pituitary adenomas (NPA) and 60 blood samples from IPA, NPA and healthy individuals. Differentially methylated regions (DMRs) and differentially expressed genes (DEGs) were identified in tissues. Pearson correlation analysis was used to identify associations between DNA methylation status and gene expression, as well as the effect of methylation on gene expression at different sites. Blood samples were analyzed to detect DMRs and DEGs, correlating with tissue-derived findings. Finally, ROC analysis and a random forest model were used to identify biomarkers for discriminating invasive from non-invasive phenotypes. Results We identified 347 DMRs between IPA and NPA, of which 63% (219/347) were hypomethylated. Additionally, 543 mRNAs showed differential expression, with 350 upregulated and 193 downregulated. 17 genes demonstrated concurrent aberrant methylation and expression, primarily within introns, promoters, and CpG islands (CGIs). Notably, only protein tyrosine phosphatase receptor type T (PTPRT) exhibited a remarkably high correlation (r = 0.81) between its DNA methylation levels and mRNA expression levels. This correlation was observed within the intronic region/opensea of the gene's CGIs. Plasma sample analysis revealed 852 DMRs between IPA and NPA, with 52% (447/852) being hypomethylated. Three tumor tissue-derived blood biomarkers (MIR4535, SLC8A1-AS1, and TTC34) accurately discriminated between IPA and NPA patients with a combined AUC of 0.980. These markers also differentiated NPA from healthy controls, though with different methylation patterns. Conclusions The relationship between DNA methylation and gene expression is complex. Plasma-based DNA methylation markers can effectively discriminate between IPA and NPA, as well as between NPA and healthy individuals (N group). Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04164-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63168fb628263eeafdc395f9c006371c823b818" target='_blank'>
              Molecular signatures of invasive and non-invasive pituitary adenomas: a comprehensive analysis of DNA methylation and gene expression
              </a>
            </td>
          <td>
            Yike Chen, Ningning Zhao, Jiahao Zhang, Xinyi Wu, Jian Huang, Xiaohui Xu, Feng Cai, Sheng Chen, Liyin Xu, Wei Yan, Yuan Hong, Yunfei Wang, Hui Ling, Jianxiong Ji, Gao Chen, Hongcang Gu, Jianmin Zhang, Qun Wu
          </td>
          <td>2025-07-01</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Papillary thyroid cancer (PTC) is an ever-increasing cancer type worldwide, and greatly decreases the life quality and affects survival time of patients during its development and progression, but the underlying mechanisms and key factors for PTC progression are not clear. Recent studies demonstrated the potassium channel protein SK4 participates in the progression of many cancers, while it lacks the molecular mechanism study for SK4 function. In this study, we performed functional and molecular explorations for SK4 by overexpressing its level in thyroid cancer BHT101 cells. Cellular proliferation and invasion experiments were performed to assess the influences of SK4 on cell behaviors. Further, whole transcriptome sequencing (RNA-seq) analysis helped us systematically investigated the targets of SK4, including differentially expressed genes (DEGs) and regulated alternative splicing events (RASEs), and validated several related DEGs and RASEs by RT-qPCR experiment. In thyroid cancer patients, SK4 expression was completely lost in normal tissues and significantly increased in every stage of tumor tissues compared with normal tissues, which probably results from the low DNA methylation level at its promoter region. Consistent with previous study, SK4 overexpression (SK4-OE) promoted proliferation and invasion ability of BHT10 cells compared with negative control (NC). By analyzing the RNA-seq data, we detected dozens of DEGs and found that up DEGs were enriched in negative regulation of apoptotic progress, including VTCN1, MSX1, FATE1, TEK, and PRAMEF2. More importantly, we found SK4-OE globally changed the alternative splicing (AS) pattern and identified 1,639 RASEs. The genes of RASEs were enriched in DNA damage/repair, viral process, translation, and mRNA splicing pathways, which were tightly associated with the pathogenesis and progression of cancers. The splicing regulatory genes from regulated alternative splicing genes (RASGs) could partly explain the reason of global AS dysregulation by SK4-OE in BHT101 cells. Finally, we found the expression of VTCN1, EDN1, SLC29A4, RP11-473M20.16, and CH507-513H4.4 were validated by RT-qPCR, as well as the AS pattern of TMEM116. In summary, we highlight that SK4-regulated AS pattern probably is a novel regulatory mechanism for SK4 in PTC. The identified DEGs and RASEs, as well as SK4 itself, could be used as potential therapeutic targets for PTC treatment in future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bf9b607a916a81e5a08e0bc05a7b09aa5fe042f" target='_blank'>
              SK4 potentially modulates the alternative splicing profile associated with papillary thyroid cancer development in BHT101 cells
              </a>
            </td>
          <td>
            Alimujiang Wupuer, Xuekelaiti Kuerban, Gong Ye
          </td>
          <td>2025-05-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Over the past decade, advancements in next-generation sequencing have significantly enhanced our understanding of the molecular pathogenesis of adamantinomatous craniopharyngiomas (ACP). Objective This study integrated methylome and transcriptome analyses in ACP samples to explore the potential interplay between DNA methylation and RNA expression signatures for diagnostic and prognostic applications in ACP patients. Methods This cross-sectional study evaluated clinicopathological features, DNA methylation, and gene expression profiles in 15 patients with ACP (33% women, age range: 3–55 years, 53% diagnosed before 18 years) treated at Ribeirao Preto Medical School, University of São Paulo. Results Multidimensional scaling and principal component analysis identified two distinct clusters (ACP-A: n=9, ACP-B: n=6) with consistent composition across DNA methylation and gene expression profiles. While most clinical and histopathological characteristics were similar between clusters, ACP-A exhibited a longer median progression-free survival. ACP-B showed a higher prevalence of hypomethylated probes in CGI sites, and 63% of differentially methylated positions (DMPs) located in gene body regions. Differential methylation patterns were categorized into Methyl-Set1 (hypomethylated in ACP-A and hypermethylated in ACP-B) and Methyl-Set2 (hypermethylated in ACP-A and hypomethylated in ACP-B). Clustering analyses based on the methylation levels of probes and expression levels of the stringently filtered 212- and 37-gene sets further confirmed these two distinct ACP subgroups. Functional enrichment analysis highlighted key roles in synaptic modulation, nervous system development, cell adhesion, as well as pathways linked to RAS signaling, GTPase activity, and membrane potential regulation. Conclusion Although clinical characteristics were largely comparable between the clusters, ACP-B patients exhibited shorter median progression-free survival, suggesting a more aggressive phenotype. The higher prevalence of hypomethylation in ACP-B indicates increased transcriptional activation, potentially driving tumor aggressiveness. The strong concordance between methylation and transcriptomic data in the 212- and 37-gene sets underscores their potential as a clinically relevant molecular biomarker panel. These gene sets demonstrate robustness in distinguishing ACP clusters, making it a promising tool for clinical sample classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1321c37193ba8ab134af26f85d3133bdcee99c23" target='_blank'>
              Methylation and gene expression patterns in adamantinomatous craniopharyngioma highlight a panel of genes associated with disease progression-free survival
              </a>
            </td>
          <td>
            J. Marrero-Gutiérrez, A. Bueno, C. Martins, Rui M Patrício Silva-Júnior, Antônio Carlos dos Santos, M. V. Santos, L. E. Wildemberg, Ximene Antunes, Mônica R Gadelha, A. C. Moreira, R. N. Vêncio, S. R. Antonini, Margaret de Castro
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis.We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome.Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities.Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18c7e36e3b876db6b83aaa4bdaf8e2434f722c8a" target='_blank'>
              Oncogenic fusions converge on shared mechanisms in initiating astroblastoma.
              </a>
            </td>
          <td>
            Yixing Shi, Qianqian Sun, Fuchuan Jia, Xiangyu Xie, Xiangyu Zhou, Rong Guo, Yangfan Zeng, Shanshan Chen, Zhenzhen Guo, Wenli Sun, Tong Guo, Yu Xia, Wenlong Li, Li Zhang, Wei Shi, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Nature</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Patients with malignant gliomas with methylated MGMT promoters are generally more sensitive to alkylating chemotherapy, as this modification impedes DNA repair. However, inconsistencies in the predictive accuracy of MGMT promoter methylation have been observed. We hypothesize that these variations may be partially explained by a counteracting influence of MGMT gene body methylation. Data from The Cancer Genome Atlas (TCGA) were analyzed to assess correlations between MGMT promoter and body methylation with transcript production across cancer types and within glioma subcohorts. Thirty-six human glioma cell lines underwent molecular profiling via Illumina 850k Methylation Arrays and RNA sequencing. A subset was further tested for MGMT protein levels and carmustine response. Correlations and linear regression analyses were conducted to investigate association of carmustine sensitivity with different levels of MGMT expression. MGMT mRNA expression was positively correlated with body methylation and negatively correlated with promoter methylation across cancers from TCGA. Body and promoter methylation were anticorrelated in the non-glioma cohort and IDH1/2 wild type glioma subcohort but not correlated in the IDH1/2 mutated subcohort. Most glioma cell lines did not express MGMT mRNA. In the cell lines tested for carmustine response, sensitivity was negatively correlated with body methylation and mRNA expression and positively correlated with promoter methylation. Our findings further expound the relationship between MGMT methylation patterns and alkylating agent response, with body methylation playing a significant role. The identified role of gene body methylation underscores the need to integrate the interplay between promoter and body methylation in clinical testing and predicting treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf4ebe875a9f744f600e2f94aecc5978a94fe9bb" target='_blank'>
              Beyond the Promoter: Total MGMT Gene Methylation Modulates Response to DNA Alkylating Agents in Glioma.
              </a>
            </td>
          <td>
            Nicole J Briceno, Jinkyu Jung, Aiguo Li, Chunzhang Yang, Mioara Larion, L. Pongor, Fathi Elloumi, Sudhir Varma, William C Reinhold, Y. Pommier, Mark R Gilbert, O. Celiku
          </td>
          <td>2025-06-04</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f42a49dad6477b432ac875de1bc87a28316b76b" target='_blank'>
              High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer
              </a>
            </td>
          <td>
            Michelli F Oliveira, J. P. Romero, Meii Chung, Stephen R. Williams, Andrew D. Gottscho, Anushka Gupta, Susan E. Pilipauskas, Seayar Mohabbat, Nandhini Raman, David Sukovich, David Patterson, Sarah E B Taylor
          </td>
          <td>2025-06-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 2519


 Background:
 Tumor heterogeneity and low mutational burden limits the availability of effective immunotherapy targets. Aberrant RNA-splicing (neojunctions) represents an underexplored yet promising source of neoantigens. To address this, we developed a neoantigen discovery platform (SNIPP) that characterizes a novel class of clonally-expressed, splicing-derived neoantigens. Furthermore, we validated the immunogenicity of these neoantigens by identifying specific TCRs that drive CD8+ T-cell-mediated tumor killing.
 Methods:
 SNIPP identified public neojunctions by analyzing TCGA RNA-seq data, selecting neojunctions with a positive sample rate (PSR) > 10% and filtering out those found in GTEx normal tissue RNA-seq data (PSR < 1%) across 12 cancer types. To characterize intratumorally conserved neojunctions, we performed maximally-distanced multi-site biopsies (
 n
 = 535) within glioma patients (
 n
 = 56) and generated RNA-seq data for each intratumoral site. Two independent algorithms were utilized to predict peptide processing likelihood and HLA-binding affinity of splicing-derived neoantigen candidates. Neoantigen-specific TCR sequences were identified via
 in vitro
 sensitization of PBMCs and subsequent 10x V(D)J scRNA-seq. These TCRs were transduced into CD8+ T-cells, which were tested downstream for immunogenicity and cytotoxicity against glioma cell lines.
 Results:
 Our pipeline identified 789 public neojunctions, including 32 neojunctions concurrently detected in transcriptomic and proteomic glioma datasets and confidently predicted to be presented by HLA-A*02:01. IVS and subsequent 10x V(D)J scRNA-seq identified TCR clonotypes reactive against neojunctions in
 RPL22
 (
 n
 = 7) and
 GNAS
 (
 n
 = 1), with the latter exhibiting high intratumoral conservation (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). TCR-transduced CD8+ T-cells recognized and were immunogenically activated and demonstrated cytotoxicity against endogenously processed and presented neoantigens in GBM and melanoma lines. Additionally, IDH1-mutant oligodendrogliomas exhibited significantly higher neojunction expression compared to IDH1-mutant astrocytomas and IDH1wt subtypes. Differential gene expression analysis (DESeq2) revealed reduced expression of splicing factors in oligodendrogliomas, attributed to their specific co-deletion of chromosomes 1p and 19q. CRISPRi-mediated knockdown of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding neojunctions.
 Conclusions:
 Our study highlights a novel class of neoantigens derived from tumor-wide aberrant RNA splicing. The SNIPP platform effectively identifies public intratumorally-conserved neojunctions with strong therapeutic potential. Furthermore, elevated neojunction expression in oligodendroglioma underscores the mechanistic link between dysregulated splicing factor expression and RNA splicing abnormalities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476976fc09e028193d5d0a3658a6d5d31763d4d8" target='_blank'>
              Tumor-wide RNA splicing aberrations and their potential as therapeutic neoantigen targets.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O. Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee-may Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Aaron A. Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideho Okada
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is among the most prevalent malignancies worldwide. Cuproptosis, a copper-induced form of cell death, has been identified as a key process in LUAD progression; however, the molecular mechanisms underlying cuproptosis in LUAD and potential therapeutic targets remain unclear. The present study utilized The Cancer Genome Atlas database to retrieve mRNA expression profiles and clinical information of LUAD, identifying 10 candidate genes from differentially expressed genes associated with cuproptosis. Protein-protein interaction analysis indicated that CDK inhibitor 2A (CDKN2A), an upregulated gene in LUAD, may function as a hub gene. Furthermore, multiple online databases were used to analyze Spi-1 proto-oncogene (SPI1), a transcription factor upstream of CDKN2A, which was downregulated in LUAD cuproptosis. The LinkedOmics database identified the p53-mediated cuproptosis-related pathway regulated by CDKN2A. Gene expression patterns were examined through Gene Expression Profiling Interactive Analysis, the Human Protein Atlas and reverse transcription-quantitative polymerase chain reaction. Prognostic significance was assessed using the UALCAN and Kaplan-Meier plotter databases. In vitro experiments demonstrated that CDKN2A knockdown and SPI1 overexpression inhibited the proliferation and migration of the H1975 cell line. After copper-induced cuproptosis in H1975 cells, SPI1 expression was upregulated, whereas CDKN2A expression was downregulated. When H1975 cells were pretreated with tetrathiomolybdate, the upregulation of SPI1 was inhibited and the downregulation of CDKN2A was also suppressed. Cell Counting Kit-8 assays indicated that SPI1 overexpression and CDKN2A knockdown facilitated elesclomol-CuCl2-induced cuproptosis. Western blot analysis revealed an inverse association between SPI1 overexpression and CDKN2A/p53 levels. In conclusion, the present study demonstrated the role of the SPI1/CDKN2A/p53 axis in LUAD cuproptosis, providing insights into potential therapeutic targets and contributing to clinical research on treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a265e6ec17f3bc3d7df377a278752607ec192b44" target='_blank'>
              Role of the SPI1/CDKN2A/p53 signaling pathway in cuproptosis of lung adenocarcinoma cells
              </a>
            </td>
          <td>
            Jiayue An, Wei-Wei Song, Qin Wang, Boyu Tan, Xuan Fei, Ruoxi Wang, Siyan Li, Xiyu Lu, Youjie Li, Ning Xie
          </td>
          <td>2025-05-19</td>
          <td>Oncology Letters</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A. Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genome instability is a major force driving tumorigenesis. The ssDNA-binding protein complex CTC1-STN1-TEN1 (CST) plays a pivotal role in maintaining genome stability by countering replication stress, modulating DNA damage repair, and maintaining telomere integrity. Despite its well-documented role in genome maintenance, the involvement of CST in skin cancer development has yet to be investigated. We recently found that CST localizes at stalled DNA replication sites after UV exposure and may suppress the unwanted repriming activity, suggesting a potential role of CST in suppressing genome instability caused by UV damage. In this study, we first analyzed CST expression and alterations in cutaneous melanoma database and found that the CST genes are frequently altered in cutaneous melanoma and their expression is significantly downregulated in melanoma samples compared to normal tissues. We then generated a conditional knockout (cKO) mouse model with STN1 deficiency specifically in melanocytes to investigate its role in skin cancer formation. Upon chronic exposure to UV irradiation, STN1-deficient mice exhibit no obvious difference in melanoma incidence compared to control littermates, suggesting that STN1 downregulation in mature melanocytes has no significant effect on UV-induced skin cancer development in lab mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b040e5d51366f43fd311948a74f723a17b17afb" target='_blank'>
              In vivo investigation of STN1 downregulation in melanoma formation in adult mice following UV irradiation
              </a>
            </td>
          <td>
            Sara Knowles, Fang Wang, Maarten C Bosland, S. Gaddameedhi, Weihang Chai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The IgLON family of tumor suppressor genes (TSG) impact a variety of cellular processes involved in cancer and non-cancer biology. OPCML is a member of this family and its inactivation is an important control point in oncogenesis and tumor growth. Here, we analyze RNA-Seq expression ratios in ovarian cancers from The Cancer Genome Atlas (TCGA) (189 subjects at Stage III) to identify genes that exhibit a cooperative survival impact (via Kaplan–Meier survival curves) with OPCML expression. Using enrichment analyses, we reconstruct functional pathway impacts revealing interactions of OPCML, and then validate these in independent cohorts of ovarian cancer. These results emphasize the role of OPCML’s regulation of receptor tyrosine kinase (RTK) signaling pathways (PI3K/AKT and MEK/ERK) while identifying three new potential RTK transcriptomic linkages to KIT, TEK, and ROS1 in ovarian cancer. We show that other known extracellular signaling receptor ligands are also transcriptionally linked to OPCML. Several key genes were validated in GEO datasets, including KIT and TEK. Considering the range of OPCML impacts evident in our analyses on both external membrane interactions and cytosolic signal transduction, we expand the understanding of OPCML’s broad cellular influences, demonstrating a multi-functional, pleiotropic, tumor suppressor, in keeping with prior published studies of OPCML function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95469dd0a784b5b2812df87071418bdff5bea009" target='_blank'>
              Bioinformatic RNA-Seq Functional Profiling of the Tumor Suppressor Gene OPCML in Ovarian Cancers: The Multifunctional, Pleiotropic Impacts of Having Three Ig Domains
              </a>
            </td>
          <td>
            Adam G. Marsh, Franziska Görtler, S. Hafizi, Hani Gabra
          </td>
          <td>2025-05-29</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell-cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti-tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non-neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations.
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of internal medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db40ae083dc749116865ffafc0606bdc245362b3" target='_blank'>
              The penetrant chordoid glioma PRKCA mutation is an oncogenic gain-of-function kinase inactivation eliciting early onset chondrosarcoma in mice
              </a>
            </td>
          <td>
            V. Calleja, Jack C Henry, Mathias Cobbaut, Joanne Sewell, Karine Rizzoti, Francesca Houghton, Stefan Boeing, Nneka Anyanwu, Sunita Varsani-Brown, Thomas Snoeks, A. Suárez-Bonnet, S. Priestnall, N. Q. McDonald, A. Cameron, Peter J. Parker
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 e14558


 Background:
 DNA polymerases ε (
 POLE
 ) and δ1 (
 POLD1
 ) encode enzymes involved in DNA replication and proofreading. Somatic
 POLE
 /
 D1
 mutations occur in 1-3% of cancers. Loss-of-function alterations typically reside within the
 POLE
 /
 D1
 exonuclease domain and can result in high tumor mutation burden (TMB). Ultrahigh TMB (≥100 mut/Mb) is often associated with
 POLE/D1
 mutations, and results in near-universal responses to immune checkpoint inhibitors (ICI). Deciphering the contribution of
 POLE
 /
 D1
 mutations to TMB can be challenging, as these often co-occur with microsatellite instability (MSI-H) which also drives TMB.
 Methods:
 Samples were analyzed by NGS (NextSeq or NovaSeq) (Caris Life Sciences, Phx, AZ). TMB (totaled somatic mutations found per megabase pair) was evaluated in patients with
 POLE
 /
 POLD1
 /
 D2
 /
 D3
 /
 D4
 -altered cancers. MSI-H or microsatellite stable (MSS) status was determined by IHC and NGS. Cohorts were divided into pathogenic and likely pathogenic variants (i.e. exonuclease domain mutations) versus VUS alterations (i.e. variants outside the exonuclease domain). Differences in TMB between cohorts were assessed using Fisher’s exact test.
 Results:
 Pan-cancer, TMB ≥20 / ≥50 was identified in 10.9% / 3.4% of
 POLE
 VUS MSS cancers compared to 3.1% / 0.4% of MSS cancers with
 POLE
 /
 D1
 WT status (
 p
 < 0.0001). 0.94% of
 POLE
 VUS cancers had ultrahigh TMB versus 0.05% of MSS
 POLE
 /
 D1
 WT cancers (
 p
 < 0.0001). Similar rates were observed in MSS cancers with
 POLD1
 VUSs (Table). MSS cancers with pathogenic
 POLE
 variants exhibited variable rates of ultrahigh TMB depending on cancer type, ranging from 52.6% (ovarian) to 62.9% (endometrial) to 80.0% (colorectal). Notably, ultrahigh TMB occurred in 0.53%, 0.44%, and 0.44% of MSS patients with
 POLD2
 ,
 POLD3
 , and
 POLD4
 mutations (
 p
 < 0.0001), after excluding coexisting
 POLE
 /
 D1
 mutations.
 Conclusions:
 We propose that certain non-exonuclease
 POLE
 /
 D1
 mutations can result in loss-of-function and ultrahigh TMB in MSS patients. Further, tumor type appears to influence the impact of established
 POLE
 /
 D1
 driver mutations on TMB. Finally,
 POLD2
 -
 4
 mutations (not previously studied) may also contribute to genome instability and mutational load. These findings argue for deeper characterization of DNA polymerase mutations to identify patients that may benefit from ICI.






 POLE
 /
 D1
 WT + MSS[n (%)]


 POLE
 VUS Mutation + MSS[n (%)]


 POLE
 Pathogenic Mutation + MSS[n (%)]


 POLD1
 VUS Mutation + MSS[n (%)]


 POLD1
 Pathogenic Mutation + MSS[n (%)]





 Total
 139,842
 5,658
 557
 1,793
 11


 TMB ≥20
 4,378(3.1%)[ref]

 614(10.9%)
 p
 <0.0001


 529(95.0%)
 p
 <0.0001


 168(9.4%)
 p
 <0.0001


 5(45.5%)
 p
 <0.0001



 TMB ≥50
 586(0.42%)[ref]

 193(3.4%)
 p
 <0.0001


 467(83.8%)
 p
 <0.0001


 79(4.4%)
 p
 <0.0001


 2(18.2%)
 p
 =0.0009



 TMB ≥100
 73(0.05%)[ref]

 53(0.94%)
 p
 <0.0001


 343(61.6%)
 p
 <0.0001


 28(1.6%)
 p
 <0.0001


 1(9.1%)
 p
 =0.0058







 Evaluated by Fisher’s exact test comparing to respective POLE/D1 WT + MSS cohort.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb38b846148e42ca1249ce42b9e558a15a34574" target='_blank'>
              Potential impact of non-exonuclease
 POLE
 and
 POLD1
 mutations on high and ultrahigh TMB across cancers.
              </a>
            </td>
          <td>
            Alexander K. Tsai, S. Deshmukh, Sharon Wu, Dave S. B. Hoon, W. El-Deiry, Axel Grothey, Justin H Hwang, E. Antonarakis
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="In recent years, the incidence of cancer has increased, and early diagnosis and treatment are key to cancer treatment. Nucleolar and spindle-associated protein 1 (NUSAP1) is a crucial mitotic regulator. NUSAP1 binds to microtubules and mediates their attachment to chromosomes, ensuring the accurate distribution of genetic material to the two daughter cells. NUSAP1 is located on chromosome 15q15.1 and is typically expressed in proliferating cells, suggesting a potential oncogenic role. Studies have shown that NUSAP1 plays a significant role in various cancers, including breast cancer, cervical cancer, lung cancer, prostate cancer, and hepatocellular carcinoma. Furthermore, NUSAP1 may serve as a promising biomarker for cancer diagnosis and prognosis. In this review, we discuss the roles and mechanisms of NUSAP1 in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17ed1053e2b63b1afa28756e4052dc1859c3cc0" target='_blank'>
              The role and therapeutic value of NUSAP1 in human cancers
              </a>
            </td>
          <td>
            Shenyuan Xu, Ding Nan, Ruiqi Liu, Chenting Liu, Yuefeng Hu, Luanluan Huang, Xiaoyan Chen, Hailong Sheng, Yanwei Lu, Haibo Zhang
          </td>
          <td>2025-07-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Genomic imprinting is the parent-of-origin specific monoallelic expression of genes that result from complex epigenetic interactions. It is often achieved by monoallelic 5-methylcytosine, resulting in the formation of differentially methylated regions (DMRs). These show a bias towards oocyte-derived methylation and survive reprogramming in the pre-implantation embryo. Imprinting is widespread in the human placenta. We have recently performed whole-genome screens for novel imprinted placenta-specific germline DMRs (gDMRs) by comparing methylomes of gametes, blastocysts and various somatic tissues, including placenta. We observe that, unlike conventional imprinting, for which methylation at gDMRs is observed in all tissues, placenta-specific imprinting is associated with transient gDMRs, present only in the pre-implantation embryo and extra-embryonic lineages. To expand the list of bona fide imprinted genes subject to placenta-specific imprinting, we reinvestigated our list of candidate loci and characterized two novel imprinted genes, PIK3R1 and G0S2, both of which display polymorphic imprinting. Interrogation of placenta single-cell RNA-seq datasets, as well as cell-type methylation profiles, revealed complex cell-type specificity. We further interrogated their methylation and expression in placental samples from complicated pregnancies, but failed to identify differences between intrauterine growth restricted or pre-eclamptic samples and controls, suggesting they are not involved in these conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b750b53f10506274107b5e1972c3a2fd8cd37694" target='_blank'>
              PIK3R1 and G0S2 are human placenta-specific imprinted genes associated with germline-inherited maternal DNA methylation
              </a>
            </td>
          <td>
            Dagne Daskeviciute, Becky Sainty, Louise Chappell-Maor, Caitlin Bone, Sarah Russell, Isabel Iglesias-Platas, Philippe Arnaud, Ana Monteagudo-Sánchez, Maxim V. C. Greenberg, Keran Chen, Africa Manerao-Azua, G. Perez de Nanclares, Jon Lartey, David Monk
          </td>
          <td>2025-06-26</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The relationship between epigenetic modifications that influence gene regulation, such as histone deacetylation, microRNA regulation and DNA methylation, and human diseases has become increasingly recognised. Dysregulation of DNA methylation has been implicated in genomic imprinting and tumourigenesis. Previous studies identified several epigenetic modifier genes that directly cause genomic DNA (gDNA) methylation such as DNA methyltransferases (DNMTs).DNMTaberrations usually affect genome-wide methylation profiles, deactivating many tumour suppressor genes by methylation of their promoters. In this study, we describe a unique case with several tumours of different lineages, likely caused by a germline pathogenic DNA (cytosine-5-)-methyltransferase 3 alpha (DNMT3A)variant.We evaluated a middle-aged female who suffered from multiple metachronous and synchronous tumours, mostly benign, including subcutaneous lipomas and mesenchymal tumours, multinodular goitre, a parathyroid adenoma, a complex meningioma, a benign breast tumour, bilateral non-functioning adrenal nodules, hepatic and renal cysts and a pancreatic neuroendocrine tumour. The patient’s gDNA was subjected to whole-exome sequencing and strict bioinformatics analysis. The possible effect of the variant was investigated using multiple in-silico analysis tools and protein three-dimensional modelling.A heterozygous missenseDNMT3Avariant (c.2063G>A, p.R688H) was identified. The identified variant was Sanger sequenced in all available patient’s samples. Protein modelling strongly supports the pathogenic role of this variant in the development of the several tumours that this patient had.We report the c.2063G>ADNMT3Avariant as a potential cause of multiple unrelated tumours. This is the first report of such a case and one of a few reports ofDNMT3Agermline variants causing tumourigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/225d0c30ea3c8c06200141f21fb55e55e3264e9c" target='_blank'>
              Multiple tumours of different lineages likely related to aDNMT3Apathogenic R688H variant
              </a>
            </td>
          <td>
            Kheloud M. Alhamoudi, Meshael M. Alswailem, B. Alghamdi, Maram Alshuwaymi, Karla A. Peña-Guerra, Stefan T. Arold, H. Al-Hindi, Ali S Alzahrani
          </td>
          <td>2025-06-01</td>
          <td>BMJ Connections Clinical Genetics and Genomics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer ranks as a primary cause of mortality globally, and the study of its molecular markers and regulatory mechanisms holds paramount importance. N6-methyladenosine (m⁶A) represents the predominant modification in messenger RNA (mRNA), influencing key biological processes including RNA stability, splicing, and translation. The dynamic modulation of m⁶A modification is mediated by an array of enzymes comprising methyltransferases (“writers”), demethylases (“erasers”), and m⁶A-binding proteins (“readers”).As a pivotal member of the m⁶A “writer” family, RNA binding motif protein 15 (RBM15) facilitates the recruitment of the methyltransferase complex (MTC) to mRNA, thus orchestrating the addition of m⁶A modifications. Although prior research has underscored the critical role of m⁶A in oncogenesis, the precise mechanisms through which RBM15 operates in cancer are yet to be elucidated. This study endeavors to elucidate the structural characteristics and functional roles of RBM15, investigate its potential regulatory mechanisms across diverse tumors, uncover its distinct functions in tumor genesis, progression, and metastasis, and evaluate the therapeutic potential of targeting RBM15 in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667b10bc6d4f24effd64babb427f758acf2b1122" target='_blank'>
              Role of N6-methyladenosine methyltransferase component RBM15 in cancer progression and its therapeutic potential
              </a>
            </td>
          <td>
            Wenxiang Shen, Yulong Ma, Chunwang Yang, Shishun Yan, Kaishan Ye
          </td>
          <td>2025-05-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA binding proteins associates with harmful R-loops.
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-02</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) constitute a large class of non-coding RNAs with gene regulatory capabilities, mainly through microRNA binding, a mechanism that has been linked to cancer development. The circRNA ciRS-7 (also known as CDR1as) is an interesting candidate as it harbors over 60 binding sites for miR-7, which is known to have tumor-suppressing properties. Here, we investigated the spatial expression patterns and epigenetic regulation of ciRS-7 across nine different adenocarcinomas originating from the colon, pancreas, ovary, endometrium, breast, stomach, bile duct, lung, and prostate. The study included primary patient samples and 18 different cell lines. ciRS-7 expression was analyzed using Reverse Transcription-quantitative PCR (RT-qPCR), single molecule in situ hybridization, and Nanostring nCounter, while epigenetic modifications were examined through bisulfite sequencing, Sensitive Melting Analysis after Real Time – Methylation Specific PCR (SMART-MSP), and chromatin immunoprecipitation. The functional relevance of epigenetic modifications was examined using DNA methyltransferase and histone deacetylase inhibitors. Across all adenocarcinomas, ciRS-7 was absent in the cancer cells in most of the primary tumor specimens, except for the breast tumors, while being expressed in the tumor microenvironment (TME). In line with this, ciRS-7 was not detected in most of the cell lines. Moreover, we demonstrated that DNA methylation and H3K9 acetylation, but not H3K27 methylation, are important epigenetic modifications that impact ciRS-7 expression. In conclusion, our data show that ciRS-7 is mainly expressed in the TME and is regulated through DNA methylation and histone acetylation across all major types of adenocarcinomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd458d2d2876867a8c5e16e63beed678fb4697c" target='_blank'>
              ciRS-7 expression is epigenetically regulated in cancer cells across human adenocarcinomas
              </a>
            </td>
          <td>
            Thea Petersen Paasch, Morten T Jarlstad Olesen, J. L. García-Rodríguez, A. Assmus, Robert A Fenton, Jørgen Kjems, Henrik Hager, L. S. Kristensen
          </td>
          <td>2025-06-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bcb6bbbf4e6c64ab8d5a6fa755b5306451c07b" target='_blank'>
              EHMT1 mediates cellular motility in embryonal rhabdomyosarcoma by activating SOX8 expression.
              </a>
            </td>
          <td>
            Upasana Bajaj, Dipanwita Das, Jia Yu Leung, Nurul Fateha Binti Abdul Rashi, R. Taneja
          </td>
          <td>2025-06-04</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Histone methylation–mediated epigenetic modification significantly influences gene transcription and expression regulation. This study examined the effects of histone 3 lysine 9 trimethylation (H3K9me3) methyltransferase SUV39H1 and its specific inhibitor chaetocin on recombinant protein expression in Chinese hamster ovary (CHO) cells. Results indicated that stable SUV39H1-knockdown CHO cells exhibited reduced H3K9me3 levels while showing increased expression of recombinant adalimumab (rADM) and human serum albumin (rHSA) by approximately 45% and 136%, respectively. Furthermore, treatment with 20 nM chaetocin, a SUV39H1-specific inhibitor, enhanced expression of enhanced green fluorescent protein (EGFP), rADM, and rHSA in CHO cells. These findings demonstrate that both stable SUV39H1 knockdown and pharmacological inhibition through chaetocin effectively reduce H3K9me3 modification levels in CHO cells while significantly boosting recombinant protein production. The results strongly suggest SUV39H1’s critical regulatory role in recombinant protein expression within CHO cell systems. This research establishes a methodological foundation for developing engineered cell lines and optimizing high-efficiency CHO expression systems through cell engineering approaches. Key points •SUV39H1 knockdown boosted recombinant protein expression and decreased H3 K9 me3 levels. •Treatment with the SUV39H1-specific inhibitor chaetocin (20 nM) enhanced recombinant protein expression. •It provides a basis for developing efficient epigenetically regulated CHO expression systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18d94857c0e12ac3a5176672a07c8f4463243640" target='_blank'>
              Epigenetic regulation of histone methyltransferase SUV39H1 on the expression of recombinant protein in CHO cells
              </a>
            </td>
          <td>
            L. Yang, Miao Zhang, Jia-Liang Guo, Ming-Ming Han, Le-le Qiu, Li-Jie Gao, Liu-Su Cui, Xiao-Yin Wang, Tian-Yun Wang, Yan-Long Jia
          </td>
          <td>2025-05-10</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell cycle structures vary significantly across cell types, which exhibit distinct phase compositions. Asynchronous DNA replication and dynamic cellular characteristics during the cell cycle result in considerable heterogeneity in DNA dosage, chromatin accessibility, methylation, and expression. Nonetheless, the consequences of cell cycle disruption in the interpretation of multi-omics data remain unclear. Here, we systematically assessed the influence of distinct cell phase structures on the interpretation of omics features in proliferating cells, and proposed solutions for each omics dataset. For copy number variation (CNV) calling, asynchronous replication timing (RT) interference induces false CNVs in cells with high S-phase ratio (SPR), which are significantly decreased following replication timing domain (RTD) correction. Similar noise is observed in the chromatin accessibility data. Moreover, for DNA methylation and transcriptomic analyses, cell cycle-sorted data outperformed direct comparison in elucidating the biological features of compared cells. Additionally, we established an integrated pipeline to identify differentially expressed genes (DEGs) after cell cycle phasing. Consequently, our study demonstrated extensive cell-cycle heterogeneity, warranting consideration in future studies involving cells with diverse cell-cycle structures. RTD correction or phase-specific comparison could reduce the influence of cell cycle composition on the analysis of the differences observed between stem and differentiated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7f0e00dfe690501686ff54d8b38db35d315d13" target='_blank'>
              Mitigating Cell Cycle Effects in Multi-Omics Data: Solutions and Analytical Frameworks.
              </a>
            </td>
          <td>
            Rui Nie, Caihong Zheng, Likun Ren, Yue Teng, Yaoyu Sun, Lifei Wang, Junya Li, Jun Cai
          </td>
          <td>2025-05-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is an aggressive cancer of bone and soft tissue that predominantly affects children and young adults. A chromosomal translocation joins the low-complexity domain (LCD) of the RNA-binding protein EWS (EWSLCD) with the DNA-binding domain of Friend leukemia integration 1 (FLI1DBD), creating EWS::FLI1, a potent fusion oncoprotein essential for EwS development and responsible for over 85% of EwS tumors. EWS::FLI1 forms biomolecular condensates in vivo and promotes tumorigenesis through mediation of aberrant transcriptional changes and by interfering with the normal functions of nucleic acid-binding proteins like EWS through a dominant-negative mechanism. In particular, the expression of EWS::FLI1 in EwS directly interferes with the biological functions of EWS leading to alternate splicing events and defects in DNA-damage repair pathways. Though the EWSLCD is capable of phase separation, here we report a direct interaction between FLI1DBD and EWSLCD that enhances condensate formation and alters the physical properties of the condensate. This effect was conserved for three related E-twenty-six transformation-specific (ETS) DNA-binding domains (DBDs) while DNA binding blocked the interaction with EWSLCD and inhibited EWS::FLI1 condensate formation. NMR spectroscopy and mutagenesis studies confirmed that ETS DBDs transiently interact with EWSLCD via the ETS DBDs "wings." Together these results revealed that ETS DBDs, particularly FLI1DBD, enhance EWSLCD condensate formation and rigidity, supporting a model in which electrostatic and structural interactions drive condensate dynamics with implications for EWS::FLI1-mediated transcriptional regulation in EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90380493c2e0c6b1757985f2684e59828098025" target='_blank'>
              Phase separation of the oncogenic fusion protein EWS::FLI1 is modulated by its DNA-binding domain.
              </a>
            </td>
          <td>
            Emily E. Selig, Erich J Sohn, Aiola P. Stoja, Alma K. Moreno-Romero, Shivani Akula, Xiaoping Xu, Alexander J R Bishop, David S. Libich
          </td>
          <td>2025-05-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Karyotyping, SNP arrays, fluorescence in situ hybridisation and next generation sequencing techniques have greatly improved our understanding of the genetic drivers of acute lymphoblastic leukemia (ALL). In the past years, another layer of genetic data has been added by the study of mutational signatures, patterns of somatic mutations that represent specific mutational mechanisms. Mutational signatures can give insight in tumor development, but also reveal mutagenic side-effects of treatment in relapse samples. Multiple treatment-related mutational signatures have been detected in relapsed ALL that could play a role in therapy resistance and relapse development. In fact, multiple pathogenic driver mutations have been attributed to these treatment-related mutational processes, including a recurrent TP53 mutation. Studies in childhood ALL revealed that thiopurine exposure is the most common source of therapy-related mutagenicity in ALL and presents differently when patients are DNA mismatch repair deficient. Thiopurine-induced DNA damage indicates that leukemic cells were able to survive thiopurine exposure. This could be due to metabolic defects, acquired mutations that induce thiopurine resistance during treatment or resistance to drugs synergizing with thiopurines. In this review, we discuss the types and prevalence of treatment-related mutational signatures in ALL, and explore mechanisms of thiopurine cytotoxicity and mutagenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Merkel Cell Polyomavirus (MCPyV) is recognized as the major aetiological agent of Merkel Cell Carcinoma (MCC), an aggressive skin tumor. MCPyV-mediated oncogenesis is strictly dependent on viral integration and the expression of a truncated form of the Large T Antigen (LT). Moreover, like other oncogenic DNA viruses, MCPyV may interfere with the DNA damage response (DDR) machinery, thus promoting genomic instability and tumorigenesis. Therefore, the objective of this study was to characterize MCPyV infection in 7 MCC patients and to elucidate the plausible role of the virus in the DDR pathway. MCPyV DNA was detected in 3/7 MCC patients and, as expected, viral integration and LT truncation were observed in virus-positive MCCs, along with the expression of early genes only. Over-expression of DDR genes such as ATM, ATR and their downstream kinases Chk1 and Chk2 was reported in MCPyV-positive MCCs supporting the potential role of the virus in interfering with DDR. Our findings support the established viral aetiology of MCC, and describe, for the first time, an over-expression of DDR components in MCPyV-positive MCC, laying the basis for future studies aimed at investigating the contribution of this pathway to MCPyV-mediated carcinogenesis and exploring the plausible clinical implications of host DDR factors for the treatment of MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cee01380ef93ba0651f961c5653ef76df058128" target='_blank'>
              Investigation of mRNA expression levels of DNA damage response genes in Merkel Cell Polyomavirus-positive Merkel Cell Carcinoma: a pilot study
              </a>
            </td>
          <td>
            S. Passerini, M. Fracella, A. Ferlosio, U. Moens, C. Scagnolari, Guido Antonelli, Marco Ciotti, V. Pietropaolo
          </td>
          <td>2025-05-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, Emanuela Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND AIMS
Intrahepatic cholangiocarcinomas (ICCs) are aggressive liver tumors with high heterogeneity and limited therapeutic options. Although traditionally thought to arise from biliary cells, recent findings suggest that hepatocytes may also serve as a cellular origin for ICC. However, the mechanisms underlying hepatocyte malignant transformation and ICC initiation remain poorly understood.


APPROACH AND RESULTS
We employed oncogene-driven and chemically induced ICC murine models, along with cellular models, to recapitulate the transformation of hepatocytes into ICC. Our findings demonstrate that mature hepatocytes undergo a significant hyperpolyploid state during ICC initiation. Hyperpolyploidy promotes aberrant cell division and chromosomal instability, accelerating hepatocyte transformation and ICC onset. Furthermore, we identified the chromatin remodeling factor Arid1a as a critical suppressor of hyperpolyploidy. Arid1a deficiency disrupts mitotic machinery at the centrosome, driving hyperpolyploidization and ICC tumorigenesis.


CONCLUSIONS
Hepatocytes can transform into ICC through a process involving hyperpolyploidizaiton. This study offers new insights into the pathogenesis of ICC, particularly in patients harboring frequent ARID1A mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb35330c9db09a4f3ded3d2167db23a4e30f36f2" target='_blank'>
              Arid1a deficiency promotes hepatocyte hyperpolyploidy and drives intrahepatic cholangiocarcinoma in mice.
              </a>
            </td>
          <td>
            Qi Bian, Shu Wang, Zimin Song, Fang Liu, Muqing Cao, Nan Yang, Jun Ying, Jia-Syuan Hu, Xinyuan Xiong, Huiqin Zhu, Jun Wu, Jie Yang, Xiaonan Wang, Shunli Shen, Xuxu Sun
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) greatly contribute to the growth, invasion, metastasis and drug resistance of neuroblastoma (NB). This study aimed to construct a CAF-related prognostic model and identify the immune status of patients with NB via single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq). Methods ScRNA-seq data of NB acquired from the Gene Expression Omnibus (GEO) database were used to identify cellular subpopulations. Bulk gene expression data were downloaded from GEO, The Cancer Genome Atlas (TCGA) and ArrayExpress databases and the prognostic model was constructed using univariate Cox and Least Absolute Shrinkage and Selection Operator (LASSO) analyses. Differences in immune infiltration, therapeutic responses and signaling pathways between the high- and low-risk groups were investigated. Finally, immunohistochemistry was performed to evaluate the protein expressions and a nomogram based on the risk signature and clinical characteristics was constructed. Results ScRNA-seq data of eight NB samples were integrated to identify 253 marker genes for CAF. An eight-gene prognostic CAF-related signature was established based on the GEO data. The CAF model was strongly associated with immune infiltration, drug response and active signaling pathways in tumors. Univariate and multivariate Cox regression analyses verified that the CAF model was as an independent prognostic indicator, and a nomogram integrating the clinical signature and CAF-related risk signature was constructed for clinical prediction. Conclusions The CAF-related signature can effectively predict the prognosis of NB and provide new genomic evidence for anti-CAF immunotherapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03023-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0c7a00465eed83a4f94e4167616471e1046f423" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis establishes a cancer associated fibroblast-related signature for predicting prognosis and therapeutic responses in neuroblastoma
              </a>
            </td>
          <td>
            Zhiyao Cao, Qi Wang, Yali Han, Jianwei Lin, Qi Wu, Chencheng Xu, Jingchun Lv, Lei Zhang, Hongxiang Gao, Dapeng Jiang
          </td>
          <td>2025-07-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epitranscriptomic changes in the transcripts of cancer related genes could modulate protein levels. RNA editing, particularly A-to-I(G) editing catalyzed by ADAR1, has been implicated in cancer progression. RNA editing events in the 3’ untranslated region (3’UTR) can regulate mRNA stability, localization, and translation, underscoring the importance of exploring their impact in cancer. Here, we performed an in silico analysis to detect breast cancer enriched RNA editing sites using the TCGA breast cancer RNA-seq dataset. Notably, the majority of differential editing events mapped to 3’ untranslated regions (3’UTRs). We confirmed A-to-I(G) editing in the 3’UTRs of MDM2 (Mouse Double Minute 2 homolog), GINS1 (GINS Complex Subunit 1), and F11R (Junctional Adhesion Molecule A) in breast cancer cells. RNA immunoprecipitation with ADAR1 antibody confirmed the interaction between ADAR1 and MDM2, GINS1, and F11R 3’UTRs. ADAR1 knockdown revealed decreased editing levels, establishing ADAR1 as the editing enzyme. A reporter assay for MDM2, an oncogene overexpressed mostly in luminal breast cancers, demonstrated that RNA editing enhances protein expression, in agreement with reduced MDM2 protein levels in ADAR1 knockdown cells. Further exploration into the mechanisms of 3’UTR editing events revealed an interaction between ADAR1 and CSTF2, a core component of the polyadenylation machinery, as identified through biotin-based proximity labeling mass spectroscopy, and co-immunoprecipitation experiments. Furthermore, CSTF2 knockdown reduced both ADAR1 and MDM2 protein levels. Our findings highlight implications for MDM2 regulation by ADAR1-dependent 3’UTR RNA editing and present an interplay between RNA editing on 3’UTRs and the mRNA polyadenylation machinery. These results improve our understanding of ADAR1’s role in cancer-associated 3’ UTR RNA editing and its potential as a therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01611-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdc55a1fd09d07d58250cf9a1734cfe432cac703" target='_blank'>
              3’UTR RNA editing driven by ADAR1 modulates MDM2 expression in breast cancer cells
              </a>
            </td>
          <td>
            Elanur Almeric, Deniz Karagozoglu, Mustafa Cicek, Didem Naz Dioken, Huseyin Avni Tac, Esra Çiçek, Busra Aytul Kirim, Irmak Gurcuoglu, O. U. Sezerman, Nurhan Ozlu, A. E. Erson-Bensan
          </td>
          <td>2025-05-17</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Temple syndrome is an imprinting disorder resulting from abnormal genomic or epigenomic aberrations of chromosome 14 including maternal uniparental disomy (matUPD), paternal deletion of 14q32, or aberrant methylation of the imprinting control regions at 14q32. Understanding the underlying molecular mechanism is essential to understanding the recurrence risk and physical effects. Currently, diagnosis requires the detection of aberrant methylation and copy number loss via methylation-sensitive assays such as methylation-specific multiplex ligation-dependent probe amplification, and short tandem repeat analysis to detect matUPD and the presence of epimutation. Therefore, a one-step approach that can detect aberrant methylation and underlying genetic mechanisms would be of high clinical value. Here we use nanopore sequencing to delineate the molecular diagnosis of a case with Temple syndrome. We demonstrate the application of nanopore sequencing to detect aberrant methylation and underlying genetic mechanisms simultaneously in this case, thus providing a proof of concept for a one-step approach for molecular diagnosis of this disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c13f17671e59c8d86a86517592417eb48513c1" target='_blank'>
              Using long-read sequencing to detect and subtype a case with Temple syndrome
              </a>
            </td>
          <td>
            Sarah Dada, Vahid Akbari, Duha Hejla, Yaoqing Shen, Katherine Dixon, S. Choufani, Rosanna Weksberg, Cornelius F Boerkoel, Laura Stewart, Kamilla Schlade-Bartusiak, Emma Strong, Danya Fox, Daniel Gamu, William T. Gibson, Steven J M Jones
          </td>
          <td>2025-05-29</td>
          <td>Journal of Medical Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Aim: Chemotherapy resistance, often linked to the development of resistance against genotoxic agents, is a major obstacle in cancer treatment. FoxM1, a transcription factor frequently overexpressed in malignancies such as breast cancer, is strongly associated with genotoxic therapy resistance. The aim of this study is to conduct a comparative analysis of the effects of thiostrepton (THIO), a FoxM1 inhibitor, on the DNA damage response in HUVEC cells (non-malignant) and MDA-MB-231 breast cancer cells (malignant) 
Materials and Methods: THIO's impact on cell viability were evaluated in both cell lines using the MTT assay. Oxidative DNA damage levels were measured with the 8-OHdG kit, and apoptosis was assessed using the Caspase 3 ELISA kit. The expression levels of DNA damage response genes (BRCA-1, DNAPKC, FOXM1, RAD51, MRE11 and XRCC1) were analyzed by RT-PCR. 
Results: MDA-MB-231 cells exhibited greater sensitivity to the cytotoxic effects of THIO than HUVEC cells. In HUVEC cells, THIO caused a significant increase in oxidative DNA damage, whereas no such effect was observed in MDA-MB-231 cell lines. Conversely, breast cancer cells showed a significant increase in Caspase 3 levels. RT-PCR results revealed a marked downregulation of DNA damage response genes, particularly BRCA-1, DNAPKC, MRE11, FOXM1, and XRCC1, in both cell types. 
Conclusion: THIO downregulates DNA damage response genes by inhibiting FoxM1 expression in both malignant and non-malignant cells. This effect suggests that THIO may enhance the sensitivity of breast cancer cells to therapy by disrupting DNA repair pathways, presenting potential for overcoming chemotherapy resistance. Further studies are required to validate these findings and explore THIO's therapeutic utility in breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0302ae4a15025cfb10555437aa0808dee744207" target='_blank'>
              FoxM1 inhibition by Thiostrepton downregulates DNA damage response genes, enhancing sensitivity of breast cancer cells to therapy
              </a>
            </td>
          <td>
            F. Demırtaş Korkmaz, Zekeriya Düzgün, Fadime Mutlu İçduygu
          </td>
          <td>2025-06-10</td>
          <td>Ege Tıp Dergisi</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ad49026046d30dd6909e4b9cd5b26991a13bef" target='_blank'>
              Tumor aneuploidy as a prognostic and predictive biomarker in immune checkpoint blockade.
              </a>
            </td>
          <td>
            Daniel Huang, L. F. Spurr, R. Weichselbaum, S. Pitroda
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Loss of the Y chromosome (LOY) in peripheral blood mononuclear cells (PBMCs) is the most common somatic alteration in men and is associated with higher mortality from epithelial cancers1–3. In tumours, epithelial LOY is also associated with poor survival4–7. This raises several fundamental questions, such as why LOY in PBMCs drives cancer mortality and whether there is a relationship between LOY in PBMCs, PBMC-derived immune cells and cancer cells (and, if so, what its consequences are). We sought to answer these questions through a comprehensive pan-cancer analysis of bulk and single-cell RNA sequencing data from 29 human tumour types, along with autochthonous and syngeneic mouse models. In human and mouse tumours, malignant epithelial cells had the highest LOY prevalence, yet LOY was also present in tumour stromal and immune cells, with LOY in malignant epithelial cells predicting LOY in benign cells. LOY also correlated between paired tumour and PBMC samples from patients. Among benign cells, LOY induced the strongest shift in CD4+ and CD8+ T cells, with both showing transcriptomic signatures of immunosuppression. Furthermore, the magnitude of LOY in epithelial cells, CD4+ T cells and CD8+ T cells independently predicts survival, with tumours exhibiting concurrent epithelial and T cell LOY having the worst outcomes. Here we establish a model that links LOY in immune cells to LOY in malignant cells, which may explain in part why LOY in PBMCs is associated with increased cancer mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/521330e6a0acc4c997107eda98c17326ccf4fe70" target='_blank'>
              Concurrent loss of the Y chromosome in cancer and T cells impacts outcome
              </a>
            </td>
          <td>
            Xingyu Chen, Yiling Shen, Suhyeon Choi, Hany A Abdel-Hafiz, Mukta Basu, Lena Hoelzen, Martina Tufano, Saravana Kumar Kailasam Mani, Maryam Ranjpour, Jiani Zhu, V. K. Ramanujan, E. Koltsova, Vinicius F Calsavara, S. Knott, Dan Theodorescu
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background EpCAM (epithelial cell adhesion molecule) is a key regulator of epithelial cell–cell adhesion, signal transduction, tissue regeneration, and serves as a stem cell marker. It is frequently overexpressed in epithelial cancers and is linked to tumor progression, survival, and metastasis. However, the functional impact of EpCAM mutations in cancer remains poorly understood. Methods To investigate the role of EpCAM mutations, we performed a comprehensive analysis of cancer cohorts from multiple genomic datasets, identifying novel somatic EpCAM mutations across diverse epithelial cancers. Using bioinformatics tools (SIFT, PolyPhen-2, Mutation Assessor) and molecular modeling, we assessed the potential impact of these mutations. Further, homology modeling and all-atom molecular dynamics (MD) simulations were conducted to evaluate structural changes. From an analysis of 300 studies comprising 300,300 cancer samples, we identified 160 recurrent somatic mutations across epithelial cancers. Of these, seven mutations most frequently associated with lung cancer were further validated through molecular dynamics simulations, evaluation of ERK signaling activity, and assessment of sensitivity to the MEK inhibitor Trametinib. Results Our findings revealed that cancer-associated mutations, particularly in the TY-1 and RCD regions, induce structural instability in EpCAM, leading to altered functional properties. Patient cohort analyses indicated that EpCAM mutations correlate with reduced survival rates in colon and hepatocellular carcinoma and contribute to early tumor progression in lung cancer. Moreover, introducing these mutations into lung cancer cells enhanced their sensitivity to MEK inhibitors, suggesting a potential therapeutic vulnerability. Conclusion This study provides novel insights into the structural and functional consequences of EpCAM mutations in cancer, demonstrating their association with reduced survival, tumor progression, and drug sensitivity. These findings highlight EpCAM as a promising therapeutic target in epithelial cancers. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14455-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593d8c16237b699357d475f55f406f3effbf3755" target='_blank'>
              Landscape of cancer associated EpCAM mutations: molecular modeling, predictive insights and impact on patient survival
              </a>
            </td>
          <td>
            P. S. Dhotare, Audrey C. Bochi-Layec, Timothy P. Fleming, William E Gillanders, Ross M. Bremner, K. Sonawane, N. Sankpal
          </td>
          <td>2025-07-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In vitro hematopoiesis systems can be used to define mechanisms for blood cell formation and function, produce cell therapeutics, and model blood cell contributions to systemic disease. Hematopoietic progenitor cell (HPC) production remains inefficient, precluded by knowledge gaps related to specification and morphogenesis of specialized hemogenic endothelial cells, which undergo an endothelial-to-hematopoietic transition (EHT) to form HPCs. We elected to define changes in gene expression and chromatin organization during HPC formation to reveal regulatory mechanisms. Using paired single cell RNA/ATAC sequencing together with Hi-C, we profiled cells before and after EHT. Pathway analysis and pseudotime inferences confirmed a continuum of stromal and endothelial cells undergoing development into HE cells and lineage-based HPCs in vitro. In these cell types, we characterize cis-regulatory elements and transcriptional regulatory activities that facilitate EHT and HPC homeostasis, including for SNAI1, SOX17, TGFβ, STAT4, as well as for GFI1b and KLF1 in megakaryocyte- and erythroid-biased progenitors, respectively. We then leveraged our insights into chromatin organization among in vitro-derived cells to assess enrichments corresponding to human trait variation reported in human genome wide association studies. HPCs revealed locus enrichment for quantitative blood traits and autoimmune disease predisposition, which were particularly enriched in myeloid- and lymphoid-biased populations. Stromal and endothelial cells from our in vitro cultures were specifically enriched for accessible chromatin at blood pressure loci. Our findings reveal genes and mechanisms governing in vitro hematopoietic development and blood cell-related disease pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4774749a5a1117ef1a36110f9d8a3ed3849f1988" target='_blank'>
              Multimodal analysis of in vitro hematopoiesis reveals blood cell-specific genetic impacts on complex disease traits
              </a>
            </td>
          <td>
            Rong Qiu, Khanh B. Trang, Carson Shalaby, J. Pippin, James Garifallou, Struan F. A. Grant, Christopher S. Thom
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 What is the prevalence of mitotic-origin aneuploidies in human blastocysts using a PGT platform capable of identifying cell division origin of aneuploidy? A single-center experience



 Although uncommon, some embryos are diagnosed with aneuploidies of mitotic origin only. Diagnosing aneuploidies without determining their cellular origin risks discarding embryos with potential viability.



 Chromosomal abnormalities in human embryos are a significant factor contributing to reproductive failure. Next-generation sequencing (NGS) platforms are widely accepted for evaluating chromosomal abnormalities, as they are copy number-focused technologies. However, in some cases, biopsies reveal a mix of normal and abnormal cells, referred to as mosaic embryos. The developmental potential of these mosaic embryos remains unclear, although successful outcomes, including normal live births, have been reported. Moreover, novel PGT approaches, such as high-density single nucleotide polymorphism (SNP) arrays, offer the capability to predict the cell division origin of aneuploidies, moving beyond a sole focus on chromosome copy numbers.



 This retrospective study analyzed 310 consecutive PGT-A biopsies obtained from 72 patients across 91 treatment cycles conducted between May 2023 and December 2024 at a private fertility clinic. Both autologous and donor cycles were included, and all blastocysts were evaluated using trophectoderm (TE) biopsy and DNA samples from the biological parents.



 Trophectoderm biopsies were obtained, with 5–10 cells collected for each sample. Parental DNA samples were also required to serve as reference material for the tested embryos. Trophectoderm biopsies and parental DNA were processed using SNP arrays to identify the cell division origin of aneuploidy, integrating both genotyping and copy-number data. Aneuploidies were classified by origin as maternal meiotic (from the oocyte), paternal meiotic (from the sperm), or mitotic (embryonic, arising post-zygotic division).



 Among the total embryos studied (n = 310), 44.8% (139/310) were identified as aneuploid. Of these, 34.8% (108/310) exhibited at least one maternal meiotic aneuploidy, with an average maternal age of 36.3 ± 4.4 years, consistent with the well-established correlation between advancing maternal age and chromosomal anomalies. Additionally, 7.7% (24/310) of the aneuploid embryos displayed paternal meiotic errors, while 8.7% (27/310) presented mitotic errors originating post-fertilization. Notably, 5.16% (16/310) of the embryos displayed exclusively mitotic-origin aneuploidies, underscoring the importance of mitotic anomalies in early embryonic development. Among the 139 abnormal embryos, 81.3% (113/139) exhibited whole chromosome aneuploidies, consistent with meiotic segregation errors involving entire chromosomes. Conversely, 13.67% (19/139) of the abnormal embryos exhibited segmental aneuploidies, involving duplications or deletions of specific chromosomal regions. A small subset of embryos, accounting for 5.04% (7/139), displayed karyotypes that included both whole chromosome aneuploidies and segmental abnormalities, reflecting the complexity of chromosomal errors in human embryos.



 This is a retrospective study with a relatively small number of embryos analyzed. Expanding the cohort of blastocysts and evaluating the clinical implications of aneuploidy origin prediction are necessary. Additionally, further research is needed to assess the reproductive potential of embryos with segmental chromosome aneuploidies.



 Identifying the cell division origin of aneuploidies provides deeper insights into embryonic chromosomal abnormalities and their developmental potential. This approach could improve clinical decision-making in assisted reproduction, enhancing embryo selection strategies and informing future studies on the implications of mitotic and segmental aneuploidies in human reproduction.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/839a948ebef49cd1b00902c78815394f99d075d1" target='_blank'>
              P-242 From Cell Division to Clinical Decisions: Unveiling the True Prevalence of Mitotic-Origin Aneuploidies in Preimplantation Embryos
              </a>
            </td>
          <td>
            M. J. Gomez Cuesta, D. Marin Chavarria, M. I. Gutierrez Estrada, M. Garcia Galicia, G. D. Cortes Ruiz, M. Alonso de Mendieta, C. Kandel Varsano, C. Lopez Valencia, E. Cervantes Bravo
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gliomas are the most common and lethal malignant primary brain tumors in adults, associated with the highest number of years of potential life lost. The latest WHO classification for central nervous system tumors highlighted the need for new biomarkers of diagnosis, prognosis, and response to therapy. The PI3K/Akt signaling pathway is clearly implicated in tumorigenesis, being one of the most frequently altered pathways in cancer. Activating PI3KCA mutations are oncogenic and can influence both prognosis and treatment response in various tumor types. In gliomas, however, studies have reported inconsistent PIK3CA mutational frequencies, ranging from 0% to 30%. Furthermore, the impact of these alterations on glioma diagnosis, prognosis, and therapy response remains unclear. Current evidence suggests that PIK3CA mutations may represent early and constitutive events in glioma development, associated with worse glioblastoma prognoses, earlier recurrences, and widespread disease. Among these, the hotspot mutation H1047R has been particularly associated with a more aggressive phenotype while also modulating the neuronal microenvironment. In this review, we examine the clinical relevance of PIK3CA mutations across different cancers, with a particular focus on their emerging role in glioma. Moreover, we also discuss the therapeutic potential and challenges of targeting PIK3CA mutations in the context of glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6d6f1fbe9d06843c9afa611b81f785472f82b9" target='_blank'>
              PIK3CA Mutations: Are They a Relevant Target in Adult Diffuse Gliomas?
              </a>
            </td>
          <td>
            A. Tomás, M. Pojo
          </td>
          <td>2025-05-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, Reva K Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Numerous long noncoding RNAs (lncRNAs) are generated in response to external stimuli, but the scope and functions of such activity are not known. Here, this study provides insight into how the transcription of lncRNAs is connected to DNA damage response by identifying the lncRNA ZFAS1, which is required for cell cycle arrest, transcription regulation, and DNA repair. Mechanistically, ZFAS1 facilitates dynamic changes in hyperphosphorylated forms of the large subunit of RNA polymerase II (RNAPII) around transcription initiation sites by directly targeting the regulated genes. It is shown that extensive transcription shutdown and concomitant stimulated engagement of RNAPII-Ser2P are crucial for repair and cell survival upon genotoxic stress. Finally, ZFAS1 knockout in mice dampened nucleotide excision repair (NER) and led to kidney dysplasia. Overall, the findings extend the understanding of lncRNAs in DNA damage repair (DDR) and imply a protective role of lncRNA against DDR-deficient developmental disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd71da5d91ac1bdf35ea5bcddbb6ad4cb3db5ea" target='_blank'>
              IncRNA-ZFAS1, an Emerging Gate-Keeper in DNA Damage-Dependent Transcriptional Regulation.
              </a>
            </td>
          <td>
            Jiena Liu, Qing Lu, Zixuan Fan, Jiahui Lin, Nan He, Xin Zhang, Zhaoya Han, Tingting Zhu, Zhenzhen Wu, Yingying Xu, Yuming Wang
          </td>
          <td>2025-05-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The distribution of m6A across various RNA isoforms and its heterogeneity within single cells are still not well understood. Here, we develop m6A-isoSC-seq, which employs both Oxford Nanopore long-read and Illumina short-read sequencing on the same 10x Genomics single-cell cDNA library with APOBEC1-YTH induced C-to-U mutations near m6A sites. Through m6A-isoSC-seq on a pooled sample of three cell line origins, we unveil a profound degree of m6A heterogeneity at both the isoform and single-cell levels. Through comparisons across single cells, we identify widespread specific m6A methylation on certain RNA isoforms, usually those misprocessed RNA isoforms. Compared to the coding isoforms of the same genes, the expression of highly methylated misprocessed RNA isoforms is more sensitive to METTL3 depletion. These misprocessed RNAs tend to have excessive m6A sites in coding regions, which are targets of CDS-m6A decay (CMD). This study offers undocumented insights into the role of m6A in RNA surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3d1afb6146144c07522f09b53199013967becd6" target='_blank'>
              Isoform characterization of m6A in single cells identifies its role in RNA surveillance
              </a>
            </td>
          <td>
            Zhijun Ren, Jialiang He, Xiang Huang, Yan Gao, Chuanchuan Wei, Zehong Wu, Wenbing Guo, Feng Wang, Qingquan Zhao, Xiang Sun, Jie Zhang, Nan Cao, Lan Lin, Jinkai Wang, Yixian Cun
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 8624


 Background:
 Lung adenocarcinoma (LUAD), the most common lung cancer subtype, has poor survival rates and limited treatment options. Among its molecular drivers, loss of methylthioadenosine phosphorylase (MTAP) is linked to aggressive development and poor outcomes, yet its downstream effects remain unclear. MTAP is essential for purine biosynthesis and S-adenosyl methionine (SAM) production, a key methyl donor for epigenetic regulation. MTAP-loss leads to methylthioadenosine (MTA) accumulation, which inhibits methyltransferase activity and disrupts epigenetic control. However, the impact of MTAP deficiency on transcriptional and metabolic programming in LUAD is still largely unknown, highlighting a critical gap for targeted therapy. This study examines these molecular consequences to identify potential therapeutic vulnerabilities in MTAP-deficient LUAD.
 Methods:
 RNA-sequencing data for 510 LUAD samples were obtained from The Cancer Genome Atlas (TCGA) via cBioPortal. Samples were classified as MTAP-loss (n=64) or MTAP-normal (n=446) based on mutational data. Differential gene expression was performed using the DESeq2 package for R and the Benjamini-Hochberg procedure was used to control for false discovery rate. DNA methylation (Illumina Human Methylation 450k) was compared between MTAP-loss (n=46) and MTAP-normal (n=404) cohorts. Significantly altered probes were mapped to differentially expressed genes (DEGs). Pathway and gene ontology (GO) analyses were conducted using KEGG, GO, and EnrichR to identify dysregulated pathways.
 Results:
 DNA methylation analysis revealed 581 hypomethylated and only 51 hypermethylated probes in MTAP-loss samples, underscoring a global hypomethylation phenotype likely driven by MTA accumulation. We identified 343 differentially expressed, hypomethylated genes in MTAP-loss LUAD samples. Of these, upregulated genes were highly enriched in mitochondrial function and stress response pathways, whereas downregulated genes were linked to cell differentiation and developmental processes, suggesting an epigenetically driven metabolic reprogramming. Notably, these alterations may confer heightened cellular survival and adaptability under stress, while curtailing normal differentiation programs.
 Conclusions:
 Our findings indicate that MTAP loss in LUAD leads to a coordinated shift in DNA methylation and gene expression, promoting survival-focused metabolic and stress responses at the expense of normal regulatory pathways. These results highlight novel vulnerabilities in MTAP-deficient tumors and suggest potential targets for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bac3c5fbe15b9c30b43fa429b943e30d1102b5e9" target='_blank'>
              Epigenetic and transcriptional consequences of MTAP-loss in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Swati Pothukuchi, Jihao Xu, Ssu-Wei Hsu, Ching-Hsien Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA damage is a serious threat to cellular viability, and it is implicated as the major cause of normal ageing. Hence, targeting DNA damage therapeutically may counteract age-related cellular dysfunction and disease, such as neurodegenerative conditions and cancer. Identifying novel DNA repair mechanisms therefore reveals new therapeutic interventions for multiple human diseases. In neurons, non-homologous end-joining (NHEJ) is the only mechanism available to repair double-stranded DNA breaks (DSB), which is much more error prone than other DNA repair processes. However, there are no therapeutic interventions to enhance DNA repair in diseases affecting neurons. NHEJ is also a useful target for DNA repair-based cancer therapies to selectively kill tumour cells. Protein disulphide isomerase (PDI) participates in many diseases, but its roles in these conditions remain poorly defined. PDI exhibits both chaperone and redox-dependent oxidoreductase activity, and while primarily localised in the endoplasmic reticulum it has also been detected in other cellular locations. We describe here a novel role for PDI in DSB repair following at least two types of DNA damage. PDI functions in NHEJ, and following DNA damage, it relocates to the nucleus, where it co-localises with critical DSB repair proteins at DNA damage foci. A redox-inactive mutant of PDI lacking its two active site cysteine residues was not protective, however. Hence, the redox activity of PDI mediates DNA repair, highlighting these cysteines as targets for therapeutic intervention. The therapeutic potential of PDI was also confirmed by its protective activity in a whole organism against DNA damage induced in vivo in zebrafish. Hence, harnessing the redox function of PDI has potential as a novel therapeutic target against DSB DNA damage relevant to several human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c93ace9388c019ac0f5bc12962ca74d85497c2d9" target='_blank'>
              The Redox Activity of Protein Disulphide Isomerase Functions in Non-Homologous End-Joining Repair to Prevent DNA Damage.
              </a>
            </td>
          <td>
            S. Shadfar, Fabiha Farzana, S. Saravanabavan, Ashley M Rozario, M. Vidal, Cyril J Jagaraj, S. Parakh, Esmeralda Paric, Kristy C Yuan, Mariana Brocardo, Donna R Whelan, Angela S Laird, J. Atkin
          </td>
          <td>2025-05-15</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) have a unique molecular profile compared to extracranial metastases (ECM). Description of the biological features and clinical outcomes of MBM will facilitate the design of rational therapies. Methods We examined the mutational landscape and gene expression profiles of MBM (74 patients) and ECM (34 patients) in paired patient samples from a previously published dataset with whole-exome sequencing (WES) and RNA sequencing (RNAseq) data from MD Anderson Cancer Center (MDACC). We also present findings from MBM from a new cohort of 14 patients from Duke University to strengthen investigation of somatic mutations and gene expression profiles. Gene Set Enrichment Analysis (GSEA) was used to compare paired MBM versus lymph node (LN) metastases and skin metastases. Relative immune cell abundance was inferred using deconvolution methods. Survival outcomes from craniotomy and associations with biological features, BRAF mutation status, and PTEN expression were assessed. Results GSEA found that autophagy signaling pathways are enriched in MBM versus LN and skin metastases. BRAF was the most frequently mutated clinically relevant gene in MBM and ECM, with NRAS and PTEN also frequently altered in MBM. The most strongly upregulated genes in autophagy pathways were glial fibrillary acidic protein (GFAP) and hemoglobin beta (HBB). An increased proportion of immune-suppressive M2 compared to tumor-suppressive M1 macrophages in MBM and ECM was identified. There was not sufficient evidence for an association between BRAF V600 mutation status or expression and overall survival (OS) from craniotomy. Conclusions The mutational landscape and gene expression of MBM from the Duke cohort resembled those previously reported in the MDACC cohort. Upregulation of autophagy pathways was observed in patient-matched MBM versus LN and skin metastases due to upregulation of two genes, GFAP and HBB. In MBM, higher M2:M1 ratio may contribute to a therapeutically relevant immune-suppressive tumor microenvironment (TME). Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02035-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2acbbd5230dc401bd3ec3c002ad4d3ee572431" target='_blank'>
              A comprehensive, multi-center, immunogenomic analysis of melanoma brain metastases
              </a>
            </td>
          <td>
            L. B. Kennedy, Amanda E. D. Van Swearingen, Marissa R Lee, Layne W. Rogers, A. Sibley, Jeff Sheng, Dadong Zhang, Xiaodi Qin, E.S. Lipp, Swaminathan Kumar, Aron Y Joon, Pixu Shi, Michael A. Davies, K. Owzar, Carey K. Anders, April K S Salama
          </td>
          <td>2025-06-02</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Recent studies suggest that tail regeneration in lizards begins with a tumor-like stage usually termed regenerative blastema. Oncogenes and tumor suppressors are activated in blastema cells, resulting in a balanced cell proliferation that does not turn the blastema into a tumor. This outgrowth elongates forming new tissues and tail. We previously showed that physiological extracts from regenerating lizard tissues inhibit the growth of cancer cells in vitro within 2-4 days of administration, demonstrating that the growing lizard blastema contains regulatory molecules which can also influence human cancer cells. The molecules responsible for this inhibition were not identified in that initial study. In the present experimental study, after specific extractions of RNAs and/or proteins from the regenerating tail of lizard, we have confirmed the inhibition of breast cancer cell vitality in vitro within 2-3 days from their addition to the culture medium. Proteolysis or heat denaturation of proteins abolished the inhibitory effect. RNA delivered to breast cancer cells in vitro through lipid vesicles (liposomes) showed the highest inhibition of cancer cells vitality. Cell degeneration, detected by microscopy, revealed that RNA is more effective than proteins extracted from regenerating tissues. The present observations further suggest that RNAs coding for known tumor suppressor proteins, and non-coding RNAs that are highly expressed in the regenerating tail, may be key inhibitors (tumor suppressors) of blastema and cancer cell proliferation. The evolution of a mechanism for the self-remission of tumor growth in lizards remains uncertain, but continuing study of this reptile may help uncover natural mechanisms for tumor growth inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbbc4431106b7f4d802c168046ca986e622ee0cd" target='_blank'>
              RNA and proteins extracted from the regenerating tail of lizards determine inhibition of cancer cell proliferation.
              </a>
            </td>
          <td>
            Nicola Greco, Maurizio Onisto, Lorenzo Alibardi
          </td>
          <td>2025-06-13</td>
          <td>The International journal of developmental biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases (CLKs) serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type (mct) tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. The full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a5712679744f74f4987e0de1a42d21bdcab518c" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas E. Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H Doroshow
          </td>
          <td>2025-05-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DNA methylation is essential for transcriptional regulation and the maintenance of chromosome stability, and its precise inheritance upon DNA replication is indispensable for cellular homeostasis. The DNMT1/UHRF1 complex is critical in copying DNA methylation with accessory proteins, including CDCA7 and HELLS. The DNMT1/UHRF1 complex is also crucial for maintaining DNA methylation at imprinting control regions during preimplantation development against genome-wide DNA demethylation, an essential process for early embryos to acquire totipotency. Pathogenic variants in the genes involved in the mechanism of DNA methylation maintenance result in immunodeficiency, centromeric instability, and facial anomalies (ICF) syndrome, multilocus imprinting disturbance (MLID), autosomal dominant cerebellar ataxia-deafness and narcolepsy (ADCADN), neuropathy, hereditary sensory, type 1E (HSN1E), Kleefstra syndrome 1 (KLEFS1), and immunodeficiency 96 (IMD96). This review discusses recent progress in understanding the possible molecular pathogenesis of these diseases, with a particular focus on recent progress in understanding ICF syndrome and MLID.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0311b04391f3ff48779575d6888611f38c56b5" target='_blank'>
              Congenital diseases with defects in DNA methylation maintenance: Focusing on ICF syndrome and multilocus imprinting disturbance.
              </a>
            </td>
          <td>
            Motoko Unoki
          </td>
          <td>2025-06-12</td>
          <td>Genes & genetic systems</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Allelic gene-specific regulatory events are crucial mechanisms in organisms, pivotal to many fundamental biological processes such as embryonic development and chromosome inactivation. Allelic gene imbalance manifests at both RNA expression and epigenetic levels. Recent research has unveiled allelic-specific regulation of RNA N6-methyladenosine (m6A), emphasizing the need for its precise identification. However, prevailing approaches primarily focus on screening allele-specific genetic variations associated with m6A, but not truly identify allelic m6A events. Therefore, the construction of a novel algorithm dedicated to identifying allele-specific m6A (ASm6A) signals is still necessary for comprehensively understanding the regulatory mechanism of ASm6A. Findings To address this limitation, we have developed a meta-analysis approach using hierarchical Bayesian models to accurately detect ASm6A events at the peak level from MeRIP-seq data. For user convenience, we introduce a unified analysis pipeline named M6Allele, streamlining the assessment of significant ASm6A across single and paired samples. Applying M6Allele to MeRIP-seq data analysis of pulmonary fibrosis and lung adenocarcinoma reveals enrichment of ASm6A events in key regulatory genes associated with these diseases, suggesting their potential involvement in disease regulation. Conclusions Our effort provides a method for precisely identifying ASm6A events at the peak level, elucidates the interplay of m6A with human health and disease genetics, and paves a new visual angle for disease research. The M6Allele software is freely available at https://github.com/RenLabBioinformatics/M6Allele under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a8b2fc14a09819d025933f902cc0922e8465cf" target='_blank'>
              M6Allele: a toolkit for detection of allele-specific RNA N6-methyladenosine modifications
              </a>
            </td>
          <td>
            Yin Zhang, Lin Tang, Shengyao Zhi, Bosu Hu, Zhixiang Zuo, Jian Ren, Yubin Xie, Xiaotong Luo
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CHROMR is a primate-specific long noncoding RNA with emerging roles in homeostasis and pathophysiology. Elevated blood levels of CHROMR have been observed in patients with cardiovascular disease and several cancers, where it is correlated with poor clinical outcomes. Like many lncRNAs, CHROMR accumulates in both the nucleus and the cytoplasm, and it assumes distinct functions in each of these cellular compartments. In the nucleus, CHROMR sequesters a transcriptional repressor complex to activate interferon-stimulated gene expression and antiviral immunity. In the cytoplasm, CHROMR competitively inhibits microRNAs involved in cholesterol efflux and cell cycle regulation, thereby impacting gene pathways involved in reverse cholesterol transport, HDL biogenesis, and tumor growth. In this review, we detail the multifaceted functions of CHROMR in cholesterol metabolism, innate immunity, and cancer progression. We also explore the potential molecular mechanisms that govern its expression and dynamic subcellular localization, which may be key to its functional versatility. Advancing our understanding of the regulatory networks and cellular environments that shape CHROMR activity will be critical for assessing its promise as a therapeutic target and diagnostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e434fd00ac8710c07c05ad122a8af40ef9af54ab" target='_blank'>
              The Multifaceted Roles of CHROMR in Innate Immunity, Cancer, and Cholesterol Homeostasis
              </a>
            </td>
          <td>
            Emma R. Blaustein, Coen van Solingen
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract As the most abundant internal modification on mRNAs, N6-methyladenosine (m6A) has been discovered to be involved in different biological processes. Mostly determined by m6A methyl-transferases (m6A writers) and demethylases (m6A erasers), different cell types possess differential m6A profiles of transcriptomes. However, the interpretation of the m6A-encoded epitranscriptomic information needs m6A readers to bind and recruit different machinery for regulating the target mRNAs, which in turn, may regulate cell fates. The functions of the m6A readers in the regulation of the maintenance and differentiation of human embryonic stem cells (hESCs) remain largely unknown. In this study, we deleted the whole genomic region of the m6A reader YTHDF2 and discovered that YTHDF2 is dispensable for the maintenance, but important for the differentiation of hESCs, especially for the differentiation towards ectoderm. Furthermore, we identified the m6A-modified ROBO1 mRNAs as potential targets by YTHDF2 in regulating hESC to neuroectoderm differentiation. This study reveals the potential roles of the m6A reader YTHDF2 in regulating the specification of neuroectodermal cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9de0dadb60bdb47e4e4e901d35dbdaddd88a733" target='_blank'>
              Transcriptomic analysis of the m6A reader YTHDF2 in the maintenance and differentiation of human embryonic stem cells
              </a>
            </td>
          <td>
            Boshi Feng, Yanxi Chen, Huanchang Tu, Jin Zhang, Lingling Tong, Xiaohan Lyu, Aaron Trent Irving, Di Chen
          </td>
          <td>2025-05-26</td>
          <td>Stem Cells</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Mutational signatures of single-base substitutions (SBSs) characterize somatic mutation processes which contribute to cancer development and progression. However, current mutational signatures do not distinguish the two independent steps that generate SBSs: the initial DNA damage followed by erroneous repair. To address this modelling gap we developed DAMUTA, a hierarchical Bayesian probabilistic model that infers separate signatures for each process, and captures their sample-specific interaction. We applied DAMUTA to 18,974 pan-cancer whole genome sequencing mutation catalogues from 23 cancer types and show that tissue-specificity in mutation patterns is driven largely by variability in damage processes. We also show that misrepair processes are predictive of DNA damage response deficiencies. Unlike existing approaches, DAMUTA distinguishes damage from misrepair contributions, and we demonstrate significant improvements over a mutational-burden baseline or signatures from the COSMIC database. Our analysis reveals a shared pan-cancer pattern of early clonal transition-mutations which shifts to a more uniform substitution pattern consistent with increased reliance on translesion synthesis for damage tolerance. DAMUTA thus generates a compact set of signatures which resolves redundancies of current signature models, disentangles the effects of DNA damage and misrepair processes, and facilitates improved stratification of tumours, while providing a framework towards a unified pan-cancer model of the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3964588c1648b53d4e30a576d480451079a470ad" target='_blank'>
              Damage and Misrepair Signatures: Compact Representations of Pan-cancer Mutational Processes
              </a>
            </td>
          <td>
            Caitlin F. Harrigan, Kieran R. Campbell, Quaid Morris, Tyler Funnell
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cell spheroids autonomously form in the ascites fluid and are considered a conduit for epithelial ovarian cancer metastasis within the peritoneal cavity. Spheroids are homotypic, avascular 3D structures that acquire resistance to anoikis to remain viable after cellular detachment. We used in vitro spheroid model systems to interrogate pathways critical for spheroid cell proliferation, distinct from those driving monolayer cancer cell proliferation. Using the 105C and KOC-7c human ovarian clear cell carcinoma (OCCC) cell lines, which have distinct proliferative phenotypes as spheroids but the same prototypical OCCC gene mutation profile of constitutively activated AKT signaling with the loss of ARID1A, we revealed therapeutic targets that efficiently kill cells in spheroids. RNA-seq analyses compared the transcriptome of 3-day monolayer and spheroid cells from these lines and identified the characteristics of dormant spheroid cell survival, which included the G2/M checkpoint, autophagy, and other stress pathways induced in 105C spheroids, in sharp contrast to the proliferating spheroid cells of the KOC-7c cell line. Next, we assessed levels of various G2/M checkpoint regulators and found a consistent reduction in steady-state levels of checkpoint regulators in dormant spheroid cells, but not proliferative spheroids. Our studies showed that proliferative spheroid cells were sensitive to Wee1 inhibition by AZD1775, but the dormant spheroid cells showed a degree of resistance to AZD1775, both in terms of EC50 values and spheroid reattachment abilities. Thus, we identified biomarkers of dormant spheroids, including the G2/M checkpoint regulators Wee1, Cdc25c, and PLK1, and showed that, when compared to proliferating spheroid cells, the transcriptome of dormant OCCC spheroids is a source of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877fd898a1fb77cad11de61e359dcf933c175ace" target='_blank'>
              Transcriptomic Analyses of Ovarian Clear Cell Carcinoma Spheroids Reveal Distinct Proliferative Phenotypes and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Bartlomiej Kolendowski, Sylvia Cheng, Yudith Ramos Valdés, Trevor G. Shepherd, G. DiMattia
          </td>
          <td>2025-05-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding the regulation of N6-methyladenosine (m6A), the prominent internal modification in mRNA, fosters the development of potential therapeutic strategies for human cancers. While the m6A demethylases FTO and ALKBH5 are recognized for their crucial roles in various cancers, their impact on lymphoid leukemia remains uncertain. Using T cell acute lymphoblastic leukemia (T-ALL) as a model system, we identify FTO as a unique vulnerability in T cell leukemia. Knockout of FTO, but not ALKBH5, significantly suppresses leukemia initiation and progression. Mechanistic analysis reveals that FTO heightens ELK3 mRNA stability in an m6A-dependent manner. Elevated ELK3 in turn transcriptionally activates the expression of glycolytic genes. Pharmacological inhibition of FTO suppresses ELK3 expression, hampers glycolysis and manifests remarkable antileukemia efficacy. Our findings unravel the crucial role of FTO in T-ALL and highlight the FTO-ELK3 axis as a key nodule during leukemogenesis, thereby providing a fundamental basis to harness selective FTO antagonist for T-ALL therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1aa1387d7c00ac34face8f8d24a3ea8d142c284" target='_blank'>
              FTO regulates ELK3-mediated metabolic rewiring and represents a unique therapeutic target in T cell leukemia
              </a>
            </td>
          <td>
            Hao Huang, Xinlu Li, Jinlian Luo, Chuan Gao, Mengjie Yang, Jin Xu, Ting Xie, Zhi Chen, Donghai Wang, Yuan Wang, Hua-Bing Li, Jinyan Huang, Yu Liu, Haojian Zhang, Panagiotis Ntziachristos, Yun Zhao, Guoliang Qing, Hudan Liu
          </td>
          <td>2025-05-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Medulloblastoma is a common embryonic malignant tumor in children. Patients with Group 3 medulloblastoma exhibit the poorest prognosis among all subgroups, and approximately 20% of these patients carry an amplification of MYC. Metabolic reprogramming, a hallmark of cancer, includes the pentose phosphate pathway (PPP) as a branch of glucose metabolism, providing cells with ribose-5-phosphate (R5P) and nicotinamide adenine dinucleotide phosphate (NADPH). The role of PPP in medulloblastoma remains unclear. In this study, we utilized transcriptomic data to identify that high expression of transketolase (TKT) correlates with worse overall survival (OS) in Group 3 patients. We found that TKT promotes proliferation of Group 3 medulloblastoma cell line cells both in vitro and in vivo. Additionally, TKT enhances R5P synthesis, increasing the proportion of S-phase cells and promoting proliferation. TKT also facilitates NADPH synthesis, which reduces intracellular reactive oxygen species (ROS) levels, inhibits ROS-induced cell death, and strengthens cellular resistance to ROS-induced injury. Subsequently, we demonstrated that inhibition of MYC leads to decreased TKT protein levels, and MYC promotes cell proliferation and suppresses cell death via TKT. Chromatin immunoprecipitation-quantitative real-time polymerase chain reaction (ChIP-qPCR) confirmed that employing the antibody targeting MYC enables the immunoprecipitation of DNA localized to the promoter region of TKT. Using luciferase assay and western blot, we verified that MYC and specificity protein 1 (SP1) co-regulate the transcription of TKT and consequently elevates TKT protein levels. Collectively, our study reports that MYC facilitates the proliferation of Group 3 medulloblastoma cells and mitigates ROS-induced damage through TKT, suggesting TKT as a potential therapeutic target for MYC-driven Group 3 medulloblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02051-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db08bc589d5182c54c613ec25df14812c7e9c6c6" target='_blank'>
              MYC promotes group 3 medulloblastoma cell proliferation and alleviates ROS-induced cell death by upregulating transketolase
              </a>
            </td>
          <td>
            Shizun Wang, Dan Zhang, Chunlong Wang, Yuqin Liu
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e15032


 Background:
 Homologous recombination deficiency (HRD) is an event triggered by impairment of the homologous recombination repair (HRR) pathway which acts on DNA double-strand breaks and interstrand cross-links (ICL). Although HRD has been shown to elevate the overall genomic instability in tumors inducing favorable responses to immune checkpoint inhibitors (ICI), its clinical utility is observed in specific cancer types like breast, ovarian, low-grade glioma, testicular germ cell tumors and kidney cancers. On the contrary, in many other cancer types, HRD is conducive to the development of immune-cold tumor microenvironment leading to unfavorable treatment responses to ICI. We present, HRD predictive scores with the corresponding IO biomarkers viz. microsatellite instability (MSI) and tumor mutation burden (TMB) pan-cancers.
 Methods:
 Retrospectively, we investigated the biomarker profile (HRD, MSI, TMB) and genomic (HRR pathway genes) profiles of 642 pan-cancer patients using a comprehensive genomic assay using OncoIndx.
 Results:
 Of the cohort of 642 pan-cancer patients, a total of 12.3% (n = 79/642) of patients were detected with high HRD scores, another 16.0% (n = 103/642) were detected with intermediate HRD scores and while 71.7% (n = 460/642) of patients showed low HRD scores. Among the cohorts of high, intermediate and low HRD scores, BRCA1/2 mutation-positive patients were 6.3% (n = 5/79), 5.8% (n = 6/103), and 5.2 (n = 24/460) respectively and HRR gene mutation-positive patients were 17.7% (n = 14/79), 25.2% (n = 26/103), 17.17% (n = 79/460) respectively. Most importantly, the immune-oncology biomarkers MSI and TMB scores were high only in 1.3% (n = 1/79) and 8.9% (n = 7/79) in HRD-high patients which were not significant compared to 1.7% (n = 8/460) and 10.4% (n = 48/460) observed in HRD-low patient cohort. Even in the cohort of patients with intermediate HRD scores, MSI and TMB did not vary significantly. Overall, no significant correlation was observed between high HRD scores vs BRCA1/2 mutations, HRR pathway gene mutations, MSI and TMB statuses. Thus, HRD scores could be predicted high, irrespective of all the associated biomarkers and pathway mutations. This indicates the utility of additional treatment options in BRCA1/2 wild-type or HRR pathway gene wild-type patients.
 Conclusions:
 Although HRD-high patients could be predictive of favorable PARPi and platinum-based therapies, their role in immune activation and ICI therapy response should be carefully considered. Even at a high score, HRD seems to be inducing non-inflammatory tumor microenvironment as opposed to triggering novel cell surface antigens and high TMB aiding in immunotherapy response. However, HRD as an individual biomarker alone adds immense therapeutic benefits even among BRCA-negative patients who could be potential for PARPi/platinum-based therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92c61b0baeee9c5a60d5b38acb29b5a9cb82e2f" target='_blank'>
              The non-canonical role of homologous recombination deficiency in the development of immune-cold tumor microenvironment.
              </a>
            </td>
          <td>
            A. Vasudevan, Aarthi Ramesh, Ankit Jain, Sandhya Iyer, Nidhi Patel, A. Joshi, M. K. Singhal, Sameer Shrirangwar, Chinmaya Kumar Pani, Ganapathi Bhat, V. Patil, Dr T. Raja, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0e72d616aeb3ee8d41accd4d7a574cb580739" target='_blank'>
              A continuum reaction-diffusion model for spread of gene silencing in chromosomal inactivation
              </a>
            </td>
          <td>
            Shibashis Paul, Sha Sun, Karmella A. Haynes, Tian Hong
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA repair is essential for preserving genome integrity. Podocytes, postmitotic epithelial cells of the kidney filtration unit, bear limited regenerative capacity, yet their survival is indispensable for kidney health. Podocyte loss is a hallmark of the aging process and of many diseases, but the underlying factors remain unclear. We investigated the consequences of DNA damage in a podocyte-specific knockout mouse model for DNA excision repair protein Ercc1 and in cultured podocytes under genomic stress. Furthermore, we characterized DNA damage-related alterations in mouse and human renal tissue of different ages and patients with minimal change disease and focal segmental glomerulosclerosis. Ercc1 knockout resulted in accumulation of DNA damage and ensuing albuminuria and kidney disease. Podocytes reacted to genomic stress by activating mTOR complex 1 (mTORC1) signaling in vitro and in vivo. This was abrogated by inhibiting DNA damage signaling through DNA-dependent protein kinase (DNA-PK) and ataxia teleangiectasia mutated (ATM) kinases, and inhibition of mTORC1 modulated the development of glomerulosclerosis. Perturbed DNA repair gene expression and genomic stress in podocytes were also detected in focal segmental glomerulosclerosis. Beyond that, DNA damage signaling occurred in podocytes of healthy aging mice and humans. We provide evidence that genome maintenance in podocytes is linked to the mTORC1 pathway and is involved in the aging process as well as the development of glomerulosclerosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e1092b41bf7562a413f9fd7c4ddd40708218ce" target='_blank'>
              Loss of genome maintenance is linked to mTOR complex 1 signaling and accelerates podocyte damage
              </a>
            </td>
          <td>
            Fabian Braun, Amrei M. Mandel, Linda Blomberg, M. N. Wong, G. Chatzinikolaou, David H Meyer, Anna Reinelt, V. Nair, Roman Akbar-Haase, P. McCown, Fabian Haas, He Chen, Mahdieh Rahmatollahi, Damian Fermin, Robin Ebbestad, G. Slaats, Tillmann Bork, C. Schell, Sybille Koehler, P. Brinkkoetter, Maja T. Lindenmeyer, C. D. Cohen, M. Kann, D. Unnersjö-Jess, Wilhelm Bloch, Matthew G Sampson, Martijn Dollé, Victor G. Puelles, M. Kretzler, G. Garinis, Tobias B. Huber, Bernhard Schermer, Thomas Benzing, Björn Schumacher, Christine E. Kurschat
          </td>
          <td>2025-05-20</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="
 Introduction: Epigenetic changes play a crucial role in cancer development. Among these, DNA methylation is one of the most significant due to its impact on gene expression and genome stability. Hyper-methylation generally suppresses transcription and hypo-methylation promotes it. In breast cancer research, tumor-adjacent tissue is often used as a reference for characterizing epigenetic alterations, but numerous studies suggest that the tissue adjacent to a tumor shares genomic characteristics with the tumor itself. The extent to which epigenetic profiles overlap between tumor, histologically normal tissues from the involved and uninvolved breast, and benign tissues from cancer-free individuals is unknown. Our study aims to address this gap by systematically comparing the DNA methylation profiles across these different tissue types. Methods: A total of 72 cancer patients were selected from Northwestern University, while data from 182 cancer-free women was obtained from tissue donated to the Komen Tissue Bank. From each individual cancer patient, four regional samples were collected: tumor tissue (TU), tumor-adjacent normal tissue (AN), benign tissue from the opposite quadrant of the involved breast (OQ), and benign tissue from the contralateral uninvolved breast (CUB). From cancer-free patients, normal breast tissue was collected (CFN). From these, we created a “tumor proximity axis”: CFN→CUB →OQ→AN→TU. Methylation profiles were assayed using Illumina’s Infinium Methylation EPIC v1.0 BeadChip. Differential methylation analysis was conducted in two sets of four pairwise comparisons: (1) comparing tumor samples to the four benign tissue categories, and (2) comparing tissue categories that are adjacent along the tumor proximity axis. The differentially methylated CpGs were analyzed for enrichment of transcription factor binding sites (TFBS) and genes. Results: Following data processing and quality control, there were 72 TU, 63 AN, 70 OQ, 72 CUB, and 182 CFN samples for analysis. Differential methylation analysis showed that TU samples had distinct methylation profiles, with more hypomethylation events compared to hypermethylation events relative to benign tissues. Case-benign tissues (AN, OQ, CUB) exhibited similar methylation profiles, distinct from CFN. Transcription factor binding site (TFBS) enrichment analysis revealed that case-benign samples, even those distant from tumor, i.e. CUB, showed breast cancer-related methylation changes. Hypomethylated sites in CUB compared to CFN were enriched for TF binding sites TP63, GATA3, ESR1, PR, AR, NR3C1, and GREB1. TU hypermethylation events were enriched for Polycomb-repressive complex 2 (PRC2) binding, including EZH2, SUZ12, and JARID2. ER+ and ER- tumors had distinct methylation profiles. Both ER+ and ER- tumors showed hypermethylation enrichment for PRC2-related binding motifs. However, hypomethylation events differed, with ER+ tumors enriched for hormone receptor-related pathways and ER- tumors enriched for hematopoiesis/immune-related pathways. We did not find any differential methylation between benign tissues from patients with ER+ vs. ER- tumors. Conclusions: DNA methylation changes profoundly at two points on the tumor proximity axis: CFN to CUB and AN to TU. Specifically, all case-benign tissues, including CUB, exhibited changes associated with breast cancer when compared to CFN. Although the methylation profiles of ER+ and ER- tumors differed, benign tissues showed no differences as a function of ER status of the tumor. These findings demonstrate that benign case tissue is not epigenetically “normal” and provide insights for further investigation into the process of tumorigenesis.
 Citation Format: Saya Dennis, Takahiro Tsukioki, Gannon Cottone, Wanding Zhou, Yuan Luo, Patricia A. Ganz, Mary E. Sehl, Seema Khan, Susan Clare. DNA methylation patterns are similar in benign tissue from ipsilateral and contralateral breast while different from matched breast cancer, and healthy controls [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr RF1-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e532f44a3009f75c42b2bf7806776eed3af9ac" target='_blank'>
              Abstract RF1-05: DNA methylation patterns are similar in benign tissue from ipsilateral and contralateral breast while different from matched breast cancer, and healthy controls
              </a>
            </td>
          <td>
            Saya R. Dennis, Takahiro Tsukioki, Gannon Cottone, Wanding Zhou, Yuan Luo, Patricia A. Ganz, M. Sehl, Seema A. Khan, Susan E Clare
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Synovial sarcoma (SS) is an aggressive malignant soft tissue sarcoma of undetermined tissue origin that harbors potential metastases and recurrences, leading to a poor prognosis. Polo-like kinase 1 (PLK1) is an essential member of the serine/threonine kinase family, which plays a key role in cell cycle progression and other cellular processes. Therefore, PLK1 could act as a promising target for treating various cancers, including a low incidence of sarcoma. Methods SS paraffin-embedded tissue samples were selected to detect the expression of PLK1 and apoptosis- and autophagy-related proteins using immunohistochemistry. We analyzed the effects of low PLK1 expression on the SS cell cycle and function using the PLK1 inhibitor BI2536 in SW982 and SSX1 cells (cells with high expression of the SYT-SSX1 fusion gene). Western blotting was used to detect the expression of PLK1 and apoptotic proteins in different groups. Finally, the antitumor effects and safety of BI2536 were evaluated using xenograft tumor experiments in nude mice. Results In SS tissues, PLK1 was overexpressed and autophagy-related proteins were expressed at low levels, leading to poor prognosis. Suppressing PLK1 expression using the PLK1 inhibitor BI2536 decreased the proliferation, migration, and invasion abilities of SW982 and SYT-SSX1 cells and promoted the expression of the apoptotic protein Bax. Meanwhile, we confirmed that SS cell apoptosis increased and G2/M cycle arrest occurred after PLK1 expression was inhibited. We found that BI2536 significantly inhibited the growth of SS-xenografted tumors in nude mice, and the expression of stem cell-related factors significantly decreased. Conclusion PLK1 overexpression was closely associated with poor prognosis. Suppressing PLK1 expression significantly inhibited the growth, invasion, and migration of SS cells, promoted apoptosis, and blocked the G2/M phase of the cell cycle. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06740-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37d396cfbb289d363c88b01a59ea841faba59fb" target='_blank'>
              Overexpression of oncogenic polo-like kinase 1 disrupts the invasiveness, cell cycle, and apoptosis in synovial sarcoma
              </a>
            </td>
          <td>
            Jun Zhang, Zhong Li, Minghui Xie, Ning Wang, Hao Liu, Feng Li, Li-Juan Pang, Yan Qi
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Significance Here, we address a major limitation in the effectiveness of poly(ADP-ribose) polymerase inhibitors (PARPi) in breast cancer gene (BRCA)-mutant prostate cancer treatment: Only ~50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARPi resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss-of-function mutations in the DNA prereplication complex (pre-RC) confer PARPi resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-RC, can restore PARPi sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95a3ba5883a1720490b54e4008aa7d7b0986c0b" target='_blank'>
              BRCA2 reversion mutation–independent resistance to PARP inhibition through impaired DNA prereplication complex function
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, Nikolaus Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles L Sawyers
          </td>
          <td>2025-06-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="
 e17631


 Background:
 Poly (ADP-ribose) polymerase inhibitors (PARPi) has been studied to show positive efficacy in endometrial cancer (EC). However, rational candidates for precision treatment with PARPi need to be further explored. Recently, it has been reported that tumors with homologous recombination deficiency (HRD) are sensitive to PARPi. Here, we aimed to assess the relationship between HRD and molecular subgroups in EC.
 Methods:
 HRD statues and gene characteristics of 372 EC patients were retrospectively collected from August, 2022 to August, 2024. HRD score was calculated as the sum of loss of heterozygosity (LOH), telomeric allelic imbalance (TAI), and large-scale state transitions (LST) scores based on the next-generation sequencing (NGS) data.
 Results:
 Of the EC that underwent NGS testing, there were 5.38% (20/372) patients with
 BRCA1/2
 mutations, and the top two
 BRCA1/2
 mutated subgroups were microsatellite instability-high (MSI-H) (2.96%) and copy number low (CN-L) (1.88%) subgroups. However, there were 10.75% (40/372) patients with positive HRD, which is largely restricted to copy number high (CN-H) (4.57%), CN-L (2.96%) and MSI-H (2.96%) subgroup. Interestingly, based on HRD score with the cutoff of 30, there were mostly CN-H and CN-L subgroups and almost no
 POLE
 mutation and MSI-H subgroups in non
 BRCA1/2
 mutation and positive HRD patients.
 Conclusions:
 BRCA1/2
 mutations and positive HRD were seen in 5.38% and 10.75% EC patients, respectively. HRD score testing, based on LOH, TAI, and LST, may help select patients benefiting from PARPi with particular reference to CN-H and CN-L subgroups.


 Prevalence of HRD statues in different molecular subgroups of EC.





 HRD
 Non-HRD




 Molecular Subgroups
 All

 BRCA1/2
 mutation

 HRD score≥30




 POLE
 mut

 1(0.27%)
 1(0.27%)
 0(0%)
 8(2.15%)


 MSI-H
 11(2.96%)
 11(2.96%)
 0(0%)
 64(17.20%)


 CN-H
 17(4.57%)
 1(0.27%)
 16(4.30%)
 64(17.20%)


 CN-L
 11(2.96%)
 7(1.88%)
 4(1.08%)
 196(52.69%)






 HRD: Homologous recombination deficiency; EC: endometrial carcinomas;
 POLE
 mut:
 POLE
 mutations; MSI-H: microsatellite instability-high; CN-L: copy number low; CN-H: copy number high.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ff21a16e023a00431cd6937437785071b41ecc" target='_blank'>
              Prevalence of homologous recombination deficiency in different molecular subgroups of endometrial carcinomas.
              </a>
            </td>
          <td>
            Y. Rao, Wenwen Zhang, Na Zhang, Han Han, Yong Xiao, Ping Liu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA methylation (DNAme) is the most commonly studied epigenetic mark in human populations. DNAme has gained attention in the Developmental Origins of Health and Disease field due to its gene expression regulation and potential long-term stability. Genetic variation and environmental exposures are amongst the main factors influencing inter-individual DNAme variability. However, the proportion and genomic distribution of their individual, additive and interactive effects on the DNA methylome remains unclear. Here, we introduce RAMEN, a Findable, Accessible, Interoperable, and Reusable (FAIR) framework tailored for DNAme microarrays. Using machine learning and statistical techniques, RAMEN models and dissects gene-environment contributions to genome-wide Variably Methylated Regions (VMRs), while controlling for spurious associations. To comprehensively test the power of RAMEN, we analyzed and characterized VMRs from cord blood samples from two independent cohorts (CHILD and PREDO; overall n=1,662). We identified genetics as a consistent key contributor to DNAme variability, usually in additive and interactive combinations with the environment, with genetic terms explaining the largest proportion of DNAme variance, compared to environmental and interaction terms. Operationalizing RAMEN as an R package to conduct scalable genome-exposome contribution analyses, our results highlighted the importance of genetic variation in sculpting DNAme patterns in early life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c30cafaf767429a5cebcd39d59a57129c4df90" target='_blank'>
              RAMEN: Dissecting individual, additive and interactive gene-environment contributions to DNA methylome variability in cord blood
              </a>
            </td>
          <td>
            Erick I. Navarro-Delgado, D. Czamara, Karlie Edwards, Maggie P. Fu, S. Merrill, Chaini Konwar, J. MacIsaac, D. Lin, P. Mandhane, Elinor Simons, P. Subbarao, Theo J. Moraes, J. Lahti, Gregory E. Miller, E. Binder, K. Räikkönen, S. Turvey, Keegan D. Korthauer, Michael S. Kobor
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background: Skin cutaneous melanoma (SKCM) is a highly aggressive cancer with significant mortality, necessitating novel prognostic markers and therapeutic strategies. Centrosome amplification (CA), a hallmark of genomic instability, contributes to cancer progression, but its role in SKCM remains unclear.
Methods: Transcriptomic data from SKCM patients were analyzed to identify differentially expressed genes (DEGs) between SKCM and normal tissues. Centrosome amplification-related genes (CA-RGs) were selected based on centrosomal functions. Prognostic CA-RGs were identified using Cox regression and LASSO analyses, resulting in a CA-RG-based risk model. Single-cell RNA sequencing (scRNA-seq) was employed to investigate cellular mechanisms, and immune infiltration analyses were conducted to assess CA-RGs’ impacts on the tumor microenvironment.
Results: Four CA-RGs (CDK2, KAT2B, NUBP1, CEP120) were identified as prognostic markers. A risk model effectively stratified patients by survival outcomes and was validated in external datasets. Immune infiltration analysis showed that low-risk patients had higher immune and stromal scores, with increased CD8+ T cells and M1 macrophages. ScRNA-seq analysis revealed interactions among fibroblasts, keratinocytes, and malignant cells, indicating CDK2 and KAT2B may promote tumor progression through intercellular signaling.
Conclusions: This study identifies novel CA-RGs and establishes a robust risk model for SKCM prognosis. Insights into the immune microenvironment and intercellular interactions provide a foundation for targeted therapies, including immunotherapy, offering potential strategies for improving SKCM management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528b17b9ac5ffbf7680a8973f6e27632744d159b" target='_blank'>
              Identification and targeting of centrosome amplification related signature genes for prognosis and therapy in skin cutaneous melanoma
              </a>
            </td>
          <td>
            Wendong Chen, Dawei Wang, Zhenyu Wu, Shengrong Cheng, Hailin Yao, Fei Zhu
          </td>
          <td>2025-05-14</td>
          <td>Cell Conflux</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07bd79e38eeb67868a3c867814b6e2e9bf7277a8" target='_blank'>
              Sequential PIDD1 auto-processing is essential for ploidy control in the liver and heart
              </a>
            </td>
          <td>
            F. Eichin, V. Sladky, Matthäus A. Reiner, M. Leone, Ernesto Abila, André F. Rendeiro, Ralph Böttcher, M. Dahlhoff, Thomas Kolbe, A. Villunger
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tumor development is mainly marked by the gradual transformation of cells that acquire capacities such as sustained growth signaling, evasion of growth suppression, resistance to cell death, and induction of angiogenesis, achieving replicative immortality and activating invasion and metastasis. How different epigenetic alterations like m1A, m5C, and m6A contribute to tumor development is a field that still needs to be investigated. The immune modulators, CD70, CD80, and TIGIT, mainly regulate T-cell activation and consequently the immune evasion of tumors. Here, we explored the presence and the potential consequences of RNA modifications in these regulators in pan-cancer. Our findings highlight the critical role of the m6A, m5C, and m1A in regulating CD70, CD80, and TIGIT across multiple solid tumors. By combining epitranscriptomics data with functional enrichment and survival modeling, we show that RNA modification enzymes not only modulate immune-related gene expression but also serve as potential biomarkers for patient prognosis. By constructing a robust four-gene prognostic signature involving YTHDF3, RBM15B, IGF2BP2, and TRMT61A, we demonstrate that RNA modification profiles can accurately stratify patients into risk groups with distinct overall survival outcomes. The performance of this model across eight cancer types underscores the translational promise of epitranscriptomic markers in both mechanistic understanding and personalized oncology. Altogether, our study bridges the gap between the mechanistic regulation of immune checkpoints and their clinical utility, offering novel insights into how the epitranscriptome can be leveraged to improve cancer prognosis and potentially enhance immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d03e20bb526dd9e6903a8377f001f578b9d1d1d" target='_blank'>
              Epitranscriptomics Regulation of CD70, CD80, and TIGIT in Cancer Immunity
              </a>
            </td>
          <td>
            C. Rigopoulos, Marios Gkoris, Ilias Georgakopoulos-Soares, I. Boulalas, A. Zaravinos
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive primary brain tumor in adults. The success of modern multimodal standards approved in anti-glioblastoma therapy remains limited. Consequently, new therapeutics are urgently needed. In this study, utilizing ex vivo, in silico, and in vitro approaches, we investigated the LCS1269 effects on two potential targets, DNA and Top I. We also elucidated the influence of LCS1269 on signaling pathways and GBM cell viability. Based on our docking data and competition studies results, we demonstrated that LCS1269 may bind to DNA, demonstrating selectivity toward AT-rich regions. We also showed that LCS1269 could dock both Top I/DNA binary complex and Top I active sites. LCS1269 caused Top I dysfunction and downregulated the expression of Top I. Moreover, the LCS1269 treatment of GBM cells facilitated DNA damage and the activation of the ATM/Chk1/BRCA1/Rad51 pathway. Meanwhile, DNA damage response induction and ATM/Chk1/BRCA1/Rad51 pathway activation were insufficient to prevent GBM cell death triggered by LCS1269 treatment. Our work shows that DNA and Top I are promising molecular targets of LCS1269, thus providing insight on several novel mechanisms of its anti-tumor activity. Nonetheless, we did not perform a biophysical validation of the LCS1269–DNA interaction, which is a limitation of our study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed93b81cfed77de24035b1bc33dd66f27dbf51de" target='_blank'>
              Targeting Topoisomerase I and DNA with LCS1269 Drives Glioblastoma Cell Death Despite ATM/Chk1/BRCA1/RAD51 Signaling Pathway Activation
              </a>
            </td>
          <td>
            Nikolay Kalitin, Ekaterina Savchenko, N. Samoylenkova, Natalia Koroleva, Anna Lushnikova, Aida Karamysheva, G. Pavlova
          </td>
          <td>2025-06-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction: Colorectal cancer (CRC) is a heterogeneous and multifactorial malignancy driven by a series of genetic and epigenetic alterations. In this field, telomere/telomerase dysfunction contributes to CRC carcinogenesis by impairing genomic stability and cellular replication. Objective: This study aimed to evaluate genetic and epigenetic alterations in CRC by examining mutation rates in the human telomerase reverse transcriptase (hTERT) promoter region, relative telomere length (RTL), hTERT gene expression, and DNA methylation in the TERT hypermethylated oncological region (THOR). Methods: A total of 45 CRC and 34 adjacent normal tissue samples from Moroccan patients were analyzed using molecular approaches, such as Sanger sequencing, quantitative PCR (qPCR), reverse transcription qPCR (RT-qPCR), and methylation-specific PCR (MSP). Results: No mutations in the hTERT promoter region were identified. However, hypermethylation in the THOR region was reported in 82.2% of CRC samples and 79.4% of adjacent normal tissues. High hTERT expression was detected in 50% of CRC patients. In addition, telomere length was significantly shorter (p=0.002) in cancerous tissues (1.41 [1.36 – 1.43]) compared to normal mucosa (1.559 [1.46 – 1.63]), with an RTL ratio less than 1 (0.90 [0.86 – 0.95]). No significant differences were found between clinicopathological features and hTERT expression, THOR methylation, or RTL, except for a significant correlation between THOR hypermethylation and smaller tumor size (p=0.017) and between THOR methylation and RTL in CRC tissues (p=0.034). Conclusion: These results suggest that telomere lengthening is crucial for CRC initiation and progression, and cancer cells tend to shorten telomeres to maintain the chromosomal instability (CIN) required for tumor progression. Further research is needed to elucidate the mechanisms underlying telomere shortening in CRC and understand the role of telomerase/telomere complex in CRC initiation and progression, which could provide new diagnostic, prognostic, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e32cd6246bc493b71aa8671eea1af0f3ca4e1f7" target='_blank'>
              Exploring genetic and epigenetic alterations in hTERT gene in colorectal cancer
              </a>
            </td>
          <td>
            Meryem Jafari, Boutaina Addoum, A. Laraqui, Mohammed Oukabli, Ahmed Bounaim, Abdelmounim Ait Ali, Sidi Mohammed Bouchantouf, Noureddine Njoumi, Belkouchi Abdelkader, Fatima Zohra BenMoula, Hicham El Annaz, Khalid Ennibi, I. Chaoui, Youssef Bakri, Mohammed Attaleb
          </td>
          <td>2025-06-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99ef5f46003666cce9bfbffdcaf4920a36fe552" target='_blank'>
              RNA G-quadruplexes: emerging regulators of gene expression and therapeutic targets.
              </a>
            </td>
          <td>
            Zukela Ruzi, Wei Zha, Huang Yuan Yuan, Jiaorui Liu
          </td>
          <td>2025-07-03</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Two epigenetically labile subsets of genes that link embryonic environmental exposures with adult disease susceptibility are those that are imprinted and those with metastable epialleles. The expression of genes with metastable epialleles, like the agouti gene in Agouti viable yellow (Avy) mice, is highly variable between individuals but uniform in tissues within an individual. We used the Avy mouse to demonstrate that exposure to nutritional supplements, chemical toxicants, and low-dose ionizing radiation during embryogenesis alters adult disease susceptibility by modifying the epigenome. Genomic imprinting is a unique species-dependent epigenetic form of gene regulation that evolved approximately 150 million years ago in a common ancestor to Therian mammals. It resulted in monoallelic parent-of-origin-dependent gene silencing. Thus, imprinted genes are functionally haploid disease susceptibility loci, since only a single genetic or epigenetic event is required to alter their function. Expression of imprinted genes in the human genome is regulated by hemi-methylated imprint control regions (ICRs) in the human imprintome. Furthermore, human imprintome ICRs associated with chronic diseases (e.g., cancer, diabetes, and obesity) and behavioral disorders (e.g., autism, bipolar disorder, psychopathy, and schizophrenia) can now be identified with the use of cells from peripheral samples and the human imprintome array. The importance of metastable epialleles and imprinted genes in the etiology of environmentally induced human chronic diseases is discussed in this review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960e9fe6811efda91a98ebc102150c171e03b920" target='_blank'>
              Epigenetics, human imprintome, and chronic diseases.
              </a>
            </td>
          <td>
            Randy L Jirtle
          </td>
          <td>2025-07-04</td>
          <td>Essays in biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian clear cell carcinoma (CCC) is a highly aggressive malignancy characterized by poor prognosis due to its resistance to chemotherapy, particularly in the advanced stages. This neoplasm originates from endometriotic cysts containing high levels of iron, derived from blood. This accumulation of iron generates oxidative stress, leading to DNA damage and carcinogenesis. Genomic analyses have identified frequent mutations in ARID1A and PIK3CA, which are also present in normal endometrial gland cells, suggesting that the cells originated from normal endometrium. A unique gene expression profile, termed the OCCC signature, is characterized by the upregulation of HNF1B signaling and oxidative stress-related genes. These genes are induced by the stressful microenvironment of endometriotic cysts. Comprehensive DNA methylation analysis has revealed that ovarian CCC exhibits a distinct epigenetic landscape, marked by global hypomethylation of HNF1B-related transcriptional signals, resulting in their activation. Functionally, ovarian CCC is characterized by enhanced anaerobic metabolism, mitochondrial dysfunction, and resistance to ferroptosis, an iron-dependent form of cell death. Notably, the suppression of HNF1B has been associated with increased sensitivity to chemotherapy, highlighting its role in drug resistance. These findings suggest that ovarian CCC develops through epigenomic carcinogenesis, wherein chronic oxidative stress in endometriotic cysts drives epigenetic alterations that promote tumor formation and therapeutic resistance, representing “epigenetic dispositions”. The elucidation of these mechanisms may provide novel insights into potential therapeutic targets to overcome the aggressive behaviors of ovarian CCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea7200b12a1379b5f745ae281bb319bfb01f6d0" target='_blank'>
              Epigenetic dispositions in ovarian clear cell carcinoma: a nexus of genomic, epigenetic, and metabolic alterations
              </a>
            </td>
          <td>
            Ken Yamaguchi, M. Jiang, T. Miyamoto, Rin Mizuno, Mana Taki, Koji Yamanoi, R. Murakami, J. Hamanishi, Y. Hosoe, T. Baba, N. Matsumura, Ikuo Konishi, Shingo Fujii, Masaki Mandai
          </td>
          <td>2025-06-02</td>
          <td>Academia Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>