<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-27 09:32:24 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>40</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>84</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>118</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>374</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>324</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, Britney Jiayu He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db525943b0ddbcca73f6e3d8fd119fef42f8fb23" target='_blank'>
              Aberrant inheritance of extrachromosomal DNA amplifications promotes cancer evolution
              </a>
            </td>
          <td>
            Shir Marom, Inbar Lifshits Dayan, Venkata Narasimha Kadali, Mădălina Giurgiu-Kraljič, Gabriela Koifman, Karen Hakeny, Madhuri Chaurasia, Orléna Benamozig, R. Nevo, I. Azuri, Julia Ryvkin, R. Rotkopf, Gil Stelzer, Nino Oniashvili, Jacques Mardoukh, Sarah Pollock, Nika Iremadze, Z. Shipony, Meital Kupervaser, N. Wigoda, Dena Leshkowitz, Z. Elazar, Esther Berko, A. Henssen, Ofer Shoshani
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (eccDNA) refers to small, circular DNA molecules that originate from chromosomal sequences and are prevalent across nearly all eukaryotic organisms. In humans, eccDNAs are widely distributed in normal tissues, cancerous tissues, and body fluids, where they play important roles in tumorigenesis and are often associated with poor clinical outcomes. Given their biological and clinical significance, a well-integrated and high-quality database is essential for advancing eccDNA-related research. To address this need, we developed eccDNABase, a comprehensive and curated resource for browsing, searching, and analyzing eccDNAs across multiple species. The database systematically catalogs eccDNA–disease associations from diverse tissues and organisms. Currently, eccDNABase contains 1,875,452 eccDNA–disease associations, encompassing 8,398 ecDNA entries across nine species, 63 diseases, and healthy individuals. Each entry provides detailed information, including eccDNA ID, type, chromosomal localization, species, tissue or cell line source, disease name and Disease Ontology ID, overlap length and percentage with genes, oncogene overlap, detection method, and links to literature and source databases. Given its extensive and curated datasets, eccDNABase serves as a valuable resource for both basic and translational research, offering deeper insights into the role of eccDNA in health and disease. The database is publicly accessible at http://cgga.org.cn/eccDNABase/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: A Comprehensive and High-Quality Database for Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNA (eccDNA) has emerged as a critical driver of oncogenesis, yet its functional roles in high‐grade serous ovarian cancer (HGSOC) remain poorly characterized. This highlights the need for comprehensive investigations into the abundance, biogenesis, and functional implications of eccDNA in HGSOC. Methods To characterize eccDNA in HGSOC, we performed comprehensive Circle‐seq analysis to assess eccDNA abundance and genomic annotation in HGSOC tissues compared to normal ovarian tissue. For mechanistic validation of eccDNA biogenesis pathways, targeted knockdown experiments of microhomology‐mediated end‐joining (MMEJ) dependent on LIG3 and POLQ were conducted. Functional characterization of HGSOC‐specific eccDNA‐harboring precursor microRNAs (eccMIRs) included in vitro assays using HGSOC cells and in vivo tumor growth experiments. Results Circle‐seq analysis revealed a 13‐fold increase in eccDNA abundance in HGSOC compared to normal ovarian tissue, with significant enrichment in promoter and coding regions. The MMEJ pathway was identified as the predominant pathway for eccDNA biogenesis in HGSOC, supported by characteristic microhomologies at junction sites and validation via LIG3 and POLQ knockdown experiments. Notably, HGSOC‐specific eccDNA frequently contained functional eccMIRs (eccMIR3661, eccMIR618, and eccMIR2277), which generate oncogenic miRNAs. These miRNAs promote tumor progression by downregulating tumor suppressor genes and activating key oncogenic pathways. Functional assays confirmed that these eccMIRs significantly enhanced HGSOC cell proliferation, migration, and invasion in vitro and promoted tumor growth in vivo. Conclusions These results underscore eccDNA as an oncogenic driver in HGSOC through non‐coding RNA‐mediated regulatory mechanisms, revealing novel therapeutic opportunities for targeting eccDNA biogenesis in this aggressive malignancy. Key points This study revealed a 13‐fold increase of eccDNA in HGSOC compared to normal tissues, with significant enrichment in promoter and coding regions. eccDNA‐derived miRNAs (eccMIRs) were shown to enhance cancer cell proliferation, invasion, and tumor growth through the expression of oncogenic miRNA sequences. The study highlights the importance of the MMEJ pathway in eccDNA generation and proposes that targeting eccDNA biogenesis in this aggressive malignancy presents a novel therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21093a2da7863b736808acd2b5cbd885658bf0" target='_blank'>
              Extrachromosomal circular DNA expressing miRNA promotes ovarian cancer progression
              </a>
            </td>
          <td>
            Ning Wu, Ling Wei, Qiyu Liu, T. He, Cuiyu Huang, Yunpeng Jiang, Kailong Li, Hongyan Guo, Fengbiao Mao, Xiaolu Zhao
          </td>
          <td>2025-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background This article presents an extensive review of the advancements in research concerning extrachromosomal circular DNA (ecDNA) within the context of breast cancer. As a distinct form of DNA, ecDNA is critically involved in the initiation, progression, diagnosis, and treatment of breast cancer. Methods The article provides a comprehensive analysis of the mechanisms underlying the formation of ecDNA, highlighting factors such as aberrant DNA damage repair and chromosomal rearrangements. It further examines the biological roles of ecDNA in augmenting oncogene expression, fostering tumor heterogeneity, and facilitating immune evasion. Results Epidemiological studies indicate significant variability in the distribution of ecDNA across different breast cancer subtypes, with a notable association with invasive subtypes, such as triple‐negative breast cancer. The presence of ecDNA is linked to poor prognosis and treatment resistance in patients. Current detection technologies for ecDNA include molecular biology and imaging techniques, and its detectability in plasma underscores its potential as a biomarker for liquid biopsy. Conclusions The development of ecDNA‐targeted therapies and their integration with immunotherapy strategies offers promising new avenues for breast cancer treatment. Despite challenges such as incomplete elucidation of mechanisms, standardization of detection methods, and ethical considerations, ecDNA remains a valuable biomarker and therapeutic target in breast cancer. Future research is anticipated to advance its clinical transformation and application in individualized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3be1ca65e1813757b54ee00006ec2026ac1e51f0" target='_blank'>
              Research Advances of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Xiaodan Zhu, Yiyang Qian, Quan Tang, Jiahui Li, Yuqing Geng, Shixia Gong, Chunhui Jin
          </td>
          <td>2025-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, , Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, Rosa Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G Swanson, Jeffrey M. Weiss, SMaHT Assembly, Claudia Carvalho, Tim H. H. Coorens, Kelley Harris, , E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke‐positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single‐cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low‐pass whole‐genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke‐endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability‐associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a629089847ef194ea5479a4ede2bc91a09e2d" target='_blank'>
              Detection and monitoring of translocation renal cell carcinoma via plasma cell-free epigenomic profiling
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ananthan Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Panday, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, Prathyusha Konda, Brad Fortunato, Rashad Nawfal, Razane El Hajj Chehade, Ze Zhang, Jillian O’Toole, J. Horst, Dory A Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, B. Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, Srinivas R Viswanathan
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration is recognized as a hallmark event in cervical carcinogenesis. However, it does not represent a routine phase of the viral life cycle but rather a stochastic occurrence, often constituting a dead-end pathway for the virus. High-risk human papillomavirus (hr-HPV) exhibits a greater propensity for integration. The progression from initial infection to genomic integration constitutes a dynamic multi-step oncogenic process in the development of cervical cancer (CC). This process involves viral entry, immune evasion, persistent infection, and ultimately integration. This article innovatively provides a comprehensive overview of this multi-stage mechanism: HPV, via the L1/L2 proteins, mediates internalization and establishes infection. Subsequently, under the influence of factors such as the host’s genetic background, vaginal microbiota imbalance, and immune evasion, the host’s DNA damage response (DDR) pathways are activated. Viral DNA integrates into host genome vulnerable sites (e.g., 3q28 and 8q24) through microhomology-mediated end joining (MMEJ) or other alternative pathways. Following integration, the expression of viral oncogenes persists, triggering host genomic rearrangements, aberrant epigenetic modifications, and immune microenvironment remodeling, all of which collectively drive cervical cancer progression. The study further reveals the clinical potential of HPV integration as a highly specific molecular biomarker, offering new perspectives for precision screening and targeted therapy. This dynamic model deepens our understanding of the HPV carcinogenic mechanism and provides a theoretical basis for intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2f3540f21b81b3f4af2b24e5ab200e71a6ae9d" target='_blank'>
              From Viral Infection to Genome Reshaping: The Triggering Role of HPV Integration in Cervical Cancer
              </a>
            </td>
          <td>
            Junlan Li, Shuang Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide vari- ants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alter- ations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational frame- work for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by indepen- dent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cryptococcus neoformans is a fungal pathogen of humans that causes life-threatening meningoencephalitis. During infection, enlarged, polyploid titan cells are produced that promote survival, evade immune cells, and generate diverse progeny. These titan cells divide to produce haploid, aneuploid or diploid daughter cells with enhanced stress tolerance; however, how ploidy reduction occurs is poorly understood. Here, we show that titan cells developed from diploid strains predominantly produce diploid daughter cells with haploid daughters observed infrequently. We further demonstrate that meiosis-specific genes, including DMC1 and SPO11, are critical for stable inheritance of a diploid genome in the daughter cells. Specifically, deletion of these genes in a heterozygous diploid background results in: 1) titan cells with a significantly reduced capacity to produce daughter cells; 2) increased phenotypic variation among daughter cells produced by the titan cells, including traits that could be relevant to cell growth and viability; and 3) daughter cells produced by the titan cells exhibiting high levels of loss of heterozygosity (LOH) and aneuploidy, suggested elevated genome instability. Taken together, these findings demonstrate the importance of meiosis-specific genes in the ploidy reduction process of titan cells derived from a heterozygous diploid background in an important human fungal pathogen. SIGNIFICANCE Polyploid titan cells are central to Cryptococcus neoformans persistence during infection, yet how they reduce ploidy to generate daughter cells was unclear. We show that the meiosis-specific factors Dmc1 and Spo11 are critical for titan cells to produce viable daughter cells specifically in a heterozygous diploid context, ensuring chromosome stability without introducing detectable canonical meiotic recombination. Loss of either gene compromises daughter cell production and causes genome instability (LOH, aneuploidy) and phenotypic heterogeneity. These findings reveal a “para-meiosis” safeguard that preserves genome balance during titan cell depolyploidization in C. neoformans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b707195ab1586edfb629b8392c64a813e56aa4" target='_blank'>
              Meiosis-specific genes play roles in ploidy reduction in Cryptococcus neoformans titan cells
              </a>
            </td>
          <td>
            Zhuyun Bian, Kayla Wilhoit, Julian Liber, Anushka Peer, Ziyan Xu, Paul M. Magwene, Sheng Sun, Joseph Heitman
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e712e19562ce359834d0d186aafed4a0127af156" target='_blank'>
              HCFC1 and YY1 mediate recruitment of COMPASS and Integrator to initiate X chromosome inactivation
              </a>
            </td>
          <td>
            Jeffrey Boeren, , Sarra Merzouk, E. Rentmeester, Rien van Haperen, Samuel J. Luchsinger-Morcelle, Cheryl Maduro, Richelle Rietdijk, Büşra Göynük, W. V. van Ijcken, J. Demmers, C. Gontan, Hegias Mira Bontenbal, J. Gribnau
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f59455ffad8232534ff09a38ff5659981429a5" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes.
              </a>
            </td>
          <td>
            L. G. de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2025-09-24</td>
          <td>Nature</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3aef172eea9f38f7ce840cbcf6314703f46a861" target='_blank'>
              Sister chromatid separation determines the proliferative characteristics upon whole-genome duplication through homologous chromosome rearrangement
              </a>
            </td>
          <td>
            Masaya Inoko, Guang Yang, Yuki Tsukada, Ryota Uehara
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Extrachromosomally amplified oncogenes are shown to drive evolution in a large subset of IDH wild-type glioblastomas, and the spatial patterns of extrachromosomal DNA contain information on a tumor’s evolutionary history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA–Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer Discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc4a7f8e64bdd86a0871fb665e1084f20e21063" target='_blank'>
              A mobile circular DNA element drives memory-related processes in mice
              </a>
            </td>
          <td>
            Haobin Ren, Hao Gong, W. Liau, Qiongyi Zhao, Alexander P. Walsh, Mason R. B. Musgrove, J. Davies, Esmi L. Zajackowski, Sachithrani U. Madugalle, P. Marshall, Tian Lan, Jichun Shi, Jiazhi Jiang, Wei Wei, Xiang Li, T. Bredy
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb77c980017aee37cb31324f45fa9fe45acb96" target='_blank'>
              Moonlighting Role of Meiotic SYCP1 in Breast Cancer: A Chromatin-Bound Regulator of DNA Repair, Transcription, and Drug Resistance
              </a>
            </td>
          <td>
            LC Brennan, OV Grinchuk, M. Pachon, IF Sou, C. Fawcett, CG Nogueira, M. Guthrie, AD Bates, M. Hine, A. Thomaz, Z. Hu, AB Fielding, OR Davies, W-W Tee, UL McClurg
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b95d07f47eb2caf5e3c1769c6acf5b8c92ae573" target='_blank'>
              Enhancers mediate euchromatin hopping at chromatin contact points
              </a>
            </td>
          <td>
            Shanelle Mullany, Tiegh Taylor, Hangpeng Li, Yaqing Zhao, Alisha Suri, Tom Sexton, James O. J. Davies, 
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The interplay between DNA methylation and super-enhancer activity is emerging as a key focus in cancer research. Super-enhancers, a specialized class of enhancers, strongly activate transcription of their target genes due to their dense clustering with essential transcription factors (TFs) and cofactors. In cancer, especially, these super-enhancers control key oncogenic drivers and often display abnormal DNA methylation patterns that can repress or overexpress target genes in both solid and blood cancers. Furthermore, DNA methylation in the super-enhancer region has been found to influence their regulatory capacity. Although enhancers are typically characterized by low DNA methylation, dysregulated methylation at super-enhancers is seen in most malignancies, affecting TF and chromatin regulator recruitment. Hypomethylation at these sites often accompanies oncogene hyperactivation, while hypermethylation can repress tumor suppressor mechanisms. Recent research highlights DNA methylation as a promising source of cancer biomarkers. This review examines the intricate relationship between DNA methylation and super-enhancer activity in cancer, concentrating on how methylation regulates super-enhancers, modulates oncogene expression, promotes oncogenesis, and serves as a target for novel oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856f95cde56634ccf673f640abf635b4018a59" target='_blank'>
              Super-enhancer DNA methylation in cancer: the mechanism of action and therapeutic directions
              </a>
            </td>
          <td>
            Yue Bi, Kang Zou, Ke Wang
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b8c268f4003d4cab2a7a6a760ce10e09bcebe0" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The genome is compacted in the nucleus through a hierarchical chromatin organization, ranging from chromosome territories to compartments, topologically associating domains (TADs), and individual nucleosomes. Nucleosome remodeling complexes hydrolyze ATP to translocate DNA and thereby mobilize histone proteins. While nucleosome remodeling complexes have been extensively studied for their roles in regulating nucleosome positioning and accessibility, their contributions to higher-order chromatin architecture remain less well understood. Here, we investigate the roles of two key nucleosome remodelers, esBAF and INO80C, in shaping 3D genome organization in mouse embryonic stem cells. Using Hi-C, we find that loss of either remodeler has minimal effects on global compartment or TAD structures. In contrast, subcompartment organization is notably altered, suggesting that esBAF and INO80C contribute to finer-scale chromatin topology. To overcome the limited resolution of Hi-C for detecting regulatory loops, we employed promoter capture Micro-C (PCMC), which revealed that the loss of esBAF or INO80C alters a subset of promoter anchored looping interactions. Although these changes occur at distinct genomic loci for each remodeler, the affected sites are commonly enriched for bivalent chromatin regions bound by OCT4, SOX2, and NANOG (OSN), as well as BRG1 and INO80 themselves. Together, our findings reveal that esBAF and INO80C selectively influence subcompartment identity and enhancer–promoter communication at key regulatory loci, highlighting a previously underappreciated role for nucleosome remodelers in higher-order chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e95264023995f85a35edad63b59e9eef19fae0c" target='_blank'>
              esBAF and INO80C fine-tune subcompartments and differentially regulate enhancer-promoter interactions
              </a>
            </td>
          <td>
            Braulio Bonilla, Benjamin J. Patty, Snehal Sambare, Job Dekker, T. Fazzio, Sarah J. Hainer
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany B. Johnson, Christopher Markovic, C. Fronick, Robert S. Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd28764a1bb58dc6ea970c4ddb741f750a9e8bc" target='_blank'>
              Genome-wide DNA bridging by H-NS reshapes the stationary phase nucleoid and transcriptional landscape
              </a>
            </td>
          <td>
            Lindsey E. Way, Xiaofeng Dai, Emma E Wiesler, Lauren A McCarthy, Zhongqing Ren, Gail G Hardy, D. E. Fuller, Labeeb Hossain, Anne S. Meyer, E. Abbondanzieri, Julie S. Biteen, Xindan Wang
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. V. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation can lead to breaks in the double-stranded structure of DNA. Although cells have mechanisms that can repair such breaks, misrejoining events can occur that lead to mutations in an individual’s genome. When mutations affect certain genes, such as oncogenes and tumor-suppressor genes, cancer can result. Although ionizing radiation is a known cause of cancer, it has been difficult to determine how much of an excess cancer risk results from low-dose radiation exposure. Radiation signatures that distinguish radiation-induced cancers from spontaneous cancers are thus useful for assessing cancer risks after exposure to low-dose radiation. Recent advances in high-throughput sequencing technologies have enabled the comprehensive profiling of mutations in individual cancer cells. Analyses of radiation-associated cancer genomes have identified an increase in small deletions and chromosome rearrangements, including large deletions, inversions, and translocations, which are likely to be formed by the misrejoining of DNA double-strand breaks. Among these, chromosome rearrangements leading to cancer-related mutational events are potentially useful as biomarkers and as therapeutic targets for treating radiation-induced cancer. This review summarizes research on radiation-induced mutational signatures found in animal and human cancers, consolidates current knowledge, and discusses the future perspectives in this research field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b1867c4003902c154bf44ef7de285c3bf97bccb" target='_blank'>
              Selective mRNA translation determines adaptative mutability of melanoma cells to anti-BRAF/MEK combination therapy
              </a>
            </td>
          <td>
            Lucilla Fabbri, Lucie Lagadec, Eva Guérin, Hélène Lecourt, Dorothée Baille, Laetitia Besse, Cédric Messaoudi, Laurent Désaubry, Hussein Abou-Hamdan, Sévérine Roy, Bérangère Lombard, D. Loew, Jean Yves Scoazec, Caroline Robert, S. Vagner
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="During cell division, chromosomes reorganise into compact bodies in which centromeres localise precisely at the chromatin surface1–4 to enable kinetochore-microtubule interactions essential for genome segregation5–8. The physical principles guiding this centromere positioning remain unknown. Here, we reveal that human core centromeres are directed to the chromatin surface by repulsion of centromere-associated proteins – independent of condensin-mediated loop extrusion and microtubule engagement. Using cellular perturbations, biochemical reconstitution, and multiscale molecular dynamics simulations, we show that chromatin surface localisation emerges from repulsion between condensed chromatin and both the kinetochore and the highly negatively charged centromere protein, CENP-B. Together, these elements form a centromeric region composed of two domains with opposing affinities, one favouring integration within the mitotic chromosome and the other favouring exposure to the surrounding cytoplasm, thereby driving surface positioning. Tethering synthetic negatively charged proteins to chromatin was sufficient to recapitulate this surface localisation in cells and in vitro, indicating that electrostatic repulsion is a key determinant of surface localisation. These findings demonstrate that centromere layering is not hardwired by chromatin folding patterns but instead emerges from phase separation in chromatin. Our work uncovers electrostatic polarity as a general and programmable mechanism to spatially organise chromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/add6a5979959c53f5ac5902a80961d55a68cfb5f" target='_blank'>
              An electrostatic repulsion model of centromere organisation
              </a>
            </td>
          <td>
            Caelan Bell, Lifeng Chen, M. J. Maristany, C. Blaukopf, Huabin Zhou, Jan Huertas, Jose I. Perez Lopez, Christoph C. H. Langer, Thomas L. Steinacker, Nikki Schuette, L. Doolittle, Jorge R. Espinosa, Sy Redding, Rosana Collepardo-Guevara, Michael K. Rosen, Daniel W. Gerlich
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f85b37be71af2ec4cbf186601f4b1c5fc232717" target='_blank'>
              Influence of cis-regulatory elements on regulatory divergence in human segmental duplications
              </a>
            </td>
          <td>
            Colin J. Shew, Gulhan Kaya, Sean P McGinty, Megan Y. Dennis
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Stem cell lineage commitment is governed by intricate interactions between epigenetic mechanisms and 3D genome organization. Traditional linear epigenetics, including DNA methylation and histone modifications, cannot fully elucidate the complex spatiotemporal regulation of gene expression. Recent advances in spatial genomics technologies, such as high‐throughput chromosome conformation capture (Hi‐C), single‐cell Hi‐C, and Chromatin immunoprecipitation combined with Hi‐C (Hi‐ChIP), have provided unprecedented insights into genome architecture, revealing key structural units like chromatin compartments, topologically associating domains (TADs), and chromatin loops. These structures dynamically reorganize during differentiation, influencing transcriptional accessibility and lineage‐specific gene activation. Additionally, liquid‐liquid phase separation (LLPS)‐mediated transcriptional condensates, such as transcription factories and super‐enhancers, have emerged as essential regulators of gene expression patterns during cell fate transitions. The integration of multiomics data and artificial intelligence‐driven predictive modeling further enhances the understanding of these regulatory networks. Despite ongoing technical challenges, including limitations in resolution, data complexity, and causal inference, recent advances continue to push the field forward. Engineered interventions such as CRISPR‐based spatial genome editing and AI‐powered computational platforms hold great promise for translating structural insights into targeted therapeutic strategies in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229c578a9eb772253ad14d19bcc178909f1f3d92" target='_blank'>
              3D Genome Architecture in Stem Cell Lineage Commitment: from Structural Organization to Precision Regulation
              </a>
            </td>
          <td>
            Yanchi He, Wenrui Li, Lin Li, Ying Yang, Yutong Lu, Yufei Pan, Qing Wang, Yuqiang Sun, Yuxuan Xie, Mingyue Wu, Peng Luo, Wansu Sun, Hengguo Zhang
          </td>
          <td>2025-09-01</td>
          <td>Advanced Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. Methods BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. Results BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. Conclusions These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02449-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcb21ab8df09d5daf66bd07fdf4abe868e528e74" target='_blank'>
              RIPPLET: Mutation-Only Gene and Pathway Profiling for Precision Oncology
              </a>
            </td>
          <td>
            Fabius Wiesmann, Cassandra Litchfield, M. Mayoux, Tumor Profiler Consortium, A. Wicki, Burkhard Becher, V. Koelzer, H. Moch, Marija Buljan, Sonia Tugues, Bettina Sobottka
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Chromosome conformation within the nucleus is essential for genome function. These have primarily been studied at the scale of loops and compartments, or at lower spatial resolution using traditional in situ hybridization in chemically fixed samples. However, the mesoscale organization of single chromosomes in vivo, shaped by the interplay between chromatin architectural proteins and histone modifications, remains partially understood. In this study, we interrogated the mesoscale conformations of interphase chromosomes in live human osteoblasts and transformed osteosarcoma cells, focusing on chromosome 19. Results Chromosome conformations were quantified by the aspect ratio of the principal axes of gyration tensors. In osteoblasts, approximately 81% of chromosome 19 are observed to consist of regions characterized by highly extended organizations, with aspect ratios approximately four times greater than those of spheres. In contrast, in osteosarcoma cells, the chromosome displays an extensively collapsed conformation, with aspect ratios more closely approximately that of a sphere. In both cell types, the chromosome’s conformation is bimodal and the balance between these two modes differs very significantly between the two cell types. While the mesoscopic conformation is considerably stable, it is superimposed on dynamic, smaller scale regions. Additional results reveal that this significant conformational shift is independent of the cell cycle but co-regulated by CTCF, cohesion, and H3K27 modifications. Conclusions Our findings provide new insights into the coordinated complex regulatory mechanisms governing mesoscale chromosome organization in normal and transformed osteogenic tissues. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03785-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f8561f1fa7ab9bbf02e738cec3deefcabedf878" target='_blank'>
              Differential regulation of mesoscale chromosome conformations in osteoblasts and osteosarcoma
              </a>
            </td>
          <td>
            Madhoolika Bisht, Yu-Chieh Chung, Siou-Luan He, Sydney Willey, Benjamin D. Sunkel, Meng Wang, Benjamin Z. Stanton, Li-Chun Tu
          </td>
          <td>2025-09-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab22d511d2948a185e5a280612430d61e3b4b0a" target='_blank'>
              Linking Germline Telomere Removal to Global Programmed DNA Elimination in Tetrahymena Genome Differentiation
              </a>
            </td>
          <td>
            Kohei Nagao, Kazufumi Mochizuki
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Retroelements in the human genome are silenced via multiple mechanisms, including DNA methylation, to prevent their potential mutagenic effect. Retroelement activity, demonstrated by their expression and somatic retrotransposition events, was shown to be deregulated in multiple tumors but not yet in leukemia. We hypothesized that treatment with hypomethylating agents, commonly used in myelodysplastic syndromes and acute myeloid leukemia, could lead to increased retroelement activity and somatic retrotranspositions, thus contributing to disease progression. To address this hypothesis, we induced the expression of ORF1p protein with hypomethylating agents in DAMI and HL‐60 myeloid cell lines. To study whether long‐term hypomethylating agent therapy induces somatic retrotranspositions, we analyzed (i) both cell lines treated for 4 weeks, and (ii) sequential samples from 17 patients with myelodysplastic syndrome treated with hypomethylating agents. Using a sensitive next‐generation sequencing (NGS)‐based method, no retroelement events were identified. To conclude, we show that although hypomethylating agents induce the expression of LINE‐1‐encoded proteins in myeloid cell lines, de novo somatic retrotransposition events do not arise during the long‐term exposure to these agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6777d30e90cc8e94bcf1ee3f59b0243f01da267f" target='_blank'>
              Hypomethylating agents increase L1 retroelement expression without inducing novel insertions in myeloid malignancies
              </a>
            </td>
          <td>
            S. Pavlova, H. Svozilová, Marcela Krzyžánková, Radim Sonnek, Anastasiya Volakhava, Anastasia Smirnova, Tatiana Grigoreva, Zuzana Jašková, Hana Synáčková, Dennis Wahl, Michaela Bilčíková, L. Červinek, Š. Pospíšilová, Ilgar Mamedov, K. Plevová
          </td>
          <td>2025-09-04</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, Nikki L Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-21082-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f693038dd949f6166800477057cd05b931fa2da" target='_blank'>
              SLFN11 restricts escape from telomere crisis to prevent alternative lengthening of telomeres
              </a>
            </td>
          <td>
            Sandra Segura-Bayona, V. Borel, Tyler H Stanage, Marija Maric, Melanie Walter, Rafaela A Oliveira, Shudong Li, Aurora I Idilli, Martina Peritore, Graeme Hewitt, M. Hekkelman, Daniel M. Snell, Sam T Jones, Chris Cheshire, Areda Elezi, Probir Chakravarty, Richard Mitter, Anna Mikolajczak, Harriet E Gee, E. Nye, Roderick J. O’Sullivan, A. Singhi, T. Brummelkamp, Anthony J. Cesare, S. Boulton
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human papillomaviruses (HPVs) maintain their genomes as minichromosomes in the nuclei of infected keratinocytes. This study investigates the association of HPV31 genomes with host chromatin using both HiC and 4C-seq chromosome conformation capture techniques. We show that HPV31 genomes preferentially associate with transcriptionally active A compartments of host chromatin, regions of open chromatin defined by ATAC-seq, and super-enhancers defined by Brd4 and H3K27ac ChIP-seq. The viral genome association sites were also highly correlated with genomic loci previously identified as common HPV integration sites in cervical cancers. Recent studies have shown that transcriptionally active sites are prone to dsDNA breaks, and we find a strong correlation among dsBREAK datasets with transcriptionally active and open regions of host chromatin and the HPV31 genome association sites defined in our study. These findings suggest that HPV genomes associate with cellular transcriptional epicenters to maintain active viral gene expression during persistent infection, but also indicate that the susceptibility of these regions to dsDNA breaks could explain their propensity for viral DNA integration in HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ecc91def6b800389ed078be4c219004519589f" target='_blank'>
              Human papillomavirus genomes associate with active host chromatin during persistent viral infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fd4d1f853d699db191c9c34adb97eb8ebac4f" target='_blank'>
              Protamine lacunae preserve the paternal chromatin landscape in sperm
              </a>
            </td>
          <td>
            Thomas W. Tullius, Rachel A Heuer, Stephanie Bohaczuk, Ben Mallory, Danilo Dubocanin, Jane E. Ranchalis, Ahmet Ayaz, Christopher E. Mason, Emre Seli, A. Phillippy, Andrew B. Stergachis, Bluma J. Lesch
          </td>
          <td>2025-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Oncomicrobes are estimated to cause 15% of cancers worldwide. When cancer whole-genome sequencing (WGS) data are collected, the microbes present are also sequenced, allowing the investigation of potential etiological and clinical associations. Interrogating the microbial community for 8908 patients encompassing 22 cancer types from the Genomics England WGS dataset revealed that only colorectal tumors exhibited unmistakably distinct microbial communities that can reliably be used to distinguish anatomical site [positive predictive value (PPV) = 0.95]. This pattern was validated in two independent datasets. Potential clinical relevance uncovered by our analyses included accurate detection of alphapapillomaviruses [human papillomavirus (HPV)] in oral cancers, when compared with current clinical standards, and the detection of rare, highly pathogenic viruses such as human T-lymphotropic virus-1. Biomarker investigations demonstrated statistically significant associations (P < 0.05) between a subset of anaerobic bacteria and survival in certain subtypes of sarcoma. Our results contradict previous claims that each cancer type has a distinct microbiological signature but highlight the potential value of microbial analysis for certain cancers as WGS of tumor samples becomes common in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e28dcd99723ccc30fe571b2d59df15e8ee150f" target='_blank'>
              The landscape of microbial associations in human cancer.
              </a>
            </td>
          <td>
            Abraham Gihawi, H.M. Wood, Jeremy Clark, Justin O'Grady, Rosalind A. Eeles, David C Wedge, G. M. Jakobsdottir, G. Magiorkinis, A. Schache, Liam Masterson, Matt Lechner, Tim R. Fenton, Terence M Jones, Adrienne M Flanagan, Solange De Noon, Alex Rubinsteyn, Rachel Hurst, Colin S Cooper, Daniel S. Brewer
          </td>
          <td>2025-09-03</td>
          <td>Science translational medicine</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517574b6bc8a73f48a3a9ee64c0be54874afb37" target='_blank'>
              TET CpG sequence context specific DNA demethylation shapes progression of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Y. Hoogstrate, S. Ghisai, L. van Hijfte, R. Head, I. D. de Heer, Marta Padovan, Maurice de Wit, Wies R. Vallentgoed, A. Dipasquale, M. Wijnenga, Bas Weenink, R. Luning, S. Maas, Adela Brzobohata, Michael Weller, Tobias Weiss, M. Mair, A. Berghoff, A. Wöhrer, A. Jeltsch, J. Koekkoek, H. M. Hazelbag, M. Kouwenhoven, Yongsoo Kim, Bart A. Westerman, B. Ylstra, A. Niers, K. Johnson, F. Varn, R. Verhaak, Mustafa Khasraw, M. J. van den Bent, Pieter Wesseling, P. French
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Genomic organization requires an intricate balance between the compact storage of genetic material and the ability to finely tune gene regulation. Chromatin looping achieves this balance by organizing concordantly regulated groups of genes and their regulatory elements into loops while also condensing DNA to fit into the small volume of a nucleus. A number of DNA-binding and associated proteins, including CTCF and cohesin, act as chromatin looping factors that mediate this process. Given the tight association between chromatin looping and gene expression, disordered genomic organization has been linked to disease development, including cancer. Recurrent mutations in chromatin looping factors are common in cancer, in particular blood cancers such as leukemia and myelodysplastic syndromes. In this review, we describe the evolution of our understanding of the chromatin looping process in healthy and malignant hematopoiesis and discuss the therapeutic potential of targeting chromatin looping factors in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05682afd3dc6513232a3e809f87b0eed8c538362" target='_blank'>
              Role of Chromatin Looping Factors in Leukemia.
              </a>
            </td>
          <td>
            Shira G Glushakow-Smith, Zuzana Tothova
          </td>
          <td>2025-09-15</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a28c3b0eeec817792757ba08ffa47e832addea" target='_blank'>
              The mitotic stopwatch limits the proliferation of whole genome doubled cells
              </a>
            </td>
          <td>
            LA Allan, R. Foy, M. Keli, AT Saurin
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human papillomavirus (HPV) is a prime target for genome-editing therapy as its E6 and E7 oncogenes are crucial for cancer development and maintenance. A key challenge in CRISPR/Cas9 therapy is the off-target effects. This study utilized a double-nicking technique to introduce DNA breaks in the E6 and E7 regions of HPV16. From 146 gRNA candidates, 16 double-nicking pairs were selected. Multiple combinations of double-nicking (DN)-gRNA pairs were delivered to HPV16-positive cells via lentiviruses, followed by Cas9 nickase (Cas9n) expression. Combinations of 3–4 DN-gRNA pairs effectively killed HPV16-positive cells while sparing HPV-negative cells. Off-target effects were reduced by nearly three orders of magnitude. An “all-in-one” adenovirus (AdV) system expressing four gRNA pairs and Cas9n showed promise in inhibiting tumor growth in HPV16-positive cancer models, demonstrating its potential as a safe and effective treatment for HPV-induced tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844375df7b66806693c2d5d8d8018d6b8709b6bc" target='_blank'>
              Disruption of Human Papillomavirus 16 E6/E7 Genes Using All-in-One Adenovirus Vectors Expressing Eight Double-Nicking Guide RNAs
              </a>
            </td>
          <td>
            Megumi Yamaji, Tomomi Nakahara, Tomoko Nakanishi, Satomi Aoyama-Kikawa, Kiyoshi Yamaguchi, Yoichi Furukawa, Mariko Nakamura, Tadashi Okada, Hirotaka Tabata, Ryoko Fuse, Eigo Shimizu, Rika Kasajima, S. Imoto, I. Kukimoto, Izumu Saito, Tohru Kiyono
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5023d83ec2a5ceab2bac53b66c2b863c5e8ad9" target='_blank'>
              Non-centromeric CENP-A epigenetically regulates epithelial-mesenchymal plasticity and heterogeneity in human cells
              </a>
            </td>
          <td>
            Charlène Renaud-Pageot, Daniele Capocefalo, Sébastien Lemaire, Audrey Forest, Laura Cantini, G. Almouzni
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ea6290df028815fb911512fc1b477390936835" target='_blank'>
              Pan-Cancer Single-Cell Profiling Uncovers the Biological Characteristics of Cancer-Testis Genes
              </a>
            </td>
          <td>
            Chunyang Fu, Ke Liu
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1980148077174708f4b58d5435d5f4b71871118c" target='_blank'>
              Determination of Copy Number Variations and Affected Gene Networks in Breast Cancer from Mexican Patients
              </a>
            </td>
          <td>
            Voleta Larios-Serrato, Hilda A. Valdez-Salazar, Javier Torres, M. Camorlinga-Ponce, P. Piña-Sánchez, Héctor Mayani, M. Ruiz-Tachiquín
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the predominant subtype of ovarian cancer and is characterized by a high rate of relapse after platinum-based chemotherapy. Herein, we present a comprehensive analysis of 111 Korean HGSOC samples using next-generation sequencing technology to elucidate their transcriptomic landscapes. Our investigation revealed the existence of four distinct transcriptional subtypes of ovarian cancer: immunoreactive, mesenchymal, proliferative, and differentiated, which is comparable to those of TCGA HGSOC transcriptional subgroups. Each subtype exhibited unique correlation networks and their immune cell composition was computationally determined. Notably, the immunoreactive cluster displayed the highest immune score, even in the context of pan solid-cancer types, accompanied by heightened expression of CD4+ and CD8+ T cells (P < 0.05), along with notable associations with neutrophil degranulation and antigen presentation pathways (FDR < 0.01). Conversely, the differentiated cluster demonstrated immunodepleted characteristics, featuring an elevated proportion of overexpressed cancer-germline antigens. We also identified several cancer-germline HGSOC antigens that could be further investigated as potential targets for immunological intervention in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffdfd9df4e6bf6e48b7483936ad622c3c2f2e59a" target='_blank'>
              Integrative transcriptomic analysis of Korean high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hyondeog Kim, Youngwook Kim, Juyeong Park, Dong-Min Shin, Ji Hyun Kim, Madhawa Gunathilake, So Young Kwon, Junyoung Shin, Chong Woo Yoo, S. Park, Myong Cheol Lim, Jeongseon Kim
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Telomere maintenance enables unlimited cell proliferation by counteracting telomere erosion. While the majority of tumors activate telomerase, a significant subset—approximately 10–15%—utilizes alternative lengthening of telomeres (ALT), a recombination-based mechanism. ALT-positive cancers are classically associated with genomic instability, anaphase bridges, chromosomal rearrangements, and resistance to DNA-damaging therapies. This process is closely associated with genetic instability, which contributes to chromosomal rearrangements and tumor evolution. Consequently, ALT has traditionally been considered an adverse prognostic marker in aggressive malignancies such as osteosarcoma, pancreatic neuroendocrine tumors, and high-grade sarcomas. Paradoxically, recent evidence demonstrates that ALT positivity correlates with improved survival in glioblastoma (GBM) and chondrosarcoma, two tumor types that have historically been regarded as immune-cold and therapeutically intractable. This favorable outcome likely reflects a convergence of factors, including replication stress and DNA damage that impose a fitness cost in slow-growing or metabolically constrained tumors. Loss of ATRX/DAXX, while enabling ALT, further amplifies chromatin fragility, and ALT-mediated instability may paradoxically enhance immunogenicity within immune-quiescent microenvironments. Moreover, ALT-positive cells exhibit unique therapeutic vulnerabilities, particularly to ATR and PARP inhibitors. Together, these observations support a context-dependent model in which ALT functions as a double-edged sword, acting as a driver of malignant aggressiveness in rapidly proliferating cancers while serving as a relative liability in slower-growing, immune-cold tumors. Understanding this duality not only refines prognostic stratification but also opens opportunities for precision oncology. By integrating ALT-specific biomarkers into clinical workflows and exploiting ALT-related DNA repair dependencies, clinicians may transform a once uniformly negative prognostic factor into an actionable therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6ed85bfa1a7b22ce5afc4a049e72d82de6bf4" target='_blank'>
              Alternative Lengthening of Telomeres: A Prognostic Paradox in Cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung
          </td>
          <td>2025-10-17</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f04f35397873ebcf0a05890d5a35b22af43957" target='_blank'>
              LncRNA LINC00941 Links Oncogenic KRAS Signaling to Aggressiveness and Chemoresistance in Pancreatic Cancer
              </a>
            </td>
          <td>
            Thalita Bueno Corrêa, Gabriel Lucas da Fonseca, Rauni Borges Marques, S. M. Gomes-Filho, Lutero Hasenkamp, Y. Magalhães, L. Russo, N. Hoch, F. Forti, Elena Levantini, E. M. Reis, D. S. Bassères
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Liquid biopsy represents a transformative approach in oncology, enabling noninvasive disease detection and monitoring through epigenetic signals in circulating tumor DNA (ctDNA), nucleosomes, and noncoding RNAs. Tumor initiation is driven by epigenetic modifications, including DNA methylation, histone alterations, and dysregulated noncoding RNAs, which disrupt gene regulation, cell cycle control, DNA repair, and metastatic processes. This review systematically examines recent evidence on DNA methylation, histone marks (e.g., H3K27me3, H3K18ac), and noncoding RNAs (miRNAs, lncRNAs) as biomarkers for early cancer detection, prognosis, and therapeutic response. Particular focus is placed on aberrant DNA methylation (e.g., hypermethylation of CDKN2A, RASSF1A) and altered histone modifications (e.g., EZH2‐mediated silencing) as indicators of tumor heterogeneity and evolution. Stable and specific in biofluids, noncoding RNAs such as oncogenic miR‐21, tumor‐suppressive miR‐34a, and metastasis‐associated MALAT1/HOTAIR further enhance clinical applicability. Recent detection methods, including bisulfite sequencing, ChIP‐seq, and RNA‐seq, have advanced biomarker profiling, though challenges remain in standardization and low‐abundance detection. With over 12 active clinical studies validating their utility, integration of epigenetic markers with AI and multiomics holds promise for individualized, dynamically guided oncology care. Future innovations, such as chromatin accessibility analysis and cfDNA fragmentation profiling, may further refine diagnostic precision and therapeutic monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f80572fe0354408f2985b22c2727d7355f5dac" target='_blank'>
              Transforming Cancer Diagnostics: The Emergence of Liquid Biopsy and Epigenetic Markers
              </a>
            </td>
          <td>
            Debalina Saha, Pritam Kanjilal, Mandeep Kaur, Soumya V. Menon, Ayash Ashraf, M. Kumar, T. Alqahtani, Shikha Atteri, D. Uti, Bikram Dhara
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>2</td>
          <td>17</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-14</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres.
              </a>
            </td>
          <td>
            Bailey A Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, Tadafumi Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1499e0bd4fc727a3ec07930d21e6808e9e69593" target='_blank'>
              Global Mapping of Combinatorial Chromatin Regulatory Events Using Hi-Plex CUT&Tag
              </a>
            </td>
          <td>
            Yuan Liao, Weiqiang Zhou, , Kuai Yu, Zixiao Jolene Wang, Thomas Drummond, Meng Ding, Yingming Zhao, Ignacio Pino, Sean D. Taverna, Hongkai Ji, Heng Zhu
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Introduction The epigenetic silencing of O6-methylguanine DNA methyltransferase (MGMT) is associated with reduced DNA repair capacity, carcinogenesis and increased sensitivity to alkylating chemotherapy. However, the biological role and clinical significance of MGMT overexpression in cancer remains poorly understood. Methods Using multiplexed quantitative immunofluorescence we measured the localized levels of MGMT protein, γH2AX and CD8+ T cells in multiple retrospective colorectal cancer (CRC) cohorts. Genomic and transcriptomic features of selected cases were also studied with whole exome DNA sequencing and genome-wide methylation analysis. MGMT-methylated human CRC cells SW620 were transfected with an MGMT-containing plasmid and co-cultured with allogeneic peripheral blood mononuclear cells. Results A subset of CRCs showed MGMT protein upregulation associated with lower γH2AX, reduced CD8+ tumor infiltrating lymphocytes (TILs), mismatch repair proficient (pMMR) status and shorter survival. CD8+ TILs were more distant from MGMT-expressing cells than MGMT-negative cells and the MGMT promoter methylation status did not highly correlate with MGMT protein levels in CRC. In genomic/transcriptomic analysis, high MGMT expression was associated with a lower nonsynonymous somatic mutational burden, higher transition-to-transversion mutation ratio, increased deleterious TP53 variants and distinct transcriptomic profiles. The exogenous expression of MGMT in SW620 CRC cells reduced the number of spontaneous nonsynonymous mutations, reproduced mutational features of MGMT-high CRC and limited the in vitro T-cell-mediated killing of malignant cells induced by proinflammatory cytokines in tumor/immune cell co-cultures. Conclusions MGMT overexpression identifies a previously undescribed subset of CRCs with distinct biological and clinical properties including reduced mutagenesis, adaptive immune evasion, predominantly pMMR phenotype and aggressive clinical course. Direct, quantitative assessment of MGMT protein expression using spatially resolved analysis is more reliable than inference of MGMT expression by promoter methylation status in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cc0eaa491cd34a4c5b928fccd34ad143431db3" target='_blank'>
              High MGMT expression identifies aggressive colorectal cancer with distinct genomic features and immune evasion properties
              </a>
            </td>
          <td>
            J. Zhang, B. K. Rajendran, Shruti S Desai, Joanna Gibson, Jassim DiPalermo, Patricia LoRusso, Yong Kong, Hongyu Zhao, Michael Cecchini, K. Schalper
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane E. Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L. Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>82</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Centromeres are crucial for precise chromosome segregation and maintaining genome stability during cell division. However, their evolutionary dynamics, particularly in polyploid organisms with complex genomic architectures, remain largely enigmatic. Allopolyploid wheat, with its well-defined hierarchical ploidy series and recent polyploidization history, serves as an excellent model to explore centromere evolution. Results In this study, we perform a systematic comparative analysis of centromeres in common wheat and its corresponding ancestral species, utilizing the latest comprehensive reference genome assembly available. Our findings reveal that wheat centromeres predominantly consist of five types of centromeric-specific retrotransposon elements (CRWs), with CRW1 and CRW2 being the most prevalent. We identify distinct evolutionary trajectories in the functional centromeres of each subgenome, characterized by variations in copy number, insertion age, and CRW composition. By utilizing CENH3-ChIP data across various ploidy levels, we uncover a series of CRW invasion events that have shaped the evolution of AA subgenome centromeres. Conversely, the evolutionary process of the DD subgenome centromeres involves their expansion from diploid to hexaploid wheat, facilitating adaptation to a larger genomic context. Integration of complete einkorn centromere assemblies and Aegilops tauschii pan-genomes further revealed subgenome-specific centromere evolutionary trajectories. By inclusion of synthetic hexaploid from S2-S3 generations, alongside 2x/6 × natural accessions, we demonstrate that DD subgenome centromere expansion represents a gradual evolutionary process rather than an immediate response to polyploidization. Conclusions Our study provides a comprehensive landscape of centromere adaptation, evolution, and maturation, along with insights into how retrotransposon invasions drive centromere evolution in polyploid wheat. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03759-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f98dee703cbbec473e46d88966ab2318a6f43c" target='_blank'>
              Distinct evolutionary trajectories of subgenomic centromeres in polyploid wheat
              </a>
            </td>
          <td>
            Yuhong Huang, Yang Liu, Chang Liu, Congyang Yi, Jinsheng Lai, Hongqing Ling, H. Su, Fang Han
          </td>
          <td>2025-09-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Numerous cellular processes rely on biomolecular condensates formed through liquid–liquid phase separation (LLPS). Recently, it has become evident that somatic mutations can interfere with or over-activate the formation of phase-separated condensates. Results Here, we set out to systematically study the connection between cancer and biological condensation, specifically mapping the extent to which LLPS is affected in cancer and understanding the molecular pathomechanisms and therapeutic consequences of mutations affecting LLPS scaffolds. We identify both known and novel combinations of molecular functions that are specific to oncogenic fusion proteins and thus have a high potential for driving tumorigenesis. Protein regions driving condensate formation show an increased association with DNA- or chromatin-binding domains of transcription regulators within oncogenic fusion proteins, indicating a common molecular mechanism underlying several soft tissue sarcomas and hematologic malignancies where phase-separation-prone oncogenic fusion proteins form abnormal condensates along the DNA and thereby dysregulate gene expression programs. Conclusions We find that proteins initiating LLPS are frequently implicated in somatic cancers, even surpassing their involvement in neurodegeneration. Our data shows that cancer-driving LLPS scaffolds tend to be potent oncogenes, giving rise to dominant phenotypes and lacking targeting options by current FDA-approved drugs. Finding the currently missing drugs to shut down oncogenic fusion proteins, to disrupt the condensation enabled by them, and to offset their downstream effects could provide cancer drugs widely applicable to diverse cancer incidences previously defying standard treatments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03787-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54d550a595352429b7ad7fdab681cd564170f566" target='_blank'>
              Phase-separating fusion proteins drive cancer by upsetting transcription regulation
              </a>
            </td>
          <td>
            Nazanin Farahi, Tamas Lazar, Peter Tompa, Bálint Mészáros, Rita Pancsa
          </td>
          <td>2025-10-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49580bd801398c666e64f8508dfb8bf04f7a7c7d" target='_blank'>
              Epidrugs in cancer: mechanisms, applications, and future direction.
              </a>
            </td>
          <td>
            Roshni Bibi, Lakshmi Varshetha, Rakshit Kamal Lahary, Jahnavi Namburi, Fardeen Ahmed Laskar, Koustav Sarkar
          </td>
          <td>2025-10-08</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41521f18f4da65cbb988d3434d4d07459d2803e" target='_blank'>
              Epigenetic characterization of pseudogenes across human tissues
              </a>
            </td>
          <td>
            Yunzhe Jiang, Beatrice Borsari, Mark B. Gerstein
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Overexpression of the proto-oncogene MYC occurs in >70% of cancers and is especially prevalent in breast cancer. Myc partners with transcription factor Max to bind to the E-box DNA response element. By patterning our frankenproteins on the basic region/helix-loop-helix/leucine zipper motif of Max, we designed MEF and MEF/C93 to bind to the E-box. In bacterial one-hybrid assays and quantitative electrophoretic mobility shift assay, both proteins bound specifically to the E-box with high sequence-specificity and affinity (Kd = 8 nM) rivaling native transcription factors. Quantitative PCR revealed that MEF and MEF/C93 selectively downregulated Myc target genes in Myc-dependent MDA-MB-231 breast cancer cells, but not in Myc-independent MCF-7 cells. Fluorescence colocalization demonstrated transport into cell nuclei. Our proteins displayed IC50 values of 1-2 μM in cell viability assays in MDA-MB-231, compared with ∼25 μM in MCF-7. These results demonstrate the specificity of targeting Myc-dependent cancer cells and mark significant progress toward protein-based therapies aimed at Myc-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cc39bbdc2499f24c52dd2589dd7f515be4ffe6" target='_blank'>
              Designer Frankenproteins That Halt the Proliferation of Myc-Driven Cancer Cells.
              </a>
            </td>
          <td>
            Maryam Ali, Raneem Akel, Francine He, Micheline Piquette-Miller, Jumi A Shin
          </td>
          <td>2025-09-03</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Repetitive DNA sequences, as transposable elements (TEs) and satellite DNA (satDNA) spread and diversify within host genomes, impacting genome biology in numerous ways. In the first part of this review, we emphasize the evolutionary importance of satDNAs and TEs, providing a short summary of their roles and the mechanisms by which they influence the structure and function of genomes. We also discuss the broad, complex, and extensive relationships between TEs and satDNAs. Following that, we bring together different mechanisms on the generation of satDNA from TE, as it has been demonstrated that almost any part of any type of TE can undergo tandemization and produce novel satDNAs. Importantly, we here present a hypothesis that would explain the existence of particular types of monomers, namely composite satDNA monomers which display multiple subsequent stretches of similarity to various TEs, for which the explanation was lacking so far. We propose that even highly shuffled and degraded TE remnants residing in heterochromatin "TE graveyards" can give rise to new satDNA sequence monomers, transforming these genomic loci into DNA "recycling yards." Furthermore, we emphasize important evolutionary questions regarding the causes, mechanisms, and frequency of these occurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f67b41471ebec0a1b45c2936430e5e0e7ad266" target='_blank'>
              Satellite DNAs rising from the transposon graveyards.
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Patrik Majcen, Miroslav Plohl
          </td>
          <td>2025-09-30</td>
          <td>DNA research : an international journal for rapid publication of reports on genes and genomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome in bladder cancer drives metabolic reprogramming
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86eafc869c572b41b230c166bb3e0741b603485e" target='_blank'>
              Aberrant Hippo-YAP/TEAD signaling drives malignant transcriptional reprogramming in external auditory canal squamous cell carcinoma
              </a>
            </td>
          <td>
            Kuniaki Sato, N. Komune, Mayumi Ono, Takahiro Hongo, T. Nakano, Kensuke Koike, Shinsaku Itoyama, Kenichi Taguchi, Koshi Mimori, J. Gutkind, Muneyuki Masuda, Takashi Nakagawa
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Lower-grade glioma (LGG) is a common primary brain tumor with potential progression to glioblastoma. Its development is closely linked to DNA damage repair. DNA polymerase delta subunit 3 (POLD3), a subunit of the DNA polymerase δ complex, participates in DNA replication and repair. POLD3 may act as a therapeutic target and prognostic biomarker in LGG. Methods We collected primary data from The Cancer Genome Atlas (TCGA), the Chinese Glioma Genome Atlas (CGGA), the Tumor Immune Estimation Resource (TIMER), and Gene Set Enrichment Analysis (GSEA). The CGGA dataset included messenger RNA (mRNA) sequencing and clinical data from 403 LGG cases; its microarray subset contained 142 additional cases. The TCGA dataset provided clinical data for 503 LGG cases. Employing bioinformatics tools, we investigated the correlation of POLD3 expression with the clinical characteristics, pathological features, survival outcomes, and tumor immune microenvironment in LGG. Subsequent functional validation of POLD3’s impact in LGG was performed using both cellular and animal models. Results Integrative analysis combining bioinformatics and experimental approaches revealed significant POLD3 overexpression in LGG. Elevated POLD3 levels exhibited significant associations with distinct clinicopathological features, altered immune cell infiltration profiles, and adverse clinical outcomes. Functionally, POLD3 was demonstrated to drive LGG cell proliferation, invasion, and modulate the tumor microenvironment. Conclusion This study establishes POLD3 as a critical molecular determinant in LGG pathogenesis. Our findings position POLD3 as a promising multifaceted target, with significant implications for advancing LGG diagnostics, therapeutic development, immune modulation strategies, and prognostic assessment, thereby providing a fresh conceptual framework for understanding and managing this tumor. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14934-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b766711209d5df5ec31c2b94f1204021afe9d8" target='_blank'>
              POLD3 knockdown effects on low-grade glioma: insights from bioinformatics and experimental validation
              </a>
            </td>
          <td>
            Zhihui Yang, Zhonghao Dou, Hao Wang, Zhaojiang Hu, Yanbo Gao, Yilei Bai, Junfeng Liu, Rongjun Qian
          </td>
          <td>2025-10-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetically encoded DNA recorders convert transient biological events into stable genomic mutations, offering a means to reconstruct past cellular states. However, current approaches to log historical events by modifying genomic DNA have limited capacity to record the magnitude of biological signals within individual cells. Here, we introduce MitoScribe, a mitochondrial DNA (mtDNA)-based recording platform that uses mtDNA base editors (DdCBEs) to write graded biological signals into mtDNA as neutral, single-nucleotide substitutions at a defined site. Taking advantage of the hundreds to thousands of mitochondrial genome copies per cell, we demonstrate MitoScribe enables reproducible, highly sensitive, non-destructive, durable, and high-throughput measurements of molecular signals, including hypoxia, NF-κB activity, BMP and Wnt signaling. We show multiple modes of operation, including multiplexed recordings of two independent signals, and coincidence detection of temporally overlapping signals. Coupling MitoScribe with single-cell RNA sequencing and mitochondrial transcript enrichment, we further reconstruct signaling dynamics at the single-cell transcriptome level. Applying this approach during the directed differentiation of human induced pluripotent stem cells (iPSCs) toward mesoderm, we show that early heterogeneity in response to a differentiation cue predicts the later cell state. Together, MitoScribe provides a scalable platform for high-resolution molecular recording in complex cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03af0fb96ee61d1661672c9f2ce3319d8f9ca337" target='_blank'>
              MitoScribe single-cell molecular recorder logs graded signaling dynamics into mitochondrial DNA
              </a>
            </td>
          <td>
            Linhan Wang, Nikolaos Poulis, Deepak Srivastava, Seth L Shipman
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Berberine (BBR) is an isoquinoline alkaloid with a variety of biological activities, including anti‐microbial and anti‐tumoral activities. However, the cellular targets of BBR and the roles of BBR in the radiosensitivity of breast cancer cells are not well defined. In this study, we investigated the effects of BBR on the radiosensitivity of BT549 triple‐negative breast cancer cells. Through RNA‐seq results, we found that BBR significantly down‐regulated the expression of DNA ligases. The results of western blot and comet assay confirmed that BBR attenuated DNA ligase III (LIGIII) expression and caused DNA damage in a dose‐dependent manner. The results of electron microscopy showed that BBR decreased mitochondrial copies and induced mitochondrial autophagy. Clonal formation analysis showed that BBR sensitised breast cancer cells to irradiation. The genetic complementation and interference assays showed that the effect of BBR on the radiosensitivity of BT549 breast cancer cells is dependent on the expression of LIGIII. These results contribute to the understanding of the effects of BBR on cellular DNA repair and the clinical application of BBR in breast cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7148d132ad7e87d8b8ae13f9a1fb9fcaf3cd90" target='_blank'>
              Berberine Sensitises Breast Cancer Cells to Radiation via the Attenuation of DNA Ligase III
              </a>
            </td>
          <td>
            Yuxin Sun, Cong Li, Hang Yin, Kunyan Li, Ying Wang, Shuailong Zhang, Yi Zhao, Jing Wang, Weifeng Mao
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Many gene fusions have been implicated in the development and progression of human cancers. Our group and others have identified specific gene fusions involving the MAML2 gene in human cancers, and we were interested in searching for additional novel MAML2 fusions. Our analysis with the Caris Life Sciences molecular database demonstrated that most of the novel fusions were associated with increased fusion burden and genomic loss of heterozygosity, which are potentially suggestive of genomic instability, except for ATXN3::MAML2 which warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa520a41486a278de5084f190cbe8e828a82921" target='_blank'>
              Novel MAML2 Fusions in Human Malignancy
              </a>
            </td>
          <td>
            Takefumi Komiya, K. Sweeney, Chao H. Huang, A. Crymes, E. S. Antonarakis, Andrew Elliott, Matt Oberley, Mark G. Evans
          </td>
          <td>2025-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Breast cancer (BCa) is a major contributor to female mortality worldwide. Treatment resistance and tumor heterogeneity contribute to the lack of effective therapeutic targets, posing a significant challenge in BCa management. CENPI, a core centromere protein involved in chromosome segregation, has emerging evidence implicating it in oncogenesis across diverse malignancies. However, its functional and molecular mechanisms in BCa remain unclear. Methods We analyzed CENPI expression and its clinical significance by using the BCa dataset from the Cancer Genome Atlas (TCGA) and immunohistochemical staining of 3 human BCa tissue samples. Cellular functional assays and mice xenograft models were utilized to assess the effects of CENPI on BCa growth. RNA sequencing combined with bioinformatics analysis was conducted to elucidate the molecular mechanisms underlying CENPI function, with further validation through Western blotting, immunofluorescence, and TOP/FOP flash assays. Results CENPI was aberrantly overexpressed in BCa, with elevated expression levels strongly associated with disease progression and poor prognosis. Functional assays demonstrated that CENPI significantly promoted breast carcinogenesis in both cellular and animal models. Mechanistically, CENPI increased BCa progression and malignant phenotypes by modulating the Wnt/β-catenin axis. Conclusions CENPI is a critical oncogene in BCa, driving tumorigenesis and disease progression via the Wnt/β-catenin axis, which represents a promising biomarker and therapeutic target for BCa. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-04001-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33eef316f873dd58287f9f4d71241b7970f53004" target='_blank'>
              Centromere protein I facilitates breast cancer tumorigenesis and disease progression through modulation of Wnt/β-Catenin signaling
              </a>
            </td>
          <td>
            Chaoshen Wu, Yibing Zhou, Yuxiao Mu, Yijie Cheng, Wenqian Xu, Mengna Huang, Jingyuan Xu, Ming Li, Jianfeng Gu, Xuli Meng, Da Qian
          </td>
          <td>2025-10-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily L. Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612aed154e4338b00f169091d56c905e3a95eb57" target='_blank'>
              Demonstration of SLU7 as a new pan-cancer target
              </a>
            </td>
          <td>
            C. Rojo, A. Otero, M. Elizalde, M. Azkona, R. Barbero, MU. Latasa, I. Uriarte, A. Gutierrez-Uzquiza, D. Alignani, L. Guembe, A. Lujambio, F. Pastor, MG. Fernández-Barrena, MA. Ávila, M. Arechederra, C. Berasain, Prof. Carmen Berasain
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7f9e82789ed7b8469f2663f27086e180c73bd6" target='_blank'>
              The mechanical code of DNA impacts its interaction with DNA gyrase
              </a>
            </td>
          <td>
            Bailey G. Forbes, Rosella Pinckney, Sisi Liu, Aditi Biswas, Heather Potter, Elizabeth R. Morris, Aakash Basu-Biswas
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1147c4ec4d5050e89e9fab52610e8f2d66fcc3cc" target='_blank'>
              Integrative multi-modal analysis reveals the contribution of noncoding RNAs to post-treatment progression of IDH-mutant astrocytomas
              </a>
            </td>
          <td>
            A. Hartewig, S. Jaatinen, M. Annala, J. Haapasalo, Wei Zhang, H. Haapasalo, M. Nykter, Xin Lai, K. Rautajoki
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27406ea76dc0545d7ca7511116aac0edd200faab" target='_blank'>
              WSB-1 regulates DNA repair and the response to DNA damage response inhibitors in breast cancer
              </a>
            </td>
          <td>
            Effrosyni Antonopoulou, Chun Li, Amen Shamin, Scott Taylor, Flore-Anne Poujade, Michael Lind, Rajarshi Roy, A. Choudhury, Isabel M. Pires
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is one of the most prevalent adult leukemias, derived from mature B-cells and exhibiting a highly heterogeneous disease course. Standard cytogenetic analysis of CLL includes FISH and karyotyping. However, conventional chromosome analysis of cancer specimens is often constrained by low chromosomal resolution, and FISH analysis is limited by the number of probes that can be applied. This study highlights the application of optical genome mapping (OGM), a high-resolution cytogenomic tool that visualizes ultra-long, sequence-labeled DNA molecules, to uncover the structural complexity of the cancer genome and assess the clinical relevance of chromothripsis in CLL. Comprehensive cytogenetic analysis was conducted on a 43-year-old male diagnosed with chronic lymphocytic leukemia. Karyotyping revealed a complex rearrangement: 46,XY,der(3)t(3;13)(p2?3;q14.3),der(4)t(?3;4)(p23;p16),add(11)(p13),del(13)(q14)[12]/46,sl,del(11)(q2?2.2q23.3)[6]. FISH analysis further identified the loss of ATM and a partial deletion of the D13S319 locus. OGM analysis performed on bone marrow revealed a complex genotype including chromothripsis of chromosome 13, and structural rearrangements involving chromosomes 3, 4, and 11. Additionally, multiple intrachromosomal translocations and interstitial microdeletions of chromosome 13 were identified. The resolution of these aberrations has been significantly enhanced with examples including: ogm[GRCh38] t(3;13)(p26.3;q33.1)(2,706,645~2,721,113;103,142,901~103,154,241][VAF0.45], ogm[GRCh38] t(4;13)(p15.31;q32.1)(20,869,721~20,907,265;96,617,837~96,630,317)[VAF0.42],. In conclusion, OGM revealed the intricate structural alterations of the cancer genome. The high resolution provided by OGM could facilitate the discovery of oncogenic mechanisms, novel fusion genes, prognostic markers, and potential therapeutic targets. OGM serves as a powerful tool for revisiting CLL disease classification by offering deeper insights into complex genomic rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4656aef9b3e1a0de1b2f5258ea921c03f0e3137" target='_blank'>
              Case Report: Unraveling complex genomic alterations in a case of chronic lymphocytic leukemia using optical genome mapping
              </a>
            </td>
          <td>
            Leila Youssefian, Trilochan Sahoo, Jia-Chi Wang
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3493648729eb60ce21f81ce27852f8a58bffd93" target='_blank'>
              Influence of Epigenetic Regulators Expression on Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            Lucas Brabo Rotella, A. Khayat, Monique Feitoza, Victoria Pereira Costa, Diego Pereira, Fabiano Cordeiro Moreira, S. Demachki, Samir Mansour Moraes Casseb, G. Ishak, W. Barra, R. Burbano, P. P. D. Assumpção
          </td>
          <td>2025-09-15</td>
          <td>None</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20084-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Subtelomeres are imperfect repeats adjacent to telomeres that are repressed by heterochromatin. Although essential for genome integrity, their repetitive nature has thwarted dissection of local heterochromatin assembly and maintenance mechanisms. Here, we engineered Schizosaccharomyces pombe strains carrying fluorescent reporters at a single subtelomere. We find that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites. Telomere-proximal regions depend on canonical shelterin or RNA interference nucleation pathways, while telomere-distal regions require nucleosome remodelers, histone chaperones, and boundary-associated factors. Using multi-generational live imaging and targeted perturbations, we show that subtelomeric subdomains display position-specific, clonally variable silencing across a spectrum of robust to fragile epigenetic states. This clonal variegation is also induced by naturally occurring subtelomeric structural variants. These findings demonstrate that subtelomeric heterochromatin maintenance is not uniform but rather governed by local chromatin context and architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1892593b4d09d954d617fd50a949c2f67aa1bce" target='_blank'>
              Discrete Subdomains Establish Epigenetic Diversity in Subtelomeric Heterochromatin
              </a>
            </td>
          <td>
            Agnisrota Mazumder, John Cooper, Can Goksal, J. S. Khanduja, Richard I. Joh, Johannes J. Groos, Rosario Y.J. Brockhausen, Junko Kanoh, Mohammad R. Motamedi, Ilya J. Finkelstein, Bassem Al-Sady, S. Braun
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cellular senescence is accompanied by extensive genomic reorganization, such as senescence-associated heterochromatin foci and expanded interchromatin compartments, to ultimately affect gene expression. Here, we demonstrate that chromatin structural changes in senescent cells drive significant alterations in the phase behavior and motility of paraspeckles, a type of interchromatin compartment condensate. We observe increased numbers, size, and elongation of paraspeckles harboring NONO and NEAT1_2, driven by elevated levels of those components, consistent with the micellization model of longitudinal growth rather than condensate coalescence. Enhanced paraspeckle motility is associated with HP1α-mediated heterochromatin condensation and interchromatin expansion found in cellular senescence. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03757-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c589595d2d429fd53faabd150c9bf96e091d470b" target='_blank'>
              Imbalanced chromatin distribution in cellular senescence specifies paraspeckle dynamics
              </a>
            </td>
          <td>
            Joonwoo Lee, Jinmi Choi, Jeongeun Park, Seonduk Lee, Eun-Jung Cho, Youngdae Gwon
          </td>
          <td>2025-09-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The circular, double-stranded DNA genomes of Human papillomaviruses (HPV) exist in a nucleosomal state throughout the infectious cycle and rely on host histone epigenetic modifications and chromatin assembly processes to promote various phases of the viral life cycle. Here, we show that the histone H3.3 chaperone HIRA and its associated complex members are recruited to HPV replication factories during the late phase of the HPV life cycle. HIRA is also recruited to HPV replication factories generated by amplification of a replicon with a minimal origin and expression of the viral replication proteins E1 and E2, demonstrating that the E1 and E2 proteins are sufficient for HIRA recruitment. Downregulation of HIRA expression reduces HPV31 DNA amplification and viral transcription in differentiated keratinocytes. Histone H3.3 that is highly phosphorylated on serine residue 31 is also enriched at sites of HPV replication and this modification links the DNA damage response to chromatin that supports rapid gene activation. We propose that deposition of histone H3.3 generates viral minichromosomes that are highly primed to support the late stages of the HPV life cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b34a817df65cd52a7a01714509e40fb97f7ea" target='_blank'>
              Chromatin assembly by the histone chaperone HIRA facilitates Human Papillomavirus replication
              </a>
            </td>
          <td>
            Ashley N. Della Fera, Dan Chen, Alison A. McBride
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03020-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a primary driver of cervical cancer. Integration of HPV into the human genome causes persistent expression of viral oncogenes E6 and E7, which promote carcinogenesis and disrupt host genomic function. However, the impact of integration on host gene expression remains incompletely understood. We used multimodal RNA sequencing, combining total RNA-seq and Cap Analysis of Gene Expression (CAGE), to clarify virus–host interactions after HPV integration. HPV-derived transcripts were detected in 17 of 20 clinical samples. In most specimens, transcriptional start sites (TSSs) showed predominant early promoter usage, and transcript patterns differed with detectable E4 RNA region. Notably, the high RNA expressions of E4 region and viral-human chimeric RNAs were mutually exclusive. Chimeric RNAs were identified in 13 of 17 samples, revealing 16 viral integration sites (ISs). CAGE data revealed two patterns of TSS upregulation centered on the ISs: a two-sided pattern (43.8%) and a one-sided pattern (31.3%). Total RNA-seq showed upregulation of 12 putative cancer-related genes near ISs, including MAGI1-AS1, HAS3, CASC8, BIRC2, and MMP12. These findings indicate that HPV integration drives transcriptional activation near ISs, enhancing expression of adjacent oncogenes. Our study deepens understanding of HPV-induced carcinogenesis and informs precision medicine strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48263c0732371718c3701a40772227c4abc1885c" target='_blank'>
              Elucidating Alterations in Viral and Human Gene Expression Due to Human Papillomavirus Integration by Using Multimodal RNA Sequencing
              </a>
            </td>
          <td>
            Kana Tamai, S. Kinjo, Ayumi Taguchi, K. Nagasaka, Daisuke Yoshimoto, Anh Quynh Duong, Yoko Yamamoto, Hitoshi Iuchi, M. Mori, K. Sone, Michiaki Hamada, Kei Kawana, Kazuho Ikeo, Yasushi Hirota, Yutaka Osuga
          </td>
          <td>2025-10-06</td>
          <td>Viruses</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9f14cca351b80c4eaf8fa50610851b494729d1" target='_blank'>
              Noncanonical chromosomal-end-specific telomeric arrays in naturally telomerase-negative yeasts
              </a>
            </td>
          <td>
            Broňa Brejová, V. Hodorová, Hana Lichancová, A. Gafurov, Dominik Bujna, Filip Brázdovič, Filip Červenák, Tomáš Petrík, Eva Hegedűsová, M. Jakúbková, M. Neboháčová, Ľubomír Tomáška, Matthias Sipiczki, T. Vinař, J. Nosek
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Centromeres, characterized by their unique chromatin attributes, are indispensable for safeguarding genomic stability. Due to their intricate and fragile nature, centromeres are susceptible to chromosomal rearrangements. However, the mechanisms preserving their functional integrity and supporting nucleus homeostasis following breakages remained enigmatic. In this study, we use wheat ditelosomic stocks, which arise from centromere breakage, to explore the genetic and epigenetic alterations in damaged centromeres. Our investigations unveil novel chromosome end structures marked by de novo addition of telomeres, as well as localized chromosomal shattering, including segment deletions and duplications near centromere breakpoints. We reveal that the damaged centromeres possess a remarkable capacity for self-regulation, through employing structural modifications such as expansion, contraction, and neocentromere formation to maintain their functional integrity. Centromere breakage triggers nucleosome remodeling and is accompanied by local transcription changes and chromatin reorganization, and subsequently may contribute to the stabilization of broken chromosomes. Our findings highlight the resilience and adaptability of plant chromosomes in response to centromere breakage, and provide valuable insights into the stability of centromeres, thereby offering promising prospects to manipulate centromeres for targeted chromosomal innovation and crop genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c301bcc4744c9e967b82a2d0c48310497c9c67e9" target='_blank'>
              Adaptation of centromere to breakage through local genomic and epigenomic remodeling in wheat.
              </a>
            </td>
          <td>
            Jingwei Zhou, Yuhong Huang, Huan Ma, Yiqian Chen, Chuanye Chen, Fang Han, H. Su
          </td>
          <td>2025-09-30</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract DNA methylation is a key epigenetic modification underlying cellular identity. Conventional methods based on CpG site-level data often lack sensitivity in detecting low-frequency methylation signals. Here, we present Alpha, a novel method combining unbiased segmentation with robust read-level identification of low frequency cell-type-specific methylation signals. Methylation markers identified by Alpha exhibited significant enrichment in regulatory genomic elements such as enhancers, active promoters, and transcription factor binding sites. In simulated cell-type admixtures, Alpha-derived markers demonstrated improved deconvolution performance, exhibiting lower error metrics compared to beta-value based methods (DSS), even with limited marker numbers (N < 50). We combined Alpha with a non-negative least squares approach (Alpha-NNLS) to enable sensitive detection of circulating tumor DNA (ctDNA) in simulated cell-free DNA from breast and colon cancers, outperforming existing read-level methylation-based tumor fraction estimation methods (CelFEER and UXM). We applied Alpha-NNLS to targeted bisulfite sequencing data from early-stage colon cancer plasma samples and demonstrated strong concordance with existing approaches (R2 = 0.98), supporting its potential for sensitive detection of ctDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b744cb55a0d128bbe65417c922bf8eb9598e2" target='_blank'>
              Read-level DNA methylation deconvolution enhances circulating tumor DNA detection
              </a>
            </td>
          <td>
            Ting Qi, L. Lakshmanan, Yuwei Yang, Ying Zhou, Min Pan, A. J. Skanderup, Qinyu Ge
          </td>
          <td>2025-08-31</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. Methods We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). Results We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Conclusions Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15053-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Major Facilitator Superfamily Domain-containing 12 (MFSD12) has emerged as a critical transmembrane protein with increasingly recognized roles in various cancers. The complex pathogenesis and therapeutic resistance of liver hepatocellular carcinoma (LIHC) present significant clinical challenges. This study investigates MFSD12’s potential involvement in LIHC progression.We performed an extensive pan-cancer analysis of MFSD12 utilizing integrated datasets from The Cancer Genome Atlas (TCGA), the Gene Expression Omnibus (GEO), and the ArrayExpress database. Our investigation focused on evaluating its prognostic significance, clinical implications, associated signaling pathways, immune cell infiltration, gene mutations, and sensitivity to chemotherapeutic agents. Through the application of R and various online analytical tools, our study demonstrated that MFSD12 expression levels were significantly higher in LIHC compared to other cancer types within the TCGA pan-cancer dataset. This finding highlights the specificity of MFSD12 expression in LIHC, a conclusion further validated by immunohistochemical analysis. Survival analysis indicated that this upregulation is associated with unfavorable clinical outcomes. Furthermore, single-cell RNA sequencing revealed that MFSD12 was predominantly expressed in tumor cells and innate lymphoid cells (ILCs) within the tumor microenvironment. Functional vitro studies showed MFSD12-siRNA treatment effectively suppressed LIHC cell proliferation, migration, and invasion. Mechanistically, MFSD12-siRNA enhanced E-cadherin while reducing vimentin, MMP-2, and MMP-9 levels. Further analyses revealed significant associations between MFSD12 expression and immune infiltration, immune checkpoint molecules, tumor mutation burden, and microsatellite instability in LIHC. Notably, MFSD12-siRNA decreased HAVCR2(TIM3) and its ligand galectin-9 (LGALS9) expression in LIHC cells.Our findings demonstrated that MFSD12 upregulation in LIHC strongly correlates with poor prognosis. This association was potentially attributed to MFSD12’s dual roles: promoting tumor cell proliferation, migration, and metastasis while critically modulating the tumor immune microenvironment, particularly through interaction with the HAVCR2/LGALS9 immune checkpoint axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7027a93a5163d82ee6cd2a90832eaa20bb8870bd" target='_blank'>
              MFSD12 promotes proliferation, metastasis and invasion of hepatocellular carcinoma cells and its potential correlation with HAVCR2/LGALS9 immune checkpoint axis
              </a>
            </td>
          <td>
            Kai Sun, Song Wen, Shou-jun Guo, Qingyun Pan, Ke Wang
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromatin organization is important for cell division, epigenetic memory, and gene regulation. It is often reflected in the non-random positioning of centromeres, but the underlying mechanisms and functions remain largely unknown. Here, we demonstrate that asymmetrically dividing Drosophila neural stem cells cluster centromeres near the nuclear envelope, adjacent to the apical centrosome. This asymmetric centromere positioning is regulated through microtubules, originating from the apical centrosome that connect to apical nuclear pore complexes. The minus-end directed motor protein Dynein, its binding partner Mushroom body defect, and Lamin are also required. Asymmetric centromere positioning persists throughout interphase in neural stem cells but is lost in more differentiated progeny. We also reveal that the genes hunchback (hb) and pendulin (pen) occupy specific nuclear regions, correlating with polarized centromere localization. We propose that fly neural stem cells translate their inherent polarity into stereotypical chromatin organization, potentially influencing cell fate decisions and stem cell behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64eb7f82d741bd10286ea05dd86a3dd52c15ebf7" target='_blank'>
              Asymmetric centromere and gene locus positioning in Drosophila neural stem cells
              </a>
            </td>
          <td>
            Jennifer A. Taylor, Clemens Cabernard
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b15ed383fa686f49d7de283de5e0eaf7d1cc0c" target='_blank'>
              A Rare Multipotent Peg-like Epithelial Cell is a Candidate Cell-of-Origin for High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Megan L. Ritting, Wenmei Yang, Syed Mohamed Musheer Aalam, Hui Zhao, Liang Feng, Jianning Song, Mihai G. Dumbrava, W. Ismail, Dong-Gi Mun, Chunling Hu, Opalina Roy, Keyur Chaludiya, Geng-Xian Shi, Daphne Norma Crasta, Kenneth Schaufelberger, Jeffrey R. Janus, Guruprasad Kalthur, S. Weroha, Scott H. Kaufmann, A. Sadanandam, David J. H. F. Knapp, Chen Wang, Akhilesh Pandey, Alexandre Gaspar-Maia, F. Couch, Mark E. Sherman, J. Bakkum-Gamez, Nagarajan Kannan
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>125</td>
        </tr>

        <tr id="


 MTAP encodes a key enzyme in methylthioadenosine (MTA) catabolism. MTAP loss, frequent across human cancers, leads to MTA accumulation and inhibition of PRMT5, a critical regulator of cell cycle and DNA repair. This metabolic vulnerability renders MTAP-deficient tumors selectively sensitive to PRMT5 inhibitors such as TNG-462 and TNG-908. Emerging data indicate that MTAP can also be epigenetically silenced via promoter hypermethylation, independent of genomic deletion, thereby evading detection by DNA-based diagnostics and excluding patients from PRMT5-targeted therapies. Here, we integrate MTAP methylation and expression profiles and evaluate PRMT5 inhibitor sensitivity across diverse tumor types. A comprehensive cohort comprising 6,187 samples with matched DNA methylation and gene expression data was assembled from 17 distinct TCGA projects. Data retrieval, raw RNA-seq read counts, and methylation data analysis were performed using the TCGAbiolinks R package, DESeq2 R package and HM450K arrays, respectively. In addition, we incorporated DNA methylation profiles from established human cell lines obtained from the ENCODE project. In vitro, MTAP-wild type cell lines (human: HCC3153 and HS578T and mouse: GL261); MTAP-null cell lines (human: MDA MB-231, RT4, RT112, UM-UC-3) and MTAP-hypermethylated cell lines (human: U87 and HepG2) were used for viability MTT assays after TNG-462 or TNG-908 treatment. Array assays revealed cg25162921 and cg00230302 probes targeting CpG sites associated with the MTAPlocus. The promoter-proximal probe cg25162921 exhibited a strong inverse correlation with MTAP expression in multiple tumor types, including lung squamous cell carcinoma (TCGA-LUSC; r=-0.48), bladder urothelial carcinoma (TCGA-BLCA; r=-0.42), and glioblastoma multiforme (TCGA-GBM; r=-0.45). We next investigated whether the mechanism of MTAP inactivation, genomic deletion and promoter hypermethylation, influenced cellular sensitivity to PRMT5 inhibition by viability MTT assy. MTAP genomic status and methylation levels were stratified at the promoter-proximal CpG site cg25162921 using DNA methylation from ENCODE project. In viability assays, MTAP-null cell lines or MTAP-hypermethylated models were more sensitive to TNG-462 and TNG-908 with a reduced IC50 compared to MTAP-wild type cell lines. We identify promoter hypermethylation as a distinct mechanism of MTAP silencing, broadening the spectrum of MTAP-low tumors beyond genomic loss. These findings support integration of epigenetic biomarkers to refine patient selection and enhance response rates in PRMT5-targeted clinical trials.



 Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, Esther Cabañas Morafraile, Bernard Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, Juan A Ardura, Balazs Gyorffy, Victor Moreno, Alberto Ocaña. Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de65a469e9e37ed4b177eca66de3a9ae88f86296" target='_blank'>
              Abstract C026: Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers
              </a>
            </td>
          <td>
            Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, E. C. Morafraile, B. Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, J. Ardura, Balázs Győrffy, Víctor Moreno, Alberto Ocaña
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-07</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction: Long interspersed element-1 (L1) and Alu elements are reported to regulate tumor development and progression by forming chimeric transcripts with diverse sequences. While these elements have been implicated in various cancers, the landscape of chimeric transcript formation in breast cancer remains poorly characterized.
Objective: Therefore, in this study, we comprehensively investigated the profiles of L1 and Alu chimeric transcripts in breast tumors by analyzing RNA sequences from The Cancer Genome Atlas (TCGA) database.
Methods: More than 50 billion paired-end RNA sequencing reads from 606 primary breast tumor tissues and 59 matched adjacent normal tissues from the TCGA database were aligned to L1 and Alu consensus sequences with Bowtie2 software and further analyzed.
Results: Our analysis revealed significantly elevated expression levels of L1 and Alu retrotransposons in breast tumor tissues compared to adjacent non-tumor tissues. Importantly, higher expression of L1 and Alu in breast tumors was associated with poorer 10-year overall survival, suggesting their potential as prognostic biomarkers. Further exploration demonstrated that L1 and Alu were extensively expressed in chimeric form with adjacent transcripts. A significant difference in the frequency of 651 L1-mRNA and 1,525 Alu-mRNA chimeric transcripts was observed between non-tumor and tumor tissues. Similarly, 1,009 L1-long non-coding RNA (lncRNA) and 2,575 Alu-lncRNA chimeric transcripts exhibited significantly different frequencies between non-tumor and tumor tissues. Functional enrichment analysis revealed that these differentially expressed chimeric genes played crucial roles in multiple facets of tumorigenesis, including metabolism, signal transduction, immunity, cell cycle regulation, and apoptosis.
Conclusion: Our findings demonstrate that Chimeric transcript formation with L1 and Alu retrotransposons represents a widespread phenomenon in breast cancer that impacts multiple oncogenic pathways. These results significantly expand our understanding of retrotransposon involvement in breast cancer and highlight potential novel targets for diagnostic and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e34ef62da2b088b93dd7d2cad99c0994dc0a0e" target='_blank'>
              Landscape of long interspersed element-1 and Alu chimeric transcripts in breast tumors
              </a>
            </td>
          <td>
            Jian Li, Yin Zhang, Yiting Zhou, Jianyou Liao, Yabin Guo
          </td>
          <td>2025-09-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a83ebfb77b99a082f33dfac7fdc4e834369069a" target='_blank'>
              Gastric epithelium from BRCA1 and BRCA2 carriers harbors increased double-stranded DNA damage and augmented growth
              </a>
            </td>
          <td>
            Kole H. Buckley, Michaela Dungan, Kevin Dinh, Gregory M. Kelly, Ryan Hausler, Kate E. Bennett, Daniel Clay, Julia E. Youngman, Ariana D. Majer, Keely Beyries, Blake A. Niccum, Sydney M. Shaffer, Tatiana A. Karakasheva, Kathryn E. Hamilton, M. Kochman, Gregory G Ginsberg, Nuzhat Ahmad, Kara N Maxwell, Bryson W. Katona
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. Methods We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. Results ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Conclusion Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07216-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Objectives Human RECQL4 and BLM helicases participate in all DNA dependent processes, including replication stress, DNA damage repair. Both helicases are overexpressed in glioblastoma (GBM), a lethal primary brain tumour, characterised by resistance to radio- and chemotherapy. BLM-depleted glioma cells exhibit senescence-associated or polypoid phenotype when exposed to temozolomide (TMZ) and olaparib (OLA), a PARP inhibitor. This study aims to investigate how RECQL4 depletion influences the response of malignant gliomas to chemotherapeutics. Methods We investigated the effect of RECQL4 depletion in glioma cells on cell growth, apoptosis, senescence and polyploidy in the response to combined TMZ and OLA treatment. We compared transcriptomes of RECQL4- and BLM-depleted LN18 and LN229 glioma cells. Drug-induced cytotoxicity, senescence-associated phenotypes, cell cycle alterations, and polyploidy were assessed using the MTT metabolic assay, β-galactosidase activity assay, and propidium iodide staining. Results RECQL4 depletion modestly affected basal glioma cell viability and proliferation, similarly to knock out of the BLM protein. Deletion of RECQL4 in glioma cells (RQ4 KO) induced profound transcriptomic alterations, dissimilar to BLM depletion. RECQL4-depleted glioma cells treated with TMZ and OLA exhibited reduced viability and increased levels of apoptosis markers. The treatment induced cell cycle arrest, however, RQ4 KO cells did not show signs of senescence phenotype or polyploidisation, when compared to BLM KO glioma cells. Interestingly, both RQ4 KO and BLM KO cells were more resistant to WP744, a doxorubicin derivative, when compared to WT LN229 glioma cells. Conclusion Our results highlight the distinct roles of RecQ helicases in a response to chemotherapeutics and support a rationale for targeting RECQL4 as a therapeutic strategy in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14932-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d631e6439cbcbbf960a1d54b75f9eb6885b3c7c" target='_blank'>
              Shared and non-overlapping functions of RECQL4 and BLM helicases in chemotherapeutics-induced glioma cell responses
              </a>
            </td>
          <td>
            K. Wojnicki, B. Wojtaś, I. Ciechomska, B. Kaza, Matthew Guille, Waldemar Priebe, Bozena Kaminska
          </td>
          <td>2025-09-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Neural tumors represent diverse malignancies with distinct molecular profiles and present particular challenges due to the blood-brain barrier, heterogeneous molecular etiology including epigenetic dysregulation, and the affected organ’s critical nature. KCC-07, a selective and blood-brain barrier penetrable MBD2 (methyl CpG binding domain protein 2) inhibitor, can suppress tumor development by inducing p53 signaling, proven only in medulloblastoma. Here we demonstrate KCC-07 treatment’s application to other neural tumors. KCC-07 treatment reduced proliferation rates of U-87MG (glioma cell line) and SH-SY5Y (neuroblastoma cell line). p53 stabilization occurred in these cell lines without significantly affecting programmed cell death factors under KCC-07 exposure. Furthermore, tumor cell growth inhibition was enhanced when combined with DNA damaging reagents. Both phleomycin (radiomimetic agent inducing DNA double strand breaks) and etoposide (topoisomerase II inhibitor inducing DNA double strand breaks) treatment activated p53-dependent signaling for apoptosis and cell cycle arrest, consequently suppressing tumor cell growth. Dual treatment with KCC-07 (epigenetic modifier) and DNA damaging reagents augmented tumor cell suppression, suggesting greater benefits of combinatorial therapy for neural tumors than previously demonstrated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2851b772d55ed5f2efe1f2d0e893e572ba0f756d" target='_blank'>
              KCC-07, MBD2 Inhibitor, Expands the Therapeutic Window of DNA Damage Inducing Reagents in Neural Tumor Cells
              </a>
            </td>
          <td>
            Darom Lee, Junyoung Kim, Keeeun Kim, Youngsoo Lee
          </td>
          <td>2025-08-31</td>
          <td>Experimental Neurobiology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In mammals, repression of germline-specific gene expression is essential for preserving somatic cell identity and preventing disease. Germline gene silencing is often dependent on the presence of promoter 5-cytosine DNA methylation (5meC). Early mammalian development, however, is marked by a dramatic loss of 5meC levels genome-wide. Recent research has highlighted a specialized variant of the Polycomb Repressive Complex 1, PRC1.6, as a key regulator that maintains transient silencing of germline genes in this embryonic window. Eventually, PRC1.6 seems to stimulate the recruitment of de novo DNA methyltransferases (DNMTs), although the precise mechanisms remain to be fully elucidated. Evidence suggests a coordinated epigenetic relay, potentially involving direct protein interactions and shaping the local histone modification landscape. This review explores PRC1.6 as a central epigenetic hub that connects Polycomb repression, histone 3 lysine 9 (H3K9) methylation, and DNA methylation pathways. Unraveling this interplay will provide key insights into the mechanisms that maintain the critical barrier between the germline and the soma, essential not only for proper development but also for preserving somatic cell function and health throughout life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ed478b995bf1b398b0bfe5ba03fe4e97c29208" target='_blank'>
              Epigenetic relay: Polycomb-directed DNA methylation in mammalian development
              </a>
            </td>
          <td>
            Teresa Urli, Maxim V. C. Greenberg
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bcd7b5a2fa30868681ad29d4c2f132679b84407" target='_blank'>
              Site-specific genome engineering of primary human natural killer cells for programmable anti-tumor function
              </a>
            </td>
          <td>
            Vincent Allain, A. Rothrock, Pierre-Louis Bernard, William A. Nyberg, Alexis Talbot, Joseph J. Muldoon, Jing-Yi Chung, Angela To, Christopher R. Chang, Gabriella R. Kimmerly, Chang Liu, T. Tsao, Yasaman Mortazavi, Jin Seo, Zhongmei Li, A. Shemesh, Ralf Schmidt, Carl C Ward, Alex Marson, Lewis L. Lanier, Oscar A. Aguilar, Justin Eyquem
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The submandibular glands (SMGs), along with the parotid and sublingual glands, generate the majority of saliva and play critical roles in maintaining oral and systemic health. Despite their physiological importance, long-term therapeutic options for salivary gland dysfunction remain limited, highlighting the need for a deeper molecular understanding of SMG biology, particularly in humans. To address this knowledge gap, we have performed transcriptomic- and epigenomic-based analyses and molecular characterization of the human SMG. Our integrated analysis of multiorgan RNA-sequencing datasets has identified an SMG-enriched gene expression signature comprising 289 protein-coding and 75 long non-coding RNA (lncRNA) genes that include both known regulators of salivary gland function and several novel candidates ripe for future exploration. To complement these transcriptomic studies, we have generated chromatin immunoprecipitation sequencing (ChIP-seq) datasets of key histone modifications on human SMGs. Our epigenomic analyses have allowed us to identify genome-wide enhancers and super-enhancers that are likely to drive genes and regulatory pathways that are important in human SMG biology. Finally, comparative analysis with mouse and human SMG and other tissue datasets reveals evolutionary conserved gene and regulatory networks, underscoring fundamental mechanisms of salivary gland biology. Collectively, this study offers a valuable knowledge-based resource that can facilitate targeted research on salivary gland dysfunction in human patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d76e2d3fba6c49c8bced153f47a39c20dda0dbf" target='_blank'>
              Multimodal Exploration Offers Novel Insights into the Transcriptomic and Epigenomic Landscape of the Human Submandibular Glands
              </a>
            </td>
          <td>
            E. Horeth, T. Wrynn, Jason M. Osinski, Alexandra Glathar, J. Bard, Mark S. Burke, Saurin Popat, Thom Loree, Michael Nagai, Robert Phillips, J.L. Tapia, Jennifer Frustino, Jill M Kramer, Satrajit Sinha, R. Romano
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background The genetic characteristics of surgically removed cancerous tissues are examined using cancer gene panel testing in cancer genome medicine to detect the pathogenic variants involved in the proliferation and progression of cancer cells. An antitumor drug is prescribed if it directly acts on the detected pathogenic variant; however, some aspects require careful consideration by medical professionals in such cases. The genetic mutations involved in the progression or onset of malignant tumors differ with race. Furthermore, genetic mutations that are variants of unknown significance (VUS) may be involved in the progression or onset of malignant tumors in some races according to the ClinVar results from the National Center for Biotechnology Information. Single nucleotide variations can result in silent mutations or splice sites. Methods We therefore reexamined the CGP results (VUS) of patients suspected of developing hereditary tumors based on their family background using IGV and RT-PCR. Results KRAS Q61K, which is found in many gastrointestinal cancers, was identified as a VUS by ClinVar, but this gene mutation was found to cause splicing. The cancer gene panel test of a 41-year-old male patient with paraganglioma identified succinate dehydrogenase complex, iron-sulfur subunit B (SDHB) G642T as a VUS. However, this mutation was later discovered to cause the splicing site to shift, preventing SDHB from translating from the correct mRNA. In addition, a cancer gene panel test of a 47-year-old patient with right breast cancer determined that breast cancer susceptibility gene 2 (BRCA2) 631 3A>T was a VUS. However, this mutation may create a splicing site, which means that the correct BRCA2 mRNA for BRCA2 is not produced. Conclusions The diagnosis of gene mutations based on the results of cancer gene panel testing may not always be correct, and a detailed examination of gene mutations is necessary. Our medical staff has performed cancer gene panel testing on approximately 5,500 cases of intractable malignant tumors to date and are investigating new treatments for these tumors. In this article, we discuss our experience with cancer gene panel testing as well as the problems encountered and new findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a638c15c6a8b23fbe602cb20d96ff59a3c3e13d" target='_blank'>
              Problems in Cancer Genome Medicine: Base Mutations Cause Intron Start Signals, Resulting in Unexpected Splicing
              </a>
            </td>
          <td>
            Takuma Hayashi, I. Konishi
          </td>
          <td>2025-09-17</td>
          <td>World Journal of Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CTCF is a multifunctional protein that mediates long-range cis-DNA interactions in mammalian genomes. Chromatin architecture governs spatial and functional interactions of gene regulatory elements at various loci and is impacted by the ability of CTCF to restrict cohesin complex dependent chromatin extrusion. In addition, at antigen receptor loci, long-range interactions facilitate spatial proximity of gene segments for VDJ recombination that generates functional genes encoding immunoglobulins and T-cell receptors in developing lymphocytes. To investigate the role of CTCF in VDJ recombination, we mutated CTCF binding sites (CBS) of murine Tcrb locus. Our analysis revealed that CBS interspersed in the domain encompassing variable gene segments (Vb) are not redundant. They exhibit independent but additive effects on dynamic chromatin organization leading to distinct VDJ recombination profiles in CBS mutants depending on positions of mutated CBS relative to Vb segments. Further, inversion of a single CBS drastically altered the chromatin loop organization and VDJ recombination profile. Our results demonstrate the critical importance of chromatin extrusion for generation of chromatin loops for VDJ recombination and underscore its dynamic impediment by CTCF binding at specific points within Vb segment domain to be essential to diversify the usage of Vb segments for VDJ recombination at Tcrb locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac90d7bd76dd65f015613a39e4dba67c4c978f" target='_blank'>
              Site Specific and Orientation Dependent CTCF Binding Determines VDJ Recombination at Murine Tcrb Locus.
              </a>
            </td>
          <td>
            Monika Yadav, Samriddhi Sharma, S. R. Rajalekshmi, Nidhi Bhasin, Jeanne Yimdjo, Karl Pfeifer, M. Srivastava
          </td>
          <td>2025-09-18</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="While endometrioid and mucinous ovarian carcinomas represent nearly a fifth of ovarian cancers, their molecular characteristics and pathologic origins are poorly understood. To identify the genomic and epigenomic alterations characteristic of these ovarian cancer subtypes and to evaluate links to morphologically similar tumors from other sites, we performed a combination of sequence, copy number, mutation signature, and rearrangement analyses from tumor samples and matched normal tissues of 133 patients, as well as methylation analyses of these tumors and tissues of 150 patients from The Cancer Genome Atlas. Genomic analyses included samples from patients with ovarian endometrioid (n=44), ovarian mucinous (n=43), uterine endometrioid (n=15), and gastrointestinal mucinous carcinomas (n=31), including mucinous carcinomas of the stomach, colon, and pancreas. In addition to identifying genes previously known to be involved in these tumors, we identified alterations in RAD51C, NOTCH4, SMARCA1/4 and JAK1 in ovarian endometrioid, ESR1 in uterine endometrioid, and SMARCA4 in ovarian mucinous carcinomas. Whole genome sequencing revealed rearrangements involving PTEN, NF1, and NF2 in ovarian endometrioid carcinomas, and NF1 and MED1 in ovarian mucinous carcinomas. The number of alterations, affected genes and genome-wide methylation profiles were not distinguishable between ovarian and uterine endometrioid carcinomas, supporting the hypothesis that these tumors share a tissue of origin. In contrast, mutation and methylation patterns in ovarian mucinous carcinomas were different from gastrointestinal mucinous carcinomas. These analyses provide insights into the genomic landscapes and origins of mucinous and endometrioid ovarian carcinomas, providing new avenues for early clinical intervention and management of patients with these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c6457a2db7e3ebae9f1aae5ab7a9dc09208e54" target='_blank'>
              Genomic landscapes of endometrioid and mucinous ovarian cancers and morphologically similar tumor types.
              </a>
            </td>
          <td>
            Dorothy Hallberg, A. Eastman, Shashikant Koul, D. C. Bruhm, E. Papp, Simon Davenport, V. Adleff, Leonardo Ferreira, N. Niknafs, J. Medina, S. Cristiano, C. Hruban, J. Fiksel, Kaui P Lebarbenchon, Luis Aparicio, Nicholas A. Vulpescu, Kuan-Ting Kuo, Nita Ahuja, Ronny Drapkin, Euihye Jung, Sarah H Kim, Mark A. Eckert, E. Lengyel, Kentaro Nakayama, Ayse Ayhan, I.-M. Shih, Tian-Li Wang, O. Lavie, G. Rennert, H. Easwaran, Stephen B Baylin, Michael F Press, V. Velculescu, R. Scharpf
          </td>
          <td>2025-09-22</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="Abstract The O6-methylguanine-DNA methyltransferase (MGMT) plays a significant role in the pathogenesis and progression of glioma. Numerous enhancer variants, including those within the MGMT gene region and adjacent gene regions, have been found to be associated with cancer development and progression. We investigated the significance of enhancer variants located in the intergenic spacer far from the MGMT gene in relation to glioma susceptibility and progression. We recruited 402 glioma patients and 654 controls for this investigation using Sequenom MassARRAY genotyping. We identified a significantly elevated risk of glioma among carriers with the rs11016629 TG genotype compared to those with the GG genotype (OR = 1.41, 95% CI 1.03–1.93; P = 0.034). Subgroup analyses revealed that rs11016629 was significantly associated with glioma risk in subjects with WHO grade IV tumor (OR = 1.59, 95% CI 1.07–2.38; P = 0.023) and high-grade glioma (OR = 1.57, 95% CI 1.11–2.21; P = 0.011). Patients who underwent gross total resection with TG/TT genotypes exhibited a 2.66-fold higher risk of disease progression than GG carriers (HR = 2.66, 95% CI 1.23–5.79; P = 0.014). The study demonstrates that a MGMT enhancer variant rs11016629 contributes to both glioma susceptibility and progression. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s10571-025-01605-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ffd3767daacb020b869e9275c5c5550f74b90c" target='_blank'>
              A MGMT Enhancer Variant is Associated with Glioma Susceptibility and Progression
              </a>
            </td>
          <td>
            Shu Zhang, Yiji Chen, Wenshen Xu, Danfang Yan, Meiqin Chen, Jiayi Zheng, Liming Huang
          </td>
          <td>2025-10-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9a242ccc6ebd9d79ab2d3fe8d1c436d8e52055" target='_blank'>
              Enabling whole-genome DNA methylation-based classification of central nervous system tumors
              </a>
            </td>
          <td>
            Jonghoon Lee, Young Seok Ju, B.-L. Oh, S. Choi, Se Hoon Kim, Ji Won Lee, J. Phi, Y. Jung
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Mutations in RNA-binding proteins are increasingly identified in cancers through tumor sequencing and are correlated with disease progression, therapy response, and overall patient outcomes, underscoring the need to study them. Here, we focus on the RNA-binding protein Poly-C binding protein 1 (PCBP1), which binds target RNAs through K-homology (KH) domains to regulate RNA fate. PCBP1 is a tumor suppressor gene and hotspot missense mutations at leucine residues 100 and 102 are observed in colorectal cancer (CRC). PCBP1 mutations have been recurrently reported in CRC genome-wide mutation studies and are associated with poor clinical outcomes; however, their effects on PCBP1 expression and function remain largely unexplored. We show that cancer-associated mutations substituting leucine 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover. The L100/L102 residues occur at the interface of the RNA-binding KH1 and KH2 domains, and our molecular dynamics simulations show that mutations at these residues disrupt the secondary structure of PCBP1. Additionally, these mutants display increased cytoplasmic localization. Importantly, mutant PCBP1 physically interacts with wild type PCBP1 and suppresses its expression through a dominant-negative mechanism. Together, our data demonstrate that CRC-associated PCBP1 mutations destabilize the protein and act as dominant-negative variants, revealing a novel mechanism of tumor suppressor inactivation in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a7a507c9272e7ca6986d3ffe683c348245fff39" target='_blank'>
              Colorectal cancer-associated PCBP1 mutations disrupt protein stability in a dominant negative manner
              </a>
            </td>
          <td>
            Paige V. Blinkiewicz, Nicole C. Hoenes, Myra X. Afzal, Yihong Liu, Ulysses J. Hill, B. Bjarnadóttir, T. E. Burgin, P. Malaney
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b73f806f958dd1db9fdcf8f058c71382d2407" target='_blank'>
              Antagonistic histone H2A variants and autonomous heterochromatin formation shape epigenomic patterns in Arabidopsis
              </a>
            </td>
          <td>
            Shoko Oda, Sayaka Tominaga, Shumpei Takeuchi, A. Osakabe, Akira Kawabe, Frédéric Berger, T. Kakutani, T. To
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Transcription factor E3 (TFE3) oncofusions are frequently detected in the Microphthalmia transcription factor (MiT) family translocation renal cell carcinoma (tRCC), a rare pediatric renal cancer with limited treatment options. The mechanisms by which TFE3 oncofusions promote tRCC malignancy remain inadequately defined. Here, we demonstrate that the RNA-binding capability conferred by TFE3 fusion partners drives the formation of TFE3 condensates. This further enables TFE3 oncofusions to co-condensate with RNA polymerase II (RNAPII) and other RNA-binding proteins, such as paraspeckle component 1 (PSPC1), ultimately driving the formation of transcriptional hubs to promote pro-oncogenic transcription. Dissolution of oncofusion condensates through nanobody-based chemogenetic manipulation effectively curtails tRCC cell growth both in vitro and in vivo, suggesting the therapeutic potential for targeting oncofusion condensation in tRCC. Collectively, our study establishes the causal role of RNA and RNA-binding proteins in facilitating oncofusion condensation to promote renal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79cdc6e21b779dea89a691271b52beca3b734a9" target='_blank'>
              RNA-mediated condensation of TFE3 oncofusions facilitates transcriptional hub formation to promote translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Lei Guo, Rongjie Zhao, Yi-Tsang Lee, Junhua Huang, James Wengler, Logan Rivera, Tingting Hong, Tianlu Wang, Kunjal Rathod, Ashley Suris, Yitian Wu, Xiaoli Cai, Rui Wang, Yubin Zhou, Yun Huang
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. Methods We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. Results EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. Conclusion This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03631-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Waleed K. Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Tumor metastasis is a major contributor for mortality in patients with pancreatic ductal adenocarcinoma (PDAC). While circular RNAs (circRNAs) have emerged as pivotal regulators of tumor progression and metastasis, their functional roles in PDAC remain poorly understood. Through comprehensive circRNA profiling of 10 pairs of PDAC tumors and adjacent normal tissues, we identified circPRKD3 (hsa_circ_0000992) as being substantially up-regulated in malignant specimens. Functional characterization demonstrated that circPRKD3 overexpression potently enhanced cellular migration, invasion, and metastatic capacity in vitro and in vivo, whereas its knockdown produced opposite phenotypic effects. Mechanistic investigations revealed that circPRKD3 directly interacted with the oncogenic RNA-binding protein Musashi-2, protecting it from β-TRCP-mediated ubiquitination and subsequent proteasomal degradation. Clinical correlation analysis revealed a close association between elevated circPRKD3 expression and shorter survival of PDAC patients. Notably, we validated the translational potential of circPRKD3 as a liquid biopsy marker, showing that serum detection, when combined with conventional biomarkers (CA19-9, CEA, and CA125), dramatically improved diagnostic performance. These findings not only delineate a novel circRNA-mediated regulatory axis in PDAC metastasis but also identify circPRKD3 as a promising diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1441280c9bbaab596bfcee43907f7ebe16003d78" target='_blank'>
              A Circular RNA Promotes Tumor Metastasis through Stabilizing MSI2 Protein in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Zhang Li, Lanyang Gao, Yang Yang, Yue Ming, Wenrong Liu, Tingting Zhang, Zixia Ye, Fuyan Xu, Juan He, Jiao Li, Jiawei Guo, Xiaojuan Yang, Qing Zhu, Yong Peng
          </td>
          <td>2025-09-10</td>
          <td>Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5723caf7254adf5d6ab924c41ca2fe7255ce2a" target='_blank'>
              Structure of Bacillus subtilis Ku-mediated DNA synaptic complex.
              </a>
            </td>
          <td>
            Whan-Jong Kim, Jieun Kim, Mingyu Jo, Youngjin Kim, Min-Sung Kim
          </td>
          <td>2025-10-21</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Our previous work demonstrated that oncogenic KRAS mutations in pancreatic cancer induce aberrant transcription-replication conflicts (TRCs). TRCs arise when DNA replication forks collide with transcription complexes due to loss of spatial and temporal coordination between these processes. TRC resolution mechanisms are and adaptive dependency to KRAS-mediated oncogenic stress. Here we identify a key pathway that regulates TRCs in PDAC and performs mechanistic studies towards clinical translation.



 TRC factors were identified through an siRNA screen against 200 DNA repair genes in Panc1 cells using a high-throughput proximity ligation assay (HiPLA). Knock-out and complementation experiments were performed using CRISPR-Cas9 systems. Cell proliferation was measured using incucyte. Replication stress and DNA damage were measured using multiple orthogonal methods. Protein-protein interactions were assessed by co-immunoprecipitation mass spectrometry and validated by western blot. Chromatin occupancy of key factors was determined using ChIP-seq.



 Through high-throughput proximity ligation assay (HiPLA) screening of 200 known DDR genes, we identified the BER pathway as a key mechanism for TRC resolution. Among these, Neil2 - a bifunctional glycosylase in the base excision repair (BER) pathway - emerged as one of the highest-scoring BER factors; and the only BER glycosylase involved—likely due to its transcription-coupled repair activity. Using CRISPR/Cas9-mediated knockdown, we found that loss of Neil2 led to elevated double-stranded DNA breaks (DSBs) and replication stress. Notably, this significantly impaired cell cycle progression. colony formation, and proliferation in pancreatic cancer cell lines, with minimal effects on normal pancreatic epithelial cells, highlighting Neil2’s selective oncogenic function. To elucidate the underlying mechanisms, we combined proteomic approaches—including co-immunoprecipitation and iPOND-mass spectrometry—with transcriptomic and chromatin profiling with ChIP-seq and RNAseq. Our data revealed that Neil2 plays a critical role in transcription termination, and that its dysregulation promotes RNA Polymerase II readthrough, likely contributing to R-loop accumulation, downstream TRCs, and subsequent genotoxicity. Intriguingly, deletion of Neil2’s DNA binding domain—which abolishes its DNA lesion binding capability—did not compromise its regulation of TRCs or support of cancer cell viability. This suggests that Neil2’s oncogenic role in pancreatic cancer may be DNA-binding independent.



 We uncovered a noncanonical and essential function of Neil2 in TRC regulation and oncogenic maintenance in pancreatic cancer, and we propose Neil2 as a promising therapeutic target for selective intervention in this disease.



 Fan Meng, Mustafa Raoof. Neil2 is a potential theratputic targert in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d45ba09d3e2c9d65e7a73127234b8bec15900c3d" target='_blank'>
              Abstract A041: Neil2 is a potential theratputic targert in pancreatic cancer
              </a>
            </td>
          <td>
            Fan Meng, Mustafa Raoof
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Four decades ago, the discovery of centromere protein-A (CENP-A) marked a pivotal breakthrough in chromosome biology, revealing the epigenetic foundation of centromere identity. CENP-A, a histone H3 variant, directs the formation of the microtubule-binding kinetochore complex, designating the chromosomal site for its assembly and underpins the accurate partitioning of genetic material during cell division. Errors in cell division can give rise to DNA instability and aneuploidy, implicated in human diseases such as cancer. Therefore, discovering the underlying pathways and mechanisms responsible for the formation, regulation and maintenance of the centromere is important to our understanding of genome stability, epigenetic inheritance, and in providing the knowledge to help generate possible treatments and therapeutics. Here, we review various molecular pathways and mechanisms implicated in maintaining centromere identity and highlight some of the key outstanding questions with a focus on the human centromere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481df10bbcc98403846b3169d8fe656f17c65c42" target='_blank'>
              Preserving centromere identity: right amounts of CENP-A at the right place and time
              </a>
            </td>
          <td>
            Zofia Pukało, Bethan Medina-Pritchard, M. A. Abad, A. A. Jeyaprakash
          </td>
          <td>2025-09-30</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="NUPTs are DNA sequences of plastid origin present in plant nuclear genomes to varying, though typically low, amounts. It is assumed that they are continuously formed and, due to their potentially mutagenic effect, removed at a constant turnover rate, which should result in an exponential decay of their age distributions and a negative correlation between age and size. However, these assumptions are based on analysis from a limited number of species and have never been explicitly tested. To gain insight into the mechanisms driving the origin and evolution of NUPTs, here we surveyed the plastid and nuclear genomes of 30 species representing the main angiosperm (flowering plants) lineages. By modeling the distribution of ages and sizes, examining their linear arrangement across the plastid genome, and statistically assessing spatial biases with respect to other genomic features, we showed that NUPTs are i) formed by both continuous and episodic mechanisms; ii) unevenly represented across the plastid genome; iii) consistently associated with certain classes of RNA genes, in particular rRNA, tRNA and regulatory RNA genes; iv) differentially contributing to structural genes; and v) closer than expected to different superfamilies of transposons in a species-specific manner. Our results reveal the unexpected complexity in the mechanisms driving the origin of NUPTs, which do not only involve their continuous formation but also episodic, highlight their role as a major source of non-coding RNA genes and other genomic features and provide a more complete picture of the different drivers of evolutionary change at the genome level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cddf52eeced1161dba2e2d7fa0a44d7221d0674f" target='_blank'>
              Episodic and ongoing mechanisms drive plastid-derived nuclear DNA evolution in angiosperms.
              </a>
            </td>
          <td>
            Juan Pablo Marczuk-Rojas, Ana D Maldonado, Lorenzo Carretero-Paulet
          </td>
          <td>2025-10-13</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34db9ef54149ec0403bbd81c404fd10ca8b5442e" target='_blank'>
              Integrative Transcriptomic Analysis Identifies Hypoxia-Responsive Cell Cycle Hub Genes as Prognostic Markers in Glioblastoma
              </a>
            </td>
          <td>
            Manish K. Sharma, Jonita Chongtham, Ashish Bhushan, K. Chosdol, Subrata Sinha, Tapasya Srivastava
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Each cell type exhibits a unique chromatin organization and transcriptome that define its identity and function. Chromatin architectural proteins like CTCF and the cohesin complex are pivotal in shaping this regulatory landscape. Understanding how they drive cell-type-specific gene expression is essential for decoding complex traits and designing targeted therapies for diseases rooted in cellular identity. Endothelial cells (ECs), which line blood vessels and maintain vascular homeostasis, are key regulators of blood pressure (BP) and possess distinct transcriptional programs and
 cis
 -regulatory elements governed by chromatin architecture.

 We hypothesized that CTCF and RAD21 (a cohesin subunit) play specialized roles in establishing the EC-specific transcriptome by modulating chromatin organization. To test this hypothesis, we depleted CTCF and RAD21 in human induced pluripotent stem cells (iPSCs) and iPSC-derived ECs (iECs), representing undifferentiated and lineage-committed states, respectively. We performed RNA sequencing to assess transcriptional changes and CUT&Tag to profile CTCF and RAD21 binding in both cell types.
 Transcriptomic analysis revealed that both factors regulate EC-specific gene expression through shared and distinct pathways. Consistent with these transcriptional changes, CTCF or RAD21 depletion in iECs led to prominent loss of binding at promoters, suggesting direct regulatory roles, whereas in iPSCs, loss occurred mainly at intronic regions, indicating broader structural functions in the undifferentiated state. We also identified a potential EC-specific association between RAD21 and the ETS transcription factor FLI1, suggesting a novel regulatory axis in vascular gene control. Supporting these distinct roles, integration with Micro-C data showed that CTCF- and RAD21-sensitive loops in iECs were largely promoter-centric, whereas in iPSCs, RAD21-sensitive loops engaged more with non-promoter regions. Notably, we identified disrupted CTCF/Rad21 binding promoter regions that harbor SNPs associated with BP and vascular traits, linking chromatin architecture to noncoding genetic risk.
 In conclusion, our study reveals distinct yet cooperative roles of CTCF and RAD21 in shaping EC-specific gene regulation. These findings provide insight into chromatin-mediated vascular identity and highlight regulatory loci with therapeutic relevance to hypertension and vascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ee78ddac4277d3f74eaac1bb7823a42a27b02a" target='_blank'>
              Abstract TH192: CTCF and Cohesin Mediated Endothelial Cell-specific Chromatin Organization and Transcriptional Control at Loci Harboring Blood Pressure-Associated Genetic Variants
              </a>
            </td>
          <td>
            Atrayee Ray, Rajan Pandey, Josiah Murray, C. Stelloh, Pengyuan Liu, Yong Liu, Qiongzi Qiu, Paul L. Auer, Michael E. Widlansky, A. Geurts, Allen W Cowley, A. Kwitek, Andrew S. Greene, Mingyu Liang, Sridhar Rao
          </td>
          <td>2025-09-01</td>
          <td>Hypertension</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e634454da0093b29671a652ab0cbf1f25b766878" target='_blank'>
              Integrated Genomic and Methylome Profiling Reveals Promoter Repression and Age-Linked CpGs in the California Mussel
              </a>
            </td>
          <td>
            L. Hiebert, Ava Soesbe, Tzung-Fu Hsieh, Qirui Cui, Soojin V Yi
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="



Oncohistones represent a distinct class of cancer-driving mutations affecting core histone proteins, particularly histone H3 variants. Unlike conventional oncogenes and tumor suppressors, these single amino acid substitutions exert their oncogenic effects through widespread disruption of chromatin architecture and epigenetic regulation. This narrative review examines the molecular mechanisms underlying oncohistone function, with particular focus on H3K27M mutations found in pediatric diffuse midline gliomas and H3K36M mutations characteristic of chondroblastomas. We explore how these mutations interfere with normal chromatin-modifying enzymes, including polycomb repressive complex 2 (PRC2) and SETD2 methyltransferase, leading to aberrant gene expression patterns that promote tumorigenesis. The review discusses the clinical significance of oncohistone mutations in cancer diagnosis and classification, highlighting their integration into current diagnostic frameworks. Additionally, we examine emerging therapeutic approaches targeting oncohistone-driven cancers, including epigenetic modulators, immunotherapeutic strategies, and novel synthetic lethality approaches. Finally, we address current challenges in the field, including therapeutic resistance mechanisms and the complexity of targeting global epigenetic alterations, while identifying key areas for future research to advance our understanding and treatment of oncohistone-driven malignancies. 



">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4872fffa3cd46351e5594106479bc2c6f61544" target='_blank'>
              Oncohistones in Cancer: A Narrative Review of Molecular Mechanisms and Targeted Therapeutic Strategies
              </a>
            </td>
          <td>
            A. R. Al-Zaalan, Hussam Saadi Aziz
          </td>
          <td>2025-10-03</td>
          <td>European Journal of Medical and Health Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5ee3b9a577273caee9d178991970953321a673" target='_blank'>
              Integrated Genomic Profiling Reveals Mechanisms of Broad Drug Resistance and Opportunities for Phenotypic Reprogramming in Cancer Cells
              </a>
            </td>
          <td>
            Ian Mersich, Brian S. J. Blagg, Aktar Ali
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Alterations in the mitochondrial genome (mtDNA) have been shown to be key in cancer development and could be useful as biomarkers for diagnosis, prognosis, and treatment. To identify mtDNA variants associated with breast cancer, we analyzed the whole mtDNA sequence from paired tissues (tumor–peripheral blood) of women with this malignancy and from peripheral blood samples of healthy women. The mtDNA mutational landscape, heteroplasmy levels of the variants, and mitochondrial ancestry were established. Comparative analysis between cases and controls revealed significant differences in the number and location of variants, as well as in the heteroplasmy levels. Cases showed higher mutation number in MT-ND5, tRNAs, and rRNAs genes; increased proportion of missense variants; and elevated mtDNA content, than controls. Notably, a high blood mtDNA mutational burden (OR = 3.83, CI: 1.89–7.95, p = 5.3 × 10−5) and five mtDNA variants showed association with the risk of breast cancer. Furthermore, a low tumor mutational burden (HR = 7.82, CI: 1.0–63.6, p = 0.05) and the haplogroup L (HR = 12.16, CI: 2.0–72.8, p = 0.0062) were associated with decreased overall and disease-free survival, respectively. Our study adds evidence of the potential usefulness of mtDNA variants as risk and prognosis biomarkers for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b89daf629cb05af3adbfba97c1f371049ff76ce" target='_blank'>
              Mitogenomic Alterations in Breast Cancer: Identification of Potential Biomarkers of Risk and Prognosis
              </a>
            </td>
          <td>
            C. Pérez-Amado, Amellalli Bazan-Cordoba, L. Gómez-Romero, Julian Ramírez-Bello, V. Bautista-Piña, A. Tenorio-Torres, E. Ruvalcaba-Limón, Felipe Villegas-Carlos, Diana Karen Mendiola-Soto, A. Hidalgo-Miranda, S. Jiménez-Morales
          </td>
          <td>2025-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662898df81b718780224755f0765bfbdce810fd9" target='_blank'>
              Low nucleus to cytoplasmic ratio compromises cytokinesis, genome stability and development in haploid embryonic cells
              </a>
            </td>
          <td>
            M. Zuccaro, Daniela Georgieva, Shuangyi Xu, Matthew Dennen, Qian Du, Gloryn Chia, Selma Amrane, Lisa C. Grossman, Alejandro De Los Angeles, Robert Prosser, R. Lobo, Dieter Egli
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The proposed existence of a widespread “tumor microbiome” has sparked considerable interest, yet recent evidence suggests many microbial DNA signals in cancer genomes arise from contamination rather than true colonization. A September 2025 Science Translational Medicine analysis of over 5,700 cancer genomes demonstrated that most detected sequences reflected artifacts, underscoring earlier concerns about low-biomass sequencing studies. While enthusiasm for a universal tumor microbiome must be tempered, robust associations remain in specific cancers, including Helicobacter pylori in gastric carcinoma, Fusobacterium nucleatum in colorectal cancer, and human papillomavirus in cervical malignancies. This reassessment highlights the urgent need for rigorous methodological standards, such as robust negative controls, contamination-aware pipelines, and transparent reporting, to ensure reproducibility. Rather than discouragement, this represents an opportunity to refocus research on biologically plausible, clinically relevant cancer-microbe interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1b7d3da6d9e90a3a12169f914f9edbbd5cf84a" target='_blank'>
              Reassessment of microbial DNA in cancer genomes: addressing contamination controversies
              </a>
            </td>
          <td>
            Muhammad Shahid Mehmood, Fatima Hajj
          </td>
          <td>2025-10-13</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis. Significance In this study we uncovered a novel regulatory link between ADAR1 and the METTL-family of RNA methyltransferases in hematopoietic stem cells. Overexpression of ADAR1 uniquely upregulated METTL13 while suppressing other METTL genes. Loss of orphan gene METTL13 affected proliferation, apoptosis and p53 signaling in hematopoietic stem cells. Furthermore, loss of METTL13 suppressed cell proliferation and survival in pediatric T-cell acute lymphoblastic leukemia. Our findings suggest a potential role for METTL13 in pre-leukemia transformation and oncogenic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba22ede5cb07736eac6d115a03c89fcfe2e86a6e" target='_blank'>
              Evolutionary Variation of Poxvirus Genome Architecture
              </a>
            </td>
          <td>
            J. Enow, Arkaidy Garber, Betram Jacobs, G. McFadden, Masmudur M. Rahman
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Cellular organisms store heritable information in two forms, genetic and epigenetic, the latter being largely dependent on cytosine methylation (5mC). Chargaff’s Second Parity Rule (CSPR) describes the nucleotide composition of cellular genomes in terms of intra-strand DNA symmetry. However, it remains unknown whether DNA methylation patterns display intra-strand DNA symmetry. Computational analysis was conducted of the DNA methylation patterns observed in human cell lines and in tissue samples from healthy donors. Analysis of 5mC marks in mutually reverse-complementary pairs of short oligomers, containing CpG dinucleotide in the middle, revealed deviations from CSPR and methylation asymmetry that can be observed for two non-overlapping mirror groups defined by CpG methylation values. Deviations from CSPR, together with combinatorial probabilities of pattern distributions and computer simulations, highlight the non-random nature of methylation processes and enabled us to identify specific cell types as outliers. Further analysis revealed a compensatory methylation asymmetry that reduces deviations from intra-strand symmetry and implies the existence of strand-specific methylation during cell differentiation. Among six pairs of reverse-complementary tetranucleotides, four pairs with specific sequence motifs display pronounced methylation asymmetry. This mirror asymmetry may be associated with chromosome folding and the formation of a complex three-dimensional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387c52e696253beba52f792a823e42cb5be6a863" target='_blank'>
              Study of the Patterns of DNA Methylation in Human Cells Through the Prism of Intra-Strand DNA Symmetry
              </a>
            </td>
          <td>
            Z. Ramazanova, Aizhan Alikul, D.A. Begimbetova, S. Taipakova, Bakhyt Matkarimov, Murat K. Saparbaev
          </td>
          <td>2025-09-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Estrogen receptor alpha (ERα)-positive (ER+) breast cancers are driven by 17β-estradiol (E2) binding to ERα, which transcriptionally regulates downstream target genes. Although microarrays and conventional RNA sequencing have identified E2 target genes, pre-designed probes and short read lengths are limited in accurately capturing complex transcript structures. Long-read RNA sequencing offers a solution by spanning entire transcripts, providing a more complete view of the transcriptome. Here, we employed nanopore long-read direct RNA sequencing (DRS) complemented with 3’-end sequencing, in vitro experiments, and deep learning-based protein modeling to explore the intricate landscape of E2-responsive transcriptome and protein level implications. Our analysis revealed a range of E2-responsive non-coding and coding isoforms, including intronically polyadenylated (IPA) mRNAs. One of these IPA isoforms was detected for TLE1 (Transducin-like enhancer protein 1), which positively assists ERα-chromatin interactions for a subset of E2 target genes. The IPA isoform produces a C-terminus truncated protein, lacking the WDR interaction domain, but retains dimerization/tetramerization capacity through its intact N-terminus Q-domain. Structural modeling and protein-based assays confirmed the truncated protein’s dimerization potential and nuclear localization. Functional assays showed that overexpression of truncated TLE1 reduces the E2-induced upregulation of GREB1, an E2-responsive gene, thereby disrupting transcriptional regulation. Importantly, a lower IPA isoform ratio is associated with worse survival in ER+ patients, highlighting clinical relevance. Our study uncovers new layers of complexity in the E2-regulated transcriptome, providing insights into truncated proteins. These findings contribute to a deeper understanding of gene regulation and may help the development of new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0bac48baf3ce0314e9eaa5b424b9e568bd79fa" target='_blank'>
              E2-regulated transcriptome complexity revealed by long-read direct RNA sequencing: from isoform discovery to truncated proteins
              </a>
            </td>
          <td>
            Didem Naz Dioken, Ibrahim Ozgul, Irem Yilmazbilek, Elanur Almeric, I. C. Eroglu, Mustafa Cicek, Utku Cem Yilmaz, Deniz Karagozoglu, Esra Çiçek, E. Karaca, Tolga Can, A. E. Erson-Bensan
          </td>
          <td>2025-08-02</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract BRCA1 (breast cancer-associated protein 1) plays a central role in homologous recombination (HR) through interactions with multiple proteins across its various domains. The C-terminal BRCT domains bind HR regulators such as ABRAXAS1, CtIP, and BRIP1, each contributing to distinct, sometimes opposing, functions. While pathogenic mutations frequently cluster within the canonical BRCT phospho-binding pocket, the broader mutational landscape and its functional consequences remain poorly understood. Here, we used a site-saturation mutagenesis library of the BRCT domains to test >4,000 single-residue variants for their ability to bind ABRAXAS1 and CtIP. Using a yeast two-hybrid screen, we systematically assessed these interactions and validated key findings in mammalian cells. The resulting interaction map identified previously uncharacterized residues critical for partner binding and demonstrated their detrimental impact on HR. Importantly, we identified separation-of-function mutations that selectively disrupt individual protein interactions, enabling a more detailed analysis of each partner’s contribution to HR. Functional assays suggested that disruption of CtIP binding had the most pronounced impact on HR. Furthermore, integration of our data with clinical variant data revealed a strong correlation between loss of protein binding and pathogenicity, highlighting the potential utility of our interaction map for clinical variant interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c047e85b483d99dee8271604efee8cc044f9d733" target='_blank'>
              Profiling BRCA1-BRCT interactions and their functional relevance at amino acid resolution
              </a>
            </td>
          <td>
            Venda Mangkusaputra, A. G. Murachelli, Zhengzhou Yu, Annouche den Hollander, R. Menafra, Anne Schreuder, S. Kloet, T. Sixma, Sylvie M. Noordermeer
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03559-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains one of the leading causes of cancer-related mortality worldwide due to its aggressive nature and high metastatic potential. Circular RNAs (circRNAs), characterized by covalently closed loop structures and high molecular stability, have emerged as important regulatory molecules in cancer biology. However, the biological functions and molecular mechanisms of most circRNAs in GC remain largely unexplored. Methods We conducted circRNA microarray profiling on three pairs of primary GC tissues and matched adjacent normal tissues to identify differentially expressed circRNAs. The expression of circ-ANKRD36C was validated by quantitative reverse transcription polymerase chain reaction (qRT-PCR) in additional clinical specimens. Functional assays were performed to assess the effects of circ-ANKRD36C knockdown on GC cell proliferation, migration, and invasion. Dual-luciferase reporter assays provided evidence of direct interaction between circ-ANKRD36C and miR-15a-5p. Clinicopathological correlations and survival outcomes were analyzed in 84 paired GC and normal tissue samples. Results Circ-ANKRD36C was significantly upregulated in GC tissues. Knockdown of circ-ANKRD36C suppressed GC cell proliferation, migration, and invasion in vitro. Dual-luciferase assays supported its direct interaction with miR-15a-5p, potentially influencing oncogenic targets such as BMI1 and CCND1. Clinically, high circ-ANKRD36C expression was associated with shorter overall survival (OS). Multivariate Cox regression analysis suggested its independent prognostic value (hazard ratio [HR] = 2.41; 95% confidence interval [CI] = 1.22–4.77; P = 0.011). Conclusion Circ-ANKRD36C is upregulated in gastric cancer and may serve as a prognostic biomarker. Functional evidence suggests it could act as a candidate regulator, but its mechanistic role requires further validation. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-04287-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec96a53ffb97eb8a42c6e93ec17c0d2fe31bed9" target='_blank'>
              Circular RNA profiling and functional analysis implicate circ-ANKRD36C as a potential prognostic biomarker in gastric cancer
              </a>
            </td>
          <td>
            Rongpeng Chen, Tianxing Ji, Yuanqing Chu, Feng Xiao, Zhicheng Feng, Guoqiang Wang
          </td>
          <td>2025-09-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c93dca3eb5be61d1472406ead880481fa79563f" target='_blank'>
              Genetic diversity and evolution of rice centromeres.
              </a>
            </td>
          <td>
            Lingjuan Xie, Yujie Huang, Wei Huang, L. Shang, Yanqing Sun, Quanyu Chen, Shuangtian Bi, Mingyu Suo, Shiyu Zhang, Chentao Yang, Xiaoming Zheng, Weiwei Jin, Qian Qian, Longjiang Fan, Dongya Wu
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1– 3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Objective: Prostate cancer, a leading cause of cancer deaths in men, involves genetic alterations and DNA damage. Oxidative stress–induced DNA damage is crucial in its development, necessitating efficient DNA repair mechanisms. Base excision repair (BER), a key pathway, addresses oxidative DNA damage with enzymes like Apurinic/Apyrimidinic Endonuclease 1 (APE1). Recent findings show that altered APE1 expression disrupts BER balance, contributing to genomic instability and cancer progression. This study examines APE1 expression in prostate cancer tissues to clarify its role in cancer development and therapeutic implications. Assessing APE1 expression in prostate cancer may elucidate its role in tumor progression, therapeutic resistance, and genomic instability, offering deeper insights into the mechanistic relevance of the BER pathway in oncogenesis. Methods: A total of 50 male patients diagnosed with prostate cancer were included in this study. Tumor and adjacent normal tissue samples were collected from consecutive male patients who underwent Tru-cut prostate biopsies due to clinical suspicion of prostate cancer, and gene expression levels were analyzed using real-time polymerase chain reaction. Results: The results showed no significant difference in APE1 expression between tumor and normal tissues (P > .05). Clinical factors like smoking, age, diabetes, hypertension, PIRADS, and Gleason scores did not significantly correlate with APE1 expression. However, a statistically significant correlation was observed between PSA levels and APE1 expression (r: 0.287, P = .04). Conclusion: The authors’ findings suggest that while no substantial changes in APE1 expression were observed in prostate cancer tissues, the enzyme’s role in DNA repair remains crucial. Abnormalities in APE1 expression and function can contribute to genomic instability and cancer progression. Future studies with larger sample sizes and standardized methodologies are necessary to better understand the potential role of APE1 as a biomarker and therapeutic target in prostate cancer.Cite this article as: Rouhi V, Habibi S, Baykal Koca S, Köseoğlu H, Güven M. Investigation of apurinic/apyrimidinic endonuclease 1 expression changes in prostate cancer. Cerrahpaşa Med J 2025, 49, 0009, doi: 10.5152/cjm.2025.25009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5a4b0fb3242b36fc3de3889e7200ec9f3f9fad" target='_blank'>
              Investigation of Apurinic/Apyrimidinic Endonuclease 1 Expression Changes in Prostate Cancer
              </a>
            </td>
          <td>
            Vahid Rouhi, Sanaz Habibi, Sevim Baykal Koca, Hikmet Köseoğlu, Mehmet Güven
          </td>
          <td>2025-08-29</td>
          <td>Cerrahpaşa Medical Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c52f42712d52bb1c07df522d1d35f4efd725478" target='_blank'>
              The genetic basis for DNA methylation variation across tissues and development
              </a>
            </td>
          <td>
            Jonathan Rosenski, O. Sabag, Eitan Marcus, Netanel Loyfer, Y. Dor, Howard Cedar, Tommy Kaplan
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="E2F5 is a member of E2F transcription factor superfamily, controlling many molecular activities, such as cell proliferation, cell differentiation, DNA repair and cell death. Therefore, it is closely related to the occurrence, development and prognosis of a variety of cancers. In recent years, with the rapid development of bioinformatics, genomics and epigenetics, this study has further elucidated of E2F5 in the tumor cell cycle. Based on the latest research reports, this study reviewed the structural composition, dynamic activity regulation of E2F5, and how its transcription program driven by carcinogenic activity changed the progress of various tumor cell cycles, especially how it converted from a “gatekeeper” to a “destroyer”, thus affecting abnormal biological behaviors of tumor cells. Our aim is to provide a new direction for the development of E2F5 targeted therapy strategies and drug resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04822ad21c2d133c4bb16efee92ced6e01833c81" target='_blank'>
              The Roles of E2F5 in Tumor Cell Cycle: The Gatekeeper or Destroyer?
              </a>
            </td>
          <td>
            Yingwen Du, Danyun Wang, Aidi Liang, Xinru Tang, Jiansen Chen, Ruizhi Yao, Lei Meng, Jianxing Xie, Ming Chen, Songtao Xiang, Canbin Lin
          </td>
          <td>2025-10-21</td>
          <td>Urology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. Results In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Conclusion Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements. Supplementary Information The online version contains supplementary material available at 10.1186/s40348-025-00208-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff9dd6e8b4e7b70e0de9451cb1c5dd37501bb1c" target='_blank'>
              A comprehensive genomic framework for identifying genes predisposing to homologous recombination repair deficient breast cancer
              </a>
            </td>
          <td>
            José Camacho-Valenzuela, Thibaut Matis, Carla Roca, Jorge Luis, Cuamatzi Flores, N. Hamel, Barbara Rivera, Simon Gravel, P. Polak, Carla Daniela, Robles-Espinoza, William D. Foulkes
          </td>
          <td>2025-10-23</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) holds promise for animal cloning but remains limited by low efficiency and phenotypic abnormalities, often attributed to incomplete nuclear reprogramming. This study presents an integrative genomic and epigenomic analysis of cloned buffaloes and their respective donors using long-read Oxford Nanopore sequencing. Our results showed a high degree of genomic similarity between clones and donors, with most variations located in non-coding regions and structural variants (SV) distributions highly correlated at the chromosomal level. Gene and protein level overlap of SV-affected loci revealed 70.9–73.3% gene-level and 69.7–72.5% protein-level similarity. Despite this genetic similarity, DNA methylation analysis identified differentially methylated regions (DMRs), particularly in intergenic and promoter regions. Clones exhibited slightly lower CpG methylation than the donors. The DMRs in donor vs. clone comparisons indicated higher hypomethylated regions than hypermethylated regions. Functional enrichment of DMR-associated genes highlighted pathways linked to mitochondrial function, oxidative phosphorylation, and reproductive processes. Although clones showed moderate genome-wide methylation correlation with donors, key differences in methylation suggest incomplete epigenetic reprogramming. Despite these epigenetic differences, all clones were phenotypically normal and healthy into adulthood. This study offers the first comprehensive SV and methylome profile of SCNT-derived buffaloes and emphasizes the role of epigenetic mechanisms in clone development and health, providing valuable insights to enhance cloning efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0c9146dce33964d38cc27a7e6aa2903443aa1" target='_blank'>
              Unravelling High Nuclear Genomic Similarity and Mitochondria Linked Epigenetic Divergence in SCNT Derived Buffalo Clones via Long-Read Nanopore Genome Sequencing
              </a>
            </td>
          <td>
            Meeti Punetha, Dharmendra Kumar, Satish Kumar, Bhavya Maggo, Priya Dahiya, Pradeep Kumar, Rakesh K. Sharma, Yash Pal, P. S. Yadav
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract UVB radiation (280–320 nm) from sunlight induces skin cancer through DNA damage-induced mutations. Melanomas carry mutational signatures associated with UVB-induced cyclobutane pyrimidine dimers (CPDs). However, there are several other melanoma signatures of unknown origin. To test if these signatures are linked to UVA, we exposed human melanocytes to UVA (340–400 nm) and to UVB for comparison. We mapped DNA damage in the form of CPDs or 8-oxoguanines (8-oxoG) genome-wide at base resolution. We then determined mutational patterns in single melanocyte cell clones by whole genome sequencing. Different from UVB, UVA induces CPDs more selectively at TT sequences resembling melanoma signature SBS7d. We did not observe rising CPD levels after cessation of radiation (dark CPDs). The UVA-induced CPDs were not mutagenic in the mutation analysis. 8-oxoG was present in melanocytes but did not substantially increase after UVA. G/C to T/A mutations were prominent in melanocyte single-cell clones with no major shift after UVA radiation. These mutations matched SBS18, a signature present in melanomas. Although UVA damages DNA, it has a surprisingly limited mutagenic effect on human melanocytes. However, the oxidative base lesions in melanocytes and their associated mutations may be linked to a subset of melanoma mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b21e19c0b878b06400096dc62c3652b28e09c65" target='_blank'>
              UVA-induced DNA damage and mutations in human melanocytes: relevance for melanoma mutations
              </a>
            </td>
          <td>
            , Jennifer Johnson, Paul Y Kim, Gerd P Pfeifer
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich DNA secondary structures with known regulatory roles in eukaryotes, but their functions in bacteria remain largely unexplored. Here, we systematically analyzed potential G-quadruplexes (PG4s) across 1566 bacterial genomes. PG4s showed non-random distributions, with strong enrichment in intergenic regions, particularly near gene start sites, suggesting potential roles in transcriptional regulation. Structural analysis revealed a higher proportion of short-loop G4s in these regions, indicating enhanced stability. Comparisons with randomized genomes confirmed that the observed patterns cannot be explained by GC content or genome size alone. Ecological analysis further revealed that bacteria associated with homothermic hosts exhibit significantly lower G4 densities than free-living species, suggesting reduced reliance on G4-mediated regulation in stable environments. Together, these findings highlight G4s as conserved yet context-dependent elements of bacterial genome organization and provide new insights into their potential regulatory roles in prokaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969b849342d5007389accc241718dd3c89890d08" target='_blank'>
              Non-Random Distribution of G-Quadruplex Structures Reveals Regulatory and Ecological Adaptations in Bacterial Genomes
              </a>
            </td>
          <td>
            Jiye Fu, Ke Xiao, Yukun He, Jing Tu
          </td>
          <td>2025-10-15</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming.
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Unlike other cancer types the determinants of anti-PD-1 response in clear cell RCC are not well defined; ccRCC exhibits a low mutational burden and CD8+ T cells infiltration is associated with worse prognosis. Cytomegalovirus (CMV) and human papillomavirus (HPV) infections are postulated to modulate the anti-tumor immune responses within the tumor microenvironment. We sought to investigate the clinical and immunologic impact of CMV and HPV viral transcriptomic signatures, as defined by prior studies, in patients with RCC. Methods RNA-seq data from three RCC prospetive trials were upper-quartile normalized, log2-transformed (TPM + 1), and batch-corrected using ComBat. Transcriptomic scores for CMV (75 genes) and HPV (79 genes) were computed using geometric means (71 and 77 genes were found, respectively). CD8 immunofluorescence (IF) was performed on tumor samples. Overall survival (OS) and progression-free survival (PFS) were assessed using multivariable Cox regression, adjusting for age at therapy start, gender, IMDC risk, sarcomatoid/rhabdoid features, and number of previous lines received. Results A high intratumoral CMV transcriptomic score was significantly correlated with T-effector signature (ρ  =  0.54, p = 4.8 × 10−6), IF tumor core CD8+ (ρ  =  0.43, P < .001), IF tumor margin CD8+ (ρ  =  0.41, P < .001), and myeloid signature (ρ  =  0.40, P < .001). The CMV score was significantly higher in infiltrated versus excluded/desert tumors (P = .0079). The CMV score was associated with worse OS among patients receiving immune checkpoint blockade (ICB) (HR 4.98, P = .039). In contrast, the HPV transcriptomic score showed only a modest correlation with myeloid signature (ρ  =  0.24, P = .051) and no association was found with OS nor PFS. Conclusions In metastatic RCC, a high tumor CMV transcriptomic score was associated with a paradoxical immune landscape, marked by CD8+ infiltration yet co-enriched with suppressive myeloid signatures, that predicts inferior overall survival following ICB. These viral signatures within the complex immune architecture of RCC may help understand immune responsiveness and resistance within distinct genomic and therapeutic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c3c656ee910ac2c2d9287a96ba1a9501c97839" target='_blank'>
              39Intratumoral viral transcriptomic signatures stratify immune phenotypes and clinical outcomes in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            Mustafa Saleh, E. Saad, Pablo Barrios, M. Machaalani, Liliana Ascione, Wassim Daoud Khatoun, Jad El Masri, Razane ElHajj Chehade, Marc Eid, Rashad Nawfal, Karl Semaan, E. Yekeduz, Clara Steiner, Weiwei Bian, Maxine Sun, S. Signoretti, E. V. Van Allen, D. Braun, Alexander Gusev, Toni Choueri
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="


 DNA damaging agents remain the mainstay for high grade serous ovarian cancer (HGSOC) therapy. While many HGSOCs are sensitized to DNA damaging agents via dysregulated DNA repair, those patients harboring tumors with amplification or overexpression of CCNE1 (encoding cyclin E1) respond poorly in part due to proficiency in the homologous recombination (HR) DNA repair pathway. Here, we put forth a novel paradigm that HR proficiency of CCNE1-driven HGSOCs is promoted by altered cellular metabolism. We found that alpha-ketoglutarate (aKG) is upregulated in cyclin E1-driven HGSOC cells, and suppression of aKG sensitized these cells to DNA damaging agents both in vitro and in vivo, demonstrating that aKG drives chemoresistance in these models. aKG is required for the activity of aKG-dependent dioxygenases (aKGDD). There are >60 known aKGDD, and prior work has almost exclusively focused on aKGDD-mediated histone demethylation. Through a targeted CRISPR knockout library, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for chemoresistance to DNA damaging agents in cyclin E1-driven cells. Unexpectedly, aKG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. These data point to a previously unrecognized role for aKG in acetylation reactions. The increase in histone acetylation via aKG-dependent carnitine synthesis promoted HR-mediated repair through site-specific histone acetylation. Additionally, analysis of HGSOC patient samples demonstrated that TMLHE positively correlates with histone acetylation, and both high tumor TMLHE or high serum acetylcarnitine are associated with worse progression free survival (PFS) in HGSOC patients treated with DNA damaging agents. Pharmacological targeting of carnitine synthesis via mildronate in vivo was sufficient to decrease tumor burden >30% in combination with cisplatin. Since mildronate is a non-toxic and extensively used drug in humans, this combination shows potential applicability of inhibition of this axis in humans. These data demonstrate for the first time that HR-proficiency in cyclin E1-driven HGSOCs is mediated through a metabolic axis that directly influences histone acetylation and highlight the translational potential of targeting this axis to induce HR deficiency and sensitize these tumors to DNA damaging agents.



 Apoorva Uboveja, Nathaniel W. Snyder, Katherine M. Aird. aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7139762235894a9ff9c39c95032bcf77d46290e" target='_blank'>
              Abstract B031: aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation
              </a>
            </td>
          <td>
            Apoorva Uboveja, Nathaniel W. Snyder, K. Aird
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b7b8c6e90f21e12e992c023df4f70131487ed6c" target='_blank'>
              The protein-protein interaction landscape of Fanconi anemia protein A (FANCA) reveals its contributions to mRNA processes
              </a>
            </td>
          <td>
            Virgile Gibert, Frédérique Maczkowiak-Chartois, Benedetta Mancini, E. Gautier, Johanna Bruce, Morgane Le Gall, S. Urbach, A. Constantinou, Anne Helbling-Leclerc, J. Basbous, Filippo Rosselli
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5a74f2db9883066216ff43d03114dc692d9adc" target='_blank'>
              Robust methylome analysis and tumour-normal classification in TCGA-COAD: a reproducible workflow
              </a>
            </td>
          <td>
            M. Ekum, C. Heumann, E. E. E. Akarawak, K. O. Adekoya, S. I. Oke, A. S. Ogunsnaya, C. Zhang, O. J. Akintande, J. A. Akinyemi, I. Ajiboye, O. D. Amusa
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters. Methods Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis. Results We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1. Conclusion These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. U. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Highlights What are the main findings? Poor prognostic ligand–receptor pairs are localized in specific regions within glioblastoma tissues. What is the implication of the main finding? Our findings provide new insights into pathological cell–cell interactions and treatment for glioblastoma. Abstract Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Graphical Abstract Abstract Dysfunction of several WD40 family proteins causes diverse endocrine diseases. Until recently, MEP50, a WD40 protein, was considered a gene of unknown significance because no inherited disease had been linked to its function. However, genetic inactivation of MEP50 in mouse models or somatic mutations in humans drives oncogenesis in several endocrine-related cancers, including those of the prostate, breast, and uterus. In this study, we generate new knowledge through a multi-tier integration of evolutionary genomics, sequence and structural analyses, molecular mechanic calculations, and dynamic simulations of wild-type and cancer-mutated MEP50 proteins. Indeed, we find that a conserved splicing event across evolution generates an alternative MEP50 isoform, which is smaller than the canonical MEP50 and lacks the final β-sheet of the first WD40 domain, the entirety of the second WD40 domain, and the first β-sheet of the third WD40 domain. Notably, we find that this novel, short MEP50 (s-MEP50) transcript encodes a 278 amino acid protein that retains aspects of the key regulatory and interaction sites, including those critical for androgen receptor and PRMT5 binding. Finally, we analyze the mutational landscape of MEP50 in endocrine-regulated cancers and use molecular mechanic calculations and dynamic simulations to reveal that cancer-associated mutations disrupt conserved bonding networks and induce widespread structural destabilization within the WD40 domain architecture. Thus, by combining evolutionary, structural, and biophysical approaches, we advance the understanding of MEP50 genomics, providing significant mechanistic and clinically relevant insights into endocrine-regulated tissues and their cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d5f2f3bba5deb45827f67cca6b9cab9291043" target='_blank'>
              Genomic features of the methylosome protein MEP50 and its implications in hormone signaling and cancer
              </a>
            </td>
          <td>
            Gareth Pollin, Young-In Chi, Raul Urrutia, G. Lomberk
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In the initiation and development of cancer, DNA damage plays a very crucial role. In the case of environmental perturbation, mutation events in critical control genes may occur as the integrity of genetic material is lost, either due to replication errors, oxidative stress, etc. And provided that these are proto-oncogenes or tumour suppressor genes, they can interfere with normal control of the cell cycle, programmed cell death (apoptosis) and repair of damaged DNA(Branzei & Foiani, 2023). Consequently, they can start to multiply out of control, which can result in the appearance of growth and malignancy. This abstract discusses the molecular processes that connect the occurrences of DNA damage and mutations with the out-of-control growth of cancer cells, pointing to their importance in the development of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cd781c4b40bc6486e36eec6ae725a99f2c36b2" target='_blank'>
              Exploring How DNA Damage And Mutations Lead To The Uncontrolled Growth Of Cancer Cells?
              </a>
            </td>
          <td>
            Yashvi Chhabra
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Scientific and Research Publications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background/Objectives: Oral squamous cell carcinoma (OSCC) is a common and aggressive oral cancer with high recurrence and mortality rates, largely due to late diagnosis and metastasis. Epigenetic regulation, particularly aberrant DNA methylation, plays a critical role in cancer progression. Altered methylation patterns disrupt cancer-related gene regulation. Our previous study found that oral cancer patients exhibit increased synthesis of S-adenosyl-L-methionine, a key methyl donor for cytosine methylation. Therefore, the aim of this study was to explore the relationship between global DNA methylation and OSCC progression and to evaluate the impact of DNA methylation heterogeneity on oral cancer cells. Methods: Immunohistochemistry (IHC) and immunofluorescence (IF) staining were used to examine 5-methylcytosine (5-mC) expression in OSCC clinical specimens and oral cancer cells. The DNA methyltransferase inhibitor 5-Aza-dC was used to assess the effects of DNA methylation on cell function and gene expression. RNA sequencing was used to identify key differentially expressed genes affected by 5-Aza-dC treatment. Cell migration was assessed using a wound closure assay. Protein and gene expression were analyzed using Western blotting and quantitative PCR. Results: An inverse relationship was found between 5-mC levels and cancer differentiation—poorly differentiated OSCC exhibited higher 5-mC levels. Additionally, higher 5-mC staining was observed at the invasion front of oral cancer tissues. In OSCC cells, 5-mC content correlated with migration ability. Furthermore, conditioned medium from cancer-associated fibroblasts enhanced both methylation levels and migration of OSCC cells. Treatment with 5-Aza-dC significantly increased epithelial differentiation, reduced epithelial-to-mesenchymal transition and cell adhesion-related genes, and inhibited OSCC cell migration. Conclusions: The findings highlight the critical role of DNA hypermethylation in OSCC progression, particularly in regulating differentiation, migration, and EMT. The interplay between the tumor microenvironment and epigenetic modifications underscores the complexity of OSCC biology and opens avenues for innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb0ec7659aed82eb5b415d59b960ec03d7ac79b" target='_blank'>
              DNA Hypermethylation at the Invasive Front of Oral Squamous Cell Carcinoma Confers Poorly Differentiated Characteristics and Promotes Migration of Cancer Cells
              </a>
            </td>
          <td>
            Li-Po Wang, Chien-Ya Li, Yu-Hsueh Wu, Meng-Yen Chen, Yi-Ping Hsieh, Tze-Ta Huang, T. Hong, Yuh-Ling Chen
          </td>
          <td>2025-09-27</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Despite aggressive treatment, glioblastoma patients suffer from an invariably poor prognosis. Whilst substan- tial progress has been made in understanding the role of DNA methylation in brain tumour pathology, methy- lation of RNA has received comparatively little attention, despite being a key driver of cellular differentiation and neurodevelopment. N6-methyladenosine (m6A) is the most abundant RNA modiﬁcation and is, crucially, reversible. The immense intra-tumoral heterogeneity and phenotypic plasticity of glioblastoma are major barri- ers to successful treatment, and dynamic m6A methylation may mediate such behaviour, by ﬂexibly inﬂuencing transcriptional networks via the regulation of mRNA metabolism.



 Nanopore sequencing technology uniquely enables direct sequencing of RNA, and therefore detection of RNA modiﬁcations within their sequence context. We present nanopore direct RNA sequencing data from four in- house glioma-derived cell lines. We validate this approach using a novel drug inhibitor of the major m6A methyl- transferase, METTL3.



 We deﬁne transcriptome-wide RNA methylome proﬁles for four cell lines and identify common candidate path- ways which exhibit methylation. We conﬁrm robust reduction in m6A marks following treatment with a novel METTL3 inhibitor. We further compare the transcriptional landscape of wild-type tumours against these treated samples, demonstrating differential expression of biological processes linked to RNA metabolism.



 We demonstrate that nanopore direct RNA sequencing can resolve m6A modiﬁcations at single-base resolu- tion, preserving m6A stoichiometry. We validate this approach with a known inhibitor of METTL3, and further identify differentially expressed pathways in response to such inhibition. We show that glioma cell lines demon- strate extensive methylation throughout the transcriptome, and exhibit a degree of inter-tumoral heterogeneity of these m6A signatures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad119dfd67a68d80f7976dca916d99c54e9f801" target='_blank'>
              EXPLORING THE GLIOMA M6A RNA EPITRANSCRIPTOME WITH DIRECT RNA SEQUENCING
              </a>
            </td>
          <td>
            Simon Deacon, M. Radhi, Lauryn Walker, J. Rowlinson, S. Paine, M. Loose, Stuart Smith
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, Madlen Müller, Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>2</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T- and B-cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B-cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by-product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B-cell leukemia.
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Cancer cells edit gene expression to survive and evade immunosurveillance. However, genome-wide studies of gene editing during early tumorigenesis are lacking because of difficulty capturing emerging tumors. We used single-cell RNA sequencing in an aggressive breast cancer genetically engineered mouse model (GEMM), driven by an inducible constitutively activate Her2 oncogene co-expressed with eGFP in mammary epithelia, to identify edited genes without bias. Early tumors captured 1 week after Her2 induction were compared to late tumors, analyzed 1 mo after induction. Late tumors epigenetically repressed genes that stimulate immune responses, reducing infiltrating immune cells and tumor-immune cell communications. Changes in tumor gene expression from early to late tumors concentrated on suppression of genes that promote tumor recognition by innate and adaptive immune cells, rather than on changes in the expression of genes that promote tumor cell-intrinsic malignancy. Innate immune genes, especially interferon-stimulated genes, dominated the down-regulated tumor genes. These data captured the rapid evolution of tumor immune evasion as naïve and activated CD8+ T cells in early tumors were replaced with exhausted or precursor-exhausted cells in late tumors. Recent unpublished work in progress uses multiplexed error-robust fluorescence in situ hybridization (MERFISH) to assess the spatial expression of 680 genes in the tumor microenvironment of these GEMM mice that identify cell subsets, functional markers, cytokines/chemokines and their receptors, and pathways involved in tumorigenesis and immune evasion, to compare normal mammary gland to early and late GEMM tumors. Repression of immune stimulating genes was reversed by inhibiting DNA methylation using low-dose decitabine, which suppressed tumor growth and restored immune control, increasing the numbers, functionality and memory of tumor-infiltrating lymphocytes and reducing myeloid suppressor cells. Decitabine induced key interferon, pyroptosis and necroptosis genes and tumor cell inflammatory cell death, all of which contributed to immune control of GEMM and implanted breast and melanoma tumors.



 Judy Lieberman, Ying Zhang, Pourya Nadheri Yeganeh, Haiwei Zhang, Bowen Gu, Simon Yuan Wang, Zhouyihan Li, Dian-Jang Lee, Zhibin Zhang, Hao Wu, Eric Greer, Jeffrey Moffitt, Winston Hide. Tumor editing suppresses innate and adaptive anti-tumor immunity and is reversed by inhibiting DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr IA07.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b587a4b5eb2a1a6ec68820d823fbb4b4d501930" target='_blank'>
              Abstract IA07: Tumor editing suppresses innate and adaptive anti-tumor immunity and is reversed by inhibiting DNA methylation
              </a>
            </td>
          <td>
            Judy Lieberman, Ying Zhang, Pourya Nadheri Yeganeh, Haiwei Zhang, Bowen Gu, Simon Yuan Wang, Zhouyihan Li, Dian-Jang Lee, Zhibin Zhang, Haoying Wu, E. Greer, Jeffrey Moffitt, Winston Hide
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The remarkable advances in cancer therapies significantly enhance the survival rates and longevity of cancer patients. Among childhood, adolescent, and young adult female cancer survivors, however, anti-cancer agents frequently cause primary ovarian insufficiency (POI), early menopause, and infertility, primarily due to the depletion of the ovarian reserve. Oocytes, the female germ cells, exhibit a notable susceptibility to DNA damage, given that they remain in meiotic arrest of prophase I for prolonged durations from months to years, which increases the risks of accumulating DNA damage overtime. To counteract this, a tightly controlled DNA damage response (DDR) signaling ensures that only oocytes with an intact genome progress to ovulation, fertilization, and next generations. Chemotherapeutic anti-cancer agents, including doxorubicin, cisplatin, cyclophosphamide, along with irradiation, elicit DNA damage via various mechanisms, including DNA crosslinking, single and double-strand DNA breaks, and oxidative stress. The genotoxic insults activate DDR in the oocytes, which detect and repair DNA damage or initiate apoptosis to eliminate impaired oocytes. Although several protein molecules such as DNA damage-sensing kinases, checkpoint kinases, p53 family transcription factors, and pro-apoptotic molecules, have been discovered, the precise mechanisms of DDR in determining the fate of oocytes, particularly how they differ from those in somatic cells and cancer cells, remain poorly understood. From an oncofertility perspective, the current review analyzes the molecular mechanisms of anti-cancer agent-induced DDR in oocytes and discuss knowledge gaps and urgent future research directions for preserving the ovarian reserve, fertility, and endocrine functions of young female cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3c0ab5ecbb10104dc6e55fe017fd113001dfb6" target='_blank'>
              DNA damage response signaling in oocytes from an oncofertility perspective.
              </a>
            </td>
          <td>
            Yunman (Sonia) Song, Jiyang Zhang, Yingnan Bo, Qiang Zhang, So-Youn Kim, Shuo Xiao
          </td>
          <td>2025-09-12</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, G. Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, , S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly aggressive malignancy with poor prognosis and significant molecular heterogeneity. This study investigates the role of tumor-specific enhancers in metabolic reprogramming, focusing on glucose-6-phosphate dehydrogenase (G6PD) and the pentose phosphate pathway (PPP). Using native elongating transcript-cap analysis of gene expression and single-cell RNA sequencing, tumor-specific enhancers driving G6PD overexpression were identified in ICC tumor epithelial cells. Functional assays demonstrated that G6PD promotes tumor proliferation by enhancing PPP activity and maintaining redox homeostasis, which provides NADPH to counter oxidative stress. Enhancer knockdown disrupted G6PD expression and PPP activity, increasing reactive oxygen species levels and reducing the NADPH/NADP+ ratio. These metabolic changes impaired tumor cell proliferation and sensitized ICC cells to cisplatin, emphasizing the dual therapeutic potential of targeting G6PD to inhibit tumor growth and overcome chemoresistance. Survival analyses showed that high G6PD expression correlates strongly with poor overall survival in ICC patients. While previous studies have recognized the roles of G6PD and PPP in cancer metabolism, this study uniquely links enhancer-mediated regulation to these processes in ICC, offering novel insights into epigenetic mechanisms driving metabolic reprogramming. Moreover, the findings highlight tumor-specific enhancers as critical epigenetic drivers of ICC progression, with potential as therapeutic targets. Future research should explore the integration of enhancer profiling into precision medicine frameworks and the development of novel enhancer-targeting strategies. These efforts could uncover additional metabolic vulnerabilities and provide effective treatments for this highly aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f09ce5004e00e224dd82b9d53133eeef55290e" target='_blank'>
              Epigenetic Regulation of G6PD Drives Metabolic Reprogramming in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Yusuke Nakano, Miwa Tanaka, Takeharu Sakamoto, M. Hashimoto, Taro Tobo, Hideyuki Saito, Tadashi Abe, Tomohiko Ikehara, Takashi Ofuchi, Koto Kawata, Takaaki Masuda, T. Ogino, Mamoru Uemura, H. Eguchi, Y. Doki, K. Mimori
          </td>
          <td>2025-10-07</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="ABSTRACT: Gene editing exploits endogenous DNA repair pathways to introduce precise modifications into the genome. The CRISPR system was first identified in Escherichia coli (1987) and, based on Cas protein architecture, is divided into Class I (types I, III, IV) and Class II (types II, V, VI). The widely adopted CRISPR/Cas9 system comprises a guide RNA (gRNA) and the Cas9 endonuclease, which orchestrate genome editing through a tripartite mechanism: target sequence recognition via gRNA binding to a complementary DNA site adjacent to a protospacer adjacent motif (PAM), double-stranded DNA cleavage by two nuclease domains (RuvC cleaving the non‑target strand and HNH the target strand), and repair of the resulting break by cellular pathways. This approach has been applied to in vitro fertilization-derived embryos and meiotically developing oocytes to disrupt or correct genes, offering potential for eliminating heritable diseases. However, concerns remain regarding off-target effects that introduces unwanted genetic mutations, necessitating improved specificity and ethical scrutiny.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e32b11816cdfe9dbe37c9b7f4fa9f2f1d5d3ed5" target='_blank'>
              CRISPR/CAS9-Mediated Gene Editing in Human Gametes: A Review
              </a>
            </td>
          <td>
            Esha Kumari, Neha Banu, Kathrina Marbaniang, Faridha Jane R.M. Momin, B. C. Hynniewta
          </td>
          <td>2025-09-30</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Chromophobe renal cell carcinoma (ChRCC) is the second most common non-clear cell renal cell carcinoma (ncRCC). A series of 496 ChRCC from Memorial Sloan Kettering Cancer Center demonstrated that sarcomatoid features, which were present in about 1.2% of the cases, are strongly correlated with metastatic disease and reduced overall survival. Little is known about the biological and immunological characteristics of the sarcomatoid variant of ChRCC and how it differs from its classic, non-sarcomatoid form. Methods Eight ChRCC tumor samples containing both classic and sarcomatoid components were selected. Xenium Prime 5K assay (5000 curated genes related to oncoimmunology) was used for high-resolution spatial transcriptomic profiling. Data were analyzed using Seurat and custom scripts to annotate cell types and conduct detailed phenotypic analysis. Pathway enrichment and transcription factor regulatory networks were analyzed using the SENIC package. Cell-cell communication was inferred using CellChat, and spatial proximity of immune and tumor cells (“cellular neighborhoods”) was analyzed, which model spatial gene expression patterns and intercellular distances. Results Spatial transcriptomic analyses revealed a significantly more immune-inflamed tumor microenvironment in sarcomatoid regions compared to their classic counterparts. We identified distinct spatial immune “neighborhoods” in sarcomatoid areas that were enriched in CD8+ T cells, polarized macrophages, and dendritic cells. These neighborhoods exhibited organized proximity to tumor cells expressing immune evasion markers such as PD-L1. In contrast, classic regions were relatively immune-deserted. Analysis of transcription factor activity showed upregulation of IRF1 and STAT1-related programs in the immune infiltrates of sarcomatoid regions, consistent with inflammatory activation. Importantly, comparison of matched sarcomatoid and classic regions within the same tumors enabled us to define a “sarcomatoid-enriched gene signature” composed of immune-related, pro-metastatic, and epithelial-to-mesenchymal transition (EMT) genes. This signature was validated in bulk RNA-seq data from The Cancer Genome Atlas (TCGA), where it correlated with poor prognosis in ChRCC patients. Conclusions This study presents the first spatial transcriptomic atlas of sarcomatoid ChRCC in comparison to matched non-sarcomatoid (classic) from the same patient. Profound immune microenvironmental differences were identified between tumor compartments. The sarcomatoid variant exhibits hallmarks of an immune-inflamed but potentially immune-evasive phenotype, including enriched immune infiltration, pro-inflammatory signaling, and EMT-associated transcriptional programs. These findings highlight the biological differences between sarcomatoid and classic ChRCC and uncover specific immune pathways and spatial cell-cell interactions that may be targeted in future immunotherapeutic strategies. Our work also establishes a foundational spatial map for ChRCC that offers new insight into the disease heterogeneity and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aba1786fb42a662d7a501d0a751ee46eaa895c9" target='_blank'>
              52Spatial transcriptomic mapping reveals immune-inflamed niches in sarcomatoid chromophobe renal cell carcinoma
              </a>
            </td>
          <td>
            Yan Tang, Tiegang Han, Hadi Mansour, M. Hirsch, Katrina Collins, Elizabeth P Henske
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

Breast cancer is the main cause of death for women, even with major improvements
in treatment. Through processes like DNA damage, estrogen metabolism, and immunological
regulation, bacterial populations have been shown to have an impact on breast cancer development
in recent studies.



This review aimed to examine and evaluate current research on the involvement of bacteria
in breast cancer progression, with an emphasis on gene action mechanisms and potential future
treatments targeting the microbiome.



A thorough literature analysis was carried out to identify pertinent research published between
1989-2024 across various databases, including PubMed, Google Scholar, Google, and Scopus.



Bacterial dysbiosis in the gut and breast tissue contributes to the progression of breast cancer
through different pathways. Double-strand breaks in DNA are linked to various bacteria, like
Escherichia coli, Staphylococcus epidermidis, Helicobacter pylori, and Fusobacterium, which contribute
to genomic instability. Breast cancers are influenced by hormones that are influenced by gut
microbiota, namely the estrobolome, which regulates estrogen levels. Bacteria also impact immune
responses by preventing anti-tumor immunity. These results suggest that restoring microbial balance
to specific bacterial taxa may open up new treatment options. Different genes may contribute
to variations, including an increase in regulatory T (Treg) cells, while FOXP3+ T cells are linked to
shorter relapse-free survival. Understanding the microbiota's role in DNA damage, hormone regulation,
and immune modulation is important.



Bacteria contribute significantly to the development of breast cancer through genelevel
processes. Probiotics, immunomodulatory techniques, and microbiome-targeted treatments are
potential future developments that could improve therapy effectiveness and reduce resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a52b781fcea2d4ef9f890f1a8d1daa10502bc" target='_blank'>
              The Bacterial Role in the Progression of Breast Cancer through Mechanism of Gene Action: Future Prospects with Existing Studies
              </a>
            </td>
          <td>
            Yasir Nawaz, Saba Munir, Sidra Aslam, Fizza Rimal Butt, Fouzia Tanvir, Basit Nawaz, Kazam Bashir, Rikza Tahreem
          </td>
          <td>2025-09-22</td>
          <td>Current Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background DNA methylation of CpG islands is altered in cancer cells. Hypermethylation of single CpG islands in the promoter regions of tumor-suppressor genes occurs already in the early stages of cancer. These methylation changes are cancer-type specific and therefore can serve as early cancer biomarker. Identifying good and reliable biomarkers is crucial for the development of diagnostic tests and their application in clinical practice and remains the most significant challenge to date. Results Here, we present a generic workflow for the discovery and design of DNA methylation-specific PCR (MSP) biomarkers using nanopore sequencing. We show that nanopore sequencing of three control and three tumor tissue samples was sufficient to predict differentially methylated regions between head and neck squamous cell carcinoma (HNSCC) and healthy control tissue samples and to design functional MSP primers. When applied to a validation cohort of 48 HNSCC and 46 control samples, four out of six designed MSP singleplex assays achieved good sensitivity and specificity with an AUC above 0.8. Conclusion Our resulting DNA methylation-based workflow demonstrates how long-read methylation data enable the design of adaptable, clinically relevant epigenetic assays, even with low coverage and small initial sample numbers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca77d339f993655ed91f8a895c93505240e806a" target='_blank'>
              Nanopore sequencing-derived methylation biomarker prediction for methylation-specific PCR in patients with head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Daria Meyer, Anne Hennig, Anna-Bawany Hums, Orlando Guntinas-Lichius, Martina Schmitz, Manja Marz
          </td>
          <td>2025-09-13</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BRCA1/2 are crucial in the homologous recombination repair (HRR) pathway, with loss-of-function (LoF) alterations predicting sensitivity to PARP-inhibitors (PARPi). Whether other HRR-gene alterations confer PARPi sensitivity remains unclear. In the Drug Rediscovery Protocol, patients receive off-label drugs matched to their tumor molecular profile. Here, olaparib efficacy and safety were evaluated in adult patients with treatment-refractory, progressive malignancies harboring LoF alterations in ATM (cohort A) or other HRR-genes including CDK12, PPP2R2A, CHEK1/2, and RAD51B (cohort B). Primary endpoints were clinical benefit (CB: confirmed objective response or stable disease ≥16 weeks) and safety. Pre-treatment biopsies were analyzed by whole-genome sequencing (WGS) for target validation. CB was observed in 8/25 patients (32%) in cohort A (prostate cancer: n = 6, adenoid cystic carcinoma: n = 1, endometrial cancer: n = 1). No effectiveness was seen in patients with colorectal cancer (n = 8). Median progression-free survival (PFS) and overall survival (OS) were 3.4 months (95% CI 1.8-5.3) and 9.2 months (95% CI 5.2-21.3), respectively. In cohort B, the CB rate was 41.7% (10/24) with median PFS and OS of 3.5 months (95% CI 3.4-6.6) and 8.1 months (95% CI 6.6-14.2), respectively. CB was observed in CKD12 (n = 7), RAD51B (n = 2), and CHEK2-altered tumors (n = 1), but not in PPP2R2A (n = 6) or CHEK1-altered tumors (n = 1). No unexpected toxicities occurred. WGS confirmed inclusion target in 84% of tested patients. In conclusion, PARPi sensitivity varies across HRR-genes, indicating that relying solely on an altered common mechanistic pathway is insufficient to predict response. Future studies should target specific HRR-genes to assess subgroup-specific benefits and determine proper use of molecular diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/307d50a92ba3f268facbb2ba41b55d6d57fef8c8" target='_blank'>
              Olaparib for patients with tumors harboring alterations in homologous recombination repair genes: Results from the drug rediscovery protocol.
              </a>
            </td>
          <td>
            Ilse A C Spiekman, N. Mehra, L. Zeverijn, B. Geurts, K. Verkerk, S. H. Haj Mohammad, V. van der Noort, P. Roepman, W. D. de Leng, A. Jansen, A. V. D. van de Luijtgaarden, T. van Voorthuizen, T. Buffart, H. Gelderblom, E. Voest, Henk M. Verheul
          </td>
          <td>2025-10-16</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract BRCA2 mutations contribute to the pathogenesis and treatment sensitivity of a subset of ovarian, breast, prostate, and pancreatic cancers. When these cancers become therapy resistant, secondary mutations that restore the BRCA2 open reading frame are found in half the cases, but other causes of resistance remain incompletely understood. Here, we identified translational readthrough of a premature termination codon (PTC) as a cause of resistance to poly(ADP-ribose) polymerase inhibitors (PARPis) and cisplatin in cells derived from the BRCA2-mutated ovarian cancer line PEO1 by PARPi selection. Despite persistence of the signature 4965C > G (p.Y1655X) BRCA2 mutation, low-level expression of full-length BRCA2 protein was detectable in these cells by immunoblotting and tandem mass spectrometry. Either BRCA2 knockdown or gene interruption 5′ or 3′ to the PTC restored treatment sensitivity, implicating BRCA2 in the resistance. Reporter assays demonstrated UAG-selective readthrough in the resistant clones but not parental cells. Moreover, custom searching of global proteomic data indicated readthrough of stop codons, particularly UAGs, in additional proteins in the resistant clones. Finally, multi-omic analysis identified multiple changes in the nonsense-mediated decay and termination machineries that favor readthrough. Accordingly, the present results identify PTC readthrough as a potential mechanism of drug resistance in cells with BRCA2 nonsense mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5f812b097b8ef89a4b208e38861c4246f71092" target='_blank'>
              Codon specific readthrough as a mechanism of BRCA2 restoration in acquired PARP inhibitor and chemotherapy resistance
              </a>
            </td>
          <td>
            Cordelia McGehee, X Wei Meng, Xinyan Wu, Cristina Correia, A. Venkatachalam, K. Flatten, Kevin L. Peterson, , Olivia K. Rossman, Clara Wong, R. Hurley, Jill M. Wagner, Paula A. Schneider, Haiming Dai, Guy G Poirier, Andrea E. Wahner Hendrickson, Elizabeth M. Swisher, Akhilesh Pandey, Hu Li, Scott H. Kaufmann
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Cancer remains a leading cause of global morbidity and mortality, underscoring the need for innovative diagnostic and prognostic biomarkers. PIWI-interacting RNAs (piRNAs) have emerged as critical regulators of gene expression and genomic stability, with increasing evidence linking their dysregulation to oncogenesis. piRNAs can modulate DNA methylation through interactions with DNA methyltransferases (DNMTs), contributing to tumor suppressor gene silencing and broader epigenetic reprogramming. However, the functional implications of piRNA-mediated methylation changes in cancer are not fully elucidated. This study investigates the circulating levels of piR-823 and piR-651, their associations with DNMT3B expression and global DNA methylation, and their correlations with clinical and molecular markers in colorectal, breast, and prostate cancers. Materials and methods A total of 300 participants, including 150 patients with histologically confirmed cancers and 150 age- and sex-matched healthy controls, were enrolled. Plasma piRNA levels were measured via qRT-PCR, while global 5-methylcytosine (5mC) and DNMT3B concentrations were quantified using ELISA. Immunohistochemistry was performed on resected tumor tissues to assess DNMT3B, Ki-67, p53, E-cadherin, vimentin, HER2, and androgen receptor (AR). Correlations were analyzed using Pearson’s test and multivariate regression models. Diagnostic performance was evaluated via Receiver Operating Characteristic (ROC) curves. Results piR-823 was significantly upregulated in colorectal and breast cancers (p < 0.001), while piR-651 was markedly downregulated in prostate cancer (p = 0.002). Cancer patients exhibited elevated levels of DNMT3B and 5mC (both p < 0.001), which correlated strongly with Ki-67 and p53 expression (r > 0.7, p < 0.001). E-cadherin expression was inversely correlated with DNMT3B and piR-823, consistent with a role in epithelial–mesenchymal transition (EMT). cfDNA levels were also significantly elevated in cancer cases (p < 0.001), suggesting potential utility as a non-invasive biomarker. Conclusion The observed associations between piRNA dysregulation, epigenetic markers, and tumor aggressiveness indicators support the potential of circulating piRNAs—particularly piR-823 and piR-651—as novel biomarkers for cancer detection and progression monitoring. These findings warrant further mechanistic studies to validate their functional roles in tumor epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec55cb867b96ef67ad7cc822f9de2fa5e24bf5e8" target='_blank'>
              Epigenetic functions and biomarker potential of PIWI-interacting RNAs in oncogenesis
              </a>
            </td>
          <td>
            Kldiashvili Ekaterina, Abiatari Ivane, Kekelia Elene, Iordanishvili Saba, Metreveli Tornike, Dumbadze Eter
          </td>
          <td>2025-09-30</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are pervasively present in human cancers and have a fundamental role in treatment failure and disease recurrence. Identifying critical elements that sustain the CSC phenotype may lead to novel strategies for cancer treatment. Here, we provide evidence of an essential link between the σ1 receptor (σ1R), a ligand-regulated chaperone protein residing preferentially at the endoplasmic reticulum-mitochondria contact sites, and CSCs in castration-resistant prostate cancers (CRPCs). Integrating functional assays in multiple preclinical models with transcriptomic and proteomic data, we found that σ1R controls CSC self-renewal capacity and tumorigenic proficiency by coordinating mitochondrial dynamics and mitochondrial-nuclear signaling. Inhibiting σ1R with synthetic antagonists and RNA interference led to the progressive exhaustion and loss of tumorigenicity of the CSC progeny. Mechanistically, interfering with σ1R function disrupted mitochondria homeostasis and triggered β-catenin degradation. Examining clinical CRPC samples, we found a tight correlation between σ1R and mitochondrial gene expression. Furthermore, σ1R and β-catenin protein levels were highly correlated in prostate tumors with significant upregulation in metastatic CRPCs, sustaining a role of the σ1R-mitochondria-β-catenin axis in disease progression. This σ1R-centered axis is essential for preserving the self-renewal and tumorigenic capability of CSCs and represents a critical vulnerability exploitable for discovering novel CSC-directed therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Approximately 50% of colorectal cancer (CRC) patients exhibit high levels of Fusobacterium nucleatum (Fn), which is associated with chemotherapy resistance. As Fn itself cannot be directly targeted for therapy in vivo, elucidating the mechanism underlying Fn-induced chemotherapy resistance is crucial, though it remains unclear. Methods qPCR was used to analyze the correlation between Fn abundance and clinical parameters in 80 human colon cancer samples. The effect of Fn on the sensitivity of colon cancer cells to oxaliplatin was evaluated by clone formation, EdU proliferation and apoptosis assays. RNA sequencing was performed on Fn-infected colon cancer cells to identify differentially expressed genes. Mechanistic studies explored the interaction between PVT1 and ATAD3A, and the role of TLR4/NF-κB pathway in regulating PVT1 expression. Results qPCR experiments showed that Fn abundance was associated with later clinical stage and shorter RFS. Clone formation, EdU proliferation assay and apoptosis assay showed that Fn could change the sensitivity of colon cancer cells to oxaliplatin. Further RNA sequencing showed that Fn infection could upregulate the expression of long noncoding RNA plasmacytoma variant translocation 1 (PVT1) in colon cancer cells. The abundance of Fn in colon cancer tissues was positively correlated with the level of PVT1, and the mechanism was that PVT1 binds to AAA domain protein 3 (ATAD3A) and prevents its ubiquitination. Fn upregulates ATAD3A expression through PVT1 and inhibits ER stress-mediated cell death. In addition, in colon cancer cells co-cultured with Fn, PVT1 expression is regulated by the TLR4/NF-κB pathway. Conclusions This study delineates a pathway where Fn infection promotes oxaliplatin resistancein colon cancer cells by upregulating PVT1 via the TLR4/NF-κB pathway. PVT1 subsequently stabilizes ATAD3A, suppressing cell death. PVT1 is a potential target to overcome the high abundance of Fusobacterium nucleatum leading to oxaliplatin resistance in colon cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07226-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ee35bc02bdd6b31bfdba892f3700bff37eb548" target='_blank'>
              Fusobacterium nucleatum enhances oxaliplatin resistance in colon cancer by increasing PVT1 expression
              </a>
            </td>
          <td>
            Kangqi Gao, Jianqing Zhang, Chen Liu, Yang Yang, Jianlong Wang, Jianfeng Zhang, H. Ma, Fei-fei Wang, Lianmei Zhao, Guiying Wang
          </td>
          <td>2025-10-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="for 273 TCGA colorecatal cancer subjects in a general linear model fitting a negative binomial distribution. TCGA annotations from the Covariables Univariate analysis The general linear model compared covariates This research first identifies that CXCL10 is significantly downregulated in Black colorectal cancers in two independent datatsets. We also observe that immune related pathways and cells are significantly different between the self identified Black and White Americans from The Cancer Genome Atlas. We use a combination of RT-qPCR, bulk RNA seq anal to show that CXCL10 expression is higher in the MMRdeficient CRC subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652519820bbd179c6d3708acf1e63564e6b14588" target='_blank'>
              The Effect of Mismatch Repair - Deficient/ Microsatellite Instability on Black Minority Colorectal Cancer Gene Expression
              </a>
            </td>
          <td>
            Dimitri F. Joseph, Joseph F. LaComb, Andrew Fu, Ricardo E. Flores, Ellen Li, Alexandra Guillaume, Bin Chen
          </td>
          <td>2025-10-01</td>
          <td>Spartan Medical Research Journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d93739961372e0101853c9916cb4256544b480" target='_blank'>
              SenLect: a genetically encoded system to purify senescent cells
              </a>
            </td>
          <td>
            Hourieh Tousian, Rachael A Protzman, T. Sargeant, Julian M. Carosi
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Centromeres, essential for chromosome segregation, exhibit remarkable evolutionary dynamism in sequence composition and structural organization. Here, we report the first haplotype-resolved, telomere-to-telomere genome assembly of Populus lasiocarpa (PLAS) and precisely map all 38 functional centromeres through CENH3 ChIP-Seq. Unlike classical satellite-rich centromeres in model plants, PLAS centromeres lack abundant satellite arrays but are dominated by retrotransposons, particularly RLG and RIL elements, which form intricate nested TE arrays within the functional centromeric regions, disrupting their structural integrity and driving their evolution. Comparative analysis with P. trichocarpa reveals a conserved retrotransposon-dominated architecture, despite minimal sequence conservation. We propose a cyclic model of centromere evolution in which autonomous retrotransposons destabilize functional centromeres through epigenetic erosion, triggering neocentromere formation at pericentromeric sites enriched in transposable elements (TEs) and tandem repeats (TRs). These neocentromeres either succumb to recurrent retrotransposon invasions or stabilize through KARMA-mediated TR expansion, ultimately giving rise to satellite-rich centromeres. Our work redefines centromeres as dynamic, epigenetically plastic domains shaped by retrotransposon-TR antagonism, challenging the satellite-centric paradigm and offering novel insights into plant genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d93a2539e4bb564c83252e1feb03b93990abbf" target='_blank'>
              Haplotype-resolved telomere-to-telomere genome assembly of Populus lasiocarpa unveils retrotransposon-driven centromere evolution.
              </a>
            </td>
          <td>
            Tengfei Shen, Yihang Ning, Yaolin Wang, Zihe Song, Mengli Xi, Huixin Pan, Meng Xu
          </td>
          <td>2025-09-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are promising biomarkers for cancer diagnosis and prognosis due to their ability to carry specific biomolecular cargo, including DNA. However, the clinical utility of DNA methylation-based liquid biopsies using EV-DNA remains underexplored. The low quantity and relatively long length of EV-DNA complicate whole-genome methylation profiling. To address this, we develop Tn5-assisted Enzymatic Methyl-sequencing with Post-conversion Tailing (TEMPT), a bisulfite-free whole-genome profiling method for EV-DNA. TEMPT employs single-adapter Tn5 tagmentation, enzymatic conversion of unmodified cytosines, and post-conversion tailing to generate high-depth whole-genome EV-DNA methylomes. We apply TEMPT to EV-DNA from 58 gastric cancer and polyp samples, generating methylomes from sub-nanogram inputs and identifying differentially methylated regions (DMRs) that distinguish cancer from controls. We identify potential cancer biomarkers through DMR-associated genes, highlighting the roles of EVs in cellular communication. Our findings suggest that immune cells may serve as an alternative source of EV-DNA. This approach holds significant promise for advancing EV-DNA research and its applications in early disease diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad67a59c8fc16bd26c5fe5c57db4a838137c7af1" target='_blank'>
              Whole-genome methylation profiling of extracellular vesicle DNA in gastric cancer identifies intercellular communication features
              </a>
            </td>
          <td>
            Bingqian Lin, Zhenna Jiao, Shouquan Dong, Weikai Yan, Jinting Jiang, Yanfang Du, Xiaocheng Weng, Hongling Wang, Zhiyuan Hu, Yibin Liu, Xiang Zhou
          </td>
          <td>2025-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer among women globally, disproportionately impacting low- and middle-income countries despite the existence of HPV vaccines. While DNA methylation has been studied extensively as a biomarker, other epigenetic mechanisms remain underexplored. This scoping review aims to report such underexplored epigenetic biomarkers linked to cervical cancer, shifting the focus beyond global nuclear DNA methylation. Literature searches were performed using Google Scholar via Publish or Perish software including studies published until January 2025. Our review focused on mitochondrial DNA, non-coding RNA, histone modifications, and repetitive elements. Mitochondrial DNA methylation has been proposed as a cervical cancer biomarker, although supporting evidence is limited. Histone modifications are more consistently reported to be involved both in cervical cancer onset and aggressiveness. Similarly, aberrant expression of lncRNAs, circRNAs, miRNAs, and piRNAs has been associated with poor prognosis. Finally, hypomethylation in repetitive elements such as LINE-1 and Alu is often observed in cervical cancer, contributing to genomic instability and tumorigenesis. Highlighting these alternative epigenetic mechanisms, our review emphasizes the importance of expanding biomarker discovery beyond the traditional nuclear DNA methylation. Understanding these mechanisms may improve early detection and personalized disease management strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7819cfc0864d5095515e01e909ecdbee12a80f" target='_blank'>
              Epigenetic Biomarkers for Cervical Cancer Progression: A Scoping Review
              </a>
            </td>
          <td>
            Efthymios Ladoukakis, Gracia Andriamiadana, Fatema Hajizadah, Lewis G. E. James, Belinda Nedjai
          </td>
          <td>2025-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres.
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Although intragenic CpG dinucleotides are conserved during evolution, they are also sensitive to methylation-dependent mechanisms. Methylation status of the TP53 introns 1, 3 and 4 have been analysed in stage III ovarian carcinoma (OC). In the present study, the methylation of exon 4 (10 CpG pairs) was analysed in advanced-stage OC to investigate TP53 methylation and compare exon and intron 4 methylation patterns. A total of 80 samples from patients with advanced-stage OC and metastatic lesions were examined, along with 80 samples derived from healthy patients who had never been diagnosed with cancer. Methylation analysis of the human A2780 ovarian cancer cell line was also performed. Exon and intron 4 were methylated in OC, corresponding metastases and paired healthy tissue. The DNA from the human A2780 ovarian cancer cell line and the normal samples from healthy subjects was also methylated. The data indicate the existence of an intragenic mechanism of regulation of TP53 activity that involves demethylation/methylation processes. This mechanism provides the ability to alter the response from cell cycle arrest to apoptosis by manipulating only the expression of long or short p53 isoforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba65eba61b813922b785bd71652e055bf783b2a" target='_blank'>
              Exon 4 and intron 4 TP53 are both methylated in advanced-stage ovarian carcinomas
              </a>
            </td>
          <td>
            Wiktor Szewczuk, Oksana Szewczuk, Krzysztof Czajkowski, R. Gromadka, Maciej Waledziak, Marek Semczuk, Andrzej Semczuk
          </td>
          <td>2025-09-12</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Many pediatric cancers, are driven by fusion oncoproteins. Many of the oncogenic fusions are to kinases such as ALK, ROS1, RET, NTRK3, FGFR3, MET, ABL1, JAK2, and BRAF. The mechanism for these are believe to be the consequence of altered function such as constitutive dimerization, loss of autoinhibition or aberrant cellular localization. Fibrolamellar carcinoma (FLC) is driven by the fusion of the first exon of DNAJB1, a heat shock protein, to the bulk of the coding region of PRKACA, the catalytic subunit of protein kinase A (PKA), forming DNAJB1::PRKACA. The active sites of the DNAJB1::PRKACA and native PRKACA are indistinguishable structurally and enzymatically raising the question of how does formation of the DNAJB1::PRKACA lead to transformation.



 We examined the possibility that increased activity levels of protein Kinase A induces FLC, independent of the fusion adduct, DNAJB. In FLC, expression of DNAJB1::PRKACA fusion is driven by the heat shock promoter. PKA is a holoenzyme of two catalytic subunits and two regulatory subunits, who inhibit and localize the catalytic activity.



 Quantitative mass spectrometry demonstrated that in patient normal liver there is a quantitative excess of regulatory subunit but in FLC tumor tissue there is an excess of catalytic subunit. Further, many of the regulatory subunits in condensates which are low, or missing catalytic subunit. The net result is a large increase of catalytic activity and a large increase of catalytic subunit in the nucleus. Overexpression of either the native PRKACA or the fusion DNAJB1::PRKACA in primary human hepatocytes produced indistinguishable alterations of the transcriptome, and they both recapitulated the transcriptome in patient tumors. We found patients who had fusions of a different domain to PRKACA, which also resulted in overexpression of the catalytic subunit, and their transcriptomes were the same as in FLC and their drug response properties were the same. We found patients with no fusion to the catalytic subunit but they were missing the regulatory subunit and their transcriptome and histopathology was indistinguishable from DNAJB1::PRKACA driven FLC.



 FLC is driven by the overexpression of the catalytic activity of protein kinase A, not by an altered or gain of function.



 Mahsa Shirani, David Requena, Denise Ng, Philip Coffino, Barbara Lyons, Sanford M. Simon. Mechanisms underlying Fibrolamellar Carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf547caa84006c646b1fb348ef865412d45329c2" target='_blank'>
              Abstract B034: Mechanisms underlying Fibrolamellar Carcinoma
              </a>
            </td>
          <td>
            Mahsa Shirani, David Requena, D. Ng, Philip Coffino, Barbara Lyons, Sanford M. Simon
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Galectin-3 (Gal-3), encoded by LGALS3, is a β-galactoside-binding lectin involved in diverse tumor-associated processes, including immune modulation, cell cycle regulation, and stress adaptation. Despite its known roles in cancer biology, the full extent of its molecular functions and prognostic relevance across tumor types remains incompletely understood. This study aimed to systematically investigate the transcriptomic impact of LGALS3 deletion and assess its clinical significance in cancer. Methods: We analyzed CRISPR-Cas9 knockout transcriptomic data from the SigCom LINCS database to characterize the consensus gene signature associated with LGALS3 loss using functional enrichment analyses. Pan-cancer survival analyses were conducted using TIMER2.0. Differential Gal-3 protein levels in ductal adenocarcinoma and normal pancreatic tissues were evaluated using the Human Protein Atlas. Finally, functional analyses were performed in pancreatic ductal adenocarcinoma (PDAC). Results: LGALS3 deletion across multiple cancer cell lines led to transcriptomic changes involving mitotic progression, stress responses, and axonal guidance pathways. High LGALS3 expression was significantly associated with worse overall survival in lower-grade glioma, PDAC, uveal melanoma, and kidney renal papillary cell carcinoma. LGALS3 knockout in YAPC cells recapitulated the pan-cancer findings, linking LGALS3 to cell morphogenesis and proliferation. Conclusions: These findings identify Galectin-3 as a key regulator of oncogenic programs and a potential prognostic biomarker in PDAC and other malignancies, with implications for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0894155b476f9fae4b8588df639cdd5d50fcb2b" target='_blank'>
              Transcriptomic and Clinical Profiling Reveals LGALS3 as a Prognostic Oncogene in Pancreatic Cancer
              </a>
            </td>
          <td>
            Grazia Scuderi, S. Mijatović, D. Maksimović‐Ivanić, M. Di Rosa, J. Muñóz-Valle, Alexis Missael Vizcaíno-Quirarte, Gian Marco Leone, K. Mangano, P. Fagone, Ferdinando Nicoletti
          </td>
          <td>2025-10-03</td>
          <td>Genes</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cdfd2697bacdb8fd34847d7d23a2cee0df40bf" target='_blank'>
              Circulating Tumor DNA Profiling Reveals Genomic Evolution in Recurrent Gastric or Gastroesophageal Junction Cancer.
              </a>
            </td>
          <td>
            Ryohei Kawabata, Hiroyuki Takeda, Atsushi Ishiguro, S. Nishina, M. Takahashi, Shuhei Suzuki, Takahisa Suzuki, Jin Matsuyama, Y. Otsuki, Y. Akamaru, Naoki Takegawa, T. Nomura, Y. Kito, H. Yabusaki, Yuji Negoro, A. Makiyama, Masato Nakamura, M. Takahashi, Yu Sunakawa
          </td>
          <td>2025-09-01</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c629e7baaceae40832311e536a58f80a6d8ec6a8" target='_blank'>
              Transcriptomic Mutational Profiling of Gastric Adenocarcinoma in Northern Brazil
              </a>
            </td>
          <td>
            Rubem Ferreira da Silva, Diego Pereira, Mateus Silva Tavares, Daniel de Souza Avelar, Jéssica Manoelli Costa da Silva, R. Mourão, Valéria Cristiane Santos da Silva, Juliana Barreto Albuquerque Pinto, Samir Mansour Moraes Casseb, S. Demachki, Tatiane Neotti, Ana Karyssa Mendes Anaissi, W. Barra, A. Vidal, G. Ishak, P. P. de Assumpção, R. Burbano, Fabiano Cordeiro Moreira
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="As a highly invasive malignant tumor, pancreatic cancer (PC) has an increasing incidence rate and a long-term low 5-year survival rate. At present, CRISPR has been used in the research of PC to analyze the function of oncogenes, explore drug resistance mechanisms and develop treatment strategies. Most of them are in the stage of cell and animal experiments. However, there are still research gaps in tumor microenvironment penetration, low delivery efficiency in vivo, off target effects and treatment limitations caused by tumor heterogeneity. This article systematically reviews the CRISPR foundation, analyzes the pathogenesis of high-frequency mutant genes in PC, elaborates three major treatment strategies based on CRISPR, combs the progress of second-generation CRISPR, clarifies the current technical challenges, and proposes targeted vector development, high fidelity Cas protein application and other corresponding programs. This study provides a comprehensive reference for the clinical transformation of CRISPR in the field of PC, helping to break through the bottleneck of traditional treatment; However, the long-term safety of this technology in the human body still needs to be verified. Future research can rely on the third-generation CRISPR to optimize the targeted delivery system, explore the joint strategy with immunotherapy and chemotherapy, promote the technology to clinical implementation, and provide more accurate treatment schemes for PC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1280741d28338ece3f7cdcb3f5fc9bdf722360" target='_blank'>
              Application of CRISPR in the Treatment of Pancreatic Cancer
              </a>
            </td>
          <td>
            Zixi Tian
          </td>
          <td>2025-10-23</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d18456bf0573eaf86f58e770fae2b90f97839de" target='_blank'>
              USP44, ZNF454, and GPRC5B ctDNA Methylation Markers in Breast Cancer: Limited Clinical Relevance for Disease Monitoring and Tumor Characteristics.
              </a>
            </td>
          <td>
            Y. Jeong, Chang-Ho Jeon, Hoon Kyu Oh, Jeong Kyu Kim, Chunseok Yang, Na-Rang Lee, Doyeon Kim
          </td>
          <td>2025-09-08</td>
          <td>Asia-Pacific journal of clinical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="This study examines the contribution of dysregulated long non-coding RNAs (lncRNAs) to disease progression and prognosis in breast invasive carcinoma (BRCA) and cervical squamous cell carcinoma with endocervical adenocarcinoma (CESC) using data from The Cancer Genome Atlas (TCGA). Comparative analyses identified both shared and cancer-specific lncRNAs linked to tumorigenesis. Survival analysis using the Kaplan–Meier method highlighted several candidates with prognostic relevance. Notably, FAM83H-AS1 showed increased expression in advanced stages of BRCA and was linked with lymph node involvement and metastatic status. Its elevated expression was associated with poor survival outcomes in both BRCA and CESC cohorts. In contrast, lncRNAs such as FGF14-AS2, which were downregulated, demonstrated links to tumor-suppressive mechanisms and improved survival in BRCA patients. Subcellular localization analysis indicated predominant cytoplasmic presence of FAM83H-AS1, implying possible roles in post-transcriptional regulation. To support our in-silico observations, we carried out qRT-PCR experiments in breast cancer cell lines with varying degrees of aggressiveness, which confirmed the differential expression patterns of these lncRNAs. Overall, our findings highlight the clinical importance of lncRNA dysregulation in gynaecological cancers and point to their potential utility as biomarkers for prognosis and targeted therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16436-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a627efdc32c519561de180942228807a9ccfd0" target='_blank'>
              Functional characterization of dysregulated LncRNAs in BRCA and CESC highlights their role in cancer prognosis
              </a>
            </td>
          <td>
            Madhur Sharma, Nidhi Chourasia, Priyanka Priyanka, D. P. Sarkar, A. Nag, Sandeep Saxena
          </td>
          <td>2025-10-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd59fd638b49487acab5f65a27e1f99a668332d" target='_blank'>
              Mechanistic insights into MET exon 14 skipping mutations and their role in tumor progression.
              </a>
            </td>
          <td>
            Promita Ghosh, Isabella Pecora, Morag Park
          </td>
          <td>2025-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889c401af57de3889b95cf28e228d76bc1fc7c6d" target='_blank'>
              Repeated losses of self-fertility shaped heterozygosity and polyploidy in yeast evolution
              </a>
            </td>
          <td>
            Nina Vittorelli, C. Gómez-Muñoz, Irina Andriushchenko, Louis Ollivier, Nicolas Agier, Stéphane Delmas, Yann Corbeau, Guillaume Achaz, M. Cosentino Lagomarsino, Gianni Liti, Bertrand Llorente, G. Fischer
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The initiation and progression of tumors are closely linked to aberrant regulation of multiple genes. Among these genes, sister chromatid cohesion acetyltransferase 2 (ESCO2) has emerged as an important target in cancer research. ESCO2 exhibits complex and heterogeneous expression patterns across various types of malignancies and tumor progression stages. Extensive studies have shown that ESCO2 primarily functions in chromosomal cohesion, significantly contributing to genomic stability maintenance. Dysregulated expression of ESCO2 in tumor cells is often correlated with malignant behaviors. Given its essential role in tumorigenesis, ESCO2 has increasingly been explored as a promising therapeutic target in recent years. This review article summarizes recent research progress regarding ESCO2 in malignant tumors, aiming to identify a novel target for their treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03164-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b1b540beced4302a863f075307c9fa3cb6930a" target='_blank'>
              The expression of establishment of cohesion 1 homolog 2 (ESCO2) in tumor cells and its research progress as a therapeutic target
              </a>
            </td>
          <td>
            Chunmei Lu, Hanchi Wu, Huiyu Wang
          </td>
          <td>2025-10-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The DREAM complex has emerged as a central repressor of DNA repair, raising questions as to whether such repression exerts long-term effects on human health. Here we establish that DREAM activity significantly impacts lifetime somatic mutation burden, and that such effects are linked to altered lifespan and age-related disease pathology. First, joint profiling of DREAM activity and somatic mutations across a single-cell atlas of 21 mouse tissues shows that cellular niches with lower DREAM activity have decreased mutation rates. Second, DREAM activity predicts the varied lifespans observed across 92 mammals, with low activity marking longer-lived species. Third, reduced DREAM activity in Alzheimer’s patients predicts late disease onset and decreased risk for severe neuropathology. Finally, we show DREAM knockout protects against mutation accumulation in vivo, reducing single-base substitutions by 4.2% and insertion/deletions by 19.6% in brains of mice. These findings position DREAM as a key regulator of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ead31f56ffdb72707b88fc850bc30a7d800daa3" target='_blank'>
              The DREAM complex links somatic mutation, lifespan, and disease
              </a>
            </td>
          <td>
            Zane Koch, Shuvro P. Nandi, Kate Licon, Arturo Bujarrabal-Dueso, David H. Meyer, Safa Saeed, Pirunthan Perampalam, Frederick A. Dick, B. Schumacher, Ludmil B. Alexandrov, T. Ideker
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Due to the persistently high global incidence and mortality rates of cancer, developing novel therapeutic strategies is imperative. The replication factor C (RFC) family, a critical subset of DNA replication and repair, serves multifaceted roles in tumor progression. Despite its widely recognized importance, the pleiotropic mechanisms of the RFC family lack systematic illustration, particularly regarding each member specific contributions to cancer hallmarks. In the present review, mRNA expression of each RFC family member in pan-cancer was profiled and the associations between their expression levels and tumor types evaluated. In addition, the effect of RFC expression on patients' survival across malignancies is assessed. Furthermore, the present review summarized current research on RFC family members in various malignancies with particular emphasis on the RFC-like complexes, highlighting key findings and advancements in understanding their role in tumor biology. The signaling pathways associated with RFC family members are discussed and the molecular mechanisms elucidated. Finally, the clinical importance of RFC family members including prognosis, potential inhibitors and combination treatments are also discussed. The present review aimed to provide innovative perspectives for developing combinatorial molecular targeted therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41b28d0169fef1334d7ef3750ca63ec9bbc80ab" target='_blank'>
              Multiple roles of replication factor C family in pan-cancer (Review)
              </a>
            </td>
          <td>
            Dan Luo, Bo Li, Xueping Jiang
          </td>
          <td>2025-09-25</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Cervical cancer remains a major cause of cancer-related death among women worldwide. Despite advances in treatment, prognosis remains poor for many patients due to tumor heterogeneity. DNA methylation, an epigenetic modification, is known to influence tumor development, but its role in defining molecular subtypes and prognostic stratification in cervical cancer remains inadequately understood.


METHODS
We analyzed DNA methylation profiles from 287 cervical cancer samples obtained from the UCSC Xena database. Univariate and multivariate Cox regression analyses were applied to identify prognostic CpG sites, as these models allow evaluation of individual and combined effects of methylation sites on patient survival. Consensus clustering was performed to define robust molecular subtypes based on methylation patterns, providing insights into tumor heterogeneity. Differentially methylated regions were identified using the Quantitative Differentially Methylated Regions (QDMR) software, an entropy-based tool validated for detecting subtype-specific methylation markers. A Bayesian classifier was constructed and validated in training and test cohorts to evaluate the predictive accuracy of these markers for subtype classification. Additionally, immune cell infiltration was estimated using computational algorithms to assess tumor microenvironment differences, and chemosensitivity was predicted to explore potential clinical implications of the methylation subtypes.


RESULTS
Four distinct methylation-based subtypes differed in methylation patterns, histological types, clinical stages, and metastatic status. A total of 501 subtype-specific methylation sites were identified. The Bayesian classifier demonstrated strong predictive performance, with an area under the receiver operating characteristic (ROC) curve (AUC) of 0.824 based on 10-fold cross-validation, indicating high classification accuracy and robustness. The immune microenvironment composition varied markedly among subtypes. Notably, Cluster 1 had elevated infiltration of central memory CD8+ and effector memory CD4+ T cells, whereas Cluster 4 exhibited reduced immune activation and the lowest immune checkpoint expression. These findings indicate subtype-specific differences in potential responsiveness to immunotherapy.


CONCLUSIONS
These DNA methylation-driven subtypes highlight the heterogeneity of cervical cancer and offer new insights for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e9636174b92b528395146a5a9794249738dcd9" target='_blank'>
              Excavation of Molecular Subtypes of Cervical Cancer Based on DNA Methylation Patterns.
              </a>
            </td>
          <td>
            Yiwei Zhao, Chutong Zhao, Jiyun Zhao, Yuhan Ma, Shunjin Zhang, Yujie Liu, Yuan Wang, Sijia Liu, Yunyan Zhang
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite numerous studies investigating intratumoral microorganisms and their significant roles in cancer initiation, progression, and treatment efficacy, a systematic understanding of intratumoral microorganisms remains lacking. Herein, we conducted a study using 16S rDNA data on seven types of cancer, comprising a total of 783 samples. It’s worth noting that Pseudomonas, Streptococcus, and Prevotella were found to be shared with the microbial communities of the seven cancers, suggesting that these may be associated with the occurrence and development of cancers. We anticipate establishing a foundation for related research and exploring potential methods for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80274119fe0660855f0a9a793c84d8230ca38ecb" target='_blank'>
              Pan-cancer analysis reveals intratumoral microbial diversity in multiple cancers by amplicon technology
              </a>
            </td>
          <td>
            Xudong Liu, Yuteng Yao, Aoyi Xiao, Dingyan Cao, Jingheng Zhang, Yanan Shi, Qing Zhong, Zilong He, Wenming Wu
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b65d7224e1ea2a592113167b5b5ec427e6e93fc" target='_blank'>
              Mapping Molecular Diversity in Prostate Cancer with a Combined Multiplex IHC and RNA-ISH Assay
              </a>
            </td>
          <td>
            Shannon Carskadon, Sean Williamson, Sangeetha Jyothilingam, Nilesh Gupta, N. Palanisamy
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Objective Increasing evidence indicated substantial involvement of non-Helicobacter pylori microbiota in gastric tumorigenesis. We aimed to elucidate detailed relationship of microbiome dynamics between two different steps in gastric cancer (GC) such as cancer initiation and progression, and assessed their associations with clinicopathological and molecular changes. Methods We systemically characterized gastric microbiome during GC initiation and progression using 944 biopsies from primary GC, non-cancerous gastric mucosa from both GC and non-cancer subjects. The association between specific microbial characteristics and GC risk, prognosis and molecular changes such as TP53 mutation, DNA methylation and telomere shortening were also evaluated. Results Microbial α-diversity in the gastric mucosa was decreased in relation to the GC occurrence, while it increased in primary GC tissue. Such paradoxical change was also observed in specific groups of bacteria during GC occurrence and its progression. GC risk-related microbiome was associated with differentiated GC, severe intestinal metaplasia, associated DNA methylation and telomere shortening, while GC tissue-specific microbiome was associated with more aggressive features of GC and TP53 mutation status. Conclusions Our findings suggested the different role of non-Helicobacter pylori microbiota in GC initiation and progression steps. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07046-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22ed5bfbd3e11bb7f9081dc75ec2feb3feea4022" target='_blank'>
              Gastric microbiome in gastric cancer sequence depicts diverse microbial structures associated with cancer risk and prognosis
              </a>
            </td>
          <td>
            Tsubasa Shimogama, Tomomitsu Tahara, Takuya Shijimaya, Jumpei Yamazaki, Sanshiro Kobayashi, Naohiro Nakamura, Yu Takahashi, T. Tomiyama, Yusuke Honzawa, T. Fukui, Makoto Naganuma
          </td>
          <td>2025-10-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae85fb3f25140318cac9ff830fec7b82bf6f487" target='_blank'>
              Characterization of Human Endogenous Retroviruses in Gastric Cancer with Helicobacter pylori: A Study from Northern Brazil
              </a>
            </td>
          <td>
            Marcos da Conceição, Juliana Barreto Albuquerque Pinto, Diego Pereira, Rubem Ferreira da Silva, Sérgio Augusto Antunes Ramos, Daniel de Souza Avelar, Jéssica Manoelli Costa da Silva, Ronald Matheus da Silva Mourão, Valéria Cristiane Santos da Silva, Samir Mansour Moraes Casseb, S. Demachki, Tatiane Neotti, A. Anaissi, W. Barra, A. Vidal, G. Ishak, P. P. de Assumpção, R. Burbano, Fabiano Cordeiro Moreira
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis through their capacity for self-renewal and multilineage differentiation. However, the isolation and functional characterization of HSCs remain challenging due to their cellular heterogeneity and dynamically regulated transcriptional and epigenetic landscapes. Advances in experimental and computational biology, including single-cell RNA sequencing (scRNA-seq), chromatin immunoprecipitation sequencing (ChIP-seq), network inference algorithms, and machine learning, have improved our ability to resolve transcriptional states, trace lineage trajectories, and reconstruct gene regulatory networks (GRN) at single-cell resolution. These approaches enable the discovery of novel HSC subtypes and regulatory factors, and facilitate the integration of multi-omics data to uncover epigenetic and transcriptional mechanisms that drive stem cell fate decisions. Additionally, machine learning models trained on high-throughput datasets provide predictive power for identifying novel enhancers, transcription factors, and therapeutic targets. This review underscores the synergistic role of computational tools in deciphering HSC biology and highlights their potential to improve stem cell therapies and precision treatments for hematologic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3c26c697c565c33ced832e7a44009a59ac4768" target='_blank'>
              Unlocking hematopoietic stem cell potential: integrative computational approaches for genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            P. Raghav, Basudha Banerjee, Rajni Chadha
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA repair is a multifaceted biological process that involves multiple pathways to counter the types of damage the genome encounters throughout life. In the past decade zebrafish became a popular model organism to study various aspects of vertebrate DNA repair, and the characterization of several mutant lines deficient in key players of the repair pathways has significantly contributed to our understanding of the roles the corresponding proteins play in the maintenance of genomic integrity. Interestingly, the base-excision repair (BER) pathway remained one of the less characterized DNA repair processes in fish. Here we provide a detailed characterization of zebrafish deficient in one of the key components of BER, the uracil-DNA glycosylase Unga. We show that while these fish are viable, they display an altered response to genotoxic stress and unga mutant males show an interesting form of subfertility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a43c49560d6251c2408bc41d8723ce8251daad0" target='_blank'>
              Zebrafish Unga Is Required for Genomic Maintenance upon Genotoxic Stress and Male Fertility
              </a>
            </td>
          <td>
            Latifa Kazzazy, Flóra Huba, Bálint Lóránt Hausz, Dávid Mező, Viktória Perey-Simon, Bálint Jezsó, Abdulrahman Seddik, Zoran Marinović, Judit Tóth, Angéla Békési, Beáta G. Vértessy, Máté Varga
          </td>
          <td>2025-09-01</td>
          <td>Journal of Developmental Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>