<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-24 09:26:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>28</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>66</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>105</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>314</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>268</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) are circular DNA bodies that play critical roles in tumor progression and treatment resistance by amplifying oncogenes across a wide range of cancer types. ecDNA lack centromeres and are thus not constrained by typical Mendelian segregation, enabling their unequal accumulation within daughter cells and associated increases in copy number. Despite intrinsic links to their oncogenic potential, the fidelity and mechanisms of ecDNA inheritance are poorly understood. Here, we show that ecDNA are protected against cytosolic mis-segregation through mitotic clustering and by tethering to the telomeric and subtelomeric regions of mitotic chromosomes. ecDNA-chromosome tethering depends on BRD4 transcriptional co-activation and mitotic transcription of the long non-coding RNA PVT1, which is co-amplified with MYC in colorectal and prostate cancer cell lines. Disruption of ecDNA-chromosome tethering through BRD4 inhibition, PVT1 depletion, or inhibiting mitotic transcription results in cytosolic mis-segregation, ecDNA reintegration, and the formation of homogeneously staining regions (HSRs). We propose that nuclear inheritance of ecDNA is facilitated by an RNA-mediated physical tether that links ecDNA to mitotic chromosomes and thus protects against cytosolic mis-segregation and chromosomal integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>3</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations, and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated copies of large genomic segments, identify multi-way interactions, distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Chromosomal instability in gastric cancer cells is associated with the amplification of oncogenes that encode receptor tyrosine kinases (RTKs), such as HER2 and FGFR2; such gene amplification varies from cell to cell and manifests as genetic heterogeneity within tumours. The intratumoural genetic heterogeneity of RTK gene amplification causes heterogeneity in RTK protein expression, which has been suggested to be associated with therapeutic resistance to RTK inhibitors; however, the underlying mechanism is not fully understood. Here, we show that extrachromosomal DNA (ecDNA) causes intratumoural genetic heterogeneity in RTKs and drug resistance due to diverse dynamic changes. We analysed the dynamics of FGFR2 and MYC ecDNA in a gastric cancer cell line after single-cell cloning. Similar to those in parental cells, the copy numbers of FGFR2 and MYC in subclones differed significantly between cells, indicating intraclonal genetic heterogeneity. Furthermore, the ecDNA composition differed between subclones, which affected FGFR2 protein expression and drug sensitivity. Interestingly, clone cells that were resistant to the FGFR2 inhibitor AZD4547 presented diverse changes in ecDNA, including chimeric ecDNA, large ecDNA and increased ecDNA numbers; these changes were associated with high expression and rephosphorylation of FGFR2. Conversely, when resistant clone cells were cultured under conditions that excluded AZD4547, the ecDNA status became similar to that of the original clone cells, and the inhibitory effect on cell growth was restored. Implications: Our results show that dynamic quantitative and qualitative changes in ecDNA can drive the intratumoural genetic heterogeneity of RTKs and resistance to RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome organization recapitulates function, yet ciliates like Oxytricha trifallax possess highly-specialized germline genomes, which are largely transcriptionally silent. During post-zygotic development, Oxytricha’s germline undergoes large-scale genome editing, rearranging precursor genome elements into a transcriptionally-active genome with thousands of gene-sized nanochromosomes. Transgenerationally-inherited RNAs, derived from the parental somatic genome, program the retention and reordering of germline fragments. Retained and eliminated DNA must be distinguished and processed separately, but the role of chromatin organization in this process is unknown. We developed tools for studying Oxytricha nuclei and apply them to map the 3D organization of precursor and developmental states using Hi-C. We find that the precursor conformation primes the germline for development, while a massive spatial reorganization during development differentiates retained from eliminated regions before DNA rearrangement. Further experiments suggest a role for RNA-DNA interactions and chromatin remodeling in this process, implying a critical role for 3D architecture in programmed genome rearrangement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381a65f6ebad009521fa755048f58360bf57403c" target='_blank'>
              Widespread 3D genome reorganization precedes programmed DNA rearrangement in Oxytricha trifallax
              </a>
            </td>
          <td>
            D. Villano, Manasa Prahlad, Ankush Singhal, K. Sanbonmatsu, L. Landweber
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="High grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy, killing more than 9,000 women each year in the United States alone. Nearly 80% of patients with HGSOC tumors will experience recurrence within 5 years, but little is known about the mechanisms that drive this process. Intratumor heterogeneity is believed to be a key feature of recurrence and resistance in HGSOC tumors, with early studies reporting diverse and complex mechanisms of the seeding of metastatic sites, including metastasis reseeding the primary tumor. Few studies have investigated temporal changes to clonality and structural variants through disease recurrence and the development of chemoresistance as surgical debulking is not frequently performed at this later stage. We performed multi-omic profiling in paired chemo-naive and chemoresistant tumors from 32 HGSOC patients to investigate the mutational and genomic landscape of disease progression. Somatic mutation profiles were largely conserved through disease progression. Mutational burdens did not significantly differ across recurrence but were driven by homologous recombination repair deficiency status. Clonal composition and dynamics were measured through variant allele frequency alterations as tumors progressed from primary chemo-naïve to recurrent chemo-resistant tumors. A novel candidate driver gene, MDC1, from the homologous recombination repair pathway, was significantly mutated and over-represented in patients with homologous recombination proficient tumors, with somatic mutations clustered in a single exon. Tumor evolution and phylogeny revealed that few changes in clonal abundance and complexity occur across the disease course in these patients. Taking structural variants into account, homologous recombination repair proficient (HRP) tumors tend to be polyclonal while homologous recombination repair deficient (HRD) tumors tend to be monoclonal, accompanied by a longer progression-free survival than the HRP patients. Three distinct classes of tumors were identified by structural variant signature analysis: tumors defined by DNA losses, tumors defined by DNA gains, and tumors defined by copy number neutral changes, which were largely defined by HRD status. Each class displayed distinct regions of the chromosome that were frequently affected by large scale SV events (>5Mb). Although no regions were frequently altered in recurrent tumors, GO analysis revealed that recurrent tumors have a significantly reduced immune response, which was not seen in the primary tumors. Ultra long read sequencing validated a majority of the SVs identified in short read sequencing and identified additional SVs undetected by short reads. These analyses identify that the phenotype of high grade serous ovarian tumors as defined by mutation and clonality profiles is established early in disease development and remain largely unchanged through chemotherapy and recurrence. This, when considered with the significant inter-patient heterogeneity identified in HGSOC, demonstrates the need for personalized therapies based on tumor profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="All cells possess mechanisms to maintain and replicate their genomes, whose integrity and transmission are constantly challenged by DNA damage and replication impediments. In eukaryotes, the protein kinase Ataxia-Telangiectasia and Rad3-related (ATR), a member of the phosphatidylinositol 3-kinase-like family acts as a master regulator of the eukaryotic response to DNA injuries, ensuring DNA replication completion and genome stability. Here we aimed to investigate the functional relevance of the ATR homolog in the DNA metabolism of Leishmania major, a protozoan parasite with a remarkably plastic genome. CRISPR/cas9 genome editing was used to generate a Myc-tagged ATR cell line (mycATR), and a Myc-tagged C-terminal knockout of ATR (mycATRΔC-/-). We show that the nuclear localisation of ATR depends upon its C-terminus. Moreover, its deletion results in single-stranded DNA accumulation, impaired cell cycle control, increased levels of DNA damage, and delayed DNA replication restart after replication stress. In addition, we show that ATR plays a key role in maintaining L. major’s unusual DNA replication program, where larger chromosomes duplicate later than smaller chromosomes. Our data reveals loss of the ATR C-terminus promotes the accumulation of replication signal around replicative stress fragile sites, which are enriched in larger chromosomes. Finally, we show that these alterations to the DNA replication program promote chromosome instability. In summary, our work shows that ATR acts to moderate DNA replication timing thus limiting the plasticity of the Leishmania genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3499060635df2f56eb484fb5207c0367f3df1797" target='_blank'>
              ATR, a DNA damage kinase, modulates DNA replication timing in Leishmania major
              </a>
            </td>
          <td>
            Gabriel L. A. da Silva, J. Damasceno, J. A. Black, R. McCulloch, L. Tosi
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 241


 Background:
 In prostate cancer, identifying biomarkers to predict patients who are at higher risk for metastatic disease remains a key question. The interaction of oncogenes such as
 MYC
 with the adaptive immune system may be a potential marker for worse progression; for example,
 MYC
 -amplified neuroblastoma has been shown to be immunologically cold both in vitro and with respect to the amount of adaptive immune receptor (IR) recombinations recovered from genomics files. Thus, we performed a correlative study identifying the relationship between
 MYC
 amplification and detection of adaptive IR recombinations in genomics files in prostate cancer.
 Methods:
 We utilized datasets for this study: The Cancer Genome Atlas-Prostate Adenocarcinoma (TCGA-PRAD, n = 475), Count Me In-Metastatic Prostate Cancer (CMI-MPC, n= 63) and West Coast Dream Team – Metastatic Castration Resistant Prostate Cancer (WCDT-MCRPC, n = 76). Copy number variation (CNV) assessments were done for the cases in these studies, identifying oncogene amplification as those with high ratios of exome, tumor to blood sequencing read counts for a given gene. We also mined productive IR recombination sequencing reads from RNAseq files, with associated V and J-IDs, as described in Chobrutskiy et al. 2020, which yielded reads for
 TRA
 ,
 TRB
 ,
 TRD
 ,
 TRG
 ,
 IGH
 ,
 IGK
 , and
 IGL
 . Then, MYC-amplified cases and MYC-deficient cases were compared with a two-proportion test for progression-free survival and with a t-test for the counts of IR receptors.
 Results:
 We first determined that, for TCGA-PRAD dataset, cases representing the top 20
 MYC
 read tumor to blood count ratios, in comparison to all other cases, represented a poorer progression free survival (p = 0.01) at the 70-month timepoint. We also noted that average
 MYC
 CN in both metastatic prostate cancer datasets (CMI-MPC and WCDT-MCRPC) was higher than in the TCGA-PRAD dataset, which represented primary tumor samples (p < 0.001 for both analyses). Further, for the WCDT-MCRPC dataset, we found that
 MYC
 -amplified cases were associated with significantly reduced
 IGH
 ,
 IGK
 , and
 IGL
 recombination reads in RNA-seq files (p = 0.005, 0.01, 0.004 respectively). For the WCDT-MCPRC dataset, we found that
 MYC
 -amplified cases were associated with reduced
 TRA
 and
 TRB
 recombination reads in WGS files (p = 0.008 and 0.017 respectively).
 Conclusions:
 This report further evidences the negative effects of MYC amplification with regard to immune evasion and being a key driver of prostate cancer metastasis. This report thus supports risk stratification by MYC-amplification, as well as noting that patients with MYC-amplification may benefit more from immune-boosting therapies. In particular, this report identifies immunoglobulins and by proxy, B-cells, as a major deficiency in those with MYC-amplified prostate cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95aa7d35ecefdd4814d5a8d637f37fa162af89f5" target='_blank'>
              Characterizing immunogenicity of
 MYC-
 amplified, metastatic prostate cancer as reflected by the recovery rates of adaptive immune receptor recombinations from RNAseq files.
              </a>
            </td>
          <td>
            Sunny Kahlon, Vayda R. Barker, Mallika Varkhedi, Alex Wang, Taha I. Huda, George Blanck
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Centromeres are chromosomal loci that ensure proper chromosome segregation by providing a platform for kinetochore assembly and spindle force transduction during cell division. Human centromeres are defined primarily by a unique chromatin domain featuring the histone H3 variant, Centromere Protein A (CENP-A), that marks a single active centromere locus per chromosome. CENP-A chromatin typically occupies a small subregion of low DNA methylation within multi-megabase arrays of hypermethylated alpha-satellite repeats and constitutive pericentric heterochromatin. However, the mechanisms defining and maintaining precise centromere position and domain size, and the role of the underlying alpha satellite DNA sequence, are poorly characterised. Using an experimentally-induced neocentromere in RPE1 cells, we discovered that the SUV39H1 and H2 methyltransferases tri-methylate H3K9 at neocentromere boundaries to maintain CENP-A domain size independent of DNA methylation or satellite sequences. Furthermore, we found that the CENP-A domain at canonical alpha-satellite-based centromeres is characterized by local depletion of H3K9me3-mediated heterochromatin, coinciding with the DNA methylation dip region. We identified the SETDB1 methyltransferase as key to maintaining H3K9me3 within flanking active higher order alpha satellite arrays while SUV39s and SUZ12 contribute to globally heterochromatinize both alpha satellites and neighbouring repeats. Loss of this heterochromatin boundary results in the progressive expansion of the primary CENP-A domain, erosion of DNA methylation, and the nucleation of new centromeres across alpha satellite sequences. Our study identifies the functional specialization of different H3K9 methyltransferases across centromeric and pericentric domains, crucial for maintaining centromere domain size and number.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21ac02ba5e831d24deadbae2a16d8fc0cb0af4d" target='_blank'>
              Heterochromatin boundaries maintain centromere position, size and number
              </a>
            </td>
          <td>
            Ben L. Carty, Danilo Dubocanin, Marina Murillo-Pineda, Marie Dumont, Emilia Volpe, Pawel Mikulski, Julia Humes, Oliver Whittingham, D. Fachinetti, Simona Giunta, Nicolas Altemose, L. Jansen
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="One fundamental yet open question is how eukaryotic chromosomes fold into segregated territories, a process essential for gene transcription and cell fate. Through analyzing Hi-C and chromatin-tracing DNA-FISH data, we identify long-range chromo skeleton loop structures that span over 100 Mb, extending beyond the reach of loop extrusion models. Spatial density analyses point to assembly formation independent of major nuclear structures. A subset of genomic loci serves as nucleation centers, driving loop clustering. These complexes are highly stable, as shown by live-cell imaging with sequence-specific fluorescent labeling, and biophysical model analyses reveal a multivalent binding mechanism. Our findings suggest a redundant, distributed cluster mechanism that ensures robustness across cell types and against mutations, guiding both chromosome compaction and the formation of smaller-scale chromosomal structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04b5ac6a8c671d63e4c9bd85fcca4ea46155b48" target='_blank'>
              Long-range genomic loci stochastically assemble into combinatorial forms of chromosome skeleton
              </a>
            </td>
          <td>
            Jingyu Zhang, Siyuan Wang, Simon C. Watkins, Jianhua Xing
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06ca40e33657af743c749690c1200f7526a655d0" target='_blank'>
              NKX6.3 modulation of mitotic dynamics and genomic stability in gastric carcinogenesis
              </a>
            </td>
          <td>
            J. H. Yoon, J. Kim, J. Eun, H. Ashktorab, Duane T. Smoot, S. Nam, Won Sang Park
          </td>
          <td>2025-01-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Two decades after the initial report on increased micronuclei in human chronic liver disease (CLD) and hepatocellular carcinoma (HCC), their role in HCC development is still poorly understood. Here we show that micronuclei in hepatocytes trigger hepatic immune response and promote HCC development via an increased level of extrachromosomal circular DNA (eccDNA). Livers from a CLD model (Mcl1Δhep mice) show increased micronuclei in parallel to eccDNA. Circular sequencing confirms higher eccDNA levels in micronuclei compared to primary nuclei. We developed nuclei-segregated DNA fiber (NuSeF) assay to show that micronuclei are more susceptible to replication stress, showing increased replication fork slowing. By comparing different murine liver disease models, high eccDNA is correlated with increased tumor incidence. EccDNA is a strong immunostimulant and promotes a crosstalk between hepatocytes and immune cells through the cGAS-STING pathway. Deletion of Sting1 in Mcl1Δhep mice reduces immune cell chemotaxis, as well as tumor incidence. Our findings suggest that eccDNA from micronuclei mediates inflammation-driven liver carcinogenesis in CLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Ductal carcinoma in situ (DCIS) harbors genomic copy number changes reflecting early events in advancing invasive breast cancer, and changes in specific cell states related to cancer initiation and progression. DNA methylation is a covalent modification to cytosines with cell type specificity, is stable in archival samples, and provides most of the specificity in multiple noninvasive pan-cancer screens. However, the creation of single-cell DNA methylomes is technically challenging and analysis is immature, leaving cell type and state level interpretation understudied. This has led to a gap in knowledge: it is not understood how the methylome reacts to gene dosage changes brought on by early copy number events, and it is not known how long these potential methylation changes persist. To this end, we have created and optimized a high throughput assay to measure these methylation changes at single-cell level, pairing our analysis with a multimodal view of these early copy number events. Here we describe three major steps in our work on elucidating methylome response to early breast cancer events: 1) We have generated methylome libraries for breast cancer cell lines with known large scale copy number alterations and validated our copy number calling in methylomes with those of gold-standard single-cell whole genome libraries. 2) We have generated thousands of single cell methylomes on normal breast samples attained from reduction surgeries paired with our groups’ work on the Human Breast Cell Atlas project. We were able to identify all major cell types of the human breast using methylation data for the first time. We used this data set to generate high-quality pseudobulk cell type specific methylomes. 3) We have profiled 5 DCIS and 1 synchronous invasive ductal carcinoma sample with subclonal copy number events, paired with single-cell transcriptomics. Through this we identified subclones in our methylome libraries and validated these through inferred copy number changes in our single-cell transcriptome libraries. We have uncovered thousands of hypomethylated regions enriched for specific transcription factor motifs, and widespread methylation changes across DCIS subclones. Notably many differential methylation changes occur outside of the genomic boundaries of copy number effects, suggesting distal changes by transcription factor gene dosage, chromatin perturbation, or tumor microenvironment interactions. These data will serve as a basis for future analyses into long-term follow through of archival samples and expand our understanding of this understudied epigenomic mark and its importance in early cancer evolution.
 Citation Format: Ryan M Mulqueen, Mariam Mosaad, Shanshan Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin. Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="In the malaria parasite Plasmodium falciparum, the expression of many genes is regulated by heterochromatin (HC) based on the histone mark tri-methylation of histone H3 lysine 9 (H3K9me3). HC assembly involves three distinct steps: de novo nucleation, spreading and maintenance. Nucleation, which consists in formation of HC in a previously euchromatic region, determines at which specific regions of the genome HC occurs. This process is not well understood in malaria parasites. Here we investigated the DNA sequence cis determinants of HC nucleation in P. falciparum, using a screening approach based on integration of fragments from different heterochromatic genes into an euchromatic locus, followed by H3K9me3 chromatin immunoprecipitation (ChIP) analysis. We found that fragments of var gene upstream regions nucleated HC efficiently, whereas fragments from the pfap2-g upstream region or from the mspdbl2 locus did not nucleate HC. Fragments from the beginning of the coding sequence (CDS) of pfap2-g nucleated HC with low efficiency, as evidenced by nucleation requiring long fragments of ∼2 Kb and occurring only in a fraction of the parasites. These results demonstrate that the primary DNA sequence is a main determinant of HC nucleation in P. falciparum. We also studied HC maintenance at the pfap2-g locus, which demonstrated that specific parts of the upstream region, different from the regions competent for HC nucleation, are required for maintenance. Together, our results provide initial insight into how HC is directed to specific loci and maintained in P. falciparum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64e9f134e6f82303aa4ebb3c8014708931204d53" target='_blank'>
              Heterochromatin de novo formation and maintenance in Plasmodium falciparum
              </a>
            </td>
          <td>
            Alba Pérez-Cantero, Oriol Llorà-Batlle, Íngrid Peláez-Conde, César Martínez-Guardiola, Alfred Cortés
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Wang Yuh-Hwa
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3d3384fa00e95094541c65edba3285e24efeab9" target='_blank'>
              Genetic regulation of TERT splicing affects cancer risk by altering cellular longevity and replicative potential
              </a>
            </td>
          <td>
            Oscar Florez-Vargas, Michelle Ho, Maxwell H Hogshead, B. Papenberg, Chia-Han Lee, Kaitlin Forsythe, Kristine Jones, Wen Luo, Kedest Teshome, Cornelis Blauwendraat, Kimberly J Billingsley, M. Kolmogorov, Melissa Meredith, B. Paten, Raj Chari, Chi Zhang, John S Schneekloth, M. Machiela, S. Chanock, S. Gadalla, S. Savage, Sam M Mbulaiteye, Ludmila Prokunina-Olsson
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Activation of synthetic centromeres on chromosome 4 in maize leads to its breakage and formation of trisomic fragments called neochromosomes. A limitation of neochromosomes is their low and unpredictable transmission rates due to trisomy. Here we report that selecting for dicentric recombinants through male crosses uncovers stabilized chromosome 4 fission events which split it into 4a-4b complementary chromosome pairs, where 4a carries a native centromere and 4b a synthetic one. The cells rapidly stabilized chromosome ends by de novo telomere formation and the new centromeres spread among genes without altering their expression. When both 4a and 4b chromosomes were present in a homozygous state, they segregated through meiosis indistinguishably from wild-type, and gave rise to healthy plants with normal seed set. This work leverages synthetic centromeres to engineer chromosome fission, effectively raising the diploid chromosome number of maize from 20 to 22. TEASER Using synthetic centromeres, we divide one maize chromosome into two chromosomes, and show that both chromosomes are fully functional.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa270994f0b80ef6d572c92b9c5ee380872d96e" target='_blank'>
              Increased maize chromosome number by engineered chromosome fission
              </a>
            </td>
          <td>
            Yibing Zeng, Mingyu Wang, J. Gent, R. Dawe
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-08</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Deletion of the long q arm of chromosome 22 (22qDEL) is the most frequently identified recurrent somatic copy number alteration observed in papillary thyroid carcinoma (PTC). Since its role in PTC is not fully understood, we conducted a pooled analysis of genomic characteristics and clinical correlates in 1094 primary tumors from four published PTC genomic studies. The majority of PTC cases with 22qDEL exhibited arm-level loss of heterozygosity (86%); nearly all PTC cases with 22qDEL had losses in 22q12 and 13, which together constitute 70% of the q arm. Our analysis confirmed that 22qDEL occurs more frequently with RAS point mutations (50.4%), particularly HRAS (70.3%), compared with other PTC drivers (9.3%), supporting the conclusion that 22qDEL is unlikely to be a solitary driver of PTC but possibly an important co-factor in carcinogenesis, particularly in PTCs with RAS driver mutations. Differential RNA expression analyses revealed downregulation of most genes located on chromosome 22 in cases with 22qDEL compared to those without 22qDEL. Many differentially expressed genes are drawn from immune response and regulation pathways. These findings highlight the value of further investigations into the contributions of 22qDEL events to PTC, perhaps mediated through immune perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0fd29e42888016677b1946cca20fdf0013b09" target='_blank'>
              Somatic copy number deletion of chromosome 22q in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Olivia W Lee, D. Karyadi, Stephen W Hartley, Weyin Zhou, M. Machiela, Shahriar A. Zamani, L. Zurnadzhy, John N Weinstein, Young Joo Park, Jeong-Sun Seo, Gerry A Thomas, Tetiana I Bogdanova, M. Tronko, Lindsay M. Morton, S. Chanock
          </td>
          <td>2025-01-01</td>
          <td>European Thyroid Journal</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genome-wide hypertranscription is common in human cancer and predicts poor prognosis. To understand how hypertranscription might drive cancer, we applied our formalin-fixed paraffin-embedded (FFPE)-cleavage under targeted accessible chromatin method for mapping RNA polymerase II (RNAPII) genome-wide in FFPE sections. We demonstrate global RNAPII elevations in mouse gliomas and assorted human tumors in small clinical samples and discover regional elevations corresponding to de novo HER2 amplifications punctuated by likely selective sweeps. RNAPII occupancy at S-phase-dependent histone genes correlated with WHO grade in meningiomas, accurately predicted rapid recurrence, and corresponded to whole-arm chromosome losses. Elevated RNAPII at histone genes in meningiomas and diverse breast cancers is consistent with histone production being rate-limiting for S-phase progression and histone gene hypertranscription driving overproliferation and aneuploidy in cancer, with general implications for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d46087a5dee58a70e59310b431eba431b7c66c3" target='_blank'>
              RNA polymerase II at histone genes predicts outcome in human cancer.
              </a>
            </td>
          <td>
            S. Henikoff, Ye Zheng, Ronald M. Paranal, Yiling Xu, Jacob E. Greene, J. Henikoff, Zachary R. Russell, Frank Szulzewsky, H. N. Thirimanne, S. Kugel, Eric C. Holland, K. Ahmad
          </td>
          <td>2025-01-02</td>
          <td>Science</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ(WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3985e13774d5b1194193c3f3fcbde897bc3a759f" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Bertram Aschenbrenner, Alexander Howard, Emily Lay, Emma Bailey, Natalia Krajic, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Mitotic fidelity depends on proper chromosome alignment at the spindle equator, a process known as chromosome congression, driven by well-established protein networks. Whereas RNA-binding proteins and noncoding RNAs have been implicated in cell division, their interplay during this process remains unknown. Here, we discover that RSRC2, an arginine/serine-rich RNA-binding protein, plays an essential role in cell division by interacting with the long non-coding RNA C1QTNF1-AS1. The loss of either RSRC2 or C1QTNF1-AS1 results in defects in chromosome congression and mitotic progression. We show that RSRC2 interacts with distinct sets of proteins involved in splicing and centrosome biogenesis, contributing to the fidelity of cell division through two different mechanisms: one linked to the splicing of mitotic regulators and the other by localising to mitotic centrosomes for which the interaction with the C1QTNF1-AS1 RNA is required. Our study uncovers RSRC2 as a new regulator of cell division and illustrates how RNA-protein complexes promote error-free mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb0d898e39c9e3e90e4923dfd61d6149838f2b92" target='_blank'>
              The RNA-binding protein RSRC2 promotes mitotic fidelity by interacting with the lncRNA C1QTNF1-AS1
              </a>
            </td>
          <td>
            Kaliya Georgieva, Alice O Coomer, Parnia Babaei, Giulia Guiducci, Martin Dodel, Eleni Maniati, Anisha Thind, S. Krishnamurthy, Anna Nawrocka, Sam Wallis, Alena Shkumatava, Jun Wang, F. Mardakheh, Lovorka Stojic
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 DNA methylation alterations are observed across multiple cancers, occurring early during tumor initiation. However, evolution and drivers of these changes remain poorly understood and difficult to assess at the clonal level. Consequently, we developed a method for profiling genome-wide DNA methylation from low DNA amounts and applied it to the organoid model of BRAFV600E oncogene-induced colon cancer (CC) tumorigenesis. Single organoid clones, derived from single stem cells, were profiled every month for 5 months to map nucleotide-level dynamic heterogeneity in methylation changes. Uninduced control organoids exhibit a pattern of epigenetic drift leading towards clonal convergence not until 5 months. Whereas BRAFV600E-induced organoids exhibit a step-wise divergence throughout the different time points, likely explained by progressive clonal takeovers. The methylation changes in BRAFV600E induced organoids are explained by the age-in-culture associated methylation changes; especially BRAFV600E did not induce epigenetic changes, as previously thought. The CpG sites affected in the control and BRAFV600E comparison were not shared but mapped to the same genomic elements, indicating common drivers of methylation changes regardless of oncogene. Investigation of promoter CpG-island methylation gains revealed that they are decoupled from replication. External validation in other pre-cancerous and CC datasets, and experimental cell line data sets, show that methylation gains at regulatory elements are a function of “biological aging” and independent of replication or cell division cycles. Our studies provide new insights into the early epigenetic changes involved in BRAFV600E driven carcinogenesis rooted in the age-dependent drivers of epigenetic alterations.
 Citation Format: Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, Thomas R. Pisanic II, Stephen B Baylin, Hariharan Easwaran. Promoter methylation gains in aging and cancer are independent of replication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, whose roles are under debate. We report that double strand breaks (DSB) induce de novo formation of chromatin loops, with the break positioned at the loop base. These loops form only in S/G2 phases and occur during repair via homologous recombination (HR), concomitant with DNA end resection and RAD51 assembly. RAD51 showed two-tiered accumulation around DSBs, with a broad (~Mb) domain arising from the homology search. This domain is regulated by cohesin unloader, is constrained by TAD boundaries, and it overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of NIPBL results in reduced HR, and this effect is more pronounced when the HR donor is far (~100 kb) from the break. Our data indicates that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions within the damaged TAD. One-Sentence Summary High spatiotemporal resolution analysis of double strand beak repair in 3D genome revealed the role of cohesin-driven loop extrusion in the homology search.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e040dd0709d7779dc334b56e7eb34dab3396724" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search.
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam Rybczynski, Namrata Nilavar, Daniel Nguyen, Violetta Karwacki-Neisius, Andrew G. Li, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The three-dimensional conformation and packaging of chromosomes modulates the spatial organization of the nucleus, orchestrating DNA replication and repair, maintaining genome stability and integrity, and regulating gene expression. Hi-C methods provide high-resolution data on chromatin-to-chromatin interactions both within and among chromosomes at a genome-wide scale. Hi-C resolves topologically-associated domains (TADs) and chromatin loops that are linked to cell-specific transcriptional control. We present a comprehensive Hi-C dataset generated from the frontal cortex of laboratory rats, encompassing a diverse group of inbred strains (SHR/OlaIpcv, BN-Lx/Cub, BXH6/Cub, HXB2/Ipcv, HXB10/Ipcv, HXB23/Ipcv, HXB31/Ipcv, LE/Stm, F344/Stm) and an F1 hybrid (SHR/Olalpcv x BN/NHsdMcwi). This dataset serves as a valuable resource for studying the mechanisms by which three-dimensional chromatin architecture governs gene expression in the brain. Strain-specific variations in genome organization can illuminate the influence of chromatin structure on gene expression, neuronal functionality, and the predisposition to neurological and behavioral disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aea183cfc31164b4b805a5b0b8bbd3945529688" target='_blank'>
              Hi-C sequencing data from cortex of laboratory rats
              </a>
            </td>
          <td>
            Panjun Kim, Rachel R Ward, Burt M. Sharp, Robert W. Williams, Hao Chen
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, , Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0491183160fe7bbdae2e8ae2e68e3e5bf8212367" target='_blank'>
              Intratumor heterogeneity of HPV integration in HPV-associated head and neck cancer
              </a>
            </td>
          <td>
            Noa Sasa, T. Kishikawa, Masashi Mori, R. Ito, Yumie Mizoro, Masami Suzuki, Hirotaka Eguchi, Hidenori Tanaka, T. Fukusumi, Motoyuki Suzuki, Y. Takenaka, K. Nimura, Yukinori Okada, H. Inohara
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is initiated during early mammalian embryogenesis by a long non-coding RNA (lncRNA) XIST, which coats one of the two X-chromosomes and facilitates epigenetic transcriptional silencing. A second, evolutionarily recent primate-specific lncRNA XACT was proposed to antagonize XIST’s ability to induce XCI. XACT expression is restricted to pluripotent states and early embryonic stages and coats the active X-chromosome in both females and males. Here, we report a novel XACT transcript expressed in normal and cancerous somatic cells from both the inactive (Xi) and active (Xa) X chromosomes. It coexists with XIST on the Xi without affecting XIST expression or (re)activating X-linked genes inactivated by XCI. During hematopoietic stem cell (HSC) differentiation, XACT is primarily expressed in myeloid progenitors in both sexes. XACT expression is activated in HSCs and peaks in megakaryocyte-erythrocyte progenitor cells (MEPs) before rapidly declining as the MEPs differentiate into megakaryocytes or erythrocytes. By combining CRISPR-based XACT perturbation with epigenomic and transcriptional studies, we revealed the key role of XACT in the self-renewal and differentiation of erythroid progenitors into erythrocytes, by recruiting cis-regulatory proteins and regulating transcription through ETS and AP-1 transcription factors. Furthermore, XACT is expressed in a subset of acute myeloid leukemia (AML) patients, with high levels found in erythroid-megakaryocytic blast cells; suggesting XACT’s potential as a marker for AML subtypes. Thus, we identified a novel XACT transcript with a unique expression profile and important roles in normal and cancer cells, revealing XACT lncRNA functions beyond its previously proposed role in XCI during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7754eb09126d1d587b2765e70f76183499d8bcbf" target='_blank'>
              A Novel XACT lncRNA Transcript with Functions Transcending X-Chromosome Inactivation
              </a>
            </td>
          <td>
            Peifen Zhang, Paula Haro Angles, Hong Zhang, Tingyue Li, Roger Mulet-Lazaro, W. V. van Ijcken, S. Philipsen, Ruud Delwel, J. Gribnau, D. Huylebroeck, Frank Grosveld, C. Rougeulle, Eskeatnaf Mulugeta
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are widely considered the most cytotoxic DNA lesions occurring in cells because they physically disrupt the connectivity of the DNA double helix. Homologous recombination (HR) is a high-fidelity DSB repair pathway that copies the sequence spanning the DNA break from a homologous template, most commonly the sister chromatid. How both DNA ends, and the sister chromatid are held in close proximity during HR is unknown. Here we demonstrate that the PST repeat region of MDC1 is a mutlivalent nucleosome binding domain, sufficient to tether chromatin in multiple contexts. In mitotic cells the affinity of the PST repeats for chromatin is downregulated by phosphorylation to prevent chromosome missegregation, while still contributing to DNA break tethering by the MDC1-TOPBP1-CIP2A complex. In interphase, the PST repeat region is critical for RAD51 focus formation but not the recruitment of 53BP1 to DNA breaks, consistent with a chromatin tethering function. In total, this work demonstrates that the PST repeat region of MDC1 is a multivalent chromatin binding domain with tunable affinity that contributes to DNA break tethering during HR and in mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dbf6e7c3f0e969876e0503353c0c93165dcec37" target='_blank'>
              The PST repeat region of MDC1 is a tunable multivalent chromatin tethering domain
              </a>
            </td>
          <td>
            Joshua R. Heyza, Maria Mikhova, Gloria I Perez, David G. Broadbent, Jens C. Schmidt
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Hepatocellular carcinoma is the most prevalent primary liver cancer and arises from hepatocytes, which are very metabolically active and contain abundant mitochondria. Despite extensive characterisation of nuclear driver genes and pathways, the role of crosstalk between the mitochondrial and nuclear genomes in mitochondrial mutagenesis remains poorly understood. To address this, we leveraged a chemical carcinogenesis model of liver cancer in four mouse strains. We analysed hundreds of tumours with paired whole-genome sequencing and total RNA sequencing to comprehensively characterise mitochondrial mutations and expression. Following de novo assembly of three new mitochondrial genomes, we devised an accurate and efficient heteroplasmy detection approach and analysed mitochondrial DNA (mtDNA) content and transcription. Across strains, there was high concordance of the number of heteroplasmies, mutation signatures, and variant allele frequencies. In contrast, the heteroplasmy rate and genic locations differed between strains, suggesting strain-specificity in DNA repair mechanisms in early tumour development. Independent of age or environmental factors, tumours had lower mtDNA content than adjacent normal liver, suggesting mitochondrial loss during tumourigenesis. Additionally, within individual strains, mtDNA content differences were associated with nuclear driver gene choice. Our comprehensive characterisation of mitochondrial genomes and expression patterns demonstrates that both germline and somatic genetic variation influence tumour mtDNA content, mutation patterns, and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7507e40e7369ffa2dd7aab3b39dffb6074ecd240" target='_blank'>
              Germline and somatic variation influence nuclear-mitochondrial crosstalk in tumourigenesis
              </a>
            </td>
          <td>
            Maëlle Daunesse, Vasavi Sundaram, Elissavet Kentepozidou, John Connelly, Craig J. Anderson, Núria López-Bigas, Colin A. Semple, D. T. Odom, Martin S. Taylor, Maša Roller, Sarah J. Aitken, Paul Flicek
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Solid tumors often have an abundance of collagen-I that stiffens the tissue, and they are invariably driven by mutations that include chromosome losses and gains. These observations are linked here by showing that 3D matrix stiffness induces heritable changes to a cell’s DNA. We use live-cell chromosome reporters (ChReporters) and hydrogels of tunable stiffness to show mitotic compression, micronuclei counts, ChReporter losses and heterogeneity all increase as functions of stiffness. Increased mistakes occur despite suppressed cell division in stiff matrix and minimal size variation between spheroids. Colonies of ChReporter-negative cells within cancer spheroids align with Luria-Delbruck’s seminal theory for heritable mutations, which predicts inter-spheroid variances that exceed Poisson statistics. Suppression of the contractility motor Myosin-II also increases chromosome loss in 3D but not 2D and does not affect spheroid growth – thus clarifying Myosin-II’s putative role as a tumor suppressor. Consistent with experiments, pan-cancer analyses of clinical data associates chromosome losses and gains with collagen-I levels and genetic variation. Stiff extracellular matrix thus drives mechano-evolution of solid tumors as a Darwin-Lamarck process with heterogeneity that complicates therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal and mobile genetic elements, including transposable elements, plasmids, and accessory chromosomes, are found across all domains of life. The replication and segregation of elements integrated into the host chromosome are ensured by the host life cycle. In contrast, the persistence of autonomously replicating elements relies on their ability to remain within the host population. Here, we assess the evolutionary success of different persistence strategies of prokaryotic plasmids. Through intracellular competitions between plasmid genotypes, we find that combining active partitioning during cell division with a toxin-antitoxin (TA) system for post-segregational killing is more effective than either strategy alone. Mathematical modeling of long-term plasmid evolution, calibrated with empirical plasmid persistence probabilities, further supports these findings. A survey of enterobacterial genomes indicates that partitioning and TA systems are core features of large conjugative plasmids. Indeed, we confirm the presence of a previously unrecognized type I TA system in IncX3 plasmids, which serve as important vectors of antibiotic resistance in human pathogens. These findings suggest that large conjugative plasmids likely encode TA systems, including those yet to be identified. The combination of TA and partitioning systems emerges as the most effective strategy for the persistence of low-copy extrachromosomal elements with unstable segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95b0cde585ef4a9250597f7819434970a3e22ed3" target='_blank'>
              Persistence of large conjugative plasmids relies on the combination of active partitioning and addiction mechanisms
              </a>
            </td>
          <td>
            Johannes Effe, M. Santer, Yiqing Wang, Theresa E. Feenstra, Nils F. Hülter, Tal Dagan
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2326c18bde268cb024eef77addf1c66d8cdb8f" target='_blank'>
              KDM6A facilitates Xist upregulation at the onset of X inactivation
              </a>
            </td>
          <td>
            Josephine Lin, Jinli Zhang, Li Ma, H. Fang, Rui Ma, Camille Groneck, Galina N. Filippova, Xinxian Deng, Chizuru Kinoshita, Jessica E. Young, Wenxiu Ma, Christine M. Disteche, J. B. Berletch
          </td>
          <td>2025-01-03</td>
          <td>Biology of Sex Differences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Efficient DNA lesion repair is crucial for cell survival, especially within actively transcribed DNA regions that contain essential genetic information. Additionally, DNA breaks in regions of active transcription are prone to generating insertions and deletions, which are hallmark features of cancer genomes. Cockayne syndrome protein B (CSB) is the sole ATP-dependent chromatin remodeler that is essential for coupling DNA repair pathways with transcription, leading to more efficient DNA repair in regions of active transcription. CSB is best known for its essential function in transcription-coupled nucleotide excision repair (TC-NER), a process that rapidly removes helix-distorting DNA lesions that stall RNA polymerase II, such as those created by chemotherapeutic platinum compounds and UV irradiation. In addition to NER, CSB has also been reported to couple homologous recombination to transcription. Most recently, CSB has also been shown to couple single-strand DNA break repair to transcription. In this review, we will discuss the overlapping and distinct mechanisms by which CSB couples these different DNA repair pathways to transcription. We will also discuss how these CSB functions may account for Cockayne syndrome and the emerging roles of CSB as an innovative target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59205832acc879122f4c23f208dc50f2ad1f678e" target='_blank'>
              ATP-Dependent Chromatin Remodeler CSB Couples DNA Repair Pathways to Transcription with Implications for Cockayne Syndrome and Cancer Therapy
              </a>
            </td>
          <td>
            Rabeya Bilkis, Robert J. Lake, Hua-Ying Fan
          </td>
          <td>2025-02-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="New therapeutic approaches that antagonize tumour-promoting phenotypes in lung cancer are needed to improve patient outcomes. Chromosomal instability (CIN) is a hallmark of lung cancer characterized by the ongoing acquisition of genetic alterations that include the gain and loss of whole chromosomes or segments of chromosomes as well as chromosomal rearrangements during cell division. Although it provides genetic diversity that fuels tumour evolution and enables the acquisition of aggressive phenotypes like immune evasion, metastasis, and drug resistance, too much CIN can be lethal because it creates genetic imbalances that disrupt essential genes and induce severe proteotoxic and metabolic stress. As such, sustaining advantageous levels of CIN that are compatible with survival is a fine balance in cancer cells, and potentiating CIN to levels that exceed a tolerable threshold is a promising treatment strategy for inherently unstable tumours like lung cancer. Kinesins are a superfamily of motor proteins with many members having functions in mitosis that are critical for the correct segregation of chromosomes and, consequently, maintaining genomic integrity. Accordingly, inhibition of such kinesins has been shown to exacerbate CIN. Therefore, inhibiting mitotic kinesins represents a promising strategy for amplifying CIN to lethal levels in vulnerable cancer cells. In this review, we describe the concept of CIN as a therapeutic vulnerability and comprehensively summarize studies reporting the clinical and functional relevance of kinesins in lung cancer, with the goal of outlining how kinesin inhibition, or “targeting kinesins”, holds great potential as an effective strategy for treating lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-18</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial–mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Approximately 97% of the human genome comprises non-coding sequences, with nearly half originating from transposable elements. Among these, retrotransposons represent a critical subclass that replicates via a "copy-and-paste" mechanism and significantly influences the regulation of host genomes. In both normal and pathological contexts, retrotransposons contribute a vast reservoir of regulatory elements that can modulate the expression of genes. If left unchecked, retrotransposons can substantially affect host transcriptional programs and genomic integrity. Therefore, various mechanisms, including epigenetic modifications, are employed to mitigate their potentially deleterious effects. In diseases such as cancers, the epigenome is often significantly reprogrammed, which can lead to retrotransposon dysregulation. Drawing insights from recent studies conducted in human and murine cells, this review examines how retrotransposons expand the complexity of mammalian genomes, describes the impact of their epigenetic dysregulation in cancer development, and highlights the potential of targeting these sequences for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, but the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a hypermutation phenomenon at transcription start sites in the human germline, spanning several hundred base pairs in both directions. We link this TSS mutational hotspot to divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, revealing a recombination-independent mechanism distinct from known processes. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants often filtered out in de novo mutation calls, indicating that the hotspot arises during early embryogenesis. Mutational signature analysis reinforces these findings and implicates alternative non-homologous end joining and maternal mutation clusters. Our study provides the first detailed description of a germline TSS mutation hotspot, with broad evolutionary and biomedical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3da1619adeabbe4ded6c11c45076e2a69d9a307" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fuelled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, V. Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b8ef7036ff1fb3a03cc5984eb57541db6e9f2c" target='_blank'>
              Functional screen identifies RBM42 as a mediator of oncogenic mRNA translation specificity.
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2025-02-04</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Because transposable elements (TEs) can cause heritable genetic changes, past work on TE mobility in Arabidopsis thaliana has mostly focused on new TE insertions in the germline of hypomethylated plants. It is, however, well-known that TEs can also be active in the soma, although the high-confidence detection of somatic events has been challenging. Here, we leveraged the high accuracy of PacBio HiFi long reads to evaluate the somatic mobility of TEs in individuals of an A. thaliana non-reference strain lacking activity of METHYLTRANSFERASE1 (MET1), a major component of the DNA methylation maintenance machinery. Most somatically mobile families coincided with those found in germline studies of hypomethylated genotypes, although the exact TE copies differed. We also discovered mobile elements that had been missed by standard TE annotation methods. Somatic TE activity was variable among individual plants, but also within TE families. Finally, our approach pointed to the possible involvement of alternative transposition as a cause for somatic hypermutability in a region that contains two closely spaced VANDAL21 elements. We conclude that long-read sequencing can reveal widespread TE transposition in the soma of A. thaliana hypomethylated mutants. Assessing somatic instead of germline mobilization is a fast and reliable method to investigate different aspects of TE mobility at the single plant level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62607e49f83191e3ad9371253c4b7877fe10686a" target='_blank'>
              Long-read detection of transposable element mobilization in the soma of hypomethylated Arabidopsis thaliana individuals
              </a>
            </td>
          <td>
            Andrea Movilli, Svitlana Sushko, Fernando A. Rabanal, Detlef Weigel
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, Gary C. Hon
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="PURPOSE
Previous research has shown, that ABC transporters gene expression level can predict the efficacy of therapy. However, other mechanisms of gene activity are rarely considered, especially in non-small cell lung cancer (NSCLC). Thus, the purpose of the work was to assess chromosomal aberrations of all 49 ABC transporters genes and the expression levels of some ABC genes, as well as their correlation with survival.


MATERIALS AND METHODS
The surgical material of 104 patients with NSCLC was used in this study. Treatment included surgery and 3 courses of adjuvant chemotherapy with "platinum doublets". DNA and RNA were isolated from the samples, followed by microarray analysis to assess the expression and chromosomal aberrations (deletions and amplifications) of ABC genes.


RESULTS
Metastatic-free survival (MFS) was higher with ABCC1, ABCC2, and ABCG1 hypoexpression at a statistically significant level (p=0.01). The presence of deletion in ABCB1 correlates with 100% MFS (p=0.001). The survival rates with ABCG1 amplification are not higher than 45% (p<0.0001). ABCA11 deletion is associated with a low MFS rate (38%) versus 91% with normal copy number (p=0.006). ABCB9 analysis showed opposite results, with survival rates of 55% and 91% in the presence of amplification and normal copy number, respectively (p=0.006). ABCC subfamily genes showed a similar result in the presence of amplification, where ABCC3 and ABCC10 account for 64% and 60% survival, respectively (p=0.005, p=0.01).


CONCLUSION
Thus, not only expression but also chromosomal aberrations were found to be associated with patient survival. These findings could be a potential marker of metastatic-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1581dfdb2ff097ba8f6291fda2517229cc004d6d" target='_blank'>
              Influence of DNA Copy Number Aberrations in ABC Transporter Family Genes on the Survival of Patients with Primary Operatable Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Matvey Mikhailovich Tsyganov, M. Ibragimova, E. Rodionov, Anastavia Alekseevna Frolova, Irina Aleksandrovna Tsydenova, E. A. Lutzkaya, S. Miller
          </td>
          <td>2025-01-31</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 recruits the host protein CTCF to viral episomes to control the differentiation-dependent viral transcriptional programme. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18 may also manipulate CTCF to contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701c7f934a3a0661690be118144129e2077bd9be" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, Leanne S Jones, Joanne D. Stockton, C. Várnai, M. West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2025-01-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Telomere elongation is essential for the proliferation of cancer cells. Telomere length control is achieved by either the activation of the telomerase enzyme or the recombination-based Alternative Lengthening of Telomeres (ALT) pathway. ALT is active in about 10-15% of human cancers, but its molecular underpinnings remain poorly understood, preventing the discovery of potential novel therapeutic targets. Pooled CRISPR-based functional genomic screens enable the unbiased discovery of molecular factors involved in cancer biology. Recently, Optical Pooled Screens (OPS) have significantly extended the capabilities of pooled functional genomics screens to enable sensitive imaging-based readouts at the single cell level and large scale. To gain a better understanding of the ALT pathway, we developed a novel OPS assay that employs telomeric native DNA FISH (nFISH) as an optical quantitative readout to measure ALT activity. The assay uses standard OPS protocols for library preparation and sequencing. As a critical element, an optimized nFISH protocol is performed before in situ sequencing to maximize the assay performance. We show that the modified nFISH protocol faithfully detects changes in ALT activity upon CRISPR knock-out (KO) of the FANCM and BLM genes which were previously implicated in ALT. Overall, the OPS-nFISH assay is a reliable method that can provide deep insights into the ALT pathway in a high-throughput format.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="During classical non-homologous end joining (cNHEJ), DNA-dependent protein kinase (DNA-PK) encapsulates free DNA ends, forming a recruitment platform for downstream end-joining factors including Ligase 4 (LIG4)1. DNA-PK can also bind telomeres and regulate their resection2–4, but does not initiate cNHEJ at this position. How the end joining process is regulated in this context-specific manner is currently unclear. Here we show that the shelterin components TRF2 and RAP1 form a complex with DNA-PK that directly represses its end joining function at telomeres. Biochemical experiments and cryo-electron microscopy reveal that when bound to TRF2, RAP1 establishes a network of interactions with KU and DNA that prevents DNA-PK from recruiting LIG4. In mouse and human cells, RAP1 is redundant with the Apollo nuclease in repressing cNHEJ at chromosome ends, demonstrating that the inhibition of DNA-PK prevents telomere fusions in parallel with overhang-dependent mechanisms. Our experiments show that the end joining function of DNA-PK is directly and specifically repressed at telomeres, establishing a molecular mechanism for how individual linear chromosomes are maintained in mammalian cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797e7286a7a007c6d465328b33c7892e6e693028" target='_blank'>
              Chromosome end protection by RAP1-mediated inhibition of DNA-PK
              </a>
            </td>
          <td>
            Patrik Eickhoff, Ceylan Sonmez, Charlotte E.L. Fisher, Oviya Inian, Theodoros I. Roumeliotis, Angela dello Stritto, Jörg Mansfeld, J. Choudhary, Sebastian Guettler, Francisca Lottersberger, Max E Douglas
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18’s role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18−/− and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Chromosomes, as the fundamental unit of genetic material located within the cell nucleus, have undergone extensive and complex changes throughout the evolutionary history of eukaryotes. Many of these patterns and mechanisms of change share commonalities across various diseases, including cancer. For a long time, biologists were limited to research methods with relatively low resolution, such as fluorescence in situ hybridization (FISH). However, the rapid advancement of high-throughput sequencing technologies is revolutionizing our understanding of chromosomal variations across different species, among individuals of the same species, and even at the cellular level within a single individual. In this review, we focus on the chromosomal evolution in vertebrates, and provide an overview of the role of chromosom rearrangements in speciation, the molecular mechanisms of chromosomal rearrangements, the evolutionary patterns from ancestral chromosomes to extant chromosomes, and the significance of sex chromosomes as a general paradigm for studying chromosomal evolution. Finally, we discuss the new opportunities and challenges that synthetic biology brings to the field of chromosomal evolution research, with the aim of offering new insights and references for understanding and studying vertebrate chromosomal evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bbc3a9efead12e9b20f611725318c5fc08d7716" target='_blank'>
              The evolution of sequences and spatial conformation in vertebrate chromosomes.
              </a>
            </td>
          <td>
            Jing Liu, Qi Zhou
          </td>
          <td>2025-02-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b62083d3dd11f079792a9e41da3af07b399c20f" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Olivia Dove Estrella, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Tumor protein 53 (TP53) is a tumor repressor gene that is highly responsible for activating cell cycle arrest in the event of DNA damage, stopping the replication of damaged cells, which may cause cancer. However, when mutated, TP53 is an oncogene, with mutated variants of it being seen in over half of all cancers diagnosed including the aforementioned four. With groundbreaking CRISPR-Cas9 gene editing technology coming to the forefront of biotechnological research, the applications to cancer research and gene therapy have been increasing. This review explores CRISPR-Cas9 and its uses for treatment and medication investigation in various cancers, including lung, prostate, breast, and bone cancer through the lens of the TP53 gene. Due to TP53s commonality in cancer, CRISPR-Cas9 gene editing technology can be used to its maximum potential in this context as a treatment or investigative tool, with it yielding positive results in reverting tumor growth, as well as great versatility when it comes to screening related genes that inhibit or encourage tumor growth which can be targeted by drugs, or even the screening of the drugs effectiveness themselves. Thus, CRISPR-Cas9s applications in TP53-related cancer treatment and chemotherapeutic fields prove this technology to be successful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da087d15d0c931988cddd598ffd040ef96f3cb9b" target='_blank'>
              Using CRISPR/Cas9 Technology to Target TP53 Related Cancers
              </a>
            </td>
          <td>
            Man Ho Damien Hsieh
          </td>
          <td>2025-01-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Molecular targeted drugs are frequently used in precision cancer therapies. However, the prolonged administration of these agents can result in drug resistance. Cancer cells exhibiting defects in homologous recombination are particularly susceptible to genotoxic stress, such as that induced by PARP inhibitors (PARPi), resulting in synthetic lethality. Here, we show that sustained treatment of such cancer cells with PARPi selectively upregulates YAP1-2α, a minor isoform of YAP1. Elevated YAP1-2α heterodimerizes with TAZ to induce liquid-liquid phase separation (LLPS) in the nucleus. This LLPS-driven process generates nuclear condensates that activate a super-enhancer comprising YAP1-2α, TAZ, TEAD, and BRD4. The super-enhancer reconfigures transcriptional networks to enhance cancer stem-like malignant properties, thereby potentiating cell resilience and augmenting cell resistance to adverse conditions, including precision cancer therapies. Our study reveals the presence of a super-enhancer that promotes malignant tumor progression. Targeting the YAP1-2α/TAZ-induced super-enhancer is a promising strategy for overcoming resistance to anticancer modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0a7f11bb9d0a8943e8f9ed992501df776b3e17" target='_blank'>
              A super-enhancer actuated by heterodimerization of a specific YAP1 isoform, YAP1-2α, with TAZ endows cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 The lysine methyltransferase KMT2A/MLL1 plays a crucial role in normal development and hematopoiesis. Rearrangement of KMT2A results in the formation of a KMT2A fusion protein with aberrant activity, which is a well-characterized driver of leukemogenesis and is therapeutically targetable. In contrast to the widely studied KMT2A translocation, the molecular consequences of other KMT2A alterations, including KMT2A amplification, remain unknown. KMT2A amplification is detected in highly aggressive and treatment-resistant myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). It strongly associates with TP53 mutations and complex karyotypes. Unlike KMT2A rearrangement, KMT2A amplification is reported mostly in patients older than 65 and treated with alkylating agents. Here, we conducted a comprehensive molecular analysis of KMT2A-amplified MDS/AML by long-read nanopore sequencing. We have revealed the unique epigenetic landscape, including DNA methylation, histone modification, and chromatin accessibility, associated with KMT2A amplification. We also explored the potential of targeting KMT2A or KMT2A-medicated pathways to block leukemogenesis. These results provide a foundation for targeting the KMT2A complex in KMT2A-amplified disease, representing a promising first step toward treating MDS/AML associated with KMT2A amplification.
 Citation Format: Jayme Ogino, Yang Liu, Wentao Yang, Madina Sukhanova, Sheng Li, Yali Dou. Epigenetic dynamics in KMT2A-amplified leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245971e55eeef33c3dd5df76ff7015895c3cd83a" target='_blank'>
              Abstract A008: Epigenetic dynamics in KMT2A-amplified leukemia
              </a>
            </td>
          <td>
            Jayme Ogino, Yang Liu, Wentao Yang, M. Sukhanova, Sheng Li, Yali Dou
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70f85dd49412748b7fb2dbe319b405d34092c20d" target='_blank'>
              Multi-omics analysis reveals the panoramic picture of TOP2A in pan-cancer
              </a>
            </td>
          <td>
            Jin Zhang, Tianxiao Yang, Kenie Wang, Jie Pan, Junjun Qiu, Shengnai Zheng, Xuesong Li, Guanghao Li
          </td>
          <td>2025-02-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome functions are regulated by the means of chromatin and 3D chromosome organization. Dynamic condensation and relaxation of chromosomes during cell cycle is largely controlled by the Condensin complexes. We mapped mutants in Condensin II subunits SMC2A, CAP-D3 and CAP-H2 as hypersensitive to DNA-protein crosslink (DPC) inducer zebularine. This suggested that Condensin II complex is required for resistance to genotoxic stress in Arabidopsis and prompted us to explore the underlying phenotypes. We show that the Condensin II role in resistance to zebularine is independent of the DNA damage response signaling by SOG1 and the homology-directed repair pathway. Furthermore, we found that Arabidopsis Condensin II mutants have incompletely condensed mitotic chromosomes and show abnormal chromatin connections during anaphase. Based on the structural and temporal analyses, we propose that the connections represent catenated chromatids. The chromosome connections were more frequent when the Condensin II mutants were treated by zebularine or TOP2 crosslinker and inhibitor ICRF-187. Altogether, we demonstrate that the proper large-scale chromatin organization by Condensin II is important for resistance to DNA damage response-inducing agents in Arabidopsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6084d605a281f1ce54dec4a816d64fd93e825187" target='_blank'>
              Condensin II mediates efficient chromatid resolution and resistance to genotoxic stress in Arabidopsis
              </a>
            </td>
          <td>
            Jovanka Vladejić, Klára Procházková, E. Tomaštíková, Zynke de Cock, Jana Zwyrtková, Kateřina Kaduchová, Markéta Pavlíková, A. Pečinka
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Super-enhancers (SEs) are expansive cis-regulatory elements known for amplifying oncogene expression across various cancers. However, their role in cervical cancer (CC), a remarkable global malignancy affecting women, remains underexplored. Here we applied integrated epigenomic and transcriptomic profiling to delineate the distinct SE landscape in CC by analyzing paired tumor and normal tissues. Our study identifies a tumor-specific SE at the EFNA1 locus that drives EFNA1 expression in CC. Mechanically, the EFNA1 SE region contains consensus sequences for the transcription factor FOSL2, whose knockdown markedly suppressed luciferase activity and diminished H3K27ac enrichment within the SE region. Functional analyses further underlined EFNA1's oncogenic role in CC, linking its overexpression to poor patient outcomes. EFNA1 knockdown strikingly reduced CC cell proliferation, migration, and tumor growth. Moreover, EFNA1 cis-interacted with its receptor EphA2, leading to decreased EphA2 tyrosine phosphorylation and subsequent activation of the Src/AKT/STAT3 forward signaling pathway. Inhibition of this pathway with specific inhibitors substantially attenuated the tumorigenic capacity of EFNA1-overexpressing CC cells in both in vitro and in vivo models. Collectively, our study unveils the critical role of SEs in promoting tumor progression through the FOSL2-EFNA1-EphA2-Src/AKT/STAT3 axis, providing new prognostic and therapeutic avenues for CC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3e483c9466e24512f997df84699534d6031c04" target='_blank'>
              Super-enhancer-driven EFNA1 fuels tumor progression in cervical cancer via the FOSL2-Src/AKT/STAT3 axis.
              </a>
            </td>
          <td>
            Shu-Qiang Liu, Xi-Xi Cheng, Shuai He, Tao Xia, Yi-Qi Li, Wan Peng, Yaqing Zhou, Zi-Hao Xu, , Yang Liu, P.-P. Wei, Song-Hua Yuan, Chang Liu, Shu-Lan Sun, Dong-Ling Zou, Min Zheng, Chun-Yan Lan, Chun-Ling Luo, Jin-Xin Bei
          </td>
          <td>2025-02-18</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Human colorectal cancer (CRC) most often arises as a result of spontaneous genetic driver mutations such as APC, BRAF, KRAS, SMAD4, and TP53. These are among the most common drivers of CRC and are thought to contribute directly to cancer initiation and growth. CRC tumors harboring BRAFV600E mutation exhibit a CpG Island Methylator Phenotype (CIMP). Other drivers are also more weakly associated with distinct DNA methylation profiles. It is not clear whether these DNA methylation profiles are a result of differences in the cells-of-origin for these cancers, with pre-existing methylation patterns or whether there is selective pressure for various DNA methylation alterations that cooperate with each driver. To address this question, we have modeled this in mice by inducing colon tumors in a controlled cell population at the same age, with either BRAFF-V600E, KRASLSLG12D, Smad4, or Trp53 drivers in combination with Apc heterozygosity. We have profiled DNA methylation in tumors arising in these mice to determine the interaction between different driver mutations and the tumor methylome. Using a multiple linear regression model, we have identified DNA methylation alterations that are shared across all tumor samples, as well as changes that are specific to each of the different genetic drivers. The DNA methylation profiles accurately recapitulate well-established findings, including high levels of hypermethylation in KRASLSLG12D and BRAF F-V600E tumors reminiscent of CpG Island Methylator Phenotype (CIMP), universal tumor hypermethylation at binding sites for transcription factors involved in intestinal epithelial differentiation, and hypomethylation at AP1 complex component binding sites. Driver-specific histology findings broadly align with those observed in human cancers, such as sessile serrated morphology in BRAFF-V600E mutated tumors. The tumor models with different driver mutations also show variation in hypermethylation of polycomb repressive complex target sites and replicative history. We are using the differences in the DNA methylation profiles between the driver tumor models in conjunction with RNA-seq analysis and conservation of the alterations in human CRC to identify epigenetic events cooperative with the drivers and functionally relevant to tumorigenesis.
 Citation Format: Manpreet Kalkat, Toshinori Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich. Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Deregulated MYCN is a driver of aggressive pediatric and adult neuroendocrine tumors, but critical oncogenic processes downstream of MYCN remain poorly defined. In neuroblastoma, MYCN interacts with and activates the Aurora-A kinase. Here we show that Aurora-A is a CDK-activating kinase for CDK12 by phosphorylating T893 in the T-loop, thereby enhancing its kinase activity. Aurora-A-dependent activation of CDK12 controls phosphorylation of T4 of RNA polymerase and recruits transcription termination complexes, thereby preventing transcription-replication conflicts. Enhanced crosslinking and immunoprecipitation sequencing reveals that Aurora-A associates with splice sites on nascent RNA. RNA-bound Aurora-A is catalytically inactive. MYCN competes with RNA for binding to Aurora-A and displaces Aurora-A from RNA in cells, promoting its CDK12 kinase activity. Combining Aurora-A and CDK12 inhibition potently suppresses the growth of MYCN-amplified neuroblastoma cells and patient-derived xenografts. Our data demonstrate that an Aurora-A/CDK12-dependent transcription termination pathway is a critical and targetable dependency of MYCN-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5883a603efd4d96de04c7785382de3b59b13dcb8" target='_blank'>
              The MYCN/Aurora-A complex is a cyclin activating kinase for CDK12
              </a>
            </td>
          <td>
            Mareike Müller, Lorenz Eing, Leonie Uhl, Maximilian Schmitz, Iliyana Kaneva, Dominic P Byrne, Daniel Fleischhauer, S. G. Burgess, Sophia Fischer, Nick Gebauer, Vanessa Nancy-Portebois, I. H. Kaltheuner, Stefanie Anh Ha, Christina Schülein-Völk, Daniel Solvie, Mark W. Richards, Raphael Vidal, Matthias Brand, Dimitrios Papadopoulos, Richard Bayliss, P. Eyers, C. Eyers, Matthias Geyer, Martin Eilers, G. Büchel
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Selection of a suitable chromatin environment during retroviral integration is a tightly regulated and multilayered process that involves interplay between viral and host factors. However, whether intrinsic chromatin dynamics during mitosis modulate retroviral genome invasion is currently poorly described. Direct interaction between the spumaretrovirus prototype foamy virus (PFV) Gag protein and cellular chromatin has been described as a major determinant for integration site selection. A previous Gag chromatin-binding site (CBS)–nucleosome co-crystal structure revealed an interaction with the histone H2A-H2B acidic patch via a highly conserved arginine anchor residue. Yet, the molecular mechanisms regulating Gag-chromatin capture during PFV infection remain obscure. Here, we investigated the kinetics of Gag-chromatin interactions during mitosis and proviral integration of PFV-infected synchronized cells. Using Gag CBS variant viruses, we showed that alteration of Gag affinity for nucleosome binding induced untimely chromatin tethering during mitosis, decreased infectivity and redistributed viral integration sites to markers associated with late replication timing of host chromosomes. Mutant Gag proteins were moreover defective in their ability to displace the histone H4 tail from the nucleosome acidic patch of highly condensed mitotic chromatin. These data indicate that the mitotic chromatin landscape during Gag–nucleosome interactions hosts PFV integration site selection determinants and that spumaretroviruses evolved high-affinity chromatin binding to overcome early mitosis chromatin condensation for optimal viral DNA tethering, integration and infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c034dba5335395c865d3f3e018bafaae4ee93b" target='_blank'>
              Timed chromatin invasion during mitosis governs prototype foamy virus integration site selection and infectivity
              </a>
            </td>
          <td>
            F. Lagadec, P. K. Singh, C. Calmels, D. Lapaillerie, D. Lindemann, V. Parissi, P. Cherepanov, A. N. Engelman, P. Lesbats
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab65a9ee1a176544b4d48a367f9b18440b017c" target='_blank'>
              R-loops acted on by RNase H1 influence DNA replication timing and genome stability in Leishmania
              </a>
            </td>
          <td>
            J. Damasceno, E. Briggs, M. Krasiļņikova, Catarina A Marques, C. Lapsley, R. McCulloch
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Transposable elements (TEs) comprise the majority of the human genome. In most somatic tissues, TEs are silenced by DNA methylation and histone modifications. Tumors exhibit changes in DNA methylation including global loss of methylation at regions that are silenced for genome stability, like TEs, and gain of methylation at the promoter regions of tumor suppressor genes. We showed that low doses of DNMT inhibitors (DNMTis) activate type I interferon signaling by reducing methylation and increasing expression of TEs that activate dsRNA sensors. DNMTis increase TEs in murine models of cancer, activating interferon signaling and recruiting CD8+ T cells to kill cancer cells and sensitize to immune checkpoint blockade therapy. We performed a genome-wide analysis of the response to DNMTi in a panel of ovarian cancer cell lines. Alu elements, which directly bind to the dsRNA sensor MDA5, are the most significantly demethylated TEs after DNMTi treatment. Our RNA-sequencing analyses indicated that A-to-I RNA editing of TEs by the RNA editing enzyme ADAR1 was increased after DNMTi treatment. This A-to-I editing makes TE RNA less immunogenic and inhibits the immune response to DNMTi. We find that combining ADAR1 knockdown and DNMTi treatment significantly increases type I interferon signaling and cancer cell secretion of downstream cytokines, including CCL5 and CXCL10. Further, this combination increased recruitment and activation of host immune cells and significantly improved survival in the aggressive ID8 Trp53-/- murine model of ovarian cancer. CITE-Seq (cellular indexing of transcriptomes and epitopes, a type of single-cell RNA sequencing that utilizes antibody tagging of each cell type before sequencing) demonstrated significantly higher infiltration of T and B cells in the tumor microenvironment. To understand the immune cell types causing the DNMTi anti-tumor response, we used antibodies to deplete CD8 T cells and NK cells in the ID8 Trp53-/- model with ADAR1 knockdown and DNMTi treatment. In addition, we performed these experiments in the C57BL/6-Tg(IghelMD4)4Ccg/J mice, which only produce antibodies against “Hel” antigen and have no other B cell responses. Of these lymphocyte populations, depleting T cells had the biggest effect and we observed specific killing of tumor cells by T cells isolated from the mice. We performed immunopeptidomics on cancer cell lines with and without DNMTi and showed that DNMTi upregulate TE peptide presentation by MHC molecules on the surface of tumor cells. Using a novel immuno-engineering approach, we are able to generate antigen-specific T cells targeting TEs and other antigens in murine and human cancer cell line models. These T cells specifically recognize (produce IFNgamma and TNFalpha) and kill the target tumor cells. As DNMTi have also been shown to demethylate genes crucial for T cell function, including IFNgamma and TCF7, we are currently combining these T cells with DNMTis for a novel cell therapy against cancer.
 Citation Format: Katherine B Chiappinelli. Epigenetic regulation of transposable elements to reverse cancer immune evasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Gene expression is abnormal in disease compared to normal tissue same as the regulatory elements. Regulatory element binding with transcription factors managed transcription of gene, which usually require chromatin accessible.To reveal potential epigenetic mechanism during hydronephrosis, we first used RNA-seq to finger out the disfunction genes during hydronephrosis, then combined with ATAC-seq, and BS-seq to reveal the related disfunction regulatory elements.Finally, we find that 860 differentially genes and 2429 dynamic chromatin open regions between normal and hydronephrosis tissue. Though, most of disfunction genes and regulatory elements significantly enriched in chronic kidney disease GO term, only small part of regulatory element target genes overlapped with truly disfunction genes. And we also find out an important gene OTUD6B, which overexpression in disease tissue is manipulated by distal regulatory element through chromatin loop, and confirm the importance of epigenetic mechanism in disease.In summary, we found many hub genes and potential therapeutic target during hydronephrosis, and also confirmed that epigenetic play important role in gene expression and relevant in disease progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f83d4eab3a922e50690ecc129e3b53c9f6fd24" target='_blank'>
              Alteration of chromatin states perturb the transcription regulation of gene during hydronephrosis
              </a>
            </td>
          <td>
            Xiaohui Wang, Shu-Feng Zhang, Hai-Ying Wu, Jian Gao, Lin Wang, Yao Yin, Xuhui Wang
          </td>
          <td>2025-02-17</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Whole-genome duplications (WGD) and transposable element (TE) activity can act synergistically in genome evolution. WGDs can increase TE activity directly through cellular stress or indirectly by relaxing selection against TE insertions in functionally redundant, duplicated regions. Because TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE activity following WGD are therefore likely to impact evolution of gene regulation. Yet, the role of TEs in genome regulatory evolution after WGDs is not well understood. Here we used Atlantic salmon as a model system to explore how TE activity after the salmonid WGD ~100MYA shaped CRE evolution. We identified 55,080 putative TE-CREs using chromatin accessibility data from liver and brain. Retroelements were both the dominant source of TE-CREs and had higher regulatory activity in MPRA experiments compared to DNA elements. A minority of TE-subfamilies (16%) accounted for 46% of TE-CREs, but these "CRE-superspreaders" were mostly active prior to the WGD. Analysis of individual TE insertions, however, revealed enrichment of TE-CREs originating from WGD-associated TE activity, particularly for the DTT (Tc1-Mariner) DNA elements. Furthermore, coexpression analyses supported the presence of TE-driven gene regulatory network evolution, including DTT elements active at the time of WGD. In conclusion, our study supports a scenario where TE activity has been important in genome regulatory evolution, either through relaxed selective constraints, or strong selection to recalibrate optimal gene expression phenotypes, during a transient period following genome doubling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08496aae0238cce493a4d3baa60b2ce62c4ee152" target='_blank'>
              The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication.
              </a>
            </td>
          <td>
            Oystein Monsen, L. Grønvold, Alex Datsomor, T. Harvey, James Kijas, Alexander Suh, T. R. Hvidsten, S. Sandve
          </td>
          <td>2025-02-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal (CRC), stomach (STAD) and endometrial (UCEC) cancers, particularly in microsatellite instable (MSI) tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Results Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors. Our analyses reveal that tumor STR profiles effectively capture both MSI phenotype and population structure. While most eSTRs are cancer-specific, shared eSTRs across multiple cancers exhibit consistent effects on gene expression. Notably, coding-region eSTRs identified in all three cancer types show positive correlations with nearby gene expression. We further validate the functional effects of eSTRs by demonstrating associations between somatic eSTR mutations and gene expression changes during the transition from normal to tumor tissues, suggesting their potential roles in tumorigenesis. Combined with DNA methylation data, we perform the first quantitative analysis of the interplay between STR variations and DNA methylation in tumors. We identify eSTRs where repeat lengths are associated with methylation levels of nearby CpG sites (meSTRs) and show that over 70% of eSTRs are significantly linked to local DNA methylation. Importantly, the effects of meSTRs on DNA methylation remain consistent across cancer types. Conclusions Overall, our findings enhance the understanding of how functional STR variations influence gene expression and DNA methylation. Our study highlights shared regulatory mechanisms of STRs across multiple cancers, offering a foundation for future research into their broader implications in tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a625ce20d0d6f7f915320a2751a41cca4a94608" target='_blank'>
              Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms
              </a>
            </td>
          <td>
            Feifei Xia, M. Verbiest, Oxana Lundström, Tugce Bilgin Sonay, M. Baudis, Maria Anisimova
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f345edfe89fa16dab1e87b151db2db75ec0ac8dc" target='_blank'>
              Insulin signaling regulates R2 retrotransposon expression to orchestrate transgenerational rDNA copy number maintenance
              </a>
            </td>
          <td>
            Jonathan O. Nelson, Alyssa Slicko, Amelie A Raz, Yukiko M. Yamashita
          </td>
          <td>2025-01-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background UCK2 (Uridine-Cytidine Kinase 2) is a promising prognostic marker for malignant tumors, but its association with immune infiltration and cancer stemness in pan-cancer remains to be fully understood. we find that gene UCK2 is closed related to RNA stemness scores (RNAss) and DNA stemness scores (DNAss), which is measured the tumor stemness. We also discover an association between UCK2 expression and immune cells by CIBERSORT algorithm, ESTIMATE algorithm and ssGSEA algorithm, especially, related to T cell, monocytes, mast cells, and macrophages. This study aims to shed light on the role and possible mechanism of UCK2 in pan-cancer. Methods We used the R programming language for pan-cancer bulk sequencing data analysis, which were obtained from the University of California, Santa Cruz (UCSC) datasets. UCSC database is a very useful for explore data from TCGA and other cancer genomics datasets, The data we explored at the UCK2 transcriptome level came from TCGA data in the UCSC database. We explored differential UCK2 expression between tumor and normal samples. Immunohistochemistry (IHC) was utilized to validate the expression of UCK2 in different types cancers using tumor tissue chips. The correlations of UCK2 with prognosis, genetic instability, DNA repair, cancer stem cell characteristics, and immune cell infiltration were investigated. Furthermore, single-cell datasets, acquired from the Gene Expression Omnibus (GEO) database, were used to validate the relationship between UCK2 and immune cells. GEO is a famous public genomics database supporting freely disseminates microarray data. Finally, we analyzed the correlation between UCK2 and drug sensitivity. Results UCK2 expression was observed to be high in most cancers and was remarkably related to the prognosis of pan-cancers. We found that the increased UCK2 expression was associated with higher genetic instability. Additionally, positive relationships were observed between UCK2 expression and mismatch repair genes, homologous recombination repair genes, and cancer stemness across different cancer types. There were significant correlations between UCK2 and T cells, monocytes, mast cells, and macrophages. Moreover, as expected, the immune checkpoint human leucocyte antigen (HLA) was found to be negatively related to UCK2. Similarly, UCK2 was also observed to have a negative association with major histocompatibility complex (MHC) genes. We noted that UCK2 had significant correlations with the sensitivity to various anti-cancer drug. Conclusion We have observed that UCK2 plays pivotal roles in prognosis and tumor immunity, and it is associated with DNA repair and cancer stemness. The UCK2 gene exhibits a strong correlation with the immune checkpoints HLA. This study highlights its potential impact on drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3d615de6acd1c023835e11b2fb04b8e2aecd39" target='_blank'>
              Uridine-cytidine kinase 2 is correlated with immune, DNA damage repair and promotion of cancer stemness in pan-cancer
              </a>
            </td>
          <td>
            Jinlong Tian, Yanlei Li, Yu Tong, Yuan Zhang, Tingxiao Zhao, Yao Kang, Qing Bi
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, , C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-15</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d06e293f47afb8c1e77d741db124642957b624" target='_blank'>
              Characterization of RNF144B and PPP2R2A identified by a novel approach using TCGA data in ovarian cancer
              </a>
            </td>
          <td>
            P. Manasa, S. Krishnapriya, C. Sidhanth, S. Vasudevan, Kanchan Murhekar, T. S. Ganesan
          </td>
          <td>2025-02-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The human fungal pathobiont Candida albicans displays extensive genomic plasticity, including large-scale chromosomal changes such as aneuploidy. Chromosome trisomy appears frequently in natural and laboratory strains of C. albicans. Trisomy of specific chromosomes has been linked to large phenotypic effects, such as increased murine gut colonization by strains trisomic for chromosome 7 (Chr7). However, studies of whole-chromosome aneuploidy are generally limited to the SC5314 genome reference strain, making it unclear whether the imparted phenotypes are conserved across C. albicans genetic backgrounds. Here, we report the presence of a Chr7 trisomy in the “commensal-like” oral candidiasis strain, 529L, and dissect the contribution of Chr7 trisomy to colonization and virulence in 529L and SC5314. These experiments show that strain background and homolog identity (i.e., AAB vs ABB) interact with Chr7 trisomy to alter commensal and virulence phenotypes in multiple host niches. In vitro filamentation was the only phenotype altered by Chr7 trisomy in similar ways across the two strain backgrounds. Oral colonization of mice was increased by the presence of a Chr7 trisomy in 529L but not SC5314; conversely, virulence during systemic infection was reduced by Chr7 trisomy in SC5314 but not 529L. Strikingly, the AAB Chr7 trisomy in the SC5314 background rendered this strain avirulent in murine systemic infection. Increased dosage of NRG1 failed to reproduce most of the Chr7 trisomy phenotypes. Our results demonstrate that aneuploidy interacts with background genetic variation to produce complex phenotypic patterns that deviate from our current understanding in the genome reference strain. AUTHOR SUMMARY Candida albicans is a clinically important fungus of humans that is also part of the microbiota that typically colonizes our bodies. Changes in the number of copies of chromosome 7 (Chr7) in the genome of the reference strain of C. albicans enhances host colonization. We identified a third copy of Chr7 in a strain classified as a good colonizer of humans that doesn’t cause disease, mimicking the increased colonization of the reference strain. Here, we demonstrate that the number of copies of Chr7 alters phenotypes differently between these strains, and three copies of Chr7 does not universally produce a strain that is only a better commensal strain. Additionally, which copy of Chr7 in this diploid species exists as the third copy influences commensal and pathogen phenotypes. Therefore, trisomic changes of the same chromosome in different C. albicans strains have substantially different effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2332e9447e18d3247be54ad40b31742ed672d8fa" target='_blank'>
              Strain background interacts with chromosome 7 aneuploidy to determine commensal and virulence phenotypes in Candida albicans
              </a>
            </td>
          <td>
            Abhishek Mishra, N. Solis, Siobhan M Dietz, Audra L Crouch, S. Filler, Matthew Z. Anderson
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f817e6c27d9a43bff904bfe868d13c1b02f3616e" target='_blank'>
              Varieties of altered TFE3 can occur in MiT-family-related renal cell carcinomas.
              </a>
            </td>
          <td>
            Allison K Kennedy, Hassan D. Kanaan, Kanika Arora, Harry Zhang, Jason M Hafron, Shelly L Kaufman, Mark A Micale, Ping L Zhang
          </td>
          <td>2025-02-18</td>
          <td>International urology and nephrology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The choice between somatic and germline fates is essential for species survival. This choice occurs in embryonic epiblast cells, as these cells are competent for both somatic and germline differentiation. The transcription factor OTX2 regulates the choice between somatic and germline fates, as Otx2-null epiblast-like cells (EpiLCs) form primordial germ cell-like cells (PGCLCs) with enhanced efficiency. However, the mechanisms by which OTX2 achieves this function are not fully characterized. Here we show that OTX2 controls chromatin accessibility at specific chromatin loci to modulate gene transcription and enable somatic differentiation. By performing CUT&RUN for OTX2 and ATAC-seq in wild-type and Otx2-null embryonic stem cells and EpiLCs, we identified regions where OTX2 binding opens chromatin, either alone or in cooperation with a ZIC family protein. Enforced OTX2 expression maintains accessibility at these regions and, in addition, induces opening of ∼4,000 somatic-associated regions in cells differentiating in the presence of PGC-inducing cytokines. Once cells have acquired germline identity, these additional somatic-associated regions can no longer respond to OTX2 and remain closed. Our results indicate that OTX2 works in cells with dual competence for both somatic and germline differentiation to increase accessibility of somatic regulatory regions and induce the somatic fate at the expense of the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df9739be940a44f0aa93afedff668896ab7f1ef" target='_blank'>
              OTX2 controls chromatin accessibility to direct somatic versus germline differentiation
              </a>
            </td>
          <td>
            Elisa Barbieri, Ian Chambers
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1494cc598957b612d5a237787552f3ac7309c3c0" target='_blank'>
              Dynamic interplay of cNHEJ and MMEJ pathways of DNA double-strand break repair during embryonic development in zebrafish
              </a>
            </td>
          <td>
            Mathieu Carrara, Anne-Laure Gaillard, Alice Brion, Evelyne Duvernois-Berthet, Carine Giovannangeli, J. Concordet, Guillaume Pézeron
          </td>
          <td>2025-02-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="X Chromosome Inactivation (XCI) equalizes X-linked gene expression between sexes. B cells exhibit dynamic XCI, with Xist RNA/heterochromatic marks absent on the inactive X (Xi) in naive B cells but returning following mitogenic stimulation. The impact of dynamic XCI on Xi structure and maintenance was previously unknown. Here, we find dosage compensation of the Xi with state-specific XCI escape genes in naive and in vitro activated B cells. Allele-specific OligoPaints indicate similar Xi and Xa territories in B cells that are less compact than in fibroblasts. Allele-specific Hi-C reveals a lack of TAD-like structures on the Xi of naive B cells, and stimulation-induced alterations in TAD-like boundary strength independent of gene expression. Notably, Xist deletion in B cells changes TAD boundaries and large-scale Xi compaction. Altogether, our results uncover B cell-specific Xi plasticity which could underlie sex-biased biological mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf439585ea0599081bd08309bf4216f355afcfa4" target='_blank'>
              B cell stimulation changes the structure and higher-order organization of the inactive X chromosome
              </a>
            </td>
          <td>
            Isabel Sierra, Natalie E. Toothacre, Robin H. van der Weide, Claudia D. Lovell, Son C. Nguyen, R. Jordan Barnett, Ashley L. Cook, Han-Seul Ryu, Sarah Pyfrom, Harrison Wang, D. Beiting, Jennifer E. Philips-Cremins, Eric F. Joyce, Montserrat C. Anguera
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, and mutation hotspots in human cancer genomes co-localize with alternative DNA structure-forming sequences (e.g. H-DNA), implicating them in cancer etiology. H-DNA has been shown to stimulate genetic instability in mammals. Here, we demonstrate a new paradigm of genetic instability, where a cancer-associated H-DNA-forming sequence accumulates more oxidative lesions than B-DNA under conditions of oxidative stress (OS), often found in tumor microenvironments. We show that OS results in destabilization of the H-DNA structure and attenuates the fold increase in H-DNA-induced mutations over control B-DNA in mammalian cells. Furthermore, the mutation spectra revealed that the damaged H-DNA-containing region was processed differently compared to H-DNA in the absence of oxidative damage in mammalian cells. The oxidatively modified H-DNA elicits differential recruitment of DNA repair proteins from both the base excision repair and nucleotide excision repair mechanisms. Altogether, these results suggest a new model of genetic instability whereby H-DNA-forming regions are hotspots for DNA damage in oxidative microenvironments, resulting in its altered mutagenic processing. Our findings provide valuable insights into the role of OS in DNA structure-induced genetic instability and may establish H-DNA-forming sequences as promising genomic biomarkers and potential therapeutic targets for genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560e3fa66aab00812738293c545080a843274a94" target='_blank'>
              Oxidative damage within alternative DNA structures results in aberrant mutagenic processing
              </a>
            </td>
          <td>
            Maha Zewail-Foote, Imee M A Del Mundo, Alex W. Klattenhoff, Karen M. Vasquez
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="H3K4me3 is a fundamental and highly conserved chromatin mark across eukaryotes, playing a central role in many genome-related processes, including transcription, maintenance of cell identity, DNA damage repair, and meiotic recombination. However, identifying the causal function of H3K4me3 in these diverse pathways remains a challenge, and we lack the tools to manipulate it for agricultural benefit. Here we use the CRISPR-based SunTag system to direct H3K4me3 methyltransferases in the model plant, Arabidopsis thaliana. Targeting of SunTag-SDG2 activates the expression of the endogenous reporter gene, FWA. We show that SunTag-SDG2 can be employed to increase pathogen resistance by targeting the H3K4me3-dependent disease resistance gene, SNC1. Meiotic crossover recombination rates impose a limit on the speed with which new traits can be transferred to elite crop varieties. We demonstrate that targeting of SunTag-SDG2 to low recombining centromeric regions can significantly stimulate crossover formation. Finally, we reveal that the effect is not specific to SDG2 and is likely dependent on the H3K4me3 mark itself, as the orthogonal mammalian-derived H3K4me3 methyltransferase, PRDM9, produces a similar effect on gene expression with reduced off-target potential. Overall, our study supports an instructive role for H3K4me3 in transcription and meiotic recombination and opens the door to precise modulation of important agricultural traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c9ac7e7e4f2e661d415bc49c2d94c638f0dbf9" target='_blank'>
              CRISPR targeting of H3K4me3 activates gene expression and unlocks centromeric crossover recombination in Arabidopsis
              </a>
            </td>
          <td>
            Jenia Binenbaum, V. Adamkova, Hannah Fryer, Linhao Xu, Nicola Gorringe, Piotr Włodzimierz, Robin Burns, Ashot Papikian, Steve Jacobsen, Ian R Henderson, C. Harris
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BORIS (brother of the regulator of imprinted sites), the paralog of the genome organizer CTCF, originated at least 318 million years ago (Mya), in the ancestor of amniotes (mammals, reptiles, and birds). Based on results from chicken (Gallus gallus), the gene was thought to be absent from birds. Using comparative genomics of 59 bird species, we show that birds possess BORIS, but frequently experience severe degradation of the gene, as observed in Gallus gallus. The degradation events are restricted to neognathous birds, specific for the BORIS coding sequence, and occur multiple times independently on different branches. They comprise a wide range of molecular decay, from individual point mutations to the inactivation and/or loss of particular zinc fingers, to the almost complete disintegration of the gene. The decay is accompanied by relaxed evolutionary constraints on BORIS codons across neognathous birds and coincides with the accumulation of species-specific repetitive elements in degenerate loci. BORIS represents a case of a presently ongoing, convergent, and specific gene loss within a lineage. As possible explanation, we propose a link between the loss of BORIS and a shift in sperm and/or genital morphology during the evolution of Neognathae. Significance Statement The gene BORIS was believed to be absent from birds. However, genome analysis of 59 bird species reveals its presence, though it often underwent severe degradation in neognathous birds. These independent degradation events affect the BORIS coding sequence and range from point mutations to near complete gene disintegration. This decay correlates with relaxed evolutionary constraints and species-specific accumulation of repetitive elements. A potential link between BORIS loss and changes in sperm or genital morphology during neognathous bird evolution is suggested.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5ac27681f042f5b8962df02e8f366f155fc2b2c" target='_blank'>
              Decay of the CTCF paralog BORIS in neognathous birds
              </a>
            </td>
          <td>
            Katja Palitzsch, T. Wiehe, P. Heger
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="At the moment, many non-coding RNAs that perform a variety of functions in the regulation of chromatin processes are known. An increasing number of protocols allow researchers to study RNA-DNA interactions and shed light on new aspects of the RNA–chromatin interactome. The Hi-C protocol, which enables the study of chromatin’s three-dimensional organization, has already led to numerous discoveries in the field of genome 3D organization. We conducted a comprehensive joint analysis of the RNA-DNA interactome and chromatin structure across different human and mouse cell lines. We show that these two phenomena are closely related in many respects, with the nature of this relationship being both tissue specific and conserved across humans and mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a06857f95843b2d45dd7445d355aa4f08736932d" target='_blank'>
              A Joint Analysis of RNA-DNA and DNA-DNA Interactomes Reveals Their Strong Association
              </a>
            </td>
          <td>
            Dmitry S. Zvezdin, Artyom A. Tyukaev, A. Zharikova, A. Mironov
          </td>
          <td>2025-01-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff1e1e85866d4cc9a2e1386acb32f48ee6d86bb6" target='_blank'>
              AZIN1 level is increased in medulloblastoma and correlates with c-Myc activity and tumor phenotype
              </a>
            </td>
          <td>
            Julie Sesen, Tyra Martinez, Sara Busatto, L. Poluben, Hassan Nassour, Caroline Stone, Karthik Ashok, Marsha A. Moses, Edward R. Smith, Aram Ghalali
          </td>
          <td>2025-02-17</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Protein intrinsically disordered regions (IDRs) are critical gene-regulatory components and aberrant fusions between IDRs and DNA-binding/chromatin-associating domains cause diverse human cancers. Despite this importance, how IDRs influence gene expression, and how aberrant IDR fusion proteins provoke oncogenesis, remains incompletely understood. Here we develop a series of synthetic dCas9-IDR fusions to establish that locus-specific recruitment of IDRs can be sufficient to stimulate endogenous gene expression. Using dCas9 fused to the paradigmatic leukemogenic NUP98 IDR, we also demonstrate that IDRs can activate transcription via localized biomolecular condensation and in a manner that is dependent upon overall IDR concentration, local binding density, and amino acid composition. To better clarify the oncogenic role of IDRs, we construct clinically observed NUP98 IDR fusions and show that, while generally non-overlapping, oncogenic NUP98-IDR fusions convergently drive a core leukemogenic gene expression program in donor-derived human hematopoietic stem cells. Interestingly, we find that this leukemic program arises through differing mechanistic routes based upon IDR fusion partner; either distributed intragenic binding and intrachromosomal looping, or dense binding at promoters. Altogether, our studies clarify the gene-regulatory roles of IDRs and, for the NUP98 IDR, connect this capacity to pathological cellular programs, creating potential opportunities for generalized and mechanistically tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de0eb272216630f4634f73b7ac6d3e80cb86fd8" target='_blank'>
              Biomolecular condensation of human IDRs initiates endogenous transcription via intrachromosomal looping or high-density promoter localization
              </a>
            </td>
          <td>
            Jing Li, Shizhe Liu, Sunghwan Kim, Jacob Goell, Zachary Allen Drum, John Patrick Flores, Alex J Ma, Barun Mahata, Mario Escobar, Alex Raterink, Jeong Hyun Ahn, Erik R Terán, R. S. Guerra-Resendez, Yuhao Zhou, Bo Yu, Michael R. Diehl, Gang Greg Wang, Anna-Karin Gustavsson, D. Phanstiel, Isaac B. Hilton
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Transposable elements (TEs), once considered as "junk" DNA, are now recognized for their epigenetic and regulatory influence, especially in cancer. Cryptic regulatory elements within TEs can be epigenetically reactivated in cancer, contributing to oncogenesis and altering gene expression patterns. This reactivation results in TE-derived transcripts, which have the potential to be translated and form neoantigens that can be presented by the human leukocyte antigens (HLA) complex and recognized by T cells, ultimately impacting the tumor immune response. In this study, we analyzed RNA-seq data from melanoma patients undergoing PD1/PDL1 immunotherapy. By identifying tumor specific TE-derived transcripts and utilizing in silico translation, we detected TE-derived neoantigens in patients with differing responses to immunotherapy. Our findings indicate that the number of TE-derived neoantigens is correlated with patients' response to immunotherapy, with a higher number of neoantigens observed in patients who responded to treatment compared to those who did not. These results imply that TE-derived neoantigens may serve as potential biomarkers for predicting the prognosis of PD1/PDL1 immunotherapy treatment. Incorporating TE-derived neoantigen profiles into clinical decision-making could enhance patient selection for immunotherapy, thereby improving treatment outcomes.
 Citation Format: Yixin Wu, Ting Wang, Yonghao Liang. Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) patients with microsatellite-stable (MSS) tumors are mostly treated with chemotherapy. Clinical benefits of targeted therapies depend on mutational states and tumor location. Many tumors carry mutations in KRAS proto-oncogene, GTPase (KRAS) or B-Raf proto-oncogene, serine/threonine kinase (BRAF), rendering them more resistant to therapies. We performed whole-exome sequencing and RNA-Sequencing of 28 tumors of the Athens Comprehensive Cancer Center CRC cohort, and molecularly characterized CRC patients based on their microsatellite instability (MSI) status, single-nucleotide variations (SNVs)/copy number alterations (CNAs), and pathway/transcription factor activities at the individual patient level. Variants were classified using a computational score for integrative cancer variant annotation and prioritization. Complementing this with public multi-omics datasets, we identified activation of transforming growth factor beta (TGFβ) signaling to be more strongly activated in MSS patients, whereas Janus kinase (JAK)-signal transducer and activator of transcription (STAT) and mitogen-activated protein kinase (MAPK) molecular cascades were activated specifically in MSI tumors. We unraveled mechanisms consistently perturbed in the transcriptional and mutational circuits and identified Runt-related transcription factors (RUNX transcription factors) as putative biomarkers in CRC, given their role in the regulation of pathways involved in tumor progression and immune evasion. Assessing the immunogenicity of CRC tumors in the context of RAS/RAF mutations and MSI/MSS status revealed a critical impact that KRAS mutations have on immunogenicity, particularly in the MSS patient subgroup, with implications for diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74b1bcf5fa54757058aca72542a0424e2f6a831" target='_blank'>
              Molecular and functional profiling unravels targetable vulnerabilities in colorectal cancer.
              </a>
            </td>
          <td>
            E. Vlachavas, Konstantinos Voutetakis, Vivian Kosmidou, Spyridon Tsikalakis, Spyridon Roditis, Konstantinos Pateas, Ryangguk Kim, Kymberleigh Pagel, Stephan Wolf, Gregor Warsow, Antonia Dimitrakopoulou-Strauss, Georgios N Zografos, Alexander Pintzas, J. Betge, O. Papadodima, Stefan Wiemann
          </td>
          <td>2025-01-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053c90f2e024c025545d0a836542f7f28e9a61c7" target='_blank'>
              KLF5 loss sensitizes cells to ATR inhibition and is synthetic lethal with ARID1A deficiency
              </a>
            </td>
          <td>
            Samah W. Awwad, Colm Doyle, Josie Coulthard, Aldo S. Bader, Nadia Gueorguieva, Simon Lam, Vipul Gupta, R. Belotserkovskaya, Tuan-Anh Tran, Shankar Balasubramanian, S. P. Jackson
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c7a49a64fc95173d062b597a609f1c34ffa6ed" target='_blank'>
              PIF/harbinger transposon-derived protein promotes 7SL expression to enhance pathogen resistance.
              </a>
            </td>
          <td>
            S. Geng, Xing Lv, Tianjun Xu
          </td>
          <td>2025-01-30</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d43986e3ec1bf10fa7949b61cdcc071a4afe56" target='_blank'>
              Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, Masashi Kariya, Ayami Osaka, S. Ishino, Masataka Yamauchi, Tomonori Tamura, I. Hamachi, Shoji Takada, Y. Ishino, H. Atomi
          </td>
          <td>2025-02-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Since the discovery of the Musashi (MSI) protein, its ability to affect the mitosis of Drosophila progenitor cells has garnered significant interest among scientists. In the following 20 years, it has lived up to expectations. A substantial body of evidence has demonstrated that it is closely related to the development, metastasis, migration, and drug resistance of malignant tumors. In recent years, research on the MSI protein has advanced, and many novel viewpoints and drug resistance attempts have been derived; for example, tumor protein p53 mutations and MSI-binding proteins lead to resistance to protein arginine N-methyltransferase 5-targeted therapy in lymphoma patients. Moreover, the high expression of MSI2 in pancreatic cancer might suppress its development and progression. As a significant member of the MSI family, MSI2 is closely associated with multiple malignant tumors, including hematological disorders, common abdominal tumors, and other tumor types (e.g., glioblastoma, breast cancer). MSI2 is highly expressed in the majority of tumors and is related to a poor disease prognosis. However, its specific expression levels and regulatory mechanisms may differ based on the tumor type. This review summarizes the research progress related to MSI2 in recent years, including its occurrence, migration mechanism, and drug resistance, as well as the prospect of developing tumor immunosuppressants and biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dd0dc9ec8f8cc8ab893bc32f47ba299917d963" target='_blank'>
              Update on the Progress of Musashi-2 in Malignant Tumors.
              </a>
            </td>
          <td>
            Yiting Niu, Tao Zhou, Yanjun Li
          </td>
          <td>2025-01-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Repetitive elements, mostly derived from transposable elements (TEs), account for half the DNA in human and other mammalian genomes. Although epigenetic mechanisms, including DNA methylation and repressive histone modifications, have evolved to suppress TE activities, TEs have substantially shaped the regulatory landscape of the host genome by contributing regulatory sequences to it. TE-derived sequences are often highly repetitive and thus have low mappability, making it difficult to profile the genomics of TEs using short-read sequencing technology. Many specialized bioinformatics tools have been developed for TE-related analysis, but meaningfully visualizing, navigating, and interpreting such data remains challenging. Here, we describe the WashU Repeat Browser to host genomics profiles of human and mouse TEs using data produced by the ENCODE Project and to support the navigation, interactive visualization, integration, comparison, and analysis in the context of TEs. WashU Repeat Browser is a web-based platform allowing users to browse genomic and statistical signals over repetitive elements derived from ENCODE, Roadmap, and FANTOM datasets. The Browser provides a TE-centric view including TE subfamily enrichments, TE subfamily profiling, as well as overviews of genomic signals on individual TE loci where we extend the WashU Epigenome Browser to display user-selected datasets and TE loci. These features could help to close the gaps in our understanding of the repetitive sequences and their putative regulatory functions and aid investigators in formulating new hypotheses by integrating their data with public data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217e299b91fe8b545cc86ee95ab6ebdd4d9d3fb8" target='_blank'>
              Exploring the epigenome profiles of repetitive elements with the WashU Repeat Browser.
              </a>
            </td>
          <td>
            Jiawei Shen, Siyuan Cheng, Deepak Purushotham, Xiaoyu Zhuo, Alan Y. Du, Wenjin Zhang, Daofeng Li, Ting Wang
          </td>
          <td>2025-02-21</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The function of GCNT2 has been documented to act as an oncogenic driver or tumor suppressor in different types of tumor, but the role of GCNT2 and the epigenetic regulation mechanism in AML, however, has not yet been clarified. This study aimed to assay the expression and methylation profile of GCNT2 in AML, and further elucidate the clinical significance.Multiple datasets from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas projects (TCGA) were used to explore the expression and methylation profile of GCNT2 in normal hematopoiesis and AML. A pan-cancer analysis was performed to define the survival implications of GCNT2 across multiple cancers including AML. The relationships between GCNT2 expression/methylation and clinicopathologic features were investigated using a TCGA-AML dataset. Correlation analysis was performed to explore the relationship between transcriptional expression and DNA methylation. Differentially expressed genes (DEGs) on the KEGG pathway and GO terms were visualized using DAVID. Gene Set Enrichment Analysis (GESA) was carried out to assess the underlying mechanism. The relationship between methylation and immune cell infiltration was also examined.GCNT2 expression was highest in hematopoietic stem cells (HSC) but gradually decreased during the hematopoiesis differentiation, the monocytes, however, remained a high level of GCNT2 as an exception. In AML, GCNT2 was down-regulated as compared to normal hematopoiesis but was much higher in contrast to normal peripheral blood samples. Data from a pan-cancer analysis revealed that high-expressed GCNT2 contributed to a worse OS for AML. DNA methylation of GCNT2 showed a distinctive co-methylation pattern in AML and significantly negatively correlated with transcriptional expression. Methylation in the transcriptional start site of isoform A plays a critical role in the epigenetic regulation of GCNT2 expression. The silence of GCNT2 in AML was attributed to DNA methylation. Hypomethylation of isoform A significantly predicted poor survival in AML, linking to several cytogenetic and molecular abnormalities, such as t (8:21), inv (16), t (15;17), and genes mutations of DNMT3A, CEBPA, RUNX1, and WT1. Enrichment analysis disclosed that hypomethylation of isoform A was involved in the immune system, and it was further revealed that hypomethylation of isoform A was tightly associated with immune cell infiltration and could be served as a promising indicator for immunotherapy.Our comprehensive research demonstrated that GCNT2 acted as an oncogene in AML, and was epigenetically regulated by DNA methylation in isoform A. Hypomethylation of isoform A could be served as a promising indicator to identify the high-risk AML patients who might be responsive to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57f0f5a218991075d83e79f90c940a4728b2de34" target='_blank'>
              Hypomethylation of GCNT2 isoform A correlates with transcriptional expression and is associated with poor survival in acute myeloid leukemia
              </a>
            </td>
          <td>
            Delong Wu, , Jing Xu, Jiang Lin, Jun Qian
          </td>
          <td>2025-02-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e667edb95ec839a056e02fa834f44d25b6553f6" target='_blank'>
              Cohesin positions the epigenetic reader Phf2 within the genome
              </a>
            </td>
          <td>
            W. Tang, Lorenzo Costantino, Roman R. Stocsits, G. Wutz, Rene Ladurner, Otto Hudecz, K. Mechtler, Jan-Michael Peters
          </td>
          <td>2025-01-02</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Among the many features of cancer cells, they commonly display aberrant accumulation of RNA-DNA hybrids known as R-loops. R-loops are formed in healthy cells during many cellular processes, including transcription and DNA replication, and are crucial for proper maintenance of telomeres and centromeres, as well as for adequate mitochondrial function. However, aberrant accumulation of R-loops can lead to increases in DNA damage and inflammation. R-loop accumulation has been documented in different types of cancer, including BRCA1-mutant breast cancer, diffuse large B cell lymphoma, and myelodysplastic syndrome (MDS). Indeed, accumulation of R-loops has been proposed as a unifying mechanism underlying MDS and other myeloid malignancies. Ten-eleven translocation enzymes (TET1, TET2, TET3) are 5-methylcytosine dioxygenases that undergo frequent mutational or functional inactivation in human cancers. We have shown that mice with profound, acute TET deficiency induced by deletion of all Tet genes (hereafter referred to as Tet iTKO mice) show rapid onset of an uncontrolled malignant myeloid expansion. Expanded myeloid cells from Tet iTKO mice show increased levels of R-loops, together with increased DNA damage and inflammation, compared to control myeloid cells. Using Tet iTKO hematopoietic stem and progenitor cells (HSPC), we performed an in vivo CRISPR knockout screen of genes encoding RNA and DNA sensors, as well as R-loop regulators, to find how loss of genes related to nucleic acid sensing and innate immune response would affect the proliferation of TET-deficient myeloid cells. We observed that loss of enzymes that prevent the accumulation of R-loops impaired the uncontrolled malignant myeloid expansion of Tet iTKO cells. CRISPR-mediated depletion of the gene encoding RNase H1, an enzyme that specifically degrades the RNA strand in R-loops, further increased R-loop levels and DNA damage, and decreased proliferation and in vivo expansion of TET-deficient myeloid cancer cells, compared to Tet iTKO cells treated with non-targeting sgRNAs. Moreover, the accumulation of DNA breaks in Tet iTKO cells upon Rnaseh1 loss was not observed in Tet wild-type cells similarly depleted for RNase H1, suggesting a role for TET enzymes and potentially TET-oxidized 5-methylcytosines in DNA repair, which will be discussed in this presentation. Overall, these findings indicate that further increasing R-loops in TET-deficient cancer cells blocks their proliferation, and suggest that RNase H depletion or inhibition could be a new therapeutic strategy for cancer treatment in cancers that display augmented levels of R-loops, independent of the presence of TET mutations.
 Citation Format: Isaac F López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, Arthur X Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao. Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The Chromosome-Centric Human Proteome Project (C-HPP) is an international initiative. It aims to create a protein list expressed in human cells by each chromosomal and mitochondrial DNA to enhance our understanding of disease mechanisms, akin to the gene list generated by the Human Genome Project. Transmembrane protein 160 (TMEM160) is a member of the transmembrane proteins (TMEM) family. TMEM proteins have been implicated in cancer-related processes, including cell proliferation, migration, epithelial-mesenchymal transition, metastasis, and resistance to chemotherapy and radiotherapy. This study aimed to investigate the role of TMEM160 in non-small cell lung cancer and cervical cancer using cell lines, clinical samples, and xenograft studies. Our findings demonstrated that TMEM160 knockdown decreased the proliferation of lung and cervical cancer cell lines. We observed that TMEM160 is localized in the nucleus and cytoplasm and dynamic localization during mitosis of cancer cells and discovered a novel interaction between TMEM160 and nuclear proteins such as NUP50. Furthermore, the TMEM160 interactome was enriched in processes associated with apical junctions, xenobiotic metabolism, glycolysis, epithelial-mesenchymal transition, reactive oxygen species, UV response DNA, the P53 pathway, and the mitotic spindle. This study provides an initial understanding of the function of TMEM160 in lung and cervical cancer progression and clarifies the need to continue investigating the participation of TMEM160 in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e13c463e1e4f1ddbb09b31ff1b7171d0bdfe61c" target='_blank'>
              TMEM160 Promotes Tumor Growth in Lung Adenocarcinoma and Cervical Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Gloria Angelina Herrera-Quiterio, H. A. Valencia-González, K. G. de la Cruz-López, D. L. Fernández-Coto, Jeovanis Gil, Gyorgy Marko-Varga, Josué Morales-Gálvez, Nilda C. Sánchez, Rubén Rodríguez-Bautista, A. Avilés-Salas, Oscar Arrieta, Alejandro García-Carrancá, S. Encarnación-Guevara
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 The molecular mechanisms underlying estrogen receptor (ER)-positive breast carcinogenesis and drug resistance remain incompletely understood. Elevated expression of CCND1 is linked to enhanced invasiveness, poorer prognosis, and resistance to drug therapies in ER-positive breast cancer. In this study, we identify a highly expressed circular RNA (circRNA) derived from
 FOXK2
 , called circFOXK2, which plays a key role in stabilizing
 CCND1
 mRNA, thereby promoting cell cycle progression, cell growth, and endocrine therapy resistance in ER-positive breast cancer cells. Mechanistically, circFOXK2 binds directly to
 CCND1
 mRNA via RNA–RNA pairing and recruits the RNA-binding protein ELAVL1/HuR, stabilizing the
 CCND1
 mRNA and enhancing CCND1 protein levels. This results in activation of the CCND1–CDK4/6–p-RB–E2F signaling axis, driving the transcription of downstream E2F target genes and facilitating the G1/S transition during cell cycle progression. Notably, targeting circFOXK2 with antisense oligonucleotide (ASO-circFOXK2) suppresses ER-positive breast cancer cell growth both in vitro and in vivo. Moreover, combination therapy with ASO-circFOXK2 and tamoxifen exhibits synergistic effects and restores tamoxifen sensitivity in tamoxifen-resistant cells. Clinically, high circFOXK2 expression is positively correlated with
 CCND1
 levels in both ER-positive breast cancer cell lines and patient tumor tissues. Overall, our findings reveal the critical role of circFOXK2 in stabilizing the oncogene
 CCND1
 and promoting cancer progression, positioning circFOXK2 as a potential therapeutic target for ER-positive breast cancer in clinical settings.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a572f11006892aa8b4e142fead4939f8a023665b" target='_blank'>
              A circRNA–mRNA pairing mechanism regulates tumor growth and endocrine therapy resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Jia Yi, Jiao Du, Xue Chen, Rui-chao Nie, Guo-Sheng Hu, Lei Wang, Yue-ying Zhang, Shang Chen, , , Hua He, Wen Liu
          </td>
          <td>2025-02-18</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Resistance to radiotherapy remains a critical barrier in treating colorectal cancer (CRC), particularly in cases of locally advanced rectal cancer (LARC). To identify key kinases involved in CRC radioresistance, we employed a kinase‐targeted CRISPR‐Cas9 library screen. This approach aimed to identify potential kinase inhibitors as radiosensitizers. Our screening identified salt‐inducible kinase 2 (SIK2) as a critical factor in CRC radioresistance. Increased SIK2 expression correlated with reduced tumor regression and poorer outcomes in LARC patients undergoing neoadjuvant chemoradiotherapy. The depletion of SIK2 significantly enhanced radiation‐induced apoptosis and tumor regression. Mechanistically, SIK2 interacts with valosin‐containing protein (VCP), promoting its hyperphosphorylation. This modification improves VCP's capacity to extract K48‐linked ubiquitin‐conjugated proteins from chromatin, thus aiding the recruitment of RPA and RAD51 to DNA damage sites. This mechanism strengthens homologous recombination–mediated DNA repair, which contributes to radioresistance. Importantly, ARN‐3236, a SIK2 inhibitor, markedly sensitized CRC cells to radiation both in vivo and in vitro, providing a potential strategy to overcome radioresistance. In summary, our findings reveal a novel mechanism by which SIK2 contributes to the radioresistance of CRC, proposing SIK2 as a potential therapeutic target with its inhibitor significantly enhancing CRC radiotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27d0bffcf12a04202c03cb4fe50e95cbe2133af6" target='_blank'>
              Salt‐inducible kinase 2 confers radioresistance in colorectal cancer by facilitating homologous recombination repair
              </a>
            </td>
          <td>
            Yuan Meng, Shuo Li, Da‐Shan Lu, Xue Chen, Lu Li, , Gao‐yuan Wang, Wenlin Huang, Ran-Yi Liu
          </td>
          <td>2025-01-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc8fc819ffccd10115ccb040d51408d1fd6a5f7" target='_blank'>
              Nanopore sequencing reveals that DNA replication compartmentalisation dictates genome stability and instability in Trypanosoma brucei
              </a>
            </td>
          <td>
            M. Krasiļņikova, Catarina A Marques, E. Briggs, C. Lapsley, Graham Hamilton, Dario Beraldi, K. Crouch, R. McCulloch
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations in the NPM1 gene represent the most common (>30% of patients) genetic alteration in Acute Myeloid Leukaemia (AML) and results in the mis-localisation of the mutated NPM1 protein from a predominantly nucleolar localisation to a predominantly cytoplasmic distribution. Numerous studies of NPM1 mutated AML have focussed on the aberrant cytoplasmic localisation of the mutated protein but efforts to reverse this mis-localisation therapeutically have so far resulted in limited clinical benefit. More recently, attention has shifted towards the nucleus with studies showing that mutant NPM1 binds to specific chromatin regions, where it directly regulates oncogenic gene expression. Here, we use high resolution imaging to demonstrate that Nucleophosmin (NPM1) is critical for maintaining normal nucleoli architecture and specifically the integrity of the nucleoli rim. We report for the first time that NPM1 mutated cell lines and primary samples have aberrant nucleoli architecture and demonstrate that the abnormal nucleoli phenotype is reversible. We also report the novel finding that NPM1 mutated protein forms distinct aggregates in NPM1 mutated cells and characterise these for the first time. This work reveals how nucleolar organisation contributes to the molecular mechanisms underpinning NPM1 driven AML and reveals unexpected novel vulnerabilities to be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b14a9c2136580651e7ba63ac9928b96ae556480" target='_blank'>
              Nucleophosmin mutations lead to abnormal, but reversible, nucleoli architecture and aggregate formation – Implications for NPM1-targeting therapies in AML
              </a>
            </td>
          <td>
            Martin Grundy, Kellie Lucken, Xiaomeng Xing, Eva L Simpson, Ahmed Bayyoomi, Alison J. Beckett, Ian A. Prior, Daniel G. Booth, C. Seedhouse
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mitochondrial diseases, caused by mutations in either nuclear or mitochondrial DNA (mtDNA), currently have limited treatment options. For mtDNA mutations, reducing mutant-to-wild-type mtDNA ratio (heteroplasmy shift) is a promising therapeutic option, though current approaches face significant challenges. Previous research has shown that severe mitochondrial dysfunction triggers an adaptive nuclear epigenetic response, characterized by changes in DNA methylation, which does not occur or is less important when mitochondrial impairment is subtle. Building on this, we hypothesized that targeting nuclear DNA methylation could selectively compromise cells with high levels of mutant mtDNA, favor ones with lower mutant load and thereby reduce overall heteroplasmy. Using cybrid models harboring two disease-causing mtDNA mutations—m.13513G>A and m.8344A>G—at varying heteroplasmy levels, we discovered that both the mutation type and load distinctly shape the nuclear DNA methylome. We found this methylation pattern to be critical for the survival of high-heteroplasmy cells but not for the low-heteroplasmy ones. Consequently, by disrupting this epigenetic programming with FDA approved DNA methylation inhibitors we managed to selectively impact high-heteroplasmy cybrids and reduce heteroplasmy. These findings were validated in both cultured cells and an in vivo xenograft model. Our study reveals a previously unrecognized role for nuclear DNA methylation in regulating cell survival in the context of mitochondrial heteroplasmy. This insight not only advances our understanding of mitochondrial-nuclear interactions but also introduces epigenetic modulation as a possible therapeutic avenue for mitochondrial diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9968dea2596dbecaa52b8340be510957ba04234e" target='_blank'>
              Rewriting nuclear epigenetic scripts in mitochondrial diseases as a strategy for heteroplasmy control
              </a>
            </td>
          <td>
            María J. Pérez, Rocío B. Colombo, S. Real, M. T. Branham, S. Laurito, Carlos T. Moraes, L. Mayorga
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The nuclear pore complex (NPC) is an evolutionarily conserved structure that maintains the traffic between the nucleus and cytoplasm. Here, we profiled the expression of nucleoporins (NUPs) in glioblastoma stem cells (GSCs) and found that NUP98 promoted GSC maintenance and therapeutic resistance. GSCs preferentially expressed NUP98, which is essential for GSC tumorigenesis in vitro and in vivo. RNA sequencing demonstrated that NUP98 regulated the expression of key DNA damage and repair pathways. NUP98 formed a complex with transcription factor p65 to directly activate genes involved in homologous repair. Attenuation of NUP98 or p65 expression induced unrepaired intrinsic DNA damage and sensitized GSC to ionizing radiation. Clinically, overexpression of NUP98 informs poor clinical outcome among glioblastoma (GBM) patients. Collectively, our results demonstrate that NUP98-p65 represents a novel node in the regulation of DNA repair, suggesting a therapeutic strategy with potential clinical benefits for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary This review presents an overview of recent findings on various epigenetic and post-transcriptional modifications of the epidermal growth factor receptor (EGFR), which activates the oncogenes RAS and MYC. Alterations in EGFR, RAS, and MYC play a crucial role in dysregulated processes that contribute to oncogenesis. Focused, targeted research aimed at understanding the distinct effects of these oncogenes and their carcinogenic modifications has facilitated the successful treatment of various malignancies using inhibitors of EGFR tyrosine kinase activity and other therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6048dcc607aa9e108befb3b09588362de65007" target='_blank'>
              Dynamic Multilevel Regulation of EGFR, KRAS, and MYC Oncogenes: Driving Cancer Cell Proliferation Through (Epi)Genetic and Post-Transcriptional/Translational Pathways
              </a>
            </td>
          <td>
            M. Šereš, Katarina Spacayova, Z. Sulová, Jana Špaldová, Albert Breier, L. Pavlikova
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="LINE-1 retrotransposons, comprising 17% of the genome, drive cancer instability through hypomethylation. The DIAMOND assay, targeting LINE-1 hypomethylation with bisulfite sequencing of cell free DNA, achieved AUCs of 88% to 100% across six cancer types, surpassing mutation-based diagnostics and suggesting utility in early cancer detection and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57cf07128fb65aaad9c55c65c3a93161bd10c47" target='_blank'>
              LINE-1 Retrotransposons as Cell-free DNA Biomarkers for Multi-Cancer Early Detection.
              </a>
            </td>
          <td>
            J. Szymanski, Pradeep S Chauhan, A. Chaudhuri
          </td>
          <td>2025-01-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8360ddf104ac6e4b2843cdfd9b7edf6a857c0d" target='_blank'>
              Dysregulation of transposable elements and PIWI-interacting RNAs in myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Z. Krejcík, D. Kundrát, Jirí Kléma, A. Hruštincová, I. Trsova, M. Belickova, Jaroslav Čermák, A. Jonášová, Jiri Dostal, M. Dostalova Merkerova
          </td>
          <td>2025-01-23</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction Traditional drug discovery efforts primarily target rapid, reversible protein-mediated adaptations to counteract cancer cell resistance. However, cancer cells also utilize DNA-based strategies, often perceived as slow, irreversible changes like point mutations or drug-resistant clone selection. Extrachromosomal DNA (ecDNA), in contrast, represents a rapid, reversible, and predictable DNA alteration critical for cancer’s adaptive response. Methods In this study, we developed a novel post-processing pipeline for automated detection and quantification of ecDNA in metaphase Fluorescence in situ Hybridization (FISH) images, leveraging the Microscopy Image Analyzer (MIA) tool. This pipeline is tailored to monitor ecDNA dynamics during drug treatment. Results Our approach effectively quantified ecDNA changes, providing a robust framework for analyzing the adaptive responses of cancer cells under therapeutic pressure. Discussion The pipeline not only serves as a valuable resource for automating ecDNA detection in metaphase FISH images but also highlights the role of ecDNA in facilitating swift and reversible adaptation to epigenetic remodeling agents such as JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed078aa348654cdad2d2c2164c322729a574c316" target='_blank'>
              Leveraging AI to automate detection and quantification of extrachromosomal DNA to decode drug responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Christina Ford, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The autonomous and active Long-Interspersed Element-1 (LINE-1, L1) and the non-autonomous Alu retrotransposon elements, contributing to 30% of the human genome, are the most abundant repeated sequences. With more than 90% of their sequences being methylated in normal cells, these elements undeniably contribute to the global DNA methylation level and constitute a major part of circulating-cell-free DNA (cfDNA). So far, the hypomethylation status of LINE-1 and Alu in cellular and extracellular DNA has long been considered a prevailing hallmark of ageing-related diseases and cancer. This study demonstrated that errors in LINE-1 and Alu methylation level measurements were caused by an excessive input quantity of genomic DNA used for bisulfite conversion. Using the minuscule DNA amount of 0.5 ng, much less than what has been used and recommended so far (500 ng-2 μg) or 1 μL of cfDNA extracted from 1 mL of blood, we revealed hypermethylation of LINE-1 and Alu in 407 tumour samples of primary breast, colon and lung cancers when compared with the corresponding pair-matched adjacent normal tissue samples (P < 0.05–0.001), and in cfDNA from 296 samples of lung cancers as compared with 477 samples from healthy controls (P < 0.0001). More importantly, LINE-1 hypermethylation in cfDNA is associated with healthy ageing. Our results have not only contributed to the standardized bisulfite-based protocols for DNA methylation assays, particularly in applications on repeated sequences but also provided another perspective for other repetitive sequences whose epigenetic properties may have crucial impacts on genome architecture and human health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af9de458a2650aad970d8197a4d94f0fb4283e9" target='_blank'>
              An appropriate DNA input for bisulfite conversion reveals LINE-1 and Alu hypermethylation in tissues and circulating cell-free DNA from cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Tung The Pham, Than Thi Nguyen, Trang Hien Do, Phương Thi Thu Luu, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Son Van Ho, Hang Viet Dao, Tong Van Hoang, Lan Thi Thuong Vo
          </td>
          <td>2024-12-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Chromosomal aneuploidies are a major cause of developmental failure and pregnancy loss. To investigate the possible consequences of aneuploidy on early embryonic development in vitro, we focused on primed pluripotent stem cells that are relatable to the epiblast of post-implantation embryos in vivo. We used human induced pluripotent stem cells (iPSCs) as an epiblast model and altered chromosome numbers by treating with reversine, a small-molecule inhibitor of monopolar spindle 1 kinase (MSP1) that inactivates the spindle assembly checkpoint, which has been strongly implicated in chromosome mis-segregation and aneuploidy generation. Upon reversine treatment, we obtained cells with varied chromosomal content that retained pluripotency and potential to differentiate into cells of three germ lineages. However, these cells displayed lagging chromosomes, increased micronuclei content, high p53 expression and excessive apoptotic activity. Cell proliferation was not affected. Prolonged in vitro culture of these cells resulted in a selective pool of cells with supernumerary chromosomes, which exhibited cellular hypertrophy, enlarged nuclei, and overproduction of total RNAs and proteins. We conclude that increased DNA damage responses, apoptosis, and improper cellular mass and functions are possible mechanisms that contribute to abnormal epiblast development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d165ff1955e4cb3146937b90c98c9776c59f3b72" target='_blank'>
              Chromosome number alterations cause apoptosis and cellular hypertrophy in induced pluripotent stem cell models of embryonic epiblast cells
              </a>
            </td>
          <td>
            Althea Stella Anil Martis, Loshini Soundararajan, Pallavi Shetty, Syed Moin, Tejashree Vanje, Yogeshwaran Jai Sankar, Shagufta Parveen
          </td>
          <td>2025-01-15</td>
          <td>Biology Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Cellular senescence plays a significant role in tissue aging. Senescent cells, which resist apoptosis while remaining metabolically active, generate endogenous DNA-damaging agents, primarily reactive oxygen species. Efficient DNA repair is therefore crucial in these cells, especially when they undergo senescence escape, resuming DNA replication and cellular proliferation. To investigate whether senescent cell transcriptomes reflect adequate DNA repair capacity, we conducted a comprehensive meta-analysis of 60 transcriptomic datasets comparing senescent to proliferating cells. Our analysis revealed a striking downregulation of genes encoding essential components across DNA repair pathways in senescent cells. This includes pathways active in different cell cycle phases such as nucleotide excision repair, base excision repair, nonhomologous end joining and homologous recombination repair of double-strand breaks, mismatch repair and interstrand crosslink repair. The downregulation observed suggests a significant accumulation of DNA lesions. Experimental monitoring of DNA repair readouts in cells that underwent radiation-induced senescence supported this conclusion. This phenomenon was consistent across various senescence triggers and was also observed in primary cell lines from aging individuals. These findings highlight the potential of senescent cells as ‘ticking bombs’ in aging-related diseases and tumors recurring following therapy-induced senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e8d9a9bd09842f0e1e6fb75af8a57d50e982d1" target='_blank'>
              Broad repression of DNA repair genes in senescent cells identified by integration of transcriptomic data
              </a>
            </td>
          <td>
            Yann Frey, M. Haj, Y. Ziv, Ran Elkon, Yosef Shiloh
          </td>
          <td>2024-12-31</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression.
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Formalin preservation of museum specimens has long been considered a barrier to molecular research due to extensive crosslinking and chemical modification. However, recent optimisation of hot alkaline lysis and proteinase K digestion DNA extraction methods have enabled a growing number of studies to overcome these challenges and conduct genome-wide re-sequencing and targeted locus-specific sequencing. The newest, and perhaps most unexpected utility of formalin preservation in archival samples is its ability to preserve in situ DNA-protein interactions at a molecular level. Retrieving this signal provides information about the relative compaction or accessibility of the genome to the transcriptional machinery required for gene expression. Thus, exposure to formalin essentially corresponds to taking a snapshot of organism-wide gene expression at the time of death. While DNA methylation and RNA-Seq analyses of dried tissues have provided glimpses into historical gene regulation, these techniques were previously limited to skeletal or desiccated remains, offering only partial insights. By examining fluid-preserved specimens, molecular tools can now be applied to a broader range of tissues, enabling more detailed tissue-specific gene regulation profiling across vertebrates. In this review, we chronicle the historical use of formaldehyde in collections and discuss how targeted chromatin profiling with assays like MNase-seq and FAIRE-seq are surmounting fixation challenges and unlocking invaluable insights into historical genomes and gene expression profiles. The deeper integration of molecular genetics with museum collections bridges the gap between past and present and provides a vital tool that could help us predict and mitigate some of the impacts of future environmental change, novel pathogens, or invasive species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71cdced5ded68073de0b09e74b9af4ad05548ef" target='_blank'>
              Reframing Formalin: A Molecular Opportunity Enabling Historical Epigenomics and Retrospective Gene Expression Studies.
              </a>
            </td>
          <td>
            C. E. Holleley, Erin E. Hahn
          </td>
          <td>2025-01-02</td>
          <td>Molecular ecology resources</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Fungal genome sizes exhibit more than a 100-fold variation, largely driven by the expansion of repetitive sequences such as transposable elements (TEs). Genomic defenses have evolved to counteract TE proliferation through silencing mechanisms operating at the epigenetic or transcript level. In fungi, repeat-induced point mutation (RIP) targets TEs by recognizing repetitive sequences and inducing mutagenesis. However, the prevalence of RIP across the fungal kingdom and the fidelity of the canonical C-to-T mutation signatures remain unclear. In this study, we address these gaps by tracking shifts in genome architecture across the fungal kingdom. We show that the loss of RIP components is associated with an 80% (∼30 Mb) increase in genome size in ascomycetes (class of Leotiomycetes). To track the impact of genome defenses, we designed a quantitative screen for RIP-like mutation signatures. The phylum of ascomycetes was unique in showing enrichment in mutation signatures in non-coding and repetitive sequences, consistent with a phylogenetically restricted occurrence of RIP-like genome defense systems. Then, we gene functions associated in a phylogenetic context with RIP-like mutation signatures. We identified a zinc-finger protein as the strongest candidate underpinning a novel mechanism of genome defenses. Our findings reveal the multifaceted drivers of genome defense systems and their close ties to genome size evolution in fungi, highlighting how proximate molecular mechanisms can shape genome evolution on deep phylogenetic scales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6148de5b7f0cf13898d5f28034ac9ab633d52a0" target='_blank'>
              A systematic screen for genetic factors underpinning transposon defense systems across the fungal kingdom
              </a>
            </td>
          <td>
            Thomas Badet, D. Croll
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352956df3f50dd3a8802a00f3ce2fb6c110258ba" target='_blank'>
              SMC translocation is unaffected by an excess of nucleoid associated proteins in vivo
              </a>
            </td>
          <td>
            Zhongqing Ren, Lindsey E. Way, Xindan Wang
          </td>
          <td>2025-01-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Most of the risk factors associated with chronic and complex diseases, such as cancer, stem from exogenous and endogenous exposures experienced throughout an individual’s life, collectively known as the exposome. These exposures can modify DNA, which can subsequently lead to the somatic mutations found in all normal and tumor tissues. Understanding the precise origins of specific somatic mutations has been challenging due to multitude of DNA adducts (i.e. the DNA adductome) and their diverse positions within the genome. Thus far, this limitation has prevented researchers from precisely linking exposures to DNA adducts and DNA adducts to subsequent mutational outcomes. Indeed, many common mutations observed in human cancers appear to originate from error-prone endogenous processes. Consequently, it remains unclear whether these mutations result from exposure-induced DNA adducts, or arise indirectly from endogenous processes or are a combination of both. In this review, we summarize approaches that aim to bridge our understanding of the mechanism by which exposure leads to DNA damage and then to mutation and highlight some of the remaining challenges and shortcomings to fully supporting this paradigm. We emphasize the need to integrate cellular DNA adductomics, long read-based mapping, single-molecule duplex sequencing of native DNA molecules and advanced computational analysis. This proposed holistic approach aims to unveil the causal connections between key DNA modifications and the mutational landscape, whether they originate from external exposures, internal processes or a combination of both, thereby addressing key questions in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdf82ec8445e0cef06c9a8eb260342f4ef9a96b" target='_blank'>
              Investigating the origins of the mutational signatures in cancer
              </a>
            </td>
          <td>
            Gunnar Boysen, Ludmil B. Alexandrov, R. Rahbari, I. Nookaew, Dave Ussery, M. Chao, Chiung-Wen Hu, Marcus S. Cooke
          </td>
          <td>2025-01-07</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The escalating global cancer burden necessitates the development of biomarkers with enhanced specificity and sensitivity for early diagnosis and therapeutic efficacy monitoring. The CNIH4 gene, an emerging biomarker, is increasingly recognized for its role in the malignant progression across various cancers.We conducted a comprehensive multi-omics analysis of CNIH4, including pan-cancer expression profiles, epigenetic alterations, immune microenvironment characteristics, and therapeutic response patterns. Our focus was on clinical features, molecular underpinnings, and drug sensitivity in breast cancer (BRCA) associated with CNIH4. In vitro studies were also performed to assess the effects of CNIH4 knockdown on cell proliferation and cell cycle in the MDA-MB-231 cell line.CNIH4 upregulation was observed in multiple cancers, significantly correlating with genomic instability. High CNIH4 expression levels were linked to poor prognosis across cancers and associated with key cancer-related pathways, particularly those in cell cycle regulation and DNA repair. Correlation analyses suggest a role for CNIH4 in the tumor immune microenvironment, as evidenced by its association with immune subtypes, immune-related genes, and immune cell infiltration. Single-cell and spatial transcriptome analyses confirmed that CNIH4 expression in BRCA predicts tumor malignancy. Drug sensitivity analysis revealed a significant correlation between CNIH4 and responsiveness to various kinase inhibitors and chemotherapeutic agents. In vitro experiments demonstrated that CNIH4 knockdown significantly impacts the proliferation and cell cycle of MDA-MB-231 cells.Our study highlights CNIH4 as a promising pan-cancer biomarker with significant implications for tumor progression and a critical role in cell cycle regulation in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7a2eade185851de2705c33f65b5768b5aafc69" target='_blank'>
              Deciphering the role of CNIH4 in pan-cancer landscapes and its significance in breast cancer progression
              </a>
            </td>
          <td>
            Yao Xu, Zengzhen Lai, Chaolin Li
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MET fusions (MET-Fs) are oncogenic drivers that remain poorly characterized. Analysis of 56 MET-F-positive tumors from an institutional cohort of 91,119 patients (79,864 DNA-seq plus 11,255 RNA-seq) uncovered two forms of MET-F pathobiology. The first group featured 5' partners with homodimerization domains fused in-frame with MET-tyrosine kinase domain (TKD), primarily originated from translocations, frequently excluded MET exon 14, mediated oncogenesis through cytoplasmic aggregation and constitutive activation, and were markedly sensitive to MET tyrosine kinase inhibitors (TKI) in pre-clinical models and patients with lung cancer. The second group lacked partner homodimerization motifs and retained MET transmembrane and extracellular domains. Their pathogenesis involved intrachromosomal rearrangements, resulting in partner selection for promoter hijacking and fusion allele amplification. Membrane-bound fusions were enriched in gliomas with RTK co-alterations. We provide a framework to comprehend the heterogeneous landscape of MET-Fs, supporting that fusion oncogenicity and MET-TKI sensitivity are determined by structural topology and pathogenomic context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020be0c73016e3bdb9e553a5f923d18e28c49838" target='_blank'>
              Pan-cancer analysis of oncogenic MET fusions reveals distinct pathogenomic subsets with differential sensitivity to MET-targeted therapy.
              </a>
            </td>
          <td>
            Christopher A Febres-Aldana, M. Vojnic, I. Odintsov, Tom Zhang, Ryan Cheng, Catherine Z. Beach, Daniel Lu, M. Mattar, Andrea M Gazzo, Leo Gili, M. Harshan, Ali Ameri, S. Machnicki, Xiuying Xiao, William W Lockwood, Xiaoyan Zhou, Qianlan Yao, A. Drilon, Natasha Rekhtman, Nameeta Shah, Anqi Li, Zebing Liu, Soo-Ryum Yang, M. Davare, Marc Ladanyi, R. Somwar
          </td>
          <td>2025-02-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="From RNA interference to chromatin silencing, diverse genome defense pathways silence selfish genetic elements to safeguard genome integrity. Despite their diversity, different defense pathways share a modular organization, where numerous specificity factors identify diverse targets and common effectors silence them. In the PIWI-interacting RNA (piRNA) pathway, target RNAs are first identified by complementary base pairing with piRNAs and then silenced by PIWI-clade nucleases. Such a binary architecture allows the defense systems to be readily adaptable, where new targets can be captured via innovation of specificity factors. Thus, our current understanding of genome defense against lineage-specific selfish genes has been largely limited to specificity factor innovations, while it remains poorly understood whether other types of innovations are required. Here, we describe a new type of innovation, which escalates the genome defense capacity to control a recently expanded selfish gene in Drosophila melanogaster. Through a targeted RNAi screen for repressors of Stellate-a recently evolved meiotic driver-we identified a defense factor, Trailblazer. Trailblazer is a transcription factor that promotes the expression of two PIWI-clade nucleases, Aub and AGO3, to match Stellate in abundance. Recent innovation in the DNA-binding domain of Trailblazer enabled it to elevate Aub and AGO3 expression, thereby escalating the silencing capacity of piRNA pathway to tame expanded Stellate and safeguard fertility. As copy-number expansion is a recurrent feature of diverse selfish genes across the tree of life, we envision that augmenting the defense capacity to quantitatively match selfish genes is a repeatedly employed defense strategy in evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b459a2d133898ca4732a04488f0f54cfda8ecf00" target='_blank'>
              Escalation of genome defense capacity enables control of an expanding meiotic driver.
              </a>
            </td>
          <td>
            Peiwei Chen, Katherine C. Pan, Eunice H. Park, Yicheng Luo, Y. C. G. Lee, A. Aravin
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b123662301192f16a8e24048c0300f73a9a5cc" target='_blank'>
              SiCLAT: simultaneous imaging of chromatin loops and active transcription in living cells
              </a>
            </td>
          <td>
            Xin Wan, Jie Kong, Xiaodi Hu, Lulu Liu, Yuanping Yang, Hu Li, Gaoao Liu, Xingchen Niu, Fengling Chen, Dan Zhang, Dahai Zhu, Yong Zhang
          </td>
          <td>2025-01-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a614193ffb0a35548300d5126509bf2d1bf067aa" target='_blank'>
              Defining ortholog-specific UHRF1 inhibition by STELLA for cancer therapy
              </a>
            </td>
          <td>
            Wenjing Bai, Jinxin Xu, Wenbin Gu, Danyang Wang, Ying Cui, Weidong Rong, Xiaoan Du, Xiaoxia Li, Cuicui Xia, Qingqing Gan, Guantao He, Huahui Guo, Jinfeng Deng, Yuqiong Wu, R. Yen, Srinivasan Yegnasubramanian, S. Rothbart, Cheng Luo, Linping Wu, Jinsong Liu, Stephen B Baylin, Xiangqian Kong
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Importance Chromatin accessibility is vital for gene regulation, determining the ability of DNA-binding proteins to access the genomic regions and drive transcriptional activity, reflecting environmental changes. Although human and murine studies have advanced the understanding of chromatin dynamics, domestic animals remain comparatively underexplored despite their importance in agriculture and veterinary medicine. Investigating the accessibility of chromatin in these species is crucial for improving traits such as productivity, disease resistance, and environmental adaptation. This review assessed chromatin accessibility research in domestic animals, highlighting its significance in understanding and improving livestock traits. Observations This review outlines chromatin accessibility research in domestic animals, focusing on critical developmental processes, tissue-specific regulation, and economically significant traits. Advances in techniques, such as Assay for Transposase-Accessible Chromatin using sequencing, have enabled detailed mapping of regulatory elements, shedding light on epigenetic regulation of traits, such as muscle development and productivity. Comparative studies have uncovered conserved and species-specific cis-regulatory elements across multiple species. These findings offer insights into regulatory mechanisms that can enhance breeding strategies and animal management. In addition, high-throughput techniques, such as single-cell analysis and deep-learning models, have advanced the study of chromatin accessibility in lesser-studied species. Conclusions and Relevance Chromatin accessibility is crucial in gene regulation in domestic animals, influencing development, immune response, and productivity. Despite the progress, more comprehensive epigenomic datasets and cross-species analytical tools are needed to harness chromatin accessibility in domestic animal research. Understanding these mechanisms has practical applications in improving livestock traits, advancing breeding programs, and developing disease-resistant animals, highlighting the importance of integrating epigenetic and genomic tools for enhancing animal health and productivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb96ae4313a5cd652a9a3747efeb45514dfc039c" target='_blank'>
              Comparative epigenetics of domestic animals: focusing on DNA accessibility and its impact on gene regulation and traits
              </a>
            </td>
          <td>
            Jeong-Woon Lee, Je-Yoel Cho
          </td>
          <td>2025-01-01</td>
          <td>Journal of Veterinary Science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Evidence suggests that increases in ploidy have occurred frequently in the evolutionary history of organisms and can serve adaptive functions to specialized somatic cells in multicellular organisms. However, the sudden multiplication of all chromosome content may present physiological challenges to the cells in which it occurs. Experimental studies have associated increases in ploidy with reduced cell survival and proliferation. To understand the physiological challenges that suddenly increased chromosome content imposes on cells, we used S. cerevisiae to ask how much chromosomal DNA cells may contain and what determines this limit. We generated polyploid cells using 2 distinct methods causing cells to undergo endoreplication and identified the maximum ploidy of these cells, 32–64C. We found that physical determinants that alleviate or exacerbate cell surface stress increase and decrease the limit to ploidy, respectively. We also used these cells to investigate gene expression changes associated with increased ploidy and identified the repression of genes involved in ergosterol biosynthesis. We propose that ploidy is inherently limited by the impacts of growth in size, which accompany whole-genome duplication, to cell surface integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c9419ff84380a674e570b7e466126c557aa8fb" target='_blank'>
              Cell integrity limits ploidy in budding yeast
              </a>
            </td>
          <td>
            Juliet Barker, Andrew W. Murray, Stephen P Bell
          </td>
          <td>2025-01-13</td>
          <td>G3: Genes | Genomes | Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine.
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, Jun Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 226


 Background:
 Metastatic prostate cancer (PC) accumulates significant genomic alterations as the disease progresses, exhibiting the highest levels of chromosomal instability (CIN) among all metastatic tumors (Bakhoum, Nature 2018). Furthermore, tumor suppressor genes (TSG;
 RB1
 ,
 PTEN
 and
 TP53
 ) are frequently altered in metastatic PC and are associated with adverse outcomes. Here, we aim to investigate CIN in metastatic hormone-sensitive PC (mHSPC) harbouring alterations in TSG.
 Methods:
 TSG and whole transcriptome gene expression was assessed by nCounter platform (N=354) and RNA-seq (N=60) in mHPSC samples, respectively. TSG
 low
 was considered when ≥2 out of 3 TSG presented low expression of a previously stablished cut-off, and TSG
 wt
 in the remaining cases. Differential gene expression analysis between TSG
 low
 vs TSG
 wt
 tumors of our RNA-seq cohort (N=60) and TCGA cohort (N=333) was performed. Differential expressed genes (DEGs) were considered if FDR <0.05 and log2FoldChange (LFC)>0.5. DEGs were correlated with castration-resistance PC-free survival (CRPC-FS) and overall survival (OS) by Cox analysis. CIN gene signatures were obtained from literature and the rest of Hallmark signatures from MSigDB.
 Results:
 In our cohort of mHSPC patients treated with androgen deprivation therapy + docetaxel, TSG
 low
 tumors (16.7%) displayed higher levels of CIN compared with TSG
 wt
 (CIN25 normalized enrichment score [NES] 2.13, p<0.001 and CIN70 NES 2.54, p<0001). Cell cycle and DNA-repair signatures were also overexpressed in TSG
 low
 tumors, contrasting with a low expression of androgen receptor pathway. Results were validated in the TCGA cohort, where TSG
 low
 presented also overexpression of CIN signatures. In our RNA-seq cohort, we found 22 DEGs associated with CIN (29.3% of the total DEGs) in TSG
 low
 tumors compared with TSG
 wt
 . 51 genes were significantly correlated with CRPC-FS in a multivariate analysis and 16 of them (31,3%) were CIN-associated genes. Among these genes, 5 encode proteins that are targetable for metastatic PC, with specific inhibitors already developed:
 BUB1
 (LFC 1.06, FDR 0.034),
 CDC7
 (LFC 12.4 FDR 0.003),
 KIFC1
 , (LFC 11,7, FDR 0.01),
 PLK1
 (LFC 1.08, FDR <0.001) and
 WEE1
 (LFC 0.79, FDR 0.011).
 Conclusions:
 TSG-altered mHSPC tumors harbor high levels of CIN, and differentially expressed CIN-associated genes could potentially be targets in mHSPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent developments in PacBio high-fidelity (HiFi) sequencing technologies have transformed genomic research, with circular consensus sequencing now achieving 99.9% accuracy for long (up to 25 kb) single-molecule reads. This method circumvents biases intrinsic to amplification-based approaches, enabling thorough analysis of complex genomic regions [including tandem repeats, segmental duplications, ribosomal DNA (rDNA) arrays, and centromeres] as well as direct detection of base modifications, furnishing both sequence and epigenetic data concurrently. This has streamlined a number of tasks including genome assembly, variant detection, and full-length transcript analysis. This review provides a comprehensive overview of the applications and challenges of HiFi sequencing across various fields, including genomics, transcriptomics, and epigenetics. By delineating the evolving landscape of HiFi sequencing in multi-omics research, we highlight its potential to deepen our understanding of genetic mechanisms and to advance precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584bbd4f29ec6dc8d7d0e916a78b0dd2a1379b9e" target='_blank'>
              Long and Accurate: How HiFi Sequencing is Transforming Genomics.
              </a>
            </td>
          <td>
            Bo Wang, Peng Jia, Shenghan Gao, Huanhuan Zhao, Gaoyang Zheng, Linfeng Xu, Kai Ye
          </td>
          <td>2025-02-07</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="An autosomal dominant GGGGCC repeat expansion in intron 1 of the C9orf72 gene is the most common genetic cause of both amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). Here, we set out to engineer a gene targeted mouse model harbouring a pathogenic length humanised C9orf72 repeat expansion allele, in order to model pathological mechanisms in a physiological context. In human disease, pathogenic repeats typically range from the hundreds to thousands of units in length, representing a considerable challenge for cellular and in vivo model generation given the instability of GC rich and repetitive DNA sequences during molecular cloning. To overcome this challenge, we developed new methodology to synthetically and iteratively build pure GGGGCC repeats within a linear vector system, which we then seamlessly and scarlessly embedded within the native human genomic sequence. This created a gene targeting DNA vector for homologous recombination of the human sequence in mouse embryonic stem cells. We used this novel targeting vector to generate a new gene targeted mouse allele, C9orf72h370, that for the first time has mouse C9orf72 intron 1 scarlessly replaced with human intron 1 including a pure (GGGGCC)370 hexanucleotide repeat expansion. We confirm that the mouse model expresses human intron 1-derived RNA and produces dipeptide repeat proteins derived from the GGGGCC repeat expansion. We now provide this model as a new freely available resource for the field. In addition, we demonstrate the utility of our cloning method for engineering diverse repeat expansion sequences for modelling other disorders, such as Fragile X Syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc6f010739dd766ccf0d0601df2a57ea4b4217f" target='_blank'>
              Generation of C9orf72h370 mice, an intron 1 humanised C9orf72 repeat-expansion knock-in model
              </a>
            </td>
          <td>
            R. Nair, M. Carcolé, David Thompson, Charlotte Tibbit, Ross McLeod, Alexander J. Cammack, Tatiana Jakubcova, D. Biggs, Matthew Wyles, Matthew Parker, Adam Caulder, L. Teboul, Chloe L Fisher-Ward, Ali R. Awan, M. Flower, Benjamin Davies, A. Isaacs, Elizabeth M. C. Fisher, Thomas J. Cunningham
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eedbdbd3810726b7db1ec867ee9197cd7581b73a" target='_blank'>
              Epigene functional diversity: isoform usage, disordered domain content, and variable binding partners
              </a>
            </td>
          <td>
            Leroy Bondhus, Aileen A. Nava, Isabelle S. Liu, Valerie A. Arboleda
          </td>
          <td>2025-02-01</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Post-transcriptional regulation by RNA-binding proteins (RBPs) is critical for mRNA stability, localization, and translation, primarily through interaction with the 3’-untranslated region. Dysregulation of RBPs has been associated with various cancers, with fragile X-related protein 1 (FXR1) emerging as a critical RBP involved in tumorigenesis through its interactions with target mRNAs. Despite its significance, the specific role of FXRI in cancer progression remains underexplored. In this study, we investigated FXR1’s function using SH-SY5Y cells. RNA immunoprecipitation (RNA-IP) assay was employed to isolate RNA complexes associated with FXR1. We generated stable cell lines with either FXR1 overexpression or silencing to assess the impact of FXRI binding to mRNA complexes. Subsequent analyses, including quantitative reverse transcription polymerase chain reaction, correlation analysis, gene expression profiling, and survival analysis, were performed to validate the interactions of FXR1 with target mRNAs. Our RNA-IP analysis identified several mRNAs significantly enriched in FXR1-bound RNA complexes, including SHISAL1, SLC43A3, NBAT1, PDZK1IP1, ACKR3, KCNN3, NECAB2, ANO5, ATOH8, IGFBP7, LEMD1, GPR35, WNT7A, and F2RL3. Notably, we observed changes in the expression levels of these genes following FXR1 overexpression or depletion, indicating FXR1-mediated functional regulation. Co-expression analysis further supported FXR1’s association with these target mRNAs. These findings highlight the significant role of FXR1 in regulating the stability and expression of key mRNAs implicated in malignancies. The dysregulation of FXR1 and its interaction with these target transcripts suggest that FXR1 plays a critical role in tumor biology, potentially offering new avenues for therapeutic interventions. This study provides a deeper understanding of FXR1’s involvement in cancer and underscores the importance of RBPs in the post-transcriptional regulatory landscape of cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68584e4b61388edd6ff9ff31b6a17585ebe8fc4" target='_blank'>
              FXR1 modulates the expression of oncogenes and tumor suppressor genes associated with poor cancer prognosis
              </a>
            </td>
          <td>
            Faiz Ali Khan, Xin-Ying Ji, Shaoping Ji, Jingcheng Dong
          </td>
          <td>2025-02-05</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5202cd39039a85ffc734b688a9ee572ce5b91b15" target='_blank'>
              Inhibition of GSK3β is synthetic lethal with FHIT loss in lung cancer by blocking homologous recombination repair
              </a>
            </td>
          <td>
            Shishi Tao, Yue Pu, E. Yang, Guowen Ren, Changxiang Shi, Li-Jie Chen, Liang Chen, J. Shim
          </td>
          <td>2025-01-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258ec54986c78fd95aedc9a7b2b989006fafe5e2" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2824e967c6a4d06b3abc17af2f0703099dbde0" target='_blank'>
              Uncovering the whole genome silencers of human cells via Ss-STARR-seq
              </a>
            </td>
          <td>
            Xiusheng Zhu, Lei Huang, Chao Wang, Guoli Li, Biao Deng, Dashuai Kong, Xiaoxiao Wang, Rongrong Chang, Yi Gu, Qiuhan Wen, Siyuan Kong, Yuwen Liu, Yubo Zhang
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, Yaokang Wu, Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c26d06bb50d508e1ca0550e77b81ac840e53150" target='_blank'>
              PARP4 deficiency enhances sensitivity to ATM inhibitor by impairing DNA damage repair in melanoma
              </a>
            </td>
          <td>
            Yuehua Li, Yu Liu, Jingjing Ma, Yuqi Yang, Q. Yue, G. Zhu, Weinan Guo, Tianwen Gao, Qiong Shi, Chunying Li
          </td>
          <td>2025-01-30</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Identifying genes involved in cancer is crucial for understanding the underlying molecular mechanisms of the disease and developing effective treatment strategies. Differential expression analysis (DEA) is the predominant method used to identify cancer-related genes. This approach involves comparing gene expression levels between different samples, such as cancerous and non-cancerous tissues, to identify genes that are significantly upregulated or downregulated in cancer. DEA is based on the commonly believed assumption that genes upregulated in cancerous tissues have the potential to function as oncogenes. Their expression levels often correlate with cancer advancement and unfavorable prognosis, whereas downregulated genes display the opposite correlation. However, contrary to the prevailing belief, our analysis utilizing The Cancer Genome Atlas (TCGA) databases revealed that the upregulated or downregulated genes in cancer do not always align with cancer progression or prognosis. These findings emphasize the need for alternative approaches for identifying cancer-related genes that may be more accurate and effective. To address this need, we compared the effectiveness of machine learning (ML) methods with that of traditional DEA in the identification of cancer-related genes. ML algorithms have the advantage of being able to analyze large-scale genomic data and identify complex patterns that may go unnoticed by traditional methods. Our results demonstrated that ML methods significantly outperformed DEA in the screening of cancer-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Ewing sarcoma (EwS) is a rare and aggressive disease affecting the bone and soft tissue of children and young adults. A FET::ETS fusion protein, typically EWSR1-FLI1 is pathognomonic for this disease. EwS is genomically quiet with widespread dysregulation driven by aberrant fusion activity leading to elevated endogenous levels of replication stress and epigenetic drift. While five-year survival rates are moderate for localized primary disease (∼70%), metastasized, relapsed, and refractory disease have much lower rates of survival (15-35%) and are associated with increased epigenetic heterogeneity. Treatment options are currently limited to systemic therapies which have not seen major advancement since the development of interval compression of chemotherapy and often lead to secondary complications for these patients. Our lab has identified a novel combination of replication stress response inhibitors (RSRi), DDK (DDKi) and WEE1 (WEE1i), which sensitize EwS cells to their own elevated levels of replication stress by removing repair strategies and forcing mitotic catastrophe and apoptosis. In one in vivo study, cell line derived xenografts in NSG mice were treated with DDKi (simurosertib) and WEE1i (adavosertib) in the following design: 1) vehicle control (n=5), 2) Irinotecan + Temozolomide (n=9), 3) Irinotecan (full dose) + DDKi + WEE1i (n=7), 4) Irinotecan (half dose) + DDKi + WEE1i (n=9), for a total of 30 samples. We have shown that this RSRi combination is able to effectively limit tumor growth over time even in the context of decreased chemotherapeutic dose. As increased replication stress can cause epigenetic drift, changes in methylation associated with treatment condition were investigated by reduced representation bisulfite sequencing. Differential methylation analysis was performed on individual CpG sites, CpG structures (islands, shores) and transcriptional elements (promoters, enhancers, transcription start sites). In this context, key pathways were found to be differentially methylated between treatment conditions, including those involved in cell signaling, motility, metabolism, and neuronal processes. Overall, this targeted therapy presents a promising therapeutic option for this pediatric patient population.
 Citation Format: Emily Isenhart, Ajay Gupta, Joyce Ohm. The effect of a novel therapy on DNA methylation in Ewing Sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b49082a0a3b2e18331aaf18415f528d694b06" target='_blank'>
              Abstract B012: The effect of a novel therapy on DNA methylation in Ewing Sarcoma
              </a>
            </td>
          <td>
            E. Isenhart, Ajay Gupta, Joyce Ohm
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying the roles of genes in cancer is critical in discovering potential genetic therapies for cancer care. Translocon-associated protein delta (TRAPδ), also known as signal sequence receptor 4 (SSR4), is a constituent unit in the TRAP/SSR complex that resides in the endoplasmic reticulum and plays a key role in transporting newly synthesized proteins into the endoplasmic reticulumn. However, its biological role in disease development remains unknown to date. This is the first study to identify the role of TRAPδ/SSR4 in colorectal cancer cells in vitro. Upon successful transient knockdown of TRAPδ/SSR4, we observed significant reduction of cell viability in all colorectal cancer cell lines tested. Both HCT 116 and SW480 cell lines were significantly arrested at S and G1 phases, while DLD-1 cells were significantly apoptotic. Moreover, TRAPδ/SSR4 stable knockdown HCT 116 and SW480 cells showed significantly lower viability, anchorage-independent growth, and increased S and G1 phase arrests. Overall, we conclude TRAPδ/SSR4 is a potential oncogene in human colorectal cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c566aa31384f1776417a4c1880816c68b25b7d5" target='_blank'>
              Identification of Translocon-associated Protein Delta as An Oncogene in Human Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Darshika Amarakoon, Wu-Joo Lee, Jing Peng, Seong-Ho Lee
          </td>
          <td>2024-12-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA damage is a common event in cells, resulting from both internal and external factors. The maintenance of genomic integrity is vital for cellular function and physiological processes. The inadequate repair of DNA damage results in the genomic instability, which has been associated with the development and progression of various human diseases. Accumulation of DNA damage can lead to multiple diseases, such as neurodegenerative disorders, cancers, immune deficiencies, infertility and aging. This comprehensive review delves the impact of alterations in DNA damage response genes (DDR) and tries to elucidate how and to what extent the same traits modulate diverse major human diseases, such as cancer, neurodegenerative diseases, and immunological disorders. DDR is apparently the trait connecting important complex disorders in humans. However, the pathogenesis of the above disorders and diseases are different and leading to the divergent consequences. It is important to discover the switch(es) that direct further the pathogenic process either to proliferative, or degenerative diseases. Our understanding the influence of DNA damage on diverse human disorders may enable a development of the strategies to prevent, diagnose, and treat these diseases. In our article, we analysed publicly available GWAS summary statistics from the NHGRI-EBI GWAS Catalog and identified 12,009 single nucleotide polymorphisms (SNPs) associated with cancer. Among these, 119 SNPs were found in DDR pathways, exhibiting significant p-values. Additionally, we identified 44 SNPs linked to various cancer types and neurodegenerative diseases (NDDs), including four located in DDR-related genes: ATM, CUX2, and WNT3. Furthermore, 402 SNPs were associated with both cancer and immunological disorders, with two found in DDR gene RAD51B. This highlights the versatility of the DDR pathway in multifactorial diseases. However, the specific mechanisms that regulate DDR to initiate distinct pathogenic processes remain to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Among the world's biggest causes of death is cancer. DNA damage is prevented by P53, a transcriptional factor that is encoded by the TP53 gene. A TP53 mutation was found in more than 50% of human cancers. In cancer cell lines with mutant or wild-type p53, CP-31398 has been used to stabilize the active state of p53 and increase p53 activity. Nonetheless, several issues persist, including deficiencies that directly impact the ensuing gene development and the absence of p53 selectivity. In vitro, CP-31398 demonstrates its anti-cancer properties by successfully stabilizing the DNA core domain of the tumor suppressor protein p53. According to the present study, CP-31398 saved the mutant p53 protein's transcriptional activation function by restoring the ability of mice and human tissue culture cells to bind DNA. Future research is thus necessary to fully assess CP-31398's potential. CP-31398 does not, it has been shown in some research, stabilize p53. The DNA-p53 core domain complex has also been seen to be altered and harmed by CP-31398, which is contained in DNA. Moreover, CP-31398 decreased both the His 273 mutant and wild-type p53's sequence-specific DNA binding. CP-3138's therapeutic potential has been shown in a number of in vitro and in vivo cancer models, such as hepatocellular carcinoma (HCC), colorectal cancer, pancreatic cancer, breast cancer, bladder cancer, neck squamous cell carcinoma, etc. In order to facilitate the applications of CP-31398, more research is still needed to comprehend the mechanism of CP-31398 in malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627c38440ba1e99cf398af06c70c6994116a801" target='_blank'>
              Applications of CP-31398 Targeting P53 on Cancers
              </a>
            </td>
          <td>
            Xintong Cai
          </td>
          <td>2025-01-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="p27Kip1 is a key cell cycle gatekeeper governing the timing of Cyclin-dependent kinase (CDK) activation/inactivation and, consequently, cell proliferation. Structurally, the protein is largely unfolded, a feature that strongly increases its plasticity and interactors and enhances the number of regulated cellular processes. p27Kip1, like other intrinsically unstructured proteins, is post-translationally modified on several residues. These modifications affect its cellular localization and address p27Kip1 for specific interactions/functions. Several germline or somatic CDKN1B (the p27Kip1 encoding gene) mutations have been demonstrated to be associated with multiple endocrine neoplasia type 4 (MEN4), hairy cell leukemia, small-intestine neuroendocrine tumors, and breast and prostate cancers. Here, we analyzed the effect of four CDKN1B missense and nonsense mutations found in patients affected by MEN4 or cancers, namely, c.349C>T, p.P117S; c.397C>A, p.P133T; c.487C>T, p.Q163*; and c.511G>T, p.E171*. By transfecting breast cancer cell lines, we observed increased growth and cell motility for all the investigated mutants compared to wild-type p27Kip1 transfected cells. Furthermore, we discovered that the mutant forms exhibited altered phosphorylation on key residues and different localization or degradation mechanisms in comparison to the wild-type protein and suggested a possible region as crucial for the lysosome-dependent degradation of the protein. Finally, the loss of p27Kip1 ability in blocking cell proliferation was in part explained through the different binding efficiency that mutant p27Kip1 forms exhibited with Cyclin/Cyclin-dependent Kinase complexes (or proteins involved indirectly in that binding) with respect to the WT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b858b52e6f58495fd391a4dfcc6f1900f00849e" target='_blank'>
              p27Kip1 and Tumors: Characterization of CDKN1B Variants Identified in MEN4 and Breast Cancer
              </a>
            </td>
          <td>
            Debora Bencivenga, E. Stampone, Jahanzaib Azhar, Daniela Parente, Waqar Ali, Vitale Del Vecchio, Fulvio Della Ragione, A. Borriello
          </td>
          <td>2025-01-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="A pseudogene is a non-functional copy of a protein-coding gene. Processed pseudogenes, which are created by the reverse transcription of mRNA and subsequent integration of the resulting cDNA into the genome, being a major pseudogene class, represent a significant challenge in genome analysis due to their high sequence similarity to the parent genes and their frequent absence in the reference genome. This homology can lead to errors in variant identification, as sequences derived from processed pseudogenes can be incorrectly assigned to parental genes, complicating correct variant calling. In this study, we quantified the occurrence of variant calling errors associated with pseudogenes, generated by the most popular germline variant callers, namely GATK-HC, DRAGEN, and DeepVariant, when analysing 30x human whole-genome sequencing data (n = 13,307). The results show that the presence of pseudogenes can interfere with variant calling, leading to false positive identifications of potentially clinically relevant variants. Compared to other approaches, DeepVariant was the most effective in correcting these errors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dbcef1abd35252b28627cdf942fe6a6b40cb02f" target='_blank'>
              Quantitative Analysis of Pseudogene-Associated Errors During Germline Variant Calling
              </a>
            </td>
          <td>
            Artem Podvalnyi, Arina Kopernik, Mariia Sayganova, M. Woroncow, G. Zobkova, A. Smirnova, Anton Esibov, A. Deviatkin, P. Volchkov, Eugene Albert
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma.
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c0dc0c7a111e57e84b3bce78e4c3f18ae8735f7" target='_blank'>
              Acetyltransferase NAT10 promotes gastric cancer progression by regulating the Wnt/β-catenin signaling pathway and enhances chemotherapy resistance
              </a>
            </td>
          <td>
            Yawen Chen, Jian Yang, Yadan Du, Zaihua Yan, Jieyun Gao, Haoyang Zhang, Qiong Wu, Bowen Nian, Xiujuan Huang, Mingxu Da
          </td>
          <td>2025-02-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Stable inheritance of DNA N6-methyladenine (6mA) is crucial for its biological functions in eukaryotes. Here, we identify two distinct methyltransferase (MTase) complexes, both sharing the catalytic subunit AMT1, but featuring AMT6 and AMT7 as their unique components, respectively. While the two complexes are jointly responsible for 6mA maintenance methylation, they exhibit distinct enzymology, DNA/chromatin affinity, genomic distribution, and knockout phenotypes. AMT7 complex, featuring high MTase activity and processivity, is connected to transcription-associated epigenetic marks, including H2A.Z and H3K4me3, and is required for the bulk of maintenance methylation. In contrast, AMT6 complex, with reduced activity and processivity, is recruited by PCNA to initiate maintenance methylation immediately after DNA replication. These two complexes coordinate in maintenance methylation. By integrating signals from both replication and transcription, this mechanism ensures the faithful and efficient transmission of 6mA as an epigenetic mark in eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2893bf0a4630bdeb041736c2ee7ebc2881aa61a" target='_blank'>
              Dual modes of DNA N6-methyladenine maintenance by distinct methyltransferase complexes.
              </a>
            </td>
          <td>
            Yuanyuan Wang, Bei Nan, Fei Ye, Zhe Zhang, Wentao Yang, Bo Pan, Fan Wei, Lili Duan, Haicheng Li, Junhua Niu, Aili Ju, Yongqiang Liu, Dantong Wang, Wenxin Zhang, Yifan Liu, Shan Gao
          </td>
          <td>2025-01-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
α thalassemia/mental retardation syndrome X-linked (ATRX) serves as a part of the sucrose nonfermenting 2 (SNF2) chromatin-remodeling complex. In interphase, ATRX localizes to pericentromeric heterochromatin, contributing to DNA double-strand break repair, DNA replication, and telomere maintenance. During mitosis, most ATRX proteins are removed from chromosomal arms, leaving a pool near the centromere region in mammalian cells, which is critical for accurate chromosome congression and sister chromatid cohesion protection. However, the function and localization mechanisms of ATRX at mitotic centromeres remain largely unresolved.


METHODS
The clustered regularly interspaced short palindromic repeats with CRISPR-associated protein 9 (CRISPR-Cas9) system and overexpression approaches were employed alongside immunofluorescence to investigate the mechanism of ATRX localization at the centromere. To study the binding mechanism between ATRX and heterochromatin protein 1 (HP1), both full-length and truncated mutants of hemagglutinin (HA)-ATRX were generated for co-immunoprecipitation and glutathione S-transferase (GST)-pull assays. Wild-type ATRX and HP1 binding-deficient mutants were created to investigate the role of ATRX binding to HP1 during mitosis, with the Z-Leu-Leu-Leu-al (MG132) maintenance assay, cohesion function assay, and kinetochore distance measurement.


RESULTS AND CONCLUSIONS
Our research demonstrated that HP1α, HP1β, and HP1γ facilitate the positioning of ATRX within the mitotic centromere area through their interaction with the first two [P/L]-X-V-X-[M/L/V] (PxVxL)motifs at the N-terminus of ATRX. ATRX deficiency causes aberrant mitosis and decreased centromeric cohesion. Furthermore, reducing Wapl activity can bypass the need for ATRX to protect centromeric cohesion. These results provide insights into the mechanism of ATRX's centromeric localization and its critical function in preserving centromeric cohesion by reducing Wapl activity in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2508ee19573d906286f1694bdd429742f1bd10" target='_blank'>
              HP1 Promotes the Centromeric Localization of ATRX and Protects Cohesion by Interfering Wapl Activity in Mitosis.
              </a>
            </td>
          <td>
            Erchen Zhang, Lei Peng, Kejia Yuan, Zexian Ding, Qi Yi
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic gamma herpesviruses, including Epstein–Barr Virus (EBV) and Kaposi’s Sarcoma-associated Herpesvirus (KSHV), are opportunistic cancer-causing viruses and induces oncogenesis through complex mechanisms, which involves manipulation of cellular physiology as well as epigenetic and epitranscriptomic reprogramming. In this review, we describe the intricate processes by which these viruses interact with the epigenetic machinery, leading to alterations in DNA methylation, histone modifications, and the involvement of non-coding RNAs. The key viral proteins such as EBNA1 and LMP1 encoded by EBV; LANA and vGPCR encoded by KSHV; play pivotal roles in these modifications by interacting with host factors, and dysregulating signaling pathways. The resultant reprogramming can lead to activation of oncogenes, silencing of tumor suppressor genes, and evasion of the immune response, which ultimately contributes to the oncogenic potential of these viruses. Furthermore, in this review, we explore current therapeutic strategies targeting these epigenetic alterations and discuss future directions for research and treatment. Through this comprehensive examination of the epigenetic and epitranscriptomic reprogramming mechanisms employed by oncogenic gamma herpesviruses, we aim to provide valuable insights into potential avenues for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d106bf10134bf9b8889820e2db155164b9012c0" target='_blank'>
              Epigenetic and epitranscriptomic regulation during oncogenic γ-herpesvirus infection
              </a>
            </td>
          <td>
            Rajnish Kumar Singh, Ramakrishna Vangala, Atharva S. Torne, Dipayan Bose, E. Robertson
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While the mutational landscape across early T-cell precursor acute lymphoblastic leukemia (ETP-ALL) and ETP-like leukemia is known, establishing a unified framework that activates stem cell genes characteristic of these tumors remains elusive. Using complementary mouse and human models, chromatin mapping, and enhancer profiling, we show that the coactivator ZMIZ1 promotes normal and malignant ETP population growth by inducing the transcription factor MYB in feedforward circuits to convergently activate oncogenes (MEF2C, MYCN, and BCL2) through essential enhancers. A key superenhancer, the N-Myc regulating enhancer (NMRE), drives malignant ETP population growth but is dispensable for normal lymphopoiesis. This network of stem cell superenhancers identifies treatment-resistant tumors and poor survival outcomes; unifies diverse ETP-ALLs; and contributes to cardinal features of the recently genomically identified high-risk bone marrow progenitor-like (BMP-like) ETP-ALL tumor-stem cell/myeloid gene expression, inhibited NOTCH1-induced T-cell development, aggressive clinical behavior, and venetoclax sensitivity. Since ZMIZ1 is dispensable for essential homeostasis, it might be possible to safely target this network to treat high-risk diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c779b27de9c7947d2736cb472ee433351114fe43" target='_blank'>
              Native stem cell transcriptional circuits define cardinal features of high-risk leukemia.
              </a>
            </td>
          <td>
            Qing Wang, Francesco Boccalatte, Jason Xu, Giovanni Gambi, Bettina Nadorp, Fatema Akter, Carea Mullin, A. Melnick, Elizabeth Choe, Anna C. McCarter, Nicole A Jerome, Siyi Chen, Karena Lin, Sarah Khan, R. Kodgule, Jonathan H Sussman, P. Pölönen, Javier Rodriguez-Hernaez, Sonali Narang, Kleopatra Avrampou, B. King, A. Tsirigos, R. Ryan, C. Mullighan, D. Teachey, Kai Tan, I. Aifantis, M. Chiang
          </td>
          <td>2025-02-19</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association.
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-19</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background/Objectives: The highly polymorphic Major Histocompatibility Complex (MHC) genomic region, located on the short arm of chromosome 6, is implicated genetically in Parkinson’s disease (PD), a progressive neurodegenerative disorder with motor and non-motor symptoms. Previously, we reported significant associations between SINE-VNTR-Alu (SVA) expression quantitative trait loci (eQTLs) and Human Leucocyte Antigen (HLA) class I genotypes in PD. In this study, we aimed to evaluate SVA associations and their regulatory effects on transposable element (TE) transcription in the MHC class I region. Methods: Transcriptome data from the peripheral blood cells of 1530 individuals in the Parkinson’s Progression Markers Initiative (PPMI) cohort were reanalyzed for TE and gene expression using publicly available bioinformatics tools, including Salmon and Matrix-eQTL. Results: Four structurally polymorphic SVAs regulated the transcription of 18 distinct clusters of 235 TE loci, comprising LINEs (33%), SINEs (19%), LTRs (35%), and ancient transposon DNA elements (12%) located near HLA genes. The transcribed TEs were predominantly short, with an average length of 445 nucleotides. The regulatory effects of these SVAs varied significantly in terms of TE types, numbers, and transcriptional activation or repression. The SVA-regulated TE RNAs in blood cells appear to function as enhancer-like elements, differentially influencing the expression of HLA class I genes, non-HLA genes, and noncoding RNAs. Conclusions: These findings highlight the roles of SVAs and their associated TEs in the complex regulatory networks governing coding and noncoding gene expression in the MHC class I region, with potential implications for immune function and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7af8d419a6f31034c910704dd0806ace1362d2" target='_blank'>
              The Transcription of Transposable Elements Differentially Regulated by SVAs in the Major Histocompatibility Complex Class I Region of a Parkinson’s Progression Markers Initiative Cohort
              </a>
            </td>
          <td>
            Jerzy K. Kulski, A. Pfaff, S. Kõks
          </td>
          <td>2025-01-06</td>
          <td>Journal of Molecular Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98077b8e7b73940125f0da65abe02bf159f00a4" target='_blank'>
              Multi-omics analyses reveal biological and clinical insights in recurrent stage I non-small cell lung cancer
              </a>
            </td>
          <td>
            Chengdi Wang, Jingwei Li, Jingyao Chen, Zhoufeng Wang, Guonian Zhu, Lujia Song, Jiayang Wu, Changshu Li, Rong Qiu, Xuelan Chen, Li Zhang, Weimin Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Recent studies have demonstrated that the production of bidirectional enhancer-derived transcripts (eRNAs) is a characteristic of an active Cis-regulatory element (CRE). Higher levels of eRNA synthesis correlate with the activation of histone modifications, a potentially valuable tool for deciphering the complexity of the gene regulatory network. To understand the changes of CREs during gonadal development in mice, we collected gonadal WT1-positive cells from the piggyBac-Wt1-mCherry-2A-EGFP (PBWt1-RG) reporter strain at E13.5, E16.5, and P0 in both sexes and conducted Cap Analysis of Gene Expression analysis (CAGE) which is capable to capture transcriptional starting site (TSS). We compared the levels of intergenic bidirectional RNA, i.e, potentially eRNA, according to sex at each stage (testis somatic cells vs ovary somatic cells at E13.5, E16.5, and P0) and stage in each sex (E13.5 vs E16.5, E16.5 vs P0, and E13.5 vs P0 in testis somatic cells or ovary somatic cells). Intergenic RNAs with significant changes (|Log2FC| > 1, p < 0.05) were selected. The TSS profile of intergenic RNA changed more profoundly in testis somatic cells than in ovary somatic cells, suggesting embryonic testicular development is driven by larger changes in a transcriptional regulatory network than ovarian development. Based on the profiles of the predicted transcription factors (TFs) that would bind to the active CREs during gonadal development, the NR4A, EGR, and TCF3 families would be novel TFs to play pivotal roles in gonadal development. Identifying active CREs using eRNAs would provide a means to comprehensively understand the transcriptional regulatory system, leading to valuable insights into the gonadal development of male and female individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8877e8a45402ec1d0fc04d7b02b6aa2118971232" target='_blank'>
              Analysis of Functional Cis-regulatory Elements Reveals Novel Transcriptional Regulatory Mechanisms in Gonadal Development.
              </a>
            </td>
          <td>
            S. Kirino, Ryuichi Nakagawa, Maki Gau, K. Takasawa, Yasuhiro Murakawa, H. Kawaji, Y. Hayashizaki, Tomohiro Morio, K. Kashimada
          </td>
          <td>2025-01-20</td>
          <td>Sexual development : genetics, molecular biology, evolution, endocrinology, embryology, and pathology of sex determination and differentiation</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f70d920d0162e98d15c899f1702b4ba2ed290a37" target='_blank'>
              BRD4 interacting genes as prognostic biomarkers in hepatocellular carcinoma for optimized treatment strategies
              </a>
            </td>
          <td>
            Huapeng Zhang, Jie Yu, Feng Zhang, Qi Liu, Jie Pan, Peihao Wen, Danyang Lu, Rong Liu, Tong Wang, Senyan Wang
          </td>
          <td>2025-02-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee69beabd8a047c43345e066d4bca544bc872682" target='_blank'>
              Satellite DNA shapes dictate pericentromere packaging in female meiosis.
              </a>
            </td>
          <td>
            Damian Dudka, Jennine M. Dawicki-McKenna, Xueqi Sun, Keagan Beeravolu, Takashi Akera, Micheal A. Lampson, Ben E. Black
          </td>
          <td>2025-01-08</td>
          <td>Nature</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="
 High-grade serous carcinoma (HGSC) remains the deadliest gynecological cancer, mostly due to the widespread metastases in abdominal cavities already at diagnosis. To improve therapy success, understanding metastasis routes and their molecular characteristics is crucial. Herein, we used >400 samples subjected to whole-genome bisulfite (WGBS) and RNA sequencing from 125 patients with HGSC, enrolled to prospective, longitudinal, multi-region, observational DECIDER trial (NCT04846933). These real-world epigenetics data allowed, for the first time, to delineate metastatic routes in HGSC. Our results show that cancer cells released from the site-of-origin sites seed to both mid-abdomen and upper abdomen in parallel. The dominant mechanism responsible for almost half of the spread was direct surface-to-surface release of ovarian tumor cells into the peritoneal cavity. The major source of cancer cells in the pelvic area was ovary, from which cancer disseminated to all intra-abdominal organs, including omentum. Tumor cells from pelvic bowel displayed strong tropism for virtually all metastatic deposits, including such distant metastases, as liver, vagina, and lungs, which suggested hematogenous spread. Ascites play a prominent role in fueling colonization, accounting for additional 20% of direct spread. The seeding via ascitic fluid remained localized within the abdominal area, however, ascites colonized primarily para-aortic and inguinal lymph nodes. Lymph node metastases were noted in one-third of possible spreading modes. HGSC dissemination via inguinal lymph nodes represented a rare, but aggressive trajectory, as these tumors, originated directly from tube were prone to invade all distant locations. Comparison of the paired samples before and after chemotherapy revealed that most sensitive to neoadjuvant chemotherapy (NACT) tumors are adnexal and omental tumors. Peritoneum and mesentery showed prominent resistance to treatment, as well as lymph nodes. Thus, patients with metastases in inguinal lymph nodes and bowel are at the highest risk of developing widespread intra-abdominal and distant metastases. Furthermore, the standard-of-care platinum/taxane therapy, along with maintenance relapse therapies are ineffective for eradication of these tumors. Ascites displayed an intriguing case: being virtually non-responding to NACT, epigenetic profiles at relapse exhibited a large change. This implies that epigenetic processes in ascites evolve along with the cancer progression, driving chemoresistance. Taken together, using DNA methylation changes from >400 samples we revealed metastasis routes in HGSC, as well as tissue-specific chemotherapy treatment response. Dissection of the key regulatory pathways of metastatic routes helps identify patients at the highest risk of distant metastases and suggests new targets for therapies.
 Citation Format: Alexandra Lahtinen, Giovanni Marchi, Kari Lavikka, Daria Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, Taru A Muranen, Jaana Oikkonen, Antti Häkkinen, Anni Virtanen, Johanna Hynninen, Sampsa Hautaniemi. DNA methylation reveals metastatic colonization routes in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f704aa07c09b6c48c3c1f8d354bbb48ab6359478" target='_blank'>
              Abstract PR003: DNA methylation reveals metastatic colonization routes in ovarian cancer
              </a>
            </td>
          <td>
            A. Lahtinen, G. Marchi, K. Lavikka, D. Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, T. Muranen, J. Oikkonen, Antti Häkkinen, A. Virtanen, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a24af38423fb1cff9abd834b78aff5969e9b847" target='_blank'>
              Establishment and characterization of three gemcitabine-resistant human intrahepatic cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Jiandong Li, Yanxin Hu, Jiayao Zhang, Weiguang Zhang, Jianhua Yu, Baochun Lu
          </td>
          <td>2025-02-09</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Proteolysis-targeting chimeras (PROTACs) are an innovative and promising technology designed to degrade harmful proteins associated with cancer. While these drugs have demonstrated significant potential in treating various cancer types, their application in pediatric acute myeloid leukemia (AML) remains underexplored. In this study, we evaluated two PROTAC drugs, CC-90009 and GU3341, both of which target the degradation of the protein GSPT1. The treatment effectively inhibited tumor growth, suppressed cancer cell proliferation, and induced cell death. Notably, it showed pronounced efficacy in two subtypes of pediatric AML driven by specific gene fusions: RUNX1::RUNX1T1 and FUS::ERG. Additionally, the degradation of GSPT1 led to a reduction in the levels of these leukemia-causing gene fusions, highlighting the critical role of GSPT1 in the disease. These findings suggest a promising new approach for treating pediatric AML by targeting proteins essential for leukemic cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4d8da65827555a74171c7326a8703bfbb7aabcf" target='_blank'>
              PROTAC-Mediated GSPT1 Degradation Impairs the Expression of Fusion Genes in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            A. Perzolli, C. Steinebach, Jan Krönke, Michael Gütschow, C. Zwaan, F. Barneh, Olaf Heidenreich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Disruption of cellular communication that regulates normal physiology is often key factor in the development of disease, including cancer. Extracellular vesicles (EVs) are mediators of cell-cell communication, modulating local and distant microenvironments and playing an important role influencing tumor progression at both early and late stages. Indeed, EV-mediated communication participate in the initial steps of primary tumor transformation and proliferation, as well as the preparation of the pre-metastatic niche and subsequent metastasis. In this context, the presence of DNA in EVs (EV-DNA) is particularly intriguing, with important biological implications and significant potential as a biomarker in liquid biopsies. In this review we will discuss the mechanisms involved in EV-shed DNA and the potential impact in tumor evolution. In addition, it has become apparent in recent years that the secretion of EVs also influences the behavior of the surrounding microenvironment. An important unresolved challenge in oncology is the resistance of tumors to treatment, one of the primary causes of high cancer mortality. The role of EVs in therapy resistance has garnered considerable interest. In the later part of this review, we will also examine the potential involvement of EVs in resistance to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412b9e12320c3e56e9fe1304933d6b3b6a3cf101" target='_blank'>
              The influence of extracellular vesicles on tumor evolution and resistance to therapy.
              </a>
            </td>
          <td>
            Enrique Bastón, Juan García-Agulló, Héctor Peinado
          </td>
          <td>2025-02-21</td>
          <td>Physiological reviews</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7d2fbf8a9f9c82b98c7984b3782b85218b8319" target='_blank'>
              Crypt density and recruited enhancers underlie intestinal tumour initiation.
              </a>
            </td>
          <td>
            Liam T. Gaynor, Harshabad Singh, Guodong Tie, Krithika Badarinath, Shariq Madha, Andrew Mancini, Swarnabh Bhattacharya, Mikio Hoshino, Frederic J. de Sauvage, Kazutaka Murata, Unmesh Jadhav, Ramesh A. Shivdasani
          </td>
          <td>2025-01-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 255


 Background:
 Homologous recombination repair (HRR) genes, including DNA repair genes, are increasingly recognized for impacting treatment resistance and prognosis of metastatic prostate cancer (mPC). Next-generation sequencing of circulating tumor DNA (ctDNA) enables non-invasive, longitudinal monitoring of molecular alterations in patients (pts) undergoing systemic therapies. This study explores the dynamic changes in ctDNA profiles in mPC, particularly focusing on the HRR alterations.
 Methods:
 We utilized GuardantINFORM, a clinical-genomics database containing de-identified ctDNA test results and commercial payer-claims data. Eligible pts had a diagnosis of mPC and underwent ctDNA testing pre (within 3 months before initiation) and post (within 3 months of discontinuation before initiating subsequent treatment) Androgen Receptor Pathway Inhibitors (ARPi), poly ADP-ribose polymerase inhibitors (PARPi), and taxane chemotherapy. Mann-Whitney test assessed the differences in mutations. We also examined the association between ctDNA burden and overall survival (OS).
 Results:
 From a database of 21,682 pts with mPC, 145 had ctDNA collected pre/post-ARPi, 54 pre/post PARPi and 115 pre/post taxane chemotherapy. Pre-to-post-ARPI, individual HRR alterations increased from 21 to 34, with the most common (%pre/%post) genes to be altered, including ATM (5.5/7.6), BRCA2 (3/6) and CDK12 (3/2). In the PARPi group, the most prevalent (%pre/%post) genes to be altered were ATM (28/22), BRCA2 (28/20) and BRCA1 (4/6). Although overall BRCA2 prevalence declined, 11 new unique alterations were detected post-PARPi, along with an increase in unique BRCA1 (2 to 4) and BRCA2 (16 to 21) alterations. Three of 25 patients (12%) who underwent serial testing pre- and post-PARPi exhibited BRCA reversion mutations. In the taxane group, notable (%pre/%post) HRR genes to be altered included ATM (4/3), BRCA1 (1/3), and BRCA2 (1/3). Patients with higher ctDNA burden in pre-treatment samples exhibited worse OS.
 Conclusions:
 This study reveals dynamic changes in HRR gene alterations in mPC using ctDNA profiling. We observed increased unique alterations in ATM, BRCA1, and BRCA2 post-treatment, along with BRCA reversion mutations, indicating potential treatment resistance mechanisms. These findings underscore the importance of tailoring therapeutic approaches based on HRR gene profiles in mPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f25e9aede0dd0d8017cba72b4efd6a1c07f803" target='_blank'>
              Molecular evolution of HRR gene alterations in metastatic prostate cancer: A ctDNA-based study.
              </a>
            </td>
          <td>
            Chinmay T Jani, Eli Tran, Nicole Zhang, J. Tsai, Jianjie Dong, Elizabeth Pan, Justine Panian, Rana R McKay
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, Judith Mary Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 DNA methylation (DNAm) reflects regions of the genome that are transcriptionally repressed or active depending on the methylation level. Multitudes of DNAm “clocks” have been created to predict biological age and health outcomes. Deviations in biological age from chronological age reflect an alteration in the normal biological aging process. Clonal hematopoiesis of indeterminate potential (CHIP) arises in hematopoietic stem cells that have accumulated somatic mutations in key driver genes which provide a selective advantage for clonal expansion. Subjects with CHIP mutations are more prone to negative clinical outcomes such as coronary heart disease. While DNAm and CHIP have been studied for the past decade, the associations between the two are only now starting to become apparent. In this study we propose an improved measurement of aging based on the Jenson-Shannon distance (JSD) and associate this measurement with CHIP and adverse clinical outcomes. Bisulfite amplicon sequencing of 20 CpG-rich targets that change in methylation with age was performed on an initial cohort of 155 individuals from the National Institute of Neurological Disorders and Stroke (NINDS). JSD was used to construct a DNA methylation chaos clock (DMC) on these targets. DMC provided better predictions of chronological age than DNAm alone (r=0.9; p<0.0001). We validated our findings in 286 individuals from Cooper University Hospital for Management of Trauma where the accuracy of the DMC was confirmed in correlations of predicted vs actual age (r=0.86; p<0.0001). We next examined the association between DMC age acceleration and clinical outcomes. It was found that in individuals over the age of 50, DMC acceleration was associated with smoking and disease incidence (e.g. congestive heart failure and chronic obstructive pulmonary disease) independent of chronological age. The incidence of CHIP mutations in three key driver genes (DNMT3A, TET2, and ASXL1) was assayed in a subset of 345 individuals from both cohorts. Individuals with CHIP mutations in key driver genes had greater deviations in their predicted age than individuals without CHIP mutations (p<0.001). This study demonstrates that the DMC assay is a candidate for improvements in our measurements of biological age and it reflects important changes in an individual’s biological aging process.
 Citation Format: Anthony J Pompetti, Woonbok Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre Issa. DNA methylation variability in aging and Clonal Hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadd86e514d184ff56ec546d04b1399f1bbf484d" target='_blank'>
              Abstract A016: DNA methylation variability in aging and Clonal Hematopoiesis
              </a>
            </td>
          <td>
            Anthony J Pompetti, W. Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 845


 Background:
 CAUBC (metastatic and surgically incurable advanced UBC) is a challenging with significant need for improvement in systemic therapies. We aimed to explore HRD sig status and related biomarkers, including other GA, to generate hypothesis and inform clinical trial designs for these patients.
 Methods:
 8825 cases of CAUBC underwent comprehensive genomic profiling (CGP) to examine all classes of genomic alterations (GA). MSI high status, tumor mutation burden (TMB) levels, genomic ancestry and trinucleotide mutational signatures were determined from the sequencing data. HRDsig status was calculated using a broad set of genome-wide copy number features. PD-L1 was determined by IHC using the Dako 22C3 tumor proportional score (TPS) system. Results were compared using the Fisher exact system with the Benjamini-Hochberg adjustment to correct for false discovery.
 Results:
 387 (4.4%) of CAUBC cases featured a positive HRDsig status (HRDsig+). The median age (62-63) was similar in the HRDsig+ and HRDsig- groups and both groups were predominantly male (75-77%). The GA/tumor frequencies were similar (5-6). MSI-high status was extremely low in both groups (0.8% vs 0%). The median TMB (10 vs 6.3; P<.0001) and frequency of TMB > 10 mutations/Mb (50.9% vs 34.8%; P=0.016) was higher in the HRDsig+. Trinucleotide mutational signature distribution, PD-L1 low (1-49% TPS) and PD-L1 high (≥50% TPS) expression were similar in both groups. As anticipated, GA in genes associated with HRD including
 BRCA1
 (9.0% vs 1.6%; P<0.0001),
 BRCA2
 (17.1% vs 2.5%; P<0.0001),
 ATM
 (9.6% vs 5.3%; P=0.0003) and
 RAD21
 (8.5% vs 3.4%; P<0.0001) were more frequent in the HRDsig+ cases. In the HRDsig- group, 83.7%/87.0% of
 BRCA1
 /
 BRCA2
 mutated CAUBC were mono-allelic, likely non-driver GA, respectively.
 FGFR3
 GA were significantly more frequent in the HRDsig- CAUBC (18.1% vs 10.9%; P=0.001). GA in
 ERBB2
 (17.8-18.5%) and
 ERBB3
 (6.2-7.8%) were similar in both groups. MTOR pathway related mutations, including
 PTEN
 (4.5-6.2%) and
 PIK3CA
 (18.6-21.7%) were similar in both groups. GA in
 TERT
 were more frequent in the HRDsig- group (78.9% vs 57.6%; P<0.0001) and GA in
 TP53
 were more frequent in the HRDsig+ cases (72.1% vs 60.5%; P<0.0001).
 Conclusions:
 With a 4.4% frequency, HRDsig+ status is a relatively uncommon biomarker in CAUBC. This finding may inform clinical trial designs, for example with PARP inhibitors and other ‘HRD-targeting’ agents. Limitations include the retrospective nature, potential selection bias and lack of clinical outcomes annotation.


 Genomic differences between HRDsig- and HRDsig+ CAUBC groups.





 CAUBC HRDsig-
 CAUBC HRDsig+
 P value




 EUR Ancestry
 85.2%
 78.6%
 0.003



 ATM

 5.3%
 9.6%
 0.003



 BRCA1

 1.6%
 9.0%
 <.0001



 BRCA2

 2.5%
 17.1%
 <.0001



 FGFR3

 18.1%
 10.9%
 0.001



 RAD21

 3.4%
 8.5%
 <.0001



 TP53

 60.5%
 72.1%
 <.0001



 RB1

 2.7%
 5.4%
 <.0001


 TMB≥10 mut/Mb
 34.8%
 50.9%
 0.016




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acfe5ecebe48c685d0ade29fb13cedf2e576caf" target='_blank'>
              Frequency and impact of a genome-wide homologous recombination deficiency signature (HRDsig+) on the genomic landscape of clinically advanced urothelial bladder carcinoma (CAUBC).
              </a>
            </td>
          <td>
            Michael F. Basin, Andrea Necchi, Roger Li, Philippe E. Spiess, A. Kamat, P. Grivas, Shilpa Gupta, Neeraj Agarwal, Liang Cheng, D. Pavlick, E. Sokol, Ryon P Graf, Douglas I. Lin, , A. Schrock, J. Quintanilha, Gerald Li, J. Jacob, A. Basnet, G. Bratslavsky
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="In eukaryotes, mismatch repair begins with MutS homolog (MSH) complexes, which scan newly replicated DNA for mismatches. Upon mismatch detection, MSH complexes recruit the PCNA- stimulated endonuclease Mlh1-Pms1/PMS2 (yeast/human), which nicks the DNA to allow downstream proteins to remove the mismatch. Past work has shown that although Mlh1-Pms1 is an ATPase and this activity is important in vivo, ATP is not required to nick DNA. Our data, using yeast as a model, suggests that Mlh1-Pms1 forms oligomeric complexes that drive DNA conformational rearrangements using the protein’s ATPase activity. Experiments with non-B-form DNA structures, common in microsatellite regions, show that these structures inhibit Mlh1-Pms1’s activities, likely through impeding Mlh1-Pms1-dependent DNA conformational changes. This could explain an additional mode for instability in these regions of the genome. These findings highlight the importance of DNA compaction and topological rearrangements in Mlh1-Pms1’s function and provide insight into how mismatch repair relies on DNA structure to coordinate events. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7d6132318c01588b39e51a01fee2c96e324d60" target='_blank'>
              The mismatch repair factor Mlh1-Pms1 uses ATP to compact and remodel DNA
              </a>
            </td>
          <td>
            Bryce W Collingwood, Amruta N. Bhalkar, Carol M. Manhart
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Non-coding RNA transcribed from active enhancers, known as enhancer RNA (eRNA), is a critical element in gene regulation with a highly specific expression pattern in the regulatory networks of tumor-infiltrating cells. Therefore, eRNA signatures could potentially be applied to represent anti-tumor immune cells and to improve cancer immunotherapy. Herein, we identified thousands of eRNAs that were significantly correlated with infiltrating immune cell abundance in more than 10,000 patient samples across a variety of cancer types. The expression of these eRNAs was mediated by transcription factors with high expression in anti-tumor immune cells, as identified through single-cell assays. An eRNA immunotherapy signature (eRIS) developed using the anti-tumor eRNAs was highly associated with the objective response rate (ORR) of immunotherapy and was elevated in patients who benefited from immune checkpoint blockade (ICB) treatment. In comparison with a signature based on protein-coding genes, the eRIS was more effective in predicting the response to immunotherapy. Integration of the eRIS with pharmacogenomic data revealed hundreds of anti-cancer drugs that have the potential to enhance immunotherapy efficacy. Finally, treatment of a mouse model of IDH mutant glioma with the histone deacetylase inhibitor vorinostat improved the effects of anti-PD-1 immunotherapy through increased abundance of infiltrating immune cells. Taken together, this study developed an eRIS with demonstrated efficacy in predicting immunotherapy response and used the eRIS to identify a series of effective combination drugs, thus highlighting the clinical utility of the eRIS in immunotherapy enhancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb404431d7b2aee513f3b1e9892ee993135259d" target='_blank'>
              Characterization of an Enhancer RNA Signature Reveals Treatment Strategies for Improving Immunotherapy Efficacy in Cancer.
              </a>
            </td>
          <td>
            Chenyang Zhang, Yan-Yan Chen, Shuyu Chen, Yunzhe Wang, Yifan Yuan, Xiwen Yang, Wei Hu, Bo Chen, Zengxin Qi, Jason T Huse, Yun Liu, Bo Wen, Xiuping Liu, Leng Han, Yuxiang Wang, Zhao Zhang
          </td>
          <td>2025-02-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose
Molecular characteristics of synchronous colorectal cancers (SCRCs) remain incompletely elucidated, despite their importance in targeted therapy selection. We compared the molecular characteristics and somatic mutations between SCRCs.


Materials and Methods
This retrospective study (2012-2014) included 98 consecutive patients with surgically resected SCRCs. Molecular characteristics, including microsatellite instability (MSI) and tumor-infiltrating lymphocytes (TILs), were analyzed for all cancer lesions. The intertumoral heterogeneity of SCRCs was evaluated using whole-exome sequencing (WES) for 18 cancers from 9 patients with at least one MSI-high (MSI-H) tumor.


Results
Twelve patients had at least one MSI-H tumor; five showed discordant MSI status. Mucinous adenocarcinoma frequency and TIL density were higher in patients with at least one MSI-H tumor than in those with only microsatellite-stable tumors. WES revealed that, except one patient (6.5%), most synchronous cancers shared few variants in each patient (0.09-0.36%). The concordance rates for BRAF, KRAS, NRAS, and PIK3CA in synchronous cancers from each patient were 66.7%, 66.7%, 66.7%, and 55.6%, respectively.


Conclusion
Although synchronous cancers shared a mutated gene, the mutation subtypes differed. SCRCs exhibited 5.1% MSI status discordance rate and a high discordance rate in somatic mutational variants. As intertumoral heterogeneity may affect the targeted therapy response, molecular analysis of all tumors is recommended for patients with SCRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d64426c5ce00acb3fa51471d3649240227f5078" target='_blank'>
              Molecular Mosaics: Unveiling Heterogeneity in Synchronous Colorectal Cancers.
              </a>
            </td>
          <td>
            H. Lee, Yeseul Kim, Mi-Ju Kim, Yeon Wook Kim, Sun-Young Jun, Deokhoon Kim, In Ja Park, Seung-Mo Hong
          </td>
          <td>2025-02-18</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67046daa5df3cebbea573edba163f96ed0a30fda" target='_blank'>
              Visualization of chromosomal reorganization induced by heterologous fusions in the mammalian nucleus
              </a>
            </td>
          <td>
            Meng Yan, X. Zhang, Zhenhua Yang, Miao Jia, Rongyu Liao, Jinsong Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Synthetic lethality approaches in BRCA1/2-mutated cancers have focused on poly(ADP-ribose) polymerase (PARP) inhibitors, which are subject to high rates of innate or acquired resistance in patients. Here, we used CRISPR/Cas9-based screening to identify DNA Ligase I (LIG1) as a novel target for synthetic lethality in BRCA1-mutated cancers. Publicly available data supported LIG1 hyperdependence of BRCA1-mutant cells across a variety of breast and ovarian cancer cell lines. We used CRISPRn, CRISPRi, RNAi, and protein degradation to confirm the lethal effect of LIG1 inactivation at the DNA, RNA, and protein level in BRCA1-mutant cells in vitro. LIG1 inactivation resulted in viability loss across multiple BRCA1-mutated cell lines, whereas no effect was observed in BRCA1/2 wild-type cell lines, demonstrating target selectivity for the BRCA1-mutant context. On-target nature of the phenotype was demonstrated through rescue of viability with exogenous wild-type LIG1 cDNA. Next, we demonstrated a concentration-dependent relationship of LIG1 protein expression and BRCA1 mutant cell viability using a titratable, degradable LIG1 fusion protein. BRCA1 mutant viability required LIG1 catalytic activity, as catalytically dead mutant LIG1K568A failed to rescue viability loss caused by endogenous LIG1 depletion. LIG1 perturbation produced proportional increases in PAR staining in BRCA1 mutant cells, indicating a mechanism consistent with the function of LIG1 in sealing ssDNA nicks. Finally, we confirmed LIG1 hyperdependence in vivo using a xenograft model in which LIG1 loss resulted in tumor stasis in all mice. Our cumulative findings demonstrate that LIG1 is a promising synthetic lethal target for development in patients with BRCA1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831e285c6dc25f79bcb13593b00dacbe2a024fd9" target='_blank'>
              LIG1 is a synthetic lethal target in BRCA1 mutant cancers.
              </a>
            </td>
          <td>
            Lauren Martires, Leanne G Ahronian, Charlotte B Pratt, Nikitha M Das, Xiaobin Zhang, D. Whittington, Hongxiang Zhang, B. Shen, Jon Come, Patrick McCarren, Mu-Sen Liu, Chengyin Min, Tianshu Feng, Haris Jahic, Janid A Ali, Daniel R Aird, Fang Li, Jannik N Andersen, A. Huang, William D Mallender, Hilary E. Nicholson
          </td>
          <td>2025-01-27</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c103a4b54230f99fc3c742a3396a3568699c7f8" target='_blank'>
              H3K56 acetylation regulates chromatin maturation following DNA replication
              </a>
            </td>
          <td>
            Shoufu Duan, I. Nodelman, Hui Zhou, T. Tsukiyama, G. Bowman, Zhiguo Zhang
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Structural variation (SV) typically refers to alterations in DNA fragments at least 50 base pairs long in the human genome. It can alter thousands of DNA nucleotides and thus significantly influence human health, disease, and clinical phenotypes. There is a shared and growing recognition that the emergence of effective computational tools and high-throughput technologies such as short-read sequencing and long-read sequencing offers novel insight into SV and, by extension, diseases affecting planetary health. However, numerous available SV tools exist with varying strengths and weaknesses. This is currently hampering the abilities of scholars to select the optimal tools to study SVs. Here, we reviewed 175 tools developed in the past two decades for SV detection, annotation, visualization, and downstream analysis of human genomics. In this expert review, we provide a comprehensive catalog of SV-related tools across different technology platforms and summarize their features, strengths, and limitations with an eye to accelerate systems science and planetary health innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7c38ace80c2689d1a82e07697646da72182a99" target='_blank'>
              Computational Tools for Studying Genome Structural Variation.
              </a>
            </td>
          <td>
            Xingyu Chen, Siyu Wei, Chen Sun, Zelin Yi, Zihan Wang, Yingyi Wu, Jing Xu, Junxian Tao, Haiyan Chen, Mingming Zhang, Yongshuai Jiang, Hongchao Lv, Chen Huang
          </td>
          <td>2025-02-05</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-14</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 RNaseH2 plays a critical role in removing mis-bounded RNA from the DNA double strand helix1. Absence of RNaseH2 results in spontaneous double-stranded DNA breaks and cytosolic release of dsDNA2. We have previously shown that epithelial deficiency of RNaseH2b (H2bΔIEC) leads to a p53-dependent stem cell suppression and additional deficiency of p53 results in spontaneous intestinal carcinogenesis3. Due to the central role of cGAS/STING in sensing dsDNA, we aim to investigate the role of epithelial STING in a mouse model of chronic epithelial DNA damage and tumor prevention in this study.



 DNA damage was assessed using comet assay in ModeK cell line with cGAS and STING knock out. Protein kinase activity were assessed by Pamgene Kinase Assay. Intestinal organoids were generated from H2bΔIEC and H2bΔIEC/Tmem173-/-mice. Colony forming assay and EdU staining were performed in all genotypes to assess the impact of STING on epithelial stem cell function and proliferation were assessed by CellTiter Glo assay.
 RNA sequencing was performed in intestinal organoids. Spontaneous carcinogenesis was assessed in 1-year aged mice.



 Loss of RNaseH2b results in activation of cGAS/STING pathway and subsequent upregulation of IFN-I signature. Deficiency of cGAS/STING lead to impaired IFN-I independent DNA damage repair which were accessed by WB and comet assay. During the DNA damage response, STING but not cGAS mediates the P53 pathway and the P21-CDK axis. Organoid EdU stanning displayed restored proliferation in H2bΔIEC/Tmem173-/- compared to H2bΔIEC. RNA sequencing from H2bΔIEC/Tmem173-/-organoid further confirmed decreased G2/M DNA damage checkpoint and P53 signature compared to H2bΔIEC. In vivo, we show that H2bΔIEC/Tmem173-/- mice have downregulated DNA damage repair and developed spontaneous small bowel adenocarcinoma after 53 weeks. Tumor derived from H2bΔIEC/Tmem173-/- mice show sensitivity to CDK inhibitor consistent with in vitro validation.



 We provide evidence that non-canonical role of epithelial STING mediates DNA damage response and guides P21-CDK1 axis in preventing tumor genesis. Our research provided evidence understand the role of STING in DNA damage response and intestine carcinogenesis.



 1.Reijns, Martin A.M., et al., Enzymatic Removal of Ribonucleotides from DNA Is Essential for Mammalian Genome Integrity and Development. Cell, 2012. 149(5): p. 1008-1022.
 2.Bartsch, K., et al., Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Hum Mol Genet, 2017. 26(20): p. 3960-3972.
 3.Aden, K., et al., Epithelial RNase H2 Maintains Genome Integrity and Prevents Intestinal Tumorigenesis in Mice. Gastroenterology, 2019. 156(1): p. 145-159 e19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0539a26d1a5ba9dc2728ef562232f66e1b1c39" target='_blank'>
              P0212 Non-canonical STING signaling in intestinal epithelial cells guides DNA damage-driven stem cell function and protects from intestinal carcinogenesis
              </a>
            </td>
          <td>
            H. Xiang, C. Woitaske-Proske, J. Bernardes, G. R. Martini, F. Wottawa, J. Bornhaeuser, N. Kakavand, Y. Guang, Q. Wu, J. Kugler, L. Welz, B. Konukiewitz, C. Peifer, S. Schreiber, M. Tschurtschenthaler, P. Rosenstiel, K. Aden
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract In eukaryotes, chromatin is compacted within nuclei under the principle of compartmentalization. On top of that, condensin II establishes eukaryotic chromosome territories, while cohesin organizes the vertebrate genome by extruding chromatin loops and forming topologically associating domains (TADs). Thus far, the formation and roles of these chromatin structures in plants remain poorly understood. This study integrates Hi‐C data from diverse plant species, demonstrating that nuclear DNA content influences large‐scale chromosome conformation and affects the finer details of compartmental patterns. These contrasting compartmental patterns are distinguished by gene‐to‐gene loops and validated through cytological observations. Additionally, a novel chromatin domain type associated with tandem duplicate gene clusters is identified. These domains are independent of H3K27me3‐mediated chromatin compartmentalization and exhibit evolutionary conservation across species. Gene pairs within TAD‐like domains are younger and show higher levels of coexpression. These domains potentially promote the formation of tandem duplicates, a property appears unique to the Actinidia family. Overall, this study reveals functional chromatin domains in plants and provides evidence for the role of three‐dimensional chromatin architecture in gene regulation and genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1f811c2a347e9ec96c4d8e9fe6d632b6cffed1" target='_blank'>
              Chromatin Topological Domains Associate With the Rapid Formation of Tandem Duplicates in Plants
              </a>
            </td>
          <td>
            Ni Ma, Xiaopeng Li, Dong Ci, Haiyue Zeng, Congxiao Zhang, Xiaodong Xie, Caihong Zhong, Xing Wang Deng, Dawei Li, Hang He
          </td>
          <td>2024-12-27</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Background Glioma stem cells (GSCs) contribute to the initiation, recurrence, metastasis, and drug resistance of glioblastoma multiforme (GBM). Long noncoding RNAs (lncRNAs) are critical modulators in the development and progression of GBM; however, specific lncRNAs related to GSCs remain largely unexplored. This study aims to identify dysregulated lncRNAs in GSCs, unravel their contributions to GBM progression, and propose new targets for diagnosis and treatment. Methods GeneChip analysis was utilized to identify lncRNAs in GSCs. The expression of RNAs was examined using quantitative real‐time PCR. Cell Counting Kit‐8, tmorsphere formation assay, limiting dilution assay, apoptosis detection and intracranial xenograft models were performed to assess the stemness and radioresistance of GSCs. Transcriptomics analysis, RNA immunoprecipitation and dual‐luciferase experiments were conducted for mechanistic studies. Results NONHSAT141192.2 exhibited elevated expression levels in aggressive GBM tissues compared to lower‐grade gliomas. Silencing NONHSAT141192.2 resulted in a considerable decrease in GSC proliferation, tumor sphere formation, self‐renewal and the expression of key stem cell markers. Furthermore, depletion of NONHSAT141192.2 enhanced GSC sensitivity to radiation, indicated by diminished viability and tumorsphere formation, increased cell apoptosis, and decreased tumor growth in intracranial xenograft models. Mechanistically, NONHSAT141192.2 upregulates the expression of SOX2 and PIK3R3 by sponging miR‐4279, influencing GSC characteristics and their resistance to radiation. Conclusion The study highlights a significant relationship between NONHSAT141192.2, GSC stemness, and radioresistance, emphasizing its potential as a therapeutic target for GBM treatment and radiosensitization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a0dff6fa5081c93a6854a7de8ed920196452d" target='_blank'>
              NONHSAT141192.2 Facilitates the Stemness and Radioresistance of Glioma Stem Cells via the Regulation of PIK3R3 and SOX2
              </a>
            </td>
          <td>
            Sihan Wang, Haolang Ming, Zhen Wang, Xingye Zhai, Xinyue Zhang, Di Wu, Yin Bo, Hang Wang, Yuanbo Luo, Zhenfeng Han, Lingyu Hao, Yijia Xiang, Xu Han, Zengguang Wang, Yi Wang
          </td>
          <td>2025-02-01</td>
          <td>CNS Neuroscience & Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The target theory is the most classical hypothesis explaining radiation-induced cell death, yet the physical or biological nature of the target remains ambiguous. This study hypothesizes that the distribution of DNA double-strand breaks (DSBs) within the 3D genome is a pivotal factor affecting the probability of cell death. We propose that clustered DSBs in DNA segments with high interaction frequencies are more susceptible to leading to cell death than isolated DSBs. Topologically associating domain (TAD) can be regarded as the reference unit for evaluating the impact of DSB clustering in the 3D genome. To quantify this correlation between the DSB distribution in 3D genome and radiation-induced effect, we developed a simplified model taking into account the DSB distribution across TADs. Utilizing track-structure Monte Carlo codes to simulate the electron and carbon ion irradiation, we calculated the incidence of each DSB case across a variety of radiation doses and linear energy transfers (LETs). Our simulation results indicate that DSBs in TADs with frequent interactions (case 3) are more likely to induce cell death than clustered DSBs within a single TAD (case 2). Moreover, case 2 is more likely to induce cell death than isolated DSBs (case 1). The curves of the incidence of case 2 and case 3 vs LETs have a similar shape to the radiation quality factor used in radiation protection. This indicates that these two cases are also associated with the stochastic effects of high LET irradiation. Our study underscores the crucial significance of the 3D genome structure in the fundamental mechanisms of radiobiological effects. The hypothesis proposed in our research offers novel perspectives on the mechanisms that regulate radiobiological effects. Moreover, it serves as a valuable reference for the establishment of mechanistic models that can predict cell survival under different LETs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6def23c77c6de48e6f1d51e1e9fb8ac9032862" target='_blank'>
              Correlation Between DNA Double-Strand Break Distribution in 3D Genome and Radiation-Induced Cell Death
              </a>
            </td>
          <td>
            A. Hu, Wanyi Zhou, Xiyu Luo, Rui Qiu, Junli Li
          </td>
          <td>2024-12-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In the M-phase, the nuclear membrane is broken down, nucleosomes are condensed as mitotic chromosomes, and transcription factors are generally known to be dislocated from their recognition sequences and dispersed to the cytoplasm. However, some transcription factors have recently been reported to remain on mitotic chromosomes and facilitate the rapid re-activation of the target genes in early G1-phase. Paired-like homeobox 2B (PHOX2B) is a transcription factor exhibiting chromosomal localization during M-phase. PHOX2B mutations are associated with congenital central hypoventilation syndrome, Hirschsprung disease, and neuroblastoma. In this study, we investigated PHOX2B chromosomal localization during M-phase through immunostaining and fluorescence recovery after photobleaching analysis to determine whether the chromosomal localization of disease-associated PHOX2B mutants is altered during M-phase. Missense mutations in the homeodomain and the frameshift mutation in the C-terminal domain disrupted the chromosomal localization of PHOX2B in M-phase, leading to its dispersion in the cell. Furthermore, a PHOX2B mutant with polyalanine expansion showed a line-shaped localization to the restricted region of mitotic chromosomes. Our findings suggest an association between the disease-associated mutations and defective chromosomal localization of transcription factors during M-phase. Further investigations of PHOX2B chromosomal localization during M-phase could reveal pathogenic mechanisms of such diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc280a650e26ad4fbfbcabfa8901f734f1fa074d" target='_blank'>
              Chromosomal localization of PHOX2B during M-phase is disrupted in disease-associated mutants.
              </a>
            </td>
          <td>
            Yuki Sato, Shin-ichiro Hayashi, Souichi Oe, T. Koike, Y. Nakano, Ryohei Seki-Omura, Hikaru Iwashita, Y. Hirahara, Masaaki Kitada
          </td>
          <td>2025-02-11</td>
          <td>Development, growth & differentiation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a819a83329e7d0a414c5918852d0411278218a5" target='_blank'>
              High CDC20 levels increase sensitivity of cancer cells to MPS1 inhibitors.
              </a>
            </td>
          <td>
            Siqi Zheng, Linoy Raz, Lin Zhou, Yael Cohen-Sharir, Ruifang Tian, M. Ippolito, Sara Gianotti, Ron Saad, René Wardenaar, Mathilde Broekhuis, Maria F Suarez Peredo Rodriguez, Soraya Wobben, A. van den Brink, Petra Bakker, S. Santaguida, F. Foijer, Uri Ben-David
          </td>
          <td>2025-01-21</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cytosine (DNA) methylation plays important roles in silencing transposable elements, plant development, genomic imprinting, stress responses, and maintenance of genome stability. To better understand the functions of this epigenetic modification, several tools have been developed to manipulate DNA methylation levels. These include mutants of DNA methylation writers and readers, targeted manipulation of locus-specific methylation, and the use of chemical inhibitors. Here, we summarize the effects of commonly used cytidine analog chemical inhibitors represented by zebularine, 5-azacytidine, and their related compounds on plants. These analogs are incorporated into the chromosomal DNA, where they block the activity of the replicative CG DNA methyltransferase 1 (MET1). This leads to manifold alterations in plant epigenome, modified developmental programs, or suppression of hybridization barriers. We also highlight the DNA-damaging effects of cytidine analogs, particularly the formation of stable DNA-protein crosslinks between DNA and MET1. This sheds new light on specific phenotypes observed upon cytidine analog treatments. In conclusion, cytidine analogs remain a vital tool for plant genome research and have the potential to open new promising avenues for applications in plant biotechnology and breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347eb5947bdb94f55760bce2a406c75b6df28aaf" target='_blank'>
              Cytidine analogs in plant epigenetic research and beyond.
              </a>
            </td>
          <td>
            Eva Dvořák Tomaštíková, A. Pečinka
          </td>
          <td>2024-12-28</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c43b2e6ebcea9dd9d569a152511cd9284977827" target='_blank'>
              Establishment and characterization of ovarian clear cell carcinoma patient-derived xenografts
              </a>
            </td>
          <td>
            Joseph J. Caumanns, Shang Li, G. Meersma, E. Duiker, A. van der Zee, G. Wisman, Steven de Jong
          </td>
          <td>2025-02-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Z-DNA is an alternative left-handed helical form of DNA with a zigzag-shaped backbone that differs from the right-handed canonical B-DNA helix. Z-DNA has been implicated in various biological processes, including transcription, replication, and DNA repair, and can induce genetic instability. Repetitive sequences of alternating purines and pyrimidines have the potential to adopt Z-DNA structures. ZSeeker is a novel computational tool developed for the accurate detection of potential Z-DNA-forming sequences in genomes, addressing limitations of prior methods. By introducing a novel methodology informed and validated by experimental data, ZSeeker enables the refined detection of potential Z-DNA-forming sequences. Built both as a standalone Python package and as an accessible web interface, ZSeeker allows users to input genomic sequences, adjust detection parameters, and view potential Z-DNA sequence distributions and Z-scores via downloadable visualizations. Our Web Platform provides a no-code solution for Z-DNA identification, with a focus on accessibility, user-friendliness, speed and customizability. By providing efficient, high-throughput analysis and enhanced detection accuracy, ZSeeker has the potential to support significant advancements in understanding the roles of Z-DNA in normal cellular functions, genetic instability, and its implications in human diseases. Availability ZSeeker is released as a Python package under the GPL license as a multi-platform application and is available at: https://github.com/Georgakopoulos-Soares-lab/ZSeeker. A web-interface of ZSeeker is publicly available at https://zseeker.netlify.app/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dacb297300a8904d26cbd58791bfe930f3f58ff0" target='_blank'>
              ZSeeker: An optimized algorithm for Z-DNA detection in genomic sequences
              </a>
            </td>
          <td>
            Guliang Wang, I. Mouratidis, Kimonas Provatas, Nikol Chantzi, Michail Patsakis, Ilias Georgakopoulos-Soares, Karen M. Vasquez
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Graphical abstract The chromosomal status of Day 5 blastocysts was determined based on trophectoderm biopsy for all 24 chromosomes by array comparative genomic hybridization (array-CGH). The blastocoel fluid (BF) was extruded and amplified, and DNA concentration was read by a Qubit fluorometer. Apoptotic gene expression (TNFRSF10B, CASP2, BAX and CASP3) was analyzed by real-time quantitative PCR (RT-qPCR). Our findings suggest BF-DNA may be released in the blastocoel cavity of aneuploidy blastocysts under the influence of the apoptosis mechanism, and it appears that quantifying BF-DNA has the potential to aid in the selection of viable embryos. Abstract It is believed that aneuploid embryos release cell-free DNA (cfDNA) into the blastocyst cavity during the self-correction process through the apoptotic mechanism. This study aimed to develop less invasive methods for predicting ploidy status by investigating how ploidy status affects blastocoel fluid DNA (BF-DNA) levels and apoptotic gene expression as indicators of embryo viability. Human blastocysts were classified into three groups; survivable embryo (SE), fatal single and double aneuploidy (FSDA) and multiple aneuploidy (MA) using array comparative genomic hybridization (array-CGH) by trophectoderm biopsy. Following BF aspiration and whole genome amplification, BF-DNA level was quantified. Apoptotic activity was assessed by measuring the genes TNFRSF10B, CASP2, BAX and CASP3 using real-time quantitative PCR. Day-5 intracytoplasmic sperm injection blastocysts were scored according to the Gardner and Schoolcraft system. BF-DNA levels were significantly higher in the MA vs SE group (P = 0.01), while these were not statistically significant differences between the MA and FSDA groups or between the FSDA and SE groups, P = 0.17 and P = 0.38, respectively. TNFRSF10B, CASP2 and CASP3 were overexpressed in the MA and FSDA groups compared to SE, while BAX was downregulated. We found a significant correlation between the amount of BF-DNA and apoptosis marker genes. No significant correlation was found between embryo morphology score and BF-DNA level, BF volume or apoptosis marker gene expression levels. We observed that the correlation between apoptotic activity and BF-DNA levels is influenced by the embryo’s ploidy status. These findings suggest that BF-DNA level evaluation can be applicable in selecting viable embryos for transfer. Lay summary Preimplantation genetic testing helps doctors choose healthy human embryos for transfer during fertility treatments, but it can be expensive, invasive and time-consuming. Recently, scientists have found a less invasive way to study embryos by looking at DNA in a fluid inside the embryo’s cavity. This fluid may give us clues about how embryos try to fix problems with their chromosomes through a natural process of cell death. Our study shows that the amount of DNA in this fluid and the activity of some genes are influenced by chromosomal problems in the embryo. We also discovered that just looking at an embryo under a microscope is not enough to evaluate its genetic health. Our findings suggest that measuring the DNA in this fluid may be a promising approach for picking the best embryos for transfer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ea8bae39e64106e3ac6f98389c15425d00eec3c" target='_blank'>
              Correlation among blastocoel fluid DNA level, apoptotic genes expression and preimplantation aneuploidy
              </a>
            </td>
          <td>
            F. Khajehoseini, Zahra Noormohammadi, P. Eftekhari-Yazdi, H. Gourabi, R. Pazhoomand, S. Hosseinishenatal, M. Bazrgar
          </td>
          <td>2025-01-01</td>
          <td>Reproduction & Fertility</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In the last years, it has been proved that some viruses are able to re-structure chromatin organization and alter the epigenomic landscape of the host genome. In addition, they are able to affect the physical mechanisms shaping chromatin 3D structure, with a consequent impact on gene activity. Here, we investigate with polymer physics genome re-organization of the host genome upon SARS-CoV-2 viral infection and how it can impact structural variability within the population of single-cell chromatin configurations. Using published Hi-C data and molecular dynamics simulations, we build ensembles of 3D configurations representing single-cell chromatin conformations in control and SARS-CoV-2 infected conditions. We focus on genomic length scales of TADs and consider, as a case study, models of real loci containing DDX58 and IL6 genes, belonging, respectively, to the antiviral interferon response and pro-inflammatory genes. Clustering analysis applied to the ensemble of polymer configurations reveals a generally increased variability and a more heterogeneous population of 3D structures in infected conditions. This points toward a scenario in which viral infection leads to a loss of chromatin structural specificity with, likely, a consequent impact on the correct regulation of host cell genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c633c3cbdf5554cee929ae69b09bc4c4042bdce3" target='_blank'>
              Loss of structural specificity in 3D genome organization upon viral infection is predicted by polymer physics.
              </a>
            </td>
          <td>
            Andrea Fontana, Simona Bianco, Fabrizio Tafuri, Andrea Esposito, Alex Abraham, Mattia Conte, Francesca Vercellone, Florinda Di Pierno, Sumanta Kundu, Sougata Guha, Ciro Di Carluccio, A. Prisco, Mario Nicodemi, A. Chiariello
          </td>
          <td>2025-01-02</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Circular RNA (CircRNA)s, a newly discovered type of noncoding RNAs, have been found to play a role in controlling the development and aggressiveness of tumors. Abnormal control of circRNA has been observed in various types of human cancers, including bladder cancer, hepatocellular carcinoma (HCC), breast cancer, and gastric cancer (GC). CircRNAs possess binding sites for microRNAs (miRNAs) and function as miRNA sponges in posttranscriptional regulation. This mechanism has been documented to influence the course of cancer. Significantly, among these putative circRNAs, circular RNA protein tyrosine kinase 2 (circPTK2) exhibited increased expression and displayed a substantial association with adverse clinical characteristics and a negative prognosis. The production of these transcripts occurs via a back-splicing mechanism. The enclosed conformation of circRNAs shields them from destruction and enhances their potential as biomarkers. Gaining insight into the molecular mechanisms involved in these processes would aid in the development of treatment approaches and the discovery of new tumor markers. This article provides a comprehensive assessment of the latest research on the biosynthesis and features of circRNAs. It examines the role of circPTK2 in the diagnosis, treatment, and prognosis evaluation of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd1e91ef7563824a8ebebdc925d90d66cc38471" target='_blank'>
              CircPTK2 as a Valuable Biomarker and Treatment Target in Cancer.
              </a>
            </td>
          <td>
            Chengqiu Yan, Yu Du, Lihong Cui, Han Bao, Hui Li
          </td>
          <td>2025-01-30</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba51e0ae284150abf200e92c2ae4133d35b65e6d" target='_blank'>
              Heterozygous mutation in BRCA2 induces accelerated age-dependent decline in sperm quality with male subfertility in rats
              </a>
            </td>
          <td>
            Yashiro Motooka, Hideaki Tanaka, Yu-ichi Maeda, Misako Katabuchi, Tomoji Mashimo, Shinya Toyokuni
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background/Objectives: Genetics and epigenetics play an important role in the pathogenesis of cutaneous melanoma. The majority of cases harbor mutations in genes associated with the MAPK signaling pathway, i.e., BRAF, NRAS, or NF1. The remaining neoplasms, often located on acral sites, are condensed as the triple-wildtype subtype and are characterized by other molecular drivers. This study aimed to elucidate genetic and epigenetic differences within cutaneous melanoma and to compare it with melanocytic nevi. Methods: DNA was extracted from archived tissue samples of cutaneous melanoma (n = 19), melanocytic nevi (n = 11), and skin controls (n = 11) and subsequently analyzed by massive parallel (next generation) gene panel sequencing and genome-wide DNA methylation array analysis. The sample size was increased by including repository data from an external study. Results: There were major differences in the genomic landscape of MAPK-altered and triple-wildtype cutaneous melanoma, the latter presenting with a lower number of mutations, a different pattern of copy number variants, and a low frequency of TERT promoter mutations. Dimensional reduction of DNA methylation array analysis clearly separated cutaneous melanoma from melanocytic nevi but revealed no major differences between classical cutaneous melanoma and the triple-wildtype cases. However, it identified a possible biological subgroup characterized by intermediately methylated CpGs. Conclusions: Dimensional reduction of methylation array data is a useful tool for the analysis of melanocytic tumors to differentiate between malignant and benign lesions and may be able to identify biologically distinct subtypes of cutaneous melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd4730aeef318378ac4a266dcf9f011ba2c44628" target='_blank'>
              DNA Methylation Array Analysis Identifies Biological Subgroups of Cutaneous Melanoma and Reveals Extensive Differences with Benign Melanocytic Nevi
              </a>
            </td>
          <td>
            Simon Schwendinger, Wolfram Jaschke, Theresa Walder, J. Hench, Verena Vogi, Stephan Frank, Per Hoffmann, S. Herms, Johannes Zschocke, Van Anh Nguyen, Matthias Schmuth, E. Jukic
          </td>
          <td>2025-02-21</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Ribosome biogenesis is one of the most fundamental and energetically demanding cellular processes. In humans, the ribosomal DNA (rDNA) repeats span a large region of DNA and comprise 200 to 600 copies of a ~43 kb unit spread over five different chromosomes. Control over ribosome biogenesis is closely tied to the regulation of the chromatin environment of this large genomic region. The proportion of rDNA loci which are active or silent is altered depending on the proliferative or metabolic state of the cell. Repeat silencing is driven by epigenetic changes culminating in a repressive heterochromatin environment. One group of proteins facilitating these epigenetic changes in response to growth or metabolic demands are ATP-dependent chromatin remodeling protein complexes that use ATP hydrolysis to reposition nucleosomes. Indeed, some chromatin remodelers are known to have indispensable roles in regulating the chromatin environment of rDNA. In this review, we highlight these proteins and their complexes and describe their mechanistic roles at rDNA. We also introduce the developmental disorders arising from the dysfunction of these proteins and discuss how the consequent dysregulation of rDNA loci may be reflected in the phenotypes observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b44d8b12e96514a790a07032e2485fc056c2a29" target='_blank'>
              It Takes a Village of Chromatin Remodelers to Regulate rDNA Expression
              </a>
            </td>
          <td>
            Mathieu G. Levesque, D. Picketts
          </td>
          <td>2025-02-19</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa0f09862c96ff9cec51a61499fdcc67366281e" target='_blank'>
              Analysis of microarray and single-cell RNA-seq identifies gene co-expression, cell–cell communication, and tumor environment associated with metabolite interconversion enzyme in prostate cancer
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Ali Shakeri Abroudi, Nadia Forghani, Sobhan Bavandi, Melika Djamali, Hamoon Baghaei, Sana Shafaeitilaki, Ehsan HasanZadeh
          </td>
          <td>2025-02-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f50dea3e9f7300d7800974584eb194f23dc33f9" target='_blank'>
              Whole-genome bisulfite sequencing of cell-free DNA unveils age-dependent and ALS-associated methylation alterations
              </a>
            </td>
          <td>
            Yulin Jin, Karen N Conneely, Wenjing Ma, Robert K Naviaux, Teepu Siddique, Emily G. Allen, Sandra Guingrich, Robert M Pascuzzi, Peng Jin
          </td>
          <td>2025-02-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Mitochondrial DNA (mtDNA) mutagenesis remains poorly understood despite its crucial role in disease, aging, and evolutionary tracing. In this study, we reconstructed a comprehensive 192-component mtDNA mutational spectrum for chordates by analyzing 118,397 synonymous mutations in the CytB gene across 1,697 species and five classes. This analysis revealed three primary forces shaping mtDNA mutagenesis: (i) symmetrical, replication-driven errors by mitochondrial polymerase (POLG), resulting in C > T and A > G mutations that are highly conserved across classes; (ii) asymmetrical, damage-driven C > T mutations on the single-stranded heavy strand with clock-like dynamics; and (iii) asymmetrical A > G mutations on the heavy strand, with dynamics suggesting sensitivity to oxidative damage. The third component, sensitive to oxidative damage, positions mtDNA mutagenesis as a promising marker for metabolic and physiological processes across various classes, species, organisms, tissues, and cells. The deconvolution of the mutational spectra into mutational signatures uncovered deficiencies in both base excision repair (BER) and mismatch repair (MMR) pathways. Further analysis of mutation hotspots, abasic sites, and mutational asymmetries underscores the critical role of single-stranded DNA damage (components ii and iii), which, uncorrected due to BER and MMR deficiencies, contributes roughly as many mutations as POLG-induced errors (component i).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4269bc12502a52f2f9e09037077d3dcfaff9d7df" target='_blank'>
              Deciphering the Foundations of Mitochondrial Mutational Spectra: Replication-Driven and Damage-Induced Signatures Across Chordate Classes
              </a>
            </td>
          <td>
            D. Iliushchenko, B. Efimenko, A. G. Mikhailova, Victor Shamanskiy, Murat K. Saparbaev, B. Matkarimov, I. Mazunin, Alexander Voronka, Dmitry A Knorre, Wolfram S. Kunz, Philipp Kapranov, S. Denisov, J. Fellay, Konstantin Khrapko, K. Gunbin, K. Popadin
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Noncoding miRNAs tend to evolve within introns of coding genes who provide them with transcriptional opportunity. As an outcome of natural selection, the intragenic position of miRNAs is crucial for their expression, evolution and functional cooperation with the host gene. Therefore, understanding the evolution of intragenic miRNA structures may bring novel insights into genetic and phenotypic evolution. However, it remains largely unexplored. Here, by analysing miRNA genomics in 34 metazoan species, we found that the majority (630/1154) of miRNA families originated from introns of coding genes which provided them with initial transcriptional capacity. The most rapid expansion of intragenic miRNAs happened at the advent of vertebrates when 21 miRNA families emerged from introns of neural genes and reorganized the gene regulatory network, leading to the rise of vertebrate-specific neural crest (NC) cells, which transformed the invertebrate head and enabled the ecological shift from filter feeding to active predation. Intragenic miRNAs gradually gain independence from their host genes, which is accelerated by whole genome duplications (WGDs). After a WGD, the purging of redundant host genes often set an orphaned miRNA "free" to diversify with the transcriptional elements inherited from the host. WGDs facilitated a dramatic miRNA diversification during the initial divergence of vertebrates, as the intragenic status of twelve NC-regulating miRNAs was retained in jawed vertebrates but was lost in jawless cyclostomes, which diverged their neural crest development. We propose that coding genes not only facilitate the origination of new miRNAs, but also "sacrifice" themselves to help miRNA diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec14a5673e5811f8ba88226799b41e9414365cb1" target='_blank'>
              Coding genes helped the origination and diversification of intragenic miRNAs.
              </a>
            </td>
          <td>
            Chengzhang Liu, Xiaojun Zhang, Jianbo Yuan, J. Xiang, Fuhua Li
          </td>
          <td>2025-02-11</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Viral mimicry refers to an active antiviral response triggered by the activation of endogenous retroviruses (ERVs), usually manifested by the formation of double-stranded RNA (dsRNA) and activation of the cellular interferon response, which activates the immune system and produces anti-tumor effects. Epigenetic studies have shown that epigenetic modifications (e.g. DNA methylation, histone modifications, etc.) play a crucial role in tumorigenesis, progression, and treatment resistance. Particularly, alterations in DNA methylation may be closely associated with the suppression of ERVs expression, and treatment by demethylation may restore ERVs activity and thus strengthen the tumor immune response. Therefore, we propose that viral mimicry can induce immune responses in the tumor microenvironment by activating the expression of ERVs, and that epigenetic alterations may play a key regulatory role in this process. In this paper, we review the intersection of viral mimicry, epigenetics and tumor immunotherapy, and explore the possible interactions and synergistic effects among the three, aiming to provide a new theoretical basis and potential strategies for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916b8820f91cde5f4b984b3eba25d2fd40a973b1" target='_blank'>
              Exploring viral mimicry combined with epigenetics and tumor immunity: new perspectives in cancer therapy
              </a>
            </td>
          <td>
            Ruirui Wang, Xin Dong, Xiongjian Zhang, Jinzhuang Liao, Wei Cui, Wei Li
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e63083f2ebe39f75bcbba803ff0b3dd9f4e01b8c" target='_blank'>
              The role of exosomal non-coding RNAs in the breast cancer tumor microenvironment.
              </a>
            </td>
          <td>
            Mohamed J Saadh, O. Q. B. Allela, Radhwan Abdul Kareem, S. Ballal, Mamata Chahar, Suman Saini, G. V. S. Prasad, Hayder Naji Sameer, Atheer Khdyair Hamad, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="INhibitor of Growth (ING1-5) proteins are epigenetic readers that target histone acetyltransferase (HAT) or histone deacetylase (HDAC) complexes to the H3K4Me3 mark of active transcription. ING5 targets Moz/Morf and HBO1 HAT complexes that alter acetylation of H3 and H4 core histones, affecting gene expression. Previous experiments in vitro indicated that ING5 functions to maintain stem cell character in normal and in cancer stem cells. Here we find that CRISPR/Cas9 ING5 knockout (KO) mice are sub-fertile but show no decrease in lifespan or ability to heal wounds despite indications of depleted stem cell pools in several tissues. ING5 KO mouse embryo fibroblasts accumulate in G2 of the cell cycle, have high levels of abnormal nuclei and show high basal levels of the γH2AX indicator of DNA damage. KO animals also develop severe dermatitis at a 5-fold higher rate that wild-type littermates. Consistent with ING5 serving a tumor suppressive role, ING5 KO mice developed germinal centre diffuse large B-cell lymphomas at a rate 6-fold higher than control mice at 18 months of age. These data suggest that ING5 functions in vivo to maintain stem cell character in multiple organs, that reduction of stem cell populations is not limiting for murine lifespan and that like a subset of other ING family members, ING5 functions as a tumor suppressor in hematopoietic cells in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7856dbb73e87aa51549dc70eedb5371d011208e6" target='_blank'>
              Knockout of the ING5 epigenetic regulator confirms roles in stem cell maintenance and tumor suppression in vivo
              </a>
            </td>
          <td>
            Buthaina Al Shueili, Arthur Dantas, Etienne Mahe, Tak-Ho Chu, Yang Yang, Elodie Labit, E. Kutluberk, Nicolas Lasaleta, Anand Masson, Hiba Omairi, Kenichi Ito, Roman J. Krawetz, R. Midha, Gregory Cairncross, Karl Riabowol
          </td>
          <td>2025-01-09</td>
          <td>PLOS ONE</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract Occurrence of degenerative interactions is thought to serve as a mechanism underlying hybrid unfitness in most animal systems. However, the molecular mechanisms underpinning the genetic interaction and how they contribute to overall hybrid incompatibilities are limited to only a handful of examples. A vertebrate model organism, Xiphophorus, is used to study hybrid dysfunction, and it has been shown from this model that diseases, such as melanoma, can occur in certain interspecies hybrids. Melanoma development is due to hybrid inheritance of an oncogene, xmrk, and loss of a co-evolved tumor modifier. It was recently found that adgre5, a G protein-coupled receptor involved in cell adhesion, is a tumor regulator gene in naturally hybridizing Xiphophorus species Xiphophorus birchmanni (X. birchmanni) and Xiphophorus malinche (X. malinche). We hypothesized that 1 of the 2 parental alleles of adgre5 is involved in regulation of cell growth, migration, and melanomagenesis. Accordingly, we assessed the function of adgre5 alleles from each parental species of the melanoma-bearing hybrids using in vitro cell growth and migration assays. In addition, we expressed each adgre5 allele with the xmrk oncogene in transgenic medaka. We found that cells transfected with the X. birchmanni adgre5 exhibited decreased growth and migration compared to those with the X. malinche allele. Moreover, X. birchmanni allele of adgre5 completely inhibited melanoma development in xmrk-transgenic medaka, while X. malinche adgre5 expression did not exhibit melanoma suppressive activity in medaka. These findings provide evidence that adgre5 is a natural melanoma suppressor and provide new insight in melanoma etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35473f18fdca14702fd87c87ffcf454b6421341" target='_blank'>
              Functional test of a naturally occurred tumor modifier gene provides insights to melanoma development
              </a>
            </td>
          <td>
            Mateo Garcia-Olazabal, M. Adolfi, B. Wilde, Anita Hufnagel, Rupesh Paudel, Yuan Lu, Svenja Meierjohann, Gil Rosenthal, Manfred Schartl
          </td>
          <td>2025-01-17</td>
          <td>G3: Genes | Genomes | Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cell division cycle-associated (CDCA) genes are dysregulated in carcinomas. Our study aims to identify similarities and differences of the clinical roles of CDCAs in breast cancer (BRCA) and to explore their potential mechanisms. In GEPIA, compared to normal tissues, expressions of CDCAs were higher in BRCA and sub-types. In addition, CDCAs were significantly positively related to stages and predicted worse survival in BRCA. In CancerSEA, expression levels of most CDCAs were strongly positively related to cell cycle, DNA damage, DNA repair, and proliferation. In TIMER, CDCAs were linked with immune infiltration levels of BRCA, including Dendritic cell, B cell and so on, and were positively related to most of the common markers of immune cells, especially CD38 of B cell and IL12RB2 of Th1. In GeneMANIA, there were complex interactions and co-expression relationships between CDCAs and cell division-associated genes. In addition, CDCA1, CDCA3, CDCA5, CDCA6 and CDCA8 had a high proportion of amplification in BRCA, and CDCA1, CDCA2, CDCA5, CDCA7 and CDCA8 had high levels of body DNA methylation. Among 11 transcription factors possibly combining promoters of all CDCAs, FOXP3 and YY1 were significantly higher in BRCA in comparison to normal tissues, and both had a positive relationship with all CDCAs in GEPIA and IHC. In addition, silencing FOXP3 or YY1 decreased levels of CDCAs in MDA-MB-231. In summary, CDCAs have various similarities in clinical functions, functional states, immune infiltration, and mechanisms, and they may become novel potential biomarkers for BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0573bd3b9dd89f3992773c13c07581660383688" target='_blank'>
              Roles of the CDCA gene family in breast carcinoma
              </a>
            </td>
          <td>
            Wei Ding, Wei Han, Chun-Tao Shi, Li-qian Yao, Zhi-wei Liang, Ming-hui Zhou, Hao-nan Wang
          </td>
          <td>2025-01-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer therapies have attempted to target the transcription factor p53, a gene also described as the “guardian of the genome,” for decades. However, the approach has faced numerous barriers to clinical efficacy due to several factors: mutations in p53 occur in almost half of all human cancers, mutations are cancer-specific, and the associated genomic changes grant mutant p53 with oncogenic potential unique from that of wild-type p53. A host of new therapeutic agents have emerged that work to target mutant p53. These agents can broadly be classified into six categories: the viral approach, direct modifiers of the p53 pathway, epigenetic modifiers of the p53 pathway, synthetic lethal agents, structural reactivators, and immune activating vaccines. Even these strategies have been met with limited success. Bypassing p53 entirely may be the next avenue in cancer therapeutics to kill tumor cells regardless of p53’s mutation pattern.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1fb68a093354533852b3fe76935ba3e2890db" target='_blank'>
              Side-stepping the guardian of the genome: current cancer therapeutics targeting mutant p53
              </a>
            </td>
          <td>
            I. Taritsa, Eric T. Fossel
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The DNA Damage Tolerance pathway (DDT) is one of the major mechanisms for resolving replication fork blocks. A central factor for DDT is the fork-associated clamp, PCNA, which can undergo to mono- or polyubiquitination, leading to error-free or error-prone modes of repair, respectively. The Rad5 protein in the yeast Saccharomyces cerevisiae plays important roles in both pathways: promoting the error-free mode by PCNA polyubiquitination and interacting with specialized DNA polymerases involved in the error-prone pathway. Rad5 also associates with telomeres, the repetitive DNA regions present at the ends of chromosomes. Telomeric DNA, tightly bound by tandem proteins arrays, poses unique challenges to replication fork progression. Here, I review the current understanding of the link between Rad5 and telomeres and present evidence that Rad5 is associated with yeast telomeres, with notable enrichment during telomere replication. This data highlights a connection between telomeres and a key DDT factor in unperturbed wild-type cells, raising intriguing possibilities about the functional interplay between telomere replication and DNA damage tolerance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55178fe5efcefe7e662ad3c10eadc34e3ab5e75a" target='_blank'>
              Rad5 and Telomere Replication
              </a>
            </td>
          <td>
            Stefano Mattarocci
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The diagnosis of Inflammatory Bowel Disease, which continues to increase year by year, is complex; these disorders can be confirmed by investigations such as fecal calprotectin, endoscopy, histology of biopsies, and CT scan; despite this, there is a lack of highly specific and minimally invasive techniques that allow not only diagnosis but also determination of the severity of the disease and its evolution. Given these challenges, the search for reliable and specific predictors that can direct the clinician toward personalized patient care is very important. In this context, can help extrachromosomal circular DNA (eccDNA) that are eukaryotic nuclear circle DNA molecules, independent from chromosomes, with a length ranging from a few hundred to a few thousand base pairs. In addition to its already known role in cancer, in IBD increased levels of eccDNA have been found compared with healthy controls. Circular DNA could represent a potential biomarker that can be isolated from peripheral blood or intestinal mucosa and is useful for making early diagnoses and assessing disease status.



 We examined peripheral blood, collected by blood sampling . "Technical Tests" were performed in order to determine the right procedure for storing blood samples and optimize the results obtained in terms of eccDNA extraction . The samples will be divided into two groups, for the first group the procedure is to centrifuge the blood at 2000g for 15 minutes at different timepoints with or without storing the plasma at 4° C; for the second group the sample is centrifuged immediately after blood collection and the plasma is stored in the refrigerator at different timepoints [Table 1]. Then for both groups we proceed with eccDNA extraction by the "Circle-Seq" analysis.



 No significant differences are shown between the two groups in terms of the amount of eccDNA [Figure 1]. In particular, the non-immediate centrifugation and separation of plasma from whole blood does not negatively affect the extraction yield, highlighting that degradative factors are not present in whole blood. In addition, refrigerated storage for a longer period of time is a positive factor for eccDNA concentration, probably because in this way circular DNA found within microvesicles has time to be extruded, increasing its concentration in plasma.



 Circulating eccDNAs could represent potential biomarkers to be detected in plasma by liquid biopsy, as the method is minimally invasive and easily reproducible in analytical laboratories, becoming a diagnostic and prognostic tool to be brought into clinical practice.



 Fundings: European Union’s Horizon 2020 Research and Innovation Programme (899417)


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59481b0f3955e21977dbcb882ab498e62539a2db" target='_blank'>
              P0167 Circular extrachromosomal DNA (eccDNA): a potential diagnostic and prognostic tool to be introduced into the clinical practice of Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            E. Foscarini, V. Emoli, V. Petito, F. Di Vincenzo, L. Masi, S. Troisi, C. Pane, A. Piazzesi, L. Turchini, D. Pugliese, L. Laterza, L. Lopetuso, D. Gerovska, M. Arauzo, L. Putignani, B. Regenberg, A. Gasbarrini, F. Scaldaferri
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Telomeres, the protective caps at the ends of human chromosomes, shorten with each round of cell division, representing a counter in the form of a biological clock. Beyond 50 to 60 divisions, the protective function of the telomeres will become unsustainable, and cells will usually experience senescence and death. Loss of cell function is manifested in the form of aging and the onset of diseases, including cancer. Cancer cells have found a way around this by overexpressing an enzyme, called telomerase, which counteracts the telomere shortening, thus allowing the neoplastic cells to keep replicating. This narrative review outlines why telomeres undergo shortening and how cancer cells exploit and take advantage of that phenomenon. The fact that malignant cells derail the biological telomere clock could be targeted for therapeutic benefit. The review also highlights the diverse telomere-based strategies explored in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0525525651060dbbe10ba3d71b4ebb15d3a5fe92" target='_blank'>
              Telomere Length and Cancer
              </a>
            </td>
          <td>
            Ismail Ibrahim Al-Janabi
          </td>
          <td>2025-01-09</td>
          <td>Al-Rafidain Journal of Medical Sciences ( ISSN 2789-3219 )</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067a6a45cdd25069d169d43fc6989270e4c626cb" target='_blank'>
              SMARCB1-driven EGFR-GLI1 epigenetic alterations in lung cancer progression and therapy are differentially modulated by MEOX2 and GLI-1.
              </a>
            </td>
          <td>
            Octavio A Trejo-Villegas, Priscila Pineda-Villegas, Leonel Armas-López, C. Mendoza-Milla, I. Peralta-Arrieta, Oscar Arrieta, I. Heijink, Joaquín Zúñiga, F. Ávila-Moreno
          </td>
          <td>2025-02-19</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [4, 13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>