<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-23 09:31:18 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>35</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>72</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>337</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>297</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Cisplatin resistance is a major cause of tumor recurrence and mortality in high-grade serous ovarian cancer (HGSOC). Extrachromosomal circular DNA (eccDNA) has emerged as a critical factor in tumor evolution and drug resistance. However, the specific contribution of eccDNA to cisplatin resistance in HGSOC remains unclear. Methods: We performed whole-genome sequencing, Circle-Seq, and RNA-Seq in four pairs of primary and cisplatin-resistant (cisR) HGSOC cell lines to characterize genome-wide eccDNA distribution and features. Functional enrichment analyses were subsequently conducted on differentially expressed eccDNA-related genes. Results: In the SKOV3 cisR cell line, we identified a large extrachromosomal circular DNA (ecDNA) carrying the HIF1A gene, which regulates DNA repair, drug efflux, and epithelial–mesenchymal transition, contributing to cisplatin resistance. Using Circle-Seq, we detected a total of 161,062 eccDNAs, most of which were less than 1000 bp and distributed across all chromosomes. Notably, the number of eccDNAs on chromosome 21 differed significantly between the primary and cisR cell lines. Additionally, eccDNAs were predominantly located in non-coding repetitive elements. Functional analysis of eccDNA-related differentially expressed genes revealed that, compared to primary cell lines, cisR cell lines were associated with mitotic spindle assembly, regulation of vascular permeability, and cell differentiation. eccDNA-related genes involved in these pathways include MISP, WIPF1, RHOD, KRT80, and PLVAP. Conclusions: Our findings suggest that eccDNAs, particularly ecDNA amplifications like HIF1A, contribute significantly to cisplatin resistance mechanisms in HGSOC. These insights highlight eccDNA as a potential target for overcoming therapeutic resistance and improving treatment outcomes in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7813e60dab9b0878e0aa308d86aa5f7fb84d46" target='_blank'>
              Characterization of Extrachromosomal Circular DNA in Primary and Cisplatin-Resistant High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Youya Wang, He Li, Qinglan Li, Yi Li, Hao Wu, Yan Ge, Xingnuo Zhu, Zhiguo Zheng, Zhongsheng Sun
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Abstract Focal amplifications of oncogenes are important cancer drivers. They can occur on chromosomes or in the context of circular extrachromosomal DNA (ecDNA). Many key features of ecDNAs were described in the 1960s to 1980s, including their “unstable” nature and their ability to confer drug resistance. With the benefit of new technologies, our understanding of ecDNAs has advanced dramatically in the last decade, both in breadth and in depth, including the remarkable discovery that ecDNAs are present in 17% of all cancers and are associated with worse patient outcomes. In this study, we present a guide to ecDNA tools and biology. Significance: Focal amplifications on ecDNAs are commonly found in cancer and are associated with poor patient outcomes and distinct biology. In this review, we provide a guide to ecDNA biology and available tools as well as our perspective on this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8be723b80118653ca09b51446fe4768c5505b8" target='_blank'>
              A Guide to Extrachromosomal DNA: Cancer’s Dynamic Circular Genome
              </a>
            </td>
          <td>
            Natasha E Weiser, Thomas B K Watkins, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-27</td>
          <td>Cancer Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Gastric cancer (GC) has a high incidence and mortality rate globally. Double minutes (DMs) are extrachromosomal circular chromosomes that carry amplified oncogenes or drug resistance genes, and they are closely associated with tumorigenesis and drug resistance. To investigate the role and regulatory mechanisms of double minutes in the malignant progression of gastric cancer, we utilize bioinformatics methods to analyze genomic copy number variation data from the CCLE cell line database and TCGA solid tumor database. We analyze and summarize the genomic features of tumor samples with double minutes. Based on these features, we classify gastric cancer samples. We use the FANTOM5 database, R package "GenomicRanges", Bedtools, and MEME SUITE to analyze the functions of the enhancers on double minutes and their regulated target genes in gastric cancer. Next, we apply the CIBERSORT package and the GDSC drug database to analyze immune cell infiltration and drug resistance in gastric cancer samples with and without double minutes from TCGA. The results show that the genome with a copy number greater than 10 and genomic fragments longer than 50 kb play a significant role. The regulatory role of double minutes enhancers affects drug resistance and tumor immunity, with disruptions in the enhancer-transcription factor-target gene regulatory loops also linked to tumor immunity. Furthermore, the target genes regulated by double minutes enhancers not only alter the expression of immune-related genes but also contribute to increased drug resistance to common chemotherapy agents in gastric cancer samples. High expression of drug resistance-related target genes in gastric cancer samples with double minutes is closely associated with poor prognosis. This study provides new insights for the treatment of gastric cancer patients with double minutes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7809d316520a84fae49c8004779285724bae28b" target='_blank'>
              Analysis of regulatory mechanisms of enhancers in gastric cancer with double minute chromosomes based on bioinformatics.
              </a>
            </td>
          <td>
            Meng-Ting An, Guan-Lin Guo, Jie Wu, Wen-Jing Sun, Xue-Yuan Jia
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have gained attention as key players in cancer heterogeneity, potentially associated with elevated oncogene copy numbers in many cancers. While the presence of eccDNA in both normal and cancer cells is confirmed, its influence on gene-level alterations in cancer cells remains largely unexplored. This study delves into the genomic profiles of eccDNA in cholangiocarcinoma (CCA), an aggressive biliary tract cancer with extensive heterogeneity and diverse molecular alterations, using a modified long-read CircleSeq method. We reveal distinct eccDNA characteristics in CCA compared to non-tumor cells, focusing on genic components and chromosomal origins. Analysing read depth differences in oncogene-containing eccDNA; we identified potential eccDNA candidates that may be relevant for CCA biology. Subsequent bioinformatics analysis was performed using the established CReSIL tool, revealing distinct patterns of these oncogenes, particularly genes in the RAS/BRAF pathway, suggesting a potential functional role. These findings highlight the remarkable heterogeneity and diverse origins of eccDNA in CCA. This study establishes the first profiling of eccDNA in cholangiocarcinoma and paves the way for further investigation of its potential contribution to oncogene amplification and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb865d79e02176dea33b880a5f277801ea0b0e0" target='_blank'>
              Characterization and identification of extrachromosomal circular DNA in cholangiocarcinoma
              </a>
            </td>
          <td>
            Zar Zar Win, Hasaya Dokduang, Siriyakorn Kulwong, W. Loilome, N. Namwat, Jutarop Phetcharaburanin, T. Wongsurawat, Piroon Jenjaroenpun, P. Klanrit, Arporn Wangwiwatsin
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6086eff81e816ca3205d5bed8a0f1a0d7841fa7" target='_blank'>
              Oncogene aberrations drive medulloblastoma progression, not initiation.
              </a>
            </td>
          <td>
            K. Okonechnikov, P. Joshi, Verena Körber, Anne Rademacher, Michele Bortolomeazzi, Jan-Philipp Mallm, Jan Vaillant, P. B. da Silva, Britta Statz, Mari Sepp, Ioannis Sarropoulos, Tetsuya Yamada, A. Wittmann, Kathrin Schramm, Mirjam Blattner-Johnson, Petra Fiesel, Barbara Jones, N. Jäger, T. Milde, K. Pajtler, Cornelis M. van Tilburg, Olaf Witt, Konrad Bochennek, Katharina Johanna Weber, Lisa Nonnenmacher, Christian Reimann, David R. Ghasemi, U. Schüller, M. Mynarek, S. Rutkowski, David T. W. Jones, A. Korshunov, K. Rippe, Frank Westermann, S. Thongjuea, Thomas Höfer, Henrick Kaessmann, Lena M. Kutscher, Stefan M. Pfister
          </td>
          <td>2025-05-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>111</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Tandem Duplicator Phenotypes (TDPs) are a class of genome instability profiles characterized by the presence of hundreds of somatic head-to-tail tandem duplications (TDs) evenly distributed throughout the cancer genome. Different TDPs are characterized by distinct but recurrent TD span profiles, they manifest with different frequencies across different tumor types, and driven by distinct genetic perturbations (i.e., BRCA1 loss, activation of the CCNE1/CCNA2 pathways or CDK12 loss). Occurring in over 14% of all cancers, TDs in TDP disrupt tumor suppressors and duplicate oncogene. Thus, TDP is a genomic instability profile that enhances the oncogenic burden of a tumor. To better understand the potential mechanisms of TD formation in the context of the TDPs, we performed a pancancer meta-analysis of the local genomic features associated with somatic structural variation breakpoints. Specifically, we grouped, classified and characterized the four basic types of somatic structural variation breakpoints (TDs, deletions, inversions and translocations) based on their span size, their TDP context as well as its associated genetic perturbation. Using both a discovery and a validation datasets, comprising 3,084 and 4,784 independent cancer genomes, respectively, we found that cyclin-related TD breakpoints, from small (∼10 Kb), medium (∼200-330 Kb) and large (∼1-2Mb) TDs, consistently form in regions with the highest likelihood of generating a replication fork barrier due to either secondary DNA conformations or transcription-replication collisions. On the other hand, both small and medium BRCA1-linked TD breakpoints show significantly weaker enrichment in those same regions. This suggests that the signal of BRCA1-linked TD breakpoints mapping to these regions may be diluted by a more homogenous distribution of breakpoints throughout the genome, potentially reflecting attempted replication across ssDNA breaks in the parental template followed by their conversion to single ended breaks. Interestingly, neither the medium nor the large TDs linked to CDK12 deficiency associated with local DNA features indicative of replication fork stalling, suggesting a different mechanism for CDK12-linked TD formation. Converging results were obtained in a sample-based analysis, where each genome was classified based on its TDP/genetic driver status and analyzed as a unit: cyclin-related TDP genomes are enriched for TD breakpoints mapping to regions of early replication/replication origin, active chromatin, high transcription, G4 and Rloop formation and regions prone to double strand break, when compared to non-TDP genomes that are wild type for all drivers examined. Conversely, BRCA1-TDP and CDK12-TDP genomes show no enrichment of these genomic features at their TD breakpoints. Of interest, CCNE1 amplified genomes and FBXW7 mutated genomes that do not classify as TDP have different breakpoint profiles compared to their TDP counterparts, suggesting that additional genetic factors must be at play to synergize with cyclin driven replication stress in inducing TDP.Taken together, our analyses indicates that the different drivers for TDPs favor distinct genomic features for SV formation. This work provides novel insights in the mutational processes that shape the genomic landscape of cancer cells.


 Francesca Menghi, Edison T. Liu. Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a critical area of cancer research due to its ubiquitous presence in tumour cells and significant role in tumorigenesis, progression and drug resistance. Recent studies demonstrate that eccDNA promotes cancer progression by influencing genomic instability, amplifying oncogenes, regulating gene expression and enhancing tumour cell adaptability to adverse conditions. While the precise mechanisms underlying eccDNA formation and its biological functions remain unclear, its potential applications in cancer diagnosis, prognosis and targeted therapy are gaining increasing recognition. This review summarises the latest advancements in eccDNA research, highlighting its potential as both a biomarker and a therapeutic target. Additionally, it emphasises the translational potential of eccDNA in clinical diagnostics and personalised treatment strategies, offering new perspectives for future cancer research and innovative therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell Proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Simple Summary High-risk human papillomaviruses (HR-HPVs), especially types 16 and 18, contribute to the development of various cancers by integrating their genomes into the host DNA, activating oncogenes or disrupting tumor suppressor genes. This review discusses the limitations of short-read sequencing (SR-Seq) for integration analyses and emphasizes the importance of using long-read sequencing (LR-Seq) technologies, which enable high-resolution mapping of complex integration events, including those previously inaccessible for analysis, i.e., repeat-rich genomic regions. This review highlights recent findings from LR-Seq studies on HPV integration and compares its mechanisms with those used by other viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bd3d524ef4cdc3c87b3f69c07be4376d8e44c3" target='_blank'>
              Novel Avenues for the Detection of Cancer-Associated Viral Genome Integrations Using Long-Read Sequencing Technologies
              </a>
            </td>
          <td>
            Larissa-Anna Bergmann, Alicja Pacholewska, Michal R. Schweiger
          </td>
          <td>2025-05-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10436da9e9bcdf67b6e4cfcf17b6e58df4672a34" target='_blank'>
              Spatio-temporal chromosomal arrangements by late-replicating heterochromatin
              </a>
            </td>
          <td>
            Satyadev Polisetty, Shuvadip Dutta, Rakesh Netha Vadnala, D. Notani, R. Padinhateeri, K. Sanyal
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Despite the advances in single cell whole genome sequencing, the ability to detect structural variations from mitotic crossovers remains challenging, while sister chromatid exchanges remain undetectable. Here we describe sci-L3-Strand-seq, a combinatorial indexing method with linear amplification for DNA template strand sequencing that cost-effectively scales to millions of single cells. We provide a computational framework to fully leverage the throughput, as well as the relatively sparse but multifaceted genotype information within each cell that includes strandedness, digital counting of copy numbers, and haplotype-aware chromosome segmentation, to systematically distinguish seven possible types of mitotic crossover outcomes and resulting genome instability. We showcase the power of sci-L3-Strand-seq by quantifying the rates of error-free and mutational crossovers in thousands of cells, enabling us to explore enrichment patterns of genomic and epigenomic features. The throughput of sci-L3-Strand-seq also gave us the ability to measure subtle phenotypes, opening the door for future large mutational screens. Furthermore, mapping clonal lineages provided insights into the temporal order of certain genome instability events, showcasing the potential to dissect cancer evolution. Altogether, we show the wide applicability of sci-L3-Strand-seq to the study of DNA repair, structural variations, and genome instability in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867f71c31d23d231530c8f668476d966ca7500c8" target='_blank'>
              High-throughput mapping of spontaneous mitotic crossover and genome instability events with sci-L3-Strand-seq
              </a>
            </td>
          <td>
            Peter Chovanec, Trevor Ridgley, Yi Yin
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Centromeres are pivotal for accurate chromosome segregation, yet their regulation and evolutionary dynamics remain poorly understood. Here, we investigate centromeres of the diploid species Gossypium anomalum (Ga, B-genome) that were transferred into tetraploid cotton G. hirsutum (Gh, AD-genome) as either an additional or integrated chromosome, as well as in synthetic allohexaploid (AABBDD) lines. We demonstrate consistent size reduction for all Ga centromeres in the Gh background. Histone modification profiling across 10 marks revealed heightened levels of both active and repressive chromatin marks within the Ga centromeres when transferred into the Gh background, particularly for H3K36me2. The centromeric histone modification perturbation extended into pericentromeric regions, with variable CENH3-binding domains consistently exhibiting a more pronounced increase in histone modification levels compared to stable centromere regions, highlighting the role of histone modification elevation in centromere dynamics. In addition, we observed enhanced chromatin accessibility and the presence of non-B-form DNA motifs, such as A-phased DNA repeats within stable centromere domains that are correlated with centromere stability. Hi-C analysis reveals a reorganized 3D chromatin architecture within the introgression line centromeres, including the formation of new topologically associating domains linked to H3K36me2 dynamics, emphasizing the importance of H3K36me2 in centromere organization. Together, these findings elucidate epigenetic mechanisms underlying centromere composition following intergenomic hybridization and allopolyploid formation, offering insights into centromere evolution in plants and its myriad epigenetic and potentially functional dimensions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cd7b66a7f1ffe44365de20596b44e77ed523aee" target='_blank'>
              Centromere-size reduction and chromatin state dynamics following intergenomic hybridization in cotton
              </a>
            </td>
          <td>
            Jinlei Han, Guanjing Hu, Yan Dai, Xin Zhang, Jingjing Tian, Jialiang Zhou, Xinqi Xu, Qi Chen, Xiaobing Kou, Lei Xu, Xinyu Wu, Ziying Sun, Jiahui Geng, Lin Li, Chenyu Qiu, Teame Gereziher Mehari, Baohua Wang, Hui Zhang, Xinlian Shen, Zhenzhen Xu, Jonathan F. Wendel, Kai Wang
          </td>
          <td>2025-05-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de502f1073df10f45cbf431033ea87108c552c" target='_blank'>
              ARID1A stabilises non-homologous end joining factors at DNA breaks induced by the G4 ligand pyridostatin
              </a>
            </td>
          <td>
            Susana Simões-Sousa, Noa Amin, Karen A. Lane, Alison Harrod, Malin Pedersen, Mercedes Pardo, Christine Rasetti-Escargueil, Z. Kozik, Katheryn A. G. Begg, Alison Ribeiro, Susana Banerjee, Manuel Stucki, Kevin J. Harrington, Jyoti S. Choudhary, Jessica A. Downs
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer, closely associated with tumor evolution, metastasis, immune evasion, and resistance to treatment. CIN is driven by persistent chromosome missegregation, resulting in abnormal chromosomal copy numbers and promoting tumor progression through mitotic errors, faulty chromosome segregation, and the formation of micronuclei or chromosome bridges. Previous studies have demonstrated that p62 localizes to micronuclei, where it interacts with mitochondria, enhancing ROS-mediated cysteine oxidation and promoting p62 homo-oligomerization. This disrupts repair by recruiting CHMP7 and other ESCRT-III components, impairing their function in restoring the micronuclear envelope. Notably, we observed a time-dependent increase in nuclear p62 levels following radiation therapy, which correlated with enhanced chromatin fragmentation and chromothripsis. While p62 inhibits homologous recombination repair and promotes non-homologous end joining, we hypothesize that radiation-induced nuclear p62 accumulation impairs nuclear envelope resealing by inhibiting ESCRT-III components, thereby contributing to chromosomal fragmentation and chromothripsis. These chromosomal alterations may play a significant role in tumor evolution and treatment resistance. Our findings suggest that irradiation-induced p62 accumulation is crucial for chromothripsis and may affect nuclear repair processes, impacting chromosomal stability. Future research will focus on elucidating how p62 contributes to these abnormalities and their implications for tumor progression and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a93ad5e20684542464f1d1902962e0915d7957" target='_blank'>
              Irradiation-induced increase in nuclear p62 levels contributes to chromosomal fragmentation and chromothripsis.
              </a>
            </td>
          <td>
            Xiaojuan Yang, Ying Zhou, Buzhe Zhang, Xue Tao, Weili Qi, Hong Wu
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogene activation is a key driver of cancer development, inducing aberrant cellular proliferation and DNA replication stress. This in turn, leads to DNA damage-which accumulates in specific genomic regions-contributing to genomic instability in cancer. However, the interplay between oncogene-induced DNA damage and chromatin organization is still poorly understood. In this study, we introduce a QUantitative ANalysis of DNA cOunterstains (QUANDO) to investigate the subnuclear localization of DNA damage in single-cell nuclei of U937-PR9 cells, an in vitro model of acute promyelocytic leukemia (APL). Using advanced imaging techniques, including DNA intensity analysis and colocalization by image cross-correlation spectroscopy (ICCS), we map DNA damage foci and correlate them with chromatin regions of different density. QUANDO is applied to dual-color confocal images of the DNA damage marker γ-H2AX and the DNA counterstain DAPI, allowing single-cell measurements of foci distribution within areas of low or high DNA density. We find that spontaneous DNA damage and DNA damage induced by the activation of PML-RARα oncogene predominantly localize in euchromatic regions. Conversely, when DNA damage is induced by the radiomimetic agent neocarzinostatin (NCS), the foci appear more evenly distributed in euchromatic and heterochromatic regions. These findings underscore the complex interplay between oncogene activation and chromatin organization, revealing how disruptions in DNA damage distribution can contribute to genomic instability and offering new insights for targeting DNA repair mechanisms in cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524273685b3a7b2a132bed81549714ac0e395f5e" target='_blank'>
              Location of oncogene-induced DNA damage sites revealed by quantitative analysis of a DNA counterstain.
              </a>
            </td>
          <td>
            Greta Paternò, Silvia Scalisi, G. Dellino, M. Faretta, P. Pelicci, Alberto Diaspro, Luca Lanzanò
          </td>
          <td>2025-05-07</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How germ cells balance retrotransposon-induced genome innovation with the need for genomic integrity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR-retrotransposon nomad completes the full mobilization cascade—including mRNA export, protein translation, and reverse transcription—to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings suggest that ecDNA formation acts as a protective mechanism to sequester retrotransposon-derived DNA and prevent widespread genomic integration during spermatogenesis, thereby preserving genome stability while allowing limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e357c0e1274581f338acdf17e4b08fe74b5c828" target='_blank'>
              Safeguarding spermatogenesis from retrotransposon insertions by forming ecDNA
              </a>
            </td>
          <td>
            Lauren Tracy, Z. Zhang
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Genetic instability is a major hazard threatening the fate of cells and ultimately of organisms. DNA double-strand break (DSB) is a highly toxic lesion, jeopardizing genome stability. Using cytogenetic and differential exome sequencing, we show here that upregulation of the kinase PKB/AKT1 leads to genomic rearrangements and chromosome fusions. By combining various approaches, at the genome scale and at precise loci, we show that PKB/AKT1 stimulates DSB end-joining, leading to inter- and intrachromosomal genomic rearrangements. The MRE11–RAD50–NBS1 (MRN) complex plays an essential role in the early steps of DSB signaling/repair. We show here that PKB/AKT1 favors the assembly of MRN, leading to the stimulation of DSB signaling via the MRE11/ATM axis. We identify MRE11 as a phosphorylation effector of PKB/AKT1 and reveal several sites whose phosphorylation is required for PKB-mediated stimulation of DSB end-joining and chromosome fusions. These data reveal that PKB/AKT1 actively promotes genetic instability by increasing the efficiency of DSB end-joining through MRE11 phosphorylation on these sites. These results highlight that not only a defect of DSB signaling/repair but also its stimulation, can lead to genome rearrangements and underline the importance of a precise regulation of the DNA damage response to maintain genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d7a378d4ec28789fd47d885beb264a6236c373" target='_blank'>
              Genome rearrangements induced by the stimulation of end-joining of DNA double strand breaks through multiple phosphorylation of MRE11 by the kinase PKB/AKT1
              </a>
            </td>
          <td>
            J. Guirouilh-Barbat, Iman Litchy Boueya, C. Gelot, Gaëlle Pennarun, Christine Granotier-Beckers, E. Dardillac, Wei Yu, C. Lescale, E. Rass, Olivier Ariste, Nicolas Siaud, Benjamin Renouf, Armêl Millet, Nadine Puget, Pascale Bertrand, Pierre de la Grange, Erika Brunet, L. Deriano, Bernard S. Lopez
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="PURPOSE
Immunotherapy has demonstrated promise in small cell lung cancer (SCLC), but certain patients encounter limited benefits, highlighting the need for immunosuppressive biomarkers. Extrachromosomal circular DNA (ecDNA) promotes amplification of MYC-paralogs (MYC, MYCN and MYCL), driving cross-resistance in SCLC. Here, we aim to investigate whether ecDNA-mediated MYC-paralogs amplification (ecMYC+) represents immunosuppressive features in SCLC.


EXPERIMENTAL DESIGN
Bulk RNA sequencing data were retrieved from public database and paraffin-embedded samples. The overexpression and amplification of MYC-paralogs were identified using immunohistochemistry and fluorescence in situ hybridization. Imaging mass cytometry and multiplex immunohistochemistry were used to characterize spatial distribution of tumor immune microenvironment (TIME). The copy number of MYC-paralogs was investigated using real-time polymerase chain reaction. RNA-sequencing and flow cytometry were performed in SCLC cell lines.


RESULTS
The mean copy number of ecDNAs and the frequency of ecMYC+ cell lines were higher in SCLC than that in the other lineages (SCLC 22/47 vs others 15/282). In ecMYC+ SCLC, multiple immune-related pathways were downregulated while nucleotide metabolism processes were upregulated. Inhibition of nucleotide metabolism induced ecDNA elimination, along with activated antigen presenting pathways. Highly dispersed MYC-paralogs amplifications were detected in resected treatment-naïve SCLC samples. Through the resolution of 103,341 cells from 24 pathological regions, we observed higher expression of MKI67, VEGFA, FAP and FOXP3 and reduced T cell infiltration in ecMYC+ samples. Moreover, ecMYC+ samples exhibited elevated cellular neighborhoods dominated by Ki67+ tumors, with reduced spatial interaction with immune cells.


CONCLUSIONS
Extrachromosomal amplification of MYC-paralogs shapes suppressive TIME, identifying potential subgroup of immunotherapy resistant patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal 11q13.3 amplification is the most common gene copy-number variation event in head and neck squamous cell carcinoma (HNSCC) that corresponds with poor prognosis. Although cyclin D1, a G1/S phase cell-cycle regulatory protein at this locus, is considered as a key driver of malignant progression, further exploration is needed to develop more effective targets for cases with this amplification. Using CRISPR-based gene knockout screening of genes located in chr11q13.3, we found that loss of the gene encoding the Fas-associated death domain (FADD) protein, a well-recognized adapter to caspase-8 that induces cell apoptosis, significantly reduced cancer cell proliferation. FADD expression was elevated in chr11q13.3-amplified tumors and correlated with poor prognosis. RNA sequencing, mass spectrometry, and proteomics analyses revealed a direct relationship between FADD and the DNA helicase MCM5 in the S phase. FADD and cyclin D1 acted at different stages of the cell cycle to synergistically induce proliferation, and caspase-8 deficiency was required for the oncogenic activity of FADD. In a patient-derived xenograft model with chr11q13.3 amplification, combined administration of the DNA helicase complex inhibitor and CDK4/6 inhibitor effectively curtailed tumor growth. Overall, this study identified a nonclassic oncogenic role for FADD in mediating tumor progression in HNSCC and provided a feasible treatment option for patients with chr11q13.3 amplification. Significance: FADD promotes progression of tumors with chr11q13.3 amplification by binding to the DNA helicase complex, which can be targeted in combination with cyclin D1 as a viable therapeutic strategy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Polymerase theta (POLθ) inhibitors were developed to overcome resistance to PARP inhibitor treatment in homologous recombination (HR) deficient cancer. Biomarkers identifying PARP inhibitor-resistant cancer cases that specifically rely on POLθ activity for cancer cell survival will help to identify the sensitive patient population. From the TCGA RNAseq data, we determined POLθ expression levels, and from whole-exome and whole-genome sequencing data, we determined the number of POLθ-associated mutational signatures in solid tumors with various HR deficiency status. We found that POLθ expression levels did not differ significantly between HR-proficient and HR-deficient cancers. However, POLθ expression correlated strongly with proliferation-associated gene expression signatures and was predominantly observed in the S and G2/M phases of the cell cycle. POLθ-associated mutational signatures are correlated with POLθ expression levels only in BRCA2-deficient cancers. POLθ expression level and POLθ-associated mutational signatures may be indicative of POLθ inhibitor sensitivity in BRCA2-deficient tumors, but are unlikely to be informative in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498ae751ba9f700cb39c5f192432c94acf2d269" target='_blank'>
              Polymerase theta expression is correlated with proliferative capacity but not with DNA repair deficiency status in solid tumors
              </a>
            </td>
          <td>
            Z. Sztupinszki, Regina Fiam, O. Pipek, M. Dióssy, J. Börcsök, Aurél Prósz, I. Csabai, Zoltan Szallasi
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles are unique, allowing sample stratification into ATRT subtypes. The TE activity signature in ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC shows broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC is also unique in having significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and-MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88b1746e8b34806ab31721a10a88339f9bbc0904" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription-replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription-induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription-Replication Conflicts: Unlocking New Frontiers in Cancer.
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Ploidy determination across the genome has been challenging for low-pass-WGS tumor-only samples. We present BACDAC, a method that calculates tumor ploidy down to 1.2X effective tumor coverage. Allele fraction patterns displayed in the Constellation Plot verify tumor ploidy and reveal subclonal populations. BACDAC outputs a metric, 2N+LOH, that when combined with ploidy better distinguishes near-diploid from high-ploidy tumors. Validated using TCGA, BACDAC had good agreement with other methods and 88% agreement with experimental methods. Discrepancies occur mainly when BACDAC predicts diploidy with subclones rather than high-ploidy. Applied to 653 low-pass-WGS samples spanning 12 cancer subtypes, BACDAC calls 40% as high-ploidy. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03599-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a5801328a2687dfdef502362bf6b982a0712bd" target='_blank'>
              Tumor ploidy determination in low-pass whole genome sequencing and allelic copy number visualization using the Constellation Plot
              </a>
            </td>
          <td>
            Sarah H. Johnson, J. Smadbeck, Roman M. Ženka, Michael T. Barrett, Athanasios Gaitatzes, Arnav Solanki, Angela B. Florio, M. Borad, John C. Cheville, G. Vasmatzis
          </td>
          <td>2025-05-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6786fefc42e4d77082eb7a0b7de5ec1cad7e96" target='_blank'>
              MITF maintains genome stability in non-melanocytic cell lineages and suppresses Hippo pathway signaling
              </a>
            </td>
          <td>
            Drifa H. Gudmundsdottir, Adrián López García de Lomana, Thejus B. Venkatesh, Kritika Kirty, S. Sigurdsson, Linda Vidarsdottir, R. Dilshat, Erla Sveinbjornsdottir, Snædís Ragnarsdóttir, Daniel H Magnusson, Maria R. Bustos, E. Steingrímsson, Thorkell Gudjónsson, Stefan Sigurdsson
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are the effective lesion of cancer radiotherapy and induce gene editing. 53BP1 accumulates at DSBs and is implicated in end joining (EJ) repair, but its influence on DSB repair is distinct from canonical non-homologous end joining (C-NHEJ). We sought to define the genetic interplay of 53BP1 with C-NHEJ, focusing on the DNA-PKcs kinase. We examined Cas9 DSBs, which largely generates blunt DSBs, since blunt DSB EJ is dependent on C-NHEJ. Loss of 53BP1 does not affect blunt DSB EJ, but causes a reduction in such repair in DNA-PKcs deficient cells. In contrast, disrupting 53BP1 and DNA-PKcs, both alone and together, has similar effects on the type of deletion mutation (increase in microhomology deletions). We found similar effects on EJ with RIF1 loss, which is a downstream effector of 53BP1. Thus, 53BP1/RIF1 appear to play a backup role for DNA-PKcs during blunt DSB EJ, but function in the same pathway to suppress microhomology deletions. In contrast, 53BP1 and DNA-PKcs function independently to suppress homology-directed repair. Finally, DNA-PKcs kinase inhibition causes marked radiosensitivity, which is not additive with loss of 53BP1 and RIF1. Altogether, 53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse DSB repair outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb1f8550e47aed05c0a51b4b294e0030758aa9f" target='_blank'>
              53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse chromosomal break repair outcomes
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779c13815a53f4713be7d07b72908d9198ac23e6" target='_blank'>
              Nucleosome-mediated conformational switches in micro-eccDNAs
              </a>
            </td>
          <td>
            Sarah Harris, Victor Velasco Berrelleza, William Sandel, T. Sutthibutpong, Victoria N. Parikh, Euan A. Ashley, Andrew Fire, S. Levene, Thomas C Bishop, M. Shoura
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62dd3d2788ffe055d52a584d9b884866f23c266" target='_blank'>
              Delayed indel formation after Cas9 cleavage promotes mosaicism and aneuploidy in human embryos
              </a>
            </td>
          <td>
            Alex Robles, Julie Sung, Stepan Jerabek, Jishnu Talukdar, Diego Marin, Jia Xu, Nathan Treff, Dieter Egli
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The lower Dipteran fungus gnat, Bradysia (aka Sciara) coprophila, has compelling chromosome biology. Paternal chromosomes are eliminated during male meiosis I and both maternal X sister chromatids are retained in male meiosis II. Embryos start with three copies of the X chromosome, but 1–2 copies are eliminated from somatic cells as part of sex determination, and one is eliminated in the germline to restore diploidy. In addition, there is gene amplification in larval polytene chromosomes, and the X polytene chromosome folds back on itself mediated by extremely long-range interactions between three loci. These developmentally normal events present opportunities to study chromosome behaviors that are unusual in other systems. Moreover, little is known about the centromeric and telomeric sequences of lower Dipterans in general, and there are recent claims of horizontally-transferred genes in fungus gnats. Overall, there is a pressing need to learn more about the fungus gnat chromosome sequences. Results We produced the first chromosome-scale models of the X and autosomal chromosomes where each somatic chromosome is represented by a single scaffold. Extensive analysis supports the chromosome identity and structural accuracy of the scaffolds, demonstrating they are co-linear with historical polytene maps, consistent with evolutionary expectations, and have accurate centromere positions, chromosome lengths, and copy numbers. The positions of alleged horizontally-transferred genes in the nuclear chromosomes were broadly confirmed by genomic analyses of the chromosome scaffolds using Hi-C and single-molecule long-read datasets. The chromosomal context of repeats shows family-specific biases, such as retrotransposons correlated with the centromeres. Moreover, scaffold termini were enriched with arrays of retrotransposon-related sequence as well as nucleosome-length (~ 175 bp) satellite repeats. Finally, the Hi-C data captured Mb-scale physical interactions on the X chromosome that are seen in polytene spreads, and we characterize these interesting “fold-back regions” at the sequence level for the first time. Conclusions The chromosome scaffolds were shown to be of exceptional quality, including loci harboring horizontally-transferred genes. Repeat analyses demonstrate family-specific biases and telomere repeat candidates. Hi-C analyses revealed the sequences of ultra-long-range interactions on the X chromosome. The chromosome-scale scaffolds pave the way for further studies of the unusual chromosome movements in Bradysia coprophila. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11573-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93782530cade9be39edb287fa99478b4aa55f399" target='_blank'>
              Chromosome-scale scaffolds of the fungus gnat genome reveal multi-Mb-scale chromosome-folding interactions, centromeric enrichments of retrotransposons, and candidate telomere sequences
              </a>
            </td>
          <td>
            John M. Urban, S. Gerbi, Allan C Spradling
          </td>
          <td>2025-05-05</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48b921317e55f0a7a7e7ce1f3c412ed182dc30b" target='_blank'>
              DNA2 and FANCM function in two distinctive pathways in disrupting TERRA R-loops and suppressing replication stress at ALT telomeres
              </a>
            </td>
          <td>
            Ashwin Ragupathi, Heba Z. Abid, Yun Chen, Rongwei Zhao, Settapong T. Kosiyatrakul, Derin I. Yetil, Julia Neiswander, Rachel Feltman, Shannon Thomas, Benjamin Yusupov, Manrose Singh, Li Zheng, Binghui Shen, Huaiying Zhang, H. C. Chu, Carl L. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Introduction CRISPR/Cas9-edited induced pluripotent stem cells (iPSCs) are valuable research models for mechanistic studies. However, gene conversion between a gene-pseudogene pair that share high sequence identity and form direct repeats in proximity on the same chromosome can interfere with the precision of gene editing. Mutations in the human beta-glucocerebrosidase gene (GBA1) are associated with Gaucher disease, Parkinson’s disease, and Lewy body dementia. During the creation of a GBA1 KO iPSC line, we detected about 70% gene conversion from its pseudogene GBAP1. These events maintained the reading frame and resulted from GBA1-specific cleavage by CRISPR/Cas9, without disrupting the GBA1 gene. Method To increase the percentage of alleles with out-of-frame indels for triggering nonsense-mediated decay of the GBA1 mRNA, we supplied the cells with two single-stranded oligodeoxynucleotide (ssODN) donors as homology-directed repair (HDR) templates. Results We demonstrate that HDR using the ssODN templates effectively competes with gene conversion and enabled biallelic KO clone isolation, whereas the nonallelic homologous recombination (NAHR)-based deletion rate remained the same. Discussion Here, we report a generalizable method to direct cellular DNA repair of double strand breaks at a target gene towards the HDR pathway using exogenous ssODN templates, allowing specific editing of one gene in a gene-pseudogene pair without disturbing the other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522033033b99ee07b46d656d628a06d2bb8f2da5" target='_blank'>
              Efficient GBA1 editing via HDR with ssODNs by outcompeting pseudogene-mediated gene conversion upon CRISPR/Cas9 cleavage
              </a>
            </td>
          <td>
            Joseph S. Lagas, M. Sentmanat, Xiaoxia Cui
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/518da9a2a0196077fdef21b0d95582e41f10777c" target='_blank'>
              The H3K36M oncohistone inhibits NSD2 to activate a SETD2-dependent antiviral-like immune response in KRAS-driven lung cancer
              </a>
            </td>
          <td>
            Amy C. Gladstein, Carson D. Poltorack, Akino Mercy Charles Solomon, Sharan Venkatesh, Keren M. Adler, Maggie R. Robertson, Stephanie Stransky, Valerie M. Irizarry-Negron, Dain A Ruiz, Nelson F. Freeburg, Simone Sidoli, Irfan A. Asangani, Sydney M. Shaffer, D. Feldser
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Reduced copy number of the D4Z4 macrosatellite at human chromosome 4q35 is associated with facioscapulohumeral muscular dystrophy (FSHD). A pervasive idea is that chromatin alterations at the 4q35 locus following D4Z4 repeat unit deletion lead to disease via inappropriate expression of nearby genes. Here, we sought to analyze transcription and chromatin characteristics at specific regions of 4q35 and how these are affected by D4Z4 deletions and exogenous stresses. Results We found that the 4q subtelomere is subdivided into discrete domains, each with characteristic chromatin features associated with distinct gene expression profiles. Centromeric genes within 4q35 (SLC25A4, FAT1 and FRG1) display active histone marks at their promoters. In contrast, poised or repressed markings are present at telomeric loci including FRG2, DBE-T and D4Z4. We discovered that these discrete domains undergo region-specific chromatin changes upon treatment with chromatin enzyme inhibitors or genotoxic drugs. We demonstrated that the 4q35 telomeric FRG2, DBE-T and D4Z4-derived transcripts are induced upon DNA damage to levels inversely correlated with the D4Z4 repeat number, are stabilized through posttranscriptional mechanisms upon DNA damage and are bound to chromatin. Conclusion Our study reveals unforeseen biochemical features of RNAs from clustered transcription units within the 4q35 subtelomere. Specifically, the FRG2, DBE-T and D4Z4-derived transcripts are chromatin-associated and are stabilized posttranscriptionally after induction by genotoxic stress. Remarkably, the extent of this response is modulated by the copy number of the D4Z4 repeats, raising new hypotheses about their regulation and function in human biology and disease. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01881-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d56bf16939905dc775b2c8dc8f0fd044f19fa7f2" target='_blank'>
              Posttranscriptional RNA stabilization of telomeric RNAs FRG2, DBE-T, D4Z4 at human 4q35 in response to genotoxic stress and D4Z4 macrosatellite repeat length
              </a>
            </td>
          <td>
            Valentina Salsi, Francesca Losi, M. Salani, Paul D. Kaufman, Rossella Tupler
          </td>
          <td>2025-05-04</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Diffuse gliomas are the commonest malignant primary brain tumour in adults. Herein, we present analysis of the genomic landscape of adult glioma, by whole genome sequencing of 403 tumours (256 glioblastoma, 89 astrocytoma, 58 oligodendroglioma; 338 primary, 65 recurrence). We identify an extended catalogue of recurrent coding and non-coding genetic mutations that represents a source for future studies and provides a high-resolution map of structural variants, copy number changes and global genome features including telomere length, mutational signatures and extrachromosomal DNA. Finally, we relate these to clinical outcome. As well as identifying drug targets for treatment of glioma our findings offer the prospect of improving treatment allocation with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba058669a48e25adcfe1a09c6668631baae8f882" target='_blank'>
              Genomic landscape of diffuse glioma revealed by whole genome sequencing
              </a>
            </td>
          <td>
            B. Kinnersley, Josephine Jung, A. Cornish, D. Chubb, Ross Laxton, A. Frangou, Andreas J. Gruber, Amit Sud, G. Caravagna, A. Sottoriva, David C Wedge, Thomas C. Booth, S. Al-Sarraj, Sam Lawrence, Erminia Albanese, Giulio Anichini, David Baxter, Alexandros Boukas, Yasir A. Chowdhury, Pietro D'Urso, Robert Corns, Andrew Dapaah, E. Edlmann, Fay Greenway, Paul L Grundy, C. Hill, Michael D. Jenkinson, Sandhya Trichinopoly Krishna, Stuart Smith, S. Manivannan, Andrew J Martin, Samir Matloob, Soumya Mukherjee, Kevin O'Neill, P. Plaha, Jonathan Pollock, Stephen J. Price, Ola Rominiyi, Bobby Sachdev, Fozia Saeed, Saurabh Sinha, Lewis Thorne, I. Ughratdar, Peter Whitfield, A. S. Youshani, H. Bulbeck, Prabhu Arumugam, R. Houlston, K. Ashkan
          </td>
          <td>2025-05-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10fbe2acee93fcc04d4cf719859f00b1fc38a341" target='_blank'>
              Megabase-scale human genome rearrangement with programmable bridge recombinases
              </a>
            </td>
          <td>
            Nicholas T. Perry, Liam J. Bartie, Dhruva Katrekar, Gabriel A. Gonzalez, Matthew G. Durrant, James J. Pai, Alison Fanton, Masahiro Hiraizumi, Chiara Ricci-Tam, Hiroshi Nishimasu, Silvana Konermann, Patrick D. Hsu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy. Although prognosis is favorable for patients with an early-stage disease, those with recurrent or more advanced disease have low response rates to chemotherapy and poor clinical outcomes. Previously, we have shown that DNA repair gene (NEIL3) is required for retaining replication fork integrity during replication stress. Here, we examined whether the overexpression of NEIL3 in endometrial cancer associated with altered genomic instability, tumor immunogenicity and anti-tumor immunity in endometrial tumor. In this study, we show that endometrial cancer patients with tumors that a have high NEIL3 expression associated with worse overall survival (OS) outcomes in patients. In addition, tumor with high NEIL3 expression is associated with high number of mutation and chromosomal instability. Furthermore, NEIL3 expression in EC tumors positively correlated with mutation of DNA polymerase eta (POLE) and TP53 as well as high expression of replicative polymerases genes (POLE, POLD1 and POLA1). In contrast, tumor with high NEIL3 expression exhibit low tumor immunogenicity and poor anti-tumor immune cell infiltration. Our findings may have important clinical implications for utilizing NEIL3 as a potential prognostic biomarker to stratify EC patients and as a target to enhance immunotherapy response in endometrial cancer. However, our NEIL3 overexpression associated observation still requires further experimental-based scientific validation studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00393-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229a73859c8660361d382a29f27f2f93cf0da384" target='_blank'>
              DNA glycosylase (NEIL3) overexpression associated with low tumor immune infiltration and poor overall patient survival in endometrial cancer
              </a>
            </td>
          <td>
            Cristofer Barry, Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2025-05-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Medulloblastoma is the most common malignant brain tumor of childhood. The highest-risk tumors are driven by recurrent Myc amplifications (Myc-MB) and experience poorer outcomes despite intensive multimodal therapy. The Myc transcription factor defines core regulatory circuitry for these tumors and acts to broadly amplify downstream pro-survival transcriptional programs. Therapeutic targeting of Myc directly has proven elusive, but inhibiting transcriptional cofactors may present an indirect means of drugging the oncogenic transcriptional circuitry sustaining Myc-MB.


METHODS
Independent CRISPR-Cas9 screens were pooled to identify conserved dependencies in Myc-MB. We performed chromatin conformation capture (Hi-C) from primary patient Myc-MB samples to map enhancer-promoter interactions. We then treated in vitro and xenograft models with CDK9/7 inhibitors to evaluate effect on Myc-driven programs and tumor growth.


RESULTS
Eight CRISPR-Cas9 screens performed across three independent labs identify CDK9 as a conserved dependency in Myc-MB. Myc-MB cells are susceptible to CDK9 inhibition, which is synergistic with concurrent inhibition of CDK7. Inhibition of transcriptional CDKs disrupts enhancer-promoter activity in Myc-MB and downregulates Myc-driven transcriptional programs, exerting potent anti-tumor effect.


CONCLUSIONS
Our findings identify CDK9 inhibition as a translationally promising strategy for the treatment of Myc-MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51ba77a6e26484b146e2f28eb319b79477cb1029" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma.
              </a>
            </td>
          <td>
            L. M. Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Etienne P. Danis, Pradeep Bompada, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-05-15</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSOC) is driven by chromosomal instability and patterns of genomic rearrangement have predictive and prognostic value in this disease. Nearly all next generation sequencing performed to date rely on short read sequencing, which is unable to resolve complex rearrangements or analyze repetitive centromeric or telomeric regions. Here we combine long-read sequencing (LRS) platform with the recently completed telomere-to-telomere (T2T-CHM13) reference genome to report the first comparison of whole genome variation in BRCA1-mutated and CCNE1-amplified HGSOC.



 High-molecular-weight DNA was extracted from cryopreserved, pre-treatment tumor and matched blood samples from six HGSOC patients and sequenced using Oxford Nanopore PromethION platform. Base-called reads were aligned to both the GRCh38 and T2T-CHM13 reference genomes for the analysis of SNVs, indels, allele-specific CNVs, complexed structural variants, transposable element insertions, and CpG methylation patterns.



 Somatic pathogenic TP53 mutations were identified in all tumors (P1-P6). Germline pathogenic BRCA1 mutations were present in two cases (P1, P2), while CCNE1 amplification was observed in the remaining four cases (P3-P6). Homologous recombination deficiency (HRD) scores were elevated in BRCA1-mutated tumors (73 and 84) compared to CCNE1-amplified tumors (71, 43, 53, and 29, respectively). Complete loss of heterozygosity in chromosome 17 was observed in all tumors. Chromoplexy was detected in five tumors (P1, P2, P4-P6). Focal amplifications associated with extrachromosomal DNA and breakage-fusion-bridge cycles were observed in P2 and P6. Telomere length was significantly shortened in all tumors compared to matched normal samples. Additionally, CpG methylation levels were significantly reduced in centromeric satellite repeats and inserted transposable elements in all tumors than in normal samples.



 This study establishes the first LRS-based analysis workflow for comprehensive genomic characterization of HGSOC, uncovering novel alterations in previously inaccessible genomic regions. These findings highlight the potential of LRS as a new gold standard for whole genome analysis in HGSOC, warranting further investigation for its clinical and research applications.



 Shiro Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena Vuttaradhi, Allison Brodsky, Barrett Lawson, Robert T. Hillman. Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB347.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841194928a5faee8a089ace06821025bbd9fd89a" target='_blank'>
              Abstract LB347: Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, B. Lawson, R. Hillman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Mutation, deletion, or silencing of genes encoding cellular metabolism factors occurs frequently in human malignancies. Neomorphic mutations in isocitrate dehydrogenases 1 and 2 (IDH1/2) promoting the production of R-2-hydroxyglutarate (R-2HG) instead of α-ketoglutarate (αKG) are recurrent in human brain cancers and constitute an early event in low-grade gliomagenesis. Due to its structural similarity with αKG, R-2HG acts as an inhibitor of αKG-dependent enzymes. These include the JUMONJI family of lysine demethylases, among which KDM4A is particularly sensitive to R-2HG-mediated inhibition. However, the precise molecular mechanism through which inhibition of αKG-dependent enzymes by R-2HG promotes gliomagenesis remains poorly understood. Here, we show that treatment with R-2HG induces cellular senescence in a p53-dependent manner. Furthermore, expression of mutated IDH1R132H or exposure to R-2HG, which leads to KDM4A inhibition, causes telomeric dysfunction. We demonstrate that KDM4A localizes to telomeric repeats and regulates abundance of H3K9(me3) at telomeres. We show that R-2HG caused reduced replication fork progression, and that depletion of SMARCAL1, a helicase involved in replication fork reversal, rescues telomeric defects caused by R-2HG or KDM4A depletion. These results establish a model whereby IDH1/2 mutations cause R-2HG-mediated inhibition of KDM4A, leading to telomeric DNA replication defects, telomere dysfunction, and associated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728014013df3b457cbd616bc46c4a758e940b3f8" target='_blank'>
              R-2-hydroxyglutarate-mediated inhibition of KDM4A compromises telomere integrity
              </a>
            </td>
          <td>
            Florence Couteau, Laurence M Gagné, Karine Boulay, Philippe Rousseau, Mélissa Carbonneau, Mary McQuaid, Jyoti Sharma, C. Sawchyn, Erlinda Fernandez, Dagmar Glatz, Rana Rizk, Marie-Eve Lalonde, Yosra Mehrjoo, Tsz Wai Chu, G. Moquin-Beaudry, Christian Beauséjour, Mikhail Sergeev, Santiago Costantino, D. Avizonis, I. Topisirovic, Nada Jabado, Hugo Wurtele, Chantal Autexier, Frédérick A. Mallette
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background/Aim: The over-activation of oncogenes is a critical genetic event in the development and progression of solid malignancies. Gene amplification and specific mutations represent the prominent mechanisms that convert proto-oncogenes into their active, oncogenic forms. C-myc oncogene (gene locus: 8q24.21) regulates crucial cell and tissue functions, whereas its deregulation is implicated in carcinogenesis. Our research aimed to investigate the impact C-myc numerical imbalances on a series of laryngeal squamous cell carcinomas (LSCCs) characterized by different clinic-pathological features. Materials and Methods: We analyzed 55 LSCC archival tissue sections by implementing a dual-color gene/chromosome fluorescent in situ hybridization assay (C-myc/Chr 8 FISH). Results: According to the extracted C-myc gene/Chr 8 signal ratios, we identified 18 cases with numerical imbalances. The majority of them were gene amplifications (pure n=15, amplification and polysomy n=1), whereas the last two (n=2) cases presented multiple gene copies as a result of Chr 8 polysomy. C-myc amplification was correlated with advanced stage. Stage III/IV cases demonstrated the highest levels of gene amplification (p=0.024). Conclusion: C-myc amplification, combined or not with Chr 8 polysomy, is a relatively frequent event in LSCC. This numerical imbalance is associated with an aggressive phenotype, and it seems to be a significant key element among other oncogenes and suppressor genes that form specific genomic signatures in the corresponding patients with LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf169522cd061e72ca9d71b7495c4317a28f61a5" target='_blank'>
              C-myc Oncogene Numerical Imbalances Analysis in Laryngeal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Sotirios Papouliakos, V. Papanikolaou, A. Chrysovergis, E. Tsiambas, D. Roukas, Sofianiki Mastronikoli, G. Tsouvelas, G. Agrogiannis, Nikolaos Kavantzas, Vasileios Ragos, Andreas C. Lazaris, E. Kyrodimos
          </td>
          <td>2025-05-27</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1a766c9476f757f76140a52e67514cf195c7f" target='_blank'>
              Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination.
              </a>
            </td>
          <td>
            K. Zhao, J. Vos, S. Lam, Lillian A Boe, Daniel Muldoon, Catherine Y Han, Cristina Valero, Mark Lee, C. Fitzgerald, A. Lee, Manu Prasad, Swati Jain, Xinzhu Deng, Timothy A Chan, Michael F. Berger, C. Bandlamudi, Xi Kathy Zhou, Luc G. T. Morris
          </td>
          <td>2025-06-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Targeted inhibition of DNA damage response proteins has received significant clinical attention owing to the success of PARP inhibitors. Due to the loss of the G1/S checkpoint, cancer cells are reliant on the G2/M checkpoint to cope with elevated DNA replication stress. We previously demonstrated a single induction of 8-oxo-guanine at telomeres in cancer cells was sufficient to induce replication stress, but was well tolerated at the cellular level. Here, we found inhibition of ATR, Chk1, or Wee1 after induction of telomere oxidative stress significantly induced genome instability and reduced cell viability. This occurred at doses markedly less than those required to increase instability in non-cancer cells. We determined the mechanism of this instability is due to cells progressing through S-phase with telomere damage and exiting G2-phase prematurely, prolonging their mitosis. This study demonstrates targeted oxidative base damage at telomeres can enhance the therapeutic efficacy of ATR inhibition in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520fb2116ae4feadf96591801b8f58200ddc5cec" target='_blank'>
              Oxidative Base Damage to Telomeres Sensitizes Cancer Cells to ATR Inhibition
              </a>
            </td>
          <td>
            Alex Garbouchian, Natalia Cestari Moreno, A. Dey, Patricia Opresko, Ryan Barnes
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ARID1A, a key subunit of the SWI/SNF chromatin remodeling complex, is frequently mutated in cancers. However, effective clinical treatments for patients with this mutation are limited, highlighting a demand for new therapeutic strategies. Here, we establish Werner syndrome ATP-dependent helicase (WRN) as a critical vulnerability target in ARID1A-mutated cancers. Upon genetic and pharmacological inhibition of WRN, ARID1A-mutated cells had defective Chk1-mediated DNA damage signaling, resulting in compensatory Chk2 activation, leading to G1-phase arrest and apoptosis, whereas ARID1A-proficient cells underwent Chk1-dependent G2/M arrest. Additional p21 inhibition, combined with WRN suppression, drove G1-arrested cells to re-enter the cell cycle, triggering mitotic catastrophe. The anti-tumor efficacy of WRN inhibition was validated using in vivo cell lines and patient-derived xenograft mouse models. Our findings define WRN as a selective therapeutic target in ARID1A-mutated cancers and suggest a combinatorial strategy of WRN and p21 inhibition as a therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee45b7e29b6454158a945bb535aaf35dcedcd93" target='_blank'>
              Differential DNA damage response to WRN inhibition identifies a targetable vulnerability in ARID1A-mutated cancers
              </a>
            </td>
          <td>
            Jiwon Kim, Jaeik Oh, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Sang Eun Lee, Yoojin Yang, Dohee Kim, Ae-Jin Choi, Hae-Rim Jung, Yumi Oh, Sung-yup Cho
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is a group of bone and soft tissue cancers in children and young adults. The EWS-FLI1 protein, the oncogenic driver in EwS, is a fusion of the FET protein, EWSR1, at its amino terminus, with the carboxy terminus of the transcription factor, FLI1. Since EwS cells have pronounced sensitivity to radiation and chemotherapy-induced DNA damage, the role of EWS-FLI1 in DNA repair is likely. Here, we demonstrate that EWS-FLI1 causes a defect in microhomology-mediated end-joining (MMEJ) repair. EWSR1 is a splicing factor that promotes the faithful splicing of the POLQ pre-mRNA, required for the expression of POLΘ, a critical protein in the MMEJ pathway. Expression of EWS-FLI1, or loss of EWSR1, causes exon 25 skipping of the POLQ transcript, decreased POLΘ protein expression, impaired MMEJ, and cellular sensitivity to inhibitors of NHEJ repair and HR repair, through the mechanism of synthetic lethality. Knockdown of EWS-FLI1 expression restores POLΘ mitotic foci and increases MMEJ activity. Based on this mechanism, inhibitors of NHEJ or HR may provide a targeted therapy for patients with Ewing sarcoma. Highlights - Ewing sarcoma tumors have a deficiency in POLθ expression and a corresponding loss of MMEJ activity - EWSR1 is a splicing factor that interacts with other splicing factors such as FUBP1 and KHSRP/FUBP2 to accurately splice the POLQ mRNA. - The EWS-FLI1 fusion oncoprotein, or loss of EWSR1, causes a splicing defect, leading to exon25 skipping of the POLQ pre-mRNA and loss of POLθ expression - The MMEJ deficiency of EwS cells results in cellular sensitivity to inhibitors of Homologous Recombination or Non-Homologous End-Joining - Exon 25 skipping of POLQ mRNA is a predictive biomarker for HR inhibitors in human cancers">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1eaf5b79d22ae4a1a31716b8447290969ad6a6" target='_blank'>
              Disruption of Microhomology-mediated End-joining in Ewing Sarcoma
              </a>
            </td>
          <td>
            Shuhei Asada, Guangli Zhu, J. Abeykoon, Yutaro Tanaka, Huy Nguyen, Yuna Hirohashi, Divya R Iyer, N. Ashton, Sirisha Mukkavalli, Lige Jiang, Miles Del Busso, J. Thangaiah, Steven I. Robinson, K. Parmar, E. V. Van Allen, R. Gillani, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6032320cee71742e5a132f4662a3a337579e09" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-05-03</td>
          <td>Genetics</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, Tulip Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Micronuclei originate from DNA damage generated by clastogenic and/or by aneugenic effects. Depending on the pattern of damage, they may have distinct genomic origin and composition. Sequences of the centromere, telomere and rDNA have been identified in plant micronuclei. However, other DNA sequences may also be present in the micronuclei, as well as their DNA contents may be different. Here, we investigate the DNA content, genomic composition and origin of micronuclei induced in Zea mays by methyl methanesulfonate (MMS). DNA contents showed a wide range of distribution, suggesting their diverse genomic origins and illustrating how much of the nuclear genome can be lost due to mutagen effects. Micronuclei diversity was also evidenced by in situ probing with different DNA sequences (5S and 18S rDNAs, 180-bp knob and Grande LTR-retrotransposon) and by 6-diamidino-2 phenylindole (DAPI) fluorochrome. Perhaps these sequences are hotspots for MMS damage, especially the Grande LTR-retrotransposon, 5S and 18S rDNAs, which are rich in guanine. In addition, probe pools were constructed from individual genomic DNA of two microdissected micronuclei. These probe pools hybridized on all Z. mays chromosomes. However, the centromere, knob and secondary constriction were hybridized by only one probe pool, evidencing the distinct genomic composition of the micronuclei. We illustrate the micronuclei genomic diversity as they originated from several different chromosomes following the MMS treatment, and demonstrate the extent of the genotoxic damage to the genome. We provide some insights into micronuclei structure and diversity, and show that they can be further explored in mutagenesis research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31c4f2a6572ceed6e9b3fda5f1cc0379119cd48" target='_blank'>
              DNA content, repeatome composition and origin of the Zea mays micronuclei
              </a>
            </td>
          <td>
            Nathália Vállery Tostes, Marcos Vitor Rosa Ferreira, Fernanda Aparecida Ferrari Soares, J. C. Silva, L. Bhering, W. Clarindo
          </td>
          <td>2025-04-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is crucial for maintaining genomic stability by repairing DNA damage. Despite its importance, HRR's role in cancer progression is not fully elucidated. Here, this work shows that nuclear‐localized branched‐chain α‐ketoacid dehydrogenase kinase (BCKDK) acts as a modulator of HRR, promoting cell resistance against DNA damage‐inducing therapy in breast cancer. Mechanistically, this work demonstrates that BCKDK is localized in the nucleus and phosphorylates RNF8 at Ser157, preventing the ubiquitin‐mediated degradation of RAD51, thereby facilitating HRR‐mediated DNA repair under replication stress. Notably, aberrant expression of the BCKDK/p‐RNF8/RAD51 axis correlates with breast cancer progression and poor patient survival. Furthermore, this work identifies a small molecule inhibitor of BCKDK, GSK180736A, that disrupts its HRR function and exhibits strong tumor suppression when combined with DNA damage‐inducing drugs. Collectively, this study reveals a new role of BCKDK in regulating HRR, independent of its metabolic function, presenting it as a potential therapeutic target and predictive biomarker in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac4a9c9be6494a377793b137fd2fa59c3a2efc2e" target='_blank'>
              Nuclear‐Localized BCKDK Facilitates Homologous Recombination Repair to Support Breast Cancer Progression and Therapy Resistance
              </a>
            </td>
          <td>
            Haiying Liu, Jiaqian Feng, Tingting Pan, Pinggen Zhang, Ling Ye, Zetan Jiang, Zilong Zhou, Qiankun Mao, Jian Li, Xinyi Yang, P. Gao, De Huang, Huafeng Zhang
          </td>
          <td>2025-04-29</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="RNA–DNA interactions are fundamental to cellular physiology, playing critical roles in genome integrity, gene expression, and stress responses. This review highlights the diverse structures of RNA–DNA hybrids, including R-loops, RNA–DNA triplexes, and RNA–DNA hybrid G-quadruplexes (hG4s) and their relevance in physiology. R-loops are formed during transcription and replication, which regulate gene expression and chromatin dynamics but can also threaten genome stability. RNA–DNA triplexes, often formed by long noncoding RNAs (lncRNAs) such as FENDRR and MEG3, recruit chromatin modifiers like Polycomb repressive complex 2 to modulate gene expression, influencing organogenesis and cell specification. hG4s, formed by guanine-rich sequences in RNA and DNA, regulate transcription termination and telomere stability. Through this, hG4s can affect gene suppression and replication regulation. RNA–DNA hybrids are tightly regulated by helicases, RNase H enzymes, and topoisomerases, with altered regulation linked to genomic instability and disease. This review discusses the complexity of RNA–DNA interactions and their recently identified contributions to cellular physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e76333f352e42183b0f9c7b2686b45523c8684f6" target='_blank'>
              The relevance of RNA–DNA interactions as regulators of physiological functions
              </a>
            </td>
          <td>
            Julia Stötzel, T. Warwick, Praveenya Tirunagari, Ralf Peter Brandes, M. Leisegang
          </td>
          <td>2025-05-21</td>
          <td>Pflugers Archiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Inter-chromosomal interactions play a crucial role in 3D genome organization, yet the organizational principles and functional significances remain elusive. In general, lncRNA loci and transcripts are frequently associated with transcriptional programs modulated by long-range chromatin interactions. Here, we identified a novel lncRNA named Gm26793, which is abundantly distributed in the primitive streak and mesodermal cells of embryonic day 7.5 mouse gastrula. Through genetic ablation of Gm26793, we observed a preferential responsiveness to primitive endoderm lineage during stem cell differentiation, as well as enhanced occurrence of transient and degenerative state cells in early mouse embryos when the cell fate segregates between epiblast and primitive endoderm. Mechanistically, we revealed that the genomic locus of Gm26793, rather than the lncRNA transcript or adjacent gene, governs the cell fate preference towards primitive endoderm. Concretely, Gm26793 locus (Chromosome 7) forms an inter-chromosomal molecular lock with Cubn (Chromosome 2) via CTCF, restraining the expression of Cubn and maintaining a natural epigenetic landscape, thus ensuring the proper lineage specification in vitro and in vivo. Overall, our study provides a clear paradigm that inter-chromosomal interaction collaborates with architectural factors to stabilize nuclear conformation and guarantee faithful gene expression during stem cell differentiation and mammalian embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5737d88dd5a339188134898f4ba7c9851ba9807" target='_blank'>
              Genomic locus of lncRNA-Gm26793 forms an inter-chromosomal interaction with Cubn to ensure proper stem cell differentiation in vitro and in vivo
              </a>
            </td>
          <td>
            Zhiwen Liu, Xin Wan, Jiehui Chen, Yongjian Ma, Yonggao Fu, Yingying Chen, Mingzhu Wen, Yun Yang, Yun Qian, Yong Zhang, Dahai Zhu, Jinsong Li, Naihe Jing, Xianfa Yang
          </td>
          <td>2025-06-03</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Tumor-associated macrophages (TAMs) play a crucial role in the tumor microenvironment, regulating immune escape and promoting cancer progression. Understanding the role of phagocytosis regulators in female-specific cancers is essential for developing effective therapeutic strategies. Methods We performed comprehensive analyses of public databases to evaluate the expression, somatic mutations, and copy number variations of phagocytosis regulators. DNA methylation patterns, biological pathways, survival outcomes, and drug sensitivity were assessed. Additionally, immune modulators, immune cell infiltration, and single-cell sequencing were used to explore alterations in phagocytosis and their cellular origins. The functional role of HMGB2 in tumor cell behavior was validated through in vitro assays. Results Phagocytosis regulators exhibited differential expression across various female-specific cancers, with key genes such as CD47 and FOXO1 playing significant roles in modulating tumor progression. High-frequency mutations were found in PTEN, ARID1A, and UBR4. Genes like COX5B and MS4A1 emerged as potential predictors of clinical outcomes and therapeutic response. HMGB2 knockdown significantly inhibited cancer cell proliferation, migration, and invasion in female-specific cancers. HMGB2 knockdown in macrophages led to a significant impairment in phagocytosis of breast, cervical, ovarian, and endometrial cancer cells. Furthermore, when HMGB2 knockdown was combined with Palbociclib treatment, a significant decrease in tumor cell proliferation was observed across multiple cancer models. Conclusion This study highlights the pivotal role of phagocytosis regulators, particularly HMGB2, in the progression of female-specific cancers. Targeting HMGB2 offers promising therapeutic opportunities, potentially enhancing precision oncology and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04106b265eee3d853f545357f94ba192188932f5" target='_blank'>
              Pan-cancer analysis of phagocytosis regulators in female-specific cancers: a focus on HMGB2
              </a>
            </td>
          <td>
            Xiaoqin Lu, Dan Ren, Panpan Zhao, Yanfang Li, Zhenhui Wang, Jingyan Zhang
          </td>
          <td>2025-05-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Defective DNA repair and metabolic rewiring are highly intertwined in promoting the development and progression of cancer. However, the molecular players at their interface remain poorly understood. Here we show that an RNF20-HIF1α axis links the DNA damage response and metabolic reprogramming in lung cancer. We demonstrate that RNF20, which catalyzes monoubiquitylation of histone H2B (H2Bub1), controls Rbx1 expression and thereby the activity of the VHL ubiquitin ligase complex and HIF1α levels. Ablation of a single Rnf20 allele significantly increases the incidence of lung tumors in mice. Mechanistically, Rnf20 haploinsufficiency results in inadequate tumor suppression via the Rnf20-H2Bub1-p53 axis and induces DNA damage, cell growth, epithelial-mesenchymal transition (EMT), and metabolic rewiring through HIF1α-mediated RNA polymerase II promoter-proximal pause release, which is independent of H2Bub1. Importantly, decreased RNF20 levels correlate with increased expression of HIF1α and its target genes, suggesting HIF1α inhibition as a promising therapeutic approach for lung cancer patients with reduced RNF20 activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f3b68771f5a1b6eb47d9ec90154cc6952cdc20" target='_blank'>
              RNF20 links the DNA damage response and metabolic rewiring in lung cancer through HIF1α
              </a>
            </td>
          <td>
            Hao Liu, Yongqin Tang, Anshu Singh, Joaquim Vong, Julio Cordero, Arthur Mathes, Rui Gao, Yanhan Jia, B. Garvalov, T. Acker, Gernot Poschet, R. Hell, Marc A Schneider, J. Heineke, Thomas Wieland, Guillermo Barreto, A. Cerwenka, Michael Potente, Sofia-Iris Bibli, R. Savai, Gergana Dobreva
          </td>
          <td>2025-05-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Aims Glioma-associated oncogene homologue 1 (GLI1) was recently shown to be coamplified with mouse double minute 2 (MDM2), cyclin-dependent kinase 4 (CDK4) and some other adjacent genes in a significant subset of GLI1-altered mesenchymal tumours and well-differentiated/dedifferentiated liposarcomas, which are characterised by MDM2 amplification. Given that MDM2 is also amplified in low-grade osteosarcoma (LGOS), we investigated the prevalence of GLI1 amplifications/fusions in a series of 15 cases of MDM2-amplified LGOS, an area that has not been previously explored. Methods This study conducted a retrospective analysis and examined GLI1 amplifications/fusions in 15 cases of MDM2-amplified LGOS and 46 cases of other bone tumours and tumour-like lesions using fluorescence in situ hybridisation with a GLI1 amplification probe and a GLI1 break-apart probe. Six cases of LGOS were also tested by next-generation sequencing. Results Fluorescence in situ hybridisation analysis revealed that 13 of 15 (87%) LGOS cases exhibited GLI1 amplification; no fusion gene was found. Next-generation sequencing revealed that all six tested cases showed GLI1 amplification and one case had both GLI1 amplification and GLI1 gene fusion (PPM1H::GLI1). All 46 cases of other bone tumours and tumour-like lesions were negative for GLI1 amplification and GLI1 fusion. Conclusion These results indicate that GLI1 amplification is common in LGOS, and GLI1 fusion could occur in LGOS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c1647776de5620ccdffd388773db804d1f24630" target='_blank'>
              GLI1 amplification and fusion in MDM2-amplified low-grade osteosarcoma
              </a>
            </td>
          <td>
            Lan Li, Ming Zhang, Xiaoqi Sun, Yuzi Zhang, Yongbin Su, Rongfang Dong, Tingting Zhang, Ziyi Wang, Yi Ding
          </td>
          <td>2025-05-02</td>
          <td>Journal of Clinical Pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1225ffe2d0c6c775628165cdb9bef1963448726" target='_blank'>
              A First-in-Class Therapy Targeting Transcription-Replication Conflicts Drives Synthetic Lethality in Homologous Recombination Deficient Cancers
              </a>
            </td>
          <td>
            Yi Chieh Lim, Soak-Kuan Lai, Ru Han Yang, Yong Kang Chua, Rémi Patouret, Vaishnervi Manoj, Colin Chun Xuan Soh, Benjamin Lebeau, Ning Cham, Wei Meng, Sze Keong Tey, Szu Ling Yeap, Songjing Zhang, Jessie Yong Xing Woon, Zahin Ali, Marek Mutwil, Pritpal Singh, M. Fullwood, Linda LD Zhong, Xue Ming Dong, Chiba Shunsuke, Cheng-Gee Koh, H. Li
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a457589b3ad117eb4f8b901ba9a44303f44812" target='_blank'>
              Mechanistic insights into transcriptional regulation of ARHGAP36 expression identify a factor predictive of neuroblastoma survival
              </a>
            </td>
          <td>
            S. Havrylov, Armin M Gamper, O. Lehmann
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cddde94c859320bea53863c22a0c0f005dca0fc" target='_blank'>
              Comprehensive analysis of FAM83B in pan-cancer and preliminary exploration in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Wei Guo, Xixi Zhao, Xinran Huang, Ruijuan Zhang, Yuchen Wang, Xinyu He, Xiangyun Ma, Yu Hao, Shangyi Geng, S. Pan, Hongbing Ma
          </td>
          <td>2025-05-26</td>
          <td>Journal of molecular histology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ed51bfd7d0538e3c3fdaf6bc3f9553670a3712" target='_blank'>
              Cancer-Associated USP28 Missense Mutations Disrupt 53BP1 Interaction and p53 Stabilization
              </a>
            </td>
          <td>
            Hazrat Belal, Esther Feng Ying Ng, Midori Ohta, Franz Meitinger
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 e16580


 Background:
 Non-muscle invasive bladder cancer (NMIBC) presents a significant clinical challenge due to its high recurrence rate. Recently, a deeper understanding of tumor epigenetics has provided new opportunities for cancer detection and treatment. While significant efforts have been made to understand NMIBC biology, the molecular and epigenetic mechanisms driving its progression remain elusive.
 Methods:
 We used multiomics analysis including expression, DNA methylation, and cell-free DNA chromatin immunoprecipitation sequencing (cfChIP-seq) assays to uncover the molecular and epigenetic landscape of NMIBC from human tissue samples and urine.
 Results:
 High-throughput sequencing revealed more than 6,000 differentially methylated regions (DMRs) within regulatory elements, many of which were linked to cancer-related pathways and exhibited distinct signatures in cancer tissues compared to healthy controls. Next, we distinguished high and low-grade tumors based on unique DNA methylation profiles. These patterns correlated with transcription level of genes associated with cell cycle regulators, consistent with the higher proliferative capacity of high-grade tumors. Moreover, Among the DMRs, we also identified genes involved in T cell regulation. Supporting this, we demonstrated reduced CD8+ T cells infiltration in high-grade tumors, further highlighting immune dysregulation in the more aggressive disease. We extended our investigation to urine samples, performing cfChIP-seq on samples from healthy donors and NMIBC patients. By using the H3K4me3 promoter histone mark on cfDNA we could detect cell origin. Utilizing this assay, we were able to identify changes in the urine of NMIBC patients, owing to cf-nucleosome contributions from both the tumor and immune cells. Additionally, we stratified the cancer cohort by tumor grade and found enrichment in the number of differentially marked genes in high-grade tumors.
 Conclusions:
 Our detection of unique epigenetic signatures offer opportunities for biomarker development, enabling early detection through non-invasive urine based diagnostic approach. This could grade tumor more accurately and tailor personalized patient care and monitoring.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dd958dabb53d9b68b4717906734c6068947417" target='_blank'>
              Elucidating the epigenetic landscape of non-muscle invasive bladder cancer.
              </a>
            </td>
          <td>
            T. Falick Michaeli, Batia Azria, M. Lotem, Nir Friedman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef02803fbaf5cb1abbd79351dc6a4ca1c68ba79" target='_blank'>
              Constitutional copy number amplifications: rare or under-evaluated? Revisiting a 25-year-old cold case.
              </a>
            </td>
          <td>
            Eliana Salvo, Romano Tenconi, R. Giorda, Sara Bertuzzo, Luca Cesana, Roberta Murru, S. Giglio, Mana M Mehrjouy, Niels Tommerup, Orsetta Zuffardi, M. Bonaglia
          </td>
          <td>2025-06-04</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Background: Cell-free plasma DNA (cfDNA) from cancer patients contains tumour-derived DNA fragments (ctDNA), allowing a non-invasive method for analysing tumour molecular information in the blood of cancer patients. This allows genetic tracking of tumours and the identification of patients that may not respond or stop responding to treatments. CDK4/6 inhibitors (CDK4/6i) in combination with endocrine therapy are the standard of care first-line treatment in ER+/HER2-negative advanced breast cancer. Clinical trials showed improvement of PFS and OS with the use of CDK4/6i. Currently there are no robust biomarkers of response or resistance to CDK4/6i and a rapid disease progression is often seen following failure of response to CDK4/6i. Early identification of response is crucial and has multiple clinical and financial implications. Here, we use shallow whole genome sequencing (sWGS) to detect copy number alterations (CNAs) and calculate a Genome Instability (GI) score with the aim to investigate correlations of GI with CDK4/6i treatment and clinical outcomes. Methods: Blood samples were collected from a small cohort of metastatic ER+/HER2 negative breast cancer patients (n=36), including patients on CDK4/6i and patients on chemotherapy following progression on treatment with aromatase inhibitor. Bloods were processed to collect the plasma containing the cfDNA which was extracted, quantified and analysed using Agilent Bioanalysis. Samples demonstrating a clean cfDNA profile with minimal contaminating larger sized leukocyte DNA were taken forward for library preparation and sequencing. The resulting genomic data were normalised following binning and background correction resulting in genome wide copy number patterns. GI scores were calculated as a read out of genomic disruption with respect to control samples. Analysis of PFS and OS were carried out with respect to treatment type and GI score thresholds. Results: Patients receiving chemotherapy demonstrated significantly higher GI scores than those treated with CDK4/6i (median chemotherapy = 319.5, CDK4/6i = 263.7, T-test P-value = 0.02), evident particularly over chromosomes 3,4,5 & 21. Additionally, patients on CDK4/6i for <1 year demonstrated significantly higher GI scores than patients on CDK4/6i for >1 year (Poor response with <1year on CDKI = 328.7, good response and remain on CDKI > 1 year = 231.7). Survival analysis showed that patients on CDK4/6i for <1 year also demonstrated worse progression free survival (PFS) and overall survival (OS) then those who remained on CDK4/6i for >1 year (median difference PFS = 630 days; OS = 377 days). Conclusion: Preliminary results examining the GIN scores from our sWGS of cfDNA suggest that high GI scores were associated with shorter progression free survival on CDK4/6i plus endocrine treatment. Patients with higher GI scores at the time of diagnosis are less likely to respond well to CDK4/6i predicting poor clinical outcomes.
 Citation Format: Olga O|ikonomidou, Fiona Semple, John P. Thomson, Natalie Wilson. Shallow whole genome sequencing of cell free DNA to predict response to CDK4/6 inhibitor treatment in ER+/HER2- metastatic breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-03-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/534fdb80bc4cf6d65a8e58970a1c813e1941062f" target='_blank'>
              Abstract P2-03-22: Shallow whole genome sequencing of cell free DNA to predict response to CDK4/6 inhibitor treatment in ER+/HER2- metastatic breast cancer
              </a>
            </td>
          <td>
            Olga O|ikonomidou, Fiona Semple, John P. Thomson, Natalie Wilson
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Fluoroquinolones induce double-strand breaks in bacterial DNA, triggering the SOS response, a major DNA damage response that ensures the expression of repair proteins but also promotes the emergence and spread of antibiotic resistance. Fluoroquinolone resistance, particularly in Escherichia coli, is a growing global health concern. Understanding bacterial responses to these antibiotics is critical for developing preventive strategies and novel treatments to combat resistance development. This study investigates DNA morphology in E. coli following exposure to ciprofloxacin (CIP), a fluoroquinolone antibiotic. We show that CIP induces a stepwise DNA reorganization, culminating in a highly dense nucleoid structure at midcell-a process we term DNA supercompaction. This phenomenon occurred also with other genotoxic agents. Live-cell imaging revealed that RecN, a structural maintenance of chromosomes (SMC)-like protein, is required for DNA supercompaction, and that RecN's dynamics and activity in this response depend on RecA. Additionally, RecN and RecA frequently colocalized at nucleoid-associated positions. We suggest that RecN and RecA play active roles in DNA supercompaction following severe DNA damage, that their interplay is part of a prompt universal survival response to DNA double-strand breaks in E. coli, and that the extent of the compaction response depends on the DNA damage severity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506e392d68ea028b658f17b6f901be5aeef9ee4e" target='_blank'>
              RecN and RecA orchestrate an ordered DNA supercompaction response following ciprofloxacin-induced DNA damage in Escherichia coli.
              </a>
            </td>
          <td>
            Krister Vikedal, S. B. Ræder, Ida Mathilde Marstein Riisnæs, Magnar Bjørås, J. Booth, Kirsten Skarstad, Emily Helgesen
          </td>
          <td>2025-05-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Tumors are characterized by global changes in epigenetic modifications such as DNA methylation and histone modifications that are functionally linked to tumor progression. Accordingly, several drugs targeting the epigenome have been proposed for cancer therapy, notably, histone deacetylase inhibitors (HDACi) such as vorinostat and DNA methyltransferase inhibitors (DNMTi) such as zebularine. However, a fundamental challenge with such approaches is the lack of genomic specificity, i.e., the transcriptional changes at different genomic loci can be highly variable, thus making it difficult to predict the consequences on the global transcriptome and drug response. For instance, treatment with DNMTi may upregulate the expression of not only a tumor suppressor but also an oncogene, leading to unintended adverse effect. Methods Given the pre-treatment transcriptome and epigenomic profile of a sample, we assessed the extent of predictability of locus-specific changes in gene expression upon treatment with HDACi using machine learning. Results We found that in two cell lines (HCT116 treated with Largazole at eight doses and RH4 treated with Entinostat at 1µM) where the appropriate data (pre-treatment transcriptome and epigenome as well as post-treatment transcriptome) is available, our model distinguished the post-treatment up versus downregulated genes with high accuracy (up to ROC of 0.89). Furthermore, a model trained on one cell line is applicable to another cell line suggesting generalizability of the model. Conclusions Here we present a first assessment of the predictability of genome-wide transcriptomic changes upon treatment with HDACi. Lack of appropriate omics data from clinical trials of epigenetic drugs currently hampers the assessment of applicability of our approach in clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a8c2bc7236dcb12e89963dabb5f55fdfd61138" target='_blank'>
              Predicting gene expression changes upon epigenomic drug treatment
              </a>
            </td>
          <td>
            P. Agrawal, Vishaka Gopalan, M. A. Rumi, Sridhar Hannenhalli
          </td>
          <td>2025-05-02</td>
          <td>F1000Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is the leading cause of cancer-related mortality worldwide owing to the lack of effective and early diagnostic tools and therapeutic approaches. DNAJC7, a member of the DnaJ heat shock family, is crucial in protein folding and stability; however, its specific functions and mechanisms in HCC remain unclear. Objective This study aimed to explore the role of DNAJC7 in HCC progression and evaluate its potential clinical significance as a prognostic marker. Methods Public databases (TCGA, ICGC, GEO, and TIMER) were used to assess DNAJC7 expression, correlations with clinical parameters, and related signaling pathways. Proliferation, migration, invasion, and cell cycle assays were performed to evaluate the function of DNAJC7 in HCC. Immune infiltration and associations with checkpoint proteins were analyzed using TIMER, and a Gene Set Enrichment Analysis (GSEA) was used to explore enriched pathways. Results DNAJC7 expression was higher in HCC tissues than in adjacent normal tissues and was associated with advanced malignancy and poor prognosis, including a lower overall survival, progression-free survival, and disease-free survival. DNAJC7 knockdown resulted in reduced malignant behavior of HCC cells, leading to S-phase cell cycle arrest. Increased DNAJC7 expression was associated with immune cell infiltration and the presence of immunological checkpoint molecules, including CTLA4 and PD-1. GSEA highlighted the activation of key pathways, including WNT signaling and cell cycle regulation. Conclusion DNAJC7 regulates tumor cell proliferation, migration, invasion, and immune evasion by acting as an oncogene in HCC. It can serve as a diagnostic and prognostic biomarker and potential treatment target for HCC. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06202-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5d2c109594fa970efe3bba40f733c0078c3d83a" target='_blank'>
              Increased expression of DNAJC7 promotes the progression of hepatocellular carcinoma by influencing the cell cycle and immune microenvironment
              </a>
            </td>
          <td>
            Jiaxing Chen, Zhizhao Yang, Yongqiang Cui, Zhilei Zhao, D. Deng, Zhihao Fu, Xiao Zhang
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Micronuclei exhibit defective proteomes rendering their chromatin vulnerable to fragmentation. This fragmentation process, known as chromothripsis, promotes tumorigenesis by catalysing the activation of oncogenes and the silencing of tumor suppressors. With this role in mind, micronuclei serve as promising targets for therapeutic intervention. This review will explore recent discoveries regarding how micronuclei form, their function in catalysing chromothripsis and how chromothripsis provides a selective advantage for cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ac4a5ea94375865c4d842a9f93ca345cb31b4e0" target='_blank'>
              Cancer cells' chamber of secrets: the link between micronuclei, chromothripsis and malignancy.
              </a>
            </td>
          <td>
            Truman McNeil, Sweta Sikder, Yamini Dalal
          </td>
          <td>2025-05-01</td>
          <td>Open biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Circulating cell-free DNA (cfDNA) has been widely used as a prognostic marker for different cancers. Objective: In this study, we used 30 cfDNA samples from oral squamous cell carcinoma (OSCC), 199 public OSCC samples, and 192 normal samples to study various correlation factors that could improve the early-stage diagnostics and/or prognosis of OSCC. Methods: The statistical correlation between healthy and OSCC patients was done and deep sequencing analyses was performed to study various genomic alterations likes copy number variation (CNV), and single nucleotide variants (SNVs), gene fusion and genomic integration of viruses. Results: We found that the OSCC patient cfDNA concentration can serve as an indicator of tumor stage, malignancy, and survival prognosis. Deep genome sequencing of cfDNA revealed genomic alterations, such as CNVs, fusion genes, and viral integrations. The CNV analysis suggested a correlation with amplification and deletion in chromosomes at loci 1q, 2q, 3p, 3q, and chromosome 8 at loci q22. Moreover, at these loci, amplification of TP53, PIK3CA, and other genes related to keratinization in OSCC patients was observed. In addition, we identified a novel abundant fusion gene, TRMO-TRNT1 ‘chimera’, in seven high-grade tumor samples. The parental genes of this chimera, TRMO and TRNT1, are known to play roles in tRNA modification and DNA repair, respectively. We have identified SNVs in our OSCC cohort. Some of these SNVs, like KMT2C, MUC3A, and MUC6, have been identified as common cases in different cancer populations. Finally, we detected contigs integrations of human papillomavirus, simian virus, and enterovirus in the OSCC samples, which may point to the potential causes of OSCC. Conclusions: Our results indicate that the liquid biopsy technique may thus serve as a sensitive tool to study OSCC patient genomic alterations by exploring cfDNA circulating in the plasma, providing an easy-to-use blood test in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c52e96cadb3f868e649f16559709defe911b5c" target='_blank'>
              Applications for Circulating Cell-Free DNA in Oral Squamous Cell Carcinoma: A Non-Invasive Approach for Detecting Structural Variants, Fusions, and Oncoviruses
              </a>
            </td>
          <td>
            M. Bhattacharya, D. Yaniv, D. P. D’Souza, Eyal Yosefof, Sharon Tzelnick, R. Detroja, Tal Wax, A. Levy-Barda, G. Baum, A. Mizrachi, G. Bachar, Milana Frenkel Morgenstern
          </td>
          <td>2025-06-06</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct “fragmentomic” characteristics that are related with their cellular origins. Despite the wealth of low-pass whole genome sequencing data from recent cancer liquid biopsy studies, somatic variants within this data have often been overlooked without detailed characterization. Here, we reported that in low-pass sequencing data, somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, our investigation into cfDNA fragmentomics showed that even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like traits, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilized artificial intelligence to integrate genomic, fragmentomic, and epigenomic features, developing diagnostic models named FreeSV (Fragmentomic and epigenomic examinations of Somatic Variants in cfDNA) and FreeSV-m. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features were also included, the AUCs of FreeSV-m model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f64ed6f9ef46f8ac541b4cf9a70df9227b7d0f90" target='_blank'>
              Cancer-like fragmentomic characteristics of somatic variants in cell-free DNA
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Wanqiu Wang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, Anthony P Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ea5280910649de8b63e584d7975f22a13b00b4" target='_blank'>
              Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code
              </a>
            </td>
          <td>
            Marina Murillo-Recio, Marina Salvadores, Adrián Gabriel Torres, Lina Tsapanou, Aina Vaquer Picó, F. Supek, L. R. de Pouplana
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The conceptional shifts on genome organization and hereditary variability occurred during transition from classical mendelian to current mobile or dynamic genetics. The main changed premises of this transition are firstly presented in detail. Mendelian genetics mainly conceived genome as the set of chromosomes with of all genes. Now genome semantics is changed. It comprises entire hereditary constitution of the cell, including both structural and dynamic aspects of coding, storage and transfer of species-specific information. There are three kinds of heritable changes: mutations, variations and epigenetic alterations. It is reasonable to discriminate in the genome two subsystems: Obligate genetic elements (OGE) and Facultative genetic elements (FGE). FGEs comprise various kinds of repeated DNA, mobile elements, amplicons, inserted viral and foreign DNA, B-chromosomes and cytobionts. FGEs are predominant genome content of many plants. The number and cell topography of FGEs are different in different cells/tissues and most eukaryote individuals. Changes in the structure or order of OGEs correspond to classical mutations. Various changes in FGEs it is reasonable to call variations. Facultative elements and their variations are the first genomic reaction on biotic and environmental challenges. Together with epigenetic alterations they implement the operational genomic memory. Three template genome processes Replication, Transcription, Translation and three basic genetic processes – Repair, Recombination and Segregation are capable to facultative expression according to principle: the unity of the whole and freedom of the parts. This is the essence of the presented generalized concept of the genome organization and hereditary variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08181d3ed9f40145f41c1bd8a5154dfef8c50437" target='_blank'>
              Beyond Mendel and Morgan to the dynamic genome
              </a>
            </td>
          <td>
            M. D. Golubovsky
          </td>
          <td>2025-05-11</td>
          <td>Vavilovia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GB) is a highly aggressive brain tumor resistant to chemoradiotherapy, largely due to glioma stem-like cells (GSCs) with robust DNA damage repair capabilities. Here we reveal that GSCs enhance their DNA repair capacity by activating non-homologous end-joining (NHEJ) through upregulation of the apoptosis antagonizing transcription factor (AATF), thereby promoting therapeutic resistance in GB. AATF interacts with XRCC4, a core NHEJ subunit, preventing its degradation via ubiquitin-mediated proteasomal processes. Upon DNA damage, AATF undergoes phosphorylation at Ser189 by ATM, leading to its dissociation from XRCC4 and rapid recruitment of XRCC4 to DNA break sites for efficient NHEJ repair. Moreover, AATF depletion or deficient AATF phosphorylation impedes NHEJ in GSCs, sensitizing GB xenografts to chemoradiotherapy. Additionally, elevated levels of AATF inform poor prognosis in GB patients. Collectively, our findings unveil a crucial role of AATF in XRCC4-mediated NHEJ repair, and underscore targeting AATF as a potential strategy to overcome GB resistance to chemoradiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23caccae8c29a0e0c13c3a812c074646bd230d3d" target='_blank'>
              Elevated nonhomologous end-joining by AATF enables efficient DNA damage repair and therapeutic resistance in glioblastoma
              </a>
            </td>
          <td>
            Lanjuan Mi, Yan Cai, Ji Qi, Lishu Chen, Yuanyuan Li, Songyang Zhang, H. Ran, Qinghui Qi, Cheng Zhang, Huiran Wu, Shuailiang Cao, Haohao Huang, Dake Xiao, Xinzheng Wang, Bohan Li, Jiong Xie, Fangye Li, Qiuying Han, Qiulian Wu, Tao Li, Ailing Li, Jeremy N. Rich, T. Zhou, Jianghong Man
          </td>
          <td>2025-05-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="G-quadruplex (G4) is a noncanonical nucleic acid secondary structure self-assembled by guanine-rich sequences. Recent studies have not only revealed the key role of G4 in gene regulation, DNA replication, and telomere maintenance but also showed that it plays a core role in regulating the tumor immune microenvironment. G4 participates in tumor immune escape and the inhibition of immune response by regulating immune checkpoint molecules, cytokine expression, immune cell function, and their interaction network, thus significantly affecting the effect of tumor immunotherapy. This article systematically reviews the molecular mechanism of G4 in tumor immune regulation, especially gastrointestinal tumors, and explores the potential and application prospects of G4-targeted drug strategies in improving anti-tumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a58fcc7d1212e9631e420ea9bf16080447b4f9a" target='_blank'>
              G-Quadruplexes in Tumor Immune Regulation: Molecular Mechanisms and Therapeutic Prospects in Gastrointestinal Cancers
              </a>
            </td>
          <td>
            Yunxia Zhou, Difei Xu, Ying Zhang, Huaixiang Zhou
          </td>
          <td>2025-04-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Eukaryotic centromeres highlight the remarkable plasticity of eukaryotic chromosomes through their conserved functionality and sequence divergence. Holocentric chromosomes, where centromere activity is distributed along the entire chromosome length, offer a unique model for investigating the molecular mechanisms underlying adaptive evolution between centromeres and chromosomes. In this review, we summarise and speculate on the multiple changes and prerequisites potentially involved in the evolution of holocentromeres. The interplay between environmental factors, chromosomal rearrangements, and centromere plasticity drives the transition from regional to holocentric characteristics. The centromeric histone H3 (CenH3) protein mediates neocentromere formation by recognising non-centromeric chromosomal regions with appropriate AT content, thereby facilitating chromosome restructuring in the transition from regional to holocentric chromosomes. Dynamic changes in repetitive sequences provide functional sites for centromere assembly, chromosomal recombination and repair and centromere spreading and maturation. Epigenetic modifications maintain functional coordination among multiple centromeric units by modulating chromatin states, CenH3 localisation, and kinetochore assembly. This review provides a comprehensive framework for understanding the evolutionary mechanisms of holocentromeres derived from monocentromere and offers insights into the design of artificial centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34691ddef62fc1465f67e23ac70d8368228f19a" target='_blank'>
              Eukaryotic Centromere Remodeling: Plasticity, Dynamics, and Holocentromere Formation.
              </a>
            </td>
          <td>
            Dan Yang, Zhaoxin Xiao, Ke Li, Jiayi Hou, Fengfeng Zhang, Jianjun Qiao, Ning Li, Mingzhang Wen
          </td>
          <td>2025-05-27</td>
          <td>Plant, cell & environment</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Many anticancer drugs are ineffective in tumors that have functional DNA repair mechanisms. In contrast, trabectedin, a tetrahydroisoquinoline alkaloid marine natural product, stands out as it is more lethal to cancer cells with active DNA repair, particularly transcription-coupled nucleotide excision repair (TC-NER), making it an intriguing alternative to standard chemotherapeutic agents. To optimize trabectedin's use in precision oncology, it is essential to understand how its toxicity depends on TC-NER. In this study, we reveal that incomplete TC-NER of trabectedin-DNA adducts generates persistent single-strand breaks (SSBs). These adducts are found to obstruct the second of two sequential NER-mediated DNA incisions. By mapping the 3'-hydroxyl groups of SSBs resulting from the first NER incision at trabectedin-DNA adducts, we achieve genome-wide visualization of TC-NER. Our findings show that trabectedin-induced SSBs predominantly occur in the transcribed strands of active genes, accumulating near transcription start sites. This work provides new insights into how trabectedin can be leveraged for targeted cancer therapies and for studying TC-NER and transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0f5289cf2f72912257662ee75e14bb3d65e6cf" target='_blank'>
              When DNA Repair Backfires - Trabectedin Induces DNA Breaks in Active Genes.
              </a>
            </td>
          <td>
            V. Takhaveev, Kook Son, Visesato Mor, Hobin Yu, Emma Dillier, Nicola Zilo, N. Püllen, Dmitri Ivanov, Helle D. Ulrich, Shana J. Sturia, Orlando D. Schärer
          </td>
          <td>2025-04-30</td>
          <td>Chimia</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Ewing sarcoma (EWS) is an aggressive cancer in adolescents and young adults with frequent relapse rates and poor outcomes in recurrent or metastatic cases. Schlafen family member 11 (SLFN11) gene is associated with the sensitivity to DNA-damaging agents (DDAs). The knockout of SLFN11 is associated with acquired chemoresistance in both cell lines and preclinical models. Here, we aimed to elucidate the metabolic underpinnings of SLFN11-loss associated chemoresistance in patient derived cell lines of EWS. Our integrated transcriptomic and metabolomic analyses revealed downregulation of mitochondrial glycerol-3-phosphate dehydrogenase 2 (GPD2) gene, which was accompanied by the upregulation of glycerophospholipid (GPL) biosynthesis pathway. Further, therapeutic targeting of lipid synthesis with the glycerol-3-phosphate acyltransferase 1 (GPAT1) inhibitor (FSG67) enhanced the efficacy of the DDA (SN-38) in SLFN11−/− cells. These findings indicate that SLFN11 loss-mediated chemoresistance can be targeted by blocking GPL biosynthesis in addition to DDA administration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b87872e0edc8f5c332129087a9e413795323d43d" target='_blank'>
              SLFN11 Loss-Induced Chemoresistance is Associated with Overexpression of Glycerophospholipid Biosynthesis in Ewing Sarcoma
              </a>
            </td>
          <td>
            Kasturee Chakraborty, Ritambhar Burman, Saharsh Satheesh, Matthew Kieffer, Chandni Karuhatty, Zuo-Fei Yuan, Haiyan Tan, Ankhbayar Lkhagva, Anthony A. High, Xusheng Wang, Alaa Refaat, Weixing Zhang, Yaxu Wang, Yiping Fan, M. M. Babu, A. Shelat, Elizabeth Stewart, Michael A Dyer, P. Bagga
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b94228db7f216ff8d9ace96b3b303dbe02cf6e0" target='_blank'>
              Circular RNA rising star: new findings of circPRMT5 in cancer.
              </a>
            </td>
          <td>
            DuanZhong Chen, Caiyan Yin, Chao Li, Jun He, Peng Wu
          </td>
          <td>2025-06-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Historically, DNA sequence mutability has been considered relatively uniform and low in tumors with chromosomal instability (CIN), based on the assumption that high mutability would be detrimental in karyotypically aberrant contexts. Recent in silico analyses have challenged this view, suggesting some heterogeneity in mutation rates across CIN tumors; however, these predictions lack experimental validation. It also remains unclear how the intertumor variability of mutation rates compares to intratumor diversification and evolves along disease progression, whether mutation rates are functionally relevant in CIN cancers, and which mutational processes shape mutational accrual during CIN tumor onset and evolution. To address these gaps, we performed mutation accumulation experiments using clonal populations of patient-derived tumoroids from seven CIN, microsatellite-stable colorectal cancers (CRCs), and one microsatellite-unstable CRC. Each tumor exhibited a distinctive mutation rate footprint that was conserved among different clones from the same ancestor. In contrast, mutation rates diverged markedly across different tumors, with variations in magnitude within microsatellite-stable tumors as prominent as those distinguishing them from microsatellite-unstable tumors. New mutations reflected mutational processes associated with defective DNA replication and repair, which were not detected in normal tissues. Last, both mutation accumulation assays and high-depth whole-exome sequencing of subclonal variants showed higher mutation rates in metastatic lesions compared with matched primary tumors, suggesting positive selection for cells with increasing mutability during cancer dissemination. By providing an empirical assessment of mutation rates in human cancer, our data delineate heterogeneity, heritability, and progression-associated evolvability of DNA mutational instability as hallmarks of microsatellite-stable CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c57088e09b71e7ad2e34a58e69bb450d1b43d18" target='_blank'>
              Heterogeneity and evolution of DNA mutation rates in microsatellite stable colorectal cancer.
              </a>
            </td>
          <td>
            Elena Grassi, V. Vurchio, George D. Cresswell, Irene Catalano, B. Lupo, F. Sassi, Francesco Galimi, Sofia Borgato, Martina Ferri, Marco Viviani, Simone Pompei, Gianvito Urgese, Bingjie Chen, E. Zanella, F. Cottino, Mariangela Russo, G. Mauri, F. Pietrantonio, M. G. Zampino, L. Lazzari, S. Marsoni, A. Bardelli, M. Lagomarsino, A. Sottoriva, L. Trusolino, Andrea Bertotti
          </td>
          <td>2025-05-21</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Ageing is a gradual biological process marked by a decline in physiological function, increasing susceptibility to disease, and mortality. Transposable elements (TEs) are repetitive DNA sequences capable of moving within the genome and thus potentially inducing mutations and disrupting normal cellular functions. Their mobile nature contributes to genomic variation, as transposition events can alter gene expression, chromosome structure, and the epigenetic landscape. To mitigate TE-induced damage, cells rely on epigenetic mechanisms, such as DNA methylation, histone modifications, and small RNAs, to repress TE activity. However, these silencing mechanisms become less effective with age, leading to increased TE activation. This review explores the dual role of TEs as both a cause and consequence of ageing, suggesting a complex relationship between TEs and the ageing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b56d174d42a8f2b0813e61df3277c11215871" target='_blank'>
              Exploring the Relationship of Transposable Elements and Ageing: Causes and Consequences
              </a>
            </td>
          <td>
            Miriam Merenciano, Anaïs Larue, Chloé Garambois, William Vilas Boas Nunes, Cristina Vieira
          </td>
          <td>2025-05-15</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background NDC80, a crucial component of the kinetochore, plays a pivotal role in regulating cell mitosis. Recent studies reported that NDC80 regulates the proliferation of cancer cells and may be related to poor prognosis in cancer. However, the biological function and mechanism of NDC80 in esophageal squamous cell carcinoma (ESCC) have not yet been elucidated. In this study, we analyzed the expression and mutation of NDC80 in ESCC tissue and assessed its impact on ESCC cells in vitro. Methods A total of 52 ESCC tissues samples were collected and NDC80 gene status was analyzed by whole exome sequence. shRNA was used to knockdown NDC80 expression in two ESCC cell lines TE1 and ECA109 by targeting silence NDC80 gene. Cellular proliferation, apoptosis, migration, and invasion were examined. Results Copy number variants (CNVs) of NDC80 were detected in 40.4% (21/52) cancer tissues. Genes with local CNVs values greater than 0.3 were classified as “gain”. 23.1% (12/52) of cases showed CNVs gain and 58.3% (7/12) of which with genotype of AAB. Compared with patients of other NDC80 CNVs status, CNVs gain patients had relatively shorter overall survival, however with no statistic difference (19.9 ± 0.8 months vs. 30.9 ± 5.1 months, p = 0.124). Single nucleotide polymorphism mutation of NDC80 was detected in only 5.8% (3/52) cancer tissues including two previous reported mutation rs2271754 and rs6506019, and one novel SNP of C/A-T/G in 2,577,492, GRCh38.p13 intron variant in one case. The knockdown of NDC80 in vitro assays significantly suppressed cellular proliferation and induced increased apoptosis in both TE1 and ECA109 cells. Furthermore, the migration and invasion ability of both TE1 and ECA109 cells were also attenuated upon knockdown of NDC80. Conclusion CNVs were identified as the predominant genetic variation of NDC80 gene in ESCC patients. In vitro assays suggested that NDC80 was not only involved into proliferation and apoptosis but also migration and invasion of ESCC cells. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-04048-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16d73cd77e8485bddfd9d633b8f7422ebd5f99a7" target='_blank'>
              Expression of kinetochore component NDC80 promotes esophageal squamous cell carcinoma cells proliferation and migration
              </a>
            </td>
          <td>
            Youwu Shi, Jing Sun, Feng Du, Zhi-wei Sun, Ying Yang, Jing Yu, Yanjie Xiao, Xiaoyu Sun, Wen Zhang, Hongkun Zheng, Xiaodong Zhang, Jun Jia
          </td>
          <td>2025-06-05</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PURPOSE
A combination of two HER2-directed antibodies, pertuzumab and trastuzumab (P + T), has antitumor activity in HER2-positive colorectal cancer. Although liquid biopsies are increasingly being used in clinical oncology, the association between tumor and ctDNA ERBB2 status and ctDNA monitoring for early response and resistance are unknown.


PATIENTS AND METHODS
Eighty-five patients with ERBB2-amplified and/or -overexpressed colorectal cancer were treated with P + T in the MyPathway trial; 42 had ctDNA testing at cycle (C) 1 day (D) 1, and 38 had longitudinal plasma tested for ctDNA. We analyzed the ctDNA versus tissue ERBB2 concordance, genomic co-alterations, and ctDNA dynamics and association with response.


RESULTS
Forty-one (98%) of 42 patients had genomic alterations detected in ctDNA at C1D1, and 29 (69%) had ERBB2 amplification in ctDNA. There was a strong correlation between the ERBB2 copy number on next-generation sequencing in tissue and C1D1 ERBB2 ctDNA copy number. Thirty-seven percent achieved a molecular response by C3D1 on P + T, which was associated with prolonged progression-free survival and overall survival. CDKN2A and KRAS mutations were associated with shorter overall survival, and a trend was seen with PIK3CA mutations. Several emerging co-alterations were identified in ctDNA at progression, including in the MAPK and PI3K pathways and other tyrosine receptor kinases.


CONCLUSIONS
ctDNA can detect ERBB2 amplification in many, but not all, patients with ERBB2 amplification detected in tumor samples. ctDNA molecular response was associated with better survival, and ctDNA co-alterations may offer insights into mechanisms of intrinsic and acquired resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4a3dd9af2fb367f227e204e5af43163baadfbf" target='_blank'>
              ctDNA Analysis in ERBB2-Amplified Colorectal Cancer: Biomarker Analysis of the MyPathway Trial.
              </a>
            </td>
          <td>
            Funda Meric-Bernstam, K. Raghav, C. Sweeney, C. Swanton, D. Spigel, R. Bose, Howard A Burris, C. Friedman, Carin R. Espenschied, Jessica Grindheim, J. Malato, Katja Schulze, R. Price, R. Kurzrock
          </td>
          <td>2025-06-13</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Homology, or relationship among characters by common descent, has been notoriously difficult to assess for many morphological features, and cell types in particular. The ontogenetic origin of morphological traits means that the only physically inherited information is encoded in the genomes. However, the complexity of the underlying gene regulatory network and often miniscule changes that can impact gene expression, make it practically impossible to postulate a clear demarcation line for what molecular signature should "define" a homologous cell type between two deeply branching animals. In this Hypothesis article, we propose the use of the recently characterized irreversible genomic states, that occur after chromosomal and sub-chromosomal mixing of genes and regulatory elements, to dissect regulatory signatures of each cell type into irreversible and reversible configurations. While many of such states will be non-functional, some may permanently impact gene expression in a given cell type. Our proposal is that such evolutionarily irreversible, and thus synapomorphic, functional genomic states can constitute a criterion for the timing of the origin of deep evolutionary cell type homologies. Our proposal thus aims to close the gap between the clearly defined homology of the individual genomic characters and their genomic states to the homology at the phenotypic level through the identification of the underlying evolutionarily irreversible and regulatory linked states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1c7265ffd0750ec9e349f309b4f00f20641d114" target='_blank'>
              The application of irreversible genomic states to define and trace ancient cell type homologies
              </a>
            </td>
          <td>
            Oleg Simakov, Günter P. Wagner
          </td>
          <td>2025-05-03</td>
          <td>EvoDevo</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Mammalian cells have numerous DNA repair pathways to repair lesions generated by replication errors, metabolism, and exogenous agents. Cells can sense and respond to DNA damage within seconds, suggesting that there is a highly effective sensor of lesions although the mechanistic details are unclear. The DNA damage response in mammalian cells results in a localized transient de-condensation of chromatin, loss of linker histones and the recruitment of DNA repair proteins such as PARP1 and chromatin remodelers. Results Here we investigated the interactions between poly(ADP-ribose) polymerase-1 (PARP1), the linker histone H1.0 and linker histone chaperone Prothymosin α (PTMA). Using H1.0 tagged with a photoconvertible fluorescent protein, we observed a significant increase in the initial rate of exit of H1.0 from regions of chromatin containing microirradiation-induced DNA lesions. Surprisingly, this was also seen in Parp1−/− cells but not in stable cell lines with homozygous null mutations in the PTMA gene (Ptma−/−). The recruitment of PARP1 to damaged DNA was inhibited by overexpression of a mutant of H1.0 with a tighter chromatin-binding affinity or by reduced expression of PTMA. Relative to the wild type, Ptma−/− cell lines displayed increased sensitivity to DNA-damaging agents. Conclusion We suggest that DNA damage alters the interaction of H1.0 with the nucleosome to allow the chaperone PTMA to bind and promote release of linker histones thereby initiating the local chromatin de-condensation necessary for the efficient recruitment of repair proteins such as PARP1. In this context linker histones may serve as in situ “sensors” of DNA damage. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00599-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b434110b02c9c576f6e6173ea01f3a42ac50b9" target='_blank'>
              The linker histone chaperone Prothymosin α (PTMA) is essential for efficient DNA damage repair and the recruitment of PARP1
              </a>
            </td>
          <td>
            Ciara A McKnight, Mary E Graichen, Eric M George, David T. Brown
          </td>
          <td>2025-06-05</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593951b80dada0fabedb9d7b8680116321df128" target='_blank'>
              Development of Paclitaxel Resistance in Triple-Negative Breast Cancer Is Associated with Extensive DNA Methylation Changes That Are Partially Reversed by Decitabine
              </a>
            </td>
          <td>
            Lenka Trnkova, M. Burikova, A. Soltysova, A. Ficek, Jana Plava, Andrea Cumova, Lucia Rojikova, Kristina Jakic, Eva Sedlackova, Boris Tichý, V. Bystrý, F. Busato, Yimin Shen, M. Matúšková, L. Kucerova, J. Tost, S. Miklikova, M. Cihova, V. Buocikova, B. Smolková
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Significance Aneuploidy is a major cause of birth defects, a hallmark of cancer, and a reflection of the dosage sensitivity of genes involved in quantitative traits. Most studies on how aneuploidy affects gene expression have examined single genomic regions at a time. Here, we examined combination trisomies, combination monosomies, and combinations of trisomy for one region and monosomy for another. The effects on global gene expression were found to be cumulative, rebalanced, or in some cases multiplicative. Most aneuploidies alter the total transcriptome size. The results have implications for the stoichiometric control of gene expression, the progression of aneuploidy in cancer, the evolution of sex chromosomes, the role of the triploid block in polyploid evolution, and the control of quantitative traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/644edaec4bdaa378dc76678350b3c6e4610cba95" target='_blank'>
              Global modulation of gene expression and transcriptome size in aneuploid combinations of maize
              </a>
            </td>
          <td>
            Hua Yang, Vincent A. Brennan, Zhi Gao, Jian Liu, Frimpong Boadu, Jianlin Cheng, J. Birchler
          </td>
          <td>2025-05-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca7843e2f4903653396c94ffa8fe3fdaa395232" target='_blank'>
              ZNF280A links DNA double-strand break repair to human 22q11.2 distal deletion syndrome.
              </a>
            </td>
          <td>
            Thomas L Clarke, Hyo Min Cho, Ilaria Ceppi, Boya Gao, Tribhuwan Yadav, Giorgia G Silveira, Ruben Boon, Barbara Martinez-Pastor, Nana Yaa A Amoh, Belen Machin, Tiziano Bernasocchi, Dua Ashfaq, Josefina Mendez, Zeeba Kamaliyan, José Del Río Pantoja, Giuliana Sardi Rogines, Blaine T Crowley, D. McGinn, V. Giunta, Oanh Tran, E. Zackai, Li Lan, Lee Zou, B. S. Emanuel, Donna M. McDonald-McGinn, Petr Cejka, Raul Mostoslavsky
          </td>
          <td>2025-06-16</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy with a 5-year overall survival (OS) rate of approximately 12%. More than 90% of PDAC patients harbor oncogenic mutations in the Kirsten rat sarcoma viral homolog (KRAS) gene. MRTX1133 (MRTX), a novel inhibitor of KRASG12D (the most common KRAS mutation found in pancreatic and colon cancers) has shown promise as a therapeutic agent. To address reported resistance to MRTX, we adapted our anti-leukemia co-targeting strategy and evaluated a combination of MRTX and Bedaquiline (BED), an FDA-approved inhibitor of mitochondrial ATP production, in in vitro human PDAC models. The combination of MRTX and BED demonstrated enhanced cytotoxic effects by disrupting all 11 genes within the DNA helicase family (CMG complex: CDC45-MCM-GINS), which are essential for initiating DNA replication and regulating cell cycle progression. Notably, real-world data analysis from Caris Life Sciences and NCI-TCGA database revealed that low transcriptomic expression of the DNA helicase CMG complex was significantly associated with prolonged survival (e.g., low CDC45 expression and low GINS2 expression with greater than 8 months longer overall survival) in PDAC patients with KRASG12 mutations (N = 9,717; P < 0.00001). However, this combination therapy also triggered strong pro-survival nuclear reprogramming. This effect was mediated by significant genetic activation of an NFκB2-DDIT (DNA damage-induced transcript) axis, which supported tumor chromosomal integrity and DNA repair mechanisms. To overcome NFκB2-driven resistance mechanisms, we explored a triple-targeting strategy that addresses metabolic and genomic plasticity in addition to actively intercepting cell division. This approach combines MRTX1133, Bedaquiline, and the NFκB2 inhibitor SN52, offering a novel therapeutic avenue to treat aggressive pancreatic cancer and potentially improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00669-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecf14d3f8785e3a0d9d03809564ba14faf0a6c7" target='_blank'>
              Targeting DNA helicase CMG complex and NFκB2-driven drug-resistant transcriptional axis to effectively treat KRASG12D-mutated pancreatic cancer
              </a>
            </td>
          <td>
            Jeffrey Xiao, Joshua Kim, Brandon Park, David J. Baylink, Cedric Kwon, Victoria Tran, Scott Lee, Kevin Codorniz, Laren D Tan, Pamela Lobo Moreno, Amy Schill-Depew, S. Mirshahidi, David de Semir, Diana Hanna, Kiran Naqvi, H. Cao, Chien-Shing Chen, Joanne Xiu, H. Lenz, Hamid Mirshahidi, Mark E. Reeves, Yi Xu
          </td>
          <td>2025-05-26</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Addiction to oncogene-rewired transcriptional networks is a therapeutic vulnerability in cancer cells, underscoring a need to better understand mechanisms that relay oncogene signals to the transcriptional machinery. Here, using human and mouse T cell acute lymphoblastic leukemia (T-ALL) models, we identify an essential requirement for the endosomal sorting complex required for transport protein CHMP5 in T-ALL epigenetic and transcriptional programming. CHMP5 is highly expressed in T-ALL cells where it mediates recruitment of the coactivator BRD4 and the histone acetyl transferase p300 to enhancers and super-enhancers that enable transcription of T-ALL genes. Consequently, CHMP5 depletion causes severe downregulation of critical T-ALL genes, mitigates chemoresistance and impairs T-ALL initiation by oncogenic NOTCH1 in vivo. Altogether, our findings uncover a non-oncogene dependency on CHMP5 that enables T-ALL initiation and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768bb462ad3f5f7bfdc0d91a1f7eb42525d3db0" target='_blank'>
              The ESCRT protein CHMP5 promotes T cell leukemia by enabling BRD4-p300-dependent transcription
              </a>
            </td>
          <td>
            Katharine Umphred-Wilson, Shashikala Ratnayake, Qianzi Tang, Rui Wang, S. Chaudhary, Devaiah N Ballachanda, Josephine Trichka, Jan Wisniewski, Lan Zhou, Qing-rong Chen, D. Meerzaman, Dinah S. Singer, S. Adoro
          </td>
          <td>2025-05-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Although chronic inflammation is implicated in the pathogenesis of diffuse large B‐cell lymphoma (DLBCL), the mechanisms responsible are unknown. We demonstrate that the overexpression of the collagen receptor, DDR1, correlates with reduced expression of spindle checkpoint genes, with three transcriptional signatures of aneuploidy and with a higher frequency of copy number alterations, pointing to a potential role for DDR1 in the acquisition of aneuploidy in DLBCL. In support of this, we found that collagen treatment of primary germinal centre B cells transduced with DDR1, not only partially recapitulated the aberrant transcriptional programme of DLBCL but also downregulated the expression of CENPE, a mitotic spindle that has a crucial role in preventing chromosome mis‐segregation. CENPE expression was also downregulated following DDR1 activation in two B‐cell lymphoma lines and was lost in most DDR1‐expressing primary tumours. Crucially, the inhibition of CENPE and the overexpression of a constitutively activated DDR1 were able to induce aneuploidy in vitro. Our findings identify a novel mechanistic link between DDR1 signalling and chromosome instability in B cells and provide novel insights into factors driving aneuploidy in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c675893b8988879101a50cb9c9f53e9ccb29a3" target='_blank'>
              The Overexpression of Collagen Receptor DDR1 is Associated With Chromosome Instability and Aneuploidy in Diffuse Large B‐Cell Lymphoma
              </a>
            </td>
          <td>
            Sandra Margielewska-Davies, Matthew Pugh, Eszter Nagy, Ciara I Leahy, M. Ibrahim, É. Fennell, Aisling M Ross, J. Bouchal, L. Lupino, M. Care, R. Tooze, Gary Reynolds, Zbigniew Rudzki, Wenbin Wei, William Simmons, Vikki Rand, Kelly Hunter, John J Reynolds, Grant S Stewart, Kateřina Bouchalová, Iona J Douglas, K. Vrzalikova, Paul G Murray
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromodomain Helicase DNA-binding protein 1-Like (CHD1L) is a chromatin remodeling enzyme increasingly recognized as an oncogenic factor promoting tumor progression and metastatic potential by orchestrating transcriptional programs that drive epithelial-mesenchymal transition (EMT), cytoskeletal remodeling, and metastatic dissemination. In parallel, CHD1L has emerged as a master regulator of tumor cell survival by regulating DNA damage response and repair and enforcing G1 cell cycle progression. Furthermore, CHD1L plays a key role in immune evasion pathways by regulating signaling cascades and by suppressing both apoptotic and non-apoptotic cell death. In particular, CHD1L is a key suppressor of PARthanatos, a caspase-independent mechanism triggered by poly(ADP-ribose) (PAR) polymer fragmentation and apoptosis-inducing factor (AIF) activation. By regulating SPOCK1, MDM2, and TCTP, CHD1L further supports survival under cellular stress. Its overexpression correlates with metastasis, therapy resistance, and poor prognosis across many solid tumors. This review covers CHD1L’s structure, oncogenic functions, and developmental origins, and highlights emerging therapeutic strategies that target CHD1L as a druggable vulnerability in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6296c3dbcbe7b1d1919c8debad6aee4011adbed2" target='_blank'>
              CHD1L in cancer and beyond: structure, oncogenic functions, and therapeutic potential
              </a>
            </td>
          <td>
            Sophia Clune, Paul Awolade, Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-05-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The nucleolus is a membraneless organelle and an excellent stress sensor. Any changes in its architecture or composition lead to nucleolar stress, resulting in cell cycle arrest and interruption of ribosomal activity, critical factors in aging and cancer. In this study, we identified and described the pivotal role of the RNA-binding protein HNRNPK in ribosome and nucleolar dynamics. We developed an in vitro model of endogenous HNRNPK overexpression and an in vivo mouse model of ubiquitous HNRNPK overexpression. These models showed disruptions in translation as the HNRNPK overexpression caused alterations in the nucleolar structure, resulting in p53-dependent nucleolar stress, cell cycle arrest, senescence, and bone marrow failure phenotype, similar to what is observed in patients with ribosomopathies. Together, our findings identify HNRNPK as a master regulator of ribosome biogenesis and nucleolar homeostasis through p53, providing what we believe to be a new perspective on the orchestration of nucleolar integrity, ribosome function and cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f5c155c3ee2cccb1bcb1a6518d0d3272385f671" target='_blank'>
              The tumor suppressor HNRNPK induces p53-dependent nucleolar stress to drive ribosomopathies
              </a>
            </td>
          <td>
            P. Aguilar-Garrido, María Velasco-Estévez, M. A. Navarro-Aguadero, Á. Otero-Sobrino, Marta Ibáñez-Navarro, Miguel Ángel Marugal, María Hernández-Sánchez, P. Malaney, Ashley G Rodriguez, Oscar Benitez, Xiaorui Zhang, M. Aitken, Alejandra Ortiz-Ruiz, Diego Megias, M. Pérez-Martínez, Gadea Mata, J. Gómez, M. Lafarga, Orlando Domínguez, Osvaldo Graña-Castro, Eduardo Caleiras, P. Ximénez-Embún, Marta Isasa, P. J. de Andrés, S. Rodríguez-Perales, R. Torres-Ruiz, E. Revilla, R. García-Martín, Daniel Azorín, Josune Zubicaray, Julián Sevilla, O. Sirozh, V. Lafarga, Joaquín Martínez-López, S. Post, M. Gallardo
          </td>
          <td>2025-05-08</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="Background Cervical cancer progresses through distinct precancerous stages, making early screening and intervention crucial for prevention. However, conventional screening modalities, such as cytology and HPV testing, face challenges related to sensitivity, specificity, and resource dependency. Circular RNAs (circRNAs), owing to their high stability and tissue-specific expression, have emerged as promising biomarkers, though their role in cervical carcinogenesis remains underexplored. In particular, the clinical utility of circRNAs for optimizing cervical cancer screening and early diagnosis has yet to be established. This study aimed to investigate the dynamic expression profiles of circRNAs across various stages of cervical cancer progression and identify potential biomarkers to enhance early detection. Methods CircRNA sequencing was performed on cervical tissues spanning normal cervical epithelium (NCE), high-grade squamous intraepithelial lesions (HSIL), and cervical squamous cell carcinoma (CSCC). Functional assays, including cell viability, colony formation, and apoptosis, were performed to assess the oncogenic potential of circPOLD1 and its interaction with YBX1 in cervical cancer cells. BaseScope and immunohistochemistry (IHC) were applied to tissue microarrays for clincial validation and ROC curve analysis evaluated the diagnostic performance of circPOLD1 in serum as a liquid biopsy marker. Results CircRNA profiling revealed a progressive increase in circPOLD1 expression from NCE to HSIL and CSCC. Mechanistically, circPOLD1 functioned as an oncogene by binding to and phosphorylating YBX1, activating the AKT/mTOR/HIF-1α pathway to enhance glycolysis-driven tumorigenesis. BaseScope and IHC confirmed the stage-specific elevation of circPOLD1 and YBX1 in cervical lesions. The circPOLD1/YBX1 multi-marker panel demonstrated superior diagnostic performance, achieving an AUC of 0.951 for LSIL+ and 0.817 for HSIL+ detection. Furthermore, serum circPOLD1 levels exhibited a progressive increase across disease stages, underscoring its potential as a non-invasive biomarker. Conclusion circPOLD1 and YBX1 synergistically drive cervical carcinogenesis and exhibit stage-specific expression patterns. Their combined detection significantly enhanced the accuracy for cervical cancer screening and dynamic monitoring. The successful application of BaseScope and IHC highlights the immediate translational potential of these biomarkers, paving the way for refined risk stratification, improved therapeutic targeting, and reduced cervical cancer burden through early intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06494-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f6664f03d6924fc8f8933932ffe6615430fec" target='_blank'>
              The biomarker potential of circPOLD1 and its binding protein YBX1 in cervical carcinogenesis
              </a>
            </td>
          <td>
            Lu Zhao, Xin Chen, Yanan Zhang, Y. Cen, Tingjia Zhu, Lingfang Wang, L. Xia, Yang Li, Xiaodong Cheng, Xing Xie, Weiguo Lu, Junfen Xu
          </td>
          <td>2025-05-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Plant nuclear genomes contain thousands of genes of mitochondrial and plastid origin as the result of endosymbiotic gene transfer (EGT). EGT is a still-ongoing process, but the molecular mechanisms determining its frequency remain largely unknown. Here we demonstrate that nuclear double-strand break (DSB) repair is a strong suppressor of EGT. Through large-scale genetic screens in tobacco plants, we found that EGT from plastids to the nucleus occurs more frequently in somatic cells when individual DSB repair pathways are inactive. This effect is explained by the expected increase in the number and residence time of DSBs available as integration sites for organellar DNA. We also show that impaired DSB repair causes EGT to increase 5- to 20-fold in the male gametophyte. Together, our data (1) uncover DSB levels as a key determinant of EGT frequency, (2) reveal the strong mutagenic potential of organellar DNA and (3) suggest that changes in DNA repair capacity can impact EGT across evolutionary timescales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7706b7ef66d1fdcd1d30d7b318635c18156686f3" target='_blank'>
              Suppression of plastid-to-nucleus gene transfer by DNA double-strand break repair
              </a>
            </td>
          <td>
            Enrique Gonzalez-Duran, Xenia Kroop, Anne Schadach, Ralph Bock
          </td>
          <td>2025-05-16</td>
          <td>Nature Plants</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human papillomavirus associated head and neck cancer (HPV + HNC) is rising globally, emphasizing the need for improved therapeutic and screening strategies. To test novel therapies and study HPV-related disease progression, it is vital to develop relevant preclinical models. However, many fail to address critical concerns, including generating a representative immune microenvironment and adequately modeling HPV-driven malignant transformation. Recent multi-omics studies reveal the significance of HPV integration location in HPV-related carcinogenesis and highlight the necessity of targeted treatment methods. Thus, we have developed a murine model of HPV16 + HNC modifying the published CRISPR-based Somatic Oncogene kNock-In for Cancer Modeling (CRISPR-SONIC) system for precise integration of HPV oncogenes. We showed that CRISPR-SONIC knock-in of KrasG12D, HPV16 E6 and E7, and a luciferase reporter at the murine β-actin 3’-UTR locus could induce spontaneous buccal tumors with sarcomatous morphology under transient or selective immunosuppression. Both preventative and therapeutic pNGVL4a-CRT/E7(detox) DNA vaccination could induce HPV16 E7-specific immune response and reduce tumor growth. Furthermore, CRISPR-SONIC knock-in of HPV16 E6 and E7 with co-delivery of HNC-relevant oncogenes AKT and c-Myc produced tumors in NSG mice capturing the characteristic carcinomic morphology of HPV + HNC. Overall, our model offers a robust platform for evaluating new therapies and exploring HPV-related carcinogenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01427-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c396727d3082a56902b37a32cd0299a9e91f92a0" target='_blank'>
              Generation of a spontaneous murine HPV + oral cancer model with site-specific oncogene insertion using CRISPR-SONIC
              </a>
            </td>
          <td>
            Julia Tao, Jason Murray, Hsin-Fang Tu, Darrell Fan, , Ming-Hung Hu, Annie A Wu, Deyin Xing, Chien-Fu Hung, T-C Wu
          </td>
          <td>2025-06-18</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Mitosis, the shortest and most dynamic phase of the cell cycle, entails a complex sequence of events that result in the equal distribution of genetic material to two daughter cells. It has been proposed that mitochondrial respiration increases during mitosis to ensure sufficient ATP for this highly energy-demanding process. However, beyond bioenergetics, how concomitant metabolic changes influence mitotic progression and DNA inheritance remains unclear. To visualize these metabolic changes, we introduce fluorescent-based, genetically encoded reporters into cell lines and human tumor organoids, targeting different cellular compartments to track metabolic and redox dynamics during mitosis with high temporal and subcellular resolution. We observe an increase in ATP production in mitochondria and, notably, an increase in the NAD+/NADH ratio in mitotic chromatin upon nuclear envelope breakdown. Strikingly, disrupting this redox change at mitotic entry alters the epigenetic landscape, leading to a hyperacetylated chromatin state that affects chromatin compaction and Aurora B activity. Importantly, this series of events, initiated by metabolic perturbation, induces mitotic errors that result in genetically distinct daughter cells. Altogether, these findings uncover a novel role for mitochondria in regulating genetic inheritance through NAD(H) redox control, a mechanism with the potential to underlie genomic instability in cancer, where lactate accumulation due to the Warburg effect perturbs redox homeostasis.


 Maria J. Rodriguez Colman. Multifaceted mitochondria in the regulation of chromosome inheritance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY22-03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="AIMS & METHODS
DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages.


RESULTS
Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example.


CONCLUSIONS
These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epstein-Barr virus (EBV), a common gamma-herpesvirus linked to various malignancies, exploits host cellular mechanisms to promote oncogenesis. Our previous research identified the zinc finger protein ZC3H18 as a novel component of the cellular DNA replication machinery in the context of EBV-driven tumorigenesis. We now demonstrate that ZC3H18 expression is upregulated in EBV-transformed and cancer cell lines, as well as in EBV-positive diffuse large B-cell lymphomas from AIDS patients, compared to their EBV-negative counterparts, supporting its activation by EBV. Our experiments show that ZC3H18 expression is regulated by the key oncogenic factors STAT3 and MYC, as well as the essential viral protein EBNA1. Using inhibitors and genetic knockdown, we find that suppressing STAT3, MYC, or EBNA1 leads to decreased ZC3H18 levels, reduced cell viability, and increased apoptosis in EBV-positive B lymphoma cells. Furthermore, ZC3H18, STAT3, MYC, and EBNA1 mutually support each other’s expression through a complex transcriptional network. Notably also, ZC3H18 transcriptionally enhances components of the NF-κB pathway, contributing to NF-κB signaling even in the absence of the EBV oncoprotein LMP1, which is crucial for cell proliferation and survival of several EBV-associated malignancies. Our findings reveal a novel regulatory axis in EBV-positive cancer cells involving STAT3, MYC, EBNA1, & ZC3H18, also linking ZC3H18 to the NF-κB pathway independently of LMP1. The involvement of EBNA1 in this network may explain, at least in part, the preferential upregulation of ZC3H18 in EBV-associated tumors and highlights predictive and therapeutic possibilities for such cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a0f98f609a49371ab8fdbae5145c0b3160ec0b2" target='_blank'>
              STAT3, MYC, and EBNA1 cooperate through a ZC3H18 transcriptional network to regulate survival and proliferation of EBV-positive lymphomas
              </a>
            </td>
          <td>
            Huanzhou Xu, S. Koganti, Chenglong Li, Michael T McIntosh, Sumita Bhaduri-McIntosh
          </td>
          <td>2025-05-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Improved biomarkers of treatment response are needed for patients with high-grade serous ovarian cancer (HGSC). A challenge is substantial anatomical site-to-site variation in expression. We completed data-independent acquisition–mass spectrometry (DIA-MS) analysis of 404 fresh frozen and 78 formalin-fixed, paraffin-embedded HGSC tissue samples from the ovary (adnexal) and a common secondary site (omentum) in 11 patients. This was compared with mutation testing, gene expression, and whole-genome copy number profiling. Proteins with relatively stable intra- and variable inter-individual expression (n = 1651), included a 52-protein module reflecting interferon-mediated tissue inflammation, indicative of a cGAS-STING pathway cytosolic double-stranded (ds) DNA response. The dsDNA sensing/inflammation score was higher in the omentum compared with the ovary. Ovarian HGSC samples showed marked inter-individual differences in inflammatory and immune responses to DNA damage. Stable discriminative features of the HGSC proteome, a prerequisite for clinical predictive biomarkers, are detectable in ovary (adnexal) tissue samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b411dbc827e64e39c2f022da354cc6d36a3a9c" target='_blank'>
              Overcoming intra-tumoral heterogeneity for biomarker discovery in the high-grade serous ovarian cancer proteome
              </a>
            </td>
          <td>
            Srikanth S. Manda, Maiken M Espersen, C. Mapagu, N. Bouantoun, J. Boros, Y. Chiew, S. Srirangan, Swetansu Pattnaik, Catherine J Kennedy, A. Brand, D. W. Garsed, A. Pandey, D. Bowtell, Natasha Lucas, Dylan Xavier, S. Mahboob, Daniel Bucio-Noble, B. Tully, P. Hains, Phillip J. Robinson, Qing Zhong, R. Reddel, A. Defazio, Rosemary L Balleine
          </td>
          <td>2025-06-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of  dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-03</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, Tuğcan Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Antagonistic activities of the 53BP1 axis and the tumor suppressor BRCA1-BARD1 determine whether DNA double-strand breaks (DSBs) are repaired by end joining or homologous recombination. We show that the CTC1-STN1-TEN1 (CST) complex, a central 53BP1 axis component, suppresses DNA end resection by EXO1 and the BLM-DNA2 helicase-nuclease complex but acts by distinct mechanisms in restricting these entities. Whereas BRCA1-BARD1 alleviates the CST-imposed EXO1 blockade, it has little effect on BLM-DNA2 restriction. CST mutants impaired for DNA binding or BLM-EXO1 interaction exhibit a hyper-resection phenotype and render BRCA1-deficient cells resistant to poly(ADP-ribose) polymerase (PARP) inhibitors. Our findings mechanistically define the crucial role of CST in DNA DSB repair pathway choice and have implications for understanding cancer therapy resistance stemming from dysfunction of the 53BP1 axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edcb82922ba3661ab5becf214c56aecb500e3589" target='_blank'>
              CTC1-STN1-TEN1 controls DNA break repair pathway choice via DNA end resection blockade.
              </a>
            </td>
          <td>
            Cody M Rogers, Hardeep Kaur, Michelle L Swift, Vivek B Raina, Shuo Zhou, Ajinkya S Kawale, Shahrez Syed, Korilynn G Kelly, Angela M Jasper, Sameer Salunkhe, Youngho Kwon, Jeffrey Wang, Aida Badamchi Shabestari, J. Daley, Adam Sacks, M. Gaczynska, P. Osmulski, Yashpal Rawal, N. Tomimatsu, S. Gayther, K. Lawrenson, S. Burma, Elizabeth V. Wasmuth, Shaun K. Olsen, Weixing Zhao, R. Hromas, David S. Libich, Alexander V. Mazin, Daohong Zhou, E. Greene, Dipanjan Chowdhury, Patrick Sung
          </td>
          <td>2025-05-22</td>
          <td>Science</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Among the human leukemia cell lines described in the literature, only the MDS-L cell line has been definitively established from a patient during the myelodysplastic syndrome (MDS) phase of the disease. However, the limited studies on its genomic complexity have restricted its applicability as an in vitro model for MDS. Here, we aimed to better characterize the chromosomal and genetic alterations of MDS-L. A comprehensive approach was employed combining conventional G banding, multicolor FISH (M-FISH), SNP arrays with the novel Optical Genome Mapping (OGM) technology. In addition, the mutational landscape was defined using targeted next-generation sequencing (NGS). G-banding revealed two karyotypically distinct cell populations, both exhibiting complex karyotypes. Using G-banding and OGM, we identified previously undescribed structural alterations, including der(1)t(1;7)(q11;q11.2), del(1)(q11), der(4)t(4;5)(p16;q11.2), i(5)(p10), der(6)t(6;15)(p21.3;q15), i(8)(q10), der(9)t(9;10)(q34;p11.21), der(19)t(6;19)(p13;p22) and i(22)(q10). Both OGM and SNP microarray analyses detected multiple copy number variants and regions of homozygosity. Chromosome breakpoints were precisely defined by OGM, allowing the identification of gene disruption events. Moreover, M-FISH technique validated the origins of additional chromosomal material observed in the karyotype, identified cryptic rearrangements, and distinguished the two clonal populations within the cell line. Finally, NGS revealed mutations in CEBPA, NRAS, TET2 and TP53 genes associated with MDS pathology. This multi-technique approach has enabled a precise characterization of the MDS-L cell line’s genomic complexity, highlighting the unique contributions of each technique in uncovering various genetic alterations and establishing a valuable resource for mechanistic studies and pre-clinical drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93beb81beda87cc45be7d6f41135c2605a8afe13" target='_blank'>
              Integrated cytogenetic and genomic profiling of the MDS-L cell line
              </a>
            </td>
          <td>
            Julia Mestre, Lorea Chaparro-González, Isabel Granada, M. Mallo, Emili Cid, Estefania Mancini, Oriol Calvete, R. Risueño, D. Starczynowski, Francesc Solé
          </td>
          <td>2025-06-11</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Triggering cancer cell death by inducing DNA damage is the primary aim of radiation therapy; however, normal cells are also damaged. Here, we showed that delivery of only four synthetic guide RNAs (sgRNAs) with Cas9 endonuclease efficiently induced simultaneous DNA double-strand breaks, resulting in efficient cell death in a cell type-specific manner. Off-target effects of Cas9 endonuclease were prevented by using Cas9-nickase to induce DNA single-strand breaks and blocking their repair with poly ADP-ribose polymerase (PARP) inhibitors. When recombinant Cas9-nickase protein and multiple sgRNAs were delivered with PARP inhibitors into cultured cells, in vivo xenografts, and patient-derived cancer organoids via lipid nanoparticles, cancer cells were unable to tolerate the induced DNA damage, even in the presence of a functional BRCA2 gene. This approach has the potential to expand the use of PARP inhibitors with verified safety and thus is a potentially powerful tool for personalized genome-based anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fde0090503581f24fa582fe8e81b3d566923d8b" target='_blank'>
              Combining Multiplexed CRISPR/Cas9-Nickase and PARP Inhibitors Efficiently and Precisely Targets Cancer Cells.
              </a>
            </td>
          <td>
            Soyoung Lee, Kyunghwan Kim, Hye-Jin Jeong, Subin Choi, Himchan Cheng, Dayoung Kim, Soomin Heo, Jinhee Mun, Minjong Kim, Eunjin Lee, Yoon Ji Choi, Seon-Gyeong Lee, Eunjin Lee, Yewon Jang, Kayeong Lim, Heon Seok Kim, Euihwan Jeong, Seung-Jae Myung, Deok-Beom Jung, Chang Sik Yu, In Ho Song, M. Corces, Joo H. Kang, Kyungjae Myung, Taejoon Kwon, Tae-Eun Park, Jinmyoung Joo, Seung Woo Cho
          </td>
          <td>2025-05-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Human exonuclease 1 (EXO1), a member of the structure-specific nuclease family, plays a critical role in maintaining genome stability by processing DNA double-strand breaks (DSBs), nicks, and replication intermediates during DNA replication and repair. As its exonuclease activity is essential for homologous recombination (HR) and replication fork processing, EXO1 has emerged as a compelling therapeutic target, especially in cancers marked by heightened DNA damage and replication stress. Through high-throughput screening of 45,000 compounds, we identified seven distinct chemical scaffolds that demonstrated effective and selective inhibition of EXO1. Representative compounds from two of the most potent scaffolds, C200 and F684, underwent a comprehensive docking analysis and subsequent site-directed mutagenesis studies to evaluate their binding mechanisms. Biochemical assays further validated their potent and selective inhibition of the EXO1 nuclease activity. Tumor cell profiling experiments revealed that these inhibitors exploit synthetic lethality in BRCA1-deficient cells, emphasizing their specificity and therapeutic potential for targeting genetically HR-deficient (HRD) cancers driven by deleterious mutations in HR genes like BRCA1/2. Mechanistically, EXO1 inhibition suppressed DNA end resection, stimulated the accumulation of DNA double-strand breaks, and triggered S-phase PARylation, effectively disrupting DNA repair pathways that are essential for cancer cell survival. These findings establish EXO1 inhibitors as promising candidates for the treatment of HRD cancers and lay the groundwork for the further optimization and development of these compounds as targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy.
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background PERP may have the potential to function as an oncogene. However, the precise function, prognostic value, and predictive significance remain shrouded in ambiguity. Methods We conducted an in-depth analysis using pan-cancer RNA sequencing data and various online web tools to investigate the correlation between PERP and crucial clinical outcomes such as prognosis, tumor microenvironment, and tumor metabolism. In addition, we explored the tumor-promoting role of PERP and its potential mechanisms through models such as immunofluorescence staining, flow cytometry, cell proliferation assays, wound healing assays, cell migration assays, mass spectrometry analysis and isotope tracing. Further in vivo models confirmed the functional consistency of PERP across pan-cancer. Finally, we analyzed the potential of PERP as a predictive factor for immunotherapy sensitivity in a clinical cohort. Results PERP exhibits elevated expression in the majority of cancer types and impedes immune cell infiltration as well as immune checkpoint reactivity in pan-cancer. We confirmed that PERP can promote tumor progression by tumor cell proliferation, scratch and transwell experiments. Meanwhile, the absence of PERP restricts the flux of 13C6-glucose into glycolysis and the tricarboxylic acid (TCA) cycle. Importantly, the deficiency of PERP enhances the in vivo anti-tumor efficacy of PD1 monoclonal antibodies. In addition, low PERP expression is highly correlated with the response of head and neck squamous cell carcinoma (HNSCC) patients to immunotherapy. Conclusions PERP represents a promising predictive/diagnostic biomarker and therapeutic target for HNSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02360-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165cbe4ddcc5a029683b55fb68914aec7383f0fa" target='_blank'>
              Targeting PERP promotes anti-tumor immunity in HNSCC by regulating tumor immune microenvironment and metabolic homeostasis
              </a>
            </td>
          <td>
            Xueying Wang, Yuxi Tian, Xiaohong Wu, Yewen Zhu, Huihong Chen, Zeyao Wang, Zihan Liu, Jiaqi Tan, Zhaoyu Pan, Jiaoyan Cao, Zhenjiang Li, Xin Zhang, Zhongjie Shi, Junchen Wang, Tong Liu
          </td>
          <td>2025-06-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="HSP110 is a ubiquitous chaperone contributing to proteostasis. It has a disaggregation activity and can refold denatured proteins. It can regulate fundamental signaling pathways involved in oncogenesis, such as Wnt/β-catenin, NF-κB and STAT3 signaling pathways. In gastric and colorectal cancer, HSP110 has been detected in the nucleus, and nuclear expression has been associated with the resistance of cells to 5-FU chemotherapy. Nuclear translocation of HSP110 is promoted by the exposure of cells to DNA-damaging agents. In a previous work, we demonstrated that nuclear HSP110 participates in the NHEJ DNA repair pathway by facilitating the recruitment of DNA-PKcs to Ku70/80 heterodimers at the site of DNA double-strand breaks. In the present work, analysis of HSP110s’ nuclear interactome revealed an enrichment of components from SWI/SNF chromatin remodeling complexes. We demonstrate that HSP110 is strongly associated with chromatin in temozolomide- and oxaliplatin-treated cells and directly interacts with the core subunit SMARCC2, thereby facilitating the assembly of SWI/SNF complexes. This work expands upon the role of HSP110, which regulates not only proteostasis but also the assembly of critical nuclear macromolecular complexes involved in the adaptive stress response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c88c3e640399a73a02e0ec0eeda0b57f77e5458" target='_blank'>
              HSP110 Regulates the Assembly of the SWI/SNF Complex
              </a>
            </td>
          <td>
            Océane Pointeau, Manon Paccagnini, Natalia Borges-Bonan, L. Biziorek, Sebastien Z. Causse, Carmen Garrido, L. Dubrez
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d46f5896df3074e9079fd05c2ab487e0d0fe604" target='_blank'>
              Context-Dependent Regulation of Microhomology-Mediated End Joining in Normal Tissues: Insights into Tissue-Specific Activation of DNA Repair Pathways
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ionizing radiotherapy (RT) is a widely used treatment strategy for malignancies. In solid tumors, RT-induced double-strand breaks lead to the accumulation of indels, and their repair by non-homologous end-joining has been linked to the ID8 mutational signature in surviving cells. However, the extent of RT-induced mutagenesis in hematologic malignancies and its impact on their mutational profiles and interplay with commonly used chemotherapies has not yet been explored. Here, we interrogated 580 whole-genome sequence samples (WGS) from patients with large B-cell lymphoma, multiple myeloma, and myeloid neoplasms and identified ID8 only in relapsed disease. Yet, ID8 was detected after exposure to both RT and mutagenic chemotherapy (i.e., platinum and melphalan). Using WGS of single-cell colonies derived from treated lymphoma cells, we revealed a dose-response relationship between RT and platinum and ID8. Finally, using ID8 as a genomic barcode we demonstrate that a single RT-surviving cell may seed distant relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537c7faf04a6fea7b0b8debed425a1583b3dc618" target='_blank'>
              Mutagenic impact and evolutionary influence of chemo-radiotherapy in hematologic malignancies.
              </a>
            </td>
          <td>
            B. Diamond, Dhanvantri Chahar, Michael D Jain, A. Poos, Michael A Durante, B. Ziccheddu, Marcella Kaddoura, M. Papadimitriou, K. Maclachlan, T. Jelínek, Faith E Davies, Nicholas B Figura, Gareth J Morgan, E. Mai, K. Weisel, Roland Fenk, M. Raab, Saad Z. Usmani, O. Landgren, Frederick L Locke, Hartmut Goldschmidt, J. Schatz, Niels Weinhold, F. Maura
          </td>
          <td>2025-05-22</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Radiotherapy remains the cornerstone of treatment for glioblastoma (GBM). However, the frequent occurrence of radiation resistance presents a significant therapeutic challenge. A comprehensive understanding of the mechanisms underlying this resistance is essential for improving GBM treatment strategies. In the present study, live-dead cell staining and immunofluorescence staining were employed, and irradiation-resistant cell lines were established. It was observed that transglutaminase 2 (TGM2) plays a pivotal role in enhancing radiation resistance in GBM, facilitating cell proliferation, and promoting DNA damage repair following irradiation. Moreover, immunofluorescence and nucleoplasmic protein extraction assays revealed that TGM2 in GBM rapidly translocates into the nucleus upon irradiation. Through co-immunoprecipitation assays, TGM2 was identified as binding to an increased amount of p53 proteins, thereby promoting p53 degradation post-irradiation. Notably, inhibition of this interaction resulted in a reduction of radiation resistance in GBM. In summary, this study underscores the significance of TGM2 nuclear translocation in radiation resistance and suggests that disrupting TGM2 binding to p53 may offer novel therapeutic insights for overcoming radiation resistance in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02599-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/964bc5c1f03aa944fd2f4ebcaf77d53377579d10" target='_blank'>
              Transglutaminase 2 nuclear localization enhances glioblastoma radiation resistance
              </a>
            </td>
          <td>
            Cuiping Sun, Zhiyong Du, Wenhui Yang, Qing Wang
          </td>
          <td>2025-05-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Jose Antonio Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Chromosomal translocations are key events in cancer, driving oncogenesis by disrupting and deregulating critical genes. While specific tumor‐associated translocations are well studied, the frequencies and distributions of most remain unknown. Additionally, the role of chromosomal reshuffling in translocations has received little attention. This study presents data on the chromosomal involvement in 59 251 translocations reported in 58 tumor entities, including both benign and malignant tumors. Unlike studies focusing on tumor‐specific abnormalities identified at the chromosome band level, this study examines translocations at the chromosomal level, offering a novel perspective on their distribution. This broader approach aims to uncover patterns that do not emerge or are disregarded in studies limited to tumor‐specific aberrations. The resulting dataset provides a novel resource for deepening our understanding of the chromosomal origins of translocations in neoplasia. Comparisons of translocation frequency distributions among tumor types, when excluding the characteristic tumor‐associated translocations, revealed that the patterns of chromosomal involvement in translocations are largely unique to each tumor entity. Statistical analyses of 241 pairwise comparisons of translocation spectra within hematologic disorders, solid tumors, and between groups of hematologic malignancies and both benign and malignant solid tumors showed insignificant/very weak associations (R2 ≤ 0.3) in 98% of the comparisons. The findings hence demonstrate that different tumor types are characterized by distinct chromosomal translocation signatures, strongly suggesting that most translocations encountered in tumor cells are not merely random events. Consequently, our study highlights the potential of rare translocations to serve as indicators of disease‐specific processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c3b03e0bf3342743907df4f51e09cd0d87a1db" target='_blank'>
              Distinct Signatures of Chromosomal Involvement in 59 251 Translocations Across 58 Tumor Types. A Novel Perspective
              </a>
            </td>
          <td>
            Felix Mitelman, Nils Mandahl
          </td>
          <td>2025-05-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Oncogene activation through DNA amplification or overexpression plays a critical role in cancer initiation and progression. Chromosome 17 harbors numerous cancer-associated genetic anomalies, some of which are strongly linked to poor prognosis in breast cancer patients. The FOXK2 gene, located on 17q25, encodes a transcription factor with a forkhead DNA binding domain. Integrative analyses of publicly available genomic datasets reveal that FOXK2 is frequently amplified and overexpressed in breast cancer, correlating with poor overall survival. FOXK2 knockdown via shRNA significantly inhibited the proliferation, migration, and anchorage-independent growth of breast cancer cells, causing G0/G1 cell cycle arrest. The C-terminal nuclear localization signal (NLS) is crucial for FOXK2's oncological function. Additionally, FOXK2 knockdown in MCF7 breast cancer cells delayed tumor growth in a xenograft mouse model. Inhibiting FOXK2 expression also sensitized breast cancer cells to frontline chemotherapy agents, including doxorubicin, 5-fluorouracil, and etoposide in vitro. Co-overexpression of FOXK2 and PI3KCA with oncogenic mutations (E545K or H1047R) induced the transformation of non-tumorigenic MCF-10A cells, suggesting that FOXK2 acts as an oncogene in breast cancer and contributes to PI3KCA-driven tumorigenesis. CCNE2, PDK1, and ESR1 were identified as direct transcriptional targets of FOXK2 in MCF-7 cells. Moreover, co-inhibition of FOXK2 function, either through knockdown or inhibition of its transcriptional targets (CCNE2/CDK2 by Dinaciclib or PDK1 by DCA) and PI3KCA by alpelisib, had synergistic anti-tumor effects in breast cancer cells with FOXK2 overexpression and PI3KCA oncogenic mutations. FOXK2 functional inhibition by the CDK2 inhibitor Dinaciclib enhanced the tumor-suppressive effect of alpelisib in a MCF7 breast cancer xenograft mouse model. These findings provide compelling evidence that FOXK2 plays an oncogenic role in breast tumorigenesis. The combination of PIK3CA and CCNE2/CDK2 inhibition by alpelisib and Dinaciclib offers a potential therapeutic strategy for breast cancer with FOXK2 overexpression and PIK3CA oncogenic mutation.
 Citation Format: Hong Zhang, Yang Yu, Weng-Ming Cao, Feng Cheng, Edaise da silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang. Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy? [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-24.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caaa74a368ec1f757b26670f47abf7954355f919" target='_blank'>
              Abstract P5-02-24: Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy?
              </a>
            </td>
          <td>
            Hong Zhang, Yang Yu, Wen-Ming Cao, Feng Cheng, E. D. da Silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ce08d08aef5a794575f58122e8dce3a63ce74" target='_blank'>
              Aneuploidy alleviates the cell proliferation defect caused by mutations affecting origin licensing in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Christophe de La Roche Saint-André
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Abstract Across the tree of life, DNA damage response (DDR) proteins play a pivotal, yet dichotomous role in organismal development and evolution. Here, we present a comprehensive analysis of 432 DDR proteins encoded by 68 genomes, including that of Nucleospora cyclopteri, an intranuclear microsporidia sequenced in this study. We compared the DDR proteins encoded by these genomes to those of humans to uncover the DNA repair-ome across phylogenetically distant eukaryotes. We also performed further analyses to understand if organismal complexity and lifestyle play a role in the evolution of DDR protein length and conserved domain architecture. We observed that the genomes of extreme parasites such as Paramicrocytos, Giardia, Spironucleus, and certain microsporidian lineages encode the smallest eukaryotic repertoire of DDR proteins and that pathways involved in modulation of nucleotide pools and nucleotide excision repair are the most preserved DDR pathways in the eukaryotic genomes analysed here. We found that DDR and DNA repair proteins are consistently longer than housekeeping and metabolic proteins. This is likely due to the higher number of physical protein–protein interactions which DDR proteins are involved. We find that although DNA repair proteins are generally longer than housekeeping proteins, their functional domains occupy a relatively smaller footprint. Notably, this pattern holds true across diverse organisms and shows no dependence on either lifestyle or mitochondrial status. Finally, we observed that unicellular organisms harbour proteins that are tenfold longer than their human homologues, with the extra amino acids forming interdomain regions with a clearly novel albeit undetermined function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d2f7581b50b1d038466b2efbeb5d637e6743082" target='_blank'>
              Evolutionary Insights into the Length Variation of DNA Damage Response Proteins Across Eukaryotes
              </a>
            </td>
          <td>
            Dominic Wiredu-Boakye, Laurence Higgins, O. Gahura, A. Butenko, Guy Leonard, Mark A Freeman, Á. Kristmundsson, Karen Moore, Jamie W Harrison, Shani Mac Donald, V. Yurchenko, Bryony A P Williams, Richard Chahwan
          </td>
          <td>2025-05-19</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) affects genomic stability and has potential as a biomarker for the effectiveness of poly (ADP-ribose) polymerase (PARP) inhibitors and immune checkpoint inhibitors. However, the clinical and molecular profile of HRD in non-small cell lung cancer (NSCLC), particularly in the Chinese population, remains poorly characterized. Based on the next-generation sequencing data of 158 Chinese NSCLC patients, we analyzed the HRD scores of mutations in homologous recombination repair (HRR) genes and dissected the correlation between HRD state and programmed death-ligand 1 (PD-L1) expression. Alterations in HRR genes were observed in 8.9% of the patients, with ATM and BRCA2 being the most commonly affected genes. HRD-high (HRD-H) status was significantly associated with advanced disease stage (≥III) and lung squamous cell carcinoma (LUSC). Transcriptomic analysis revealed distinct gene expression profiles between HRD-H and HRD-low (HRD-L) subgroups, with HRD-H tumors exhibiting predominantly downregulated genes. While EGFR mutations occurred at similar frequencies across HRD status, TP53 mutations were significantly enriched in HRD-H cases. HRD-H status correlated with higher PD-L1 positivity in NSCLC overall, but not within the lung adenocarcinoma (LUAD) subgroup in our cohort. The Cancer genome atlas analysis showed higher PD-L1 protein expression in HRD-H LUAD, but not in LUSC. Kyoto Encyclopedia of Genes and Genomes analysis identified enrichment of complement and coagulation cascades, ABC transporters, and bile secretion pathways in HRD-H tumors, suggesting links to immune evasion and drug resistance. This study elucidates the genomic landscape of HRD in Chinese NSCLC patients and provides insights into its potential clinical utility for therapeutic targeting. Our findings suggest that integrated HRD scoring may guide the application of PARP inhibitors and immunotherapy in specific NSCLC patient subgroups. Further prospective clinical studies are needed to validate the predictive value of HRD scoring in NSCLC treatment and to optimize patient selection strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c403957972723c45a13a1636514a41ce8c5b8d3" target='_blank'>
              Genomic alterations of homologous recombination deficiency in Chinese NSCLC patients
              </a>
            </td>
          <td>
            Shuang Xiang, Changqiong Shen, Chun Huang, Ya-ting Yang, Jing Guo, Yi Liu, Mingzhu Yin, Song Duan
          </td>
          <td>2025-06-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study analyzed the genetics of classical Hodgkin lymphoma (cHL) by using circulating tumor DNA (ctDNA). Two genetic subtypes were identified, differing in genetic instability mechanisms: one subtype (64% of cases) showed a higher mutation load and a higher fraction of mutations associated with activation-induced cytidine deaminase and microsatellite instability signatures, while the other subtype (36% of cases) exhibited chromosomal instability with more somatic copy number alterations. Whole-genome duplication was more common in cHL compared to other B-cell tumors and emerged as a prognostic biomarker for patients undergoing ABVD-based therapy. Non-coding regulatory mutations, similar to those in diffuse large B-cell lymphoma, were highly prevalent in 86% of cHL. A recurrent somatic expression quantitative trait locus (seQTL) involving the BCL6 gene was found in 30% of cases. The seQTL of BCL6 aligned with accessible chromatin and increased H3K27 acetylation in cHL, disrupted PRDM1 binding, and co-occurred with BCL6 expression in cHL cells. Weak to strong expression of BCL6 was observed in 68% of cases and BCL6 expression associated with gene repression similarly in cHL and germinal center B cells. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was derepressed in cHL and proliferation was impaired. The number and clonality of neoantigens was associated with tumor microenvironment type and response to checkpoint blockade. Finally, ctDNA analysis was suggested as a tool to distinguish ambiguous PET/CT-positive lesions post-treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61df03d16c11da50362eb27baf0281aee3eb21d5" target='_blank'>
              A novel role for UPF1 in safeguarding DNA replication in S phase and beyond
              </a>
            </td>
          <td>
            Thomas Walne, Laura Maple, Nan Li, Isabel Christie, Carl Smythe, Ruth Thompson
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next-generation sequencing. New techniques, such as single-cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single-cell and spatial analyses. These developments enable the investigation of cell-cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cf680837c223860a295c7d973297324c3a3da0" target='_blank'>
              From predictive biomarker to therapeutic target: the dual role of SLFN11 in chemotherapy sensitivity.
              </a>
            </td>
          <td>
            Yue Feng, Yingze Li, Zhenhao Zhang, Yuxuan Dai, Xingchun Gou, Kejing Lao, Running Zhang
          </td>
          <td>2025-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Background: In HER2+ breast cancer (BC), residual disease (RD) after neoadjuvant HER2 inhibition is associated with increased risk of recurrence. However, the genetic and non-genetic resistance mechanisms underlying RD remain incompletely characterized. Here we used single nuclei (sn) RNA and DNA sequencing to profile genetic and transcriptional changes in growing, residual, and recurrent tumors from an inducible mouse tumor model of HER2+ breast cancer. Methods: We used mammary tumors / tumor bed (n=2-3/time point) from female MMTV-rtTA; TetO-ERBB2 mice at 9 weeks DoxOn (HER2on), 7 days DoxOff (HER2off) and 12-21 weeks Doxoff (recurrent). Samples were snap frozen and single nuclei isolation was performed. snRNAseq was performed using 10X 5’ v2 kits for library preparation and sequencing. snDNAseq was performed using a custom amplicon panel targeting the mouse genome and human ERBB2 locus via the MissionBio Tapestri platform. Results: For snRNAseq analysis, tumor cell clusters were identified by Epcam and ERBB2 gene expression. Tumor cells were further subclustered into 18 unique clusters that separated by ERBB2 expression. Seven subclusters of HER2on tumor cells were characterized by low ERRB2 expression: 1.38 vs 2.94 (median normalized log counts) in HER2on-ERBB2high cells. Top 5 differentially expressed genes (DEG) per condition (log2 fold change >1.5, adjusted p value < 0.05) were: HER2on: Erbb2, Csn3, Lcn2, Csn2, Wfdc18; HER2on-ERBB2high: Csn3, Igfbp5, Csn2, Lcn2, Trf; HER2on-ERBB2low: Lars2, Gm47283, Cmss1, Gm42418, Neat1; HER2off: Lars2, Hexb, Cmss1, Gphn, Gm42418; Recurrent: Capza2, Vim, Lgals1, Crabp1, S100a6. HER2on-ERBB2low cells were more closely related in transcriptional space to HER2off residual tumor cells than HER2on-ERBB2high tumor cells. Trajectory analysis suggested that HER2off RD evolved from HER2on-ERRB2low cells and formed the basis of recurrent disease. To determine clonal evolution over time, we used common top 10 DEGs between HER2on vs HER2off vs HER2low snRNAseq data and intersected that set with significantly enriched single nucleotide variants (SNVs) from our snDNAseq data. We identified 16 clones containing at least 1 SNV at >1% frequency across all samples. We observed clonal persistence, relative enrichment and extinguishment between HER2on, HER2off and recurrent samples but without clonal dominance. Conclusion: Time-series analysis of genetic and transcriptional evolution in a mouse model of HER2+ BC RD suggests that a tumor cell subpopulation characterized by an ERBB2low-associated transcriptional program drives RD after HER2 inhibition. Further functional analysis will be presented at the conference.
 Citation Format: Sheheryar Kabraji, Weihua Wang, Yali Zhang, Johann Bergholz, Tao Jiang, Vittal Kurisetty, Maia Homsi, Jianmin Wang, Jean J. Zhao. Residual disease after HER2 inhibition is driven by a primary tumor subpopulation expressing an ERBB2-low associated transcriptional program [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS4-07.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347843feebd647684f01cab5cf9d05c5c7a0be61" target='_blank'>
              Abstract PS4-07: Residual disease after HER2 inhibition is driven by a primary tumor subpopulation expressing an ERBB2-low associated transcriptional program
              </a>
            </td>
          <td>
            S. Kabraji, Weihua Wang, Yalii Zhang, Johann S. Bergholz, Tao Jiang, Vittal Kurisetty, Maia Homsi, Jianmin Wang, Jean Zhao
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Aberrant function or overactivation of exonuclease 1 (EXO1) may be associated with cancer tumor development, drug resistance, and response to immunotherapy in female-related cancers. Methods By analyzing RNA-sequencing data from The Cancer Genome Atlas database, combined with validation through quantitative polymerase chain reaction experiments, we explored the expression levels of EXO1 in breast cancer (BRCA) cell lines and assessed its multidimensional roles in various female-related cancers. Results Our experiments revealed elevated expression of EXO1 in BRCA cell lines, consistent with the RNA-sequencing data. The high expression of EXO1 is associated with poor prognosis in various female-related cancers, especially in BRCA and UCEC. It significantly correlates with clinical and pathological characteristics. In specific cancer subtypes like the basal-like subtype of BRCA, high EXO1 expression is associated with a better prognosis. Genetic mutation analysis indicates a higher frequency of EXO1 gene mutations in uterine sarcoma and BRCA. DNA methylation levels may play a role in the regulation of EXO1 gene expression in some cancers. EXO1 expression is correlated with various factors within the tumor immune microenvironment and may be associated with the sensitivity to anticancer drugs. Conclusion EXO1 exhibits multidimensional roles in female-related cancers as a prognostic biomarker and potentially influences tumor immune therapy responses and drug sensitivities. Further studies are needed to fully understand the complex mechanisms underlying these associations and to explore potential therapeutic strategies targeting EXO1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00f61f934389fb7d273be0eaa350ec5d0b17ae1" target='_blank'>
              Deciphering the multifaceted role of EXO1 in female-related cancers: implications for prognosis and therapeutic responsiveness
              </a>
            </td>
          <td>
            Cong Yu, Guoying Wu
          </td>
          <td>2025-05-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 PKMYT1, a serine/threonine kinase, selectively regulates CDK1 during the G2/M phase transition to prevent mitotic entry in the presence of DNA damage, whereas WEE1 kinase regulates both CDK1 and CDK2. Inhibition of PKMYT1 forces cancer cells to prematurely enter mitosis without repairing DNA damage, thereby leading to cell death. Therefore, PKMYT1 is an attractive target for synthetic lethality in DNA damage-sensitive cancer cells, such as those with hyperactive CCNE1/CDK2 signaling and p53 mutations.
 CCNE1 amplification (Copy number ≥ 6) is observed in approximately 8% of all solid tumor patients and is particularly prevalent in ovarian cancer (31%), uterine carcinosarcoma (37%), gastric cancer (14%), and breast cancer (8%). Despite the significant patient population harboring this biomarker, no targeted therapy has been approved, emphasizing a critical unmet medical need. CCNE1 amplification drives premature S-phase entry, resulting in genome instability due to DNA replication stress.
 VRN16 is a novel PKMYT1 kinase inhibitor with high target selectivity. KINOMEScanTM showed that VRN16 has fewer off-target kinases (1% with S(10)) than lunresertib (7% with S(10)), a clinical stage PKMYT1 inhibitor. Notably, VRN16 spares WEE1, PLKs, and other off-target kinases, including BRAF. In contrast, lunresertib inhibits BRAF kinase, confirmed to cause paradoxical activation of phospho-MEK and phospho-ERK signaling in the cells, similar to the effects observed with vemurafenib, a BRAF inhibitor. Indeed, in CCNE1+ cancer cells, VRN16 demonstrates higher proliferation inhibition potency than lunresertib. Conversely, in CCNE1-normal cells, VRN16 exhibits lower cytotoxicity compared to lunresertib.
 In three CCNE1+ CDX mouse models (OVCAR3, MKN1, and HCC1569), VRN16 demonstrated superior efficacy without observed toxicity compared to lunresertib which showed adverse effects such as stomatitis, keratosis, and edema at its MTD, 10 or 20 mg/kg, BID. Additionally, in a CCNE1+ ovarian cancer PDX model, VRN16 demonstrated dose-dependent efficacy as both monotherapy and in combination therapy with gemcitabine. Although CDK2 inhibitors are also undergoing clinical trials with CCNE1+ as a major biomarker, VRN16 demonstrated superior efficacy in combination with gemcitabine in the OVCAR3 CDX, compared to the combination of BLU-222, a CDK2 inhibitor, with paclitaxel or carboplatin.
 In a 2-week repeated-dose oral toxicity study, VRN16 achieved a wide preclinical safety margin, more than 8-fold, compared to lunresertib with a limited safety margin. This wide therapeutic window highlights VRN16 as a novel therapeutic option for patients with CCNE1+ solid tumors or DNA damage sensitive tumors.


 Kwangwoo Hwang, Seohyun Jo, Ho-Soo Lee, Youngyi Lee, Donghyun Park, Songyee Han, Kyungah Seo, Jiseon Choi, Soochan Kim, Jiyeon Kim, Donghyuk Seo, Younho Lee, Yikyung Ko, Hyunjong Kwak, Ye Ji Son, Jeonghun Lee, Soyeon Je, Hong-ryul Jung, Daekwon Kim, Sunghwan Kim, Jaeyoung Ahn. VRN16: A novel PKMYT1 kinase selective inhibitor with wide therapeutic window [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB436.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960a7979050b70a2d24fbb180b4e757ef27292f4" target='_blank'>
              Abstract LB436: VRN16: A novel PKMYT1 kinase selective inhibitor with wide therapeutic window
              </a>
            </td>
          <td>
            Kwangwoo Hwang, Seohyun Jo, Ho-Soo Lee, Youngyi Lee, Donghyun Park, Songyee Han, Kyungah Seo, Jiseon Choi, Soochan Kim, Jiyeon Kim, Dong-hyuk Seo, Younho Lee, Yikyung Ko, Hyunjong Kwak, Ye Ji Son, Jeonghun Lee, Soyeon Je, Hong-ryul Jung, Daekwon Kim, Sunghwan Kim, J. Ahn
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is characterized by high heterogeneity, and its molecular features and microenvironment complexity contribute to varied patient prognoses. Anoikis, a form of programmed cell death, is considered closely linked to the development and progression of liver cancer. This study aimed to analyze the cellular heterogeneity of HCC using single-cell sequencing and to construct an anoikis-related prognostic model for predicting patient outcomes. Methods Single-cell sequencing data of HCC patients were obtained from the GEO database. The AUCell algorithm was used to quantify anoikis gene activity across cell types. Cells were grouped based on enrichment scores. Cox regression and 101 machine learning algorithms were employed to identify prognostic genes and construct a risk model, which was validated using TCGA, ICGC, and GEO datasets. Key findings were validated using qPCR on clinical samples, and functional assays using shRNA-mediated knockdown of KPNA2 were conducted in Hep-3B cell lines to assess changes in proliferation and migration. Results Single-cell analysis revealed the cellular heterogeneity of HCC, with particular enrichment of anoikis-related genes in fibroblasts, endothelial cells, and macrophages. The prognostic model, developed using 101 algorithms, demonstrated the highest performance. High-risk patients showed poor prognosis, and the ROC curve analysis showed high accuracy for model predictions. Genetic mutation analysis indicated significant gene amplifications and deletions in the high-risk group. PCR analysis of clinical samples confirmed the expression patterns of key gene KPNA2, and gene knockdown in cell experiments further validated its oncogentic role in HCC. Conclusion This study highlights the anoikis-related cellular heterogeneity in HCC and constructs a machine learning-based prognostic model with high predictive value, enabling effective risk stratification for patients. It offers further support for the potential therapeutic application of the identified genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ae0b20f3f5d93557a996379c7107447e12e9" target='_blank'>
              Anoikis in hepatocellular carcinoma: genetic alterations and immune microenvironment insights from single-cell and computational analysis
              </a>
            </td>
          <td>
            Bing Zhou, Jialing Zhang, Mingfeng Guo
          </td>
          <td>2025-05-31</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 PARP inhibitors (PARPi) transformed the treatment of BRCA1/2-associated cancer by leveraging that tumors with homologous recombination DNA repair (HR) deficiency are vulnerable to PARPi. However, PARPi use is uncommon for patients with invasive lobular carcinoma (ILC), as ILC are rare among BRCA-mutation carriers, and ILC have very low ‘BRCA-like’ genomic scarring of HR deficiency. Importantly, though most ILC appear “low risk” (e.g. ∼95% are estrogen receptor α/ER-positive), ILC have poorer long-term outcomes than other breast cancers, are associated with treatment resistance, and lack ILC-targeted therapies. Novel treatments for ILC are needed, and our work suggests PARPi target a unique vulnerability in ILC and provide an opportunity for precision treatment. We identified a novel crosstalk between ER and DNA repair in ILC – via mediator of DNA damage checkpoint 1 (MDC1), a key double-strand break repair protein – that facilitates endocrine response and anti-estrogen resistance, but compromises DNA repair downstream of MDC1. We find homologous recombination (HR) induction and resolution is limited and inefficient in ILC; e.g., in I-SceI assays for HR versus non-homologous end-joining activity, ILC cells (MDA MB 134VI, SUM44PE) showed ≤10% of repair events via HR, while ≥40% of repair events were via HR in non-ILC breast cancer cells (MCF7, T47D). Importantly, these cell line data are paralleled by ILC tumor data, wherein subsets of ILC are defined by elevated DNA repair protein levels and increased tumor mutational burden. These ILC have the poorest disease-free and overall survival, supporting that high-risk ILC tumors also present with a novel DNA repair dysfunction. Based on this putative HR dysfunction, we tested PARPi talazoparib against ILC models. In vitro, talazoparib was equipotent against proliferation in ILC and non-ILC breast cancer cell lines (IC50 ∼25nM). However, non-ILC cells recovered after talazoparib washout, but ILC cells remained growth suppressed 7d post-washout with outgrowth reduced by 50-80%. In vivo, we tested talazoparib (0.33mg/kg p.o. 5d/w) alone ± estrogen withdrawal (to mimic aromatase inhibition/AI), with 6 weeks treatment against MDA MB 134VI or SUM44PE xenografts. In MM134, talazoparib suppressed tumor growth vs control (ANOVA p=0.0016) and durably suppressed growth as seen in vitro. Combining talazoparib with AI was superior to single agents (ANOVA p=0.0006 vs talazoparib, p= 0.0037 vs AI) and best prolonged host survival (cohort survival: ctrl = 158d; AI = 175d, log rank vs ctrl p=0.054; talazoparib = 182d, p=0.069; AI + talazoparib = >209d, p=0.0021). Combined AI + talazoparib also best suppressed SUM44PE xenograft growth (ANOVA AI vs combination, p<0.0001), and median time to tumor progression (doubling in size: ctrl = 104d; AI = 130d, log rank vs ctrl p=0.11; talazoparib = 121d, p=0.68; AI + talazoparib = >153d, p=0.0018; survival study ongoing). Our work suggests that despite the absence of the BRCA-like phenotype, ILC have a previously unappreciated form of DNA repair dysfunction that makes cells sensitive to PARPi. PARPi represent potential first-in-class precision treatments specifically targeting ILC biology. Understanding HR dysfunction in ILC can broaden PARPi use paradigms and build new conceptual frameworks for understanding DNA repair capacity and associated therapeutic vulnerability.
 Citation Format: Matthew Sikora, Joseph L. Sottnik, Madeleine T. Shackleford, Jordan M. Swartz, Camryn S. Nesiba, Carmen E. Rowland, Margaret Musick, Sanjana Mehrotra, Jennifer R. Diamond. Redefining PARP inhibitor paradigms to target DNA repair dysfunction in lobular breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b9a02a56d02ead11fe4498d60b93da534033e9" target='_blank'>
              Abstract P5-02-29: Redefining PARP inhibitor paradigms to target DNA repair dysfunction in lobular breast cancer
              </a>
            </td>
          <td>
            Matthew Sikora, Joseph L. Sottnik, Madeleine T. Shackleford, Jordan M. Swartz, Camryn S. Nesiba, Carmen E. Rowland, Margaret Musick, Sanjana Mehrotra, Jennifer R. Diamond
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The cancer cell cycle possesses an incredible capacity to adapt or change in response to oncogenic mutations or drug treatment (cell cycle “plasticity”) to generate a diversity of solutions to the cell cycle (distinct cell cycle “programs”) that differ in key regulatory effectors and mechanisms. To study the extent and role of cell cycle plasticity in tumor progression and drug resistance, we developed cell cycle mapping - a single-cell proteomic imaging platform that combines highly multiplexed immunofluorescence with manifold learning to visualize cell cycle plasticity among and within tumors. Applying cell cycle mapping to a large library of hormone receptor (HR)+ breast tumors, we observed 4 distinct cell cycle programs driving tumor growth. These programs differed in the expression/activity of key cyclins and CDKs. In many cases, we observed multiple cell cycle programs within a single tumor correlating with differences in the cellular composition of the local tumor environment. These results also highlight the challenge of targeting the cancer cell cycle (e.g. with CDK4/6 inhibitors), given the heterogeneity of cell cycle programs actin within a single tumor. To further explore the role of cell cycle plasticity in targeted therapy, we used a combination of cell cycle mapping and time-lapse imaging to reveal how the breast cancer cell cycle responds to a panel of FDA-approved drugs targeting a range of cell cycle effectors, including multiple CDK inhibitors. We observed clear heterogeneity in the mechanisms of cell cycle adaptation executed across a panel of cell lines modeling distinct molecular (HR+, HER2+, basal) and histological subtypes (invasive ductal carcinoma, invasive lobular carcinoma) of breast cancer. Altered expression/activity of key cell cycle regulators provides a new “path” through the cell cycle, a molecular detour that allows cells to continue proliferating in the presence of drug to drive therapeutic resistance.


 Yian Yang, Thu Pham, Janet McLaughlin, Steffi Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert. Cell cycle plasticity drives tumor progression and drug resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY22-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="PURPOSE Regulator of chromosome condensation 1 (RCC1) and RCC2 have been shown to play important roles in the regulation of cell cycle, DNA damage response, and nucleocytoplasmic transport. MATERIALS AND METHODS DNA (592-gene or whole exome) and RNA (whole transcriptome) sequencing was performed at Caris Life Sciences (Phoenix, AZ). Samples were stratified by RCC1 expression quartile thresholds (Q1: low, Q4: high) for small cell lung cancer (SCLC; n = 876), non–small cell lung cancer (NSCLC; n = 21,603), gastric cancer (GC; n = 1,908), pancreatic cancer (PC; n = 5,071), and colorectal cancer (CRC; n = 14,892). Statistical significance was determined using chi-square and Wilcoxon rank-sum tests and adjusted for multiple comparisons (*P < .05). Corresponding analyses were run for RCC2. RESULTS Median RCC1 mRNA expression was highest in SCLC (14.3 transcript per million [TPM]), followed by GC (9.9), NSCLC (9.9), CRC (9.8), and PC (6.9). Similar to RCC1, the median RCC2 expressions were highest in SCLC (36.2 TPM). Tumor mutational burden-high rates were positively associated with increasing RCC1 expression quartiles (Q1-4) in NSCLC (31%-41%), GC (7%-22%), and CRC (5%-17%) and with increasing RCC2 expression in NSCLC and CRC only. Higher expression with RCC1 and RCC2 was associated with worse overall survival in NSCLC (hazard ratio [HR] for RCC1 and RCC2 were 1.3 and 1.3, respectively), PC (HR for RCC1 and RCC2 were 1.5 and 1.12, respectively), and CRC (HR for RCC1 and RCC2 were 1.3 and 1.03, respectively). CONCLUSION RCC1 and RCC2 expression is a negative prognostic marker in NSCLC, PC, and CRC. Further studies to investigate RCC1 and RCC2 function at the molecular level may provide opportunities for novel targeted drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33ff6d06df5be38cb6ff5a68f5198616070f9b7f" target='_blank'>
              Multiomic Characterization of RCC1 and RCC2 Expression and Their Association With Molecular Alterations, Immune Phenotypes, and Cancer Outcomes
              </a>
            </td>
          <td>
            M. Nagasaka, Brian Warnecke, S. Ou, Sahar F. Bannoura, Chul Kim, A. Elliott, B. Halmos, Dave S.B. Hoon, S. Darabi, A. Sukari, M. Radovich, E. Lou, George W Sledge, W. El-Deiry, M. Al-Hallak, Boris C. Pasche, A. Azmi
          </td>
          <td>2025-05-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The ATP-hydrolytic ectoenzyme ENPP1 has been implicated in the metastasis and recurrence in triple-negative breast cancer (TNBC), primarily by contributing to tumor cell survival and treatment resistance. However, the precise mechanisms remain unclear. In a model of local recurrence (LR), circulating tumor cells (CTC) engrafting in the post-resection tumor bed developed a radioresistant phenotype linked to an ENPP1+-gene signature which was also identified in TNBC patients, suggesting ENPP1´s role in genome integrity. Blockade of ENPP1 using a permeable ENPP1 inhibitor (AVA-NP-695) reduced radioresistance, mechanistically attributed to decreased homologous recombination (HR) resulting in persistent DNA damage, as evidenced by enhanced tail moment and sustained γH2AX formation. This impaired DNA damage repair (DDR) sensitized tumor cells to ionizing radiation (IR). Notably, several DDR inhibitors (i) (including PARPi and ATMi) showed the highest synergy score in a targeted pharmacological screening. In vivo, dual ENPP1/ATM inhibition heightened radiosensitivity, compromised tumor cell survival and enhanced STING-TBK1 signaling by preventing ENPP1-mediated cGAMP hydrolysis. This resulted in robust innate and long-lasting adaptive antitumor immune memory responses, leading to significant tumor regression. Remarkably, combined treatment post-IR reduced spontaneous metastasis and local recurrence, and induced abscopal effects that impacted distant tumor spread in orthotopic tumor models. Thus, these findings position ENPP1 as a critical link between genome integrity and immunosuppression, offering promising translational opportunities for treating local or distant dissemination in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34af6ea74bf5effdace3557bda27f7680f31a271" target='_blank'>
              Dual ENPP1/ATM depletion blunts DNA damage repair boosting radioimmune efficacy to abrogate triple-negative breast cancer
              </a>
            </td>
          <td>
            Borja Ruiz-Fernández de Córdoba, K. Valencia, C. Welch, Haritz Moreno, S. Martínez-Canarias, C. Zandueta, Eduardo Gómez, Alfonso Calvo, N. Otegui, M. Echepare, Ignacio Garzón, D. Ajona, David Lara-Astiaso, Elisabet Guruceaga, Laura Guembe, Ruben Pio, I. Melero, Silve Vicent, Fernando Pastor, R. Martinez-Monge, F. Lecanda
          </td>
          <td>2025-06-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="MYCN, or N-Myc, is a member of the MYC family of transcription factors, which plays a key role in tumor formation by regulating genes involved in proliferation, differentiation, and apoptosis. MYCN is essential for neural development, especially for the appropriate growth and differentiation of neural progenitor cells, and its aberrant expression contributes to tumorigenesis. Gene amplification and mutations of this gene have been observed in a wide variety of cancer types, particularly in pediatric brain and non-brain tumors, such as neuroblastoma. Previous studies have provided extensive insights into the complex regulatory network of this transcription factor. Additionally, the presence of MYCN alterations in patient tumors serve as a key factor for risk stratification, as it correlates with poorer outcomes, and presents a significant challenge for treatment. Despite its clinical significance, therapeutic targeting of MYCN is challenging due to its structure, nuclear localization, and complex regulatory pathways. Efforts to target MYCN have focused on destabilizing the protein, modulating epigenetic mechanisms, and disrupting its transcriptional network. This review explores the role of MYCN in different subtypes of pediatric brain tumors and highlights novel ongoing therapeutic approaches. However, further research is necessary to develop more effective therapies and improve survival outcomes for patients with MYCN-driven tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6157e6a11e8780346fdeb79b9c8d51cc080befb3" target='_blank'>
              MYCN as an oncogene in pediatric brain tumors
              </a>
            </td>
          <td>
            Adriana Fernandez Garcia, Jayden Jackson, Poorvi Iyer, Elissa G. Oliver, Kosuke Funato
          </td>
          <td>2025-04-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The overexpressed gene MYEOV in multiple myeloma, as an oncogene, has been widely recognized for its high expression levels in various malignant tumors. MYEOV plays a significant role in multiple malignancies, particularly in diseases such as multiple myeloma, breast cancer, lung cancer, pancreatic cancer, and esophageal cancer. The presence of the open reading frame of MYEOV in humans and other primates suggests its potential protein-coding capacity, although direct evidence of functional MYEOV protein is currently lacking. The role of MYEOV in various tumors is not limited to its direct effects as an oncogene; it also includes its complex role in tumor cell signaling pathways and its ability to participate in miRNA regulatory networks as a competing endogenous RNA (ceRNA). Studies have shown that MYEOV may affect the expression of cancer-related genes through enhancer activity. This article aims to provide a comprehensive review of the role of MYEOV, its involvement in signaling pathways in tumor cells, and the latest advancements in MYEOV-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e799fb69a5049d179cc77bd3716c162048abf1c" target='_blank'>
              The role of MYEOV gene: a review and future directions
              </a>
            </td>
          <td>
            Yidan Xi, Xuefei Liu, Yanan Ge, Mingzhe Jiang, Dong Zhao, Ershu Zhao, Fei Cai, Xinlong Wang, Jiatong Li, Tingting Li, Zhendong Zheng
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Summary: The term “anaplasia” was coined in 1890 to describe chromosomal changes common to primordial cancer cells, but ever since the mechanisms whereby a cell becomes anaplastic has been the subject of much speculation. Recent results based on genomic and epigenomic profiles of cancer patient samples provide a glimpse into early events that lead to aneuploidy, the original defining feature of an anaplastic cell. We propose that the anaplastic cell is one in which RNA Polymerase II hypertranscribes S-phase-dependent histone genes, and the resulting histone excess facilitates DNA replication while competing for CENP-A, causing centromere breaks that initiate whole-arm aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e327392e26ad44de92c835846a2ea7101b5c4f9e" target='_blank'>
              Hansemann’s anaplastic theory of cancer after 135 years
              </a>
            </td>
          <td>
            S. Henikoff, K. Ahmad
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA) is present in food, water, and drugs and is considered a probable human carcinogen by the International Agency for Research on Cancer. The mechanism of action of NDMA involves the generation of carcinogenic methyl lesions such as 3-methyladenine (3MeA) on DNA bases. Alkyladenine DNA Glycosylase (AAG) removes 3MeA to initiate Base Excision Repair, leaving an intermediary lesion that is subsequently resolved by backbone cleavage, nucleotide insertion, and backbone ligation. The intermediate steps following lesion removal produce potentially toxic and mutagenic single-strand DNA breaks. Here, we explored differences between males and females regarding downstream DNA damage, toxicity, mutations and cancer arising from 3MeA in the livers of WT, Aag-/-, and Aag-overexpressing (AagTg) mice. We found that males were more susceptible to NDMA-induced mutations (WT and Aag-/-) and cancer (all genotypes). In contrast, AagTg females were more prone than males to micronucleus induction. As we showed in our prior analyses where data were pooled for males and females, Aag-/- mice were significantly more susceptible to NDMA-induced mutations and cancer, and AagTg mice displayed significantly greater toxicity. Building on these findings, our analyses of sex-related differences show that Aag deficiency and maleness are both susceptibility factors for NDMA-induced liver cancer, while Aag overexpression drives toxicity, potentially with a greater effect on females. By assessing differences between males and females, this study reveals a deeper mechanistic understanding of the underpinnings for a well-known increased risk of liver cancer in men versus women by demonstrating a higher susceptibility of male mice to both mutations and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2ad8f9213bd9ddfa98933a729d0b81e465af3f" target='_blank'>
              Sex and Alkyladenine DNA Glycosylase Expression are Key Susceptibility Factors for NDMA-induced Mutations, Toxicity, and Cancer
              </a>
            </td>
          <td>
            Jennifer E. Kay, J. Corrigan, Lindsay B Volk, Amanda L Armijo, Ilana S Nazari, D. Torous, Svetlana L. Avlasevich, R. G. Croy, D. Wadduwage, S. Dertinger, J. Essigmann, Leona D. Samson, B. Engelward
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The tumor suppressor gene NKX3.1 and the LPL gene are located in close proximity on chromosome 8, and their deletion has been reported in multiple studies. However, the significance of LPL loss may be misinterpreted due to its co-deletion with NKX3.1, a well-established event in prostate carcinogenesis. This study investigates whether LPL deletion represents a biologically relevant event or occurs merely as a bystander to NKX3.1 loss. We analyzed 28 formalin-fixed paraffin-embedded prostate cancer samples with confirmed LPL deletion and 28 without. Immunohistochemical staining was performed, and previously published whole-genome sequencing data from 103 prostate cancer patients were reanalyzed. Deletion of the 8p21.3 region was associated with higher Gleason grade groups. While NKX3.1 expression was significantly reduced in prostate cancer compared to benign prostatic hyperplasia, LPL protein expression showed no significant difference between cancerous and benign tissue, nor was it affected by the 8p21.3 deletion status. Copy number analysis confirmed the co-deletion of NKX3.1 and LPL in 54 patients. Notably, NKX3.1 loss without accompanying LPL deletion was observed in eight additional cases. These findings suggest that LPL deletion is a passenger event secondary to NKX3.1 loss and underscore the importance of cautious interpretation of cytogenetic findings involving the LPL locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74796121b67d50bf9eb77e02037db96e6cc25807" target='_blank'>
              Deletions of LPL and NKX3.1 in Prostate Cancer Progression: Game Changers or By-Standers in Tumor Evolution
              </a>
            </td>
          <td>
            Tereza Vodičková, Mária Wozniaková, Vladimír Židlík, Jana Žmolíková, Jana Dvořáčková, Adéla Kondé, Jana Schwarzerová, Michal Grepl, J. Bouchal
          </td>
          <td>2025-05-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7be43f99c39d72183ee247e245dd3934afb3150" target='_blank'>
              Decoding meningioma prognosis with multi-omics: macrophage diversity, immune-CNV interplay, and novel SPP1-targeted strategies.
              </a>
            </td>
          <td>
            Hailang Fan, Xiaojie Li, Yaqian Zhao, Lairong Song, Lanlan Sun, Zhen Wu, Junting Zhang, Haikun Zhang, Feifan Zhang, Yingdi Huang, Dake Zhang, Liang Wang
          </td>
          <td>2025-06-16</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA topoisomerases, particularly type II, are crucial for DNA processes, such as replication, transcription, and chromosome segregation, making them prime targets for cancer therapy. This review delves into the multifaceted mechanisms of action of type II topoisomerases, highlighting their essential roles beyond cancer progression. It explores recent advancements in screening and designing metallic complexes as inhibitors of topoisomerase II activity. Emphasizing the structural and functional diversity between alpha and beta isoforms, it elucidates their significance in DNA metabolism and genome integrity. Additionally, this review discusses the interplay of topoisomerase II with cellular components, underscoring its regulatory roles in gene expression. Insights into screening and design strategies for metallic complex inhibitors are provided, showcasing their therapeutic potential against cancer. Overall, this review highlights the importance of understanding topoisomerase II inhibition mechanisms and the versatility of metallic complexes in biomedical research, paving the way for novel therapeutic strategies and broader applications beyond cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a3a5eac5f306d411595288c7868532f4247f662" target='_blank'>
              Topoisomerase II Inhibition in Cancer: A Focus on Metal Complexes.
              </a>
            </td>
          <td>
            A. O. Akinyemi, Josias da Silveira Rocha, Gabriela Porto de Oliveira, Josenilton de Jesus Santos, B. Owoyemi, Fillipe Vieira Rocha
          </td>
          <td>2025-06-05</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The D4Z4 locus is a macrosatellite array on chromosome 4q that normally comprises 8 to >100 3.3-kb repeat units. Its size and repetitiveness render it refractory to most sequencing technologies; consequently its genetic and epigenetic architectures remain incompletely understood despite their relevance to human health, in particular facioscapulohumeral muscular dystrophy (FSHD). Molecular diagnosis for FSHD following clinical description currently involves complex, multi-step and low resolution assays, aiming at identifying contractions on permissive haplotypes (FSHD type 1) or epigenetic reactivation (FSHD type 2) due to pathogenic variants in the epigenetic machinery (most often in SMCHD1).Here we leverage ultra-long whole-genome and Cas9-targeted sequencing to develop a fast and accurate workflow, D4Z4End2End, to comprehensively charactere the genetics and methylation of D4Z4 alleles. We apply it to samples from patients affected by FSHD1, FSHD2, and another disease caused by SMCHD1 variants, Bosma arhinia microphthalmia syndrome (BAMS), as well as publicly-available data from the 1000 Genomes Project and Human Pangenome Reference Consortium. We attain high read depth sequencing of full-length D4Z4 arrays of up to 40 repeat units (~132 kb), accurately capture contracted arrays, genetic mosaicism, and pathogenic SMCHD1 variants, and generate accurate consensus sequences of the full set of D4Z4 alleles for variant analysis. Moreover, we identify new allelic variants, analyse complex D4Z4 rearrangements including in-cis duplications, and reveal striking length- and SMCHD1 status-dependent methylation patterns across the D4Z4 array. Our findings provide new insights into human macrosatellite genetics and epigenetics, and demonstrate the potential of long-read nanopore sequencing to accelerate FSHD research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a67b65544abaa38aef2d4ff63b752b6fdbd016b" target='_blank'>
              D4Z4End2End: complete genetic and epigenetic architecture of D4Z4 macrosatellites in FSHD, BAMS and reference cohorts
              </a>
            </td>
          <td>
            Lucinda C. Xiao, Ayush Semwal, Brianna St, Kathleen Zeglinski, Shian Su, James Lancaster, Shifeng Xue, B. Reversade, Matthew E. Ritchie, Frédérique Magdinier, M. Blewitt, Quentin Gouil
          </td>
          <td>2025-04-26</td>
          <td>None</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs), among the most deleterious forms of DNA damage, are primarily repaired by non-homologous end-joining or homologous recombination repair in mammalian cells. Although DSB repair is regulated by various posttranslational modifications, such as phosphorylation, the contribution of protein arginine methylation, catalyzed by protein arginine N-methyltransferases (PRMTs), is less understood. To explore this, we conducted a screen of human PRMTs for their recruitment to the DSB sites in living cells. Among the hits, PRMT4 (also known as coactivator-associated arginine methyltransferase 1: CARM1) was found to accumulate at the DSB sites within 1 min, with the signal dissipating by 10 min post-damage. Further analysis revealed that PRMT4 recruitment to DSB sites depended on its interaction with PARP1, in which both N- and C-terminal domains of PRMT4 were required. In addition, the catalytic activity of PRMT4 was indispensable for releasing PRMT4 from the DSB sites. Finally, the PRMT4 knockout cell line exhibited delayed DSB repair, as evidenced by the neutral comet assay. Consistently, this cell line displayed a higher residual γH2AX signal compared to the parental cells following phleomycin treatment. Collectively, our findings extend the list of PRMTs involved in maintaining genome integrity and identify PRMT4 as a novel factor promoting DSB repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec4b09c7d6c576bdaf67e9c1c84a1dda0dc8942" target='_blank'>
              PRMT4/CARM1 Is a Novel Factor Promoting DNA Double-Strand Break Repair.
              </a>
            </td>
          <td>
            Yurina Abe, Hayaki Ikegame, Yuina Tsuchiya, Ryotaro Nishi
          </td>
          <td>2025-06-02</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="To protect genome integrity from pervasive threats of damage and prevent diseases like cancer, cells employ an integrated network of signalling pathways called the DNA damage response. These pathways involve both protein and RNA components, which can act within the damaged cell or be transferred intercellularly to influence population-wide responses to damage. Here, we show that radioprotection can be conferred by damage-derived exosomes and is dependent on YBX1-packaged Y3-derived ysRNA. In recipient cells, ysRNAs are methylated on cytosine and bound by m5C reader, YBX1. YBX1/ysRNA localise at double strand break (DSB) sites to promote efficient DNA repair and cell survival through complex formation with PARP1. YBX1 modulates PARP1 auto-modification by facilitating ysRNA ADP-ribosylation, promoting increased PARP1 residency at DSBs. Our data highlight an unprecedented role for these under-studied species of small non-coding RNAs, identifying them as a novel substrate for PARP1 mediated ADP-ribosylation with a function in DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b129803142b95daeca0e6c5e6ddbee0e7b9c7d8" target='_blank'>
              Non-coding Y RNA fragments in a complex with YBX1 modulate PARP1 residency at DNA double strand breaks.
              </a>
            </td>
          <td>
            Annabelle Shaw, Kamal Ajit, Manon Chataignier, M. Gullerová
          </td>
          <td>2025-06-06</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Many eukaryotic organisms, from ciliates to mammals, employ programmed DNA elimination during their postmeiotic reproduction. The process removes specific regions from the somatic DNA and has broad functions, including the irreversible silencing of genes, sex determination, and genome protection from transposable elements or integrating viruses. Multiple mechanisms have evolved that explain the sequence selectivity of the process. In some cases, the eliminated sequences lack centromeres and are flanked by conserved sequence motifs that are specifically recognized and cleaved by designated nucleases. Upon cleavage, all DNA fragments that lack centromeres are lost during the following mitosis. Alternatively, specific sequences can be destined for elimination by complementary small RNAs (sRNAs) as in some ciliates. These sRNAs enable a PIWI-mediated recruitment of chromatin remodelers, followed up by the precise positioning of a cleavage complex formed from a transposase like PiggyBac or Tc1. Here, we review the known molecular interplay of the cellular machinery that is involved in precise sRNA-guided DNA excision, and additionally, we highlight prominent knowledge gaps. We focus on the modes through which sRNAs enable the precise localization of the cleavage complex, and how the nuclease activity is controlled to prevent off-target cleavage. A mechanistic understanding of this process could enable the development of novel eukaryotic genome editing tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561cdb775ea226e1249d7f24f55971f03ff3eed2" target='_blank'>
              Functions and mechanisms of eukaryotic RNA-guided programmed DNA elimination.
              </a>
            </td>
          <td>
            Bozhidar-Adrian Stefanov, M. Nowacki
          </td>
          <td>2025-04-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Background: Breast cancer (BC) stands as a prominent contributor to cancer-related fatalities on a global scale. Traditionally, the primary focus in assessing recurrence risk and guiding treatment decisions has revolved around scrutinizing the primary tumor’s histopathological characteristics and molecular compositions. However, recent findings suggest that exploring normal tissue adjacent to the tumor (NAT) could unveil valuable insights into the intricate characteristics of BC advancement and patient prognoses. Specifically, in our recent work, we discovered three phylogenetic tree groups of NAT and tumor tissues that were associated with distinct tumor and tumor microenvironment (TME) features using the whole genome sequencing data collected from 43 Hong Kong BC patients (Zhu et al. under review). Here, we aimed to further characterize the molecular heterogeneity of NAT by integrating methylation and gene expression profiling data in an expanded sample set, which may deepen our understanding of tumor evolution. Methods: Genome-wide DNA was extracted from fresh frozen paired tumor/NAT tissues in 188 patients diagnosed with breast cancer and treated in Hong Kong, China. DNA was then profiled using the Infinium Methylation 850K BeadChip array. Horvath clock was utilized to estimate epigenetic age, and MethylCIBERSORT was used to estimate cellular composition using the methylation data. For a subset of the patients (N= 76), we also analyzed RNA sequencing (RNA-Seq) data and used CIBERSORTx to estimate the proportions of immune cell subpopulations in NAT samples. Risk factor and clinical information was available for most patients. Results: The uniform manifold approximation and projection (UMAP) identified two distinct clusters of NAT samples (cluster 1, N= 139; cluster 2, N= 49) based on the top 10K most variable methylation probes. The two clusters were recapitulated by unsupervised clustering. The two sets of patients did not differ by chronological age at diagnosis, but when investigating epigenetic aging - as captured by Horvath clock – we found that cluster 2 displayed a lower age acceleration in NAT compared to cluster 1. Among the subset of patients with phylogenetic trees constructed in our previous work, we observed cluster 1 had a higher frequency of patients (64.3%) that showed distinct evolutionary trajectories in matched NAT and tumor samples (multiple-tree group) compared to cluster 2 (33.3%). Consistent with previous findings, we found that cluster 1, enriched with multiple-tree NAT samples, was associated with lower proportions of CD14 cell population (P=1.6x10-6), monocytes (P=7.1x10-6), and M2 macrophages (P=2.3x10-7) compared to cluster 2. Interestingly, genomic features in the matched tumors such as PAM50 subtype, TP53 mutation status, or tumor mutational burden, did not vary significantly between clusters 1 and 2. Lastly, established BC risk factors examined in this study did not show significant differences by this methylation-based cluster assignment. Conclusion: Using methylation data in an expanded dataset, we extended our previous analysis of paired tumor and NAT samples to characterize breast cancer field effect. Our main findings illuminate the diverse nature of the microenvironment surrounding the tumor and its potential influence on the genomic evolutionary path of the tumor.
 Citation Format: Hela Koka, Bin Zhu, Priscilla Lee, Kevin Wang, Avraam Tapinos, Difei Wang, Gary M. Tse, Koon-ho Tsang, Cherry Wu, Chad A. Highfill, Kristine Jones, Belynda Hicks, Amy Hutchinson, Montserrat Garcia-Closas, Stephen Chanock, David C. Wedge, Lap Ah Tse, Xiaohong R. Yang. Molecular heterogeneity in adjacent normal tissue among Chinese breast cancer patients [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f57f67b5c54214463999793ad38b86bf56cb8b35" target='_blank'>
              Abstract P2-02-06: Molecular heterogeneity in adjacent normal tissue among Chinese breast cancer patients
              </a>
            </td>
          <td>
            H. Koka, Bin Zhu, Priscilla Lee, Kevin Wang, Avraam Tapinos, Difei Wang, G. M. Tse, Koon-Ho Tsang, Cherry Wu, Chad A. Highfill, Kristine Jones, B. Hicks, A. Hutchinson, Montserrat García-Closas, Stephen Chanock, David C Wedge, L. Tse, Xiaohong R. Yang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a pervasive feature of cancer, promotes tumor evolution and inflammatory signaling, yet its influence on innate immune sensing remains incompletely understood. Ruptured micronuclei, a direct byproduct of CIN arising from missegregated chromosomes, expose out‐of‐context double‐stranded DNA that engages the cGAS‐STING pathway. In their recent study, Sasaki et al. show that micronuclei are also a source of immunogenic double‐stranded RNA (dsRNA), triggering MAVS‐dependent type I interferon responses independently of STING. The authors show that micronuclei undergo aberrant transcription, producing dsRNA from nonexonic, transcriptionally accessible loci, with many species localizing near interferon‐stimulated genes. This work suggests a feedforward loop in which type I interferon signaling reinforces its own activation through transcriptional dysregulation. Using MPS1 inhibition to induce acute CIN, the authors show that MAVS signaling promotes MHC Class I expression and immune cell recruitment. These findings reposition CIN as a dual trigger of innate immunity through cytoplasmic DNA and RNA sensing. Future work should define how these pathways integrate in the context of chronic CIN and evaluate strategies to target DNA and RNA sensing in immune‐edited tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828c0bc85c3a2a64866df460ecbc5473d78feef8" target='_blank'>
              CINs of the cytoplasm: dissecting dsRNA signaling in chromosomal instability
              </a>
            </td>
          <td>
            Aglaia Skolariki, Rose L Jady-Clark, Eileen E Parkes
          </td>
          <td>2025-05-07</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pediatric acute myeloid leukemias (AML) are aggressive malignancies characterized by rapid progression, therapy resistance, and poor outcomes, frequently originating during fetal development or early infancy. Unlike adult leukemias, which generally arise from a decades-long accumulation of genetic mutations, pediatric leukemias emerge following genetic alterations occurring prenatally or shortly after birth. However, the biological mechanisms underlying this age-dependent susceptibility and aggressiveness remain largely unexplained. To address this knowledge gap, we developed a novel humanized model of pediatric AML driven by the NUP98::NSD1 fusion oncogene, clinically associated with chemotherapy resistance and dismal prognosis, particularly when accompanied by mutations in the transcription factor WT1. Using CRISPR/Cas9 genome engineering to induce endogenous expression of this fusion oncogene directly in human primary hematopoietic stem and progenitor cells (HSPCs) isolated from distinct developmental stages, we demonstrate that leukemia initiation, lineage specification, and therapeutic response critically depend on the developmental stage of the initiating cells. We found that fetal-derived HSPCs readily underwent leukemia transformation upon expression of the NUP98::NSD1 fusion, especially when co-occurring with WT1 mutations. In contrast, postnatal HSPCs exhibited progressively diminished leukemia transformation potential. Single-cell RNA-seq and ATAC-seq analyses uncovered distinct transcriptional and epigenetic landscapes associated with developmental stage-specific leukemia initiation, leukemia stem cell hierarchies, and therapeutic vulnerabilities between fetal and postnatal-origin leukemias. Clinically, NUP98-rearranged AML exhibiting fetal-like transcriptional programs correlated strongly with worse patient outcomes. Furthermore, fetal-derived leukemias displayed unique metabolic dependencies, contributing to enhanced therapy resistance. Leveraging these insights, we identified and therapeutically targeted vulnerabilities including oxidative phosphorylation and anti-apoptotic mechanisms. Indeed, therapeutic strategies exploiting these developmental-stage-specific susceptibilities effectively overcame resistance in combination therapies and significantly reduced leukemia burden in our preclinical and patient-derived xenograft models. Together, our findings reveal an unappreciated developmental dimension in leukemia biology, illustrating how developmental context critically shapes leukemia potential, lineage fate decisions, and therapeutic response - even among leukemias with identical oncogenic mutations. This work strongly supports the development of age-specific precision oncology strategies to overcome therapy resistance and improve survival outcomes in pediatric patients with aggressive fusion-driven leukemias.


 Elvin Wagenblast. Developmental dependency and metabolic vulnerabilities in NUP98-rearranged pediatric leukemia [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY34-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a04310544d53b394a40ce917fc33a338d122cf3" target='_blank'>
              Abstract SY34-01: Developmental dependency and metabolic vulnerabilities in NUP98-rearranged pediatric leukemia
              </a>
            </td>
          <td>
            Elvin Wagenblast
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Malignant rhabdoid tumor of the kidney (MRTK) is a rare renal tumor with poor prognosis. While germline mutations of SMARCB1 are considered to be the primary cause of MRTK, emerging evidence suggests that somatic epigenetic changes also play a vital role in the development and progression of MRTK. YTHDF1, an m6A reader protein, has been implicated in regulation of tumorigenesis by influencing RNA translation and stability in several adult cancers. However, the exploration of the role of YTHDF1 in pediatric cancer, especially MRTK, remains limited. Methods In this study, CRISPR/Cas9 was employed to knockout (KO) YTHDF1 in G401 cells. The impact of YTHDF1 on the cell growth and chemoresistance were assessed using CCK-8 assays. Western blot and qRT-PCR were used to determine the changes in ferroptosis marker gene expression. Additionally, 4D-label free quantitative proteomics was conducted to uncover alterations by YTHDF1 deletion. Results We observed that the deletion of YTHDF1 in the MRTK cell line led to a significant reduction in malignancy-associated characteristics, including decreased cell motility, invasive growth, and chemoresistance. Quantitative proteomic analysis revealed that the glutathione-related signaling pathway was notably affected by YTHDF1 KO. Specifically, YTHDF1 KO resulted in a reduction of both mRNA and protein levels of Glutathione S-Transferase Mu 2 (GSTM2), a phase II metabolizing enzyme responsible for conjugating glutathione to electrophilic compounds. The decrease in GSTM2 levels following YTHDF1 KO increased the susceptibility of MRTK cells to ferroptosis. Notably, overexpression of GSTM2 in YTHDF1 KO cells partially restored the oncogenic phenotype of MRTK cells, underscoring its role in MRTK progression. Conclusions In summary, our findings provide new insights into the molecular mechanisms driving MRTK progression, highlighting YTHDF1 and GSTM2 as potential therapeutic targets for this aggressive pediatric renal tumor. Graphical Abstract YTHDF1 is highly expressed in MRTK cells. YTHDF1 recognizes the m6A modification on GSTM2 mRNA, enhancing its stability. GSTM2 protects MRTK cells against ferroptosis independent of GPX4. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10049-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248db6cfe415c7d8ca495a38efdbd0087162d544" target='_blank'>
              Loss of YTHDF1 suppresses the progression of malignant rhabdoid tumor of the kidney by regulating Glutathione S-transferase Mu 2 (GSTM2)
              </a>
            </td>
          <td>
            Qian-Wen Xiong, Yuntao Liu, Min He, Xiao-Die Shen, Manli Zhao, Shuang-Ai Liu, Gensheng Zhang, Qian Liu, Jinhu Wang, Wan-Xin Peng
          </td>
          <td>2025-06-07</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Centromere protein F (CENP‐F) is an important nuclear matrix protein that regulates mitosis and the cell cycle, and plays a crucial role in recruiting spindle checkpoint proteins to maintain the accuracy of chromosome segregation. Studies have shown that CENP‐F is closely involved in the pathogenesis of various diseases, particularly in the development and progression of malignant tumors, where it exhibits significant oncogenic activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e907c4941d09e4b8b2b05c38929140acf5c506d" target='_blank'>
              Centromere Protein F in Tumor Biology: Cancer's Achilles Heel
              </a>
            </td>
          <td>
            Zitong Wan, Miaomiao Wen, Chunlong Zheng, Ying Sun, Yinxi Zhou, Yahui Tian, Shaowei Xin, Xuejiao Wang, Xiaohong Ji, Jie Yang, Yanlu Xiong, Yong Han
          </td>
          <td>2025-05-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ef251e1dfb7904ef7b82484076e545a6c98075" target='_blank'>
              DNA remnants in red blood cells enable early detection of cancer.
              </a>
            </td>
          <td>
            Haobo Sun, Xingyun Yao, Yurong Jiao, Xiangxing Kong, Yuehua Han, Ying Li, Jianping Ge, Yanfei Cao, Hongsheng Lu, Pingli Wang, Yu Xu, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-05-09</td>
          <td>Cell research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Epithelial ovarian cancer (EOC) is a leading cause of cancer mortality in women, often diagnosed at advanced stages. While first-line treatments improve survival, relapses remain common, with 5-year survival rates below 40%. Circulating tumor DNA (ctDNA) is a promising biomarker for non-invasive EOC detection and monitoring. It may help assess treatment response, notably microscopic residual disease. Our objective was to compare two ctDNA characterization strategies in EOC for assessing tumor burden during first-line treatment: a tumor-informed approach based on somatic mutations and a tumor-type informed approach utilizing DNA methylation patterns. Methods In the tumor-informed approach, whole exome sequencing (WES) was performed on EOC tumor DNA and matched PBMCs from 22 patients to identify tumor-specific mutations. Personalized panels were then designed to track these mutations in plasma cfDNA. In the tumor-type informed approach, differentially methylated loci (DMLs) were identified by comparing EOC samples, healthy ovarian tissues, and PBMCs. A unique custom methylation panel was designed, and a support vector machine classifier was trained to distinguish between methylation profiles in plasma cfDNA from healthy donors and from EOC patients. Plasma samples from 47 advanced-stage EOC patients receiving chemotherapy and 54 healthy subjects were analyzed. Results For the tumor-informed approach, WES identified an average of 72 somatic mutations per patient. For the tumor-type informed approach, 52,173 DMLs were identified as tumor-specific markers. In 47 plasma samples tested by both approaches, ctDNA levels were significantly correlated (R = 0.56, p = 4.3 × 10-5), with 70.2% concordance in detection. At baseline, ctDNA was detected in 21/22 patients with the tumor-informed approach, and in 11/12 non-training baseline samples with the tumor-type-informed classifier. At end-of-treatment, the latter detected ctDNA in 16/22 samples, outperforming the former. Detection using this more sensitive approach was significantly associated with relapse (log-rank p = 0.009; hazard ratio = 9.44; 95% CI 1.22–73.26) and poorer overall survival (log-rank p = 0.041). Conclusion The tumor-type informed classifier demonstrated sensitivity and specificity for ctDNA detection, outperforming the tumor-informed approach in monitoring EOC progression. Requiring fewer sequencing data, it offers a practical, efficient solution for clinical management of EOC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03433-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/585c3fe59d51d5d987c4d3714b4b1b5d474cd54f" target='_blank'>
              Improved tumor-type informed compared to tumor-informed mutation tracking for ctDNA detection and microscopic residual disease assessment in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Mehdi Ben Sassi, Henri Azais, Charles Marcaillou, Sylvain Guibert, Emmanuel Martin, Jérôme Alexandre, Louise Benoit, A. de Reyniès, Émilie Laude, Cam Duong, Jacques Medioni, Bruno Borghese, A. Bats, V. Taly, Pierre Laurent-Puig
          </td>
          <td>2025-06-12</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f31d8e3aa41a4d4ed605eb1a2bff762b1dfaf76" target='_blank'>
              NUSAP1 regulates mitotic processes via KIF2C interaction and AURKA phosphorylation in primary microcephaly
              </a>
            </td>
          <td>
            Decheng Ren, Keyi Li, Zhen Liu, Yan Bi, Liangjie Liu, Lei Ji, Ke Yang, Yingying Luo, Le Luo, Yang Yan, Yang Li, Feng-ping Yang, Hua Wang, Guang He, Xiao Mao
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare, aggressive cancer affecting young women, driven by inactivating mutations in SMARCA4, a key SWI/SNF chromatin remodeling gene. To uncover its druggable vulnerabilities, we performed a compound screen and found that SCCOHT cells and tumors were sensitive to PARP inhibitors. Paradoxically, SCCOHT displayed BRCA-deficient traits despite retaining wild-type BRCA1 expression. Elevated R-loop in SCCOHT sequestered BRCA1, limiting its availability for DNA damage repair. Proximity-dependent biotin identification revealed that wild-type SMARCA4, but not its pathogenic variants, promoted RNA polymerase II (Pol II) elongation by mediating the assembly of the polymerase-associated factor 1 (PAF1) complex. Thus, SMARCA4 loss increased Pol II pausing, resulting in elevated R-loops and BRCA1 redistribution. The suppression of BRCA1 activity sensitized SMARCA4-deficient SCCOHT cells and tumors to PARP inhibitors, which was further enhanced by the addition of a CDK9 inhibitor targeting Pol II elongation. Co-targeting PARP/CDK9 also elicited synergistic effects against other undifferentiated ovarian cancer cells with SMARCA4 loss. These findings link SMARCA4 loss to perturbed Pol II elongation and compromised DNA repair by BRCA1, providing a therapeutic opportunity to target SCCOHT and other SWI/SNF-deficient ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350e8c98aa41fa4d992b18cc3ee2eb0de4ea4277" target='_blank'>
              SMARCA4 loss increases RNA Polymerase II pausing and elevates R-loops to inhibit BRCA1-mediated repair in ovarian cancer.
              </a>
            </td>
          <td>
            Xianbing Zhu, Z. Fu, Giulio Aceto, Jonathan R St-Germain, Kexin Liu, A. Arabzadeh, Yuxuan Qi, Yibo Xue, L. Witkowski, Elise Graulich, Jutta Steinberger, Selim Misirlioglu, Nicklas Bassani, Racim Sansal, Amber Yasmeen, Geneviève Morin, Jingjie Guo, A. Monast, V. Pilon, Alica Valachová, K. Pavlakis, Lili Fu, Walter H. Gotlieb, W. McCluggage, D. Huntsman, Alexander J R Bishop, Douglas A Levine, Morag Park, Yemin Wang, B. Raught, W. Foulkes, Sidong Huang
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 As we age, cells accumulate somatic mutations, undergo epigenetic alterations, and experience telomere shortening. In older individuals, normal tissues are often overtaken by expanding clones of cells that have acquired mutations promoting proliferation. Whether epigenetic changes can also provide selective advantage and induce clonal expansion in ageing is less clear.
 Here, we sequenced 719 whole genomes and methylomes of single-cell colonies derived from HSCs of three healthy individuals and three breast cancer patients. Using somatic mutations, we built lineage trees of the HSCs for each individual. To establish whether loss or gain of methylation was heritable, we developed a method that accurately infers, for each CpG site, its zygotic methylation state as well as the number and timing of ancestral methylation changes that explain the observed methylation state of each somatic cell.
 We analysed ∼24 million CpG sites per individual and find that methylation states are, in general, stably heritable over decades of life. We find that during embryogenesis, when the embryo comprises a few thousand cells, developmental cells lay down unique methylation haplotypes, spanning hundreds of base pairs in size, that are stably inherited by descendants. This phase of rewiring the methylome has striking properties: (1) thousands of regions genome-wide show heterogeneity of methylation profiles laid down during this phase; (2) these regions are often conserved across different individuals; (3) it precedes gastrulation, so that the same changes are seen in all germ layers; (4) the process happens quickly, completing within a few generations of cell division; (5) it coincides with establishment of inactive X chromosome methylation in females; and (6) the methylation affects many key regulatory regions of the genome.
 To assess whether embryonic methylation rewiring can contribute to cancer development, we studied women with breast cancer who had BRCA1 promoter methylation. The tumour samples did not have germline or somatic mutation of BRCA1 (or other DNA repair genes), but had BRCA1 methylation along with the typical mutational signatures of homologous recombination deficiency. Intriguingly, we found that the same methylation change was present heterozygously in a subset of HSCs; that these HSCs all derived from one embryonic cell that pre-dated gastrulation; and that the somatic mutations defining this lineage of HSCs were also present clonally in the breast cancer. Thus, aberrant methylation of the BRCA1 promoter in embryogenesis was transmitted across germ layers, in blood and breast epithelium, and (with loss-of-heterozygosity) drove homologous recombination deficiency in a breast clone that transformed to cancer 4-6 decades later.
 By comparing normal healthy individuals of different ages with cancer patients, we have been able to unravel the process of normal ageing from cancer development at an unprecedented resolution.


 Lori D. Kregar, Nicholas Williams, Joe Lee, Michael Spencer-Chapman, Oleksii Nikolaienko, Emily Mitchell, Liv B. Gansmo, Per E. Lønning, Elisa Laurenti, Lucy Yates, Stian Knappskog, Jyoti Nangalia, Peter Campbell. Rewiring of methylation during embryogenesis can seed cancer decades later [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB308.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993964715fda888d384887c1b0e033eb189929c" target='_blank'>
              Abstract LB308: Rewiring of methylation during embryogenesis can seed cancer decades later
              </a>
            </td>
          <td>
            Lori D. Kregar, N. Williams, Joe Lee, Michael Spencer-Chapman, O. Nikolaienko, E. Mitchell, Liv B Gansmo, P. Lønning, Elisa Laurenti, Lucy Yates, S. Knappskog, J. Nangalia, Peter J Campbell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Mutations in somatic cells are inflicted by both extrinsic and intrinsic sources and contribute over time to cancer. Tobacco smoke contains chemical carcinogens that have been causatively implicated with cancers of the lung and head & neck. APOBEC family DNA cytosine deaminases have emerged as endogenous sources of mutation in cancer, with hallmark mutational signatures (SBS2/SBS13) that often co-occur in tumors of tobacco smokers with an equally diagnostic mutational signature (SBS4). Here we challenge the dogma that mutational processes are thought to occur independently and with additive impact by showing that 4-nitroquinoline 1-oxide (NQO), a model carcinogen for tobacco exposure, sensitizes cells to APOBEC3B (A3B) mutagenesis and leads to synergistic increases in both SBS2 mutation loads and oral carcinomas in vivo. NQO-exposed/A3B-expressing mice exhibit twice as many head & neck lesions as carcinogen-exposed wildtype animals. This increase in carcinogenesis is accompanied by a synergistic increase in mutations from APOBEC signature SBS2, but not from NQO signature SBS4. Interestingly, a large proportion of A3B-catalyzed SBS2 mutations occurs as strand-coordinated pairs within 32 nucleotides of each other in transcribed regions, suggesting a mechanism in which removal of NQO-DNA adducts by nucleotide excision repair exposes short single-stranded DNA tracts to enzymatic deamination. These highly enriched pairs of APOBEC signature mutations are termed didyma (Greek for twins) and are mechanistically distinct from other types of clustered mutation (omikli and kataegis). Computational analyses of lung and head & neck tumor genomes show that both APOBEC mutagenesis and didyma are elevated in cancers from smokers compared to non-smokers. APOBEC signature mutations and didyma are also elevated in normal lung tissues in smokers prior to cancer initiation. Collectively, these results indicate that DNA adducting mutagens in tobacco smoke can amplify DNA damage and mutagenesis by endogenous APOBEC enzymes and, more broadly, suggest that mutational mechanisms can interact synergistically in both cancer initiation and promotion.


 Reuben S. Harris. Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB476.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63aefe6d3e777332427bd46910b51b4d2635b3ae" target='_blank'>
              Abstract LB476: Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            R. Harris
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="


 Genomic DNA is continually exposed to endogenous and exogenous stressors, leading to DNA damage and genomic instability. The base excision repair (BER) mechanism is crucial for maintaining genetic integrity by repairing damaged nucleobases. BER begins when DNA glycosylases cleave the N-glycosidic bond to remove oxidized pyrimidines. DNA glycosylases can be monofunctional or bifunctional. In this study, we explore the bifunctional NTHL1 variant in inflammation-associated colorectal cancer (CRC). NTHL1 tumor syndrome is linked to increased lifetime risks of CRC, breast cancer, and colorectal polyposis. The rare polymorphism rs3087468 (D239Y) results in a loss of glycosylase activity and acts as a dominant-negative mutation.



 We hypothesize that the NTHL1-D239Y variant increases CRC susceptibility under chronic inflammation in the AOM/DSS cancer initiation model.



 Male and female C57BL/6 mice (6-8 weeks old) with wild-type (NTHL1WT/WT), heterozygous (NTHL1WT/D239Y), or homozygous (NTHL1D239Y/D239Y) NTHL1 variants were used. In the first experiment, mice were injected with 7.5 mg/kg azoxymethane (AOM), followed by three cycles of 2.5% dextran sulfate sodium (DSS) (5 days DSS alternating with 14 days of normal water). The second experiment used DSS alone. We monitored body weight, collected fecal samples for 16S rRNA sequencing, and measured colon length and polyp number/size at euthanasia. Histology was assessed for inflammation and tumor development. Blood samples were analyzed for micronuclei formation, and immunohistochemistry (IHC) staining was used for tissue proliferation.



 In the AOM/DSS model, male NTHL1WT/D239Y and NTHL1D239Y/D239Y mice had significantly more polyps compared to NTHL1WT/WT controls. Female NTHL1D239Y/D239Y mice had more polyps than NTHL1WT/D239Y females. In the DSS-only experiment, male NTHL1WT/D239Y mice developed a significant number of polyps compared to wild-type males, while female NTHL1D239Y/D239Y mice developed more polyps than NTHL1WT/WT females. No significant differences in polyp size were observed in the DSS-only experiment. Tumors extending into the proximal colon were seen only in female NTHL1D239Y/D239Y mice. Male NTHL1WT/D239Y mice developed high-grade adenomas in the distal colon compared to wild-type males.Micronucleated erythrocytes were significantly higher in male NTHL1D239Y/D239Y mice compared to controls, but this was not observed in the DSS-only group. No changes in colitis score or colon length were observed. Genotype or AOM/DSS treatment did not affect microbial alpha diversity, but treatment-dependent changes in microbial composition were noted based on sex and genotype. IHC data indicated elevated KI67 expression in NTHL1D239Y/D239Y and NTHL1WT/D239Y mice following AOM/DSS treatment.



 The NTHL1-D239Y variant promotes malignancy only under chronic inflammation and AOM/DSS treatment. This variant confers a complex, sex- and zygosity-dependent susceptibility to inflammation-associated CRC, with females showing increased tumor development in the proximal colon and males displaying enhanced tumor size and progression.



 Tarek Michael Masannat, Fayez K. Ghishan, Pawel R. Kiela, Joann B. Sweasy. NTHL1-D239Y variant enhances tumorigenesis in colorectal cancer through cancer initiation and chronic inflammation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB304.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93f87b45a480b89d7a05bed3b25376e4e282650" target='_blank'>
              Abstract LB304: NTHL1-D239Y variant enhances tumorigenesis in colorectal cancer through cancer initiation and chronic inflammation
              </a>
            </td>
          <td>
            Tarek Masannat, F. Ghishan, P. Kiela, J. Sweasy
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 2047


 Background:
 Genome-wide characterization has illuminated the molecular complexity of human gliomas. Genetic alterations help predict the clinical behavior of gliomas, but variability persists. Accordingly, there is a need to expanding molecular signatures that refine prognostication. The Brain Protein I3 (BRI3) gene, localized on chromosome 7, is associated with high-grade glioma, yielding the highest hazard ratio among all high-risk genes in an in-silico glioma model. Given GBM's known chromosome 7 gain and BRI3's chromosomal location, we hypothesized that gene dosage gains and overexpression serve as prognostic biomarkers.
 Methods:
 We used the Rembrandt (n = 461 patients) and TCGA low-grade glioma (LGG) and GBM (n = 1,148 patients) databases for multi-omic analyses. RNA sequencing (Illumina HiSeq) and DNA methylation profiles (Illumina 450K) were analyzed in a combined LGG and GBM cohort, with high/low expression and hyper/hypomethylation defined by median values. CNV analysis (GISTIC 2.0) classified 2 copies as gene-copy neutral and > 2 as gene dosage gain. Somatic mutation data (SNPs/INDELs) were derived from whole-exome sequencing to decipher IDH-wt and IDH-mutant gliomas. Univariate and adjusted Cox models, Kaplan-Meier estimates, and receiver operating characteristic (ROC) curve analysis were performed.
 Results:
 Of 461 Rembrandt patients, 47.29% were GBM, 31.89% astrocytoma, 14.53% oligodendroglioma, and 6.29% normal brain. Among 1,148 TCGA LGG/GBM patients, 551 had complete molecular and clinical data, with 28% IDH-wt status, 49% MGMT hypermethylation, and 36.66% BRI3 gene dosage gains. BRI3 expression was significantly higher in GBM, grade IV tumors, IDH-wt, and mesenchymal subtypes. Among the IDH-wt cases, 75% showed BRI3 gene dosage gains with significantly elevated mRNA expression. EGFR, co-amplified in 80% of IDH-wt cases, did not affect survival (25.27 vs. 17.90 months,
 p
 = 0.346). Conversely, BRI3 gene dosage gain correlated with worse survival (79.3 vs. 17.93 mo,
 p
 = 0.001), as did BRI3 high vs. low expression (17.90 vs. 25.27 months,
 p
 = 0.015) and MGMT hypermethylation (25.27 vs. 18.63 mo,
 p
 = 0.021). Univariate analysis linked patient age (HR: 2.858 [1.781–4.586],
 p
 < 0.001), MGMT hypermethylation (HR: 2.632 [1.307–5.301],
 p
 = 0.007), BRI3 high expression (HR: 1.938 [1.131–3.320],
 p
 = 0.016) and BRI3 gene dosage gain (HR: 2.504 [1.425–4.398],
 p
 = 0.001) to worse OS. Adjusted multivariate Cox regression confirmed BRI3 gene dosage gain, age, and MGMT methylation as independent OS predictors in IDH-wt cases. ROC analysis revealed stronger prognostic performance for BRI3 gene dosage gain than MGMT hypermethylation (AUC: 0.737 vs. 0.616,
 p
 < 0.001) in IDH-wt cases.
 Conclusions:
 Elevated BRI3 gene dosage and expression portend poor prognosis and could be incorporated into models predicting disease fate in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87bc0207e2dc419835fa0d46e3df399c03320a14" target='_blank'>
              Use of brain protein I3 (BRI3) to predict disease fate in glioblastoma.
              </a>
            </td>
          <td>
            Ifeanyichukwu C Ogobuiro, Scott M. Welford, Markus Bredel
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: The study of circulating tumor DNA (ctDNA) and cell-free chromatin (cf-Chr) has become instrumental in advancing our understanding of cancer biology. This research delves into the antitumor mechanism of abemaciclib in ER-positive, HER2-negative metastatic breast cancer patients by examining cf-Chr fragmentation patterns. Methods: In this investigation, 72 ER-positive, HER2-negative metastatic breast cancer patients, as defined by ASCO/CAP guidelines and treated with abemaciclib at Chiba University Hospital in Japan from 2020 to 2022, were enrolled. Pre- and post-treatment serum samples were analyzed, with nucleic acids extracted for whole-genome sequencing (WGS) and differential enrichment analysis. Given the observed correspondence of cf-Chr fragment patterns with open chromatin regions (OCRs), indicative of gene activation zones, differential enrichment analysis was specifically focused on cell type-specific regulatory regions—namely enhancers and promoters—in luminal breast cancer cells and T cells, to elucidate the transcriptionally active regions within the genome. Results: Our investigations have revealed a significant decrease in cf-Chr concentrations, showing a 50% reduction following treatment with abemaciclib. The analysis of cf-Chr in breast cancer patients, both pre- and post-abemaciclib treatment, unveiled differential chromatin enrichment within regions tied to antitumor pathways that are modulated by the drug. Notably, the cf-Chr analysis preceding treatment disclosed disturbances in the Notch signaling pathway, thereby revealing an aspect of abemaciclib's antitumor mechanism that has not been previously established in human clinical specimens. Conclusions: Our study suggests for the first time alterations in cf-Chr in human patient specimens that may reflect the antitumor activity of abemaciclib that had previously been confirmed only in vitro. Our study presents a pivotal step in understanding the impact of abemaciclib on the tumor chromatin landscape, providing a direct observation of the drug's mechanism of action in patients. These insights pave the way for future research into the mechanisms of existing therapies and their optimization for treating metastatic breast cancer.
 Citation Format: Mamoru Takada, Sakuntha Gunarathna, Hideyuki Yamada, Muhan Yu, Takeshi Nagashima, Hiroshi Fujimoto, Junta Sakakibara, Hiroto Yamamoto, Regina Nguyen, Tsutomu Kawaguchi, Otsuka Masayuki, Motoki Takaku. Unveiling the Antitumor Mechanism of abemaciclib in Human Breast Cancer Through Circulating Tumor Chromatin Analysis [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006d74044a867b3a3ad5d2dcfe26ae4ccbc619c6" target='_blank'>
              Abstract P5-02-23: Unveiling the Antitumor Mechanism of abemaciclib in Human Breast Cancer Through Circulating Tumor Chromatin Analysis
              </a>
            </td>
          <td>
            Mamoru Takada, Sakuntha Gunarathna, Hideyuki Yamada, Muhan Yu, Takeshi Nagashima, H. Fujimoto, Junta Sakakibara, Hiroto Yamamoto, Regina Nguyen, Tsutomu Kawaguchi, Otsuka Masayuki, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Papillary thyroid cancer (PTC) is an ever-increasing cancer type worldwide, and greatly decreases the life quality and affects survival time of patients during its development and progression, but the underlying mechanisms and key factors for PTC progression are not clear. Recent studies demonstrated the potassium channel protein SK4 participates in the progression of many cancers, while it lacks the molecular mechanism study for SK4 function. In this study, we performed functional and molecular explorations for SK4 by overexpressing its level in thyroid cancer BHT101 cells. Cellular proliferation and invasion experiments were performed to assess the influences of SK4 on cell behaviors. Further, whole transcriptome sequencing (RNA-seq) analysis helped us systematically investigated the targets of SK4, including differentially expressed genes (DEGs) and regulated alternative splicing events (RASEs), and validated several related DEGs and RASEs by RT-qPCR experiment. In thyroid cancer patients, SK4 expression was completely lost in normal tissues and significantly increased in every stage of tumor tissues compared with normal tissues, which probably results from the low DNA methylation level at its promoter region. Consistent with previous study, SK4 overexpression (SK4-OE) promoted proliferation and invasion ability of BHT10 cells compared with negative control (NC). By analyzing the RNA-seq data, we detected dozens of DEGs and found that up DEGs were enriched in negative regulation of apoptotic progress, including VTCN1, MSX1, FATE1, TEK, and PRAMEF2. More importantly, we found SK4-OE globally changed the alternative splicing (AS) pattern and identified 1,639 RASEs. The genes of RASEs were enriched in DNA damage/repair, viral process, translation, and mRNA splicing pathways, which were tightly associated with the pathogenesis and progression of cancers. The splicing regulatory genes from regulated alternative splicing genes (RASGs) could partly explain the reason of global AS dysregulation by SK4-OE in BHT101 cells. Finally, we found the expression of VTCN1, EDN1, SLC29A4, RP11-473M20.16, and CH507-513H4.4 were validated by RT-qPCR, as well as the AS pattern of TMEM116. In summary, we highlight that SK4-regulated AS pattern probably is a novel regulatory mechanism for SK4 in PTC. The identified DEGs and RASEs, as well as SK4 itself, could be used as potential therapeutic targets for PTC treatment in future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bf9b607a916a81e5a08e0bc05a7b09aa5fe042f" target='_blank'>
              SK4 potentially modulates the alternative splicing profile associated with papillary thyroid cancer development in BHT101 cells
              </a>
            </td>
          <td>
            Alimujiang Wupuer, Xuekelaiti Kuerban, Gong Ye
          </td>
          <td>2025-05-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Newcastle disease virus (NDV) is a promising oncolytic agent with a non-segmented, negative-sense single-stranded RNA (ssRNA) genome of approximately 15 kb. While NDV selectively replicates in and lyses a wide range of human cancer cells, a subset of these cells develop persistent infections, potentially compromising the therapeutic efficacy of NDV-based treatments. To investigate the molecular basis of persistent infection, we performed transcriptome profiling of TCCSUP bladder cancer cells persistently infected with the NDV AF2240 strain. Deep sequencing using Illumina HiSeq 2000 was conducted in triplicate, and the resulting viral and host reads were separated and analyzed. Using Integrative Genomic Viewer (IGV) software, we identified several nucleotide variants linked to persistence. Specifically, nucleotide alterations included a deletion at 359A→C and a substitution at 1653C→T within the nucleoprotein (NP) gene, as well as an insertion at 3338C→T in the matrix protein (M) gene. Additionally, a GGG base insertion was detected at position 2290 in the phosphoprotein (P) gene. Crucially, we observed truncations in the hemagglutinin-neuraminidase (HN) gene (nt 8263–8390) and the large polymerase (L) gene (nt 6203–6342). These mutations and truncations suggest significant disruptions in viral replication, assembly, and host cell attachment, potentially facilitating viral persistence in bladder cancer cells. Understanding these genomic alterations provides valuable insights into the mechanisms driving viral persistence and could inform strategies to optimize the oncolytic efficacy of NDV in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46c1e75575001a56bcabf1b3e8302abfbe59f19d" target='_blank'>
              Genomic alterations in persistently infecting oncolytic Newcastle disease virus reveal mechanisms of viral persistence in bladder cancer cells
              </a>
            </td>
          <td>
            U. Ahmad, D. Chau, SC Chan, K. Yusoff, A. Veerakumarasivam, S. Abdullah
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7defb0dc0b2dcfb00df9b73a0a0c72cd7d43d4" target='_blank'>
              Transposable elements as instructors of the immune system.
              </a>
            </td>
          <td>
            Lisa Schmidleithner, Philipp Stüve, Markus Feuerer
          </td>
          <td>2025-04-29</td>
          <td>Nature reviews. Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="R-loop is a three-stranded nucleic acid structure composed by a DNA:RNA hybrid and a displaced single-stranded DNA. It plays pivotal roles in a variety of essential biological processes across both prokaryotic and eukaryotic organisms, including DNA replication and repair, transcriptional regulation, and epigenetic modifications. However, excessive accumulation of R-loops can lead to DNA damage, genomic instability, and chromatin rearrangements, which in turn can trigger the development of various tumors and non-tumorous diseases. To identify the genomic locations of R-loops, several detection methods have been developed, including affinity enrichment techniques (based on S9.6 antibodies or the hybrid-binding domain of RNase H1) and non-affinity enrichment techniques (such as spKAS-seq and RIAN-seq). This review focuses on recent advances in understanding R-loop formation mechanisms, biological functions, and detection technologies, aiming to provide a conceptual framework for understanding the biological significance of R-loops and their roles in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cac47e0796778e4ede95c2be9a5977402c16793" target='_blank'>
              Research advances in R-loop functions and detection technologies
              </a>
            </td>
          <td>
            Qiannan He, Hongjie Yao
          </td>
          <td>2025-05-01</td>
          <td>SCIENTIA SINICA Vitae</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Gastric cancer (GC) is a leading cause of cancer-related deaths globally. Emerging evidence implicates aberrant DNA polymerase beta (POLB) expression in genomic instability and tumorigenesis. Previously, we demonstrated that the human gastric cancer-associated variant of POLB (Leu22Pro or L22P) lacks dRP lyase function in vitro and induces replication associated genomic instability and cellular transformation. In addition , in vivo mouse model experiment has shown that L22P mutation of POLB , leads to inefficient BER and exacerbate genomic instability and carcinogenesis .



 We conducted an in-silico analysis of GC datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) to assess POLB expression levels and their association with the tumor immune microenvironment (TIME). Survival analyses were performed using Kaplan-Meier estimates, and the TIMER algorithm was employed to evaluate immune cell infiltration. In parallel, we utilized a dRP lyase-deficient POLB variant (Leu22Pro, L22P) in vivo model to investigate the impact of dRP lyase deficient POLB on genomic stability and inflammatory signaling. Also, Mitotic dysfunction and cytosolic DNA-mediated inflammatory responses were assessed, with and without poly(ADP-ribose) polymerase 1 (PARP1) inhibition.



 Our analysis revealed that POLB is overexpressed in approximately 32% of GC cases, significantly correlating with the intestinal subtype of GC. POLB overexpression was associated with reduced expression of innate immune signaling genes and lower infiltration of immune cells, indicating a suppressed immune microenvironment. Conversely, tumors with high POLB expression exhibited increased mutation frequency and microsatellite instability (MSI). Notably, patients with POLB-overexpressing tumors and high immune score demonstrated improved overall survival. ROC analysis suggested that POLB overexpression holds potential as a prognostic marker for GC. In in vivo model, the L22P POLB variant increased mitotic dysfunction-associated genomic instability, leading to a cytosolic DNA-mediated inflammatory response. PARP1 inhibition exacerbated chromosomal instability and enhanced the inflammatory response in these cells.



 Aberrant POLB expression in GC is associated with a dysregulated TIME and may serve as a prognostic indicator. Defective POLB function contributes to genomic instability and activates cytosolic DNA-mediated inflammatory signaling. These findings suggest that deregulated POLB in tumor likely provide a potential opportunity to patient stratification for better immune-based therapies in GC.



 Aashirwad Shahi. Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB386.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21340650146a334354ebce820c4c26cb9ab3ba1b" target='_blank'>
              Abstract LB386: Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer
              </a>
            </td>
          <td>
            Aashirwad Shahi
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ovarian metastases originating from colorectal cancer (CRCOM) develops rapidly and lethally. Previously, the genetic alterations and metastatic pathway in CRCOM were not well understood. The aim of this study is to explore the special molecular phenotype and dissemination patterns of CRCOM.The whole-exome sequencing (WES) was performed on 65 matched tissue samples from 11 CRCOM patients, including 11 primary colorectal cancer (CRC) with 11 matched normal tissues, and 43 multi-site metastases (including 15 CRCOMs and 4 patients had bilateral ovarian metastases (OMs). Genetic landscape, neoantigens, tumor clonal origin and spread of CRCOMs were analyzed. TCGA-COAD dataset combined with our data were used for survival analysis and validation of the findings.There was significant intertumoral heterogeneity among patients with CRCOM and intra-tumoral heterogeneity among multiorgan metastases. 19 genes were inferred as the potential driver genes of CRCOM. USP7 and RPA1 were HRD-related mutations and potential to serve as predictive biomarkers in OM. The putative neoantigen number of the primary CRC and OM varies widely among patients. The OM showed an immune desert state, extremely deficient in each subtype of immune cells. According to COSMIC signatures features, the CRCOM patients were divided into two groups, which are different in overall survival (OS) (median OS, 720 days vs 360 days, P = 0.074) and genetic alterations. Two metastatic patterns of CRCOM were summarized, which were primary CRC to OM, and metastases to metastases (including lymph node metastases (LNM) to OM, peritoneal metastases (PM) to OM, and other metastases to OM). Interestingly, the sources of bilateral OM might be different in the two patients.This study presents a better understanding the heterogeneity of the genetic characterizations and metastatic pattern in CRCOM. The subtypes of CRCOM with USP7 mutation, more copy number alterations, lower neoantigens, and immunoscore have a worse prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f79bcc3ee858322f861b93f18ac7e9a5b5f8b2c" target='_blank'>
              The heterogeneity of genomic alterations, metastatic patterns and immune microenvironment in metastatic ovarian cancer originating from colorectal cancer
              </a>
            </td>
          <td>
            Chao Chen, Jian Wang, Binjie Sun, Yuyan Zheng, Xiaoxu Ge, Zhiyuan Gong, Haochen Gu, Zhiwei Zhang, Akao Zhu, Yingkuan Shao, Yeting Hu, Lijia Ma, Yini Li, Kefeng Ding, Da Wang, Lifeng Sun
          </td>
          <td>2025-06-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 The single-stranded DNA cytosine deaminase enzyme, APOBEC3B (A3B), has been implicated as a mutational driver in multiple human cancers1,2. Breast cancer in particular shows high levels of A3B expression and positive associations with poor clinical outcomes3. Recent studies have also demonstrated that human A3B expression in mice is carcinogenic with significantly shortened life expectancies4. The only genetic factor in breast cancer thus far associated with A3B expression and APOBEC signature single base substitutions is p53 mutation5. Therefore, here we test the hypothesis in vivo that human A3B expression will show pathlogical synergy in the form of accelerated mammary tumor penetrance upon combination with p53 haploinsufficiency due to a heterozygous deletion mutation spanning exons 2-10. Custom and commercially available animals were subjected to standard breeding practices to generate experimental (A3B, p53Delta exons 2-10/+) and control groups (p53Deltanexons 2-10/+). Surprisingly, in contrast to A3B-accelerated tumor development on a wildtype genetic background, A3B had no effect on the rates of tumor development in p53 heterozygous animals (median 13.5 months). The observed genetic epistasis was not due to a lack of A3B expression or activity, as evidenced by strong IHC positivity and a clear acculumlation of APOBEC signature single base substitution mutation in tumors. We conclude that A3B-accelerated tumor development requires full p53 function, and that a haploinsufficiency in p53 enables tumor cells (or their precursors) to better tolerate DNA damage lesions induced by A3B.
 Selected References:
 1. Petljak, M. et al. Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer. Nat Genet 54, 1599-1608 (2022).
 2. Butler, K. & Banday, A. R. APOBEC3-mediated mutagenesis in cancer: causes, clinical significance and therapeutic potential. J Hematol Oncol 16, 31 (2023).
 3. Roelofs, P. A. et al. Clinical implications of APOBEC3-mediated mutagenesis in breast cancer. Clin Cancer Res 29, 1658-1669 (2023).
 4. Durfee, C. et al. Human APOBEC3B promotes tumor development in vivo including signature mutations and metastases. Cell Rep Med 4, 101211 (2023).
 5. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).\
 Citation Format: Joshua Proehl, Cameron Durfee, Yuan Zhao, Nuri Alpay Temiz, Reuben Harris. p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8de4891dd8d8ec2c399682371385c3645385994" target='_blank'>
              Abstract P3-06-26: p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo
              </a>
            </td>
          <td>
            Joshua Proehl, Cameron Durfee, Yuan Zhao, N. A. Temiz, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="All biological processes, from embryonic development to cancer, are tightly controlled by the interactions between genetics and epigenetics. An array of epigenetic modifications, such as DNA methylation, histone/chromatin modifications, and noncoding RNA-mediated targeting, are essential to regulate the heritable changes that occur during multiple cellular processes. A failure in proper regulation results in inappropriate gene expression that ultimately leads to pathological states. Groundbreaking advances in genomics and transcriptomics have revealed the potential involvement of epigenetics in various physiological and pathological states. The promising clinical and preclinical results shown by epigenetics drugs further underscore the central role of epigenetics in multiple human diseases, including cancer. AT rich interaction domain (ARID)-containing proteins are a family of evolutionarily conserved DNA binding proteins that regulate epigenetic modifications. Genome sequencing has revealed the existence of 15 ARID family proteins that are divided into 7 subfamilies based on their sequence and domain homology. Although the ARID family of proteins are implicated in cell growth, development, differentiation, and cancer, the diverse biological functions of many family members remain to be elucidated. Here, we focus on ARID4B to summarize its prominent role in embryonic stem cell differentiation and human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72fba835b7a96d97fab7eb6f68f890cb10aeda31" target='_blank'>
              ARID4B: An Orchestrator from Stem Cell Fate to Carcinogenesis
              </a>
            </td>
          <td>
            R. R. K. Kandy, M. Arumugam, Mukesh Pratap Yadav, Bibhuti Bhusan Mishra, Jyotika Sharma
          </td>
          <td>2025-06-10</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA topoisomerase 1 (TOP1) is essential for transcription, replication, and repair. Its function relies on two catalytic steps, DNA breakage and rejoining. Inhibitors of the second step prevent DNA rejoining and lead to persistent DNA breaks, acting as topoisomerase poisons, used as anticancer drugs. However, reliable inhibitors of the first step are not available. Here, we provide genetic and molecular evidence supporting that Patulin and, to a lesser extent, Xestoquinol inhibit the first catalytic step of TOP1 in vitro, in yeast and in human cells. Particularly, Patulin prevents the accumulation of TOP1 cleavage complexes caused by the TOP1 poison camptothecin (CPT) in human cells. Moreover, Patulin pretreatment of human or yeast cells reduces DNA damage and the accumulation of DNA breaks upon CPT exposure. Consistent with the protective role of TOP1 against harmful R-loops, Patulin treatment increases R-loops and R-loop-associated cytotoxicity, mimicking the effect of TOP1 silencing. Altogether our findings indicate that Patulin and Xestoquinol are nonpoisoning inhibitors of TOP1, which should potentiate new research approaches in molecular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c91b4af793ee46a1c397d0e749fad65c6dce6ba" target='_blank'>
              Patulin and Xestoquinol are inhibitors of DNA topoisomerase 1.
              </a>
            </td>
          <td>
            E. Tumini, R. Wellinger, Emilia Herrera-Moyano, Patricia Navarro-Cansino, María L. García-Rubio, Daniel Salas-Lloret, Alejandro Losada, M. Muñoz-Alonso, H. Gaillard, Rosa Luna, A. Aguilera
          </td>
          <td>2025-04-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 In the 20 years since the completion of the Human Genome Project, cancer biology remains rooted in the belief that the human genome encodes ∼20,000 protein-coding genes. This assumption has been fundamentally challenged by the discovery of microproteins, which are encoded across the genome by thousands of non-canonical open reading frames (ORF). Whether these microproteins reflect a previously untapped source of cancer genes may therefore have a profound impact on our understanding of cancer biology. To address this question, we have interrogated the prevalence and impact of non-canonical ORF translation across multiple cancer types. Using ribosome profiling, ORF cDNA libraries, expression profiling and CRISPR/Cas9 screens, we find that thousands of non-canonical ORFs are translated in cancer cells; nearly half (46%) of tested non-canonical ORFs may make a detectable microprotein; nearly 30% induced gene expression changes in cancer cell lines; and 10% induced a cancer cell viability defect when knocked-out. High-resolution CRISPR saturation mutagenesis identified a subset of non-canonical ORFs with exquisite genetic selectivity in about one-third of cases, providing high-confidence genetic evidence for a causal role of the non-canonical ORF in cancer cell viability. To position specific candidates in the context of cancer biology, we have characterized two cancer-associated microproteins encoded by non-canonical ORFs. In breast cancer, we described G029442 - renamed glycine-rich extracellular protein-1 (GREP1) - as a secreted protein that is a preferential genetic dependency in this disease through its regulation of the oncogenic cytokine, GDF15. Moreover, GREP1 expression is prognostic for poor patient outcomes. In childhood medulloblastoma, we found that MYC-driven medulloblastoma cells preferentially upregulated an upstream open reading frame in the ASNSD1 5’ untranslated region, termed ASNSD1-uORF. ASNSD1 uORF was a top genetic vulnerability in multiple models of medulloblastoma and it functions as a component of the prefoldin-like complex, which post-translationally regulates nascent proteins. Together, ASNSD1-uORF and the prefoldin-like complex impacted cancer cell cycle through the proteomic regulation of proteins involved in mitosis and chromosome spindle complexes. In summary, our work supports a generalizable principle that ncORFs commonly encode biologically-active microproteins in diverse malignancies. Ongoing investigation of ncORFs therefore represents a new frontier in cancer research with the potential to define the next generation of therapeutic target genes.


 John R. Prensner. Mapping the functional microproteome in cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY04-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3048f7fc1ab2dec5d8c000f11dda761b0959e" target='_blank'>
              Abstract SY04-01: Mapping the functional microproteome in cancer
              </a>
            </td>
          <td>
            John R. Prensner
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is among the most prevalent malignancies worldwide. Cuproptosis, a copper-induced form of cell death, has been identified as a key process in LUAD progression; however, the molecular mechanisms underlying cuproptosis in LUAD and potential therapeutic targets remain unclear. The present study utilized The Cancer Genome Atlas database to retrieve mRNA expression profiles and clinical information of LUAD, identifying 10 candidate genes from differentially expressed genes associated with cuproptosis. Protein-protein interaction analysis indicated that CDK inhibitor 2A (CDKN2A), an upregulated gene in LUAD, may function as a hub gene. Furthermore, multiple online databases were used to analyze Spi-1 proto-oncogene (SPI1), a transcription factor upstream of CDKN2A, which was downregulated in LUAD cuproptosis. The LinkedOmics database identified the p53-mediated cuproptosis-related pathway regulated by CDKN2A. Gene expression patterns were examined through Gene Expression Profiling Interactive Analysis, the Human Protein Atlas and reverse transcription-quantitative polymerase chain reaction. Prognostic significance was assessed using the UALCAN and Kaplan-Meier plotter databases. In vitro experiments demonstrated that CDKN2A knockdown and SPI1 overexpression inhibited the proliferation and migration of the H1975 cell line. After copper-induced cuproptosis in H1975 cells, SPI1 expression was upregulated, whereas CDKN2A expression was downregulated. When H1975 cells were pretreated with tetrathiomolybdate, the upregulation of SPI1 was inhibited and the downregulation of CDKN2A was also suppressed. Cell Counting Kit-8 assays indicated that SPI1 overexpression and CDKN2A knockdown facilitated elesclomol-CuCl2-induced cuproptosis. Western blot analysis revealed an inverse association between SPI1 overexpression and CDKN2A/p53 levels. In conclusion, the present study demonstrated the role of the SPI1/CDKN2A/p53 axis in LUAD cuproptosis, providing insights into potential therapeutic targets and contributing to clinical research on treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a265e6ec17f3bc3d7df377a278752607ec192b44" target='_blank'>
              Role of the SPI1/CDKN2A/p53 signaling pathway in cuproptosis of lung adenocarcinoma cells
              </a>
            </td>
          <td>
            Jiayue An, Wei-Wei Song, Qin Wang, Boyu Tan, Xuan Fei, Ruoxi Wang, Siyan Li, Xiyu Lu, Youjie Li, Ning Xie
          </td>
          <td>2025-05-19</td>
          <td>Oncology Letters</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f02ba901248eaa8a5acde4255c7a7dbd0983784" target='_blank'>
              Efficacy of DNA Methyltransferase Inhibitor Immune Priming Therapy in Combination with PD-1 Inhibitors to Treat High-Risk Pediatric Brain Tumors
              </a>
            </td>
          <td>
            Deepak Mishra, Shelli M. Morris, Dean Popovski, Emily J. Girard, Andrew Bondoc, S. Kumar, Augusto Faria Andrade, Xiaoting Zhu, Fupan Yao, M. Brusniak, Umaru Banlanjo, Erin E. Crotty, Ken Brasel, Fiona Pakiam, Caterina Russo, M. Zeinieh, Matt C. Biery, Margo Coxon, Heather Conti, Midori Clarke, Mei Lu, James T Rutka, Dhana Llivichuzhca-Loja, Liza Konnikova, M. Fouladi, Nada Jabado, Annie Huang, Jim Olson, R. Drissi
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, YeJi An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genome instability is a major force driving tumorigenesis. The ssDNA-binding protein complex CTC1-STN1-TEN1 (CST) plays a pivotal role in maintaining genome stability by countering replication stress, modulating DNA damage repair, and maintaining telomere integrity. Despite its well-documented role in genome maintenance, the involvement of CST in skin cancer development has yet to be investigated. We recently found that CST localizes at stalled DNA replication sites after UV exposure and may suppress the unwanted repriming activity, suggesting a potential role of CST in suppressing genome instability caused by UV damage. In this study, we first analyzed CST expression and alterations in cutaneous melanoma database and found that the CST genes are frequently altered in cutaneous melanoma and their expression is significantly downregulated in melanoma samples compared to normal tissues. We then generated a conditional knockout (cKO) mouse model with STN1 deficiency specifically in melanocytes to investigate its role in skin cancer formation. Upon chronic exposure to UV irradiation, STN1-deficient mice exhibit no obvious difference in melanoma incidence compared to control littermates, suggesting that STN1 downregulation in mature melanocytes has no significant effect on UV-induced skin cancer development in lab mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b040e5d51366f43fd311948a74f723a17b17afb" target='_blank'>
              In vivo investigation of STN1 downregulation in melanoma formation in adult mice following UV irradiation
              </a>
            </td>
          <td>
            Sara Knowles, Fang Wang, Maarten C Bosland, S. Gaddameedhi, Weihang Chai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Neuroblastoma (NB), the common extracranial solid tumor in children, is associated with a poor prognosis, particularly in high‐risk patients. MYCN amplification stands as the most prominent molecular hallmark within this high‐risk subgroup. However, MYCN protein is considered “undruggable” due to its lack of a conventional enzymatic binding pocket and its predominant nuclear localization, which precludes targeting by standard small‐molecule inhibitors or antibody‐based therapeutics. Consequently, current therapeutic strategies have achieved limited efficacy against MYCN‐driven NB. Notably, MYCN not only orchestrates diverse metabolic reprogramming pathways in tumors but also exerts a pivotal influence on cellular differentiation. To overcome this therapeutic barrier, we seek to elucidate the contribution of purine metabolism to stemness maintenance in MYCN‐amplified NBs and to discover novel small‐molecule inhibitors capable of inducing differentiation in high‐risk NBs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02766936ba001495072dbbf9dc777d193e23d838" target='_blank'>
              Inhibition of Purine Metabolism Promotes the Differentiation of Neuroblastoma Driven by MYCN
              </a>
            </td>
          <td>
            Yufeng Jiang, Hui Xiao, Yi Yang, Guoyu Chen, Yingwen Zhang, Xiaoyu Wu, Qi Zhang, Qingyi Huang, Hongxiang Gao, Yali Han, Anan Zhang, Dapeng Jiang, Liang Zheng, Yan-Xun Li, Yijin Gao
          </td>
          <td>2025-05-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="
 3577


 Background:
 HRD is linked to sensitivity to platinum-based chemotherapy and poly (ADP-ribose) polymerase inhibitors across various tumor types. However, the presence of HRD, driver events (such as pathogenic mutations in HR genes or other genomic alterations), and its clinical relevance in mCRC remain underexplored.
 Methods:
 We performed the VHIO-300 test, an ISO15189 accredited custom NGS panel profiling over 450 genes (including HR-related genes:
 ATM, BRCA1, BRCA2, BRIP1, CHEK2 or PALB2
 ) on 356 Stage IV mCRC patients (corresponding to 247 primary colorectal and 109 to metastatic samples) enrolled in the Vall d’Hebron Institute of Oncology’s Molecular Prescreening Program from June 2021 to December 2024. All samples had a tumor cellularity > 40% as per pathologist evaluation
 .
 An HR score (sHR) based on genome-wide copy number alterations (CNA) and loss of heterozygosity (LOH) patterns is generated. After cross-validation with Myriad MyChoice, sHR ≥ 56 was established based on a cohort of ovarian tumors and used to identify HRD in mCRC tumor samples.
 Results:
 HRD prevalence was 3.4% in our mCRC cohort (12/356), but, much higher in metastatic lesions, (6.4%, 7/109) than in primary samples (2.03%, 5/247) (p = 0.05). In fact, the median sHR between primary (21) vs. metastases (30) in CRC was significantly different (p < 0.01). Regarding HR gene status, HR-mutated samples were not significantly within the HRD group (p = 0.27) and only 6.5% (2/31) were HRD. Noteworthy,
 BRCA2
 exhibits a frameshift deletion in a homopolymer stretch, that is a frequent hotspot in microsatellite instable (MSI) tumors, but this event was not found to be associated with HRD. In fact, all HRD tumors (n = 12) were microsatellite stable (MSS). Other frequent alterations in mCRC were studied and
 BRAF
 mutations were found to be present in 42% of HRD tumors (p < 0.01). Inversely, HRD was rare in
 KRAS
 -mutated samples (0.7%; 1/140) and, in fact, highly correlated with non-HRD status (p = 0.02), especially the G12 mutation (p < 0.01). Interestingly, CNA profiles also revealed a strong association between the
 BCL2L1
 loci gain and HRD (p < 0.01). Clinically, HRD was not significantly associated with prognostic value nor clinical benefit to oxaliplatin-based combinations. However, the limited sample size and heterogeneous treatment lines restricted robust statistical analysis.
 Conclusions:
 This study identified a small, yet significant subset of mCRC that displays HRD. sHR and HRD rates were higher in metastatic lesions vs primary tumors, indicating HR scarring could be accumulating over time in some mCRC patients. No clear association between pathogenic HR gene mutations and HRD were found, suggesting the involvement of alternative molecular mechanisms in this process. Frequent co-occurring events, such as
 BRAF
 mutations or
 BCL2L1
 gains could be drivers in CRC HRD, and shape, eventually, new therapeutic options for these patients in the metastatic setting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14cea3ee627505991df2a2a9cb9438dafa79fb0" target='_blank'>
              Drivers of homologous recombination deficiency (HRD) in metastatic colorectal cancer (mCRC).
              </a>
            </td>
          <td>
            Paula Romero Lozano, Javier Ros Montañá, Eduardo García-Galea, Maria Vila-Casadesús, R. Comas, Ágatha Martín, Jenifer González-Zorelle, Paolo Nuciforo, H. G. Pálmer, J. Tabernero, E. Élez, Ana Vivancos-Prellezo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfdae0d9888d7167ae3464de1089da14e882abe" target='_blank'>
              Multi-cellular phenotypic dynamics during the progression of breast tumors
              </a>
            </td>
          <td>
            Louise Gsell, Spencer S. Watson, Iva Sutevski, Matteo Massara, Klara Soukup, Alper Eroglu, Jeff E Mold, Antony Cougnoux, Johanna A. Joyce, Jean Hausser
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Disorders of sex development (DSDs) are characterized by discrepancies among karyotype, the gonadal phenotype and gonadal anatomy following birth. Among these, 46, XY DSD is the most complex sub-type and a major cause of birth defects associated with sexual development. However, due to the considerable heterogeneity in pathogenic genes, numerous cases remain genetically undiagnosed. In the present study, a novel gain-of-function variant was identified in the mitogen-activated protein 3 kinase 1 (MAP3K1) gene, contributing to 46, XY DSD through the induction of abnormal cell apoptosis. Genetic analysis of a pediatric male patient and his family revealed a heterozygous MAP3K1 c.4445 G>A variant, resulting in an arginine-to-glutamine substitution. The variant site is highly conserved across species and the amino acid transition induced structural changes in the MAP3K1 protein. In vitro experiments demonstrated that the MAP3K1 c.4445 G>A variant markedly increased apoptotic signaling, leading to abnormal cell apoptosis, disruption of the cell cycle and reduced cell viability. Moreover, the variant exhibited increased levels of ERK1/2 and p38 phosphorylation, indicative of a gain-of-function effect. Subsequent analysis revealed increased expression of the testis-determining gene, SOX9 and reduced expression of the ovary-determining gene, FOXL2. Notably, alterations in gene expression were associated with the MAP3K1 c.4445 G>A variant, providing a mechanistic basis for the pathogenesis of 46, XY DSD. Collectively, these findings offer novel insights into the anti-apoptotic role of MAP3K1, advancing genetic diagnosis and pre-natal screening for individuals with DSDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2799a19a1fd170b1fea72ad9ddd73906ece00960" target='_blank'>
              A novel variant in the MAP3K1 genomic locus reveals abnormal cell apoptosis as a potential pathogenic mechanism in 46, XY disorders of sex development
              </a>
            </td>
          <td>
            Yufu Lu, Sijia Wei, Shuang Wang, Jingzhi Zhang, Yongjie Xu, Changyudong Huang, Wei Pan, Zhengrong Wang
          </td>
          <td>2025-06-05</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The supernumerary B chromosome of maize has a drive mechanism to maintain itself in a population despite being dispensible. This involves nondisjunction of the B centromere at the second pollen mitosis that produces the two sperm followed by preferential fertilization of the egg by the B containing sperm during double fertilization. During an introgression of the supernumerary B chromosome into the inbred line B73, an unusually high frequency of trisomies for A chromosomes was observed. Due to parallels to the High Loss phenomenon in which three or more B chromosomes in a specific genetic background cause chromosomal breakage at heterochromatic knob sites during the second pollen mitosis as well as ploidy changes, this phenomenon was revisited. Examination of pollen of the High Loss line revealed a high frequency of single sperm in the presence of the B chromosomes, which was previously not realized. Crosses to tetraploid females confirmed that the single sperm were diploid and functional but also revealed the presence of diploids with their A chromosomes derived solely from the tetraploid parent indicating a "diploid induction". Collectively, the results reveal two backgrounds in which the B drive mechanism is not confined to this chromosome causing detrimental effects by adherence of heterochromatic knobs and apparently A centromeres at the mitosis preceding sperm development. In most genetic backgrounds this process is restricted to the B chromosome but in B73 and the High Loss line, there is spillover to the normal chromosomes in distinct ways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822f11adca9e71b6e0ff40e6fc00f85ee3a4109f" target='_blank'>
              Aneuploidy and ploidy variation conditioned by the B chromosome of maize.
              </a>
            </td>
          <td>
            Patrice S. Albert, Hua Yang, Zhi Gao, Cassidy DeVore, J. Birchler
          </td>
          <td>2025-04-25</td>
          <td>Heredity</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The epitranscriptomic modification N6-methyladenosine (m6A) has emerged as a central regulator of RNA metabolism, influencing diverse physiological and pathological processes. As the most abundant RNA modification in eukaryotic cells, m6A dynamically modulates gene expression by regulating RNA processing, stability, and translation. Dysregulation of m6A has been implicated in key oncogenic processes, including tumor initiation, progression, metastasis, therapy resistance, and interactions within the tumor microenvironment. Despite these advances, the role of m6A in prostate cancer remains poorly understood, highlighting the need for further investigation. In a recent study, Xu et al. utilized refined m6A meRIP-seq to profile 162 primary prostate cancer patient samples, unveiling novel insights into the epitranscriptomic landscape of this malignancy. These findings not only enhance our understanding of m6A regulation in cancer biology but also underscore its potential to inform diagnostic, prognostic, and therapeutic strategies for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5516fc4406b04626bff5988fb10a8ec93da2c74b" target='_blank'>
              Unraveling the Role of N6-Methyladenosine in Prostate Cancer: Implications for Prognosis and Tumor Aggression
              </a>
            </td>
          <td>
            Xin Xu, Xinqiao Zheng, Housheng He
          </td>
          <td>2025-05-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Synthetic lethality, a phenomenon in which alterations in two or more genes together induce cell death while changes in either gene alone do not, provides a promising approach for cancer treatment. Indeed, various inhibitors targeting synthetic lethal interactions are approved by the Food and Drug Administration or in clinical trials. Seeking to nominate additional synthetic lethal targets, we analyzed large-scale CRISPR knockout screening data and identified Pelota mRNA surveillance and ribosome rescue factor (PELO) as a promising therapeutic target for two independent and frequently observed molecular subtypes of cancer: biallelic deletion of chromosomal region 9p21.3 (9p21.3-/-) and microsatellite instability-high (MSI-H).
 Homozygous deletion of chromosomal region 9p21.3 is one of the most frequently observed somatic copy number alterations in human cancers, occurring in approximately 15% of all cancers. Many 9p21.3-/- cancers are associated with poor clinical outcomes including subsets of glioblastoma, mesothelioma, urothelial, pancreatic, esophageal, and non-small cell lung cancers. The primary driver of 9p21.3-/- is thought to be loss of the tumor suppressor cyclin-dependent kinase inhibitor 2A. Additionally, studies have linked the loss of the interferon gene cluster on 9p21.3 to immune evasion and primary resistance to immune checkpoint inhibitors. Extensive deletions of chromosomal regions can provide opportunities for cancer treatment. For instance, PRMT5 and MAT2A were found to be promising targets in 9p21.3-/- cancers due to synthetic lethal interactions with deletions of the 9p21.3 gene methylthioadenosine phosphorylase. While clinical trials are underway examining PRMT5 and MAT2A inhibitors, the frequency and poor outcomes of many 9p21.3−/− cancers underscores the urgent need for additional therapies to treat this diverse group of cancers. Using large-scale functional genomic datasets, we identified PELO as the top preferential dependency in 9p21.3-/- cell lines.
 We also observed that some cell lines with intact 9p21.3 (9p21.3+) were dependent on PELO for survival. Examining other genomic features, we found that these cells were characterized as MSI-H, a hypermutable state observed in subsets of colon, endometrial, gastric, and ovarian cancers. Indeed, when we compared 9p21.3+/MSI-H and 9p21.3+/microsatellite stable (MSS) cell lines, we found that PELO scored as a strong preferential dependency in MSI-H cell lines, second only to the previously described synthetic lethal target Werner helicase (WRN).
 To validate these findings, we interrogated the viability effects of PELO knockdown across a panel of cell lines representing 9p21.3+/MSS, 9p21.3-/-/MSS, or 9p21.3+/MSI-H. CRISPR interference (CRISPRi)-mediated PELO knockdown with three distinct guide RNAs impaired the viability of 9p21.3-/-/MSS cell lines and 9p21.3+/MSI-H cell lines, but not 9p21.3+/MSS cell lines. We also sought to validate PELO dependency in tumor organoid models for which pre-existing CRISPR data were unavailable, and therefore, we could only predict their dependence on PELO based on the two identified biomarkers. We demonstrated that 9p21.3-/- or MSI-H tumor organoids were preferentially dependent on PELO as compared to 9p21.3+/MSS models. We also sought to evaluate the effect of PELO knockdown in tumor maintenance in vivo. Leveraging a doxycycline-inducible CRISPRi system to suppress PELO expression, we showed that PELO knockdown significantly shrank 9p21.3-/- tumors grown on the flanks of nude mice.
 We next pursued the culprit lesion in 9p21.3-/- cancers that conferred PELO dependency. Hypothesizing that the loss of a 9p21.3 gene confers PELO dependency, we performed a focused loss-of-function CRISPR/Cas12a screen targeting 9p21.3 genes in the context of PELO knockdown or control conditions. We identified the mRNA surveillance gene FOCAD as a 9p21.3 gene whose knockout lead to preferential impairment in cells with PELO depletion. This led us to hypothesize that FOCAD deletions were conferring PELO dependency. In focused validation, we showed that FOCAD deletion sensitizes cell lines to PELO knockdown. Furthermore, we discovered that restoring FOCAD rescued 9p21.3-/- cells from PELO depletion. Together, these data demonstrated FOCAD loss was both necessary and sufficient for PELO dependency in the 9p21.3-/- context.
 Next, we sought to determine the mechanistic basis for PELO dependency in MSI-H cancers. We hypothesized that a mutation associated with MSI-H sensitizes cells to PELO loss. We observed that deletions in the splicing acceptor site at intron 29 of the superkiller complex (SKIc) member tetratricopeptide repeat domain 37 (TTC37) was the most correlated microsatellite mutation with PELO dependency. Thymidine deletions in this splicing acceptor site were predictive of decreased TTC37 protein levels in cell lines. These data led us to hypothesize that functional loss of TTC37 is the culprit lesion that renders MSI-H cancers dependent on PELO for survival. Evaluating this hypothesis, we found that TTC37 knockout sensitized cells to PELO depletion. We also showed that exogenous expression of TTC37, but not control, cDNA rescued the viability of MSI-H cell lines from PELO knockdown. Taken together, these data demonstrated that functional TTC37 impairment is sufficient and necessary for PELO dependency in the MSI-H setting and could serve as a refined predictive biomarker for PELO dependency.
 We next sought to investigate how FOCAD loss and TTC37 mutations converge to confer PELO dependency. TTC37 interacts with Ski2-like RNA helicase (SKIV2L) and WD Repeat Containing Protein 61 (WDR61) to form the SKIc, which promotes the 3’-5’ exosome degradation of mRNA from stalled ribosomes. We examined the Cancer Cell Line Encyclopedia proteomics dataset and demonstrated positive correlations between FOCAD, TTC37, and SKIV2L. Consistent with these datasets, we observed decreased TTC37 and SKIV2L protein levels in 9p21.3-/- cell lines and MSI-H cell lines. Based on these results, we asked if FOCAD was required and/or sufficient to maintain TTC37 and SKIV2L protein stability. We expressed FOCAD cDNA in 9p21.3-/- cells and observed increased TTC37 and SKIV2L protein levels. Furthermore, FOCAD knockout reduced SKIV2L and TTC37 protein levels in 9p21.3+/MSS cells. Together, these data support FOCAD as a critical regulator of SKIV2L and TTC37 stability. We hypothesized that FOCAD deletions and TTC37 mutations were capturing SKIc loss, thereby increasing reliance on PELO for survival. We reasoned that if functional loss of the SKIc was responsible for PELO dependency, SKIV2L deletion would also confer increased PELO dependency. We knocked SKIV2L out and observed that loss of SKIV2L sensitized cells to PELO loss. These data support our hypothesis that loss of SKIc function confers increased dependence on PELO for survival.
 To assess how SKIc-deficient cancer cells respond to PELO suppression, we performed gene expressing profiling. We found that SKIc-deficient cells had a robust transcriptional response with gene set enrichment analyses demonstrating upregulation of the unfolded protein response, a signaling network responding to aggregated unfolded or misfolded proteins. We validated these findings by demonstrating that PELO depletion preferentially upregulated C/ebp homologous protein (CHOP) expression and induced X-BOX binding protein (XPB1) splicing, markers of UPR activation, in the SKIc-deficient setting.
 Together, our observations reveal that MSI-H associated mutations and large 9p21.3 deletions involving TTC37 and FOCAD, respectively, independently impair the SKIc and confer a synthetic lethal relationship with PELO. Since MSI-H and large 9p21.3 deletions are frequently observed in patients, a PELO-based therapeutic could have broad implications for clinical oncology.


 Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel Lubicki, Ashir Borah, Nicholas Lofaso, Sohani Das Sharma, Riya V. Kishen, Joshua Dempster, Francisca Vazquez, Edmond M. Chan. Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Triple-negative breast cancer (TNBC) is an aggressive and hard-to-treat form of breast cancer that has limited treatment options. In TNBC, changes in DNA (mutations) associated with the cancer, frequently occur in the TP53 gene. The TP53 gene normally helps cells repair damage or self-destruct when they are not functioning properly; however when mutated, TP53 cannot function properly. In this study, we found that different types of TP53 mutations, affect how cancer cells manage calcium—an important signal that controls how cells grow and die. Some of these mutations made the cancer cells less likely to die under stress, which may explain why they can resist certain treatments. To address this problem, we tested a TP53 reactivator that helps restore the normal functions of TP53. This TP53 reactivator increased calcium levels in the cancer cells and made them more responsive to treatment. Our findings help explain why some cancers are more aggressive and suggest a new approach to treatment by targeting both the mutant TP53 and the disrupted calcium signals. This could lead to better therapies and improved outcomes for people with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd38d72cf5acc6bf0f18946326e613c63624e9b4" target='_blank'>
              TP53 Mutation-Specific Dysregulation of Store-Operated Calcium Entry and Apoptotic Sensitivity in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Kaneez E. Rabab, Paul J. Buchanan, Grace Colley, Anita White, Aisling Murphy, Chloe McCormack, Alexander J. Eustace
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell-cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti-tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non-neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations.
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of internal medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Chromosomal instability (CIN+) characterizes a continuous mitotic dysfunction leading to karyotypic abnormalities and aneuploidy, commonly observed in various human cancers, including ovarian, triple-negative breast (TNBC), and head-and-neck tumors, with significant implications for tumor progression and prognosis. KIF18A, a mitotic kinesin motor protein essential for chromosome alignment during cell division, is critical for the proliferation of CIN+ cells but dispensable in near-diploid cells. Inhibition of KIF18A results in mitotic delays, multipolar spindle formation, and elevated cell death in CIN+ tumor cells. YF550 is a highly selective and potent inhibitor in KIF18A inhibitor and CIN+ cellular proliferation assays, as well as in phospho-Histone H3 mitotic marker evaluation. YF550 exhibits exceptional selectivity for KIF18A over related protein Eg5 and several other proteins. In contrast to the Eg5 inhibitor Ispinesib, YF550 does not inhibit normal PBMC proliferation. Notably, the cellular potency of YF550 correlates with the aneuploidy score of CIN+ cancer cells, which may serve as a potential biomarker for patient stratification. YF550 has demonstrated synergistic effects when combined with various agents, including Olaparib (PARP inhibitor), PF-07104091 (CDK2 inhibitor), Elimusertib (ATR inhibitor), Volasertib (PLK1 inhibitor), and MMAE, in cellular proliferation assays. In the OVCAR3 ovarian cancer xenograft model, YF550 has shown superior efficacy compared to KIF18A inhibitor AMG650 at a 5 mg/kg dose, with no significant impact on body weight. Furthermore, YF550 induced tumor regression in both the OVCAR8 ovarian and JIMT-1 HER2 positive breast cancer xenograft models. YF550 exhibits superior ADME and PK property suitable for clinical development for CIN+ cancers with high aneuploidy score and has the potential for combination therapy with PD-1/L1 antibodies, PARP1 inhibitors, and MMAE based ADC.


 Yaolin Wang, Rongfeng Liu, Hong Yang, Yong Luo, Younong Yu, Jifang Weng, Yan Weng, Hong Mei, David Dai. YF550 is a potent and selective inhibitor of KIF18A, specifically targeting CIN+ cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB188.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe15ddac83bc61bb3a18904050bca997f44ada15" target='_blank'>
              Abstract LB188: YF550 is a potent and selective inhibitor of KIF18A, specifically targeting CIN+ cancer cells
              </a>
            </td>
          <td>
            Yaolin Wang, Rongfeng Liu, Hong Yang, Yong Luo, Younong Yu, Jifang Weng, Yan Weng, Hong Mei, David Dai
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The relationship between epigenetic modifications that influence gene regulation, such as histone deacetylation, microRNA regulation and DNA methylation, and human diseases has become increasingly recognised. Dysregulation of DNA methylation has been implicated in genomic imprinting and tumourigenesis. Previous studies identified several epigenetic modifier genes that directly cause genomic DNA (gDNA) methylation such as DNA methyltransferases (DNMTs).DNMTaberrations usually affect genome-wide methylation profiles, deactivating many tumour suppressor genes by methylation of their promoters. In this study, we describe a unique case with several tumours of different lineages, likely caused by a germline pathogenic DNA (cytosine-5-)-methyltransferase 3 alpha (DNMT3A)variant.We evaluated a middle-aged female who suffered from multiple metachronous and synchronous tumours, mostly benign, including subcutaneous lipomas and mesenchymal tumours, multinodular goitre, a parathyroid adenoma, a complex meningioma, a benign breast tumour, bilateral non-functioning adrenal nodules, hepatic and renal cysts and a pancreatic neuroendocrine tumour. The patient’s gDNA was subjected to whole-exome sequencing and strict bioinformatics analysis. The possible effect of the variant was investigated using multiple in-silico analysis tools and protein three-dimensional modelling.A heterozygous missenseDNMT3Avariant (c.2063G>A, p.R688H) was identified. The identified variant was Sanger sequenced in all available patient’s samples. Protein modelling strongly supports the pathogenic role of this variant in the development of the several tumours that this patient had.We report the c.2063G>ADNMT3Avariant as a potential cause of multiple unrelated tumours. This is the first report of such a case and one of a few reports ofDNMT3Agermline variants causing tumourigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/225d0c30ea3c8c06200141f21fb55e55e3264e9c" target='_blank'>
              Multiple tumours of different lineages likely related to aDNMT3Apathogenic R688H variant
              </a>
            </td>
          <td>
            Kheloud M. Alhamoudi, Meshael M. Alswailem, B. Alghamdi, Maram Alshuwaymi, Karla A. Peña-Guerra, Stefan T. Arold, H. Al-Hindi, Ali S Alzahrani
          </td>
          <td>2025-06-01</td>
          <td>BMJ Connections Clinical Genetics and Genomics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bcb6bbbf4e6c64ab8d5a6fa755b5306451c07b" target='_blank'>
              EHMT1 mediates cellular motility in embryonal rhabdomyosarcoma by activating SOX8 expression.
              </a>
            </td>
          <td>
            Upasana Bajaj, Dipanwita Das, Jia Yu Leung, Nurul Fateha Binti Abdul Rashi, R. Taneja
          </td>
          <td>2025-06-04</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is an aggressive cancer of bone and soft tissue that predominantly affects children and young adults. A chromosomal translocation joins the low-complexity domain (LCD) of the RNA-binding protein EWS (EWSLCD) with the DNA-binding domain of Friend leukemia integration 1 (FLI1DBD), creating EWS::FLI1, a potent fusion oncoprotein essential for EwS development and responsible for over 85% of EwS tumors. EWS::FLI1 forms biomolecular condensates in vivo and promotes tumorigenesis through mediation of aberrant transcriptional changes and by interfering with the normal functions of nucleic acid-binding proteins like EWS through a dominant-negative mechanism. In particular, the expression of EWS::FLI1 in EwS directly interferes with the biological functions of EWS leading to alternate splicing events and defects in DNA-damage repair pathways. Though the EWSLCD is capable of phase separation, here we report a direct interaction between FLI1DBD and EWSLCD that enhances condensate formation and alters the physical properties of the condensate. This effect was conserved for three related E-twenty-six transformation-specific (ETS) DNA-binding domains (DBDs) while DNA binding blocked the interaction with EWSLCD and inhibited EWS::FLI1 condensate formation. NMR spectroscopy and mutagenesis studies confirmed that ETS DBDs transiently interact with EWSLCD via the ETS DBDs "wings." Together these results revealed that ETS DBDs, particularly FLI1DBD, enhance EWSLCD condensate formation and rigidity, supporting a model in which electrostatic and structural interactions drive condensate dynamics with implications for EWS::FLI1-mediated transcriptional regulation in EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90380493c2e0c6b1757985f2684e59828098025" target='_blank'>
              Phase separation of the oncogenic fusion protein EWS::FLI1 is modulated by its DNA-binding domain.
              </a>
            </td>
          <td>
            Emily E. Selig, Erich J Sohn, Aiola P. Stoja, Alma K. Moreno-Romero, Shivani Akula, Xiaoping Xu, Alexander J R Bishop, David S. Libich
          </td>
          <td>2025-05-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce179919f141331dbb2b8eba6f85098cd0966d6" target='_blank'>
              Complex associations of genetic/epigenetic variations of CGG repeats with patient phenotypes in oculopharyngodistal myopathy
              </a>
            </td>
          <td>
            Nobuyuki Eura, Satoru Noguchi, M. Ogawa, K. Sonehara, Ai Yamanaka, Shinichiro Hayashi, Yukinori Okada, Kazuma Sugie, I. Nishino
          </td>
          <td>2025-05-15</td>
          <td>None</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The miniature RNA‐guided endonuclease IscB, as the evolutionary progenitor of Cas9, is attracting increased attention for genome editing due to its compact size and suitability for in vivo delivery. However, the poor editing efficiency of IscB in eukaryotic cells presents a significant challenge to its widespread application in precise site‐specific human genome editing. In this study, we employed structure‐guided rational design and protein engineering to optimize OgeuIscB, resulting in the identification of enIscB‐F138R, which further enhanced editing activity up to 3.49‐fold in mammalian cells compared to the high‐activity OgeuIscB variant enIscB. Furthermore, we engineered an enIscB‐F138R nickase‐based adenine base editor, termed miABE‐F138R, exhibiting enhanced base editing efficiency relative to miABE. To illustrate the practical applications of miABE‐F138R, we applied it to rectify the prevalent R560C mutation in Pde6β associated with autosomal recessive retinitis pigmentosa, resulting in a significant improvement in activity compared to miABE. In conclusion, enIscB‐F138R and miABE‐F138R offer adaptable platforms for genome editing with potential significance in future biomedical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b215d27435524a8988149d395c9b504ac3e4e7d" target='_blank'>
              Directed Evolution of OgeuIscB With Enhanced Activity in Human Cells
              </a>
            </td>
          <td>
            Jineng Lv, Jiang Jin, Liujun Ding, Lue Xiang, Bintao Xie, Kunchao Wu, Qi Chen
          </td>
          <td>2025-04-25</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Karyotyping, SNP arrays, fluorescence in situ hybridisation and next generation sequencing techniques have greatly improved our understanding of the genetic drivers of acute lymphoblastic leukemia (ALL). In the past years, another layer of genetic data has been added by the study of mutational signatures, patterns of somatic mutations that represent specific mutational mechanisms. Mutational signatures can give insight in tumor development, but also reveal mutagenic side-effects of treatment in relapse samples. Multiple treatment-related mutational signatures have been detected in relapsed ALL that could play a role in therapy resistance and relapse development. In fact, multiple pathogenic driver mutations have been attributed to these treatment-related mutational processes, including a recurrent TP53 mutation. Studies in childhood ALL revealed that thiopurine exposure is the most common source of therapy-related mutagenicity in ALL and presents differently when patients are DNA mismatch repair deficient. Thiopurine-induced DNA damage indicates that leukemic cells were able to survive thiopurine exposure. This could be due to metabolic defects, acquired mutations that induce thiopurine resistance during treatment or resistance to drugs synergizing with thiopurines. In this review, we discuss the types and prevalence of treatment-related mutational signatures in ALL, and explore mechanisms of thiopurine cytotoxicity and mutagenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c85dc6ce574d477720be7189fec938db38846c36" target='_blank'>
              NOTCH1 Acts as a Tumor Suppressor That Induces Early Differentiation in Head and Neck Cancer
              </a>
            </td>
          <td>
            Chenfei Huang, Shhyam Moorthy, Qiuli Li, K.M. Ahmed, Defeng Deng, Jiping Wang, X. Rao, Jiexin Zhang, Yuanxin Xi, Jing Wang, Zhiyi Liu, Noriaki Tanaka, David A. Wheeler, Eve Shibrot, Rami Saade, C. Pickering, T. Xie, Adel K. El-Naggar, A. Osman, Kunal Rai, P. Zweidler-McKay, J. Heymach, L. Byers, Faye M. Johnson, V.C. Sandulache, Jeffrey N. Myers, P. Yadollahi, M.J. Frederick
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Merkel Cell Polyomavirus (MCPyV) is recognized as the major aetiological agent of Merkel Cell Carcinoma (MCC), an aggressive skin tumor. MCPyV-mediated oncogenesis is strictly dependent on viral integration and the expression of a truncated form of the Large T Antigen (LT). Moreover, like other oncogenic DNA viruses, MCPyV may interfere with the DNA damage response (DDR) machinery, thus promoting genomic instability and tumorigenesis. Therefore, the objective of this study was to characterize MCPyV infection in 7 MCC patients and to elucidate the plausible role of the virus in the DDR pathway. MCPyV DNA was detected in 3/7 MCC patients and, as expected, viral integration and LT truncation were observed in virus-positive MCCs, along with the expression of early genes only. Over-expression of DDR genes such as ATM, ATR and their downstream kinases Chk1 and Chk2 was reported in MCPyV-positive MCCs supporting the potential role of the virus in interfering with DDR. Our findings support the established viral aetiology of MCC, and describe, for the first time, an over-expression of DDR components in MCPyV-positive MCC, laying the basis for future studies aimed at investigating the contribution of this pathway to MCPyV-mediated carcinogenesis and exploring the plausible clinical implications of host DDR factors for the treatment of MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cee01380ef93ba0651f961c5653ef76df058128" target='_blank'>
              Investigation of mRNA expression levels of DNA damage response genes in Merkel Cell Polyomavirus-positive Merkel Cell Carcinoma: a pilot study
              </a>
            </td>
          <td>
            S. Passerini, M. Fracella, A. Ferlosio, U. Moens, C. Scagnolari, Guido Antonelli, Marco Ciotti, V. Pietropaolo
          </td>
          <td>2025-05-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6116cf0972c0f5ba1621a1614ba9f7a9a3add325" target='_blank'>
              The SOX gene superfamily in oncogenesis: unraveling links to ncRNAs, key pathways, chemoresistance, and gene editing approaches.
              </a>
            </td>
          <td>
            Ammar Yasir Ahmed, Subasini Uthirapathy, Enwa Felix Oghenemaro, R. M, Rohit Kumawat, Yasser Fakri Mustafa, Muthena Kariem, A. Kadhim, Shilpa Sharma, M. R. Kumar
          </td>
          <td>2025-05-20</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The conserved and essential Cdc13/CTC1-Stn1-Ten1 telomeric complex (CST) ensures chromosome stability by protecting telomere ends and regulating telomerase accessibility. In a recent study, we uncovered mutants of the S. cerevisiae CST, in which damage was sensed by the two major G2/M spindle checkpoints (one is Bub2-dependent and the other one Mad2-dependent), as well as the major G2/M DNA damage checkpoint (Mec1-dependent). In this study, we found, by fluorescence microscopy, that the stability of the mitotic tubulin spindle was profoundly affected in the best-studied of these mutants, stn1-sz2. Additional data from genetic analyses suggested the potential involvement of Stu1 and Stu2, as well as Slk19, in these defects. Throughout this study, we compared the phenotypes of stn1-sz2 with those of cdc13-1, the best-studied CST mutant, which also serves as a prototype of telomere-damage-characterized CST mutants. We propose that stn1-sz2 represents the prototype of cst mutants characterized by tubulin spindle damage. These newly described phenotypes potentially represent the basis for identifying new functions of the CST telomeric complex. These functions might consist of ensuring correct chromosome segregation through the stabilization of the mitotic spindle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e08d1c3e5372f1f89371c0ad710d60359f02ea1" target='_blank'>
              The stn1-sz2 Mutant Provides New Insight into the Impacts of Telomeric Cdc13-Stn1-Ten1 Dysfunction on Cell Cycle Progression
              </a>
            </td>
          <td>
            N. Grandin, Michel Charbonneau
          </td>
          <td>2025-05-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="BACKGROUND AIMS
Intrahepatic cholangiocarcinomas (ICCs) are aggressive liver tumors with high heterogeneity and limited therapeutic options. Although traditionally thought to arise from biliary cells, recent findings suggest that hepatocytes may also serve as a cellular origin for ICC. However, the mechanisms underlying hepatocyte malignant transformation and ICC initiation remain poorly understood.


APPROACH AND RESULTS
We employed oncogene-driven and chemically induced ICC murine models, along with cellular models, to recapitulate the transformation of hepatocytes into ICC. Our findings demonstrate that mature hepatocytes undergo a significant hyperpolyploid state during ICC initiation. Hyperpolyploidy promotes aberrant cell division and chromosomal instability, accelerating hepatocyte transformation and ICC onset. Furthermore, we identified the chromatin remodeling factor Arid1a as a critical suppressor of hyperpolyploidy. Arid1a deficiency disrupts mitotic machinery at the centrosome, driving hyperpolyploidization and ICC tumorigenesis.


CONCLUSIONS
Hepatocytes can transform into ICC through a process involving hyperpolyploidizaiton. This study offers new insights into the pathogenesis of ICC, particularly in patients harboring frequent ARID1A mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb35330c9db09a4f3ded3d2167db23a4e30f36f2" target='_blank'>
              Arid1a deficiency promotes hepatocyte hyperpolyploidy and drives intrahepatic cholangiocarcinoma in mice.
              </a>
            </td>
          <td>
            Qi Bian, Shu Wang, Zimin Song, Fang Liu, Muqing Cao, Nan Yang, Jun Ying, Jia-Syuan Hu, Xinyuan Xiong, Huiqin Zhu, Jun Wu, Jie Yang, Xiaonan Wang, Shunli Shen, Xuxu Sun
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Tumors with BRCA1/2 mutations demonstrate homologous recombination (HR) repair deficiencies, leading to genomic instability. Genomic instability is a defining hallmark of numerous carcinomas, making them susceptible to therapies targeting the DNA damage response (DDR) pathways. Among these, poly (ADP-ribose) polymerase (PARP) inhibitors have shown representative therapeutic options. However, incomplete disease control and the emergence of resistance remain challenges for many patients.
 Ubiquitin-specific protease 1 (USP1) plays a critical role in regulating DNA damage repair including the pathway of translesion synthesis (TLS) and Fanconi anemia (FA), which enables resistance bypass mechanisms against PARP inhibitors (PARPi). Therefore, USP1 has been considered as a synthetic lethal target with BRCA1/2-deficient and PARPi resistant tumors. Many investigational USP1 inhibitors have been discovered and developed, but key leading programs showed limited safety or efficacy. Of these, the phase 1/2 clinical trial of TNG348 was terminated due to toxicity observed in the initial study cohorts and KSQ-4279 showed limited efficacy and hematologic toxicity in its early clinical study. Based on these findings, it is necessary to understand if heme and liver toxicities are induced by on-target inhibition of USP1, and whether efficacy can be improved through higher exposure and tolerability.
 VRN19, a novel oral allosteric USP1 inhibitor, exhibited comparable efficacy, favorable safety, and pharmacological profiles, indicating that the previously identified limitations of USP1 programs may be overcome. VRN19 has been designed to increase target binding affinity, minimize toxicity, and increase solubility. Briefly, VRN19 inhibits the catalytic activity of the USP1-UAF1 complex by directly binding to USP1, confirmed by determining CryoEM complex structures. In the cellular assay, VRN19 showed double-digit nanomolar (nM) GI50, cytotoxic activity against BRCA1-mutated cancer cells, sparing BRCA WT cancer cells over 500-fold, indicating high selectivity to target cells. Moreover, unlike KSQ-4279, VRN19’s oral administration demonstrated dose-proportional drug exposure in the plasma at doses up to 300 mg/kg in mice and effective dissolution in FaSSGF (Fasted State Simulated Gastric Fluid) at 2 mg/mL. In the BRCA1-mutated CDX model (MDA-MB-436) and PDX with resistance to PARP inhibitors, VRN19 demonstrated synergistic effect with PARP inhibitors, particularly with olaparib, and showed efficacy comparable to KSQ-4279 without toxicity. In a 2-week toxicity study in rats, administration of 300 mg/kg VRN19 showed no significant changes in anemia markers, such as RBC, Hb, and reticulocytes, nor hepatotoxicity indicators like GGT and bilirubin. However, administration of 300 mg/kg KSQ-4279 significantly elevated anemia-related markers and GGT levels, consistent with the adverse events observed in clinical trials with KSQ-4279.
 In conclusion, VRN19 is a promising novel USP1 inhibitor with favorable safety and improved therapeutic outcomes.


 Se-Hyuk Kim, Seohyun Jo, Kibum Kim, Seonah Hwang, hayeong Kim, Donghyun Park, Soochan Kim, Youngyi Lee, Junhee Kim, Kyungah Seo, Younho Lee, Daekwon Kim, Sunghwan Kim, Jaeyoung Ahn. VRN19: A novel USP1 Inhibitor with anti-tumor efficacy and favorable safety profile [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58c3658dec4970ad21f7599fdefb7a3f37efb23" target='_blank'>
              Abstract LB015: VRN19: A novel USP1 Inhibitor with anti-tumor efficacy and favorable safety profile
              </a>
            </td>
          <td>
            Se-Hyuk Kim, Seohyun Jo, Kibum Kim, Seonah Hwang, hayeong Kim, Donghyun Park, Soochan Kim, Youngyi Lee, Junhee Kim, Kyungah Seo, Younho Lee, Daekwon Kim, Sunghwan Kim, J. Ahn
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b13792be21e8e149fad24f47b79b2c67bef2640" target='_blank'>
              The protein deacetylase HDAC10 controls DNA replication in malignant lymphoid cells.
              </a>
            </td>
          <td>
            Andreas O Mieland, Giuseppe Petrosino, Mario Dejung, Jia-Xuan Chen, Amitkumar Fulzele, F. Mahmoudi, J.-W. Tu, Al-Hassan M. Mustafa, Yanira Zeyn, Christoph Hieber, Matthias Bros, T. Schnöder, F. Heidel, Sara Najafi, Ina Oehme, Ilse Hofmann, Mike Schutkowski, Sebastian Hilscher, Christian Kosan, F. Butter, S. Bhatia, W. Sippl, Oliver H. Krämer
          </td>
          <td>2025-04-29</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, Reva K Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Numerous long noncoding RNAs (lncRNAs) are generated in response to external stimuli, but the scope and functions of such activity are not known. Here, this study provides insight into how the transcription of lncRNAs is connected to DNA damage response by identifying the lncRNA ZFAS1, which is required for cell cycle arrest, transcription regulation, and DNA repair. Mechanistically, ZFAS1 facilitates dynamic changes in hyperphosphorylated forms of the large subunit of RNA polymerase II (RNAPII) around transcription initiation sites by directly targeting the regulated genes. It is shown that extensive transcription shutdown and concomitant stimulated engagement of RNAPII-Ser2P are crucial for repair and cell survival upon genotoxic stress. Finally, ZFAS1 knockout in mice dampened nucleotide excision repair (NER) and led to kidney dysplasia. Overall, the findings extend the understanding of lncRNAs in DNA damage repair (DDR) and imply a protective role of lncRNA against DDR-deficient developmental disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd71da5d91ac1bdf35ea5bcddbb6ad4cb3db5ea" target='_blank'>
              IncRNA-ZFAS1, an Emerging Gate-Keeper in DNA Damage-Dependent Transcriptional Regulation.
              </a>
            </td>
          <td>
            Jiena Liu, Qing Lu, Zixuan Fan, Jiahui Lin, Nan He, Xin Zhang, Zhaoya Han, Tingting Zhu, Zhenzhen Wu, Yingying Xu, Yuming Wang
          </td>
          <td>2025-05-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal instability in Alzheimer’s disease (AD) neurons has been previously reported. This pilot study aimed to establish a quantitative technique for assessing X chromosome centromere signals using fluorescence in situ hybridization (FISH). Hippocampal brain tissue was collected at autopsy from sporadic AD patients and age- and gender-matched controls. FISH was utilized to detect and measure the intensity of hybridization signals for X chromosome centromeres in the interphase nuclei of hippocampal brain cells. The premature centromere division (PCD) phenomenon, marked by a close bipartite signal appearing as two separated FISH spots, was examined to see if the hybridized DNA amount in each spot matched the expected centromere DNA amount. The technique effectively distinguished between PCD+ and PCD− signals. The average PCD frequency of the X chromosome in the AD group was 7 ± 1%, compared with 3.2 ± 0.84% in the controls. This quantitative approach supports qualitative analyses of FISH centromere spots, reinforcing findings of chromosomal instability in AD. The presence of a double signal at the centromere of a single X chromosome indicates re-entered cell cycles, DNA replication, and PCD in hippocampal neurons. This technique provides a reliable method for identifying PCD + signals and contributes to understanding chromosomal instability in AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682335163a6ea498c4fd17acd3e21f0876fd8d01" target='_blank'>
              Assessment of X Chromosome Centromere Instability in Alzheimer’s Disease: A Quantitative FISH Approach
              </a>
            </td>
          <td>
            B. Spremo-Potparević, Petar Popović, Dijana Topalović, Andrea Pirković, George Perry, L. Živković
          </td>
          <td>2025-06-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2d2fcb3a36276bb7c44a383c03f0aef02728e9" target='_blank'>
              Targeting DNA damage in ageing: towards supercharging DNA repair.
              </a>
            </td>
          <td>
            Arturo Bujarrabal-Dueso, G. Garinis, Paul D. Robbins, J. Vijg, B. Schumacher
          </td>
          <td>2025-06-12</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="As many as 90% of human pancreatic ductal adenocarcinoma (PDAC) tumors harbor gain-of-function mutations in the KRAS oncogene. Recently, inhibitors of the most common KRAS mutation, KRASG12D, have entered the clinical arena. However, early evidence suggests that as monotherapy, KRASG12D inhibitors such as MRTX1133 at best provide brief periods of disease stabilization. Hence, there is a growing interest in understanding the mechanisms through which tumors acquire resistance to KRAS inhibition. In the present study, we generated in vitro models of MRTX1133 resistance and subjected parental and drug-resistant cell lines to RNA sequencing. This suggested that MRTX1133-resistant tumor cells undergo a global shift toward histone acetylation. Inhibition of the histone acetyltransferase EP300 reversed the drug-resistant phenotype in vitro, which subsequent RNA sequencing experiments determined was associated with the suppression of pro-survival FOSL1 signaling. Accordingly, siFOSL1 reversed the MRTX1133-resistant phenotype with similar effects on pro-survival signaling. Given the lack of clinically useful EP300 or FOSL1 inhibitors, we next explored whether inhibitors of the acetylation scanning BET proteins would be similarly effective. The addition of BET inhibitors re-sensitized several resistant cell lines to MRTX1133 and impaired FOSL1-mediated survival signaling in vitro. In murine models of MRTX1133-resistant PDAC, BET inhibition cooperated with MRTX1133 to markedly extend overall survival. As BET inhibitors are currently under clinical testing, the combination of MRTX1133 and BET inhibitors warrants further investigation, particularly in tumors that have developed resistance to KRAS inhibition. SIGNIFICANCE Here, we demonstrate that BET inhibition is effective in PDAC with acquired resistance to KRAS inhibitors. As BET inhibitors are under clinical testing, the combination of KRAS and BET inhibitors warrants consideration in PDAC patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31571cc4d771e5d8cbcfb30a223ba711a12ce0b" target='_blank'>
              Resistance to the KRASG12D Inhibitor MRTX1133 is Associated with Increased Sensitivity to BET Inhibition
              </a>
            </td>
          <td>
            Daniel R. Principe, Jeffrey H Becker, Anastasia E. Metropulos, Alejandra M Marinelarena, Thao D Pham, A. Aissa, H. Munshi
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/818a125f74f34d1eae53b9d769f4c38e62475f7b" target='_blank'>
              In vivo guanine quadruplex structure dynamics and role in genome maintenance in Deinococcus radiodurans
              </a>
            </td>
          <td>
            Himani Tewari, Shruti Mishra, Swathi Kota
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is an aggressive form of cancer, characterized by limited treatment options and a poor prognosis. Immunological therapy is an emerging and promising strategy that has the potential to enhance treatment outcomes and extend the survival of patients with ICC. The role of CD4+ T cells in the development of cancer has attracted attention in previous years. However, the complexities of the tumor microenvironment (TME) impede the full understanding of the roles of CD4+ T cells in cancer. The present study used single-cell RNA sequencing to explore the heterogeneity of the TME during the development of ICC. The results demonstrated that CD4+ T cells were enriched in the TME of ICC and the ratio of regulatory T cells (CD4-forkhead box P3) to central memory T cells (CD4-interleukin 7 receptor) was markedly increased. Secreted phosphoprotein 1 (SPP1) and CD44 showed increased expression levels in tumor cells and T cells from ICC tumor tissues, respectively. Additionally, SPP1 gene expression levels were higher and the ratio of regulatory T cells to central memory T cells was increased in the late stage of ICC compared with the early stage. Elevated levels of SPP1 were associated with a poor prognosis for patients with ICC. Finally, analysis of cell-cell interactions, utilizing established receptor-ligand pairs, demonstrated that ICC tumor cells may engage with immune cells through an SPP1-CD44 axis. Therefore, the results suggest that ICC tumor cells impact CD4+ T-cell differentiation, which could alter the immune TME in ICC and potentially promote ICC tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cf95ea2ea04750f960c2db9f956491fba608b38" target='_blank'>
              Single-cell RNA sequencing analysis of intrahepatic cholangiocarcinoma reveals SPP1 facilitates disease progression via interaction with CD4+ T cells
              </a>
            </td>
          <td>
            Xian Lin, Huanyan Peng, Ke Liu, Wenqi Chen, X. Shao, Chun Ning, Hongchang Li, Dongye Yang
          </td>
          <td>2025-06-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aneuploidy, characterized by the gain or loss of chromosomes, plays a critical role in both cancer and congenital aneuploidy syndromes. For any aneuploidy, we can distinguish between its general effects and its chromosome-specific effects. General effects refer to the common cellular stresses induced by aneuploidy, such as impaired proliferation, proteotoxic stress, and altered metabolism, which occur regardless of the specific chromosome involved and profoundly impact cellular and organismal functions. These generalized stresses often hinder cell fitness but can also, under certain conditions, contribute to cancer progression. In contrast, chromosome-specific effects arise from the altered dosage of particular genes on the gained or lost chromosome. These effects vary depending on the chromosome involved and can provide specific fitness effects in cancer cells or distinct developmental phenotypes in congenital aneuploidies like Down syndrome. Understanding the interplay between these two levels of effects is crucial for deciphering the outcomes of aneuploidy. This review synthesizes current knowledge and discusses future directions for unraveling the hallmarks of aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72635ce4b3e9d86105cc5d392c530bed90825c86" target='_blank'>
              The Hallmarks of Aneuploidy in Cancer and Congenital Syndromes.
              </a>
            </td>
          <td>
            Pan Cheng, Karan Singh, Roger H. Reeves, Teresa Davoli
          </td>
          <td>2025-05-07</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) represents a major fraction of the total cancer burden worldwide. It has been recently identified that various high‐risk Human Papillomaviruses (HPVs) are present in human CRCs, where they play a critical role in the development and progression of the cancer. Aims In this study, we explored the synergistic effect of the E6/E7 viral oncoproteins of the two most frequently observed HPV types (16 and 18) on KRAS and TP53 mutant CRC cell models. Methods We performed an experimental in vitro study utilizing lipofection to transfect KRAS and TP53 mutant CRC cell models (HCT 116 and HT‐29 respectively) with E6/E7 oncoproteins of HPV types 16 and 18 individually and in combination. Subsequently, we assessed their synergistic effect on cell proliferation, invasion, migration, and survival. In addition, we also compared the protein expression patterns of key epithelial‐mesenchymal transition (EMT) biomarkers like E‐cadherin, fascin, and vimentin among transfected, co‐transfected, and wild‐type cells. Results We found that the co‐expression of E6/E7 of HPV types 16 and 18 enhanced cell proliferation, invasion, migration, and survival in both cell models. Interestingly, this was also accompanied by the deregulation of all three EMT biomarkers, E‐cadherin, fascin, and vimentin. The synergistic effect of the viral oncoproteins in promoting cancer was more pronounced in TP53 mutant cells (HT‐29) as compared to KRAS mutant cells (HCT 116). We also report that HPV type 18 can induce a greater and more sustained oncogenic outcome as compared to HPV type 16. Conclusion Our data indicate that co‐expression of the E6/E7 oncoproteins of HPV types 16 and 18 can enhance oncogenic processes in CRC, especially TP53 mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5af237b0e42715fbf2b20a0bcbc3cb0ed6da7ba" target='_blank'>
              Crosstalk Between the Oncoproteins of High‐Risk Human Papillomaviruses Types 16 and 18 in Colorectal Cancer Cell Models
              </a>
            </td>
          <td>
            Queenie Fernandes, V. Inchakalody, Sarra Mestiri, Takwa Bedhiafi, Shereena Hydrose, Sara S. Bashraheel, M. Merhi, Said Dermime, A. Al Moustafa
          </td>
          <td>2025-06-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 e15032


 Background:
 Homologous recombination deficiency (HRD) is an event triggered by impairment of the homologous recombination repair (HRR) pathway which acts on DNA double-strand breaks and interstrand cross-links (ICL). Although HRD has been shown to elevate the overall genomic instability in tumors inducing favorable responses to immune checkpoint inhibitors (ICI), its clinical utility is observed in specific cancer types like breast, ovarian, low-grade glioma, testicular germ cell tumors and kidney cancers. On the contrary, in many other cancer types, HRD is conducive to the development of immune-cold tumor microenvironment leading to unfavorable treatment responses to ICI. We present, HRD predictive scores with the corresponding IO biomarkers viz. microsatellite instability (MSI) and tumor mutation burden (TMB) pan-cancers.
 Methods:
 Retrospectively, we investigated the biomarker profile (HRD, MSI, TMB) and genomic (HRR pathway genes) profiles of 642 pan-cancer patients using a comprehensive genomic assay using OncoIndx.
 Results:
 Of the cohort of 642 pan-cancer patients, a total of 12.3% (n = 79/642) of patients were detected with high HRD scores, another 16.0% (n = 103/642) were detected with intermediate HRD scores and while 71.7% (n = 460/642) of patients showed low HRD scores. Among the cohorts of high, intermediate and low HRD scores, BRCA1/2 mutation-positive patients were 6.3% (n = 5/79), 5.8% (n = 6/103), and 5.2 (n = 24/460) respectively and HRR gene mutation-positive patients were 17.7% (n = 14/79), 25.2% (n = 26/103), 17.17% (n = 79/460) respectively. Most importantly, the immune-oncology biomarkers MSI and TMB scores were high only in 1.3% (n = 1/79) and 8.9% (n = 7/79) in HRD-high patients which were not significant compared to 1.7% (n = 8/460) and 10.4% (n = 48/460) observed in HRD-low patient cohort. Even in the cohort of patients with intermediate HRD scores, MSI and TMB did not vary significantly. Overall, no significant correlation was observed between high HRD scores vs BRCA1/2 mutations, HRR pathway gene mutations, MSI and TMB statuses. Thus, HRD scores could be predicted high, irrespective of all the associated biomarkers and pathway mutations. This indicates the utility of additional treatment options in BRCA1/2 wild-type or HRR pathway gene wild-type patients.
 Conclusions:
 Although HRD-high patients could be predictive of favorable PARPi and platinum-based therapies, their role in immune activation and ICI therapy response should be carefully considered. Even at a high score, HRD seems to be inducing non-inflammatory tumor microenvironment as opposed to triggering novel cell surface antigens and high TMB aiding in immunotherapy response. However, HRD as an individual biomarker alone adds immense therapeutic benefits even among BRCA-negative patients who could be potential for PARPi/platinum-based therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92c61b0baeee9c5a60d5b38acb29b5a9cb82e2f" target='_blank'>
              The non-canonical role of homologous recombination deficiency in the development of immune-cold tumor microenvironment.
              </a>
            </td>
          <td>
            A. Vasudevan, Aarthi Ramesh, Ankit Jain, Sandhya Iyer, Nidhi Patel, A. Joshi, M. K. Singhal, Sameer Shrirangwar, Chinmaya Kumar Pani, Ganapathi Bhat, V. Patil, Dr T. Raja, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Background: Breast cancer is the most frequently diagnosed cancer and the leading cause of cancer-related death among women worldwide. Triple negative breast cancer (TNBC) represents 15-20% of all incident breast cancers and is a highly aggressive subtype. Neo(adjuvant) chemotherapy (NAC) is standard for early stage TNBC; however, for those with residual breast cancer following NAC, recurrence is common and associated with few effective therapeutic approaches. Thus, identification of novel resistance mechanisms and alternative treatment strategies remain an unmet clinical need. Circular RNAs (circRNAs) are a newly identified class of largely noncoding RNA molecules with covalently closed circular structures. Recent studies suggest that circRNAs play crucial roles in regulating tumor development, progression and chemoresistance. However, the role of circRNAs in mediating chemotherapy resistance remains unknown. Methods: As a first step towards identifying circRNAs that participate in the development of chemoresistance, and to determine if targeting such circRNAs may be an efficacious strategy, doxorubicin-resistant (Doxo-R), paclitaxel-resistant (PTX-R), and double-resistant (DP-R) cell lines were generated from MDA-MB-231 cell lines. Human circRNA microarrays were utilized to profile the expression of approximately 14,000 known circRNAs in normal breast tissue, matched TNBC patient-derived xenografts (PDX) generated prior to and following NAC, and TNBC chemosensitive and chemoresistant cell lines. Top hits were validated using RT-PCR. Bioinformatic approaches were employed to identify circRNA-miRNA-mRNA networks regulated by differentially expressed circRNA transcripts. A reverse genetics approach was also employed using CRISPR to interrogate the contributions of over 4000 specific circRNA in driving the development of resistance to doxorubicin and paclitaxel. Results: circRNA microarray profiling identified 429 and 310 transcripts differentially expressed in doxorubicin and paclitaxel resistant cells, respectively, compared to parental chemosensitive cell lines (|FC| ≥ 1.5; p value < 0.05). Given that circRNAs can act as microRNA (miRNA) sponges, thereby inhibiting miRNA activity, we computationally identified circRNA-miRNA pairs, and assessed the abundance of mRNA transcripts predicted to be targets of paired miRNAs via RNA-seq. A differentially expressed circRNA-miRNA-mRNA network was further constructed. Dysregulated canonical pathways and gene ontology (GO) enrichment analysis were determined from these networks and revealed multiple GO term included negative regulation of miRNA transcription, miRNA and RNA metabolic process, and endoplasmic reticulum-associated degradation (ERAD) pathway. The CRISPR-Cas13 screen is ongoing at this time but results will be available prior to the meeting. Conclusions: We have identified cancer specific circRNAs in TNBC cell lines, PDX models and patient tumors in the context of chemotherapy sensitive and chemotherapy resistant disease. Specific biological processes, pathways and signaling cascades have been implicated in the down-stream consequences of dysregulated circRNAs and in the potential development of resistance. Our ongoing studies seek to identify specific circRNAs sufficient to drive resistance to chemotherapy. Outcomes of these studies highlight the potential of specific circRNAs to serve as predictive/prognostic biomarkers and novel therapeutic targets for advanced forms of TNBC.
 Citation Format: Xiyin Wang, Michael J. Emch, Xiaojia Tang, Esther P.B. Rodman, Liewei Wang, Judy C. Boughey, Matthew P. Goetz, Krishna R. Kalari, John R. Hawse. The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb284df6818a5ca75827ea02126a84e48b2886e9" target='_blank'>
              Abstract P3-02-22: The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance
              </a>
            </td>
          <td>
            Xiyin Wang, M. Emch, Xiaojia Tang, E. Rodman, Liewei Wang, Judy C. Boughey, Matthew P. Goetz, K. Kalari, John R. Hawse
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Synovial sarcoma (SS) is an aggressive malignant soft tissue sarcoma of undetermined tissue origin that harbors potential metastases and recurrences, leading to a poor prognosis. Polo-like kinase 1 (PLK1) is an essential member of the serine/threonine kinase family, which plays a key role in cell cycle progression and other cellular processes. Therefore, PLK1 could act as a promising target for treating various cancers, including a low incidence of sarcoma. Methods SS paraffin-embedded tissue samples were selected to detect the expression of PLK1 and apoptosis- and autophagy-related proteins using immunohistochemistry. We analyzed the effects of low PLK1 expression on the SS cell cycle and function using the PLK1 inhibitor BI2536 in SW982 and SSX1 cells (cells with high expression of the SYT-SSX1 fusion gene). Western blotting was used to detect the expression of PLK1 and apoptotic proteins in different groups. Finally, the antitumor effects and safety of BI2536 were evaluated using xenograft tumor experiments in nude mice. Results In SS tissues, PLK1 was overexpressed and autophagy-related proteins were expressed at low levels, leading to poor prognosis. Suppressing PLK1 expression using the PLK1 inhibitor BI2536 decreased the proliferation, migration, and invasion abilities of SW982 and SYT-SSX1 cells and promoted the expression of the apoptotic protein Bax. Meanwhile, we confirmed that SS cell apoptosis increased and G2/M cycle arrest occurred after PLK1 expression was inhibited. We found that BI2536 significantly inhibited the growth of SS-xenografted tumors in nude mice, and the expression of stem cell-related factors significantly decreased. Conclusion PLK1 overexpression was closely associated with poor prognosis. Suppressing PLK1 expression significantly inhibited the growth, invasion, and migration of SS cells, promoted apoptosis, and blocked the G2/M phase of the cell cycle. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06740-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37d396cfbb289d363c88b01a59ea841faba59fb" target='_blank'>
              Overexpression of oncogenic polo-like kinase 1 disrupts the invasiveness, cell cycle, and apoptosis in synovial sarcoma
              </a>
            </td>
          <td>
            Jun Zhang, Zhong Li, Minghui Xie, Ning Wang, Hao Liu, Feng Li, Li-Juan Pang, Yan Qi
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 e17631


 Background:
 Poly (ADP-ribose) polymerase inhibitors (PARPi) has been studied to show positive efficacy in endometrial cancer (EC). However, rational candidates for precision treatment with PARPi need to be further explored. Recently, it has been reported that tumors with homologous recombination deficiency (HRD) are sensitive to PARPi. Here, we aimed to assess the relationship between HRD and molecular subgroups in EC.
 Methods:
 HRD statues and gene characteristics of 372 EC patients were retrospectively collected from August, 2022 to August, 2024. HRD score was calculated as the sum of loss of heterozygosity (LOH), telomeric allelic imbalance (TAI), and large-scale state transitions (LST) scores based on the next-generation sequencing (NGS) data.
 Results:
 Of the EC that underwent NGS testing, there were 5.38% (20/372) patients with
 BRCA1/2
 mutations, and the top two
 BRCA1/2
 mutated subgroups were microsatellite instability-high (MSI-H) (2.96%) and copy number low (CN-L) (1.88%) subgroups. However, there were 10.75% (40/372) patients with positive HRD, which is largely restricted to copy number high (CN-H) (4.57%), CN-L (2.96%) and MSI-H (2.96%) subgroup. Interestingly, based on HRD score with the cutoff of 30, there were mostly CN-H and CN-L subgroups and almost no
 POLE
 mutation and MSI-H subgroups in non
 BRCA1/2
 mutation and positive HRD patients.
 Conclusions:
 BRCA1/2
 mutations and positive HRD were seen in 5.38% and 10.75% EC patients, respectively. HRD score testing, based on LOH, TAI, and LST, may help select patients benefiting from PARPi with particular reference to CN-H and CN-L subgroups.


 Prevalence of HRD statues in different molecular subgroups of EC.





 HRD
 Non-HRD




 Molecular Subgroups
 All

 BRCA1/2
 mutation

 HRD score≥30




 POLE
 mut

 1(0.27%)
 1(0.27%)
 0(0%)
 8(2.15%)


 MSI-H
 11(2.96%)
 11(2.96%)
 0(0%)
 64(17.20%)


 CN-H
 17(4.57%)
 1(0.27%)
 16(4.30%)
 64(17.20%)


 CN-L
 11(2.96%)
 7(1.88%)
 4(1.08%)
 196(52.69%)






 HRD: Homologous recombination deficiency; EC: endometrial carcinomas;
 POLE
 mut:
 POLE
 mutations; MSI-H: microsatellite instability-high; CN-L: copy number low; CN-H: copy number high.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ff21a16e023a00431cd6937437785071b41ecc" target='_blank'>
              Prevalence of homologous recombination deficiency in different molecular subgroups of endometrial carcinomas.
              </a>
            </td>
          <td>
            Yang Rao, Wenwen Zhang, Na Zhang, Han Han, Yong Xiao, Ping Liu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891242f777da10d08f786d978bb8ecf69ade37e9" target='_blank'>
              Replaying germinal center evolution on a quantified affinity landscape
              </a>
            </td>
          <td>
            William S DeWitt, Ashni A. Vora, Tatsuya Araki, Jared G. Galloway, T. Alkutkar, Juliana Bortolatto, Tiago B.R. Castro, Will Dumm, Chris Jennings-Shaffer, Tongqiu Jia, L. Mesin, G. Ozorowski, Juhee Pae, Duncan K. Ralph, Jesse D. Bloom, Armita Nourmohammad, Yun S. Song, A. Ward, Tyler N. Starr, Frederick A. Matsen, G. Victora
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Branched-chain amino acid (BCAA) metabolism is dysregulated in colorectal cancer (CRC), with elevated plasma BCAA levels significantly associated with an increased risk of developing the disease. However, whether BCAAs directly promote CRC progression and their underlying mechanisms remain unclear. Methods In this study, we investigated the metabolic alterations in KRAS-mutant CRC. We examined the effects of restricting BCAA supply on the proliferation and metastasis of KRAS-mutant CRC cells both in vitro and in vivo. Results We found that in KRAS-mutant CRC, BCAAs and their metabolic products accumulate markedly. Restricting the BCAA supply specifically inhibits the proliferation of KRAS-mutant CRC cells but does not affect metastasis. In these cancer cells, enoyl-CoA hydratase-1 (ECHS1), a key enzyme in BCAA metabolism, is downregulated. Furthermore, BCAAs enhance the acetylation of lysine 204 on ECHS1, impairing its ability to bind enoyl-CoA and reducing its catalytic activity. This modification triggers the ubiquitination of ECHS1 and its subsequent degradation, diminishing BCAA catabolism and leading to its cellular accumulation. This accumulation activates the mTORC1 signaling pathway, which induces the transcriptional activation of downstream target proteins and promotes the malignant progression of CRC. Conclusions Limiting BCAA intake not only suppresses tumor growth in KRAS-mutant CRC but also enhances the efficacy of the KRAS G12D inhibitor MRTX1133 and the monoclonal antibody bevacizumab. Our findings reveal a previously unknown regulatory mechanism of ECHS1 in CRC and offer new potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03399-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa2272525b1d439b86c30cb63146c406262aab3" target='_blank'>
              Acetylation-induced degradation of ECHS1 enhances BCAA accumulation and proliferation in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Zhenkang Li, Zhengyu Liu, Mingdao Lin, Huayang Pan, Yuechen Liu, Yang Liu, Yuwen Xie, Jinchao Zhang, Shenyuan Guan, Yongsheng Li, Mulan Zhu, Yuan Fang, Zhiyong Shen, Haijun Deng
          </td>
          <td>2025-05-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Many cancers, such as fibrolamellar carcinoma (FLC), Ewing’s sarcoma, chronic myelogenous leukemia and rhabdomyosarcoma, are driven by fusion oncoproteins (1-6). Since each fusion partner originates from a native protein with essential functions, it is important but difficult to selectively target fusion oncoproteins while sparing native components. FLC is a usually lethal primary liver tumor of adolescents and young adults driven by an oncogenic fusion DNAJB1::PRKACA, which results from the fusion of the first exon of DNAJB1, a heat shock protein, to the bulk of the coding region of PRKACA, the catalytic subunit of protein kinase A (PKA) (7-10), forming DNAJB1::PRKACA. The fusion protein not only triggers the tumor, but drives the tumor and the tumor is oncogenically addicted to the fusion oncoprotein. Elimination of the mRNA encoding DNAJB1::PRKACA with an shRNA (11) or an siRNA (12) is sufficient to stop growth and kill the tumor. However, because the active sites of the DNAJB1::PRKACA and native PRKACA are structurally indistinguishable (13), designing inhibitors specific to the fusion protein is challenging.
 To test therapeutics, we developed a number of model systems including organoids and patient-derived xenografts grown in mice, genetically engineered mouse models, genetically engineered human hepatocytes and FLC tumors cells directly from patient surgical resection. These were validated for their transcriptome, proteome, phosphoproteome, metabolome and drug-sensitivity. Currently, there is no accepted therapy for FLC. Our testing of agents that are currently being tried in the clinic, found no therapeutic window between the FLC cells and normal human hepatocytes. We identified several potential upregulated targets in FLC tumors at both the transcriptomic and proteomic level (14,15). However, inhibitors against these targets showed no therapeutic advantage (16).
 Here, we present three therapeutic strategies that were successful and are applicable to other Fusion Oncoprotein-driven Malignancies.
 Each of these three approaches are in, or moving towards, clinical trials and each of these pipelines are applicable to other Fusion Oncoprotein-driven Malignancies.
 References1. Verma, S.K., Witkin, K.L., Sharman, A., and Smith, M.A. (2024). Targeting fusion oncoproteins in childhood cancers: challenges and future opportunities for developing therapeutics. Journal of the National Cancer Institute 116, 1012-1018. 10.1093/jnci/djae075.
 2. Wachtel, M., Surdez, D., Grunewald, T.G.P., and Schafer, B.W. (2024). Functional Classification of Fusion Proteins in Sarcoma. Cancers (Basel) 16. 10.3390/cancers16071355.
 3. Gao, Q., Liang, W.W., Foltz, S.M., Mutharasu, G., Jayasinghe, R.G., Cao, S., Liao, W.W., Reynolds, S.M., Wyczalkowski, M.A., Yao, L., et al. (2018). Driver Fusions and Their Implications in the Development and Treatment of Human Cancers. Cell Rep 23, 227-238 e223. 10.1016/j.celrep.2018.03.050.
 4. Simon, S.M. (2023). Fighting rare cancers: lessons from fibrolamellar hepatocellular carcinoma. Nature reviews. Cancer 23, 335-346. 10.1038/s41568-023-00554-w.
 5. Crompton, B.D., Stewart, C., Taylor-Weiner, A., Alexe, G., Kurek, K.C., Calicchio, M.L., Kiezun, A., Carter, S.L., Shukla, S.A., Mehta, S.S., et al. (2014). The genomic landscape of pediatric Ewing sarcoma. Cancer Discov 4, 1326-1341. 10.1158/2159-8290.CD-13-1037.
 6. Delattre, O., Zucman, J., Plougastel, B., Desmaze, C., Melot, T., Peter, M., Kovar, H., Joubert, I., de Jong, P., Rouleau, G., and et al. (1992). Gene fusion with an ETS DNA-binding domain caused by chromosome translocation in human tumours. Nature 359, 162-165. 10.1038/359162a0.
 7. Honeyman, J.N., Simon, E.P., Robine, N., Chiaroni-Clarke, R., Darcy, D.G., Lim, II, Gleason, C.E., Murphy, J.M., Rosenberg, B.R., Teegan, L., et al. (2014). Detection of a recurrent DNAJB1-PRKACA chimeric transcript in fibrolamellar hepatocellular carcinoma. Science 343, 1010-1014. 10.1126/science.1249484.
 8. Kastenhuber, E.R., Lalazar, G., Houlihan, S.L., Tschaharganeh, D.F., Baslan, T., Chen, C.C., Requena, D., Tian, S., Bosbach, B., Wilkinson, J.E., et al. (2017). DNAJB1-PRKACA fusion kinase interacts with beta-catenin and the liver regenerative response to drive fibrolamellar hepatocellular carcinoma. Proceedings of the National Academy of Sciences of the United States of America 114, 13076-13084. 10.1073/pnas.1716483114.
 9. Shirani, M., Levin, S., Shebl, B., Requena, D., Finkelstein, T.M., Johnson, D.S., Ng, D., Lalazar, G., Heissel, S., Hojrup, P., et al. (2024). Increased Protein Kinase A Activity Induces Fibrolamellar Hepatocellular Carcinoma Features Independent of DNAJB1. Cancer research 84, 2626-2644. 10.1158/0008-5472.CAN-23-4110.
 10. Graham, R.P., Lackner, C., Terracciano, L., Gonzalez-Cantu, Y., Maleszewski, J.J., Greipp, P.T., Simon, S.M., and Torbenson, M.S. (2018). Fibrolamellar carcinoma in the Carney complex: PRKAR1A loss instead of the classic DNAJB1-PRKACA fusion. Hepatology 68, 1441-1447. 10.1002/hep.29719.
 11. Neumayer, C., Ng, D., Jiang, C.S., Qureshi, A., Lalazar, G., Vaughan, R., and Simon, S.M. (2023). Oncogenic Addiction of Fibrolamellar Hepatocellular Carcinoma to the Fusion Kinase DNAJB1-PRKACA. Clinical cancer research : an official journal of the American Association for Cancer Research 29, 271-278. 10.1158/1078-0432.CCR-22-1851.
 12. Neumayer, C., Ng, D., Requena, D., Jiang, C.S., Qureshi, A., Vaughan, R., Prakash, T.P., Revenko, A., and Simon, S.M. (2024). GalNAc-conjugated siRNA targeting the DNAJB1-PRKACA fusion junction in Fibrolamellar Hepatocellular Carcinoma. Molecular therapy : the journal of the American Society of Gene Therapy 32, 140-151. 10.1016/j.ymthe.2023.11.012.
 13. Tomasini, M.D., Wang, Y., Karamafrooz, A., Li, G., Beuming, T., Gao, J., Taylor, S.S., Veglia, G., and Simon, S.M. (2018). Conformational Landscape of the PRKACA-DNAJB1 Chimeric Kinase, the Driver for Fibrolamellar Hepatocellular Carcinoma. Scientific reports 8, 720. 10.1038/s41598-017-18956-w.
 14. Simon, E.P., Freije, C.A., Farber, B.A., Lalazar, G., Darcy, D.G., Honeyman, J.N., Chiaroni-Clarke, R., Dill, B.D., Molina, H., Bhanot, U.K., et al. (2015). Transcriptomic characterization of fibrolamellar hepatocellular carcinoma. Proceedings of the National Academy of Sciences of the United States of America 112, E5916-5925. 10.1073/pnas.1424894112.
 15. Requena, D., Medico, J.A., Soto-Ugaldi, L.F., Shirani, M., Saltsman, J.A., 3rd, Torbenson, M.S., Coffino, P., and Simon, S.M. (2024). Liver cancer multiomics reveals diverse protein kinase A disruptions convergently produce fibrolamellar hepatocellular carcinoma. Nature communications 15, 10887. 10.1038/s41467-024-55238-2.
 16. Lalazar, G., Requena, D., Ramos-Espiritu, L., Ng, D., Bhola, P.D., de Jong, Y.P., Wang, R., Narayan, N.J.C., Shebl, B., Levin, S., et al. (2021). Identification of Novel Therapeutic Targets for Fibrolamellar Carcinoma Using Patient-Derived Xenografts and Direct-from-Patient Screening. Cancer Discov 11, 2544-2563. 10.1158/2159-8290.CD-20-0872.



 Sanford M. Simon. Developing therapeutics for cancers driven by fusion proteins: Lessons from fibrolamellar carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY34-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/032817b86f9d559dea96a2c9d23f3d4df585f300" target='_blank'>
              Abstract SY34-02: Developing therapeutics for cancers driven by fusion proteins: Lessons from fibrolamellar carcinoma
              </a>
            </td>
          <td>
            Sanford M. Simon
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e359c7ef64214d02915af71ac1f575ebe2c2dd" target='_blank'>
              Chinmo defines the region-specific oncogenic competence in the Drosophila central nervous system
              </a>
            </td>
          <td>
            Phuong-Khanh Nguyen, Francesca Froldi, Louise Y. Cheng
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 2633


 Background:
 HER2/neu mutations or amplifications, present in up to 23% of non-small cell lung cancers (NSCLCs), activate STAT3, promoting inflammation and suppressing adaptive immune responses. Elevated systemic inflammation-immune index (SII) in small cell lung cancer (SCLC) correlates with reduced response to PD-1/L1 immune checkpoint inhibitors (ICIs), leading to worse overall survival (OS) and progression-free survival (PFS). These findings highlight the need to mitigate inflammation in lung cancers to enhance ICI response and improve survival. Biomarker studies can identify genes that differentiate inflammatory and immune pathways, offering therapeutic insights.
 Methods:
 To identify genes regulating the HER2/neu oncogene, HER2/neu-overexpressing mice were crossed with genetically Diverse Outbred (DO) mice. The resulting DO F1 offspring were monitored for HER2/neu-expressing tumor onset and growth. Tumor phenotypes were associated with mouse haplotypes to identify quantitative trait loci (QTL) on chromosomes (chr) 2, 6, and X linked with aggressive HER2/neu tumors. Of the 35 candidate genes harboring protein-coding changes, homologous genes on human chr 1, 10, and X were analyzed using Caris Life Sciences datasets of NSCLC and SCLC: 25,143 primary NSCLC; 12,365 metastatic NSCLC; 621 primary SCLC; and 971 metastatic SCLC where primary biopsies are those taken from the lung and metastatic biopsies from sites other than the lung. Patient cohorts stratified by gene expression (top vs. bottom 50%) were correlated with OS, ICI response, and time on pembrolizumab or atezolizumab treatment (TOT).
 Results:
 Several homologous candidate genes correlated with lung cancer survival in both NSCLC and SCLC. Notably, high expression of
 LILRB4
 , a macrophage-specific checkpoint molecule, was associated with improved survival (HR 0.66–0.84,
 p
 < 0.001) across all lung cancer types and sites. Additionally, elevated
 TSC22D3
 , a glucocorticoid receptor-activated gene regulating anti-inflammatory pathways, including
 LILRB4
 expression, was positively associated with OS in metastatic NSCLC (HR 0.82,
 p
 < 0.00001) and SCLC (HR 0.77,
 p
 < 0.001). Expression of
 LILRB4
 and
 TSC22D3
 further enhanced survival in ICI-treated patients (SCLC: HR 0.72; NSCLC: HR 0.82;
 p
 < 0.001). In NSCLC, high
 LILRB4
 expression conferred an 18% improved TOT with pembrolizumab (HR 0.858,
 p
 < 0.0001).
 Conclusions:
 LILRB4
 and
 TSC22D3
 are key genes linked to improved survival outcomes in both SCLC and NSCLC, likely through their roles in mitigating inflammation. Their association with enhanced ICI responses underscores their potential as therapeutic targets. Future research will evaluate the relationship between
 LILRB4
 and
 TSC22D3
 expression and HER2/neu status in NSCLC and validate protein-level correlations in positive cells. Our long-term goal is to determine the therapeutic potential of
 LILRB4
 and
 TSC22D3
 in SCLC, NSCLC and HER2/neu-positive NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0e9d9066dedda6abe73bbff9473b3f5d6c28e8" target='_blank'>
              Effect of elevated expression of
 LILRB4
 and
 TSC22D3
 on survival in lung cancer.
              </a>
            </td>
          <td>
            B. Hrinczenko, T. Adeyelu, Wei-Zen Wei, A. Elliott, G. Bepler, A. Vanderwalde, B. Halmos, E. Antonarakis, Maryam B Lustberg, Jennifer Jacob
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) is a prevalent haematologic malignancy characterized by significant heterogeneity. Despite the application of aggressive therapeutic approaches, AML remains associated with poor prognosis. Circular RNAs (circRNAs) constitute a unique class of single-stranded RNAs featuring covalently closed loop structures that are ubiquitous across species. These molecules perform crucial regulatory functions in the pathogenesis of various diseases through diverse mechanisms, including acting as miRNA sponges, interacting with DNA or proteins, and encoding functional proteins/polypeptides. Recently, numerous circRNAs have been confirmed to have aberrant expression patterns in AML patients. In particular, certain circRNAs are closely associated with specific clinicopathological characteristics and thus have great potential as diagnostic/prognostic biomarkers and therapeutic targets in AML. Herein, we systematically summarize the biogenesis, degradation, and functional mechanisms of circRNAs while highlighting their clinical relevance. We also outline a series of online databases and analytical tools available to facilitate circRNA research. Finally, we discuss the current challenges and future research priorities in this evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76678c89dbd9eb9ed5d4360a7dd0170b4e909fb0" target='_blank'>
              The emerging functions and clinical implications of circRNAs in acute myeloid leukaemia
              </a>
            </td>
          <td>
            Shuiqing Liu, Xingyu Wan, Yang Gou, Wucheng Yang, Wei Xu, Yuxuan Du, Xiangui Peng, Xiaoqi Wang, Xi Zhang
          </td>
          <td>2025-04-29</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Eliciting DNA damage in tumor cells continues to be one of the most successful strategies against cancer. This is the case for classical chemotherapy drugs and radiotherapy. In the modern era of personalized medicine, this strategy tries to identify specific vulnerabilities found in each patient’s tumor, to inflict DNA damage in certain cell contexts that end up in massive cancer cell death. Cells rely on multiple DNA repair pathways to fix DNA damage, but cancer cells frequently exhibit defects in these pathways, many times being tolerant to the damage. Key vulnerabilities, such as BRCA1/BRCA2 mutations, have been exploited with PARP inhibitors, leveraging synthetic lethality to selectively kill tumor cells and improving patients’ survival. In the DNA damage response (DDR) network, kinases ATM, ATR, Chk1, and Chk2 coordinate DNA repair, cell cycle arrest, and apoptosis. Inhibiting these proteins enhances tumor sensitivity to DNA-damaging therapies, especially in DDR-deficient cancers. Several small-molecule inhibitors targeting ATM/Chk2 or ATR/Chk1 are currently being tested in preclinical and/or clinical settings, showing promise in cancer models and patients. Additionally, pharmacological blockade of ATM/Chk2 and ATR/Chk1 axes enhances the effects of immunotherapy by increasing tumor immunogenicity, promoting T-cell infiltration and activating immune responses. Combining ATM/Chk2- or ATR/Chk1-targeting drugs with conventional chemotherapy, radiotherapy or immune checkpoint inhibitors offers a compelling strategy to improve treatment efficacy, overcome resistance, and enhance patients’ survival in modern oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24f87b378053b6c4aa81e36eb090c1658bdb9c58" target='_blank'>
              Cancer Vulnerabilities Through Targeting the ATR/Chk1 and ATM/Chk2 Axes in the Context of DNA Damage
              </a>
            </td>
          <td>
            Anell Fernandez, Maider Artola, Sergio Leon, N. Otegui, Aroa Jimeno, Diego Serrano, Alfonso Calvo
          </td>
          <td>2025-05-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spindle Pole Body Component 25 (SPC25), a critical component of the NDC80 kinetochore complex, plays an essential role in maintaining chromosomal stability during mitosis. Recent studies have revealed its aberrant expression in various cancers, highlighting its potential as both a diagnostic biomarker and a therapeutic target. This review provides a comprehensive analysis of the molecular mechanisms underlying SPC25’s involvement in tumorigenesis, including its regulation of cell cycle progression and interaction with key oncogenic pathways. Furthermore, we discuss its prognostic significance across different cancer types and its potential impact on therapy resistance. The emerging evidence underscores SPC25’s multifaceted role in cancer biology, offering novel insights into its clinical applications. We conclude by exploring future research directions, emphasizing the need for in-depth studies to unravel the precise molecular functions of SPC25 and its therapeutic potential in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c39ca98ea2816bab12e16bae857d1896574f49" target='_blank'>
              The multifaceted functions of SPC25 in cancer: from molecular pathways to targeted therapy
              </a>
            </td>
          <td>
            Yigang Jin, Minjie Chen, Fei Chen, Zhao-feng Gao, Xiaoping Li, Lingyu Hu, Dandan Cai, Siqi Zhao, Zhengwei Song
          </td>
          <td>2025-05-07</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignancy among women globally, with its complexity linked to genetic variations and metabolic alterations within tumor cells. This study investigates the role of fumarate hydratase (FH), a key enzyme in the tricarboxylic acid (TCA) cycle, in breast cancer progression. Our findings reveal that FH mRNA and protein levels are significantly upregulated in breast cancer tissues and correlate with poor patient prognosis and aggressive tumor characteristics. Using in vitro and in vivo models, we demonstrate that FH overexpression enhances breast cancer cell proliferation, migration, and invasion through metabolic reprogramming and by increasing reactive oxygen species (ROS) production. Furthermore, we identify matrix metalloproteinase 1 (MMP1) as a downstream effector of FH, linked to p21 downregulation, elucidating a novel regulatory pathway influencing tumor behavior. Interestingly, unlike its tumor-suppressing role in other cancer types, this study highlights FH's oncogenic potential in breast cancer. Our results suggest that FH enhances cancer cell viability and aggressiveness via both catalytic and non-catalytic mechanisms. This work not only underscores the metabolic adaptations of breast cancer cells but also proposes FH as a potential biomarker and therapeutic target for breast cancer management. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00397-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/757f91ceb4c980ac76d1ca46a85f3acd5f2e6714" target='_blank'>
              Oncogenic role of fumarate hydratase in breast cancer: metabolic reprogramming and mechanistic insights
              </a>
            </td>
          <td>
            Shyng-Shiou F Yuan, Anupama Vadhan, Hieu D. H. Nguyen, Pang-Yu Chen, Chih-Huang Tseng, Ching-Hu Wu, Yu-Chieh Chen, Yi-Chia Wu, Stephen Chu‐Sung Hu, S. Lo, Ming-Feng Hou, Yen-Yun Wang
          </td>
          <td>2025-05-29</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recent advances in screening programs and the development of innovative therapeutic strategies have significantly improved the clinical outcomes of cancer patients. However, many patients still experience treatment failure, primarily due to inherent or acquired drug resistance mechanisms. This challenge underscores the urgent need for novel therapeutic targets for the effective treatment of malignancies, as well as cancer-specific biomarkers to enhance early diagnosis and guide interventions. Epigenetic mechanisms, including DNA methylation, have recently garnered growing interest as key regulators of gene expression under both physiological and pathological conditions. Although epigenetic dysregulations are reliable tumor hallmarks, DNA methylation is still not routinely integrated into clinical practice, highlighting the need for further research to translate preclinical findings from the bench to the bedside. On these bases, the present review aims to illustrate the state of the art regarding the role of DNA methylation in cancer, describing the technologies currently available for DNA methylation profiling. Furthermore, the latest evidence on the application of DNA methylation hotspots in cancer diagnosis and prognosis, as well as the impact of epidrugs in cancer care, is discussed to provide a comprehensive overview of the potential clinical relevance of DNA methylation in advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7000e596c1c5da0f5cbf0989fd9d5a1a4286bc4e" target='_blank'>
              Recent advances on gene-related DNA methylation in cancer diagnosis, prognosis, and treatment: a clinical perspective
              </a>
            </td>
          <td>
            Alessandro Lavoro, Daria Ricci, Giuseppe Gattuso, Federica Longo, Graziana Spoto, Anastasia Cristina Venera Vitale, Maria Chiara Giuliana, L. Falzone, Massimo Libra, S. Candido
          </td>
          <td>2025-05-05</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="Research on myeloid neoplasms, a field that has been driving scientific advances in cancer for over 50 years, has yielded many discoveries that have fundamentally reshaped our understanding of cancer biology. These insights, often the product of leukemia research, have been instrumental in developing more mechanism-based treatments in the early 2000s [1]. Recognizing epigenetic dysregulation as a common disease mechanism in myeloid cancers has been groundbreaking regarding recent treatment developments that exploit chromatin-based oncogenic mechanisms. In the case of acute myeloid leukemia (AML), sequencing studies aimed at assessing the complement of genetic alterations demonstrated that more than 60% of AML cases harbored disease-driving mutations in epigenetic regulators. This high prevalence underscores the importance of epigenetic dysregulation in AML pathogenesis [2, 3]. Chromatin regulators commonly control disease-specific transcriptional programs, making them attractive therapeutic targets to manipulate neoplastic gene expression programs, particularly in myeloid neoplasms. Several drugs targeting epigenetic mechanisms and exploiting myeloid disease-specific dependencies have recently been approved for treating myeloid neoplasms. Many additional drugs are currently being investigated in clinical trials, and numerous new compound developments are being studied in preclinical studies. This manuscript will review (1) chromatin-based disease mechanisms, such as DNA methylation, chromatin regulatory complexes, and histone modifications, currently investigated for therapeutic exploitation in myeloid malignancies, and (2) therapeutic developments already approved or investigated for treating these diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9903badc85679a15da9a5a2e6c4c1cc4ffc6496" target='_blank'>
              The evolving landscape of epigenetic target molecules and therapies in myeloid cancers: focus on acute myeloid leukemia and myeloproliferative neoplasms.
              </a>
            </td>
          <td>
            Michael W M Kühn, Naveen Pemmaraju, F. Heidel
          </td>
          <td>2025-05-15</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Serine metabolism is of growing biologic and therapeutic interest in cancer. Upregulation of the serine synthesis pathway (SSP) can fuel tumor growth, and cancers with this phenotype are often sensitive to SSP inhibitors. In parallel, dietary restriction of serine and glycine (SG) can suppress some cancers, but the determinants of sensitivity to this approach are poorly understood. This is especially true in acute myeloid leukemia (AML), where serine metabolism has been less explored. We report that a subset of human AML cell lines and primary samples are completely dependent on external serine, known as serine auxotrophy. These leukemias consistently suppressed the SSP enzyme PSAT1, failed to synthesize serine, responded to SG restriction in vivo, and were rescued by restoring PSAT1. We also found that AML with an SF3B1 K700E mutation showed additional dependence on the SSP enzyme PHGDH, that SG restriction synergized with venetoclax in serine auxotrophic AML, and that MECOM rearrangement was strongly associated with PSAT1 suppression and serine auxotrophy. These findings define a metabolically distinct AML subtype and nominate it for targeting by SG restriction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c5b66553b8e0f26c1d2319a4806fb04112b3ce" target='_blank'>
              Serine auxotrophy is a targetable vulnerability driven by PSAT1 suppression in AML
              </a>
            </td>
          <td>
            Ilias Sinanidis, Panagiotis Tsakiroglou, Benjamin G. Dubner, Rebecca Foertsch, Julia Gondek, InYoung Choi, Bogdan Paun, Liang Zhao, Gabriel Ghiaur, W. Dalton
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e23526


 Background:
 Kaposi sarcoma (KS) is a tumor caused by Kaposi sarcoma herpesvirus (KSHV), also known as human herpesvirus 8 (HHV-8). Unlike other virus-associated tumors, where disease progression often continues even after viral eradication, KS requires the persistent presence of KSHV for disease progression. This unique dependency underscores the potential of strategies targeting the virus to also target the tumor. This study aims to characterize the expression of KSHV viral proteins and identify factors associated with differences in viral gene expression, ultimately formulating treatment targets.
 Methods:
 A total of 43 unique samples were collected at a single academic institution between 2001 and 2024. RNA was extracted from formalin-fixed paraffin-embedded (FFPE) tissue blocks and subjected to bulk RNA sequencing. Reads were mapped to the human genome (GRCh38.p14) using STAR and to the KSHV genome (NC_009333) using Salmon. Transcript abundance in units of Transcripts Per Million (TPM) and estimated read counts to each transcript was quantified with Salmon. Clinical and demographic data from the time of sample collection were extracted from medical records. Differential gene expression analysis, PCA using DESeq2 and hierarchical clustering were done in R.
 Results:
 The 43 samples analyzed were derived from 25 patients, with 23 (92%) being male. Of the 18 patients with known HIV status, 4 (22%) were HIV-positive. None had a history of organ transplantation. Among the 14 patients with available disease status, 5 (38%) had disseminated disease at the time of sample collection. KSHV gene expression was detected in 42 out of 43 samples. The most consistently expressed genes across all samples were ORF75 (42 samples, proximal to the poly-A tail), ORF72/v-cyclin (42 samples), and ORF73/LANA (39 samples). A pronounced pattern of high expression was observed in the latency-associated region of the KSHV genome, which was conserved across all KS tumors. The three clusters of samples generated from hierarchical clustering exhibited different lytic and latent gene profile (lytic, latent, mixed), which are similar to previous studies. The differentially expressed genes between different tumor morphology or HIV status of the host showed a mix profile of lytic and latent genes.
 Conclusions:
 This study highlights the latency-associated region of KSHV as the most consistently and highly expressed region, regardless of baseline characteristics, presenting a potential conserved therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69dfe0589aae838d9a59485221f8f4d5df8fd8a5" target='_blank'>
              Transcriptomic landscape of Kaposi sarcoma: Insights into therapeutic targeting of KSHV.
              </a>
            </td>
          <td>
            Yuewei Fei, P. Costa, M. Junejo, Michelle Li, Curtis J. Perry, William Damsky, J. Ishizuka
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 We aim to identify a genomic driver leading to overexpression of benign hamartomatous growths seen in the excessive thoracic angiomas of breast cancer patients. In our practice, we have observed that 4% of Caucasian postmenopausal and 1% of premenopausal breast cancer patients demonstrate large numbers of cutaneous angiomas, largely on the anterior trunk. Genetic analysis of “cherry angiomas” have found that they frequently carry specific somatic missense mutations in the GNAQ and GNA11 (Q209H) genes. Tumorigenesis has been correlated with abnormal expression and receptors, and associated G proteins (GNAS, GNAQ, or GNA11). Oncogenic mutations in G protein coupled receptors and guanine protein encoding genes have been identified in a significant number of tumors such as uveal melanoma and other malignancies. If a common genomic driver is identified in these breast cancer patients with excessive thoracic angiomas, there may be targeted treatment available for this genetic tumor subset in the future. To test the hypothesis that a driver for increased number of angiomas may also be a driver for developing breast cancer, ten breast cancer patients with 100 or more thoracic angiomas were consented to have somatic whole exome sequencing of their archival breast cancer tissue through CARIS Life Sciences. At the time of abstract submission, the batch of 10 patients’ tumor samples have been sent to CARIS for sequencing. Once the results are received, we will include further specific information on distribution of angiomas in Caucasian women in pre-and post-menopausal breast cancer patients.
 Citation Format: Rachel Geiser, Lorinda Baker, Peaches Ulrich, Albert Wendt. Genomic Investigation of Angioma in Mammary Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-04-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac67f33d671dc7d498dce88340e0014410e65cc5" target='_blank'>
              Abstract P5-04-23: Genomic Investigation of Angioma in Mammary Cancer
              </a>
            </td>
          <td>
            Rachel Geiser, Lorinda Baker, Peaches Ulrich, Albert Wendt
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background and Objectives Cancer continues to be a predominant cause of mortality worldwide, underscoring the critical need to identify and develop novel biomarkers to improve prognostic accuracy and therapeutic approaches. The dysregulation of ELAVL1 is linked to various diseases, including cancer. Nevertheless, its role across different cancer types remains insufficiently investigated. Methods We conducted a systematic investigation into the expression patterns, prognostic significance, genomic alterations, modifications, and functional implications of ELAVL1 in pan-cancer types. Besides, we performed in vitro and in vivo experiments to confirm the role of ELAVL1 in nasopharyngeal carcinoma (NPC). Results By utilizing multi-omics datasets, we found obvious overexpression of ELAVL1 in various cancer types at both the mRNA and protein levels, with predominant expression in malignant cells. Survival analysis revealed that increased ELAVL1 expression was linked to unfavorable outcomes in certain cancers; however, its effect difers among various cancer types. Additionally, we found that the genomic alterations and modifications of ELAVL1 were related to tumor progression. We discovered that ELAVL1 was elevated in NPC tissues. In addition, survival analysis indicated that NPC patients with higher ELAVL1 expression had worse prognoses. Functional assays demonstrated that ELAVL1 suppression led to decreased proliferation and migration in NPC cell lines. Moreover, ELAVL1 knockdown effectively inhibited NPC progression in the lymph node and lung metastasis models. Conclusions In summary, ELAVL1 exhibits diverse and complex involvement in tumor progression. Targeting it might inhibit tumor progression, making it a promising biomarker and therapeutic target for enhancing cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3845989ed113cdceeda02fbe18eb5b7d67ac9" target='_blank'>
              Deciphering the role of ELAVL1: Insights from pan-cancer multiomics analyses with emphasis on nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Jindong Xie, Yi Xie, Wencheng Tan, Yimeng Ye, Xueqi Ou, Xiong Zou, Zhiqing He, Jiarong Wu, X. Deng, Hailin Tang, Longjun He, Kailai Li, Peng Luo, Kunhao Bai, Guoxian Huang, Jianjun Li
          </td>
          <td>2025-04-01</td>
          <td>Journal of Translational Internal Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>