<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-22 09:30:59 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>79</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>356</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>313</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction, whereas we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA. Significance: ecDNA is an essential factor in cancer progression. We found that a group of cancer cells with ecDNA is selectively depleted after lentiviral infection. This finding provides promise for ecDNA-specific targeting, suggests the need for caution in using lentivirus, and offers alternative ways to study ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective Depletion of Cancer Cells with Extrachromosomal DNA via Lentiviral Infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement, that we applied to both bulk DNA and to ecDNA isolated with FINE (Fluorescence-activated cell sorting-based Isolation of Native ecDNA), a new method for isolating, chromatinized ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec71fc330fef12833948b0057870ba85688ca5b1" target='_blank'>
              ecDNA replication is disorganized and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J Jaworski, Pauline L Pfuderer, Pawel Czyz, Gianluca Petris, M. Boemo, Julian E. Sale
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e02ec2c65d55813a962cf1f4c6e68eaaf0b28784" target='_blank'>
              Panorama of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Yang Yang, Xiaoming Zhong, William Phillips, Wei Zhao, P. Hoang, Christopher Wirth, S. Yang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, Philippe Joubert, Robert J Homer, Jian Sang, Azhar Khandekar, John P. McElderry, Thi-Van-Trinh Tran, Caleb Hartman, Mona Miraftab, Monjoy Saha, Olivia W. Lee, Sunandini Sharma, , M. Díaz-Gay, E. Edell, Ś. JacoboMartínezSantamaría, , Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, Beata wi tkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, V. Janout, D. Mates, Simona Ognjanovic, Milan Savic, M. Kontić, Y. Bossé, B. G. Rothberg, D. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Maria Pik Wong, Kin Chung Leung, Chih-Yi Chen, Chao Agnes, Hsiung, A. Pesatori, Dario Consonni, Nathaniel Rothman, Qing Lan, Martin A. Nowak, David C Wedge, Ludmil B. Alexandrov, Stephen J Chanock, Jianxin Shi, Tongwu Zhang, Lixing Yang, Marion Landi
          </td>
          <td>2025-09-12</td>
          <td>None</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Extrachromosomal DNA amplification is associated with poor cancer prognoses1. Large numbers of excised signal circles (ESCs) are produced as by-products of antigen receptor rearrangement during V(D)J recombination2,3. However, current dogma states that ESCs are progressively lost through cell division4. Here we show that ESCs replicate and persist through many cell generations and share many properties in common with circular extrachromosomal DNAs. Increased ESC copy numbers at diagnosis of B cell precursor acute lymphoblastic leukaemia were highly correlated with subsequent relapse. By taking advantage of the matching recombination footprint that is formed upon the generation of each ESC, we measured ESC persistence and replication and found increased ESC replication in patients who later relapsed. This increased replication is controlled by cell-intrinsic factors and corresponds to increased expression of DNA replication- and repair-associated genes. Consistent with high ESC levels having a role in disease progression, the number of mutations typical of those caused by the V(D)J recombinase–ESC complex was significantly increased at diagnosis in patients who later relapsed. The number of such mutations in genes associated with relapse increased between diagnosis and relapse, and corresponded to clonal expansion of cells with high ESC copy numbers. These data demonstrate that the by-product of V(D)J recombination, when increased in abundance, potently associates with the V(D)J recombinase to cause adverse disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Oncogenes amplified on extrachromosomal DNA (ecDNA) contribute to treatment resistance and poor survival across cancers. Currently, the spatiotemporal evolution of ecDNA remains poorly understood. In this study, we integrate computational modeling with samples from 94 treatment-naive human glioblastomas (GBM) to investigate the spatiotemporal evolution of ecDNA. We observe oncogene-specific patterns of ecDNA spatial heterogeneity, emerging from random ecDNA segregation and differing fitness advantages. Unlike PDGFRA-ecDNAs, EGFR-ecDNAs often accumulate prior to clonal expansions, conferring strong fitness advantages and reaching high abundances. In corroboration, we observe pretumor ecDNA accumulation in vivo in genetically engineered mouse neural stem cells. Variant and wild-type EGFR-ecDNAs often coexist in GBM. Those variant EGFR-ecDNAs, most commonly EGFRvIII-ecDNA, always derive from preexisting wild-type EGFR-ecDNAs, occur early, and reach high abundance. Our results suggest that the ecDNA oncogenic makeup determines unique evolutionary trajectories. New concepts such as ecDNA clonality and heteroplasmy require a refined evolutionary interpretation of genomic data in a large subset of GBMs.


SIGNIFICANCE
We study spatial patterns of ecDNA-amplified oncogenes and their evolutionary properties in human GBM, revealing an ecDNA landscape and ecDNA oncogene-specific evolutionary histories. ecDNA accumulation can precede clonal expansion, facilitating the emergence of EGFR oncogenic variants, reframing our interpretation of genomic data in a large subset of GBMs. See related article by Korsah et al., p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin Wlodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P Ji, Nancy R. Zhang
          </td>
          <td>2025-09-15</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract The MYC proto-oncogene is crucial for neoplasia in most tumors. Overexpressed, oncogenic MYC amplifies the flux through most major processes but does not specify a unique carcinogenic pathway. This “amplifier” model suggests that MYC must exceed an expression threshold to become oncogenic. We designed a genetic test of this model, using the mouse Trp53 null mutant ( p53KO ) as a highly robust tumor generator to examine the effect of a modest change in the endogenous Myc level ( Myc +/- ). Strikingly, tumor-free survival is greatly extended in p53KO mice with haploid Myc gene-dosage, yet in the tumors that do develop (mainly hemangiosarcomas and thymic lymphomas), their Myc deficit has been invariably compensated either by increasing Myc genomic dosage (hemangiosarcomas) or expression (lymphomas). Furthermore, acutely halving the endogenous Myc gene-dosage in established tumor allografts curtails growth rates. These results indicate that even an incremental reduction of MYC activity can be salutary in cancer and that one of the major tumor suppressor functions of p53 derives from its ability to prevent MYC overexpression. Myc generates acute DNA damage by several mechanisms and accordingly, p53’s anti-Myc function may be inextricably linked to its role in genome integrity surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d1bbb94f2291cffaa63f9e01f8612f258ae8036" target='_blank'>
              A tumorigenesis threshold for endogenous Myc revealed by dosage-compensation for Myc haploinsufficiency in the absence of p53
              </a>
            </td>
          <td>
            Xiaozhong Bao, Zied Abdullaev, , Valerie Zgonc, Kris Ylaya, Stefania Pittaluga, Stephen M. Hewitt, J. M. Sedivy, Svetlana J. Pack, Susan Mackem, David Levens
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Multiple metrics for read-level DNA methylation pattern analysis have provided new insights into DNA methylation modifications. However, the performance of these metrics and their relationship with DNA methylation ratios in identifying biologically meaningful regions have remained unclear. Here, we systematically benchmarked five read-level DNA methylation metrics using whole-genome bisulfite sequencing data from 59 individuals across six healthy tissue types and six tumor types. We found that DNA methylation concurrence (MCR) effectively captured tissue-specific features independent of the DNA methylation ratios. Regions that exhibited decreased MCR (MCDRs) in tumors were significantly enriched in promoter and intergenic regions and strongly overlapped with tumor-gained chromatin accessibility sites. The further analysis of histone modifications, including H3K4me3, H3K27ac, and H3K9ac, confirmed that MCDRs marked active gene regulatory elements. Motif enrichment analysis revealed a strong preference for CTCF binding within MCDRs. Additionally, 3D genome analysis supported a model in which MCDRs, independent of DNA methylation ratios, contribute to active gene regulation by facilitating CTCF binding and long-range chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e01ae2e6ef4d74e2fae8d606fc1cf846f2e7bba" target='_blank'>
              DNA Methylation Concurrence, Independent of DNA Methylation Ratios, Is Associated with Chromatin Accessibility and 3D Genome Architecture
              </a>
            </td>
          <td>
            Guian Zhang, Yixian Yang, Dan Cui, Jia Li
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome-derived circular DNAs have different characters from linear DNAs and RNAs, which exert distinctive functions, and participate in a widespread range of physiological and pathological processes. Although, the composition, structure and genome-wide frequency of circular DNAs have not yet been illuminated systemically, circular DNAs are identified to act as unanticipated major sources of somatic rearrangements, which is an important genomic feature in tumorigenesis and represents a multihit and ongoing mutagenic process. And, this process remarkably contributes to oncogenic remodeling and oncogenes amplification through chimeric circularization and reintegration of circular DNAs into the linear genome, which is closely associated with high cancer mortality and morbidity. It has been found that circular DNAs-stimulated oncogenes amplification and overexpression implicate in diverse cytopathological processes during cancer progression, including cell proliferation, apoptosis, autophagy, epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) stiffness and angiogenesis. Herein, we comprehensively introduced circular DNAs’ formation and characteristics, and emphatically elaborated circular DNAs-induced oncogenes amplification and overexpression and their functional mechanisms regulating cancer behaviors. Lastly, some mysteries in oncogenic circular DNAs’ study were introduced, and the perspectives were also discussed in cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825b11b400d7213811736a403636f24a5a529f78" target='_blank'>
              Circular DNAs Participate in Tumorigenesis and Its Mechanisms
              </a>
            </td>
          <td>
            Yuan Tan, Faqing Tang
          </td>
          <td>2025-07-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a lethal malignancy with poor prognosis. We investigated intratumoral deoxyribonucleic acid methylation heterogeneity by analyzing 44 tumor samples and 5 normal samples from 6 cases of PDAC by using high-resolution methylation arrays. Two distinct methylation profiles were identified: T1, which is similar to normal pancreatic tissue and is associated with well-differentiated histology, and T2, which is significantly different from normal tissue and is linked to poorly differentiated morphology and squamous features. Validation using The Cancer Genome Atlas (TCGA) confirmed these profiles and revealed the association of T2 with shorter disease-free survival (p = 0.04). Differentially methylated region analysis identified the substantial hypomethylation of transcription regulation genes in T2 profiles (false discovery rate [FDR] q < 0.001). Gene set enrichment analysis with TCGA gene expression data demonstrated the upregulation of DNA repair and MYC target genes in T2 samples (FDR q < 0.001). Phylogenetic analysis with our multi-sampling dataset suggested an evolutionary trajectory from T1 to T2 profiles coinciding with aggressive phenotypes and increased genomic instability. Cases exhibited varying degrees of intratumoral heterogeneity from distinctly separated clusters to minimal differences. This comprehensive characterization of the epigenetic landscape of PDAC provides insights into tumor evolution and heterogeneity with potential implications for patient stratification and the development of epigenetic-based diagnostic and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13919-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd12eab03975f1f1da59d2699d262a5bc0933404" target='_blank'>
              Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            K. Kitahama, Yu-Jui Ho, Kaishi Satomi, Takahiro Shibayama, Kiyotaka Nagahama, K. Ohtsuka, Hiroaki Ohnishi, Yoshihiro Sakamoto, Junji Shibahara, Akimasa Hayashi
          </td>
          <td>2025-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Increasing evidence underscores the driving role of coding and non-coding variants in cancer development. Analyzing gene sets in biological processes offers deeper insights into the molecular mechanisms of carcinogenesis. Here, we developed geMER to identify candidate driver genes genome-wide by detecting mutation enrichment regions within coding and non-coding elements. We subsequently designed a pipeline to identify a core driver gene set (CDGS) that broadly promotes carcinogenesis across multiple cancers. CDGS comprising 25 genes for 25 cancers displayed instability in DNA aberrations. Variants within the TTN enrichment region may influence the folding of the I-set domain by altering local polarity or side-chain chemistry properties of amino acids, potentially disrupting its antigen-binding capacity in LUAD. Multi-omics analysis revealed that APOB emerged as a candidate oncogene in LIHC, whose genetic alterations within the enrichment region may activate key TFs, upregulate DNA methylation levels, modulate critical histone modifications, and enhance transcriptional activity in the HepG2 and A549 cell lines compared to Panc1. Additionally, CDGS mutation status was an independent prognostic factor for the pan-cancer cohort. High-risk patients tended to develop an immunosuppressive microenvironment and demonstrated a higher likelihood of responding to ICI therapy. Finally, we provided a user-friendly web interface to explore candidate driver genes using geMER (http://bio-bigdata.hrbmu.edu.cn/geMER/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe1ede7baa4480b19bb1878aa07140d59ffd789" target='_blank'>
              A core driver gene set identified based on geMER reveals its potential driver mechanism in pan-cancer
              </a>
            </td>
          <td>
            Jing Gan, Yuncong Wang, Zhuoran Shi, Haoyu Hu, Manyi Xu, Xinrong Li, Wenbo Dong, Jiaheng He, Yusen Zhao, Yakun Zhang, Yue Sun, Caiyu Zhang, Qianyi Lu, Shangwei Ning, Yan Jin, Hui Zhi
          </td>
          <td>2025-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of both variants of each pair were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a prophage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variants
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acral melanoma (AM) is an aggressive melanoma subtype with limited therapeutic options and poor outcomes. In non-European descent and admixed populations, like those residing in Latin America, AM accounts for a significant proportion of cutaneous melanoma cases. Here, we performed comprehensive genomic and functional profiling of AM from a uniquely diverse Brazilian cohort. Whole-exome and transcriptome sequencing revealed low mutation burden and predominance of copy number alterations, including high-amplitude focal amplifications termed hailstorms. These hailstorms frequently affected chromosomes 11, 5 and 22 and key oncogenes such as CCND1, GAB2, CDK4, and TERT. The presence of hailstorms in the long arms of chromosomes 11 and 22 was associated with higher focal copy number burden and loss of DNA damage response genes (ATM, CHEK1), suggesting a permissive genomic environment driving structural instability. To explore the unique genomic context of AM, we established a comprehensive collection of patient-derived xenograft (AM-PDX) models that faithfully retain the histopathological and genomic features of the original tumours. Functional exploration of AM-specific vulnerabilities through pharmacological and CRISPR/Cas9 knockout screenings identified strong sensitivity to targeting MAPK, CDK4/6, MDM2, and WEE1 pathways. Notably, the pan-RAS(ON) inhibitor RMC-7977 effectively reduced viability in NRAS-, KRAS-, and KIT-mutant AM cell lines. Finally, CRISPR screens revealed AM-enriched dependencies, including CRKL and SF3B4, highlighting previously unrecognized vulnerabilities. Our findings emphasize the distinct biology of AM compared to other subtypes of melanoma, provide a valuable resource of models reflective of Latin American ancestry, and identify potential drivers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0407eee77f76a83d31f10dc4bc4ede91f52ba5a" target='_blank'>
              Modelling Acral Melanoma in Admixed Brazilians Uncovers Genomic Drivers and Targetable Pathways
              </a>
            </td>
          <td>
            Annie Cristhine Moraes Sousa-Squiavinato, Sara Santos Bernardes, Flavia C. Aguiar, Antonio C. Facciolo, Martín del Castillo Velasco Herrera, J. R. C. Wong-Ramirez, P. Basurto-Lozada, Aretha Brito Nobre, Geethanjali Annamalai, Rebecca Martins Cadimo do Nascimento, Jacqueline M Boccacino, Rafaela Fagundes, Pedro Sodré do R. Barros, Mariana de Moraes Pitombo, R. Olvera-León, Larissa Satiko Alcantara Sekimoto Matsuyama, J. Billington, I. Vermes, Irving Simonin-Wilmer, D. G. Carvalho, João Pedro Cavalcante Simoes, Priscila Valverde Fernandes, Luiz Fernando Nunes, Andreia Cristina de Melo, Jadivan Leite de Oliveira, M. Herlyn, Andrew E. Aplin, C. D. Robles-Espinoza, David J. Adams, P. Possik
          </td>
          <td>2025-08-12</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d343e029068c438cb11fe609f9a8f46f24c108" target='_blank'>
              Multi-Layered Dysregulation of NRCAM in Gliomas: Insights from TCGA Copy Number and Epigenetic Analyses
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter H. Rheinstein
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mammalian genomes are subdivided into euchromatic A compartments that contain mostly active chromatin, and inactive, heterochromatic B compartments. However, it is not well understood how A and B genome compartments are established and maintained. Here we study SMCHD1, an SMC-like protein best known for its role in X chromosome inactivation, in human male myoblasts. SMCHD1 colocalizes with Lamin B1 and the heterochromatin mark H3K9me3. Loss of SMCHD1 leads to extensive heterochromatin and Lamin B1 depletion at the nuclear lamina, acquisition of active chromatin states and increased DNA methylation along chromosomes. In absence of SMCHD1, long range intra-chromosomal contacts between B compartments are lost while many new TADs and loops are formed. Inactivation of SMCHD1 promotes numerous B to A compartment transitions accompanied by activation of silenced genes. The data suggests that SMCHD1 functions as an anchor for heterochromatin domains at the nuclear lamina ensuring that these domains are poorly accessible to DNA methyltransferases and to epigenome modification enzymes that typically operate in active chromatin. Thus, the properties of SMCHD1 in heterochromatin maintenance extend well beyond its role in X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d576a82dc3af4516675e7e1ac94325c2cc7c05" target='_blank'>
              SMCHD1 maintains heterochromatin, genome compartments and epigenome landscape in human myoblasts
              </a>
            </td>
          <td>
            Zhijun Huang, Wei Cui, Ishara Ratnayake, Kristin Gallik, Lorna Cohen, R. Tawil, Gerd P Pfeifer
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke-positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single-cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low-pass whole-genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke-endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability-associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand.
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84b7fddece1de57c4f545b96bbc40fa5c54a7ad7" target='_blank'>
              Enhancer regulation in cancer: from epigenetics to m6A RNA modification.
              </a>
            </td>
          <td>
            Arathy Vasukutty, Poshan Yugal Bhattarai, Hong Seok Choi
          </td>
          <td>2025-08-19</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e777b5adbe2904319c350a6fa32537ac2efa7f" target='_blank'>
              Expansion of satellite DNAs derived from transposable elements in beetles with reduced diploid numbers.
              </a>
            </td>
          <td>
            José M. Rico-Porras, P. Mora, Ana E. Gasparotto, V. B. Bardella, T. Palomeque, Pedro Lorite, D. C. Cabral-de-Mello
          </td>
          <td>2025-08-24</td>
          <td>Heredity</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/952548d2d250db0e8e23c4ea5a20ae1aba4ce302" target='_blank'>
              Coregulators determine androgen receptor activity in prostate cancer.
              </a>
            </td>
          <td>
            Kerim Yavuz, Nathan A. Lack
          </td>
          <td>2025-08-20</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation – DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b0e24503d3afcd35a7b4c4c2e06e9d2060436" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f0d40328372c75f3f03439d0b2ad59f2263c92" target='_blank'>
              Transcriptional regulation of Ligase IV by an intronic regulatory element directs thymocyte development.
              </a>
            </td>
          <td>
            Matthew D Estrada, Christopher J Gebhardt, Mariam A Salem, Christina N Rau, Kruthika Sharma, Rebecca A Glynn, Craig H Bassing, Eugene M Oltz, Patrick L. Collins
          </td>
          <td>2025-09-05</td>
          <td>Genes and immunity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Non-Coding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions.
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54525feaa0ad7b16a4eefd8a46c2d6fd3ccc01e5" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of Single-Nucleotide Variant and Copy Number Aberration Evolution from Single-Cell DNA Sequencing of Tumors.
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2025-09-15</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma is the most common extracranial solid tumors in children, and patients with high-risk disease have poor survival despite intensive multimodal therapies. MYCN amplification is a defining feature of high-risk neuroblastoma and drives a transcriptional program that maintains an undifferentiated and proliferative tumor state. Although MYCN is a well-established oncogenic driver, it remains a difficult therapeutic target. In this study, we investigated the role of NIPBL, a cohesin loading factor, in regulating MYCN-driven transcriptional programs. We found that elevated NIPBL expression is associated with poorly differentiated tumor phenotypes and worse clinical outcomes. Importantly, NIPBL depletion led to downregulation of MYCN expression and induced transcriptional reprogramming consistent with neuronal differentiation. These findings suggest that NIPBL sustains the oncogenic activity of MYCN and may serve as a tractable therapeutic vulnerability in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876bbc7066492a73f26bd266ecac38324c99091c" target='_blank'>
              Cohesin Loading Factor NIPBL Is Essential for MYCN Expression and MYCN-Driven Oncogenic Transcription in Neuroblastoma
              </a>
            </td>
          <td>
            Jeeyoun Kang, Kaitlyn Tremble, Philip Homan, Carol J Thiele
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Significance Modified cytosine bases are essential components of mammalian genomes, but how they are distributed in double-stranded DNA remains unclear. Here, we report a method to simultaneously sequence the two most abundant modified bases—5-methylcytosine and 5-hydroxymethylcytosine (hmC)—across both strands of genomic DNA. Applying this method to DNA from mouse embryonic stem cells, we find that at CpG sites, hmC forms different combinations with cytosine variants across the DNA double helix, exhibiting unique distributions at enhancers and gene bodies, and distinct relationships with transcription. Our findings demonstrate that the double-stranded context of hmC is a key feature of the epigenome, revealing a further layer of regulatory information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8822428faae43abf3590008b224a007bfe559580" target='_blank'>
              SCoTCH-seq reveals that 5-hydroxymethylcytosine encodes regulatory information across DNA strands
              </a>
            </td>
          <td>
            Jack S Hardwick, Somdutta Dhir, Angie Kirchner, A. Simeone, Sean M. Flynn, James M. Edgerton, Rafael de Cesaris Araujo Tavares, Isabel Esain-Garcia, D. Tannahill, Pauline Y. Golder, Jack M Monahan, Walraj S. Gosal, Shankar Balasubramanian
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The DNA-incorporating nucleoside analogs azacytidine (AZA) and decitabine (DEC) have clinical efficacy in blood cancers, yet the precise mechanism by which these agents kill cancer cells has remained unresolved – specifically, whether their anti-tumor activity arises from conventional DNA damage or DNA hypomethylation via DNA methyltransferase 1 (DNMT1) inhibition. This incomplete mechanistic understanding has limited their broader therapeutic application, particularly in solid tumors, where early clinical trials showed limited efficacy. Here, through the assessment of drug sensitivity in over 600 human cancer models and comparison to a non-DNA-damaging DNMT1 inhibitor (GSK-3685032), we establish DNA hypomethylation, rather than DNA damage, as the primary killing mechanism of AZA and DEC across diverse cancer types. In further support of an epigenetic killing mechanism, CRISPR drug modifier screens identified a core set of chromatin regulators, most notably the histone deubiquitinase USP48, as AZA and DEC protective factors. We show that USP48 is recruited to newly hypomethylated CpG islands and deubiquitinates non-canonical histones, establishing USP48 as a key molecular link between the two components of epigenetic gene regulation: DNA methylation and chromatin modification. Furthermore, loss of USP48, which occurs naturally through biallelic deletions in human cancers, sensitized both hematologic and solid tumors to DNMT1 inhibition in vitro and in vivo. Our findings elucidate the epigenetic mechanism of action of AZA and DEC and identify a homeostatic link between DNA methylation and chromatin state, revealing new therapeutic opportunities for DNMT1 inhibitors in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4849479727b73a833dd0ff467a2ebd5a79ba5e8" target='_blank'>
              Discovery of chromatin-based determinants of azacytidine and decitabine anti-cancer activity
              </a>
            </td>
          <td>
            Rishi V. Puram, Qiangzong Yin, YuhJong Liu, Justine C. Rutter, Daniel Bondeson, Maria C. Saberi, Lisa Miller, Michael Du, Khanh Nguyen, Donovan L. Batzli, Hilina B. Woldemichael, Christian Taeger, Anna Goldstein, Ming Y. Chu, Qi Guo, D. Mani, Michael Naumann, M. Ronan, M. Rees, Blanche C. Ip, Mustafa Kocak, Mikołaj Słabicki, John G Doench, Jennifer A. Roth, Steven A. Carr, N. Udeshi, Jingyi Wu, Todd R. Golub
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, N. Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="This review considers fundamental issues related to the genomics of birds (Aves), including the special organization and evolution of their chromosomes. In particular, we address the capabilities of molecular genetic/genomic approaches to clarify aspects of their evolutionary history, including how they have adapted to multiple habitats. We contemplate general genomic organization, including the small size and typical number of micro/macrochromosomes. We discuss recent genome sequencing efforts and how this relates to cytogenomic studies. We consider the emergence of this unique organization ~245 million years ago, examining examples where the “norm” is not followed. We address the functional role of synteny disruptions, centromere repositioning, repetitive elements, evolutionary breakpoints, synteny blocks and the role of the unique ZW sex chromosome system. By analyzing the cytogenetic maps and chromosomal rearrangements of eight species, the possibility of successfully applying modern genomic methods/technologies to identify general and specific features of genomic organization and an in-depth understanding of the fundamental patterns of the evolution of avian genomes are demonstrated. An interpretation of the observed genomic “variadicity” and specific chromosomal rearrangements is subsequently proposed. We also present a mathematical assessment of cross-species bacterial artificial chromosome (BAC) hybridization during genomic mapping in the white-throated sparrow, a species considered a key model of avian behavior. Building on model species (e.g., chicken), avian cytogenomics now encompasses hundreds of genomes across nearly all families, revealing remarkable genomic conservation with many dynamic aspects. Combining classical cytogenetics, high-throughput sequencing and emerging technologies provides increasingly detailed insights into the structure, function and evolutionary organization of these remarkable genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d522936fb056b50f35fb472ce8bf96b8ff9560" target='_blank'>
              Avian Cytogenomics: Small Chromosomes, Long Evolutionary History
              </a>
            </td>
          <td>
            Darren K. Griffin, Rafael Kretschmer, D. Larkin, Kornsorn Srikulnath, Worapong Singchat, Valery G Narushin, R. O’Connor, Michael N. Romanov
          </td>
          <td>2025-08-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Comprehensive epigenomic studies in multiple myeloma (MM) that unravel the connections between major epigenetic regulators, their intertwined collaboration and the potential of combinatorial targeting remain limited. Utilizing ChIP-seq, ATAC-seq, RNA-seq, and DNA methylation (DNAme) data, we generated whole-genome chromatin annotations from normal plasma cells and MM patients, revealing epigenomic re-configuration affecting downstream genes involved in tumour growth and survival. Primary MM samples showed global DNA hypomethylation but site-specific hypermethylation was observed at transcription start sites, promoters, and enhancers. Moreover, increased deposition of H3K27me3 was observed in clinically relevant functional chromatin clusters. Combined EZH2 and DNMTs inhibition resulted in extensive epigenomic alterations activating apoptosis and cell cycle genes, leading to increased G2/M arrest and apoptosis in MM cell lines. Our findings provide novel insights into the role of epigenetic gene silencing in MM tumorigenesis and the interplay between the Polycomb repressive complex 2 and DNAme. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17093-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dfe4ebbad2d35b5cf5e512e8911342a3522d08" target='_blank'>
              Combinatorial DNMTs and EZH2 inhibition reprograms the H3K27me3 and DNAme-mediated onco-epigenome to suppress multiple myeloma proliferation
              </a>
            </td>
          <td>
            Alba Atienza Párraga, P. Nylund, K. Diamanti, Berta Garrido-Zabala, Stefania Iliana Tziola, Louella Vasquez, P. Pyl, D. Raykova, Aron T. Skaftason, Anqi Ma, Jian Jin, J. Martín-Subero, Fredrik Öberg, Elke De Bruyne, Jan Komorowski, Helena Jernberg Wiklund, A. Kalushkova
          </td>
          <td>2025-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic polyploid macronucleus is lost. The formation of a new macronucleus involves programmed genome rearrangements, introducing DNA DSBs at ∼45 000 loci. Paramecium tetraurelia employs a non-homologous end joining (NHEJ) mechanism for the faithful repair of these DSBs. There are four genes encoding DNA polymerases of family X in the genome, one of which was found recently to colocalize with other NHEJ proteins in the nucleus. Here we have characterized all four enzymes and shown that they are generally very faithful. They fall into two functional classes that may specialize in the distinct repair contexts encountered during DSB DNA repair. Biochemical assays, site-directed mutagenesis, and X-ray structures of mutants of human Polλ incorporating sequence determinants from P. tetraurelia PolX or metazoan Polβ are used to investigate the origin of their fidelity. Our findings suggest that Paramecium PolX enzymes may represent evolutionary intermediates between metazoan Polβ and Polλ. A general classification of DNA PolXs based on clustering methods indicates that our results can be generalized to plant DNA PolXs (Polλ-like) involved in DSB DNA repair generated by CRISPR–Cas9 engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e54e53d0ac06ac21d27fa1e914141a18179e83e" target='_blank'>
              Fidelity, specialization, and evolution of Paramecium PolX DNA polymerases involved in programmed double-strand break DNA repair
              </a>
            </td>
          <td>
            Nourisson Antonin, Missoury Sophia, Lucas-Staat Soizick, Haouz Ahmed, Delarue Marc
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract The murine endogenous retrovirus MERVL is dynamically activated in a small population of in vitro cultured mouse embryonic stem cells (mESCs) exhibiting totipotent-like features. Yet, the relationship between MERVL activation and cell fate decisions of mESCs is incompletely understood. Through a genome-wide knockout screen, we discovered that MERVL activity is intrinsically linked to DNA damage response pathways. Loss of Ints7, a backbone subunit of the Integrator complex, increased DNA damage and triggered MERVL expression. Mechanistically, Ints7 depletion induced phosphorylation of Kap1, increased chromatin accessibility at MERVL loci, and activated the p53–Dux axis to drive MERVL transcription. Intriguingly, DNA damage-induced MERVL resurgence followed the cleavage of caspase-3, often accompanying a process known as anastasis—cell survival after transient apoptotic signaling. Collectively, our study uncovered that MERVL activation in mESCs is integrated into the cellular circuit for decision-making in response to DNA damage, suggesting that sublethal caspase activation can influence the developmental potential of stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9647f46def2d9561823487f197db2f3a1327b93a" target='_blank'>
              Ints7 deficiency activates DNA damage response to elicit resurgence of endogenous retrovirus MERVL and anastasis of embryonic stem cells
              </a>
            </td>
          <td>
            Yunfan Shen, Li Li, Huiling Ni, Hui Li, Mingrui Xu, Xiaoyang Tan, Zhangjie Li, Pishun Li, Fang Chen, Song Mao, Gongping Sun, Kai Yuan
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a629089847ef194ea5479a4ede2bc91a09e2d" target='_blank'>
              Detection and monitoring of translocation renal cell carcinoma via plasma cell-free epigenomic profiling.
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ananthan Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Panday, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, Prathyusha Konda, Brad Fortunato, R. Nawfal, Razane El Hajj Chehade, Ze Zhang, Jillian O’Toole, J. Horst, Dory A Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D. Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, Srinivas R Viswanathan
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract DNA double-strand breaks (DSBs) represent critical events in genome integrity, arising from both endogenous cellular processes and exogenous factors. These breaks are implicated in various genomic aberrations and chromosomal rearrangements, leading to cancers and genetic disorders. Common and rare fragile sites, containing repetitive elements and non-B DNA structures, are particularly prone to breakage under replication stress, which play a pivotal role in cancer development and genetic diseases. Accurate quantification of DNA breaks in the context of repetitive sequences such as microsatellites or non-B DNA structures is technically challenging. We have been comparing four different methods to reliably quantify DSBs in repetitive DNA, namely Southern blot, DSB-PCR, real-time DSB-qPCR, and digital PCR (dPCR). We show here that dPCR offers enhanced sensitivity and specificity compared to other methods. This provides significant applications for future disease diagnosis, understanding molecular mechanisms generating chromosomal breakage and for the development of gene therapies for microsatellite expansion disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccab30f2a498f797c04595f2b9078ba3ca2f3603" target='_blank'>
              Fast and accurate quantification of double-strand breaks in microsatellites by digital PCR
              </a>
            </td>
          <td>
            Cécile Palao, Adèle Kovacs, M. Teixeira, G. Richard
          </td>
          <td>2025-08-09</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities all receive the same therapeutic regimen. Although several pre-clinical models of Ewing sarcoma have been established, such as cell lines and patient-derived tumor xenografts, few models exist for other SRCS. Here, we describe a pediatric SRCS tumor organoid (tumoroid) biobank containing long-term tumoroid cultures with different translocations, including EWSR1::FLI1, EWSR1::ERG, CIC::DUX4, and BCOR-rearrangements. Using histology, whole genome sequencing and RNA sequencing, we demonstrate that these tumoroids retain histological characteristics, known marker gene expression and chromosomal rearangements of their matching patient tumors. In addition, we compare mutation clusters in the tumoroids across patient-matched longitudinal samples, which shows that cellular heterogeneity is maintained. Drug screening on the tumoroid models unveils entity-specific drug sensitivity to various cytotoxic compounds and targeted compounds, including MCL-1 inhibitors for CIC::DUX4 sarcomas. Taken together, this newly established SRCS patient-derived tumoroid biobank represents a promising source of material for future basic cancer research and drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d54a3ce373321b31d8cc83feaca65eb77c05c889" target='_blank'>
              Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition
              </a>
            </td>
          <td>
            F. Ringnalda, Gijs J. F. van Son, Laurens H. G. Verweij, Seok-Young Kim, Vicky Amo-Addae, U. E. Flucke, L. Hiemcke-Jiwa, Karin P S Langenberg, Jos A. M. Bramer, L. Heimans, M. A. van de Sande, W. V. van Houdt, Max M. van Noesel, H. Kerstens, Marcel Santoso, Georg Seifert, O. Delattre, Katia Scotlandi, B. Geoerger, J. H. Merks, J. Molenaar, R. van Boxtel, Marc van de Wetering, K. Sanders, H. Clevers
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cyclin-dependent kinases (CDKs) coordinate DNA replication and cell division, and play key roles in tissue homeostasis, genome stability and cancer development. The first step in replication is origin licensing, when minichromosome maintenance (MCM) helicases are loaded onto DNA by CDC6, CDT1 and the origin recognition complex (ORC). In yeast, origin licensing starts when CDK activity plummets in G1 phase, reinforcing the view that CDKs inhibit licensing. Here we show that, in human cells, CDK4/6 activity promotes origin licensing. By combining rapid protein degradation and time-resolved EdU-sequencing, we find that CDK4/6 activity acts epistatically to CDC6 and CDT1 in G1 phase and counteracts RB pocket proteins to promote origin licensing. Therapeutic CDK4/6 inhibitors block MCM and ORC6 loading, which we exploit to trigger mitosis with unreplicated DNA in p53-deficient cells. The CDK4/6-RB axis thus links replication licensing to proliferation, which has implications for human cell fate control and cancer therapy design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1df687f2a244dc847159544edc08f4fb36a5cf" target='_blank'>
              Temporal control of human DNA replication licensing by CDK4/6-RB signalling and chemical genetics
              </a>
            </td>
          <td>
            Anastasia Sosenko Piscitello, Ann-Sofie Nilsson, Michael Hawgood, Abid H Sayyid, V. Dionellis, Giovanni Giglio, Bruno Urién, Pratikiran Bajgain, Sotirios G. Ntallis, J. Bartek, T. Halazonetis, B. Lemmens
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Over 95% of ependymomas that arise in the cortex are driven by a gene fusion involving the zinc finger translocation-associated (ZFTA) protein. Here, using super-resolution and lattice light-sheet microscopy, we demonstrate that the most frequent fusion variant, ZFTA–RELA (ZR), forms dynamic nuclear condensates that are required for oncogene expression and tumorigenesis. Mutagenesis studies of ZR reveal a key intrinsically disordered region (IDR) in RELA that governs condensate formation. Condensate-modulating IDR mutations introduced into ZR impaired its genomic occupancy at oncogenic loci and inhibited the recruitment of transcriptional effector proteins, such as MED1, BRD4 and RNA polymerase II. Using nuclear magnetic resonance spectroscopy, we examined the DNA-binding residues of the critical zinc finger (ZF1) found in ZR and characterized their significance for condensate formation, genomic binding and oncogene activation. We generated synthetic ZFTA fusion proteins where IDRs from known condensate-forming proteins were grafted into ZR. Synthetic ZFTA fusion oncoproteins utilizing IDRs from EWS and FUS restored condensate formation, oncogene transcription and tumour initiation in mice. These findings provide key insights into the oncogenic mechanism of ZR and the importance of IDR acquisition in fusion oncoproteins in brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d5d4e3905b6133ceff9f0311d9fcea6d57257e" target='_blank'>
              Synthetic ZFTA fusions pinpoint disordered protein domain acquisition as a mechanism of brain tumorigenesis
              </a>
            </td>
          <td>
            A. Arabzade, H. K. Shirnekhi, S. Varadharajan, S. M. Ippagunta, A. Phillips, N. Laboe, D. W. Baggett, Wahiduzzaman, M. Jo, T. Zheng, R. Pathak, D. Gee, D. Bhimsaria, H. Wu, X. Gao, J. Liu, E. Emanus, A. Bland, A. Kardian, A. Hancock, B. Holcomb, T. Wright, T. Bugbee, H. Sun, M. Zhai, E. Caesar, M. Park, S. Tripathi, A. Shirinifard, K. Lowe, A. Khalighifar, R. A. Petersen, S. King, D. Stabley, A. Pitre, G. Campbell, C-G Park, W. T. Freyaldenhoven, B. Chandra, Y. Xia, E. Bonten, A. Achari, S. Kandikonda, A. Carisey, S. Pounds, J. Xu, D. W. Ellison, B. Deneen, K. C. Bertrand, R. Kriwacki, S. C. Mack
          </td>
          <td>2025-08-27</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer arises from a convergence of genetic mutations and epigenetic alterations. Among the consequences of epigenetic deregulation is the aberrant activation of transposable elements, particularly retrotransposons, the most expansive and functionally impactful class. This review examines the paradoxical role of retrotransposons in cancer, emphasizing their dual capacity to support oncogenesis while simultaneously triggering innate and adaptive immune responses that may limit tumor growth. A central question is discussed: Is retrotransposon activation merely a stochastic byproduct of epigenetic instability, or could it represent an intrinsic layer of early tumor surveillance that inadvertently turns against the host as malignancy progresses? Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00368-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a2bf5f051e3bd6c0fa28e6e14cd088f5983472" target='_blank'>
              A plan or pandemonium? The conundrum of retrotransposon activation in cancer
              </a>
            </td>
          <td>
            Elina Zueva
          </td>
          <td>2025-08-09</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic p53 mutations (Onc-p53) are frequent in lung and many other solid tumors often associated with chromosome aberrations. Why cells with Onc-p53 develop chromosomal aberrations and whether the abnormalities contribute to tumor growth remain elusive. Evidence in this communication demonstrate for the first time that replication stress induced by Onc-p53 triggers re-copying of DNA replication forks, which generates replication intermediates that cause persistent mitotic aberration and DNA segregation errors. Replication intermediates from re-copied replication forks induced by Onc-p53 activate ATM signaling, which stabilizes Onc-p53, reinforces its ability to upregulate replication factors for sustaining replication stress, thus generating a feedforward cycle accelerating tumor formation. In agreement with this observation our time lapse video microscopy show in real time that persistent mitotic aberration and DNA segregation errors induced by Onc-p53 confer selective growth advantage. Accordingly, human lung tumors with Onc-p53 show selection of cells with mitotic aberration during serial passages. Knock down of active replication forks reduces re-copied fork generation by Onc-p53 and specifically induces apoptotic death of lung cancer cells expressing Onc-p53 in xenograft lung tumors synergistically in cooperation with inhibitors of ATM activation, deselecting cells with Onc-p53 with mitotic errors. This communication reveals a novel mechanism which interconnects replication stress induced by Onc-p53 to its stabilization and ability to generate chromosomal aberration in lung cancer cells that both accelerate tumor growth and serve as a targetable therapeutic vulnerability. These findings will be extremely valuable for tumor-specific treatment of a high percentage of cancer patients with p53 mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77d562f308b7a32a1524622e9f1f8039e4924f9" target='_blank'>
              Oncogenic p53 induces mitotic errors in lung cancer cells by recopying DNA replication forks conferring targetable proliferation advantage
              </a>
            </td>
          <td>
            S. Deb, Shilpa Singh, Lilia Gheghiani, R. Frum, Steven Grossman, Brad Windle, S. Deb
          </td>
          <td>2025-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Transcription factor (TF) fusion oncoproteins represent cancer-specific alterations that arise from chromosomal rearrangements. Through target gene recognition, TF fusions can disseminate transcriptional responses that collectively work to drive tumorigenesis. Thus, identifying the molecular targets that operate as a disease-driving network can potentially uncover key actionable dependencies. We have taken this strategy to dissect the underlying biological mechanism by which CIC::DUX4, a fusion oncoprotein associated with dismal outcomes, drives sarcomagenesis. We and others have defined a CIC::DUX4 fusion-mediated network that dysregulates cell-cycle and DNA replication checkpoints. Specifically, CIC::DUX4-mediated CCNE1 upregulation compromises the G1/S transition, leading to high DNA replication stress and conferring a dependence on the G2/M checkpoint kinase, WEE1. WEE1 provides a molecular brake to enable effective DNA repair prior to mitotic entry. Importantly, the mechanism by which CIC::DUX4 regulates DNA repair remains unknown. Here we show that the catalytic subunit of DNA polymerase epsilon (POLE) is essential for DNA integrity and cellular division in CIC::DUX4 sarcoma. Mechanistically, POLE loss increases DNA damage and induces p21-mediated cellular senescence to limit CIC::DUX4 tumor growth in vitro and tumor formation in vivo. Collectively, we credential POLE as a CIC::DUX4 target and further characterize a functional network through which CIC::DUX4 operates to drive tumor progression and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e39cce89c2554835e6bd04e09986a82ce912ced" target='_blank'>
              The CIC::DUX4 oncoprotein maintains DNA integrity through direct regulation of the catalytic subunit of DNA polymerase epsilon (POLE)
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-08-04</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="CNS embryonal tumors with PLAGL amplification (ET, PLAGL) are a recently described tumor type marked by amplification of one of the PLAG family genes, PLAGL1 or PLAGL2. Separately, a supratentorial, ependymoma-like CNS tumor type with PLAG family alteration, namely PLAGL1 fusion, was also reported (NET_PLAGL1). Here, we use DNA methylation profiling in combination with copy number, RNA-seq, and histological analysis to characterize and classify a novel group of CNS embryonal tumors harboring PLAG1 gene fusions (n=12). Through our screening, we identified a subset of CNS tumors (n=12) epigenetically distinct from other known CNS tumor types, but clustering close to the PLAGL1- and PLAGL2-amplified ET, PLAGL subtypes in our t-SNE analysis. Copy number profiles indicated putative PLAG1 fusions, which were confirmed in 9/12 tumors (not determined in 3/12). Different 5’ fusion partners (ASAP1, ADGRG1, TMEM68, TCF4, CHD7, NCALD, HNRNPK, LOC105378102) were identified that upregulate wild-type PLAG1 through promoter hijacking. Expression analysis shows upregulation of PLAG1 as well as IGF2, DLK1, Desmin, CYP2W1, and RET, which are also robustly expressed in PLAGL1/2-amplified tumors. Patient characteristics, survival data, and clinical/imaging analysis show additional similarities to PLAGL1/2-amplified tumors. Median age at diagnosis was 5 years, tumors were located throughout the neuroaxis, and original histological diagnoses were heterogeneous. The tumors demonstrated morphologic heterogeneity, with most composed of densely cellular areas of primitive small blue cells, alongside focal regions showing clear cell morphology, microcystic changes, and ependymoma-like perivascular pseudorosettes. Applied treatment regimens were also heterogeneous, but some favorable responses to therapy were observed. In summary, we describe a third subtype of PLAG family-altered pediatric CNS embryonal tumor characterized by PLAG1 gene fusion, which leads to upregulation of PLAG1 and downstream genes. We therefore propose to rename ET, PLAGL to ET, PLAG (CNS embryonal tumor with PLAG family gene alteration) together with a specification of the respective subtype. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02917-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48e98c986a7d6a88f7b4564b94e380215cf7f5b" target='_blank'>
              PLAG1 fusions define a third subtype of CNS embryonal tumor with PLAG family gene alteration
              </a>
            </td>
          <td>
            Michaela-Kristina Keck, M. Al-Hussaini, N. Amayiri, Akosua Adoma Boakye Yiadom, Gabriel Chamyan, E. Cheesman, Cécile Faure-Conter, Miguel Garcia-Ariza, Guillaume Gauchotte, M. Hasselblatt, Mette Jorgensen, J. Kilday, Gabriela Lamas, C. Lavarino, Marilyn M Li, Fabiana Lubieniecki, O. Maher, D. Meyronet, Jan Mueller, Mariarita Santi, U. Schüller, A. L. Seidinger, M. Sill, S. Sudhakar, María Tallón García, A. Tauziède‐Espariat, Pascale Varlet, A. Vasiljevic, A. Wittmann, Andreas von Deimling, D. A. Solomon, F. Sahm, A. Tietze, K. von Hoff, P. Sievers, David T W Jones
          </td>
          <td>2025-08-02</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Cancer genomes frequently carry APOBEC (apolipoprotein B mRNA editing catalytic polypeptide-like)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and evolution. However, the pure mutagenic impacts of the specific enzymes among this family remain unclear in human normal cell lineages. Here, we investigated the comparative mutagenic activities of APOBEC3A and APOBEC3B, through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC expression cassettes. Our findings demonstrated that transcriptional upregulation of APOBEC3A led to the acquisition of a massive number of genomic mutations in just a few cell cycles. By contrast, despite clear deaminase activity and DNA damage, APOBEC3B upregulation did not generate a significant increase in mutations in the gastric epithelium. APOBEC3B-associated mutagenesis remained minimal even in the context of TP53 inactivation. Further analysis of the mutational landscape following APOBEC3A upregulation revealed a detailed spectrum of APOBEC3A-associated mutations, including indels, primarily 1 bp deletions, clustered mutations, and evidence of selective pressures acting on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ca9c435f5c5a8910902a03b815334d801814d13" target='_blank'>
              APOBEC3A drives deaminase mutagenesis in human gastric epithelium.
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Thomas M. Klompstra, Haeun Lee, Jinju Han, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2025-08-26</td>
          <td>Genome research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Repetitive DNA sequences are abundant in genomes and can adopt alternative DNA structures (i.e., non-B DNA). One such structure, Z-DNA, has been shown to stimulate genetic instability in a variety of organisms, including human cells and mice. Z-DNA-forming sequences are enriched at mutation hotspots in human cancer genomes, implicating them in cancer etiology. Aging is a known risk factor for the development of cancer, and genetic instability is a hallmark of both aging and cancer. However, how aging affects the mutagenic potential of Z-DNA has not yet been investigated. Here, we explored the effects of aging on the mutagenic processing of Z-DNA using a transgenic mouse model. Surprisingly, Z-DNA-induced mutations decreased or remained unchanged with increasing age. Cleavage of Z-DNA was unaffected with increasing age, suggesting that downstream repair processing, such as double-strand break repair processes, may be involved in the age-related changes in Z-DNA-induced mutagenesis in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d857eba11e4fc492ce4ba5d13294865a2445fdd6" target='_blank'>
              Effects of Aging on Z-DNA-Induced Genetic Instability In Vivo
              </a>
            </td>
          <td>
            Tonia T. Li, Alexandra M D'Amico, Laura A. Christensen, Karen M. Vasquez
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Somatic mutations are desirable targets for selective cancer elimination, yet most are found within noncoding regions. We have adapted CRISPR-Cas9 as a novel, cancer-specific killing strategy by targeting the subset of somatic mutations that create protospacer adjacent motifs (PAMs), which have evolutionally allowed bacterial cells to distinguish between self and non-self DNA for Cas9-induced double strand breaks (DSBs). We hypothesized that somatic 5’-NGG-3’ PAMs arising from base substitutions (e.g. NCG→NGG) could serve as CRISPR-Cas9 targets for selective cancer elimination through the induction of multiple DSBs. Whole genome sequencing (WGS) of paired tumor-normal (T-N) samples from three PC patients were put through PAMfinder (software for somatic PAM discovery). On average, 417 somatic PAMs per PC case were found (>99% of them in non-exons) and 97% of the selected subset were Sanger confirmed. We then analysed T-N samples from ICGC which revealed medians of 455 somatic PAMs per tumor in two PC cohorts (N=174 combined), 2790 in lung (N=29), and 3236 in esophageal (N=388) cancer cohorts. We performed γH2AX staining on PC cell lines transduced with 4-9 sgRNAs designed from our PAM discovery approach, and observed more foci with increased number of sgRNAs in the target cell line but not in the non-target cell line. Deep sequencing and WGS on clones transduced with cancer-specific sgRNAs also demonstrated on-target activity in the target cell line but not in either the patient’s normal cells or an irrelevant cancer cell line. No off-target events were detected. To test selective toxicity in vitro and in vivo, we transduced 4-9 sgRNAs into PC co-cultures and found 69-99% selective cell death of three target cell lines through flow cytometry. Finally, using mouse xenograft models, 92% weight reduction (N=10, P=0.0004) of tumors transduced with a pool of 9 cancer-specific sgRNAs was achieved compared to tumors transduced with non-targeting sgRNAs (N=8).



 Selina Shiqing K. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, Michael Goldstein, Nicholas J. Roberts, and James R. Eshleman. CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b50e15f1763d831356c9974d95d4708f8806a3e" target='_blank'>
              Abstract LT02: CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers
              </a>
            </td>
          <td>
            S. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, M. Goldstein, Nicholas J. Roberts, James R. Eshleman
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-12</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Gastric cancer (GC) is classified into four molecular subtypes according to the Epstein-Barr virus-positive status, microsatellite instability, genomic stability and chromosomal instability (CIN). The CIN subtype is characterized by a high frequency of gene amplifications in receptor tyrosine kinases (RTKs) and a poor prognosis. In addition, the CIN subtype often exhibits intratumoral heterogeneity and indicates insensitivity to targeted drugs. Elucidating the molecular mechanisms of CIN in GC is therapeutically crucial; however, the molecular mechanisms involved are not yet fully understood. The kinetochore-associated 1 (KNTC1) gene encodes kinetochore-associated protein 1 (KNTC1), a major component of the outer kinetochore. The downregulation of KNTC1 causes a high frequency of lagging chromosomes and consequent aneuploidy and CIN in Drosophila and Caenorhabditis elegans. However, the association between KNTC1 and CIN in GC has not yet been clarified. Therefore, the present study investigated the role of KNTC1 in GC CIN. It was found that GC cell lines with a high frequency of lagging chromosomes had a low KNTC1 mRNA expression. Notably, KNTC1 knockdown increased the frequency of lagging chromosomes in GC cell lines. In particular, GC cell lines with the amplification of RTK genes exhibited a significant increase in the frequency of lagging chromosomes. On the whole, the findings of the present study suggest that the suppression of KNTC1 expression may contribute to CIN in GC and may be involved in the generation of intratumoral genetic heterogeneity in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4f34e8d31ebf5a0db6caf271a09a3caf4a8164" target='_blank'>
              Downregulation of kinetochore-associated 1 gene increases lagging chromosomes and contributes to chromosomal instability in gastric cancer cells
              </a>
            </td>
          <td>
            Daiki Ohsaki, Kazuki Kanayama
          </td>
          <td>2025-08-04</td>
          <td>Medicine International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Optical genome mapping (OGM) has recently emerged as a new technology in the clinical cytogenomics laboratories. This methodology has the ability to detect balanced and unbalanced structural rearrangements using ultra-high molecular weight DNA. This article discusses the uses of this new technology in both constitutional and somatic settings, its advantages as well as opportunity for improvements. Methods: We reviewed the medical and scientific literature for methodology and current clinical uses of OGM. Results: OGM is a recent addition to the methods used in cytogenomics laboratories and can detect a wide range of structural and copy number variations across a plethora of diseases. Conclusions: Clinical cytogenomics is an important laboratory specialty for which various technologies have been validated over the last several decades to improve detection of copy number and structural variations and their association to human disease. OGM has proven to be a powerful tool in the arsenal of clinical laboratories and provides a unified workflow for the detection of chromosomal aberrations across a wide range of diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/415f99beee1f8b4d54a0090a8bf5feba2107610f" target='_blank'>
              Optical Genome Mapping: A New Tool for Cytogenomic Analysis
              </a>
            </td>
          <td>
            Brynn Levy, Rachel D. Burnside, Y. Akkari
          </td>
          <td>2025-07-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The majority of life-threatening cancers arise from epithelial tissues. These epithelial cancers include cutaneous squamous cell carcinoma (cSCC), the second-most common cancer. cSCC is highly invasive and accounts for an estimated 15,000 deaths each year. We identified SRCAP, a chromatin remodeler that regulates the chromatin occupancy of the histone H2A variant H2A.Z, as a frequently mutated gene in cSCC. Analysis of cSCC mutations in epithelial cancers identified a hotspot truncating mutation in SRCAP, which removes 42% of the protein sequences after amino acid 1879. While SRCAP mutations have been previously connected to the pathogenesis of Floating-Harbor syndrome (FHS), these typically occur downstream, with a hotspot mutation leading to protein truncation after amino acid 2444. We found that expressing the SRCAP-1879 truncation in an HRas-CDK4-driven cSCC model was sufficient to increase proliferation, impair terminal differentiation, and accelerate invasion. Mechanistically, the expression of SRCAP-1879 in primary human keratinocytes was sufficient to dysregulate genes crucial for carcinogenesis (e.g., proliferation, differentiation, and motility) without altering H2A.Z occupancy. In particular, the expression of SRCAP-1879 truncation led to strong induction of the matrix metalloproteinase MMP9 expression level, accompanied by increased keratinocyte cell motility, which was sensitive to matrix metalloprotease inhibition. In contrast, the expression of the SRCAP-FHS truncation did not increase but instead reduced cell motility as well as the expression of MMP9. Taken together, our findings identify a previously under-characterized role of the SRCAP-1879 truncating mutation in promoting multiple aspects of epithelial cancer progression, including invasion, distinct from the well-recognized roles of SRCAP mutations in FHS pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b5e5845afc4ca4f1754b4e5d0d5637c1e90c8a" target='_blank'>
              A dominant SRCAP truncating mutation promotes squamous cell carcinoma progression
              </a>
            </td>
          <td>
            S. Droll, Elena I. O. Dewar, Celia Xue, Maxwell C Levine, Benny J Zhang, Xiaomin Bao
          </td>
          <td>2025-08-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Centromeres are crucial for precise chromosome segregation and maintaining genome stability during cell division. However, their evolutionary dynamics, particularly in polyploid organisms with complex genomic architectures, remain largely enigmatic. Allopolyploid wheat, with its well-defined hierarchical ploidy series and recent polyploidization history, serves as an excellent model to explore centromere evolution. Results In this study, we perform a systematic comparative analysis of centromeres in common wheat and its corresponding ancestral species, utilizing the latest comprehensive reference genome assembly available. Our findings reveal that wheat centromeres predominantly consist of five types of centromeric-specific retrotransposon elements (CRWs), with CRW1 and CRW2 being the most prevalent. We identify distinct evolutionary trajectories in the functional centromeres of each subgenome, characterized by variations in copy number, insertion age, and CRW composition. By utilizing CENH3-ChIP data across various ploidy levels, we uncover a series of CRW invasion events that have shaped the evolution of AA subgenome centromeres. Conversely, the evolutionary process of the DD subgenome centromeres involves their expansion from diploid to hexaploid wheat, facilitating adaptation to a larger genomic context. Integration of complete einkorn centromere assemblies and Aegilops tauschii pan-genomes further revealed subgenome-specific centromere evolutionary trajectories. By inclusion of synthetic hexaploid from S2-S3 generations, alongside 2x/6 × natural accessions, we demonstrate that DD subgenome centromere expansion represents a gradual evolutionary process rather than an immediate response to polyploidization. Conclusions Our study provides a comprehensive landscape of centromere adaptation, evolution, and maturation, along with insights into how retrotransposon invasions drive centromere evolution in polyploid wheat. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03759-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f98dee703cbbec473e46d88966ab2318a6f43c" target='_blank'>
              Distinct evolutionary trajectories of subgenomic centromeres in polyploid wheat
              </a>
            </td>
          <td>
            Yuhong Huang, Yang Liu, Chang Liu, Congyang Yi, Jinsheng Lai, Hongqing Ling, Handong Su, Fangpu Han
          </td>
          <td>2025-09-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogene targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance.1 Therapeutic approaches to treat tumours which have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms.2,3 Rather than treating resistance after it emerges, it may possible to prevent it by inhibiting the adaptive processes which initiate resistance but these are poorly understood.4 Here we report that residual cancer persister cells that survive oncogene targeted therapy are growth arrested by drug stress-induced intrinsic Type I interferon (IFN) signaling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA Fragmentation Factor B (DFFB, also known as Caspase-Activated DNase (CAD)) which induces DNA damage, mutagenesis, and stress response factor Activating Transcription Factor 3 (ATF3). ATF3 limits Activator Protein-1 (AP1)-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DFFB or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes regrowth of residual cancer cells that survive drug treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21f810637300567b18dd70e40b7019befcd5ee94" target='_blank'>
              DFFB suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X. Wang, B.E. Mauch, Rhea Plawat, Ariel H Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J Hangauer
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-invasive urine tests for bladder cancer (BC) could reduce dependence on flexible cystoscopy for diagnosis and surveillance. Most recent developments in urine testing are based on targeted detection of genomic and/or epigenomic markers. We hypothesised that long-read whole-genome sequencing of urinary DNA with direct methylation profiling may allow accurate BC detection and insights into disease biology. However, the feasibility of such an approach has not yet been reported. Methods We applied long-read whole-genome sequencing with direct methylation detection to urine cell pellet DNA (ucpDNA) from 21 haematuria clinic patients: 13 BCs and 8 non-BCs. The modkit Hidden Markov Model algorithm was used to define differentially methylated regions across the genome. The ability to discriminate between BC and non-BC, and the cellular pathways affected were tested using PCA, h-clust and GSEA. Results We observed global hypomethylation and cancer-specific patterns of promoter hypermethylation in urine from BC patients. Sequencing of a single ucpDNA sample per flow cell yielded read depths of 18-34x; furthermore, BC methylation patterns were also evident with 2–5x multiplex sequencing. Copy number changes were also evident in ucpDNAs from BC patients. A limitation of the study is the small number of samples analysed; however, the detection of cancer-specific events demonstrates the feasibility of the approach, both in single and multiplexed flow-cell runs. Conclusions Even at low-read depths, genome-wide methylation patterns in urinary DNA reflect the presence of BC, potentially permitting rapid, non-invasive and cost-effective BC detection. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01946-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390cf4d4c35b389c0ed48f2ce2f3090aabaa5c3" target='_blank'>
              Detection of genome-wide methylation changes in bladder cancer by long-read sequencing of urinary DNA
              </a>
            </td>
          <td>
            A. Goel, Benjamin J. Tura, Joanne D. Stockton, Nicholas Tovey, Luke Ames, Andrew D. Beggs, M. P. Zeegers, Nicholas D. James, K. Cheng, Richard T. Bryan, D. G. Ward, R. Arnold
          </td>
          <td>2025-08-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 DNA repair and autophagy are distinct biological processes vital for cell survival. Although autophagy helps maintain genome stability, there is no evidence of its direct role in the repair of DNA lesions. We discovered that lysosomes process Topoisomerase 1-cleavage complexes (TOP1cc) DNA lesions in human cancer cell. TOP1 resolves DNA topological stress ahead of DNA replication and transcription. If not promptly repaired, TOP1cc hinder the progression of DNA replication and transcription fostering genomic instability. Thus, stabilising TOP1cc with TOP1 poisons, such as Camptothecin or its analogue, is highly effective and widely used in cancer therapy. Proteolysis of TOP1cc by the proteasome is described at excessively high doses of CPT that are not relevant in a clinical setting, calling into question the relevance of the proteasome role in TOP1cc repair for cancer therapy and patient response. Here we described that repair of TOP1cc induced by clinically relevant dose of Camptothecin depends on selective autophagy. Proteomics, sequencing, biochemistry techniques and live imaging all demonstrate the lysosomal uptake of TOP1cc upon induction of replication stress. Selective degradation of TOP1cc by autophagy directs DNA damage repair and cell survival. TOP1cc are exported from the nucleus to lysosomes through transient alteration of the nuclear envelope and independent of the proteasome. Mechanistically, the autophagy receptor TEX264 acts as a TOP1cc sensor at DNA replication forks, triggering TOP1cc processing by the p97 ATPase and mediating the delivery of TOP1cc to lysosomes in an MRE11 nuclease and ATR kinase-dependent manner. We found an evolutionarily conserved role for selective autophagy in DNA damage repair that enables cell survival, protects genome stability, and is clinically relevant for colorectal cancer patients.



 Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Antičević, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, Loïc Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, and Kristijan Ramadan. TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ada9e8b0a0e36cefc3f3e65806469a480324003" target='_blank'>
              Abstract P11: TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL
              </a>
            </td>
          <td>
            Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Anticevic, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, L. Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, Kristijan Ramadan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a type of hematological cancer that arises from uncontrolled proliferation of plasma cells. In addition to frequent genetic mutations, malignant plasma cells are characterized by alterations to the epigenome. Myeloma cells display a genome-wide loss of DNA methylation and a corresponding increase in ‘active’ chromatin modifications. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements, which can impact genome regulation. Through paired epigenome and transcriptome profiling of patient derived MM samples, we have found that loss of DNA methylation in MM genomes results in the formation of partially methylated domains that are variable across patients. This loss of DNA methylation coincides with the expression of hundreds of transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. MM samples can be stratified based on L1 activity with distinct gene expression signatures. The high L1 samples are characterized by a more proliferative, less differentiated state as well as inhibition of interferon and genome defense pathways. Several L1 promoters generate chimeric transcripts with adjacent oncogenes. We further find that KRAB-zinc finger proteins (KZFPs) that are responsible for the epigenetic silencing of L1s have abnormally low abundance in MM samples with high L1 activity. These results indicate that cell proliferation in MM is associated with a loss of KZFP expression and activation of L1 elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8c94d95f4ddee7048f28a27ac43f9a3d704355" target='_blank'>
              Methylation variability and LINE-1 activation in multiple myeloma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Y. Song, Dustin Schones
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are prevalent at promoters and superenhancers (SEs), exclude nucleosomes, and recruit transcription factors. This study sought to determine whether the nucleosome exclusion affects the recruitment of the SE marker BRD4, which typically binds to acetylated histones and facilitates SE–promoter contacts via the phase separation-dependent mechanism. Analyses of the available whole-genome data revealed that SEs with the highest G4 density were depleted of nucleosomes but not of BRD4. This led us to test the possibility of histone-independent BRD4 maintenance at G4-rich SEs. A typical SE G4 destabilized a nearby nucleosome in vitro and, unlike B-DNA, bound weakly to BRD4 bromodomains. Similar to an acetylated nucleosome, the G4 promoted phase separation in BRD4 solutions. This effect was not altered by the histone competitor JQ1. However, it was attenuated by two known G4 ligands, suggesting that they could disrupt SE–promoter communication in cells. Consistently, these ligands downregulated several genes regulated by G4-rich SE-contacting promoters more efficiently than they did SE-independent genes. Our findings underscore the significance of G4-rich SEs as transcriptional regulators and provide new insights into their organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca6b9e0a4bd9850a35faa9e07c7255af9d5b35c" target='_blank'>
              G-quadruplexes as potential traps for superenhancer marker BRD4: ligand-sensitive binding and co-separation in vitro
              </a>
            </td>
          <td>
            Iuliia Pavlova, Olga M. Ivanova, Mikhail S Iudin, Anastasiya V Surdina, Nikolay A Barinov, M. E. Bogomiakova, Sergey D Oreshkov, Z. Shenkarev, V. Severov, Dmitriy V Klinov, Victoria O Shender, A. Bogomazova, M. Lagarkova, A. Varizhuk, Vladimir B Tsvetkov
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Focal tumor irradiation, a cornerstone of cancer therapy, has been increasingly recognized for its capacity to provoke systemic immunogenic responses that extend beyond localized tumor control. Recent advances highlight DNA damage, especially DNA double-strand breaks (DSBs), as a central mediator linking radiotherapy to anti-tumor immune activation. Importantly, DNA misrepair, prevalent in cancer cells with deficient or dysregulated repair machinery, serves as a double-edged sword: while fostering tumor adaptation and genomic instability, it also fuels immune recognition through the accumulation of neoantigens, extracellular DNA release, immunogenic cell death, and the modulation of immune-related cytokines and chemokines. This review critically synthesizes the latest clinical and preclinical insights into the dynamic interplay between DNA damage, repair fidelity, and the immunogenic consequences of tumor irradiation. By focusing on the impact of DSB induction and misrepair processes, we underscore the emerging therapeutic opportunities of modulating DNA repair pathways during radiotherapy to potentiate anti-tumor immunity, particularly in synergy with immune checkpoint blockade. This article provides a comprehensive perspective on the molecular underpinnings and translational potential of harnessing irradiation-induced immunogenicity, offering a roadmap for future therapeutic strategies in radiation oncology and cancer immunotherapy. Clinical trial number Not applicable">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fbc9f0b1b1a3aac4d0995d587b7e709f21a5ad" target='_blank'>
              Tumor irradiation induced immunogenic response: the impact of DNA damage induction and misrepair
              </a>
            </td>
          <td>
            Qi Liu, Xiaoran Shi, Nilupaier Tayier, Lin Ma
          </td>
          <td>2025-08-21</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Structural maintenance of chromosomes (SMC) complexes organize genomes by extruding DNA loops, while replisomes duplicate entire chromosomes. These essential molecular machines must collide frequently in every cell cycle, yet how such collisions are resolved in vivo remains poorly understood. Taking advantage of the ability to load SMC complexes at defined sites in the Bacillus subtilis genome, we engineered head-on and head-to-tail collisions between SMC complexes and the replisome. Replisome progression was monitored by genome-wide marker frequency analysis, and SMC translocation was monitored by time-resolved ChIP-seq and Hi-C. We found that SMC complexes do not impede replisome progression. By contrast, replisomes restrict SMC translocation regardless of collision orientations. Combining experimental data with simulations, we determined that SMC complexes are blocked by the replisome and then released from the chromosome. Occasionally, SMC complexes can bypass the replisome and continue translocating. Our findings establish that the replisome is a barrier to SMC-mediated DNA-loop extrusion in vivo, with implications for processes such as chromosome segregation, DNA repair, and gene regulation that require dynamic chromosome organization in all organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686025c7ec346de8dd0c0dabfe2ed3d2aaa2a6a1" target='_blank'>
              Replisomes restrict SMC translocation in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Malate dehydrogenase 1 (MDH1), an NAD(H)-dependent isoenzyme, is a key component of the malate-aspartate shuttle (MAS). A significant association has been observed between MDH1 expression and various characteristics of the tumor microenvironment across different cancer types. Methods This study provides comprehensive pan-cancer analyses exploring the expression patterns, clinical and pathological correlations, genetic alterations, immunogenomic profiles, single-cell dynamics, alternative splicing signatures, and pharmacological sensitivities related to MDH1. Drug sensitivity profiling and molecular docking techniques have been employed to identify potential anti-cancer compounds targeting MDH1. Experiments have also been conducted to investigate the biological function of MDH1 in lung adenocarcinoma (LUAD) and to confirm the interaction between MDH1 and macrophages using immunofluorescence assays. Results MDH1 expression levels are elevated across a wide range of malignancies, and overexpression of MDH1 was consistently linked to poor prognosis in multiple cancer subtypes. Moreover, MDH1 expression shows complex correlations with various immune cell populations, particularly macrophages, and cohort analysis of both bulk and pan-cancer single-cell immunotherapy data suggest that MDH1 could serve as a predictive marker for immunotherapy responses. Moreover, knockdown of MDH1 suppresses macrophage invasion. To investigate the role of MDH1 in LUAD cells, a potential inhibitor of MDH1 was identified, BI-2536, and has been confirmed to impact MDH1 activity and impede the growth of LUAD cells. Conclusion Our findings indicate that MDH1 may serve as a potential prognostic marker and a promising target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3176f0dff692eaebf853d9454461d636ffecd8e7" target='_blank'>
              Panoramic view of MDH1: driving cancer progression and shaping the tumor immune microenvironment
              </a>
            </td>
          <td>
            Yunchen Lou, Yunwei Lou, Yao Cheng, Beining Xu, Hanbin Chen, Yinwei Dai
          </td>
          <td>2025-08-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-10</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: a comprehensive and high-quality database for extrachromosomal circular DNA.
              </a>
            </td>
          <td>
            Jin-Hao Zhang, , , Ji Shi, , Ying Zhang, , Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract R-loops or DNA–RNA hybrids are prominent nucleic acid structures that commonly arise during transcription. These structures play important biological functions, such as regulating gene expression and DNA repair. However, when unresolved by nucleic acid processing factors, pathological R-loops can be harmful and lead to genome instability. N6-Methyladenosine (m6A), the most prevalent modification in messenger RNA, has been recently identified to be crucial for regulating R-loop balance and maintaining genome stability. Strikingly, m6A-modified R-loop formation can have opposing consequences, either stabilization or resolution, depending on the biological context. In this review, we discuss the current knowledge of the regulatory roles of m6A on R-loops across various processes, including gene transcription, DNA repair, and centromere and telomere stability. Additionally, we explore other m6A-mediated processes, such as nascent transcription and chromatin landscape, that potentially affect R-loop dynamics. Finally, we discuss the current limitations and future directions of studying the m6A–R-loop axis, as well as the opportunities to target this pathway as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5d8ee286e956882ce3a5149e41ece752c38e0d" target='_blank'>
              m6A modification in R-loop homeostasis: a potential target for cancer therapeutics
              </a>
            </td>
          <td>
            Minh-Anh Vu, Manuela Spagnuolo, Chun-Long Chen
          </td>
          <td>2025-08-11</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant DNA methylation has been shown to be a fingerprint characteristic in human colorectal tumors. In this study, we hypothesize that investigating global DNA methylation could offer potential candidates for clinical application in CRC. The epigenome-wide association analysis was conducted in both the tumor area (N = 27) and the adjacent tumor-free (NAT) area (N = 15). We found 78,935 differentially methylated CpG sites (DMCs) (FDR < 0.05), 42,888 hypomethylated and 36,047 hypermethylation showing overall hypomethylation. Gene ontology and KEGG analysis of differentially methylated genes showed significant enrichment in developmental genes, as well as in genes involved in metabolic processes and the cell cycle, such as the TFGβ and cAMP signaling pathways. Through filtered analysis, we identified TNXB as the most epigenetically dysregulated gene, hypomethylated and downregulated in CRC (both with p < 0.001) and associated with poor overall survival. In the functional analysis, TNXB was epigenetically regulated in a dose-dependent manner, suggesting a potential role in CRC. The epigenetic dysregulation and functional role of TNXB in CRC could have clinical implications, serving as indicators of malignant potential, with adverse effects associated with disease origin and progression in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5735874128c08fad50b2549a2c915b47f637e3f9" target='_blank'>
              Unraveling TNXB Epigenetic Alterations Through Genome-Wide DNA Methylation Analysis and Their Implications for Colorectal Cancer
              </a>
            </td>
          <td>
            Jesús Pilo, Alejandro Rego-Calvo, L. García-Flores, I. Arranz-Salas, Ana Isabel Álvarez-Mancha, A. G. Izquierdo, A. Crujeiras, Julia Alcaide, María Ortega-Castan, H. Boughanem, M. Macías-González
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James O. J. Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="R-loops, chromatin structures containing DNA-RNA hybrids with displaced single-stranded DNA, play crucial roles in various cellular processes. Their formation is influenced by factors such as DNA topology, RNA stability, and the presence of GC-rich regions. However, excessive or uncontrolled R-loop accumulation can threaten genomic stability, leading to DNA damage, particularly double-strand breaks. To preserve genome integrity, cells have developed mechanisms to regulate R-loop formation and resolution. Dysregulation of these processes is linked to several diseases, including cancer and neurodegenerative disorders. In this review, we will explore the dynamics of R-loop formation and resolution and how they are detected, their roles in DNA damage and repair, and how their dysregulation may lead to immune responses and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3728d9fd65e0fa0dfc3340b92946a313355d5d1" target='_blank'>
              The paradox of R-loops: guardians of the genome or drivers of disease?
              </a>
            </td>
          <td>
            Shachinthaka D Dissanayaka Mudiyanselage, Phillip Wulfridge, K. Sarma
          </td>
          <td>2025-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d25046bd1017d2fa8c917d20b560de58a89044" target='_blank'>
              Effects of mating system and adaptedness on the evolution of fitness and mtDNA copy number in mitonuclear mismatched C. elegans.
              </a>
            </td>
          <td>
            Zachary P Dietz, Devshuvam Banerji, J. A. Sullins, Brent W. Bever, S. Christy, U. Bergthorsson, Vaishali Katju, Suzanne Estes
          </td>
          <td>2025-07-28</td>
          <td>Heredity</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Esophageal carcinoma (ESCA) is a common tumor in the digestive system, resulting in approximately 300,000 death worldwide every year. ALDH18A1, a potential RNA binding protein (RBP), is significantly overexpressed as in ESCA, while its functions and mechanism is unclear. In this study, we silenced ALDH18A1 in KYSE150 cells using small interfering RNA (siALDH18A1), and evaluated its effect on cell characteristics. Finally, we conducted whole transcriptome sequencing (RNA-seq) and RNA immunoprecipitation (iRIP-seq) experiments to explore the downstream targets of ALDH18A1, and verified their change patterns using quantitative polymerase chain reaction. We found that ALDH18A1 exhibited a high expression level in ESCA patients using online database. The proliferation, invasion, and migration abilities of KYSE150 cells were significantly inhibited in siALDH18A1 samples. Through the investigation of RNA-seq data, we found that ALDH18A1 globally regulates the expression and alternative splicing profiles of KYSE150 cells by identifying 437 differentially expressed genes (DEGs) and 1071 ALDH18A1-regulated AS events (RASEs), respectively. The functions of RASE genes (RASGs) were enriched in pathways such as DNA repair and gene expression. In addition, through the integration analysis of iRIP-seq data and DEGs, we found that ALDH18A1 bound to and regulated the expression of 59 DEGs, including LAMB3 and IL1B, which were verified by RT-qPCR experiment. In addition, we performed an overlap analysis between the genes bound by ALDH18A1 and RASGs and found 339 bound RASGs. The functional enrichment of these genes revealed pathways such as DNA repair, cell proliferation, and DNA damage response. Among them, we found significant alternative splicing events of MTA1, which were confirmed by PCR experiments. In summary, our study comprehensively explored the functions and downstream targets of ALDH18A1 in ESCA cells, highlighting the molecular mechanism of ALDH18A1 on the carcinogenic characteristics of ESCA. The identified molecular targets, especially DNA repair-related genes, can serve as potential therapeutic targets for future ESCA treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08006-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4696a54f3a85524e5b23e6278282dcbdfa70bd9a" target='_blank'>
              ALDH18A1 has carcinogenic functions and regulates alternative splicing events of DNA repair-related genes in esophageal carcinoma cells
              </a>
            </td>
          <td>
            Yongkang Wang, Maimaiti Yisireyili, Kelimu Abudureyimu, Atsushi Enomoto
          </td>
          <td>2025-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ionizing radiation can damage DNA, leading to mutations, and is a risk factor for cancer. Based on the assumption that all radiation exposure poses a risk in linear proportion to its dose, ionizing radiation is considered a non-threshold carcinogen. However, most epidemiological studies have had insufficient statistical power to detect excess cancer risks from low-dose radiation exposure. Therefore, research is needed to identify radiation signatures that distinguish radiation-induced cancers from spontaneously developed cancers. In rodent cancer models, interstitial chromosomal deletions of specific tumor-suppressor gene loci are characteristically found in cancers from irradiated animals. In humans, a high frequency of small deletions and chromosome rearrangements, such as large deletions, inversions, and translocations, has also been reported in second cancers that develop in patients who received radiotherapy and in thyroid cancers diagnosed in residents after the Chornobyl accident. These genomic alterations are likely to be generated as a consequence of the processing of radiation-induced DNA double-strand breaks. Particularly, chromosome rearrangements that occur at loci directly linked to tumor formation after ionizing-radiation exposure are potentially useful as biomarkers and as therapeutic targets for radiation-induced cancer. Here we provide an overview of the radiation-induced mutational signatures observed in animal and human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b074d01f6fb78ae72372c0d26a26df7f99fb224" target='_blank'>
              Fungi as models of centromere innovation: from DNA sequence to 3-dimensional arrangement.
              </a>
            </td>
          <td>
            Srijana Dutta, Krishna Bhat, Rashi Aggarwal, K. Sanyal
          </td>
          <td>2025-08-11</td>
          <td>Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Ribophorin 1 (RPN1), a key component of the oligosaccharyltransferase complex, is implicated in tumor progression through glycosylation-mediated pathways, yet its pan-cancer roles remain unexplored. This study presents a comprehensive multi-omics analysis of RPN1 across 33 cancers such as sarcoma (SARC), integrating genomic, transcriptomic, and proteomic data from TCGA, GTEx, and CPTAC. RPN1 was significantly overexpressed in 14 malignancies and correlated with advanced tumor stages and poor prognosis in glioblastoma (GBM), lower-grade glioma, SARC and hepatocellular carcinoma, validated in an independent glioma cohort (n = 151). Genomically, RPN1 amplification linked to homologous recombination deficiency and elevated tumor mutational burden, suggesting a role in genomic instability. Critically, multiplex immunofluorescence demonstrates RPN1 overexpression colocalizes with CD206+ M2 macrophages in tumor microenvironments, while in vitro coculture experiments confirm RPN1-dependent microglial recruitment and M2 polarization. RPN1 expression negatively correlates with CD8+ T cell infiltration and predicts resistance to chemotherapy (GBM, ovarian cancer) and immunotherapy (GBM, esophageal carcinoma), though it associates with PD-1 inhibitor sensitivity in bladder cancer. Functional validation shows RPN1 knockdown suppresses proliferation, migration, and invasion in GBM cells. Pathway enrichment connects RPN1 to endoplasmic reticulum stress, glycosylation, DNA repair, and immune checkpoint regulation. These findings position RPN1 as a multimodal oncogenic driver promoting genomic instability, immunosuppressive microenvironment remodeling, and context-dependent therapeutic vulnerabilities across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fad4cdee9a5624b27cf2ed12f95c27e148fc7b5" target='_blank'>
              RPN1 Is Associated With Immunosuppression in Pan-Cancer and Affects the Malignant Phenotype of Tumor.
              </a>
            </td>
          <td>
            Shaoru Lin, Changwu Wu, Qing Liu, Fushu Luo, Jun Tan
          </td>
          <td>2025-08-26</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC), an aggressive neuroendocrine tumor strongly associated with exposure to tobacco carcinogens, is characterized by early dissemination and dismal prognosis with a five-year overall survival of less than 7%. High-frequency gain-of-function mutations in oncogenes are rarely reported, and intratumor heterogeneity (ITH) remains to be determined in SCLC. Here, via multiomics analyses of 314 SCLCs, we found that the ASCL1+/MKI67+ and ASCL1+/CRIP2+ clusters accounted for 74.38% of the 190,313 SCLC cancer cells from 39 patients, with the ASCL1+SOX1+ stem-like cell cluster across SCLC subtypes. The major histocompatibility complex (MHC) class I molecules were expressed at low levels in six and high levels in five cancer cell clusters and were inversely associated with the KI67 expression level. Abnormal splicing of mRNAs was a feature of SCLC, with focal adhesion kinase (FAK) splicing variants identified in 119 (77.3%) of 154 patients. FAK variants exhibited elevated kinase activity, were associated with the worst prognosis, and were sensitive to FAK inhibitors in patient-derived organoids and xenograft models. Eleven high-frequency mutations were identified in addition to TP53 and RB1, and smoking status and tumor stage did not affect microbiota variance in SCLC. Taken together, our data further revealed the complicated ITH and discovered that FAK splicing variants represent high-frequency gain-of-function alterations in oncogene in SCLC and potential therapeutic targets for this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3bde80c19d7166cce5785be8ae3a4a5e971fbbd" target='_blank'>
              Characterization of the extrinsic and intrinsic signatures and therapeutic vulnerability of small cell lung cancers
              </a>
            </td>
          <td>
            Gui-zhen Wang, Zheng Wang, Shi-Hao Bai, Yun Tan, Wen-Zhao Zhong, Guo-Gui Sun, Yu-Tao Liu, B. Pan, Chen Huang, Di Wang, Beibei Sun, Dong-Ni Chen, Bin Zhang, Yong-Chun Zhou, Sheng Li, Xiang-Wei Zhang, Si-Chong Han, Fu-ying Yang, Xue-Yan Shi, Xiao-Liang Jie, Yu-Ke Shen, L. Liang, , Li Zhang, Mingkun Li, Na Wang, Jin-Song Liu, Ying Dong, Manli Wang, Yan Wang, Chang-Li Wang, Dawei Xie, Ze-Guang Han, Jian-Ming Ying, Chong Chen, Yunchao Huang, Hongbin Ji, Yuan-Yuan Zhang, Yan Yu, 
          </td>
          <td>2025-09-10</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is one of the most prevalent adult leukemias, derived from mature B-cells and exhibiting a highly heterogeneous disease course. Standard cytogenetic analysis of CLL includes FISH and karyotyping. However, conventional chromosome analysis of cancer specimens is often constrained by low chromosomal resolution, and FISH analysis is limited by the number of probes that can be applied. This study highlights the application of optical genome mapping (OGM), a high-resolution cytogenomic tool that visualizes ultra-long, sequence-labeled DNA molecules, to uncover the structural complexity of the cancer genome and assess the clinical relevance of chromothripsis in CLL. Comprehensive cytogenetic analysis was conducted on a 43-year-old male diagnosed with chronic lymphocytic leukemia. Karyotyping revealed a complex rearrangement: 46,XY,der(3)t(3;13)(p2?3;q14.3),der(4)t(?3;4)(p23;p16),add(11)(p13),del(13)(q14)[12]/46,sl,del(11)(q2?2.2q23.3)[6]. FISH analysis further identified the loss of ATM and a partial deletion of the D13S319 locus. OGM analysis performed on bone marrow revealed a complex genotype including chromothripsis of chromosome 13, and structural rearrangements involving chromosomes 3, 4, and 11. Additionally, multiple intrachromosomal translocations and interstitial microdeletions of chromosome 13 were identified. The resolution of these aberrations has been significantly enhanced with examples including: ogm[GRCh38] t(3;13)(p26.3;q33.1)(2,706,645~2,721,113;103,142,901~103,154,241][VAF0.45], ogm[GRCh38] t(4;13)(p15.31;q32.1)(20,869,721~20,907,265;96,617,837~96,630,317)[VAF0.42],. In conclusion, OGM revealed the intricate structural alterations of the cancer genome. The high resolution provided by OGM could facilitate the discovery of oncogenic mechanisms, novel fusion genes, prognostic markers, and potential therapeutic targets. OGM serves as a powerful tool for revisiting CLL disease classification by offering deeper insights into complex genomic rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4656aef9b3e1a0de1b2f5258ea921c03f0e3137" target='_blank'>
              Case Report: Unraveling complex genomic alterations in a case of chronic lymphocytic leukemia using optical genome mapping
              </a>
            </td>
          <td>
            Leila Youssefian, Trilochan Sahoo, Jia-Chi Wang
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C- and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e4731f01013d1812909194114a8c4a4ce7f91c" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Ashley Y Cheng, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Herpesviruses are ubiquitous pathogens that cause a wide range of disease. Upon nuclear entry, their genomes associate with histones and chromatin modifying enzymes that regulate the progression of viral transcription and outcome of infection. While the composition and modification of viral chromatin has been extensively studied on bulk populations of infected cells by chromatin immunoprecipitation, this key regulatory process remains poorly defined at single-genome resolution. Here we use high-resolution quantitative imaging to investigate the spatial proximity of canonical and variant histones at individual Herpes Simplex Virus 1 (HSV-1) genomes within the first 90 minutes of infection. We identify significant population heterogeneity in the stable enrichment and spatial proximity of canonical histones (H2A, H2B, H3.1) at viral DNA (vDNA) relative to established promyelocytic leukaemia nuclear body (PML-NB) host factors that are actively recruited to viral genomes upon nuclear entry. We show the replication-independent histone H3.3/H4 chaperone Daxx to cooperate with PML to mediate the enrichment and spatial localization of variant histone H3.3 at vDNA and limit the rate of HSV-1 genome decompaction. This host response is counteracted by the viral ubiquitin ligase ICP0, which degrades PML to disperse Daxx and variant histone H3.3 from vDNA to stimulate the progression of viral immediate-early (IE) transcription, genome expansion, and onset of HSV-1 replication. Our data support a model of intermediate and sequential histone assembly initiated by Daxx that limits the rate of HSV-1 genome decompaction independently of the stable enrichment of histones H2A and H2B at vDNA required to facilitate canonical nucleosome assembly. We identify HSV-1 genome decompaction upon nuclear infection to play a key role in the initiation and functional outcome of HSV-1 lytic infection, findings pertinent to the transcriptional regulation of many nuclear replicating herpesvirus pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617d8ae459ea016f8dca2b2644f05eeab5725c80" target='_blank'>
              Daxx mediated histone H3.3 deposition on HSV-1 DNA restricts genome decompaction and the progression of immediate-early transcription
              </a>
            </td>
          <td>
            Ashley P. E. Roberts, Anne Orr, V. Iliev, Lauren Orr, S. McFarlane, Zhousiyu Yang, I. Epifano, C. Loney, Mila Collados Rodriguez, Anna. R Cliffe, Kristen L Conn, Chris Boutell
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Cytosine DNA methylation patterns vary widely across eukaryotes, with its ancestral roles being understood to have included both transposable element (TE) silencing and host gene regulation. To further explore these claims, in this study, we reevaluate the evolutionary origins of DNA methyltransferases and characterize the roles of cytosine methylation on underexplored lineages, including the amoebozoan Acanthamoeba castellanii, the glaucophyte Cyanophora paradoxa, and the heterolobosean Naegleria gruberi. Our analysis of DNA methyltransferase evolution reveals a rich ancestral eukaryotic repertoire, with several eukaryotic lineages likely subsequently acquiring enzymes through lateral gene transfer (LGT). In the three species examined, DNA methylation is enriched on young TEs and silenced genes, suggesting an ancestral repressive function, without the transcription-linked gene body methylation of plants and animals. Consistent with this link with silencing, methylated genomic regions co-localize with heterochromatin marks, including H3K9me3 and H3K27me3. Notably, the closest homologs of many of the silenced, methylated genes in diverse eukaryotes belong to viruses, including giant viruses. Given the widespread occurrence of this pattern across diverse eukaryotic groups, we propose that cytosine methylation was a silencing mechanism originally acquired from bacterial donors, which was used to mitigate the expression of both transposable and viral elements, and that this function may persist in creating a permissive atmosphere for LGT in diverse eukaryotic lineages. These findings further highlight the importance of epigenetic information to annotate eukaryotic genomes, as it helps delimit potentially adaptive LGTs from silenced parasitic elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48579c32d97033a0fd54ac2bedb8cb3bebb700ed" target='_blank'>
              Repressive Cytosine Methylation is a Marker of Viral Gene Transfer Across Divergent Eukaryotes
              </a>
            </td>
          <td>
            Luke A. Sarre, Giselle Azucena Gastellou Peralta, Pedro Romero Charria, Vladimir Ovchinnikov, Alex de Mendoza
          </td>
          <td>2025-07-25</td>
          <td>Molecular Biology and Evolution</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cellular senescence is accompanied by extensive genomic reorganization, such as senescence-associated heterochromatin foci and expanded interchromatin compartments, to ultimately affect gene expression. Here, we demonstrate that chromatin structural changes in senescent cells drive significant alterations in the phase behavior and motility of paraspeckles, a type of interchromatin compartment condensate. We observe increased numbers, size, and elongation of paraspeckles harboring NONO and NEAT1_2, driven by elevated levels of those components, consistent with the micellization model of longitudinal growth rather than condensate coalescence. Enhanced paraspeckle motility is associated with HP1α-mediated heterochromatin condensation and interchromatin expansion found in cellular senescence. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03757-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c589595d2d429fd53faabd150c9bf96e091d470b" target='_blank'>
              Imbalanced chromatin distribution in cellular senescence specifies paraspeckle dynamics
              </a>
            </td>
          <td>
            Joonwoo Lee, Jinmi Choi, Jeongeun Park, Seonduk Lee, Eun-Jung Cho, Youngdae Gwon
          </td>
          <td>2025-09-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da656da84cf3e9dc1144dda4829e9c83aef2891" target='_blank'>
              CHAMP1 Complex Promotes Heterochromatin Assembly and Reduces Replication Stress.
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and leukemogenic function remain unclear. A major challenge has been their large size, which has impeded preclinical model development and mechanistic studies. Results We employ a domain-focused truncation strategy to generate de novo murine models of K/C and K/P fusions, which faithfully recapitulate the morphological, immunophenotypic, and transcriptomic features of KAT6A-rearranged AML. Genomic profiling reveals that KAT6A fusions preferentially localize to H3K4me2/3-marked regions, while biochemical analyses uncover that KAT6A interacts with the Nucleosome Remodeling Factor (NURF), a key H3K4me2/3 reader. Disrupting NURF-chromatin interactions via depletion or small-molecule inhibition of its subunit, Bromodomain PHD Finger Transcription Factor (BPTF), impairs K/C recruitment and disrupts MLL/COMPASS-mediated H3K4me2 deposition, defining a functional epigenetic module involving KAT6A chimeras, NURF, and MLL/COMPASS. Notably, CBP/P300 inhibition reduces histone acetylation and chromatin accessibility, further impairing the recruitment of this epigenetic module. Targeting this module via NURF or CBP/P300 inhibition demonstrates efficacy in K/C leukemia models, with enhanced therapeutic effects observed when combined. Conclusions Our study identifies a self-reinforcing epigenetic module of histone modifiers and readers in KAT6A-rearranged AML, providing mechanistic insights into the genomic targeting of KAT6A chimeras and highlighting promising combinatorial therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03743-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b1939d769bfdab61413c545354658eef418fa8" target='_blank'>
              KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML
              </a>
            </td>
          <td>
            Junhui Lv, Zhinang Yin, Conghui Li, Honglin Wen, Jian Ni, Peiyuan Yang, Zemin Song, Ying Xiang, Honghong Wang, Rui Lu, Li Huang, Ying Zhou, Hai-Bing Zhou, Ruijing Xiao, Pingping Fang, Kaiwei Liang
          </td>
          <td>2025-08-19</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND Hepatocellular carcinoma (HCC) remains a lethal malignancy due to its molecular complexity and chemoresistance. Rac family small GTPase 3 (RAC3), a tumorigenic GTPase understudied in HCC, drives recurrence via E2F transcription factor 1 (E2F1)-mediated transcriptional activation. This study integrates multiomics and clustered regularly interspaced short palindromic repeats (CRISPR) screening to delineate RAC3’s roles. RAC3 overexpression correlates with advanced HCC and patient age, while its knockout suppresses proliferation. Mechanistically, RAC3 dysregulates cell-cycle checkpoints through E2F1 binding. Pharmacological RAC3 inhibition disrupts tumor growth and synergizes with chemotherapy to overcome resistance. AIM To explore RAC3’s expression, clinical links, and HCC mechanisms via multiomics and functional genomics. METHODS Multiomic integration of The Cancer Genome Atlas (TCGA), Gene Expression Omnibus, and Genotype-Tissue Expression datasets was performed to analyze RAC3 mRNA expression. Immunohistochemistry quantified RAC3 protein in 108 HCC/adjacent tissue pairs. Kaplan–Meier/Cox regression assessed prognostic significance using TCGA data. CRISPR screening validated RAC3’s necessity for HCC proliferation. Functional enrichment identified associated pathways; hTFtarget/JASPAR predicted transcription factors, validated via chromatin immunoprecipitation sequencing (ChIP-seq). RESULTS RAC3 exhibited significant mRNA and protein overexpression in HCC tissues, which was correlated with advanced tumor stages and reduced overall survival rates (hazard ratio = 1.82, 95%CI: 1.31–2.53). Genetic ablation of RAC3 suppressed HCC cell proliferation across 16 cell lines. Pathway analysis revealed RAC3’s predominant involvement in cell-cycle regulation, DNA replication, and nucleocytoplasmic transport. Mechanistic investigations identified E2F1 as a pivotal upstream transcriptional regulator, and ChIP-seq analysis validated its direct binding to the RAC3 promoter region. These findings suggest that RAC3 drives HCC progression through E2F1-mediated cell-cycle dysregulation. CONCLUSION This study identified RAC3 as a key HCC oncogenic driver; its overexpression links to poor prognosis/resistance. Targeting the RAC3/E2F1 axis offers a new therapy, which highlights RAC3 as a biomarker/target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/702736239ad578eee490b2f86a99dd33f569093b" target='_blank'>
              Deciphering the oncogenic role of Rac family small GTPase 3 in hepatocellular carcinoma through multiomics integration
              </a>
            </td>
          <td>
            Run Liu, Jin-Cheng Li, S. Li, Jian-Di Li, Rong-Quan He, Gang Chen, Zhen-Bo Feng, Jia-Liang Wei
          </td>
          <td>2025-07-27</td>
          <td>World Journal of Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Mutations that initiate AML can cause clonal expansion without transformation ("clonal hematopoiesis"). Cooperating mutations, usually in signaling genes, are needed to cause overt disease, but these may require a specific "fitness state" to be tolerated. Here, we show that nearly all AMLs arising in a mouse model expressing two common AML initiating mutations (Dnmt3aR878H and Npm1cA) acquire a single copy amplification of chromosome 7, followed by activating mutations in signaling genes. We show that overexpression of a single gene on chromosome 7 (Gab2, which coordinates signaling pathways) is tolerated in the presence of the Npm1cA mutation, can accelerate the development of AML, and is important for survival of fully transformed AML cells. GAB2 is likewise overexpressed in many human AMLs with mutations in NPM1 and/or signaling genes, and also in Acute Promyelocytic Leukemia initiated by PML::RARA; the PML::RARA fusion protein may activate GAB2 by directly binding to its 5' flanking region. A similar pattern of GAB2 overexpression preceding mutations in signaling genes has been described in other human malignancies. GAB2 overexpression may represent an oncogene-driven adaptation that facilitates the action of signaling mutations, suggesting an important (and potentially targetable) "missing link" between the initiating and progression mutations associated with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6f3ad6142f75e1cb2d2a0d8414ccd605f8e467" target='_blank'>
              Overexpression of the signaling-coordinator GAB2 can play an important role in Acute Myeloid Leukemia progression.
              </a>
            </td>
          <td>
            Michael H. Kramer, Stephanie N Richardson, Yang Li, Tiankai Yin, Nichole M. Helton, Daniel R. George, Michelle Cai, S. M. Ramakrishnan, C. Katerndahl, Christopher A. Miller, T. Ley
          </td>
          <td>2025-08-07</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07119c6c29989e52a1007b94a87a49c0954c86a" target='_blank'>
              Phase separation of ERCC6L2-CtIP regulates the extent of DNA end resection.
              </a>
            </td>
          <td>
            Yixin Yin, Jinlong Lin, Xiaoxia Cai, Runcong Nie, Jiliang Lin, Shuting Li, Zhicheng Xiang, Yihong Ling, Yiyang Zhang, Jie Zhou, Jiewei Chen, Wenping Lin, Jinling Duan, Xueyi Zheng, Dan Xie, Mu-yan Cai
          </td>
          <td>2025-09-05</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Ovarian cancer (OV) has the highest mortality rate among gynecological cancers and shows varied responses to chemotherapy combined with PARP inhibitors based on homologous recombination deficiency (HRD) subtypes. Methods This study enrolled 143 Chinese OV patients to determine the HRD score grouping threshold using genomic features, dividing patients into HRD-high and HRD-low groups. Multi-omics sequencing was conducted on 70 patients receiving adjuvant chemotherapy with PARP inhibitors. Results In this study, TP53 mutations enriched in the HRD-high group, while ARID1A, PIK3CA, and PTEN mutations were more common in the HRD-low group. HRD-high patients exhibited stronger immune activation, including elevated STAT1 expression, HLA signatures, and increased M1 macrophage infiltration, correlating with better prognosis. Additionally, peripheral blood analysis revealed higher bMSI and maxVAF levels in HRD-low patients compared to HRD-high patients, suggesting ctDNA as a potential tool for dynamic monitoring post-treatment. Conclusions This study identified distinct molecular and immune profiles between HRD subgroups in Chinese ovarian cancer patients. Patients with HRD-high and STAT1 expression ≥ 74 suggests PARPi benefit. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00806-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5abcfeae4983ac959a97accf690879bf81c76cc2" target='_blank'>
              Genomic and transcriptomic data reveal molecular differences between homologous recombination deficiency subgroups in Chinese ovarian cancer patients
              </a>
            </td>
          <td>
            Hongxia Wang, Wenhong Zhao, Wenhao Zhou, Na Wang, Yijie Li, Kaiyun Qin, Jingde Jia, Jiaqian Wang, Congcong Song, Yu Yu, Fenghua Zhang, Xu Cui, Lanlan Zhao, Haitao Luo, Zhengmao Zhang
          </td>
          <td>2025-08-31</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Transposable elements (TEs) have hijacked cellular machineries to replicate and spread throughout host genomes. TEs now make up a significant portion of eukaryotic genomes and play notable roles in genomic evolution, driving both speciation and providing raw material for genetic innovation. Barbara McClintock's pioneering work on these “jumping genes” laid the foundation for modern TE research; however, her paradigm‐shifting theories in which TEs act as “controlling elements” were initially rejected due to the long‐held belief that TEs were “junk” or parasitic DNA elements. Historically, the highly repetitive nature of TEs made it challenging to both identify and investigate functions. However, recent advances in genomics have greatly accelerated our understanding of TEs. Despite their potential to cause insertional mutagenesis and disease, many transposable elements have been co‐opted by host genomes to contribute to gene regulation and development. In contrast to protein‐coding genes that typically begin their journey as DNA, are transcribed into RNA, and reach their ultimate functional form as proteins, TEs can function as cis‐regulatory DNA, functional RNA, and in rare cases, domesticated proteins and fusion events between TE and host genes. Driven by rapidly advancing technologies, the roles of TEs in both development and disease are being uncovered faster than ever, making current and future work an exciting continuation of Barbara McClintock's groundbreaking legacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831a0252c52d2b92a8d6f28251facae6b2cf9eff" target='_blank'>
              From Junk DNA to Genomic Treasure: Impacts of Transposable Element DNA, RNA, and Protein in Mammalian Development and Disease
              </a>
            </td>
          <td>
            Ten D. Li, Katelyn Toohill, A. Modzelewski
          </td>
          <td>2025-07-01</td>
          <td>Wiley Interdisciplinary Reviews. RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly prevalent malignant tumor worldwide, and its development is closely associated with dysregulated non-coding RNA expression. The long non-coding RNA MALAT1 is overexpressed in multiple cancers, but its precise mechanistic role and downstream regulatory network in HCC remain incompletely understood. Additionally, PRKAG1, a regulatory subunit of AMPK, has an unclear function in tumor progression. This study aimed to investigate the role and clinical significance of the MALAT1-PRKAG1 axis in HCC pathogenesis. Methods A multi-omics approach was employed to systematically dissect the mechanism of the MALAT1-PRKAG1 axis in HCC. Bioinformatics analysis using GEPIA2 and TCGA databases revealed that MALAT1 and PRKAG1 were significantly upregulated in HCC and correlated with poor prognosis. Cellular experiments demonstrated that knockdown of MALAT1 or PRKAG1 suppressed HCC cell proliferation, migration, and invasion. Mechanistic studies indicated that MALAT1 upregulates PRKAG1 expression by competitively binding miR-383-5p. Further analyses showed that PRKAG1 remodels the tumor immune microenvironment by modulating immune cell infiltration (CIBERSORT analysis) and intercellular communication (single-cell RNA sequencing), while also activating key pathways such as the cell cycle and DNA repair (GO/KEGG enrichment analysis). Results This study uncovered the critical role of the MALAT1-PRKAG1 regulatory axis in HCC. MALAT1 was significantly overexpressed in HCC tissues and cell lines, and its expression was associated with poor patient prognosis. Knockdown of MALAT1 markedly inhibited HCC cell proliferation, migration, and invasion. PRKAG1, as a downstream target of MALAT1, was also highly expressed in HCC and correlated with tumor stage and adverse outcomes. Mechanistically, MALAT1 competitively binds miR-383-5p to relieve its suppression of PRKAG1, thereby activating the P53 and AKT signaling pathways. Additionally, PRKAG1 modulated immune cell infiltration (particularly CD4+ T cells and M0 macrophages) and promoted intercellular communication via the MIF signaling network. Multi-omics analysis revealed that PRKAG1-associated genes were primarily enriched in critical pathways, including the cell cycle and DNA repair, collectively driving HCC progression. Conclusion This study elucidates the oncogenic role of the MALAT1/miR-383-5p/PRKAG1 axis in HCC, demonstrating that PRKAG1 promotes tumor progression by regulating cell proliferation, the immune microenvironment, and key signaling pathways. These findings provide potential novel targets for HCC prognosis assessment and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94f1c7d406fef526691446ae8363bec6c20c63d2" target='_blank'>
              Molecular mechanism of LncRNA MALAT1 in regulating hepatocellular carcinoma progression via the miR-383-5p/PRKAG1 axis and its role in the tumor immune microenvironment
              </a>
            </td>
          <td>
            Rui Zhang, Mingze Li, Shan Lu, Anni Wang, Shujun Zhang
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The impact of ionizing radiation (IR) with induction of various DNA damage is based not only on genetic but also on epigenetic effects. Epigenetic modifications determine the chromatin structure and DNA accessibility, thereby regulating cellular functions through the expression of individual genes or entire groups of genes. However, the influence of DNA repair processes on the restoration of local chromatin structures and global nuclear architectures is still insufficiently understood. In multicellular organisms, epigenetic mechanisms control diverse cellular functions of specific cell types through precise temporal and spatial regulation of gene expression and silencing. How altered epigenetic mechanisms regulate the pathophysiological function of cells, tissues, and ultimately entire organs following IR exposure remains to be investigated in detail. Radiation-induced epigenetic processes are particularly critical for immature cell populations such as tissue-specific stem and progenitor cells during development and differentiation of organ tissues. Genome-wide patterns of DNA and histone modifications are established cell types—specifically during the development and differentiation of organ tissues but can also be fundamentally altered in adult organism by stress responses, such as radiation-induced DNA damage. Following IR exposure, epigenetic factors are not always fully restored to their original state, resulting in epigenetic dysfunction that causes cells to lose their original identity and function. Moreover, severe radiation-induced DNA damage can induce premature senescence of cells in complex tissues, which ultimately leads to signs of aging and age-related diseases such as cancer. In this work, we provide an overview of the most important epigenetic changes following IR exposure and their pathophysiological significance for the development of acute and chronic radiation reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78d105ae28e3df5bffe779131184bc03d9d06867" target='_blank'>
              Role of Ionizing Radiation in Shaping the Complex Multi-Layered Epigenome
              </a>
            </td>
          <td>
            C. Rübe, M. A. A. Al-razaq, Carola Meier, Markus Hecht, Christian Rübe
          </td>
          <td>2025-08-08</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Knowledge of the structure, regulation, and function of genes is a fundamental pillar of molecular biology and modern medicine. Genes are DNA sequences that contain the information necessary to produce functional proteins or RNA, and are not only made up of coding regions (exons), but also introns and regulatory elements such as promoters, enhancers, and silencers. Mutations in any of these regions can alter normal cellular function and lead to various diseases. The regulation of gene expression ensures that not all genes are activated at the same time, but rather in a controlled manner depending on the cell type, stage of development, or environmental conditions. This control is mediated by transcription factors, epigenetic modifications such as DNA methylation, and the action of non-coding RNAs, mechanisms that allow multiple specialized cell types to be generated from the same DNA sequence. The function of genes is reflected in the products they encode, whether proteins or RNA, which participate in vital processes such as metabolism, cell signaling, or DNA repair. Genetic variants can cause loss of function, as in the case of cystic fibrosis due to mutations in the CFTR gene, or gain of function, as occurs in certain oncogenes involved in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ccb5bb302f5b6f126acbfc92ab37e76b9351d4d" target='_blank'>
              Estructura de un gen y sus elementos de control
              </a>
            </td>
          <td>
            Ana Laura Téllez García, Kevin Chung, Karen Hou, Stephania Hylton, J. Mosquera, Crital Zhang
          </td>
          <td>2025-08-31</td>
          <td>Genética y Genómica Clínica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epstein–Barr virus, the first identified human DNA tumour virus, is detectable in more than 90% of nasopharyngeal carcinoma patients in endemic regions. The 3D chromosome conformation analysis reveals that virus‒host chromatin interactions induce the spatial reorganisation of loops and compartments, resulting in “enhancer infestation” and switch of “H3K27 bivalency” at EBV-interacting regions. Through this mechanism, EBV hijacks KDM5B, a gatekeeper of genome stability, and its binding targets, leading to aberrant activation of an EBVIR-enhancer-KDM5B signature. Cancer cells with this signature present increased MYC activation, DNA damage responses, and epigenetic plasticity of epithelial-immune dual features with metastatic potential. Our multicentre multiomics study confirms that this signature is the prerequisite for chromosome instability and can be utilised as a risk factor for distant metastasis. This study highlights a mechanism in which latent viral episomes can alter the host high-order epigenotype, promoting transcriptional rewiring and metastasis in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24b257eddf1aac8b8f7ed9ed3a3ecfd24278af6" target='_blank'>
              Virus-human chromatin interactions reorganise 3D genome and hijack KDM5B for promoting metastasis in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            D. Chung, Zhaozheng Hou, Ying Wang, Kazi Anisha Islam, Songran Liu, Jiayan Liu, Larry Ka-Yue Chow, Yuki Yuk-Wun Wong, Cyrus Paak-Ting Chak, Yingpei Zhang, Lanqi Gong, Ziyang Qi, K. Cheng, Zhuoyou Yu, Ping Feng, Zilu Huang, R. Ngan, Xin-Yuan Guan, W. Ng, Zhonghua Liu, A. Tsang, D. Kwong, Ann Wing-Mui Lee, V. H. Lee, Honglin Chen, Yunfei Xia, Wei Dai
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract The presence of CDKN2A/B homozygous deletion (HD) denotes a higher WHO grade and more aggressive behavior in gliomas and meningiomas. The gold standard assays for detecting copy number variation (CNV) include chromosomal microarray, next generation sequencing, and fluorescence in situ hybridization (FISH). However, these methods are costly, time-consuming, and are limited in resource-sparse centers. Thus, an inexpensive and time-efficient assay for detecting CDKN2A/B HD is necessary for routine neuropathology cases. Chromogenic in situ hybridization (CISH) combines DNA hybridization techniques utilized in FISH with conventional peroxidase reactions utilized in immunohistochemistry to interrogate CNV with brightfield microscopy. We hypothesize that CISH is a sensitive and specific method for detecting CDKN2A/B HD in gliomas and meningiomas. We performed CDKN2A CISH on a retrospective cohort of whole slide tissue sections of 5 gliomas and 12 meningiomas using the ABNOVA CDKN2A/Centromere 9 (CEP9) CISH probes and kit. Two pathologists independently counted and scored 100 cells from each case showing any CEP9 (red) signals and no CDKN2A (green) signals as HD, and cells showing red and green signals as CDKN2A/B intact. The pathologists’ scores were averaged and compared to the CDKN2A/B status for each of the glioma and meningioma cases, acquired via FISH and methylation profiling, respectively. Our results demonstrate that detection of CDKN2A/B HD with CISH showed 100% concordance with CDKN2A/B locus interrogation with FISH and methylation profiling. CISH is therefore a sensitive and specific method for detecting CDKN2A/B HD in glioma and meningioma and is a practical alternative to traditional CNV detection methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee0c49f58caa417ea1856c56abdf8eb9dd1a185" target='_blank'>
              25 CHROMOGENIC IN SITU HYBRIDIZATION FOR DETECTION OF CDKN2A/B HOMOZYGOUS DELETION IN GLIOMA AND MENINGIOMA
              </a>
            </td>
          <td>
            K. C. Martin, Tara Spence, Janine Senz, Julia Naso, Andrew Churg, Stephen Yip
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2093fb2bf2cfa5129572c94a9428ff0b8a8b0c1b" target='_blank'>
              Gene signatures characterizing driver mutations in lung squamous carcinoma are predictive of the progression of pre-cancer lesions.
              </a>
            </td>
          <td>
            Yupei Lin, Venugopalareddy Mekala, Jianrong Li, Xiang Wang, M. Aminu, Jia Wu, Jianjun Zhang, R. Ripley, Christopher I Amos, Chao Cheng
          </td>
          <td>2025-07-30</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction Retrotransposons (REs) constitute nearly half of the genome and include long terminal repeat (LTR) elements, Long INterspersed Elements (LINE), and Short INterspersed Elements (SINE). REs are typically silenced in somatic tissues via DNA methylation but can be reactivated through DNA hypomethylation, potentially impacting gene regulation. Here, we investigate genome-scale profiles of RE methylation in high-grade serous ovarian carcinoma (HGSOC) and associations with survival among Black women. Methods Methylation levels of LTR, LINE-1, and Alu (type of SINE) in 200 HGSOC tumors were predicted using a random forest approach and clustered using multiple consensus algorithms. Associations between RE methylation clusters and survival were evaluated using Cox proportional hazard regression, adjusting for age, stage, and debulking status. We performed sensitivity analyses restricted to women with late-stage disease and with adjustment for BRCA1/BRCA2 mutations. Results Two RE methylation clusters were identified. Cluster 1 exhibited a more hypomethylated RE profile (“Active”), while Cluster 2 was more hypermethylated (“Repressed”). No statistically significant differences in patient or clinical characteristics were observed between clusters. Compared to the Active Cluster, the Repressed Cluster was associated with an increased risk of mortality (HR = 2.41; 95% CI 1.04–5.59) and had a lower proportion of T cells. This association was consistent in sensitivity analyses. Conclusion A more hypermethylated RE profile was linked to worse survival among Black women with HGSOC, highlighting the potential of RE methylation as a prognostic biomarker. Further research is needed to understand the underlying biological mechanisms and their implications in ovarian cancer biology and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01942-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc642af0cdc56a04adbb30045ba5a19d537e41" target='_blank'>
              Retrotransposon methylation profiles and survival in Black women with high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Christelle M. Colin-Leitzinger, Katherine A Lawson-Michod, Courtney E Johnson, Irma M Vlasac, Sean Yoder, Tania Mesa, Dana Roeber, Chad D Huff, M. Hildebrandt, Kristen Haller, Anthony J Alberg, E. Bandera, Melissa Bondy, Michele L Cote, Theresa A Hastert, Edward S Peters, Paul D Terry, Andrew B. Lawson, Andy Berchuck, Brooke L Fridley, J. Chern, J. Doherty, Jeffrey R Marks, J. Schildkraut, Brock C. Christensen, Lucas A Salas, Lauren C. Peres
          </td>
          <td>2025-07-30</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Genomic instability and loss of proteostasis are two of the primary Hallmarks of Aging. Although these hallmarks are well-defined in the literature, the mechanisms that drive genomic instability and loss of proteostasis as cells age are still largely unknown. Using budding yeast replicative lifespan as a model for aging in actively dividing cells, we identified nuclear proteins that were depleted in the earliest stages of aging. We found that many age-depleted proteins were involved in ribosome biogenesis, specifically in ribosome processing, or in maintenance of chromatin stability. We focused on topoisomerase I (Top1) as a novel age-depleted nuclear protein and found that its depletion in the early stages of aging was not a result of transcriptional changes or changes in protein turnover. Despite the stark depletion of Top1 in early aging, rescue of this age-dependent depletion was actually harmful to replicative lifespan. We found that Top1, when overexpressed, disrupts the stoichiometry of the RENT complex by pulling Sir2 away from the ribosomal DNA (rDNA), a phenotype which is further enhanced when the overexpressed Top1 is catalytically dead. Loss of Sir2 from the rDNA via the overexpression of catalytically dead Top1 decreases RNA Pol II silencing of a reporter gene inside or adjacent to the rDNA, consistent with the lifespan defect. Finally, we found that the catalytic activity of Top1 plays an important role in the establishment of rDNA silencing, raising the possibility that rDNA secondary structure/DNA topology is important for RNA Pol I-dependent spreading of silent chromatin across the rDNA locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb1885993b368b41abc338c8a39ec0f62ce5c9c" target='_blank'>
              Age-dependent topoisomerase I depletion alters recruitment of rDNA silencing complexes
              </a>
            </td>
          <td>
            Lindsey N. Power, Natalia Zawrotna, M. Dinda, Abigail E. Weir, Bishal P. Paudel, Oshil Ghimire, Karolina Kisiel, Christopher T. Letai, Kevin A. Janes, Jeffrey S Smith
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd145cac2df335e27d4a9869d5f4c02876f7c50" target='_blank'>
              RNA reinforces condensate nucleation on chromatin to amplify oncogenic transcription.
              </a>
            </td>
          <td>
            Krista A Budinich, Xinyi Yao, Chujie Gong, Lele Song, Michelle C Lee, Qinglan Li, Sylvia Tang, Kaeli M Mathias, Yiman Liu, Son C Nguyen, Eric F. Joyce, Yuanyuan Li, Haitao Li, Liling Wan
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most lethal gynecological malignancy. Therapeutic options remain limited for patients lacking predictive biomarkers, particularly those with BRCA wild-type tumors or those who have acquired resistance to both PARP inhibitors and platinum-based chemotherapy. Replication stress, TP53 mutations, and genomic instability characterize HGSOC. The cellular response to replication stress is primarily mediated by checkpoint kinases; however, this mechanism is frequently impaired in tumor cells. Consequently, cancer cells become increasingly dependent on the replication stress response (RSR) pathway for survival, and susceptible to therapies targeting the ATR-CHK1-WEE1 axis—a key regulator of genomic integrity. Inhibition of these checkpoint kinases can disrupt cell cycle control, inducing mitotic catastrophe and subsequent cancer cell death. Another defining feature of HGSOC is its immunosuppressive tumor microenvironment (TME), which has limited the efficacy of immune checkpoint inhibitors. Emerging evidence suggests that inhibition of the RSR pathway may not only exploit intrinsic tumor vulnerabilities but also modulate the TME to enhance anti-tumor immune responses. This provides rationale for combination approaches integrating RSR pathway inhibitors with innovative immune checkpoint blockade (ICB). This review examines the mechanistic rationale and therapeutic potential of such combinations, drawing on both preclinical and clinical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa92875288685395bcc326c756f9f278710481b" target='_blank'>
              Interplay of replication stress response and immune microenvironment in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Laura Venegas, Stephanie Lheureux
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59d7e7dcc2e2b5046587aac4504172a2317b36b" target='_blank'>
              Coordinated gene expression within sustained STAT3-associated chromatin conformations contributes to hepatocellular carcinoma progression.
              </a>
            </td>
          <td>
            Sunyoung Jang, Sumin Yoon, Hyeokjun Yang, Nayun Choi, Su-hyang Han, Lark Kyun Kim, Hyoung-Pyo Kim, Jong Hoon Park, Daeyoup Lee, Kyung Hyun Yoo
          </td>
          <td>2025-07-26</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="In mammals, repression of germline-specific gene expression is essential for preserving somatic cell identity and preventing disease. Germline gene silencing is often dependent on the presence of promoter 5-cytosine DNA methylation (5meC). Early mammalian development, however, is marked by a dramatic loss of 5meC levels genome-wide. Recent research has highlighted a specialized variant of the Polycomb Repressive Complex 1, PRC1.6, as a key regulator that maintains transient silencing of germline genes in this embryonic window. Eventually, PRC1.6 seems to stimulate the recruitment of de novo DNA methyltransferases (DNMTs), although the precise mechanisms remain to be fully elucidated. Evidence suggests a coordinated epigenetic relay, potentially involving direct protein interactions and shaping the local histone modification landscape. This review explores PRC1.6 as a central epigenetic hub that connects Polycomb repression, histone 3 lysine 9 (H3K9) methylation, and DNA methylation pathways. Unraveling this interplay will provide key insights into the mechanisms that maintain the critical barrier between the germline and the soma, essential not only for proper development but also for preserving somatic cell function and health throughout life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ed478b995bf1b398b0bfe5ba03fe4e97c29208" target='_blank'>
              Epigenetic relay: Polycomb-directed DNA methylation in mammalian development
              </a>
            </td>
          <td>
            Teresa Urli, Maxim V. C. Greenberg
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Insertion sequence (IS) elements are key drivers of bacterial genome plasticity, yet the overall regulation of their transposition remains poorly understood. This is especially true for the multiple-layer regulation at the donor site, which has been largely overlooked. Using multiple mutation assays, genetic manipulations and reporter genes, this study focuses on characterizing how endogenous DNA sequences, transcriptional and translational factors, and genomic context regulate IS1 transposition from its donor site. Out of six elements within the chromosome of E. coli strain BW25113, IS1A and IS1E (both with the consensus sequence) contribute to over 99.9% of the overall IS1 transposition within the genome while the other four elements without the non-consensus sequence are essentially incapable of transposing. Inducing a ribosomal -1 frameshift at the A6C motif increases transposition over 1000-fold, but this enhancement is largely reversed by restoring InsA-mediated transcriptional regulation. Strikingly, genomic sequences flanking IS1 elements appreciably modulate transposition by promoting transcription or facilitating formation of transpososomes, a phenomenon that remains under-studied. Finally, IS1 was confirmed to undergo replicative transposition intramolecularly, a mechanism shown here to be independent of transposase levels in the cell. These findings contribute to our understanding of mobile genetic element regulation and potentially offer strategies for mitigating their potentially harmful effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed2e184755f4f8d507b9afcfaae112ea127ba2c1" target='_blank'>
              Examining the Roles of Genomic Context and Endogenous Regulatory Elements on IS1 Transposition Within the Escherichia coli Genome
              </a>
            </td>
          <td>
            Sofia Smith, Zhongge Zhang, Allyson Ho, Tusha Karnani, Jack W Ord, Milton H. Saier
          </td>
          <td>2025-08-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Microscopic examination of biopsy tissues remains essential for cancer diagnosis, despite advancements in sequencing technologies. Alterations in nuclear size or the nuclear-to-cytoplasmic ratio are hallmark features of cancer cells and often correlate with disease progression. However, the mechanisms underlying nuclear size abnormalities and their impact on tumor progression remain unclear. In this study, we demonstrate that nuclear hypertrophy occurs in response to enhanced DNA replication stress, a key characteristic of cancer cells. Increased actin polymerization within the nucleus appears to be the primary mechanism driving nuclear hypertrophy downstream of the ATR-CHEK1 pathway. Replication stress-induced nuclear hypertrophy alters transcriptomic profiles and chromatin topology, while reducing the migratory and metastatic capacity of cancer cells. In addition, nuclear hypertrophy in cancer cells is associated with increased infiltration of antitumor immune cells. Our findings suggest that cell-autonomous effects of nuclear hypertrophy do not promote cellular fitness or aggressive characteristics in cancer cells. This may explain why cells with nuclear hypertrophy are not positively selected and persist as a subpopulation during tumor progression and metastasis. Furthermore, the link between replication stress and nuclear hypertrophy provides insights into why enlarged nuclei are consistently observed in advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd09203e34a2e72f651d176a77deed8567aab11" target='_blank'>
              Replication stress-induced nuclear hypertrophy alters chromatin topology and impacts cancer cell fitness.
              </a>
            </td>
          <td>
            Changgon Kim, Semyeong Hong, Soo Hyeon Ma, Joochan Lee, Hee So, Jun Young Kim, Eunbie Shin, Kippeum Lee, Sohee Choi, Juyeon Park, YongKeun Park, You-Me Kim, Ji Hun Kim, Joon Kim
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. M. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences that constitute a significant portion of mammalian genomes. While typically silenced by epigenetic mechanisms, mounting evidence indicates TEs can regulate gene expression and chromatin architecture. However, their regulatory roles under various environmental exposures remain largely unexplored. In this study, we investigate the regulatory functions of TEs in mouse liver tissue following early-life exposure to environmental toxicants, including arsenic (As), lead (Pb), bisphenol A (BPA), tributyltin (TBT), di-2-ethylhexyl phthalate (DEHP), tetrachlorodibenzo-p-dioxin (TCDD), and particulate matter less than 2.5 micrometers (PM2.5). These toxicants are linked to various health issues, including neurodevelopmental deficits, metabolic and immune dysfunction, and increased cancer risks. Integrative analysis of 351 multi-omics datasets from liver tissues of 5-month-old mice indicated that early-life environmental exposures significantly altered chromatin accessibility and expression of TEs in later life stage, revealing distinct exposure-specific signatures and sex-dependent responses. 6,699 TEs were identified with altered chromatin accessibility, mostly in non-coding regions, suggesting potential impact on gene regulation. Within these TEs, LINE elements were enriched in genes involved in metabolic pathways, while LTR elements, particularly the ORR1E subfamily, were predominantly associated with immune-related genes. Additionally, we identified 140 TE-gene chimeric transcripts with TE-derived novel transcription start sites, highlighting TE-contributed transcriptional plasticity. Our findings depict a comprehensive landscape of TE regulation under early-life toxicant exposures, offering insights into TEs biology and their impact on health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5f0711b9c755ac942e42700386af14ea62ffc2" target='_blank'>
              The impact of environmental exposures on the epigenomic and transcriptomic landscape of transposable elements
              </a>
            </td>
          <td>
            Benpeng Miao, Yu Zhang, Wanqing Shao, Shuhua Fu, Marisa S. Bartolomei, Cheryl Walker, Cristian Coarfa, Bo A Zhang, Ting Wang
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, Ye-Ke Wang, Yuan-Ru Xiong, Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, Dao-Qiong Zheng
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Over the past two decades, genetic and proteomic screens have identified the Hippo pathway as a complex signaling network that controls tissue growth and human cancer. Despite these advances, our understanding of how Hippo signaling regulates transcription is less clear. To address this, we used live microscopy to study the nuclear behavior of the major Hippo pathway transcription effectors, YAP and TEADs. We reveal that TEADs are a major determinant of YAP DNA binding and nuclear mobility, while YAP minorly influences TEADs. YAP and TEAD1 associate with DNA for longer periods in cells with intrinsically low Hippo pathway activity and upon acute Hippo pathway perturbation. TEAD1 binds the genome on a broad range of timescales, and this is extended substantially in nuclear condensates. Last, a cancer-associated YAP fusion protein exhibits substantially different biophysical behavior than either YAP or TEAD1. Thus, we reveal that Hippo signaling regulates transcription, in part, by influencing the DNA binding times of YAP and TEADs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd586bf6ea2612de1fe233a16f92fef5a5055813" target='_blank'>
              Hippo signaling regulates the nuclear behavior and DNA binding times of YAP and TEAD to control transcription
              </a>
            </td>
          <td>
            Benjamin Kroeger, Samuel A. Manning, Varshini Mohan, Jieqiong Lou, Guizhi Sun, Sara Lamont, Alex J. McCann, Mathias Francois, J. Polo, Elizabeth Hinde, Kieran F. Harvey
          </td>
          <td>2025-07-25</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Diversification and selective propagation are the main driving forces of evolution, resulting in the emergence of organisms possessing various strategies. Here, we conducted in vitro evolution of the replication origin (oriC) under the pressure of an AT-rich mini-chromosome amplification using a reconstituted Escherichia coli replication-cycle reaction (RCR). Using next-generation sequencing, we identified that these evolved oriCs contain select mutations within the duplex unwinding element (DUE) and DnaA-binding site (DnaA box) regions, which are crucial for replication initiation. Real-time detection of RCR amplification (real-time RCR) revealed that the DUE mutations, which decreased the GC content, along with the introduction of a specific A/T sequence in DUE-M and a consecutive KAK (K = T or G) motif in DUE-R, enhanced the RCR amplification efficiency compared to the wild-type oriC (oriCwt). A competitive amplification assay also elucidated that the DnaA box mutations confer a competitive advantage over coexistent oriCwt. Although these DUE and DnaA box mutations were selected through the same amplification reaction, they exhibited distinct competitive amplification strategies. The DUE mutant represents a faster propagation strategy (r-strategy), while the DnaA box mutant represents an adaptive strategy with a competitive advantage (K-strategy), representing the r/K-selection theory in molecular evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d8adc8e642b9b4fca6ba9da0b2f823fd1341a1" target='_blank'>
              Adaptive evolution of oriC through in vitro propagation of a mini-chromosome in RCR
              </a>
            </td>
          <td>
            Shota Suzuki, M. Su’etsugu
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Intratumor heterogeneity (ITH) is associated with anti-tumoral immune response and with the efficiency of cancer immunotherapy, yet overcoming ITH remains a significant challenge. Notably, cellular MYC (c-MYC) has been shown to be a pivotal orchestrator of this ITH progression. Here, we develop a c-MYC-based sensing circuit (cMSC) that is activated exclusively by aberrant c-MYC levels, along with an exosome-based cell-to-cell (CtC) system that augments communication among tumor cells, effectively targeting all cells in tumors circumventing the limitations imposed by ITH. Further expression of multifunctional immunostimulatory agents in these cMSC-reprogrammed cancer cells remodels the tumor microenvironment, enhancing selective T-cell-mediated oncolysis. Our cMSC/CtC platform specifically senses aberrant c-MYC expression and subsequently triggers a robust cancer immunotherapeutic response. These findings offer a promising avenue for targeting cancers via precisely sensing c-MYC, overcoming the limitations of ITH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd1994e5ac75ac89b2d1f1610daef6efdae5bcc0" target='_blank'>
              Reprogramming the tumor microenvironment with c-MYC-based gene circuit platform to enhance specific cancer immunotherapy
              </a>
            </td>
          <td>
            Hengji Zhan, Hongjin Wang, Bolin Pan, Junlin Lu, Kanghua Xiao, Jiajian Lai, Zehua Chen, Kaiwen Jie, Siting Chen, Hong Li, Tianxin Lin, Xu Chen
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ubiquitin carboxyl terminal hydrolase L3 (UCHL3) is a deubiquitinase belonging to UCH protease family. Previous studies have demonstrated that abnormal expression of UCHL3 is correlated with the development of human diseases, especially in tumors. Notably, UCHL3 exhibits contradictory biological functions across different cancer types. Initial studies identified UCHL3 as a tumor suppressor in prostate cancer. However, emerging evidence suggests that UCHL3 serves as an oncogene in other cancers. Several UCHL3 inhibitors have shown a promising anti-tumor effect in vitro and in vivo. UCHL3 has also been linked to DNA damage repair, which is essential for genome stability. Overexpression of UCHL3 enhances DNA damage repair induced resistance to chemotherapy or radiotherapy in certain tumor types. This review aims to summarize the promoting role of UCHL3 in DNA damage repair and its dual, paradoxical roles in tumor progression, along with the associated mechanisms, and to provide insights into the use of UCHL3 as a therapeutic target for overcoming DNA damage repair-mediated resistance to chemotherapy and radiotherapy, as well as for treating tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3d5f1b68a091ec87b927bf44542627cac87c63" target='_blank'>
              UCHL3: a crucial deubiquitinase in DNA damage repair and tumor progression
              </a>
            </td>
          <td>
            Jiahao Liu, Shulu Hu, Junqi Xiao, Jumei Zhou
          </td>
          <td>2025-07-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Current cancer chemotherapy uses etoposide to induce DNA double-strand breaks for cell killing, which can cause fluctuations in stemness promoters like OCT4A, SOX-2, or NANOG in relation to senescence promoters p21Cip1, p27, or p16ink4A, and mitigate anti-cancer therapy. These fluctuations take place over several days and are associated with changes in the organization of euchromatin and heterochromatin. Here, we show how the induction of double-strand breaks by etoposide led to the compaction of euchromatin and de-condensation of heterochromatin on day 3. This was accompanied by a co-localization of euchromatin and heterochromatin marks as detected by Single-Molecule Localization Microscopy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a3f66309c268ad41eb3fdc8156d8889c120362" target='_blank'>
              Chemotherapy (Etoposide)-Induced Intermingling of Heterochromatin and Euchromatin Compartments in Senescent PA-1 Embryonal Carcinoma Cells
              </a>
            </td>
          <td>
            Marc Bayer, Jaroslava Zajakina, Myriam Schäfer, K. Salmina, Fēlikss Rūmnieks, J. Jansons, F. Bestvater, R. Kurg, J. Erenpreisa, Michael Hausmann
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dps is the most abundant nucleoid-associated protein in starved Escherichia coli with ~180,000 copies per cell. Dps binds DNA and oxidises iron, facilitating survival in harsh environments. Dps-DNA complexes can form crystalline structures, leading to the proposed model that Dps reorganises the starved E. coli nucleoid into a compact liquid crystal, slowing chromosome dynamics and limiting access of other proteins to DNA. In this work, we directly tested this model using live-cell super-resolution microscopy and Hi-C analysis. We found that after 96 h of starvation, Dps compacts the nucleoid and increases short-range DNA-DNA interactions, but does not affect chromosome accessibility to large protein nanocages or small restriction enzymes. We also report that chromosome dynamics and organisation are primarily impacted by the bacterial growth phase; the effect of Dps is relatively minor. Our work clarifies the role of Dps in modulating nucleoid properties, and we propose an updated model for Dps-DNA interactions in which Dps binds, protects and compacts DNA largely without influencing chromosome access, dynamics and organisation. Additionally, this work provides a general framework for assessing the impact of nucleoid-associated proteins on key aspects of chromosome function in live cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a6930c67bb9911b075c952097a3099d7e9b19fb" target='_blank'>
              Dps binds and protects DNA in starved Escherichia coli with minimal effect on chromosome accessibility, dynamics and organisation
              </a>
            </td>
          <td>
            Lauren A. McCarthy, Lindsey E. Way, Xiaofeng Dai, Zhongqing Ren, David E. H. Fuller, Ishika Dhiman, Linaria Larkin, Jurriaan J. D. Sieben, Ilja Westerlaken, E. Abbondanzieri, Gail G Hardy, Anne S. Meyer, Xindan Wang, Julie S. Biteen
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Quiescent cells (Q) are seemingly inactive, developmentally arrested cells, whose universal characteristic is the ability to promptly reenter the cell cycle upon sensing of external cues. Q cells are responsive to the environment and flexible enough to adapt to available resources. In budding yeast, quiescent nuclear features are drastically distinct from those observed in nutrient replete conditions: the nuclear volume is reduced, the telomeres relocate from the nuclear periphery to the center of the nucleus into a hypercluster, chromatin is found in a compacted, hypoacetylated state, and transcription is globally shutdown. Yet, Q cells can restart transcription within minutes of refeeding. Here, we follow the global decrease of transcription in sorted, developing Q populations, and its reactivation upon release. We find that transcription and telomere clustering dynamics in and out of quiescence are independent events. We report a genome-wide redistribution of the transcription machinery as cells progress into quiescence. Although most genes are shut down, 3% of coding genes remain active. Furthermore, RNAPII accumulates at one third of gene promoters. The corresponding genes are highly enriched among those showing a high level of transcription and high frequency of expression in individual cells, shortly after cells are refed, as monitored by single-cell RNA-seq. Our results point toward a role for quiescent-specific RNAPII distribution to ensure a rapid and robust transcriptional response upon return to growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79310aef16945d91c92b6960ca1816e65764f91e" target='_blank'>
              Intergenic accumulation of RNAPII maintains the potential for swift transcriptional restart upon release from quiescence.
              </a>
            </td>
          <td>
            Manuela Baquero Pérez, Gertjan Laenen, Isabelle Loïodice, Mickael Garnier, Ugo Szachnowski, Antonin Morillon, Myriam Ruault, Angela Taddei
          </td>
          <td>2025-08-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Topoisomerase I (Top1) alleviates DNA supercoiling during replication and transcription, but its catalytic cycle can be hijacked by chemotherapeutic agents such as camptothecin (CPT), stabilizing Top1-DNA covalent complexes (Top1cc) that threaten genome integrity. Efficient resolution of these trapped intermediates is crucial to prevent replication stress, DNA breaks, and cell death. Poly (ADP-ribose) polymerase 1 (PARP1) is a key sensor of Top1cc, facilitating repair by recruiting tyrosyl-DNA phosphodiesterase 1 (TDP1) and modifying chromatin to promote lesion accessibility. Beyond this canonical pathway, emerging evidence highlights PARP1-independent mechanisms such as endo nucleolytic cleavage, proteolytic degradation of Top1 and replication-associated processing. Intriguingly, PARP1 appears to act as a molecular switch between TDP1 and the endonuclease pathway for the repair of Top1cc. This review highlights mechanisms of PARP1-dependent and -independent Top1cc repair pathways, their interplay and redundancy, and how their targeting can enhance Top1-based cancer therapies and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d35356a5858a286f35b40059a6b371da2f3da9f3" target='_blank'>
              PARP1-Dependent and Independent Pathways for Resolution of Trapped Topoisomerase I Covalent Complexes.
              </a>
            </td>
          <td>
            B. Das, Banhi Chowdhury, Sarita Das, Asmit Banerjee
          </td>
          <td>2025-09-16</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da4d10721badd9a65f0ec25fedb84c3d3183aaf" target='_blank'>
              Direct functional HOXA9/DNA-binding competitors versus epigenetic inhibitors of HOXA9 expression on cell proliferation, death and differentiation processes in the model of MLL-rearranged acute myeloid leukemia.
              </a>
            </td>
          <td>
            Julie Vrevin, Mélanie Lambert, Marine Andrique, Nathalie Jouy, M. David-Cordonnier
          </td>
          <td>2025-08-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="R-loops are three-stranded nucleic acid structures consisting of a DNA/RNA hybrid and a displaced single-stranded DNA. Once considered rare byproducts of transcription, R-loops are now recognized as important regulators of various nuclear processes. In particular, evidence indicates a role for R-loops in regulating DNA methylation dynamics. R-loops have been shown to promote active DNA demethylation—the enzymatic reversal of 5-methylcytosine (5mC) back into cytosine—by recruiting associated proteins, providing an attractive targeting mechanism. Nevertheless, many important aspects of this process, including whether the associated proteins bind to and function on R-loops, remain to be substantiated. In this study, we demonstrate for the first time that thymine DNA glycosylase (TDG), a key enzyme in the active DNA demethylation pathway, binds tightly to R-loops in vitro and can excise DNA demethylation intermediates 5-formylcytosine (5fC) and 5-carboxycytosine (5caC) from DNA in DNA/RNA hybrid duplexes. We also show that R-loops guide the strand-specific activity of TDG at CpG sites, potentially explaining the asymmetric distribution of 5fC/5caC at gene promoters. Furthermore, we provide important mechanistic insights into base excision on DNA/RNA hybrid duplexes using 19F NMR. Finally, our findings suggest that TDG–R-loop interactions may be widespread in human cells. Collectively, our results provide strong evidence that R-loops play a critical role in DNA demethylation and support a mechanism in which 5fC/5caC are directly removed from DNA/RNA hybrids in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f720c25376cfceef4fd5beb6c69c7fd9ad1d0f50" target='_blank'>
              Thymine DNA Glycosylase Binds to R-Loops and Excises 5-Formyl and 5-Carboxyl Cytosine from DNA/RNA Hybrids
              </a>
            </td>
          <td>
            Baiyu Zhu, L. Pidugu, Mary E. Cook, Xinyu Y Nie, E. A. P. T. Amarasekara, Jerome S. Menet, A. Drohat, Jonathan T. Sczepanski
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Variations in DNA methylation within the DNA damage response (DDR) mechanism could have significant implications for glioma prognosis and immune responses. This study aimed to explore the global DNA methylation landscape of DDR genes in gliomas and identify key epigenetically regulated genes influencing glioma biology and immunity. Methods This study incorporated a range of public and local glioma datasets. Multiple clinical, bioinformatic, and in vitro experimental analyses were conducted to explore clinical and biological aspects. Results Global DNA methylation variations in DDR genes correlated with distinct glioma prognoses, with five CpGs identified as potent predictors. Hierarchical clustering and a risk-score model based on these CpGs unveiled immune-related prognostic subgroups in glioblastomas (GBMs) and lower-grade gliomas (LGGs). NSUN5, epigenetically regulated by one of these CpGs, highlighted the biological significance of the DDR CpG panel. In vitro, NSUN5 displayed tumor-suppressor-like activities in GBM cells, but clinically, it was an unfavorable prognostic marker. Depletion of NSUN5 shifted cytosolic DNA sensing from a STING-dependent (cGAS-STING) pathway to a STING-independent (DNA-PK-HSPA8) pathway, leading to a delayed but more robust type I interferon (IFN) response in GBM cells and enhancing microglial M1 polarization and chemotaxis. This may partially account for the functional discrepancy of NSUN5 observed between experimental and clinical contexts. Conclusion This study highlights the complex interplay between DNA methylation, the DDR mechanism, cytosolic DNA sensing, and glioma immunity. These findings may inspire novel strategies for DNA sensing-based immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06360-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc54757ac8e99a0979bc781abac88c0c6433dc38" target='_blank'>
              DNA methylation variations of DNA damage response in glioblastoma: NSUN5 modulates tumor-intrinsic cytosolic DNA-sensing and microglial behavior
              </a>
            </td>
          <td>
            An-An Yin, Yan-Qiu Yao, Yi-Feng Liu, Yu-Sha Ji, Yalong He, Tian-Chi Ma, Wen-Heng Guo, A. Etcheverry, M. Aubry, Jean Mosser, Wei Lin, Yuan-Ming Wu, Kun Chen
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="GGH (Gamma-glutamyl hydrolase) is a folate metabolism enzyme that hydrolyzes intracellular polyglutamylated folates and is highly expressed in various cancers. It remains unclear whether GGH functions as an oncogene and its underlying mechanisms in tumor progression. Here, it is reported that GGH silencing inhibited the growth of lung cancer cells in vivo and in vitro. The oncogenic function of GGH relied on its non-canonical role as a novel RNA-binding protein, which maintained the cell cycle and DNA replication by stabilizing target mRNAs. Furthermore, GGH bound to the GC-rich motif in the 5' untranslated region of mRNAs, such as CDC6 and CCND1. Additionally, GGH directly interacts with HuR (Human Antigen R), a well-characterized RNA-binding protein critical for mRNA stability in cancer. GGH, HuR, and their mRNA targets formed a ternary complex, which may facilitate the induction of a circular mRNA conformation, potentially enhancing RNA stability. Finally, it is found that GGH is highly expressed in lung cancer tissues, and its elevated expression correlates with worse patient survival in lung cancer. This discovery offered novel insights and identified potential therapeutic targets for the prevention and treatment of lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71630599b9ea7d81e787095673033ab508c0165a" target='_blank'>
              The GGH/HuR Complex Binds and Stabilizes mRNAs to Maintain Tumor Cell Cycle and DNA Replication.
              </a>
            </td>
          <td>
            Yu Li, Xinrui Li, Yuhui Du, Sijie Chen, Xiaoniu He, Zhangrong Xie, Zhiqing Zhou, Huijie Zhao, Xiaofei Zeng, Guoan Chen
          </td>
          <td>2025-08-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Integrating conjugative elements (ICEs) are mobile genetic elements conferring a wide range of beneficial functions upon their bacterial hosts. Generally, they can be activated from their integrated states to undergo horizontal gene transfer via conjugation. In the case of the human gastric pathogen Helicobacter pylori , a paradigm for extensive genetic diversity, highly efficient natural transformation and recombination processes may superimpose canonical transfer of its two ICEs termed ICE Hptfs3 and ICE Hptfs4 , and thus shape their composition substantially. Here, as a part of the Helicobacter pylori Genome Project (Hp GP ) initiative, we have analyzed high-quality genome sequences from 1011 clinical strains with respect to their ICE content and variability. We show that both elements are highly prevalent in all H. pylori populations, but have a strong tendency for gene erosion. ICE sequence variations reflect the population structure and show a clear signature of increased horizontal transfer. A detailed map of ICE integration sites revealed local preferences, but also how recombination processes result in hybrid elements or genome rearrangements. Population-specific differences in ICE cargo genes might reflect distinct requirements in the biological functions provided by these mobile elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573f2ef5afd055f208fa4bb18b449324318e3244" target='_blank'>
              Global diversity of integrating conjugative elements (ICEs) in Helicobacter pylori and their influence on genome architecture
              </a>
            </td>
          <td>
            A. Gutiérrez-Escobar, Mrinalini Srinivasan, Z. Muñoz-Ramirez, Filipa F. Vale, Difei Wang, Santiago Sandoval-Motta, J. P. Dekker, Kaisa Thorell, M. C. Camargo, Yoshio Yamaoka, Wolfgang Fischer
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a59a21b3122614696fcbf37a9973f0709eb05a0" target='_blank'>
              Unravelling the genetics and epigenetics of the ageing tumour microenvironment in cancer.
              </a>
            </td>
          <td>
            H. Easwaran, A. Weeraratna
          </td>
          <td>2025-09-08</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background DNA methylation plays a critical role in the dynamics of gene expression regulation and the development of various disorders. Whole-genome bisulfite sequencing can provide single-base resolution of CpG methylation levels and is the “gold standard” for DNA methylation quantification, but its high cost limits its widespread application. In contrast, targeted sequencing provides an optimal, cost-effective solution when focusing on specific candidate regions while providing sufficient sequencing depth. Here, we present a targeted bisulfite sequencing approach in which nanopore sequencing is used to study the methylation status of regions of interest. Methods We applied this workflow to study the promoters of candidate genes associated with severe preterm delivery in a Latin American population. We amplified fragments greater than 1 kilobase in length from 12 genes via long PCR. Each sample was barcoded and pooled for sequencing in MinION flow cells. Results This approach achieves high sequencing depths, ensuring robust DNA methylation (DNAm) estimates. We detected significant hypomethylation of MIR155HG and hypermethylation of the ANKRD24 gene promoter in severe preterm birth samples, which is concordant with previously reported gene expression changes. Conclusions This approach represents a scalable and cost-effective method suitable for targeted promoter methylation profiling across several samples. Our study provides a proof-of-concept for larger studies, demonstrating the broad applicability and scalability of our assay to any locus of interest. These features render the method especially apt for clinical diagnostics and precision medicine. Supplementary Information The online version contains supplementary material available at 10.1186/s12920-025-02193-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c52694e5de9b6dee435d0c8482235c952d5198d" target='_blank'>
              Cost-effective promoter methylation analysis via target long-read bisulfite sequencing: a case study in severe preterm birth
              </a>
            </td>
          <td>
            Silvana Pereyra, Angela Sardina, Rita Neumann, Celia May, R. Sapiro, Bernardo Bertoni, M. Cappetta
          </td>
          <td>2025-07-29</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are remnants of retroviral infections that have become stably integrated into host germline genomes. Far beyond passive genomic elements, ERVs actively shape host evolution through complex mechanisms involving genetic innovation, immune modulation, and species adaptation. This review provides a comprehensive synthesis of ERV biology, highlighting recent advances in their classification, amplification mechanisms, and epigenetic silencing. Particular emphasis is placed on the cross-talk between ERVs and exogenous retroviruses (XRVs), demonstrating how receptor competition, recombination, and immune evasion contribute to virus-host co-evolution. We explore ERVs as molecular markers for phylogenetic reconstruction, with case studies such as Koala retrovirus (KoRV) and HERV-K illustrating regional transmission dynamics and co-opted immune functions. Additionally, we discuss the functional domestication of ERVs into regulatory elements, non-coding RNAs, and envelope-derived fusion proteins that influence gene expression, antiviral defense, and placental development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff7f347985d489007229fb4a5b8a3cd06f14b32" target='_blank'>
              Endogenous Retroviruses in Host-Virus Coevolution: From Genomic Domestication to Functional Innovation
              </a>
            </td>
          <td>
            Ruqi Jiang, Jingjun Zhou, Yue Liu, Guanjin Zhou, Dongdong Fan, Lixin Xiang, Ye Chen, Jianzhong Shao
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281dda7d54e63559872a9e12f7b32f74d4425ae4" target='_blank'>
              The tumor-associated fibroblasts regulate urothelial carcinoma progression.
              </a>
            </td>
          <td>
            Yu Xiao, Junfeng Yang, Mengjie Sun, Yongfu Li, Qinyin Liu, Jinjun Leng, Maolin Yang, Jinrui Wang, Hongju Li, Caifeng Yang, Changfen Luo, Jiahong Li, Longli Kang, Fen Huang, Yanhong Yu, Chuanmao Zhang
          </td>
          <td>2025-09-19</td>
          <td>Journal of molecular cell biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaa6c9f0d6e6a493ccdb8dd3165ee404adbcef4e" target='_blank'>
              Nuclear ribonucleoprotein condensates as platforms for gene expression regulation.
              </a>
            </td>
          <td>
            Sunkyung Choi, Kee K. Kim
          </td>
          <td>2025-08-04</td>
          <td>Genes & genomics</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Maintaining genome integrity is essential for the proper functioning and development of organisms. An intriguing aspect is that neocentromeres can form at non-centromeric sites. CENP-A, a key epigenetic marker of centromeres, is often mislocalized to ectopic sites in cancers when overexpressed. Its deposition on centromeres relies on transcription of centromeric non-coding RNAs. Subsequently, ectopic CENP-A is frequently found at transcriptionally active and chromosome breakpoint regions. We previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing PCAT2, a non-centromeric oncogenic lncRNA that recruits CENP-A to its transcribing locus. We tracked cells with this transgene to analyze the longevity of ectopic CENP-A. We discovered that this induced epigenetic memory was lost due to suppression by epigenetic silencing mechanisms, restoring CENP-A to previous levels. These findings suggest that cells have mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription unless selective pressure favors it">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7b7b4ef76af3d3bde1c36b2a65ab677cc6c45" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-07-31</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604d54b8cce75407680ad35bfbf26bb3a7a65d49" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification.
              </a>
            </td>
          <td>
            Jane E Blackmer, Erin A. Jezuit, Archan Chakraborty, Satya N Yalamanchi, Ruth A. Montague, Erin E. Dickert, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-07-31</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract CRISPR–Cas12a enzymes are RNA-guided nucleases widely used for programmable genome editing and diagnostics. Perfect complementarity between guide RNA and target DNA is essential for efficient binding and cleavage by Cas12a. However, we report that a particular ortholog of Cas12a, Acidaminococcus sp. Cas12a (AsCas12a), shows an unexpected tolerance to noncomplementary insertions at various positions in its DNA target. AsCas12a remains functional despite DNA bubbles or loops in the CRISPR–RNA (crRNA)–target DNA duplex, displaying both cis- and trans-cleavage activities even when the target harbors insertions of lengths 1–20 nucleotides in the crRNA-binding region. This activity is sequence-independent and works for ssDNA and is observed on dsDNA in vitro for specific insertion lengths/positions and DNA topologies but is strongly diminished in cells. Among 12 Cas12a orthologs tested, only AsCas12a exhibits this tolerance, making it a unique member of the Cas12a family. Structural analysis suggests a distinctive α-helix in AsCas12a’s WED domain is required for this flexibility. Upon deleting this α-helix, AsCas12a loses its ability to tolerate insertions. This discovery can be utilized to detect single-nucleotide polymorphisms and enable protospacer adjacent motif (PAM)–flexible DNA cleavage with Cas12a. Our findings expand our fundamental understanding of CRISPR–Cas12a systems. In conclusion, we uncover and characterize a unique property of AsCas12a to tolerate insertions in its target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f53d3b750d5121f2f4f8da7f48ee085b5fa25d6" target='_blank'>
              AsCas12a tolerates insertions in target DNA
              </a>
            </td>
          <td>
            Santosh R. Rananaware, Grace M. Shoemaker, B. Pizzano, Emma K. Vesco, L. S. W. Sandoval, Jordan G Lewis, August P Bodin, Sarah J Flannery, Ian H Lange, Dheeraj Pedada, Anne Fang, Sydney G. Antal, Daisy Aguilar, Noah R. Rakestraw, Vedant N Karalkar, Katelyn S. Meister, Long T. Nguyen, Piyush K Jain
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA transactions introduce torsional constraints that pose an inherent risk to genome integrity. While topoisomerase 1 (TOP1) activity is essential for DNA supercoil removal, the aberrant stabilization of TOP1:DNA cleavage complexes (TOP1ccs) can result in cytotoxic DNA lesions. What protects genomic hot spots of topological stress from excessive TOP1cc accumulation remains unknown. Here, we identify chromatin context as an essential means to coordinate TOP1cc resolution. Through its ability to bind poly(ADP-ribose) (PAR), the histone variant macroH2A1.1 facilitates TOP1cc repair factor recruitment and lesion turnover, thereby preventing DNA damage in response to transcription-associated topological stress. The alternatively spliced macroH2A1.2 isoform is unable to bind PAR or protect from TOP1ccs. Impaired macroH2A1.1 splicing, a frequent cancer feature, was predictive of increased sensitivity to TOP1 poisons in a pharmaco-genomic screen in breast cancer cells, and macroH2A1.1 inactivation mirrored this effect. We propose macroH2A1 alternative splicing as an epigenetic modulator of TOP1-associated genome maintenance and a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdec4fbd7a5287d669fa0b66871ebd9fb8a9c98" target='_blank'>
              Epigenetic control of topoisomerase 1 activity presents a cancer vulnerability
              </a>
            </td>
          <td>
            Tae-Hee Lee, Colina X Qiao, Vladislav Kuzin, Yuepeng Shi, Marina Farkas, Zhiyan Zhao, Vijayalalitha Ramanarayanan, Tongyu Wu, Tianyi Guan, Xianzhen Zhou, David Corujo, M. Buschbeck, Laura Baranello, Philipp Oberdoerffer
          </td>
          <td>2025-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA replication initiation is orchestrated in bacteria by the replication initiator DnaA. Two models for regulation of DnaA activity in Escherichia coli have been proposed: the switch between an active and inactive form, and the titration of DnaA on the chromosome. Although proposed decades ago, experimental evidence of a titration-based control mechanism is still lacking. Here, we first identified a conserved high-density region of binding motifs near the origin of replication, an advantageous trait for titration of DnaA. We then investigated the mobility of DnaA by visualising single proteins inside single cells of wild-type and deletion mutants E. coli strains, while monitoring cellular size and DNA content. Our results indicate that the chromosome of E. coli controls the free amount of DnaA in a growth rate-dependent fashion. Moreover, they address long-standing questions on the relevance of DnaA titration in stabilising DNA replication by preventing re-initiation events during slow growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9245293eecd0e45715a401c652d843bd210c464" target='_blank'>
              The Escherichia coli replication initiator DnaA is titrated on the chromosome
              </a>
            </td>
          <td>
            Lorenzo Olivi, Stephan Köstlbacher, C. Ludwig, Mees Langendoen, Nico J. Claassens, Thijs J. G. Ettema, J. van der Oost, P. R. ten Wolde, Johannes Hohlbein, R. Staals
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Post-transcriptional modifications on mRNA are crucial for mRNA fate and function. The current lack of a comprehensive method for high-coverage and sensitive quantitative analysis of mRNA modifications significantly limits the discovery of new mRNA modifications and understanding mRNA modifications’ occurrence, dynamics and function. Here, a highly sensitive, high-throughput and robust LC-MS/MS-based technique, mRQuant, was developed to simultaneously detect and quantify 84 modified ribonucleosides in cellular mRNA. Using mRQuant, we quantified 32–34 modified ribonucleosides across several human cancer and non-cancer cell lines and uncovered cancer- and cancer type-specific signatures. Analyses of cisplatin- and paclitaxel-treated HeLa cells and drug-resistant variants revealed several drug resistance-associated modifications. Among them, m1A exhibited significant differences across multiple cell types and between cancerous and non-cancerous cells. Knocking down mRNA m1A writer or eraser protein resulted in altered cell viability, cell cycle and apoptosis in HeLa cells, suggesting a role of mRNA m1A in cancer. Transcriptomic and proteomic analyses further revealed the molecular mechanisms underlying the phenotypic variation. Supplementary information The online version contains supplementary material available at 10.1007/s00018-025-05805-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b614524ca44941ff0254c44b4d860cba08475a2" target='_blank'>
              Quantifying the mRNA epitranscriptome reveals epitranscriptome signatures and roles in cancer
              </a>
            </td>
          <td>
            Ying Feng, Xiaoli He, Mingxin Guo, Ying Tang, Guantong Qi, Qian Huang, Wenran Ma, Hong Chen, Yifan Qin, Ruiqi Li, Jin Wang, Yu Liu
          </td>
          <td>2025-07-28</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H.A. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Double Homeobox 4 (DUX4) is a potent transcription factor encoded by a retrogene mapped in D4Z4 repeated elements on chromosome 4q35. DUX4 has emerged as pivotal in the pathomechanisms of facioscapulohumeral muscular dystrophy (FSHD), a relatively common hereditary muscle wasting condition, although classified as a rare disease. DUX4 contributes to zygote genome activation before its expression is repressed in most somatic tissues through epigenetic mechanisms, including DNA methylation and chromatin modifications. In FSHD, inappropriate activation of DUX4 expression is driven by a complex interplay of genomic and epigenetic alterations. The ectopic presence of DUX4 in skeletal muscle cells activates genes, viral elements and pathways that are typical of very early embryonic development, disturbing cell function and ultimately contributing to muscle weakness and wasting. This review first traces the history of DUX4, from the FSHD genetic linkage studies in the early 1990s, through to identification and characterization of the DUX4 gene in 1999. We then discuss the seminal studies that showed how and why DUX4 is expressed in FSHD and the effects of this ectopic expression in muscle, notably cellular toxicity. Other pathological roles of DUX4, such as participation in cancer and viral infection, are also highlighted. Maintenance of DUX4 in the genome was explained by discovery of the function of DUX4 in zygotic genome activation to institute the totipotent cells of the embryo. Thus, we encompass the gradual transition of DUX4 over the past 25 years from being considered a pseudogene in “junk DNA” to becoming central to understanding the molecular pathogenesis of FSHD and the primary focus for FSHD therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad639b9256d2a222879537cb6ffce2084c7d6dd4" target='_blank'>
              DUX4 at 25: how it emerged from “junk DNA” to become the cause of facioscapulohumeral muscular dystrophy
              </a>
            </td>
          <td>
            Alexandra Belayew, Alberto L. Rosa, Peter S. Zammit
          </td>
          <td>2025-08-25</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Genes encoding the RNA-binding proteins FUS, EWSR1, and TAF15 (FET proteins) are involved in chromosomal translocations in rare sarcomas. The encoded fusion oncoproteins typically include a DNA-binding domain and function as oncogenic transcription factors. FET-rearranged sarcomas are often aggressive malignancies affecting patients of all ages. Oncogenic fusion transcription factors are challenging to target directly, and new therapies are needed. Treatment with the small molecule SP-2509 results in reversal of the transcriptional activity of the FET fusion that causes Ewing sarcoma, EWSR1::FLI1. A similar molecule, seclidemstat (SP-2577), is currently in clinical trials for FET-rearranged sarcomas (NCT03600649), but its pharmacologic activity in FET fusion–positive sarcomas has not been demonstrated. We found potent seclidemstat cytotoxicity against both FET-rearranged and other fusion-positive sarcoma cell lines in vitro, including Ewing sarcoma, desmoplastic small round cell tumor, clear cell sarcoma, myxoid liposarcoma, and fusion-positive rhabdomyosarcoma. We also define transcriptomic effects of seclidemstat with bulk RNA sequencing. Seclidemstat recapitulated much of SP-2509 transcriptional activity in Ewing sarcoma. Widespread transcriptional changes were seen in all tested cell lines after seclidemstat treatment, regardless of the fusion protein expressed. This included reversal of FET fusion transcriptional signatures for EWSR1::WT1, EWSR1::ATF1, and EWSR1::ERG. Though novel inhibitors are unlikely to display single-agent efficacy in the clinic, these data suggest that seclidemstat remains a promising new treatment strategy for patients with FET-rearranged and other fusion-positive sarcomas. Significance: In this study, we show seclidemstat has in vitro activity in multiple rare and aggressive sarcomas caused by FET fusion proteins. With 13 RNA sequencing experiments, including multiple FET-rearranged sarcoma cell lines, this dataset is a rich resource for those studying FET-rearranged sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb78384576a9a5b2487874950835b687b754a22" target='_blank'>
              Seclidemstat (SP-2577) Induces Transcriptomic Reprogramming and Cytotoxicity in Multiple Fusion–Positive Sarcomas
              </a>
            </td>
          <td>
            G. Rask, C. Taslim, A. Bayanjargal, Rachel D. Dreher, Matthew V. Cannon, Julia Selich-Anderson, Jesse C. Crow, Aundrietta Duncan, Emily R. Theisen
          </td>
          <td>2025-08-25</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. H. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, Marwa A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The genomes of 43 distinct lactococcal strains were reconstructed by a combination of long- and short-read sequencing, resolving the plasmid complement and methylome of these strains. The genomes comprised 43 chromosomes of approximately 2.5 Mb each and 269 plasmids ranging from 2 to 211 kb (at an average occurrence of 6 per strain). A total of 953 antiphage genes representing 538 phage defence systems were identified in the 43 strains and were catalogued and cross-correlated with co-occurrent mobile elements, which indicated that almost 60% of these systems are predicted to be mobile. Detailed analysis established that restriction-modification (R-M) systems form a significant portion of this mobile phage defensome. As such, all detected Type I, II, and III-associated methylated motifs (46 of which were unique to this study) were matched to their corresponding methylating enzymes by homology detection or molecular cloning. The cumulative antiphage activity of selected systems and the ability of truncated R-M genes to contribute to methylation were demonstrated. This study reveals, for the first time, the dairy lactococcal plasmidome to be a rich reservoir of orphan HsdS-encoding genes, in a comprehensive survey of (mobile) phage defence systems in lactic acid bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6afd146a6e3d233d4624dbd25f3bf7a0d3905b" target='_blank'>
              Lactococcal mobile genetic elements harbour a diverse phage defensome rich in restriction-modification systems
              </a>
            </td>
          <td>
            Brian McDonnell, P. Kelleher, Alexey Fomenkov, Guillermo Ortiz Charneco, Keith Coughlan, Pascal Quénée, S. Kulakauskas, Christian Cambillau, Brian P. Anton, Paul P. de Waal, N. V. van Peij, Francesca Bottacini, J. Mahony, R. J. Roberts, D. van Sinderen
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-04</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Germline mutations in the DNA repair helicase XPD can cause the diseases xeroderma pigmentosum (XP) and trichothiodystrophy (TTD). XP patients bear an increased risk of skin cancer including melanoma. This is not observed for TTD patients despite DNA repair defects. To examine whether TTD cells harbor features counteracting tumorigenesis, we developed a TTD melanoma cell model containing the XPD variant R722W. Intriguingly, TTD melanoma cells exhibited reduced proliferation and an increased signature of the melanocyte lineage factor MITF, along with a strong basal upregulation of REDD2, an inhibitor of the mTOR/S6K/4EBP1-dependent messenger RNA (mRNA) translation machinery. REDD2 levels were partially driven by MITF and contributed to reduced melanoma proliferation. In a TTD model for melanocytes—the progenitor cells of melanoma—the MITF gene signature was also increased, but here without affecting REDD2 expression. However, ribosomal protein synthesis was reduced particularly in R722W melanocytes after UV stress, indicating a compromised mRNA translation machinery. Impaired translation was also demonstrated for the TTD XPD variant A725P, but not for an XP variant. Concludingly, the impaired translation and reduced fitness observed in TTD melanocytes and melanoma cells, particularly after UV stress, offer a possible explanation why TTD patients do not develop melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6262e92871597f9c2f4cf51c63e9ae2bb27bcd01" target='_blank'>
              Anti-tumorigenic properties by trichothiodystrophy mutations in melanocytic cells
              </a>
            </td>
          <td>
            Rupesh Paudel, Lena F Sorger, Anita Hufnagel, Mira Pasemann, Tamsanqa Hove, André Marquardt, Susanne Kneitz, Andreas Schlosser, Caroline Kisker, Jochen Kuper, Svenja Meierjohann
          </td>
          <td>2025-08-27</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Peroxisome proliferator-activated receptor γ (PPARγ), which is expressed in a variety of malignancies, governs biological functions through transcriptional programs. Defining the molecular mechanisms governing the selection of canonical versus non-canonical PPARγ binding sequences may provide the opportunity to design regulators with distinct functions and side effects. Acetylation at K268/293 in mouse Pparγ2 participates in the regulation of adipose tissue differentiation, and the conserved lysine residues (K154/155) in mouse Pparγ1 governs lipogenesis in breast cancer cells. Herein, the PPARγ1 acetylated residues K154/155 were shown to be essential for oncogenic ErbB2 driven breast cancer growth and mammary tumor stem cell expansion in vivo. The induction of transcriptional modules governing growth factor signaling, lipogenesis, cellular apoptosis, and stem cell expansion were dependent upon K154/155. The acetylation status of the K154/155 residues determined the selection of genome-wide DNA binding sites, altering the selection from canonical to non-canonical (C/EBP) DNA sequence-specific binding. The gene signature reflecting the acetylation-dependent genomic occupancy in lipogenesis provided predictive value in survival outcomes of ErbB2+ breast cancer. The Pparγ1 acetylation site is critical for ErbB2-induced breast cancer tumor growth and may represent a relevant target for therapeutic coextinction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bf36b7f21c4b94702b8969aa76f9a853c780b1" target='_blank'>
              PPARγ acetylation governs mammary adenocarcinoma tumor growth via acetylated residues that determine DNA sequence-specific binding
              </a>
            </td>
          <td>
            Lifeng Tian, X. Jiao, Chenguang Wang, Danni Li, A. Ertel, Joanna Achinger-Kawecka, S. Addya, Raymond E Soccio, Eric R. Chen, Balázs Győrffy, Gabriele Di Sante, Zhijiu Zhong, Haidar Alkhafaji, Nina Entcheva, Elyssa M. Campbell, Peter A McCue, Andrew V. Kossenkov, Rita Pancsa, Peter Tompa, Susan J. Clark, Richard G Pestell
          </td>
          <td>2025-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Targeting the DNA damage response (DDR) exhibits potent efficacy in inducing immune activation and enhancing patient prognosis. However, the benefits of DDR regulation are not universally observed across all patients, owing to the intricate compensatory mechanisms operative in certain cancers. There still exists a gap in the function of activated DDR protein in esophageal squamous cell carcinoma (ESCC). Here, we demonstrate that increased expression of DDR genes contributes to the progression of esophageal squamous cell carcinoma and suppresses the tumor immune microenvironment. Notably, the abundant presence of the DDR protein KIN in ESCC tissues facilitates efficient DNA damage clearance and promotes escape from apoptosis. Depletion of KIN significantly inhibited proliferation and induced DNA damage accumulation in ESCC cells. Mechanistically, KIN functions to support the recruitment of the R-loop regulator DHX9 to R-loop sites, thereby addressing DNA damage associated R-loops. Intriguingly, the depletion of KIN activates the STING pathway via NFκB signaling, which is induced by the accumulation of R-loops, ultimately initiating an innate immune response. Depletion of KIN improved the immune microenvironment and the effect of immune therapy in mouse model. Collectively, our findings identify KIN as a novel R-loop binding protein that facilitates the recruitment of the R-loop resolution complex and suppresses tumor-intrinsic innate immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e3c03a2576fc82b57363fed2fb18edfc9c71a5" target='_blank'>
              DNA/RNA-binding protein KIN17 supports esophageal cancer progression via resolving noncanonical STING activation induced by R-loop
              </a>
            </td>
          <td>
            Zichao Wei, N. Zhao, Lu Kuang, J. Cong, Sujuan Zheng, Yi Li, Zhihua Liu
          </td>
          <td>2025-08-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Radiotherapy is a mainstay of treatment for head and neck squamous cell carcinoma (HNSCC), either definitive or adjuvant to surgery. Biological factors known to affect radiation response are hypoxia and DNA repair capacity, but several lines of evidence indicate that other genes and pathways in the tumor cells might be involved that have not been elucidated. Here, we report the results of a genome-wide CRISPR-Cas9 functional genomics screen in HNSCC cells to identify radiosensitizing genes. Remarkably, microtubule organizing genes were identified with CLASP1 as most unexpected radiosensitizing hit. Clonogenic assay confirmed the radiosensitizing effect of CLASP1 knockout. Functional analysis showed that CLASP1 knockout has major impact during S-phase, and resulted in mitotic cells with broken chromosomes and cell death. CLASP1 and possibly the microtubule machinery in broader sense seem involved in protection of HNSCC cells against radiation–induced DNA damage. This newly identified mechanism provides an outlook for novel treatment approaches in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284fd55d39146101abef95bbef149eda9c32d030" target='_blank'>
              An unexpected role of CLASP1 in radiation response and S-phase regulation of head and neck cancer cells
              </a>
            </td>
          <td>
            R. D. de Roest, Marijke Buijze, Myrthe Veth, Klaas de Lint, G. Pai, M. Rooimans, Rob M. F. Wolthuis, A. Brink, J. Poell, R. Brakenhoff
          </td>
          <td>2025-08-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="


 Medulloblastomas, which occurs most commonly in children, is defined as an embryonal tumor derived from cells in the early stages of development. However, it is still unclear whether these embryonal tumors arise from embryonic cells in the uterus or as a result of single oncogenic event in fetal cells that rapidly progress to genetic instability. Meanwhile, potential transcription factor ZNF536 is most abundant in brain, especially in developing central nervous system. ZNF536 has been reported to have an inhibitory effect on neuronal differentiation, but the detailed regulatory mechanism has not been revealed. In particular, its pathological function in tumor development or progression of tumors is unknown. In this study, spontaneous tumor formation was observed in Zfp536+/- heteromice about 1 year old, and knock-down of ZNF536 in normal neuronal cells led cancerous phenotypes, such as neurosphere formation. Additionally, when comparing RNA expression levels in molecular genetic subgroups of medulloblastoma, ZNF536 expression was observed to be significantly reduced in group3, which is known to have the most malignant medulloblastoma. Based on our observation that loss of ZNF536 expression led spontaneous cancer development, we investigated the effect on cancer phenotypes after ZNF536 modulation. Besides, by performing ChIP-sequencing and RNA-sequencing, we identified the several downstream candidate targets of ZNF536 and further demonstrated comprehensive mechanisms of ZNF536 involved in cancer development. In summary, we suggested new factors including ZNF536 as a regulator of medulloblastoma development and progress, also we demonstrated regulation mechanisms by ZNF536. These results will provide insights understanding medulloblastoma development and new targeting strategies.



 Eun-Jin Yun and Seung Tae Baek. DISCOVERY OF NEW FACTORS INVOLVED IN MEDULLOBLASTOMA DEVELOPMENT AND INVESTIGATION OF ITS REGULATORY MECHANISM [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P77.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2442789bfe7e730bd10bb94e1ebf6df3c2d39113" target='_blank'>
              Abstract P77: DISCOVERY OF NEW FACTORS INVOLVED IN MEDULLOBLASTOMA DEVELOPMENT AND INVESTIGATION OF ITS REGULATORY MECHANISM
              </a>
            </td>
          <td>
            Eun-Jin Yun, Seung Tae Baek
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b570c21348c29c543cb996a976f642797b0edb0" target='_blank'>
              A SETD2-CDK1-lamin axis maintains nuclear morphology and genome stability.
              </a>
            </td>
          <td>
            Abid Khan, Cheng Zhang, Phu G Nguyen, James M. Metts, Lucas C. Collins, Kanishk Jain, C. A. Mills, Logan Vlach, Kelin Li, Amanda L. Brademeyer, Brittany M. Bowman, M. Major, Jeffrey Aubé, Laura E. Herring, W. Rathmell, Frank M. Mason, Ian J. Davis, Qing Zhang, B. Strahl
          </td>
          <td>2025-08-01</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Transcription factors (TFs) play a crucial role in tumorigenesis by driving oncogene expression in key signaling pathways. However, their small size and flat surfaces make them challenging targets for small-molecule inhibitors, while macromolecular therapies struggle to cross the cell membrane. Modulating TF activity at the genetic level offers a promising alternative. Antisense oligonucleotides (ASOs), which regulate protein expression by targeting mRNA, have emerged as effective therapeutics for previously undruggable proteins, including TFs. Over the past two decades, ASO therapeutics have advanced significantly, demonstrating long-lasting efficacy by promoting mRNA degradation. c-Myc, a key regulator of oncogene expression, drives cancer cell growth and proliferation but remains undruggable due to its nuclear localization and dynamic structure. In this study, we utilized our ASO development platform to design ASOs targeting c-Myc. Our sequence optimization algorithm achieved high accuracy, with one of three designed ASOs successfully silencing c-Myc. Ex vivo validation showed that ASO3 inhibited A549 cell growth with an IC50 of 152.5 nM. At the molecular level, ASO3 significantly reduced both c-Myc mRNA and protein expression. Functional assays, including trypan blue exclusion assay and CCK-8, confirmed that ASO3 decreased cell viability, suppressed proliferation, and induced apoptosis. These findings highlight ASO3's therapeutic potential and support further investigation as an anti-cancer agent targeting c-Myc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b33b3079149d5ba644ab8a2a4958404ec130feef" target='_blank'>
              Targeting c-Myc with antisense oligonucleotides to induce apoptosis in tumor cells.
              </a>
            </td>
          <td>
            Yuemei Ye, Yanhui Wang, Zhaoyun Zong, Shiyu Chen
          </td>
          <td>2025-08-12</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Initially, pseudogenes were considered to be “junk DNA”, and their biological role was unclear. However, some of the pseudogenes are engaged in the process of cancerogenesis and perform essential functions in competing for endogenous ribonucleic acid (ceRNA) networks and competing for RNA binding proteins (RBPs). They either positively or negatively regulate gene expression and act as suppressive and oncogenic transcripts. In this review, we look at some of the pseudogenes that play a role in the epithelial-to-mesenchymal transition (EMT) process and the maintenance of cancer-initiating cells (CIC), which are essential in understanding cancer development and progression metastasis and resistance to commonly used therapies. The group of discussed pseudogenes consists of CHIAP2, PTENP1, SUMO1P3, NANOGP8, OCT4-PG1/4, or HMGA1-P6, which are connected with different molecular pathways. Moreover, we discussed pseudogenes as potential diagnostic molecules that can be used as a new class of biomarkers. This potential usage may be valuable for oncology and personalized medicine in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8fe0d4350cd2131303d74c06087d56d8dfd818f" target='_blank'>
              Pseudogenes in the carcinogenesis: epithelial-to-mesenchymal transition process and cancer initiating cells
              </a>
            </td>
          <td>
            Tomasz Kolenda, Piotr Chałaj, Aleksandra Cichowicz, Aleksandra Trojańska, Andrzej Bałoniak, Malwina Kwaśniewska, Magda Odrobińska, Kacper Guglas, Joanna Kozłowska-Masłoń, Paulina Gieremek, Paulina Poter, Marlena Janiczek-Polewska, Anna Florczak-Substyk, Anna Przybyła, Patrycja Mantaj, Katarzyna Regulska, Beata J Stanisz, Zefiryn Cybulski, U. Kazimierczak
          </td>
          <td>2025-08-08</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Systematic discovery of transcription factor (TF) landscapes in low-input samples and at single cell level is a major challenge in the fields of molecular biology, genetics, and epigenetics. Here, we present cleavage under Dynamic targets and Tagmentation (DynaTag), enabling robust mapping of TF-DNA interactions using a physiological salt solution during sample preparation. DynaTag uncovers occupancy alterations for 15 TFs in stem cell and cancer tissue models. We highlight changes in TF-DNA binding for NANOG, MYC, and OCT4, during stem-cell differentiation, at both bulk and single-cell resolutions. DynaTag surpasses CUT&RUN and ChIP-seq in signal-to-background ratio and resolution. Furthermore, using tumours of a small cell lung cancer model derived from a single female donor, DynaTag reveals increased chromatin occupancy of FOXA1, MYC, and the mutant p53 R248Q at enriched gene pathways (e.g. epithelial-mesenchymal transition), following chemotherapy treatment. Collectively, we believe that DynaTag represents a significant technological advancement, facilitating precise characterization of TF landscapes across diverse biological systems and complex models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a8af886e5fec85aa2e58ed1b6a6ad04cf228fa" target='_blank'>
              DynaTag for efficient mapping of transcription factors in low-input samples and at single-cell resolution
              </a>
            </td>
          <td>
            Pascal Hunold, Giulia Pizzolato, Nadia Heramvand, L. Kaiser, Giulia Barbiera, Olivia van Ray, Roman K Thomas, Julie George, M. Peifer, Robert Hänsel-Hertsch
          </td>
          <td>2025-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Summary/Abstract Glioblastoma stem cell (GSC) cultures are initiated from glioblastoma (GBM) surgical resection tissue. They can capture and propagate key GBM primary tumor molecular and cellular features. We have deeply characterized four isocitrate dehydrogenase (IDH)-expressing (or IDH+) GSC cultures from unrelated adults to serve as cellular models for the majority of adult primary GBM. We demonstrate that GSC cultures can be continuously propagated in defined, serum-free media and 5% oxygen without requiring specialized growth substrates; have well-defined genomic and mtDNA variants and gene/protein expression profiles; and highly reproducible dose-survival curves when treated with the GBM standard-of-care therapies of ionizing radiation (IR) and temozolomide (TMZ). We also illustrate how expressed lentiviral barcodes, mtDNA variants and single cell gene expression profiling can be used to define and track cellular heterogeneity over 40 days after IR treatment. These well-characterized IDH+ GSC cultures can support many high throughput in vitro assay formats, including xenograft, organoid and other GBM disease modeling protocols. They should prove a useful resource to better understand GBM biology, and to identify new and more effective GBM therapies and treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b852883dfd3891f1795ac6382d7546d2b8963f5d" target='_blank'>
              Cellular heterogeneity and therapeutic response profiling of human IDH+ glioma stem cell cultures
              </a>
            </td>
          <td>
            N. Chambwe, Scott R. Kennedy, Brendan F. Kohrn, Pavlo Lazarchuk, Mario Leutert, Guangrong Qin, Bahar Tercan, M. Sanchez-Contreras, Weiliang Tang, Jerome J. Graber, Patrick J. Paddison, Judit Villén, Ilya Shmulevich, Raymond J Monnat
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract UVB radiation (280–320 nm) from sunlight induces skin cancer through DNA damage-induced mutations. Melanomas carry mutational signatures associated with UVB-induced cyclobutane pyrimidine dimers (CPDs). However, there are several other melanoma signatures of unknown origin. To test if these signatures are linked to UVA, we exposed human melanocytes to UVA (340–400 nm) and to UVB for comparison. We mapped DNA damage in the form of CPDs or 8-oxoguanines (8-oxoG) genome-wide at base resolution. We then determined mutational patterns in single melanocyte cell clones by whole genome sequencing. Different from UVB, UVA induces CPDs more selectively at TT sequences resembling melanoma signature SBS7d. We did not observe rising CPD levels after cessation of radiation (dark CPDs). The UVA-induced CPDs were not mutagenic in the mutation analysis. 8-oxoG was present in melanocytes but did not substantially increase after UVA. G/C to T/A mutations were prominent in melanocyte single-cell clones with no major shift after UVA radiation. These mutations matched SBS18, a signature present in melanomas. Although UVA damages DNA, it has a surprisingly limited mutagenic effect on human melanocytes. However, the oxidative base lesions in melanocytes and their associated mutations may be linked to a subset of melanoma mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b21e19c0b878b06400096dc62c3652b28e09c65" target='_blank'>
              UVA-induced DNA damage and mutations in human melanocytes: relevance for melanoma mutations
              </a>
            </td>
          <td>
            , Jennifer Johnson, Paul Y Kim, Gerd P Pfeifer
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Specialized DNA polymerases facilitate various cellular processes. Despite extensive research, the mutagenic effects of these error-prone enzymes on genomes are not fully understood. Here we show that Pol IV promotes genomic instability in Pseudomonas aeruginosa by misincorporating oxidized guanine nucleotides. This activity led to a distinctive mutational signature, characterized by A-to-C transversions occurring preferentially at AT sites flanked by a 5’G and/or 3’C. Furthermore, Pol IV preferentially targeted pathogenicity genes located at specific chromosomal locations near the replication termination region and rRNA-encoding operons. Half of the mutation events catalyzed by Pol IV impaired gene function. This can be attributed to the bias of Pol IV for mutating codons with its preferred sequence contexts, leading to substitutions to unreactive alanine and glycine residues. Remarkably, mutation signatures identified for Pol IV were found in clinical isolate genomes of P. aeruginosa, providing compelling evidence for its role in genetic diversification during pathogen adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f7c02761e12b8b73eebc31b5beda14fbe3ac0c" target='_blank'>
              The low-fidelity DNA Pol IV accelerates evolution of pathogenicity genes in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Sofía D. Castell, Consuelo M. Fernandez, Ignacio N. Tumas, L. M. Margara, María C. Miserendino, Danilo Ceschin, R. Pezza, Mariela R. Monti
          </td>
          <td>2025-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Bladder cancer pathogenesis is closely linked to epigenetic alterations, particularly DNA methylation and demethylation processes. Environmental carcinogens and persistent inflammatory stimuli—such as recurrent urinary tract infections—can induce aberrant DNA methylation, altering gene expression profiles and contributing to malignant transformation. This review synthesizes current evidence on the role of DNA methyltransferases (DNMT1, DNMT3a, DNMT3b) and the hypermethylation of key tumour suppressor genes, including A2BP1, NPTX2, SOX11, PENK, NKX6-2, DBC1, MYO3A, and CA10, in bladder cancer. It also evaluates the therapeutic application of DNA-demethylating agents such as 5-azacytidine and highlights the impact of chronic inflammation on epigenetic regulation. Promoter hypermethylation of tumour suppressor genes leads to transcriptional silencing and unchecked cell proliferation. Urine-based DNA methylation assays provide a sensitive and specific method for non-invasive early detection, with single-target approaches offering high diagnostic precision. Animal models are increasingly employed to validate these findings, allowing the study of methylation dynamics and gene–environment interactions in vivo. DNA methylation represents a key epigenetic mechanism in bladder cancer, with significant diagnostic, prognostic, and therapeutic implications. Integration of human and experimental data supports the use of methylation-based biomarkers for early detection and targeted treatment, paving the way for personalized approaches in bladder cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f706e4ed85cef4708cafccd425d482bfabf2666" target='_blank'>
              DNA Methylation in Bladder Cancer: Diagnostic and Therapeutic Perspectives—A Narrative Review
              </a>
            </td>
          <td>
            D. Puia, M. Ivănuță, C. Pricop
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In mammals, primordial germ cells (PGCs) undergo global erasure of DNA methylation with delayed demethylation of germline genes and selective retention of DNA methylation at evolutionarily young retrotransposons. However, the molecular mechanisms of persistent DNA methylation in PGCs remain unclear. Here we report that resistance to DNA methylation reprogramming in PGCs requires UHRF2, the paralog of the DNMT1 cofactor UHRF1. PGCs from Uhrf2 knock-out mice show loss of retrotransposon DNA methylation, while DNA methylation is unaffected in somatic cells. This is not associated with changes in the expression of retrotransposons in E13.5 PGCs, indicating that other mechanisms compensate for retrotransposon control at this stage. Furthermore, Uhrf2-deficient PGCs show precocious demethylation of germline genes and overexpress meiotic genes in females. Subsequently, Uhrf2-deficient mice show impaired oocyte development and female-specific reduced fertility, as well as incomplete remethylation of retrotransposons during spermatogenesis. These findings reveal a crucial function for the UHRF1 paralog UHRF2 in controlling DNA methylation in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea0f6cead9b7479f9deb5d526e8e54ad404a663" target='_blank'>
              UHRF2 mediates resistance to DNA methylation reprogramming in primordial germ cells
              </a>
            </td>
          <td>
            Ambre Bender, Marion Morel, Michael Dumas, M. Klopfenstein, Naël Osmani, Maxim V. C. Greenberg, Déborah Bourc'his, Norbert B Ghyselinck, Michael Weber
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="INTRODUCTION
The epigenetic silencing of O6-methylguanine DNA methyltransferase (MGMT) is associated with reduced DNA repair capacity, carcinogenesis and increased sensitivity to alkylating chemotherapy. However, the biological role and clinical significance of MGMT overexpression in cancer remains poorly understood.


METHODS
Using multiplexed quantitative immunofluorescence we measured the localized levels of MGMT protein, γH2AX and CD8+ T cells in multiple retrospective colorectal cancer (CRC) cohorts. Genomic and transcriptomic features of selected cases were also studied with whole exome DNA sequencing and genome-wide methylation analysis. MGMT-methylated human CRC cells SW620 were transfected with an MGMT-containing plasmid and co-cultured with allogeneic peripheral blood mononuclear cells.


RESULTS
A subset of CRCs showed MGMT protein upregulation associated with lower γH2AX, reduced CD8+ tumor infiltrating lymphocytes (TILs), mismatch repair proficient (pMMR) status and shorter survival. CD8+ TILs were more distant from MGMT-expressing cells than MGMT-negative cells and the MGMT promoter methylation status did not highly correlate with MGMT protein levels in CRC. In genomic/transcriptomic analysis, high MGMT expression was associated with a lower nonsynonymous somatic mutational burden, higher transition-to-transversion mutation ratio, increased deleterious TP53 variants and distinct transcriptomic profiles. The exogenous expression of MGMT in SW620 CRC cells reduced the number of spontaneous nonsynonymous mutations, reproduced mutational features of MGMT-high CRC and limited the in vitro T-cell-mediated killing of malignant cells induced by proinflammatory cytokines in tumor/immune cell co-cultures.


CONCLUSIONS
MGMT overexpression identifies a previously undescribed subset of CRCs with distinct biological and clinical properties including reduced mutagenesis, adaptive immune evasion, predominantly pMMR phenotype and aggressive clinical course. Direct, quantitative assessment of MGMT protein expression using spatially resolved analysis is more reliable than inference of MGMT expression by promoter methylation status in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cc0eaa491cd34a4c5b928fccd34ad143431db3" target='_blank'>
              High MGMT expression identifies aggressive colorectal cancer with distinct genomic features and immune evasion properties.
              </a>
            </td>
          <td>
            J. Zhang, B. K. Rajendran, Shruti S Desai, Joanna Gibson, Jassim DiPalermo, Patricia LoRusso, Yong Kong, Hongyu Zhao, Michael Cecchini, K. Schalper
          </td>
          <td>2025-09-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, Hafiz Muhammad Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) serve as important drivers mediating polyploidization events and phenotypic diversification in plant genomes. However, the dynamic changes in various TE subclasses post-polyploidization and their mechanisms of influencing phenotypic variation require further investigation. The allopolyploid Gossypium species, originating from two diploid progenitors, provide an ideal model for studying TE dynamics following polyploidization. This study investigated TE dynamics post-polyploidization based on 21 diploid and 7 polyploid cotton genomes. The Tekay subclass of the Gypsy serves as a major driver of Gossypium genome evolution, as it underwent two burst events in the At-subgenome and its progenitor, exhibiting the highest abundance, longest length, and largest proportion among all TE subclasses. In contrast, the Gopia superfamily Tork subclass has lower abundance but greater genic association, facilitating environmental adaptation and phenotypic variation. Additionally, a pan-TE-related structural variation, the pan-TRV map, was constructed by integrating resequencing data from 256 accessions. Genome-wide analysis of 28 cotton genomes identified 142,802 TRVs, among which 72,116 showed polymorphisms in the 256 G. hirsutum accessions. The Gypsy superfamily, particularly the Tekay subclass, has been identified as a major source of TRVs, while Copia-type elements demonstrate significantly greater enrichment in gene-proximal genomic regions. A total of 334 TRVs exhibiting statistically significant associations with 10 key phenotypic traits, including 164 TRVs affecting yield components and 170 TRVs determining fiber quality. This investigation delineates the evolutionary significance of transposable elements in Gossypium genome diversification while simultaneously providing novel functional markers and potential editing targets for genetic dissection and molecular breeding of key agronomic traits in cotton.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae18df9212dca9b2e483245a02819881c81bf4b6" target='_blank'>
              Transposon Dynamics Drive Genome Evolution and Regulate Genetic Mechanisms of Agronomic Traits in Cotton
              </a>
            </td>
          <td>
            Zeyu Dong, Shangkun Jin, Yupeng Hao, Ting Zhao, Haihong Shang, Zhiyuan Zhang, Lei Fang, Zhihong Zheng, Jun Li
          </td>
          <td>2025-08-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dysregulated expression of circular RNAs (circRNAs) has been implicated in the initiation and progression of various diseases, including cancer. In this study, we investigated the role of hsa_circ_0005571 in modulating the biological characteristics of colorectal cancer (CRC) cells. Initially, the circRNA expression profiles of CRC tissues and corresponding normal tissues were analyzed using bioinformatics to identify differentially expressed circRNAs. Subsequently, quantitative real-time PCR (qRT‒PCR) was used to validate the expression levels of hsa_circ_0005571 in both CRC tissues and cell lines. In vitro assays, including the Cell Counting Kit-8 (CCK-8), colony formation, 5-ethynyl-2′-deoxyuridine (EdU) incorporation, wound healing, and Transwell invasion assays, were employed to assess the effects of hsa_circ_0005571 on CRC cell proliferation and invasion. Additionally, tumorigenic potential was examined through in vivo tumorigenesis experiments in nude mice. Bioinformatic predictions and experimental evidence indicated that hsa_circ_0005571 may function as a sponge for miR-520f-3p, which was found to be downregulated in CRC cells. Moreover, LRP8 was predicted and confirmed as a direct downstream target of miR-520f-3p, with both the mRNA and protein levels of LRP8 significantly elevated in CRC cells. Overall, our results suggest that hsa_circ_0005571 enhances CRC proliferation and invasion by modulating the miR-520f-3p/LRP8 axis. Furthermore, elevated hsa_circ_0005571 expression in CRC tissue samples and cell lines relative to that in normal controls was positively correlated with metastasis (p = 0.029) and advanced clinical stage (p = 0.006). Survival analyses revealed that CRC patients with high levels of hsa_circ_0005571 expression had significantly lower overall survival rates than did those with low expression (p < 0.05). Functional assays confirmed that hsa_circ_0005571 facilitates CRC cell proliferation and migration, whereas its knockdown results in reduced migration, proliferation, and tumorigenic potential in vivo. Finally, Western blot analyses demonstrated that LRP8 expression varied in accordance with the levels of hsa_circ_0005571 and miR-520f-3p, further confirming the involvement of the hsa_circ_0005571/miR-520f-3p/LRP8 regulatory axis in CRC. Our study suggested that hsa_circRNA_0005571 promotes proliferation and migration through the miR-520f-3p/LRP8 axis in CRC. Consequently, hsa_circ_0005571 may represent a promising novel gene target for the diagnosis and treatment of CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16450-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3bdc932b4f1529cee6d8b185f8814c7089e79a" target='_blank'>
              Hsa_circ_0005571 promotes the proliferation and invasion of colorectal cancer cells
              </a>
            </td>
          <td>
            Haipeng Ge, Jiaxuan Zhang, Ran Tao, Li Ding, Song Jiang, Jiale Shen, L. Bian, Jun-Jiao Qin
          </td>
          <td>2025-08-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila.
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Elucidating the mechanisms of radioresistance in highly radiotolerant organisms can provide valuable insights into the adaptation and evolution of organisms. However, research has been limited on many naturally occurring radioresistant organisms due to a lack of information regarding their genetic and biochemical characteristics and the difficulty of handling them experimentally. To address this, we conducted an experiment on adaptive evolution using gamma radiation as the selection pressure to generate evolved Escherichia coli with gamma radiation resistance approximately one order of magnitude greater than that of wild-type E. coli. Gene expressions in all wild-type and evolved radioresistant E. coli in the presence or absence of gamma irradiation were analyzed and compared using RNA sequencing. Under steady-state conditions, the genes involved in survival, cell recovery, DNA repair, and response following stress exposure were upregulated in evolved E. coli compared with those in wild-type E. coli. Furthermore, the evolved E. coli induced these genes more efficiently following gamma irradiation and greater DNA repair activity than that in the wild-type E. coli. Our results indicate that an increased steady-state expression of various anti-stress genes, including DNA repair-related genes, and their highly efficient induction under irradiation are responsible for the remarkable radioresistance of evolved E. coli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13cc80bdc29b64dbce3840bea6a235a4ab6a6a91" target='_blank'>
              Acquired Radioresistance Through Adaptive Evolution with Gamma Radiation as Selection Pressure: Increased Expression and Induction of Anti-Stress Genes
              </a>
            </td>
          <td>
            Takeshi Saito, Hiroaki Terato
          </td>
          <td>2025-07-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Survival strategy of bacteria is expanded by extrachromosomal elements (ECEs). However, their genetic diversity and functional roles for adaptability are largely unknown. Here, we discover a novel family of intracellular ECEs using 56 saliva samples by developing an efficient microbial DNA extraction method coupled with long-read metagenomics assembly. Even though this ECE family was not hitherto identified, our global prevalence analysis using 476 salivary metagenomic datasets elucidates that these ECEs reside in 74% of the population. These ECEs, which we named, “Inocles”, are giant plasmid-like circular genomic elements of 395 kb in length, including Streptococcus as a host bacterium. Inocles encode a series of genes that contribute to intracellular stress tolerance, such as oxidative stress and DNA damage, and cell wall biosynthesis and modification involved in the interactions with oral epithelial cells. Moreover, Inocles exhibit significant positive correlations with immune cells and proteins responding to microbial infection in peripheral blood. Intriguingly, we examine and find their marked reductions among 68 patients of head and neck cancers and colorectal cancers, suggesting its potential usage for a novel biomarker of gastrointestinal cancers. Our results suggest that Inocles potentially boost the adaptive capacity of host bacteria against various stressors in the oral environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ef6b2491714273e3a5acd966b2b147c6dd3e3d" target='_blank'>
              Giant extrachromosomal element “Inocle” potentially expands the adaptive capacity of the human oral microbiome
              </a>
            </td>
          <td>
            Yuya Kiguchi, Nagisa Hamamoto, Y. Kashima, L. Runtuwene, A. Ishizaka, Yuta Kuze, T. Enokida, N. Tanaka, Makato Tahara, S. Kageyama, T. Fujisawa, Riu Yamashita, A. Kanai, Josef S. B. Tuda, Taketoshi Mizutani, Yutaka Suzuki
          </td>
          <td>2025-08-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="RNAs adopt heterogenous folded structures that are essential for function and thus play critical roles in cellular biology. An example of this is the ribosome, a complex, three-dimensionally folded macromolecular machine that coordinates protein synthesis. Advances in RNA biochemistry, structural and molecular biology, and bioinformatics have unveiled other non-coding RNAs whose functions are dictated by their structure. It is not surprising that aberrantly folded RNA structures enable disease. 
R-loops are nucleic acid structures encompassing an RNA-DNA hybrid and a displaced single-stranded DNA (ssDNA) (Hegazy et al., 2019). Once thought to be uncommon transcriptional byproducts, but reclassified as prevalent and functionally significant in a variety of eukaryotic and prokaryotic species. During transcription, R-loops naturally form when the developing RNA hybridizes with the DNA template strands; displacing the non-template strand and creating a three-stranded structure. RNA loops are fundamental secondary structural elements that play crucial roles in RNA function and interaction within cells. Recent developments in immunoprecipitation techniques (e.g., DRIP-seq) and high-throughput sequencing have made it possible to profile R-loops throughout the genome, demonstrating their occurrence in both coding and non-coding areas. The RNA loops are involved in a wide range of biological processes spanning genome regulation, physiology and repair. Mounting evidence also supports R-loop deregulation as a frequent, initiating, event during the development of several human pathologies, such as cancer and neurological disorder. Several applications of RNAloop in biomedical research, clinical and diagnostics underscores their application potential. With this background, we update the current available literature nd review diverse aspects of RNAloops traversing from their roles in cell biology and physiology, genome regulation and instability, the cellular mechanisms and methods of detection. Finally, we in detail curate literature of their roles in Human diseases with emphasis on neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3db695265d37192e791c7dff9f5cd030549c5d" target='_blank'>
              RNA-Loops (DNA/RNA Hybrids) Mechanisms, Genomic and Cellular Roles-A Review with Emphasis on Neuropsychiatric Diseases
              </a>
            </td>
          <td>
            Kiran Kumar H.B, Mahalakshmi B.R, Priya M.D., Rajesha S, R. H. S, Latha K, Ranjini. P, Shankar J, K. R. Kini, S.Chandra Nayak, Shailasree Sekhar
          </td>
          <td>2025-08-07</td>
          <td>Journal of Applied Life Sciences International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Mucosa-associated invariant T (MAIT) cells represent a unique population of innate-like T lymphocytes capable of detecting non-peptide antigens in the context of monomorphic antigen-presenting molecules. Due to their abundance in barrier tissues, reactivity to local inflammatory cues, and cytotoxic and regulatory functions, MAIT cells are poised to shape the dynamics of various tumor microenvironments. Growing evidence suggests that MAIT cells can exert protumor and/or antitumor effects in cancers arising from or metastasizing to mucosal tissues. However, MAIT cell roles in bladder cancer (BCa) remains unclear. Methods To begin to identify MAIT cells in BCa, we stained bladder tumor biopsies for T cell receptor (TCR) Vα7.2+ cells. We then refined a human MAIT cell signature, which enabled us to interrogate a bulk RNA sequencing dataset and conduct correlation analyses linking intratumoral MAIT cell abundance and mortality from BCa. To extend our work to an in vivo setting, we employed a clinically relevant mouse model in which Mr1+/+ B6-MAITCAST (MAIT-sufficient) and Mr1−/− B6-MAITCAST (MAIT-deficient) mice were exposed to N-butyl-N-(4-hydroxybutyl)nitrosamine, a chemical carcinogen associated with tobacco smoke. In additional experiments, MAIT cells were functionally removed through acetyl-6-formylpterin (Ac-6-FP) administration. Effector and regulatory cell types were phenotyped by flow cytometry, and BCa tumor burden and progression were assessed by MRI and/or H&E and Ki67 staining. Results TCR Vα7.2+ cells were readily detectable in several BCa biopsies, and our bioinformatic analyses correlated heavier MAIT cell presence in BCa tumors with poorer overall survival. Similarly, we found higher tumor burdens in Mr1+/+ B6-MAITCAST mice than in Mr1−/− or Ac-6-FP-treated animals. Bladder MAIT cells from tumor-bearing mice exhibited phenotypic MAIT17 bias based on transcription factors they harbored along with increased interleukin-17A and tumor necrosis factor-α production capacities upon stimulation. Finally, FoxP3+ regulatory T (Treg) cell frequencies were elevated in Mr1+/+ mouse bladder tumors, likely contributing to an immunosuppressive tumor microenvironment, a finding that could be recapitulated in our transcriptomic studies on human BCa. Conclusions MAIT cells are abundant in BCa tumor microenvironments where they potentiate Treg cell accumulation and play protumor roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/904ce037c95ea168f087ef993f969eaddff0121a" target='_blank'>
              MAIT cells promote cancer progression and regulatory T cell accumulation in bladder tumor microenvironment
              </a>
            </td>
          <td>
            Inbar A Habaz, Xuejin Ou, Nicole I Wang, Angela Schincaglia, Valeryia Shydlouskaya, Agetha Mahendran, F. Martinez-Santiesteban, Dylan Doolabi, Yige Bao, Ahmad R Movasseghi, Julius Haruna, Nicholas E. Power, Timothy J Scholl, Melissa J Huynh, Alp Sener, S. M. Haeryfar, Dr S M Mansour Haeryfar
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The bacterial cytoplasm is characterized by a distinctive membrane-less organelle, the nucleoid, which harbors chromosomal DNA. We investigate the effects of dynamic processes associated with transcription and translation on the structure of this organelle, using coarse-grained molecular dynamics (MD) simulations implemented with out-of-equilibrium reactions. Our model captures the scale of the entire cell and incorporates a reaction-diffusion system for ribosomes and polyribosomes, combining their out-of-equilibrium dynamics with excluded volume interactions with DNA. Our findings demonstrate that out-of-equilibrium reactions increase the size of the nucleoid. In addition, we show that the nucleoid size increase is proportional to transcriptional activity. Our model reproduces the time-dependent change in nucleoid size observed in rifampicin treatment experiments, where the pool of polyribosomes is depleted. Furthermore, we find these active processes are essential for complete sister chromosome separation and correct nucleoid positioning within the cell. Overall, our study reveals the effects of the central dogma processes on the internal organization and localization of bacterial nucleoids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5869c17c0b2f03dcafbf482f5c2ecb59859979e3" target='_blank'>
              Effects of central dogma processes on the compaction and segregation of bacterial nucleoids
              </a>
            </td>
          <td>
            Mu-Hung Chang, M. Lavrentovich, Jan-Michael Y. Carrillo, Jaan Männik
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d05a5f9576a9619318986d87c824d703b9e97b8" target='_blank'>
              Targeted DNA ADP-ribosylation triggers templated repair in bacteria and base mutagenesis in eukaryotes.
              </a>
            </td>
          <td>
            Darshana Gupta, Constantinos Patinios, H. V. Bassett, A. Kibe, Scott P. Collins, Charlotte Kamm, Yanyan Wang, Chengsong Zhao, Katie Vollen, C. Toussaint, Irene Calvin, G. Cullot, Eric J. Aird, Kathryn M. Polkoff, T. Nguyen-Vo, Angela Migur, Friso T Schut, Ibrahim S. Al’Abri, Tatjana Achmedov, Alessandro Del Re, Jacob E. Corn, A. Saliba, Nathan C. Crook, A. Stepanova, J. M. Alonso, Chase L. Beisel
          </td>
          <td>2025-09-04</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive subtype of breast cancer with high metastatic potential, limited treatment options, and low patient survival rates. By combining functional proteomics and genomics approaches, we identified an oncogenic transcriptional network in mesenchymal and invasive TNBC involving the glucocorticoid receptor (GR), GATA6, MYC, and AP-1 transcription factors. Although these transcription factors bound extensively to shared enhancers, they utilized different enhancer repertoires from this shared enhancer pool to drive distinct downstream oncogenic pathways. The SWI/SNF chromatin remodeling complex was a common gatekeeper of chromatin access for these transcription factors, and SWI/SNF inhibition decommissioned the distinct enhancer networks and pro-tumorigenic gene programs they control. Consistently, SWI/SNF inhibition repressed TNBC proliferation, glucocorticoid-induced chemoresistance, and invasion in vitro and suppressed tumor growth and metastasis in vivo. Thus, these findings indicate that SWI/SNF-mediated chromatin remodeling drives TNBC progression, positioning SWI/SNF as a promising therapeutic target in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5aecd9fb0bc36400b799414dcc4be0d90b23c11" target='_blank'>
              SWI/SNF Functions as a Gatekeeper of Enhancer Chromatin Access to Control Progression of Mesenchymal Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Rikkemaja Isahbell J Møldrup, M. Terp, S. T. Jakobsen, Tina Ravnsborg, M. S. Madsen, Daniel D Brown, Laurits R Sloth, Michael C. Gorry, S. Oesterreich, Adrian V. Lee, Henrik J Ditzel, Ole N. Jensen, Rasmus Siersbæk
          </td>
          <td>2025-09-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background DNA changes induced by stress may be stable through cell division and passed to subsequent generations. Plant improvement programs require that individuals develop a new heritable trait. The action of mutagens applied in classical mutagenesis is known from their transgenerational inheritance, observed as mutations. DNA methylation is an epigenetic mechanism that generates heritable phenotypic variation by influencing gene expression and modifying DNA accessibility to mutagens. Fagopyrum esculentum (common buckwheat) and F. tataricum (Tartary buckwheat) are essential for food production and valuable for medical purposes. Results In this study, we aimed to investigate the transgenerational changes in the genome and epigenome of two Fagopyrum species, by analysing 1st and 3rd generations, followed by treatment with gamma rays. The genomic instability, observed as DNA fragmentation, micronuclei, and changes in cell cycle profile, was observed in 1st generation, whereas 3rd generation shows recovery in DNA stability. Gamma-ray stress caused alterations in the pattern and level of DNA methylation in the roots of both analysed generations. The response to gamma rays depends on the species – in 1st generation, a higher increase in DNA methylation level in F. esculentum and only slight changes in F. tataricum were observed. The DNA methylation level in the 3rd generation changed only in F. esculentum. Conclusions Gamma ray is a stress factor that affects the pattern and level of DNA methylation in F. esculentum and F. tataricum. Significant differences were observed between control and 3rd generation only in F. esculentum. Differences in DNA methylation following gamma irradiation between the analysed Fagopyrum species may result from various forms of pollination in the analysed species. By using gamma ray-induced mutation, we got a highly stable mutant of F. tataricum, which may result in a self-pollinating trait. Understanding the changes in DNA methylation following mutagenic treatment can be essential in a breeding programme. Supplementary Information The online version contains supplementary material available at 10.1186/s12870-025-07033-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c669739a40e63380b2299a72c170061851584f" target='_blank'>
              Transgenerational genetic and epigenetic changes induced by gamma-ray in Fagopyrum species
              </a>
            </td>
          <td>
            Katarzyna Sala-Cholewa, Magdalena Rojek-Jelonek, Jolanta Kwasniewska
          </td>
          <td>2025-07-31</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The type II topoisomerases, gyrase and topoisomerase IV, are essential enzymes in nearly all bacteria and are the targets of fluoroquinolones, which are some of the most widely prescribed broad-spectrum antibacterials in clinical use. As part of their catalytic cycle, gyrase and topoisomerase IV transiently cleave DNA in a sequence-dependent manner. However, it is unclear whether this sequence-dependence is species-specific. Therefore, using our recently developed SHAN-seq method, we mapped and compared cleavage sites for type II topoisomerases from three different pathogenic bacterial species, Escherichia coli, Bacillus anthracis, and Mycobacterium tuberculosis in the presence of the fluoroquinolone, ciprofloxacin. We found that the enzymes have substantially different DNA cleavage specificities that vary between gyrase and topoisomerase IV, across species, with supercoil chirality, and in response to ciprofloxacin. Our results demonstrate that bacterial species fine-tune the DNA cleavage specificity of their type II topoisomerases. This finding suggests that cleavage specificity may play important physiological roles and, in turn, may affect the susceptibility of bacteria to fluoroquinolone antibacterials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e082902e5461cf61bfe338f2489bf47116a6cee" target='_blank'>
              Bacterial type II topoisomerases cleave DNA in a species-specific manner
              </a>
            </td>
          <td>
            Ian L. Morgan, Jeffrey Y Jian, N. Osheroff, K. Neuman
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Extrachromosomal and mobile genetic elements, including plasmids, and accessory chromosomes, are prevalent in all life domains. Elements integrated into the host chromosome replicate and segregate via the host life cycle. In contrast, the persistence of autonomously replicating elements relies on their ability to remain within the host population. Here, we compare the evolutionary advantage of different persistence strategies found in prokaryotic plasmids. Through intracellular competitions between plasmid genotypes, we find that the combination of active partitioning during cell division with a toxin-antitoxin (TA) system for post-segregational killing increases plasmid fitness more than either strategy alone. Mathematical modeling of long-term plasmid evolution, calibrated with empirical plasmid loss dynamics, further supports these findings. A survey of enterobacterial genomes indicates that partitioning and TA systems are core features of large plasmids. Indeed, we confirm the presence of a previously unrecognized type I TA system in conjugative IncX3 plasmids, which serve as important vectors of antibiotic resistance in human pathogens. These findings suggest that large plasmids – including conjugative and mobilizable types – encode TA systems, some of which have yet to be identified. The combination of TA and partitioning systems emerges as the most effective strategy for the evolutionary success of low-copy extrachromosomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ddba67c058e73d4520b91af55962bc53698843f" target='_blank'>
              The combination of active partitioning and toxin-antitoxin systems is most advantageous for low-copy plasmid fitness
              </a>
            </td>
          <td>
            Johannes Effe, M. Santer, Yiqing Wang, Theresa E. Feenstra, Nils F. Hülter, Tal Dagan
          </td>
          <td>2025-08-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Alkyladenine DNA glycosylase (AAG) is a critical enzyme in the base excision repair (BER) pathway, responsible for removing a broad spectrum of alkylated DNA lesions. While AAG maintains genomic stability, dysregulated activity has been implicated in cancer development, drug resistance, and neurodegenerative diseases. This review synthesizes the current knowledge on AAG’s structure, catalytic mechanism, and polymorphic variants, highlighting their potential roles in disease pathogenesis. A comprehensive bioinformatics analysis of over 370 AAG single-nucleotide polymorphisms (SNPs) is presented, identifying ~40% as high-risk variants likely to impair enzymatic function. Notably, 151 SNPs were predicted to be damaging by multiple algorithms, including substitutions at catalytic residues and non-conserved sites with unknown functional consequences. Analysis of cancer databases (COSMIC, cBioPortal, NCBI) revealed 93 tumor-associated AAG variants, with 18 classified as high-impact mutations. This work underscores the need for mechanistic studies of AAG variants using structural biology, cellular models, and clinical correlation analyses. Deciphering AAG’s polymorphic landscape may unlock personalized strategies for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35650f59a2a5d69d14473a41169b47ec06f0170" target='_blank'>
              The Link Between Human Alkyladenine DNA Glycosylase and Cancer Development
              </a>
            </td>
          <td>
            O. A. Kladova, A. A. Kuznetsova
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Despite aggressive treatment, glioblastoma patients suffer from an invariably poor prognosis. Whilst substan- tial progress has been made in understanding the role of DNA methylation in brain tumour pathology, methy- lation of RNA has received comparatively little attention, despite being a key driver of cellular differentiation and neurodevelopment. N6-methyladenosine (m6A) is the most abundant RNA modiﬁcation and is, crucially, reversible. The immense intra-tumoral heterogeneity and phenotypic plasticity of glioblastoma are major barri- ers to successful treatment, and dynamic m6A methylation may mediate such behaviour, by ﬂexibly inﬂuencing transcriptional networks via the regulation of mRNA metabolism.



 Nanopore sequencing technology uniquely enables direct sequencing of RNA, and therefore detection of RNA modiﬁcations within their sequence context. We present nanopore direct RNA sequencing data from four in- house glioma-derived cell lines. We validate this approach using a novel drug inhibitor of the major m6A methyl- transferase, METTL3.



 We deﬁne transcriptome-wide RNA methylome proﬁles for four cell lines and identify common candidate path- ways which exhibit methylation. We conﬁrm robust reduction in m6A marks following treatment with a novel METTL3 inhibitor. We further compare the transcriptional landscape of wild-type tumours against these treated samples, demonstrating differential expression of biological processes linked to RNA metabolism.



 We demonstrate that nanopore direct RNA sequencing can resolve m6A modiﬁcations at single-base resolu- tion, preserving m6A stoichiometry. We validate this approach with a known inhibitor of METTL3, and further identify differentially expressed pathways in response to such inhibition. We show that glioma cell lines demon- strate extensive methylation throughout the transcriptome, and exhibit a degree of inter-tumoral heterogeneity of these m6A signatures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad119dfd67a68d80f7976dca916d99c54e9f801" target='_blank'>
              EXPLORING THE GLIOMA M6A RNA EPITRANSCRIPTOME WITH DIRECT RNA SEQUENCING
              </a>
            </td>
          <td>
            Simon Deacon, M. Radhi, Lauryn Walker, J. Rowlinson, S. Paine, M. Loose, Stuart Smith
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Many recent lines of evidence from the human microbiome and other fields indicate bacterial involvement in various types of cancer. Helicobacter pylori has been recognized as the major cause of stomach cancer (gastric cancer), but the mechanism by which it destabilizes the human genome to cause cancer remains unclear. Our recent studies have identified a unique family of toxic restriction enzymes that excise a base (A: adenine) from their recognition sequence (5′-GTAC). At the resulting abasic sites (5′-GT_C), its inherent endonuclease activity or that of a separate endonuclease may yield atypical strand breaks that resist repair by ligation. Here, we present evidence demonstrating involvement of its H. pylori member, HpPabI, in stomach carcinogenesis: (i) Association of intact HpPabI gene with gastric cancer in the global H. pylori Genome Project and the open genomes; (ii) Frequent mutations at A in 5′-GTAC in the gastric cancer genomes as well as in H. pylori genomes; (iii) Its induction of chromosomal double-strand breaks in infected human cells and of mutagenesis in bacterial test systems. In addition, its unique regions that interact with DNA exhibit signs of diversifying selection. Our further analysis revealed similar oncogenic bacterium–restriction–enzyme pairs for other types of cancer. These results set another stage for cancer research and medicine around oncogenic restriction enzymes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f484817b57e3d6d7ad5b587893b556976204531a" target='_blank'>
              Helicobacter pylori base-excision restriction enzyme in stomach carcinogenesis
              </a>
            </td>
          <td>
            Masaki Fukuyo, Noriko Takahashi, Katsuhiro Hanada, Ken Ishikawa, eslovas Venclovas, K. Yahara, Hideo Yonezawa, Takeshi Terabayashi, Yukako Katsura, Naoki Osada, Atsushi Kaneda, M. C. Camargo, Charles S Rabkin, I. Uchiyama, Takako Osaki, Ichizo Kobayashi, Edited By, Asima Bhattacharyya
          </td>
          <td>2025-08-01</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Recent evidence highlights the role of extrachromosomal circular DNAs (eccDNAs) in cancers. However, reports regarding its role in hepatocellular carcinoma (HCC) are infrequent. The abundance of eccDNAs from five HCC/adjacent tissue pairs was explored using Circle-Sequencing. eccDNA PAFAH1B3 was selected as one of the objects. The effect of eccDNA PAFAH1B3 on HCC progression was determined using EdU, Transwell, and apoptosis assays. Additionally, the expressions of eccDNA PAFAH1B3, mRNA PAFAH1B3, and epithelial–mesenchymal transition (EMT)-related markers were determined using RT-PCR and WB. A xenograft tumor model was established to explore the function of PAFAH1B3 in vivo, and EMT-related markers were detected using RT-PCR and IHC analyses. The abundance of eccDNA PAFAH1B3 was significantly increased in HCC cell lines after transfection with eccDNA PAFAH1B3, and promoted the proliferation, migration, and invasion of liver cells while inhibiting apoptosis. The levels of mRNA PAFAH1B3 were also upregulated. Furthermore, intratumoral injection of PAFAH1B3 inhibitor suppressed tumor growth, and PAFAH1B3 knockdown increased and decreased the levels of the E-cadherin and N-cadherin, respectively. Our study findings reveal that eccDNA PAFAH1B3 may promote the occurrence and development of HCC by enhancing the expression of PAFAH1B3 and regulating EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173a3d83ec58d1fbf7f0f7dcd22ef9f85da2bb94" target='_blank'>
              PAFAH1B3 Exists in Linear Chromosomal and Extrachromosomal Circular DNA and Promotes HCC Progression via EMT
              </a>
            </td>
          <td>
            Dandan Li, Huishan Sun, Yingjie Wang, Yicong Yin, Ying Zhu, Xia Qian, Shanshan Wang, Longhao Zhang, Haitao Zhao, Ling Qiu
          </td>
          <td>2025-09-10</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Chromatin relaxation is a permissiven progress for DNA repair through enabling repair factors to access the damaged DNA. Linker histone H1 is important in maintaining chromatin compaction under physiological state. The recent evidence highlights the importance of H1 modifications in response to cellular stress. Following DNA double‐strand breaks, the metabolic enzyme phosphorylated CTP synthase 1 (CTPS1) functions as a deamidase, catalyzing the rapid conversion of H1 residues Asn76 and Asn77 into aspartate. This modification enables subsequent acetylation at Lys75 by the histone acetyltransferase p300, thereby reducing H1‐DNA affinity and promoting chromatin decompaction. This sequential modification‐H1 deamidation followed by acetylation‐facilitates the recruitment of repair factors involving both homologous recombination and non‐homologous end joining repair pathways, and consequently promoting DNA repair. Importantly, high expression of CTPS1 is associated with resistance to radiotherapy in mouse models and clinical cancer patients, suggesting that the CTPS1 may serve as a potential therapeutic target. While targeting CTPS1 may offer opportunities to enhance radiosensitivity of cancer patients, challenges related to specificity and off‐target effects require further studies. This article highlights an emerging role of H1 modification in the DNA damage repair and discusses the therapeutic potential of manipulating H1 deamidation in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ef2febfbdf0354039134d0a6f7b0aac3792561" target='_blank'>
              H1N76/77 deamidation facilitates chromatin remodelling and genome stability during DNA damage repair
              </a>
            </td>
          <td>
            Tingting Feng, Mengyan Li, Chenmiao Hu, Yuan Tian, Wei-Guo Zhu
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, Madlen Müller, Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41c845deccf83a402934c97c50b9796b93645c5b" target='_blank'>
              Establishment of chromatin architecture interplays with embryo hypertranscription.
              </a>
            </td>
          <td>
            Guang Yu, Kai Xu, Weikun Xia, Ke Zhang, Qianhua Xu, Lijia Li, Zili Lin, Ling Liu, Bofeng Liu, Zhenhai Du, Xia Chen, Qiang Fan, Fangnong Lai, Wenying Wang, Lijuan Wang, Feng Kong, Chao Wang, Haiqiang Dai, Huili Wang, Wei Xie
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Nasopharyngeal Carcinoma (NPC) is a highly metastatic malignancy, often diagnosed at advanced stages with poor prognosis and survival rates. It is strongly associated with Epstein-Barr virus (EBV) exposure and Chinese/Asian ethnic backgrounds, leading to approximately 300 new cases annually in Singapore. EBV plays a critical role in NPC development, with both its latent and lytic phases contributing to tumorigenesis. The virus precisely controls its methylation pattern to regulate latent gene expression and lytic reactivation. In contrast, disruptions in DNA modification patterns in humans are frequently observed in tumorigenesis, suggesting transcriptional dysregulation. Currently, few studies have explored limited loci in human and EBV epigenomes, with most only profiling population-level average methylation status in NPC. These studies cannot distinguish between 5-hydroxymethylcytosine (5hmC) and 5-methylcytosine (5mC), which have opposing regulatory roles. Moreover, existing methods fail to access repetitive regions, which comprise significant portions of both the EBV and human genomes. To address these gaps, we generated whole-genome, single-molecule, single-base resolution profiles of 5mC and 5hmC modifications in EBV and human DNA from different NPC tumor samples and cell lines using direct Nanopore ultralong-read sequencing. This approach offers unique insights into the complexities of EBV and human DNA modifications during NPC development. We discovered that distinct EBV strains cause contrasting 5mC changes in both human and EBV epigenomes, with some inducing hypermethylation and others hypomethylation. While hypomethylation was previously linked to APOBEC activity, our findings suggest that EBV strain-specific sequences play a significant role. Notably, in EBV epigenome, 5mC differences are concentrated in repetitive regions, whereas 5hmC profiles are consistent across strains, with persistent 5hmC modifications at specific loci. Our study provides valuable epigenome knowledge to the field of NPC research. Further data and detailed mechanistic studies are crucial to deepening our understanding of disease onset, progression, and advancing early detection and therapeutic strategies.



 Li Zhe, Yao Fei, Sia Yee Yen, Melvin Chua Lee Kiang, Joshua Tay, Liu Jianjun. Towards Profiling DNA Modifications of Virus and Human Host DNA in Epstein-Barr Virus (EBV)-positive Nasopharyngeal Carcinoma (NPC) with Nanopore Ultra-Long Reads Sequencing [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P52.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e731f932485dff94f0e6e21fa72516a251e45df3" target='_blank'>
              Abstract P52: Towards Profiling DNA Modifications of Virus and Human Host DNA in Epstein-Barr Virus (EBV)-positive Nasopharyngeal Carcinoma (NPC) with Nanopore Ultra-Long Reads Sequencing
              </a>
            </td>
          <td>
            Li Zhe, Yao Fei, Sia Yee Yen, Melvin Chua Lee Kiang, Joshua Tay, Jianjun Liu
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Lynch syndrome (LS) and familial adenomatous polyposis (FAP) are hereditary cancer predisposing syndromes characterized by increased risk of especially early-onset colorectal cancer. Predisposition to LS is caused by germline mutations in DNA mismatch repair genes leading to elevated cancer progression and microsatellite instability. FAP is associated with germline mutations in APC promoting cancer initiation and chromosomal instability. DNA methylation is an important epigenetic mechanism in early tumorigenesis via, e.g., field defects in non-neoplastic colon. Our aim was to study genome-wide methylation changes in colorectal specimens (adenomas and carcinomas supplemented with paired normal colon) obtained during colonoscopy surveillance, and explore the role of such alterations in tumorigenesis, with a special focus on early changes. To our best knowledge, this study is the first one to compare altered DNA methylation genome-wide in LS and FAP-associated colorectal neoplasia. Results DNA methylation alterations were subtle in FAP adenomas, whereas in LS adenomas, changes were abundant when compared to their normal counterparts. When FAP normal and LS normal colon were compared, DNA methylation changes of FAP normal colon mirrored those occurring in LS tumors, suggesting that colorectal tumorigenesis-associated DNA methylation alterations take place already in FAP normal colon mucosa. DNA methylation age was more variable in LS than FAP normal colon, and in proximal than distal colon, when compared to individuals’ age at the time of sampling. In LS tumors, DNA methylation changes (hyper- and hypomethylation) were abundant even in adenomas with low-grade dysplasia and stable microsatellites and peaked in adenomas with high-grade dysplasia. LINE-1 hypomethylation was more prominent in LS adenomas than FAP adenomas, but normal colon of LS and FAP displayed similar levels of LINE-1 methylation. Conclusions Genome-wide DNA methylation changes are an integral part of FAP and LS-associated colorectal tumorigenesis. Occurrence at early stages, even in non-neoplastic colonic mucosa, and increased prevalence with progressive dysplasia suggest a role in tumor development. Overlap of many of the topmost DNA methylation alterations between LS and FAP, and previous reports of their occurrence in sporadic colorectal and other tumors as well, imply their broad biological relevance and possible biomarker potential for clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01940-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f732072d5015ea229362c0044955d5acfa518d6c" target='_blank'>
              Genome-wide DNA methylation profiles of colorectal tumors in Lynch syndrome and familial adenomatous polyposis
              </a>
            </td>
          <td>
            Satu Mäki-Nevala, Anni K Kauppinen, A. Olkinuora, Aleksi Laiho, Petri Törönen, L. Renkonen-Sinisalo, A. Lepistö, T. Seppälä, J. Mecklin, P. Peltomäki
          </td>
          <td>2025-08-02</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Plasmid fusion, the recombination of distinct plasmids into a single plasmid, constitutes a critical mechanism shaping bacterial evolution. This process facilitates genetic resource consolidation, enabling bacterial populations to rapidly adapt to selective pressures such as antibiotic exposure. The growing recognition of plasmid fusion as a widespread genomic recombination event necessitates a comprehensive analysis of its prerequisites, molecular mechanisms, and functional consequences. This review synthesizes current knowledge on the triggers, molecular mechanisms, and outcomes of plasmid fusion. We demonstrate that conjugative transfer and antibiotic selection pressure drive plasmid fusion via DNA double-strand breaks and SOS response activation, with formation mechanisms dominated by insertion sequence-mediated homologous recombination or transposon-mediated recombination. Fusion plasmids amplify antimicrobial resistance (AMR) dissemination by creating multidrug-resistant megaplasmids, enable cross-host transfer of non-conjugative plasmids, and foster virulence-resistance hybrid elements. Furthermore, we propose a novel framework for investigating, specifically addressing how fusion plasmids achieve compensatory mechanisms to balance functional redundancies among critical genetic modules. Elucidating the evolutionary drivers underlying the pervasive dissemination of fusion plasmids will not only advance our understanding of their ecological success but also identify critical intervention targets to disrupt their dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e76ccd851ef2713c4c63665c2a723bfdeba73e" target='_blank'>
              Molecular drivers of fusion plasmid: mechanistic insights and evolutionary implications.
              </a>
            </td>
          <td>
            Ziyi Liu, Ying Tang, Mingxing He, Chenggang Xu
          </td>
          <td>2025-08-20</td>
          <td>The Journal of antimicrobial chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Cancer research has long focused on mutations in normal body cells, but this approach has not produced major breakthroughs for most cancers. Our study explores a different concept that some aggressive cancers may actually arise from early reproductive cells called primordial germ cells, which normally develop into eggs and sperm. We created a new experimental model showing how a virus can transform human primordial germ cell-like cells into virus-positive Merkel cell carcinoma, a rare but deadly skin cancer. This model shows that cancers can emerge through changes in developmental states rather than relying solely on genetic mutations. By linking cancer development to early germ cells, our findings suggest a unifying explanation for both germ cell cancers and body cancers. This new perspective may guide more effective approaches to study, diagnose, and treat cancer by focusing on early human development rather than only DNA mutations and later developmental stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de5f636d947e93ff13e3e8ff3f419d1f1d6cb51" target='_blank'>
              A Polyomavirus-Positive Merkel Cell Carcinoma Mouse Model Supports a Unified Origin for Somatic and Germ Cell Cancers
              </a>
            </td>
          <td>
            Wendy Yang, Sara Contente, Sarah Rahman
          </td>
          <td>2025-08-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3c0ab5ecbb10104dc6e55fe017fd113001dfb6" target='_blank'>
              DNA damage response signaling in oocytes from an oncofertility perspective.
              </a>
            </td>
          <td>
            Yunman Sonia Song, Jiyang Zhang, Yingnan Bo, Qiang Zhang, So-Youn Kim, Shuo Xiao
          </td>
          <td>2025-09-12</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c548af869e91466a22ceb7887ea1b6fdb37dc0" target='_blank'>
              Integrative proteogenomics and forward genetics reveals a novel mitotic vulnerability in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Nicholas J Neill, S. Satpathy, Karsten Krug, Jitendra K Meena, Nivetha Ramesh Babu, Cheyenne Calderon, Desmon Reed, Marcus J Weber, L. Dobrolecki, Alaina Lewis, Christina Sallas, M. Anurag, Kimberly R Holloway, Chen Huang, S. Vasaikar, Maria F Cardenas, Beom-Jun Kim, Doug W. Chan, Shayan C. Avanessian, S. Tyagi, Mayra Orellana, Sufeng Mao, Heyuan Li, Fade Gong, S. Kurley, K. Meerbrey, Calla M Olson, Amritha Nair, Tingting Sun, Hsiang-Ching Chung, Elizabeth A. Bowling, Jarey H Wang, Pengju Zhang, Peng Xiao, Duxiao Yang, F. Stossi, Mei-Yin C Polley, Alexander B. Saltzman, F. Mundt, D. Mani, Michael A Gillette, S. Hilsenbeck, George Miles, Carolina Gutierrez, C. K. Osborne, Charles Y. Lin, N. Gray, Jinpeng Sun, David A. Wheeler, C. Perou, A. Malovannaya, Michael T. Lewis, Bing Zhang, Matthew J Ellis, Steven A. Carr, Thomas F Westbrook
          </td>
          <td>2025-08-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="Background Gastric cancer (GC) ranks as the fifth most prevalent malignancy and the third leading cause of cancer-related mortality worldwide, with complex pathogenesis driven by genetic and epigenetic alterations. While genetic contributors to GC have been extensively studied, the functional role of N6-methyladenosine (m6A) RNA methylation—the most abundant eukaryotic RNA modification—in gastric carcinogenesis remains insufficiently characterized. This study aimed to investigate transcriptome-wide m6A methylation dysregulation and its mechanistic implications in GC progression. Methods Methylated RNA immunoprecipitation sequencing (MeRIP-seq) was performed to map m6A epitranscriptomes in paired GC and adjacent normal tissues. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses were conducted on 253 hypomethylated mRNAs to delineate the biological pathways associated with m6A dysregulation. The transcriptomic profiles were analyzed using RNA-seq, while a retrospective cohort analysis (n = 58) evaluated the correlations between IGF2BP3 expression and the clinicopathological characteristics of patients with GC. Results MeRIP-seq analysis demonstrated transcriptome-wide m6A hypomethylation in GC tissues, identifying 271 significantly reduced peaks (p < 0.01). Functional annotation revealed enrichment of hypomethylated transcripts in metabolic pathways and transcriptional dysregulation. Notably, m6A-related genes exhibited widespread activation in GC, with IGF2BP3 showing the most pronounced upregulation (106-fold increase, p < 0.0001). Clinically, elevated IGF2BP3 expression significantly correlated with lymph node involvement (p = 0.016) and advanced TNM staging (p = 0.028). Conclusion This study establishes m6A methylation dysregulation as a critical mechanism in GC pathogenesis and identifies IGF2BP3 as both a potential therapeutic target and a prognostic biomarker in GC. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00802-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a9f3e1f9eee919cc6ee02a44855814e67eb1ac" target='_blank'>
              Clinical and prognostic significance of m6A hypomethylation and IGF2BP3 overexpression in gastric cancer: an integrated epigenomic-transcriptomic analysis
              </a>
            </td>
          <td>
            Xiang-yun Hu, Zhe Wang, Youwei Kou, Yujing Huang
          </td>
          <td>2025-08-22</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f34ee7a0d8147abc850afa171aa9019409c2cc66" target='_blank'>
              DNA methylation alterations of PITX2 and PANCR are associated with left-right asymmetry of the heart and left ventricle size
              </a>
            </td>
          <td>
            Steffan Noe, Niikanoff Christiansen, K. B. Olsen, Sara Tangmose Larsen, Pernille Heimdal Holm, M. K. Palsøe, M. Kampmann, S. B. Jacobsen, Mikkel Meyer Andersen, J. Banner, N. Morling, Jacob Tfelt-Hansen, J. D. Andersen
          </td>
          <td>2025-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Pleomorphic giant cell adenocarcinoma (PGCA) of the prostate is a rare, aggressive variant characterized by multinucleated giant cells, sarcomatoid features, and resistance to conventional therapies. Despite its recognition in the WHO 2016 guidelines, the molecular drivers and clinicopathological correlates of PGCA remain poorly characterized. This study presents the first integrative clinicogenomic profiling of PGCA, revealing a novel prognostic gene signature with direct implications for diagnosis and treatment. Methods We conducted comprehensive clinicopathological and genomic analyses of a treatment-refractory PGCA case using histology, immunohistochemistry (IHC), whole-exome sequencing (WES), clonal evolution modeling, and multicohort validation. IHC assessed key prostate cancer markers (AR, AMACR, KLK3, PTEN, NKX3-1, VIM), while WES compared somatic alterations in PGCA, adjacent adenocarcinoma, and stromal tissue. Public datasets (prostate_dkfz_2018, prad_tcga, prad_mcspc_mskcc_2020) were used for external validation. Results PGCA displayed profound pleomorphism, necrosis, and complete loss of luminal markers (AR/AMACR/KLK3), along with strong vimentin (VIM) expression, consistent with epithelial–mesenchymal transition. WES revealed PGCA-specific mutations enriched in cell-cycle and inflammatory response pathways, distinct from metabolic alterations in the adjacent adenocarcinoma. Clonal evolution analysis showed divergent progression from a shared ancestral clone. Importantly, mutations in ADAMTS7, CDH1, DRD5, MGAT5, and TP53 emerged as a robust five-gene signature predictive of biochemical recurrence, metastasis, and poor survival, validated across multiple independent cohorts. Conclusion Our study provides the first molecular roadmap of prostatic PGCA to date, establishing a novel five-gene prognostic signature and revealing fundamental insights into its pathogenesis through divergent evolution from conventional adenocarcinoma. These insights offer new opportunities for precise diagnosis, prognostic stratification, and targeted therapeutic strategies for this lethal prostate cancer variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3fd48864bfffc9bb6aed51ef499e504db68fa6c" target='_blank'>
              Clonal divergence and genomic meltdown in prostatic pleomorphic giant cell adenocarcinoma
              </a>
            </td>
          <td>
            Xiaoshi Ma, Kun Chen, Jing Zhang, Liming Liu, Jiping Luo, Kaipeng Huang, Hongying Zhang, Danni Liu, Jizhou Gou, Changyin Feng, Xia Zhao, Wanying Li, Lipeng Chen, Li Yin, Xianlin Meng, Zhiqiang Cheng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract While the majority of viruses falls within a handful of well-defined realms, many novel viruses and virus-like genetic elements remain to be discovered. Given the abundance and diversity of small, circular DNA genomes in animal gut viromes, we hypothesized that viruses of similar size might have been overlooked in other environments. Re-analyzing existing ocean virome datasets for small viruses, we discover a novel group of circular mobile genetic elements of ∼4.6kb in size, likely with double-stranded DNA, which we term Charybdis elements. Charybdis elements form a singular group based on cohesive gene content, encode around 10 hypothetical genes but apparently contain no virus, transposon or plasmid associated hallmark proteins. They are detected exclusively in marine datasets ranging from surface water to hydrothermal vents and sediments, from the Arctic to the Antarctic. Virome datasets separating virus-like particles by density gradients show that Charybdis elements accumulate in the fraction representing extracellular vesicles and small, tail-less viruses that are often associated with lipid membranes. One specific clade of Charybdis elements is shown to integrate into the genomes of the common marine group II archaeon candidate order Poseidoniales, the putative host of these elements. Analysis of thousands genomes shows a common genome architecture consisting of two modules: An accessory module containing a diverse range methyltransferases and other genes likely involved in host interactions, and a more conserved (putative) structural module. The structural module encodes proteins that are confidently predicted to form striking tri- or pentameric structures. These structures show stark resemblance and in some cases homology to protrusion or tail-elements on the capsids of viruses from the realms Varidnaviria and Duplodnaviria . Given these traits, Charybdis elements likely represent a hitherto undiscovered group of virus or virus-satellites, with at least one protein of a novel double-Greek-key fold serving as a potential capsid element. However, since their hosts remain unculturable, the true nature of these ubiquitous marine elements remains as of yet unresolvable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d6ae6d60b7d2c07e786f432ef49127c628ff31" target='_blank'>
              A novel mobile genetic element with virus-like characteristics is widespread in the world’s oceans
              </a>
            </td>
          <td>
            P. Kirchberger, Antoni Luque
          </td>
          <td>2025-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94f6767c83b49c5f93547ae2e51f853756e7c466" target='_blank'>
              DNA methylation influences human centromere positioning and function.
              </a>
            </td>
          <td>
            Catalina Salinas-Luypaert, Danilo Dubocanin, Rosa Jooyoung Lee, Lorena Andrade Ruiz, Riccardo Gamba, Marine Grison, Léonid Velikovsky, Annapaola Angrisani, Andrea Scelfo, Yuan Xu, Marie Dumont, Viviana Barra, Therese Wilhelm, Guillaume Velasco, Marialucrezia Losito, René Wardenaar, C. Francastel, F. Foijer, G. Kops, K. Miga, Nicolas Altemose, D. Fachinetti
          </td>
          <td>2025-09-04</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Menin protein is encoded by the multiple endocrine neoplasia type 1 gene, which typically serves an oncogenic role in endocrine organs. However, in mice, menin protein is a key mediator of leukemic transformation, particularly in acute myeloid leukemia (AML), and is involved in the disease process through epigenetic regulatory mechanisms. This functional paradox may be due to the unique gene expression regulatory properties of menin, which can both activate and inhibit the expression of target genes. At the molecular level, menin protein regulates the gene transcription process by interacting with multiple protein complexes and forms a complex network with multiple signaling pathways. As the core hub of transcriptional regulation, menin protein is essential for the maintenance of cellular homeostasis and its aberrant function can lead to gene expression disorders, which contributes to the development of AML. Despite the druggability challenges of several transcriptionally regulated proteins, inhibitors of menin protein have made breakthroughs in clinical development. Particularly in AML subtypes [for example, lysine methyltransferase 2A (KMT2A) rearrangement or nucleophosmin (NPM1) mutation], menin protein inhibitors have demonstrated favorable efficacy. The present study systematically reviewed biological functions of the menin protein and its application in targeted therapy for specific AML subtypes. Notably, menin inhibitors have demonstrated potential in KMT2A/NPM1-mutant leukemia, however, the off-target effects, resistance mechanisms and a lack of biomarkers due to the extensive nature of their binding interface remains to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18b23d4e7fbebe4140893833cf2070b2e3e6ebf" target='_blank'>
              Targeting menin in lysine methyltransferase 2A/nucleophosmin-mutated leukemia: A novel strategy from epigenetic dysregulation to clinical therapy (Review)
              </a>
            </td>
          <td>
            Junjie Bi, Hongjun Zhou
          </td>
          <td>2025-09-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background. Studies of comorbid (syntropic) and inversely comorbid (rarely occurring together, i.e., dystropic) diseases have focused on the search for molecular causes of this phenomenon. Materials. We investigated DNA methylation levels in regulatory regions of 23 apoptosis-associated genes as candidate loci associated with the “cancer–neurodegeneration” dystropy in patients with Huntington’s disease (HD) and patients with non–small cell lung cancer (LC). Results. Statistically significant differences in methylation levels between the HD and LC groups were found for 41 CpG sites in 16 genes. The results show that five genes (SETDB1, TWIST1, HDAC1, SP1, and GRIA2) are probably involved in the phenomenon of inverse comorbidity of these diseases. For these genes, the methylation levels of the studied CpG sites were altered in opposite directions in the two groups of patients, compared to the control group. Conclusions. For the SP1 gene, the above hypothesis is supported by our analysis of open-access data on gene expression in patients with the aforementioned diagnoses and fits a probable mechanism of the “HD–LC” dystropy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea115150dd05fa7cf3e3d8dd1a0ac32240561c27" target='_blank'>
              DNA Methylation Status of Regulatory Regions of Apoptosis-Associated Genes in Dystropy «Huntington’s Disease—Non-Small Cell Lung Cancer»
              </a>
            </td>
          <td>
            Nadezhda P. Babushkina, E. Bragina, D. Gomboeva, Iuliia A. Koroleva, S. Illarioshkin, S. Klyushnikov, N. Abramycheva, Maria A. Nikitina, Valentina M. Alifirova, N. Litviakov, M. K. Ibragimova, M. M. Tsyganov, I. A. Tsydenova, Aleksei A. Zarubin, I. Goncharova, M. Golubenko, R. R. Salakhov, A. A. Sleptcov, Aksana N. Kucher, M. S. Nazarenko, Valery P. Puzyrev
          </td>
          <td>2025-08-07</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="There is a continued need for identification of novel disease drivers of acute myeloid leukemia as many patients experience relapse and have poor clinical outcomes. Analyses from our study and publicly available datasets predicted CEBPD as a novel tumor suppressor gene in acute myeloid leukemia. Consistent with the analyses, CEBPD knockdown experiments showed activation of MAPK signaling with concomitant increase in cell growth rate, while upregulation experiments suggested induction of myeloid differentiation marker CD14 expression in AML cell lines OCI-AML2 and OCI-AML5. Consistent with a previous report, our genomics analyses and azacytidine treatment experiments suggested a role for DNA methylation in downregulation of CEBPD expression during AML pathogenesis. Altogether, our results provide experimental evidence for a tumor suppressor function of CEBPD in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6db75144a5e5ecd2e5d64e0dad8460e5b0da483" target='_blank'>
              CCAAT-enhancer binding protein delta functions as a tumor suppressor gene in acute myeloid leukemia
              </a>
            </td>
          <td>
            Subhash Prajapati, Cem Meydan, Yaseswini Neelamraju, Zhenjia Wang, Hao Fan, Nicholas Dunham, Richard Dillon, Jorge A Gandara, Tak C Lee, Caroline Sheridan, P. Zumbo, Michael W. Becker, Lars Bullinger, Martin P Carroll, Richard J J D'Andrea, Ross L Levine, Christopher E Mason, Ari Melnick, Chongzhi Zang, Stefan Bekiranov, Francine E. Garrett-Bakelman
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Endometrial cancer (EC), a type of uterine cancer, is witnessing a global increase in incidence. Despite advancement in diagnosis and treatment, metastatic or recurrent EC often exhibits a poor prognosis, necessitating novel therapeutic strategies. DEAD-box helicase 17 (DDX17) is implicated in several cancers. Our study aimed to uncover the biological function and molecular mechanism of DDX17 in EC. Methods EC and matched adjacent normal tissues from 80 patients were analyzed; DDX17 mRNA/protein expression was quantified via RT-qPCR and immunoblotting in clinical specimens and cell lines (HEC-1A, HEC-1B, Ishikawa), with functional assays (proliferation/migration/invasion) performed following DDX17 overexpression in vitro, while xenograft modeling in BALB/c nude mice enabled in vivo validation through immunofluorescence and immunohistochemical staining; mechanistic studies employed RNA immunoprecipitation (RIP-PCR), m6A-specific RNA immunoprecipitation (MeRIP-PCR), and protein interaction analyses. Results DDX17 was significantly downregulated in EC tissues/cells, correlating with poor prognosis in clinical cohorts. Overexpression of DDX17 suppressed tumorigenesis both in vitro and in vivo through PI3K/AKT pathway inactivation. METTL3-mediated m6A modification stabilized DDX17 mRNA, with IGF2BP2 specifically recognizing m6A-modified transcripts. Critically, METTL3 ablation reversed DDX17 stabilization and abolished its tumor-suppressive effects, while PI3K inhibition (LY294002) phenocopied METTL3 restoration in rescuing DDX17 deficiency-induced oncogenicity. Conclusion METTL3-mediated m6A modification stabilizes DDX17 to suppress EC cell proliferation, migration, and invasion through an IGF2BP2-dependent mechanism by inactivating the PI3K/AKT pathway. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01817-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc74064b9143aa7c166a780986e41cb0e801d2f4" target='_blank'>
              METTL3 stabilizes DDX17 mRNA via IGF2BP2-mediated m6A modification to suppress endometrial cancer progression
              </a>
            </td>
          <td>
            Yuan Zhang, Mengjun Liu, Jing Liu, Jing Zhang
          </td>
          <td>2025-08-07</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Nuclear architecture and chromosome folding are often speculated to influence genome replication. In yeasts, centromeres cluster close to the spindle pole body and telomeres position at the nuclear periphery. This ‘Rabl configuration’ spatially segregates the most early and late replicating parts of the genome, centromeres and telomeres, respectively, suggesting that origin position along the centromere–telomere axis may influence origin activity. Here, we investigated DNA replication in a wild-type, 16-chromosome Saccharomyces cerevisiae strain and in its single-chromosome counterpart engineered by chromosome fusion and elimination of all but two telomeres and one centromere, which strongly affects genome folding and abrogates the Rabl conformation. Using nanopore sequencing-based methods, we found that the DNA replication program of both strains was virtually indistinguishable, with the exception of origin inactivation next to deleted centromeres and changes in origin efficiency and fork direction at chromosome fusions, as anticipated from the known origin-regulation properties of centromeres and telomeres. Only a handful of replication changes, mostly due to local origin repression, were observed elsewhere. Fork speed was also unaffected except at deleted centromeres. In conclusion, the DNA replication program of budding yeast is remarkably resilient to perturbations of chromosome folding and loss of the Rabl conformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e28f5bc4a9bd5e110c57ffd165964c5f1a57708" target='_blank'>
              Replication program of a single-chromosome budding yeast strain
              </a>
            </td>
          <td>
            Jade Pellet, Laurent Lacroix, Bertrand Theulot, Emma Simonin Chavignier, Alan Tourancheau, F. Proux, Sylvie Hermann-Le Denmat, G. Millot, Benoît Le Tallec, O. Hyrien
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3759561e15319d6f3bc64b2d73976dc58b1df0ff" target='_blank'>
              Off-pore nucleoporin sPOM121 transcriptionally propels β-Catenin driven tumor progression and immune escape in prostate cancer.
              </a>
            </td>
          <td>
            Perumalraja Kirthika, Vijayakumar Jawalagatti, Peiyao Li, Michael Xu, M. Carceles-Cordon, Adam Ertel, S. A. Quinn, D. Petrylak, J. Orme, S. Dehm, E. Antonarakis, Elisabeth I Heath, Mukesh K. Pandey, Carlos Cordon-Cardo, Haojie Huang, Haidong Dong, Shijia Zhu, Josep Domingo-Domenech, V. Rodriguez-Bravo
          </td>
          <td>2025-07-28</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) is the most common hematological malignancy. More than half of DLBCL patients achieve long‐term remission after treatment, but a third relapse after conventional Rituximab (R)‐based chemotherapy regimens, such as CHOP (cyclophosphamide, doxorubicin, vincristine and prednisone). Cancer cells are exposed to chronic replication stress, which impedes the duplication of their genome. Functional DNA repair pathways are therefore important for the survival of cancer cells. This dependence can be exploited therapeutically to hamper repair of the intrinsic DNA damage occurring during replication or to exacerbate DNA damage induced by chemotherapy. Using CRISPR‐Cas9 screening, we identified CHEK1, WEE1, ATR and RAD51 DNA repair factors as essential genes in DLBCL cells. According to these results, we investigated the effect of small molecules targeting DNA replication stress and DNA repair mechanisms, alone or in combination with the R‐CHOP genotoxic agents, cyclophosphamide and doxorubicin. Applying a standard threshold of 2 SDs below the IC50 of the genotoxic agent alone, a total of 3 synthetic lethal combinations have been identified including cyclophosphamide with CHK1/2 inhibitor, cyclophosphamide and ATR inhibitor and doxorubicin with DNAPK inhibitor. Co‐treatment with these molecules led to cell death, DNA damage induction and cell cycle arrest in DLBCL cells more efficiently than genotoxic agents alone. These data have been validated using primary DLBCL cells from patients. Our results open new perspectives for therapeutic approaches exploiting the synthetic lethality of genotoxic agents with DNA repair inhibitors to improve the therapeutic outcome of patients with DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ea8280a7e78b4aa4b06c11d4f0a4eaa409f0ab" target='_blank'>
              Synthetic Lethal Combinations of DNA Repair Inhibitors and Genotoxic Agents to Target High‐Risk Diffuse Large B Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Ovejero, Julie Devin, Laura Alibert, Camille Soun, Yea-Lih Lin, Laure Dutrieux, M. Abouladze, Elvira Garcia de Paco, Ouissem Karmous Gadacha, A. Constantinou, Guillaume Cartron, C. Herbaux, O. Elemento, Philippe Pasero, Sandrine Roulland, J. Moreaux, C. Bret
          </td>
          <td>2025-08-23</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb04dfc69e03ddcb4aa348ab8eef9592fa55873" target='_blank'>
              p300/CBP is an essential driver of pathogenic enhancer activity and gene expression in Ewing sarcoma.
              </a>
            </td>
          <td>
            Laura C Godfrey, Brandon Regalado, Sydney R Schweber, Charlie Hatton, Daniela V. Wenge, Yanhe Wen, Meaghan Boileau, Maria Wessels, Jun Qi, Christopher J Ott, Kimberly Stegmaier, Miguel N Rivera, Scott A Armstrong
          </td>
          <td>2025-09-01</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) evolves as an aggressive phenotype during prolonged androgen deprivation therapy, lacking effective clinical management. Here, we elucidated a reciprocal metabolic-epigenetic mechanism involving a positive feedback loop between glycolysis and the transcription factor LHX2 that promotes PCa progression. Mechanistically, enzalutamide-mediated inhibition of the androgen receptor (AR) led to upregulation of key glycolytic enzymes. Elevated glycolysis resulted in lactate accumulation and subsequent histone lactylation, which in turn enhanced LHX2 expression. Reciprocally, LHX2 transcriptionally activated the lactate dehydrogenase LDHA, which further amplified lactate production. Furthermore, LHX2 augmented the expression of the lineage-determining enzyme DNMT1, potentiating neuroendocrine gene expression and tumor growth. Significantly, the antiviral agent paritaprevir could directly bind to LHX2, effectively suppressing neuroendocrine marker expression and tumor progression. These findings uncover a potential role for LHX2 in orchestrating NEPC development, offering putative therapeutic targets for treating this aggressive cancer phenotype and overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef2fefc98e5fd3352706deab4254aa3b33bda2c" target='_blank'>
              LHX2 Rewires the Metabolic and Epigenetic Landscape to Drive Tumor Progression in Prostate Cancer.
              </a>
            </td>
          <td>
            Jun Jiang, Shaojie Liu, Chao Xu, Limin He, Hongji Li, Yike Zhou, Zhengxuan Li, Yu Li, Fa Yang, Yuchen Wei, Tong Lu, Keying Zhang, Jingliang Zhang, Pang Wang, Li Guo, Changhong Shi, Weihong Wen, Rui Zhang, Donghui Han, Weijun Qin
          </td>
          <td>2025-09-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="SUMMARY The ring-shaped sliding clamp PCNA enables DNA polymerases to perform processive DNA synthesis during replication and repair. The loading of PCNA onto DNA is catalyzed by the ATPase clamp loader RFC. Using a single-molecule platform to visualize the dynamic interplay between PCNA and RFC on DNA, we unexpectedly discovered that RFC continues to associate with PCNA after loading, contrary to the conventional view. Functionally, this clamp-loader/clamp complex is required for processive DNA synthesis by polymerase δ (Polδ), as the PCNA-Polδ assembly is inherently unstable. This architectural role of RFC is dependent on the BRCT domain of Rfc1, and mutation of its DNA-binding residues causes sensitivity to DNA damage in vivo. We further showed the FEN1 flap endonuclease can also stabilize the PCNA-Polδ interaction and mediate robust synthesis. Overall, our work revealed that, beyond their canonical enzymatic functions, PCNA-binding proteins harbor non-catalytic functions essential for DNA replication and genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00baaea7b1484df608f6f240f34fa8c252f35c53" target='_blank'>
              A non-catalytic role for RFC in PCNA-mediated processive DNA synthesis
              </a>
            </td>
          <td>
            Gabriella N. L. Chua, Emily C. Beckwitt, Victoria Miller-Browne, O. Yurieva, Dan Zhang, Bryce Katch, John W Watters, Kaitlin Abrantes, Ryogo Funabiki, Xiaolan Zhao, Michael E. O’Donnell, Shixin Liu
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Fanconi anemia (FA) is a rare genetic disorder caused by defects in the repair of DNA interstrand crosslinks (ICLs)—highly toxic lesions that impede essential processes like DNA replication and transcription, leading to severe genome instability. Clinically, FA presents with a broad spectrum of symptoms, including progressive bone marrow failure, congenital abnormalities, and an elevated predisposition to various malignancies, particularly acute myeloid leukemia and squamous cell carcinomas. This review provides a comprehensive overview of both the endogenous and exogenous sources of ICLs and the DNA repair pathways responsible for their resolution, with a primary focus on the FA pathway. We also discuss the tumorigenic consequences of FA pathway deficiencies, highlighting the molecular mechanisms that contribute to the heightened cancer risk observed in FA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6689c7c75058ca5686e353d84e63237862e17744" target='_blank'>
              Comprehensive review on Fanconi anemia: insights into DNA interstrand cross-links, repair pathways, and associated tumors
              </a>
            </td>
          <td>
            Chenyan Fang, Zhoujun Zhu, Jun Cao, Jun Huang, Yipeng Xu
          </td>
          <td>2025-07-30</td>
          <td>Orphanet Journal of Rare Diseases</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Some selfish genetic elements drive at meiosis to achieve transmission distortion, breaking the rules of Mendelian segregation to enhance their own evolutionary success. It has been shown that enhancers of drive must act in cis in order to gain the selfish benefit of drive and that suppressors of drive will be selected at unlinked loci. Here, we model the evolution of an autosomal trans-acting gene (Distorter) that causes the Y chromosome (or even 0 chromosome) to drive without driving itself, a phenomenon we call 'remote-control meiotic drive'. We show that such a gene may spread in the population when linked to a second locus, Assister, whose alleles are transmitted at different frequencies through sperm as compared to eggs, for which we consider various scenarios, such as sexually antagonistic selection or sex-limited drive. Depending on the mechanistic details of sex-chromosome drive, Distorter's spread can additionally facilitate transitions between XY and X0 sex determination. Our results provide a proof of principle that stretches the current understanding of segregation modifier and sex allocation theory. Moreover, we identify alternative evolutionary trajectories that could also lead to remote control drive and discuss its potential applications in developing synthetic sex-ratio-distorting elements for use in pest management, for example.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e8563304f91f2028efd43ad9bed1b9ed15be83" target='_blank'>
              Autosomally-encoded segregation distortion of sex chromosomes.
              </a>
            </td>
          <td>
            Naomi L. Greenberg, M. Patten, M. Schenkel
          </td>
          <td>2025-08-01</td>
          <td>Proceedings. Biological sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastasis is a hallmark of cancer and is responsible for the majority of cancer-related deaths. Evidence suggests that even a single cancer cell can spread and seed a secondary tumour. However, not all circulating tumour cells have this ability, which implies that dissemination and distal growth require adaptative mechanisms during circulation. Here we report that constriction during microcapillary transit will trigger reprogramming of melanoma cells to a tumorigenic cancer stem cell-like state. Using a microfluidic device mimicking physiological flow rates and gradual capillary narrowing, we showed that compression through narrow channels lead to cell and nuclear deformation, rapid changes in chromatin state and increased calcium handling through the mechanosensor PIEZO1. Within minutes of microcapillary transit, cells show increased regulation of transcripts associated with metabolic reprogramming and metastatic processes, which culminates in the adoption of cancer stem cell-like properties. Squeezed cells displayed elevated melanoma stem cell markers, increased propensity for trans-endothelium invasion, and characteristics of enhanced tumorigenicity in vitro and in vivo. Pharmacological disruption of channel activity inhibited the stem cell-like state, while the selective PIEZO1 activator Yoda1 primed this state irrespective of constriction. Finally, deletion of PIEZO1 led to complete abrogation of the constriction-induced stem cell-like state. Together, this work demonstrates that compressive forces during circulation can reprogram circulating cancer cells to tumorigenic stem cell-like states that are primed for extravasation and metastatic colonization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3810403f83f02e75d7eba1f1e769e5b2b7eed299" target='_blank'>
              Capillary constrictions prime cancer cell tumorigenicity through PIEZO1
              </a>
            </td>
          <td>
            G. Silvani, C. Kopecky, S. Romanazzo, V. Rodríguez, A. Das, E. Pandzic, J. G. Lock, C. Chaffer, K. Poole, K. Kilian
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Colorectal cancer with deficient mismatch repair (dMMR)/microsatellite instability (MSI) constitutes a distinct clinicopathologic and immunologic subtype, characterized by high sensitivity to immune checkpoint inhibitors. However, prognosis and therapeutic response vary considerably among dMMR/MSI colorectal cancers, underscoring the need for molecular markers to refine patient stratification. In this study, we systematically investigated cancer cell–intrinsic expression profiles of 188 glycosyltransferase genes by integrating single-cell, bulk, and cell line RNA sequencing datasets. This approach identified five glycosyltransferases, including GALNT7, expression of which differed consistently according to MSI status. The clinical and prognostic relevance of these glycosyltransferases was further analyzed across large-scale transcriptomic, proteomic, and IHC cohorts, comprising 662 dMMR/MSI and 3,483 proficient mismatch-repair (pMMR)/microsatellite-stable (MSS) colorectal cancers. A five-gene glycosyltransferase signature effectively distinguished MSI from MSS colorectal cancers across 18 datasets. Among the five genes, GALNT7 expression was robustly associated with favorable prognosis in four independent transcriptomic and IHC cohorts of dMMR/MSI colorectal cancers while showing little or no prognostic impact in pMMR/MSS colorectal cancers. Notably, GALNT7 expression was inversely correlated with PD-L1 expression at both the mRNA and protein levels in multiple datasets exclusively within dMMR/MSI colorectal cancers, but not in pMMR/MSS CRCs. Functional assays and lectin microarray analysis using MSI colorectal cancer cell lines revealed that GALNT7 knockdown enhanced IFNγ-induced PD-L1 expression without altering cell-surface glycosylation. In conclusion, GALNT7 expression stratified dMMR/MSI colorectal cancers into distinct subsets with differential tumor cell PD-L1 expression and diverse survival outcomes, highlighting its potential as a prognostic biomarker to guide treatment strategies. Significance: We identified glycosyltransferases with altered expression depending on MMR/MSI status. Our findings indicate the existence of two molecularly defined subtypes within dMMR/MSI colorectal cancers based on GALNT7 expression, characterized by differential tumor cell PD-L1 levels and distinct survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b90eff0e535356049051428a68485da55a57d5c1" target='_blank'>
              GALNT7 Stratifies dMMR/MSI Colorectal Cancer into Distinct Molecular Subsets Associated with Prognosis and PD-L1 Expression
              </a>
            </td>
          <td>
            Hiroya Suzuki, H. Okayama, S. Nakajima, Katsuharu Saito, Ryo Kanoda, Yuya Maruyama, Akira Matsuishi, Takuro Matsumoto, Misato Ito, Shun Chida, W. Sakamoto, M. Saito, Z. Saze, T. Momma, Kosaku Mimura, K. Kono
          </td>
          <td>2025-08-18</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The MRE11-RAD50-NBS1 (MRN) complex is a central, multifunctional factor in the detection, signaling and nucleolytic processing of DNA double-strand breaks (DSBs). To clarify how human MRN binds generic and telomeric DNA ends and can separate DNA end sensing from nuclease activities, we determined cryo-electron microscopy (cryo-EM) structures of human MRN bound to DNA and to DNA and the telomere protection factor TRF2. MRN senses DSBs through a tight clamp-like sensing state with closed coiled-coil domains, but auto-inhibited MRE11 nuclease. NBS1 wraps around the MRE11 dimer, with NBS1’s ATM recruitment motif sequestered by binding to the regulatory RAD50 S site, necessitating a switch in the NBS1 C helix for ATM activation. At telomeric DNA, TRF2 blocks the second S site via the iDDR motif to prevent nuclease and ATM activation. Our results provide a structural framework for DNA sensing via a gating mechanism and separation of sensing, signaling and processing activities of mammalian MRN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7645e8c109e5a6f384b9d506f7906e33acb9e2b8" target='_blank'>
              Structural basis for DNA break sensing by human MRE11-RAD50-NBS1 and its regulation by telomeric factor TRF2
              </a>
            </td>
          <td>
            Yilan Fan, Filiz Kuybu, Hengjun Cui, K. Lammens, Jia-Xuan Chen, Michael Kugler, Christophe Jung, K. Hopfner
          </td>
          <td>2025-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="DNA N6-methyladenine (6mA) is an emerging epigenetic mark in the mammalian genome. ALKBH1 preferentially exhibits 6mA demethylase activity for single-stranded DNA (ssDNA) or bubbled/bulged DNA, but not for double-stranded DNA (dsDNA). Nevertheless, ALKBH1 significantly decreases the cellular 6mA level in genomic DNA, whose prevailing DNA conformation in living mammalian cells is dsDNA. Therefore, the demethylase activity of ALKBH1 toward 6mA in genomic DNA, especially dsDNA, remains largely debated. Here, we found that YTHDF3 increases the 6mA demethylase activity of ALKBH1 in genomic DNA with different conformations, including dsDNA. Compared with ALKBH1, YTHDF3 preferentially recognizes and binds to 6mA-modified DNA with different conformations. YTHDF3 recognizes 6mA in genomic DNA, and binds ALKBH1 to recruit it to sites near 6mA in genomic DNA, thereby facilitating the ALKBH1-mediated removal of 6mA in genomic dsDNA. In summary, YTHDF3 is a novel genomic DNA reader and guides ALKBH1 to remove 6mA in human genomic DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9589673dbf4b0dcfb9624f20bcd04da6bc6ed357" target='_blank'>
              YTHDF3 recognizes DNA N6-methyladenine and recruits ALKBH1 for 6mA removal from genomic DNA
              </a>
            </td>
          <td>
            Xin-Hui Chen, Zi-Lu Wang, Jincui Yang, Min Chen, Si-Yi Zhao, Kun-Xiong Guo, Xuelong Zheng, Zhengwei Zhao, Xiaoqiang Chen, Jing Li, Minmin Zhang, Ling Ran, Huifang Zhu, Xiaofeng Gu, Guang-Rong Yan
          </td>
          <td>2025-07-25</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In addition to the main chromosome, approximately 10% bacterial genomes have one or more large secondary replicons, including a unique type of replicon known as "chromid," which has plasmid-type replication and partition systems but carries core genes. Their nucleotide composition of chromids is very similar to that of their related chromosomes. However, the distribution, characteristics, functions, and origins of these chromids remain unclear. In this study, we established a workflow to identify chromids from complete bacterial genomes and screened 36,322 complete bacterial genomes, resulting in the identification of 1,104 bacterial genomes with chromids as secondary replicons. These chromid-carrying bacteria belong to eight phyla and 73 genera, exhibiting diversity and a wide global distribution. We analyzed the characteristics of chromids and found that their average size is larger than that of "megaplasmids" and that multi-chromid bacteria exist. Furthermore, chromids encode genes related to bacterial respiratory chain enzyme complexes and antiviral systems, expanding bacterial metabolic capabilities and enhancing their antiviral defenses. In addition, we developed an automated identification program, Chromid-Finder, for identifying chromid sequences in metagenomic data, which has demonstrated outstanding performance. To demonstrate its application, we analyzed 92,143 metagenome-assembled genomes (MAGs) from the human gut microbiome. We found that the distribution of chromid-carrying bacteria in the human gut is closely associated with host age, health status, and geographic location. Species with chromids exhibit unique functional capabilities, showing good separation at the phylum level.IMPORTANCEIn this study, we have developed a workflow to identify chromids from complete bacterial genomes. We utilized this workflow to search for chromids in the latest NCBI RefSeq databases, to map the distribution of bacteria carrying chromids, to identify the characteristics of bacterial chromids, to discuss their origins, and to investigate their roles in bacterial life. To address the growing volume of metagenomic data, we developed a high-performance automated identification program, Chromid-Finder, designed to identify chromids and their corresponding bacterial main chromosomes within extensive metagenomic data sets. Using this tool, we analyzed 92,143 metagenome-assembled genomes (MAGs) from the human gut microbiome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1858682f6d07c026bc78fe000bacfa219ec68e47" target='_blank'>
              Unexplored diversity and potential functions of extra-chromosomal elements.
              </a>
            </td>
          <td>
            Haoyu Liu, Jia Sun, JuanJuan Si, Yi Liao, Jiaqi Bai, Xia Li, Limin Wang, Kuojun Cai, Wei Ni, Ping Zhou, Shengwei Hu
          </td>
          <td>2025-08-19</td>
          <td>mSystems</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The human genome is routinely subject to endogenous (e.g. oxidative stress) and exogenous (e.g. radiation, UV) factors that induce DNA lesions. These lesions may accumulate and impair cell function or cause cancer. Hence, DNA damage repair and preservation are prerequisites for normal cellular function and maintaining genome integrity. Current research suggests that long non-coding RNAs (lncRNA) play a role in DNA damage response (DDR). lncRNAs are RNA transcripts >200 nt that do not code for proteins. After transcription, they interact with DNA, protein, mRNA, or other non-coding RNA to influence gene expression and protein function. However, there is a lack of research concerning the role of lncRNA in DDR. This study aimed to investigate and quantify the expression of DNA-damage-responsive lncRNAs in a time-course experiment. To do so, human embryonic kidney 293T (HEK293T) cells were exposed to ionizing radiation (IR) at 2 Gy to induce 60-100 double-stranded breaks (DSB) per cell. Following this exposure, RNA was collected at 4 different time points (1, 4, 8, and 24 hrs). Finally, Real-time Quantitative PCR was performed to quantify the expression of select lncRNA genes. Our findings demonstrate an association between DDR and lncRNA gene expression. We found a temporal upregulation or downregulation of lncRNA genes, specifically ENSG00000250519, ENSG00000231595, ENSG00000254338, and ENSG00000223749. The influence of DNA lesions on the expression of these lncRNA increased over 24 hrs. This finding asserts that lncRNA is an epigenetic regulator that may influence signalling networks related to DNA damage. Hence, lncRNAs may serve as biomarkers for the diagnosis, treatment, and prognosis of cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c3f3f72f4b73fe0339171d4fca65b2b98d68e4" target='_blank'>
              Investigating DNA-damage responsive long non-coding RNA (lncRNA)
              </a>
            </td>
          <td>
            Lindsay Yu
          </td>
          <td>2025-08-28</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e71faadb3e7b7c4954d9338f65e8dc404cbfd9" target='_blank'>
              The role of HOXA1 in cancer and targeted therapy.
              </a>
            </td>
          <td>
            Jianshu Dong, Huimin Gong, Jian Li, Xinli Ma, Bilal Tayyab, Xia Xu
          </td>
          <td>2025-08-04</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hypoxia and aberrant activation of epidermal growth factor receptor (EGFR) are considered important features of various malignancies. However, whether hypoxia can directly trigger EGFR activation and its clinical implications remain unclear. In this study, we demonstrated that in oral cancer, a typical hypoxic tumor, hypoxia can induce chronic but constitutive phosphorylation of wild-type EGFR in the absence of ligands. Oral cancer cell lines exhibit different EGFR phosphorylation responses to hypoxia. In hypoxic HN4 and HN6 cells, ubiquitination-mediated endocytosis, lysosomal sorting, and degradation lead to low levels of EGFR phosphorylation. However, in CAL-27 and HN30 cells, a novel HIF-1α-induced long noncoding RNA (lncRNA), EUDAL, can compete with the E3 ligase/adaptor complex c-Cbl/Grb2 for binding to EGFR, stabilizing phosphorylated EGFR (pEGFR) and resulting in sustained activation of EGFR and its downstream STAT3/BNIP3 signaling. STAT3/BNIP3-mediated autophagy leads to antitumor drug resistance. A high EUDAL/EGFR/STAT3/autophagy pathway activation predicts poor response to chemotherapy in oral cancer patients. Collectively, hypoxia can induce noncanonical ligand-independent EGFR phosphorylation. High EUDAL expression facilitates sustained EGFR phosphorylation in hypoxic tumor cells and leads to autophagy-related drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cd45bed925400684752deec925406ffbecfc7f9" target='_blank'>
              LncRNA EUDAL shapes tumor cell response to hypoxia-induced constitutive EGFR activation and promotes chemoresistance in oral cancer
              </a>
            </td>
          <td>
            Shengkai Chen, Zhenlin Dai, Jianbo Shi, Mengyu Rui, Zhiyuan Zhang, Qin Xu
          </td>
          <td>2025-09-12</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3eb3248f534742d8b8e67a9753e5d185982f4a" target='_blank'>
              Systematic investigation reveals extensive Epstein-Barr virus transcriptional regulation of the human genome.
              </a>
            </td>
          <td>
            Phillip J Dexheimer, Matthew R. Hass, Lee E Edsall, Arame A. Diouf, Sydney H. Jones, Omer Donmez, Cailing Yin, Katelyn A. Dunn, Carmy Forney, Hayley K. Hesse, Andrew VonHandorf, Xiaoting Chen, Sreeja Parameswaran, Olivia E. Gittens, Kenyatta C M F Viel, R. Razavi, B. Gewurz, Bo Zhao, Lucinda P. Lawson, T. Hughes, Leah C. Kottyan, M. Weirauch
          </td>
          <td>2025-07-30</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Many pathogenic fungi display 'two-speed genome', with the fast-evolving genomic compartments enriched with repetitive sequences, particularly the transposons, which have been shown to drive the variation of pathogenicity-associated genes. Supernumerary chromosomes (SCs) are known to facilitate genomic variation in fungal pathogens, but their specific role in such processes remains understudied. In this study, we assessed the transferability of SCs between asexual Magnaporthe oryzae strains during co-culture and co-infection, and investigated their role in genome reconstruction through experimental evolution assays. We found that SCs could be horizontally transferred between M. oryzae strains and revealed frequent structural variations facilitated by SCs, including deletions, duplications, translocations, and SC-core chromosome recombinations during and after horizontal transfer. Remarkably, all observed intra- and inter-chromosome rearrangements were confined to core chromosome ends and SCs, indicating a robust role of SCs in facilitating genetic exchange within fast-evolving genomic compartments. Additionally, SC carrying the avirulence gene AvrPikE modulates M. oryzae virulence against Pikh rice through horizontal transfer, loss of whole SC, and segmental deletions. Our findings establish SCs as critical players in shaping the diversity and dynamics of the pathogenic fungal genomes, highlighting them as a cradle for the variation of pathogenicity-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30fc7ba98841d16bfd287154023ca3410a86f26c" target='_blank'>
              Experimental insights into genome reconstruction driven by horizontal transfer of supernumerary chromosomes in Magnaporthe oryzae.
              </a>
            </td>
          <td>
            Zhenyu Fang, Yuyong Li, Jianqiang Huang, Meilian Chen, Xiali Chen, Xueting Mo, Z. Zhong, Xiuxiu Li, G. Lu, Guifang Lin, Zonghua Wang, Huakun Zheng
          </td>
          <td>2025-08-03</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="

Cancer remains one of the most complex diseases globally, driven by genetic and epigenetic
alterations that disrupt normal cellular processes. This complexity is further amplified by
mechanisms such as uncontrolled cell proliferation, inhibition of apoptosis, angiogenesis, immune
evasion, and metabolic reprogramming, all of which contribute to tumor growth and metastasis.
While advancements in diagnostic technologies, including next-generation sequencing and liquid
biopsies, have facilitated early detection and the development of personalized therapeutic strategies,
traditional treatments such as chemotherapy, radiotherapy, and immunotherapy continue to
face significant limitations, including drug resistance, adverse side effects, and tumor heterogeneity.
Recent studies have emphasized the potential of bacteriophages in cancer treatment. As viruses
that specifically target bacteria, phages offer unique advantages due to their genetic modifiability,
roles in immunotherapy, and capabilities in targeted drug delivery. They can reshape the tumor microenvironment,
enhance immune responses, and deliver therapeutic agents directly to cancer
cells. Moreover, phages hold promise in addressing antibiotic resistance in cancer patients and improving
the efficacy of traditional therapies through combination strategies. However, challenges
remain regarding the optimization of phage stability, bioavailability, and tumor-specific targeting.
Research and clinical trials on the therapeutic application of phages are critical to unlocking their
full potential as innovative approaches to overcome the limitations of current cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f176f476b0e17d22a339146ab7610aed3ab80193" target='_blank'>
              Bacteriophages in Cancer Treatment
              </a>
            </td>
          <td>
            Mustafa Ustundag, Berrin Ustundag
          </td>
          <td>2025-08-22</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, Hisato Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Both DNA methylation and homologous recombination (HR) are extensively studied. In bacteria, Dam methylation is the most studied DNA modification, while RecA-mediated HR is a primary mechanism to repair DNA damages including double-stranded breaks, single-stranded gaps, and stalled replication forks. While HR regulation by proteins is extensively studied, whether methylation of DNA itself directly affects the functions of RecA and HR remains unclear. Mainly by single-molecule experiments, we report that Dam methylation of single-stranded DNA (ssDNA) promotes RecA assembly, partially by reducing the effective charge of ssDNA under counterion screening. Furthermore, Dam methylation of double-stranded DNA promotes homologous pairing, joint molecule growth, and strand exchange. In cellular experiments, dam deletion impairs HR, whereas hypermethylation of the adenines in the genome enhances HR in P1 transduction assays and DNA-damage sensitivity tests without significantly upregulated HR-related genes. In addition, the preference of RecA for Dam-methylated DNA in RecA assembly and homologous pairing is conserved across divergent species covering a gram-negative bacterium Klebsiella pneumoniae, a gram-positive bacterium Bacillus subtilis, and a flowering plant Arabidopsis thaliana. Dam methylation of ssDNA increases the ATPase activity of molecular motors such as RecQ helicase that containing RecA-like domains. These findings reveal effects of DNA methylation and mechanisms regulating RecA-mediated HR and molecular motors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a366efaa4f71dbaa4fd5242b1fedea2c3fbc0149" target='_blank'>
              N6-methyladenine modification of DNA enhances RecA-mediated homologous recombination.
              </a>
            </td>
          <td>
            Xiaocong Zhao, Bin Wu, Ya-Jun Yang, Ying Li, Qi-Yuan Qiu, Liu Wang, Ya-Peng Xu, Han Gong, Lun Song, Xuejie Wang, Jing-Yao Shi, Xuefeng Chen, Shao-Ran Zhang, Qi Zong, Liang Dai, Shi-Shen Du, Yan Zhang, Wenqiang Wu, Xinghua Zhang
          </td>
          <td>2025-08-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A. Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition.
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="There is growing interest in understanding the mechanisms underlying differences in cancer incidence among species (comparative oncology). The naked mole-rat (NMR) is often referenced as "cancer-resistant" and prior studies focused on identifying mechanisms explaining this. However, efforts to assess this in vivo have been limited. Herein, we provide evidence that the NMR presents as a novel autochthonous model of lung tumor initiation, driven by an introduction of the oncogenic Eml4-Alk fusion protein using CRISPR-mediated genome editing. Whereas in mice the inversion alone is sufficient to drive tumorigenesis, the inversion alone was insufficient to drive tumorigenesis in the NMR lung and tumor development required additional losses of the tumor suppressors p53 and pRb. Our findings suggest that the proposed "resistance" of the NMR to the development of cancer may reflect that the genetic events leading to tumor initiation are likely to be comparable to those present in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a53eef9b5cb3d97ea0c88db9955706dc37ce138" target='_blank'>
              An Autochthonous Model of Lung Cancer Identifies Requirements for Cellular Transformation in the Naked Mole-Rat.
              </a>
            </td>
          <td>
            A. Shepard, Daniel K Lester, Scott Troutman, Sany Hoxha, Walid T Khaled, E. S. Smith, Thomas J. Park, Rochelle Buffenstein, Dongliang Du, Mingxiang Teng, C. Dengler-Crish, Kenneth Y Tsai, Elsa R Flores, Andrea Ventura, Joseph L. Kissil
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018e4ec4dd8ef81f5b174f65fbe205bcc992d85a" target='_blank'>
              The Impact of Polyploid Giant Cancer Cells: The Root of Stress Resilience.
              </a>
            </td>
          <td>
            Yuta Ogawa, Lydia Fisher, Tomonori Matsumoto
          </td>
          <td>2025-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The protein p53, often referred to as the “guardian of the genome,” is essential for preserving cellular balance and preventing cancerous transformations. As one of the most commonly altered genes in human cancers, its impaired function is associated with tumor initiation, development, and resistance to treatment. Exploring the diverse roles of p53, which include regulating the cell cycle, repairing DNA, inducing apoptosis, reprogramming metabolism, and modulating immunity, provides valuable insights into cancer mechanisms and potential treatments. This review integrates recent findings on p53′s dual nature, functioning as both a tumor suppressor and an oncogenic promoter, depending on the context. Wild-type p53 suppresses tumors by inducing cell cycle arrest or apoptosis in response to genotoxic stress, while mutated variants often lose these functions or gain novel pro-oncogenic activities. Emerging evidence highlights p53′s involvement in non-canonical pathways, such as regulating tumor microenvironment interactions, metabolic flexibility, and immune evasion mechanisms. For instance, p53 modulates immune checkpoint expression and influences the efficacy of immunotherapies, including PD-1/PD-L1 blockade. Furthermore, advancements in precision diagnostics, such as liquid biopsy-based detection of p53 mutations and AI-driven bioinformatics tools, enable early cancer identification and stratification of patients likely to benefit from targeted therapies. Therapeutic strategies targeting p53 pathways are rapidly evolving. Small molecules restoring wild-type p53 activity or disrupting mutant p53 interactions, such as APR-246 and MDM2 inhibitors, show promise in clinical trials. Combination approaches integrating gene editing with synthetic lethal strategies aim to exploit p53-dependent vulnerabilities. Additionally, leveraging p53′s immunomodulatory effects through vaccine development or adjuvants may enhance immunotherapy responses. In conclusion, deciphering p53′s complex biology underscores its unparalleled potential as a biomarker and therapeutic target. Integrating multi-omics analyses, functional genomic screens, and real-world clinical data will accelerate the translation of p53-focused research into precision oncology breakthroughs, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6708657f0ff7cd8075abb4af281defbd7a1577" target='_blank'>
              The Multifaceted Role of p53 in Cancer Molecular Biology: Insights for Precision Diagnosis and Therapeutic Breakthroughs
              </a>
            </td>
          <td>
            Bolong Xu, Ayitila Maimaitijiang, Dawuti Nuerbiyamu, Zhengding Su, Wenfang Li
          </td>
          <td>2025-07-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8304854a46599698bac2a6e2d8f83da3bfb9d71b" target='_blank'>
              DNA2 enables growth by restricting recombination-restarted replication.
              </a>
            </td>
          <td>
            Jessica J R Hudson, Rowin Appanah, David Jones, Kathryn Davidson, Alice M Budden, Alina Vaitsiankova, Kok-Lung Chan, Keith W. Caldecott, Antony M. Carr, Ulrich Rass
          </td>
          <td>2025-09-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Mediator complex subunit 10 (MED10) serves as a critical regulator of eukaryotic gene expression by facilitating RNA polymerase II activity. Our investigation aims to characterize MED10's functional contributions and underlying molecular pathways in hepatocellular carcinoma (HCC) development.


METHODS
MED10 expression patterns in HCC and their correlation with clinicopathological parameters and patient outcomes were examined using bioinformatics databases and immunohistochemistry. Subsequently, we systematically investigated the biological functions of MED10 in the malignant progression of HCC through comprehensive in vitro experiments, including assessments of cell migration (transwell and wound healing assays), proliferative capacity (cell counting kit-8, colony formation, and 5-Ethynyl-2'-deoxyuridine assays), and cell cycle progression (flow cytometry analysis). Furthermore, we elucidated the underlying molecular mechanisms using real-time quantitative PCR (RT-qPCR), western blotting, immunofluorescence staining, and public database analyses. Furthermore, an in vivo subcutaneous xenograft model was employed to validate MED10's impact on tumor growth.


RESULTS
The results revealed a marked increase in MED10 expression levels within HCC tissues, showing a strong association with unfavorable clinical outcomes. Mechanistically, MED10 induced the epithelial-mesenchymal transition (EMT) and enhanced HCC cell migration. Moreover, MED10 overexpression drives HCC cell cycle progression and proliferation by activating rapidly accelerated fibrosarcoma 1 (RAF1), a process potentially mediated through the mitogen-activated protein kinase (MEK)/extracellular signal-regulated kinase (ERK)/cellular myelocytomatosis oncogene (c-Myc) signaling axis.


CONCLUSION
MED10 promotes HCC cell migration and EMT but, more importantly, also drives cell cycle progression and proliferation via RAF1 activation, and is related to the MEK/ERK/c-Myc axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83554affa0ce8872e89c5a91b5147c43bb5338b7" target='_blank'>
              MED10 as a Novel Oncogenic Driver in HCC: Promoting Cell Cycle Progression and Proliferation Through RAF1 Activation.
              </a>
            </td>
          <td>
            Junhao Liu, Yongxue Lv, Kejun Liu, Zhengquan Li, Bendong Chen, Yang Bu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) remains a global health burden, with disproportionately high mortality in China’s Guangxi region, where endemic Clonorchis sinensis (C. sinensis) infection coincides with elevated HCC incidence. Preliminary single-cell sequencing revealed marked overexpression of cluster of differentiation 24 (CD24) in HCC tissues, suggesting its potential pathological role. This study aims to elucidate the oncogenic mechanisms of C. sinensis excretory-secretory products (CsESPs) and their link to CD24-mediated HCC progression. Methods We employed an integrated clinical and experimental approach. First, clinical cohort analysis assessed CD24 expression in C. sinensis-associated HCC cases. Multiplatform bioinformatics validation (GEPIA/UALCAN/TIMER) evaluated CD24’s prognostic significance and immune microenvironment modulation. Functional studies (quantitative polymerase chain reaction (qPCR), Western blotting, CCK-8 assays, flow cytometry) examined CsESPs’ effects on CD24 expression, cell proliferation, and apoptosis. Mechanistic investigations (chromatin immunoprecipitation, dual-luciferase reporter assays) identified E2F1-mediated transcriptional activation of CD24. siRNA-mediated CD24 knockdown validated its role in CsESPs-driven oncogenesis. Additionally, the expression of immune checkpoint (CTLA-4, LAG-3) was assessed in the co-cultures of peripheral blood mononuclear cells (PBMCs)–HCC cells. Results Clinical cohort analysis confirmed significant CD24 upregulation in HCC, particularly in C. sinensis-infected cases. Bioinformatic analyses linked high CD24 expression to poor prognosis and immune microenvironment alterations. Functional assays demonstrated that CsESPs enhance CD24 expression, promoting proliferation and inhibiting apoptosis. Mechanistically, E2F1 directly binds to CD24 promoter, driving its transcription upon CsESPs exposure. CD24 silencing reversed CsESPs-induced oncogenic effects. Furthermore, CsESPs upregulated immune checkpoints (CTLA-4, LAG-3) in the co-cultures of PBMC–HCC cells, an effect reversed by CD24 knockdown. Conclusions Our findings establish a novel parasitic carcinogenesis paradigm wherein C. sinensis promotes HCC development through E2F1-mediated transcriptional activation of CD24, simultaneously identifying prognostic biomarkers and therapeutic targets while suggesting combinatory immunotherapy strategies for parasite-associated HCC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13071-025-06979-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44a225526ec9b8357a014fc963659daa1b7f9c" target='_blank'>
              Clonorchis sinensis-driven hepatocarcinogenesis via E2F1-CD24 transcriptional axis: mechanistic and therapeutic implications
              </a>
            </td>
          <td>
            Wen-Min Lu, Jin Yan, Zhao-Ji Liu, Yong Wu, Qian-Ru Cui, Ji Feng, Yu Chen, Guangzhi Zhu, Tao Peng, Jing Zhou, Guo-Dong Lu
          </td>
          <td>2025-08-19</td>
          <td>Parasites & Vectors</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcf3324435cca52660fd9b480c9db910a863585b" target='_blank'>
              Teniposide Triggers DNA Repair Inhibition by Binding and Ubiquitination of Apurinic/Apyrimidinic Endonuclease 1 to Boost Oxidative DNA Damage for Lung Cancer Destruction.
              </a>
            </td>
          <td>
            Ning Han, Jun Hu, Zhong-Xiong Xie, Lei Zhang, Liu-Gen Li, Xin-Ran Yu, Cunqing Kong, Fan Leng, Cui Hu, Qiufang Zhang, Mei-Fang Wang, Tong-fei Li
          </td>
          <td>2025-08-08</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Human topoisomerase II alpha (TOP2α) resolves DNA intertwines between sister chromatids during mitosis. How cohesin, an SMC complex that holds sister chromatids, affects TOP2α decatenation is unclear. To addres this, we developed a quadruple-trap optical tweezers assay to create DNA braids and study TOP2α decatenation at the single-molecule level in real-time. We show that TOP2α resolves both single and multiple braids but becomes inefficient at forces exceeding 28 pN. TOP2α is sensitive to DNA geometry, exhibiting a chiral preference for right-handed braid crossings, and requires loading directly at DNA crossovers to act. Pre-loading TOP2α onto individual DNA strands before braid formation, in the presence of ATP, prevents decatenation. Finally, we show that human cohesin, but not condensin I, binds stably to DNA braids and blocks TOP2α activity. Our study provides novel insights into the role of substrate accessibility in regulating TOP2α‘s activity and highlights cohesin as a barrier to decatenation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7fc5b97ef55e14d8e05557d238a795889fb56aa" target='_blank'>
              Substrate accessibility regulation of human TopIIa decatenation by cohesin
              </a>
            </td>
          <td>
            E. Cutts, Sanjana Saravanan, Paul Girvan, Benjamin Ambrose, Gemma L M Fisher, David S. Rueda, Luis Aragon
          </td>
          <td>2025-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Stem cells are essential for tissue maintenance, repair, and regeneration, yet their dysregulation gives rise to cancer stem cells (CSCs), which drive tumor progression, metastasis, and therapy resistance. Despite extensive research on stemness and oncogenesis, a critical gap remains in our understanding of how the transcriptomic landscapes of normal somatic stem cells (SSCs) diverge from those of CSCs to enable malignancy. This review synthesizes current knowledge of the key signaling pathways (Wnt, Notch, Hedgehog, TGF-β), transcription factors (Oct4, Sox2, Nanog, c-Myc, YAP/TAZ), and epigenetic mechanisms (chromatin remodeling, DNA methylation, microRNA regulation) that govern stemness in SSCs and are hijacked or dysregulated in CSCs. We highlight how context-specific modulation of these pathways distinguishes physiological regeneration from tumorigenesis. Importantly, we discuss the role of epithelial–mesenchymal transition (EMT), cellular plasticity, and microenvironmental cues in reprogramming and maintaining CSC phenotypes. By integrating transcriptomic and epigenetic insights across cancer biology and regenerative medicine, this review provides a framework for identifying vulnerabilities specific to CSCs while still preserving normal stem cell function. Understanding these distinctions is essential for the development of targeted therapies that minimize damage to healthy tissues and advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb9e90607e3b0ca792b0cb08f2440605b878350" target='_blank'>
              Transcriptomic Comparisons of Somatic and Cancer Stem Cells
              </a>
            </td>
          <td>
            Austin Drysch, Arun Ahuja, Dillan Prasad, Rishi Jain, Sharbel Romanos, A. Alwakeal, Christopher S. Ahuja
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f06d8c3c1ce1c0e2c42d69d2aefb0ff5769089e" target='_blank'>
              STAT3 sustains tumorigenicity following mutant KRAS ablation.
              </a>
            </td>
          <td>
            S. D'Amico, V. Kirillov, Jingxuan Liu, Zhijuan Qiu, Xinyuan Lei, Hong Qin, Brian S. Sheridan, Nancy C. Reich
          </td>
          <td>2025-08-26</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="We investigated whether DNA methylation and cell-free DNA (cfDNA) fragmentation patterns can improve circulating tumor DNA (ctDNA) detection in advanced pancreatic cancer. In a cohort of 33 patients, ctDNA detection was performed in a tumor-agnostic fashion using DNA methylation, cfDNA fragment lengths, and 4-mer 5' end motifs. Machine learning models estimating ctDNA levels were built for each individual detection method and their combination. All models significantly differentiated ctDNA levels in patients from healthy individuals (P < 0.001). Using the highest estimated levels in healthy volunteers as cutoffs, ctDNA was detected in 79%, 67%, 67%, and 55% of patients using methylation, fragment length, end motifs, and the combined model, respectively. Univariable Cox regression showed that all ctDNA level estimates were associated with increased hazard ratios (HR, all P < 0.001) for progression-free survival (PFS) and overall survival (OS). Multivariable Cox regression confirmed ctDNA levels as an independent predictor of PFS (HR = 1.9, P < 0.001) and OS (HR = 2.7, P < 0.001). Our findings suggest that machine learning models based on DNA methylation, cfDNA fragment lengths, and cfDNA end motifs can estimate ctDNA levels and predict clinical outcomes in advanced pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a97d214cc95d4d912f037f8e83c7c1e9be3b6c1d" target='_blank'>
              Tumor-agnostic detection of circulating tumor DNA in patients with advanced pancreatic cancer using targeted DNA methylation sequencing and cell-free DNA fragmentomics.
              </a>
            </td>
          <td>
            Morten Lapin, Kjersti Tjensvoll, Karin H Edland, Satu Oltedal, H. Garresori, Bjørnar Gilje, Saga Ekedal, T. Eftestøl, J. Kvaløy, Filip Janku, Oddmund Nordgård
          </td>
          <td>2025-08-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract RECQ1, the most abundant RecQ helicase in human cells, is involved in telomere maintenance in ALT cells and plays a critical role in maintaining genomic integrity and stability. Here, we present five high-resolution crystal structures that systematically reveal a novel mechanism by which the RECQ1 helicase recognizes and regulates G-quadruplex (G4) DNA structures. Our results demonstrate that DNA binding induces intra-subunit rearrangement in RECQ1, transitioning it from a closed to an open conformation. This rearrangement alters the stability of the dimer interface. G4 recognition and unwinding are driven by coordinated interactions between the D1/D2 domains and the single-stranded DNA (ssDNA)-binding channel. This dual engagement aligns the G4 tetrad in a geometry favorable for unwinding. ATP hydrolysis facilitates ssDNA translocation, positioning the β-hairpin to disrupt hydrogen bonds—unraveling G4 structures in a manner analogous to the unwinding of dsDNA. This study proposes a mechanistic model for RECQ1-mediated G4 unwinding and elucidates how RECQ1 recognizes and unwinds distinct DNA structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b4dc7d9e0b357d6d1b7be83e6503a6870b2fd66" target='_blank'>
              Structural mechanism of RECQ1 helicase in unfolding G-quadruplexes compared with duplex DNA
              </a>
            </td>
          <td>
            Ze-Yu Song, Xin Zhang, Xia Ai, , , Philippe Fossé, , Olivier Mauffret, Stéphane Réty, Xu-Guang Xi
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b54093d228fc592b42597f1baffae61141e09d6b" target='_blank'>
              DNA Methylation in Ovarian and Endometrial Cancers: Predictive and Mechanistic Roles in PARP Inhibitor and ICI Response.
              </a>
            </td>
          <td>
            Shuhei Kitamura, Ayumi Taguchi, Kana Tamai, Yoko Yamamoto, Anh Quynh Duong, Daisuke Yoshimoto, Ayako Mori, Aya Ishizaka, Saki Tsuchimochi, K. Sone, Masahito Kawazu, K. Oda, Yasushi Hirota
          </td>
          <td>2025-09-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Systems that perform continuous hypermutation of designated genes without compromising the integrity of the host genome can substantially accelerate the evolution of new or enhanced protein functions. We describe an orthogonal DNA replication system in Escherichia coli based on the controlled expression of the replisome of bacteriophage T7 (T7-ORACLE). The system replicates circular plasmids that enable high transformation efficiencies and seamless integration into standard molecular biology workflows. Engineering of T7 DNA polymerase yielded variant proteins with mutation rates of 1.7 × 10-5 substitutions per base in vivo-100,000-fold above the genomic mutation rate. We demonstrated continuous evolution using the T7 replisome by expanding the substrate scope of TEM-1 β-lactamase and increasing activity 5000-fold against clinically relevant monobactam and cephalosporin antibiotics in less than 1 week.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed28e398210a42a2a56f735d7fe9f761f9cf0a6" target='_blank'>
              An orthogonal T7 replisome for continuous hypermutation and accelerated evolution in E. coli.
              </a>
            </td>
          <td>
            Christian S Diercks, Philipp Sondermann, Cynthia Rong, Thomas G Gillis, Yahui Ban, Celine Wang, David A. Dik, Peter G. Schultz
          </td>
          <td>2025-08-07</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objective Patients diagnosed with metastatic pheochromocytoma/paraganglioma (PCC/PGL) have limited treatment options. In some cases, peptide receptor radionuclide therapy (PRRT) is followed by an eruption of metastases, possibly originating from tumor cells with a radioresistant phenotype. However, the underlying mechanisms of radioresistance in PCC/PGL remain largely unknown and appropriate models are missing. Methods Two genetically modified mouse pheochromocytoma (MPC) cell lines, one positive and one negative for hypoxia-inducible factor 2α expression (MPC+HIF-2α and MPC+EV [empty vector], respectively), were X-ray-conditioned through fractionated irradiation at sublethal doses. Two procedures were tested: one allowed for recovery between each irradiation step (recIR), while the other demanded daily irradiation (dayIR). Changes in cell morphology, growth rates, and DNA repair (γH2AX immunostaining) were characterized in response to irradiation. Results We generated two MPC+HIF-2α- and two MPC+EV-derived cell lines that tolerate irradiations with X-rays at dose fractions of 2 Gy per day without significant growth inhibition. All recIR-and dayIR-conditioned cell lines showed increased DNA repair capacity. Morphological changes toward stronger clustering and slower growth were more pronounced in dayIR-conditioned than in recIR-conditioned cell lines. X-ray-conditioned MPC+HIF-2α cells showed the highest increase in resistance to X-ray-treatment with dose fractions up to 5 Gy per day. Conclusion The herein established X-ray-conditioned MPC cell lines represent PCC/PGL models with a radioresistant phenotype. Further investigations on the radiation-induced genetic responses of these cell lines, their corresponding tumor spheroids, and tumor allografts in mice will help to elucidate the underlying mechanisms of acquired radioresistance and radionuclide therapy-induced metastatic eruption in PCC/PGL. Lastly, the suitability of advanced PRRT and complementary treatments can be tested to improve theranostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42539527ca697e5d2588e236622b3f72185855c8" target='_blank'>
              Radioresistant mouse pheochromocytoma cell lines
              </a>
            </td>
          <td>
            S. Lemm, Marcel Gebhardt, Thomas Groß, Susan Richter, M. Ullrich, Jens Pietzsch
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Kaposi’s sarcoma herpesvirus (KSHV) establishes lifelong oncogenic infection in lymphatic endothelial cells (LECs) by ensuring episomal maintenance of its genome via the viral protein LANA. Efficient viral genome maintenance typically involves host DNA replication and episome tethering, but the extent of cell-type-specific regulation remains unclear. Here, we identify that KSHV hijacks the pioneering function of the endothelial-specific transcription factor SOX18 to facilitate persistence of viral episomes. Upon infection, LANA co-opts SOX18 to recruit the SWI/SNF chromatin-remodeling complex via its ATPase subunit BRG1, enhancing chromatin accessibility and enabling efficient viral genome persistence. Disruption of SOX18 or BRG1—genetically or pharmacologically—leads to reduced episome load and attenuated hallmarks of virus infection. This work highlights how viruses can harness lineage-specific transcriptional regulators to establish persistent nuclear retention of their episome into the host genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d22064019de27118b6d0d659b640014ad8ce7faf" target='_blank'>
              Oncogenic virus hijacks SOX18 pioneer function to enhance viral persistence
              </a>
            </td>
          <td>
            P. Ojala, Krista Tuohinto, Matthew S. Graus, Peyton Staab, Ville Tiusanen, Liangru Fei, Susav Pradhan, Y. Wong, Simon Weissmann, Jieqiong Lou, Elizabeth Hinde, Justin Wong, Quintin Lee, Alexey Terskikh, Martin Alvarez-Kuglen, Tara Karnezis, Thomas Günther, A. Grundhoff, Biswajyoti Sahu, Mathias Francois
          </td>
          <td>2025-08-18</td>
          <td>Research Square</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer.
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Primary cilia have been considered tumor-suppressing organelles in cholangiocarcinoma (CCA), though the mechanisms behind their protective role are not fully understood. This study investigates how the loss of primary cilia affects DNA damage response (DDR) and DNA repair processes. Human cholangiocyte cell lines were used to examine the colocalization of DNA repair proteins at the cilia and assess the impact of experimental deciliation on DNA repair pathways. Deciliation was induced using shRNA knockdown or CRISPR knockout of IFT20, IFT88, or KIF3A, followed by exposure to the genotoxic agents cisplatin, methyl methanesulfonate (MMS), or irradiation. Cell survival, cell cycle progression, and apoptosis rates were evaluated, and DNA damage was assessed using comet assays and γH2AX quantification. An in vivo liver-specific IFT88 knockout model, generated using Albumin-Cre/Lox recombination, was used to study the loss of primary cilia in the liver. Results showed that RAD51 localized predominantly at the base of the cilium, while ATR, PARP1, CHK1, and CHK2 were also detected within the ciliary shaft. Deciliated cells displayed dysregulation in critical DNA repair pathways. These cells also showed reduced survival and increased S-phase arrest after genotoxic challenges as compared to ciliated cells. Enhanced DNA damage was observed via increased γH2AX signals and comet assay results. An increase in γH2AX expression was also observed in our in vivo model, indicating elevated DNA damage. Additionally, key DDR proteins such as ATM, p53, and p21, were downregulated in deciliated cells after irradiation. This study underscores the crucial role of primary cilia in regulating DNA repair and suggests that targeting cilia-related mechanisms could present a novel therapeutic approach for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/911ceef3e7f2a0c36abb13e2df295806ae55af3b" target='_blank'>
              Cholangiocytes' Primary Cilia Regulate DNA Damage Response and Repair.
              </a>
            </td>
          <td>
            E. Peixoto, Kishor Pant, Seth Richard, Juan Pablo Popoca, J. Abrahante, Wioletta Czaja, S. Gradilone
          </td>
          <td>2025-08-12</td>
          <td>American journal of physiology. Gastrointestinal and liver physiology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>