<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-14 09:36:26 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>36</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>73</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>341</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>299</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865d140b6544d7dfc0dca54fb01a54fdf212f1a" target='_blank'>
              Mitotic BLM functions are required to maintain genomic stability
              </a>
            </td>
          <td>
            Tamara Eleanore Hamann, A. Wieland, Andrea Tirincsi, Kruno Vukušić, Farbod Mohseni, René Wardenaar, Marialucrezia Losito, Ipek Ilgin Gönenc, B. Wollnik, F. Foijer, I. Tolic, Zuzana Storchová, M. Räschle
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extreme Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cfe224e356c6e68ceccf6ec7ce94377f472868" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Zhangshuai Dai, Julia Thiel, Meghana Manjunath, Nadine Schacherer, Bishakha Joyeeta Saha, Yara Souto, A. Abdelrahman, Frank Essmann, Sven Beyes, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10436da9e9bcdf67b6e4cfcf17b6e58df4672a34" target='_blank'>
              Spatio-temporal chromosomal arrangements by late-replicating heterochromatin
              </a>
            </td>
          <td>
            Satyadev Polisetty, Shuvadip Dutta, Rakesh Netha Vadnala, D. Notani, R. Padinhateeri, K. Sanyal
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of differentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, differential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and differentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by differential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be sufficient for gene regulation. Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS - The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops - Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis - Chromatin domain boundaries are weakened during cancer progression - Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops - Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression - Prominent changes in chromatin loops occur at a small subset of differentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Platinum (Pt) drugs are widely utilized in cancer chemotherapy. Although cytotoxic and resistance mechanisms of Pt drugs have been thoroughly explored, it remains elusive what factors affect the receptiveness of DNA to drug-induced damage in nuclei. Here, we demonstrate that nuclear locations of chromatin play a key role in Pt drug-induced DNA damage susceptibility in vivo. By integrating data from damage-seq experiments with 3D genome structure information, we show that nuclear locations of chromatin relative to specific nuclear bodies and compartments explain patterns of cisplatin DNA damage susceptibility. This aligns with observations of cisplatin enrichment in biomolecular condensates at certain nuclear bodies. Finally, 3D structure mapping of DNA damage reveals characteristic differences between nuclear distributions of oxaliplatin-induced DNA damage in drug resistant versus sensitive cells. DNA damage increases in gene-poor chromatin at the nuclear periphery, while it decreases in gene-rich regions located at nuclear speckles. This suggests a strategic redistribution of Pt drug-induced damage in nuclei during chemoresistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad670ec02d61bb33260095b67444d0174970be28" target='_blank'>
              3D genome organization shapes DNA damage susceptibility to platinum-based drugs
              </a>
            </td>
          <td>
            Ye Wang, Aslı Yıldırım, L. Boninsegna, Valentina Christian, Sung-Hae L Kang, X. J. Zhou, Frank Alber
          </td>
          <td>2025-05-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aebba9c32cce818d2e97fcfa124c22eff76b4f3" target='_blank'>
              LSH-mediated resolution of R-loops mitigates transcription-replication conflicts to preserve genomic stability in prostate cancer cells
              </a>
            </td>
          <td>
            Kai Ni, Yu Zhang, Kun Zheng, Jianwen Huang, Nuorong Xiong, Rong Liang, Yingxiao Chen, Honglin Jin, Youlong Hai, Wang Li, Jiayi Ma, Ranxing Yang, Guanggai Xia, Yueqing Bai, Lujie Song, Xiaoyong Hu, Zijian Tang, Qiang Fu
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c19d5f1ca11d81a47f9f7924f6f192beb7cb85" target='_blank'>
              Cancer type-specific association of p53 deficiency with aneuploidy and chromosome losses
              </a>
            </td>
          <td>
            Joana F. Marques, Marco António Dias Louro, Teresa Davoli, G. Kops
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, Alice Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer, closely associated with tumor evolution, metastasis, immune evasion, and resistance to treatment. CIN is driven by persistent chromosome missegregation, resulting in abnormal chromosomal copy numbers and promoting tumor progression through mitotic errors, faulty chromosome segregation, and the formation of micronuclei or chromosome bridges. Previous studies have demonstrated that p62 localizes to micronuclei, where it interacts with mitochondria, enhancing ROS-mediated cysteine oxidation and promoting p62 homo-oligomerization. This disrupts repair by recruiting CHMP7 and other ESCRT-III components, impairing their function in restoring the micronuclear envelope. Notably, we observed a time-dependent increase in nuclear p62 levels following radiation therapy, which correlated with enhanced chromatin fragmentation and chromothripsis. While p62 inhibits homologous recombination repair and promotes non-homologous end joining, we hypothesize that radiation-induced nuclear p62 accumulation impairs nuclear envelope resealing by inhibiting ESCRT-III components, thereby contributing to chromosomal fragmentation and chromothripsis. These chromosomal alterations may play a significant role in tumor evolution and treatment resistance. Our findings suggest that irradiation-induced p62 accumulation is crucial for chromothripsis and may affect nuclear repair processes, impacting chromosomal stability. Future research will focus on elucidating how p62 contributes to these abnormalities and their implications for tumor progression and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a93ad5e20684542464f1d1902962e0915d7957" target='_blank'>
              Irradiation-induced increase in nuclear p62 levels contributes to chromosomal fragmentation and chromothripsis.
              </a>
            </td>
          <td>
            Xiaojuan Yang, Ying Zhou, Buzhe Zhang, Xue Tao, Weili Qi, Hong Wu
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Plasmids have fundamentally transformed how we resolve regulatory grammar across the tree of life. However, although chromatin plays an integral role in regulating the function of regulatory elements along the nuclear genome, our understanding of how, or whether, similar chromatin structures form on plasmids transfected into mammalian cells remains limited. We demonstrate that plasmid single-molecule chromatin fiber sequencing (plasmid Fiber-seq) can accurately map chromatin architectures along individual, full-length transfected plasmid molecules at near single-nucleotide resolution. Application of plasmid Fiber-seq to diverse plasmids and cell lines demonstrates that plasmids are chromatinized in a sequence-dependent organized manner and adopt a heterogeneous and incomplete chromatin architecture relative to nuclear-encoded chromatin fibers. We show that the focal occupancy of nucleosomes and transcription factors along transfected plasmids is central to their transcriptional activity within mammalian cells, and demonstrate that plasmids indeed are capable of recapitulating nuclear genome-encoded chromatin architectures, although not always. Finally, we demonstrate that combining plasmid Fiber-seq with high-throughput reporter assays can establish the molecular mechanisms underlying pathogenic non-coding variants, including disentangling the effects of transcriptional activators and repressors with near-single-nucleotide resolution. Overall, our findings reveal the principles by which plasmid-based assays can be used for accurate fine-scale mapping of chromatin-dependent regulatory grammar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d71ed390846f18de84c61671f46cfb09ecd9af" target='_blank'>
              Principles and functional consequences of plasmid chromatinization in mammalian cells
              </a>
            </td>
          <td>
            Ben Mallory, Thomas W. Tullius, Carina G. Biar, Jonas A. Gustafson, Stephanie C. Bohaczuk, Danilo Dubocanin, L. Starita, A. Stergachis
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The evolution of techniques used to identify structural variants (SVs) and copy number variants (CNVs) in genomes have seen significant development in the last decade. With the growing use of more technologies including chromosomal microarray, genome sequencing and genome mapping in clinical cytogenetics laboratories, reporting the frequency of SVs and CNVs has increased the complexity of genomic results. In conventional testing (e.g. karyotype or FISH) individual cells are analyzed and abnormalities are reported at the single cell level directly as a proportion of the analyzed cells. Whereas for bulk genome assays structural and sequence changes are often reported as variant allele frequencies and fractional copy number states. The International System of Cytogenomic Nomenclature (ISCN) recommends converting these values into a “proportion of the sample”, which requires different calculations and underlying assumptions based on the data type. This review illustrates how the different methods of interpreting and reporting data are performed and identifies challenges in the conversion of these values to a proportion of the sample. We stress the need for careful interpretation of data with consideration for factors that may alter how proportions are reported including overlapping SVs and CNVs or regions with acquired homozygosity. We also demonstrate, using validation data of SVs and CNVs tested by multiple techniques how results are largely consistent across methodologies, but can show dramatic differences in rare circumstances. This review focuses on illustrating many of the challenges with aligning reporting using different techniques and their underlying assumptions. As hematologic disease classifications start to incorporate numeric limits (e.g. VAF defining thresholds), it is important for laboratory geneticists, pathologists and clinicians to appreciate the differences in methodologies, potential pitfalls and the nuances when comparing bulk genome analyses to the more conventional single cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883b8047db9fe9c01ef1b4982d92e6a1387b4be0" target='_blank'>
              What the VAF? A guide to the interpretation of variant allele fraction, percent mosaicism, and copy number in cancer
              </a>
            </td>
          <td>
            Adam C Smith, Hubert Tsui, Sila Usta, Jose-Mario Capo-Chichi
          </td>
          <td>2025-07-08</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/037c750c2146efe1a36aa3818222e02772636491" target='_blank'>
              Mitotic phosphorylation of ADAR1 regulates its centromeric localization and is required for faithful mitotic progression
              </a>
            </td>
          <td>
            Yuxi Yang, Mai Kubota, Kokone Hasegawa, Chao Zeng, Kazuko Nishikura, Michiaki Hamada, Masayuki Sakurai
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription-replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription-induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription-Replication Conflicts: Unlocking New Frontiers in Cancer.
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc53004c4a4eae46c08e03815bc43c29345a6bec" target='_blank'>
              The landscape and consequences of transcription stress
              </a>
            </td>
          <td>
            Osama Hidmi, Sara Oster Flayshman, Diala Shatleh, Jonathan Monin, Rami I. Aqeilan
          </td>
          <td>2025-05-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ploidy determination across the genome has been challenging for low-pass-WGS tumor-only samples. We present BACDAC, a method that calculates tumor ploidy down to 1.2X effective tumor coverage. Allele fraction patterns displayed in the Constellation Plot verify tumor ploidy and reveal subclonal populations. BACDAC outputs a metric, 2N+LOH, that when combined with ploidy better distinguishes near-diploid from high-ploidy tumors. Validated using TCGA, BACDAC had good agreement with other methods and 88% agreement with experimental methods. Discrepancies occur mainly when BACDAC predicts diploidy with subclones rather than high-ploidy. Applied to 653 low-pass-WGS samples spanning 12 cancer subtypes, BACDAC calls 40% as high-ploidy. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03599-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a5801328a2687dfdef502362bf6b982a0712bd" target='_blank'>
              Tumor ploidy determination in low-pass whole genome sequencing and allelic copy number visualization using the Constellation Plot
              </a>
            </td>
          <td>
            Sarah H. Johnson, J. Smadbeck, Roman M. Ženka, Michael T. Barrett, Athanasios Gaitatzes, Arnav Solanki, Angela B. Florio, M. Borad, John C. Cheville, G. Vasmatzis
          </td>
          <td>2025-05-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779c13815a53f4713be7d07b72908d9198ac23e6" target='_blank'>
              Nucleosome-mediated conformational switches in micro-eccDNAs
              </a>
            </td>
          <td>
            Sarah Harris, Victor Velasco Berrelleza, William Sandel, T. Sutthibutpong, Victoria N. Parikh, Euan A. Ashley, Andrew Fire, S. Levene, Thomas C Bishop, M. Shoura
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer.
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, Ping Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Imprinted genes are epigenetically regulated in normal tissues to follow monoallelic expression according to the parent of origin of each allele. Some of these patterns are dysregulated in cancer.


RESULTS
We developed a novel computational multi-omic pipeline to evaluate monoallelic and biallelic expression patterns based on matched RNA-seq expression data, whole-exome sequencing information, and copy number data. We analyzed allelic expression of the entire genes, individual isoforms, and each exon of 59,283 autosomal protein-coding and ncRNA genes, with a focus on 94 genes previously reported to be imprinted. We analyzed 108 cell lines from 9 different tumor histologies using molecular data from the DepMap Portal for the Cancer Cell Line Encyclopedia. Allelic expression patterns of imprinted genes and isoforms in tumor cells were variable. We also identified additional genes and isoforms with predominantly monoallelic expression due to a variety of potential mechanisms. We provide a novel public dataset of transcriptome-wide allelic expression patterns in cell lines from diverse tumor categories, which can serve as a resource for future cancer studies. We examined associations of in vitro cell line response to antitumor agents and repurposed drugs with allelic patterns and overall levels of isoform expression of imprinted genes and of additional genes with predominantly monoallelic expression. Drug response was associated with isoform expression patterns of multiple imprinted genes including CPA4, DGCR6, DNMT1, GNAS, GRB10, H19, NAA60, OSBPL5, PHACTR2, and ZFAT, predominantly monoallelically expressed MAP2K5 and BCLAF1, and additional predominantly monoallelically expressed genes. Multiple associations may be related to mechanisms of drug activity, including associations between the response to the DNA damaging agents and allelic expression of ZFAT, CDC27, and BCLAF1 isoforms, and the response to inhibitors of multiple signaling pathways with expression patterns of GNAS isoforms.


CONCLUSIONS
Tumor cells have a range of monoallelic and biallelic expression patterns in both imprinted and non-imprinted genes and are likely affected by the complex interplay among changes in allelic expression, sequence variants, copy number changes, and expression changes of biologically important genes. Multiple isoform-specific patterns of allelic expression were associated with drug response, indicating complex mechanisms of cancer chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e53122e0648c0b26f74137cf51cf8b0a5877a45c" target='_blank'>
              Allelic expression patterns of imprinted and non-imprinted genes in cancer cell lines from multiple histologies
              </a>
            </td>
          <td>
            Julia Krushkal, Travis L. Jensen, George Wright, Yingdong Zhao
          </td>
          <td>2025-05-25</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e17121


 Background:
 Chromosome arm 8q is frequently amplified in prostate cancer, harboring genes that drive tumor progression, therapy resistance, and poor outcomes. Key loci include
 MYC
 , which regulates cell proliferation;
 AR
 , central to prostate cancer biology and resistance; and
 RAD21
 , essential for chromosomal stability. While the prognostic implications of 8q gains have been explored, their role in driving racial disparities in prostate cancer outcomes remains underexplored. This study investigates chromosome 8q amplification as a prognostic biomarker, examining its differential prevalence and impact across racial groups to elucidate its contribution to disparities in disease progression and patient survival.
 Methods:
 Using cBioPortal's clinical and genomic data from Memorial Sloan Kettering Cancer Center, this work examined two datasets from 2021 and 2024. Comprehensive genomic profiling included copy number alterations, mutational co-occurrence of important 8q genes, and mutational analysis. Multivariate Cox regression and Kaplan-Meier survival analysis were conducted to evaluate the impact of 8q amplifications on overall survival (OS) and disease outcomes, stratified by racial demographic groups and disease stages.
 Results:
 In this cohort of 1,577 prostate cancer patients, 652 (41%) exhibited chromosome 8q amplification, which correlated with aggressive tumor phenotypes. The most common amplifications were observed in PTEN (21%), FOXA1 (17%), AR (16%), and MYC (7%),, with notable co-occurrences observed between MYC-RAD21, MYC-NBN, and PRDM14-ELOC (
 P
 = <0.001), linking these alterations to genomic instability. Kaplan-Meier analysis showed patients with 8q gain had significantly inferior overall survival (HR = 2.31, 95% CI: 1.99 - 2.69,
 P
 < 0.001, mOS gain vs neutral: 35mo vs 55mo). Black patients exhibited a higher frequency of 8q gain (10% vs. 6.1%,
 P
 = 0.003) and distinct oncogenic profiles, including MYC (12%), RAD21 (7%), and LYN (4%) alterations. While Black patients experienced inferior overall outcomes, chromosome 8q amplification was a consistent predictor of poor prognosis across all racial groups.
 Conclusions:
 This study establishes chromosome 8q amplification as a robust prognostic biomarker in prostate cancer. The integration of clinical and genomic data highlights the potential of 8q amplification to guide personalized therapeutic strategies. These findings underscore the value of incorporating 8q analysis into clinical risk stratification frameworks to enhance treatment decision-making and address racial disparities in prostate cancer outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844ee49cccb708b6dfbb629567a7cde15db25ac6" target='_blank'>
              Prognostic impact and racial disparities of chromosome 8q gains in prostate cancer: Insights from integrated molecular profiling.
              </a>
            </td>
          <td>
            Chidiebube J. Ugwu, G. Laliotis, U. C. Okere, Nneoma Ubah, Elvis Obomanu, Colton Jones, Muluken Megiso, Nzubechukwu J. Ugochukwu, Gabor Varadi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="To systematically investigate the molecular and pathological mechanisms of enhancer RNA (eRNA)–mediated transcriptional regulation in glioma recurrence and progression, transcriptomic, regulatome, and genomic data were integrated to analyze eRNA behavior in lower-grade gliomas (stages II/III) and glioblastomas (stage IV). Most eRNAs exhibited dynamic expression during glioma progression, regulated by master transcription factors (TFs) and affected by genomic mutations. The constructed perturbed TF-mediated eRNA-promoter regulatory landscape revealed that rewiring eRNA-promoter networks contributed to glioma malignancy. Drug response–related eRNAs associated with poor prognosis were identified, highlighting their clinical potential. Overall, integrative analysis highlights the critical role of eRNA-mediated regulatory rewiring in glioma progression, providing valuable insights into transcriptional mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b9a681aad2033d6a1c3522f91909b48b06e614" target='_blank'>
              Enhancer RNA–mediated transcriptional regulatory programs reveal the malignant progression of glioma
              </a>
            </td>
          <td>
            Yingying Ma, Shuyuan Wang, Yijie Jia, Shijie Wang, Wendian Liu, Jiaxin Yu, Jun Wang, Jingkai Geng, Jiayi Zhao, Zheng Zhao, Juan Xu, Hui Liu
          </td>
          <td>2025-06-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48b921317e55f0a7a7e7ce1f3c412ed182dc30b" target='_blank'>
              DNA2 and FANCM function in two distinctive pathways in disrupting TERRA R-loops and suppressing replication stress at ALT telomeres
              </a>
            </td>
          <td>
            Ashwin Ragupathi, Heba Z. Abid, Yun Chen, Rongwei Zhao, Settapong T. Kosiyatrakul, Derin I. Yetil, Julia Neiswander, Rachel Feltman, Shannon Thomas, Benjamin Yusupov, Manrose Singh, Li Zheng, Binghui Shen, Huaiying Zhang, H. C. Chu, Carl L. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulator often disrupted in cancer, yet how promoter methylation dynamics translate into transcriptional changes during cancer progression remains incompletely understood. Here, we employed targeted bisulfite sequencing and RNA-seq on paired tumor and non-tumor tissues from 80 Korean colorectal cancer (CRC) patients to map promoter methylation and gene expression dynamics. Promoters with high baseline methylation in non-tumor tissues tended to become hypomethylated in tumors, while those with low baseline methylation underwent partial hypermethylation. However, these changes did not consistently correlate with gene silencing or activation. Strikingly, promoters marked by Polycomb (PcG+) in non-tumor tissue were prone to hypermethylation yet often remained transcriptionally active in tumors, a paradox most prominent in transcription factor (TF) genes. In contrast, hypermethylation in PcG− promoters was more consistently associated with transcriptional repression. Our findings suggest that epigenetic plasticity at PcG+ TF gene promoters can override the typically repressive effects of DNA methylation, potentially enabling tumors to maintain or enhance the expression of key regulatory genes. This highlights the importance of PcG occupancy in shaping the functional consequences of methylation changes during colorectal tumorigenesis, warranting deeper investigation into how these epigenetic adaptations drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65ef4383977e8a75d030b12da77f7092d6182f23" target='_blank'>
              Transcription factors overcome the repressive impact of Polycomb-associated methylation in tumors
              </a>
            </td>
          <td>
            Min-Kyeong Kwon, Goeun Park, Dayoung Go, Donghyun Park, Sridhar Hannenhalli, Sun Shim Choi
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24cfd9851764715ead654d1c47914a25b505dff" target='_blank'>
              Die Vermessung des Genoms
              </a>
            </td>
          <td>
            Clemens Steinek, H. Leonhardt
          </td>
          <td>2025-06-01</td>
          <td>BIOspektrum</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The capacity of a plasmid to express genes is constrained by its length and copy number. However, the interplay between these parameters and their constraints on plasmid evolution have remained elusive due to the absence of comprehensive quantitative analyses. Here, we present ‘Pseudoalignment and Probabilistic Iterative Read Assignment’ (pseuPIRA), a computational method that overcomes previous computational bottlenecks, enabling rapid and accurate determination of plasmid copy numbers at large scale. We apply pseuPIRA to all microbial genomes in the NCBI RefSeq database with linked short-read sequencing data (4644 bacterial and archaeal genomes including 12,006 plasmids). The analysis reveals three scaling laws of plasmids: first, an inverse power-law correlation between plasmid copy number and plasmid length; second, a positive linear correlation between protein-coding genes and plasmid length; and third, a positive correlation between metabolic genes per plasmid and plasmid length, particularly for large plasmids. These scaling laws imply fundamental constraints on plasmid evolution and functional organization, indicating that as plasmids increase in length, they converge toward chromosomal characteristics in copy number and functional content.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d0a945dacbeb8801a3cb57a4c7f7eb8870d06a" target='_blank'>
              Scaling laws of bacterial and archaeal plasmids
              </a>
            </td>
          <td>
            Rohan Maddamsetti, Irida Shyti, Maggie L. Wilson, Hye-in Son, Yasa Baig, Zhengqing Zhou, Jia Lu, Lingchong You
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf2932e90bcc5681634229bcd158ed1d080d8b" target='_blank'>
              Single-cell Characterization of DNA Hydroxymethylation of the Mouse Brain During Aging
              </a>
            </td>
          <td>
            Yali Bai, Tianjiao Yuan, Liuhao Ren, Yuhang Huan, Fa Yang, Yutong Li, Aicui Zhang, Yueyang Liu, Tian Tian, Ningxia Kang, Danni Chen, Xuchao Xie, Qiang Wang, Wei Chen, Yinghui Zheng, Xiaoliang Sunney Xie, Yunlong Cao
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4f9af4dcbefbfc2af9850cbacc2d2628f2e0e1e" target='_blank'>
              TEITbase: a database for transposable element (TE)-initiated transcripts in human cancers
              </a>
            </td>
          <td>
            Yun Zhang, Jianqi She, Xueyan Hu, Yueqi Jin, Changyu Tao, Minghao Du, Ence Yang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Mutation, deletion, or silencing of genes encoding cellular metabolism factors occurs frequently in human malignancies. Neomorphic mutations in isocitrate dehydrogenases 1 and 2 (IDH1/2) promoting the production of R-2-hydroxyglutarate (R-2HG) instead of α-ketoglutarate (αKG) are recurrent in human brain cancers and constitute an early event in low-grade gliomagenesis. Due to its structural similarity with αKG, R-2HG acts as an inhibitor of αKG-dependent enzymes. These include the JUMONJI family of lysine demethylases, among which KDM4A is particularly sensitive to R-2HG-mediated inhibition. However, the precise molecular mechanism through which inhibition of αKG-dependent enzymes by R-2HG promotes gliomagenesis remains poorly understood. Here, we show that treatment with R-2HG induces cellular senescence in a p53-dependent manner. Furthermore, expression of mutated IDH1R132H or exposure to R-2HG, which leads to KDM4A inhibition, causes telomeric dysfunction. We demonstrate that KDM4A localizes to telomeric repeats and regulates abundance of H3K9(me3) at telomeres. We show that R-2HG caused reduced replication fork progression, and that depletion of SMARCAL1, a helicase involved in replication fork reversal, rescues telomeric defects caused by R-2HG or KDM4A depletion. These results establish a model whereby IDH1/2 mutations cause R-2HG-mediated inhibition of KDM4A, leading to telomeric DNA replication defects, telomere dysfunction, and associated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728014013df3b457cbd616bc46c4a758e940b3f8" target='_blank'>
              R-2-hydroxyglutarate-mediated inhibition of KDM4A compromises telomere integrity
              </a>
            </td>
          <td>
            Florence Couteau, Laurence M Gagné, Karine Boulay, Philippe Rousseau, Mélissa Carbonneau, Mary McQuaid, Jyoti Sharma, C. Sawchyn, Erlinda Fernandez, Dagmar Glatz, Rana Rizk, Marie-Eve Lalonde, Yosra Mehrjoo, Tsz Wai Chu, G. Moquin-Beaudry, Christian Beauséjour, Mikhail Sergeev, Santiago Costantino, D. Avizonis, I. Topisirovic, Nada Jabado, Hugo Wurtele, Chantal Autexier, Frédérick A. Mallette
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The human genome is widely transcribed, with part of these transcribed regions producing stably expressed protein-coding or non-coding RNAs. Long intergenic non-coding RNAs (lincRNAs) are significantly differentially expressed in various cell lines and tissues. However, the influence of their transcription events remains unclear. In this study, we constructed a human genomic interaction network and found frequent interactions between lincRNA genes and protein-coding genes that are highly related to the occupancy of RNA polymerase II on the lincRNA gene. Interestingly, in the human genome interaction networks, the degree of lincRNA genes was significantly higher than that of protein-coding genes. The promoter regions of the protein-coding genes interacting with the lincRNA genes are enriched with R-loop structures, indicating that lincRNA may influence the target genes through R-loop structures. These promoters were enriched in more transcription factor binding sites. Furthermore, the whole network and sub-network could be utilized to explore potential biomarkers of leukemia. We found that zinc finger protein 668 (ZNF668), eosinophil granule ontogeny transcript (EGOT), and glutamate metabotropic receptor 7 (GRM7) could serve as novel biomarkers for acute myeloid leukemia (LMAL). Pasireotide acetate (CAS No. 396091-76-2) represents a potential drug for LMAL patients. These results suggested that potential biomarkers and corresponding drugs for cancer could be identified based on lincRNA–promoter network/sub-network topological parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fd4f873b33226dc62ac979d88627e1f00373ee" target='_blank'>
              Genome-Wide Detection of Leukemia Biomarkers from lincRNA–Protein-Coding Gene Interaction Networks in the Three-Dimensional Chromatin Structure
              </a>
            </td>
          <td>
            Yue Hou, Wei Ning, Muren Huhe, Chuanjun Shu
          </td>
          <td>2025-05-22</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4968170702089395233b0c88bfdd298c3048f328" target='_blank'>
              Role of Chromosomal Microarray and RNA Fusion Analysis in Detecting KMT2A Partial Tandem Duplication with High Mortality in Myeloid Neoplasms: Retrospective Study
              </a>
            </td>
          <td>
            MD¹ Bhaumik Shah, MD¹ Jianming Pei, MD¹ Roniya Francis, MD² Peter Abdelmessieh, MD¹ Mariusz Wasik, MD¹ Nicholas Mackrides, MD¹ Reza Nejati
          </td>
          <td>2025-05-28</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Aim: The over-activation of oncogenes is a critical genetic event in the development and progression of solid malignancies. Gene amplification and specific mutations represent the prominent mechanisms that convert proto-oncogenes into their active, oncogenic forms. C-myc oncogene (gene locus: 8q24.21) regulates crucial cell and tissue functions, whereas its deregulation is implicated in carcinogenesis. Our research aimed to investigate the impact C-myc numerical imbalances on a series of laryngeal squamous cell carcinomas (LSCCs) characterized by different clinic-pathological features. Materials and Methods: We analyzed 55 LSCC archival tissue sections by implementing a dual-color gene/chromosome fluorescent in situ hybridization assay (C-myc/Chr 8 FISH). Results: According to the extracted C-myc gene/Chr 8 signal ratios, we identified 18 cases with numerical imbalances. The majority of them were gene amplifications (pure n=15, amplification and polysomy n=1), whereas the last two (n=2) cases presented multiple gene copies as a result of Chr 8 polysomy. C-myc amplification was correlated with advanced stage. Stage III/IV cases demonstrated the highest levels of gene amplification (p=0.024). Conclusion: C-myc amplification, combined or not with Chr 8 polysomy, is a relatively frequent event in LSCC. This numerical imbalance is associated with an aggressive phenotype, and it seems to be a significant key element among other oncogenes and suppressor genes that form specific genomic signatures in the corresponding patients with LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf169522cd061e72ca9d71b7495c4317a28f61a5" target='_blank'>
              C-myc Oncogene Numerical Imbalances Analysis in Laryngeal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Sotirios Papouliakos, V. Papanikolaou, A. Chrysovergis, E. Tsiambas, D. Roukas, Sofianiki Mastronikoli, G. Tsouvelas, G. Agrogiannis, Nikolaos Kavantzas, Vasileios Ragos, Andreas C. Lazaris, E. Kyrodimos
          </td>
          <td>2025-05-27</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="A characteristic of triple-negative breast cancer (TNBC) is the epigenetic regulation of tumor suppressor genes, leading to TNBC heterogeneity and treatment resistance in patients. TNBC exhibits high methylation rates, leading to the silencing of numerous tumor suppressor genes. DNA methyltransferase inhibitors (DNMTis) have shown limited clinical efficacy in TNBC treatment. This study aims to uncover a target that could be used to reverse the epigenetic silencing of tumor suppressor genes in TNBC. The Western blot analysis demonstrated that ROR1 knockdown, an oncofetal gene, reduced DNMT3A and DNMT3B protein expression in the TNBC cell lines MDA-MB-231 and HCC1806, as well as a non-malignant breast cell line, MCF10A. The reduced representation bisulfite sequencing (RRBS) analysis identified differential methylation of CREB3L1 when ROR1 is knocked down in TNBC cell lines. CREB3L1 is a transcription factor that plays tumor-suppressive roles in TNBC and is commonly epigenetically silenced in patients. This study shows that ROR1 requires pSTAT3 activation to upregulate DNMT3A and DNMT3B expression to induce CREB3L1 epigenetic silencing in TNBC. ROR1 knockdown resulted in the re-expression of CREB3L1 in TNBC cells. The data provide evidence that ROR1 inhibition, in combination with DNMTis, could enhance patient outcomes as a therapeutic approach for TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac855cfb53ff2a45c0c45fe36069b3895f3153ea" target='_blank'>
              Uncovering a Novel Role of ROR1 in the Epigenetic Regulation of Tumor Suppressor Gene CREB3L1 in Triple-Negative Breast Cancer Cells
              </a>
            </td>
          <td>
            Victoria L. Reed, Eric Lalu, Leena Yoon, Norman Fultang, B. Peethambaran
          </td>
          <td>2025-05-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Human development and health depend on the precise expression of relevant genes in specific cells, at precise times and in response to appropriate stimuli. This is known as context-dependent gene regulation and relies on the activities of a diverse 'zoo' of DNA elements within the genome that are collectively called context-dependent cis-regulatory elements (cdCREs). cdCREs may comprise as much as 10% of the genome and include better-known sequences such as enhancers, silencers and promoters that form the basis of complex multicellularity. Diverse vertebrate body plans not only share considerable phenotypic similarities but also cell types, the genes they express and, in a growing number of cases, in the function and nucleotide sequence of cdCREs. The current review will critically evaluate current methodologies to identify cdCREs and re-evaluate a place for comparative genomics amongst them. We will also explore the function of cdCREs and discuss methods of analysing their function in disease-associated physiologies and behaviours using in vivo models such as CRISPR-generated GA mouse lines. Finally, we will study the effects of epigenetic mechanisms such as DNA methylation on cdCRE activity and examine how genetics and epigenetics can interact to alter disease susceptibility. Given that genome-wide association studies (GWAS) suggest that 95% of disease-associated genomic variation reside in the 98% of the less understood noncoding genome, the need to understand the role of conserved vertebrate cdCREs in development and health in vivo has never been more pressing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382d555c7aff11257c9330d9a7647ebcfb9ae6bb" target='_blank'>
              Exploring the roles of conserved context-dependent cis-regulatory elements (cdCREs) in multicellularity, human health and disease.
              </a>
            </td>
          <td>
            Andrew McEwan, Alexander Rattray, Greg Hutchings, Elizabeth A. Hay, Chris Murgatroyd, A. MacKenzie
          </td>
          <td>2025-07-04</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39136831a82d4694ee219ef38ce5763db7fa3a19" target='_blank'>
              ECCFP: a consecutive full pass based bioinformatic analysis for eccDNA identification using Nanopore sequencing data
              </a>
            </td>
          <td>
            Tangxuan Zhang, Wang Li, Qingsong Zen, Jun Zhang, Biyuan Miao, Mengting Li, Juan Luo, Tianliang Liu, Shifu Chen, Shaogui Wan
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1a766c9476f757f76140a52e67514cf195c7f" target='_blank'>
              Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination.
              </a>
            </td>
          <td>
            K. Zhao, J. Vos, S. Lam, Lillian A Boe, Daniel Muldoon, Catherine Y Han, Cristina Valero, Mark Lee, C. Fitzgerald, A. Lee, Manu Prasad, Swati Jain, Xinzhu Deng, Timothy A Chan, Michael F. Berger, C. Bandlamudi, Xi Kathy Zhou, Luc G. T. Morris
          </td>
          <td>2025-06-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0065672d86f3f94440ecf420ec518fca8aa5a1ec" target='_blank'>
              Integrated Spatial Analysis of Ovarian Precancerous Lesions
              </a>
            </td>
          <td>
            Tu-Yung Chang, Yen-Wei Chien, Szu-Hua Chen, Annelise Sokolow, Yeh Wang, Brant G. Wang, Tricia A Numan, M. Herman Chui, Rebecca Stone, T. Pisanic, Nicholas Papadopoulos, Tian-Li Wang, Christopher Douville, Leslie Cope, I.-M. Shih
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41db77b7599b92a63937c7f3610506611aa97e8f" target='_blank'>
              A survey of human cancer-germline genes : linking X chromosome localization, DNA methylation and sex-biased expression in early embryos
              </a>
            </td>
          <td>
            A. Loriot, Julie Devis, Laurent Gatto, C. De Smet
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Every generation, the human genome is shuffled during meiosis and a single fertilized egg gives rise to all of the cells of the body1. Meiotic errors leading to chromosomal abnormalities are known causes of pregnancy loss2,3, but genetic aetiologies of euploid pregnancy loss remain largely unexplained4. Here we characterize sequence diversity in early pregnancy loss through whole-genome sequencing of 1,007 fetal samples and 934 parental samples from 467 trios affected by pregnancy loss (fetus, mother and father). Sequenced parental genomes enabled us to determine both the parental and meiotic origins of chromosomal abnormalities, detected in half of our set. It further enabled us to assess de novo mutations on both homologous chromosomes from parents transmitting extra chromosomes, and date them, revealing that 6.6% of maternal mutations occurred before sister chromatid formation in fetal oocytes. We find a similar number of de novo mutations in the trios affected by pregnancy loss as in 9,651 adult trios, but three times the number of pathogenic small (<50 bp) sequence variant genotypes in the loss cases compared with adults. Overall, our findings indicate that around 1 in 136 pregnancies is lost due to a pathogenic small sequence variant genotype in the fetus. Our results highlight the vast sequence diversity that is lost in early pregnancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa3651219511c4ef3423241fcef77876aaae5715" target='_blank'>
              Sequence diversity lost in early pregnancy
              </a>
            </td>
          <td>
            G. Arnadottir, Hákon Jónsson, T. Hartwig, J. Gruhn, P. L. Møller, Arnaldur Gylfason, D. Westergaard, Andrew Chi-ho Chan, A. Oddsson, L. Stefánsdóttir, L. Roux, V. Steinthorsdottir, Kristjan H Swerford Moore, S. Olafsson, Pall I. Olason, H. Eggertsson, G. Halldorsson, G. Walters, H. Stefánsson, S. A. Gudjonsson, G. Pálsson, B. Jensson, Run Fridriksdottir, Jesper Friis Petersen, , H. Nielsen, S. Bliddal, K. Banasik, M. Krog, A. Kolte, N. la Cour Freiesleben, S. Ostrowski, Erik Sørensen, Mille Løhr, Markus J. Herregård, E. Hoffmann, J. Gruhn, Andrew Chi-ho Chan, U. Þorsteinsdóttir, Kári Stefánsson, Hákon Jónsson, O. Magnusson, L. Ambye, Lone Schmidt, P. Kamstrup, M. Nyegaard, Ellen Christine Leth Løkkegaard, H. E. Bredkjær, H. W. Hvidman, Agnar Helgason, G. Norddahl, P. D. Rohde, J. Saemundsdottir, O. Magnusson, B. Halldórsson, D. Gudbjartsson, P. Sulem, U. Thorsteinsdóttir, E. Hoffmann, H. Nielsen, Kári Stefánsson
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>2</td>
          <td>184</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="For chromosome abnormalities (CA), such as Down syndrome (DS), the influence of genomic variations to chromosome conformation and gene transcription remains elusive. Based on the complete genomic sequence from the parents of the DS trisomy patient, we systematically delineated an atlas of parental-specific haploid single nucleotide polymorphism (SNP), copy number variation (CNV), three-dimensional (3D) genome, and RNA expression profiles of the diencephalon in the DS patient. The integrated haploid multi-omics analysis demonstrated that one-dimensional genomic variations including SNPs and CNVs in the DS patient are highly correlated with the alterations of the 3D genome and the subsequent gene transcription. The correlation relationship remains valid in haploid-levels. Moreover, we revealed the 3D genome alteration associated mis-regulation of DS-related genes, which facilitates to understanding the pathogenesis of CA. Together, our study contributes to decipher the coding from one-dimensional genomic variations to 3D genomic architecture and the subsequent gene transcription in healthy and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9de44c4aa76967640e3414bc7acd3b2037c35" target='_blank'>
              Deciphering Haploid Chromosome Conformation Alternation in Down Syndrome by Multiple Haploid Omics Analysis.
              </a>
            </td>
          <td>
            Chengchao Wu, Tianshu Zhou, Wenfu Ke, Wei Xiong, Zhihui Zhang, Siheng Zhang, Jinyue Wang, Lulu Deng, Keji Yan, Man Wang, Shenglong He, Qi Gong, Chao Ma, Xiaping Chen, Yan Li, He Long, Chong Guo, Gang Cao, Zhijun Zhang
          </td>
          <td>2025-06-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, , Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The mutations found in a cancer genome are shaped by diverse processes, each displaying a characteristic mutational signature that may be influenced by the genome’s architecture. While prior analyses have evaluated the effect of topographical genomic features on mutational signatures, there has been no computational tool that can comprehensively examine this interplay. Here, we present SigProfilerTopography, a Python package that allows evaluating the effect of chromatin organization, histone modifications, transcription factor binding, DNA replication, and DNA transcription on the activities of different mutational processes. SigProfilerTopography elucidates the unique topographical characteristics of mutational signatures, unveiling their underlying biological and molecular mechanisms. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03612-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f430e923a52d6b701a30fa1d7bf099c2ea45b513" target='_blank'>
              Evaluating topography of mutational signatures with SigProfilerTopography
              </a>
            </td>
          <td>
            Burçak Otlu, Ludmil B. Alexandrov
          </td>
          <td>2025-05-20</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef02803fbaf5cb1abbd79351dc6a4ca1c68ba79" target='_blank'>
              Constitutional copy number amplifications: rare or under-evaluated? Revisiting a 25-year-old cold case.
              </a>
            </td>
          <td>
            Eliana Salvo, Romano Tenconi, R. Giorda, Sara Bertuzzo, Luca Cesana, Roberta Murru, S. Giglio, Mana M Mehrjouy, Niels Tommerup, Orsetta Zuffardi, M. Bonaglia
          </td>
          <td>2025-06-04</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The human genome encodes 19 adenosine and cytidine deaminase genes, classified as A-to-I versus C-to-U editors. A-to-I editors have been widely identified as a promising therapeutic target in various cancers. Conversely, the investigation into C-to-U editors is relatively limited. This study evaluated RNA-editing genes in prostate cancer (PCa). Notably, the APOBEC3 genes are clustered in terms of their chromosomal locations, and their transcriptional changes exhibit significant positive correlations in both primary PCa and castration-resistant prostate cancer (CRPC). One member of this family, APOBEC3C, is demonstrated here as an androgen receptor (AR)-repressed gene. Consistently, APOBEC3 loci are epigenetically inhibited in PCa progression, with APOBEC3C level lower in PSA-high patients. APOBEC3C-low PCa cohorts exhibit increased resistance to Abiraterone and Enzalutamide. Clinicopathological profiling further confirmed APOBEC3C downregulation along PCa progression to advanced phases (grade IV/V, stage III-IV, and pathological stage T3-4), underscoring its prognostic value. Additionally, APOBEC3C expression inversely correlates with PCa relapse and mortality, and low APOBEC3C levels are linked to unfavorable survival. Notably, integrated analyses identified APOBEC3C as the sole RNA-editing gene with significance in both differential expression and PCa prognosis, and APOBEC3C had the best diagnostic performance among 19 genes. Our efforts provide a foundation for further RNA editors research in PCa diagnosis and therapy, and grant APOBEC3C as a candidate tumor suppressor. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00169-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0fae2048158500534064188ceb1148d343ad87" target='_blank'>
              Characterization of RNA editing gene APOBEC3C as a candidate tumor suppressor in prostate cancer
              </a>
            </td>
          <td>
            Li-Yang Wang, Ji Shi, Mo-Fei Wang, Yi-Meng Liu, Hong-Shan Guo, Jin-Cheng Wang, Shu Jiang, Jia-Qian Liang, Xinghua Liao, Shao-Yong Chen
          </td>
          <td>2025-05-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Endogenized giant viruses are emerging as major contributors to the genome evolution of microbial eukaryotes, with both degraded and fully functional latent viruses being found integrated in diverse lineages. The mechanisms that determine the fate of viral integrants are poorly understood, however. Acanthamoeba is a unicellular eukaryote known for undergoing lateral gene transfer (LGT) with viruses. Here we have leveraged chromosome-scale assemblies of two strains of Acanthamoeba, Neff and C3, to investigate the genomic mechanisms that mediate the fate of viral integrations in eukaryotic genomes. Results Viral integrations in the C3 and Neff genomes are largely non-overlapping and disproportionately found in sub-telomeric regions. Multiple partial copies of these insertions are found throughout the Neff genome, but they are not expressed, do not obviously encode functions associated with their own mobility, and are colonized by host mobile elements. Viral regions are hypermethylated and highly condensed, suggesting that the expression of recently acquired viral DNA is suppressed in heterochromatic regions. Conclusions We propose a model for the trajectory of viral sequences in Acanthamoeba: (i) integration of DNA from giant viruses, (ii) epigenetic suppression of the viral DNAs, allowing them to persist in the genome, and (iii) deterioration of viral genomes by point mutation, mobile element colonization, and intra- and inter-chromosomal recombination. Viral integrations in Acanthamoeba spp. are transient and may not have long-lasting effects on the fitness of the amoeba. Our work highlights the importance of host genome dynamics and epigenetic silencing for understanding the evolution of endogenized viral elements. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02280-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f54d3754da37f780fd27a13a0f10078e856d51" target='_blank'>
              Epigenetic silencing and genome dynamics determine the fate of giant virus endogenizations in Acanthamoeba
              </a>
            </td>
          <td>
            Cédric Blais, Morgan J. Colp, Luke A. Sarre, Alex de Mendoza, John M. Archibald
          </td>
          <td>2025-07-01</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Fluoroquinolones induce double-strand breaks in bacterial DNA, triggering the SOS response, a major DNA damage response that ensures the expression of repair proteins but also promotes the emergence and spread of antibiotic resistance. Fluoroquinolone resistance, particularly in Escherichia coli, is a growing global health concern. Understanding bacterial responses to these antibiotics is critical for developing preventive strategies and novel treatments to combat resistance development. This study investigates DNA morphology in E. coli following exposure to ciprofloxacin (CIP), a fluoroquinolone antibiotic. We show that CIP induces a stepwise DNA reorganization, culminating in a highly dense nucleoid structure at midcell-a process we term DNA supercompaction. This phenomenon occurred also with other genotoxic agents. Live-cell imaging revealed that RecN, a structural maintenance of chromosomes (SMC)-like protein, is required for DNA supercompaction, and that RecN's dynamics and activity in this response depend on RecA. Additionally, RecN and RecA frequently colocalized at nucleoid-associated positions. We suggest that RecN and RecA play active roles in DNA supercompaction following severe DNA damage, that their interplay is part of a prompt universal survival response to DNA double-strand breaks in E. coli, and that the extent of the compaction response depends on the DNA damage severity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506e392d68ea028b658f17b6f901be5aeef9ee4e" target='_blank'>
              RecN and RecA orchestrate an ordered DNA supercompaction response following ciprofloxacin-induced DNA damage in Escherichia coli.
              </a>
            </td>
          <td>
            Krister Vikedal, S. B. Ræder, Ida Mathilde Marstein Riisnæs, Magnar Bjørås, J. Booth, Kirsten Skarstad, Emily Helgesen
          </td>
          <td>2025-05-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, A. Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 10615


 Background:
 ATM
 is a tumor suppressor gene involved in DNA repair and telomere maintenance.
 ATM
 biallelic pathogenic or likely pathogenic germline variants (gPV) are associated with

 ataxia-telangiectasia syndrome. Monoallelic ATM gPV are associated with increased cancer risk; however, their contribution to carcinogenesis has not been elucidated. We sought to characterize the genomic landscape of gPV
 ATM
 -associated cancers in a pan-cancer cohort.
 Methods:
 ATM
 alterations (germline and somatic) were identified in patients (Pts) with a solid tumor diagnosis sequenced with MSK-IMPACT, an FDA-approved, tumor-normal paired targeted NGS. Genetic-imputed ancestry clinicopathologic characteristics and FACETS estimated allele-specific copy number profiles were evaluated. Samples with purity <0.2 or with undetermined copy number profiles were excluded. HRD score was evaluated using FACETS profiles. Both somatic and germline sequencing data were analyzed within protocol NCT01775072.
 Results:
 Among 40,136 Pts with cancer who underwent germline testing, 1.1% (n=442) harbored an
 ATM
 gPV, inclusive of 2 Pts with biallelic
 ATM
 gPVs. The most frequent mutations were: R2547_S2549del (n=17), K2756* (n=14) and V1268* (n=11). Among these, R2547_S2549del and K2756* were identified exclusively in Pts with European ancestry, while E343Ifs*2 (n= 9/10) and c.1065+1G>T (n= 6/7) were most prevalent in Pts with Ashkenazi Jewish (AJ) ancestry. Concomitant gPVs in other genes were observed in 10% (n=47), with the most frequent ones being
 APC
 (I1307K),
 MUTYH
 ,
 BRCA2, CHEK2
 , and
 BRCA1
 . 63,270 out of 86,039 tumor samples had usable FACETS profiles.
 ATM
 somatic allele-specific information was available for 67% (297/442) of Pts. 62% (197/317) of tumor samples from Pts with monoallelic
 ATM
 gPVs exhibited somatic biallelic (Bi)
 ATM
 inactivation, while 38% (120/317) retained monoallelic (Mono) ATM status. The underlying mechanism of Bi-ATM inactivation was loss of heterozygosity in 75% (149/197) and additional somatic
 ATM
 mutations in 25% (48/197). Samples from Pts with tumors known to be associated with ATM gPVs (Breast, Pancreatic and Prostate cancers) had significant enrichment in Bi-ATM inactivation compared to tumors without strong association with ATM gPVs (88% Vs 58%; p<0.01). Although samples with Bi-
 ATM
 had higher overall HRD-scores compared to Mono-
 ATM
 (median = 34 Vs 20; p<0.01), no significant enrichment in HRD-High phenotype was seen in Bi-
 ATM
 (23 Vs 18; p=0.49).
 Conclusions:
 Evaluation of a pan-cancer Pt population with
 ATM
 gPVs demonstrated a high prevalence of biallelic somatic inactivation. Most Pts with
 ATM
 gPVs who had malignancies implicated in
 ATM
 -associated cancer risk, had biallelic somatic inactivation in their tumors suggestive of their contribution to tumorigenesis. While biallelic ATM inactivation is associated with higher genomic instability, no enrichment in HRD phenotype was observed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171b6148e94d64a55d3944e70772bbb94d153ffa" target='_blank'>
              Clinicopathologic and allele-specific analysis of germline
 ATM
 alterations in a pan-cancer cohort.
              </a>
            </td>
          <td>
            Matteo Repetto, Daniel Muldoon, P. Selenica, Kanika Arora, Ying L. Liu, M. Carlo, Lauren Banaszak, Alicia J Latham, C. Bandlamudi, E. O'Reilly, M. E. Robson, B. Rousseau, D. Mandelker, A. Drilon, Britta Weigelt, K. Offit, Michael F. Berger, Y. Murciano-Goroff, Z. Stadler, Mohammad Abbass
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract MYC proteins are potent oncoproteins that drive tumorigenesis in a wide range of cancers, making it critical to understand their oncogenic functions and underlying mechanisms. Although MYC overexpression induces transcriptional and replication‐associated stress, recent studies have paradoxically identified MYC as a key resilience factor that protects cancer cells from these stressors. In this review, we explore the dual role of MYC in both driving and mitigating cellular stress to achieve its oncogenic function. We also examine how MYC‐induced transcriptional and replicative stress generates potentially immunogenic nucleic acid species while simultaneously helping cancer cells evade host immune recognition. We propose a model in which MYC plays a critical role in managing the stress it induces, thereby maintaining a balance that promotes tumor growth. Based on this model, we discuss potential therapeutic strategies targeting MYC‐dependent stress responses, offering new avenues for cancer treatment and highlighting the complexity of MYC‐driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ce286c658baf7f61e9c844f34e8e87797cdc79" target='_blank'>
              MYC: The Guardian of Its Own Chaos
              </a>
            </td>
          <td>
            A. Gaballa, Bastian Krenz, Leonie Uhl
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Nucleoporin 98 (NUP98) fusion oncoproteins are strong drivers of pediatric acute myeloid leukemia (AML) with poor prognosis. Here we show that NUP98 fusion-expressing AML harbors an epigenetic signature that is characterized by increased accessibility of hematopoietic stem cell genes and enrichment of activating histone marks. We employ an AML model for ligand-induced degradation of the NUP98::KDM5A fusion oncoprotein to identify epigenetic programs and transcriptional targets that are directly regulated by NUP98::KDM5A through CUT&Tag and nascent RNA-seq. Orthogonal genome-wide CRISPR/Cas9 screening identifies 12 direct NUP98::KDM5A target genes, which are essential for AML cell growth. Among these, we validate cyclin-dependent kinase 12 (CDK12) as a druggable vulnerability in NUP98::KDM5A-expressing AML. In line with its role in the transcription of DNA damage repair genes, small-molecule-mediated CDK12 inactivation causes increased DNA damage, leading to AML cell death. Altogether, we show that NUP98::KDM5A directly regulates a core set of essential target genes and reveal CDK12 as an actionable vulnerability in AML with oncogenic NUP98 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac2ff38eb74ff54c2c90e5931c96b450175dda5" target='_blank'>
              Transcriptional and epigenetic rewiring by the NUP98::KDM5A fusion oncoprotein directly activates CDK12
              </a>
            </td>
          <td>
            Selina Troester, Thomas Eder, Nadja Wukowits, M. Piontek, P. Fernández-Pernas, Johannes Schmoellerl, B. Haladik, G. Manhart, M. Allram, Margarita Maurer-Granofszky, Nastassja Scheidegger, K. Nebral, Giulio Superti-Furga, Roland Meisel, B. Bornhauser, P. Valent, Michael N Dworzak, Johannes Zuber, Kaan Boztug, Florian Grebien
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, M. Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37e8efeea3f80c12980365d094cd561d8a7d326" target='_blank'>
              LINE-1 ORF1p is a shared and immunogenic antigen in cancer
              </a>
            </td>
          <td>
            Wilson McKerrow, Megan Snyder, Chong Chu, Deena Kelly, Heike Keilhack, Liyang Diao
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Aging and cancer, though distinct biological processes, share overlapping molecular pathways, particularly in epigenetic regulation. Among these, DNA methylation is central to mediating gene expression, maintaining cellular identity, and regulating genome stability. This review explores how age-associated changes in DNA methylation, characterized by both global hypomethylation and focal hypermethylation, contribute to the emergence of cancer. We discuss mechanisms of DNA methylation drift, the development of epigenetic clocks, and the role of entropy and epigenetic mosaicism, in aging and tumorigenesis. Emphasis is placed on how stochastic methylation errors accumulate in aging cells and lead to epiallelic shifts and gene silencing, predisposing tissues to malignant transformation, even despite recently increased cancer incidences at younger ages. We also highlight the translational potential of DNA methylation-based biomarkers, and therapeutic targets, in age-related diseases. By framing cancer as a disease of accelerated epigenetic aging, this review offers a unifying perspective and calls for age-aware approaches to both basic research and clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80ee6ccb4c2cbc13d84b76cf48a4e779ceb4234" target='_blank'>
              DNA Methylation, Aging, and Cancer
              </a>
            </td>
          <td>
            Himani Vaidya, Jaroslav Jelinek, Jean-Pierre J. Issa
          </td>
          <td>2025-06-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bb823c01319c257e9b78eae2e9bd2984b27b68" target='_blank'>
              CRISPR activation screens map the genomic landscape of cancer glycome remodeling
              </a>
            </td>
          <td>
            John Daly, Lidia Piatnitca, Mohammed Al-Seragi, Vignesh K Krishnamoorthy, S. Wisnovsky
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="BAP31, a transmembrane protein in the endoplasmic reticulum, is known for its oncogenic properties, but its role in immunotherapy is not well understood. While BAP31’s involvement in liver, gastric, and cervical cancers has been documented, its role in pan-cancer immune regulation, particularly in breast cancer, remains unexplored. Using TCGA data, analysis via the Xiantao academic and GEPIA2 database showed that BAP31 upregulation correlates with advanced clinical stages and a poor prognosis. ROC analysis demonstrated BAP31’s high accuracy in distinguishing cancerous tissue from normal tissues. Additionally, BAP31 expression is associated with CNV, methylation, TMB, and MSI. In breast cancer, TIMER database analysis revealed that BAP31 expression is inversely correlated with the infiltration levels of myeloid-derived suppressor cells (MDSCs), macrophages, T lymphocytes, B lymphocytes, and neutrophils. Additionally, we investigated the relationship between BAP31 and the expression of major histocompatibility complex (MHC) molecules and chemokine receptors utilizing the TISIDB database. LinkedOmics analysis demonstrated associations between BAP31 and various immune-inflammatory pathways, while also indicating a negative correlation between BAP31 expression and four critical pathways: extracellular matrix receptor interaction, focal adhesion, JAK-STAT signaling, and TGF-β signaling. Furthermore, loss-of-function experiments employing shRNA-mediated knockdown of BAP31 resulted in a marked reduction in cell proliferation and an increase in apoptosis in breast cancer cells, thereby confirming its role in tumor promotion. These findings suggest that BAP31 may serve as a promising prognostic biomarker and a potential target for immunotherapy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/451aa3997b56f0c6b2a58e6dd234249f11454823" target='_blank'>
              Upregulated BAP31 Links to Poor Prognosis and Tumor Immune Microenvironment in Breast Cancer
              </a>
            </td>
          <td>
            Z. Hao, Bo Zhao, Xiaoshuang Zhu, Wanting Zhang, Bing Wang
          </td>
          <td>2025-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Objective This study aimed to investigate the role of poly(A) binding protein nuclear 1 (PABPN1) as a potential pan-cancer biomarker for prognosis and immunotherapy. Methods The original datasets were acquired from TCGA and GEO databases. PABPN1 expression analysis was conducted utilizing the Oncomine, TIMER, GEPIA, and BioGPS databases. Prognostic implications of PABPN1 were assessed through GEPIA, Kaplan-Meier plotter, and the PrognoScan database. Correlations between PABPN1 expression and immune checkpoints (ICP), tumor mutational burden (TMB), microsatellite instability (MSI), and neoantigens in human cancers were examined using the SangerBox database. Additionally, the association between PABPN1 and marker genes of tumor-infiltrated immune cells in urogenital cancers was confirmed. Differential expression of PABPN1 in urogenital cancers with distinct clinical characteristics was assessed using the UALCAN database. Finally, experiments of T24, 5637, HLF and MCF-7 cells were performed to verify the above results. Results The expression of PABPN1 tended to be higher in human cancers compared to paired normal tissues. Its expression levels showed strong associations with TMB, MSI, and neoantigens. Additionally, significant correlations existed between PABPN1 expression and tumor immune-infiltrated cells (TILs) in many human cancers, with marker genes of TILs showing significant relationships with PABPN1 expression, particularly in urogenital cancers. The coexpression networks of PABPN1 were predominantly involved in the regulation of immune response, antigen processing, and presentation. After down expression of PABPN1, mRNA expression levels of MRPS15 and GPx (Glutathione peroxidase) decreased significantly in T24, 5637 HLF and MCF-7 cells. Conclusion PABPN1 was expected to be an important role element in cancer research, serving as a potential prognostic and immunological pan-cancer biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec05ce952a72ca8439b0f8535adcae1a481e737a" target='_blank'>
              PABPN1 as a pan-cancer biomarker: prognostic significance and association with tumor immune microenvironment
              </a>
            </td>
          <td>
            Hong-xing Li, Xiao-ling Ma, Wang-bin Ma, Tian-yu Jia, Xiao-Hong Sun, Xiao-Xia He, Li-li Zhang, Ya-Ming Xi
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ffec3efbb372d77f7720ecc19ce6460ab0ebe6" target='_blank'>
              Caspase-8 is a novel modulator of Homologous Recombination Repair in response to ionizing radiations
              </a>
            </td>
          <td>
            Alessandra Ferri, Claudia Contadini, Claudia Di Girolamo, C. Cirotti, G. Fiscon, Paola Paci, M. Marzullo, M. Gentileschi, Tatsuro Yamamoto, Robert Strauss, D. Del Bufalo, L. Ciapponi, D. Barilà
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Historically, DNA sequence mutability has been considered relatively uniform and low in tumors with chromosomal instability (CIN), based on the assumption that high mutability would be detrimental in karyotypically aberrant contexts. Recent in silico analyses have challenged this view, suggesting some heterogeneity in mutation rates across CIN tumors; however, these predictions lack experimental validation. It also remains unclear how the intertumor variability of mutation rates compares to intratumor diversification and evolves along disease progression, whether mutation rates are functionally relevant in CIN cancers, and which mutational processes shape mutational accrual during CIN tumor onset and evolution. To address these gaps, we performed mutation accumulation experiments using clonal populations of patient-derived tumoroids from seven CIN, microsatellite-stable colorectal cancers (CRCs), and one microsatellite-unstable CRC. Each tumor exhibited a distinctive mutation rate footprint that was conserved among different clones from the same ancestor. In contrast, mutation rates diverged markedly across different tumors, with variations in magnitude within microsatellite-stable tumors as prominent as those distinguishing them from microsatellite-unstable tumors. New mutations reflected mutational processes associated with defective DNA replication and repair, which were not detected in normal tissues. Last, both mutation accumulation assays and high-depth whole-exome sequencing of subclonal variants showed higher mutation rates in metastatic lesions compared with matched primary tumors, suggesting positive selection for cells with increasing mutability during cancer dissemination. By providing an empirical assessment of mutation rates in human cancer, our data delineate heterogeneity, heritability, and progression-associated evolvability of DNA mutational instability as hallmarks of microsatellite-stable CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c57088e09b71e7ad2e34a58e69bb450d1b43d18" target='_blank'>
              Heterogeneity and evolution of DNA mutation rates in microsatellite stable colorectal cancer.
              </a>
            </td>
          <td>
            Elena Grassi, V. Vurchio, George D. Cresswell, Irene Catalano, B. Lupo, F. Sassi, Francesco Galimi, Sofia Borgato, Martina Ferri, Marco Viviani, Simone Pompei, Gianvito Urgese, Bingjie Chen, E. Zanella, F. Cottino, Mariangela Russo, G. Mauri, F. Pietrantonio, M. G. Zampino, L. Lazzari, S. Marsoni, A. Bardelli, M. Lagomarsino, A. Sottoriva, L. Trusolino, Andrea Bertotti
          </td>
          <td>2025-05-21</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Ultraviolet (UV) light causes cyclobutane pyrimidine dimers (CPDs) and other DNA lesions that must be efficiently repaired to prevent cell death and mutagenesis. While mammals utilize the nucleotide excision repair (NER) pathway to repair CPDs, many species primarily utilize photolyase enzymes to repair UV damage. Our understanding of how different genomic and chromatin features impact photolyase repair across a eukaryotic genome is limited. Here, we map repair of CPDs by photolyase across the yeast genome at single-nucleotide resolution. Our data indicate that yeast photolyase repairs CPDs more rapidly than NER, but photolyase activity is inhibited at certain classes of transcription factor binding sites and in nucleosomes. Repair in nucleosomes is particularly inhibited when CPDs are located along the 3′ side of the nucleosomal DNA or at minor-in rotational settings. Our data indicate that photolyase efficiently repairs the non-transcribed strand of yeast genes, but repair of the transcribed strand (TS) is inhibited. Genome-wide analysis of UV-induced mutations in NER-deficient photoreactivated yeast reveals a striking enrichment of mutations along the TS of yeast genes. These data indicate that inhibition of photolyase repair along the TS, likely due to occlusion of CPDs by RNA polymerase II stalling, promotes UV mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7e81bd67cc35d46ad0fa3937b62cb144d357b6" target='_blank'>
              Genome-wide maps of UV damage repair and mutation suppression by CPD photolyase
              </a>
            </td>
          <td>
            Kaitlynne A. Bohm, Marian F. Laughery, P. A. Mieczkowski, Steven A. Roberts, John J Wyrick Data Curation
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary A comprehensive study analyzed circulating cell-free DNA (cfDNA) from 30 oral squamous cell carcinoma (OSCC) patients, 199 public OSCC, and 192 normal samples to explore its potential in early-stage diagnostics and prognosis. The findings suggest that the cfDNA concentration correlates with the tumor stage, malignancy, and survival prognosis in OSCC patients. Deep genome sequencing of cfDNA revealed several genomic alterations: 1. Copy number variation analysis identified amplifications and deletions at loci 1q, 2q, 3p, 3q, and 8q22. Notably, amplifications of TP53 and PIK3CA, along with other genes related to keratinization, were identified in relapsed OSCC cases. Single nucleotide variations were observed in genes highly mutated in OSCC. 2. A novel fusion gene, TRMO-TRNT1, was detected in seven high-grade tumor samples. The parental genes, TRMO and TRNT1, are involved in tRNA modification and DNA repair, respectively. 3. Integrations of human papillomavirus, simian virus, and enterovirus were found in the OSCC samples, potentially indicating viral involvement in OSCC pathogenesis. These results underscore the utility of cfDNA as a non-invasive biomarker for OSCC, offering insights into tumor genomics and aiding in early detection and prognosis assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c52e96cadb3f868e649f16559709defe911b5c" target='_blank'>
              Applications for Circulating Cell-Free DNA in Oral Squamous Cell Carcinoma: A Non-Invasive Approach for Detecting Structural Variants, Fusions, and Oncoviruses
              </a>
            </td>
          <td>
            M. Bhattacharya, D. Yaniv, D. P. D’Souza, Eyal Yosefof, Sharon Tzelnick, R. Detroja, Tal Wax, A. Levy-Barda, G. Baum, A. Mizrachi, G. Bachar, Milana Frenkel Morgenstern
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The spatial resolution of omics dynamics is fundamental to understanding tissue biology. Spatial profiling of DNA methylation, which is a canonical epigenetic mark extensively implicated in transcriptional regulation, remains an unmet demand. Here, we introduce a method for whole genome spatial co-profiling of DNA methylation and transcriptome of the same tissue section at near single-cell resolution. Applying this technology to mouse embryogenesis and postnatal brain resulted in rich DNA-RNA bimodal tissue maps. These maps revealed the spatial context of known methylation biology and its interplay with gene expression. The two modalities’ concordance and distinction in spatial patterns highlighted a synergistic molecular definition of cell identity in spatial programming of mammalian development and brain function. By integrating spatial maps of mouse embryos at two different developmental stages, we reconstructed the dynamics of both epigenome and transcriptome underlying mammalian embryogenesis, revealing details in sequence, cell type, and region-specific methylation-mediated transcriptional regulation. This method extends the scope of spatial omics to DNA cytosine methylation for a more comprehensive understanding of tissue biology over development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e580ba04766eb95572371b96391de4351133faff" target='_blank'>
              Spatial joint profiling of DNA methylome and transcriptome in mammalian tissues
              </a>
            </td>
          <td>
            Chin Nien Lee, Hongxiang Fu, Angelysia Cardilla, Wanding Zhou, Yanxiang Deng
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889307242b28376f292bef10300827443677a91d" target='_blank'>
              EGR1 drives sex-differences in glioblastoma tumorigenicity
              </a>
            </td>
          <td>
            Tamara J. Abou-Antoun, Lorida Llaci, Jason P. Wong, Lihua Yang, N. Warrington, Lloyd Tripp, Jingqin Luo, Brian D. Muegge, Joshua B. Rubin, R. Mitra
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The epigenetic deregulation of CpG islands (CGIs) plays a crucial role in cancer initiation and progression. CGIs comprise 1-2% of the human genome and are rich in differentially methylated regions (DMRs) that can serve as cancer biomarkers in clinical samples and liquid biopsies. Focusing epigenetic sequencing on CpG-rich sequences, including CGIs and avoiding non-informative regions, offers an efficient and sensitive approach for cancer identification and tracking, especially within samples containing excess of unaltered, normal DNA. To this end, we have developed Adaptor-anchored Methylation amplification via Proximity Primers (aMAPP), a versatile PCR-based enrichment method. aMAPP employs specially designed primers to selectively enrich either methylated or unmethylated CpGs, depending on the upstream methylation conversion method employed. aMAPP achieves high coverage of genome-wide CGIs and detects hundreds of DMRs in tumor samples compared to adjacent normal tissue using ultra-low depth sequencing (∼300,000 reads). It enables tracing of aberrant methylation down to allelic frequency 0.01% in dilutions of tumor DNA and in cell-free DNA samples, can be applied using picogram amounts of DNA, and can be adapted to enrich either small panels of cancer-specific DMRs, or the majority (>90%) of genomic CGIs and CpGs. aMAPP offers a simple, cost-effective, and highly sensitive approach for capturing the epigenetic footprint of genome-wide CpGs and identifying aberrantly methylated or un-methylated genomic regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61956d7589478355803ed0351b120e6a0e356110" target='_blank'>
              SLFN11 Enhances Cisplatin Sensitivity in Pediatric Cancer via Activation of Stress-Response and Suppression of Survival Pathways
              </a>
            </td>
          <td>
            A. Jabeen, Dina Awartani, S. Sherif, Eiman I. Ahmed, R. Alanany, Ayman Saleh, W. Hendrickx, C. Raynaud
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Retroviral and retrotransposon invasion pose a constant threat to genome integrity and have driven the evolution of host defense pathways able to counter these attacks. The human silencing hub (HUSH complex) is an epigenetic transcriptional repressor complex that recognizes and silences newly integrated retroelements through the establishment of ectopic heterochromatin and chromatin compaction. HUSH provides a genome-wide immunosurveillance system whose challenging task is to detect and silence any newly integrated retroelements, and it thus plays a key role in host defense. To distinguish self from nonself genomic DNA, HUSH recognizes long single-exon (intronless) DNA, the essential hallmark of reverse transcription. Retroelements, being RNA derived, lack classical, noncoding cellular introns, so a long, intronless sequence of DNA is the abnormal molecular pattern that allows HUSH to distinguish invading retroelements from intron-containing host genes. As a newly identified component of the innate immune system, HUSH protects the genome from the reverse flow of genetic information from RNA to DNA, revealing an unanticipated mechanism of postintegration genome immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b1d5426e615e5a445a45b06d1bcad8bd22f0c5" target='_blank'>
              Silencing by the HUSH Epigenetic Transcriptional Repressor Complex.
              </a>
            </td>
          <td>
            Paul J. Lehner
          </td>
          <td>2025-06-01</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D. Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromodomain Helicase DNA-binding protein 1-Like (CHD1L) is a chromatin remodeling enzyme increasingly recognized as an oncogenic factor promoting tumor progression and metastatic potential by orchestrating transcriptional programs that drive epithelial-mesenchymal transition (EMT), cytoskeletal remodeling, and metastatic dissemination. In parallel, CHD1L has emerged as a master regulator of tumor cell survival by regulating DNA damage response and repair and enforcing G1 cell cycle progression. Furthermore, CHD1L plays a key role in immune evasion pathways by regulating signaling cascades and by suppressing both apoptotic and non-apoptotic cell death. In particular, CHD1L is a key suppressor of PARthanatos, a caspase-independent mechanism triggered by poly(ADP-ribose) (PAR) polymer fragmentation and apoptosis-inducing factor (AIF) activation. By regulating SPOCK1, MDM2, and TCTP, CHD1L further supports survival under cellular stress. Its overexpression correlates with metastasis, therapy resistance, and poor prognosis across many solid tumors. This review covers CHD1L’s structure, oncogenic functions, and developmental origins, and highlights emerging therapeutic strategies that target CHD1L as a druggable vulnerability in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6296c3dbcbe7b1d1919c8debad6aee4011adbed2" target='_blank'>
              CHD1L in cancer and beyond: structure, oncogenic functions, and therapeutic potential
              </a>
            </td>
          <td>
            Sophia Clune, Paul Awolade, Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-05-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Among the various genotypes of the human papillomavirus (HPV), the HPV16 genotype is the most oncogenic by far, and some (sub)lineages may be associated with an increased risk of cancer. Thus, characterizing the full genome is important to understand the link between the HPV16 genome variability and its transforming role. We set up a multiplex PCR approach combined with Oxford Nanopore Technologies for the HPV16 full-genome characterization. The primers were designed with PrimalScheme, and the optimization of the monoplex/multiplex PCR was performed on SiHa cells. After library preparation, the sequencing was performed on a GridION sequencing device to determine the sensitivity, the specificity, and the performance of the method. Fourteen primer pairs were selected to span the full genome. With the monoplex PCR, 12 primer pairs showed good amplification, while a double concentration of two primers was required to improve the amplification. We then performed a multiplex PCR approach by generating two pools comprising non-overlapping primer pairs. The multiplex PCR showed good sensitivity, allowing the amplification of HPV16 with a Ct value below 27 and providing a coverage greater than 99.9% and a sequencing depth greater than 100×. In addition, the specificity of the method was validated by the absence of amplification of other high-risk HPV genotypes compared with HPV16. We set up an amplicon-based approach to characterize the full genome and diversity of the HPV16 (sub)lineages. This approach shows good sensibility and specificity with limited cost, opening new perspectives in the field of whole-genome HPV sequencing. IMPORTANCE HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field. HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836ca2c5fafa3f7dae222db6854bdcbbe566eaa9" target='_blank'>
              An amplicon-based approach for full-genome characterization of HPV16
              </a>
            </td>
          <td>
            I. Malet, Inès Draa, V. Leducq, Fanny Vuong, Pascale Bonnafous, A. Marcelin, V. Calvez, A. Jary
          </td>
          <td>2025-06-09</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Plant nuclear genomes contain thousands of genes of mitochondrial and plastid origin as the result of endosymbiotic gene transfer (EGT). EGT is a still-ongoing process, but the molecular mechanisms determining its frequency remain largely unknown. Here we demonstrate that nuclear double-strand break (DSB) repair is a strong suppressor of EGT. Through large-scale genetic screens in tobacco plants, we found that EGT from plastids to the nucleus occurs more frequently in somatic cells when individual DSB repair pathways are inactive. This effect is explained by the expected increase in the number and residence time of DSBs available as integration sites for organellar DNA. We also show that impaired DSB repair causes EGT to increase 5- to 20-fold in the male gametophyte. Together, our data (1) uncover DSB levels as a key determinant of EGT frequency, (2) reveal the strong mutagenic potential of organellar DNA and (3) suggest that changes in DNA repair capacity can impact EGT across evolutionary timescales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7706b7ef66d1fdcd1d30d7b318635c18156686f3" target='_blank'>
              Suppression of plastid-to-nucleus gene transfer by DNA double-strand break repair
              </a>
            </td>
          <td>
            Enrique Gonzalez-Duran, Xenia Kroop, Anne Schadach, Ralph Bock
          </td>
          <td>2025-05-16</td>
          <td>Nature Plants</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Background Ewing sarcomas (ESs) are aggressive paediatric tumours of bone and soft tissues afflicting children and adolescents. Despite current therapies having improved the 5-year survival rate to 70% in patients with localized disease, 25% of patients relapse and most have metastasis at diagnosis. Resistance to chemotherapy, together with the high propensity to metastasize, remain the main causes of treatment failure. Thus, identifying novel targets for alternative therapeutic approaches is urgently needed. Methods Biochemical and functional analyses were carried out to elucidate the mechanism of regulation of YAP1 expression by pncRNA_YAP1-1 in ES cells. Results Here, we identified a novel promoter-associated noncoding RNA, pncRNA_YAP1-1, transcribed from the YAP1 promoter in ES cells. We found that pncRNA_YAP1-1 level exerts antitumour effects on ES by destabilizing YAP1 protein. The molecular mechanism relies on the interaction of pncRNA_YAP1-1 with the RNA binding protein FUS, which stabilizes the transcript. Furthermore, pncRNA_YAP1-1 binding to TEAD impairs its interaction with YAP1, thus determining YAP1 translocation into the cytoplasm, its phosphorylation and degradation. Conclusions Overall, our findings reveal a novel layer of regulation of YAP1 protein expression by pncRNA_YAP1-1 in Ewing sarcoma. Considering the role of YAP1 in therapy response and cell propensity to metastasize, our results indicate pncRNA_YAP1-1 as an actionable target that could be exploited to enhance chemotherapy efficacy in Ewing sarcoma. Significance PncRNA_YAP1-1 counteracts the YAP1 oncogenic transcriptional program in Ewing sarcoma cells by interfering with YAP1-TEAD interaction and impairing YAP1 protein stability. These findings uncover a novel treatment option for Ewing sarcoma. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00736-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945aa0f8515a96b425b539864e7b5d32769ff4ff" target='_blank'>
              YAP1 promoter-associated noncoding RNA affects Ewing sarcoma cell tumorigenicity by regulating YAP1 expression
              </a>
            </td>
          <td>
            L. Chellini, Arianna Del Verme, Veronica Riccioni, M. Paronetto
          </td>
          <td>2025-05-25</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ead10baece9545ae1e3801dde43ce3d766e518" target='_blank'>
              METTL5 Blockade Enhances Anti-Tumor Immune Response via Inducing Neoantigen Generation
              </a>
            </td>
          <td>
            Yangyi Zhang, Xiaoyan Shi, Yuci Wang, Ruiqi Wang, Yanlan Cao, Hao Chen
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) contributes to 3% of all childhood cancers with roughly 400-500 cases diagnosed each year in the United States. The World Health Organization classifies rhabdomyosarcoma into four histological subtypes which include alveolar, embryonal, spindle-cell and pleomorphic. The primary genetic drivers in a subset of alveolar and spindle-cell histological subtypes are gene fusions. This review explores the fusion oncogenes identified in RMS such as PAX-and NCOA2-based fusions, along with discussing studies defining fusion oncogene biology and tumorigenic mechanisms. Focus areas include data around transformation events and progression along with dysregulated biological processes. Furthermore, we summarize model systems, ranging from cell to animal models, that have been implemented to study fusion oncogenes identified in RMS. With the constant identification of novel fusion oncogenes, this review also emphasizes the need for genetically characterizing RMS tumors and rapidly developing new model systems. These models are critical to study fusion oncogene activity and to delineate key regulatory players and potential therapeutic targets that suppress tumorigenesis. The identification of RMS fusion oncogenes and integration with animal and cell culture models will help identify conserved molecular targets, optimize therapeutic approaches, and ultimately improve clinical outcomes for children with RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23d84102b5431b0bc6bd415779fd94bf54425c" target='_blank'>
              Fusion oncogenes in rhabdomyosarcoma: model systems, mechanisms of tumorigenesis, and therapeutic implications
              </a>
            </td>
          <td>
            Chinmay S. Sankhe, Lisa Hall, Genevieve C. Kendall
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5be1cdd710efff61ea77e5d24b0d7c017a38d79" target='_blank'>
              Temporal and spatial coordination of DNA segregation and cell division in an archaeon
              </a>
            </td>
          <td>
            Joe Parham, Valerio Sorichetti, Alice Cezanne, Baukje Hoogenberg, Sherman Foo, Yin-wei Kuo, Eloise Mawdesley, Lydia Daniels Gatward, Jerome Boulanger, Ulrike Schulze, A. Šarić, B. Baum
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abnormal nuclear morphology is a hallmark of human diseases, including cancers and age-related disorders. Previously, maintenance of nuclear morphology and integrity was thought to be solely dependent on a force balance between nuclear mechanical resistance and actin antagonism. However, our recent work revealed that inhibiting RNA polymerase II suppresses nuclear blebbing independent of altering force balance, but the mechanism remains unknown. Through removing cell culture media serum and then adding it back, we can decrease and then restore transcriptional activity. Decreasing transcriptional activity decreases nuclear bleb formation, stability, and rupture while returning transcriptional activity restores nuclear blebbing. These modulations of transcriptional activity did not alter nuclear or actin mechanics. The mean square displacement (MSD) of chromatin domains labeled via transfected Cy3-dNTPs revealed that transcription activity regulates chromatin motion. To determine if increasing chromatin motion is a mechanism to increase nuclear blebbing, we used an established RAD51 inhibitor BO2. We verified BO2 increases chromatin domain motion which resulted in increased nuclear blebbing. We reveal the mechanism by which transcriptional activity drives nuclear blebbing is through chromatin motion. Thus, two hallmarks of human disease are directly linked via transcriptional activity and abnormal nuclear shape. Statement of Significance Nuclear blebs are hallmarks of disease progression that cause dysfunction, but how they are formed remains unanswered. We find that chromatin motion generated by transcriptional activity is essential for both nuclear bleb formation and stability. This was independent of changes in nuclear stiffness or actin antagonism. This finding provides a key advancement in our understanding of nuclear bleb formation. Furthermore, it reveals transcriptional activity as a novel contributor to nuclear blebbing in addition to the paradigm of nuclear shape determined as a force balance between nuclear resistance and actin antagonism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a26f910c21dd15469983a962e8b18f8de0bd93e" target='_blank'>
              Transcriptional activity generates chromatin motion that drives nuclear blebbing
              </a>
            </td>
          <td>
            Kelsey Prince, Katie Lin, Andy Li, Nick Borowski, A. Stephens
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774b7db029018d408461488af66408e59c0c7b28" target='_blank'>
              Physical principles of phase-separation action on chromatin looping associated to pathogenic gene activation
              </a>
            </td>
          <td>
            Sougata Guha, Andrea Fontana, Alex Abraham, Simona Bianco, Andrea Esposito, Mattia Conte, Sumanta Kundu, Ciro Di Carluccio, Francesca Vercellone, Florinda Di Pierno, Mario Nicodemi, A. Chiariello
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The tumor suppressor gene NKX3.1 and the LPL gene are located in close proximity on chromosome 8, and their deletion has been reported in multiple studies. However, the significance of LPL loss may be misinterpreted due to its co-deletion with NKX3.1, a well-established event in prostate carcinogenesis. This study investigates whether LPL deletion represents a biologically relevant event or occurs merely as a bystander to NKX3.1 loss. We analyzed 28 formalin-fixed paraffin-embedded prostate cancer samples with confirmed LPL deletion and 28 without. Immunohistochemical staining was performed, and previously published whole-genome sequencing data from 103 prostate cancer patients were reanalyzed. Deletion of the 8p21.3 region was associated with higher Gleason grade groups. While NKX3.1 expression was significantly reduced in prostate cancer compared to benign prostatic hyperplasia, LPL protein expression showed no significant difference between cancerous and benign tissue, nor was it affected by the 8p21.3 deletion status. Copy number analysis confirmed the co-deletion of NKX3.1 and LPL in 54 patients. Notably, NKX3.1 loss without accompanying LPL deletion was observed in eight additional cases. These findings suggest that LPL deletion is a passenger event secondary to NKX3.1 loss and underscore the importance of cautious interpretation of cytogenetic findings involving the LPL locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74796121b67d50bf9eb77e02037db96e6cc25807" target='_blank'>
              Deletions of LPL and NKX3.1 in Prostate Cancer Progression: Game Changers or By-Standers in Tumor Evolution
              </a>
            </td>
          <td>
            Tereza Vodičková, Mária Wozniaková, Vladimír Židlík, Jana Žmolíková, Jana Dvořáčková, Adéla Kondé, Jana Schwarzerová, Michal Grepl, J. Bouchal
          </td>
          <td>2025-05-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is an aggressive hematopoietic malignancy, necessitating the exploration of innovative therapeutic approaches. Targeting epigenetic mechanisms has emerged as a promising avenue for cancer treatment. EP300 belongs to the KAT3 family of histone/non-histone lysine acetyltransferases, regulating gene expression by acetylating H3K27. However, the role of EP300 and its potential as a targeted therapy in DLBCL remains unknown. Methods Public datasets were collected to evaluate the expression and clinical significance of epigenetic modification-related genes in patients with DLBCL. Flow cytometry, colony formation, and western blotting were conducted to investigate the function of EP300. CCK8, proliferation, cell cycle, and apoptosis assays, as well as experiments in tumor-bearing mouse models were conducted to determine the therapeutic effect of the EP300 inhibitor A485 alone or in combination with the XPO1 inhibitor KPT8602. RNA-seq was used to investigate the molecular mechanisms underlying the inhibition of DLBCL development by A485. Results EP300 is frequently overexpressed in DLBCL and is associated with poor prognosis, highlighting its potential role in lymphoma progression. In this study, we found that A485, a novel small-molecule inhibitor targeting the conserved histone acetyltransferase (HAT) domain of EP300, significantly reduced H3K27Ac levels and demonstrated potent antitumor effects in DLBCL cells, both in vitro and in vivo. Furthermore, we showed that A485 attenuated DLBCL progression by inhibiting the MYC and E2F1 pathways. Notably, the combination of A485 with the XPO1 inhibitor KPT8602 produced synergistic anti-lymphoma in vitro and in vivo effects in DLBCL cell lines. This combination therapy resulted in enhanced tumor suppression in a DLBCL xenograft model with minimal toxicity. These findings suggested that targeting EP300, particularly in conjunction with XPO1 inhibition, could represent a promising therapeutic strategy for DLBCL treatment. Conclusions Our study elucidated that EP300 inhibition, especially in combination with XPO1 blockade, could serve as a promising therapeutic strategy for the treatment of DLBCL. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14257-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920e8f7451cedecae320b60bb672cda4294ef282" target='_blank'>
              Targeting EP300 in diffuse large b-cell lymphoma: efficacy of A485 and synergistic effects with XPO1 inhibition
              </a>
            </td>
          <td>
            Yanan Jiang, Donghui Xing, Xiang He, Wenqi Wu, Hong Xu, H. Sun, Yixin Zhai, Kaiping Luo, Zhigang Zhao
          </td>
          <td>2025-05-28</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a013cf13f270b70a82c0a8355212488d6a1c6ef" target='_blank'>
              Microhomology-Mediated End Joining as a Novel Mechanism Underlying Androgen Insensitivity Syndrome
              </a>
            </td>
          <td>
            J. M. Marques, R. M. Ramos, N. D. R. D' Alessandre, G. D. A. Guardia, A. F. Afonso, B. L. Braga, M. F. D. A. Funari, M. Y. Nishi, P. Asprino, S. Domenice, P. A. F. Galante, B. B. Mendonca, R. L. Batista
          </td>
          <td>2025-06-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background
 DNA 5-methylcytosine (5mC) and RNA N6-methyladenosine (m6A) methylation are prevalent modifications in eukaryotes, both playing crucial roles in gene regulation. Recent studies have explored their crosstalk and impact on transcription. However, the intricate relationships among 5mC, m6A, and gene expression remain incompletely elucidated. Results We collect data on 5mC, m6A, and gene expression from samples from three tissues from each of four pregnant cattle and sheep. We construct a comprehensive genome-wide self-interaction (same gene) and across-interaction (across genes) network of 5mC and m6A within gene-bodies or promoters and gene expression in both species. Qualitative analysis identifies uniquely expressed genes with specific m6A methylation in each tissue from both species. A quantitative comparison of gene expression ratio between methylated and unmethylated genes for m6A within gene body and promoter, and 5mC within gene body and promoter confirms the positive effect of RNA methylation on gene expression. Importantly, the influence of RNA methylation on gene expression is stronger than that of DNA methylation. The predominant self- and across-interactions are between RNA methylation within gene bodies and gene expression, as well as between RNA methylation within promoters and gene expression in both species. Conclusions RNA methylation has a stronger effect on gene expression than does DNA methylation within gene bodies and promoters. DNA and RNA methylation in gene-bodies has a greater impact on gene expression than those in promoters. These findings deepen comprehension of the dynamics and complex relationships among the epigenome, epitranscriptome, and transcriptome, offering fresh insights for advancing epigenetics research. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03617-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79390cad38f810c7589608ce6b54c5711ffacee" target='_blank'>
              Analyzing the relationship of RNA and DNA methylation with gene expression
              </a>
            </td>
          <td>
            Shangqian Xie, Darren Hagen, G. Becker, Kimberly M. Davenport, Katie A Shira, Morgan R Stegemiller, Jacob W. Thorne, Sarem F. Khilji, D. Konetchy, Patricia Villamediana, Brenda M Murdoch, S. McKay
          </td>
          <td>2025-05-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The extent of genetic variation and its influence on gene expression across multiple tissue and cellular contexts is still being characterized, with germline Structural Variants (SVs) being historically understudied. DNA methylation also represents a component of normal germline variation across individuals. Here, we combine germline SVs (by short-read sequencing) with tumor DNA methylation across 1292 pediatric brain tumor patients. For thousands of methylation probes for CpG Islands (CGIs) or enhancers, rare and common SV breakpoints upstream or downstream associate with differential methylation in tumors spanning various histologic types, a significant subset involving genes with SV-associated differential expression. Cancer predisposition genes involving SV-associated differential methylation and expression include MSH2, RSPA, and PALB2. SV breakpoints falling within CGIs or histone marks H3K36me3 or H3K9me3 associate with differential CGI methylation. Genes with SVs and CGI methylation associated with patient survival include POLD4. Our results capture a class of normal phenotypic variation having disease implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f12392b4958c7351af5ca7ce23e9c5777fb52b" target='_blank'>
              Global DNA methylation differences involving germline structural variation impact gene expression in pediatric brain tumors
              </a>
            </td>
          <td>
            Fengju Chen, Yiqun Zhang, Wei Li, F. Sedlazeck, Lanlan Shen, Chad J. Creighton
          </td>
          <td>2025-05-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pediatric brain tumors, the most devastating cancers affecting children, are believed to originate from neural stem/progenitor cells in developing brain. In precise timing and specific regions during the brain development, chromatin deregulation plays crucial roles in redirecting normal neuronal differentiation pathways toward tumorigenesis. Indeed, epigenomic abnormalities are thought to be more important for brain tumor formation especially in children than adults, as pediatric brain tumors generally exhibit fewer genetic mutations compared to adult brain tumors. Given the small number of mutations, targeting such limited alterations involved in cancer epigenomes is expected to be more effective in pediatric brain tumors. The mechanisms of cancer epigenomes include mutation or dysregulation of chromatin remodelers, histone modifiers, histones themselves, and DNA methylation enzymes. Furthermore, genomic rearrangements and/or higher-order chromatin topology also contribute to these epigenomic mechanisms. These mechanisms are commonly observed in various types of pediatric brain tumors. However, alterations in chromatin regulatory factors differ across tumor types, reflecting the unique epigenetic landscapes shaped by their tumor origins. Accordingly, clarifying their functional similarities and differences across tumor types could offer valuable insights for finding new therapeutic strategies. Thus, this review article focuses on elucidating how pediatric brain tumors arise from epigenomic deregulation and what epigenetic molecules or mechanisms could serve as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a334571a5c8d012851148672d620a490ae61353" target='_blank'>
              Epigenetic modifications and their roles in pediatric brain tumor formation: emerging insights from chromatin dysregulation
              </a>
            </td>
          <td>
            Kento Kawata, Owen S. Chapman, Satoshi Narumi, Daisuke Kawauchi
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mosaic chromosomal alterations (mCAs) in blood, a form of clonal hematopoiesis, have been linked to various diseases, but their role in Alzheimer's disease (AD) remains unclear. We analyzed blood whole-genome sequencing (WGS) data from 24,049 individuals in the Alzheimer's Disease Sequencing Project and found that autosomal mCAs were significantly associated with increased AD risk (odds ratio = 1.27; P = 1.3 x 10-5 ). This association varied by ancestry, mCA subtype, APOE {epsilon}4 allele status, and chromosomal location. Using matched blood WGS and brain single-nucleus RNA-seq data, we identified microglia-annotated cells in the brain carrying the same mCAs found in blood. These findings suggest that blood mCAs may contribute to AD pathogenesis, potentially through infiltration into the brain and influencing local immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3af27139f0dd37c6b832ce61aa2ff78b9b4570d" target='_blank'>
              Mosaic chromosomal alterations in blood are associated with an increased risk of Alzheimer’s disease
              </a>
            </td>
          <td>
            T. Naito, Kosei Hirata, Beomjin Jang, C. Lakhani, Alice Buonfiglioli, Wan-Ping Lee, O. Valladares, Li-San Wang, Yukinori Okada, Hong-Hee Won, E. Papapetrou, Samuele G. Marro, D. A. Knowles, T. Raj
          </td>
          <td>2025-06-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, plays a crucial role in maintaining genomic stability by facilitating DNA double-strand break (DSB) repair through chromatin loop formation and transcriptional regulation. Its deficiency is frequently observed in various cancers and is associated with impaired DNA repair mechanisms and increased accumulation of cytosolic DNA. This triggers the activation of the CGAS-STING pathway, a cytosolic DNA-sensing mechanism that induces type I interferon (IFN-I) responses and expression of interferon-stimulated genes (ISGs) such as IFIT1, ISG15, CXCL9, CXCL10, and CCL5. These immune modulators enhance antitumor immunity by promoting immune cell infiltration into the tumor microenvironment. Experimental studies using ARID1A-deficient cancer cell lines—including MDA-MB-231 and AID-Diva—demonstrated increased expression of IFN-α, IL-6, and chemokines upon DNA damage, confirming CGAS-STING pathway activation. Clinically, these downstream genes serve as potential biomarkers for predicting response to DNA-damaging agents and immune checkpoint inhibitors. The findings support the potential of targeting DNA damage-induced innate immune signaling to enhance therapeutic efficacy in ARID1A-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86c82b6f917ff940d43e0ac6c93824eea43df3f7" target='_blank'>
              Review: Immune Gene Regulation via cGAS-STING in ARID1A-deficient cancers and their response to therapeutic stress
              </a>
            </td>
          <td>
            Syakirah Shafwah Rahmadani, Awik Puji Dyah Nurhayati
          </td>
          <td>2025-05-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="VIRMA (also known as KIAA1429), as a core regulatory subunit of the m6A methyltransferase complex, plays a key role in tumorigenesis and progression by dynamically regulating RNA methylation modifications. Studies have shown that VIRMA is aberrantly overexpressed in more than 20 types of malignant tumors, including liver cancer, breast cancer, and lung cancer, and is significantly associated with chromosome 8q amplification and poor prognosis. Its mechanism of action involves regulating the expression of tumor-associated genes through both m6A-dependent and m6A-independent pathways, thereby promoting tumor proliferation, metastasis, and drug resistance. These findings suggest that VIRMA has the potential to serve as a pan-cancer diagnostic and prognostic biomarker. This review summarizes the role of VIRMA in malignant tumors from multiple perspectives and explores its potential applications in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1f0fb758f358894db60bb1aa6efc139d365cc4" target='_blank'>
              The Multifaceted Role of VIRMA, a Core Component of the Methyltransferase Complex, in Cancer and Cancer Therapy
              </a>
            </td>
          <td>
            Jinmeng Lu, Chengyu Zhang, Mengshuang Yin, Huili You, Chao Xiong, Jing Wu, Ying Gong, Zhangang Xiao, Jing Shen
          </td>
          <td>2025-06-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Inflammatory breast cancer (IBC), of which HER2 + is the predominant subtype, is extremely aggressive and difficult to treat. Previous studies have suggested that targeting the tumour microenvironment (TME) may provide new directions for IBC diagnosis and treatment. Methods In this study, we used single-cell transcriptome technology (scRNA-seq) to investigate the molecular features of the TME of HER2 + IBC patients and performed a comprehensive and detailed comparison of the cellular components and molecular phenotypes of the TME between IBC patients and noninflammatory breast cancer (nIBC) patients to elucidate the cell types that are specifically enriched in the TME of IBC patients, as well as the molecular features that are responsible for the preferential remodelling of the cellular functional state in the TME. Results A total of 15,832 cells, including epithelial cells, endothelial cells, stromal cells, T cells, B cells, antibody secreting cells (ASCs) and myeloid cells, were obtained from tumour tissues from 3 HER2 + IBC patients for scRNA analysis. By comparing the TME with that of HER2 + nIBC patients in a public database, we found that the TME of HER2 + IBC patients had a greater level of lymphocyte infiltration than that of nIBC patients did, and an especially significant enrichment of ASCs (mainly plasmablasts or plasma cells). In the TME of HER2 + IBC patients, tumour-infiltrating T cells exhibited a dual molecular phenotype of high activation and high exhaustion, with tumour-infiltrating B cells preferring the extrafollicular developmental pathway, and tumour-infiltrating myeloid and mesenchymal cells exhibiting a greater immunosuppressive status. By performing a cellular interaction analysis, we revealed that PTN molecules were significantly overexpressed in HER2 + IBC tumour cells and that the cellular interactions mediated by these molecules were strongly correlated with the functional polarisation of the cellular components in the TME. We observed that HER2 + IBC tumour cells have an active interferon response and epithelial mesenchymal transition (EMT) signalling, and that their malignant process is strongly correlated with the inflammatory response. Moreover, we found that HER2 + IBC tumour-infiltrating B cells promoted necroptosis of endothelial cells through high expression of TNF, thus promoting inflammatory responses. Conclusion We found a strong correlation between high expression of PTN molecules in HER2 + IBC tumour cells and their highly invasive characteristics and highly immunosuppressive TME. These results suggest that HER2 + IBC tumour cells can promote an inflammatory response by upregulating the expression of TNF molecules in B cells via PTN molecules and that the enhanced inflammatory response in turn promotes tumour progression, a malignant cycle that shapes a more immunosuppressive TME. Therefore, diagnostic and therapeutic strategies targeting the PTN-TNF molecular axis may have considerable potential for development in HER2 + IBC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03454-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c28eda21e6edafced575cd62b5c13be2df5e23" target='_blank'>
              Single-cell transcriptome analysis reveals the malignant characteristics of tumour cells and the immunosuppressive landscape in HER2-positive inflammatory breast cancer
              </a>
            </td>
          <td>
            Juan Huang, Yongwei Zhu, Wenjing Zeng, Yulong Zhang, Weizhi Xia, Fan Xia, Liyu Liu, Kuansong Wang, Yidi Guan, Taohong Shen, Bingjian Jiang, Lunquan Sun, Ayong Cao, Shouman Wang, Zhi Li
          </td>
          <td>2025-07-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In mammals, the binding of transcriptional activators leads to the repositioning of silent loci from the nuclear periphery to the nuclear interior. However, it remained unknown whether the same mechanism functions in Drosophila. Here, using FISH and DamID, we have shown that binding the GAL4 activator to the silent loci causes weakening of their interactions with the nuclear lamina and relocalization inside nuclei in Drosophila salivary gland cells and neurons. This mimics the removal from the nuclear periphery of a neuron-specific gene upon its activation in neurons. Salivary gland cells contain polytene chromosomes with mechanical properties, different from chromosomes of diploid cells, while neurons represent predominantly non-dividing cell type. Our results indicate a causal relationship between transcriptional activator binding and changes in the intranuclear position of loci in Drosophila. They also point to the similarity in general chromatin dynamics in mammals and Drosophila, thus strengthening the role of model organisms in studying genome architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a72e89c238e1e68a064b0436b325874b1c8e025b" target='_blank'>
              Binding of Transcriptional Activator to Silent Loci Causes Their Detachment from the Nuclear Lamina in Drosophila Neurons and Salivary Gland Cells
              </a>
            </td>
          <td>
            R. Simonov, O. Olenkina, V. Nenasheva, Yuri A. Abramov, Sergey A. Lavrov, A. Fedotova, Y. Y. Shevelyov
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaee95d8b33babc294494d923d1f29bffe920cba" target='_blank'>
              Gain of Function p53 mutant R273H confers distinct methylation profiles and consequent partial or full EMT states to colon tumour
              </a>
            </td>
          <td>
            H. Rani, Seemadri Subhadarshini, M. Jolly, V. Mahadevan
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="
 10043


 Background:
 Risk stratification in osteosarcoma relies on metastatic status and tumor necrosis after chemotherapy. Despite a complex genomic landscape, genomic biomarkers are not yet used for predicting therapy response.
 MYC
 amplification has previously been identified as a potential biomarker for chemotherapy resistance but studies of MYC expression are limited. This study evaluated the relationship between
 MYC
 amplification and protein expression and between these biologic features and survival in pediatric and young adult osteosarcoma.
 Methods:
 In a cohort of 93 patients with high-grade osteosarcoma,
 MYC
 copy number was evaluated using a targeted sequencing panel, and MYC protein expression was quantified via IHC with an H-Score. The relationship between copy number and protein expression were evaluated via spearman correlation. Amplification (AMP) was defined as > 7 copies, and high expression (EXP) as > 175 H-score. The primary outcome, overall survival (OS), was assessed using Kaplan Meier analysis (median [IQR]) for unadjusted models and cox proportional hazard analysis (HR±SE) with models adjusted for metastatic status at diagnosis.
 Results:
 Among the 93 patients, 64% were male, with a median age of 14 years [range: 4–29].
 MYC
 AMP was present in 16 (17%) patients, high MYC EXP in 19 (20%), and both AMP + high EXP in 8 (9%) patients. Copy number status was positively correlated with expression (r = 0.57, p < 0.0001). Patients with AMP were more likely metastatic at diagnosis (75% vs 38%, p = 0.011) and more so for AMP + high EXP (100% vs 39%, p = 0.0009), but not high EXP alone (63% vs 39%, p = 0.11). OS was reduced for AMP compared to non-AMP (median OS: 1.3 [1.0, 2.6] vs 6.2 [2.8, 12.8] years, p < 0.0001; Adj HR 3.2±0.4, p = 0.001) and high EXP compared to low EXP (median OS: 1.5 [1.0, 3.1] vs. 6.2 [2.8, 12.8] years, p < 0.0001; Adj HR 5.6±0.4, p < 0.0001). A dose-response relationship was seen with higher copy number or expression linked to reduced OS. Compared to non-AMP + low EXP (median OS: 7.23 [3.1, 12.8] years) there was reduced OS for AMP-only/high EXP-only (median OS: 3.1 [2.0, 4.0] years, p = 0.01; Adj HR 2.8±0.4, p = 0.009) and further reduced OS for AMP + high EXP (median OS: 1.0 [0.5, 1.0] years, p < 0.0001; Adj HR 15.3±0.5, p < 0.0001). Metastasis at diagnosis predicted reduced OS compared to localized disease (median OS: 2.0 [1.0, 4.5] vs 6.2 [4.0, 12.8] years, p < 0.0001), but age, sex, and tumor necrosis were not associated with OS.
 Conclusions:
 MYC
 amplification and protein expression are positively correlated, and both independently and synergistically predict poor OS in osteosarcoma, even after adjusting for metastatic status at diagnosis. Incorporating
 MYC
 amplification and expression status into risk stratification may help identify patients with worse prognosis. These data also underscore the need for therapeutic approaches tailored to the genetic basis of osteosarcoma tumor biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bf0231245d22b844f02a713a0c76283c5de3ea" target='_blank'>
              MYC amplification and protein expression as prognostic markers in pediatric and young adult osteosarcoma.
              </a>
            </td>
          <td>
            Matthew Nagy, Olivia Puopolo, Erin Alston, Sreekar Challa, E. Ceca, Yvonne Y. Li, Andrew D Cherniack, Lorena Lazo de la Vega, A. Church, K. Janeway
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Glioma is a highly malignant primary tumor of the central nervous system, characterized by highly invasive behavior and a high recurrence rate. The current treatment options for gliomaoften have unsatisfactory outcomes. Replication protein A1 (RPA1), a component of the RPA heterotrimer, has been identified as a potential oncogene implicated in the development and clinical prognosis of various types of solid tumors, including liver cancer, nasopharyngeal carcinoma and gastrointestinal cancer. However, studies on the associations between RPA1 expression and glioma cell proliferation or patient prognosis are limited, and the underlying mechanisms remain unclear. The aim of the present study was to explore the expression of RPA1 in glioma and its clinical relevance in the clinicopathology and prognosis of glioma. In addition, the biological functions and signal transduction pathways mediated by RPA1 were predicted using bioinformatics analysis to provide basic theoretical support for further mechanistic research and in vivo experiments. The Cancer Genome Atlas (TCGA) data analysis, and reverse transcription-quantitative PCR, western blot analysis and immunohistochemical (IHC) staining were performed to analyze RPA1 expression in glioma cells or tissues of different grades. The associations between RPA1 expression and various pathological parameters related to tumor cell proliferation and patient prognosis were also investigated. Furthermore, the biological processes and signaling pathways influenced by RPA1-related target genes were predicted. The results revealed that RPA1 expression varies among different grades of gliomas, with higher World Health Organization (WHO) grades exhibiting higher RPA expression levels. Furthermore, both bioinformatics analysis and IHC staining results revealed that high RPA1 expression was associated with a shorter overall survival (OS). In addition, RPA1 expression exhibited a significant association with WHO glioma grade and key markers of malignancy, namely Ki-67 expression and p53 mutation status. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes enrichment analyses suggested that RPA1 may promote the development of glioma through pathways including ‘DNA replication’, ‘Fanconi anemia pathway’, ‘mismatch repair’, ‘homologous recombination’, ‘nucleotide excision repair’ and ‘cell cycle’. In conclusion, RPA1 expression in glioma tissues and cells is positively associated with the degree of glioma malignancy, and high RPA1 expression is associated with a shorter OS in patients with glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb9154b84dfbf0461fd48c7dd062feb43da7f114" target='_blank'>
              Role and mechanism of RPA1 in the development and progression of glioma
              </a>
            </td>
          <td>
            Zhiming Zhao, Yu Sun, Ping Yin, Quan Zhang, Jianfu Zhang
          </td>
          <td>2025-06-26</td>
          <td>Experimental and Therapeutic Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78533e7da1fcbb6085c8100dfa95e814a9a6b0e7" target='_blank'>
              Deficiencies in the Fanconi Anemia or the Homologous Recombination pathway enhance the antitumor effects of the novel hypoxia-activated prodrug CP-506
              </a>
            </td>
          <td>
            Lesley Schuitmaker, A. M. van der Wiel, N. Lieuwes, R. Biemans, Nikki A.M. Mutsters, Jennifer Jung, Victoria Claudino Bastos, Èlia Prades Sagarra, Sheng Kuang, S. A. Langie, J. Setton, J. Theys, A. Yaromina, Ludwig J. Dubois, P. Lambin
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61df03d16c11da50362eb27baf0281aee3eb21d5" target='_blank'>
              A novel role for UPF1 in safeguarding DNA replication in S phase and beyond
              </a>
            </td>
          <td>
            Thomas Walne, Laura Maple, Nan Li, Isabel Christie, Carl Smythe, Ruth Thompson
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cf680837c223860a295c7d973297324c3a3da0" target='_blank'>
              From predictive biomarker to therapeutic target: the dual role of SLFN11 in chemotherapy sensitivity.
              </a>
            </td>
          <td>
            Yue Feng, Yingze Li, Zhenhao Zhang, Yuxuan Dai, Xingchun Gou, Kejing Lao, Running Zhang
          </td>
          <td>2025-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861ff3d8af6cc40f991267fd62388116259dc4df" target='_blank'>
              EWSR1::ETS-low cells promote metabolic reprogramming of the tryptophan-kynurenine-AHR axis, immunosuppression, and poor outcome in Ewing sarcoma
              </a>
            </td>
          <td>
            Martha J. Carreño-Gonzalez, Kimberley M. Hanssen, Ahmed Sadik, Alessa Henneberg, Anna C. Ehlers, A. K. Ceranski, Zuzanna Kolodynska, Malenka Zimmermann, R. Imle, Florian H. Geyer, Alina Ritter, A. Banito, M. Knott, S. Ohmura, F. Cidre-Aranaz, Christiane Opitz, T. G. Grünewald
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background In-solution hybridization enrichment of genetic markers is a method of choice in paleogenomic studies, where the DNA of interest is generally heavily fragmented and contaminated with environmental DNA, and where the retrieval of genetic data comparable between individuals is challenging. Here, we benchmark the commercial “Twist Ancient DNA” reagent from Twist Biosciences using sequencing libraries from ancient human samples of diverse demographic origin with low to high endogenous DNA content (0.1–44%). For each library, we tested one and two rounds of enrichment and assessed performance compared to deep shotgun sequencing. Results We find that the “Twist Ancient DNA” assay provides robust enrichment of approximately 1.2M target SNPs without introducing allelic bias that may interfere with downstream population genetics analyses. Additionally, we show that pooling up to 4 sequencing libraries and performing two rounds of enrichment is both reliable and cost-effective for libraries with less than 27% endogenous DNA content. Above 38% endogenous content, a maximum of one round of enrichment is recommended for cost-effectiveness and to preserve library complexity. Conclusions In conclusion, we provide researchers in the field of human paleogenomics with a comprehensive understanding of the strengths and limitations of different sequencing and enrichment strategies, and our results offer practical guidance for optimizing experimental protocols. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03622-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c477c495125e3cd66f843708e054b0141348cbd8" target='_blank'>
              Optimized in-solution enrichment of over a million ancient human SNPs
              </a>
            </td>
          <td>
            Roberta Davidson, Xavier Roca-Rada, Shyamsundar Ravishankar, Leonard Taufik, Christian Haarkötter, Evelyn Collen, Matthew P. Williams, Peter Webb, M. I. Mahmud, Erlin Novita, Idje Djami, Gludhug A. Purnomo, Cristina Santos, A. Malgosa, Linda R. Manzanilla, Ana Maria Silva, Sofia Tereso, Vítor Matos, Pedro C. Carvalho, Teresa Fernandes, A. Maurer, João C. Teixeira, Ray Tobler, Lars Fehren-Schmitz, B. Llamas
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract The homologous recombination (HR) system repairs DNA double-strand breaks produced by the DNA damage response, which is a complex signaling pathway consisting of the key proteins BRCA1/2 and other DNA repair proteins, such as the ATM, PALB2, BARD1, RAD51, and Fanconi anemia proteins. Mutations and epigenetic alterations in HR-related genes may lead to homologous recombination deficiency (HRD), resulting in genomic instability and contributing to the development of certain solid tumors. The biological significance and molecular mechanism of BRCA1/2 mutation-related HRD are well understood, but the relationships of other HR-related genes and their variant forms with HRD have not been sufficiently studied. These genes exhibit multiple forms of variation, including one or more HR genes, germline or somatic mutations, monoallelic or biallelic variants, and not all variants present HRD. Therefore, HRD is usually defined as HR-related gene variation, but recent studies have shown that defining it as the combined score of loss of heterozygosity, LST and TAI, known as the HRD score, can more accurately assess genomic instability. In patients with HRD, platinum-based therapy and poly ADP-ribose polymerase enzyme inhibitor (PARPi) have been shown to perform well in ovarian, breast, and prostate cancers. For gastrointestinal cancer (GI cancer), HRD has been relatively well studied in pancreatic cancer, but its role in other cancers has rarely been reported. Herein, we review the pathogenesis and predictive value of HRD, including the use of platinum drugs, PARPi, and immunotherapy, in digestive system tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b86ad0d0eb3efaff60321645af70d194c1741a2" target='_blank'>
              The prognostic and predictive value of homologous recombination deficiency in gastrointestinal cancer
              </a>
            </td>
          <td>
            Beibei Jiang, Yile Shang, Xiang Zhang, Wenguang He, Hanju Hua, Feng Ye, Xile Zhou, Yandong Li, Weixiang Zhong, Wei-wei Jiang, Guosheng Wu
          </td>
          <td>2025-06-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Chromosomal instability in Alzheimer’s disease (AD) neurons has been previously reported. This pilot study aimed to establish a quantitative technique for assessing X chromosome centromere signals using fluorescence in situ hybridization (FISH). Hippocampal brain tissue was collected at autopsy from sporadic AD patients and age- and gender-matched controls. FISH was utilized to detect and measure the intensity of hybridization signals for X chromosome centromeres in the interphase nuclei of hippocampal brain cells. The premature centromere division (PCD) phenomenon, marked by a close bipartite signal appearing as two separated FISH spots, was examined to see if the hybridized DNA amount in each spot matched the expected centromere DNA amount. The technique effectively distinguished between PCD+ and PCD− signals. The average PCD frequency of the X chromosome in the AD group was 7 ± 1%, compared with 3.2 ± 0.84% in the controls. This quantitative approach supports qualitative analyses of FISH centromere spots, reinforcing findings of chromosomal instability in AD. The presence of a double signal at the centromere of a single X chromosome indicates re-entered cell cycles, DNA replication, and PCD in hippocampal neurons. This technique provides a reliable method for identifying PCD + signals and contributes to understanding chromosomal instability in AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682335163a6ea498c4fd17acd3e21f0876fd8d01" target='_blank'>
              Assessment of X Chromosome Centromere Instability in Alzheimer’s Disease: A Quantitative FISH Approach
              </a>
            </td>
          <td>
            B. Spremo-Potparević, Petar Popović, Dijana Topalović, Andrea Pirković, George Perry, L. Živković
          </td>
          <td>2025-06-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary This study investigated circRNA_4083, which is a stable circular RNA which originates from the MSH3 gene of chicken. It is highly expressed in cardiac, pulmonary and renal tissues. Its biogenesis depends on flanking genomic sequences. CircRNA_4083 overexpression reduced apoptosis and enhanced cell proliferation, indicating its role in promoting cellular viability. Bioinformatics analysis indicates that circRNA_4083 interacts with RNA-binding molecules and regulates pathways that are crucial for genomic stability, possibly protecting the genome from damage. These findings underscore its importance in maintaining cellular health. By elucidating circRNA-driven mechanisms, this work advances the understanding of non-coding RNA functions in poultry biology, offering insights for developing strategies to improve disease resilience and productivity in agricultural practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c0e289cb6c7acbb75967e45fbe212b55b3a552a" target='_blank'>
              Mechanism of circRNA_4083 Circularization and Its Role in Regulating Cell Viability
              </a>
            </td>
          <td>
            Wenhao Li, Ting Yang, Haojie Wang, Hao Bai, Guobin Chang, Lingling Qiu
          </td>
          <td>2025-05-23</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The ATP-hydrolytic ectoenzyme ENPP1 has been implicated in the metastasis and recurrence in triple-negative breast cancer (TNBC), primarily by contributing to tumor cell survival and treatment resistance. However, the precise mechanisms remain unclear. In a model of local recurrence (LR), circulating tumor cells (CTC) engrafting in the post-resection tumor bed developed a radioresistant phenotype linked to an ENPP1+-gene signature which was also identified in TNBC patients, suggesting ENPP1´s role in genome integrity. Blockade of ENPP1 using a permeable ENPP1 inhibitor (AVA-NP-695) reduced radioresistance, mechanistically attributed to decreased homologous recombination (HR) resulting in persistent DNA damage, as evidenced by enhanced tail moment and sustained γH2AX formation. This impaired DNA damage repair (DDR) sensitized tumor cells to ionizing radiation (IR). Notably, several DDR inhibitors (i) (including PARPi and ATMi) showed the highest synergy score in a targeted pharmacological screening. In vivo, dual ENPP1/ATM inhibition heightened radiosensitivity, compromised tumor cell survival and enhanced STING-TBK1 signaling by preventing ENPP1-mediated cGAMP hydrolysis. This resulted in robust innate and long-lasting adaptive antitumor immune memory responses, leading to significant tumor regression. Remarkably, combined treatment post-IR reduced spontaneous metastasis and local recurrence, and induced abscopal effects that impacted distant tumor spread in orthotopic tumor models. Thus, these findings position ENPP1 as a critical link between genome integrity and immunosuppression, offering promising translational opportunities for treating local or distant dissemination in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34af6ea74bf5effdace3557bda27f7680f31a271" target='_blank'>
              Dual ENPP1/ATM depletion blunts DNA damage repair boosting radioimmune efficacy to abrogate triple-negative breast cancer
              </a>
            </td>
          <td>
            Borja Ruiz-Fernández de Córdoba, K. Valencia, C. Welch, Haritz Moreno, S. Martínez-Canarias, C. Zandueta, Eduardo Gómez, Alfonso Calvo, N. Otegui, M. Echepare, Ignacio Garzón, D. Ajona, David Lara-Astiaso, Elisabet Guruceaga, Laura Guembe, Ruben Pio, I. Melero, Silve Vicent, Fernando Pastor, R. Martinez-Monge, F. Lecanda
          </td>
          <td>2025-06-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Gram-negative anaerobe Fusobacterium nucleatum is an oral oncobacterium that promotes colorectal cancer (CRC) development with the amyloid-forming cell surface adhesin FadA integral to CRC tumorigenesis. We describe here molecular genetic studies uncovering a novel mode of metabolic regulation of FadA-mediated tumor formation by a highly conserved respiratory enzyme known as the Rnf complex. First, we show that genetic disruption of Rnf, via rnfC deletion, significantly reduces the level of fadA transcript, accompanied by a near-complete abolishment of the precursor form of FadA (pFadA), reduced assembly of FadA at the mature cell pole, and severe defects in the osmotic stress-induced formation of FadA amyloids. We show further that the Rnf complex regulates three response regulators (CarR, ArlR, and S1), which modulate the expression of pFadA, without affecting fadA transcript. Consistent with our hypothesis that these response regulators control factors that process FadA, deletion of rnfC, carR, arlR, or s1 each impairs expression of the signal peptidase gene lepB, and FadA production is nearly abolished by CRISPR-induced depletion of lepB. Importantly, while rnfC deletion does not affect the ability of the mutant cells to adhere to CRC cells, rnfC deficiency significantly diminishes the fusobacterial invasion of CRC cells and formation of spheroid tumors in vitro. Evidently, the Rnf complex modulates the expression of the FadA adhesin and tumorigenesis through a gene regulatory network consisting of multiple response regulators, each controlling a signal peptidase that is critical for the post-translational processing of FadA and surface assembly of FadA amyloids. IMPORTANCE The Rhodobacter nitrogen-fixation (Rnf) complex of Fusobacterium nucleatum plays an important role in the pathophysiology of this oral pathobiont, since genetic disruption of this conserved respiratory enzyme negatively impacts a wide range of metabolic pathways, as well as bacterial virulence in mice. Nonetheless, how Rnf deficiency weakens the virulence potential of F. nucleatum is not well understood. Here, we show that genetic disruption of the Rnf complex reduces surface assembly of adhesin FadA and FadA-mediated amyloid formation, via regulation of signal peptidase LepB by multiple response regulators. As FadA is critical in the carcinogenesis of colorectal cancer (CRC), the ability to invade CRC cells and promote spheroid tumor growth is strongly diminished in an Rnf-deficient mutant. Thus, this work uncovers a molecular linkage between the Rnf complex and LepB-regulated processing of FadA – likely via metabolic signaling – that maintains the virulence potential of this oncobacterium in various cellular niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe80828055ab4b08cc5ea46b200a6e60a644da8" target='_blank'>
              Inactivation of the Fusobacterium nucleatum Rnf complex reduces FadA-mediated amyloid formation and tumor development
              </a>
            </td>
          <td>
            Timmie A. Britton, Ju Huck Lee, Chungyu Chang, A. Bhat, Yi-Wei Chen, Rusul Mohammed Ali, Chenggang Wu, A. Das, Hung Ton-That
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The overexpressed gene MYEOV in multiple myeloma, as an oncogene, has been widely recognized for its high expression levels in various malignant tumors. MYEOV plays a significant role in multiple malignancies, particularly in diseases such as multiple myeloma, breast cancer, lung cancer, pancreatic cancer, and esophageal cancer. The presence of the open reading frame of MYEOV in humans and other primates suggests its potential protein-coding capacity, although direct evidence of functional MYEOV protein is currently lacking. The role of MYEOV in various tumors is not limited to its direct effects as an oncogene; it also includes its complex role in tumor cell signaling pathways and its ability to participate in miRNA regulatory networks as a competing endogenous RNA (ceRNA). Studies have shown that MYEOV may affect the expression of cancer-related genes through enhancer activity. This article aims to provide a comprehensive review of the role of MYEOV, its involvement in signaling pathways in tumor cells, and the latest advancements in MYEOV-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e799fb69a5049d179cc77bd3716c162048abf1c" target='_blank'>
              The role of MYEOV gene: a review and future directions
              </a>
            </td>
          <td>
            Yidan Xi, Xuefei Liu, Yanan Ge, Mingzhe Jiang, Dong Zhao, Ershu Zhao, Fei Cai, Xinlong Wang, Jiatong Li, Tingting Li, Zhendong Zheng
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The Gram-negative anaerobe Fusobacterium nucleatum is an oral oncobacterium that promotes colorectal cancer (CRC) development with the amyloid-forming cell surface adhesin FadA integral to CRC tumorigenesis. We describe here molecular genetic studies uncovering a novel mode of metabolic regulation of FadA-mediated tumor formation by a highly conserved respiratory enzyme known as the Rnf complex. First, we show that genetic disruption of Rnf, via rnfC deletion, significantly reduces the level of fadA transcript, accompanied by a near-complete abolishment of the precursor form of FadA (pFadA), reduced assembly of FadA at the mature cell pole, and severe defects in the osmotic stress-induced formation of FadA amyloids. We show further that the Rnf complex regulates three response regulators (CarR, ArlR, and S1), which modulate the expression of pFadA, without affecting fadA transcript. Consistent with our hypothesis that these response regulators control factors that process FadA, deletion of rnfC, carR, arlR, or s1 each impairs expression of the signal peptidase gene lepB, and FadA production is nearly abolished by CRISPR-induced depletion of lepB. Importantly, while rnfC deletion does not affect the ability of the mutant cells to adhere to CRC cells, rnfC deficiency significantly diminishes the fusobacterial invasion of CRC cells and formation of spheroid tumors in vitro. Evidently, the Rnf complex modulates the expression of the FadA adhesin and tumorigenesis through a gene regulatory network consisting of multiple response regulators, each controlling a signal peptidase that is critical for the post-translational processing of FadA and surface assembly of FadA amyloids. IMPORTANCE The Rhodobacter nitrogen-fixation (Rnf) complex of Fusobacterium nucleatum plays an important role in the pathophysiology of this oral pathobiont since genetic disruption of this conserved respiratory enzyme negatively impacts a wide range of metabolic pathways, as well as bacterial virulence in mice. Nonetheless, how Rnf deficiency weakens the virulence potential of F. nucleatum is not well understood. Here, we show that genetic disruption of the Rnf complex reduces surface assembly of adhesin FadA and FadA-mediated amyloid formation, via regulation of signal peptidase LepB by multiple response regulators. As FadA is critical in the carcinogenesis of colorectal cancer (CRC), the ability to invade CRC cells and promote spheroid tumor growth is strongly diminished in an Rnf-deficient mutant. Thus, this work uncovers a molecular linkage between the Rnf complex and LepB-regulated processing of FadA—likely via metabolic signaling—that maintains the virulence potential of this oncobacterium in various cellular niches. The Rhodobacter nitrogen-fixation (Rnf) complex of Fusobacterium nucleatum plays an important role in the pathophysiology of this oral pathobiont since genetic disruption of this conserved respiratory enzyme negatively impacts a wide range of metabolic pathways, as well as bacterial virulence in mice. Nonetheless, how Rnf deficiency weakens the virulence potential of F. nucleatum is not well understood. Here, we show that genetic disruption of the Rnf complex reduces surface assembly of adhesin FadA and FadA-mediated amyloid formation, via regulation of signal peptidase LepB by multiple response regulators. As FadA is critical in the carcinogenesis of colorectal cancer (CRC), the ability to invade CRC cells and promote spheroid tumor growth is strongly diminished in an Rnf-deficient mutant. Thus, this work uncovers a molecular linkage between the Rnf complex and LepB-regulated processing of FadA—likely via metabolic signaling—that maintains the virulence potential of this oncobacterium in various cellular niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b853997bcaf224c53b9af4dc2ba5437887be127" target='_blank'>
              Inactivation of the Fusobacterium nucleatum Rnf complex reduces FadA-mediated amyloid formation and tumor development
              </a>
            </td>
          <td>
            Timmie A. Britton, Ju Huck Lee, Chungyu Chang, A. Bhat, Yi-Wei Chen, Rusul Mohammed Ali, Chenggang Wu, A. Das, Hung Ton-That
          </td>
          <td>2025-05-22</td>
          <td>mBio</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Several landmark studies over the past decade have uncovered a critical role of the CRL3KBTBD4 ubiquitin ligase complex in regulating stability of corepressor of repressor element 1 silencing transcription factor (CoREST) complex proteins and normal hematopoietic stem cell self-renewal. There is now mounting evidence that the CoREST complex plays oncogenic roles, although the contributions of its catalytic versus noncatalytic functions remain unclear. Here, we summarize and discuss mechanisms whereby the CoREST complex coopts tissue-specific transcription factors to elicit pathogenic activity in cancer and neurodegenerative disease. We also identify tumor types with selective dependencies on the scaffolding properties of the CoREST complex. We argue that these tumor types may benefit from a KBTBD4-activating/CoREST complex degrader therapy, which could also enhance antitumor immunity and sensitize resistant tumors to immunotherapy. Overall, understanding how the CoREST complex operates abnormally and differences between its targeting through catalytic inhibitors or protein degraders will help discern all possible applications for targeting therapies now in clinical development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7a455584707201e7d88db50f466735a5ef7372" target='_blank'>
              CoREST in pieces: Dismantling the CoREST complex for cancer therapy and beyond
              </a>
            </td>
          <td>
            Houssam Ismail, J. Chagraoui, Guy Sauvageau
          </td>
          <td>2025-06-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Alterations in the homologous recombination repair genes, such as BRCA1 and BRCA2, are prevalent in various cancers, presenting a unique opportunity to develop synthetic lethal strategies that target homologous recombination deficiency (HRD). Poly ADP-ribose polymerase inhibitors (PARPis) have been developed to induce synthetic lethality in tumors with HRD by inhibiting the repair of single-strand DNA breaks. Beyond the initial approach to target cancers associated with HRD, the utility of PARPis has expanded to combination therapy with immune checkpoint inhibitors, anti-angiogenic drugs, or anti-androgen drugs based on the molecular biological rationale. In the field of genitourinary (GU) cancer, PARPis, such as olaparib, rucaparib, and talazoparib, are approved by the Food and Drug Administration in metastatic prostate cancer patients with BRCA1/2 mutations, sometimes in combination with other agents (e.g., olaparib plus abiraterone acetate, or talazoparib plus enzalutamide). More recently, pivotal clinical trials have broadened the potential of PARPis to the other GU cancers, including urothelial carcinoma and renal cell carcinoma. In this review, we examine the biomarkers for the response to PARPis beyond mutations in BRCA1/2 and discuss the current state and future perspectives of PARPis in GU cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ab480620686400ea1a47db842007f871807c33" target='_blank'>
              PARP Inhibitors in Genitourinary Cancer: A New Paradigm Beyond Prostate Cancer.
              </a>
            </td>
          <td>
            Yohei Okuda, Taigo Kato, Y. Ishizuya, Takuji Hayashi, Y. Yamamoto, K. Hatano, A. Kawashima, Junko Murai, Norio Nonomura
          </td>
          <td>2025-06-09</td>
          <td>International journal of urology : official journal of the Japanese Urological Association</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a common and aggressive malignancy. While significant advances have been made in the management of low-grade cancer, treatment of advanced HNSCC remains challenging. Here, we used a proteomic approach to find binding partners of the oncogene p63, the most frequently amplified transcription factor in HNSCC. We identified a zinc finger protein, ZNF148, which is coexpressed and physically binds p63 in carcinoma cells. Genome occupancy analyses identified a functional transcribed enhancer-derived RNA (eRNA) upstream of the CCND1 gene occupied by both factors. Mechanistically, p63 and ZNF148 control transcription of this eRNA, leading to overexpression of cyclin D1 to reinforce tumor cell proliferation. Importantly, this axis is specific for cancer cells and remains inactive in normal epithelial cells. The expression levels of these factors and the eRNA are positively correlated in cancer and associated with advanced stage and metastasis. Collectively, our data reveal a molecular pathway controlling HNSCC progression and identify potential selective targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e810ce5ca8ccc51a55dbf7814b137367f41651" target='_blank'>
              p63 and ZNF148 cooperate to regulate head and neck squamous cell carcinoma.
              </a>
            </td>
          <td>
            R. Pecorari, Mara Mancini, Artem Smirnov, Giacomo Corleone, Flavia Novelli, A. Lena, Mariacristina Franzese Canonico, M. Montanaro, A. Cappello, A. Sacconi, Gabriella Misso, Michela Falco, Chiara Tammaro, F. Ciccosanti, Mauro Piacentini, M. Scimeca, M. Caraglia, G. Blandino, A. Mauriello, Maurizio Fanciulli, G. Melino, E. Candi
          </td>
          <td>2025-07-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="While it is straightforward to understand why most mutations affecting functional sequence are harmful, how genomic changes result in new beneficial traits is harder to understand. Domestication of transposable elements (TEs) is an important source of both new genes and new regulatory systems as, for their own propagation, TEs need to have transcription factor binding sites and functional products that predispose to their recruitment. But are such predispositions to gain-of-function sufficient? Here we consider the case of the endogenous retrovirus, HERVH. Knockdown data supports HERVH having roles in pluripotency, self-renewal and defence against transpositionally-active retroelements in the early human embryo. We clarify the pluripotent cell types associated with HERVH expression and, in the process, note a key unresolved issue, framed by the unwanted transcript hypothesis: how can some cell types have 2% of their transcripts being HERVH-derived but survive the multiplicity of cellular devices that suppress foreign transcripts, be this by transcriptional repression or post-transcriptional filtering? We note a common coupling between novelty generation and suppression evasion. For example, pluripotency-associated KLF4 binding is thought to compete with transcriptional suppressor binding. Similarly, HERVH has a strong splice site enabling efficient novel chimeric transcript formation, the resulting exon–intron junctions enabling evasion of the unwanted transcript filters that recognize low or absent intron presence. We conclude that to better understand domestication, a focus on predispositions to avoidance of unwanted transcripts filters, as well as predispositions to gain of functions, is necessary. The same insights will be valuable for transgene design (eg for gene therapy) and instructive of gain-of-function in tumours, as HERVH is known to be involved in onco-exaptation events. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00361-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1580e3fbff7b45d8d3d4ce0772f34b993e7ad4c0" target='_blank'>
              Co-option of an endogenous retrovirus (LTR7-HERVH) in early human embryogenesis: becoming useful and going unnoticed
              </a>
            </td>
          <td>
            Z. Izsvák, Jin Ma, Manvendra Singh, Laurence D. Hurst
          </td>
          <td>2025-07-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b16c1cd80bd43a019c039162e9f1b0a15b5cf6d2" target='_blank'>
              Integrative Multi-Omics Analysis Identifies Nuclear Factor I as a Key Driver of Dysregulated Purine Metabolism in DIPG
              </a>
            </td>
          <td>
            I. Mersich, Sunny Congrove, Matthew Horchar, Roman Caceres, Ranjithmenon Muraleedharan, Janki M Desai, Pankaj B Desai, Larry Sallans, J. Reisz, Abby Grier, Matthew R. Hass, Omer Donmez, Cailing Yin, M. Weirauch, Leah C. Kottyan, Charles B. Stevenson, Claire X. Sun, P. D. de Blank, N. Pillay-Smiley, T. Hummel, N. Elumalai, Ali Tavassoli, Ron Firestein, T. Phoenix, A. D'alessandro, Biplab Dasgupta
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer progression is a complex, multi-stage development process characterized by dynamic changes at the molecular level. Understanding these changes may provide new insights into tumorigenesis and potential therapeutic targets. This study focuses on the evolutionary transcriptomics of cancer, specifically analyzing the Transcriptome Age Index (TAI) across different pathological stages. By examining various cancers at four distinct pathological stages, we identify a significant «hourglass» pattern in TAI indices of ductal carcinoma of the breast, bladder carcinoma, and liver carcinoma, suggesting a conserved evolutionary trajectory during tumor development. The results reveal that early and late stages of these cancers exhibit higher TAI values, indicative of more novel gene expression, while intermediate stages show a dip in TAI, reflecting a more ancient evolutionary origin of expressed genes. This «hourglass» pattern underscores the evolutionary constraints and innovations at play during tumor progression. Our findings contribute to the growing body of evidence that evolutionary principles are deeply embedded in cancer biology, offering new perspectives on the dynamics of gene expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d899ba20c349660eb4cd72c9aa668578d579a2" target='_blank'>
              Evolutionary Transcriptomics of Cancer Development
              </a>
            </td>
          <td>
            Roman Ivanov, D. Afonnikov, Yury Matushkin, S. Lashin
          </td>
          <td>2025-05-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) remains the leading cause of cancer deaths in men. The prostate-specific antigen (PSA) test is widely used for PCa screening, but it lacks specificity and can lead to over-diagnosis and over-treatment. New, effective and affordable markers are therefore needed. Results Using enzymatic methyl sequencing (EM-Seq), methylation-specific PCR (MS-PCR), and transcriptomics including a spatial approach, we analyzed tumor and non-tumor samples from radical prostatectomy specimens. Comprehensive methylome was performed in 15 paired samples of prostate cancer and their adjacent non-tumor tissue by EM-Seq. From over 4-million differentially methylated CpG sites, we identified 66 CpGs sites representing eight genes: CLDN5, GSTP1, NBEAL2, PRICKLE2, SALL3, TAMALIN/GRASP, TJP2, and TMEM106A which were hypermethylated in PCa tissues (p-value < 0.0001), and were confirmed by MS-PCR. A very good correlation between EM-Seq and MS-PCR results was observed (Pearson’s correlation of 0.93). Differential expression of these candidate genes was analyzed first, using an Affymetrix RNA array dataset from a cohort of 68 non-tumor samples and 101 tumors with different aggressiveness patterns and, second, by in situ expression using Visium 10X spatial genomics transcriptomics on eight prostate tissue sections with different tumor grades and non-tumor glands. Lower expression level was found, using RNA arrays, in tumor compared to non-tumor tissues for six of the eight genes (p ≤ 0.0001) and in tumor glands with high aggressiveness compared to non-tumor glands (p <  0.0001) for the eight genes using in situ transcriptomics. Conclusions Our study identifies promising DNA methylation markers for the diagnosis of prostate cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01930-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb9c219d2e7e435e19ce0212a2b160af0c7cac6d" target='_blank'>
              Multiple highly methylated CpG sites as potential epigenetic markers for the diagnosis of prostate cancer
              </a>
            </td>
          <td>
            J. Roperch, Guillaume Charbonnier, S. Figiel, Alastair D. Lamb, Ian Mills, Claude Hennion, G. Cancel-Tassin, Olivier Cussenot
          </td>
          <td>2025-07-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81a8c4d68f310ed7b4d9b39860d8a3a3880f7f9c" target='_blank'>
              Macroscopic Analyses of RNA-Seq Data to Reveal Chromatin Modifications in Aging and Disease
              </a>
            </td>
          <td>
            Achal Mahajan, Francesca Ratti, Ban Wang, Hana El-Samad, James H. Kaufman, Vishrawas Gopalakrishnan
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Short DNA catenanes [circular double-stranded DNA (dsDNA)] have attracted considerable interest for constructing nanostructures and nanomachines, as well as understanding DNA topology. The study of topoisomers of a circular dsDNA with a definite linking number (Lk) is essential but very difficult for simplifying the complex problems about DNA topology. The topoisomers are difficult to prepare, especially in the case that two strands are completely complementary. In this study, using a model system, we prepared all eight topoisomers (Lk0–Lk7) of a 79-bp-long circular dsDNA (8–14 nm in size) by utilizing aid-DNA to prevent undesired hybridization. By rapid ligation before strand displacement, high selectivity (>75%) for most topoisomers (31% for Lk1) was achieved under the strict topological control. All eight topoisomers with high purity were obtained after purification. Using a gel shift assay with Z-DNA-specific binding proteins, as well as by circular dichroism chromatography and enzymatic digestion, it was found that Z-DNA forms for topoisomers Lk0–Lk6, and Lk0–Lk5 can be converted to Lk6 by topoisomerase I. The approach developed in this study can significantly contribute to DNA or RNA topology, particularly the effect of topological constraints on DNA structures and functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48fcb40e10339d3441e51c8ab2560bd4c7c636" target='_blank'>
              Preparation of topoisomers of short circular dsDNA with defined linking number by accurate topological control
              </a>
            </td>
          <td>
            Mengqin Liu, Ziyi Wang, Ran An, Angda Li, Xingguo Liang
          </td>
          <td>2025-05-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Nuclear atypia is associated with increased malignancy in numerous cancers, including glioblastoma (GBM). Here, we found that GBM stem cells display small nuclear size, prompting investigation of mechanisms underlying nuclear size regulation in the tumor hierarchy.


METHODS
We performed comparative gene expression and proteomics in GBM stem cells (GSCs) and neural stem cells (NSCs) to discover potential regulators of nuclear size. Through transcriptomic analysis, mass spectrometry, and pharmacologic inhibition, we interrogated the functional significance of nuclear size regulation.


RESULTS
GSCs were enriched for a nuclear sterol reductase, Lamin B Receptor (LBR). Targeting LBR increased nuclear size and decreased GSC viability and tumor initiation. Regulation of nuclear cholesterol synthesis underlaid LBR-dependency in GSCs. Loss of LBR or reduction of cholesterol levels induced double-strand DNA breaks (DSBs), activating P53-dependent DNA damage responses (DDR). The GSC proteomic LBR interactome revealed DDR mediators, including DEAD-box RNA helicase DDX5 that resolves R-loops at DSBs. Genetic targeting of LBR reduced the DDX5-R loop interaction, leading to increased R-loop formation rescued by cholesterol supplementation. Pharmacological sterol reductase inhibition mirrored genetic LBR targeting by reducing the DDX5-R loop interaction and increasing R-loops and DSBs. Targeting LBR genetically and pharmacologically inhibits GSC growth in vivo and synergizes with irradiation.


CONCLUSIONS
Stem-like GBM cells display reduced nuclear size, driven by nuclear cholesterol synthesis to regulate radiation responses, revealing a novel therapeutic paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac67f62707eaa5f9643989ac2e6171dc64366dea" target='_blank'>
              Nuclear cholesterol regulates nuclear size and DNA damage responses in cancer stem cells.
              </a>
            </td>
          <td>
            Tingting Duan, S. Taori, S. Bhargava, Sisi Lai, Cuiqing Zhong, Shira Yomtoubian, Huairui Yuan, Xujia Wu, Po Zhang, Tengfei Huang, Donghai Wang, Fanen Yuan, Daqi Li, Huan Li, Hailong Mi, Weichi Wu, Rui Wang, Ahmed Habib, Farrukh Hammed, F. Vendetti, Pascal O. Zinn, Christopher J. Bakkenist, Marlies Meisel, Qiulian Wu, Jeremy N. Rich
          </td>
          <td>2025-06-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Previously, we found that an uncharacterized protein CXorf38 is significantly downregulated in human ZIP8-knockout (KO) cells. Given that ZIP8 regulates essential micronutrients linked to diseases including cancer, this study aims to characterize CXorf38 and evaluate its role in lung adenocarcinoma. Methods: iTRAQ-based proteomics was previously used to identify the abundance of proteins in ZIP8-KO cells. Cell proliferation and colony formation assays were used to examine the function of CXorf38 by overexpressing the gene in lung adenocarcinoma cell lines. Kaplan–Meier survival analysis was used to assess the prognostic value of CXorf38, while TCGA clinical database analysis was used to evaluate its expression in lung cancer tissues, particularly in smokers. Bioinformatics analyses (GO, KEGG, PPI, and ICI) were performed on CXorf38-coexpressed genes derived from patients with lung cancer. Results: CXorf38 overexpression suppressed lung cancer cell proliferation and colony formation, suggesting a tumor-suppressive role. Higher CXorf38 expression correlated with improved survival in patients with lung adenocarcinoma, but not in lung squamous cell carcinoma. Clinical data showed CXorf38 downregulation with lung cancer tissues of smokers, indicating a potential role in smoking-induced cancer progression and treatment. Functional analysis using bioinformatics linked CXorf38 to immune response regulation, suggesting involvement in the tumor immune microenvironment. Conclusions: Our study reveals for the first time that CXorf38 is a potential tumor suppressor, prognostic biomarker, and/or tumor immune regulator in lung adenocarcinoma—further research is warranted to explore its role in tumor immunity and its therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de769da2616c4ecb2cadcb3f426fcabdecfbd13" target='_blank'>
              Chromosome X Open Reading Frame 38 (CXorf38) Is a Tumor Suppressor and Potential Prognostic Biomarker in Lung Adenocarcinoma: The First Characterization
              </a>
            </td>
          <td>
            Rui Yan, H. Tan, Na-Li Cai, Le Yu, Yan Gao, Yan-Ming Xu, Andy T. Y. Lau
          </td>
          <td>2025-06-01</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H. Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f31d8e3aa41a4d4ed605eb1a2bff762b1dfaf76" target='_blank'>
              NUSAP1 regulates mitotic processes via KIF2C interaction and AURKA phosphorylation in primary microcephaly
              </a>
            </td>
          <td>
            Decheng Ren, Keyi Li, Zhen Liu, Yan Bi, Liangjie Liu, Lei Ji, Ke Yang, Yingying Luo, Le Luo, Yang Yan, Yang Li, Feng-ping Yang, Hua Wang, Guang He, Xiao Mao
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Women who harbor germline heterozygous mutations of BRCA1 or BRCA2 have a high risk of breast cancer. Our previous study showed that patient-derived, ostensibly normal BRCA2mut/+ luminal progenitor (LP) cells are more prone to exhibit sub-chromosomal copy number variations and associated DNA damage relative to non-carriers, potentially reflecting early breast tumorigenesis. Clinically assessable biomarkers for early pathological changes of BRCA1/2 mutation in LP cells remain unknown. Single-cell RNA sequencing (scRNAseq) of LP cells of BRCA1/2mut/+ carriers revealed enrichment of KIT expression (KIT+) and subsequent transcriptional factor activations were observed in LP cells of BRCA1/2mut/+ carriers relative to non-carriers. Moreover, pathway analysis uncovered that KIT+ BRCA-mutated LP cells were enriched in pathways involving DNA binding transcription activator activity and oxidative phosphorylation. These gene signature profiles were recapitulated in bulk RNA-seq of BRCA2 mut/+ carrier LP cells. Collectively, our preliminary data suggest such analyses may identify potential biomarkers for near-term risk prediction that reflect early pathogenesis of BRCA1/2-associated carcinogenesis. Following further validation via tissue microassay analysis and clinical studies, our findings may eventually assist patients and clinicians in decision making regarding the timing and necessity of preventive surgeries for BRCA1/2 mutation carriers.
 Citation Format: Zuen Ren, Siang Boon Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, Veerle Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W. Ellisen. Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a865906cd027bf501016cc53a811825bba9923" target='_blank'>
              Abstract P3-04-06: Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer
              </a>
            </td>
          <td>
            Zuen Ren, S. Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, V. Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W Ellisen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The study explores the role of non-coding ribonucleic acids (ncRNAs) and higher enhancer of zeste homolog 2 (EZH2) gene in breast cancer progression, examining how microRNA (miRNA) and long noncoding RNA (lncRNA) control EZH2, potentially influencing oncogene growth and treatment failure. Materials and methods The databases used in the study included Cyclebase 3.0 and CellTracer to determine EZH2’s role in cell cycle, Oncomine, OncoMX and The University of Alabama At Birmingham Cancer Data Analysis Portal (UALCAN) for Pan-cancer analysis, The Cancer Genomic Atlas Portal (TCGA Portal), Gene Expression Profiling Interactive Analysis (GEPIA2), OncoDB, CR2Cancer, Encyclopaedia of RNA Interactomes (ENCORI), and The Cancer Genome Altas Analyzer (TCGAnalyzeR v1.0) for differential expression analysis, CR2Cancer, OncoDB, MethMarkerDB, and Wanderer databases for epigenetic alteration analysis, Kaplan-Meier Plotter for survival analysis, Breast Cancer Gene Expression Miner (bc-GenExMiner v5.0) for hormone receptor analysis, Tumor-Immune System Interaction Database (TISIDB), Cancer Single Cell State Atlas (CancerSEA), TNMplot, DriverDBv4, and ENCORI for biological processes, cell cycle checkpoints and metastasis analysis, Enrichr, Tumor Immune Estimation Resource (TIMER 2.0), Gepia2 for transcription factor analysis, miRNet, Transcriptome Alterations in Cancer Omnibus (TACCO), and CancerMIRome for miRNA analysis, Enrichr, UALCAN, and ENCORI for lncRNA analysis. Results The EZH2 gene is overexpressed in breast cancer (BRCA) tumors, metastatic tissues, and circulating tumor cells, potentially leading to cancer progression. Patients with high EZH2 expression have shorter overall survival (OS), distant metastasis-free survival (DMFS), and relapse-free survival (RFS) compared to those with low expression. Estrogen receptot (ER)-negative BRCA tumors and PR-negative tumors have high EZH2 gene and eukaryotic transcription factor (E2F2) levels. The EZH2/E2F2 axis may assist ER/PR-negative BRCA by sponging homosapiens microRNA family (hsa-let-7b-5p) through lncRNA-thymopoietin antisense transcript 1 (TMPO-AS1). Overexpression of the EZH2 protein is associated with BRCA metastasis. Conclusion EZH2 overexpression in basal-like BRCA is mediated by a competing endogenous RNA (ceRNA) network and regulating their expression levels may facilitate better survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7050692d6924a95d1983e91c0ebe8eef26bc9ef7" target='_blank'>
              TMPO-AS1-hsa-let-7b-5p-EZH2-RNA network predicts poor survival in basal-like breast cancer patients
              </a>
            </td>
          <td>
            Prerna Vats, Bhavika Baweja, Sakshi Nirmal, Aditi Singh, Rajeev Nema
          </td>
          <td>2025-06-07</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Personalized immunotherapy of pancreatic ductal adenocarcinoma (PDAC) through T-cell mediated targeting of tumor-specific, mutanome-encoded neoepitopes may offer new opportunities to combat this disease, in particular by countering recurrence after primary tumor resection. However, the sensitive and accurate calling of somatic mutations in PDAC tissue samples is compromised by the low tumor cell content. Moreover, the repertoire of immunogenic neoepitopes in PDAC is limited due to the low mutational load of the majority of these tumors. Methods We developed a workflow involving the combined analysis of next-generation DNA and RNA sequencing data from matched pairs of primary tumor samples and patient-derived xenograft models towards the enhanced detection of driver mutations as well as single nucleotide variants encoding potentially immunogenic T-cell neoepitopes. Subsequently, we immunized HLA/human T-cell receptor (TCR) locus-transgenic mice with synthetic peptides representing candidate neoepitopes, and molecularly cloned the genes encoding TCRs targeting these epitopes. Result Application of our pipeline resulted in the identification of greater numbers of non-synonymous mutations encoding candidate neoepitopes with increased confidence. Furthermore, we provide proof of concept for the successful isolation of HLA-restricted TCRs from humanized mice immunized with different neoepitopes, several of which would not have been selected based on mutanome analysis of PDAC tissue samples alone. These TCRs mediate specific T-cell reactivity against the tumor cells in which the corresponding mutations were identified. Conclusion Enhanced mutanome analysis and candidate neoepitope selection increase the likelihood of identifying therapeutically relevant neoepitopes, and thereby support the optimization of personalized immunotherapy for PDAC and other poorly immunogenic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74726107b1853e699ed8135ce18a77b1befecac" target='_blank'>
              Enhanced mutanome analysis towards the induction of neoepitope-reactive T-cell responses for personalized immunotherapy of pancreatic cancer
              </a>
            </td>
          <td>
            M. Volkmar, Dana Hoser, Claudia Lauenstein, Janina Rebmann, Agnes Hotz-Wagenblatt, Jan Rieger, I. Poschke, Jonas P. Becker, Angelika B. Riemer, M. Sprick, A. Trumpp, Oliver Strobel, Thomas Blankenstein, G. Willimsky, R. Offringa
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f43529197d7ae7361f9e30b546cc7da63fdb07" target='_blank'>
              Hypomorphic Lig4 gene mutation in mice predisposes to Th1-skewing intestinal inflammation
              </a>
            </td>
          <td>
            Yusuke Yamashita, Hideki Kosako, Takashi Kato, I. Sasaki, Sadahiro Iwabuchi, Tadashi Okamura, Misato Tane, Shotaro Tabata, Kazutaka Nakashima, Ken Tanaka, Kazunori Shiraishi, Yuki Uchihara, Daisuke Okuzaki, Atsushi Shibata, Tsunehiro Mizushima, Hiroaki Hemmi, Nobuo Kanazawa, Seiji Kodama, Kouichi Ohshima, Shinichi Hashimoto, Yoshio Fujitani, T. Sonoki, S. Tamura, T. Kaisho
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ribosomal DNA transcription is essential for ribosome biogenesis and the production of proteins. Using a combination of droplet digital PCR and deep bisulfite sequencing, we have quantified both the absolute number as well as the methylation level of individual rDNA transcription units (TU) in blood samples of 139 young healthy individuals and 141 sex- and age-matched individuals with unsolved syndromal developmental delay (DD), ranging from 0.02 to 18.4 years in age. There were no between-group differences in average promoter methylation, absolute copy number (CN), extreme CN, and hypomethylated (0-10%) presumably active CN. This largely excludes rDNA CN as a modulating factor in DD. The absolute CN in all 280 samples was 423.7 ± 108.4 (median 410, range 153 to 1,000) and the active CN was 175.0 ± 36.4 (median 174, range 70 to 376). Similar to adults, the absolute CN did not change from birth to sexual maturity but was strongly (Pearson ρ = 0.64, P < 0.001) correlated with promoter methylation. In contrast to adults, there was no significant correlation between age and promoter methylation and no age-related loss of active copies from birth to < 20 years. The number of completely unmethylated copies even significantly (Pearson ρ = 0.15; P = 0.01) increased during childhood and adolescence. Our results suggest that rDNA promoter methylation and the age-related loss of active rDNA TU, which are a hallmark of the aging process, start only after reaching sexual maturity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274c60d47b8d0482945a4f0864a98745972db01c" target='_blank'>
              rDNA copy number variation and methylation from birth to sexual maturity
              </a>
            </td>
          <td>
            Alina Michler, Sarah Kießling, Jana Durackova, R. Potabattula, Asuman Koparir, Thomas Haaf
          </td>
          <td>2025-06-16</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis.We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome.Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities.Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 e15180


 Background:
 Pancreatic cancer has apoor prognosis and limited therapeutic options. While
 K-RAS
 mutations are prevalent in this malignancy, a subset of patients harboring wild-type
 K-RAS
 presents distinct clinical behavior. Gene fusions have been discerned in pancreatic cancer and are promising candidates for administering targeted therapies.
 Methods:
 We conducted a retrospective analysis of 926 pancreatic cancer patients of which 165 (18%) carried a wild-type
 K-RAS
 gene, by next generation sequencing (NGS) or polymerase chain reaction (PCR) testing. Tumors were analyzed with a clinical-grade hybrid capture-based RNA-seq clinical fusionome test, evaluating gene fusions between 1104 genes or to other known or novel partners as well as gene expression transcript analysis. Single nucleotide variants/insertions deletions (SNV/Indels) were tested with a clinical-grade NGS assay targeting > 300 cancer genes.
 Results:
 Gene fusions were detected in 32% of
 K-RAS
 wild-type pancreatic cancers, with 75% involving known oncogenic, clinically actionable genes including
 FGFR
 (two
 FGFR2
 and one
 FGFR1
 fusion), two
 NTRK3
 and one each of
 RAF1, RET, ROS1
 and
 BRAF
 fusions. Deleterious SNVs were frequently found in both fusion-positive and fusion-negative
 K-RAS
 wild-type pancreatic cancers. Transcriptomic expression revealed additional potentially druggable targets, e.g.,
 ROS1
 and
 FGFR3
 overexpression in a tumor harboring an
 NTRK3
 fusion,
 BRAF
 overexpression in the context of an otherwise non-actionable
 ETV6
 fusion, and
 MET
 overexpression in a
 TES::TFEC
 -fusion bearing cancer.
 Conclusions:
 Druggable fusions are frequent in
 K-RAS
 wild-type pancreatic cancers. Moreover, RNA fusion assays as well as transcriptomic expression testing can identify candidate personalized therapies for patients with
 K-RAS
 wild-type pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723fc53e1c67e373fda744cec67f0720b9c9b122" target='_blank'>
              The cancer fusionome and overexpressed druggable oncogenic signals in
 K-RAS
 wild-type pancreatic cancer.
              </a>
            </td>
          <td>
            Michal Krawczyk, Roisin Puentes, Nathan D Montgomery, Derek D Lyle, R. Kurzrock, F. Lopez-Diaz
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f661d74703dc0b2d864c7015c79812dfe307ea" target='_blank'>
              Decoding the Transcriptional Complexity of the Human BRCA2 DNA Repair Gene Using Hybrid-seq.
              </a>
            </td>
          <td>
            P. Adamopoulos, Michaela A. Boti, Konstantina Athanasopoulou, P. Tsiakanikas, Glykeria N. Daneva, Andreas Scorilas
          </td>
          <td>2025-07-10</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 2519


 Background:
 Tumor heterogeneity and low mutational burden limits the availability of effective immunotherapy targets. Aberrant RNA-splicing (neojunctions) represents an underexplored yet promising source of neoantigens. To address this, we developed a neoantigen discovery platform (SNIPP) that characterizes a novel class of clonally-expressed, splicing-derived neoantigens. Furthermore, we validated the immunogenicity of these neoantigens by identifying specific TCRs that drive CD8+ T-cell-mediated tumor killing.
 Methods:
 SNIPP identified public neojunctions by analyzing TCGA RNA-seq data, selecting neojunctions with a positive sample rate (PSR) > 10% and filtering out those found in GTEx normal tissue RNA-seq data (PSR < 1%) across 12 cancer types. To characterize intratumorally conserved neojunctions, we performed maximally-distanced multi-site biopsies (
 n
 = 535) within glioma patients (
 n
 = 56) and generated RNA-seq data for each intratumoral site. Two independent algorithms were utilized to predict peptide processing likelihood and HLA-binding affinity of splicing-derived neoantigen candidates. Neoantigen-specific TCR sequences were identified via
 in vitro
 sensitization of PBMCs and subsequent 10x V(D)J scRNA-seq. These TCRs were transduced into CD8+ T-cells, which were tested downstream for immunogenicity and cytotoxicity against glioma cell lines.
 Results:
 Our pipeline identified 789 public neojunctions, including 32 neojunctions concurrently detected in transcriptomic and proteomic glioma datasets and confidently predicted to be presented by HLA-A*02:01. IVS and subsequent 10x V(D)J scRNA-seq identified TCR clonotypes reactive against neojunctions in
 RPL22
 (
 n
 = 7) and
 GNAS
 (
 n
 = 1), with the latter exhibiting high intratumoral conservation (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). TCR-transduced CD8+ T-cells recognized and were immunogenically activated and demonstrated cytotoxicity against endogenously processed and presented neoantigens in GBM and melanoma lines. Additionally, IDH1-mutant oligodendrogliomas exhibited significantly higher neojunction expression compared to IDH1-mutant astrocytomas and IDH1wt subtypes. Differential gene expression analysis (DESeq2) revealed reduced expression of splicing factors in oligodendrogliomas, attributed to their specific co-deletion of chromosomes 1p and 19q. CRISPRi-mediated knockdown of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding neojunctions.
 Conclusions:
 Our study highlights a novel class of neoantigens derived from tumor-wide aberrant RNA splicing. The SNIPP platform effectively identifies public intratumorally-conserved neojunctions with strong therapeutic potential. Furthermore, elevated neojunction expression in oligodendroglioma underscores the mechanistic link between dysregulated splicing factor expression and RNA splicing abnormalities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476976fc09e028193d5d0a3658a6d5d31763d4d8" target='_blank'>
              Tumor-wide RNA splicing aberrations and their potential as therapeutic neoantigen targets.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O. Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee-may Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Aaron A. Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideho Okada
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f02ba901248eaa8a5acde4255c7a7dbd0983784" target='_blank'>
              Efficacy of DNA Methyltransferase Inhibitor Immune Priming Therapy in Combination with PD-1 Inhibitors to Treat High-Risk Pediatric Brain Tumors
              </a>
            </td>
          <td>
            Deepak Mishra, Shelli M. Morris, Dean Popovski, Emily J. Girard, Andrew Bondoc, S. Kumar, Augusto Faria Andrade, Xiaoting Zhu, Fupan Yao, M. Brusniak, Umaru Banlanjo, Erin E. Crotty, Ken Brasel, Fiona Pakiam, Caterina Russo, M. Zeinieh, Matt C. Biery, Margo Coxon, Heather Conti, Midori Clarke, Mei Lu, James T Rutka, Dhana Llivichuzhca-Loja, Liza Konnikova, M. Fouladi, Nada Jabado, Annie Huang, Jim Olson, R. Drissi
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A. Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Tripartite motif 47 (TRIM47) plays a vital role in the carcinogenesis and drug resistance of various cancers, whereas the function of TRIM47 in thyroid carcinoma (TC) remains unclear. Methods Human study and animal experiments were performed. Mass spectrometry, cellular invasion/metastasis assay, chemo-resistance assay, and ubiquitination evaluation were conducted to investigate the interaction between TRIM47 and adenosine deaminases acting on RNA (ADAR). Results TRIM47 expression was increased in human tissues and cell lines of TC. Functional experiments demonstrated that TRIM47 expression enhanced malignant biological behaviors. With mass spectrometry, TRIM47 silencing could significantly decrease the chemo-resistance of TC cells to chemotherapeutic drugs. The interaction between TRIM47 and ADAR was mediated through the ubiquitin–proteasome pathway (UPP), which was approved by RNA interference procedure and co-immunoprecipitation. Conclusion Comprehensively, glycogen synthase kinase-3β (GSK-3β)-associated ubiquitination is critical in the TRIM47-ADAR-GSK-3β axis. This study demonstrates that TRIM47 interacted with ADAR to facilitate ADAR protein degradation via ubiquitination and GSK-3β-associated phosphorylation, which serves as a novel therapeutic avenue for TC. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25e7844b5fbe0336e7b85b77d576485e48bcfefe" target='_blank'>
              Tripartite motif 47 promotes the development of thyroid carcinoma through ADAR ubiquitination
              </a>
            </td>
          <td>
            Hongzhou Liu, Xiaodong Hu, Tan Li, Yuhan Wang, Xiaomin Fu
          </td>
          <td>2025-07-05</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The IgLON family of tumor suppressor genes (TSG) impact a variety of cellular processes involved in cancer and non-cancer biology. OPCML is a member of this family and its inactivation is an important control point in oncogenesis and tumor growth. Here, we analyze RNA-Seq expression ratios in ovarian cancers from The Cancer Genome Atlas (TCGA) (189 subjects at Stage III) to identify genes that exhibit a cooperative survival impact (via Kaplan–Meier survival curves) with OPCML expression. Using enrichment analyses, we reconstruct functional pathway impacts revealing interactions of OPCML, and then validate these in independent cohorts of ovarian cancer. These results emphasize the role of OPCML’s regulation of receptor tyrosine kinase (RTK) signaling pathways (PI3K/AKT and MEK/ERK) while identifying three new potential RTK transcriptomic linkages to KIT, TEK, and ROS1 in ovarian cancer. We show that other known extracellular signaling receptor ligands are also transcriptionally linked to OPCML. Several key genes were validated in GEO datasets, including KIT and TEK. Considering the range of OPCML impacts evident in our analyses on both external membrane interactions and cytosolic signal transduction, we expand the understanding of OPCML’s broad cellular influences, demonstrating a multi-functional, pleiotropic, tumor suppressor, in keeping with prior published studies of OPCML function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95469dd0a784b5b2812df87071418bdff5bea009" target='_blank'>
              Bioinformatic RNA-Seq Functional Profiling of the Tumor Suppressor Gene OPCML in Ovarian Cancers: The Multifunctional, Pleiotropic Impacts of Having Three Ig Domains
              </a>
            </td>
          <td>
            Adam G. Marsh, Franziska Görtler, S. Hafizi, Hani Gabra
          </td>
          <td>2025-05-29</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db40ae083dc749116865ffafc0606bdc245362b3" target='_blank'>
              The penetrant chordoid glioma PRKCA mutation is an oncogenic gain-of-function kinase inactivation eliciting early onset chondrosarcoma in mice
              </a>
            </td>
          <td>
            V. Calleja, Jack C Henry, Mathias Cobbaut, Joanne Sewell, Karine Rizzoti, Francesca Houghton, Stefan Boeing, Nneka Anyanwu, Sunita Varsani-Brown, Thomas Snoeks, A. Suárez-Bonnet, S. Priestnall, N. Q. McDonald, A. Cameron, Peter J. Parker
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In recent years, the incidence of cancer has increased, and early diagnosis and treatment are key to cancer treatment. Nucleolar and spindle-associated protein 1 (NUSAP1) is a crucial mitotic regulator. NUSAP1 binds to microtubules and mediates their attachment to chromosomes, ensuring the accurate distribution of genetic material to the two daughter cells. NUSAP1 is located on chromosome 15q15.1 and is typically expressed in proliferating cells, suggesting a potential oncogenic role. Studies have shown that NUSAP1 plays a significant role in various cancers, including breast cancer, cervical cancer, lung cancer, prostate cancer, and hepatocellular carcinoma. Furthermore, NUSAP1 may serve as a promising biomarker for cancer diagnosis and prognosis. In this review, we discuss the roles and mechanisms of NUSAP1 in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17ed1053e2b63b1afa28756e4052dc1859c3cc0" target='_blank'>
              The role and therapeutic value of NUSAP1 in human cancers
              </a>
            </td>
          <td>
            Shenyuan Xu, Ding Nan, Ruiqi Liu, Chenting Liu, Yuefeng Hu, Luanluan Huang, Xiaoyan Chen, Hailong Sheng, Yanwei Lu, Haibo Zhang
          </td>
          <td>2025-07-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Radiotherapy (RT), a cornerstone of cancer treatment, exerts its therapeutic effects primarily by inducing DNA damage in tumor cells and modulating the tumor immune microenvironment (TIME). Despite its efficacy, RT is often counteracted by tumor-intrinsic mechanisms, such as DNA damage repair, as well as immune-suppressive responses. YTHDF proteins, key N6-methyladenosine (m6A) readers, have emerged as pivotal regulators of tumor progression, DNA repair, and immune cell function, making them promising targets for enhancing RT efficacy. In this review, we explore the dual roles of YTHDF proteins in modulating both tumor-intrinsic and immune-mediated responses to RT. We summarize their influence on DNA damage repair pathways in tumor cells and their impact on the TIME, which collectively shape the antitumor efficacy of RT. Furthermore, we discuss recent advances in the development of YTHDF-targeting inhibitors and their potential to synergize with RT and immunotherapy, offering new avenues to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff5dd1a3e9e346144ede7ce1c93e024a38d8fd6" target='_blank'>
              YTHDFs as radiotherapy checkpoints in tumor immunity.
              </a>
            </td>
          <td>
            Chuangyu Wen, Emile Z Naccasha, Chuan He, H. Liang, R. Weichselbaum
          </td>
          <td>2025-06-05</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90517c7621a56cfd44f51891279305813cf3e121" target='_blank'>
              Notch3 regulates pericyte phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            N. Chalkidi, Athanasia Stavropoulou, V. Arvaniti, Christina Paraskeva, Artemis Monogyiou, Maria Sakkou, Christoforos Nikolaou, V. Koliaraki
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Genomic imprinting is the parent-of-origin specific monoallelic expression of genes that result from complex epigenetic interactions. It is often achieved by monoallelic 5-methylcytosine, resulting in the formation of differentially methylated regions (DMRs). These show a bias towards oocyte-derived methylation and survive reprogramming in the pre-implantation embryo. Imprinting is widespread in the human placenta. We have recently performed whole-genome screens for novel imprinted placenta-specific germline DMRs (gDMRs) by comparing methylomes of gametes, blastocysts and various somatic tissues, including placenta. We observe that, unlike conventional imprinting, for which methylation at gDMRs is observed in all tissues, placenta-specific imprinting is associated with transient gDMRs, present only in the pre-implantation embryo and extra-embryonic lineages. To expand the list of bona fide imprinted genes subject to placenta-specific imprinting, we reinvestigated our list of candidate loci and characterized two novel imprinted genes, PIK3R1 and G0S2, both of which display polymorphic imprinting. Interrogation of placenta single-cell RNA-seq datasets, as well as cell-type methylation profiles, revealed complex cell-type specificity. We further interrogated their methylation and expression in placental samples from complicated pregnancies, but failed to identify differences between intrauterine growth restricted or pre-eclamptic samples and controls, suggesting they are not involved in these conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b750b53f10506274107b5e1972c3a2fd8cd37694" target='_blank'>
              PIK3R1 and G0S2 are human placenta-specific imprinted genes associated with germline-inherited maternal DNA methylation
              </a>
            </td>
          <td>
            Dagne Daskeviciute, Becky Sainty, Louise Chappell-Maor, Caitlin Bone, Sarah Russell, Isabel Iglesias-Platas, Philippe Arnaud, Ana Monteagudo-Sánchez, Maxim V. C. Greenberg, Keran Chen, Africa Manerao-Azua, G. Perez de Nanclares, Jon Lartey, David Monk
          </td>
          <td>2025-06-26</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bcb6bbbf4e6c64ab8d5a6fa755b5306451c07b" target='_blank'>
              EHMT1 mediates cellular motility in embryonal rhabdomyosarcoma by activating SOX8 expression.
              </a>
            </td>
          <td>
            Upasana Bajaj, Dipanwita Das, Jia Yu Leung, Nurul Fateha Binti Abdul Rashi, R. Taneja
          </td>
          <td>2025-06-04</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The kinesin family member 18 A (KIF18A) is an essential regulator of microtubule dynamics and chromosome alignment during mitosis. Functional dependency on KIF18A varies by cell type and genetic context but the heritable factors that influence this dependency remain unknown. To address this, we took advantage of the variable penetrance observed in different mouse strain backgrounds to screen for loci that modulate germ cell depletion in the absence of KIF18A. We found a significant association at a Chr5 locus where anaphase promoting complex subunits 5 (Anapc5) and 7 (Anapc7) were the top candidate genes. We found that both genes were differentially expressed in a sensitive strain background when compared to resistant strain background at key timepoints in gonadal development. We also identified a novel retroviral insertion in Anapc7 that may in part explain the observed expression differences. In cell line models, we found that depletion of KIF18A induced mitotic arrest, which was partially rescued by co-depletion of ANAPC7 (APC7) and exacerbated by co-depletion of ANAPC5 (APC5). These findings suggest that differential expression and activity of Anapc5 and Anapc7 may influence sensitivity to KIF18A depletion in germ cells and CIN cells, with potential implications for optimizing antineoplastic therapies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08766-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/965c0dbbb6128d27bea8370bf402a688e4926421" target='_blank'>
              Anapc5 and Anapc7 as genetic modifiers of KIF18A function in fertility and mitotic progression
              </a>
            </td>
          <td>
            Carleigh Nesbit, Whitney Martin, A. Czechanski, Candice Byers, Narayanan Raghupathy, Ardian Ferraj, Jason K. Stumpff, L. Reinholdt
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cell cycle structures vary significantly across cell types, which exhibit distinct phase compositions. Asynchronous DNA replication and dynamic cellular characteristics during the cell cycle result in considerable heterogeneity in DNA dosage, chromatin accessibility, methylation, and expression. Nonetheless, the consequences of cell cycle disruption in the interpretation of multi-omics data remain unclear. Here, we systematically assessed the influence of distinct cell phase structures on the interpretation of omics features in proliferating cells, and proposed solutions for each omics dataset. For copy number variation (CNV) calling, asynchronous replication timing (RT) interference induces false CNVs in cells with high S-phase ratio (SPR), which are significantly decreased following replication timing domain (RTD) correction. Similar noise is observed in the chromatin accessibility data. Moreover, for DNA methylation and transcriptomic analyses, cell cycle-sorted data outperformed direct comparison in elucidating the biological features of compared cells. Additionally, we established an integrated pipeline to identify differentially expressed genes (DEGs) after cell cycle phasing. Consequently, our study demonstrated extensive cell-cycle heterogeneity, warranting consideration in future studies involving cells with diverse cell-cycle structures. RTD correction or phase-specific comparison could reduce the influence of cell cycle composition on the analysis of the differences observed between stem and differentiated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7f0e00dfe690501686ff54d8b38db35d315d13" target='_blank'>
              Mitigating Cell Cycle Effects in Multi-Omics Data: Solutions and Analytical Frameworks.
              </a>
            </td>
          <td>
            Rui Nie, Caihong Zheng, Likun Ren, Yue Teng, Yaoyu Sun, Lifei Wang, Junya Li, Jun Cai
          </td>
          <td>2025-05-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is an aggressive cancer of bone and soft tissue that predominantly affects children and young adults. A chromosomal translocation joins the low-complexity domain (LCD) of the RNA-binding protein EWS (EWSLCD) with the DNA-binding domain of Friend leukemia integration 1 (FLI1DBD), creating EWS::FLI1, a potent fusion oncoprotein essential for EwS development and responsible for over 85% of EwS tumors. EWS::FLI1 forms biomolecular condensates in vivo and promotes tumorigenesis through mediation of aberrant transcriptional changes and by interfering with the normal functions of nucleic acid-binding proteins like EWS through a dominant-negative mechanism. In particular, the expression of EWS::FLI1 in EwS directly interferes with the biological functions of EWS leading to alternate splicing events and defects in DNA-damage repair pathways. Though the EWSLCD is capable of phase separation, here we report a direct interaction between FLI1DBD and EWSLCD that enhances condensate formation and alters the physical properties of the condensate. This effect was conserved for three related E-twenty-six transformation-specific (ETS) DNA-binding domains (DBDs) while DNA binding blocked the interaction with EWSLCD and inhibited EWS::FLI1 condensate formation. NMR spectroscopy and mutagenesis studies confirmed that ETS DBDs transiently interact with EWSLCD via the ETS DBDs "wings." Together these results revealed that ETS DBDs, particularly FLI1DBD, enhance EWSLCD condensate formation and rigidity, supporting a model in which electrostatic and structural interactions drive condensate dynamics with implications for EWS::FLI1-mediated transcriptional regulation in EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90380493c2e0c6b1757985f2684e59828098025" target='_blank'>
              Phase separation of the oncogenic fusion protein EWS::FLI1 is modulated by its DNA-binding domain.
              </a>
            </td>
          <td>
            Emily E. Selig, Erich J Sohn, Aiola P. Stoja, Alma K. Moreno-Romero, Shivani Akula, Xiaoping Xu, Alexander J R Bishop, David S. Libich
          </td>
          <td>2025-05-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Merkel Cell Polyomavirus (MCPyV) is recognized as the major aetiological agent of Merkel Cell Carcinoma (MCC), an aggressive skin tumor. MCPyV-mediated oncogenesis is strictly dependent on viral integration and the expression of a truncated form of the Large T Antigen (LT). Moreover, like other oncogenic DNA viruses, MCPyV may interfere with the DNA damage response (DDR) machinery, thus promoting genomic instability and tumorigenesis. Therefore, the objective of this study was to characterize MCPyV infection in 7 MCC patients and to elucidate the plausible role of the virus in the DDR pathway. MCPyV DNA was detected in 3/7 MCC patients and, as expected, viral integration and LT truncation were observed in virus-positive MCCs, along with the expression of early genes only. Over-expression of DDR genes such as ATM, ATR and their downstream kinases Chk1 and Chk2 was reported in MCPyV-positive MCCs supporting the potential role of the virus in interfering with DDR. Our findings support the established viral aetiology of MCC, and describe, for the first time, an over-expression of DDR components in MCPyV-positive MCC, laying the basis for future studies aimed at investigating the contribution of this pathway to MCPyV-mediated carcinogenesis and exploring the plausible clinical implications of host DDR factors for the treatment of MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cee01380ef93ba0651f961c5653ef76df058128" target='_blank'>
              Investigation of mRNA expression levels of DNA damage response genes in Merkel Cell Polyomavirus-positive Merkel Cell Carcinoma: a pilot study
              </a>
            </td>
          <td>
            S. Passerini, M. Fracella, A. Ferlosio, U. Moens, C. Scagnolari, Guido Antonelli, Marco Ciotti, V. Pietropaolo
          </td>
          <td>2025-05-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, Emanuela Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) greatly contribute to the growth, invasion, metastasis and drug resistance of neuroblastoma (NB). This study aimed to construct a CAF-related prognostic model and identify the immune status of patients with NB via single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq). Methods ScRNA-seq data of NB acquired from the Gene Expression Omnibus (GEO) database were used to identify cellular subpopulations. Bulk gene expression data were downloaded from GEO, The Cancer Genome Atlas (TCGA) and ArrayExpress databases and the prognostic model was constructed using univariate Cox and Least Absolute Shrinkage and Selection Operator (LASSO) analyses. Differences in immune infiltration, therapeutic responses and signaling pathways between the high- and low-risk groups were investigated. Finally, immunohistochemistry was performed to evaluate the protein expressions and a nomogram based on the risk signature and clinical characteristics was constructed. Results ScRNA-seq data of eight NB samples were integrated to identify 253 marker genes for CAF. An eight-gene prognostic CAF-related signature was established based on the GEO data. The CAF model was strongly associated with immune infiltration, drug response and active signaling pathways in tumors. Univariate and multivariate Cox regression analyses verified that the CAF model was as an independent prognostic indicator, and a nomogram integrating the clinical signature and CAF-related risk signature was constructed for clinical prediction. Conclusions The CAF-related signature can effectively predict the prognosis of NB and provide new genomic evidence for anti-CAF immunotherapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03023-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0c7a00465eed83a4f94e4167616471e1046f423" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis establishes a cancer associated fibroblast-related signature for predicting prognosis and therapeutic responses in neuroblastoma
              </a>
            </td>
          <td>
            Zhiyao Cao, Qi Wang, Yali Han, Jianwei Lin, Qi Wu, Chencheng Xu, Jingchun Lv, Lei Zhang, Hongxiang Gao, Dapeng Jiang
          </td>
          <td>2025-07-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aim: Chemotherapy resistance, often linked to the development of resistance against genotoxic agents, is a major obstacle in cancer treatment. FoxM1, a transcription factor frequently overexpressed in malignancies such as breast cancer, is strongly associated with genotoxic therapy resistance. The aim of this study is to conduct a comparative analysis of the effects of thiostrepton (THIO), a FoxM1 inhibitor, on the DNA damage response in HUVEC cells (non-malignant) and MDA-MB-231 breast cancer cells (malignant) 
Materials and Methods: THIO's impact on cell viability were evaluated in both cell lines using the MTT assay. Oxidative DNA damage levels were measured with the 8-OHdG kit, and apoptosis was assessed using the Caspase 3 ELISA kit. The expression levels of DNA damage response genes (BRCA-1, DNAPKC, FOXM1, RAD51, MRE11 and XRCC1) were analyzed by RT-PCR. 
Results: MDA-MB-231 cells exhibited greater sensitivity to the cytotoxic effects of THIO than HUVEC cells. In HUVEC cells, THIO caused a significant increase in oxidative DNA damage, whereas no such effect was observed in MDA-MB-231 cell lines. Conversely, breast cancer cells showed a significant increase in Caspase 3 levels. RT-PCR results revealed a marked downregulation of DNA damage response genes, particularly BRCA-1, DNAPKC, MRE11, FOXM1, and XRCC1, in both cell types. 
Conclusion: THIO downregulates DNA damage response genes by inhibiting FoxM1 expression in both malignant and non-malignant cells. This effect suggests that THIO may enhance the sensitivity of breast cancer cells to therapy by disrupting DNA repair pathways, presenting potential for overcoming chemotherapy resistance. Further studies are required to validate these findings and explore THIO's therapeutic utility in breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0302ae4a15025cfb10555437aa0808dee744207" target='_blank'>
              FoxM1 inhibition by Thiostrepton downregulates DNA damage response genes, enhancing sensitivity of breast cancer cells to therapy
              </a>
            </td>
          <td>
            F. Demırtaş Korkmaz, Zekeriya Düzgün, Fadime Mutlu İçduygu
          </td>
          <td>2025-06-10</td>
          <td>Ege Tıp Dergisi</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ad49026046d30dd6909e4b9cd5b26991a13bef" target='_blank'>
              Tumor aneuploidy as a prognostic and predictive biomarker in immune checkpoint blockade.
              </a>
            </td>
          <td>
            Daniel Huang, L. F. Spurr, R. Weichselbaum, S. Pitroda
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cholangiocarcinoma is a highly heterogeneous malignant tumor, including intrahepatic cholangiocarcinoma, hepatoportal cholangiocarcinoma and distal cholangiocarcinoma. Its incidence is increasing worldwide and currently accounts for approximately 15% of all primary liver cancers and 3% of all gastrointestinal malignancies. There is a lack of early diagnostic methods for cholangiocarcinoma, and the overall treatment effect is poor, with a 5-year survival rate of less than 25%. New biomarkers are urgently needed in clinical practice to improve the current diagnosis and treatment status. Circulating tumor DNA (ctDNA) is DNA fragments released by tumor cells, which can show tumor-specific gene mutations (such as IDH1/2, FGFR2 fusion) and epigenetic modifications (such as abnormal methylation). With the rapid development of tumor liquid biopsy technology, ctDNA has been gradually applied in solid tumors such as lung cancer and colorectal cancer due to its high sensitivity and dynamic monitoring capabilities. This review systematically introduces ctDNA technology and its progress in early screening, early diagnosis, treatment response, and prognosis monitoring of cholangiocarcinoma. In addition, this review also summarizes the challenges and limitations of current ctDNA technology and analyzes future hot research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/582e6b4bb69a2b394d91aff713a3ea41c9e46004" target='_blank'>
              Circulating tumor DNA in cholangiocarcinoma: current clinical applications and future perspectives
              </a>
            </td>
          <td>
            Yi Wang, Yike Li, Zhong Liang, Yuqiao Zhang, Tong Li, Chenjun Tian, Jinyu Zhao, Boru Jin, Jie Cao, Yanyan Lin
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Loss of the Y chromosome (LOY) in peripheral blood mononuclear cells (PBMCs) is the most common somatic alteration in men and is associated with higher mortality from epithelial cancers1–3. In tumours, epithelial LOY is also associated with poor survival4–7. This raises several fundamental questions, such as why LOY in PBMCs drives cancer mortality and whether there is a relationship between LOY in PBMCs, PBMC-derived immune cells and cancer cells (and, if so, what its consequences are). We sought to answer these questions through a comprehensive pan-cancer analysis of bulk and single-cell RNA sequencing data from 29 human tumour types, along with autochthonous and syngeneic mouse models. In human and mouse tumours, malignant epithelial cells had the highest LOY prevalence, yet LOY was also present in tumour stromal and immune cells, with LOY in malignant epithelial cells predicting LOY in benign cells. LOY also correlated between paired tumour and PBMC samples from patients. Among benign cells, LOY induced the strongest shift in CD4+ and CD8+ T cells, with both showing transcriptomic signatures of immunosuppression. Furthermore, the magnitude of LOY in epithelial cells, CD4+ T cells and CD8+ T cells independently predicts survival, with tumours exhibiting concurrent epithelial and T cell LOY having the worst outcomes. Here we establish a model that links LOY in immune cells to LOY in malignant cells, which may explain in part why LOY in PBMCs is associated with increased cancer mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/521330e6a0acc4c997107eda98c17326ccf4fe70" target='_blank'>
              Concurrent loss of the Y chromosome in cancer and T cells impacts outcome
              </a>
            </td>
          <td>
            Xingyu Chen, Yiling Shen, Suhyeon Choi, Hany A Abdel-Hafiz, Mukta Basu, Lena Hoelzen, Martina Tufano, Saravana Kumar Kailasam Mani, Maryam Ranjpour, Jiani Zhu, V. K. Ramanujan, E. Koltsova, Vinicius F Calsavara, S. Knott, Dan Theodorescu
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e86dbb7bd01d6e90b441b62dcc070c2bc321518" target='_blank'>
              High-fidelity long-read sequencing reveals a complex RCCX locus at the single-nucleotide level in Korean patients with congenital adrenal hyperplasia
              </a>
            </td>
          <td>
            , , D. Kim, S. Park, G.-H. Kim,, , J.-H. Choi,, J. Kim, J. Kim
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 What is the prevalence of mitotic-origin aneuploidies in human blastocysts using a PGT platform capable of identifying cell division origin of aneuploidy? A single-center experience



 Although uncommon, some embryos are diagnosed with aneuploidies of mitotic origin only. Diagnosing aneuploidies without determining their cellular origin risks discarding embryos with potential viability.



 Chromosomal abnormalities in human embryos are a significant factor contributing to reproductive failure. Next-generation sequencing (NGS) platforms are widely accepted for evaluating chromosomal abnormalities, as they are copy number-focused technologies. However, in some cases, biopsies reveal a mix of normal and abnormal cells, referred to as mosaic embryos. The developmental potential of these mosaic embryos remains unclear, although successful outcomes, including normal live births, have been reported. Moreover, novel PGT approaches, such as high-density single nucleotide polymorphism (SNP) arrays, offer the capability to predict the cell division origin of aneuploidies, moving beyond a sole focus on chromosome copy numbers.



 This retrospective study analyzed 310 consecutive PGT-A biopsies obtained from 72 patients across 91 treatment cycles conducted between May 2023 and December 2024 at a private fertility clinic. Both autologous and donor cycles were included, and all blastocysts were evaluated using trophectoderm (TE) biopsy and DNA samples from the biological parents.



 Trophectoderm biopsies were obtained, with 5–10 cells collected for each sample. Parental DNA samples were also required to serve as reference material for the tested embryos. Trophectoderm biopsies and parental DNA were processed using SNP arrays to identify the cell division origin of aneuploidy, integrating both genotyping and copy-number data. Aneuploidies were classified by origin as maternal meiotic (from the oocyte), paternal meiotic (from the sperm), or mitotic (embryonic, arising post-zygotic division).



 Among the total embryos studied (n = 310), 44.8% (139/310) were identified as aneuploid. Of these, 34.8% (108/310) exhibited at least one maternal meiotic aneuploidy, with an average maternal age of 36.3 ± 4.4 years, consistent with the well-established correlation between advancing maternal age and chromosomal anomalies. Additionally, 7.7% (24/310) of the aneuploid embryos displayed paternal meiotic errors, while 8.7% (27/310) presented mitotic errors originating post-fertilization. Notably, 5.16% (16/310) of the embryos displayed exclusively mitotic-origin aneuploidies, underscoring the importance of mitotic anomalies in early embryonic development. Among the 139 abnormal embryos, 81.3% (113/139) exhibited whole chromosome aneuploidies, consistent with meiotic segregation errors involving entire chromosomes. Conversely, 13.67% (19/139) of the abnormal embryos exhibited segmental aneuploidies, involving duplications or deletions of specific chromosomal regions. A small subset of embryos, accounting for 5.04% (7/139), displayed karyotypes that included both whole chromosome aneuploidies and segmental abnormalities, reflecting the complexity of chromosomal errors in human embryos.



 This is a retrospective study with a relatively small number of embryos analyzed. Expanding the cohort of blastocysts and evaluating the clinical implications of aneuploidy origin prediction are necessary. Additionally, further research is needed to assess the reproductive potential of embryos with segmental chromosome aneuploidies.



 Identifying the cell division origin of aneuploidies provides deeper insights into embryonic chromosomal abnormalities and their developmental potential. This approach could improve clinical decision-making in assisted reproduction, enhancing embryo selection strategies and informing future studies on the implications of mitotic and segmental aneuploidies in human reproduction.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/839a948ebef49cd1b00902c78815394f99d075d1" target='_blank'>
              P-242 From Cell Division to Clinical Decisions: Unveiling the True Prevalence of Mitotic-Origin Aneuploidies in Preimplantation Embryos
              </a>
            </td>
          <td>
            M. J. Gomez Cuesta, D. Marin Chavarria, M. I. Gutierrez Estrada, M. Garcia Galicia, G. D. Cortes Ruiz, M. Alonso de Mendieta, C. Kandel Varsano, C. Lopez Valencia, E. Cervantes Bravo
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Objective Epstein-Barr virus nuclear antigen 1 (EBNA1) is a key viral protein expressed in all latency phases and EBV-associated tumors. It can modulate the expression of various host and viral genes. This study aimed to investigate the impact of EBNA1 on the expression levels of two cellular genes involved in p53 pathway regulation—MDM2 and MDM4—in HeLa cells. This investigation was conducted as part of our broader research on EBV-related oncogenic mechanisms. Results HeLa cells were transfected with either an EBNA1-expressing plasmid or a control plasmid. Gene expression levels of MDM2 and MDM4 were analyzed using real-time PCR. The results demonstrated a statistically significant increase in MDM4 expression in EBNA1-transfected cells compared to controls (p = 0.028). Although MDM2 expression was also elevated, the difference was not statistically significant (p = 0.11). These findings suggest that EBNA1 may play a role in cervical cancer development by upregulating genes that inhibit p53 tumor suppressor activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eedbb8118d780d80afe4e5adaa5555f34e322dec" target='_blank'>
              Epstein-Barr virus nuclear antigen 1 (EBNA1) increases the expression levels of MDM2 and MDM4 genes in HeLa cells: a review on MDM2 and MDM4 roles in cancer
              </a>
            </td>
          <td>
            Seyed Mohammad Ali Hashemi, Amir Hossein Alipour, Sima Emamifar, Ali Farhadi, J. Sarvari
          </td>
          <td>2025-05-17</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 3627


 Background:
 A subset of colorectal cancer (CRC) features amplification (amp) of the ERBB2 gene. The prognostic role and treatment in these patients are poorly understood. This study aims to characterize the clinical outcomes and molecular profiles of ERBB2-amp CRC.
 Methods:
 This is a retrospective analysis of clinical outcomes data and next-generation sequencing at MD Anderson from 2006-2025 in patients with ERBB2-amp CRC including all classes of genomic alterations (GA) in other genes. Progression-free survival (PFS) and overall survival (OS) were assessed by Kaplan-Meier, log-rank, and cox regression. Chi square goodness of fit was assessed for the proportion of left versus right sided CRC.
 Results:
 100 pts with ERBB2-amp CRC were identified. Patients (Table) who received ERBB2-directed therapy (n = 36) did not have significantly longer OS compared to those who received other systemic therapy (n = 57) (53.8 vs. 45.5 m, p = 0.35). Patients who underwent surgery (n = 64) (56.8 vs. 23.2 m, p = 0.01) or non-surgical localized therapy (n = 45) had longer OS (64.4 vs 45.5 m, p = 0.01). ERBB2-amp was more associated with left sided CRC (n = 80) than right-sided CRC (n = 19) (X
 2
 = 39.34, df = 1, p < 0.0001) but there was no significant OS difference (53.8 vs. 45.5 m, p = 0.76). Median PFS (mPFS) for all 1st (n = 93), 2nd (n = 80), and 3rd line (n = 61) systemic therapy was 8.0, 5.1, and 4.9 m, respectively. mPFS for all ERBB-2 directed therapy (n = 59) was 3.5 m. The most common concurrent co-GA were TP53 (90%) and APC (64%) and most common co-amp were CDK12 (31%) and EGFR (23%). Concurrent APC mutation was associated with improved OS (53.8 vs. 26.2 m, p = 0.003). No other co-GA or co-amp, including KRAS, showed significant survival outcome associations. TP53 co-GA was present in 90% of both left and right-sided ERBB-2 CRC and KRAS co-GA in 15% and 32%, respectively.
 Conclusions:
 There was no difference in OS in ERBB2 amp CRC in patients who received ERBB2 directed therapy, however patients had improved OS with surgery or localized therapy options and should be offered to eligible patients if possible. Co-GA with APC mutation correlated with improvement in survival outcomes.


 Baseline data, initial staging, treatments, and mutational profile for ERBB2 co-amp CRC.




 Age at Diagnosis (median)
 55 years




 Gender – no.
 M (56), F (44)


 Stage at Initial Diagnosis – no.
 I (6), II (6), III (18), IV (69)


 Location of Primary Tumor – no.
 R-sided (19), L-sided (80), Cecum (7), Ascending (8), Transverse (5), Descending (7), Sigmoid/Rectum (73)


 Surgery – no.
 Primary tumor (55), Liver metastectomy (24), Lung metastectomy (8)


 Localized Therapy – no.
 RT (36), Ablation (6), Y90 (4), Cryotherapy (1)


 ERBB2 Directed Therapy (excluded if n=1) – no.
 Trastuzumab (TRA) + Pertuzumab (20), ERBB2-directed trial drug (14), Trastuzumab deruxtecan (8), TRA (7), TRA + Tucatinib (4)


 Mutational Profile
 Co-GA: TP53 (90%), APC (64%), SMAD4 (18%), KRAS (18%)


 Co-amp: CDK12 (31%), EGFR (23%), MYC (19%), BRAF (13%)




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d0f42d0d349a777ee7b1fb85c5f8c10e3a8f7ef" target='_blank'>
              Characterizing clinical outcomes and DNA co-alterations of ERBB2-amplified colorectal cancer.
              </a>
            </td>
          <td>
            Mohamed Nuh, Monica Hsiang, Sunyoung S. Lee
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Telomere maintenance mechanisms including MYCN amplification (MYCN-A) (via upregulated telomerase reverse transcriptase [TERT] expression), TERT rearrangements (TERT-RA), and ATRX mutations confer neoplastic immortality in neuroblastoma (NB) cells. Clinical characterization of patients with NB harboring activating somatic TERT promoter point mutations (TERT-PM) in NB may improve stratification.


METHODS
To identify TERT-PM and TERT-RA, tumors were profiled by the Memorial Sloan Kettering Cancer Center-Integrated Mutation Profiling of Actionable Cancer Targets targeted next-generation sequencing platform and whole-genome sequencing, respectively. Associated clinical outcomes were studied.


RESULTS
TERT-PM and TERT-RA were detected in 17 of 603 (2.8%) and 31of 168 (18.4%) tumors from individual patients, respectively. The median age at diagnosis was 32 (range, 15-79) and 48 (range, 3-168) months for TERT-PM and TERT-RA, respectively. TERT-PM were located at canonical hotspots (C228T [16/17] and C250T [1/17]) and were concurrent with MYCN-A in 9 of 17 (53%). By contrast, MYCN-A occurred in 1 of 31 TERT-RA tumors. Most patients with TERT-PM had stage M (94%) and high-risk NB (HR-NB) (81%) at diagnosis. Twenty-eight patients with TERT-RA had HR-NB (90%). After risk appropriate therapy, complete response was achieved in 7 of 17 (41%) and 17 of 31 (55%) patients with TERT-PM and TERT-RA, respectively. The median progression-free (PFS) and overall survival (OS) were 9.7 ± 1.2 and 16.5 ± 2 months for patients with TERT-PM. Corresponding PFS/OS for patients with TERT-RA were 20 ± 6.4/56.8 ± 8.6 months (P = .015 for OS). Excluding MYCN-A status produced no significant survival difference (P > .1) between the two groups. CNS relapse occurred in 11 of 31 (35%) patients with TERT-RA versus 1 of 17 (6%) patients with TERT-PM.


CONCLUSION
TERT-PM is rarer than TERT-RA but both are associated with HR-NB and poor prognosis. MYCN-A frequently co-occurs with TERT-PM and might represent an ultra-high-risk subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/556a0902c3148a7408804dc2c0f9318f277250b6" target='_blank'>
              Clinical Significance of TERT Promoter Mutations in Neuroblastoma.
              </a>
            </td>
          <td>
            Natasha Persaud, G. Gundem, B. Kushner, Marc Ladanyi, Neerav Shukla, S. Modak
          </td>
          <td>2025-07-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract HELB is a human helicase involved in initiation of DNA replication, the replication stress response, and regulation of double-strand DNA break repair. rs75770066 is a low-frequency single-nucleotide polymorphism (SNP) in the HELB gene that affects age at natural menopause (ANM). rs75770066 results in a D506G substitution in a HELB-specific motif in the 1A domain of the helicase that contains amino acids known to interact with RPA. We found that this amino acid change has no effect on the enzymatic activity of HELB on naked DNA substrates but reduces the rate of unwinding by HELB on RPA coated substrates, likely because D506G substitution in HELB reduces interaction with RPA. This impaired interaction of D506G HELB with RPA dramatically impairs the cellular function of HELB and likely results in the effects of rs75770066 as this reduces recruitment of HELB to sites of DNA damage. Reduced recruitment of D506G–HELB to double-strand DNA breaks and the concomitant increase in homologous recombination likely alters the levels of meiotic recombination, which affects the viability of gametes. Because menopause occurs when oocyte levels drop below a minimum threshold, altered repair of meiotic double-stranded DNA breaks has the potential to directly affect the ANM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/725c12f189337a8b7320ea03b087656f731a6b6a" target='_blank'>
              Rare SNP in the HELB gene interferes with RPA interaction and cellular function of HELB
              </a>
            </td>
          <td>
            Bertha Osei, Benjamin H. May, Joseph S Beard, Matthew D Thompson, Duah Alkam, M. Zafar, Erik Bergstrom, Stephanie D. Byrum, Eric J Enemark, Kirk L. West, Alicia K. Byrd
          </td>
          <td>2025-04-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) have a unique molecular profile compared to extracranial metastases (ECM). Description of the biological features and clinical outcomes of MBM will facilitate the design of rational therapies. Methods We examined the mutational landscape and gene expression profiles of MBM (74 patients) and ECM (34 patients) in paired patient samples from a previously published dataset with whole-exome sequencing (WES) and RNA sequencing (RNAseq) data from MD Anderson Cancer Center (MDACC). We also present findings from MBM from a new cohort of 14 patients from Duke University to strengthen investigation of somatic mutations and gene expression profiles. Gene Set Enrichment Analysis (GSEA) was used to compare paired MBM versus lymph node (LN) metastases and skin metastases. Relative immune cell abundance was inferred using deconvolution methods. Survival outcomes from craniotomy and associations with biological features, BRAF mutation status, and PTEN expression were assessed. Results GSEA found that autophagy signaling pathways are enriched in MBM versus LN and skin metastases. BRAF was the most frequently mutated clinically relevant gene in MBM and ECM, with NRAS and PTEN also frequently altered in MBM. The most strongly upregulated genes in autophagy pathways were glial fibrillary acidic protein (GFAP) and hemoglobin beta (HBB). An increased proportion of immune-suppressive M2 compared to tumor-suppressive M1 macrophages in MBM and ECM was identified. There was not sufficient evidence for an association between BRAF V600 mutation status or expression and overall survival (OS) from craniotomy. Conclusions The mutational landscape and gene expression of MBM from the Duke cohort resembled those previously reported in the MDACC cohort. Upregulation of autophagy pathways was observed in patient-matched MBM versus LN and skin metastases due to upregulation of two genes, GFAP and HBB. In MBM, higher M2:M1 ratio may contribute to a therapeutically relevant immune-suppressive tumor microenvironment (TME). Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02035-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2acbbd5230dc401bd3ec3c002ad4d3ee572431" target='_blank'>
              A comprehensive, multi-center, immunogenomic analysis of melanoma brain metastases
              </a>
            </td>
          <td>
            L. B. Kennedy, Amanda E. D. Van Swearingen, Marissa R Lee, Layne W. Rogers, A. Sibley, Jeff Sheng, Dadong Zhang, Xiaodi Qin, E.S. Lipp, Swaminathan Kumar, Aron Y Joon, Pixu Shi, Michael A. Davies, K. Owzar, Carey K. Anders, April K S Salama
          </td>
          <td>2025-06-02</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Allelic gene-specific regulatory events are crucial mechanisms in organisms, pivotal to many fundamental biological processes such as embryonic development and chromosome inactivation. Allelic gene imbalance manifests at both RNA expression and epigenetic levels. Recent research has unveiled allelic-specific regulation of RNA N6-methyladenosine (m6A), emphasizing the need for its precise identification. However, prevailing approaches primarily focus on screening allele-specific genetic variations associated with m6A, but not truly identify allelic m6A events. Therefore, the construction of a novel algorithm dedicated to identifying allele-specific m6A (ASm6A) signals is still necessary for comprehensively understanding the regulatory mechanism of ASm6A. Findings To address this limitation, we have developed a meta-analysis approach using hierarchical Bayesian models to accurately detect ASm6A events at the peak level from MeRIP-seq data. For user convenience, we introduce a unified analysis pipeline named M6Allele, streamlining the assessment of significant ASm6A across single and paired samples. Applying M6Allele to MeRIP-seq data analysis of pulmonary fibrosis and lung adenocarcinoma reveals enrichment of ASm6A events in key regulatory genes associated with these diseases, suggesting their potential involvement in disease regulation. Conclusions Our effort provides a method for precisely identifying ASm6A events at the peak level, elucidates the interplay of m6A with human health and disease genetics, and paves a new visual angle for disease research. The M6Allele software is freely available at https://github.com/RenLabBioinformatics/M6Allele under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a8b2fc14a09819d025933f902cc0922e8465cf" target='_blank'>
              M6Allele: a toolkit for detection of allele-specific RNA N6-methyladenosine modifications
              </a>
            </td>
          <td>
            Yin Zhang, Lin Tang, Shengyao Zhi, Bosu Hu, Zhixiang Zuo, Jian Ren, Yubin Xie, Xiaotong Luo
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CHROMR is a primate-specific long noncoding RNA with emerging roles in homeostasis and pathophysiology. Elevated blood levels of CHROMR have been observed in patients with cardiovascular disease and several cancers, where it is correlated with poor clinical outcomes. Like many lncRNAs, CHROMR accumulates in both the nucleus and the cytoplasm, and it assumes distinct functions in each of these cellular compartments. In the nucleus, CHROMR sequesters a transcriptional repressor complex to activate interferon-stimulated gene expression and antiviral immunity. In the cytoplasm, CHROMR competitively inhibits microRNAs involved in cholesterol efflux and cell cycle regulation, thereby impacting gene pathways involved in reverse cholesterol transport, HDL biogenesis, and tumor growth. In this review, we detail the multifaceted functions of CHROMR in cholesterol metabolism, innate immunity, and cancer progression. We also explore the potential molecular mechanisms that govern its expression and dynamic subcellular localization, which may be key to its functional versatility. Advancing our understanding of the regulatory networks and cellular environments that shape CHROMR activity will be critical for assessing its promise as a therapeutic target and diagnostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e434fd00ac8710c07c05ad122a8af40ef9af54ab" target='_blank'>
              The Multifaceted Roles of CHROMR in Innate Immunity, Cancer, and Cholesterol Homeostasis
              </a>
            </td>
          <td>
            Emma R. Blaustein, Coen van Solingen
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a leading cause of cancer-related mortality in men worldwide, with tumor heterogeneity, therapy resistance, and lineage plasticity posing significant clinical challenges. Androgen deprivation therapy (ADT), while initially effective, often culminates in castration-resistant prostate cancer (CRPC), may fueled by castration-resistant prostate cancer stem cells (CSCs) with adaptive self-renewal and regenerative capacities. Emerging evidence implicates prostate CSCs as pivotal contributors to tumor heterogeneity, drug resistance, and disease recurrence. These stem subpopulations exhibit intrinsic adaptability through genetic, epigenetic, and microenvironmental reprogramming, enabling survival under androgen-deprived conditions and fostering clonal diversification. This review synthesizes current knowledge on the identity, regulation, and functional dynamics of prostate stem cells, emphasizing their role in shaping the tumor ecosystem. By dissecting the genetic drivers, epigenetic alteration and crosstalk with microenvironment, this review underscores critical triggers on prostate CSCs determination and differentiation. Additionally, we discuss emerging strategies to target prostate CSCs-specific vulnerabilities, including molecular drivers of stemness and plasticity, to improve therapeutic outcomes for advanced prostate malignancies. This synthesis underscores the critical need to unravel prostate stem cells biology for developing precision therapies against prostate CSCs-driven adaptation in prostate diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/749e1d03aa4d620992d251430c919ea9331b3e5d" target='_blank'>
              Prostate cancer stem cell dynamics in the evolution of drug resistance.
              </a>
            </td>
          <td>
            Xiaoyu Zhang, Xiaoqing Zhang, Xinpei Deng, Jingna Cao, Qing Bao, Huan Wang, Dong Chen, Hailin Tang
          </td>
          <td>2025-06-04</td>
          <td>International journal of surgery</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome segregation is essential for cellular proliferation. Unlike eukaryotes, bacteria lack cytoskeleton-based machinery to segregate their chromosomal DNA (nucleoid). The bacterial ParABS system segregates the duplicated chromosomal regions near the origin of replication. However, this function does not explain how bacterial cells partition the rest (bulk) of the chromosomal material. Furthermore, some bacteria, including Escherichia coli, lack a ParABS system. Yet, E. coli faithfully segregates nucleoids across various growth rates. Here, we provide theoretical and experimental evidence that polysome production during chromosomal gene expression helps compact, split, segregate, and position nucleoids in E. coli through nonequilibrium dynamics that depend on polysome synthesis, degradation (through mRNA decay), and exclusion from the DNA meshwork. These dynamics inherently couple chromosome segregation to biomass growth across nutritional conditions. Halting chromosomal gene expression and thus polysome production immediately stops sister nucleoid migration, while ensuing polysome depletion gradually reverses nucleoid segregation. Redirecting gene expression away from the chromosome and toward plasmids causes ectopic polysome accumulations that are sufficient to drive aberrant nucleoid dynamics. Cell width enlargement experiments suggest that limiting the exchange of polysomes across DNA-free regions ensures nucleoid segregation along the cell length. Our findings suggest a self-organizing mechanism for coupling nucleoid compaction and segregation to cell growth without the apparent requirement of regulatory molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38632c47e3d185d80d1fdc81e020f23db296ddb" target='_blank'>
              Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli
              </a>
            </td>
          <td>
            Alexandros Papagiannakis, Qiwei Yu, Sander K. Govers, Wei-Hsiang Lin, N. Wingreen, C. Jacobs-Wagner
          </td>
          <td>2025-06-24</td>
          <td>eLife</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cell spheroids autonomously form in the ascites fluid and are considered a conduit for epithelial ovarian cancer metastasis within the peritoneal cavity. Spheroids are homotypic, avascular 3D structures that acquire resistance to anoikis to remain viable after cellular detachment. We used in vitro spheroid model systems to interrogate pathways critical for spheroid cell proliferation, distinct from those driving monolayer cancer cell proliferation. Using the 105C and KOC-7c human ovarian clear cell carcinoma (OCCC) cell lines, which have distinct proliferative phenotypes as spheroids but the same prototypical OCCC gene mutation profile of constitutively activated AKT signaling with the loss of ARID1A, we revealed therapeutic targets that efficiently kill cells in spheroids. RNA-seq analyses compared the transcriptome of 3-day monolayer and spheroid cells from these lines and identified the characteristics of dormant spheroid cell survival, which included the G2/M checkpoint, autophagy, and other stress pathways induced in 105C spheroids, in sharp contrast to the proliferating spheroid cells of the KOC-7c cell line. Next, we assessed levels of various G2/M checkpoint regulators and found a consistent reduction in steady-state levels of checkpoint regulators in dormant spheroid cells, but not proliferative spheroids. Our studies showed that proliferative spheroid cells were sensitive to Wee1 inhibition by AZD1775, but the dormant spheroid cells showed a degree of resistance to AZD1775, both in terms of EC50 values and spheroid reattachment abilities. Thus, we identified biomarkers of dormant spheroids, including the G2/M checkpoint regulators Wee1, Cdc25c, and PLK1, and showed that, when compared to proliferating spheroid cells, the transcriptome of dormant OCCC spheroids is a source of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877fd898a1fb77cad11de61e359dcf933c175ace" target='_blank'>
              Transcriptomic Analyses of Ovarian Clear Cell Carcinoma Spheroids Reveal Distinct Proliferative Phenotypes and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Bartlomiej Kolendowski, Sylvia Cheng, Yudith Ramos Valdés, Trevor G. Shepherd, G. DiMattia
          </td>
          <td>2025-05-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 e15032


 Background:
 Homologous recombination deficiency (HRD) is an event triggered by impairment of the homologous recombination repair (HRR) pathway which acts on DNA double-strand breaks and interstrand cross-links (ICL). Although HRD has been shown to elevate the overall genomic instability in tumors inducing favorable responses to immune checkpoint inhibitors (ICI), its clinical utility is observed in specific cancer types like breast, ovarian, low-grade glioma, testicular germ cell tumors and kidney cancers. On the contrary, in many other cancer types, HRD is conducive to the development of immune-cold tumor microenvironment leading to unfavorable treatment responses to ICI. We present, HRD predictive scores with the corresponding IO biomarkers viz. microsatellite instability (MSI) and tumor mutation burden (TMB) pan-cancers.
 Methods:
 Retrospectively, we investigated the biomarker profile (HRD, MSI, TMB) and genomic (HRR pathway genes) profiles of 642 pan-cancer patients using a comprehensive genomic assay using OncoIndx.
 Results:
 Of the cohort of 642 pan-cancer patients, a total of 12.3% (n = 79/642) of patients were detected with high HRD scores, another 16.0% (n = 103/642) were detected with intermediate HRD scores and while 71.7% (n = 460/642) of patients showed low HRD scores. Among the cohorts of high, intermediate and low HRD scores, BRCA1/2 mutation-positive patients were 6.3% (n = 5/79), 5.8% (n = 6/103), and 5.2 (n = 24/460) respectively and HRR gene mutation-positive patients were 17.7% (n = 14/79), 25.2% (n = 26/103), 17.17% (n = 79/460) respectively. Most importantly, the immune-oncology biomarkers MSI and TMB scores were high only in 1.3% (n = 1/79) and 8.9% (n = 7/79) in HRD-high patients which were not significant compared to 1.7% (n = 8/460) and 10.4% (n = 48/460) observed in HRD-low patient cohort. Even in the cohort of patients with intermediate HRD scores, MSI and TMB did not vary significantly. Overall, no significant correlation was observed between high HRD scores vs BRCA1/2 mutations, HRR pathway gene mutations, MSI and TMB statuses. Thus, HRD scores could be predicted high, irrespective of all the associated biomarkers and pathway mutations. This indicates the utility of additional treatment options in BRCA1/2 wild-type or HRR pathway gene wild-type patients.
 Conclusions:
 Although HRD-high patients could be predictive of favorable PARPi and platinum-based therapies, their role in immune activation and ICI therapy response should be carefully considered. Even at a high score, HRD seems to be inducing non-inflammatory tumor microenvironment as opposed to triggering novel cell surface antigens and high TMB aiding in immunotherapy response. However, HRD as an individual biomarker alone adds immense therapeutic benefits even among BRCA-negative patients who could be potential for PARPi/platinum-based therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92c61b0baeee9c5a60d5b38acb29b5a9cb82e2f" target='_blank'>
              The non-canonical role of homologous recombination deficiency in the development of immune-cold tumor microenvironment.
              </a>
            </td>
          <td>
            A. Vasudevan, Aarthi Ramesh, Ankit Jain, Sandhya Iyer, Nidhi Patel, A. Joshi, M. K. Singhal, Sameer Shrirangwar, Chinmaya Kumar Pani, Ganapathi Bhat, V. Patil, Dr T. Raja, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0e72d616aeb3ee8d41accd4d7a574cb580739" target='_blank'>
              A continuum reaction-diffusion model for spread of gene silencing in chromosomal inactivation
              </a>
            </td>
          <td>
            Shibashis Paul, Sha Sun, Karmella A. Haynes, Tian Hong
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 5072


 Background:
 STEAP
 1 and 2 (six-transmembrane epithelial antigen of prostate) are metalloreductase proteins involved in a variety of biologic processes.
 STEAP1/2
 are tumor-associated cell surface antigens highly expressed in prostate cancer (PC), although their role in cancer is poorly understood.
 STEAP1/2
 have emerged as successful targets for adoptive T-cell therapy trials for PC. We employed a multi-omics approach to investigate the molecular features associated with
 STEAP1
 and
 STEAP2
 expression in PC.
 Methods:
 NextGen Sequencing of DNA (592 genes or whole exome) and RNA (whole transcriptome) was performed for PC tumors (n = 7089) submitted to Caris Life Sciences (Phoenix, AZ). PC samples were stratified by
 STEAP1/2
 mRNA levels into top (high) and bottom quartile (low). Immune cell infiltration in the tumor microenvironment (TME) was inferred by quanTIseq. Transcriptomic signatures of androgen receptor signaling (AR), neuroendocrine classification (NEPC), and interferon gamma signaling (IFN) were calculated. Mann-Whitney U and X2/Fisher-Exact tests were applied where appropriate, with P-values adjusted for multiple comparisons (q< .05).
 Results:
 Of 7,089 samples, 63.2% were from the prostate; 11.7% from lymph node metastases (LNM); 7.3% from bone; and 17.8% from visceral/soft tissue metastases (V/STM).
 STEAP
 -1 and -2 were significantly correlated to each other (R= 0.90, p<.001), with significantly higher expression of
 STEAP1
 observed in primary prostate and LNMs, compared with reduced expression in V/STM (
 STEAP1
 TPM: 105.2 vs 140.6 vs 91.9 p<.001). Mutations in
 AR
 (3.8% v 1.9%),
 KDM6A
 (4.2% v 2.2%),
 SPOP
 (10.9% v 8.4%) and
 AR
 V7 (23.0% v 10.5%) were enriched in
 STEAP1
 high PC (each q<.01). Mutations in
 KDM6A
 (3.8% v 2.5%) and
 AR
 V7 (17.6% v 14.3%) were enriched in
 STEAP2
 high PC (each q<.05).
 STEAP1/2
 expression negatively correlated with TMB count (R =-0.03, p<.05) and IFN score (R = -0.26, p<.001). Concordantly, fewer proinflammatory immune cell fractions (M1 Macrophages, NK cells, CD4+/CD8+ T cells, myeloid dendritic cells) were observed within the TME of
 STEAP1/2
 high PC (p<.0001). However,
 STEAP1/2
 expression correlated positively with the AR signature (R = 0.39, p<.001) and androgen response pathways, while correlating negatively with the NEPC signature (R = -0.15, p<.001).
 Conclusions:
 PC tumors expressing high
 STEAP1/2
 display distinct genomic and transcriptomic profiles compared to
 STEAP1/2
 -low PC, and
 STEAP 1/2
 expression varies across sites of metastases. Immune biomarkers and immune cell infiltration data suggest that STEAP1/2 may be associated with a cold TME. The recent success of STEAP1-targeting T-cell redirecting therapies mechanisms by which adoptive T-cell strategies may overcome immunosuppressive factors within the TME. Ongoing development of T-cell immunotherapeutics targeting STEAP1 may account for the differential expression profiles in guiding patient selection and combination strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3db32deff1c58e26b5169abd853a15c53e234839" target='_blank'>
              Molecular characterization of STEAP1 and -2 in advanced prostate cancer.
              </a>
            </td>
          <td>
            Kevin K. Zarrabi, T. Adeyelu, A. Elliott, D. Geynisman, D. Oh, Carissa E. Chu, Rana R McKay, Nicholas A Zorko, E. Antonarakis, Lucia R Languino, Pedro C. Barata, D. Danila, Norm Smith, William Kevin Kelly
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA repair is essential for preserving genome integrity. Podocytes, postmitotic epithelial cells of the kidney filtration unit, bear limited regenerative capacity, yet their survival is indispensable for kidney health. Podocyte loss is a hallmark of the aging process and of many diseases, but the underlying factors remain unclear. We investigated the consequences of DNA damage in a podocyte-specific knockout mouse model for DNA excision repair protein Ercc1 and in cultured podocytes under genomic stress. Furthermore, we characterized DNA damage-related alterations in mouse and human renal tissue of different ages and patients with minimal change disease and focal segmental glomerulosclerosis. Ercc1 knockout resulted in accumulation of DNA damage and ensuing albuminuria and kidney disease. Podocytes reacted to genomic stress by activating mTOR complex 1 (mTORC1) signaling in vitro and in vivo. This was abrogated by inhibiting DNA damage signaling through DNA-dependent protein kinase (DNA-PK) and ataxia teleangiectasia mutated (ATM) kinases, and inhibition of mTORC1 modulated the development of glomerulosclerosis. Perturbed DNA repair gene expression and genomic stress in podocytes were also detected in focal segmental glomerulosclerosis. Beyond that, DNA damage signaling occurred in podocytes of healthy aging mice and humans. We provide evidence that genome maintenance in podocytes is linked to the mTORC1 pathway and is involved in the aging process as well as the development of glomerulosclerosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e1092b41bf7562a413f9fd7c4ddd40708218ce" target='_blank'>
              Loss of genome maintenance is linked to mTOR complex 1 signaling and accelerates podocyte damage
              </a>
            </td>
          <td>
            Fabian Braun, Amrei M. Mandel, Linda Blomberg, M. N. Wong, G. Chatzinikolaou, David H Meyer, Anna Reinelt, V. Nair, Roman Akbar-Haase, P. McCown, Fabian Haas, He Chen, Mahdieh Rahmatollahi, Damian Fermin, Robin Ebbestad, G. Slaats, Tillmann Bork, C. Schell, Sybille Koehler, P. Brinkkoetter, Maja T. Lindenmeyer, C. D. Cohen, M. Kann, D. Unnersjö-Jess, Wilhelm Bloch, Matthew G Sampson, Martijn Dollé, Victor G. Puelles, M. Kretzler, G. Garinis, Tobias B. Huber, Bernhard Schermer, Thomas Benzing, Björn Schumacher, Christine E. Kurschat
          </td>
          <td>2025-05-20</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction: Epigenetic changes play a crucial role in cancer development. Among these, DNA methylation is one of the most significant due to its impact on gene expression and genome stability. Hyper-methylation generally suppresses transcription and hypo-methylation promotes it. In breast cancer research, tumor-adjacent tissue is often used as a reference for characterizing epigenetic alterations, but numerous studies suggest that the tissue adjacent to a tumor shares genomic characteristics with the tumor itself. The extent to which epigenetic profiles overlap between tumor, histologically normal tissues from the involved and uninvolved breast, and benign tissues from cancer-free individuals is unknown. Our study aims to address this gap by systematically comparing the DNA methylation profiles across these different tissue types. Methods: A total of 72 cancer patients were selected from Northwestern University, while data from 182 cancer-free women was obtained from tissue donated to the Komen Tissue Bank. From each individual cancer patient, four regional samples were collected: tumor tissue (TU), tumor-adjacent normal tissue (AN), benign tissue from the opposite quadrant of the involved breast (OQ), and benign tissue from the contralateral uninvolved breast (CUB). From cancer-free patients, normal breast tissue was collected (CFN). From these, we created a “tumor proximity axis”: CFN→CUB →OQ→AN→TU. Methylation profiles were assayed using Illumina’s Infinium Methylation EPIC v1.0 BeadChip. Differential methylation analysis was conducted in two sets of four pairwise comparisons: (1) comparing tumor samples to the four benign tissue categories, and (2) comparing tissue categories that are adjacent along the tumor proximity axis. The differentially methylated CpGs were analyzed for enrichment of transcription factor binding sites (TFBS) and genes. Results: Following data processing and quality control, there were 72 TU, 63 AN, 70 OQ, 72 CUB, and 182 CFN samples for analysis. Differential methylation analysis showed that TU samples had distinct methylation profiles, with more hypomethylation events compared to hypermethylation events relative to benign tissues. Case-benign tissues (AN, OQ, CUB) exhibited similar methylation profiles, distinct from CFN. Transcription factor binding site (TFBS) enrichment analysis revealed that case-benign samples, even those distant from tumor, i.e. CUB, showed breast cancer-related methylation changes. Hypomethylated sites in CUB compared to CFN were enriched for TF binding sites TP63, GATA3, ESR1, PR, AR, NR3C1, and GREB1. TU hypermethylation events were enriched for Polycomb-repressive complex 2 (PRC2) binding, including EZH2, SUZ12, and JARID2. ER+ and ER- tumors had distinct methylation profiles. Both ER+ and ER- tumors showed hypermethylation enrichment for PRC2-related binding motifs. However, hypomethylation events differed, with ER+ tumors enriched for hormone receptor-related pathways and ER- tumors enriched for hematopoiesis/immune-related pathways. We did not find any differential methylation between benign tissues from patients with ER+ vs. ER- tumors. Conclusions: DNA methylation changes profoundly at two points on the tumor proximity axis: CFN to CUB and AN to TU. Specifically, all case-benign tissues, including CUB, exhibited changes associated with breast cancer when compared to CFN. Although the methylation profiles of ER+ and ER- tumors differed, benign tissues showed no differences as a function of ER status of the tumor. These findings demonstrate that benign case tissue is not epigenetically “normal” and provide insights for further investigation into the process of tumorigenesis.
 Citation Format: Saya Dennis, Takahiro Tsukioki, Gannon Cottone, Wanding Zhou, Yuan Luo, Patricia A. Ganz, Mary E. Sehl, Seema Khan, Susan Clare. DNA methylation patterns are similar in benign tissue from ipsilateral and contralateral breast while different from matched breast cancer, and healthy controls [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr RF1-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e532f44a3009f75c42b2bf7806776eed3af9ac" target='_blank'>
              Abstract RF1-05: DNA methylation patterns are similar in benign tissue from ipsilateral and contralateral breast while different from matched breast cancer, and healthy controls
              </a>
            </td>
          <td>
            Saya R. Dennis, Takahiro Tsukioki, Gannon Cottone, Wanding Zhou, Yuan Luo, Patricia A. Ganz, M. Sehl, Seema A. Khan, Susan E Clare
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Here, we address a major limitation in the effectiveness of poly(ADP-ribose) polymerase inhibitors (PARPi) in breast cancer gene (BRCA)-mutant prostate cancer treatment: Only ~50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARPi resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss-of-function mutations in the DNA prereplication complex (pre-RC) confer PARPi resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-RC, can restore PARPi sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95a3ba5883a1720490b54e4008aa7d7b0986c0b" target='_blank'>
              BRCA2 reversion mutation–independent resistance to PARP inhibition through impaired DNA prereplication complex function
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, Nikolaus Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles L Sawyers
          </td>
          <td>2025-06-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="Simple Summary Telomere shortening plays a significant role in the development of age-related diseases and cancer. However, this process can be reversed through telomerase, which is highly active in tumors. This narrative review examines the molecular mechanisms underlying increased telomere attrition, which contributes to an elevated risk of leukemias, highlighting the potential prognostic value of telomere length in leukemias. It emphasizes the connection between oxidative stress and mitochondrial dysfunction in accelerating telomere shortening, which contributes to a higher onset and progression of leukemia. Additionally, telomerase emerges as a therapeutic target in leukemias. For this reason, telomerase inhibitors and telomerase-based immunotherapy are analyzed in the fight against leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f5367fd1dcbeb8eeebf72138504efa7d9e2b057" target='_blank'>
              The Telomere Length Signature in Leukemias—From Molecular Mechanisms Underlying Telomere Shortening to Immunotherapeutic Options Against Telomerase
              </a>
            </td>
          <td>
            Stella Baliou, Iordanis Pelagiadis, M. Apetroaei, E. Vakonaki, A. Arsene, Eleftheria Hatzidaki, M. Tzatzarakis, P. Ioannou, Artistidis Tsatsakis, E. Stiakaki
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891242f777da10d08f786d978bb8ecf69ade37e9" target='_blank'>
              Replaying germinal center evolution on a quantified affinity landscape
              </a>
            </td>
          <td>
            William S DeWitt, Ashni A. Vora, Tatsuya Araki, Jared G. Galloway, T. Alkutkar, Juliana Bortolatto, Tiago B.R. Castro, Will Dumm, Chris Jennings-Shaffer, Tongqiu Jia, L. Mesin, G. Ozorowski, Juhee Pae, Duncan K. Ralph, Jesse D. Bloom, Armita Nourmohammad, Yun S. Song, A. Ward, Tyler N. Starr, Frederick A. Matsen, G. Victora
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction: Colorectal cancer (CRC) is a heterogeneous and multifactorial malignancy driven by a series of genetic and epigenetic alterations. In this field, telomere/telomerase dysfunction contributes to CRC carcinogenesis by impairing genomic stability and cellular replication. Objective: This study aimed to evaluate genetic and epigenetic alterations in CRC by examining mutation rates in the human telomerase reverse transcriptase (hTERT) promoter region, relative telomere length (RTL), hTERT gene expression, and DNA methylation in the TERT hypermethylated oncological region (THOR). Methods: A total of 45 CRC and 34 adjacent normal tissue samples from Moroccan patients were analyzed using molecular approaches, such as Sanger sequencing, quantitative PCR (qPCR), reverse transcription qPCR (RT-qPCR), and methylation-specific PCR (MSP). Results: No mutations in the hTERT promoter region were identified. However, hypermethylation in the THOR region was reported in 82.2% of CRC samples and 79.4% of adjacent normal tissues. High hTERT expression was detected in 50% of CRC patients. In addition, telomere length was significantly shorter (p=0.002) in cancerous tissues (1.41 [1.36 – 1.43]) compared to normal mucosa (1.559 [1.46 – 1.63]), with an RTL ratio less than 1 (0.90 [0.86 – 0.95]). No significant differences were found between clinicopathological features and hTERT expression, THOR methylation, or RTL, except for a significant correlation between THOR hypermethylation and smaller tumor size (p=0.017) and between THOR methylation and RTL in CRC tissues (p=0.034). Conclusion: These results suggest that telomere lengthening is crucial for CRC initiation and progression, and cancer cells tend to shorten telomeres to maintain the chromosomal instability (CIN) required for tumor progression. Further research is needed to elucidate the mechanisms underlying telomere shortening in CRC and understand the role of telomerase/telomere complex in CRC initiation and progression, which could provide new diagnostic, prognostic, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e32cd6246bc493b71aa8671eea1af0f3ca4e1f7" target='_blank'>
              Exploring genetic and epigenetic alterations in hTERT gene in colorectal cancer
              </a>
            </td>
          <td>
            Meryem Jafari, Boutaina Addoum, A. Laraqui, Mohammed Oukabli, Ahmed Bounaim, Abdelmounim Ait Ali, Sidi Mohammed Bouchantouf, Noureddine Njoumi, Belkouchi Abdelkader, Fatima Zohra BenMoula, Hicham El Annaz, Khalid Ennibi, I. Chaoui, Youssef Bakri, Mohammed Attaleb
          </td>
          <td>2025-06-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a889f8e2ddcb6a4b7bab00376e74479c448ebb" target='_blank'>
              Structural basis of Fanconi anemia pathway activation by FANCM.
              </a>
            </td>
          <td>
            R. Bythell-Douglas, Sylvie van Twest, Lara Abbouche, Elyse Dunn, Rachel J Coulthard, D. Briggs, Vincent J. Murphy, Xinxin Zhang, Winnie Tan, Sarah S Henrikus, Dongming Qian, Yin Wu, Jana Wolf, Laurent Rigoreau, S. Shakeel, Kathryn L Chapman, N. Q. McDonald, A. Deans
          </td>
          <td>2025-05-30</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99ef5f46003666cce9bfbffdcaf4920a36fe552" target='_blank'>
              RNA G-quadruplexes: emerging regulators of gene expression and therapeutic targets.
              </a>
            </td>
          <td>
            Zukela Ruzi, Wei Zha, Huang Yuan Yuan, Jiaorui Liu
          </td>
          <td>2025-07-03</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Two epigenetically labile subsets of genes that link embryonic environmental exposures with adult disease susceptibility are those that are imprinted and those with metastable epialleles. The expression of genes with metastable epialleles, like the agouti gene in Agouti viable yellow (Avy) mice, is highly variable between individuals but uniform in tissues within an individual. We used the Avy mouse to demonstrate that exposure to nutritional supplements, chemical toxicants, and low-dose ionizing radiation during embryogenesis alters adult disease susceptibility by modifying the epigenome. Genomic imprinting is a unique species-dependent epigenetic form of gene regulation that evolved approximately 150 million years ago in a common ancestor to Therian mammals. It resulted in monoallelic parent-of-origin-dependent gene silencing. Thus, imprinted genes are functionally haploid disease susceptibility loci, since only a single genetic or epigenetic event is required to alter their function. Expression of imprinted genes in the human genome is regulated by hemi-methylated imprint control regions (ICRs) in the human imprintome. Furthermore, human imprintome ICRs associated with chronic diseases (e.g., cancer, diabetes, and obesity) and behavioral disorders (e.g., autism, bipolar disorder, psychopathy, and schizophrenia) can now be identified with the use of cells from peripheral samples and the human imprintome array. The importance of metastable epialleles and imprinted genes in the etiology of environmentally induced human chronic diseases is discussed in this review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960e9fe6811efda91a98ebc102150c171e03b920" target='_blank'>
              Epigenetics, human imprintome, and chronic diseases.
              </a>
            </td>
          <td>
            Randy L Jirtle
          </td>
          <td>2025-07-04</td>
          <td>Essays in biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is a highly heterogeneous malignant tumor. Among the various epigenetic alterations involved in its pathogenesis, the N6-methyladenosine (m6A) modification of eukaryotic mRNA plays a pivotal role in tumor biology and holds significant potential for diagnostic and therapeutic applications. In this study, we identified and validated an 8-gene risk score that robustly predicts prognosis in HCC patients, achieving area under the ROC curve (AUC) values greater than 0.75 across both internal and external cohorts. A high-risk score was significantly associated with poorer clinical outcomes and reduced immune tolerance, while also indicating a potentially enhanced response to immunotherapy. Single-cell RNA sequencing revealed a higher proportion of T cells and a decreased presence of immunosuppressive T cells in the high-risk group, which may account for their improved responsiveness to immune checkpoint inhibitors. Finally, in vitro experiments and immunohistochemical staining showed the biological function of the key gene ANLN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea94ab482ecd557d67969f8038f92a2cc7e807c" target='_blank'>
              Stratification of Hepatocellular Carcinoma Using N6-Methyladenosine
              </a>
            </td>
          <td>
            Nan Wang, Jia-Xin Shi, M. Bartneck, E. Dahl, Junqing Wang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e359c7ef64214d02915af71ac1f575ebe2c2dd" target='_blank'>
              Chinmo defines the region-specific oncogenic competence in the Drosophila central nervous system
              </a>
            </td>
          <td>
            Phuong-Khanh Nguyen, Francesca Froldi, Louise Y. Cheng
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 2633


 Background:
 HER2/neu mutations or amplifications, present in up to 23% of non-small cell lung cancers (NSCLCs), activate STAT3, promoting inflammation and suppressing adaptive immune responses. Elevated systemic inflammation-immune index (SII) in small cell lung cancer (SCLC) correlates with reduced response to PD-1/L1 immune checkpoint inhibitors (ICIs), leading to worse overall survival (OS) and progression-free survival (PFS). These findings highlight the need to mitigate inflammation in lung cancers to enhance ICI response and improve survival. Biomarker studies can identify genes that differentiate inflammatory and immune pathways, offering therapeutic insights.
 Methods:
 To identify genes regulating the HER2/neu oncogene, HER2/neu-overexpressing mice were crossed with genetically Diverse Outbred (DO) mice. The resulting DO F1 offspring were monitored for HER2/neu-expressing tumor onset and growth. Tumor phenotypes were associated with mouse haplotypes to identify quantitative trait loci (QTL) on chromosomes (chr) 2, 6, and X linked with aggressive HER2/neu tumors. Of the 35 candidate genes harboring protein-coding changes, homologous genes on human chr 1, 10, and X were analyzed using Caris Life Sciences datasets of NSCLC and SCLC: 25,143 primary NSCLC; 12,365 metastatic NSCLC; 621 primary SCLC; and 971 metastatic SCLC where primary biopsies are those taken from the lung and metastatic biopsies from sites other than the lung. Patient cohorts stratified by gene expression (top vs. bottom 50%) were correlated with OS, ICI response, and time on pembrolizumab or atezolizumab treatment (TOT).
 Results:
 Several homologous candidate genes correlated with lung cancer survival in both NSCLC and SCLC. Notably, high expression of
 LILRB4
 , a macrophage-specific checkpoint molecule, was associated with improved survival (HR 0.66–0.84,
 p
 < 0.001) across all lung cancer types and sites. Additionally, elevated
 TSC22D3
 , a glucocorticoid receptor-activated gene regulating anti-inflammatory pathways, including
 LILRB4
 expression, was positively associated with OS in metastatic NSCLC (HR 0.82,
 p
 < 0.00001) and SCLC (HR 0.77,
 p
 < 0.001). Expression of
 LILRB4
 and
 TSC22D3
 further enhanced survival in ICI-treated patients (SCLC: HR 0.72; NSCLC: HR 0.82;
 p
 < 0.001). In NSCLC, high
 LILRB4
 expression conferred an 18% improved TOT with pembrolizumab (HR 0.858,
 p
 < 0.0001).
 Conclusions:
 LILRB4
 and
 TSC22D3
 are key genes linked to improved survival outcomes in both SCLC and NSCLC, likely through their roles in mitigating inflammation. Their association with enhanced ICI responses underscores their potential as therapeutic targets. Future research will evaluate the relationship between
 LILRB4
 and
 TSC22D3
 expression and HER2/neu status in NSCLC and validate protein-level correlations in positive cells. Our long-term goal is to determine the therapeutic potential of
 LILRB4
 and
 TSC22D3
 in SCLC, NSCLC and HER2/neu-positive NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0e9d9066dedda6abe73bbff9473b3f5d6c28e8" target='_blank'>
              Effect of elevated expression of
 LILRB4
 and
 TSC22D3
 on survival in lung cancer.
              </a>
            </td>
          <td>
            B. Hrinczenko, T. Adeyelu, Wei-Zen Wei, A. Elliott, G. Bepler, A. Vanderwalde, B. Halmos, E. Antonarakis, Maryam B Lustberg, Jennifer Jacob
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Oral squamous cell carcinoma (OSCC) is associated with poor prognosis due to extensive local invasiveness and lymph node metastasis, often leading to a significant decrease in aesthetics and function after surgery. Therefore, elucidation of the molecular mechanisms underlying OSCC is necessary for its early detection and treatment. N6-methyladenosine (m6A) modification of mRNA is the most common form of post-transcriptional RNA methylation and is often involved in the progression of cancer by regulating the expression of various genes. Recent studies reported the tumor-promoting effects of vir-like m6A methyltransferase associated (VIRMA, also termed KIAA1429), a novel molecule involved in m6A modification; however, its role in OSCC remains poorly understood. Methods In the present study, we determined the total m6A levels and VIRMA expression in OSCC using immunohistochemistry of tissue specimens and evaluated their association with clinicopathologic characteristics. We also performed gene expression analysis of VIRMA/KIAA1429 using public datasets. Results We found that the m6A levels were significantly higher in the OSCC specimens of patients with a more advanced clinical stage (P = 0.0063), lymph node metastasis (P = 0.0323), and venous invasion (P = 0.0380) compared to those without. The analysis of the public datasets revealed that VIRMA/KIAA1429 expression levels were higher in head and neck SCC than in normal mucosa, whereas immunohistochemistry revealed that VIRMA-expressing OSCC was associated with a significantly shorter disease-free survival (P = 0.0043) and was an independent poor prognostic factor (P = 0.0179). Conclusions These results highlight the potential utility of RNA methylation and VIRMA expression for the diagnosis and treatment of OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4045080f537d11c412b6f7373ebe4ef6923638e2" target='_blank'>
              Total m6A RNA levels and VIRMA expression as potential diagnostic and prognostic markers in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Kaori Shima, Yudai Shimojukkoku, Yasunobu Oku, Kanako Higashimoto, Takahiro Tsuchiyama, Yuka Kajiya, Miyako Kurihara-Shimomura, Tomonori Sasahira
          </td>
          <td>2025-07-03</td>
          <td>Diagnostic Pathology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Introduction: BRCA1 promoter hypermethylation (BRCA1meth), which is a driver of homologous recombination deficiency (HRD), is observed in up to 25% of triple negative breast cancers (TNBCs) and ovarian carcinomas (OvCas). BRCA1meth is associated with worse outcome compared to BRCA1 mutations (Menghi et al., PMID: 35857626), and, when found in individuals without cancer (i.e. constitutional BRCA1meth), it is associated with increased risk of breast and ovarian cancer (Lonning, et al., PMID: 36074460). Current methodologies to determine the presence of BRCA1meth in a cancer sample require tissue biopsy and have not been adopted for standard clinical assessment. The GRAIL cell-free DNA (cfDNA)-based targeted methylation platform is a robust, biopsy-free, scalable assay that was optimized to distinguish cancer methylation patterns between different cancer signal origins. Here, we describe the GRAIL platform’s ability to detect and quantify BRCA1meth. Methods: As part of the CCGA study (NCT02889978), clinical data were recorded and plasma samples were collected and processed on GRAIL’s targeted methylation platform from 2,958 patients with cancer prior to any treatment, and from 2,790 individuals without cancer. Samples were evaluated from patients with a variety of cancers: breast, ovarian, cervical, uterine, lung, prostate, head and neck, kidney, bladder, stomach, esophageal, colorectal, and hematological malignancies. A dilution series of contrived samples was analyzed to establish analytical characteristics of BRCA1meth detection using the GRAIL assay. Results: When applied to contrived samples, the GRAIL platform was able to detect BRCA1meth at an LoD95 of <1% (as measured by BRCA1meth fraction = methylated molecules / total molecules). When applied to a cohort of 2,790 individuals without cancer, we found 4% of individuals harbored evidence of BRCA1meth in the fractional range of 0.4% to 37%, with the majority of individuals (81%) having <5% BRCA1meth fraction, similar to surveys using peripheral blood cell DNA (Lonning et al., PMID: 36074460). Consistent with recent data of constitutional BRCA1meth prevalence in umbilical cord blood of newborns (Nikolaienko, et al., PMID: 38053165), we see lower incidence of BRCA1meth in males without cancer (3%) compared to females without cancer (5%, p=0.01). BRCA1meth was significantly enriched in the cfDNA of patients with TNBC and OvCa compared to female individuals without cancer (17/112 (15%), adjusted p=0.001 for TNBC; 14/101 (14%), adjusted p=0.01 for OvCa), in concordance with previous assessments of BRCA1meth prevalence across different tumor types (Menghi et al., PMID: 35857626). In TNBC and OvCa, the per-sample BRCA1meth fraction ranged from 0% to 54% and was correlated with the methylation-based estimate of tumor fraction in both TNBC (R=0.68, p=0.0052) and OvCa (R=0.65, p=0.03). High BRCA1meth fractions (up to 16-21%) were observed in four individuals with uterine, colorectal, bladder, or kidney cancer. This suggests that BRCA1 deficiency may be implicated in these rare cases, which would imply HRD, making them candidates for specific treatments. Finally, across tumor types that affect both sexes, we found consistently lower percentages of BRCA1meth in male vs. female patients, reflecting the differential rates of constitutional BRCA1meth across the two sexes. This suggests that the dynamics of BRCA1 promoter methylation and its consequences for the development of cancer may differ by sex. Conclusions: This work shows that the GRAIL platform, as a single blood-based assay, can sensitively detect and quantify BRCA1meth from cfDNA, which may inform cancer susceptibility in non-cancer bearing individuals and may also have potential treatment selection implications for cancer patients.
 Citation Format: Francesca Menghi, Margaret Antonio, William Cance, Tracy Nance, Edison T. Liu. Quantification of Breast Cancer 1 gene (BRCA1) promoter methylation in individuals with and without cancer using cell-free DNA [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ede0da8ea6d210ede4f24e81580ed41fdb9cdd" target='_blank'>
              Abstract P2-11-03: Quantification of Breast Cancer 1 gene (BRCA1) promoter methylation in individuals with and without cancer using cell-free DNA
              </a>
            </td>
          <td>
            F. Menghi, Margaret Antonio, William Cance, Tracy Nance, Edison T. Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Here we review the historical background and contemporary insights into genetic dominance, focusing on haploinsufficiency (HI), that is, when the function of only one allele of a gene is not enough to ensure a normal phenotype in a diploid organism. A related phenomenon is triplosensitivity, that is, pathogenic effects when there are three instead of two copies of some 'genes'. The importance of gene dosage issues was realized in humans when whole chromosomal abnormalities (aneuploidy) could be linked to clinical phenotypes such as Down, Edwards, and Patau syndromes. Subsequently, subtler chromosomal deletions and duplications have been shown to be responsible for many developmental syndromes. In several cases, a dosage‐sensitive gene mapping to the relevant regions has been implicated as causal. We delve into the mechanisms of HI, especially due to direct protein insufficiency and subunit imbalances in the context of multi‐subunit complexes. We show how the nonlinearity inherent to the relationship between genotype and phenotype is responsible for the dominance of the underlying genetic variants. We also explore why increased gene dosage can lead to abnormal phenotypes. Examples include trisomy or segmental genomic duplications in humans and oncogene amplification in cancers. Finally, we examine a few cases of genetic synergy, where the combined effect of two or more variants amplifies their individual effects, underlying a distinguishable phenotype. Further research is required to elucidate the dynamics of multicomponent interactions to unravel the mechanistic complexities of genetic dominance, inter‐gene interactions, and their implications for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53a61642f9ad382de56a8be421a6a2e4cf23bf4" target='_blank'>
              Gene Dosage Sensitivity and Human Genetic Diseases
              </a>
            </td>
          <td>
            R. Veitia, Johannes Zschocke, James A. Birchler
          </td>
          <td>2025-07-01</td>
          <td>Journal of Inherited Metabolic Disease</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 DNA repair processes are the foundation for genome integrity and survival, especially in extreme environments where DNA damage occurs more frequently and where archaea are found. Nevertheless, first-hand experimental information on repair pathways in archaea is scarce, and assignment of repair proteins is currently largely based on homology. We showed previously that DNA lesions induced by CRISPR-Cas self-targeting are repaired by microhomology mediated end joining (MMEJ). To identify proteins involved in the archaeal MMEJ pathway, we used deletion strains devoid of proteins assigned to the key steps of MMEJ, to examine changes in the repair outcome. In addition we used aphidicolin to inhibit the activity of the essential PolB1 protein. For the first time, we were thereby able to experimentally identify proteins involved in this repair pathway in the euryarchaeal model organism Haloferax volcanii. The present study confirms that Mre11, Rad50, Fen1, PolB1, LigA and LigN take part in MMEJ, as previously inferred. In addition, we show that Cas1 and Hel308a are also involved in the MMEJ pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af42e41866e31a1c7b25ea6eee79e0cec5ab57c" target='_blank'>
              CRISPR-Cas induced self-targeting identifies key players in archaeal microhomology mediated end joining
              </a>
            </td>
          <td>
            A-L. Sailer, J. Wörtz, V. Smith, A. Stachler, F. Blau, M. Daratha, L-K Maier, T. Allers, A. Marchfelder
          </td>
          <td>2025-07-08</td>
          <td>microLife</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignancy among women globally, with its complexity linked to genetic variations and metabolic alterations within tumor cells. This study investigates the role of fumarate hydratase (FH), a key enzyme in the tricarboxylic acid (TCA) cycle, in breast cancer progression. Our findings reveal that FH mRNA and protein levels are significantly upregulated in breast cancer tissues and correlate with poor patient prognosis and aggressive tumor characteristics. Using in vitro and in vivo models, we demonstrate that FH overexpression enhances breast cancer cell proliferation, migration, and invasion through metabolic reprogramming and by increasing reactive oxygen species (ROS) production. Furthermore, we identify matrix metalloproteinase 1 (MMP1) as a downstream effector of FH, linked to p21 downregulation, elucidating a novel regulatory pathway influencing tumor behavior. Interestingly, unlike its tumor-suppressing role in other cancer types, this study highlights FH's oncogenic potential in breast cancer. Our results suggest that FH enhances cancer cell viability and aggressiveness via both catalytic and non-catalytic mechanisms. This work not only underscores the metabolic adaptations of breast cancer cells but also proposes FH as a potential biomarker and therapeutic target for breast cancer management. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00397-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/757f91ceb4c980ac76d1ca46a85f3acd5f2e6714" target='_blank'>
              Oncogenic role of fumarate hydratase in breast cancer: metabolic reprogramming and mechanistic insights
              </a>
            </td>
          <td>
            Shyng-Shiou F Yuan, Anupama Vadhan, Hieu D. H. Nguyen, Pang-Yu Chen, Chih-Huang Tseng, Ching-Hu Wu, Yu-Chieh Chen, Yi-Chia Wu, Stephen Chu‐Sung Hu, S. Lo, Ming-Feng Hou, Yen-Yun Wang
          </td>
          <td>2025-05-29</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Nucleic acid amplification is a pivotal technology for gene investigation in molecular biology as well as an indispensable tool for nucleic acid-based analyses across various disciplines. However, the aerosolized amplicons render the subsequent reaction susceptible to carryover contamination, leading to serious false positive results, particularly in scenarios involving repeated amplification of identical sequences. Although the uracil-DNA glycosylase-based strategy can address this issue for natural DNA amplification, the uracil-containing DNA required for bisulfite conversion-based DNA methylation analysis fails to be analyzed because both aerosol and the intended template can be hydrolyzed. In this study, an innovative strategy for one-pot elimination of carryover contamination is established, applicable to both natural and uracil-containing DNA analyses using either thermal cycle-based or isotherm-based nucleic acid amplification. By incorporation of deoxyinosine triphosphate, hypoxanthine is inserted into amplicons, resulting in a different base composition from the original template. Hypoxanthine, a damaged base as the recognition site of endonuclease V, acts as a label for aerosol cleavage, while the intended template lacking hypoxanthine is unaffected. By systematic optimization, abundant aerosols are thoroughly hydrolyzed within a brief time. Despite the use of deoxyinosine triphosphate as an aberrant deoxyribonucleoside triphosphate, the amplification efficiency, sensitivity, and specificity are insusceptible. In addition, the hypoxanthine-containing amplicons can be analyzed not only for length identification by gel and capillary electrophoresis but also for sequence identification by sanger sequencing, pyrosequencing, and massively parallel sequencing. Moreover, the hypoxanthine-containing amplicons can be further used for cloning and restriction enzyme digestion, indicating the significant potential of this anti-aerosol strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5373512e1ee71db9407b5d72c170aac51f9f8cf9" target='_blank'>
              Elimination of Carryover Contamination during Nucleic Acid Amplification Based on Damaged Base Excision by DNA Repair Enzymes.
              </a>
            </td>
          <td>
            Xiaonan Liu, Wenjing Hu, Huyun Zhou, Jiaxing Zhang, Mengyang Zhao, Yali Cui, Jiangwei Yan
          </td>
          <td>2025-06-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the last two decades, significant improvements have been made in the understanding of the genomic and biological bases of acute lymphoblastic leukemia (ALL), resulting in enhanced genomic classification, more precise risk stratification and improved long-term outcomes. ALL is a hematologic malignancy defined by uncontrolled proliferation of immature B- or T- lymphoid blasts in the bone marrow, blood, and other extramedullary tissues. It affects most commonly children, representing the most common childhood cancer, but it also occurs in adults where outcome tends to be poorer compared to pediatric patients. A variety of genetic aberrations, including structural and numerical chromosome alterations, translocations generating fusion oncoproteins, cryptic genomic rearrangements, sequence mutations and genomic copy number changes, define multiple genomic subtypes, influence risk stratification and determine response to therapeutic strategies. In this review, we describe the updated genomic classification of ALL highlighting new biological insights and discussing their implications for prognostication and outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf200ea2c50518790c91d4e35c6ba5760ccd542" target='_blank'>
              Genomic Subclassification and Risk Stratification of ALL.
              </a>
            </td>
          <td>
            Anna Østergaard, Ilaria Iacobucci
          </td>
          <td>2025-06-30</td>
          <td>Acta haematologica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc4d33cb171c95693c151dadaf1339c2b7a61c7" target='_blank'>
              SCLC-TumorMiner: A Directly Accessible Genomics Resource for Precision Oncology – Big Data for Small Cells
              </a>
            </td>
          <td>
            Fathi Elloumi, A. Dhall, Daiki Taniyama, Augustin Luna, Yasuhiro Arakawa, Sudhir Varma, Jeffrey Wang, Anisha Tehim, Mark Raffeld, Kenneth Aldape, Christophe E. Redon, Roshan Shresta, William C Reinhold, M. Aladjem, Jaydira Del Rivero, Nitin Roper, Anish Thomas, Y. Pommier
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract Background Small cell lung cancer (SCLC) is a lethal lung malignancy and patients are often diagnosed with distant metastasis. Nearly all patients suffer from disease relapsing with inherent chemoresistance. Lack of targeted SCLC therapies further worsens disease outcomes, making it highly desirable to identify novel and effective therapeutic targets. Methods To search for potential therapeutic targets in SCLC, we analyzed publicly available single-cell and bulk RNA-sequencing (RNA-seq) data from normal, lung adenocarcinoma, and SCLC tumor tissues. To assess the targeting potential of FOXM1, we developed various in vitro models, including DOX-On-shFOXM1 (Tet-ON) inducible stable knockdown systems. Cisplatin resistant human and murine SCLC cell lines were generated to assess the role of FOXM1 in chemotherapy resistance. Immunoblotting, immunohistochemistry (IHC), and immuno-fluorescence were used to analyze the expression of FOXM1 and target proteins. ChIP-assay was used to study protein-gene interactions. Further, multicolor flow cytometry was employed to study the effect of FOXM1 inhibition on human T cells activation and differentiation. Subcutaneous xenograft and SCLC spontaneous (RPM: RB fl/fl ; TP53 fl/fl ; LSL-MYC T58A ) mouse models were used to evaluate the efficacy of FOXM1 inhibitors. Results Single-cell as well as bulk RNA-seq data revealed that FOXM1, an oncogenic transcription factor, is overexpressed in SCLC, and it was recapitulated in human and murine SCLC tissues and cell lines. Interestingly, chemo-resistant (CR) SCLC showed a substantially higher FOXM1 expression than naïve SCLC. Silencing FOXM1 genetically or pharmacologically by FOXM1 inhibitors revealed a marked reduction in cell viability, colony formation, migration and sphere formation in naïve and CR SCLC cells. Moreover, FOXM1 inhibition induced apoptosis and cell cycle arrest in SCLC cells. Furthermore, FOXM1 inhibition in combination with first-line platinum-based chemotherapy showed synergistic anticancer effects in both xenograft and RPM mouse models of SCLC. Our RNA-seq analysis revealed that FOXM1 inhibition altered the Aurora Kinase B (AURKB) signaling pathway, which is dysregulated in SCLC. Moreover, we found FOXM1 inhibition enhanced T cell activation and supported the differentiation of CD8 + cytotoxic T cells, and T cell-mediated killing of cancer cells. Conclusions Our study demonstrates that FOXM1 targeting using small molecule inhibitors has the potential to be a novel therapeutic strategy to combat SCLC progression including chemotherapeutic resistance and reshaping the anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf5514bc8f1dd3db6ac08b065580f4ed8890ea4" target='_blank'>
              FOXM1 targeting alters AURKB activity and reshapes antitumor immunity to curb the progression of small cell lung cancer
              </a>
            </td>
          <td>
            Md Arafat Khan, Parvez Khan, Mahek Fatima, Asad Ur Rehman, Laiba Anwar, Z. Alsafwani, Aatiya Ahmad, Mohammad Ali Abbas Zaidi, Jesse L. Cox, Areem Zahid, Sameer Mohiuddin, Sung Hoon Kim, Juan A. Santamaria-Barria, Imayavaramban Lakshmanan, B. Katzenellenbogen, John A Katzenellenbogen, A. Ganti, Surinder K Batra, M. Nasser
          </td>
          <td>2025-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Cervical cancer is the fourth most common malignant tumor in women, lacking specific and sensitive markers. TP53 induced glycolysis regulatory phosphatase (TIGAR) plays an important role in the initiation and progression of several solid tumors. This study aims to investigate the diagnostic and prognostic and immunotherapeutic role of TIGAR in cervical cancer. We observed that TIGAR expression was upregulated in cervical cancer cells and tissues using experiments and bioinformatics analyses. Moreover, elevated TIGAR expression was associated with poor prognosis using Kaplan–Meier survival analysis, and its diagnostic value was confirmed using the Receiver operating characteristic (ROC) curve. We also analyzed the correlation between TIGAR expression and immune infiltration and assessed its correlation with a clinical cohort of cervical cancer, and found that TIGAR expression was significantly associated with the infiltration levels of Foxp3 + Treg cells, CD11b + MDSC, CTLA4, and HAVCR2. TIGAR is a potential diagnostic and prognostic biomarker of cervical cancer and is expected to offer new possibilities for cervical cancer immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07201-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91786186f8b506b70f9791c3768dec5816483289" target='_blank'>
              TIGAR regulated by HPV E6 is correlated with disease stage, drug sensitivity, and immune microenvironment in cervical cancer
              </a>
            </td>
          <td>
            Yameng Liu, Yu Zhang, Hailong Zhang, Dianqin Xu, Xinzhu Zhou, Luhong Xie, Hongli Xi, Yurong Zhu, Hanlin Yang, Xiaoyu Zhu, Ji Ren, Shaoju Min, Yujie Tan
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Nuclear morphology is a defining cellular feature, differing based on cell type, tissue type, and species. In healthy cells, nuclear morphology is generally tightly regulated and maintained; however, dynamic changes in nuclear morphology are observed under certain conditions, for instance in early embryos and in some immune cells. Deviations in normal nuclear morphology are linked to numerous diseases, including most cancers and premature aging syndromes. Many regulators of nuclear morphology have been identified, encompassing both intranuclear, cytoplasmic, and extracellular factors. Of note, recent studies have converged on chromatin and chromatin-associated proteins as key determinants of nuclear morphology and dynamics. In this review we discuss how the chromatin landscape regulates nuclear morphology in both normal and diseased cellular states. Additionally, we highlight emerging technologies that promise to bridge critical gaps in our understanding of nuclear morphology, including new approaches to probe nuclear structure and the use of synthetic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adc2c44f577e2c2679b995d4aca9623ddc124e59" target='_blank'>
              How the chromatin landscape influences nuclear morphology
              </a>
            </td>
          <td>
            Sourabh Sengupta, Haritha Prabha, Daniel L. Levy
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction: Anal squamous cell carcinoma (ASCC) is a rare but increasingly common gastrointestinal malignancy, mainly associated with oncogenic human papillomaviruses (HPVs). The role of non-coding RNAs (ncRNAs) in tumorigenesis is recognized, but the impact of viral ncRNAs on host gene expression remains unclear. Methods: We re-analyzed total RNA-Seq data from 70 anal biopsies: 31 low-grade squamous intraepithelial lesions (LGSIL), 16 high-grade SIL (HGSIL), and 23 ASCC cases. Microbial composition was assessed taxonomically. Novel viral miRNAs were predicted using vsRNAfinder and linked to host targets using TargetScan and expression correlation analyses. Results: Microbial profiling revealed significant differences in abundance, with Alphapapillomaviruses types 9, 10, and 14 enriched across lesion grades. We identified 90 novel viral miRNAs and 177 significant anti-correlated miRNA–mRNA interactions. Target genes were enriched in pathways related to cell cycle, epithelial–mesenchymal transition, lipid metabolism, immune modulation, and viral replication. Discussion: Our findings suggest that HPV-derived miRNAs, including those from low-risk types, may contribute to neoplastic transformation by modulating host regulatory networks. Conclusion: This study highlights viral miRNAs as potential drivers of HPV-related anal cancer and supports their utility as early biomarkers and therapeutic targets in ASCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911fe59aaa1647471be1a4e931c53c959630dc5" target='_blank'>
              Human Papillomavirus-Encoded microRNAs as Regulators of Human Gene Expression in Anal Squamous Cell Carcinoma: A Meta-Transcriptomics Study
              </a>
            </td>
          <td>
            Daniel J. García, Marco A. Pulpillo-Berrocal, José L. Ruiz, E. Andrés-León, L. Terrón-Camero
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f2b837ac11b06bf8917b835488e0f8a39575dc9" target='_blank'>
              Post-translational modifications in DNA damage repair: mechanisms underlying temozolomide resistance in glioblastoma.
              </a>
            </td>
          <td>
            Yike Chen, Kaikai Ding, Shuyu Zheng, Songting Gao, Xiaohui Xu, Haijian Wu, Fengqi Zhou, Yongjie Wang, Jinfang Xu, Chun Wang, C. Ling, Jing Xu, Lin Wang, Qun Wu, G. Giamas, Gao Chen, Jianmin Zhang, Chenggang Yi, Jianxiong Ji
          </td>
          <td>2025-05-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0fba16907f99aaf84ad95347cb0fae9bffa9f" target='_blank'>
              Biallelic loss-of-function variants in ZNF142 are associated with a robust DNA methylation signature affecting a limited number of genomic loci.
              </a>
            </td>
          <td>
            Mathis Hildonen, A. Ciolfi, Marco Ferilli, Camilla Cappelletti, Chadi Al Alam, David J. Amor, T. S. Barakat, Valérie Benoit, O. S. Birk, B. Callewaert, Ana Cazurro-Gutiérrez, Matthias De Wachter, Martine Doco-Fenzy, P. Gómez-Puertas, T. Hammer, R. Jamra, R. Kaiyrzhanov, Shinichi Kameyama, B. Keren, Christina Kresge, Ilona Krey, Damien Lederer, Í. Marcos-Alcalde, Reza Maroofian, Naomichi Matsumoto, T. Mizuguchi, L. Moey, Angela T. Morgan, Francina Munell, K. Platzer, Beth Pletcher, D. Ros-Pardo, Lynne Rumping, K. Szakszon, K. Van Schil, Edgard Verdura, Julie Vogt, E. Wassmer, Mina Zamani, Z. Tümer, M. Tartaglia
          </td>
          <td>2025-05-23</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Plasmodium parasites, the causative agents of malaria, undergo crucial developments within the mosquito vector, initiated by the formation of male and female gametes. Male gametogenesis involves three rapid rounds of mitosis without nuclear or cell division, followed by a single round of DNA segregation and nuclear division during gamete budding. How the cell organizes the segregation of eight genomes from a single octoploid nucleus into eight haploid gametes is currently unknown. Here we discovered an atypical Arp2/3 complex in Plasmodium important for DNA segregation during male gametogenesis. Unlike the canonical Arp2/3 complex found in other eukaryotes, Plasmodium Arp2/3 localizes to endomitotic spindles and interacts with a kinetochore-associated protein. Disruption of Arp2/3 subunits or actin polymerization interferes with kinetochore–spindle association, causes the formation of subhaploid gametes, and blocks transmission. Our work identified an evolutionary divergent Arp2/3 complex in malaria parasites, provides insights into gametogenesis, and reveals potential targets for transmission-blocking interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0eb8cd0287d7e4de3c5f68362eedb92788d67" target='_blank'>
              An atypical Arp2/3 complex is required for Plasmodium DNA segregation and malaria transmission
              </a>
            </td>
          <td>
            Franziska Hentzschel, David Jewanski, Yvonne Sokolowski, Pratika Agarwal, Anna Kraeft, Kolja Hildenbrand, Lilian P. Dorner, Mirko Singer, Matthias Marti, Friedrich Frischknecht
          </td>
          <td>2025-06-13</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e23526


 Background:
 Kaposi sarcoma (KS) is a tumor caused by Kaposi sarcoma herpesvirus (KSHV), also known as human herpesvirus 8 (HHV-8). Unlike other virus-associated tumors, where disease progression often continues even after viral eradication, KS requires the persistent presence of KSHV for disease progression. This unique dependency underscores the potential of strategies targeting the virus to also target the tumor. This study aims to characterize the expression of KSHV viral proteins and identify factors associated with differences in viral gene expression, ultimately formulating treatment targets.
 Methods:
 A total of 43 unique samples were collected at a single academic institution between 2001 and 2024. RNA was extracted from formalin-fixed paraffin-embedded (FFPE) tissue blocks and subjected to bulk RNA sequencing. Reads were mapped to the human genome (GRCh38.p14) using STAR and to the KSHV genome (NC_009333) using Salmon. Transcript abundance in units of Transcripts Per Million (TPM) and estimated read counts to each transcript was quantified with Salmon. Clinical and demographic data from the time of sample collection were extracted from medical records. Differential gene expression analysis, PCA using DESeq2 and hierarchical clustering were done in R.
 Results:
 The 43 samples analyzed were derived from 25 patients, with 23 (92%) being male. Of the 18 patients with known HIV status, 4 (22%) were HIV-positive. None had a history of organ transplantation. Among the 14 patients with available disease status, 5 (38%) had disseminated disease at the time of sample collection. KSHV gene expression was detected in 42 out of 43 samples. The most consistently expressed genes across all samples were ORF75 (42 samples, proximal to the poly-A tail), ORF72/v-cyclin (42 samples), and ORF73/LANA (39 samples). A pronounced pattern of high expression was observed in the latency-associated region of the KSHV genome, which was conserved across all KS tumors. The three clusters of samples generated from hierarchical clustering exhibited different lytic and latent gene profile (lytic, latent, mixed), which are similar to previous studies. The differentially expressed genes between different tumor morphology or HIV status of the host showed a mix profile of lytic and latent genes.
 Conclusions:
 This study highlights the latency-associated region of KSHV as the most consistently and highly expressed region, regardless of baseline characteristics, presenting a potential conserved therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69dfe0589aae838d9a59485221f8f4d5df8fd8a5" target='_blank'>
              Transcriptomic landscape of Kaposi sarcoma: Insights into therapeutic targeting of KSHV.
              </a>
            </td>
          <td>
            Yuewei Fei, P. Costa, M. Junejo, Michelle Li, Curtis J. Perry, William Damsky, J. Ishizuka
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, Dariusz Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>