<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-24 09:34:01 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>41</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>86</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>121</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>382</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>331</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres and segregate stochastically during cell division3–6 yet persist over many generations. EcDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei over 40 years ago with unknown mechanism3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. We develop Retain-seq, a genome-scale assay that reveals thousands of human retention elements conferring generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromatin conformation capture show that retention elements physically interact with mitotic chromosomes at regions which are mitotically bookmarked by transcription factors and chromatin proteins, intermolecularly recapitulating promoter-enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated; targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, suggesting that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, revealing the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Emergence of drug resistance is the main cause of therapeutic failure in patients with high-grade serous ovarian cancer (HGSOC)1. To study drug resistance in patients, we developed CloneSeq-SV, which combines single-cell whole-genome sequencing2 with targeted deep sequencing of clone-specific genomic structural variants in time-series cell-free DNA. CloneSeq-SV exploits tumour clone-specific structural variants as highly sensitive endogenous cell-free DNA markers, enabling the relative abundance measurements and evolutionary analysis of co-existing clonal populations over the therapeutic time course. Here, using this approach, we studied 18 patients with HGSOC over a multi-year period from diagnosis to recurrence and showed that drug resistance typically arose from selective expansion of a single or small subset of clones present at diagnosis. Drug-resistant clones frequently showed interpretable and distinctive genomic features, including chromothripsis, whole-genome doubling, and high-level amplifications of oncogenes such as CCNE1, RAB25, MYC and NOTCH3. Phenotypic analysis of matched single-cell RNA sequencing data3 indicated pre-existing and clone-specific transcriptional states such as upregulation of epithelial-to-mesenchymal transition and VEGF pathways, linked to drug resistance. In one notable case, clone-specific ERBB2 amplification affected the efficacy of a secondary targeted therapy with a positive patient outcome. Together, our findings indicate that drug-resistant states in HGSOC pre-exist at diagnosis, leading to positive selection and reduced clonal complexity at relapse. We suggest these findings motivate investigation of evolution-informed adaptive treatment regimens to ablate drug resistance in future HGSOC studies. A combined sequencing technique assesses 18 patients with high-grade serous ovarian cancer over a multi-year period from diagnosis to recurrence and shows drug resistance typically arises from selective expansion of one or a few clones present at diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, C. Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, , Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, , J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Oncogenesis involves widespread genetic and epigenetic alterations, yet the full spectrum of somatic variation genome-wide remains unresolved. We generated a near-telomere-to-telomere (T2T) diploid assembly of a donor paired with deep short- and long-read sequencing of their melanoma. This revealed that 16% of somatic variants occur in sequences absent from GRCh38, with satellite repeats acting as hotspots for UV-induced damage due to sequence-intrinsic mutability and inefficient repair. Centromere kinetochore domains emerged as focal sites of structural, genetic, and epigenetic variation, leading to remodeling of centromere kinetochore binding domains during tumor evolution. Single-molecule telomere reconstructions uncovered cycles of attrition, deletion, and telomerase-mediated extension that shape cancer telomeres. Finally, diploid chromatin maps exposed that copy number alterations and epimutations, rather than point mutations, predominate in rewiring cancer regulatory programs. These findings define the full landscape of a cancer’s somatic variation and their functional impact, establishing a blueprint for T2T studies of mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, A. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, A. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Aneuploidy, an abnormal number of chromosomes, is a hallmark of cancer and has been proposed as an initiating event in tumorigenesis. In glioblastoma (GBM), a highly aggressive brain tumor, cells almost universally display gain of chromosome 7 and loss of chromosome 10. However, it remains unclear whether these alterations arise de novo during malignant transformation or reflect pre-existing chromosomal instability in normal brain tissue. Here, we used single-nucleus whole-genome sequencing (snWGS) on 225 NeuN-negative (non-neuronal) cortical nuclei from 12 healthy individuals and 6 GBM patients, including matched tumor cores and non-tumor brain regions. In healthy brains, approximately 15% of glial nuclei harbored somatic aneuploidies, most often involving chromosome arms, with recurrent 16p alterations detected in up to 3% of nuclei from both healthy controls and GBM non-tumor tissue. These findings establish 16p is a hotspot of structural variation in adult glia. Non-tumor regions in GBM patients closely resembled healthy controls in aneuploidy burden and chromosomal instability metrics and lacked hallmark tumor alterations. In contrast, GBM tumors exhibited significantly elevated aneuploidy (~50%), enrichment for canonical chromosomal instability-driven events, and sex-specific karyotype patterns, consistent with transformation-associated chromosomal instability. Thus, aneuploidy is a recurrent but constrained feature of normal adult glia, whereas chromosome instability and GBM-defining aneuploidies emerge only during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ca098d5f35d207827ccedcf67f15fa391301a1" target='_blank'>
              The Somatic Aneuploidy Landscape of Adult Glia Reveals 16p as a Hotspot and Differentiates Mosaicism in Normal Glia from Chromosomal Instability in Glioblastoma
              </a>
            </td>
          <td>
            Cristina Montagna, Olivia Albert, Shixiang Sun, Jhih-Rong Lin, Moonsook Lee, Chang Chan, Alex Maslov, Lisa Ellerby, Anita Huttner, Zhengdong Zhang, J. Vijg
          </td>
          <td>2025-11-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Telomeres are essential for genome integrity, but the accurate, high-resolution mapping of their three-dimensional (3D) chromatin interactions, a process thought to mediate gene regulation and telomere maintenance, has been limited by the repetitive nature of subtelomeric DNA and shortcomings of traditional sequencing-based chromatin conformation capture methodologies. To overcome these challenges, we developed Telomere Conformation Capture by sequencing (Telomere-C), a novel high-throughput approach for the genome-wide identification of telomeric chromatin interactions. We applied Telomere-C to three normal fibroblast cell lines (BJ, IMR90, WI38), one telomerase-positive cancer cell line (HeLa S3) and two cancer cell lines (U2-OS and WI38-VA13) that maintain telomeres by alternative lengthening of telomeres (ALT). We generated millions of telomere-associated reads per sample and identified up to 41,026 interaction peaks. We successfully validated previously-described telomeric interaction sites (e.g., TERT and DUX4), and critically, revealed a novel class of ultra-long-range interactions extending over 5 Mb away from the telomere, which surprisingly constitute over 80% of all identified contacts. Contrary to prior studies focusing on protein coding gene interactions, we discovered that the telomeric 3D interactome is overwhelmingly anchored at repetitive element hubs, particularly interstitial telomeric sequence (ITS), telomere-associated repeat 1 (TAR1) and D20S16 elements. We found that the clustering of these interactions correlates strongly with cell-type-specific telomere maintenance. Most notably, D20S16 telomeric interactions were uniquely and highly enriched in ALT cancer cells, suggesting a mechanistic link. Taken together, our study effectively constructs the first high resolution maps of the telomeric 3D interactome, redefining its scope to be dominated by ultra-long-range contacts with repetitive elements. This work provides fundamental insights into telomeric nuclear organization and establishes the telomeric D20S16 interaction as a molecular signature for the ALT pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="It has been more than 20 years since the publication of the C57BL/6J mouse reference genome, which has been a key catalyst for understanding the biology of mammalian diseases. However, the mouse reference genome still lacks telomeres and centromeres, contains 281 chromosomal sequence gaps and only partially represents many biomedically relevant loci. Here we present the first telomere-to-telomere (T2T) mouse genomes for two key inbred strains, C57BL/6J and CAST/EiJ. These T2T genomes reveal substantial variability in telomere and centromere sizes and structural organization. We thus add an additional 213 Mb of new sequence to the reference genome, which contains 517 protein-coding genes. We also examined two important but incomplete loci in the mouse genome—the pseudoautosomal region (PAR) on the sex chromosomes and KRAB zinc-finger protein loci. We identified distant locations of the PAR boundary, different copy numbers and sizes of segmental duplications and a multitude of amino acid substitution mutations in PAR genes. Telomere-to-telomere assemblies of two mouse inbred strains, C57BL/6J and CAST/EiJ, offer improvements over the current mouse reference genome by adding telomere and centromere sequences that lead to insights into variability in telomere and centromere sizes and organization, and the discovery of 225 and 355 new genes for C57BL/6J and CAST/EiJ, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, T. Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C. Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation. Radiation-induced malignantly transformed cells undergo multi-scale 3D genome reorganization. TP63 acts as a key mediator, creating accessible chromatin sites that form tumour-specific loops activating the MYC oncogene during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) activates pro-inflammatory cGAS/STING signaling, positioning it as a biomarker for combining immune checkpoint blockade (ICB) and PARP inhibition (PARPi). However, the consequences of HRD on the immune landscape across cancers remain unclear. Here, we applied a pan-cancer HRD classifier to >10,000 tumors from The Cancer Genome Atlas and uncovered striking heterogeneity in immune activity. Compared to HR-proficient tumors, HRD tumors showed elevated inflammation in breast, ovarian, and endometrial cancers. These tumors exhibited robust activation of innate and adaptive immune pathways (IFN, NF-κB) and transcriptional hallmarks of senescence, angiogenesis, and adenosine signaling. In contrast, lung, head and neck, and melanoma HRD tumors displayed suppressed inflammation and evidence of immune escape through large-scale loss-of-heterozygosity (LOH) at IFNA/B, STING, and other loci. These tumors also frequently presented HLA LOH and oncogene amplifications, suggesting selection under immune pressure and replication stress. Together, our study resolves HRD tumors into two immune archetypes, immune-inflamed and immune-evasive, linked to chromosomal instability and lineage, informing biomarker-driven evaluation of immune checkpoint blockade/PARPi combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b68316c5391afa0d45aa42e226e37099bf84d45" target='_blank'>
              Tissue specificity and chromosomal alterations shape divergent immune programs in HRD tumors
              </a>
            </td>
          <td>
            D. Gulhan, David Barras, Marco Mina, Eleonora Ghisoni, Yoo-Na Kim, V. Viswanadham, Hu Jin, Florian Huber, K. Homicsko, M. Bassani-Sternberg, G. Ciriello, Peter J. Park, Denarda Dangaj Laniti
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-01</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The mortality rate of ovarian cancer remains disproportionately high compared to its incidence. This is partly due to a high level of intra-tumoral heterogeneity that promotes disease recurrence and treatment failure. In this study, we describe degrees of heterogeneity revealed by single-cell whole genome sequencing and spatial transcriptomics of five epithelial ovarian carcinomas. At the cellular level, we describe pseudo-diploid cells that match the malignant cell population in both somatic variant and copy number patterns. At the clonal and subclonal levels, we describe diversification associated with copy number gains and whole genome doubling. In multi-clonal samples, we infer evolutionary relationships from single cell copy number, loss of heterozygosity analysis, and somatic variant detection, and correlate these with tissue histology and gene expression programs. In one sample, we identify functionally consequential copy number alterations that contribute to molecular diversity, cell proliferation, and inflammation in a minor clone that persisted without major expansion alongside a more complex major clone. In another, we describe a complex evolutionary history including a spontaneous reversion of a driver mutation in a secondary clone, which correlated with a switch in oncogenic expression programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Non-canonical (non-B) DNA secondary structures, e.g., G-quadruplexes and triplex DNA, are mutation hotspots and genome regulators contributing to disease and evolution. Yet they remain uncharacterized in complete genomes in vivo. Here we exploited the fact that many non-B DNA structures form single-stranded DNA (ssDNA). Using permanganate/S1 footprinting across 14 cell lines, we generated ssDNA profiles for human and six non-human ape telomere-to-telomere (T2T) genomes. Newly resolved satellite arrays—e.g., at ribosomal DNA and centromeres—displayed high ssDNA levels, implicating non-B DNA in satellite expansion and function. Hidden Markov Models applied to our ssDNA data revealed active genomic domains with specific functions—e.g., replication, transcription, or recombination—each enriched in particular non-B DNA types. Human-specific ssDNA domains correlated with nervous system genes, whereas cancer and embryonic cells showed increased ssDNA in transposable elements. Our ssDNA analysis across ape T2T genomes uncovered conserved and species-specific DNA structural dynamics central to genome regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2c0bd20d360ee798769e48db594be4df9b6168" target='_blank'>
              Comparative analysis of single-stranded and non-canonical DNA formation in human and other ape cells with telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Jacob P. Sieg, Huiqing Zeng, Hana Pálová, Linnéa Smeds, S. Mohanty, Angelika Lahnsteiner, Francesca Chiaromonte, Kateryna D. Makova
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) assays are being widely adopted in oncology and maternal-fetal medicine. Patterns of cfDNA fragmentation can provide useful information about gene regulation and expression in human disease from a blood draw. Here, we demonstrate that enhancer RNA expression – a marker of enhancer activity – can be inferred from local patterns of cfDNA fragmentation. We define a transcriptional activation score (TAS) that predicts expression of enhancers and genes based on cfDNA fragment sizes and positions near transcriptional start sites (TSSs). The TAS identifies activity of cancer-associated enhancers in patients with cancer, distinguishes clinically relevant cancer subtypes, and identifies activation of enhancers associated with treatment resistance and therapy response. We propose a simple model to account for our findings based on chromatin fiber structure and the depletion of H1 histone proteins near active TSSs. Our model provides a unified framework that reconciles seemingly conflicting observations from prior fragmentomics studies. Broadly, this work enables blood-based assessments of gene regulation in cancer and non-oncologic diseases to inform pathobiology, diagnosis, and treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caacf7712f535881a1313780e0d16e1ccea5aabe" target='_blank'>
              A chromatin fiber model explains cell-free DNA fragmentation signatures of active regulatory elements
              </a>
            </td>
          <td>
            Alexis Yang, G. Lee, Surya B. Chhetri, Razane El Hajj Chehade, Gunsagar S. Gulati, Medha Pandey, Doris Fu, Yoo-Na Kim, Shahabeddin Sotudian, H. Savignano, Sadia D. Shahbazi, Chelsea Philpot, Taylor A. Hoggood, D. Gulhan, M. Kellis, Jacob E Berchuck, S. Baca
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Approximately half of high-grade serous ovarian carcinomas (HGSOCs) demonstrate homologous recombination deficiency (HRD) with characteristic genomic rearrangements. However, the impact of HRD on centromeres and transposable elements remains largely unexplored in HGSOC since conventional short-read sequencing is unable to interrogate these repetitive regions. We employed Oxford Nanopore long-read sequencing (LRS) to investigate genomic and epigenetic alterations in these regions. Pre-treatment archival cryopreserved tumor and matched blood samples were obtained for six patients with HGSOC. High-molecular-weight DNA was sequenced using Oxford Nanopore R10.4 flow cells and aligned to both the GRCh38 and the telomere-to-telomere T2T-CHM13 reference genome. Pathogenic gene mutations, allele-specific copy number variations, structural variants, and CpG methylation were analyzed. All six tumors had pathogenic TP53 mutations. Two carried germline BRCA1 mutations, while three showed CCNE1 amplifications. HRD scores and mutational signatures associated with HRD were elevated in the BRCA1-mutated tumors. Centromeric regions were significantly hypomethylated in tumors and their methylation profiles distinctly separated HRD tumors from non-HRD tumors. LINE1 and ERV transposable elements showed marked hypomethylation in tumors without germline BRCA1 mutations. Chromosome arm-specific telomere lengths were significantly shortened in tumors. Allele-specific hypermethylation in the TERT hypermethylated oncological region was detected in three tumors. LRS uncovered HRD-related genomic and epigenomic alterations in previously inaccessible repetitive regions of HGSOC, including centromeric and transposable element hypomethylation. These findings highlight the potential of such abnormalities as novel biomarkers for HGSOC and warrant further application of the methods to larger cohorts in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a227daba2577321ec904774ceb23e3c2f8c0883" target='_blank'>
              Long read sequencing reveals novel genomic and epigenomic alterations in repetitive regions of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Y Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, Barrett Lawson, R. Hillman
          </td>
          <td>2025-10-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Extrachromosomal circular DNA (EccDNA) is widespread in various heterogeneous tumors and closely associated with tumor resistance and progression. Methods: Circle-seq and mRNA-seq were done on samples from three multiple myeloma (MM) patients, one when they had a complete response and one when they relapsed. Results: A large number of EccDNA molecules were detected with high heterogeneity among the six samples. Circle-seq combined with mRNA-seq analyses revealed that there is no linear relationship between mRNA expression and EccDNA quantity. Chromosome 19 presented the highest density of differentially expressed EccDNA genes, followed by chromosome 17. Only the T3 sample showed del(17p) by fluorescence in situ hybridization at the time of relapse. Approximately 92% of all upregulated EccDNA genes from chromosome 17 (137) were present in the T3 sample. By integrating Circle-seq and mRNA-seq data, we obtained several potentially functional candidate protein-coding genes and miRNAs from chromosome 17. We further assessed the prognostic value of the three protein-coding genes in the MMRF-COMPASS clinical trial and found that all three genes were poor prognostic indicators for MM. Furthermore, WB, CCK8 and Annexin V-FITC/PI assays revealed that the overexpression of 2',3'-cyclic nucleotide 3' phosphodiesterase (CNP) downregulated the apoptotic pathway and increased bortezomib tolerance in MM cells. Conclusion: We performed a Circle-seq analysis of MM and investigated the heterogeneity of EccDNA. Analysis of data from clinical studies and basic experiments revealed that the gene carried on EccDNA most likely contributed to the increased tolerance of bortezomib in MM with del(17p).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0af596876441646ebdb29f843eea2b6d455fc1" target='_blank'>
              Extrachromosomal circular DNA of multiple myeloma
              </a>
            </td>
          <td>
            Fangfang Li, Xinyi Long, Sishi Tang, Jing Liu, Yunfeng Fu
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Alternative DNA conformation formed by sequences called flipons are thought to play an important role in regulating various genomic processes, either repressing or enhancing transcription, chromatin organization, DNA repair, telomere maintenance, RNA splicing, translation, and stress responses. The formation of G-quadruplexes (GQs) has been investigated experimentally using various methodologies with varying degrees of overlap between the results underscoring the need for a gold-standard GQ dataset. With this aim we trained a large language model, GQ-DNABERT using EndoQuad, the most comprehensive human GQ dataset. GQ-DNABERT recalled the training data and predicted de novo GQs in intergenic and intronic regions, enriched for cis-regulatory elements (cCREs) and ATAC-seq peaks. We evaluated the predicted GQ-DNABERT proximal enhancer–promoter (pEP) pairs, using annotations from ENdb, ENCODE, Zoonomia, Chromium multiomics scATAC-seq and scRNA-seq data from normal cells, and cCREs from normal-cancer pairs. We found GQ pEP pairs correlating with gene expression, with some pairings potentially acting as tissue-specific switches. Genes with GQ pEP pairs in cancer cells are enriched in different processes compared to the corresponding normal tissues. Overall, GQ-DNABERT is a valuable tool for extending and harmonizing data collected ex vivo. We demonstrate the usefulness of GQ-DNABERT for investigating transcriptional regulation in single-cell experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57f048fcb6a847ca70df111a471d2de991b3cd9c" target='_blank'>
              GQ-DNABERT reveals GQ proximal enhancer–promoter interactions associated with tissue-specific transcription
              </a>
            </td>
          <td>
            Dmitry Konovalov, Dmitry Umerenkov, Alan Herbert, M. Poptsova
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) is a notable exception to genome integrity, characterized by significant DNA loss during development. In many nematodes, PDE is initiated by DNA double-strand breaks (DSBs), which lead to chromosome fragmentation and subsequent DNA loss. However, the mechanism of nematode DNA breakage remains largely unclear. Interestingly, in the human and pig parasitic nematode Ascaris, no conserved motif or sequence structures are present at chromosomal breakage regions (CBRs), suggesting the recognition of CBRs may be sequence-independent. Using Hi-C, we revealed that Ascaris CBRs engage in three-dimensional (3D) interactions before PDE, indicating that physical contacts of the break regions may contribute to the PDE process. The 3D interactions are established in both Ascaris male and female germlines, demonstrating inherent genome organization associated with the CBRs and to-be-eliminated sequences. In contrast, in the unichromosomal horse parasite Parascaris univalens, transient and pairwise interactions between two neighboring CBRs that would form the ends of future somatic chromosomes were observed only during PDE. Intriguingly, we found specific spatial compartmentalization changes in the Ascaris chromosomes after PDE converts 24 germline chromosomes into 36 somatic ones. Remarkably, Parascaris PDE generates the same set of 36 somatic chromosomes, and the compartment changes following PDE are consistent between the two species. Overall, our findings contribute to a model that the 3D genome facilitates the recognition of sequence-independent break sites for Ascaris PDE. It also suggests an evolutionary and developmentally conserved 3D genome reorganization of nematode somatic chromosomes following PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (∼750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b8c268f4003d4cab2a7a6a760ce10e09bcebe0" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b8e7f8962ad8fcf01d30be8350fefcd7ba1a81" target='_blank'>
              CRISPR live-cell imaging reveals chromatin dynamics and enhancer interactions at multiple non-repetitive loci.
              </a>
            </td>
          <td>
            Meishuo Liu, Keyun Huang, Jie Zhang, Qingyang Li, Xinming Wang, Buming Gu, Hao Tang, Zhenhai Du, Liangjun Hu, Shutao Qi, Yu Ma, Hongtao Yu, Wei Xie, Xianyang Fang, Haifeng Wang
          </td>
          <td>2025-11-06</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The interplay between DNA methylation and super-enhancer activity is emerging as a key focus in cancer research. Super-enhancers, a specialized class of enhancers, strongly activate transcription of their target genes due to their dense clustering with essential transcription factors (TFs) and cofactors. In cancer, especially, these super-enhancers control key oncogenic drivers and often display abnormal DNA methylation patterns that can repress or overexpress target genes in both solid and blood cancers. Furthermore, DNA methylation in the super-enhancer region has been found to influence their regulatory capacity. Although enhancers are typically characterized by low DNA methylation, dysregulated methylation at super-enhancers is seen in most malignancies, affecting TF and chromatin regulator recruitment. Hypomethylation at these sites often accompanies oncogene hyperactivation, while hypermethylation can repress tumor suppressor mechanisms. Recent research highlights DNA methylation as a promising source of cancer biomarkers. This review examines the intricate relationship between DNA methylation and super-enhancer activity in cancer, concentrating on how methylation regulates super-enhancers, modulates oncogene expression, promotes oncogenesis, and serves as a target for novel oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856f95cde56634ccf673f640abf635b4018a59" target='_blank'>
              Super-enhancer DNA methylation in cancer: the mechanism of action and therapeutic directions
              </a>
            </td>
          <td>
            Yue Bi, Kang Zou, Ke Wang
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Bacterial nucleoid-associated proteins (NAPs) structure the chromosome and regulate gene expression, but how these two functions are related is unclear. H-NS is a well-studied NAP that acts as a global gene silencer capable of bridging and looping DNA in vitro. Here, using high-throughput chromosome conformation capture assays, we show that H-NS mediates genome-wide long-range DNA looping in the stationary-phase nucleoid of Escherichia coli. Chromatin immunoprecipitation assays demonstrate that high levels of H-NS are present at the base of DNA loops. Super-resolution imaging and single-particle tracking show that H-NS binds more tightly in stationary phase and compacts the nucleoid mesh. Transcriptomic analyses indicate H-NS represses gene expression more strongly in the looped nucleoid and enables higher expression of genes outside of H-NS-bound regions. Overall, our study demonstrates that H-NS bridges distal DNA regions along the genome upon nutrient limitation, causing reduced nucleoid accessibility, stronger transcriptional repression, and a shifted transcriptional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd28764a1bb58dc6ea970c4ddb741f750a9e8bc" target='_blank'>
              Genome-wide DNA bridging by H-NS reshapes the stationary phase nucleoid and transcriptional landscape
              </a>
            </td>
          <td>
            Lindsey E. Way, Xiaofeng Dai, Emma E Wiesler, Lauren A McCarthy, Zhongqing Ren, Gail G Hardy, D. E. Fuller, Labeeb Hossain, Anne S. Meyer, E. Abbondanzieri, Julie S. Biteen, Xindan Wang
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background/Aim: Lineage plasticity enables prostate cancer cells to bypass androgen receptor (AR) dependence, contributing to metastasis, treatment resistance, and lethality. The developmental transcription factor PROX1 was recently identified as an early driver of lineage plasticity. The TMPRSS2-ERG fusion, a common genomic event in prostate cancer, induces ERG overexpression and aggressive tumor behavior. We investigated the relationship between PROX1 expression and ERG expression in the context of TMPRSS2-ERG fusion in prostate cancer. Materials and Methods: We analyzed data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) dataset (n=492 samples). The UCSC Xena Browser was used to visualize and integrate gene expression, copy number alterations, and fusion status. Complementary analyses were performed using cBioPortal. Genomic instability was assessed using both fraction of the genome altered (FGA) and mutation count. The correlation between PROX1 and ERG expression was evaluated in fusion-positive and fusion-negative tumors. Results: PROX1 expression showed a strong positive correlation with ERG expression in TMPRSS2-ERG fusion-positive tumors (r=0.4, p=3.2×10−16). Fusion-positive samples exhibited elevated PROX1 expression, compared to fusion-negative samples. This pattern suggests that PROX1 induction is associated with ERG-driven transcriptional reprogramming linked to the TMPRSS2-ERG fusion. Conclusion: PROX1 expression is significantly associated with ERG expression and TMPRSS2-ERG fusion status in prostate cancer. These findings reinforce the role of PROX1 as an early marker and potential mediator of lineage plasticity. Targeting PROX1 or its regulatory pathways may offer a novel therapeutic strategy to mitigate plasticity-driven progression and treatment resistance in TMPRSS2-ERG fusion-positive prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c9168d60bbc1c5a4a0d510943f10c27a506bc7" target='_blank'>
              PROX1 Expression Is Significantly Associated With ERG Expression and the TMPRSS2-ERG Fusion Gene in Prostate Cancer
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2025-09-26</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Comprising up to 90% of eukaryotic genomes, transposable elements (TEs) are mobile genetic units that play fundamental roles in evolution. Brown algae, one of the most complex multicellular eukaryotic groups that evolved independently from plants, fungi, and animals, are particularly underexplored in their transposon biology, especially when studied in a developmental context. Here, we explore the TE landscape of the model brown alga Ectocarpus, using a high-quality genome assembly complemented by extensive manual curation. TEs account for 28% of the genome, with a predominance of evolutionarily young elements. DNA transposons represent the most abundant and diverse TE subclass. Notably, TEs are significantly enriched along the sex chromosomes, a pattern potentially driven by local transposition events from the non-recombining sex-determining region into the pseudoautosomal regions. The genome harbors a high density of intronic TEs, which show minimal impact on host gene expression; however, intronic TEs tend to be shorter and more degraded than intergenic copies, suggesting selective pressures on their retention in the genome. Intact and potentially active TEs are preferentially associated with small RNAs and the histone modification H3K79me2, with over 70% of H3K79me2-marked intact TEs also enriched in small RNAs. This stable association indicates tight and sustained silencing of intact TEs throughout the life cycle of Ectocarpus. Our study highlights the genetic diversity of the Ectocarpus mobilome and presents a complex, multilayered landscape of TE regulation mechanisms which involves small RNAs and chromatin modifications in the absence of an epigenetic silencing machinery that would be comparable to animals or plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/656c205c60e9226c179215faa881ebfaff946d52" target='_blank'>
              Characterization of the transposable element landscape shaping the Ectocarpus genome
              </a>
            </td>
          <td>
            Erica Dinatale, Rory J. Craig, Claudia Martinho, Hajk-Georg Drost, Susana M. Coelho
          </td>
          <td>2025-09-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer and is a potential vulnerability that can be selectively targeted. To systematically identify genes that affect the incidence and fitness of aneuploid cells, we conducted a genome-wide CRISPR/Cas9 screen using NMS-P715, an inhibitor of the spindle assembly checkpoint (SAC) kinase Mps1/TTK. In this study, we identified a number of genes known to regulate aneuploidy and mitosis, and subsequently focused on PRR14L, a ubiquitously expressed gene previously implicated in chronic myelomonocytic leukemia (CMML). Proximity labeling of PRR14L using TurboID revealed several cell division proteins, including the PP2A-B56 phosphatase complex and the spindle assembly factor TACC3, as PRR14L-interacting proteins. Loss of PRR14L prolongs SAC-dependent mitotic arrest in response to microtubule depolymerization but, paradoxically, leads to catastrophic mitotic errors upon SAC abrogation by MPS1 inhibitors. A model derived from our findings provides a rationale for exploiting MPS1 inhibition as a potential vulnerability in cancers containing either PRR14L loss of function mutations or FGFR-TACC3 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The distinctive three-dimensional (3D) chromatin architecture adopted by chromosomes during meiosis is essential for fertility, yet the functional implications of the fine-scale spatial organization remain poorly understood. Here, we investigate the impact of RAD21L deletion, a meiosis-specific cohesin subunit, on the 3D genome architecture and gene expression in the male germ line. Using fluorescence-activated cell sorting, high-throughput chromosome conformation capture, and single-cell RNA sequencing, we demonstrate that RAD21L deficiency impairs meiotic chromatin organization and inter/intrachromosomal interactions. This conspicuous 3D genome reorganization also disrupts bouquet formation, resulting in increased telomeric interactions between heterologous chromosomes in primary spermatocytes. Genome reorganization was accompanied by detectable transcriptional dysregulation in spermatogonia and primary spermatocytes, mainly affecting sex chromosomes. Collectively, our findings establish the cohesin RAD21L as a germline-specific critical regulator of genome-wide 3D genome reorganization during spermatogenesis and a modulator of the male germline transcriptional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2272faa3ad286c58e292f1de1607d234286a48c" target='_blank'>
              Meiotic cohesin RAD21L shapes 3D genome structure and transcription in the male germline
              </a>
            </td>
          <td>
            Laia Marín-Gual, Covadonga Vara, Raquel Sainz-Urruela, Yasmina Cuartero, Lucía Álvarez-González, Natalia Felipe-Medina, Francisca Garcia, E. Llano, M. Martí-Renom, A. Pendás, A. Ruiz-Herrera
          </td>
          <td>2025-10-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC) is a stable epigenomic mark associated with gene regulation and tumorigenesis. To evaluate its potential for multi-cancer detection, epigenomic distribution of 5hmC was analyzed in tumor tissues (n = 217), normal tissues (n = 50), and cell-free DNA (cfDNA) from cancer (n = 1009) and non-cancer subjects (n = 1678) across breast, colon, lung, ovarian, and pancreatic cancers. Differential analysis revealed extensive redistribution of 5hmC in early-stage tumors that persisted into the late-stage, while global 5hmC abundance decreased in all tumors. Two distinct patterns of hydroxymethylation distribution were identified: one common across cancers and another tissue specific. Differential hydroxymethylation regions (DhmRs) that were tumor tissue specific had significantly improved cancer prediction in cfDNA. A multinomial logistic regression model was trained on tissue-specific DhmRs to predict Tissue of Tumor Origin (TOTO) of cfDNA with 85.2% accuracy. These findings demonstrate that stable, cancer-specific 5hmC signatures appear to be established early in tumorigenesis persist throughout disease progression, suggesting 5hmC as a potential pan-cancer marker for early and late-stage tumor detection via liquid biopsy. Analysis of 5hmC in tumor tissues and cfDNA reveals concordant cancer signatures that show potential for tissue origin prediction via liquid biopsy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27052f03426a9d599b2be2b2dba739f7a1a6978" target='_blank'>
              5-hydroxymethylcytosine analysis reveals stable epigenomic changes in tumor tissue that enable cancer detection in cell-free DNA
              </a>
            </td>
          <td>
            Yuan Xue, Y. Ning, V. Friedl, David Haan, Anna Bergamaschi, G. Guler, K. Hazen, Aaron Scott, T. Phillips, E. McCarthy, Chris Ellison, R. Malta, A. Nguyen, Vanessa Lopez, R. Cavet, M. Peters, J. Sheard, William J Gibb, Shimul Chowdhury, Wayne Volkmuth, S. Levy
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155a629eff7b8ba829fe935812c535143d1302b5" target='_blank'>
              Rewiring cancer epigenome: lncRNA as modulator of chromatin architecture and neoplastic transformation
              </a>
            </td>
          <td>
            Santosh Kumar, S. Kundu, S. Sharawat, Ashok Sharma
          </td>
          <td>2025-11-19</td>
          <td>Mammalian Genome</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) selectively removes portions of the genome during development, deviating from the genome constancy rule. Our understanding of PDE mechanisms, including the involvement of small RNAs and transposases, is mainly based on studies in the single-cell ciliates, whereas in metazoans, the PDE mechanisms remain largely unknown. In the parasitic nematode Ascaris, PDE removes and remodels all germline chromosome ends, splits fused chromosomes into somatic chromosomes, leading to dramatic karyotype changes. To probe the possible involvement of small RNAs in Ascaris PDE, we identified two Argonautes, AsWAGO-2 and AsWAGO-3, associated with PDE. Using cellular fractionation followed by nuclease-treated immunoprecipitation, we characterized WAGO-associated small RNAs (siRNAs) and their targets in chromatin, nucleoplasm, and cytoplasm. We found that AsWAGO-3 is enriched on eliminated chromatin, with its associated siRNAs mostly targeting genes. However, a subset of AsWAGO-3 siRNAs targets repeats in the eliminated regions specifically located in the middle of chromosomes that originated from the ends of ancestral chromosomes. In contrast, AsWAGO-2 transiently stains the retained chromosomes during elimination mitoses, and the associated siRNAs are enriched in the retained regions only in the chromatin fraction. Overall, our data established a link between WAGOs and Ascaris PDE, demonstrating the flexibility of small RNA pathways that selectively distinguish retained and eliminated DNA. We suggest that some of the targeting may be a response to the karyotype changes associated with chromosome fusion and PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f77d30c5b35d3bee66492e606ea00f82e5f1ceb" target='_blank'>
              Argonautes and small RNAs associated with nematode programmed DNA elimination
              </a>
            </td>
          <td>
            M. Zagoskin, Yuanyuan Kang, Richard E Davis, Jianbin Wang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a primary driver of cervical cancer. Integration of HPV into the human genome causes persistent expression of viral oncogenes E6 and E7, which promote carcinogenesis and disrupt host genomic function. However, the impact of integration on host gene expression remains incompletely understood. We used multimodal RNA sequencing, combining total RNA-seq and Cap Analysis of Gene Expression (CAGE), to clarify virus–host interactions after HPV integration. HPV-derived transcripts were detected in 17 of 20 clinical samples. In most specimens, transcriptional start sites (TSSs) showed predominant early promoter usage, and transcript patterns differed with detectable E4 RNA region. Notably, the high RNA expressions of E4 region and viral-human chimeric RNAs were mutually exclusive. Chimeric RNAs were identified in 13 of 17 samples, revealing 16 viral integration sites (ISs). CAGE data revealed two patterns of TSS upregulation centered on the ISs: a two-sided pattern (43.8%) and a one-sided pattern (31.3%). Total RNA-seq showed upregulation of 12 putative cancer-related genes near ISs, including MAGI1-AS1, HAS3, CASC8, BIRC2, and MMP12. These findings indicate that HPV integration drives transcriptional activation near ISs, enhancing expression of adjacent oncogenes. Our study deepens understanding of HPV-induced carcinogenesis and informs precision medicine strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48263c0732371718c3701a40772227c4abc1885c" target='_blank'>
              Elucidating Alterations in Viral and Human Gene Expression Due to Human Papillomavirus Integration by Using Multimodal RNA Sequencing
              </a>
            </td>
          <td>
            Kana Tamai, S. Kinjo, Ayumi Taguchi, K. Nagasaka, Daisuke Yoshimoto, Anh Quynh Duong, Yoko Yamamoto, Hitoshi Iuchi, M. Mori, K. Sone, Michiaki Hamada, Kei Kawana, Kazuho Ikeo, Yasushi Hirota, Yutaka Osuga
          </td>
          <td>2025-10-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation - DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1655457ebd077b5d78f8c0d2ef963c19750b7b0" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris.
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-11-12</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Repetitive DNA sequences, as transposable elements (TEs) and satellite DNA (satDNA) spread and diversify within host genomes, impacting genome biology in numerous ways. In the first part of this review, we emphasize the evolutionary importance of satDNAs and TEs, providing a short summary of their roles and the mechanisms by which they influence the structure and function of genomes. We also discuss the broad, complex, and extensive relationships between TEs and satDNAs. Following that, we bring together different mechanisms on the generation of satDNA from TE, as it has been demonstrated that almost any part of any type of TE can undergo tandemization and produce novel satDNAs. Importantly, we here present a hypothesis that would explain the existence of particular types of monomers, namely composite satDNA monomers which display multiple subsequent stretches of similarity to various TEs, for which the explanation was lacking so far. We propose that even highly shuffled and degraded TE remnants residing in heterochromatin ‘TE graveyards’ can give rise to new satDNA sequence monomers, transforming these genomic loci into DNA ‘recycling yards’. Furthermore, we emphasize important evolutionary questions regarding the causes, mechanisms, and frequency of these occurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f67b41471ebec0a1b45c2936430e5e0e7ad266" target='_blank'>
              Satellite DNAs rising from the transposon graveyards
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Patrik Majcen, Miroslav Plohl
          </td>
          <td>2025-09-30</td>
          <td>DNA Research: An International Journal for Rapid Publication of Reports on Genes and Genomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms. Cancer cell plasticity is often linked to treatment failure. Here, the authors discuss the role of epigenome in DNA repair and, how DNA repair-induced chromatin changes influence cancer cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d85f000ce66e3391a720ff68ea8a6722807028f" target='_blank'>
              TAOK1 regulates chemo- and radiosensitivity in BRCA1/2-deficient tumors
              </a>
            </td>
          <td>
            Lea Lingg, M. Mutlu, A. Naguleswaran, I. Klebic, E. Gogola, Dilara Akhoundova Sanoyan, Pascale Martine Schürch, Raviprasad Kuthethur, Carmen Fonseca, Simone de Brot, M. Everts, Martín González-Fernández, Saskia Hussung, E. Vassella, M. V. van Vugt, Arnab Ray Chaudhuri, Jos Jonkers, Sven Rottenberg, P. Francica
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Despite shared genetic driver alterations and histology, the genomic fidelity of most mouse tumor models, including those genetically engineered (GEMM), to their human counterparts is unknown. In this study, we developed MiMouse, a mouse comprehensive genomic profiling panel for high-throughput credentialing applicable to routine formalin-fixed, paraffin-embedded tumors. Through simulation/validation, we focused on considerations for cross-species mutation prioritization, strain determination, and aneuploidy detection. Using MiMouse, we profiled >250 tumors from high-grade serous carcinoma GEMMs based on conditional inactivation of Brca1 (B), Trp53 (P), Pten (Pt), Rb1 (R), and/or Nf1 (N) and a colorectal carcinoma GEMM based on conditional inactivation of Apc, Kras, and/or P. We confirmed increased genomic instability in high-grade serous carcinoma tumors, with BPPt cancers having both the shortest latency and the least genomic instability. In colorectal cancer, focusing on fidelity to human colorectal cancer aneuploidy events, our results highlighted the critical importance of synteny in transgenic studies, as not only was loss of mouse chromosome 18 (containing the tumor suppressor gene Smad4) a significant aneuploidy event (18%), additional tumors harbored focal Smad4 copy loss, potentially due to the mouse-specific proximity of Apc (mouse and human chromosomes 18 and 5, respectively). Likewise, mouse chromosome 5, the only significantly gained (46%) chromosome in our colorectal cancer models, has syntenic blocks from human chromosomes 7p, 7q, and 13q, including Cdx2, which is both a lineage-specific colorectal cancer oncogene and the colorectal cancer GEMM promoter source. Given the importance of mice to translational cancer research, this study highlights the considerations and utility of approaches for comprehensive genomic credentialing. Significance: The genomic fidelity of most mouse tumor models is unknown. Considering cross-species issues, we develop MiMouse for high-throughput genomic credentialing and profile >250 tumors from fallopian tube and colorectal tumor models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc16f185cf42f303d66c6b3a8187a11fa08b90af" target='_blank'>
              Development and Application of MiMouse, a Comprehensive Genomic Profiling Panel for Credentialing Mouse Tumor Models
              </a>
            </td>
          <td>
            Kevin Hu, C. Liu, Zhaoping Qin, Aaron M Udager, M. Cieslik, S. Tomlins
          </td>
          <td>2025-10-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The human genome attains an amazing spatial organization in the packaging of 2 m of DNA into a 10-μm nucleus. Such structural organization is achieved by the folding of chromatin and the regulation exerted by architectural proteins such as insulators. Chromatin insulators are boundary elements of the genome that, through enhancing blocking activities, demarcation of chromatin domains, and chromatin looping, regulate transcription. The review focuses on the identification and characterization of insulators in various species, discussing mainly the functions of the CCCTC-binding factor (CTCF) in mammals and functionally equivalent insulator proteins in Drosophila melanogaster. We review here the mechanisms of enhancer blocking, barrier activity, and loop extrusion, emphasizing their effects on topologically associating domains and chromatin architecture. Furthermore, we discuss new concepts that have come into prominence: tethering elements and redundancy among the insulator proteins, which contribute to chromatin organization. Advances in methodology, including chromosome conformation capture and high-resolution imaging techniques, have transformed our view of the dynamic interplay between the architecture of chromatin and transcription regulation. This review discusses the importance of insulators for genome organization and describes future directions in investigating their roles in both gene regulation and three-dimensional genomic architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eda983a70eb60f914e84aedb4a80af4895d808" target='_blank'>
              Chromatin insulators in gene regulation and 3D genome organization.
              </a>
            </td>
          <td>
            Hina Sultana, Rohit Kunar, A. G. Matera
          </td>
          <td>2025-10-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Upon DNA breakage, a genomic locus undergoes alterations in three-dimensional chromatin architecture to facilitate signaling and repair. Although cells possess mechanisms to repair damaged DNA, it is unknown whether the surrounding chromatin is restored to its naïve state. We show that a single DNA double-strand break (DSB) within a topologically associated domain (TAD) harboring conformation-sensitive genes causes lasting chromatin alterations, which persist after completion of DNA repair and feature topological rearrangements and loss of local RNA species. These newly acquired features of postrepair chromatin are transmitted to daughter cells and manifest as heritable impairments of gene expression. These findings uncover a hitherto concealed dimension of DNA breakage, which we term postrepair chromatin fatigue and which confers heritable impairment of gene function beyond DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70850e6eacefc0378a65ec703a07a0084a7eaca" target='_blank'>
              Repair of DNA double-strand breaks leaves heritable impairment to genome function.
              </a>
            </td>
          <td>
            Susanne C. S. Bantele, Irene Mordini, A. Biran, Nicolás Alcaraz, Gijs Zonderland, Alice Wenger, Nils Krietenstein, Anja Groth, Jiri Lukas
          </td>
          <td>2025-11-06</td>
          <td>Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea4b63fc938ba67b178443696fb6b3e033db699" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, 
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3066257378e89ecfacf6bd297cae20bfa3bfa3db" target='_blank'>
              p53 activity licenses transcriptional regulation by YAP/TAZ-TEAD to shape expression landscapes during tumorigenesis
              </a>
            </td>
          <td>
            Cole Martin, William B. Sullivan, Scott Bang, Joseph R. Gomes, A. C. Edwards, C. Der, Cheng Fan, Yusha Liu, John P. Morris
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517574b6bc8a73f48a3a9ee64c0be54874afb37" target='_blank'>
              TET CpG sequence context specific DNA demethylation shapes progression of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Y. Hoogstrate, S. Ghisai, L. van Hijfte, R. Head, I. D. de Heer, Marta Padovan, Maurice de Wit, Wies R. Vallentgoed, A. Dipasquale, M. Wijnenga, Bas Weenink, R. Luning, S. Maas, Adela Brzobohata, Michael Weller, Tobias Weiss, M. Mair, A. Berghoff, A. Wöhrer, A. Jeltsch, J. Koekkoek, H. M. Hazelbag, M. Kouwenhoven, Yongsoo Kim, Bart A. Westerman, B. Ylstra, A. Niers, K. Johnson, F. Varn, R. Verhaak, Mustafa Khasraw, M. J. van den Bent, Pieter Wesseling, P. French
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Exonuclease EXO1 performs multiple roles in DNA replication and DNA damage repair (DDR). However, EXO1 loss is well-tolerated, suggesting the existence of compensatory mechanisms that could be exploited in DDR-deficient cancers. Using CRISPR screening, we find EXO1 loss as synthetic lethal with many DDR genes somatically inactivated in cancers, including Fanconi Anaemia (FA) pathway and BRCA1-A complex genes. We also identify the spliceosome factor and tumour suppressor ZRSR2 as synthetic lethal with loss of EXO1 and show that ZRSR2-deficient cells are attenuated for FA pathway activation, exhibiting cisplatin sensitivity and radial chromosome formation. Furthermore, FA or ZRSR2 deficiencies depend on EXO1 nuclease activity and can be potentiated in combination with PARP inhibitors or ionizing radiation. Finally, we uncover dysregulated replication-coupled repair as the driver of synthetic lethality between EXO1 and FA pathway attributable to defective fork reversal, elevated replication fork speeds, post-replicative single stranded DNA exposure and DNA damage. These findings implicate EXO1 as a synthetic lethal vulnerability and promising drug target in a broad spectrum of DDR-deficient cancers unaddressed by current therapies. EXO1 performs multiple roles in DNA replication and DNA damage repair (DDR), but its role in DDR-deficient cancers remains unclear. Here, the authors find EXO1 loss as synthetic lethal with many DDR genes involved in various cancers, including genes from Fanconi Anaemia pathway, BRCA1-A complex, and spliceosome factor ZRSR2; such interactions represent potential clinical targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef09b9a20a2a66068a4fe7db533a048d468729c" target='_blank'>
              EXO1 as a therapeutic target for Fanconi Anaemia, ZRSR2 and BRCA1-A complex deficient cancers
              </a>
            </td>
          <td>
            Marija Maric, Sandra Segura-Bayona, Raviprasad Kuthethur, Tohru Takaki, V. Borel, Tyler H Stanage, Miroslav P Ivanov, Nishita Parnandi, Graeme Hewitt, Rhona Millar, Carmen Fonseca, Harshil Patel, Miriam Llorian, Scott Warchal, Michael Howell, A. Chaudhuri, Panagiotis Kotsantis, S. Boulton
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Highlights What are the main findings? ALT functions as a context-dependent double-edged sword: it drives aggressiveness in rapidly proliferating cancers but imposes replicative and metabolic stress in slow-growing, immune-cold tumors such as glioblastoma and chondrosarcoma. Loss of ATRX/DAXX and ALT activation creates unique DNA repair vulnerabilities, including hypersensitivity to ATR and PARP inhibition, opening avenues for targeted therapy. What is the implication of the main finding? Refines prognostic interpretation of ALT positivity—transforming it from a uniformly negative marker into a tumor-type-specific indicator of clinical outcome. Positions ALT as a precision oncology target, where integrated biomarkers (C-circle as say, APB detection, ATRX/DAXX mutation profiling) enable patient stratification and therapeutic decision-making. Abstract Telomere maintenance enables unlimited cell proliferation by counteracting telomere erosion. While the majority of tumors activate telomerase, a significant subset—approximately 10–15%—utilizes alternative lengthening of telomeres (ALT), a recombination-based mechanism. ALT-positive cancers are classically associated with genomic instability, anaphase bridges, chromosomal rearrangements, and resistance to DNA-damaging therapies. This process is closely associated with genetic instability, which contributes to chromosomal rearrangements and tumor evolution. Consequently, ALT has traditionally been considered an adverse prognostic marker in aggressive malignancies such as osteosarcoma, pancreatic neuroendocrine tumors, and high-grade sarcomas. Paradoxically, recent evidence demonstrates that ALT positivity correlates with improved survival in glioblastoma (GBM) and chondrosarcoma, two tumor types that have historically been regarded as immune-cold and therapeutically intractable. This favorable outcome likely reflects a convergence of factors, including replication stress and DNA damage that impose a fitness cost in slow-growing or metabolically constrained tumors. Loss of ATRX/DAXX, while enabling ALT, further amplifies chromatin fragility, and ALT-mediated instability may paradoxically enhance immunogenicity within immune-quiescent microenvironments. Moreover, ALT-positive cells exhibit unique therapeutic vulnerabilities, particularly to ATR and PARP inhibitors. Together, these observations support a context-dependent model in which ALT functions as a double-edged sword, acting as a driver of malignant aggressiveness in rapidly proliferating cancers while serving as a relative liability in slower-growing, immune-cold tumors. Understanding this duality not only refines prognostic stratification but also opens opportunities for precision oncology. By integrating ALT-specific biomarkers into clinical workflows and exploiting ALT-related DNA repair dependencies, clinicians may transform a once uniformly negative prognostic factor into an actionable therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6ed85bfa1a7b22ce5afc4a049e72d82de6bf4" target='_blank'>
              Alternative Lengthening of Telomeres: A Prognostic Paradox in Cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary Enhancer RNAs (eRNAs) are long non-coding RNAs that activate specific genes by interacting with enhancer regions. In gliomas, they are increasingly recognized as molecular regulators of tumor growth, treatment resistance, and patient outcomes. This review summarizes current evidence on glioma-associated eRNAs and their clinical implications. Several eRNAs, including TMZR1-eRNA and LINC02454, influence sensitivity to temozolomide (the main chemotherapeutic agent for glioma) by modulating STAT3 and DDR1 signaling. Others, such as HOXDeRNA and CRNDE, correlate with tumor aggressiveness and survival. By linking enhancer activity to tumor behavior, eRNAs represent promising biomarkers for diagnosis, disease monitoring, and personalized therapy development in neuro-oncology. Abstract Background: Enhancer RNAs (eRNAs), a subclass of long non-coding RNAs transcribed from enhancer regions, have emerged as dynamic regulators of gene expression, tumor progression, and therapeutic response. In gliomas, their biological and clinical significance is only recently being elucidated. This systematic review aimed to synthesize current evidence regarding the role of eRNAs in gliomagenesis, chemoresistance, and prognosis. Methods: We conducted a systematic review following PRISMA 2020 guidelines. PubMed/MEDLINE and Scopus databases were searched on September 2025 using a predefined strategy. Eligible studies included clinical or pre-clinical analyses of eRNAs in gliomas, reporting associations with tumorigenicity, survival, or resistance to temozolomide (TMZ). Risk of bias was assessed using ROBINS-I (Version 2), and findings were qualitatively synthesized. Results: From 26 retrieved records, 10 studies were included, encompassing 22 unique eRNAs. Two studies demonstrated that TMZR1-eRNA and LINC02454* modulate TMZ sensitivity by regulating STAT3, SORBS2, and DDR1 pathways. Seven studies evaluated prognostic implications: 12 eRNAs (e.g., AC003092.1, CYP1B1-AS1, CRNDE) were consistently associated with poor survival, while seven (e.g., LINC00844, ENSR00000260547) correlated with favorable outcomes, particularly in low-grade gliomas. One mechanistic study showed that HOXDeRNA directly promotes gliomagenesis by displacing PRC2 repression at key transcription factor promoters and activating oncogenic super-enhancers. Conclusions: eRNAs are not passive transcriptional by-products but active modulators of glioma biology. They influence tumor initiation, therapeutic resistance, and survival outcomes, underscoring their potential as prognostic biomarkers and therapeutic targets. Future research should validate these findings in larger clinical cohorts and explore strategies for eRNA-directed therapies in precision neuro-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc4bce08879520ae703d36596d7175b40d9d4f73" target='_blank'>
              Unveiling Enhancer RNAs in Gliomas: A Systematic Review and Qualitative Synthesis
              </a>
            </td>
          <td>
            Matteo Palermo, Giovanni Pennisi, Benedetta Burattini, P. Bruzzaniti, A. Talacchi, Alessandro Olivi, C. Sturiale
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf3ca6346229918f35167d16749694c1e3b771aa" target='_blank'>
              STAG2 cohesin cooperates with DREAM to maintain quiescence and suppress tumourigenesis in the urothelium
              </a>
            </td>
          <td>
            Maria Ramal, E. Lapi, M. Kalisz, Jaime Martínez de Villarreal, Meritxell Novillo-Font, M. Marqués, E. Zarzuela, Marta Isasa, Osvaldo Graña-Castro, Jianming Xu, Ana Cayuela, François Radvanyi, Mireia Novell, Jorge Martínez-Torrecuadrada, M. Barallobre, I. Bernard-Pierrot, María Marti-Marimon, Maria L Arbonés, M. Marti-Renom, Francisco X. Real
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The complications with identifying cells at the origin of cancer and tracking their early divisions impede studies of cancer initiation. Recently, it was shown that some DNA lesions generated by a pulse of damage-inducing mutagen persist over multiple rounds of replication. Segregation of DNA lesions in the early genealogy of an expanding clone leaves a statistically interpretable footprint of cancer initiating events. Specifically, it allows for estimating the number of cell divisions between the initiating DNA lesion and the most recent common ancestor of the tumor. Here, we analyze footprints of segregating lesions from a previously published experimental mouse system, as well as post-chemotherapy human metastatic tumors and the blood of chemotherapy treated patients. In all contexts, clones tend to start early, usually within the span of 4 cell generations from mutagen exposure. Using a branching process model, we show that fitness advantage of early cancer drivers exceeds 30%, with each early division leading to at least 1.3 self-renewing cells. We highlight an example of a blood-derived single cell phylogeny with major subclones separated by just two cell divisions. Broadly, our approach allows inference of tumor initiation and growth parameters based on events preceding the most recent common ancestor of the initiating clone as opposed to characteristics of fully grown tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d535d363c465b65e62fbbcdfa920c1c51d8d6da" target='_blank'>
              Segregating DNA lesions point to high selective advantage of tumor initiating cells
              </a>
            </td>
          <td>
            Vladimir Seplyarskiy, Maha Shady, Maria A Andrianova, Michael Spencer Chapman, E. V. Van Allen, S. Sunyaev
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The WHO 2021 classification of central nervous system tumors incorporated molecular markers to better define diffuse glioma subgroups. We analyzed array-based copy number, targeted next-generation sequencing, germline genotyping, and Illumina EPIC array methylation data from 404 primary IDH-mutant astrocytomas to further understand the relationship between copy number and outcomes. Loss of chromosome arm 13q was associated with poor overall survival after adjusting for age, sex, tumor grade, and treatment status (hazard ratio [HR]=2.04 (95% confidence interval: 1.25-3.33), p=0.004). Chromosome 13q loss was also significantly associated with the total number of copy number alterations (p<0.001). Loss of arm 11p was associated with poor survival in the adjusted analysis (HR=2.29 (1.38-3.79), p=0.001). Unsupervised clustering of the raw copy number data identified 12 clusters, where each cluster was defined by distinct copy number alteration(s). Clusters that contained 13q loss exhibited worse overall survival compared to other clusters. The clusters were significantly associated with age at diagnosis (p=0.015), tumor grade (p<0.001) and copy number complexity (p<0.001). The clusters with a high proportion of grade 4 tumors and with the highest complexity had the poorest overall survival. These findings highlight the prognostic relevance of chromosome 13q and 11p losses in IDH-mutant astrocytomas and support the integration of additional copy number alterations into molecular classification frameworks of IDH-mutant astrocytomas to improve risk stratification and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253265c2822339aab418366b6f6dea35282c7e8a" target='_blank'>
              PATH-36. DNA copy number clustering reveals alterations that influence survival in primary IDH-mutant astrocytoma patients
              </a>
            </td>
          <td>
            K. Drucker, Arijit Panda, Paul A Decker, M. Kosel, Shameel Shafqat, Muhammad Asad Maqbool, A. Landry, Y. Ellenbogen, Vikas Patil, Caterina Giannini, R. Vaubel, Aditya Raghunathan, I. Parney, T. Burns, Cristiane M Ida, S. Kizilbash, G. Zadeh, Daniel H Lachance, A. Abyzov, J. Eckel-Passow, R. Jenkins
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, Sameh Youssef, Alain de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma.
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Acquired resistance to immune checkpoint inhibitors (ICIs) remains a significant challenge in the treatment of metastatic melanoma. Phenotypic plasticity, such as dedifferentiation and transdifferentiation, is an increasingly recognized mechanism for treatment resistance. We present a case of a man in his 70s with metastatic melanoma who experienced progression through sequential treatments including pembrolizumab in combination with the HDAC inhibitor entinostat, and ipilimumab. During treatment a histologically distinct pleomorphic rhabdomyosarcoma (RMS) emerged at metastatic sites. Longitudinally acquired tumor samples representing both phenotypes were analyzed using whole- exome sequencing (WES), RNA sequencing (RNA-seq) and high-plex tissue imaging (spatial proteomics). WES revealed driver mutations (e.g. NRAS, NF1) and loss-of-heterozygosity (LOH) shared between phenotypes indicating a common ancestral clone. Phylogenetic analysis demonstrated an early divergence of the phenotypes, with each later acquiring unique mutations. RNA-seq showed mutually exclusive expression of lineage-specific markers as well as epithelial-mesenchymal transition and myogenic gene set enrichment in the RMS samples. High-plex imaging identified distinct tumor microenvironments, with RMS lesions enriched in CD163+ macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="BARD1 variants are associated with hereditary breast cancer and neuroblastoma, yet, 98% of missense variants remain variants of uncertain significance (VUS). We applied Saturation Genome Editing (SGE) to assess 8,818 SNVs and 2,097 3-base pair deletions for their effects on cell survival and RNA abundance. We found that 13% of missense variants are loss-of-function (LoF), with 98% in BARD1's three folded domains. LoF missense variants in the ankyrin repeat and BRCT domains were enriched in breast cancer cohorts, linking their molecular functions to cancer risk. SGE discriminated known pathogenic from benign variants with exceptional accuracy (AUC = 0.99) and resolved 95.4% of VUS, demonstrating high clinical utility. The single nucleotide resolution allowed discrimination of variant effects on RNA abundance from those affecting protein function and provided further evidence linking BARD1's function in homology directed repair to tumor suppression. This comprehensive variant effect dataset informs inherited cancer risk and treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c2cc869774eac711c785244c7b4a415374fa2e8" target='_blank'>
              Saturation genome editing of BARD1 resolves VUS and provides insight into BRCA1-BARD1 tumor suppression
              </a>
            </td>
          <td>
            I. Woo, S. Casadei, M. W. Snyder, N. T. Smith, S. Best, M. Tejura, P. Gupta, A. E. McEwen, M. Post, A. Hamm, M. Dawood, A. Hosokai, A. Xu, R. K. Garge, S. Fayer, T. Brannan, M. E. Richardson, S. Pendyala, S. Heidl, L. Muffley, D. M. Fowler, L. Starita
          </td>
          <td>2025-11-06</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Human oncogenic viruses contribute significantly to the global health burden and include seven types: Epstein–Barr virus, hepatitis B virus, human T-cell leukemia virus type 1, human papillomavirus, hepatitis C virus, Kaposi’s sarcoma-associated herpesvirus, and Merkel cell polyomavirus. While the roles of latent or integrated viral genomes in cancer have been documented, emerging evidence highlights the contribution of defective viruses—those carrying intragenic deletions or loss-of-function mutations—in promoting viral oncogenesis. These altered genomes often lack genes essential for lytic replication or immune recognition, which enhances their persistence and immune evasion. In virus-associated diseases, specific patterns of gene retention and deletion suggest that host-driven selective pressures drive the emergence of these altered genomes. This review examines the generation, prevalence, and functional impact of these viruses, reframing them as active participants in disease development and progression. Recognizing their role offers new insights into viral tumor evolution and creates opportunities for applications in viral diagnostics and targeted intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389381b25524966b33d29c79924df9e5f4c7676f" target='_blank'>
              Defective but tumorigenic: the evolutionary and functional roles of mutated oncoviruses
              </a>
            </td>
          <td>
            Yoshitaka Sato, Yusuke Okuno, Takayuki Murata, Hiroshi Kimura
          </td>
          <td>2025-09-26</td>
          <td>FEMS Microbiology Reviews</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7bcb7a0494ca2d2085913525073bd8291d09b9" target='_blank'>
              EGFR suppression and drug-induced potentiation are widespread features of oncogenic RTK fusions
              </a>
            </td>
          <td>
            Yuzhi (Carol) Gao, D. Gonzalez-Martinez, Sofia Wissert, Hana Bader, Nidhi Sahni, Anh Le, R. Doebele, Lukasz J. Bugaj
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA), a probable human carcinogen, induces toxic and mutagenic O6-methylguanine lesions that are repaired by the O6-methylguanine methyltransferase (MGMT). To elucidate mechanisms of NDMA-induced liver cancer progression, we performed longitudinal analyses of phenomic, transcriptomic, and phosphoproteomic changes in wild-type and MGMT-deficient mice, observing amplified responses in the deficient genotype. Early molecular rewiring indicative of a DNA damage response was detected by phosphoproteomic and transcriptomic profiling within days post-exposure. Transcriptomic analyses identified a persistent and robust interferon response as the dominant activated pathway. This chronic interferon signaling, which remained unresolved, correlated with extensive clonal expansion, an early hallmark of oncogenesis. Spatial transcriptomics further revealed pathway alterations favoring tumorigenesis within clonally expanded cells. These findings delineate the cascade of molecular events triggered by acute early-life NDMA exposure, culminating in cancer development months later. Our study unveils potential predictive biomarkers and strategies for disease mitigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily M. Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Piwi-interacting RNAs (piRNAs) are small non-coding RNAs primarily recognized for their roles in germline development and genome stability. However, their contribution to breast cancer remains poorly understood. In this study, we profiled piRNA expression in 96 primary breast cancer tissues and explored the prognostic and functional significance of piR-775. Higher expression of piR-775 was significantly associated with improved relapse-free survival, suggesting its potential as a favorable prognostic biomarker. Functional assays in triple-negative breast cancer (TNBC) models demonstrated that piR-775 overexpression significantly suppressed colony formation and three-dimensional tumor growth. Using an organ-on-chip platform, piR-775 overexpression also inhibited TNBC cell migration and invasion. Transcriptomic analyses following piR-775 re-expression identified multiple downregulated target genes involved in cell cycle progression and DNA damage repair pathways, including POLE, BARD1, TPX2, BIRC5, and XRCC2, which were further validated by RT-qPCR. Integration with CRISPR screen data highlighted 47 essential piR-775 targets required for TNBC survival. Additionally, DNA damage assays indicated increased genomic instability upon piR-775 expression. Notably, combining piR-775 with the PARP inhibitor Olaparib led to enhanced cytotoxic effects, suggesting a synthetic lethal interaction. Furthermore, transcriptomic analysis of breast cancer revealed consistent downregulation of the PIWI proteins PIWIL2 and PIWIL4, indicating disruption of the piRNA–PIWI pathway in breast cancer. Collectively, these findings establish piR-775 as a tumor-suppressive piRNA with both prognostic and therapeutic relevance in breast cancer, particularly in TNBC. By impairing oncogenic signaling and enhancing the efficacy of PARP inhibitors, piR-775 emerges as a promising biomarker and a candidate for combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858c8d8d1032bace8d569826ea440beaf678ad07" target='_blank'>
              Piwi-interacting RNA 775 (piR-775) predicts favorable prognosis and regulates cell cycle and DNA damage response pathways in breast cancer
              </a>
            </td>
          <td>
            Bushra Yasin Abohalawa, Hend Ghasan Eldous, R. Elango, Radhakrishnan Vishnubalaji, Sameera Rashid, Khalid Ouararhni, Ayman Al Haj Zen, Nehad M. Alajez
          </td>
          <td>2025-11-04</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Acute promyelocytic leukemia (APL) is a subtype of acute myeloid leukemia (AML), characterized by a fusion between the PML and RARA genes and by a block in the myeloid maturation at the promyelocytic stage. This study investigates the epigenetic landscape of APL by integrating ChIP-seq data on eight histone modifications and RNA-seq in APL as well as non-APL AML. APL showed a distinct chromatin profile that differed from non-APL AML. We describe APL-specific changes in H3K27ac, H3K9me3, and H3K27me3 with impact on enhancer activity, repression of transposable elements, and Polycomb regulated gene repression. The APL-specific H3K27ac pattern identifies APL-specific enhancer and super-enhancer regions, including a subset of enhancers that are bound by the PML-RARA fusion protein. While chromatin bound specifically by PML-RARA were dominantly active, APL was also characterized by gain of APL-specific heterochromatin states with significant gains of H3K9me3 enriched lamina-associated domains and the transposable elements LINE, LTR, and SINE. These findings suggest a unique enhancer and heterochromatin profile in APL, with implications for transcription regulation and treatment response. These findings offer novel insights into the pathogenesis of APL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4d4d5354c6d704bfe8c1f882f567114dfdc0fb" target='_blank'>
              Disease-specific epigenetic deregulation of enhancers, transposons, and polycomb targets in acute promyelocytic leukemia
              </a>
            </td>
          <td>
            Xiangfu Zhong, Lina Cordeddu, Angelica Gamboa-Cedeno, S. Bengtzén, Karl Ekwall, Andreas Lennartsson, Sören Lehmann
          </td>
          <td>2025-10-30</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Although NUP85 is a member of the nuclear pore complex (NPC) and is associated with chromosome variation and tumor regulation, its specific role in cancer development remains unclear and requires further research. This study analyzes the mRNA and protein levels of NUP85 in normal and tumor tissues using the TCGA, GTEx, and HPA databases. We investigated the relationship between NUP85 and survival rates, as well as clinical features, in various tumors by analyzing the TCGA database. The expression pattern of NUP85 in cancer cells is analyzed using single-cell sequencing data from the TISCH database. Two types of prostate cancer cell lines are utilized to investigate the impact of NUP85 on cell proliferation, migration, invasion, and apoptosis, as well as its regulatory mechanism. These analyses aim to uncover the role of NUP85 in cancer, particularly in prostate cancer. NUP85 is observed to exhibit elevated expression levels across multiple malignancies, with its heightened expression showing consistent associations with poorer clinical prognoses. Bioinformatic analyses further reveal that NUP85 expression patterns demonstrate significant correlations with the activity of cancer-related pathways and immunological interactions involving macrophages and T cell populations. Notably, experimental studies using prostate cancer models have documented reduced cellular proliferation following NUP85 knockout, suggesting a potential functional connection warranting further mechanistic investigation. The results indicate that elevated NUP85 expression shows strong correlations with cancer initiation and progression. These findings support its potential utility as a candidate biomarker for disease monitoring across various malignancies, though mechanistic validation remains necessary to establish clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d9fcec4dab152579fb48a48d3474142345170" target='_blank'>
              Comprehensive pan-cancer evaluation of NUP85 prognosis, immune infiltration response, and validation in prostate cancer
              </a>
            </td>
          <td>
            Zhijun Chen, H. Guan, Long Chen, Xinwei Yuan, Wenyan Sun, Ming Chen
          </td>
          <td>2025-10-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is an aggressive malignancy with high mortality, and the identification of upstream regulators of stemness represents a critical step toward developing more effective targeted therapies. This study aimed to define the role of NR5A2 in CRC, particularly in the context of cancer stem cells (CSCs). We profiled DNA copy number alterations, DNA methylation status, and mRNA expression in 30 CRC specimens. Primary CRC cells and CSC-enriched sphere cultures were established in vitro. Functional assays included RNA sequencing, sphere- and colony-formation assays, cell viability and cytotoxicity assessments, qRT-PCR, western blotting, immunofluorescence staining, chromatin immunoprecipitation (ChIP), and in vivo patient-derived xenograft (PDX) models. Integrated genomic and epigenomic analyses implicated NR5A2 in the progression of an aggressive CRC subtype characterized by stemness gene expression. Our findings demonstrate that NR5A2 plays a key role in CRC. Mechanistically, NR5A2 promoted CSC maintenance by directly binding to the promoter and enhancer regions of NANOG, thereby upregulating its expression. Pharmacological inhibition of NR5A2 using Cmp3 significantly sensitized a subset of CRC PDX models to standard chemotherapy, resulting in enhanced tumor regression. This study identifies NR5A2 as a novel, actionable therapeutic target in CRC. Pharmacological modulation of NR5A2 disrupts CSC-driven stemness, potentially preventing relapse and improving treatment outcomes. These findings provide a strong rationale for the development of NR5A2-targeted therapies, either as monotherapy or in combination with chemotherapy, to optimize CRC patient care. 1. NR5A2 was identified as a key therapeutic target in colorectal cancer (CRC) through a dual mechanism involving the regulation of NANOG. 2. A significant proportion of CRC tumors demonstrated responsiveness to NR5A2 inhibition, particularly when combined with tumor-debulking chemotherapy. 3. These findings support NR5A2-targeted therapy as a promising strategy to improve clinical outcomes in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd8c74c01c7fc95987f53308ec60368af31f2dc" target='_blank'>
              Epigenetic regulation of NR5A2 influences colorectal cancer cell stemness via a stemness-related transcription factor NANOG
              </a>
            </td>
          <td>
            Jia Liu, Li Li, Liang Zhang, Wenpeng Wang, Lei Zheng, Dalu Kong, Jiefu Wang, Yang Zhan
          </td>
          <td>2025-11-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Purpose Profilin 1 (Pfn1) has been implicated in cytoskeletal regulation; however, its role in breast cancer progression and DNA replication remains unclear. This study investigated the functional significance of Pfn1 nuclear-cytoplasmic shuttling in breast cancer. Methods We analyzed Pfn1 expression and its correlation with DNA replication, repair, and oncogenic markers in breast cancer cell lines. Chromatin-bound and soluble Pfn1 levels were quantified by western blotting. The effects of nuclear (nuclear localization sequence-Pfn1) and cytoplasmic (nuclear export sequence-Pfn1) localization on cell growth, DNA replication, and stemness were assessed using colony formation, Alamar blue fluorescence, replication protein A 32-kDa foci staining, and DNA fiber assays. Mouse xenografts of breast cancer cells were used to determine the effect of Pfn1 localization on tumor growth in vivo. We identified the direct interactors of nuclear Pfn1 by immunoprecipitation, and their affinity was determined using bio-layer interferometry. Results Pfn1 expression was positively correlated with DNA replication, repair, p53, and MYC expression. Chromatin-bound Pfn1 was significantly degraded in breast cancer cell lines compared to that in non-cancerous MCF10a cells. Nuclear Pfn1 inhibited cell growth and DNA replication in SKBR3 cells, while cytoplasmic Pfn1 promoted cell survival and DNA replication in MCF10a cells. Loss of nuclear Pfn1 in SKBR3 cells inhibited their growth in vivo. Additionally, cytoplasmic Pfn1 upregulated stemness markers (c-Myc, B lymphoma Mo-MLV insertion region 1, and Nijmegen breakage syndrome 1). Pfn1 regulated cell stemness by binding to the nucleosome remodeler sucrose non-fermenting 2 homolog. Conclusion Our findings revealed that nuclear Pfn1 acts as a tumor suppressor by inhibiting DNA replication and cell growth, while cytoplasmic Pfn1 promotes tumorigenesis by enhancing stemness and replication efficiency. These results highlight the dual role of Pfn1 in breast cancer progression, governed by its subcellular localization. They suggested that modulating Pfn1 nuclear-cytoplasmic shuttling may be a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1361eb9668d89561237c423978b3412e7cc545f" target='_blank'>
              Loss of Nuclear Profilin 1 Triggers Oncogenic Reprogramming of Mammary Epithelial Cells Through Dysregulated DNA Replication in Breast Cancer
              </a>
            </td>
          <td>
            Zheng Zhao, Yingbin Huang, Junhao Mai, Fei Cao, Q. Fang, Di Wu, Ziqian Li, Xuekui Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal of Breast Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Ends of linear chromosomes require protection from host repair machinery that otherwise will mistake them for damaged DNA. The E. coli bacteriophage N15 harbors a linear genome with covalently closed hairpin ends formed by the phage-encoded telomere resolvase TelN. The double-strand break repair complex Mre11-Rad50 (MR, SbcCD in E. coli) specifically targets DNA termini, yet how hairpin telomeres evade host nuclease degradation in bacteria remains unknown. Here, we demonstrate that TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from MR processing in E. coli. Using a combination of genetic and biochemical approaches, we show that this protective function requires both TelN sequence-specific DNA binding and species-specific protein-protein interactions. Notably, we found that protection is independent of TelN’s resolution activity and does not require the C-terminal domains of TelN. Our findings reveal a potentially broad mechanism of telomere protection, providing insights into a conserved regulation of MR activity at chromosome ends across the tree of life. Chromosome ends require protection from the DNA repair machinery. This work demonstrates that TelN protein, the telomere resolvase of E. coli phage N15, protects hairpin telomeres from processing by the host Mre11-Rad50 nuclease through distinct molecular mechanisms. TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from Mre11-Rad50 processing. Protective functions require both TelN sequence-specific DNA binding and species-specific protein-protein interactions with the E. coli MR complex. Hairpin telomere protection is independent of TelN's resolution activity and of the C-terminal domains of TelN, locating a core protection mechanism to conserved N-terminal domains. TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from Mre11-Rad50 processing. Protective functions require both TelN sequence-specific DNA binding and species-specific protein-protein interactions with the E. coli MR complex. Hairpin telomere protection is independent of TelN's resolution activity and of the C-terminal domains of TelN, locating a core protection mechanism to conserved N-terminal domains. Bacteriophages with linear genomes evade telomere processing by bacterial host MR complexes via a telomere-resolution-independent function of their telomere resolvase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e29067dbe60d16b5dac9d8ede2b48b79efe83f8" target='_blank'>
              Apoptosis reveals human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Centrosome amplification (CA) has been observed in various solid tumors and contributes to chromosomal instability (CIN) and poor clinical prognosis in patients with cancer. CA also inhibits cell proliferation by inducing cell-cycle arrest and cell death. However, the mechanism of regulation of centrosome number in cancer cells and its effect on CIN and cell proliferation remains elusive. Here, we report that tuftelin (TUFT1) is a novel centrosomal protein that localizes to the proximal ends of parent centrioles. TUFT1 prevents CA and mitotic spindle multipolarity by suppressing premature polo-like kinase 1 activation. In addition, TUFT1 is phosphorylated by NIMA-related kinase 2 (NEK2), and the phosphorylation status of TUFT1 is essential for coordinating centrosome number and cell proliferation in cervical and breast cancers. Data from clinical breast cancer samples indicate that the combined detection of TUFT1 and NEK2 expression reflects tumor malignancy and patient survival with higher precision. Overall, these results reveal a crucial role of TUFT1 in the regulation of tumor progression through centrosome number control. Thus, TUFT1 represents a promising target for diagnostic and therapeutic approaches for cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efc995dda20af305e1ea407bedfd835bb75716c" target='_blank'>
              TUFT1 regulates cancer progression by suppressing centrosome amplification and mitotic spindle multipolarity
              </a>
            </td>
          <td>
            Shaojie Feng, Mengmeng Zhao, Donghui Zhang, Yan Zhang, Lingyuan Min, Xianqiang Liu, Huan Shi, Tianning Wang
          </td>
          <td>2025-09-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past decade, non-viral DNA template delivery has been used with engineered nucleases to target single-stranded DNA sequences in hematopoietic stem and progenitor cells. While effective for gene therapy, this method is limited to short DNA donor templates, restricting its applications to gene corrections. To expand its scope, we developed an editing process using kilobase-long circular single-stranded DNA donor templates and TALEN technology. Our results show that the CssDNA editing process achieves high gene insertion frequency in HSPCs. Compared to AAV-edited HSPCs, CssDNA-edited HSPCs show a higher propensity to engraft and maintain gene edits in a female NCG murine model. This positive outcome is partly due to higher levels of primitive edited HSPCs, a more quiescent metabolic state, and elevated expression of bone marrow niche adhesion markers. Our findings highlight the strong potential of CssDNA as a universal, scalable and efficient non-viral DNA template for gene therapy applications. Letort et al. demonstrate that Circular single-stranded DNA enables efficient and precise gene insertion in hematopoietic stem cells, outperforming other DNA delivery formats and supporting progress toward scalable next-generation gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8334d5fc2c8d3325ef3a32ecf5b25cddb9e9e5c9" target='_blank'>
              Circular single stranded DNA potentiates non-viral gene insertion in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, Diane Le Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd50f18e47ccb7ad438fcdc8ecebf2ceabf04540" target='_blank'>
              Polyclonal origins of human premalignant colorectal lesions
              </a>
            </td>
          <td>
            D. Van Egeren, Ryan O. Schenck, Aziz Khan, Aaron M. Horning, Shanlan Mo, Clemens L. Weiß, Edward D. Esplin, Winston R. Becker, Si Wu, Casey Hanson, Nasim Barapour, Lihua Jiang, K. Contrepois, Hayan Lee, Stephanie A. Nevins, Tuhin K. Guha, Hao Zhang, Zhen He, Zhicheng Ma, Emma Monte, Thomas V. Karathanos, Rozelle Laquindanum, M. Mills, Hassan Chaib, R. Chiu, Ruiqi Jian, Joanne Chan, M. Ellenberger, Bahareh Bahmani, Basil Michael, Annika K. Weimer, D. G. Esplin, Samuel Lancaster, Jeanne Shen, U. Ladabaum, Teri A. Longacre, Anshul B Kundaje, William J. Greenleaf, Zheng Hu, James M Ford, Michael P. Snyder, Christina Curtis
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, D. D. Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe27e8860b31d75137bdc5403fdf3407806c753" target='_blank'>
              Global re-organisation of genome architecture at the transition to gametogenesis
              </a>
            </td>
          <td>
            Tien-Chi Huang, Maria Rigau, V. Malysheva, Chad Whildling, Stella Siciliani, Jingyu Li, Irina Balaguer Balsells, Pavel Artemov, Camille Dion, M. Spivakov, Juan M. Vaquerizas, Petra Hajkova
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bdb65fc0708e21a785db03ef4faf2be314ea734" target='_blank'>
              E2 displacement of CIP2A from TOPBP1 activates the DNA damage response during papillomavirus life cycles
              </a>
            </td>
          <td>
            A. T. Prabhakar, C. James, Allison Q. Nguyen, Phoebe Bridy, Xu Wang, Ryan J. Zoldork, Jenny D Roe, Austin J. Witt, Srimanya S. Panidepu, Molly L Bristol, Krista Dalton, Andreas Wieland, Ahmed Diab, Anthony C. Faber, Elliot J Androphy, Devraj Basu, Paul F. Lambert, Megan E. Spurgeon, I. Morgan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution.
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Determining the correct structure of large, interspersed duplications and related complex genomic rearrangements in genetic disease is critical when establishing causal roles and requires a technology able to span the entire duplicated segment(s) on single molecules. We assessed the use of Bionano optical genome mapping (OGM) for this purpose. We combined OGM, Illumina short-read sequencing and fluorescence in situ hybridisation (FISH) to characterise three large interspersed duplications/triplications, and used the deepC algorithm to predict impact on local topologically-associating domains (TADs), assisting functional interpretation. Case 1 harboured paired interspersed duplications (244/323 kb) on chromosome 13. By analysing multiple molecules > 300 kb completely spanning the smaller duplication, we unambiguously determined the correct structure, which potentially alters the TAD containing FGF9, a candidate gene. In Case 2, involving a child with hypertrichosis and gingival hyperplasia (HTC), duplications on chromosomes 16 (2.01 Mb) and 17 (564 kb) were linked on short-read sequencing. By obtaining three OGM molecules spanning the 564 kb segment, we deduced that a t(16;17) translocation was present, which we confirmed by FISH. This interpretation has important implications for clinical risk and highlights KCNJ2 as a potential driver of the HTC3 locus at 17q24.3. Case 3 involved a complex chromoanasynthesis event on chromosome 20. OGM readily resolved all but two of 12 alternative structures; however full resolution required reads to span a 627 kb duplication, which we could not achieve consistently. OGM represents a powerful tool for disambiguating complex structural variants, but requires multiple individual reads to completely span the duplicated segment. In our hands the upper size of duplications that could be resolved was ~ 550 kb. Deducing the correct configuration is critical both for mechanistic understanding of pathogenesis and accurate recurrence risk counselling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e568829e237f4a57a0d68d6063ed9794d08d203" target='_blank'>
              Exploring the size limits of Bionano optical genome mapping to resolve alternative structures of linked interspersed chromosomal duplications
              </a>
            </td>
          <td>
            Yang Pei, E. Calpena, Jill M. Brown, Ron Schwessinger, Lucy Platts, Simon J. McGowan, Tazeen Ashraf, F. Forzano, Jane A. Hurst, Wendy D. Jones, Ajoy Sarkar, Richard J. Gibbons, S. Twigg, Andrew O. M. Wilkie
          </td>
          <td>2025-11-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d88265df3748494692babcea1f6ced027e33fc6b" target='_blank'>
              Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver
              </a>
            </td>
          <td>
            Rémi Tellier, Marie Fernandes, Audrey Vinchent, Sonia Paget, Agathe Laratte, Céline Vuillier, Elisabeth Werkmeister, C. Villenet, Jean-Pascal Meneboo, A. Chotteau-Lelièvre, Eric Wasielewski, Z. Kherrouche, Marie-José Truong, C. Descarpentries, Luca Grumolato, M. Figeac, Alexis B. Cortot, Laurent Poulain, Andréa Paradisi, David Tulasne
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract NUPTs are DNA sequences of plastid origin that are present in plant nuclear genomes in varying, though typically low, amounts. It is assumed that they are continuously formed and, due to their potentially mutagenic effect, they are removed at a constant turnover rate, which should result in an exponential decay of their age distributions and a negative correlation between age and size. However, these assumptions are based on analyses from a limited number of species and have never been explicitly tested. To gain insight into the mechanisms driving the origin and evolution of NUPTs, here we surveyed the plastid and nuclear genomes of 30 species representing the main angiosperm (flowering plants) lineages. By modeling the distribution of ages and sizes, examining their linear arrangement across the plastid genome, and statistically assessing spatial biases with respect to other genomic features, we showed that NUPTs are: (i) formed by both continuous and episodic mechanisms; (ii) unevenly represented across the plastid genome; (iii) consistently associated with certain classes of RNA genes, in particular rRNA, tRNA, and regulatory RNA genes; (iv) differentially contributing to structural genes; and (v) closer than expected to different superfamilies of transposons in a species-specific manner. Our results reveal the unexpected complexity in the mechanisms driving the origin of NUPTs, which not only involve their continuous formation but also episodic events, highlight their role as a major source of noncoding RNA genes and other genomic features, and provide a more complete picture of the different drivers of evolutionary change at the genome level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cddf52eeced1161dba2e2d7fa0a44d7221d0674f" target='_blank'>
              Episodic and Ongoing Mechanisms Drive Plastid-Derived Nuclear DNA Evolution in Angiosperms
              </a>
            </td>
          <td>
            Juan Pablo Marczuk-Rojas, Ana D Maldonado, Lorenzo Carretero-Paulet
          </td>
          <td>2025-10-13</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/289d25354f1aa0c7f9f35bc3f3f67c1438d2a225" target='_blank'>
              Myosin VI orchestrates estrogen-driven gene expression in breast cancer cells
              </a>
            </td>
          <td>
            Yukti Hari-Gupta, Danielle Lambert, Isabel W. Shahid-Fuente, Natalia L. Fili, Ália dos Santos, Alexandria Sprules, Hyejeong Rosemary Kim, Alexander W. Cook, P. Ellis, Jesse Aaron, Teng-Leong Chew, Lin Wang, C. Toseland
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Epstein-Barr virus associated gastric cancer (EBVaGC) accounts for ∼9–10% of gastric cancers worldwide and is defined by a distinctive molecular profile, including extreme hypermethylation of the DNA. Targeting this aberrant methylation may be a potential therapeutic strategy. EBV+ gastric cancer cell lines (YCCEL1, SNU719) and EBV- lines (AGS, SNU16, MKN74) were treated with a DNA methyl-transferase inhibitor (DNMT), decitabine (DCB), for three days followed by RNA sequencing to identify EBV-specific responses. DNA methylation profiling by reduced representation bisulfite sequencing (RRBS) was performed in EBV+ cell-lines and integrated with expression data to identify epigenetically regulated networks. While DCB induced broad transcriptional changes across all lines, EBV+ cells exhibited the strongest transcriptional response, sharing many upregulated genes. Many of these EBV+ specific genes were expressed at lower baseline levels in EBV+ tumors from TCGA. DCB predominantly reduced methylation at highly methylated intergenic CpGs, with a subset of promoters undergoing significant demethylation. Integrated analysis revealed a strong inverse correlation between promoter demethylation and gene expression, implicating multiple cancer-relevant pathways. Upstream regulator analysis and motif enrichment indicated that regions losing methylation were enriched for retinoic acid receptor α (RARα) binding motifs, suggesting that DCB-mediated demethylation restores RA pathway accessibility and transcriptional activity. Further, inhibiting RAR signaling reduced DCB induced apoptosis. Although DCB can induce both host gene re-expression and viral lytic gene activation in EBV-positive tumors, its impact on RA signaling in EBVaGC has not been studied. Decitabine promotes extensive epigenetic reprogramming in EBVaGC, with preferential effects in CIMP-positive, EBV-infected cell lines. Importance EBV+ gastric cancer contains hypermethylated DNA and despite this distinct molecular phenotype there are currently no EBV-specific treatments available. Using an FDA approved inhibitor to target hypermethylated DNA and multi-omics approach to study the cellular response, we uncovered epigenetically altered transcriptional networks that may be further exploited to improve potential therapy. Among the pathways disrupted, retinoic acid signaling is of particular interest, as retinoid receptors such as RARα and RARβ are frequently hypermethylated and repressed in EBVaGC. Our findings indicate that DNMT inhibition can partially reverse RA receptor silencing, supporting further investigation of DNMTi–RA combination strategies as a novel therapy for EBV+ gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74a4f19fc0f137d7c09390e6a4bff845ed706aa" target='_blank'>
              Integrated transcriptomic and methylome analysis reveals retinoic acid pathway activation after decitabine treatment in EBV associated gastric cancer
              </a>
            </td>
          <td>
            Sarah Preston-Alp, Yin Wang, Matteo Bergonzoni, Andrew V Kossenkov, Samantha Soldan, L. B. Caruso, Davide Maestri, Paul M. Lieberman, B. Gewurz, I. Tempera
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA cytosine modifications, including 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), are key epigenetic regulators with distinct functions. Dissecting the ternary code (C, 5mC, 5hmC) across tissues and cell types remains a critical priority due to the limitations of traditional profiling methods based on bisulfite conversion. Here, we leverage the combined bisulfite and enzymatic (bACE) conversion with the Mouse Methylation BeadChip to generate 265 base-resolution ternary-code modification maps of 5mC and 5hmC across 29 mouse tissue types spanning 8–76 weeks of age and both sexes. Our atlas reveals a complex grammar of 5hmC distribution, jointly shaped by cell mitotic activity, chromatin states, and interplay with 5mC at the same and neighboring CpG sites. Of note, we demonstrate that 5hmC significantly complements 5mC-based biomarkers in delineating cell identity in both brain and non-brain tissues. Each modification state, including 5hmC alone, accurately discriminates tissue types, enabling high-precision machine learning classification of epigenetic identity. Furthermore, the ternary methylome variations extensively implicate gene transcriptional variation, with age-related changes correlated with gene expression in a tissue-dependent manner. Our work reveals how tissue, sex, and age jointly govern the dynamics of the two cytosine modifications, augments the scope of DNA modification biomarker discovery, and provides a reference atlas to explore epigenetic dynamics in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf56e8a468cf8f7c8ba62217313ee3ee7ed7a698" target='_blank'>
              A ternary-code DNA methylome atlas of mouse tissues
              </a>
            </td>
          <td>
            S. Lee, David C Goldberg, Cameron Cloud, Jared B. Parker, Christopher J. Krapp, Christian E. Loo, Elliot Kim, Ivan Zhao, Chengcheng Jin, R. Porecha, M. Bartolomei, Rahul M. Kohli, Wanding Zhou
          </td>
          <td>2025-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is a cellular pathway for high-fidelity double strand DNA break repair that uses the sister chromatid as a guide to ensure chromosomal integrity and cell viability. Deficiency in the HRR pathway (HRD) can sensitize tumors to poly (ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapy, offering an avenue to identify patients who may benefit from targeted therapies. HRD signature (HRDsig) is a pan-solid-tumor biomarker on the FoundationOne®CDx (F1CDx®) assay that employs a DNA scar-based approach to calculate a score based on copy number features (e.g., segment size, oscillation patterns, and breakpoints per chromosome arm) and does not rely on HRR gene alterations, enabling detection of genomic and epigenetic mechanisms of HRD. After finalizing the HRDsig algorithm, analytical validation was conducted in a CAP-accredited, CLIA-certified laboratory on 278 solid tumor and normal tissue specimens. HRDsig results were compared with an independent HRD biomarker, defined by the presence of a reversion mutation restoring HRR gene function. In this evaluation, 100 HRD-positive and 126 HRD-negative samples showed a positive percent agreement of 90.00% and a negative percent agreement of 94.44%. The limit of detection (LoD) was estimated at 23.04% tumor purity, with the limit of blank (LoB) confirmed as zero in 60 normal tissue replicates. Reproducibility testing on 11 positive and 11 negative samples across multiple labs, reagent lots, and sequencers yielded agreement in 99.49% of positive and 99.73% of negative replicates. HRDsig status remained consistent in the presence of interfering substances, demonstrating 100% concordance in spiked samples. These validation results underscore the high analytical concordance, low false-positive rate, and overall robustness of HRDsig for reliable assessment of homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e556ef8cfb0b30b29bee7661332474ddfe38a5dd" target='_blank'>
              Analytical validation of a homologous recombination deficiency signature (HRDsig) in pan-tumor tissue samples
              </a>
            </td>
          <td>
            Wenshu Li, Jeffrey A. Leibowitz, Shuoguo Wang, Louisa Walker, Chang Xu, Kuei-Ting Chen, Alexa B Schrock, Jason D Hughes, Nimesh Patel, J. Elvin, Lauren L. Ritterhouse, E. Sokol, Garrett Frampton, Lucas Dennis, Bahar Yilmazel, Brennan Decker
          </td>
          <td>2025-11-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Gene expression and regulation with or without alternative splicing are crucial for tissues and cells to function correctly. They have been studied from three almost independent perspectives at the genome level: 1- Recognition of splicing quantitative trait loci (sQTLs), 2- Expression quantitative trait loci (eQTLs) recognition, and 3- Recognition of longer-range physical chromatin interactions between genome segments which model 3D dynamics of cells and tissues. Even though the associations between eQTLs and longer-range chromatin interactions have been previously studied, a similar relationship between sQTLs and chromatin interactions has yet to be analyzed. In this case, examining whether sQTLs control the alternative splicing of their target genes’ mRNA via physically interacting genome segments is crucial. We have jointly analyzed high-throughput chromatin conformation capture (Hi-C) and sQTL datasets over eight human cancer tissues. We have discovered a positive association between the number of genes having sQTLs and chromatin interaction frequency. Such a positive association still exists when we also control for eQTLs. Additionally, sQTLs and their target genes generally exist inside identical topologically associating domains (TADs). These findings are observed over the whole set of analyzed cancer types and functional subsets of the sQTL dataset, such as survival-related sQTLs. Furthermore, tissue-specific sQTLs are statistically enriched in tissue-specific frequently interacting regions (FIREs) in 6 out of 8 human cancer tissues (Chronic Myeloid Leukemia, Colon Adenocarcinoma, Acute Myeloid Leukemia, Lung Adenocarcinoma, Prostate Cancer, Sarcoma). Our sQTL and Hi-C datasets have shown the existence of closer spatial distance between sQTLs and their target genes with possible alternative splicing across several different cancer types in humans. Such a closer spatial distance also exists, independent of whether we integrate eQTLs into the analysis. We found that sQTLs regulate alternative splicing through chromatin interactions. Source code of the analysis in this research is available on https://github.com/seferlab/sqtlhic .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d307f9f6f23c37ffce97f36f862782001b30f0c" target='_blank'>
              Joint analysis of sQTL and Hi-C reveals spatial proximity between sQTLs and target genes in cancer tissues
              </a>
            </td>
          <td>
            Batuhan Eralp, Emre Sefer
          </td>
          <td>2025-10-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Lower-grade glioma (LGG) is a common primary brain tumor with potential progression to glioblastoma. Its development is closely linked to DNA damage repair. DNA polymerase delta subunit 3 (POLD3), a subunit of the DNA polymerase δ complex, participates in DNA replication and repair. POLD3 may act as a therapeutic target and prognostic biomarker in LGG. We collected primary data from The Cancer Genome Atlas (TCGA), the Chinese Glioma Genome Atlas (CGGA), the Tumor Immune Estimation Resource (TIMER), and Gene Set Enrichment Analysis (GSEA). The CGGA dataset included messenger RNA (mRNA) sequencing and clinical data from 403 LGG cases; its microarray subset contained 142 additional cases. The TCGA dataset provided clinical data for 503 LGG cases. Employing bioinformatics tools, we investigated the correlation of POLD3 expression with the clinical characteristics, pathological features, survival outcomes, and tumor immune microenvironment in LGG. Subsequent functional validation of POLD3’s impact in LGG was performed using both cellular and animal models. Integrative analysis combining bioinformatics and experimental approaches revealed significant POLD3 overexpression in LGG. Elevated POLD3 levels exhibited significant associations with distinct clinicopathological features, altered immune cell infiltration profiles, and adverse clinical outcomes. Functionally, POLD3 was demonstrated to drive LGG cell proliferation, invasion, and modulate the tumor microenvironment. This study establishes POLD3 as a critical molecular determinant in LGG pathogenesis. Our findings position POLD3 as a promising multifaceted target, with significant implications for advancing LGG diagnostics, therapeutic development, immune modulation strategies, and prognostic assessment, thereby providing a fresh conceptual framework for understanding and managing this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b766711209d5df5ec31c2b94f1204021afe9d8" target='_blank'>
              POLD3 knockdown effects on low-grade glioma: insights from bioinformatics and experimental validation
              </a>
            </td>
          <td>
            Zhihui Yang, Zhonghao Dou, Hao Wang, Zhaojiang Hu, Yanbo Gao, Yilei Bai, Junfeng Liu, Rongjun Qian
          </td>
          <td>2025-10-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999aa4e6a47ef459adcf0a9e608c45b2e8af7576" target='_blank'>
              A human supergene: BRCA1, CCDC200, and U2 snRNAs
              </a>
            </td>
          <td>
            Pelle Scholten, Michael Segel, David Haig
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancers are molecularly heterogeneous, with subtype-specific differences in transcriptional programs, chromatin architecture, and therapeutic responses. While PARP1 has been extensively studied in the context of DNA repair, emerging evidence implicates its catalytic activity in a broader set of cellular processes, including the regulation of gene expression. Here, we employed an NAD analog-sensitive PARP1 (asPARP1) chemical genetics approach combined with mass spectrometry to map the ADP-ribosylated proteome across six human breast cancer cell lines representing luminal and basal/triple negative subtypes. We identified thousands of PARP1 substrates and hundreds of Glu/Asp ADPRylation sites, revealing both shared and subtype-specific modifications in cell lines maintained under basal growth conditions. Luminal-specific substrates were enriched in chromatin and transcriptional regulators, whereas basal-specific substrates were preferentially linked to translation and RNA processing, highlighting lineage-dependent PARP1 activity. Transcription factors emerged as major substrates, with TFAP2A serving as a proof-of-concept; it is selectively ADPRylated in luminal cells and inhibition of PARP1-mediated ADPRylation modulates its promoter occupancy in a subtype-specific manner. Our data provide a new resource for studying subtype-specific PARP1-mediated ADPRylation in breast cancer cells. Collectively, our findings expand the conceptual framework for PARP1 function beyond DNA repair, offering mechanistic insights into subtype-specific gene regulation and potential determinants of PARP inhibitor sensitivity in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4df105f2ecd1754e819d2661c1298b7b3761196" target='_blank'>
              Mapping the Subtype-Specific PARP1 ADP-ribosylated Proteome in Breast Cancer Cells
              </a>
            </td>
          <td>
            Sneh Koul, Minjung Kwon, Poulami Tapadar, Yangyang Dai, T. Nandu, Dan Huang, C. Camacho, W. Kraus
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Ribosomal DNA (rDNA), the most abundantly expressed locus in the human genome, is represented by hundreds of units per cell. Each unit includes a 13 kb long region (47S rDNA) containing the genes of ribosomal particles, and a 30 kb long intergenic spacer (IGS). The 47S rDNA is transcribed with varying intensity in different units, some of which remain permanently silent. A key intermediator of this silencing is the promoter-associated RNA (pRNA) produced from a 2 kb long gene situated upstream of the rDNA transcription start site. Recent studies, including ours, suggest that the sequence variability, which normally occurs in mammalian cells, may account for the selective transcription of different rDNA units. The present work is based on the deep sequencing of a pRNA gene fragment and its RNA product and subsequent bioinformatic analysis. We found that a certain SNV, which converts the CCC motif into CCT, as well as deletions which reduce the number of (CCCT) tandem repeats, were significantly more frequent in the DNA than in the respective transcripts. These findings allowed us to establish directly the inhibitory effect of DNA variants on the expression of pRNA and thus (indirectly) the promoting effect on the production of ribosomal RNA. Our results also suggest that (CCCT)n/(GGGA)n DNA repeats and the respective (GGGA)n RNA repeats may form triplex structures, facilitating the function of pRNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891924cd9bf55653e2539c5b0034f6d9fcdd744" target='_blank'>
              Deletions in (CCCT)n repeat regions belonging to the human pRNA gene inhibit its expression
              </a>
            </td>
          <td>
            Nikola Chmúrčiaková, Adam Nógell, E. Smirnov, Dušan Cmarko
          </td>
          <td>2025-10-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Gemykibivirus, an emerging single-stranded DNA (ssDNA) virus of the recently established genus in the family of Genomoviridae, had been discovered in human blood and cerebrospinal fluid and a variety of other body fluids. However, the molecular mechanisms of gemykibivirus entrance into the host cells and its pathogenicity remain poorly understood. To investigate the host response of gemykibivirus, we used an infectious clone of gemykibivirus previously established through molecular biology techniques to rescue virus in HEK293T cells and analyzed the changes in the host transcriptome during the infection period by RNA-Seq. Our findings indicate that gemykibivirus can both express viral proteins and accomplish replication, and high-throughput transcriptome analysis identified a total 1732 significantly different genes. Functional enrichment analysis of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways for differentially expressed genes (DEGs) showed gemykibivirus involving several important pathways, including MAPK signaling pathway, Chemical carcinogenesis-reactive oxygen species and Oxidative phosphorylation. Interestingly, mitochondrial DNA-encoded mRNAs exhibited varying levels of upregulation, suggesting that gemykibivirus may be involved in mitochondrial fission and the regulation of mitochondrial function. Subsequently, a series of experiments proved that gemykibivirus can lead an increase in mitochondrial DNA copy number, promote the release of mtDNA into the cytoplasm, enhance reactive oxygen species production and trigger other cellular antiviral responses. Overall, we lay a foundation for revealing the relationship between Gemykibivirus and human diseases through mitochondrial functional alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67fc3d43e01f79ea8442e27ecafc7aa6d559e12" target='_blank'>
              Transcriptome Analysis Reveals Gemykibivirus Infection Induces Mitochondrial DNA Release in HEK293T Cells
              </a>
            </td>
          <td>
            Runbo Yang, Hao Yan, Yifan Wang, Wenqing Yang, Jianru Qin
          </td>
          <td>2025-09-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Resistance to genotoxic therapies remains a major contributor to tumor recurrence and treatment failure, yet the mechanisms by which cancer cells escape these therapies through DNA damage response (DDR) activation are not fully understood. Here, we identify a DDR regulatory pathway in which glycogen synthase kinase 3 β (GSK3B), a multifunctional serine/threonine kinase, governs DNA double-strand break (DSB) repair pathway choice by phosphorylating 53BP1 at threonine 334 (T334) — a site distinct from canonical ATM targets. This phosphorylation event disrupts 53BP1’s interaction with nonhomologous end joining (NHEJ) effectors PTIP and RIF1, promoting their dissociation from DSBs and inhibiting 53BP1-driven NHEJ. Simultaneously, T334 phosphorylation facilitates the recruitment of CtIP and RPA32 for DNA end resection and promotes homologous recombination (HR) by enabling BRCA1 and RAD51 loading. Notably, the phospho-deficient T334A mutant of 53BP1, unlike 53BP1 loss, accumulates aberrantly at DSBs along with PTIP/RIF1, impairs end resection, and suppresses HR activity. Importantly, both genetic and pharmacologic disruption of the GSK3B–53BP1 axis sensitizes tumors to PARP inhibitors (PARPi) independently of BRCA1 status. Together, these findings reveal a GSK3B-dependent mechanism that regulates DSB repair pathway choice and provide a rationale for targeting this axis to enhance PARPi efficacy in solid tumors regardless of BRCA1 status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7947f267430312493c565ecaa4ae8dfb1f98a221" target='_blank'>
              GSK3B directs DNA repair choice and determines tumor response to PARP1 inhibition independent of BRCA1
              </a>
            </td>
          <td>
            H. Allam, Scarlett Acklin-Wehnert, Ratan Sadhukhan, M. Patra, F. Xia
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="


 Chromosomal instability is a well-recognized hallmark of tumorigenesis. However, it is not known how markers for chromosomal instability, such as total distance of DNA altered or total copy number variations (CNV), affect survival, particularly in IDH wild type glioblastoma. Additionally, whether alterations in pathways that impact cellular responses to chromosomal instability, such as the TP53 or RB1 axis, influence survival, similarly remain poorly understood.



 To better characterize potential prognostic impact of markers of chromosomal instability on patient outcomes, we assessed the Mayo Clinic cohort of IDH wild type glioblastomas (n=449) for whom chromosomal microarray data was available.



 We found that while total distance of DNA altered on microarray analysis was not associated with overall survival (OS) (p=0.4030, univariate analysis), the least altered tumors (stratified by quartiles) when assessed by numbers of CNVs demonstrated improved OS (p=0.0057, univariate analysis). We next evaluated alterations in the MDM2/4-TP53 signaling axis, as well as the CDKN2A/B-CDK4/6-RB1 axis, as both amplifications and losses impacting these pathways have been well characterized in adult diffuse gliomas. While TP53 status was not predictive of OS (p=0.1504, univariate analysis), amplification of MDM2 or MDM4 was predictive of decreased OS (p=0.0071, univariate analysis). We also found that tumors without alterations in the CDKN2A/B-CDK4/6-RB1 axis had improved OS when compared to altered tumors (p<0.0001, univariate analysis). Multivariate Cox regression analysis, adjusting for age and sex, confirmed prognostic impact of MDM2/4 amplification status (HR 1.4090, p=0.0370) and CNV status (HR 0.6798, p=0.00312) on OS.



 CNV status and MDM2/4 amplification represent previously unappreciated molecular markers for prognostication in IDH wild type glioblastoma. Further work is underway to confirm our findings through multi-institutional collaborations and to account for additional potential prognostic factors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7bd1ec122e6536bdfe28952f14c544a17f01a98" target='_blank'>
              EPCO-33. MOLECULAR MARKERS OF CHROMOSOMAL INSTABILITY, INCLUDING COPY NUMBER VARIATION AND MDM2/4 AMPLIFICATION, ARE PROGNOSTIC IN IDH WILD TYPE GLIOBLASTOMA
              </a>
            </td>
          <td>
            Sanghee Lim, Ahauve Orusa, J. Eckel-Passow, R. Jenkins, M. Ruff
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) has a poor prognosis due to late diagnosis and limited treatment options. While protein-coding genes are known to contribute to disease pathogenesis, their specific roles require further investigation. We analyzed RNA-seq and clinical data from 179 PAAD patients in TCGA to examine TICRR (TOPBP1 Interacting Checkpoint And Replication Regulator). Prognostic analyses included survival and risk assessment. Complementary cell-based experiments explored TICRR and associated non-coding RNAs (ncRNAs). Multi-omics profiling examined potential relationships between TICRR expression and tumor microenvironment features. Higher TICRR expression showed association with poorer survival outcomes. Experimental modulation of MIR659 and AC074099.1 expression appeared to influence malignant phenotypes in vitro. Multi-omics data suggested potential links between TICRR-associated molecular changes and features of immune evasion and genomic instability. Our findings suggest possible connections between TICRR expression and PAAD progression. The observed phenotypic changes following ncRNA modulation may indicate their involvement in related oncogenic processes. These preliminary observations warrant further investigation into potential regulatory relationships between TICRR and associated ncRNAs, which could inform future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dcdb22943faec5ffead6df5f9d9a524741a08be" target='_blank'>
              Deciphering TICRR's oncogenic landscape in pancreatic adenocarcinoma: molecular insights and clinical implications
              </a>
            </td>
          <td>
            Ke‐Jie He, Haitao Wang
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3251b6b3c185d73484bc445eaf225c467c2a8c93" target='_blank'>
              Recurrent enhancer-promoter interactions across samples
              </a>
            </td>
          <td>
            Mathew Weston, Satvik Gunjala, Haiyan Hu, X. Li
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 MTAP encodes a key enzyme in methylthioadenosine (MTA) catabolism. MTAP loss, frequent across human cancers, leads to MTA accumulation and inhibition of PRMT5, a critical regulator of cell cycle and DNA repair. This metabolic vulnerability renders MTAP-deficient tumors selectively sensitive to PRMT5 inhibitors such as TNG-462 and TNG-908. Emerging data indicate that MTAP can also be epigenetically silenced via promoter hypermethylation, independent of genomic deletion, thereby evading detection by DNA-based diagnostics and excluding patients from PRMT5-targeted therapies. Here, we integrate MTAP methylation and expression profiles and evaluate PRMT5 inhibitor sensitivity across diverse tumor types. A comprehensive cohort comprising 6,187 samples with matched DNA methylation and gene expression data was assembled from 17 distinct TCGA projects. Data retrieval, raw RNA-seq read counts, and methylation data analysis were performed using the TCGAbiolinks R package, DESeq2 R package and HM450K arrays, respectively. In addition, we incorporated DNA methylation profiles from established human cell lines obtained from the ENCODE project. In vitro, MTAP-wild type cell lines (human: HCC3153 and HS578T and mouse: GL261); MTAP-null cell lines (human: MDA MB-231, RT4, RT112, UM-UC-3) and MTAP-hypermethylated cell lines (human: U87 and HepG2) were used for viability MTT assays after TNG-462 or TNG-908 treatment. Array assays revealed cg25162921 and cg00230302 probes targeting CpG sites associated with the MTAPlocus. The promoter-proximal probe cg25162921 exhibited a strong inverse correlation with MTAP expression in multiple tumor types, including lung squamous cell carcinoma (TCGA-LUSC; r=-0.48), bladder urothelial carcinoma (TCGA-BLCA; r=-0.42), and glioblastoma multiforme (TCGA-GBM; r=-0.45). We next investigated whether the mechanism of MTAP inactivation, genomic deletion and promoter hypermethylation, influenced cellular sensitivity to PRMT5 inhibition by viability MTT assy. MTAP genomic status and methylation levels were stratified at the promoter-proximal CpG site cg25162921 using DNA methylation from ENCODE project. In viability assays, MTAP-null cell lines or MTAP-hypermethylated models were more sensitive to TNG-462 and TNG-908 with a reduced IC50 compared to MTAP-wild type cell lines. We identify promoter hypermethylation as a distinct mechanism of MTAP silencing, broadening the spectrum of MTAP-low tumors beyond genomic loss. These findings support integration of epigenetic biomarkers to refine patient selection and enhance response rates in PRMT5-targeted clinical trials.



 Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, Esther Cabañas Morafraile, Bernard Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, Juan A Ardura, Balazs Gyorffy, Victor Moreno, Alberto Ocaña. Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de65a469e9e37ed4b177eca66de3a9ae88f86296" target='_blank'>
              Abstract C026: Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers
              </a>
            </td>
          <td>
            Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, E. C. Morafraile, B. Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, J. Ardura, Balázs Győrffy, Víctor Moreno, Alberto Ocaña
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Glioblastoma are hallmarked by clonal evolution and transcriptional heterogeneity linked to genetic and environmental diversity resulting in therapy resistance and disease progression. In this study, we integrate heritable molecular barcoding and spatial transcriptomics to enable high-resolution cell lineage tracing in the spatial context of the glioblastoma TME.



 Patient-derived glioblastoma cells from two individual cases were isolated from anatomically and pathologically distinct regions: the primary tumor core, the tumor periphery, and the recurrent lesion. Each subline was transduced with predefined FlexTrack lentiviral plasmids encoding GFP, resulting in uniquely barcoded (9N) cell lines. Immediately before injection, the barcoded glioblastoma populations were pooled and introduced into ex vivo human cortical slices. Live-imaging was used to trace GFP-positive, barcoded cell populations. After incubation, the fresh-frozen slices were processed for spatial transcriptomics and the line-specific barcodes were mapped according to the 3′ poly(A) tails of captured transcripts. Graph neural networks based on single-cell deconvolution were employed to spatially analyze the cellular ecosystems.



 We successfully deconvoluted and reconstructed the trajectories of injected glioblastoma cells linking their transcriptome phenotypes along with clonal identity. Multi-omic integration of spatial copy-number alterations and barcoding based pseudo-trajectories enabled reconstruction of the temporal transcriptional dynamics linked to the heterogeneity of tumor ecosystems.



 This study serves as a proof-of-concept for integrating molecular barcoding and spatial transcriptomics in cortical slice culture. Our approach to molecular barcoding in complex model systems opens new avenues to investigate the contextual dynamics of clonal evolution in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f785ed6bb81bfa9c1d1d1cee449d66b313a3ee5" target='_blank'>
              P02.23.B MOLECULAR BARCODING OF GLIOBLASTOMA IN HUMAN ORGANOTYPIC CORTICAL SLICE CULTURE
              </a>
            </td>
          <td>
            G. Kastner, M. Mühlbauer, J. Jung, S. Langer, L. Müller, A. Baginska, P. Berger, C. Dobersalske, S. Ertmer, F. M. Gerach, N. Neidert, K. Herrmann, L. Rauschenbach, L. van Hijfte, U. Sure, K. D. Kokkaliaris, B. M. Grüner, D. Heiland, B. Scheffler
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The adoption of liquid biopsy approaches in clinical practice has triggered a significant paradigm shift in the diagnostic, prognostic, and predictive outcomes for cancer patients. Circulating tumor DNA (ctDNA) is considered a valuable biomarker for monitoring tumor burden and its mutational dynamics. In this context, not all cell-free DNA (cfDNA) molecules are derived from tumor cells. Furthermore, due to tumor heterogeneity, not all ctDNA molecules contain cancer-associated alleles, complicating the direct quantification of the circulating tumor allele fraction (cTF) within the total cfDNA. Cancer arises from the accumulation of multiple genetic and epigenetic changes. Each of these molecular features can be exploited as the basis of methodological strategies used in ctDNA quantification. Different layers of omics data, from genomics, evaluating mutational analysis of somatic single-nucleotide variants and copy number alterations, to epigenomics, primarily consisting of the evaluation of methylation profiles and fragmentation patterns, can be used for this purpose. Some of these approaches can be effective in a multi-modal manner. To date, the quantification approaches for estimating cTF vary enormously, making direct comparisons and an assessment of their translational value challenging. Moreover, the lack of regulatory approval for many of these assays is a critical barrier to their widespread clinical adoption. This review explores the different omics approaches described for ctDNA quantification, outlining strengths and limitations, and highlighting their valuable applications in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65d286916cbf91ea842f3c5cc84289bd6ae22c9" target='_blank'>
              Genomics and Epigenomics Approaches for the Quantification of Circulating Tumor DNA in Liquid Biopsy: Relevance of a Multimodal Strategy
              </a>
            </td>
          <td>
            Elisa De Paolis, Alessia Perrucci, Gabriele Albertini Petroni, Alessandra Conca, Matteo Corsi, Andrea Urbani, A. Minucci
          </td>
          <td>2025-11-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In most solid tumors, hypoxia is a critical physical attribute that reprograms malignant cells into a highly metastatic state. Specifically, hypoxia is a well-established inducer of cellular plasticity, which is associated with treatment resistance and metastasis. Furthermore, hypoxia exacerbates chromosomal instability (CIN), a hallmark of cancer that can be initiated by the loss of Trp53 and a key contributor to metastasis. Despite this, the mechanisms by which malignant cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains unknown. Here we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. Mechanistically, Kdm8 suppression in normoxia recapitulates major aspects of the global epigenetic changes and the transcriptomic rewiring induced by hypoxia. Moreover, Kdm8 deficiency leads to mitotic defects, increased micronuclei formation, Kras copy number gains, and enhanced CIN. Of note, disruption of Kdm8’s demethylase function phenocopies the effects of Kdm8 loss, whereas expression of hypermorphic Kdm8 variants that are resistant to hypoxic suppression reduces metastasis beyond the levels achieved by the wildtype counterpart. Through the suppression of Kdm8 demethylase activity, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7bb4ad19cff7d1c29fd11327f4df4bccd23cd26" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Shin-Heng Chiou, P. Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Roy, Yousef Masoudpoor, Arthur Roberts, K. Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, Zhouli Ni, Guoping Gu, Cristina Montagna
          </td>
          <td>2025-10-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) is an aggressive malignancy with a poor prognosis and limited treatment options. Previous studies have shown that selective downregulation of the MADD (MAP-kinase-activating death domain-containing protein) gene isoform increases ATC cell susceptibility to TRAIL-induced apoptosis. However, the existence of multiple MADD gene isoforms raised the possibility of functional compensation. This study aimed to definitively evaluate the role of MADD in ATC by employing CRISPR-Cas9 to Cas9 to target exon 3, a conserved exon expressed in all known MADD isoforms, resulting in functional knockout of MADD expression. CRISPR-Cas9-mediated MADD knockout, performed in three ATC cell lines (8505 C, C643, and HTH7) with distinct mutational backgrounds, significantly impaired ATC cell function in vitro, as indicated by reduced viability, increased apoptosis, decreased migration, and G0/G1 cell cycle arrest. RNA-seq analysis revealed alterations in genes related to cell survival, proliferation, and metastasis. In the orthotopic ATC mouse model, MADD deletion dramatically suppressed tumor growth, reduced lung metastases, and prolonged survival. Our findings demonstrate that MADD plays a crucial role in ATC cell survival, proliferation, and metastasis. The consistent effects observed across multiple cell lines and in vivo models suggest that MADD may represent a promising therapeutic target for this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c491eafd711a4e0b81ae9148ad43109c7f7479cf" target='_blank'>
              CRISPR/Cas9-mediated deletion of MADD induces cell cycle arrest and apoptosis in anaplastic thyroid cancer cells
              </a>
            </td>
          <td>
            Velavan Bakthavachalam, Mark A. Sanborn, Manikannan Mathayan, Satyajeet S. Salunkhe, , M. Maienschein-Cline, Suman Setty, B. Kabeer, Jalees Rehman, Bellur S. Prabhakar
          </td>
          <td>2025-11-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Triple-negative breast cancer (TNBC) is a major cause of cancer mortality, with distant metastases presenting a significant clinical challenge. While epigenetic mechanisms like DNA methylation are known to influence TNBC progression, their specific role in driving metastasis remains underexplored. To address this gap, we performed comprehensive multi-omics analysis, integrating epigenomic and transcriptomic profiles from TNBC xenograft models and patient cohorts. Genome-wide DNA methylation profiling of primary tumors, lymph nodes, and lung metastases from xenografts revealed pronounced global hypomethylation in lung lesions, consistent with findings from the clinical cohort. Promoter-methylation changes were enriched in pathways linked to invasion and proliferation, and transcriptomic integration identified 22 epigenetically regulated genes. Among these, elevated AK1 , SLC2A5 , TPI1 , and ZBTB17 expression correlated with a higher risk of lung dissemination. These findings highlight altered DNA methylation as a driver of TNBC lung colonization and identify candidate prognostic markers, emphasizing the importance of epigenetic reprogramming in organ-specific lung metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcba6d1c1c788aab2b1bb08a4a08046f1d4d0352" target='_blank'>
              Epigenetic programs shaping lung metastasis in triple-negative breast cancer
              </a>
            </td>
          <td>
            D. Marzese, Andrés F. Bedoya-López, Javier I. J. Orozco, Miquel Ensenyat-Mendez, Betsy J. Valdez, Sandra Íñiguez-Muñoz, Alexander D. Boiko, Huiwen Xie, Maggie L. DiNome, P. Llinàs-Arias
          </td>
          <td>2025-11-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6bb7c74dc3226bfd915b13024e794addeb2154" target='_blank'>
              Diversity and distribution of the subtelomeric Y’ elements across Saccharomyces cerevisiae strains
              </a>
            </td>
          <td>
            Liébaut Dudragne, Juliana Silva Bernardes, Zhou Xu
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Numerous covalently closed circular RNAs (circRNAs) are produced from back-splicing of eukaryotic exons, and multiple circRNAs can be generated from a single gene locus through alternative circularization (AC). However, functions of most AC circRNAs remain poorly understood. Here, we profile the landscape of AC across multiple cell lines and colorectal cancer (CRC) tissues, identifying predominantly expressed circRNAs (pe-circRNA) in each AC gene locus. Other than cell-type-specific expression of most top pe-circRNAs, circMAN1A2(2,3,4,5), a universally expressed pe-circRNA in examined samples, plays an important role in cell proliferation and CRC progression. Mechanistically, circMAN1A2(2,3,4,5) directly interacts with the 3′ untranslated region of Centromere Protein B (CENPB) mRNA via its characteristic back-splicing junction site, which enhances IGF2BP2-mediated CENPB mRNA stability. Inhibition of circMAN1A2(2,3,4,5) expression with locked nucleic acids represses CRC progression. These results uncover the prevalence of AC and the regulatory role of a specific AC circRNA, circMAN1A2(2,3,4,5), in cell proliferation and tumor progression. The functional roles of circular RNAs (circRNAs) in the progression of colorectal cancer remain to be studied. Here the authors report that an alternatively-circularized circMAN1A2(2,3,4,5) promotes colorectal cancer cell proliferation via enhancing target mRNA stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/333a174f8178b96c1b80005e5f597d344cda4871" target='_blank'>
              Direct circMAN1A2(2,3,4,5)-CENPB mRNA interaction regulates cell proliferation and cancer progression
              </a>
            </td>
          <td>
            Mei Cao, Guo-Hua Yuan, Shi-Meng Cao, Yu-Xin Liu, Chu-Xiao Liu, Yi-Feng Xu, Jia Wei, Yi-Lin Guo, Ling-Ling Chen, Li Yang
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The modulation of DNA replication dynamics has emerged as a key area of study in understanding genome stability and its perturbations in various physiological and pathological contexts. Replication fork rate is influenced by a variety of factors, including DNA repair pathways, origin firing, chromatin organization, transcription, and oncogenic signaling. This review highlights recent findings on the molecular mechanisms driving replication fork acceleration, focusing on scenarios such as PARP inhibition, oncogene activation, depletion of replication factors, and defects in Okazaki fragment processing. We discuss how reduced origin firing, R-loop resolution, and metabolic changes contribute to fork rate modulation, as well as the involvement of innate immune signaling, particularly through pathways such as cGAS-STING and ISG15. Special attention is given to consequences of accelerated replication forks for genome stability and their role in disease progression, particularly cancer. By unraveling the molecular mechanisms of fork acceleration, this Mini Review underscores its critical role in shaping genome integrity and cellular homeostasis, providing insights into future research directions and therapeutic strategies. A mini review explores molecular drivers of replication fork acceleration and their consequences for genome stability, innate immunity, and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edcef1b767fb22ecf30bb9d1778a6566d3ca870a" target='_blank'>
              The need for speed: drivers and consequences of accelerated replication forks
              </a>
            </td>
          <td>
            Dávid Lukáč, Katarína Chromá, Pavel Moudrý
          </td>
          <td>2025-11-06</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The transcription of satellite DNA is highly sensitive to environmental factors and represents a source of genomic instability. Therefore, tight regulation of (peri)centromeric transcription is essential for genome maintenance. Antibiotics are routinely used for in vitro studies and for medical treatment, however, their effect on pericentromeric satellite DNA transcription was not investigated. Here we show that antibiotics geneticin and hygromycin B, conveniently used in cell culture, as well as rifampicin (along with five other antibiotics), used to treat bacterial infections, increase transcription of a major human pericentromeric alpha satellite DNA in cell lines at standard concentrations. However, response differs among cell lines - maximal increase in A-1235 cells is obtained by rifampicin while in HeLa cells and fibroblasts by geneticin. There is also a positive correlation between antibiotic concentration and the level of alpha satellite transcription. The increase of transcription is accompanied with either H3K9me3 decrease or H3K18ac increase at tandemly arranged alpha satellite arrays while H3K4me2 remains unchanged. Our results suggest that induced alpha satellite DNA transcription upon antibiotic stress could be linked to epigenetic changes - histone modifications H3K9me3 and H3K18ac, which are associated with transcription of heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03ef88bdc2c47f36c0fdbaf887b60122c711579a" target='_blank'>
              Antibiotics induce overexpression of alpha satellite DNA accompanied with epigenetic changes at alpha satellite arrays as well as genome-wide
              </a>
            </td>
          <td>
            Sven Ljubić, Maja Matulić, Damir Đermić, Maria Chiara Feliciello, Alfredo Procino, Đurđica Ugarković, I. Feliciello
          </td>
          <td>2025-09-26</td>
          <td>Epigenetics & Chromatin</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Cyclin-dependent kinase 12 (CDK12) regulates general gene transcription elongation, and plays multiple roles in RNA splicing, DNA damage-response, cell cycle and genomic stability. However, transcriptional partners that guide CDK12-specific gene programs have not been identified. Genomic alterations in CDK12 have been observed in multiple cancers, exhibiting both pro-tumorigenic and tumor-suppressive functions, suggesting a context-dependent mechanism of action.


METHODS
CDK12 copy number alterations and gene expression levels were analyzed in matched primary and brain metastatic patient tumors. Clinical significance was assessed by immunohistochemistry in a large cohort of primary breast cancer patient tumors. RNA-sequencing, ChIP-sequencing and molecular studies were conducted to explore CDK12's mechanism of action, and pharmacological studies were performed both in vitro and in vivo using models of advanced (endocrine-resistant and metastatic) estrogen receptor positive (ER+) disease.


RESULTS
CDK12 amplifications and gene overexpression were observed in brain metastatic tumors. In ER+ primary patient tumors, high CDK12 protein expression significantly associated with poor overall survival, particularly within the ER+/HER2-negative group. In ER+ endocrine resistant models, CDK12 regulated estrogen signaling pathways, with ER/MED1 identified as the master transcriptional complex directing CDK12-specific pro-tumorigenic gene programs. Pharmacological inhibition of CDK12 significantly reduced viability in endocrine resistant and metastatic cell and organoid models in vitro, and decreased metastatic spread in vivo.


CONCLUSION
This work describes a novel mechanism for CDK12, suggesting a potential vulnerability in ER+ breast cancer. These findings provide a basis for further investigation into the role of CDK12 inhibition as a therapeutic approach, particularly in advanced disease settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af116245324e2c76800300a12fb01f632badb6d3" target='_blank'>
              Targeting CDK12 disrupts estrogen-receptor chromatin recruitment and ER-MED1 transcription in advanced ER+ breast cancer.
              </a>
            </td>
          <td>
            Daniela Ottaviani, Mihaela Ola, A. Allotta, Yasmine Maati Chaibi, Seán Hickey, Petra Jagust, Nicola Cosgrove, S. Cocchiglia, F. Bane, Ramón Fallon, Gordon Daly, A. Hegarty, Lance Hudson, Katherine Sheehan, Shannon Kalsi, Stephen Shovlin, Aoibhín M Powell, Ashwani K. Bahl, Ed Ainscow, S. Oesterreich, Adrian V. Lee, F. Couch, Arnold D K Hill, D. Varešlija, Leonie S Young
          </td>
          <td>2025-10-15</td>
          <td>Journal of the National Cancer Institute</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a malignant neoplasm of plasma cells leading to bone destruction and marrow failure. Prognosis and management of MM rely on cytogenetic determination of copy number alterations (CNAs). Nevertheless, karyotype analysis difficult due to the presence of few plasma cells and their low proliferative activity. A shallow whole-genome sequencing (sWGS) technology named LeukoPrint was used to detect genome-wide CNAs in MM patients (n = 128), which can cover the entire genome without involving cell culture. Compared with karyotyping and fluorescent in situ hybridization (FISH), LeukoPrint demonstrated a significantly higher detection rate of copy number alterations (CNAs), increasing from 8.0% (karyotyping) and 44.2% (FISH) to 75.0% (n = 96), provided new CNA information and redefined the prognostic stratification in 20.3% of patients according to mSMART guidelines. Hyperdiploidy was the most common CNA feature (39.6%) in this cohort. A high concordance of 90.7% was observed in CNA between matched bone marrow and peripheral blood samples. LeukoPrint can be regarded as an automated, convenient and cost-effective approach to describe genomic CNA profiles. With the advantage of detecting CNAs of short segments and incorporating routine diagnostic methods, LeukoPrint can add value to conventional karyotyping with improved prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368c2b78c175919af87d99afc97d9ec591c9a64f" target='_blank'>
              Identification of Copy Number Alterations From Shallow Whole-Genome Sequencing in Multiple Myeloma.
              </a>
            </td>
          <td>
            Jiali Li, Shaobing Gao, Zhenling Li, Yuexin Cheng, J. Rao, X. Xiang, Yunjng Zeng, Xi Zhang, Li Gao
          </td>
          <td>2025-10-23</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="NCAPH is mainly involved in the transformation of nuclear chromatin in the intercellular phase into a highly heliform nuclear chromosome, encoded by a gene on chromosome 2q11.2. Studies have found that NCAPH, as an oncogene, plays important roles in the occurrence and development of several cancers, significantly affecting the survival and prognosis of patients. However, the role of NCAPH in low-grade glioma (LGG) has been largely unexplored. Here, we conduct a comprehensive analysis of NCAPH expression, abundance of cell subsets in single-cell cohorts, prognosis, co-localization, epigenetic alterations, functional enrichment, tumor immune-related features, immunotherapy response, drug sensitivity, and molecular docking in LGG. Our findings suggest that NCAPH is significantly overexpressed in LGG and is strongly relevant to poor prognosis, and that NCAPH plays important roles in reshaping the tumor microenvironment, which may promote immune tolerance of LGG and thus become a potential immunotherapeutic target. The sensitivity of LGG patients with high NCAPH expression to chemotherapy agents such as temozolomide also suggests the potential therapeutic effect of chemotherapy combined with immunotherapy. Furthermore, NCAPH was positively correlated with cell cycle, proliferation, DNA damage and repair, EMT, invasion, and apoptosis, while negatively associated with inflammation, quiescence and angiogenesis. Together, this study provides a comprehensive understanding of the role of NCAPH in LGG and suggests that NCAPH may be a potential prognostic biomarker for LGG patients, with potential for drug development and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36556ad2c639805f53cbd4b6ee89f252754798c8" target='_blank'>
              NCAPH as a potential prognostic signaling biomarker regulating low-grade glioma cell proliferation, migration, invasion, immune microenvironment, and drug sensitivity
              </a>
            </td>
          <td>
            Lirui Dai, Shu Jiang, Peizhi Zhou
          </td>
          <td>2025-09-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Esophageal squamous cell cancer (ESCC) is one of the most common and lethal malignancies. It is urgent to uncover the underlying mechanisms for ESCC progression. Aberrant DNA methylation is observed in ESCC. However, the precise relationship between DNA methylation alterations and clinical features of ESCC is still largely unknown. We aimed to further explore the potential role of DNA methylation in the development of ESCC. Abnormal DNA methylation and gene expression were analyzed with TCGA ESCA cohort data. Differentially methylated CpGs were identified with Mann-Whitney U test. Differentially expressed genes (DEGs) were analyzed with limma package and student’s t-test. Prognosis-related CpGs were screened by Cox regression analysis. ESCC subgroups were identified with ConsensusClusterPlus package. Epithelial-mesenchymal transition (EMT) score was calculated with epithelial and mesenchymal gene expression data. CIBERSORTx was employed to analyze immune cell infiltration. Drug sensitivity was predicted with oncoPredict package. Gene Expression Omnibus (GEO) dataset GSE53624 was used to verify CFAP52 expression in ESCC, and GSE160269 and GSE203115 were utilized to investigate the expression of CFAP52 in different cell types. Seurat package was used for single-cell RNA-seq data analysis. Western blot was used to assess CFAP52 expression. MTS, transwell, scratch wound healing, colony formation and cell cycle assays were conducted to verify the function of CFAP52. The genes affected by CFAP52 overexpression were explored via RNA-seq. Arachidonic acid level was quantified with UPLC-MS/MS system. Immunohistochemistry was used to check the protein level of CFAP52. 3443 differentially methylated CpGs and 43 related DEGs were identified between males and females. Besides, 1219 alcohol-related CpGs were screened, and 9 related genes were differentially expressed. 175 ESCC prognosis-related CpGs were screened, and ESCC subgroups were identified with these CpGs. Poor prognosis group had higher EMT score, and showed elevated proportion of regulatory T cells and M2 macrophages and higher estimated IC50s of cisplatin_1496, gemcitabine_135 and docetaxel_1007. CFAP52 was identified as a stage-related gene, and mainly expressed in epithelial cells, especially in malignant epithelial cells. Furthermore, CFAP52 was upregulated in ESCC cells and promoted cell proliferation and migration. CFAP52 might affect arachidonic acid metabolism in ESCC cells. CFAP52 protein level increased in ESCC tissues, and high CFAP52 was associated with poor survival of ESCC patients. Treatment with DNA methylation inhibitor downregulated CFAP52. Collectively, our investigations show that DNA methylation is associated with sex and alcohol consumption in ESCC. Besides, DNA methylation is linked to ESCC prognosis. Moreover, we uncover the potential oncogenic role of CFAP52 in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199b70207cbb95c05493015ce40595ab05a81d9e" target='_blank'>
              Analysis of the associations between DNA methylation and clinical features reveals the potential oncogenic effect of CFAP52 in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Guangchao Wang, Mengzhu Lv, Yan Wang, Qimin Zhan
          </td>
          <td>2025-11-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff090d0a0cb56557fe9a6d37cfd5b8f1167336b" target='_blank'>
              ATRX loss in sarcomas is associated with dysregulated gene and transposable element expression, loss of DNA methylation, and worse survival
              </a>
            </td>
          <td>
            K. Demanelis, J. Leix, M. Burgess, B. Nacev
          </td>
          <td>2025-11-17</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fb5c415bb77b25342add7bc3b95281a99e2297" target='_blank'>
              Metagenomic approaches for studying ubiquitous yet diverse nucleoid associated proteins in microbial communities: challenges and advances
              </a>
            </td>
          <td>
            Harsh V Purohit, Joydeep Chakraborty
          </td>
          <td>2025-10-01</td>
          <td>World Journal of Microbiology and Biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Four decades ago, the discovery of centromere protein-A (CENP-A) marked a pivotal breakthrough in chromosome biology, revealing the epigenetic foundation of centromere identity. CENP-A, a histone H3 variant, directs the formation of the microtubule-binding kinetochore complex, designating the chromosomal site for its assembly and underpins the accurate partitioning of genetic material during cell division. Errors in cell division can give rise to DNA instability and aneuploidy, implicated in human diseases such as cancer. Therefore, discovering the underlying pathways and mechanisms responsible for the formation, regulation and maintenance of the centromere is important to our understanding of genome stability, epigenetic inheritance, and in providing the knowledge to help generate possible treatments and therapeutics. Here, we review various molecular pathways and mechanisms implicated in maintaining centromere identity and highlight some of the key outstanding questions with a focus on the human centromere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481df10bbcc98403846b3169d8fe656f17c65c42" target='_blank'>
              Preserving centromere identity: right amounts of CENP-A at the right place and time
              </a>
            </td>
          <td>
            Zofia Pukało, Bethan Medina-Pritchard, M. A. Abad, A. A. Jeyaprakash
          </td>
          <td>2025-09-30</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many chemotherapies are effective against cancers that display high levels of genome instability by disrupting or overwhelming the DNA damage response to induce cell death. PARP inhibitors (PARPi) exploit this vulnerability by stalling DNA repair particularly in homologous recombination (HR)-deficient cancer cells. Although PARPi are now used to treat BRCA1/2-mutated cancers such as ovarian and breast cancers, they are still limited to a narrow range of clinical indications and are susceptible to acquired resistance. Here, we introduce “DNA Damage Chemical Inducers of Proximity” (DD-CIPs), bivalent molecules that rewire the mechanism of action of conventional PARPi. The DD-CIPs function through chemical induced proximity between PARP1/2 and the chromatin remodeling protein, BRD4. From a candidate library of DD-CIPs, we identified DD-CIP1 which induces the DNA damage response (DDR) and apoptosis to a range of cancer lines at two-digit nanomolar concentrations. Further optimization yielded DD-CIP2, which induces tumor cell death at nanomolar concentrations across diverse blood and solid cancer cells, including cancer types that are insensitive to PARPi. Using small-cell lung cancer (SCLC) as a model, we found that DD-CIP2 triggers DDR, cell cycle arrest, and apoptosis in vitro, leading to anti-tumor efficacy without substantial toxicity in preclinical SCLC xenograft models at well tolerated doses. Our findings demonstrate that DD-CIPs may provide an opportunity to address the limitations of traditional PARPi and establish chemical induced proximity as a strategy for modulating the DDR in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f7c1744fc2f1ed50403a7b109097213e18ec402" target='_blank'>
              Design and Development of DNA Damage Chemical Inducers of Proximity (DD-CIP) for Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Tian Qiu, Yeuan Ting Lee, Brendan G. Dwyer, Yi Jer Tan, Ting Chen, Bryan A. Romero, Yanlan Wang, Jiehui Deng, Tinghu Zhang, G. R. Crabtree, Stephen M. Hinshaw, Kwok-Kin Wong, N. Gray
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573448152ca34657743158301dc6d5b323113cd1" target='_blank'>
              TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition
              </a>
            </td>
          <td>
            Jean-François Spinella, J. Chagraoui, I. Boivin, Guillaume Richard Carpentier, Céline Moison, N. Mayotte, François Béliveau, Vincent-Philippe Lavallée, J. Hébert, Guy Sauvageau
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-10</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT MYC is among the most frequently dysregulated oncogenes in human cancer, yet its direct targeting remains a significant challenge. Here, we present an in‐silico integrative screening approach to identify compounds and combinations that can block MYC's oncogenic function by specifically disrupting its transcriptional regulatory function. Using a doxycycline (DOX)‐inducible model, we established a MYC loss‐of‐function (LOF) gene signature that specifically captures the molecular consequences corresponding to the loss of MYC's ability in transcriptional regulation. By integrating large‐scale post‐perturbation transcriptomic profiling from the CMAP database, we screened over 8300 drug‐induced profiles and identified 70 recurrent compounds that are predicted to antagonize MYC's transcriptional programs. To further enhance their therapeutic potential, we also developed an orthogonality analysis to pinpoint synergistic drug combinations that suppress MYC activity more effectively than single agents. Our scalable framework enables a rational and systematic identification of compounds with potential to antagonize MYC's oncogenic function by disrupting its transcriptional regulatory ability without necessarily decreasing its abundance. Our approach provides new insights on utilizing existing anticancer drugs to indirectly target MYC in MYC‐driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f19d2dd4a2e3e24bf8c740f6f2a3bb3a7026742" target='_blank'>
              An Integrative Drug‐Induced Transcriptomic Analysis Identifies Novel MYC Antagonists and Potential Synergistic Drug Combinations
              </a>
            </td>
          <td>
            Anthony Aceto, Yue Wang, Da Yang
          </td>
          <td>2025-09-30</td>
          <td>Molecular Carcinogenesis</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2879c47c1c36c88a865d8e833a060834b8e01c61" target='_blank'>
              LMP2A-The Other EBV Oncogene.
              </a>
            </td>
          <td>
            Mariah Riel, Eric C Johannsen
          </td>
          <td>2025-10-08</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Trypanosomatid parasites and the diseases they cause affect more than 30 million people annually worldwide. To develop treatments for these diseases, it is critical to understand how trypanosomatid biology protects the parasite, so that these mechanisms may be exploited as drug targets. An important aspect of trypanosomatid survival is protection from oxidative damage inflicted by the host. Reactive oxygen species produced by the host can damage nuclear DNA and kinetoplast, the mitochondrion DNA. DNA damage must be repaired or bypassed for the trypanosomatid to continue to replicate its genome. Trypanosomatids also possess specialized redox pathways that neutralize reactive oxygen species (ROS) from host-derived attacks and endogenous mitochondrion metabolism. This Review Article focuses on how trypanosomatids employ microhomology-mediated end-joining to repair DNA double-strand breaks and translesion DNA synthesis to bypass oxidatively damaged bases in nuclear and kinetoplast DNA. While the deleterious effects of ROS must be managed to protect the parasite’s genome, the redox status generated by oxidative assault is crucial for intracellular signaling, DNA synthesis, and kinetoplast homeostasis. This Review will also comment on the effectiveness of current treatments for trypanosomatid-caused diseases and the role of oxidative damage in trypanosomatid diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/280cba51b176c0cec98145ce4753c330f4d4fd5a" target='_blank'>
              Translesion synthesis and microhomology-mediated end-joining repair in trypanosomatids
              </a>
            </td>
          <td>
            Lea Drogalis Beckham, Anne K. Snyder, Sylvie Doublié, Bruno Martorelli di Genova
          </td>
          <td>2025-10-01</td>
          <td>PLOS Neglected Tropical Diseases</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract The incidence of colorectal cancer in young adults (age of diagnosis <50 years) has been rapidly increasing. Although ∼20% of early-onset colorectal cancer (EOCRC) cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Nongenetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as alterations in the epigenome, microbiome, and immunome. We characterized the DNA methylation (DNAm) signature and measured DNAm age in EOCRC by using The Cancer Genome Atlas (TCGA). Furthermore, we carefully identified intratumoral microbes from TCGA and the Oncology Research Information Exchange Network datasets and then related the microbes to deconvolved immune cell abundances in EOCRC. We observed that the DNAm age in the EOCRC cohort was 12 years older when compared with the average-onset colorectal cancer (AOCRC) cohort, using three different epigenetic clocks. Differentially methylated sites associated with gene expression include cAMP-responsive element binding protein signaling in neurons, G protein–coupled receptor signaling, phagosome formation, and S100 family signaling. These differences were validated in the gene expression data from TCGA and the Oncology Research Information Exchange Network. When comparing the intratumoral microbes between EOCRC and AOCRC, no consistent differences were observed. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more and larger positive correlations in EOCRC. These data suggest that epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. Significance: We investigated whether environmentally driven factors contribute to EOCRC. We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than in AOCRC. These data suggest a dysregulation of the immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c12561621c15cb4ece79f124013162ea3c517b" target='_blank'>
              Epigenetic Modulation, Intratumoral Microbiome, and Immunity in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            N. Jin, R. Hoyd, Ayse S Yilmaz, Jiangjiang Zhu, Yunzhou Liu, Malvenderjit S Jagjit Singh, Dennis J. Grencewicz, Xiaokui Mo, Matthew F Kalady, Daniel W Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos H F Chan, M. Churchman, N. Denko, Frances Di Clemente, Rebecca D. Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, Alexandra P Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, Afaf E G Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A. Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, A. Tan, Daniel J. Spakowicz
          </td>
          <td>2025-10-24</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 SETD2 is a tumor suppressor gene responsible for trimethylation of H3K36, crucial for DNA mismatch repair and homologous recombination. Novel synthetic lethality agents, ATR inhibitors (ATRi), have demonstrated anti-tumor activity in preclinical models and early clinical trials for pan-tumors. SETD2 and H3K36 deficient cells have been demonstrated to display Micro Satellite Instability (MSI) with increased tumor mutational burden (TMB). Deleterious SETD2 alterations occur in 5-10% of brain tumor patients with glioblastoma, IDH wild type, WHO grade 4, (GBM). These patients have a poor prognosis. This study aims to evaluate targeted treatment strategies in central nervous system (CNS) tumors, by investigating the potential of ATRi for tumors harboring deleterious SETD2 alterations and immuno-therapy for tumors with increased mutational burden, with the objective of improving patient outcomes.



 This study analyzes genomic profiles from 434 patients diagnosed with central nervous system (CNS) tumors between January 1st, 2019, and February 28th, 2025, enrolled in the prospective Neurogenome and CoPPO studies. Rigshospitalet has one of the largest single-institution datasets with comprehensive data on SETD2 alterations derived from whole exome/genome sequencing including structural variants, copy number variations, and RNA expression.



 The study cohort included 434 patients with CNS tumors (166 female, 268 male) with a median age at diagnosis of 59 years (interquartile range: 49-66 years). GBM was the predominant tumor type (86%), followed by astrocytoma, IDH mutated (mut), WHO grade 2-4 (12%) and oligodendroglioma, IDH-mut, 1p/19q co-deletion, WHO-grade 2-3 (1.2%). Other CNS tumors were diagnosed in 6 (1.4%) patients. Deleterious SETD2 alterations were found in 13 (3%) patients: 11 with GBM, 1 astrocytoma IDH mut, and 1 oligodendroglioma. Twelve patients had mutations: 6 premature stop codons, 5 frameshift mutations and 1 splice site variant, and one case with biallelic SETD2 loss. Data on tumor mutational burden (TMB) was assessable for 12 SETD2-deficient patients, none exceeding 10mut/mb; mean TMB was 3.3 mut/mb vs. 6.2 mut/mb in the SETD2-wildtype (wt) group. No SETD2-deficient patients had received temozolomide (TMZ) treatment prior to sampling. By contrast, some samples in the SETD2-wt cohort were obtained at relapse, which may have inflated TMB due to TMZ treatment. COSMIC signatures were available for 10 SETD2-deficient patients with none having indications of MSI based on COSMIC signature evaluation.



 Loss of SETD2 function in primary CNS-tumors is relatively rare and was predominantly driven by mutations, with copy number analysis revealing only 1 biallelic deletion in 434 patients. SETD2 alterations was primarily seen in GBM patients. The SETD2 deficient patients did not exhibit indications of MSI or high TMB. However, the impact of patient relapse samples with TMZ induced mutations in the SETD2-wt cohort is being explored further together with pan-tumor markers for DNA damage repair.



 Tine Østergaard, Christoffer L. Halvorsen, Dorte S. Nørøxe, Vincent N. Fougner, Iben Spanggaard, Kristoffer S. Rohrberg, Martin Højgaard. Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in SETD2 without associated increase in mutational burden [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5615e3697959292b283838db14154a56c4e82cf6" target='_blank'>
              Abstract C037: Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in
 SETD2
 without associated increase in mutational burden
              </a>
            </td>
          <td>
            Tine Østergaard, Christoffer L. Halvorsen, D. Nørøxe, V. Fougner, I. Spanggaard, K. S. Rohrberg, M. Højgaard
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d6f72ebd8a3733c60e8ee75cfbcd9b5da5d9c6" target='_blank'>
              Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes
              </a>
            </td>
          <td>
            Zhe Liu, Yihang Bao, An Gu, Weichen Song, Guan Ning Lin
          </td>
          <td>2025-09-26</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abnormal regulation of microRNA (miRNA) production is increasingly recognized as a hallmark of cancer development. Genetic alterations, disrupted transcriptional control, and deficiencies in processing machinery all contribute to changes in miRNA levels. Depending on the cellular environment, a single miRNA may act either in favor of tumor growth or as a barrier against malignancy. These small RNAs influence key oncogenic features, including uncontrolled cell division, suppression of growth-inhibitory pathways, evasion of programmed cell death, metastatic spread, and angiogenesis. Because of this broad impact, numerous studies have explored the potential of miRNAs as biomarkers for early diagnosis and prognosis, although further validation is still required before clinical translation. Among these regulators, hsa-miR-1296 has recently gained attention. Both strands of its precursor, miR-1296-5p and miR-1296-3p, are involved in modulating multiple signaling cascades across different tumor types. Depending on tissue context, miR-1296 can function as either a tumor suppressor or, less frequently, as an oncogenic factor by targeting genes central to cell-cycle progression, migration, invasion, and therapy resistance. Evidence from breast, lung, liver, colorectal, gastric, prostate, and bone cancers shows that miR-1296 interacts with networks of protein-coding genes and noncoding RNAs, shaping tumor behavior and influencing patient outcomes. Relevant studies were identified through searches in PubMed and Scopus databases up to 2025, focusing on experimental and clinical investigations of miR-1296 in human cancers and its ceRNA-mediated regulatory mechanisms. This review summarizes the current knowledge of miR-1296 biology and highlights its dual role in cancer, as well as its emerging value as a diagnostic and therapeutic candidate for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840b03bfeaf1a1d94eb0ab4a83679a414d0bc0e8" target='_blank'>
              miR-1296 in Human Cancers: Context-Dependent Tumor Suppressor and Oncogene Orchestrated by ceRNA Networks
              </a>
            </td>
          <td>
            Zahra Diba
          </td>
          <td>2025-10-28</td>
          <td>Journal of Cancer Biomoleculars and Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study investigates the expression patterns and clinical relevance of KLHL35 (Kelch-Like Family Member 35) across various cancer types, with a focus on colorectal cancer. Comprehensive analysis of RNA sequencing data from The Cancer Genome Atlas (TCGA) and protein expression data from the Human Protein Atlas (HPA) revealed significant overexpression of KLHL35 in 13 tumor types, including colorectal cancer. Increased KLHL35 expression was associated with poor clinical outcomes, including overall survival (OS), progression-free interval (PFI), and disease-specific survival (DSS). Receiver operating characteristic (ROC) curve analysis suggested promising but exploratory diagnostic value for KLHL35, with area under the curve (AUC) values exceeding 0.9 in colorectal cancer. Functional enrichment analyses suggested potential associations between KLHL35 and cell cycle/apoptosis pathways, while immune infiltration correlations hinted at a possible role in immune microenvironment remodeling. In vitro assays confirmed KLHL35’s role in promoting cancer cell proliferation, invasion, and migration. These findings suggest that KLHL35 could serve as a valuable diagnostic and prognostic biomarker and a potential therapeutic target, particularly in colorectal cancer. While these findings nominate KLHL35 as a candidate for therapeutic targeting, further mechanistic studies are required to validate its function.”">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25129ee960e072c4c7a11405bd5b1a6a5d1f179" target='_blank'>
              Comprehensive analysis of KLHL35 expression and its prognostic value in cancer: implications for colorectal cancer diagnosis and therapy
              </a>
            </td>
          <td>
            Rong Qin, Yiyao Duan, Shasha Bao, Hui Wang, Jing Zhou, Xirui Fan, Xiang Li, Guoyu Li, Jun Hu
          </td>
          <td>2025-10-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND Colorectal cancer (CRC) is the second leading cause of cancer-related death, largely due to limited treatment options in advanced stages. Genomic alterations in advanced CRC (aCRC) are complex and not fully characterized, with only 30% of patients benefiting from targeted therapies. AIM To investigate the molecular heterogeneity of primary aCRC in order to identify clinically relevant genomic alterations. METHODS We conducted a retrospective molecular analysis of 73 consecutive patients with histologically confirmed primary aCRC (stage pT4a-b). All molecular findings were correlated with available clinicopathological data. In addition, we performed survival analyses using publicly available datasets and tools. RESULTS Genetic abnormalities identified in primary tumors were most frequently mutations in tumor protein p53 (58% of cases), Kirsten rat sarcoma viral oncogene homolog (52%), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (25%), B-Raf kinase (11%) and fibroblast growth factor receptor 3 (8%), as well as R-spondin 3 (RSPO3) fusions (8%). Alterations in the tumor protein p53 and neuroblastoma RAS viral oncogene homolog genes were predominantly observed in tumors from the left colon, whereas B-Raf kinase mutations and RSPO3 fusions were more frequently detected in the right or transverse colon. We also show a strong association between the presence of RSPO3 rearrangements and patients with small tumors, normal carcinoembryonic antigen levels, and microsatellite stable tumors. Furthermore, aCRC patients with protein tyrosine phosphatase receptor type k::RSPO3 fusions exhibited a higher mortality rate. Elevated RSPO3 gene expression levels were also significantly correlated with poorer OS across two large, independent CRC cohorts. CONCLUSION This study identifies a relatively high incidence of RSPO3 rearrangements in aCRC and a strong association with clinical features. Furthermore, we find that RSPO3 fusions are associated with poorer OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b1b9374204622db5b697e06c7d6c2484792d29" target='_blank'>
              RSPO3 rearrangements in advanced colorectal cancer patients and their relationship with disease characteristics
              </a>
            </td>
          <td>
            Raquel Tur, M. Abad, Elena Filipovich, Maria Belen Rivas, Marta Rodriguez, Juan Carlos Montero, J. Sayagués
          </td>
          <td>2025-11-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Human RECQL4 and BLM helicases participate in all DNA dependent processes, including replication stress, DNA damage repair. Both helicases are overexpressed in glioblastoma (GBM), a lethal primary brain tumour, characterised by resistance to radio- and chemotherapy. BLM-depleted glioma cells exhibit senescence-associated or polypoid phenotype when exposed to temozolomide (TMZ) and olaparib (OLA), a PARP inhibitor. This study aims to investigate how RECQL4 depletion influences the response of malignant gliomas to chemotherapeutics. We investigated the effect of RECQL4 depletion in glioma cells on cell growth, apoptosis, senescence and polyploidy in the response to combined TMZ and OLA treatment. We compared transcriptomes of RECQL4- and BLM-depleted LN18 and LN229 glioma cells. Drug-induced cytotoxicity, senescence-associated phenotypes, cell cycle alterations, and polyploidy were assessed using the MTT metabolic assay, β-galactosidase activity assay, and propidium iodide staining. RECQL4 depletion modestly affected basal glioma cell viability and proliferation, similarly to knock out of the BLM protein. Deletion of RECQL4 in glioma cells (RQ4 KO) induced profound transcriptomic alterations, dissimilar to BLM depletion. RECQL4-depleted glioma cells treated with TMZ and OLA exhibited reduced viability and increased levels of apoptosis markers. The treatment induced cell cycle arrest, however, RQ4 KO cells did not show signs of senescence phenotype or polyploidisation, when compared to BLM KO glioma cells. Interestingly, both RQ4 KO and BLM KO cells were more resistant to WP744, a doxorubicin derivative, when compared to WT LN229 glioma cells. Our results highlight the distinct roles of RecQ helicases in a response to chemotherapeutics and support a rationale for targeting RECQL4 as a therapeutic strategy in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d631e6439cbcbbf960a1d54b75f9eb6885b3c7c" target='_blank'>
              Shared and non-overlapping functions of RECQL4 and BLM helicases in chemotherapeutics-induced glioma cell responses
              </a>
            </td>
          <td>
            K. Wojnicki, B. Wojtaś, I. Ciechomska, B. Kaza, Matthew Guille, Waldemar Priebe, B. Kaminska
          </td>
          <td>2025-09-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a global health challenge, with limited treatment options and a high mortality rate. We recently found that lung cancer cells that express more genes that are typically restricted to the germline (germ cell cancer genes, GC genes) repair DNA double-strand breaks more rapidly, show higher rates of proliferation, and are more resistant to ionizing radiation, compared to cells that express fewer GC genes. Moreover, we found that the gene encoding TRIP13 (thyroid hormone receptor interactor 13) plays a significant role in this malignant phenotype. Here, we demonstrate that melatonin (MT), a hormone synthesized in the pineal gland, downregulates the expression of TRIP13, particularly in lung cancer cells with high expression of TRIP13. Moreover, this downregulation of TRIP13 by MT further inhibits the DNA repair proteins RAD51 and XRCC5, thereby impairing DNA repair via homologous recombination and non-homologous end joining. We further found that the melatonin receptor 1B (MTNR1B), rather than melatonin receptor 1 A (MTNR1A), is essential for MT mediated TRIP13 downregulation. Because we also found that treatment with MT still decreases cell proliferation of TRIP13-KO cells, combining MT with the TRIP13 inhibitor DCZ0415 would likely have an additive anti-proliferative therapeutic effect in the treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Jumping translocations (JTs) can lead to partial trisomies. A breakpoint within the gene known as Jumping Translocation Breakpoint (JTB) has previously been associated with JTs involving the long arm of human chromosome 1 (1q). These 1q+ amplifications are frequently observed in cancer. JTB was initially mapped to the Epidermal Differentiation Complex (EDC) at 1q21, and earlier studies primarily focused on its role in malignant or adult tissues. Using updated genomic data, we refined the mapping of JTB. We employed RNA in situ hybridization (RISH) to visualize Jtb expression with organ, tissue, and cell-level resolution. We demonstrate that human JTB and murine Jtb are located outside the EDC. In midgestational wild-type mouse embryos, Jtb is expressed in multiple tissues, including the developing heart and vertebral column, and shows partial overlap with the expression of early markers of the neural crest cell lineage. Our findings suggest that the oncogenic potential associated with human JTB translocations is likely unrelated to its previously assumed location within the EDC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed6323abd411d00d62b1282d50fbc327ee57f1b" target='_blank'>
              Jumping Translocation Breakpoint Expression in Midgestation Mouse Embryos
              </a>
            </td>
          <td>
            Carley McGrath, Taniya M Jayaweera, T. Lufkin, Costel C. Darie, A. Neagu, Petra Kraus
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43450515e8b62fe85e717aebfac6a246b0159e06" target='_blank'>
              Mus Musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, Sang Hyun Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich DNA secondary structures with known regulatory roles in eukaryotes, but their functions in bacteria remain largely unexplored. Here, we systematically analyzed potential G-quadruplexes (PG4s) across 1566 bacterial genomes. PG4s showed non-random distributions, with strong enrichment in intergenic regions, particularly near gene start sites, suggesting potential roles in transcriptional regulation. Structural analysis revealed a higher proportion of short-loop G4s in these regions, indicating enhanced stability. Comparisons with randomized genomes confirmed that the observed patterns cannot be explained by GC content or genome size alone. Ecological analysis further revealed that bacteria associated with homothermic hosts exhibit significantly lower G4 densities than free-living species, suggesting reduced reliance on G4-mediated regulation in stable environments. Together, these findings highlight G4s as conserved yet context-dependent elements of bacterial genome organization and provide new insights into their potential regulatory roles in prokaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969b849342d5007389accc241718dd3c89890d08" target='_blank'>
              Non-Random Distribution of G-Quadruplex Structures Reveals Regulatory and Ecological Adaptations in Bacterial Genomes
              </a>
            </td>
          <td>
            Jiye Fu, Ke Xiao, Yukun He, Jing Tu
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recombinant adeno-associated virus (rAAV) is a replication-defective viral vector used in hundreds of human gene therapy trials, resulting in five FDA-approved therapies. Despite this success, rAAV-based gene therapies suffer from dose-limiting toxicities, resulting in several severe adverse reactions, including death. Previously, we discovered that rAAV rapidly kills mouse NPCs in vitro and in vivo. This vector contains a minimal genome comprised of 145-base pair inverted terminal repeats (ITRs) with a T-shaped hairpin structure that appears to be necessary and sufficient for this toxicity. However, the mechanism for AAV ITR toxicity is not known, and there have been few attempts to engineer ITRs to attenuate rAAV toxicity. In the current study, we explore the molecular mechanisms that drive dose-dependent rAAV toxicity in dividing human NPCs (hNPCs) and test whether disrupting these mechanisms mitigates this toxicity. Recombinant AAV infection induces aberrant cell cycle progression with activation of the ATM /CHK1/CHK2 pathway and expression of the DNA damage markers γH2AX and 53BP1. Affinity-based proteomics indicate that AAV ITRs bind to Poly-(ADP-Ribose)polymerase 1 (PARP1) and other DNA damage response (DDR) proteins involved in single-strand break repair (SSBR). Recombinant AAV infection attenuates poly-(ADP-ribose) (PAR) formation and mimics the antiproliferative effects of pharmacological PARP inhibitors used in cancer therapy. Moreover, treatment of hNPCs with PARP inhibitors is sufficient to reproduce many features of rAAV-induced toxicity. Finally, we demonstrate that eliminating the T-shaped hairpin within the AAV ITR reduces binding to SSBR proteins and the resulting rAAV toxicity. These findings suggest that rAAV infection induces replication stress and cell death in dividing hNPCs by functionally depleting PARP1 and other DDR proteins that are essential for DNA replication. This work fills substantial gaps in the understanding of the mechanisms of rAAV toxicity and has important implications for the development of safer rAAV-based human gene therapies. One Sentence Summary The rAAV genome binds to and depletes PARP1 and other SSBR proteins that are essential for DNA replication, resulting in DNA double stranded breaks, checkpoint activation, and cell death in dividing cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44901dfbd5176301295029fab9bb4b40320a082" target='_blank'>
              AAV Kills Dividing Cells by Depleting PARP1 and Other DNA Damage Response Proteins
              </a>
            </td>
          <td>
            Sasha Friese, Junjie Zai, Grace Luzbetak, Nyonika Khanna, Johanna Gesperger, Chien-Hung Liu, Wynand P. Roos, Rasha Q. Al-Rahahleh, Matisse Willardson, Feng Yang, Duoduo Fu, Yiming Han, Nadia Lintag, Jaiden Saykham, Joshua Le, Emeen Al-Delaimy, Nolan Soutipan, Ellen Duong, Jeremy Rich, Maria Carolina Marchetto, Michael Rosenfeld, Robert W. Sobol, Matthew Shtrahman
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Telomere length maintenance is strongly linked to cellular aging, as telomeres progressively shorten with each cell division. This phenomenon is well-documented in mitotic, or dividing, cells. However, neurons are post-mitotic and do not undergo mitosis, meaning they lack the classical mechanisms through which telomere shortening occurs. Despite this, neurons retain telomeres that protect chromosomal ends. The role of telomeres in neurons has gained interest, particularly in the context of neurodegenerative diseases such as amyotrophic lateral sclerosis (ALS), where aging is a major risk factor. This has sparked interest in investigating telomere maintenance mechanisms in post-mitotic neurons. Nevertheless, most existing telomere analysis techniques were developed for and optimized using mitotic cells, posing challenges for studying telomeres in non-dividing neuronal cells. Thus, this protocol adapts an already established technique, the combined immunofluorescence and telomere fluorescent in situ hybridization (IF-FISH) on mitotic cells to study the processes occurring at telomeres in cortical neurons of the mouse ALS transgenic model, TDP-43 rNLS. Specifically, it determines the occurrence of DNA damage and the alternative lengthening of telomeres (ALT) mechanism through simultaneous labeling of the DNA damage marker, γH2AX, or the ALT marker, promyelocytic leukemia (PML) protein, together with telomeres. Therefore, the protocol enables the visualization of DNA damage (γH2AX) or the ALT marker (PML) concurrently with telomeres. This technique can be successfully applied to brain tissue and enables the investigation of telomeres specifically in cortical neurons, rather than in bulk tissue, offering a significant advantage over Southern blot or qPCR-based techniques. Key features • This protocol enables the labeling of telomeres in mouse brain tissue prepared from paraffin-embedded brain sections. • This method facilitates concurrent labeling of proteins that are colocalized at telomere sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1e2280b8fd8c35a0ab81f6940db9809bdef4684" target='_blank'>
              Colocalizing Telomeres With PML or γH2AX Foci by IF-FISH in Mouse Brain Neurons
              </a>
            </td>
          <td>
            Anna Konopka
          </td>
          <td>2025-11-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Objective To explore the molecular genetic characteristics of childhood acute lymphoblastic leukemia (ALL) and their relationships with clinical phenotypes, chromosomal abnormalities, and prognosis, so as to provide references for precise diagnosis and treatment. Methods A total of 302 newly diagnosed children with ALL were included. Real-time fluorescent quantitative PCR, high-throughput sequencing and other technologies were used to detect common fusion genes, rare fusion genes and Ph-like ALL-related molecules. Combined with chromosomal karyotype analysis, immunophenotyping and minimal residual disease (MRD) monitoring, the associations between molecular genetic characteristics and clinical indicators as well as prognosis were analyzed. Results Among the 302 children, the total positive rate of leukemia genes was 50.66%, and the gene detection rate in B-ALL children (52.90%) was significantly higher than that in T-ALL (37.21%). Common fusion genes were mainly ETV6/RUNX1 (19.54%), MLL (9.27%) and BCR/ABL (6.29%), with age- and immune subtype-specific distributions. Children with ETV6/RUNX1 positivity had the highest complete remission rate (93.2%) and the best 2-year event-free survival rate (89.8%), while those with BCR/ABL positivity had the worst prognosis (complete remission rate 57.9%, 2-year event-free survival rate 42.1%). There were differences in the consistency between fusion genes and chromosomal abnormalities: ETV6/RUNX1 and BCR/ABL showed 100% consistency with specific chromosomal translocations, while E2A/PBX1 and MLL showed about 50% consistency. The dynamic changes of MRD were closely related to gene types, with significantly higher MRD positive rates in children with high-risk genes. Conclusion The molecular genetic characteristics of childhood ALL have clear clinical significance. Fusion gene detection can be used for disease classification, risk stratification and prognosis evaluation, providing an important basis for the formulation of individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/735c362fca7d080732163ea6bc8f870abd0cd0da" target='_blank'>
              Molecular genetic characteristics and clinical significance of childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenzhen Zhu, Junchen Yan
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Pediatrics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba22ede5cb07736eac6d115a03c89fcfe2e86a6e" target='_blank'>
              Evolutionary Variation of Poxvirus Genome Architecture
              </a>
            </td>
          <td>
            J. Enow, Arkaidy Garber, Betram Jacobs, G. McFadden, Masmudur M. Rahman
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The extracellular space is a critical environment for discriminating self versus non-self nucleic acids and initiating the appropriate immune responses through signaling cascades to relay information about extracellular nucleic acids. Here, we provide evidence that oxidized mitochondrial DNA is tethered to the surface of select mammalian cells through cell surface proteins and heparan sulfate proteoglycans. We demonstrate that cell surface DNA accumulates in large clusters that partially overlap with domains enriched in RNA binding proteins. Finally, we show that human and murine B cell surfaces contain DNA that can be cleared by the secreted nuclease DNASE1L3, and that patients with a DNASE1L3 missense variant associated with increased risk for autoimmune disease harbor increased levels of surface DNA on B and T cells. Taken together, this work expands the scope of cell surface nucleic acid biology and provides a mechanistic link between cell surface molecules and DNA targeting in autoimmune disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b04efe93fb72cdef3529bf2ec79fd71db767d8a" target='_blank'>
              DNASE1L3 surveils mitochondrial DNA on the surface of distinct mammalian cells
              </a>
            </td>
          <td>
            Jennifer Porat, Valentina Poli, Karim Almahayni, Benson M. George, F. N. de Luna Vitorino, Li Yi, Carolina Bras Costa, Yitong Zhou, Andrew Crompton, Hongxiu Wen, Leon Zheng, Peter K. Gregersen, Suneet Agarwal, Benjamin A. Garcia, Yixuan Xie, Leonhard Möckl, Ivan Zanoni, K. Simpfendorfer, Ryan A. Flynn
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Glioblastoma (GBM) is one of the most aggressive and common tumors of the central nervous system (CNS), distinguished by its significant intratumoral heterogeneity and poor prognosis, with a median overall survival of approximately 15 months. Recent advances in single-cell RNA sequencing (scRNA-seq) have provided deeper insights into the molecular subtypes and heterogeneity of GBM, which include the proneural (PN), classical (CL), and mesenchymal (MES) subtypes. Objectives: This study hypothesizes that constructing gene regulatory networks (GRNs) using single-cell RNA sequencing (scRNA-seq) data can elucidate the complex regulatory mechanisms underlying glioblastoma (GBM) heterogeneity. The primary aim is to identify key regulators and subtype-specific markers in GBM through the integration of scRNA-seq and GRNs, with the ultimate goal of discovering novel prognostic biomarkers that can improve patient stratification and treatment outcomes. Materials and Methods: Malignant cells were identified based on inferred copy number variations (CNVs) from scRNA-seq data. Consensus clustering of expression profiles was used to assign GBM subtypes. Analyses of differential gene expression, ligand-receptor interactions, pathway activities, and responses to immune checkpoint inhibitors were conducted. GRNs were constructed by retaining co-expressed transcription factor (TF)-gene pairs with enriched transcription-binding motifs. Bulk RNA-seq data were then used to validate marker expression and assess prognostic value. Results: Distinct functional roles and molecular characteristics were observed across GBM subtypes. Subtype-specific GRNs revealed both unique and shared regulatory features. GATA binding protein 3 (GATA3) emerged as a potential prognostic marker, showing correlation with homologous recombination deficiency (HRD) and patient outcomes. Conclusions: Subtype-resolved GRN analysis of GBM provides valuable insights into intratumoral heterogeneity and regulatory mechanisms. GATA3 may serve as a novel biomarker for prognosis and therapeutic stratification in GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4258a2482ce3fcfd666558255236c3cd210b1ec2" target='_blank'>
              Single-Cell RNA-Seq Dissects Specific Marker in Glioblastoma Based on Gene Regulatory Networks
              </a>
            </td>
          <td>
            Shenglan Li, Yanjie Lan, Jiachen Wang, Zhuang Kang, Mengqian Huang, Rong-Jie Zhang, Wenbin Li
          </td>
          <td>2025-10-01</td>
          <td>Iranian Journal of Biotechnology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad49d001bc6357de47f285935ec8cbde300765bb" target='_blank'>
              MEDUSA: Maintaining Entire DNA Duplexes for Utmost Sequencing Accuracy
              </a>
            </td>
          <td>
            Kan Xiong, Rachel Li, Ning Li, Ruolin Liu, Azeet Narayan, J. Rhoades, Catherine Song, R. C. Lindsley, Heather A. Parsons, G. M. Makrigiorgos, V. Adalsteinsson
          </td>
          <td>2025-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="E2F5 is a member of E2F transcription factor superfamily, controlling many molecular activities, such as cell proliferation, cell differentiation, DNA repair and cell death. Therefore, it is closely related to the occurrence, development and prognosis of a variety of cancers. In recent years, with the rapid development of bioinformatics, genomics and epigenetics, this study has further elucidated of E2F5 in the tumor cell cycle. Based on the latest research reports, this study reviewed the structural composition, dynamic activity regulation of E2F5, and how its transcription program driven by carcinogenic activity changed the progress of various tumor cell cycles, especially how it converted from a “gatekeeper” to a “destroyer”, thus affecting abnormal biological behaviors of tumor cells. Our aim is to provide a new direction for the development of E2F5 targeted therapy strategies and drug resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04822ad21c2d133c4bb16efee92ced6e01833c81" target='_blank'>
              The Roles of E2F5 in Tumor Cell Cycle: The Gatekeeper or Destroyer?
              </a>
            </td>
          <td>
            Yingwen Du, Danyun Wang, Aidi Liang, Xinru Tang, Jiansen Chen, Ruizhi Yao, Lei Meng, Jianxing Xie, Ming Chen, Songtao Xiang, Canbin Lin
          </td>
          <td>2025-10-21</td>
          <td>Urology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 H3K27 mutant(H3K27M) tumors are the most common type of diffuse midline gliomas and are lethal childhood tumors with a median survival of less than 12 months. Radiation is a critical treatment, yet these tumors recur rapidly within the radiation field due to resistance to radiation-induced DNA damage, allowing the tumor cells to survive. However, the interactions mediating efficient DNA repair, radiation resistance, and recurrence are poorly understood. Our goal was to identify mediators of treatment resistance in H3K27M tumors for therapeutic targeting to improve survival.



 We developed an in vivo recurrence model to assess mediators of treatment resistance. Bulk RNA, single cell RNA, and spatial transcriptomics analyses were used to determine drivers of resistance, and genes of interest were mechanistically assessed for their role in resistance.



 We determined that H3K27M epigenetic reprogramming results in activation of cell-surface protein THY1 which is highly upregulated in all H3K27M tumors. We then confirmed in numerous in vitro and in vivo models that THY1 mediates significant treatment resistance. We determined that THY1 mediates resistance through rapid and efficient DNA repair. We then performed extensive mechanistic analyses which revealed that THY1 signaling activates Fyn, a Src family kinase, resulting in downstream DNA repair. Remarkably both biological knockdown of Fyn and pharmacological Fyn inhibition, with brain-penetrating saracatinib, result in delayed DNA repair and tumor cell death.



 We have uncovered that the Thy1-Fyn signaling cascade mediates rapid DNA repair and treatment resistance. Pharmacological Fyn inhibition successfully delays DNA repair which could improve patient survival. As saracatinib is highly brain penetrant(0.5:1 Brain: Plasma ratio) and has demonstrated safety in Phase I/II Alzheimer’s clinical trials, it is an ideal drug for clinical translation. We are currently designing a Phase 0/1 study to assess drug penetration, efficacy, and safety with saracatinib and radiation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c2f1401d86d8dd15e7bf8324894f0a446afbce" target='_blank'>
              EXTH-65. THY1 Mediates Treatment Resistance in H3K7M Pediatric Tumors and is Therapeutically Targetable
              </a>
            </td>
          <td>
            S. Syed, Sunjong Ji, Arushi Tripathy, Yazen Alomary, Reka Chakravarthy, Mateus Mota, Dah-Luen Huang, Zhuwen Wang, Zian Li, Qiang Zhang, Daniel Nava de la Martin, T. Lawrence, D. Wahl, C. Koschmann, S. Venneti, A. Rehemtulla, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with limited therapeutic options and poor prognosis. Growing evidence suggests that transposable elements (TEs) may play critical roles in cancer by promoting genomic instability, altering gene expression through insertional mutagenesis, and modulating immune responses via the expression of TE-derived neoantigens, but their specific impact on IDH-wildtype GBM remains unclear. We investigated the association of TE abundance with survival and oncogenic pathways in GBM.



 We analyzed RNA-sequencing data from 151 IDH-wildtype GBM tumors, classified per WHO 2021 guidelines. TE abundance was quantified in tumor samples across genomic regions (whole genomic, exonic, intronic, intergenic) using REdiscoverTE. Associations with overall survival were evaluated using Cox proportional hazards models. Pathway enrichment analyses were performed for genes correlated with survival-associated TEs.



 Increased abundance of several TEs was significantly associated with reduced survival. Notably, whole-genome BC200 abundance was linked to poorer survival (HR1.75, 95% CI:1.28-2.22, p= 0.0042), with high expressors having a median survival of 9.81 vs 14.7 months. Similarly, LTR7Y (HR=2.24, 95%CI: 1.55-3.25, p=0.011), MER61D (HR=1.48, 95%CI: 1.20-1.82, p=0.055), and exonic L1ME3B (HR=2.27, 95% CI: 1.5-3.44, p=0.022) were significantly associated with inferior survival. These survival associations were independent of MGMT promoter methylation and CDKN2A deletion status. Gene expression correlated with TE abundance revealed enrichment in pathways related to RNA processing (BC200) L1mE3B), cellular motility and axon guidance (LTR7Y, MER61D), and immune regulation (LTR16A), suggesting potential multifaceted roles in GBM progression.



 Our findings demonstrate that specific TEs are significantly associated with inferior survival in IDH-wildtype GBM, highlighting their potential as prognostic markers alongside known somatic mutations. Their association with key tumorigenic processes such as immune evasion and genomic instability further suggests a contributory role in tumor progression and identifies them as potential therapeutic targets. Future studies should investigate the mechanistic functions, regulatory dynamics, and tissue specificity of TEs using longitudinal datasets to better elucidate their role in GBM biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d64d64059b2e2f2e271240ac062c311929bc6e" target='_blank'>
              EPID-30. Abundance of Transposable Elements is Associated with inferior Survival in IDH-wildtype Glioblastoma: A TCGA analysis
              </a>
            </td>
          <td>
            Eduardo Rodriguez Almaraz, G. Guerra, George Wendt, David V Glidden, Aaron Scheffler, Susan M Chang, S. Francis
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Recent studies reveal that 5-18% of children with cancer harbor pathogenic variants in known cancer predisposing genes. However, DNA damage repair (DDR) genes, which are frequently somatically altered in pediatric tumors, have not been systematically examined as a source of novel cancer predisposing signals.


METHODS
To address this gap, we interrogated 189 DDR genes for presence of germline predisposing variants (PV) among 5,993 childhood cancer cases and 14,477 adult non-cancer controls (discovery cohort). PV were determined using a tiered approach incorporating ClinVar annotations, InterVar classification, and in silico tools (REVEL, CADD, MetaSVM). Using logistic and firth regression, we identified genes with PV statistically enriched in tumors and replicated findings among 1,497additional childhood cancer cases across three independent cohorts.


FINDINGS
Analysis across all cancers revealed enrichment of TP53 PV. Cancer-specific analyses confirmed known associations including germline TP53 PV in adrenocortical carcinoma, high-grade glioma (HGG), and medulloblastoma (MB), PMS2 in HGG and non-Hodgkin lymphoma (NHL), MLH1 in HGG, BRCA2 in NHL, and BARD1 in neuroblastoma. In addition, four novel associations were uncovered, including BRCA1 in ependymoma, SPIDR in HGG, SMC5 in MB, and SMARCAL1 in osteosarcoma (OS). Importantly, the SMARCAL1:OS association was significant in the discovery (6/230, 2.6%, FDRlogistic=0.0189) as well as all three replication cohorts (Childhood Cancer Survivor Study: 8/275, 2.9%; PFisher<0.0001; German Childhood Cancer Registry: 4/135, 3%, PFisher=0.002; INdividualized Therapy FOr Relapsed Malignancies in Childhood: 4/217, 1.8%, PFisher=0.012). The remaining wildtype SMARCAL1 allele was deleted in three of four OS tumors with available data.


INTERPRETATION
Our study confirms the relevance DDR genetic variation in pediatric cancer risk and establishes SMARCAL1 as a novel OS predisposing gene, providing insights into tumor biology and creating opportunities to optimize care for patients with this challenging tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c07cd6375ece0142b345b31eccaeaa135c76e101" target='_blank'>
              Investigation of DNA damage response genes validates the role of DNA repair in pediatric cancer risk and identifies SMARCAL1 as novel osteosarcoma predisposition gene.
              </a>
            </td>
          <td>
            Ninad Oak, Wenan Chen, Alise K Blake, Lynn Harrison, Martha O'Brien, C. Previti, Gnanaprakash Balasubramanian, K. Maass, S. Hirsch, Judith Penkert, Barbara C Jones, Kathrin Schramm, Michaela Nathrath, K. Pajtler, David T W Jones, Olaf Witt, U. Dirksen, Jiaming Li, Y. Sapkota, Kirsten K. Ness, Lillian M Guenther, Stefan M. Pfister, Christian P. Kratz, Zhaoming Wang, Greg T Armstrong, M. Hudson, Gang Wu, R. Autry, Kim E. Nichols, Richa Sharma
          </td>
          <td>2025-10-09</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Glioblastoma multiforme (GBM) is one of the most aggressive and lethal forms of brain cancer, characterized by cellular heterogeneity, poor prognosis and resistance to conventional therapies. One of the major challenges in treating GBM is its diffuse nature, which complicates surgical resection and contributes to high recurrence rates. Our research focuses on the role of biomolecular condensates - membraneless compartments formed by phase separation - in driving oncogenic processes. These condensates organize cellular biochemical processes by compartmentalizing proteins and nucleic acids, thereby regulating gene expression, signal transduction, and metabolism. We are particularly interested in the formation of aberrant condensates in glioblastoma cells and their contribution to malignant transformation, metabolic reprogramming and treatment resistance.



 Using patient-derived GBM models, CRISPR/Cas9 technology, super-resolution microscopy, and proteomic analyses, we are investigating the structural and functional properties of these condensates.



 Our goal is to determine how their dysregulation facilitates cancer progression and to identify novel vulnerabilities that could be therapeutically targeted.



 This research advances our understanding of the pathophysiological roles of biomolecular condensates in cancer and opens new avenues for innovative treatment strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc0d78352f5d6b9af0c7f40eba7db59a9e99146d" target='_blank'>
              P01.03.A SPATIOTEMPORAL ORGANIZATION OF BIOMOLECULES IN GLIOBLASTOMA MULTIFORME CELLS BY PHASE SEPARATION
              </a>
            </td>
          <td>
            M. Sztacho, A. Chytła, C. Carrera Bravo, S. Abbasi Sani, E. Herencia Lagunar, R. Gönül
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly lethal malignancy characterized by a complex pathological mechanism involving multiple genes and progressive stages. The efficacy of targeted and immunotherapy remains limited, highlighting the urgent need for a reliable model to predict prognosis and response to immune checkpoint inhibitors (ICIs). We developed an integrated model based on genes related to autophagy, senescence, dormancy, mitochondrial function, and tumor stemness. The predictive capability of this model for HCC prognosis and ICI response was evaluated. Single-cell transcriptomic analysis and immunocompetent mouse models were further utilized to elucidate the role of model-associated genes in regulating the tumor immune microenvironment. A 16-gene integrated model was constructed using genes associated with mitochondrial function, autophagy, dormancy, stemness, and senescence. This model demonstrated robust predictive power for HCC prognosis and ICI responsiveness. Single-cell trajectory analysis revealed that EZH2 plays a crucial role in immune cell infiltration, activation, and HCC progression. Additionally, in vivo mouse models further indicated that EZH2 may regulate CD8+ T cell activation and exhaustion through the MIF-CD74 signaling pathway. The integrated model holds potential as a prognostic and predictive tool for HCC immunotherapy. EZH2 may influence CD8+ T cell activation and exhaustion via the MIF-CD74 axis, providing insights for patient stratification and potential therapeutic strategies to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7937b56d5b80810f28cf602b4f87ac567637d0a" target='_blank'>
              Integrated single-cell RNA-seq analysis reveals that EZH2 regulates the MIF-CD74 axis to modulate T cell activation and exhaustion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yangyang Zhou, Yi Xu, Mengfan Ye, Jiayi Xing, Di Chen, Al-Ameer Wail Hussein Ahmed, Al-Rahabi Rehab Abdullah Mohammed Ali, Chenwei Pan, Xiangchou Yang, Zan Shen
          </td>
          <td>2025-10-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="JC virus (JCV) replicates within the nuclei of glial cells in the human brain and causes progressive multifocal leukoencephalopathy. JCV possesses a small, circular, double-stranded DNA genome, divided into early and late protein-coding regions. The non-coding control region (NCCR) functions bidirectionally for both early and late genes, and the agnogene is located downstream of TCR and upstream of three capsid proteins in the late region. Previously, in cell culture systems, we demonstrated that these capsid proteins accumulate in intranuclear domains known as promyelocytic leukemia nuclear bodies (PML-NBs), where they assemble into virus-like particles (VLPs). To investigate the agnogene’s function, VLPs were formed in its presence or absence, and differential gene expression was analyzed using microarray technology. The results revealed altered expression of histone-modifying enzymes, including methyltransferases (EHMT1, PRMT7) and demethylases (KDM2B, KDM5C, KDM6B), as well as various kinases and phosphatases. Notably, CTDP1, which dephosphorylates the C-terminal domain of an RNA polymerase II subunit, was also differentially expressed. The changes were predominant in the presence of the agnogene. These findings indicate that the agnogene and/or its protein product likely influence epigenetic regulation associated with PML-NBs, which may influence cell cycle control. Consistently, in human brain tissue, JCV-infected glial cells displayed maintenance of a diploid chromosomal complement, likely through G2 arrest. The precise mechanism of this, however, remains to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b19411275496dccfccf4c9581e91d1c44a41e774" target='_blank'>
              JC Virus Agnogene Regulates Histone-Modifying Enzymes via PML-NBs: Transcriptomics in VLP-Expressing Cells
              </a>
            </td>
          <td>
            Yukiko Shishido-Hara, Takeshi Yaoi
          </td>
          <td>2025-10-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Background/Aim: Hepatocellular carcinoma (HCC) is a heterogeneous liver cancer with limited treatment options and a poor prognosis in advanced stages. To identify novel biomarkers and therapeutic targets, we investigated the role of chromosome 1 open reading frame 50 (C1orf50), a gene with a previously uncharacterized function in HCC. Materials and Methods: We performed a comprehensive transcriptome data analysis of the human hepatocellular carcinoma project from The Cancer Genome Atlas (TCGA) and subsequently validated the oncogenic roles of C1orf50 using HCC cell lines. Results: Using transcriptomic and clinical data from TCGA, we stratified 355 primary HCC samples based on C1orf50 expression levels. Patients with high C1orf50 expression exhibited significantly shorter overall survival, suggesting its association with aggressive tumor behavior. Differential expression and enrichment analyses revealed that C1orf50-high tumors were enriched in oncogenic pathways, including epithelial-mesenchymal transition (EMT), cell cycle activation, and stemness-related properties. Transcriptional regulatory network analysis detected 456 significantly dysregulated regulons, including ZEB1/2 and E2F2, key drivers of EMT and cell cycle, in the C1orf50-high group. In addition, we observed increased YAP1/TAZ signaling, further linking C1orf50 to stemness and therapeutic resistance. Functional data from CRISPR-based dependency screening suggested that several transcription factors up-regulated in the C1orf50-high state, such as ZBTB11 and CTCE, are essential for the survival of HCC cells. These findings indicate potential therapeutic vulnerabilities and support the rationale for targeting C1orf50-associated pathways. Conclusion: C1orf50 is a novel biomarker of poor prognosis in HCC and a key regulator of oncogenic features such as EMT, cell cycle progression, and stemness. This study highlights the therapeutic potential of targeting C1orf50-related networks in aggressive subtypes of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cf5fd5b7c144989d6a903de5542f0ee787b2ca" target='_blank'>
              C1orf50 Accelerates Epithelial-Mesenchymal Transition and the Cell Cycle of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Atsushi Tanaka, Yusuke Otani, Masaki Maekawa, Anna Rogachevskaya, Tirso Peña, Vanessa D. Chin, Shinichi Toyooka, Michael H. Roehrl, Atsushi Fujimura
          </td>
          <td>2025-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This study explores the role of 5mC regulatory genes in head and neck squamous cell carcinoma (HNSCC), focusing on their association with tumor progression, metastasis, and prognosis. A prognostic model was also developed to predict patient survival. Using expression profile data from the GEO and TCGA databases, the study analyzed the differential expression of 5mC-related genes, their connection with somatic mutations, and their impact on the immune microenvironment. A prognostic risk model based on these genes was constructed through the differential expression of 5mC-related genes. (1) Several 5mC regulatory factors were found to be abnormally expressed in HNSCC, with a significant association with somatic mutations. (2) Different 5mC methylation regulation patterns showed variations in immune infiltration, immune checkpoint expression, tumor mutation burden, and cancer stem cell gene set expression, potentially influencing immunotherapy effectiveness and prognosis. (3) Prognosis-related risk genes, such as DNMT1 and TET2, were identified as predictors of survival outcomes in HNSCC patients. The methylation patterns of 5mC regulatory factors are critical in HNSCC progression and are closely linked to tumor microenvironment infiltration. These findings offer new perspectives for treatment strategies and prognosis prediction in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2107e5b0f7cdb8234aeae2307aae6371c1f39db" target='_blank'>
              Identification of 5mC regulatory factors mediating methylation patterns and prognosis models in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Yanwei Li, Yunfeng Cao, Nuerbiyamu Maimaitiyiming, Yue Li
          </td>
          <td>2025-11-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genomic instability is a hallmark of aging and cancer. A key contributor to genomic instability includes alternative DNA structures, such as cruciform-forming inverted repeats (IRs). Short IRs (< 100 bps) are abundant in the human genome, mutagenic, and enriched at mutation hotspots in human cancer genomes. Using an innovative mutation-reporter mouse model, we showed that short IRs are mutagenic in vivo. Further, we found that aging exacerbates IR-induced genomic instability, as evidenced by increased mutation frequencies and altered spectra in the spleen and brain of mice harboring either a short IR or control B-DNA sequence at 2 and 24 months of age. These findings establish a link between aging and enhanced mutagenesis at short IRs, providing a unique in vivo platform to investigate age-related mechanisms of DNA structure-mediated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdc4812813b7511aa9fae9d1c0a6308731d4e5f" target='_blank'>
              Aging Increases Short, Inverted Repeat-Mediated Genomic Instability In Vivo.
              </a>
            </td>
          <td>
            Pooja Mandke, Pallavi Kompella, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-09-30</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Head and neck cutaneous squamous cell carcinoma (HNCSCC) remains underexplored compared to oropharyngeal squamous cell carcinoma, particularly in relation to human papillomavirus (HPV) and molecular markers such as p16 and p53. While p16 is a well-established surrogate for HPV in oropharyngeal cancer, our review highlights its unreliable role in HNCSCC, where positivity is instead associated with recurrence and metastasis. Similarly, p53 illustrates a dual role - wild-type as a genomic safeguard, mutated as an oncogenic driver - complicating prognostication. Methodological considerations, including the limitations of immunohistochemistry for HPV detection, underscore the need for multi-method and molecular validation in future studies. Ultraviolet radiation is posited as a key modifier of p16 function, decoupling expression from tumor suppression. To contextualize these findings, we draw parallels to glioblastoma (GBM), where subclonal evolution, p53 dysfunction, and intratumoral heterogeneity drive relapse despite aggressive multimodal therapies. GBM exemplifies how bulk-level biomarker generalizations often obscure dynamic cellular ecosystems, reinforcing the necessity of single-cell and spatial approaches. Multi-omics integration - encompassing genome, transcriptome, proteome, and tumor microenvironment mapping - coupled with single-cell RNA sequencing and spatial transcriptomics, offers a path forward for resolving subclonal dynamics in both HNCSCC and GBM. These technologies provide the resolution needed to track tumor-immune-stromal co-evolution, identify therapy-resistant clones, and anticipate recurrence. We argue for a N-of-1, patient- and cell-centric paradigm that reframes biomarkers not as static surrogates but as dynamic readouts of cancer evolution across time and tissue contexts. Conceptually, we propose kinetic and microenvironmental frameworks (e.g., “load-and-lock” barriers; dormancy and immune-synapse stabilization) as hypothesis-generating avenues to stall clonal handoffs and improve outcome prediction. Together, these perspectives argue for revised biomarker frameworks in HNCSCC and ethically inclusive, mechanism-anchored studies that bridge discovery with individualized care. By bridging insights from HNCSCC with the lessons of GBM, this review underscores the need for ethically inclusive, mechanistically informed frameworks that integrate subclonal evolution, biomarker re-interpretation, and precision-personalized hybrid models. Such an approach will be essential for advancing from one-size-fits-all strategies to individualized lifetime cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa038c0ad9d152c7382847ca5a53a02fc9fd0417" target='_blank'>
              Rethinking p16, p53, and HPV in HNCSCC through lessons from glioblastoma subclonal evolution toward patient-centric N-of-1 single-cell RNA sequencing paradigm
              </a>
            </td>
          <td>
            Henry Michael Lee, S. Li
          </td>
          <td>2025-11-16</td>
          <td>World Journal of Clinical Cases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human papillomavirus-associated head and neck squamous cell carcinomas (HPV+ HNSCCs) lack early diagnostics and continue to rise in incidence. HPV16 has been detected in ∼90% of HPV+ oropharyngeal cancers (HPV+ OPCs), an anatomical subset of HNSCC, with the majority retaining episomal viral genomes. Despite this, existing episomal HPV16+ OPC cell lines are especially scarce. UMSCC104s were initially reported as episomal; however, the literature contains conflicting reports regarding the genome status of these cells. We now show that UMSCC104s rapidly undergo integration and E2 loss under standard monoculture, and later lots are fully integrated. These findings resolve prior discrepancies and underscore the instability of episomes in monoculture. Accurate models of HPV-driven cancers are critically needed. We propose fibroblast co-culture methods, traditionally utilized for HPV+ keratinocyte models, as a strategy to preserve episomal status in cancer models by supporting viral and host genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79383ce31e18bce44c59616029f77e566e53fa0" target='_blank'>
              The Utility of Fibroblast Co-culture for the Maintenance of Episomes in Human Papillomavirus-Associated Cancer Models
              </a>
            </td>
          <td>
            Austin J. Witt, Rachel L. Lewis, Xu Wang, Phoebe Bridy, Jenny D Roe, I. Morgan, C. James, Molly L Bristol
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) is the fifth-leading cause of cancer-related mortality, with a 5-year survival rate less than 20%. It develops from preneoplastic lesions to adenocarcinoma, but these early genetic alterations remain poorly understood. Therefore, we aimed to identify early genetic drivers underlying the development of preneoplastic lesions and the initiation of gastric carcinogenesis. Methods We characterized preneoplastic and early gastric adenocarcinoma using 48 samples from 16 Guatemalan patients, a country with a high incidence of GC. We sequenced a panel of 127 genes to identify early genetic drivers and possible actionable targets. Results We identified extensive genetic heterogeneity, including single nucleotide and copy number variations. After comparing our data with other studies, we identified TP53 and APC as the most mutated genes in preneoplastic lesions and early GC. Our mean tumor mutational burden was higher in diffuse (0.017 mutations/Mb) and intestinal adenocarcinomas (0.015) than in chronic gastritis (0.005), and analysis of the mutational signatures revealed several processes acting at different stages of the disease. Signatures S15 (DNA mismatch repair deficiency) and S03 (homologous recombination deficiency) were more frequent in early adenocarcinoma than in chronic gastritis, intestinal metaplasia, necrosis, tubular adenoma, and atrophy. Notably, 10 of 16 patients (62.5%) had at least one actionable mutation in their preneoplastic lesions or gastric adenocarcinomas. Conclusions We show that at the preneoplastic and earliest stages, GC is genetically heterogeneous and presents key cancer-driving mutations that may participate in neoplastic transformation and progression, with 62.5% of lesions having the potential for treatment. This study expands the limited research on early GC and highlights key opportunities for precision medicine in populations with high GC incidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f131a17bcf2decc26bf9525e923b46c1c5c1bd" target='_blank'>
              Genetic heterogeneity and key driver mutations in the preneoplastic and earliest stages of gastric cancer
              </a>
            </td>
          <td>
            Eduardo Alfonso Marroquín-Estrada, Héctor Martínez-Gregorio, C. Díaz-Velásquez, Fernando Ambriz-Barrera, Eduardo Emilio Córdoba-García, Laura Itzel Hernández-Romero, Miguel Ruiz De-La-Cruz, Armando Cáceres, Elisa Hernández, José Rodolfo Gil, Elizabeth Orellana, Daniel Penados, Carolina Richter, Marcelino Díaz, Felipe Vaca-Paniagua
          </td>
          <td>2025-10-09</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) are emerging as important regulators of gene expression in both normal biological systems and disease. Among them, metastasis-associated lung adenocarcinoma transcript 1 (MALAT1) has attracted considerable attention due to its high nuclear expression and evolutionary conservation. It was first identified as a biomarker for lung cancer metastasis, but since then, its involvement has been reported in a wide range of cancers. MALAT1 influences several key cancer-related processes, including cell proliferation, migration, angiogenesis, apoptosis, epithelial–mesenchymal transition, and the behavior of cancer stem cells. In this review, we take a closer look at how MALAT1 functions through different mechanisms, such as regulating RNA splicing, altering chromatin states, acting as a sponge for microRNAs, or modifying protein interactions, to drive these changes. We also discuss its growing role in shaping the immune landscape of tumors, particularly its involvement in immune evasion, inflammatory signaling, and regulation of immunogenic cell death. In addition, MALAT1 contributes to resistance against therapy, rewiring of cellular metabolism, and communication between tumor and stromal cells via exosomes. While many studies describe MALAT1 as an oncogene, others suggest it may also play a suppressive role depending on the cancer type and biological context. Here, we aim to offer a comprehensive overview of MALAT1’s diverse actions and evaluate its potential as a diagnostic marker and therapeutic target in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73cb56fb6ad82b5646d4f05c255a6ee44614bc12" target='_blank'>
              MALAT1 as a molecular driver of tumor progression, immune evasion, and resistance to therapy
              </a>
            </td>
          <td>
            A. Bitaraf, Alireza Zafarani, Pardis Jahandideh, Benyamin Hakak-Zargar, A. Haghi, Golareh Asgaritarghi, S. Babashah
          </td>
          <td>2025-10-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8e14b9adf83be97392ebad1ece35384d8c5812" target='_blank'>
              Phosphoproteomic dysregulation drives tumor proliferation in Cushing’s disease
              </a>
            </td>
          <td>
            David T. Asuzu, Dustin Mullaney, Debjani Mandal, Diana Nwokoye, Sheelu Varghese, Daniela Tortoza Lopez, Dhruval Bhatt, Kory Johnson, Abdel Elkahloun, Zied Abdullaev, Kenneth J Aldape, D. Maric, Clarisse Quignon, Nasir S. Malik, Joseph P. Steiner, Yan Li, Susan Wray, L. Nieman, Christina Tatsi, P. Chittiboina
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Simple Summary We compared the results using a 108-gene RNA-seq panel with those derived from optical genome mapping (OGM) in 467 acute leukemias. The overall concordance rate was 88.1%. For clinically relevant events (n = 234), OGM uniquely identified 15.8%, whereas RNA-seq uniquely identified 9.4%. Concordance was poor for enhancer-hijacking lesions (20.6%), including MECOM, BCL11B, and IGH rearrangements, many of which were not detected by RNA-seq. Conversely, RNA-seq slightly outperformed OGM for fusions arising from intrachromosomal deletions that were sometimes labeled by OGM as simple deletions. Overall, RNA-seq is more sensitive for detecting expressed chimeric fusions, whereas OGM is better for detecting cryptic, enhancer-driven events. Using both platforms provides the most comprehensive structural variant detection in acute leukemia cases. Abstract Background/Objectives: Gene rearrangements involving oncogenes are major drivers in acute leukemia, influencing disease classification, prognosis, and therapeutic decision-making. Targeted RNA sequencing (RNA-Seq) panels capable of detecting intergenic and intragenic fusions across multiple genes are increasingly used in diagnostic settings. However, comparative evaluation with orthogonal technologies remains limited. Material and Methods: We compared the performance of a 108-gene anchored multiplex PCR (AMP)-based RNA-Seq panel with that of Optical Genome Mapping (OGM) in 467 acute leukemia cases. The cohort included 360 cases of acute myeloid leukemia (AML), 89 B-lymphoblastic leukemia (B-ALL), 12 T-lymphoblastic leukemia (T-ALL), and 6 cases of mixed phenotype acute leukemia (MPAL). Results: Results of both methods were concordant in 175 (74.7%) of 234 detected gene/rearrangement fusions. The concordance rate varied significantly across different leukemia types, ranging from 80.2% in B-ALL to 41.7% in T-ALL (p < 0.001) OGM uniquely detected 37 of 234 (15.8%) clinically relevant rearrangements, whereas RNA-Seq exclusively identified 22 of 234 (9.4%). Enhancer-hijacking lesions, including MECOM and BCL11B rearrangements, CDK6::MNX1, and IGH rearrangements, had a markedly lower concordance (20.6%) compared with all other aberrations (93.1%) (p < 0.001). Conversely, some gene fusions arising from intrachromosomal deletions were interpreted by OGM as simple deletions rather than rearrangements or fusions. Conclusions: Targeted RNA-Seq was effective for detecting chimeric fusion transcripts and showed slightly better performance in identifying fusions resulting from deletions. However, OGM was effective for detecting enhancer-hijacking events that do not generate fusion transcripts. Both methods are complementary for the workup of acute leukemia cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/783106fd704ddbf425128b4173188409188a77b1" target='_blank'>
              Comparative Analysis of Targeted RNA-Seq and Optical Genome Mapping for Detecting Gene Rearrangements in Acute Leukemia
              </a>
            </td>
          <td>
            C. Ok, Guilin Tang, S. Loghavi, Shimin Hu, Qing Wei, Andrés E. Quesada, M. Routbort, R. Kanagal-Shamanna, C. Yin, Iman Sarami, Sofia A. Garces, Nitin K. Agarwal, R. Luthra, Hong Fang, F. Jelloul, Julian Bryan, L. Medeiros, Keyur P. Patel, Gokce A. Toruner
          </td>
          <td>2025-10-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a1d7b33f411a72234eadba08fcc3ddda8c8f36" target='_blank'>
              Integrative single-cell and bulk RNA-seq analysis identifies glycosyltransferases-related signature in triple negative breast cancer
              </a>
            </td>
          <td>
            Junyi Hu, Ningning Yuan, Zhenglan Huang, Yancheng Liu, Bohan Tu, Xinyue Yu, Tianli Hui, Guowei Zuo
          </td>
          <td>2025-11-08</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff9dd6e8b4e7b70e0de9451cb1c5dd37501bb1c" target='_blank'>
              A comprehensive genomic framework for identifying genes predisposing to homologous recombination repair deficient breast cancer
              </a>
            </td>
          <td>
            José Camacho-Valenzuela, Thibaut Matis, Carla Roca, Jorge Luis, Cuamatzi Flores, N. Hamel, Barbara Rivera, Simon Gravel, P. Polak, Carla Daniela, Robles-Espinoza, William D. Foulkes
          </td>
          <td>2025-10-23</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Shillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Mutator phenotypes are short-lived due to the rapid accumulation of deleterious mutations. Yet, recent observations reveal that certain fungi can undergo prolonged accelerated evolution after losing genes involved in DNA repair. Here, we surveyed 1,154 yeast genomes representing nearly all known yeast species of the subphylum Saccharomycotina (phylum Ascomycota) to examine the relationship between reduced gene repertoires broadly associated with genome stability functions (eg DNA repair, cell cycle) and elevated evolutionary rates. We identified 3 distantly related lineages—encompassing 12% of species—that had both the most streamlined sets of genes involved in genome stability (specifically DNA repair) and the highest evolutionary rates in the entire subphylum. Two of these “faster-evolving lineages” (FELs)—a subclade within the order Pichiales and the Wickerhamiella/Starmerella (W/S) clade (order Dipodascales)—are described here for the first time, while the third corresponds to a previously documented Hanseniaspora FEL. Examination of genome stability gene repertoires revealed a set of genes predominantly absent in these 3 FELs, suggesting a potential role in the observed acceleration of evolutionary rates. In the W/S clade, genomic signatures are consistent with a substantial mutational burden, including pronounced A|T bias and endogenous DNA damage. Interestingly, we found that the W/S clade also contains DNA repair genes possibly acquired through horizontal gene transfer, including a photolyase of bacterial origin. These findings highlight how hypermutators can persist across macroevolutionary timescales, potentially linked to the loss of genes related to genome stability, with horizontal gene transfer as a possible avenue for partial functional compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6d95c002d856c172eff01b267c3c0451ed62ab" target='_blank'>
              Stable Hypermutators Revealed by the Genomic Landscape of Genes Involved in Genome Stability Among Yeast Species
              </a>
            </td>
          <td>
            Carla Gonçalves, J. Steenwyk, D. Rinker, D. A. Opulente, A. Labella, Marie-Claire Harrison, John F. Wolters, Xiaofan Zhou, Xingxing Shen, S. Covo, M. Groenewald, C. T. Hittinger, A. Rokas
          </td>
          <td>2025-11-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Abstract Background Unlike other cancer types the determinants of anti-PD-1 response in clear cell RCC are not well defined; ccRCC exhibits a low mutational burden and CD8+ T cells infiltration is associated with worse prognosis. Cytomegalovirus (CMV) and human papillomavirus (HPV) infections are postulated to modulate the anti-tumor immune responses within the tumor microenvironment. We sought to investigate the clinical and immunologic impact of CMV and HPV viral transcriptomic signatures, as defined by prior studies, in patients with RCC. Methods RNA-seq data from three RCC prospetive trials were upper-quartile normalized, log2-transformed (TPM + 1), and batch-corrected using ComBat. Transcriptomic scores for CMV (75 genes) and HPV (79 genes) were computed using geometric means (71 and 77 genes were found, respectively). CD8 immunofluorescence (IF) was performed on tumor samples. Overall survival (OS) and progression-free survival (PFS) were assessed using multivariable Cox regression, adjusting for age at therapy start, gender, IMDC risk, sarcomatoid/rhabdoid features, and number of previous lines received. Results A high intratumoral CMV transcriptomic score was significantly correlated with T-effector signature (ρ  =  0.54, p = 4.8 × 10−6), IF tumor core CD8+ (ρ  =  0.43, P < .001), IF tumor margin CD8+ (ρ  =  0.41, P < .001), and myeloid signature (ρ  =  0.40, P < .001). The CMV score was significantly higher in infiltrated versus excluded/desert tumors (P = .0079). The CMV score was associated with worse OS among patients receiving immune checkpoint blockade (ICB) (HR 4.98, P = .039). In contrast, the HPV transcriptomic score showed only a modest correlation with myeloid signature (ρ  =  0.24, P = .051) and no association was found with OS nor PFS. Conclusions In metastatic RCC, a high tumor CMV transcriptomic score was associated with a paradoxical immune landscape, marked by CD8+ infiltration yet co-enriched with suppressive myeloid signatures, that predicts inferior overall survival following ICB. These viral signatures within the complex immune architecture of RCC may help understand immune responsiveness and resistance within distinct genomic and therapeutic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c3c656ee910ac2c2d9287a96ba1a9501c97839" target='_blank'>
              39Intratumoral viral transcriptomic signatures stratify immune phenotypes and clinical outcomes in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            Mustafa Saleh, E. Saad, Pablo Barrios, M. Machaalani, Liliana Ascione, W. Daoud Khatoun, Jad El Masri, Razane ElHajj Chehade, Marc Eid, Rashad Nawfal, Karl Semaan, E. Yekeduz, Clara Steiner, Weiwei Bian, Maxine Sun, S. Signoretti, E. V. Van Allen, D. Braun, Alexander Gusev, Toni Choueri
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="The biological mechanisms driving the long survival in glioblastoma (GBM). Five‐year long‐term survival (LTS) and 10‐year survival very long‐term survival (VLTS) remain significantly understudied. Here we molecularly detailed two cases. AR10‐046 (VLTS) was affected by a giant cell‐GBM, classified as the pedHGG_RTK1a subtype according to the v12.5 Heidelberg brain tumor methylation classifier. Somatic and germline MSH6 mutations, typically in Lynch syndrome, and high tumour mutational burden were detected. The copy number variation plots showed chromosome 1q gain and chromosome 13 loss with no other typical GBM alterations. AR10‐037 (LTS) suffered from a classical GBM, identified as pedHGG_MYCN subclass. Apart from the canonical chromosome 7 gain and chromosome 10q loss, we observed MDM2 gene amplification and possible rearrangements on chromosome 12 and 18 with the typical aspect of chromothripsis, harbouring two putative new gene fusions: CPSF6::CPM and PTPRR::RAB3IP. We described two patients with peculiar tumour molecular profile, widening the scenario of clinical and molecular variability in such patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8551ad58743a80e7a359a8ec1bcd9af76ef1c7" target='_blank'>
              Unique Genetic and Epigenetic Alterations in Glioblastoma Long‐Term Survivors: Insights From Two Clinical Cases
              </a>
            </td>
          <td>
            E. Anghileri, E. Miele, S. Patrizi, S. Barresi, E. Lazzarini, L. Maddaloni, M. Patanè, L. Pedace, R. Paterra, A. Silvani, Franco Locatelli, S. Indraccolo, B. Pollo
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0025e7aa342389077db174680eb853f811df7b03" target='_blank'>
              Liquid biopsy epigenetics: establishing a molecular profile based on cell-free DNA.
              </a>
            </td>
          <td>
            C. T. Maansson, A. L. Nielsen, B.S. Sørensen
          </td>
          <td>2025-11-21</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The rate of DNA synthesis is crucial for full DNA duplication. We report a key role of p21 in controlling this rate. During normal replication, p21 promotes nascent DNA synthesis alongside the DNA polymerase iota (Pol ι)/p53 complex. When p21 is down-regulated but detectable, nascent DNA tracks are longer and discontinuous and rely on primase and DNA polymerase (PrimPol). With the complete elimination of p21, nascent DNA tracks become shorter and continuous and depend on Pol kappa (κ). Endogenous p21 levels are critical for genomic stability, as both PrimPol- and Pol κ–mediated syntheses can induce chromosomal instability. The residual expression of p21 in p53-null cells influences the involvement of PrimPol or Pol κ in nascent DNA synthesis and subsequent chromosomal instability. Our results demonstrate that endogenous levels of p21 in cycling cells, insufficient for cyclin-dependent kinase inhibition, prevent genomic instability through proliferating cell nuclear antigen binding (PCNA), limiting PrimPol and Pol κ’s role in nascent DNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31f81e290a07076ab9200968e4922ab77b5675a" target='_blank'>
              Endogenous p21 levels protect genomic stability by suppressing both excess and restrained nascent DNA syntheses
              </a>
            </td>
          <td>
            N. L. Calzetta, S. Mansilla, Candelaria Mares Ahlers, Agostina P. Bertolin, Lilen I. Caimi, Sofía Venerus Arbilla, Jingkun Zeng, Lisa Wiesmüller, V. Gottifredi
          </td>
          <td>2025-10-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 The genomic regions surrounding tumor suppressor genes are often lost as collateral damage during tumorigenesis, providing novel therapeutic vulnerabilities. To systematically identify synthetic lethal pairs of genes within DNA damage repair (DDR) pathways, we conducted a combinatorial genome-wide CRISPR screen. We found that RNASEH2B, a gene frequently co-deleted with RB1 in prostate cancers, emerged as a top vulnerability with knockout of APEX2, which is a nuclease involved in DNA repair. This interaction was validated in RNASEH2B isogenic cell line pairs using whole-genome CRISPR screens and further confirmed across multiple cancer lineages using a focused mini-pool sgRNA library. Single gene knockouts demonstrated that loss of APEX2, specifically its catalytic activity, is sufficient to eliminate RNASEH2B mutant cells both in vitro and in vivo. Importantly, loss of RB1 did not impact the synthetic lethal interaction between RNASEH2B and APEX2. Consistent with prior reports, APEX2 knockout also impairs growth of BRCA1 mutant cells and enhances sensitivity to PARP inhibitors. Overall, these findings position APEX2 as a compelling synthetic lethal target in tumors with RNASEH2B deletion or BRCA1 mutations and support the development of APEX2 inhibitors as a potential monotherapy to improve patient outcome.



 Antoine Simoneau, Steve Lombardo, Ruthie Swain, Serge Gueroussov, Binzhang Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H. Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, Douglas A. Whittington, Yi Yu, Jannik N. Andersen, Teng Teng. APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f840d30e01fa153da53003edd382b7d06deea42" target='_blank'>
              Abstract B039: APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors
              </a>
            </td>
          <td>
            Antoine Simoneau, Steven A. Lombardo, Ruthie Swain, Serge Gueroussov, B. Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, D. Whittington, Yi Yu, Jannik N Andersen, T. Teng
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Significance How new genes arise and gain function is a central question in biology. New genes can evolve from nongenic DNA, yet their adaptive potential remains unclear. Here, we use millions of (semi-)random sequences as experimental models of emerging genes and find that thousands confer phage resistance in Escherichia coli. Expressed random sequences can produce both protein- and RNA-based functions that reprogram cellular systems to counter viral infection. Resistance arises through activation of a cell envelope stress response or downregulation of membrane receptor expression. Our results reveal that genetic novelty, in the form of genes appearing for the first time, can shape host–virus interactions, providing insight into microbial evolution and the surprising ease with which functional genes can emerge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c17bf5a40c5c5bfce399a75abe69e8bb312d9" target='_blank'>
              Emergence of antiphage functions from random sequence libraries reveals mechanisms of gene birth
              </a>
            </td>
          <td>
            Idan Frumkin, Christopher N Vassallo, Yi Hua Chen, Michael T. Laub
          </td>
          <td>2025-10-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification has emerged as a critical epigenetic mechanism regulating gene expression in diverse physiological and pathological processes, including cancer. Methyltransferase-like 16 (METTL16), a recently identified m6A methyltransferase, has been shown to influence tumor progression through m6A-dependent regulation of key target genes. Accumulating evidence indicates that METTL16 exerts tumor-suppressive or tumor-promoting roles in a context-dependent manner, affecting cell proliferation, apoptosis, autophagy, and chemotherapeutic response across multiple cancer types such as bladder cancer, lung cancer, colorectal cancer, and acute myeloid leukemia. Mechanistically, METTL16 modifies the mRNA stability and translation of oncogenes or tumor suppressors via recognition of m6A sites, and its expression can be regulated by upstream factors including transcription factors and hypoxia-inducible signals. Recent evidence suggests that METTL16 also modulates the tumor microenvironment (TME), potentially affecting immune cell infiltration, immune checkpoint expression, and tumor immune evasion. Collectively, METTL16 emerges as a pivotal epitranscriptomic regulator linking RNA modification, tumor progression, and immune modulation, offering new avenues for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7cac623df0454861cf59d7b80f15a7189f66d09" target='_blank'>
              METTL16 emerges as a pivotal epitranscriptomic regulator, linking RNA modification, tumor progression, and immune modulation
              </a>
            </td>
          <td>
            Qiang Wang, Xiulin Jiang, Yixiao Yuan, Chunhong Li
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Retinoblastoma (RB) is the most common pediatric eye cancer. Most cases of RB are initiated by bi-allelic mutational inactivation of the RB1 gene, yet most RB tumors harbor additional genomic aberrations that may promote tumor progression. After RB1, the gene that is most commonly mutated gene in RB is BCOR, which is mutated in approximately 20% of RB tumors and is associated with a more aggressive tumor phenotype and worse patient outcomes. Despite its importance, little is known about the role of BCOR in RB. Here, we interrogated BCOR in low passage RB cell lines using mass spectrometry, chromatin immunoprecipitation sequencing, and RNA sequencing. We show that the BCOR protein interacts with members of the ncPRC1.1 Polycomb Repressive Complex and localizes at gene loci with traditionally activating and repressing chromatin markers. Loss of BCOR downregulates the expression of genes associated with cell cycle regulation and upregulates genes associated with hypoxic adaptation. We conclude that BCOR mutations slow cell proliferation and drive hypoxic adaptation in RB via epigenetic mechanisms that may be amenable to targeted therapy. Implications: This study reveals that BCOR may play a noncanonical, multi-faceted role in retinoblastoma with implications in cell cycle, differentiation, and hypoxic adaptation, ultimately shedding insight into its molecular framework for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba892d255c691eee277b20e16deae8de79fd0b76" target='_blank'>
              BCOR mutations deregulate cell cycle and hypoxic adaptation pathways in retinoblastoma.
              </a>
            </td>
          <td>
            Michelle G. Zhang, J. Kuznetsoff, Natalie C Cetta, Sebastian Salazar, Renata L Volonterio, Stefan Kurtenbach, Z. Corrêa, Daniel Pelaez, J. W. Harbour
          </td>
          <td>2025-11-05</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. Y. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Inherited mutations in BRCA1 are among the primary causes of hereditary breast and ovarian cancer (HBOC). Genetic testing has identified numerous pathogenic and benign mutations in BRCA1, but also thousands of variants of uncertain significance (VUS) with unclear functional consequences. Accurate cancer risk prediction for carriers of these VUS requires functional assays to assess their pathogenicity. In this study, we analyzed 16 BRCA1 VUS detected at the Hereditary Cancer Genetic Counseling Laboratory in Salamanca, Spain, with the goal of providing additional data to guide their reclassification. Since homologous recombination (HR) is the primary tumor-suppressive function of BRCA1, we employed two complementary HR repair assays to evaluate HR efficiency. The first, an already established assay, uses a HeLa-DR cell line harboring an HR reporter cassette and involves silencing endogenous BRCA1 followed by complementation with the VUS under study. The second, developed in this work, is also based in complementation assays that restore the green fluorescence protein (GFP) gene, and employs a breast cancer cell line with a similar HR reporter system (HCC1937-HR) that lacks BRCA1 expression, allowing direct complementation. Both assays consistently identified five VUS (p.V11G, p.H888Y, p.G1201S, p.Q1395R, and p.F1734L) as pathogenic due to significantly reduced HR efficiency, whereas the remaining 11 VUS were benign in terms of HR function. Notably, two pathogenic variants (p.H888Y, p.G1201S) were located outside the known functional domains of BRCA1. The five HR-deficient variants were further evaluated for their sensitivity to ionizing radiation, which confirmed the deleterious impact on DNA repair for variants p.V11G, p.H888Y, p.G1201S and p.F1734L. Sensitivity to the PARP inhibitor olaparib revealed hypersensitivity only in cells expressing p.V11G and p.F1734L, suggesting variant-specific mechanistic effects with potential therapeutic relevance. In conclusion, recombination-based functional assays complemented with sensitivity assays are effective tools for assessing the pathogenicity of BRCA1 VUS and can provide valuable information to support clinical decision-making. We propose that these assays can be completed within a timeframe compatible with clinical needs, offering critical insights for genetic counseling and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c220541b6ad477ac28279a59bfc46ba5327a7db4" target='_blank'>
              Functional characterization of BRCA1 variants of unknown significance using homologous recombination repair assays
              </a>
            </td>
          <td>
            Paloma Martín-Bejarano, E. M. Sánchez-Tapia, Pérez Jessica, Teresa Martín-Gómez, Rosario Vidal Tocino, Rogelio González-Sarmiento, A. B. Herrero
          </td>
          <td>2025-10-07</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 In frame fusions between Fibroblast Growth Factor Receptor 3 (FGFR3) and Transforming Acidic Coiled-Coil containing protein 3 (TACC3) genes on chromosome 4p16.3 have been reported across diverse cancers, including bladder cancer (∼5%) and IDH-wildtype glioblastoma (4.1%), as well as in small subsets of non-small cell lung (∼0.5%), esophageal squamous cell (1.4–2.1%), cervical (1.9%), head and neck (3.7%), and rare nasopharyngeal (2.5%) carcinomas, and other cancer types. These chromosomal translocations generate FGFR3-TACC3 (F3T3) driver oncofusion proteins which promote tumor cell growth and survival. The F3T3-driven cancer cells are critically dependent on the constitutively active oncofusion kinase activity for survival and proliferation, a phenomenon known as "oncogene addiction". While inhibitors of FGFR kinases, such as erdafitinib and fexagratinib, are the treatment of choice for F3T3-driven cancers, most treated patients do not respond to the drug, suggesting insufficient inhibition of the oncofusion activity. Additionally, most tumor responses to FGFR kinase inhibitors are partial and short-lived, followed by development of resistance and disease progression within few months. Based on the oncogene addiction of cancer cells on the F3T3 activity, we hypothesized that rapid ablation of this oncofusion protein can induce robust apoptotic response, leading to significant tumor regression and sustained remissions. To this end, we explored the use of Proteolysis Targeting Chimeric (PROTAC) bifunctional small molecules which can rapidly degrade F3T3 oncofusion protein by harnessing the cell's own proteasome machinery. We synthesized more than 100 novel PROTAC small molecules targeting F3T3 fusion through combination of previously reported TACC3 ligands with various linkers and ligands of E3 ubiquitin ligases CRBN, VHL or XIAP. We evaluated our new anti-F3T3 PROTAC molecules using F3T3-driven bladder (RT4 and RT112) and breast (SUM185PE) cancer cell lines, which express two of the most common oncofusion variants. Several PROTACs were able to degrade the native F3T3 oncofusion protein and suppressed the growth of F3T3-driven cells. One PROTAC (KHS-C4) degraded F3T3 at low nanomolar concentrations, strongly suppressed the oncogenic pathway downstream F3T3 fusion, had excellent predicted pharmacological properties, acceptable in vitro liver microsome stability (Clint = 59 mL/min/kg for human and 318 mL/min/kg for mouse microsomes) and no in vivo toxicity. Importantly, low concentrations of KHS101-C4 (but not of FGFR3 kinase inhibitors) eradicated F3T3-driven cancer cells (but not cancer cells that do not express the F3T3 oncofusion) in vitro. These data suggest that chemically-induced degradation of the F3T3 fusion protein can lead to oncogenic signal ablation sufficiently potent to induce a lethal oncogenic shock and can thus yield more durable responses than FGFR3 kinase inhibitors. KHS-C4 was selected for further evaluation to establish in vivo proof of concept for the PROTAC-based therapeutic strategy against F3T3-driven cancers.



 Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea. Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5bb4e0b8d5df4124ad0dd08279a3f1803abead8" target='_blank'>
              Abstract B120: Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The onset of acquired resistance limits the efficacy of KRASG12C inhibitors in lung cancer patients. Experiments in preclinical model systems and retrospective analyses of patients treated with these inhibitors have both suggested that the activation of DNA repair pathways and G2/M kinases is common in lung cancers with acquired resistance to KRASG12C inhibitors. Here, we identified a shared vulnerability to the CDK12/13 selective inhibitor SR-4835 in several KRASG12C-mutant cell lines with acquired resistance to sotorasib. In pairs of KRASG12C-mutant parental and sotorasib-resistant cell lines, CDK12/13 inhibition suppressed DNA repair gene expression and induced mitotic arrest, which was exacerbated in sotorasib-resistant cells by failure to activate the G2/M checkpoint and suppression of an oncogenic isoform of TP73. Combined treatment with both sotorasib and SR-4835 yielded an additive response and suppressed the development of acquired resistance to either inhibitor. The inhibitor combination caused additive G1 and G2/M arrest, and the increased sensitivity of emerging drug-resistant cell populations to SR-4835 extended the duration of response over single-agent treatment in cell culture and in mouse models. These results support combining KRAS and CDK12/13 inhibitors to extend the duration of response in KRAS-mutant lung cancer. Patients with acquired resistance to sotorasib may benefit from follow-up monotherapy with CDK12/13 inhibitors, the first of which recently entered clinical trials. Targeting CDK12/13 thus offers a promising strategy to overcome or prevent resistance to KRAS inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5df2549b13618f9401530fb07b8c41d871bb9c6" target='_blank'>
              Targeting CDK12/13 Drives Mitotic Arrest to Overcome Resistance to KRASG12C Inhibitors.
              </a>
            </td>
          <td>
            Yaakov E. Stern, Pompom Ghosh, John Peroza, Hitendra S Solanki, Denis Imbody, Liznair Bridenstine, Thiyagamurthy Pandurangan, Sylvia M Frydman, Kathryn N Nickens, Hannah L Walker-Mimms, Kruthi Suvarna, Neelkamal Chaudhary, Andrii Monastyrskyi, Derek Duckett, E. Haura
          </td>
          <td>2025-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Reprogramming gene expression at the translational level drives intestinal tumorigenesis. Codon decoding during translation elongation relies on tRNA modifications, while their pathological relevance in colorectal cancer remains to be elucidated. Here, we show that AlkB homolog 8 (ALKBH8), a uridine 34 (U34) tRNA methyltransferase, is a direct target of Wnt/β-catenin and is upregulated in colorectal cancer. Genetic ablation of ALKBH8 inhibits the development of intestinal tumors in Apcmin/+, azoxymethane/dextran sulfate sodium (AOM/DSS), and xenograft models. Loss of ALKBH8 induces ribosome pausing at adenine-ending codons, impairing the translation elongation of mRNAs enriched with these codons. Specifically, ALKBH8 regulates the translation of KRAS proto-oncogene in a codon-dependent manner. Rescue experiments demonstrate that the methyltransferase activity of ALKBH8 is required for its translation-promoting function. Together, our findings reveal ALKBH8-dependent mRNA translation as a critical mediator of intestinal tumorigenesis, underscoring its potential as a promising target for colorectal cancer therapy. ALKBH8 is a tRNA methyltransferase, specifically modifying the wobble uridine (U34) in tRNAs. Here, the authors discover that ALKBH8 is a direct transcriptional target of β-catenin, and that mediates intestinal tumorigenesis through regulation of the translation of KRAS in a codon-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741e7e4c4eea499624351c733df3953216fc634e" target='_blank'>
              ALKBH8-mediated codon-specific translation promotes colorectal tumorigenesis
              </a>
            </td>
          <td>
            Yu Qian, Canlan Wu, Saisai Wei, Sujun Yan, Junxuan Peng, Lei Yu, Yunyi Gao, Jingyu Hou, Wentao Yu, Zhanghui Chen, Jun Zhang, Xiangwei Gao
          </td>
          <td>2025-10-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Centromeres are the chromosomal sites at which the kinetochore forms and attaches to spindle microtubules, directing chromosome segregation. Plasmids based on centromeres can maintain stability and distribute accurately during cell division, supporting them as effective genetic tools for research and biotechnological applications. Here, the centromeric regions on seven chromosomes of Ogataea polymorpha, a methylotrophic non‐conventional yeast with great potential in biotechnology, were located by ChIP‐seq with the native Cse4. The actual centromeric sequences of chromosomes 1, 2 and 5 were characterised, which are very different from those of other eukaryotes and each unique. Although long terminal repeats (LTR) were found in these centromeres, they are ‘solo LTR elements’ and not important for the function of centromeres. Hence, the O. polymorpha centromeres were categorised into small regional centromeres. O. polymorpha centromeric plasmids were constructed for the first time, which exhibit high genetic stability and compatibility. Application potential for multigene or multicopy expression was validated by the production of uricase and triterpene squalene. This research elucidates the structural features of O. polymorpha centromeres and constructs new, stable centromeric plasmids, expanding the phenomenal diversity of centromeres and providing a powerful genetic toolbox for multi‐node and multiple‐layered engineering of cell metabolism and physiological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8595896eccb7f21123d64f19b9f60bcb2fa63adb" target='_blank'>
              Centromeric Sequences in Ogataea polymorpha Genome Enable Development of Stable Multigene Expression Plasmid Tools
              </a>
            </td>
          <td>
            Yanfei Cheng, Yuanyuan Shi, Ning Sun, Xin Zhang, Mengwei Sun, Xiuping He
          </td>
          <td>2025-11-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Early disease progression, including histological transformation, can occur in follicular lymphoma, but tumor heterogeneity makes comparison difficult. We analyzed cell-free DNA from 30 patients with follicular lymphoma to assess cell-free DNA concentration and gene mutations. Mutational profiles were also examined in 16 cases using DNA from formalin-fixed, paraffin-embedded tissue samples. Cell-free DNA concentrations were higher in cases with histological transformation. Both the number of mutations per patient and the number of mutated genes were greater in cell-free DNA than in formalin-fixed, paraffin-embedded tissue-derived DNA. Notably, TP53 mutations were more frequently detected in cell-free DNA. CIITA mutations were more commonly observed in cases with a low frequency of somatic hypermutation in VH genes. Cell-free DNA analysis in follicular lymphoma may become a complementary approach, overcoming the limitations of assessing disease status from single biopsy sites. Further studies will examine the usefulness of cell-free DNA in predicting disease progression in follicular lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc0cb12618903a19d09041fcad50558a34c24c" target='_blank'>
              Genetic analysis of cell-free DNA in follicular lymphoma in comparison with tissue-derived DNA.
              </a>
            </td>
          <td>
            Yoshikazu Hori, Hiroki Hosoi, M. Osuga, Ryuta Iwamoto, Fumiyo Maekawa, Tadashi Okamura, S. Murata, T. Mushino, Motomi Osato, Hitoshi Ohno, Nobuyuki Yamamoto, Shin-ichi Murata, Yasuhiro Koh, Sonoki Takashi
          </td>
          <td>2025-10-15</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) remains the most aggressive primary brain tumour in adults, marked by pronounced cellular heterogeneity, diffuse infiltration, and resistance to conventional treatment. In recent years, transcriptomic profiling has provided valuable insights into the molecular mechanisms that govern the progression of glioblastoma. This systematic review aims to synthesise the current literature on dysregulated gene expression in GBM, focusing on gene signatures associated with stemness, immune modulation, extracellular matrix remodelling, metabolic adaptation, and therapeutic resistance. Methods We conducted a systematic search of PubMed, The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and the GlioVis portal for studies published between January 2005 and April 2025, limited to English-language reports. Studies were eligible if they included adult glioblastoma tissue or patient-derived datasets and reported gene-level expression or clinical associations. Reviews, commentaries, and studies on non-GBM gliomas were excluded. Screening followed the PRISMA 2020 checklist, with 410 records initially identified, 90 duplicates removed, and 125 studies retained after full-text review. Data were synthesised descriptively, and findings were validated against TCGA/CGGA expression datasets to ensure consistency across cohorts. Results We categorised recurrently dysregulated genes by their biological function, including transcription factors (SOX2, ZEB2), growth factor receptors (EGFR, PDGFRA), immune-related markers (PD-L1, TAP1, B2M), extracellular matrix regulators (MMP2, LAMC1, HAS2), and metabolic genes (SLC7A11, PRMT5, NRF2). For each group, we examine the functional consequences of transcriptional alterations and their role in driving key glioblastoma phenotypes, including angiogenesis, immunosuppression, invasiveness, and recurrence. Conclusion We further discuss the prognostic implications of these gene signatures and evaluate their potential utility in precision medicine, including current clinical trials that target molecular pathways identified through transcriptomic data. This review highlights the power of gene expression profiling to stratify glioblastoma subtypes and improve personalised therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f178b970f37a5487e44c1e368727e8a6a513f43" target='_blank'>
              RNA Expression Signatures in Glioblastoma: A Systematic Review of Tumour Biology and Therapeutic Targets
              </a>
            </td>
          <td>
            Amber Hassan, Badr Hafiz, Taghreed A. Alsinani, Rakan Bokhari, D. Mirdad, Awab T. Tayyib, Alaa Alkhotani, Ahmad Fallata, Iman Mirza, Eyad Faizo, Saleh Baeesa, Huda Alghefari, Maher Kurdi
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In the initiation and development of cancer, DNA damage plays a very crucial role. In the case of environmental perturbation, mutation events in critical control genes may occur as the integrity of genetic material is lost, either due to replication errors, oxidative stress, etc. And provided that these are proto-oncogenes or tumour suppressor genes, they can interfere with normal control of the cell cycle, programmed cell death (apoptosis) and repair of damaged DNA(Branzei & Foiani, 2023). Consequently, they can start to multiply out of control, which can result in the appearance of growth and malignancy. This abstract discusses the molecular processes that connect the occurrences of DNA damage and mutations with the out-of-control growth of cancer cells, pointing to their importance in the development of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cd781c4b40bc6486e36eec6ae725a99f2c36b2" target='_blank'>
              Exploring How DNA Damage And Mutations Lead To The Uncontrolled Growth Of Cancer Cells?
              </a>
            </td>
          <td>
            Yashvi Chhabra
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Scientific and Research Publications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, G. Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, J. E. Meyer, S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery.
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Platinum-based drugs (Pt drugs) are widely used in cancer chemotherapy, yet their genome-wide binding patterns remain incompletely understood. Here, we present Pt sequencing (Pt-seq), an antibody-assisted, genome-wide method for mapping Pt-DNA adducts at single-base resolution. By using exo- and endonucleases to remove background, Pt-seq enables robust and sensitive profiling of binding sites for cisplatin, oxaliplatin, lobaplatin, and a Pt(IV) complex. Using Pt-seq, we identified tens to hundreds of binding clusters that are 10 to 20 kilobases in median length and highly consistent among different drugs, predominantly localizing to centromeric and ribosomal DNA regions. In cisplatin-resistant cells, we found significantly reduced binding within these regions. Moreover, we found that mutations in cancer cells can generate previously unidentified binding sites. On this basis, we demonstrated that ICR-191, an acridine orange compound capable of inducing G insertions, enhances cisplatin-DNA binding and sensitizes cells to cisplatin. Collectively, Pt-seq sensitively profiles Pt–DNA interactions and deepens our understanding of the genome-wide effect of chemotherapeutic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8b8af89d2820c69ce2eeb05e85318ac412b23f" target='_blank'>
              Pt-seq unveils the genomic binding pattern of platinum-based drugs
              </a>
            </td>
          <td>
            Chuyun Shao, Mingyang Li, Liuyi Liu, Dongsheng Bai, Jinmin Yang, Cong Liu, Linxiang Cai, Xiaoyong Wang, Jinying Peng, Zijian Guo, C. Yi
          </td>
          <td>2025-10-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive clonal hematopoietic malignancy, characterized by marked biological heterogeneity and variable clinical outcomes. Among its rarer genetic subsets is AML with rearrangements of the MDS1 and EVI1 complex locus (MECOM), occurring in fewer than 2% of newly diagnosed cases. This review examines the biology and clinical significance of MECOM-rearranged AML, with a focus on its diverse mechanisms of leukemogenesis, including chromosomal inversion and translocation involving 3q26. We discuss how aberrant EVI1/MECOM activity alters gene expression networks and drives malignant transformation. Current therapeutic approaches—including intensive chemotherapy, hypomethylating agents in combination with venetoclax, and allogeneic stem cell transplantation—are evaluated with particular emphasis on inv(3) and other t(3q26) subtypes. Despite these treatment strategies, outcomes remain poor, underscoring the urgent need for novel, more effective therapies for this high-risk form of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26f62836eceb821380878862a26c45df4a04887" target='_blank'>
              MECOM-Rearranged Acute Myeloid Leukemia: Pathobiology and Management Strategies
              </a>
            </td>
          <td>
            Utsav Joshi, R. Shallis
          </td>
          <td>2025-10-31</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to establish divergent gene regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under "cellular constraint". We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and noncoding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share signals of positive and negative selection, suggesting a dynamic role in influencing human traits and cellular phenotypes. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd414947d1caaf71463120749f3e2d01e73021fa" target='_blank'>
              Conserved facultative heterochromatin across cell types identify regulatory sequences underpinning cell identity and disease.
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, , Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2025-10-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39534227ca18263af64802892a94b5211231b6b" target='_blank'>
              Failure of DNA repair leads to the tumorigenic transformation of a regenerating blastema in Drosophila imaginal discs
              </a>
            </td>
          <td>
            Tanroop Kaur, Ada Repiso, Ying Liu, Paula Doria, Joan Urgell, Luis Rodriguez-Escudero, Katerina Karkali, Yanhui Hu, N. Perrimon, E. Martín-Blanco
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84210062581c67c09a67215ecb3a57b62da836b" target='_blank'>
              Adenine DNA methylation associated with transcriptionally permissive chromatin is widespread across eukaryotes.
              </a>
            </td>
          <td>
            Pedro Romero Charria, Cristina Navarrete, Vladimir Ovchinnikov, Lan Xu, Luke A. Sarre, Victoria Shabardina, Ewa Księżopolska, Elena Casacuberta, David Lara-Astiaso, Arnau Sebé-Pedrós, Alex de Mendoza
          </td>
          <td>2025-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b086f25cb86ff33c2e536b095d10812da2afa3" target='_blank'>
              POLR3A mutations cause nucleolus abnormalities and aberrant telomerase RNA metabolism in induced pluripotent stem cells from Wiedemann-Rautenstrauch premature aging syndrome patient
              </a>
            </td>
          <td>
            Evgeniya A Orlova, V. K. Abdyev, V. Morgunova, Anna A Schukina, Anastasiia L. Kungurtseva, Peter A Vasiluev, V. Tabakov, Ekaterina O Vorontsova, Elena V Zinina, Marina Izvolskaia, Marina E Minzhenkova, A. Vitebskaya, Alla Kalmykova
          </td>
          <td>2025-10-13</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de4f907940d94d84af062c9c3da4d52a4ad33d5" target='_blank'>
              Uncovering a role for METTL13 in malignant transformation of human hematopoietic stem cells and in the progression of pediatric leukemia
              </a>
            </td>
          <td>
            Frida Holm, Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, S. S. Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang
          </td>
          <td>2025-10-23</td>
          <td>Research Square</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Chromophobe renal cell carcinoma (ChRCC) is the second most common non-clear cell renal cell carcinoma (ncRCC). A series of 496 ChRCC from Memorial Sloan Kettering Cancer Center demonstrated that sarcomatoid features, which were present in about 1.2% of the cases, are strongly correlated with metastatic disease and reduced overall survival. Little is known about the biological and immunological characteristics of the sarcomatoid variant of ChRCC and how it differs from its classic, non-sarcomatoid form. Methods Eight ChRCC tumor samples containing both classic and sarcomatoid components were selected. Xenium Prime 5K assay (5000 curated genes related to oncoimmunology) was used for high-resolution spatial transcriptomic profiling. Data were analyzed using Seurat and custom scripts to annotate cell types and conduct detailed phenotypic analysis. Pathway enrichment and transcription factor regulatory networks were analyzed using the SENIC package. Cell-cell communication was inferred using CellChat, and spatial proximity of immune and tumor cells (“cellular neighborhoods”) was analyzed, which model spatial gene expression patterns and intercellular distances. Results Spatial transcriptomic analyses revealed a significantly more immune-inflamed tumor microenvironment in sarcomatoid regions compared to their classic counterparts. We identified distinct spatial immune “neighborhoods” in sarcomatoid areas that were enriched in CD8+ T cells, polarized macrophages, and dendritic cells. These neighborhoods exhibited organized proximity to tumor cells expressing immune evasion markers such as PD-L1. In contrast, classic regions were relatively immune-deserted. Analysis of transcription factor activity showed upregulation of IRF1 and STAT1-related programs in the immune infiltrates of sarcomatoid regions, consistent with inflammatory activation. Importantly, comparison of matched sarcomatoid and classic regions within the same tumors enabled us to define a “sarcomatoid-enriched gene signature” composed of immune-related, pro-metastatic, and epithelial-to-mesenchymal transition (EMT) genes. This signature was validated in bulk RNA-seq data from The Cancer Genome Atlas (TCGA), where it correlated with poor prognosis in ChRCC patients. Conclusions This study presents the first spatial transcriptomic atlas of sarcomatoid ChRCC in comparison to matched non-sarcomatoid (classic) from the same patient. Profound immune microenvironmental differences were identified between tumor compartments. The sarcomatoid variant exhibits hallmarks of an immune-inflamed but potentially immune-evasive phenotype, including enriched immune infiltration, pro-inflammatory signaling, and EMT-associated transcriptional programs. These findings highlight the biological differences between sarcomatoid and classic ChRCC and uncover specific immune pathways and spatial cell-cell interactions that may be targeted in future immunotherapeutic strategies. Our work also establishes a foundational spatial map for ChRCC that offers new insight into the disease heterogeneity and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aba1786fb42a662d7a501d0a751ee46eaa895c9" target='_blank'>
              52Spatial transcriptomic mapping reveals immune-inflamed niches in sarcomatoid chromophobe renal cell carcinoma
              </a>
            </td>
          <td>
            Yan Tang, Tiegang Han, Hadi Mansour, M. Hirsch, Katrina Collins, Elizabeth P. Henske
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Several autonomous mechanisms regulate protein expression, such as transcription, translation, post-translational modifications, and epigenetic changes. Rarely, these processes are controlled by the same molecular player with overlapping roles. Here, we reveal that transcription factor NFATc1 regulates both transcription and degradation of the Ca2+ channel Orai3 in a context-dependent manner. We demonstrate that NFATc1 drives Orai3 transcription in non-metastatic pancreatic cancer cells. In invasive and metastatic pancreatic cancer cells, NFATc1 induces Orai3 lysosomal degradation by transcriptionally enhancing MARCH8 E3-ubiquitin ligase. We show that MARCH8 physically interacts with Orai3 intracellular loop eventually resulting in its ubiquitination at the N-terminal. Mechanistically, the dichotomy in the regulation of Orai3 expression emerges from the differences in MARCH8 epigenetic landscape. We uncover that MARCH8 promoter is hyper-methylated in non-metastatic cells. Importantly, we demonstrate that MARCH8 restricts pancreatic cancer metastasis by targeting Orai3 degradation, thereby highlighting the pathophysiological importance of this signaling module. Taken together, we report a unique and clinically relevant scenario wherein the same transcription factor both enhances and curtails the expression of a target protein in cancer. By harnessing the epigenetic differences in the promoter of MARCH8 E3 ubiquitin ligase, the transcription factor NFATc1 drives Orai3 transcription and protein degradation, regulating pancreatic cancer in context-dependent manner. NFATc1 transcriptionally upregulates Orai3 Ca2+ channel in non-metastatic cells while inducing Orai3 lysosomal degradation via MARCH8 E3 ubiquitin ligase in metastatic cells. MARCH8 physically interacts with Orai3 and induces ubiquitination at the Orai3 N-terminal K2 residue. NFATc1’s dichotomous function derives from differences in the methylation status of the MARCH8 promoter. MARCH8 inhibits pancreatic cancer metastasis by driving Orai3 degradation. NFATc1 transcriptionally upregulates Orai3 Ca2+ channel in non-metastatic cells while inducing Orai3 lysosomal degradation via MARCH8 E3 ubiquitin ligase in metastatic cells. MARCH8 physically interacts with Orai3 and induces ubiquitination at the Orai3 N-terminal K2 residue. NFATc1’s dichotomous function derives from differences in the methylation status of the MARCH8 promoter. MARCH8 inhibits pancreatic cancer metastasis by driving Orai3 degradation. Dichotomous control by NFATc1 defines cell state-specific roles of the Ca2+ channel Orai3 in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc7315e3b7c364310dce4b187714fe3a0717601" target='_blank'>
              NFATc1 drives Orai3 transcription and proteolysis by harnessing epigenome differences in the MARCH8 promoter
              </a>
            </td>
          <td>
            Sharon Raju, Akshay Sharma, Sakshi Dahiya, Gyan Ranjan, Rajender K. Motiani
          </td>
          <td>2025-09-29</td>
          <td>The EMBO Journal</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Epithelioid hemangioendothelioma (EHE) is a difficult to treat vascular sarcoma defined by TAZ-CAMTA1 (TC) or YAP-TFE3 (YT) fusion proteins. Human cell lines needed to further understand the pathogenesis of EHE have been lacking. Herein, we describe a method to generate EHE extended primary cell cultures. An integrated multi –omic and functional approach was used to characterize these cultures. The cell cultures, relatively homogenous by single cell RNA-Seq, demonstrated established characteristics of EHE including increased proliferation, anchorage independent growth, as well as the overall gene expression profile and secondary genetic alterations seen in EHE. Whole genome sequencing (WGS) identified links to epigenetic modifying complexes, metabolic processes, and pointed to the importance of the extracellular matrix (ECM) in these tumors. Bulk RNA-Seq demonstrated upregulation of pathways including PI3K-Akt signaling, ECM/ECM receptor interaction, and the Hippo signaling pathway. Development of these extended primary cell cultures allowed for single-cell profiling which demonstrated different cell compartments within the cultures. Furthermore, the cultures served as a therapeutic platform to test the efficacy of TEAD inhibitors in vitro. Overall, the development of primary EHE cell cultures will aid in the mechanistic understanding of this sarcoma and serve as a model system to test new therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4728cebf1ce6a2c3cd2884745cce7881e7bc18e9" target='_blank'>
              EHE cell cultures are a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M Bellizzi, Anthony N. Snow, Patrick Breheny, M. Chimenti, M. Tanas
          </td>
          <td>2025-10-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is the leading cause of death in women under 50. The majority of breast cancers are estrogen receptor α-positive (ER+) and are commonly treated with hormonal therapies such as tamoxifen that inhibit ER activity. The TP53 tumor suppressor gene, encoding the p53 protein, is the most frequently mutated gene in breast cancer, and TP53 mutations are associated with diminished tamoxifen response and worse prognosis for breast cancer patients. Here, we report that in breast cancer cells p53 and ER cooperate to regulate the transcription of a set of genes encoding chromatin modifiers. The net result is a global increase in H3K4me3 and decrease in H3K9me3 chromatin marks. The resultant "open" chromatin is associated with increased transcription of luminal cell identity genes and enhanced tamoxifen sensitivity. Conversely, diminished p53 control of these chromatin modulators is associated with the evolution of tamoxifen resistance and cancer stem cell properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a29e75c12c65ca56f17dc6c777618735a34c52" target='_blank'>
              p53 regulates the expression of histone modifiers to restrict stemness and maintain differentiated luminal identity in breast cancer.
              </a>
            </td>
          <td>
            Yael Aylon, N. Furth, A. Pirona, Adi Lavie, Olga Fedorova, Ori Hassin, Nuno Padrão, Maxime Steinmetz, Avital Sarusi-Potuguez, Liat Fellus-Alyagor, Irit Shimoni, B. Dassa, W. Zwart, E. Shema, Moshe Oren
          </td>
          <td>2025-10-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT The E7 oncoprotein of mouse papillomavirus (MmuPV1) plays a pivotal role in both viral infection and cancer development. While earlier studies have identified key cellular targets of MmuPV1 E7, such as pRB and PTPN14, the broader impact of MmuPV1 E7 on keratinocyte homeostasis and shared activities with human papillomavirus (HPV) E7 remains unclear. In this study, we employed proteomic and transcriptomic analyses using our established mouse keratinocyte model—previously instrumental in uncovering a novel function of MmuPV1 E6—to investigate the biological consequences of MmuPV1 E7 expression in mouse keratinocytes. Our findings reveal that MmuPV1 E7 induces cellular changes reminiscent of those driven by “high-risk” HPV infection implicated in cervical cancer. Notably, MmuPV1 E7 did not activate canonical E2F-responsive gene expression or promote proliferation, reinforcing the idea that MmuPV1 E6 is the primary driver of cell cycle activation. However, MmuPV1 E7 expression led to a significant accumulation of stress keratin 17, a marker associated with immune evasion and elevated in both HPV16 transgenic models and MmuPV1 infections. Additionally, we observed enhanced PI3K-AKT-mTOR signaling, with increased levels of phosphorylated S6 kinase and heightened sensitivity to epidermal growth factor stimulation. Collectively, these results underscore the role of MmuPV1 E7 in promoting oncogenic phenotypes and highlight its relevance as a model for studying the molecular underpinnings of “high-risk” HPV-driven disease. IMPORTANCE In this study, we determined the ability of the MmuPV1 E7 oncoprotein in promoting disruption of keratinocyte homeostasis in mouse keratinocytes. Using a multiomics approach, we observed that MmuPV1 E7 promoted several phenotypes associated with “high-risk” human papillomavirus (HPV) infection. Specifically, we confirmed that MmuPV1 E7 does not increase E2F-responsive gene expression and proliferation of mouse keratinocytes. We did find that MmuPV1 E7 was able to increase the expression of stress keratin 17, which promotes immune evasion in papillomavirus infections. Finally, MmuPV1 E7 showed increased expression of genes associated with PI3K-AKT-mTOR signaling. Consistent with this observation, MmuPV1 E7-expressing mouse keratinocytes had elevated phosphorylation of S6 kinase. We also found that MmuPV1 E7 potentiates this signaling through increased sensitivity to epidermal growth factor stimulation. Our collective data show that MmuPV1 E7 promotes several phenotypes associated with “high-risk” HPV infection and cancers. In this study, we determined the ability of the MmuPV1 E7 oncoprotein in promoting disruption of keratinocyte homeostasis in mouse keratinocytes. Using a multiomics approach, we observed that MmuPV1 E7 promoted several phenotypes associated with “high-risk” human papillomavirus (HPV) infection. Specifically, we confirmed that MmuPV1 E7 does not increase E2F-responsive gene expression and proliferation of mouse keratinocytes. We did find that MmuPV1 E7 was able to increase the expression of stress keratin 17, which promotes immune evasion in papillomavirus infections. Finally, MmuPV1 E7 showed increased expression of genes associated with PI3K-AKT-mTOR signaling. Consistent with this observation, MmuPV1 E7-expressing mouse keratinocytes had elevated phosphorylation of S6 kinase. We also found that MmuPV1 E7 potentiates this signaling through increased sensitivity to epidermal growth factor stimulation. Our collective data show that MmuPV1 E7 promotes several phenotypes associated with “high-risk” HPV infection and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc11c43dd44972330a70bdfb67ca963643009c95" target='_blank'>
              MmuPV1 E7 promotes phenotypes associated with “high-risk” HPV infection in mouse keratinocytes
              </a>
            </td>
          <td>
            Kayla R. Duxbury, Liyan Zhang, Laura K. Muelhbauer, Mitchell Hayes, Joshua J. Coon, Megha Padi, James C. Romero-Masters
          </td>
          <td>2025-10-30</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) remains one of the most lethal malignancies, primarily due to its robust resistance to genotoxic therapies, such as chemotherapy and radiotherapy. Understanding the mechanisms underlying this resistance is essential to improve clinical outcomes. Here, we identified trimethylguanosine synthase 1 (TGS1), previously known for its role in RNA modification, as a critical mediator of homologous recombination (HR) repair that specifically contributes to resistance in PAAD. TGS1 was significantly overexpressed in PAAD tissues, correlating strongly with advanced disease stages, therapy resistance, and poor patient prognosis. Following DNA damage, ATM kinase phosphorylated TGS1 at serine residues S389 and S531, which mediated its direct interaction with BRCA1 and subsequent recruitment of BRCA1 to DNA damage sites. The phosphorylation-dependent interaction enhanced HR repair efficiency, enabling cancer cells to survive genotoxic stress. Depletion or pharmacological inhibition of TGS1 induced HR deficiency and markedly enhanced sensitivity to DNA-damaging agents, particularly PARP inhibitors, in PAAD cell lines in vitro and in both cell line-derived and patient-derived xenograft models in vivo. Collectively, these findings uncover an ATM-TGS1-BRCA1 signaling axis that promotes DNA repair and resistance to genotoxic therapies in pancreatic cancer, positioning TGS1 as a promising predictive biomarker and therapeutic target to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abe0d36d758071ca95a0b7efd96da1ea51c63685" target='_blank'>
              Targeting the ATM-TGS1-BRCA1 Axis Overcomes Genotoxic Therapy Resistance in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Changying Li, Xuran Zhao, Xinyan Li, Chunyan Wang, Zhenyu Huo, Xin Xu, Wenjuan Kang, S. Nowsheen, Khaled Aziz, Guangyi Sun, Zhihua Liu, Zhenkun Lou, Min Deng
          </td>
          <td>2025-11-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="G-quadruplexes have been identified as a promising anti-cancer target because of their ability to modulate the stability of mRNAs encoding oncogenes, tumor suppressor genes, and other potential therapeutic targets. Deregulation of DNA damage and Unfolded Protein Response pathways in cancer cells may create vulnerabilities that can be exploited therapeutically. Previous studies have shown variations in the relative expression of DDR and UPR components in canine lymphoma and leukemia cell lines CLBL-1, CLB70, and GL-1. In the present study, we report the presence of G-quadruplex structures in these canine cell lines. Downregulation of the expression of DDR and UPR components at the mRNA level was observed in the CLBL-1 and CLB70 cell lines after stabilization of G4 structures using the ligand PhenDC3. In contrast, in GL-1 cells, important components of the DDR pathway, such as PARP1, GADD45A, and PIK3CB were upregulated in response to PhenDC3 treatment. Downregulation of DDIT4 mRNA expression, which encodes an important UPR component, was detected in the CLBL-1 and GL-1 cell lines after PhenDC3 exposure. These results suggest that G4 structures can be used to manipulate the expression of potential targets to treat lymphoma in dogs. A substantial enrichment of DNA replication and pyrimidine metabolism pathways was found in the GL-1 cell line after G4 stabilization. This finding suggests that PhenDC3 may induce DNA replication stress in this cell line. Collectively, these results support the feasibility of employing canine cancer cells as a model system to investigate the role of G-quadruplex structures in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266b5cfaedc49c459cf184dd7b4aeb1ea558a388" target='_blank'>
              Stabilization of G-Quadruplexes Modulates the Expression of DNA Damage and Unfolded Protein Response Genes in Canine Lymphoma/Leukemia Cells
              </a>
            </td>
          <td>
            Beatriz Hernández-Suárez, David A. Gillespie, Ewa Dejnaka, B. Obmińska-Mrukowicz, A. Pawlak
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) embryos exhibit widespread epigenetic defects, particularly aberrant DNA methylation. DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development, yet its role in SCNT embryos remains largely unknown. Here, the genome-wide 5hmC landscapes in mouse SCNT embryos are systematically profiled with parental allele specificity. It is revealed that both maternal and paternal genomes of donor somatic cells acquire a transient, sperm-like but attenuated and allele symmetric distribution of 5hmC at the 2-cell stage, distinct from the parental asymmetric pattern observed in naturally fertilized eggs. This is characterized by insufficient DNA hydroxymethylation of the X chromosome in female SCNT embryos, as well as resistance to 5hmC-associated DNA demethylation at germline imprinting control regions (gICRs). While de novo 5hmC generation is closely associated with initial DNA demethylation during somatic-to-zygotic transition, it later becomes uncoupled from ongoing methylation changes. Importantly, global elevation of 5hmC via Tet3 overexpression leads to premature activation of developmental genes at the 2-cell stage and severely impairs SCNT embryo development. These findings reveal unique dynamics and functional consequences of abnormal 5hmC remodeling in SCNT embryos, highlighting the precise regulation of 5hmC generation as a key epigenetic event for successful mammalian cloning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e7a45e185e638b752c93b5629a29c7ec2760ec" target='_blank'>
              5-Hydroxymethylcytosine Dynamics Reveals Coordinated Reprogramming of Parental Genomes and X Chromosome Dosage Balance in Mouse SCNT Embryos.
              </a>
            </td>
          <td>
            Zeming Xiang, Rui Yan, Jing Guo, Mengyao Wang, Xin Cheng, Fan Zhang, Tianzi Guo, Xin Long, Fan Guo, Dan Liang
          </td>
          <td>2025-11-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43fc1a06533a0a2592fc97d8e105f6672f41ebb" target='_blank'>
              Somatic TYK2 activating mutations in tumor-infiltrating T cells promote anti-cancer immunity
              </a>
            </td>
          <td>
            Zhijie Li, Meng-Hsiung Hsieh, Alisa Arutyunova, John Evans, Tao Liu, A. Lawson, Pantelis A. Nicola, Gerda Kildisiute, Heather E. Machado, Ahmad Al Kawam, Rui Wang, Qiyu Zeng, Nisi Jiang, Xun Wang, Greg Mannino, Xiongzhao Ren, Xiangyi Fang, Tripti Sharma, Suman Komjeti, David Hsiehchen, Scott Hayton, Simon Brunner, I. Martincorena, Peter J. Campbell, Hao Zhu
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="BRCA1/2 are crucial in the homologous recombination repair (HRR) pathway, with loss-of-function (LoF) alterations predicting sensitivity to PARP-inhibitors (PARPi). Whether other HRR-gene alterations confer PARPi sensitivity remains unclear. In the Drug Rediscovery Protocol, patients receive off-label drugs matched to their tumor molecular profile. Here, olaparib efficacy and safety were evaluated in adult patients with treatment-refractory, progressive malignancies harboring LoF alterations in ATM (cohort A) or other HRR-genes including CDK12, PPP2R2A, CHEK1/2, and RAD51B (cohort B). Primary endpoints were clinical benefit (CB: confirmed objective response or stable disease ≥16 weeks) and safety. Pre-treatment biopsies were analyzed by whole-genome sequencing (WGS) for target validation. CB was observed in 8/25 patients (32%) in cohort A (prostate cancer: n = 6, adenoid cystic carcinoma: n = 1, endometrial cancer: n = 1). No effectiveness was seen in patients with colorectal cancer (n = 8). Median progression-free survival (PFS) and overall survival (OS) were 3.4 months (95% CI 1.8-5.3) and 9.2 months (95% CI 5.2-21.3), respectively. In cohort B, the CB rate was 41.7% (10/24) with median PFS and OS of 3.5 months (95% CI 3.4-6.6) and 8.1 months (95% CI 6.6-14.2), respectively. CB was observed in CKD12 (n = 7), RAD51B (n = 2), and CHEK2-altered tumors (n = 1), but not in PPP2R2A (n = 6) or CHEK1-altered tumors (n = 1). No unexpected toxicities occurred. WGS confirmed inclusion target in 84% of tested patients. In conclusion, PARPi sensitivity varies across HRR-genes, indicating that relying solely on an altered common mechanistic pathway is insufficient to predict response. Future studies should target specific HRR-genes to assess subgroup-specific benefits and determine proper use of molecular diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/307d50a92ba3f268facbb2ba41b55d6d57fef8c8" target='_blank'>
              Olaparib for patients with tumors harboring alterations in homologous recombination repair genes: Results from the drug rediscovery protocol.
              </a>
            </td>
          <td>
            I. A. C. Spiekman, N. Mehra, L. Zeverijn, B. Geurts, K. Verkerk, S. H. Haj Mohammad, V. van der Noort, P. Roepman, W. D. de Leng, A. Jansen, A. V. D. van de Luijtgaarden, T. van Voorthuizen, T. Buffart, H. Gelderblom, E. Voest, Henk M. Verheul
          </td>
          <td>2025-10-16</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reversine is a small-molecule Aurora kinase inhibitor known for its pro-apoptotic effects and potential to remodel chromatin architecture. Although its impact on mitotic regulation is established, its effects on telomere dynamics and nuclear organization in chronic myeloid leukemia (CML) remain unclear. This study aimed to investigate the effects of reversine on telomere architecture, genomic instability, and apoptosis in CML cell lines (K-562 and MEG-01). Reversine was applied at increasing concentrations, and cytotoxicity was assessed using caspase-3/7 activation assays. Quantitative PCR was used to measure AURKA and AURKB mRNA expressions. Three-dimensional telomere architecture was analyzed with TeloView® v1.03 software after Q-FISH labeling to quantify telomere number, signal intensity, aggregation, nuclear volume, and a/c ratio. Reversine induced a dose- and time-dependent apoptotic response in both cell lines and significantly downregulated AURKA and AURKB expressions. Three-dimensional telomere analysis revealed a marked reduction in telomere number and aggregates, signal intensity, and nuclear volume. While reduced signal intensity may indicate telomere shortening, the concurrent decrease in aggregation and altered spatial parameters suggests telomeric reorganization rather than progressive instability. These features reflect structural nuclear remodeling and early apoptotic commitment. Differences between K-562 and MEG-01 responses underscore potential heterogeneity in telomere maintenance mechanisms. Reversine modulates genomic stability in CML cells through dual mechanisms involving Aurora kinase inhibition and telomere architecture remodeling. The integration of 3D telomere profiling highlights reversine’s potential as a therapeutic agent targeting nuclear disorganization and mitotic dysregulation in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e06bedd101bf31c7285fbbc7d8f35b57bf8952e" target='_blank'>
              Reversine-Induced Telomere Architecture Remodeling in Chronic Myeloid Leukemia Cell Lines: Insights from TeloView® Analysis of 3D Nuclear Architecture
              </a>
            </td>
          <td>
            Fábio Morato de Oliveira, Isabela Dias Cruvinel, Bruno Machado Rezende Ferreira, Sabine Mai
          </td>
          <td>2025-10-31</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly aggressive malignancy with poor prognosis and significant molecular heterogeneity. This study investigates the role of tumor-specific enhancers in metabolic reprogramming, focusing on glucose-6-phosphate dehydrogenase (G6PD) and the pentose phosphate pathway (PPP). Using native elongating transcript-cap analysis of gene expression and single-cell RNA sequencing, tumor-specific enhancers driving G6PD overexpression were identified in ICC tumor epithelial cells. Functional assays demonstrated that G6PD promotes tumor proliferation by enhancing PPP activity and maintaining redox homeostasis, which provides NADPH to counter oxidative stress. Enhancer knockdown disrupted G6PD expression and PPP activity, increasing reactive oxygen species levels and reducing the NADPH/NADP+ ratio. These metabolic changes impaired tumor cell proliferation and sensitized ICC cells to cisplatin, emphasizing the dual therapeutic potential of targeting G6PD to inhibit tumor growth and overcome chemoresistance. Survival analyses showed that high G6PD expression correlates strongly with poor overall survival in ICC patients. While previous studies have recognized the roles of G6PD and PPP in cancer metabolism, this study uniquely links enhancer-mediated regulation to these processes in ICC, offering novel insights into epigenetic mechanisms driving metabolic reprogramming. Moreover, the findings highlight tumor-specific enhancers as critical epigenetic drivers of ICC progression, with potential as therapeutic targets. Future research should explore the integration of enhancer profiling into precision medicine frameworks and the development of novel enhancer-targeting strategies. These efforts could uncover additional metabolic vulnerabilities and provide effective treatments for this highly aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f09ce5004e00e224dd82b9d53133eeef55290e" target='_blank'>
              Epigenetic Regulation of G6PD Drives Metabolic Reprogramming in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Yusuke Nakano, Miwa Tanaka, Takeharu Sakamoto, M. Hashimoto, Taro Tobo, Hideyuki Saito, Tadashi Abe, Tomohiko Ikehara, Takashi Ofuchi, Koto Kawata, Takaaki Masuda, T. Ogino, Mamoru Uemura, H. Eguchi, Y. Doki, K. Mimori
          </td>
          <td>2025-10-07</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, A. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe180fb4ec176d493964db2a8374c986c41b6b32" target='_blank'>
              Spatial Proximity Sequencing Maps Developmental Dynamics in the Germinal Center
              </a>
            </td>
          <td>
            Huili Wang, Junjie Xia, Peer Mohamed Suhail Mujibur Rahman, Bijentimala Keisham, Abinash Padhi, Yiyu Deng, Yan Li, Luke Vistain, Sarah Kim, Gabriel Mercado-Vásquez, Aly A. Khan, Marcus R. Clark, Savaş Tay
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d46c8e39bf3ac8b82b288f4203116930391492" target='_blank'>
              Unveiling The Origin of Human High-Grade Serous “Ovarian” Cancer Using Bulk and Single-Nucleus RNA Sequencing
              </a>
            </td>
          <td>
            Qian Li, Keren Cheng, Lili Sun, Wei Yan
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>