<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 09:25:36 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>27</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>64</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>308</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>265</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Assay for transposase-accessible chromatin using sequencing (ATAC-seq) is a cornerstone for epigenomic profiling, yet its potential for genomic characterization remains poorly explored. Here, we systematically benchmarked bulk ATAC-seq against whole-genome sequencing (WGS) to assess its capacity for detecting small variants, copy number variations (CNVs), telomeric content, and mitochondrial mutations. Using paired datasets from six melanoma cell lines, we demonstrated that ATAC-seq achieves reasonable precision in SNP detection, robustly profiles CNVs in the nuclear genome, and effectively quantifies mitochondrial heteroplasmy, with strong concordance to WGS at classical sequencing depths. Notably, we present the first systematic evaluation of telomeric length by ATAC-seq, revealing its untapped potential for studying cellular aging and genome stability. By bridging genomic and epigenomic insights into a single genome-wide approach, bulk ATAC-seq emerges as a cost-effective and versatile tool poised to transform cancer and aging research and advance clinical diagnostics through its integrative capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee80f71d844365f30482c6aaffb03ba85ca20c8d" target='_blank'>
              ATAC-seq as a versatile tool to portray genomes and epigenomes
              </a>
            </td>
          <td>
            Islem Toumi, Laurent Lecam, Pierre-François Roux
          </td>
          <td>2025-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Human satellitess(HSats) are pericentric, tandemly repeating satellite DNA sequences in the human genome. While silent in normal cells, a subset of HSat2 noncoding RNA is expressed and accumulates in the nucleus of cancer cells. We developed a FISH-based approach for identification of the distribution of three subfamilies of HSat2 (A1, A2, B) sequences on individual human chromosomes. Further, using the HSat subfamily annotations in the T2T completed centromere satellite (CenSat) sequence, we isolated, defined and mapped differentially expressed sequence variants of nuclear-restricted HSat2 and HSat3 RNA from cancer cell lines and heat-shocked cells. We identified chromosome-specific and subfamily-specific expression of HSat2 and HSat3 and established a computational pipeline for differential expression analysis of tandemly repeated satellite sequences. Results suggest the differential expression of chromosome-specific HSat2 arrays in the human genome may underlie their accumulation in cancer cells and that specific HSat3 loci are upregulated upon heat shock.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ef06daf0097b2cbd7221f330d3d882d1511e90" target='_blank'>
              Locus-specific differential expression of human satellite sequences in the nuclei of cancer cells and heat-shocked cells
              </a>
            </td>
          <td>
            Christina A. Rabeler, Nicholas J. Paterna, Rajiv Potluri, Lia R. D’Alessandro, Anusha Bhatia, Shu Yi Chen, Johanna Lee, Bereketab Abeje, Benjamin Lipchin, Benjamin R Carone, Dawn M. Carone
          </td>
          <td>2024-12-01</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="All cells possess mechanisms to maintain and replicate their genomes, whose integrity and transmission are constantly challenged by DNA damage and replication impediments. In eukaryotes, the protein kinase Ataxia-Telangiectasia and Rad3-related (ATR), a member of the phosphatidylinositol 3-kinase-like family acts as a master regulator of the eukaryotic response to DNA injuries, ensuring DNA replication completion and genome stability. Here we aimed to investigate the functional relevance of the ATR homolog in the DNA metabolism of Leishmania major, a protozoan parasite with a remarkably plastic genome. CRISPR/cas9 genome editing was used to generate a Myc-tagged ATR cell line (mycATR), and a Myc-tagged C-terminal knockout of ATR (mycATRΔC-/-). We show that the nuclear localisation of ATR depends upon its C-terminus. Moreover, its deletion results in single-stranded DNA accumulation, impaired cell cycle control, increased levels of DNA damage, and delayed DNA replication restart after replication stress. In addition, we show that ATR plays a key role in maintaining L. major’s unusual DNA replication program, where larger chromosomes duplicate later than smaller chromosomes. Our data reveals loss of the ATR C-terminus promotes the accumulation of replication signal around replicative stress fragile sites, which are enriched in larger chromosomes. Finally, we show that these alterations to the DNA replication program promote chromosome instability. In summary, our work shows that ATR acts to moderate DNA replication timing thus limiting the plasticity of the Leishmania genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3499060635df2f56eb484fb5207c0367f3df1797" target='_blank'>
              ATR, a DNA damage kinase, modulates DNA replication timing in Leishmania major
              </a>
            </td>
          <td>
            Gabriel L. A. da Silva, J. Damasceno, J. A. Black, R. McCulloch, L. Tosi
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f0a75813c454a5e8e9bd04b719f1b32b01034b" target='_blank'>
              ZIC1 is a context-dependent medulloblastoma driver in the rhombic lip
              </a>
            </td>
          <td>
            John J. Y. Lee, Ran Tao, Z. You, P. Haldipur, Anders W Erickson, H. Farooq, Liam D Hendriske, Namal Abeysundara, Cory Richman, Evan Y Wang, Neha Das Gupta, Jennifer L. Hadley, Melissa Batts, Christopher W. Mount, Xiaochong Wu, Alexandra Rasnitsyn, Swneke Bailey, F. Cavalli, Sorana A. Morrissy, Livia Garzia, K. Michealraj, Abhi Visvanathan, Vernon Fong, Jonelle Palotta, Raúl A. Suárez, Bryn Livingston, Miao Liu, B. Luu, Craig Daniels, J. Loukides, A. Bendel, P. French, J. Kros, Andrey Korshunov, Marcel Kool, F. Chico Ponce de León, M. Perezpeña-Diazconti, Boleslaw Lach, Sheila K Singh, S. Leary, Byung-Kyu Cho, Seung-Ki Kim, Kyu-Chang Wang, Ji-Yeoun Lee, Teiji Tominaga, William A. Weiss, Joanna J Phillips, Shizhong Dai, Gelareh Zadeh, Ali G Saad, L. Bognár, Á. Klekner, Ian F. Pollack, Ronald L Hamilton, Young-Shin Ra, W. Grajkowska, M. Perek-Polnik, Reid C. Thompson, Anna M. Kenney, Michael K. Cooper, S C Mack, Nada Jabado, Mathieu Lupien, Marco Gallo, Vijay Ramaswamy, M. Suvà, Hiromishi Suzuki, K. Millen, L. F. Huang, P. Northcott, Michael D. Taylor
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865e3224ea8530a73a268350cf8b48819bfe3161" target='_blank'>
              Inter-chromosomal transcription hubs shape the 3D genome architecture of African trypanosomes
              </a>
            </td>
          <td>
            Claudia Rabuffo, Markus R Schmidt, Prateek Yadav, Pin Tong, Roberta Carloni, Anna Barcons-Simon, Raúl O. Cosentino, Stefan Krebs, Keith R. Matthews, Robin C. Allshire, Nicolai Siegel
          </td>
          <td>2024-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Characterization of tumor epigenetic aberrations is integral to understanding the mechanisms of tumorigenesis and provide diagnostic, prognostic, and predictive information of high clinical relevance. Among the different tumor-associated epigenetic signatures, 5 methyl-cytosine (5mC) and 5-hydroxymethylcytosine (5hmC) are the two most well-characterized DNA methylation alterations linked to cancer pathogenesis. 5hmC has a tissue-specific distribution and its abundance is subjected to changes in tumor DNA, making it a promising biomarker. Detecting tumor-related DNA methylation alterations in tissues is highly invasive, while the analysis of the cell-free DNA (cfDNA) is poised to supplement, if not replace, surgical biopsies. Despite many studies attempted to identify new epigenetic targets for liquid biopsy assays, little is known about the regulatory roles of 5hmC, its impacts on the molecular phenotypes in tumors. Most importantly, whether the oncogenic-associated 5hmC signatures found in tumor tissues can be recapitulated in patient cfDNA. In this study, we performed the unbiased and simultaneous detection of 5mC and 5hmC whole-genome DNA modifications at base-resolution from two distinct cancer cohorts, from patients with bladder cancer or B-Cell lymphoma, their corresponding normal tissues, and cfDNAs from plasma. We analyzed tissue-specific methylation patters and searched for signatures in gene coding and regulatory regions linked to cancerous states. We then looked for methylation signatures in patients cfDNA to determine if they were consistent with the tumor-specific patterns. We determined the functional significance of 5hmC in tissue specific transcription and uncovered hundreds of tumor-associated 5hmC signatures. These tumor-associated 5hmC changes, particularly in genes and enhancers, were functionally significant in tumorigenesis pathways and correlated with tumor specific gene expression. To investigate if cfDNA is a faithful surrogate for tumor-associated 5hmC, we devised a targeted capture strategy to examine the alterations of 5hmC in cfDNA from patients with bladder cancer and lymphoma with sufficient sensitivity and specificity and confirmed that they recapitulated the patterns we observed in tumor tissues. Our results provide analytic validation of 5hmC as a cancer-specific biomarker. The methods described here for systematic characterization of 5hmC at functional elements open new avenues to discover epigenetic markers for non-invasive diagnosis, monitoring, and stratifying cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233a1b3a2e680f52000d36e68abfc546f2d4b33" target='_blank'>
              Global DNA methylomes reveal oncogenic-associated 5-hydroxylmethylated cytosine (5hmC) signatures in the cell-free DNA of cancer patients
              </a>
            </td>
          <td>
            G. E. Rech, A. C. Lau, R. L. Goldfeder, R. Maurya, A. V. Danilov, C.-L. Wei
          </td>
          <td>2025-01-15</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer. To resolve genome dysregulation associated with HPV integration, we performed Oxford Nanopore long-read sequencing on 72 cervical cancer genomes from an Ugandan dataset that was previously characterized using short-read sequencing. We found recurrent structural rearrangement patterns at HPV integration events, which we categorized as: del(etion)-like, dup(lication)-like, translocation, multibreakpoint, or repeat region integrations. Integrations involving amplified HPV-human concatemers, particularly multibreakpoint events, frequently harbored heterogeneous forms and copy numbers of the viral genome. Transcriptionally active integrants were characterized by unmethylated regions in both the viral and human genomes downstream from the viral transcription start site, resulting in HPV-human fusion transcripts. In contrast, integrants without evidence of expression lacked consistent methylation patterns. Furthermore, whereas transcriptional dysregulation was limited to genes within 200 kilobases of an HPV integrant, dysregulation of the human epigenome in the form of allelic differentially methylated regions affected megabase expanses of the genome, irrespective of the integrant's transcriptional status. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8457c68dc264c60760742af32b0cda2346039" target='_blank'>
              Rearrangements of viral and human genomes at human papillomavirus integration events and their allele-specific impacts on cancer genome regulation.
              </a>
            </td>
          <td>
            Vanessa L. Porter, Michelle Ng, Kieran O’Neill, Signe MacLennan, R. Corbett, Luka Culibrk, Zeid Hamadeh, Marissa Iden, Rachel Schmidt, Shirng-Wern Tsaih, C. Nakisige, Martin Origa, Jackson Orem, Glenn Chang, Jeremy Fan, K. Nip, Vahid Akbari, Simon K. Chan, James Hopkins, Richard A. Moore, E. Chuah, K. Mungall, A. Mungall, Inanҫ Birol, S. Jones, Janet S. Rader, M. Marra
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="RNase H2 is a heterotrimeric endoribonuclease that resolves RNA:DNA hybrids and genome-embedded ribonucleotides, which are implicated in DNA replication stress and cancer development. Protein and/or mRNA levels of individual RNase H2 subunits are elevated in some cancers, but little is known about the mechanisms or consequences of RNase H2 upregulation. We report that RNase H2 subunits are upregulated at the protein level in response to replication stress induced by oncogenes and chemotherapy drugs in human cancer and non-cancer cell lines. We show that inducible overexpression of the RNASEH2B subunit increases levels of the active RNase H2 heterotrimer. While causing only subtle changes to gene expression, RNASEH2B overexpression is unexpectedly associated with increased RNA:DNA hybrid levels. RNASEH2B overexpression prevents further increases in RNA:DNA hybrid levels by camptothecin or hydroxyurea and reduces replication fork stalling in presence of these drugs. Surprisingly, RNase H2 levels do not strongly impact survival of chemotherapy treatments but appear to have more subtle effects on genome instability and innate immune signalling. In contrast, increased RNase H2 levels in presence of oncogenic HRAS limit not only RAS-induced replication fork stalling but also cell death. Our findings shed new light on the functions of RNase H2 and suggest that upregulation of RNase H2 may be an important aspect of replication stress responses in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54af3146eb7acac3bc5186f9ac5d0287ea7b019a" target='_blank'>
              Human RNase H2 upregulation counteracts oncogene- and chemotherapy-induced replication stress
              </a>
            </td>
          <td>
            Rosanna J. Wilkins, Abirami Kannan, Siobhan A. Plass, Claire Wilson, Richard D. W. Kelly, Claire H. M. Tang, P. Kotsantis, Martin A. M. Reijns, Aditi Kanhere, Eva Petermann
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Human gene expression is controlled from distance via enhancers, which can form longer ‘super-enhancer’-regions of intense regulatory activity. Whether super-enhancers constitute a separate regulatory paradigm remains unclear, largely due to the difficulty of dissecting the contributions and interactions of individual elements within their natural chromosomal context. To address this challenge, we developed enhancer scrambling, a high-throughput strategy to generate stochastic inversions and deletions of targeted enhancer regions by combining CRISPR prime editing insertion of symmetrical loxP sites with Cre recombinase-induced rearrangements. We applied our approach to dissect a distal super-enhancer of the OTX2 gene, generating up to 134 alternative regulatory configurations in a single experiment, and establishing how they drive gene expression and chromatin accessibility, as well as the individual contributions of its elements to this activity. Surprisingly, the presence of the sequence containing a single DNase I hypersensitive site predominantly controls OTX2 expression. Our findings highlight that enhancer-driven regulation of some highly expressed, cell-type-specific genes can rely on an individual element within a cluster of non-interacting, dispensable components, and suggest a simple functional core to a subset of super-enhancers. The targeted randomisation method to scramble enhancers can scale to resolve many super-enhancers and human gene regulatory landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cecb86ee79d0fa4af20cbd670e19b80d640e006e" target='_blank'>
              Resolution of a human super-enhancer by targeted genome randomisation
              </a>
            </td>
          <td>
            Jonas Koeppel, Pierre Murat, Gareth Girling, Elin Madli Peets, Mélanie Gouley, Valentin Rebernig, Anant Maheshwari, Jacob Hepkema, Juliane Weller, Jenie Hannah Johnkingsly Jebaraj, Ronnie Crawford, Fabio Giuseppe Liberante, Leopold Parts
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="All great apes differ karyotypically from humans due to the fusion of chromosomes 2a and 2b, resulting in human chromosome 2. Yet, the structure, function, and evolutionary history of the genomic regions associated with this fusion remain poorly understood. Here, we analyze finished telomere-to-telomere chromosomes in great apes and macaques to show that the fusion was associated with multiple pericentric inversions, segmental duplications (SDs), and the rapid turnover of subterminal repetitive DNA. We characterized the fusion site at single-base-pair resolution and identified three distinct SDs that originated more than 5 million years ago. These three distinct SDs were differentially distributed among African great apes as a result of incomplete lineage sorting (ILS) and lineage-specific duplication. Most conspicuously, one of these SDs shares homology to a hypomethylated SD spacer sequence present in hundreds of copies in the subterminal heterochromatin of chimpanzees and bonobos. The fusion in human was accompanied by a systematic degradation of the three divergent α-satellite arrays representing the ancestral centromere creating five distinct structural haplotypes in humans. CRISPR/Cas9-mediated depletion of the fusion site in human cell lines significantly alters the expression of 108 genes, indicating a potential regulatory consequence to this human-specific karyotypic change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b16610644425427f06758c18d45678c36f10190" target='_blank'>
              Incomplete lineage sorting of segmental duplications defines the human chromosome 2 fusion site early during African great ape speciation
              </a>
            </td>
          <td>
            Xinrui Jiang, Lu Zhang, Zikun Yang, Xiangyu Yang, Kaiyue Ma, DongAhn Yoo, Yong Lu, Shilong Zhang, Jieyi Chen, Yanhong Nie, Xinyan Bian, Junmin Han, Lianting Fu, Juan Zhang, Guojie Zhang, Qiang Sun, E. Eichler, Yafei Mao
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Wang Yuh-Hwa
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Ovarian cancer is a deadly gynecological disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for HR-deficient high-grade serous ovarian cancers (HGSOCs). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both HSF1 and MYC, wherein these two genes are co-amplified in over 30% of HGSCO patients. We further found that HSF1 and MYC transcriptional activity was highly correlated in human HGSOC tumors and cell lines, suggesting they may cooperate in the disease. CUT&RUN for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein-protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1-MYC co-amplified HGSOC cells compared to those with wild-type HSF1 and MYC copy number. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 in a precision medicine approach using HSF1-MYC co-amplification as a biomarker in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High Grade Serous Ovarian Cancer.
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6960e9d1d69ddbed72b6eb04c8ec6b5d4992a9b7" target='_blank'>
              Structural polymorphism and diversity of human segmental duplications.
              </a>
            </td>
          <td>
            Hyeonsoo Jeong, Philip C. Dishuck, DongAhn Yoo, William T. Harvey, Katherine M. Munson, Alexandra P. Lewis, Jennifer R. Kordosky, Gage H. Garcia, F. Yilmaz, P. Hallast, Charles Lee, T. Pastinen, E. Eichler
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNA G-quadruplexes (G4s) are secondary structures with significant roles in regulating genome function and stability. Dysregulation of the dynamic formation of G4s is linked to genomic instability and disease, but the underlying mechanisms are not fully understood. In this study, we conducted a screen of chromatin-modifying enzymes and identified nine potential inhibitors of G4 formation, including seven that were not previously characterized. Among these, we highlight the role of BAZ2 chromatin remodelers as key suppressors of G4 DNA and G4-related genome instability. Depletion of BAZ2 subunits led to increased G4 formation, especially at transcriptional regulatory elements. BAZ2B was found to associate with G4 loci, suggesting that it plays a direct role in suppressing G4s. While BAZ2-deficient cells exhibited modest genomic instability, treatment with the G4-stabilizing ligand BRACO19 exacerbated double-strand breaks (DSBs), highlighting its utility as a tool to study G4-dependent genome instability. DSB profiling using INDUCE-seq uncovered distinct breakage patterns around G4s, further underscoring the impact of G4s on genome integrity. Notably, we found that within G4s, G repeats were more susceptible to DSBs than loops. These results establish BAZ2 chromatin remodeling complexes as direct regulators of G4 dynamics and provide new insights into G4-dependent genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a708bc98c35ab68fd43151d91f694aa98b85bda" target='_blank'>
              The Role of BAZ2-dependent Chromatin Remodeling in Suppressing G4 DNA Structures and Associated Genomic Instability
              </a>
            </td>
          <td>
            Adrianna L. Vandeuren, Kierney O’Dare, Rosemary H. C. Wilson, Patrick van Eijk, Lindsay R. Julio, Shannon G. MacLeod, Ella Chee, Annika Salpukas, Emma M. Kriz, George A. Lantz, Shellaina J. V. Gordon, Simon J. Elsässer, Simon H. Reed, Tovah A. Day
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcd0f4cc9961826189852b167ab5b2ef51c1fec" target='_blank'>
              Integrating Circle-Seq with transcriptomics reveals genome-wide characterization of extrachromosomal circular DNA for dilated cardiomyopathy
              </a>
            </td>
          <td>
            Zhenhao Lin, Fangjie Dai, Bo Li, Yongchao Zhao, Changqian Wang
          </td>
          <td>2024-11-29</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ(WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3985e13774d5b1194193c3f3fcbde897bc3a759f" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Bertram Aschenbrenner, Alexander Howard, Emily Lay, Emma Bailey, Natalia Krajic, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genome duplication, critical for cell survival and identity, requires precise origin activation to ensure accurate DNA replication and chromatin structure maintenance. In colorectal cancer cells, prolonged replication stress does not hinder cell proliferation, although the mechanisms driving cancer cell adaptability remain largely unclear. Here, we demonstrate that upon recovery, cancer cells are able to activate new replication origins in distinct domains, causing persistent changes in chromatin accessibility, nuclear morphology, and replication timing, ultimately promoting chromatin instability. Chromatin accessibility, particularly in promoter regions, increased following replication stress in a subset of cells, correlating with altered gene expression and nuclear expansion. Additionally, some genes with enhanced promoter accessibility displayed sustained expression changes, further suggesting a transcriptional shift linked to stress adaptation. Our findings reveal that colorectal cancer cells recover from severe replication stress through new origin activation, a mechanism that not only maintains cell proliferation under stress but may also accelerate tumour heterogeneity. This research underscores replication origin activation as a potential therapeutic target in combating cancer cell resilience. GRAPHICAL ABSTRACT MAIN HIGHLIGHTS Colorectal cancer cells under severe replication stress activate new replication origins upon recovery. The firing of new origins in distinct replication domains results in long-lasting changes in chromatin accessibility, especially in promoter regions, and is associated with alterations in nuclear morphology and replication timing. Increased chromatin accessibility in certain promoter regions correlates with prolonged gene expression changes. The capacity of CRC cells to recover from severe replication stress fosters chromatin instability and tumour heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cc1950d705dd8a02a8a5d1f21cfa69a97d0c9c" target='_blank'>
              Adaptive replication origin activation alters chromatin dynamics and stability in cancer cells
              </a>
            </td>
          <td>
            Alejandro Pérez-Venteo, Fernando Unzueta, Sonia Feu, P. Llinàs-Arias, Caroline Mauvezin, Neus Agell
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="INTRODUCTION
The SEWA cell line, which is derived from a virus induced murine osteosarcoma (OS) ascites was established in the 1980s from a serially transplanted male derived tumor that first published in 1961. It has been applied in about 50 studies but was never genetically characterized in detail; this study fills that gap.


METHODS
The SEWA cell line was analyzed for its chromosomal constitution using molecular cytogenetic approaches. Array comparative genomic hybridization was performed to characterize copy number alterations.


RESULTS
SEWA has a near diploid karyotype without Y chromosome material. The complex karyotype includes neocentrics and simple and complex rearrangements. Amplification of MYC oncogene was detected in two homogeneously staining regions on two different derivative chromosomes.


CONCLUSION
An in silico translation of the obtained results to the human genome indicated that SEWA is suitable as a model for advanced human OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec328ac3891c2b11391d74ef5192bbc587c6a2" target='_blank'>
              Cytogenomic characterization of murine osteosarcoma cell line SEWA.
              </a>
            </td>
          <td>
            Thomas Liehr, Martina Rinčic
          </td>
          <td>2024-12-11</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal inversions play a crucial role in evolution by influencing phenotypes through the linkage of co-adapted alleles. While inversions have been found across a large number of taxa, mapping and characterizing inversion breakpoint regions remains challenging, often due to the presence of complex tandem repeats and transposable elements (TEs). Here, we identify and quantify TEs in the breakpoints of the four large-scale inversions previously reported in Atlantic cod, leveraging on three high-quality long-read-based reference genome assemblies for the Norwegian Coastal cod, the Northeast Arctic cod and Celtic cod ecotypes. We detected a significant enrichment of TE orders and superfamilies with terminal inverted repeats (TIRs) within the inversion breakpoints of chromosomes 1, 7 and 12. Notably, we discovered a tandem accumulation of miniature inverted-repeat transposable elements (MITEs) belonging to a family of hAT transposons, exclusively residing in the breakpoints of the inverted haplotype on chromosomes 1 and 7 found in the Northeast Arctic cod. The accumulation of tandemly arranged TEs in breakpoint regions suggests that they have driven the appearance of inversions through ectopic recombination, further supporting the potential of TEs in facilitating chromosomal reorganizations with large evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c69b85f1eed1a39a185be57ad812eafc541ce3b" target='_blank'>
              Chromosomal inversions mediated by tandem insertions of transposable elements
              </a>
            </td>
          <td>
            Robin Aasegg Araya, William B. Reinar, O. K. Tørresen, C. Goubert, Tara J Daughton, S. Hoff, H. T. Baalsrud, Marine Servane Ono Brieuc, Anna Komisarczuk, S. Jentoft, José Cerca, K.S. Jakobsen
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is initiated during early mammalian embryogenesis by a long non-coding RNA (lncRNA) XIST, which coats one of the two X-chromosomes and facilitates epigenetic transcriptional silencing. A second, evolutionarily recent primate-specific lncRNA XACT was proposed to antagonize XIST’s ability to induce XCI. XACT expression is restricted to pluripotent states and early embryonic stages and coats the active X-chromosome in both females and males. Here, we report a novel XACT transcript expressed in normal and cancerous somatic cells from both the inactive (Xi) and active (Xa) X chromosomes. It coexists with XIST on the Xi without affecting XIST expression or (re)activating X-linked genes inactivated by XCI. During hematopoietic stem cell (HSC) differentiation, XACT is primarily expressed in myeloid progenitors in both sexes. XACT expression is activated in HSCs and peaks in megakaryocyte-erythrocyte progenitor cells (MEPs) before rapidly declining as the MEPs differentiate into megakaryocytes or erythrocytes. By combining CRISPR-based XACT perturbation with epigenomic and transcriptional studies, we revealed the key role of XACT in the self-renewal and differentiation of erythroid progenitors into erythrocytes, by recruiting cis-regulatory proteins and regulating transcription through ETS and AP-1 transcription factors. Furthermore, XACT is expressed in a subset of acute myeloid leukemia (AML) patients, with high levels found in erythroid-megakaryocytic blast cells; suggesting XACT’s potential as a marker for AML subtypes. Thus, we identified a novel XACT transcript with a unique expression profile and important roles in normal and cancer cells, revealing XACT lncRNA functions beyond its previously proposed role in XCI during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7754eb09126d1d587b2765e70f76183499d8bcbf" target='_blank'>
              A Novel XACT lncRNA Transcript with Functions Transcending X-Chromosome Inactivation
              </a>
            </td>
          <td>
            Peifen Zhang, Paula Haro Angles, Hong Zhang, Tingyue Li, Roger Mulet-Lazaro, W. V. van Ijcken, S. Philipsen, Ruud Delwel, J. Gribnau, D. Huylebroeck, Frank Grosveld, C. Rougeulle, Eskeatnaf Mulugeta
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Rearrangements associated with gene amplification are frequently initiated by the formation of inverted repeats. Among these rearrangements are breakage-fusion-bridge cycles and tandem inverted duplications (TIDs), both of which can be mediated through palindromic sequences. We have investigated spontaneous rearrangements within the genome of the filamentous fungus Aspergillus flavus by high-coverage third-generation sequencing of vegetatively cultured strains. Analysis of sequence data identified a class of rare and transient rearrangements having structures typical of TIDs. These putative TIDs (pTIDs) form both unprocessed and processed species and have a variably sized deletion at the junction between direct and inverted sequences. Most intact junctions contain the predicted palindromic sequences, supporting the TID model. However, palindrome-free junctions are also infrequently observed. pTIDs are highly enriched in heterochromatic centromeric and subtelomeric A+T rich regions, suggesting that this process is a source of genetic instability in these domains. Consistent with this finding, A+T rich regions contain elevated levels of direct, inverted, and perfect palindromic repeats. We propose a replication-based model in which palindromes and short inverted repeats, under condition of replication stress, are responsible for pTID structures. TEASER The dynamic formation of duplication/inversions clustered in fungal chromosomes reveals the genetic instability of centromeric and subtelomeric regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728781bd2be224742eb5106c8451f24b72cd7b56" target='_blank'>
              Clustering of Tandem Inverse Duplications in Centromeric and Subtelomeric Chromosomal Regions of Aspergillus flavus
              </a>
            </td>
          <td>
            Jeffrey W. Cary, Amna Malik, Alexie Smith, Erik K Flemington, Arthur J Lustig
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types.


METHODS
In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts.


RESULTS
FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types.


CONCLUSIONS
Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor.
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-06</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The SNF2 family chromatin remodeler HELLS has emerged as an important regulator of cell proliferation, genome stability, and several cancer pathways. Significant upregulation of HELLS has been reported in 33 human cancer types. While HELLS has been implicated in DNA damage response, its function in DNA repair is poorly understood. Here we report a new regulatory link between HELLS and single-strand break (SSB) repair in cellular responses to DNA alkylation damage. We found that loss of HELLS impairs SSB repair, and selectively sensitizes cells to DNA alkylating agents and PARP inhibitors (PARPi). Furthermore, we found that HELLS is co-expressed with PARP1 in cancer cells, and its loss is synthetic lethal with homologous recombination deficiency (HRD). This work unveils new functions of HELLS in modulating SSB repair and responses to clinically relevant DNA alkylation damage, thus offering new insights into the potential therapeutic value of targeting HELLS in cancer. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd4d063ed0793e8f2d3e58d28ea55c516edbff0" target='_blank'>
              Chromatin regulator HELLS mediates SSB repair and responses to DNA alkylation damage
              </a>
            </td>
          <td>
            Joyous T Joseph, Christine M Wright, Estanislao Peixoto, Asad Khan, Yong Li, Jason S Romero Neidigk, Olivia Decker, Azait Imtiaz, Brianna A Jones, Yangfeng Zhang, Etsuko Shiabata, S. Gradilone, Zachary A. Lewis, Rafael Contreras-Galindo, Arko Sen, A. Dutta, Wioletta Czaja
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6201c33f4b59a0efc18c440991fab3c666877247" target='_blank'>
              Integration of CRISPR/dCas9-Based methylation editing with guide positioning sequencing identifies dynamic changes of mrDEGs in breast cancer progression
              </a>
            </td>
          <td>
            Baolong Zhang, Jin Li, Wenqiang Yu
          </td>
          <td>2025-01-21</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Long interspersed nuclear element 1 (L1) retrotransposons represent a vast source of divergent genetic information. However, mechanistic analysis of whether and how L1s contribute to human developmental programs is lacking, in part due to the challenges associated with specific profiling and manipulation of human L1 expression. Here we show that thousands of hominoid-specific L1 integrants are expressed in human induced pluripotent stem cells and cerebral organoids. The activity of individual L1 promoters is surprisingly divergent and correlates with an active epigenetic state. Efficient on-target CRISPRi silencing of L1s revealed nearly a hundred co-opted L1-derived chimeric transcripts and L1 silencing resulted in changes in neural differentiation programs and reduced cerebral organoid size. Together, these data implicate L1s and L1-derived transcripts in hominoid-specific CNS developmental processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76dd8be460b1842f859ff053be4194a72fd59c71" target='_blank'>
              LINE-1 retrotransposons regulate the exit of human pluripotency and early brain development
              </a>
            </td>
          <td>
            Anita Adami, R. Garza, Patricia Gerdes, Pia A. Johansson, F. Dorazehi, Symela Koutounidou, Laura Castilla-Vallmanya, D. Atacho, Yogita Sharma, Jenny G. Johansson, Oliver Tam, A. Kirkeby, Roger A. Barker, Molly Gale-Hammell, C. Douse, Johan Jakobsson
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Lomentospora prolificans is a fungal pathogen responsible for serious, often fatal, illness in patients with compromised immune systems. Treatment is rarely successful because L. prolificans is inherently resistant to all major classes of antifungal drugs. In this study we publish three chromosome-level de novo genome assemblies, including the first complete-level assembly of L. prolificans, along with genome annotations. The L. prolificans genome is packaged in 11 nuclear chromosomes and one mitochondrial chromosome, has 36.7-37.1 Mb, and encodes for a putative 7357-7640 genes. The length and composition of contigs in one stain varied from those of the other two strains, supporting the hypothesis that a chromosomal translocation took place. These assemblies were confirmed with pulsed-field gel electrophoresis. The availability of more complete genomes will hopefully help the search for new antifungal drugs and provides insights into the evolutionary history of this pathogenic fungus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f38c6f4c1713d9e763d43ccdd74bd10162966e74" target='_blank'>
              Three annotated chromosome-level de novo genome assemblies of Lomentospora prolificans provide evidence for a chromosomal translocation event
              </a>
            </td>
          <td>
            Nina Grossman, Yunfan Fan, Aleksey V. Zimin, Maggie P. Wear, Anne Jedlicka, Amanda Dziedzic, Livia Liporagi-Lopes, W. Timp, A. Casadevall
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Once considered rare in eukaryotes, polycistronic mRNA expression has been identified in kinetoplastids and, more recently, green algae, red algae, and certain fungi. This study provides comprehensive evidence supporting the existence of polycistronic mRNA expression in the apicomplexan parasite Cryptosporidium parvum. Leveraging long-read RNA-seq data from different parasite strains and using multiple long-read technologies, we demonstrate the existence of defined polycistronic transcripts containing 2-4 protein encoding genes, several validated with RT-PCR. Some polycistrons exhibit differential expression profiles, usually involving the generation of internal monocistronic transcripts at different times during development. ATAC-seq in sporozoites reveals that polycistronic transcripts usually have a single open chromatin peak at their 5-prime ends, which contains a single E2F binding site motif. Polycistronic genes do not appear enriched for either male or female exclusive genes. This study elucidates a potentially complex layer of gene regulation with distinct chromatin accessibility akin to monocistronic transcripts. This is the first report of polycistronic transcription in an apicomplexan and expands our understanding of gene expression strategies in this medically important organism. Significance Cryptosporidium parvum is a parasite that causes human gastrointestinal disease, profoundly impacting infants and immunocompromised individuals worldwide. C. parvum isolation, culturing, cloning, and its compact 9.2 Mb genome pose challenges for research. We have discovered that, unlike most eukaryotes, which express one gene per mRNA, C. parvum exhibits widespread polycistronic transcription, whereby 10% of its protein encoding genes are expressed as multiple genes per single mRNA molecule. Mining the published C. parvum single-cell atlas with proxy sequences for polycistronic expression, we observed patterns of polycistron expression that vary with developmental stages, suggesting a role in the parasite’s lifecycle. By illuminating this unique aspect of C. parvum biology, our findings provide novel insights into Cryptosporidium gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f58f28759f061ba55a0de0ea34cad8756bccaf2c" target='_blank'>
              Deciphering Transcription in Cryptosporidium parvum: Polycistronic Gene Expression and Chromatin Accessibility
              </a>
            </td>
          <td>
            Rui Xiao, Rodrigo P. Baptista, Fiifi Agyabeng-Dadzie, Yiran Li, Yinxin Dong, Robert J. Schmitz, TC Glenn, J. Kissinger
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The study investigates the impact of targeted chromosome engineering on telomere dynamics, chromatin structure, gene expression, and phenotypic stability in Arabidopsis thaliana. Using precise CRISPR/Cas-based engineering, reciprocal translocations of chromosome arms were introduced between non-homologous chromosomes. The subsequent homozygous generations of plants were assessed for phenotype, transcriptomic changes and chromatin modifications near translocation breakpoints, and telomere length maintenance. Phenotypically, translocated lines were indistinguishable from wild-type plants, as confirmed through morphological assessments and principal component analysis. Gene expression profiling detected minimal differential expression, with affected genes dispersed across the genome, indicating negligible transcriptional impact. Similarly, ChIPseq analysis showed no substantial alterations in the enrichment of key histone marks (H3K27me3, H3K4me1, H3K56ac) near junction sites or across the genome. Finally, bulk and arm-specific telomere lengths remained stable across multiple generations, except for minor variations in one translocation line. These findings highlight the remarkable genomic and phenotypic robustness of A. thaliana despite large-scale chromosomal rearrangements. The study offers insights into the cis-acting mechanisms underlying chromosome arm-specific telomere length setting and establishes the feasibility of chromosome engineering for studies of plant genome evolution and crop improvement strategies. Significance statement This study demonstrates the robustness of Arabidopsis thaliana in maintaining telomere stability, chromatin integrity, and wild-type phenotype despite large-scale chromosomal translocations. It underscores the potential of chromosome engineering in advancing genome evolution research and crop improvement, contributes to a deeper understanding of genome dynamics, and opens new avenues for precision breeding in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c4a5f7dc0b1cbe9e13ab94b7dc2055482d7e96c" target='_blank'>
              Chromosome engineering points to the cis-acting mechanism of chromosome arm-specific telomere length setting and robustness of plant phenotype, chromatin structure and gene expression
              </a>
            </td>
          <td>
            Ondřej Helia, Barbora Matúšová, Kateřina Havlová, Anna Hýsková, Martin Lyčka, Natalja Beying, Holger Puchta, Jíří Fajkus, M. Fojtová
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Methylome profiling is an emerging clinical tool for tumor classification and liquid biopsies. Here, we developed FLEXseq, a genome-wide methylation profiler that enriches and sequences the fragments of DNA flanking the CCGG motif. FLEXseq strongly correlates (Pearson’s r = 0.97) with whole genome bisulfite sequencing (WGBS) while enriching 18-fold. To demonstrate the broad applicability of FLEXseq, we verified its usage across cells, body fluids, and formalin-fixed paraffin-embedded (FFPE) tissues. DNA dilutions down to 250 pg decreased CpG coverage, but bias in methylation remained low (Pearson’s r ≥ 0.90) compared to a 10 ng input. FLEXseq offers a cost-efficient, base-pair resolution methylome with potential as a diagnostic tool for tissue and liquid biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54be57d79dde7a0380f12d6db796d5b66a6e3b0a" target='_blank'>
              Enriched Methylomes of Low-input and Fragmented DNA Using Fragment Ligation EXclusive Methylation Sequencing (FLEXseq)
              </a>
            </td>
          <td>
            Jingru Yu, Lauren S. Ahmann, Yvette Y. Yao, Wei Gu
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial-mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-17</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Simple Summary Large genomic rearrangements of the PALB2 gene, especially deletions and duplications, are associated with hereditary breast–ovarian cancer. This study investigates intronic breakpoints linked to rearrangements in PALB2 exon 11, which is crucial for understanding the mechanisms affecting patients with hereditary breast and ovarian syndrome. Through next-generation sequencing, one duplication and three deletions were identified, confirmed by Multiplex Ligation-Dependent Probe Amplification. Detailed characterization revealed a tandem duplication of 5134 base pairs mediated by AluY repeats and identical deletions in three unrelated patients promoted by AluSx elements, resulting in a truncated PALB2 protein. These findings highlight the instability of intronic regions flanking exon 11 and suggest directions for future research on the prevalence and functional implications of these genomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30610e3b0b4ee5300bddce9e6a181cf9f25f21c9" target='_blank'>
              Alu–Mediated Duplication and Deletion of Exon 11 Are Frequent Mechanisms of PALB2 Inactivation, Predisposing Individuals to Hereditary Breast–Ovarian Cancer Syndrome
              </a>
            </td>
          <td>
            Diletta Sidoti, Valeria Margotta, Diletta Calosci, Erika Fiorentini, C. Bacci, Francesca Gensini, Laura Papi, Marco Montini
          </td>
          <td>2024-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Polyploidization is a driving force of wheat evolution and speciation, yet its impact on epigenetic regulation and gene expression remains unclear. Here, we constructed a high-resolution epigenetic landscape across leaves, spikes, and roots of hexaploidy wheat and its tetraploid and diploid relatives. Inter-species stable-expression genes exhibited conserved amino acid sequences under strong purifying selection, while dynamic-expression genes were linked to species-specific adaptation. During hexaploidization, dominant D-subgenome homoeolog expression was suppressed via reduced activating epigenetic signals, converging expression with the A and B subgenomes. Proximal chromatin regions near genes were more stable, whereas distal regions, particularly enhancer-like elements mediated by H3K27ac and H3K4me3, exhibit higher dynamism. Sequence variations in these enhancers lead to differential gene regulation, influencing traits such as spike development. For instance, the two haplotypes of dCRE region of TaDEP-B1 resulted in significant differences in its expression and spikelet numbers. We also observed a coevolution of transcription factors and their binding sites, particularly within the expanded ERF family, which regulates spike morphology. This study highlights the interplay between sequence variation and epigenetic modifications in shaping transcriptional regulation during wheat speciation, offering valuable insights for genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3eb492b176e5f2e28bdd73ac3853852b58a5d2c" target='_blank'>
              Reshaping epigenomic landscapes in facilitating the speciation of bread wheat
              </a>
            </td>
          <td>
            Zhaoheng Zhang, Xuelei Lin, Jingjing Yue, Yong-Chang Xu, Lingfeng Miao, Wenqiang Tang, Weilong Guo, Jun Xiao
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070c22030bfb86f8cf3bc0c2063075e7b4258116" target='_blank'>
              Identification of the clinical and genetic characteristics of gliomas with gene fusions by integrated genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            Guo-zhong Yi, Hua-yang Zhang, Tian-shi Que, Shan-qiang Qu, Zhi-yong Li, Song-tao Qi, Wen-yan Feng, Guang-long Huang
          </td>
          <td>2025-01-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Phased telomere to telomere (T2T) genome assemblies are revolutionising our understanding of long hidden genome biology “dark matter” such as centromeres, rDNA repeats, inter-haplotype variation, and allele specific expression (ASE). Yet insights into dikaryotic fungi that separate their haploid genomes into distinct nuclei is limited. Here we explore the impact of dikaryotism on the genome biology of a long-term asexual clone of the wheat pathogenic fungus Puccinia striiformis f. sp. tritici. We use Oxford Nanopore (ONT) duplex sequencing combined with Hi-C to generate a T2T nuclear-phased assembly with >99.999% consensus accuracy. We show that this fungus has large regional centromeres enriched in LTR retrotransposons, with a single centromeric dip in methylation that suggests one kinetochore attachment site per chromosomes. The centromeres of chromosomes pairs are most often highly diverse in sequence and kinetochore attachment sites are not always positionally conserved. Each nucleus carries a unique array of rDNAs with >200 copies that harbour nucleus-specific sequence variations. The inter-haplotype diversity between the two nuclear genomes is caused by large-scale structural variations linked to transposable elements. Nanopore long-read cDNA analysis across distinct infection conditions revealed pervasive allele specific expression for nearly 20% of all heterozygous gene pairs. Genes involved in plant infection were significantly enriched in ASE genes which appears to be mediated by elevated CpG gene body methylation of the lower expressed pair. This suggests that epigenetically regulated ASE is likely a previously overlooked mechanism facilitating plant infection. Overall, our study reveals how dikaryotism uniquely shapes key eukaryotic genome features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebe895c868a2f8ac32e5a5a400c1fdc993b92b30" target='_blank'>
              Long-read genomics reveal extensive nuclear-specific evolution and allele-specific expression in a dikaryotic fungus
              </a>
            </td>
          <td>
            Rita Tam, Mareike Möller, Runpeng Luo, Zhenyan Luo, Ashley W. Jones, Sambasivam K. Periyannan, J. Rathjen, B. Schwessinger
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="A major challenge in epigenetics is uncovering the dynamic distribution of nucleosomes and other DNA-binding proteins, which plays a crucial role in regulating cellular functions. Established approaches such as ATAC-seq, ChIP-seq, and CUT&RUN provide valuable insights but are limited by the ensemble nature of their data, masking the cellular and molecular heterogeneity that is often functionally significant. Recently, long-read sequencing technologies, particularly Single Molecule, Real-Time (SMRT/PacBio) sequencing, have introduced transformative capabilities, such as N6-methyladenine (6mA) footprinting. This technique enables the detection of joint binding events and long-range chromatin organization on individual DNA molecules spanning multiple kilobases. Despite the potential of 6mA footprinting, existing analytical tools for 6mA detection in SMRT sequencing data suffer from significant limitations in both performance and applicability, especially with the latest sequencing platform. To address these gaps, we developed a novel 6mA-calling pipeline based on polymerase kinetics analysis. Our approach significantly outperforms current tools in terms of accuracy and computational efficiency, setting a new benchmark for 6mA detection. Utilizing our optimized experimental and computational framework, we extensively mapped nucleosome positioning and transcription factor occupancy at the single-molecule level, revealing critical features of the transcription-associated epigenetic landscape. Additionally, our work established high-resolution, long-range binding events in mitochondrial DNA, revealing simultaneous loading of two sets of replication machinery onto the displacement loop (D-loop). Our study highlights the potential of 6mA footprinting in capturing the coordinate binding of nucleoproteins and unraveling heterogeneous epigenetic states with unprecedented resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39f69332a7cd1e65bdcc77503302539796aff1ef" target='_blank'>
              Revealing long-range heterogeneous organization of nucleoproteins with N6-methyladenine footprinting
              </a>
            </td>
          <td>
            Wentao Yang, Xue Qing Wang, Aaron M. Wenger, Fan Wei, Jingqi Yu, Yifan Liu, Y. Dou
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Myeloid leukemias are heterogeneous cancers with diverse mutations, sometimes in genes with unclear roles and unknown functional partners. PHF6 and PHIP are two poorly-understood chromatin-binding proteins recurrently mutated in acute myeloid leukemia (AML). PHF6 mutations are associated with poorer outcomes, while PHIP was recently identified as the most common selective mutation in Black patients in AML. Here, we show that PHF6 is a transcriptional repressor that suppresses a stemness gene network, and that PHF6 missense mutations, classified by current clinical algorithms as variants of unknown significance, produce unstable or non-functional protein. We present multiple lines of evidence converging on a critical mechanistic connection between PHF6 and PHIP. We show that PHIP loss phenocopies PHF6 loss, and that PHF6 requires PHIP to occupy chromatin and exert its downstream transcriptional program. Our work unifies PHF6 and PHIP, two disparate leukemia-mutated proteins, into a common functional complex that suppresses AML stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acc625ad647593a9481f172008906d4067da9a6" target='_blank'>
              Leukemia-mutated proteins PHF6 and PHIP form a chromatin complex that represses acute myeloid leukemia stemness
              </a>
            </td>
          <td>
            Aishwarya S Pawar, Patrick Somers, Aleena Alex, Subin S. George, Charles Antony, Roman Verner, Sanese K. White-Brown, Mohit Khera, M. S. Mendoza-Figueroa, K. F. Liu, Jennifer Morrissette, V. Paralkar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D. Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, Gary C. Hon
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Caviid gammaherpesvirus 1 (CaGHV-1), formerly known as the guinea pig herpes-like virus, is an oncogenic gammaherpesvirus with a sequenced genome but an as-yet uncharacterized transcriptome. Using nanopore long-read RNA sequencing, we annotated the CaGHV-1 genome and constructed a detailed transcriptomic atlas. Our findings reveal diverse viral mRNAs and non-coding RNAs, along with mapped promoter elements for each viral gene. We demonstrated that the CaGHV-1 RTA lytic cycle transcription factor activates its own promoter, similar to KSHV, and that the CaGHV-1 ORF50 promoter responds to RTA proteins from other gammaherpesviruses, highlighting the evolutionary conservation of RTA-mediated transcriptional mechanisms. Additionally, our analysis uncovered extensive transcriptional overlap within the viral genome, suggesting a role in regulating global gene expression. Given its tumorigenic properties, broad host range, and non-human pathogenicity, this work establishes CaGHV-1 as a promising small animal model for investigating human gammaherpesvirus pathogenesis. IMPORTANCE The molecular underpinnings of gammaherpesvirus pathogenesis remain poorly understood, partly due to limited animal models. This study provides the first comprehensive transcriptomic atlas of CaGHV-1, highlighting both coding and non-coding RNAs and revealing regulatory elements that drive viral gene expression. Functional studies of the CaGHV-1 RTA transcription factor demonstrated its ability to self-activate and cross-activate promoters from homologous gammaherpesviruses, reflecting conserved mechanisms of transcriptional control. These findings solidify CaGHV-1 as a unique and versatile small animal model, offering new opportunities to investigate gammaherpesvirus replication, transcriptional regulation, and tumorigenesis in a controlled experimental system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60c145bd10389c6f47e8b493a56c969c78162109" target='_blank'>
              Long-read Transcriptomics of Caviid Gammaherpesvirus 1: Compiling a Comprehensive RNA Atlas
              </a>
            </td>
          <td>
            Gábor Torma, Ákos Dörmő, Á. Fülöp, Dóra Tombácz, M. Mizik, Amanda M. Pretory, Seeyoun Lee, Z. Tóth, Z. Boldogkői
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In most eukaryotic cells, euchromatin is localized in the nuclear interior, whereas heterochromatin is enriched at the nuclear envelope (NE). This conventional chromatin organization is established by heterochromatin tethering to the NE, however its importance for cellular homeostasis is largely unexplored. Peripheral heterochromatin localization relies on redundant NE-tethering systems. One tether is constituted by the lamin B receptor (LBR) in mammals, but the enigmatic nature of the other tethers has hampered functional analyses. Here we demonstrate that the downregulation of abundant, ubiquitous NE proteins can induce the global detachment of heterochromatin from the NE. Among these factors, we identify LBR and LAP2 as major players in bulk heterochromatin attachment to the NE in pluripotent and differentiated mammalian cells. Their loss leads to repositioning of heterochromatin to the nuclear interior, changes in chromatin accessibility, deregulation of gene expression including activation of antiviral innate immunity, and defects in cell fate determination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fde3d7b0c4771c759bd6fcaac94c205c62bfd4c" target='_blank'>
              LBR and LAP2 mediate heterochromatin tethering to the nuclear periphery to preserve genome homeostasis
              </a>
            </td>
          <td>
            Renard Lewis, Virginia Sinigiani, K. Koos, Cristiana Bersaglieri, C. Ashiono, Raffaella Santoro, C. Ciaudo, Peter Horvath, Puneet Sharma, Ulrike Kutay
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5–34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18's role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18-/- and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination.
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-20</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="SETD6 is a non-histone lysine methylatransferase, previously shown to participate in several housekeeping signaling pathways such as the NFkB pathway, Wnt signaling pathway, mitosis and more. In the current study we show evidence that SETD6 methylation is involved in the regulation of cytokinesis - the final process that divides cell contents into two daughter cells. SETD6 depleted HeLa cells presented high levels of chromatin bridges and actin patches, which are commonly observed following chromosomal segregation errors. In a proteomic screen we identified Aurora-B as a novel SETD6 substrate. Aurora-B kinase is an essential regulator of cytokinesis, known to actively delay cytokinesis as a response to the presence of chromatin in the midzone. We found that SETD6 binds and methylates Aurora-B on two adjacent lysine residues. Upon replication stress, Aurora-B methylation by SETD6 increases but is abolished when the two lysine methylation targets are substituted. In addition, replication stress led to a high tendency of SETD6 depleted cells to multinucleate, a major chromosomal-instability (CIN) phenotype. We detected a significant reduction in the Aurora-B kinase activity during cytokinesis in SETD6 knockout cells upon replication stress, which could be the mechanism underlying the accumulation of CIN phenotypes in these cells. CIN is a hallmark of cancer and is associated with tumor cell malignancy. Our findings suggest that Aurora-B methylation by SETD6 carries meaningful implications on tumorigenic cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30079fd8588b1378d0602ce8bccaf091eb97d109" target='_blank'>
              AuroraB-kinase methylation by SETD6 regulates cytokinesis and protects cells from chromosomal instability
              </a>
            </td>
          <td>
            Michal Feldman, Anand Chopra, Dikla Nachmias, K. Biggar, Daniel Sevilla, Natalie Elia, D. Levy
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumor protein 53 (TP53) is a tumor repressor gene that is highly responsible for activating cell cycle arrest in the event of DNA damage, stopping the replication of damaged cells, which may cause cancer. However, when mutated, TP53 is an oncogene, with mutated variants of it being seen in over half of all cancers diagnosed including the aforementioned four. With groundbreaking CRISPR-Cas9 gene editing technology coming to the forefront of biotechnological research, the applications to cancer research and gene therapy have been increasing. This review explores CRISPR-Cas9 and its uses for treatment and medication investigation in various cancers, including lung, prostate, breast, and bone cancer through the lens of the TP53 gene. Due to TP53s commonality in cancer, CRISPR-Cas9 gene editing technology can be used to its maximum potential in this context as a treatment or investigative tool, with it yielding positive results in reverting tumor growth, as well as great versatility when it comes to screening related genes that inhibit or encourage tumor growth which can be targeted by drugs, or even the screening of the drugs effectiveness themselves. Thus, CRISPR-Cas9s applications in TP53-related cancer treatment and chemotherapeutic fields prove this technology to be successful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da087d15d0c931988cddd598ffd040ef96f3cb9b" target='_blank'>
              Using CRISPR/Cas9 Technology to Target TP53 Related Cancers
              </a>
            </td>
          <td>
            Man Ho Damien Hsieh
          </td>
          <td>2025-01-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is recognized as the most aggressive type of primary brain tumor. Despite recent advances in understanding the molecular mechanisms involved in the biology of glioblastoma, patient survival rates remain disappointing, primarily due to the lack of effective treatment options. Tumor necrosis factor receptor-associated protein 1 (TRAP1), a member of the heat shock protein 90 (Hsp90) family, refers to a protein predominantly localized in the mitochondria that regulates both cellular metabolic reprogramming and mitochondrial apoptosis. This protein is highly expressed in several types of tumors, including colorectal cancer, breast cancer, prostate cancer, and lung cancer, and is often associated with drug resistance. However, TRAP1 is also downregulated in certain cancers such as ovarian cancer, bladder cancer, and renal cancer, where its lower expression correlates with poorer prognoses and chemoresistance. The role of TRAP1 lies in enhancing or suppressing oxidative phosphorylation, with the impact of such regulation on tumor development and progression being a matter of ongoing debate. These observations prompt further investigation into the mechanisms responsible for the dual role of TRAP1 as both an oncogene and a tumor suppressor in specific types of tumors, particularly glioblastoma. The present study reviews the role of TRAP1 in the development and progression of glioblastoma and discusses the potential of targeting TRAP1 as a novel therapeutic approach against tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/236ff07b8d8eaf07109bcd843d2d90948b45e7ad" target='_blank'>
              Role of TRAP1 Protein in the Development and Progression of Glioblastoma
              </a>
            </td>
          <td>
            I. Gareev, A. Yasinskaya, S. Roumiantsev
          </td>
          <td>2024-12-28</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4cf48abf7265148385bfea0bb2728f6fd6101" target='_blank'>
              SMYD3 drives cell cycle and epithelial-mesenchymal transition pathways through dual gene transcriptional repression and activation in HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Madhavi Murali, Abbas Saeed, Sohyoung Kim, K. Burkitt, Hui Cheng, Arfa Moshiri, Jawad Akhtar, D. Tsai, Marie Luff, Baktiar Karim, V. Saloura
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Alterations in DNA methylation profiles are typically found in cancer cells, combining genome-wide hypomethylation with hypermethylation of specific regions, such as CpG islands, which are normally unmethylated. Driving effects in cancer development have been associated with alteration of DNA methylation in certain regions, inducing, for example, the repression of tumor suppressor genes or the activation of oncogenes and retrotransposons. These alterations represent prime candidates for the development of specific markers for the detection, diagnosis and prognosis of cancer. In particular, these markers, distributed along the genome, provide a wealth of information that offers potential for innovation in the field of liquid biopsy, in particular thanks to the emergence of artificial intelligence for diagnostic purposes. This could overcome the limitations related to sensitivities and specificities, which remain too low for the most difficult applications in oncology: the detection of cancers at an early stage, the monitoring of residual disease and the analysis of brain tumors. In addition, targeting the enzymatic processes that control the epigenome offers new therapeutic strategies that could reverse the regulatory anomalies of these altered epigenomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6734e14b7462be1947bbbaa4edaec73c090d31d" target='_blank'>
              [Epigenetics and cancer: the role of DNA methylation].
              </a>
            </td>
          <td>
            Marine Gorse, Charline Bianchi, Charlotte Proudhon
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Structural variation (SV), defined as balanced and unbalanced chromosomal rearrangements >1 kb, is a major contributor to germline and neoplastic disease. Large variants have historically been evaluated by chromosome analysis and now are commonly recognized by chromosomal microarray analysis (CMA). The increasing application of genome sequencing (GS) in the clinic and the relatively high incidence of chromosomal abnormalities in sick newborns and children highlights the need for accurate SV interpretation and reporting. In this review, we describe SV patterns of common cytogenetic abnormalities for laboratorians who review GS data.


CONTENT
GS has the potential to detect diverse chromosomal abnormalities and sequence breakpoint junctions to clarify variant structure. No single GS analysis pipeline can detect all SV, and visualization of sequence data is crucial to recognize specific patterns. Here we describe genomic signatures of translocations, inverted duplications adjacent to terminal deletions, recombinant chromosomes, marker chromosomes, ring chromosomes, isodicentric and isochromosomes, and mosaic aneuploidy. Distinguishing these more complex abnormalities from simple deletions and duplications is critical for phenotypic interpretation and recurrence risk recommendations.


SUMMARY
Unlike single-nucleotide variant calling, identification of chromosome rearrangements by GS requires further processing and multiple callers. SV databases have caveats and limitations depending on the platform (CMA vs sequencing) and resolution (exome vs genome). In the rapidly evolving era of clinical genomics, where a single test can identify both sequence and structural variants, optimal patient care stems from the integration of molecular and cytogenetic expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a2a826f1810ca4839038af6a42ce3daef01b34" target='_blank'>
              Structural Variation Interpretation in the Genome Sequencing Era: Lessons from Cytogenetics.
              </a>
            </td>
          <td>
            Lucilla Pizzo, M. K. Rudd
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic therapies are emerging for pediatric cancers. Due to the relatively low mutational burden in pediatric tumors, epigenetic dysregulation and differentiation blockade is a hallmark of oncogenesis in some childhood cancers. By targeting epigenetic regulators that maintain tumor cells in a primitive developmental state, epigenetic therapies may induce differentiation. The most well-studied and clinically advanced epigenetic-targeted therapies include azacitidine and decitabine, which inhibit DNA methylation through competitive inhibition of the enzymatic activity of the DNA methyltransferase family enzymes. These DNA hypomethylating agents are Food and Drug Administration (FDA) approved for hematologic malignancies. The discovery that DNA hypermethylation occurs in patients with isocitrate dehydrogenase (IDH) mutations has led to the development and FDA approval of IDH inhibitors for hematologic and solid tumors. Epigenetic dysregulation in pediatric tumors is also driven by changes in the "histone code" that either promote oncogene expression or repress tumor suppressors. Cancers whose chromatin landscape is characterized by such aberrant histone posttranslational modifications may be amenable to targeted therapies that inhibit the chromatin-modifying enzymes that read, write, and erase these histone modifications. Small molecules that inhibit the enzymatic activity of histone deacetylases, acetyltransferases, and methyltransferases have been approved for the treatment of some adult cancers, and these agents are currently under investigation in various pediatric tumors. Chromatin regulatory complexes can be hijacked by oncogenic fusion proteins that are produced by chromosomal translocations, which are common drivers in pediatric cancer. Small molecules that disrupt oncogenic fusion protein activity and their associated chromatin complexes have demonstrated remarkable promise, and this approach has become the standard treatment for a subset of leukemias driven by the PML-RARA oncogenic fusion protein. A deeper understanding of the mechanisms that drive epigenetic dysregulation in pediatric cancer may hold the key to future success in this field, as the landscape of druggable epigenetic targets is also expanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a807fa68d0895be1a6925ed8f18dcca64f6453e" target='_blank'>
              Epigenetic Therapies.
              </a>
            </td>
          <td>
            Wallace Bourgeois, Scott A Armstrong, Emily B. Heikamp
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge.


EXPERIMENTAL DESIGN
We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. The DIAMOND (Detection of Long Interspersed Nuclear Element Altered Methylation ON plasma DNA) method targets 30-40,000 young L1 scattered throughout the genome, covering about 100,000 CpG sites and is based on a reference-free analysis pipeline.


RESULTS
Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers (colorectal, breast, lung, ovarian, gastric cancers and uveal melanoma including localized stages) in two independent cohorts (AUC = 88% to 100%, N = 747). DIAMOND can also be used to perform copy number alterations (CNA) analysis which improves cancer detection.


CONCLUSIONS
This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321c21ecf5626d3880de9fbd35f995ec3b32a1f7" target='_blank'>
              Non-invasive multi-cancer detection using DNA hypomethylation of LINE-1 retrotransposons.
              </a>
            </td>
          <td>
            M. Michel, Maryam Heidary, Anissa Mechri, Kévin Da Silva, Marine Gorse, Victoria Dixon, Klaus von Grafenstein, Charline Bianchi, C. Hego, A. Rampanou, C. Lamy, M. Kamal, C. Le Tourneau, M. Séné, I. Bieche, Cécile Reyes, D. Gentien, Marc-Henri Stern, Olivier Lantz, L. Cabel, J-Y Pierga, F. Bidard, Chloé-Agathe Azencott, C. Proudhon
          </td>
          <td>2024-12-02</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The long non-coding RNA (lncRNA), HAR1A is emerging as a putative tumour suppressor. In non-neoplastic brain cells, REST suppresses HAR1A expression. In gliomas REST acts as an oncogene and is a potential therapeutic target. It is therefore conceivable that REST promotes glioma progression by down-regulating HAR1A. To test this hypothesis, glioma clinical databases were analysed to study: (I) HAR1A/REST correlation; (II) HAR1A and REST prognostic role; (III) molecular pathways associated with these genes. HAR1A expression and subcellular localization were studied in glioblastoma and paediatric glioma cells. REST function was also studied in these cells, by observing the effects of gene silencing on: (I) HAR1A expression; (II) cancer cell proliferation, apoptosis, migration; (III) expression of neural differentiation genes. The same phenotypes (and cell morphology) were studied in HAR1A overexpressing cells. Our results show that REST and HAR1A are negatively correlated in gliomas. Higher REST expression predicts worse prognosis in low-grade gliomas (the opposite is true for HAR1A). REST-silencing induces HAR1A upregulation. HAR1A is primarily detected in the nucleus. REST-silencing dramatically reduces cell proliferation and induces apoptosis, but HAR1A overexpression has no major effect on investigated cell phenotypes. We also show that REST regulates the expression of neural differentiation genes and that its oncogenic function is primarily HAR1A-independent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3631f5927fc44541e398927b977abcbc19ff8853" target='_blank'>
              REST-dependent glioma progression occurs independently of the repression of the long non-coding RNA HAR1A.
              </a>
            </td>
          <td>
            Ella Waters, Perla Pucci, Ruman Rahman, A. P. Yatsyshyna, Harry Porter, Mark Hirst, R. Gromnicova, Igor Kraev, V. Mongiardini, Benedetto Grimaldi, Jon Golding, Helen L Fillmore, Balázs Győrffy, Priyadarsini Gangadharannambiar, Christos N Velanis, Christopher J Heath, F. Crea
          </td>
          <td>2024-11-27</td>
          <td>PloS one</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA, our findings rather suggest that protein-DNA co-condensates ‘sense’ the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d84d86b264f5c1ad8de5c81e2ae317173c400dd" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Thejas Satheesh, Rohit Kumar Singh, Prakshi Gaur, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA double-strand break (DSB) always occurs within genome. The pathways that repair the DSB are essential in keeping cell viability. This review will focus on the recent understanding of TMEJ, especially two key protein factors involved in the TMEJ repair pathway2, DNA polymerase theta (Polθ) and DNA polymerase delta (Polδ), and discuss the potential mechanism of choosing repair pathways when there is a DNA Double Strand Break. We dissect TMEJ's unique mechanism of action, including recognition of DNA ends, microhomology search, end pairing, and DNA synthesis, by focusing on the enzyme Polθ as the central player to understand the repair pathway choice. We also discuss how BRCA-mutated cancer uses the TMEJ repair pathway to facilitate breast cancer cells' growth3. This review aims to provide a comprehensive overview of TMEJ's cellular functions, regulatory mechanisms, and pivotal role in BRCA-mutated breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba6a1c7fb204a897897b1b79890ca6faef600e" target='_blank'>
              Polymerase Theta-mediated DNA End-Joining Repair in the Maintenance of Genome Stability
              </a>
            </td>
          <td>
            Xiangyu Yan
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e175d7f8eb691015e68d15e186cd0072f7b5db" target='_blank'>
              hnRNPU-mediated pathogenic alternative splicing drives gastric cancer progression
              </a>
            </td>
          <td>
            Guoguo Jin, Yanming Song, Shaobo Fang, Mingyang Yan, Zhaojie Yang, Yang Shao, Kexin Zhao, Meng Liu, Zhenwei Wang, Zhiping Guo, Zigang Dong
          </td>
          <td>2025-01-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Gene fusions are nucleotide sequences formed due to errors in replication and transcription control. These errors, resulting from chromosomal translocation, transcriptional errors or trans-splicing, vary from cell to cell. The identification of fusions has become critical as key biomarkers for disease diagnosis and therapy in various cancers, significantly influencing modern medicine. Chimeric Transcripts and RNA-Sequencing database version 8.0 (ChiTaRS 8.0; http://biosrv.org/chitars) is a specialized repository for human chimeric transcripts, containing 47 445 curated RNA transcripts and over 100 000 chimeric sequences in humans. This updated database provides unique information on 1055 chimeric breakpoints derived from public datasets using chromosome conformation capture techniques (the Hi-C datasets). It also includes an expanded list of gene fusions that are potential drug targets, and chimeric breakpoints across 934 cell lines, positioning ChiTaRS 8.0 as a valuable resource for testing personalized cancer therapies. By utilizing text mining on a curated selection of disease-specific RNA-sequencing data from public datasets, as well as patient blood and plasma samples, we have identified novel chimeras—particularly in diseases such as oral squamous cell carcinoma and glioblastoma—now catalogued in ChiTaRS. Thus, ChiTaRS 8.0 serves as an enhanced fusion transcript repository that incorporates insights into the functional landscape of chimeras in cancers and other complex diseases, based on liquid biopsy results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733a293484c1701290a845d632917cdf236590b" target='_blank'>
              ChiTaRS 8.0: the comprehensive database of chimeric transcripts and RNA-seq data with applications in liquid biopsy
              </a>
            </td>
          <td>
            Dylan DSouza, Lihi Bik, Olawumi Giwa, Shahaf Cohen, Hilit Levy Barazany, T. Siegal, M. Frenkel-Morgenstern
          </td>
          <td>2024-12-16</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The nucleolus is a major subnuclear compartment where ribosomal DNA (rDNA) is transcribed and ribosomes are assembled. In addition, recent studies have shown that the nucleolus is a dynamic organizer of chromatin architecture that modulates developmental gene expression. rDNA gene units are assembled into arrays located in the p-arms of five human acrocentric chromosomes. Distal junctions (DJs) are ∼400 kb sequences adjacent to rDNA arrays that are thought to anchor them at the nucleolus, although the underlying regulatory elements remain unclear. Here we show that DJs display a dynamic chromosome conformation profile in human embryonic stem cells (hESCs). We identified a primate-specific, full-length insertion of the retrotransposon long interspersed nuclear element 1 (LINE1) in a conserved position across all human DJs. This DJ-LINE1 locus interacts with specific regions of the DJ and is upregulated in naïve hESCs. CRISPR-based deletion and interference approaches revealed that DJ-LINE1 contributes to nucleolar positioning of the DJs. Moreover, we found that the expression of DJ-LINE1 is required for maintenance of the structure and transcriptional output of the nucleolus in hESCs. Silencing of DJ-LINE1 leads to loss of self-renewal, disruption of the landscape of chromatin accessibility, and derepression of earlier developmental programs in naïve hESCs. This work uncovers specific LINE1 elements with a fundamental role in nucleolar organization in hESCs and provides new insights into how the nucleolus functions as a key genome-organizing hub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6f03971b0054eedbed60fad051bf9b167c6012" target='_blank'>
              LINE1 elements at distal junctions of rDNA repeats regulate nucleolar organization in human embryonic stem cells.
              </a>
            </td>
          <td>
            Lamisa Ataei, Juan Zhang, Simon Monis, Krystyna Giemza, Kirti Mittal, Joshua Yang, Mayu Shimomura, B. McStay, Michael D. Wilson, M. Ramalho-Santos
          </td>
          <td>2024-12-20</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) imposes a significant health burden, necessitating innovative therapeutic strategies to enhance treatment efficacy. Current treatments such as surgery, radiation, and chemotherapy have limited effectiveness and yield severe side effects, emphasizing the need for targeted therapies. We have focused on DNA polymerase beta (Polβ) and its roles in replication stress, cellular responses to DNA damaging therapies, and DNA damage response modifiers. Our investigations reveal a regulatory role for base excision repair (BER) proteins, including Polβ, in the cellular response to inhibitors of poly(ADP-ribose) glycohydrolase (PARG), an enzyme involved in poly(ADP-ribose) (PAR) degradation. The inhibition of PARG, in HNSCC cells, elicits replication stress. Further, this activates the PAR-induced S-phase/ATR checkpoint, leading to a block to replication, cell cycle arrest, and the onset of apoptosis. However, Polβ overexpression mitigates this response, reducing replication-stress induced PAR foci formation, suggesting a modulation of replication checkpoint activation. We found that PARG inhibitor treatment is ineffective on HNSCC cells that overexpress Polβ, implying that the PARG inhibitor-induced PAR and apoptotic response is dependent on the level of Polβ. Further, our in vitro experiments demonstrate that combining PARG and ATR/CHK1 inhibitors overcomes Polβ-mediated treatment resistance in HNSCC cells, producing enhanced effects as compared to the individual treatment conditions. Our findings suggest a possible treatment paradigm for HNSCC, employing ATR or CHK1 inhibitors in combination with PARG inhibitors. This strategy offers a promising path for more effective HNSCC treatments, potentially overcoming Polβ-related resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546af29aab21a1579fd4129cca1f86d70f7aaf27" target='_blank'>
              DNA polymerase beta expression in head & neck cancer modulates the poly(ADP-ribose)-mediated replication checkpoint
              </a>
            </td>
          <td>
            Md Maruf Khan, Wynand P. Roos, Anusha Angajala, Denise Y. Gibbs, Jeffrey C. Liu, Camille Ragin, Robert W. Sobol
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Insertion sequences (ISs) are mobile pieces of DNA that are widespread in bacterial genomes. IS movements typically involve (i) excision of the IS element, (ii) cutting of target site DNA, and (iii) IS element insertion. This process generates a new copy of the IS element and a short duplication at the target site. It has been noted that, for some extant IS copies, no target site duplications (TSDs) are readily identifiable. TSD absence has been attributed to degeneration of the TSD after the insertion event, recombination between identical ISs, or adjacent deletions. Indeed, the latter two—recombination between ISs and adjacent deletions—are frequent causes for the absence of TSDs, which we demonstrate here in an analysis of genome sequence data from the Lenski long-term evolution experiment. Furthermore, we propose that some IS movements—namely, those that occur in association with large-scale genomic rearrangements—do not generate TSDs, and occur without evidence for recombination between ISs or adjacent deletions. In support of this hypothesis, we provide two direct, empirical observations of such IS transposition events: an IS5 movement plus a large deletion in Escherichia coli C, and an IS481 movement occurring with a large duplication in Pseudomonas fluorescens SBW25. Although unlikely, it is possible that the observed deletion and associated IS movement occurred in two successive events in one overnight culture. However, an IS at the center of a large-scale duplication is not readily explained, suggesting that IS element activity may promote both large-scale deletions and duplications. IMPORTANCE Insertion sequences are the most common mobile genetic elements found in bacterial genomes, and hence they significantly impact bacterial evolution. We observe insertion sequence movement at the center of large-scale deletions and duplications that occurred during laboratory evolution experiments with Escherichia coli and Pseudomonas fluorescens, involving three distinct types of transposase. We raise the possibility that the transposase does not mediate DNA cleavage but instead inserts into existing DNA breaks. Our research highlights the importance of insertion sequences for the generation of large-scale genomic rearrangements and raises questions concerning the mechanistic basis of these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00001ae08c5c01e65cbd4f4a4b234bdd427cfa24" target='_blank'>
              A more significant role for insertion sequences in large-scale rearrangements in bacterial genomes
              </a>
            </td>
          <td>
            Wing Y. Ngan, Lavisha Parab, Frederic Bertels, J. Gallie
          </td>
          <td>2024-12-05</td>
          <td>mBio</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="SUMMARY DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild‐type Arabidopsis thaliana somatic cells exposed either to UV‐B, to UV‐C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia‐telangiectasia‐mutated) and ATR (Ataxia‐telangiectasia‐mutated and Rad3‐related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations‐induced deleted regions allowed determining that exposure to UV‐B, UV‐C and protons induced the microhomology‐mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations‐induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Despite the evolutionary importance of supergenes, their properties in polyploids remain unexplored. Polyploid genomes are expected to undergo chromosomal rearrangements and gene losses over time, potentially affecting supergene architecture. The iconic distyly supergene (S-locus), controlling a floral heteromorphism with two self-incompatible morphs, has been well-documented in diploids, but remains unknown in polyploids. Primula, the classic model for distyly since Darwin, is ancestrally diploid and distylous, yet polyploid, homostylous species with a single, self-compatible floral morph evolved repeatedly. The intraspecific loss of distyly is associated with small loss-of-function mutations in the S-locus CYPT gene controlling style length and female self-incompatibility. Over longer timescales, relaxed selection on CYPT should generate greater accumulation of larger mutations, including exon and gene loss. By analyzing the first assembled genome of an allotetraploid, homostylous species (Primula grandis) in a comparative framework, we discovered two, nearly identical S-locus alleles in the same subgenome, suggesting it originated via inter-specific hybridization between a homostylous and a distylous progenitor. Conformant to predictions from theory, the macroevolutionary loss of distyly coincided with considerable degeneration of CYPT, while other S-locus genes remained largely unaffected, suggesting the shift to homostyly preceded and facilitated polyploid establishment. At the whole-genome level, we found minimal subgenome dominance — as expected, given the inferred recent origin of P. grandis — and highly reduced genetic diversity, congruently with its narrow distribution and self-compatibility. This study provides the first comparison of a supergene across ploidy levels and reproductive systems, contributing new knowledge on the previously unknown fate of supergenes in polyploids. SIGNIFICANCE This study advances knowledge on genome evolution by elucidating how supergenes (clusters of tightly linked genes) evolve across species with different sets of chromosomes and reproductive systems. By analyzing the newly assembled genome of the polyploid, self-compatible Primula grandis in a broad framework, we provide the first comparison of the distyly supergene between diploid outcrossers and polyploid self-fertilizers. We discovered one pair of identical supergene alleles in the same subgenome, rather than one pair per subgenome, revealing the species originated via a cross between a self-compatible and a self-incompatible progenitor. Conformant to theory, the gene controlling female self-incompatibility and style length (CYPT) was considerably degenerated, because of relaxed selection over time, with the rest of the supergene largely unaffected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99c78bc08b60ac655953b43bdc71e27d5d4f87" target='_blank'>
              Fate of a supergene in the shift from diploidy to polyploidy
              </a>
            </td>
          <td>
            E. Mora-Carrera, N. Yousefi, G. Potente, R. Stubbs, B. Keller, Étienne Léveillé-Bourret, Grob Stefan, F. Celep, G. Tedoradze, E. Conti
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86fef8bdb07b67c65a912b81891de3ba97bc74cb" target='_blank'>
              Osmotic disruption of chromatin induces Topoisomerase 2 activity at sites of transcriptional stress
              </a>
            </td>
          <td>
            William H. Gittens, Rachal M. Allison, Ellie M Wright, George G B Brown, Matthew J. Neale
          </td>
          <td>2024-12-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff1090b04839914362d5c68932f9ab6b30614f" target='_blank'>
              Emerging roles of cohesin-STAG2 in cancer.
              </a>
            </td>
          <td>
            Julia S Scott, Loubna Al Ayadi, Emmanouela Epeslidou, Roan H van Scheppingen, Anna Mukha, Lucas J T Kaaij, C. Lutz, S. Prekovic
          </td>
          <td>2024-11-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here we show that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells display increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence is rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimics the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slows growth of multiple PAX3-FOXO1+ tumor xenograft models, but not fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50231dc8772bfbbe71ed944e599efde5187dc0b" target='_blank'>
              PAX3-FOXO1 drives targetable cell state-dependent metabolic vulnerabilities in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, , Angela M. Montero, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, Andrew L Kung, Daoqi You, Filemon Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), a highly aggressive brain tumor, frequently develops resistance to temozolomide (TMZ), the current standard chemotherapy. Our study investigates the potential of combining TMZ with the poly (ADP-ribose) polymerase inhibitor Olaparib (OLA) to overcome TMZ resistance. Using in vitro models, including U251 cell lines and patient-derived GBM primary cultures, we demonstrate that OLA enhances TMZ efficacy by disrupting base excision repair and potentiating DNA damage-induced cytotoxicity. A CRISPR knockout screen identified DNA mismatch repair (MMR) gene deficiencies as key drivers of TMZ resistance, which OLA effectively counteracts. Co-treatment with TMZ and OLA significantly reduced tumor cell viability, even in MMR-deficient and MGMT-expressing contexts, suggesting a synergistic mechanism. Gene expression analysis revealed that the combination therapy impacts cell cycle regulation, stress responses, and extracellular matrix integrity, leading to mitotic catastrophe and apoptosis. These findings propose the TMZ-OLA combination as a promising therapeutic strategy for overcoming chemoresistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edc2d11f92b839420cb397da78a6ec7ab2aa82c" target='_blank'>
              PARP inhibitor counteracts Temozolomide Resistance in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Edouard Samarut, Tabourel Gaston, Briand Joséphine, Landais Yuna, Gratas Catherine, Cartron Pierre-François, Vallette François, Oliver Lisa, Sérandour A. Aurélien
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="T2T genome is a telomere-to-telomere high-quality genome with exceptional accuracy, continuity and integrity. It significantly filled some gaps of chromosome. However, the genomic features unique to the filled regions assembled in the T2T genome across species remain unclear. Here, we collected and compared T2T genomes with their older version from human (Homo sapiens), animal (Mus musculus), and plants (Arabidopsis thaliana, Musa acuminata, and Fragaria vesca). We explored the newly filled regions generated by the T2T genome in comparison to the old one, and focused on: (1) the comparison of their locations, (2) the distribution of genomic features, and (3) gene annotation and gene function for filled regions. Research indicated that the filled regions of the T2T genome ordinarily appear in the centromere and telomere regions, contain a great number of repetitive sequences. The T2T genome revealed the identification of a certain number of newly annotated genes, alongside significant variations in common annotated genes. Interestingly, the genomic features of filled regions exhibit distinct patterns between plants and human/animal. T2T genomes feature analysis serves as a crucial resource for studying pattern characteristics across diverse species and advancing the in-depth development of genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88844fa94943a306afc6b801a890b4cc98392746" target='_blank'>
              Genomic feature of filled regions of telomere-to-telomere genomes in five model species
              </a>
            </td>
          <td>
            Chu Xiong, Jun Zheng, Hui Zhang, Yunpeng Zhang, Lihong Hao, Min Wang, Zhen Liang
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="TNK2 is a ubiquitously expressed nonreceptor-type tyrosine kinase. TNK2 participates in tumorigenesis, and TNK2 activation has been found in various cancers; therefore, TNK2 is a promising target for cancer chemotherapy. While the TNK2 inhibitor XMD16-5 is highly selective, it inhibits cytokinesis at higher concentrations by targeting Aurora B kinase, a key enzyme for cell division. Cytokinesis failure frequently generates polyploid cells, and the surviving polyploid cells risk leading to cancer development and malignant progression via chromosome instability. In this study, to investigate the possibility that (R)-9bMS, a TNK2 inhibitor structurally related to XMD16-5, drives malignant progression by inducing abnormal cell division, we examined its effects on cell division, Aurora B autophosphorylation, and colony formation. Cell count results showed a reduction in the number of A431, HeLa S3, HCT116, and MCF7 cells upon TNK2 inhibitor treatment. Microscopic observation indicated the formation of multinucleated and nucleus-enlarged cells. An increase in DNA content was confirmed with flow cytometry, which was underpinned by an increased number of centrosomes. Time-lapse imaging revealed mitotic failure, such as mitotic slippage and cytokinesis failure, as a cause of polyploidization. Of note, TNK2 knockdown significantly increased multinucleated cells, but the effect was quite weak, suggesting that TNK2 inhibition may only partially contribute to mitotic failure and polyploidization. Expectedly, Aurora B phosphorylation was reduced by (R)-9bMS like XMD16-5, but not by TNK2 knockdown. Collectively, TNK2 inhibitors (R)-9bMS and XMD16-5 induce polyploidization via mitotic failure caused by the inhibition of Aurora B kinase rather than TNK2. Notably, (R)-9bMS treatment promoted anchorage-independent colony formation, a hallmark of cancer. Our findings suggest that (R)-9bMS at a high concentration risks promoting cancer development or malignant progression. Therefore, caution should be used when using TNK2 inhibitors for cancers where TNK2 activation is not the transforming mutation and higher concentrations of TNK2 inhibitors are required to slow proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83021c8768a54cea8919a992b5bb8d7150bf2fc1" target='_blank'>
              TNK2 Inhibitor (R)-9bMS Causes Polyploidization Through Mitotic Failure by Targeting Aurora B.
              </a>
            </td>
          <td>
            Mayu Murata, Hiroki Kuwajima, Junna Tanaka, Nanami Hasegawa, Ryuzaburo Yuki, Youhei Saito, Yuji Nakayama
          </td>
          <td>2024-12-01</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, Nour H Marzouk, Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks as the third most prevalent cancer globally and is the second leading cause of cancer mortality. FAM49B, a member of the FAM49 gene family, is a recently identified, evolutionarily conserved gene. Emerging studies indicate that FAM49B plays a role in various cancers, though its specific mechanism in CRC remains largely unexplored. In this study, we observed that FAM49B was abnormally expressed in CRC tissues and cell lines, with elevated expression correlating with poor patient prognosis. FAM49B knockdown markedly suppressed CRC cell proliferation by arresting the cell cycle and reducing cell migration and invasion. Single-cell RNA-seq (ScRNA-seq) analysis revealed that high FAM49B expression in malignant epithelial cell clusters was strongly linked to c-Myc oncogene activation. Further, FAM49B knockdown significantly reduced c-Myc expression by enhancing its K48 ubiquitination. We identified NEK9 as a direct interacting partner of FAM49B, with FAM49B knockdown inhibiting NEK9-Thr210 phosphorylation. Similarly, high NEK9 expression was linked to unfavorable prognosis in CRC. In FAM49B-overexpressing CRC cells, NEK9 knockdown significantly suppressed c-Myc expression, c-Myc-ser62 phosphorylation, and reduced cell proliferation, migration, and invasion. Thus, directly targeting the FAM49B/NEK9/c-Myc pathway presents a promising therapeutic approach for c-Myc positive CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aecd72da94ed813728ad7c7716f5f1747f110c" target='_blank'>
              FAM49B drives colorectal cancer progression by stabilizing c-Myc through NEK9 phosphorylation.
              </a>
            </td>
          <td>
            Chen Lu, Tianyu Liu, E. Yimin, Lin Miao, Chunzhao Yu, Jianping Zhang, Xiagang Luo
          </td>
          <td>2025-01-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pseudouridine (Ψ) is a post-transcriptional modifier of RNA, often referred to as the ‘fifth nucleotide’ owing to its regulatory role in various biological functions as well as because of its significant involvement in the pathogenesis of human cancer. In recent years, research has revealed various Ψ modifications in different RNA types, including messenger RNA, transfer RNA, ribosomal RNA, small nuclear RNA, and long noncoding RNA. Pseudouridylation can significantly alter RNA structure and thermodynamic stability, as the Ψ-adenine (A) base pair is more stable than the typical uridine (U)-A base pair is due to its structural similarity to adenine. Studies have linked Ψ expression to the development and progression of several digestive system cancers, such as liver cancer and colorectal cancer, and nondigestive system cancers, such as breast cancer, non-small cell lung cancer, prostate cancer, glioblastoma, ovarian cancer, oral squamous cell carcinoma, and pituitary cancer. The present review briefly outlines the chemical structure, synthesis, and regulatory mechanisms of Ψ. This review summarizes the effects of pseudouridylation on various substrates of RNA and briefly discusses methods for detecting Ψ. Last, it focuses on how RNA pseudouridylation influences different cancers, emphasizing the search for novel approaches to cancer diagnosis, treatment, and prognosis through Ψ modification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9466c719e8cf6ff18a146061c808211210df53e3" target='_blank'>
              Advancements in pseudouridine modifying enzyme and cancer
              </a>
            </td>
          <td>
            Kaijie Liu, Shujun Zhang, Yafeng Liu, Xinjun Hu, Xinyu Gu
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are important regulators of gene expression whose dysregulation is widely linked to tumourigenesis, tumour progression and Epithelial-Mesenchymal Transition (EMT), a developmental process that promotes metastasis when inappropriately activated. However, controversy has emerged regarding how many functional miRNAs are encoded in the genome, and to what extent non-regulatory products of RNA degradation have been mis-identified as miRNAs. Central to miRNA function is their capacity to associate with an Argonaute (AGO) protein and form an RNA-Induced Silencing Complex (RISC), which mediates target mRNA suppression. We report that numerous “miRNAs” previously reported in EMT and cancer contexts, are not incorporated into RISC and are not capable of endogenously silencing target genes, despite the fact that hundreds of publications in the cancer field describe their roles. Apparent function can be driven through the expression of artificial miRNA mimics which is not necessarily reflective of any endogenous gene regulatory function. We present biochemical and bioinformatic criteria that can be used to distinguish functional miRNAs from mistakenly annotated RNA fragments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1ebefc068bdba0793a707a24cbd7cd0a5f423e" target='_blank'>
              Chasing non-existent “microRNAs” in cancer
              </a>
            </td>
          <td>
            Ayla Orang, Nicholas I Warnock, M. Migault, B. Dredge, A. Bert, Julie M Bracken, Philip A Gregory, Katherine A. Pillman, Gregory J Goodall, C. Bracken
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The kinesin family member 18A (KIF18A) is an essential regulator of microtubule dynamics and chromosome alignment during mitosis. Functional dependency on KIF18A varies by cell type and genetic context but the heritable factors that influence this dependency remain unknown. To address this, we took advantage of the variable penetrance observed in different mouse strain backgrounds to screen for loci that modulate germ cell depletion in the absence of KIF18A. We found a significant association at a Chr5 locus where anaphase promoting complex subunits 5 (Anapc5) and 7 (Anapc7) were the top candidate genes. We found that both genes were differentially expressed in a sensitive strain background when compared to resistant strain background at key timepoints in gonadal development. We also identified a novel retroviral insertion in Anapc7 that may in part explain the observed expression differences. In cell line models, we found that depletion of KIF18A induced mitotic arrest, which was partially rescued by co-depletion of ANAPC7 (APC7) and exacerbated by co-depletion of ANAPC5 (APC5). These findings suggest that differential expression and activity of Anapc5 and Anapc7 may influence sensitivity to KIF18A depletion in germ cells and CIN cells, with potential implications for optimizing antineoplastic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22896bf7bb510f98354d16e22622adcfeda18ffb" target='_blank'>
              Anapc5 and Anapc7 as genetic modifiers of KIF18A function in fertility and mitotic progression
              </a>
            </td>
          <td>
            Carleigh Nesbit, Whitney Martin, A. Czechanski, Candice Byers, Narayanan Raghupathy, Ardian Ferraj, Jason K. Stumpff, L. Reinholdt
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The kinetochore, an intricate macromolecular protein complex located on chromosomes, plays a pivotal role in orchestrating chromosome segregation. It functions as a versatile platform for microtubule assembly, diligently monitors microtubule binding fidelity, and acts as a force coupler. Comprising over 100 distinct proteins, many of which exist in multiple copies, the kinetochore’s composition dynamically changes throughout the cell cycle, responding to specific timing and conditions. This dynamicity is important for establishing functional kinetochores, yet the regulatory mechanisms of these dynamics have largely remained elusive. In this study, we employed advanced quantitative immunofluorescence techniques to meticulously chart the dynamics of kinetochore protein levels across the cell cycle. These findings offer a comprehensive view of the dynamic landscape of kinetochore architecture, shedding light on the detailed mechanisms of microtubule interaction and the nuanced characteristics of kinetochore proteins. This study significantly advances our understanding of the molecular coordination underlying chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe1c4335d69edcb64d4e5d971e98a59daf3526c" target='_blank'>
              CENcyclopedia: Dynamic Landscape of Kinetochore Architecture Throughout the Cell Cycle
              </a>
            </td>
          <td>
            Yu-Chia Chen, Ece Kilic, Evelyn Wang, Will Rossman, Aussie Suzuki
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="With the practical implementation of the CRISPR/Cas technology for targeted genome editing, it has become possible to carry out genetic engineering manipulations with eukaryotic genomes with high efficiency. One of the key stages of this technology is the targeted induction of site-specific DNA cleavages (breaks). The cell repairs these breaks via one of two pathways: nonhomologous end joining or homologous recombination. The choice of DNA repair pathway is determined by the architecture of the sites at the DNA break area formed as a result of terminal resection and depends on the phases of the cell cycle. Nonhomologous end joining is the main pathway for repair of double-stranded DNA breaks in mammalian cells. It involves a nonspecific ligation reaction, the accuracy of which depends on the structure of the ends of the break, and can result in various insertions or deletions in the target region of the genome. Integration of the desired sequence into the genome occurs along the path of homologous recombination, the implementation of which requires a matrix with homology regions on both sides of the double-strand break. The introduction of a genetic construct into a given location in the genome is an important, but currently complex and labor-intensive task. At the same time, for fundamental studies of gene function and the creation of animal models of human diseases, the choice of the repair pathway can be of fundamental importance. This review is an attempt to combine and structure all known information on approaches to increasing the efficiency of DNA repair involving homologous recombination. The article lists the most effective strategies to shift the balance towards homologous repair, such as the use of inhibitors of the non-homologous end joining mechanism, regulation of key factors of homologous recombination, control of the cell cycle, chromatin status, construction of templates for homologous recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23816bdc012e17808d654e03c85cd07d45917c31" target='_blank'>
              How to shift the equilibrium of dna break repair in favor of homology recombination
              </a>
            </td>
          <td>
            O. A. Averina, S. A. Kuznetsova, O. Permyakov, P. V. Sergiev
          </td>
          <td>2024-12-30</td>
          <td>Molekulârnaâ biologiâ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transposable elements (TEs) are significant drivers of genome evolution, yet their recent dynamics and impacts within and among species, as well as the roles of host genes and non-coding RNAs in the transposition process, remain elusive. With advancements in large-scale pan-genome sequencing and the development of open data sharing, large-scale comparative genomics studies have become feasible. Here, we performed complete de novo TE annotations and identified active TEs in 310 plant genome assemblies across 119 species and seven crop populations. Using 811 high-quality genomes, we detected 13 844 553 TE-induced structural variants (TE-SVs), providing unprecedented resolution in delineating recent TE activities. Our integrative analysis revealed a mutual evolutionary relationship between TEs and host genomes. On one hand, host genes and ncRNAs are involved in the transposition process, as evidenced by their colocalization and coactivation with TEs, and may play a role in chromatin regulation. On the other hand, TEs drive genetic innovation by promoting the duplication of host genes and inserting into regulatory regions. Moreover, genes influenced by active TEs are linked to plant growth, nutrient absorption, storage metabolism and environmental adaptation, aiding in crop domestication and adaptation. This TE dynamics atlas not only reveals evolutionary and functional features linked to transposition activity but also highlights the role of TEs in crop domestication and adaptation, paving the way for future exploration of TE-mediated genome evolution and crop improvement strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3e6dd3ee1f3baea529da4c3c4ab4bf7659f210" target='_blank'>
              Deciphering recent transposition patterns in plants through comparison of 811 genome assemblies.
              </a>
            </td>
          <td>
            Yan Huang, Sunil Kumar Sahu, Xin Liu
          </td>
          <td>2025-01-10</td>
          <td>Plant biotechnology journal</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and showed that STAG2 is uniquely tumor-suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility, and 3D genome conformation that impact the cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relate STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin-mediated mechanisms of lung tumor suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77c9e3bd051e20d8c03518cc31ea52a29e0f1f0" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis.
              </a>
            </td>
          <td>
            Emily L Ashkin, Yuning J Tang, Haiqing Xu, King L. Hung, J. Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-12-09</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Targeting DNA damage response (DDR) pathways represents one of the principal approaches in cancer therapy. However, defects in DDR mechanisms, exhibited by various tumors, can also promote tumor progression and resistance to therapy, negatively impacting patient survival. Therefore, identifying new molecules from natural extracts could provide a powerful source of novel compounds for cancer treatment strategies. In this context, we investigated the role of oleanolic acid (OA), identified in fermented Aglianico red grape pomace, in modulating the DDR in response to camptothecin (CPT), an inhibitor of topoisomerase I. Specifically, we found that OA can influence the choice of DNA repair pathway upon CPT treatment, shifting the repair process from homologous recombination gene conversion to single-strand annealing. Moreover, our data demonstrate that combining sub-lethal concentrations of OA with CPT enhances the efficacy of topoisomerase I inhibition compared to CPT alone. Overall, these findings highlight a new role for OA in the DDR, leading to a more mutagenic DNA repair pathway and increased sensitivity in the HeLa cancer cell line.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba28716fad27e1778281b97439ea3f957928277" target='_blank'>
              Oleanolic Acid Modulates DNA Damage Response to Camptothecin Increasing Cancer Cell Death
              </a>
            </td>
          <td>
            Giulio Mazzarotti, Maria Cuomo, Maria Carmen Ragosta, Andrea Russo, Margherita D’Angelo, Annamaria Medugno, Giuseppe Maria Napolitano, C. A. Iannuzzi, I. Forte, Rosa Camerlingo, Sharon Burk, F. Errichiello, Luigi Frusciante, M. Forino, M. R. Campitiello, M. de Laurentiis, A. Giordano, Luigi Alfano
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The human genome is packaged within a three-dimensional (3D) nucleus and organized into structural units known as compartments, topologically associating domains (TADs), and loops. TAD boundaries, separating adjacent TADs, have been found to be well conserved across mammalian species and more evolutionarily constrained than TADs themselves. Recent studies show that structural variants (SVs) can modify 3D genomes through the disruption of TADs, which play an essential role in insulating genes from outside regulatory elements' aberrant regulation. However, how SV affects the 3D genome structure and their association among different aspects of gene regulation and candidate cis-regulatory elements (cCREs) have rarely been studied systematically. Here, we assess the impact of SVs intersecting with TAD boundaries by developing an integrative Hi-C analysis pipeline, which enables the generation of an in-depth catalog of TADs and TAD boundaries in human lymphoblastoid cell lines (LCLs) to fill the gap of limited resources. Our catalog contains 18,865 TADs, including 4596 sub-TADs, with 185 SVs (TAD-SVs) that alter chromatin architecture. By leveraging the ENCODE registry of cCREs in humans, we determine that 34 of 185 TAD-SVs intersect with cCREs and observe significant enrichment of TAD-SVs within cCREs. This study provides a database of TADs and TAD-SVs in the human genome that will facilitate future investigations of the impact of SVs on chromatin structure and gene regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe7b730dd8aa8992f72d3009d5b16854b631e1b" target='_blank'>
              An integrative TAD catalog in lymphoblastoid cell lines discloses the functional impact of deletions and insertions in human genomes.
              </a>
            </td>
          <td>
            Chong Li, M. Bonder, Sabriya Syed, Matthew Jensen, Mark B. Gerstein, Michael C Zody, Mark J. P. Chaisson, M. Talkowski, T. Marschall, Jan O. Korbel, E. Eichler, Charles Lee, Xinghua Shi
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Gene conversion is a process in which genetic material from a donor sequence is unidirectionally copied to an acceptor sequence during the homologous recombination repair of a DNA double-strand break. Although gene conversion has been widely studied in the context of meiosis, hereditary diseases, and cancer development, gene conversion between parental homologs in the zygotes remains controversial. Here, we developed a method to detect interparental gene conversions by focusing on Mendelian errors and identified gene conversion events in one out of every 21.8 births. Some of these events were observed in genetic regions, potentially affecting offspring phenotypes. Interparental gene conversion leads to the offspring inheriting two identical alleles from one parent, resulting in a loss of heterozygosity. Our findings suggest that naturally occurring interparental gene conversions may provide a novel mechanism for the development of certain genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ddbca341c42c36a2e20339125b0cf1e5d09505" target='_blank'>
              Interparental Gene Conversion in General Population: A Novel Mechanism For Loss of Heterozygosity
              </a>
            </td>
          <td>
            Jumpei Toratani, Masahito Tachibana, Junichi Sugawara, Atsushi Sugawara, Takeki Sato, Yuri Takahashi, Hiroaki Hiraga, Emi Yokoyama, Zen Watanabe, Masatoshi Saito, Nobuo Yaegashi, Gen Tamiya, J. Takayama
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Bidirectional promoters (BDPs) regulate the transcription of two adjacent, oppositely oriented genes, offering a compact structure with significant potential for multigene expression systems. Although BDPs are evolutionarily conserved, their regulatory roles and sequence characteristics vary across species, with limited studies in fish. Here, we systematically analyzed the distribution, sequence features, and expression patterns of BDPs in the medaka (Oryzias latipes) genome. A total of 1737 divergent gene pairs, representing 13% of medaka genes, were identified as potentially regulated by BDPs. These genes are enriched in essential biological processes, including organelle function, RNA processing, and ribosome biogenesis. Transcriptomic analysis revealed that co-regulation (co-expression and co-silencing) is a prominent feature of these gene pairs, with variability influenced by tissue and sex. Sequence analysis showed that medaka BDPs are compact, with most fragments under 400 bp and an average GC content of 42.06%. Validation experiments confirmed the bidirectional transcriptional activity of three histone-related BDPs in both medaka SG3 cells and embryos, demonstrating effective and robust regulatory efficiency. This study enhances our understanding of the genomic organization and transcriptional regulation in fish and provides a valuable reference for developing species-specific multigene expression systems in fish genetic engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f319b14c5f2d24693577df92e252c6762c9458fc" target='_blank'>
              Genomic Analysis and Functional Validation of Bidirectional Promoters in Medaka (Oryzias latipes)
              </a>
            </td>
          <td>
            Jingjie Liang, Yan Huang, Jiangling Li, Ruoxue Chen, Yanlong Lin, Haiqing Li, Xiangrui Cao, Tiansheng Chen
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, Liyue Sun
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc8fc819ffccd10115ccb040d51408d1fd6a5f7" target='_blank'>
              Nanopore sequencing reveals that DNA replication compartmentalisation dictates genome stability and instability in Trypanosoma brucei
              </a>
            </td>
          <td>
            M. Krasiļņikova, Catarina A Marques, E. Briggs, C. Lapsley, Graham Hamilton, Dario Beraldi, K. Crouch, R. McCulloch
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Protein phosphatases are critical for regulating cell signaling, cell cycle, and cell fate decisions, and their dysregulation leads to an array of human diseases like cancer. The dual specificity phosphatases (DUSPs) have emerged as important factors driving tumorigenesis and cancer therapy resistance. DUSP12 is a poorly characterized atypical DUSP widely conserved throughout evolution. Although no direct substrate has been firmly established, DUSP12 that has been implicated in protecting cells from stress, regulating ribosomal biogenesis, and modulating cellular DNA content. In this study, we used affinity- and proximity-based biochemical purification approaches coupled to mass spectrometry to identify the zinc finger protein ZPR9 as a novel DUSP12 interactor, which was validated by in-cell and in-vitro IP assays. Interestingly, ZPR9 binds to the unique zinc-binding domain of DUSP12, which previous reports indicated was important for many of DUSP12’s functions within the cell. Prior studies had implicated ZPR9 as a modulator of apoptosis, but it remained unclear if and how ZPR9 participated in the cell cycle and, more so, how it promoted cell death. Using mass spectrometry analyses, we found that overexpression of DUSP12 promoted de-phosphorylation of ZPR9 at Ser143. Overexpression of ZPR9, but not Ser143 phosphomimetic and phosphorylation-deficient mutants, led to an increase in pre-metaphase mitotic defects while knockdown of DUSP12 also showed mitotic defects in metaphase. Furthermore, knockdown of DUSP12 promoted, while knockdown of ZPR9 suppressed, stress-induced apoptosis. Our results support a model where DUSP12 protects cells from stress-induced apoptosis by promoting de-phosphorylation of ZPR9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7479ca7476284506ba203b0cbc2b3084cb60b315" target='_blank'>
              DUSP12 promotes cell cycle progression and protects cells from cell death by regulating ZPR9
              </a>
            </td>
          <td>
            Mai Abdusamad, Xiao Guo, Ivan Ramirez, Erick F. Velasquez, Whitaker Cohn, Ankur A. Gholkar, Julian P Whitelegge, Jorge Z. Torres
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract H-DNA is an intramolecular DNA triplex formed by homopurine/homopyrimidine mirror repeats. Since its discovery, the field has advanced from characterizing the structure in vitro to discovering its existence and role in vivo. H-DNA interacts with cellular machinery in unique ways, stalling DNA and RNA polymerases and causing genome instability. The foundational S1 nuclease and chemical probing technologies originally used to show H-DNA formation have been updated and combined with genome-wide sequencing methods for large-scale mapping of secondary structures. There is evidence for triplex H-DNA’s role in polycystic kidney disease (PKD), cancer, and numerous repeat expansion diseases (REDs). In PKD, an H-DNA forming repeat region within the PKD1 gene stalls DNA replication and induces fragility. H-DNA-forming repeats in various genes have a role in cancer; the most well-studied examples involve H-DNA-mediated fragility causing translocations in multiple lymphomas. Lastly, H-DNA-forming repeats have been implicated in four REDs: Friedreich’s ataxia, GAA-FGF14-related ataxia, X-linked Dystonia Parkinsonism, and cerebellar ataxia, neuropathy and vestibular areflexia syndrome. In this review, we summarize H-DNA’s discovery and characterization, evidence for its existence and function in vivo, and the field’s current knowledge on its role in physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d258222649c5f8f00b18b553e105c9ceb00bafd" target='_blank'>
              Triplex H-DNA structure: the long and winding road from the discovery to its role in human disease
              </a>
            </td>
          <td>
            Julia A. Hisey, Chiara Masnovo, S. Mirkin
          </td>
          <td>2024-10-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Given the presence of highly repetitive genomic regions such as subtelomeric regions, understanding human genomic evolution remains challenging. Recently, long-read sequencing technology has facilitated the identification of complex genetic variants, including structural variants (SVs), at the single-nucleotide level. Here, we resolved SVs and their underlying DNA damage–repair mechanisms in subtelomeric regions, which are among the most uncharted genomic regions. We generated ∼20 × high-fidelity long-read sequencing data from three Korean individuals and their partially phased high-quality de novo genome assemblies (contig N50: 6.3–58.2 Mb). We identified 131 138 deletion and 121 461 insertion SVs, 41.6% of which were prevalent in the East Asian population. The commonality of the SVs identified among the Korean population was examined by short-read sequencing data from 103 Korean individuals, providing the first comprehensive SV set representing the population based on the long-read assemblies. Manual investigation of 19 large subtelomeric SVs (≥5 kb) and their associated repair signatures revealed the potential repair mechanisms leading to the formation of these SVs. Our study provides mechanistic insight into human telomere evolution and can facilitate our understanding of human SV formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/660e990dc5f20027c8956f073fbc6b43cc05d5aa" target='_blank'>
              Highly accurate Korean draft genomes reveal structural variation highlighting human telomere evolution
              </a>
            </td>
          <td>
            Jun Kim, Jong Lyul Park, Jin Ok Yang, Sangok Kim, Soobok Joe, Gunwoo Park, Taeyeon Hwang, Mun-Jeong Cho, Seung-Jae Lee, Jong-Eun Lee, Ji-Hwan Park, Min-Kyung Yeo, Seon-Young Kim
          </td>
          <td>2025-01-07</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067cac427326fdea9459d93348affd2bc0e1346c" target='_blank'>
              Nucleus-translocated GCLM promotes chemoresistance in colorectal cancer through a moonlighting function
              </a>
            </td>
          <td>
            Jin-Fei Lin, Ze-xian Liu, Dong-Liang Chen, , Fen Cao, Kai Yu, Ting Li, Hai-Yu Mo, Hui Sheng, Zhi-Bing Liang, Kun Liao, Yi Han, Shan-Shan Li, Zhao-Lei Zeng, Song Gao, Huai-qiang Ju, Rui-Hua Xu
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4623fb2b0df3467d1b2cf2d35e43a732c3aed069" target='_blank'>
              DNA targeting by compact Cas9d and its resurrected ancestor
              </a>
            </td>
          <td>
            Rodrigo Fregoso Ocampo, Jack P. K. Bravo, Tyler L. Dangerfield, Isabel Nocedal, Samatar Jirde, Lisa M. Alexander, Nicole C. Thomas, Anjali Das, Sarah Nielson, Kenneth A. Johnson, Christopher T. Brown, Cristina N. Butterfield, Daniela S. A. Goltsman, David W. Taylor
          </td>
          <td>2025-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary Histone methyltransferases (HMTs) and histone demethylases (HDMs) are enzymes that modify histones, the proteins around which DNA is wrapped. These modifications play a crucial role in regulating gene expression by adding or removing methyl groups, which can activate or silence genes. In this review, we explore the roles of HMTs and HDMs in the development of cancer and neurodegenerative diseases, specifically, Alzheimer’s, Parkinson’s, and Huntington’s diseases. Regarding cancer, HMTs and HDMs can drive tumor growth and progression by altering gene expression patterns that regulate cell proliferation and survival. For example, the dysregulation of enzymes like KMT2D (an HMT) and KDM2A (an HDM) is linked to various cancers including gastric cancer by influencing the chromatin structure and gene activity. In neurodegenerative diseases, these enzymes impact the health of neurons by modifying genes involved in brain function. In Alzheimer’s, Parkinson’s, and Huntington’s diseases, changes in HMT and HDM activity can lead to the dysregulation of genes critical for neuron survival, contributing to cognitive decline and motor dysfunction. Herein, we highlight the similarities and differences in how HMTs and HDMs function in cancer and neurodegeneration. Understanding these roles may reveal new therapeutic targets that address the epigenetic underpinnings of both cancer and neurodegenerative diseases, offering hope for innovative treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb434586ac7de7e0c22b593818a08f0a3dc6b5b6" target='_blank'>
              Epigenetic Modifiers: Exploring the Roles of Histone Methyltransferases and Demethylases in Cancer and Neurodegeneration
              </a>
            </td>
          <td>
            Lauren Reed, Janak Abraham, Shay Patel, Shilpa S. Dhar
          </td>
          <td>2024-12-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245a4598675a5f03862517e366728182b100458" target='_blank'>
              Molecular mechanism targeting condensin for chromosome condensation.
              </a>
            </td>
          <td>
            Menglu Wang, Daniel Robertson, Juan Zou, Christos Spanos, J. Rappsilber, A. Marston
          </td>
          <td>2024-12-17</td>
          <td>The EMBO journal</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="An ADP‐ribosylating enzyme called poly (ADP–ribose) polymerase 1 (PARP1) is necessary to start several types of DNA repair. PARP1 also contributes significantly to the regulation of gene expression through enzyme‐independent motif recognition and enzyme‐dependent chromatin remodeling. Thus, synthetic lethality is achieved in the therapy of tumors and cancers by blocking undesired DNA repair by the inhibition of its enzyme activity with small molecules. In addition to outlining the most recent research from 2019 to 2024 on the precise ways in which PARP1 regulates each process independently, we discussed how transcription and DNA repair are interdependent enough that perturbations caused by PARP1 enzymatic inhibition, disease‐related enzyme hyperactivation. Herein, this concept review systematically summarizes the different approach for designing selective PARP1 inhibitor that are i) PARP1–DNA trapping approach, ii) hybrid approach, iii) PROTAC approach, and iv) dual target inhibitor approach and updates on clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec80be41f9967e872ba7ea36021cfd284d0a854" target='_blank'>
              Recent Progress in Poly(ADP–Ribose) Polymerase‐1 Inhibitors for Application in Cancer
              </a>
            </td>
          <td>
            Anuradha Mehra, Shahnaz Islam, Rekha Sangwan
          </td>
          <td>2025-01-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA is frequently damaged by genotoxic stresses such as ionizing radiation, reactive oxygen species, and nitrogen species. DNA damage is a key contributor to cancer initiation and progression, and thus the precise and timely repair of these harmful lesions is required. Recent studies revealed transcription as a source of genome instability, and transcription-coupled DNA damage has been a focus in cancer research. Impaired mRNA export is closely related to DNA damage through R-loop formation. The molecular machineries of transcription-coupled DNA damage have been extensively analyzed in Saccharomyces cerevisiae. However, the molecular basis of these phenomena in higher eukaryotes remains elusive. In this review, we focus on the relationship between deregulated mRNA export through the transcription-export-2 (TREX-2) complex and cancer development. Particularly, the expression of germinal center-associated nuclear protein (GANP), a molecular scaffold in the TREX-2 complex, is highly associated with tumorigenesis in mice and humans. Although the deregulated expression of other components in the TREX-2 complex might affect cancer development, we have directly demonstrated the significance of GANP in tumorigenesis using genetically modified mice. Additionally, we describe recent evidence for medical applications demonstrating that the downregulation of the other components may be a good candidate for a chemotherapeutic target in terms of reducing the side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e429b3bb884f7cc3d479875de747738fafbdce" target='_blank'>
              Tumorigenesis Caused by Aberrant Expression of GANP, a Central Component in the Mammalian TREX-2 Complex—Lessons from Transcription-Coupled DNA Damages
              </a>
            </td>
          <td>
            Andri Rezano, Naomi Gondo, Yasuhiro Sakai, Yuko Nakamura, S. Phimsen, Tokio Tani, Akihiko Ito, Seiji Okada, Kazuhiko Kuwahara
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Polycomb repressive complex 2 (PRC2) is known to regulate gene expression and chromatin structure as it methylates H3K27, resulting in gene silencing. Studies have shown that PRC2 has dual functions in oncogenesis that allow it to function as both an oncogene and a tumor suppressor. Because of this, nuanced strategies are necessary to promote or inhibit PRC2 activity therapeutically. Given the therapeutic vulnerabilities and associated risks in oncological applications, a structured literature review on PRC2 was conducted to showcase similar cofactor competitor inhibitors of PRC2. Key inhibitors such as Tazemetostat, GSK126, Valemetostat, and UNC1999 have shown promise for clinical use within various studies. Tazemetostat and GSK126 are both highly selective for wild-type and lymphoma-associated EZH2 mutants. Valemetostat and UNC1999 have shown promise as orally bioavailable and SAM-competitive inhibitors of both EZH1 and EZH2, giving them greater efficacy against potential drug resistance. The development of other PRC2 inhibitors, particularly inhibitors targeting the EED or SUZ12 subunit, is also being explored with the development of drugs like EED 226. This review aims to bridge gaps in the current literature and provide a unified perspective on promising PRC2 inhibitors as therapeutic agents in the treatment of lymphomas and solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d39c17f73d3909cec9400437d3632effbc2233b4" target='_blank'>
              Pharmacological Advancements of PRC2 in Cancer Therapy: A Narrative Review
              </a>
            </td>
          <td>
            Michael S. Wang, Jonathan H Sussman, Jessica A. Xu, Reema Patel, Omar Elghawy, Prashanth Rawla
          </td>
          <td>2024-12-01</td>
          <td>Life</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated oncogenes in cancer and is involved in various tumorigenic processes. Among its diverse roles, KRAS mutations have been shown to upregulate macropinocytosis, a form of endocytosis that allows cells to engulf extracellular fluid and its contained solutes, subsequently supporting the increased need for cancer cells' growth and proliferation. The nutrient uptake function of macropinocytosis was initially described within the framework of KRAS-driven pancreatic tumors. Although constitutive macropinocytosis can be induced by activating mutations of proteins that are commonly found in cancer, KRAS-induced macropinocytosis constitutes a fundamental area of research due to the high occurrence of mutated KRAS in cancer (~1/3 of all cancers).  Furthermore, KRAS-mutated macropinocytosis not only contributes to tumorigenesis but also plays a critical role in developing resistance to treatments, as it was found to be implicated in Multi-Drug Resistance (MDR) in cancer cells. This mini-review aims to synthesize current knowledge of mechanisms of KRAS-mutated macropinocytosis briefly and examines the relationship between KRAS and macropinocytosis in the light of its role in cancer progression and drug resistance, highlighting therapeutic implications, targeting potential vulnerabilities and outlining clinical advancements in pertinent therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4b47f4761d206b1edf858bd1fc0d092897567" target='_blank'>
              The Role of KRAS in Macropinocytosis and its Implications of Multi-Drug Resistance in Cancer
              </a>
            </td>
          <td>
            F. Fenyvesi, Zeinab Ibrahim
          </td>
          <td>2024-12-23</td>
          <td>De Remediis</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Telomere maintenance is crucial for preventing the linear eukaryotic chromosome ends from being mistaken for DNA double-strand breaks, thereby avoiding chromosome fusions and the loss of genetic material. Unlike most eukaryotes that use telomerase for telomere maintenance, Drosophila relies on retrotransposable elements-specifically HeT-A, TAHRE, and TART (collectively referred to as HTT)-which are regulated and precisely targeted to chromosome ends. Drosophila telomere protection is mediated by a set of fast-evolving proteins, termed terminin, which bind to chromosome termini without sequence specificity, balancing DNA damage response factors to avoid erroneous repair mechanisms. This unique telomere capping mechanism highlights an alternative evolutionary strategy to compensate for telomerase loss. The modulation of recombination and transcription at Drosophila telomeres offers insights into the diverse mechanisms of telomere maintenance. Recent studies at the population level have begun to reveal the architecture of telomere arrays, the diversity among the HTT subfamilies, and their relative frequencies, aiming to understand whether and how these elements have evolved to reach an equilibrium with the host and to resolve genetic conflicts. Further studies may shed light on the complex relationships between telomere transcription, recombination, and maintenance, underscoring the adaptive plasticity of telomeric complexes across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc71fe25065e2464b3633178f88c0066a196f35c" target='_blank'>
              Maintaining Telomeres without Telomerase in Drosophila: Novel Mechanisms and Rapid Evolution to Save a Genus.
              </a>
            </td>
          <td>
            S. Cacchione, G. Cenci, Anne-Marie Dion-Côté, D. Barbash, G. Raffa
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87208750323d073805f3062a30f76b4849f02905" target='_blank'>
              Unveiling cell-type-specific microRNA networks through alternative polyadenylation in glioblastoma
              </a>
            </td>
          <td>
            Mert Cihan, Greta Schmauck, Maximilian Sprang, Miguel Andrade
          </td>
          <td>2025-01-21</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) regulates gene expression through two primary mechanisms: as a growth factor in the nucleus, where it translocates upon binding its ligand, or via its intrinsic tyrosine kinase activity in the cytosol, where it modulates key signaling pathways such as RAS/MYC, PI3K, PLCγ, and STAT3. During tumorigenesis, these pathways become deregulated, leading to uncontrolled proliferation, enhanced migratory and metastatic capabilities, evasion of programmed cell death, and resistance to chemotherapy or radiotherapy. The RAS and MYC oncogenes are pivotal in tumorigenesis, driving processes such as resistance to apoptosis, replicative immortality, cellular invasion and metastasis, and metabolic reprogramming. These oncogenes are subject to regulation by a range of epigenetic and post-transcriptional modifications. This review focuses on the deregulation of EGFR, RAS, and MYC expression caused by (epi)genetic alterations and post-translational modifications. It also explores the therapeutic potential of targeting these regulatory proteins, emphasizing the importance of phenotyping neoplastic tissues to inform the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6048dcc607aa9e108befb3b09588362de65007" target='_blank'>
              Dynamic Multilevel Regulation of EGFR, KRAS, and MYC Oncogenes: Driving Cancer Cell Proliferation Through (Epi)Genetic and Post-Transcriptional/Translational Pathways.
              </a>
            </td>
          <td>
            M. Šereš, Katarina Spacayova, Z. Sulová, Jana Špaldová, Albert Breier, L. Pavlikova
          </td>
          <td>2025-01-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant global health challenge, underscoring the need for innovative therapeutic strategies. Oncogenic miRNAs (oncomiRs) play a significant biological role in the initiation and progression of colorectal cancer. Inspired by the cooperative mechanisms of plant nanovirus, which employ multiple circular single-stranded DNA (CssDNA) genomes, it is hypothesized that the development and delivery of CssDNA to target oncomiRs would achieve therapeutic benefits in CRC. In this study, a multi-omics approach is utilized to identify key tumor suppressor genes (TSGs) and their related oncomiRs implicated in CRC, followed by the development of CssDNA, each of which is loaded with multiple miRNA binding sites targeting one oncomiR. When transfected into the cells, these CssDNA can effectively target and sequester the corresponding oncomiRs to restore the expression of TSGs, leading to a marked reduction in CRC development both in vitro and in vivo. The findings highlight the therapeutic potential of nanovirus-inspired CssDNA in modulating the miRNA-mediated regulatory network in CRC. This study lays the groundwork for the development of non-coding DNA-based therapies with broad implications for the treatment of colorectal cancer and potentially other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a389d3e4c09149bfce4cb4ae8bf6fdc4a619a01" target='_blank'>
              Circular Single-Stranded DNA-Based Artificial Nanoviruses Mitigate Colorectal Cancer Development.
              </a>
            </td>
          <td>
            Jinghao Wang, Pengfei Zhang, Yonglian Huang, Guang Hu, Kexuan Zou, Songtao Zhou, Dandan Shao, Jianming Wang, Jie Song
          </td>
          <td>2025-01-21</td>
          <td>Small methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In many eukaryotes, meiotic recombination occurs preferentially at discrete sites, called recombination hotspots. In various lineages, recombination hotspots are located in regions with promoter-like features and are evolutionarily stable. Conversely, in some mammals, hotspots are driven by PRDM9 that targets recombination away from promoters. Paradoxically, PRDM9 induces the self-destruction of its targets and this triggers an ultra-fast evolution of mammalian hotspots. PRDM9 is ancestral to all animals, suggesting a critical importance for the meiotic program, but has been lost in many lineages with surprisingly little effect on meiosis success. However, it is unclear whether the function of PRDM9 described in mammals is shared by other species. To investigate this, we analyzed the recombination landscape of several salmonids, the genome of which harbors one full-length PRDM9 and several truncated paralogs. We identified recombination initiation sites in Oncorhynchus mykiss by mapping meiotic DNA double-strand breaks (DSBs). We found that DSBs clustered at hotspots positioned away from promoters, enriched for the H3K4me3 and H3K36me3 and the location of which depended on the genotype of full-length Prdm9. We observed a high level of polymorphism in the zinc finger domain of full-length Prdm9, indicating diversification driven by positive selection. Moreover, population-scaled recombination maps in O. mykiss, Oncorhynchus kisutch and Salmo salar revealed a rapid turnover of recombination hotspots caused by PRDM9 target motif erosion. Our results imply that PRDM9 function is conserved across vertebrates and that the peculiar evolutionary runaway caused by PRDM9 has been active for several hundred million years.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f09c603402d3b52a99c356601f8730c3715c9b0" target='_blank'>
              PRDM9 drives the location and rapid evolution of recombination hotspots in salmonid fish
              </a>
            </td>
          <td>
            Marie-Laure Raynaud, Paola Sanna, Julien Joseph, J. Clément, Yukiko Imai, J. Lareyre, Audrey Laurent, Nicolas Galtier, F. Baudat, Laurent Duret, P. Gagnaire, B. de Massy
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Background Gene methylation in cells is an important factor in tumorigenesis, and radiotherapy can change DNA methylation in cells. In this study, complete genome methylation sequencing (BS-Seq) technology was used to analyze the genome-wide methylation of patients with cervical cancer before and after radiotherapy. Methods Three pairs of cervical squamous cell carcinoma samples were collected from patients before and after radiotherapy in July 2020. Genome-wide DNA methylation profiles were generated using WGBS. Bioinformatics analysis was conducted to identify differential methylation regions (DMRs) and their associated genes and pathways. The study focused on the methylation changes of LHX2, LHX5, and LHX9 genes, assessing their expression levels using qRT-PCR and correlating these changes with cervical cancer stages. Results MCG was the main way of genomic DNA methylation in the three patients. The DNA methylation level and methylation density on each chromosome varied greatly. As revealed by comparison of methylation before and after radiation in the three patients, 1287, 1261 and 789 differential methylation genes were identified, respectively. 3) Combined with clinical treatment, methylation level difference and correlation enrichment analysis, it was found that LHX2, LHX5 and LHX9 were closely related to the occurrence and development of cervical cancer. After 5-Aza-DC and radiotherapy, the methylation of the CpG islands in LHX2, LHX5 and LHX9 genes in these patients was decreased (p < 0.01), and the mRNA and protein expression levels were relatively increased (p < 0.01). Conclusion In our present work, genome-wide DNA methylation maps of cervical cancer tissues before and after radiotherapy were successfully constructed. We found that LHX5 and LHX9 genes are closely related to cervical cancer. LHX5 and LHX9 have a negative effect on cervical cancer. The migration ability of LHX9 silenced cells was significantly enhanced after irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fea2269d56f3d3d8896922414dbb511f2559dfa8" target='_blank'>
              Whole-Genome Methylation Sequencing Analysis and Functional Verification of LIM-Homeobox Family Genes in Cervical Cancer
              </a>
            </td>
          <td>
            Rong Yu, Qin Yu, Jie Shi, Xue Meng, Zhiyuan Deng, Jing Suo, Hao Yang
          </td>
          <td>2025-01-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Introduction Cancer’s inherent heterogeneity, marked by diverse genetic and molecular alterations, presents significant challenges for developing effective treatments. One such alteration is the regulation of disulfidoptosis, a recently discovered programmed cell death pathway. RPN1, a key regulator associated with disulfidoptosis, may influence various aspects of tumor biology, including immune evasion and cellular senescence. This study aims to dissect the role of RPN1 in pan-cancer and its potential as a therapeutic target. Methods We employed a pan-cancer analysis to explore RPN1 expression and its association with clinical outcomes across multiple tumor types. Immune cell infiltration and expression of immune checkpoint genes were analyzed in relation to RPN1. Additionally, cellular senescence markers were assessed in RPN1 knockdown tumor cells. Gene regulatory mechanisms were studied through gene copy number variations, DNA methylation analysis, and transcriptional regulation by SP1. Results RPN1 is overexpressed in a wide range of tumor types and correlates with poor clinical outcomes, including overall survival, disease-specific survival, and progression-free intervals. Our analysis shows that RPN1 is involved in immune evasion, correlating with the presence of myeloid dendritic cells, macrophages, and tumor-associated fibroblasts, and influencing T-cell activity. RPN1 knockdown led to reduced tumor cell proliferation and induced cellular senescence, marked by increased senescence-associated biomarkers and β-galactosidase activity. RPN1 expression was found to be regulated by gene copy number variations, reduced DNA methylation, and transcriptional control via SP1. Discussion These findings highlight RPN1 as a key pan-cancer regulator, influencing immune microenvironment interactions and cellular senescence. The regulation of disulfidoptosis by RPN1 presents a promising avenue for therapeutic intervention. Targeting RPN1 could enhance immunotherapy efficacy and help mitigate tumor progression, offering a potential strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2952a313a9a344ecc3d40275b633ebc51cd3bfe9" target='_blank'>
              Dissecting the functions and regulatory mechanisms of disulfidoptosis-related RPN1 in pan-cancer: modulation of immune microenvironment and cellular senescence
              </a>
            </td>
          <td>
            Lexin Qin, Tingting Liang, Hailong Zhang, Xian Gong, Meidan Wei, Xiangrong Song, Yaoyu Hu, Xinyu Zhu, Wentao Hu, Jianxiang Li, Jin Wang
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Human papillomaviruses (HPV) from the genus beta have been implicated in the development of cutaneous squamous cell cancer in epidermodysplasia verruciformis and organ transplant patients. In contrast to alpha-high-risk HPV, which cause ano-genital and oropharyngeal cancers, beta-HPV replication is not well understood. The beta-HPV49 transcriptome was analyzed by RNA sequencing using stable keratinocyte cell lines maintaining high levels of extrachromosomally replicating E8- genomes, which can be established due to a lack of the viral E8^E2 repressor protein. This analysis indicated the presence of four transcription start sites, two polyadenylation signals, and splice donor (SD) and acceptor sites consistent with the conserved gene expression patterns of animal and human PV. Surprisingly, a novel SD in the E6 oncogene (SD217) was identified resembling the SD in E6 of carcinogenic alpha-HPV. Mutation of SD217 enhanced E6 protein expression but had no influence on the growth of keratinocytes transduced with retroviral HPV49 E6 and E7 expression vectors. Inactivation of SD217 in the context of the HPV49 wild-type genome did not enable immortalization and prevented immortalization in the context of the E8- genome. The analysis of SD217 mutant genomes revealed a strong down-regulation of SD217 usage, but only weak effects on other viral transcripts. This suggests that SD217 does not contribute to immortalization by modulating viral gene expression. Usage of SD217 is increased in immortalized E8- cell lines compared with transiently transfected cells, which may indicate that long-term extrachromosomal maintenance requires reduced E6 protein levels.IMPORTANCEHigh-risk (hr) human papillomaviruses (HPV) from the genus alpha cause ano-genital and oropharyngeal cancers, whereas beta-HPV have been implicated to cause skin cancer in epidermodysplasia verruciformis and organ transplant patients. In contrast to alpha hr-HPV, the replication cycle of beta-HPV is not very well understood. Transcriptional profiling of beta-HPV49 by RNA sequencing reveals transcription start sites and splice sites conserved among HPV. Surprisingly, a splice donor site in the E6 oncogene (SD217), previously only described for hr-HPV, was identified that controls E6 oncoprotein levels and is required for immortalization of keratinocytes by the HPV49 genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b466540ee80d69ec5d5848e7f47b4a1492c73735" target='_blank'>
              A splice donor in E6 influences keratinocyte immortalization by beta-HPV49.
              </a>
            </td>
          <td>
            Tina M. Rehm, T. Iftner, F. Stubenrauch
          </td>
          <td>2025-01-22</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="BACKGROUND
 Although transmembrane protein 106C (TMEM106C) has been elucidated to be overexpressed in cancers, its underlying mechanisms have not yet been fully understood.
 AIM
 To investigate the expression levels and molecular mechanisms of TMEM106C across 34 different cancer types, including liver hepatocellular carcinoma (LIHC).
 METHODS
 We analyzed TMEM106C expression patterns in pan-cancers using microenvironment cell populations counter to evaluate its association with the tumor microenvironment. Gene set enrichment analysis was conducted to identify molecular pathways related to TMEM106C. Chromatin immunoprecipitation followed by sequencing (ChIP-seq) analysis was conducted to identify upstream transcriptional regulators of TMEM106C. In LIHC, we examined mRNA profiles, performed in-house quantitative polymerase chain reaction, immunohistochemistry, and constructed a co-expression gene network. Functional assays, including cell counting kit-8, cell cycle, apoptosis, migration, and invasion, were conducted. The effect of nitidine chloride (NC) on LIHC xenograft was evaluated through RNA sequencing and molecular docking. Finally, potential therapeutic agents targeting TMEM106C were predicted.
 RESULTS
 TMEM106C was significantly overexpressed in 27 different cancer types and presaged poor prognosis in four of these types, including LIHC. Across pan-cancers, TMEM106C was inversely correlated to the abundances of immune and stromal cells. Furthermore, TMEM106C was significantly linked to cell cycle and DNA replication pathways in pan-cancers. ChIP-seq analysis predicted CCCTC-binding factor as a pivotal transcriptional factor targeting the TMEM106C gene in pan-cancers. Integrated analysis showed that TMEM106C was upregulated in 4657 LIHC compared with 3652 normal liver tissue [combined standardized mean difference = 1.31 (1.09, 1.52)]. In-house LIHC samples verified the expression status of TMEM106C. Higher TMEM106C expression signified worse survival conditions in LIHC patients treated with sorafenib, a tyrosine kinase inhibitor (TKI). Co-expressed analysis revealed that TMEM106C were significantly enriched in the cell cycle pathway. Knockout experiments demonstrated that TMEM106C plays a crucial role in LIHC cell proliferation, migration, and invasion, with cell cycle arrest occurring at the DNA synthesis phase, and increased apoptosis. Notably, TMEM106C upregulation was attenuated by NC treatment. Finally, TMEM106C expression levels were significantly correlated with the drug sensitivity of anti-hepatocellular carcinoma agents, including JNJ-42756493, a TKI agent.
 CONCLUSION
 Overexpressed TMEM106C was predicted as an oncogene in pan-cancers, which may serve as a promising therapeutic target for various cancers, including LIHC. Targeting TMEM106C could potentially offer a novel direction in overcoming TKI resistance specifically in LIHC. Future research directions include in-depth experimental validation and exploration of TMEM106C’s role in other cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b72383ec9aa12d705a14afde59b61b7e89e0cf78" target='_blank'>
              Unveiling expression patterns, mechanisms, and therapeutic opportunities of transmembrane protein 106C: From pan-cancers to hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jian-Di Li, Rong-Quan He, Y. Dang, Zhi-guang Huang, Dan-dan Xiong, Lu Zhang, Xiu-Fang Du, Gang Chen
          </td>
          <td>2025-02-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Mitochondrial mRNA modifications are suggested to play a role in fine-tuning mitochondrial gene expression and function. However, the epitranscriptomic landscape of mitochondrial mRNA (mt-mRNA) remains poorly explored. Here, we uncover N3-methylcytosine (m3C) as a novel mt-mRNA that is catalyzed by METTL8, an enzyme previously known to modify mt-tRNA. Transcriptome-wide mapping reveals that METTL8-dependent m3C is enriched in mt-mRNAs encoding complex I subunits of the respiratory chain. Additionally, METTL8 is highly expressed in various cancers, notably in cervical cancer. METTL8 depletion impairs cell migration in vitro and reduces tumor growth in mouse xenografts. Finally, transcriptomic analyses further link METTL8 expression to oncogenic pathways, mitochondrial functions and complex I activity. Together, our results reveal a novel mitochondrial mRNA modification that promotes cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1ff335d47d70a5bda6740843d412195cf845e67" target='_blank'>
              m3C is a mitochondrial mRNA modification which promotes tumor progression
              </a>
            </td>
          <td>
            Audrey Penning, Kun Wang, Margot Célérier, É. Collignon, Irina Primac, Louis Van der Linden, M. Bizet, E. Calonne, Bouchra Hassabi, Céline Hubert, Pascale Putmans, Frédéric Murisier, L. Larbanoix, Younes Achouri, Jie Lan, Ivan Nemazanyy, Sophie Laurent, Patrick Jacquemin, Chengqi Yi, Rachel Deplus, François Fuks
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Anelloviruses are a group of small, circular, single-stranded DNA viruses that are found ubiquitously across mammalian hosts. Here, we explored a large number of publicly available human microbiome datasets and retrieved a total of 829 anellovirus genomes, substantially expanding the known diversity of these viruses. The majority of new genomes fall within the three major human anellovirus genera: Alphatorquevirus, Betatorquevirus, and Gammatorquevirus, while we also present new genomes of the under-sampled Hetorquevirus, Memtorquevirus, and Samektorquevirus genera. We performed recombination analysis and show evidence of extensive recombination across all human anelloviruses. Interestingly, more than 95% of the detected events are between members of the same genus and only 15 inter-genus recombination events were detected. The breakpoints of recombination cluster in hotspots at the ends and outside of the ORF1 gene, while a recombination coldspot was detected within the gene. Our analysis suggests that anellovirus evolution is governed by homologous recombination; however, events between distant viruses or ones producing chimaeric ORF1s likely lead to nonviable recombinants. The large number of genomes further allowed us to examine how essential genomic features vary across anelloviruses. These include functional domains in the ORF1 protein and the nucleotide motif of the replication loop region, required for the viruses’ rolling-circle replication. A subset of the genomes assembled in both this and previous studies are completely lacking these essential elements, opening up the possibility that anellovirus intracellular populations contain nonstandard viral genomes. However, low-read depth of the metagenomically assembled contigs may partly explain the lack of some features. Overall, our study highlights key features of anellovirus genomics and evolution, a largely understudied group of viruses whose potential in virus-based therapeutics is recently being explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773b8f1792eaa5d1aac5fe47e3d01f3b2b675543" target='_blank'>
              Expanding the genomic diversity of human anelloviruses
              </a>
            </td>
          <td>
            S. Modha, J. Hughes, R. Orton, Spyros Lytras
          </td>
          <td>2025-01-07</td>
          <td>Virus Evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Background Chromosomal instability (CIN), a hallmark of cancer, is commonly linked to poor prognosis in high‐grade prostate cancer (PCa). Paradoxically, excessively high levels of CIN may impair cancer cell viability. Consequently, understanding how tumours adapt to CIN is critical for identifying novel therapeutic targets. Methods Bioinformatic analyses were conducted to identify genes overexpressed in PCa tissues using The Cancer Genome Atlas (TCGA) and GEO datasets. Western blotting and immunohistochemistry assays were applied to determine the expression levels of euchromatic histone lysine methyltransferase 2 (EHMT2), pT232‐Aurora B and Cullin 3 (CUL3). The proliferation of cells was measured through CCK‐8 tests, clonogenesis and subcutaneous xenografts of human PCa cells in BALB/c nude mice. Live cell imaging, immunofluorescence (IF) and flow cytometry were used to confirm the role of EHMT2 in PCa cell mitosis. Co‐immunoprecipitation, Western blotting and IF assays further elucidated the underlying molecular mechanisms. Results EHMT2 was highly expressed in metastatic PCa tissues exhibiting elevated CIN and was strongly associated with adverse clinical outcomes in patients with PCa. Silencing EHMT2 impaired cell division, inducing G2/M‐phase arrest and mitotic catastrophe in PCa cells. Mechanistically, EHMT2 is indispensable to ensure the full activation of Aurora B through centromeric R‐loop‐driven ATR–CHK1 pathway, with EHMT2 protein expression peaking during the G2/M‐phase. Moreover, CUL3 was identified as a binding partner of EHMT2, mediating its polyubiquitination and destabilising its protein levels. Conclusions This study reveals a CUL3–EHMT2–Aurora B regulatory axis that safeguards accurate chromosome segregation in PCa cells, supporting the potential therapeutic application of EHMT2 inhibitors. Key points Euchromatic histone lysine methyltransferase 2 (EHMT2) is overexpressed in advanced prostate cancer, restraining catastrophic chromosomal instability (CIN) and enhancing cell fitness. EHMT2 functions via the centromeric R‐loop‐driven ATR–CHK1–Aurora B pathway to promote chromosomal stability. EHMT2 confers enzalutamide resistance via activating Aurora B. Cullin 3 (CUL3) promotes EHMT2 destabilisation via deubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc5ece350ce05d9ec1131335e290684b93468c8" target='_blank'>
              EHMT2‐mediated R‐loop formation promotes the malignant progression of prostate cancer via activating Aurora B
              </a>
            </td>
          <td>
            Yuyang Zhang, Mingqin Su, Yiming Chen, Li Cui, Wei Xia, Renfang Xu, Dong Xue, Xiansheng Zhang, Xingliang Feng
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ebfde7a932000ba8510eefbb0bb716c35e7fa" target='_blank'>
              The landscape of fusion transcripts in plants: a new insight into genome complexity
              </a>
            </td>
          <td>
            P. Chitkara, Ajeet Singh, Rashmi Gangwar, Rohan Bhardwaj, Shafaque Zahra, Simran Arora, Fiza Hamid, Ajay Arya, Namrata Sahu, Srija Chakraborty, Madhulika Ramesh, Shailesh Kumar
          </td>
          <td>2024-12-04</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22203f23dc18f7e477336e6b95f3919f3954fe27" target='_blank'>
              Mechanisms and therapeutic implications of gene expression regulation by circRNA-protein interactions in cancer
              </a>
            </td>
          <td>
            Nan Zhang, Xinjia Wang, Yu Li, Yiwei Lu, Chengcheng Sheng, Yumeng Sun, Ningye Ma, Yisheng Jiao
          </td>
          <td>2025-01-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell cycle checkpoints are stringent quality control mechanisms that regulate cell cycle progression and division. Cancer cells often develop a dependency on the G2/M cell cycle checkpoint to facilitate DNA repair and resolve intrinsic or therapy-induced DNA damage. This dependency leads to therapy resistance, continuous cell division, and disease progression. Targeting G2/M checkpoints has been heavily pursued over the past two decades and has progressed into clinical studies. Recent genome-scale functional genomic studies have revealed that Myt1 kinase, an essential but previously overlooked molecule for the G2/M checkpoint, is a promising target for multiple types of cancers. In this work, we summarize the latest discoveries in molecular targeting of Myt1 kinase and discuss the challenges and limitations in expanding its clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17deb173d71c056d6283f804746aaf46a913ddbb" target='_blank'>
              Myt1 kinase: An Emerging Cell Cycle Regulator for Cancer Therapeutics.
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Javeria Zaheer, Diego Luis Ribeiro, Chunzhang Yang
          </td>
          <td>2025-01-16</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b0e375dc93cf14802320945eec674fcdd4837e" target='_blank'>
              Identification of MORF4L1 as an endogenous substrate of CRBN and its potential role as a therapeutic target in cancer
              </a>
            </td>
          <td>
            Luyao Zhang, Shunfang Liu, Jingliang He, Zhongke Hu, Lan Zhu, Hongyi Huang, Qi Gao, Dan Wang, Lu Chen, Xingyu Zhang, Ruotong Liu, Jiayun Wang, Yizhuo Song, Kaile Zeng, Xiuming Li, Yulu Chen, Xun Zou, Shaojie Ma, Xiujun Wang, Guofeng Xu, Wei Liu, Bin Liu
          </td>
          <td>2025-01-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is among the most aggressive and lethal human tumors. Many functional studies have demonstrated the role of non-coding RNAs (ncRNA), particularly microRNAs (miRNA), in the regulation of hepatocarcinogenesis driving pathways. MiR-125a-5p (miR-125a) has been consistently reported as an oncosuppressive miRNA, as demonstrated in vivo and in vitro. However, its HCC relevant targets and molecular mechanisms are still largely unknown. Here, a genome-wide perspective of the whole miR-125a targetome has been achieved. In particular, two different HCC cell lines were subjected to a miRNA boosting by mimic transfections, and consequently many genes were de-regulated, as observed by a transcriptomic approach. The merging of down-regulated genes with results from bioinformatic predictive tools yielded a number of candidate direct targets that were further experimentally validated by luciferase-based reporter assays. Different novel targets were found, in particular ARID3A, CCNJ, LIPA, NR6A1, and NUP210, oncogenes in various tumors and here also related to HCC through miR-125a regulation. The RNA interactions investigated in this work could pave the way to piece together the RNA regulatory networks governed by the miRNA impacting on hepatocarcinogenesis, and be exploited in the future for identifying novel biomarkers and therapeutic targets in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62eb68119553b65d1027aab3b456532606e32c8" target='_blank'>
              Transcriptomic-Based Identification of miR-125a Novel Targets in Human Hepatocarcinoma Cells.
              </a>
            </td>
          <td>
            Ilenia De Leo, Nicola Mosca, Mariaceleste Pezzullo, Danila Valletta, Francesco Manfrevola, Vincenza Grazia Mele, Rosanna Chianese, A. Russo, N. Potenza
          </td>
          <td>2025-01-18</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf2e1956ceee35cd1cab8cf25a742bffadc692d" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zong-xiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Identifying the roles of genes in cancer is critical in discovering potential genetic therapies for cancer care. Translocon-associated protein delta (TRAPδ), also known as signal sequence receptor 4 (SSR4), is a constituent unit in the TRAP/SSR complex that resides in the endoplasmic reticulum and plays a key role in transporting newly synthesized proteins into the endoplasmic reticulumn. However, its biological role in disease development remains unknown to date. This is the first study to identify the role of TRAPδ/SSR4 in colorectal cancer cells in vitro. Upon successful transient knockdown of TRAPδ/SSR4, we observed significant reduction of cell viability in all colorectal cancer cell lines tested. Both HCT 116 and SW480 cell lines were significantly arrested at S and G1 phases, while DLD-1 cells were significantly apoptotic. Moreover, TRAPδ/SSR4 stable knockdown HCT 116 and SW480 cells showed significantly lower viability, anchorage-independent growth, and increased S and G1 phase arrests. Overall, we conclude TRAPδ/SSR4 is a potential oncogene in human colorectal cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c566aa31384f1776417a4c1880816c68b25b7d5" target='_blank'>
              Identification of Translocon-associated Protein Delta as An Oncogene in Human Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Darshika Amarakoon, Wu-Joo Lee, Jing Peng, Seong-Ho Lee
          </td>
          <td>2024-12-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract As a chromatin remodelling factor, high mobility group A1 (HMGA1) plays various roles in both physiological and pathological conditions. However, its role in DNA damage response and DNA damage‐based chemotherapy remains largely unexplored. In this study, we report the poly ADP‐ribosylation (PARylation) of HMGA1 during DNA damage, leading to desensitization of esophageal squamous cell carcinoma (ESCC) cells to the poly(ADP‐ribose) polymerase 1 (PARP1) inhibitor, olaparib. We found that HMGA1 accumulates at sites of DNA damage, where it interacts with PARP1 and undergoes PARylation at residues E47 and E50 in its conserved AT‐hook domain. This modification enhances the accumulation of Ku70/Ku80 at the site of DNA damage and activates the DNA‐dependent protein kinase catalytic subunit, facilitating nonhomologous end‐joining repair. In both subcutaneous tumour models and genetically engineered mouse models of in situ esophageal cancer, HMGA1 interference increased tumour sensitivity to olaparib. Moreover, HMGA1 was highly expressed in ESCC tissues and positively correlated with PARP1 levels as well as poor prognosis in ESCC patients. Taken together, these findings reveal a mechanistic link between HMGA1 and PARP1 in regulating cell responses to DNA damage and suggest that targeting HMGA1 could be a promising strategy to increase cancer cell sensitivity to olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/419174c65c94af6161e05b00b1d3b8c1060c25e0" target='_blank'>
              PARylation of HMGA1 desensitizes esophageal squamous cell carcinoma to olaparib
              </a>
            </td>
          <td>
            Xin-Yuan Lei, Kai-Yue He, Qiu-Tong Li, Lei Zhang, Dan-Hui Wu, Jing-Yu Yang, Jin-Rong Guo, Meng-Jie Liu, Zi-Long Zhao, Jun-Qi Li, Huai Liu, Yuan Zhao, Yu-Jia Li, Qian-Hui Sun, Chen-Guang Wu, Yun-Fan Wang, Geng-Sheng Cao, Gang Wang, Yong-Ping Jian, Z. Xu
          </td>
          <td>2024-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma in children, but the outcomes of high-grade RMS patients remain poor, underscoring the critical need for novel therapeutic strategies. Although metabolic pathways in RMS are incompletely characterized, emerging evidence suggests that metabolic adaptations in RMS resemble those in other malignancies. Here, we identify elevated cholesterol biosynthesis driven by the PROX1 transcription factor as a defining feature of RMS. Our findings demonstrate that the cholesterol biosynthesis pathway is essential for RMS cell growth, proliferation, and survival. Blocking this pathway through genetic or pharmacological inhibition of the key cholesterol biosynthesis enzymes significantly impairs RMS cell proliferation, halts cell cycle progression, and triggers apoptosis through activation of endoplasmic reticulum stress pathways. We furthermore validate the critical role of cholesterol biosynthesis in RMS progression in tumor xenograft models, demonstrating that silencing of the DHCR7 gene significantly suppresses tumor growth. Transcriptomic analysis revealed widespread downregulation of cell cycle-related genes following DHCR7 silencing, further supporting the role of cholesterol metabolism in cell cycle regulation. These results highlight the vulnerability of RMS cells to cholesterol biosynthesis inhibition and suggest that targeting this metabolic pathway as a promising therapeutic approach for improving RMS outcomes. Our findings provide a rationale for the development of novel therapies targeted to cholesterol biosynthesis in this aggressive cancer. Significance This study reveals that targeting cholesterol biosynthesis in rhabdomyosarcoma induces ER-stress, apoptosis and cell cycle arrest, highlighting a potential therapeutic strategy for treating this aggressive pediatric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1168b7a6afff81a3b6df61f48c435d90b22e7cf3" target='_blank'>
              Targeting de novo cholesterol synthesis in rhabdomyosarcoma induces cell cycle arrest and triggers apoptosis through ER stress-mediated pathways
              </a>
            </td>
          <td>
            N. Y. Gizaw, Kalle Kolari, Kari Alitalo, R. Kivelä
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Animal genomes are packaged into chromatin, a highly dynamic macromolecular structure of DNA and histone proteins organised into nucleosomes. This accommodates packaging of lengthy genomic sequences within the physical confines of the nucleus while also enabling precise regulation of access to genetic information. However, histones existed before chromatin and have lesser-known functions beyond genome regulation. Most notably, histones are potent antimicrobial agents, and the release of chromatin to the extracellular space is a defence mechanism nearly as ancient and widespread as chromatin itself. Histone sequences have changed very little throughout evolution, suggesting the possibility that some of their 'non-canonical' functions are at play in parallel or in concert with their genome regulatory functions. In this Review, we take an evolutionary perspective of histone, nuclear chromatin and extracellular chromatin biology and describe the known extranuclear and extracellular functions of histones. We detail molecular mechanisms of chromatin release and extracellular chromatin sensing, and we discuss their roles in physiology and disease. Finally, we present evidence and give a perspective on the potential of extracellular histones to act as bioactive, cell modulatory factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2006f62e95e3010581233345151a9c7075d4f96c" target='_blank'>
              Fantastic proteins and where to find them - histones, in the nucleus and beyond.
              </a>
            </td>
          <td>
            Johanna Grinat, Noah P Shriever, M. A. Christophorou
          </td>
          <td>2024-12-15</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef930bd56fde447e683b8e74f0950f751b9d7434" target='_blank'>
              Reassessing the roles of oxidative DNA base lesion 8-oxoGua and repair enzyme OGG1 in tumorigenesis
              </a>
            </td>
          <td>
            Jing Wang, Chunshuang Li, Jinling Han, Yaoyao Xue, Xu Zheng, Ruoxi Wang, Z. Radák, Y. Nakabeppu, Istvan Boldogh, Xueqing Ba
          </td>
          <td>2025-01-01</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Enhancer of zeste homolog 2 (EZH2) is a methyltransferase involved in cell cycle regulation, cell differentiation, and cell death and plays a role in modulating the immune response. Although it mainly functions by catalyzing the tri-methylation of H3 histone on K27 (H3K27), to inhibit the transcription of target genes, EZH2 can directly methylate several transcription factors or form complexes with them, regulating their functions. EZH2 expression/activity is often dysregulated in cancer, contributing to carcinogenesis and immune escape, thereby representing an important target in anti-cancer therapy. This review summarizes some of the mechanisms through which EZH2 regulates the expression and function of tumor suppressor genes and oncogenic molecules such as STAT3, mutant p53, and c-Myc and how it modulates the anti-cancer immune response. The influence of posttranslational modifications on EZH2 activity and stability and the possible strategies leading to its inhibition are also reviewed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0220899c23d1ac05dd7a74006eb2a6d3a973aaf" target='_blank'>
              Targeting EZH2 in Cancer: Mechanisms, Pathways, and Therapeutic Potential
              </a>
            </td>
          <td>
            M. S. Gilardini Montani, Rossella Benedetti, M. Cirone
          </td>
          <td>2024-12-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Genetic variation underpins evolutionary change, but accumulation of slightly deleterious mutations also increases mutation load. There are multiple factors affecting the extent of load such as population size and breeding system, yet other potential determinants remain unexplored. A common macromutation, whole-genome duplication (WGD) occurs broadly across Eukaryotes, yet we lack a clear understanding of how WGD impacts neutral and selective processes within a population. Using forward simulations and empirical analysis of 632 short- and 16 long-read sequenced individuals of Arabidopsis arenosa (23 diploid and 42 natural autotetraploid populations), we test for the effects of WGD on genome-wide diversity and mutation load. Our simulations show how genetic variation gradually rises in autotetraploids due to increase of mutational target size. Moreover, mutation load increases due to relaxed purifying selection when deleterious mutations are masked by additional chromosome copies. Empirical data confirm these patterns, showing significant increase in nucleotide diversity, ratios of non-synonymous to synonymous SNPs, and number of indels and large structural variants in A. arenosa autotetraploids. However, a rather modest increase in load proxies together with a broad distribution and niche of autotetraploids suggests load accumulation has not (yet) limited their successful expansion. Overall, we demonstrate a complex interplay between neutral processes and purifying selection in shaping genetic variation following WGD and highlight ploidy as an important determinant of genetic diversity and mutation load in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86bbacd2db01892da50f5a93e5f2a705f35e9543" target='_blank'>
              Whole-genome duplication increases genetic diversity and load in outcrossing Arabidopsis
              </a>
            </td>
          <td>
            Jakub Vlček, Tuomas Hämälä, Cristina Vives Cobo, Emma Curran, G. Šrámková, T. Slotte, R. Schmickl, Levi Yant, Filip Kolář
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma.
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogenic gamma herpesviruses, including Epstein–Barr Virus (EBV) and Kaposi’s Sarcoma-associated Herpesvirus (KSHV), are opportunistic cancer-causing viruses and induces oncogenesis through complex mechanisms, which involves manipulation of cellular physiology as well as epigenetic and epitranscriptomic reprogramming. In this review, we describe the intricate processes by which these viruses interact with the epigenetic machinery, leading to alterations in DNA methylation, histone modifications, and the involvement of non-coding RNAs. The key viral proteins such as EBNA1 and LMP1 encoded by EBV; LANA and vGPCR encoded by KSHV; play pivotal roles in these modifications by interacting with host factors, and dysregulating signaling pathways. The resultant reprogramming can lead to activation of oncogenes, silencing of tumor suppressor genes, and evasion of the immune response, which ultimately contributes to the oncogenic potential of these viruses. Furthermore, in this review, we explore current therapeutic strategies targeting these epigenetic alterations and discuss future directions for research and treatment. Through this comprehensive examination of the epigenetic and epitranscriptomic reprogramming mechanisms employed by oncogenic gamma herpesviruses, we aim to provide valuable insights into potential avenues for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d106bf10134bf9b8889820e2db155164b9012c0" target='_blank'>
              Epigenetic and epitranscriptomic regulation during oncogenic γ-herpesvirus infection
              </a>
            </td>
          <td>
            Rajnish Kumar Singh, Ramakrishna Vangala, Atharva S. Torne, Dipayan Bose, E. Robertson
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Myc hyperactivation coordinately regulates numerous metabolic processes to drive lymphomagenesis. Here, we elucidate the temporal and functional relationships between the medley of pathways, factors, and mechanisms that cooperate to control redox homeostasis in Myc-overexpressing B cell lymphomas. We find that Myc overexpression rapidly stimulates the oxidative pentose phosphate pathway (oxPPP), nucleotide synthesis, and mitochondrial respiration, which collectively steers cellular equilibrium to a more oxidative state. We identify Myc-dependent hyperactivation of the phosphoribosyl pyrophosphate synthetase (PRPS) enzyme as a primary regulator of redox status in lymphoma cells. Mechanistically, we show that genetic inactivation of the PRPS2 isozyme, but not PRPS1, in Myc-driven lymphoma cells leads to elevated NADPH levels and reductive stress-mediated death. Employing a pharmacological screen, we demonstrate how targeting PRPS1 or PRPS2 elicits opposing sensitivity or resistance, respectively, to chemotherapeutic agents affecting the thioredoxin and glutathione network, thus providing a therapeutic blueprint for treating Myc-driven lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654099247bbca4f140ebd9c31e764d3e528bd649" target='_blank'>
              PRPS activity tunes redox homeostasis in Myc-driven lymphoma
              </a>
            </td>
          <td>
            Austin C. Macmillan, Bibek R. Karki, Juechen Yang, Karmela R. Gertz, Samantha Zumwalde, M. Czyzyk-Krzeska, Jarek Meller, J. T. Cunningham
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Viruses represent the most abundant biological entities on Earth, with bacteriophages (or phages) specifically infecting bacteria. Typically, double-strand DNA phages possess genomes around 50 kilobase pairs (kbp) in size, although the largest known genome surpasses 735 kbp. This raises intriguing questions about the potential maximum size of phage genomes. In this study, we present the first phage genome over 800 kbp, named BF3, which was reconstructed from oil reservoir production water. BF3 encodes 1,164 protein-coding genes (code 11) and 46 tRNA genes, and is classified within the Caudoviricetes, though its bacterial host remains unidentified. We utilized ColabFold to predict the structures of 744 protein-coding genes with high confidence, finding that 395 and 591 of these genes corresponded with known structures in the Protein Data Bank and the Big Fantastic Virus Database (BFVD), respectively. Notably, 153 of BF3’s predicted structures exhibited no similarity to those catalogued in the BFVD, which is currently the most extensive viral protein structure database. This study not only expands our understanding of phage genome capacities but also underscores the need for specialized analytical tools and pipelines to investigate exceptionally large phages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3cdea26be54a6e6da8e0b634e3f223b9cc7aee6" target='_blank'>
              Discovery and analysis of an 841 kbp phage genome: the largest known to date
              </a>
            </td>
          <td>
            Linxing Chen
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV), a γ-herpesvirus, is predominantly associated with Kaposi’s sarcoma (KS) as well as two lymphoproliferative disorders: primary effusion lymphoma (PEL) and multicentric Castleman disease (MCD). Like other herpesviruses, KSHV employs two distinct life cycles: latency and lytic replication. To establish a lifelong persistent infection, KSHV has evolved various strategies to manipulate the epigenetic machinery of the host. In latently infected cells, most viral genes are epigenetically silenced by components of cellular chromatin, DNA methylation and histone post-translational modifications. However, some specific latent genes are preserved and actively expressed to maintain the virus’s latent state within the host cell. Latency is not a dead end, but the virus has the ability to reactivate. This reactivation is a complex process that involves the removal of repressive chromatin modifications and increased accessibility for both viral and cellular factors, allowing the activation of the full transcriptional program necessary for the subsequent lytic replication. This review will introduce the roles of epigenetic modifications in KSHV latent and lytic life cycles, including DNA methylation, histone methylation and acetylation modifications, chromatin remodeling, genome conformation, and non-coding RNA expression. Additionally, we will also review the transcriptional regulation of viral genes and host factors in KSHV infection. This review aims to enhance our understanding of the molecular mechanisms of epigenetic modifications and transcriptional regulation in the KSHV life cycle, providing insights for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65f8fc61715f58044e9d19bdbbd3b68cc4bfb9d" target='_blank'>
              Rewriting Viral Fate: Epigenetic and Transcriptional Dynamics in KSHV Infection
              </a>
            </td>
          <td>
            Chunyan Han, Danping Niu, Ke Lan
          </td>
          <td>2024-11-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, Tamara Basta
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Noncoding satellite DNA repeats are abundant at the pericentromeric heterochromatin of eukaryotic chromosomes. During interphase, sequence-specific DNA-binding proteins cluster these repeats from multiple chromosomes into nuclear foci known as chromocenters. Despite the pivotal role of chromocenters in cellular processes like genome encapsulation and gene repression, the associated proteins remain incompletely characterized. Here, we use 2 satellite DNA-binding proteins, D1 and Prod, as baits to characterize the chromocenter-associated proteome in Drosophila embryos, ovaries, and testes through quantitative mass spectrometry. We identify D1- and Prod-associated proteins, including known heterochromatin proteins as well as proteins previously unlinked to satellite DNA or chromocenters, thereby laying the foundation for a comprehensive understanding of cellular functions enabled by satellite DNA repeats and their associated proteins. Interestingly, we find that multiple components of the transposon-silencing piRNA pathway are associated with D1 and Prod in embryos. Using genetics, transcriptomics, and small RNA profiling, we show that flies lacking D1 during embryogenesis exhibit transposon expression and gonadal atrophy as adults. We further demonstrate that this gonadal atrophy can be rescued by mutating the checkpoint kinase, Chk2, which mediates germ cell arrest in response to transposon mobilization. Thus, we reveal that a satellite DNA-binding protein functions during embryogenesis to silence transposons, in a manner that is heritable across later stages of development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26b7e353a809cd613bd6598d2720c5f84058d838" target='_blank'>
              Multi-tissue characterization of the constitutive heterochromatin proteome in Drosophila identifies a link between satellite DNA organization and transposon repression
              </a>
            </td>
          <td>
            Ankita Chavan, Lena Skrutl, Federico Uliana, Mélanie Pfister, Franziska Brändle, László Tirián, Delora Baptista, Dominik Handler, David Burke, Anna Sintsova, P. Beltrao, Julius Brennecke, Madhav Jagannathan
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/692b233441e797a73e14defb069a03a55d2ca324" target='_blank'>
              DNA methylation profiling at base-pair resolution reveals unique epigenetic features of early-onset colorectal cancer in underrepresented populations
              </a>
            </td>
          <td>
            J. S. Li, Karen Riggins, Li Yang, Chaorong Chen, Patricia Castro, Wedad Alfarkh, Neda Zarrin-Khameh, Michael E. Scheurer, Chad J. Creighton, Benjamin Musher, Wei Li, Lanlan Shen
          </td>
          <td>2025-01-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ubiquitination is cells' second most abundant posttranslational protein modification after phosphorylation. The ubiquitin-proteasome system (UPS) is critical in maintaining essential life processes such as cell cycle control, DNA damage repair, and apoptosis. Mutations in ubiquitination pathway genes are strongly linked to the development and spread of multiple cancers since several of the UPS family members possess oncogenic or tumor suppressor activities. This comprehensive review delves into understanding the ubiquitin code, shedding light on its role in cancer cell biology and immune evasion. Furthermore, we highlighted recent advances in the field for targeting the UPS pathway members for effective therapeutic intervention against human cancers. We also discussed the recent update on small-molecule inhibitors and PROTACs and their progress in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc272aae9b08cad5711453e2ddaf7f491ca23dc" target='_blank'>
              Ubiquitination Enzymes in Cancer, Cancer Immune Evasion, and Potential Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Aiman B. Awan, Maryiam Jama Ali Osman, Omar M. Khan
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mitochondrial function and dynamics are crucial for maintaining cellular homeostasis and overall health. Disruptions in these processes can lead to various diseases, including cancer. The Hippo signaling pathway, a universal growth regulator, plays central roles in cancer through its main effector, the Yes-associated protein (YAP), known as Yorkie (Yki) in Drosophila. Yki upregulation drives benign tissue overgrowth in Drosophila imaginal discs. Our research demonstrates that the conserved metabolic regulator dPGC1 limits Yki-driven tissue hyperplasia and maintains tissue homeostasis in vivo. Yki upregulation and dPGC1 depletion lead to tumors characterized by enlarged and hyperfused mitochondria, a condition both necessary and sufficient for Yki-driven oncogenic growth. Our findings demonstrate that mitochondrial hyperfusion elevates the levels of the cell cycle regulator Cyclin E, which is crucial for tumor development. Our findings identify dPGC1 as a context-dependent tumor suppressor that coordinates mitochondrial dynamics and cell cycle regulation in response to oncogene activation. Defects in these processes are commonly found in cancer cells, highlighting major implications for cancer development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/238724bd493ec2c48bc12750dc3d9f0183bf9e62" target='_blank'>
              Control of mitochondrial dynamics by dPGC1 limits Yorkie-induced oncogenic growth in Drosophila
              </a>
            </td>
          <td>
            Wei Qi Guinevere Sew, Zhiquan Li, L. J. Rasmussen, Héctor Herranz
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7d2fbf8a9f9c82b98c7984b3782b85218b8319" target='_blank'>
              Crypt density and recruited enhancers underlie intestinal tumour initiation.
              </a>
            </td>
          <td>
            Liam T. Gaynor, Harshabad Singh, Guodong Tie, Krithika Badarinath, Shariq Madha, Andrew Mancini, Swarnabh Bhattacharya, Mikio Hoshino, Frederic J. de Sauvage, Kazutaka Murata, Unmesh Jadhav, Ramesh A. Shivdasani
          </td>
          <td>2025-01-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The strong size increase of the human neocortex is supported both by the amplification and the basal translocation of a neural stem cell population, the basal radial glial cells (or bRG cells). Using live imaging of second trimester human fetal tissue and cortical organoids, we identify two independent translocation modes for bRG cell colonization of the human neocortex. On top of an actomyosin-dependent movement called mitotic somal translocation (MST), we identify a microtubule-dependent motion occurring during interphase, that we call interphasic somal translocation (IST). We show that IST is driven by the LINC complex, through the nuclear envelope recruitment of the dynein motor and of its activator LIS1. Consequently, IST severely altered in LIS1 patient-derived cortical organoids. We also demonstrate that MST occurs during prometaphase and is a mitotic spindle translocation event. MST is controlled by the mitotic cell rounding molecular pathway, via Moesin and Vimentin, driving translocation. We report that 85% of bRG cell translocation is due to IST, for a total movement of 0,67 mm per month of human fetal gestation. Our work identifies how bRG cells colonize the human fetal cortex, and further shows that IST and MST are conserved in bRG-related migrating glioblastoma cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5162f39b5b35fd089916583d60a7b8a2e59a995" target='_blank'>
              Two independent translocation modes drive neural stem cell dissemination into the human fetal cortex
              </a>
            </td>
          <td>
            Ryszard Wimmer, Pauline Lestienne, Clarisse Brunet, L. Coquand, Amandine Di Cicco, Christophe Chehade, A. Artioli, Matthieu Cortes, Anne-Sophie Macé, Xiuyu Wang, Jean-Baptiste Manneville, Bettina Bessière, Ananya Roy, K. Forsberg-Nilsson, Julia Ladewig, Fabien Guimiot, Alexandre D. Baffet
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="DNA methylation is a crucial epigenetic modification that orchestrates chromatin remodelers that suppress transcription, and aberrations in DNA methylation result in a variety of conditions such as cancers and developmental disorders. While it is understood that methylation occurs at CpG-rich DNA regions, it is less understood how distinct methylation profiles are established within various cell types. In this work, we develop a molecular-transport model that depicts the genomic exploration of DNA methyltransferase within a multiscale DNA environment, incorporating biologically relevant factors like methylation rate and CpG density to predict how patterns are established. Our model predicts DNA methylation-state correlation distributions arising from the transport and kinetic properties that are crucial for the establishment of unique methylation profiles. We model the methylation correlation distributions of nine cancerous human cell types to determine how these properties affect the epigenetic profile. Our theory is capable of recapitulating experimental methylation patterns, suggesting the importance of DNA methyltransferase transport in epigenetic regulation. Through this work, we propose a mechanistic description for the establishment of methylation profiles, capturing the key behavioral characteristics of methyltransferase that lead to aberrant methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e458fbb01165e95120272a65d7850681688311a0" target='_blank'>
              Modeling DNA methyltransferase function to predict epigenetic correlation patterns in healthy and cancer cells.
              </a>
            </td>
          <td>
            Ariana Y Tse, A. Spakowitz
          </td>
          <td>2025-01-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10-10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5'-TGT*-3' motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential.IMPORTANCEYarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica.
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and environmental microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Deoxyribose nucleic acid (DNA) methylation is an important epigenetic modification that plays an important role in the occurrence and development of tumors. Identifying key methylation-driven genes that affect the prognosis of lung squamous cell carcinoma (LUSC) can provide direction for targeted therapy research. Methods and results Methylation and RNA-seq data were downloaded from The Cancer Genome Atlas (TCGA). The MethylMix package was used to integrate and analyze the methylation and gene expression data from TCGA, and the LUSC dataset (GSE37745) was downloaded from GEO for validation. Forty-five DNA-methylation-driven genes (MDGs) were obtained, and 3 genes (TRIM61, SMIM22, and ALDH7A1) were significantly associated with survival by using univariate and multivariate Cox regression. A risk model was constructed. KM analysis showed that patients with high-risk scores had poor survival. A nomination plot for prognosis prediction of LUSC patients was constructed, which showed a good predictive efficiency for tumor prognosis. The high expression of ALDH7A1 was an independent risk factor for poor prognosis in LUSC. The expression of ALDH7A1 in LUSC was negatively correlated with its methylation status (COR = −0.655). GSEA analysis showed that high expression of ALDH7A1 could activate multiple signaling pathways (JAK-STAT signaling pathway and mTOR signaling pathway). In vitro cell experiments confirmed that in LUSC, silencing ALDH7A1 could inhibit tumor progression, while overexpression of ALDH7A1 could promote tumor progression. Conclusion Our results indicated that ALDH7A1, a newly discovered MDG in LUSC, could act as an independent prognostic factor for OS in LUSC, with the potential to become a potential target for LUSC diagnosis and treatment. High expression of ALDH7A1 in LUSC could promote the occurrence and development of tumors. Signaling pathways, such as JAK-STAT and mTOR signaling pathways, might regulate the high expression of ALDH7A1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa4fa0f5654e845798a1c36db8d638da005aaf3" target='_blank'>
              Identification of ALDH7A1 as a DNA-methylation-driven gene in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Gaofeng Liang, Jinxian He, Tian Chen, Liang Zhang, Kaizhong Yu, Weiyu Shen
          </td>
          <td>2024-12-23</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite its relatively recent emergence, single-cell sequencing has cemented its place in scientific research. It has grown exponentially in less than two decades since its start, with broad impact in the biological sciences. The blood represents an attractive system for early development and application of single-cell technologies. As a result, single-cell analyses in blood and leukemia have led the way in describing how cellular heterogeneity affects cancer progression. In this review, we discuss the technological and conceptual advances brought by single-cell genomics, ranging from genetic evolution and differentiation states that mediate drug resistance to the complex interactions required for immunotherapy responses. These high-resolution insights are starting to enter clinical assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fa592467bc471ca2cb2ea46b2951c1d4aa2628" target='_blank'>
              Insights into Leukemia from Single-Cell Genomics
              </a>
            </td>
          <td>
            Nurefsan Sariipek, Peter van Galen
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc6572745a00744a0ffe9484307c354e0370541" target='_blank'>
              Short tandem repeats delineate gene bodies across eukaryotes
              </a>
            </td>
          <td>
            William B. Reinar, A. Krabberød, V. O. Lalun, M. Butenko, K.S. Jakobsen
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The presence of specific genetic mutations in patients with glioblastoma multiforme (GBM) is associated with improved survival outcomes. Disruption of the DNA damage response (DDR) pathway in tumor cells enhances the effectiveness of radiotherapy drugs, while increased mutational burden following tumor cell damage also facilitates the efficacy of immunotherapy. The ATRX gene, located on chromosome X, plays a crucial role in DDR. The aim of this research is to elucidate the correlation between ATRX mutations and GBM. Dataset obtained from TCGA-GBM were conducted an analysis on the genomic features, biological characteristics, immunopathological markers, and clinical prognosis of patients carrying ATRX mutations. Our findings revealed a significantly elevated level of microsatellite instability in individuals with ATRX mutants, along with significant alterations in the receptor-tyrosine kinase (RTK)-ras pathway among patients exhibiting combined ATRX mutations. TCGA-GBM patients with concurrent ATRX mutations exhibited sensitivity to 26 chemotherapeutic and anticancer drugs, which exerted their effects by modulating the DDR of tumor cells through highly correlated mechanisms involving the RTK-ras pathway. Additionally, we observed an enrichment of ATRX mutations in specific pathways associated with DDR among TCGA-GBM patients. Our model also demonstrated prolonged overall survival in patients carrying ATRX mutations, particularly showing strong predictive value for 3- and 5-year survival rates. Furthermore, additional protective factors such as younger age, female gender, combined IDH mutations, and TP53 mutations were identified. The results underscore the protective role and prognostic significance of ATRX mutations in GBM as a potential therapeutic target and biomarker for patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89334fb95e9947311852d880d8c780526377835e" target='_blank'>
              ATRX mutation modifies the DNA damage response in glioblastoma multiforme tumor cells and enhances patient prognosis
              </a>
            </td>
          <td>
            Rou Yu, Keru Huang, Xinyan He, Jingwen Zhang, Yushan Ma, Hui Liu
          </td>
          <td>2025-01-10</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) has been found to be widespread and functional in various organisms. However, comparative analyses of pre- and post-infection of virus are rarely known. Herein, we investigated the changes in expression patterns of eccDNA following infection with Bombyx mori cytoplasmic polyhedrosis virus (BmCPV) and explore the role of eccDNA in viral infection. Circle-seq was used to analyze eccDNAs in the midgut of BmCPV-infected and BmCPV-uninfected silkworms. A total of 5508 eccDNAs were identified, with sizes varying from 72 bp to 17 kb. Most of eccDNAs are between 100 to 1000 bp in size. EccDNA abundance in BmCPV-infected silkworms was significantly higher than in BmCPV-uninfected silkworms. GO and KEGG analysis of genes carried by eccDNAs reveals that most are involved in microtubule motor activity, phosphatidic acid binding, cAMP signaling pathway, and pancreatic secretion signaling pathways. Several eccDNAs contain sequences of the transcription factor SOX6, sem-2, sp8b, or Foxa2. Association analysis of eccDNA-mRNA/miRNA/circRNA revealed that some highly expressed genes are transcribed from relevant sequences of eccDNA and the transcription of protein coding genes influenced the frequency of eccDNA. BmCPV infection resulted in changes in the expression levels of six miRNAs, but no known miRNAs with altered expression levels due to changes in eccDNA abundance were identified. Moreover, it was found that 1287 and 924 sequences representing back-spliced junctions of circRNAs were shared by the junctions of eccDNAs in the BmCPV-infected and uninfected silkworms, respectively, and some eccDNAs loci were shared by circRNAs on Chromosomes 2, 7, 11, 14, and 24, suggesting some eccDNAs may exert its function by being transcribed into circRNAs. These findings suggest that BmCPV infection alter the expression pattern of eccDNAs, leading to changes in RNA transcription levels, which may play roles in regulating BmCPV replication. In the future, further experiments are needed to verify the association between eccDNA-mRNA/miRNA/circRNA and its function in BmCPV infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cfcd5648670a234c50e6896700101007b2cd3a1" target='_blank'>
              Genome-Wide Characterization of Extrachromosomal Circular DNA in the Midgut of BmCPV-Infected Silkworms and Its Potential Role in Antiviral Responses.
              </a>
            </td>
          <td>
            Xinyu Tong, Chao Lei, Yilin Liu, Mei Yin, Huan Peng, Qunnan Qiu, Yongjie Feng, Xiaolong Hu, C. Gong, Min Zhu
          </td>
          <td>2025-01-19</td>
          <td>International journal of molecular sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceda101cbd27e107d725418d6adeff717071b560" target='_blank'>
              RTEL1 is upregulated in gastric cancer and promotes tumor growth
              </a>
            </td>
          <td>
            Chunyu Yang, Suzeng Wang, Ge Gao, Peiwen Xu, Mengyuan Qian, Yuan Yin, Surui Yao, Zhaohui Huang, Zehua Bian
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a rare aggressive haematological malignancy characterised by the clonal expansion of immature T-cell precursors. It accounts for 15% of paediatric and 25% of adult ALL. T-ALL is associated with the overexpression of major transcription factors (TLX1/3, TAL1, HOXA) that drive specific transcriptional programmes and constitute the molecular classifying subgroups of T-ALL. Although the dysregulation of transcription factor oncogenes is frequently associated with chromosomal translocations in T-ALL, epigenetic dysregulation resulting in changes to post-translational modifications of histones has also been reported. This includes non-coding intergenic mutations that form oncogenic neo-enhancers. This review will focus on the known epigenetically activating intergenic mutations reported in T-ALL, and will discuss the wider implications of neo-enhancer mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f059393054a743b5ecf3a79dba4386f2b175eb" target='_blank'>
              Neo-enhancers in T-cell acute lymphoblastic Leukaemia (T-ALL) and beyond.
              </a>
            </td>
          <td>
            Charlotte Smith, V. Asnafi, A. Touzart
          </td>
          <td>2025-01-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 RNaseH2 plays a critical role in removing mis-bounded RNA from the DNA double strand helix1. Absence of RNaseH2 results in spontaneous double-stranded DNA breaks and cytosolic release of dsDNA2. We have previously shown that epithelial deficiency of RNaseH2b (H2bΔIEC) leads to a p53-dependent stem cell suppression and additional deficiency of p53 results in spontaneous intestinal carcinogenesis3. Due to the central role of cGAS/STING in sensing dsDNA, we aim to investigate the role of epithelial STING in a mouse model of chronic epithelial DNA damage and tumor prevention in this study.



 DNA damage was assessed using comet assay in ModeK cell line with cGAS and STING knock out. Protein kinase activity were assessed by Pamgene Kinase Assay. Intestinal organoids were generated from H2bΔIEC and H2bΔIEC/Tmem173-/-mice. Colony forming assay and EdU staining were performed in all genotypes to assess the impact of STING on epithelial stem cell function and proliferation were assessed by CellTiter Glo assay.
 RNA sequencing was performed in intestinal organoids. Spontaneous carcinogenesis was assessed in 1-year aged mice.



 Loss of RNaseH2b results in activation of cGAS/STING pathway and subsequent upregulation of IFN-I signature. Deficiency of cGAS/STING lead to impaired IFN-I independent DNA damage repair which were accessed by WB and comet assay. During the DNA damage response, STING but not cGAS mediates the P53 pathway and the P21-CDK axis. Organoid EdU stanning displayed restored proliferation in H2bΔIEC/Tmem173-/- compared to H2bΔIEC. RNA sequencing from H2bΔIEC/Tmem173-/-organoid further confirmed decreased G2/M DNA damage checkpoint and P53 signature compared to H2bΔIEC. In vivo, we show that H2bΔIEC/Tmem173-/- mice have downregulated DNA damage repair and developed spontaneous small bowel adenocarcinoma after 53 weeks. Tumor derived from H2bΔIEC/Tmem173-/- mice show sensitivity to CDK inhibitor consistent with in vitro validation.



 We provide evidence that non-canonical role of epithelial STING mediates DNA damage response and guides P21-CDK1 axis in preventing tumor genesis. Our research provided evidence understand the role of STING in DNA damage response and intestine carcinogenesis.



 1.Reijns, Martin A.M., et al., Enzymatic Removal of Ribonucleotides from DNA Is Essential for Mammalian Genome Integrity and Development. Cell, 2012. 149(5): p. 1008-1022.
 2.Bartsch, K., et al., Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Hum Mol Genet, 2017. 26(20): p. 3960-3972.
 3.Aden, K., et al., Epithelial RNase H2 Maintains Genome Integrity and Prevents Intestinal Tumorigenesis in Mice. Gastroenterology, 2019. 156(1): p. 145-159 e19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0539a26d1a5ba9dc2728ef562232f66e1b1c39" target='_blank'>
              P0212 Non-canonical STING signaling in intestinal epithelial cells guides DNA damage-driven stem cell function and protects from intestinal carcinogenesis
              </a>
            </td>
          <td>
            H. Xiang, C. Woitaske-Proske, J. Bernardes, G. R. Martini, F. Wottawa, J. Bornhaeuser, N. Kakavand, Y. Guang, Q. Wu, J. Kugler, L. Welz, B. Konukiewitz, C. Peifer, S. Schreiber, M. Tschurtschenthaler, P. Rosenstiel, K. Aden
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92332ae486c3b94c41f8d10e6867cf69c3624d" target='_blank'>
              Unveiling the molecular profile of a prostate carcinoma: implications for personalized medicine
              </a>
            </td>
          <td>
            Massimiliano Agostini, Erica Giacobbi, F. Servadei, Julia Bishof, Likas Funke, Giuseppe Sica, Valentina Rovella, Marco Carilli, V. Iacovelli, Yufang Shi, Jianquan Hou, E. Candi, G. Melino, Giulio Cervelli, M. Scimeca, A. Mauriello, Pierluigi Bove
          </td>
          <td>2024-12-31</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cytosine (DNA) methylation plays important roles in silencing transposable elements, plant development, genomic imprinting, stress responses, and maintenance of genome stability. To better understand the functions of this epigenetic modification, several tools have been developed to manipulate DNA methylation levels. These include mutants of DNA methylation writers and readers, targeted manipulation of locus-specific methylation, and the use of chemical inhibitors. Here, we summarize the effects of commonly used cytidine analog chemical inhibitors represented by zebularine, 5-azacytidine, and their related compounds on plants. These analogs are incorporated into the chromosomal DNA, where they block the activity of the replicative CG DNA methyltransferase 1 (MET1). This leads to manifold alterations in plant epigenome, modified developmental programs, or suppression of hybridization barriers. We also highlight the DNA-damaging effects of cytidine analogs, particularly the formation of stable DNA-protein crosslinks between DNA and MET1. This sheds new light on specific phenotypes observed upon cytidine analog treatments. In conclusion, cytidine analogs remain a vital tool for plant genome research and have the potential to open new promising avenues for applications in plant biotechnology and breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347eb5947bdb94f55760bce2a406c75b6df28aaf" target='_blank'>
              Cytidine analogs in plant epigenetic research and beyond.
              </a>
            </td>
          <td>
            Eva Dvořák Tomaštíková, A. Pečinka
          </td>
          <td>2024-12-28</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumorigenesis and progression are complex process with multifarious molecules and related mechanisms, while most of them are still elusive. Most research into the mechanisms of cancer today are focusing on encoding RNAs. However, for the past few years as the emergence and development of epigenetics, a kind of new molecule: non-coding RNAs (ncRNAs), RNAs without protein-coding function, have gained prominence in recent scientific discussions. Since the first ncRNA was discovered in 1990s, extensive research has shown that ncRNAs significantly influence processes such as cell growth, differentiation, metabolic regulation, and programmed cell death at both transcriptional and post-transcriptional stages. By means of acting as the tumor suppression and oncogenesis, research has identified several ncRNAs with abnormal expression patterns in cancer cells, marking them as primary oncogenic factors or valuable targets in cancer treatment. This review focus on three kinds of ncRNA, MicroRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs), the molecule structure, generative mechanism, characteristic function and regulatory mechanism in tumor progression. In addition, we will discuss the application value and future expiration of the therapeutic that are focusing on ncRNA, that will open up a new idea and methods for the treatment of cancer for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026f448482a66bffaadbd5d2efa692a3ff825932" target='_blank'>
              Non-coding RNAs in Tumor Biology: Exploring miRNAs, lncRNAs, and circRNAs Roles and Therapeutic Potentials
              </a>
            </td>
          <td>
            Tianhao Zhang
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecological malignancy, and there is still an unmet medical need to deepen basic research on its origins and mechanisms of progression. Patient-derived organoids of high-grade serous ovarian cancer (HGSOC-PDO) are a powerful model to study the complexity of ovarian cancer as they maintain, in vitro, the mutational profile and cellular architecture of the cancer tissue. Genetic modifications by lentiviral transduction allow novel insights into signaling pathways and the potential identification of biomarkers regarding the evolution of drug resistance. Here, we provide an in-depth and detailed protocol to successfully modify the gene expression of HGSOC-PDOs by lentiviral transduction. As an example, we validate our protocol and create a stable knockdown of the MACC1 oncogene with an efficacy of ≥72% in two HGSOC-PDO lines, which remained stable for >3 months in culture. Moreover, we explain step-by-step the sample preparation for the validation procedures on transcriptional (qPCR) and protein (Western Blot) levels. Sustained downregulation of specific genes by lentiviral transduction enables the analysis of the resulting phenotypic and morphological changes. It serves as a valuable in-vitro model to study the mechanisms of ovarian cancer pathogenesis and allows for the evaluation of therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c972ee3bf2a62f9085f299cf8d2e6e93c66fc34c" target='_blank'>
              Establishment of Stable Knockdown of MACC1 Oncogene in Patient-Derived Ovarian Cancer Organoids
              </a>
            </td>
          <td>
            Sophia Hierlmayer, Liliia Hladchenko, Juliane Reichenbach, Christoph Klein, Sven Mahner, F. Trillsch, Mirjana Kessler, A. Chelariu-Raicu
          </td>
          <td>2024-12-01</td>
          <td>Methods and Protocols</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481e81722a6ce523752049f844fedd532d84c573" target='_blank'>
              Transcriptomic landscape of Hras12V oncogene-induced hepatocarcinogenesis with gender disparity
              </a>
            </td>
          <td>
            Huaiyuan Di, Zhuona Rong, Nan Mao, Huiling Li, Jun Chen, Renwu Liu, Aiguo Wang
          </td>
          <td>2025-01-16</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Kinetoplastids display a single, large mitochondrion per cell, with their mitochondrial DNA referred to as the kinetoplast. This kinetoplast is a network of concatenated circular molecules comprising a maxicircle (20-64 kb) and up to thousands of minicircles varying in size depending on the species (0.5-10 kb). In Trypanosoma cruzi, maxicircles contain typical mitochondrial genes found in other eukaryotes. They consist of coding and divergent/variable regions, complicating their assembly due to repetitive elements. However, next-generation sequencing (NGS) methods have resolved these issues, enabling the complete sequencing of maxicircles from different strains. Furthermore, several insertions and deletions in the maxicircle sequences have been identified among strains, affecting specific genes. Unique to kinetoplastids, minicircles play a crucial role in a particular U-insertion/deletion RNA editing system by encoding guide RNAs (gRNAs). These gRNAs are essential for editing and maturing maxicircle mRNAs. In Trypanosoma cruzi, although only a few studies have utilized NGS methods to date, the structure of these molecules suggests a classification into four main groups of minicircles. This classification is based on their size and the number of highly conserved regions (mHCRs) and hypervariable regions (mHVRs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05d10adddf5fb9302615fcadcf95b2c5e7dc1271" target='_blank'>
              Mitochondrial DNA Structure in Trypanosoma cruzi.
              </a>
            </td>
          <td>
            Alfonso Herreros-Cabello, Francisco Callejas-Hernández, Manuel Fresno, N. Gironès
          </td>
          <td>2025-01-14</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7be5c20b4cd0ad9d6c8146c5936042d6f39bb61" target='_blank'>
              CRYβB2 alters cell adhesion to promote invasion in a triple-negative breast cancer cell line
              </a>
            </td>
          <td>
            Amr A. Waly, London Harper, Jodie M. Fleming, Lindsey M Costantini
          </td>
          <td>2025-01-21</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02fb4ad81cdcfd96f458e312f0500d09b4e4efb2" target='_blank'>
              Computational analysis of MYC gene variants: structural and functional impact of non-synonymous SNPs.
              </a>
            </td>
          <td>
            Plabita Bhuyan, Varshabi Bharali, Sangju Basumatary, Aido Lego, Juman Sarma, Debasish Borbora
          </td>
          <td>2024-12-14</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract HAP1 is a near-haploid human cell line commonly used for mutagenesis and genome editing studies due to its hemizygous nature. We noticed an unusual hypersensitivity of HAP1 to camptothecin, an antineoplastic drug that stabilizes topoisomerase I cleavage complexes (TOP1ccs). We have attributed this hypersensitivity to a deficiency of TDP1, a key phosphodiesterase involved in resolving abortive TOP1ccs. Through whole-exome sequencing and subsequent restoration of TDP1 protein via CRISPR-Cas9 endogenous genome editing, we demonstrate that TDP1 deficiency and camptothecin hypersensitivity in HAP1 cells are a result of a splice-site mutation (TDP1 c.660–1G > A) that causes exon skipping and TDP1 loss of function. The lack of TDP1 in HAP1 cells should be considered when studying topoisomerase-associated DNA lesions and when generalizing mechanisms of DNA damage repair using HAP1 cells. Finally, we also report the generation of HAP1 STAR clones with restored TDP1 expression and function, which may be useful in further studies to probe cellular phenotypes relating to TOP1cc repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891032e07f96c869d97db1867e4b8a4b2832cd9" target='_blank'>
              TDP1 splice-site mutation causes HAP1 cell hypersensitivity to topoisomerase I inhibition
              </a>
            </td>
          <td>
            Chen Gang Goh, Aldo S. Bader, Tuan-Anh Tran, R. Belotserkovskaya, Giuseppina D'Alessandro, S. P. Jackson
          </td>
          <td>2024-12-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast cancer, the most prevalent cancer in females, is a heterogeneous disease with various molecular subtypes, which presents challenges in diagnosis and treatment. Ubiquitination is one of the most critical post-translational protein modifications, that plays regulatory roles in numerous cellular processes including cell cycle progression, DNA replication & repair, apoptosis, transcription regulation, protein localization, trafficking and signal transduction. This modification can be reversed by deubiquitinases, or DUBs, a superfamily of cysteine proteases and metalloproteases that cleave ubiquitin-protein bonds. Dysregulation of DUBs has been associated to various diseases including cancer, making them promising targets for cancer therapy. We leveraged publicly available breast cancer datasets and employed various bioinformatics tools to identify differentially expressed DUBs in breast cancer. Our analysis identified six genes (COPS5, EIF3H, MINDY1, MINDY2, PSMD14 and USP26) with significant differential expression and survival implications. We further validated our findings experimentally and found upregulation of COPS5, EIF3H and MINDY 1 in MCF-7 and T47D breast cancer cell lines using qPCR analysis. To identify the role of these genes, EIF3H and COPS5, in disease progression, we constructed a protein-protein interaction (PPI) network with genes associated with metastasis and explored their correlation at the gene expression level in breast cancer patients. Together, this comprehensive study sheds light on DUB gene expression patterns in breast cancer with the potential to identify novel targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1182059f20169c0a82d0630262bfb5d46b482ee5" target='_blank'>
              Deubiquitinating enzymes in breast cancer: in silico analysis of gene expression and metastatic correlation.
              </a>
            </td>
          <td>
            Gaurav Sahoo, Shruti Bandyopadhyay, Ekta Tripathi, Prashanthi Karyala
          </td>
          <td>2024-12-13</td>
          <td>Journal of biomolecular structure & dynamics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Occurrence of degenerative interactions is thought to serve as a mechanism underlying hybrid unfitness in most animal systems. However, the molecular mechanisms underpinning the genetic interaction and how they contribute to overall hybrid incompatibilities are limited to only a handful of examples. A vertebrate model organism, Xiphophorus, is used to study hybrid dysfunction, and it has been shown from this model that diseases, such as melanoma, can occur in certain interspecies hybrids. Melanoma development is due to hybrid inheritance of an oncogene, xmrk, and loss of a co-evolved tumor modifier. It was recently found that adgre5, a G protein-coupled receptor involved in cell adhesion, is a tumor regulator gene in naturally hybridizing Xiphophorus species Xiphophorus birchmanni (X. birchmanni) and Xiphophorus malinche (X. malinche). We hypothesized that 1 of the 2 parental alleles of adgre5 is involved in regulation of cell growth, migration, and melanomagenesis. Accordingly, we assessed the function of adgre5 alleles from each parental species of the melanoma-bearing hybrids using in vitro cell growth and migration assays. In addition, we expressed each adgre5 allele with the xmrk oncogene in transgenic medaka. We found that cells transfected with the X. birchmanni adgre5 exhibited decreased growth and migration compared to those with the X. malinche allele. Moreover, X. birchmanni allele of adgre5 completely inhibited melanoma development in xmrk-transgenic medaka, while X. malinche adgre5 expression did not exhibit melanoma suppressive activity in medaka. These findings provide evidence that adgre5 is a natural melanoma suppressor and provide new insight in melanoma etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35473f18fdca14702fd87c87ffcf454b6421341" target='_blank'>
              Functional test of a naturally occurred tumor modifier gene provides insights to melanoma development.
              </a>
            </td>
          <td>
            Mateo Garcia-Olazabal, M. Adolfi, B. Wilde, Anita Hufnagel, Rupesh Paudel, Yuan Lu, Svenja Meierjohann, Gil Rosenthal, Manfred Schartl
          </td>
          <td>2025-01-17</td>
          <td>G3</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The memory of cellular identity is crucial for the correct development of an individual and is maintained throughout life by the epigenome. Chromatin marks, such as DNA methylation and histone modifications, ensure the stability of gene expression programmes over time and through cell division. Loss of these marks can lead to severe pathologies, including cancer and developmental syndromes. However, reprogramming of cellular identity is also a natural phenomenon that occurs early in mammalian development, particularly in the germ line, which enables the production of mature and functional gametes. The germ line transmits genetic and epigenetic information to the next generation, contributing to the survival of the species. Primordial germ cells (PGCs) undergo extensive chromatin remodelling, including global DNA demethylation and erasure of the parental imprints. This review introduces the concept of epigenetic reprogramming, its discovery and key steps, as well as the transcriptional and chromatin changes that accompany germ cell formation in mice. Finally, we discuss the epigenetic mechanisms of genomic imprinting, its discovery, regulation and relevance to human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2611fcc1b3dfdfc6647da0d47d718b5970da542" target='_blank'>
              [Epigenetic reprogramming, germline and genomic imprinting].
              </a>
            </td>
          <td>
            Clara Roidor, Karim Chebli, Maud Borensztein
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452c65f4f9faa535e4f661db53c177e8e8787cfc" target='_blank'>
              KANK1 promotes breast cancer development by compromising Scribble-mediated Hippo activation
              </a>
            </td>
          <td>
            S. S. Guo, Zhiying Liu, Guan M. Wang, Zhiqi Sun, Kaikai Yu, James P Fawcett, Reinhard Buettner, Bo Gao, Reinhard Fässler
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic integrity is critical for cellular homeostasis, preventing the accumulation of mutations that can drive diseases such as cancer. Among the mechanisms safeguarding genomic stability, the Base Excision Repair (BER) pathway plays a pivotal role in counteracting oxidative DNA damage caused by reactive oxygen species. Central to this pathway are enzymes like 8-oxoguanine glycosylase 1 (OGG1), which recognize and excise 8-oxo-7,8-dihydro-2'-deoxyguanosine (8-oxodG) lesions, thereby initiating a series of repair processes that restore DNA integrity. BER inhibitors have recently been identified as a promising approach in cancer therapy, increasing the sensitivity of cancer cells to radiotherapy and chemotherapy. By exploiting tumor-specific DNA repair dependencies and synthetic lethal interactions, these inhibitors could be used to selectively target cancer cells while sparing normal cells. This review provides a robust reference for scientific researchers, offering an updated perspective on small-molecule inhibitors targeting the 8-oxodG-BER pathway and highlighting their potential role in expanding cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e72df5cdd609ef616fe0cccc3b1963c63863d272" target='_blank'>
              Targeting the 8-oxodG Base Excision Repair Pathway for Cancer Therapy.
              </a>
            </td>
          <td>
            Anna Piscone, Francesca Gorini, Susanna Ambrosio, Anna Noviello, Giovanni Scala, Barbara Majello, Stefano Amente
          </td>
          <td>2025-01-14</td>
          <td>Cells</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/092427beef72a968e48463433a28bcb1fbe24bce" target='_blank'>
              Diversity and consequences of structural variation in the human genome.
              </a>
            </td>
          <td>
            Ryan L. Collins, M. Talkowski
          </td>
          <td>2025-01-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774b26878c155e7c22bc9f18e20f8b922e64a4e4" target='_blank'>
              Dissecting the Kaiso binding profile in clear renal cancer cells
              </a>
            </td>
          <td>
            Alexey Starshin, Pavel Abramov, Yaroslava Lobanova, Fedor Sharko, G. Filonova, Dmitry Kaluzhny, D. Kaplun, Igor Deyev, Alexander Mazur, Egor Prokchortchouk, S. Zhenilo
          </td>
          <td>2024-12-19</td>
          <td>Epigenetics & Chromatin</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amidst the prevalent and notable characteristic of a hypoxic microenvironment present in the majority of solid tumors, a burgeoning number of studies have revealed the significance of noncoding RNAs (ncRNAs) in hypoxic tumor regions. The transcriptome of cancers is highly heterogeneous, with noncoding transcripts playing crucial roles. Long noncoding RNAs (lncRNAs) and circular RNAs (circRNAs) are two distinctive classes of ncRNA that are garnering increasing attention. Biologically, they possess intriguing properties and possess significant regulatory functions. Clinically, they present as promising biomarkers and therapeutic targets. Additionally, recent research has evaluated the clinical applications of these ncRNAs in RNA-based treatments and noninvasive liquid biopsies. This review provides a comprehensive summary of recent studies on lncRNAs and circRNAs within the hypoxic tumor microenvironment. Furthermore, the clinical significance of lncRNAs and circRNAs in cancer diagnosis and treatment is emphasized, which could pave the way for the development of effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/334eb766e645372ee9515024ced95b053cc01f29" target='_blank'>
              Landscape of Noncoding RNA in the Hypoxic Tumor Microenvironment
              </a>
            </td>
          <td>
            Lianfeng Gong, Chuanxin Zou, Haixia Zhang, Fei Yang, Gui Qi, Zhaowu Ma
          </td>
          <td>2025-01-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to drugs is one of the major issues affecting the response to pharmacological treatments for tumors. Different mechanisms have been proposed to explain the development of cancer drug resistance (CDR), and several approaches to overcome it have been suggested. However, the biological basis of CDR remains unclear. Here, we investigated whether mitochondrial damage and consequent mitochondrial dysfunction are major causes of drug resistance in different tumors. To this end, we used cell lines from three tumors: hepatocellular carcinoma, breast cancer, and colon cancer. We then applied a protocol that recapitulates chemotherapy regimens in patients, rendering each cell line resistant to the drug commonly used in their respective treatments. The combination of cellular respiration analysis, gene expression analysis of cytochrome c oxidase isoforms, and mass spectrometry assessment of cardiolipin reveals that mitochondrial dysfunction is the underlying cause of the resistant phenotype. Importantly, we disclosed for the first time the rapid inhibition of oxidative phosphorylation (OXPHOS) by L-lactate, the major product of fermentation. Finally, we demonstrated that inhibition of lactic acid fermentation and activation of OXPHOS can increase drug sensitivity in all tested drug-resistant cancer cells. Taken together, our results suggest that inhibiting fermentation and enhancing mitochondrial function in cancer cells may be a concrete option to control the worrisome phenomenon of CDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb4a2eced91fb616948e3a8a82b2b795dd7c9f" target='_blank'>
              Dysfunctional mitochondrial bioenergetics sustains drug resistance in cancer cells.
              </a>
            </td>
          <td>
            D. Gnocchi, D. Nikolic, S. Russo, M. L. Matrella, R. R. Paparella, Sujeet Kumar, S. Karki, Carlo Sabbà, Tiziana Cocco, Simona Lobasso, A. Mazzocca
          </td>
          <td>2025-01-24</td>
          <td>American journal of physiology. Cell physiology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33f540ba07c7d118037674f121aaa35d22c9a7" target='_blank'>
              Prolonged persistence of mutagenic DNA lesions in somatic cells.
              </a>
            </td>
          <td>
            M. Spencer Chapman, E. Mitchell, Kenichi Yoshida, N. Williams, Margarete A. Fabre, A. Ranzoni, Philip S. Robinson, Lori D. Kregar, Matthias Wilk, Steffen Boettcher, Krishnaa Mahbubani, Kourosh Saeb Parsy, K. Gowers, Sam M Janes, Stanley W K Ng, Matt Hoare, Anthony R Green, G. Vassiliou, A. Cvejic, M. Manz, E. Laurenti, I. Martincorena, Michael R Stratton, J. Nangalia, Tim H. H. Coorens, P. J. Campbell
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="To achieve replicative immortality, cancer cells must activate telomere maintenance mechanisms. In 10 to 15% of cancers, this is enabled by recombination-based alternative lengthening of telomeres pathways (ALT). ALT cells display several hallmarks including heterogeneous telomere length, extrachromosomal telomeric repeats, and ALT-associated PML bodies. ALT cells also have high telomeric replication stress (RS) enhanced by fork-stalling structures (R-loops and G4s) and altered chromatin states. In ALT cells, telomeric RS promotes telomere elongation but above a certain threshold becomes detrimental to cell survival. Manipulating RS at telomeres has thus been proposed as a therapeutic strategy against ALT cancers. Through analysis of genome-wide CRISPR fitness screens, we identified ALT-specific vulnerabilities and describe here our characterization of the roles of SUB1, a ssDNA-binding protein, in telomere stability. SUB1 depletion increases RS at ALT telomeres, profoundly impairing ALT cell growth without impacting telomerase-positive cells. During RS, SUB1 is recruited to stalled forks and ALT telomeres via its ssDNA-binding domain. This recruitment is potentiated by RPA depletion, suggesting that these factors may compete for ssDNA. The viability of ALT cells and their resilience toward RS also requires ssDNA binding by SUB1. SUB1 depletion accelerates cell death induced by FANCM depletion, triggering unsustainable levels of telomeric damage in ALT cells. Finally, combining SUB1 depletion with RS-inducing drugs rapidly induces replication catastrophe in ALT cells. Altogether, our work identifies SUB1 as an ALT susceptibility with roles in the mitigation of RS at ALT telomeres and suggests advanced therapeutic strategies for a host of still poorly managed cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6497c277f2efbf01c25fef9e39f419c5577c8b02" target='_blank'>
              The single-stranded DNA-binding factor SUB1/PC4 alleviates replication stress at telomeres and is a vulnerability of ALT cancer cells.
              </a>
            </td>
          <td>
            Jean-Christophe Dubois, Erin Bonnell, Amélie Filion, Julie Frion, Samuel Zimmer, Muhammad Riaz Khan, Gabriela M. Teplitz, Lisa Casimir, Élie Méthot, Isabelle Marois, Mouhamed Idrissou, Pierre-Étienne Jacques, R. Wellinger, Alexandre Maréchal
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Pituitary neuroendocrine tumors (PitNETs) are unusual among neoplasms in that sporadic tumors are not typically associated with genetic mutations. Instead, epigenetics, a non-mutational process by which gene expression is modified via a variety of mechanisms, may be a driving factor in PitNET growth and behavior. DNA methylation is one of the most well understood forms of epigenetic modification. Research on DNA methylation profiles of PitNETs has identified a large number of genes silenced or upregulated by DNA methylation, particularly when methylated at CpG islands of gene promoter regions. Global patterns of DNA methylation may provide valuable insight into origins of pituitary adenoma subtypes, assist with PitNet diagnostics and have been found to correspond to the current WHO classification of PitNETs based on transcription factor lineage. Analysis of differentially methylated regions of individual genes may have prognostic value as well as guide research toward nonsurgical therapeutic strategies. Pituitary epigenetics and DNA methylation analysis are rapidly growing areas of interest with the potential to shape the future of pituitary tumor diagnostics and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91afb11897061f0dd790a1feb3aca845da9bfbb" target='_blank'>
              Current understanding of the role of DNA methylation in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Racheal Peterson, David J Cote, G. Zada
          </td>
          <td>2025-01-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is the sixth most common malignancy in women and the poor 5-year survival emphasises the need for novel therapies. NK cells play an important role in the control of malignant disease but the nature of tumour-infiltrating and peripheral NK cells in OC remains unclear. Methods Using flow cytometric analysis, we studied the phenotype and function of NK cells in blood, primary tumour and metastatic tissue in 80 women with OC. The cell type contexture of metastatic OC tissue was explored utilising scRNAseq analysis, with a focus on portraying an immunogenic tumour microenvironment and determining the characteristics of a dysfunctional NK cell population. Results The proportion of peripheral NK cells was markedly elevated with a highly activated profile and increased cytotoxicity. In contrast, NK cell numbers in primary tumour and metastasis were substantially reduced, with downregulation of activatory receptors together with elevated PD-1 expression. scRNA-Seq identified 5 NK cell subpopulations along with increased exhausted and immature NK cells within tumour tissue compared to normal tissue. These features were attenuated following chemotherapy where higher levels of activated and cytotoxic NK cells associated with improved disease-free survival. Correlation of NK cell phenotype with clinical outcomes revealed high levels of DNAM-1 expression on tissue-localised and peripheral NK cells to be associated with reduced survival. Expression of PVR, the DNAM-1 ligand, was significantly increased on tumours and DNAM-1 mediated NK cell lysis of primary tumour tissue was observed in vitro. Discussion These findings reveal profound modulation of the tumour tissue and systemic profile of NK cells which likely contributes to the high rates of local progression and metastasis seen with OC. Immunotherapeutic approaches that overcome local immune suppression and enhance DNAM-1-targeted lysis of OC offer the potential to improve disease control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec3263f862b7dea4d3ddaf31f9b8ccd7a6ea974" target='_blank'>
              The emergence of DNAM-1 as the facilitator of NK cell-mediated killing in ovarian cancer
              </a>
            </td>
          <td>
            R. Pounds, W. Croft, Hayden Pearce, Tasnia Hossain, Kavita Singh, J. Bálega, D. Jeevan, S. Sundar, S. Kehoe, J. Yap, Paul Moss, J. Zuo
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Long non-coding RNA (lncRNA) is a type of non-coding RNA distinguished by a length exceeding 200 nucleotides. Recent studies indicated that lncRNAs participate in various biological processes, such as chromatin remodeling, transcriptional and post-transcriptional regulation, and the modulation of cell proliferation, death, and differentiation, hence influencing gene expression and cellular function. ADAMTS9-AS1, an antisense long non-coding RNA situated on human chromosome 3p14.1, has garnered significant interest due to its pivotal involvement in the advancement and spread of diverse malignant tumors. ADAMTS9-AS1 functions as a competitive endogenous RNA (ceRNA) that interacts with multiple microRNAs (miRNAs) and plays a crucial role in regulating gene expression and cellular functions by modulating essential signaling pathways, including PI3K/AKT/mTOR, Wnt/β-catenin, and Ras/MAPK pathways. Dysregulation of this factor has been linked to tumor development, migration, invasion, and resistance to apoptotic mechanisms, including as iron-induced apoptosis, underscoring its intricate function in cancer pathology. While current research has clarified certain pathways involved in cancer formation, additional clinical and in vivo investigations are necessary to enhance comprehension of its specific involvement across various cancer types. This review encapsulates the recent discoveries on the correlation of ADAMTS9-AS1 with numerous malignancies, clarifying its molecular mechanisms and its prospective role as a therapeutic target in oncology. Furthermore, it identifies ADAMTS9-AS1 as a potential early diagnostic biomarker and therapeutic target, offering novel opportunities for targeted intervention in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd8babd07e146bd73db5bdfb569c7497c29a95ac" target='_blank'>
              The Dual Role of ADAMTS9-AS1 in Various Human Cancers: Molecular Pathogenesis and Clinical Implications.
              </a>
            </td>
          <td>
            Haodong He, Jingjie Yang, Yan Zhou, Xinyan Zheng, Lihan Chen, Zhujun Mao, Chuyuan Liao, Tongtong Li, Haoran Liu, Gang Zhou, Houdong Li, Chengfu Yuan
          </td>
          <td>2025-01-03</td>
          <td>Anti-cancer agents in medicinal chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="SET domain bifurcated histone lysine methyltransferase 1 (SETDB1/ESET), a pivotal H3K9 methyltransferase, has been extensively studied since its discovery over two decades ago. SETDB1 plays critical roles in immune regulation, including B cell maturation, T-cell activity modulation, and endogenous retrovirus (ERV) silencing. While essential for normal immune cell function, SETDB1 overexpression in cancer cells disrupts immune responses by suppressing tumor immunogenicity and facilitating immune evasion. This is achieved through the repression of anti-tumor immune cell production, ERV silencing, and interference with the type I interferon pathway leading to inhibiting immune checkpoint blockade (ICB) efficacy. Beyond its immunological implications, SETDB1 overexpression fosters tumor growth and metastasis via transcriptional silencing of tumor suppressor genes through histone regulation and activating oncogenic signaling by non-histone regulation. These multifaceted roles make SETDB1 an attractive epigenetic target for novel cancer therapies. This review explores SETDB1's dual function in immune regulation and tumor progression, emphasizing its potential in the development of innovative cancer treatments targeting epigenetic dysregulation and oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a673433d5768545e8da0592cd5b401ba83b5720" target='_blank'>
              Targeting SETDB1 in cancer and immune regulation: Potential therapeutic strategies in cancer.
              </a>
            </td>
          <td>
            Bo-Syong Pan, Cheng‐Yu Lin, Gilbert Aaron Lee, Hui-Kuan Lin
          </td>
          <td>2025-01-07</td>
          <td>The Kaohsiung journal of medical sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="While optimizing cancer cell growth conditions, we discovered that cancer stem cells can generate second membranes outside the plasma membranes forming compartments separated from the extracellular matrix. The encapsulating membranes can extend and generate long cytocapsular tubes, wherein multiple cells can migrate. SILAC proteomics of the second cytocapsular membranes identified 400 membrane proteins, and a small subset of them are highly upregulated in cytocapsular cancers compared to normal tissues. The ATP-dependent calcium pump PMCA2 is one of the highest upregulated factors of the cytocapsular membrane, and antibodies serve as biomarkers for malignant tumors, as checked for 293 subtypes of cancers. Cytocapsular tumors have not been described before, possibly because the CC membranes do not exhibit epitopes targeted by conventional methods, and no efforts have been made to search for new cancer specific organelles. Antibodies against PMCA2 can now be used to map cancer evolution pathways in human bodies by comparisons of more than 12 000 annotated specimens from tissue banks worldwide. The current research reveals that the native malignant cancer cell is enclosed in a cytocapsular membrane. With the PMCA2 cancer biomarker available, the development of human cancers can be studied from cancer tissue banks and clinical cancer biopsies with a previously unknown diversity. The emerging knowledge on cancer-driving biomarkers opens doors for new routes in cancer diagnosis, surgery, therapy, and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4e5b795160ba02ecb2832797cada359373d1df" target='_blank'>
              Discovery of the Cytocapsular Membrane as Hallmark of Malignant Tumors.
              </a>
            </td>
          <td>
            Tingfang Yi, G. Wagner
          </td>
          <td>2024-12-16</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: The progression of colorectal cancer through clinically and histopathologically well-defined stages is driven by specific mutations that activate oncogenes or inactivate tumor-suppressor genes. In addition, pre-cancerous/cancer cells respond to cues from the tissue microenvironment that support tumorigenesis and progression, many of which are transmitted through integrin receptors for the extracellular matrix. Integrin α3β1 has pro-tumorigenic/pro-metastatic roles in many cancers, but it also has suppressive roles in some cancers or at specific stages of progression, indicating that its potential value as a therapeutic target cannot be extrapolated across cancer types or stages. In this study, we investigated roles for α3β1 in colorectal cancer using cellular and genetic models that represent different stages. Methods: We generated mice with colon-specific α3 knockout in a tamoxifen-inducible model of KRAS-mutated colorectal cancer to assess the effects of α3β1 ablation on early dysplasia. We also used siRNA to suppress α3β1 in human colorectal cancer cells, then assessed effects on motility and invasion in vitro. Results: Genetic deletion of α3β1 in the colon did not alter dysplasia in mice predisposed to KRAS-mutated colorectal cancer, and it was accompanied by an increase in the colocalization of α6 integrin with laminin-332 (a matrix ligand for both integrins), suggesting functional compensation. However, suppression of α3β1 caused an approximately 40% to 60% reduction in the motility/invasion of human colorectal cancer cells. Conclusions: Our findings that α3β1 is not required for pre-cancerous dysplasia but promotes colorectal cancer cell motility/invasion indicate an important role for pro-migratory functions of this integrin at later stages of progression when cells invade from the primary tumor, suggesting that strategies to target α3β1 in colorectal cancer should be aimed at distinct stages of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af466585d725fe3d69feef187a621b0cba9e5fc" target='_blank'>
              Integrin α3β1 Is Not Required for Onset of Dysplasia in Genetic Model of Colon Cancer but Promotes Motility of Colon Cancer Cells
              </a>
            </td>
          <td>
            Kathryn E. Ottaviano, Sita Subbaram, Lei Wu, Kiley Stahl, Antoinette J. Mastrangelo, Hwajeong Lee, C. DiPersio
          </td>
          <td>2025-01-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The progression of colorectal cancer is strongly influenced by environmental and genetic conditions. One of the key factors is tumor heterogeneity which is extensively studied by cfDNA and bulk sequencing methods; however, we lack knowledge regarding its effects at the single-cell level. Motivated by this, we aimed to employ an end-to-end single-cell sequencing workflow from tissue-derived sample isolation to exome sequencing. Our main goal was to investigate the heterogeneity patterns by laser microdissecting samples from different locations of a tissue slide. Moreover, by studying healthy colon control, tumor-associated normal, and colorectal cancer tissues, we explored tissue-specific heterogeneity motifs. For completeness, we also compared the performance of the whole-exome bulk, cfDNA, and single-cell sequencing methods based on variation at the level of a single nucleotide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9266548dfc591e54ed5a1417777c06bd133c4150" target='_blank'>
              Investigation of Exome-Wide Tumor Heterogeneity on Colorectal Tissue-Based Single Cells.
              </a>
            </td>
          <td>
            Nikolett Szakállas, A. Kalmár, B. Barták, Z. Nagy, Gábor Valcz, Tamás Richárd Linkner, Kristóf Róbert Rada, István Takács, Béla Molnár
          </td>
          <td>2025-01-16</td>
          <td>International journal of molecular sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract N4‐acetylcytidine (ac4C) modification is a crucial RNA modification widely present in eukaryotic RNA. Previous studies have demonstrated that ac4C plays a pivotal role in viral infections. Despite numerous studies highlighting the strong correlation between ac4C modification and cancer progression, its detailed roles and molecular mechanisms in normal physiological processes and cancer progression remain incompletely understood. This review first outlines the key regulatory enzyme mediating ac4C modification, N‐acetyltransferase 10 (NAT10), including its critical roles in regulating RNA stability, transcriptional efficiency, and translational fidelity. Additionally, it systematically summarizes the essential functions and mechanisms of ac4C modification in normal biological processes, including stem cell fate determination, spermatogenesis and oogenesis, embryonic development, cellular senescence, and bone remodeling. Furthermore, this review delves into the central roles and molecular mechanisms of ac4C modification in regulating malignant proliferation, cell cycle arrest, EMT, drug resistance, cell death, cancer metabolism, and tumor immunotherapy. It also emphasizes the potential of NAT10 as a prognostic biomarker and its therapeutic potential as a target for disease treatment. In summary, this review clarifies the multifaceted roles of ac4C modification in both health and disease and explores NAT10‐targeted therapies with the aim of advancing cancer research and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207de8afe077c354fcc57810c217c15a2fb1cd47" target='_blank'>
              RNA N4‐acetylcytidine modification and its role in health and diseases
              </a>
            </td>
          <td>
            Qiang Wang, Yixiao Yuan, Qiang Zhou, Yingdong Jia, Jing Liu, GuangJun Xiao, Chunhong Li, Xiulin Jiang
          </td>
          <td>2025-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DREAM (dimerization partner, RB-like, E2F, and multi-vulval class B) complex is an evolutionarily conserved transcriptional repression complex that coordinates nearly one thousand target genes, primarily associated with the cell cycle processes. The formation of the DREAM complex consequently inhibits cell cycle progression and induces cellular quiescence. Given its unique role in cell cycle control, the DREAM complex has gained significant interest across various physiological and pathological contexts, particularly in conditions marked by dysregulated cell cycles, such as cancer. However, the specific cancer types most significantly affected by alterations in the DREAM complex are yet to be determined. Moreover, the possibility of restoring or pharmacologically targeting the DREAM complex as a therapeutic intervention against cancer remains a relatively unexplored area of research and is currently under active investigation. In this review, we provide an overview of the latest advances in understanding the DREAM complex, focusing on its role in cancer. We also explore strategies for targeting the DREAM complex as a potential approach for cancer therapeutics. Advances in understanding the precise role of the DREAM complex in cancer, combined with ongoing efforts to develop targeted therapies, may pave the way for new options in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f83080159a75138829f6897856a0dcb6a27d73" target='_blank'>
              Emerging Role of the DREAM Complex in Cancer and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Ye-Jin Hwang, Moon Jong Kim
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Infections have imposed strong selection pressures throughout human evolution, making the study of natural selection's effects on immunity genes highly complementary to disease-focused research. This review discusses how ancient DNA studies, which have revolutionized evolutionary genetics, increase our understanding of the evolution of human immunity. These studies have shown that interbreeding between modern humans and Neanderthals or Denisovans has influenced present-day immune responses, particularly to viruses. Additionally, ancient genomics enables the tracking of how human immunity has evolved across cultural transitions, highlighting strong selection since the Bronze Age in Europe (<4,500 years) and potential genetic adaptations to epidemics raging during the Middle Ages and the European colonization of the Americas. Furthermore, ancient genomic studies suggest that the genetic risk for noninfectious immune disorders has gradually increased over millennia because alleles associated with increased risk for autoimmunity and inflammation once conferred resistance to infections. The challenge now is to extend these findings to diverse, non-European populations and to provide a more global understanding of the evolution of human immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b729418da43553d3cfa26e68fd325ebc987180" target='_blank'>
              Tracing the Evolution of Human Immunity Through Ancient DNA.
              </a>
            </td>
          <td>
            E. Patin, L. Quintana-Murci
          </td>
          <td>2024-12-20</td>
          <td>Annual review of immunology</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a01f4d4fe80d230f00a3e911e4c3749e3d5d282" target='_blank'>
              Chromosome-scale genome assembly and gene annotation of the Alligator Gar (Atractosteus spatula)
              </a>
            </td>
          <td>
            Qing Wang, Qianqian Yu, Xiangqian Dong, Hengjin Chen, Xue Tian, Peng Qi, Haitao Wu, Yuxiang Yuan
          </td>
          <td>2024-12-18</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04e5297d8cec280df0e89a9099a47d824b54a4d" target='_blank'>
              Molecular leveraging of HOX-embedded non-coding RNAs in the progression of acute myeloid leukemia.
              </a>
            </td>
          <td>
            C. Wilson, Priyanka Swaroop, Sachin Kumar, Anita Chopra, S. Sharawat
          </td>
          <td>2024-11-30</td>
          <td>Human cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Bladder cancer is one of the most common malignancies of the urinary system and has always presented great challenges in treatment due to its intricate biological features and high recurrence rates. Although great developments were achieved in immunotherapy and targeted therapies within the last decade, therapeutic outcomes for a great number of patients remain unsatisfactory, particularly as to long-term efficacy. Review discusses the molecular mechanisms developed during the process of bladder cancer progression: genetic and epigenetic alterations, dynamics of the tumor microenvironment (TME), and dysregulation and abnormal activation of various signaling pathways—all contributing to therapeutic resistance. It is genetic mutation, especially in both low- and high-grade tumors, that, alongside epigenetic modifications, plays a considerable role in tumor aggressiveness and drug resistance. TME, comprising cancer-associated fibroblasts (CAFs), immunosuppressive cells, and different components of the extracellular matrix (ECM), orchestrates a setting that fosters tumor growth and immune evasion and confers resistance on any therapeutic regime that might be used. The review also provides an overview of PI3K/AKT and MAPK signaling pathways in the progression of bladder cancer and the development of targeted therapies against them. Further, it discusses the challenges and mechanisms of resistance to immunotherapy, including those involving immune checkpoint inhibitors. Other promising approaches include the development of new therapeutic strategies that target not only the signaling pathways but also immune checkpoints in combination therapies. This review aims to contribute to the elaboration of more effective and personalized treatment strategies by fully understanding the underlying mechanisms involved in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee79f09b15911cd50c65c34f5cba7cd83105949" target='_blank'>
              Redefining bladder cancer treatment: innovations in overcoming drug resistance and immune evasion
              </a>
            </td>
          <td>
            Peng Huang, Jie Wang, Zongze Yu, Jiaan Lu, Zhou Sun, Zhigui Chen
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive cancer with distant metastasis. Accumulated evidence has demonstrated that exosomes are involved in TNBC metastasis. Elucidating the mechanism underlying TNBC metastasis has important clinical significance. Extracellular vesicles (EVs) present a promising avenue for diagnosing and treating triple-negative breast cancer (TNBC) through a technique called "liquid biopsy," offering a new wellspring of biomarkers. These tiny lipid bilayer vesicles, released by most cells, carry a diverse array of RNA molecules that can influence the behaviour of recipient cells. Among these, circular RNAs (circRNAs) have emerged as a subtype of noncoding RNAs capable of modulating gene expression by sponging microRNAs, thus playing crucial roles in various aspects of cancer development and progression, including TNBC. Despite their significance, our understanding of circRNAs involvement in TNBC remains incomplete. However, studies have shown that circRNAs are abundant in EVs, with exosomal circRNAs (exo-circRNAs) particularly influential in cancer biology. These exo-circRNAs can be taken up by neighboring or distant cells, impacting numerous aspects of their physiological states, thereby enhancing cell communication and tumor dissemination. This review provides an overview of EVs key characteristics and functions before delving into exo-circRNAs potential roles in driving or suppressing TNBC, as well as their implications for cancer diagnosis, prognosis, and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bbb3d750dda1ec3bc792d436bc16135579839b9" target='_blank'>
              CircRNAs in extracellular vesicles associated with triple-negative breast cancer.
              </a>
            </td>
          <td>
            A. A. Qurtam
          </td>
          <td>2025-01-12</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc7636cc7694bca6c468bc6e4c750272c2be1508" target='_blank'>
              SPRTN metalloprotease participates in repair of ROS-mediated DNA-protein crosslinks
              </a>
            </td>
          <td>
            Luke Erber, Arnold Scott Groehler IV, Cesar I. Cyuzuzo, Jahan Baker-Wainwright, Reeja S. Maskey, Lei Li, Yuichi J. Machida, Natalia Tretyakova
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Senescence is a tumor suppressor mechanism triggered by oncogene expression and chemotherapy treatment. It orchestrates a definitive cessation of cell proliferation through the activation of the p53-p21 and p16-Rb pathways, coupled with the compaction of proliferative genes within heterochromatin regions. Some cancer cells have the ability to elude this proliferative arrest but the signaling pathways involved in circumventing senescence remain to be characterized. We have recently described that malignant cells capable of evading senescence have an increased expression of specific tRNAs, such as tRNA-Leu-CAA and tRNA-Tyr-GTA, alongside the activation of their corresponding tRNA ligases, namely LARS and YARS. We have previously shown that YARS promotes senescence escape by activating proliferation and cell cycle genes but its functions during this proliferative arrest remain largely unknown. In this study, we have continued to characterize the functions of YARS, describing non-canonical transcriptional functions of the ligase. Our results show that YARS is present in the nucleus of proliferating and senescent cells and interacts with the Trim28 transcriptional regulator. Importantly, YARS binds to the LIN9 promoter, a critical member of the Dream complex responsible for regulating cell cycle gene transcription. The ligase facilitates the binding and the phosphorylation of the type II RNA polymerase and promotes the deposition of activating epigenetic marks on the LIN9 promoter. Consequently, during senescence escape, YARS activates LIN9 expression and both proteins are necessary to induce the proliferation of emergent cells. These results underscore unconventional transcriptional functions of YARS in activating LIN9 expression in proliferating cells and during senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c36e3380440225cd14c09aec61c163d6f628c872" target='_blank'>
              A non-canonical role for the tyrosyl tRNA synthetase: YARS regulates senescence induction and escape and controls the transcription of LIN9.
              </a>
            </td>
          <td>
            H. Coquelet, Geraldine Leman, Amine Maarouf, Coralie Petit, Bertrand Toutain, Cécile Henry, Alice Boissard, Catherine Guette, Eric Lelièvre, Pierre Vidi, Jordan Guillon, O. Coqueret
          </td>
          <td>2025-01-05</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to "intrinsic resistance", e.g., acquired by random mutations, cancer cells typically escape from certain treatments ("acquired resistance") by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies.
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Integration of DNA replication with DNA repair, cell cycle progression, and other biological processes is crucial for preserving genome stability and fundamentally important for all life. Ataxia-telangiectasia mutated and RAD3-related (ATR) and its partner ATR-interacting protein (ATRIP) function as a critical proximal sensor and transducer of the DNA Damage Response (DDR). Several ATR substrates, including p53 and CHK1, are crucial for coordination of cell cycle phase transitions, transcription, and DNA repair when cells sustain DNA damage. While much is known about ATR activation mechanisms, it is less clear how ATR signaling is negatively regulated in cells. Here, we identify the DNA repair protein REV7 as a novel direct binding partner of ATRIP. We define a REV7-interaction motif in ATRIP, which when mutated abrogates the REV7-ATRIP interaction in vitro and in intact cells. Using in vitro kinase assays, we show that REV7 inhibits ATR-mediated phosphorylation of its substrates, including p53. Disruption of the REV7-ATRIP interaction also enhances phosphorylation of CHK1 at Ser317 (a known ATR target site) in intact cells. Taken together our results establish REV7 as a critical negative regulator of ATR signaling. REV7 has pleiotropic roles in multiple DDR pathways including Trans-Lesion Synthesis, DNA Double-Strand Break resection, and p53 stability and may play a central role in the integration of multiple genome maintenance pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728ca1439c5470e6d387db9f0926461c6798d50b" target='_blank'>
              REV7 associates with ATRIP and inhibits ATR kinase activity
              </a>
            </td>
          <td>
            Megan Biller, Sara Kabir, Sarah Nipper, Sydney Allen, Yara Kayali, Skyler Kuncik, Hiroyuki Sasanuma, Pei Zhou, Cyrus Vaziri, Junya Tomida
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are non-coding RNA (ncRNA) molecules that, due to their covalent ring structure and lack of free ends, have a very high intracellular stability compared to their linear counterparts. In general, circRNAs are expressed in mammalian cells and exhibit tissue/cell-specific expression patterns. Mounting evidence is indicative that circRNAs regulate a variety of cellular processes by acting as miRNA sponges, transcriptional regulators, protein sponges, molecular scaffolds, and protein/peptide translators. The emergence of the biological functions of circRNAs has brought a novel outlook to our better understanding of cellular physiology and disease pathogenesis. CircRNAs have also been shown to play a critical role in the occurrence, development and progression of cancers. Their participation in the pathophysiology of various diseases including cardiovascular diseases, diabetes and neurological disorders is very important. Such characteristics have led to more studies investigating circRNAs as promising tools in molecular medicine and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b727ea7526fd99e7ae1cdc4440e60473676ede8" target='_blank'>
              Circular RNAs: history, metabolism, mechanisms of function, and regulatory roles at a glance
              </a>
            </td>
          <td>
            Roqaye Karimi, Ehsan Javandoost, Nooshin Asadmasjedi, Amir Atashi, Alireza Soleimani, Mahin Behzadifard
          </td>
          <td>2024-12-19</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Long non-coding RNA Rhabdomyosarcoma 2-associated Transcript (RMST) is a crucial regulator in various biological processes, particularly in neurogenesis and cancer progression. This review summarizes current knowledge on structure, expression patterns, and functional roles across different organs and diseases of RMST. RMST exhibits tissue-specific expression, notably in brain tissues and vascular endothelial cells, and plays a significant role in neuronal differentiation through interaction with SRY-box 2 (SOX2). In cancer, RMST predominantly functions as a tumor suppressor, with context-dependent roles observed across different cancer types. RMST is also implicated in neurological disorders, cardiovascular diseases, and Hirschsprung's disease. Mechanistically, RMST acts as a competing endogenous RNA and a transcriptional regulator, interacting with various microRNAs and proteins to modulate gene expression. The potential of RMST as a biomarker and therapeutic target is increasingly recognized, particularly in atherosclerosis and cancer. While current findings are promising, further research is needed to fully elucidate the functions and translate these insights into clinical applications of RMST. This review underscores the significance of RMST in cellular processes and disease pathogenesis, highlighting its potential as a novel target for diagnostic and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274785be6ba4141ed74a6cfcddf7cc469ab67f46" target='_blank'>
              RMST: A long noncoding RNA involved in cancer and disease.
              </a>
            </td>
          <td>
            Hidenori Tani
          </td>
          <td>2024-11-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most lethal and aggressive breast cancer among all the breast cancer subtypes. Despite several attempts, to date, there is an extensive lack of therapeutic intervention. Hence, there is a dire need for an effective biomarker to timely diagnose TNBC. Here, utilizing super-resolution microscopy, the remodeling structural aspects of euchromatin and heterochromatin in TNBC are studied and the results are compared with non-cancerous and non-TNBC cell lines. The nanoscopic visualization reveals a distinct difference in chromatin remodeling in TNBC in comparison to the other two cell lines. While the euchromatin density is found to increase, the heterochromatin is found to decrease. A complete switching of the heterochromatin-euchromatin ratio is observed in TNBC cells thus proposing that chromatin remodeling and chromatin morphological changes can be pursued as one of the targets for diagnostic purposes. Increased expression of structure specific recognition protein-1(SSRP-1) protein supports the increased rate of chromatin remodeling in breast cancer cell lines. The results may lead to developing a new strategy for diagnosing TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43773d49f46d80ebb65965ee2d14a82a4ab43944" target='_blank'>
              Chromatin Marks H3K4me3 and H3K9me3 in Triple-Negative Breast Cancer Cell Lines.
              </a>
            </td>
          <td>
            Farhan Anjum, Kush Kaushik, Abdul Salam, C. K. Nandi
          </td>
          <td>2025-01-22</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epigenetics which examines the regulation of genes without modification of the DNA sequence, plays a crucial role in various biological processes and disease mechanisms. Among the different forms of epigenetic modifications, histone post-translational modifications (PTMs) are important for modulating chromatin structure and gene expression. Aberrant levels of histone PTMs are implicated in a wide range of diseases, including cancer, making them promising targets for biomarker discovery and therapeutic intervention. In this context, blood, tissues, or cells serve as valuable resources for epigenetic research and analysis. Traditional methods such as mass spectrometry and western blotting are widely used to study histone PTMs, providing qualitative and (semi)quantitative information. However, these techniques often face limitations that could include throughput and scalability, particularly when applied to clinical samples. To overcome these challenges, we developed and validated 13 Nu.Q® immunoassays to detect and quantify specific histone PTM-nucleosomes from K2EDTA plasma samples. Then, we tested these assays on other types of samples, including chromatin extracts from frozen tissues, as well as cell lines and white blood cells Our findings demonstrate that the Nu.Q® assays offer high specificity, sensitivity, precision and linearity, making them effective tools for epigenetic profiling. A comparative analysis of HeLa cells using mass spectrometry, Western blot, and Nu.Q® immunoassays revealed a consistent histone PTMs signature, further validating the effectiveness of these assays. Additionally, we successfully applied Nu.Q® assays across various biological samples, including human tissues from different organs and specific white blood cell subtypes, highlighting their versatility and applicability in diverse biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea54269010715f8d7295e217ed70c9663ed44c1" target='_blank'>
              High-Throughput Epigenetic Profiling Immunoassays for Accelerated Disease Research and Clinical Development
              </a>
            </td>
          <td>
            P. Van den Ackerveken, Clotilde Hannart, Dorian Pamart, Robin Varsebroucq, Marion Wargnies, Olivia Thiry, Marie Lurkin, Séverine Vincent, Muriel Chapelier, Guillaume Rommelaere, M. Herzog
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Reverse-transcribing viruses (RTVs) characterized by reverse transcription required for their replication infect nearly all the eukaryotes. After decades of extensive analyses and discoveries, the understanding of the diversity of RTVs has largely stagnated. Herein, we discover a previously neglected lineage of RTVs, designated Kuafuorterviruses, in animals. Through screening over 8000 eukaryote genomes, we identify the presence of endogenous Kuafuorterviruses in the genomes of 169 eumetazoans dispersed across 11 animal phyla. Phylogenetic analyses and sequence similarity networks indicate that Kuafuorterviruses constitute a novel major lineage of RTVs. The discovery of Kuafuorterviruses refines our understanding of the diversity, evolution, and classification of RTVs and has implications in annotating animal genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09fd4b26c457880f808962c9907b8488777fe4fe" target='_blank'>
              Kuafuorterviruses, a novel major lineage of reverse-transcribing viruses
              </a>
            </td>
          <td>
            Zhen Gong, Guan-Zhu Han
          </td>
          <td>2024-12-19</td>
          <td>Virus Evolution</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Cancer research has historically focused on the somatic mutation theory, viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information-processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking cancer evolution: from genetic mutations to complex information systems in tumor reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lung cancer is one of the most common cancers in humans. However, there is still a need to understand the underlying mechanisms of a normal cell developing into a cancer cell. Here, we develop the chromosome dynamic structural model and quantify the important characteristics of the chromosome structural ensemble of the normal lung cell and the lung cancer A549 cell. Our results demonstrate the essential relationship among the chromosome ensemble, the epigenetic marks, and the gene expressions, which suggests the linkage between chromosome structure and function. The analysis reveals that the lung cancer cell may have a higher level of relative ensemble fluctuation (micro CFI) and a higher degree of phase separation between the two compartments than the normal lung cell. In addition, the significant conformational "switching off" events (from compartment A to B) are more than the significant conformational "switching on" events during the lung cancerization. We identify "nucleation seeds" or hot spots in chromosomes, which initiate the transitions and determine the mechanisms. The hot spots and interaction network results reveal that the lung cancerization process (from normal lung to A549) and the reversion process have different mechanisms. These investigations have revealed the cell fate determination mechanism of the lung cancer process, which will be helpful for the further prevention and control of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ced4f0cd9c3dbc1395b811ded415faaab1d4100" target='_blank'>
              Uncovering the lung cancer mechanisms through the chromosome structural ensemble characteristics and nucleation seeds.
              </a>
            </td>
          <td>
            Wen-Ting Chu, Jin Wang
          </td>
          <td>2024-12-11</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Noncoding RNA 886 has emerged as a pivotal regulatory RNA with distinct functions across tissues, acting as a regulator of protein activity by directly binding to protein partners. While it is well recognized as a tumor suppressor in prostate cancer, the underlying molecular mechanisms remain elusive. To discover the principal pathways regulated by nc886 in prostate cancer, we used a transcriptomic and proteomic approach analyzing malignant DU145, LNCaP, PC3, and benign RWPE-1 prostate cell line models transiently transfected with in vitro transcribed nc886 or antisense oligonucleotides. Multiomics revelead a significant enrichment of immune system-related pathways across the cell lines, including cytokines and interferon signaling. The interferon response provoked by nc886 was validated by functional assays. The invariability of PKR phosphorylation and NF-κB activity in the gain/loss of nc886 function experiments and the positive regulation of innate immunity suggest a PKR-independent mechanism of nc886 action. Accordingly, the GSEA of the PRAD-TCGA data set revealed immune stimulation as the nc886 most associated node also in the clinical setting. Our study showed that the reduction of nc886 leads to a blunted immune response in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f4cce51821d8413b62bbd710ee2168d2423850" target='_blank'>
              Multi-Omics Study Reveals Nc886/vtRNA2-1 as a Positive Regulator of Prostate Cancer Cell Immunity.
              </a>
            </td>
          <td>
            Carolina Oliveira-Rizzo, Camilla L Colantuono, Ana J Fernández-Alvarez, Graciela L Boccaccio, Beatriz Garat, J. Sotelo-Silveira, Shahbaz Khan, V. Ignatchenko, Yong Sun Lee, T. Kislinger, Stanley K Liu, R. Fort, M. A. Duhagon
          </td>
          <td>2024-12-26</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Stemness-associated cell states are linked to chemotherapy resistance in AML. We uncovered a direct mechanistic link between expression of the stem cell transcription factor GATA2 and drug resistance. The GATA-binding protein 2 (GATA2) plays a central role in blood stem cell generation and maintenance. We find substantial intra- and inter-patient variability in GATA2 expression across AML patient samples. GATA2 expression varies by molecular subtype and has been linked to outcome. In a murine model, KMT2A-MLL3 driven AML originating from a stem cell or immature progenitor cell population have higher Gata2 expression and are more resistant to the standard AML chemotherapy agent doxorubicin. Deletion of Gata2 resulted in more robust induction of p53 following exposure to doxorubicin. ChIP-Seq, RNA-Seq and functional studies revealed that GATA2 regulates the expression of RASSF4, a modulator of the p53 inhibitor MDM2. GATA2 and RASSF4 are anti-correlated in human cell lines and AML patient cell bulk and single cell expression datasets. Knockdown of Rassf4 in Gata2 low cells resulted in doxorubicin or nutlin-3 resistance. Conversely, overexpression of Rassf4 results in sensitization of cells expressing high levels of Gata2. Finally, doxorubicin and nutlin-3 are synergistic in Gata2-high murine AML, as well as AML patient samples. We discovered a previously unappreciated role for GATA2 in dampening p53-mediated apoptosis via transcriptional regulation of RASSF4, a modulator of MDM2. This role for GATA2 directly links the expression of a stemness associated transcription factor to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e4d0de77d706d929171b654f103344701ea6a" target='_blank'>
              GATA2 links stemness to chemotherapy resistance in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Fatemeh Alikarami, Hongbo M Xie, Simone S. Riedel, Haley Goodrow, D. Barrett, Leila Mahdavi, Alexandra Lenard, Changya Chen, T. Yamauchi, Etienne Danis, Zhendong Cao, Vu L. Tran, M. Jung, Yapeng Li, Hua Huang, Junwei Shi, Kai Tan, D. Teachey, E. Bresnick, Tobias Neff, Kathrin M Bernt
          </td>
          <td>2025-01-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [4, 2],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>