<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-01 09:37:20 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>43</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>88</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>122</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>385</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>336</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres and segregate stochastically during cell division3–6 yet persist over many generations. EcDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei over 40 years ago with unknown mechanism3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. We develop Retain-seq, a genome-scale assay that reveals thousands of human retention elements conferring generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromatin conformation capture show that retention elements physically interact with mitotic chromosomes at regions which are mitotically bookmarked by transcription factors and chromatin proteins, intermolecularly recapitulating promoter-enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated; targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, suggesting that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, revealing the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells.
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, , J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Oncogenesis involves widespread genetic and epigenetic alterations, yet the full spectrum of somatic variation genome-wide remains unresolved. We generated a near-telomere-to-telomere (T2T) diploid assembly of a donor paired with deep short- and long-read sequencing of their melanoma. This revealed that 16% of somatic variants occur in sequences absent from GRCh38, with satellite repeats acting as hotspots for UV-induced damage due to sequence-intrinsic mutability and inefficient repair. Centromere kinetochore domains emerged as focal sites of structural, genetic, and epigenetic variation, leading to remodeling of centromere kinetochore binding domains during tumor evolution. Single-molecule telomere reconstructions uncovered cycles of attrition, deletion, and telomerase-mediated extension that shape cancer telomeres. Finally, diploid chromatin maps exposed that copy number alterations and epimutations, rather than point mutations, predominate in rewiring cancer regulatory programs. These findings define the full landscape of a cancer’s somatic variation and their functional impact, establishing a blueprint for T2T studies of mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Aneuploidy, an abnormal number of chromosomes, is a hallmark of cancer and has been proposed as an initiating event in tumorigenesis. In glioblastoma (GBM), a highly aggressive brain tumor, cells almost universally display gain of chromosome 7 and loss of chromosome 10. However, it remains unclear whether these alterations arise de novo during malignant transformation or reflect pre-existing chromosomal instability in normal brain tissue. Here, we used single-nucleus whole-genome sequencing (snWGS) on 225 NeuN-negative (non-neuronal) cortical nuclei from 12 healthy individuals and 6 GBM patients, including matched tumor cores and non-tumor brain regions. In healthy brains, approximately 15% of glial nuclei harbored somatic aneuploidies, most often involving chromosome arms, with recurrent 16p alterations detected in up to 3% of nuclei from both healthy controls and GBM non-tumor tissue. These findings establish 16p is a hotspot of structural variation in adult glia. Non-tumor regions in GBM patients closely resembled healthy controls in aneuploidy burden and chromosomal instability metrics and lacked hallmark tumor alterations. In contrast, GBM tumors exhibited significantly elevated aneuploidy (~50%), enrichment for canonical chromosomal instability-driven events, and sex-specific karyotype patterns, consistent with transformation-associated chromosomal instability. Thus, aneuploidy is a recurrent but constrained feature of normal adult glia, whereas chromosome instability and GBM-defining aneuploidies emerge only during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ca098d5f35d207827ccedcf67f15fa391301a1" target='_blank'>
              The Somatic Aneuploidy Landscape of Adult Glia Reveals 16p as a Hotspot and Differentiates Mosaicism in Normal Glia from Chromosomal Instability in Glioblastoma
              </a>
            </td>
          <td>
            Cristina Montagna, Olivia Albert, Shixiang Sun, Jhih-Rong Lin, Moonsook Lee, Chang Chan, Alex Maslov, Lisa Ellerby, Anita Huttner, Zhengdong Zhang, J. Vijg
          </td>
          <td>2025-11-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="APOBEC3 cytidine deaminases drive cancer evolution. There is an unmet need to target cancer cells with APOBEC3 activity. Here, we identify error-prone theta-mediated end joining (TMEJ) as the main pathway for repairing APOBEC3-induced double-strand breaks (DSBs). Using fluorescent DSB repair reporters and a novel biochemical assay, we demonstrate that APOBEC3A competes with replication protein A (RPA) for single-stranded DNA overhangs, exposing microhomologous sequences to shift DSB repair towards error-prone TMEJ. Genomic analysis of clinical tumor samples confirmed the co-occurrence and proximity between APOBEC3-induced mutational footprints, microhomology-mediated deletions (MMDs), and TMEJ-associated chromosomal instability signatures. Crucially, inhibition of DNA polymerase theta (Polθ) synergizes with APOBEC3A-induced DSBs to induce synthetic lethality in vitro and in vivo. Collectively, our findings identify TMEJ as the preferred mechanism for repairing APOBEC3A-induced DSBs and establish Polθ inhibition as a novel promising strategy to eliminate cancer cells with APOBEC3A activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1126d43689098cc7096980496ac4ba1303c669a" target='_blank'>
              APOBEC3A-Induced DNA Damage Drives Polymerase θ Dependency and Synthetic Lethality in Cancer
              </a>
            </td>
          <td>
            Abhishek Bose, Weisi Liu, Paul Yoo, Alina Sami, Michal T. Boniecki, M. Deshpande, Mohamed Osman, Duy D. Nguyen, U. D. Castillo, P. Giannakakou, B. Bhinder, William F Hooper, O. Elemento, N. Robine, Milaid Granadillo Rodríguez, J. Gerhardt, K. Mouw, L. Chelico, B. M. Faltas
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Telomeres are essential for genome integrity, but the accurate, high-resolution mapping of their three-dimensional (3D) chromatin interactions, a process thought to mediate gene regulation and telomere maintenance, has been limited by the repetitive nature of subtelomeric DNA and shortcomings of traditional sequencing-based chromatin conformation capture methodologies. To overcome these challenges, we developed Telomere Conformation Capture by sequencing (Telomere-C), a novel high-throughput approach for the genome-wide identification of telomeric chromatin interactions. We applied Telomere-C to three normal fibroblast cell lines (BJ, IMR90, WI38), one telomerase-positive cancer cell line (HeLa S3) and two cancer cell lines (U2-OS and WI38-VA13) that maintain telomeres by alternative lengthening of telomeres (ALT). We generated millions of telomere-associated reads per sample and identified up to 41,026 interaction peaks. We successfully validated previously-described telomeric interaction sites (e.g., TERT and DUX4), and critically, revealed a novel class of ultra-long-range interactions extending over 5 Mb away from the telomere, which surprisingly constitute over 80% of all identified contacts. Contrary to prior studies focusing on protein coding gene interactions, we discovered that the telomeric 3D interactome is overwhelmingly anchored at repetitive element hubs, particularly interstitial telomeric sequence (ITS), telomere-associated repeat 1 (TAR1) and D20S16 elements. We found that the clustering of these interactions correlates strongly with cell-type-specific telomere maintenance. Most notably, D20S16 telomeric interactions were uniquely and highly enriched in ALT cancer cells, suggesting a mechanistic link. Taken together, our study effectively constructs the first high resolution maps of the telomeric 3D interactome, redefining its scope to be dominated by ultra-long-range contacts with repetitive elements. This work provides fundamental insights into telomeric nuclear organization and establishes the telomeric D20S16 interaction as a molecular signature for the ALT pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="It has been more than 20 years since the publication of the C57BL/6J mouse reference genome, which has been a key catalyst for understanding the biology of mammalian diseases. However, the mouse reference genome still lacks telomeres and centromeres, contains 281 chromosomal sequence gaps and only partially represents many biomedically relevant loci. Here we present the first telomere-to-telomere (T2T) mouse genomes for two key inbred strains, C57BL/6J and CAST/EiJ. These T2T genomes reveal substantial variability in telomere and centromere sizes and structural organization. We thus add an additional 213 Mb of new sequence to the reference genome, which contains 517 protein-coding genes. We also examined two important but incomplete loci in the mouse genome—the pseudoautosomal region (PAR) on the sex chromosomes and KRAB zinc-finger protein loci. We identified distant locations of the PAR boundary, different copy numbers and sizes of segmental duplications and a multitude of amino acid substitution mutations in PAR genes. Telomere-to-telomere assemblies of two mouse inbred strains, C57BL/6J and CAST/EiJ, offer improvements over the current mouse reference genome by adding telomere and centromere sequences that lead to insights into variability in telomere and centromere sizes and organization, and the discovery of 225 and 355 new genes for C57BL/6J and CAST/EiJ, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, T. Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C. Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f51ce65d6aee0f0ac9a07259643549e19bbbc0c" target='_blank'>
              Acidic transcription factors position the genome at nuclear speckles through transcription dependent and independent mechanisms
              </a>
            </td>
          <td>
            Pankaj Chaturvedi, Purnam Ghosh, Liguo Zhang, Meng Zhang, Huimin Zhao, Andrew S. Belmont
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Non-canonical (non-B) DNA secondary structures, e.g., G-quadruplexes and triplex DNA, are mutation hotspots and genome regulators contributing to disease and evolution. Yet they remain uncharacterized in complete genomes in vivo. Here we exploited the fact that many non-B DNA structures form single-stranded DNA (ssDNA). Using permanganate/S1 footprinting across 14 cell lines, we generated ssDNA profiles for human and six non-human ape telomere-to-telomere (T2T) genomes. Newly resolved satellite arrays—e.g., at ribosomal DNA and centromeres—displayed high ssDNA levels, implicating non-B DNA in satellite expansion and function. Hidden Markov Models applied to our ssDNA data revealed active genomic domains with specific functions—e.g., replication, transcription, or recombination—each enriched in particular non-B DNA types. Human-specific ssDNA domains correlated with nervous system genes, whereas cancer and embryonic cells showed increased ssDNA in transposable elements. Our ssDNA analysis across ape T2T genomes uncovered conserved and species-specific DNA structural dynamics central to genome regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2c0bd20d360ee798769e48db594be4df9b6168" target='_blank'>
              Comparative analysis of single-stranded and non-canonical DNA formation in human and other ape cells with telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Jacob P. Sieg, Huiqing Zeng, Hana Pálová, Linnéa Smeds, S. Mohanty, Angelika Lahnsteiner, Francesca Chiaromonte, Kateryna D. Makova
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singal, Valentina Opazo-Medallo, Anthony Rodriguez, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzales-Velozo, Irving L Weissman, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human type-II topoisomerases, TOP2A and TOP2B, resolve transcription-associated DNA supercoiling, thereby influencing gene expression programs, and have been recently linked to three-dimensional genome architecture through yet poorly understood mechanisms. Here, we investigate the regulatory roles of TOP2 paralogs using estrogen signaling, which triggers an acute transcriptional induction that involves extensive rewiring of genome organization, as a model system. Unexpectedly, we find that estrogen treatment strongly inhibits TOP2B catalytic activity—although not its binding—specifically at estrogen-responsive enhancers and promoters. This inhibition results in an accumulation of negative DNA supercoiling and promotes the formation of regulatory chromatin contacts. Estrogen-mediated inhibition of TOP2B activity depends on estrogen receptor α, a noncatalytic function of TOP2A, and, most importantly, the action of zinc finger protein associated with tyrosyl-DNA phosphodiesterase 2 and TOP2, an atypical small ubiquitin-like modifier ligase that directly inhibits TOP2 activity. This mechanism of transcriptional control, involving the fine-tuning of DNA supercoiling levels, highlights the role of DNA topoisomerases as central regulators of genome dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fc329f060fc8318d5c30aa1bf3dc055d01aee37" target='_blank'>
              TOP2B modulates DNA supercoiling and chromatin contacts during transcriptional induction
              </a>
            </td>
          <td>
            José Terrón-Bautista, Marina Bejarano-Franco, María del Mar Martínez-Sánchez, Laura López-Hernández, Héctor Díaz-Maldonado, Angélica Santiago-Gómez, Sara Kidane, Ananda Ayyappan, Jaguva Vasudevan, Mario García-Domínguez, R. S. Williams, A. Aguilera, Gonzalo Millán-Zambrano, Felipe Cortés-Ledesma
          </td>
          <td>2025-11-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) assays are being widely adopted in oncology and maternal-fetal medicine. Patterns of cfDNA fragmentation can provide useful information about gene regulation and expression in human disease from a blood draw. Here, we demonstrate that enhancer RNA expression – a marker of enhancer activity – can be inferred from local patterns of cfDNA fragmentation. We define a transcriptional activation score (TAS) that predicts expression of enhancers and genes based on cfDNA fragment sizes and positions near transcriptional start sites (TSSs). The TAS identifies activity of cancer-associated enhancers in patients with cancer, distinguishes clinically relevant cancer subtypes, and identifies activation of enhancers associated with treatment resistance and therapy response. We propose a simple model to account for our findings based on chromatin fiber structure and the depletion of H1 histone proteins near active TSSs. Our model provides a unified framework that reconciles seemingly conflicting observations from prior fragmentomics studies. Broadly, this work enables blood-based assessments of gene regulation in cancer and non-oncologic diseases to inform pathobiology, diagnosis, and treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caacf7712f535881a1313780e0d16e1ccea5aabe" target='_blank'>
              A chromatin fiber model explains cell-free DNA fragmentation signatures of active regulatory elements
              </a>
            </td>
          <td>
            Alexis Yang, G. Lee, Surya B. Chhetri, Razane El Hajj Chehade, Gunsagar S. Gulati, Medha Pandey, Doris Fu, Yoo-Na Kim, Shahabeddin Sotudian, H. Savignano, Sadia D. Shahbazi, Chelsea Philpot, Taylor A. Hoggood, D. Gulhan, M. Kellis, Jacob E Berchuck, S. Baca
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Approximately half of high-grade serous ovarian carcinomas (HGSOCs) demonstrate homologous recombination deficiency (HRD) with characteristic genomic rearrangements. However, the impact of HRD on centromeres and transposable elements remains largely unexplored in HGSOC since conventional short-read sequencing is unable to interrogate these repetitive regions. We employed Oxford Nanopore long-read sequencing (LRS) to investigate genomic and epigenetic alterations in these regions. Pre-treatment archival cryopreserved tumor and matched blood samples were obtained for six patients with HGSOC. High-molecular-weight DNA was sequenced using Oxford Nanopore R10.4 flow cells and aligned to both the GRCh38 and the telomere-to-telomere T2T-CHM13 reference genome. Pathogenic gene mutations, allele-specific copy number variations, structural variants, and CpG methylation were analyzed. All six tumors had pathogenic TP53 mutations. Two carried germline BRCA1 mutations, while three showed CCNE1 amplifications. HRD scores and mutational signatures associated with HRD were elevated in the BRCA1-mutated tumors. Centromeric regions were significantly hypomethylated in tumors and their methylation profiles distinctly separated HRD tumors from non-HRD tumors. LINE1 and ERV transposable elements showed marked hypomethylation in tumors without germline BRCA1 mutations. Chromosome arm-specific telomere lengths were significantly shortened in tumors. Allele-specific hypermethylation in the TERT hypermethylated oncological region was detected in three tumors. LRS uncovered HRD-related genomic and epigenomic alterations in previously inaccessible repetitive regions of HGSOC, including centromeric and transposable element hypomethylation. These findings highlight the potential of such abnormalities as novel biomarkers for HGSOC and warrant further application of the methods to larger cohorts in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a227daba2577321ec904774ceb23e3c2f8c0883" target='_blank'>
              Long read sequencing reveals novel genomic and epigenomic alterations in repetitive regions of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Y Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, Barrett Lawson, R. Hillman
          </td>
          <td>2025-10-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Alternative DNA conformation formed by sequences called flipons are thought to play an important role in regulating various genomic processes, either repressing or enhancing transcription, chromatin organization, DNA repair, telomere maintenance, RNA splicing, translation, and stress responses. The formation of G-quadruplexes (GQs) has been investigated experimentally using various methodologies with varying degrees of overlap between the results underscoring the need for a gold-standard GQ dataset. With this aim we trained a large language model, GQ-DNABERT using EndoQuad, the most comprehensive human GQ dataset. GQ-DNABERT recalled the training data and predicted de novo GQs in intergenic and intronic regions, enriched for cis-regulatory elements (cCREs) and ATAC-seq peaks. We evaluated the predicted GQ-DNABERT proximal enhancer–promoter (pEP) pairs, using annotations from ENdb, ENCODE, Zoonomia, Chromium multiomics scATAC-seq and scRNA-seq data from normal cells, and cCREs from normal-cancer pairs. We found GQ pEP pairs correlating with gene expression, with some pairings potentially acting as tissue-specific switches. Genes with GQ pEP pairs in cancer cells are enriched in different processes compared to the corresponding normal tissues. Overall, GQ-DNABERT is a valuable tool for extending and harmonizing data collected ex vivo. We demonstrate the usefulness of GQ-DNABERT for investigating transcriptional regulation in single-cell experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57f048fcb6a847ca70df111a471d2de991b3cd9c" target='_blank'>
              GQ-DNABERT reveals GQ proximal enhancer–promoter interactions associated with tissue-specific transcription
              </a>
            </td>
          <td>
            Dmitry Konovalov, Dmitry Umerenkov, Alan Herbert, M. Poptsova
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa16651a01af6730b55901a5d23dad38d167b3c" target='_blank'>
              Reporter CRISPR screens decipher cis-regulatory and trans-regulatory principles at the Xist locus.
              </a>
            </td>
          <td>
            Till Schwämmle, Gemma Noviello, Eleni Kanata, Jonathan J Froehlich, Melissa Bothe, Alexandra Martitz, Aybuge Altay, Jade Scouarnec, Vivi-Yun Feng, Heleen Mallie, Martin Vingron, Edda G. Schulz
          </td>
          <td>2025-10-06</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) is a notable exception to genome integrity, characterized by significant DNA loss during development. In many nematodes, PDE is initiated by DNA double-strand breaks (DSBs), which lead to chromosome fragmentation and subsequent DNA loss. However, the mechanism of nematode DNA breakage remains largely unclear. Interestingly, in the human and pig parasitic nematode Ascaris, no conserved motif or sequence structures are present at chromosomal breakage regions (CBRs), suggesting the recognition of CBRs may be sequence-independent. Using Hi-C, we revealed that Ascaris CBRs engage in three-dimensional (3D) interactions before PDE, indicating that physical contacts of the break regions may contribute to the PDE process. The 3D interactions are established in both Ascaris male and female germlines, demonstrating inherent genome organization associated with the CBRs and to-be-eliminated sequences. In contrast, in the unichromosomal horse parasite Parascaris univalens, transient and pairwise interactions between two neighboring CBRs that would form the ends of future somatic chromosomes were observed only during PDE. Intriguingly, we found specific spatial compartmentalization changes in the Ascaris chromosomes after PDE converts 24 germline chromosomes into 36 somatic ones. Remarkably, Parascaris PDE generates the same set of 36 somatic chromosomes, and the compartment changes following PDE are consistent between the two species. Overall, our findings contribute to a model that the 3D genome facilitates the recognition of sequence-independent break sites for Ascaris PDE. It also suggests an evolutionary and developmentally conserved 3D genome reorganization of nematode somatic chromosomes following PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (∼750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b8c268f4003d4cab2a7a6a760ce10e09bcebe0" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b8e7f8962ad8fcf01d30be8350fefcd7ba1a81" target='_blank'>
              CRISPR live-cell imaging reveals chromatin dynamics and enhancer interactions at multiple non-repetitive loci.
              </a>
            </td>
          <td>
            Meishuo Liu, Keyun Huang, Jie Zhang, Qingyang Li, Xinming Wang, Buming Gu, Hao Tang, Zhenhai Du, Liangjun Hu, Shutao Qi, Yu Ma, Hongtao Yu, Wei Xie, Xianyang Fang, Haifeng Wang
          </td>
          <td>2025-11-06</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cleavage and polyadenylation (CPA) is a co-transcriptional mRNA processing mechanism that is central to mRNA and protein function. Dysregulation of CPA is widespread in cancer, promotes oncogenic programs, and affects patient outcomes. The CPA machinery is composed of multiple factors, and while prior research has investigated the impact of CPA gene expression on cancer phenotypes, the contribution of genomic alterations, such as mutations and copy number variations, remains largely unexplored. In this study, we conducted a pan-cancer analysis of genomic alterations in CPA genes. While numerous CPA genes harbor somatic mutations, these mutations do not significantly impact mRNA expression or provide prognostic value. In contrast, we found that copy number alterations in CPA genes have substantial clinical relevance. Notably, we identified the cleavage and polyadenylation specificity factor 1 (CPSF1) gene as the most frequently amplified CPA gene in cancer. While amplification of CPSF1 and MYC are co-occuring, CPSF1 amplification independently correlates with poor prognosis. We also found that CPSF1 amplification can impact 3′UTR length regardless of MYC status. Our study highlights the importance of CPSF1 as a promising prognostic factor in cancer and as a therapeutic intervention target to study in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4081cc4fba35f80ef2fd0752f0deca0cef14763a" target='_blank'>
              CPSF1 Is Co-Amplified with MYC but Is Independently Associated with Alternative Polyadenylation in Cancer
              </a>
            </td>
          <td>
            A. Alahmari
          </td>
          <td>2025-11-21</td>
          <td>Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e564310285f85398ebc0aef6f674b82cb349e5" target='_blank'>
              Decoding Gene Responsiveness to Synthetic Chromatin Reader-Actuators with Multi-Modal Epigenomic Profiling
              </a>
            </td>
          <td>
            S. Kim, Isioma Enwerem-Lackland, Natecia L. Williams, Rachel Fisher, Christopher L. Plaisier, Karmella A. Haynes
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            Guangli Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The 3D chromatin architecture establishes a complex network of genes and regulatory elements necessary for transcriptomic regulation in development and disease. This network can be modelled by cis-regulatory hubs (CRH), which underscore the local functional interactions between enhancers and promoter regions and differ from other higher-order chromatin structures such as topologically associated domains (TAD). The Activity-by-contact (ABC) model of enhancer-promoter regulation has been recently used in the identification of these CRHs, but little is known about the role of CTCF on the ABC scores and the consequent impact on CRHs. Here we show that the loss of CTCF leads to a reorganization of the ABC-derived rankings of the putative enhancers in the mouse heart, a global reduction of the total number of CRHs and an increase in the size of the CRHs. Furthermore, CTCF loss leads to a higher percentage of CRHs that cross TAD boundaries. These results provide another layer of evidence to support the importance of CTCF in the formation of regulatory networks necessary for gene regulation. Summary Deletion of CTCF in mouse cardiomyocytes led to reorganization of the activity-by-contact scores of the heart enhancers and changes in the cis-regulatory hubs">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/979e8629c76cf287372d3b0a0fa1b5a83502bf9c" target='_blank'>
              CTCF Mediates the Cis-Regulatory Hubs in Mouse Hearts
              </a>
            </td>
          <td>
            Mick Lee, Loïc Mangnier, Cory C. Padilla, D. Lee, Wilson Tan, Wen Hao Zheng, L. Gan, Ching Kit Chen, Y. Lim, Rina Miao Qin Wang, Peter Y. Q. Li, Yonglin Zhu, Steve Bilodeau, Alexandre Bureau, R. Foo, C. Anene-Nzelu
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction Chromatin interactions, particularly those between promoters and distal enhancers, enable precise gene regulation in specialized tissues, like the cochlea in the inner ear. Disruptions in these long-range interactions between enhancers and gene promoters are linked to hereditary hearing loss. For many genes important to cochlear development and function, the distal regulatory elements that control their expression remain unknown. Identifying these elements and studying their regulatory roles is challenging due to their distance from target genes and the spatial complexity of chromatin architecture. Methods To address this, we employed Micro-C, a high-resolution chromatin conformation capture technique for mapping chromatin interactions, to construct a cochlea-specific chromatin interaction map. We then integrated epigenomic and transcriptomic data to interpret enhancer-promoter interactions involved in gene regulation. Results Our analysis revealed unbiased tissue-specific long-range interactions, and some of those interactions overlapped with disease-associated deletions and active regulatory elements, such as the NR2F1 locus, which is involved in Bosch-Boonstra-Schaaf optic atrophy syndrome (BBSOAS), and the DLX5/6 locus, which is linked to Split-Hand/Foot Malformation Type 1 (SHFM1), suggesting that structural variants disrupting local chromatin architecture cause transcriptional dysregulation. Discussion This study establishes a high-resolution interaction map of the cochlea, demonstrating how non-coding variants can impair tissue-specific gene regulation in hearing loss. Our dataset provides a foundational resource for analyzing hereditary hearing loss mutations and investigating transcriptional regulation in the cochlea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac1a573c10cae04e963fe41193ca45b22c9aa173" target='_blank'>
              Mapping of chromatin architecture and enhancer-promoter interactions in the cochlea
              </a>
            </td>
          <td>
            Tuba Ege, Celia Bloom, Mi Zhou, Huizhan Liu, Litao Tao
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prime editing, a flexible "search-and-replace" genome editing technique, has become a potent instrument for accurately altering the genome without the need for donor templates or double-strand DNA breaks. The vast non-coding regulatory genome, which includes enhancers, promoters, untranslated regions (UTRs), and non-coding RNAs, is now understood to be a major driver of tumorigenesis, despite the fact that the majority of cancer research has concentrated on protein-coding mutations. Oncogene dysregulation, tumor suppressor silencing, and treatment resistance can all result from mutations in these areas. The convergence of these two fields is examined in this review, which also describes the use of prime editing to fix particular, harmful non-coding mutations. In order to restore normal gene expression patterns, we describe methods for focusing on regulatory elements, present a landscape of non-coding drivers in cancer, and talk about the fundamentals of prime editing. Lastly, we look at delivery methods, therapeutic factors, and how prime editing might be incorporated into the precision oncology paradigm going forward. We emphasize how prime editing could help usher in a new era of targeted therapy and functional genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9d66804e5625efa917cbd699bfc770f8da5aa5" target='_blank'>
              Prime Editing the Non-Coding Genome: A Molecular Review of Correcting Regulatory Mutations for Precision Cancer Therapy
              </a>
            </td>
          <td>
            Alina Mansoor, Midhat Narmeen Mustafa, Abdul Ahad Mehboob, Nureen Shafaqat, Sanam Dehraj, Amna Noor, Sehresh Murtaza, Mahnoor Javed
          </td>
          <td>2025-10-30</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer and is a potential vulnerability that can be selectively targeted. To systematically identify genes that affect the incidence and fitness of aneuploid cells, we conducted a genome-wide CRISPR/Cas9 screen using NMS-P715, an inhibitor of the spindle assembly checkpoint (SAC) kinase Mps1/TTK. In this study, we identified a number of genes known to regulate aneuploidy and mitosis, and subsequently focused on PRR14L, a ubiquitously expressed gene previously implicated in chronic myelomonocytic leukemia (CMML). Proximity labeling of PRR14L using TurboID revealed several cell division proteins, including the PP2A-B56 phosphatase complex and the spindle assembly factor TACC3, as PRR14L-interacting proteins. Loss of PRR14L prolongs SAC-dependent mitotic arrest in response to microtubule depolymerization but, paradoxically, leads to catastrophic mitotic errors upon SAC abrogation by MPS1 inhibitors. A model derived from our findings provides a rationale for exploiting MPS1 inhibition as a potential vulnerability in cancers containing either PRR14L loss of function mutations or FGFR-TACC3 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Meiotic drivers are selfish genetic elements that subvert Mendelian inheritance to increase their own transmission, yet they are typically found at low frequencies across natural populations. The factors that limit their spread remain unclear. To investigate this paradox, we studied the Segregation Distorter (SD) system, a selfish coadapted gene complex in Drosophila melanogaster. SD biases its transmission by killing sperm carrying a homologous chromosome bearing a target locus, Responder (Rsp), which appear as satellite repeats. Such selfish killing impairs male fertility and imposes selective pressure on the host genome to evolve resistance, either by deleting Rsp copies or acquiring unlinked suppressors. To characterize the spectrum of Rsp alleles and the frequency of segregating suppressors, we surveyed 90 strains from the Drosophila Genome Reference Panel. Rather than loss of Rsp, we found that over half of the strains (52/90) harbor suppressors located on the X chromosome or autosomes, but not the Y chromosome. The widespread presence of strong suppressors limited the resolution of our genome-wide association mapping; however, recombination analysis identified a strong X-linked suppressor to a ∼300 kb interval on the chromosome. Together, our findings suggest that pervasive, multilocus suppression constrains the spread of SD in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e04c6f5ef9fa6ac101d2f55bfc644ec979072" target='_blank'>
              Pervasive suppressors halt the spread of selfish Segregation Distorter in a natural population
              </a>
            </td>
          <td>
            Ching-Ho Chang, Tyler Handler, Nicholas J. Fuda, Danielle Pascua, Taylar Mouton, A. Larracuente
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract The SNF2 family chromatin remodeler HELLS has emerged as an important regulator of cell proliferation, genome stability, and several cancer pathways. Significant upregulation of HELLS has been reported in 33 human cancer types. While HELLS has been implicated in DNA damage response, its function in DNA repair is poorly understood. Here, we report a new regulatory link between HELLS and single-strand break (SSB) repair in cellular responses to DNA alkylation damage. We found that loss of HELLS impairs SSB repair and selectively sensitizes cells to DNA alkylating agents and PARP inhibitors (PARPi). Our data reveal non-epistatic interactions between HELLS and PARP1 and suggest that HELLS functionally compensates for PARP1 deficiency in promoting cell survival in response to DNA alkylation damage. Furthermore, we found that HELLS is co-expressed with PARP1 in cancer cells, and its loss is synthetic lethal with homologous recombination deficiency (HRD). This work unveils new functions of HELLS in modulating SSB repair and responses to clinically relevant DNA alkylation damage, thus offering new insights into the potential therapeutic value of targeting HELLS in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dfda2eb1be90c33c77ac8f6428991258f5ea18" target='_blank'>
              The chromatin regulator HELLS mediates SSB repair and responses to DNA alkylation damage
              </a>
            </td>
          <td>
            Joyous T Joseph, Christine M Wright, Estanislao Peixoto, Etsuko Shibata, Asad Khan, Yong Li, J. R. Neidigk, Bo-Ruei Chen, Saba Tufail, Aaiyas Abdulhamid Mujawar, Olivia Decker, Krishna Reethika Kadali, Azait Imtiaz, Brianna A Jones, Yanfeng Zhang, S. Gradilone, Zachary A Lewis, Rafael Contreras-Galindo, Arko Sen, Anindya Dutta, Wioletta Czaja
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Systemic sclerosis (SSc) is a fibrotic autoimmune disease in which genomic sources of instability and their immunological consequences remain poorly defined. We show that bleomycin, a widely used SSc fibrosis model, induces DNA double-strand breaks (DSBs) at active centromeres. Comparable centromeric damage was detected in fibroblasts from patients with limited cutaneous SSc, with a substantial fraction of cells affected. Quantification of α-satellite repeat length revealed deletions, paralleling alterations observed in diffuse and limited SSc. These breaks are repaired primarily by ATM-dependent RAD51-mediated homologous recombination, but repair remains incomplete. Incomplete repair of centromeric breaks disrupts kinetochore assembly, leading to missegregation and formation of micronuclei and irregular extranuclear chromatin enriched in centromere proteins. BANF1 redistribution was observed on nuclei and micronuclei, and CENP-A/B–positive cytoplasmic chromatin colocalized with MHC class II (HLA-DRB1), suggesting a route for centromeric antigen exposure. Our findings establish BLM exposure as a tractable experimental model for inducing centromere deletions, producing widespread double-strand breaks across active centromeres with signatures consistent with chromatin structure–dependent cleavage. Together, these findings support a model in which centromere instability is an important contributor to chromosomal instability, immune activation, and fibrosis in SSc, providing a mechanistic link between genome instability and autoimmunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="5-hydroxymethylcytosine (5hmC) is a stable epigenomic mark associated with gene regulation and tumorigenesis. To evaluate its potential for multi-cancer detection, epigenomic distribution of 5hmC was analyzed in tumor tissues (n = 217), normal tissues (n = 50), and cell-free DNA (cfDNA) from cancer (n = 1009) and non-cancer subjects (n = 1678) across breast, colon, lung, ovarian, and pancreatic cancers. Differential analysis revealed extensive redistribution of 5hmC in early-stage tumors that persisted into the late-stage, while global 5hmC abundance decreased in all tumors. Two distinct patterns of hydroxymethylation distribution were identified: one common across cancers and another tissue specific. Differential hydroxymethylation regions (DhmRs) that were tumor tissue specific had significantly improved cancer prediction in cfDNA. A multinomial logistic regression model was trained on tissue-specific DhmRs to predict Tissue of Tumor Origin (TOTO) of cfDNA with 85.2% accuracy. These findings demonstrate that stable, cancer-specific 5hmC signatures appear to be established early in tumorigenesis persist throughout disease progression, suggesting 5hmC as a potential pan-cancer marker for early and late-stage tumor detection via liquid biopsy. Analysis of 5hmC in tumor tissues and cfDNA reveals concordant cancer signatures that show potential for tissue origin prediction via liquid biopsy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27052f03426a9d599b2be2b2dba739f7a1a6978" target='_blank'>
              5-hydroxymethylcytosine analysis reveals stable epigenomic changes in tumor tissue that enable cancer detection in cell-free DNA
              </a>
            </td>
          <td>
            Yuan Xue, Y. Ning, V. Friedl, David Haan, Anna Bergamaschi, G. Guler, K. Hazen, Aaron Scott, T. Phillips, E. McCarthy, Chris Ellison, R. Malta, A. Nguyen, Vanessa Lopez, R. Cavet, M. Peters, J. Sheard, William J Gibb, Shimul Chowdhury, Wayne Volkmuth, S. Levy
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155a629eff7b8ba829fe935812c535143d1302b5" target='_blank'>
              Rewiring cancer epigenome: lncRNA as modulator of chromatin architecture and neoplastic transformation
              </a>
            </td>
          <td>
            Santosh Kumar, S. Kundu, S. Sharawat, Ashok Sharma
          </td>
          <td>2025-11-19</td>
          <td>Mammalian Genome</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The human malaria parasite, Plasmodium falciparum, faces unique DNA repair challenges; it is haploid, undergoes asynchronous mitosis termed schizogony, and lacks canonical non-homologous end joining (C-NHEJ). Yet, it has adapted DNA repair pathways that enable survival in distinct environments, including human erythrocytes and hepatocytes, as well as the mosquito vector. Plasmodium falciparum chromosomes are partitioned into a conserved core genome and highly diverse subtelomeric regions containing hypervariable, multicopy gene families, including var, which encodes a critical parasite virulence factor. The molecular mechanisms maintaining this chromosomal structure remain unclear. Here, we describe specific DNA repair pathways that distinguish hypervariable subtelomeric regions from the conserved core genome. By disrupting the DNA repair enzyme PfRad51 and TLS polymerases PfPolζ and PfRev1, we identified differential irradiation hypersensitivity across the cell cycle for TLSΔ parasites and uniform hypersensitivity for PfRad51Δ parasites, highlighting variable roles for these repair pathways. Repair of targeted double-strand breaks demonstrated that PfRad51 is essential for HR-mediated repair in the core genome, whereas a Rad51-independent, homology-directed repair pathway was observed in subtelomeric regions. This previously unidentified alternative repair pathway was independent of TLS polymerases. We propose that these differential DNA repair responses maintain the unique structure that defines P. falciparum chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865a5a32816c7b5bb2a2b106850064ee7a2becd" target='_blank'>
              Plasmodium falciparum DNA repair dynamics reveal unique roles for TLS polymerases and PfRad51 in genome diversification
              </a>
            </td>
          <td>
            Akshay Vishwanatha, Xu Zhang, Yi Jing Liu, Annie Leung, Mikayla Herring, Joseph E. Visone, Amanda Chan, Susanah Calhoun, Kirk Deitsch, Laura Kirkman
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The vast majority of recurrent somatic mutations arising in tumors affect protein-coding genes in the nuclear genome. Here, through population-scale analysis of 14,106 whole tumor genomes, we report the discovery of highly recurrent mutations affecting both the small (12S, MT-RNR1) and large (16S, MT-RNR2) mitochondrial RNA subunits of the mitochondrial ribosome encoded within mitochondrial DNA (mtDNA). Compared to non-hotspot positions, mitochondrial rRNA hotspots preferentially affected positions under purifying selection in the germline and demonstrated structural clustering within the mitoribosome at mRNA and tRNA interacting positions. Using precision mtDNA base editing, we engineered models of an exemplar MT-RNR1 hotspot mutation, m.1227G>A. Multimodal profiling revealed a heteroplasmy-dependent decrease in mitochondrial function and loss of respiratory chain subunits from a heteroplasmic dosage of ~10%. Mutation of conserved positions in ribosomal RNA that disrupt mitochondrial translation therefore represent a class of functionally dominant, pathogenic mtDNA mutations that are under positive selection in cancer genomes. Analysis of 14,106 tumor genomes highlights recurrent mutations in mitochondrial ribosomal RNA encoded within the mitochondrial genome. Mutations occur at hotspot positions and are under strong purifying selection in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eda054c049bfbd95fe327306b7323841c22683" target='_blank'>
              Functionally dominant hotspot mutations of mitochondrial ribosomal RNA genes in cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Jacqueline Tait-Mulder, Minsoo Kim, Cerise Tang, Tricia Park, Flora McNulty, S. Lilla, S. Zanivan, Alejandro Huerta-Uribe, Benan N. Nalbant, Mark Zucker, D. Sumpton, Geoffray Monteuuis, Christopher B Jackson, Wei Wei, P. Chinnery, R. Chaligné, Caleb A. Lareau, Ed Reznik, P. Gammage
          </td>
          <td>2025-11-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The regulation of cell-type-specific transcription relies on complex 3D interactions between promoters and distal regulatory elements. Although Hi-C has advanced our understanding of genome architecture, its high sequencing demand limits use in large-scale or time course experiments. We introduce Micro-C-ChIP, a strategy combining Micro-C with chromatin immunoprecipitation to map 3D genome organization at nucleosome resolution for defined histone modifications. We profile H3K4me3 and H3K27me3-specific 3D genome architecture in mouse embryonic stem cells (mESC), hTERT-immortalized human retinal pigment epithelial (hTERT-RPE1) cells, and HCT-116 RAD21-mAID-mClover (HCT-116 RAD21-mAC) cells. We validate that Micro-C-ChIP reveals genuine 3D genome features that are not driven by ChIP-enrichment bias. We identify extensive promoter–promoter contact networks in mESCs and hTERT-RPE1, and resolve the distinct 3D architecture of bivalent promoters in mESCs. Together, our results demonstrate that Micro-C-ChIP is a high-resolution, cost-efficient approach to study histone-modification-specific chromatin folding. Here, the authors introduce Micro-C-ChIP, a strategy combining Micro-C with chromatin immunoprecipitation to map 3D genome organization at nucleosome resolution for defined histone modifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c676f752c46fdc2b4fdcb566bbb4f4b14cf261fb" target='_blank'>
              Deciphering histone mark-specific fine-scale chromatin organization at high resolution with Micro-C-ChIP
              </a>
            </td>
          <td>
            Mariia Metelova, M. Vigh, Nils Krietenstein
          </td>
          <td>2025-10-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation - DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1655457ebd077b5d78f8c0d2ef963c19750b7b0" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris.
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-11-12</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df1cc841e4301399a031d8eb117c88f836d020ed" target='_blank'>
              SURFR: genome-free discovery of cancer-unique small RNAs from cohort sequencing data
              </a>
            </td>
          <td>
            Panagiotis Kalogeropoulos, Konstantin Danilov, Maria Alejandra Ulloa, Asimina Talanti, Bastian Fromm, Simon Ekman, Per Hydbring, Marc R. Friedländer
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="It is unclear how CRISPR editing outcomes vary across the genome and whether undesirable events such as structural variants (SVs) are predictable or preventable. To define a genome-wide map of editability, we performed whole-genome sequencing on 1875 budding yeast clones edited across 16 chromosomes by CRISPR-Cas9 and donor-templated repair. We found that unintended edits, including short indels and SVs, were enriched in specific genomic and sequence contexts. We developed a predictive model, SCORE (System for CRISPR Outcome and Risk Evaluation), which revealed 4.8% of the genome as SV prone, consisting of 562 SV hotspots. Donor repair-enhancing strategies suppressed SV formation in regions with moderate, but not high, predicted risk. Applying SCORE to the Sc2.0 synthetic yeast genome revealed a markedly altered SV landscape due to the removal of endogenous repetitive elements and the insertion of loxP sites. Our study provides the genome-scale map of SV hotspots after CRISPR editing and predictive and experimental tools to mitigate their formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c353e50628925b1279c5062797bfebe0f4ebe5a" target='_blank'>
              The editable landscape of the yeast genome reveals hotspots of structural variant formation
              </a>
            </td>
          <td>
            Shengdi Li, Sibylle C. Vonesch, Kevin R. Roy, Chelsea Szu Tu, Friederike Steudle, Michelle Nguyen, Cosimo Jann, Lars M. Steinmetz
          </td>
          <td>2025-10-31</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract DNA damage formation and repair are influenced by the genomic landscape, yet how chromatin and transcriptional activity shape these processes at a whole-organism scale remains incompletely understood. Using Caenorhabditis elegans, a widely used model organism to study DNA repair and related processes, we present comprehensive, time-course maps of ultraviolet-induced DNA damage and excision repair, revealing how chromatin context and transcription dictate the spatiotemporal patterns of damage and repair. Of the two repair pathways—global repair and transcription-coupled repair—global repair predominates, removing the majority of the lesions; and notably, (6–4) photoproducts are removed by transcription-coupled repair at an extent comparable to cyclobutane pyrimidine dimers, a feature not previously observed in animals. Integration of damage and repair profiles with chromatin features reveals that, despite non-uniform damage formation, repair efficiency is the primary determinant of residual damage. Finally, repair around accessible regions exhibit nucleosome-size periodicity, reflecting underlying nucleosome architecture. Together, these findings establish C. elegans as a valuable model organism for interrogating damage formation and repair within a chromatin context and reveal species-specific features that broaden our understanding of DNA repair mechanisms across metazoans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb94f64070b41f072e581840675d0dbf1b209a0d" target='_blank'>
              Chromatin context shapes DNA damage formation and nucleotide excision repair dynamics in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Cansu Kose, Cem Azgari, L. Lindsey-Boltz, Ogun Adebali, A. Sancar
          </td>
          <td>2025-10-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d97982af89e80beb34eb068ca73e5435c3e26dc" target='_blank'>
              High-throughput functional characterization of enhancers in totipotent-like cells
              </a>
            </td>
          <td>
            Lingyue Yang, Tianran Peng, Yating Zhu, Xuzhao Zhai, Boyan Huang, Ling Li, Tao Zhang, Jiekai Chen, Dan Liang, Jiangping He, Man Zhang
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Theileria annulata-infected host leukocytes exhibit cancer-like phenotypes, driven by mechanisms that remain incompletely understood. This study explores the genomic alterations underlying these transformations using whole-genome sequencing and bioinformatic analyses of six clinically relevant T. annulata-infected cell lines. Here we identify 7867 exon-linked somatic mutations shared across all cell lines, with significant enrichment in oncogenes (e.g., FLT4, NOTCH2, MAP3K1, DAXX, FCGR2B, ROS1) and tumor suppressor genes (e.g., BARD1, KMT2C, GRIN2A, BAP1). These mutations are associated with critical cancer-related pathways. Functional studies revealed that inhibition of the mutated oncogene ROS1 using crizotinib induces death in infected leukocytes, confirming its role in transformation. Additionally, we observe mutations in genes linked to genomic instability and the DNA damage response (DDR) pathways, highlighting potential parallels with cancer biology. Suppression of TP53, a key tumor suppressor, is implicated in the immortalization of infected cells, while upregulation of the DNA mutator enzyme APOBEC3H suggests a parasite-driven, mutation-inducing mechanism. Our findings provide new insights into how T. annulata reprograms host cells through genomic instability and mutations, identifying ROS1 and TP53 as critical targets for therapeutic intervention. This work advances understanding of parasite-induced oncogenic transformation and offers pathways for future research. Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes sheds light on parasite-induced genomic changes that might drive the acquisition of cancer-like phenotypes in host cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847bdcc03008fe46129bac5e3efd5e339af0347f" target='_blank'>
              Genomic characterization of host gene alterations in Theileria annulata-transformed leukocytes
              </a>
            </td>
          <td>
            Debabrata Dandasena, Akash Suresh, Roli Budhwar, Jeffrey Godwin, Sakshi Singh, Madhusmita Subudhi, Vengatachala Moorthy A, Amruthanjali T, Sonti Roy, V. Bhandari, Paresh Sharma
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The human genome attains an amazing spatial organization in the packaging of 2 m of DNA into a 10-μm nucleus. Such structural organization is achieved by the folding of chromatin and the regulation exerted by architectural proteins such as insulators. Chromatin insulators are boundary elements of the genome that, through enhancing blocking activities, demarcation of chromatin domains, and chromatin looping, regulate transcription. The review focuses on the identification and characterization of insulators in various species, discussing mainly the functions of the CCCTC-binding factor (CTCF) in mammals and functionally equivalent insulator proteins in Drosophila melanogaster. We review here the mechanisms of enhancer blocking, barrier activity, and loop extrusion, emphasizing their effects on topologically associating domains and chromatin architecture. Furthermore, we discuss new concepts that have come into prominence: tethering elements and redundancy among the insulator proteins, which contribute to chromatin organization. Advances in methodology, including chromosome conformation capture and high-resolution imaging techniques, have transformed our view of the dynamic interplay between the architecture of chromatin and transcription regulation. This review discusses the importance of insulators for genome organization and describes future directions in investigating their roles in both gene regulation and three-dimensional genomic architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eda983a70eb60f914e84aedb4a80af4895d808" target='_blank'>
              Chromatin insulators in gene regulation and 3D genome organization.
              </a>
            </td>
          <td>
            Hina Sultana, Rohit Kunar, A. G. Matera
          </td>
          <td>2025-10-01</td>
          <td>Biochemical Society transactions</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Chromatin-based mechanisms contribute to the exquisite regulation of gene expression during animal development. But how those mechanisms evolved remains elusive. Here we investigate chromatin regulatory features in the closest relatives of animals, choanoflagellates. In a model choanoflagellate Salpingoeca rosetta, we compare chromatin accessibility and histone modifications to gene expression. Accessible genomic regions in S. rosetta primarily correspond to gene promoters, and we find no evidence of distal gene regulatory elements that resemble enhancers deployed to regulate developmental genes in animals. Remarkably, the histone modification H3K27me3 decorates genes with cell type-specific expression, revealing a functional similarity in S. rosetta and animals. Additionally, H3K27me3 marks LTR retrotransposons, retaining a potential ancestral role in regulating these elements. We further uncover a putative bivalent chromatin state at cell type-specific genes that consists of H3K27me3 and H3K4me1. Together, these data support the emergence of gene-associated histone modification states that underpin development before the evolution of animal multicellularity. Here, the authors investigate chromatin-based gene regulation in the closest relative of animal, choanoflagellates. They uncover a putative dual role for the histone modification H3K27me3 in regulating of protein-coding genes during differentiation and transposable elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5504ea78d0c86caaeac861a13005f7e8b5070b2" target='_blank'>
              Chromatin profiling identifies putative dual roles for H3K27me3 in regulating cell type-specific genes and transposable elements in choanoflagellates
              </a>
            </td>
          <td>
            JM Gahan, Lily W. Helfrich, Laura A. Wetzel, N. Bhanu, , Benjamin A. Garcia, R. Klose, Alex de Mendoza, David S. Booth
          </td>
          <td>2025-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="53BP1 accumulates at DNA double strand breaks (DSBs) and is implicated in non-homologous end joining (NHEJ), but the genetic interplay of 53BP1 with the NHEJ pathway (e.g., DNA-PKcs) is poorly understood. We examine blunt DSB NHEJ of Cas9 DSBs, which is dependent on core NHEJ factors, and find that loss of 53BP1 does not affect such repair but causes a reduction when combined with DNA-PKcs disruption. In contrast, disrupting 53BP1 and DNA-PKcs, alone and together, has similar effects on the type of deletion mutation (increase in microhomology deletions). We find similar effects with RIF1, such that 53BP1-RIF1 appear to play a backup role for DNA-PKcs during blunt DSB NHEJ, but function in the same pathway to suppress microhomology deletions. Finally, DNA-PKcs kinase inhibition causes increased radiosensitivity and homology-directed repair that is not additive with loss of 53BP1. Altogether, 53BP1-RIF1 and DNA-PKcs show distinct genetic interactions with diverse DSB repair outcomes. DNA double strand breaks can be repaired by several pathways leading to different genetic outcomes. Here, the authors define the interplay between the DNA damage response factors 53BP1-RIF1 and core non-homologous end joining factors on diverse repair outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6dc22c93555e7812dfc2b85eba7a6ca2131899" target='_blank'>
              53BP1-RIF1 and DNA-PKcs show distinct genetic interactions with diverse chromosomal break repair outcomes
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chemoresistance remains a major challenge in addressing T-cell lymphoblastic lymphoma/leukemia (T-LBL/ALL), underscoring the necessity for novel strategies to unravel the molecular factors driving resistance. Through transcriptomic profiling, circBPTF was found to be markedly overexpressed in chemoresistant samples. Further functional experiments demonstrated that BPTF-665aa, the protein product of circBPTF, plays a pivotal role in mediating resistance. Notably, BPTF-665aa prevents the ubiquitination degradation of full-length BPTF, and promotes chromatin accessibility at key promoter sites, such as that of c-Myc promter 2 (P2), facilitating transcriptional activation crucial for cellular survival and proliferation under therapeutic stress. Structural studies confirmed the motifs of BPTF-665aa, including the Plant Homeodomain (PHD) finger and Bromodomain, essential for its chromatin remodeling function. HY-B0509 was identified as a small-molecule inhibitor of BPTF-665aa, with molecular docking and dynamics simulations showing stable binding to critical residues within the protein’s active site. Overall, this study introduces a new mechanism where circBPTF affects chromatin accessibility, causing chemoresistance, making BPTF-665aa as a potential therapeutic target for treating T-LBL/ALLs. This study identifies BPTF-665aa, a protein translated from circBPTF, as a key regulator of chromatin accessibility in T-cell lymphoblastic lymphoma/leukemia (T-LBL/ALL). BPTF-665aa promotes chromatin remodeling at oncogenic loci, including the c-Myc promoter, facilitating transcriptional activation of resistance-associated genes. Furthermore, we identify a small molecule inhibitor targeting BPTF-665aa, highlighting its potential as a therapeutic strategy. These findings establish a new paradigm linking circRNA-derived proteins and chromatin architecture, positioning BPTF-665aa as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17596bccd73211f83b30b6a17b37b615a3eeedbc" target='_blank'>
              BPTF-665aa mediate chromatin remodeling drives chemoresistance in T-LBL/ALL
              </a>
            </td>
          <td>
            Rong-Hui Chen, Mei Li, Zhen-Zhong Zhou, , Yong Zhu, Yuan Zhang, Xu Liu, Hai-Long Li, Jing Feng, Li-Yan Song, Rong-Min Yu, Tian-Xiao Gao, Xiao-Peng Tian, Wei-Juan Huang
          </td>
          <td>2025-11-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Upon DNA breakage, a genomic locus undergoes alterations in three-dimensional chromatin architecture to facilitate signaling and repair. Although cells possess mechanisms to repair damaged DNA, it is unknown whether the surrounding chromatin is restored to its naïve state. We show that a single DNA double-strand break (DSB) within a topologically associated domain (TAD) harboring conformation-sensitive genes causes lasting chromatin alterations, which persist after completion of DNA repair and feature topological rearrangements and loss of local RNA species. These newly acquired features of postrepair chromatin are transmitted to daughter cells and manifest as heritable impairments of gene expression. These findings uncover a hitherto concealed dimension of DNA breakage, which we term postrepair chromatin fatigue and which confers heritable impairment of gene function beyond DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70850e6eacefc0378a65ec703a07a0084a7eaca" target='_blank'>
              Repair of DNA double-strand breaks leaves heritable impairment to genome function.
              </a>
            </td>
          <td>
            Susanne C. S. Bantele, Irene Mordini, A. Biran, Nicolás Alcaraz, Gijs Zonderland, Alice Wenger, Nils Krietenstein, Anja Groth, Jiri Lukas
          </td>
          <td>2025-11-06</td>
          <td>Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea4b63fc938ba67b178443696fb6b3e033db699" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The adoption of liquid biopsy approaches in clinical practice has triggered a significant paradigm shift in the diagnostic, prognostic, and predictive outcomes for cancer patients. Circulating tumor DNA (ctDNA) is considered a valuable biomarker for monitoring tumor burden and its mutational dynamics. In this context, not all cell-free DNA (cfDNA) molecules are derived from tumor cells. Furthermore, due to tumor heterogeneity, not all ctDNA molecules contain cancer-associated alleles, complicating the direct quantification of the circulating tumor allele fraction (cTF) within the total cfDNA. Cancer arises from the accumulation of multiple genetic and epigenetic changes. Each of these molecular features can be exploited as the basis of methodological strategies used in ctDNA quantification. Different layers of omics data, from genomics, evaluating mutational analysis of somatic single-nucleotide variants and copy number alterations, to epigenomics, primarily consisting of the evaluation of methylation profiles and fragmentation patterns, can be used for this purpose. Some of these approaches can be effective in a multi-modal manner. To date, the quantification approaches for estimating cTF vary enormously, making direct comparisons and an assessment of their translational value challenging. Moreover, the lack of regulatory approval for many of these assays is a critical barrier to their widespread clinical adoption. This review explores the different omics approaches described for ctDNA quantification, outlining strengths and limitations, and highlighting their valuable applications in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65d286916cbf91ea842f3c5cc84289bd6ae22c9" target='_blank'>
              Genomics and Epigenomics Approaches for the Quantification of Circulating Tumor DNA in Liquid Biopsy: Relevance of a Multimodal Strategy
              </a>
            </td>
          <td>
            Elisa De Paolis, Alessia Perrucci, Gabriele Albertini Petroni, Alessandra Conca, Matteo Corsi, Andrea Urbani, A. Minucci
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="L1 comprises 17% of the human genome, with 50-150 full-length sequences capable of retrotransposition. Although largely inactive in normal somatic tissues, tumorigenesis leads to L1 derepression and overexpression of its RNA binding chaperone protein, ORF1p. A potential cancer biomarker, ORF1p expression is a hallmark of multiple cancers and an early event in precursor lesions. Our study provides a comprehensive pan-cancer analysis of ORF1p using CPTAC proteogenomic data, revealing a dichotomous role in modulating immune responses. Integrated analyses and supervised learning models divide ORF1p-high tumors into two groups: (1) high ORF1p associates with immunosuppression, reduced interferon signaling and diminished immune cell infiltration (HNSCC and LSCC), and (2) ORF1p-high tumors associate with immune activation (UCEC). Linear regression models reveal that cancer-specific aneuploidies may underlie this immune dichotomy, highlighting the prognostic significance of ORF1p and informing new strategies to leverage detection of plasma-circulating ORF1p to enhance immunotherapeutic efficacy in different tumor contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3642f23461379e5a0adfe4119834fff5e5e8de7" target='_blank'>
              Pan-cancer analysis reveals context-dependent roles of LINE-1 ORF1p in immune regulation and copy number alterations
              </a>
            </td>
          <td>
            E. Kaparos, Wenjing Zhang, Ryusei Miyanaga, Wilson H. McKerrow, J. Boeke, Teresa Davoli, P. Mita, Kelly Ruggles, David Fenyö
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Homologous recombination (HR) and non-homologous end joining (NHEJ) are two main repair pathways of DNA double-strand breaks (DSBs). The regulation of repair pathway choice is crucial for maintaining genomic stability and preventing carcinogenesis. Consequently, there is increasing interest in elucidating the molecular mechanisms that govern DSB repair pathway selection. Here, we identify chromosome 8 open reading frame 33 (C8orf33) as a novel regulator of DSB repair choice. We show that C8orf33 is a nuclear protein localized predominantly to the nucleolus and is recruited to DSB sites within both nuclear and nucleolar regions. We demonstrate that C8orf33 promotes the recruitment of 53BP1, thus channeling DSB repair toward NHEJ. Consequently, C8orf33 inhibits DNA end resection and counteracts the recruitment of HR factors, BRCA1 and RAD51, to DSB sites. Mechanistically, chromatin profiling analysis reveals that C8orf33 antagonizes the chromatin association of KAT8 acetyltransferase at DSB sites leading to reduced histone 4 lysine 16 acetylation (H4K16ac) levels. Accordingly, the loss of C8orf33 enhances KAT8 chromatin binding that increases H4K16ac levels. This promotes the recruitment of HR factors while suppressing the accumulation of NHEJ factors at DSB sites, thereby favoring HR over NHEJ. Additionally, we demonstrate that the elevated HR activity in C8orf33-deficient cells causes genomic instability, as evidenced by accelerated loss of ribosomal DNA repeats and increases cell death. Collectively, our findings establish C8orf33 as a critical regulator of DSB repair pathway choice, safeguarding genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac1d5cfef9dd20c22a9cd88671f578821b21baa" target='_blank'>
              C8orf33 dictates DNA double-strand break repair choice by modulating KAT8-mediated H4K16 acetylation
              </a>
            </td>
          <td>
            Laila A. Bishara, Enas R. Abu-Zhayia, Marian Nicola, N. Ayoub
          </td>
          <td>2025-11-17</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1913940e2ad0a9ac39d6ab69328331969ad3d74" target='_blank'>
              HPV16 E6 oncoprotein promotes microhomology-mediated viral integration by upregulating POLQ expression
              </a>
            </td>
          <td>
            Guangli Zhu, S. Asada, J. Abeykoon, Lifang Sun, S. Mukkavalli, Alan D. D’Andrea
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517574b6bc8a73f48a3a9ee64c0be54874afb37" target='_blank'>
              TET CpG sequence context specific DNA demethylation shapes progression of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Y. Hoogstrate, S. Ghisai, L. van Hijfte, R. Head, I. D. de Heer, Marta Padovan, Maurice de Wit, Wies R. Vallentgoed, A. Dipasquale, M. Wijnenga, Bas Weenink, R. Luning, S. Maas, Adela Brzobohata, Michael Weller, Tobias Weiss, M. Mair, A. Berghoff, A. Wöhrer, A. Jeltsch, J. Koekkoek, H. M. Hazelbag, M. Kouwenhoven, Yongsoo Kim, Bart A. Westerman, B. Ylstra, A. Niers, K. Johnson, F. Varn, R. Verhaak, Mustafa Khasraw, M. J. van den Bent, Pieter Wesseling, P. French
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transcriptional regulation is mediated by enhancers, yet how genetic perturbations alter enhancer activity and gene expression remains poorly understood. We developed UDI-UMI-STARR-seq, which integrates dual indexes and unique molecular identifiers, and combined it with RNA-seq to profile the effects of perturbations on enhancer activity and target gene expression. We applied this approach to a library of 253,632 fragments representing 46,142 cell type–specific candidate enhancers and assessed the impact of CRISPR/Cas9-mediated deletion of six transcription factors (or TFs; ATF2, CTCF, FOXA1, LEF1, TCF7L2, and SCRT1) with diverse regulatory roles. Across knockout lines, we identified responsive enhancers that were either repressed or induced, often through motifs such as the p53 family of TFs. Enhancer–gene mapping revealed TF-specific programs, including repression of Wnt/p53 targets with ATF2 or LEF1 loss, downregulation of the FIRRE locus with CTCF loss, and compensatory upregulation of RNA polymerase II regulators following FOXA1 depletion. A deep learning model trained on enhancer sequences recapitulated core principles of enhancer grammar, including cooperative motif syntax and the influence of flanking sequence context. Applying this framework to the neurodevelopmental disorder-associated 16p12.1 deletion identified responsive enhancers linked to genes involved in axon guidance, synaptic plasticity, and translational control, providing a scalable readout of enhancer dynamics generalizable to genetic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e599253ce970a00d4735ed417209ae82c988f97" target='_blank'>
              Quantifying the impact of genetic mutations on enhancer dynamics
              </a>
            </td>
          <td>
            Maitreya Das, Deepro Banerjee, Ayaan Hossain, Matthew Jensen, S. Mogre, Jiawan Sun, Joseph Mao, Adam B Glick, Howard M Salis, S. Girirajan
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffefccab81cd9b2f6152654f3b31bd309e658b64" target='_blank'>
              HDAC1 has a role in double strand break repair by regulating γH2AX signalling
              </a>
            </td>
          <td>
            José Javier Marqueta-Gracia, Marina Bejarano-Franco, Salome Spaag, Sónia Silva, A. Aguilera, Belén Gómez‐González
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The complications with identifying cells at the origin of cancer and tracking their early divisions impede studies of cancer initiation. Recently, it was shown that some DNA lesions generated by a pulse of damage-inducing mutagen persist over multiple rounds of replication. Segregation of DNA lesions in the early genealogy of an expanding clone leaves a statistically interpretable footprint of cancer initiating events. Specifically, it allows for estimating the number of cell divisions between the initiating DNA lesion and the most recent common ancestor of the tumor. Here, we analyze footprints of segregating lesions from a previously published experimental mouse system, as well as post-chemotherapy human metastatic tumors and the blood of chemotherapy treated patients. In all contexts, clones tend to start early, usually within the span of 4 cell generations from mutagen exposure. Using a branching process model, we show that fitness advantage of early cancer drivers exceeds 30%, with each early division leading to at least 1.3 self-renewing cells. We highlight an example of a blood-derived single cell phylogeny with major subclones separated by just two cell divisions. Broadly, our approach allows inference of tumor initiation and growth parameters based on events preceding the most recent common ancestor of the initiating clone as opposed to characteristics of fully grown tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d535d363c465b65e62fbbcdfa920c1c51d8d6da" target='_blank'>
              Segregating DNA lesions point to high selective advantage of tumor initiating cells
              </a>
            </td>
          <td>
            Vladimir Seplyarskiy, Maha Shady, Maria A Andrianova, M. S. Chapman, E. V. Van Allen, S. Sunyaev
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The WHO 2021 classification of central nervous system tumors incorporated molecular markers to better define diffuse glioma subgroups. We analyzed array-based copy number, targeted next-generation sequencing, germline genotyping, and Illumina EPIC array methylation data from 404 primary IDH-mutant astrocytomas to further understand the relationship between copy number and outcomes. Loss of chromosome arm 13q was associated with poor overall survival after adjusting for age, sex, tumor grade, and treatment status (hazard ratio [HR]=2.04 (95% confidence interval: 1.25-3.33), p=0.004). Chromosome 13q loss was also significantly associated with the total number of copy number alterations (p<0.001). Loss of arm 11p was associated with poor survival in the adjusted analysis (HR=2.29 (1.38-3.79), p=0.001). Unsupervised clustering of the raw copy number data identified 12 clusters, where each cluster was defined by distinct copy number alteration(s). Clusters that contained 13q loss exhibited worse overall survival compared to other clusters. The clusters were significantly associated with age at diagnosis (p=0.015), tumor grade (p<0.001) and copy number complexity (p<0.001). The clusters with a high proportion of grade 4 tumors and with the highest complexity had the poorest overall survival. These findings highlight the prognostic relevance of chromosome 13q and 11p losses in IDH-mutant astrocytomas and support the integration of additional copy number alterations into molecular classification frameworks of IDH-mutant astrocytomas to improve risk stratification and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253265c2822339aab418366b6f6dea35282c7e8a" target='_blank'>
              PATH-36. DNA copy number clustering reveals alterations that influence survival in primary IDH-mutant astrocytoma patients
              </a>
            </td>
          <td>
            K. Drucker, Arijit Panda, Paul A Decker, M. Kosel, Shameel Shafqat, Muhammad Asad Maqbool, A. Landry, Y. Ellenbogen, Vikas Patil, Caterina Giannini, R. Vaubel, Aditya Raghunathan, I. Parney, T. Burns, Cristiane M Ida, S. Kizilbash, G. Zadeh, Daniel H Lachance, A. Abyzov, J. Eckel-Passow, R. Jenkins
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa35e38064580145b981499573e43bdd54174f5" target='_blank'>
              Allele-specific correction of ATAD3A pathogenic variants via template-free CRISPR-Cas9 editing and gene conversion
              </a>
            </td>
          <td>
            Taejeong Bae, Yohan Park, Anel LaGrone, M. Šuvakov, Ping Zhang, Hyekyung Park, Holly Van Remmen, James R. Lupski, Tamar Harel, Jean J. Kim, A. Abyzov, Wan Hee Yoon
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7412496bc3addf8ed94aacf8c02fb82569e6b" target='_blank'>
              Sensitizing tumor response to topoisomerase I antibody drug conjugate by selective CDK7 inhibition
              </a>
            </td>
          <td>
            Hyerim Jung, Yoonji Lee, Donghoon Yu, Y. Jeon, Hwankyu Kang, Dahun Um, Mooyoung Seo, Dongsik Park, Jeongjun Kim, Seung-Joo Lee, Jaeseung Kim, Lauren Escobedo, Yilun Sun, Anish Thomas, Kiyean Nam, Tae-Kyung Kim
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4af295adbd4b81c6a3532b7a185c58a2b03e28" target='_blank'>
              Pervasive binding of the stem cell transcription factor SALL4 shapes the chromatin landscape
              </a>
            </td>
          <td>
            Kashyap Chhatbar, Sara Giuliani, Timo Quante, Beatrice Alexander-Howden, Jim Selfridge, Jacky Guy, Tatsiana Auchynnikava, Christos Spanos, Tricia Mathieson, Guido Sanguinetti, Adrian Bird, Raphaël Pantier
          </td>
          <td>2025-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma.
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer is generally thought to be caused by expansion of a single mutant cell1. However, analyses of early colorectal cancer lesions suggest that tumors may instead originate from multiple, genetically distinct cell populations2,3. Detecting polyclonal tumor initiation is challenging in patients, as it requires profiling early-stage lesions before clonal sweeps obscure diversity. To investigate this, we analyzed normal colorectal mucosa, benign and dysplastic premalignant polyps, and malignant adenocarcinomas (123 samples) from six individuals with familial adenomatous polyposis (FAP). Individuals with FAP have a germline heterozygous APC mutation, predisposing them to colorectal cancer and numerous premalignant polyps by early adulthood4. Whole-genome and/or whole-exome sequencing revealed that many premalignant polyps—40% with benign histology and 28% with dysplasia—were composed of multiple genetic lineages that diverged early, consistent with polyclonal origins. This conclusion was reinforced by whole-genome sequencing of single crypts from multiple polyps in additional patients which showed limited sharing of mutations among crypts within the same lesion. In some cases, multiple distinct APC mutations co-existed in different lineages of a single polyp, consistent with polyclonality. These findings reshape our understanding of early neoplastic events, demonstrating that tumor initiation can arise from the convergence of diverse mutant clones. They also suggest that cell-intrinsic growth advantages alone may not fully explain tumor initiation, highlighting the importance of microenvironmental and tissue-level factors in early cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed79fdee85797c4fc9c9d2549d5e711e9476ac0" target='_blank'>
              Polyclonal origins of human premalignant colorectal lesions
              </a>
            </td>
          <td>
            D. Van Egeren, Ryan O. Schenck, Aziz Khan, Aaron M. Horning, Shanlan Mo, Clemens L. Weiß, Edward D Esplin, Winston R. Becker, Si Wu, Casey Hanson, Nasim Barapour, Lihua Jiang, K. Contrepois, Hayan Lee, Stephanie A. Nevins, Tuhin K. Guha, Hao Zhang, Zhen He, Zhicheng Ma, Emma Monte, Thomas V. Karathanos, Rozelle Laquindanum, M. Mills, Hassan Chaib, R. Chiu, Ruiqi Jian, Joanne Chan, M. Ellenberger, Bahareh Bahmani, Basil Michael, Annika K. Weimer, D. G. Esplin, Samuel Lancaster, Jeanne Shen, U. Ladabaum, Teri A. Longacre, Anshul B Kundaje, William J. Greenleaf, Zheng Hu, James M. Ford, Michael P. Snyder, Christina Curtis
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91539f6c1730b35b91b8ad7b77a643d6d6aba2da" target='_blank'>
              Complete and haplotype-resolved maps of genomic and epigenetic discordance in monozygotic twins
              </a>
            </td>
          <td>
            Tengxue Lou, Dongming Fang, G. Lin, Zheng Jia, Yafang Qin, Yabin Tian, Lingxin Qiu, Xin Jin, Lei Cheng, Dongya Wu, Kuo Zhang, Chentao Yang, Jie Huang
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, with the most consequential mutations occurring in coding exons and regulatory regions. However, the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a mutational hotspot at transcription start sites (TSSs) in the human germline, spanning several hundred base pairs in both directions. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants, many of which are excluded from de novo mutation calls, indicating that the hotspot partly arises during early embryogenesis. We associate the TSS mutational hotspot with divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, suggesting a recombination-independent mechanism distinct from known processes. Our findings are reinforced by mutational signature analysis, which highlights alternative double-strand break repair and transcription-associated mutagenesis. These insights reveal a germline mutational phenomenon with evolutionary and biomedical implications, particularly affecting genes linked to cancer and developmental phenotypes. The impact of transcription on germline mutagenesis remains poorly understood. Here, the authors identify a mutational hotspot at transcription start sites in the human germline that is significantly enriched with early mosaic variants; they also analyse the molecular mechanisms that could cause it and explore its potential consequences for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de526e07a78fe794f2e553a66798f6d31a5db32" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fueled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, Vladimir Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract The establishment of cell type–specific chromatin landscapes is essential for cellular identity, but how these landscapes are generated remains poorly understood. Here, we demonstrate that the chromatin remodeler SMARCA5 establishes epigenetic priming that is required for retinoic acid (RA)–induced differentiation in the male germline. Germ cell–specific deletion of Smarca5 results in a complete loss of differentiating spermatogonia, phenocopying vitamin A-deficient mice that lack RA signaling. During the perinatal transition from prospermatogonia to undifferentiated spermatogonia, SMARCA5 is recruited to binding sites of the transcription factor DMRT1, which are located at distal putative enhancers and promoters of germline genes. The SMARCA5–DMRT1 pioneer complex establishes chromatin accessibility at these loci, generating poised enhancers and promoters that serve as RA receptor (RAR)–binding sites. Thus, SMARCA5 licenses transcriptional responses to RA that enable spermatogenic differentiation. Our findings uncover a mechanism linking pioneer factor activity to external signal responsiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c84c702820dd3c4b715831624f8ab0a6a3dbe7" target='_blank'>
              The SMARCA5–DMRT1 Pioneer Complex Establishes Epigenetic Priming to Direct Male Germline Development
              </a>
            </td>
          <td>
            Yuka Kitamura, Yasuhisa Munakata, Hironori Abe, Mengwen Hu, S. Oya, Shanmathi Murugesan, Mahnoor Rizwan, Shawna P. Katz, David Picketts, Richard M Schultz, Satoshi Namekawa
          </td>
          <td>2025-10-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/144b42b6b9399f428f0119f05417665f87b81650" target='_blank'>
              Long-term prevention of aneuploidy in human pluripotent stem cells by fine-tuning GSK3 activity
              </a>
            </td>
          <td>
            A. De Jaime-Soguero, M. Romitti, S. Costagliola, A. Jauch, K. Willert, F. Foijer, S. Acebrón
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acquired resistance to immune checkpoint inhibitors (ICIs) remains a significant challenge in the treatment of metastatic melanoma. Phenotypic plasticity, such as dedifferentiation and transdifferentiation, is an increasingly recognized mechanism for treatment resistance. We present a case of a man in his 70s with metastatic melanoma who experienced progression through sequential treatments including pembrolizumab in combination with the HDAC inhibitor entinostat, and ipilimumab. During treatment a histologically distinct pleomorphic rhabdomyosarcoma (RMS) emerged at metastatic sites. Longitudinally acquired tumor samples representing both phenotypes were analyzed using whole- exome sequencing (WES), RNA sequencing (RNA-seq) and high-plex tissue imaging (spatial proteomics). WES revealed driver mutations (e.g. NRAS, NF1) and loss-of-heterozygosity (LOH) shared between phenotypes indicating a common ancestral clone. Phylogenetic analysis demonstrated an early divergence of the phenotypes, with each later acquiring unique mutations. RNA-seq showed mutually exclusive expression of lineage-specific markers as well as epithelial-mesenchymal transition and myogenic gene set enrichment in the RMS samples. High-plex imaging identified distinct tumor microenvironments, with RMS lesions enriched in CD163+ macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Genome rearrangement is a hallmark of cancer and ageing. DNA double-strand breaks (DSBs) are highly toxic lesions that can generate genome rearrangements. Several pathways compete for DSB repair, and selection of the appropriate repair process is critical for genetic stability. DSB repair acts according to two steps: (1) the choice between nonhomologous end-joining (NHEJ) and the resection of DNA ends, generating single-stranded DNA (ssDNA) and (2) on generated ssDNA, homologous recombination (HR) and sub-processes. Here, we show that 53BP1, which plays a prime role in the first step by protecting against resection and fostering NHEJ, physically interacts with the catalytic subunits of cAMP-dependent protein kinase A (PKAcs). PKA favours the recruitment of 53BP1 at DNA damage sites and consistently prevents resection, favouring NHEJ. Inhibition of PKA stimulated resection and concurrently reduced NHEJ. Conversely, the activation of PKA with 8-Bromo-cAMP stimulated NHEJ and reduced HR. These data provide new avenues for potential anticancer strategies. More generally, these findings underscore the high complexity of the regulation of DSB repair, identifying PKA as a novel participant in the DNA damage response that acts in the DSB repair process, which may be essential for maintaining genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff78c19cf3ac765f9d790f02f928cce37461ed5" target='_blank'>
              PKA prevents the resection of DNA double-strand breaks and favours nonhomologous end-joining
              </a>
            </td>
          <td>
            Houaria Khater-Sellou, E. Dardillac, Elise Rabut, I. Magdalou, Vasily Ogryzko, J. Guirouilh-Barbat, Bernard S. Lopez
          </td>
          <td>2025-11-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="BARD1 variants are associated with hereditary breast cancer and neuroblastoma, yet, 98% of missense variants remain variants of uncertain significance (VUS). We applied Saturation Genome Editing (SGE) to assess 8,818 SNVs and 2,097 3-base pair deletions for their effects on cell survival and RNA abundance. We found that 13% of missense variants are loss-of-function (LoF), with 98% in BARD1's three folded domains. LoF missense variants in the ankyrin repeat and BRCT domains were enriched in breast cancer cohorts, linking their molecular functions to cancer risk. SGE discriminated known pathogenic from benign variants with exceptional accuracy (AUC = 0.99) and resolved 95.4% of VUS, demonstrating high clinical utility. The single nucleotide resolution allowed discrimination of variant effects on RNA abundance from those affecting protein function and provided further evidence linking BARD1's function in homology directed repair to tumor suppression. This comprehensive variant effect dataset informs inherited cancer risk and treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c2cc869774eac711c785244c7b4a415374fa2e8" target='_blank'>
              Saturation genome editing of BARD1 resolves VUS and provides insight into BRCA1-BARD1 tumor suppression
              </a>
            </td>
          <td>
            I. Woo, S. Casadei, M. W. Snyder, N. T. Smith, S. Best, M. Tejura, P. Gupta, A. E. McEwen, M. Post, A. Hamm, M. Dawood, A. Hosokai, A. Xu, R. K. Garge, S. Fayer, T. Brannan, M. E. Richardson, S. Pendyala, S. Heidl, L. Muffley, D. M. Fowler, L. Starita
          </td>
          <td>2025-11-06</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/325705b2c45e84020614b3300cac4317c5da4a9b" target='_blank'>
              Integrated Proteomic and Epigenomic Analysis Reveals IGF2 as a Vulnerability in PRC2-Deficient Malignant Peripheral Nerve Sheath Tumors
              </a>
            </td>
          <td>
            Joanna K. Lempiäinen, Kirill Miachin, Xingyu Liu, Kuangying Yang, Cynthia Horth, E. Bareke, Mitchell F. Grinwald, Wesley N. Saintilnord, Ting Wang, Jacek Majewski, A. Hirbe, Benjamin A. Garcia
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c948196f370f90e50659421ebd98d4fcdbcb30c7" target='_blank'>
              Differentiation-Dependent Proximity Proteomics Identifies Novel Host Factors Linked to HPV16 E2 Function
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, A. T. Prabhakar, Jenny D Roe, Elinor Lu, Austin J. Witt, S. Giri, Molly L Bristol, Phoebe Bridy, Xu Wang, Arjun Rijal, Charles Lyons, I. Morgan
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Piwi-interacting RNAs (piRNAs) are small non-coding RNAs primarily recognized for their roles in germline development and genome stability. However, their contribution to breast cancer remains poorly understood. In this study, we profiled piRNA expression in 96 primary breast cancer tissues and explored the prognostic and functional significance of piR-775. Higher expression of piR-775 was significantly associated with improved relapse-free survival, suggesting its potential as a favorable prognostic biomarker. Functional assays in triple-negative breast cancer (TNBC) models demonstrated that piR-775 overexpression significantly suppressed colony formation and three-dimensional tumor growth. Using an organ-on-chip platform, piR-775 overexpression also inhibited TNBC cell migration and invasion. Transcriptomic analyses following piR-775 re-expression identified multiple downregulated target genes involved in cell cycle progression and DNA damage repair pathways, including POLE, BARD1, TPX2, BIRC5, and XRCC2, which were further validated by RT-qPCR. Integration with CRISPR screen data highlighted 47 essential piR-775 targets required for TNBC survival. Additionally, DNA damage assays indicated increased genomic instability upon piR-775 expression. Notably, combining piR-775 with the PARP inhibitor Olaparib led to enhanced cytotoxic effects, suggesting a synthetic lethal interaction. Furthermore, transcriptomic analysis of breast cancer revealed consistent downregulation of the PIWI proteins PIWIL2 and PIWIL4, indicating disruption of the piRNA–PIWI pathway in breast cancer. Collectively, these findings establish piR-775 as a tumor-suppressive piRNA with both prognostic and therapeutic relevance in breast cancer, particularly in TNBC. By impairing oncogenic signaling and enhancing the efficacy of PARP inhibitors, piR-775 emerges as a promising biomarker and a candidate for combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858c8d8d1032bace8d569826ea440beaf678ad07" target='_blank'>
              Piwi-interacting RNA 775 (piR-775) predicts favorable prognosis and regulates cell cycle and DNA damage response pathways in breast cancer
              </a>
            </td>
          <td>
            Bushra Yasin Abohalawa, Hend Ghasan Eldous, R. Elango, Radhakrishnan Vishnubalaji, Sameera Rashid, Khalid Ouararhni, Ayman Al Haj Zen, Nehad M. Alajez
          </td>
          <td>2025-11-04</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Somatic structural variants (SVs) play a crucial role in tumor development and evolution, yet their identification remains challenging, partly due to limitations in current reference genomes. We present a systematic evaluation of diploid donor-specific assemblies (DSAs)—generated based on hybrid long-read sequencing data—as the reference genome for detecting somatic SVs. We analyzed six tumor-normal cell line pairs, using the EchoSV tool we developed to consolidate haplotype-based SVs into a single DSA-based set and to compare SVs across reference genomes. Across Illumina, PacBio HiFi, and Oxford Nanopore Technology (ONT) data, DSA-based analysis improved read-mapping quality, identified over 20% additional SVs compared to GRCh38 and CHM13, and reduced germline artifacts. Most DSA-specific SVs were isolated deletions or insertions in repetitive elements, especially in satellite regions. By tracking sequence-context differences, we identified scenarios in which DSAs enabled detection of SVs missed on GRCh38/CHM13, and confirmed their functional impact with RNA-seq. These results highlight the value of integrating diploid DSAs into somatic SV analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a18c3e5a3efb8d34cfabcd22cb306cbbbfb02310" target='_blank'>
              Diploid donor-specific assembly enhances somatic structural variant detection in cancer genomes
              </a>
            </td>
          <td>
            Yuwei Zhang, Han Qu, Qian Qin, Heng Li, Peter J. Park
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute promyelocytic leukemia (APL) is a subtype of acute myeloid leukemia (AML), characterized by a fusion between the PML and RARA genes and by a block in the myeloid maturation at the promyelocytic stage. This study investigates the epigenetic landscape of APL by integrating ChIP-seq data on eight histone modifications and RNA-seq in APL as well as non-APL AML. APL showed a distinct chromatin profile that differed from non-APL AML. We describe APL-specific changes in H3K27ac, H3K9me3, and H3K27me3 with impact on enhancer activity, repression of transposable elements, and Polycomb regulated gene repression. The APL-specific H3K27ac pattern identifies APL-specific enhancer and super-enhancer regions, including a subset of enhancers that are bound by the PML-RARA fusion protein. While chromatin bound specifically by PML-RARA were dominantly active, APL was also characterized by gain of APL-specific heterochromatin states with significant gains of H3K9me3 enriched lamina-associated domains and the transposable elements LINE, LTR, and SINE. These findings suggest a unique enhancer and heterochromatin profile in APL, with implications for transcription regulation and treatment response. These findings offer novel insights into the pathogenesis of APL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4d4d5354c6d704bfe8c1f882f567114dfdc0fb" target='_blank'>
              Disease-specific epigenetic deregulation of enhancers, transposons, and polycomb targets in acute promyelocytic leukemia
              </a>
            </td>
          <td>
            Xiangfu Zhong, Lina Cordeddu, Angelica Gamboa-Cedeno, S. Bengtzén, Karl Ekwall, Andreas Lennartsson, Sören Lehmann
          </td>
          <td>2025-10-30</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1499e0bd4fc727a3ec07930d21e6808e9e69593" target='_blank'>
              Global Mapping of Combinatorial Chromatin Regulatory Events Using Hi-Plex CUT&Tag
              </a>
            </td>
          <td>
            Yuan Liao, Weiqiang Zhou, Min-Zhi Jiang, Kuai Yu, Zixiao Jolene Wang, Thomas Drummond, Meng Ding, Yingming Zhao, Ignacio Pino, Sean D. Taverna, Hongkai Ji, Heng Zhu
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Chromatin organization shapes gene regulation by linking distal elements across megabase scales, yet most predictive genomics models still treat the genome as linear, without incorporating three-dimensional structure. Hi-C provides genome-wide chromatin conformation information, but its contact maps are population-averaged, distance-biased, and noisy, obscuring the biologically specific contacts. We present CHROME, a framework built on a self-avoiding polymer ensemble null model that identifies physically specific, non-random Hi-C contacts. By integrating these contacts into graph representations, CHROME enables efficient information transfer across spatially connected loci. It integrates sequence, chromatin accessibility, or pre-trained embeddings into a graph attention architecture to predict cell line-specific ChIP-seq profiles, consistently outperforming local encoder baselines and generalizing to an unseen cell line. The resulting graph embeddings also enhance prediction on tissue-specific eQTL and ClinVar variant pathogenicity, outperforming local sequence-based embeddings. Beyond predictive performance, CHROME provides interpretability through attention-derived neighbor-to-center contributions that reveal how spatially connected loci influence local regulatory activity over multi-megabase distances. Together, these results show that incorporating physically validated chromatin interactions enables more accurate and interpretable modeling of gene regulation and variant effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0bce1e03b52a16226b35a4a2a7a29a22d1542bc" target='_blank'>
              A Chromatin-Structure-Guided Framework for Predictive and Interpretable Regulatory Genomics
              </a>
            </td>
          <td>
            Bowei Ye, Lin Du, Min Chen, Yang Dai, Ao Ma, Jie Liang
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2134381d445b860d33da1c40fb581783fd188ec" target='_blank'>
              Widespread low-affinity motifs enhance chromatin accessibility and regulatory potential in mESCs
              </a>
            </td>
          <td>
            Melanie Weilert, Kaelan J. Brennan, Khyati Dalal, S. Krueger, Haining Jiang, Rosa Martinez-Corral, J. Zeitlinger
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="DNA double-strand break (DSB) repair via homologous recombination (HR) is critical for maintaining genomic integrity and requires proper DNA end resection to generate single-stranded DNA (ssDNA) overhangs. However, the mechanisms governing this critical step in cells remains poorly understood. Here, we report that GNL3, a nucleolar GTP-binding protein, plays a key role in HR repair via regulating DNA end resection dependently on its SUMOylation. Ectopic expression of wild-type, but not the SUMO-defective K196R mutant, GNL3 completely abolished DNA damage response induced by knockdown of endogenous GNL3. GNL3 interacts with the BLM-DNA2 helicase-nuclease complex and is critical for DNA end resection and subsequent loading of RPA and RAD51. This interaction requires SUMOylation and SUMO-interacting motifs (SIMs) in both proteins. We further demonstrate that USP36, a nucleolar deubiquitinating enzyme, functions as a novel SUMO ligase for GNL3, while the SUMO protease SENP3 deSUMOylates GNL3. Notably, several breast cancer-derived GNL3 variants that disrupt its SUMOylation or SIM fail to interact with the BLM-DNA2 complex. Knockdown of GNL3 sensitizes HR-proficient breast cancer cells to etoposide or Olaparib treatment. Together, our results reveal that GNL3 SUMOylation is crucial for HR repair and suggest that targeting GNL3 SUMOylation may induce HR deficiency, thereby sensitizing breast cancers to DNA damage-inducing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae37de99c671716653a1ca2bda6a664db37f1de" target='_blank'>
              GNL3 SUMOylation is essential for DNA double-strand break repair by homologous recombination
              </a>
            </td>
          <td>
            Yunhan Yang, Yanping Li, Roselyn S. Dai, Canping Chen, Keith Zientek, Ashok Reddy, Zheng Xia, Rosie Sears, Xiao-Xin Sun, Mu-Shui Dai
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Polycomb targeted loci form long-range chromatin interactions independent of CTCF-cohesion and are demarcated by low DNA methylation(1–3). Polycomb targets form extremely long-range loops (long Polycomb loops) that can separate anchors up to 60 Mb. These loops predominantly occur in cells of self-renewal status such as human hematopoietic stem cells (HSC) and mouse embryonic stem cells (ESC), but rarely in cell lines. To identify long Polycomb loops in cancer, we initiated a pan-cancer survey of long Polycomb loops in a collection of 235 tumor samples (33 acute myeloid leukemias, 17 Breast cancer samples, 63 pediatric brain tumors, 70 prostate cancers, and 42 colon cancers). We found most cancers, including all prostate cancers and colon cancers, lack long Polycomb loops except pediatric brain tumors and certain AMLs. In pediatric brain tumors, we found the ependymoma PFA subtype (which shows globally depleted H3K27me3) notably displayed strong long Polycomb loop interactions. AML displayed more diverse levels of long Polycomb loop interactions. Most AMLs lost both long Polycomb loops with Polycomb binding loss at the loop anchors. Whereas 10% of AMLs retain long Polycomb loops as strong as in HSCs. These AMLs recurrently carry mutations in CEBPA and STAG2, which are not associated with the Polycomb complex or DNA methylation machinery. We found that long Polycomb loop strong AML is sensitive to EZH2 inhibition, which induces cell differentiation. Conversely, PRC1 component dependency can be used to predict long Polycomb loop formation in B-cell lymphoblastic leukemia cell lines. Our analysis suggests that the oncogenesis process antagonizes the long Polycomb loop maintenance in most cancers, yet certain cancers may still preserve strong long Polycomb loops from cell-of-origin. The maintenance of long Polycomb loops sensitizes cells to Polycomb inhibition, indicating that such loops could be an epigenomic biomarker for pharmacological or genetic Polycomb inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd40648aae14140db85bd195a593affdaa280e98" target='_blank'>
              Pan-cancer 3D genomic analysis revealed extremely long Polycomb loops as the biomarker for sensitivity to Polycomb inhibition
              </a>
            </td>
          <td>
            Sean Moran, Zhong Fan, Miriam Zanovello, Feng Fan, David Xue Qing Wang, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Ends of linear chromosomes require protection from host repair machinery that otherwise will mistake them for damaged DNA. The E. coli bacteriophage N15 harbors a linear genome with covalently closed hairpin ends formed by the phage-encoded telomere resolvase TelN. The double-strand break repair complex Mre11-Rad50 (MR, SbcCD in E. coli) specifically targets DNA termini, yet how hairpin telomeres evade host nuclease degradation in bacteria remains unknown. Here, we demonstrate that TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from MR processing in E. coli. Using a combination of genetic and biochemical approaches, we show that this protective function requires both TelN sequence-specific DNA binding and species-specific protein-protein interactions. Notably, we found that protection is independent of TelN’s resolution activity and does not require the C-terminal domains of TelN. Our findings reveal a potentially broad mechanism of telomere protection, providing insights into a conserved regulation of MR activity at chromosome ends across the tree of life. Chromosome ends require protection from the DNA repair machinery. This work demonstrates that TelN protein, the telomere resolvase of E. coli phage N15, protects hairpin telomeres from processing by the host Mre11-Rad50 nuclease through distinct molecular mechanisms. TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from Mre11-Rad50 processing. Protective functions require both TelN sequence-specific DNA binding and species-specific protein-protein interactions with the E. coli MR complex. Hairpin telomere protection is independent of TelN's resolution activity and of the C-terminal domains of TelN, locating a core protection mechanism to conserved N-terminal domains. TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from Mre11-Rad50 processing. Protective functions require both TelN sequence-specific DNA binding and species-specific protein-protein interactions with the E. coli MR complex. Hairpin telomere protection is independent of TelN's resolution activity and of the C-terminal domains of TelN, locating a core protection mechanism to conserved N-terminal domains. Bacteriophages with linear genomes evade telomere processing by bacterial host MR complexes via a telomere-resolution-independent function of their telomere resolvase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Core cis-regulatory elements known as enhancers play a central role in enabling precise transcriptional regulation of target genes that control diverse cellular functions and developmental processes. These enhancers are often transcribed in both directions, producing long non-coding transcripts referred to as enhancer RNAs (eRNAs). The expression of eRNAs is closely linked to active chromatin features, such as H3K27ac and co-activator recruitment, and functionally contributes to the transcriptional activation of target genes. Nevertheless, the detection and quantification of eRNAs remain challenging, especially when they overlap with host-gene transcription. To address this, we present a standardized, user-friendly computational workflow for analyzing enhancer transcription from nascent RNA sequencing data. The protocol guides users through data preprocessing, read mapping, and quality control, followed by strand-specific quantification of enhancer-associated transcription, with dedicated procedures for intragenic enhancers where signal assignment is complex. Visualization modules enable clear inspection of enhancer activity across genomic contexts, and built-in options support analyses of both intergenic and intragenic enhancers. Designed for researchers with limited bioinformatics expertise, this workflow provides a practical framework for consistent, reproducible, and scalable studies of enhancer transcription, facilitating broader application of enhancer biology across diverse systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f855810e0abb37436dd298cbb475c3b494ab8147" target='_blank'>
              A Computational Pipeline for Intergenic/Intragenic Enhancer RNA Quantification in Mouse Embryonic Stem Cells.
              </a>
            </td>
          <td>
            Myunggeun Oh, Seunghwa Jeong, Seung-Kyoon Kim
          </td>
          <td>2025-10-28</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past decade, non-viral DNA template delivery has been used with engineered nucleases to target single-stranded DNA sequences in hematopoietic stem and progenitor cells. While effective for gene therapy, this method is limited to short DNA donor templates, restricting its applications to gene corrections. To expand its scope, we developed an editing process using kilobase-long circular single-stranded DNA donor templates and TALEN technology. Our results show that the CssDNA editing process achieves high gene insertion frequency in HSPCs. Compared to AAV-edited HSPCs, CssDNA-edited HSPCs show a higher propensity to engraft and maintain gene edits in a female NCG murine model. This positive outcome is partly due to higher levels of primitive edited HSPCs, a more quiescent metabolic state, and elevated expression of bone marrow niche adhesion markers. Our findings highlight the strong potential of CssDNA as a universal, scalable and efficient non-viral DNA template for gene therapy applications. Letort et al. demonstrate that Circular single-stranded DNA enables efficient and precise gene insertion in hematopoietic stem cells, outperforming other DNA delivery formats and supporting progress toward scalable next-generation gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8334d5fc2c8d3325ef3a32ecf5b25cddb9e9e5c9" target='_blank'>
              Circular single stranded DNA potentiates non-viral gene insertion in hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, Diane Le Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ce53b192feb215e073df6046be982c6e7a222cf" target='_blank'>
              Analysis of the three-dimensional genome of exhausted CD8+ T cells reveals a critical role of IRF8 in their differentiation and functions in cancer
              </a>
            </td>
          <td>
            Ruifeng Li, Kun Wei, Qinli Sun, Bowen Xie, Peng Wei, Tian Xie, Xinyi Guo, Yongzhen Chen, Birui Pan, Zixuan Zhao, Ling Ni, Chen Dong
          </td>
          <td>2025-11-18</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db26ccc6dc7e76ac6a63d600396d97f0ca6b89f2" target='_blank'>
              Passenger mutations link cellular origin and transcriptional identity in human lung adenocarcinomas.
              </a>
            </td>
          <td>
            Sukanya Panja, Padmaja Mantri, K. E. Johnson, J. Andrade-Martínez, S. Yang, Aditya Deshpande, Huasong Tian, Shaham Beg, Kentaro Ohara, Alessandro Leal, Joel Rosiene, Marlon Stoeckius, Peter Smibert, William D. Travis, J. Mosquera, P. Polak, Marcin Imieliński
          </td>
          <td>2025-11-27</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Dynamic patterns of gene expression drive animal development. These transcriptional dynamics depend on the interplay of diverse developmental enhancers within complex regulatory landscapes. Here, we employ a single-molecule genomic method, Fiber-Seq, to capture the simultaneous transcriptional and regulatory states of millions of 20-30 kb chromatin fibers in the 2-4hr Drosophila embryo. These snapshots of genes and their surrounding regulatory context reveal that the basic unit of transcription at highly transcribed genes is a convoy of 2-14 tightly spaced polymerases (Pol II). Further, a subset of templates exhibits hyperbursting, whereby most or all nucleosomes are evicted from the gene body facilitating maximum rates of transcription. We demonstrate that hyperbursting is achieved through cooperation of numerous cis-regulatory elements, and that the resulting nucleosome eviction appears to trigger a novel silencer in the ftz 3’ UTR. We anticipate that similar mechanisms are used by vertebrate processes requiring intense transcription like somitogenesis and erythropoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ae68e3fd0860af19fc0e759f8ef3715a02c5a3" target='_blank'>
              Comprehensive Single Molecule View of Transcriptional Dynamics in Development
              </a>
            </td>
          <td>
            Thomas W. Tullius, Tohn Borjigin, Michael Levine
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe27e8860b31d75137bdc5403fdf3407806c753" target='_blank'>
              Global re-organisation of genome architecture at the transition to gametogenesis
              </a>
            </td>
          <td>
            Tien-Chi Huang, Maria Rigau, V. Malysheva, Chad Whildling, Stella Siciliani, Jingyu Li, Irina Balaguer Balsells, Pavel Artemov, Camille Dion, M. Spivakov, Juan M. Vaquerizas, Petra Hajkova
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Among them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed a synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2446ce02999836acb1c038122ee522f1878bf0c3" target='_blank'>
              TopBP1 biomolecular condensates as a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, B. Bordignon, C. Hassen-Khodja, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Laurent Chaloin, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2025-10-21</td>
          <td>eLife</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ZNF16 (also known as HZF1 and KOX9) is a multi-C2H2 zinc finger protein first identified via its expression in human T-cells and shown to have a role in blood cell differentiation. ZNF16 was later shown to be ubiquitously expressed in a variety of fetal and adult tissues, suggesting a broader function. In this study, we confirm the ubiquitous expression of ZNF16 in a variety of cancer and non-cancer cell lines and show that ZNF16 depletion reduces cell viability in all cell lines tested. Furthermore, we show that ZNF16 localizes to the nucleolus in a transcription-dependent manner, interacts with the intergenic spacer region of the rDNA and promotes rDNA transcription. Additionally, RNA-seq experiments after ZNF16 depletion revealed that ZNF16 also has roles in a variety of pathways including ECM-receptor interaction, focal adhesions, cytokine-cytokine receptor interactions, human papillomavirus (HPV) infection and cancer pathways. These findings are consistent with broader roles for ZNF16, including the regulation of nucleolar function, a process that is essential for all cells, and provide evidence at the cellular/molecular level of its role in the regulation of cancer-associated genes (e.g., NRAS, BIRC3, EGFR).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dc067e77fc25c3bffbc7d4cd3e9f4b28158e78" target='_blank'>
              ZNF16 is a nucleolar-associated protein that regulates expression of the rDNA and cancer-associated genes
              </a>
            </td>
          <td>
            Chelsea L. George, Laura A. Espinoza Quevedo, Jason Paratore, Matthew J. Alcaraz, Arlene P. Levario, Y. Rahmatallah, Galina V. Glazko, Nathan S. Reyna, Laura A. Diaz-Martinez
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bdb65fc0708e21a785db03ef4faf2be314ea734" target='_blank'>
              E2 displacement of CIP2A from TOPBP1 activates the DNA damage response during papillomavirus life cycles
              </a>
            </td>
          <td>
            A. T. Prabhakar, C. James, Allison Q. Nguyen, Phoebe Bridy, Xu Wang, Ryan J. Zoldork, Jenny D Roe, Austin J. Witt, Srimanya S. Panidepu, Molly L Bristol, Krista Dalton, Andreas Wieland, Ahmed Diab, Anthony C. Faber, Elliot J Androphy, Devraj Basu, Paul F. Lambert, Megan E. Spurgeon, I. Morgan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0957eb10bf767ffc74bc27558270080adbc791c6" target='_blank'>
              Pan-cancer analysis reveals mtDNA copy number as a key determinant of mutational load and disease progression in cancer
              </a>
            </td>
          <td>
            Anamika Acharyya, Riddhiman Dhar
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45dd551579bb493b99c7ec5a7c44f7e95f00ebc0" target='_blank'>
              Single cell whole genome and transcriptome sequencing links somatic mutations to cell identity and ancestry
              </a>
            </td>
          <td>
            Abhiram Natu, Mrunal K. Dehankar, Reenal Pattni, M. Šuvakov, Dmitrii Olisov, Livia Tomasini, Yeongjun Jang, Yiling Huang, Eva Benito-Garragori, Patrick Hasenfeld, Jan O. Korbel, Alexander E. Urban, A. Abyzov, F. Vaccarino
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9cba62bd659ab0fed0834794015eb81e0685b90" target='_blank'>
              Integrative Multi-Omics Identifies CDK1 as a Key Signaling Regulator of CD4+ T Cell Effector Function
              </a>
            </td>
          <td>
            N. Servaas, H. G. Bauersachs, Luisa Abreu, A. Claringbould, I. Berest, Jennifer J. Schwarz, Frank Stein, Maria Fälth-Savitski, Lena Eismann, James P. Reddington, Judith B. Zaugg
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Transposable elements (TEs) are often epigenetically repressed in eukaryotic cells, but still affect the molecular state of the cell in certain contexts. A flurry of recent studies have elucidated new effects of TE sequences in eukaryotic cells. We review these emerging molecular effects of TEs, including a variety of new mechanisms by which TE sequences affect the cell, including pre- and post-transcriptional regulation of gene expression; cell-to-cell transmission of genes within a multicellular organism through virus-like activity; and RNA-guided DNA insertion. Recent demonstration of TE-guided genome editing underscores the importance of these investigations for both basic and translational research. Future work is needed to continue to unravel the molecular effects of TE sequences across developmental stages, across cell types, and in various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3445e991f8e7a3fbd17d3feddadbe0f7c8e2d56" target='_blank'>
              Molecular effects of transposable element sequences in mammalian cells
              </a>
            </td>
          <td>
            Ming-Ching C. Wen, Joshua D. Welch
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Centromeres are fundamental chromosomal structures that ensure accurate chromosome segregation during cell division. Despite their conserved and essential role in maintaining genomic stability, centromeres are subject to rapid evolutionary change. At the heart of centromere identity is the histone H3 variant CENP-A, an epigenetic mark that defines and propagates active centromeres and is essential for their function. Recent evidence supports a rapid evolution of centromere DNA sequences but also suggests a certain degree of flexibility in CENP-A deposition and propagation. The phenomenon of centromere drift, recently observed in humans, highlights how the dynamic repositioning of CENP-A and associated epigenetic environment over time maintains a regulated equilibrium, ensuring centromere function despite positional variation. Understanding these processes is crucial for unraveling centromere dynamics and their broader implications for genome stability and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0e9c5778af501dce621770da4beeabba7210f5" target='_blank'>
              The dynamic centromere
              </a>
            </td>
          <td>
            Angela Enriquez, Yael Nechemia-Arbely
          </td>
          <td>2025-10-16</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Pioneer transcription factors act at the top of gene-regulatory networks by promoting accessible chromatin at the cis-regulatory regions that drive gene expression. Despite their ability to bind closed chromatin, pioneer factors occupy distinct binding sites in different tissues. The pioneer factor Zelda promotes the undifferentiated fate in both the early Drosophila embryo and in the neural stem cells (neuroblasts) of the larval brain. Tissue-specific binding by Zelda identifies cell-type specific enhancers, which are enriched for different DNA-sequence motifs. We investigated the features that promoted cell-type specific occupancy by testing the role of conserved, structured protein domains in the capacity of Zelda to promote the embryonic and neuroblast cell fates. We unexpectedly identified that the most deeply conserved region in Zelda, the second zinc finger, has opposing functions in the embryo and neuroblasts. We showed that this zinc finger is a previously unrecognized DNA-binding domain that is specifically required for Zelda binding to a G-rich motif in larval neuroblasts. The pioneering function of Zelda depends largely on the C-terminal cluster of zinc fingers that promotes binding in the early embryo, suggesting that pioneer function may depend on how Zelda engages the genome. As opposed to co-factor expression or chromatin environment, our data identify tissue-specific usage of two, widely separated DNA-binding domains as the mechanism controlling tissue-specific binding and activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a61d22a6d0218cee59e6ebeaffed9e6e8bc1147b" target='_blank'>
              Differential usage of two, distinct DNA-binding domains regulates tissue-specific occupancy of the pioneer factor Zelda
              </a>
            </td>
          <td>
            Eliana F. Torres-Zelada, H. Komori, Hsiao-Yun Liu, Elizabeth D. Larson, A. Yang, Zoe A. Fitzpatrick, Timothy R. Hughes, C. Rushlow, Cheng-Yu Lee, Melissa M. Harrison
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The cattle genome is crucial for understanding ruminant biology, but it remains incomplete. Here we present a telomere-to-telomere haplotype-resolved X chromosome and four autosomes of cattle in a near-complete assembly that is 431 Mb (16%) longer than the current reference genome. Using this assembly (UOA_Wagyu_1) we identify 738 new protein-coding genes and support the characterization of centromeric repeats, identification of transposable elements, and enabled the detection of 2397 more structural variants from 20 Wagyu animals than using ARS-UCD2.0. We find that the cattle X centromere is a natural neocentromere with highly identical inverted repeats, no bovine satellite repeats, low CENP-A signal, low methylation, and low CpG content, in contrast to the autosomal centromeres that are comprised of typical bovine satellite repeats and epigenetic features. Our results suggest it likely formed from transposable element expansion and CpG deamination, suggesting dynamic evolution. We find eighteen X-pseudoautosomal region genes have conserved testes expression between cattle and apes. We also find all cattle X neocentromere protein-coding genes are expressed in testes, which suggests they potentially play a role in reproduction. Advancements in sequencing technologies and assemblers have enabled us to generate a complete, haplotype-resolved X chromosome in cattle. This study discovers the cattle X centromere is a natural neocentromere and characterises its genetic and epigenetic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3dbd28f2812d9774920d7d01884482758e1cf6a" target='_blank'>
              Insights into natural neocentromere evolution from a cattle T2T X chromosome
              </a>
            </td>
          <td>
            Paulene S Pineda, C. Macphillamy, Yan Ren, Tong Chen, Luan Zhong, David L. Adelson, Carey Dessaix, Jose Perez-Silva, Leanne Haggerty, Fergal J. Martin, Cynthia D. K. Bottema, Wayne S Pitchford, Benjamin D. Rosen, Timothy P. L. Smith, W. Low
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA cytosine modifications, including 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), are key epigenetic regulators with distinct functions. Dissecting the ternary code (C, 5mC, 5hmC) across tissues and cell types remains a critical priority due to the limitations of traditional profiling methods based on bisulfite conversion. Here, we leverage the combined bisulfite and enzymatic (bACE) conversion with the Mouse Methylation BeadChip to generate 265 base-resolution ternary-code modification maps of 5mC and 5hmC across 29 mouse tissue types spanning 8–76 weeks of age and both sexes. Our atlas reveals a complex grammar of 5hmC distribution, jointly shaped by cell mitotic activity, chromatin states, and interplay with 5mC at the same and neighboring CpG sites. Of note, we demonstrate that 5hmC significantly complements 5mC-based biomarkers in delineating cell identity in both brain and non-brain tissues. Each modification state, including 5hmC alone, accurately discriminates tissue types, enabling high-precision machine learning classification of epigenetic identity. Furthermore, the ternary methylome variations extensively implicate gene transcriptional variation, with age-related changes correlated with gene expression in a tissue-dependent manner. Our work reveals how tissue, sex, and age jointly govern the dynamics of the two cytosine modifications, augments the scope of DNA modification biomarker discovery, and provides a reference atlas to explore epigenetic dynamics in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf56e8a468cf8f7c8ba62217313ee3ee7ed7a698" target='_blank'>
              A ternary-code DNA methylome atlas of mouse tissues
              </a>
            </td>
          <td>
            S. Lee, David C Goldberg, Cameron Cloud, Jared B. Parker, Christopher J. Krapp, Christian E. Loo, Elliot Kim, Ivan Zhao, Chengcheng Jin, R. Porecha, M. Bartolomei, Rahul M. Kohli, Wanding Zhou
          </td>
          <td>2025-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Homologous recombination repair (HRR) is a cellular pathway for high-fidelity double strand DNA break repair that uses the sister chromatid as a guide to ensure chromosomal integrity and cell viability. Deficiency in the HRR pathway (HRD) can sensitize tumors to poly (ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapy, offering an avenue to identify patients who may benefit from targeted therapies. HRD signature (HRDsig) is a pan-solid-tumor biomarker on the FoundationOne®CDx (F1CDx®) assay that employs a DNA scar-based approach to calculate a score based on copy number features (e.g., segment size, oscillation patterns, and breakpoints per chromosome arm) and does not rely on HRR gene alterations, enabling detection of genomic and epigenetic mechanisms of HRD. After finalizing the HRDsig algorithm, analytical validation was conducted in a CAP-accredited, CLIA-certified laboratory on 278 solid tumor and normal tissue specimens. HRDsig results were compared with an independent HRD biomarker, defined by the presence of a reversion mutation restoring HRR gene function. In this evaluation, 100 HRD-positive and 126 HRD-negative samples showed a positive percent agreement of 90.00% and a negative percent agreement of 94.44%. The limit of detection (LoD) was estimated at 23.04% tumor purity, with the limit of blank (LoB) confirmed as zero in 60 normal tissue replicates. Reproducibility testing on 11 positive and 11 negative samples across multiple labs, reagent lots, and sequencers yielded agreement in 99.49% of positive and 99.73% of negative replicates. HRDsig status remained consistent in the presence of interfering substances, demonstrating 100% concordance in spiked samples. These validation results underscore the high analytical concordance, low false-positive rate, and overall robustness of HRDsig for reliable assessment of homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e556ef8cfb0b30b29bee7661332474ddfe38a5dd" target='_blank'>
              Analytical validation of a homologous recombination deficiency signature (HRDsig) in pan-tumor tissue samples
              </a>
            </td>
          <td>
            Wenshu Li, Jeffrey A. Leibowitz, Shuoguo Wang, Louisa Walker, Chang Xu, Kuei-Ting Chen, Alexa B Schrock, Jason D Hughes, Nimesh R. Patel, J. Elvin, Lauren L. Ritterhouse, E. Sokol, Garrett Frampton, Lucas Dennis, Bahar Yilmazel, Brennan Decker
          </td>
          <td>2025-11-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background The identification and characterisation of somatic cancer driver mutations in the non-coding genome remains challenging. Objective To broadly characterise non-coding driver mutations for pancreatic ductal adenocarcinoma (PDAC). Design Using mutation calls from whole-genome sequence data in PDACs and genome-scale maps of accessible gene regulatory regions in normal-derived and tumour-derived pancreatic samples, we analysed enrichment of non-coding mutations in gene regulatory regions relevant to normal-derived and tumour-derived pancreatic contexts. Functional follow-up of potential driver mutations was performed using chromatin interaction analyses, massively parallel reporter assays (MPRA) and targeted analysis of selected non-coding somatic mutations (NCSMs). Results We first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3614 NCSMs. Chromatin interaction analysis identified 416 potential target genes and MPRA revealed 178 NCSMs impacting reporter activity (19.45% of those tested). Targeted luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near ZFP36L2 and CDKN2A. For the former, CRISPR interference identified ZFP36L2 as a target gene (16.0–24.0% reduced expression, p=0.023–0.0047), and growth inhibition after overexpression of ZFP36L2 (4.1–14.1-fold reduction, p=6.0×10–4 − 3.2×10–3) implicates a possible tumour suppressor function. Conclusion Our integrative approach provides a catalogue of potential non-coding driver mutations and nominates ZFP36L2 as a novel PDAC driver gene with a likely tumour suppressor function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4adc9cdafe8f3e335c7d27583752292d933404cf" target='_blank'>
              Large-scale multiomic analysis identifies non-coding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jun Zhong, Aidan O'Brien, Minal B Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Sudipto Das, Maura J. O’Neill, Courtney D Dill, Andrew D. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E Connelly, Jianxin Shi, Efsun Arda, Jason W Hoskins, L. Amundadottir
          </td>
          <td>2025-10-08</td>
          <td>Gut</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Summary Humans exhibit significant phenotypic differences from other great apes, yet pinpointing the underlying genetic changes has been limited by incomplete reference genomes and a reliance on single assemblies to represent a species. We aligned 20 telomere-to-telomere (T2T) assemblies spanning great ape divergence and variation to define 1,596 Consensus HAQERs (consensus human ancestor quickly evolved regions), regions that diverged rapidly between the human-chimpanzee ancestor and an ancestral node of modern humans. Unlike prior HAQER sets, Consensus HAQERs incorporate population variation, reducing the likelihood of intraspecies variation appearing as interspecies divergence. Consensus HAQERs exhibit signatures of elevated mutation rates, ancient positive selection, bivalent regulatory function, are enriched in disease-linked loci, and often emerged in previously inaccessible repetitive DNA. Through multiplex, single-cell enhancer assays, we identify HAQERs as active enhancers in the developing brain and cardiomyocytes, highlighting their potential contributions to human-specific gene regulation across multiple tissues. Highlights ● Telomere-to-telomere alignments of diverse human and great ape genomes identify 1,596 Consensus HAQERs, regions of rapid sequence divergence separating human ancestors from other great apes. ● Consensus HAQERs exhibit signatures of both elevated mutation rates and ancient positive selection. ● HAQERs are enriched in bivalent regulatory elements and disease-linked loci. ● Multiplex, single-cell gene regulatory assays identify HAQERs as enhancers in the developing heart and brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00bd3c45b8187e63a71fea1f33c5572d84f0153e" target='_blank'>
              Intraspecific sequence variation and complete genomes refine the identification of rapidly evolved regions in humans
              </a>
            </td>
          <td>
            Yanting Luo, Riley J. Mangan, Seth Weaver, Sarah A. Zhao, Federica Mosti, Michael C. Thomas, Ravi Karra, D. Silver, M. Kellis, Craig B. Lowe
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Central nervous system germ cell tumors (CNSGCTs) are the most enigmatic brain tumors. They consist of several histologically distinct types of tumors, which are generally sub-grouped into germinomas or non-germinomatous germ cell tumors (NGGCTs). While germinomas are characterized by alterations in the MAPK/PI3K-pathway genes and global hypomethylation, pathogenesis of NGGCTs is largely unknown. To investigate genetic and epigenetic mechanisms of CNSGCT development, 33 CNSGCTs including 7 germinomas and 26 NGGCTs of various histology collected through the Intracranial Germ Cell Tumor Genome Analysis Consortium and Japan Children’s Cancer Group, together with their matched normal DNA, were subjected to a long-read whole genome sequencing using the PromethION system from Oxford Nanopore Technologies. The mean reading depth was 39X for tumor specimen and 31x for blood DNA. The mean N50 value was 9,565 bp. The mean/median number of structural abnormalities was 16.6/16 for germinomas and 27.9/10.5 for NGGCTs, which was not statistically significantly different. A 2 kb deletion involving an exon of USP28 was observed in a single mixed-germ cell tumor case. Deletions involving PRKN were observed in 4 cases. The mean number of ectopic somatic insertions of the LINE-1 element was 1.8 per case. A single mixed GCT had 21 LINE-1 insertions, one of which was located within an intron of NTRK2. The mean of the whole genome DNA methylation level was 38.2% in germinomas and 75.9% in NGGCTs (p=0.0005). Germinomas mostly had biallelic DNA hypomethylation in the imprinted genes, consistent with their presumed primordial germ cell (PGC) origin. On the other hand, NGGCTs showed various levels of methylation in the imprinted genes, suggesting that they may also have originated from PGCs, however at different stages from that of germinomas. Taken together, our whole genome DNA methylation haplotyping provided a new clue to elucidate the pathogenesis of NGGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a0bfd993cd8d6ea3ee9b1101edf3e65daa2bfa" target='_blank'>
              EPCO-07. A WHOLE GENOME DNA METHYLATION ANALYSIS REVEALED DISTINCT MECHANISMS OF DEVELOPMENT IN DIVERSE TYPES OF CENTRAL NERVOUS SYSTEM GERM CELL TUMORS
              </a>
            </td>
          <td>
            Shoko Yoshimoto, Hirokazu Takami, Erina Ishikawa, Yuta Kuze, Takako Yoshioka, Junko Hirato, Nobuhito Saito, Toshihiro Kumabe, Ryo Nishikawa, Yutaka Suzuki, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801ad174dd38ca1cc6ebc678d046b17d34c65d92" target='_blank'>
              Epigenetic gene regulation is controlled by distinct regulatory complexes utilizing specialized paralogs of TELOMERE REPEAT BINDING FACTORS
              </a>
            </td>
          <td>
            Maik Mendler, Kristin Krause, Simone Zündorf, Prathamesh Sannak, Petra Tänzler, S. Stolze, Hirofumi Nakagami, F. Turck
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Gene expression and regulation with or without alternative splicing are crucial for tissues and cells to function correctly. They have been studied from three almost independent perspectives at the genome level: 1- Recognition of splicing quantitative trait loci (sQTLs), 2- Expression quantitative trait loci (eQTLs) recognition, and 3- Recognition of longer-range physical chromatin interactions between genome segments which model 3D dynamics of cells and tissues. Even though the associations between eQTLs and longer-range chromatin interactions have been previously studied, a similar relationship between sQTLs and chromatin interactions has yet to be analyzed. In this case, examining whether sQTLs control the alternative splicing of their target genes’ mRNA via physically interacting genome segments is crucial. We have jointly analyzed high-throughput chromatin conformation capture (Hi-C) and sQTL datasets over eight human cancer tissues. We have discovered a positive association between the number of genes having sQTLs and chromatin interaction frequency. Such a positive association still exists when we also control for eQTLs. Additionally, sQTLs and their target genes generally exist inside identical topologically associating domains (TADs). These findings are observed over the whole set of analyzed cancer types and functional subsets of the sQTL dataset, such as survival-related sQTLs. Furthermore, tissue-specific sQTLs are statistically enriched in tissue-specific frequently interacting regions (FIREs) in 6 out of 8 human cancer tissues (Chronic Myeloid Leukemia, Colon Adenocarcinoma, Acute Myeloid Leukemia, Lung Adenocarcinoma, Prostate Cancer, Sarcoma). Our sQTL and Hi-C datasets have shown the existence of closer spatial distance between sQTLs and their target genes with possible alternative splicing across several different cancer types in humans. Such a closer spatial distance also exists, independent of whether we integrate eQTLs into the analysis. We found that sQTLs regulate alternative splicing through chromatin interactions. Source code of the analysis in this research is available on https://github.com/seferlab/sqtlhic .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d307f9f6f23c37ffce97f36f862782001b30f0c" target='_blank'>
              Joint analysis of sQTL and Hi-C reveals spatial proximity between sQTLs and target genes in cancer tissues
              </a>
            </td>
          <td>
            Batuhan Eralp, Emre Sefer
          </td>
          <td>2025-10-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Natural chromosomal transformation (NCT) in Bacillus subtilis requires RecA and its accessory proteins including RecX and RecD2. Inactivation of the nucleoid-associated protein (NAP) Hbsu, Rok, or LrpC results in a dual effect: it enhances NCT but exacerbates the NCT defect of ΔrecX cells. Purified EbfC exhibits characteristic features of NAPs: it binds both single- and double-stranded DNA, protects them from degradation, and forms higher-order protein–DNA complexes via its DNA bridging activity. NCT is reduced upon EbfC inactivation but enhanced by ebfC overexpression. hbs55, Δrok, or ΔlrpC mutations suppress the NCT defect of ΔebfC, yet synergistically increase NCT upon ebfC overexpression. The NCT defect in ΔrecD2 cells is worsened by ebfC overexpression or by hbs55, Δrok, or ΔlrpC mutations. The nucleoid was more compacted in ΔebfC cells, an effect counteracted by hbs55, Δrok, or ΔlrpC mutations. EbfC contributes to DNA repair, and ebfC is epistatic to hbs or lrpC in response to DNA damage. We propose that chromosome folding, modulated by NAPs, plays a critical role in NCT and DNA repair. In this context, EbfC, by regulating nucleoid dynamics as a NAP, opposes the functions of Hbsu, Rok, and LrpC in NCT, while their interconnected roles contribute to DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328ab39c4a3f04975076ada74d7385c75dacff21" target='_blank'>
              Nucleoid structure and dynamics influence natural chromosomal transformation in Bacillus subtilis: the role of EbfC
              </a>
            </td>
          <td>
            Rubén Torres, María López-Sanz, Yuri Ushijima, Kazuya Morikawa, Juan C Alonso
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405fa3aef04b79361776fea48c1010530e4d930c" target='_blank'>
              Targeted Transcriptional Repression by Induced Proximity
              </a>
            </td>
          <td>
            Christian E. Stieger, Xinru Chen, Dustin Dovala, Fabian Wu, Nicolas Pizzato, J. McKenna, Cory Johannessen, Barna D. Fodor, Markus Schirle, Daniel K. Nomura
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The maintenance of a stable genome requires constant repair. Congenital DNA repair defects lead to cancer susceptibility and progeroid (premature aging-like) syndromes. Even with intact repair, DNA lesions accumulate in aging organisms, leading to replication and transcription stress and age-dependent somatic mutations. These, in turn, can compromise cellular function and elevate cancer risk. DNA damage response (DDR) mechanisms can lead to cellular death and senescence, and targeting the DDR has emerged as therapeutic strategy not only in cancer but also to protect from age-associated phenotypes. Inhibiting DNA repair can promote cancer cell death. Eliminating senescent cells may alleviate proinflammatory consequences on their tissue environment. Moreover, strategies to limit DNA damage and augment repair in normal cells are in active development. Here, we review emerging concepts for targeting genome maintenance mechanisms to lower cancer risk and lengthen healthy lifespan by extending the integrity and functionality of somatic genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d199d017f9a52318b0cbb47dc234cf34a0dca5e" target='_blank'>
              Targeting Genome Stability to Mitigate Human Aging and Disease.
              </a>
            </td>
          <td>
            Debra Toiber, Björn Schumacher
          </td>
          <td>2025-10-14</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Chromosomal instability is a well-recognized hallmark of tumorigenesis. However, it is not known how markers for chromosomal instability, such as total distance of DNA altered or total copy number variations (CNV), affect survival, particularly in IDH wild type glioblastoma. Additionally, whether alterations in pathways that impact cellular responses to chromosomal instability, such as the TP53 or RB1 axis, influence survival, similarly remain poorly understood.



 To better characterize potential prognostic impact of markers of chromosomal instability on patient outcomes, we assessed the Mayo Clinic cohort of IDH wild type glioblastomas (n=449) for whom chromosomal microarray data was available.



 We found that while total distance of DNA altered on microarray analysis was not associated with overall survival (OS) (p=0.4030, univariate analysis), the least altered tumors (stratified by quartiles) when assessed by numbers of CNVs demonstrated improved OS (p=0.0057, univariate analysis). We next evaluated alterations in the MDM2/4-TP53 signaling axis, as well as the CDKN2A/B-CDK4/6-RB1 axis, as both amplifications and losses impacting these pathways have been well characterized in adult diffuse gliomas. While TP53 status was not predictive of OS (p=0.1504, univariate analysis), amplification of MDM2 or MDM4 was predictive of decreased OS (p=0.0071, univariate analysis). We also found that tumors without alterations in the CDKN2A/B-CDK4/6-RB1 axis had improved OS when compared to altered tumors (p<0.0001, univariate analysis). Multivariate Cox regression analysis, adjusting for age and sex, confirmed prognostic impact of MDM2/4 amplification status (HR 1.4090, p=0.0370) and CNV status (HR 0.6798, p=0.00312) on OS.



 CNV status and MDM2/4 amplification represent previously unappreciated molecular markers for prognostication in IDH wild type glioblastoma. Further work is underway to confirm our findings through multi-institutional collaborations and to account for additional potential prognostic factors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7bd1ec122e6536bdfe28952f14c544a17f01a98" target='_blank'>
              EPCO-33. MOLECULAR MARKERS OF CHROMOSOMAL INSTABILITY, INCLUDING COPY NUMBER VARIATION AND MDM2/4 AMPLIFICATION, ARE PROGNOSTIC IN IDH WILD TYPE GLIOBLASTOMA
              </a>
            </td>
          <td>
            Sanghee Lim, Ahauve Orusa, J. Eckel-Passow, R. Jenkins, M. Ruff
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retrovirus K (HERV-K) proviruses of the HML-2 subgroup are the most recently integrated retroviral elements within the human genome. The HERV-K Rec protein, a functional homolog of HIV Rev, is essential for the expression of viral mRNAs with retained introns. However, their diversity and functional capacities have remained largely unexplored. We identified Rec coding sequences in the human genome and selected intact Rec proteins from 28 proviral loci for functional characterization. Using a dual-color fluorescent reporter containing the HERV-K Rec-response element and a complementary enzyme-linked immunosorbent assay, we found that Rec proteins from nine proviral genomic loci promoted function at the post-transcriptional level. All but one of these proviruses are insertionally polymorphic in the human genome. In addition, several of the non-functional Rec proteins were trans-dominant negative. Detailed mutational analysis of the most potent inhibitory variant, encoded by the HERV-K provirus 12q14.1, displayed only two amino acid changes (N2H and E34del) relative to the prototypical functional Rec protein encoded by the reconstructed consensus HERV-K. Insertion of a glutamic acid at position 34 in this mutant fully restored the functional activity. Our results unveil an unexpected complexity in HERV-K (HML-2) post-transcriptional regulation, with a few Rec variants showing function, while others are trans-dominant negative. These findings significantly expand our understanding of the various Rec proteins that can be expressed in human cells and how they function at the post-transcriptional level. IMPORTANCE In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects. In this study, we compared Rec sequences from 58 type 2 human endogenous retrovirus K proviruses and showed that only 9 encode Rec proteins that are functional in post-transcriptional regulation of RNAs with retained introns. We also showed that several Rec proteins have trans-dominant negative activity when co-expressed with a functional Rec protein. While previous studies have demonstrated the expression of Rec mRNAs and proteins in human cells, this is the first study to define which loci have the potential to encode either functional or trans-dominant negative Rec proteins. Additionally, our reporter system will also enable future investigations to easily determine whether functional and/or trans-dominant negative Rec proteins are expressed in any cell. These findings are also important for future studies that aim to link Rec post-transcriptional function to physiological or pathological effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42022bba88ac63086a6e5b60150a1271cd5595a" target='_blank'>
              Characterization of HERV-K (HML-2) Rec proteins encoded in the human genome and their post-transcriptional function
              </a>
            </td>
          <td>
            Katarzyna Zurowska, Godfrey Dzhivhuho, David Grabski, D. Rekosh, M. Hammarskjold
          </td>
          <td>2025-11-18</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Reversine is a small-molecule Aurora kinase inhibitor known for its pro-apoptotic effects and potential to remodel chromatin architecture. Although its impact on mitotic regulation is established, its effects on telomere dynamics and nuclear organization in chronic myeloid leukemia (CML) remain unclear. This study aimed to investigate the effects of reversine on telomere architecture, genomic instability, and apoptosis in CML cell lines (K-562 and MEG-01). Reversine was applied at increasing concentrations, and cytotoxicity was assessed using caspase-3/7 activation assays. Quantitative PCR was used to measure AURKA and AURKB mRNA expressions. Three-dimensional telomere architecture was analyzed with TeloView® v1.03 software after Q-FISH labeling to quantify telomere number, signal intensity, aggregation, nuclear volume, and a/c ratio. Reversine induced a dose- and time-dependent apoptotic response in both cell lines and significantly downregulated AURKA and AURKB expressions. Three-dimensional telomere analysis revealed a marked reduction in telomere number and aggregates, signal intensity, and nuclear volume. While reduced signal intensity may indicate telomere shortening, the concurrent decrease in aggregation and altered spatial parameters suggests telomeric reorganization rather than progressive instability. These features reflect structural nuclear remodeling and early apoptotic commitment. Differences between K-562 and MEG-01 responses underscore potential heterogeneity in telomere maintenance mechanisms. Reversine modulates genomic stability in CML cells through dual mechanisms involving Aurora kinase inhibition and telomere architecture remodeling. The integration of 3D telomere profiling highlights reversine’s potential as a therapeutic agent targeting nuclear disorganization and mitotic dysregulation in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e06bedd101bf31c7285fbbc7d8f35b57bf8952e" target='_blank'>
              Reversine-Induced Telomere Architecture Remodeling in Chronic Myeloid Leukemia Cell Lines: Insights from TeloView® Analysis of 3D Nuclear Architecture
              </a>
            </td>
          <td>
            Fábio Morato de Oliveira, Isabela Dias Cruvinel, Bruno Machado Rezende Ferreira, Sabine Mai
          </td>
          <td>2025-10-31</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic dysfunction‐associated steatohepatitis (MASH) is anticipated to become the leading cause of hepatocellular carcinoma (HCC). Accumulating evidence indicates that N6‐methyladenosine (m6A)‐modified circular RNAs (circRNAs) play key roles in tumor malignant progression. However, the precise molecular mechanisms by which circRNAs and their m6A modification regulatory networks respond to metabolic reprogramming, such as lipid overload stress, to drive malignant tumor progression in the context of MASH‐related HCC remain unclear. This study aimed to investigate the role and regulatory network of m6A‐modified circRNAs in MASH‐related HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71765296ca87fa312e7fb8f2bfee13dadb3e17c0" target='_blank'>
              Intranuclear paraspeckle‐circular RNA TACC3 assembly forms RNA‐DNA hybrids to facilitate MASH‐related hepatocellular carcinoma growth in an m6A‐dependent manner
              </a>
            </td>
          <td>
            Jingbo Fu, Yanping Wei, Yun Yang, Xinwei Yang, Ouyang Tao, Xianming Wang, Shuzhen Chen, Zenglin Liu, Yu Su, Jingbo Fu, Miao Yu, Haihua Qian, Hao Song, Shuoyuan Xu, Ru Zhao, Xue Jiang, Yunfei Huo, Man Zhang, Pinhua Yang, Zhao Yang, Kui Wang, Liang Li, Hongyang Wang
          </td>
          <td>2025-10-16</td>
          <td>Cancer Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 MTAP encodes a key enzyme in methylthioadenosine (MTA) catabolism. MTAP loss, frequent across human cancers, leads to MTA accumulation and inhibition of PRMT5, a critical regulator of cell cycle and DNA repair. This metabolic vulnerability renders MTAP-deficient tumors selectively sensitive to PRMT5 inhibitors such as TNG-462 and TNG-908. Emerging data indicate that MTAP can also be epigenetically silenced via promoter hypermethylation, independent of genomic deletion, thereby evading detection by DNA-based diagnostics and excluding patients from PRMT5-targeted therapies. Here, we integrate MTAP methylation and expression profiles and evaluate PRMT5 inhibitor sensitivity across diverse tumor types. A comprehensive cohort comprising 6,187 samples with matched DNA methylation and gene expression data was assembled from 17 distinct TCGA projects. Data retrieval, raw RNA-seq read counts, and methylation data analysis were performed using the TCGAbiolinks R package, DESeq2 R package and HM450K arrays, respectively. In addition, we incorporated DNA methylation profiles from established human cell lines obtained from the ENCODE project. In vitro, MTAP-wild type cell lines (human: HCC3153 and HS578T and mouse: GL261); MTAP-null cell lines (human: MDA MB-231, RT4, RT112, UM-UC-3) and MTAP-hypermethylated cell lines (human: U87 and HepG2) were used for viability MTT assays after TNG-462 or TNG-908 treatment. Array assays revealed cg25162921 and cg00230302 probes targeting CpG sites associated with the MTAPlocus. The promoter-proximal probe cg25162921 exhibited a strong inverse correlation with MTAP expression in multiple tumor types, including lung squamous cell carcinoma (TCGA-LUSC; r=-0.48), bladder urothelial carcinoma (TCGA-BLCA; r=-0.42), and glioblastoma multiforme (TCGA-GBM; r=-0.45). We next investigated whether the mechanism of MTAP inactivation, genomic deletion and promoter hypermethylation, influenced cellular sensitivity to PRMT5 inhibition by viability MTT assy. MTAP genomic status and methylation levels were stratified at the promoter-proximal CpG site cg25162921 using DNA methylation from ENCODE project. In viability assays, MTAP-null cell lines or MTAP-hypermethylated models were more sensitive to TNG-462 and TNG-908 with a reduced IC50 compared to MTAP-wild type cell lines. We identify promoter hypermethylation as a distinct mechanism of MTAP silencing, broadening the spectrum of MTAP-low tumors beyond genomic loss. These findings support integration of epigenetic biomarkers to refine patient selection and enhance response rates in PRMT5-targeted clinical trials.



 Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, Esther Cabañas Morafraile, Bernard Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, Juan A Ardura, Balazs Gyorffy, Victor Moreno, Alberto Ocaña. Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de65a469e9e37ed4b177eca66de3a9ae88f86296" target='_blank'>
              Abstract C026: Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers
              </a>
            </td>
          <td>
            Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, E. C. Morafraile, B. Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, J. Ardura, Balázs Győrffy, Víctor Moreno, Alberto Ocaña
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Ubiquitin-specific protease 1 (USP1) is a critical regulator of DNA damage response (DDR) pathways, exerting its function via the deubiquitination of key DNA repair substrates, including FANCD2 and proliferating cell nuclear antigen (PCNA). These deubiquitination events are essential for the functional integrity of the Fanconi anemia (FA) pathway and translesion DNA synthesis (TLS), both of which are vital for resolving DNA interstrand crosslinks and replication-associated DNA lesions. DNA-damaging agents, a broad class of therapeutics exert their cytotoxic effects by inducing various forms of DNA lesions, including base modifications, single-strand breaks (SSBs), double-strand breaks (DSBs), and DNA crosslinks. These lesions disrupt DNA replication and transcription, triggering replication stress, genomic instability, and ultimately cell death if left unrepaired. However, cancer cells often activate DDR pathways that coordinate DNA repair, cell cycle checkpoints, and survival responses, which can confer resistance to DNA-damaging treatments by enhancing lesion resolution and maintaining genomic integrity. Our previous studies showed that USP1 inhibition impaired the resolution of DNA lesions, resulting in persistent activation of DNA damage markers such as γH2AX. This indicates that USP1 inhibition could deepens the genotoxic stress imposed by DNA-damaging agents and promotes apoptotic cell death by blocking effective DNA repair In this study, we investigated whether pharmacological inhibition of USP1 could enhance the cellular response to DNA-damaging agents by disrupting key DNA repair mechanisms. Human cancer cell lines were treated with our proprietary USP1 inhibitor, VRTX531, either alone or in combination with DNA-damaging agents. VRTX531 treatment alone led to the accumulation of monoubiquitinated FANCD2 and PCNA, confirming effective USP1 inhibition and disruption of FA and TLS pathway activity. Combined treatment with VRTX531 and DNA-damaging agents resulted in a synergistic increase in cytotoxicity, as reflected by significantly reduced cell viability relative to DNA damage alone. These findings underscore the broader therapeutic potential of USP1 inhibition in not only sensitizing cancer cells to DNA-damaging agents but also overcoming chemo-resistance driven by DDR pathway activation.



 Prashant K. Bhavar, Uday K. Surampudi, Partha Pratim. Sarma, Appaji Mandhare, Neetu Singh, Nandini Narayani, Anuj Kshisagar. USP1 inhibition disrupts DNA repair and potentiates the cytotoxic effects of DNA-damaging agents via persistent DNA damage in cancer cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06d19eca0105605db82afdf528e4bb45ce22555e" target='_blank'>
              Abstract A034: USP1 inhibition disrupts DNA repair and potentiates the cytotoxic effects of DNA-damaging agents via persistent DNA damage in cancer cells
              </a>
            </td>
          <td>
            P. Bhavar, U. Surampudi, P. Sarma, A. Mandhare, Neetu Singh, Nandini Narayani, Anuj Kshisagar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Epstein-Barr virus (EBV) lytic infection contributes to virus-associated cancers and autoimmunity and depends on viral subversion of host DNA damage responses (DDR). Here, we use high-content screening (HCS) for morphologic and pseudotemporal profiling of single-cell EBV reactivation and DDR dynamics spanning seven lytic induction treatments in nine B cell lymphoma models. We generated an atlas (>750,000 cells) of spatiotemporally distinct phenotypes of immediate-early (IE) and late lytic proteins, viral and cellular DNA replication, and double-stranded break (DSB) DDR factors. Single-cell segmentation, feature extraction, and clustering identify treatment- and model-dependent cell responses and lytic induction. Notably, genotoxin-induced DDR profiles differ in lytic versus latent cells, and lytic protein localization varies across pharmacologic and physiologic stimuli. Pseudotime trajectories of physiologic reactivation reveal viral replication compartment (VRC) nucleation and expansion alongside concomitant host DDR localization. The early DDR marker γH2AX is depleted from VRCs but widespread across host chromatin throughout reactivation. Surprisingly, the lytic-essential late DDR protein 53BP1 is present in lytic cells prior to viral genome replication but subsequently absent from VRCs and host chromatin, indicating spatial and kinetic DDR dysregulation during EBV reactivation. Collectively, these data support a model wherein EBV transiently employs host DSB DDR mediators to initiate genome replication while a host-targeted DDR is initiated but impaired. We further demonstrate biological generalizability and utility of the method across confocal and epifluorescence systems. Thus, this work demonstrates a powerful technique to recover host-virus dynamics from static timepoints and will support future high-throughput single-cell virology applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8351d90d1e76d2976355d93ad064503b2904e7b0" target='_blank'>
              Reconstructing EBV reactivation and DNA damage response kinetics in morphologic pseudotime
              </a>
            </td>
          <td>
            Dina G. Tekle, Jonathan Z. Sexton, E. SoRelle
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6bb7c74dc3226bfd915b13024e794addeb2154" target='_blank'>
              Diversity and distribution of the subtelomeric Y’ elements across Saccharomyces cerevisiae strains
              </a>
            </td>
          <td>
            Liébaut Dudragne, Juliana Silva Bernardes, Zhou Xu
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ab74cb4a8563ca5e2fb3e18483b3566ef1290b3" target='_blank'>
              The TDG protein environment connects active DNA demethylation with chromatin and RNA biology.
              </a>
            </td>
          <td>
            Federica Richina, Faiza Noreen, Christina Bauer, A. Weber, Christophe Kunz, Katarzyna Buczak, S. Schwarz, Fabian Wu, David Schürmann, Primo Schär
          </td>
          <td>2025-11-25</td>
          <td>Cellular and molecular life sciences : CMLS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea5a3932257b70ecdafde06e4be4b8b58b0afee" target='_blank'>
              SWING domains prime chromatin for nuclear body–mediated gene regulation
              </a>
            </td>
          <td>
            Ruofan Yu, Hanna Sas-Nowosielska, A. S. Dessai, Amal Oubarri, Hannah H. Kim, Christine Faunce, Benjamin Vanderkruk, Katherine A Alexander, Son C Nguyen, Ian D. Krantz, Eric F Joyce, Melike Lakadamyali, Shelley L. Berger
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The E. coli genome is encoded on a contiguous ~4.6 Mb-long DNA molecule, compacted inside a micron-cubed cell. When reconstituted in vitro, chromosomes expand in a bulk that is challenging for probing single-chromosome DNA transactions and conformational changes. Here, we report transplanting E. coli chromosomes into 2D semi-open microfluidic compartments, enabling exchange of conditions, stretching by electric field, mapping DNA-bound proteins, and cell-free transcription-translation at steady-state. We find transplanted chromosomes emerge as intact, compacted, blob-like structures, decorated with native proteins from the donor cell. The blobs include clusters of condensin proteins and exclude sparse bright ribosome foci, whereas RNA polymerases uniformly decorate the chromosome. Introducing a transcription-translation system, we measure genome-average transcription rates and image the birth of individual proteins from a reporter gene on the chromosome. Our data suggest a dilute regime without translational amplification or multiple synthesis events per gene. The removal of native proteins reveals a conformation transition from expanded to compacted state upon increased molecular crowding. Interestingly, transcription has a swelling effect, pushing the compaction transition to higher crowding levels. Our work opens a window into genome-scale DNA transactions outside a cell and helps tackle the bottom-up assembly of autonomous artificial cells. Chromosomes are long DNA polymers, compacted in cells and fragile once released from the host. Here, authors report on a method to gently transplant bacterial chromosomes into 2D microfluidic compartments, enabling analysis of their conformations and function within a native transcription-translation system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e9cf05890236fa52fe259a47a2db9febf64295" target='_blank'>
              Bacterial chromosome conformation and cell-free gene expression in synthetic 2D compartments
              </a>
            </td>
          <td>
            Ferdinand Greiss, S. Daube, V. Noireaux, R. Bar-Ziv
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lamin A/C (LMNA), a key component of the nuclear envelope, is essential for maintaining nuclear integrity and genome organization [W. Xie et al., Curr. Biol. 26, 2651-2658 (2016)]. While LMNA dysregulation has been implicated in genomic instability across cancer and aging, the underlying mechanisms remain poorly understood [S. Graziano et al., Nucleus 9, 258-275 (2018)]. Here, we define a mechanistic role for LMNA in preserving genome stability in small-cell lung cancer (SCLC), a malignancy marked by extreme genomic instability [N. Takahashi et al., Cancer Res. Commun. 2, 503-517 (2022)]. LMNA depletion promotes R-loop accumulation, transcription-replication conflicts, replication stress, DNA breaks, and micronuclei formation. Mechanistically, LMNA deficiency disrupts nuclear pore complex organization, specifically reducing phenylalanine-glycine (FG)-nucleoporin incorporation, resulting in impaired RNA export and nuclear retention of RNA. LMNA expression is repressed by EZH2 and reexpressed during SCLC differentiation from neuroendocrine (NE) to non-NE states, and low LMNA levels correlate with poor clinical outcomes. These findings establish LMNA as a key regulator of nuclear transport and genome integrity, linking nuclear architecture to SCLC progression and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1b936a6a58b416fb0e62e291846a3a4d6fb1c3" target='_blank'>
              Lamin A/C loss promotes R-loop-mediated genomic instability and poor survival in small-cell lung cancer.
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Ravi P Shuklah, Robin Sebastian, Umeshkumar M Vekariya, Shahbaz Ahmed, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, V. Aksenova, Elizabeth Giordano, N. Takahashi, Michael Nirula, Mohit Arora, Chiori Tabe, M. Thomas, Rajesh Kumar, Yashuhiro Arakawa, Ukhyun Jo, Tomasz Skorski, Beverly A. Teicher, Roshan Shreshta, M. Aladjem, Stephen Lockett, M. Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-10-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Tardigrades exhibit extraordinary ionizing radiation (IR) resistance, with DNA repair as a core mechanism, but interactions between classical repair factors and tardigrade-specific proteins remain unclear. The non-homologous end joining (NHEJ) core factor XRCC5 (Ku80) is strongly upregulated post-IR and critical for survival. TDR1, a tardigrade-specific intrinsically disordered protein, is IR-induced, co-localizes with DNA damage foci, binds DNA in vitro, and is synchronously overexpressed with XRCC5 under IR, suggesting potential synergy. Using Hypsibius exemplars, this study conducted two experiments: 1) Radiation gradient treatment CRISPR/Cas9-mediated TDR1/XRCC5 knockout to measure -H2AX (DNA double-strand break marker) and counterpart gene mRNA/protein at 4 h post-2000 Gy IR. Predicted results: Low-dose IR triggers XRCC5 (4 h) but not TDR1; medium-high-dose IR upregulates both (4 h). TDR1 knockout may reduce XRCC5 protein and increase -H2AX; XRCC5 knockout may not affect TDR1 but cause persistent -H2AX.This study clarifies TDR1-XRCC5 synergy in IR-induced DNA repair, illuminating tardigrades radiation resistance mechanisms and informing radio-sensitive system optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd7968ce6d15f78681776f4a0b93359ef37c2fd8" target='_blank'>
              The Functional Interplay Between Tardigrade-Specific IDP TDR1 and XRCC5 in Radiation Resistance
              </a>
            </td>
          <td>
            Haoran Hao
          </td>
          <td>2025-11-27</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Transposable elements (TEs) make up half of the human and mouse genome but remain understudied. Evidence is growing that TEs play important roles in immunity as a subset of TE enhancers regulate innate immunity genes. However, the cis-regulatory contribution of TEs across immune cell development is not well understood. Mining more than 300 chromatin accessibility datasets (ATAC-seq) across mouse hematopoietic cell lineages, we identified two subfamilies of rodent-specific retroviral LTRs, ORR1E and ORR1D2 (ODE), which are enriched in accessible chromatin across all immune cell types. Unsupervised clustering show that ∼2,900 ODEs stratify by cell type-specific accessibility. Each ODE cluster is enriched for a unique combination of sequence motifs for cell lineage-specifying TFs, such as PU.1, IRF8, and RORγ, which we validated with ChIP-seq. By correlating ODE accessibility with gene expression (RNA-seq), we predicted hundreds of ODE-gene interactions. We generated Micro-C (DNA-DNA contact) in CD8+ T cells and validated ∼120 ODE-gene interactions. Predicted target genes are more expressed in mouse than in human CD8+ T cells. Lastly, accessible ODEs have higher sequence conservation across rodents than other ODEs. Together, these data suggest that despite their common origin, ODE LTRs have subfunctionalized into cell type-specific enhancers across mouse hematopoiesis. We propose that ODE enhancers were co-opted during rodent evolution to reinforce cell type identity.



 F31AI183775



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c25d5af8adc02b84a26acb002fe252a06d958df" target='_blank'>
              Functionalization of transposable element-derived enhancers across mouse hematopoiesis 3835
              </a>
            </td>
          <td>
            Jason D. Chobirko, Elizabeth A. Fogarty, Cedric Feschotte, Andrew Grimson
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Esophageal squamous cell cancer (ESCC) is one of the most common and lethal malignancies. It is urgent to uncover the underlying mechanisms for ESCC progression. Aberrant DNA methylation is observed in ESCC. However, the precise relationship between DNA methylation alterations and clinical features of ESCC is still largely unknown. We aimed to further explore the potential role of DNA methylation in the development of ESCC. Abnormal DNA methylation and gene expression were analyzed with TCGA ESCA cohort data. Differentially methylated CpGs were identified with Mann-Whitney U test. Differentially expressed genes (DEGs) were analyzed with limma package and student’s t-test. Prognosis-related CpGs were screened by Cox regression analysis. ESCC subgroups were identified with ConsensusClusterPlus package. Epithelial-mesenchymal transition (EMT) score was calculated with epithelial and mesenchymal gene expression data. CIBERSORTx was employed to analyze immune cell infiltration. Drug sensitivity was predicted with oncoPredict package. Gene Expression Omnibus (GEO) dataset GSE53624 was used to verify CFAP52 expression in ESCC, and GSE160269 and GSE203115 were utilized to investigate the expression of CFAP52 in different cell types. Seurat package was used for single-cell RNA-seq data analysis. Western blot was used to assess CFAP52 expression. MTS, transwell, scratch wound healing, colony formation and cell cycle assays were conducted to verify the function of CFAP52. The genes affected by CFAP52 overexpression were explored via RNA-seq. Arachidonic acid level was quantified with UPLC-MS/MS system. Immunohistochemistry was used to check the protein level of CFAP52. 3443 differentially methylated CpGs and 43 related DEGs were identified between males and females. Besides, 1219 alcohol-related CpGs were screened, and 9 related genes were differentially expressed. 175 ESCC prognosis-related CpGs were screened, and ESCC subgroups were identified with these CpGs. Poor prognosis group had higher EMT score, and showed elevated proportion of regulatory T cells and M2 macrophages and higher estimated IC50s of cisplatin_1496, gemcitabine_135 and docetaxel_1007. CFAP52 was identified as a stage-related gene, and mainly expressed in epithelial cells, especially in malignant epithelial cells. Furthermore, CFAP52 was upregulated in ESCC cells and promoted cell proliferation and migration. CFAP52 might affect arachidonic acid metabolism in ESCC cells. CFAP52 protein level increased in ESCC tissues, and high CFAP52 was associated with poor survival of ESCC patients. Treatment with DNA methylation inhibitor downregulated CFAP52. Collectively, our investigations show that DNA methylation is associated with sex and alcohol consumption in ESCC. Besides, DNA methylation is linked to ESCC prognosis. Moreover, we uncover the potential oncogenic role of CFAP52 in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199b70207cbb95c05493015ce40595ab05a81d9e" target='_blank'>
              Analysis of the associations between DNA methylation and clinical features reveals the potential oncogenic effect of CFAP52 in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Guangchao Wang, Mengzhu Lv, Yan Wang, Qimin Zhan
          </td>
          <td>2025-11-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Protein kinase CK2 is an important regulator of cell, embryo, and organism function whose transcript levels are often dysregulated in disease. Previous studies have primarily focused on the regulation of CK2 gene expression via the proximal promoter. Here, we analyzed the core promoter of the CK2 genes and pseudogene to assess the structure and potential regulatory elements. Our analysis showed that CSNK2A1 contained 14 exons, rather than 13 exons as previously reported. Using FANTOM5 and DBTTS data, we found that transcription start sites were broadly distributed across a 100-nucleotide region in the CK2 gene core promoters, consistent with “broad” class promoter architecture. Using these databases, we found a dissimilar transcription start site usage between adult and cancer tissues compared to fetal tissues for each of the CK2 gene promoters. A further analysis of the CK2 gene core promoter subregions showed instances of core promoter subregion switching. All CK2 gene core promoters contained canonical and non-canonical initiator motifs, suggesting their potential as dual-initiator core promoters, while CSNK2A3 only had canonical initiator motifs. Additionally, all CK2 gene core promoters contain DCE motifs and pause buttons. In contrast, Wnt/β-catenin target genes c-MYC and CCND1 had DPEs, which can be regulated by protein kinase CK2. Collectively, our data provides new insights into the transcriptional regulation of CK2 genes and opens new avenues for research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e833dc4b836cd2e9bdb5ff4433218f7e2c6e4e0" target='_blank'>
              Bioinformatic Investigation of Regulatory Elements in the Core Promoters of CK2 Genes and Pseudogene
              </a>
            </td>
          <td>
            Nicholas G. Wilson, Jesse S. Basra, Isabel Dominguez
          </td>
          <td>2025-11-04</td>
          <td>Kinases and Phosphatases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA replication strips off all chromatin proteins, which must be reassembled behind the replication fork. To track chromatin reassembly on newly synthesized DNA, we developed Nascent CUT&Tag, a chromatin profiling method that uses antibody-targeted in situ tagmentation to directly measure transcription factor binding on nascent chromatin. Using Nascent CUT&Tag, we tracked the recovery of GAGA factor (GAF) in Drosophila Kc cells. We find that GAF is displaced from chromatin during DNA replication and shows a broad spectrum of recovery times, ranging from minutes to hours. Early recovering peaks are characterized by shorter GAF motifs and are associated with functions related to cell cycle progression. Conversely, late recovering peaks are characterized by longer, degenerate GAF motifs and are associated with developmental functions. We also show that GAF recovery on newly synthesized DNA requires chromatin remodeling by Brahma Associated Factor (BAF), implying that nucleosome eviction is critical to fully reestablish GAF binding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0e968f2d6d90515e2129f89a9da1138e1f3aeeb" target='_blank'>
              Nascent CUT&Tag captures transcription factor binding after chromatin duplication
              </a>
            </td>
          <td>
            Matthew Wooten, Kevin Nguyen, Brittany N Takushi, K. Ahmad, Steven Henikoff
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff090d0a0cb56557fe9a6d37cfd5b8f1167336b" target='_blank'>
              ATRX loss in sarcomas is associated with dysregulated gene and transposable element expression, loss of DNA methylation, and worse survival
              </a>
            </td>
          <td>
            K. Demanelis, J. Leix, M. Burgess, B. Nacev
          </td>
          <td>2025-11-17</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transcriptional regulation is a crucial biological process that enables accurate gene expression, allowing cells to maintain their identity and respond to environmental stimuli. CCCTC-binding factor is a fundamental protein that actively involves itself in transcriptional regulation, serving as a highly conserved architectural regulator. CTCF is traditionally acknowledged for its function in chromatin organization and insulation. It coordinates active or repressed transcription by establishing topologically associated domains and also helps in preserving enhancer-promoter identity. In addition to its DNA-binding roles, CTCF significantly participates in RNA biology. It engages with nascent RNA, pre-mRNA, and long non-coding RNAs via its RNA-binding domain, thereby affecting different transcriptional dynamics. Simultaneously, the swiftly advancing domain of epitranscriptomics has revealed other RNA modifications, such as N6-methyladenosine (m6A), 5-methylcytosine (m5C), and pseudouridine (Ψ), which jointly regulate RNA stability, transport, translation, and destruction. Hence, the dysregulation of these changes or CTCFactivity is closely linked to oncogenesis, developmental problems, and resistance to therapy. The intersection of CTCF-mediated genomic architecture with epitranscriptomic regulation highlights its function as a complex integrator of chromatin and RNA networks. This review consolidates contemporary understanding of CTCF's twin activities in DNA and RNA binding, examining how their interaction influences transcriptional regulation, RNA processing, and disease relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9880a67af650c419bef7c172c588e4f299705f1f" target='_blank'>
              Beyond DNA binding: CTCF at the crossroads of chromatin architecture and epitranscriptomic regulation.
              </a>
            </td>
          <td>
            Oishi Mukherjee, Koustav Sarkar
          </td>
          <td>2025-11-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Chromosomal translocations involving the mixed lineage leukemia (also known as MLL or KMT2A ) gene occur in approximately 40% of children with acute myeloid leukemia (AML) under the age of 3. The fusion partners of MLL can vary widely with over 130 different ones identified to date. These fusions result in constitutive activation of MLL and ultimately drive the overexpression of key downstream target genes including MEIS1 and HOXA cluster genes that leads to leukemia development. Critically, MLL ‐ rearranged ( MLL ‐ r) leukemias all depend on menin for growth and survival that in turn has driven the development of menin inhibitors (e">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3856a5480d6c33f52e5c3762c44d1fd0ad0e257b" target='_blank'>
              A new KAT on the block rewrites the epigenetic script in menin inhibitor‐resistant leukemia
              </a>
            </td>
          <td>
            Yizhou Huang, Charles E. de Bock
          </td>
          <td>2025-11-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Mycobacterium tuberculosis (Mtb), the causative agent of tuberculosis (TB), is estimated to infect nearly one-quarter of the global population. A key factor in its resilience and persistence is its robust DNA repair capacity. Non-homologous end joining (NHEJ) is the primary pathway for repairing DNA double-strand breaks (DSBs) in many organisms, including Mtb, where it is mediated by the Ku protein and the multifunctional LigD enzyme. In this study, we demonstrate that Ku is essential for mycobacterial survival under DNA-damaging conditions. Using cryogenic electron microscopy (cryo-EM), we solved high-resolution structures of both the apo and DNA-bound forms of the Ku-Mtb homodimer. Our structural and biophysical analyses reveal that Ku forms an extended proteo-filament upon binding DNA. We identify critical residues involved in filament formation and DNA synapsis and show that their mutation severely impairs bacterial viability. Furthermore, we propose a model in which the C-terminus of Ku regulates DNA binding and loading and facilitates subsequent recruitment of LigD. These findings provide unique insights into bacterial DNA repair and guide future therapeutics. Mycobacterium tuberculosis protein Ku is involved in DNA repair and a potential drug target. Here, using cryo-EM and complementary approaches, the authors obtain insights into Ku oligomerization and mechanisms of function in DNA synapsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a761f6b26651521fdd3367949d0523c2f80c9db2" target='_blank'>
              Oligomerisation of Ku from Mycobacterium tuberculosis promotes DNA synapsis
              </a>
            </td>
          <td>
            Sayma Zahid, S. Baconnais, Henrietta Smith, Saseela Atwal, Lucy Bates, Harriet Read, Ankita Chadda, Florian Morati, Tom Bedwell, Emil G. P. Stender, Joanne Walter, Steven W. Hardwick, Fredrik Westerlund, E. Galburt, É. Le Cam, Alice L. B. Pyne, G. Mukamolova, Amanda K. Chaplin
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7806041ee18d34c6b180e4244d83e2a940ee9f4" target='_blank'>
              Finding the perfect promoter: Correlating single-cell transcriptome data with gene drive performance
              </a>
            </td>
          <td>
            Yingke Wu, Yunchen Xia, Ziyin Yao, Weizhe Chen, Xihua Jia, Nan Liang, Jackson Champer
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant childhood cerebellar tumor comprising molecularly and clinically distinct subgroups. Groups 3 and 4-MB originate from overlapping progenitor pools of the developing rhombic lip when neuronal differentiation hierarchies are subverted through somatic alterations. Mutations suspected to deregulate the activity of chromatin-modifying genes are pervasive in Group 3/4-MB. However, molecular consequences of these alterations remain largely undefined. We characterized the chromatin landscape of fifty-two primary MBs using chromatin immunoprecipitation followed by sequencing (ChIP-seq) for six histone modifications and performed multi-modal integration with sample-matched somatic lesions, genome-wide DNA methylation, and transcriptomic profiles. Analysis of differential chromatin states across MB subgroups revealed significant enrichment of the bivalent enhancer state (EnhBiv; marked by coincident H3K4me1 and H3K27me3) in Group 3/4-MB overlapping neurodevelopmental genes. Integrative bioinformatics coupled with CUT&RUN analysis revealed significant enrichment of KDM2B, a histone lysine demethylase overexpressed in Group 4-MB, in gene promoters marked by the EnhBiv state. CRISPR gene targeting or targeted protein degradation of KDM2B selectively suppressed growth of Group 3 and Group 4-MB models in vitro and in vivo, which was dependent on KDM2B DNA-binding domain but not demethylase activity. Acute and chronic KDM2B degradation impaired EZH2 recruitment through direct physical interactions and indirect KDM2B-mediated PRC1 activity, reducing H3K27me3 deposition and chromatin bivalency, while profoundly derepressing PRC2 targets involved in neuronal differentiation. Our comprehensive characterization of the MB chromatin landscape in a large cohort of primary tumors provides unprecedented insights into the epigenetic basis of MB subgroups, implicating novel mechanisms of tumorigenesis, and disclosing a unique subgroup-specific dependency conferred by KDM2B-mediated Polycomb activity that warrants consideration as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff433ac7a60b68bf5a2b5c1d8d57a287494ba1b0" target='_blank'>
              EPCO-22. DIVERGENT MEDULLOBLASTOMA CHROMATIN STATES DISCLOSE KDM2B AS A SELECTIVE DEPENDENCY
              </a>
            </td>
          <td>
            Ran Tao, Serap Erkek-Ozhan, Beisi Xu, Yurika Matsui, Priya Mittal, Kyle S Smith, Yiran Li, Qingsong Gao, Emily Darrow, Ruijie Xu, Nadhir Djekidel, Richa Bajpai, Rahul Kumar, Colleen Reilly, T. Soliman, Jennifer L. Hadley, Melissa Batts, Natarajan Bhanu, B. Gudenas, Leena Paul, Hong Lin, K. Lowe, L. Kutscher, Benjamin Garcia, S. Pruett-Miller, Xin Zhou, Brent A. Orr, G. Robinson, Gang Wu, Jan O. Korbel, Jamy C Peng, Stefan M. Pfister, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Loss of DNA mismatch repair (MMR) increases mutagenesis and tumorigenesis. mutS homolog 2 (MSH2), a central component of the MMR pathway, is essential for correcting base–base mismatches and insertion/deletion loops during DNA replication. To investigate how Msh2 deficiency cooperates with oxidative stress to drive mutagenesis and tumorigenesis, we employed an rpsL reporter gene assay using normal tissues before tumor development following treatment with an oxidizing agent. The background mutation frequency in the small intestines of Msh2-/- mice was over 20-fold higher than that of wild-type mice. In addition to G > A base substitutions, frequent 1-bp deletions in adenine mononucleotide repeats ((A)n) in the rpsL gene were observed. Potassium bromate treatment further increased the mutation frequency, particularly insertion-deletion mutations (indel), in the normal small intestinal epithelium of Msh2-/- mice before tumor development. Mutation signature analysis from next-generation sequencing data revealed that signatures associated with MMR deficiency (SBS15, SBS44, and ID2) and clock-like processes (SBS1 and SBS5) were consistently detected across all Msh2-/- tumors, similar to those observed in human MMR-deficient cancers. ID2, which involves 1-base deletions occurring in (A/T)n tracts of six bases or longer, supports the findings of the rpsL assay. Microsatellite instability (MSI) analysis showed that indel mutations at (A)n loci detected using the rpsL assay reflect genome-wide MSI. Msh2-/- tumors frequently harbored driver mutations, such as frameshift mutations in short tandem repeats within Apc and G > A substitutions in Ctnnb1, both of which activate the Wnt signaling pathway. Oxidative stress further accelerated these mutational processes. Oxidative stress promotes repeat-associated mutagenesis, which manifests as MSI and base substitutions in MMR-deficient intestinal tissues, thereby enhancing the mutator phenotype and increasing the overall mutation burden. This process can be sensitively captured using our rpsL assay, which serves as a functional indicator of MMR deficiency and replication instability in normal tissues before tumor formation. This increases the likelihood of driver mutations in oncogenes and tumor suppressor genes, ultimately accelerating early tumorigenesis. This study demonstrated that MSH2 is essential for maintaining genome stability under oxidative conditions and functions as a key suppressor of oxidative stress–induced tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c307ffc3ea1fca1fb4bfcae167ad12e85c5a8ce" target='_blank'>
              Oxidative stress accelerates repeat sequence instability and base substitutions promoting gastrointestinal driver mutations in MSH2 deficient mice
              </a>
            </td>
          <td>
            M. Ohno, Noriko Takano, Kyoko Hidaka, Fumiko Sasaki, Yasunobu Aoki, Takehiko Nohmi, Teruhisa Tsuzuki
          </td>
          <td>2025-10-23</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Enhancers are critical genetic elements controlling transcription from promoters, yet how they convey regulatory information across large genomic distances remains unclear. Here, we engineer pluripotent stem cells in which cohesin loop extrusion can be inducibly disrupted without confounding cell cycle defects. Transcriptional dysregulation is cell type-specific, and not all loci with distal enhancers depend equally on cohesin extrusion. Using comparative genome editing, we demonstrate that enhancer-promoter communication over just 20 kilobases can require cohesin. However, promoter-proximal elements can support long-range, cohesin-independent enhancer action - even across strong CTCF insulators. Finally, transcriptional dynamics and the emergence of embryonic cell types remain largely robust despite disrupted extrusion. Beyond establishing strategies to study cohesin in enhancer biology, our work provides mechanistic insight into cell type- and genomic context-specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08370d10326bee6e2fd937c0c8c1af297bd762df" target='_blank'>
              Synergy between regulatory elements can render cohesin dispensable for distal enhancer function.
              </a>
            </td>
          <td>
            Karissa L. Hansen, Annie S. Adachi, L. Braccioli, Smit Kadvani, Ryan M. Boileau, Moreno Martinovic, Bozhena Pokorny, Rini Shah, Erika C. Anderson, Kaite Zhang, Irié Carel, Kenya Bonitto, R. Blelloch, Geoffrey Fudenberg, Elzo de Wit, Elphѐge P. Nora
          </td>
          <td>2025-11-27</td>
          <td>Science</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554ff6de17aca3fc14055b3c5a6203467aaf1fe3" target='_blank'>
              A gap-free telomere-to-telomere pig reference genome provides insight in centromere evolution and breed-specific selection
              </a>
            </td>
          <td>
            Dong Li, Yulong Wang, Tiantian Yuan, Xiang Li, Yu Quan, Wansheng Liu, K. Schachtschneider, Lawrence B Schook, R. Crooijmans, M. Groenen, M. Derks, Taiyong Yu
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract The incidence of colorectal cancer in young adults (age of diagnosis <50 years) has been rapidly increasing. Although ∼20% of early-onset colorectal cancer (EOCRC) cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Nongenetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as alterations in the epigenome, microbiome, and immunome. We characterized the DNA methylation (DNAm) signature and measured DNAm age in EOCRC by using The Cancer Genome Atlas (TCGA). Furthermore, we carefully identified intratumoral microbes from TCGA and the Oncology Research Information Exchange Network datasets and then related the microbes to deconvolved immune cell abundances in EOCRC. We observed that the DNAm age in the EOCRC cohort was 12 years older when compared with the average-onset colorectal cancer (AOCRC) cohort, using three different epigenetic clocks. Differentially methylated sites associated with gene expression include cAMP-responsive element binding protein signaling in neurons, G protein–coupled receptor signaling, phagosome formation, and S100 family signaling. These differences were validated in the gene expression data from TCGA and the Oncology Research Information Exchange Network. When comparing the intratumoral microbes between EOCRC and AOCRC, no consistent differences were observed. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more and larger positive correlations in EOCRC. These data suggest that epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. Significance: We investigated whether environmentally driven factors contribute to EOCRC. We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than in AOCRC. These data suggest a dysregulation of the immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c12561621c15cb4ece79f124013162ea3c517b" target='_blank'>
              Epigenetic Modulation, Intratumoral Microbiome, and Immunity in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            N. Jin, R. Hoyd, Ayse S Yilmaz, Jiangjiang Zhu, Yunzhou Liu, Malvenderjit S Jagjit Singh, Dennis J. Grencewicz, Xiaokui Mo, Matthew F Kalady, Daniel W Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos H F Chan, M. Churchman, N. Denko, Frances Di Clemente, Rebecca D. Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, Alexandra P Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, Afaf E G Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A. Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, A. Tan, Daniel J. Spakowicz
          </td>
          <td>2025-10-24</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 SETD2 is a tumor suppressor gene responsible for trimethylation of H3K36, crucial for DNA mismatch repair and homologous recombination. Novel synthetic lethality agents, ATR inhibitors (ATRi), have demonstrated anti-tumor activity in preclinical models and early clinical trials for pan-tumors. SETD2 and H3K36 deficient cells have been demonstrated to display Micro Satellite Instability (MSI) with increased tumor mutational burden (TMB). Deleterious SETD2 alterations occur in 5-10% of brain tumor patients with glioblastoma, IDH wild type, WHO grade 4, (GBM). These patients have a poor prognosis. This study aims to evaluate targeted treatment strategies in central nervous system (CNS) tumors, by investigating the potential of ATRi for tumors harboring deleterious SETD2 alterations and immuno-therapy for tumors with increased mutational burden, with the objective of improving patient outcomes.



 This study analyzes genomic profiles from 434 patients diagnosed with central nervous system (CNS) tumors between January 1st, 2019, and February 28th, 2025, enrolled in the prospective Neurogenome and CoPPO studies. Rigshospitalet has one of the largest single-institution datasets with comprehensive data on SETD2 alterations derived from whole exome/genome sequencing including structural variants, copy number variations, and RNA expression.



 The study cohort included 434 patients with CNS tumors (166 female, 268 male) with a median age at diagnosis of 59 years (interquartile range: 49-66 years). GBM was the predominant tumor type (86%), followed by astrocytoma, IDH mutated (mut), WHO grade 2-4 (12%) and oligodendroglioma, IDH-mut, 1p/19q co-deletion, WHO-grade 2-3 (1.2%). Other CNS tumors were diagnosed in 6 (1.4%) patients. Deleterious SETD2 alterations were found in 13 (3%) patients: 11 with GBM, 1 astrocytoma IDH mut, and 1 oligodendroglioma. Twelve patients had mutations: 6 premature stop codons, 5 frameshift mutations and 1 splice site variant, and one case with biallelic SETD2 loss. Data on tumor mutational burden (TMB) was assessable for 12 SETD2-deficient patients, none exceeding 10mut/mb; mean TMB was 3.3 mut/mb vs. 6.2 mut/mb in the SETD2-wildtype (wt) group. No SETD2-deficient patients had received temozolomide (TMZ) treatment prior to sampling. By contrast, some samples in the SETD2-wt cohort were obtained at relapse, which may have inflated TMB due to TMZ treatment. COSMIC signatures were available for 10 SETD2-deficient patients with none having indications of MSI based on COSMIC signature evaluation.



 Loss of SETD2 function in primary CNS-tumors is relatively rare and was predominantly driven by mutations, with copy number analysis revealing only 1 biallelic deletion in 434 patients. SETD2 alterations was primarily seen in GBM patients. The SETD2 deficient patients did not exhibit indications of MSI or high TMB. However, the impact of patient relapse samples with TMZ induced mutations in the SETD2-wt cohort is being explored further together with pan-tumor markers for DNA damage repair.



 Tine Østergaard, Christoffer L. Halvorsen, Dorte S. Nørøxe, Vincent N. Fougner, Iben Spanggaard, Kristoffer S. Rohrberg, Martin Højgaard. Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in SETD2 without associated increase in mutational burden [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5615e3697959292b283838db14154a56c4e82cf6" target='_blank'>
              Abstract C037: Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in
 SETD2
 without associated increase in mutational burden
              </a>
            </td>
          <td>
            Tine Østergaard, Christoffer L. Halvorsen, D. Nørøxe, V. Fougner, I. Spanggaard, K. S. Rohrberg, M. Højgaard
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND Colorectal cancer (CRC) is the second leading cause of cancer-related death, largely due to limited treatment options in advanced stages. Genomic alterations in advanced CRC (aCRC) are complex and not fully characterized, with only 30% of patients benefiting from targeted therapies. AIM To investigate the molecular heterogeneity of primary aCRC in order to identify clinically relevant genomic alterations. METHODS We conducted a retrospective molecular analysis of 73 consecutive patients with histologically confirmed primary aCRC (stage pT4a-b). All molecular findings were correlated with available clinicopathological data. In addition, we performed survival analyses using publicly available datasets and tools. RESULTS Genetic abnormalities identified in primary tumors were most frequently mutations in tumor protein p53 (58% of cases), Kirsten rat sarcoma viral oncogene homolog (52%), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (25%), B-Raf kinase (11%) and fibroblast growth factor receptor 3 (8%), as well as R-spondin 3 (RSPO3) fusions (8%). Alterations in the tumor protein p53 and neuroblastoma RAS viral oncogene homolog genes were predominantly observed in tumors from the left colon, whereas B-Raf kinase mutations and RSPO3 fusions were more frequently detected in the right or transverse colon. We also show a strong association between the presence of RSPO3 rearrangements and patients with small tumors, normal carcinoembryonic antigen levels, and microsatellite stable tumors. Furthermore, aCRC patients with protein tyrosine phosphatase receptor type k::RSPO3 fusions exhibited a higher mortality rate. Elevated RSPO3 gene expression levels were also significantly correlated with poorer OS across two large, independent CRC cohorts. CONCLUSION This study identifies a relatively high incidence of RSPO3 rearrangements in aCRC and a strong association with clinical features. Furthermore, we find that RSPO3 fusions are associated with poorer OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b1b9374204622db5b697e06c7d6c2484792d29" target='_blank'>
              RSPO3 rearrangements in advanced colorectal cancer patients and their relationship with disease characteristics
              </a>
            </td>
          <td>
            Raquel Tur, M. Abad, Elena Filipovich, Maria Belen Rivas, Marta Rodriguez, Juan Carlos Montero, J. Sayagués
          </td>
          <td>2025-11-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, Mark R. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="During cellular proliferation DNA undergoes frequent replication cycles in which errors inevitably accumulate. DNA simultaneously faces continuous damage from endogenous sources [e.g., reactive oxygen species (ROS)] and environmental stressors, such as ultraviolet (UV) and ionizing radiation (IR). Such lesions compromise genomic stability and may escalate into DNA double-strand breaks (DSBs). Failure to repair DSBs can ultimately trigger cell death 1 . The DNA damage response (DDR) involves a coordinated, multi-step process with precise lesion detection followed by controlled chromatin remodeling to facilitate repair. Initially, the cell identifies and localizes damage sites. Then, the cell remodels condensed chromatin and constructs a platform to recruit various DNA damage repair factors to access DNA lesions. These factors excise or replace the damaged segment to repair DNA. Following repair completion, dedicated machinery restores chromatin to a closed conformation, thereby maintaining genomic integrity, a process collectively termed the Access-Repair-Restore (ARR) model 2 . Chromatin, which is comprised of DNA coiled around histone octamers (H2A/H2B/H3/H4) and stabilized by a linker (histone H1), compacts the genome into a nucleosome. This condensed architecture intrinsically obstructs DNA repair machinery, impeding repair factor recruitment. The earliest chromatin relaxation, which is essential for initiating repair, remains mechanistically opaque">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fcd919f57e7852ef39f1e75151221d2c8bd6d31" target='_blank'>
              Chromatin relaxation dynamics and histone PTMs in the early DNA damage response
              </a>
            </td>
          <td>
            Jinqin Qian, Zhongyi Xie, Liqun Zhou, Wei-Guo Zhu
          </td>
          <td>2025-10-09</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Jumping translocations (JTs) can lead to partial trisomies. A breakpoint within the gene known as Jumping Translocation Breakpoint (JTB) has previously been associated with JTs involving the long arm of human chromosome 1 (1q). These 1q+ amplifications are frequently observed in cancer. JTB was initially mapped to the Epidermal Differentiation Complex (EDC) at 1q21, and earlier studies primarily focused on its role in malignant or adult tissues. Using updated genomic data, we refined the mapping of JTB. We employed RNA in situ hybridization (RISH) to visualize Jtb expression with organ, tissue, and cell-level resolution. We demonstrate that human JTB and murine Jtb are located outside the EDC. In midgestational wild-type mouse embryos, Jtb is expressed in multiple tissues, including the developing heart and vertebral column, and shows partial overlap with the expression of early markers of the neural crest cell lineage. Our findings suggest that the oncogenic potential associated with human JTB translocations is likely unrelated to its previously assumed location within the EDC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed6323abd411d00d62b1282d50fbc327ee57f1b" target='_blank'>
              Jumping Translocation Breakpoint Expression in Midgestation Mouse Embryos
              </a>
            </td>
          <td>
            Carley McGrath, Taniya M Jayaweera, T. Lufkin, Costel C. Darie, A. Neagu, Petra Kraus
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The single-stranded DNA deaminase APOBEC3B (A3B) is capable of potently restricting the replication of a range of viruses including retroviruses (cDNA) and herpesviruses (genomic DNA). However, these and likely other DNA virus families have evolved host species-specific counter-defenses that are equally potent and serve to protect viral DNA from restriction. Although high-risk human papillomavirus (HPV) infection triggers A3B upregulation, potentially as part of an antiviral response, the impact of this restriction factor on papillomavirus replication and pathogenesis has yet to be assessed. To study human A3B antiviral function in the absence of a species-specific counter-defense, here we ask whether human A3B is capable of restricting Mus musculus papillomavirus (MmuPV1) in cellulo and in vivo. First, we created human A3B and catalytic mutant A3B-E255A expressing FVB/N mice. Second, MmuPV1 gene expression and replication was quantified in primary keratinocytes from these animals and, surprisingly, enzymatically active human A3B caused no measurable impairment in viral transcript or DNA accumulation. Third, A3B, catalytic mutant A3B-E255A, and nontransgenic FVB/N animals were infected with MmuPV1 and similar pathologies were found regardless of A3B functionality. Thus, despite likely never being exposed to human A3B during evolution, MmuPV1 appears to be unaffected by this potent, primate-specific antiviral factor. These results suggest that MmuPV1 and perhaps papillomaviruses more broadly possess a conserved mechanism to efficiently escape restriction by human A3B and related DNA deaminases. IMPORTANCE Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes such as A3B.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43450515e8b62fe85e717aebfac6a246b0159e06" target='_blank'>
              Mus Musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, Sang Hyun Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Telomere length maintenance is strongly linked to cellular aging, as telomeres progressively shorten with each cell division. This phenomenon is well-documented in mitotic, or dividing, cells. However, neurons are post-mitotic and do not undergo mitosis, meaning they lack the classical mechanisms through which telomere shortening occurs. Despite this, neurons retain telomeres that protect chromosomal ends. The role of telomeres in neurons has gained interest, particularly in the context of neurodegenerative diseases such as amyotrophic lateral sclerosis (ALS), where aging is a major risk factor. This has sparked interest in investigating telomere maintenance mechanisms in post-mitotic neurons. Nevertheless, most existing telomere analysis techniques were developed for and optimized using mitotic cells, posing challenges for studying telomeres in non-dividing neuronal cells. Thus, this protocol adapts an already established technique, the combined immunofluorescence and telomere fluorescent in situ hybridization (IF-FISH) on mitotic cells to study the processes occurring at telomeres in cortical neurons of the mouse ALS transgenic model, TDP-43 rNLS. Specifically, it determines the occurrence of DNA damage and the alternative lengthening of telomeres (ALT) mechanism through simultaneous labeling of the DNA damage marker, γH2AX, or the ALT marker, promyelocytic leukemia (PML) protein, together with telomeres. Therefore, the protocol enables the visualization of DNA damage (γH2AX) or the ALT marker (PML) concurrently with telomeres. This technique can be successfully applied to brain tissue and enables the investigation of telomeres specifically in cortical neurons, rather than in bulk tissue, offering a significant advantage over Southern blot or qPCR-based techniques. Key features • This protocol enables the labeling of telomeres in mouse brain tissue prepared from paraffin-embedded brain sections. • This method facilitates concurrent labeling of proteins that are colocalized at telomere sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1e2280b8fd8c35a0ab81f6940db9809bdef4684" target='_blank'>
              Colocalizing Telomeres With PML or γH2AX Foci by IF-FISH in Mouse Brain Neurons
              </a>
            </td>
          <td>
            Anna Konopka
          </td>
          <td>2025-11-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 IDH-mutant adult-type diffuse gliomas show variable clinical outcomes influenced by patient age. Recent studies have linked HOX gene activity to age, survival, and tumor aggressiveness in both IDH-mutant astrocytoma (IDH-A) and oligodendroglioma (IDH-O), but the extent to which HOX activity defines shared biology across these entities remains unclear. To address this, we assembled a large DNA methylation dataset of IDH-O (N=1,445) and IDH-A (N=2,232), integrating published cohorts, TCGA, and an in-house NIH collection. Multi-omic analyses included DNA methylation (450K, EPIC, EPICv2), RNA-Seq, somatic copy number alterations (SCNA), mutation, pathology, and clinical features, enabling a comprehensive characterization of HOX-associated biology. Unsupervised clustering of HOX gene body methylation consistently identified two subgroups—HOX-High and HOX-Low—in both IDH-O and IDH-A across all platforms. HOX High tumors were associated with older age, higher tumor grade, increased Ki-67%, shorter survival, and higher SCNA and mutation burdens. Differential methylation analyses revealed that most probes distinguishing HOX High and HOX Low subgroups in IDH-O were also differentially methylated in IDH-A, with similar effect sizes, and vice versa. Notably, probes within the HOXD locus showed the greatest differences in both tumor types. RNA-Seq data revealed congruent differences in HOXD gene expression. Gene ontology analyses showed highly correlated enrichment scores between IDH-O and IDH-A, even after excluding HOX-related genes, suggesting broader shared biology. Importantly, HOX-associated subgroups were observed in both tumor types regardless of 1p/19q-codeletion status, indicating that this signature is independent of the canonical molecular subclassification. Our results support the existence of a pan-IDH-mutant glioma biological program, transcending traditional molecular subtypes. Identification of these HOX-associated subgroups provides new opportunities for biologically informed risk stratification and may inform the development of targeted therapies for IDH-mutant gliomas. Further prospective studies are warranted to explore the clinical utility of HOX-based biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8be8f07e09f14a3e3736db95a5e2b18ba6610668" target='_blank'>
              PATH-15. HOX-Associated Differences between IDH-mutant Astrocytoma and Oligodendroglioma are Independent of 1p/19q-Codeletion
              </a>
            </td>
          <td>
            Nicholas Nuechterlein, Vinny Ha, Drew W Pratt, P. Cimino
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Polyomaviruses (PyVs) are small, circular double-stranded DNA viruses that establish lifelong persistent infections. PyV infections are typically asymptomatic, but they can cause severe disease particularly in immunosuppressed individuals, including BKPyV-associated nephropathy, and Merkel cell carcinoma (MCC) caused by Merkel cell polyomavirus (MCPyV). While most individuals are infected with PyVs during childhood, it is unclear how these viruses persist in the host and what cellular signals prompt their reactivation. The early regions of PyV genomes encode regulatory proteins that control viral replication and host cell transformation in the case of MCPyV, SV40 and murine polyomavirus (MuPyV). All PyV early regions encode Large and Small Tumor antigens (LT, ST), which are splice variants of a common transcript. PyV LTs bind and inhibit RB1 via a conserved LxCxE motif, deregulating E2F transcriptional activity and cell cycle control to promote viral replication and tumorigenesis. The existing paradigm in the field is during normal PyV infection, LT is expressed and drives the host cell into S phase. However, a recent study revealed that cells infected with BKPyV only express LT and replicate the viral genome when already in S phase, but the mechanisms activating LT expression during S phase are unknown. The Non-Coding Control Region (NCCR), found in all PyVs, contains the viral origin of replication and predicted transcription factor (TF) binding sites, yet only a handful have been experimentally validated and shown to regulate PyV early transcription. Here, we set out to identify host TFs that bind the MCPyV NCCR to regulate early transcription in both MCC cells and in an in vitro model of MCPyV infection. We identified E2F1-3 as critical regulators of MCPyV early transcription, which bind to the MCPyV NCCR via a consensus E2 site just upstream of LT/ST transcriptional start site. We discovered E2 sites in the NCCRs of several other polyomaviruses, including MuPyV but not BKPyV. Deletion of the E2 sites in PyV NCCRs downregulates early transcription in luciferase assays. Despite not containing a consensus E2 site, we detected weak E2F binding to the NCCRs of BKPyV and SV40, suggesting that E2Fs also contribute to transcription of their early genes. Overall, our findings challenge the existing paradigm of PyV infection, that LT expression activates E2F signaling via RB1 inhibition, and suggest that E2Fs must already be active in the PyV-infected host cell for LT/ST expression and viral replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38026f5b1f8b196429ba7f6a171e6ca72452e0dc" target='_blank'>
              E2F1-3 activate polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Njh Salisbury, S. Amonkar, A. Roman, DA Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2508a7d83b4a1f4cf3672318d8edc8fabc927a1" target='_blank'>
              Evidence for improved DNA repair in long-lived bowhead whale.
              </a>
            </td>
          <td>
            Denis Firsanov, Max Zacher, Xiao Tian, T. Sformo, Yang Zhao, Greg Tombline, J. Y. Lu, Zhizhong Zheng, Luigi Perelli, Enrico Gurreri, Li Zhang, Jing Guo, Anatoly Korotkov, Valentin Volobaev, S. A. Biashad, Zhihui Zhang, Johanna Heid, Alex Maslov, Shixiang Sun, Zhuoer Wu, Jonathan Gigas, Eric C Hillpot, John C. Martinez, Minseon Lee, Alyssa Williams, Abbey Gilman, Nicholas Hamilton, Ekaterina Strelkova, Ena Haseljic, Avnee Patel, M. Straight, Nalani Miller, J. Ablaeva, L. M. Tam, Chloé Couderc, M. Hoopmann, Robert L. Moritz, Shingo Fujii, Amandine Pelletier, Dan J Hayman, Hongrui Liu, Yuxuan Cai, Anthony K. L. Leung, Zhengdong Zhang, C. B. Nelson, Lisa M. Abegglen, Joshua D. Schiffman, V. Gladyshev, Carlo C. Maley, Mauro Modesti, G. Genovese, M. Simons, J. Vijg, A. Seluanov, Vera Gorbunova
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>1</td>
          <td>66</td>
        </tr>

        <tr id="The extracellular space is a critical environment for discriminating self versus non-self nucleic acids and initiating the appropriate immune responses through signaling cascades to relay information about extracellular nucleic acids. Here, we provide evidence that oxidized mitochondrial DNA is tethered to the surface of select mammalian cells through cell surface proteins and heparan sulfate proteoglycans. We demonstrate that cell surface DNA accumulates in large clusters that partially overlap with domains enriched in RNA binding proteins. Finally, we show that human and murine B cell surfaces contain DNA that can be cleared by the secreted nuclease DNASE1L3, and that patients with a DNASE1L3 missense variant associated with increased risk for autoimmune disease harbor increased levels of surface DNA on B and T cells. Taken together, this work expands the scope of cell surface nucleic acid biology and provides a mechanistic link between cell surface molecules and DNA targeting in autoimmune disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b04efe93fb72cdef3529bf2ec79fd71db767d8a" target='_blank'>
              DNASE1L3 surveils mitochondrial DNA on the surface of distinct mammalian cells
              </a>
            </td>
          <td>
            Jennifer Porat, Valentina Poli, Karim Almahayni, Benson M. George, F. N. de Luna Vitorino, Li Yi, Carolina Bras Costa, Yitong Zhou, Andrew Crompton, Hongxiu Wen, Leon Zheng, Peter K. Gregersen, Suneet Agarwal, Benjamin A. Garcia, Yixuan Xie, Leonhard Möckl, Ivan Zanoni, K. Simpfendorfer, Ryan A. Flynn
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) with isolated trisomies (ITs) represents a distinct cytogenetic subgroup with heterogeneous clinical behaviour and incompletely defined molecular features. To explore its genomic and transcriptomic landscape, we performed next‐generation sequencing (NGS) on 14 AML patients harbouring isolated trisomies of chromosomes 8, 9, 10, 13, 14, 21 and 22. RNA sequencing (RNA‐Seq) was conducted on 15 samples, including 12 with IT and 3 cytogenetically normal AML cases (normal karyotype, NK‐AML) serving as controls. Trisomy 8 was most frequent, followed by chromosomes 13, 14 and 21. Recurrent mutations were identified in epigenetic regulators (DNMT3A, IDH1/2, ASXL1, TET2). Transcriptomic profiling stratified cases into IT‐8, IT‐21 and IT‐13+22 subgroups. Gene set enrichment analysis (GSEA) revealed shared downregulation of cell cycle‐related pathways (e.g., G2M checkpoint) and subgroup‐specific patterns: oxidative stress and unfolded protein response in IT‐8; epithelial‐mesenchymal transition and oxidative phosphorylation in IT‐21; inflammatory signalling (IL‐6/JAK/STAT, TNF‐α/NF‐κB) in IT‐13+22. A core set of 60 differentially expressed genes (DEGs) was shared, with nine hub genes related to cell cycle (MCM4, CDC7, CDC25A, DHFR), proteostasis (HSPA5, DNAJC3, CALR, HSP90B1) and inflammation. Drug sensitivity profiling revealed subgroup‐specific vulnerabilities: IT‐8 to DNA damage checkpoint inhibitors, IT‐21 to PLK/mTOR inhibitors and IT‐13+22 to BRAF/EGFR‐targeted agents. These findings highlight AML‐IT heterogeneity and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a07ac5d29af86fd0590c16e27dfff205a7ea13" target='_blank'>
              Integrated Genomic and Transcriptomic Profiling of Isolated Trisomies in AML Reveals Cell Cycle Dysregulation and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            J. H. Maués, Bruno Kosa L Duarte, M. Svidnicki, Herton Luiz A S Filho, F. Niemann, Adriana da Silva S Duarte, Paula de Melo Campos, P. Moraes-Vieira, S. T. Olalla Saad
          </td>
          <td>2025-11-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="E2F5 is a member of E2F transcription factor superfamily, controlling many molecular activities, such as cell proliferation, cell differentiation, DNA repair and cell death. Therefore, it is closely related to the occurrence, development and prognosis of a variety of cancers. In recent years, with the rapid development of bioinformatics, genomics and epigenetics, this study has further elucidated of E2F5 in the tumor cell cycle. Based on the latest research reports, this study reviewed the structural composition, dynamic activity regulation of E2F5, and how its transcription program driven by carcinogenic activity changed the progress of various tumor cell cycles, especially how it converted from a “gatekeeper” to a “destroyer”, thus affecting abnormal biological behaviors of tumor cells. Our aim is to provide a new direction for the development of E2F5 targeted therapy strategies and drug resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04822ad21c2d133c4bb16efee92ced6e01833c81" target='_blank'>
              The Roles of E2F5 in Tumor Cell Cycle: The Gatekeeper or Destroyer?
              </a>
            </td>
          <td>
            Yingwen Du, Danyun Wang, Aidi Liang, Xinru Tang, Jiansen Chen, Ruizhi Yao, Lei Meng, Jianxing Xie, Ming Chen, Songtao Xiang, Canbin Lin
          </td>
          <td>2025-10-21</td>
          <td>Urology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis, creating new combinations of the genome while also introducing mutations, and influencing GC content. Here we report direct detection of recombination events using highly accurate long-read sequencing from testis tissue of 16 individuals across six primate species and three human sperm samples. Based on methylation patterns, we classify sequencing reads as originating from either somatic or germline cells. We identify 2881 crossovers, 2314 simple gene conversions, and 555 complex events, and analyze their chromosomal distribution. Crossovers are more telomeric, showing stronger concordance with recombination maps than gene conversions. Human samples align with a double-strand break map, whereas other species differ, consistent with variation in PRDM9-directed breaks, although the recombination process is otherwise conserved. Gene conversion tracts are short and of similar length across species (mean 22–95 bp), implying that most non-crossover events are undetectable. We observe GC-biased gene conversion for both single and multiple-SNV events, including sites flanking crossovers. We infer longer gene conversion tracts associated with crossovers (318–688 bp) than with non-crossovers. Highly accurate long-read sequencing combined with methylation-based classification of reads to specific cell types provides a powerful way of studying recombination events in single individuals of any mammalian species. Homologous recombination rearranges genetic information during meiosis to generate new combinations in the genome. Here, the authors report direct, single-individual detection of recombination in primates and humans using long-read sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be14431c3fb0a34e6bd96874bd9859d957c5877b" target='_blank'>
              Long-read sequencing of primate testis and human sperm allows identification of recombination events in individuals
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. Winge, C. Hvilsom, Carmen Oroperv, Lasse Thorup Hansen, J. Berner, Marta Pelizzola, S. Laurentino, N. Neuhaus, Asger Hobolth, Thomas Bataillon, S. Besenbacher, Kristian Almstrup, M. Schierup
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most common (~80%) and lethal ovarian cancer subtype in the United States, characterized by TP53 mutations and DNA repair defects causing chromosomal instability (CIN). KIF18A is an essential cytoskeletal motor protein for cell division in CIN+ cancer cells, but it is not necessary for cell division in normal cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CIN+ cancers. We investigated the use of a novel KIF18A inhibitor ATX020, for selectively targeting CIN+ HGSOC cells using growth inhibition assays, invasion assays, immunoassays, cell cycle analysis, and immunofluorescence techniques. Using DepMap and flow cytometry, we classified a panel of HGSOC cell lines based on aneuploidy scores (AS) and ploidy levels and identified a correlation between these classifications and sensitivity against ATX020. ATX020 induced cytotoxicity through mitotic arrest and DNA damage, and reduced tumor growth in HGSOC with high aneuploidy scores (AS). Mechanistically, ATX020 blocks KIF18A’s plus-end movement on spindle fibers, increasing spindle length, resulting in chromosomal mis-segregation, aneuploidy, and DNA damage. Our findings suggest that ATX020 inhibits CIN+ HGSOC cells mainly by inducing mitotic arrest and DNA damage, disrupting KIF18A’s function crucial for mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f42a6d74edbdfa8c878ba6b61be09681cd9d76" target='_blank'>
              The KIF18A Inhibitor ATX020 Induces Mitotic Arrest and DNA Damage in Chromosomally Instable High-Grade Serous Ovarian Cancer Cells
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Tzu-Ting Huang, Maureen Lynes, Sanjoy Khan, Serena J Silver, Jung-min Lee
          </td>
          <td>2025-11-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Herpesvirales are an ancient viral order that causes lifelong infections in species from mollusks to humans. They export their capsids from the nucleus to the cytoplasm by a noncanonical nuclear egress route that involves capsid budding at the inner nuclear membrane followed by fusion of this temporary envelope with the outer nuclear membrane. Here, using a whole-genome CRISPR screen, we identify ER protein CLCC1 as important for the fusion stage of nuclear egress in herpes simplex virus 1. We also find that the genomes of Herpesvirales that infect mollusks and fish encode CLCC1 genes acquired from host genomes by horizontal gene transfer. In uninfected cells, loss of CLCC1 causes a nuclear blebbing defect, suggesting a role in host nuclear export. We hypothesize that CLCC1 facilitates an ancient cellular membrane fusion mechanism that Herpesvirales have hijacked or co-opted for capsid export and propose a mechanistic model. Herpesvirales utilize a unique nuclear egress route for capsid export. Here the authors show that herpesviruses exploit a cellular membrane protein, once thought to transport chloride, to facilitate membrane fusion and egress from the nucleus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4fb4c70f2d2d09bbfa7bbd4ae09de6478c50bc" target='_blank'>
              ER protein CLCC1 promotes nuclear envelope fusion in herpesviral and host processes
              </a>
            </td>
          <td>
            Bing Dai, Adrian W. Sperl, Lucas Polack, Isabel Mejia, Haley Dame, Tien Huynh, Chloe Deveney, Nathalie Lavoie, Chanyoung Lee, John G Doench, M. Daugherty, E. Heldwein
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ca54f63f6feb36492a0d66c69a169994ddd7a" target='_blank'>
              Leukemia Risk Factor ARID5B Coordinates HDAC-Mediated Transcriptional Repression
              </a>
            </td>
          <td>
            A. P. Kutschat, Fabian Frommelt, Brianda L Santini, Sophie Müller, P. Batty, G. Karbon, Giulio Superti-Furga, Davide Seruggia
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="CCCTC-binding factor (CTCF) is essential for chromatin organization. CTCF interacts with endogenous RNAs, and deletion of its ZF1 RNA binding region (∆ZF1) disrupts chromatin loops in mouse embryonic stem cells (ESCs). However, the functional significance of CTCF-ZF1 RNA interactions during cell differentiation is unknown. Using an ESC–to–neural progenitor cell (NPC) differentiation model, we show that CTCF-ZF1 is crucial for maintaining cell type–specific chromatin loops. Expression of CTCF-∆ZF1 leads to disrupted loops and dysregulation of genes within these loops, particularly those involved in neuronal development and function. We identified NPC-specific, CTCF-ZF1 interacting RNAs. Truncation of two such coding RNAs, Podxl and Grb10, disrupted chromatin loops in cis, similar to the disruption seen in CTCF-∆ZF1–expressing NPCs. These findings underscore the inherent importance of CTCF-ZF1 RNA interactions in preserving cell-specific genome structure and cellular identity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7cc7cc49039ba2724b97571ec654c24aaaca4af" target='_blank'>
              CTCF-RNA interactions orchestrate cell-specific chromatin loop organization
              </a>
            </td>
          <td>
            Kimberly Lucero, Sungwook Han, , Xiang Qiu, Esteban O Mazzoni, Danny Reinberg
          </td>
          <td>2025-11-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 H3K27 mutant(H3K27M) tumors are the most common type of diffuse midline gliomas and are lethal childhood tumors with a median survival of less than 12 months. Radiation is a critical treatment, yet these tumors recur rapidly within the radiation field due to resistance to radiation-induced DNA damage, allowing the tumor cells to survive. However, the interactions mediating efficient DNA repair, radiation resistance, and recurrence are poorly understood. Our goal was to identify mediators of treatment resistance in H3K27M tumors for therapeutic targeting to improve survival.



 We developed an in vivo recurrence model to assess mediators of treatment resistance. Bulk RNA, single cell RNA, and spatial transcriptomics analyses were used to determine drivers of resistance, and genes of interest were mechanistically assessed for their role in resistance.



 We determined that H3K27M epigenetic reprogramming results in activation of cell-surface protein THY1 which is highly upregulated in all H3K27M tumors. We then confirmed in numerous in vitro and in vivo models that THY1 mediates significant treatment resistance. We determined that THY1 mediates resistance through rapid and efficient DNA repair. We then performed extensive mechanistic analyses which revealed that THY1 signaling activates Fyn, a Src family kinase, resulting in downstream DNA repair. Remarkably both biological knockdown of Fyn and pharmacological Fyn inhibition, with brain-penetrating saracatinib, result in delayed DNA repair and tumor cell death.



 We have uncovered that the Thy1-Fyn signaling cascade mediates rapid DNA repair and treatment resistance. Pharmacological Fyn inhibition successfully delays DNA repair which could improve patient survival. As saracatinib is highly brain penetrant(0.5:1 Brain: Plasma ratio) and has demonstrated safety in Phase I/II Alzheimer’s clinical trials, it is an ideal drug for clinical translation. We are currently designing a Phase 0/1 study to assess drug penetration, efficacy, and safety with saracatinib and radiation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c2f1401d86d8dd15e7bf8324894f0a446afbce" target='_blank'>
              EXTH-65. THY1 Mediates Treatment Resistance in H3K7M Pediatric Tumors and is Therapeutically Targetable
              </a>
            </td>
          <td>
            S. Syed, Sunjong Ji, Arushi Tripathy, Yazen Alomary, Reka Chakravarthy, Mateus Mota, Dah-Luen Huang, Zhuwen Wang, Zian Li, Qiang Zhang, Daniel Nava de la Martin, T. Lawrence, D. Wahl, C. Koschmann, S. Venneti, A. Rehemtulla, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with limited therapeutic options and poor prognosis. Growing evidence suggests that transposable elements (TEs) may play critical roles in cancer by promoting genomic instability, altering gene expression through insertional mutagenesis, and modulating immune responses via the expression of TE-derived neoantigens, but their specific impact on IDH-wildtype GBM remains unclear. We investigated the association of TE abundance with survival and oncogenic pathways in GBM.



 We analyzed RNA-sequencing data from 151 IDH-wildtype GBM tumors, classified per WHO 2021 guidelines. TE abundance was quantified in tumor samples across genomic regions (whole genomic, exonic, intronic, intergenic) using REdiscoverTE. Associations with overall survival were evaluated using Cox proportional hazards models. Pathway enrichment analyses were performed for genes correlated with survival-associated TEs.



 Increased abundance of several TEs was significantly associated with reduced survival. Notably, whole-genome BC200 abundance was linked to poorer survival (HR1.75, 95% CI:1.28-2.22, p= 0.0042), with high expressors having a median survival of 9.81 vs 14.7 months. Similarly, LTR7Y (HR=2.24, 95%CI: 1.55-3.25, p=0.011), MER61D (HR=1.48, 95%CI: 1.20-1.82, p=0.055), and exonic L1ME3B (HR=2.27, 95% CI: 1.5-3.44, p=0.022) were significantly associated with inferior survival. These survival associations were independent of MGMT promoter methylation and CDKN2A deletion status. Gene expression correlated with TE abundance revealed enrichment in pathways related to RNA processing (BC200) L1mE3B), cellular motility and axon guidance (LTR7Y, MER61D), and immune regulation (LTR16A), suggesting potential multifaceted roles in GBM progression.



 Our findings demonstrate that specific TEs are significantly associated with inferior survival in IDH-wildtype GBM, highlighting their potential as prognostic markers alongside known somatic mutations. Their association with key tumorigenic processes such as immune evasion and genomic instability further suggests a contributory role in tumor progression and identifies them as potential therapeutic targets. Future studies should investigate the mechanistic functions, regulatory dynamics, and tissue specificity of TEs using longitudinal datasets to better elucidate their role in GBM biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d64d64059b2e2f2e271240ac062c311929bc6e" target='_blank'>
              EPID-30. Abundance of Transposable Elements is Associated with inferior Survival in IDH-wildtype Glioblastoma: A TCGA analysis
              </a>
            </td>
          <td>
            Eduardo Rodriguez Almaraz, G. Guerra, George Wendt, David V Glidden, Aaron Scheffler, Susan M Chang, S. Francis
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastomas (GBMs) are highly aggressive, infiltrative, and heterogeneous brain tumors driven by complex genetic alterations. The basic-helix-loop-helix (bHLH) transcription factors ASCL1 and OLIG2 are dynamically co-expressed in GBMs; however, their combinatorial roles in regulating the plasticity and heterogeneity of GBM cells are unclear. Here, we show that induction of somatic mutations in subventricular zone (SVZ) progenitor cells leads to the dysregulation of ASCL1 and OLIG2, which then function redundantly and are required for brain tumor formation in a mouse model of GBM. Subsequently, the binding of ASCL1 and OLIG2 to each other’s loci and to downstream target genes then determines the cell types and degree of migration of tumor cells. Single-cell RNA sequencing (scRNA-seq) reveals that a high level of ASCL1 is key in specifying highly migratory neural stem cell (NSC)/astrocyte-like tumor cell types, which are marked by upregulation of ribosomal protein, oxidative phosphorylation, cancer metastasis, and therapeutic resistance genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f56d67f3fa305af969dda22297460569970ba3" target='_blank'>
              CNSC-16. TRANSCRIPTION FACTORS ASCL1 AND OLIG2 DRIVE GLIOBLASTOMA INITIATION AND CO-REGULATE TUMOR CELL TYPES AND MIGRATION
              </a>
            </td>
          <td>
            Bianca L. Myers, T. Vue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, posing a serious threat to public health. The ASPN protein has emerged as a key player in certain cancers; however, its precise mechanisms and regulatory roles are still not well-defined. We explored the expression of ASPN and its prognostic significance across various cancers. Functional enrichment analyses were conducted using the Kyoto Encyclopedia of Genes and Genomes (KEGG) and Gene Ontology (GO). The association between ASPN expression and factors including immune cell infiltration, immune checkpoint genes, microsatellite instability (MSI), tumor mutational burden (TMB), drug sensitivity, promoter methylation, and tumor mutation types was examined. Finally, the function of ASPN in stomach adenocarcinoma (STAD) was examined using RT-qPCR, WB, Cell Counting Kit-8, Transwell, scratch, and flow cytometry assays, focusing on the effects of ASPN on cell proliferation, migration, invasion, and apoptosis. ASPN expression was high in several cancers, where higher expression correlated with poorer prognosis. Gene enrichment analysis indicates ASPN is involved in several cellular pathways and correlates with immune cells infiltration, TMB, and MSI. The single-cell analysis showed that ASPN was mainly expressed in fibroblasts, and ASPN mutations predominantly consisted of missense mutations. After using siRNA knockdown of ASPN, CCK-8 assay and wound healing assays exhibited that suppressed cell viability and migration ability of gastric cells, Transwell assay displayed migration and invasion abilities of two gastric cell lines was decreased. In accordance with the result of cell apoptosis, reducing ASPN elevated the rate of apoptosis in two gastric cancer cells. ASPN inhibition leads to upregulation of pro-apoptotic BAX and cleaved Caspase-3, and downregulation of anti-apoptotic BCL2. These findings suggest that ASPN contributes to gastric cancer aggressiveness. Elevated ASPN expression across cancers is associated with poorer prognosis, highlighting its potential as a prognostic biomarker and therapeutic target, particularly in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/810d24b761f28b7df03e982a8efd99836df98c4f" target='_blank'>
              Exploring ASPN as a pan-cancer biomarker with a focus on gastric cancer
              </a>
            </td>
          <td>
            Yue Feng, Sijiang Zhou, Bing Yang, Jian Yang, Shengyu Wang, Zhu Yu, Chao Quan, Junqiang Chen
          </td>
          <td>2025-11-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background/Aim: Hepatocellular carcinoma (HCC) is a heterogeneous liver cancer with limited treatment options and a poor prognosis in advanced stages. To identify novel biomarkers and therapeutic targets, we investigated the role of chromosome 1 open reading frame 50 (C1orf50), a gene with a previously uncharacterized function in HCC. Materials and Methods: We performed a comprehensive transcriptome data analysis of the human hepatocellular carcinoma project from The Cancer Genome Atlas (TCGA) and subsequently validated the oncogenic roles of C1orf50 using HCC cell lines. Results: Using transcriptomic and clinical data from TCGA, we stratified 355 primary HCC samples based on C1orf50 expression levels. Patients with high C1orf50 expression exhibited significantly shorter overall survival, suggesting its association with aggressive tumor behavior. Differential expression and enrichment analyses revealed that C1orf50-high tumors were enriched in oncogenic pathways, including epithelial-mesenchymal transition (EMT), cell cycle activation, and stemness-related properties. Transcriptional regulatory network analysis detected 456 significantly dysregulated regulons, including ZEB1/2 and E2F2, key drivers of EMT and cell cycle, in the C1orf50-high group. In addition, we observed increased YAP1/TAZ signaling, further linking C1orf50 to stemness and therapeutic resistance. Functional data from CRISPR-based dependency screening suggested that several transcription factors up-regulated in the C1orf50-high state, such as ZBTB11 and CTCE, are essential for the survival of HCC cells. These findings indicate potential therapeutic vulnerabilities and support the rationale for targeting C1orf50-associated pathways. Conclusion: C1orf50 is a novel biomarker of poor prognosis in HCC and a key regulator of oncogenic features such as EMT, cell cycle progression, and stemness. This study highlights the therapeutic potential of targeting C1orf50-related networks in aggressive subtypes of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cf5fd5b7c144989d6a903de5542f0ee787b2ca" target='_blank'>
              C1orf50 Accelerates Epithelial-Mesenchymal Transition and the Cell Cycle of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Atsushi Tanaka, Yusuke Otani, Masaki Maekawa, Anna Rogachevskaya, Tirso Peña, Vanessa D. Chin, Shinichi Toyooka, Michael H. Roehrl, Atsushi Fujimura
          </td>
          <td>2025-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chinese hamster ovary (CHO) cells are the leading mammalian system for recombinant therapeutic protein production. However, optimizing transgene expression remains challenging due to the limited understanding of the regulatory mechanisms controlling gene expression in CHO cells. Towards overcoming this barrier, here we provide a systematic characterization of cis-regulatory elements in CHO cells. Using genome-wide STARR-seq, a high-throughput method for quantifying enhancer strength, we identified regions with enhancer activity in the CHO cell genome. By integrating these data with ATAC-seq and histone modification profiles, we were able to characterize the chromatin state of these regions. Our analysis revealed thousands of newly identified enhancer sequences. The most active sequences could drive transgene expression at levels similar to or higher than strong viral enhancers. Notably, half of the regions found to have enhancer activity were within inaccessible chromatin in their native context. We observed that accessible enhancers were primarily near to transcriptional start sites and associated with ubiquitously-expressed genes, whereas inaccessible enhancers were predominantly intergenic and associated with tissue-specific genes. Additionally, through a deep-learning-based approach ETS and YY1 transcription factor (TF) binding motifs were identified as key determinants of enhancer identity and strength. Disrupting YY1 binding motifs led to reduced enhancer activity, thereby highlighting the importance of YY1 as a transcriptional activator in CHO cells. Our study demonstrates the first comprehensive map of functionally-validated enhancers in CHO cells and generates new insights into gene regulation and the role of TFs in determining enhancer strength. This study helps to lay the foundation for strategic engineering of CHO cell transcriptional networks to achieve enhanced biopharmaceutical production.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa658ab6545520fb2736494338b1f0f7330dffa7" target='_blank'>
              Comprehensive Mapping of Functional Enhancers in Chinese Hamster Ovary Cells.
              </a>
            </td>
          <td>
            María Santos, Yusuf B. Johari, Laura Biggins, Natalie C Elliott, S. Schoenfelder, Mounika Boddireddy, Daniel K Fabian, Michael Anbar, Peter O'Callaghan, P. Rugg-Gunn
          </td>
          <td>2025-10-03</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This study explores the role of 5mC regulatory genes in head and neck squamous cell carcinoma (HNSCC), focusing on their association with tumor progression, metastasis, and prognosis. A prognostic model was also developed to predict patient survival. Using expression profile data from the GEO and TCGA databases, the study analyzed the differential expression of 5mC-related genes, their connection with somatic mutations, and their impact on the immune microenvironment. A prognostic risk model based on these genes was constructed through the differential expression of 5mC-related genes. (1) Several 5mC regulatory factors were found to be abnormally expressed in HNSCC, with a significant association with somatic mutations. (2) Different 5mC methylation regulation patterns showed variations in immune infiltration, immune checkpoint expression, tumor mutation burden, and cancer stem cell gene set expression, potentially influencing immunotherapy effectiveness and prognosis. (3) Prognosis-related risk genes, such as DNMT1 and TET2, were identified as predictors of survival outcomes in HNSCC patients. The methylation patterns of 5mC regulatory factors are critical in HNSCC progression and are closely linked to tumor microenvironment infiltration. These findings offer new perspectives for treatment strategies and prognosis prediction in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2107e5b0f7cdb8234aeae2307aae6371c1f39db" target='_blank'>
              Identification of 5mC regulatory factors mediating methylation patterns and prognosis models in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Yanwei Li, Yunfeng Cao, Nuerbiyamu Maimaitiyiming, Yue Li
          </td>
          <td>2025-11-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human papillomavirus-associated head and neck squamous cell carcinomas (HPV+ HNSCCs) lack early diagnostics and continue to rise in incidence. HPV16 has been detected in ∼90% of HPV+ oropharyngeal cancers (HPV+ OPCs), an anatomical subset of HNSCC, with the majority retaining episomal viral genomes. Despite this, existing episomal HPV16+ OPC cell lines are especially scarce. UMSCC104s were initially reported as episomal; however, the literature contains conflicting reports regarding the genome status of these cells. We now show that UMSCC104s rapidly undergo integration and E2 loss under standard monoculture, and later lots are fully integrated. These findings resolve prior discrepancies and underscore the instability of episomes in monoculture. Accurate models of HPV-driven cancers are critically needed. We propose fibroblast co-culture methods, traditionally utilized for HPV+ keratinocyte models, as a strategy to preserve episomal status in cancer models by supporting viral and host genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79383ce31e18bce44c59616029f77e566e53fa0" target='_blank'>
              The Utility of Fibroblast Co-culture for the Maintenance of Episomes in Human Papillomavirus-Associated Cancer Models
              </a>
            </td>
          <td>
            Austin J. Witt, Rachel L. Lewis, Xu Wang, Phoebe Bridy, Jenny D Roe, I. Morgan, C. James, Molly L Bristol
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex
 virome
 , defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored.




 We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe.




 We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma:
 Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1
 , and
 Cyprinid herpesvirus 2
 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75).




 Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) is the fifth-leading cause of cancer-related mortality, with a 5-year survival rate less than 20%. It develops from preneoplastic lesions to adenocarcinoma, but these early genetic alterations remain poorly understood. Therefore, we aimed to identify early genetic drivers underlying the development of preneoplastic lesions and the initiation of gastric carcinogenesis. Methods We characterized preneoplastic and early gastric adenocarcinoma using 48 samples from 16 Guatemalan patients, a country with a high incidence of GC. We sequenced a panel of 127 genes to identify early genetic drivers and possible actionable targets. Results We identified extensive genetic heterogeneity, including single nucleotide and copy number variations. After comparing our data with other studies, we identified TP53 and APC as the most mutated genes in preneoplastic lesions and early GC. Our mean tumor mutational burden was higher in diffuse (0.017 mutations/Mb) and intestinal adenocarcinomas (0.015) than in chronic gastritis (0.005), and analysis of the mutational signatures revealed several processes acting at different stages of the disease. Signatures S15 (DNA mismatch repair deficiency) and S03 (homologous recombination deficiency) were more frequent in early adenocarcinoma than in chronic gastritis, intestinal metaplasia, necrosis, tubular adenoma, and atrophy. Notably, 10 of 16 patients (62.5%) had at least one actionable mutation in their preneoplastic lesions or gastric adenocarcinomas. Conclusions We show that at the preneoplastic and earliest stages, GC is genetically heterogeneous and presents key cancer-driving mutations that may participate in neoplastic transformation and progression, with 62.5% of lesions having the potential for treatment. This study expands the limited research on early GC and highlights key opportunities for precision medicine in populations with high GC incidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f131a17bcf2decc26bf9525e923b46c1c5c1bd" target='_blank'>
              Genetic heterogeneity and key driver mutations in the preneoplastic and earliest stages of gastric cancer
              </a>
            </td>
          <td>
            Eduardo Alfonso Marroquín-Estrada, Héctor Martínez-Gregorio, C. Díaz-Velásquez, Fernando Ambriz-Barrera, Eduardo Emilio Córdoba-García, Laura Itzel Hernández-Romero, Miguel Ruiz De-La-Cruz, Armando Cáceres, Elisa Hernández, José Rodolfo Gil, Elizabeth Orellana, Daniel Penados, Carolina Richter, Marcelino Díaz, Felipe Vaca-Paniagua
          </td>
          <td>2025-10-09</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary We compared the results using a 108-gene RNA-seq panel with those derived from optical genome mapping (OGM) in 467 acute leukemias. The overall concordance rate was 88.1%. For clinically relevant events (n = 234), OGM uniquely identified 15.8%, whereas RNA-seq uniquely identified 9.4%. Concordance was poor for enhancer-hijacking lesions (20.6%), including MECOM, BCL11B, and IGH rearrangements, many of which were not detected by RNA-seq. Conversely, RNA-seq slightly outperformed OGM for fusions arising from intrachromosomal deletions that were sometimes labeled by OGM as simple deletions. Overall, RNA-seq is more sensitive for detecting expressed chimeric fusions, whereas OGM is better for detecting cryptic, enhancer-driven events. Using both platforms provides the most comprehensive structural variant detection in acute leukemia cases. Abstract Background/Objectives: Gene rearrangements involving oncogenes are major drivers in acute leukemia, influencing disease classification, prognosis, and therapeutic decision-making. Targeted RNA sequencing (RNA-Seq) panels capable of detecting intergenic and intragenic fusions across multiple genes are increasingly used in diagnostic settings. However, comparative evaluation with orthogonal technologies remains limited. Material and Methods: We compared the performance of a 108-gene anchored multiplex PCR (AMP)-based RNA-Seq panel with that of Optical Genome Mapping (OGM) in 467 acute leukemia cases. The cohort included 360 cases of acute myeloid leukemia (AML), 89 B-lymphoblastic leukemia (B-ALL), 12 T-lymphoblastic leukemia (T-ALL), and 6 cases of mixed phenotype acute leukemia (MPAL). Results: Results of both methods were concordant in 175 (74.7%) of 234 detected gene/rearrangement fusions. The concordance rate varied significantly across different leukemia types, ranging from 80.2% in B-ALL to 41.7% in T-ALL (p < 0.001) OGM uniquely detected 37 of 234 (15.8%) clinically relevant rearrangements, whereas RNA-Seq exclusively identified 22 of 234 (9.4%). Enhancer-hijacking lesions, including MECOM and BCL11B rearrangements, CDK6::MNX1, and IGH rearrangements, had a markedly lower concordance (20.6%) compared with all other aberrations (93.1%) (p < 0.001). Conversely, some gene fusions arising from intrachromosomal deletions were interpreted by OGM as simple deletions rather than rearrangements or fusions. Conclusions: Targeted RNA-Seq was effective for detecting chimeric fusion transcripts and showed slightly better performance in identifying fusions resulting from deletions. However, OGM was effective for detecting enhancer-hijacking events that do not generate fusion transcripts. Both methods are complementary for the workup of acute leukemia cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/783106fd704ddbf425128b4173188409188a77b1" target='_blank'>
              Comparative Analysis of Targeted RNA-Seq and Optical Genome Mapping for Detecting Gene Rearrangements in Acute Leukemia
              </a>
            </td>
          <td>
            C. Ok, Guilin Tang, S. Loghavi, Shimin Hu, Qing Wei, Andres E. Quesada, M. Routbort, R. Kanagal-Shamanna, C. Yin, Iman Sarami, Sofia A. Garces, Nitin K. Agarwal, R. Luthra, Hong Fang, F. Jelloul, Julian Bryan, L. Medeiros, Keyur P. Patel, Gokce A. Toruner
          </td>
          <td>2025-10-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Liquid biopsies containing circulating tumor DNA (ctDNA) are important biomarkers across several forms of cancer. The detection of mutations in cell-free DNA (cfDNA) indicates the presence of ctDNA. However, unsatisfactory ctDNA mutation sensitivities, issues with sequencing errors, and clonal hematopoiesis variants have limited the clinical utility of mutation-based ctDNA assays. Recently, a new avenue of cfDNA assays has been developed, focusing on cfDNA epigenetics. Here, we outline the recent advancements in cfDNA epigenetics, focusing on cfDNA methylation, fragmentomics, and post-translational modifications (PTMs) of circulating nucleosomes. We present various methylation strategies concerning ctDNA detection and tissue of origin (TOO) analyses. cfDNA fragmentomics focuses on cfDNA fragment lengths, fragment end motifs, and nucleosome positioning to infer gene expression and estimate the ctDNA fraction. Lastly, we discuss the development of cell-free chromatin immunoprecipitation of circulating nucleosomes with PTMs. This method has been implemented to detect tumor gene expression, TOO, and treatment resistance. Combining the epigenetic features of cfDNA will expand the utility of liquid biopsies to give a more comprehensive insight into tumor biology, treatment response, and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0025e7aa342389077db174680eb853f811df7b03" target='_blank'>
              Liquid biopsy epigenetics: establishing a molecular profile based on cell-free DNA.
              </a>
            </td>
          <td>
            C. T. Maansson, A. L. Nielsen, B.S. Sørensen
          </td>
          <td>2025-11-21</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The rate of DNA synthesis is crucial for full DNA duplication. We report a key role of p21 in controlling this rate. During normal replication, p21 promotes nascent DNA synthesis alongside the DNA polymerase iota (Pol ι)/p53 complex. When p21 is down-regulated but detectable, nascent DNA tracks are longer and discontinuous and rely on primase and DNA polymerase (PrimPol). With the complete elimination of p21, nascent DNA tracks become shorter and continuous and depend on Pol kappa (κ). Endogenous p21 levels are critical for genomic stability, as both PrimPol- and Pol κ–mediated syntheses can induce chromosomal instability. The residual expression of p21 in p53-null cells influences the involvement of PrimPol or Pol κ in nascent DNA synthesis and subsequent chromosomal instability. Our results demonstrate that endogenous levels of p21 in cycling cells, insufficient for cyclin-dependent kinase inhibition, prevent genomic instability through proliferating cell nuclear antigen binding (PCNA), limiting PrimPol and Pol κ’s role in nascent DNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31f81e290a07076ab9200968e4922ab77b5675a" target='_blank'>
              Endogenous p21 levels protect genomic stability by suppressing both excess and restrained nascent DNA syntheses
              </a>
            </td>
          <td>
            N. L. Calzetta, S. Mansilla, Candelaria Mares Ahlers, Agostina P. Bertolin, Lilen I. Caimi, Sofía Venerus Arbilla, Jingkun Zeng, Lisa Wiesmüller, V. Gottifredi
          </td>
          <td>2025-10-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 The genomic regions surrounding tumor suppressor genes are often lost as collateral damage during tumorigenesis, providing novel therapeutic vulnerabilities. To systematically identify synthetic lethal pairs of genes within DNA damage repair (DDR) pathways, we conducted a combinatorial genome-wide CRISPR screen. We found that RNASEH2B, a gene frequently co-deleted with RB1 in prostate cancers, emerged as a top vulnerability with knockout of APEX2, which is a nuclease involved in DNA repair. This interaction was validated in RNASEH2B isogenic cell line pairs using whole-genome CRISPR screens and further confirmed across multiple cancer lineages using a focused mini-pool sgRNA library. Single gene knockouts demonstrated that loss of APEX2, specifically its catalytic activity, is sufficient to eliminate RNASEH2B mutant cells both in vitro and in vivo. Importantly, loss of RB1 did not impact the synthetic lethal interaction between RNASEH2B and APEX2. Consistent with prior reports, APEX2 knockout also impairs growth of BRCA1 mutant cells and enhances sensitivity to PARP inhibitors. Overall, these findings position APEX2 as a compelling synthetic lethal target in tumors with RNASEH2B deletion or BRCA1 mutations and support the development of APEX2 inhibitors as a potential monotherapy to improve patient outcome.



 Antoine Simoneau, Steve Lombardo, Ruthie Swain, Serge Gueroussov, Binzhang Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H. Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, Douglas A. Whittington, Yi Yu, Jannik N. Andersen, Teng Teng. APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f840d30e01fa153da53003edd382b7d06deea42" target='_blank'>
              Abstract B039: APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors
              </a>
            </td>
          <td>
            Antoine Simoneau, Steven A. Lombardo, Ruthie Swain, Serge Gueroussov, B. Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, D. Whittington, Yi Yu, Jannik N Andersen, T. Teng
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Significance How new genes arise and gain function is a central question in biology. New genes can evolve from nongenic DNA, yet their adaptive potential remains unclear. Here, we use millions of (semi-)random sequences as experimental models of emerging genes and find that thousands confer phage resistance in Escherichia coli. Expressed random sequences can produce both protein- and RNA-based functions that reprogram cellular systems to counter viral infection. Resistance arises through activation of a cell envelope stress response or downregulation of membrane receptor expression. Our results reveal that genetic novelty, in the form of genes appearing for the first time, can shape host–virus interactions, providing insight into microbial evolution and the surprising ease with which functional genes can emerge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c17bf5a40c5c5bfce399a75abe69e8bb312d9" target='_blank'>
              Emergence of antiphage functions from random sequence libraries reveals mechanisms of gene birth
              </a>
            </td>
          <td>
            Idan Frumkin, Christopher N Vassallo, Yi Hua Chen, Michael T. Laub
          </td>
          <td>2025-10-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc89e3f57ea3c181c1e8b1d97a4236bdf0e388" target='_blank'>
              Disentangling the architectural and non-architectural functions of CTCF and cohesin in gene regulation.
              </a>
            </td>
          <td>
            Takeo Narita, Sinan Kılıç, Yoshiki Higashijima, Natalie M Scherer, Georgios Pappas, Elina Maskey, Chunaram Choudhary
          </td>
          <td>2025-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. Y. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="




 Fusobacterium nucleatum

 , an oral commensal bacterium, has been increasingly recognized for its oncogenic role in gastrointestinal malignancies. In esophageal squamous cell carcinoma (ESCC),

 F. nucleatum

 has been implicated in promoting tumor progression and facilitating immune evasion. However, its relationship with stromal remodeling and the tumor microenvironment (TME) remains unclear.





 We performed integrative analyses using metagenomic profiling and transcriptomic deconvolution, and histopathological assessment of 93 The Cancer Genome Atlas (TCGA)‐ESCC cases and an independent cohort of 126 resected tumors.

 F. nucleatum

 status was determined by microbial abundance and quantitative Polymerase Chain Reaction (q‐PCR).






 F. nucleatum

 ‐positive tumors showed significant enrichment of TNFα/NF‐κB signaling and reduced CD8
 +
 T cell infiltration. Stromal analysis revealed a marked increase in cancer‐associated fibroblasts (CAFs) in

 F. nucleatum

 ‐positive tumors, confirmed by transcriptomic deconvolution and α‐SMA immunohistochemistry. Notably, immunohistochemical analysis demonstrated increased nuclear localization of NF‐κB p65, indicating

 F. nucleatum

 ‐induced NF‐κB activation in tumor cells. Clinically, among elderly patients with poor performance status, the prevalence of

 F. nucleatum

 positivity was significantly higher.






 F. nucleatum

 may contribute to the progression of ESCC by inducing NF‐κB–mediated inflammatory signaling in tumor cells and promoting CAFs activation. Its presence may facilitate immune exclusion and tumor invasion through stromal remodeling. Furthermore,

 F. nucleatum

 positivity may reflect broader host vulnerability, particularly in frail elderly individuals. These findings highlight

 F. nucleatum

 as a potential biomarker of tumor immune dynamics and suggest the importance of maintaining good oral hygiene to reduce

 F. nucleatum

 colonization.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bc0b986ba2712cc4f56e16a39af3c218af77cb8" target='_blank'>
              Intratumoral

 Fusobacterium nucleatum

 Drives Cancer‐Associated Fibroblasts Enrichment and Immune Exclusion in Esophageal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Takashi Ofuchi, Q. Hu, Kazuki Omachi, Kosuke Kanemitsu, Hajime Otsu, Yusuke Yonemura, Taro Tobo, Y. Baba, Masaaki Iwatsuki, Koshi Mimori
          </td>
          <td>2025-11-13</td>
          <td>Annals of Gastroenterological Surgery</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="This study aims to develop a ceRNA network associated with the chromosomal passenger complex (CPC) and identify a prognostic signature in lung cancer, the most diagnosed globally, to better understand the molecular mechanisms underlying tumor progression. The study used R packages and publicly available databases to conduct multi-omics In-silico analyses on CPC. These tools facilitated gene expression profiling, prognostic assessment, exploration of miRNA (microRNA), lncRNA (long non-coding RNA), transcription factor interactions, and pathway enrichment analysis. Molecular docking tools were used to evaluate binding affinities of CPC proteins with tobacco carcinogens, selective Aurora kinase B inhibitors, FDA-approved chemotherapeutics, and natural compounds. Immune landscape analysis was conducted using the SPRING viewer to visualize immune cell subpopulations in NSCLC, validated by correlation analysis using the GSCA database. The study reveals that CPC genes—BIRC5, CDCA8, INCENP, and AURKB—are overexpressed in lung adenocarcinoma (LUAD) and are associated with poor overall survival, especially in smokers. A dysregulated ceRNA axis involving lncRNA TMPO-AS1 and miRNA-hsa-let-7b-5p was identified, along with transcription factor E2F1, which shows a strong correlation with the CPC genes. Notably, TMPO-AS1 and E2F1 are positively correlated, while hsa-let-7b-5p is negatively correlated with the CPCs, contributing to tumor progression. Downregulation of hsa-let-7b-5p is linked to poorer outcomes, highlighting its potential as a therapeutic target. Nicotine and NNK show stable binding, suggesting potential roles in activating the CPC pathway and contributing to LUAD progression. CPCs have a strong binding affinity with Hesperidin, a natural bioflavonoid, compared to known chemotherapeutic agents like docetaxel and paclitaxel. CPC genes are negatively correlated with CD4⁺ T cells, indicating a role in promoting an immunosuppressive tumor microenvironment. Lung adenocarcinoma patients have poorer prognosis due to higher levels of CPCs, TMPO-AS1, and E2F1. A sponge complex between TMPO-AS1 and hsa-let-7b-5p may contribute to the tumor progression, and targeting CPCs with natural compounds could offer therapeutic potential. Highlights The overexpression of chromosomal passenger complex genes, AURKB, BIRC5, CDCA8, and INCENP is significantly associated with poor prognosis in lung adenocarcinoma (LUAD), particularly among smokers. The competing endogenous RNA (ceRNA) axis, which involves the long non-coding RNA TMPO-AS1 and the miRNA hsa-let-7b-5p, regulates the expression of these CPC genes. TMPO-AS1 shows a positive correlation with CPC genes, while hsa-let-7b-5p shows a negative correlation. Survival analysis indicates that the combined expression of CPC genes, TMPO-AS1, hsa-let-7b-5p, and E2F1 may serve as a reliable prognostic biomarker panel for LUAD in smokers. Hesperidin exhibits a strong binding affinity to CPC proteins, particularly AURKB, when compared to Barasertib, Docetaxel, and Paclitaxel, highlighting its potential as a therapeutic agent. The overexpression of CPC genes, E2F1, and TMPO-AS1 in LUAD is strongly associated with reduced infiltration of CD4⁺ T cells, indicating their role in promoting an immunosuppressive tumor microenvironment. The overexpression of chromosomal passenger complex genes, AURKB, BIRC5, CDCA8, and INCENP is significantly associated with poor prognosis in lung adenocarcinoma (LUAD), particularly among smokers. The competing endogenous RNA (ceRNA) axis, which involves the long non-coding RNA TMPO-AS1 and the miRNA hsa-let-7b-5p, regulates the expression of these CPC genes. TMPO-AS1 shows a positive correlation with CPC genes, while hsa-let-7b-5p shows a negative correlation. Survival analysis indicates that the combined expression of CPC genes, TMPO-AS1, hsa-let-7b-5p, and E2F1 may serve as a reliable prognostic biomarker panel for LUAD in smokers. Hesperidin exhibits a strong binding affinity to CPC proteins, particularly AURKB, when compared to Barasertib, Docetaxel, and Paclitaxel, highlighting its potential as a therapeutic agent. The overexpression of CPC genes, E2F1, and TMPO-AS1 in LUAD is strongly associated with reduced infiltration of CD4⁺ T cells, indicating their role in promoting an immunosuppressive tumor microenvironment. Schematic representation of the regulatory axis involving TMPO-AS1, hsa-let-7b-5p, and CPC genes in lung tissue. A Normal lung microenvironment: Under physiological conditions, low expression of lncRNA TMPO-AS1 and high levels of miRNA hsa-let-7b-5p maintain controlled expression of CPC genes. The upregulated miRNA binds CPC transcripts, leading to their degradation or translational repression, thereby preserving chromosomal stability and supporting an active immune microenvironment with effective CD4+ T cell infiltration. B Tumor lung microenvironment (LUAD associated with smoking): Chronic exposure to tobacco carcinogens leads to upregulation of TMPO-AS1 and downregulation of hsa-let-7b-5p. TMPO-AS1 sponges hsa-let-7b-5p, limiting its ability to suppress CPC transcripts, resulting in CPC overexpression. This promotes chromosomal instability, supports tumor growth, and contributes to the establishment of an immunosuppressive microenvironment characterized by reduced CD4+ T cell infiltration Schematic representation of the regulatory axis involving TMPO-AS1, hsa-let-7b-5p, and CPC genes in lung tissue. A Normal lung microenvironment: Under physiological conditions, low expression of lncRNA TMPO-AS1 and high levels of miRNA hsa-let-7b-5p maintain controlled expression of CPC genes. The upregulated miRNA binds CPC transcripts, leading to their degradation or translational repression, thereby preserving chromosomal stability and supporting an active immune microenvironment with effective CD4+ T cell infiltration. B Tumor lung microenvironment (LUAD associated with smoking): Chronic exposure to tobacco carcinogens leads to upregulation of TMPO-AS1 and downregulation of hsa-let-7b-5p. TMPO-AS1 sponges hsa-let-7b-5p, limiting its ability to suppress CPC transcripts, resulting in CPC overexpression. This promotes chromosomal instability, supports tumor growth, and contributes to the establishment of an immunosuppressive microenvironment characterized by reduced CD4+ T cell infiltration">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab8191e4b2596144719c4592edb4b9f92354b1" target='_blank'>
              Long noncoding RNA TMPO-AS1 upregulates chromosomal passenger complex expression to promote cell proliferation in lung cancer via sponging microRNA let-7b-5p
              </a>
            </td>
          <td>
            Prerna Vats, Bhavika Baweja, Chainsee Saini, Sakshi Soni, Tripti Tungariya, Aditi Singh, Ashok Kumar, Rajeev Nema
          </td>
          <td>2025-10-15</td>
          <td>Cell Division</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 In frame fusions between Fibroblast Growth Factor Receptor 3 (FGFR3) and Transforming Acidic Coiled-Coil containing protein 3 (TACC3) genes on chromosome 4p16.3 have been reported across diverse cancers, including bladder cancer (∼5%) and IDH-wildtype glioblastoma (4.1%), as well as in small subsets of non-small cell lung (∼0.5%), esophageal squamous cell (1.4–2.1%), cervical (1.9%), head and neck (3.7%), and rare nasopharyngeal (2.5%) carcinomas, and other cancer types. These chromosomal translocations generate FGFR3-TACC3 (F3T3) driver oncofusion proteins which promote tumor cell growth and survival. The F3T3-driven cancer cells are critically dependent on the constitutively active oncofusion kinase activity for survival and proliferation, a phenomenon known as "oncogene addiction". While inhibitors of FGFR kinases, such as erdafitinib and fexagratinib, are the treatment of choice for F3T3-driven cancers, most treated patients do not respond to the drug, suggesting insufficient inhibition of the oncofusion activity. Additionally, most tumor responses to FGFR kinase inhibitors are partial and short-lived, followed by development of resistance and disease progression within few months. Based on the oncogene addiction of cancer cells on the F3T3 activity, we hypothesized that rapid ablation of this oncofusion protein can induce robust apoptotic response, leading to significant tumor regression and sustained remissions. To this end, we explored the use of Proteolysis Targeting Chimeric (PROTAC) bifunctional small molecules which can rapidly degrade F3T3 oncofusion protein by harnessing the cell's own proteasome machinery. We synthesized more than 100 novel PROTAC small molecules targeting F3T3 fusion through combination of previously reported TACC3 ligands with various linkers and ligands of E3 ubiquitin ligases CRBN, VHL or XIAP. We evaluated our new anti-F3T3 PROTAC molecules using F3T3-driven bladder (RT4 and RT112) and breast (SUM185PE) cancer cell lines, which express two of the most common oncofusion variants. Several PROTACs were able to degrade the native F3T3 oncofusion protein and suppressed the growth of F3T3-driven cells. One PROTAC (KHS-C4) degraded F3T3 at low nanomolar concentrations, strongly suppressed the oncogenic pathway downstream F3T3 fusion, had excellent predicted pharmacological properties, acceptable in vitro liver microsome stability (Clint = 59 mL/min/kg for human and 318 mL/min/kg for mouse microsomes) and no in vivo toxicity. Importantly, low concentrations of KHS101-C4 (but not of FGFR3 kinase inhibitors) eradicated F3T3-driven cancer cells (but not cancer cells that do not express the F3T3 oncofusion) in vitro. These data suggest that chemically-induced degradation of the F3T3 fusion protein can lead to oncogenic signal ablation sufficiently potent to induce a lethal oncogenic shock and can thus yield more durable responses than FGFR3 kinase inhibitors. KHS-C4 was selected for further evaluation to establish in vivo proof of concept for the PROTAC-based therapeutic strategy against F3T3-driven cancers.



 Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea. Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5bb4e0b8d5df4124ad0dd08279a3f1803abead8" target='_blank'>
              Abstract B120: Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Early disease progression, including histological transformation, can occur in follicular lymphoma, but tumor heterogeneity makes comparison difficult. We analyzed cell-free DNA from 30 patients with follicular lymphoma to assess cell-free DNA concentration and gene mutations. Mutational profiles were also examined in 16 cases using DNA from formalin-fixed, paraffin-embedded tissue samples. Cell-free DNA concentrations were higher in cases with histological transformation. Both the number of mutations per patient and the number of mutated genes were greater in cell-free DNA than in formalin-fixed, paraffin-embedded tissue-derived DNA. Notably, TP53 mutations were more frequently detected in cell-free DNA. CIITA mutations were more commonly observed in cases with a low frequency of somatic hypermutation in VH genes. Cell-free DNA analysis in follicular lymphoma may become a complementary approach, overcoming the limitations of assessing disease status from single biopsy sites. Further studies will examine the usefulness of cell-free DNA in predicting disease progression in follicular lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc0cb12618903a19d09041fcad50558a34c24c" target='_blank'>
              Genetic analysis of cell-free DNA in follicular lymphoma in comparison with tissue-derived DNA.
              </a>
            </td>
          <td>
            Yoshikazu Hori, Hiroki Hosoi, M. Osuga, Ryuta Iwamoto, Fumiyo Maekawa, Tadashi Okamura, S. Murata, T. Mushino, Motomi Osato, Hitoshi Ohno, Nobuyuki Yamamoto, Shin-ichi Murata, Yasuhiro Koh, Sonoki Takashi
          </td>
          <td>2025-10-15</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf04ff0b527b927234e0bc04a121079916f0a15" target='_blank'>
              HERV-R (ERV3-1) Env gene knockout reduces invasion, migration, and tumor growth in DLD1 colon cancer cells.
              </a>
            </td>
          <td>
            Min-Hye Kim, Eun-Ji Ko, Hyojin An, Seongsoo Choi, Heui-Soo Kim, S. Leem, H. Cha
          </td>
          <td>2025-11-20</td>
          <td>BMB reports</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="DNA single-strand break (SSB) repair defects lead to hereditary neurological syndromes. Spinocerebellar ataxia with axonal neuropathy type 1 (SCAN1), is caused by the homozygous H493R mutation in tyrosyl-DNA phosphodiesterase 1 (TDP1), an enzyme that initiates the repair of DNA topoisomerase 1 (TOP1)-induced SSBs by unlinking the TOP1 peptide from the break. Although TDP1 also initiates the repair of TOP1-induced DNA double-strand breaks (DSBs) associated with transcription, the role of TOP1-induced DSBs in SCAN1 pathology remains unclear. Here, we have addressed the impact of the SCAN1/H493R mutation on the repair of TOP1-induced DSBs. We demonstrate that while TDP1 loss delays the repair of these breaks, SCAN1/H493R completely blocks it in RPE-1 quiescent cells. This blockage is specific to DSBs and is accompanied by a prolonged trapping of mutated TDP1 on DNA, but not of TOP1 cleavage complexes (TOP1cc). Intriguingly, the H263A inactivating mutation of TDP1, which accumulates TOP1cc, also blocks TOP1-induced DSB repair. Importantly, both SCAN1/H493R and H263A mutations exhibit genome instability and cell death. Moreover, we demonstrate that tyrosyl-DNA phosphodiesterase 2 (TDP2) can compensate for TDP1 loss in RPE-1 quiescent cells. Collectively, our data support the potential role of TOP1-induced DSBs as a main contributor to certain hereditary neurological syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebb4417da0d9de2b620ef12c2f66c797e441d074" target='_blank'>
              H263A and SCAN1/H493R mutant TDP1 block TOP1-induced double-strand break repair during gene transcription in quiescent cells and promote cell death
              </a>
            </td>
          <td>
            Diana Rubio-Contreras, Daniel Hidalgo-García, Carmen Angulo-Jiménez, Esperanza Granado-Calle, Margarita Sabio-Bonilla, , Fernando Gómez-Herreros
          </td>
          <td>2025-11-28</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eeb5a1876276ca40bae5f7f5ce7e3154844650e9" target='_blank'>
              Regulation of nucleolar dominance through allopolyploidization of hexaploid wheat
              </a>
            </td>
          <td>
            Yohta Hyuga, Tohru Nakano, Hirokazu Handa
          </td>
          <td>2025-11-06</td>
          <td>Theoretical and Applied Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is a common, aggressive tumor with treatment challenges due to varied patient responses and ineffective biomarkers. mRNA-based therapeutic approaches, particularly in cancer immunology, have become an important area of research. This study investigates nucleoside diphosphate linked moiety X hydrolase 16 (NUDT16), a key enzyme involved in mRNA quality control through 5’ cap-dependent decapping, focusing on its expression profile and prognostic implications in clear cell renal cell carcinoma (ccRCC). Utilizing a comprehensive multi-omics strategy, we integrated mRNA expression profiles from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression (GTEx), and Gene Expression Omnibus (GEO) datasets (GSE68417 and GSE66270). We systematically evaluated NUDT16 expression across pan-cancer, ccRCC, and normal tissues, and assessed its association with clinical parameters. A prognostic model was developed using Cox proportional hazards regression and subsequently validated. Co-expressed genes of NUDT16 were subjected to functional enrichment analysis, and a protein-protein interaction network was constructed. We further investigated NUDT16’s role in immune infiltration, immunotherapy response, methylation status, and CpG site prediction in ccRCC. Single-cell omics analysis and functional assays were performed to elucidate NUDT16’s biological functions. NUDT16 expression was significantly downregulated in ccRCC compared to normal tissues, with progressive reduction observed across advancing pathological stages and histological grades. The prognostic model incorporating NUDT16 expression and clinicopathological features demonstrated robust predictive accuracy for ccRCC patient survival. Gene Set Enrichment Analysis (GSEA) of NUDT16 co-expressed genes revealed significant enrichment in biological processes related to proximal tubule transport and DNA methylation. NUDT16 expression exhibited significant correlations with immune infiltration patterns, particularly NK CD56bright, TReg, and Tcm cells. Reduced NUDT16 expression was associated with enhanced response to CTLA4(+)PD1(-) immune phenotype scores, increased tumor immune dysfunction and exclusion scores, and elevated tumor mutation burden. Promoter region hypermethylation of NUDT16 was significantly elevated in ccRCC and correlated with adverse survival outcomes. NUDT16 overexpression inhibited proliferation, colony formation, migration, and invasion in ccRCC cell lines (ACHN and 786-O). NUDT16 emerges as a potential prognostic biomarker in ccRCC, with its downregulation significantly associated with unfavorable clinical outcomes, altered immune infiltration profiles, and promoter region hypermethylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a6f28f97dded165be34551d1a4dd4dc158bf25" target='_blank'>
              Prognostic, molecular characterization, and immune infiltration analysis of NUDT16 in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Chen Chen, Zhuojing Hu, Wei Zhao, Yunbo Ma, Qinghua Xia
          </td>
          <td>2025-11-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Centromeres are the chromosomal sites at which the kinetochore forms and attaches to spindle microtubules, directing chromosome segregation. Plasmids based on centromeres can maintain stability and distribute accurately during cell division, supporting them as effective genetic tools for research and biotechnological applications. Here, the centromeric regions on seven chromosomes of Ogataea polymorpha, a methylotrophic non‐conventional yeast with great potential in biotechnology, were located by ChIP‐seq with the native Cse4. The actual centromeric sequences of chromosomes 1, 2 and 5 were characterised, which are very different from those of other eukaryotes and each unique. Although long terminal repeats (LTR) were found in these centromeres, they are ‘solo LTR elements’ and not important for the function of centromeres. Hence, the O. polymorpha centromeres were categorised into small regional centromeres. O. polymorpha centromeric plasmids were constructed for the first time, which exhibit high genetic stability and compatibility. Application potential for multigene or multicopy expression was validated by the production of uricase and triterpene squalene. This research elucidates the structural features of O. polymorpha centromeres and constructs new, stable centromeric plasmids, expanding the phenomenal diversity of centromeres and providing a powerful genetic toolbox for multi‐node and multiple‐layered engineering of cell metabolism and physiological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8595896eccb7f21123d64f19b9f60bcb2fa63adb" target='_blank'>
              Centromeric Sequences in Ogataea polymorpha Genome Enable Development of Stable Multigene Expression Plasmid Tools
              </a>
            </td>
          <td>
            Yanfei Cheng, Yuanyuan Shi, Ning Sun, Xin Zhang, Mengwei Sun, Xiuping He
          </td>
          <td>2025-11-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf4c8fc173b098438a554dbc52df28600269e7a7" target='_blank'>
              G-Quadruplex and i-Motif Structures in the SHMT1 5’UTR Modulate Gene Expression
              </a>
            </td>
          <td>
            Rosalia Palumbo, Manju Kasaju, Sophia C. Hershey, Morgan E. McCann, Zoe H. Woon, David B. Heisler, M. Mihailescu
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA damage activates the DNA damage response (DDR) machinery. However, aging impairs DDR in neurons, thereby increasing susceptibility to neurodegenerative diseases, such as Huntington’s disease (HD). The mutant huntingtin (mHTT) protein interferes with DNA repair, leading to DNA lesions and a feedback loop of cellular stress that accelerates neurodegeneration. Although the individual roles of FBXW7, ATM, and checkpoint kinase (CHK) are well-known in DDR, their combined roles in HD remain unclear. In this study, we investigated the FBXW7-mediated CHK2 pathway in HD, in which mHTT levels increase, whereas wild-type (WT) HTT levels decrease. HD cells containing mHTT or expanded polyQ-HTT were more prone to DNA damage than cells containing wtHTT or normal-length polyQ, demonstrating the increased vulnerability of HD neurons. Downregulating the expression of FBXW7 reduces susceptibility to DNA damage and promotes cellular stability. Additionally, FBXW7 specifically prevented CHK2 degradation, but not CHK1 degradation. This suggests a selective role in DDR regulation. Thus, the FBXW7-CHK2 pathway may alleviate DNA damage in HD by supporting DDR and inducing cell cycle arrest. The intricate relationship between DDR and HTT is fundamental to the pathophysiology of HD. Elucidating these mechanisms could facilitate the development of new therapeutic strategies that enhance DNA repair or correct DDR dysfunction, thereby slowing disease progression or delaying symptom onset. Understanding this pathway may provide insights into the targeting of DNA repair defects in HD and related neurodegenerative disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed23932fd450c3ceb432313a0f6f217089345cd1" target='_blank'>
              FBXW7-mediated CHK2 regulation modulates DNA damage response and cellular stability in Huntington’s disease
              </a>
            </td>
          <td>
            Tae Eun Kang, Yu Min Lee, Seung Ho Choi, Kyoungjoo Cho
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cells typically regulate their size within a relatively tight range by coupling growth to the cell division cycle using a dedicated set of molecular mechanisms. In budding yeast, cells are born with a similar amount of the G1/S inhibitor protein Whi5 that is then diluted by growth throughout G1. As cells grow, Whi5 concentration decreases and cells become more likely to enter the cell cycle. Cells are born in G1 with similar amounts of Whi5 because of the size-independent (sub-scaling) expression of WHI5 mRNA during S/G2/M phases and the equal partitioning of Whi5 protein at division. While the latter is known to be achieved by association with chromatin before anaphase, the mechanism for the former is poorly understood. Through systematic mutations of the WHI5 promoter, we discovered that WHI5’s core promoter region located -126 to -75 base pairs upstream of the start codon is responsible for sub-scaling expression. This sequence contains a repeating array of binding sites for the transcription factors Fkh1 and Fkh2. Mutation of any of these sites, deletion of either FKH1 or FKH2, or preventing Fkh1 or Fkh2 dimerization weakens the sub-scaling of WHI5 transcription. Taken together with structural predictions and a mathematical model of cooperative Fkh-DNA binding, we conclude that WHI5’s sub-scaling transcription is regulated by a Fkh1/2 heteropolymer binding an array of sites in its core promoter.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c744df826b38252b0c2344e1ddfca307457bd790" target='_blank'>
              A Fkh1/2 binding site array in the WHI5 promoter drives sub-scaling transcription
              </a>
            </td>
          <td>
            Jacob Kim, Shicong Xie, L. Valenzuela, Jordan Xiao, Michael C. Lanz, Xin Gao, Matthew P. Swaffer, Corbin E. Mitchell, Seth M. Rubin, K. Schmoller, J. Skotheim
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, G. Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, J. E. Meyer, S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb3cfc8c5b653932644073b1e9b5efeba85464a2" target='_blank'>
              MAPK3 Modulates Enhancer-Promoter Interactions of SKAP2 in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Yanping Hu, Fang Chen, Tingjie Wang, Bo Wang, Zhizhong Wang, Jiawen Zheng, Yifen Shen, Yihang Shen, Jie Ma, Yongjun Guo
          </td>
          <td>2025-10-25</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to establish divergent gene regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under "cellular constraint". We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and noncoding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share signals of positive and negative selection, suggesting a dynamic role in influencing human traits and cellular phenotypes. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd414947d1caaf71463120749f3e2d01e73021fa" target='_blank'>
              Conserved facultative heterochromatin across cell types identify regulatory sequences underpinning cell identity and disease.
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, Fei-Fei Cheng, Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2025-10-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea9805a426118d493f4c3e4084ab4ad0614ebdb" target='_blank'>
              DNA Repair in Ensuring Genome Stability and Human Health
              </a>
            </td>
          <td>
            N. Rechkunova, O. I. Lavrik
          </td>
          <td>2025-11-01</td>
          <td>Russian Journal of Genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Platinum-based drugs (Pt drugs) are widely used in cancer chemotherapy, yet their genome-wide binding patterns remain incompletely understood. Here, we present Pt sequencing (Pt-seq), an antibody-assisted, genome-wide method for mapping Pt-DNA adducts at single-base resolution. By using exo- and endonucleases to remove background, Pt-seq enables robust and sensitive profiling of binding sites for cisplatin, oxaliplatin, lobaplatin, and a Pt(IV) complex. Using Pt-seq, we identified tens to hundreds of binding clusters that are 10 to 20 kilobases in median length and highly consistent among different drugs, predominantly localizing to centromeric and ribosomal DNA regions. In cisplatin-resistant cells, we found significantly reduced binding within these regions. Moreover, we found that mutations in cancer cells can generate previously unidentified binding sites. On this basis, we demonstrated that ICR-191, an acridine orange compound capable of inducing G insertions, enhances cisplatin-DNA binding and sensitizes cells to cisplatin. Collectively, Pt-seq sensitively profiles Pt–DNA interactions and deepens our understanding of the genome-wide effect of chemotherapeutic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8b8af89d2820c69ce2eeb05e85318ac412b23f" target='_blank'>
              Pt-seq unveils the genomic binding pattern of platinum-based drugs
              </a>
            </td>
          <td>
            Chuyun Shao, Mingyang Li, Liuyi Liu, Dongsheng Bai, Jinmin Yang, Cong Liu, Linxiang Cai, Xiaoyong Wang, Jinying Peng, Zijian Guo, C. Yi
          </td>
          <td>2025-10-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Organisms rely on coordinated networks of DNA repair pathways to protect genomes against toxic double-strand breaks (DSBs), particularly in germ cells. All repair mechanisms must successfully negotiate the local chromatin environment in order to access DNA. For example, nucleosomes can be repositioned by the highly conserved Nucleosome Remodeling and Deacetylase (NuRD) complex. In Caenorhabditis elegans, NuRD functions in the germline to repair DSBs - the loss of NuRD's ATPase subunit, LET-418/CHD4, prevents DSB resolution and therefore reduces fertility. In this study, we challenge germlines with exogenous DNA damage to better understand NuRD's role in repairing DSBs. We find that let-418 mutants are sensitive to cisplatin and hydroxyurea: exposure to either mutagen impedes DSB repair, generates aneuploid oocytes, and reduces fertility and embryonic survival. These defects resemble those seen when the Fanconi anemia (FA) DNA repair pathway is compromised, and we find that LET-418's activity is epistatic to that of the FA component FCD-2/FANCD2. We propose a model in which NuRD is recruited to the site of DNA lesions to remodel chromatin and allow access for FA pathway components. Together, these results implicate NuRD in the repair of both endogenous DSBs and exogenous DNA lesions to preserve genome integrity in developing germ cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77bfc2c8ad230781b0b0137c6deec8663495ed6" target='_blank'>
              NuRD chromatin remodeling contributes to repairing exogenous DSBs in the Caenorhabditis elegans germline.
              </a>
            </td>
          <td>
            Deepshikha Ananthaswamy, Kelin Funes, Thiago Borges, Scott Roques, Nina Fassnacht, Sereen El Jamal, P. Checchi, T. W. Lee
          </td>
          <td>2025-11-17</td>
          <td>Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, 
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Disclosure: C. Springer: None. T. Janas: None. K.H. Stopsack: None. D.R. Schmidt: None. D. Ma: None. Z. Li: None. M.G. Vander Heiden: None. K.L. Penny: None. P.A. Scheet: None. T.L. Lotan: None. A. Amon: None. L.A. Mucci: None. X. Su: None. Prostate cancer (PCa) remains the second leading cause of cancer-related mortality among men in the United States. Aneuploidy, characterized by imbalanced chromosome numbers, correlates with lethal PCa progression. Chromosome 8q (chr8q) gain is one of the most frequent aneuploidy events, occurring in 23% of PCa cases. Recently, we ranked odds ratio (OR) for each chr8q gene to assess the long-term risk of metastases and death from PCa (lethal disease) within 403 patients in the HPFS and PHS cohorts. We observed that the squalene monooxygenase (SQLE) is one of the top ranked genes on chr8q, with an OR of 2.2. SQLE, encoding the key enzyme in cholesterol biosynthesis pathway, has been shown to be associated with poor prognosis in PCa, suggesting it as a potential target to treat aggressive PCa. Datasets such as The Cancer Genome Atlas (TCGA) and the Prostate Cancer Atlas were used to correlate the copy number and mRNA expression levels and the progression of PCa. We focused on mouse prostate organoids (both normal and cancer-like) and human metastatic prostate cell line, VCaP cell, expressing TMPRSS2-ERG gene, which approximately 50% of PCa cases harbors. We used lentiviral transduction to introduce either SQLE overexpression in prostate organoids (which harbor euploid SQLE) or SQLE shRNA in VCaP cells (which harbor increased SQLE). Molecular and cellular approaches were employed to study the influences of altered SQLE levels on target prostate cells. Ultra performance liquid chromatography (UPLC) with mass spectrometry (MS) was used to analyze the changed levels of squalene and lipid profiles when SQLE levels were altered. Overexpression of SQLE in cancer-like mouse prostate organoid models led to increased invasive structures and proliferation. RNA sequencing data suggested that alteration in SQLE levels changes gene signatures related to lipid metabolisms. Interestingly, SQLE overexpression decreased TMPRSS2-ERG protein levels in the organoids. In contrast, SQLE knockdown in VCaP cells resulted in significantly increased TMPRSS2-ERG protein levels. ERG is associated with fatty acid metabolism in PCa cohorts. Our findings suggest that aneuploidy-associated overexpression of SQLE drives PCa aggressiveness by modulating lipid metabolism, particularly through its impact on cholesterol biosynthesis and TMPRSS2-ERG regulation. Targeting SQLE could present a novel therapeutic approach to treat aggressive PCa. Presentation: Saturday, July 12, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f061b46783468debe5f7cd035eb3bdd4d59bdd4b" target='_blank'>
              SAT-314 Elevated Expression Of Squalene Epoxidase Gene (sqle) Promotes Prostate Cancer Aggressiveness By Altering Lipid Metabolism
              </a>
            </td>
          <td>
            Chloe Springer, Thomas Janas, Konrad H. Stopsack, Daniel R. Schmidt, Duanduan Ma, Zhe Li, M. V. Vander Heiden, Kathryn L Penny, Paul A. Scheet, Tamara L. Lotan, Angelika Amon, Lorelei A Mucci, Xiaofeng Su
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84210062581c67c09a67215ecb3a57b62da836b" target='_blank'>
              Adenine DNA methylation associated with transcriptionally permissive chromatin is widespread across eukaryotes.
              </a>
            </td>
          <td>
            Pedro Romero Charria, Cristina Navarrete, Vladimir Ovchinnikov, Lan Xu, Luke A. Sarre, Victoria Shabardina, Ewa Księżopolska, Elena Casacuberta, David Lara-Astiaso, Arnau Sebé-Pedrós, Alex de Mendoza
          </td>
          <td>2025-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Herpes simplex virus (HSV) has a complicated life cycle including stages of primary lytic infection, latent infection, and reactivation. Although the HSV genomic DNA within the viral capsid is devoid of histones, it rapidly associates with histones upon entering the nucleus to form viral chromatin. This chromatin is not integrated into the host chromosome and displays features distinct from the cellular chromatin. The composition, structure, and post-translational modifications of the HSV chromatin change over the course of infection due to the actions of numerous viral and host molecules. In turn, the chromatin states influence the transcription profiles of viral genes at all stages of the viral life cycle and may dictate the outcomes of the lytic-latent balance. These mechanisms may be exploited to develop new antiviral therapeutics. This review summarizes current knowledge about the formation, regulation, and functions of the HSV chromatin and discusses the questions remaining to be answered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83596318a5b50f9a1cdf7828926f8638397595da" target='_blank'>
              Chromatin Regulation of HSV Gene Transcription
              </a>
            </td>
          <td>
            Yuxuan Zheng, Juncheng Zhang, Dongli Pan
          </td>
          <td>2025-10-23</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epithelioid hemangioendothelioma (EHE) is a difficult to treat vascular sarcoma defined by TAZ-CAMTA1 (TC) or YAP-TFE3 (YT) fusion proteins. Human cell lines needed to further understand the pathogenesis of EHE have been lacking. Herein, we describe a method to generate EHE extended primary cell cultures. An integrated multi –omic and functional approach was used to characterize these cultures. The cell cultures, relatively homogenous by single cell RNA-Seq, demonstrated established characteristics of EHE including increased proliferation, anchorage independent growth, as well as the overall gene expression profile and secondary genetic alterations seen in EHE. Whole genome sequencing (WGS) identified links to epigenetic modifying complexes, metabolic processes, and pointed to the importance of the extracellular matrix (ECM) in these tumors. Bulk RNA-Seq demonstrated upregulation of pathways including PI3K-Akt signaling, ECM/ECM receptor interaction, and the Hippo signaling pathway. Development of these extended primary cell cultures allowed for single-cell profiling which demonstrated different cell compartments within the cultures. Furthermore, the cultures served as a therapeutic platform to test the efficacy of TEAD inhibitors in vitro. Overall, the development of primary EHE cell cultures will aid in the mechanistic understanding of this sarcoma and serve as a model system to test new therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4728cebf1ce6a2c3cd2884745cce7881e7bc18e9" target='_blank'>
              EHE cell cultures are a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M Bellizzi, Anthony N. Snow, Patrick Breheny, M. Chimenti, M. Tanas
          </td>
          <td>2025-10-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ad0cbeca9935096e33fe6872d833adddc5508b" target='_blank'>
              Arabidopsis YEATS domain proteins facilitate DNA double strand break repair via homologous recombination pathways
              </a>
            </td>
          <td>
            Neeraja Vegesna, Clara Bourbousse, Y. Jami-Alahmadi, Ana Marie S. Palanca, James A. Wohlschlegel, Julie A. Law
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Abstract INTRODUCTION Faulty DNA repair and epigenetic regulation contribute to neurodegeneration in Alzheimer's disease. Long‐read sequencing enables analysis of “dark regions” that are difficult to study via traditional sequencing methodologies. METHODS Using nanopore whole‐genome DNA sequencing of post mortem brain from early‐ and late‐ stage Alzheimer's disease cases and controls, we analyzed retrotransposition, non‐allelic homologous recombination (NAHR), structural variation, and DNA methylation within repetitive regions. RESULTS Retrotransposon insertions and NAHR were enriched in centromeric/pericentromeric and ribosomal DNA (rDNA) in the aged brain. Putatively somatic AluY retrotransposon insertions trended upward in late‐stage disease. Enrichment of NAHR between repetitive regions and DNA demethylation were detected in early disease. Differential methylation of dark regions within specific Alzheimer's disease risk genes and repetitive elements occurred across disease stage. DISCUSSION This study provides the first long‐read analysis of repetitive elements in the aged human brain and identifies these regions as hotspots for genomic variation in Alzheimer's disease. Highlights Long‐read sequencing enables analysis of “dark” regions Alzheimer's disease brains Somatic AluY retrotransposon insertions may be elevated in late‐stage disease Retrotransposon‐associated non‐allelic homologous recombination (NAHR) is enriched in repetitive regions in early disease DNA demethylation within the centro/pericentromere and ribosomal DNA (rDNA) occur across disease stage Dark regions of risk genes are differentially methylated across disease stage">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f2a36cf2db9b5da1ca7fa6928e8ec70338fa2ed" target='_blank'>
              Long‐read sequencing reveals genomic and epigenomic variation in the dark genome of human Alzheimer's disease
              </a>
            </td>
          <td>
            Paulino Ramirez, Wenyan Sun, Shiva Kazempour Dehkordi, Habil Zare, G. Pascarella, P. Carninci, Bernard Fongang, Kevin F. Bieniek, Bess Frost
          </td>
          <td>2025-11-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="G-quadruplexes have been identified as a promising anti-cancer target because of their ability to modulate the stability of mRNAs encoding oncogenes, tumor suppressor genes, and other potential therapeutic targets. Deregulation of DNA damage and Unfolded Protein Response pathways in cancer cells may create vulnerabilities that can be exploited therapeutically. Previous studies have shown variations in the relative expression of DDR and UPR components in canine lymphoma and leukemia cell lines CLBL-1, CLB70, and GL-1. In the present study, we report the presence of G-quadruplex structures in these canine cell lines. Downregulation of the expression of DDR and UPR components at the mRNA level was observed in the CLBL-1 and CLB70 cell lines after stabilization of G4 structures using the ligand PhenDC3. In contrast, in GL-1 cells, important components of the DDR pathway, such as PARP1, GADD45A, and PIK3CB were upregulated in response to PhenDC3 treatment. Downregulation of DDIT4 mRNA expression, which encodes an important UPR component, was detected in the CLBL-1 and GL-1 cell lines after PhenDC3 exposure. These results suggest that G4 structures can be used to manipulate the expression of potential targets to treat lymphoma in dogs. A substantial enrichment of DNA replication and pyrimidine metabolism pathways was found in the GL-1 cell line after G4 stabilization. This finding suggests that PhenDC3 may induce DNA replication stress in this cell line. Collectively, these results support the feasibility of employing canine cancer cells as a model system to investigate the role of G-quadruplex structures in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266b5cfaedc49c459cf184dd7b4aeb1ea558a388" target='_blank'>
              Stabilization of G-Quadruplexes Modulates the Expression of DNA Damage and Unfolded Protein Response Genes in Canine Lymphoma/Leukemia Cells
              </a>
            </td>
          <td>
            Beatriz Hernández-Suárez, David A. Gillespie, Ewa Dejnaka, B. Obmińska-Mrukowicz, A. Pawlak
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) embryos exhibit widespread epigenetic defects, particularly aberrant DNA methylation. DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development, yet its role in SCNT embryos remains largely unknown. Here, the genome-wide 5hmC landscapes in mouse SCNT embryos are systematically profiled with parental allele specificity. It is revealed that both maternal and paternal genomes of donor somatic cells acquire a transient, sperm-like but attenuated and allele symmetric distribution of 5hmC at the 2-cell stage, distinct from the parental asymmetric pattern observed in naturally fertilized eggs. This is characterized by insufficient DNA hydroxymethylation of the X chromosome in female SCNT embryos, as well as resistance to 5hmC-associated DNA demethylation at germline imprinting control regions (gICRs). While de novo 5hmC generation is closely associated with initial DNA demethylation during somatic-to-zygotic transition, it later becomes uncoupled from ongoing methylation changes. Importantly, global elevation of 5hmC via Tet3 overexpression leads to premature activation of developmental genes at the 2-cell stage and severely impairs SCNT embryo development. These findings reveal unique dynamics and functional consequences of abnormal 5hmC remodeling in SCNT embryos, highlighting the precise regulation of 5hmC generation as a key epigenetic event for successful mammalian cloning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e7a45e185e638b752c93b5629a29c7ec2760ec" target='_blank'>
              5-Hydroxymethylcytosine Dynamics Reveals Coordinated Reprogramming of Parental Genomes and X Chromosome Dosage Balance in Mouse SCNT Embryos.
              </a>
            </td>
          <td>
            Zeming Xiang, Rui Yan, Jing Guo, Mengyao Wang, Xin Cheng, Fan Zhang, Tianzi Guo, Xin Long, Fan Guo, Dan Liang
          </td>
          <td>2025-11-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3160e385fe96505bcb09a45df54fdae63937a125" target='_blank'>
              NONO as a Sensor of Intracellular Oxidation: Relevance to Neuroblastoma Cell Death
              </a>
            </td>
          <td>
            S. S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal structural changes happen when genomic stability compromised, such as in disease, or in species hybrids. In these contexts, diminished control of repetitive elements have been reported, but the reasons for this are not yet well understood. There are causal associations between repetitive elements and phenotypes such as disease progression, leading us to the hypothesis that chromosomal structure may be affected by transposable elements (TEs). In an intraspecific hybrid Drosophila melanogaster cell line (PnM), the degree of pairing among trans homologous chromosomes was affected by the presence of nearby TEs; in particular, LINE and LTR elements, such as Baggins1 or Gypsy. Chromosomal pairing was significantly lower in windows containing TEs than in windows without any TEs. Pairing was also affected by TEs in mouse, which suggests a possible general association between TEs and pairing that is highly conserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b05bf5cfcd82a8fc82e293a52e5e86b57f277d" target='_blank'>
              Haplotype-resolved Chromatin Conformation Data reveals relationship between Transposable Elements and Chromosomal Pairing.
              </a>
            </td>
          <td>
            Luke K Genutis, Patrick Villanueva, Rita M. Graze, Jumana Alhaj Abed, Lauren M McIntyre, S. Nuzhdin
          </td>
          <td>2025-11-26</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="BRCA1/2 are crucial in the homologous recombination repair (HRR) pathway, with loss-of-function (LoF) alterations predicting sensitivity to PARP-inhibitors (PARPi). Whether other HRR-gene alterations confer PARPi sensitivity remains unclear. In the Drug Rediscovery Protocol, patients receive off-label drugs matched to their tumor molecular profile. Here, olaparib efficacy and safety were evaluated in adult patients with treatment-refractory, progressive malignancies harboring LoF alterations in ATM (cohort A) or other HRR-genes including CDK12, PPP2R2A, CHEK1/2, and RAD51B (cohort B). Primary endpoints were clinical benefit (CB: confirmed objective response or stable disease ≥16 weeks) and safety. Pre-treatment biopsies were analyzed by whole-genome sequencing (WGS) for target validation. CB was observed in 8/25 patients (32%) in cohort A (prostate cancer: n = 6, adenoid cystic carcinoma: n = 1, endometrial cancer: n = 1). No effectiveness was seen in patients with colorectal cancer (n = 8). Median progression-free survival (PFS) and overall survival (OS) were 3.4 months (95% CI 1.8-5.3) and 9.2 months (95% CI 5.2-21.3), respectively. In cohort B, the CB rate was 41.7% (10/24) with median PFS and OS of 3.5 months (95% CI 3.4-6.6) and 8.1 months (95% CI 6.6-14.2), respectively. CB was observed in CKD12 (n = 7), RAD51B (n = 2), and CHEK2-altered tumors (n = 1), but not in PPP2R2A (n = 6) or CHEK1-altered tumors (n = 1). No unexpected toxicities occurred. WGS confirmed inclusion target in 84% of tested patients. In conclusion, PARPi sensitivity varies across HRR-genes, indicating that relying solely on an altered common mechanistic pathway is insufficient to predict response. Future studies should target specific HRR-genes to assess subgroup-specific benefits and determine proper use of molecular diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/307d50a92ba3f268facbb2ba41b55d6d57fef8c8" target='_blank'>
              Olaparib for patients with tumors harboring alterations in homologous recombination repair genes: Results from the drug rediscovery protocol.
              </a>
            </td>
          <td>
            I. A. C. Spiekman, N. Mehra, L. Zeverijn, B. Geurts, K. Verkerk, S. H. Haj Mohammad, V. van der Noort, P. Roepman, W. D. de Leng, A. Jansen, A. V. D. van de Luijtgaarden, T. van Voorthuizen, T. Buffart, H. Gelderblom, E. Voest, Henk M. Verheul
          </td>
          <td>2025-10-16</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) gaps are a hallmark of BRCA-deficient cells, yet the mechanisms that safeguard these lesions remain unclear. Through a genome-wide CRISPR screen, we identified the RAD9A-HUS1-RAD1 (9-1-1) complex as essential for the survival of BRCA2-deficient cells through an ATR-independent mechanism. Loss of 9-1-1 in this context leads to the accumulation of PRIMPOL-dependent gaps that fail to undergo post-replicat ive repair, resulting in pathological expansion and increased DNA damage. This instability is driven by excessive EXO1-mediated degradation, as EXO1 depletion rescues the phenotype. We further demonstrate that the 9-1-1 complex is required for POLζ-dependent gap filling. We propose a model in which ssDNA gaps, when extended beyond a critical length, become inaccessible to TLS-mediated repair and are fully reliant on homologous recombination. These findings establish the 9-1-1 complex as key regulator of ssDNA gap stability and a promising therapeutic target in BRCA2-deficient cancers. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8631064e162ad6c938aaf5491d07fd82b069ed0" target='_blank'>
              The 9-1-1 complex protects ssDNA gaps in BRCA2-deficient cancer
              </a>
            </td>
          <td>
            Helen E. Grimsley, Katherine Courtemanche, Shane Cox, Niamh McDermott, Aman Sharma, Jackson Bright, J. Setton, Mehmet Orman, Simon N. Powell
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular senescence is a hallmark of cancer and induces senescence-associated inflammatory responses in the tumor microenvironment (TME). CDKN1A, an important cellular senescence marker, plays significant roles in cell proliferation, invasion, migration, and apoptosis. Previous studies have implied its drug resistance role in certain cancer types. However, its impact on immunotherapy efficacy in advanced LUAD remains unclear. Using TCGA and SU2C-MARK cohorts, we investigated CDKN1A’s biological features in advanced LUAD, analyzing pathway regulation, immune infiltration, and immunotherapy associations. Single-cell RNA sequencing (GSE148071) validated TME and cellular communication differences between CDKN1A high/low expressing tumors in advanced LUAD. CDKN1A expression was positively associated with immunosuppressive environment including extracellular matrix, cancer-associated fibroblasts (CAFs) and myeloid-derived suppressor cells (MDSCs) in advanced LUAD for both TCGA and SU2C-MARK cohorts (P < 0.05). CDKN1A showed significantly positive correlations with many senescence genes in advanced LUAD (P < 0.05), which were also positively associated with endothelial cells, epithelial cells, fibroblast cells and macrophages, but negatively associated with immune cells (P < 0.05). In multivariable cox regression, patients with high CDKN1A expression had worse OS (HR = 2.74, 95% CI = 1.31–5.73, P = 0.007) and PFS (HR = 1.78, 95% CI = 1.01–3.11, P = 0.045) than those with low CDKN1A expression when treated with immunotherapy. In contrast, the high CDKN1A expression was not associated with PFS and OS in the TCGA cohort, in which the LUAD patients received standard chemotherapy, suggesting the immunotherapy predictive role instead prognostic role of CDKN1A. Moreover, single-cell analysis revealed that CDKN1A highly expressed tumors were accompanied by an enrichment of stromal and endothelial cells within the tumor microenvironment, along with enhanced activity of SMAD3/4, the downstream transcription factors of TGFB signaling. These tumors exhibited increased cell–cell communication with stromal cells (COL1A1/COL1A2-ITGA1/ITGB1/SDC1) and endothelial cells (NAMPT-ITGAS/ITGB1/INSR). CDKN1A expression was associated with cellular senescence, immunosuppressive environment and exhibited resistance to immunotherapy in advanced LUAD, suggesting a potential combination strategy with senolytic or senomorphic therapies to overcome immunotherapeutic resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f34f805e0422258abbde7dbe983d9c6f010ed5" target='_blank'>
              CDKN1A and cellular senescence are associated with immune resistance in advanced lung adenocarcinoma
              </a>
            </td>
          <td>
            Wang Shen, Feidie Duan, Shuiting Fu, Peng Cui, Yezhen Shi, Lei Zhang, Xiao Mi, Yu Xu, Guoqiang Wang, S. Cai, Xuebing Li, Xiaojun Tang
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c234ce326d72e3bdffc7fdf7789f2aeb60a1a1" target='_blank'>
              Inactivation of a protein hydroxylase complex impairs replication fork restart in cancer and neurodevelopmental disorders
              </a>
            </td>
          <td>
            Tristan J. Kennedy, Sally C. Fletcher, Uncaar Boora, Erin Fraser-Williams, Gareth W. Hughes, Eline Hendrix, Joanna R Morris, Stephen J Smerdon, M. Coleman
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe180fb4ec176d493964db2a8374c986c41b6b32" target='_blank'>
              Spatial Proximity Sequencing Maps Developmental Dynamics in the Germinal Center
              </a>
            </td>
          <td>
            Huili Wang, Junjie Xia, Peer Mohamed Suhail Mujibur Rahman, Bijentimala Keisham, Abinash Padhi, Yiyu Deng, Yan Li, Luke Vistain, Sarah Kim, Gabriel Mercado-Vásquez, Aly A. Khan, Marcus R. Clark, Savaş Tay
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d0438573ac5783b09f8434fbbdee10888d90b8" target='_blank'>
              ETV5 transcriptionally activates LGR4 and promotes cancer cell invasion and migration of nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Lei Zeng, Qiwei Luo, Nan Liu, Aochen Shi, LuLu Liu, Lei Tao, KaiBin Zhu, Yan Liu, Xianming He, Jianze Zhang, Zequn Huang, Su Deng, Jiayin Wu, Ping Zhang, Yi Sang
          </td>
          <td>2025-10-29</td>
          <td>Journal of Molecular Histology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Genotoxic therapy resistance in glioblastoma (GBM) is mediated by rapid DNA repair. We hypothesized that non-malignant cells in the heterogenous GBM microenvironment assist rapid DNA repair in neoplastic cells. Supporting this hypothesis, intracranial GBM tumors (orthotopic GBM PDXs and syngeneic murine GBMs) in mice had rapid DNA repair and poorer survival after radiation compared to flank GBM tumors of identical genotype. Gamma-H2AX immunofluorescence analysis of irradiated intracranial tumors revealed spatial heterogeneity in resolution of DNA damage. Coupling these analyses with spatial transcriptomics in irradiated intracranial syngeneic murine GBM tissues suggested that regions with fast repair were rich in microglia. Coculture of patient derived GBM cells with human primary microglia ex vivo enhanced DNA repair post irradiation in GBM cells. This phenotype was independent of cell contact, suggesting a microglia-derived soluble factor. Indeed, mass spectrometry analysis revealed that microglia secrete acetylated amino acids that are then consumed by neoplastic cells. Supplementation of acetylated amino acids hastened the repair of radiation-induced DNA damage and promoted an invasive phenotype. Metabolomic analysis of GBM cells supplemented with acetylated amino acids showed that glioma cells re-route glucose utilization towards nucleotide synthesis. Spatial transcriptomic analysis of human GBM tissue explant treated with radiation and lung cancer metastasis to the brain resected after irradiation, showed higher expression of genes essential for synthesis of the acetylated amino acids in myeloid cells. Furthermore, supplementation of acetylated amino acids increased acetylation of H2AX post radiation in GBM cells. Hence, we show that microglia in the GBM microenvironment secrete acetylated amino acids which are consumed by GBM cells and used to repair radiation induced DNA damage by repurposing glucose for nucleotide synthesis and increase H2AX acetylation, thereby conferring treatment resistance to the GBM cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef28425f01748e3f178727e96499580896a377fc" target='_blank'>
              TMIC-85. Microenvironment-derived acetylated amino acids promote glioblastoma DNA repair by rerouting glucose fates and enhancing histone acetylation
              </a>
            </td>
          <td>
            S. Palavalasa, Jack Freeman, Aditri Gokul, Ava Singer, A. Scott, K. Wilder-Romans, Emily Baker, N. Korimerla, A. O'Brien, A. Lin, Erik R Peterson, Ningning Liang, Jie Xu, V. Ravikumar, P. Sajjakulnukit, Harrison Wong, A. Andren, Li Zhang, Sunita Shankar, Sagnik Bhadury, J. Heth, Weihua Zhou, Arvind Rao, D. Nagrath, W. Al-Holou, C. Lyssiotis, D. Wahl
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>