<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-08 09:36:53 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>43</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>89</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>123</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>386</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>337</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres and segregate stochastically during cell division3–6 yet persist over many generations. EcDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei over 40 years ago with unknown mechanism3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. We develop Retain-seq, a genome-scale assay that reveals thousands of human retention elements conferring generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromatin conformation capture show that retention elements physically interact with mitotic chromosomes at regions which are mitotically bookmarked by transcription factors and chromatin proteins, intermolecularly recapitulating promoter-enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated; targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, suggesting that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, revealing the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, Ivy Tsz-Lo Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f24f51bf1d13b62d74eb0b4db2b03297c30b25d" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells.
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, , Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Oncogenesis involves widespread genetic and epigenetic alterations, yet the full spectrum of somatic variation genome-wide remains unresolved. We generated a near-telomere-to-telomere (T2T) diploid assembly of a donor paired with deep short- and long-read sequencing of their melanoma. This revealed that 16% of somatic variants occur in sequences absent from GRCh38, with satellite repeats acting as hotspots for UV-induced damage due to sequence-intrinsic mutability and inefficient repair. Centromere kinetochore domains emerged as focal sites of structural, genetic, and epigenetic variation, leading to remodeling of centromere kinetochore binding domains during tumor evolution. Single-molecule telomere reconstructions uncovered cycles of attrition, deletion, and telomerase-mediated extension that shape cancer telomeres. Finally, diploid chromatin maps exposed that copy number alterations and epimutations, rather than point mutations, predominate in rewiring cancer regulatory programs. These findings define the full landscape of a cancer’s somatic variation and their functional impact, establishing a blueprint for T2T studies of mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, S. Wg, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>35</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Understanding of the heterogeneity in the formation of the Ph has evolved as technology has advanced. In the pre-TKI era, FISH revealed derivative 9 deletions that were associated with shorter survival. The poor outcomes were overcome with imatinib. Whole exome and transcriptome sequencing revealed greater complexity: inversions; novel 9;22 fusions; sequence fragmentation; and random reassembly. These were termed Ph-associated rearrangements. They occurred on the Ph and/or the derivative 9, were thought to occur at the same time as the Ph and correlated with poorer response to imatinib.



 To use whole genome sequencing (WGS) to examine derivative chromosomes and further resolve the complexity of Ph-associated rearrangements in patients at diagnosis and pan-TKI resistant cell lines.



 WGS was performed for 7 chronic phase CML patients at diagnosis and 7 matched sensitive/resistant Ph-positive cell lines. Resistance was induced by exposure to increasing concentrations of imatinib. Data was analyzed using Oncoanalyzer. Derivative chromosome structure was resolved using the LINX algorithm, which clusters structural variants that are highly unlikely to occur independently. Rearrangements were validated using orthogonal methods, including Optical Genome Mapping and SNP Array. Six of 7 patients had pre-characterized Ph-associated fusions identified using RNAseq.



 For most patients, a balanced reciprocal Ph translocation involves 2 rearrangements: 1 that generates the Ph and BCR::ABL1; and 1 reciprocal derivative 9 with ABL1::BCR. These structures were resolved for 1 patient without pre-characterized Ph-associated rearrangements and for 5 of 7 parental cell lines.
 A distinct class of complex structural rearrangement called chromoplexy was identified involving the Ph translocation in 11 samples. These Ph-associated rearrangements had the highest number of linked structural variants within the individual genomes of 10 of the 11 samples. Chromoplexy involves the formation of extensive chained rearrangements affecting multiple chromosomes that create an intricate series of inter/intra-chromosomal fusions, deletions, duplications and inversions. Many DNA strands were broken and ligated in new configurations in the affected samples. Multiple genes on the derivative chromosomes were disrupted, including recurrent cancer drivers.
 For 3 of 7 matched sensitive/resistant cell lines, Ph-associated rearrangements were acquired in the resistant lines. The most complex events occurred in the LAMA84 resistant line. The parallel sensitive line had the standard 2 rearrangements. However, the resistant line gained 17 rearrangements with highly amplified Ph sequences, inversions and novel genomic fusions. Remarkably, small fragments from 6 chromosomes (2, 4, 10, 13, 17, X) were inserted upstream and downstream of BCR::ABL1, and disrupted the NUP214 driver gene.
 The K562 sensitive line displayed 24 rearrangements with non-contiguous amplified Ph regions of different copy number. Events included NOTCH1 deletion, 9;22 NUP214 fusion and an adjacent 9;13 fusion. Six fragments from chromosome 13 were incorporated into the Ph. The K562 resistant line displayed further evolution with 31 Ph rearrangements.
 The AR230 sensitive line displayed Ph chromoplexy with 9 rearrangements. The resistant line evolved with 19 additional Ph rearrangements. A fragment from chromosome 21 was inserted upstream of BCR::ABL1 and disrupted the MAPK1 driver gene. The LZTR1 tumor suppressor was deleted. All other cell lines maintained the standard Ph configuration.
 Ph-associated chromoplexy was evident in 6 patient samples: average 10 rearrangements. The patient with the highest number (15) progressed to blast phase at 4 months of imatinib. Four Ph-associated subclones were detected at diagnosis with non-contiguous amplified regions, consistent with the evolved cell lines. Furthermore, the LZTR1 tumor suppressor gene was deleted, as detected in AR230. The blast phase sample was unavailable. Deep molecular response was achieved in 1 patient on imatinib and 4 on frontline or switch to nilotinib/dasatinib.



 We have demonstrated that Ph chromoplectic rearrangement can be present at diagnosis but can also be initiated during leukemic progression and disrupt multiple genes, including cancer drivers. The Ph may be prone to extensive DNA breakage. Ph chromoplexy could constitute a new mechanism that contributes to acquired TKI resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed250c9c3a664955ee35598f9870a47e6c84f2b" target='_blank'>
              Whole genome sequencing reveals that the Philadelphia chromosome (Ph) can evolve and gain complex chromoplectic sequence rearrangements associated with tyrosine kinase inhibitor (TKI) resistance
              </a>
            </td>
          <td>
            S. Branford, C. Wadham, N. Shahrin, V. Saunders, Rosalie R. Kenyon, Ming Lin, Dominic Kaczorowski, Muneeza Maqsood, Hamish Scott, John Toubia, N. Shanmuganathan, C. Kok, Timothy Hughes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aneuploidy, an abnormal number of chromosomes, is a hallmark of cancer and has been proposed as an initiating event in tumorigenesis. In glioblastoma (GBM), a highly aggressive brain tumor, cells almost universally display gain of chromosome 7 and loss of chromosome 10. However, it remains unclear whether these alterations arise de novo during malignant transformation or reflect pre-existing chromosomal instability in normal brain tissue. Here, we used single-nucleus whole-genome sequencing (snWGS) on 225 NeuN-negative (non-neuronal) cortical nuclei from 12 healthy individuals and 6 GBM patients, including matched tumor cores and non-tumor brain regions. In healthy brains, approximately 15% of glial nuclei harbored somatic aneuploidies, most often involving chromosome arms, with recurrent 16p alterations detected in up to 3% of nuclei from both healthy controls and GBM non-tumor tissue. These findings establish 16p is a hotspot of structural variation in adult glia. Non-tumor regions in GBM patients closely resembled healthy controls in aneuploidy burden and chromosomal instability metrics and lacked hallmark tumor alterations. In contrast, GBM tumors exhibited significantly elevated aneuploidy (~50%), enrichment for canonical chromosomal instability-driven events, and sex-specific karyotype patterns, consistent with transformation-associated chromosomal instability. Thus, aneuploidy is a recurrent but constrained feature of normal adult glia, whereas chromosome instability and GBM-defining aneuploidies emerge only during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ca098d5f35d207827ccedcf67f15fa391301a1" target='_blank'>
              The Somatic Aneuploidy Landscape of Adult Glia Reveals 16p as a Hotspot and Differentiates Mosaicism in Normal Glia from Chromosomal Instability in Glioblastoma
              </a>
            </td>
          <td>
            Cristina Montagna, Olivia Albert, Shixiang Sun, Jhih-Rong Lin, Moonsook Lee, Chang Chan, Alex Maslov, Lisa Ellerby, Anita Huttner, Zhengdong Zhang, J. Vijg
          </td>
          <td>2025-11-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="APOBEC3 cytidine deaminases drive cancer evolution. There is an unmet need to target cancer cells with APOBEC3 activity. Here, we identify error-prone theta-mediated end joining (TMEJ) as the main pathway for repairing APOBEC3-induced double-strand breaks (DSBs). Using fluorescent DSB repair reporters and a novel biochemical assay, we demonstrate that APOBEC3A competes with replication protein A (RPA) for single-stranded DNA overhangs, exposing microhomologous sequences to shift DSB repair towards error-prone TMEJ. Genomic analysis of clinical tumor samples confirmed the co-occurrence and proximity between APOBEC3-induced mutational footprints, microhomology-mediated deletions (MMDs), and TMEJ-associated chromosomal instability signatures. Crucially, inhibition of DNA polymerase theta (Polθ) synergizes with APOBEC3A-induced DSBs to induce synthetic lethality in vitro and in vivo. Collectively, our findings identify TMEJ as the preferred mechanism for repairing APOBEC3A-induced DSBs and establish Polθ inhibition as a novel promising strategy to eliminate cancer cells with APOBEC3A activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1126d43689098cc7096980496ac4ba1303c669a" target='_blank'>
              APOBEC3A-Induced DNA Damage Drives Polymerase θ Dependency and Synthetic Lethality in Cancer
              </a>
            </td>
          <td>
            Abhishek Bose, Weisi Liu, Paul Yoo, Alina Sami, Michal T. Boniecki, M. Deshpande, Mohamed Osman, Duy D. Nguyen, U. D. Castillo, P. Giannakakou, B. Bhinder, William F Hooper, O. Elemento, N. Robine, Milaid Granadillo Rodríguez, J. Gerhardt, K. Mouw, L. Chelico, B. M. Faltas
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Telomeres are essential for genome integrity, but the accurate, high-resolution mapping of their three-dimensional (3D) chromatin interactions, a process thought to mediate gene regulation and telomere maintenance, has been limited by the repetitive nature of subtelomeric DNA and shortcomings of traditional sequencing-based chromatin conformation capture methodologies. To overcome these challenges, we developed Telomere Conformation Capture by sequencing (Telomere-C), a novel high-throughput approach for the genome-wide identification of telomeric chromatin interactions. We applied Telomere-C to three normal fibroblast cell lines (BJ, IMR90, WI38), one telomerase-positive cancer cell line (HeLa S3) and two cancer cell lines (U2-OS and WI38-VA13) that maintain telomeres by alternative lengthening of telomeres (ALT). We generated millions of telomere-associated reads per sample and identified up to 41,026 interaction peaks. We successfully validated previously-described telomeric interaction sites (e.g., TERT and DUX4), and critically, revealed a novel class of ultra-long-range interactions extending over 5 Mb away from the telomere, which surprisingly constitute over 80% of all identified contacts. Contrary to prior studies focusing on protein coding gene interactions, we discovered that the telomeric 3D interactome is overwhelmingly anchored at repetitive element hubs, particularly interstitial telomeric sequence (ITS), telomere-associated repeat 1 (TAR1) and D20S16 elements. We found that the clustering of these interactions correlates strongly with cell-type-specific telomere maintenance. Most notably, D20S16 telomeric interactions were uniquely and highly enriched in ALT cancer cells, suggesting a mechanistic link. Taken together, our study effectively constructs the first high resolution maps of the telomeric 3D interactome, redefining its scope to be dominated by ultra-long-range contacts with repetitive elements. This work provides fundamental insights into telomeric nuclear organization and establishes the telomeric D20S16 interaction as a molecular signature for the ALT pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Background: Chromothripsis, a catastrophic event of chromosomal shattering and reassembly, is a key driver of oncogenesis and is associated with poor prognosis in multiple myeloma (MM). However, the specific functional consequences and the molecular mechanisms by which it rewires the MM regulatory landscape remained largely unknown.
 Methods: Using previous definitions, we identified chromothripsis (≥10 breakpoints, ≥7 CN oscillations, ≥200 kb segment sizes), chromothripsis-like (5-9 breakpoints and 3-6 CN oscillation) and non-chromothripsis (WT) samples. We utilized multiomic data from the CoMMpass dataset (n=1,066) and an independent Indiana University (IU) dataset (n=134), as well as 14 patient-derived xenograft (PDX) models with short-/long-read whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities and structural variation (SV), as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) to identify 3D chromatin architecture/folding inside the cell and enhancer-target interactions.
 Results: We identified 417 chromothripsis events in 268 patients (25%) from the CoMMpass dataset. Events were identified more frequently in relapsed and refractory (RRMM) samples compared to newly diagnosed (NDMM) samples (30% vs. 22%), suggesting an association between chromothripsis and disease progression. The number of events found in chromosomes highly correlated with chromosome sizes (r=0.63, Pearson correlation). However, events were particularly enriched on chromosomes 1, and 17 (0.26 and 0.30 events per million bps). Breakpoints of these events were enriched on 17q and 1p (odds ratio=2.2 and 1.7, p=0.0003 and 0.001). Chromothriptic samples were associated with inferior progression-free (PFS; 970 vs. 1227 days, p=0.003) and overall survival (2176 vs. not reached, p=0.006) compared to WT. In chromothriptic samples, 34% exhibited breakpoints on >1 chromosome and an higher number of affected chromosomes correlated with worse PFS (≥3 vs. 2, p=0.03; ≥3 vs. 1, p=0.02). Meanwhile, CN oscillations plus breakpoints was an independent factor associated with inferior PFS(p=0.05,: 700 vs. 1358 days). Del(TENT5C), del(TP53)and t(4;14) were enriched in the chromothripsis group (32%, 18% and 20%, odds ratio=2.6, 3.4 and 2.4, p<0.001) and are associated with genomic instability. We applied the same approach and further identified 19 chromothriptic samples from the IU dataset, where t(4;14) and del(TP53) were also enriched (odds ratio=2.6 and inf.) We identified differentially expressed genes (DEGs) between the chromothripsis group and a subset of WT group with balanced frequencies of high-risk events (e.g. t(4;14) & del(TP53)) to generate a chromothripsis signature. Pathway analysis on 37 consistent DEGs between the two datasets identified ‘G2/M checkpoint’ and ‘interferon α/γ responses‘(p<0.05), suggesting selection for a growth advantage and activation of the cGAS/STING pathway. A neuron network model for predicting chromothripsis was trained with DEGs and reached performance of AUC=0.90 and AUPR=0.87.
 We utilized multi-omics data from established PDX models to further examine the functional consequences of chromothripsis events. Among 9 events detected in 7 samples, 5 events exhibited significantly enriched APOBEC mutational signature and kataegis within events. Copy number change alone did not account for changes in gene expression within the chromothripsis regions, with 71% of genes dysregulated through neo-TAD formation and rewiring of the epigenome, specifically through juxtaposition of super-enhancers and allele-specific expression associated with DNA methylation changes. Key dysregulated genes included MYCN (Log2FC=6.3), the tumor suppressor PMAIP1 (Log2FC=-4.9), and cell adhesion proteins (EPCAM/CD326, Log2FC=3.5) that were rarely expressed in other MM samples.
 Conclusion: We revealed novel molecular characteristics of chromothripsis and the mechanisms caused it. We determined that the number of events and CN abnormalities are associated with poor outcome and that epigenetic states are reprogrammed within the events. These findings advance the clinical risk stratification and reveal unique therapeutic vulnerabilities tied to chromothripsis-associated epigenetic dysregulation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eeb625eee780834dfe72e26b192d78e2662c5ec" target='_blank'>
              Chromothripsis drives high-risk multiple myeloma by copy number abnormalities and rewiring the epigenome to activate oncogenic programs.
              </a>
            </td>
          <td>
            Enze Liu, Nathan Becker, Sreejeta Mondal, A. Suvannasankha, D. Coffey, B. Diamond, Abhishek Pandey, Kelvin Lee, D. Kazandjian, R. Abonour, Aneta Mikulasova, G. Morgan, O. Landgren, Brian Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Non-canonical (non-B) DNA secondary structures, e.g., G-quadruplexes and triplex DNA, are mutation hotspots and genome regulators contributing to disease and evolution. Yet they remain uncharacterized in complete genomes in vivo. Here we exploited the fact that many non-B DNA structures form single-stranded DNA (ssDNA). Using permanganate/S1 footprinting across 14 cell lines, we generated ssDNA profiles for human and six non-human ape telomere-to-telomere (T2T) genomes. Newly resolved satellite arrays—e.g., at ribosomal DNA and centromeres—displayed high ssDNA levels, implicating non-B DNA in satellite expansion and function. Hidden Markov Models applied to our ssDNA data revealed active genomic domains with specific functions—e.g., replication, transcription, or recombination—each enriched in particular non-B DNA types. Human-specific ssDNA domains correlated with nervous system genes, whereas cancer and embryonic cells showed increased ssDNA in transposable elements. Our ssDNA analysis across ape T2T genomes uncovered conserved and species-specific DNA structural dynamics central to genome regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2c0bd20d360ee798769e48db594be4df9b6168" target='_blank'>
              Comparative analysis of single-stranded and non-canonical DNA formation in human and other ape cells with telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Jacob P. Sieg, Huiqing Zeng, Hana Pálová, Linnéa Smeds, S. Mohanty, Angelika Lahnsteiner, Francesca Chiaromonte, Kateryna D. Makova
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah Burget, Tobias Friedrich, So-Yoon Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circulating cell-free DNA (cfDNA) assays are being widely adopted in oncology and maternal-fetal medicine. Patterns of cfDNA fragmentation can provide useful information about gene regulation and expression in human disease from a blood draw. Here, we demonstrate that enhancer RNA expression – a marker of enhancer activity – can be inferred from local patterns of cfDNA fragmentation. We define a transcriptional activation score (TAS) that predicts expression of enhancers and genes based on cfDNA fragment sizes and positions near transcriptional start sites (TSSs). The TAS identifies activity of cancer-associated enhancers in patients with cancer, distinguishes clinically relevant cancer subtypes, and identifies activation of enhancers associated with treatment resistance and therapy response. We propose a simple model to account for our findings based on chromatin fiber structure and the depletion of H1 histone proteins near active TSSs. Our model provides a unified framework that reconciles seemingly conflicting observations from prior fragmentomics studies. Broadly, this work enables blood-based assessments of gene regulation in cancer and non-oncologic diseases to inform pathobiology, diagnosis, and treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caacf7712f535881a1313780e0d16e1ccea5aabe" target='_blank'>
              A chromatin fiber model explains cell-free DNA fragmentation signatures of active regulatory elements
              </a>
            </td>
          <td>
            Alexis Yang, G. Lee, Surya B. Chhetri, Razane El Hajj Chehade, Gunsagar S. Gulati, Medha Pandey, Doris Fu, Yoo-Na Kim, Shahabeddin Sotudian, H. Savignano, Sadia D. Shahbazi, Chelsea Philpot, Taylor A. Hoggood, D. Gulhan, M. Kellis, Jacob E Berchuck, S. Baca
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Approximately half of high-grade serous ovarian carcinomas (HGSOCs) demonstrate homologous recombination deficiency (HRD) with characteristic genomic rearrangements. However, the impact of HRD on centromeres and transposable elements remains largely unexplored in HGSOC since conventional short-read sequencing is unable to interrogate these repetitive regions. We employed Oxford Nanopore long-read sequencing (LRS) to investigate genomic and epigenetic alterations in these regions. Pre-treatment archival cryopreserved tumor and matched blood samples were obtained for six patients with HGSOC. High-molecular-weight DNA was sequenced using Oxford Nanopore R10.4 flow cells and aligned to both the GRCh38 and the telomere-to-telomere T2T-CHM13 reference genome. Pathogenic gene mutations, allele-specific copy number variations, structural variants, and CpG methylation were analyzed. All six tumors had pathogenic TP53 mutations. Two carried germline BRCA1 mutations, while three showed CCNE1 amplifications. HRD scores and mutational signatures associated with HRD were elevated in the BRCA1-mutated tumors. Centromeric regions were significantly hypomethylated in tumors and their methylation profiles distinctly separated HRD tumors from non-HRD tumors. LINE1 and ERV transposable elements showed marked hypomethylation in tumors without germline BRCA1 mutations. Chromosome arm-specific telomere lengths were significantly shortened in tumors. Allele-specific hypermethylation in the TERT hypermethylated oncological region was detected in three tumors. LRS uncovered HRD-related genomic and epigenomic alterations in previously inaccessible repetitive regions of HGSOC, including centromeric and transposable element hypomethylation. These findings highlight the potential of such abnormalities as novel biomarkers for HGSOC and warrant further application of the methods to larger cohorts in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a227daba2577321ec904774ceb23e3c2f8c0883" target='_blank'>
              Long read sequencing reveals novel genomic and epigenomic alterations in repetitive regions of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Y Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, Barrett Lawson, R. Hillman
          </td>
          <td>2025-10-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Alternative DNA conformation formed by sequences called flipons are thought to play an important role in regulating various genomic processes, either repressing or enhancing transcription, chromatin organization, DNA repair, telomere maintenance, RNA splicing, translation, and stress responses. The formation of G-quadruplexes (GQs) has been investigated experimentally using various methodologies with varying degrees of overlap between the results underscoring the need for a gold-standard GQ dataset. With this aim we trained a large language model, GQ-DNABERT using EndoQuad, the most comprehensive human GQ dataset. GQ-DNABERT recalled the training data and predicted de novo GQs in intergenic and intronic regions, enriched for cis-regulatory elements (cCREs) and ATAC-seq peaks. We evaluated the predicted GQ-DNABERT proximal enhancer–promoter (pEP) pairs, using annotations from ENdb, ENCODE, Zoonomia, Chromium multiomics scATAC-seq and scRNA-seq data from normal cells, and cCREs from normal-cancer pairs. We found GQ pEP pairs correlating with gene expression, with some pairings potentially acting as tissue-specific switches. Genes with GQ pEP pairs in cancer cells are enriched in different processes compared to the corresponding normal tissues. Overall, GQ-DNABERT is a valuable tool for extending and harmonizing data collected ex vivo. We demonstrate the usefulness of GQ-DNABERT for investigating transcriptional regulation in single-cell experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57f048fcb6a847ca70df111a471d2de991b3cd9c" target='_blank'>
              GQ-DNABERT reveals GQ proximal enhancer–promoter interactions associated with tissue-specific transcription
              </a>
            </td>
          <td>
            Dmitry Konovalov, Dmitry Umerenkov, Alan Herbert, M. Poptsova
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Programmed DNA elimination (PDE) is a notable exception to genome integrity, characterized by significant DNA loss during development. In many nematodes, PDE is initiated by DNA double-strand breaks (DSBs), which lead to chromosome fragmentation and subsequent DNA loss. However, the mechanism of nematode DNA breakage remains largely unclear. Interestingly, in the human and pig parasitic nematode Ascaris, no conserved motif or sequence structures are present at chromosomal breakage regions (CBRs), suggesting the recognition of CBRs may be sequence-independent. Using Hi-C, we revealed that Ascaris CBRs engage in three-dimensional (3D) interactions before PDE, indicating that physical contacts of the break regions may contribute to the PDE process. The 3D interactions are established in both Ascaris male and female germlines, demonstrating inherent genome organization associated with the CBRs and to-be-eliminated sequences. In contrast, in the unichromosomal horse parasite Parascaris univalens, transient and pairwise interactions between two neighboring CBRs that would form the ends of future somatic chromosomes were observed only during PDE. Intriguingly, we found specific spatial compartmentalization changes in the Ascaris chromosomes after PDE converts 24 germline chromosomes into 36 somatic ones. Remarkably, Parascaris PDE generates the same set of 36 somatic chromosomes, and the compartment changes following PDE are consistent between the two species. Overall, our findings contribute to a model that the 3D genome facilitates the recognition of sequence-independent break sites for Ascaris PDE. It also suggests an evolutionary and developmentally conserved 3D genome reorganization of nematode somatic chromosomes following PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Background and Aims: Genomic instability, a prominent feature of cancer and precancerous conditions, leads to constant acquisition of genomic changes some of which enable these cells to progress through worsening stages of disease, overcome immune surveillance and survive through therapeutic interventions. The purpose of this study was to identify and validate genes and pathways driving genomic evolution in human cancer, with special focus on myeloma (MM).
 Methods: Gene expression and copy number data of ten human cancers (MM, hepatocellular carcinoma, hormone positive breast cancer, triple negative breast cancer and adenocarcinomas of esophagus, stomach, pancreas, lung, prostate and ovary) were used in a stepwise manner in each cancer. First, the extent of genomic instability in each patient was assessed by calculating total copy number events and then integrated with expression data to identify genes whose expression correlated with genomic instability. Subsequently the genes in this signature were investigated for correlation with survival in different MM datasets. Polo-like kinase 4, a member of this signature, was investigated in loss and gain of function studies for its impact on growth and chromosomal integrity.
 Results: Expression of 32 genes correlated with genomic instability in ten human cancers including MM. Functional enrichment analysis demonstrated that microtubule motor, RNA/DNA hybrid ribonuclease, flap endonuclease and 5‘-3‘ exonuclease activities were prominent molecular functions whereas spindle organization, chromosome segregation and nuclear division were major biological processes associated with these genes. Expression of 11 out of 32 genes associated with poor OS and/or EFS in three different MM datasets [MMRF, IFM70 (n=170) and GSE24080 (n=559)]. Of these 11 genes, polo-like kinase 4 (PLK4), with the highest relative expression in MMRF dataset was investigated further. Evaluation by Western blotting confirmed that expression of PLK4 was low in PBMC (n=2) whereas markedly elevated in all 10 MM cell lines tested. PLK4 knockdown significantly inhibited whereas its overexpression increased DNA breaks and genomic instability (as assessed from micronucleus assay) as well as growth rate of MM cells. Evaluation of cell cycle in PLK4-overexpressing MM as well as non-cancerous (HS5) cells identified a smaller third peak after G2 indicating a sub population of cells with DNA content > 4N. Consistently, the karyotypic evaluation of these MM cells demonstrated that after 8 weeks in culture, control cells had only 1 out of 70 (1.4%) abnormal metaphases, whereas 8 out of 70 (11.4%) metaphases in PLK4-overexpressing cells were abnormal. Abnormalities in PLK4-overexpressing cells included fused, broken and polyploid chromosomes. In non-cancerous immortal (HS5) cells, 5 out of 70 (7%) abnormal metaphases were observed in control cells whereas 15 out of 70 (21.4%) were abnormal in PLK4-overexpressing cells. Abnormalities included fused, broken, triradial and polyploid chromosomes. Evaluation by fluorescence in situ hybridization (FISH) demonstrated c-myc amplification (with 3 to > 5 copies) in 30/500 (6%) control cells whereas 56/500 (11.2%) PLK4-overexpressing cells. Similar observations were made when a centromere probe was used to investigate these samples by FISH, indicating an increase in intrachromosomal instability by PLK4. A specific PLK4 inhibitor caused inhibition of cell viability in all 4 MM cell lines tested (with IC50 ranging from 11 nM – 71 nM) with a minimal impact observed in 2 normal PBMC samples (IC50 > 50 M) indicating a high therapeutic index. Furthermore, we observed that PLK4 regulates expression and/or phosphorylation of PBK which in turn phosphorylates FOXM1 leading to increased genomic instability (through expression of DNA repair genes including RAD51) as well as contributes to ongoing proliferation of MM cells(through inhibition of DREAM complex).
 Conclusions: Using an integrated genomic strategy, we identified a global gene signature (comprised of 32 genes) associated with genomic instability and poor survival in human cancers including MM. One of the most prominent amongst these, PLK4, is identified as an important component of this signature impacting development of chromosomal changes including hyperdiploidy, a major defining feature in half of MM . With its role in cell cycle and and chromosome integrity, it is a promising target in MM to inhibit growth and progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a26341b8978a6f53aca199b7ea675509c5318f5" target='_blank'>
              Integrated genomic analysis identifies polo-like kinase 4 (PLK4) as a key regulator of chromosomal stability and growth in multiple myeloma
              </a>
            </td>
          <td>
            Leutz Buon, Poonam Kalhotra, S. Talluri, Jiangning Zhao, Harshini Sriram, C. Chakraborty, Masood Shammas, Nikhil C Munshi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b8e7f8962ad8fcf01d30be8350fefcd7ba1a81" target='_blank'>
              CRISPR live-cell imaging reveals chromatin dynamics and enhancer interactions at multiple non-repetitive loci.
              </a>
            </td>
          <td>
            Meishuo Liu, Keyun Huang, Jie Zhang, Qingyang Li, Xinming Wang, Buming Gu, Hao Tang, Zhenhai Du, Liangjun Hu, Shutao Qi, Yu Ma, Hongtao Yu, Wei Xie, Xianyang Fang, Haifeng Wang
          </td>
          <td>2025-11-06</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4935b57359aad70cac663f1f7ac7eb81e40945" target='_blank'>
              AP-1 promotes oncogenic transcription in lung cancer cells by bridging promoter-enhancer interactions.
              </a>
            </td>
          <td>
            Xianglong Tan, Michael Kroneberg, Fei Sun, Kevin Avelar Diaz, Alisha Flora, Michael F Carey
          </td>
          <td>2025-12-03</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Rhabdoid tumors (RTs) are highly aggressive cancers driven by biallelic mutation of SMARCB1, a core subunit of the BAF (SWI/SNF) complex. We found that SMARCB1-deficient tumors have a defect in the microhomology-mediated end-joining (MMEJ) pathway, and SMARCB1 is essential for maintaining the protein level of the core MMEJ protein Polymerase Q (POLQ). Mechanistically, SMARCB1 facilitates the nuclear export of the POLQ mRNA through its interaction with the nuclear pore complex. Interestingly, loss of MMEJ in RT cells leads to a compensatory activation of, and a hyper-dependence on, the Fanconi Anemia (FA)/BRCA pathway. Knockout or inhibition of this pathway selectively kills RT cells. Notably, RBM39 degraders, novel splicing modulators, effectively inhibit the FA/BRCA pathway and kill RT cells. SMARCB1 and other cBAF/pBAF components are important for maintenance of MMEJ activity and POLQ protein level, suggesting that BAF-deficient cancers more broadly may be treated by targeted inhibition of the FA/BRCA pathway. Teaser SMARCB1-deficient tumors lose POLQ protein and MMEJ repair, forcing a hyper-dependence on the FA/BRCA pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f010170f43c4678dd73e5bdabf76a69257f6e847" target='_blank'>
              Defective Microhomology-Mediated End-joining in SMARCB1-Deficient Tumors
              </a>
            </td>
          <td>
            Guangli Zhu, S. Asada, Huy Nguyen, Y. Hirohashi, Lifang Sun, Avneesh Saravanapavan, Sirisha Mukkavalli, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf3c669fabe8f21d1e26ee85c7f96eb437ce3f" target='_blank'>
              DNA Circles as Vehicles for Genes to Evade Chromosomal Discipline.
              </a>
            </td>
          <td>
            Monica Rojas-Triana, J. J. Boomsma, Birgitte Regenberg
          </td>
          <td>2025-12-03</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Prime editing, a flexible "search-and-replace" genome editing technique, has become a potent instrument for accurately altering the genome without the need for donor templates or double-strand DNA breaks. The vast non-coding regulatory genome, which includes enhancers, promoters, untranslated regions (UTRs), and non-coding RNAs, is now understood to be a major driver of tumorigenesis, despite the fact that the majority of cancer research has concentrated on protein-coding mutations. Oncogene dysregulation, tumor suppressor silencing, and treatment resistance can all result from mutations in these areas. The convergence of these two fields is examined in this review, which also describes the use of prime editing to fix particular, harmful non-coding mutations. In order to restore normal gene expression patterns, we describe methods for focusing on regulatory elements, present a landscape of non-coding drivers in cancer, and talk about the fundamentals of prime editing. Lastly, we look at delivery methods, therapeutic factors, and how prime editing might be incorporated into the precision oncology paradigm going forward. We emphasize how prime editing could help usher in a new era of targeted therapy and functional genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9d66804e5625efa917cbd699bfc770f8da5aa5" target='_blank'>
              Prime Editing the Non-Coding Genome: A Molecular Review of Correcting Regulatory Mutations for Precision Cancer Therapy
              </a>
            </td>
          <td>
            Alina Mansoor, Midhat Narmeen Mustafa, Abdul Ahad Mehboob, Nureen Shafaqat, Sanam Dehraj, Amna Noor, Sehresh Murtaza, Mahnoor Javed
          </td>
          <td>2025-10-30</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA copy number changes are the most frequent genomic alterations in cancer cells. Here, we demonstrate that Rad27/FEN1, a structure-specific nuclease in budding yeast, plays a crucial role in maintaining the stability of the ribosomal DNA (rDNA) repeats. Severe rDNA instability is observed in the rad27∆ mutant, independently of Fob1-mediated DNA replication fork arrest and DNA double-strand break (DSB) formation in the rDNA. The rad27Δ mutant accumulates Okazaki fragments in the rDNA region, without inducing the formation of detectable DSBs. Similar rDNA instability is observed in DNA ligaseCdc9-deficient cells. Furthermore, Exonuclease 1 and PCNA partially compensate for the loss of Rad27 in rDNA stabilization. These findings highlight the importance of proper Okazaki fragment maturation in the maintenance of rDNA stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f06a68a9173b59a7c4386ddec1b029c06486a7" target='_blank'>
              Rad27/FEN1 prevents accumulation of Okazaki fragments and ribosomal DNA copy number changes.
              </a>
            </td>
          <td>
            Tsugumi Yamaji, Yuko Katayama, Nanase Arata, Mariko Sasaki
          </td>
          <td>2025-10-27</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer and is a potential vulnerability that can be selectively targeted. To systematically identify genes that affect the incidence and fitness of aneuploid cells, we conducted a genome-wide CRISPR/Cas9 screen using NMS-P715, an inhibitor of the spindle assembly checkpoint (SAC) kinase Mps1/TTK. In this study, we identified a number of genes known to regulate aneuploidy and mitosis, and subsequently focused on PRR14L, a ubiquitously expressed gene previously implicated in chronic myelomonocytic leukemia (CMML). Proximity labeling of PRR14L using TurboID revealed several cell division proteins, including the PP2A-B56 phosphatase complex and the spindle assembly factor TACC3, as PRR14L-interacting proteins. Loss of PRR14L prolongs SAC-dependent mitotic arrest in response to microtubule depolymerization but, paradoxically, leads to catastrophic mitotic errors upon SAC abrogation by MPS1 inhibitors. A model derived from our findings provides a rationale for exploiting MPS1 inhibition as a potential vulnerability in cancers containing either PRR14L loss of function mutations or FGFR-TACC3 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Mutations in non-coding regulatory elements are increasingly recognized as critical drivers of cancer development and progression. Among these, structural alterations such as duplications of super-enhancers (SEs), large clusters of enhancers that regulate key oncogenes, have emerged as significant contributors to oncogenesis. In T-cell acute lymphoblastic leukemia (T-ALL), a particularly aggressive hematologic malignancy, SE duplications have been identified in approximately 5% of human cases. These duplications frequently involve a non-coding region located approximately 1.3 megabases downstream of the Myc oncogene, a region known as N-Me. Despite the growing recognition of the role of SE duplications in T-ALL, functional studies to determine their molecular origins and contributions to leukemogenesis have been limited, largely due to the absence of suitable and genetically tractable preclinical models. This gap in knowledge hinders both our understanding of disease pathogenesis and our ability to design targeted therapeutic interventions.
 Here, we report the generation and characterization of a novel genetically engineered mouse model of T-ALL that spontaneously develops oncogenic SE duplications at the Myc locus. This model offers the first spontaneous in vivo system to investigate the origin, regulation, and oncogenic function of SE duplications. This model was created by conditionally stabilizing β-catenin in CD4⁺CD8⁺ double-positive (DP) thymocytes in combination with targeted deletion of the transcription factor HEB. This combination is sufficient to drive the development of aggressive T-cell leukemias that consistently harbor duplications of the N-Me super-enhancer. This finding recapitulates the genomic alteration observed in a subset of human T-ALL and provides a unique platform for mechanistic studies.
 We further demonstrate that the molecular events leading to SE duplication and oncogenic Myc activation unfold in a stepwise manner. At the preleukemic stage, stabilization of β-catenin alone is sufficient to enhance recruitment of HEB to the N-Me region. The simultaneous loss of HEB greatly increased chromatin accessibility at the N-Me enhancer. This altered chromatin landscape permits aberrant binding of transcription factors including TCF-1, which plays a critical role in T-cell development and leads to N-Me SE activation associated with significant three-dimensional chromatin reorganization. Chromosome conformation analyses in the leukemic cells reveal the formation of a novel chromatin loop that brings the activated N-Me SE into close spatial proximity with the Myc promoter, enabling oncogenic Myc transcription.
 These findings uncover a novel mechanistic pathway by which β-catenin and HEB interact to reshape the enhancer landscape and three-dimensional chromatin architecture, ultimately promoting SE duplication and aberrant Myc expression in T-ALL. β-catenin appears to play a dual role, first facilitating HEB recruitment under physiological conditions and then, upon HEB loss, permitting the reprogramming of enhancer activity through chromatin opening and ectopic transcription factor binding. Our data suggest that the cooperative interplay between signaling pathways and transcriptional regulators at enhancer elements can initiate structural genomic changes that drive malignancy.
 This work provides a long-sought preclinical model that faithfully mirrors key features of human T-ALL. Our findings open new avenues for understanding how non-coding regulatory mutations arise and exert oncogenic effects and have the potential to provide insights that may extend beyond T-ALL to other cancers driven by enhancer dysregulation. Moreover, by revealing molecular intermediates such as β-catenin, HEB, and TCF-1 in the regulation of enhancer activity, our findings identify potential therapeutic targets to disrupt the oncogenic enhancer-promoter interactions that underlie Myc-driven leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c3354f2ef9255003e9c97bc93c32af47ddbd359" target='_blank'>
              Myc super-enhancer dynamics in T-ALL
              </a>
            </td>
          <td>
            Georgios Tousinas, Vasili Toliopoulos, Melissa Tracy, Akinola Emmanuel, K. Khazaie, Thomais Papamarcaki, P. L. Bergsagel, F. Gounari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background Centromere breakage has been associated with anti-centromere antibodies in systemic sclerosis (SSc), yet the origins of centromere damage and its link to immune activation remain unclear. The bleomycin induced fibrosis model is widely used as an experimental model of SSc. Here, we investigated whether bleomycin selectively disrupts centromeres and whether such damage contributes to chromosome instability and immunogenic chromatin mislocalization. Methods We evaluated centromere integrity and downstream genome damage-response phenotypes across complementary systems: the bleomycin mouse model of skin fibrosis, hTERT-immortalized human fibroblasts, and primary dermal fibroblasts from SSc patients. Centromeric α-satellite copy number, DNA damage and repair markers, micronuclei, nuclear envelope rupture, and colocalization of centromeric chromatin with antigen presentation machinery were assessed by qPCR, RT-qPCR, and immunofluorescence microscopy. Results Bleomycin induced selective depletion of α-satellite repeats at centromeres in mouse skin. In human fibroblasts, bleomycin generated double-strand breaks by γH2AX foci that preferentially colocalized with CENP-A-marked active centromeres and produced persistent centromere loss. Damage signaling required ATM and relied mainly on RAD51-mediated homologous-recombination repair, yet centromere restoration remained incomplete. Bleomycin also increased micronuclei, cytoplasmic centromeric foci, and BANF1-marked nuclear-envelope rupture, consistent with mis-segregation-driven chromatin leakage. CENP-B colocalized with HLA-DRB1 in bleomycin-treated fibroblasts and limited cutaneous SSc (lccSSc) fibroblasts, revealing centromere-derived antigen-presentation signatures. Conclusions This study identifies active centromeres as selectively vulnerable and incompletely repaired targets of bleomycin-induced DNA damage and demonstrates that centromere instability produces mislocalized chromatin capable of engaging antigen-presentation pathways. These findings provide mechanistic insight into SSc autoantibody specificity and link genome instability to immune activation in fibrotic autoimmunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, , Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Telomeres are known to be important for Leishmania biology but the mechanistic of how the process of telomere maintenance contributes to genome stability remains an unanswered question. Their maintenance is most commonly facilitated by the telomerase ribonucleoprotein complex that elongates telomeres countering their natural shortening due to the incomplete DNA replication in each cell cycle. In some organisms, telomere maintenance is achieved through telomerase-independent mechanisms, such as the Alternative Lengthening of Telomeres (ALT) pathways described in yeasts with dysfunctional telomerase and some rare telomerase-negative human cancer cells. Molecular markers for the ALT pathway include presence of the heterogeneous (in their length and sequence) telomeres, high level of telomeric exchange between the sister chromatids, increased expression of Rad51 and associated proteins, and occurrence of extrachromosomal telomeric repeats that can be present in either linear or circular form. Here, we used third-generation sequencing techniques in combination with other approaches and analyzed telomeres of L. mexicana at unprecedented high-level resolution. We demonstrate that Ku80 ablation-driven telomere elongation varies between chromosomes, possibly due to the chromosome-specific recombination rates, which are sequence/content dependent and associated with the structure of the telomeric tandemly repeated sequence, TTAGGG. Moreover, this telomere length heterogeneity is accompanied by an increased level of C-circles, a subclass of circular telomeric DNA highly specific for ALT activity. Our findings underscore that L. mexicana promastigotes have an inherent ability to utilize ALT, and the loss of Ku80 and/or TERT further enhanced this trait. These proteins work together to maintain telomere integrity, inhibit recombination, and stabilize telomere lengths. Our data suggest that ALT may be a fundamental and readily activated feature of Leishmania biology, and that telomere regulation in this organism significantly differs from what has been observed in other eukaryotic model species, including iconic T. brucei.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e070c6fd10f58ebc83ce00a4b0839a86950d886" target='_blank'>
              Leishmania mexicana telomeres at high resolution: Ku80, TERT, and alternative lengthening mechanisms
              </a>
            </td>
          <td>
            Edubiel A. Alpizar-Sosa, Andreu Saura, Petr Fajkus, Ester Poláková, Kateřina Havlová, Amanda T. S. Albanaz, Jiří Fajkus, Vyacheslav Yurchenko
          </td>
          <td>2025-10-24</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Meiotic drivers are selfish genetic elements that subvert Mendelian inheritance to increase their own transmission, yet they are typically found at low frequencies across natural populations. The factors that limit their spread remain unclear. To investigate this paradox, we studied the Segregation Distorter (SD) system, a selfish coadapted gene complex in Drosophila melanogaster. SD biases its transmission by killing sperm carrying a homologous chromosome bearing a target locus, Responder (Rsp), which appear as satellite repeats. Such selfish killing impairs male fertility and imposes selective pressure on the host genome to evolve resistance, either by deleting Rsp copies or acquiring unlinked suppressors. To characterize the spectrum of Rsp alleles and the frequency of segregating suppressors, we surveyed 90 strains from the Drosophila Genome Reference Panel. Rather than loss of Rsp, we found that over half of the strains (52/90) harbor suppressors located on the X chromosome or autosomes, but not the Y chromosome. The widespread presence of strong suppressors limited the resolution of our genome-wide association mapping; however, recombination analysis identified a strong X-linked suppressor to a ∼300 kb interval on the chromosome. Together, our findings suggest that pervasive, multilocus suppression constrains the spread of SD in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e04c6f5ef9fa6ac101d2f55bfc644ec979072" target='_blank'>
              Pervasive suppressors halt the spread of selfish Segregation Distorter in a natural population
              </a>
            </td>
          <td>
            Ching-Ho Chang, Tyler Handler, Nicholas J. Fuda, Danielle Pascua, Taylar Mouton, A. Larracuente
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-12-01</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155a629eff7b8ba829fe935812c535143d1302b5" target='_blank'>
              Rewiring cancer epigenome: lncRNA as modulator of chromatin architecture and neoplastic transformation
              </a>
            </td>
          <td>
            Santosh Kumar, S. Kundu, S. Sharawat, Ashok Sharma
          </td>
          <td>2025-11-19</td>
          <td>Mammalian Genome</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Human tissues require a mechanism to generate durable, yet modifiable, transcriptional memories to sustain cell function across a lifetime. Previously, it was demonstrated that nanoscale packing domains couple heterochromatin (cores) and euchromatin (outer zone) into unified reaction volumes that can generate transcriptional memory. In prior work, this framework demonstrates that RNA synthesis occurred within the ideal zone (intermediate density) portions of the domain. Naturally, this creates a question of where genes are positioned in relation to the packing domain architecture and which genetic material fills the domain core to sustain transcription. Here, it is proposed that this can be solved by the encoded positioning of introns, intergenic segments, and exons as a projection of the functional packing layers of domains. This suggests that introns and intergenic segments are coupled to adjacent exons to generate coherent packing domain volumes. How this organization will reconcile contradictions in epigenetic patterns, non-randomness in oncogenic mutations, and produce durable transcriptional memory is illustrated. The study concludes by showing that this genome geometry may have coincided with the rapid evolution of body-plan complexity, suggesting that chromatin geometry could be fundamental to metazoan evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28ea2cbe5621f89d0e90527ffbe4497e1a12d59" target='_blank'>
              Geometrically Encoded Positioning of Introns, Intergenic Segments, and Exons in the Human Genome.
              </a>
            </td>
          <td>
            L. Almassalha, Kyle L MacQuarrie, Marcelo Carignano, Cody L Dunton, Ruyi Gong, Joe Ibarra, Lucas M Carter, Wing Shun Li, Rikkert J. Nap, P. Dulai, I. Szleifer, Vadim Backman
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="The vast majority of recurrent somatic mutations arising in tumors affect protein-coding genes in the nuclear genome. Here, through population-scale analysis of 14,106 whole tumor genomes, we report the discovery of highly recurrent mutations affecting both the small (12S, MT-RNR1) and large (16S, MT-RNR2) mitochondrial RNA subunits of the mitochondrial ribosome encoded within mitochondrial DNA (mtDNA). Compared to non-hotspot positions, mitochondrial rRNA hotspots preferentially affected positions under purifying selection in the germline and demonstrated structural clustering within the mitoribosome at mRNA and tRNA interacting positions. Using precision mtDNA base editing, we engineered models of an exemplar MT-RNR1 hotspot mutation, m.1227G>A. Multimodal profiling revealed a heteroplasmy-dependent decrease in mitochondrial function and loss of respiratory chain subunits from a heteroplasmic dosage of ~10%. Mutation of conserved positions in ribosomal RNA that disrupt mitochondrial translation therefore represent a class of functionally dominant, pathogenic mtDNA mutations that are under positive selection in cancer genomes. Analysis of 14,106 tumor genomes highlights recurrent mutations in mitochondrial ribosomal RNA encoded within the mitochondrial genome. Mutations occur at hotspot positions and are under strong purifying selection in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eda054c049bfbd95fe327306b7323841c22683" target='_blank'>
              Functionally dominant hotspot mutations of mitochondrial ribosomal RNA genes in cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Jacqueline Tait-Mulder, Minsoo Kim, Cerise Tang, Tricia Park, Flora McNulty, S. Lilla, S. Zanivan, Alejandro Huerta-Uribe, Benan N. Nalbant, Mark Zucker, D. Sumpton, Geoffray Monteuuis, Christopher B Jackson, Wei Wei, P. Chinnery, R. Chaligné, Caleb A. Lareau, Ed Reznik, P. Gammage
          </td>
          <td>2025-11-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840b437f838fbeb4893166e383606b0c1b14d251" target='_blank'>
              Integrated Metabolic Complex Genetic Interaction Network of Chromosome 4p Loss in Basal Breast Cancer
              </a>
            </td>
          <td>
            Lynn Karam, Alain Pacis, Rohan Dandage, Michael Schwartz, Gabriel Alzial, Anastasia Gherghi, H. Djambazian, Gerardo Zapata, Marina A Schapfl, Paria Asadi, Mayerly Castrillón, Alysh Orr, Belen Hernandez, Pegah Kargar, Michelle Vandeloo, Sylvia Santosa, A. Villunger, Traver Hart, Jiannis Ragoussis, Geneviève Deblois, Andreas Bergdahl, Morag Park, Guillaume Bourque, Elena Kuzmin
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Background: Multiple Myeloma (MM) is a plasma cell malignancy characterized by genomic instability, often manifesting as chromothripsis. Although CNVs in DNA repair genes are recurrent in MM, the combined effect of germline defects and somatic alterations in driving aggressive, therapy-resistant disease remains incompletely understood.
 Methods: We performed whole-genome sequencing (WGS) on tumor DNA from a patient with newly diagnosed (NDMM), progressive, treatment-resistant multiple myeloma and CNV gains across multiple chromosomes. This patient was from the original cohort of 17 patients we reported where we detected a possible correlation of copy number gains in DDR genes and aggressive myeloma.1 Analysis included detection of SNVs, indels, CNVs, structural variants (SVs), HRD scoring, mutational signatures, and pathway enrichment.
 Results: Germline mutations were identified in key DDR genes (RAD50, BRCA1, CHD1, SEMA4D), resulting in an HR-deficient phenotype (high HRD score/ high Genomic Instability). Variants in CYP2B6, CYP2D6, CYP3A5 indicated impaired drug metabolism. Somatic mutations in TRAF3andARID5B affected immune signaling and transcription. The genome showed near-triploidy (ploidy score: 2.98), high SV burden, and dominant CNV signatures, associated with whole-genome duplication and chromothripsis (or, chromoplexy).
 Mutational signatures included SBS1, SBS5, and DBS17, indicating age-related processes, DNA repair defects, and oxidative stress. A high MATH score along with local hypermutation events reflected tumor heterogeneity. SV analysis revealed extensive deletions, duplications, and complex rearrangements, especially on chromosomes 1 and 11. Widespread Alu element duplications and Alu-mediated rearrangements suggest non-allelic homologous recombination and L1 retrotransposition events.
 Pathway analysis revealed:DNA repair: HRR and NHEJ dysfunction; BRCAness without BRCA1/2 biallelic inactivation.Cell cycle: Impaired G2/M checkpoints.Immune signaling: TRAF3 mutation disrupted cytokine and interferon pathways.Gene transcription: TP53 pathway dysregulation.Metabolism: CYP variants affected drug/xenobiotic metabolism.Signal transduction: CHD1 mutation impacted nuclear receptor signaling.Developmental biology: SEMA4D and BRCA1 affected migration and differentiation.Hemostasis: F5 mutation altered fibrin clot formation, suggesting thrombotic risk, which is a known feature of myeloma.
 Discussion: Thibaud et al.2 found pathogenic germline variants in BRCA1/2 contributing to hereditary risk in 10 percent myeloma cases. Nazaryan-Petersen et al.3 showed germline chromothripsis driven by L1-Mediated Retrotransposition and Alu/Alu Homologous Recombination. Aoki et al.4 found hypomethylation of LINE elements correlated with progression and poorer prognosis of myeloma. They detected an association between common breakpoint loci at commonly lost regions with increased LINE-1 densities. Our findings of widespread Alu duplications in this case suggests the possibility that chromoplexy in myeloma may be induced by retrotransposon activity.
 This case illustrates a “two-hit” MM pathogenesis model: germline mediated HRD creates a permissive genomic background for catastrophic somatic events, including chromothripsis (or, chromoplexy) and immune gene disruption. Similar “two-hit” model involving tumor suppressor genes and oncogenes has been reported by Rodrigues et al.5 These findings emphasize the value of integrated germline-somatic multimodal analysis to identify high-risk disease biology and guide therapeutic strategies. Pawlyn et al.6 have suggested using PARP inhibitors in HRD-LOH MM, which correlated with high-risk disease in the relapsed MM (RRMM) setting.
 Our case suggests that such high-risk presentation can occur in newly diagnosed myeloma (NDMM) too and begs the question: should PARP inhibitors be evaluated for upfront induction treatment of such cases?
 References: Raychaudhuri J et al. Abstract 507/Transplantation and Cellular Therapy 30 2S (2024) S370-S398 Thibaud et al. Blood Cancer Discovery (2024);5: 428–41 Nazaryan-Petersen L et al. Hum Mutat. (2016) Apr;37(4):385-95 Aoki et al. Genome Medicine (2012), 4:101 Rodrigues et al. iScience, (2025) 28, 111620, January 17, Pawlyn C et al. Leukemia (2018) 32:1561–1566
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faab97a25ec8eddcce44ce6eb8cdc61a206c2acd" target='_blank'>
              Germline mediated HR deficiency, alu element duplication, and immune pathway alterations define aggressive, drug-resistant multiple myeloma
              </a>
            </td>
          <td>
            Anindyajit Banerjee, Sombuddharoy Bhowmick, S. Vasudevan, Anurag Mehta, N. Agrawal, V. Sheth, D. Bhurani, Neeraj Arora, Jyotishankar Raychaudhuri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation - DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1655457ebd077b5d78f8c0d2ef963c19750b7b0" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris.
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-11-12</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, , S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="It is unclear how CRISPR editing outcomes vary across the genome and whether undesirable events such as structural variants (SVs) are predictable or preventable. To define a genome-wide map of editability, we performed whole-genome sequencing on 1875 budding yeast clones edited across 16 chromosomes by CRISPR-Cas9 and donor-templated repair. We found that unintended edits, including short indels and SVs, were enriched in specific genomic and sequence contexts. We developed a predictive model, SCORE (System for CRISPR Outcome and Risk Evaluation), which revealed 4.8% of the genome as SV prone, consisting of 562 SV hotspots. Donor repair-enhancing strategies suppressed SV formation in regions with moderate, but not high, predicted risk. Applying SCORE to the Sc2.0 synthetic yeast genome revealed a markedly altered SV landscape due to the removal of endogenous repetitive elements and the insertion of loxP sites. Our study provides the genome-scale map of SV hotspots after CRISPR editing and predictive and experimental tools to mitigate their formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c353e50628925b1279c5062797bfebe0f4ebe5a" target='_blank'>
              The editable landscape of the yeast genome reveals hotspots of structural variant formation
              </a>
            </td>
          <td>
            Shengdi Li, Sibylle C. Vonesch, Kevin R. Roy, Chelsea Szu Tu, Friederike Steudle, Michelle Nguyen, Cosimo Jann, Lars M. Steinmetz
          </td>
          <td>2025-10-31</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16016e64ba44704fcf2f544c64c327f4597e0734" target='_blank'>
              Lamin B1 Safeguards the B Cell Genome and Shapes Lymphoma Outcome
              </a>
            </td>
          <td>
            Filip Filipsky, Katarina B Chapman, Johannes Bloehdorn, Jun Wang, John Gribben, Michael Hausmann, Christoph Cremer, Andrejs Braun, Marta C Sallán, Tanya Klymenko
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The human genome attains an amazing spatial organization in the packaging of 2 m of DNA into a 10-μm nucleus. Such structural organization is achieved by the folding of chromatin and the regulation exerted by architectural proteins such as insulators. Chromatin insulators are boundary elements of the genome that, through enhancing blocking activities, demarcation of chromatin domains, and chromatin looping, regulate transcription. The review focuses on the identification and characterization of insulators in various species, discussing mainly the functions of the CCCTC-binding factor (CTCF) in mammals and functionally equivalent insulator proteins in Drosophila melanogaster. We review here the mechanisms of enhancer blocking, barrier activity, and loop extrusion, emphasizing their effects on topologically associating domains and chromatin architecture. Furthermore, we discuss new concepts that have come into prominence: tethering elements and redundancy among the insulator proteins, which contribute to chromatin organization. Advances in methodology, including chromosome conformation capture and high-resolution imaging techniques, have transformed our view of the dynamic interplay between the architecture of chromatin and transcription regulation. This review discusses the importance of insulators for genome organization and describes future directions in investigating their roles in both gene regulation and three-dimensional genomic architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eda983a70eb60f914e84aedb4a80af4895d808" target='_blank'>
              Chromatin insulators in gene regulation and 3D genome organization.
              </a>
            </td>
          <td>
            Hina Sultana, Rohit Kunar, A. G. Matera
          </td>
          <td>2025-10-01</td>
          <td>Biochemical Society transactions</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Chromatin-based mechanisms contribute to the exquisite regulation of gene expression during animal development. But how those mechanisms evolved remains elusive. Here we investigate chromatin regulatory features in the closest relatives of animals, choanoflagellates. In a model choanoflagellate Salpingoeca rosetta, we compare chromatin accessibility and histone modifications to gene expression. Accessible genomic regions in S. rosetta primarily correspond to gene promoters, and we find no evidence of distal gene regulatory elements that resemble enhancers deployed to regulate developmental genes in animals. Remarkably, the histone modification H3K27me3 decorates genes with cell type-specific expression, revealing a functional similarity in S. rosetta and animals. Additionally, H3K27me3 marks LTR retrotransposons, retaining a potential ancestral role in regulating these elements. We further uncover a putative bivalent chromatin state at cell type-specific genes that consists of H3K27me3 and H3K4me1. Together, these data support the emergence of gene-associated histone modification states that underpin development before the evolution of animal multicellularity. Here, the authors investigate chromatin-based gene regulation in the closest relative of animal, choanoflagellates. They uncover a putative dual role for the histone modification H3K27me3 in regulating of protein-coding genes during differentiation and transposable elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5504ea78d0c86caaeac861a13005f7e8b5070b2" target='_blank'>
              Chromatin profiling identifies putative dual roles for H3K27me3 in regulating cell type-specific genes and transposable elements in choanoflagellates
              </a>
            </td>
          <td>
            JM Gahan, Lily W. Helfrich, Laura A. Wetzel, N. Bhanu, , Benjamin A. Garcia, R. Klose, Alex de Mendoza, David S. Booth
          </td>
          <td>2025-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Upon DNA breakage, a genomic locus undergoes alterations in three-dimensional chromatin architecture to facilitate signaling and repair. Although cells possess mechanisms to repair damaged DNA, it is unknown whether the surrounding chromatin is restored to its naïve state. We show that a single DNA double-strand break (DSB) within a topologically associated domain (TAD) harboring conformation-sensitive genes causes lasting chromatin alterations, which persist after completion of DNA repair and feature topological rearrangements and loss of local RNA species. These newly acquired features of postrepair chromatin are transmitted to daughter cells and manifest as heritable impairments of gene expression. These findings uncover a hitherto concealed dimension of DNA breakage, which we term postrepair chromatin fatigue and which confers heritable impairment of gene function beyond DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70850e6eacefc0378a65ec703a07a0084a7eaca" target='_blank'>
              Repair of DNA double-strand breaks leaves heritable impairment to genome function.
              </a>
            </td>
          <td>
            Susanne C. S. Bantele, Irene Mordini, A. Biran, Nicolás Alcaraz, Gijs Zonderland, Alice Wenger, Nils Krietenstein, Anja Groth, Jiri Lukas
          </td>
          <td>2025-11-06</td>
          <td>Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Bantele and colleagues recently reported that repair of a single CRISPR/Cas9-induced DNA double-strand break (DSB) in the c-MYC topologically associated domain leads to a persistent depletion of chromatin interactions and long-term transcriptional attenuation across multiple generations of human cells. They interpret this observation as evidence for a previously unrecognized principle--"chromatin fatigue"--in which DSB repair generates a stable architectural defect that acts as a heritable impairment to genome function. Such an idea, if correct, would carry profound implications for genome biology, epigenetic inheritance, cancer evolution, aging, and the safety of therapeutic genome editing. However, our detailed reassessment of the experimental design, underlying assumptions, and data interpretation reveals that the evidence provided is inadequate to support these sweeping conclusions. Instead, the observed outcomes are more plausibly explained by a combination of Cas9 persistence, off-target DNA damage, repair-factor retention, MYC enhancer plasticity, and the well-documented genomic instability of HeLa cells. The study does not demonstrate mechanistic causality, does not exclude simpler explanations, and does not provide data consistent with true chromatin memory or heritable architectural change. Moreover, its statistical inferences are based on noisy measurements that fall within expected variability of unstable oncogenic loci. Here, we present a comprehensive critical analysis showing that the proposed model of chromatin fatigue is unsupported by the available evidence. We offer a corrected interpretation in which the chromatin landscape experiences a temporary, repair-associated perturbation that resolves without leaving enduring or heritable impairment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b088cad1c1339b9432ed9215feaa7f00a0cdad" target='_blank'>
              Comment on"Repair of DNA Double-Strand Breaks Leaves Heritable Impairment to Genome Function"
              </a>
            </td>
          <td>
            Yi Wang, Shu-Feng Zhou
          </td>
          <td>2025-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1913940e2ad0a9ac39d6ab69328331969ad3d74" target='_blank'>
              HPV16 E6 oncoprotein promotes microhomology-mediated viral integration by upregulating POLQ expression
              </a>
            </td>
          <td>
            Guangli Zhu, S. Asada, J. Abeykoon, Lifang Sun, S. Mukkavalli, Alan D. D’Andrea
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Super enhancers (SEs) are clusters of enhancers with exceptionally high transcriptional activity, crucial for determining cell identity and regulating gene expression. They function as key regulatory hubs, governing gene networks essential for normal hematopoiesis while also driving the pathogenesis of hematological malignancies. This review summarizes the role of SEs in maintaining hematopoietic lineage identity and examines how their dysregulation in acute myeloid leukemia (AML), myelodysplastic neoplasms (MDS), adult T-cell leukemia (ATL), acute lymphoblastic leukemia (ALL), and multiple myeloma (MM) leads to oncogenic activation. By regulating key oncogenes, SEs represent promising therapeutic targets. Emerging strategies-such as BET inhibitors, CDK7/9 inhibitors, and rational drug combinations-effectively disrupt SE-driven transcriptional programs and show potential to overcome treatment resistance in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335f0fb009eae6a66b9d878f35670a097b643167" target='_blank'>
              Super enhancers as key drivers of gene regulatory networks in normal and malignant hematopoiesis
              </a>
            </td>
          <td>
            Ying Cheng, Guangxin Pei, Hengchao Zhang, Yan Hou, Lei Sun, Hongdi Xu, Yuning Lv, Xiuyun Wu
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rapid prophase chromosome movements ensure faithful alignment of the parental homologous chromosomes and successful synapsis formation during meiosis. These movements are driven by cytoplasmic forces transmitted to the nuclear periphery, where chromosome ends are attached through transmembrane proteins. During many developmental stages a specific genome architecture with chromatin nuclear periphery contacts mediates specific gene expression. Whether chromatin is removed from the nuclear periphery as a consequence of chromosome motions or by a specific mechanism is not fully understood. Here, we identify a mechanism to remove chromatin from the nuclear periphery through vaccinia related kinase (VRK-1)–dependent phosphorylation of Barrier to Autointegration Factor 1 (BAF-1) in Caenorhabditis elegans early prophase of meiosis. Interfering with chromatin removal delays chromosome pairing, impairs synapsis, produces oocytes with abnormal chromosomes and elevated apoptosis. Long read sequencing reveals deletions and duplications in offspring lacking VRK-1 underscoring the importance of the BAF-1–VRK-1 module in preserving genome stability in gametes during rapid chromosome movements. Rapid chromosome movements support the side-by-side alignment of the homologous chromosomes during meiotic prophase. Here, the authors show that absence of BAF-1–VRK-1 mediated release of meiotic chromosomes during ongoing movements leads to chromosome abnormalities, a reduced oocyte pool and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4371516d2fea02020dbd0ac17cf83398bf3bc3" target='_blank'>
              BAF-1–VRK-1 mediated release of meiotic chromosomes from the nuclear periphery is important for genome integrity
              </a>
            </td>
          <td>
            Dimitra Paouneskou, A. Baudrimont, Reka K. Kelemen, Marwan Elkrewi, Angela Graf, Shehab Moukbel Ali Aldawla, Claudia Kölbl, Irene Tiemann-Boege, Beatriz Viçoso, V. Jantsch
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Characterizing tissue-specific (TSp) gene expression is crucial for understanding development and disease; however, traditional expression-based methods often overlook the latent “regulatory grammar” embedded in the non-coding DNA, particularly in distal promoter regions. Here, we introduce TSProm, a framework that specializes a DNA foundation model (DNABERT2) to decipher the long-range regulatory logic of TSp promoters at the gene isoform level. The contributions of our work are two-fold. First, we propose a novel comparative design that trains two distinct models, A: for general promoter biology and B: for TSp regulation. These models enable the precise isolation of sequence motifs around the transcription start site that uniquely define tissue identity. Second, we introduce a comprehensive explainable AI (xAI) module that integrates attention-based discovery with model-agnostic SHAP analysis to provide robust, cross-validated interpretations of learned features. Applying this framework to human brain, liver, and testis promoters, we identified and validated clinically relevant transcription factors (TFs) in the brain, including SP1, MYC, and HES6, and confirmed their known roles in diseases such as gliomas and neuroblastomas. Our analysis further revealed that C2H2 Zinc Finger proteins are a dominant feature of the global landscape of TSp gene regulation. TSProm provides a novel and interpretable framework for identifying TSp gene regulatory elements, offering powerful computational tools for the study of tissue-specific gene regulation in normal and disease conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a83e73abe66e3648e75b598545eed1991706b17" target='_blank'>
              TSProm: Deciphering the Genomic Context of Tissue Specificity
              </a>
            </td>
          <td>
            Pallavi Surana, Pratik Dutta, Nimisha Papineni, Rekha Sathian, Zhihan Zhou, Han Liu, Ramana V. Davuluri
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219b8e66a168673e1981b45ca84c3b808ce85a83" target='_blank'>
              BMAL2 is a druggable target for ARID1A-wildtype ovarian clear cell carcinoma (OCCC)
              </a>
            </td>
          <td>
            Grace Y. T. Tan, Pei-Yi Lin, Li-Tzu Cheng, Yu-Sheng Tsai Yuan, Shih-Han Huang, C. Yu, Chao-Tsen Chen, P. Chi, Wendy W. Hwang-Verslues
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma.
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell death & disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The search for novel tumor-specific markers and therapeutic targets is driving the development of more effective and personalized treatment strategies for cancer patients. This article focuses on investigating a promising new source of biomarkers—readthrough transcripts, or downstream-of-gene (DoG) transcripts. These transcripts are extended products of gene transcription that continue into intergenic regions and can overlap neighboring genes, sometimes giving rise to cis-splicing of adjacent gene (cis-SAGe) transcripts. Recent studies suggest that besides frequently being a “transcriptional noise”, DoG transcripts can perform regulatory functions, serve as a source of novel protein products, and act as prognostic markers of patient survival across various cancers. This article aims to investigate the regulatory mechanisms and functional significance of readthrough transcripts in tumors, to identify currently known tumor-specific variants with potential utility as cancer biomarkers or therapeutic targets, and to evaluate the most suitable approaches for their detection. The knowledge gained through this research may provide a foundation for the development of diagnostic test systems and the design of new anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/005f6133a4fcac56d4aa30b3f8401ba10e194f84" target='_blank'>
              Regulatory Mechanisms and Functional Roles of Readthrough Transcripts in Tumorigenesis
              </a>
            </td>
          <td>
            Aleksander Modestov, G. Zakharova, E. Poddubskaya, Anton Buzdin
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b33cda1e67d599b36aad32c46e387766265d476f" target='_blank'>
              ssG4-seq for global profiling of strand-specific G-quadruplex structures in mammalian genomes.
              </a>
            </td>
          <td>
            Sheng Li, Ruoyan Wang, Jinyue Zhang, Changchang Cao, Jiangshan Bai, Zhaokui Cai, Rong Ye, Juan Chen, Wanglong Liu, Chen Lu, Ruitian Li, Yuanchao Xue
          </td>
          <td>2025-12-04</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7dd9b9ee24ce0fa62bc6faefc53ea6b0b9ed0f" target='_blank'>
              DHX36 plays an oncogenic role in breast cancer progression and invasiveness.
              </a>
            </td>
          <td>
            Chunli Wei, Dongmei Xu, Jingliang Cheng, Meiling Zheng, Ting Li, Jing Yang, Jiayue He, Ali El-Far, Lianmei Zhang, Xiuli Xiao, Tao He, Junjiang Fu
          </td>
          <td>2025-12-02</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Polycomb targeted loci form long-range chromatin interactions independent of CTCF-cohesion and are demarcated by low DNA methylation(1–3). Polycomb targets form extremely long-range loops (long Polycomb loops) that can separate anchors up to 60 Mb. These loops predominantly occur in cells of self-renewal status, such as human hematopoietic stem cells (HSC) and mouse embryonic stem cells (ESC), but rarely in cell lines. To identify long Polycomb loops in cancer, we initiated a pan-cancer survey of long Polycomb loops in a collection of 264 tumor samples (33 acute myeloid leukemias, 17 T cell lymphoblastic leukemia, 29 Breast cancer samples, 63 pediatric brain tumors, 70 prostate cancers, and 42 colon cancers). We found most cancers, including all prostate cancers and colon cancers, lack long Polycomb loops except pediatric brain tumors and certain AMLs. In pediatric brain tumors, we found 30% of the ependymoma PFA subtype (which shows globally depleted H3K27me3) notably displayed strong long Polycomb loop interactions. AML displayed more diverse levels of long Polycomb loop interactions. Most AMLs lost both long Polycomb loops with Polycomb binding loss at the loop anchors. Whereas 10% of AMLs retain long Polycomb loops as strong as in HSCs. These AMLs recurrently carry mutations in CEBPA and STAG2, which are not associated with the Polycomb complex or DNA methylation machinery. We found that long Polycomb loop strong AML is sensitive to EZH2 inhibition, which induces cell differentiation. Conversely, PRC1 component dependency can be used to predict long Polycomb loop formation in B-cell lymphoblastic leukemia cell lines. Our analysis suggests that the oncogenesis process antagonizes the long Polycomb loop maintenance in most cancers, yet certain cancers may still preserve strong long Polycomb loops from cell-of-origin. The maintenance of long Polycomb loops sensitizes cells to Polycomb inhibition, indicating that such loops could be an epigenomic biomarker for pharmacological or genetic Polycomb inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd40648aae14140db85bd195a593affdaa280e98" target='_blank'>
              Pan-cancer 3D genomic analysis revealed extremely long Polycomb loops as the biomarker for sensitivity to Polycomb inhibition
              </a>
            </td>
          <td>
            Sean Moran, Zhong Fan, M. Zanovello, Feng Fan, David Xue Qing Wang, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7bcb7a0494ca2d2085913525073bd8291d09b9" target='_blank'>
              EGFR suppression and drug-induced potentiation are widespread features of oncogenic RTK fusions
              </a>
            </td>
          <td>
            Yuzhi (Carol) Gao, D. Gonzalez-Martinez, Sofia Wissert, Hana Bader, Nidhi Sahni, Anh Le, R. Doebele, Lukasz J. Bugaj
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c948196f370f90e50659421ebd98d4fcdbcb30c7" target='_blank'>
              Differentiation-Dependent Proximity Proteomics Identifies Novel Host Factors Linked to HPV16 E2 Function
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, A. T. Prabhakar, Jenny D Roe, Elinor Lu, Austin J. Witt, S. Giri, Molly L Bristol, Phoebe Bridy, Xu Wang, Arjun Rijal, Charles Lyons, I. Morgan
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Although NUP85 is a member of the nuclear pore complex (NPC) and is associated with chromosome variation and tumor regulation, its specific role in cancer development remains unclear and requires further research. This study analyzes the mRNA and protein levels of NUP85 in normal and tumor tissues using the TCGA, GTEx, and HPA databases. We investigated the relationship between NUP85 and survival rates, as well as clinical features, in various tumors by analyzing the TCGA database. The expression pattern of NUP85 in cancer cells is analyzed using single-cell sequencing data from the TISCH database. Two types of prostate cancer cell lines are utilized to investigate the impact of NUP85 on cell proliferation, migration, invasion, and apoptosis, as well as its regulatory mechanism. These analyses aim to uncover the role of NUP85 in cancer, particularly in prostate cancer. NUP85 is observed to exhibit elevated expression levels across multiple malignancies, with its heightened expression showing consistent associations with poorer clinical prognoses. Bioinformatic analyses further reveal that NUP85 expression patterns demonstrate significant correlations with the activity of cancer-related pathways and immunological interactions involving macrophages and T cell populations. Notably, experimental studies using prostate cancer models have documented reduced cellular proliferation following NUP85 knockout, suggesting a potential functional connection warranting further mechanistic investigation. The results indicate that elevated NUP85 expression shows strong correlations with cancer initiation and progression. These findings support its potential utility as a candidate biomarker for disease monitoring across various malignancies, though mechanistic validation remains necessary to establish clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d9fcec4dab152579fb48a48d3474142345170" target='_blank'>
              Comprehensive pan-cancer evaluation of NUP85 prognosis, immune infiltration response, and validation in prostate cancer
              </a>
            </td>
          <td>
            Zhijun Chen, H. Guan, Long Chen, Xinwei Yuan, Wenyan Sun, Ming Chen
          </td>
          <td>2025-10-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute promyelocytic leukemia (APL) is a subtype of acute myeloid leukemia (AML), characterized by a fusion between the PML and RARA genes and by a block in the myeloid maturation at the promyelocytic stage. This study investigates the epigenetic landscape of APL by integrating ChIP-seq data on eight histone modifications and RNA-seq in APL as well as non-APL AML. APL showed a distinct chromatin profile that differed from non-APL AML. We describe APL-specific changes in H3K27ac, H3K9me3, and H3K27me3 with impact on enhancer activity, repression of transposable elements, and Polycomb regulated gene repression. The APL-specific H3K27ac pattern identifies APL-specific enhancer and super-enhancer regions, including a subset of enhancers that are bound by the PML-RARA fusion protein. While chromatin bound specifically by PML-RARA were dominantly active, APL was also characterized by gain of APL-specific heterochromatin states with significant gains of H3K9me3 enriched lamina-associated domains and the transposable elements LINE, LTR, and SINE. These findings suggest a unique enhancer and heterochromatin profile in APL, with implications for transcription regulation and treatment response. These findings offer novel insights into the pathogenesis of APL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4d4d5354c6d704bfe8c1f882f567114dfdc0fb" target='_blank'>
              Disease-specific epigenetic deregulation of enhancers, transposons, and polycomb targets in acute promyelocytic leukemia
              </a>
            </td>
          <td>
            Xiangfu Zhong, Lina Cordeddu, Angelica Gamboa-Cedeno, S. Bengtzén, Karl Ekwall, Andreas Lennartsson, Sören Lehmann
          </td>
          <td>2025-10-30</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="KDELR2, a retrograde transport receptor involved in ER protein homeostasis, has emerged as a potential contributor to cancer development. However, its comprehensive expression profile and immunological roles across diverse human tumors remain unclear. We performed an integrative pan-cancer analysis of KDELR2 using the TCGA and GTEx datasets, and examined its expression patterns, genetic alterations, methylation status, prognostic significance, and associations with immune infiltration and hallmark cancer pathways. Functional insights were derived from GSEA, GSVA, and immune landscape analyses. Wound healing, transwell and sphere formation assays were performed to investigate the role of KDELR2 in the stem-like properties, migration and invasion of pancreatic cancer cells. KDELR2 was broadly overexpressed in various tumor types and significantly associated with advanced tumor stages and poor clinical outcomes. Copy number amplification was the predominant genomic alteration, while promoter hypomethylation also contributed to its dysregulation. KDELR2 expression was correlated with the activation of oncogenic pathways, including those related to the ER stress response, glycosylation, and membrane trafficking. Moreover, KDELR2 was linked to immunosuppressive microenvironmental features, including enrichment of Tregs, TAMs, and MDSCs and reduced infiltration of cytotoxic immune cells. These trends were validated across multiple immune databases. KDELR2 also showed strong associations with immune checkpoint molecules and chemokine signaling networks, suggesting a dual role in tumor progression and immune evasion. Additionally, in vitro experiments revealed that KDELR2 overexpression promoted the stem-like properties, migration and invasion of PANC-1 cells, whereas KDELR2 knockdown led to the opposite results. KDELR2 serves as a potential pan-cancer biomarker associated with tumor aggressiveness and immune modulation. Our findings support KDELR2 as a promising candidate for prognostic evaluation and targeted intervention, particularly in the context of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5799135994e476648644420f015087628fa2dcda" target='_blank'>
              A pan-cancer analysis of the oncogenic role of KDELR2 in human cancers
              </a>
            </td>
          <td>
            Yu Xie, Dong Chen, Qi Zhang
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D. Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Ends of linear chromosomes require protection from host repair machinery that otherwise will mistake them for damaged DNA. The E. coli bacteriophage N15 harbors a linear genome with covalently closed hairpin ends formed by the phage-encoded telomere resolvase TelN. The double-strand break repair complex Mre11-Rad50 (MR, SbcCD in E. coli) specifically targets DNA termini, yet how hairpin telomeres evade host nuclease degradation in bacteria remains unknown. Here, we demonstrate that TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from MR processing in E. coli. Using a combination of genetic and biochemical approaches, we show that this protective function requires both TelN sequence-specific DNA binding and species-specific protein-protein interactions. Notably, we found that protection is independent of TelN’s resolution activity and does not require the C-terminal domains of TelN. Our findings reveal a potentially broad mechanism of telomere protection, providing insights into a conserved regulation of MR activity at chromosome ends across the tree of life. Chromosome ends require protection from the DNA repair machinery. This work demonstrates that TelN protein, the telomere resolvase of E. coli phage N15, protects hairpin telomeres from processing by the host Mre11-Rad50 nuclease through distinct molecular mechanisms. TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from Mre11-Rad50 processing. Protective functions require both TelN sequence-specific DNA binding and species-specific protein-protein interactions with the E. coli MR complex. Hairpin telomere protection is independent of TelN's resolution activity and of the C-terminal domains of TelN, locating a core protection mechanism to conserved N-terminal domains. TelN is essential and sufficient to protect N15 phage-derived hairpin telomeres from Mre11-Rad50 processing. Protective functions require both TelN sequence-specific DNA binding and species-specific protein-protein interactions with the E. coli MR complex. Hairpin telomere protection is independent of TelN's resolution activity and of the C-terminal domains of TelN, locating a core protection mechanism to conserved N-terminal domains. Bacteriophages with linear genomes evade telomere processing by bacterial host MR complexes via a telomere-resolution-independent function of their telomere resolvase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese M. Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718cc7516178174bad744d62d1cf513b5e9fb3f2" target='_blank'>
              Apoptosis uncovers human evolutionary heritage sequestered in non-coding DNA
              </a>
            </td>
          <td>
            Ruchi Joshi, Anushka Roy, Relestina Lopes, Laxmi Kata, Rutuja Selukar, Mrunmayi Markam, Subhangi Banerji, G. V. Raghuram, Snehal Shabrish, Indraneel Mittra
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, JangKeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416974525ed871e08b5c483505e950ab4b46e07f" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation.
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah Burget, Tobias Friedrich, So-Yoon Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-12-02</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bdb65fc0708e21a785db03ef4faf2be314ea734" target='_blank'>
              E2 displacement of CIP2A from TOPBP1 activates the DNA damage response during papillomavirus life cycles
              </a>
            </td>
          <td>
            A. T. Prabhakar, C. James, Allison Q. Nguyen, Phoebe Bridy, Xu Wang, Ryan J. Zoldork, Jenny D Roe, Austin J. Witt, Srimanya S. Panidepu, Molly L Bristol, Krista Dalton, Andreas Wieland, Ahmed Diab, Anthony C. Faber, Elliot J Androphy, Devraj Basu, Paul F. Lambert, Megan E. Spurgeon, I. Morgan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Determining the correct structure of large, interspersed duplications and related complex genomic rearrangements in genetic disease is critical when establishing causal roles and requires a technology able to span the entire duplicated segment(s) on single molecules. We assessed the use of Bionano optical genome mapping (OGM) for this purpose. We combined OGM, Illumina short-read sequencing and fluorescence in situ hybridisation (FISH) to characterise three large interspersed duplications/triplications, and used the deepC algorithm to predict impact on local topologically-associating domains (TADs), assisting functional interpretation. Case 1 harboured paired interspersed duplications (244/323 kb) on chromosome 13. By analysing multiple molecules > 300 kb completely spanning the smaller duplication, we unambiguously determined the correct structure, which potentially alters the TAD containing FGF9, a candidate gene. In Case 2, involving a child with hypertrichosis and gingival hyperplasia (HTC), duplications on chromosomes 16 (2.01 Mb) and 17 (564 kb) were linked on short-read sequencing. By obtaining three OGM molecules spanning the 564 kb segment, we deduced that a t(16;17) translocation was present, which we confirmed by FISH. This interpretation has important implications for clinical risk and highlights KCNJ2 as a potential driver of the HTC3 locus at 17q24.3. Case 3 involved a complex chromoanasynthesis event on chromosome 20. OGM readily resolved all but two of 12 alternative structures; however full resolution required reads to span a 627 kb duplication, which we could not achieve consistently. OGM represents a powerful tool for disambiguating complex structural variants, but requires multiple individual reads to completely span the duplicated segment. In our hands the upper size of duplications that could be resolved was ~ 550 kb. Deducing the correct configuration is critical both for mechanistic understanding of pathogenesis and accurate recurrence risk counselling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e568829e237f4a57a0d68d6063ed9794d08d203" target='_blank'>
              Exploring the size limits of Bionano optical genome mapping to resolve alternative structures of linked interspersed chromosomal duplications
              </a>
            </td>
          <td>
            Yang Pei, E. Calpena, Jill M. Brown, Ron Schwessinger, Lucy Platts, Simon J. McGowan, Tazeen Ashraf, F. Forzano, Jane A. Hurst, Wendy D. Jones, Ajoy Sarkar, Richard J. Gibbons, S. Twigg, Andrew O. M. Wilkie
          </td>
          <td>2025-11-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d88265df3748494692babcea1f6ced027e33fc6b" target='_blank'>
              Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver
              </a>
            </td>
          <td>
            Rémi Tellier, Marie Fernandes, Audrey Vinchent, Sonia Paget, Agathe Laratte, Céline Vuillier, Elisabeth Werkmeister, C. Villenet, Jean-Pascal Meneboo, A. Chotteau-Lelièvre, Eric Wasielewski, Z. Kherrouche, Marie-José Truong, C. Descarpentries, Luca Grumolato, M. Figeac, Alexis B. Cortot, Laurent Poulain, Andréa Paradisi, David Tulasne
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="p190A has been studied across various cancer types, and mutation rates of up to 20% were observed in some cancers, supporting the significance of p190A in carcinogenesis. Since the relevance of p190A in bladder cancer has not been addressed so far, we attempted to explore it. Evaluation of TCGA high-throughput sequencing datasets revealed p190A mutations in up to 8% of bladder cancer samples across several studies. Employing immunohistochemistry on tissue microarray that included 202 BC patient samples, we observed that lower p190A expression correlates with increased invasiveness and poorer survival outcomes. These findings suggested that p190A may have a tumor suppressor function in bladder tissue, consistent with its anticipated function in ploidy-control. p190A knockdown resulted in chromosomal instability in ureter-derived epithelial cells with otherwise normal karyotype, supporting its potential involvement in tumorigenesis. Loss-of-function studies in low-invasive bladder RT4 cell line and gain-of-function experiments in two highly invasive bladder cancer cell lines (T24 and BFTC) demonstrated that p190A influences cell migration and invasion in vitro, as determined by scratch assay and Boyden chamber approaches. This conclusion was further validated by ex vivo porcine bladder invasion approach, invadopodia formation, and gelatin degradation assays. Pathway analysis revealed that altered p190A expression influences both the Rho-ROCK-dependent LIMK1-cofilin pathway and the phosphorylation of cortactin by focal adhesion kinase, both of which regulate critical cellular processes such as actin network organization and polarization to facilitate efficient, coordinated cell migration and division. In summary, our results indicate that p190A has a genuine role in controlling cell-ploidy and regulates actin dynamics in the bladder urothelium, while loss of p190A results in genome instability and drives bladder cancer initiation and progression through deregulated actin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1d93f50c63b74c7bd204430b76069621fb9b43" target='_blank'>
              Loss of p190A RhoGAP induces aneuploidy and enhances bladder cancer cell migration and invasion by modulating actin dynamics
              </a>
            </td>
          <td>
            Qianyu Kang, Xue Kong, Gregoire Najjar, A. Azoitei, Markus Eckstein, A. John, F. Zengerling, Felix Wezel, Christian Bolenz, Cagatay Günes
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transposable elements (TEs) are often epigenetically repressed in eukaryotic cells, but still affect the molecular state of the cell in certain contexts. A flurry of recent studies have elucidated new effects of TE sequences in eukaryotic cells. We review these emerging molecular effects of TEs, including a variety of new mechanisms by which TE sequences affect the cell, including pre- and post-transcriptional regulation of gene expression; cell-to-cell transmission of genes within a multicellular organism through virus-like activity; and RNA-guided DNA insertion. Recent demonstration of TE-guided genome editing underscores the importance of these investigations for both basic and translational research. Future work is needed to continue to unravel the molecular effects of TE sequences across developmental stages, across cell types, and in various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3445e991f8e7a3fbd17d3feddadbe0f7c8e2d56" target='_blank'>
              Molecular effects of transposable element sequences in mammalian cells
              </a>
            </td>
          <td>
            Ming-Ching C. Wen, Joshua D. Welch
          </td>
          <td>2025-11-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Radiotherapy (RT) is a crucial treatment for hepatocellular carcinoma (HCC); however, resistance to radiation remains a major challenge to its effectiveness. Circular RNAs (circRNAs), traditionally considered stable non-coding RNAs, have recently gained attention for their potential to encode proteins or peptides that play crucial roles in HCC radioresistance. In this study, circSNX25 (has_circ_0004874) was identified as a novel circRNA with coding potential related to radiation resistance in HCC through circRNA sequencing, Northern blotting, and mass spectrometry (MS). We demonstrated that SNX25-215, a novel protein encoded by circSNX25, rather than the parental circSNX25 itself, promotes resistance to radiotherapy in HCC cells both in vitro and in vivo. Co-immunoprecipitation (co-IP) and molecular docking revealed that amino acids H207 and E214 of SNX25-215 are critical for binding to Golgi to ER traffic protein 4 homolog (GET4). This interaction inhibits the binding of BCL2-associated athanogene 6 (BAG6) to GET4, thereby exposing the nuclear localization sequence (NLS) of BAG6 and facilitating its nuclear translocation. This, in turn, enhances DNA damage repair, ultimately increasing resistance to ionizing radiation (IR) in HCC cells. Importantly, elevated levels of SNX25-215 lead to nuclear localization of BAG6, endowing HCC cells with radioresistant activity, which is further supported by clinical evidence. Our findings highlight the potential of circSNX25 as a prognostic biomarker and a therapeutic target for overcoming radioresistance in HCC. This study provides deeper insights into the roles that circRNA-encoded proteins play in HCC radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16daa4d6e57dc11c1f72d49cb0e599242e4afab2" target='_blank'>
              Circular SNX25 encoded radioresistance augmenter facilitates DNA damage repair in hepatocellular carcinoma by targeting BAG6-GET4 interaction
              </a>
            </td>
          <td>
            Shuping Li, Ying Gao, Yuhao Tang, Aoran Dong, Shuifang Hu, Ruizhi Wang, Xiaoyong Gong, Zhenwei Peng
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="The hypomethylating agents (HMAs) 5-azacytidine (vidaza-AZA) and 5-aza-2′-deoxycytidine (decitabine-DAC) are part of the standard of care for the treatment of myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). However, the molecular events mediated by HMAs in MDS and AML are poorly understood, and the efficacy of MDS and AML treatments is still improvable. The majority of CpG dinucleotides are located at satellite repeats, and their methylation levels are known to play a fundamental role in ensuring the genomic stability of cells. HMAs are believed to act through a plethora of effects, including DNA demethylation and the consequent re-expression of aberrantly silenced genes, DNA damage due to the covalent trapping of DNA methyltransferases (DNMTs) on DNA, and endogenous retroelements (EREs) reactivation associated with the induction of a cell-intrinsic antiviral response. DNA demethylation of satellite repeats and the consequent genomic destabilization and mitotic impairment of leukemic cells are also believed to play important roles. Although the demethylating activity of HMAs on gene promoters has been extensively investigated, little is known about their effects on satellite DNA methylation during treatment, especially when the selective pressure of the treatment ends. Here, we characterized the dynamics of satellite 2 DNA methylation mediated by decitabine in a human AML cell line model (U937 cells). We demonstrate that the initial demethylation of satellite 2 repeats is followed by complete recovery after 48 h of culture. The observed regain of methylation is associated with increased expression of DNMT3B, the de novo DNMT known to target satellite 2 repeats. In the intent of deciphering the regulation of DNMT3B expression, we found that DAC significantly increased the level of H3 acetylation at the DNMT3B promoter. These preliminary data shed light on DAC-mediated methylation dynamics at satellite 2 repeats, suggesting that satellite 2 remethylation could limit the genomic-destabilizing effects mediated by HMAs in tumor cells and, thus, the future evaluation of strategies to impair this methylation regain and to improve HMAs activity against tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22e258ac309c61d90db3c701350a9fa1bf79d64" target='_blank'>
              Decitabine-mediated DNA methylation dynamics at pericentromeric satellite 2 repeats
              </a>
            </td>
          <td>
            Enrica Sordini, Eugenia Ciurlia, Alessia Zanella, B. Fogliardi, D. Lame, Antonella Poloni, S. Amatori, M. Fanelli
          </td>
          <td>2025-11-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alu elements are short repeats that occupy approximately 10% of the human genome 1,2. Saturation of primate genomes with Alu sequences occurred at the prosimian/new-world monkey evolutionary juncture. Alu elements have clearly driven unique aspects of higher primate evolution, but their presence can be detrimental to genomic stability 3. The expansion of Alu sequences in the genomes of higher primates precisely coincides with a substantial increase in the ubiquitous expression of the three polypeptides of the DNA-dependent protein kinase (DNA-PK), the Ku70/80 heterodimer and DNA-PKcs 4. Previous work suggests that the elevated levels of Ku70/80 are required to prevent the activation of innate immune signaling pathways triggered by RNA molecules derived from Alu elements 5. Here we demonstrate that Ku ablation dramatically alters mRNA splicing, by allowing the use of alternative splice sites contained in intronic antisense Alu elements, which are known to directly associate with Ku70/80 5. Dysregulation of mRNA splicing precedes cell death and preferentially impacts genes involved in essential RNA metabolism processes, including splicing and ribosome biogenesis, likely impacting cell viability. In addition, we demonstrate that cell death after Ku70 depletion cannot be rescued by expression of its prosimian homologue, which suggests that primate Ku70 has evolved specific molecular features to suppress deleterious effects of an Alu element rich genome. We propose a model in which Ku binding of antisense Alu elements in introns of nascent RNAs modulates the use of alternative splice sites to balance beneficial and detrimental contributions of Alu repeats within primate genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6dca09c7acf567685ebeaa9e9f19797124182b" target='_blank'>
              Ku limits aberrant mRNA splicing promoted by intronic antisense Alu elements
              </a>
            </td>
          <td>
            G. Pascarella, Mariia Mikhova, Gargi Parkhi, Jared Godfrey, Joshua R. Heyza, Tomáš Janovič, Andrew J. Olive, P. Carninci, Jens C. Schmidt, Katheryn Meek
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Centromeres are fundamental chromosomal structures that ensure accurate chromosome segregation during cell division. Despite their conserved and essential role in maintaining genomic stability, centromeres are subject to rapid evolutionary change. At the heart of centromere identity is the histone H3 variant CENP-A, an epigenetic mark that defines and propagates active centromeres and is essential for their function. Recent evidence supports a rapid evolution of centromere DNA sequences but also suggests a certain degree of flexibility in CENP-A deposition and propagation. The phenomenon of centromere drift, recently observed in humans, highlights how the dynamic repositioning of CENP-A and associated epigenetic environment over time maintains a regulated equilibrium, ensuring centromere function despite positional variation. Understanding these processes is crucial for unraveling centromere dynamics and their broader implications for genome stability and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0e9c5778af501dce621770da4beeabba7210f5" target='_blank'>
              The dynamic centromere
              </a>
            </td>
          <td>
            Angela Enriquez, Yael Nechemia-Arbely
          </td>
          <td>2025-10-16</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The cattle genome is crucial for understanding ruminant biology, but it remains incomplete. Here we present a telomere-to-telomere haplotype-resolved X chromosome and four autosomes of cattle in a near-complete assembly that is 431 Mb (16%) longer than the current reference genome. Using this assembly (UOA_Wagyu_1) we identify 738 new protein-coding genes and support the characterization of centromeric repeats, identification of transposable elements, and enabled the detection of 2397 more structural variants from 20 Wagyu animals than using ARS-UCD2.0. We find that the cattle X centromere is a natural neocentromere with highly identical inverted repeats, no bovine satellite repeats, low CENP-A signal, low methylation, and low CpG content, in contrast to the autosomal centromeres that are comprised of typical bovine satellite repeats and epigenetic features. Our results suggest it likely formed from transposable element expansion and CpG deamination, suggesting dynamic evolution. We find eighteen X-pseudoautosomal region genes have conserved testes expression between cattle and apes. We also find all cattle X neocentromere protein-coding genes are expressed in testes, which suggests they potentially play a role in reproduction. Advancements in sequencing technologies and assemblers have enabled us to generate a complete, haplotype-resolved X chromosome in cattle. This study discovers the cattle X centromere is a natural neocentromere and characterises its genetic and epigenetic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3dbd28f2812d9774920d7d01884482758e1cf6a" target='_blank'>
              Insights into natural neocentromere evolution from a cattle T2T X chromosome
              </a>
            </td>
          <td>
            Paulene S Pineda, C. Macphillamy, Yan Ren, Tong Chen, Luan Zhong, David L. Adelson, Carey Dessaix, Jose Perez-Silva, Leanne Haggerty, Fergal J. Martin, Cynthia D. K. Bottema, Wayne S Pitchford, Benjamin D. Rosen, Timothy P. L. Smith, W. Low
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Copy Number Alterations (CNAs), including chromosomal gains and losses, shape the genetics and clinical management of acute myeloid leukemia (AML), particularly in complex karyotypes (e.g. CK-AML). Traditionally, AML associated CNAs have been studied using cytogenetic approaches, i.e., karyotyping, that are limited in resolution and sensitivity. Here, low-coverage, sparse whole genome sequencing followed by machine learning applied to over 600 AML patient samples (ECOG-ARCIN trials: E1900, E3999) reveals novel genetic, biologic, and clinical correlates in AML disease. We find that multi-variate Cox-regression analysis applied to the data identifies select patients exhibiting focal and cryptic hemizygous deletions at 17q, 21q, and 3p, in otherwise diploid genomes, associate with inferior survival compared to other patients (median overall survival; OS = 65.5 compared to 421 days, Chi-square test pval=6x10-11), independent of factors such as age, treatment, and karyotype complexity status. In addition, we confirm that amplifications at 11q23, encoding the KM2TA gene, are associated with inferior outcome, and further identify unique breakpoint event features as well as transcriptional signatures suggestive of distinctive KM2TA-translocation independent biology. Applying machine learning approaches to the data, more specifically Non-negative Matrix Factorization (NMF), we develop a prognostic model that captures clinically adverse, cryptic variation such as focal 5q deletions, missed by conventional chromosome analysis and thereby, adds novel, complementary information to traditional, cytogenetic based risk-stratification (e.g. ELN). The model also associates gain of chromosome 8 in Intermediate risk patients with worse outcome compared to other Intermediate risk patients (median OS = 186 compared to 446 days, Chi-square test pval=0.005) and identifies a subgroup of patients within the adverse risk category that lack deletions on chromosomes 5, 7, and 17, and that fare better than the rest of the adverse risk patients (median OS = 188 compared to 97.5 days, Chi-square test pval=0.02), likely reflecting previously identified “atypical” CK-AML patients. Last, we develop a prognostic classifier based on a combination of Neural-Network and Random-Forest based learning, that predicts disease outcome with high accuracy and is based on a three genomic feature list on chromosome bands 5q31.2, 7q31.33, and 8q24, suggesting potential for clinical implementation. Altogether, our results show that low coverage, sparse whole genome sequencing combined with advanced learning methods can be applied clinically to support the diagnosis and clinical management of AML disease as well as serve as a powerful tool to further a basic understanding of the genetics and biology of CNAs in myeloid neoplasms and other cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b53d83e6519034a37c996567686fb00fab394d" target='_blank'>
              Sparse whole genome sequencing and machine learning of AML genomes reveals novel, clinically relevant genetics.
              </a>
            </td>
          <td>
            T. Baslan, Pascal Belleau, Astrid Deschênes, Yanming Zhang, Zhuoxin Sun, Francine Garrett-Bakelman, Maria Kleppe, E. Paietta, Larry Cripe, Hugo Fernandez, Martin S Tallman, J. Rowe, Hillard J. Lazarus, Ari M. Melnick, M. Litzow, Ross L Levine, Nevenka Dimitrova, Selina M Luger, Alexander Krasnitz, Scott Lowe
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Extrachromosomal circular DNA (eccDNA) has emerged as a critical player in cancer biology. Accumulating evidence indicates that eccDNA promotes tumor progression by inducing genomic instability, amplifying oncogenes, and regulating gene expression. However, its biological function and mechanistic role in acute myeloid leukemia (AML) remain unexplored. In this study, we investigated the functional significance and regulatory mechanisms of eccDNA in AML, aiming to identify potential therapeutic targets.



 Bone marrow samples were collected from newly-diagnosed and chemo-resistant AML patients with informed consent. High-throughput eccDNA sequencing was performed to determine eccDNA length distribution. The presence of gene-containing eccDNAs was validated by PCR and fluorescence in situ hybridization (FISH), and synthetic eccDNA constructs were generated to investigate their functional roles in AML. In vivo, we evaluated the impact of eccDNA-encoded LIPIN1 (ecc-LIPIN1) on AML progression and chemoresistance, and developed a PEI/PC7A nanoparticle system for therapeutic intervention. Proteomic and transcriptomic analyses were performed to explore the downstream targets.



 High-throughput eccDNA sequencing revealed abundant eccDNAs (0.3-1 kb) across newly-diagnosed and chemo-resistant AML samples, mapped to diverse genomic regions. Analysis of the GEO dataset (GSE165430) identified genes upregulated in chemo-resistant AML patients. Integrating these data with eccDNA-derived genes highlighted ecc-LIPIN1 as a potential resistance driver.
 PCR and FISH confirmed the presence of eccDNA harboring LIPIN1 nucleotide fragments. Ecc-LIPIN1 levels were significantly elevated in AML samples, particularly in chemo-resistant cases, and associated with poor prognosis. A circular DNA containing LIPIN1 was synthesized based on sequencing data and transfected into AML cells, resulting in elevated LIPIN1 expression. Functional assays demonstrated that ecc-LIPIN1 promoted AML cell proliferation and chemoresistance.
 Given its metabolic role, metabolomic profiling assay revealed that ecc-LIPIN1 primarily regulated lipid metabolism. Fluorescence microscopy and flow cytometry confirmed that ecc-LIPIN1 enhanced lipid droplet formation, contributing to AML progression and chemo-resistance. Proteomic and transcriptomic analyses of ecc-LIPIN1-overexpressed AML cells identified DGAT1, a key enzyme in lipid metabolism, as a downstream effector. Silencing DGAT1 in ecc-LIPIN1-overexpressed AML cells markedly reduced lipid droplet accumulation, indicating that ecc-LIPIN1 drives lipid storage through DGAT1.
 Further analyses revealed that LIPIN1 also modulated endoplasmic reticulum (ER) stress pathways. In LIPIN1-overexpressed AML cells, ER-stress markers CHOP and ATF4 were markedly elevated. Silencing DGAT1 reversed ER-stress activation, suggesting that LIPIN1 promoted AML progression by regulating ER stress via DGAT1.
 To validate the oncogenic function of ecc-LIPIN1, we generated eccDNA constructs containing either wild-type or mutant LIPIN1 nucleotide sequences and transfected them into AML cells. The wild-type ecc-LIPIN1 markedly enhanced AML progression and chemo-resistance both in vitro and in vivo. To counteract this effect, we developed a PEI/PC7A/siLIPIN1 nanoparticle system. Treatment with these nanoparticles efficiently suppressed ecc-LIPIN1 expression, inhibited AML progression, and mitigated chemo-resistance.



 This study provides preclinical evidence that eccDNA plays a critical role in AML progression and chemoresistance by encoding LIPIN1. Mechanistically, ecc-LIPIN1 promotes lipid droplet formation and ER stress via DGAT1, ultimately driving chemoresistance. These findings identify ecc-LIPIN1 as a novel oncogenic factor in AML and highlight its potential as a therapeutic target. Future research into eccDNA biogenesis in AML may uncover new strategies for combating drug resistance and disease progression.



 Extrachromosomal circular DNA; Acute Myeloid Leukemia; drug resistance; lipid metabolism; LIPIN1.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cecd3734536748347f09719465be9ffb74475eb" target='_blank'>
              Eccdna-encoded LIPIN1 drives DGAT1-dependent lipid droplet biogenesis to promote Acute Myeloid Leukemia progression and chemoresistance
              </a>
            </td>
          <td>
            Wancheng Liu, Amin Zhang, XiangLing Xing, Daoxin Ma
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gene expression and regulation with or without alternative splicing are crucial for tissues and cells to function correctly. They have been studied from three almost independent perspectives at the genome level: 1- Recognition of splicing quantitative trait loci (sQTLs), 2- Expression quantitative trait loci (eQTLs) recognition, and 3- Recognition of longer-range physical chromatin interactions between genome segments which model 3D dynamics of cells and tissues. Even though the associations between eQTLs and longer-range chromatin interactions have been previously studied, a similar relationship between sQTLs and chromatin interactions has yet to be analyzed. In this case, examining whether sQTLs control the alternative splicing of their target genes’ mRNA via physically interacting genome segments is crucial. We have jointly analyzed high-throughput chromatin conformation capture (Hi-C) and sQTL datasets over eight human cancer tissues. We have discovered a positive association between the number of genes having sQTLs and chromatin interaction frequency. Such a positive association still exists when we also control for eQTLs. Additionally, sQTLs and their target genes generally exist inside identical topologically associating domains (TADs). These findings are observed over the whole set of analyzed cancer types and functional subsets of the sQTL dataset, such as survival-related sQTLs. Furthermore, tissue-specific sQTLs are statistically enriched in tissue-specific frequently interacting regions (FIREs) in 6 out of 8 human cancer tissues (Chronic Myeloid Leukemia, Colon Adenocarcinoma, Acute Myeloid Leukemia, Lung Adenocarcinoma, Prostate Cancer, Sarcoma). Our sQTL and Hi-C datasets have shown the existence of closer spatial distance between sQTLs and their target genes with possible alternative splicing across several different cancer types in humans. Such a closer spatial distance also exists, independent of whether we integrate eQTLs into the analysis. We found that sQTLs regulate alternative splicing through chromatin interactions. Source code of the analysis in this research is available on https://github.com/seferlab/sqtlhic .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d307f9f6f23c37ffce97f36f862782001b30f0c" target='_blank'>
              Joint analysis of sQTL and Hi-C reveals spatial proximity between sQTLs and target genes in cancer tissues
              </a>
            </td>
          <td>
            Batuhan Eralp, Emre Sefer
          </td>
          <td>2025-10-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Telomerase reactivation via mutations in the telomerase reverse transcriptase promoter (TERTp) is a recurrent feature of glioblastoma, enabling replicative immortality in ~80% of patient cases and presents a critical oncogenic event attractive for therapeutic targeting. TERTp hotspot mutations in glioblastoma generate de novo transcription factor binding sites that recruit the transcription factor GABP, driving aberrant TERT expression. CRISPRoff is an engineered CRISPR/Cas9 fusion protein that can potently and heritably silence cis-regulatory elements such as promoters and enhancers through combined action of DNA and histone methylation. Here, we use CRISPRoff to perform targeted silencing of the TERTp in three patient-derived primary glioblastoma models harboring TERTp mutations. We delivered mRNA encoding CRISPRoff and TERTp-targeting sgRNAs using optimized lipid nanoparticles (LNPs) into glioblastoma primary cultures and organoids. Epigenetic and transcriptional phenotypes were profiled using RT-qPCR, RNA-seq, targeted bisulfite DNA sequencing, long read native DNA sequencing using Oxford Nanopore, ChIP-qPCR, and CUT&TAG. A single transient CRISPRoff delivery was sufficient to stably silence TERT expression by >95% in primary glioblastoma cultures, leading to induction of apoptosis over time and complete replicative senescence in clonal populations. The CpG island encompassing TERTp exhibited ~5-fold increases in cytosine DNA methylation and long read Nanopore sequencing revealed that increased 5mC DNA methylation was specific to the mutant allele of TERTp. Analysis of 5hmC revealed enrichment of this modification upstream of the -124C>T promoter mutation, which was lost upon CRISPRoff activity. ChIP-qPCR revealed a 68% decrease in GABPA occupancy at the TERT promoter. Analysis of H3K9me3 enrichment revealed deposition of heterochromatin within the CpG island of TERTp, and this enrichment was among the most statistically significant increased H3K9me3 peaks genome-wide. In sum, the molecular characterization of CRISPRoff activity against TERTp revealed the ability to achieve mutant allele-specific reprogramming of epigenetic state that could be harnessed for epigenetic therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fd2ebf3520d8717e155c35619009f3a21ba993" target='_blank'>
              EPCO-23. MOLECULAR CHARACTERIZATION OF CRISPROFF EPIGENETIC EDITOR ACTIVITY REVEALS MUTANT ALLELE-SPECIFIC
 TERT
 REPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Akane Hubbard, Zerong Ma, Nicholas O. Stevers, C. Zou, K. Lin, Krish Raval, Rohit Sharma, L. Goudy, Luke A. Gilbert, Niren Murthy, Mitchell S. Berger, Joseph F. Costello, S. J. Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="SINE variable number tandem repeat Alu elements (SVAs) are a unique group of hominid-specific composite retrotransposons with highly variable internal structure. They represent the youngest TE family in humans and contribute to genetic diversity, evolution, and disease. Recent findings indicate that SVA mobilization rates may exceed previous estimates, and many SVAs exhibit insertion polymorphism. SVAs facilitate transduction (TD) events when transcription initiates upstream of a source element, or when their internal termination signal is bypassed, mobilizing adjacent 5’ and/or 3’ sequence. To investigate features of non-reference SVA elements currently polymorphic in the human genome, we analyzed a structural variant callset built upon 35 diverse human genomes generated by the Human Genome Structural Variation Consortium. In our curated dataset of 543 polymorphic, non-reference SVAs, we identify insertions representing the three youngest subfamilies: D (7%), E (38%), and F (55%). Of the latter, we determine that at least 47% are actually SVA_F1, a more recently discovered human-specific subfamily, indicating that F1 is a major contributor to SVA expansion in the human population. We further uncover that 40% of non-reference SVAs carry a TD on their 5’ and/or 3’ ends. Of these, the majority (69%) harbor sequence originating in a gene, including 14 exonic events and the mobilization of a processed pseudogene, supporting the role of SVA in exon shuffling. In addition, we identified a so-called “orphan” TD, defined by the absence of SVA sequence at the insertion site. Leveraging TD origin coordinates, we identify 55 active source elements, including nine non-reference and 46 across GRCh38 and T2T-CHM13, giving rise to 84% of TD-carrying SVAs. Our analyses indicate that SVA_F1 is more active than previously described and is a main driver of SVA expansion. We find two-fold more TD events compared to previous estimates, with an unexpected bias toward 3’ events. Finally, we postulate that the discrepant SVA mobilization rate may be attributed to inter-individual variation in the presence/absence of source elements, a recent uptick in mobilization supported by overall low allele frequencies, and/or negative selection against deleterious insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c58ab3c46bd44059e8817d00743b60c2481618a6" target='_blank'>
              Structural and transduction patterns of human-specific polymorphic SVA insertions
              </a>
            </td>
          <td>
            Ashley E Kirby, Mark Loftus, Emily C. Golba, , H. Abel, Hufsah Ashraf, P. Audano, A. Basile, M. Bonder, H. Brand, M. Byrska-Bishop, Mark J. P. Chaisson, Junjie Chen, Yu Chen, Zechen Chong, Nelson Chuang, Wayne E. Clarke, A. Corvelo, S. Devine, Peter Ebert, Jana Ebler, U. S. Evani, S. Fairley, Paul Flicek, Mark B. Gerstein, M. Ghareghani, Ira M. Hall, William T. Harvey, Patrick Hasenfeld, Alex R Hastie, Wolfram Höps, Pinghsun Hsieh, Sushant Kumar, Joyce Lee, Alexandra P. Lewis, Chong Li, Yang I Li, Jiadong Lin, Tsung-Yu Lu, Rebecca Serra Mari, Ryan E. Mills, Zepeng Mu, Katherine M. Munson, David Porubsky, Benjamin Raeder, Tobias Rausch, Allison A. Regier, Jingwen Ren, Bernardo Rodriguez-Martin, A. Sanders, Martín Santamarina, Xinghua Shi, Oliver Stegle, Arvis Sulovari, M. Talkowski, Luke J. Tallon, J. Tubío, A. Wenger, Xiaofei Yang, Kai Ye, F. Yilmaz, Xuefang Zhao, Weichen Zhou, Qihui Zhu, Michael C. Zody, Jan O. Korbel, Tobias Marschall, Evan E. Eichler, Charles Lee, Miriam K. Konkel
          </td>
          <td>2025-11-06</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease that arises from dysregulated myeloid proliferation. We performed a high-throughput CRISPR interference (CRISPRi) screen in the THP-1 monocytic cancer cell line to identify long noncoding RNAs (lncRNAs) that play a role in contributing to cell proliferation. Our screen identified INSTAR (Intergenic Nuclear Suppressor lncRNA Targeting Adjacent Regulator SFMBT2) as a top candidate. RNA-seq on INSTAR deficient THP-1 cells revealed transcriptional changes in genes involved in cell proliferation as well as other cellular processes. Loss of INSTAR selectively reduced expression of its neighboring gene, SFMBT2. Functional assays confirmed that both genes suppress cell growth, revealing a cis-regulatory mechanism in which INSTAR regulates SFMBT2 expression to control monocyte proliferation. Here, we leverage high-throughput screening to rapidly pinpoint functional lncRNAs providing novel insights into a key regulatory locus consisting of INSTAR and SFMBT2 which could be critical for better understanding dysregulation contributing to acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR , in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Natural chromosomal transformation (NCT) in Bacillus subtilis requires RecA and its accessory proteins including RecX and RecD2. Inactivation of the nucleoid-associated protein (NAP) Hbsu, Rok, or LrpC results in a dual effect: it enhances NCT but exacerbates the NCT defect of ΔrecX cells. Purified EbfC exhibits characteristic features of NAPs: it binds both single- and double-stranded DNA, protects them from degradation, and forms higher-order protein–DNA complexes via its DNA bridging activity. NCT is reduced upon EbfC inactivation but enhanced by ebfC overexpression. hbs55, Δrok, or ΔlrpC mutations suppress the NCT defect of ΔebfC, yet synergistically increase NCT upon ebfC overexpression. The NCT defect in ΔrecD2 cells is worsened by ebfC overexpression or by hbs55, Δrok, or ΔlrpC mutations. The nucleoid was more compacted in ΔebfC cells, an effect counteracted by hbs55, Δrok, or ΔlrpC mutations. EbfC contributes to DNA repair, and ebfC is epistatic to hbs or lrpC in response to DNA damage. We propose that chromosome folding, modulated by NAPs, plays a critical role in NCT and DNA repair. In this context, EbfC, by regulating nucleoid dynamics as a NAP, opposes the functions of Hbsu, Rok, and LrpC in NCT, while their interconnected roles contribute to DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328ab39c4a3f04975076ada74d7385c75dacff21" target='_blank'>
              Nucleoid structure and dynamics influence natural chromosomal transformation in Bacillus subtilis: the role of EbfC
              </a>
            </td>
          <td>
            Rubén Torres, María López-Sanz, Yuri Ushijima, Kazuya Morikawa, Juan C Alonso
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Introduction: During the past two decades, our knowledge about the molecular pathogenesis of myeloid neoplasms (MNs), including acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) has dramatically been improved through the identification of major driver alterations using next generation sequencing. However, with the lack of large-scale analysis using whole genome sequencing (WGS) with sufficient sequencing depths, the analyses have mainly focused on the alterations affecting protein-coding sequences, while the role of non-coding alterations and structural variations (SVs) in myeloid leukemogenesis has not fully been investigated. The etiology of mutagenic processes has also been poorly understood. In the current study, we therefore conducted large-scale deep WGS to address these issues.
 Methods: We performed deep WGS with matched normal samples (mean sequence depth 127.0/30.3 for tumor/normal, respectively) for a total of 903 patients with MN (494 AML and 419 MDS), with an additional 263 cases (94 AML and 169 MDS) currently under analysis. Most cases were also analyzed using deep targeted capture sequencing (n=886) of 446 known/putative driver genes. In addition, RNA sequencing was newly performed for nearly half of cases (n=406). Mutation calling was performed using Mutect2 and GRIDSS2, with in-house modifications to control false positivity in the face of frequent contaminations of tumor cells into germline samples.
 Results: We identified 1,159,638 SNVs and 76,842 sInDels with median mutation burden of 0.46 SNVs and 0.030 sInDels per Mb in AML and 0.39 and 0.022 per Mb in MDS. Validation by deep targeted capture sequencing estimated a true-positive rate of 95% and 78.1% sensitivity for known drivers. In the analysis of mutational signatures, we found that clock-like signatures SBS1 and SBS5 were predominant. SBS19 and SBS32 were detected in 62%/14% and 68%/14% of AML and MDS cases, respectively. As expected, chemotherapy related signatures (SBS25, 31 and 99) were significantly enriched in cases with therapy-related MN cases. The SBS18 signature, which is implicated in oxidative stress–induced mutagenesis, was highly enriched in AML cases carrying t(8;21) and inv(16), but rarely found in those with MDS, highlighting the role of oxidative stress in core binding factor leukemia. Other novel signatures such as SBS24, 39 and 40b were detected in a small number of cases (~1%).
 Our study of a large cohort of MNs also revealed novel candidate driver genes. By evaluating mutation enrichment, we identified a total of 82 candidates of driver genes. Among these, 15 genes were not previously reported as driver genes in MNs, including 7 genes listed in COSMIC cancer gene census database as associated with other cancer types. RNA-seq enabled the identification of intronic mutations predicted to cause alternative splicing in known driver genes, including TET2 and DNMT3A, as well as in novel candidates, observed in 2.1% of cases. In addition to coding genes, mutations in non-coding RNAs with potential driver role (e.g. seed region of mir-142) were detected in 3.3% of the cases.
 We identified a total of 11,050 SV events, of which 48 disrupted common tumor suppressor genes, such as RUNX1, ETV6, CBL, and TP53. Chromothripsis events were detected in 5.3% of cases, which were highly enriched in TP53-mutated cases and frequently affected KMT2A, ETS1, ETV6, TP53, EPOR and ERG. Median of 4 chromosomes were affected in a single chromothripsis event.
 We detected a total of 340 fusion events in 22.6% of all cases, of which 105 fusions involved previously known targets, whereas the remaining 235 represented non-recurrent in-frame gene fusions detected in 11.7% of cases. Although such fusion events were significantly associated with TP53 mutations, 5.6% of TP53-wild-type cases also had in-frame gene fusions, some of which were suggestive of oncogenic potential (e.g. KAT6A::NUTM1). Although low in frequency (1.1%), analysis of expression change revealed SVs affecting cis-regulatory elements including potential enhancer hijacking of a RAS pathway gene.
 Conclusion: Through deep WGS of a large cohort of MN cases, we have delineated a comprehensive landscape of driver mutations and detected new mutational signatures and candidates for driver mutations, including those affecting non-coding regions, underscoring the importance of WGS for better understanding of the pathogenesis of MNs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4576f85031bf75aa25ad786753882bdfc5ce2543" target='_blank'>
              Repertoire of driver events in coding and non-coding regions identified by whole-genome sequencing of myeloid neoplasms
              </a>
            </td>
          <td>
            Koji Okazaki, R. Saiki, Y. Ochi, Kenichi Yoshida, K. Katayama, Hidehito Fukushima, Kaito Mimura, M. Nakagawa, Masanori Motomura, A. Yoda, R. Okuda, Lanying Zhao, Motohiro Kato, Tatsuki Ogasawara, Shigeo Fuji, Lee-yung Shih, N. Uoshima, Mitsumasa Watanabe, Yasushi Miyazaki, M. Ichikawa, Takeshi Maeda, Toshiyuki Kitano, Satoshi Yoshihara, Kazunori Imada, Kinuko Mitani, J. Jansen, M. Sakata-Yanagimoto, Luca Malcovati, Akihiko Gotoh, Hitoshi Kiyoi, N. Kanemura, N. Sezaki, N. Hiramoto, S. Kasahara, S. Imoto, A. Takaori-Kondo, Y. Nannya, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Ribosomal DNA (rDNA), the most abundantly expressed locus in the human genome, is represented by hundreds of units per cell. Each unit includes a 13 kb long region (47S rDNA) containing the genes of ribosomal particles, and a 30 kb long intergenic spacer (IGS). The 47S rDNA is transcribed with varying intensity in different units, some of which remain permanently silent. A key intermediator of this silencing is the promoter-associated RNA (pRNA) produced from a 2 kb long gene situated upstream of the rDNA transcription start site. Recent studies, including ours, suggest that the sequence variability, which normally occurs in mammalian cells, may account for the selective transcription of different rDNA units. The present work is based on the deep sequencing of a pRNA gene fragment and its RNA product and subsequent bioinformatic analysis. We found that a certain SNV, which converts the CCC motif into CCT, as well as deletions which reduce the number of (CCCT) tandem repeats, were significantly more frequent in the DNA than in the respective transcripts. These findings allowed us to establish directly the inhibitory effect of DNA variants on the expression of pRNA and thus (indirectly) the promoting effect on the production of ribosomal RNA. Our results also suggest that (CCCT)n/(GGGA)n DNA repeats and the respective (GGGA)n RNA repeats may form triplex structures, facilitating the function of pRNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891924cd9bf55653e2539c5b0034f6d9fcdd744" target='_blank'>
              Deletions in (CCCT)n repeat regions belonging to the human pRNA gene inhibit its expression
              </a>
            </td>
          <td>
            Nikola Chmúrčiaková, Adam Nógell, E. Smirnov, Dušan Cmarko
          </td>
          <td>2025-10-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b990cf440846339c231eccc76316041249b0f0d" target='_blank'>
              UHRF1 Overexpression Generates Distinct Senescent States with Different Tp53 Dependencies
              </a>
            </td>
          <td>
            Elena Magnani, Charlene Chen, F. Macchi, Bhavani P Madakashira, Tijana Randic, Ian McBain, Kirsten C. Sadler
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The human cancer genome contains a substantial proportion of nonconserved regions that are transcribed into long noncoding RNAs (lncRNAs) and hosted in areas recurrently implicated in cancer. Due to its high responsiveness to extracellular cues, the aberrant lncRNA transcriptome represents a major source of molecular innovation for cancer cells, which are subjected to intense evolutionary pressure during progression and therapy resistance. As such, lncRNAs contribute to the aberrant rewiring of the molecular networks in cancer cells by functioning as molecular sponges and scaffolds, modulating the activity and localization of other biomolecules. This largely untapped reservoir of regulatory elements holds significant potential for addressing the current clinical challenges of cancer progression and therapy resistance. This review synthesizes current insights into the molecular strategies by which lncRNAs subvert homeostatic regulation across diverse cellular compartments and within the extracellular milieu. Further, it explores their multifaceted contributions to cancer therapy resistance, underscoring their emerging prominence as both actionable therapeutic targets and informative biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4243c72a246e400410e4a6a6c6ec766b30aef2" target='_blank'>
              lncRNAs as Targetable Nodes of Oncogenic Networks Underlying Cancer Progression and Therapy Resistance
              </a>
            </td>
          <td>
            Eleonora Leucci
          </td>
          <td>2025-11-25</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Chromosomal loops are CTCF–cohesin-mediated 3D genomic structures in the mammalian nucleus. In addition to these CTCF–cohesin loops, other epigenomic features can also form long-range interactions. Polycomb-targeted loci, regulated by the Polycomb repressive complex and trithorax group during development, form long-range chromatin interactions independent of CTCF–cohesin and are demarcated by regions of low DNA methylation—referred to as DNA methylation canyons.
 We previously identified extremely long Polycomb loops occurring between DNA methylation Canyon demarked Polycomb-targeted loci, spanning distances of up to 60 Mb (Zhang et al, 2020). These loops are found exclusively in self-renewing cells, such as human hematopoietic stem cells and mouse embryonic stem cells.
 In hematopoietic malignancies, both DNA methylation and Polycomb binding are significantly altered. To explore this further, we conducted a pan-cancer survey of long Polycomb loops across 223 tumor samples, with a focus on hematopoietic malignancies—given that their likely cell-of-origin, the hematopoietic stem cell, exhibits strong long Polycomb loops. Our cohort included 32 acute myeloid leukemias (AMLs), 24 T-cell lymphoblastic leukemias, 61 pediatric brain tumors, 80 prostate cancers, and 26 colon cancers. We found that most cancers—including all prostate and colon cancers—lack long Polycomb loops. However, long Polycomb loops are retained in pediatric brain tumors and a subset of AMLs. Interestingly, strong long Polycomb loop interactions are observed in normal developing brain tissue and hematopoietic stem cell. Our data suggest that the presence of long Polycomb loops is likely inherited from the epigenomic state of the cell of origin, as they are also observed in normal pediatric brain tissue and hematopoietic stem cells.
 Loss of long Polycomb loops in primary cancer samples is accompanied by DNA hypermethylation and reduced Polycomb binding at loop anchor loci, and these disruptions are further exacerbated in cultured cell lines. Interestingly, in AML, many of these previously silenced loci—often mesodermal transcription factors—become activated and form de novo 3D interaction anchors. For example, we observed the formation of a new domain around the leukemogenic ZEB2 gene, driven by leukemia-specific HOXA9 binding at an upstream enhancer within the TEX41 locus.
 Notably, AML samples displayed a wide range of long Polycomb loop strength. While most AMLs lose these loops, approximately 12% (4 out of 33) retain strong long Polycomb loops comparable to hematopoietic stem cells. These cases recurrently harbor somatic mutations in CEBPA (2 out of 4) and STAG2 (2 out of 4). These genes are not directly linked to Polycomb or DNA methylation machinery, suggesting alternative mechanisms of loop maintenance. The CEBPA mutant AMLs also exhibited H3K27me3 spreading to non-Polycomb target loci across the genome.
 We therefore tested whether EZH2 inhibition could disrupt long Polycomb loops in these AMLs and impact disease maintenance. Indeed, EZH2 inhibition attenuated long Polycomb loops, reduced colony-forming capacity, and promoted differentiation in long Polycomb loop–retaining AMLs. Treated AML cells activated a macrophage differentiation program and showed downregulation of cell cycle and DNA replication genes. These findings suggest that AMLs retaining long Polycomb loops are dependent on this 3D chromatin architecture. This rare but strong sensitivity to EZH2 inhibition in AML indicates that long Polycomb loops could serve as an epigenomic biomarker for EZH2 or other Polycomb-targeted therapies in cancer rather than the current genomic marker such as EZH2 gain-of-function mutations and SWI/SNF loss-of-function mutations.
 Overall, long Polycomb loops are commonly lost during leukemogenesis due to epigenomic disruption. However, a subset of AMLs maintains these loops from the cell of origin and appears to rely on the Polycomb network to sustain self-renewal.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2c56d775b22a7b9218367e039106ed0000301e" target='_blank'>
              Pan-leukemia 3D genomic profile reveals extreme long polycomb loop in subset of CEBPA mutated AMLs sensitive to EZH2 inhibition
              </a>
            </td>
          <td>
            Sean Moran, M. Zanovello, Zhong Fan, Chao Lu, Jie Liu, Xiaotian Zhang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to genotoxic therapies remains a major contributor to tumor recurrence and treatment failure, yet the mechanisms by which cancer cells escape these therapies through DNA damage response (DDR) activation are not fully understood. Here, we identify a DDR regulatory pathway in which glycogen synthase kinase 3 β (GSK3B), a multifunctional serine/threonine kinase, governs DNA double-strand break (DSB) repair pathway choice by phosphorylating 53BP1 at threonine 334 (T334) — a site distinct from canonical ATM targets. This phosphorylation event disrupts 53BP1’s interaction with nonhomologous end joining (NHEJ) effectors PTIP and RIF1, promoting their dissociation from DSBs and inhibiting 53BP1-driven NHEJ. Simultaneously, T334 phosphorylation facilitates the recruitment of CtIP and RPA32 for DNA end resection and promotes homologous recombination (HR) by enabling BRCA1 and RAD51 loading. Notably, the phospho-deficient T334A mutant of 53BP1, unlike 53BP1 loss, accumulates aberrantly at DSBs along with PTIP/RIF1, impairs end resection, and suppresses HR activity. Importantly, both genetic and pharmacologic disruption of the GSK3B–53BP1 axis sensitizes tumors to PARP inhibitors (PARPi) independently of BRCA1 status. Together, these findings reveal a GSK3B-dependent mechanism that regulates DSB repair pathway choice and provide a rationale for targeting this axis to enhance PARPi efficacy in solid tumors regardless of BRCA1 status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7947f267430312493c565ecaa4ae8dfb1f98a221" target='_blank'>
              GSK3B directs DNA repair choice and determines tumor response to PARP1 inhibition independent of BRCA1
              </a>
            </td>
          <td>
            H. Allam, Scarlett Acklin-Wehnert, Ratan Sadhukhan, M. Patra, F. Xia
          </td>
          <td>2025-11-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reversine is a small-molecule Aurora kinase inhibitor known for its pro-apoptotic effects and potential to remodel chromatin architecture. Although its impact on mitotic regulation is established, its effects on telomere dynamics and nuclear organization in chronic myeloid leukemia (CML) remain unclear. This study aimed to investigate the effects of reversine on telomere architecture, genomic instability, and apoptosis in CML cell lines (K-562 and MEG-01). Reversine was applied at increasing concentrations, and cytotoxicity was assessed using caspase-3/7 activation assays. Quantitative PCR was used to measure AURKA and AURKB mRNA expressions. Three-dimensional telomere architecture was analyzed with TeloView® v1.03 software after Q-FISH labeling to quantify telomere number, signal intensity, aggregation, nuclear volume, and a/c ratio. Reversine induced a dose- and time-dependent apoptotic response in both cell lines and significantly downregulated AURKA and AURKB expressions. Three-dimensional telomere analysis revealed a marked reduction in telomere number and aggregates, signal intensity, and nuclear volume. While reduced signal intensity may indicate telomere shortening, the concurrent decrease in aggregation and altered spatial parameters suggests telomeric reorganization rather than progressive instability. These features reflect structural nuclear remodeling and early apoptotic commitment. Differences between K-562 and MEG-01 responses underscore potential heterogeneity in telomere maintenance mechanisms. Reversine modulates genomic stability in CML cells through dual mechanisms involving Aurora kinase inhibition and telomere architecture remodeling. The integration of 3D telomere profiling highlights reversine’s potential as a therapeutic agent targeting nuclear disorganization and mitotic dysregulation in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e06bedd101bf31c7285fbbc7d8f35b57bf8952e" target='_blank'>
              Reversine-Induced Telomere Architecture Remodeling in Chronic Myeloid Leukemia Cell Lines: Insights from TeloView® Analysis of 3D Nuclear Architecture
              </a>
            </td>
          <td>
            Fábio Morato de Oliveira, Isabela Dias Cruvinel, Bruno Machado Rezende Ferreira, Sabine Mai
          </td>
          <td>2025-10-31</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ab2b82b250c7fe0872534fbb6e35f7aaf3d36d" target='_blank'>
              Cell State Chaos Underpins the Evolution of SMARCA4-Deficient Dedifferentiated Endometrial Cancer
              </a>
            </td>
          <td>
            M. Brandon-Coatham, Julia Vassalakis, Hannah Plummer, Jasleen Kaur, Farzaneh Afzali, Zhihua Xu, Guihua Zhang, Jiahui Liu, Wei Wang, Einav Wajsbrot Renert, Tyler T. Cooper, Alan Dimitriev, Cierra Perron, Andrew Garven, Simatsidk Haregu Abebe, David M Berman, Amber L Simpson, G. Wong, Gilles A. Lajoie, Felix K.F. Kommos, Andreas von Diemlin, Sheela A. Abraham, David J.H.F. Knapp, Alan Underhill, M. Koebel, Cheng Han-Lee, Lynne-Marie Postovit
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Introduction: ASXL1-mutant (mt) Chronic Myelomonocytic Leukemia (CMML) is an aggressive myeloid neoplasm characterized by increased proliferation, resistance to epigenetic therapies, and poor survival. Epigenetic remodeling and up-regulation of leukemogenic driver genes are hallmarks of ASXL1mt CMML. Truncating mutations in ASXL1 activate enhancers that drive the expression of key leukemogenic driver genes such as HOXA9 and its co-factor MEIS1. Here, we investigate the chromatin accessibility of these leukemogenic enhancers by ASXL1 genotype at single-cell resolution (GoT-ChA-seq).
 Methods: After IRB approval, we performed bulk RNA-seq and ChIP-seq (H3K4me1, H3K4me3, H3K27ac, H3K27me3, H2AK119ub, H2BK20ac) on bone marrow (BM) mononuclear cells from 40 patients with CMML (19 ASXL1mt, 21 -wildtype/wt). After quality control, differential analyses of gene expression and histone modification occupancy between ASXL1mt and wt patients were conducted. A consensus peak calling framework was used to define genomic regions with ASXL1mt-specific histone modifications. Canonical enhancers were defined as genomic regions with a co-occupancy of H3K4me1 and H3K27ac. We then added regions marked by H2BK20ac to capture regulatory regions associated with p300 (Narita et al. Nat Genet 2023). We used an in silico prediction strategy to associate regulatory regions marked by strong histone modification occupancy with highly up-regulated leukemogenic driver genes (HOXA6-10, MEIS1). The ReMap database (Hammal et al. Nucleic Acids Res 2022) was queried for transcription factor binding in the genomic regions of interest. We then performed GoT-ChA-seq (Izzo et al. Nature 2024) to investigate chromatin accessibility for ASXL1mt (n=3) and wildtype BM single cells from CMML patients. In vitro colony forming assays were conducted on primary CMML BM cells (11- ASXL1mt and 9 wt CMML), assessing the efficacy of EP31670, a novel, oral, dual inhibitor of p300/BRD4.
 Results: This study included 40 patients with CMML, 19 ASXL1mt and 21 wt. The variant allele frequencies for ASXL1 were compatible with heterozygosity (median 0.41) and TET2 mutations were balanced between the two groups. We identified 149 candidate regulatory regions correlating with the expression of up-regulated target genes of interest (HOXA6-10, MEIS1) by bulk ChIP- and RNA-seq. Publicly available ChIP-seq data from myeloid cell lines demonstrated enrichment of p300 (in THP-1 cells, enrichment effect size +2.58, p=0.002) and BRD4 (in K-562 cells, enrichment effect size +1.58, p=0.045) in these candidate regulatory regions. Next, we interrogated the chromatin accessibility of ASXL1mt (n=13620) and ASXL1wt (n=24269) single cells from 3 CMML patients using GoT-ChA-seq. Differential accessibility analysis revealed 144 genomic regions, all of which were preferentially accessible in ASXL1mt cells compared to ASXL1wt cells (p<0.05 for all regions). Four of these preferentially accessible genomic regions (p<0.001 for all regions) overlapped with the previously identified candidate regulatory regions for MEIS1. The H3K27ac and H2BK20ac occupancy levels in these regions was strongly associated with the expression of MEIS1 (median Pearson r=0.73, p=1.16x10-7). These candidate regulatory regions were annotated in ENCODE as distal and proximal enhancers. Targeting this biology with the dual p300/BRD4 inhibitor EP31670 in progenitor colony forming assays led to a preferential therapeutic effect in ASXL1mt BM samples (median IC50 = 25nM) compared to in ASXL1wt samples (median IC50 = 74nM, p=0.03). Treatment with EP31670 was also associated with increased differentiation (increased blast forming units-erythroid; BFU-E) and organization of colonies in ASXL1mt vs wt CMML.
 Conclusions: ASXL1mt CMML is characterized by the up-regulation of several leukemogenic driver genes including HOXA9 and its co-factor MEIS1, secondary to the activation of genotype-specific enhancers. The preferential accessibility of these enhancers can be defined between ASXL1mt and ASXL1wt cells within patients. The enhancers of interest interact with p300 and BRD4 in myeloid cell lines and targeting them with the novel dual p300/BRD4 inhibitor, EP31670, induces therapeutic responses at low nanomolar drug concentrations in vitro, preferentially in ASXL1-mutant cells. These data solidify the biological rationale for targeting p300/BRD4 in ASXL1mt chronic myeloid neoplasms for therapeutic benefit (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d58c09cc34747d2000f4bf32710a735358c8669" target='_blank'>
              Clone-specific epigenetic regulatory mechanisms in ASXL1-mutant chronic myelomonocytic leukemia
              </a>
            </td>
          <td>
            Moritz Binder, T. Lasho, Jenna A. Fernandez, Wazim Mohammed Ismail, Amelia Mazzone, C. Finke, Pankaj Pradeep, A. Mangaonkar, Nickolas Steinauer, Huihuang Yan, Liguo Wang, Alexandre Gaspar-Maia, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Colibactin is a secondary metabolite produced by bacteria present in the human gut and is implicated in the development of colorectal cancer. This genotoxin alkylates deoxyadenosines on opposite strands of host cell DNA to produce DNA interstrand cross-links. While cells have evolved multiple mechanisms to resolve (“unhook”) interstrand cross-links, little is known about which of these pathways promote resistance to colibactin. Here, we use Xenopus egg extracts to investigate replication-coupled repair of colibactin-induced interstrand cross-links. We show that replication fork stalling at a colibactin-induced interstrand cross-link activates the Fanconi anemia interstrand cross-link repair pathway, which unhooks the interstrand cross-link through nucleolytic incisions. These incisions generate a DNA double-strand break intermediate in one sister chromatid, which can be repaired by homologous recombination, and a monoadduct (“interstrand cross-link remnant”) in the other. Translesion synthesis past the colibactin-induced interstrand cross-link remnant depends on Pol η and the Pol κ-REV1-Pol ζ polymerase complex and introduces predominantly T>A point mutations at the sites of colibactin alkylation. Taken together, our work provides a molecular framework for understanding how cells tolerate a naturally occurring and clinically relevant interstrand cross-link. The bacterial genotoxin colibactin induces DNA interstrand cross-links which pose a barrier to DNA replication. Here, the authors use Xenopus egg extracts to show that the Fanconi anemia pathway is responsible for repairing these cross-links.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3af9af8aec073ccec4ec8bcc56779d2d58c22a2a" target='_blank'>
              The Fanconi anemia pathway repairs colibactin-induced DNA interstrand cross-links
              </a>
            </td>
          <td>
            Maria Altshuller, Xu He, Elliot J. MacKrell, Kevin M. Wernke, Yougant Airan, , Selene Sellés-Baiget, T. Wang, Tsui-Fen Chou, Julien P. Duxin, E. Balskus, Seth B. Herzon, Daniel R. Semlow
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Introduction: Extensive characterization of the genetic landscape in AML has revealed a set of driver mutations sufficient to initiate leukemic transformation. However, some cases do not contain any of these known drivers and thus pose a clinical challenge for prognostication and targeted therapy. To address this, we employed a novel, proximity ligation based, linked-read whole genome NGS assay to identify single nucleotide variants (SNVs), short Insertions/Deletions (InDels), copy number variation (CNV), structural variants (SVs) and 3D genome contact maps in AML. By employing this approach, we investigate whether comprehensive detection of genetic alterations in conjunction with insights into the epigenetic state of leukemic cells can reveal important biological information in cases lacking canonical driver mutations.
 Methods: We curated a cohort of patient samples where concomitant targeted sequencing by NGS and conventional karyotyping showed no detectable driver mutations. Further, we included control specimens with either known AML drivers as well as remission specimens after allogeneic transplantation. We utilized a multi-purpose NGS assay (Dovetail® LinkPrep™) to detect SNVs, InDels, CNVs, SVs and 3D genome contact maps. Peripheral blood or bone marrow cells were thawed and immediately fixed, subjected to crosslinking and in situ tagmentation, followed by library prep. Libraries were sequenced on Illumina sequencers with a target of 400M reads per library. Data analysis was performed using both published and custom tools to call SNVs/InDels (DeepVariant), Copy Number estimates (Purple), Structural Variants (HiC-Breakfinder, DTG-SELVA) and 3D contact maps (Juicebox).
 Results: We sequenced whole genome libraries to achieve an average coverage of 50.4x (range 41.1x - 58.1x) showing a long-range linkage percentage of 26.2% (range 17.9% - 31.8%). SNVs and short InDels, such as likely pre-leukemic mutations in DNMT3A, TET2, and mutations in GATA2 were detected reliably. Of note, the linked-read nature of this assay allowed us to perform phasing in a case with two TET2 mutations and confirm the compound heterozygous nature of these two TET2 mutations at a distance of 24kb. Further, known AML drivers in control specimens in genes such as NPM1, FLT3 and KRAS were reliably detected with variant allele frequencies as detected by targeted sequencing. Surprisingly, however, we found new structural variants in half of our cohort, including the diagnostic specimens with no detectable driver mutations, despite all specimens having no chromosomal abnormalities by conventional karyotyping. These new SVs included large intrachromosomal inversions and unbalanced translocations. We validated the presence of all SVs in sorted leukemic blasts and absence in sorted T cell fractions by PCR and Sanger sequencing. We subsequently examined the 3D genome contact maps and performed RNA-Sequencing to determine the epigenetic state and potential biological consequences across these novel breakpoints. We found several neoloops connecting enhancers with promoters across these breakpoints in 3D contact maps, and gene expression dysregulation proximal to the SVs by RNA-Seq. When we evaluated the expression of these putative driver genes across a cohort of healthy donors and AML patients, we found lack of expression in healthy cells but significantly increased expression in 9-10% of patients with AML (n=777 patients). Finally, we also identified an undetected structural variant in engrafted donor cells from a patient with myelodysplastic neoplasms (MDS) in remission after allogeneic transplantation, highlighting the potential to assay SVs in the context of non-leukemic hematopoiesis as well.
 Conclusion: In this exploratory cohort, we find several otherwise undetected structural variants in AML cases without known driver lesions using a proximity ligation based, linked-read whole genome NGS assay while maintaining our ability to detect recurrent, known driver lesions. This approach allowed us to interrogate the biological consequences of these genomic alterations by leveraging the ability to not only detect genomic variants but also analyze the epigenetic state through 3D genome contact maps. Our results imply that our current understanding of the genomic landscape in AML might be underestimating a sizable number of structural variants that are difficult to detect using standard assays, implying putative novel driver genes in AML pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f033c80701d9a0710865b0dce2b79cfc9a9b774" target='_blank'>
              Sensitive detection of novel structural variants and 3D chromosome conformation reveals likely novel drivers including enhancer hijacking in AML
              </a>
            </td>
          <td>
            Thomas Koehnke, A. Ediriwickrema, Mital S. Bhakta, Alexander Fortuna, Charu Tiwari, Zackary Sanborn, Tian Zhang, Lisa Munding, Ravi Majeti
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Ubiquitin-specific protease 1 (USP1) is a critical regulator of DNA damage response (DDR) pathways, exerting its function via the deubiquitination of key DNA repair substrates, including FANCD2 and proliferating cell nuclear antigen (PCNA). These deubiquitination events are essential for the functional integrity of the Fanconi anemia (FA) pathway and translesion DNA synthesis (TLS), both of which are vital for resolving DNA interstrand crosslinks and replication-associated DNA lesions. DNA-damaging agents, a broad class of therapeutics exert their cytotoxic effects by inducing various forms of DNA lesions, including base modifications, single-strand breaks (SSBs), double-strand breaks (DSBs), and DNA crosslinks. These lesions disrupt DNA replication and transcription, triggering replication stress, genomic instability, and ultimately cell death if left unrepaired. However, cancer cells often activate DDR pathways that coordinate DNA repair, cell cycle checkpoints, and survival responses, which can confer resistance to DNA-damaging treatments by enhancing lesion resolution and maintaining genomic integrity. Our previous studies showed that USP1 inhibition impaired the resolution of DNA lesions, resulting in persistent activation of DNA damage markers such as γH2AX. This indicates that USP1 inhibition could deepens the genotoxic stress imposed by DNA-damaging agents and promotes apoptotic cell death by blocking effective DNA repair In this study, we investigated whether pharmacological inhibition of USP1 could enhance the cellular response to DNA-damaging agents by disrupting key DNA repair mechanisms. Human cancer cell lines were treated with our proprietary USP1 inhibitor, VRTX531, either alone or in combination with DNA-damaging agents. VRTX531 treatment alone led to the accumulation of monoubiquitinated FANCD2 and PCNA, confirming effective USP1 inhibition and disruption of FA and TLS pathway activity. Combined treatment with VRTX531 and DNA-damaging agents resulted in a synergistic increase in cytotoxicity, as reflected by significantly reduced cell viability relative to DNA damage alone. These findings underscore the broader therapeutic potential of USP1 inhibition in not only sensitizing cancer cells to DNA-damaging agents but also overcoming chemo-resistance driven by DDR pathway activation.



 Prashant K. Bhavar, Uday K. Surampudi, Partha Pratim. Sarma, Appaji Mandhare, Neetu Singh, Nandini Narayani, Anuj Kshisagar. USP1 inhibition disrupts DNA repair and potentiates the cytotoxic effects of DNA-damaging agents via persistent DNA damage in cancer cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06d19eca0105605db82afdf528e4bb45ce22555e" target='_blank'>
              Abstract A034: USP1 inhibition disrupts DNA repair and potentiates the cytotoxic effects of DNA-damaging agents via persistent DNA damage in cancer cells
              </a>
            </td>
          <td>
            P. Bhavar, U. Surampudi, P. Sarma, A. Mandhare, Neetu Singh, Nandini Narayani, Anuj Kshisagar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bf9f556241d0007b43f3963aee0e9a310306573" target='_blank'>
              lncRNA EGFR-AS1 promotes DNA damage repair by enhancing PARP1-mediated PARylation.
              </a>
            </td>
          <td>
            Kun Gao, Ruiya Shi, Chenying Xu, Zhaoyang Mao, Changying Guo, Liang Jin
          </td>
          <td>2026-01-05</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The modulation of DNA replication dynamics has emerged as a key area of study in understanding genome stability and its perturbations in various physiological and pathological contexts. Replication fork rate is influenced by a variety of factors, including DNA repair pathways, origin firing, chromatin organization, transcription, and oncogenic signaling. This review highlights recent findings on the molecular mechanisms driving replication fork acceleration, focusing on scenarios such as PARP inhibition, oncogene activation, depletion of replication factors, and defects in Okazaki fragment processing. We discuss how reduced origin firing, R-loop resolution, and metabolic changes contribute to fork rate modulation, as well as the involvement of innate immune signaling, particularly through pathways such as cGAS-STING and ISG15. Special attention is given to consequences of accelerated replication forks for genome stability and their role in disease progression, particularly cancer. By unraveling the molecular mechanisms of fork acceleration, this Mini Review underscores its critical role in shaping genome integrity and cellular homeostasis, providing insights into future research directions and therapeutic strategies. A mini review explores molecular drivers of replication fork acceleration and their consequences for genome stability, innate immunity, and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edcef1b767fb22ecf30bb9d1778a6566d3ca870a" target='_blank'>
              The need for speed: drivers and consequences of accelerated replication forks
              </a>
            </td>
          <td>
            Dávid Lukáč, Katarína Chromá, Pavel Moudrý
          </td>
          <td>2025-11-06</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lamin A/C (LMNA), a key component of the nuclear envelope, is essential for maintaining nuclear integrity and genome organization [W. Xie et al., Curr. Biol. 26, 2651-2658 (2016)]. While LMNA dysregulation has been implicated in genomic instability across cancer and aging, the underlying mechanisms remain poorly understood [S. Graziano et al., Nucleus 9, 258-275 (2018)]. Here, we define a mechanistic role for LMNA in preserving genome stability in small-cell lung cancer (SCLC), a malignancy marked by extreme genomic instability [N. Takahashi et al., Cancer Res. Commun. 2, 503-517 (2022)]. LMNA depletion promotes R-loop accumulation, transcription-replication conflicts, replication stress, DNA breaks, and micronuclei formation. Mechanistically, LMNA deficiency disrupts nuclear pore complex organization, specifically reducing phenylalanine-glycine (FG)-nucleoporin incorporation, resulting in impaired RNA export and nuclear retention of RNA. LMNA expression is repressed by EZH2 and reexpressed during SCLC differentiation from neuroendocrine (NE) to non-NE states, and low LMNA levels correlate with poor clinical outcomes. These findings establish LMNA as a key regulator of nuclear transport and genome integrity, linking nuclear architecture to SCLC progression and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1b936a6a58b416fb0e62e291846a3a4d6fb1c3" target='_blank'>
              Lamin A/C loss promotes R-loop-mediated genomic instability and poor survival in small-cell lung cancer.
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Ravi P Shuklah, Robin Sebastian, Umeshkumar M Vekariya, Shahbaz Ahmed, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, V. Aksenova, Elizabeth Giordano, N. Takahashi, Michael Nirula, Mohit Arora, Chiori Tabe, M. Thomas, Rajesh Kumar, Yashuhiro Arakawa, Ukhyun Jo, Tomasz Skorski, Beverly A. Teicher, Roshan Shreshta, M. Aladjem, Stephen Lockett, M. Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-10-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="BACKGROUND
Cyclin-dependent kinase 12 (CDK12) regulates general gene transcription elongation, and plays multiple roles in RNA splicing, DNA damage-response, cell cycle and genomic stability. However, transcriptional partners that guide CDK12-specific gene programs have not been identified. Genomic alterations in CDK12 have been observed in multiple cancers, exhibiting both pro-tumorigenic and tumor-suppressive functions, suggesting a context-dependent mechanism of action.


METHODS
CDK12 copy number alterations and gene expression levels were analyzed in matched primary and brain metastatic patient tumors. Clinical significance was assessed by immunohistochemistry in a large cohort of primary breast cancer patient tumors. RNA-sequencing, ChIP-sequencing and molecular studies were conducted to explore CDK12's mechanism of action, and pharmacological studies were performed both in vitro and in vivo using models of advanced (endocrine-resistant and metastatic) estrogen receptor positive (ER+) disease.


RESULTS
CDK12 amplifications and gene overexpression were observed in brain metastatic tumors. In ER+ primary patient tumors, high CDK12 protein expression significantly associated with poor overall survival, particularly within the ER+/HER2-negative group. In ER+ endocrine resistant models, CDK12 regulated estrogen signaling pathways, with ER/MED1 identified as the master transcriptional complex directing CDK12-specific pro-tumorigenic gene programs. Pharmacological inhibition of CDK12 significantly reduced viability in endocrine resistant and metastatic cell and organoid models in vitro, and decreased metastatic spread in vivo.


CONCLUSION
This work describes a novel mechanism for CDK12, suggesting a potential vulnerability in ER+ breast cancer. These findings provide a basis for further investigation into the role of CDK12 inhibition as a therapeutic approach, particularly in advanced disease settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af116245324e2c76800300a12fb01f632badb6d3" target='_blank'>
              Targeting CDK12 disrupts estrogen-receptor chromatin recruitment and ER-MED1 transcription in advanced ER+ breast cancer.
              </a>
            </td>
          <td>
            Daniela Ottaviani, Mihaela Ola, A. Allotta, Yasmine Maati Chaibi, Seán Hickey, Petra Jagust, Nicola Cosgrove, S. Cocchiglia, F. Bane, Ramón Fallon, Gordon Daly, A. Hegarty, Lance Hudson, Katherine Sheehan, Shannon Kalsi, Stephen Shovlin, Aoibhín M Powell, Ashwani K. Bahl, Ed Ainscow, S. Oesterreich, Adrian V. Lee, F. Couch, Arnold D K Hill, D. Varešlija, Leonie S Young
          </td>
          <td>2025-10-15</td>
          <td>Journal of the National Cancer Institute</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Tardigrades exhibit extraordinary ionizing radiation (IR) resistance, with DNA repair as a core mechanism, but interactions between classical repair factors and tardigrade-specific proteins remain unclear. The non-homologous end joining (NHEJ) core factor XRCC5 (Ku80) is strongly upregulated post-IR and critical for survival. TDR1, a tardigrade-specific intrinsically disordered protein, is IR-induced, co-localizes with DNA damage foci, binds DNA in vitro, and is synchronously overexpressed with XRCC5 under IR, suggesting potential synergy. Using Hypsibius exemplars, this study conducted two experiments: 1) Radiation gradient treatment CRISPR/Cas9-mediated TDR1/XRCC5 knockout to measure -H2AX (DNA double-strand break marker) and counterpart gene mRNA/protein at 4 h post-2000 Gy IR. Predicted results: Low-dose IR triggers XRCC5 (4 h) but not TDR1; medium-high-dose IR upregulates both (4 h). TDR1 knockout may reduce XRCC5 protein and increase -H2AX; XRCC5 knockout may not affect TDR1 but cause persistent -H2AX.This study clarifies TDR1-XRCC5 synergy in IR-induced DNA repair, illuminating tardigrades radiation resistance mechanisms and informing radio-sensitive system optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd7968ce6d15f78681776f4a0b93359ef37c2fd8" target='_blank'>
              The Functional Interplay Between Tardigrade-Specific IDP TDR1 and XRCC5 in Radiation Resistance
              </a>
            </td>
          <td>
            Haoran Hao
          </td>
          <td>2025-11-27</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Transposable elements (TEs) make up half of the human and mouse genome but remain understudied. Evidence is growing that TEs play important roles in immunity as a subset of TE enhancers regulate innate immunity genes. However, the cis-regulatory contribution of TEs across immune cell development is not well understood. Mining more than 300 chromatin accessibility datasets (ATAC-seq) across mouse hematopoietic cell lineages, we identified two subfamilies of rodent-specific retroviral LTRs, ORR1E and ORR1D2 (ODE), which are enriched in accessible chromatin across all immune cell types. Unsupervised clustering show that ∼2,900 ODEs stratify by cell type-specific accessibility. Each ODE cluster is enriched for a unique combination of sequence motifs for cell lineage-specifying TFs, such as PU.1, IRF8, and RORγ, which we validated with ChIP-seq. By correlating ODE accessibility with gene expression (RNA-seq), we predicted hundreds of ODE-gene interactions. We generated Micro-C (DNA-DNA contact) in CD8+ T cells and validated ∼120 ODE-gene interactions. Predicted target genes are more expressed in mouse than in human CD8+ T cells. Lastly, accessible ODEs have higher sequence conservation across rodents than other ODEs. Together, these data suggest that despite their common origin, ODE LTRs have subfunctionalized into cell type-specific enhancers across mouse hematopoiesis. We propose that ODE enhancers were co-opted during rodent evolution to reinforce cell type identity.



 F31AI183775



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c25d5af8adc02b84a26acb002fe252a06d958df" target='_blank'>
              Functionalization of transposable element-derived enhancers across mouse hematopoiesis 3835
              </a>
            </td>
          <td>
            Jason D. Chobirko, Elizabeth A. Fogarty, Cedric Feschotte, Andrew Grimson
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mycobacterium tuberculosis (Mtb), the causative agent of tuberculosis (TB), is estimated to infect nearly one-quarter of the global population. A key factor in its resilience and persistence is its robust DNA repair capacity. Non-homologous end joining (NHEJ) is the primary pathway for repairing DNA double-strand breaks (DSBs) in many organisms, including Mtb, where it is mediated by the Ku protein and the multifunctional LigD enzyme. In this study, we demonstrate that Ku is essential for mycobacterial survival under DNA-damaging conditions. Using cryogenic electron microscopy (cryo-EM), we solved high-resolution structures of both the apo and DNA-bound forms of the Ku-Mtb homodimer. Our structural and biophysical analyses reveal that Ku forms an extended proteo-filament upon binding DNA. We identify critical residues involved in filament formation and DNA synapsis and show that their mutation severely impairs bacterial viability. Furthermore, we propose a model in which the C-terminus of Ku regulates DNA binding and loading and facilitates subsequent recruitment of LigD. These findings provide unique insights into bacterial DNA repair and guide future therapeutics. Mycobacterium tuberculosis protein Ku is involved in DNA repair and a potential drug target. Here, using cryo-EM and complementary approaches, the authors obtain insights into Ku oligomerization and mechanisms of function in DNA synapsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a761f6b26651521fdd3367949d0523c2f80c9db2" target='_blank'>
              Oligomerisation of Ku from Mycobacterium tuberculosis promotes DNA synapsis
              </a>
            </td>
          <td>
            Sayma Zahid, S. Baconnais, Henrietta Smith, Saseela Atwal, Lucy Bates, Harriet Read, Ankita Chadda, Florian Morati, Tom Bedwell, Emil G. P. Stender, Joanne Walter, Steven W. Hardwick, Fredrik Westerlund, E. Galburt, É. Le Cam, Alice L. B. Pyne, G. Mukamolova, Amanda K. Chaplin
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Introduction: Ribosomal DNA (rDNA) encodes ribosomal RNA (rRNA), which undergoes processing to become ribosomes. Although rDNA dysregulation has been observed in various pathologies, comprehensive epigenetic profiling in cancer has been limited due to its repetitive sequence. To overcome this challenge, a customized genome has been developed to map human and mouse rDNA using high-throughput sequencing data. Using this genome and more than 2,200 ChIP-seq datasets, Antony et al performed a systematic analysis to identify the transcription factors, e.g., C/EBP-A, that support rRNA production in hematopoietic cells. Our previous publications suggested that DNA cytosine modifications influence genomic binding of C/EBP. Accumulated studies, including our own, have reported aberrant DNA methylation and hydroxymethylation in myeloid malignancies. Yet, it remains unclear whether the leukemia-associated DNA methylation dysregulation extends to genomic regions encoding rRNA (rDNA), and how these epigenetic changes in rDNA affect downstream targets.
 Methods: To address these critical knowledge gaps, we employed innovative rDNA assembly methods, along with a DNA methylation analysis pipeline, to evaluate the whole-genome bisulfite sequencing (WGBS) data from 136 cancer patients covering 9 cancer types, aiming to uncover DNA methylation changes in rDNA in cancer. To further investigate unique patterns observed in hematological malignancies, we expanded our analysis to a dataset containing DNA methylation profiles from 292 AML patients. Additional AML datasets were analyzed to validate our initial findings with integrative methylation analysis methods. Furthermore, we analyzed published nanopore long-read data to independently confirm the methylation status of rDNA.
 Results: Our pan-cancer analysis revealed several patterns of rDNA methylation across cancer types, often at odds with non-rDNA methylation patterns. Intriguingly, we discovered that hematological malignancies exhibited distinct DNA methylation signatures in rDNA, with higher DNA methylation in the rDNA-coding region compared with other cancer types. This finding was validated in multiple independent datasets totaling 484 AML samples. Additionally, long-read nanopore sequencing verified these methylation patterns across the entire rDNA tandem array. Further characterization of this hypermethylation in AML patient samples revealed CpG loci that show highly conserved changes. A significant correlation was observed between hypermethylation at these loci and decreased time-to-relapse, suggesting that increased rDNA coding region methylation may serve as a biomarker. Interestingly, along with prognostic significance related to time-to-relapse, we revealed distinct patterns of rDNA methylation in patients treated with hypomethylating agents (HMA). To study potential mechanisms, we first validated published findings that many transcription factors exhibit consistent binding patterns on rDNA. To elucidate the connection between DNA methylation and transcription factor binding within rDNA, we first performed a motif analysis of the rDNA sequence, revealing 4000+ canonical binding motifs. We then used our large AML datasets to identify motifs with differential methylation between healthy bone marrow and AML patients, both at diagnosis and relapse. Across both datasets, we identified several potential transcription factors, e.g., CEBP families, for downstream analysis. We will use multiple leukemia cell lines to validate the correlation between DNA methylation and TF binding at rDNA. We will further use catalytically dead Cas9 (dCas9) mediated epigenome editing to probe the causal relationship to support our hypothesis that rDNA methylation can influence TF binding.
 Conclusions: Our data revealed distinct DNA methylation signatures in rDNA specific to myeloid leukemia. Using multiple published DNA methylation datasets from AML patients, we identified unique DNA methylation changes in rDNA, highlighting its potential diagnostic and prognostic value. At the molecular level, we observed differential DNA methylation levels within several TF binding motifs, which might alter TF binding and impact their transcriptional activity in rDNA, ultimately contributing to aberrant ribosomal biogenesis. Overall, our study provides a comprehensive analysis of previously neglected DNA methylation changes in rDNA in AML, uncovering novel epigenetic mechanisms with potential clinical relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba5a2d1360b7125af2d5a2084348209fa94e181" target='_blank'>
              Hypermethylation of the ribosomal DNA coding sequence and disordered transcription factor binding as hallmarks of acute myeloid leukemia
              </a>
            </td>
          <td>
            James Wengler, Y. Huang, Leng Han, Lei Guo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, Michael Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species.
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant childhood cerebellar tumor comprising molecularly and clinically distinct subgroups. Groups 3 and 4-MB originate from overlapping progenitor pools of the developing rhombic lip when neuronal differentiation hierarchies are subverted through somatic alterations. Mutations suspected to deregulate the activity of chromatin-modifying genes are pervasive in Group 3/4-MB. However, molecular consequences of these alterations remain largely undefined. We characterized the chromatin landscape of fifty-two primary MBs using chromatin immunoprecipitation followed by sequencing (ChIP-seq) for six histone modifications and performed multi-modal integration with sample-matched somatic lesions, genome-wide DNA methylation, and transcriptomic profiles. Analysis of differential chromatin states across MB subgroups revealed significant enrichment of the bivalent enhancer state (EnhBiv; marked by coincident H3K4me1 and H3K27me3) in Group 3/4-MB overlapping neurodevelopmental genes. Integrative bioinformatics coupled with CUT&RUN analysis revealed significant enrichment of KDM2B, a histone lysine demethylase overexpressed in Group 4-MB, in gene promoters marked by the EnhBiv state. CRISPR gene targeting or targeted protein degradation of KDM2B selectively suppressed growth of Group 3 and Group 4-MB models in vitro and in vivo, which was dependent on KDM2B DNA-binding domain but not demethylase activity. Acute and chronic KDM2B degradation impaired EZH2 recruitment through direct physical interactions and indirect KDM2B-mediated PRC1 activity, reducing H3K27me3 deposition and chromatin bivalency, while profoundly derepressing PRC2 targets involved in neuronal differentiation. Our comprehensive characterization of the MB chromatin landscape in a large cohort of primary tumors provides unprecedented insights into the epigenetic basis of MB subgroups, implicating novel mechanisms of tumorigenesis, and disclosing a unique subgroup-specific dependency conferred by KDM2B-mediated Polycomb activity that warrants consideration as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff433ac7a60b68bf5a2b5c1d8d57a287494ba1b0" target='_blank'>
              EPCO-22. DIVERGENT MEDULLOBLASTOMA CHROMATIN STATES DISCLOSE KDM2B AS A SELECTIVE DEPENDENCY
              </a>
            </td>
          <td>
            Ran Tao, Serap Erkek-Ozhan, Beisi Xu, Yurika Matsui, Priya Mittal, Kyle S Smith, Yiran Li, Qingsong Gao, Emily Darrow, Ruijie Xu, Nadhir Djekidel, Richa Bajpai, Rahul Kumar, Colleen Reilly, T. Soliman, Jennifer L. Hadley, Melissa Batts, Natarajan Bhanu, B. Gudenas, Leena Paul, Hong Lin, K. Lowe, L. Kutscher, Benjamin Garcia, S. Pruett-Miller, Xin Zhou, Brent A. Orr, G. Robinson, Gang Wu, Jan O. Korbel, Jamy C Peng, Stefan M. Pfister, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 While radiation therapy offers strong anti-cancer benefits, it can also have serious side effects for children undergoing brain tumor treatment. These include disruptions in white matter development, delays in functional and cognitive growth, and an increased risk of secondary tumors within the area that was treated. Age-dependent susceptibility to these adverse effects is tightly linked to critical windows of neurodevelopment, particularly during the periods of active neural cell differentiation, maturation, and the rapid onset of myelination in infancy and early childhood. We previously demonstrated that oligodendrocyte precursor cells (OPCs) exhibit heightened sensitivity to IR-induced DNA damage compared to other neural stem/progenitor cells (NSPC) populations – a difference that is associated with their decreased ability to form RAD51 filaments (a critical step in RAD51-mediated homologous recombination repair of DNA double-strand breaks). Recently, we performed genome-wide chromatin immunoprecipitation followed by sequencing (ChIP-seq) to map endogenous γH2AX binding sites—a sensitive marker of DNA damage response—in both S-phase OPCs and NSPCs. Our data identified 71,794 γH2AX-enriched regions in OPCs versus 29,154 in NSPCs, highlighting the elevated genotoxic stress burden in OPCs. Interestingly, we observed γH2AX binding is depleted at transcription start sites and transcription termination sites, however OPCs show more γH2AX peaks compare to NSPCs across all protein-coding genes. Furthermore, our data show cell type specific γH2AX binding sites. Together, these findings establish a comprehensive and unbiased genomic framework for elucidating the mechanisms underlying OPC vulnerability to DNA damage. Future studies will examine different DNA damage pattern in response to radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d211959973f2a6f48caa461ee162af3afdec9c4b" target='_blank'>
              DNAR-11. Differential DNA Damage and Repair burden in proliferating neural progenitor cells
              </a>
            </td>
          <td>
            Askar Yimit, Peter Brownlee, Sean Nesdoly, Sorana A. Morrissy, Jennifer A Chan, Aaron Goodarzi
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c7b8cd9fd3a432269be76a19451b459fe1ecd" target='_blank'>
              EBNA2 and EBNA-LP: The Earliest Viral Latency Proteins.
              </a>
            </td>
          <td>
            Jana M Cable, Jenna C Grabowski, Micah A. Luftig
          </td>
          <td>2025-12-03</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a cancer of unmet need. Identifying biomarkers for patient stratification and prediction of response to therapy is a priority. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere attrition has been shown to drive genome instability and clonal evolution in several cancers. We examined the relationship between telomere dysfunction, fusion and clinical outcome in GBM patients.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR coupled with long-read nanopore sequencing was used to characterise telomere fusions.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009), this was more pronounced in patients with complete resection at index surgery (HR 21.55, p=0.0074). Widespread telomere fusions were observed that revealed microhomology, subtelomeric deletions and insertions of genomic loci within complex chromosomal rearrangements. Insertions often included protein-coding genes with driver mutations, long non-coding RNA, telomeric repeats, and subtelomeric and peri-centromeric regions. This mutational spectrum suggested utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The data defines patient subgroups (MGMT unmethylated, short TL) which may benefit from alternatives to TMZ, and aberrant DDR signatures could help inform stratification within adaptive platform trials of DDR inhibitors. Results from a prospective cohort of all-grade diffuse gliomas with 5-ALA and image-guided multi-site sampling will be shared at this meeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccc6bd14a72e4b9daf2cb4615f11e794725f9f98" target='_blank'>
              DNAR-09. Telomere dysfunction and fusions in glioblastoma - defining prognosis and genome instability
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, William Gray, Kathreena M Kurian, D. Baird
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The incidence of colorectal cancer in young adults (age of diagnosis <50 years) has been rapidly increasing. Although ∼20% of early-onset colorectal cancer (EOCRC) cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Nongenetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as alterations in the epigenome, microbiome, and immunome. We characterized the DNA methylation (DNAm) signature and measured DNAm age in EOCRC by using The Cancer Genome Atlas (TCGA). Furthermore, we carefully identified intratumoral microbes from TCGA and the Oncology Research Information Exchange Network datasets and then related the microbes to deconvolved immune cell abundances in EOCRC. We observed that the DNAm age in the EOCRC cohort was 12 years older when compared with the average-onset colorectal cancer (AOCRC) cohort, using three different epigenetic clocks. Differentially methylated sites associated with gene expression include cAMP-responsive element binding protein signaling in neurons, G protein–coupled receptor signaling, phagosome formation, and S100 family signaling. These differences were validated in the gene expression data from TCGA and the Oncology Research Information Exchange Network. When comparing the intratumoral microbes between EOCRC and AOCRC, no consistent differences were observed. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more and larger positive correlations in EOCRC. These data suggest that epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. Significance: We investigated whether environmentally driven factors contribute to EOCRC. We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than in AOCRC. These data suggest a dysregulation of the immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c12561621c15cb4ece79f124013162ea3c517b" target='_blank'>
              Epigenetic Modulation, Intratumoral Microbiome, and Immunity in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            N. Jin, R. Hoyd, Ayse S Yilmaz, Jiangjiang Zhu, Yunzhou Liu, Malvenderjit S Jagjit Singh, Dennis J. Grencewicz, Xiaokui Mo, Matthew F Kalady, Daniel W Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos H F Chan, M. Churchman, N. Denko, Frances Di Clemente, Rebecca D. Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, Alexandra P Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, Afaf E G Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A. Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, A. Tan, Daniel J. Spakowicz
          </td>
          <td>2025-10-24</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de768ce4bdb5a4b567cbf3cdac89e7caf46688f" target='_blank'>
              Genomic and transcriptomic dynamics in the stepwise progression of lung adenocarcinoma.
              </a>
            </td>
          <td>
            F. Fu, Jun Shang, Yueren Yan, He Jiang, Han Han, Hui Yuan, Zhendong Gao, Jingcheng Yang, Jian Gao, Jun Wang, Yunjian Pan, Yicong Lin, Ting Ye, Yiliang Zhang, Yawei Zhang, Jiaqing Xiang, Hong Hu, Zhiwei Cao, Yuanting Zheng, Yuan Li, Yang Zhang, Li Jin, Leming Shi, Haiquan Chen
          </td>
          <td>2025-12-04</td>
          <td>Cell research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abnormal regulation of microRNA (miRNA) production is increasingly recognized as a hallmark of cancer development. Genetic alterations, disrupted transcriptional control, and deficiencies in processing machinery all contribute to changes in miRNA levels. Depending on the cellular environment, a single miRNA may act either in favor of tumor growth or as a barrier against malignancy. These small RNAs influence key oncogenic features, including uncontrolled cell division, suppression of growth-inhibitory pathways, evasion of programmed cell death, metastatic spread, and angiogenesis. Because of this broad impact, numerous studies have explored the potential of miRNAs as biomarkers for early diagnosis and prognosis, although further validation is still required before clinical translation. Among these regulators, hsa-miR-1296 has recently gained attention. Both strands of its precursor, miR-1296-5p and miR-1296-3p, are involved in modulating multiple signaling cascades across different tumor types. Depending on tissue context, miR-1296 can function as either a tumor suppressor or, less frequently, as an oncogenic factor by targeting genes central to cell-cycle progression, migration, invasion, and therapy resistance. Evidence from breast, lung, liver, colorectal, gastric, prostate, and bone cancers shows that miR-1296 interacts with networks of protein-coding genes and noncoding RNAs, shaping tumor behavior and influencing patient outcomes. Relevant studies were identified through searches in PubMed and Scopus databases up to 2025, focusing on experimental and clinical investigations of miR-1296 in human cancers and its ceRNA-mediated regulatory mechanisms. This review summarizes the current knowledge of miR-1296 biology and highlights its dual role in cancer, as well as its emerging value as a diagnostic and therapeutic candidate for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840b03bfeaf1a1d94eb0ab4a83679a414d0bc0e8" target='_blank'>
              miR-1296 in Human Cancers: Context-Dependent Tumor Suppressor and Oncogene Orchestrated by ceRNA Networks
              </a>
            </td>
          <td>
            Zahra Diba
          </td>
          <td>2025-10-28</td>
          <td>Journal of Cancer Biomoleculars and Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="RhoGDI2 is a RhoGTPase regulator that has roles in cytoskeleton organization and cell survival, amongst others. It is differentially expressed in many cell types and tissues, including several human cancers, where its expression has been correlated with either good or bad prognosis. To identify the underlying mechanisms, we knocked down its expression in human cancer cell lines. We observed that repression of RhoGDI2 expression, but not that of the closely related RhoGDI1, significantly reduces their proliferation rate. In parallel, RhoGDI2 suppression induces supernumerary centrosomes and inhibits ciliogenesis. As RhoGDIs are regulators of GTPases, we checked whether key RhoGTPases are involved in these effects. We found that silencing RhoA partially rescued the induction of supernumerary centrosomes and ciliary defects observed upon RhoGDI2 silencing. It was previously shown that RhoGDI2 is strongly expressed in immune cells and that there are striking similarities between primary cilia and immune synapses. Based on this knowledge, we silenced RhoGDI2 in NK cells and could demonstrate that this strongly affects their immune synapse-related cancer cell-killing activity. Altogether, these data suggest novel roles for RhoGDI2 in centrosome functions in human cancer and immune synapses in immune cells, which provides an explanation for its reported dual role in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c0ac8c4a86b7bca62ea906513e7218a435f267" target='_blank'>
              Reduced RhoGDI2 Expression Disrupts Centrosome Functions and Promotes Mitotic Errors
              </a>
            </td>
          <td>
            M. Tripathi, N. Garbacki, J. Willems, G. Cobraiville, Marianne Fillet, Alain Colige, Christophe F. Deroanne
          </td>
          <td>2025-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND Colorectal cancer (CRC) is the second leading cause of cancer-related death, largely due to limited treatment options in advanced stages. Genomic alterations in advanced CRC (aCRC) are complex and not fully characterized, with only 30% of patients benefiting from targeted therapies. AIM To investigate the molecular heterogeneity of primary aCRC in order to identify clinically relevant genomic alterations. METHODS We conducted a retrospective molecular analysis of 73 consecutive patients with histologically confirmed primary aCRC (stage pT4a-b). All molecular findings were correlated with available clinicopathological data. In addition, we performed survival analyses using publicly available datasets and tools. RESULTS Genetic abnormalities identified in primary tumors were most frequently mutations in tumor protein p53 (58% of cases), Kirsten rat sarcoma viral oncogene homolog (52%), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (25%), B-Raf kinase (11%) and fibroblast growth factor receptor 3 (8%), as well as R-spondin 3 (RSPO3) fusions (8%). Alterations in the tumor protein p53 and neuroblastoma RAS viral oncogene homolog genes were predominantly observed in tumors from the left colon, whereas B-Raf kinase mutations and RSPO3 fusions were more frequently detected in the right or transverse colon. We also show a strong association between the presence of RSPO3 rearrangements and patients with small tumors, normal carcinoembryonic antigen levels, and microsatellite stable tumors. Furthermore, aCRC patients with protein tyrosine phosphatase receptor type k::RSPO3 fusions exhibited a higher mortality rate. Elevated RSPO3 gene expression levels were also significantly correlated with poorer OS across two large, independent CRC cohorts. CONCLUSION This study identifies a relatively high incidence of RSPO3 rearrangements in aCRC and a strong association with clinical features. Furthermore, we find that RSPO3 fusions are associated with poorer OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b1b9374204622db5b697e06c7d6c2484792d29" target='_blank'>
              RSPO3 rearrangements in advanced colorectal cancer patients and their relationship with disease characteristics
              </a>
            </td>
          <td>
            Raquel Tur, M. Abad, Elena Filipovich, Maria Belen Rivas, Marta Rodriguez, Juan Carlos Montero, J. Sayagués
          </td>
          <td>2025-11-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="There is an urgent need to find targeted agents for T-cell acute lymphoblastic leukemia (T-ALL). NOTCH1 is the most frequently mutated oncogene in T-ALL, but clinical trials showed that pan-Notch inhibitors caused dose-limiting toxicities. Thus, we shifted our focus to ETS1, which is one of the transcription factors that most frequently co-bind Notch-occupied regulatory elements in the T-ALL context. To identify the most essential enhancers, we performed a genome-wide CRISPR interference screen of the strongest ETS1-dependent regulatory elements. The #1-ranked element is located in an intron of AHI1 that interacts with the MYB promoter and is amplified with MYB in ~8.5% of T-ALL patients. Using mouse models, we showed that this enhancer promotes self-renewal of hematopoietic stem cells and T-cell leukemogenesis, maintains early T-cell precursors, and restrains myeloid expansion with aging. We named this enhancer the hematopoietic stem cell MYB enhancer (H-Me). The H-Me shows limited activity and function in committed T-cell progenitors but is accessed during leukemogenesis. In one T-ALL context, ETS1 binds the ETS motif in the H-Me to recruit cBAF to promote chromatin accessibility and activation. ETS1 or cBAF degraders impaired H-Me function. Thus, we identified a targetable stem cell element that is co-opted for T-cell transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b0d046a416194ebc6608468d656972dd5460330" target='_blank'>
              The hematopoietic stem cell MYB enhancer is essential and recurrently amplified during T-cell leukemogenesis.
              </a>
            </td>
          <td>
            Carea Mullin, Karena Lin, Elizabeth Choe, Cher Sha, Zeel Shukla, Koral Campbell, Anna C. McCarter, Annie Wang, Jannaldo Nieves-Salva, Sarah Khan, Theresa M. Keeley, Shannon Liang, Qing Wang, A. Melnick, Pearl Evans, Alexander C. Monovich, Ashwin Iyer, R. Kodgule, Yamei Deng, Felipe da Veiga Leprevost, Kelly R. Barnett, P. Pölönen, R. Khoriaty, Daniel Savic, D. Teachey, C. Mullighan, Marcin Cieslik, Alexey I. Nesvizhskii, Linda C. Samuelson, Morgan Jones, Qing Li, Russell J H Ryan, M. Chiang
          </td>
          <td>2025-10-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, , Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a global health challenge, with limited treatment options and a high mortality rate. We recently found that lung cancer cells that express more genes that are typically restricted to the germline (germ cell cancer genes, GC genes) repair DNA double-strand breaks more rapidly, show higher rates of proliferation, and are more resistant to ionizing radiation, compared to cells that express fewer GC genes. Moreover, we found that the gene encoding TRIP13 (thyroid hormone receptor interactor 13) plays a significant role in this malignant phenotype. Here, we demonstrate that melatonin (MT), a hormone synthesized in the pineal gland, downregulates the expression of TRIP13, particularly in lung cancer cells with high expression of TRIP13. Moreover, this downregulation of TRIP13 by MT further inhibits the DNA repair proteins RAD51 and XRCC5, thereby impairing DNA repair via homologous recombination and non-homologous end joining. We further found that the melatonin receptor 1B (MTNR1B), rather than melatonin receptor 1 A (MTNR1A), is essential for MT mediated TRIP13 downregulation. Because we also found that treatment with MT still decreases cell proliferation of TRIP13-KO cells, combining MT with the TRIP13 inhibitor DCZ0415 would likely have an additive anti-proliferative therapeutic effect in the treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Jumping translocations (JTs) can lead to partial trisomies. A breakpoint within the gene known as Jumping Translocation Breakpoint (JTB) has previously been associated with JTs involving the long arm of human chromosome 1 (1q). These 1q+ amplifications are frequently observed in cancer. JTB was initially mapped to the Epidermal Differentiation Complex (EDC) at 1q21, and earlier studies primarily focused on its role in malignant or adult tissues. Using updated genomic data, we refined the mapping of JTB. We employed RNA in situ hybridization (RISH) to visualize Jtb expression with organ, tissue, and cell-level resolution. We demonstrate that human JTB and murine Jtb are located outside the EDC. In midgestational wild-type mouse embryos, Jtb is expressed in multiple tissues, including the developing heart and vertebral column, and shows partial overlap with the expression of early markers of the neural crest cell lineage. Our findings suggest that the oncogenic potential associated with human JTB translocations is likely unrelated to its previously assumed location within the EDC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed6323abd411d00d62b1282d50fbc327ee57f1b" target='_blank'>
              Jumping Translocation Breakpoint Expression in Midgestation Mouse Embryos
              </a>
            </td>
          <td>
            Carley McGrath, Taniya M Jayaweera, T. Lufkin, Costel C. Darie, A. Neagu, Petra Kraus
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Radiotherapy is essential in the treatment of colorectal cancer (CRC), but the presence of drug resistance leads to poor prognosis for CRC patients. Identifying targets and mechanisms for regulating radiotherapy resistance has high clinical value. This study identifies CCR4-NOT transcription complex subunit 7 (CNOT7) as a key factor mediating radiotherapy resistance in CRC by stabilizing XRCC6 protein and enhancing non-homologous end joining (NHEJ) mediated DNA damage repair (DDR) pathway. Proteomic analysis of 45 CRC tissues revealed that elevated CNOT7 expression correlates with poorer responses to neoadjuvant radiotherapy and lower disease control rate (DCR). We demonstrated that CNOT7 knockdown enhances radiosensitivity by impairing NHEJ mediated double-strand breaks (DSBs) repair and promoting apoptosis in vitro and in vivo. Mechanistically, CNOT7 interacts with XRCC6 to stabilize its protein levels by inhibiting TRIM21-mediated K48-linked ubiquitination at lysine 526, thereby facilitating efficient DNA repair. CNOT7 accelerates degradation of TRIM21 mRNA through its deadenylase activity. Additionally, the combination of STL127705, an inhibitor of the XRCC6/XRCC5 heterodimer, with radiotherapy notably suppressed tumor growth in patient-derived xenograft (PDX) and cell line mouse transplant tumor models, especially in the context of CNOT7 deficiency. These findings elucidate the function of CNOT7 in promoting DNA repair and radiotherapy resistance in CRC, highlighting that targeting the CNOT7-TRIM21-XRCC6 axis provides a promising therapeutic approach to overcome radiotherapy resistance and improve clinical outcomes for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82682cf583ba655b911eee025ad39a84a6800b0e" target='_blank'>
              CNOT7 facilitates radiation resistance in colorectal cancer through TRIM21/XRCC6-mediated non-homologous end joining repair
              </a>
            </td>
          <td>
            Yien Li, Luying Cui, Shaoke Wang, Qunye Zhao, Fenqi Du, Songtao Du, Chenfeng Yu, Mingyu Xia, Shihui Zhao, Tian Luan, Yanlong Liu, Binbin Cui
          </td>
          <td>2025-11-17</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Telomere length maintenance is strongly linked to cellular aging, as telomeres progressively shorten with each cell division. This phenomenon is well-documented in mitotic, or dividing, cells. However, neurons are post-mitotic and do not undergo mitosis, meaning they lack the classical mechanisms through which telomere shortening occurs. Despite this, neurons retain telomeres that protect chromosomal ends. The role of telomeres in neurons has gained interest, particularly in the context of neurodegenerative diseases such as amyotrophic lateral sclerosis (ALS), where aging is a major risk factor. This has sparked interest in investigating telomere maintenance mechanisms in post-mitotic neurons. Nevertheless, most existing telomere analysis techniques were developed for and optimized using mitotic cells, posing challenges for studying telomeres in non-dividing neuronal cells. Thus, this protocol adapts an already established technique, the combined immunofluorescence and telomere fluorescent in situ hybridization (IF-FISH) on mitotic cells to study the processes occurring at telomeres in cortical neurons of the mouse ALS transgenic model, TDP-43 rNLS. Specifically, it determines the occurrence of DNA damage and the alternative lengthening of telomeres (ALT) mechanism through simultaneous labeling of the DNA damage marker, γH2AX, or the ALT marker, promyelocytic leukemia (PML) protein, together with telomeres. Therefore, the protocol enables the visualization of DNA damage (γH2AX) or the ALT marker (PML) concurrently with telomeres. This technique can be successfully applied to brain tissue and enables the investigation of telomeres specifically in cortical neurons, rather than in bulk tissue, offering a significant advantage over Southern blot or qPCR-based techniques. Key features • This protocol enables the labeling of telomeres in mouse brain tissue prepared from paraffin-embedded brain sections. • This method facilitates concurrent labeling of proteins that are colocalized at telomere sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1e2280b8fd8c35a0ab81f6940db9809bdef4684" target='_blank'>
              Colocalizing Telomeres With PML or γH2AX Foci by IF-FISH in Mouse Brain Neurons
              </a>
            </td>
          <td>
            Anna Konopka
          </td>
          <td>2025-11-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Background:
 Chronic lymphocytic leukemia (CLL) is a heterogeneous B-cell malignancy characterized by clonal expansion and immune dysfunction. Although peripheral blood (PB) is frequently used for genomic and transcriptomic profiling, it may not fully capture the biological activity of the disease. The lymph node (LN) microenvironment plays a central role in promoting malignant cell survival, proliferation, and immune evasion. Genomic alterations such as TP53, NOTCH1, and SF3B1 mutations are associated with adverse outcomes, but it remains unclear how these mutations interact with the local immune contexts across compartments. We investigated how clonal architecture and immune composition differ between PB and LN, and how these features relate to evolutionary dynamics.
 Methods:
 We performed whole exome sequencing (WES) and bulk RNA sequencing on paired LN biopsies and CD19⁺-purified PB samples from 26 patients with CLL. We selected samples from patients with known CLL driver mutations. Somatic variants, copy number alterations, and clonal architecture were inferred using SuperFreq, and cancer cell fraction (CCF) was used to define clonal distribution. Variants were filtered to include only those with >3 alternate reads, >20 total reads, a variant allele frequency (VAF) >0.02, and classified as non-synonymous mutations. CCF was considered equivalent to VAF if >0.6, otherwise CCF was calculated as VAF * 2. CCF was adjusted by dividing CCF by the ancestral or founding clone CCF as a proxy for tumor cell fraction. Clones were considered compartmentalized if the absolute CCF difference between LN and PB exceeded 0.25. Exome alteration was calculated by summing the length of exonic targets falling in copy-number segments with an absolute log2 copy-ratio >0.2 and dividing by the total captured exome length. LN immune composition was estimated from bulk RNA-seq data using CIBERSORTx.
 Results:
 Of the 26 patients, 12 were treatment-naive and 18 were IGHV unmutated. In the PB, mutational frequency was as follows: 46.1% NOTCH1, 42.3% TP53, and 26.9% SF3B1. Oncoplot analysis revealed the presence of multi-hit alterations and a predominance of missense and frameshift variants.
 Analysis of paired PB and LN samples revealed notable heterogeneity in clonal architecture. Using SuperFreq, we identified between 2-6 clones per patient (median: 4). The median number of subclones did not differ between LN and PB (4 vs. 4), treatment naive vs. relapsed/refractory status (4 vs. 4) or between IGHV unmutated vs mutated (4.5 vs. 4). We evaluated genomic instability and found significantly higher exome alteration in patients who were IGHV unmutated (P = .03). Clonal compartmentalization (CCF difference of |.25|) was observed in 18 of 26 (69%) patients, with 5 patients showing clonal evolution in the LN (28%), 6 in the PB (33%), and 7 with distinct clones evolving in both LN and PB (39%).
 SF3B1 mutations were common in clones with CCF trending higher in the LN compared to PB (median ΔCCF 0.16, P = .11). Interestingly, CCF of NOTCH1 mutations was balanced across compartments (median ΔCCF = -0.02; P = .36), and CCFof TP53 mutations was numerically slightly lower in LN than PB (median ΔCCF –0.05; P = .97).
 LN samples with stable clonal populations had higher proportions of CD4⁺ naïve T cells compared to those with evolving clones (P = .02), but no other significant changes in T cell populations were noted. Patients with NOTCH1 or SF3B1 mutations had reduced dendritic cell abundance in LN (P = .04 and P = .06, respectively) compared to those with TP53 mutations or no known driver mutations.
 Conclusions:
 This study demonstrates distinct clonal and immune features between PB and LN compartments in CLL, a heterogeneous clonal architecture across compartments. Differences in clonal composition across LN sites within individual patients suggest that localized immune contexts play a role in clonal selection. Evolving LN clones were more common in LNs with relative decreases in CD4⁺ T cell subsets, and the presence of common driver mutations was associated with reductions in dendritic cell content. Together these observations suggest an interplay of supportive vs restrictive roles of the LN TME in CLL progression. These findings highlight the added biological insight provided by LN sampling and suggest that PB alone may not fully capture the clonal heterogeneity of CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121dea9d74142004efb29c8ec3326a61ca23f8a3" target='_blank'>
              Role of the immune microenvironment in the clonal evolution of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Victoria Rogness, Bharath Dileep Kumar, Chingiz Underbayev, Pia Nierman, Clare Sun, Christine Gruessner, A. Wiestner
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Polyomaviruses (PyVs) are small, circular double-stranded DNA viruses that establish lifelong persistent infections. PyV infections are typically asymptomatic, but they can cause severe disease particularly in immunosuppressed individuals, including BKPyV-associated nephropathy, and Merkel cell carcinoma (MCC) caused by Merkel cell polyomavirus (MCPyV). While most individuals are infected with PyVs during childhood, it is unclear how these viruses persist in the host and what cellular signals prompt their reactivation. The early regions of PyV genomes encode regulatory proteins that control viral replication and host cell transformation in the case of MCPyV, SV40 and murine polyomavirus (MuPyV). All PyV early regions encode Large and Small Tumor antigens (LT, ST), which are splice variants of a common transcript. PyV LTs bind and inhibit RB1 via a conserved LxCxE motif, deregulating E2F transcriptional activity and cell cycle control to promote viral replication and tumorigenesis. The existing paradigm in the field is during normal PyV infection, LT is expressed and drives the host cell into S phase. However, a recent study revealed that cells infected with BKPyV only express LT and replicate the viral genome when already in S phase, but the mechanisms activating LT expression during S phase are unknown. The Non-Coding Control Region (NCCR), found in all PyVs, contains the viral origin of replication and predicted transcription factor (TF) binding sites, yet only a handful have been experimentally validated and shown to regulate PyV early transcription. Here, we set out to identify host TFs that bind the MCPyV NCCR to regulate early transcription in both MCC cells and in an in vitro model of MCPyV infection. We identified E2F1-3 as critical regulators of MCPyV early transcription, which bind to the MCPyV NCCR via a consensus E2 site just upstream of LT/ST transcriptional start site. We discovered E2 sites in the NCCRs of several other polyomaviruses, including MuPyV but not BKPyV. Deletion of the E2 sites in PyV NCCRs downregulates early transcription in luciferase assays. Despite not containing a consensus E2 site, we detected weak E2F binding to the NCCRs of BKPyV and SV40, suggesting that E2Fs also contribute to transcription of their early genes. Overall, our findings challenge the existing paradigm of PyV infection, that LT expression activates E2F signaling via RB1 inhibition, and suggest that E2Fs must already be active in the PyV-infected host cell for LT/ST expression and viral replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38026f5b1f8b196429ba7f6a171e6ca72452e0dc" target='_blank'>
              E2F1-3 activate polyomavirus early transcription and replication
              </a>
            </td>
          <td>
            Njh Salisbury, S. Amonkar, A. Roman, DA Galloway
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor suppressor protein 53 (p53) is a transcription factor that is deregulated in 50% of cancers. Often termed the guardian of the genome, p53 is responsible for maintenance of genomic stability, cell cycle arrest, DNA repair, senescence, and apoptosis. In cancer cells, deregulation of p53 often occurs through mutations in the DNA binding domain which lead to a loss of the transcriptional activity. While 100’s of somatic mutations in the DNA binding domain are known, a small number of mutants are enriched in cancer, suggesting a gain-of-function role. Here we deploy an intein-based approach to localize µMap photoproximity labeling to p53 to define novel interactions contributing to the loss and gain of function roles of 5 separate hotspot mutants. These data revealed that G245S and R273H binds to RNA through its C-terminal domain. We show through CLIP experiments that mutant p53 has an RNA binding motif that conserved across mutants and is enriched in 3’UTRs, promoting ribosomal localization and labeling of proteins at the mitochondrial surface. We further demonstrate that the RNA binding ability of mutant p53 promotes altered miRNA processing and mitochondrial dysfunction providing mechanistic rationale for historically reported but poorly understood phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba8d438eaa840ed5fde080176675c184a40b5ec" target='_blank'>
              Mutant p53 binds RNA to drive mitochondrial dysfunction
              </a>
            </td>
          <td>
            Wuyue Zhou, Alice Long, Cameron J. Douglas, Dalila Gallegos, James M. Burke, David W. C. MacMillan, Ezgi Hacisuleyman, Ciaran P. Seath
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2508a7d83b4a1f4cf3672318d8edc8fabc927a1" target='_blank'>
              Evidence for improved DNA repair in long-lived bowhead whale.
              </a>
            </td>
          <td>
            Denis Firsanov, Max Zacher, Xiao Tian, T. Sformo, Yang Zhao, Greg Tombline, J. Y. Lu, Zhizhong Zheng, Luigi Perelli, Enrico Gurreri, Li Zhang, Jing Guo, Anatoly Korotkov, Valentin Volobaev, S. A. Biashad, Zhihui Zhang, Johanna Heid, Alex Maslov, Shixiang Sun, Zhuoer Wu, Jonathan Gigas, Eric C Hillpot, John C. Martinez, Minseon Lee, Alyssa Williams, Abbey Gilman, Nicholas Hamilton, Ekaterina Strelkova, Ena Haseljic, Avnee Patel, M. Straight, Nalani Miller, J. Ablaeva, L. M. Tam, Chloé Couderc, M. Hoopmann, Robert L. Moritz, Shingo Fujii, Amandine Pelletier, Dan J Hayman, Hongrui Liu, Yuxuan Cai, Anthony K. L. Leung, Zhengdong Zhang, C. B. Nelson, Lisa M. Abegglen, Joshua D. Schiffman, V. Gladyshev, Carlo C. Maley, Mauro Modesti, G. Genovese, M. Simons, J. Vijg, A. Seluanov, Vera Gorbunova
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1209057bae1912ac3d93ac4a3b90f31b4f8bca" target='_blank'>
              RMND5A upregulation via IGF2BP3-mediated m6A RNA modification enhances malignant traits and immune evasion in laryngeal squamous cell carcinoma cells
              </a>
            </td>
          <td>
            Shu Zhang, Xiaojun Li, Miao Zhang, Bo Kou
          </td>
          <td>2025-11-24</td>
          <td>Human Cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Adenosine-to-inosine (A-to-I) editing is an endogenous RNA modification in eukaryotes, catalyzed by adenosine deaminases acting on RNA (ADARs). This modification modulates the gene expression by influencing splicing, RNA stability, and coding potential, depending on the site of editing. Although recent studies suggest a crosstalk between A-to-I editing and transcripts involved in DNA repair, the extent and functional significance of this interaction remain unclear. To investigate this, we applied the EpiPlex RNA assay—a method enabling epitranscriptome-wide detection of RNA modifications—in human lymphoblastoid TK6 cells. Across two biological replicates, we identified 869 transcripts bearing A-to-I–modifications. Gene Ontology analysis revealed significant enrichment in genome maintenance pathways, including “chromatin remodeling” and “DNA repair.” Notably, 27 transcripts encoding DNA repair proteins—such as ATM, FANCA, BRCA1, POLH, and XPA—contained A-to-I sites within introns or 3′ untranslated regions. To assess the isoform-specific contributions of ADAR enzymes—specifically ADAR1 p150 and p110—to RNA editing, we generated p150-deficient (p150 KO) and p150/p110-deficient (p150/p110 KO) TK6 cells. A-to-I editing peaks were reduced by ∼73.4% in p150 KO cells and nearly abolished (99.9%) in p150/p110 KO cells, indicating that most editing sites are p150-dependent, while a notable subset relies on p110. Importantly, a novel splice variant of XPA emerged in ADAR1-deficient cells, suggesting a role for RNA editing in alternative splicing regulation. Our epitranscriptomic analysis of A-to-I RNA editing underscores a multifaceted role for ADAR1-dependent editing in preserving genome integrity through posttranscriptional regulation of DNA repair genes, laying the groundwork for future studies into RNA-based mechanisms of genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13057a5d11575213c7ae970530837618192ad08" target='_blank'>
              Epitranscriptome-wide profiling identifies RNA editing events regulated by ADAR1 that are associated with DNA repair mechanisms in human TK6 cells
              </a>
            </td>
          <td>
            Akito Yoshida, Yuqian Song, Hotaru Takaine, Sujin Song, Yu-Hsien Hwang-Fu, Zachary Johnson, Kiyoe Ura, Akira Sassa
          </td>
          <td>2025-07-12</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a9a257557fe4b01c761e5ab2bf1e53a00ef799" target='_blank'>
              TIF1γ drives oral cancer recurrence by the transcriptional regulation of self-renewal genes, such as HES1
              </a>
            </td>
          <td>
            K. P. Padmaja, A. Mohan, G. M. Nair, Jyothy S Prabhu, Hafsa Shabeer, V. P. Snijesh, P. G. Balagopal, A. Thameem, Alan Jose, N. George, Jiss Maria Louis, Parvathy Visakh, Gayathri V Nair, Anna P Joseph, V. S. Jaikumar, Riya Ann Paul, Jackson James, T. T. Maliekal
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad49d001bc6357de47f285935ec8cbde300765bb" target='_blank'>
              MEDUSA: Maintaining Entire DNA Duplexes for Utmost Sequencing Accuracy
              </a>
            </td>
          <td>
            Kan Xiong, Rachel Li, Ning Li, Ruolin Liu, Azeet Narayan, J. Rhoades, Catherine Song, R. C. Lindsley, Heather A. Parsons, G. M. Makrigiorgos, V. Adalsteinsson
          </td>
          <td>2025-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) with isolated trisomies (ITs) represents a distinct cytogenetic subgroup with heterogeneous clinical behaviour and incompletely defined molecular features. To explore its genomic and transcriptomic landscape, we performed next‐generation sequencing (NGS) on 14 AML patients harbouring isolated trisomies of chromosomes 8, 9, 10, 13, 14, 21 and 22. RNA sequencing (RNA‐Seq) was conducted on 15 samples, including 12 with IT and 3 cytogenetically normal AML cases (normal karyotype, NK‐AML) serving as controls. Trisomy 8 was most frequent, followed by chromosomes 13, 14 and 21. Recurrent mutations were identified in epigenetic regulators (DNMT3A, IDH1/2, ASXL1, TET2). Transcriptomic profiling stratified cases into IT‐8, IT‐21 and IT‐13+22 subgroups. Gene set enrichment analysis (GSEA) revealed shared downregulation of cell cycle‐related pathways (e.g., G2M checkpoint) and subgroup‐specific patterns: oxidative stress and unfolded protein response in IT‐8; epithelial‐mesenchymal transition and oxidative phosphorylation in IT‐21; inflammatory signalling (IL‐6/JAK/STAT, TNF‐α/NF‐κB) in IT‐13+22. A core set of 60 differentially expressed genes (DEGs) was shared, with nine hub genes related to cell cycle (MCM4, CDC7, CDC25A, DHFR), proteostasis (HSPA5, DNAJC3, CALR, HSP90B1) and inflammation. Drug sensitivity profiling revealed subgroup‐specific vulnerabilities: IT‐8 to DNA damage checkpoint inhibitors, IT‐21 to PLK/mTOR inhibitors and IT‐13+22 to BRAF/EGFR‐targeted agents. These findings highlight AML‐IT heterogeneity and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a07ac5d29af86fd0590c16e27dfff205a7ea13" target='_blank'>
              Integrated Genomic and Transcriptomic Profiling of Isolated Trisomies in AML Reveals Cell Cycle Dysregulation and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            J. H. Maués, Bruno Kosa L Duarte, M. Svidnicki, Herton Luiz A S Filho, F. Niemann, Adriana da Silva S Duarte, Paula de Melo Campos, P. Moraes-Vieira, S. T. Olalla Saad
          </td>
          <td>2025-11-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Chromatin remodelers regulate DNA-nucleosome interactions, influencing chromatin structure, accessibility, and gene expression (Gourisankar, S., Nat Rev Genet, 2024). Their dysregulation is frequently observed in cancer and often represents a therapeutic vulnerability (Johnstone, S. E., Science, 2022). HELLS, a key chromatin remodeler, maintains genome stability and regulates gene expression in aggressive T-cell lymphomas, such as ALK-negative anaplastic large cell lymphoma (ALK- ALCL) (Fragliasso V, Leukemia 2020; Tameni A., Cell Death Dis. 2021). Previous studies have demonstrated that HELLS knockdown sensitizes ALCL cells to chemotherapy, supporting its potential as a therapeutic target (Tameni A., Nucleic Acids Res. 2024).
 This study aimed to define the transcriptional role of HELLS and evaluate its therapeutic relevance in ALCL. To achieve this, we analyzed formalin-fixed, paraffin-embedded biopsies from a cohort of 44 ALCL patients and performed an integrated in vitro multi-omics analysis (RNA-seq, ChIP-seq, ATAC-seq, and Connectivity Map), alongside functional assays.
 To understand how HELLS coordinates and supports transcription, we integrated HELLS Chromatin immunoprecipitation-sequencing (ChIP-seq) data from the TLBR-2 cell line with RNA-sequencing data from TLBR-2 HELLS-knockdown (KD) and control cells. Of the 729 genes significantly affected by HELLS KD, 467 (64%) were identified as direct HELLS targets based on ChIP-seq analysis. Gene-set enrichment analysis of these HELLS-direct genes (HDGs) revealed their association with diverse biological processes, including the JAK/STAT signaling pathway and chemokine- and cytokine-mediated inflammatory signaling. The clinical relevance of HDGs was confirmed by nCounter profiling in a cohort of 44 ALCL patients (15 ALK+, 29 ALK- ALCL).
 To assess the contribution of HELLS to HDG transcription, we investigated the distribution of RNA Polymerase II (RNAPII) via ChIP-seq in TLBR-2 HELLS-KD and control cells. RNAPII ChIP-seq data revealed RNAPII elongation defects in approximately 60% of HDGs and altered RNAPII occupancy in ~40% of HDGs following HELLS depletion. Gene ontology analysis of this latter group of HDGs highlighted an enrichment in immune-related pathways, including T-cell-mediated immunity, cytokine signaling, and JAK/STAT signaling. Furthermore, ATAC-seq and H3K4me3 profiling indicated that HELLS promotes chromatin accessibility and transcriptional activation at immune-related loci.
 To evaluate the therapeutic relevance of HELLS, we employed a drug repurposing approach based on the HDG signature. This led to the identification of 10 classes of synergistic pathways, with PI3K, JAK/STAT, and DNA-PK emerging as top-scoring synergistic targets. In several ALCL cell lines, HELLS depletion synergized with an IC20 dose of Ruxolitinib (a JAK inhibitor) or AZD7648 (a DNA-PK inhibitor), resulting in significant synthetic lethality compared to single-agent treatment or control cells.
 In conclusion, this study demonstrates that HELLS drives the expression of immune-related genes through chromatin remodeling in aggressive ALK- ALCL. Its inhibition uncovers combinatorial vulnerabilities, providing a rationale for dual-targeted therapies involving JAK/STAT or DNA-PK pathway inhibition.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/088deae46f89f2511cdb635d4408d6da56d67fdf" target='_blank'>
              Targeting hells‑mediated chromatin accessibility to sensitize ALK‑negative anaplastic large cell lymphoma to JAK/STAT and DNA‑PK inhibition
              </a>
            </td>
          <td>
            Giulia Gambarelli, Selene Mallia, B. Donati, N. Puccio, Magda Zanelli, Emanuele Vitale, Federica Torricelli, V. Manicardi, E. Salviato, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, 
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis, creating new combinations of the genome while also introducing mutations, and influencing GC content. Here we report direct detection of recombination events using highly accurate long-read sequencing from testis tissue of 16 individuals across six primate species and three human sperm samples. Based on methylation patterns, we classify sequencing reads as originating from either somatic or germline cells. We identify 2881 crossovers, 2314 simple gene conversions, and 555 complex events, and analyze their chromosomal distribution. Crossovers are more telomeric, showing stronger concordance with recombination maps than gene conversions. Human samples align with a double-strand break map, whereas other species differ, consistent with variation in PRDM9-directed breaks, although the recombination process is otherwise conserved. Gene conversion tracts are short and of similar length across species (mean 22–95 bp), implying that most non-crossover events are undetectable. We observe GC-biased gene conversion for both single and multiple-SNV events, including sites flanking crossovers. We infer longer gene conversion tracts associated with crossovers (318–688 bp) than with non-crossovers. Highly accurate long-read sequencing combined with methylation-based classification of reads to specific cell types provides a powerful way of studying recombination events in single individuals of any mammalian species. Homologous recombination rearranges genetic information during meiosis to generate new combinations in the genome. Here, the authors report direct, single-individual detection of recombination in primates and humans using long-read sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be14431c3fb0a34e6bd96874bd9859d957c5877b" target='_blank'>
              Long-read sequencing of primate testis and human sperm allows identification of recombination events in individuals
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. Winge, C. Hvilsom, Carmen Oroperv, Lasse Thorup Hansen, J. Berner, Marta Pelizzola, S. Laurentino, N. Neuhaus, Asger Hobolth, Thomas Bataillon, S. Besenbacher, Kristian Almstrup, M. Schierup
          </td>
          <td>2025-11-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most common (~80%) and lethal ovarian cancer subtype in the United States, characterized by TP53 mutations and DNA repair defects causing chromosomal instability (CIN). KIF18A is an essential cytoskeletal motor protein for cell division in CIN+ cancer cells, but it is not necessary for cell division in normal cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CIN+ cancers. We investigated the use of a novel KIF18A inhibitor ATX020, for selectively targeting CIN+ HGSOC cells using growth inhibition assays, invasion assays, immunoassays, cell cycle analysis, and immunofluorescence techniques. Using DepMap and flow cytometry, we classified a panel of HGSOC cell lines based on aneuploidy scores (AS) and ploidy levels and identified a correlation between these classifications and sensitivity against ATX020. ATX020 induced cytotoxicity through mitotic arrest and DNA damage, and reduced tumor growth in HGSOC with high aneuploidy scores (AS). Mechanistically, ATX020 blocks KIF18A’s plus-end movement on spindle fibers, increasing spindle length, resulting in chromosomal mis-segregation, aneuploidy, and DNA damage. Our findings suggest that ATX020 inhibits CIN+ HGSOC cells mainly by inducing mitotic arrest and DNA damage, disrupting KIF18A’s function crucial for mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f42a6d74edbdfa8c878ba6b61be09681cd9d76" target='_blank'>
              The KIF18A Inhibitor ATX020 Induces Mitotic Arrest and DNA Damage in Chromosomally Instable High-Grade Serous Ovarian Cancer Cells
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Tzu-Ting Huang, Maureen Lynes, Sanjoy Khan, Serena J Silver, Jung-min Lee
          </td>
          <td>2025-11-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ca54f63f6feb36492a0d66c69a169994ddd7a" target='_blank'>
              Leukemia Risk Factor ARID5B Coordinates HDAC-Mediated Transcriptional Repression
              </a>
            </td>
          <td>
            A. P. Kutschat, Fabian Frommelt, Brianda L Santini, Sophie Müller, P. Batty, G. Karbon, Giulio Superti-Furga, Davide Seruggia
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 H3K27 mutant(H3K27M) tumors are the most common type of diffuse midline gliomas and are lethal childhood tumors with a median survival of less than 12 months. Radiation is a critical treatment, yet these tumors recur rapidly within the radiation field due to resistance to radiation-induced DNA damage, allowing the tumor cells to survive. However, the interactions mediating efficient DNA repair, radiation resistance, and recurrence are poorly understood. Our goal was to identify mediators of treatment resistance in H3K27M tumors for therapeutic targeting to improve survival.



 We developed an in vivo recurrence model to assess mediators of treatment resistance. Bulk RNA, single cell RNA, and spatial transcriptomics analyses were used to determine drivers of resistance, and genes of interest were mechanistically assessed for their role in resistance.



 We determined that H3K27M epigenetic reprogramming results in activation of cell-surface protein THY1 which is highly upregulated in all H3K27M tumors. We then confirmed in numerous in vitro and in vivo models that THY1 mediates significant treatment resistance. We determined that THY1 mediates resistance through rapid and efficient DNA repair. We then performed extensive mechanistic analyses which revealed that THY1 signaling activates Fyn, a Src family kinase, resulting in downstream DNA repair. Remarkably both biological knockdown of Fyn and pharmacological Fyn inhibition, with brain-penetrating saracatinib, result in delayed DNA repair and tumor cell death.



 We have uncovered that the Thy1-Fyn signaling cascade mediates rapid DNA repair and treatment resistance. Pharmacological Fyn inhibition successfully delays DNA repair which could improve patient survival. As saracatinib is highly brain penetrant(0.5:1 Brain: Plasma ratio) and has demonstrated safety in Phase I/II Alzheimer’s clinical trials, it is an ideal drug for clinical translation. We are currently designing a Phase 0/1 study to assess drug penetration, efficacy, and safety with saracatinib and radiation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c2f1401d86d8dd15e7bf8324894f0a446afbce" target='_blank'>
              EXTH-65. THY1 Mediates Treatment Resistance in H3K7M Pediatric Tumors and is Therapeutically Targetable
              </a>
            </td>
          <td>
            S. Syed, Sunjong Ji, Arushi Tripathy, Yazen Alomary, Reka Chakravarthy, Mateus Mota, Dah-Luen Huang, Zhuwen Wang, Zian Li, Qiang Zhang, Daniel Nava de la Martin, T. Lawrence, D. Wahl, C. Koschmann, S. Venneti, A. Rehemtulla, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="One of the seminal discoveries from genetic studies of autism spectrum disorder and related neurodevelopmental disorders (NDD) has been that loss-of-function (LoF) mutations in many genes that impact chromatin and transcriptional regulation confer substantial liability to NDD. Haploinsufficiency of the epigenetic regulator POGZ represents one of the strongest such associations; however, little is known about the direct or indirect regulatory targets of POGZ, or the mechanisms by which loss of this chromatin modifier alters early neuronal development and synaptic functions. Here, we created an allelic series of CRISPR-engineered human induced pluripotent stem cell (hiPSC) clones harboring mono- and biallelic POGZ deletions. In hiPSC-derived neural stem cells (NSC) and Neurogenin 2-induced neurons (iN), POGZ LoF altered the expression of genes associated with critical cellular processes and neuronal functions, including synaptic and intracellular signaling and extracellular matrix organization. Our multiomics profiling also showed altered footprinting of critical transcription factors (e.g., activator protein 1 complexes) that were enriched at promoters of differentially expressed genes associated with synaptic function. To further interrogate the shared molecular changes in neuronal development associated with NDD and POGZ regulation, we compared our results to deletions of the transcription factor MEF2C and the sodium channel gene SCN2A that we generated in these same isogenic iN. These analyses revealed strong enrichment of extracellular matrix and intracellular signaling disruption associated with POGZ and MEF2C deletion, whereas POGZ and SCN2A haploinsufficiency exhibited shared transcriptional effects on gene modules enriched for NDD-associated genes with opposing regulatory effects. Notably, we also observed alterations to synaptic firing rate and neurite extension with biallelic deletions, but not heterozygous lines, suggesting subtle effects in neuronal development associated with haploinsufficiency. Overall, these shared molecular consequences suggest key points of convergence that connect epigenetic regulation to neuronal function in the etiology of neurodevelopmental pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed6138b4f1663b7339e409062cbd4f7a6fbbd84" target='_blank'>
              CRISPR-engineered deletion of POGZ alters transcription factor binding at promoters of genes involved in synaptic signaling
              </a>
            </td>
          <td>
            M. Moyses-Oliveira, Yating Liu, Serkan Erdin, Dadi Gao, Riya Bhavsar, Kiana Mohajeri, Kathryn O’Keefe, Philip M. Boone, Gabriela Xavier, Calwing Liao, Aiqun Li, Rachita Yadav, M. Salani, D. Lucente, Benjamin Currall, C. D. de Esch, D. J. Tai, D. Ruderfer, Kristen J. Brennand, James F. Gusella, M. Talkowski
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is an aggressive form of primary brain tumor, characterized by high malignancy and genetic heterogeneity, and standard treatment for GBM has remained unchanged in the last decade. A feature of GBM that makes it particularly difficult to treat is its heterogeneity, in part created by glioma stem cells (GSCs) within the solid tumor. GSCs have high plasticity and the ability to self-renew and differentiate into various heterogeneous cancer cell populations. GSCs are kept in a ‘stem’- like state with unlimited self-replication without differentiation and maturation, which helps maintain tumor heterogeneity and growth. This state also prevents them from being targeted effectively by traditional therapeutics. Histone modifications keep late-glial genes inaccessible by transcription factors, promoting transcription of replication and early-glial genes, maintaining stem-ness. A major player in this process is the imitation switch (ISWI) family of ATP-dependent chromatin remodelers, which have been found to have widespread aberrant expression in cancers, including gliomas. The ISWI complex can change the spacing of histones on the chromatin, and is primarily involved in the repression of gene expression. It contains a catalytic subunits, SMARCA5, and our data support that inhibition of SMARCA5 reprograms GSCs to differentiate. RNA-Seq after genetic depletion and pharmacologic inhibition of SMARCA5 in GSCs leads to loss of expression of early glial lineage genes and increased expression of late glial genes. We have developed a first-in-class small molecule SMARCA5 inhibitor that passes the BBB. In pilot studies, SMARCA5 depletion through genetic and pharmacologic inhibition extended animal survival in GSC-derived xenograft models of GBM. ATAC-Seq and single-cell RNA-Seq of tumors further support that SMARCA5 inhibition facilitated transition from stem/progenitor to differentiated cell states. Additionally, the SMARCA5 pharmacological inhibitor was well tolerated in vivo, suggesting that targeting SMARCA5 may have a favorable therapeutic window and high clinical translation potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea3b1fa5c18282fa0b9ccc1bb0d300ed3d16cc6" target='_blank'>
              STEM-03. SMARCA5 AS A THERAPEUTIC TARGET FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            Molly Guthrie, Himanshu Dashora, Le Xiong, Meghana Budankayala, Hongyu Zhao, Hariti Shah, Julia Yoo, Tapas Behera, Dalya Khalife, Y. Saunthararajah, Liangqi Xie, Jennifer Yu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Glioblastomas (GBMs) are highly aggressive, infiltrative, and heterogeneous brain tumors driven by complex genetic alterations. The basic-helix-loop-helix (bHLH) transcription factors ASCL1 and OLIG2 are dynamically co-expressed in GBMs; however, their combinatorial roles in regulating the plasticity and heterogeneity of GBM cells are unclear. Here, we show that induction of somatic mutations in subventricular zone (SVZ) progenitor cells leads to the dysregulation of ASCL1 and OLIG2, which then function redundantly and are required for brain tumor formation in a mouse model of GBM. Subsequently, the binding of ASCL1 and OLIG2 to each other’s loci and to downstream target genes then determines the cell types and degree of migration of tumor cells. Single-cell RNA sequencing (scRNA-seq) reveals that a high level of ASCL1 is key in specifying highly migratory neural stem cell (NSC)/astrocyte-like tumor cell types, which are marked by upregulation of ribosomal protein, oxidative phosphorylation, cancer metastasis, and therapeutic resistance genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f56d67f3fa305af969dda22297460569970ba3" target='_blank'>
              CNSC-16. TRANSCRIPTION FACTORS ASCL1 AND OLIG2 DRIVE GLIOBLASTOMA INITIATION AND CO-REGULATE TUMOR CELL TYPES AND MIGRATION
              </a>
            </td>
          <td>
            Bianca L. Myers, T. Vue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Potassium is the most abundant intracellular cation. Potassium flux across cell membranes is controlled by potassium channels. While cytoplasmic potassium has been extensively studied, very little is known about the function of potassium in the cell nucleus. Here, we discover that a nuclear potassium channel complex composed of KCNG1 and KCNB1 subunits is a regulator of nuclear potassium in medulloblastoma, the most common malignant pediatric brain tumour. Using genetically encoded potassium sensors, we show that KCNG1 regulates nuclear potassium level independently of cytoplasmic potassium level. KCNG1 knockdown alters the genomic architecture of medulloblastoma cells by stabilizing G-quadruplex DNA and inducing DNA damage. Ultimately, KCNG1 deficiency results in nuclear swelling, thereby compromising nuclear envelope integrity, promoting the formation of micronuclei, and inducing DNA spillage into the cytoplasm to activate the immune-stimulatory cGAS/STING pathway. Kcng1 knockout in a genetically engineered mouse model of medulloblastoma remodels the immune microenvironment, reduces tumour growth, and prolongs mouse survival. Altogether, we identified a potassium channel complex that regulates nuclear potassium to govern nuclear integrity, tumour immune microenvironment, and medulloblastoma growth. Our results demonstrate that targeting nuclear potassium can be leveraged to treat medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3156f37831fc433a976d1d47ab5180bff944abd" target='_blank'>
              DNAR-02. Nuclear potassium governs nuclear integrity and tumour growth in medulloblastoma
              </a>
            </td>
          <td>
            Jade Chan, Xi Huang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, the roles of which are under debate. We found that double-strand breaks (DSBs) induce de novo formation of chromatin loops in human cells, with the loop base positioned at the DSB site. These loops form in the S and G2 phases of the cell cycle during homologous recombination repair, concomitantly with DNA end resection and radiation-sensitive protein 51 (RAD51) recruitment. RAD51 shows a broad (megabase-sized) chromatin domain reflective of the homology search. This domain is regulated by cohesin unloader and overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of the loop-extruding cohesin subunit NIPBL lowers homologous recombination in mouse embryonic stem cells, and this effect is more pronounced when the homologous recombination donor is hundreds of kilobases from the DSB. Our data indicate that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e61307e4b5e3ffa40f9315b36ee4db40b3f94b6" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search.
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam T. Rybczynski, Namrata Nilavar, Daniel Nguyen, Andrew G. Li, Violetta Karwacki-Neisius, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-12-04</td>
          <td>Science</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL), the most common type of pediatric leukemia, is frequently driven by fusion genes generated by chromosomal rearrangements. Compared with wild-type genes, many oncogenic fusions show increased expression and sustained functional activity that drives tumorigenesis. However, the mechanisms by which chromosomal rearrangements lead to functional enhancement remain largely elusive. In addition, although large-scale sequencing has identified numerous fusion events, the functional significance of most remains unclear. Here, we demonstrate that enhanced mRNA stability represents an important tumorigenic mechanism for oncogenic fusions, including classical PAX5 fusions. Based on this mechanism, we characterize a novel oncogenic fusion, STK38-PXT1, which exhibits upregulated STK38 mRNA levels and drives the development of ALL. Mechanistically, the increased mRNA stability results primarily from enhanced m6A modification of oncogenic fusions, which is attributable to "gene truncation" (as in PAX5 fusions) and "partner collaboration" (as in STK38-PXT1). Furthermore, the m6A reader IGF2BP3 is crucial for maintaining the high mRNA stability of oncogenic fusions. We further propose venetoclax as an innovative and clinically available therapy for ALL driven by these oncogenic fusions characterized by high mRNA stability. Our study not only highlights mRNA stabilization as a crucial mechanism by which oncogenic fusions to drive tumorigenesis, but also presents a promising therapeutic strategy for patients with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c354105c3c8ac3504d3a9759fd49b409b956d3c" target='_blank'>
              Chromosomal rearrangement-enhanced mRNA stability drives the oncogenic potential of fusion genes in pediatric leukemia.
              </a>
            </td>
          <td>
            Xuejing Shao, Zhimei Xia, Minyi Cai, Chen Shao, Shaowei Bing, Tianrui Wang, Wenxin Du, Jiayi Liu, Diying Shen, Ji Cao, Bo Yang, Qiaojun He, Xiaojun Xu, Jingying Zhang, Meidan Ying
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Paclitaxel is a widely used chemotherapeutic agent that plays a vital role in the treatment of various cancers, including breast, ovarian, and lung cancers. Despite its success, the development of resistance significantly limits its long-term efficacy. Paclitaxel resistance involves a range of molecular mechanisms, including alterations in drug transport, mutations in β-tubulin, and activation of pro-survival signaling pathways. Recent studies have highlighted the crucial role of circular RNAs (circRNAs) in mediating paclitaxel resistance. CircRNAs are characterized by their stable, covalently closed structures, which protect them from exonuclease degradation. They regulate drug resistance through various mechanisms, including modulating transcription factors and influencing cellular processes such as apoptosis, immune response, and cancer stem cell dynamics by sponging microRNAs. This review focuses on the mechanisms by which circRNAs contribute to paclitaxel resistance and discusses their dual roles as both oncogenes and tumor suppressors. Furthermore, we explore the potential of circRNAs as novel therapeutic targets to overcome paclitaxel resistance. By targeting specific circRNAs or restoring tumor-suppressive circRNAs, it may be possible to enhance paclitaxel sensitivity and improve treatment outcomes. Future research is essential to further understand the role of circRNAs in paclitaxel resistance and to develop effective circRNA-targeted therapies for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b2a64750b1e25583cb541b2ce40bf7626415eaa" target='_blank'>
              The dual role of circular RNAs in paclitaxel resistance: insights into molecular mechanisms and clinical implications.
              </a>
            </td>
          <td>
            Ehsan Maghrebi-Ghojogh, O. Q. B. Allela, Karar H. Alfarttoosi, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq, Mitra Zareanshahraki, Hossein Gandomkar
          </td>
          <td>2025-10-17</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex
 virome
 , defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored.




 We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe.




 We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma:
 Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1
 , and
 Cyprinid herpesvirus 2
 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75).




 Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bfffe74b13f5a75dbf73aa9e042097d2141c09" target='_blank'>
              POLQ variants with aberrant DNA polymerase activity protect against UV-induced cell death
              </a>
            </td>
          <td>
            Steven E Weicksel, Corey Thomas, Ethan Hall, Kylie Davis, Chase Michalczik, Sreerupa Ray, Jamie B Towle-Weicksel
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec13fad98d61dff3c83ecb3a4a35f96c11ec562" target='_blank'>
              Targeting MORF4L1-mediated DNA repair potentiates RT-induced antitumor immunity via cGAS-STING activation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si-wei Wang, Wei-Feng Hong, Yi-Lan Huang, Yi-Min Zheng, Qiu-Yi Zheng, Jun-Jie Cheng, Bu-Fu Tang, Gen-Wen Chen, Bu-Gang Liang, Jia-Cheng Lu, Li Yuan, Shujung Hsu, Yang Zhang, Xiao-Bin Zheng, Zhaochong Zeng, Jian Zhou, Jia Fan, Aiwu Ke, Chao Gao, Shi-Suo Du
          </td>
          <td>2025-11-05</td>
          <td>Cellular & Molecular Immunology</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Dysregulation of gene expression is an essential feature of most human cancers, including acute myeloid leukemia (AML). Despite distinct biology of different subtypes of AML, there is also increasing evidence for shared pathologic mechanisms. Varied genetic subsets of AML exhibit shared dependencies on specific transcription factors and gene expression regulatory mechanisms. In particular, MYB is aberrantly activated in most studied forms of AML, where it is required for blockade of differentiation and self-renewal of leukemia cells. MYB is rarely mutated and generally not over-expressed in most human myeloid leukemias, presenting a key question concerning mechanisms of its activation that should be targeted therapeutically.
 To elucidate potential mechanisms of MYB activation, we investigated post-translational modifications (PTMs) of endogenous MYB proteins purified from human AML cells. Using comprehensive bottom-up high-resolution mass spectrometry, we achieved 95% MYB peptide sequence coverage and identified 10 unique MYB amino acid residues containing phosphorylated, acetylated, methylated, or ubiquitin-like PTMs. This included phosphorylation of S11 and S12 in the N-terminal domain (NTD) proximal to the DNA-binding domain of MYB, as validated using phospho-specific MYB antibody, mutagenesis, and dephosphorylation experiments. Notably, S11/S12 phosphorylation was preferentially detected across multiple AML cell lines and patient specimens from diverse molecular subtypes as compared to healthy human blood progenitor and stem cells.
 We found that MYB is required for the growth and survival of leukemia cells, which can be rescued by the expression of wild-type MYB but not MYB mutants that cannot be phosphorylated at S11/S12 in its NTD. Using label-free quantitative mass spectrometry combined with affinity purifications, we defined specific protein co-factors that are selectively assembled with phosphorylated as compared to non-phosphorylated MYB in the nuclei of AML cells, as well as cells expressing MYB-PLEKHO1 and MYB-ZFAT fusion oncogenes from blastic plasmacytoid dendritic cell neoplasm (BPDCN) cells. We found that S11-phosphorylated MYB preferentially assembles with other hematopoietic transcription factors including LMO2, LYL1, JUN, and TCF12. A subset of transcription factors including ZEB2, SATB1, IKZF1, SPI1, and RUNX1, 2, and 3 were also found to be co-assembled with BPDCN MYB oncogene fusion proteins. Co-assembled MYB transcription factors were associated with cooperative chromatin complexes regulating genes controlling leukemia cell differentiation, growth and survival, as defined using gene expression, chromatin profiling, and functional genomic assays.
 In all, these studies identify DNA-proximal phosphorylation of MYB and MYB DNA-binding domain fusions as a critical regulatory mechanism required for MYB oncogenesis through cooperative organization of transcription factors. Modifications of DNA-proximal NTD leading to MYB enhanceosome assembly suggest a new target for therapeutic intervention of MYB-dependent transcriptional circuits in AML and other transcription factor-dependent malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97715039497852e85a982a7d2a4af32af301c7f5" target='_blank'>
              DNA-proximal MYB phosphorylation is required for oncogenic transcription in acute myeloid leukemia
              </a>
            </td>
          <td>
            Shuyuan Cheng, Katarzyna Kulej, Christopher A G Booth, Masahiro Uni, Nicole Cruz, Andrew Lane, R. Koche, A. Kentsis
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 In normal hematopoiesis, HOXA9 maintains early hematopoietic progenitors and is silenced during differentiation. Failure to silence HOXA9 blocks hematopoietic differentiation and leads to leukemia. In acute myeloid leukemia (AML), it is overexpressed in over 70% of cases with diverse genetic alternations and is a strong predictor of poor prognosis. Our previous work has demonstrated that disruption of lineage-specific transcription factors can drive leukemogenesis by reorganizing the 3D genome structure and regulatory networks critical for cell fate determination.
 As a pioneer factor in AML, HOXA9 is essential for leukemic transformation, and its leukemogenic activity is markedly enhanced by co-overexpression with its partner MEIS1. However, the specific regulatory mechanisms underlying HOXA9-driven leukemia progression, and its role in establishing oncogenic regulatory networks that drive AML pathogenesis, remain poorly understood.
 To address this, we developed a primary cell system co-expressing HOXA9 and MEIS1, combined with a PROTAC-based degron strategy for rapid and inducible degradation of HOXA9 protein. This approach avoids complex genetic alterations and enables precise, temporal dissection of HOXA9 function under controlled conditions. We isolated hematopoietic stem and progenitor cells (HSPCs) from the bone marrow of C57BL/6J mice and transduced with a single retroviral construct co-expressing FKBP12F36V degron-tagged HOXA9 and MEIS1 (dA9M). Treatment of these transduced cells with dTAGV-1 resulted in rapid and complete degradation of HOXA9 within 1 hour. Importantly, dTAGV-1 treatment led to reduced cell viability of dA9M cells, but had no effect on HSPCs transformed by non-degron-tagged HOXA9 and MEIS1.
 To precisely detect HOXA9-mediated transcriptional, epigenetic, and higher-order chromatin changes in AML, we collected 3 biological replicates at 0, 1.5, 6, 12, 24, 48, and 96 hours after dTAGV-1 treatment. Integrating transcriptomic and epigenomic profiles, we classified 3 regulatory clusters: immediate loss, early decrease, and late decrease, based on chromatin accessibility changes upon HOXA9 depletion. Chromatin accessibility alterations at early stages (beginning at 1.5 hours) comprised 2 clusters. The first cluster included immediate loss regions, which exhibited chromatin accessibility loss at 1.5 hours with exclusive enrichment for HOXA9 motifs. The second cluster consisted of early decrease regions enriched for HOXA9 along with differentiation associated transcription factors. Overall, at early stages, genes involved in leukemic stem cell self-renewal and myeloid differentiation pathways were broadly downregulated. To directly measure the immediate transcriptional consequences of HOXA9 loss, we performed SLAM-seq (thiol(SH)-linked alkylation for the metabolic sequencing of RNA) at 1.5 hours after dTAGV-1 treatment which revealed a rapid and significant decrease in the transcription rates of 308 genes. This result confirmed that their downregulation was a direct consequence of HOXA9 depletion, establishing them as immediate transcriptional targets of HOXA9. To determine the functional relevance of these genes, we further validated that 42 of them are critical for leukemia cell survival through both in vivo and in vitro screening. Late stage alterations emerged at 48 hours as the third cluster, which was enriched for CTCF and cohesin components including SMC3 and RAD21. These epigenomic changes affected genes involved in chromatin organization and cell cycle regulation, thereby constraining leukemic cell proliferation.
 At the level of 3D genome organization, HOXA9 depletion induced widespread reorganization events across multiple scales, consistent with the associated epigenetic changes. Notably, loss of HOXA9 led to chromatin compaction, disruption of topologically associating domain (TAD) structures, and reshaping of enhancer promoter interactions. In particular, SOX4 expression was linked to disrupted enhancer-promoter interactions at a TAD boundary.
 Together, our study provides a higher order chromatin regulatory network mediated by HOXA9 in AML. Furthermore, these results demonstrate that HOXA9 governs specific transcriptional programs and 3D genome organization in AML and thus provides potential targets for disrupting the core regulatory mechanisms that sustain the disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efe38a73a45ae0a4a8a8206075f4eb2ab0aa216" target='_blank'>
              Acute depletion of HOXA9 reveals its direct role in cis-elements regulation and 3D genome organization during Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Qili Shi, Sajesan Aryal, Qiushi Jin, Xinyue Zhou, Chenhui He, Huijie Yang, Rui Lu, Yu Luan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aim: The A2UCOE, a well-characterized chromatin-modifying element originating from a constitutively expressed human gene locus, promotes consistent and durable gene activity, even when foreign DNA sequences are inserted into densely packed heterochromatin regions. The aim of this study is to examine the Daedalus fragment in A2UCOE to identify key sub-regions responsible for its prominent chromatin opening activity. This would enable the discovery of a minimal yet fully functional element, allowing for more efficient utilization of vector capacity to accommodate therapeutic transgenes when incorporated into lentiviral vectors (LVs).
Method: To assess the chromatin-opening activity of the Daedalus sub-region within the A2UCOE, lentiviral vectors carrying eGFP reporter constructs were generated and transduced into murine embryonal carcinoma cell lines (P19 and F9). Transgene expression stability was assessed across both pluripotent cells and those differentiated with retinoic acid. HEK293T cells were used for lentiviral packaging. Plasmid propagation was performed in E. coli DH5α, and standard molecular cloning techniques were applied. Flow cytometry was used to measure eGFP expression levels, and significance was assessed via Student’s t-test (p < 0.05).
Results: Flow cytometry and RT-qPCR analyses revealed that the positive control vector (1.5A2UCOE-SEW) maintained stable eGFP expression in both differentiated states of P19 and F9 cells, whereas the UCOE vectors named Daedalus-F and Daedalus-R were unable to prevent transgene silencing. In undifferentiated cultures, eGFP expression from these vectors decreased by 40–50% within two weeks, with a similar decline observed in differentiated cells. The average vector copy numbers remained stable, indicating transcriptional silencing rather than vector loss.
Conclusion: The Daedalus sub-region of A2UCOE alone could not prevent transgene silencing in pluripotent or differentiated cells, indicating additional A2UCOE elements are needed for stable long-term expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b98626fce66cdb8386ed52347362958929e350" target='_blank'>
              Daedalus fragment from the CBX3 ubiquitous chromatin opening element as an optimum UCOE candidate
              </a>
            </td>
          <td>
            Ali Osman Arslan, Ömer Faruk Anakök
          </td>
          <td>2025-10-26</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary We compared the results using a 108-gene RNA-seq panel with those derived from optical genome mapping (OGM) in 467 acute leukemias. The overall concordance rate was 88.1%. For clinically relevant events (n = 234), OGM uniquely identified 15.8%, whereas RNA-seq uniquely identified 9.4%. Concordance was poor for enhancer-hijacking lesions (20.6%), including MECOM, BCL11B, and IGH rearrangements, many of which were not detected by RNA-seq. Conversely, RNA-seq slightly outperformed OGM for fusions arising from intrachromosomal deletions that were sometimes labeled by OGM as simple deletions. Overall, RNA-seq is more sensitive for detecting expressed chimeric fusions, whereas OGM is better for detecting cryptic, enhancer-driven events. Using both platforms provides the most comprehensive structural variant detection in acute leukemia cases. Abstract Background/Objectives: Gene rearrangements involving oncogenes are major drivers in acute leukemia, influencing disease classification, prognosis, and therapeutic decision-making. Targeted RNA sequencing (RNA-Seq) panels capable of detecting intergenic and intragenic fusions across multiple genes are increasingly used in diagnostic settings. However, comparative evaluation with orthogonal technologies remains limited. Material and Methods: We compared the performance of a 108-gene anchored multiplex PCR (AMP)-based RNA-Seq panel with that of Optical Genome Mapping (OGM) in 467 acute leukemia cases. The cohort included 360 cases of acute myeloid leukemia (AML), 89 B-lymphoblastic leukemia (B-ALL), 12 T-lymphoblastic leukemia (T-ALL), and 6 cases of mixed phenotype acute leukemia (MPAL). Results: Results of both methods were concordant in 175 (74.7%) of 234 detected gene/rearrangement fusions. The concordance rate varied significantly across different leukemia types, ranging from 80.2% in B-ALL to 41.7% in T-ALL (p < 0.001) OGM uniquely detected 37 of 234 (15.8%) clinically relevant rearrangements, whereas RNA-Seq exclusively identified 22 of 234 (9.4%). Enhancer-hijacking lesions, including MECOM and BCL11B rearrangements, CDK6::MNX1, and IGH rearrangements, had a markedly lower concordance (20.6%) compared with all other aberrations (93.1%) (p < 0.001). Conversely, some gene fusions arising from intrachromosomal deletions were interpreted by OGM as simple deletions rather than rearrangements or fusions. Conclusions: Targeted RNA-Seq was effective for detecting chimeric fusion transcripts and showed slightly better performance in identifying fusions resulting from deletions. However, OGM was effective for detecting enhancer-hijacking events that do not generate fusion transcripts. Both methods are complementary for the workup of acute leukemia cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/783106fd704ddbf425128b4173188409188a77b1" target='_blank'>
              Comparative Analysis of Targeted RNA-Seq and Optical Genome Mapping for Detecting Gene Rearrangements in Acute Leukemia
              </a>
            </td>
          <td>
            C. Ok, Guilin Tang, S. Loghavi, Shimin Hu, Qing Wei, Andres E. Quesada, M. Routbort, R. Kanagal-Shamanna, C. Yin, Iman Sarami, Sofia A. Garces, Nitin K. Agarwal, R. Luthra, Hong Fang, F. Jelloul, Julian Bryan, L. Medeiros, Keyur P. Patel, Gokce A. Toruner
          </td>
          <td>2025-10-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="


 Mature T-cell lymphomas (TCL) are a heterogeneous group of non-Hodgkin lymphomas arising from the malignant transformation of post-thymic T and NK cells. These lymphomas are characterized by poor response to chemotherapy, high relapse rates, and dismal survival outcomes. Genomic analyses of TCL have revealed recurrent alterations affecting intracellular signaling pathways and epigenetic regulators. The frequent involvement of epigenetic modifiers suggests that epigenetic dysregulation is a critical step in TCL pathogenesis.
 To identify the epigenetic changes driving TCL transformation, we analyzed chromatin modifications and gene expression profiles in murine TCL models that closely recapitulate human disease. Our data revealed that Id2, a repressor of basic helix-loop-helix proteins, is consistently overexpressed at both RNA and protein levels across multiple murine lymphoma models. Supporting this, analysis of the Cancer Cell Encyclopedia identified TCL as one of the tumor types with the highest ID2 expression, which we confirmed by Western blot in human TCL-derived cell lines. Furthermore, analysis of single-nuclei RNAseq data from a cohort of primary angioimmunoblastic T-cell lymphoma (AITL) and peripheral T-cell lymphoma not otherwise specified (PTCL, NOS) patient samples generated in our laboratory, demonstrated significant ID2 overexpression in a subset of cases (12/27; 44%) compared to tonsil and lymph node controls. Interestingly, all relapsed TCL cases in our cohort showed significant overexpression of ID2. Based on our data, we hypothesize that ID2 is epigenetically regulated during transformation and contributes to T-cell lymphomagenesis.
 To define the mechanisms leading to ID2 overexpression in TCL, we performed H3K27ac and H3K4me1 ChIP-seq, ATAC-seq, reverse-ChIP coupled with mass spectrometry, and dual-luciferase reporter assays. These analyses identified a conserved region near the ID2 locus enriched for H3K27ac and H3K4me1 marks that co-localizes with ATAC-seq peaks and is ranked as a super-enhancer by the ROSE algorithm. This region is specifically activated in mature TCL cells compared to control CD4⁺ T cells. Reporter assays using the most conserved segment within the enhancer region showed more than twofold increased luciferase activity relative to a minimal promoter, confirming its enhancer function. Interestingly, reverse-ChIP identified CTCF and ASCL1 as candidate regulators of enhancer looping and transcriptional activation, respectively. To characterize the proteins mediating the epigenetic function of ID2 in TCL, we employed a biotin ligase proximity labeling assay (BioID) approach coupled with mass spectrometry. Our results identified known ID2 partners, including the E-proteins TCF3, TCF4 and TCF12, key regulators of T-cell differentiation, as well as novel partners including elements of the SWI/SNF chromatin-remodeling complex, suggesting that ID2 overexpression indirectly regulates transcriptional programs critical for lymphomagenesis.
 To determine the functional role of ID2 in TCL, we characterized the effects of both genetic and pharmacological inhibition of ID2, using shRNA and CRISPR-Cas9 mediated knockout and novel pan-ID inhibitors AGX51 and AGXA, respectively. Loss of ID2 leads to decreased cell viability and increased apoptosis in in vitro TCL models. Interestingly, RNA-seq analysis revealed enrichment in signatures associated with glucose metabolism, suggesting a novel role for ID2 in tumor metabolic regulation. Finally, in vivo treatment with the pan-ID inhibitor AGX51 in a murine lymphoma model induced a modest but consistent reduction in tumor burden, spleen weight, and activated CD69⁺ T cells population in AGX51-treated mice compared to vehicle-treated controls.
 In summary, we have identified a novel TCL-specific super-enhancer that drives ID2 overexpression in TCL and demonstrated an essential role for ID2 in supporting lymphoma proliferation and survival. More importantly, pharmacological targeting of ID2 reduces tumor burden in vitro and in vivo, highlighting its potential as a therapeutic target in mature T-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31f2dc0fbeb09da6f80dbe4eff065e3af3490a0" target='_blank'>
              Inhibitor of DNA binding 2 (ID2) is epigenetically regulated during lymphomagenesis and constitutes a novel therapeutic target in mature T-cell lymphoma
              </a>
            </td>
          <td>
            Anouchka P. Laurent, Hannah Miller, Ryan D. Najac, R. Albero, Jose R. Cortés, Cindy Ma, W. Lin, Anqi Wang, Ruth Alonso Alonso, Laura Quevedo, Rajesh Soni, O. Ouerfelli, Raul Rabadan, R. Benezra, Teresa Palomero
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Small nucleolar RNAs (snoRNAs) constitute a class of non-coding RNAs traditionally associated with ribosomal RNA modification. However, recent research has revealed that these molecules participate in a wide range of cellular processes with significant implications for cancer biology. This review examines the emerging evidence on snoRNA dysregulation in various tumor types, where they can function as both oncogenes and tumor suppressors. We analyze the multiple molecular mechanisms through which snoRNAs contribute to malignant transformation, including the generation of specialized ribosomes, alternative splicing regulation, epigenetic modulation, and post-transcriptional regulatory functions. We discuss the potential of snoRNAs as non-invasive biomarkers for cancer diagnosis and prognosis, as well as emerging strategies for their utilization as therapeutic targets. The integration of these findings positions snoRNAs as fundamental players in oncogenesis and opens new avenues for the development of clinical tools in precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e57200138dcd57354ce391e23390f926ba3abb" target='_blank'>
              Small Nucleolar RNAs as Emerging Players in Cancer Biology and Precision Medicine
              </a>
            </td>
          <td>
            H. Lorenzo
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="This review outlines the mechanisms and regulation of DNA replication in Escherichia coli within the oriC region, focusing on DnaA nucleotide state and stepwise occupation of high and lowaffinity boxes, origin DNA topology, and architectural proteins (IHF, Fis, SeqA). We outline helicase loading by the DnaBDnaC complex, the role of the clamp and Hda in controlling reinitiation, and how small RNAs such as DsrA remodel transcriptional programs that impinge on initiation fidelity through the regulation of proteins associated with the DNA replication process and inhibition of. We discuss coupling between replication and transcription, the contribution of DNA gyrase to the need for unwound DNA to replicate, and strategies cells use to avoid replicationtranscription conflicts. Finally, we highlight synthetic biology applications, including orthogonal replication systems in E. coli that decouple engineered DNA propagation from the host chromosome. These insights provide an integrated view of initiation control and suggest opportunities to engineer replication for biotechnology and genome stability research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1d8ddca5d996affebfe1d9eec679ba452b2314" target='_blank'>
              Genetic Regulatory Network of DNA Replication Initiation in E. coli and Its Biological Significance
              </a>
            </td>
          <td>
            Zenas Deng
          </td>
          <td>2025-10-28</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f741cfc314db9bdb828e55bb38f137b5fb40cdaa" target='_blank'>
              Plasmid copy number affects the DNA methylation-driven expression dynamics of the CfrBI restriction-modification system and impacts phage restriction
              </a>
            </td>
          <td>
            Johannes Gibhardt, Trung Duc Hoang, Konstantin Severinov, Mikhail A. Khodorkovskii, N. Morozova
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 The genomic regions surrounding tumor suppressor genes are often lost as collateral damage during tumorigenesis, providing novel therapeutic vulnerabilities. To systematically identify synthetic lethal pairs of genes within DNA damage repair (DDR) pathways, we conducted a combinatorial genome-wide CRISPR screen. We found that RNASEH2B, a gene frequently co-deleted with RB1 in prostate cancers, emerged as a top vulnerability with knockout of APEX2, which is a nuclease involved in DNA repair. This interaction was validated in RNASEH2B isogenic cell line pairs using whole-genome CRISPR screens and further confirmed across multiple cancer lineages using a focused mini-pool sgRNA library. Single gene knockouts demonstrated that loss of APEX2, specifically its catalytic activity, is sufficient to eliminate RNASEH2B mutant cells both in vitro and in vivo. Importantly, loss of RB1 did not impact the synthetic lethal interaction between RNASEH2B and APEX2. Consistent with prior reports, APEX2 knockout also impairs growth of BRCA1 mutant cells and enhances sensitivity to PARP inhibitors. Overall, these findings position APEX2 as a compelling synthetic lethal target in tumors with RNASEH2B deletion or BRCA1 mutations and support the development of APEX2 inhibitors as a potential monotherapy to improve patient outcome.



 Antoine Simoneau, Steve Lombardo, Ruthie Swain, Serge Gueroussov, Binzhang Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H. Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, Douglas A. Whittington, Yi Yu, Jannik N. Andersen, Teng Teng. APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f840d30e01fa153da53003edd382b7d06deea42" target='_blank'>
              Abstract B039: APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors
              </a>
            </td>
          <td>
            Antoine Simoneau, Steven A. Lombardo, Ruthie Swain, Serge Gueroussov, B. Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, D. Whittington, Yi Yu, Jannik N Andersen, T. Teng
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte-rich malignant tumours with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte-rich tissues, and then performed genome-wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis-regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour-specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D-loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour-specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome-wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas.
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Retinoblastoma (RB) is the most common pediatric eye cancer. Most cases of RB are initiated by bi-allelic mutational inactivation of the RB1 gene, yet most RB tumors harbor additional genomic aberrations that may promote tumor progression. After RB1, the gene that is most commonly mutated gene in RB is BCOR, which is mutated in approximately 20% of RB tumors and is associated with a more aggressive tumor phenotype and worse patient outcomes. Despite its importance, little is known about the role of BCOR in RB. Here, we interrogated BCOR in low passage RB cell lines using mass spectrometry, chromatin immunoprecipitation sequencing, and RNA sequencing. We show that the BCOR protein interacts with members of the ncPRC1.1 Polycomb Repressive Complex and localizes at gene loci with traditionally activating and repressing chromatin markers. Loss of BCOR downregulates the expression of genes associated with cell cycle regulation and upregulates genes associated with hypoxic adaptation. We conclude that BCOR mutations slow cell proliferation and drive hypoxic adaptation in RB via epigenetic mechanisms that may be amenable to targeted therapy. Implications: This study reveals that BCOR may play a noncanonical, multi-faceted role in retinoblastoma with implications in cell cycle, differentiation, and hypoxic adaptation, ultimately shedding insight into its molecular framework for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba892d255c691eee277b20e16deae8de79fd0b76" target='_blank'>
              BCOR mutations deregulate cell cycle and hypoxic adaptation pathways in retinoblastoma.
              </a>
            </td>
          <td>
            Michelle G. Zhang, J. Kuznetsoff, Natalie C Cetta, Sebastian Salazar, Renata L Volonterio, Stefan Kurtenbach, Z. Corrêa, Daniel Pelaez, J. W. Harbour
          </td>
          <td>2025-11-05</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f7970eb0b2f0e2ae86ba00f6f0eaa976b226bb" target='_blank'>
              DAZAP1 promotes cancer progression and chemotherapy resistance by stabilizing PIN1 protein in gastric cancer.
              </a>
            </td>
          <td>
            Peiling Zhang, Lujuan Ma, Yitian Wei, Qian Peng, Hong Xiang, Xisheng Fang, Cheng-yin Weng, Yong Wu, Lin Lu
          </td>
          <td>2025-12-02</td>
          <td>Cell biology and toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Introduction Ph-like B-ALL is a poor-risk B-ALL subtype defined by gene expression signatures that resemble the BCR::ABL1-associated transcriptional program in the absence of the BCR-ABL1 oncoprotein. Recent studies have demonstrated that up to 80% of Ph-like B-ALLs have recurrent gene fusions involving cytokine receptors or tyrosine kinases that can be identified via cytogenetics, fluorescence in situ hybridization (FISH) or RNA sequencing. However, the inability to rapidly and accurately identify these alterations in the clinical setting is a major impediment to developing therapeutic strategies that may have efficacy in this B-ALL subtype. Here we used 88 patient (pt) samples to validate a rapid, streamlined clinical WGS assay (ChromoSeq) for B-ALL and assessed the diagnostic yield for Ph-like-associated SVs in 136 pts enrolled in the Alliance A041501 clinical trial.
 Methods ChromoSeq is a clinical WGS assay for hematologic malignancies that uses rapid laboratory workflows and focused analysis methods to identify clinically relevant SVs, copy number alterations and gene-level mutations. We updated the ChromoSeq pipeline for use in B-ALL pts and validated it using 88 retrospective B-ALL samples (80 adult/young adult and 8 pediatric from Washington University School of Medicine and St. Louis Children's Hospital), 54 of which were analyzed via RNAseq for gene fusions and a 6-gene Ph-like B-ALL signature (CA6, SPATS2L, MUC4, JCHAIN, BMPR1B, ADGRF1, NRXN3 and CRLF2). The “ChromoSeqV2” assay was then used to analyze 136 pretreatment blood or bone marrow samples from the Alliance A041501 (trial NCT03150693). These results were compared to data from RNAseq obtained for research purposes (N=35), clinical cytogenetics (N=69) and Ph-like classification via low-density microarray (LDA) card analysis (all pts).
 Results ChromoSeqV2 was validated in a CLIA-licensed setting using cohort of 88 retrospective B-ALL samples, 45 of which had ICC and WHO class-defining SVs based on G-banded karyotyping, FISH or RNAseq. ChromoSeqV2 detected 100% of the non-Ph-like SVs (37/37), including BCR::ABL1 (N=20), KMT2Ar (N=9), ETV6::RUNX1 (N=3), TCF3::PBX1 (N=1), and rearrangements involving MYC (N=2) or MEF2D (N=2). Fifteen patients met criteria for Ph-like B-ALL via cytogenetics or RNAseq either due to rearrangements of CRLF2 (N=5), JAK2 (N=2) or ABL2 (N=1) or high expression of a 6-gene Ph-like signature (N=7). ChromoSeqV2 detected a Ph-like SV in 93% of these cases (14/15) with 100% specificity in samples negative by RNAseq (N=23).
 ChromoSeqV2 was then used to profile 136 samples from the Alliance A041501 trial, where it identified ICC and WHO class-defining SVs in 87 pts (64%; 95% CI: 55-72%), including Ph-like SVs (56/136, 41%), KMT2A rearrangements (N=5), TCF3::PBX1 (N=3) and SVs involving MYC (N=3), MEF2D (N=2), DUX4 (N=2), and ZNF384 (N=1). Ph-like SVs were dominated by CRLF2 rearrangements (N=45) and included SVs involving EPOR (N=4), JAK2 (N=3), ABL1/ABL2 (N=2) and PDGFRB (N=1). IGH was the most common CRLF2 partner (N=32), followed by P2RY8::CRLF2 (N=7) and other genes (N=5). Multiple CRLF2 SVs were detected in 4 pts, and 33% (15/45) of pts with a CRLF2 rearrangement had a co-occurring CRLF2 gene mutation. The yield of ChromoSeqV2 for SVs in pts with the Ph-like expression signature via LDA card analysis was 66% (54/82; 95% CI 55%-76%) compared to 24% (11/45; 95% CI 13%-40%) by cytogenetics when these studies were successful and 70% (12/17; 95% CI 44%-90%) vs RNAseq in pts with available data. The sensitivity of ChromoSeqV2 for Ph-like SVs identified by either cytogenetics, FISH or RNAseq was 100% (21/21; 95% CI 84%-100%) and the positive predictive value of Ph-like SVs for the Ph-like expression signature by the LDA card assay was 96% (54/56; 95% CI 88%-99%).
 Conclusions Genomic assessment of 136 B-ALL pts from the Alliance A041501 trial using the ChromoSeqV2 rapid clinical WGS assay was 100% sensitive for Ph-like SVs identified by other molecular methods and had a 96% PPV for the Ph-like expression signature via LDA card analysis. The diagnostic yield of SV pts with the Ph-like signature was 66% using ChromoSeqV2, which was similar to RNAseq and superior to cytogenetics. These results demonstrate that ChromoSeqV2 is an effective approach for identifying potentially targetable rearrangements in B-ALL pts suspected of having the Ph-like phenotype. Support: U10CA180821, U10CA180882; . Servier; Pfizer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/496ee390ef7f76b531cb6af7732693c780f64abe" target='_blank'>
              Increased yield of recurrent structural variants (SVs) in Ph-like B-cell acute lymphoblastic leukemia (B-ALL) using a streamlined clinical whole-genome sequencing (WGS) assay
              </a>
            </td>
          <td>
            Andrew Hughes, Feiyu Du, Michele O'Laughlin, Gregory Sprenger, Mohamed Mahgoub, Nidhi Davarapalli, Heidi Struthers, Haley Abel, Christopher Manring, R. Harvey, C. Saygin, Jun Yin, , D. DeAngelo, S. Dinner, Richard M. Stone, Marina Konopleva, Ann-Kathrin Eisfeld, Molly Schroeder, Eric Duncavage, A. Advani, Wendy Stock, Geoffrey Uy, David Spencer
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="


 In frame fusions between Fibroblast Growth Factor Receptor 3 (FGFR3) and Transforming Acidic Coiled-Coil containing protein 3 (TACC3) genes on chromosome 4p16.3 have been reported across diverse cancers, including bladder cancer (∼5%) and IDH-wildtype glioblastoma (4.1%), as well as in small subsets of non-small cell lung (∼0.5%), esophageal squamous cell (1.4–2.1%), cervical (1.9%), head and neck (3.7%), and rare nasopharyngeal (2.5%) carcinomas, and other cancer types. These chromosomal translocations generate FGFR3-TACC3 (F3T3) driver oncofusion proteins which promote tumor cell growth and survival. The F3T3-driven cancer cells are critically dependent on the constitutively active oncofusion kinase activity for survival and proliferation, a phenomenon known as "oncogene addiction". While inhibitors of FGFR kinases, such as erdafitinib and fexagratinib, are the treatment of choice for F3T3-driven cancers, most treated patients do not respond to the drug, suggesting insufficient inhibition of the oncofusion activity. Additionally, most tumor responses to FGFR kinase inhibitors are partial and short-lived, followed by development of resistance and disease progression within few months. Based on the oncogene addiction of cancer cells on the F3T3 activity, we hypothesized that rapid ablation of this oncofusion protein can induce robust apoptotic response, leading to significant tumor regression and sustained remissions. To this end, we explored the use of Proteolysis Targeting Chimeric (PROTAC) bifunctional small molecules which can rapidly degrade F3T3 oncofusion protein by harnessing the cell's own proteasome machinery. We synthesized more than 100 novel PROTAC small molecules targeting F3T3 fusion through combination of previously reported TACC3 ligands with various linkers and ligands of E3 ubiquitin ligases CRBN, VHL or XIAP. We evaluated our new anti-F3T3 PROTAC molecules using F3T3-driven bladder (RT4 and RT112) and breast (SUM185PE) cancer cell lines, which express two of the most common oncofusion variants. Several PROTACs were able to degrade the native F3T3 oncofusion protein and suppressed the growth of F3T3-driven cells. One PROTAC (KHS-C4) degraded F3T3 at low nanomolar concentrations, strongly suppressed the oncogenic pathway downstream F3T3 fusion, had excellent predicted pharmacological properties, acceptable in vitro liver microsome stability (Clint = 59 mL/min/kg for human and 318 mL/min/kg for mouse microsomes) and no in vivo toxicity. Importantly, low concentrations of KHS101-C4 (but not of FGFR3 kinase inhibitors) eradicated F3T3-driven cancer cells (but not cancer cells that do not express the F3T3 oncofusion) in vitro. These data suggest that chemically-induced degradation of the F3T3 fusion protein can lead to oncogenic signal ablation sufficiently potent to induce a lethal oncogenic shock and can thus yield more durable responses than FGFR3 kinase inhibitors. KHS-C4 was selected for further evaluation to establish in vivo proof of concept for the PROTAC-based therapeutic strategy against F3T3-driven cancers.



 Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea. Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5bb4e0b8d5df4124ad0dd08279a3f1803abead8" target='_blank'>
              Abstract B120: Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The onset of acquired resistance limits the efficacy of KRASG12C inhibitors in lung cancer patients. Experiments in preclinical model systems and retrospective analyses of patients treated with these inhibitors have both suggested that the activation of DNA repair pathways and G2/M kinases is common in lung cancers with acquired resistance to KRASG12C inhibitors. Here, we identified a shared vulnerability to the CDK12/13 selective inhibitor SR-4835 in several KRASG12C-mutant cell lines with acquired resistance to sotorasib. In pairs of KRASG12C-mutant parental and sotorasib-resistant cell lines, CDK12/13 inhibition suppressed DNA repair gene expression and induced mitotic arrest, which was exacerbated in sotorasib-resistant cells by failure to activate the G2/M checkpoint and suppression of an oncogenic isoform of TP73. Combined treatment with both sotorasib and SR-4835 yielded an additive response and suppressed the development of acquired resistance to either inhibitor. The inhibitor combination caused additive G1 and G2/M arrest, and the increased sensitivity of emerging drug-resistant cell populations to SR-4835 extended the duration of response over single-agent treatment in cell culture and in mouse models. These results support combining KRAS and CDK12/13 inhibitors to extend the duration of response in KRAS-mutant lung cancer. Patients with acquired resistance to sotorasib may benefit from follow-up monotherapy with CDK12/13 inhibitors, the first of which recently entered clinical trials. Targeting CDK12/13 thus offers a promising strategy to overcome or prevent resistance to KRAS inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5df2549b13618f9401530fb07b8c41d871bb9c6" target='_blank'>
              Targeting CDK12/13 Drives Mitotic Arrest to Overcome Resistance to KRASG12C Inhibitors.
              </a>
            </td>
          <td>
            Yaakov E. Stern, Pompom Ghosh, John Peroza, Hitendra S Solanki, Denis Imbody, Liznair Bridenstine, Thiyagamurthy Pandurangan, Sylvia M Frydman, Kathryn N Nickens, Hannah L Walker-Mimms, Kruthi Suvarna, Neelkamal Chaudhary, Andrii Monastyrskyi, Derek Duckett, E. Haura
          </td>
          <td>2025-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Reprogramming gene expression at the translational level drives intestinal tumorigenesis. Codon decoding during translation elongation relies on tRNA modifications, while their pathological relevance in colorectal cancer remains to be elucidated. Here, we show that AlkB homolog 8 (ALKBH8), a uridine 34 (U34) tRNA methyltransferase, is a direct target of Wnt/β-catenin and is upregulated in colorectal cancer. Genetic ablation of ALKBH8 inhibits the development of intestinal tumors in Apcmin/+, azoxymethane/dextran sulfate sodium (AOM/DSS), and xenograft models. Loss of ALKBH8 induces ribosome pausing at adenine-ending codons, impairing the translation elongation of mRNAs enriched with these codons. Specifically, ALKBH8 regulates the translation of KRAS proto-oncogene in a codon-dependent manner. Rescue experiments demonstrate that the methyltransferase activity of ALKBH8 is required for its translation-promoting function. Together, our findings reveal ALKBH8-dependent mRNA translation as a critical mediator of intestinal tumorigenesis, underscoring its potential as a promising target for colorectal cancer therapy. ALKBH8 is a tRNA methyltransferase, specifically modifying the wobble uridine (U34) in tRNAs. Here, the authors discover that ALKBH8 is a direct transcriptional target of β-catenin, and that mediates intestinal tumorigenesis through regulation of the translation of KRAS in a codon-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741e7e4c4eea499624351c733df3953216fc634e" target='_blank'>
              ALKBH8-mediated codon-specific translation promotes colorectal tumorigenesis
              </a>
            </td>
          <td>
            Yu Qian, Canlan Wu, Saisai Wei, Sujun Yan, Junxuan Peng, Lei Yu, Yunyi Gao, Jingyu Hou, Wentao Yu, Zhanghui Chen, Jun Zhang, Xiangwei Gao
          </td>
          <td>2025-10-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Early disease progression, including histological transformation, can occur in follicular lymphoma, but tumor heterogeneity makes comparison difficult. We analyzed cell-free DNA from 30 patients with follicular lymphoma to assess cell-free DNA concentration and gene mutations. Mutational profiles were also examined in 16 cases using DNA from formalin-fixed, paraffin-embedded tissue samples. Cell-free DNA concentrations were higher in cases with histological transformation. Both the number of mutations per patient and the number of mutated genes were greater in cell-free DNA than in formalin-fixed, paraffin-embedded tissue-derived DNA. Notably, TP53 mutations were more frequently detected in cell-free DNA. CIITA mutations were more commonly observed in cases with a low frequency of somatic hypermutation in VH genes. Cell-free DNA analysis in follicular lymphoma may become a complementary approach, overcoming the limitations of assessing disease status from single biopsy sites. Further studies will examine the usefulness of cell-free DNA in predicting disease progression in follicular lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc0cb12618903a19d09041fcad50558a34c24c" target='_blank'>
              Genetic analysis of cell-free DNA in follicular lymphoma in comparison with tissue-derived DNA.
              </a>
            </td>
          <td>
            Yoshikazu Hori, Hiroki Hosoi, M. Osuga, Ryuta Iwamoto, Fumiyo Maekawa, Tadashi Okamura, S. Murata, T. Mushino, Motomi Osato, Hitoshi Ohno, Nobuyuki Yamamoto, Shin-ichi Murata, Yasuhiro Koh, Sonoki Takashi
          </td>
          <td>2025-10-15</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eeb5a1876276ca40bae5f7f5ce7e3154844650e9" target='_blank'>
              Regulation of nucleolar dominance through allopolyploidization of hexaploid wheat
              </a>
            </td>
          <td>
            Yohta Hyuga, Tohru Nakano, Hirokazu Handa
          </td>
          <td>2025-11-06</td>
          <td>Theoretical and Applied Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triplet repeat expansion underlies multiple pathologies, including Huntington’s disease, often arising in somatic non-dividing tissues such as the brain. Despite identification of genetic modifiers, mechanistic insights remain limited. Using purified human proteins, we show that MutLγ (MLH1-MLH3), stimulated by MutSβ (MSH2-MSH3), incises DNA opposite an extrahelical loop on the 5’ side. This activity, with a moderate sequence preference, generates DNA nicks enabling Polδ-mediated displacement synthesis with the loop as a template, leading to expansion. PCNA confines these MutLγ incisions near the loop. FAN1, instead, preferentially targets the looped strand. RFC-PCNA stimulate and direct FAN1 nuclease to the 3’ boundary of the loop while restricting its exonuclease activity. No pre-existing nick is required. Following FAN1-RFC-PCNA action, Polδ removes the loop and resynthesizes DNA, causing contraction. FAN1 also directly inhibits MutLγ, preventing its activation by MutSβ. Our study illuminates both repeat expansion and contraction mechanisms and reveals the protective function of FAN1. Triplet repeat expansions drive several human disorders. Here, the authors define biochemically an expansion pathway promoted by MutLγ and MutSβ, and a protective pathway catalyzed by FAN1, POLD and RFC-PCNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d493a864ff7aa67cb76ef4f0ebec83dde1bc0b" target='_blank'>
              Mechanism of trinucleotide repeat expansion by MutSβ-MutLγ and contraction by FAN1
              </a>
            </td>
          <td>
            Issam Senoussi, Valentina Mengoli, Arianna Cerana, Andrea Rinaldi, Andrés Marco, Giordano Reginato, Simone Moro, A. Acharya, Megha Roy, Akshay Jayachandran, E. Cannavó, Ilaria Ceppi, Petr Cejka
          </td>
          <td>2025-10-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Ewing sarcoma (EwS) is a group of bone and soft tissue cancers in children and young adults. The EWS-FLI1 protein, the oncogenic driver in EwS, is a fusion of the FET protein, EWSR1, at its amino terminus, with the carboxy terminus of the transcription factor, FLI1. Since EwS cells have pronounced sensitivity to radiation and chemotherapy-induced DNA damage, the role of EWS-FLI1 in DNA repair is likely. Here, we demonstrate that EWS-FLI1 causes a defect in microhomology-mediated end-joining (MMEJ) repair. EWSR1 is a splicing factor that promotes the faithful splicing of the POLQ pre-mRNA, required for the expression of POLQ, a critical protein in the MMEJ pathway. Expression of EWS-FLI1, or loss of EWSR1, causes exon 25 skipping of the POLQ transcript, decreased POLQ protein expression, impaired MMEJ, and cellular sensitivity to inhibitors of NHEJ repair and HR repair, through the mechanism of synthetic lethality. Knockdown of EWS-FLI1 expression restores POLQ mitotic foci and increases MMEJ activity. Based on this mechanism, inhibitors of NHEJ or HR may provide a targeted therapy for patients with Ewing sarcoma and a novel Synthetic Opportunity. Highlights: 1) Ewing sarcoma tumors have a deficiency in POLQ expression and a corresponding loss of MMEJ activity 2) EWSR1 is a splicing factor that interacts with other splicing factors such as FUBP1 and KHSRP/FUBP2 to accurately splice the POLQ mRNA 3) The EWS-FLI1 fusion oncoprotein, or loss of EWSR1, causes a splicing defect, leading to exon25 skipping of the POLQ pre-mRNA and loss of POLQ expression 4)The MMEJ deficiency of EwS cells results in cellular sensitivity to inhibitors of Homologous Recombination or Non-Homologous End-Joining 5) Exon 25 skipping of POLQ mRNA is a predictive biomarker for HR inhibitors in human cancers 6) Ewing Sarcoma is the first example of a human cancer with a defect in the MMEJ DNA Repair pathway.



 Shuhei Asada, Guangli Zhu, Alan D. D'Andrea. Novel Synthetic Lethality Opportunity for Ewing Sarcoma [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A028.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b636e6a5df6be529dcc0f34802ce14751fa7642" target='_blank'>
              Abstract A028: Novel Synthetic Lethality Opportunity for Ewing Sarcoma
              </a>
            </td>
          <td>
            S. Asada, Guangli Zhu, Alan D. D'Andrea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Transcription and replication both utilize genomic DNA as a template, creating the potential for frequent interference between the two machineries. Such transcription–replication conflicts (TRCs) can compromise genome stability due to the abnormal accumulation and persistence of R-loops—three-stranded nucleic acid structures comprising a DNA:RNA hybrid and a displaced single-stranded DNA strand (Bhowmick R., et al., Mol Cell. 2023). To safeguard genome integrity, cells deploy multiple R-loop resolution factors that prevent R-loop-associated replication stress (Brickner J. R., et al., Mol Cell. 2022). Persistent R-loops are also associated with the stalling of RNA polymerase II (RNAPII) on chromatin, which further impedes replication fork progression (Zardoni L., et al., NAR 2021).
 Our observation of cells depleted of the chromatin remodelerHELLS, which displayed an accumulation of R-loops and stalled RNAPII on highly transcribed genes (Tameni A., Mallia S., et al., NAR 2024) in aggressive anaplastic large cell lymphoma (ALCL), led us to hypothesize that HELLS-dependent R-loop accumulation can induce replication stress and TRCs. To test replication forks progression, we performed DNA fiber analysis by sequentially labeling the cells with two thymidine analogs, 5-chloro-2′-deoxyuridine (CldU, red) and 5-iodo-2′-deoxyuridine (IdU, green). While control cells showed a CIdU/ldU ratio close to 1, indicating the absence of stalled forks, TLBR-2 and MAC2A HELLS-KD cells displayed an CIdU/ldU ratio close to 2, strongly indicating the presence of stalled forks in absence of HELLS. Then, we performed proximity-ligation assays (PLA) for measuring the presence of TRCs. Specifically, we used antibodies against phospho-forms of RNAPII, pSer5-RNAPII and pSer2-RNAPII, corresponding to initiation and elongation transcriptional complexes respectively and PCNA, as key component of DNA replicative machinery. We found that PLA signal between PCNA and pSer5-RNAPII, but not pSer2 RNAPII, increased in HELLS-KD cells compared to control, suggesting that the depletion of HELLS increases the collision between RNAPII and replicative complex at promoter-proximal sites before pause release. Importantly, ectopic expression of RNaseH1 in HELLS-KD cells reduced R-loop-dependent TRCs, confirming the functional involvement of R-loops in this process.
 Despite the large number of genes demonstrated or proposed to regulate R-loop homeostasis and, therefore, to impact TRCs and in turn on genome integrity, our knowledge of specific HELLS-regulated factors is still limited. To explore the mechanism contributing to TRCs and thus to genome instability, we performed gene set enrichment analysis focusing on genes belonging to functional categories related to chromatin regulation previously identified as HELLS transcriptional direct genes. Among them, mini chromosome maintenance 5 (MCM5) emerged as a potential candidate. Inducible knockdown of MCM5 significantly reduced cell proliferation, increased TRCs levels in lymphoma cells, as measured by pSer5-RNAPII–PCNA PLA foci, led to forks stall and to enhanced nuclear R-loop accumulation, phenocopying HELLS depletion. Notably, the ectopic overexpression of MCM5 in TLBR-2 HELLS-KDcells fully rescues cell proliferation reducing R-loop formation and the presence of TRCs. Additionally, perturbations observed upon MCM5-KD were dramatically reduced when R-loops were suppressed by overexpression of RNaseH1 indicating that, in absence of MCM5, the persistence of R-loop disrupts the replication fork process through TRCs, mimicking HELLS. These data highligh MCM5 as the downstream target of HELLS in replicative processes.
 Integrative analysis of MCM5, and RNAPII chromatin occupancy (ChIP-seq), alongside chromatin accessibility profiles (ATAC-seq) in TLBR-2 cells, further revealed that MCM5 was excluded from accessible, actively transcribed loci, confirming the not-transcriptional role of MCM5.
 Collectively, these findings position the axis HELLS/MCM5 as a critical regulator of replicative processes in aggressive ALCL. Disruption of HELLS/MCM5 activity induces aberrant TRCs and R-loop accumulation, ultimately compromising genome integrity and highlighting a potential therapeutic vulnerability in lymphoma cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caa8a5ddfc9df7b3d2ed0ec9481cd381c40701c0" target='_blank'>
              The chromatin remodeler hells regulates transcription-replication conflicts and genome integrity in ALK-negative anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Selene Mallia, Giulia Gambarelli, V. Manicardi, B. Donati, Emanuele Vitale, Federica Torricelli, E. Salviato, Magda Zanelli, N. Puccio, S. Luminari, Francesco Merli, A. Neri, A. Ciarrocchi, Valentina Fragliasso
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Centromeres are the chromosomal sites at which the kinetochore forms and attaches to spindle microtubules, directing chromosome segregation. Plasmids based on centromeres can maintain stability and distribute accurately during cell division, supporting them as effective genetic tools for research and biotechnological applications. Here, the centromeric regions on seven chromosomes of Ogataea polymorpha, a methylotrophic non‐conventional yeast with great potential in biotechnology, were located by ChIP‐seq with the native Cse4. The actual centromeric sequences of chromosomes 1, 2 and 5 were characterised, which are very different from those of other eukaryotes and each unique. Although long terminal repeats (LTR) were found in these centromeres, they are ‘solo LTR elements’ and not important for the function of centromeres. Hence, the O. polymorpha centromeres were categorised into small regional centromeres. O. polymorpha centromeric plasmids were constructed for the first time, which exhibit high genetic stability and compatibility. Application potential for multigene or multicopy expression was validated by the production of uricase and triterpene squalene. This research elucidates the structural features of O. polymorpha centromeres and constructs new, stable centromeric plasmids, expanding the phenomenal diversity of centromeres and providing a powerful genetic toolbox for multi‐node and multiple‐layered engineering of cell metabolism and physiological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8595896eccb7f21123d64f19b9f60bcb2fa63adb" target='_blank'>
              Centromeric Sequences in Ogataea polymorpha Genome Enable Development of Stable Multigene Expression Plasmid Tools
              </a>
            </td>
          <td>
            Yanfei Cheng, Yuanyuan Shi, Ning Sun, Xin Zhang, Mengwei Sun, Xiuping He
          </td>
          <td>2025-11-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769ab003240fead51b76ade414628f11f7033a1a" target='_blank'>
              Dedifferentiation-Driven Oncogenic Stemness Promotes Tumor-Sustaining Adaptability in the Intestinal Epithelium
              </a>
            </td>
          <td>
            Kylee Zgeib, Thomson Hui, Simon Garcia, Zahra Hashemi, Shima Nejati, Crystal Lim, Dahlia Matouba, Atharva Inamdar, Christina Li, Hossein Khiabanian, Binfeng Lu, Ansu Perekatt
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, Gavin Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, G. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, J. E. Meyer, S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea9805a426118d493f4c3e4084ab4ad0614ebdb" target='_blank'>
              DNA Repair in Ensuring Genome Stability and Human Health
              </a>
            </td>
          <td>
            N. Rechkunova, O. I. Lavrik
          </td>
          <td>2025-11-01</td>
          <td>Russian Journal of Genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Platinum-based drugs (Pt drugs) are widely used in cancer chemotherapy, yet their genome-wide binding patterns remain incompletely understood. Here, we present Pt sequencing (Pt-seq), an antibody-assisted, genome-wide method for mapping Pt-DNA adducts at single-base resolution. By using exo- and endonucleases to remove background, Pt-seq enables robust and sensitive profiling of binding sites for cisplatin, oxaliplatin, lobaplatin, and a Pt(IV) complex. Using Pt-seq, we identified tens to hundreds of binding clusters that are 10 to 20 kilobases in median length and highly consistent among different drugs, predominantly localizing to centromeric and ribosomal DNA regions. In cisplatin-resistant cells, we found significantly reduced binding within these regions. Moreover, we found that mutations in cancer cells can generate previously unidentified binding sites. On this basis, we demonstrated that ICR-191, an acridine orange compound capable of inducing G insertions, enhances cisplatin-DNA binding and sensitizes cells to cisplatin. Collectively, Pt-seq sensitively profiles Pt–DNA interactions and deepens our understanding of the genome-wide effect of chemotherapeutic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8b8af89d2820c69ce2eeb05e85318ac412b23f" target='_blank'>
              Pt-seq unveils the genomic binding pattern of platinum-based drugs
              </a>
            </td>
          <td>
            Chuyun Shao, Mingyang Li, Liuyi Liu, Dongsheng Bai, Jinmin Yang, Cong Liu, Linxiang Cai, Xiaoyong Wang, Jinying Peng, Zijian Guo, C. Yi
          </td>
          <td>2025-10-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="


 Nucleophosmin mutations are among the most common aberrations in acute myeloid leukemia (AML). We previously uncovered a novel pivotal role for mutant NPM1 (NPM1c) in the nucleus, where it binds chromatin at key self-renewal gene loci, such as HOXA/B and MEIS1, directly regulating oncogenic transcription. How NPM1c regulates the transcriptional activity of its chromatin targets remains unclear. To identify potential nuclear cooperating partners of NPM1c, we employed TurboID proximity labeling of NPM1c and wild-type NPM1 (NPM1wt). Mass spectrometric analysis revealed an enrichment of nucleoporins in the NPM1c interactome, with Nucleoporin 98 (NUP98) emerging as a top candidate.
 NUP98, a key component of the nuclear pore complex (NPC), has been established as a critical oncogenic player in AML as part of oncogenic fusion proteins. In these oncofusions the N-terminal FG (phenylalanine-glycine) repeat domain of NUP98 is preserved and fused to a diverse set of partner genes. NUP98-fusion partners are believed to mediate the chromatin binding specificity of the oncoprotein while the NUP98 FG repeats have been shown to form biomolecular condensates that enhance transcriptional output. Interestingly, wild-type NUP98 (NUP98wt) itself has been linked to gene regulatory functions during embryonic development and normal hematopoiesis. However, the role of NUP98wt in transcriptional regulation in the context of leukemia remains unresolved.
 To explore a potential cooperation between NUP98wt and NPM1c we first validated their interaction by co-immunoprecipitation assays. We next wanted to determine whether NUP98 and NPM1c are recruited to the same chromatin sites. To this end, we performed ChIP-seq in an NPM1c endogenous degrader cell line. These experiments revealed co-occupancy of NUP98 at NPM1c target genes and a significant reduction of NUP98 binding upon degradation of NPM1c. These findings suggest that NPM1c is required for the recruitment of NUP98 to key oncogenic target loci.
 To dissect the functional role of NUP98wt in NPM1c AML, we performed loss of function studies. CRISPR mediated knock-out of NUP98 in NPM1 mutant (OCI-AML3) cells resulted in a significant repression of NPM1c target genes, including HOXA/B and MEIS1. Given the essential nature of NUP98, we also engineered an endogenous NUP98 degrader system in OCI-AML3 and NPM1wt (MV4-11) cells to allow for the study of the effects of rapid protein depletion. Upon degradation of NUP98, OCI-AML3 cells exhibited robust myeloid differentiation (~70% increase in CD11b) and increased apoptosis (~50% increase in Annexin V+). MV411 cells on the other hand did not show the same differentiation phenotype. To characterize the immediate transcriptional consequences of NUP98 depletion, we performed nascent RNAseq after 1 and 3 hours of degradation. In the first hours of degradation we observed a rapid and significant repression of NPM1c target genes such as HOXA/B cluster genes, MEIS1, SMC4, TNRC18, SATB1/2 and UNCX, thus demonstrating the strong dependency of the NPM1c-driven transcriptional program on the presence of NUP98. In contrast, in MV4-11 cells, NUP98 loss had no effect on HOXA/MEIS1 expression, suggesting a leukemia subtype-specific role of NUP98wt.
 To delineate the domains of NUP98 required for its chromatin function, we performed a CRISPR gene tiling screen of full-length NUP98 (635 sgRNAs) and based on this we generated several domain-deletion mutants for rescue experiments in the NUP98-degrader cell lines. Retroviral overexpression of the N-terminal FG-repeat containing domains of NUP98 was sufficient to rescue cell viability and prevent the differentiation phenotype typically observed upon NUP98 degradation. Strikingly, gene expression analysis showed that the NUP98 FG-repeats were sufficient to restore the expression of HOXA10, MEIS1, and HOXB9 expression.
 Overall, our study identifies NUP98wt as a critical cofactor for NPM1c-driven oncogenic transcription. NPM1c recruits NUP98 to its chromatin targets, functionally substituting for an oncogenic fusion partner to concentrate transcriptional condensates at oncogenic loci. This establishes a previously unrecognized, subtype-specific dependency on NUP98wt in leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ff1e4b1ddc4587a7bab297c9d60234d4ef12e51" target='_blank'>
              Wildtype NUP98 cooperates with NPM1c to drive oncogenic transcription
              </a>
            </td>
          <td>
            Sara Gama, , H. González Dammers, Elena Haarer, Sara Peruzzi, José Gonçalves-Dias, J. Klusmann, Lu Yang, Renee Chen, Chun-Wei Chen, Stamatis Papathanasiou, B. Falini, Lorenzo Brunetti, Hannah J Uckelmann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="

 The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes.



 Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis.




 Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2
 IGFBP3
 +, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation
 in vitro
 .





 We identified immunoregulatory C2
 IGFBP3
 + melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Huabao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) and RNA N⁶-methyladenosine (m6A) have been linked to leukemia drug resistance. However, whether and how lncRNAs and m6A coordinately regulate resistance remain elusive. Here, we show that many differentially expressed lncRNAs enrich m6A, and more lncRNAs tend to have higher m6A content in CML cells resistant to tyrosine kinase inhibitors (TKIs). We demonstrate the broad clinical relevance of our findings, showing that upregulation of top-ranked lncRNAs (e.g., SENCR, PROX1-AS1, LINC00892) in TKI-resistant cell lines occurs in CML patients at the diagnostic stage, blast crisis phase, or not responding to TKIs compared to the chronic phase or TKI responders, respectively. Higher lncRNAs predict drug resistance and shorter survival duration. The knockdown of SENCR, PROX1-AS1, or LINC00892 restores TKI sensitivity. Mechanistically, upregulation of PROX1-AS1, SENCR, and LINC00892 results from FTO-dependent m6A hypomethylation that stabilizes lncRNA transcripts and empowers resistant cell growth through overexpression of PI3K signaling mediators (e.g., ITGA2, F2R, COL6A1). Treatment with PI3K inhibitor alpelisib eradicates resistant cells in vitro and in vivo, with prolonged survival of leukemic mice through downregulation of F2R, ITGA2, and COL6A1. Thus, the lncRNA-m6A-PI3K cascade represents a new non-genetic predictor for drug resistance and poorer prognosis in cancer, and a pan-cancer mechanism underlying TKI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c9b002cf98c0cb00e810018cf5908e319ebaf5" target='_blank'>
              The RNA N6-methyladenosine methylome coordinates long non-coding RNAs to mediate cancer drug resistance by activating PI3K signaling
              </a>
            </td>
          <td>
            Yanhong Tan, Changli Zhou, Sicheng Bian, Huiqin Bian, Yanan Ren, W. Walter, Jiuxia Pang, Tao Cheng, Hongwei Wang, Yuchao Yang, Wenzheng Guo, Lingli Zhang, A. Al-Kali, M. Litzow, Xiaonan Han, Jianhua Yu, Rendong Yang, Gang Huang, Gregor Hoermann, William Tse, Shujun Liu
          </td>
          <td>2025-11-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, , George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, , Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84210062581c67c09a67215ecb3a57b62da836b" target='_blank'>
              Adenine DNA methylation associated with transcriptionally permissive chromatin is widespread across eukaryotes.
              </a>
            </td>
          <td>
            Pedro Romero Charria, Cristina Navarrete, Vladimir Ovchinnikov, Lan Xu, Luke A. Sarre, Victoria Shabardina, Ewa Księżopolska, Elena Casacuberta, David Lara-Astiaso, Arnau Sebé-Pedrós, Alex de Mendoza
          </td>
          <td>2025-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Introduction: The t(4;14) translocation subtype of multiple myeloma (MM) is characterized by overexpression of NSD2, a histone methyltransferase which catalyzes di-methylation of histone H3 lysine 36 (H3K36me2). About 12% of MM patients have t(4;14) and this is associated with poor outcome. If and how NSD2 drives unique epigenetic vulnerabilities in this subtype of MM is not well understood.
 Methods: Primary MM samples from the MMRF CoMMpass trial (NCT01454296) were analyzed by Whole Genome Bisulfite Sequencing (WGBS). CRISPR/Cas9 was used to genetically ablate the NSD2 translocated allele in KMS11 and KMS18 (hereafter referred to as NSD2-high and NSD2-low). Multiple single cell clones were isolated and confirmed by immunoblot analysis and sequencing. KMS11 NTKO and TKO models (Kuo et al. Molecular Cell 2011) were also used for CUT&Tag and DNAm analysis. Cell responses to decitabine (DAC) and GSK-3685032 were evaluated by flow cytometry for Annexin V and live/dead staining on day 4. Dynamic BH3 profiling (DBP) was performed similar to previously described (Matulis, ASH 2024). Gene expression was assessed using RNAseq with ribosomal RNA depletion (Kapa).
 Results: Analysis of CoMMpass samples showed higher levels of DNAm in the t(4;14) subtype compared to other subtypes (p=1.17x10-12). Consistent with DNAm being connected to NSD2 overexpression, KMS11 NSD2-high cells had higher DNAm than isogenic NSD2-low cells (p<1x10-9). Elevated DNAm in NSD2-high cells co-occurred in genomic regions with higher levels of H3K36me2 (OR=1.85, p<1x10-15). Treating KMS11 and KMS18 NSD2 isogenic cell lines with the DNA hypomethylating agent DAC showed NSD2-high cells are significantly more sensitive to DNAm inhibition (p<0.01 at ≥100 nM for KMS11 and p<0.001 at ≥500 nM for KMS18). This sensitivity to DNAm inhibition was seen again using the DNMT1 enzymatic inhibitor GSK-3685032 (p<0.001 at ≥100 nM). DBP of KMS18 showed NSD2-high cells increased mitochondrial priming when treated with DAC (p<0.001) whereas NSD2-low cells did not significantly change, consistent with the increased apoptosis observed in NSD2-high cells.
 RNAseq of KMS11 NSD2-high and NSD2-low clones treated with 0, 100, or 500 nM DAC showed 3,599 differentially expressed genes (FDR<0.01, fold-change>2). Genes downregulated with DAC treatment were common between NSD2-high and NSD2-low cells and included genes involved in cell cycle and MYC targets as indicated by Gene Set Enrichment Analysis (GSEA). Genes upregulated with DAC-treatment were more distinct between NSD2-high and NSD2-low cells, with GSEA indicating interferon response and TNFa signaling via NFKB more significantly induced in NSD2-high cells (FDR<0.0023) than in the NSD2-low cells (FDR>0.13).
 Further investigation of DAC-induced RNAseq shows an increase in transcription of endogenous retroviruses (ERVs), specifically in the NSD2-high KMS11 cells. This suggests that hypomethylating agents induce transcription of these ERVs, which then induce an intracellular interferon response, and this is specific to NSD2-high MM. Consistent with this, these ERVs had higher levels of DNAm in isogenic KMS11 NSD2-high cells as compared to NSD2-low cells (FDR<0.01). This was also observed in CoMMpass samples where higher DNA methylation was found at these ERVs in t(4;14) MM.
 Conclusions: NSD2-mediated increases in H3K36me2 in t(4;14) MM correspond with higher DNAm as compared to other MM subtypes. This is consistent with data from other cell types suggesting H3K36me2 is recognized by the PWWP domains of the DNMT3 DNA methyltransferasaes. Multiple NSD2 isogenic models suggest this creates a preferential sensitivity to DNA hypomethylating agents in NSD2-high MM. This phenomenon was shown both with DAC, a cytosine analog that incorporates into DNA, as well as with GSK-3685032, a DNMT1 enzymatic inhibitor. Both Annexin V staining and DBP indicated increased apoptosis, specific to NSD2-high cells. In addition, RNAseq data suggests this increased apoptosis is due to re-expression of ERVs that are triggering a viral defense response and interferon signaling. Consistent with this, t(4;14) cell line models and patient samples had higher levels of DNAm at ERVs, suggesting these are silenced through DNAm in t(4;14), but by other mechanisms in non-t(4;14) MM. Together, these data indicate NSD2 remodels the epigenome of t(4;14) MM creating unique epigenetic dependencies that can be exploited with readily available DNA hypomethylating agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c372d03852ab31bfaf4fcce61c8d2085a69b5cdf" target='_blank'>
              NSD2 expression remodels DNA methylation, creating a targetable epigenetic dependency in t(4;14) multiple myeloma
              </a>
            </td>
          <td>
            Anna Baj, Jonathan C Patton, Robert M Chavez, Shannon M Matulis, Doris R. Powell, Mala Shanmugam, Arun Wiita, Constantine S Mitsiades, J. Keats, Jonathan Licht, S. Lonial, Lawrence Boise, Benjamin G. Barwick
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="


 Mosaic loss of chromosome Y (LOY) is a highly prevalent somatic event in many cancers (~30% of male tumors) and is frequently observed in males with age-associated clonal hematopoiesis of indeterminate potential (CHIP; ~50% by age 70-80 years). However, the mechanistic role of LOY in cancer remains poorly understood. Some studies suggest LOY is, in part, a manifestation of inherent genomic instability and a marker of biological aging, but mounting evidence supports an alternative hypothesis — that LOY is itself a cancer driver.
 We analyzed >1500 male patients with acute myeloid leukemia (AML) and found that ~10% harbored somatic LOY. Notably, LOY was significantly enriched (p<0.01) in specific subtypes — most strikingly in 54% of male patients with RUNX1::RUNX1T1 (AML1-ETO) AML — including pediatric cases. These data suggest that LOY may actively contribute to the pathogenesis of certain AML subtypes rather than being a mere passenger event or age-related artifact. Supporting clinical relevance, Schlenk et al. (2004) reported that LOY in AML1-ETO AML predicts shorter remission and inferior overall survival.
 Several candidate Y-encoded genes — including UTY/KDM6C, KDM5D, CSF2RA, TMSB4Y, and CRLF2 — have been implicated as drivers of LOY's oncogenic effects across different malignancies. However, loss of multiple genes, rather than any single gene, may underlie the phenotypic consequences of LOY, highlighting the need for robust, lineage-specific, and otherwise isogenic models to study pathogenic mechanisms.
 To investigate this, we immortalized primary murine hematopoietic progenitors in an ex vivo culture system using a retrovirally-expressed estrogen receptor-HoxB8 fusion protein, which maintains hematopoietic cells as immature, multipotent progenitors in the presence of estradiol and allows synchronized differentiation upon estradiol removal. Using two CRISPR/Cas9 strategies targeting ChrY repeat elements, we generated isogenic progenitors with or without ChrY deletion, confirmed by qRT-PCR and DNA PCR. In short-term culture of undifferentiated myeloid progenitors, LOY had no significant effects on growth kinetics, confirmed by flow cytometry cell tracing and cell cycle analysis. However, in longer-term competitive co-culture assays, LOY cells showed significantly reduced growth versus wildtype ChrY (WTY) cells, evident by day 7. Similar paradoxical reduced fitness has been seen previously with single gene models of clonal hematopoiesis-associated mutations. Transcriptomic profiling revealed 195 upregulated and 198 downregulated genes in LOY versus WTY cells (padj<0.05). Downregulated ChrY-encoded genes including Uty, Ddx3y, and Kdm5d validated LOY, and pathway enrichment of non-ChrY genes revealed significant downregulation (p=0.008) of oxidative phosphorylation genes in LOY cells, suggesting metabolic reprogramming.
 Following estradiol withdrawal, which inactivates the HoxB8 stem cell transcription factor and induces terminal differentiation, both WTY and LOY Hoxb8 cells lost the progenitor marker Cd117 (c-Kit) and upregulated mature myeloid markers Gr-1 and Cd11b within four days. However, LOY cells consistently showed accelerated myeloid maturation, confirmed by flow cytometry and morphology.
 In contrast to cells with LOY alone, in cells co-expressing the oncogene AML-ETO9a (AE9a), LOY significantly impaired myeloid differentiation compared to WTY. RNA-seq of AE9a cells with and without LOY revealed 638 up- and 540 down-regulated genes (padj<0.05), with significant reprogramming of translation, stress response, and biosynthetic pathways. Functionally via in vitrotransformation assays, both AE9a WTY and LOY cells proliferated and survived two-fold longer than empty vector controls (6 vs 12 days); however, only AE9a LOY cells sustained long-term viability beyond 21 days, while AE9a WTY cells progressively died off. In methylcellulose colony formation assays, AE9a LOY cells were the only genotype that produced large rapidly growing colonies (>200 cells), while other groups (AE9a WTY, and LOY or WTY without AE9a) formed only microcolonies (<20 cells).Together, our findings suggest that LOY impedes myeloid differentiation and enhances self-renewal in concert with specific leukemia oncogenes, revealing a context-dependent effect to promote clonal expansion and leukemic progression. These results challenge the paradigm that LOY is merely a bystander in leukemia and support further mechanistic study of its role in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c7b5fd6dd10d7134f1746ffc1fb097a24dfdd3" target='_blank'>
              Loss of the Y chromosome drives context-dependent transcriptional reprogramming in myeloid progenitors and AML1-ETO acute myeloid leukemia (AML)
              </a>
            </td>
          <td>
            Hayden Bell, Christopher A G Booth, Andrew Lane
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose
In uveal melanoma (UM), coexistence of the fatal monosomy 3 with the benign gain of chromosome 6p occurs rarely. The spatial organization of chromosomes can be influenced by the nucleoli, which become larger under hyperglycemia. We therefore hypothesized that hyperglycemia may be responsible for chromosome-specific aberrations in UM and analyzed its effect on nucleolar organization, chromosome territories, and missegregation rates in vitro.


Methods
UM cell lines 92.1 and OMM2.5, UM cells from the primary tumors of two patients, and Tenon fibroblasts from a control were incubated in normo- or hyperglycemic medium (with 5.5 or 25 mM glucose, respectively) for one day, followed by the mitotic arrest with Nocodazole for 18-24 hours and recovery in fresh medium. Co-detection of proteins with the centromeres of chromosomes 3 and 6 was performed by two-dimensional immunofluorescent in situ hybridization.


Results
In the UM cells undergoing interphase, hyperglycemia promoted the dislocation of chromosome 3 toward the center along with nucleolar growth. During prometaphase, the mean angle between the centromeres of chromosome 3 was reduced below 90° under hyperglycemia (P = 0.02). During the later mitotic phases, hyperglycemia resulted in a 3.8-fold increase in the missegregation rate of chromosome 3 in UM cells (P < 0.001), whereas chromosome 6 rather than 3 was more prone to missegregation in the normoglycemic UM cells and hyperglycemic Tenon fibroblasts.


Conclusions
Hyperglycemia can favor chromosome-specific aneuploidies by altering chromosome territories in a cell-type dependent manner. Prevention of hyperglycemia may be a simple therapeutic approach to impede the generation of monosomy 3 in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/216ee98cebdd686eaeabfdc2e4a556eeb9b8ce09" target='_blank'>
              Missegregation of Chromosome 3 and Generation of Monosomy 3 in the Proliferating Uveal Melanoma Cells Under Hyperglycemia.
              </a>
            </td>
          <td>
            A. Tura, Svenja R Sonntag, N. C. V. von Bubnoff, Malte Spielmann, Salvatore Grisanti
          </td>
          <td>2025-12-01</td>
          <td>Investigative ophthalmology & visual science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epithelioid hemangioendothelioma (EHE) is a difficult to treat vascular sarcoma defined by TAZ-CAMTA1 (TC) or YAP-TFE3 (YT) fusion proteins. Human cell lines needed to further understand the pathogenesis of EHE have been lacking. Herein, we describe a method to generate EHE extended primary cell cultures. An integrated multi –omic and functional approach was used to characterize these cultures. The cell cultures, relatively homogenous by single cell RNA-Seq, demonstrated established characteristics of EHE including increased proliferation, anchorage independent growth, as well as the overall gene expression profile and secondary genetic alterations seen in EHE. Whole genome sequencing (WGS) identified links to epigenetic modifying complexes, metabolic processes, and pointed to the importance of the extracellular matrix (ECM) in these tumors. Bulk RNA-Seq demonstrated upregulation of pathways including PI3K-Akt signaling, ECM/ECM receptor interaction, and the Hippo signaling pathway. Development of these extended primary cell cultures allowed for single-cell profiling which demonstrated different cell compartments within the cultures. Furthermore, the cultures served as a therapeutic platform to test the efficacy of TEAD inhibitors in vitro. Overall, the development of primary EHE cell cultures will aid in the mechanistic understanding of this sarcoma and serve as a model system to test new therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4728cebf1ce6a2c3cd2884745cce7881e7bc18e9" target='_blank'>
              EHE cell cultures are a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M Bellizzi, Anthony N. Snow, Patrick Breheny, M. Chimenti, M. Tanas
          </td>
          <td>2025-10-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) remains one of the most lethal malignancies, primarily due to its robust resistance to genotoxic therapies, such as chemotherapy and radiotherapy. Understanding the mechanisms underlying this resistance is essential to improve clinical outcomes. Here, we identified trimethylguanosine synthase 1 (TGS1), previously known for its role in RNA modification, as a critical mediator of homologous recombination (HR) repair that specifically contributes to resistance in PAAD. TGS1 was significantly overexpressed in PAAD tissues, correlating strongly with advanced disease stages, therapy resistance, and poor patient prognosis. Following DNA damage, ATM kinase phosphorylated TGS1 at serine residues S389 and S531, which mediated its direct interaction with BRCA1 and subsequent recruitment of BRCA1 to DNA damage sites. The phosphorylation-dependent interaction enhanced HR repair efficiency, enabling cancer cells to survive genotoxic stress. Depletion or pharmacological inhibition of TGS1 induced HR deficiency and markedly enhanced sensitivity to DNA-damaging agents, particularly PARP inhibitors, in PAAD cell lines in vitro and in both cell line-derived and patient-derived xenograft models in vivo. Collectively, these findings uncover an ATM-TGS1-BRCA1 signaling axis that promotes DNA repair and resistance to genotoxic therapies in pancreatic cancer, positioning TGS1 as a promising predictive biomarker and therapeutic target to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abe0d36d758071ca95a0b7efd96da1ea51c63685" target='_blank'>
              Targeting the ATM-TGS1-BRCA1 Axis Overcomes Genotoxic Therapy Resistance in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Changying Li, Xuran Zhao, Xinyan Li, Chunyan Wang, Zhenyu Huo, Xin Xu, Wenjuan Kang, S. Nowsheen, Khaled Aziz, Guangyi Sun, Zhihua Liu, Zhenkun Lou, Min Deng
          </td>
          <td>2025-11-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Introduction A long-standing question in multiple myeloma (MM) is the nature of its stem cell origin and whether transformation involves compartments prior to terminal plasma cell (PC) differentiation. Of particular interest is the potential involvement of pre-germinal center (GC) naïve B cells or even earlier stages such as hematopoietic stem cells (HSCs) or B-cell precursors. Advances in sequencing and lineage-resolved cell sorting allow re-evaluation of this question, with particular relevance to the origins of relapse following deep MRD-negative responses to therapies targeting mature PCs. The extent and pattern of somatic hypermutation (SHM) in immunoglobulin (Ig) genes can provide insight into timing and nature of GC exposure during clonal evolution. We analyzed Ig gene rearrangements from whole-genome sequencing (WGS) with compartment-specific sorting to define hematopoietic relationships and pinpoint the cellular origin of MM transformation.
 Methods We performed WGS on 353 matched tumor-normal pairs from 294 MGUS, SMM, NDMM, and RR MM patients (median coverage: 64× tumor, 34× normal). Ig loci were profiled using IgCaller, which utilizes split-read and discordant-pair evidence to reconstruct V(D)J junctions, detect abnormal class-switch recombination (CSR) events, and call chromosomal translocations. Variant calling was performed via the harmonized Myeloma Genome Pipeline 1000. Our workflow, shmCaller (https://github.com/pblaney/shmCaller), was applied to sorted compartments to quantify and track SHM fingerprinting across lineages. Thirteen patients underwent flow sorting into hematopoietic and B-cell subsets: HSCs, multipotent progenitors (MPPs), lymphoid-primed MPPs, pre/pro-B cells, naïve B cells, memory B cells, normal plasma cells, and malignant plasma cells (MPCs), with an immunophenotyping panel (CD34, CD38, CD10, CD19, CD45RA, CD90, CD27, CD56, CD117 and CD138) and each fraction underwent 100x whole-exome sequencing (WES) with a uniform pipeline.
 Results Productive V(D)J rearrangements were detected in 82% of cases: IGH+IGK (35%), IGH+IGL (15%), IGK only (18%), IGL only (8%), and IGH only (6%). V, D, and J gene usage (e.g., IGHV3-30, IGKV1-5, IGLV1-40) showed no major differences across MGUS, SMM, and MM. Canonical translocations were found in 43% of cases: t(11;14) (21%), t(4;14) (12%), t(8;14) and t(6;14) (3% each), t(14;16) and t(14;20) (2% each), with breakpoints mapped to IG constant and VDJ regions. Novel non-canonical rearrangements were detected: IGH to 11q13.4 (13%), 19p13.3 (PIP5K1C, 9%), 19p13.11 (2%), and IGK to 20q11.21 upstream of BCL2A1 (14%). Intraclonal SHM analysis of IGHV sequences revealed variable mutation burdens across MGUS, SMM, and MM. Productively rearranged IGHV cases exhibited diverse SHM patterns. Several cases showed additional mutations within dominant Ig clones, consistent with ongoing or reinitiated SHM, potentially driven by GC reentry or persistent chromatin accessibility.
 We also performed low-input WES across all flow sorted compartments. Preliminary analysis revealed cases with mutational burden restricted to MPCs and others with shared mutations in bone marrow-confined pre-GC populations. One SMM case harbored a pathogenic TP53 L137Q mutation across HSCs, MPPs, and MPCs, consistent with possible early acquisition in a precursor selectively contributing to PC differentiation. These findings suggest that initiation of transformation events may arise at multiple stages of hematopoietic development, informing models of clonal evolution, disease persistence, and potential relevance of clonal hematopoiesis of indeterminate potential to MM pathogenesis. Additional high-depth sorting and sequencing are underway and will be integrated into the final analysis to refine the temporal and lineage context of MM transformation.
 Conclusion Our integrated analysis of Ig rearrangements, SHM patterns, and lineage-resolved sequencing supports a model in which multiple myeloma originates from clonal events across hematopoietic development, from early progenitors to differentiated B-cell compartments. While malignant transformation likely occurs later within the plasma cell lineage, early driver mutations in stem-like cells and ongoing SHM activity underscore the complexity of clonal evolution. These findings highlight the need to define lineage-specific vulnerabilities and temporal dynamics throughout the myeloma disease course.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d3d685814981f3ac4a04d887670caed2833481f" target='_blank'>
              On the cellular origins of multiple myeloma
              </a>
            </td>
          <td>
            Dennis Lee, P. Blaney, Reva Shenwai, Sanghoon Lee, Di Zhang, Marc J Braunstein, David Kaminetzky, Lisa Cervia, Kern Hast, Timothy Chu, L. Winterkorn, B. Diamond, N. Korde, Alexander M Lesokhin, Elizabeth Brown, N. Robine, K. Maclachlan, Francesco Maura, Saad Usmani, O. Landgren, Brian Walker, Faith Davies, Iryna Berezniuk, O. Lahoud, Christopher Y. Park, G. Morgan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Circular RNAs (circRNAs), a novel class of single-stranded, covalently closed non-coding RNA molecules, have been increasingly recognized for their roles in the cellular landscape over the past decade. These molecules are predominantly localized within the cytoplasm of eukaryotic cells and have been implicated in the pathogenesis of a spectrum of cancers. Despite this, the specific contributions of circRNAs to hepatocellular carcinoma (HCC) have yet to be fully delineated. Our research has identified that cyclic RNA0002898 is downregulated in HCC, with its reduced expression correlating with advanced tumor grade and diminished patient survival outcomes. Furthermore, we have demonstrated that the transcription factor forkhead box C2 (FOXC2) regulates the expression of cyclic RNA0002898, and diminished levels of cyclic RNA0002898 in HCC are associated with enhanced tumor cell proliferation, migration, and invasion in vitro. Mechanistic insights revealed that cyclic RNA0002898 could modulate the expression levels of the tumor suppressor a disintegrin and metalloproteinase with thrombospondin motif 5 (ADAMTS5) by antagonizing miR-145, thereby inhibiting angiogenesis and suppressing the growth of HCC cells. Our findings collectively elucidate the regulatory role, functional significance, and underlying mechanism of cyclic RNA0002898 in HCC, a previously uncharted relationship. The potential prognostic implications of cyclic RNA0002898 and its therapeutic potential as a target in HCC warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7284a80109b14d76997c2fcead1b4dc67a67eca2" target='_blank'>
              Transcription factor FOXC2 regulates miR-145/ADAMTS5 axis to inhibit angiogenesis in hepatocellular carcinoma via circular RNA 0002898
              </a>
            </td>
          <td>
            Jiangbei Yuan, Zixiang Pan, Fei Lv, Yue Zhao, Wei Zheng, Qiaoqiao Yin, LanJie Wu, Jianli Yu, Chengan Xu, Qiang He, Hongying Pan
          </td>
          <td>2025-11-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="G-quadruplexes have been identified as a promising anti-cancer target because of their ability to modulate the stability of mRNAs encoding oncogenes, tumor suppressor genes, and other potential therapeutic targets. Deregulation of DNA damage and Unfolded Protein Response pathways in cancer cells may create vulnerabilities that can be exploited therapeutically. Previous studies have shown variations in the relative expression of DDR and UPR components in canine lymphoma and leukemia cell lines CLBL-1, CLB70, and GL-1. In the present study, we report the presence of G-quadruplex structures in these canine cell lines. Downregulation of the expression of DDR and UPR components at the mRNA level was observed in the CLBL-1 and CLB70 cell lines after stabilization of G4 structures using the ligand PhenDC3. In contrast, in GL-1 cells, important components of the DDR pathway, such as PARP1, GADD45A, and PIK3CB were upregulated in response to PhenDC3 treatment. Downregulation of DDIT4 mRNA expression, which encodes an important UPR component, was detected in the CLBL-1 and GL-1 cell lines after PhenDC3 exposure. These results suggest that G4 structures can be used to manipulate the expression of potential targets to treat lymphoma in dogs. A substantial enrichment of DNA replication and pyrimidine metabolism pathways was found in the GL-1 cell line after G4 stabilization. This finding suggests that PhenDC3 may induce DNA replication stress in this cell line. Collectively, these results support the feasibility of employing canine cancer cells as a model system to investigate the role of G-quadruplex structures in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266b5cfaedc49c459cf184dd7b4aeb1ea558a388" target='_blank'>
              Stabilization of G-Quadruplexes Modulates the Expression of DNA Damage and Unfolded Protein Response Genes in Canine Lymphoma/Leukemia Cells
              </a>
            </td>
          <td>
            Beatriz Hernández-Suárez, David A. Gillespie, Ewa Dejnaka, B. Obmińska-Mrukowicz, A. Pawlak
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) embryos exhibit widespread epigenetic defects, particularly aberrant DNA methylation. DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development, yet its role in SCNT embryos remains largely unknown. Here, the genome-wide 5hmC landscapes in mouse SCNT embryos are systematically profiled with parental allele specificity. It is revealed that both maternal and paternal genomes of donor somatic cells acquire a transient, sperm-like but attenuated and allele symmetric distribution of 5hmC at the 2-cell stage, distinct from the parental asymmetric pattern observed in naturally fertilized eggs. This is characterized by insufficient DNA hydroxymethylation of the X chromosome in female SCNT embryos, as well as resistance to 5hmC-associated DNA demethylation at germline imprinting control regions (gICRs). While de novo 5hmC generation is closely associated with initial DNA demethylation during somatic-to-zygotic transition, it later becomes uncoupled from ongoing methylation changes. Importantly, global elevation of 5hmC via Tet3 overexpression leads to premature activation of developmental genes at the 2-cell stage and severely impairs SCNT embryo development. These findings reveal unique dynamics and functional consequences of abnormal 5hmC remodeling in SCNT embryos, highlighting the precise regulation of 5hmC generation as a key epigenetic event for successful mammalian cloning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e7a45e185e638b752c93b5629a29c7ec2760ec" target='_blank'>
              5-Hydroxymethylcytosine Dynamics Reveals Coordinated Reprogramming of Parental Genomes and X Chromosome Dosage Balance in Mouse SCNT Embryos.
              </a>
            </td>
          <td>
            Zeming Xiang, Rui Yan, Jing Guo, Mengyao Wang, Xin Cheng, Fan Zhang, Tianzi Guo, Xin Long, Fan Guo, Dan Liang
          </td>
          <td>2025-11-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
DNA double-strand breaks (DSBs) are the most lethal and dangerous type of lesions with significant implications for both cellular function and organismal health. The number of DSBs (NDSBs) across the genome reflects DNA damage severity. However, current quantification methods mainly rely on next-generation sequencing, which is laborious and expensive. This study aims to provide a simple, low-cost, and high-throughput standard curve-based method for quantifying genome-wide DSBs.


METHOD
Genomic DNA from human, mouse, Arabidopsis, Saccharomyces cerevisiae, and Escherichia coli was digested by seven blunt-end restriction enzymes to generate DSB standards. Theoretical NDSBs for each standard were calculated based on restriction site frequency. Ligation-mediated quantitative PCR (LM-qPCR) was performed to obtain the Ct values, which were plotted against log-transformed NDSBs to construct standard curves. Method reliability was assessed by comparing results with neutral single-cell gel electrophoresis and γ-H2AX flow cytometry.


RESULTS
All genomes were successfully digested by seven blunt-end restriction enzymes to produce standard DSB fragments. Standard curves demonstrated high linearity (R2 > 0.95), with intra- and inter-assay coefficients of variation of 1.101% and 2.528%, respectively. The detection limit was below 100 DSBs. Quantification results strongly correlated with traditional DSB detection methods (|r| > 0.9).


CONCLUSION
This standard curve-based method enables accurate, reproducible quantification of genome-wide DSBs in various organisms. It is simple, low-cost, and easily standardized, offering a promising tool for applications in genotoxicity testing, environmental exposure monitoring, and DNA damage research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b4e2232775eb0b9347348ac65a428b9b495195" target='_blank'>
              Quantitative Analysis of DNA Double-Strand Breaks in Genomic DNA Using Standard Curve Method.
              </a>
            </td>
          <td>
            Lihuang Guo, Hanying Dai, Jiancheng Li, Chenwei Li, Yue Huang, Keqian Xu
          </td>
          <td>2025-10-18</td>
          <td>Journal of clinical laboratory analysis</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Inactivation of tumor suppressor genes (TSGs) impart a cellular fitness in cancers, including acute myeloid leukemia (AML). The silencing of TSGs without direct mutations presents challenges in cancer therapy but also presents a therapeutic opportunity to restore their function. In this study, we identified the transcriptional repressor ZBTB7A as a TSG that is downregulated in AML and associated with poor survival outcomes. Loss of ZBTB7A amplifies TNF signaling, driving a dysfunctional inflammatory state that accelerates leukemia progression. Mechanistically, the mRNA decay factor ZFP36L2 binds to the 3' untranslated region (3'UTR) of ZBTB7A, promoting its transcript degradation. To uncover therapeutic strategies, we developed a CRISPR-based screening approach coupled with in situ FISH-Flow, pinpointing KDM4 as a vulnerability to restore ZBTB7A function. Pharmacologic inhibition of KDM4 enhanced ZBTB7A expression, promoted terminal differentiation of leukemic cells, and demonstrated broad anti-leukemic activity across AML subtypes while preserving normal hematopoiesis. These findings reveal critical regulatory mechanisms of ZBTB7A and support epigenetic therapy as a promising strategy to reactivate its tumor suppressor function in hematologic cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaf7a74c25b86553fb4e494e6a32f6ab68f9d8f" target='_blank'>
              Epigenetic reactivation of a tumor suppressor program in AML.
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Arnuk, I. Aifantis, Hussein A. Abbas, A. Tsirigos, Sadik Karma, S. Rajendran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Chromosomal structural changes happen when genomic stability compromised, such as in disease, or in species hybrids. In these contexts, diminished control of repetitive elements have been reported, but the reasons for this are not yet well understood. There are causal associations between repetitive elements and phenotypes such as disease progression, leading us to the hypothesis that chromosomal structure may be affected by transposable elements (TEs). In an intraspecific hybrid Drosophila melanogaster cell line (PnM), the degree of pairing among trans homologous chromosomes was affected by the presence of nearby TEs; in particular, LINE and LTR elements, such as Baggins1 or Gypsy. Chromosomal pairing was significantly lower in windows containing TEs than in windows without any TEs. Pairing was also affected by TEs in mouse, which suggests a possible general association between TEs and pairing that is highly conserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b05bf5cfcd82a8fc82e293a52e5e86b57f277d" target='_blank'>
              Haplotype-resolved Chromatin Conformation Data reveals relationship between Transposable Elements and Chromosomal Pairing.
              </a>
            </td>
          <td>
            Luke K Genutis, Patrick Villanueva, Rita M. Graze, Jumana Alhaj Abed, Lauren M McIntyre, S. Nuzhdin
          </td>
          <td>2025-11-26</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Deregulation of gene expression is a hallmark of cancer, underlying the phenotypic changes fundamental to tumorigenesis and cancer progression. This multi-step process is controlled at different levels, from transcription, RNA localization, stability and translation up to posttranslational protein modifications and protein degradation. Posttranscriptional gene regulation (PTGR) has been indicated as a key determinant of aberrant protein abundance, yet to what extent cancer cells alter system-wide PTGR control remains poorly understood. Nonetheless, it is clear that RNA does not only serve as an intermediate molecule towards protein expression, but can be a highly versatile molecule with critical regulatory functions. These functions are executed through interactions between RNA-binding proteins (RBPs) that interact with their respective target RNAs (RNA-interactome) to control all facets of PTGR, including RNA biogenesis, editing, splicing, localization, and translation. In this project, we aim to elucidate the RNA-interactome of several glioma subtypes and their dynamics under treatment, focused on the highly aggressive glioblastoma (GB). GB is the most common malignant primary brain tumour in adults, with a particularly grim prognosis. There is a need for a more fundamental understanding of the underlying mechanisms behind the therapeutic failure seen in GB patients. Previous research has indicated that GB’s RNA-interactome is linked to treatment response, but a large-scale, RNA-biotype-independent comprehensive investigation is currently lacking. We aim to provide such an investigation using GB patient-derived cell lines, representing the heterogeneous patient population. Utilizing novel methodologies to isolate RNA-protein complexes, combined with a multi-omics approach, will allow for the dissection of the composition and properties of these networks, ultimately aiding our understanding of the regulatory mechanisms behind GB treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88c625388694e2493960d4ccc0cfc43279b68d2" target='_blank'>
              PATH-24. Understanding the RNA-regulatory interactome landscape of glioma treatment response
              </a>
            </td>
          <td>
            Thijs de Kock, Eleonora Leucci, R. Quintens, F. de Smet, Nicholas Rajan
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Cohesin is a multimeric protein complex that regulates three-dimensional chromatin architecture and gene regulation. In myeloid malignancies, mutations in the cohesin subunit STAG2 are common and associated with poor outcomes. STAG2 mutations frequently emerge as secondary drivers in myelodysplastic syndrome (MDS), following mutations in epigenetic modifiers such as TET2 and ASXL1 in clonal hematopoiesis of indeterminate potential (CHIP). While a chronic inflammatory milieu can select for mutant hematopoietic stem cells (HSCs), the specific cause of STAG2-mutant clonal expansion in MDS remains unknown. Using models of cohesin-mutant MDS, here we describe endogenous retroviral element (ERV) reactivation as a driver of inflammation and stem cell transcriptional programs during MDS development.
 Interferon (IFN) signaling is crucial for antiviral responses and has been linked to the transcriptional reactivation of genomic repeats, which can in turn create double-stranded RNA (dsRNA). We found that patients with STAG2-mutant MDS (n=39) had increased expression of Type 1 IFN signaling compared to wild type (WT) patients (n=126). To understand the mechanism behind STAG2-mutant IFN signaling and its role in cohesin-mutant myeloid malignancies, we first examined isogenic cell line models with and without STAG2 mutations. STAG2-mutant cells showed increased Type 1 IFN signatures, an accumulation of dsRNA, and increased transcription of open chromatin in intergenic regions, as identified by ATAC-Seq. Given the abundance of retroviral sequences in these regions, we hypothesized that altered chromatin structure and subsequent expression of ERV loci could be key drivers of inflammation and subsequent progressionof STAG2-mutant MDS.
 To address this hypothesis, we characterized differentially expressed transposable elements and their chromatin state in multiple isogenic STAG2-KO leukemia cell lines. We observed transcription of a specific ERV subfamily, HERV3-int and its flanking LTR4 in the absence of STAG2. While these loci were accessible in WT cells, the STAG2 mutation resulted in a loss of the repressive H3K9me3 histone mark and gain of the activating H3K27ac mark, driving transcription, and in specific cases also translation of ERV-derived peptides, which were presented on MHC Class I. Further analysis revealed the presence of the AP-1 transcription factor motifs within these overexpressed ERV loci and binding of canonical AP-1 complex members Fos and Fosl1. We also observed a significant increase in ERV expression in STAG2-mutant MDS patients, further highlighting the association between STAG2 mutations, Type 1 IFN signaling, and ERV reactivation in MDS. Experimentally, repression of the HERV3-int/LTR4 subfamily loci using CRISPRi reversed Type 1 IFN signaling in STAG2-mutant cell lines.
 Finally, we examined the role or ERV expression in disease progression from CHIP to STAG2-mutant MDS. We generated transgenic mouse models of Stag2-mutant MDS in the context of most frequently co-occurring CHIP lesions. These models showed features of MDS, including macrocytic anemia, thrombocytopenia, and monocytosis with an increase in immature bone marrow populations and significant B-cell aplasia. Analogously to human cells, we identified increased expression of the murine IAPEY4 ERV subfamily and its associated LTR in hematopoietic stem and progenitor cells across all of our Stag2-mutant MDS mouse models. Using scRNA-Seq from sorted lineage-Sca1+kit+ cells, we mapped the highest abundance of IAPEY4 expression to short- and long-term HSCs. To probe the role of ERV reactivation in Stag2-mutant MDS disease progression, we simultaneously repressed multiple members of the IAPEY4 subfamily using CRISPRi in Tet2/Stag2-mutant HSCs in vivo. ERV repression led to decreased inflammatory gene expression signatures and reversed multiple features of the MDS phenotype, including the expansion of short-term HSCs and B-cell aplasia.
 In summary, our data suggest that STAG2 loss of function mutations lead to chromatin remodeling that promotes ERV expression. These transcripts can then induce an IFN response, which drives HSC dysfunction in MDS, highlighting a cell-autonomous mechanism for inflammation, clonal selection, and MDS progression. We also find that STAG2-mutant MDS cells present ERV-encoded peptides on MHC Class I molecules. Thus, our data not only inform the pathogenesis of MDS development but also have significant therapeutic implications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a4be84b55b57275de2a79256cdf07a06db357ed" target='_blank'>
              Activation of endogenous retroviral elements drives inflammation and transformation in STAG2-mutant myeloid malignancies
              </a>
            </td>
          <td>
            Johann-Christoph Jann, Ariel Swett, Yihua Wang, Rishika Vadlamudi, Benjamin Braun, Jennifer Karlow, Amy Wang, Lin Han, Griffin Lawrence, Caroline N. Pitton, Caroline A. Conway, M. Meggendorfer, K. Clauser, Steven A. Carr, Jenn Abelin, T. Haferlach, Kathleen Burns, Zuzana Tothova
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Human DNA is continuously exposed to endogenous and exogenous agents that generate over 100,000 lesions per cell each day. In addition to damage to nucleobases, deoxyribose, and phosphate groups, a particularly harmful class of lesions involves non-canonical DNA termini—structures deviating from the canonical 3′-hydroxyl and 5′-phosphate ends. These aberrant DNA ends can obstruct essential DNA transactions and, if left unrepaired, contribute to cytotoxicity and mutagenesis. Their biological significance is further highlighted by the severe pathologies linked to deficiencies in DNA end-processing enzymes, including inflammation, cancer predisposition syndromes, neurodegeneration, and aging. This review highlights recent advances in our understanding of the formation, prevalence, and repair mechanisms of several key non-canonical DNA end structures, including 3′-phosphate, 3′-phosphoglycolate, 3′-α,β-unsaturated aldehyde and its glutathione derivative, 5′-deoxyribose-5-phosphate, 2′-deoxyribonucleoside-5′-aldehyde, and 5′-adenosine monophosphate. These non-canonical DNA terminal structures arise from various sources, such as radical-induced oxidation of the 2-deoxyribose moiety and DNA repair pathways. While this review does not cover the full spectrum of non-canonical termini, the selected structures are emphasized based on quantitative data supporting their biological relevance. The review also discusses their broader implications in mitochondrial DNA maintenance and inflammatory signaling and highlights key knowledge gaps that warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3444783d397400d00489bdd7139868206b7457" target='_blank'>
              Dirty Ends: Formation, Repair, and Biological Relevance of Non-Canonical DNA Terminal Structures
              </a>
            </td>
          <td>
            Seanmory Sothy, Linlin Zhao
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 RAD51 is a highly conserved gene whose protein product, RAD51, is instrumental in the maintenance of genome integrity in eukaryotes. RAD51 is well known to function in homologous recombination repair ('HRR') in conjunction with the BRCA1 and BRCA2 gene products. It also has other less well-known functions including protection from replication stress ('RS'). RS occurs downstream of incessant oncogene signaling and is therefore typical of cancers in general, and leukemias in particular, but not normal hematopoietic cells. RS is defined by stalled replication forks, which, if unprotected, are susceptible to endonuclease-mediated digestion leading to potentially lethal double strand DNA breaks. RAD51 gathers around stalled forks and protects them by a variety of processes including fork reversal to allow survival and proliferation of cancer cells. RAD51 is, therefore, a potentially important target in leukemias: its inhibition has potential to selectively kill malignant cells.
 We have shown that lead compounds of the IBR family of RAD51 inhibitors (IBR2, IBR120 and JKYN-1) have low micromolar single agent activity in a variety of cancer cell lines including representatives of both acute ('AML') and chronic ('CML') myeloid leukemia. Furthermore, these compounds often exhibit strong synergy and at least additivity in combination with both on-label and off-label use of FDA-approved compounds, as well as investigational molecularly-targeted agents, in relevant cell lines; these include FLT3 inhibitors quizartinib, gilteritinib and tuspetinib ((in MV-4-11 and HL60 cell lines) and BCR/ABL inhibitors (imatinib and regorafenib) in the K562 cell line (AACR 2025). Interestingly, strong synergy was demonstrated with JKYN1 and quizartinib, and separately, tuspetinib, in the FLT3 WT line HL60, but only additivity between JKYN-1 and gilteritinib in that cell line; all three of these compounds were extremely potent in the FLT3 mutated MV-4-11 cell line; additivity with JKYN-1 was nonetheless demonstrable, but not synergy. IBR2 exhibited low micromolar single agent activity in the K562 CML line and synergized with both imatinib and (separately) the off-label agent regorafenib. Regorafenib was also strongly synergistic with IBR2 in multiple other solid tumor cell lines, both within and beyond its label in colorectal cancer.
 We have confirmed the IBRs inhibit multiple RAD51 functions including hydroxyurea-induced RAD51 focus formation, HRR by DR-GFP assay, and RS protection by RPA focus formation and a DNA fibre assay. They markedly reduce intracellular RAD51 levels. All the IBRs tested to date bind RAD51 strongly by surface plasmon resonance (‘SPR‘), especially JKYN-1, and JKYN-1 is more potent in this and other assays than the standard classical RAD51 inhibitor B02. Our hypothesis is that both the single agent activity and synergy is due primarily to deprotection of RS.
 JKYN-1 is a partially optimized (more soluble and more potent) version of IBR120 but is not resistant to degradation by liver microsomes or sufficiently capable of entry into target cells. We have therefore selected 15-20 IBR analogues amongst a pool of ~35,000 created by generative chemistry (InSilico Medicine, Boston) for synthesis and in vitro testing. Anti-leukemic properties of these and our current IBRs will be tested on primary patient derived cells from genetically diverse AML patients. The assay will target stemness properties of AML patients in a high throughput screen designed to provide clinically actionable data for relapsed AML. Treatment dose and combinations will be pre-screened using a highly predictive cell line as a surrogate of human cancer stem cells for subsequently AML patient panels of leukemic stem cells to be examined. These studies will focus on single agent anti-proliferative activity and synergy with a suite of standard and investigational antileukemic drugs including cytosine arabinoside, daunorubicin, quizartinib, gilteritinib, tuspetinib, venetoclax, and azacitadine.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79dc795d42c064ddfb139b88cd12f7483902f754" target='_blank'>
              Inhibitors of RAD51 as potential novel therapies in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Mick Bhatia, P. Ferguson, A. Xenocostas, James Koropatnick, Morgan Black, Mark D. Vincent
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains a clinical challenge with suboptimal long-term survival. While circular RNAs derived from the RNF220 host gene have been implicated in AML pathogenesis, the functional role and regulatory mechanisms of RNF220 itself in AML are poorly understood. Methods We integrated bioinformatics analyses of public databases (TCGA-LAML, TARGET-LAML) and local cohort with in vitro functional assays. RNF220 was knocked down and overexpressed in AML cell lines using lentivirus. Transcriptomic profiling (RNA-seq), metabolic pathway enrichment (GSVA, GSEA), and immune microenvironment deconvolution (xCELL, CIBERSORT, MCP-counter) were performed. Transcription factor binding sites were predicted across five databases (JASPAR, ENCODE, GTRD, etc.). Validation of transcriptional regulation was performed using ChIP-PCR and luciferase reporter assays. Results RNF220 overexpression correlated with poor prognosis in AML, drove an immunosuppressive microenvironment characterized by reduced CD8+ T cells, and inhibited NK activity and M2 polarization of macrophage. RNF220 promoted tumor proliferation by suppressing apoptosis and preventing G1 arrest. Knockdown of RNF220 dysregulated metabolic pathways, notably suppressing glycolysis and phenylalanine metabolism. Mechanistically, FOXA1 was identified as an upstream negative regulator of RNF220, where high FOXA1 predicted favorable survival and inversely correlated with RNF220-associated metabolic reprogramming. Conclusion NF220 acts as an oncogenic ubiquitin ligase in AML by coordinating dual pro-leukemic mechanisms: cell-intrinsic metabolic rewiring (glycolysis/phenylalanine) and immune evasion via microenvironment suppression. Targeting the FOXA1–RNF220 axis may offer novel therapeutic strategies for high-risk AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ff73f3a35554c15311058c2d83eb2628a0504d" target='_blank'>
              Dual oncogenic role of RNF220 in AML: linking metabolic rewiring to cell proliferation and immune evasion
              </a>
            </td>
          <td>
            Bi-xia Li, Shi Jiang, Yao Xu, Xiao Yan, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Background: Epigenetic remodeling is a hallmark of aging hematopoietic stem and progenitor cells (HSPCs). Loss of DNA methylation and heterochromatin dysfunction are two key epigenetic signatures in aged HSPCs, contributing to functional decline and age-associated clonal hematopoiesis (CH). CH promotes the expansion of mutant clones, increasing the risk for hematologic malignancies and other age-related diseases such as cardiovascular disease. Our previous work demonstrated that age-related DNA methylation loss in aged HSPCs significantly disrupts the nuclear organization of H3K9me3-marked heterochromatin and contributes to functional decline, yet the causal mechanisms remain unclear.
 Methods: To investigate these mechanisms, we employed an in vitro replication-dependent senescence model. This approach is based on previous findings showing that during cellular replication, human primary fibroblasts form senescence-associated heterochromatin foci (SAHF), characterized by the accumulation of H3K9me3-marked heterochromatin into distinct, bright punctate foci surrounded by H3K27me3 within the nucleus, mimicking the chromatin architecture observed in aged HSPCs. To investigate the causal relationship between DNA methylation and heterochromatin dysfunction in senescence and aging, we perturbed DNA methylation by inhibiting DNMT1 in early-passage cells or overexpressing DNMT1 in late-passage cells. SAHF formation and senescence-associated phenotypes were assessed. DNA methylation was profiled via whole-genome bisulfite sequencing (WGBS), and the genomic localization of H3K9me3 and H3K27me3 was evaluated using CUT&Tag.
 Results: DNMT1 expression significantly declined during replication, confirmed by qPCR and western blot analyses comparing early and late passages. Correspondingly, global DNA methylation levels were reduced in replication-induced senescent cells, as measured by dot-blot analysis. To further investigate whether DNMT1 loss accelerates senescence, early-passage cells were treated with a DNMT1 inhibitor. We observed a dramatic increase in senescence-associated β-galactosidase (SA-β-gal) activity, suggesting that loss of DNMT1 accelerates senescence. Conversely, DNMT1 overexpression in late-passage cells (via lentiviral transduction) restored replication capacity and significantly reduced SA-β-gal activity compared to controls, underscoring a critical role for DNMT1 in regulating cellular senescence. Immunofluorescence staining revealed a strong positive correlation between DNMT1 loss and SAHF formation, with increased internal H3K9me3 and surrounding H3K27me3 foci. Notably, young Dnmt1-deficient HSPCs displayed similar chromatin reorganization and H3K9me3 relocalization, alongside impaired colony-forming capacity and disrupted perinuclear heterochromatin, mirroring features of aged HSPCs. Restoration of DNMT1 in late-passage cells markedly reduced SAHF formation. Epigenomic profiling via WGBS and CUT&Tag demonstrated that regions with reduced DNA methylation exhibited extensive redistribution of H3K9me3 and H3K27me3, particularly at genes associated with aging, inflammation, and stem cell function.
 Conclusion: Together, these findings demonstrate the essential role of DNMT1-mediated DNA methylation in maintaining heterochromatin integrity. Age-related DNA methylation loss profoundly impacts heterochromatin organization and genome architecture, driving cellular functional decline. Critically, loss of DNA methylation is sufficient to induce SAHF formation in somatic cells and chromatin remodeling in HSPCs, linking epigenetic erosion to stem cell dysfunction. This work provides a mechanistic foundation connecting epigenetic erosion and cellular aging and highlights the potential for therapeutic targeting of epigenetic regulators to reverse senescence and age-related hematopoietic decline.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e499ccb6510b3824230532d7d381186982b03555" target='_blank'>
              DNMT1-mediated epigenetic regulation of heterochromatin in cellular aging
              </a>
            </td>
          <td>
            Maryn Cavalier, Tingting Hong, Lei Guo, Y. Huang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a major cause of cancer-related mortality globally. Previous studies have reported that oral cancer overexpression 1 (ORAOV1) is overexpressed in HCC and correlated with poor prognosis, yet its molecular mechanisms remain incompletely understood. In this study, ORAOV1 overexpression was confirmed in HCC tissues via tissue microarray analysis and functionally linked to tumor cell proliferation through a positive correlation with Ki-67 expression in the human HCC cell line MHCC-97L. Bioinformatics analyses using The Cancer Genome Atlas (TCGA) and three Gene Expression Omnibus (GEO) HCC datasets further supported these findings. Multiple mechanisms appear to drive ORAOV1 upregulation, including promoter hypomethylation, amplification of the 11q13 region, and a putative ceRNA network involving AC005332.1, AC012615.1, and hsa-miR-100-5p. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses implicated ORAOV1 in various cellular processes, such as abnormal membrane channel function, extracellular matrix–receptor interactions, IL-17 signaling, and peroxisome proliferator-activated receptor (PPAR) signaling. Co-expression analysis identified significant associations between ORAOV1 and the oncogenes TPCN2 and CCND1. Additionally, ORAOV1 expression correlated with enhanced infiltration of immunosuppressive cells, including regulatory T cells, myeloid-derived suppressor cells, and cancer-associated fibroblasts, as well as upregulation of immune checkpoint markers (PD-1, PD-L1, and CTLA-4). These results indicate that ORAOV1 may modulate the immunosuppressive tumor microenvironment and contribute to resistance against immunotherapy, highlighting its potential as a therapeutic target in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b50660a0d2a36a94ce4bc7721bd3f07db2787d9" target='_blank'>
              Overexpression of ORAOV1 and its association with immunotherapy resistance in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yuzhen Huang, Ni Yang, Su Wen, Ziwei Fang, Yucong Zhang, Zonghao Qian, Yi Huang, Tiejun Yin, Cuntai Zhang, Le Zhang
          </td>
          <td>2025-11-25</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genetic variation at 19q13.3 critically modulates chemical carcinogen-induced lung carcinogenesis, particularly in mediating the activity of benzo[a]pyrene (B[a]P), a major polycyclic aromatic hydrocarbon (PAH) carcinogen. The adjacent genes ERCC1 and iASPP within this locus respectively coordinate nucleotide excision repair of PAH-induced DNA damage and suppression of apoptotic pathways. Their synergistic interaction regulates pivotal molecular events during PAH-driven lung carcinogenesis, ultimately impacting cellular repair, proliferation, and apoptosis. Chimeric RNAs have been increasingly recognized as promising biomarkers and therapeutic targets in cancer. However, the characterization of lung cancer-specific chimeric RNAs in the context of chemical carcinogenesis remains limited. This study identifies and characterizes ERCC1-iASPP, a novel chimeric RNA derived from the neighboring genes ERCC1 and iASPP, which exerts tumor-promoting functions via coding and non-coding mechanisms. First, the chimeric transcript encodes a previously uncharacterized protein, Ei, which enhances USP45-mediated deubiquitination of ERCC1, thereby stabilizing ERCC1 protein. Additionally, ERCC1-iASPP also functions as a (long non-coding chimeric RNA, lnccRNA): in the cytoplasm, it acts as a (competing endogenous RNA, ceRNA) by sequestering miR-143-3p, leading to derepression of CDK1 and PGK1 and subsequent activation of oncogenic pathways, while in the nucleus, ERCC1-iASPP further promotes transcriptional activation by recruiting STAT4 to the PGK1 promoter. Collectively, these findings establish ERCC1-iASPP as a bifunctional RNA with both protein-coding and non-coding regulatory roles that cooperatively promote B[a]P-induced lung tumorigenesis. This study highlights ERCC1-iASPP as a potential diagnostic and therapeutic target in smoking-related lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f388b70f5c6d5aca22c21b9e5d642b75c55036fe" target='_blank'>
              A Unique Chimeric RNA: ERCC1-iASPP Drives Benzo[a]pyrene-Induced Lung Carcinogenesis via Dual Coding and Non-Coding Mechanisms.
              </a>
            </td>
          <td>
            Mingming Shan, Mingyang Xiao, Liang Zhang, Shen Wang, Jiahang Wu, Kang Cao, Yujin Li, Guopei Zhang, Junwei Xie, Xiaobo Lu
          </td>
          <td>2025-11-26</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Embryonal tumors with multilayered rosettes (ETMR) are rare pediatric brain cancers with a median survival of 14 months despite aggressive treatment. The oncogenic driver in ~90% of cases is an amplification and fusion of the chromosome 19 micro-RNA cluster (C19MC), which is yet undruggable. Due to the tumor’s rarity, comprehensive molecular characterization and the identification of therapeutic targets are challenging, and ETMR remains poorly understood.
 We used single cell RNA sequencing and multiplexed protein profiling to study intratumoral heterogeneity and oncogenic signaling in ETMR (n=11). MiRNA-mRNA interactions were investigated by enhanced crosslinking and immunoprecipitation (eCLIP) followed by next-generation sequencing. ETMR cells were treated with locked nucleic acids (LNAs) and small-molecule inhibitors, and their anti-tumor effects assessed through confluency and death assays. Here, we identified three distinct malignant ETMR cell types resembling differentiation stages during early fetal neocortex development: undifferentiated ETMR cells resembling neural-stem cells (NSC), differentiated cells corresponding to mature neurons, and intermediate cells exerting markers of neuronal lineage commitment. C19MC was overexpressed in mitotically active NSC-like cells. Using eCLIP, we uncovered widespread regulatory roles of individual C19MC members. Markers of neuronal differentiation (NR2F2, SOX4, SOX11) and lineage commitment (C2–H2 ZNF family members) were targets of abundant C19MC members. LNA-mediated blockade of key C19MC members significantly reduced ETMR cell confluency and induced apoptosis. Genome-wide cell-cell communication analysis identified fibroblast growth factor receptors (FGFRs) and NOTCH receptors as key mediators of cellular cooperation. Accordingly, FGFR and NOTCH inhibitors had significant anti-tumor effects in vitro, and erdafitinib-supplemented therapy led to partial response in one therapeutically exhausted ETMR patient. Together, we demonstrated intratumoral heterogeneity in ETMR paralleling fetal neurodevelopment. Targeting C19MC in undifferentiated NSC-like cells significantly impaired cell confluency and survival. Additionally, FGFR emerged as a clinically actionable target, showing promising anti-tumor activity in vitro and in one ETMR patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb99e7f0c8810ec820bdec1d791a67a355e38fb" target='_blank'>
              EXTH-92. Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor-ligand interactions
              </a>
            </td>
          <td>
            L. Gabler-Pamer, Alexander Beck, G. A. V. Cruzeiro, S. Lambo, McKenzie L. Shaw, Olivia A Hack, Andrezza Nascimento, S. Madlener, Daniela Lötsch, I. Slavc, C. Dorfer, C. Haberler, Lissa C. Baird, Susan N. Chi, S. Alexandrescu, J. Gojo, Kool Marcel, V. Hovestadt, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Nuclear Factor I (NFI) proteins are involved in adenovirus DNA replication and regulate gene transcription, stem cell proliferation, and differentiation. They play key roles in development, cancer, and congenital disorders. Within the NFI family, NFI-X is critical for neural stem cell biology, hematopoiesis, muscle development, muscular dystrophies, and oncogenesis. Here, we present the structural characterization of the NFI transcription factor NFI-X, both alone and bound to its consensus palindromic DNA site. Our analyses reveal a MH1-like fold within NFI-X DNA-binding domain (DBD) and identify crucial structural determinants for activity, such as a Zn²⁺ binding site, dimeric assembly, and DNA-binding specificity. Given the ~85% sequence identity within the NFI DBDs, our structural data are prototypic for the entire family, a NFI Rosetta Stone that allows decoding a wealth of biochemical and functional data and provides a precise target for drug design in a wider disease context. The Nuclear Factor I (NFI) family includes transcription factors key to development, cancer, and genetic disorders. Here, the authors present the structure and DNA-binding properties of NFI-X, a prototype for the entire NFI family.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e09b9f75cb385c6026fbb070d6c58ca1ca1038c7" target='_blank'>
              Structural basis of Nuclear Factor 1-X DNA recognition provides prototypic insight into the NFI family
              </a>
            </td>
          <td>
            Michele Tiberi, M. Lapi, L. Gourlay, A. Chaves-Sanjuan, Maurizio Polentarutti, N. Demitri, Miriam Cavinato, Diane Marie Valérie Jeanne Bonnet, Valentina Taglietti, Anna Righetti, Rachele Sala, Silvia Cauteruccio, Amit Kumawat, R. Russo, A. Barbiroli, N. Gnesutta, C. Camilloni, M. Bolognesi, Graziella Messina, Marco Nardini
          </td>
          <td>2025-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Introduction: Diffuse large B-cell lymphoma (DLBCL) is stratified into genetic subtypes (MCD, BN2, A53, N1, EZB, ST2) that differ in their gene expression profiles, oncogenic mechanisms, and response to therapy. Using paired single cell RNA (scRNA) and ATAC (scATAC) sequencing in DLBCL tumors, we previously identified gene expression themes reflecting B cell differentiation, cell growth, and cell cycle that distinguished intratumoral genetic subclones (Wang B, ASH, 2024). Here, we present a global analysis of transcription factor (TF) binding and activity in normal B cells and DLBCL tumors that revealed epigenetic heterogeneity among the DLBCL genetic subtypes, which underpins their divergent therapeutic responses.
 Methods: Paired scRNA and scATAC sequencing was performed on 102 DLBCL cases (504,444 cells) and 3 tonsils (12,227 cells). Gene expression and genetic subtypes were determined from matched bulk samples by RNA and whole exome sequencing. Computational analysis was performed using R/python and custom bioinformatic pipelines.
 Results: By linking TF activators (+/+) and repressors (-/+) to target gene expression using SCENIC+ (Bravo Gonzalez-Blas C, Nat Methods, 2023), we identified gene regulatory networks (GRNs) composed of enhancer-driven Regulons (eRegulons). In tonsillar B cell subpopulations, we identified 173 eRegulons that linked TF binding to 9,456 genomic regions and 3,535 target genes. A subset of these eRegulons were differentially active (p<0.05) in germinal center (GC) B cells (FOXO1, MEF2B, EBF1, PAX5, TCF3), plasma cells (PC; IRF4, XBP1, PRDM1) and memory B cells (KLF2, STAT1, ETV6). Importantly, cell lineage analysis traced the activity of these eRegulons along 3 differentiation trajectories stemming from naïve B cells towards either GC dark zone, PC, or memory B cells.
 Next, we used TF binding to define the epigenetic landscape of the DLBCL genetic subtypes, which could be distinguished from each other using subtype-specific gene expression signatures. Chromatin binding by TFs that regulate PC differentiation (IRF4, POU2F2, TCF4) correlated with the MCD, BN2 and A53 gene expression signatures as well as with gene expression themes reflecting PC differentiation, cell cycle, and cell growth. Binding by another group of TFs (FOXO1, MYBL1, STAT6, PAX5) was associated with the EZB signature and the GC B cell gene expression theme. Binding by BCL6 was anticorrelated with signatures of the N1 subtype and memory differentiation, suggesting that BCL6 antagonizes memory B cell differentiation and the generation of N1 DLBCL.
 To define GRNs in DLBCL genetic subtypes and genetic subclones, we used SCENIC+ to infer 289 eRegulons, comprised of 12,016 TF binding regions and 4,723 target genes. By integrating the eRegulon RNA and ATAC scores using multiomics factor analysis (Argelaguet R, Genome Biol, 2020), we identified major axes of variation that discriminated both DLBCL subtypes and normal B cell populations. The EZB and ST2 subtypes were significantly associated (p<0.05) with eRegulons that typify normal GC B cells (MEF2B, MEF2C, IRF8, FOXO1). Within these subtypes, subclones with REL amplification had significantly greater activity of a REL +/+ eRegulon than those with wild type REL (p<0.001). The MCD, A53 and BN2 subtypes were enriched (p<0.05) for the IRF4 +/+ eRegulon while MCD was additionally associated (p<0.05) with BATF, SPIB, XBP1 and PRDM1 eRegulons. A TBL1XR1 –/+ eRegulon was significantly associated with the N1 subtype (p<0.001), which is notable given that TBL1XR1 is a tumor suppressor that is frequently inactivated in N1. The subtype-associated eRegulons were also differentially active in normal B cell populations, with several MCD eRegulons active in PCs, N1 eRegulons active in memory B cells, and EZB eRegulons active in GC B cells. Accordingly, eRegulon scores correlated with the B cell differentiation themes across DLBCL subclones.
 Conclusions: By paired scRNA and scATAC sequencing, we identified GRNs present in normal and malignant B cells that highlight transcriptional states of DLBCL genetic subtypes which vary along three principal differentiation axes – GC B cell, memory B cell and PC. Our analysis illuminates the biological heterogeneity of DLBCL molecular subtypes and offers rationale targets for future therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f09806b355405a44b654842f8d2b5c5a853ff8" target='_blank'>
              Gene-regulatory axes of diffuse large B-cell lymphoma subtypes
              </a>
            </td>
          <td>
            Julius Enssle, Boya Wang, George W. Wright, Xin Yu, James D. Phelan, Yandan Yang, Arthur Shaffer, Da Wei Huang, Zana Coulibaly, Michael Kelly, Bao Tran, Stefania Pittaluga, M. Roschewski, Wyndham H Wilson, Thomas Oellerich, G. Inghirami, Louis M Staudt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Seminoma is the most common solid malignant tumor of the testis in young males, significantly impacting fertility. Approximately 20% of seminomas metastasize, markedly increasing recurrence risk and compromising quality of life. To investigate the poorly understood mechanisms driving seminoma metastasis, we performed a comprehensive analysis by integrating single-cell RNA sequencing, TCGA data mining, and cell biology assays. We identified significant intratumoral heterogeneity. A subset of tumor cells expressing DPPA4 and PSMA7 showed high stemness, enhanced self-renewal, and association with metastasis. Knockdown of these genes reduced sphere formation, tumor migration and proliferation in Tcam-2 and NCCIT cells. Conversely, tumor cells overexpressing PAGE5 and SAT1 exhibited reduced stemness, migratory capacity and proliferation. In the immune microenvironment, we identified IFNG+ T cells, which recruit and activate other antitumor immune cells. These cells secrete IFN-γ, which promotes tumor differentiation, reduces stemness, and mitigates tumor aggressiveness. Based on these findings, we further developed a molecular panel to aid in the identification of seminomas with a higher risk of metastasis. In conclusion, our study identifies unique molecular signatures that facilitate risk stratification based on metastatic potential, providing valuable insights for improving precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a200ee992d63f7f5d78cc885c7a3fb857907df52" target='_blank'>
              Single-cell analysis unravels divergent gene signatures shaping seminoma stemness and metastasis
              </a>
            </td>
          <td>
            Zhouliang Bian, Biying Chen, Jiahui Guo, Jiamin Zhang, Guangye Du, Shufang He, Haihua Yuan, Yue Zhou, Bin Jiang, Daliu Min, Dingwei Ye, Hengchuan Su, Yanjie Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death Discovery</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background: Long non-coding RNAs (lncRNAs) are regulatory molecules that have multifaceted impacts on the carcinogenic molecular landscape—with pathologic consequences when aberrantly expressed. Anaplastic thyroid cancer (ATC) is a rapidly progressing and highly lethal malignancy, with mortality rates approaching 100%. The molecular/transcriptomic signature of ATC has significant gaps in understanding; thus, a comprehensive study of ATC non-coding RNA transcript regulation is necessary. Results: The lncRNA Double Homeobox A Pseudogene 10 (DUXAP10) was identified in patient genomic datasets as a highly upregulated transcript in ATC vs. normal thyroid tissue. DUXAP10 expression was transcriptionally repressed with CRISPR-interference (CRISPRi), and data supports an extensive role of DUXAP10 in several cancer-promoting phenotypes in ATC, both in vitro and in vivo. Our two DUXAP10-CRISPRi cell lines significantly reduced the rapid growth and metastatic behaviors characteristic of ATC, affecting proliferation, viability, clonogenicity, apoptosis, invasion, migration, tumorigenesis, and metastasis. Conclusion: Thus, DUXAP10 is a proposed prognostic marker and therapeutic target for ATC disease propagation and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8742d85dd91cdd69bf0576c2eda466736d98698d" target='_blank'>
              Long Non-Coding RNA DUXAP10 Promotes Tumorigenesis and Metastasis in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            N. Desouza, Michelle Carnazza, T. Jarboe, D. Quaranto, K. Kopec, Anthony J. Centone, Kate Nielsen, R. Suriano, A. Moscatello, Humayun K. Islam, , J. Geliebter, Raj K. Tiwari
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1d5f0efc9c96fae34cd837f1b2d0514c22b1a7" target='_blank'>
              Genome-wide mapping of sex-associated autosomal DNA methylation in major blood cell types
              </a>
            </td>
          <td>
            Yutaro Yanagida, Y. Nakachi, M. Bundo, Syohei Komaki, A. Shimizu, K. Iwamoto
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Telomerase Reverse Transcriptase promoter (TERTp) mutations enable tumor cell immortality in millions of cancer patients annually. TERTp mutations are the most common non-coding mutations across all cancers, including glioblastoma, oligodendroglioma, medulloblastoma, and high-grade meningioma. A multitude of TERTp mutations, including the two hotspots, create de novo E26 transformation specific transcription factor binding sites. Among the 28 family members, we have shown that only the GA-binding protein (GABP, composed of DNA binding GABPA and transactivating GABPB subunits) activates the mutant TERTp. Prior therapies targeting the telomerase RNA component, TERC, lacked tumor selectivity and were poorly tolerated. The GABP-mediated reactivation of the mutated TERTp presents a unique therapeutic opportunity for tumor specific reversal of cellular immortality. Breakthroughs in the design of biological proteolysis-targeting chimera (bioPROTACs) offer a new approach to selectively degrade previously intractable targets such as transcription factors. We combined in silico protein-protein interaction modeling via AlphaFold and experimental validation to identify a minimal GABPB to bind GABPA and fused it with an E3 ubiquitin ligase. Introduction of this GABPA bioPROTAC into TERTp mutant glioblastoma cells depleted GABPA protein and eliminated GABPA binding to the mutant TERT promoter, reducing transcriptional activating (H3K4me3) and increasing suppressive (H3K27me3) histone marks. Reversion of mutant TERTp to an epigenetically silenced state reduced TERT expression by 73% to 95% and shortened tumor cell telomeres in a promoter mutation-specific manner. The GABP bioPROTAC reduced tumor growth and improved survival of mice bearing an orthotopic xenograft of TERTp mutated glioblastoma cells. The effects of the bioPROTAC were validated in vivo via magnetic resonance imaging of metabolic correlates of TERT expression. These data demonstrate that this GABP bioPROTAC potently degrades GABPA, leading to tumor-specific silencing of telomerase expression, and a reversal of glioblastoma tumor cell immortality. This artificial intelligence-guided approach may be applicable to other intractable cancer specific targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102d24aea88be03925f3847c09d31dfd2d63944f" target='_blank'>
              EXTH-77. An AI-engineered bioPROTAC reverses immortality of
 TERT
 promoter mutant tumors
              </a>
            </td>
          <td>
            Nicholas O. Stevers, Ellen Chen, Joshua S. Wu, T. Patel, Sara Collins, M. Montoya, Samuel H. Wu, G. Batsios, A. Gillespie, Chibo Hong, Noriyuki Kasahara, Pavithra Viswanath, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Early T-cell precursor (ETP) acute lymphoblastic leukemia/lymphoma (ALL) is an aggressive hematological malignancy characterized by the uncontrolled expansion of T-cell progenitors. Loss-of-function mutations in genes encoding epigenetic regulators, including core components of the Polycomb Repressive Complex 2 (PRC2), such as the EZH2 histone methyltransferase, are frequently observed in ETP-ALL patients.To investigate the role of EZH2 inactivation in human early T-cell leukemogenesis, we optimized a CRISPR/Cas9-based strategy to disrupt EZH2 gene expression by integrating a DNA cassette containing the constitutive human PGK-1 promoter followed by the GFP reporter. Specifically, human CD34⁺ hematopoietic stem/progenitor cells (HSPCs) derived from cord blood were first edited to knock out (KO) EZH2, then transduced with lentiviruses encoding different T-ALL oncogenes or an empty vector control, and maintained in serum-free medium supplemented to support T-cell progenitor expansion. We found that only HOXA9- and LYL1-transduced EZH2KO HSPCs expanded in vitro for up to 30 days, and were enriched for early T-cell subsets, as identified by single-cell RNA sequencing (scRNA-seq) and a 26-color flow cytometry assay. Notably, these same subsets were capable of initiating T-cell leukemia in immunocompromised (NSG) mice following transplantation, suggesting that EZH2 inactivation cooperates with HOXA9 or LYL1 overexpression to drive human T-cell transformation. To assess the relevance of these findings in primary human T-ALLs, we analyzed whether loss-of-function mutations in PRC2 genes (EZH2, EED, SUZ12) were enriched in specific T-ALL subgroups, particularly ETP-ALL, in a cohort of 1,335 patients from the Children's Oncology Group (COG) study. Interestingly, PRC2-mutated T-ALLs were significantly associated with the HOXA subgroup and the ETP-ALL phenotype, and exhibited higher levels of minimal residual disease (MRD) 30 days after therapy compared to other T-ALL cases. To further dissect the molecular effects of EZH2 loss and HOXA9 overexpression in ETP-like T-ALL, we knocked out EZH2 in the human Loucy and PEER cell lines using CRISPR/Cas9. These EZH2KOcells were then transduced with HOXA9-encoding lentiviruses or empty vectors and analyzed by RNA sequencing. In parallel, ChIP-seq analyses were performed to map HOXA9 binding sites and assess histone modifications affected by EZH2 loss. Integrated analysis of RNA-seq and ChIP-seq data revealed that HOXA9 expression in EZH2-deficient ETP-like cells led to the upregulation of a stem-like/immature T-cell gene signature enriched in transcriptional activators and chromatin remodelers (e.g., RUNX2, IRX3, SATB1, ARID5B, CHD1, INO80D).Furthermore, scRNA-seq profiling of primary PRC2-mutated T-ALL samples revealed distinct leukemic subsets with high HOXA9 expression and strong enrichment for the same gene signature, as determined by Gene Set Variation Analysis (GSVA). Importantly, this signature stratified PRC2-mutated T-ALLs and identified a HOXA positive subgroup associated with poor prognosis.
 Collectively, these findings uncover a critical epigenetic circuit involving EZH2 loss and HOXA9-driven transcriptional reprogramming that promotes stem-like, therapy-resistant T-ALL subsets and highlights novel vulnerabilities that could be therapeutically exploited to eliminate high-risk disease at its origin.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d423907fad1e15df50c67eaeeaa6ff45c0caebf" target='_blank'>
              EZH2 inactivation cooperates with HOXA9 to induce leukemic transformation of human t-cell progenitors and sustain high-risk subsets of early t-cell precursor acute lymphoblastic leukemia (ETP-ALL)
              </a>
            </td>
          <td>
            M. Colucci, Costanzo Padovano, Gaja Bruno, Elisabetta De Santis, Serena di Iasio, Chiara Di Nunzio, Andrew Weng, Vincenzo Giambra
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca6176cd27da7f6583937f65a3ed03caa145ef0" target='_blank'>
              Overexpressed PRR11-SKA2-miR301a/454 bidirectional transcription unit essentially and coordinately promotes PI3K-AKT pathway activation and lung cancer progression.
              </a>
            </td>
          <td>
            Tao Liu, Xiaofeng Zuo, Shijie Sun, Kailong Du, Chuntao Tao, X. Xia, Linli Yu, Chunxue Zhang, Zhengmei Yang, Yitao Wang, Junhong Ye, Youquan Bu
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH), the clonal expansion of mutant hematopoietic stem and progenitor cells (HSPCs), is highly prevalent in aging populations and is linked to age-related diseases such as cardiovascular disease and malignancy (Jaiswal et al., N. Engl. J. Med., 2014, 2017). DNA methylation regulators, Ten-Eleven Translocation 2 (TET2) and DNA methyltransferase 3A (DNMT3A), are among the most frequently mutated genes in CH (Genovese et al., N. Engl. J. Med. 2014). Recent studies report the essential role of TET2/DNMT3A in regulating DNA methylation in euchromatin particularly in the regulation of key genes involved in hematopoietic stem cell function and lineage specification. DNA methylation also maintains heterochromatin by silencing transposable elements (TEs). Our recent study reported a distinct role for DNMT3A and TET2 in regulating the spatial relocation of H3K9me3-marked heterochromatin during HSPC aging, which contributes to age-related CH (Hong et al., Nat. Aging, 2023). Our data suggested the upregulation of TEs correlated with the activation of interferon-stimulated genes (ISGs) and contributed to the functional decline of aged HSPCs. This decline was mitigated in TET2-mutant HSPCs, increasing their competitive capacity. However, given HSPC heterogeneity, the cell type specific TEs alterations during aging and in the pathogenesis of CH remain largely unknown.
 To further delineate the transcriptional and epigenetic regulation of TEs during aging, we purified Lin- HSPCs from C57BL/6 wild-type (WT) and TET2-knockout (TET2-KO) mice at 7 weeks (7w), 21 months (21m), and 30 months (30m) of age. We then performed single-cell (sc) RNA-seq and scATAC-seq. After preprocessing, cell type labeling, and integration of both scRNA and scATAC datasets, we quantified TE expression and accessibility at both the locus and family levels using MATES, an autoencoder-based methodology (Wang et al., Nat. Commun., 2024). We compared WT and TET2-KO aging by integrative scRNA/scATAC analysis of lineage biases and regulon activities.
 Coding-gene and TE expression profiles independently resolved the same HSPC progenitor subpopulations, indicating cell type specific TE expression. This strongly suggested the tissue specific regulation of TEs, consistent with previous publications regarding embryonic development (Miao, et al, Genome Biol 2020). Furthermore, scATAC-seq analysis revealed similar subpopulation specific TE trends, including distinct, smooth trajectories of TE accessibility along myeloid, lymphoid, and megakaryocyte-erythroid lineages. Integrating our 7w and 21m WT scRNA-seq and scATAC-seq, we found that age-dependent gains in TE accessibility and transcription varied across all major progenitor subpopulations. These age-dependent changes in TE accessibility and transcription occurred in all major classes of TEs including LINEs, SINEs and LTRs, but the changes were mitigated in TET2 deficient cells, strongly suggesting distinct regulatory mechanisms in TE during TET2 deficient HSPCs aging. The age-dependent changes in HSCs were strongly exemplified by many TE elements at the subfamily and locus levels in groups such as the MurERV40int and RLTR10 from the ERV class. Furthermore, we revealed a positive correlation between TE expression and inflammatory genes, suggesting the potential involvement of TEs in regulating intracellular innate immune response during aging. In addition to modulating the age-related changes in TEs, loss of TET2 mitigated age-associated chromatin opening and expression of age-related genes such as Stxbp4 and S100a6 (Svendsen et al., Blood, 2021). Additionally, a regulon-based analysis showed age-related decreases in regulons associated with HSC function and maintenance such as Meis1 (Kocabas, et al. Blood, 2012), this age-related decrease was mitigated in TET2-KO. Parallel analysis of aging human HSPC datasets uncovered similar gene and TE activation patterns, suggesting a conserved mechanism (Hua et al., Blood, 2019).
 Beyond extrinsic inflammation, age-related TE upregulation may trigger innate immune signaling that impairs HSPC self-renewal and specification. Our multi-omics approach elucidates the epigenetic and transcriptional mechanisms underlying TE dysregulation in aging HSPCs. Our study provides new insights into the molecular underpinnings of age-related hematopoietic dysfunction and identified potential therapeutic targets for mitigating CH and associated diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca589ad4135ba9b576c77a6dbfe352981eb6d90e" target='_blank'>
              Single-cell multi-omics uncovers transposable element regulation in hematopoietic stem and progenitor cells during aging
              </a>
            </td>
          <td>
            Logan Rivera, Tingting Hong, Y. Huang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38954ab0551e2a9866f6ea57105246fb32735740" target='_blank'>
              CellSNVReg: a multidimensional resource for SNV-mediated regulatory perturbations in single-cell and spatial omics.
              </a>
            </td>
          <td>
            Jingyi Shi, Shiqian Zhang, Zhenglin Lu, Weiwei Zhou, Weijian Lu, Gengdong Chen, Jianing Li, Wenhui Liu, Jiaqi Huang, Tiantongfei Jiang, Xiyun Jin, Juan Xu, Tingting Shao
          </td>
          <td>2025-10-31</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Children with Down syndrome are predisposed to hematological malignancies due to the presence of an extra copy of chromosome 21. Trisomy 21 (T21) alone has been shown to disrupt fetal hematopoiesis. Further examination identified that the Down syndrome critical region (DSCR) on chromosome 21 plays a key role in T21-driven leukemogenesis due to gene dosage imbalances. DSCR harbors several genes essential for hematopoietic development and leukemogenesis, such as DYRK1A, one of the dosage sensitive genes. As a dual-specificity kinase, DYRK1A plays diverse roles in cell cycle progression, stem cell differentiation, and DNA damage response. Interestingly, DYRK1A has been reported to interact with RNA polymerase II at active promoters, which shows a potential role in transcriptional regulation. DYRK1A overexpression leads to disrupted hematopoiesis, yet its distinct role in ML-DS remains unclear.
 In this study we showed DYRK1A as a novel interactor of EZH2 in CMK, a ML-DS cell line, and 3 distinct relapsed ML-DS PDX samples by co-immunoprecipitation. Given that both proteins are known to function in cytoplasmic and nuclear compartments, we performed co-immunoprecipitation in these subcellular fractions and found that the interaction only occurs in the nuclear fraction in CMK and ML-DS PDX cells. Considering the canonical role of EZH2 as a methyltransferase and of DYRK1A as a dual-specificity kinase, we examined if their canonical roles are necessary for this interaction. Therefore, we treated cells with EZH2 inhibitor (GSK126) or DYRK1A inhibitor (EHT1610) and performed co-immunoprecipitation. The data demonstrated that the interaction between DYRK1A and EZH2 was maintained even in the presence of inhibitors which suggest that the interaction between these two proteins is not dependent on either methyltransferase or kinase activity.
 Since DYRK1A has been implicated in transcriptional regulation and we showed that it interacts with EZH2 in the nucleus only, we hypothesized that DYRK1A may collaborate with EZH2 to shape the chromatin landscape in ML-DS. To explore this possibility, we performed CUT&RUN chromatin profiling of EZH2, DYRK1A, H3K27me3, and H3K4me3 in CMK cells. CUT&RUN using anti-DYRK1A antibody identified 12,348 peaks (p <0.001). These peaks partially overlapped with the H3K4me3 peaks, but they were not enriched by H3K27me3 peaks. Overlapping DYRK1A peaks with EZH2 peaks revealed 186 regions occupied by both, and these regions are highly enriched by H3K4me3, suggesting active gene expression. Pathway enrichment analysis of DYRK1A and EZH2 co-occupied regions indicated enrichment in key signal transduction pathways, including MAPK, ERK, RAS, and neurotrophin (NTRK) signaling (p < 0.05).
 To further investigate the function of EZH2 and DYRK1A in ML-DS, we depleted the expression of EZH2 or DYRK1A in CMK cells by CRISPR-cas9 mediated knockout. While depletion of EZH2 resulted in decreased protein level of DYRK1A in the nucleus, it had no effect on cytoplasmic DYRK1A. Interestingly, the downregulation of DYRK1A protein did not affect the level of EZH2 in both fractions. These data indicate that DYRK1A requires EZH2 for its nuclear localization and likely regulates expression of specific genes required for leukemogenesis.
 Together, these findings suggest that DYRK1A and EZH2 may functionally cooperate to regulate gene expression at transcriptionally active regions in ML-DS cells, especially within key signal transduction networks. This interaction may represent a previously unrecognized mechanism contributing to leukemogenesis in T21, with potential implications for targeting non-canonical EZH2 functions in ML-DS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcb96ecddff82ebd51607a31bc1f76845786142" target='_blank'>
              DYRK1A interacts with EZH2 to regulate transcriptionally active chromatin in myeloid leukemia associated with down syndrome
              </a>
            </td>
          <td>
            Kader Cicek, S. Barwe, Anilkumar Gopalakrishnapillai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Long non-coding RNAs (lncRNAs) represent an important layer of gene regulation and have emerged as significant contributors to multiple myeloma (MM) progression (Malard, F., Neri, P., Bahlis, N.J. et al.,2024). The lncRNA NEAT1 is over-expressed in MM patients, supporting malignant plasma cells proliferation and promoting therapy resistance (Taiana E.et al., 2020). Besides its conventional function in the assembly of paraspeckles (PSs), we recently demonstrated that NEAT1 is directly involved in transcriptional control (Puccio N. et al., 2024). In this study, we deepen our understanding of NEAT1's function in shaping the activity of transcriptional bodies, providing the rationale for targeted therapeutic intervention.
 Starting from a transcriptomic approach in NEAT1-silenced (KD) and CRISPRa NEAT1-overexpressing MM cells, we derived a list of 378 targets, primarily involved in mitosis. 52% of these present the cell cycle homology region motif (CHR) in their promoter region, which synchronizes their activation during late cell cycle phases. We confirmed the clinical relevance of NEAT1's transcriptional program in the coMMpass cohort of almost 900 MM patients. Unsupervised clustering analysis, based on the expression of 378 genes NEAT1's target genes, identified two different groups of patients characterized by high or low NEAT1's expression program and different clinical outcomes. Notably, the cluster with the high NEAT1-CHR program showed reduced overall survival. To explore the mechanistic contribution of NEAT1 to mitotic gene expression, we performed computational analysis through the binding model of analysis of transcription (BART) and query of the ENCODE database. These analyses revealed that FOXM1 and CDK9 as upstream regulators of NEAT1 program. Notably, FOXM1 resulted to be highly expressed in R2-ISS stages II-III of myeloma disease as compared to stage I, and in secondary plasma cell leukemia (sPCL) of myeloma origin, based on coMMpass and GSE66293 datasets analysis. We supported these observations by performing in vitro assays. We described a co-localization of NEAT1 and FOXM1 condensates in MM cells, through combined RNA-FISH/immunofluorescence, and direct binding of NEAT1-FOXM1/NEAT1-CDK9 by targeted RNA-immunoprecipitation (RIP). Additionally, with Chromatin immunoprecipitation experiments (ChIP), we demonstrated that NEAT1 KD results in a reduced occupancy of FOXM1 at the promoters of essential CHR genes. Finally, by performing high-throughput drug screening, we identified CDK9 inhibitors to have a strong synergistic effect with NEAT1 KD, consistent with a synthetic lethality effect. These results were confirmed by live cell imaging approach, with two selective CDK9 inhibitors (CDK9i) on a panel of MM cell lines. Consistently, NEAT1 overexpression confers an increased resistance to CDK9i, confirming the participation of NEAT1 to the transcriptional complex that synchronizes mitotic genes and executes the cell division program.
 These data are in line with our recent findings that demonstrate that NEAT1 controls and cooperates with Aurora kinase A (AURKA) in ensuring proficient chromosomal segregation(Puccio N. et al., 2024).
 In conclusion, our data demonstrated that NEAT1 contributes to overt phases of MM and to the clinical aggressive behaviour of the disease, by orchestrating a mitotic transcriptional program. Furthermore, through the application of a multi-dimensional approach, we revealed that the dual inhibition of NEAT1 and transcriptional regulators as CDK9 may offer a promising strategy for novel combinatorial anti-MM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0423164211e0008033b3494329d936fcc2b1eb0a" target='_blank'>
              Disruption of NEAT1-transcriptional network identifies targetable non-genetic dependencies in multiple myeloma
              </a>
            </td>
          <td>
            N. Puccio, Christian Boni, Federica Torricelli, Emanuele Vitale, E. Taiana, D. Ronchetti, Valentina Fragliasso, Selene Mallia, Giulia Gambarelli, Roberto Piva, N. Bolli, S. Luminari, A. Ciarrocchi, A. Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="


 Age-related clonal hematopoiesis (CH) and clonal cytopenia of undetermined significance (CCUS) represent pre-leukemic conditions associated with predisposition to hematological malignancies, particularly myeloid neoplasms. Frequently associated with mutations in epigenetic regulators, such as DNMT3A, TET2, and ASXL1, deregulated epigenomes are implicated in the aberrant gene expression involved in the pathogenesis of CH and CCUS, whose molecular mechanism, however, has not been investigated. The major difficulty lies in the technical limitation to simultaneously analyze gene mutation, transcriptome, and epigenetics in a small fraction of mutant cells in comparison with wild-type cells within the same patients, particularly in those cases where mutant cell fractions are very small.
 Aim/Methods In this study, we developed a novel platform that enabled simultaneous measurements of gene mutations and chromatin accessibility at the single-cell level based on the modified Mission Bio Tapestri system, which was applied to the analysis of 11 samples with TET2-mutated (TET2-MUT) CH and CCUS to investigate the impact of TET2 mutations on chromatin accessibility. In a subset of the samples (n = 3), we performed 10x Chromium single-cell ATAC+RNA-seq (Multiome) experiments and then predicted single-cell TET2 genotypes by employing ATAC peak accessibility-based logistic regression models trained on the paired Tapestri multimodal data, enabling the analysis of effects of TET2 mutation on gene regulation and transcriptional outputs at single-cell resolution.
 Results First, our platform was validated using a mixed cell line sample (HL-60 and SKM-1). ATAC data alone clearly separated the two cell lines, and genotyping accuracy for cell line-discriminative SNPs reached 94-99%, confirming robust integration of ATAC-seq and genotyping at the single-cell level.
 Next, to investigate the consequences of TET2 mutation on hematopoiesis and chromatin accessibility, we applied the platform to a total of 37,190 cells from TET2-MUT CH/CCUS patients, achieving successful genotyping in 83% (74–92%) of the cells. TET2-MUT cells exhibited a differentiation bias toward myeloid and plasmacytoid dendritic cell (pDC) lineages, with reduced mature lymphoid output. In line with this skewed differentiation, TET2-MUT hematopoietic stem/progenitor cells (HSPCs) showed an increased accessibility to the motifs of transcription factors (TFs) associated with myeloid/pDC differentiation (e.g., CEBPE, RUNX1, and IKZF1), compared to wild-type (TET2-WT) HSPCs, whereas TET2-MUT B cells had a reduced accessibility to the motifs of B cell development-associated TFs (e.g., BACH2, IRF4, and IRF8), compared with TET2-WT counterparts. In erythroblasts, TET2-MUT cells exhibited a decreased accessibility to erythroid maturation-related genes and an increased accessibility to genes involved in cell proliferation, including E2F and MYC targets, compared with TET2-WT cells, suggesting dysregulated maturation of TET2-MUT erythroblasts. Collectively, these results revealed TET2 mutation-driven alterations in chromatin accessibility affecting lineage maturation.
 Next, to investigate combined effects of TET2 mutation on TF-mediated gene regulation and transcriptional programs, we analyzed 10x Multiome data with predicted single-cell TET2 genotypes. We first analyzed TET2-MUT cell fractions within monocyte subsets and observed a depletion of mutants in the CD16(+) subset, suggesting impaired maturation of TET2-MUT cells. To uncover the underlying mechanisms, we then compared TF activity between genotypes, using SCENIC+, which integrates paired single-cell ATAC-seq and RNA-seq data to infer TF activity based on both accessibility at TF-binding enhancers and expression of their target genes. For both accessibility- and expression-based TF activities, TET2-MUT monocytes exhibited up-regulation of cell cycle-regulating TFs (e.g., TFDP1, TFDP2, and MYBL2) and down-regulation of key monocyte maturation-related TFs (e.g., CEBPB, CEBPD, and SPI1). These results suggest that TET2 mutation may impair monocyte development by epigenetically and transcriptionally suppressing maturation-related programs and sustaining proliferative states.
 ConclusionOur innovative single-cell multi-omics approaches revealed that TET2 mutations rewire chromatin accessibility and TF regulatory networks, leading to dysregulated lineage maturation and aberrant cell cycle control in CH/CCUS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8043e74c1da9433c5daf135387ded1955d981b6" target='_blank'>
              Single-cell multiomic profiling of gene mutation, chromatin accessibility, and gene expression in TET2-mutant clonal hematopoiesis
              </a>
            </td>
          <td>
            Masanori Motomura, Y. Ochi, Y. Nannya, N. Kakiuchi, N. Sezaki, S. Kasahara, Tomoe Nakagawa, Lanying Zhao, Shuichi Matsuda, M. Nakagawa, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [12, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>