<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-02 10:00:57 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>47</td>
          <td>107</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>94</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>126</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>411</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>347</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA molecules prevalent in human cancers that drive tumor evolution and drug resistance. Their circular topology, which disrupts topological domains and rewires regulatory circuits, has typically been studied via pairwise interactions. Here we develop ec3D, a computational method for reconstructing three-dimensional ecDNA structures from Hi-C data. Given a candidate ecDNA sequence and whole-genome Hi-C data, ec3D reconstructs spatial structures by maximizing the Poisson likelihood of observed interactions. We validate ec3D using simulated structures, previously characterized cancer cell lines, and microscopy imaging. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range regulatory interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex structures with duplicated segments, identify multi-way interactions, and identify potential intermolecular (trans) interactions. Our findings provide insights into how ecDNA’s spatial organization bypasses normal chromosomal constraints and contributes to increased oncogene expression. Extrachromosomal circular DNAs (ecDNAs) are prevalent in human cancers and are thought to drive tumor evolution and drug resistance by amplifying oncogenes. Here, authors develop ec3D to reconstruct three-dimensional ecDNA structures, revealing how their spatial organization rewires regulatory circuits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, Jessica Alsing, J. Luebeck, Maria E. Stefanova, Owen S. Chapman, Katerina Kraft, Shu Zhang, Jun Yi Stanley Lim, Yipeng Xie, Yoon Jung Kim, Sihan Wu, Lukas Chavez, Guy Nir, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>75</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Genome instability, including chromothripsis, is a hallmark of cancer. Cancer cells frequently contain micronuclei-small, nucleus-like structures formed by chromosome missegregation-that are susceptible to rupture, exposing chromatin to cytoplasmic nucleases. Through an unbiased, imaging-based small interfering RNA screen that targeted all 204 known and putative human nucleases, we identified a previously uncharacterized cytoplasmic endonuclease, NEDD4-binding protein 2 (N4BP2), that enters ruptured micronuclei and initiates DNA damage, leading to chromosome fragmentation. N4BP2 promoted genome rearrangements (including chromothripsis), formation of extrachromosomal DNA (ecDNA) in drug-induced gene amplification, tumorigenesis, and tumor cell proliferation in an induced model of human high-grade glioma. Analysis of more than 10,000 human cancer genomes revealed elevated N4BP2 expression to be predictive of chromothripsis and copy number amplifications, including ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e190a2f3ad6c333d8f483ff05cc8237431d7f3" target='_blank'>
              Chromothripsis and ecDNA initiated by N4BP2 nuclease fragmentation of cytoplasm-exposed chromosomes.
              </a>
            </td>
          <td>
            Ksenia Krupina, A. Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D Steele, Amy T Nguyen, Daniel L Zhang, Jonas Koeppel, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, F. Furnari, P. Campbell, Ludmil B. Alexandrov, Don W. Cleveland
          </td>
          <td>2025-12-11</td>
          <td>Science</td>
          <td>2</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling.
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Extrachromosomal DNA (ecDNA) amplifications are key drivers of human cancers. Here, we show that ecDNAs are major platforms for generating and amplifying oncogene fusion transcripts across diverse cancer types. By integrating analysis of whole genome and transcriptome sequences from 1,825 tumor samples and cancer cell lines of a wide variety of tissue types, we reveal that ecDNAs have the highest rate of oncogene fusion events of any copy number alteration. Focusing on the most common ecDNA fusion hotspot, we find that fusion of the 5' end of the long noncoding RNA gene, PVT1–with exon 1 joined to diverse 3' partners–confers increased RNA stability, and enhances MYC-dependent transcription and cancer cell survival. These results demonstrate that ecDNA fosters genome instability and frequent oncogene fusion formation under strong selection in cancer.



 Shu Zhang, Hyerim Yi, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, Ivy Tsz-Lo. Wong, Suhas Srinivasan, Ellis J. Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, Jens Luebeck, Chris Bailey, Yanding Zhao, Julia A. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, Mariam Jamal-Hanjani, Dean W. Felsher, Luke A. Gilbert, Vineet Bafna, Paul S. Mischel, Howard Y. Chang. EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68513e53c43f7bee512369c94fed2f8b73ac179" target='_blank'>
              Abstract A007: EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer
              </a>
            </td>
          <td>
            Shu Zhang, Hyerim Yi, J. Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Chris Bailey, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, M. Jamal-Hanjani, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is an aggressive malignancy associated with single-nucleotide variants in VHL, PBRM1, and SETD2. However, the role of structural variation (SV)—which have broader genomic impacts—and 3D genome architecture in ccRCC remains inadequately understood. Here, we reported a comprehensive molecular characterization. Through multi-omics analysis, we identify novel SV-associated oncogenic targets and reveal multidimensional 3D genome reorganization during ccRCC progression. We elucidate the dynamic interplay between SV and 3D chromatin architecture, demonstrating how structural rearrangements drive oncogenic dysregulation through 3D genome reorganization. Notably, an unrecognized pathogenic enhancer hijacking event is discovered and experimentally validated, leading to constitutive activation of the proto-oncogene SEMA5B. Furthermore, we developed a machine learning-based prognostic framework using enhancer hijacking signatures. Collectively, this work establishes a valuable resource for ccRCC research by elucidating how SVs and 3D genome reorganization collectively drive oncogenesis, and translates these findings into a clinically applicable prognostic tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, 
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb14a3c17daae13a9f8ee1cd3546f3c3832ec32" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>2</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce9b91a05b2c305ecf72c3bf59327984c56dba6" target='_blank'>
              Functional inactivation of the telomerase chaperone TCAB1 primes cells for the activation of ALT in osteosarcoma
              </a>
            </td>
          <td>
            Joshua Keegan, Sydney Sorbello, Joakin O. Mori, Shugo Muratani, I. Leshchiner, Christopher M. Heaphy, R. L. Flynn
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bac6b9fbac768b087852174f053e8622b039f" target='_blank'>
              Targeted editing of pericentromeric satellite DNA alters sensitivity to meiotic drive
              </a>
            </td>
          <td>
            D. Eickbush, J. Rahmat, M. Lindsay, J. Bowers, N. Fuda, A. Larracuente
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer driver mutations alone are often insufficient to fully explain tumorigenesis. We demonstrate that these mutations cooperate with somatic copy number variations (CNVs) in a tissue-specific pattern of genomic epistasis. Analyzing 93,462 tumors, we identified 54 gene-cancer type pairs with significant co-occurrence of somatic mutations and CNVs. Our new Binoculars algorithm, which resolved phased DNA/RNA reads, revealed frequent preferential amplification in oncogenic mutation alleles, including AKT1 p.E17K, BRAF p.V600E, KRAS p.G12C/D/V, NRAS p.Q61K, and a fraction of gain-of-function TP53 p.R175H. Conversely, deletions selectively targeted the reference alleles, leading to loss of heterozygosity of IDH1 p.R132H and tumor suppressor mutations, including CDKN2A and TP53 truncations. Lung cancer patients carrying co-occurrences of somatic mutation-CNVs in TP53 and KRAS showed poorer survival than those carrying the same gene mutations. These findings reveal epistasis of cancer mutations and CNVs at an allelic resolution, suggesting specific genomic events to enhance patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c67a7d5e6bdabbcb4e758c260b7827b1f0b38f4" target='_blank'>
              Widespread Epistasis between Cancer Driver Mutations and Allele-Specific Copy Number Variations
              </a>
            </td>
          <td>
            Serge Merzliakov, Guanlan Dong, Andrea Castro, Mahad Bihie, Yve Nichols-Evans, Hanna Carter, Kuan-lin Huang
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Long INterspersed Element-1 (LINE-1) retrotransposons comprise 17–20% of the human genome. These retroelements are normally silenced early in embryonic development through epigenetic mechanisms and reawakened during oncogenesis, leading to transcriptional dysregulation, genomic instability, and immune evasion. Methods: In the present study, we categorized LINE-1 transcripts across 121 non-small cell lung cancer (NSCLC) cell lines from the Cancer Cell Line Encyclopedia (CCLE) by subfamily, length, orientation, chromosomal origin, and distribution. In addition, high-prevalence insertions were mapped to nearby genes to assess potential functional interactions. Results: LINE-1 transcript abundance and length in NSCLC were dominated by evolutionarily young subfamilies, particularly L1HS and L1PA2 through L1PA5. Chromosomal patterns were conserved across NSCLC subtypes, with modest enrichment of L1HS activity on Chromosome 4 and the X Chromosome. The lung squamous cell carcinoma (LSQCC) subtype exhibited the highest total levels of L1HS expression relative to other NSCLC subtypes. Race modestly influenced LINE-1 transcript abundance, with cell lines derived from self-identified African American individuals showing elevated overall LINE-1 and L1HS expression. Age showed a weak positive correlation with total LINE-1 abundance. Integrative analysis revealed recurrent hotspots at 22q12.1 and 20p11.21 that were transcriptionally active across subtypes and coincided with previously reported intact LINE-1 elements active in epithelial cancers. Recurrent insertions were located near cancer-associated genes, including RB1, NEDD4, FTO, LAMA2, NOD1, and KCNB2, implicating LINE-1 activity in cis-regulatory remodeling of oncogenic pathways. Conclusions: Together, these findings indicate that LINE-1 transcript heterogeneity in NSCLC is shaped by host genomic architecture and conserved functional hotspots, providing new insights into the mechanisms of genetic and epigenetic dysregulation associated with LINE-1 retroelements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The short arms of human acrocentric chromosomes are characterized by nucleolar organizer regions essential for ribosome biogenesis, but their highly repetitive nature has hindered genomic analysis. Leveraging the recently completed genomes of all major ape lineages, we identified recurrent features of their acrocentrics, including enriched repeat classes, centromere repositioning by whole-arm inversion, interchromosomal sequence exchange, and birth-and-death evolution of multiple gene families. Together, these processes have enabled the repeated amplification and diversification of the FRG1 gene family over 25 million years of ape evolution, and, in gorilla, the formation and amplification of a novel IGSF3-GGT fusion gene under positive selection. Similar evolutionary events also explain the distribution of segmental duplications and heterochromatin in the modern human genome, predisposing it to karyotypic abnormalities such as Robertsonian translocations. Our findings highlight acrocentric chromosomes as key drivers of evolution in the great apes, with implications for speciation, adaptation, and clinical genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Introduction Cancer cell proliferation occurs within the context of persistent genomic instability. In this review, we propose the RS–CCD–CIN axis as a systems-level framework in which replication stress (RS), cell cycle deregulation (CCD) and chromosomal instability (CIN) form an interdependent triad that shapes tumour evolution. This axis represents a constrained metastable state in which genomic instability is tolerated and buffered. The objective of this review is to synthesize the current understanding of how the RS–CCD–CIN axis contributes to tumour heterogeneity, adaptability and therapy response. Discussion Evidence indicates that RS, CCD and CIN operate as a dynamic, interconnected network rather than as independent processes. Replication stress induces DNA damage and mutagenesis, while partial checkpoint disruption permits cells with unresolved lesions to proliferate. Chromosomal instability generates both structural and numerical alterations, contributing to intratumoural heterogeneity. Together, these processes facilitate adaptation to environmental and therapeutic pressures. Extrachromosomal DNA, micronuclei formation and cytosolic DNA signalling, including the cGAS–STING pathway, connect genomic instability to adaptive responses and immune modulation. Single-cell and spatial profiling reveal temporal and spatial variability in RS, CCD and CIN states, highlighting the limitations of static biomarkers. Therapeutically, targeting individual components often yields limited durability, whereas approaches that simultaneously perturb multiple aspects of the RS–CCD–CIN axis may improve clinical outcomes. Conclusions This review highlights the RS–CCD–CIN axis as a fragile and metastable architecture that supports cancer evolution, while also being susceptible to collapse. A deeper understanding of this interconnected framework may inform the development of therapeutic strategies and enhance the management of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294582a14e94664c9f90d67cf0b0db4402660bea" target='_blank'>
              A system-level metastable model of cancer evolution: integrating replication stress, cell cycle deregulation and chromosomal instability
              </a>
            </td>
          <td>
            Dongjing Ma, Jinbin Wang, Yicong Chen, Ling Yao, Qianya Wei, Jianjun Wu
          </td>
          <td>2026-01-28</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/755b604be820a14cd765634bf7256cd13a7ee21b" target='_blank'>
              Protein-only centromeric chromatin assembly streamlines human artificial chromosome formation
              </a>
            </td>
          <td>
            Gabriel J. Birchak, P. K. Allu, Prakriti Kashyap, G. K. Anbalagan, Shu-Cheng Chuang, Glennis A. Logsdon, Daniel G. Gibson, John I. Glass, Ben E. Black
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Aim: Prostate cancer features profound transcriptional dysregulation within the androgen receptor (AR) signaling axis. The pioneer factor FOXA1, which facilitates AR binding to chromatin, is recurrently altered in 10-40% of tumors. Recent studies classify FOXA1 mutations as Class 1 Wing 2 mutations, which enhance AR-dependent tumorigenesis, and Class 2 C-terminal truncations, which promote lineage plasticity and therapy resistance. The interplay of FOXA1 alterations with TMPRSS2-ERG fusions and PROX1 remains incompletely understood. Materials and Methods: Data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) cohort (n=492) were analyzed via UCSC Xena and cBioPortal. FOXA1 mutations were categorized following Eyunni et al. Copy number was assessed by log2(tumor/normal) ratios. Mutual exclusivity and co-occurrence were evaluated using Fisher’s exact test with false-discovery-rate correction. Associations between FOXA1 status and genomic instability were assessed using the fraction genome altered (FGA) metric. Results: FOXA1 was broadly expressed, with subsets showing elevation. Class 1 mutations localized to the Wing 2 region, while Class 2 truncations clustered in the C-terminal domain. Copy number changes were infrequent, indicating mutation-driven reprogramming as the main oncogenic mechanism. TMPRSS2 and ERG strongly co-occurred (log2 OR >3, q<0.001), whereas FOXA1 was mutually exclusive with both TMPRSS2 and ERG (q<0.001). Although FOXA1 alterations showed no significant Pearson correlation with FGA (r=−0.01, p=0.76), a moderate Spearman correlation (ρ=0.52, p<0.001) suggested enrichment in genomically unstable tumors. Conclusion: FOXA1 defines a major oncogenic axis in prostate cancer, distinct from TMPRSS2-ERG fusion and PROX1 induction. Class 1 and 2 FOXA1 mutations drive alternative transcriptional programs leading to therapy resistance, highlighting FOXA1 as a critical biomarker and target for chromatin-directed interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962282da85200138085414df96f69e7ea38e7831" target='_blank'>
              FOXA1 Alterations in Prostate Cancer: Expression, Mutation Classes, and Copy Number Changes
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450603a806f69ebb4194f3b7b3ab19ed0a5b5513" target='_blank'>
              A holocentric pangenome links karyotype evolution to meiotic recombination
              </a>
            </td>
          <td>
            Meng Zhang, Stefan Steckenborn, Marco Castellani, Laia Marín-Gual, A. S. Câmara, Laura A. Robledillo, Letícia Maria-Parteka, N. Sargheini, Magdalena Marek, Eduardo Chacón-Madrigal, Andrés Gatica Arias, L. P. Felix, William W. Thomas, B. Huettel, A. Pedrosa‐Harand, John T. Lovell, André L. L. Vanzela, Aurora Ruiz-Herrera, A. Marques
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Ribosomal DNA (rDNA) encodes the 18S, 5.8S, and 28S rRNA, accounting for ∼70% of cellular transcription. Despite its essential role and links to cancer and aging, quantifying rDNA instability in mammals remains challenging due to its repetitive organization and inherent heterogeneity. Here, we developed a murine rDNA FISH probe and genomic tools tailored for laboratory mouse strains. The results confirmed rDNA cluster locations, revealed substantial inter- and intra-strain as well as intercellular heterogeneity in rDNA organization within inbred mice and unstressed cells, and identified sources of spontaneous and replication-associated DNA double-strand breaks in the rDNA transcription termination region. Using mouse embryonic stem cells, we showed that BRCA1-mediated homologous recombination promotes rDNA instability, the non-homologous end joining factor XRCC1, but not Ku, suppresses intra-cluster deletions, and ATM kinase preserves rDNA cluster stability. Together, these findings establish a platform and tools for studying rDNA instability in animal models relevant to aging and cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377fa5f7328f7f52cd2825bb12b7a086a7b2a641" target='_blank'>
              Visualization and quantification of rDNA instabilities in mammalian cells and mouse models
              </a>
            </td>
          <td>
            Xiaolu Zhu, Wenxia Jiang, Wei Wu, Brian J. Lee, Demis Menolfi, Anthony Tubbs, Olivia M. Cupo, Eli Malkovskiy, Mattie Nester, Xiaobin S. Wang, Peter A. Sims, Chyuan-Sheng V Lin, Lorraine Symington, Andre Nussenzweig, Brian McStay, Shan Zha
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0f98bd86315e8e5aa3b48daf84b94980e7dd61" target='_blank'>
              Dynamic Supercoiling Sponsors Transcription Amplification by MYC
              </a>
            </td>
          <td>
            R. Jha, Fedor Kouzine, Bo Wang, James D. Phelan, S. K. Das, Brian A Lewis, D. Levens
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6dd597d404f6624bed3a43a3ecd045b834d7d8b" target='_blank'>
              Integration of an anellovirus genome in the SKNO-1 acute myeloid leukemia cell line
              </a>
            </td>
          <td>
            Na Cui, S. Goya, E. Piliper, Alexander L. Greninger
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Hepatocellular carcinoma (HCC) is the most common primary liver malignancy worldwide and is associated with a high mortality rate. The unique genetic features and structural properties of extrachromosomal circular DNA (eccDNA) offer a promising approach for early cancer diagnosis.


OBJECTIVE
This study aims to investigate the potential of eccDNA as a diagnostic marker by identifying its distinguishing characteristics using a two-dimensional cell culture model.


METHODS
EccDNA was isolated from the supernatant of HepG2 and THLE-2 cell cultures. Long-read sequencing and bioinformatic analysis were employed to detect and characterize eccDNA. The size distribution, chromosomal origin, and genomic annotation of eccDNA from both cell types were compared. PCR was performed to validate DNA fragments at the junctions of the closed-circular forms.


RESULTS
The total number of cell-free eccDNA derived from HepG2 cells averaged 2,669,673 bases, encompassing 2,542 sequences, while THLE-2 cells averaged 857,718 bases with 975 sequences. Size distribution analysis revealed that most cell-free eccDNA from both cell types ranged from 351 to 600 bp. Bioinformatic analysis revealed that approximately 80% of eccDNAs corresponded to gene coding regions. Ten eccDNAs were the most frequently detected in HepG2 cells, with four eccDNAs harboring CDC27P11, RAC1P3, LOC112268123, and LOC124902279 genes successfully validated and uniquely detected in HepG2 cells, suggesting their potential as biomarkers for HCC diagnosis.


CONCLUSION
Distinct eccDNA types were identified in HepG2 cells, which may serve as promising biomarkers for HCC diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2857a0fc338d1571175ba6c4c210b5ca1dafdfd9" target='_blank'>
              Identification of Extrachromosomal Circular DNA Isolated from Cell Culture Supernatant and Its Potential Applications for Hepatocellular Carcinoma Diagnosis.
              </a>
            </td>
          <td>
            Suchitraporn Sukthaworn, Suchanuch Ondee, Opas Choksupmanee, Suparat Taengchaiyaphum, Prasert Yodsawat, Ponsit Sathapondecha
          </td>
          <td>2026-01-21</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="C1orf50 encodes a small, evolutionarily conserved protein, the function of which remains unclear. Its significance across various human cancers, particularly its specific role in ovarian cancer within an immunogenomic context, is not yet fully understood. Utilizing The Cancer Genome Atlas and single-cell RNA sequencing (scRNA-seq) public datasets, we conducted a comprehensive profiling of C1orf50 across multiple cancer types, with a particular focus on ovarian cancer, to investigate its associations with copy-number status, genomic instability, tumor programs, and the immune microenvironment. Across cancer types, copy-number gain or amplification of C1orf50 was most frequent in ovarian cancer and closely tracked with higher messenger RNA levels. Higher C1orf50 expression was associated with a greater tumor mutational burden and homologous recombination deficiency, as indicated by gene-set patterns that suggested heightened cell-cycle and cellular stress responses accompanied by reduced oxidative phosphorylation, enrichment of regulatory T cells, and depletion of resting memory CD4 T cells. In ovarian cancer, focal events at chromosome 1p34.2 were accompanied by stepwise increases in C1orf50 expression by clinical stage and were linked to higher tumor mutational burden, homologous recombination deficiency, and greater loss of heterozygosity, together with more frequent gene alterations in BRCA1 or BRCA2. Immune composition clustered into profiles consistent with an immunosuppressive context in tumors with higher C1orf50 expression. The scRNA-seq data further revealed that cancer cells enhanced immune-suppressive interactions with various immune cell populations and diminished antigen-presentation signals. Analyses of genomic instability in ovarian cancer suggested mutational processes compatible with base-substitution patterns associated with cytidine deaminase activity and with insertion-deletion patterns characteristic of homologous recombination failure, while transcript-level patterns pointed to a broad downshift of canonical DNA repair activity with apparent compensatory adjustments in related pathways rather than a uniform change in any single pathway. The overexpression of C1orf50 characterizes an aggressive immunogenomic phenotype in ovarian cancer, distinguished by genomic instability, impaired DNA repair mechanisms, and extensive immunosuppression. These findings indicate that C1orf50 warrants consideration as a potential biomarker and a prospective target for therapeutic investigation. Furthermore, they advocate for the progression to prospective validation and functional studies to ascertain its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/308ec9e744b73dd3ecd51f7b2385a3a46667c2cb" target='_blank'>
              Pangenome analysis reveals the evolutionary dynamics of repeat-based holocentromeres
              </a>
            </td>
          <td>
            Piotr Włodzimierz, Estela Perez-Roman, A. S. Câmara, Laura A. Robledillo, Gokilavani Thangavel, Meng Zhang, Jacob González Isa, L. M. Parteka, Marco Castellani, B. Huettel, André L. L. Vanzela, Ian R. Henderson, Alexandros Bousios, A. Marques
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Neuroblastoma RAS viral oncogene homolog (NRAS) mutations occur in 20-30% of cutaneous melanomas, defining a distinct molecular subtype. While NRAS mutations drive unique gene expression programs, locus-specific transcriptional regulation has not been systematically explored. This study characterized coordinated transcriptional upregulation in NRAS-mutant melanoma, focusing on chromosome 1p13.2. RNA-seq and mutation data for 472 melanoma samples from The Cancer Genome Atlas Skin Cutaneous Melanoma were analyzed. Differential expression, coexpression assessment, and copy number variation (CNV) correlation were performed for 45 protein-coding genes within 1p13.2. Findings were validated in MSK-IMPACT 2021 (n = 696). In NRAS-mutant melanomas, 56% of 1p13.2 genes were upregulated (false discovery rate < 0.05) versus 12% genome-wide (P < 0.001), including TRIM33, AMPD1, GNAI3, and SLC25A24. Upregulation was NRAS-specific: 84% showed higher expression versus BRAF-mutant tumors. Six of the 10 most NRAS-correlated genes localized to 1p13.2. CNV gains showed dose-dependent effects (gains: 1.8-3.2 fold; amplifications: 3.5-5.4 fold), with strong correlations (AMPD1 ρ = 0.68, GNAI3 ρ = 0.62, TRIM33 ρ = 0.58; P < 0.001). MSK-IMPACT validation confirmed 1p13.2 upregulation. As these findings are based on transcriptomic and copy number analyses, additional protein-level and functional studies across independent cohorts are required to confirm biological significance. NRAS-mutant melanomas harbor a coordinated transcriptional program within 1p13.2, driven by CNV gains. This locus contains genes with potential druggability, offering new avenues for combinatorial targeting alongside mitogen-activated protein kinase pathway inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb61b6e16a4770e24198eb7ffe671bddbc211884" target='_blank'>
              Chromosome 1p13.2 gene cluster upregulation in neuroblastoma RAS viral oncogene homolog pathogenic variant melanoma: a potential biomarker axis.
              </a>
            </td>
          <td>
            Sophie M. Bilik, Olivia M Burke, S. Elman
          </td>
          <td>2026-01-29</td>
          <td>Melanoma research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078474078b9a084c2455a85f367e87d76cce9bc4" target='_blank'>
              Lineage-determining transcription factors EBF1 and TCF1 shape chromatin fibre folding
              </a>
            </td>
          <td>
            R. Faryabi, Yeqiao Zhou
          </td>
          <td>2025-12-10</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566c090308e26a25e5654df687dc08aebd75d6cb" target='_blank'>
              EZH2 Inhibition Reshapes 3D Chromatin Architecture to Induce Immunogenic Phenotype in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Sana Parveen, Ramesh Adhinaveni, Kun Fang, Lavanya Choppavarapu, Meijun Du, Gustavo Leone, N. De Sarkar, Victor X. Jin, 
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pedigree analysis remains the gold standard for rare disease diagnostics, yet whole genome sequencing studies typically omit critical regions like centromeres, telomeres, and acrocentric chromosome p-arms. Here, we present telomere-to-telomere (T2T) reference genomes for four self-identified African American individuals of admixed ancestry spanning three generations. Our parent-of-origin assigned, chromosome-level assemblies revealed precise meiotic recombination breakpoints in previously inaccessible regions, including recombination events across acrocentric and subtelomeric sequences. Centromeric regions were highly stable, with multi-megabase arrays inherited intact across three generations, while the position of kinetochore assembly sites remained consistent and predominantly associated with the p-arm proximal region. The relative lengths of telomeres on individual chromosomes were maintained across generations. Using a targeted rDNA assembly approach, we reconstructed a complete megabase-scale ribosomal DNA (rDNA) array corresponding to the paternal chromosome 14. This openly available pedigree provides a benchmark dataset for studying recombination and genetic and epigenetic variation across the complete genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56491af219e3e15ccd2642a4c21320991315613" target='_blank'>
              Complete genomes of a multi-generational pedigree to expand studies of genetic and epigenetic inheritance
              </a>
            </td>
          <td>
            Monika Cechova, T. Potapova, A. Rechtsteiner, Glenn Hickey, Rebecca Serra Mari, Mira Mastoras, Julian Menendez, Nikol Poláková, Prajna Hebbar, Fedor Ryabov, Hailey Loucks, Aljona Groot, Tomáš Pavlík, Mobin Asri, Shihua Dong, Stephanie M. Yan, Julian K. Lucas, Steven J. Solar, M. Borchers, Mark Mattingly, S. McKinney, M. Krátká, Catherine Mikhailova, Ondřej Hanák, Sohinee Tiffany Saha, Emily Xu, D. Antipov, S. Koren, Jordan M. Eizenga, B. McNulty, Joshua M.V. Gardner, T. Hillaker, I. Violich, Christopher Markovic, S. Kruglyak, Shawn Levy, Trevor Wolf, Matthew W. Mitchell, Laura Scheinfeldt, Haoyu Cheng, I. Alexandrov, R. McCoy, B. Paten, A. Phillippy, J. Zook, J. Gerton, Robert S. Fulton, N. Stitziel, Ting Wang, T. Marschall, C. Greider, K. Miga
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Genomic instability is a hallmark of cancer and contributes to disease progression through various mechanisms. DNA copy number variation (CNV) is one of the most significant consequences causing a phenomenon called aneuploidy. Aneuploidy is characterized by an imbalanced genome and is associated with lethal progression in prostate cancer. Currently, sequencing-based CNV analysis is widely used to identify cancer-specific aneuploidy that contribute to tumorigenesis, immune evasion, and disease progression. Performing low-pass whole genome sequencing (LP-WGS) on DNA extracted from bulk tissues or cell cultures, is a convenient approach to use for detecting aneuploidy and focal CNVs. However, conventional bulk LP-WGS methods average genomic signals across thousands of cells, often overlooking tumor heterogeneity and small populations with distinct CNV patterns. To address this limitation, we optimized a microfluid nanowell-based system to analyze aneuploidy patterns at a single-cell level using disassociated tissues and isolated nuclei from prostate samples. This pipeline is compatible with both preclinical models and clinical specimens and can be integrated with other multi-omics platforms to generate multimodal datasets for basic and translational research. By leveraging high-throughput single-cell sequencing technologies, our approach enables direct DNA CNV measurement with high sensitivity at a single-cell resolution. Ultimately, this approach aims to improve the resolution of CNV detection, enhance molecular stratification and enable multimodal data integration for better understanding of clinical prostate cancer samples.



 Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, William D. Figg, Gregory Chesnut, Matthew G. Vander Heiden, Xiaofeng A. Su. Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A060.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6642b028922ed7c8904563bae6731f81ce2d7a38" target='_blank'>
              Abstract A060: Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing
              </a>
            </td>
          <td>
            Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline M. Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, W. Figg, Gregory T. Chesnut, M. V. Vander Heiden, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f4b87b068fd900c29c6efac827028094f6b686" target='_blank'>
              Removal of nascent transcripts by TTF2 is required for efficient sister chromatid resolution in human cells
              </a>
            </td>
          <td>
            Laura Tovini, Inês Milagre, Catarina Peneda, 
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gene regulatory innovation underlies many phenotypic transitions. Transposable elements are an established mechanism for creating families of cis-acting elements with shared sequence features and the potential to establish co-regulatory networks. To understand additional mechanisms by which co-regulatory networks form, we define families of noncoding elements based on sequence similarity and cell type-specific activity. We apply this analysis framework to the human telomere-to-telomere genome assembly and embryonic stem cell chromatin accessibility data. We identify segmental duplications as the major mechanism establishing these families, creating over one thousand networks of elements with open chromatin in embryonic stem cells. We functionally validate a subset of these networks as families of regulatory elements with STARR-seq and identify their target genes with CRISPRi in embryonic stem cells. Following segmental duplication, we find that regulatory elements at times maintain their relationship to target genes, and at times rewire to form novel connections. During this rewiring, we observe proximal-acting elements gaining the ability to regulate distally-located genes and observe transcriptional enhancers rewiring to regulate genes present at the locus outside the segmental duplication. Many of these rewiring events are human specific. Finally, we find that segmental duplications have made outsized contributions to expanding regulatory element families functioning in immune cell types and specific brain regions, including the posterior cingulate gyrus. We speculate that placing regulatory elements in new genomic contexts primes regulatory elements for neofunctionalization, and that regulatory rewiring after segmental duplication was a common mechanism underlying gene regulatory change during human evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is among the most prevalent malignancies. However, the regulatory networks involved in tumor occurrence and development are still poorly understood. Human endogenous retroviruses (HERVs), a class of transposable elements, have been implicated in the development and progression of various human cancers. This study presents the first comprehensive locus-specific profiling of the expression of HERV gene transcripts in rectal cancer, revealing significantly dysregulated HERVs. Analysis of data from three Gene Expression Omnibus data sets revealed 25 upregulated HERVs and 7 downregulated HERVs. Dysregulation of HERV6196, a type of HERVH, was validated through reverse transcription quantitative PCR and droplet digital PCR in cells and tissues. Additionally, HERV6196 promoted the proliferation, inhibited the apoptosis, enhanced the colony formation ability, and enhanced the migration capability of rectal cancer cells. Moreover, HERV6196 functioned as an enhancer, promoting the expression of neighboring genes and the development of CRC. In summary, the present results revealed that HERV6196 is involved in the pathogenesis of rectal cancer, indicating the potential contribution of dysregulated HERVs to the development and progression of CRC through gene expression modulation. IMPORTANCE The role of human endogenous retroviruses (HERVs) in colorectal cancer (CRC) remains insufficiently understood. The present study revealed aberrant expression of HERV gene transcripts in cancerous tissues compared with non-cancerous tissues. HERV6196 contributes to CRC progression by regulating the expression of neighboring genes. These findings suggest that HERVs may serve as enhancers and regulate oncogenic gene expression, providing new insights for rewiring transcriptional regulatory networks in CRC pathogenesis. The role of human endogenous retroviruses (HERVs) in colorectal cancer (CRC) remains insufficiently understood. The present study revealed aberrant expression of HERV gene transcripts in cancerous tissues compared with non-cancerous tissues. HERV6196 contributes to CRC progression by regulating the expression of neighboring genes. These findings suggest that HERVs may serve as enhancers and regulate oncogenic gene expression, providing new insights for rewiring transcriptional regulatory networks in CRC pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8dcd97edf79418a2a506bad015f01e1c7fbd0b0" target='_blank'>
              HERV6196 as an enhancer with oncogenic potential in rectal cancer
              </a>
            </td>
          <td>
            Yi-Xiu Gan, Xin Jiang, Zhi-Yu Wang, Yi-Lin Yu, Ling-Dong Shao, Jianmin Wang, Jun-Xin Wu
          </td>
          <td>2025-01-06</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This review aims to synthesize current knowledge on Cyclin-dependent kinase 12 (CDK12) and Cyclin-dependent kinase 13 (CDK13), two transcriptional kinases with distinct and overlapping roles in gene regulation, genome stability, and cancer biology. These kinases control transcription elongation and co-transcriptional processing, including splicing and polyadenylation, thereby linking RNA polymerase II activity with DNA repair and replication stress response. CDK12 and CDK13 play context-specific roles across malignancies, where amplification, truncating mutations, fusions, and deletions variably alter their transcriptional output and downstream DNA repair programs. Their dysregulation contributes to oncogenesis, metastasis, and resistance to conventional therapies. There is growing therapeutic interest in CDK12/13 inhibition, especially as selective inhibitors of single DNA repair nodes have shown limited success. Selective CDK12/13-targeting agents have shown efficacy in preclinical models by inducing synthetic lethality, restoring sensitivity to PARP inhibitors, and are being explored to potentiate immune checkpoint blockade. This review highlights the biological rationale for targeting CDK12 and CDK13 in cancer, summarizes emerging therapeutic strategies, and identifies key opportunities for integrating these approaches into precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6aa4a8b6800b768181f51eaa1d8b4bd48846cd" target='_blank'>
              CDK12 and CDK13 in oncology: from RNA regulation to therapeutic targeting
              </a>
            </td>
          <td>
            Julia Dudkiewicz-Garbicz, Paweł K Włodarski
          </td>
          <td>2026-01-05</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e732ad2e253dbdf86a5a9ac4cf431aea3406d6e9" target='_blank'>
              Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells
              </a>
            </td>
          <td>
            Aritra Gupta, Siddharth Bhardwaj, K. Desai
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfde1f929fb1d5e109604b5adc00a4ea625430b8" target='_blank'>
              Localised activity of reverse gyrase at gene regulatory elements
              </a>
            </td>
          <td>
            Paul Villain, Vladislav Kuzin, Augustin Darennes-Degaugue, Florence Lorieux, Antoine Hocher, Romain Le Bars, Tobias Warnecke, Laura Baranello, Tamara Basta
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ewing sarcoma is an aggressive bone tumor of adolescence characterized by a hallmark EWSR1::ETS fusion oncogene. The resulting chimeric oncoprotein drives tumorigenesis by reshaping transcriptional and epigenetic landscapes. However, how it is transcriptionally regulated and whether additional master transcription factors (MTFs) form a core regulatory circuit (CRC) in Ewing sarcoma remain unclear. Using an extensive panel of Ewing sarcoma cell lines and primary tumors, we mapped super-enhancers and identified enrichment of GGAA microsatellites, confirming their specificity to Ewing sarcoma as compared to other pediatric cancers and normal tissues. Integrating transcriptomic, epigenetic, 3D chromatin conformation, and dependency data, we predicted a set of MTFs potentially forming a CRC. However, functional validation demonstrated that these MTFs neither establish auto-regulatory loops nor confer robust proliferative dependencies typical of CRCs in other pediatric tumors. Instead, EWSR1::FLI1 emerged as an “hegemonic” oncoprotein, regulating expression of these MTFs without reciprocal regulation. Knockdown of EWSR1::FLI1 strongly shifted H3K27ac profiles toward mesenchymal states, whereas silencing individual or combined MTFs did not alter cell growth or EWSR1::FLI1 expression. These findings highlight the absence of a classical CRC in Ewing sarcoma and emphasize EWSR1::FLI1 as the dominant oncoprotein and a major vulnerability in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The dynamic three-dimensional (3D) organization of the human genome (the 4D nucleome) is linked to genome function. Here we describe efforts by the 4D Nucleome Project1 to map and analyse the 4D nucleome in widely used H1 human embryonic stem cells and immortalized fibroblasts (HFFc6). We produced and integrated diverse genomic datasets of the 4D nucleome, each contributing unique observations, which enabled us to assemble extensive catalogues of more than 140,000 looping interactions per cell type, to generate detailed classifications and annotations of chromosomal domain types and their subnuclear positions, and to obtain single-cell 3D models of the nuclear environment of all genes including their long-range interactions with distal elements. Through extensive benchmarking, we describe the unique strengths of different genomic assays for studying the 4D nucleome, providing guidelines for future studies. Three-dimensional models of population-based and individual cell-to-cell variation in genome structure showed connections between chromosome folding, nuclear organization, chromatin looping, gene transcription and DNA replication. Finally, we demonstrate the use of computational methods to predict genome folding from DNA sequence, which will facilitate the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function. The 4D Nucleome Project demonstrates the use of genomic assays and computational methods to measure genome folding and then predict genomic structure from DNA sequence, facilitating the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>92</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aneuploidy is near-ubiquitous in cancer and contributes to tumor biology. However, the temporal evolutionary dynamics that select for aneuploidy remain uncharacterized. We performed longitudinal genomic analysis of 755 samples from 167 patients with colorectal-derived neoplasias from different stages through metastasis and treatment. Adenomas had few copy number alterations (CNA) and most were subclonal, whereas cancers had many clonal CNAs, suggesting that progression goes through a CNA bottleneck. Individual colorectal cancer glands from the same tumor had similar karyotypes, despite evidence of ongoing instability at the cell level. CNAs in metastatic lesions, after therapy, and in late recurrences were similar to the primary. Mathematical modeling indicated that these data are consistent with the action of negative selection on CNAs that "trap" cancer genomes on a fitness peak characterized by specific CNAs. Hence, progression to colorectal cancer requires traversing a rugged fitness landscape, whereas subsequent CNA evolution is constrained by negative selection.


SIGNIFICANCE
We profiled 167 long-term responders longitudinally (755 samples), documenting long-term cancer evolution. We found that a genetic bottleneck is required for progression and is associated with dramatic increase in CNAs but decrease in clonal diversity. After initiation, copy number evolution is constrained by negative selection through metastasis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c71eacb567a19cb84e29c03543a7fbd15b5975" target='_blank'>
              Negative Selection Maintains Grossly Altered but Broadly Stable Karyotypes in Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            William Cross, S. Nowinski, George D. Cresswell, M. Mossner, Abhirup Banerjee, B. Lu, Marc J. Williams, G. Vlachogiannis, Laura J Gay, Ann-Marie Baker, C. Kimberley, F. J. Whiting, Hayley L. Belnoue-Davis, Pierre Martinez, Maria Traki, Viola Walther, Kane Smith, J. Fernández-Mateos, Erika Yara-Romero, Erica A Oliveira, Salvatore Milite, G. Caravagna, Chela T James, G. Elia, A. Berner, Chang-Ho Ryan Choi, Pradeep Ramagiri, Ritika Chauhan, Nik Matthews, Jamie Murphy, A. Antoniou, Susan K Clark, M. Mitchison, J. Chin Aleong, Enric Domingo, Inma Spiteri, S. McDonald, Darryl Shibata, M. Laclé, L. Wang, Morgan Moorghen, Ian P.M. Tomlinson, Marco Novelli, Marnix Jansen, A. Watson, Nicola Valeri, Nicholas A. Wright, John Bridgewater, M. Rodriguez-Justo, C. Barnes, Hemant M Kocher, Simon J. Leedham, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2026-01-21</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d39ef025a1bb316bde2c6dad77b6828031b8946" target='_blank'>
              Functional characterization of the 9q34.13 locus identifies RAPGEF1 as modulating risk for melanoma and nevi via RAS activation
              </a>
            </td>
          <td>
            Rohit Thakur, Mai Xu, Alexandra M. Thornock, Joshuah Yon, Samuel Anyaso-Samuel, M. Lauss, Thomas Rehling, Linh Bui-Raborn, Hayley A. Sowards, Gerard Duncan, Lea Jessop, Timothy Myers, Raj Chari, Erping Long, K. Funderburk, Jinhu Yin, Rebecca Hennessey, Emory Hseih, Hannah Levin, Mitchell John Machiela, Tongwu Zhang, Goran Jonsson, D. T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Law, T. Andresson, Jiyeon Choi, Leonard I. Zon, Jianxin Shi, Kevin M. Brown
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b11efdad5d60344c302ed6d2c56a7d648ae4da7" target='_blank'>
              Targeted therapy-induced chromosomal instability dictates mitotic dependency on Aurora Kinase A
              </a>
            </td>
          <td>
            Chendi Li, Varuna Nangia, Melissa D. Vieira, Anahita Nimbalkar, Christopher J. Graser, Jeremy B Chang, Mohammad U. Syed, Yi Shen, Radhika Koranne, Lee Zou, Xueqian Gong, Sabrina L Spencer, Aaron N. Hata
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The ciliate Oxytricha undergoes massive genome rearrangements during development to produce a functional nucleus from an encrypted zygotic genome. PIWI-interacting small RNAs protect DNA regions against deletion, but how that protective mark is established has been a mystery. Recently our lab discovered MTA1, a methyltransferase that catalyzes DNA N6-adenine (6mA) methylation. Both MTA1 and the Oxytricha Piwi protein, Otiwi1, are required for development, and Otiwi1 mutation eliminates 6mA signal. To examine the role of 6mA, we analyzed its genome-wide distribution across development in wild-type and MTA1 mutant backcrossed cells. We find specific and abundant enrichment on retained sequences, suggesting a protective role for this epigenetic mark. Furthermore, programmed retention of a DNA region that is normally deleted leads to accumulation of new 6mA marks on the ectopically retained DNA sequence. Together, these results suggest that piRNA-guided 6mA DNA methylation leads to protection of DNA sequences against deletion during nuclear differentiation. Highlights DNA N6-methyladenine accumulates on retained DNA regions during Oxytricha development. mta1 mutant backcrosses have disrupted DNA methylation and a developmental delay. DNA N6-adenine methylation during genome rearrangement requires the presence of Otiwi1. Programmed retention of a germline-limited region leads to developmental methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecfb36caac2df07b165f21a1abfc41a4ec2bd5d" target='_blank'>
              A PIWI protein-dependent DNA N6-adenine methylation pathway in Oxytricha protects genomic sequences from deletion
              </a>
            </td>
          <td>
            Margarita T. Angelova, Yi Feng, D. Villano, Erhan Aslan, Michael W. Lu, P. Esteve, Sagnik Sen, William E. Jack, S. Pradhan, L. Landweber
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Over the past 5–7 million years, humans and chimpanzees have diverged in brain size, structural complexity, and cognitive abilities despite high conservation of protein-coding genes. Notably, the endogenization and proliferation of retroviral infections within host genomes has introduced numerous species-specific regulatory elements that have the potential to influence gene regulation. However, the role of these endogenous retroviruses in hominoid brain evolution remains unclear. A burst of lineage-specific PTERV1 retroviruses recently invaded the chimpanzee genome but are absent in humans. We conducted an epigenomic analysis of PTERV1 insertions in chimpanzee neural organoids and found that they are heavily covered by DNA methylation, representing more than 150 species-specific heterochromatin domains with the capacity to influence gene regulatory networks. We identified one such chimpanzee-specific PTERV1 insertion on chromosome 19 that blocks the expression of the long noncoding RNA LINC00662, via DNA methylation spread to the adjacent genomic region. The expression of LINC00662 was restored in chimpanzee induced pluripotent stem cells when we deleted the PTERV1 insertion using CRISPR editing. We found that LINC00662, a human-specific RNA, is highly expressed in the developing brain and plays an important role in the posttranscriptional control of neuronal maturation, axon outgrowth, and neural organoid development. In summary, our findings describe how endogenous retroviral insertions contributed to the functional divergence of the human and chimpanzee brains. This provides a new mechanism by which retroviral pandemics influenced primate brain speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7aafdaae0e68f3439418fdcde81bbb4ad6e768" target='_blank'>
              Retroviral insertions contributed to the divergence of human and chimpanzee brains
              </a>
            </td>
          <td>
            Patricia Gerdes, Ofelia Karlsson, R. Garza, Diahann A. M. Atacho, Pinghsun Hsieh, Bharat Prajapati, Chandramouli Muralidharan, Anita Adami, Carrie Davis-Hansson, Emily M. Johansson, Laura Castilla-Vallmanya, Wiktor West, Meghna Vinoud, Lucian Domitrovic, Pia A. Johansson, Jenny G. Johansson, C. Douse, C. Kanduri, Patric Jern, E. Eichler, J. Jakobsson
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb92c350f57793eebc90671a169ae635ef64d" target='_blank'>
              Reverse gyrase and 3D genome architecture suppress hyperthermophile genome instability arising from horizontal gene transfer
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, T. Yamagami, Yoshizumi Ishino, Haruyuki Atomi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dace37f687cf3f6f5d90e43b0621daaa3dd0b0" target='_blank'>
              Distinct p21 dynamics drive alternative routes to whole-genome duplication through a common CDK4/6-dependent polyploid G0 state
              </a>
            </td>
          <td>
            Yian Yang, Anuraag Bukkuri, Dante Poe, Janet McLaughlin, Sihan Hao, Daniel D. Brown, Katarzyna M. Kedziora, Wayne Stallaert
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In placental XX females, one X chromosome is silenced during a narrow developmental time window by X-chromosome inactivation, which is mediated by Xist noncoding RNA. Although most X-linked genes are silenced during X-chromosome inactivation, some genes can escape. Here, by increasing its endogenous level, we show that Xist RNA can silence escapees well beyond early embryogenesis both in vitro, in differentiated cells, as well as in vivo, in mouse pre- and post-implantation embryos. We further demonstrate that Xist RNA plays a role in eliminating topologically associating domain-like structures spanning clusters of escapees, and this is dependent on SPEN. The function of Xist in silencing escapees and eliminating topological domains is initially fully reversible, but sustained Xist upregulation leads to irreversible silencing and CpG island DNA methylation of escapees. Thus, gene activity and three-dimensional topology of the inactive X chromosome are directly controlled by Xist, well beyond an early developmental time window. The authors show that increased Xist RNA levels can induce de novo silencing of genes that normally escape X inactivation. SPEN depletion prevents the silencing of escape genes upon Xist RNA overexpression in neural progenitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b274cefc089887937597429bef5c9660ece124a" target='_blank'>
              Escape from X inactivation is directly modulated by Xist noncoding RNA
              </a>
            </td>
          <td>
            Antonia Hauth, Jasper Panten, Emma Kneuss, Christel Picard, N. Servant, Isabell Rall, Yuvia A. Pérez-Rico, Lena Clerquin, N. Servaas, L. Villacorta, Ferris Jung, Christy Luong, Howard Y. Chang, Judith B. Zaugg, O. Stegle, D. T. Odom, Edith Heard, Agnese Loda
          </td>
          <td>2025-12-15</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Hui Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a4185cccd9e0a3e27a9efe577810b6e58e6b70" target='_blank'>
              Polymorphic 3D genome architecture mediated by transposable elements
              </a>
            </td>
          <td>
            Harsh G. Shukla, Yuheng Huang, Zita Y. Gao, Y. C. G. Lee
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c691f3f7f0a81f36b7c4733212366543fe923c8f" target='_blank'>
              Deciphering chromosome fusion in D. miranda’s neo-sex chromosome through single-copy and repetitive oligo probes
              </a>
            </td>
          <td>
            Henry Bonilla Bruno, Isabela Almeida, Maria Dulcetti Vibranovski
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Wellcome Trust scientists have shown that “mutational signatures” in specific nucleotide contexts accumulate in genomes of mammalian tissues, providing clues to underlying causes of specific signatures. Analysis of cancer genomes has identified more than 50 single‐base substitution (SBS) signatures, with SBS1, SBS5, and SBS40 linked to aging and present in normal tissues. SBS1 results from cytosine demethylation, whereas SBS5 and SBS40 arise from unknown endogenous mechanisms. We hypothesized that loss of fragile‐site genes drives these two signatures. FHIT, located at FRA3B, is frequently deleted in cancers, and Fhit‐deficient mouse tissues exhibit a mutation profile resembling human SBS5. Data mining of cancer genome sequences showed that FHIT/Fra3B was the gene loss most significantly correlated with human SBS5 mutations, likely representing the endogenous molecular process determining SBS5 mutations in soma, germ cells, and cancers, and inversely associated with lifespan in mammals. Exome sequences of Fhit knockout tissues revealed ~1000 SBS1 and ~4000 SBS5 mutations per sample, compared to markedly fewer in wild‐type controls. Using SigProfilerAssignment, similar analyses performed with Fhit ko samples where Wwox is also frequently lost identified SBS40c. These findings implicate fragile‐site gene inactivation as a major source of clock‐like mutational signatures, contributing to lifelong mutation accumulation and potentially influencing aging, evolution, and speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b576f2a56d83e49e328a800a1ee42f5ac37825f" target='_blank'>
              Endogenous Processes Underlying Clock‐Like Mutational Signatures
              </a>
            </td>
          <td>
            T. Druck, Rami I. Aqeilan, C. Marcelo Aldaz, N. Zanesi, Kay Huebner
          </td>
          <td>2026-01-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus (HPV) and Merkel cell polyomavirus (MCPyV) are DNA tumor viruses that cause human cancer. The mechanisms by which HPV and MCPyV oncoproteins induce genomic instability are not well defined. This minireview discusses the influence of these oncoproteins on the repertoire of proteins at replicating DNA, known as the host replisome, and discusses how new technologies like isolation of proteins on nascent DNA (iPOND) can drive the discovery of viral dysregulation of the host replisome to enhance our understanding of viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f294dc7fa07420ce7e76b025f057d7e7a72cbe2a" target='_blank'>
              Endogenous retroviral elements LTR8B and MER65 regulate the PSG9 locus that promotes trophoblast syncytialization: Insights into placental evolution and pre-eclampsia pathology
              </a>
            </td>
          <td>
            Manvendra Singh, Yuliang Qu, Amit Pande, Julianna Zadora, F. Herse, M. Gauster, Xuhui Kong, Rongyan Zheng, Rabia Anwar, Katarina Stevanovic, Ralf Dechend, Marie Cohen, Attila Molvarec, Jichang Wang, Miriam K. Konkel, Bin Zhang, Cédric Feschotte, Gabriela Dveksler, S. M. Blois, Laurence D. Hurst, Z. Izsvák
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="ABSTRACT Oncogenic viruses can induce epigenetic changes in host cells, which may contribute to cell transformation and tumorigenesis by altering gene expression patterns. Human adenoviruses, while primarily known for their lytic infections, have also been implicated in oncogenic transformation in experimental settings, but the mechanisms underlying adenovirus oncoprotein-induced epigenetic dysregulation remain poorly defined. Building on our recent findings that the adenoviral oncoprotein E1B-55K coordinates cellular transformation through interactions with DNA-bound host transcription factors, we next sought to determine whether these interactions also influence the chromatin landscape. For this purpose, we investigated the epigenetic consequences of E1B-55K expression in primary human mesenchymal stromal cells using histone-targeting MNase chromatin immunoprecipitation-seq, complemented by RNA-seq analysis. We demonstrate that stable expression of HAdV-C5 E1B-55K leads to widespread changes in histone post-translational modifications across the genome. Most notably, E1B-55K expression results in a marked loss of the activating histone marks H3K4me3 and H3K27ac at specific promoters and enhancers. These epigenetic alterations correspond with significant changes in gene expression, suggesting that E1B-55K interferes with normal transcriptional programming by altering the establishment of key histone modifications. Our findings help explain how E1B-55K drives viral transformation by changing epigenetic patterns and improving our understanding of virus-host interactions at the chromatin level. By describing how human adenoviruses disrupt epigenetic homeostasis, we provide insights into the mechanisms by which oncogenic viruses promote early events in cellular transformation. IMPORTANCE Oncogenic viruses can reshape host cell epigenomes to promote transformation, yet the mechanisms by which adenoviral oncogenes exert such control remain poorly understood. Here, we reveal that the human adenovirus E1B-55K oncoprotein induces widespread loss of activating histone modifications, leading to transcriptional silencing of key cellular genes in primary human cells. Building on our previous demonstration of efficient adenoviral transformation of primary human mesenchymal stromal cells, this study provides the first comprehensive view of E1B-55K-mediated epigenetic reprogramming. Our findings uncover a link between E1B-55K-host transcription factor interactions and chromatin remodeling, offering new insight into how adenoviruses disrupt epigenetic homeostasis to initiate early events in oncogenic transformation. These results provide direct evidence for the transcriptional repression function long postulated for E1B-55K and advance understanding of virus-host interplay at the chromatin level. Oncogenic viruses can reshape host cell epigenomes to promote transformation, yet the mechanisms by which adenoviral oncogenes exert such control remain poorly understood. Here, we reveal that the human adenovirus E1B-55K oncoprotein induces widespread loss of activating histone modifications, leading to transcriptional silencing of key cellular genes in primary human cells. Building on our previous demonstration of efficient adenoviral transformation of primary human mesenchymal stromal cells, this study provides the first comprehensive view of E1B-55K-mediated epigenetic reprogramming. Our findings uncover a link between E1B-55K-host transcription factor interactions and chromatin remodeling, offering new insight into how adenoviruses disrupt epigenetic homeostasis to initiate early events in oncogenic transformation. These results provide direct evidence for the transcriptional repression function long postulated for E1B-55K and advance understanding of virus-host interplay at the chromatin level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77947d567465053bbac94ee829a515fc47178965" target='_blank'>
              The adenovirus oncoprotein E1B-55K reshapes epigenetic histone modifications in primary human cells
              </a>
            </td>
          <td>
            Konstantin von Stromberg, Laura Seddar, Britta Gornott, Thomas Dobner, L. D. Bertzbach, W. Ip
          </td>
          <td>2026-01-26</td>
          <td>mBio</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a25c976684f63347d1fe095c70af1dbd2cc6459" target='_blank'>
              A molecular framework of chromatin extrusion in plants
              </a>
            </td>
          <td>
            Sofia Tzourtzou, Zhidan Wang, Sarah Leuchtenberg, D. Latrasse, M. Benhamed, Chang Liu
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) was characterized by a highly complex genome, with structural variations (SVs) playing a significant role in its development. In this study, we employed Oxford Nanopore Technology long‐read sequencing in paired tumor and adjacent normal liver tissues from 74 Chinese HCC patients to thoroughly characterize the landscape of somatic SVs. Our analysis revealed that somatic SVs were more prevalent in hepatitis B virus (HBV)‐related HCC, with chromosome 1 emerging as a major hotspot, and several members of the chromosome 1 open reading frame (C1orf) family genes expression level exhibited significant age‐related difference. Notably, HBV‐related HCC cases exhibited a higher frequency of deletions, particularly among younger ones (≤ 35 years old). In addition, we observed an increased burden of HBV integration events in younger ones. Remarkably, the divergent‐paired related homeobox (DPRX) loci was identified as a novel gene for HBV integration in younger patients. Together, these findings delineated the somatic SV landscape in HCC and underscored age‐associated HBV‐related genomic alterations as key pathological features of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83cdc214e4351424b89dd91edf26352c6acad965" target='_blank'>
              Landscape of Somatic and Age‐Related Pathogenic Structural Variations in Hepatocellular Carcinoma Revealed by Long‐Read Sequencing
              </a>
            </td>
          <td>
            Zhewen Wei, Yinghao Cao, Hongchao Liu, Mei Liu, Bolun Zhang, Jianming Ying, Jianqiang Cai, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Zhen Huang, Jianmei Liu, Xueyan Lv, Zhiwen Luo, Zhicheng Wei, Xiaoshi Zhang, Yi Yang, Yiqiao Deng, Yanjiang Yin, Jinghua Chen, Junbo Liang, Xiaoyue Wang, Yefan Zhang, Hong Zhao
          </td>
          <td>2026-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a387e7c68c6390622f2c2d1c86f9db7afbe8e8ce" target='_blank'>
              miRISC inhibition causes mitotic defects and synergizes with genotoxic agents in cancers
              </a>
            </td>
          <td>
            Minsi Zhang, Ziqi Jiao, Ylenia Cendon Florez, Ivana Lessel, Xiaoyi Li, Melissa A. Yao, Moritz Weigl, Mateusz M. Ozimek, Turgut Dogruluk, Kevin Chen, Keith Conrad Fernandez, Vinagolu K. Rajasekhar, Olesja Popow, João A. Paulo, Davide Pradella, Tanmay Mishra, C. Mastroleo, Juliana I. Delgado, Francesco Enrico D’Amico, Vincenzo Cavalieri, Carol D. Morris, M. B. Tsou, Kevin M. Haigis, Jayanta Chaudhuri, R. Benezra, Joana A. Vidigal, Davor Lessel, Andrea Ventura, Gaspare La Rocca
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e5545d6ea0cdff6467688544916e041cb664d5" target='_blank'>
              Human CCR4-NOT suppresses pervasive transcription and retrotransposable elements
              </a>
            </td>
          <td>
            S. Kulkarni, Alexis Morrissey, A. Sebastian, O. T. Akinniyi, C. Keller, I. Albert, Shaun Mahony, J. Reese.
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Aims Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease. Materials and methods Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines. Results We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression. Conclusions We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f08e435f05db1aa85f8917f417c0101833b4e71" target='_blank'>
              Meiotic pairing through barcode-like satellite DNA repeats
              </a>
            </td>
          <td>
            Lena Skrutl, Ankita Chavan, A. Sintsova, Ilaria Ceppi, Corin J. Ropp, S. Sunagawa, Petr Cejka, Madhav Jagannathan
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oncocytic (Hürthle cell) carcinoma of the thyroid (OCT) is characterized by widespread loss of heterozygosity (LOH), mitochondrial accumulation and recurrent mitochondrial DNA mutations leading to impairment of complex I. Here, we establish and characterize a novel OCT cell line, UT946, which displays severe mitochondrial electron transport chain dysfunction and a Warburg metabolic phenotype. Using a series of cytoplasmic hybrids, we establish that the complex I defect in UT946 stems from a nuclear-encoded loss of function mutation in the complex I subunit NDUFS1. To our surprise, the mutation in NDUFS1 was inherited as a recessive germline allele that underwent LOH in the tumor to expose functional loss of complex I. A re-analysis of 91 OCT tumor genomes revealed that LOH-driven exposure of recessive germline mutations in complex I subunits was a recurrent mechanism underlying complex I inactivation in OCT. These findings unveil a new germline-driven mechanism of complex I loss and metabolic reprogramming in cancer, and provide further evidence of the strong selective pressure for complex I impairment in OCT. Teaser Germline mutations in complex I induce aerobic glycolysis in oncocytic carcinoma of the thyroid through somatic loss of heterozygosity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec415c1026a19fc77b2f19d7d97c4e52057a5cf" target='_blank'>
              Loss of heterozygosity exposes germline mutations in complex I and drives Warburg metabolism in oncocytic carcinoma of the thyroid
              </a>
            </td>
          <td>
            Celia de la Calle Arregui, A. Frank, Kelvin Mun, Jiwoong Kim, Kuntal Majmudar, Justin A. Bishop, C. Lewis, Yang Xie, David G. McFadden
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0453b8fd3ee789a5730a5cb95cb52509da58c9" target='_blank'>
              Unraveling miRNA-Driven DNA Damage Response Networks in Pancreatic Adenocarcinoma: A Multi-Omics and Machine Learning Approach
              </a>
            </td>
          <td>
            Shayori Bose, Paramita Basak Upama, S. Ahamed
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Nucleophosmin (NPM1), a nucleolar protein frequently mutated in hematopoietic malignancies, is overexpressed in several solid tumors with poorly understood functional roles. Here, we demonstrate that Npm1 is upregulated after APC loss in WNT-responsive tissues and supports WNT-driven intestinal and liver tumorigenesis. Mechanistically, NPM1 loss induces ribosome pausing and accumulation at the 5’-end of coding sequences, triggering a protein synthesis stress response and p53 activation, which mediate this antitumorigenic effect. Collectively, our data identify NPM1 as a critical WNT effector that sustains WNT-driven hyperproliferation and tumorigenesis by attenuating the integrated stress response and p53 activation. Notably, NPM1 expression correlates with elevated WNT signaling and proliferation in human colorectal cancer (CRC), while CRCs harboring NPM1 deletions exhibit preferential TP53 inactivation, underscoring the clinical relevance of our findings. Being dispensable for adult epithelial homeostasis, NPM1 represents a promising therapeutic target in p53-proficient WNT-driven tumors, including treatment-refractory KRAS-mutant CRC, and hepatic cancers. Npm1 promotes tumor formation via attenuating the integrated stress response and p53 activation in mouse WNT-driven intestinal and liver tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc59acfaa4bd757bcf8852b2707e63fc79814e81" target='_blank'>
              Nucleophosmin supports WNT-driven hyperproliferation and tumor initiation
              </a>
            </td>
          <td>
            Georgios Kanellos, Chiara Giacomelli, A. Raven, Nikola Vlahov, Hu Jin, Pauline Herviou, Sudhir B. Malla, N. Nasreddin, Patricia P. Centeno, Constantinos Alexandrou, Kathryn L Gilroy, Rachel L Baird, K. Pennel, June Munro, Joseph A. Waldron, Holly Hall, Leah Officer-Jones, Sheila Bryson, D. Strathdee, S. Lilla, S. Zanivan, Vivienne Morrison, C. Nixon, Rachel A Ridgway, Crispin J Miller, John R. P. Knight, A. Campbell, P. Dunne, John Le Quesne, Joanne Edwards, Peter J. Park, Martin Bushell, Owen J. Sansom
          </td>
          <td>2025-12-18</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 MYC overexpression, often associated with chromosome 8q24 amplification, is a well identified genetic alteration in aggressive prostate cancer, especially in metastatic castration-resistant prostate cancer (mCRPC). It has been reported that MYC amplification counteracts with androgen receptor (AR) signaling, which has significant impacts on androgen deprivation therapy efficacy. In this study, we developed a prostate cancer mouse model in which MYC overexpression is initially induced by AR but AR-independent during tumorigenesis and progression. We employed the Multiome technology integrating single-cell RNA-sequencing (scRNA-seq) and ATAC-sequencing (scATAC-seq) to profile late-stage MYC-driven non-metastatic and metastatic prostate cancers. Consistent with previous studies using AR-dependent MYC-driven models, we identified that primary tumors are composed of a large quantity of luminal cells. We have also observed a higher proportion of luminal cells in the tumors that developed metastasis, while localized primary tumors were more abundant in basal cells, and transitional cell types. Interestingly, we uncovered heterogeneous transitional populations featured by different cell cycle profiles and cell-type gene signatures. With gene set enrichment analysis (GSEA), we identified differential enrichment of inflammatory pathways in luminal cells in metastatic primary tumors relative to localized primary tumors. By integrating with scATAC-seq analysis, we also identified new differential transcriptional and epigenetic regulators that may drive the aggressiveness of these MYC-driven cancers in a tissue specific manner. In addition, we applied inferred copy number variation (CNV) analysis and showed that aggressive tumors and metastasis harbor higher levels of CNVs. Together, these findings reveal how MYC overexpression reshapes prostate epithelial lineage, chromatin landscape and genomic instability to promote AR-independent aggressiveness, providing mechanistic insights and a foundation for future therapeutic targeting of MYC-driven malignancies.



 Kathryn Echandía-Monroe, Sofia Hu, Daniel R. Schmidt, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Faith Kim, Chloe Springer, Savannah Washburn, Madeline M. Wong, Kate Lu, Marianna Trakala, William D. Figg, Matthew G. Vander Heiden, Xiaofeng A. Su. Multiomic single-cell profiling of a novel MYC-driven mouse prostate cancer model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5bcbed11fc8b18cad2c80c67908e6beb4140ba8" target='_blank'>
              Abstract B017: Multiomic single-cell profiling of a novel MYC-driven mouse prostate cancer model
              </a>
            </td>
          <td>
            Kathryn Echandía-Monroe, Sofia Hu, Daniel R. Schmidt, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Faith Kim, Chloe Springer, Savannah Washburn, Madeline M. Wong, Kate Lu, Marianna Trakala, W. Figg, M. V. Vander Heiden, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596e7d27f8f208d19adb1081b3f37426d428f0a7" target='_blank'>
              PTEN deficiency linked to chromosome 10q loss leads to aggressive NF2 mutant meningioma biology
              </a>
            </td>
          <td>
            Abigail G. Parrish, H. N. Thirimanne, Sonali Arora, C. Herold-Mende, P. Sievers, F. Sahm, Frank Szulzewsky, Eric C. Holland
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In our study, we investigated the role of the chromatin remodeler ATRX in the progression of cutaneous melanoma. We analyzed ATRX protein expression in over 350 melanomas, correlating findings with clinical data, tumor proliferation rates, and vessel density. Additionally, we examined whole-genome sequencing data from 70 melanoma metastases to assess ATRX genetic alterations and compared these with protein expression patterns. We observed a significant reduction in ATRX protein expression in metastases compared to primary tumors: 51% of primary melanomas showed ATRX positivity in over 50% of tumor cells, versus only 25% of metastases (p = 0.01). ATRX loss was associated with earlier metastasis (median 17 vs. 46 months), reduced overall survival (median 69 vs. 162.5 months), and worse tumor-specific survival (p = 0.01). ATRX expression correlated positively with vessel density (rs = 0.3) and negatively with proliferation (rs = – 0.3), suggesting a role in hypoxia response. Genetically, intronic mutations were most frequent (80%), followed by copy number variations (loss: 29%, gain: 37%). Interestingly, ATRX copy number changes did not correlate with protein levels, pointing to epigenetic regulation. Our findings highlight ATRX loss as an early and prognostically relevant event in melanoma, with potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31fd0bb9ff3e922880a552c9ee8dacf9129ecfc2" target='_blank'>
              The chromatin guardian ATRX is a strong prognostic biomarker in melanoma
              </a>
            </td>
          <td>
            Céline Arlette Frei, Cassandra Litchfield, Alanna Mihic, Fabiola Prutek, André Fitsche, Fabius Wiesmann, Miriam Wanner, Bettina Sobottka, D. Mihic-Probst, , Rudolf Aebersold, Melike Ak, Faisal S. Al-Quaddoomi, Silvana I. Albert, Jonas B. Albinus, I. Alborelli, S. Andani, Per-Olof Attinger, M. Bacac, Monica-Andreea Baciu-Drăgan, Daniel Baumhoer, Beatrice Beck-Schimmer, N. Beerenwinkel, C. Beisel, Lara Bernasconi, A. Bertolini, Bernd Bodenmiller, Ximena Bonilla, L. Bosshard, Byron Calgua, Ruben Casanova, Stéphane Chevrier, N. Chicherova, R. Coelho, Maya D’Costa, E. Danenberg, Natalie R. Davidson, Reinhard Dummer, S. Engler, Martin Erkens, Katja Eschbach, Cinzia Esposito, A. Fedier, Pedro F. Ferreira, Joanna Ficek-Pascual, Anja L. Frei, Bruno Frey, Sandra Goetze, Linda Grob, Gabriele Gut, Detlef Günther, P. Haeuptle, V. Heinzelmann-Schwarz, S. Herter, René Holtackers, Tamara Huesser, A. Immer, A. Irmisch, Francis Jacob, Andrea Jacobs, T. Jaeger, A. R. James, P. Jermann, A. Kahles, Abdullah Kahraman, V. H. Koelzer, Werner Kuebler, Jack Kuipers, C. P. Kunze, C. Kurzeder, K. Lehmann, Mitchell P. Levesque, Ulrike Lischetti, Flavio C. Lombardo, Sebastian Lugert, Gerd Maass, Markus G. Manz, Philipp Markolin, Martin Mehnert, J. Mena, Julian M. Metzler, Nicola Miglino, Emanuela S. Milani, Holger Moch, Simone Muenst, Riccardo Murri, C. K. Y. Ng, S. Nicolet, Marta Nowak, Mónica Núñez López, Patrick G. A. Pedrioli, Lucas Pelkmans, Salvatore Piscuoglio, Michael Prummer, Laurie Prélot, N. Rimmer, M. Ritter, Christian Rommel, María L. Rosano-Gonzalez, Gunnar Rätsch, N. Santacroce, Jacobo Sarabia del Castillo, R. Schlenker, Petra C. Schwalie, Severin Schwan, Tobias Schär, Gabriela Senti, Wenguang Shao, Franziska Singer, Sujana Sivapatham, B. Snijder, Bettina Sobottka, Vipin T. Sreedharan, Stefan Stark, D. Stekhoven, Tanmay Tanna, A. Theocharides, Tinu M. Thomas, Markus Tolnay, Vinko Tosevski, Nora C. Toussaint, M. Tuncel, Marina Tusup, A. van Drogen, Marcus Vetter, Tatjana Vlajnic, Sandra Weber, Walter P. Weber, R. Wegmann, Michael Weller, Fabian Wendt, Norbert Wey, Andreas Wicki, M. Wildschut, Bernd Wollscheid, Shuqing Yu, Johanna Ziegler, M. Zimmermann, M. Zoche, G. Zuend
          </td>
          <td>2025-12-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Background The Golden Mussel, Limnoperna fortunei, represents one of the most successful aquatic invaders in South America, causing significant ecological and economic impacts. Repetitive DNA sequences, particularly satellite DNAs (satDNAs), play crucial roles in genome architecture and evolution, yet the satDNA landscape of this invasive species remains largely uncharacterized. This study aimed to comprehensively analyze the satellitome of L. fortunei using integrated computational and cytogenetic approaches. Methods We employed a read-clustering approach (RepeatExplorer2) to identify satDNA families from short-read genomic data. The identified families were then mapped onto the chromosome-level reference genome in silico to determine their chromosomal distribution. The physical localization of two representative satDNA families with contrasting distribution patterns was validated through fluorescence in situ hybridization (FISH) on meiotic chromosomes. Results Our analysis identified 129 distinct satDNA families, which were estimated to comprise approximately 9.1% of the genome based on read clustering. Subsequent in silico mapping successfully localized 126 of these families to the reference genome, where they collectively represented approximately 5.3% of the assembled chromosomes. Most families showed low divergence levels (0%–5% Kimura substitutions), suggesting recent amplification events. While most satDNAs were distributed across all 15 chromosomes, FISH analysis of two distinct families revealed contrasting distribution patterns: LfoSat025 showed preferential localization to pericentromeric regions, while LfoSat004 exhibited localized enrichment in specific chromosomal regions, demonstrating diverse organizational strategies within the satellitome. Meiotic analysis revealed normal chromosome pairing (15 bivalents), with no evidence of differentiated sex chromosomes, consistent with the species’ predominantly gonochoristic reproductive mode. Conclusion This study provides the first comprehensive characterization of the L. fortunei satellitome, revealing a diverse and recently amplified repetitive landscape. The discrepancy between the genome-wide abundance (9.1%) and the mapped abundance (5.3%) highlights the challenges of assembling repetitive regions and underscores the value of using complementary methodologies. The identification of chromosome-specific satDNA markers establishes a foundation for developing molecular tools for invasion monitoring, including population genetic analyses to trace invasion routes and identify source populations. These findings contribute to understanding the role of repetitive DNA in genome evolution and the adaptive success of invasive species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecafaaf22baf704516b16401c19167437d585cf" target='_blank'>
              A diverse satellite DNA repertoire in Limnoperna fortunei: insights into genome structure and chromosomal organization
              </a>
            </td>
          <td>
            J. P. Castro, T. M. Degrandi, Allan Oliveira Leal, Augusto Luiz Ferreira Júnior, M. C. de Almeida, R. Artoni
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Circular DNA molecules—mitochondrial DNA (mtDNA) and extrachromosomal circular DNA (ecDNA)—are fundamental genetic elements whose structural integrity is essential for cellular homeostasis. Pathological alterations in their length, caused by deletions or duplications, are directly implicated in a broad spectrum of human diseases, from inherited mitochondrial disorders to aggressive cancers. This review provides a comprehensive methodological guide for detecting these specific structural changes. We systematically evaluate classical techniques (long-range PCR, Southern blotting, electron microscopy) that offer direct visualization and validation, and modern high-throughput approaches (short- and long-read sequencing) that enable genome-wide discovery. Special emphasis is placed on quantitative clinical tools like digital droplet PCR (ddPCR) for ultrasensitive monitoring. By presenting a comparative framework, we guide the selection of optimal methods based on resolution, throughput, and clinical applicability. We conclude that an integrated, multi-method strategy is indispensable for robust analysis, positioning the precise detection of circular DNA length alterations as a cornerstone of emerging precision diagnostics and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9367ea411656b4af8c5e9c55df093cb7809d8cc8" target='_blank'>
              The Circular DNA Size Code
              </a>
            </td>
          <td>
            Zurab Abashidze
          </td>
          <td>2026-01-31</td>
          <td>Longevity Horizon</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Methyl CpG Binding Protein 2 (MECP2) gene has copy number gain in a number of human cancers and functions as an oncogene through an unusual epigenetic mechanism. We explored the possibility that MECP2 might be a therapeutic target, and whether its epigenetic mode of action is amenable to specific therapy. Constitutively expressed or inducible lentiviral short hairpin RNA (shRNA) directed at MECP2 in human triple negative breast cancer (TNBC) cell lines with high or low level of MECP2 protein were grown as xenografts to assess oncogene addiction. We next evaluated the effect of DNA methylation inhibitors or histone deacetylase inhibitors on monolayer or soft agar growth of isogenic human mammary epithelial cells with or without MECP2 overexpression, and xenograft growth of MECP2-dependent or MECP2-independent human TNBC or lung cancer cell lines. We then investigated the mechanism of MECP2-induced activation of MAPK pathway and assessed the effect of drug treatment. Human cancer cell lines with MECP2 overexpression show MECP2 dependence, and epigenetic drugs are effective in these models. Activated RAS and other activators of the MAPK pathway caused resistance to these therapies, giving insight into their novel mode of action and demonstrating specificity. The kinase PAK3 is important for MECP2-mediated induction of MAPK pathway, is modulated by epigenetic drugs affecting MECP2 action as expected and may itself be a therapeutic target for MECP2-driven cancers. These preclinical studies show that tumors overexpressing MECP2 might benefit from epigenetic therapy targeting MECP2 function and demonstrate a novel mechanism of action for these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4802b6674f8b3e0a4e440436dd1fc69dc98c09c" target='_blank'>
              MECP2-Dependent Cancers Can Be Targeted by Epigenetic Drugs: A New Role for Epigenetic Cancer Therapy.
              </a>
            </td>
          <td>
            Manish Neupane, Mukesh Kumar, Malela A Mwamufiya, Colleen M Donnelly, Alsu Ibragimova, Brian Curcio, Inna Chervoneva, Irina Vasilevskaya, Daniel P Silver
          </td>
          <td>2025-12-11</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Receptor tyrosine kinase (RTK) fusions are a large class of oncoproteins found in ∼5% of cancers. Key questions remain, however, about how RTK fusions transmit oncogenic signals, including how these largely cytoplasmic proteins activate downstream pathways that originate at the plasma membrane. Fusions are multimeric and can form mesoscale condensates in cancer cells, and condensation has been implicated as an essential mechanism to enable signal transmission from the cytoplasm. However, whether condensates play a causal role, or whether smaller ‘diffuse’ assemblies are sufficient to transduce signals, has been challenging to establish. Here we apply advanced microscopy, single-cell analysis, and synthetic fusions to determine the principles by which multimerization and condensation drive signaling from cytoplasmic RTK fusions. For EML4-ALK, a prominent fusion that forms condensates, we found poor correlation between condensation and signaling. By contrast, EML4-ALK activity was abundant in the diffuse phase, and the kinetics of diffuse-phase activity aligned more closely with downstream Erk signaling than did kinetics of signaling within condensates. Synthetic RTK fusions showed that cytoplasmic ALK or RET fusion dimers—and even constitutively active monomers—were sufficient to induce strong Ras-Erk signaling despite the absence of condensates, and diffuse fusions were sufficient to transform cells in vitro and in subcutaneous tumor models. A panel of various other cancer-driving RTK fusions showed that low-order multimerization was universal across fusions, whereas mesoscale condensation was rare and did not correlate with signaling. Our results suggest that low-order fusion multimerization is sufficient to drive its phosphorylation, which is necessary and sufficient to trigger downstream oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6563f441f99dac16cec7dceb7a7c5f130e146e" target='_blank'>
              Low-order assemblies drive oncogenic RTK fusion signaling without condensation
              </a>
            </td>
          <td>
            D. Gonzalez-Martinez, Thomas R. Mumford, Delaney Wilde, Sofia Wissert, Y. Gao, Emily Brackhahn, R. Kriwacki, Elizabeth Rhoades, Lukasz J. Bugaj
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein (NAP) that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules, leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. We further validate our findings by carrying out in vivo imaging of endogenously expressing Lsr2 tagged with eGFP in Mtb cells. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA; our findings rather suggest that protein–DNA co-condensates “sense” the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gmycene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fe9c6575903dcbbc1b5c771711d6f1e938d3c5" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Prakshi Gaur, Thejas Satheesh, R. Singh, Hussain Beig, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Ayesha Dash, Amit Singh, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0dd4ab62e8dce20ab891fcaeb464ced704c0754" target='_blank'>
              PKMYT1 inhibition induces DNA damage and synergizes with immune checkpoint blockade in CCNE1-amplified gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            S. Jang, Lawrence W Wu, Jimyung Park, Sho Hangai, Cameron Shapiro, C. G. Marshall, Luke L. Cai, Benjamin Izar, Adam J. Bass, Ryan H. Moy
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2121fa65dd71d988060d54e96a929aab0104fdc6" target='_blank'>
              Targeting ALC1 can safely expand the therapeutic utility of PARP inhibitors across high-grade serous ovarian cancers
              </a>
            </td>
          <td>
            Lindsey N. Aubuchon, Deanna H. Wong, Natasha M. Ramakrishnan, Rohan Reddy, Amanda J Compadre, Elena Lomonosova, Kathleen E. Jackson, Danielle Kemper, Vrutti V. Mehta, Kellen Zoberi, D. Khabele, Elizabeth L. Christie, Mary M Mullen, Priyanka Verma
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Eukaryotic genomes contain numerous transposable elements (TEs), whose dysregulation threatens genome stability and may contribute to cancer. Pancreatic adenocarcinoma (PAAD) is among the deadliest cancers, marked by abundant stroma that obscures tumor-specific molecular signals, complicating bulk-tissue analyses. Here, using 71 patient-derived PAAD organoids, we show that TE activities may potentially promote tumorigenesis and provide a source of novel immunotherapeutic targets. We identify 16 new TE-derived transcripts fused with 15 known oncogenes, exhibiting potential oncogenic function and prognostic value. Notably, LTR7-PLAAT4, present in 29% of tumors, encodes a protein variant transcriptionally regulated by FOXM1 binding to the LTR7 promoter. LTR7-PLAAT4 isoform 2 is associated with increased cholesterol ester accumulation and lipid droplet formation mediated through BSCL2 coexpression, potentially fostering tumor progression. On the immunogenic front, HLA-I immunopeptidomics of AsPC-1 cells and DAC13 organoids identify over 11,000 peptides respectively. Althought mutation-derived neoantigens are rare, several peptides are originated from TE-chimeric transcripts, including four predicted by TEprof2. The peptide FLIQHLPLV, detected in 27% of organoids, exhibits robust immunogenicity, validated by T2 binding, mass spectrometry and ELISPOT assays with HLA-genotyped PBMCs. Together, these findings suggest that TE activities may contribute to PAAD progression and diversify its immunopeptidome, providing new opportunities for molecular subtyping and potential immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, G. Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c961a3f1619198d178054a670088d76568b3783" target='_blank'>
              Friend and Foe: Genome-Wide Analysis of the Tardigrade Dsup Protein Expressed in Yeast Reveals Trade-offs of DNA Protection
              </a>
            </td>
          <td>
            H. Gaikani, Marjan Barazandeh, Lewis Hitchens, G. Giaever, Corey Nislow
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract DNA methylation (DNAm) is widely used to leverage biological information in the context of cancer. Active regulatory elements marked by increased chromatin accessibility and specific transcription factor (TF) binding, such as enhancers, show tumor-specific DNAm patterns reflecting the biological forces shaping tumorigenesis. However, DNAm changes also occur at regions that are inactive at histone-mark level in fully developed tumors. Beyond hypermethylation of promoters, these changes are often overlooked, leaving their functional relevance poorly understood. By analyzing DNAm and chromatin state annotations from conventional mantle cell lymphoma (cMCL), we identified a subset of ~300 CpGs with homogeneous cMCL-specific demethylation patterns located in cMCL-inactive regions. We show that these regions contain cMCL-related TF motifs and are flanked by genes with cMCL-specific expression patterns, suggesting a potential regulatory role in lymphomagenesis. We hypothesize that these regions represent DNA demethylation imprints of genome activation prior to full-blown tumor formation, either present in its cell type of origin (COO) or during early stages of tumor formation. Altogether, we put forward a new, DNAm-centered approach to gain insights into potential early tumorigenic events, allowing us to generate novel, further explorable hypotheses to better understand the features playing a role in lymphomagenesis and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38a0fc34737aa002d9118359a92f24cc2e094aa9" target='_blank'>
              DNA methylation profiles aid to identify putative genome activation histories along lymphomagenesis
              </a>
            </td>
          <td>
            Leone Albinati, Irene D’Onofrio, Rabia Gül Aydin, Houyem Toukabri, Renée Beekman
          </td>
          <td>2026-01-10</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc8f584519863394cabad2d307177d0a23fce8d" target='_blank'>
              An EGR1-dependent cascade modulates genome architecture at the CSF1R locus.
              </a>
            </td>
          <td>
            Sandra Deliard, Elisa Barbieri, Marco Trizzino, Kelsey A. Leach, Avery J. Zucco, Francis Picone, Filippo Veglia, Alessandro Gardini
          </td>
          <td>2026-01-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Dps is the most abundant nucleoid-associated protein in starved Escherichia coli with ∼180 000 copies per cell. Dps binds DNA and oxidizes iron, facilitating survival in harsh environments. Dps–DNA complexes can form crystalline structures, leading to the proposed model that Dps reorganizes the starved E. coli nucleoid into a compact liquid crystal, slowing chromosome dynamics, and limiting access of other proteins to DNA. In this work, we directly tested this model using live-cell super-resolution microscopy and Hi-C analysis. We found that after 96 h of starvation, Dps compacts the nucleoid, and increases short-range DNA–DNA interactions but does not affect chromosome accessibility to large protein nanocages or small restriction enzymes. We also report that chromosome dynamics and organization are primarily impacted by the bacterial growth phase; the effect of Dps is relatively minor. Our work clarifies the role of Dps in modulating nucleoid properties, and we propose an updated model for Dps–DNA interactions in which Dps binds, protects, and compacts DNA largely without influencing chromosome access, dynamics, and organization. Additionally, this work provides a general framework for assessing the impact of nucleoid-associated proteins on key aspects of chromosome function in live cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33238ba5564d2cc1675f4271e212f02cf6fdbd2b" target='_blank'>
              Dps binds and protects DNA in starved Escherichia coli with minimal effect on chromosome accessibility, dynamics, and organization
              </a>
            </td>
          <td>
            L. A. McCarthy, Lindsey E. Way, Xiaofeng Dai, Zhongqing Ren, D. E. Fuller, Ishika Dhiman, Linaria Larkin, Jurriaan J. D. Sieben, Ilja Westerlaken, E. Abbondanzieri, Gail G. Hardy, A. S. Meyer, Xindan Wang, Julie S. Biteen
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Optical genome mapping (OGM) detects genome-wide structural variants (SVs), including balanced rearrangements and complex copy-number alterations beyond standard-of-care cytogenomic assays. In chronic lymphocytic leukemia (CLL), cytogenetic and genomic risk stratification is traditionally based on fluorescence in situ hybridization (FISH), karyotyping, targeted next-generation sequencing (NGS), and immunogenetic assessment of immunoglobulin heavy chain variable region (IGHV) somatic hypermutation status, each of which interrogates only a limited aspect of disease biology. Methods: We retrospectively evaluated fifty patients with CLL using OGM and integrated these findings with cytogenomics, targeted NGS, IGHV mutational status, and clinical time-to-first-treatment (TTFT) data. Structural variants were detected using OGM and pathogenic NGS variants were derived from a clinical heme malignancy panel. Clinical outcomes were extracted from the electronic medical record. Results: OGM identified reportable structural variants in 82% (41/50) of cases. The most frequent abnormality was del(13q), observed in 29/50 (58%) and comprising 73% (29/40) of all OGM-detected deletions with pathologic significance. Among these, 12/29 (42%) represented large RB1-spanning deletions, while 17/29 (58%) were focal deletions restricted to the miR15a/miR16-1 minimal region, mapping to the non-coding host gene DLEU2. Co-occurrence of adverse lesions, including deletion 11q/ATM, BIRC3 loss, trisomy 12, and deletion 17p/TP53, were recurrent and strongly associated with shorter TTFT. OGM also uncovered multiple cryptic rearrangements involving chromosomal loci that are not represented in the canonical CLL FISH probe panel, including IGL::CCND1, IGH::BCL2, IGH::BCL11A, IGH::BCL3, and multi-chromosomal copy-number complexity. IGHV data were available in 37/50 (74%) of patients; IGHV-unmutated status frequently co-segregated with OGM-defined high-risk profiles (del(11q), del(17p), trisomy 12 with secondary hits, and complex genomes whereas mutated IGHV predominated in OGM-negative or structurally simple del(13q) cases and aligned with indolent TTFT. Integration of OGM with NGS further improved genomic risk classification, particularly in cases with discordant or inconclusive routine testing. Conclusions: OGM provides a comprehensive, genome-wide view of structural variation in CLL, resolving deletion architecture, identifying cryptic translocations, and defining complex multi-hit genomic profiles that tracked closely with clinical behavior. Combining OGM and NGS analysis refined risk stratification beyond standard FISH panels and supports more precise, individualized management strategies in CLL. Prospective studies are warranted to evaluate the clinical utility of OGM-guided genomic profiling in contemporary treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28f49f64c743393cb2116dc5e9531fab4be082e7" target='_blank'>
              Optical Genome Mapping Enhances Structural Variant Detection and Refines Risk Stratification in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            S. Chakraborty, Michelle A. Bickford, Narcisa A. Smuliac, Kyle A. Tonseth, Jing Bao, Farzana Murad, Irma G. Domínguez Vigil, Heather B. Steinmetz, Lauren M. Wainman, Parth Shah, Elizabeth M. Bengtson, Swaroopa PonnamReddy, Gabriella A. Harmon, Liam L. Donnelly, L. Tafe, Jeremiah X. Karrs, Prabhjot Kaur, Wahab A. Khan
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Introduction The E3 ubiquitin ligase Pellino1 (PELI1) is ubiquitously expressed in human tissues and primarily modulates inflammatory and immune responses; however, its pan-cancer biological significance remains poorly characterized. Methods We employed a combination of R software and online bioinformatics platforms—including UALCAN, HPA, GEPIA2, cBioPortal, STRING, TISIDB, SRAMP, LinkedOmics, and Sangerbox—to systematically characterize PELI1 in human tumors. Our analyses encompassed its abnormal expression, genetic alterations, prognostic and diagnostic relevance, and epigenetic regulation. Focusing on liver hepatocellular carcinoma (LIHC), we further explored the oncogenic functions of PELI1, its associated signaling pathways, and immunomodulatory roles. Key bioinformatic predictions were subsequently validated through in vitro experiments using LIHC cell lines, including functional assays (proliferation, apoptosis, and cell cycle) and signaling pathway analyses. Results PELI1 was frequently overexpressed across multiple tumor types and mainly correlated with poor patient prognosis. The highest frequency of PELI1 alterations was observed in patients with diffuse large B-cell lymphoma, primarily in the form of copy number amplification, while R145Q/W represented the most recurrent mutation sites. PELI1 may drives cancer progression through epigenetic regulation involving DNA methylation and N6-methyladenosine (m6A) RNA modifications. PELI1 could potentially serve as a novel biomarker for diagnosing different cancer types. In LIHC specifically, PELI1 expression was significantly elevated compared to adjacent non-tumor tissues. Functional studies revealed that PELI1 modulated cell apoptosis, cell cycle progression, immune responses and the MAPK-ERK pathway in LIHC. Furthermore, PELI1 may regulate immune checkpoints to influence the immune responses. PELI1 expression was positively correlated with the IC50 of hepatocellular carcinoma investigational drug–CFI-402257. Experimental knockdown of PELI1 in LIHC cell lines suppressed proliferation, promoted apoptosis and induced G1-phase cell cycle arrest via MAPK-ERK pathway. Conclusions This pan-cancer study revealed the diagnostic, prognostic and immunomodulatory potential, along with the biological function and mechanism of PELI1, supporting its role as a promising therapeutic target in LIHC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b6e461ae85a0f26d5308987db714f7774b3c5b5" target='_blank'>
              PELI1 in human cancers: a pan-cancer exploration of its molecular function, clinical significance, and immunomodulatory roles
              </a>
            </td>
          <td>
            Yan Xu, Jiale Zhou, Xiaoran Chen, Xiaoqing Dong, Bing Chen
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A large number of extrachromosomal circular DNAs (eccDNAs) are found in eukaryotic cells, but the mechanism behind the formation of eccDNAs remains unknown. EccDNAs smaller than 1000 bp are commonly referred to as microDNAs. In a prior study, we identified a 542 bp microDNA, designated eccDNAfib-L, mapping to Chromosome 14: 9,692,083-9,692,624 nt in the silk gland of Bombyx mori. There is a direct short repeat “GAGT” at both 5′ and 3′ break points of eccDNAfib-L, but only one copy of “GAGT” is retained in eccDNAfib-L. Here, we find that the specific junction observed can also be detected by transfecting with a DNA fragment containing upstream sequences of the 5′ break point and downstream sequences of the 3′ break point of eccDNAfib-L. Additionally, the length of the flanking sequences and the direct short repeats “GAGT” affect the formation efficiency of eccDNAfib-L. We confirm that eccDNAfib-L formation is associated with DNA repair pathways, with the expression of eccDNAfib-L undergoing significant alterations following the silencing of genes related to DNA repair pathways. In vitro, a cell-free reaction system confirms that Polθ and the direct short repeat “GAGT” are essential for the circularization of eccDNAfib-L, and the DExH-box helicase domain of Polθ plays critical role in mediating this circularization process. Collectively, our data support a mechanism whereby Polθ can mediate the joining of linear DNA fragments with direct short repeat at the 5′ and 3′ ends to form a circular DNA through MMEJ. The results not only clarify the essential factors driving eccDNAfib-L formation but also complement previous reports on eccDNA formation. DNA repair pathways are involved in eccDNAfib-L formation. Specifically, Polθ’s DExH-box helicase domain and direct short repeat “GAGT” are essential for its MMEJ-mediated circularization, delineating key factors driving this microDNA’s biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Genomic copy number alterations cause aneuploidy, a hallmark of cancer characterized by imbalanced chromosome numbers. Aneuploidy has been shown to negatively associate with cancer immune signaling, but these mechanisms remain understudied. Of note, high levels of aneuploidy are strongly correlated with lethal progression and poor patient outcomes in prostate cancer. Our recent work identified chromosome 8q (chr8q) gain, one of the most frequent aneuploidy events in prostate tumors, as a key driver of disease progression, in part through the cohesin RAD21 gene located on the chr8q24 region. In both prostate cell line and organoid models, we demonstrated that increased RAD21 expression accelerates oncogenesis and aggressive tumor proliferation by mitigating cellular stress during early-stage tumorigenesis. Therefore, we hypothesize that elevated RAD21 levels enable cancer cells to evade immune surveillance by alleviating oncogenic stress and DNA damage. In this study, we analyzed publicly available genomic datasets and discovered that higher RAD21 expression negatively correlated with natural killer (NK) cell infiltration in prostate tumors. To further test our hypothesis, we modulated RAD21 levels in isogenic prostate cell culture models. Using an NK cell co-culture assay, we observed that cells with increased RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited greater survivorship when co-cultured with NK cells. In addition, we found that cellular levels of various pro-inflammatory cytokines, including those that stimulate NK cell activation, were significantly downregulated when RAD21 was overexpressed. These findings are consistent with our observation that tumors with high RAD21 levels are linked to reduced NK cell infiltration in prostate cancer cases. Our results suggest that amplification of RAD21, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Targeting RAD21-driven immune evasion may reveal novel therapeutic strategies to restore immune surveillance mechanisms in prostate cancer.



 Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R. Lozada, Christine Luo, Andrew Elliott4, Nicholas A. Zorko3, Justin H. Hwang3, David Takeda5, Xiaofeng A. Su. Elevated RAD21 levels promote immune evasion in prostatic malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A032.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba3556a59b07de1fb3550ddaa2e2a0eb7e783af" target='_blank'>
              Abstract A032: Elevated
 RAD21
 levels promote immune evasion in prostatic malignancies
              </a>
            </td>
          <td>
            Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R Lozada, Christine Luo, A. Elliott, Nicholas A. Zorko, Justin H. Hwang, David Takeda, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90832a3d9b2f9576b36552899274c52bd58fcd98" target='_blank'>
              RBM8A confers oxaliplatin resistance in gastric cancer by maintaining EGFR mRNA stability
              </a>
            </td>
          <td>
            Yichun Ma, Ying Xiang, Mei Yang, Yanmei Zhu, Yani Pan, Nannan Zhang, Xueni Fu, Ping Jiang, Yinya Pan, Hai Wu, Qiong Yan, Wenjun Li, Hongji Tao, Shangtao Mao, Yufei Tao, Jiale Li, Linzhe Su, Guangtao Gao, Fangmei An, Zhangding Wang, Guifang Xu
          </td>
          <td>2025-12-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbcb5b4a0b3273929b660d10aee3627b91d407c6" target='_blank'>
              Periplasmic SacB as a robust counter-selection tool for genome engineering in the polyploid bacterium Zymomonas mobilis
              </a>
            </td>
          <td>
            Katsuya Fuchino, Christodoulos Astraios, Richard A Daniel, James Grimshaw, Manuel Banzhaf, Waldemar Vollmer
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 The minor spliceosome (MiS) is a specialized RNA-processing machinery upregulated in cancer to promote expression of oncogenic genes. Here, we identify its catalytic component, U6atac snRNA, as a druggable vulnerability in prostate and breast cancers. U6atac knockdown triggers R-loop–mediated DNA damage while impairing DNA repair by downregulating key factors such as BRCA1, PARP1, TP53BP1, and CHK1/2, disabling both homologous recombination and non-homologous end joining. This dual effect sensitizes tumors to PARP inhibitors, cisplatin, and radiation, independent of BRCA status. Moreover, we uncover an adaptive resistance mechanism driven by extracellular vesicles enriched in U6atac, which amplify MiS activity and facilitate therapy escape; a process reversed by U6atac depletion. Across multiple in vitro and in vivo models, MiS targeting demonstrates tumor-selective activity with minimal toxicity. These findings position U6atac as a central regulator of genome stability and establish MiS targeting as a promising approach to potentiate genotoxic therapy and overcome resistance.



 Anke Katharina. Augspach, Mark Rubin, Paola Francica, Sven Rottenberg, Rahul Kanadia. Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c71eaaa6cc7e28ae4d142a474ab9e0350163c6b" target='_blank'>
              Abstract B003: Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, Mark A. Rubin, P. Francica, Sven Rottenberg, Rahul Kanadia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da2aeba8554dfbf99a705198a393f82144d0d18d" target='_blank'>
              Menin maintains enhancer-promoter interactions in a leukemia-specific manner
              </a>
            </td>
          <td>
            Vassilena Sharlandjieva, Catherine Chahrour, F. Lassen, Joseph C. Hamley, Andreas Damianou, N. Denny, Alastair L. Smith, Svenja S Hester, I. Vendrell, Ronald W. Stam, Marina Konopleva, A. Roy, J. O. Davies, Nicholas T. Crump, Benedikt M. Kessler, Thomas A. Milne
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="MYC genomic amplification and pathway activation is associated with aggressive behavior and poor prognosis in osteosarcoma (OS). However, a discordance exists between copy-number gains and transcriptional output from MYC in OS, and defining these mechanisms is critical to understand and intercept persistent MYC signaling. Here, we showed that cytoplasmic mRNA (poly(A)) sustains MYC activation in OS. Multi-omics profiling and single-cell transcriptomics identified TENT5A, a non-canonical RNA-binding poly(A) polymerase, as selectively upregulated in MYC-activated tumors and enriched in proliferative, stem-like populations. Biochemical and genetic evidence demonstrated that TENT5A directly bound MYC mRNA via its PAP/OAS1 domain, extended its poly(A) tail, and stabilized the transcript, thereby reinforcing MYC-driven stemness and chemoresistance. Gain- and loss-of-function assays, orthotopic xenografts, and patient-derived organoids confirmed that elevated TENT5A enhanced tumor-initiating capacity and reduced chemotherapy sensitivity. Pharmacologic inhibition of TENT5A disrupted MYC mRNA stabilization, shortened poly(A) tails, and reversed chemoresistance in preclinical models. These findings delineate a post-transcriptional RNA-stabilization pathway that reconciles the disconnect between MYC genetic alterations and transcriptional activity and nominate the RNA-binding protein TENT5A as a therapeutically tractable target in OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d515c1b7f0b534d63c1002ea77732498dd44f1" target='_blank'>
              TENT5A Maintains MYC mRNA Stability to Enhance Osteosarcoma Stemness.
              </a>
            </td>
          <td>
            Yining Tao, Qi Zhang, Haoyu Wang, Binghui Yang, Haoran Mu, Kaiyuan Liu, Weisong Zhao, Xiyu Yang, Bowen Zhao, D. Zuo, Liu Yang, Zhengdong Cai, Zongyi Wang, Hongsheng Wang, Yingqi Hua, Wei Sun
          </td>
          <td>2026-01-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Significance Our studies show the levels of type I topoisomerases are increased in human papillomaviruses (HPV)-positive cells, leading to enhanced levels of DNA breaks and activation of DNA repair pathways. Furthermore, knockdown of either TOP1α or TOP3β impaired viral replication and transcription. Importantly, these two topoisomerases act by distinct mechanisms to regulate viral functions. Elucidating the mechanisms by which topoisomerases regulate viral gene expression and replication offers a promising avenue for targeted therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/843f6cbaaa32b6f9a8d2bc8fb897a0a89ef42241" target='_blank'>
              Differential roles of type I topoisomerases in regulating HPV pathogenesis
              </a>
            </td>
          <td>
            A. Vats, Conor W Templeton, Lou Laimins
          </td>
          <td>2026-01-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The total DNA of an organism is called the genome. DNA contains all the instructions for how to build an organism and how that organism will function. Some DNA sequences are repeated thousands and thousands of times throughout the genome. Repeats lined up one after another are called satellite DNA. The number of copies of a given satellite DNA sequence can rapidly change, and differ among individuals. Although satellite DNAs were once considered useless, researchers are continuously discovering their important roles in various organisms. Satellite DNAs are essential for keeping organisms functioning properly. They help cells divide and keep the genome integrity. They can affect behavior, health, and help the organism to overcome stressful conditions. In these ways, satellite DNAs increase Earth’s biodiversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e93be9ccc179d27f89aeee0b54094657efcfefe" target='_blank'>
              Mysteries of Repetitive DNA Segments
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Miroslav Plohl
          </td>
          <td>2025-12-29</td>
          <td>Frontiers for Young Minds</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Most deaths from cancer are caused by their metastases. Along with the ability to evade the immune system and to invade and disrupt very different tissue environments, metastases usually develop resistance to whatever therapeutic approaches are tried. Their extreme adaptability requires a diversity of traits from which natural selection can choose, but current models, such as the epithelial-to-mesenchymal transition (EMT), do not directly address how this diversity is generated. We observed that single cell clones of Panc1 human pancreas cancer cells that had had crispR knock outs (KO) of the gene for activin-like kinase 4 (ALK4) had developed a markedly diverse morphology. Time-lapse and fluorescence microscopy, FACS analysis and mitotic chromosome squashes provided evidence that the cells diversify profoundly in size, behavior and chromosome number. This diversity appears to develop through cytokinesis failure, the generation of very large multinuclear cells that exhibit a high range of nuclear configurations, coupled with continued cell divisions that sometimes generate three, four, and more daughter cells in one cell division. This astounding activity often resulted in cells that were much smaller than the normal Panc1 cells, and indirect evidence suggests they were significantly sub-ploidy, yet underwent regular cell divisions. A subset of the smaller cells were highly motile, and were observed to sometimes merge together or even into larger cells. This phenomenon could provide an unappreciated venue for transferring mutated genes, chromosome fragments, whole chromosomes or sets of chromosomes into an invaded cell. We suggest that these observations may be a serendipitous illustration of the missing link between the genetic mutations implicated in the development of primary carcinoma, and the extreme genomic diversity that characterizes malignant metastases. Graphical Summary Single cell clones of Panc 1 ALK4 KO cultures changed from largely uniformly epithelioid in character (A) to cultures composed of an unexpectedly wide variety of cells, and that this variety is not explained by the traditional “EMT” (epithelial-to-mesenchymal-transition). These cells range from very small to very large, have varying degrees of motility, chromosome number, and cell division behavior. We describe the new morphologies and behaviors, and present time lapse movies and microscopic evidence that help explain the evolution of this variety. These new cell types integrate into colonies where the smallest cells gather into spherical masses (D). We believe that the heterogeneity in genomic content and behavior provides the variety needed for natural selection to select cells that are resistant to most treatments for metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6b16627321b67c2cf57a151fe0aa1bc2c09235" target='_blank'>
              A Model for the Diversity Explosion Fundamental to Metastasis
              </a>
            </td>
          <td>
            Edward Tim O’Brien
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs), a unique class of endogenous noncoding RNAs characterized by their covalently closed loop structure, have emerged as pivotal players in cancer biology due to their exceptional stability, tissue‐specific expression, and diverse regulatory functions. This comprehensive review delves into the multifaceted roles of circRNAs as epigenetic modulators in cancer pathogenesis, systematically exploring their biogenesis via back‐splicing—regulated by
 cis
 ‐elements and
 trans
 ‐factors—and their general functions, including directly recruiting chromatin‐modifying enzymes or DNA methyltransferases or by sequestering noncoding RNAs that feed back to the chromatin, thereby establishing heritable and DNA sequence–invariant expression states. A distinctive feature of this work is its focused examination of how circRNAs interface with key epigenetic regulators, such as histone modifiers, DNA methyltransferases, and chromatin remodeling complexes, thereby influencing gene expression and driving tumor initiation, metastasis, and drug resistance. The review further integrates recent insights from single‐cell sequencing and discusses stoichiometric and structural nuances that define authentic miRNA sponge functions, setting it apart from previous summaries. By synthesizing current evidence and highlighting both mechanistic depth and clinical relevance—such as the potential of circRNAs as biomarkers and therapeutic targets—this article not only advances our understanding of circRNA‐mediated epigenetic dysregulation in cancer but also outlines future research directions and technological challenges, underscoring its significance in bridging basic molecular insights with translational oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df145e1c14bbc9d33031580b823f8b55632795bd" target='_blank'>
              Circular RNAs: Epigenetic Puppeteers Pulling the Strings of Cancer Pathogenesis
              </a>
            </td>
          <td>
            Yimao Wu, Zichang Chen, Junying Lee, Gökhan Zengin, Mengyao Li
          </td>
          <td>2026-01-14</td>
          <td>Medicine Bulletin</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/422a810751b71895a76ea9dd11ff1a3be414db1d" target='_blank'>
              The FANCM-RMI1/2 complex promotes genomic instability and PARP inhibitor sensitivity in BRCA2-deficicient cells
              </a>
            </td>
          <td>
            François Dossin, Mikael Attia, Fabricio Montero Zúñiga, Daniel González Acosta, R. Brough, Fei Fei Song, Aurélien Bore, Raphaël Margueron, Massimo Lopes, C. J. Lord, L. Deriano
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus 16 (HPV16) is a causative agent of oropharyngeal, cervical, and anogenital cancers. The viral E2 protein is essential for viral genome replication, transcriptional regulation, episome maintenance, and activation of the host DNA damage response. Despite its central role, the full network of HPV16 E2 interactions with host proteins remains incompletely defined, particularly under differentiating conditions, which support the complete viral life cycle. In this study, we used TurboID-based proximity labeling to characterize the interactome of HPV16 E2 and its known host partner protein TOPBP1 in both undifferentiated monolayer and differentiating keratinocytes. We generated stable keratinocyte lines expressing doxycycline-inducible TurboID-tagged HPV16 E2 and confirmed that the tagged protein retained transcriptional, replicative, and DNA damage-inducing functions. Mass spectrometry analysis of streptavidin-enriched proteins identified both known and novel E2-associated host factors, including chromatin regulators, DNA repair proteins, and nucleolar components. Comparative analysis revealed substantial overlap between E2 and TOPBP1 interactomes, and in situ validation by proximity ligation assay identified nucleolin (NCL) as a differentiation-dependent factor whose interaction with E2 is stabilized by TOPBP1. Functional studies demonstrated that NCL is required for episomal genome maintenance, highlighting a cooperative E2–TOPBP1–NCL axis critical for viral genome stability during differentiation. These findings provide a comprehensive view of the E2-associated protein landscape in stratified epithelial cells and reveal a mechanistic pathway through which HPV16 co-opts host factors to support genome maintenance, productive replication, and persistence. IMPORTANCE Human papillomaviruses (HPVs) establish persistent infections in stratified epithelia and rely on host DNA damage and repair factors to support their replication. The E2 protein is central to viral genome replication and maintenance and depends heavily on its interaction with the host factor TOPBP1 for these functions. Here, we define the E2 and TOPBP1 interactomes in differentiating keratinocytes and identify nucleolin (NCL) as a critical differentiation- and TOPBP1-dependent E2 partner required for episomal genome stability. These findings expand the understanding of how HPV16 coordinates viral replication with host chromatin and DNA repair networks, uncovering a cooperative E2–TOPBP1–NCL axis that may represent a new target for antiviral intervention. Human papillomaviruses (HPVs) establish persistent infections in stratified epithelia and rely on host DNA damage and repair factors to support their replication. The E2 protein is central to viral genome replication and maintenance and depends heavily on its interaction with the host factor TOPBP1 for these functions. Here, we define the E2 and TOPBP1 interactomes in differentiating keratinocytes and identify nucleolin (NCL) as a critical differentiation- and TOPBP1-dependent E2 partner required for episomal genome stability. These findings expand the understanding of how HPV16 coordinates viral replication with host chromatin and DNA repair networks, uncovering a cooperative E2–TOPBP1–NCL axis that may represent a new target for antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ded8316db339bd64df9132a4a8179bca47c1336a" target='_blank'>
              Differentiation-dependent proximity proteomics identifies novel host factors linked to HPV16 E2 function
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, A. T. Prabhakar, Jenny D Roe, Elinor Lu, Austin J. Witt, S. Giri, Molly L Bristol, Phoebe Bridy, Xu Wang, Arjun Rijal, Charles Lyons, I. Morgan
          </td>
          <td>2026-01-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E. Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, T. F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Simple Summary In many types of cancer, cells double their entire set of chromosomes, a state known as tetraploidy. This change can accelerate the cancer’s growth, spread, and resistance to drugs. We wanted to understand how this chromosome doubling changes the cell’s mitochondria. We compared cells with a normal versus a doubled set of chromosomes from human cancers, yeast, and fungi. We found that in all cases, cells with doubled chromosomes were larger and contained more mitochondria, which were also more active. By identifying this shared trait, our research points to these adapted mitochondria as a potential new target for treating cancers with unstable chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e2f77cdf8945e8a66c23348025539c8ac24aa7" target='_blank'>
              Mitochondrial Adaptations Underlying Tetraploidization in Human Cancer, Fungal, and Yeast Models
              </a>
            </td>
          <td>
            Mohamed Jemaà, Ameni Bedoui, Nihel Ammous, Ali Gargouri, Mohamed Guerfali
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: PHF6 is a chromatin-binding protein located in the nucleus, and it is involved in transcriptional regulation. However, limited research exists on the specific roles and mechanisms of PHF6 across various tumors. Methods: Based on The Cancer Genome Atlas (TCGA) database, we analyzed PHF6 expression in pan-cancer. We first evaluated the relevance between PHF6 and prognosis; then, the relevance between PHF6 and immune cell infiltration in pan-cancer were analyzed. Subsequently, we explored the correlation between PHF6 and cancer heterogeneity, such as tumor mutation burden (TMB) and microsatellite instability (MSI), as well as cancer stemness. Finally, the role of PHF6 was validated in liver cancer and pancreatic cancer cell lines by cell proliferation assays. Results: PHF6 expression was higher in the vast majority of cancers than their normal counterparts. PHF6 was substantially correlated with prognosis and immune cell infiltration in various cancers. Moreover, PHF6 expression showed a strong correlation with cancer heterogeneity and stemness in certain cancer types. Additionally, the depletion of PHF6 inhibited cell proliferation in both liver and pancreatic cancer cells. Conclusions: PHF6 expression was closely associated with the occurrence and development of many types of cancer, and it might promote cancer progression by inhibiting the function of the immune microenvironment, while knockout of PHF6 significantly inhibited the tumor cells proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb4747735aa530ce0fcc12a646bd5b0adcc8b9" target='_blank'>
              Systematic Pan-Cancer Analysis Identifies PHF6 as an Immunological and Prognostic Biomarker
              </a>
            </td>
          <td>
            Yi Tao, Mingming Hu, Zixin Guo, Qian Xu, Lu Niu, Yonghong Mao, Gang Yuan
          </td>
          <td>2025-12-29</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, may also play an important role in tetraploid (4N) cancer cell physiology. Here, we showed that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. Small size was associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size may contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism.
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Essential meiotic structure‐specific endonuclease 1 (EME1) is integral to the maintenance of genomic stability in various cancers. However, its biological role and expression profile of this molecule in nasopharyngeal carcinoma (NPC) remain to be explored. In this study, we found that EME1 was overexpressed in NPC specimens compared to adjacent noncancerous tissues and was correlated with poorer overall survival outcomes. Furthermore, EME1 knockdown significantly inhibited the proliferation and migration of NPC cells in vitro and in vivo, with a corresponding sensitization to either camptothecin (CPT) or olaparib, evidenced by a further suppression of proliferation upon drug treatment. Notably, silencing EME1 significantly increased the sensitivity of NPC cell lines to CPT by enhancing ATM‐CHEK2 phosphorylation and inducing nuclear abnormalities. Collectively, our findings suggest that combining EME1 modulation with agents such as CPT or olaparib could be an effective treatment strategy for NPC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4b0d132ea96aa28bec17ad81c4b1af8056bcb" target='_blank'>
              Targeting EME1 Increases the Sensitivity of Camptothecin in Nasopharyngeal Carcinoma Cells
              </a>
            </td>
          <td>
            Xizhen Jiang, Falian Liang, Zhirui Lin, Fang Yang, Dong-ping Chen, Gengde Hong, Jinquan Liu, Wenjing Yin, Mengyao Wang, Bin Qi
          </td>
          <td>2026-01-14</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, D. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            Tanveer Ahmad, Xiaojie Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, E. Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="RecQ family of DNA helicases play pivotal roles in DNA replication, repair and responses to DNA damage or replication stress. Several human RecQ helicases are defective in diseases associated with chromosomal instability, premature aging and cancer. We recently discovered novel mutations in the RECQL4 gene in glioblastoma (GBM), the most malignant brain tumor in adults. Transcriptomic profiles of GBMs with REQCL4 mutations resembled those in REQCL4 KO glioma cells. We employ structural modelling and biochemical approaches to elucidate impacts of novel mutations on RECQL4 helicase activities. Using recombinant RECQL4P532S and RECQL4R766Q proteins we demonstrate that P532S substitution reduces the RECQL4 ability to unwind DNA and disrupts DNA-coupled ATP-hydrolysis activity. WT and mutated RECQL4 were overexpressed in RECQL4 KO glioma cells to study interactions with BLM helicases, cell viability and specific responses to UVC- and chemotherapy-induced DNA damage/repair. Overexpression of RECQL4P532S or RECQL4R766Q variants affected DNA repair and responses to chemotherapeutics in glioma cells, and RECQL4R766Q disturbed interactions with the BLM helicase. Our results reveal deleterious consequences of novel RECQL4 mutations in GBMs. The newly identified RECQL4 mutations affect RECQL4 helicases and their interactions with BLM contributing to glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650345dd7db490bb44b3ecdcc3b0ad865cd59a8" target='_blank'>
              Novel mutations in the RECQL4 gene affect its helicase functions, interactions with the BLM helicase and chemotherapeutics-induced cell death
              </a>
            </td>
          <td>
            Agnieszka Kaczmarczyk, Mikołaj Sokołowski, K. Wojnicki, Marta Pabiś, B. Wojtaś, I. Ciechomska, K. Poleszak, B. Gielniewski, S. K. Król, Matthew Guille, Sebastian Glatt, B. Kaminska
          </td>
          <td>2025-12-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The genome folds inside the cell nucleus into hierarchical architectural features, such as chromatin loops and domains. If and how this genome organization influences the regulation of gene expression remains only partially understood. The structure-function relationship of genomes has traditionally been probed by population-wide measurements after mutation of critical DNA elements or by perturbation of chromatin-associated proteins. To circumvent possible pleiotropic effects of such approaches, we have developed OptoLoop, an optogenetic system that allows direct manipulation of chromatin contacts by light in a controlled fashion. OptoLoop is based on the fusion between a nuclease-dead SpCas9 protein and the light-inducible oligomerizing protein CRY2. We demonstrate that OptoLoop can bring together genomically distant, repetitive DNA loci. As a proof-of-principle application of OptoLoop, we probed the functional role of DNA looping in the regulation of the human telomerase gene TERT by long-range contacts with the telomere. By analyzing the extent of chromatin looping and nascent RNA production at individual alleles, we find evidence for looping-mediated repression of TERT. In sum, OptoLoop represents a novel means for the interrogation of structure-function relationships in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5676ff6c24cf2aa8e41cc35c8a6bbe06bad743e9" target='_blank'>
              OptoLoop: An optogenetic tool to probe the functional role of genome organization.
              </a>
            </td>
          <td>
            Martin Stortz, Adib Keikhosravi, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2026-01-19</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Loss-of-function mutations in DNMT3A, a DNA methyltransferase, or NSD1, a histone methyltransferase, cause overgrowth syndromes. Conversely, disruption of the DNMT3A domain that binds NSD1-deposited H3K36 dimethylation (H3K36me2) results in growth restriction. To investigate the molecular basis of these opposing growth outcomes, we generated isogenic human embryonic stem cells carrying growth syndrome–associated mutations in DNMT3A and NSD1. Unexpectedly, both overgrowth- and growth restriction–associated DNMT3A mutations led to DNA hypomethylation in a shared subset of active enhancers, implicating H3K36me2 in directing enhancer methylation maintenance. In contrast, bivalent promoters—marked by both active and repressive histone modifications—showed divergent DNA methylation changes: hypermethylation in growth restriction-associated DNMT3A mutants and hypomethylation in overgrowth-associated DNMT3A or NSD1 loss-of-function mutants. These findings identify locus-specific DNA methylation defects as a common molecular feature and nominate dysregulated DNA methylation at bivalent promoters as a potential driver of abnormal growth phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f699bcc558592153abc552376bed77e07ec2c5e0" target='_blank'>
              Convergent DNA methylation abnormalities at enhancers and bivalent promoters in human growth disorders
              </a>
            </td>
          <td>
            Marie E S Wheeler, Yoshiko Takahashi, Jihye Lee, Camille T Perez, Xiaoting Chen, Yuri Lee, Zachary S. Pope, Daniella J Lu, Marcus M. Seldin, Ivan Marazzi, Hongseok Yun, M. Weirauch, Minji Byun
          </td>
          <td>2025-12-27</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a5c7aee5589d3f24718b4f0d1ab22f036f7184d" target='_blank'>
              Retrotransposon Activation in the Aged and Alzheimer’s Disease Brain Examined by Nanopore Long-read DNA Sequencing
              </a>
            </td>
          <td>
            M. Kelsey, Anjalika Chongtham, J. LaCava, Martin S. Taylor, J. Boeke, F. H. Gage, A. Seluanov, Vera Gorbunova, Ana C. Pereira, J. Sedivy
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Organisms regulate cell size and shape to function efficiently. Aberrant cell morphogenesis is commonly associated with disease, yet gene-regulatory mechanisms remain unknown. CISTR-ACT was the first lncRNA involved in inter-chromosomal proximities and Mendelian disease, and it is associated with mean corpuscular volume (red blood cell size). Here, functional dissection of CISTR-ACT’s DNA- and RNA-encoded mechanisms by in vitro and in vivo perturbations reveals that CISTR-ACT regulates cell size across cell types and species. CISTR-ACT’s locus is embedded in a stable inter-chromosomal environment which contains cell size genes that are regulated by CISTR-ACT in trans. CISTR-ACT’s RNA also has function and directly interacts with transcription factor FOSL2 to guide its regulation of cell morphogenesis and cell-cell adhesion genes. In the absence of CISTR-ACT, the FOSL2-chromatin binding is perturbed. Our study exemplifies how a functionally conserved lncRNA regulates cell size with multiple modes of action and ultimately contributes to clinically relevant phenotypes. Establishment and maintenance of appropriate cell size is a prerequisite for cells to function efficiently. Here, Kiriakopulos et al. reveal that the lncRNA CISTR-ACT maintains cell size across cell types in humans and mice by regulating cell morphogenesis genes in trans via guidance of the transcription factor FOSL2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2257e0e8bb5b6556885e88aef4e0f1ede88e699" target='_blank'>
              LncRNA CISTR-ACT regulates cell size in human and mouse by guiding FOSL2
              </a>
            </td>
          <td>
            Katerina Kiriakopulos, Katty Soleimanpour, Brandon J. McMurray, Benjamin-Israel Moke, Jordan J. Chalmers, Milad Mokhtaridoost, Jeremy D Newton, Taylor De Young, Kate Delfosse, Mai Ahmed, Cassandra J. Wong, S. Stricker, Yun Li, Brian J. Nieman, A. Gingras, Monica J. Justice, Julie L. Lefebvre, Philipp G. Maass
          </td>
          <td>2025-12-16</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>110</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eb287f4be34ac61e66b972c8521500710a820fd" target='_blank'>
              Dual inhibition of the nonsense mediated mRNA decay enhances tumour immunogenicity, drives immunoediting, and potentiates checkpoint blockade
              </a>
            </td>
          <td>
            Ivan Zadra, Davor Orsolic, Dimitri Kounis, Mario Acera, Guillermo Palou Márquez, Etna Abad, Laura Ortet, Anna Benito, Y. Kharraz, Carlos M. Martínez, Elisabetta Mereu, Fran Supek, Ana Janic
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumor (ATRT) is a highly aggressive but genetically simple pediatric central nervous system tumor, defined by biallelic inactivation of the chromatin regulator SMARCB1 with remarkably few other cooperating mutations. Despite its genetic homogeneity, ATRT exhibits profound clinical and epigenetic heterogeneity, with three major subgroups (ATRT-TYR, ATRT-MYC, and ATRT-SHH) defined by DNA methylation and transcriptional signatures. Beyond these subgroup-defining features, we aimed to investigate epigenetic variability within tumors by applying whole-genome bisulfite sequencing and probabilistic modeling to quantify stochastic DNA methylation in primary ATRT samples encompassing all three subgroups. We show that ATRT exhibits a destabilized and increasingly stochastic methylome. While ATRT global methylation patterns diverge according to subgroup, some methylation perturbations, such as hypermethylation and increased methylation entropy over bivalent promoters, are consistent across subgroups. We find that methylation stochasticity alterations map onto potential drivers of ATRT, such as LIN28a, the HOXD cluster for ATRT-MYC, and OTX2 for ATRT-TYR, and identify actionable targets, such as hypermethylation of the tumor suppressor CDKN2a across all subgroups. We investigate the sensitivity of the aberrant DNA methylation landscape of ATRT to pharmacologic DNA methyltransferase inhibition (DNMTi) and histone deacetylase inhibition (HDACi). We show that decitabine leads to profound demethylation of patient-derived ATRT cell lines, including reversal of hypermethylation at bivalent promoters and the CDKN2a locus. The addition of HDACi leads to dramatic gene expression changes, including upregulation of innate immune signaling pathways, such as STING/interferon signaling, genes under the regulation of bivalent promoters, and reactivation of the tumor suppressor CDKN2A. The combination of DNMTi and HDACi synergistically reduces cell viability. Taken together, we show that ATRT has a highly stochastic methylome sensitive to epigenetic manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e14cbda965cb78ec2d086d8bbd22a534ed3d3a3" target='_blank'>
              Disordered DNA methylation leads to targetable transcriptional plasticity in ATRT
              </a>
            </td>
          <td>
            Ashley R Tetens, Tyler R Findlay, Jordyn Craig-Schwartz, Athanasia Liapodimitri, Oscar Camacho, Kegan O Skalitzky, Adrian Idrizi, Rakel Tryggvadottir, Kayleigh Lunsford, Eric H. Raabe, Michael A Koldobskiy
          </td>
          <td>2025-12-17</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tandem amplification of genomic fragments is quite common in bacteria growing under stress conditions, while spontaneous genome amplification events are rare, unstable and generally poorly described. Plague pathogen Yersinia pestis is a unique microorganism that contains an enormous number of short repeat sequences in its genome and as a result is very prone to spontaneous genome rearrangements including large tandem genome amplification events. Eleven Y. pestis strains sequenced during this study and more than thousand read archives from SRA were analyzed in this study. It was shown that genomes of more than half of Y. pestis laboratory isolates contain tandem repeats. They are mainly caused by the presence of multicopy IS-elements but a few of them are associated with multicopy rRNA clusters, so the rearrangement mechanism is most likely RecA-dependent recombination. Four regions with unstable copy number reproduced between different bioprojects were found. One of them was identified as an integrative mobilizable element carrying a probably incomplete Type 4 secretion system. More interesting, two other reproducible regions were not identified as mobile elements but had the length and GC-content almost identical to the length and GC-content of pMT1 and pCD1 plasmids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bf8e6e37f6935779620869a8b92594a383a1b6" target='_blank'>
              Systematic analysis of spontaneous tandem genome amplification events in Yersinia pestis
              </a>
            </td>
          <td>
            D. Konanov, Olga N. Liubimova, Alexander V. Kovrizhnikov, Ignat V. Sonets, A. N. Balykova, Alexandra V. Lukina-Gronskaya, Anna S. Speranskaya, D. Krivonos, G. A. Eroshenko, Elena N. Ilina, Vadim M. Govorun, V. V. Kutyrev
          </td>
          <td>2025-12-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Compacting chromatin within the cellular nucleus presents a significant challenge for biology. Chromosomes must be both condensed and spatially organized to enable essential processes such as transcription and replication. Chromosome conformation capture experiments (e.g., Hi-C) provide valuable information about the spatial organization and, therefore, the connectivity between different genomic regions. These experiments inspired polymer models that describe the physical mechanism of the chromosomal energy landscape. The Full-Inversion Chromatin model (FI-Chrom), a data-driven approach for modeling genome organization, uses Hi-C contact maps to infer pairwise interaction potentials between all chromosomal loci. It combines Graphics Processing Unit (GPU)-accelerated simulations with efficient training of tens of millions of parameters derived from the maximum-entropy principle to determine 3D structures of chromosomes that accurately reproduce Hi-C-like data. FI-Chrom does not make any a priori assumptions regarding chromosome architecture, making it applicable to any chromosome conformation capture experiment. Its derived structural ensembles capture all essential features from the short- and long-range interactions of typical chromosome organization, such as segregated compartments, chromosome territories, and fully or partially formed loops. Although Hi-C contains only structural information, FI-Chrom extends these data by revealing an emergent dynamical mechanism encoded in the inferred energy landscape. For example, simulations show that chromatin loops are not static architectural features but rather transient structural elements. Statistical analyses further indicate that loops confined within a single compartment occur more frequently than those spanning multiple compartments, highlighting the dynamic and compartment-dependent nature of chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5e464495ba1f67ebcfcfd2e8d42c98c8759239d" target='_blank'>
              A data-driven chromatin model reveals spatial and dynamic features of genome organization.
              </a>
            </td>
          <td>
            A. B. Oliveira Junior, Matheus F. Mello, Ronaldo J Oliveira, Esteban Dodero-Rojas, Sumitabha Brahmachari, V. Contessoto, J. Onuchic
          </td>
          <td>2026-01-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer.
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Inflammation plays a critical role in cancer initiation and progression by modulating the tumor microenvironment and immune responses. Interleukin-1 receptor–associated kinase 2 (IRAK2) is a key mediator of the Toll-like receptor and interleukin-1 receptor signaling pathways, its pan-cancer expression patterns, genomic and epigenetic features, immune-related roles, and clinical relevance remain unclear. The expression patterns of IRAK2 across multiple cancer types, transcript variants, single-cell distribution, prognostic significance, and biological functions were comprehensively evaluated through analyses of multiple databases and multi-dimensional datasets. Furthermore, the correlations of IRAK2 with the immune microenvironment, epigenetic modifications, and drug sensitivity were investigated. The potential role of IRAK2 in hepatocellular carcinoma was further explored through both in vitro and in vivo experiments. Aberrant expression of IRAK2 was observed in the majority of cancer types, with a relatively high proportion of expression detected in macrophages, and was found to be associated with the prognosis of certain cancers. In most cancer types, IRAK2 expression showed significant correlations with immune cell infiltration, the cancer-immunity cycle, major histocompatibility complex molecules, immune checkpoints, tumor mutational burden, microsatellite instability, RNA modifications (including m1A, m5C, and m6A), and DNA methylation sites. Both in vitro and in vivo experiments demonstrated that knockdown of IRAK2 markedly reduced the proliferative capacity of hepatocellular carcinoma cells. The present study highlights the potential of IRAK2 expression as a novel biomarker for predicting the prognosis and immunotherapeutic response across various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c6c0ac99cada70622c4e70f3537871b6c0f3bd" target='_blank'>
              Comprehensive pan-cancer characterization of IRAK2 as a potential prognostic biomarker and therapeutic target with validation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Haijian Dong, Yuanqian Yao, Qun Niu, Xueqing Gong, Yu Mou, Zijian Zeng, Hui Li
          </td>
          <td>2025-12-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CDK9, in complex with cyclin T1 or T2, is essential for mRNA transcription by enabling paused RNA polymerase II to proceed into elongation. Increasing evidence highlights CDK9’s involvement in transcriptional addiction in cancer. Triple-negative breast cancer (TNBC) is an aggressive breast cancer subtype for which effective targeted therapies remain limited. Here, we aimed to define the therapeutic potential of novel CDK9 inhibitors in TNBC. We explored the efficacy and mechanism of action of novel CDK9 inhibitors, alone or in combination with EGFR inhibitors, using TNBC cell lines and in vivo xenograft models. Targeting CDK9 significantly impaired proliferation and induced apoptosis in multiple TNBC cell lines. Transcriptomic analyses revealed that CDK9 inhibitors induced downregulation of genes involved in transcription, cell cycle progression, and oncogenic signalling pathways, including TGF-β and Wnt/β-catenin signalling. Combined CDK9 and EGFR inhibition disrupted transcriptional programs, enhanced TNBC cell death in vitro, and acted synergistically to reduce tumour growth in PDX and Hs578T xenograft models, although this combination was also associated with increased toxicity. Our results position CDK9 as a promising therapeutic target in TNBC, either alone or in combination with EGFR inhibition, provided that side effects associated with this combination treatment can be controlled.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acbf12d986be88bb6e251fe6f6618756d9f000ef" target='_blank'>
              Disrupting CDK9 activity suppresses triple-negative breast cancer and is enhanced by EGFR Inhibition
              </a>
            </td>
          <td>
            V. V. D. Noord, R. McLaughlin, J. Karuntu, Jichao He, ·. A. M. Timmermans, Sunita K. C. Basnet, Yi Long, Sarah Al, Haj Diab, Solomon Tadesse, Natalie Proost, B. V. Gerwen, Bjørn Siteur, M. Ven, C. Pont, S. L. Dévédec, John W. M. Martens, Shudong Wang, Yinghui Zhang, B. Water
          </td>
          <td>2026-01-08</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e82aa8297f80f569a06497cad1d25c2344da9284" target='_blank'>
              The 3D Genome of Gigaspora margarita Unveils Stable Chromatin and Nucleolar Organization and Symbiont-Dependent Genome Dynamics
              </a>
            </td>
          <td>
            Ken Mugambi, Jordana Oliveira, F. Magurno, Alessandra Salvioli di Fossalunga, M. Novero, L. Lanfranco, Stefano Ghignone, Gokalp Yildirir, Yan Wang, Paola Bonfante, Nicolas Corradi
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the second leading cause of cancer-related death in men, accounting for over 35,000 deaths annually in the USA. While coding mutations have been extensively characterized, the contribution of non-coding somatic variants to PCa development and progression remains poorly understood. These variants are thought to influence tumor biology by modulating gene expression through regulatory elements. To address this, we mapped somatic mutation-enriched regions across the Pan-Cancer Analysis of Whole Genomes and Hartwig PCa cohorts and identified several recurrently altered enhancers. CRISPR activation of one such enhancer at 14q22.1 increased expression of its target gene, FERMT2 (Kindlin-2), a focal-adhesion adaptor. Functional assays using CRISPR knockout and siRNA knockdown models showed that loss of FERMT2 significantly reduced migration and clonogenic survival in androgen receptor (AR)-negative, but not AR-positive, PCa cells, indicating a context-dependent role in adhesion and growth control. Multi-omic analyses of clinical datasets demonstrated that FERMT2-high tumors are enriched for extracellular matrix and focal-adhesion signaling, whereas FERMT2-low tumors rely on proliferative and DNA-damage-repair (DDR) pathways, particularly under hypoxia. Low FERMT2 expression also correlated with shorter disease-free survival. Collectively, these findings suggest that FERMT2 is a candidate biomarker of prognostic and therapeutic relevance, offering a framework for patient stratification and informing the development of targeted interventions in PCa. Ongoing work aims to elucidate the molecular mechanisms underpinning therapy response in FERMT2-high and FERMT2-low tumors.



 Victoria Blair, Nick Orr, Sarah Maguire. Investigating FERMT2 expression as a determinant of prostate cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec6381f5325ea7ce6a42cb5ea3b084b931435af2" target='_blank'>
              Abstract A005: Investigating
 FERMT2
 expression as a determinant of prostate cancer progression
              </a>
            </td>
          <td>
            Victoria Blair, Nick Orr, Sarah Maguire
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21b610301b7b55b77a33ac6317a40f28dcdb985" target='_blank'>
              A gastric microbial chromatin remodeler drives gastric cancer progression and immune evasion by reprogramming the host epigenome
              </a>
            </td>
          <td>
            Xiaoshan Xie, Yue Wei, Zhikai Zheng, Jiaying Zheng, Xijie Chen, Jiarui Wang, Ning Ma, Xiaoling Huang, Peng Zhang, Boyu Zhang, Hanyong Cai, Li Ma, Lishi Xiao, Qingxin Liu, Wenyu Wang, Sachiyo Nomura, Shi Chen, Xiangqi Meng, Mong-Hong Lee
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background and objectives Somatic mutation patterns in cancer remain largely unexplored outside humans, despite their significance for aging and oncogenesis. Chimpanzees (Pan troglodytes), sharing >98% genomic similarity with humans, display markedly different cancer spectra. To gain comparative insights into cancer susceptibility and resistance, we sequenced chimpanzee hepatocellular carcinoma (HCC) genomes and analyzed their mutational profiles alongside human counterparts. Methodology HCC and matched non-cancerous tissues from five chimpanzees were examined using histopathology, immunohistochemistry (β-catenin, ARID1A, TSC2, FAP, vimentin, TGF-β), whole-genome sequencing (one pair), and whole-exome sequencing (four pairs). Somatic variants were identified with GATK MuTect2, annotated with Ensembl VEP, and analyzed for functional enrichment. Comparative analyses were performed with subsets of human HCC datasets (TCGA, ICGC) including TSC2-positive and TSC2-negative cases. Results Chimpanzee HCCs exhibited histological and immunohistochemical features similar to human tumors but displayed sharply divergent genomic landscapes. Chimpanzee tumors carried significantly higher coding mutation loads (mean 5632 per sample vs. 96–275 in humans). Non-synonymous TSC2 mutations occurred in 80% of chimpanzees, versus ~7% in human HCC, suggesting a species-specific oncogenic pathway linked to the scirrhous subtype. Additional recurrently mutated genes included ARID1A, FAT1–4, TP53, and FGA. Despite greater heterogeneity in chimpanzee tumors, humans showed stronger enrichment of non-synonymous single nucleotide variants, implying more intense positive selection. Shared alterations across species involved canonical drivers such as TP53, CTNNB1, FAT4, and TTN. Conclusions and implications Chimpanzee HCCs are defined by high mutational burden and frequent TSC2 alterations, contrasting with the more selectively constrained mutation spectrum of human HCC. Divergent evolutionary patterns highlight species-specific oncogenic routes while underscoring conserved pathways. Comparative primate cancer genomics offers novel insights into cancer evolution, biomarkers, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7fdead0d4830205015111b5a2fbfd3bc9dc0ec" target='_blank'>
              Divergent evolution of hepatocellular carcinoma genomes in chimpanzees and humans
              </a>
            </td>
          <td>
            Lin Kang, K. Michalak, Robin T. Varghese, R. Anandakrishnan, Edward J Dick, Z. A. Abd Elmageed, Pawel Michalak
          </td>
          <td>2025-12-15</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Ewing sarcoma is driven by the EWSR1–FLI1 fusion oncoprotein, a transcription factor created by chromosomal translocation in otherwise low-mutation tumors. EWSR1–FLI1 recruits chromatin-remodeling complexes, opens chromatin, and establishes broad H3K27ac/H3K4me1 domains. Uniquely, it binds normally silent GGAA microsatellites dispersed across the genome and converts them into de novo enhancers, elevating transcription of downstream targets. We hypothesize that EWSR1–FLI1 engagement of GGAA repeats globally destabilizes chromatin and enforces a differentiation block, and that direct inhibition of its DNA binding will restore transcriptional control. To test this, we established a propidium iodide (PI) uptake assay after exposure to the chromatin-damaging agent CBL0137 as a surrogate for chromatin accessibility, benchmarking ES against non-ES tumor and non-tumor cells, and in parallel engineered a DNA-binding–dependent reporter to screen small molecules that disrupt fusion–DNA interactions. ES cells showed greater PI uptake following CBL0137 than controls, consistent with globally relaxed chromatin. The reporter screen identified candidate compounds that decreased fusion-dependent activity in a dose-responsive manner; orthogonal luciferase assays supported potential on-target inhibition, and viability assays revealed selective toxicity in ES relative to fusion-negative sarcomas. These findings support a model in which fusion oncoproteins promote chromatin relaxation and establish a complementary screening platform that nominates direct fusion inhibitors, providing mechanistic insight and therapeutic starting points for pediatric sarcomas.



 Ayah Salameh. GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c2a09d1dd63318d72cdbf406bfb1ca4ece465c" target='_blank'>
              Abstract A010: GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform
              </a>
            </td>
          <td>
            Ayah Salameh
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Myeloid leukemias, diseases marked by aggressiveness and poor outcomes, are frequently triggered by oncogenic translocations. In the case of chronic myelogenous leukemia (CML), the BCR-ABL fusion initiates chronic phase disease with second hits allowing progression to blast crisis. Although Gleevec has been transformative for CML, blast crisis CML remains relatively drug resistant. Here, we show that MSI2-HOXA9, a translocation with an unknown role in cancer, can serve as a second hit in driving bcCML. Compared to BCR-ABL, BCR-ABL/MSI2-HOXA9 led to a more aggressive disease in vivo with decreased latency, increased lethality, and a differentiation blockade that is a hallmark of blast crisis. Domain mapping revealed that the MSI2 RNA binding domain RRM1 had a preferential impact on growth and lethality of bcCML relative to RRM2 or the HOXA9 domain. Mechanistically, MSI2-HOXA9 triggered global downstream changes with a preferential upregulation of mitochondrial components. Consistent with this, BCR-ABL/MSI2-HOXA9 cells exhibited a significant increase in mitochondrial respiration. These data suggest that MSI2-HOXA9 acts, at least in part, by increasing expression of the mitochondrial polymerase POLRMT and augmenting mitochondrial function and basal respiration in blast crisis. Collectively, our findings demonstrate for the first time that translocations involving the stem and developmental signal MSI2 can be oncogenic and suggest that MSI, which we found to be a frequent partner for an array of translocations, could also be a driver mutation across solid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0a174e1fc7002e60c4dfa2851f21e71b1198a5" target='_blank'>
              Identification of a Musashi2 translocation as a novel oncogene in myeloid leukemia
              </a>
            </td>
          <td>
            Kyle Spinler, Michael Hamilton, Jeevisha Bajaj, Yutaka Shima, Emily Diaz, M. Kritzik, T. Reya
          </td>
          <td>2026-01-07</td>
          <td>eLife</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="High mobility group A1 (HMGA1), a non-histone chromatin structural protein encoded by the HMGA1 gene, plays a critical role in cancer. Recent studies have increasingly focused on its functions in genomic stability and cell death, revealing its involvement in tumorigenesis, cancer progression, and chemotherapy resistance. Consequently, inhibiting HMGA1 represents a promising strategy for developing novel cancer therapies. This review summarizes the cellular and molecular functions of HMGA1 in regulating genomic integrity and cell death in cancer. Furthermore, we discuss current HMGA1-targeting strategies, with emphasis on approaches leveraging its structural and functional characteristics, aiming to provide new insights for future research on HMGA1-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dfc0cc9a26137eed4ac1b69fa1e9407df182b1" target='_blank'>
              The role of HMGA1 in genome stability: Implications in human cancer
              </a>
            </td>
          <td>
            , Kai-yue He, Yong-Ping Jian, 
          </td>
          <td>2026-01-26</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="PHF20 encodes plant homeodomain finger protein 20 (PHF20), a component of the KAT8‐containing nonspecific lethal (NSL) complex that deposits acetylation on histone H4 to activate gene expression. We report two unrelated individuals with developmental delay, microcephaly, and distinctive facial features, in whom exome sequencing and chromosomal microarray analysis revealed a homozygous deletion of PHF20 that segregated with the disease phenotype in their families. Breakpoint junction sequencing revealed an Alu–Alu‐mediated deletion event. Western blot in cells from an affected individual showed undetectable PHF20, while levels of other NSL complex subunits were unaltered. Transcriptomic and epigenomic analysis revealed significant downregulation of gene pathways related to cell projection and neuronal development, associated with reduced histone H4K16 acetylation at these genes. In conclusion, our data suggest that homozygous deletion of PHF20 leads to a neurodevelopmental syndrome, potentially through targeted epigenetic dysregulation and altered gene expression essential for neuronal development. Identifying additional families with biallelic PHF20 variants will further delineate the phenotypic spectrum, and molecular studies in neuronal cell lines will be essential for understanding the disease mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ebe2b3854a5aa35254b0ecad5b6631cc63aebb0" target='_blank'>
              Homozygous Deletion of the Epigenetic Regulator PHF20 in Individuals With Neurodevelopmental Disorder
              </a>
            </td>
          <td>
            Shira Yanovsky Dagan, Hongwen Xuan, Jonathan Rips, Emuna Paz-Ebstein, Talia Baer, Shira Gross, A. Frumkin, Xiaobing Shi, T. Harel
          </td>
          <td>2025-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Spermatogenesis is a tightly coordinated differentiation program that sustains male fertility while transmitting genetic and epigenetic information to the next generation. This review consolidates mechanistic evidence showing how RNA-centered regulation integrates with the epitranscriptome and three-dimensional (3D) genome architecture to orchestrate germ-cell fate transitions from spermatogonial stem cells through meiosis and spermiogenesis. Recent literature is critically surveyed and synthesized, with particular emphasis on human and primate data and on stage-resolved maps generated by single-cell and multi-omics technologies. Collectively, available studies support a layered regulatory model in which RNA-binding proteins and RNA modifications coordinate transcript processing, storage, translation, and decay; small and long noncoding RNAs shape post-transcriptional programs and transposon defense; and dynamic chromatin remodeling and 3D reconfiguration align transcriptional competence with recombination, sex-chromosome silencing, and genome packaging. Convergent nodes implicated in spermatogenic failure are highlighted, including defects in RNA metabolism, piRNA pathway integrity, epigenetic reprogramming, and nuclear architecture, and the potential of these frameworks to refine molecular phenotyping in male infertility is discussed. Finally, key gaps and priorities for causal testing in spatially informed, stage-specific experimental systems are outlined.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc54e6cdf4f079512c4f096e69dbc3f2d8520f48" target='_blank'>
              Spermatogenesis Beyond DNA: Integrated RNA Control of the Epitranscriptome and Three-Dimensional Genome Architecture
              </a>
            </td>
          <td>
            Ares Kaltsas, Maria-Anna Kyrgiafini, Z. Mamuris, Michael Chrisofos, Nikolaos Sofikitis
          </td>
          <td>2026-01-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 70% of ovarian cancer-related deaths, yet therapeutic progress remains stagnant. Among the four molecular subtypes reported for HGSC, the C5 subtype is distinguished by high proliferation and immune evasion with an unfavorable MHC-I/PD-L1 ratio. However, the molecular drivers of this immune desert state remain largely undefined. Here, we identify RNA-binding proteins (RBPs) as key regulators of immune evasion in C5-HGSC through integrated single-cell and bulk RNA sequencing. We perform a targeted loss-of-function screen in C5-like cell models and find IGF2BP1 as a central mediator of immune evasion in vitro and in vivo. Mechanistically, IGF2BP1 abrogates interferon-gamma signaling by accelerating IRF1 protein degradation, thereby suppressing MHC-I presentation. We also discover that IGF2BP1 decouples PD-L1 expression from IRF1-dependent transcription and reshapes the immune receptor landscape to limit immune cell infiltration and T cell activation. Therapeutically, the small-molecule BTYNB effectively inhibits IGF2BP1 and synergizes with PD-1 blockade to overcome immune evasion in vivo. Multi-spectral imaging confirms these findings in human HGSC tissues and highlights the role of oncofetal RBPs as molecular drivers of the C5-HGSC subtype. This subtype-wide survey uncovers a previously unrecognized RBP–interferon regulatory axis and establishes RBP inhibition as a therapeutic strategy to enhance immune checkpoint therapy in immunologically cold ovarian tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489506f242d494b2564a416c7a7d043babe1d708" target='_blank'>
              Inhibition of RNA-binding proteins enhances immunotherapy in ovarian cancer
              </a>
            </td>
          <td>
            N. Bley, Alexander Rausch, Simon Müller, Theresa Simon, Markus Glaß, Danny Misiak, Laura Schian, Lara Meret Peters, Mohammad Dipto, Ali Hmedat, Bianca Busch, Annekatrin Schott, M. Lederer, Alice Wedler, R. Rolnik, Hend Elrewany, Ehab Ghazy, W. Sippl, Martina Vetter, Markus Wallwiener, S. Hüttelmaier
          </td>
          <td>2025-12-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background & objectives Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer. Methods This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6low (negative/weak) or SMC6high (moderate/strong). Results SMC6low tumours tend to show higher CNV. SMC6high group presented poorer disease-free survival than the SMC6low group (P=0.050), mainly for the luminal subtype (P=0.005). SMC6low/ERpos were protective biomarkers for recurrence. Interpretation & Conclusions There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-11-01</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="ABSTRACT High-quality reference genomes are essential for comparative genomics and accurate genotype-phenotype mapping. Here, we corrected the Mycobacterium tuberculosis Erdman strain reference genome (ErdmanTI) using ultra-deep HiFi sequencing. Among the small variants (n = 275) between ErdmanTI and the current Erdman reference NC_020559.1 (ErdmanSTJ), numerous are likely errors in ErdmanSTJ. We identified a novel bias toward in-frame structural variations (SVs) in pe/ppe genes and 28 SVs between ErdmanTI and ErdmanSTJ, half representing likely errors in ErdmanSTJ. Other SVs were consistent with in vitro evolution, including copy number variation (CNV) of promoter tandem repeats (PTRs). PTR CNVs were polyphyletic and within isogenic populations (10−2–10−3 CNVs/chromosome), demonstrating the impact of phase-variable CNV across evolutionary timescales. These hypervariable PTRs pinpoint a genomic basis for rapidly switching nitric oxide resistance (Dop), biofilm formation (LpdA), drug tolerance (EfpA), and glycerol utilization (GlpD2) phenotypes. This work uncovers a common phase variation mechanism obscured by short-read sequencing limitations and provides an improved reference for comparative studies. IMPORTANCE Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response. Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b608d990d2aa84315a169548ac17ba8eb0b7b1e2" target='_blank'>
              Updated Erdman reveals tandem repeat copy number is phase-variable and impacts M. tuberculosis adaptation across evolutionary timescales
              </a>
            </td>
          <td>
            Samuel J. Modlin, Nachiket Thosar, Paulina M. Mejía-Ponce, Raegan L. Lunceford, Gaëlle Guiewi Makafe, Brian Weinrick, F. Valafar
          </td>
          <td>2025-12-17</td>
          <td>mSystems</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The APOBEC3 family of RNA and single stranded DNA cytidine deaminases contribute prominently to the mutagenesis of certain cancers including urothelial carcinoma of the bladder (UC). Remarkably, up to 70% of mutations in UC are attributable to the mutagenic activity of the APOBEC3 deaminases. Despite this strong association, few functional studies have investigated APOBEC3’s role in bladder cancer. We report a genetically engineered murine model with conditional knock out of Pten and Trp53 in addition to overexpression of mouse Apobec3 (UPPA). Analysis of bladder tumors from UPPA mice demonstrates that mA3 promotes tumor progression and squamous trans-differentiation. We establish that APOBEC3 promotes squamous differentiation through IL-1α and downstream activation of the AP-1 transcription factor. Bulk RNA-sequencing from human UC shows APOBEC3A as the only human APOBEC3 family member to correlate with squamous differentiation. Furthermore, single cell and spatial transcriptomics reinforces the role of APOBEC3A in fostering squamous trans-differentiation and promoting the emergence of a subpopulation of highly squamous epithelial cells. Our results demonstrate that mouse Apobec3 and human APOBEC3A promote squamous differentiation in urothelial carcinoma and that this trans-differentiation phenotype is mediated through IL-1α signaling, a target of FDA approved therapies for rheumatologic disease. Deaminases of the APOBEC3 family contribute to the mutagenesis of various cancers, including urothelial carcinoma (UC) of the bladder. Here, the authors use functional studies and transcriptomics to demonstrate that APOBEC3 promotes tumour progression and squamous trans-differentiation in UC through IL-1α and downstream activation of the AP-1 transcription factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14e67774e5f26f7adb282a687c97e8843cc0470" target='_blank'>
              APOBEC3 promotes squamous differentiation via IL-1A/AP-1 signaling
              </a>
            </td>
          <td>
            Michael S. Sturdivant, Andrew Truong, Mi Zhou, E. D. Toomer, Wolfgang Beckabir, John D. Raupp, Ujjawal Manocha, Ibardo A. Zambrano, Hung-Jui Tan, M. Bjurlin, Angela B Smith, Tracy L Rose, Matthew I. Milowsky, Sara E. Wobker, Kathryn H. Gessner, J. Damrauer, William Y. Kim
          </td>
          <td>2025-12-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f26e5c36cf6fc942909dc97f82cb84d88175870d" target='_blank'>
              Transient centrosome loss in cultured prostate epithelial cells induces chromosomal instability to produce an oncogenic genotype that correlates with poor clinical outcomes.
              </a>
            </td>
          <td>
            Jiawen Yang, John M. Ryniawec, Diogo de Oliveira Pessoa, Matthew R Coope, Daniel W. Buster, Emily Loertscher, Mengdie Wang, Chen Chen, A. Cress, Megha Padi, Gregory C Rogers
          </td>
          <td>2026-01-07</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences capable of self-replication (especially retrotransposons) within the genome, which may lead to various forms of DNA damage. The introduction of this review encompasses the diverse classes and subclasses of TEs, particularly emphasizing the most active TEs present in the human genome. An analysis of the retrotransposition process of TEs is presented, illustrating how this mechanism can result in DNA damage and gene rearrangements. Furthermore, the review meticulously examines the implications of TE insertions on gene expression and genomic organization, which may contribute to the development of various diseases, including cancer. The relationship between TE activation and the aging process is also explored, with an emphasis on that epigenetic modifications associated with aging can lead to the derepression of TEs, thereby promoting genomic instability and inflammation. These factors may play a significant role in the pathogenesis of age-related diseases, such as cancer, cardiovascular disorders, and neurodegenerative conditions. Finally, the review considers potential therapeutic approaches aimed at targeting TE activity to alleviate the impacts of aging and associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1552ca333c184d411cd1fdc06105d2ca4b0a5435" target='_blank'>
              The role of transposable elements activity in genomic instability and their relationship to aging process.
              </a>
            </td>
          <td>
            Jingran Hu, Tianhao Mao, Kainan Huang, Shangzhi Yang, Wenrui Yu, Jiacheng Huang, Shiqi Jin, Chuanyu Sun, Zeyidan Jiapaer, Xianli Wang
          </td>
          <td>2026-01-18</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is one of the most common cancers and a leading cause of cancer-related mortality worldwide, highlighting the need for novel therapeutic strategies. Proteasome 26S Subunit, Non-ATPase 12 (PSMD12), a component of the proteasomal 19S regulatory particle, is associated with tumorigenesis; however, its role in LUAD remains poorly understood. Integrative bioinformatic analysis of The Cancer Genome Atlas (TCGA) and other publicly available LUAD datasets identified PSMD12 as a candidate driver gene on chromosome 17q, a region frequently amplified in LUAD. Clinicopathological and prognostic analyses revealed that PSMD12 was significantly upregulated in tumor tissues because of DNA copy number gain. High PSMD12 expression was associated with poor prognosis and advanced pathological stages. Gene set enrichment analysis of TCGA LUAD dataset demonstrated that samples with high PSMD12 expression were enriched for cell cycle-related pathways. Using CRISPR-Cas9-mediated PSMD12 knockout and lentivirus-mediated overexpression models, we demonstrated that PSMD12 promoted tumor cell proliferation by accelerating the G2/M cell cycle transition in vitro, and xenograft experiments confirmed its tumor-promoting effect in vivo. Mechanistically, PSMD12 overexpression reduced the ubiquitination of CDK1, a key regulator of mitotic entry. Cycloheximide chase and MG132 assays confirmed that PSMD12 stabilized CDK1 by inhibiting proteasome-mediated degradation. In conclusion, we identified PSMD12 as a novel driver gene and prognostic biomarker of LUAD. PSMD12 promoted LUAD progression by modulating CDK1 ubiquitination and enhancing cell cycle progression. These findings suggest that PSMD12 is a promising molecular target for future LUAD therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb3bc3b62a546392f132deef63cdce1ec580a2a" target='_blank'>
              PSMD12 Overexpression Promotes Lung Adenocarcinoma Progression via Ubiquitin-Proteasome Pathway Dysregulation.
              </a>
            </td>
          <td>
            Yuya Ono, Hajime Otsu, Takaaki Masuda, K. Kosai, Shohei Shibuta, Kosuke Hirose, Takashi Ofuchi, Yuki Ando, Koto Kawata, Yasuo Tsuda, Yusuke Yonemura, Taro Tobo, T. Takenaka, T. Yoshizumi, Koshi Mimori
          </td>
          <td>2026-01-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hereditary paragangliomas (PGLs) caused by germline SDHD pathogenic variants (PVs) exhibit a parent‐of‐origin effect, with tumors arising almost exclusively when the PV is inherited from the paternal allele. The Hensen model proposes that a cluster of maternally expressed tumor suppressor genes (TSGs) on chromosome 11p15.5 may play a crucial role in SDHD‐related PGL tumorigenesis, wherein somatic loss of maternal 11p and wild‐type SDHD allele, in conjunction with a paternally inherited SDHD PV, triggers tumor development. To systematically localize and identify the most crucial maternal‐expressed TSGs within 11p15.5, we developed a novel single nucleotide variant (SNV)‐oriented, capture‐based targeted enrichment approach followed by next‐generation sequencing (NGS) to enable high‐resolution loss‐of‐heterozygosity (LOH) analysis. Among 13 SDHD‐related PGLs and 23 non‐SDHD‐related PGLs, a somatic loss of 11p15.5‐15.4 was detected in 92% and 47%, respectively, a significant difference (p = 0.0035). Parental genotype analysis confirmed that the lost chromosome was of maternal origin. In our studies, 12/13 SDHD‐related tumors demonstrated complete loss of the maternal 11p15.5‐15.4 region, preventing localization of a specific driver TSG. Only one exceptional SDHD‐related tumor retained this region, warranting further investigation into the mechanism underlying parent‐of‐origin tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27b365a662db25feb9efe53f1684d83f7862183" target='_blank'>
              A Novel Targeted Sequence for Chromosome 11p15.5 Maternal Loss in SDHD‐Related Paragangliomas
              </a>
            </td>
          <td>
            Huei-Pin Lai, Chen-Hui Lee, Hsuan Hu, An-Ko Chung, Pei-Lung Chen, Wei-Shiung Yang, Wan-Chen Wu
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Micronuclei are formed during cell division when acentric fragments or lagging chromosomes cannot be incorporated into the primary nucleus. Macroautophagy/autophagy may reduce chromosomal instability (CIN) by clearing isolated, atypical micronuclei. Other studies implicate that the loss of autophagy disrupts DNA repair pathways. However, whether aberrant mitosis contributing to CIN occurs when autophagy is inhibited has yet to be evaluated. We found impaired autophagy initiation contributes to CIN and facilitates the formation of micronuclei and other abnormal nuclear phenotypes either by genetic or pharmacological manipulation in multiple cell lines. We also found that loss of the integral autophagy protein ATG9A resulted in various types of mitotic errors that can contribute to the formation of micronuclei. ATG9A also localizes to centrosomes and midbody during cell division. Autophagy inhibition causes the overactivation and mislocalization of TBK1 (TANK binding kinase 1) into cytoplasmic, punctate structures that colocalize with SQSTM1/p62. This overactivation interferes with its function in cell division as a mitotic kinase and its role at the centrosome. These results indicate that loss of autophagy contributes to genomic instability from multiple angles, one of which being aberrant cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f7652d445dff3bc9d16db0af8750b841d562ae2" target='_blank'>
              Disrupted autophagy overactivates TBK1 and results in mitotic defects promoting chromosomal instability.
              </a>
            </td>
          <td>
            Swagatika Paul, Porter L Tomsick, Julia P. Milner, S. Biswas, Samantha Brindley, Nicole DeFoor, Leila Zavar, G. Wright, Yairis Soto, A. Pickrell
          </td>
          <td>2026-01-16</td>
          <td>Autophagy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V. Semenov, Irina S. Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, L. Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Chromosome 19 is the most gene-dense chromosome in the human genome, with a high frequency of segmental duplications that predispose it to genomic rearrangements. While deletions of chromosome 19 have been associated with various clinical conditions, duplications remain poorly characterised. Here, we report three cases involving 19p13.11 duplication and describe the associated clinical phenotype. Methods We describe three unrelated individuals with microduplications at 19p13.11 identified either via clinical whole-exome sequencing or chromosomal microarray. The probands underwent detailed clinical genetic evaluations, and CNVs were confirmed with parental testing when available. Sequencing reads were aligned to the GRCh37/hg19 human genome build. Results All three probands exhibited neurodevelopmental delays, attention-deficit/hyperactivity disorder and speech delay. Additional overlapping features included joint hypermobility, short stature and craniofacial anomalies. Patient-specific manifestations included haematological abnormalities, musculoskeletal asymmetries and cardiac findings. Duplicated regions spanned 1.2–1.6 Mb and encompassed 41–49 protein-coding genes. Patients 2 and 3 have CNVs that overlap 76% with those of Patient 1. Several genes have predicted high triplosensitivity scores and are associated with autosomal dominant neurodevelopmental and skeletal disorders. Patient 1, with the largest duplication, had more extensive systemic involvement, likely reflecting the broader gene dosage effect. Conclusion This is the first comprehensive clinical and molecular characterisation of 19p13.11 duplications, suggesting a recurring multisystem phenotype driven by gene dosage sensitivity. These findings support the inclusion of 19p13.11 duplications in diagnostic evaluations for neurodevelopmental and multisystem disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5b04514949e9e7f26f47e17e95907ce73d74d7a" target='_blank'>
              Clinical manifestations of chromosome 19p13.11 duplication
              </a>
            </td>
          <td>
            Dibyendu Dutta, Megan A Keeney, Nicole Matthews, Kristina Peron, Abdulrazak Alali, Ariel F Martinez, Bobbi McGivern, Claire Boring, Scott C. Smith, Ria Garg
          </td>
          <td>2026-01-16</td>
          <td>Journal of Medical Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How retrotransposon mobilization that can be retained for genome evolution persists despite negative outcomes of retrotransposon activity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR retrotransposon nomad completes the full mobilization cascade (including mRNA export, protein translation, and reverse transcription) to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings show that retrotransposon-derived DNA largely remains as ecDNA. This could prevent widespread genomic integration during spermatogenesis, potentially preserving genome stability with the presence of limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9420ae83c1dc149dca66d3f18d2a4cfd87487588" target='_blank'>
              Retrotransposon activation during spermatogenesis achieves massive ecDNA biogenesis but rare integration.
              </a>
            </td>
          <td>
            Lauren Tracy, Yao Chen, Z. Zhang
          </td>
          <td>2026-01-22</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cellular differentiation is driven by epigenetic modifiers and readers, including the methyl CpG binding protein 2 (MeCP2), whose level and mutations cause the neurological disorder Rett syndrome. During differentiation, most of the genome gets densely packed into heterochromatin, whose function has been simplistically viewed as gene silencing. However, gene expression changes reported in mutations leading to Rett syndrome have failed to be a predictor of disease severity. Here we show that MeCP2 increases nuclear stiffness in a concentration-dependent manner and dependent on its ability to cluster heterochromatin during differentiation. MeCP2-dependent stiffness increase could not be explained by changes in the expression of mechanobiology-related genes, but we found that it is disrupted by Rett syndrome mutations and correlated with disease severity. Our results highlight the impact of chromatin organization on the mechanical properties of the cell as an alternative or complementary mechanism to changes in cytoskeleton components. A correlative AFM–fluorescence approach elucidates how epigenetic control of chromatin organization contributes to nuclear stiffness and its dysregulation in Rett syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d330823c212a15c08be21ea0eb068997201f78c8" target='_blank'>
              MeCP2-driven chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Andreas Zhadan, Marah Mahmoud, Nina Trautwein, Bodo Laube, Christian Dietz, R. Stark, Cristina Cardoso
          </td>
          <td>2025-12-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="N4BP1 specifically degrades a subset of mRNA targets through their coding sequences and functions as a negative regulator of inflammation; however, its role in cancer development remains undefined. N4BP1 exhibits the highest expression in head and neck squamous cell carcinoma among all analyzed cancer types. Unlike wild-type mice, N4bp1−/− mice did not develop visible tongue tumor masses in a 4-NQO-induced oral carcinogenesis model. Furthermore, N4bp1−/− mice (86% vs 0%) exhibited significantly prolonged survival compared to wild-type mice within 26 weeks in 4-NQO-induced oral carcinogenesis model. Single-cell profiling demonstrated that N4BP1-deficient epithelial cells arrest at an early stage of cancerous transformation, while wild-type epithelial cells efficiently progress to an advanced stage of cancer. In established human cancer cell lines, N4BP1 also plays a crucial role in proliferation, migration, colony formation, and in vivo growth. Transcriptome profiling identified CCL2 and GM-CSF as downstream targets of N4BP1 in oral cancer. Apart from its intrinsic role in cancer cells, N4BP1-deficient cancer cells induce the differentiation of macrophages into the M1 phenotype. In N4BP1-deficient tissues, CCL2 and GM-CSF were significantly increased, accompanied by the accumulation of M1 macrophages and neutrophils. Our results demonstrate that N4BP1 is an essential gene in tongue cancer development. N4BP1 not only drives cancer cell evolution but also establishes an immune-suppressive microenvironment. N4BP1 is an endoribonuclease that specifically regulates a subset of mRNA targets (including CCL2 and GM-CSF) and plays an essential role in oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/453fe4fbe3e2a6087004d4f6f19727a6b189bb9b" target='_blank'>
              N4BP1 is essential for the development of oral cancer via controlling both cancer cells and immune microenvironment
              </a>
            </td>
          <td>
            Yihua Song, Rong Sun, Jie Ji, Wen Zheng, Yanli Li, Xiaohong Guo, Liuting Chen, Yuanyuan Wu, Miaomiao Chen, Xingmei Feng, Mingbing Xiao, Renfang Mao, Yihui Fan
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/315f53b4c0a6b2f6712e27d1e37b4dc249ec8768" target='_blank'>
              Tumor Suppression across the Human miRNome Associates with Guanine-Rich Precursor Terminal Loops
              </a>
            </td>
          <td>
            Amit Cohen, M. Burgos-Aceves, Yoav Smith
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, Kevin Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Foxp3+ regulatory T (Treg) cells rely on DNA demethylation to establish and maintain the gene expression program that defines their identity and suppressive function. Although sustained transcription of Treg-specific genes is known to drive this demethylation, the precise mechanism has remained unclear. Here, we show that Dot1L-catalyzed methylation of histone H3 at lysine 79 (H3K79me) is essential for Treg-specific DNA demethylation in both thymic and induced Treg lineages. H3K79me promotes chromatin activation and recruits TET family DNA demethylases to key regulatory loci. Treg-restricted deletion of Dot1L disrupts locus-specific DNA demethylation, diminishes expression of core Treg genes, and precipitates a fatal, early-onset autoimmune syndrome. Conversely, pharmacologic enhancement of TET activity during differentiation rescues DNA demethylation, restores Treg-gene expression, and reinstates suppressive function even in the presence of Dot1L inhibition. Together, these findings identify a critical epigenetic axis—Dot1L-mediated H3K79 methylation driving TET-dependent DNA demethylation—that safeguards Treg cellular identity, function, and immune homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/542929f5fc79d680747f50c5a7526c3c0b1a5dde" target='_blank'>
              Dot1L licenses DNA demethylation to establish regulatory T cell identity
              </a>
            </td>
          <td>
            James Cameron, Tyler R. Colson, Xudong Li
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immunotherapies have transformed cancer treatment by harnessing the immune system to recognize and eliminate malignant cells, offering durable clinical benefit across diverse tumor types. Despite successes with immune checkpoint inhibitors (ICIs) and other strategies like cytokines, oncolytic viruses, cancer vaccines, bispecific antibodies, and adoptive cell therapies, substantial fractions of patients still fail to respond or develop resistance. The oncogene MYC, deregulated in ~70% of human cancers, has emerged as a central driver of immune evasion and a key contributor to immunotherapy failure. MYC regulates broad transcriptional networks controlling proliferation, metabolism, angiogenesis, and cell survival, while also orchestrating profound remodeling of the tumor microenvironment (TME). Mechanistically, oncogenic MYC suppresses antigen processing and presentation, attenuates interferon signaling, and upregulates immune checkpoints such as PD-L1 and CD47. Concurrently, MYC stimulates secretion of immunosuppressive cytokines and chemokines that recruit regulatory T cells, myeloid-derived suppressor cells, and M2 macrophages, while driving metabolic reprogramming that fosters nutrient competition, hypoxia, and acidosis, impairing effector T- and NK-cell function. Through these pathways, MYC promotes primary, adaptive, and acquired resistance to immunotherapy. Targeting MYC, directly or indirectly, holds promise to restore immune surveillance and potentiate immunotherapeutic efficacy. This review highlights MYC as a master regulator of tumor–immune interactions and underscores the therapeutic potential of MYC inhibition to overcome resistance and expand the clinical impact of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437ab7ab32dc49430fcd2b151a4005d089060a97" target='_blank'>
              MYC at the tumor–immune interface: mechanisms of immune escape and immunotherapy resistance
              </a>
            </td>
          <td>
            Íñigo González-Larreategui, Manrique Valdés-Bango Martín, S. Casacuberta-Serra, Laura Soucek
          </td>
          <td>2026-01-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ea2c9c7605b557b8e4ec203f1280961cf9800c" target='_blank'>
              Exploring the oncogenic mechanism of plasmacytoma variant translocation 1 (PVT1) gene in solid cancers; emphasis on microRNA regulation pathways.
              </a>
            </td>
          <td>
            Mojtaba Zehtabi, Shirin Azizidoost, Abdolah Mousavi Salehi, Mahrokh Abouali Gale Dari, Mohammad Amin Zargar, Maryam Khombi Shooshtari, Krzysztof Data, Dominika Domagała, Julia Niebora, Małgorzata Józkowiak, P. Chmielewski, Aleksandra Partyńska, Aleksandra Górska, A. Bryja, M. Kulus, Adam Kamiński, Teresa Wysocka, Marek Spaczyński, Piotr Dzięgiel, P. Mozdziak, Maryam Farzaneh, B. Kempisty
          </td>
          <td>2026-01-29</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT HIV-1 viral core transport to the nucleus, an early infection event, triggers the redistribution of cleavage and polyadenylation specificity factors (CPSF) 5 and CPSF6 to nuclear speckles, forming puncta-like structures. CPSF5 and CPSF6 regulate alternative polyadenylation (APA), which governs approximately 70% of gene expression. APA alters the lengths of mRNA 3’-untranslated regions (3′-UTRs), which contain regulatory signals influencing RNA stability, localization, and function. We investigated whether HIV-1 infection–induced changes in CPSF5 and CPSF6 subcellular localization are accompanied by APA changes. Using two independent methodologies to assess APA in human cell lines and primary CD4+ T cells, we found that HIV-1 infection regulates APA, shaped by the interaction of CPSF6 with the viral capsid, recapitulating the APA phenotype observed in CPSF6 knockout cells. Our study demonstrates that HIV-1 infection leverages the interaction between the viral capsid and CPSF6 to co-opt cellular processes, alter gene expression, and potentially contribute to viral pathogenesis. IMPORTANCE The interaction between HIV-1 and the cellular protein CPSF6 has been known for over 15 years; however, depletion of CPSF6 does not impair productive infection. An alternative possibility is that the virus exploits this protein to modulate cellular processes. This study demonstrates that HIV-1 infection alters the cellular function of CPSF6, an essential regulator of alternative polyadenylation—a mechanism that controls 70% of gene expression. Here, we show that HIV-1 regulates gene expression by disrupting the alternative polyadenylation function of CPSF6 through direct interaction. Overall, this reveals a novel strategy employed by the virus to modulate cellular gene expression. The interaction between HIV-1 and the cellular protein CPSF6 has been known for over 15 years; however, depletion of CPSF6 does not impair productive infection. An alternative possibility is that the virus exploits this protein to modulate cellular processes. This study demonstrates that HIV-1 infection alters the cellular function of CPSF6, an essential regulator of alternative polyadenylation—a mechanism that controls 70% of gene expression. Here, we show that HIV-1 regulates gene expression by disrupting the alternative polyadenylation function of CPSF6 through direct interaction. Overall, this reveals a novel strategy employed by the virus to modulate cellular gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9341535e23a1ea4edd63fd87e99760612b583cb" target='_blank'>
              HIV-1 infection regulates gene expression by altering alternative polyadenylation correlated with CPSF6 and CPSF5 redistribution
              </a>
            </td>
          <td>
            Charlotte Luchsinger, Annie Zhi Dai, Hari Yalamanchili, Aiswarya Balakrishnan, Kai-Lieh Huang, Cinzia Bertelli, Bin Cui, Ramon Lorenzo-Redondo, Eric J. Wagner, Felipe Diaz-Griffero
          </td>
          <td>2025-12-17</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Clinical genomic sequencing has revealed that prostate cancer (PCa) progression is associated with changes to chromosome number and structure, termed chromosomal instability (or CIN). While mutations in common driver genes are rare in primary PCa, CIN is detected in >90% of cases, suggesting that CIN may be a driver of the disease. However, the underlying mechanisms that cause CIN in PCa are poorly understood. Cells are susceptible to CIN during mitosis, when the duplicated genome must be equally segregated into two daughter cells. Normally, two centrosomes (cytoplasmic, non-membranous organelles) nucleate and organize microtubules to guide assembly of the mitotic spindle, thus ensuring the fidelity of cell division. However, we recently discovered that cells within primary PCa lack centrosomes, the first cancer type to show this phenomenon. Furthermore, we showed that centrosome elimination in non-tumorigenic cells generates CIN which can transform these cells, causing them to form xenograft tumors in mice. Therefore, we sought to understand the underlying mechanisms that drive the centrosome loss observed in PCa. The prostate is a naturally hypoxic organ, averaging around 4% oxygen tension in the healthy organ which declines as men age. Additionally, tumor hypoxia is strongly correlated with high CIN, particularly in prostate cancers. However, there are few mechanistic links between hypoxia and CIN. Therefore, we hypothesized that hypoxia could cause centrosome loss and, thus, predispose cells to CIN. Indeed, we found that immortalized prostate RWPE-1 cells progressively lost centrosomes when exposed to 1% O2 for as short as 6 hours in vitro. Using ultrastructure expansion microscopy (U-ExM), we find centrosomes are replaced by non-functional microtubule husks devoid of centrosome proteins which we call ‘remnants’. These phenomena also occur in PC-3 and LNCaP prostate cancer lines. Furthermore, we observe centrosome loss specifically in the hypoxic regions of xenograft tumors. Using a combination of RNAseq and chemical inhibitors, we determined that centrosome elimination sits at the crossroads of 3 transcriptional programs: (1) centrosome elimination occurs in basal cells and basal-like cancers that express p63; (2) cells must be confluent with active Hippo pathway; and (3) hypoxia induces a HIF-independent transcriptional response. When these conditions are met, cells transcriptionally downregulate 4 key centrosome proteins (Cep63, Cep152, Cep192, and Cep215), resulting in the depletion of a protective shell of pericentriolar material (PCM) and subsequent centrosome elimination. By overexpressing the mitotic kinase Polo-like kinase 1 (PLK1), we can strengthen the PCM and prevent elimination. Finally, we identify PCa cells containing remnants within prostatectomy tissue samples. This work is significant as it identifies the first examples and mechanisms of centrosome elimination in human somatic and cancer cells, provides a mechanistic link between hypoxia and CIN, and identifies remnants as a cellular indicator for CIN predisposition.



 John M. Ryniawec, Natalya K. Seppanen, Gregory C. Rogers, Anne E. Cress. Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0fec5efccdcc580bc977b0a1bb739d683cd475" target='_blank'>
              Abstract B065: Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer
              </a>
            </td>
          <td>
            John M. Ryniawec, Natalya K. Seppanen, Gregory C Rogers, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 844


 Background:
 Homologous recombination deficiency (HRD) scores quantify genomic instability based on chromosomal alterations such as loss of heterozygosity, telomeric allelic imbalance, and large scale transitions. HRD is emerging as a potential biomarker in gastrointestinal (GI) cancers, correlating with sensitivity to DNA damaging agents, immune response modulation, and clinical outcomes. ARID1A, a chromatin remodeling gene frequently altered in GI malignancies, may play a prognostic role, although its interaction with HRD status remains unclear.
 Methods:
 Using The Cancer Genome Atlas (TCGA), we analyzed 7 individual GI cancers: colon (n=271), rectum (n=84), stomach (n=396), pancreatic (n=154), hepatocellular carcinoma (n=343), cholangiocarcinoma (n=36), and esophageal carcinoma (n=161). HRD scores were dichotomized at the median into HRD-high and HRD-low groups. ARID1A expression was similarly dichotomized and analyzed within each cancer type, adjusting for HRD status using multivariable Cox proportional hazards models. Hazard ratios (HRs) and 95% confidence intervals (CIs) were computed for overall survival.
 Results:
 In hepatocellular carcinoma, HRD high status was associated with significantly worse overall survival (HR 4.26; 95% CI 1.04–17.3; p=0.043). In contrast, pancreatic adenocarcinoma showed improved survival with HRD-low status (HR 0.61; 95% CI 0.38–0.97; p=0.039). Other tumor types, including colon, rectum, stomach, cholangiocarcinoma, and esophageal carcinoma, showed no significant differences in survival based on HRD status. When adjusting for HRD, ARID1A expression level was not significantly associated with overall survival in any GI cancer subtype. Hazard ratios ranged from 0.72 in colon adenocarcinoma to 1.58 in cholangiocarcinoma, but none reached statistical significance. In hepatocellular carcinoma, ARID1A-high expression was nearly absent, precluding meaningful survival analysis.
 Conclusions:
 HRD status exhibits tumor specific prognostic associations in GI cancers. HRD-high status appears deleterious in hepatocellular carcinoma, while HRD-low status may be protective in pancreatic cancer. ARID1A expression, however, does not independently impact survival when HRD is accounted for. These findings support further exploration of HRD as a prognostic and potentially predictive biomarker in GI malignancies and suggest limited utility for ARID1A expression alone in prognostication.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6eb30cbe8cc66aa414ce56255ebc5a4b2c61ca8" target='_blank'>
              Prognostic impact of HRD status and ARID1A expression in individual gastrointestinal cancers.
              </a>
            </td>
          <td>
            Pragya Jain, Nency Ganatra, Shivam Patel, Manan Patel, Muhammad A. Khalil
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 ZFTA-RELA (ZR) is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite the oncogenic potential of ZR, it is observed nearly exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZR.



 To this end, we performed combined single nucleus ATAC and RNA (snMultiome) sequencing of the developing mouse forebrain, as compared to ZR-driven mouse and human EPN.



 We demonstrate that specific developmental lineage programs present in transient progenitor cells and regulated by PLAG/L family transcription factors (TF) are at risk of neoplastic transformation. Binding of this chromatin network by ZR or other PLAG/L family motif targeting fusion oncoproteins lead to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human ZR EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling progenitor-like or radial glial-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy.



 These findings unravel developmental epigenomic states critical for fusion oncoprotein-driven transformation and elucidate how these states continue to shape tumor progression.



 Alisha Kardian, Hua Sun, Stephen Mack. Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885b82368395c60d63823ec56b4ae77477004d30" target='_blank'>
              Abstract LT004: Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            A. Kardian, Hua Sun, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Papillary renal cell carcinoma (pRCC) is characterized by marked intratumoral heterogeneity, which contributes to therapeutic resistance and disease progression. In this study, we identify STK38 as a key regulator of tumor heterogeneity in pRCC, functioning through non-canonical activation of the Hedgehog (Hh) signaling pathway. STK38 interacts with both KIF7 and GSK3β to promote Hh signaling by facilitating KIF7 ciliary localization and reprogramming GSK3β substrate selectivity, leading to GLI1 stabilization and β-catenin suppression. Moreover, GLI1 directly enhances STK38 transcription, establishing a positive feedback loop that reinforces pathway activation. Notably, depletion of STK38 sensitizes tumor cells to a NETosis-like chromatin release process (tNET release), a form of stress-induced nuclear expulsion associated with immune evasion and metastatic potential. Given the potential pro-metastatic consequences of STK38 inhibition, we instead targeted its downstream effector GLI1 using Glabrescione B, which potently suppressed tumor growth and induced apoptosis in both xenograft and patient-derived organoid models, particularly in STK38-high tumors. These findings position STK38 as a critical modulator of pRCC heterogeneity and support GLI1 inhibition as a promising strategy to disrupt oncogenic signaling while minimizing adverse effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a249254f9dd46bc3c20e8d81598af9b9ed7c0662" target='_blank'>
              STK38-mediated feedback loop regulation of the hedgehog pathway governing tumor heterogeneity in renal papillary carcinoma
              </a>
            </td>
          <td>
            Yifan Du, Xiuyuan Sui, Zeyuan Zheng, Zhengying Zhang, Bin Liu, Yang Bai, Yue Zhao, Qingqing Wu, Haodong Wu, Min Zhong, Liyan Li, Huimin Sun, Chen Shao
          </td>
          <td>2026-01-15</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The field of epitranscriptomics discovered N6-methyladenosine (m6A) modifications, which function as fundamental elements that control RNA metabolism properties that powerfully affect cancer biology. This review examines the way m6A modifications shape RNA stability while regulating translation, together with their eraser and reader proteins. We demonstrate that m6A modifications guide oncogene and tumor suppressor transcript outcomes, which promote tumor growth, metastasis, and therapeutic resistance. The regulatory function of m6A depends significantly on its relationship with ncRNAs that mainly include miRNAs, lncRNAs, and circRNAs. The review examines the effects of m6A on ncRNA production, stability, export, and degradation, as well as the regulation of m6A protein expression by ncRNAs, highlighting intricate reciprocal feedback loops that drive cancer progression. The interplay between m6A RNA modifications and ncRNAs provides emerging evidence on how they collectively influence the tumor microenvironment, modulate immune system responses, and contribute to resistance. Harnessing ncRNA-m6A interactions for managing drug resistance offers promising therapeutic avenues. However, advancing our understanding of the context-specific roles of m6A modifications and translating these insights into clinical applications remains a significant challenge. This review synthesizes recent findings on ncRNA-m6A crosstalk to lay the groundwork for developing epitranscriptomic strategies in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e0eb5d331194e0274cc28e14543827ba3e7f5f" target='_blank'>
              The epitranscriptome meets non-coding RNA: m6A-mediated regulation in oncogenesis and therapy
              </a>
            </td>
          <td>
            Prasanna Srinivasan Ramalingam, M. Rejili, Faouzi Haouala, Md Sadique Hussain, Yumna Khan, Mudasir Maqbool, Janaki Ramaiah Mekala, S. Arumugam
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b68b7fa61f9edb29d61947a73ec839e1cdd7c8" target='_blank'>
              CRISPR-Cas9 technology: a breakthrough in cancer gene therapy
              </a>
            </td>
          <td>
            Z. I. Jameel
          </td>
          <td>2026-01-12</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Cancer progression involves dynamic crosstalk between tumor-intrinsic pathways and microenvironmental remodeling, and identifying pan-cancer biomarkers is critical for precision oncology.
 CDKN2AIPNL
 exhibits a paradoxical role in cancer, acting as a tumor suppressor in myeloid malignancies but promoting solid tumor progression, yet its systematic pan-cancer characteristics remain unelucidated. This study aimed to comprehensively analyze
 CDKN2AIPNL’s
 expression patterns, prognostic value, genetic alterations, and molecular mechanisms across multiple tumor types using public datasets including TCGA, GTEx, HPA, and tools such as GEPIA2, cBioPortal, TIMER2, STRING, and BioGRID. We performed expression difference analysis, survival analysis (overall survival, disease-free survival, progression-free survival), genetic alteration analysis, cancer-associated fibroblast (CAF) infiltration analysis, and gene/protein interaction enrichment analysis. Results showed that CDKN2AIPNL was significantly upregulated in multiple tumors (e.g., LIHC, UVM, BRCA, LUAD) and downregulated in others (e.g., KICH, KIRP, THCA), with high tumor specificity. Elevated CDKN2AIPNL expression correlated with poor overall survival in LIHC (HR = 1.7, p = 0.0026), UVM (HR = 26, p = 2.2e-6), BRCA (HR = 26, p = 2.2e-6), LUAD (HR = 1.36, p = 0.049), PCPG (HR = 1.71, p = 0.0012), and TGCT (HR = 0.37, p=0.023), and was associated with advanced tumor stages in metabolically active cancers. Genetic alterations (amplifications and mutations) were frequent in KIRC (>5%) and ACC (>4%), with all mutations localized to the XTBD region, and amplification predicted poor prognosis in PRAD (p = 0.008) while mutations conferred favorable outcomes in BLCA. CDKN2AIPNL expression positively correlated with CAF infiltration in ESCA, KICH, UVM, and other tumors, and interacted with MYC, XRN2, and CHAMP1 to regulate metabolic reprogramming, cell cycle, and immune suppression. Our findings systematically reveal CDKN2AIPNL’s dual role in tumorigenesis and validate it as a potential pan-cancer prognostic biomarker, providing novel insights for cancer diagnosis and targeted therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1804517cd011102843d1d1b41c7599a28bf25c60" target='_blank'>
              CDKN2AIPNL: a potential pan-cancer biomarker
              </a>
            </td>
          <td>
            Yulin Yuan, Shengjie Ma, Heshi Liu, Yang Gong, Xuan Sun, Quan Wang, Weifu Zhang
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Chromosomal aberrations are frequent in cancer and play a pivotal role in disease initiation and progression. In prostate cancer (PCa), deletions at chromosome 13q14 are common and enriched in advanced disease, suggesting selective advantage in tumor progression. Beyond BRCA2, this region harbors other tumor suppressor genes, including RB1 and RNASEH2B, which impact sensitivity to androgen receptor inhibitors (ARi) and PARP inhibitors (PARPi). This study aims to elucidate the effect of individual vs concomitant loss of RB1 and RNASEH2B in response to dual AR and PARP inhibition.



 To explore the prevalence of deletions in 13q we interrogated copy number alterations in the Hartwig Medical Foundation (HMF) PCa WGS dataset (n=486). In vitro, we generated CRISPR-Cas9-mediated knock-outs (KO) of RB1, RNASEH2B, or both genes in PCa cell lines (LNCaP, C4-2, 22Rv1, DU145). Gene editing was confirmed by Sanger, and western blot analysis validated protein loss. Antitumor activity of enzalutamide (ARi), and olaparib or saruparib (PARPi) was determined using CCK8 kit and colony formation assays.



 RB1 deletions were present in 42% of HMF PCa samples (205/486), being mostly heterozygous deletions (200/205). RB1 deletions co-occurred with RNASEH2B loss in 94% of cases (192/205) and with BRCA2 deletions in 42% of cases (87/205). In vitro, RB1-KO increased resistance to ARi and PARPi in hormone-sensitive lines LNCaP and C4-2, while no change in sensitivity was detected in AR-independent models 22Rv1 and DU145. Loss of RNASEH2B increased sensitivity to PARPi in all models. Next, we generated RB1 and RNASEH2B double KO models in LNCaP and C4-2 by sequentially editing single KO populations. After single-cell seeding, we obtained: (1) RB1-KO (homozygous) clones with complete protein loss; (2) RNASEH2B-heterozygous (RNASEH2B-Het) models with partial protein expression; and (3) RB1-KO/RNASEH2B-Het models. No clones with complete RNASEH2B loss were obtained, suggesting that RNASEH2B-loss population is transient and that clones with complete loss do not proliferate. RB1-KO models maintained resistance to both treatments. RNASEH2B-Het cells showed increased sensitivity to PARPi, although to lower extent than the population with complete RNASEH2B loss. Importantly, introducing RB1-loss in RNASEH2B-Het cells restored PARPi resistance, and combined AR/PARP inhibition did not overcome resistance in RB1-KO or RB1-KO/RNASEH2B-Het cells.



 Our findings indicate that 13q14 deletions in PCa predominantly occur as heterozygous events encompassing key genes influencing ARi and PARPi responses. Heterozygous loss of RNASEH2B confers sensitivity to PARPi, although to a lesser extent than in models with complete RNASEH2B loss. RB1-loss confers higher resistance to ARi and PARPi, restoring resistance to PARPi in RNASEH2B heterozygous models. These results highlight the need for models that better mirror the heterozygous nature of 13q14 deletions to clarify their biological effects and refine therapeutic strategies for advanced PCa.



 Victor Esquefa, Pablo Cresta Morgado, Richard Norris, Lara de Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel Ramos, Daniel Aguilar, Gisela Mir, Irene Casanova-Salas, Francisco M. Barriga, Joaquin Mateo. Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15430fc4c6c79184273233030845f22a6f4ecd5a" target='_blank'>
              Abstract B018: Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents
              </a>
            </td>
          <td>
            Victor Esquefa, P. C. Morgado, Richard Norris, L. D. De Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel J. Ramos, Daniel Aguilar, Gisela Mir, I. Casanova-Salas, Francisco M. Barriga, Joaquin Mateo
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Objectives Gastric cancer (GC) remains a major global health concern, and Phosphoinositide-3-Kinase Regulatory Subunit 1 (PIK3R1), a regulatory subunit of the PI3K signaling pathway, may play a critical yet underexplored role in GC progression. This study aimed to investigate the prognostic significance of PIK3R1 in GC and its association with the tumor immune microenvironment. Methods PIK3R1 expression and its clinical relevance were analyzed using datasets from GC patients who underwent gastrectomy, including cohorts from The Cancer Genome Atlas (TCGA) and the Sun Yat-sen University Cancer Center (SYSUCC). Prognostic models integrating PIK3R1 expression with clinical parameters were constructed for both cohorts. The immune microenvironment associated with PIK3R1 expression was assessed through immunohistochemistry and single-cell RNA sequencing. In vitro assays were conducted to evaluate the effects of PIK3R1 on GC cell proliferation and migration. Results PIK3R1 was significantly overexpressed in GC tissues and was closely associated with aggressive tumor characteristics and poor clinical outcomes. A nomogram combining PIK3R1 expression with clinicopathological features effectively predicted patient prognosis. Knockdown of PIK3R1 in GC cells reduced proliferation and migration in vitro. Immunological profiling revealed that high PIK3R1 expression correlated with increased infiltration of forkhead box protein P3 (Foxp3+) and cluster of differentiation 73 (CD73+) T cells. Patients with low PIK3R1 expression and low CD73+ T cell infiltration had significantly better survival. Conclusions PIK3R1 overexpression is linked to poor prognosis in GC and influences the extent of immune cell infiltration within the tumor microenvironment. A novel prognostic model integrating PIK3R1 and CD73 expression with clinical parameters was established to stratify GC patients into distinct risk groups, offering potential value for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee81bdbf2e1928add8592ede0221b699e416daa" target='_blank'>
              PIK3R1 as a Gastric Cancer Biomarker Linked to CD73+ Treg-Mediated Immunosuppression
              </a>
            </td>
          <td>
            B. Zou, Yi-En Xu, Hui-Chan He, Zulu Ye, Da-Lei Zhou, Caiyun He, Chan Huang
          </td>
          <td>2026-01-19</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            Emily C. Williams, Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c6e2a39a6240d55a8cd9334672f5c4459bbc0c" target='_blank'>
              A compendium of chromatin interaction maps in the Giant Panda genome.
              </a>
            </td>
          <td>
            Pengliang Liu, Jiaman Zhang, Kailai Cai, Juan Wang, Hong Liu, Liang Zhang, Wei Xu, Yuliang Liu, Fujun Shen, Rong Hou, Yan Li
          </td>
          <td>2026-01-13</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 In Non-Small Cell Lung Cancer (NSCLC), low-fidelity transcription mechanisms such as ecDNA (extra-chromosomal DNA) are an increasingly relevant aspect of progression and prognosis, warranting comprehensive investigation into stability and prevalence of chimeric RNA transcripts. Here, we describe global distribution of fusion transcripts in encounter-matched primary tumor and lymph node metastases and their prognostic implications.



 RNAseq was performed on resected FFPE (encounter-matched primary and lymph nodes) from 121 stage IIB-IIIA (T1-4N1-2M0) NSCLC patients, in addition to 693 Stage IA1-IB cases (T1a-2aN0M0). STAR and STAR-Fusion were used to align data to the GRCh37 genome. Differential gene expression was based on specimen source (primary or lymph node). Aggregate analyses were also performed, including determination of a fusion burden metric, assessing the relationship between primary tumors, lymph node metastases, and recurrence. Deconvolution of Tumor Microenvironment was performed via Quantiseq.



 Comparing primary tumors to paired lymph node metastases, 7,930 ensembl IDs were differentially-expressed at FDR < 0.001 (n=101 pairs). A total of 613 fusions (685 unique transcripts) were detected from paired primaries and lymph node metastases (n = 242). Frequency of specific gene fusions in primaries ranged from 0-0.963 (median 0.019), compared to 0-0.870 (median 0.0093) in LN metastases (p=1.87×10-4). We observed fusion of TULP4-RP11-732M18.3 as pervasive in NSCLC, occurring in 84-87% of primary tumors and lymph nodes, with a 91% retention rate. Conversely, fusions involving CCDC7 or SEPT14 were preferentially observed in metastases, where 82% of CCDC7 fusions and 75% of SEPT14 fusions were not observed in the primary tumor despite occurring at high frequency (66% and 18.5%, respectively) among LN metastases. Interestingly, median fusion burden (unique gene-gene fusions) for primaries was 14 compared to 11 (p=2.06×10-7) in LN metastases (median change -4). Among N0 cases that did not receive adjuvant therapy, median fusion burden in non-recurrent tumors was 15 (n=570), compared to 13 in those that recurred over 3-60 months (n=123; p = 0.04). Interestingly, there was no significant difference in fusion burden between recurrent cases and primary tumors with LN metastases (n = 108). For both recurrent cases and LN-paired primaries, the difference in fusion burden compared to LN metastases (n = 110) was significant (p=2.10×10-4 and 2.06×10-7). A subset of LN metastases (n=27) increased in fusion burden relative to the primary tumor; these primaries uniquely exhibited 33% decrease in regulatory T cell content (p=0.008), 20% decrease in M2 macrophage content (p = 0.01 with M1 macrophages static), and 40% decrease in B-cell content (p=0.05).



 Fusion transcripts are pervasive in RNAseq data, suggesting a broader and more variable landscape than accounted for by primary tumors alone. Concurrent ongoing studies explore the role of ecDNA in fusion burden changes related to NSCLC progression and metastasis.



 Kelly M. Cagin, Keri L. Denson, Sierra R. Broad, Wara M. Naeem, Arsalan M. Khan, Michael M. Liptay, Christopher W. Seder, Jeffrey A. Borgia. A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abeb8dbe1ed35e15732a9cc5bda99a8030ed67" target='_blank'>
              Abstract B027: A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs
              </a>
            </td>
          <td>
            K. Cagin, Keri L. Denson, Sierra R. Broad, Wara Naeem, Arsalan M. Khan, Michael M. Liptay, C.W. Seder, Jeffrey Borgia
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole‐genome sequencing (WGS) has greatly expanded researchers' ability to study structural variants (SVs), that is, the variation in the presence, number, orientation or position of a DNA sequence. This has paved the way to study the eco‐evolutionary dynamics of SVs across the tree of life and within a population genomics framework. In this review, we provide the necessary fundamentals to help researchers generate and analyse population‐level SV data. We discuss the unique properties of different SV groups and how these fundamental differences interact with important biological and evolutionary processes using both empirical results and theory. This includes discussion of unresolved issues around SVs, such as technical difficulties in identification, accounting for diversity and evaluating functional effects. We explicitly integrate into this discussion transposable elements, which are an important component of SVs often identified in population‐level variant data. Finally, we focus on the practical side of SV analysis, offering a framework for SV identification and data analysis. In particular, we examine the heterogeneous nature of SV properties (type, length, sequence identity) that should be considered when studying them in ecology and evolution. This review aims to provide resources and guidelines to help researchers navigate the complexities of a relatively new field of eco‐evolutionary genomics research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4e7fdcc4f309779079c0cb9dffdc986139d5472" target='_blank'>
              A Beginner's Guide to Structural Variants in Eco‐Evolutionary Population Genomics
              </a>
            </td>
          <td>
            Katarina C. Stuart, Rebekah A. Oomen, Anna Tigano, Maren Wellenreuther, Jana Wold, David L Field, C. Mérot
          </td>
          <td>2026-01-01</td>
          <td>Molecular Ecology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf472535b902d79ea418f17502b1ecdbe549381" target='_blank'>
              Depletion of BRCA1 Potentiates Progestin-Induced Cytoskeletal Changes in an Ovarian Cancer Cell Model
              </a>
            </td>
          <td>
            Jocelyn Baquier, Noelle E. Gillis, Caroline H. Diep, Laura J. Mauro, Carol A. Lange
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colon cancer (CC) is a malignancy with high global incidence and mortality, and elucidating its underlying molecular mechanisms is critical for improving prognostic assessment and therapeutic strategies. In this study, transcriptomic data from a large cohort of CC samples and a limited number of normal controls from the TCGA database were used to construct a multigene prognostic risk model using univariate Cox and least absolute shrinkage and selection operator (LASSO) regression analyses. The expression of key prognostic genes, including immunoglobulin superfamily member 9 (IGSF9), was further validated in CC tissues by PCR and Western blotting. Functional assays were performed in HCT116 cells to investigate the biological effects of IGSF9 overexpression and its regulatory relationship with p53. The prognostic model identified IGSF9 as a gene significantly associated with patient survival. Although IGSF9 expression was reduced at both the mRNA and protein levels in CC tissues, its overexpression in vitro markedly promoted apoptosis, alleviated DNA damage, and suppressed cell migration and invasion. Notably, silencing of p53 partially reversed the tumor‐suppressive effects induced by IGSF9 overexpression, indicating that IGSF9 exerts its biological functions in a p53‐dependent manner. Collectively, these findings demonstrate that IGSF9 acts as a tumor suppressor in colorectal cancer and regulates DNA damage responses and apoptosis through a mechanism partially dependent on p53, highlighting its potential value as a prognostic biomarker and therapeutic target in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ff548ce8387a244b0397236c92065d62ddf1f1" target='_blank'>
              P53 Inhibition Diminishes IGSF9 Gene Activity to Promote DNA Repair and Exacerbate the Progression of Colon Cancer
              </a>
            </td>
          <td>
            Huan-Yu Zhang, Dan Tian, Wan-Fu Zhang, Ying-Hui Zhang, Jia-Li Feng, Juan Sheng, Xue-Qin Shang
          </td>
          <td>2026-01-01</td>
          <td>Journal of Biochemical and Molecular Toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Polycomb Repressive Complex 2 (PRC2) is an ancient, conserved chromatin-interacting complex that controls gene expression, facilitating differentiation and cellular identity during development. Its regulation is critical in most eukaryotes. EZHIP was recently characterized as a PRC2 inhibitor and ‘oncohistone mimic’ in mammals. Although Ezhip expression is typically restricted to the germline, its aberrant expression in pediatric brain tumors inhibits PRC2-mediated H3K27 methylation and drives disease progression. To gain a deeper understanding of its normal functions, we systematically examined Ezhip evolution across 70 mammals using comparative genomics, synteny analysis, and motif discovery. Bolstering previous work, we find that Ezhip originated and has been strictly retained on the X chromosome in placental mammals. In addition to the highly conserved H3K27M-like histone mimic motif, our motif analysis reveals seven previously unidentified EZHIP motifs, including a putative nuclear localization signal, and tandem repeats that are largely well-conserved in placental mammals, except in some lineages. We hypothesize that these motifs are also critical to EZHIP’s functions, including in PRC2 interaction and inhibition. We show that Ezhip has evolved under strong diversifying selection in primates and underwent dynamic expansions and losses across species. Some paralogs, such as Ezhip2 in primates, also evolved under positive selection. Based on its evolutionary attributes and germ-cell expression, we propose that Ezhip arose and evolved rapidly due to inter-parental conflict over fetal development in utero in placental mammals. Our work provides a foundation for further investigations into EZHIP’s essential, potentially multifaceted roles in mammalian reproduction and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5aaeda0a67409aa497058125c0f7abe2b1f4a7" target='_blank'>
              Dynamic evolution of EZHIP, an inhibitor of the Polycomb Repressive Complex 2 in mammals
              </a>
            </td>
          <td>
            P. Raman, Hana Khan, Janet M. Young, T. Tsukiyama, Harmit S. Malik
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38276b1e4a7d6b2c083fc831ee032111ce1755b" target='_blank'>
              A recently evolved TAF8 isoform arising from an Alu insertion increases TFIID assembly complexity in the human lineage
              </a>
            </td>
          <td>
            Andrea Bernardini, A. Gallo, E. Scheer, Bastien Morlet, D. Dolfini, Roberto Mantovani, Stéphane D. Vincent, László Tora
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id=". Breast invasive carcinoma (BRCA) is the most common type of cancer affecting women worldwide. Biomarkers such as estrogen receptor, progesterone receptor and HER2 are currently utilized in clinical practice for breast cancer diagnosis; yet their sensitivity and specificity remain limited. However, Ras‑GTPase‑activating protein SH3 domain‑binding protein 1 (G3BP1), an RNA‑binding protein, has been implicated in tumor progression in various cancers, yet its clinical relevance and mechanistic role in BRCA still remain unclear. The present study integrated multi‑omics data analysis and experimental validation to address this. G3BP1 mRNA and protein expression levels in BRCA were analyzed using The Cancer Genome Atlas, Tumor Immune Estimation Resource (TIMER) and Clinical Proteomic Tumor Analysis Consortium databases. Kaplan‑Meier survival analysis and Cox regression were employed to evaluate the prognostic value of G3BP1. Gene set enrichment analysis (GSEA) was performed to identify associated signaling pathways and immune cell infiltration correlations were assessed using CIBERSORT and TIMER. Additionally, 38 samples of BRCA and adjacent normal tissue were collected for immunohistochemical (IHC) validation and diagnostic efficacy was evaluated using receiver operating characteristic (ROC) curves. G3BP1 was significantly upregulated in BRCA tissues at both mRNA and protein levels (P<0.05). High G3BP1 expres‑ sion was associated with the advanced cancer lymph node stage (P=0.005) and a poor prognosis [overall survival, disease‑free survival (DFS), distant metastasis‑free survival and post‑progression survival; all P<0.05]. Differential gene analysis identified 67 upregulated and 9 downregulated genes. GSEA revealed G3BP1 enrichment in key pathways such as PI3K/AKT/mTOR signaling and ubiquitin‑mediated proteolysis. Immune analysis showed a significant positive association between G3BP1 and M2 macrophage infiltration (P<0.05">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f27a548b3fa216ea82907fbbff63b24c53e8d51" target='_blank'>
              High expression of G3BP1 is associated with poor prognosis in breast invasive carcinoma
              </a>
            </td>
          <td>
            Jingjing Li, Zhiqiang Zong, Jian Shen, Jiahao Wang, Xuan Zhou, Wei Shi, Jia Li, Hong Zhao, Yunwen Yan, Fanfan Li
          </td>
          <td>2025-12-18</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Fusion oncogenes drive malignant transformation by altering transcriptional programs, disrupting cellular homeostasis, and frequently rewiring protein condensate networks. Emerging evidence suggests that aberrant condensate formation is a key mechanism through which these fusion proteins exert oncogenic control. Specifically, by amplifying transcriptional output, sequestering cofactors, and enforcing pathological gene expression states. Despite their genetic validation as oncogenic drivers, many fusion proteins remain pharmacologically intractable due to the absence of catalytic domains or well-defined binding pockets. To overcome this, we developed an integrated discovery platform that directly assesses the biophysical and functional properties of fusion protein-driven condensates and enables identification of small molecules that disrupt these oncogenic assemblies. Using protein language models and in silico predictors of phase separation, we systematically mapped condensate-forming potential across recurrent oncogenic fusions catalogued in TCGA. This analysis revealed that most fusion proteins (including EML4-ALK and EWS-FLI1) harbor extensive intrinsically disordered regions (IDRs) that promote abnormal condensation and transcriptional dysregulation. We further identified disease-specific variants that alter condensate formation relative to wild-type proteins, providing mechanistic insight into oncogenic gain- and loss-of-function events. To experimentally validate and therapeutically exploit these findings, we established PhaseScan™, a high-throughput droplet microfluidics screening platform capable of quantitatively profiling condensate dynamics and screening thousands of compounds against reconstituted fusion condensates to identify therapeutically relevant small molecule inhibitors. Across multiple fusion proteins, PhaseScan™ identified small molecules that selectively disrupt aberrant condensates while sparing physiological assemblies such as nuclear speckles and stress granules. Using EML4-ALK as a representative model, we demonstrated that compound-mediated condensate dissolution disrupts oncogenic ALK signaling and reverses oncogenic phenotypes in cell-based systems. Our results highlight condensate biology as a generalizable vulnerability across fusion-driven cancers and demonstrate the feasibility of discovering small molecules against fusion proteins traditionally considered undruggable. By integrating AI-driven predictive modeling, and on-target molecular high-throughput screening, our platform provides a roadmap for rationally identifying and drugging fusion protein condensates. This approach establishes condensate-targeted therapeutics as a novel class of precision medicines for patients with fusion-positive malignancies.



 Andrew Seeber, William E. Arter, Seema Qamar, Cinzia Sgambato, Amal Alex, Kadi Saar, Julia Doh, Martin Kulander, Tuomas Knowles, Shilpi Arora. Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee0c93f691d35862b44375fdee303e25769c39e0" target='_blank'>
              Abstract PR004: Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform
              </a>
            </td>
          <td>
            A. Seeber, W. Arter, S. Qamar, Cinzia Sgambato, Amal Alex, K. Saar, Julia Doh, M. Kulander, Tuomas Knowles, Shilpi Arora
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct fragmentomic characteristics that are related with their cellular origins. Here, we report that somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like fragmentomic characteristics, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Further investigations suggest that such cancer-like traits are associated with somatic variants derived from clonal hematopoiesis. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilize AI to integrate genomic, fragmentomic, and epigenomic features to develop diagnostic models named FreeSV and FreeSV+. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features are also included, the AUCs of FreeSV+ model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d931b70db2d8495e7d9b12e6b1f59f47b890a1" target='_blank'>
              Cancer-Like Fragmentomic Characteristics of Somatic Variants in Cell-Free DNA.
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2026-01-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb26a2cdf46ae9c9709cf8355637b0c744219483" target='_blank'>
              Enhanced co-expression of cyclin F and USP7 in luminal A breast cancer correlates with endocrine resistance, high oxidative phosphorylation, and low inflammatory signatures
              </a>
            </td>
          <td>
            Savitha S. Sharma, P. S. Hari, R. Nivedhitha
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3e5486bf9671c3b3ce7043325de2e14e73fe8c9" target='_blank'>
              Connecting mitochondrial metabolism and mitotic fidelity to control vulnerability of high grade serous ovarian cancer patients to taxane-based chemotherapy
              </a>
            </td>
          <td>
            H. Moindjie, Morgane Morin, Evanthia Pangou, Diego Chianese, Cynthia Seiler, S. Rodrigues-Ferreira, Maria M. Haykal, Christine Péchoux, F. Dingli, D. Loew, Y. Kieffer, Géraldine Gentric, Patricia Pautier, Philippe Dessen, R. Argüello, Paolo Pinton, Fatima Mechta-Grigoriou, Catherine Brenner, Massimo Bonora, Clara Nahmias
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="


 Like many cancers, prostate cancer (PCa) relies on tissue- and lineage-specific transcription factors essential for normal tissue function and tumor progression. A key master factor is the androgen receptor (AR), which regulates prostate tissue identity and differentiation, maintaining normal, growth-suppressive, prostate-specific luminal programs via androgen response elements (AREs). However, during tumorigenesis, AR is co-opted to drive oncogenic transcription programs, including reprogramming factors like FOXA1 and HOXB13. Despite AR’s critical role, the mechanisms and functional implications of both its oncogenic and growth-suppressive programs in PCa remain incompletely understood. We hypothesized that the methyl-CpG reader and DNA repair protein MBD4 safeguards lineage trajectories by restraining pioneer factor engagement.



 We performed an epigenetic-focused CRISPR screen (2,508 genes) in an LNCaP ARE-activated model to nominate the regulators of AR programs. We engineered MBD4 knockout and inducible overexpression models and profiled chromatin and transcription by ATAC-seq; CUT&RUN (AR, FOXA1, H3K27ac, H3K4me3, H3K27me3); and RNA-seq across AR+ and AR- lines. Proliferation and lineage-identity programs were also evaluated in normal mouse prostate organoids. Clinical relevance was assessed using multi-cohort patient transcriptomes and cancer-dependency datasets.



 MBD4 emerged as a gatekeeper of the AR/ARE-mediated growth-suppressive program. MBD4 loss increased chromatin accessibility and enhancer acetylation at FOXA1-enriched loci, with FOXA1 binding expanding upon loss and decreasing with MBD4 overexpression. MBD4 localized to AR/FOXA1 enhancers and limited FOXA1 engagement. Functionally, MBD4 knockout accelerated proliferation in AR+ lines and slowed growth in AR- lines (e.g., PC3, DU145). In normal mouse prostate organoids, MBD4 disruption biased luminal epithelial identity programs and reprogrammed enhancer architecture. Across patient cohorts, MBD4 expression and dependency patterns tightly tracked with AR status/lineage, linking lineage context to chromatin plasticity.



 MBD4 acts as a gatekeeper of the AR/ARE-mediated growth-suppressive program by limiting lineage-specific enhancers and restraining FOXA1 engagement. Loss of MBD4 promotes FOXA1-driven reprogramming and lineage plasticity, nominating the MBD4–FOXA1 axis as a potential therapeutic target. AI DISCLOSURE: Generative AI was used to help draft the wording of this abstract; all content was supplied, reviewed, and approved by the authors.



 Xuanrong Chen, Janny Villa-Pulgarin, Jiansheng Wu, Un In Chan, Jude Owiredu, Anjali Yadav, Andrea Sboner, Christopher Barbieri. MBD4 regulates FOXA1 lineage-specific enhancers to promote prostate tumorigenesis and progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/144dd6ef15d0789330238173b369f5abfb772a9e" target='_blank'>
              Abstract A010: MBD4 regulates FOXA1 lineage-specific enhancers to promote prostate tumorigenesis and progression
              </a>
            </td>
          <td>
            Xuanrong Chen, J.A. Villa-Pulgarín, Jiansheng Wu, Un In Chan, Jude Owiredu, Anjali Yadav, A. Sboner, C. Barbieri
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Thyroid hormones (THs) regulate metabolism, proliferation, and genomic stability. Clinical studies have linked levothyroxine therapy with higher Oncotype DX Recurrence Scores in breast cancer (BC), suggesting a potential effect of thyroid hormone signaling on genomic risk. Here, we investigated the impact of triiodothyronine (T3) on DNA damage and repair pathways in estrogen receptor-positive T47D breast cancer and non-tumorigenic MCF10A cells. RNA sequencing revealed significant upregulation of RAD51 and enrichment of DNA repair pathways following 24 h T3 exposure. Consistently, T3 increased γH2AX and 53BP1 nuclear foci, indicating transient activation of the DNA damage response (DDR). These effects were transient, returning to baseline after 48 h, suggesting cellular adaptation. T3 also enhanced proliferation at 10 μM but inhibited growth at higher concentrations. Our findings indicate that acute exposure to T3 induces transient genomic stress, providing a potential mechanistic basis for the observed association between thyroid hormone therapy and increased BC recurrence risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c85003cf8a6d3c7dd5fc690ebfeb5cb1dec0bff6" target='_blank'>
              Thyroid Hormone T3 Induces DNA Damage Response in Breast Cancer Cells
              </a>
            </td>
          <td>
            Sahar Movshovitz, L. A. Sinberger, Keren Trabelsi, Amit Bar-on, A. Sonnenblick, Mali Salmon-Divon, T. Listovsky
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven in part by metabolic reprogramming and the persistence of cancer stem-like cells (CSCs). The seven in absentia homolog 2 (SIAH2)/with-no-lysine kinase 1 (WNK1) signaling axis has emerged as a potential regulator of these processes, yet its functional role in CRC metabolism and tumor–stroma crosstalk remains incompletely understood. Integrated analyses of The Cancer Genome Atlas–Colon Adenocarcinoma (TCGA-COAD) and Gene Expression Omnibus (GEO, GSE17538) datasets revealed significant upregulation of SIAH2 and WNK1 in CRC tissues, with strong positive correlations to glycolysis- and hypoxia-associated genes, including PFKP, LDHA, BPGM, ADH1A, ADH1B, and HIF-1α. Single-cell and clinical profiling further demonstrated preferential enrichment of SIAH2 in undifferentiated, stem-like tumor cell populations. Functional studies across multiple CRC cell lines showed that SIAH2 silencing suppressed proliferation, clonogenic growth, tumor sphere formation, and cell-cycle progression, whereas SIAH2 overexpression exerted opposite effects. Seahorse extracellular flux analyses established that SIAH2 promotes glycolytic capacity and metabolic flexibility. At the protein level, SIAH2 regulated glycolytic enzymes and WNK1/hypoxia-inducible factor-1α (HIF-1α) signaling, effects that were amplified by cancer-associated fibroblast (CAF)-derived conditioned medium. CAF exposure enhanced SIAH2 expression, CSC spheroid growth, and resistance to fluorouracil, leucovorin, and oxaliplatin (FOLFOX) chemotherapy, whereas SIAH2 depletion effectively abrogated these effects. Collectively, these findings identify the SIAH2/WNK1 axis as a central metabolic regulator linking glycolysis, CSC maintenance, and microenvironment-driven therapy resistance in CRC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb269b1f5618b40199d4b3cc75e161d29b2f08d9" target='_blank'>
              SIAH2–WNK1 Signaling Drives Glycolytic Metabolism and Therapeutic Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Kee-Thai Kiu, , Yi-Chiao Cheng, Min-Hsuan Yen, Ying-Wei Chen, N. W. Pikatan, Vijesh Kumar Yadav, Tung-Cheng Chang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9106ce16a20b6a59754668f52e05e0f1c8f3c2c3" target='_blank'>
              SET1/MLL complexes control transcription independently of H3K4me3
              </a>
            </td>
          <td>
            Hya Au, AT Szczurek, I. de Krijger, N. Kjelstrup-Osorio, AL Hughes, A. Lastuvkova, M. Vermeulen, RJ Klose
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Conventional cancer therapies, including radiation therapy and chemotherapy, rely on inflicting DNA damage, yet they inevitably affect normal cells, leading to severe adverse effects. The advent of precision chemotherapy exploiting tumor-specific DNA repair defects has validated the effectiveness of this approach. The first successful example is PARP inhibitors, which selectively kill homologous recombination (HR) defective cancers, such as familial breast cancer possessing HR deficiency due to BRCA gene mutations. However, the broader landscape of DNA maintenance-including DNA replication, repair, and checkpoint pathways-harbors numerous mutations in tumors that remain untargeted. Here, we propose repurposing chain-terminating nucleoside analogs (CTNAs) to target such cancers' vulnerabilities. CTNAs, long utilized as anti-cancers and anti-viral drugs, inhibit replication and thereby suppress growth, but their activity has never been systematically aligned with specific cancer mutations associated with DNA maintenance defects. Based on our recent studies, we demonstrate that CTNAs elicit synthetic lethality in cells deficient for distinct DNA maintenance systems, amplifying replication stress, leading to cell death. We highlight the spectrum of CTNA-induced lesions and repair pathways required for cellular tolerance. This framework presents a versatile "repair-defect-guided" chemotherapy that expands the clinical utility of CTNAs and improves therapeutic effect by reducing side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e63be07db0a569b7816efc4ad2f977abe094978" target='_blank'>
              Targeting Genome Maintenance Defects of Cancers Using Chain-Terminating Nucleoside Analogs.
              </a>
            </td>
          <td>
            Ryotaro Kawasumi, Rubaiat E Tabassum, Kouji Hirota
          </td>
          <td>2025-12-10</td>
          <td>Cancer science</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eabb33613261d21b9be588be9b205da952bb4e2" target='_blank'>
              Polyploidy drives structural and functional evolution in Camellia mitochondrial genomes.
              </a>
            </td>
          <td>
            Jianbin Gao, Yuan Zeng, Boyong Liao, Zuoyun Yin, Wei Wu, Yong-quan Li
          </td>
          <td>2026-01-28</td>
          <td>BMC genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Long Interspersed Nuclear Elements-1 (LINE-1, L1) are transposable elements that make up roughly 17% of the human genome. These elements can copy and insert themselves into new genomic locations (Kazazian and Moran, N Engl J Med 377:361–370, 2017). Typically, LINE-1 is repressed in healthy tissues but may become activated in various human diseases. LINE-1 expression has been associated with aging (Simon, et al., Cell Metab 29:871–885.e5, 2019; De Cecco et al. Nature 566:73–78, 2019; Della Valle et al. Nat Rev Genet 26:1–12, 2025), neurodegenerative disorders (Roy et al., Acta Neuropathol 148:75, 2024;Frost and Dubnau, Annu Rev Neurosci 47:123–143, 2024; Ravel-Godreuil et al. FEBS Lett 595:2733–2755, 2021), cancer (Rodriguez-Martin et al., Nat Genet 52:306-319, 2020; Taylor et al. Cancer Discov 13:2532–2547, 2023; Solovyov et al. Nat Commun 16:2049, 2025), and autoimmune diseases (Rice et al., N Engl J Med 379:2275–2277, 2018), (Carter et al., Arthritis Rheumatol 72:89–99, 2020). Despite the strong association between LINE-1 expression and disease, the regulatory mechanisms controlling the expression of LINE-1-encoded ORF1p and ORF2p and the link between LINE-1 activity and cancer cell survival remain poorly understood. Gaining insights into these regulatory pathways may help elucidate how LINE-1 contributes to disease pathogenesis. To identify upstream regulators of LINE-1 and genes associated with LINE-1 activity-dependent lethality, we developed a dual-reporter system that simultaneously monitors the protein levels of LINE-1-encoded ORF1p and ORF2p (wild-type or catalytically inactive EN/RT mutant). Using genome-wide CRISPR/Cas9-based screens with this system, we identified candidate genes that may influence LINE-1 regulation at multiple levels, including RNA and protein expression. Alongside known factors such as the HUSH complex, the screens revealed additional genes not previously linked to LINE-1 regulation, suggesting possible new regulatory mechanisms for ORF1p and ORF2p expression. We also identified genes whose loss correlated with reduced viability in a manner dependent on LINE-1 activity. These findings collectively provide a broad resource for exploring cellular factors that may modulate LINE-1 expression and activity. This study provides a resource for investigating the cellular regulation of LINE-1, highlighting distinct candidate factors that may modulate ORF1p and ORF2p expression and influence LINE-1 activity-associated cytotoxicity. While functional validation of these candidate regulators remains necessary, the findings offer a foundation for future studies aimed at experimentally confirming their roles and elucidating the molecular mechanisms underlying LINE-1 regulation and its potential contributions to disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc75de6176e9aaa531a302da776789c267f8ce" target='_blank'>
              Identification of novel regulators of LINE-1 expression via CRISPR/Cas9 screening
              </a>
            </td>
          <td>
            Ozgur Oksuz, Chong Chu, Cedric Arisdakessian, Liyang Diao, Dennis M. Zaller, Kimberly K Long, Heike Keilhack, Sarah Knutson
          </td>
          <td>2025-12-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A significant cancerous growth with a high global incidence and death rate, cancer of the lungs—particularly non-small cell lung cancer (NSCLC), which makes up 80% to 85% of cases—poses a serious threat to public health. Drug resistance, damage to cells that are healthy, and limited effectiveness in advanced stages represent a few of the drawbacks of conventional treatments such radiation and targeted therapy. CRISPR-Cas9 gene editing technology, consisting of key elements like Cas9 protein and guide RNA (sgRNA), enables precise DNA cleavage and editing, offering new prospects for NSCLC treatment with advantages such as strong targeting and high efficiency. The procedures for employing CRISPR-Cas9 for curing NSCLC are explained in detail in this publication, including the use of delivery vectors optimized to overcome lung physiological barriers, treatment strategies like oncogene knockout using Cas9 nickases and adjuvant CAR-T therapy through PD-1 gene knockout, and its application in detection and diagnosis for early screening. Additionally, it analyzes the technical advantages of CRISPR-Cas9 compared to traditional therapies and discusses the challenges and solutions in clinical transformation, with the intention of offering a theoretical guide for the accurate management of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3f6e41f0ec4c718c50a0a00f99bdd6b6402f217" target='_blank'>
              Applying CRISPR-Cas9 technologies for lung cancer treatment
              </a>
            </td>
          <td>
            Shimiao Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Loss of isozyme diversity (LID) refers to the selective dependency on a single isozyme following the functional collapse of its redundant counterparts, uncovering a metabolic vulnerability. This metabolic liability establishes LID as a novel framework for precision targeting strategy in cancer therapy. We integrated a large-scale breast cancer multi-omics cohort to systematically delineate isozyme expression patterns and identify therapeutically promising LID targets across intrinsic molecular subtypes. Genomic and epigenetic data were employed to decipher the underlying causes of LID. Subsequent experiments were conducted to validate the collateral metabolic vulnerabilities arising from identified LID targets and investigate the associated mechanisms. Our analysis delineated the isozyme utilization patterns for metabolic reactions in breast cancer, which revealed widespread and significant LID phenomena. We further decoded the enrichment characteristics of these LID reactions and identified a set of potential clinically relevant LID targets across different subtypes. Further investigation revealed that copy number loss and epigenetic silencing predominantly drive these LIDs, as exemplified by luminal subtype-specific N-myristoyltransferase 2 (NMT2) deficiency caused by hypermethylation. Mechanistic studies demonstrated that NMT2 deficiency confers synthetic lethality with NMT1 inhibition, potentially through impaired myristoylation of the oncogenic effector FMR1 autosomal homolog 2 (FXR2). These findings underscored the potential of NMT inhibitors as viable therapeutic strategies for NMT2-deficient tumors. In conclusion, this study constructed a comprehensive LID landscape across all breast cancer subtypes, advancing precision oncology through innovative metabolic targeting approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48d7510a6c9d424472c6f680b4af6eada3b51b02" target='_blank'>
              Collateral metabolic vulnerabilities unveiled by loss of isozyme diversity in breast cancer
              </a>
            </td>
          <td>
            Rui Ding, Tianjian Yu, Yizhou Jiang, Yi Xiao, Zhi-Ming Shao
          </td>
          <td>2025-12-12</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Sex chromosome aneuploidies (SCAs) are collectively common genetic disorders that impact diverse body systems. The molecular mechanisms by which an extra or missing sex chromosome increases clinical risk are not fully understood, but they likely involve imbalances in expression and regulation of dosage-sensitive genes. There has been a recent surge in transcriptomic and epigenomic studies on genomic effects of SCAs - making it an opportune time to: (i) map existing knowledge of SCA impacts on gene expression and regulation; (ii) resolve consensus findings on SCA dosage-sensitive genes; and (iii) define gaps and high priority areas for future research. Following the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines (protocol registered on the international Prospective Register of Systematic Reviews PROSPERO CRD42024473984), we searched nine databases and screened the titles and abstracts of 2421 records, thoroughly evaluated 161 full-text articles, and identified 57 eligible studies for abstraction of methodological features and results. Our review spans 18 years of research and encompasses samples from 930 individuals with SCAs and 2192 euploidic controls. The recent acceleration in publication rates outstrips that for biomedical research as a whole. Studies have only recently started to diversify away from the most studied SCAs (47,XXY and 45,X karyotypes), tissue types (blood-derived, gonadal) and measurement methods (transcriptomic analysis by RNAseq). We identify a core set of dosage-sensitive genes that are recurrently impacted by SCAs across multiple tissues. These genes concentrate in 3 protein-protein interaction networks that are predominantly enriched for chromatin remodelling, and represent candidate drivers of downstream phenotypes. This systematic review of SCA impacts on the human genome helps to target the future research efforts that are now needed to (i) address existing knowledge gaps by diversifying the karyotypes, tissues and genomic features analyzed, and (ii) test the causal role for recurrently dysregulated genes. Meeting these goals would provide a molecular foundation to drive both basic and clinical understanding of sex chromosome influences on human phenotypic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2f04563ca279e23a5b7a1d63c20549007f5b86a" target='_blank'>
              Sex chromosome aneuploidy impacts on human gene expression and regulation: a systematic review
              </a>
            </td>
          <td>
            Marcela Legue, Melanie Staszewski, Maya Mastronardo, Gisela Butera, Aleksandra Dakic, Armin Raznahan
          </td>
          <td>2025-12-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ffa110b89229c9f4629f47074b1313edcfc03" target='_blank'>
              Praja1 protects cells from DNA damage through direct DNA binding
              </a>
            </td>
          <td>
            Kotaro Kawasaki, Toru Asahi, Wataru Onodera
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966381538c6447576a47e18b17f5babd106c5c90" target='_blank'>
              Structural organization and function of telomeric chromatin.
              </a>
            </td>
          <td>
            Ruben van der Lugt, J. Jacobs
          </td>
          <td>2026-01-07</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Failures in chromosome segregation during meiosis result in aneuploid gametes and are the leading cause of miscarriage. The position and number of crossovers, genetic exchanges between homologous chromosomes, are essential to their accurate disjunction in meiosis. Previous research analyzing nondisjunction of acro- and telocentric chromosomes in human and Drosophila has identified altered positioning and number of crossovers that differs between meiosis I and meiosis II nondisjunction. Limited data from positioning in chromosomes that segregated normally in metacentric trisomies has suggested discrepancies between the behavior of these chromosomes and the acro/telocentrics in nondisjunction, which warrants further investigation. Here, we employ whole-genome sequencing to study spontaneous meiotic nondisjunction of the metacentric chromosome 2 in Drosophila melanogaster. Several patterns of recombination may differentially impact telo- versus metacentric chromosomes: lack of a crossover, distal crossovers, and proximal crossovers. We find that unlike meiotic nondisjunction of the Drosophila X chromosome, nondisjunction of chromosome 2 is not associated with dramatic changes in crossover landscape, but is associated with reduced recombination. Differences in the proportions of NDJ events with altered recombination patterns between chromosomes X and 2 suggest that abnormal crossover positions disparately affect chromosomes with different shapes. Taken together, these findings highlight that the underlying triggers of meiotic nondisjunction are chromosome-specific.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8208b29f245fc09b23a7d8f70794f7fb4b094e6" target='_blank'>
              Chromosome-specific differences in the recombination landscape of spontaneous meiotic nondisjunction
              </a>
            </td>
          <td>
            Carolyn A. Turcotte, J. Sekelsky
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Breast cancer remains the leading cause of cancer-related deaths in women. Therefore, developing targeted combination therapies that improve overall survival in breast cancer patients continues to pose a major clinical challenge. Etoposide (ETO), a topoisomerase II inhibitor that induces transient double-strand breaks by blocking the cleavable complex, is currently used in high doses to treat radioresistant or metastatic breast cancers. To enhance the effectiveness of radiation and chemotherapy, targeting molecular mechanisms involved in DNA repair of induced DNA lesions could selectively increase tumor cell death. Since the effects of ETO are primarily seen during the S/G2 phases of the cell cycle, its efficacy could potentially be enhanced by using an inhibitor of a DNA repair gene involved in homologous recombination, which is mainly active during these phases. In this context, synthetic lethality refers to the concept that inhibiting or mutating two or more genes simultaneously leads to greater cell death than altering them individually. The FDA approval of Olaparib, a specific PARP inhibitor for BRCA-mutated breast cancer patients, has motivated researchers to explore other synthetic lethal interactions that could increase DNA damage accumulation, leading to cancer cell death. We demonstrate that successive treatment of breast cancer cells with specific inhibitors of ATM kinase and topoisomerase II in vitro can induce increased apoptosis. ATM is an apical kinase that recognizes DNA double-strand breaks and activates the homologous recombination repair pathway, either directly or through cell cycle checkpoint control. Topoisomerase II poisons generate enzyme-mediated DNA damage, leading to permanent double-strand breaks. Cytokinesis-block micronucleus assay was performed to assess the increase in DNA damage during combination treatment with inhibitors. Additionally, cell viability tests and fluorescent staining assay were conducted to evaluate the extent of cell death. We found that targeting ETO-treated breast cancer cells with an ATM kinase inhibitor, KU-55933 (KU) induced higher chromosomal damage/aberrations, as evaluated by the cytokinesis-block micronucleus assay. The ATM kinase inhibitor also significantly reduced the viability of ETO-treated breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0930a44ae8b83df33c3c61afc28e59cfc915449b" target='_blank'>
              Increased sensitivity of etoposide-treated breast cancer cells with an ATM inhibitor
              </a>
            </td>
          <td>
            Arun R. K. Kumar, C. Shaji, Aswathi Rajagopalan, Sombodhi Bhattacharya, Wilner Martínez-López, Radha Saraswathy
          </td>
          <td>2026-01-20</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The senescence-associated secretory phenotype (SASP) exerts dual roles in tumor suppression and progression, yet how it is regulated in head and neck squamous cell carcinoma (HNSCC) remains unclear. Here, we identify LIM homeobox 1 (LHX1) as a key transcriptional suppressor of STING, whose downregulation enables evasion of SASP-mediated tumor surveillance. Notably, high LHX1 expression correlated with poor prognosis in HNSCC patients. Mechanistically, LHX1, in complex with LDB1, directly bound to the STING promoter to mediate transcriptional repression via the deposition of the repressive histone mark H3K9me3, thereby blocking SASP activation. Depletion of LHX1 restored STING-dependent SASP and impaired cancer stem cell self-renewal. Therapeutic disruption of the LHX1-LDB1 complex using engineered peptides re-activated STING signaling, induced SASP, and significantly suppressed tumor growth. In this study, we employed human and mouse-derived HNSCC cell lines, xenograft models, and clinical samples to assess the functional relevance of LHX1 in regulating SASP and tumor progression. Our findings reveal LHX1 as a master transcriptional repressor of STING-mediated senescence and highlight the therapeutic potential of targeting the LHX1-LDB1 axis to restore tumor-suppressive SASP in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c542eb07fdf0c718c2bb9764fab625fc0a071c" target='_blank'>
              Targeting the LHX1-LDB1 Complex Restores STING-dependent Senescence Surveillance and Inhibits Head and Neck Cancer Progression
              </a>
            </td>
          <td>
            Mingshu Long, Yang Chen, Ruixue Du, Jiejie Yang, Jingjing Wang, Yunqing Sun, Xuezhang Tian, Shaowei Wang, Yunhong Zhong, Weilian Liang, Junjie Zhang, Z. Shang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Androgen Receptor Pathway Inhibition (ARPI) is a cornerstone of prostate cancer treatment; however, therapeutic resistance often arises due to an AR-independent cell state emerging as a result of tumor plasticity. Dysregulation of epigenetic reprogramming factors, including Polycomb Repressive Complex 2 (PRC2), creates an environment conducive to lineage plasticity. Thus inhibiting PRC2 offers a promising strategy to overcome ARPI therapeutic resistance in prostate cancer. ORIC-944 is a potent, orally bioavailable, allosteric PRC2 inhibitor with potential best-in-class drug properties that is currently in Phase 1b clinical development. In preclinical studies, ORIC-944 in combination with ARPI demonstrated efficacy across various models including AR-mutant and -wildtype, ARPI-resistant and -sensitive, and castration-resistant and -sensitive prostate cancers. Despite the spectrum of resistance mechanisms in these AR positive models, ORIC-944 consistently enhanced signatures of luminal cell state and AR signaling. These preclinical observations reveal the potential for ORIC-944 to block prostate tumor adaptation and re-sensitize tumors to ARPI. In this study of clinical samples from the ORIC-944-01 Phase 1b clinical trial, we aimed to evaluate the mechanism of ORIC-944 in patients with prostate cancer. DNA methylation profiling from liquid biopsies was chosen as a non-invasive method for investigating the epigenetic landscape of circulating tumor DNA. We identified CpG-rich regions associated with prostate tumorigenesis and transcriptional subtypes based on publicly available DNA methylation data from 100 metastatic castration-resistant prostate cancer (mCRPC) tumors, 35 healthy blood samples, and 150+ normal tissues. These selected regions became the foundation of a custom panel utilized on liquid biopsies to profile the epigenetic states of tumors from mCRPC patients treated with single-agent ORIC-944. A new methodology for cell-free DNA methylation was established to detect and quantify changes in subtype-associated regions independently of tumor burden. This approach and the clinical DNA methylation data showed that following treatment with ORIC-944, tumors from most patients experienced a shift in methylation pattern that suggests an increase in AR signaling. Notably, this shift was detectable after one treatment cycle of single agent ORIC-944, consistent with preclinical studies. Conversely, no patients showed an increase in neuroendocrine-associated signals after one cycle of treatment. These findings suggest that a selected set of informative methylation regions evaluated in plasma can identify epigenetic-driven changes in transcriptional activity during the treatment of patients with advanced prostate cancer. This approach is a key step towards enabling non-invasive epigenomic profiling of transcriptional states of prostate cancer. Furthermore, this analysis confirms our preclinical mechanistic data and hypothesis that ORIC-944 treatment enhances AR signaling in prostate cancer.



 Amber W. Wang, Michal Pawlak, Eric Ariazi, Livia Ulicna, Anne Page, Rupal Patel, Edna Chow Maneval, Pratik S. Multani, Lori S. Friedman, Anneleen Daemen, Aleksandr Pankov. Circulating tumor DNA methylation captures epigenetic changes in patients induced by the PRC2 inhibitor ORIC-944 [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed33cfdff2ce86febb4c5b2edd0b31607a86486" target='_blank'>
              Abstract PR024: Circulating tumor DNA methylation captures epigenetic changes in patients induced by the PRC2 inhibitor ORIC-944
              </a>
            </td>
          <td>
            Amber W. Wang, Michal Pawlak, Eric Ariazi, Livia Ulicna, Anne Page, Rupal Patel, E. C. Maneval, P. Multani, Lori S Friedman, Anneleen Daemen, A. Pankov
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Estrogen receptor-positive breast cancer remains a leading cause of cancer-related death in women, with mortality largely driven by late recurrence of treatment-resistant disease. Loss of MLH1 promotes resistance to estrogen-targeting therapies by uncoupling cell cycle progression from estrogen regulation. Here, we show that even when MLH1 is abundantly expressed, aberrant cytoplasmic localization in a subset of tumor cells drives endocrine therapy resistance by enabling estrogen-independent growth. This resistance arises from failure to undergo robust cell cycle arrest in response to endocrine therapy, creating acute dependency on CDK4/6 activity. Consequently, CDK4/6 inhibitors induce strong regression in cells with cytoplasmic MLH1 compared to cells with nuclear MLH1. As cytoplasmic localization occurs in ~11% of ER+ patients, it represents a contributor to MLH1 dysregulation. Incorporating cytoplasmic MLH1 localization into diagnostics could guide the use of CDK4/6 inhibitors in this hard-to-treat subset. Mismatch repair pathway is frequently dysregulated across cancer types, commonly represented by loss of MLH1 or MSH2 gene expression. Here the authors model MLH1 missense mutations from patients to study how cytoplasmic localization of MLH1, promotes resistance to endocrine therapy but predicts response to cell cycle inhibitors in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d45973c13d385df999038297a0493e826b1cece1" target='_blank'>
              Aberrant cytoplasmic localization of MLH1 characterizes a cell population that seeds breast cancer recurrence
              </a>
            </td>
          <td>
            A. Mazumder, Jerry Dewitt, E. Oropeza, Nindo B Punturi, D. Lozano, M. Raghunathan, Jonathan Piscitelli, E. Sajjadi, Elena GueriniRocco, K. Venetis, Mariia Ivanova, Eltjona Mane, Marianna Dercole, A. Concardi, N. Fusco, Carol M. Manhart, Matthew Bainbridge, S. Haricharan
          </td>
          <td>2025-12-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35607d62497aa71c83442fc00cbc1c011c30044b" target='_blank'>
              DICER1 syndrome mutations lead to a gain of 3p-miRNA function, HERVH activity and increased metastatic potential
              </a>
            </td>
          <td>
            K. Gordon, N. Bellora, J. Witteveldt, Joanna K. Wojtus, Felix Mueller, Katharina Kases, Pilar G. Marchante, G. Peris, Shelagh Boyle, Sara R. Heras, Atlanta G. Cook, Cei Abreu-Goodger, S. Macias
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Androgen receptor (AR) signaling is critical for the survival and proliferation of prostate cancer (PCa) cells, thus making androgen deprivation therapy (ADT; e.g., castration) the mainstay for treatment. Notably, the oncogenic transcriptional functions of AR rely on a host of chromatin-binding regulatory proteins, which includes a pioneer transcription factor called FOXA1. FOXA1 de-compacts chromatin to enable DNA binding of AR and activation of target gene expression. Our lab found FOXA1 alterations to recur within three distinct structural classes in over 35% of metastatic castration-resistant PCa (mCRPC) cases in Caucasian White men. Subsequent studies reported that FOXA1 mutations are prevalent in over 40% of primary PCa in Chinese patients, thus positioning FOXA1 as a principal oncogene in this disease. Yet, the tumorigenic potential and pathobiology of FOXA1 alterations remain unexplored in vivo. Here, we have developed and characterized the first in-field transgenic mouse models that conditionally overexpress FOXA1 mutants in the prostate luminal epithelia. Our findings reveal that truncal FOXA1 class1 mutations (i.e., wing 2 alterations) in a Trp53-null background triggered high-grade adenocarcinoma mimicking human disease, characterized by abnormal induction of NSD2 expression, extensive AR reprogramming to chimeric AR-half neo-enhancers, and sensitivity to androgen deprivation therapies. In contrast, FOXA1 class 2 mutations (i.e., C-terminal truncations) that are acquired in human mCRPC do not drive prostate luminal transformation in mice. Instead, these mutations induce intra-luminal plasticity by reprogramming differentiated cells into a progenitor stem-like state within androgen-replete normal tissues—a phenomenon otherwise induced upon castration. Class 2 mutants lead to a 20-fold expansion of Ar+/Ck8+ luminal epithelia that gain expression of stemness markers such as Trop2, Ck4, and Psca. These stem cells are similar to the Club cells detected in the human prostate gland that have been implicated in driving resistance to ADT. Mechanistically, we found the cistromically-dominant Class 2 mutants to pioneer over 40,000 neo-enhancer elements that were bound by stemness-associated transcription factors, like KLF5 and AP-1, which together instruct an androgen-insensitive luminal progenitor cell fate. Consistently, we found that Class 2-mutant mouse prostates showed minimal atrophy upon castration, with immunohistological assessment revealing a higher density of Ki67+ luminal epithelial cells relative to both wild-type and Class 1-mutant tissues. Class 2-mutant organoids also showed greater subcutaneous grafting ability in limiting-dilution assays in mice. Collectively, our data establish FOXA1 as a multifaceted oncogene in AR-dependent prostate cancers, in which divergent evolution of FOXA1 mutational classes distinctly drives either cancer formation or therapy-resistant, intra-luminal plasticity during disease progression. These findings also represent the first report of FOXA1-driven prostate adenocarcinoma in mice.



 Sanjana Eyunni, Rahul Mannan, Yuping Zhang, Eleanor Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, Jean Ching-Yi Tien, James George, Mustapha Jaber, Hamzah Hakkani, Sandra E. Carson, Abigail J. Todd, Noshad Hosseini, Mahnoor Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin Cieslik, Arul M. Chinnaiyan, Abhijit Parolia. Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e838d76f934b647aed2d9fe44ea43b83b285eb1c" target='_blank'>
              Abstract PR012: Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, Jean Ching-Yi Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>170</td>
        </tr>

        <tr id="Heterochromatic position effect variegation (PEV) of the gene comprises euchromatic gene inhibition upon its transition in the area of heterochromatin either at the same chromosome, for example, in case of inversions (cis-acting PEV), or at the contact of the euchromatic gene with the heterochromatin associated with the allele of the same gene in the homologous chromosome in the three-dimentional nuclear volume (trans-acting PEV). The reverse PEV consists in heterochromatic gene inhibition in case it is placed in euchromatic environment. Most PEV data were obtained using genetic system of Drosophila melanogaster that has four chromosomes combine in the chromocenter. Heterochromatic cis-acting PEV often takes place in case of invertions such as In(2)A4; In(1)wm4. Molecular mechanisms of cis-acting PEV include expression level changes of several genes due to the changes in the quantities of specific histone modifications, heterochromatin proteins (HP1) and specific small RNA including piRNA. In case of cis-acting PEV the distribution of heterochromatic modifications (H3K9me2/3) and main heterochromatic protein (heterochromatin protein HP1a) from the heterochromatin to the euchromatin area is well studied that is associated with the expression inhibition of several genes. Heterochromatic trans-acting PEV has been thoroughly investigated only in a few cases including the invertion In(2)A4 and satellite DNA fragment insertion in brown gene (bwD). In both cases genomic rearrangements took place at the second chromosome of Drosophila melanogaster. Molecular mechanisms of trans-acting PEV are less studied than those of cis-acting PEV. It was shown that SU(VAR)2-HP2, SAYP, SETDB1 participate in trans-acting PEV in case of the inversion In(2)A4. Perspective studies in the field of cis- and trans-acting PEV include the role of prod and D1 mutations that influence the integrity of Drosophila melanogaster chromocenters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6bfc21165d15645a816b5cb7ec55e9ad20d3c8" target='_blank'>
              Molecular mechanisms of chromatin structure changes at position effect variegation
              </a>
            </td>
          <td>
            E.A. Shestakova, A. Solodovnikov, S. A. Lavrov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma (NB) is a childhood cancer that arises when certain nerve-related cells fail to mature properly and become malignant. Our study focuses on a gene called INSM1, which is normally active during early cell development but is abnormally elevated in NB tumors. We aimed to understand how INSM1 contributes to cancer’s aggressiveness and poor differentiation. We discovered that INSM1 keeps tumor cells in an immature, stem-like state and is closely linked to a chemical pathway called the methionine cycle, which affects gene regulation through epigenetic changes. Disrupting this cycle alters INSM1 activity and may help push cancer cells toward normal development. We also found that retinoic acid, a compound known to promote cell differentiation, reduces INSM1 and another cancer-driving gene, N-Myc. These findings suggest that targeting INSM1 and its metabolic regulation could be a promising strategy to treat NB by encouraging tumor cells to mature and lose their malignant properties. This research provides new insights into the biology of NB and may lead to more effective therapies for children affected by this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb4304aef088d414eab3a9ced76da697e7e9f2" target='_blank'>
              Regulation of INSM1 Gene Expression and Neuroendocrine Differentiation in High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Chiachen Chen, Siyuan Cheng, Xiuping Yu, Yisheng Lee, Michael S. Lan
          </td>
          <td>2025-12-22</td>
          <td>Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Prostate cancer is estimated to contribute to over 35,000 deaths of men residing in the United States, with the majority fatality due to metastatic disease. CDC7 is a kinase that regulates DNA replication and is found elevated during neuroendocrine transdifferentiation in lung and prostate cancer. In this study, we demonstrate that CDC7 is highly expressed in treatment-resistant prostate cancer, with even higher levels observed in treatment-resistant prostate cancer with neuroendocrine phenotype (NEPC). We further identify CDC7 as a critical regulator of prostate tumorigenesis. Downregulation of CDC7 significantly reduces prostate cancer cells growth and invasion in vitro and silencing CDC7 suppresses prostate tumor growth in vivo. Furthermore, we demonstrate that the inhibition of CDC7 using TAK-931, a selective CDC7 inhibitor, significantly reduces the proliferation, migration, and invasion of aggressive prostate cancer cells. TAK-931 treated prostate cancer cells exhibit an abnormal cell cycle profile, suggesting that CDC7 inhibition induces replication stress and promotes apoptosis. Collectively, our findings demonstrate that CDC7 is a regulator of tumor progression in prostate cancer and represents new therapeutic target in advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee39b3ba5cf310218ed8767183ac21c97e67e236" target='_blank'>
              CDC7 is a targetable regulator of advanced prostate cancer
              </a>
            </td>
          <td>
            A. B. Hartono, Hyeonji Hwang, Sidharth Paparaju, Shiqin Liu, James D. Brooks, E. Corey, Tanya Stoyanova
          </td>
          <td>2025-12-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost-effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry-based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN-associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS-STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of "Micronuclear Epigenetics" is highlighted, with its potential application in high-throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure-induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability.
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce1dd81fa69ca7596c6e2f6203643899d73e28c1" target='_blank'>
              In vivo isogenic modelling unveils a TP53-mediated relapse phenotype in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Stéphanie Gachet, S. Quentin, L. Hernandez, Loïc Maillard, M. Passet, R. Kim, Hugo Bergugnat, Melha Benlebna, Maxime Boy, Véronique Parietti, P. Fenaux, André Baruchel, Hervé Dombret, N. Boissel, François Sigaux, Hugues de Thé, E. Clappier, J. Soulier
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) of melanoma have identified numerous susceptibility loci. However, causal genes and variants underlying risk have yet to be established for most. It is becoming apparent that many functional variants underlying complex traits act via cis-regulation that may be context-specific, dependent on availability of specific transcription factors/complexes in specific cell types and cell-states. To characterize a risk locus on chromosome band 2q33.1 associated with melanoma, breast cancer, and keratinocyte cancers, we integrated fine-mapping, cell-type specific expression quantitative trait locus (eQTL) analysis, a massively parallel reporter assay, individual luciferase assays, and SNP-based proteomics. Integrated analysis implicates the presence of multiple functional variants lying primarily within a promoter for CASP8. A haplotype containing rs3769823 appeared have the largest effect on expression. Strikingly, both tumor/normal context and this risk-associated haplotype play critical roles in mediating allelic cis-regulatory activity. Quantitative mass spectrometry for rs3769823 identified both E4F1, a transcriptional repressor, and IRF2, a transcriptional activator, as binding preferentially to risk-associated rs3969823-A. The binding of these transcription factors was validated via EMSA, supershift, and chromatin immunoprecipitation (ChIP) assays. The relative levels of E4F1 and IRF2 differ by cell-type and play a role in mediating transcriptional activity in a cell-type specific manner. Our results indicate that the top credible causal set variant rs3769823 likely influences expression of CASP8 and FLACC1 in a cell-type specific manner and may be a relevant functional variant for multiple cancers associated with this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0bd03d9d9be45c3f139097e5cecc84ff17cc6a" target='_blank'>
              Functional characterization of a multi-cancer risk locus on chromosome band 2q33.1 near CASP8
              </a>
            </td>
          <td>
            Hyunkyung Kong, Jiyeon Choi, Tongwu Zhang, Cathrin Gräwe, Mai Xu, Rohit Thakur, Hayley A. Sowards, Rebecca Hennessey, Andrew Vu, Jianxin Shi, T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Vermeulen, M. Law, L. Amundadottir, Melanoma Meta-7 Analysis, Consortium, Kevin M. Brown
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="As a tumor suppressor gene, CCDC6 encodes a coiled-coil domain-containing protein that is ubiquitously expressed and involved in crucial cellular processes such as DNA damage response and apoptosis, although its precise mechanisms remain elusive. Initially identified as part of a fusion gene, CCDC6 can form fusion genes with a variety of proto-oncogenes, including both kinase- and non-kinase-coding genes, thereby facilitating oncogenesis. Alterations in CCDC6 expression across various cancers underscore its intricate role and potential influence on the efficacy of anticancer therapies. Recent findings have demonstrated that CCDC6 can undergo liquid-liquid phase separation (LLPS) and facilitate the LLPS of its associated fusion proteins, providing new perspectives on its functional characterization and potential therapeutic implications in related diseases. We present a comprehensive overview of CCDC6, encompassing its protein characteristics and physiological and genomic aspects. Furthermore, we explored the association between CCDC6 alterations and carcinogenesis, as well as their implications for therapeutic interventions. The objective of this review is to furnish the medicinal community with current information and valuable insights pertaining to diseases associated with CCDC6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872145069c4ef81fa1ace0d713f50e945a6721cc" target='_blank'>
              The multiple roles of the disordered protein CCDC6 in cancer development.
              </a>
            </td>
          <td>
            Guifeng Wei, Yiji Chen, Yichao Kong, Donglai Li, Yang Wang, Ting Qiu, Xiabin Chen
          </td>
          <td>2026-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The extraordinary repetitive content of human acrocentric short arms has prevented detailed investigations into recombination and de novo mutation. Integrating multiple sequencing technologies, we created 156 phased short arms and assessed 107 intergenerational transmissions from 23 samples in a four-generation pedigree. We observed a significant depletion (P<0.0001) of p-arm allelic recombination but one ectopic chr13–chr21 recombination breakpoint mediated by a 630 kbp segmental duplication mapping 1.6 Mbp distal to the SST1 array. In contrast, 18 maternal-biased q-arm allelic recombinations are significantly enriched within 5 Mbp of the centromere. Compared to autosomal euchromatin, the overall p-arm de novo single-nucleotide variant rate (1.33×10⁻⁷ per base pair per generation) is 10-fold higher, with a significant reduction of C>T but increased C>G and A>C mutations. We hypothesize that acrocentric sequence composition biases and the dearth of allelic recombination contribute to an elevated mutation rate and unique mutational signatures suggestive of mismatch repair defects and oxidative stress-induced DNA lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5730289623fe97755031a7f3f19f52605e3c3" target='_blank'>
              Human acrocentric chromosome short arm de novo mutation and recombination
              </a>
            </td>
          <td>
            Jiadong Lin, F. K. Mastrorosa, Michelle D. Noyes, DongAhn Yoo, A. Rhie, David Porubsky, Kendra Hoekzema, Katherine M. Munson, Nidhi Koundinya, W. S. Watkins, Lynn B. Jorde, Aaron R. Quinlan, Deborah W. Neklason, A. Phillippy, E. Eichler
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Lysine specific demethylase 1 (LSD1), encoded by the gene KDM1A, is overexpressed and correlates with poor patient prognosis in Ewing sarcoma. LSD1 and the pathognomonic fusion oncoprotein, EWSR1::FLI1, colocalize throughout the genome, suggesting LSD1 is a critical co-regulator driving the progression of Ewing sarcoma. However, therapeutic targeting of LSD1 by competitive and noncompetitive inhibitors has yielded mixed results. Irreversible, enzymatic inhibition seems ineffective, but reversible noncompetitive inhibition has predominant off target mechanisms, leaving open the question of LSD1 function in Ewing sarcoma. Here we take a robust approach through multiple methods of depletion in multiple EwS cell lines to define enzymatic and nonenzymatic contributions of LSD1 to transcriptional regulation. We define a core set of 22 genes that are commonly repressed by LSD1 in all cell lines, and that repression of these genes downregulates synapse functioning and e-cadherin target genes. Derepression of these genes with LSD1 loss is an early and sustained genotype in all cell lines tested. We further define distinct gene sets in each cell line that are regulated by enzymatic and nonenzymatic LSD1 activity and find repression of e-cadherin target genes to be nonenzymatically regulated. This finding supports the growing body of evidence that in addition to their canonical catalytic activity, chromatin regulatory enzymes serve essential noncanonical roles as well. Furthermore, we uncovered evidence through use of the irreversible inhibitor OG-L002 that 2D cytotoxicity and proliferation assays may be insufficient to determine Ewing sarcoma response to LSD1 inhibition. SIGNIFICANCE Here we address a long-standing question in the field surrounding LSD1 and define the distinct enzymatic and nonenzymatic functions of LSD1 in Ewing sarcoma. In doing so, we have created a robust data set using genetic and pharmacological techniques in multiple models to thoroughly characterize LSD1 function in Ewing sarcoma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa31c55d8000608b4f5b4507892563a7a74a977c" target='_blank'>
              LSD1 Performs Demethylase-Independent and Context-Specific Roles in Ewing Sarcoma
              </a>
            </td>
          <td>
            Rachel D. Dreher, C. Taslim, Ira Miller, John W. Sherman, Ariunaa Bayanjargal, Emily R. Theisen
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In cells, highly coordinated multivalent interactions give rise to discrete functional assemblies-commonly referred to as biomolecular condensates-that compartmentalize the molecular components required for specific biological reactions. These condensates are increasingly recognized as organizational entities with central roles in normal cellular regulation and in the pathogenesis of human cancers. In a recent issue of Cell, Datar and colleagues investigated the condensation of NPM1c, a common gene mutation in acute myeloid leukemias (AMLs). They demonstrated the necessity and sufficiency of NPM1c in forming nuclear condensates termed coordinating bodies (C-bodies), which show co-partitioning of a suite of transcriptional coactivators such as NUP98, KMT2A/MLL1, Menin and XPO1/CRM1. While C-bodies are necessary for driving NPM1c-mutant AMLs, blockade of the co-partitioned component within C-bodies, such as the XPO1/CRM1 or Menin interaction by inhibitors, significantly alters the condensate composition and functionality. Likewise, a systematic deletion study of various regions within NPM1c pointed to a role for the coordinated multivalent interaction in establishing the functional condensates, as previously reported in studies of the Wilms tumor-causing ENL mutants and AML-causing NUP98 onco-fusions. Co-mixing of C-bodies and condensates formed by the onco-fusion of NUP98 or KMT2A/MLL1 in cells suggested them to be biophysically indistinguishable, indicative of a shared pathogenic mechanism. Altogether, recent studies of multiple genetic drivers in human cancers have revealed a type of chromatin-bound multi-component onco-condensates, which shall motivate the development of onco-condensate disruptors that could potentially be used as the broad treatments for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cc999451ae4cdf83273f4d0ad6380f1649aa9ab" target='_blank'>
              A unified model: chromatin-bound multi-component onco-condensates.
              </a>
            </td>
          <td>
            Benjamin K Lau, Elena Haarer, Jeong Hyun Ahn, G. Wang
          </td>
          <td>2026-01-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The extreme rarity of certain fusion-oncogene-driven cancers limits dedicated mechanistic study and the potential development of new therapies. Beyond simple expression constructs, only a select few fusion-driven cancers have had synthetic tools engineered which enable complex and thorough biological dissection of these driver events. In this work, we focus on facilitating access to research focused on fusion genes involving capicua (CIC) and ataxin 1 (ATXN1). While prior work from our group and others has led to advances in modeling CIC::DUX4 sarcoma, no tools have existed to study fusions involving other 3’ partners including CIC::NUTM1 and CIC::LEUTX, which have different clinical characteristics compared to CIC::DUX4. We use the first models of CIC::NUTM1 and CIC::LEUTX to evaluate the gene programs they drive and the functional domains they rely on to function. We characterize a novel region within the NUTM1 fragment of CIC::NUTM1, which enables expression of hundreds of genes not strongly induced by CIC::DUX4 in 293T cells. This molecular divergence has potential implications for the identification of the CIC::NUTM1 cell of origin. We also find two LEUTX transactivating domains as key for CIC::LEUTX-mediated transcriptional activation of a CIC target gene, potentially highlighting a therapeutic vulnerability. Finally, we build and characterize the first model of ATXN1 (a known CIC interactor) containing fusions, including ATXN1::DUX4 tumors, which constitute a rare but interesting class of fusions that potentially co-opt native CIC interactions to drive tumorigenesis. Beyond transcriptional events, we have engineered tools to study CIC::DUX4 at the cellular level. Specifically, we developed tools for microscopic visualization (mStayGold tag) and proximity labelling (TurboID tag) of the CIC::DUX4 fusion oncoprotein. These tools will facilitate our understanding of CIC::DUX4-mediated cellular interactions and function, improving our understanding of it to a level comparable to that of the more well studied EWS::FLI or PAX::FOXO fusions. Collectively, we have developed a suite of ready-made tools and model systems to scientifically and clinically advance the study of CIC- and ATXN1-rearranged cancers. A major goal of this project is to provide the research community with CIC and ATXN1 fusion tools and resources to advance therapies and improve outcomes for these devastating and understudied diseases.



 Cuyler Luck, Ross A. Okimoto. Lowering barriers to studying CIC- and ATXN1-rearranged cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e32adf20145c10d47d14897eebe780ef3f650043" target='_blank'>
              Abstract A015: Lowering barriers to studying
 CIC
 - and
 ATXN1
 -rearranged cancers
              </a>
            </td>
          <td>
            Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Invasive lobular carcinoma of the breast (ILC) is typically estrogen receptor α (ER)-positive and presents with biomarkers of anti-estrogen sensitive disease. Unfortunately, patients with ILC face particularly poor long-term outcomes with high recurrence risk, suggesting a divergent endocrine response and ER function in ILC compared to other breast cancers. ER is co-regulated by the DNA repair protein MDC1 specifically in ILC cells, driving distinct ER activity. Here, we profiled the MDC1 interactome to examine how MDC1 regulates ER activity and DNA repair function in ILC. MDC1-associated proteins in ILC cells mirrored a "BRCA-like" state lacking key homologous recombination (HR) proteins, consistent with HR dysfunction but distinct from classic "BRCAness". Single-cell transcriptome and DNA repair activity analyses, along with DNA repair signaling and functional data, substantiated dysfunctional induction and execution of HR in ILC cells. In parallel, ILC tumor data were consistent with a form of HR dysfunction distinct from overt HR deficiency, lacking BRCA-like genomic scarring but showing elevated signatures of PARP inhibitor sensitivity. Treatment with the PARP inhibitor talazoparib produced a durable growth suppression both in vitro and in multiple ILC xenografts in vivo. Together, these findings reveal that ILC-specific ER:MDC1 activity comes at the cost of DNA repair dysfunction, which may be therapeutically targetable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a91ccc24a1ced8113a892a4a43c737a71e1d883" target='_blank'>
              Altered MDC1 Interactions and Dysfunctional DNA Repair in Lobular Breast Cancer Confers Sensitivity to PARP Inhibition.
              </a>
            </td>
          <td>
            Joseph L Sottnik, Madeleine T. Shackleford, Camryn S. Nesiba, Amanda L. Richer, Z. Fleischmann, Jordan M. Swartz, Carmen E. Rowland, Maggie Musick, Rui Fu, Logan R. Myler, P. Opresko, Sanjana Mehrotra, Ethan S Sokol, Jay R. Hesselberth, Jennifer R. Diamond, Matthew J. Sikora
          </td>
          <td>2026-01-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The AID/APOBEC gene family originated during evolution to perform functions related to organismal defense. At present, AID/APOBEC genes encode proteins that are structurally similar but functionally diverse. Most AID/APOBEC proteins exhibit deaminase activity toward cytidyl nucleotides in single-stranded DNA and RNA. In addition to cytidine deamination within viral genomes – which restricts infection – these deaminases are also capable of inducing mutations in the human genome. APOBEC-associated mutations represent one of the most common mutational signatures in cancer: they have been detected in approximately 75 % of cancer types and in more than 50 % of all tumors. 
The aim of this review is to analyze the role of AID/APOBEC-induced mutations in the genesis of human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631a1dbb121589589ddf78ee79bee08d68b13140" target='_blank'>
              AID/APOBEC family and its role in carcinogenesis
              </a>
            </td>
          <td>
            G. Volgareva
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6da8c58eace175b556f1165210d11f554c8a306d" target='_blank'>
              Regulatory Mechanisms of Maternal Imprinting at the Dlk1-Dio3 Domain
              </a>
            </td>
          <td>
            Ariella Weinberg-Shukron, Frances L Dearden, Ana Moreno-Barriga, Lok Ting Nick New, Marco Müller, Rebecca C. Rancourt, C. Edwards, A. Ferguson-Smith
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Prostate cancer is among the most prevalent cancer types and advanced castration-resistant prostate cancer (CRPC) remains a lethal disease. It is driven by oncogenic transcription factors, most notably the androgen receptor (AR) which represents to this day the main therapeutic target. That said, more transcription factors have emerged supporting tumor growth in late stages when prostate cancer cells lose AR expression through lineage plasticity. However, targeting this plethora of transcriptional drivers throughout disease progression is a daunting task. Yet, disrupting common enablers upstream of oncogenic transcription factors may provide an opportunity to shut down oncogenic output of prostate cancer cells regardless of their CRPC subtype. Indeed, the pioneer transcription factor FOXA1 is a strong and highly selective Achilles' heel in prostate cancer (www.depmap.org), critical for opening chromatin and enabling AR to exert its oncogenic function. Moreover, its homologue FOXA2 has been shown to be upregulated in AR-negative lineage plasticity prostate cancers as well. That said, effective FOXA1/2 inhibitors are currently not available. Here, we show that the FOXA1 homologue FOXA2 cooperates with the AR-pioneer factor FOXA1 in mediating AR-independent cell proliferation in various prostate cancer subtypes. In AR-negative cells, this colocalization is reinforced and supports the gene transcription of lineage-specific oncogenic transcription factors. Joint loss-of-function of both FOXA1 and FOXA2 leads to the collapse of lineage-specific oncogenic transcription factors. Importantly, using virtual drug screening we identify a non-covalent small molecule capable of directly binding and disrupting the interaction of the conserved forkhead domains with DNA. The inhibitors target both FOXA1 wild-type tumors and tumor characterized by recurrent driver mutations. Mechanistically, the inhibitors bind to a hydrophobic cavity within the N-terminus of the conserved forkhead domains of FOXA1 and FOXA2. Treatment with T and derivatives inhibits specifically the oncogenic transcriptional output programs related to different AR-positive and AR-negative prostate cancer subtypes and effectively blocks tumor growth in culture and after intraperitoneal injection in xenograft models in vivo. In summary, our findings uncover an unexpected collaboration of FOXA1 and FOXA2 in disease progression and a druggable key dependency for AR-positive and -negative CRPC regardless of their specific lineage-specific oncogenic program.



 Nicolo Formaggio, Jacopo Sgrignani, Gayathri Thillaiyampalam, Claudio Lorenzi, Yanick Uebelhart, Diego Camuzi Cassiano, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Raffaella Santoro, Andrea Cavalli, Jean-Philippe Paul. Theurillat. Targeting of FOXA1 and FOXA2 by small molecules disables the oncogenic output of castration resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR020.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af0e2296f18efc47feaa25f0dfc62945878e4ac" target='_blank'>
              Abstract PR020: Targeting of FOXA1 and FOXA2 by small molecules disables the oncogenic output of castration resistant prostate cancer
              </a>
            </td>
          <td>
            Nicolò Formaggio, Jacopo Sgrignani, Gayathri Thillaiyampalam, Claudio Lorenzi, Yanick Uebelhart, Diego Camuzi Cassiano, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Raffaella Santoro, Andrea Cavalli, Jean-Philippe Paul. Theurillat
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3ef8e203c378a8d1a7856db6aaa8c3b98aa293" target='_blank'>
              Dissecting the contribution of transposable elements to interphase chromosome structure.
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, Xuemeng Zhou, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhuoqi Huang, Li Sun, Jiangping He, Alexander V. Strunnikov, A. P. Hutchins
          </td>
          <td>2026-01-20</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells.
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91fe51488198f3ff4051c25194da67b2c9762e1" target='_blank'>
              Human Accelerated Regions: Functional Studies and Methodological Approaches (A Review)
              </a>
            </td>
          <td>
            A. Knyazeva, T. A. Shnaider
          </td>
          <td>2026-01-30</td>
          <td>Cell and Tissue Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transcriptional reprogramming through induced proximity has emerged as a powerful strategy for modulating the expression of oncogenic and tumor-suppressive genes. Inspired by transcriptional reprogramming approaches such as transcriptional/epigenetic chemical inducers of proximity (TCIPs) that link BCL6 inhibitors to transcriptional regulators, we sought to develop covalent ligands that rewire BCL6 proximity to selectively suppress MYC transcriptional output while derepressing BCL6 target loci. Through a chemistry-driven and chemoproteomics-enabled design strategy, we generated a panel of BCL6-based electrophile-bearing hybrid ligands and identified a nondegradative molecular glue, ZD-1-186, that potently suppresses MYC and robustly induces CDKN1A (p21) in diffuse large B-cell lymphoma cells. ZD-1-186 downregulates MYC more effectively than BCL6 inhibitors or degraders, while strongly derepressing canonical BCL6 targets, including p21. Through BCL6 pulldown proteomics, ZD-1-186 induced a selective recruitment of the noncanonical BAF complex subunit BRD9 to BCL6 and covalently modified BRD9 at C288. Pharmacologic inhibition or genetic knockdown of BRD9 attenuated ZD-1-186-mediated MYC suppression and blunted p21 induction. Transcriptomic profiling of ZD-1-186 showed simultaneous derepression of BCL6-repressive loci and suppression of MYC transcriptional programs. These findings demonstrated that ZD-1-186 acted as a transcriptional rewiring glue, recruiting BRD9 to BCL6-repressive loci to activate tumor-suppressive transcription, while also potentially redirecting BCL6 to BRD9-bound oncogenic loci. Overall, our work provides a blueprint for the rational discovery and design of electrophile-enabled, nondegradative molecular glues for targeted transcriptional rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e1cf0470c87b88a6512470358d48dec81cbfa4" target='_blank'>
              Discovery of Non-Degradative Covalent Molecular Glues for Transcriptional Reprogramming
              </a>
            </td>
          <td>
            Tuong Nghi Duong, Edward Pandji, Qian Shao, Daniel K. Nomura
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Metastatic castration-resistant prostate cancer (mCRPC) is a heterogeneous disease and often comprise molecularly distinct subtypes that differ in prognosis and therapeutic response. Super-Enhancers (SEs) are large enhancer clusters that robustly drive expression of genes controlling developmental processes and cell identity. In this study, we developed an epigenetic-based framework to determine whether mCRPC harbors subtype-specific SE programs that may reactivate distinct developmental transcriptional programs and define molecularly subtypes of disease progression.



 We developed a computational workflow, Super-Enhancer Analysis for Lineages (SEAL), to map SE landscapes, classify tumor subtypes and identify subtype-specific SE-driven genes. Using ROSE analysis on H3K27ac ChIP-seq data from the LuCaP and MURAL PDX mCRPC series, we constructed an integrated SE atlas. Molecular subtypes were defined through consensus clustering of SE regions and further characterized by genomic, transcriptional, and clinical features. We integrated matched RNA sequencing data to identify top-ranked SE-driven transcription factors, which were further evaluated via loss- and gain-of-function studies as well as ChIP-seq and RNA-seq analyses to define their cistromes, transcriptomes, and interacting networks.



 We identified five distinct SE programs in mCRPC: three AR-positive subtypes (AR-1, AR-2, AR-3) and two AR-independent subtypes (NEPC-like and DNPC-like). Among the three AR-driven subtypes, AR-1 and AR-2 displayed aggressive tumor features, enriched for cell cycle, EMT, and hypoxia pathways. Each subtype exhibited a distinct SE-driven transcriptional program. Notably, we identified TWIST1, HNF1A, and TBX10 as key SE-driven transcription factors in AR-1, AR-2, and AR-3 subtypes, respectively. Among these, HNF1A, a critical transcription factor involved in hepatic development and metabolic regulation, was exclusively expressed in the AR-2 subtype. HNF1A silencing significantly reduced proliferation in vitro and in vivo and markedly suppressed glycolytic activity. AR-2 tumors produced high levels of secreted albumin, a well-established HNF1A hepatic target, which was decreased upon HNF1A silencing. ChIP-seq and RNA-seq analyses revealed that HOXB13 co-occupied HNF1A-mediated enhancers and cooperatively regulated the hepatic transcriptional programs. Conversely, HNF1A overexpression in AR-3 cells induced the expression of hepatic markers, enhanced proliferation and migration, and recapitulated features of the AR-2 subtype, indicating that HNF1A acts as a driver of hepatic lineage reprogramming.



 Our study reveals a novel HOXB13-HNF1A transcriptional axis that governs a previously undefined hepatic reprogramming in a subset of AR-driven mCRPC tumors. These findings establish a SE-based molecular classification of mCRPC and highlight HNF1A as a key regulator of metabolic and lineage programs, with potential implications for precision therapy and biomarker development in advanced prostate cancer.



 Mingyu Liu, Songqi Zhang, Nolan D. Patten, Jared G. Lourie, Xiaolin Zi, Kai Zou, Shuai Gao, Kourosh Zarringhalam, Changmeng Cai. Super-enhancer landscape analysis reveals a HOXB13-HNF1A transcriptional axis driving hepatic reprogramming in castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c8c15e0835123218bb04bb3827f4c746859379" target='_blank'>
              Abstract PR015: Super-enhancer landscape analysis reveals a HOXB13-HNF1A transcriptional axis driving hepatic reprogramming in castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Mingyu Liu, Songqi Zhang, Nolan D. Patten, Jared Lourie, Xiaolin Zi, Kai Zou, Shuai Gao, Kourosh Zarringhalam, Changmeng Cai
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Prostate cancer lineage plasticity, characterized by histologic transformation of prostate adenocarcinoma (PRAD) to neuroendocrine prostate cancer (NEPC), is an emerging mechanism of treatment resistance. NEPC accounts for up to 15% of treatment-resistant prostate cancers and is associated with poor prognosis, highlighting an unmet need for new therapies. NEPC lineage reprogramming is primarily driven by epigenetic dysregulation including differential activity of histone and DNA methyltransferases. EZH2, the catalytic component of the Polycomb repressive complex 2 (Polycomb), is overexpressed in most treatment-resistant prostate cancers and is implicated as a driver of disease progression. In this study, we define the differential, lineage-specific action of Polycomb in both PRAD and NEPC subtypes to better understand its role in modulating differentiation and lineage plasticity, and to identify novel targetable drivers of NEPC. Epigenetic H3K27me3 CUT&Tag profiling of treatment-resistant PRAD (n=9) and NEPC (n=9) rapid autopsy clinical samples revealed that Polycomb targets, including NE-lineage transcription factors, are de-repressed in NEPC. Mechanistically, Polycomb modulates H3K4/K27me3 bivalent promoters, leading to the upregulation of NEPC-associated transcriptional drivers (e.g., ASCL1) and neuronal gene programs which facilitate forward differentiation following EZH2 targeting in NEPC patient-derived organoid/xenograft models. Notably, we identified prospero-homeobox 1 (PROX1) as an understudied cell-fate determining transcription factor that is epigenetically de-repression by differential Polycomb activity. Integrative H3K27ac CUT&RUN and Capture Hi-C analyses revealed potential enhancers of PROX1 in NEPC that may be regulated by ASCL1. We sought to functionally characterize the role of PROX1 in NEPC. An unbiased CRISPR screen in two NEPC patient-derived organoid models demonstrated high cellular dependency for PROX1; knockout of PROX1 impeded tumor growth in NEPC models, while overexpression of PROX1 in PRAD promoted tumor growth and spontaneous metastases. Transcriptomic and cistromic analyses across models of CRPC and NEPC pointed to PROX1 regulation of neuroendocrine-lineage transcriptional programs. Immunoprecipitation followed by mass spectrometry identified three novel phosphorylated sites in the DNA-binding domain of PROX1 that are critical for its stability and function. In silico analyses of these phosphorylation sites predicted CHEK1 as a potential upstream kinase which could be exploited for therapeutic targeting of PROX1. Our findings provide insights into the potential role for bivalent promoters in Polycomb-mediated lineage reprogramming, which may facilitate forward differentiation in NEPC upon EZH2 inhibition. Further investigation of de-repressed candidates defines the role of PROX1 as a driver of NEPC and a potential therapeutic target.



 Varadha Balaji Venkadakrishnan, Nathaniel C. E. Voss, Nicole Traphagen, Richa Singh, James Neiswender, Keira Prenza. Sosa, Kenny Weng, Francisca Vazquez, David S. Rickman, Myles Brown, Himisha Beltran. Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16b90ea02984495a11ed149a37089c1519a17fc" target='_blank'>
              Abstract PR022: Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity
              </a>
            </td>
          <td>
            V. B. Venkadakrishnan, Nathaniel C E Voss, Nicole A. Traphagen, Richa Singh, James Neiswender, Keira Prenza Sosa, Kenny Weng, Francisca Vazquez, D. Rickman, Myles Brown, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Genomic alterations affecting components of the fibroblast growth factor (FGF) signaling axis can trigger aberrant pathway activation and tumor development. Genomic truncation of the FGF receptor 2 (FGFR2) exon 18 (E18) disrupts the FGFR2 carboxy-terminal tail (C-tail), acting as a potent driver alteration across multiple tumor types. Here, we analyzed human oncogenomic datasets to reveal that E18 truncations are similarly prevalent in FGFR3, an FGFR2 paralog. FGFR3 E18 truncations primarily occurred due to rearrangements (REs) that involve transforming acidic coiled-coil containing protein 3 (TACC3), resulting in FGFR3ΔE18-TACC3 gene fusions. In contrast to E18-truncated FGFR2, functional in vitro and in vivo examination of Fgfr3 variants demonstrated that the truncation of Fgfr3 E18 is insufficient to promote oncogenic activity in cell lines or in the lungs and mammary glands of mice. Only the combination of an Fgfr3 E18 truncation with a RE partner gene that encodes a receptor-dimerizing domain resulted in the development of tumors, which were sensitive to FGFR inhibition. Overall, these findings suggest that patients with cancers that are positive for rearranged FGFR3 resulting in E18 truncation and a fusion to dimerizing partners should be considered for FGFR-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a9640747833b06de97dd43802396e30e71e136" target='_blank'>
              C-terminal Truncation and Fusion Partner Determine Oncogenicity of FGFR3.
              </a>
            </td>
          <td>
            Julia Yemelyanenko, J. Bhin, Eline van der Burg, Anne Paulien Drenth, Jessica K. Lee, C. Lutz, Lea Dörner, E. Wientjens, S. Klarenbeek, Ji-Ying Song, Hyeonjin Moon, S. Annunziato, Natalie Proost, Bjørn Siteur, J. S. Ross, M. van de Ven, O. van Tellingen, Shridar Ganesan, L. Wessels, Daniel Zingg, Jos Jonkers
          </td>
          <td>2025-12-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Intestinal metaplasia (IM) is a premalignant condition associated with increased risk of gastric cancer-a deadly malignancy with varying geographic incidence. High-depth targeted sequencing of more than 1,500 IM samples from six countries identified 47 significantly mutated genes, including driver genes associated with high-risk populations and worse prognosis (ARID1A), KRAS/MAPK signaling (KRAS, BRAF, MAP2K1, MAP3K1, and MAP2K4), and altered mucosal immunity (PIGR). IM whole-genome sequencing and DNA methylation analysis revealed SBS17 as a specific mutational signature separating IMs from normal gastric tissues, associated with late DNA replication, genomic hypomethylation, and tobacco exposure. Beyond epithelial-derived somatic mutations, we observed elevated clonal hematopoiesis (CH) in patients with IM associated with age, smoking, and enhanced risk of progressing to gastric cancer. Patients with CH expansions exhibited co-occurring IM PIGR truncating mutations and greater colonization of the IM microenvironment by orally derived bacteria, suggesting that CH may promote IM progression by modulating host-microbe mucosal immunity.


SIGNIFICANCE
This international study identifies recurrent IM driver genes, IM-specific mutational signatures, and alterations in IM-associated immune landscapes and microbiomes. Our results highlight a role for nonepithelial somatic alterations (CH) in IM progression to gastric cancer, offering new translational opportunities for early cancer detection and interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f6a437b6939f544182c7d61adabe509b3b8f5bd" target='_blank'>
              Mutational Signatures and Clonal Hematopoiesis in Intestinal Metaplasia across Countries with Varying Stomach Cancer Incidence.
              </a>
            </td>
          <td>
            Kiekyong Huang, T. Hagihara, Benedict Shi Xiang Lian, Zhi Xuan Ong, Shen Kiat Lim, R. Chong, Supriya Srivastava, Jason Xing Kang, May Yin Lee, A. Tan, M. Lee, Shamaine Wei Ting Ho, Siti Aishah Binte Abdul Ghani, Clara Shi Ya Ng, Ruanyi Liang, Lin Liu, S. Tay, Xuewen Ong, Feng Zhu, Hui Chen, Zhen Li, T. Ang, Takuji Gotoda, Robert J. Huang, Christopher J L Khor, Hyun-Soo Kim, L. H. Lau, Yi-Chia Lee, Ayaka Takasu, Ming Teh, M. Thian, Wai Leong Tam, Xin Lu, S. Wong, J. B. So, H. Chung, J. Lee, K. Yeoh, Patrick Tan
          </td>
          <td>2026-01-14</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ae85bd79cada9f97140c874baf065a8524c49a" target='_blank'>
              NF1 Loss Remodels Tumor Niches for Immune Evasion
              </a>
            </td>
          <td>
            Milad Ibrahim, Irineu Illa-Bochaca, Tara Muijlwijk, Ines Delclaux, K. S. Ventre, G. Jour, Paola Angulo Salgado, Shi Qiu, Agrima Dutt, Amanda W. Lund, Iman Osman, Markus Schober
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="ERK activity governs diverse cellular responses and has significant implications in cancer biology and treatment. Cellular heterogeneity is a major feature of cancer and a barrier for therapy success, allowing cancer cells to adapt and survive in challenging environments. Here, we used a genetic live-cell reporter to explore the heterogeneity of ERK signaling activity within cellular populations and colonies of glioblastoma (GB) cells. GB cells showed a wide spectrum of ERK activation levels in basal culture conditions and throughout state transitions. Treatment with the chemotherapeutic agent temozolomide increased the phenotypic heterogeneity in ERK activity within cells even in clonal populations. Using the MEK inhibitor trametinib in combination with temozolomide to homogenize ERK activity reduced cell fitness in colonies and decreased fractional killing in GB clonal cells. Our study contributes to the growing understanding of the complexity in ERK activity and dynamics, pointing out the consequences of cell-to-cell ERK phenotypic variability in fitness and therapy survival. The complexity of ERK signaling phenotypes in the context of chemotherapy treatment is shown, offering valuable insights about the intricacies of ERK signaling heterogeneity and chemotherapy treatment. Heterogeneity in ERK activity in live GBM cells is high in basal culture conditions. Temozolomide alters ERK activity in GBM cells and generates phenotypic heterogeneity. Clonal populations behave heterogeneously in ERK activity, and this heterogeneity impacts fitness. Targeting the generation of ERK heterogeneity reduces fitness and fractional killing in GBM colonies. Heterogeneity in ERK activity in live GBM cells is high in basal culture conditions. Temozolomide alters ERK activity in GBM cells and generates phenotypic heterogeneity. Clonal populations behave heterogeneously in ERK activity, and this heterogeneity impacts fitness. Targeting the generation of ERK heterogeneity reduces fitness and fractional killing in GBM colonies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eba7cfb5da57498c2e07248e865b70479b2da77" target='_blank'>
              Temozolomide increases the generation of cell heterogeneity in ERK activity in glioma cells
              </a>
            </td>
          <td>
            K. R. Begnini, J. Marcolin, Luiza Cherobini Pereira, Letícia Cunha Pereira de Souza, Frederico Kraemer-Mattos, Daphne Tórgo, Carolina Machemer, Georgia da Silva Goulart, Andrew Oliveira Silva, Jephesson A. dos Santos, Guido Lenz
          </td>
          <td>2026-01-29</td>
          <td>Journal of Molecular Medicine (Berlin, Germany)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Fusion oncoproteins generated by chromosomal translocations are defining molecular drivers of several pediatric cancers, functioning in part by reprogramming gene expression and disrupting cellular differentiation. Ewing sarcoma, an aggressive cancer of bone and soft tissue, exemplifies this paradigm and is driven in the majority of cases by the oncogenic fusion protein EWS::FLI1, formed by joining the low-complexity domain (LCD) of the RNA-binding protein EWS to the DNA-binding domain (DBD) of the transcription factor FLI1. EWS::FLI1 alters transcriptional and RNA processing programs and assembles into aberrant biomolecular condensates, yet the molecular basis for these fusion-driven assemblies remains poorly understood. Here, we integrate NMR spectroscopy, mutagenesis, biophysical and cellular assays, microscopy, and all-atom molecular dynamics simulations to dissect how sequence features and domain interactions shape the condensate properties of EWS and EWS::FLI1. We find that the EWS LCD undergoes extensive self-association mediated by tyrosine-rich motifs that generate compact intramolecular conformations and robust multivalent networks driving phase separation. The RNA-binding domains of EWS engage the LCD through transient electrostatic and aromatic interactions, modulating condensate dynamics, whereas the FLI1 DBD of the fusion protein directly interacts with the EWS LCD via flexible “wings”. Incorporation of the FLI1 DBD markedly alters the material properties of condensates, producing emergent biophysical behaviors distinct from either parental protein. These altered condensate properties possibly contribute to the cellular dysregulation and toxicity associated with EWS::FLI1, in part through mis-localization and aberrant interactions with native EWS that disrupt normal nuclear organization and RNA processing. Together, these findings define a molecular framework in which weak, multivalent, and chemically specific interactions give rise to the emergent properties of the EWS::FLI1 fusion protein. By linking sequence-encoded features, interdomain crosstalk, and condensate material states, we propose a mechanistic description of how fusion with ETS-family DBDs reprograms the physical behavior of EWS assemblies. These emergent condensate properties provide a potential unifying description of how EWS::FLI1 perturbs transcriptional regulation, RNA metabolism, and nuclear organization in pediatric Ewing sarcoma.



 David S. Libich, Erich J. Sohn, Kandarp A. Sojitra, Aiola Stoja, Courtney N. Johnson, Jeetain Mittal, Alex A J. Bishop. Emergent condensate behaviors of the EWS::FLI1 fusion protein in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6d38099a1e2be11dc3ccd83fb0db5c31397244f" target='_blank'>
              Abstract B005: Emergent condensate behaviors of the EWS::FLI1 fusion protein in Ewing sarcoma
              </a>
            </td>
          <td>
            David S. Libich, Erich J. Sohn, Kandarp A. Sojitra, Aiola P. Stoja, Courtney N. Johnson, J. Mittal, Alex A J. Bishop
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8aedd893b33c0856e32b7cec42a8d28dc249d86" target='_blank'>
              System-wide extraction of cis-regulatory rules from sequence-to-function models in human neural development
              </a>
            </td>
          <td>
            Seppe De Winter, C. Mannens, Valerie Christiaens, Roel Vandepoel, Lijuan Hu, Sten Linnarsson, S. Aerts
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncogenic mutations of SF3B1 are common in myeloid cancers, chronic lymphocytic leukemia (CLL) and select solid tumors. Their mechanistic basis for promoting oncogenesis has been investigated in detail, with the stereotyped missplicing of mRNA protein coding sequences most intensively studied. These changes, in genes such as MAP3K7, BRD9, and ABCB7, typically lead to loss-of-function, thus contributing to cancer pathogenesis.Here we systematically analyzed the impact of mutant SF3B1 on non-coding regions of mRNA transcripts across disease types, in both cell lines and primary patient specimens. This identified numerous novel and highly reproducible splicing alterations in such regions. Studies of one target gene, DCAF16, revealed multiple complex mutation-induced alterations in its 5' and 3' untranslated regions (5', 3' UTRs). Remarkably, these were mechanistically associated with increased DCAF16 protein levels in SF3B1 mutant cells, representing the first time that oncogenic SF3B1 has been shown to increase levels of a target protein in a gain-of-function manner. DCAF16 is a substrate recognition adapter for the DDB1/CUL4 E3 ubiquitin ligase complex. Novel protein degrader small molecules which co-opt DCAF16 to degrade BRD4 as a neosubstrate demonstrated preferential selectivity for SF3B1 mutant cancers and CLL primary patient specimens due to increased DCAF16 protein levels. In turn, this reveals the therapeutic relevance of mutant SF3B1 dysregulation of transcript untranslated regions and uncovers a novel strategy for the treatment of these important neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc898e0fdc805e68eb2bae19245eee2675983440" target='_blank'>
              Oncogenic SF3B1 mutations alter the splicing of mRNA noncoding regions to induce a novel therapeutic vulnerability.
              </a>
            </td>
          <td>
            Michal Sekrecki, Agata Sekrecka, Rohan Lattupally, Kenny Le, Xiaokang Jin, Chen Mozes, Brendan G. Dwyer, Zhe Zhuang, Bryan A. Romero, J. Pineda, Xunhong Cao, Linh Nguyen, Vicky W Chen, Crystal Zhou, Jared A Wallace, Kailee L Tanaka, Charu Tiwari, Austin Gabel, Won Jun Kim, Robert F Stanley, Salima Benbarche, Jaspreet Thind, Abhinaya Muruganandham, T. Y. Zhang, Peter L Greenberg, J. Gotlib, G. Mannis, William Shomali, G. Salmasi, Calvin J Kuo, T. Shanafelt, Irtisha Singh, Daichi Inoue, Finn K. Hansen, N. Gray, Capucine Van Rechem, B. Fakhri, Xiaoyu Zhang, Sydney X. Lu
          </td>
          <td>2026-01-26</td>
          <td>Blood</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c08de5eb400d357e336428d7f7a366a3a038cb3" target='_blank'>
              Identification of RPGRIP1L as an instability-maintaining gene to drive tumor growth and PD-L1 expression via Hedgehog signaling in breast cancer.
              </a>
            </td>
          <td>
            Meng Lv, Yan’e Liu, Weihong Cao, Yongmei Wang, Qi Wang, Xue-Qiang Gao, Baowei Peng, Haibo Wang, Yan Mao
          </td>
          <td>2025-12-30</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c041d5587b3ab9e813f645db331f46877e736665" target='_blank'>
              The emerging role of SPHK1 at the immune-metabolic interface: a pan-cancer integrative analysis.
              </a>
            </td>
          <td>
            Lei Wang, Guodong Zhong, Hao Luo, Qiao He, Yan Chen, Wei Li, Qiuju Wang
          </td>
          <td>2026-01-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the second leading cause of cancer-related deaths among men in the U.S. Overexpression of PVT1 exon 9 has been shown to transform non-tumorigenic prostate epithelial cells into aggressive tumor-forming cells. We have recently identified a non-canonical promoter within PVT1 that is associated with PVT1 exon 9 expression and hypothesize that epigenetic factors like DNA methylation and transcription factors regulate its overexpression in prostate cancer, given that gene expression is modulated by epigenetic alterations. Using enzymatic methyl-sequencing (EM-Seq), we profiled tumor and adjacent normal prostate tissues. At a differential methylation threshold of ≥±5% and q ≤0.05, we identified 1050 differentially methylated cytosines (DMCs), with 108 hypomethylated and 80 hypermethylated sites in tumor tissues. Regional tiling analysis (1kb windows) also revealed 172 differentially methylated regions (DMRs), including 42 hypomethylated and 16 hypermethylated DMRs in tumor tissue. PVT1 exon 9-associated promoter was visibly hypomethylated in tumors relative to adjacent normal tissues. In vitro, 5-azacytidine (a DNA methyltransferase inhibitor) treatment induced significant upregulation of PVT1 exons 4A and 9 in WPE1-NA22, DU145, PC-3 and MDA-PCa-2b cell lines versus the non-tumorigenic RWPE-1 cell line. Computational transcription factor analysis revealed CDX1 binds to the newly-identified non-canonical promoter, while CRISPR-mediated base editing of its signal sequence reduced PVT1 exon 9 expression in PVT1 exon 9 overexpressing cells. In addition, CDX1 expression was also found to have a positive correlation with high grade PCa. These findings indicate that PVT1 exon 9 overexpression and oncogenicity in PCa is driven by promoter hypomethylation and CDX1 regulation.



 Chinedum C. Udekwu, Siti Khaula. Nurazizah, Seidu Adams, Bonnacci Rachel, E.Oluwabunmi Olapade-Olaopa, Olorunseun O. Ogunwobi. The oncogenic role of PVT1 exon 9 overexpression in prostate Cancer is mediated by DNA methylation and CDX1 [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a9e20af1415b799b87aab15045ca553659c6c2" target='_blank'>
              Abstract A065: The oncogenic role of PVT1 exon 9 overexpression in prostate Cancer is mediated by DNA methylation and CDX1
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Siti Khaula. Nurazizah, Seidu Adams, Bonnacci Rachel, E. Oluwabunmi Olapade-Olaopa, O. Ogunwobi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="INTRODUCTION
Chromosomal instability (CIN), a hallmark of over 90% of solid tumors, presents both a vulnerability and an opportunity in cancer therapy. KIF18A, a kinesin motor protein essential for mitotic fidelity, becomes critical for CIN cancer cells, where it suppresses lethal mitotic errors. This selective dependency makes KIF18A an attractive therapeutic target.


AREAS COVERED
This review explores the structural and functional biology of KIF18A, its role in maintaining chromosomal alignment, and how its inhibition leads to mitotic failure specifically in cancer cells exhibiting high CIN. We summarize the development of small-molecule inhibitors, from early compounds like BTB-1 to second-generation agents including Sovilnesib and VLS-1488, the latter currently in Phase I/II trials. Clinical data show manageable toxicity and promising antitumor activity, particularly in high-grade serous ovarian cancer. New entrants like ATX-295 and GH2616 further expand the landscape. We also highlight efforts to refine KIF18A targeting through structural studies and emerging AI-driven drug discovery.


EXPERT OPINION
KIF18A inhibition represents a mechanistically grounded, tumor-selective strategy for treating CIN-driven cancers. As clinical trials progress, optimizing patient stratification and exploring synergistic combinations will be crucial. Broadening the range of druggable sites on KIF18A, especially beyond the motor domain, could mitigate resistance and help expand therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b172472dda432c5dc196b15477d3bfabc81ddaf1" target='_blank'>
              KIF18A inhibition and its rising role in cancer therapy trials: from bench to bedside.
              </a>
            </td>
          <td>
            Jia Q Truong, Tal Galper, Jessica K. Holien
          </td>
          <td>2026-01-16</td>
          <td>Expert opinion on investigational drugs</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Prostate cancer (PC) is one of the most prevalent malignancies in men, with rising incidence and mortality rates globally. Despite advances in therapeutic options such as androgen deprivation therapy and chemotherapy, effective cures, especially for advanced stages of the disease, remain limited. Recent research has highlighted the significant roles of alternative splicing (AS) and noncoding RNAs in tumor progression and drug resistance. This study aims to investigate the role of circIMP3, derived from the IMP3 gene, in prostate cancer development. Methods In this study, we employed quantitative PCR, RNA sequencing, and immunoblotting to identify and characterize circIMP3 in prostate cancer tissues and patient blood samples. Functional assays, including cell proliferation and in vivo tumorigenicity assays, were conducted to assess the biological role of circIMP3 in PC cells. RNA immunoprecipitation sequencing (RIP-seq) was used to identify alternative splicing events regulated by circIMP3. Additionally, exosome isolation and uptake assays were performed to explore the paracrine signaling function of circIMP3 within the tumor microenvironment (TME). Results We identified circIMP3, which is significantly upregulated in both prostate cancer tissues and peripheral blood of patients. CircIMP3 contains an internal ribosome entry site (IRES) and encodes a previously uncharacterized 288-amino-acid protein, circIMP3_288aa. Functional assays revealed that circIMP3_288aa promotes cell proliferation in vitro and accelerates tumor growth in vivo. Mechanistically, circIMP3_ 288aa regulates the alternative splicing of FBXW7, leading to impaired c-Myc ubiquitination and stabilization, which enhances oncogenic signaling. RIP-seq analysis identified over 2,000 alternative splicing events regulated by IMP3, with a notable enrichment in pathways related to ubiquitin-mediated proteolysis. Furthermore, circIMP3 is secreted into the TME via exosomes, where it is taken up by recipient cells, contributing to their proliferation. Discussion Our findings demonstrate that circIMP3 acts as a key regulator of both intracellular alternative splicing and extracellular paracrine signaling within the TME. The ability of circIMP3 to influence FBXW7 splicing and stabilize c-Myc provides a mechanistic basis for its role in promoting oncogenesis in prostate cancer. Clinically, high expression levels of circIMP3 correlate with poorer event-free survival in prostate cancer patients, suggesting its potential as a prognostic biomarker. Additionally, the detection of circIMP3 in peripheral blood positions it as a promising target for liquid biopsy applications in PC diagnosis and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44898723fc23dea0100abadafda89694a83449" target='_blank'>
              A novel protein encoded by circIMP3 promotes prostate cancer progression by regulating alternative splicing and tumor microenvironment
              </a>
            </td>
          <td>
            Wenren Zuo, Weizhou Huang, Haojie Chen, Yan Xu, Yang Zhang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b5f103c6a6802e94ba070307c1c9a2755a26be" target='_blank'>
              A telomere-to-telomere genome assembly for Cyperus difformis.
              </a>
            </td>
          <td>
            Jingbo Li, Jian Zhao, Wei Zheng, Yonglin Ma, Meisha Qiu, Weisheng Li, Yanhui Wang
          </td>
          <td>2026-01-15</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cyclin-dependent kinase 12 (CDK12), a transcription-associated cyclin-dependent kinase (tCDK) is a tumor suppressor in prostate cancer, with biallelic loss observed in ∼1% of newly diagnosed and ∼5% of castration-resistant prostate cancer (CRPC) cases. CDK12-lost tumors are aggressive and are associated with poorer survival and rapid progression to castration resistance. CDK12 functions are essential for cell viability, limiting the mechanistic studies of its loss-of -function. In tumors, its loss is characterized by unique genomic features, notably large tandem duplications on a largely diploid background. As a tCDK, CDK12 phosphorylates RNA polymerase II and regulates transcriptional elongation and processivity. Genetic depletion or inhibition of CDK12 causes aberrant splicing, premature polyadenylation, and transcriptional termination, particularly affecting genes in the homologous recombination (HR) pathway. These defects provided a rationale for therapies such as PARP inhibitors or immune checkpoint blockade, but clinical trials have largely failed. Thus, we hypothesized that tumor cells evolve compensatory mechanisms to sustain CDK12 loss, possibly by engaging another tCDK. To better recapitulate the chronic effects of CDK12 loss observed in tumors, we utilized patient-derived organoid (PDO) models with established CDK12 loss for our studies. Using these models, we evaluated the growth-inhibitory response of CDK12-deficient PDOs to selective inhibitors targeting various transcription-associated CDKs. Notably, we found that all the CDK12-lost PDOs (n=4) were exceptionally sensitive to THZ531, an inhibitor targeting CDK12 and CDK13 (a paralog of CDK12). Next, we analyzed the global transcriptional/co-transcriptional processes in CDK12-lost and CDK12-intact models at baseline and following treatment with THZ531. Mechanistically, we found that THZ531 treatment induced increased splicing errors in CDK12-deficient organoids compared to CDK12-intact controls. The most frequent splicing aberrations were seen in intron retention and exon skipping. These findings indicate that CDK13 plays a compensatory role in maintaining transcriptional and splicing homeostasis in CDK12-deficient settings, and that targeting CDK13 represents a potential therapeutic strategy for this aggressive prostate cancer subtype.



 Shipra Shukla, Dana Schoeps, Nicholas Terri, Jahan Rehman, Omar Abdel-Wahab, Yu Chen. Understanding tumorigenesis and identifying therapeutic targets in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B069.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5bc798d42a1b153398baf489ee461849a97c163" target='_blank'>
              Abstract B069: Understanding tumorigenesis and identifying therapeutic targets in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Shipra Shukla, Dana M. Schoeps, Nicholas Terri, Jahan Rehman, Omar Abdel-Wahab, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9ee68261862574cb9b7d1551d82bf740d99cfd" target='_blank'>
              Elevated mutation in haploid yeast driven by translesion synthesis
              </a>
            </td>
          <td>
            Jacob Fredette-Roman, Denise R. Smith, Sanad B. Omari, Nathaniel P. Sharp
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) resulting from inactivation of BRCA1/2 genes promotes chromosomal instability and renders tumor cells susceptible to platinum derivatives and PARP inhibitors (PARPi). The contribution of alterations in other homologous recombination repair (HRR) genes to HRD remains understudied. This investigation aimed to analyze the spectrum of mutations in 34 HRR genes in prostate carcinomas (PCs) and study the relationship between HRR status and HRD. HRR mutations and HRD scores were examined by NGS in 1131 and 680 PCs, respectively. Pathogenic or likely pathogenic variants in HRR genes were detected in 216/1131 cases (19.1%). HRD, defined by an HRD score cut-off of ≥42, was observed more frequently in HRR-mutated than in wild-type tumors (23/120 (19.2%) vs. 29/560 (5.2%), p < 0.0001). The highest HRD scores were detected in PCs with biallelic inactivation of the BRCA2 or PALB2 genes, as well as in tumors with BRIP1 mutations. HRD was also occasionally seen in PCs with ATM, NBN, FANCM, BRCA1 and CDK12 alterations, but never in cases with CHEK2 mutations. HRD was significantly more associated with aggressive PC features than HRR mutations. The majority of CDK12-mutated tumors exhibited a distinct type of copy number variations (CNV)–a tandem duplication phenotype. Our study suggests that the selection of PC patients for PARPi treatment requires a significant revision of existing attitudes towards tumor genetic profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4a6b0e731768bb34ffda6a1da1c394d76e62" target='_blank'>
              Complex Relationships Between Homologous Recombination Deficiency (HRD) Score and Mutational Status of Homologous Recombination Repair (HRR) Genes in Prostate Carcinomas
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, A. Shestakova, E. Kuligina, N. S. Morozova, Sergey V. Popov, Pavel V. Vyazovcev, Tatyana Y. Luchkova, Aleksey S. Peremyshlenko, Timur M. Topuzov, Olga M. Gudkova, R. V. Orlova, Andrey V. Levushkin, Daniil O. Moiseev, Oksana N Shkodenko, Demyan V. Gubarev, A. V. Dzyuba, Irina Y. Povorina, Anna V. Agaeva, V. F. Kutyan, Mhitar M. Grigoryan, Alexey N. Orlov, Spartak I. Lefterov, Aleksey V. Belousov, M. N. Nechaeva, E. Vorobyeva, Irina K. Amirkhanova, Nadezhda V. Kryukova, Lyubov I. Zatulivetrova, Aryuna B. Gomboeva, Vladimir N. Krivtsov, Olga I. Shchiglik, Natalya V. Prokudina, Natalya A. Butunina, A. M. Belyaev, E. Imyanitov
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92529c21b1d903977db7b6f99cc016a10cc689ad" target='_blank'>
              Molecular analysis of tumor recurrence using established cancer stem cell-line and drug discovery.
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, Y. Zaitsu, Chiyoko Nishime, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, T. Kawazoe, Kentaro Nonaka, Keita Natsugoe, Sachie Omori, H. Saeki, Hiroyuki Aburatani, Tatsumi Yamazaki, Yoshihiko Maehara
          </td>
          <td>2026-01-20</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Sorex araneus, the Eurasian common shrew, has seasonal brain size plasticity (Dehnel's phenomenon) and abundant intraspecific chromosomal rearrangements. Genomic contributions to these traits, however, remain unknown. We couple a chromosome-scale genome assembly with seasonal brain transcriptomes to discover relationships between molecular changes and both traits. While positively selected genes enriched the Fanconi anemia DNA repair pathway (FANCI, FAAP100), which is likely involved in chromosomal rearrangements by preventing the accumulation of chromosomal aberrations, genes under positive selection or showing seasonal differential expression in the brain implicate neurogenesis (PCDHA6, SOX9, Notch signaling) and metabolic regulation (VEGFA, SPHK2) as key mechanisms underlying Dehnel's phenomenon. We also find that both positively selected and differentially expressed genes in the hippocampus are overrepresented near Sor. araneus evolutionary breakpoints. This relates both positive selection and differential expression to accessible chromatin configuration, suggesting that chromosomal rearrangements are integral to adaptive evolution and the regulation of brain size plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e5d59bc3d794d733f5961db0a47844813d84f1" target='_blank'>
              Genomic comparisons shed light on the adaptive basis of brain size plasticity and chromosomal instability in the Eurasian common shrew.
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Cecilia Baldoni, Laia Marín-Gual, Diana D Moreno Santillán, Marta Farré, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Ying Sims, Alan Tracey, Jonathan M. D. Wood, David A. Ray, Dominik von Elverfeldt, J. Nieland, Angelique P. Corthals, A. Ruiz-Herrera, D. Dechmann, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2026-01-09</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ANLN is a highly conserved actin-binding protein that plays a critical role in cytokinesis by interacting with key cytoskeletal components such as actin, myosin, and RhoA. Increasing evidence shows that ANLN is aberrantly overexpressed in various cancers, including lung, breast, and liver cancers, and that its elevated expression is associated with enhanced tumor cell proliferation, migration, and invasion. Because of its central involvement in tumorigenesis and disease progression, ANLN has emerged as a promising prognostic biomarker and potential therapeutic target. Recent studies have demonstrated that ANLN contributes to resistance to chemotherapy, targeted therapy, and immunotherapy through multiple molecular mechanisms. This review provides a systematic overview of the physiological functions of ANLN, its roles in cancer initiation and progression, and its regulatory mechanisms in treatment resistance, offering biological insights into precision oncology and potential strategies for overcoming therapeutic resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a40af0d42ab2b0397c3e4bc1fc5ecc54b07af4" target='_blank'>
              The role of ANLN in malignant tumors: pathogenesis, treatment resistance and targeted strategies
              </a>
            </td>
          <td>
            Yixuan Wang, Yang Xiao, Jielin Yu, YuanHua Zou, Zhe Wang, Xianghong Yang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Regulation of gene expression is central to the development of immune cells and their ability to respond to infection. As part of a clinical evaluation, we identified two sisters with recurrent infections, hypogammaglobulinemia, and memory B cell deficiency, diagnosed as common variable immunodeficiency. Whole exome sequencing identified a heterozygous variant (Leu50Ser, L50S) in a conserved region of EZH2, the catalytic subunit of the epigenetic gene repressor Polycomb Repressive Complex 2 (PRC2). EZH2-catalyzed histone H3 lysine 27 methylation (H3K27me) in bulk was not overall significantly disrupted by this variant, in patient samples or cell lines expressing EZH2-L50S. EZH2-L50S protein is expressed similar to wild-type and can form PRC2. However, we find that specific genomic regions that normally have high wild-type levels of H3K27me3 are deficient in the L50S context, particularly around gene promoters. EZH2-L50S is still recruited to these sites, but is not as active. Using recombinant purified PRC2, we determine that L50S affects methylation of nucleosomes and disrupts allosteric stimulation that normally amplifies H3K27me3, consistent with the location of L50 in the allosteric regulatory region of PRC2. Thus, variation of EZH2 L50, occurring at low frequency in the population may interfere with normal B cell gene expression patterns, contributing to immunodeficiency. This study has implications for genetic variation in PRC2 in the general population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a59bd0112470a94559bebc44def6044177da01" target='_blank'>
              Characterization of an immunodeficiency-associated EZH2 variant
              </a>
            </td>
          <td>
            Francisco Perez de los Santos, Lily Beck, Emily DeCurtis, April M. Griffin, Maggie M. Balas, Charlotte Marchioni, Attila Kumánovics, Matthew R. G. Taylor, Kimberly R. Jordan, Charles H. Kirkpatrick, Aaron M. Johnson
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>133</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Systemic sclerosis (SSc) is a rare, complex autoimmune disease characterized by fibrosis of the skin and internal organs. While its pathogenesis is not fully understood, chromosomal instability and telomere attrition have emerged as significant areas of investigation. Methods: This review provides a historical narrative perspective and synthesizes current findings on the role of these genomic anomalies in SSc pathogenesis. We synthesized findings from foundational and recent research articles investigating genotoxic factors, chromosomal aberrations, and telomere biology in SSc. Results: There is a strong historical basis for chromosomal instability in SSc, manifesting as micronuclei, translocations, and breaks. This instability is driven by clastogenic factors and oxidative stress. SSc-specific autoantibodies are implicated; anti-centromere antibodies correlate with aneuploidy and micronuclei, while anti-topoisomerase I may inhibit DNA repair. SSc is also characterized by significant telomere attrition, first reported in 1996 and now confirmed by additional genetic studies. This telomere loss is associated with reduced telomerase activity and the presence of autoantibodies against telomere-associated proteins, including shelterin components. Conclusions: We conclude that inflammation, telomere attrition, and chromosomal instability are linked in a self-perpetuating cycle that drives SSc pathogenesis. We propose that an initial inflammatory stimulus leads to reactive oxygen species production, causing telomere damage and attrition. Critically short telomeres trigger faulty DNA repair mechanisms, such as breakage–fusion–bridge cycles, resulting in chromosomal instability. This genomic damage, in turn, acts as a danger signal, further activating inflammatory pathways and creating a feedback loop that perpetuates fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a5b157843adfebc89c062c9bea135fbbb56036" target='_blank'>
              Chromosomal Instability and Telomere Attrition in Systemic Sclerosis: A Historical Perspective
              </a>
            </td>
          <td>
            C. M. Artlett
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Treatments for childhood brain cancer, the leading cause of disease mortality in children, has not changed in the last three decades and constitutes surgery combined with cytotoxic chemo- and radio- therapy. RT is not always curative, and the mechanisms that promote resistance to radiation for many childhood cancers remains elusive. We found the ZFTA-RELA (ZR) fusion oncoprotein (FO), a potent driver of Ependymoma (EPN), drives a radio-resistant phenotype, but the mechanism underlying this observation is unknown. We hypothesized that radio-resistance mediated by ZR arises from direct ZR driven transcription. Analysis of numerous mouse and human datasets reveals ZR upregulates pathways associated with radio-resistance. We demonstrate the importance of ZR transcriptional programs on radiosensitivity by abrogating the DNA binding ability of ZR to sensitize EPN cells to RT. To identify ZR target genes that mediate radiosensitivity, we performed CRISPR-Cas12 dropout screens in both mouse and human EPN cell lines and identified >150 genes that sensitize EPN to RT, including canonical regulators of homologous recombination repair and the NFKβ pathway. In parallel, we performed high throughput drug screening to identify pharmacologic radiosensitizers in EPN and identified several combinations containing PARP1 inhibitors that are effective in vitro and tolerable in vivo. These data represent novel efforts to identify genetic and chemical radiosensitizers in EPN, and demonstrate how fusion oncoproteins like ZR hijack normal transcriptional programs to promote resistance to conventional therapy.



 Blake Holcomb, Ha Won Lee, Pete Hall, Shilpa Narina, Amanda Bland, Siri Ippagunta, Tuyu Zheng, Amir Arabzade, Nic Laboe, Patrick Connelly, Shondra Miler, Taosheng Chen, Stephen Mack. Elucidating the role of ZFTA-RELA in radiation resistance in ependymoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f094996aad3c3897c354e021ac68992322ee6553" target='_blank'>
              Abstract B025: Elucidating the role of ZFTA-RELA in radiation resistance in ependymoma
              </a>
            </td>
          <td>
            B. Holcomb, Ha Won Lee, Pete Hall, S. Narina, A. Bland, S. Ippagunta, T. Zheng, A. Arabzade, Nicolas Laboe, Patrick Connelly, Shondra Miler, Taosheng Chen, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aim. The current study aims to investigate changes in the expression of proprotein convertases (PCs) genes during the development of lung tumors. PCs is a family of highly specific serine endopeptidases of mammals that process precursors of various proteins and peptides. Nine genes encoding PCs have been identified in the human genome that are essential for normal body functioning. Additionally, PCs can activate proteins involved in carcinogenesis, as well as the expression levels of their genes were shown to correlate with tumor aggressiveness and patient survival. We previously evaluated the expression of all nine PC genes using quantitative PCR on paired samples of lung tumor and adjacent normal tissue. For the first time, we identified four distinct patterns of PCs gene expression change in tumor tissue, which we have called «scenarios». For three of them, covering more than two thirds of the samples, a dominant change in the expression of one PC gene in the tumor tissue was shown. These results may indicate the existence of a limited number of possible options for changes in PCs gene expression during the malignant transformation of lung cells. However, these results need to be confirmed using expanded cohort of tumor samples. Material and Methods. To confirm our previous findings, we analyzed the expression of PCs genes using modern methods of mathematical statistics and data from three previously published studies, which evaluated gene expression through high-throughput RNA sequencing in paired tumor and normal tissue samples from 194 patients with non-small cell lung cancer. Results. Our meta-analysis confirmed that the changes in PCs gene expression in lung tumor tissue compared to surrounding normal tissue follow a limited set of possible scenarios, each having a unique profile of PCs gene expression. Conclusion. The reasons for implementing each scenario may be linked to the origin of tumors, their mutation status, characteristics of the tumor microenvironment, and other factors. Correspondingly, these scenarios may correlate, for example, with tumor aggressiveness and resistance to therapy, and therefore may potentially be used to choose the treatment approach and/or to predict the course of the disease. However, this issue requires further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3af10872808404c4c77f8841898be6b45125c83" target='_blank'>
              Alterations in proprotein convertase genes’ expression during malignization of human lung cells have a limited set of scenarios
              </a>
            </td>
          <td>
            A. A. Komissarov, I. N. Vlasov, P. A. Slominsky, I. Demidyuk
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2967424876b05ad4fc08b0dd3adaa2e4feebc8f7" target='_blank'>
              Deep meta-analysis of human pangenome data reveals triplication of the IGH constant gene locus
              </a>
            </td>
          <td>
            Pietro D’Addabbo, Rossella Cianci, Domenico Frezza
          </td>
          <td>2026-01-17</td>
          <td>Discover Immunity</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is an aggressive pediatric soft tissue cancer, with approximately 350 cases per year in the US. The highly aggressive tumor subtype, Fusion-Positive Rhabdomyosarcoma (FP-RMS) is characterized by chromosomal translocations yielding oncogenic fusion transcription factors (TFs). The most common RMS fusion (PAX3::FOXO1) forms an in-frame chimera of the DNA-binding domain of PAX3 with the activator domain of FOXO1. We hypothesize that understanding the precise mechanisms of PAX3::FOXO1’s chromatin engagement and nucleosome invasion would enable precision therapy to target these mechanisms to ultimately improve patient outcomes. Our studies are providing evidence that PAX3::FOXO1 has pioneer activity, including nucleosome engagement, and an ability to recognize multiple chromatin states including both open and repressive chromatin regions. We are investigating the biochemical regulatory functions of PAX3::FOXO1 binding to specific DNA motifs embedded within chromatin or nucleosomes, developing synthetic epigenetics to investigate motifs from our genomics-scale studies. We are interested in the preservation of naturally occurring post-translational modifications from the mammalian cellular context, and have sought to build a highly versatile system for investigation of TF/nucleosome targeting from cell extracts. In our studies, we are finding new evidence for PAX3::FOXO1’s targeting of homeodomain and paired domain DNA motifs in free DNA and also in a nucleosome context. We are validating these findings through parallel approaches including coupling electrophoretic mobility shift assays (EMSAs) with westerns and antibody-targeting of complexes within EMSA experiments. We are finding that PAX3::FOXO1’s binding to high-complexity motifs, conjoining paired domain and homeodomain sequences, occurs during nucleosome invasion. Through iterative comparisons, custom unmodified mono-nucleosomes containing three major classes of DNA binding motifs reveal PAX3::FOXO1’s binding to these nucleosomes without a strong motif preference within the categories examined in our series. These findings support a mechanistic hypothesis where PAX3::FOXO1 may invade nucleosomes, altering their local positioning prior to generation of chromatin accessibility. Additional investigations are underway to unravel these mechanisms further and probe the contexts for motif specificity or generality in nucleosome engagement, and to understand conformational changes in the target nucleosomes. The mechanisms we are uncovering will contribute to the search for new therapeutic vulnerabilities as our community progresses toward targeted therapies for FP-RMS.



 Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic Barr, Michael G. Poirier, Benjamin Stanton. Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbbbd914885e1df8de9eefd492cd42d843d809e" target='_blank'>
              Abstract B016: Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic G. Barr, Michael G. Poirier, Benjamin Z. Stanton
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer (OC), the third most common gynecologic malignancy, is characterized by high mortality largely driven by chemotherapy resistance, leading to recurrence and metastasis. Using transcriptomic data from GSE73935, we constructed a weighted gene co-expression network and identified eight hub genes (IGF1R, CDH2, PDGFRA, CDKN1A, SHC1, SPP1, CAV1 and FGF18) associated with cisplatin resistance, among which CDH2 emerged as the most clinically relevant candidate. CDH2 demonstrated moderate diagnostic potential (AUC = 0.792) and was markedly upregulated in cisplatin-resistant A2780/CP70 cells. Independent validation using clinical single-cell RNA-seq data (GSE211956) confirmed its selective enrichment in resistant tumor cell subpopulations. Gene set enrichment analysis linked elevated CDH2 expression to p53 signaling, DNA replication, nucleotide excision repair, and Toll-like receptor pathways, with qPCR supporting upregulation of key downstream genes in resistant cells. Immune deconvolution further indicated that high CDH2 expression correlated with increased infiltration of NK cells, Tregs, macrophages, and neutrophils, and immunohistochemistry verified CDH2 overexpression in cisplatin-resistant tissues. In addition, virtual screening and drug sensitivity profiling identified several FDA-approved agents with potential relevance to CDH2-associated drug response. These findings indicate that CDH2 may serve as a candidate marker associated with cisplatin response in OC, and its association with immune cell infiltration provides further insight into mechanisms potentially underlying chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b261349cbc2ce5423ecbd97a555c9bd93cba98" target='_blank'>
              Integrated Analyses Identify CDH2 as a Hub Gene Associated with Cisplatin Resistance and Prognosis in Ovarian Cancer
              </a>
            </td>
          <td>
            Jun-Yi Xu, Mao-Qi Tian, Rui Yang, Zi-Xuan Li, Zi-Heng Lin, Yu-Fei Wang, Yu-Hang Chu, Wei-Ning Sun, Ya-Mei Wang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [144, 6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>