<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-17 09:40:51 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>41</td>
          <td>106</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>86</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>120</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>380</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>330</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, I. Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db525943b0ddbcca73f6e3d8fd119fef42f8fb23" target='_blank'>
              Aberrant inheritance of extrachromosomal DNA amplifications promotes cancer evolution
              </a>
            </td>
          <td>
            Shir Marom, Inbar Lifshits Dayan, Venkata Narasimha Kadali, Mădălina Giurgiu-Kraljič, Gabriela Koifman, Karen Hakeny, Madhuri Chaurasia, Orléna Benamozig, R. Nevo, I. Azuri, Julia Ryvkin, R. Rotkopf, Gil Stelzer, Nino Oniashvili, Jacques Mardoukh, Sarah Pollock, Nika Iremadze, Z. Shipony, Meital Kupervaser, N. Wigoda, Dena Leshkowitz, Z. Elazar, Esther Berko, A. Henssen, Ofer Shoshani
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21aaf9e66bf2842235cbcfb96477f51c53dc060" target='_blank'>
              Complex HPV-human DNA structures revealed by large-scale DNA analyses in an HPV-cancer derived cell line
              </a>
            </td>
          <td>
            Eleanor J. Agosta, Yoke-Chen Chang, Vandya Rao, Jessie Hollingsworth, Michelle Brown, Debajyoti Kabiraj, Mark H Einstein, A. Van Arsdale, Koenraad van Doorslaer, Chang Chan, Subhajyoti De, Advaitha Madireddy, Brian J. Haas, Danny E. Miller, Jack Lenz, Cristina Montagna
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, A. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, A. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This article presents an extensive review of the advancements in research concerning extrachromosomal circular DNA (ecDNA) within the context of breast cancer. As a distinct form of DNA, ecDNA is critically involved in the initiation, progression, diagnosis, and treatment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3be1ca65e1813757b54ee00006ec2026ac1e51f0" target='_blank'>
              Research Advances of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Xiaodan Zhu, Yiyang Qian, Quan Tang, Jiahui Li, Yuqing Geng, Shixia Gong, Chunhui Jin
          </td>
          <td>2025-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="It has been more than 20 years since the publication of the C57BL/6J mouse reference genome, which has been a key catalyst for understanding the biology of mammalian diseases. However, the mouse reference genome still lacks telomeres and centromeres, contains 281 chromosomal sequence gaps and only partially represents many biomedically relevant loci. Here we present the first telomere-to-telomere (T2T) mouse genomes for two key inbred strains, C57BL/6J and CAST/EiJ. These T2T genomes reveal substantial variability in telomere and centromere sizes and structural organization. We thus add an additional 213 Mb of new sequence to the reference genome, which contains 517 protein-coding genes. We also examined two important but incomplete loci in the mouse genome—the pseudoautosomal region (PAR) on the sex chromosomes and KRAB zinc-finger protein loci. We identified distant locations of the PAR boundary, different copy numbers and sizes of segmental duplications and a multitude of amino acid substitution mutations in PAR genes. Telomere-to-telomere assemblies of two mouse inbred strains, C57BL/6J and CAST/EiJ, offer improvements over the current mouse reference genome by adding telomere and centromere sequences that lead to insights into variability in telomere and centromere sizes and organization, and the discovery of 225 and 355 new genes for C57BL/6J and CAST/EiJ, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/526cf79eb8c8c6fe666d5ce7cd05dc886f05599e" target='_blank'>
              Complete genome assemblies of two mouse subspecies reveal structural diversity of telomeres and centromeres
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Tianzhen Wu, K. Zoltowska, Mohab Helmy, T. Kato, A. Czechanski, I. Bronner, Emma Dawson, M. A. Quail, Anne C. Ferguson-Smith, L. Reinholdt, David J Adams, Thomas M. Keane
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation. Radiation-induced malignantly transformed cells undergo multi-scale 3D genome reorganization. TP63 acts as a key mediator, creating accessible chromatin sites that form tumour-specific loops activating the MYC oncogene during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-01</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T‐ and B‐cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B‐cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by‐product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B‐cell leukemia
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6351708081597f9b8207513e5bf5a9d141739521" target='_blank'>
              OptoLoop: An optogenetic tool to probe the functional role of genome organization
              </a>
            </td>
          <td>
            Martin Stortz, Adib Keikhosravi, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2025-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration is recognized as a hallmark event in cervical carcinogenesis. However, it does not represent a routine phase of the viral life cycle but rather a stochastic occurrence, often constituting a dead-end pathway for the virus. High-risk human papillomavirus (hr-HPV) exhibits a greater propensity for integration. The progression from initial infection to genomic integration constitutes a dynamic multi-step oncogenic process in the development of cervical cancer (CC). This process involves viral entry, immune evasion, persistent infection, and ultimately integration. This article innovatively provides a comprehensive overview of this multi-stage mechanism: HPV, via the L1/L2 proteins, mediates internalization and establishes infection. Subsequently, under the influence of factors such as the host’s genetic background, vaginal microbiota imbalance, and immune evasion, the host’s DNA damage response (DDR) pathways are activated. Viral DNA integrates into host genome vulnerable sites (e.g., 3q28 and 8q24) through microhomology-mediated end joining (MMEJ) or other alternative pathways. Following integration, the expression of viral oncogenes persists, triggering host genomic rearrangements, aberrant epigenetic modifications, and immune microenvironment remodeling, all of which collectively drive cervical cancer progression. The study further reveals the clinical potential of HPV integration as a highly specific molecular biomarker, offering new perspectives for precision screening and targeted therapy. This dynamic model deepens our understanding of the HPV carcinogenic mechanism and provides a theoretical basis for intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2f3540f21b81b3f4af2b24e5ab200e71a6ae9d" target='_blank'>
              From Viral Infection to Genome Reshaping: The Triggering Role of HPV Integration in Cervical Cancer
              </a>
            </td>
          <td>
            Junlan Li, Shuang Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2c0bd20d360ee798769e48db594be4df9b6168" target='_blank'>
              Comparative analysis of single-stranded and non-canonical DNA formation in human and other ape cells with telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Jacob P. Sieg, Huiqing Zeng, Hana Pálová, Linnéa Smeds, S. Mohanty, Angelika Lahnsteiner, Francesca Chiaromonte, Kateryna D. Makova
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The genomic instability associated with cancer can result in the formation of extrachromosomal circular DNA (eccDNA), which contributes to tumor heterogeneity, gene amplification, tumor evolution, and drug resistance. However, most studies on eccDNA have been conducted on tumor tissue or cancer cell lines, and limited research has been done on eccDNA in plasma. In this study, we investigated eccDNA in non-small cell lung cancer (NSCLC) by sequencing plasma eccDNA from 32 epidermal growth factor receptor (EGFR)-mutated NSCLC patients before and during treatment with osimertinib, as well as plasma eccDNA from five healthy individuals. Plasma eccDNA was identified in all samples but with significantly higher levels in cancer patients than healthy controls. EGFR-overlapping eccDNA, eccDNA that contains part of or the whole EGFR gene, was detected in the majority of samples both at baseline and during treatment. High levels of EGFR-overlapping eccDNA during osimertinib treatment were associated with significantly shorter progression-free survival and overall survival. Plasma eccDNA represents a newly identified type of biomarker for monitoring treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07675ef736e153dd2d860214b3234fa0d7fbdddb" target='_blank'>
              Plasma extrachromosomal circular DNA as a biomarker in EGFR-targeted therapy of non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Stensgaard, Sarmad Mehmood, E. B. F. Ebert, P. Meldgaard, Anindya Dutta, B. Sørensen
          </td>
          <td>2025-10-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f59455ffad8232534ff09a38ff5659981429a5" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes.
              </a>
            </td>
          <td>
            L. G. de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2025-09-24</td>
          <td>Nature</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Approximately half of high-grade serous ovarian carcinomas (HGSOCs) demonstrate homologous recombination deficiency (HRD) with characteristic genomic rearrangements. However, the impact of HRD on centromeres and transposable elements remains largely unexplored in HGSOC since conventional short-read sequencing is unable to interrogate these repetitive regions. We employed Oxford Nanopore long-read sequencing (LRS) to investigate genomic and epigenetic alterations in these regions. Pre-treatment archival cryopreserved tumor and matched blood samples were obtained for six patients with HGSOC. High-molecular-weight DNA was sequenced using Oxford Nanopore R10.4 flow cells and aligned to both the GRCh38 and the telomere-to-telomere T2T-CHM13 reference genome. Pathogenic gene mutations, allele-specific copy number variations, structural variants, and CpG methylation were analyzed. All six tumors had pathogenic TP53 mutations. Two carried germline BRCA1 mutations, while three showed CCNE1 amplifications. HRD scores and mutational signatures associated with HRD were elevated in the BRCA1-mutated tumors. Centromeric regions were significantly hypomethylated in tumors and their methylation profiles distinctly separated HRD tumors from non-HRD tumors. LINE1 and ERV transposable elements showed marked hypomethylation in tumors without germline BRCA1 mutations. Chromosome arm-specific telomere lengths were significantly shortened in tumors. Allele-specific hypermethylation in the TERT hypermethylated oncological region was detected in three tumors. LRS uncovered HRD-related genomic and epigenomic alterations in previously inaccessible repetitive regions of HGSOC, including centromeric and transposable element hypomethylation. These findings highlight the potential of such abnormalities as novel biomarkers for HGSOC and warrant further application of the methods to larger cohorts in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a227daba2577321ec904774ceb23e3c2f8c0883" target='_blank'>
              Long read sequencing reveals novel genomic and epigenomic alterations in repetitive regions of high grade serous ovarian cancer
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Y Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, Barrett Lawson, R. Hillman
          </td>
          <td>2025-10-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Fleur Chapus, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Extrachromosomal circular DNA (EccDNA) is widespread in various heterogeneous tumors and closely associated with tumor resistance and progression. Methods: Circle-seq and mRNA-seq were done on samples from three multiple myeloma (MM) patients, one when they had a complete response and one when they relapsed. Results: A large number of EccDNA molecules were detected with high heterogeneity among the six samples. Circle-seq combined with mRNA-seq analyses revealed that there is no linear relationship between mRNA expression and EccDNA quantity. Chromosome 19 presented the highest density of differentially expressed EccDNA genes, followed by chromosome 17. Only the T3 sample showed del(17p) by fluorescence in situ hybridization at the time of relapse. Approximately 92% of all upregulated EccDNA genes from chromosome 17 (137) were present in the T3 sample. By integrating Circle-seq and mRNA-seq data, we obtained several potentially functional candidate protein-coding genes and miRNAs from chromosome 17. We further assessed the prognostic value of the three protein-coding genes in the MMRF-COMPASS clinical trial and found that all three genes were poor prognostic indicators for MM. Furthermore, WB, CCK8 and Annexin V-FITC/PI assays revealed that the overexpression of 2',3'-cyclic nucleotide 3' phosphodiesterase (CNP) downregulated the apoptotic pathway and increased bortezomib tolerance in MM cells. Conclusion: We performed a Circle-seq analysis of MM and investigated the heterogeneity of EccDNA. Analysis of data from clinical studies and basic experiments revealed that the gene carried on EccDNA most likely contributed to the increased tolerance of bortezomib in MM with del(17p).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0af596876441646ebdb29f843eea2b6d455fc1" target='_blank'>
              Extrachromosomal circular DNA of multiple myeloma
              </a>
            </td>
          <td>
            Fangfang Li, Xinyi Long, Sishi Tang, Jing Liu, Yunfeng Fu
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="OncoDB was initially developed to advance cancer research by integrating RNA expression profiles, DNA methylation patterns, clinical annotations, and oncoviral signatures derived from the Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression datasets. We now present OncoDB 2.0, an extensively expanded platform that offers a more comprehensive and integrated view of cancer omics. The updated version includes an atlas of somatic mutations discovered from combined DNA and RNA sequencing, enabling in-depth investigation of mutation patterns across tumor types and their association with clinical features. Furthermore, we have integrated proteomic data from the Clinical Proteomic Tumor Analysis Consortium (CPTAC) and chromatin accessibility data from TCGA, offering new dimensions for oncogene regulation studies. OncoDB 2.0 also introduces advanced multi-omics analysis modules that facilitate the combined exploration of RNA expression, DNA methylation, and somatic mutations, allowing researchers to examine complex cross-omic relationships with greater depth and flexibility. Together, these enhancements make OncoDB 2.0 a robust and invaluable tool for the cancer research community. OncoDB 2.0 is freely available at https://oncodb.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2597afaecc32114e7746cc65fb5c9507bfe36bf0" target='_blank'>
              OncoDB 2.0: a comprehensive platform for integrated pan-cancer omics analysis.
              </a>
            </td>
          <td>
            Minsu Cho, Gongyu Tang, Charles S Rogers, Makayla Dove, Xinyi Liu, Yuanxiang Li, Xiaowei Wang
          </td>
          <td>2025-09-23</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caacf7712f535881a1313780e0d16e1ccea5aabe" target='_blank'>
              A chromatin fiber model explains cell-free DNA fragmentation signatures of active regulatory elements
              </a>
            </td>
          <td>
            Alexis Yang, G. Lee, Surya B. Chhetri, Razane El Hajj Chehade, Gunsagar S. Gulati, Medha Pandey, Doris Fu, Yoo-Na Kim, Shahabeddin Sotudian, H. Savignano, Sadia D. Shahbazi, Chelsea Philpot, Taylor A. Hoggood, D. Gulhan, M. Kellis, Jacob E Berchuck, S. Baca
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc4a7f8e64bdd86a0871fb665e1084f20e21063" target='_blank'>
              A mobile circular DNA element drives memory-related processes in mice
              </a>
            </td>
          <td>
            Haobin Ren, Hao Gong, W. Liau, Qiongyi Zhao, Alexander P. Walsh, Mason R. B. Musgrove, J. Davies, Esmi L. Zajackowski, Sachithrani U. Madugalle, P. Marshall, Tian Lan, Jichun Shi, Jiazhi Jiang, Wei Wei, Xiang Li, T. Bredy
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c45d07da847d3ed2d6622371a3aa002460dcb6" target='_blank'>
              Spatial genome reorganization is associated with chromosomal breakage regions and karyotype changes induced by programmed DNA elimination
              </a>
            </td>
          <td>
            James R. Simmons, Tianchun Xue, R. P. McCord, Jianbin Wang
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b8c268f4003d4cab2a7a6a760ce10e09bcebe0" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b8e7f8962ad8fcf01d30be8350fefcd7ba1a81" target='_blank'>
              CRISPR live-cell imaging reveals chromatin dynamics and enhancer interactions at multiple non-repetitive loci.
              </a>
            </td>
          <td>
            Meishuo Liu, Keyun Huang, Jie Zhang, Qingyang Li, Xinming Wang, Buming Gu, Hao Tang, Zhenhai Du, Liangjun Hu, Shutao Qi, Yu Ma, Hongtao Yu, Wei Xie, Xianyang Fang, Haifeng Wang
          </td>
          <td>2025-11-06</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd28764a1bb58dc6ea970c4ddb741f750a9e8bc" target='_blank'>
              Genome-wide DNA bridging by H-NS reshapes the stationary phase nucleoid and transcriptional landscape
              </a>
            </td>
          <td>
            Lindsey E. Way, Xiaofeng Dai, Emma E Wiesler, Lauren A McCarthy, Zhongqing Ren, Gail G Hardy, D. E. Fuller, Labeeb Hossain, Anne S. Meyer, E. Abbondanzieri, Julie S. Biteen, Xindan Wang
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany B. Johnson, Christopher Markovic, C. Fronick, Robert S. Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Genome instability, a hallmark of diffuse glioma, can arise from unprotected chromosome ends due to telomere dysfunction (TD). We successfully modeled TD in normal human astrocyte (NHA) cells by dysregulating cell cycle checkpoints using retroviruses containing HPV-18 E6/E7 and expanding the resulting cells over long periods of time. TD was confirmed by the detection of progressive telomere shortening and anaphase bridges with increasing population doublings (PDLs). We hypothesized that NHA-E6E7 cells undergoing TD would progressively accumulate structural variation (SV) and aneuploidy, which was confirmed using copy number and SV analysis from long and short-read sequencing data of increasing PDLs. Notably, we detected telomeric SV junctions and found that telomere-associated SVs were more prevalent in NHA-E6E7 cells. Analysis of telomeric long reads demonstrated that reads with internal telomeric sequences were significantly enriched at later PDLs, reflecting recombination events involving telomeres. Intriguingly, we identified an SV breakpoint on chromosome 3 at an interstitial telomeric sequence (ITS). Telomeric chromatin interaction profiling using Telomere-C showed a long-range chromatin interaction involving this ITS and the telomere in both normal and cancer cells, suggesting that this site could be a site of telomeric recombination. While genome instability fuels oncogenic alterations, large-scale rearrangements can be detrimental. We propose that the exposed ITS might serve as a neo-telomere, protecting chromosome segments from further DNA damage and conferring pre-cancerous cells with survival advantages. Our findings confirmed that TD contributes to genome instability in glioma and suggest that cancer cell progression could be influenced by underlying telomere-related mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171df1c4c8ebda2935993a63f5460965f3517f82" target='_blank'>
              EPCO-15. CHROMOSOME BREAKS AT INTERSTITIAL TELOMERIC SEQUENCES ENABLE PRE-CANCEROUS ASTROCYTES TO SURVIVE GENOME INSTABILITY CAUSED BY TELOMERE DYSFUNCTION
              </a>
            </td>
          <td>
            Yue Hao, T. R. Ranallo-Benavidez, Noelle H. Fukushima, Ogechukwu Mbegbu, Maria Kyriakidou, Yi-An Chen, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Uterine leiomyomas (ULs) and breast fibroadenomas (FAs) are the most common benign tumors in women, both arising in hormone-responsive mesenchymal tissues and often co-occurring during reproductive years. Despite their shared hormonal sensitivity and frequent MED12 mutations, their downstream molecular evolution remains poorly characterized. This study aimed to investigate whether ULs and FAs, though initiated by similar genetic alterations, diverge in their oncogenic trajectories, thereby addressing the molecular basis for their distinct clinical behaviors. Methods: We performed whole-exome sequencing (WES) on 15 uterine leiomyomas and 7 publicly available fibroadenomas with matched normal controls to compare somatic mutations, copy number alterations (CNAs), and mutational signatures. All UL samples were derived from Korean patients who underwent surgical treatment at a tertiary hospital between 2017 and 2019. Somatic variants were analyzed using MuTect2 and Strelka2. FACETS was used to estimate copy number changes in individual samples, and GISTIC2 to identify recurrent and statistically significant copy number alterations across patient cohorts. Mutational processes were inferred using SigProfiler. Microsatellite instability status was determined with MSIsensor2. The study was approved by the Institutional Review Board (UC17SNSI0092). Results: Comparative whole-exome sequencing of 15 ULs and 7 FAs from matched tumor-normal samples in an East Asian cohort confirmed both tumor types harbor identical MED12 p.G44D mutations, establishing shared molecular initiation. However, post-initiation evolution diverged dramatically: ULs exhibited chromosomal instability with 15 copy number amplifications, 6 of which affected oncogenes, but relatively modest point mutations. In contrast, FAs remained chromosomally stable but hypermutated, harboring 1.47× more variants than ULs despite lower tumor purity. Notably, one histologically benign FA harbored multiple loss-of-function mutations plus an EGFR gain-of-function mutation typically associated with malignant breast cancer, challenging traditional benign-malignant classifications. Conclusions: Despite sharing a common initiating mutation in MED12, ULs and FAs evolve through fundamentally distinct genomic pathways. UL evolves through chromosomal instability, whereas FA evolves through a mutator phenotype, with important implications for understanding tumor biology and molecular-based risk stratification. These findings support a paradigm of tissue-specific oncogenic evolution and underscore the potential clinical utility of genomic profiling in distinguishing benign tumors with atypical molecular features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9889bd66df0b0830e6828b3b935a1992068cb7" target='_blank'>
              Tissue-Specific Genomic Evolution Despite Shared MED12 Mutations in Benign Tumors
              </a>
            </td>
          <td>
            Jeong Namkung, Sang Ho Park, Ayoung Hwang, Hae Seo, Jooyoung Park, Moonyoung Lee, Hyunkyung Kim, Jungmin Choi, Jaeyen Song
          </td>
          <td>2025-10-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, C. Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7cf3abcb0807fc86763b9a27cf76f2de5e62eb" target='_blank'>
              Widespread DNA off-targeting confounds studies of RNA chromatin occupancy
              </a>
            </td>
          <td>
            Micah Jonathan Goldrich, L. Delhaye, Sarah-Lee Bekaert, Bieke Decaesteker, F. van Nieuwerburgh, Frank Speleman, S. Eyckerman, Pieter Mestdagh, I. Ulitsky
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The strength of the repressive histone H3K27me3 signal varies across silencers. Focusing on regions with unusually strong signals—super-silencers—we show that B-cell super-silencers are initially linked to gene upregulation in development, with target genes highly expressed in stem cells. About 13% of B-cell super-silencers convert to super-enhancers in B-cell lymphoma; 22% of these recur in over half of patients. Genes like BCL6 and BACH2 tied to these conversions are downregulated faster by JQ1, a super-enhancer-disrupting anti-cancer agent. Super-silencers are enriched for B-cell cancer-associated variants—both somatic and germline—and translocation breakpoints, exceeding levels in other regulatory elements like CTCF binding sites. Over 80% of B-cell lymphoma t(3;14)(q27;q32) translocations fuse BCL6 super-silencers with enhancer-rich regions. Super-silencer repression depends on CpG content: CpG-rich elements block promoter–enhancer contacts; CpG-poor — inhibit looping. These findings highlight super-silencers’ key role in B-cell regulation and suggest their alteration may be a primary factor of B-cell carcinogenesis. In B-cells, super-silencers originally help activate important genes during early development. Later, they switch roles and become powerful silencers in mature B cells. Super-silencers help control gene activity in healthy B-cells, but when they malfunction or mutate, they may drive the development of B-cell cancers. In healthy, mature B cells, super-silencers keep developmental genes switched off to maintain cell identity. When disrupted by mutation or malfunction, these elements can reverse their role and contribute to the development of B-cell cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab550e747142b856a2c8f404a6e1c22dc14158c" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff952ed2ebbe43375a46000c748eba5e1f9ac44a" target='_blank'>
              The scaffold protein PRR14L links the PP2A-TACC3 axis to mitotic fidelity and sensitivity to MPS1 inhibition
              </a>
            </td>
          <td>
            Albert Z. Liu, A. Narkar, Keming Li, Thierry Bertomeu, Blake A. Johnson, J. Coulombe-Huntington, Yi Dong, Jin Zhu, Mike Tyers, Rong Li
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The distinctive three-dimensional (3D) chromatin architecture adopted by chromosomes during meiosis is essential for fertility, yet the functional implications of the fine-scale spatial organization remain poorly understood. Here, we investigate the impact of RAD21L deletion, a meiosis-specific cohesin subunit, on the 3D genome architecture and gene expression in the male germ line. Using fluorescence-activated cell sorting, high-throughput chromosome conformation capture, and single-cell RNA sequencing, we demonstrate that RAD21L deficiency impairs meiotic chromatin organization and inter/intrachromosomal interactions. This conspicuous 3D genome reorganization also disrupts bouquet formation, resulting in increased telomeric interactions between heterologous chromosomes in primary spermatocytes. Genome reorganization was accompanied by detectable transcriptional dysregulation in spermatogonia and primary spermatocytes, mainly affecting sex chromosomes. Collectively, our findings establish the cohesin RAD21L as a germline-specific critical regulator of genome-wide 3D genome reorganization during spermatogenesis and a modulator of the male germline transcriptional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2272faa3ad286c58e292f1de1607d234286a48c" target='_blank'>
              Meiotic cohesin RAD21L shapes 3D genome structure and transcription in the male germline
              </a>
            </td>
          <td>
            Laia Marín-Gual, Covadonga Vara, Raquel Sainz-Urruela, Yasmina Cuartero, Lucía Álvarez-González, Natalia Felipe-Medina, Francisca Garcia, E. Llano, M. Martí-Renom, A. Pendás, A. Ruiz-Herrera
          </td>
          <td>2025-10-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e04c6f5ef9fa6ac101d2f55bfc644ec979072" target='_blank'>
              Pervasive suppressors halt the spread of selfish Segregation Distorter in a natural population
              </a>
            </td>
          <td>
            Ching-Ho Chang, Tyler Handler, Nicholas J. Fuda, Danielle Pascua, Taylar Mouton, A. Larracuente
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Replication stress (RS), if not effectively and timely addressed, could result in DNA damage in mitosis. However, the relationship between RS and other mitotic events, such as nuclear envelope (NE) breakdown and reassembly, remains poorly understood. Here we report that RS can lead to NE defect. Importantly, rather than de novo NE rupture, the defect per se is a result of nuclear envelope reassembly defect (NERD) during mitosis. Interestingly, NERD is associated with mitotic DNA damage, and repair of the damage by DNA polymerase theta (Polθ)-mediated end joining (TMEJ) ameliorates NERD. Genomic mapping of lamina associated domains (LADs) by cleavage under targets and tagmentation (CUT&Tag) identifies a population of replication stress-sensitive LADs (RESSLADs). Strikingly, a substantial portion of RESSLADs reside in the common fragile sites (CFSs). The loss of RESSLADs-NE interaction under RS might be attributed to the sustained phosphorylation of Lamin A/C at the sites of NERD. In addition, prominent NE defect is observed under multiple conditions of synthetic lethality. Altogether, these findings establish a link between genome instability and nuclear vulnerability under replication stress. How replication stress affects nuclear integrity is not fully understood. Here, the authors report that replication stress induces DNA damage and nuclear envelope reassembly defect during mitosis, which is suppressed by DNA polymerase theta mediated end joining.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/939377138dd121885a6964852dac3b04dd8c9c53" target='_blank'>
              Mitotic DNA repair by TMEJ suppresses replication stress-induced nuclear envelope reassembly defect
              </a>
            </td>
          <td>
            Guojun Ye, Yide He, Yihui Zhang, Dongchen Li, Fuhai Liu, Yi Li, Qinglian Ge, Qiong Guo, Shuya Han, Chunyu Song, Weiping Chang, Haoyue Zhang, Qin Peng, Kun Sun, Wei-Ke Ji, Lin Deng
          </td>
          <td>2025-10-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0dc83d3c9d750e9f6591dee2fa83d60f9676308" target='_blank'>
              Dici: a novel DNA transposon reshaping the genome of the opportunistic yeast Diutina catenulata
              </a>
            </td>
          <td>
            F. Bigey, M. Wessner, Martine Pradal, Corinne Cruaud, J. Aury, Cécile Neuvéglise
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Human tissues require a mechanism to generate durable, yet modifiable, transcriptional memories to sustain cell function across a lifetime. Previously, it was demonstrated that nanoscale packing domains couple heterochromatin (cores) and euchromatin (outer zone) into unified reaction volumes that can generate transcriptional memory. In prior work, this framework demonstrates that RNA synthesis occurred within the ideal zone (intermediate density) portions of the domain. Naturally, this creates a question of where genes are positioned in relation to the packing domain architecture and which genetic material fills the domain core to sustain transcription. Here, it is proposed that this can be solved by the encoded positioning of introns, intergenic segments, and exons as a projection of the functional packing layers of domains. This suggests that introns and intergenic segments are coupled to adjacent exons to generate coherent packing domain volumes. How this organization will reconcile contradictions in epigenetic patterns, non-randomness in oncogenic mutations, and produce durable transcriptional memory is illustrated. The study concludes by showing that this genome geometry may have coincided with the rapid evolution of body-plan complexity, suggesting that chromatin geometry could be fundamental to metazoan evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28ea2cbe5621f89d0e90527ffbe4497e1a12d59" target='_blank'>
              Geometrically Encoded Positioning of Introns, Intergenic Segments, and Exons in the Human Genome.
              </a>
            </td>
          <td>
            L. Almassalha, Kyle L MacQuarrie, Marcelo Carignano, Cody L Dunton, Ruyi Gong, Joe Ibarra, Lucas M Carter, Wing Shun Li, Rikkert J. Nap, P. Dulai, I. Szleifer, Vadim Backman
          </td>
          <td>2025-10-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c9258c210cadae25150c22831d0f054794304f" target='_blank'>
              Micro-C in Solanum Uncovers Conserved Genome Folding and Epigenetically Defined Loops with Bifunctional Enhancer-Silencer Activity
              </a>
            </td>
          <td>
            Shdema Filler-Hayut, A. Hansen
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The vast majority of recurrent somatic mutations arising in tumors affect protein-coding genes in the nuclear genome. Here, through population-scale analysis of 14,106 whole tumor genomes, we report the discovery of highly recurrent mutations affecting both the small (12S, MT-RNR1) and large (16S, MT-RNR2) mitochondrial RNA subunits of the mitochondrial ribosome encoded within mitochondrial DNA (mtDNA). Compared to non-hotspot positions, mitochondrial rRNA hotspots preferentially affected positions under purifying selection in the germline and demonstrated structural clustering within the mitoribosome at mRNA and tRNA interacting positions. Using precision mtDNA base editing, we engineered models of an exemplar MT-RNR1 hotspot mutation, m.1227G>A. Multimodal profiling revealed a heteroplasmy-dependent decrease in mitochondrial function and loss of respiratory chain subunits from a heteroplasmic dosage of ~10%. Mutation of conserved positions in ribosomal RNA that disrupt mitochondrial translation therefore represent a class of functionally dominant, pathogenic mtDNA mutations that are under positive selection in cancer genomes. Analysis of 14,106 tumor genomes highlights recurrent mutations in mitochondrial ribosomal RNA encoded within the mitochondrial genome. Mutations occur at hotspot positions and are under strong purifying selection in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eda054c049bfbd95fe327306b7323841c22683" target='_blank'>
              Functionally dominant hotspot mutations of mitochondrial ribosomal RNA genes in cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Jacqueline Tait-Mulder, Minsoo Kim, Cerise Tang, Tricia Park, Flora McNulty, S. Lilla, S. Zanivan, Alejandro Huerta-Uribe, Benan N. Nalbant, Mark Zucker, D. Sumpton, Geoffray Monteuuis, Christopher B Jackson, Wei Wei, P. Chinnery, R. Chaligné, Caleb A. Lareau, Ed Reznik, P. Gammage
          </td>
          <td>2025-11-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f77d30c5b35d3bee66492e606ea00f82e5f1ceb" target='_blank'>
              Argonautes and small RNAs associated with nematode programmed DNA elimination
              </a>
            </td>
          <td>
            M. Zagoskin, Yuanyuan Kang, Richard E Davis, Jianbin Wang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a primary driver of cervical cancer. Integration of HPV into the human genome causes persistent expression of viral oncogenes E6 and E7, which promote carcinogenesis and disrupt host genomic function. However, the impact of integration on host gene expression remains incompletely understood. We used multimodal RNA sequencing, combining total RNA-seq and Cap Analysis of Gene Expression (CAGE), to clarify virus–host interactions after HPV integration. HPV-derived transcripts were detected in 17 of 20 clinical samples. In most specimens, transcriptional start sites (TSSs) showed predominant early promoter usage, and transcript patterns differed with detectable E4 RNA region. Notably, the high RNA expressions of E4 region and viral-human chimeric RNAs were mutually exclusive. Chimeric RNAs were identified in 13 of 17 samples, revealing 16 viral integration sites (ISs). CAGE data revealed two patterns of TSS upregulation centered on the ISs: a two-sided pattern (43.8%) and a one-sided pattern (31.3%). Total RNA-seq showed upregulation of 12 putative cancer-related genes near ISs, including MAGI1-AS1, HAS3, CASC8, BIRC2, and MMP12. These findings indicate that HPV integration drives transcriptional activation near ISs, enhancing expression of adjacent oncogenes. Our study deepens understanding of HPV-induced carcinogenesis and informs precision medicine strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48263c0732371718c3701a40772227c4abc1885c" target='_blank'>
              Elucidating Alterations in Viral and Human Gene Expression Due to Human Papillomavirus Integration by Using Multimodal RNA Sequencing
              </a>
            </td>
          <td>
            Kana Tamai, S. Kinjo, Ayumi Taguchi, K. Nagasaka, Daisuke Yoshimoto, Anh Quynh Duong, Yoko Yamamoto, Hitoshi Iuchi, M. Mori, K. Sone, Michiaki Hamada, Kei Kawana, Kazuho Ikeo, Yasushi Hirota, Yutaka Osuga
          </td>
          <td>2025-10-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation - DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1655457ebd077b5d78f8c0d2ef963c19750b7b0" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris.
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-11-12</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms. Cancer cell plasticity is often linked to treatment failure. Here, the authors discuss the role of epigenome in DNA repair and, how DNA repair-induced chromatin changes influence cancer cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab22d511d2948a185e5a280612430d61e3b4b0a" target='_blank'>
              Linking Germline Telomere Removal to Global Programmed DNA Elimination in Tetrahymena Genome Differentiation
              </a>
            </td>
          <td>
            Kohei Nagao, Kazufumi Mochizuki
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d97982af89e80beb34eb068ca73e5435c3e26dc" target='_blank'>
              High-throughput functional characterization of enhancers in totipotent-like cells
              </a>
            </td>
          <td>
            Lingyue Yang, Tianran Peng, Yating Zhu, Xuzhao Zhai, Boyan Huang, Ling Li, Tao Zhang, Jiekai Chen, Dan Liang, Jiangping He, Man Zhang
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9318834789bef983d4d5a7b760560db627aaee1" target='_blank'>
              Programmed DNA elimination was present in the last common ancestor of Caenorhabditis nematodes
              </a>
            </td>
          <td>
            Lewis Stevens, Simo Sun, N. Haruta, Leyun Xiao, Naoki Uwatoko, M. Kieninger, Kazuki Sato, Akemi Yoshida, Dominic E. Absolon, Joanna Collins, Asako Sugimoto, Taisei Kikuchi, Mark Blaxter
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, N. Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-throughput clonal tracing of primary human samples relies on naturally occurring barcodes, such as somatic mitochondrial DNA (mtDNA) mutations detected via single-cell ATAC-seq (mtscATAC-seq). Fresh-frozen clinical specimens preserve tissue architecture but compromise cell integrity, thereby precluding their use in multi- omic approaches such as mitochondrial genotyping at single-cell resolution. Here, we introduce Cryo-mtscATAC-seq, a broadly applicable method for diverse pathophysiological contexts to isolate nuclei with their associated mitochondria (“CryoCells”) from frozen samples for high-throughput clonal analysis. We applied Cryo-mtscATAC-seq to the neurodegenerated human brain, glioblastoma (GBM), pediatric neuroblastoma, and human aorta, and implemented mitobender, a computational tool to reduce ambient mtDNA in single-cell assays. Our approach revealed regional clonal gliogenesis and microglial expansions in amyotrophic lateral sclerosis (ALS), persistence of oligodendrocyte progenitor cell (OPC)-like clones in GBM recurrence, mtDNA depth heterogeneity after neuroblastoma chemotherapy, and oligoclonal proliferation of smooth muscle cells in human aorta. In conclusion, Cryo-mtscATAC-seq broadly extends mtDNA genotyping to archival frozen specimens across tissue types, opening new avenues for investigation of cell state- informed clonality in human health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60dee046cd06ab111daf9f8ce07ff1e2d339ed7" target='_blank'>
              Cryo-mtscATAC-seq for single-cell mitochondrial DNA genotyping and clonal tracing in archived human tissues
              </a>
            </td>
          <td>
            Maren Salla, B. Obermayer, Marie Cotta, Ekaterina Friebel, Juliana Campo-Garcia, Georgia Charalambous, Roemel Jeusep Bueno, Dustin Lieu, Patryk Dabek, Ashley Helmuth, G. Tellides, Roland Assi, Katrin Bankov, Marco Lodrini, H. Deubzer, D. Beule, Hattie Chung, Helena Radbruch, David Capper, Frank L. Heppner, Sarah C. Starossom, Caleb A. Lareau, Ilon Liu, Leif S. Ludwig
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The human genome attains an amazing spatial organization in the packaging of 2 m of DNA into a 10-μm nucleus. Such structural organization is achieved by the folding of chromatin and the regulation exerted by architectural proteins such as insulators. Chromatin insulators are boundary elements of the genome that, through enhancing blocking activities, demarcation of chromatin domains, and chromatin looping, regulate transcription. The review focuses on the identification and characterization of insulators in various species, discussing mainly the functions of the CCCTC-binding factor (CTCF) in mammals and functionally equivalent insulator proteins in Drosophila melanogaster. We review here the mechanisms of enhancer blocking, barrier activity, and loop extrusion, emphasizing their effects on topologically associating domains and chromatin architecture. Furthermore, we discuss new concepts that have come into prominence: tethering elements and redundancy among the insulator proteins, which contribute to chromatin organization. Advances in methodology, including chromosome conformation capture and high-resolution imaging techniques, have transformed our view of the dynamic interplay between the architecture of chromatin and transcription regulation. This review discusses the importance of insulators for genome organization and describes future directions in investigating their roles in both gene regulation and three-dimensional genomic architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eda983a70eb60f914e84aedb4a80af4895d808" target='_blank'>
              Chromatin insulators in gene regulation and 3D genome organization.
              </a>
            </td>
          <td>
            Hina Sultana, Rohit Kunar, A. G. Matera
          </td>
          <td>2025-10-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Whole genome doubling (WGD) triggers profound genomic and epigenetic reorganization, yet the functional dynamics of DNA methylation during this process remain incompletely resolved. Here, we integrate whole genome bisulfite sequencing (WGBS) and three-dimensional chromatin interaction data to display methylation landscapes in autotetraploid Arabidopsis thaliana. Our analysis reveals evolutionarily conserved spatial patterning of DNA methylation after WGD, with centromeric enrichment and telomeric depletion. Chromosome-level profiling identifies Chromosome 2 as the most highly methylated across CG, CHG, and CHH contexts, while Chromosome 1 shows the lowest methylation. Subcontext methylation analysis uncovers increases in methylation levels in autotetraploid Arabidopsis thaliana, most pronounced in the CHH context, yet global distribution patterns remain stable. Comparative methylation profiling around genes and transposable elements (TEs) reveals elevated CHH methylation in autotetraploid gene bodies and flanking regions, whereas TE bodies exhibit minimal changes despite minor flanking hypermethylation. Strikingly, 8% of chromatin compartments were restructured, and B-B interactions weakened in autotetraploid, while DNA methylation remained stable across shifting A/B compartments. Our findings suggest that DNA methylation serves as a resilient epigenetic modification during WGD, even if 3D chromatin architecture undergoes reorganization upon WGD in some degree.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c794ab17ab1c871d4bc77df6d82021f2b6c73a" target='_blank'>
              The Association Between DNA Methylation and Three-Dimensional Genome During Whole Genome Doubling in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Ranze Zhao, Zhongqiu Ni, Dingyu Zhang, Yuda Fang
          </td>
          <td>2025-09-24</td>
          <td>Plants</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Upon DNA breakage, a genomic locus undergoes alterations in three-dimensional chromatin architecture to facilitate signaling and repair. Although cells possess mechanisms to repair damaged DNA, it is unknown whether the surrounding chromatin is restored to its naïve state. We show that a single DNA double-strand break (DSB) within a topologically associated domain (TAD) harboring conformation-sensitive genes causes lasting chromatin alterations, which persist after completion of DNA repair and feature topological rearrangements and loss of local RNA species. These newly acquired features of postrepair chromatin are transmitted to daughter cells and manifest as heritable impairments of gene expression. These findings uncover a hitherto concealed dimension of DNA breakage, which we term postrepair chromatin fatigue and which confers heritable impairment of gene function beyond DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70850e6eacefc0378a65ec703a07a0084a7eaca" target='_blank'>
              Repair of DNA double-strand breaks leaves heritable impairment to genome function.
              </a>
            </td>
          <td>
            Susanne C. S. Bantele, Irene Mordini, A. Biran, Nicolás Alcaraz, Gijs Zonderland, Alice Wenger, Nils Krietenstein, Anja Groth, Jiri Lukas
          </td>
          <td>2025-11-06</td>
          <td>Science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromosome conformation within the nucleus is essential for genome function. These have primarily been studied at the scale of loops and compartments, or at lower spatial resolution using traditional in situ hybridization in chemically fixed samples. However, the mesoscale organization of single chromosomes in vivo, shaped by the interplay between chromatin architectural proteins and histone modifications, remains partially understood. In this study, we interrogated the mesoscale conformations of interphase chromosomes in live human osteoblasts and transformed osteosarcoma cells, focusing on chromosome 19. Chromosome conformations were quantified by the aspect ratio of the principal axes of gyration tensors. In osteoblasts, approximately 81% of chromosome 19 are observed to consist of regions characterized by highly extended organizations, with aspect ratios approximately four times greater than those of spheres. In contrast, in osteosarcoma cells, the chromosome displays an extensively collapsed conformation, with aspect ratios more closely approximately that of a sphere. In both cell types, the chromosome’s conformation is bimodal and the balance between these two modes differs very significantly between the two cell types. While the mesoscopic conformation is considerably stable, it is superimposed on dynamic, smaller scale regions. Additional results reveal that this significant conformational shift is independent of the cell cycle but co-regulated by CTCF, cohesion, and H3K27 modifications. Our findings provide new insights into the coordinated complex regulatory mechanisms governing mesoscale chromosome organization in normal and transformed osteogenic tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f8561f1fa7ab9bbf02e738cec3deefcabedf878" target='_blank'>
              Differential regulation of mesoscale chromosome conformations in osteoblasts and osteosarcoma
              </a>
            </td>
          <td>
            Madhoolika Bisht, Yu-Chieh Chung, Siou-Luan He, Sydney Willey, Benjamin D. Sunkel, Meng Wang, Benjamin Z. Stanton, Li-Chun Tu
          </td>
          <td>2025-09-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Selfish genetic elements (SGEs) are DNA sequences that enhance their own transmission, often at the expense of the host genome’s overall fitness. They include transposable elements, meiotic drivers, supernumerary B chromosomes, post-segregation killers, and sex-distorting heritable microbes or organelles. These elements generate genetic conflict among different parts of the genome—such as between nuclear, cytoplasmic, and mobile elements—due to their differing transmission strategies. Genomic research has revealed that microbial genomes are abundant in mobile genetic elements (MGEs) that spread through DNA transfer mechanisms and play major roles in genome organization, regulation, and evolution. SGEs contribute to genetic diversity, drive innovation, and influence key biological processes such as gene regulation, development, and speciation. The review highlights the diversity of SGE transmission mechanisms, the domestication of SGEs leading to novel genes and regulatory systems, and their dual role in promoting adaptability while imposing metabolic and genetic costs on the host. Moreover, understanding SGE-driven conflicts provides valuable insights for biotechnology, including the development of gene-editing tools, genetic engineering strategies, and potential therapeutic applications such as targeting cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6431d858e07f65156e8fbedcb579a4f343b92fd" target='_blank'>
              Selfish Genetic Elements and Genomic Conflicts: An Update on Microbial Elements, Their Genetics and Evolutionary Consequences
              </a>
            </td>
          <td>
            Priya M.D, Mahalakshmi B.R, Ranjini.P, Latha K, Shreya K.R, Mukunda Suryanarayana, Kiran Kumar H.B
          </td>
          <td>2025-11-06</td>
          <td>Asian Journal of Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517574b6bc8a73f48a3a9ee64c0be54874afb37" target='_blank'>
              TET CpG sequence context specific DNA demethylation shapes progression of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Y. Hoogstrate, S. Ghisai, L. van Hijfte, R. Head, I. D. de Heer, Marta Padovan, Maurice de Wit, Wies R. Vallentgoed, A. Dipasquale, M. Wijnenga, Bas Weenink, R. Luning, S. Maas, Adela Brzobohata, Michael Weller, Tobias Weiss, M. Mair, A. Berghoff, A. Wöhrer, A. Jeltsch, J. Koekkoek, H. M. Hazelbag, M. Kouwenhoven, Yongsoo Kim, Bart A. Westerman, B. Ylstra, A. Niers, K. Johnson, F. Varn, R. Verhaak, Mustafa Khasraw, M. J. van den Bent, Pieter Wesseling, P. French
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fd4d1f853d699db191c9c34adb97eb8ebac4f" target='_blank'>
              Protamine lacunae preserve the paternal chromatin landscape in sperm
              </a>
            </td>
          <td>
            Thomas W. Tullius, Rachel A Heuer, Stephanie Bohaczuk, Ben Mallory, Danilo Dubocanin, Jane E. Ranchalis, Ahmet Ayaz, Christopher E. Mason, Emre Seli, A. Phillippy, A. Stergachis, Bluma J. Lesch
          </td>
          <td>2025-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cf856a29d7dc82e41e89d6c3e46ad0a67ee3c3a" target='_blank'>
              Cell-stereotyped DNA repair outcomes are widespread during genome editing
              </a>
            </td>
          <td>
            Moritz F. Schlapansky, Markus S Schröder, Antonio J. Santinha, Tanja Rothgangl, Eleonora I Ioannidi, G. Cullot, Bohdan Lewków, G. C. Ortega, Ahmed Nouraiz, Dominic Mailänder, Louisa Selbert, Simon Lam, Aitana Egea, Jiyung J. Shin, Matteo Bordi, Mark DeWitt, Ana Gvozdenovic, Stephen P. Jackson, T. Schroeder, Helmuth Gehart, Gerald Schwank, Randall J. Platt, Jacob E. Corn
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Highlights What are the main findings? ALT functions as a context-dependent double-edged sword: it drives aggressiveness in rapidly proliferating cancers but imposes replicative and metabolic stress in slow-growing, immune-cold tumors such as glioblastoma and chondrosarcoma. Loss of ATRX/DAXX and ALT activation creates unique DNA repair vulnerabilities, including hypersensitivity to ATR and PARP inhibition, opening avenues for targeted therapy. What is the implication of the main finding? Refines prognostic interpretation of ALT positivity—transforming it from a uniformly negative marker into a tumor-type-specific indicator of clinical outcome. Positions ALT as a precision oncology target, where integrated biomarkers (C-circle as say, APB detection, ATRX/DAXX mutation profiling) enable patient stratification and therapeutic decision-making. Abstract Telomere maintenance enables unlimited cell proliferation by counteracting telomere erosion. While the majority of tumors activate telomerase, a significant subset—approximately 10–15%—utilizes alternative lengthening of telomeres (ALT), a recombination-based mechanism. ALT-positive cancers are classically associated with genomic instability, anaphase bridges, chromosomal rearrangements, and resistance to DNA-damaging therapies. This process is closely associated with genetic instability, which contributes to chromosomal rearrangements and tumor evolution. Consequently, ALT has traditionally been considered an adverse prognostic marker in aggressive malignancies such as osteosarcoma, pancreatic neuroendocrine tumors, and high-grade sarcomas. Paradoxically, recent evidence demonstrates that ALT positivity correlates with improved survival in glioblastoma (GBM) and chondrosarcoma, two tumor types that have historically been regarded as immune-cold and therapeutically intractable. This favorable outcome likely reflects a convergence of factors, including replication stress and DNA damage that impose a fitness cost in slow-growing or metabolically constrained tumors. Loss of ATRX/DAXX, while enabling ALT, further amplifies chromatin fragility, and ALT-mediated instability may paradoxically enhance immunogenicity within immune-quiescent microenvironments. Moreover, ALT-positive cells exhibit unique therapeutic vulnerabilities, particularly to ATR and PARP inhibitors. Together, these observations support a context-dependent model in which ALT functions as a double-edged sword, acting as a driver of malignant aggressiveness in rapidly proliferating cancers while serving as a relative liability in slower-growing, immune-cold tumors. Understanding this duality not only refines prognostic stratification but also opens opportunities for precision oncology. By integrating ALT-specific biomarkers into clinical workflows and exploiting ALT-related DNA repair dependencies, clinicians may transform a once uniformly negative prognostic factor into an actionable therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6ed85bfa1a7b22ce5afc4a049e72d82de6bf4" target='_blank'>
              Alternative Lengthening of Telomeres: A Prognostic Paradox in Cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2160f4dc4408443133ee4eb22a79c177cc5308" target='_blank'>
              Human cytomegalovirus regulates host DNA repair machinery for viral genome integrity
              </a>
            </td>
          <td>
            P. Longmire, S. Zeltzer, Kristen Zarrella, Olivia Daigle, Marek Svoboda, Justin M. Reitsma, S. Terhune, Carly Bobak, Giovanni Bosco, Felicia D. Goodrum
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf3ca6346229918f35167d16749694c1e3b771aa" target='_blank'>
              STAG2 cohesin cooperates with DREAM to maintain quiescence and suppress tumourigenesis in the urothelium
              </a>
            </td>
          <td>
            Maria Ramal, E. Lapi, M. Kalisz, Jaime Martínez de Villarreal, Meritxell Novillo-Font, M. Marqués, E. Zarzuela, Marta Isasa, Osvaldo Graña-Castro, Jianming Xu, Ana Cayuela, François Radvanyi, Mireia Novell, Jorge Martínez-Torrecuadrada, M. Barallobre, I. Bernard-Pierrot, María Marti-Marimon, Maria L Arbonés, M. Marti-Renom, Francisco X. Real
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d535d363c465b65e62fbbcdfa920c1c51d8d6da" target='_blank'>
              Segregating DNA lesions point to high selective advantage of tumor initiating cells
              </a>
            </td>
          <td>
            Vladimir Seplyarskiy, Maha Shady, Maria A Andrianova, Michael Spencer Chapman, E. V. Van Allen, S. Sunyaev
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="
 Meningiomas are among the most prevalent tumors of the central nervous system and are typically driven by a limited set of mutually exclusive genetic alterations. We report molecular characterization of seven meningiomas with fusions between LMNA and FOSB, in the absence of other canonical meningioma-associated mutations. LMNA, encoding nuclear envelope protein Lamin A/C, plays a central role in chromatin architecture and is widely expressed across differentiated tissues. FOSB, an AP-1 transcription factor, regulates genes involved in cellular growth and differentiation. Using integrated genomic analyses—including targeted DNA sequencing, whole-genome sequencing, and RNA-seq—we validated near identical fusions involving exons 1–2 of LMNA (chr1) and exons 2–4 of FOSB (chr19). This rearrangement retains key functional domains from both genes, notably the Ser22 phosphorylation site in Lamin A/C that associates with active enhancers, and the DNA-binding region of FOSB. Transcript levels of LMNA and FOSB remained largely unaltered in fusion-positive tumors, suggesting oncogenic activity may stem from a neomorphic fusion protein rather than dysregulated gene expression. Expression of LMNA-FOSB constructs in CH157 and IOMM-LEE meningioma cell lines led to significant alterations in chromatin structure and transcriptional profiles. Gene set enrichment analysis of upregulated genes (FC > 1, P < 0.05) shared between transfected CH157 and IOMM-LEE lines revealed enrichment of pathways related to laminin, fibronectin, and collagen binding—components commonly elevated in the meningioma extracellular matrix—as well as growth factor binding (P < 0.05). These findings were corroborated by Hi-C and ChIP-seq profiling of patient-derived frozen tumors, which demonstrated changes in chromatin conformation and transcriptional regulation. Our data establish LMNA-FOSB fusions as a novel and recurrent genetic alteration in meningiomas, illuminating a potential oncogenic mechanism involving chromatin remodeling and transcriptional reprogramming, and expanding the known functional scope of FOSB fusions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecf48230bbb5851d97d33985c7b6b81db14bbc7" target='_blank'>
              EPCO-42. RECURRENT LMNA-FOSB FUSIONS ALTER CHROMATIN TOPOLOGY AND GENE REGULATION IN MENINGIOMAS
              </a>
            </td>
          <td>
            Mateo Gomez Hjerthen, J. Wong, Juan Wang, Harrshavasan T. Congivaram, Priscilla K Brastianos, Akash Patel, C. Horbinski, James P Chandler, Zeynep E. Erson Omay, Jennifer Moliterno, Murat Günel, Stephen T. Magill, D. Raleigh, Feng Yue, Mark W. Youngblood
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91cdceed3d847b45d60afbbef0fb296224e3aa1c" target='_blank'>
              Defective proviruses cause T cell reprogramming through promoter exaptation in HIV-1 infection
              </a>
            </td>
          <td>
            M.V. Hamann, Lisa Brauckmann, Christoph Schwarz, M. Spohn, Ramon Stoeck, Sabrina M. Leddy, Julie Frouard, Maisha Adiba, F. Thiele, Sanamjeet Virdi, A. Grundhoff, Cédric Feschotte, N. Roan, E. Tolosa, Ulrike C Lange
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2871bba13184fce8594e19be2ca206ed09a430e3" target='_blank'>
              Independent evolution of holocentric chromosomes in an early branching apicomplexan parasite
              </a>
            </td>
          <td>
            Abigail Kimball, Wanyi Huang, Rui Xu, Melanie Key, Lisa J. Funkhouser-Jones, L. D. Sibley, Dr. L. David Sibley
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 The WHO 2021 classification of central nervous system tumors incorporated molecular markers to better define diffuse glioma subgroups. We analyzed array-based copy number, targeted next-generation sequencing, germline genotyping, and Illumina EPIC array methylation data from 404 primary IDH-mutant astrocytomas to further understand the relationship between copy number and outcomes. Loss of chromosome arm 13q was associated with poor overall survival after adjusting for age, sex, tumor grade, and treatment status (hazard ratio [HR]=2.04 (95% confidence interval: 1.25-3.33), p=0.004). Chromosome 13q loss was also significantly associated with the total number of copy number alterations (p<0.001). Loss of arm 11p was associated with poor survival in the adjusted analysis (HR=2.29 (1.38-3.79), p=0.001). Unsupervised clustering of the raw copy number data identified 12 clusters, where each cluster was defined by distinct copy number alteration(s). Clusters that contained 13q loss exhibited worse overall survival compared to other clusters. The clusters were significantly associated with age at diagnosis (p=0.015), tumor grade (p<0.001) and copy number complexity (p<0.001). The clusters with a high proportion of grade 4 tumors and with the highest complexity had the poorest overall survival. These findings highlight the prognostic relevance of chromosome 13q and 11p losses in IDH-mutant astrocytomas and support the integration of additional copy number alterations into molecular classification frameworks of IDH-mutant astrocytomas to improve risk stratification and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253265c2822339aab418366b6f6dea35282c7e8a" target='_blank'>
              PATH-36. DNA copy number clustering reveals alterations that influence survival in primary IDH-mutant astrocytoma patients
              </a>
            </td>
          <td>
            K. Drucker, Arijit Panda, Paul A Decker, M. Kosel, Shameel Shafqat, Muhammad Asad Maqbool, A. Landry, Y. Ellenbogen, Vikas Patil, Caterina Giannini, R. Vaubel, Aditya Raghunathan, I. Parney, T. Burns, Cristiane M Ida, S. Kizilbash, G. Zadeh, Daniel H Lachance, A. Abyzov, J. Eckel-Passow, R. Jenkins
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Telomere length variation is a hallmark of genomic instability, however, its role in shaping the structural variant (SV) landscape of diffuse astrocytoma remains poorly understood. In this study, we employed Oxford Nanopore long-read sequencing to investigate the association between SVs and chromosome arm-specific telomere length in a clinical patient cohort consisting of n = 20 individuals diagnosed with an IDH mutant astrocytoma. Oxford Nanopore long-read sequencing was performed to a depth of ~20-40× coverage. SVs were identified using Sniffles, copy number alterations were assessed using DNAcopy and chromosome arm-specific telomere length was estimated with Telogator. Statistical analyses were conducted to assess associations between telomere dynamics, structural variation, and copy number changes. Astrocytoma genomes exhibited a broad spectrum of structural variants, including deletions and insertions, with significant enrichment in the short arms of acrocentric chromosomes. A subset of samples demonstrated markedly shorter telomeres relative to the overall cohort, accompanied by an increased burden of structural variants, particularly duplications and inversions, suggesting enhanced genomic instability. Notably, a prominent accumulation of deletion and insertion SVs was observed near chromosome ends and centromeres, highlighting the (sub)telomeric and peri(centromeric) regions as SV hotspots. Statistical modeling revealed a significant association between telomere length and arm-specific variation, including telomeric deletions, inversions, and copy number losses. In summary, our findings reveal a previously underappreciated link between SVs and telomere length in astrocytomas. Genomic rearrangements caused by telomere shortening may represent critical drivers of genome instability and could serve as novel biomarkers for disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caee3d4164cb55a207716341d4255e0e2db47b7" target='_blank'>
              EPCO-26. HIGH-RESOLUTION PROFILING OF TELOMERE DYNAMICS AND GENOMIC VARIANTS IN ASTROCYTOMAS USING LONG-READ SEQUENCING
              </a>
            </td>
          <td>
            Maryam Jehangir, K. Drucker, T. R. Ranallo-Benavidez, T. Kollmeyer, Maria Kyriakidou, Daniel H Lachance, J. Eckel-Passow, R. Jenkins, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S. Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476d94c7dbe88869f2a1eef006cd2afd67f0a95c" target='_blank'>
              Spatial Organization of Chromatin of ZEB1 Gene Promoter Region in Pancreatic Ductal Adenocarcinoma Cells
              </a>
            </td>
          <td>
            M. Zinovyeva, L. G. Nikolaev
          </td>
          <td>2025-09-22</td>
          <td>Molecular Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 In GBM, the most common genomic events include chromosome 7 gain, chromosome 9p loss, or chromosome 10 loss. At least 80% of GBM tumors have one or more of these chromosome alterations. Since these gains and losses involve entire chromosome arms, we predict that more than one gene on each arm functions to promote tumorigenesis. Using a combination of computational studies and chromosome-specific CRISPR screens, we identified a novel oncogene on chromosome 7 and two novel haploinsufficient tumor suppressor genes on chromosomes 9p and 10.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44b97bf505907c010f0bd84f701cb1ae5504c188" target='_blank'>
              CSIG-36. LEVERAGING GENOMIC ALTERATIONS IN GLIOBLASTOMA MULTIFORME
              </a>
            </td>
          <td>
            Tamar R Berger, Nora McGowan, Nitin H. Shirole, Muhannad Abu-Remaileh, Keith L. Ligon, Margaret M. Cusick, R. Beroukhim, W. Kaelin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad59c3801bee389a25c11b5127d71541daed409" target='_blank'>
              The tiny germline chromosomes of Paramecium aurelia have an exceptionally high recombination rate and are capped by a new class of Helitrons
              </a>
            </td>
          <td>
            O. Arnaiz, Frédéric Guérin, Arnaud Couloux, Caridad Miró-Pina, Guillaume Pellerin, I. Nekrasova, J. Amselem, J. Aury, Simran Bhullar, Andrea Frapporti, Emmanuelle Lerat, Isabelle Luyten, Sophie Malinsky, Nathalie Mathy, Alexey Potekhin, Vinciane Régnier, Natalia Sawka‐Gądek, Amandine Touzeau, Augustin de Vanssay, Coralie Zangarelli, H. Quesneville, M. Bétermier, K. Labadie, Laurent Duret, E. Meyer, S. Duharcourt, L. Sperling
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane E. Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, A. Stergachis, Rachel J. O’Neill, K. Miga, Steven L. Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cf151a47356eecc8542d42d62be9db47d608e1" target='_blank'>
              EpiBlot: Joint Mapping of Chromatin Accessibility and Targeted Proteomics in HER2-expressing Breast Cancer Systems
              </a>
            </td>
          <td>
            A. F. Khartchenko, Trinh Lam, Nadine Goldhammer, Marin Bont, Shruti Warhadpande, Jennifer M. Rosenbluth, Amy E. Herr
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>None</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is an aggressive and lethal tumor with limited effective treatments. Recently, oncolytic herpes simplex virus (oHSV) G47Δ was approved in Japan for recurrent GBM. To enhance its therapeutic efficacy, we have explored combinations with inhibitors targeting DNA damage response (DDR). In a comprehensive screening of DDR inhibitors affecting non-homologous end joining (NHEJ), alternative NHEJ, homologous recombination (HR), and single-strand break repair pathways, we identified inhibition of DNA-dependent protein kinase catalytic subunit (DNA-PKcs), a critical regulator of NHEJ repair, as particularly promising. Inhibition of DNA-PKcs with M3814 (Peposertib), currently in clinical trials for GBM and solid tumors, synergized with two different oHSVs: G47Δ (γ34.5 and ICP47 deletions, ICP6 inactivation) and MG18L (Us3 deletion, ICP6 inactivation) in killing six patient-derived GBM stem-like cell (GSC) lines and four matched serum-cultured GBM cells, but was antagonistic in human astrocytes. This synergy was associated with degradation of DDR proteins, increased DNA damage and reduced repair, decreased stem-like cells, increased virus replication, cell cycle block, and enhanced cell death. To validate selectivity and restrict inhibition to GBM cells, we created GSCs with doxycycline-inducible DNA-PKcs shRNA, which significantly reduced sphere formation (Stemness). Intratumoral injection of oHSV into GSC tumors with transient doxycycline-induced DNA-PKcs knockdown significantly prolonged mouse survival compared to either monotherapy, achieving about 20-50% long-term survivors. Treatment of GSC-derived tumors with intratumoral oHSV and systemic M3814 also extended survival, highlighting the potential clinical utility of this strategy. These findings underscore the vulnerability of human GSCs to DNA-PKcs inhibition in the absence of DNA-damaging agents and its robust synergy with oHSV. Given that both G47Δ and M3814 have been evaluated in clinical trials, this study supports their continued clinical translation and further investigation into combination therapies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe5406a8f1e7142803bec335f17d2677766f8a69" target='_blank'>
              EXTH-45. Oncolytic Herpes Simplex Virus Synergizes with DNA-PKcs Inhibitors to Treat Glioblastoma
              </a>
            </td>
          <td>
            Hirotaka Fudaba, Hiroaki Wakimoto, R. Martuza, S. Rabkin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Abstract Human oncogenic viruses contribute significantly to the global health burden and include seven types: Epstein–Barr virus, hepatitis B virus, human T-cell leukemia virus type 1, human papillomavirus, hepatitis C virus, Kaposi’s sarcoma-associated herpesvirus, and Merkel cell polyomavirus. While the roles of latent or integrated viral genomes in cancer have been documented, emerging evidence highlights the contribution of defective viruses—those carrying intragenic deletions or loss-of-function mutations—in promoting viral oncogenesis. These altered genomes often lack genes essential for lytic replication or immune recognition, which enhances their persistence and immune evasion. In virus-associated diseases, specific patterns of gene retention and deletion suggest that host-driven selective pressures drive the emergence of these altered genomes. This review examines the generation, prevalence, and functional impact of these viruses, reframing them as active participants in disease development and progression. Recognizing their role offers new insights into viral tumor evolution and creates opportunities for applications in viral diagnostics and targeted intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389381b25524966b33d29c79924df9e5f4c7676f" target='_blank'>
              Defective but tumorigenic: the evolutionary and functional roles of mutated oncoviruses
              </a>
            </td>
          <td>
            Yoshitaka Sato, Yusuke Okuno, Takayuki Murata, Hiroshi Kimura
          </td>
          <td>2025-09-26</td>
          <td>FEMS Microbiology Reviews</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf7e3130dd32248dd76079d5a6fd192ad3103a10" target='_blank'>
              Germline-limited transposon remnants foster somatic genome diversification
              </a>
            </td>
          <td>
            Bozhidar-Adrian Stefanov, M. Nowacki
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Repetitive DNA sequences, as transposable elements (TEs) and satellite DNA (satDNA) spread and diversify within host genomes, impacting genome biology in numerous ways. In the first part of this review, we emphasize the evolutionary importance of satDNAs and TEs, providing a short summary of their roles and the mechanisms by which they influence the structure and function of genomes. We also discuss the broad, complex, and extensive relationships between TEs and satDNAs. Following that, we bring together different mechanisms on the generation of satDNA from TE, as it has been demonstrated that almost any part of any type of TE can undergo tandemization and produce novel satDNAs. Importantly, we here present a hypothesis that would explain the existence of particular types of monomers, namely composite satDNA monomers which display multiple subsequent stretches of similarity to various TEs, for which the explanation was lacking so far. We propose that even highly shuffled and degraded TE remnants residing in heterochromatin "TE graveyards" can give rise to new satDNA sequence monomers, transforming these genomic loci into DNA "recycling yards." Furthermore, we emphasize important evolutionary questions regarding the causes, mechanisms, and frequency of these occurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f67b41471ebec0a1b45c2936430e5e0e7ad266" target='_blank'>
              Satellite DNAs rising from the transposon graveyards.
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Patrik Majcen, Miroslav Plohl
          </td>
          <td>2025-09-30</td>
          <td>DNA research : an international journal for rapid publication of reports on genes and genomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Although there are known sex differences in cancer incidence, severity, and treatment, the sex chromosomes are typically excluded from genomic analyses because of the unique technical challenges associated with assessing their copy number, sequence variation, and expression. Here we assess sex chromosome complement in three widely-used human genomics datasets from normal (non-cancerous) tissues, primary tumors, and cancer cell lines and study the effects on genome-wide gene expression. Expected sex chromosome complements based on reported patient sex were observed in non-cancerous tissues, but about half of tumors and cancer cell lines showed loss of typical sex chromosome gene expression across tissue types with three categories: loss of chromosome Y (LOY), loss of chromosome X (LOX) and reactivation of the inactive X chromosome (XaXa). Genes consistently differentially expressed in tumors with loss of chromosome X, loss of chromosome Y, or loss of X chromosome inactivation are associated with the hallmarks of cancer and include both sex-linked and autosomal genes from nearly all chromosomes, druggable genes, and genes with molecular functions relevant to cancer signaling, such as kinase activity. Strikingly, tumors that are X0, including tumors from female patients that have lost an X chromosome and tumors from male patients that have lost a Y chromosome, cluster together by gene expression profile. Patients with tumors that have LOX or LOY had poorer survival outcomes compared to those with tumors that had maintained their sex chromosome complement. Further, LOX and LOY eliminates nearly all of the differential gene expression between tumors from different patient sexes, affecting sex chromosomal and autosomal gene expression. Going forward, considering patient sex as well as the entire genome, including assessment of the sex chromosome complement, will provide additional insights into personalized tumor etiology, progression, treatment, and patient outcome. Teaser Loss of typical sex chromosome complement is present in primary tumors and cancer cell lines across tissue types. We identify consistent autosomal gene expression changes across multiple cancers when the sex chromosome complement is altered, and show that loss of sex chromosomes reduces tumor gene expression differences between patient sexes. We show that loss of sex chromosomes is also associated with poorer patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2224f8588ccaf63733d9224d9f7545eea3b5b510" target='_blank'>
              Reproducible autosomal gene expression changes with loss of typical X and Y complement across tumor types
              </a>
            </td>
          <td>
            Seema B. Plaisier, Robert Phavong, Mason Farmwald, Mariah Lee, Teagen Allen, Malli Swamy, Ilsa Rodriguez, MacKenzie Wells, Nadia Phaneuf, S. Massey, Jared Del Rosario, Juvelyn Hart, Alexander Mangelsdorf, Martin Van Der Jagt, Alex R. DeCasien, K. Buetow, Melissa A. Wilson
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive soft tissue sarcoma with myogenic features affecting children and adolescents. The high-risk fusion-positive RMS subtype (FP-RMS), driven by the oncogenic chimeric transcription factor PAX3–FOXO1, shows 5-year overall survival not exceeding 30%. Here, we examine the impact of neddylation inhibition, a post-translational modification in which the NEDD8 peptide is conjugated to proteins, on the tumorigenic properties of FP-RMS. Here, we report that the NAE1 and UBA3 genes encoding the two subunits of the NEDD8-activating enzyme (NAE) heterodimer are upregulated in FP-RMS patients compared to healthy skeletal muscle tissues and highly expressed in RMS among several tumor types. Furthermore, DepMap analyses showed that FP-RMS cell lines are among the most sensitive to both NAE1 and UBA3 CRISPR-mediated knockout as well as to NAE pharmacological inhibition with MLN4924 compared to other cancer cell lines. In agreement, FP-RMS cells treated in vitro with MLN4924 (Pevonedistat) exhibited cell proliferation decrease, G2/M cell cycle arrest, senescence, and caspase- and PARP1-dependent apoptosis. These phenotypes were associated with increased γH2AX nuclear foci and protein levels, DNA double-strand breaks (DSB), and reduced RAD51 levels. NAE1 and UBA3 individual silencing mirrors the major effects of MLN4924. In addition, MLN4924 also prevented FP-RMS tumor growth in vivo. Combining MLN4924 with irradiation enhanced apoptosis and the inhibition of colony formation, cell cycle progression, and anchorage-independent and tumor spheroids growth compared to single treatments. Molecularly, MLN4924 amplified the irradiation-induced DNA damage by increasing γH2AX and DSBs, while reducing RAD51 expression and DNA-PKcs activation, both of which are involved in DNA repair. Collectively, our results suggest that the neddylation pathway is deregulated in FP-RMS, representing a potential therapeutic target. Therefore, MLN4924 could be considered as an anti-tumorigenic compound and a novel radiosensitizer in FP-RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab549fd592623cfae81765fa8c4ce22b89c2e02b" target='_blank'>
              Neddylation inhibition induces DNA double-strand breaks, hampering tumor growth in vivo, and promotes radiosensitivity in PAX3–FOXO1 rhabdomyosarcoma
              </a>
            </td>
          <td>
            F. Aiello, Lucrezia D'Archivio, Marika Attili, Erika Ferraro, Elisa Macrí, Riccardo Mazzocchi, Matteo Cassandri, S. Pomella, Valeria Tocco, M. Pezzullo, C. De Stefanis, S. Codenotti, Giovanni Barillari, C. Marchese, A. Fanzani, F. Megiorni, Janet Shipley, Marielle Yohe, S. Gatz, Peter J. Houghton, Giovanni Cenci, C. Quintarelli, F. Locatelli, Francesco Marampon, B. De Angelis, R. Rota
          </td>
          <td>2025-11-03</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Simple Summary Many gene fusions have been implicated in the development and progression of human cancers. Our group and others have identified specific gene fusions involving the MAML2 gene in human cancers, and we were interested in searching for additional novel MAML2 fusions. Our analysis with the Caris Life Sciences molecular database demonstrated that most of the novel fusions were associated with increased fusion burden and genomic loss of heterozygosity, which are potentially suggestive of genomic instability, except for ATXN3::MAML2 which warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa520a41486a278de5084f190cbe8e828a82921" target='_blank'>
              Novel MAML2 Fusions in Human Malignancy
              </a>
            </td>
          <td>
            Takefumi Komiya, K. Sweeney, Chao H. Huang, A. Crymes, E. S. Antonarakis, Andrew Elliott, Matt Oberley, Mark G. Evans
          </td>
          <td>2025-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, D. D. Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="

 Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored.



 We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls.




 AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g.,
 FLT3
 ,
 RUNX1
 , and
 CD33
 ) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML.




 This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgen Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe27e8860b31d75137bdc5403fdf3407806c753" target='_blank'>
              Global re-organisation of genome architecture at the transition to gametogenesis
              </a>
            </td>
          <td>
            Tien-Chi Huang, Maria Rigau, V. Malysheva, Chad Whildling, Stella Siciliani, Jingyu Li, Irina Balaguer Balsells, Pavel Artemov, Camille Dion, M. Spivakov, Juan M. Vaquerizas, Petra Hajkova
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bdb65fc0708e21a785db03ef4faf2be314ea734" target='_blank'>
              E2 displacement of CIP2A from TOPBP1 activates the DNA damage response during papillomavirus life cycles
              </a>
            </td>
          <td>
            A. T. Prabhakar, C. James, Allison Q. Nguyen, Phoebe Bridy, Xu Wang, Ryan J. Zoldork, Jenny D Roe, Austin J. Witt, Srimanya S. Panidepu, Molly L Bristol, Krista Dalton, Andreas Wieland, Ahmed Diab, Anthony C. Faber, Elliot J Androphy, Devraj Basu, Paul F. Lambert, Megan E. Spurgeon, I. Morgan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution.
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Determining the correct structure of large, interspersed duplications and related complex genomic rearrangements in genetic disease is critical when establishing causal roles and requires a technology able to span the entire duplicated segment(s) on single molecules. We assessed the use of Bionano optical genome mapping (OGM) for this purpose. We combined OGM, Illumina short-read sequencing and fluorescence in situ hybridisation (FISH) to characterise three large interspersed duplications/triplications, and used the deepC algorithm to predict impact on local topologically-associating domains (TADs), assisting functional interpretation. Case 1 harboured paired interspersed duplications (244/323 kb) on chromosome 13. By analysing multiple molecules > 300 kb completely spanning the smaller duplication, we unambiguously determined the correct structure, which potentially alters the TAD containing FGF9, a candidate gene. In Case 2, involving a child with hypertrichosis and gingival hyperplasia (HTC), duplications on chromosomes 16 (2.01 Mb) and 17 (564 kb) were linked on short-read sequencing. By obtaining three OGM molecules spanning the 564 kb segment, we deduced that a t(16;17) translocation was present, which we confirmed by FISH. This interpretation has important implications for clinical risk and highlights KCNJ2 as a potential driver of the HTC3 locus at 17q24.3. Case 3 involved a complex chromoanasynthesis event on chromosome 20. OGM readily resolved all but two of 12 alternative structures; however full resolution required reads to span a 627 kb duplication, which we could not achieve consistently. OGM represents a powerful tool for disambiguating complex structural variants, but requires multiple individual reads to completely span the duplicated segment. In our hands the upper size of duplications that could be resolved was ~ 550 kb. Deducing the correct configuration is critical both for mechanistic understanding of pathogenesis and accurate recurrence risk counselling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e568829e237f4a57a0d68d6063ed9794d08d203" target='_blank'>
              Exploring the size limits of Bionano optical genome mapping to resolve alternative structures of linked interspersed chromosomal duplications
              </a>
            </td>
          <td>
            Yang Pei, E. Calpena, Jill M. Brown, Ron Schwessinger, Lucy Platts, Simon J. McGowan, Tazeen Ashraf, F. Forzano, Jane A. Hurst, Wendy D. Jones, Ajoy Sarkar, Richard J. Gibbons, S. Twigg, Andrew O. M. Wilkie
          </td>
          <td>2025-11-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3983db3a7f5cc2d617ff079fb79e12e467a39c" target='_blank'>
              Hypomethylation may drive CLPB overexpression which predicts poor survival outcomes in breast cancer
              </a>
            </td>
          <td>
            Sena Sonmez, S. Noyan, T. Acun
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739af24c682d2782ba0306bcab9ddd9444127eae" target='_blank'>
              Circulating DNA reveals nucleosome occupancy patterns that are associated with nucleosome-DNA affinity and are affected in cancer
              </a>
            </td>
          <td>
            Marianne Richaud, E. Pisareva, Paul Burgat, Alain R. Thierry, Jacques Colinge
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: The A2UCOE sequence, positioned between the HNRPA2B1 and CBX3 gene promoters in the human genome, supports durable and consistent expression of integrated transgenes, even within compact heterochromatin domains such as centromeres. This project focuses on evaluating the dual-component hypothesis of A2UCOE function by analyzing alternative DNA elements that possess CpG-rich content and divergent promoter features.
Method: To investigate expression stability, lentiviral vectors carrying eGFP reporter constructs driven by novel UCOE candidates and various A2UCOE subregions were introduced into P19 and F9 mouse embryonal carcinoma cells. Expression was tracked over time, both before and after lineage-specific differentiation toward neuroectoderm and endoderm. To examine the proposed bipartite model of UCOE function, we employed two types of CpG-rich, bidirectionally transcribed elements: the endogenous SETD3–CCNK housekeeping gene pair, and a synthetically arranged divergent configuration composed of RPS11 and HNRPA2B1 promoters.
Results: Placing these regulatory elements in either orientation upstream of the SFFV-eGFP reporter gene—known for its susceptibility to transcriptional silencing—conferred a noticeable, though incomplete, resistance to silencing when compared to the full activity of the reference 1.5A2UCOE-SFFV-eGFP construct. This partial protective effect was consistently observed in both P19 and F9 cell lines, prior to and following their differentiation. In conclusion, we successfully identified a naturally occurring (SETD3–CCNK) and synthetically engineered (RPS11–HNRPA2B1) pair of divergent promoters that exhibited measurable but incomplete UCOE-like activity relative to the established HNRPA2B1–CBX3 core element.
Conclusion: This study demonstrates that natural and synthetic divergent promoter pairs confer significant, though partial, resistance to transgene silencing. This finding directly supports the A2UCOE's dual-component hypothesis, confirming that CpG-rich bidirectional architecture is key for sustaining stable expression through differentiation and in challenging genomic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ab8136cee5a33079a29d9b8aa6e3c61e332c4" target='_blank'>
              Testing of novel CpG chromatin fragments as UCOE candidates for improved gene therapy vectors
              </a>
            </td>
          <td>
            Ömer Faruk Anakök, Ali Osman Arslan
          </td>
          <td>2025-11-13</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract NUPTs are DNA sequences of plastid origin that are present in plant nuclear genomes in varying, though typically low, amounts. It is assumed that they are continuously formed and, due to their potentially mutagenic effect, they are removed at a constant turnover rate, which should result in an exponential decay of their age distributions and a negative correlation between age and size. However, these assumptions are based on analyses from a limited number of species and have never been explicitly tested. To gain insight into the mechanisms driving the origin and evolution of NUPTs, here we surveyed the plastid and nuclear genomes of 30 species representing the main angiosperm (flowering plants) lineages. By modeling the distribution of ages and sizes, examining their linear arrangement across the plastid genome, and statistically assessing spatial biases with respect to other genomic features, we showed that NUPTs are: (i) formed by both continuous and episodic mechanisms; (ii) unevenly represented across the plastid genome; (iii) consistently associated with certain classes of RNA genes, in particular rRNA, tRNA, and regulatory RNA genes; (iv) differentially contributing to structural genes; and (v) closer than expected to different superfamilies of transposons in a species-specific manner. Our results reveal the unexpected complexity in the mechanisms driving the origin of NUPTs, which not only involve their continuous formation but also episodic events, highlight their role as a major source of noncoding RNA genes and other genomic features, and provide a more complete picture of the different drivers of evolutionary change at the genome level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cddf52eeced1161dba2e2d7fa0a44d7221d0674f" target='_blank'>
              Episodic and Ongoing Mechanisms Drive Plastid-Derived Nuclear DNA Evolution in Angiosperms
              </a>
            </td>
          <td>
            Juan Pablo Marczuk-Rojas, Ana D Maldonado, Lorenzo Carretero-Paulet
          </td>
          <td>2025-10-13</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) enables the direct reprogramming of somatic cells into functional oocytes, albeit with a diploid genome. To address ploidy reduction, we investigated an experimental reductive cell division process, termed mitomeiosis, wherein non-replicated (2n2c) somatic genomes are prematurely forced to divide following transplantation into the metaphase cytoplasm of enucleated human oocytes. However, despite fertilization with sperm, SCNT oocytes remained arrested at the metaphase stage, indicating activation failure. Artificial activation using a selective cyclin-dependent kinase inhibitor successfully bypassed this arrest, inducing the segregation of somatic chromosomes into a zygotic pronucleus and a polar body. Comprehensive chromosome tracing via sequencing revealed that homologous chromosome segregation occurred randomly and without crossover recombination. Nonetheless, an average of 23 somatic chromosomes were retained within the zygote, demonstrating the feasibility of experimentally halving the diploid chromosome set. Fertilized human SCNT oocytes progressed through normal embryonic cell divisions, ultimately developing into embryos with integrated somatic and sperm-derived chromosomes. While our study demonstrates the potential of mitomeiosis for in vitro gametogenesis, at this stage it remains just a proof of concept and further research is required to ensure efficacy and safety before future clinical applications. Researchers induced ploidy reduction in human oocytes generated by somatic cell nuclear transfer, enabling fertilization and embryo development with integrated somatic and sperm chromosomes, highlighting a proof-of-concept for in vitro gametogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f10451bff8c497dce7391045b4480c619f68e2" target='_blank'>
              Induction of experimental cell division to generate cells with reduced chromosome ploidy
              </a>
            </td>
          <td>
            N. Marti Gutierrez, A. Mikhalchenko, Maria Shishimorova, D. Frana, C. Van Dyken, Ying Li, Hong Ma, A. Koski, Dan Liang, Sang-Goo Lee, Daniel Eyberg, Zahra Safaei, Eunju Kang, Yeonmi Lee, Thomas O’Leary, David Lee, Sacha A. Krieg, Diana Wu, Elizabeth Rubin, Paula Amato, S. Mitalipov
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Centromeres are fundamental chromosomal structures that ensure accurate chromosome segregation during cell division. Despite their conserved and essential role in maintaining genomic stability, centromeres are subject to rapid evolutionary change. At the heart of centromere identity is the histone H3 variant CENP-A, an epigenetic mark that defines and propagates active centromeres and is essential for their function. Recent evidence supports a rapid evolution of centromere DNA sequences but also suggests a certain degree of flexibility in CENP-A deposition and propagation. The phenomenon of centromere drift, recently observed in humans, highlights how the dynamic repositioning of CENP-A and associated epigenetic environment over time maintains a regulated equilibrium, ensuring centromere function despite positional variation. Understanding these processes is crucial for unraveling centromere dynamics and their broader implications for genome stability and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0e9c5778af501dce621770da4beeabba7210f5" target='_blank'>
              The dynamic centromere
              </a>
            </td>
          <td>
            Angela Enriquez, Yael Nechemia-Arbely
          </td>
          <td>2025-10-16</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="DNA cytosine modifications, including 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), are key epigenetic regulators with distinct functions. Dissecting the ternary code (C, 5mC, 5hmC) across tissues and cell types remains a critical priority due to the limitations of traditional profiling methods based on bisulfite conversion. Here, we leverage the combined bisulfite and enzymatic (bACE) conversion with the Mouse Methylation BeadChip to generate 265 base-resolution ternary-code modification maps of 5mC and 5hmC across 29 mouse tissue types spanning 8–76 weeks of age and both sexes. Our atlas reveals a complex grammar of 5hmC distribution, jointly shaped by cell mitotic activity, chromatin states, and interplay with 5mC at the same and neighboring CpG sites. Of note, we demonstrate that 5hmC significantly complements 5mC-based biomarkers in delineating cell identity in both brain and non-brain tissues. Each modification state, including 5hmC alone, accurately discriminates tissue types, enabling high-precision machine learning classification of epigenetic identity. Furthermore, the ternary methylome variations extensively implicate gene transcriptional variation, with age-related changes correlated with gene expression in a tissue-dependent manner. Our work reveals how tissue, sex, and age jointly govern the dynamics of the two cytosine modifications, augments the scope of DNA modification biomarker discovery, and provides a reference atlas to explore epigenetic dynamics in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf56e8a468cf8f7c8ba62217313ee3ee7ed7a698" target='_blank'>
              A ternary-code DNA methylome atlas of mouse tissues
              </a>
            </td>
          <td>
            S. Lee, David C Goldberg, Cameron Cloud, Jared B. Parker, Christopher J. Krapp, Christian E. Loo, Elliot Kim, Ivan Zhao, Chengcheng Jin, R. Porecha, M. Bartolomei, Rahul M. Kohli, Wanding Zhou
          </td>
          <td>2025-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="L1 retrotransposons are the only protein-coding active transposable elements in the human genome. While typically silenced in normal cells, they are highly expressed in many human epithelial cancers, including high-grade serous ovarian cancer (HGSC), and can integrate into the genome through retrotransposition. De novo L1 insertions are known to contribute to genomic instability and cancer evolution in epithelial malignancies, including HGSC, suggesting they might also play a role in intra-patient tumor heterogeneity. Here, we quantified de novo L1 insertions in clinical HGSC specimens and uncovered high heterogeneity in total L1 insertion events (L1 burden) between patients. HGSC tumors with high L1 burden were highly proliferative, while tumors with low or no L1 insertions showed enrichment of immune response and cell death pathways. Although the overall L1 burden was similar across different tumor sites within the same patient, the specific L1 insertions (L1 profiles) diverged significantly more than their single nucleotide variants (SNVs) profiles. Taken together, these findings demonstrate that L1 activity and retrotransposition are highly dynamic in vivo and can contribute substantially to tumor genome plasticity, especially at late stages of cancer progression. The patient-specific propensity of acquiring L1 insertions (L1 burden) could be driven by molecular properties of progenitor tumor. Retrotransposition-associated DNA damage and/or replication stress could be a potential molecular vulnerability for precision cancer medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aca4e68c0aaf55e8c323fa640292722deea9ff" target='_blank'>
              Dynamic and Ongoing De Novo L1 Retrotransposition Contributes to Genome Plasticity and Intrapatient Heterogeneity in Ovarian Cancer.
              </a>
            </td>
          <td>
            Barun Pradhan, J. Oikkonen, Kaiyang Zhang, J. M. Botto, Minna R Eriksson, Srividhya Sundaresan, Fatih Genç, T. R. Pisanic, Matias Marin Falco, Yilin Li, S. Pikkusaari, K. Lavikka, Giulia Micoli, G. Marchi, T. Muranen, K. Huhtinen, Anna Vähärautio, R. Badge, Kathleen H. Burns, S. Hietanen, J. Hynninen, G. Faulkner, S. Hautaniemi, L. Kauppi
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcc185fd751f8e84a66d907f599c28cc29b6d85" target='_blank'>
              Human Oncogene EWS::FLI1 Functions as a Pioneer Factor in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Diego Velázquez, Cristina Molnar, Jose Reina, Jaume Mora, Cayetano Gonzalez
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Centromeres, characterized by their unique chromatin attributes, are indispensable for safeguarding genomic stability. Due to their intricate and fragile nature, centromeres are susceptible to chromosomal rearrangements. However, the mechanisms preserving their functional integrity and supporting nucleus homeostasis following breakages remained enigmatic. In this study, we use wheat ditelosomic stocks, which arise from centromere breakage, to explore the genetic and epigenetic alterations in damaged centromeres. Our investigations unveil novel chromosome end structures marked by de novo addition of telomeres, as well as localized chromosomal shattering, including segment deletions and duplications near centromere breakpoints. We reveal that the damaged centromeres possess a remarkable capacity for self-regulation, through employing structural modifications such as expansion, contraction, and neocentromere formation to maintain their functional integrity. Centromere breakage triggers nucleosome remodeling and is accompanied by local transcription changes and chromatin reorganization, and subsequently may contribute to the stabilization of broken chromosomes. Our findings highlight the resilience and adaptability of plant chromosomes in response to centromere breakage, and provide valuable insights into the stability of centromeres, thereby offering promising prospects to manipulate centromeres for targeted chromosomal innovation and crop genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c301bcc4744c9e967b82a2d0c48310497c9c67e9" target='_blank'>
              Adaptation of centromere to breakage through local genomic and epigenomic remodeling in wheat.
              </a>
            </td>
          <td>
            Jingwei Zhou, Yuhong Huang, Huan Ma, Yiqian Chen, Chuanye Chen, Fang Han, H. Su
          </td>
          <td>2025-09-30</td>
          <td>Genome research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Gliomas are highly aggressive brain tumors with poor outcomes. CDC25C, a phosphatase that activates the CDK1–Cyclin B1 complex at the G₂/M checkpoint, is implicated in several cancers but its role in glioma progression and tumor immunity remains unclear. We integrated bulk transcriptomic datasets (TCGA, CGGA, GEO) and single-cell RNA sequencing data to evaluate CDC25C expression, prognostic significance, and associated pathways in gliomas. Immune cell infiltration and checkpoint gene correlations were assessed via computational deconvolution. Additionally, U87MG glioma cells were transfected with siRNAs targeting CDC25C to achieve transient knockdown. Proliferation was measured by EdU incorporation, cell-cycle distribution by flow cytometry, and expression of G₂/M regulators (Cyclin B1, CDK1, phospho-CDK1) by Western blot. CDC25C was overexpressed in gliomas compared to normal brain, and high CDC25C levels correlated with advanced tumor grade, other poor prognostic markers, and shorter overall survival. Gene set analyses revealed CDC25C-high tumors were enriched in cell-cycle and immunosuppressive pathways. High CDC25C expression was associated with greater infiltration of regulatory T cells (Tregs) and M0 macrophages, alongside upregulation of immune checkpoints PD-1, PD-L1, and CD80. Single-cell RNA-seq localized CDC25C-driven proliferation to a hyperproliferative MKI67-positive tumor cell subpopulation. Functionally, CDC25C knockdown in U87MG cells suppressed proliferation, induced G₂/M arrest, and inhibited the CDK1–Cyclin B1 mitotic axis. CDC25C is a critical driver of glioma cell proliferation and an enabler of immune evasion. Its overexpression may link unchecked mitotic progression with an immunosuppressive microenvironment, underscoring CDC25C as a promising prognostic biomarker and therapeutic target in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/133482035bbc170923eea731e0b85f3e2387d498" target='_blank'>
              Cell division cycle 25 C (CDC25C) mediates cell-cycle progression and immune evasion in glioma
              </a>
            </td>
          <td>
            Anqi Qiao, QiuYue Zhang, Jiakai He, Fan Song, Tenglu Cui, Yi Liu, Pengfei Sun
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Chromosome organization is critical for the maintenance of genetic integrity. Most studies of bacterial nucleoids have focused on growing rod-shaped organisms. Studying nucleoid dynamics during Myxococcus xanthus development offers the unique opportunity to investigate the localization of two sister chromosomes as rod-shaped cells transition into round spores. During starvation-induced multicellular development, DNA replication is required for M. xanthus rods to transition into spores with two copies of the chromosome. Here, we report novel approaches using confocal fluorescence microscopy to observe the chromosome number and arrangement, and nucleoid localization in developing cells in situ. We discovered that sister chromosomes are present in some rods and transitioning cells (TCs) early in development. The arrangement of the two chromosomes in developing cells was novel compared to predivisional growing cells studied previously. We observed segregated nucleoids in ~40% of TCs and spores. The majority of TCs contained a crescent-shaped nucleoid along one side, perhaps due to ongoing chromosome segregation, whereas most spores appeared to have undergone nucleoid decondensation. During unicellular glycerol-induced sporulation of M. xanthus, we observed segregated nucleoids in only ~10%–20% of TCs and spores. In addition, early in starvation-induced development, we discovered a subpopulation of cells that may be spheroplasts destined for lysis, which is the fate of most cells under these conditions. Chromosome segregation in developing M. xanthus may be a bet-hedging strategy to increase survival under different conditions and/or an evolutionary remnant of ancestral events that included cell division to produce spores with one copy of the chromosome. IMPORTANCE The cell cycle normally involves DNA replication, chromosome segregation, and cell division. During starvation-induced Myxococcus xanthus development, DNA replication is necessary for progression to spore formation, which occurs without cell division, resulting in spores with two copies of the chromosome. The organization of sister chromosomes during the morphological change of rod-shaped cells into round spores was unknown. We discovered that the two nucleoids often segregate during the transition from rods to spores. Mature spores contained decondensed nucleoids. Our observations raise important questions about the mechanism of chromosome segregation during M. xanthus development and the reason for its existence. We also discovered a subpopulation of developing cells with characteristics suggesting they are spheroplasts on the verge of cell death. The cell cycle normally involves DNA replication, chromosome segregation, and cell division. During starvation-induced Myxococcus xanthus development, DNA replication is necessary for progression to spore formation, which occurs without cell division, resulting in spores with two copies of the chromosome. The organization of sister chromosomes during the morphological change of rod-shaped cells into round spores was unknown. We discovered that the two nucleoids often segregate during the transition from rods to spores. Mature spores contained decondensed nucleoids. Our observations raise important questions about the mechanism of chromosome segregation during M. xanthus development and the reason for its existence. We also discovered a subpopulation of developing cells with characteristics suggesting they are spheroplasts on the verge of cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe53b6817f634726006f01b89cdd1037a4dfb40" target='_blank'>
              Segregation of sister chromosomes during the shape change of developing Myxococcus xanthus cells
              </a>
            </td>
          <td>
            Y. Hoang, Yann S Dufour, Lee Kroos
          </td>
          <td>2025-09-26</td>
          <td>Journal of Bacteriology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aging is a critical risk factor for cancer development with genetic instability presenting as a common hallmark. Alternative DNA-forming sequences are endogenous sources of genetic instability that are enriched at mutation hotspots in cancer genomes. Using a transgenic mutation-reporter mouse model containing an H-DNA-forming sequence from a mutation hotspot in Burkitt lymphoma, we demonstrate tissue-specific effects of aging on DNA structure-induced mutagenesis, with H-DNA mutation frequencies increasing in spleen and liver and decreasing in brain tissues. DNA sequencing revealed a correlation of increasing large deletions with increasing mutation frequencies. Further, we observed tissue-specific modulation of mechanisms of H-DNA processing, including decreased nucleotide excision repair activity in aged brain tissues and increased cleavage activity on H-DNA structures in aging spleen tissues. Together, these findings provide significant insights into the relationship between aging and cancer-associated genetic instability, aiding in the delineation of the underlying mechanisms of age-associated cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df234a222020175cd31562839285cb02431f2f82" target='_blank'>
              Aging alters genomic instability at endogenous mutation hotspots in mice
              </a>
            </td>
          <td>
            Alexandra D'Amico, Tonia T. Li, Dennis Wylie, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: Bladder cancer is rare in young women, and advanced presentations are exceptionally uncommon. We report a de-identified case of a previously healthy 31-year-old female who developed rapidly progressive stage IV bladder cancer within 12 months of completing a three-dose Moderna mRNA vaccination series (May 2021, June 2021, December 2021). Case Findings: Comprehensive multi-omic profiling was performed using PBIMA (Molecular Surveillance and Individualized Targeted Immunotherapy Peptide Editing) and REViSS (Spike-associated Transcription-al/Translational Instability Surveillance), incorporating analyses of plasma-derived circulating tumor DNA, whole-blood RNA, and urine exosome proteomics. Dysregulated gene expressions were identified across oncogenic driver genes (KRAS, ATM, MAPK1, NRAS, CHD4, PIK3CA, and SF3B1), auxiliary tumor-promoting signals (TOP1, PSIP1, and ERBB2), and broad evidence of genome instability with impaired DNA repair (ATM, MSH2). Within circulating tumor DNA, a host–vector chimeric read mapped to chr19:55,482,637–55,482,674 (GRCh38), in cytoband 19q13.42, positioned ~367 kb downstream of the canonical AAVS1 safe harbor and ~158 kb upstream of ZNF580 at the proximal edge of the zinc-finger (ZNF) gene cluster. This sequence aligned with perfect 20/20 bp identity to a segment (bases 5905–5924) within the Spike open reading frame (ORF) coding region (bases 3674–7480) of the Pfizer BNT162b2 DNA plasmid reference (GenBank accession OR134577.1), despite the patient only receiving Moderna vaccinations. This apparent paradox is best explained by shared Spike ORF sequences within the expression cassette across both vaccine platforms; because Moderna has not deposited its proprietary plasmid sequence in NCBI, BLAST defaults to Pfizer’s published reference as the nearest available match. The integration site was located outside the canonical AAVS1 “safe harbor” and within a gene-dense, recombination-prone regulatory region, raising concern for transcriptional disruption, fusion transcript formation, and oncogenic potential. The probability of a random 20-base sequence perfectly matching a predefined target is approximately 1 in a trillion, making this alignment statistically compelling and highly unlikely to be an incidental artifact. Conclusions: This sentinel case report provides the first documented evidence of genomic integration of mRNA vaccine-derived genetic material in a human subject, documenting a temporal association between COVID-19 mRNA vaccination and aggressive malignancy, reproducible multi-omic evidence of oncogenic signaling, and a non–safe harbor host–vector integration event. While causality cannot be established from a single case, the con-vergence of (i) close temporal proximity to vaccination, (ii) genomic integration of a vaccine plasmid–derived spike gene fragment, and (iii) consistent transcriptomic and proteomic instability across biospecimens represents a highly unusual and biologically plausible pattern. These findings highlight an urgent need for systematic genomic sur-veillance, orthogonal validation with long-read sequencing, and larger cohort studies to rigorously define the im-pact of synthetic mRNA vaccine platforms on genome integrity and cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8848f5c106afb2dfada322a3642c497ad8b443d" target='_blank'>
              Genomic Integration and Molecular Dysregulation in Aggressive Stage IV Bladder Cancer Following COVID-19 mRNA Vaccination
              </a>
            </td>
          <td>
            J. A. Catanzaro, Nicolas Hulscher, Peter A. McCullough
          </td>
          <td>2025-10-05</td>
          <td>International Journal of Innovative Research in Medical Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ribosomal DNA (rDNA), the most abundantly expressed locus in the human genome, is represented by hundreds of units per cell. Each unit includes a 13 kb long region (47S rDNA) containing the genes of ribosomal particles, and a 30 kb long intergenic spacer (IGS). The 47S rDNA is transcribed with varying intensity in different units, some of which remain permanently silent. A key intermediator of this silencing is the promoter-associated RNA (pRNA) produced from a 2 kb long gene situated upstream of the rDNA transcription start site. Recent studies, including ours, suggest that the sequence variability, which normally occurs in mammalian cells, may account for the selective transcription of different rDNA units. The present work is based on the deep sequencing of a pRNA gene fragment and its RNA product and subsequent bioinformatic analysis. We found that a certain SNV, which converts the CCC motif into CCT, as well as deletions which reduce the number of (CCCT) tandem repeats, were significantly more frequent in the DNA than in the respective transcripts. These findings allowed us to establish directly the inhibitory effect of DNA variants on the expression of pRNA and thus (indirectly) the promoting effect on the production of ribosomal RNA. Our results also suggest that (CCCT)n/(GGGA)n DNA repeats and the respective (GGGA)n RNA repeats may form triplex structures, facilitating the function of pRNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891924cd9bf55653e2539c5b0034f6d9fcdd744" target='_blank'>
              Deletions in (CCCT)n repeat regions belonging to the human pRNA gene inhibit its expression
              </a>
            </td>
          <td>
            Nikola Chmúrčiaková, Adam Nógell, E. Smirnov, Dušan Cmarko
          </td>
          <td>2025-10-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Subtelomeres are imperfect repeats adjacent to telomeres that are repressed by heterochromatin. Although essential for genome integrity, their repetitive nature has thwarted dissection of local heterochromatin assembly and maintenance mechanisms. Here, we engineered Schizosaccharomyces pombe strains carrying fluorescent reporters at a single subtelomere. We find that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites. Telomere-proximal regions depend on canonical shelterin or RNA interference nucleation pathways, while telomere-distal regions require nucleosome remodelers, histone chaperones, and boundary-associated factors. Using multi-generational live imaging and targeted perturbations, we show that subtelomeric subdomains display position-specific, clonally variable silencing across a spectrum of robust to fragile epigenetic states. This clonal variegation is also induced by naturally occurring subtelomeric structural variants. These findings demonstrate that subtelomeric heterochromatin maintenance is not uniform but rather governed by local chromatin context and architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1892593b4d09d954d617fd50a949c2f67aa1bce" target='_blank'>
              Discrete Subdomains Establish Epigenetic Diversity in Subtelomeric Heterochromatin
              </a>
            </td>
          <td>
            Agnisrota Mazumder, John Cooper, Can Goksal, J. S. Khanduja, Richard I. Joh, Johannes J. Groos, Rosario Y.J. Brockhausen, Junko Kanoh, Mohammad R. Motamedi, Ilya J. Finkelstein, Bassem Al-Sady, S. Braun
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gemykibivirus, an emerging single-stranded DNA (ssDNA) virus of the recently established genus in the family of Genomoviridae, had been discovered in human blood and cerebrospinal fluid and a variety of other body fluids. However, the molecular mechanisms of gemykibivirus entrance into the host cells and its pathogenicity remain poorly understood. To investigate the host response of gemykibivirus, we used an infectious clone of gemykibivirus previously established through molecular biology techniques to rescue virus in HEK293T cells and analyzed the changes in the host transcriptome during the infection period by RNA-Seq. Our findings indicate that gemykibivirus can both express viral proteins and accomplish replication, and high-throughput transcriptome analysis identified a total 1732 significantly different genes. Functional enrichment analysis of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways for differentially expressed genes (DEGs) showed gemykibivirus involving several important pathways, including MAPK signaling pathway, Chemical carcinogenesis-reactive oxygen species and Oxidative phosphorylation. Interestingly, mitochondrial DNA-encoded mRNAs exhibited varying levels of upregulation, suggesting that gemykibivirus may be involved in mitochondrial fission and the regulation of mitochondrial function. Subsequently, a series of experiments proved that gemykibivirus can lead an increase in mitochondrial DNA copy number, promote the release of mtDNA into the cytoplasm, enhance reactive oxygen species production and trigger other cellular antiviral responses. Overall, we lay a foundation for revealing the relationship between Gemykibivirus and human diseases through mitochondrial functional alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67fc3d43e01f79ea8442e27ecafc7aa6d559e12" target='_blank'>
              Transcriptome Analysis Reveals Gemykibivirus Infection Induces Mitochondrial DNA Release in HEK293T Cells
              </a>
            </td>
          <td>
            Runbo Yang, Hao Yan, Yifan Wang, Wenqing Yang, Jianru Qin
          </td>
          <td>2025-09-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1147c4ec4d5050e89e9fab52610e8f2d66fcc3cc" target='_blank'>
              Integrative multi-modal analysis reveals the contribution of noncoding RNAs to post-treatment progression of IDH-mutant astrocytomas
              </a>
            </td>
          <td>
            A. Hartewig, S. Jaatinen, M. Annala, J. Haapasalo, Wei Zhang, H. Haapasalo, M. Nykter, Xin Lai, K. Rautajoki
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="


 Chromosomal instability is a well-recognized hallmark of tumorigenesis. However, it is not known how markers for chromosomal instability, such as total distance of DNA altered or total copy number variations (CNV), affect survival, particularly in IDH wild type glioblastoma. Additionally, whether alterations in pathways that impact cellular responses to chromosomal instability, such as the TP53 or RB1 axis, influence survival, similarly remain poorly understood.



 To better characterize potential prognostic impact of markers of chromosomal instability on patient outcomes, we assessed the Mayo Clinic cohort of IDH wild type glioblastomas (n=449) for whom chromosomal microarray data was available.



 We found that while total distance of DNA altered on microarray analysis was not associated with overall survival (OS) (p=0.4030, univariate analysis), the least altered tumors (stratified by quartiles) when assessed by numbers of CNVs demonstrated improved OS (p=0.0057, univariate analysis). We next evaluated alterations in the MDM2/4-TP53 signaling axis, as well as the CDKN2A/B-CDK4/6-RB1 axis, as both amplifications and losses impacting these pathways have been well characterized in adult diffuse gliomas. While TP53 status was not predictive of OS (p=0.1504, univariate analysis), amplification of MDM2 or MDM4 was predictive of decreased OS (p=0.0071, univariate analysis). We also found that tumors without alterations in the CDKN2A/B-CDK4/6-RB1 axis had improved OS when compared to altered tumors (p<0.0001, univariate analysis). Multivariate Cox regression analysis, adjusting for age and sex, confirmed prognostic impact of MDM2/4 amplification status (HR 1.4090, p=0.0370) and CNV status (HR 0.6798, p=0.00312) on OS.



 CNV status and MDM2/4 amplification represent previously unappreciated molecular markers for prognostication in IDH wild type glioblastoma. Further work is underway to confirm our findings through multi-institutional collaborations and to account for additional potential prognostic factors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7bd1ec122e6536bdfe28952f14c544a17f01a98" target='_blank'>
              EPCO-33. MOLECULAR MARKERS OF CHROMOSOMAL INSTABILITY, INCLUDING COPY NUMBER VARIATION AND MDM2/4 AMPLIFICATION, ARE PROGNOSTIC IN IDH WILD TYPE GLIOBLASTOMA
              </a>
            </td>
          <td>
            Sanghee Lim, Ahauve Orusa, J. Eckel-Passow, R. Jenkins, M. Ruff
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mitotic chromosomes give genome portions the required compaction and mechanical stability for faithful inheritance during cell divisions. They are shaped by the chromosomal condensin complex. Here, we record human chromosome dimensions from their appearance in prophase over successive times in a mitotic arrest. Chromosomes first appear long and uniformly thin. Then, individual chromosome arms become discernible, which continuously shorten and thicken—the longer a chromosome arm, the thicker it becomes. In the search for a molecular explanation of this behavior, given uniform condensin density, the popular loop extrusion model provides no obvious means by which longer chromosome arms become thicker. Instead, we find that simulations of an alternative loop capture model recapitulate key features of our observations, with re-arranging chromatin rosettes underpinning the gradually developing arm length-to-width relationship. Our analyses portray chromosomes as out-of-equilibrium structures in the process of transitioning towards, but on biologically relevant time scales not typically reaching, steady state. Chromosomes continuously develop their shape during cell divisions. Mitotic chromosomes gradually shorten and thicken. Longer chromosome arms become thicker than short arms. Anaphase sets in before most longer chromosome arms reach steady state. Loop capture simulations recapitulate observed chromosome shape changes. Mitotic chromosomes gradually shorten and thicken. Longer chromosome arms become thicker than short arms. Anaphase sets in before most longer chromosome arms reach steady state. Loop capture simulations recapitulate observed chromosome shape changes. Chromosomes continuously develop their shape during cell divisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3055d8d7767edb2d5ac8b163761950bc3ca2e716" target='_blank'>
              Progressive chromosome shape changes during cell divisions
              </a>
            </td>
          <td>
            Yasutaka Kakui, Y. Kusano, Tereza Clarence, Maya Lopez, T. Fallesen, T. Hirota, Bhavin S. Khatri, Frank Uhlmann
          </td>
          <td>2025-09-23</td>
          <td>EMBO Reports</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 MTAP encodes a key enzyme in methylthioadenosine (MTA) catabolism. MTAP loss, frequent across human cancers, leads to MTA accumulation and inhibition of PRMT5, a critical regulator of cell cycle and DNA repair. This metabolic vulnerability renders MTAP-deficient tumors selectively sensitive to PRMT5 inhibitors such as TNG-462 and TNG-908. Emerging data indicate that MTAP can also be epigenetically silenced via promoter hypermethylation, independent of genomic deletion, thereby evading detection by DNA-based diagnostics and excluding patients from PRMT5-targeted therapies. Here, we integrate MTAP methylation and expression profiles and evaluate PRMT5 inhibitor sensitivity across diverse tumor types. A comprehensive cohort comprising 6,187 samples with matched DNA methylation and gene expression data was assembled from 17 distinct TCGA projects. Data retrieval, raw RNA-seq read counts, and methylation data analysis were performed using the TCGAbiolinks R package, DESeq2 R package and HM450K arrays, respectively. In addition, we incorporated DNA methylation profiles from established human cell lines obtained from the ENCODE project. In vitro, MTAP-wild type cell lines (human: HCC3153 and HS578T and mouse: GL261); MTAP-null cell lines (human: MDA MB-231, RT4, RT112, UM-UC-3) and MTAP-hypermethylated cell lines (human: U87 and HepG2) were used for viability MTT assays after TNG-462 or TNG-908 treatment. Array assays revealed cg25162921 and cg00230302 probes targeting CpG sites associated with the MTAPlocus. The promoter-proximal probe cg25162921 exhibited a strong inverse correlation with MTAP expression in multiple tumor types, including lung squamous cell carcinoma (TCGA-LUSC; r=-0.48), bladder urothelial carcinoma (TCGA-BLCA; r=-0.42), and glioblastoma multiforme (TCGA-GBM; r=-0.45). We next investigated whether the mechanism of MTAP inactivation, genomic deletion and promoter hypermethylation, influenced cellular sensitivity to PRMT5 inhibition by viability MTT assy. MTAP genomic status and methylation levels were stratified at the promoter-proximal CpG site cg25162921 using DNA methylation from ENCODE project. In viability assays, MTAP-null cell lines or MTAP-hypermethylated models were more sensitive to TNG-462 and TNG-908 with a reduced IC50 compared to MTAP-wild type cell lines. We identify promoter hypermethylation as a distinct mechanism of MTAP silencing, broadening the spectrum of MTAP-low tumors beyond genomic loss. These findings support integration of epigenetic biomarkers to refine patient selection and enhance response rates in PRMT5-targeted clinical trials.



 Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, Esther Cabañas Morafraile, Bernard Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, Juan A Ardura, Balazs Gyorffy, Victor Moreno, Alberto Ocaña. Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de65a469e9e37ed4b177eca66de3a9ae88f86296" target='_blank'>
              Abstract C026: Epigenetic silencing of MTAP defines a distinct class of PRMT5-inhibitor-sensitive cancers
              </a>
            </td>
          <td>
            Luis Álvarez, Dalma Muller, Lucia Paniagua, Verónica Alonso, Manuel Pedregal, E. C. Morafraile, B. Doger, Emiliano Calvo, Irene Moreno, Irene Gutiérrez, J. Ardura, Balázs Győrffy, Víctor Moreno, Alberto Ocaña
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Telomeres protect chromosome ends and their length varies significantly between organisms. Because telomere length variation is associated with various biomedical and eco-evolutionary phenotypes, many biological fields are interested in understanding its biological significance. Here, we introduce Topsicle, a computational method that estimates telomere length from whole genome long-read sequencing data using k-mer and change-point detection analysis. Simulations show Topsicle is robust to sequencing errors and coverage. Application of Topsicle to plant and human cancer cells shows high accuracy and comparable results to direct telomere length measurements. We predict Topsicle will be a useful tool for studying telomere biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aff7ef6bf5d73645c58ce8d4d7e64383c77d646" target='_blank'>
              Topsicle: a method for estimating telomere length from whole genome long-read sequencing data
              </a>
            </td>
          <td>
            Linh Nguyen, Jae Young Choi
          </td>
          <td>2025-09-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Genomic instability and the accumulation of DNA damage are hallmarks of cancer, often resulting from defects in DNA repair pathways. While normal cells rely on highly coordinated DNA damage response (DDR) mechanisms to maintain genomic integrity, cancer cells exploit aberrant DDR regulation to sustain uncontrolled proliferation and survival. Despite significant advancements in chemotherapy, targeted therapy, and immunotherapy, the emergence of resistance remains a major challenge in cancer treatment. Small molecule inhibitors targeting key DDR proteins have emerged as promising therapeutic agents, not only as direct anticancer drugs but also as indispensable tools for dissecting the molecular intricacies of DNA repair. Recent therapeutic approaches leverage synthetic lethality and DDR pathway vulnerabilities to selectively eradicate tumor cells while minimizing damage to normal tissues. These inhibitors provide insights into mechanisms of tumor resistance, facilitating the rational design of combination therapies to enhance treatment efficacy. This review examines the latest advancements in DNA repair-targeted therapeutics, with a focus on small molecule inhibitors currently under clinical investigation. Additionally, we discuss emerging strategies for optimizing DDR-targeted interventions, including biomarker-driven patient selection and rational drug combinations. Understanding these molecular interactions will contribute to the development of novel, more effective treatment paradigms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62086e2b8dc746fe8a2545d0353e820f59f91c95" target='_blank'>
              Targeting DNA repair mechanisms in cancer therapy: the role of small molecule DNA repair inhibitors
              </a>
            </td>
          <td>
            Seula Jeong, Y. Chung, Soomin Heo, Kyungjae Myung
          </td>
          <td>2025-11-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa4f5ddeab086fa66a7de2d9fed3fa9999b778c" target='_blank'>
              Cryptic multicellularity in wild yeast brings selective advantages under stress
              </a>
            </td>
          <td>
            W. K. Sexton, K. Schmidt, Q. Dickinson, J. Childress, F. Rosenzweig, E. Kroll
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The E. coli genome is encoded on a contiguous ~4.6 Mb-long DNA molecule, compacted inside a micron-cubed cell. When reconstituted in vitro, chromosomes expand in a bulk that is challenging for probing single-chromosome DNA transactions and conformational changes. Here, we report transplanting E. coli chromosomes into 2D semi-open microfluidic compartments, enabling exchange of conditions, stretching by electric field, mapping DNA-bound proteins, and cell-free transcription-translation at steady-state. We find transplanted chromosomes emerge as intact, compacted, blob-like structures, decorated with native proteins from the donor cell. The blobs include clusters of condensin proteins and exclude sparse bright ribosome foci, whereas RNA polymerases uniformly decorate the chromosome. Introducing a transcription-translation system, we measure genome-average transcription rates and image the birth of individual proteins from a reporter gene on the chromosome. Our data suggest a dilute regime without translational amplification or multiple synthesis events per gene. The removal of native proteins reveals a conformation transition from expanded to compacted state upon increased molecular crowding. Interestingly, transcription has a swelling effect, pushing the compaction transition to higher crowding levels. Our work opens a window into genome-scale DNA transactions outside a cell and helps tackle the bottom-up assembly of autonomous artificial cells. Chromosomes are long DNA polymers, compacted in cells and fragile once released from the host. Here, authors report on a method to gently transplant bacterial chromosomes into 2D microfluidic compartments, enabling analysis of their conformations and function within a native transcription-translation system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e9cf05890236fa52fe259a47a2db9febf64295" target='_blank'>
              Bacterial chromosome conformation and cell-free gene expression in synthetic 2D compartments
              </a>
            </td>
          <td>
            Ferdinand Greiss, S. Daube, V. Noireaux, R. Bar-Ziv
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a malignant neoplasm of plasma cells leading to bone destruction and marrow failure. Prognosis and management of MM rely on cytogenetic determination of copy number alterations (CNAs). Nevertheless, karyotype analysis difficult due to the presence of few plasma cells and their low proliferative activity. A shallow whole-genome sequencing (sWGS) technology named LeukoPrint was used to detect genome-wide CNAs in MM patients (n = 128), which can cover the entire genome without involving cell culture. Compared with karyotyping and fluorescent in situ hybridization (FISH), LeukoPrint demonstrated a significantly higher detection rate of copy number alterations (CNAs), increasing from 8.0% (karyotyping) and 44.2% (FISH) to 75.0% (n = 96), provided new CNA information and redefined the prognostic stratification in 20.3% of patients according to mSMART guidelines. Hyperdiploidy was the most common CNA feature (39.6%) in this cohort. A high concordance of 90.7% was observed in CNA between matched bone marrow and peripheral blood samples. LeukoPrint can be regarded as an automated, convenient and cost-effective approach to describe genomic CNA profiles. With the advantage of detecting CNAs of short segments and incorporating routine diagnostic methods, LeukoPrint can add value to conventional karyotyping with improved prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368c2b78c175919af87d99afc97d9ec591c9a64f" target='_blank'>
              Identification of Copy Number Alterations From Shallow Whole-Genome Sequencing in Multiple Myeloma.
              </a>
            </td>
          <td>
            Jiali Li, Shaobing Gao, Zhenling Li, Yuexin Cheng, J. Rao, X. Xiang, Yunjng Zeng, Xi Zhang, Li Gao
          </td>
          <td>2025-10-23</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The human Y chromosome (ChrY), which confers male sex determination, contains a relatively small number of protein-coding genes compared to other chromosomes; consequently, its functional impact on adult physiology was once severely unappreciated. While the repetitive structure of the ChrY once impeded sequencing, technological advances have now made it possible to identify its contents. Despite the historical view of ChrY as a virtual wasteland, we now know that it encodes a variety of genes which are hugely consequential to both human health and disease. The extreme downregulation of ChrY gene expression, resulting from partial or total loss of ChrY (LOY), is a common characteristic observed in various disease states in men, including cardiovascular, neurodegenerative, immunological health issues, and ,most notably, cancer. Additionally, mosaic LOY (mLOY) is sometimes found in primary cancerous tissues and is associated with poorer clinical outcome. Although, the reasons for these associations were once elusive, they are now understood to be linked to the activity of several ChrY genes, as well as the pleiotropic effects of their loss. In this review, we critically analyze contemporary and historic scientific literature which evaluate the clinical LOY trends seen in male exclusive/predominant cancers as well as explore the now identified mechanisms of ChrY alteration in cancer initiation, progression, and metastasis. Moreover, we discuss recent research studies which have uncovered novel mechanisms through which LOY may induce the physiological and molecular changes in the tumor microenvironment (TME) associated with malignant transformation and the evasion of innate immunity. Interestingly, the TME formed by malignant cells with LOY appears to contribute to early T cell exhaustion in infiltrating immune cells and consequent compromised tumor clearance; a phenomenon which has been profusely observed in patient samples. Furthermore, we describe the tumor-suppressive activities of the ChrY demonstrated in previous studies, as well as its newly identified roles in cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0160059eec1c9f3ab978cc420d02d0cdd3e593" target='_blank'>
              Y-chromosome alteration and its impact on cancer progression and metastasis
              </a>
            </td>
          <td>
            S. A. King, Merana Jahan, Prathiksha Prabhakaraalva, Nabila Zaman, Shipra Chaudhary, Natasha Kyprianou, Ashutosh K. Tewari, Goutam Chakraborty
          </td>
          <td>2025-11-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The O6-methylguanine-DNA methyltransferase (MGMT) plays a significant role in the pathogenesis and progression of glioma. Numerous enhancer variants, including those within the MGMT gene region and adjacent gene regions, have been found to be associated with cancer development and progression. We investigated the significance of enhancer variants located in the intergenic spacer far from the MGMT gene in relation to glioma susceptibility and progression. We recruited 402 glioma patients and 654 controls for this investigation using Sequenom MassARRAY genotyping. We identified a significantly elevated risk of glioma among carriers with the rs11016629 TG genotype compared to those with the GG genotype (OR = 1.41, 95% CI 1.03–1.93; P = 0.034). Subgroup analyses revealed that rs11016629 was significantly associated with glioma risk in subjects with WHO grade IV tumor (OR = 1.59, 95% CI 1.07–2.38; P = 0.023) and high-grade glioma (OR = 1.57, 95% CI 1.11–2.21; P = 0.011). Patients who underwent gross total resection with TG/TT genotypes exhibited a 2.66-fold higher risk of disease progression than GG carriers (HR = 2.66, 95% CI 1.23–5.79; P = 0.014). The study demonstrates that a MGMT enhancer variant rs11016629 contributes to both glioma susceptibility and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ffd3767daacb020b869e9275c5c5550f74b90c" target='_blank'>
              A MGMT Enhancer Variant is Associated with Glioma Susceptibility and Progression
              </a>
            </td>
          <td>
            Shu Zhang, Yiji Chen, Wenshen Xu, Danfang Yan, Meiqin Chen, Jiayi Zheng, Liming Huang
          </td>
          <td>2025-10-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a83ebfb77b99a082f33dfac7fdc4e834369069a" target='_blank'>
              Gastric epithelium from BRCA1 and BRCA2 carriers harbors increased double-stranded DNA damage and augmented growth
              </a>
            </td>
          <td>
            Kole H. Buckley, Michaela Dungan, Kevin Dinh, Gregory M. Kelly, Ryan Hausler, Kate E. Bennett, Daniel Clay, Julia E. Youngman, Ariana D. Majer, Keely Beyries, Blake A. Niccum, Sydney M. Shaffer, Tatiana A. Karakasheva, Kathryn E. Hamilton, M. Kochman, Gregory G Ginsberg, N. Ahmad, Kara N Maxwell, Bryson W. Katona
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Four decades ago, the discovery of centromere protein-A (CENP-A) marked a pivotal breakthrough in chromosome biology, revealing the epigenetic foundation of centromere identity. CENP-A, a histone H3 variant, directs the formation of the microtubule-binding kinetochore complex, designating the chromosomal site for its assembly and underpins the accurate partitioning of genetic material during cell division. Errors in cell division can give rise to DNA instability and aneuploidy, implicated in human diseases such as cancer. Therefore, discovering the underlying pathways and mechanisms responsible for the formation, regulation and maintenance of the centromere is important to our understanding of genome stability, epigenetic inheritance, and in providing the knowledge to help generate possible treatments and therapeutics. Here, we review various molecular pathways and mechanisms implicated in maintaining centromere identity and highlight some of the key outstanding questions with a focus on the human centromere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481df10bbcc98403846b3169d8fe656f17c65c42" target='_blank'>
              Preserving centromere identity: right amounts of CENP-A at the right place and time
              </a>
            </td>
          <td>
            Zofia Pukało, Bethan Medina-Pritchard, M. A. Abad, A. A. Jeyaprakash
          </td>
          <td>2025-09-30</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f7c1744fc2f1ed50403a7b109097213e18ec402" target='_blank'>
              Design and Development of DNA Damage Chemical Inducers of Proximity (DD-CIP) for Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Tian Qiu, Yeuan Ting Lee, Brendan G. Dwyer, Yi Jer Tan, Ting Chen, Bryan A. Romero, Yanlan Wang, Jiehui Deng, Tinghu Zhang, G. R. Crabtree, Stephen M. Hinshaw, Kwok-Kin Wong, N. Gray
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9a242ccc6ebd9d79ab2d3fe8d1c436d8e52055" target='_blank'>
              Enabling whole-genome DNA methylation-based classification of central nervous system tumors
              </a>
            </td>
          <td>
            Jonghoon Lee, Young Seok Ju, B.-L. Oh, S. Choi, Se Hoon Kim, Ji Won Lee, J. Phi, Y. Jung
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gastric cancer (GC) metastasis remains a major cause of poor prognosis, yet its molecular drivers are poorly understood. Here, we integrated single-cell RNA sequencing (scRNA-seq) of primary tumors and matched metastatic lymph nodes from six GC patients to identify a metastatic epithelial subpopulation characterized by EGR4 overexpression. Kaplan-Meier analysis revealed that high EGR4 expression correlated with reduced survival in GC patients. Mechanistically, chromatin immunoprecipitation sequencing (ChIP-seq) and luciferase assays demonstrated that EGR4 directly bound to the GDF15 promoter, driving its transcriptional activation. Functional studies showed that EGR4 promoted migration and metastasis via GDF15-mediated ErbB3/ErbB1 hetero-dimerization, which activated PI3K/AKT and MAPK/ERK pathways. Furthermore, CellChat analysis identified robust interactions between EGR4+ GC cells and cancer-associated fibroblasts (CAFs), particularly extracellular matrix (ECM)-remodeling eCAFs. Secreted GDF15 induced CAF activation through TGF-β receptor signaling, creating a pro-metastatic niche. Collectively, our study establishes the EGR4/GDF15 axis as a critical driver of GC metastasis, offering possible therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6e5035ca4934c776796dfb40ae16c3c3f2ef43" target='_blank'>
              EGR4 transcriptionally upregulates GDF15 to promote gastric cancer metastasis
              </a>
            </td>
          <td>
            Weiwei Liu, Yanyan Li, Lixin Liang, Lisheng Zheng, Rui Zeng, Congcong Zhang, Zhihao Lin, Wanying Feng, Qingling Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-10</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="We have investigated changes in chromosome conformation, nuclear organization, and transcription during differentiation and maturation of control and mutant motor neurons harboring hexanucleotide expansions in the C9orf72 gene that cause amyotrophic lateral sclerosis (ALS). Using an in vitro reprogramming, differentiation and neural maturation protocol, we obtained highly purified populations of post-mitotic motor neurons for both normal and diseased cells. As expected, as fibroblasts are reprogrammed into iPSCs, and as iPSCs differentiate into motor neurons, chromatin accessibility, chromosome conformation, and nuclear organization change along with large-scale alterations in transcriptional profiles. We find that the transcriptome changes extensively during the first three weeks of post-mitotic neuronal maturation, with thousands of genes changing expression, but then is relatively stable for the next three weeks. In contrast, chromosome conformation and nuclear organization continue to change over the entire 6-week maturation period: chromosome territoriality increases, long-range interactions along chromosomes decrease, compartmentalization strength increases, and centromeres and telomeres increasingly cluster. In motor neurons derived from ALS patients such changes in chromosome conformation were much reduced. Chromatin accessibility changes also showed delayed maturation. The transcriptome in these cells matured relatively normally but with notable changes in expression of genes involved in lipid, sterol and mitochondrial function. We conclude that neural maturation is associated with large scale post-mitotic changes in gene expression, chromosome conformation and nuclear organization, and that these processes are defective in motor neurons derived from ALS patients carrying C9orf72 hexanucleotide repeat expansions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ebb664b6f1b21244e0cfd38a3721d45d012b3f" target='_blank'>
              Dynamic changes in chromosome and nuclear architecture during maturation of normal and ALS C9orf72 motor neurons
              </a>
            </td>
          <td>
            Özgün Uyan, Snehal Sambare, Marlies E. Oomen, Nicholas Wightman, A. Schooley, Joseph R. Klim, Houda Belaghzal, Özkan Aydemir, Betul Akgol-Oksuz, Zeynep Sena Agim Uslu, Kevin Eggan, Robert H. Brown, Job Dekker
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Trypanosomatid parasites and the diseases they cause affect more than 30 million people annually worldwide. To develop treatments for these diseases, it is critical to understand how trypanosomatid biology protects the parasite, so that these mechanisms may be exploited as drug targets. An important aspect of trypanosomatid survival is protection from oxidative damage inflicted by the host. Reactive oxygen species produced by the host can damage nuclear DNA and kinetoplast, the mitochondrion DNA. DNA damage must be repaired or bypassed for the trypanosomatid to continue to replicate its genome. Trypanosomatids also possess specialized redox pathways that neutralize reactive oxygen species (ROS) from host-derived attacks and endogenous mitochondrion metabolism. This Review Article focuses on how trypanosomatids employ microhomology-mediated end-joining to repair DNA double-strand breaks and translesion DNA synthesis to bypass oxidatively damaged bases in nuclear and kinetoplast DNA. While the deleterious effects of ROS must be managed to protect the parasite’s genome, the redox status generated by oxidative assault is crucial for intracellular signaling, DNA synthesis, and kinetoplast homeostasis. This Review will also comment on the effectiveness of current treatments for trypanosomatid-caused diseases and the role of oxidative damage in trypanosomatid diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/280cba51b176c0cec98145ce4753c330f4d4fd5a" target='_blank'>
              Translesion synthesis and microhomology-mediated end-joining repair in trypanosomatids
              </a>
            </td>
          <td>
            Lea Drogalis Beckham, Anne K. Snyder, Sylvie Doublié, Bruno Martorelli di Genova
          </td>
          <td>2025-10-01</td>
          <td>PLOS Neglected Tropical Diseases</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a2eaeac8d39964b8ce29ff957548c6f095cc6b" target='_blank'>
              Beckwith-Wiedemann syndrome multiomic analysis of hepatoblastoma uncovers unique tumor heterogeneity and cellular landscapes, including transition cells leading to tumor formation
              </a>
            </td>
          <td>
            Snehal Nirgude, Elisia D Tichy, Yuanchao Zhang, Rose D Pradieu, Michael Xie, Kathrin M. Bernt, Suzanne Macfarland, J. Kalish
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 IDH-mutant gliomas include astrocytomas and oligodendrogliomas (ODG) that share a common glial precursor and IDH1/2 gene mutation. However, those entities display markedly distinct biological behaviors and clinical outcomes. Whereas ODG display relative genomic stability and are associated with favorable prognosis, astrocytomas often progress to highly aggressive forms characterized by extensive genomic rearrangements. Chromothripsis (CT), defined as a catastrophic one-off shattering and chaotic repair of one to two chromosomes, may drive abrupt genomic remodeling. Hypotheses regarding CT origin include mitotic errors such as chromosome entrapment in micronuclei, nuclear envelope rupture or chromosomal bridges. The incidence and mechanisms of CT events in IDH-mutant gliomas remain largely unexplored. To address those issues, we analyzed 235 IDH-mutant gliomas (143 astrocytomas, 92 ODG). We identified CT events using high-resolution SNP arrays and CTLPScanner algorithm, with cross-validation by visual inspection. Nuclear instability was investigated by confocal immunofluorescence microscopy on formalin-fixed paraffin-embedded sections, employing DAPI for nuclei/micronuclei, anti-lamin A/C and B1 antibodies to assess nuclear envelope (NE) integrity, and anti-BAF antibodies to show nuclear rupture or repair. Imaging employed Z-stack acquisition on a Leica TCS SP8 and analysis with FIJI. CT was observed exclusively in high-grade (WHO grade 3/4) astrocytomas (11/142, 7.8%; 7 grade 3 tumors and 4 grade 4 tumors), with higher frequency in higher-grade tumors (15% of grade 4 vs 7.6% of grade 3 astrocytomas). CT was not detected in either ODG or grade 2 astrocytomas. CT-positive tumors harbored more micronuclei and more frequent NE disruption, evidenced by anti-lamin A/C and anti-BAF immunopositivity. Thus, high-grade IDH-mutant astrocytomas were characterized by CT events and NE instability compared to lower-grade astrocytomas and ODG. Those phenomena may underlie the less favorable prognosis of astrocytomas compared to ODG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea57db6cc8ec3554e504ba2ce8b5c47bb2e0c7c2" target='_blank'>
              DNAR-12. High-grade IDH-mutant astrocytomas are characterized by chromothripsis and nuclear envelope instability compared to IDH-mutant oligodendrogliomas
              </a>
            </td>
          <td>
            Alix Fontaine, ëtitia La Basset, Joris Argentin, Maël Bouillon, E. Garcion, Audrey Rousseau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic instability may contribute to the occurrence and progression of malignant tumors of the female reproductive system. Homologous recombination repair (HRR) is vital in maintaining cellular genomic stability. RAD51‐associated protein 1 (RAD51AP1) plays a vital role in HRR, mainly participating in the formation of displacement loop (D‐loop), and is an important molecule for maintaining cellular genomic stability. Recent studies showed that RAD51AP1 was significantly overexpressed in a variety of cancer types and correlated with prognosis, suggesting that it may have a significant pro‐carcinogenic effect. However, the mechanism underlying its pro‐carcinogenic effect remains unclear, which may be closely associated with cancer stemness. Meanwhile, RAD51AP1 also plays an important role in resistance to radiotherapy and chemotherapy. Exploring RAD51AP1 and its regulatory molecules may provide new targets for overcoming cancer progression and treatment resistance. Here, we reviewed the latest research on RAD51AP1 in female reproductive system tumors and summarized its differential expression and prognostic implications. In this review, we also outlined the potential mechanisms of its procancer and drug resistance‐promoting effects to provide several potential directions for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef51ed6c508733d87385cf988511122d0230631" target='_blank'>
              Research Progress of RAD51AP1 in Malignant Tumors of the Female Reproductive System
              </a>
            </td>
          <td>
            Chengguo Zhang, Xiaoyang Liu, Zizhang Li, Jiayao Han, Jun Liang, Nan Zhou
          </td>
          <td>2025-10-19</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Alternative lengthening of telomeres (ALT) is a homologous recombination-based mechanism of telomere maintenance that promotes replicative immortality. ALT is common in high-grade gliomas (HGGs) and is associated with elevated baseline DNA damage. Despite this, ALT+ HGGs remain immunologically cold, limiting the effectiveness of current therapies. Topoisomerase I (TOP1) inhibition has been explored as a strategy to induce lethal DNA damage in gliomas. However, clinical trials using the TOP1 inhibitor irinotecan failed to improve survival in glioblastoma, partly due to poor blood–brain barrier penetration and lack of molecularly-guided patient selection. Newer TOP1 inhibitors with improved pharmacologic profiles and updated biomarker-guided tumor stratification offer renewed potential. We found ALT+ glioma cells, relative to telomerase+, are specifically vulnerable to either TOP1i or depletion of SMARCAL1, an annealing helicase involved in replication fork reversal. Replication fork reversal is a primary mechanism of the DNA damage response to TOP1 trapping; therefore, we hypothesized these strategies would synergize to amplify DNA damage and enhance immunogenicity in ALT+ HGGs. We utilized doxycycline inducible shRNAs to deplete SMARCAL1 over multiple cell cycles in ALT+ and telomerase+ HGG cell lines. In patient-derived ALT+ cell lines, SMARCAL1 depletion led to increased DNA double-strand breaks and ALT-associated PML bodies (APBs), as shown by immunofluorescence and FISH. Flow cytometry revealed that this damage includes excessive RPA binding and replication catastrophe preferentially in ALT+ cells. Combining SMARCAL1 depletion with TOP1 inhibition produced a synergistic increase in DNA damage. Importantly, SMARCAL1 loss also upregulated inflammatory signaling and immune receptor expression in ALT+ cells, enhancing recognition by phagocytes. These findings establish SMARCAL1 as a critical regulator of DNA repair and immune suppression in ALT+ HGGs. Its inhibition presents a promising therapeutic approach, either alone or in combination with TOP1 inhibitors, to increase tumor cell death and boost anti-tumor immunity in this challenging glioma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1bcd648526b6f6d565df535b79bc10e16bb1aa6" target='_blank'>
              DNAR-07. SMARCAL1 inhibition synergizes with TOP1 inhibitors to induce replication catastrophe and inflammation in ALT+ gliomas
              </a>
            </td>
          <td>
            Elise N Erman, Emiley A Gibson, Laura Strickland, Alexandrea Brown, Steve Keir, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Diverse fusions of EML4 and ALK are oncogenic drivers in lung adenocarcinomas. EML4-ALK variants have distinct breakpoints within EML4, but their functional differences remain poorly understood. Here, we use somatic genome editing to generate autochthonous mouse models of EML4-ALK-driven lung tumors and show that V3 is more oncogenic than V1. By employing multiplexed genome editing and quantifying the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth, we show that many tumor suppressor genes have variant-specific effects on tumorigenesis. Pharmacogenomic analyses further suggest that tumor genotype can influence therapeutic responses. Analysis of human EML4-ALK-positive lung cancers also identified variant-specific differences in their genomic landscapes. These findings suggest that EML4-ALK variants behave more like distinct oncogenes rather than a uniform entity and highlight the dramatic impact of oncogenic fusion partner proteins and coincident tumor suppressor gene alterations on the biology of oncogenic fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8a78e3c9e4fd68a0441eaf2f2edc5f59a5afae" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis.
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Karen Y. Shih, Óscar González-Velasco, Mulham Najajreh, Stewart Kim, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, Ilse Hofmann, S. Sivakumar, S. Sisoudiya, Ethan S Sokol, Hongchen Cai, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2025-09-23</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554ff6de17aca3fc14055b3c5a6203467aaf1fe3" target='_blank'>
              A gap-free telomere-to-telomere pig reference genome provides insight in centromere evolution and breed-specific selection
              </a>
            </td>
          <td>
            Dong Li, Yulong Wang, Tiantian Yuan, Xiang Li, Yu Quan, Wansheng Liu, K. Schachtschneider, Lawrence B Schook, R. Crooijmans, M. Groenen, M. Derks, Taiyong Yu
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0598f4e042a38ac90e8698c02adae86ab3960654" target='_blank'>
              KCTD10 is a sensor for co-directional transcription-replication conflicts.
              </a>
            </td>
          <td>
            Jake A. Kloeber, Bin Chen, Guangchao Sun, Charles S King, Zhiquan Wang, Li Wang, Zheming Wu, Shouhai Zhu, Fei Zhao, Hongran Qin, Yaobin Ouyang, Huaping Xiao, X. Tu, Jing Lu, Yanxia Jiang, K. Luo, Ping Yin, Xinyan Wu, R. Mutter, Jinzhou Huang, Zhenkun Lou
          </td>
          <td>2025-10-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Histone modifications are central to regulating gene expression, chromatin organization, and DNA repair. Tri-methylation of histone H3 on lysine 36 (H3K36me3), catalyzed exclusively by the methyltransferase SETD2, is critical for the proper execution of homologous recombination and DNA damage repair processes. In KRAS-mutant colorectal cancers, radioresistance represents a significant therapeutic hurdle. Our preliminary studies indicate that oncogenic KRAS upregulates SETD2 expression, suggesting that the KRAS-SETD2-H3K36me3 axis might contribute to the DNA repair proficiency underlying radiation resistance.



 We propose that inhibition of SETD2 will disrupt H3K36me3-dependent recruitment of DNA repair complexes, thereby sensitizing KRAS-mutant colorectal cancer cells to ionizing radiation.



 To test our hypothesis, we performed comprehensive in vitro studies using three established human KRAS-mutant colorectal cancer cell lines. Genetic inhibition was achieved via RNA interference (RNAi) targeting SETD2, and pharmacologic inhibition was conducted using EZM0414, a selective, orally bioavailable SETD2 inhibitor in preclinical development. Following treatment with either RNAi or EZM0414, cells were exposed to ionizing radiation. Radiosensitivity was quantified using clonogenic assays, while the extent of DNA damage was assessed through immunofluorescence detection of γH2AX foci. Additionally, western blot analyses were performed to evaluate the expression profiles of key DNA damage response proteins.



 Both genetic and pharmacologic abrogation of SETD2 resulted in a marked enhancement of radiation-induced DNA damage compared to radiation treatment alone. A significant increase in γH2AX foci was observed in cells treated with EZM0414 in combination with ionizing radiation, indicating an elevated accumulation of double-strand breaks. Furthermore, clonogenic assays revealed that inhibition of SETD2 substantially impaired cell survival following radiation exposure. These data collectively support the contention that SETD2-mediated H3K36me3 serves a protective role in the DNA damage response, thereby promoting radioresistance in KRAS-mutant colorectal cancer cells.



 Our findings establish a functional link between oncogenic KRAS signaling, SETD2 activity, and DNA repair competence in colorectal cancer, highlighting the SETD2-H3K36me3 axis as a key mediator of radiation resistance. By targeting SETD2, our study provides compelling preclinical evidence for a novel radiosensitization strategy. These results have immediate translational relevance and underscore the potential for integrating SETD2 inhibitors into therapeutic regimens to overcome radioresistance, ultimately improving outcomes in patients with treatment-refractory KRAS-mutant colorectal cancer.



 Lokesh Akana, Saaimatul Huq, Henrique Rodrigues, Adam R. Wolfe. Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27fb3c4df665fd773319419f1a14be068502495" target='_blank'>
              Abstract A033: Targeting SETD2 in Combination with Radiotherapy in Rectal Cancer
              </a>
            </td>
          <td>
            L.K. Akana, Saaimatul Huq, H. Rodrigues, A. Wolfe
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="FET (FUS, EWSR1, and TAF15) fusion oncoproteins are characteristic for several sarcomas and leukemias, including myxoid liposarcoma and Ewing sarcoma. FET oncoproteins interact with the SWI/SNF chromatin remodeling complex subtypes cBAF, PBAF, and GBAF, but their impact on SWI/SNF compositions, interactions, and downstream epigenetic effects remains elusive. We employ a comprehensive immunoprecipitation and quantitative mass spectrometry approach to determine the impact of FET oncoproteins on SWI/SNF composition and their interactomes. Validation of complex composition and interaction partners is performed by glycerol gradient sedimentation assays and co-immunofluorescence analysis. Furthermore, we determine the differential chromatin accessibility and gene regulation in FET sarcomas using assay for transposase-accessible chromatin sequencing and RNA sequencing, respectively. Our data show that FET sarcomas have distinct SWI/SNF complex compositions, with different subunit paralogs and subtype-specific components that utilize distinct sets of interaction partners, including specific transcription factors. We show that FET oncoproteins cause no major disruption of the SWI/SNF complex composition. Instead, FUS::DDIT3-bound SWI/SNF complexes in myxoid liposarcoma cells are enriched in PBAF and GBAF components as well as most interaction partners. These data suggest that FET oncoproteins act together with fully assembled and functional SWI/SNF complexes and recruited interaction partners. Finally, our data reveal that the SWI/SNF compositions, interactomes, and epigenetic background contribute to the tumor type in FET sarcoma. Trial registration Clinical trial number: not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7d390ca6f97d7ab986b2803f84e41f3c7c0db63" target='_blank'>
              FET fusion oncoproteins enrich SWI/SNF complex subtypes and interaction partners
              </a>
            </td>
          <td>
            M. Lindén, Lisa Andersson, Heba Albatrok, Vilma Canfjorden, Emma Jonasson, Kajsa Grönqvist, Daniel Sjövall, Pekka Jaako, Rossella Crescitelli, Henrik Fagman, Pierre Åman, Anders Ståhlberg
          </td>
          <td>2025-09-23</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 SETD2 is a tumor suppressor gene responsible for trimethylation of H3K36, crucial for DNA mismatch repair and homologous recombination. Novel synthetic lethality agents, ATR inhibitors (ATRi), have demonstrated anti-tumor activity in preclinical models and early clinical trials for pan-tumors. SETD2 and H3K36 deficient cells have been demonstrated to display Micro Satellite Instability (MSI) with increased tumor mutational burden (TMB). Deleterious SETD2 alterations occur in 5-10% of brain tumor patients with glioblastoma, IDH wild type, WHO grade 4, (GBM). These patients have a poor prognosis. This study aims to evaluate targeted treatment strategies in central nervous system (CNS) tumors, by investigating the potential of ATRi for tumors harboring deleterious SETD2 alterations and immuno-therapy for tumors with increased mutational burden, with the objective of improving patient outcomes.



 This study analyzes genomic profiles from 434 patients diagnosed with central nervous system (CNS) tumors between January 1st, 2019, and February 28th, 2025, enrolled in the prospective Neurogenome and CoPPO studies. Rigshospitalet has one of the largest single-institution datasets with comprehensive data on SETD2 alterations derived from whole exome/genome sequencing including structural variants, copy number variations, and RNA expression.



 The study cohort included 434 patients with CNS tumors (166 female, 268 male) with a median age at diagnosis of 59 years (interquartile range: 49-66 years). GBM was the predominant tumor type (86%), followed by astrocytoma, IDH mutated (mut), WHO grade 2-4 (12%) and oligodendroglioma, IDH-mut, 1p/19q co-deletion, WHO-grade 2-3 (1.2%). Other CNS tumors were diagnosed in 6 (1.4%) patients. Deleterious SETD2 alterations were found in 13 (3%) patients: 11 with GBM, 1 astrocytoma IDH mut, and 1 oligodendroglioma. Twelve patients had mutations: 6 premature stop codons, 5 frameshift mutations and 1 splice site variant, and one case with biallelic SETD2 loss. Data on tumor mutational burden (TMB) was assessable for 12 SETD2-deficient patients, none exceeding 10mut/mb; mean TMB was 3.3 mut/mb vs. 6.2 mut/mb in the SETD2-wildtype (wt) group. No SETD2-deficient patients had received temozolomide (TMZ) treatment prior to sampling. By contrast, some samples in the SETD2-wt cohort were obtained at relapse, which may have inflated TMB due to TMZ treatment. COSMIC signatures were available for 10 SETD2-deficient patients with none having indications of MSI based on COSMIC signature evaluation.



 Loss of SETD2 function in primary CNS-tumors is relatively rare and was predominantly driven by mutations, with copy number analysis revealing only 1 biallelic deletion in 434 patients. SETD2 alterations was primarily seen in GBM patients. The SETD2 deficient patients did not exhibit indications of MSI or high TMB. However, the impact of patient relapse samples with TMZ induced mutations in the SETD2-wt cohort is being explored further together with pan-tumor markers for DNA damage repair.



 Tine Østergaard, Christoffer L. Halvorsen, Dorte S. Nørøxe, Vincent N. Fougner, Iben Spanggaard, Kristoffer S. Rohrberg, Martin Højgaard. Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in SETD2 without associated increase in mutational burden [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5615e3697959292b283838db14154a56c4e82cf6" target='_blank'>
              Abstract C037: Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in
 SETD2
 without associated increase in mutational burden
              </a>
            </td>
          <td>
            Tine Østergaard, Christoffer L. Halvorsen, D. Nørøxe, V. Fougner, I. Spanggaard, K. S. Rohrberg, M. Højgaard
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b04efe93fb72cdef3529bf2ec79fd71db767d8a" target='_blank'>
              DNASE1L3 surveils mitochondrial DNA on the surface of distinct mammalian cells
              </a>
            </td>
          <td>
            Jennifer Porat, Valentina Poli, Karim Almahayni, Benson M. George, F. N. de Luna Vitorino, Li Yi, Carolina Bras Costa, Yitong Zhou, Andrew Crompton, Hongxiu Wen, Leon Zheng, Peter K. Gregersen, Suneet Agarwal, Benjamin A. Garcia, Yixuan Xie, Leonhard Möckl, Ivan Zanoni, K. Simpfendorfer, Ryan A. Flynn
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d6f72ebd8a3733c60e8ee75cfbcd9b5da5d9c6" target='_blank'>
              Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes
              </a>
            </td>
          <td>
            Zhe Liu, Yihang Bao, An Gu, Weichen Song, Guan Ning Lin
          </td>
          <td>2025-09-26</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Reverse transcriptases (RTs) have well-established roles in the replication and spread of retroviruses and retrotransposons. However, recent evidence suggests that RTs have been conscripted by cells for diverse roles in antiviral defence. Here we determine structures of a type I-A retron, which explain how RNA, DNA, RT, HNH-nuclease and four molecules of a structure maintenance of chromosome (SMC)-family ATPase assemble into a 364 kDa complex that provides phage defence. We show that phage-encoded nucleases trigger degradation of the retron-associated DNA, leading to activation of the HNH nuclease. The HNH nuclease cleaves tRNASer, stalling protein synthesis and arresting viral replication. Taken together, these data reveal diverse and paradoxical roles for RTs in the perpetuation and elimination of genetic parasites. In this study, the authors determine the structure of a Type I-A retron from E. coli FORC82 and reveal the functional interplay between Reverse Transcriptases (RTs) and Structural Maintenance of Chromosomes (SMC) ATPases.’">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78259f1a0c5b394b643034f4b28e1301ed4a03ce" target='_blank'>
              Structural basis of antiphage defence by an ATPase-associated reverse transcriptase
              </a>
            </td>
          <td>
            Jerrin Thomas George, Nathaniel Burman, R. Wilkinson, Senuri de Silva, Quynh McKelvey-Pham, M. Buyukyoruk, Adelaide Dale, Hannah Landman, A. Graham, Steven Z DeLuca, B. Wiedenheft
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b15ed383fa686f49d7de283de5e0eaf7d1cc0c" target='_blank'>
              A Rare Multipotent Peg-like Epithelial Cell is a Candidate Cell-of-Origin for High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Megan L. Ritting, Wenmei Yang, Syed Mohamed Musheer Aalam, Hui Zhao, Liang Feng, Jianning Song, Mihai G. Dumbrava, W. Ismail, Dong-Gi Mun, Chunling Hu, Opalina Roy, Keyur Chaludiya, Geng-Xian Shi, Daphne Norma Crasta, Kenneth Schaufelberger, Jeffrey R. Janus, Guruprasad Kalthur, S. Weroha, Scott H. Kaufmann, A. Sadanandam, David J. H. F. Knapp, Chen Wang, Akhilesh Pandey, Alexandre Gaspar-Maia, F. Couch, Mark E. Sherman, J. Bakkum-Gamez, Nagarajan Kannan
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>125</td>
        </tr>

        <tr id="Human RECQL4 and BLM helicases participate in all DNA dependent processes, including replication stress, DNA damage repair. Both helicases are overexpressed in glioblastoma (GBM), a lethal primary brain tumour, characterised by resistance to radio- and chemotherapy. BLM-depleted glioma cells exhibit senescence-associated or polypoid phenotype when exposed to temozolomide (TMZ) and olaparib (OLA), a PARP inhibitor. This study aims to investigate how RECQL4 depletion influences the response of malignant gliomas to chemotherapeutics. We investigated the effect of RECQL4 depletion in glioma cells on cell growth, apoptosis, senescence and polyploidy in the response to combined TMZ and OLA treatment. We compared transcriptomes of RECQL4- and BLM-depleted LN18 and LN229 glioma cells. Drug-induced cytotoxicity, senescence-associated phenotypes, cell cycle alterations, and polyploidy were assessed using the MTT metabolic assay, β-galactosidase activity assay, and propidium iodide staining. RECQL4 depletion modestly affected basal glioma cell viability and proliferation, similarly to knock out of the BLM protein. Deletion of RECQL4 in glioma cells (RQ4 KO) induced profound transcriptomic alterations, dissimilar to BLM depletion. RECQL4-depleted glioma cells treated with TMZ and OLA exhibited reduced viability and increased levels of apoptosis markers. The treatment induced cell cycle arrest, however, RQ4 KO cells did not show signs of senescence phenotype or polyploidisation, when compared to BLM KO glioma cells. Interestingly, both RQ4 KO and BLM KO cells were more resistant to WP744, a doxorubicin derivative, when compared to WT LN229 glioma cells. Our results highlight the distinct roles of RecQ helicases in a response to chemotherapeutics and support a rationale for targeting RECQL4 as a therapeutic strategy in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d631e6439cbcbbf960a1d54b75f9eb6885b3c7c" target='_blank'>
              Shared and non-overlapping functions of RECQL4 and BLM helicases in chemotherapeutics-induced glioma cell responses
              </a>
            </td>
          <td>
            K. Wojnicki, B. Wojtaś, I. Ciechomska, B. Kaza, Matthew Guille, Waldemar Priebe, B. Kaminska
          </td>
          <td>2025-09-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a global health challenge, with limited treatment options and a high mortality rate. We recently found that lung cancer cells that express more genes that are typically restricted to the germline (germ cell cancer genes, GC genes) repair DNA double-strand breaks more rapidly, show higher rates of proliferation, and are more resistant to ionizing radiation, compared to cells that express fewer GC genes. Moreover, we found that the gene encoding TRIP13 (thyroid hormone receptor interactor 13) plays a significant role in this malignant phenotype. Here, we demonstrate that melatonin (MT), a hormone synthesized in the pineal gland, downregulates the expression of TRIP13, particularly in lung cancer cells with high expression of TRIP13. Moreover, this downregulation of TRIP13 by MT further inhibits the DNA repair proteins RAD51 and XRCC5, thereby impairing DNA repair via homologous recombination and non-homologous end joining. We further found that the melatonin receptor 1B (MTNR1B), rather than melatonin receptor 1 A (MTNR1A), is essential for MT mediated TRIP13 downregulation. Because we also found that treatment with MT still decreases cell proliferation of TRIP13-KO cells, combining MT with the TRIP13 inhibitor DCZ0415 would likely have an additive anti-proliferative therapeutic effect in the treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e796861d07495168d94915331af08dc34126dd1" target='_blank'>
              Melatonin suppresses cancer cell proliferation, DNA repair and expression of the oncogene TRIP13
              </a>
            </td>
          <td>
            Wenqing Liu, Ans M. M. van Pelt, G. Hamer
          </td>
          <td>2025-10-27</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Jumping translocations (JTs) can lead to partial trisomies. A breakpoint within the gene known as Jumping Translocation Breakpoint (JTB) has previously been associated with JTs involving the long arm of human chromosome 1 (1q). These 1q+ amplifications are frequently observed in cancer. JTB was initially mapped to the Epidermal Differentiation Complex (EDC) at 1q21, and earlier studies primarily focused on its role in malignant or adult tissues. Using updated genomic data, we refined the mapping of JTB. We employed RNA in situ hybridization (RISH) to visualize Jtb expression with organ, tissue, and cell-level resolution. We demonstrate that human JTB and murine Jtb are located outside the EDC. In midgestational wild-type mouse embryos, Jtb is expressed in multiple tissues, including the developing heart and vertebral column, and shows partial overlap with the expression of early markers of the neural crest cell lineage. Our findings suggest that the oncogenic potential associated with human JTB translocations is likely unrelated to its previously assumed location within the EDC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed6323abd411d00d62b1282d50fbc327ee57f1b" target='_blank'>
              Jumping Translocation Breakpoint Expression in Midgestation Mouse Embryos
              </a>
            </td>
          <td>
            Carley McGrath, Taniya M Jayaweera, T. Lufkin, Costel C. Darie, A. Neagu, Petra Kraus
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Background/Objectives: DNA2 is a conserved nuclease–helicase that plays a crucial role in DNA replication and repair by responding to replication stress. Previous studies have established the role of DNA2 in Okazaki fragment processing, the recovery of stalled replication forks, and double-strand break repair. This study investigates the role of Drosophila melanogaster Dna2 in response to exogenous DNA damage and replication stress as well as during developmental stages involving intensive DNA replication. Methods: We used the Drosophila mutant alleles, Dna2D1 and Dna2D2, which differ in the presence of the helicase 1A domain, to assess sensitivity to mutagens that cause various types of replication stress and DNA damage. We examined reproductive fitness through Mendelian ratio calculations, fecundity, egg viability assays, and assessed DNA damage via immunostaining of ovarian germaria. Lifespan assays were also conducted to examine adult survival. Results: Dna2 mutants demonstrated significant sensitivity to replication stress induced by MMS, hydroxyurea, topotecan, and nitrogen mustard. Dna2lS/S1 mutants exhibited higher survival than Dna2lS/D2 upon exposure to topotecan and bleomycin, suggesting a possible helicase-specific role in damage response. Mutants exhibited decreased fecundity, reduced egg viability, and elevated DNA damage in mitotically active germline cells. Adult lifespan was not reduced in Dna2 mutants, implying potential compensatory stress-response mechanisms. Conclusions: Our findings support a requirement of Dna2 in managing replication stress during critical developmental phases in Drosophila. These insights clarify the nuanced contributions of the nuclease and helicase domains of DNA2, suggesting potential domain-specific functions in genomic stability and repair mechanisms. This work provides a foundation that will enable future researchers to further dissect the complex roles of DNA2 in replication and repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb07b9f9e3d5987899375139034bf1577286e9c" target='_blank'>
              Dna2 Responds to Endogenous and Exogenous Replication Stress in Drosophila melanogaster
              </a>
            </td>
          <td>
            Ivan Rivera, Sabah Shammari, Hamiya Sohail, Christian Villegas, Zoha Wasim, Sze Hang Ip, Vada Becker, Kathryn P. Kohl, Eric P Stoffregen, Christina I. Swanson, Elyse Bolterstein
          </td>
          <td>2025-09-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43450515e8b62fe85e717aebfac6a246b0159e06" target='_blank'>
              Mus Musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, Sang Hyun Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich DNA secondary structures with known regulatory roles in eukaryotes, but their functions in bacteria remain largely unexplored. Here, we systematically analyzed potential G-quadruplexes (PG4s) across 1566 bacterial genomes. PG4s showed non-random distributions, with strong enrichment in intergenic regions, particularly near gene start sites, suggesting potential roles in transcriptional regulation. Structural analysis revealed a higher proportion of short-loop G4s in these regions, indicating enhanced stability. Comparisons with randomized genomes confirmed that the observed patterns cannot be explained by GC content or genome size alone. Ecological analysis further revealed that bacteria associated with homothermic hosts exhibit significantly lower G4 densities than free-living species, suggesting reduced reliance on G4-mediated regulation in stable environments. Together, these findings highlight G4s as conserved yet context-dependent elements of bacterial genome organization and provide new insights into their potential regulatory roles in prokaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969b849342d5007389accc241718dd3c89890d08" target='_blank'>
              Non-Random Distribution of G-Quadruplex Structures Reveals Regulatory and Ecological Adaptations in Bacterial Genomes
              </a>
            </td>
          <td>
            Jiye Fu, Ke Xiao, Yukun He, Jing Tu
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies. This study introduces single-cell transcription factor (TF) sequencing, a single-cell barcoded and doxycycline-inducible TF overexpression approach that reveals dose-sensitive functional classes of TFs and cellular heterogeneity by mapping TF dose-dependent transcriptomic changes during the reprogramming of mouse embryonic multipotent stromal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793bb57e9ee8be0a292f9fe822d8b2b3a9781a8c" target='_blank'>
              Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq
              </a>
            </td>
          <td>
            Wangjie Liu, Wouter Saelens, Pernille Y Rainer, Marjan Biočanin, V. Gardeux, Antoni J. Gralak, Guido van Mierlo, Angelika Gebhart, Julie Russeil, Tingdang Liu, Wanze Chen, Bart Deplancke
          </td>
          <td>2025-10-01</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44901dfbd5176301295029fab9bb4b40320a082" target='_blank'>
              AAV Kills Dividing Cells by Depleting PARP1 and Other DNA Damage Response Proteins
              </a>
            </td>
          <td>
            Sasha Friese, Junjie Zai, Grace Luzbetak, Nyonika Khanna, Johanna Gesperger, Chien-Hung Liu, Wynand P. Roos, Rasha Q. Al-Rahahleh, Matisse Willardson, Feng Yang, Duoduo Fu, Yiming Han, Nadia Lintag, Jaiden Saykham, Joshua Le, Emeen Al-Delaimy, Nolan Soutipan, Ellen Duong, Jeremy Rich, Maria Carolina Marchetto, Michael Rosenfeld, Robert W. Sobol, Matthew Shtrahman
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 IDH-mutant adult-type diffuse gliomas show variable clinical outcomes influenced by patient age. Recent studies have linked HOX gene activity to age, survival, and tumor aggressiveness in both IDH-mutant astrocytoma (IDH-A) and oligodendroglioma (IDH-O), but the extent to which HOX activity defines shared biology across these entities remains unclear. To address this, we assembled a large DNA methylation dataset of IDH-O (N=1,445) and IDH-A (N=2,232), integrating published cohorts, TCGA, and an in-house NIH collection. Multi-omic analyses included DNA methylation (450K, EPIC, EPICv2), RNA-Seq, somatic copy number alterations (SCNA), mutation, pathology, and clinical features, enabling a comprehensive characterization of HOX-associated biology. Unsupervised clustering of HOX gene body methylation consistently identified two subgroups—HOX-High and HOX-Low—in both IDH-O and IDH-A across all platforms. HOX High tumors were associated with older age, higher tumor grade, increased Ki-67%, shorter survival, and higher SCNA and mutation burdens. Differential methylation analyses revealed that most probes distinguishing HOX High and HOX Low subgroups in IDH-O were also differentially methylated in IDH-A, with similar effect sizes, and vice versa. Notably, probes within the HOXD locus showed the greatest differences in both tumor types. RNA-Seq data revealed congruent differences in HOXD gene expression. Gene ontology analyses showed highly correlated enrichment scores between IDH-O and IDH-A, even after excluding HOX-related genes, suggesting broader shared biology. Importantly, HOX-associated subgroups were observed in both tumor types regardless of 1p/19q-codeletion status, indicating that this signature is independent of the canonical molecular subclassification. Our results support the existence of a pan-IDH-mutant glioma biological program, transcending traditional molecular subtypes. Identification of these HOX-associated subgroups provides new opportunities for biologically informed risk stratification and may inform the development of targeted therapies for IDH-mutant gliomas. Further prospective studies are warranted to explore the clinical utility of HOX-based biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8be8f07e09f14a3e3736db95a5e2b18ba6610668" target='_blank'>
              PATH-15. HOX-Associated Differences between IDH-mutant Astrocytoma and Oligodendroglioma are Independent of 1p/19q-Codeletion
              </a>
            </td>
          <td>
            Nicholas Nuechterlein, Vinny Ha, Drew W Pratt, P. Cimino
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Telomere length maintenance is strongly linked to cellular aging, as telomeres progressively shorten with each cell division. This phenomenon is well-documented in mitotic, or dividing, cells. However, neurons are post-mitotic and do not undergo mitosis, meaning they lack the classical mechanisms through which telomere shortening occurs. Despite this, neurons retain telomeres that protect chromosomal ends. The role of telomeres in neurons has gained interest, particularly in the context of neurodegenerative diseases such as amyotrophic lateral sclerosis (ALS), where aging is a major risk factor. This has sparked interest in investigating telomere maintenance mechanisms in post-mitotic neurons. Nevertheless, most existing telomere analysis techniques were developed for and optimized using mitotic cells, posing challenges for studying telomeres in non-dividing neuronal cells. Thus, this protocol adapts an already established technique, the combined immunofluorescence and telomere fluorescent in situ hybridization (IF-FISH) on mitotic cells to study the processes occurring at telomeres in cortical neurons of the mouse ALS transgenic model, TDP-43 rNLS. Specifically, it determines the occurrence of DNA damage and the alternative lengthening of telomeres (ALT) mechanism through simultaneous labeling of the DNA damage marker, γH2AX, or the ALT marker, promyelocytic leukemia (PML) protein, together with telomeres. Therefore, the protocol enables the visualization of DNA damage (γH2AX) or the ALT marker (PML) concurrently with telomeres. This technique can be successfully applied to brain tissue and enables the investigation of telomeres specifically in cortical neurons, rather than in bulk tissue, offering a significant advantage over Southern blot or qPCR-based techniques. Key features • This protocol enables the labeling of telomeres in mouse brain tissue prepared from paraffin-embedded brain sections. • This method facilitates concurrent labeling of proteins that are colocalized at telomere sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1e2280b8fd8c35a0ab81f6940db9809bdef4684" target='_blank'>
              Colocalizing Telomeres With PML or γH2AX Foci by IF-FISH in Mouse Brain Neurons
              </a>
            </td>
          <td>
            Anna Konopka
          </td>
          <td>2025-11-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Objective To explore the molecular genetic characteristics of childhood acute lymphoblastic leukemia (ALL) and their relationships with clinical phenotypes, chromosomal abnormalities, and prognosis, so as to provide references for precise diagnosis and treatment. Methods A total of 302 newly diagnosed children with ALL were included. Real-time fluorescent quantitative PCR, high-throughput sequencing and other technologies were used to detect common fusion genes, rare fusion genes and Ph-like ALL-related molecules. Combined with chromosomal karyotype analysis, immunophenotyping and minimal residual disease (MRD) monitoring, the associations between molecular genetic characteristics and clinical indicators as well as prognosis were analyzed. Results Among the 302 children, the total positive rate of leukemia genes was 50.66%, and the gene detection rate in B-ALL children (52.90%) was significantly higher than that in T-ALL (37.21%). Common fusion genes were mainly ETV6/RUNX1 (19.54%), MLL (9.27%) and BCR/ABL (6.29%), with age- and immune subtype-specific distributions. Children with ETV6/RUNX1 positivity had the highest complete remission rate (93.2%) and the best 2-year event-free survival rate (89.8%), while those with BCR/ABL positivity had the worst prognosis (complete remission rate 57.9%, 2-year event-free survival rate 42.1%). There were differences in the consistency between fusion genes and chromosomal abnormalities: ETV6/RUNX1 and BCR/ABL showed 100% consistency with specific chromosomal translocations, while E2A/PBX1 and MLL showed about 50% consistency. The dynamic changes of MRD were closely related to gene types, with significantly higher MRD positive rates in children with high-risk genes. Conclusion The molecular genetic characteristics of childhood ALL have clear clinical significance. Fusion gene detection can be used for disease classification, risk stratification and prognosis evaluation, providing an important basis for the formulation of individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/735c362fca7d080732163ea6bc8f870abd0cd0da" target='_blank'>
              Molecular genetic characteristics and clinical significance of childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenzhen Zhu, Junchen Yan
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Pediatrics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce78a1432bf55161afa66b6f861a74035f16861" target='_blank'>
              Low Dose of Lead Causes Activation of Ribosomal Cistrons 15 Chromatids Associated with Malignant Growth
              </a>
            </td>
          <td>
            T. Lezhava, T. Buadze, T. Jokhadze, M. Gaiozishvili, T. Sigua, Khatia Nadiradze, Shorena Sharia
          </td>
          <td>2025-09-21</td>
          <td>Cytology and Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="SUMMARY DNA repair is crucial for genome stability, in particular for plants which are exposed to high levels of damage arising from UV irradiation, soil pollutants and reactive oxygen species. Damage that affects both strands of the DNA duplex is harder to repair due to both the lack of a template strand and the potential for physical separation of fragmented chromosomes. As such, DNA double‐strand breaks (DSBs) and interstrand DNA crosslinks (ICL) are particularly cytotoxic forms of damage. Here we report the functions of FANCONI ANAEMIA I (FANCI), an Arabidopsis thaliana homologue of the mammalian ICL repair protein. We show that in plant cells, as in mammals, FANCI forms a nuclear localised complex with FANCD2. Genetic analysis of plants lacking FANCI displays significant hypersensitivity to the DNA crosslinking reagent mitomycin C. Furthermore, mutation of FANCI in combination with mutations in a second ICL repair factor, METHYL METHANESULFONATE AND UV‐SENSITIVE PROTEIN 81 (MUS81), results in increased levels of programmed cell death compared to the corresponding single mutants, revealing roles in maintaining plant genome stability. Sequence analysis of mutational repair of CRISPR‐Cas9‐induced DSBs revealed that FANCI promotes single nucleotide insertions and reduces longer deletions. This pattern of mutations may reflect roles for FA proteins in replication‐coupled repair of a subset of DSBs. Taken together, this analysis finds evidence for multiple roles for FANCI in the maintenance of plant genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c1e3958ce2798a3276d09d0f66000e5f49655d2" target='_blank'>
              Arabidopsis thaliana FANCONI ANAEMIA I (FANCI) has roles in the repair of interstrand crosslinks and CRISPR‐Cas9 induced DNA double strand breaks
              </a>
            </td>
          <td>
            Atheer Balobaid, W. Waterworth, Sophya F Vila Nova, B. Causier, Vinay Sharma, Madeline R Park, Manish K. Pandey, Chris West
          </td>
          <td>2025-10-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="E2F5 is a member of E2F transcription factor superfamily, controlling many molecular activities, such as cell proliferation, cell differentiation, DNA repair and cell death. Therefore, it is closely related to the occurrence, development and prognosis of a variety of cancers. In recent years, with the rapid development of bioinformatics, genomics and epigenetics, this study has further elucidated of E2F5 in the tumor cell cycle. Based on the latest research reports, this study reviewed the structural composition, dynamic activity regulation of E2F5, and how its transcription program driven by carcinogenic activity changed the progress of various tumor cell cycles, especially how it converted from a “gatekeeper” to a “destroyer”, thus affecting abnormal biological behaviors of tumor cells. Our aim is to provide a new direction for the development of E2F5 targeted therapy strategies and drug resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04822ad21c2d133c4bb16efee92ced6e01833c81" target='_blank'>
              The Roles of E2F5 in Tumor Cell Cycle: The Gatekeeper or Destroyer?
              </a>
            </td>
          <td>
            Yingwen Du, Danyun Wang, Aidi Liang, Xinru Tang, Jiansen Chen, Ruizhi Yao, Lei Meng, Jianxing Xie, Ming Chen, Songtao Xiang, Canbin Lin
          </td>
          <td>2025-10-21</td>
          <td>Urology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract DNA interstrand crosslinks (ICLs) are highly cytotoxic lesions that block essential cellular processes like replication and transcription. Endogenous ICLs can be induced by reactive aldehydes produced during normal cellular metabolism. Defective repair of these aldehyde-induced ICLs is associated with Fanconi anaemia (FA), a cancer predisposition syndrome. We previously showed that acetaldehyde-induced ICLs are repaired by the FA pathway and a novel excision-independent pathway. Here, we demonstrate that ICLs induced by acrolein, another cellular aldehyde, are also repaired by both pathways, establishing the generality of aldehyde ICL repair. Focusing on the FA pathway, we identify DNA polymerase kappa (Polκ) as the primary translesion synthesis (TLS) polymerase responsible for the insertion step during lesion bypass of unhooked aldehyde ICLs. This function requires Polκ’s catalytic activity and PCNA interaction domains but is independent of Rev1 interaction. In contrast, Polκ has a non-catalytic role in the extension step of cisplatin ICL repair that is dependent on Rev1 interaction. Our work reveals a key role for Polκ in aldehyde ICL repair and provides mechanistic insights into how different ICL structures determine the choice of TLS polymerases during repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1abf1a2fc3c509254fd2e9bc5c16c755167ac68d" target='_blank'>
              DNA polymerase kappa is the primary translesion synthesis polymerase for aldehyde ICLs
              </a>
            </td>
          <td>
            Roxanne van der Sluijs, Alex E E Verkennis, M. Hodskinson, Jamie Barnett, Victoria M. Cruz, Miguel Hernandez-Quiles, Themistoklis Liolios, Sally B Morton, Aiko Hendrikx, Collin Bos, H. Post, Christopher L. Millington, Clément Rouillon, Giulia Ricci, Francesca Mattiroli, David M Williams, M. Altelaar, Michiel Vermeulen, K. J. Patel, P. Knipscheer
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 H3K27 mutant(H3K27M) tumors are the most common type of diffuse midline gliomas and are lethal childhood tumors with a median survival of less than 12 months. Radiation is a critical treatment, yet these tumors recur rapidly within the radiation field due to resistance to radiation-induced DNA damage, allowing the tumor cells to survive. However, the interactions mediating efficient DNA repair, radiation resistance, and recurrence are poorly understood. Our goal was to identify mediators of treatment resistance in H3K27M tumors for therapeutic targeting to improve survival.



 We developed an in vivo recurrence model to assess mediators of treatment resistance. Bulk RNA, single cell RNA, and spatial transcriptomics analyses were used to determine drivers of resistance, and genes of interest were mechanistically assessed for their role in resistance.



 We determined that H3K27M epigenetic reprogramming results in activation of cell-surface protein THY1 which is highly upregulated in all H3K27M tumors. We then confirmed in numerous in vitro and in vivo models that THY1 mediates significant treatment resistance. We determined that THY1 mediates resistance through rapid and efficient DNA repair. We then performed extensive mechanistic analyses which revealed that THY1 signaling activates Fyn, a Src family kinase, resulting in downstream DNA repair. Remarkably both biological knockdown of Fyn and pharmacological Fyn inhibition, with brain-penetrating saracatinib, result in delayed DNA repair and tumor cell death.



 We have uncovered that the Thy1-Fyn signaling cascade mediates rapid DNA repair and treatment resistance. Pharmacological Fyn inhibition successfully delays DNA repair which could improve patient survival. As saracatinib is highly brain penetrant(0.5:1 Brain: Plasma ratio) and has demonstrated safety in Phase I/II Alzheimer’s clinical trials, it is an ideal drug for clinical translation. We are currently designing a Phase 0/1 study to assess drug penetration, efficacy, and safety with saracatinib and radiation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c2f1401d86d8dd15e7bf8324894f0a446afbce" target='_blank'>
              EXTH-65. THY1 Mediates Treatment Resistance in H3K7M Pediatric Tumors and is Therapeutically Targetable
              </a>
            </td>
          <td>
            S. Syed, Sunjong Ji, Arushi Tripathy, Yazen Alomary, Reka Chakravarthy, Mateus Mota, Dah-Luen Huang, Zhuwen Wang, Zian Li, Qiang Zhang, Daniel Nava de la Martin, T. Lawrence, D. Wahl, C. Koschmann, S. Venneti, A. Rehemtulla, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with limited therapeutic options and poor prognosis. Growing evidence suggests that transposable elements (TEs) may play critical roles in cancer by promoting genomic instability, altering gene expression through insertional mutagenesis, and modulating immune responses via the expression of TE-derived neoantigens, but their specific impact on IDH-wildtype GBM remains unclear. We investigated the association of TE abundance with survival and oncogenic pathways in GBM.



 We analyzed RNA-sequencing data from 151 IDH-wildtype GBM tumors, classified per WHO 2021 guidelines. TE abundance was quantified in tumor samples across genomic regions (whole genomic, exonic, intronic, intergenic) using REdiscoverTE. Associations with overall survival were evaluated using Cox proportional hazards models. Pathway enrichment analyses were performed for genes correlated with survival-associated TEs.



 Increased abundance of several TEs was significantly associated with reduced survival. Notably, whole-genome BC200 abundance was linked to poorer survival (HR1.75, 95% CI:1.28-2.22, p= 0.0042), with high expressors having a median survival of 9.81 vs 14.7 months. Similarly, LTR7Y (HR=2.24, 95%CI: 1.55-3.25, p=0.011), MER61D (HR=1.48, 95%CI: 1.20-1.82, p=0.055), and exonic L1ME3B (HR=2.27, 95% CI: 1.5-3.44, p=0.022) were significantly associated with inferior survival. These survival associations were independent of MGMT promoter methylation and CDKN2A deletion status. Gene expression correlated with TE abundance revealed enrichment in pathways related to RNA processing (BC200) L1mE3B), cellular motility and axon guidance (LTR7Y, MER61D), and immune regulation (LTR16A), suggesting potential multifaceted roles in GBM progression.



 Our findings demonstrate that specific TEs are significantly associated with inferior survival in IDH-wildtype GBM, highlighting their potential as prognostic markers alongside known somatic mutations. Their association with key tumorigenic processes such as immune evasion and genomic instability further suggests a contributory role in tumor progression and identifies them as potential therapeutic targets. Future studies should investigate the mechanistic functions, regulatory dynamics, and tissue specificity of TEs using longitudinal datasets to better elucidate their role in GBM biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d64d64059b2e2f2e271240ac062c311929bc6e" target='_blank'>
              EPID-30. Abundance of Transposable Elements is Associated with inferior Survival in IDH-wildtype Glioblastoma: A TCGA analysis
              </a>
            </td>
          <td>
            Eduardo Rodriguez Almaraz, G. Guerra, George Wendt, David V Glidden, Aaron Scheffler, Susan M Chang, S. Francis
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is an aggressive form of primary brain tumor, characterized by high malignancy and genetic heterogeneity, and standard treatment for GBM has remained unchanged in the last decade. A feature of GBM that makes it particularly difficult to treat is its heterogeneity, in part created by glioma stem cells (GSCs) within the solid tumor. GSCs have high plasticity and the ability to self-renew and differentiate into various heterogeneous cancer cell populations. GSCs are kept in a ‘stem’- like state with unlimited self-replication without differentiation and maturation, which helps maintain tumor heterogeneity and growth. This state also prevents them from being targeted effectively by traditional therapeutics. Histone modifications keep late-glial genes inaccessible by transcription factors, promoting transcription of replication and early-glial genes, maintaining stem-ness. A major player in this process is the imitation switch (ISWI) family of ATP-dependent chromatin remodelers, which have been found to have widespread aberrant expression in cancers, including gliomas. The ISWI complex can change the spacing of histones on the chromatin, and is primarily involved in the repression of gene expression. It contains a catalytic subunits, SMARCA5, and our data support that inhibition of SMARCA5 reprograms GSCs to differentiate. RNA-Seq after genetic depletion and pharmacologic inhibition of SMARCA5 in GSCs leads to loss of expression of early glial lineage genes and increased expression of late glial genes. We have developed a first-in-class small molecule SMARCA5 inhibitor that passes the BBB. In pilot studies, SMARCA5 depletion through genetic and pharmacologic inhibition extended animal survival in GSC-derived xenograft models of GBM. ATAC-Seq and single-cell RNA-Seq of tumors further support that SMARCA5 inhibition facilitated transition from stem/progenitor to differentiated cell states. Additionally, the SMARCA5 pharmacological inhibitor was well tolerated in vivo, suggesting that targeting SMARCA5 may have a favorable therapeutic window and high clinical translation potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea3b1fa5c18282fa0b9ccc1bb0d300ed3d16cc6" target='_blank'>
              STEM-03. SMARCA5 AS A THERAPEUTIC TARGET FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            Molly Guthrie, Himanshu Dashora, Le Xiong, Meghana Budankayala, Hongyu Zhao, Hariti Shah, Julia Yoo, Tapas Behera, Dalya Khalife, Y. Saunthararajah, Liangqi Xie, Jennifer Yu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6d95c002d856c172eff01b267c3c0451ed62ab" target='_blank'>
              Stable hypermutators revealed by the genomic landscape of genes involved in genome stability among yeast species.
              </a>
            </td>
          <td>
            Carla Gonçalves, J. Steenwyk, D. Rinker, D. A. Opulente, A. Labella, Marie-Claire Harrison, John F. Wolters, Xiaofan Zhou, Xingxing Shen, S. Covo, M. Groenewald, C. T. Hittinger, A. Rokas
          </td>
          <td>2025-11-03</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="


 Glioblastoma multiforme (GBM) is one of the most aggressive and lethal forms of brain cancer, characterized by cellular heterogeneity, poor prognosis and resistance to conventional therapies. One of the major challenges in treating GBM is its diffuse nature, which complicates surgical resection and contributes to high recurrence rates. Our research focuses on the role of biomolecular condensates - membraneless compartments formed by phase separation - in driving oncogenic processes. These condensates organize cellular biochemical processes by compartmentalizing proteins and nucleic acids, thereby regulating gene expression, signal transduction, and metabolism. We are particularly interested in the formation of aberrant condensates in glioblastoma cells and their contribution to malignant transformation, metabolic reprogramming and treatment resistance.



 Using patient-derived GBM models, CRISPR/Cas9 technology, super-resolution microscopy, and proteomic analyses, we are investigating the structural and functional properties of these condensates.



 Our goal is to determine how their dysregulation facilitates cancer progression and to identify novel vulnerabilities that could be therapeutically targeted.



 This research advances our understanding of the pathophysiological roles of biomolecular condensates in cancer and opens new avenues for innovative treatment strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc0d78352f5d6b9af0c7f40eba7db59a9e99146d" target='_blank'>
              P01.03.A SPATIOTEMPORAL ORGANIZATION OF BIOMOLECULES IN GLIOBLASTOMA MULTIFORME CELLS BY PHASE SEPARATION
              </a>
            </td>
          <td>
            M. Sztacho, A. Chytła, C. Carrera Bravo, S. Abbasi Sani, E. Herencia Lagunar, R. Gönül
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="JC virus (JCV) replicates within the nuclei of glial cells in the human brain and causes progressive multifocal leukoencephalopathy. JCV possesses a small, circular, double-stranded DNA genome, divided into early and late protein-coding regions. The non-coding control region (NCCR) functions bidirectionally for both early and late genes, and the agnogene is located downstream of TCR and upstream of three capsid proteins in the late region. Previously, in cell culture systems, we demonstrated that these capsid proteins accumulate in intranuclear domains known as promyelocytic leukemia nuclear bodies (PML-NBs), where they assemble into virus-like particles (VLPs). To investigate the agnogene’s function, VLPs were formed in its presence or absence, and differential gene expression was analyzed using microarray technology. The results revealed altered expression of histone-modifying enzymes, including methyltransferases (EHMT1, PRMT7) and demethylases (KDM2B, KDM5C, KDM6B), as well as various kinases and phosphatases. Notably, CTDP1, which dephosphorylates the C-terminal domain of an RNA polymerase II subunit, was also differentially expressed. The changes were predominant in the presence of the agnogene. These findings indicate that the agnogene and/or its protein product likely influence epigenetic regulation associated with PML-NBs, which may influence cell cycle control. Consistently, in human brain tissue, JCV-infected glial cells displayed maintenance of a diploid chromosomal complement, likely through G2 arrest. The precise mechanism of this, however, remains to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b19411275496dccfccf4c9581e91d1c44a41e774" target='_blank'>
              JC Virus Agnogene Regulates Histone-Modifying Enzymes via PML-NBs: Transcriptomics in VLP-Expressing Cells
              </a>
            </td>
          <td>
            Yukiko Shishido-Hara, Takeshi Yaoi
          </td>
          <td>2025-10-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer's global prevalence underscores a critical need for novel biomarkers to guide treatment and improve patient outcomes. Biomarker discovery historically focused on mutations in protein coding regions, comprising merely 1% of the genome. However, with advances in whole-genome sequencing, the functional significance of the noncoding genome-comprising the remaining 99%-has become increasingly evident. Noncoding regions play a vital role in regulating gene expression, and mutations within these regions have been associated with cancer risk, progression, and treatment response. This Review compiles and synthesizes current knowledge on cis-regulatory alterations (promoters/enhancers) and long noncoding RNAs (lncRNAs) in breast cancer. Key examples include promoter mutations [e.g., rs2279744 (Mouse double minute 2 homolog gene; MDM2)], enhancer mutations [e.g., rs4784227 (thymocyte selection-associated high mobility group box family member 3 gene; TOX3)], and lncRNAs [e.g., HOX transcript antisense intergenic RNA (HOTAIR)] linked to progression, metastasis, and poor survival. Integrating preclinical (in vitro, in vivo) and clinical findings, we emphasize the biomarker and therapeutic potential of these noncoding alterations. This Review also critically identifies the pressing need for more specific functional validation studies to fully elucidate their mechanistic roles. This emerging field offers promising opportunities to advance personalized medicine and refine prognostic/predictive strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3ed147fc4400798d8583cc23b80b7bdf1dc1ff" target='_blank'>
              Cis-regulatory and long noncoding RNA alterations in breast cancer - current insights, biomarker utility, and the critical need for functional validation.
              </a>
            </td>
          <td>
            Arnau Cuy Saqués, Aracele Martinez-Mendez, J. Crown, A. Eustace
          </td>
          <td>2025-11-13</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Potassium is the most abundant intracellular cation. Potassium flux across cell membranes is controlled by potassium channels. While cytoplasmic potassium has been extensively studied, very little is known about the function of potassium in the cell nucleus. Here, we discover that a nuclear potassium channel complex composed of KCNG1 and KCNB1 subunits is a regulator of nuclear potassium in medulloblastoma, the most common malignant pediatric brain tumour. Using genetically encoded potassium sensors, we show that KCNG1 regulates nuclear potassium level independently of cytoplasmic potassium level. KCNG1 knockdown alters the genomic architecture of medulloblastoma cells by stabilizing G-quadruplex DNA and inducing DNA damage. Ultimately, KCNG1 deficiency results in nuclear swelling, thereby compromising nuclear envelope integrity, promoting the formation of micronuclei, and inducing DNA spillage into the cytoplasm to activate the immune-stimulatory cGAS/STING pathway. Kcng1 knockout in a genetically engineered mouse model of medulloblastoma remodels the immune microenvironment, reduces tumour growth, and prolongs mouse survival. Altogether, we identified a potassium channel complex that regulates nuclear potassium to govern nuclear integrity, tumour immune microenvironment, and medulloblastoma growth. Our results demonstrate that targeting nuclear potassium can be leveraged to treat medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3156f37831fc433a976d1d47ab5180bff944abd" target='_blank'>
              DNAR-02. Nuclear potassium governs nuclear integrity and tumour growth in medulloblastoma
              </a>
            </td>
          <td>
            Jade Chan, Xi Huang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic instability is a hallmark of aging and cancer. A key contributor to genomic instability includes alternative DNA structures, such as cruciform-forming inverted repeats (IRs). Short IRs (< 100 bps) are abundant in the human genome, mutagenic, and enriched at mutation hotspots in human cancer genomes. Using an innovative mutation-reporter mouse model, we showed that short IRs are mutagenic in vivo. Further, we found that aging exacerbates IR-induced genomic instability, as evidenced by increased mutation frequencies and altered spectra in the spleen and brain of mice harboring either a short IR or control B-DNA sequence at 2 and 24 months of age. These findings establish a link between aging and enhanced mutagenesis at short IRs, providing a unique in vivo platform to investigate age-related mechanisms of DNA structure-mediated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdc4812813b7511aa9fae9d1c0a6308731d4e5f" target='_blank'>
              Aging Increases Short, Inverted Repeat-Mediated Genomic Instability In Vivo.
              </a>
            </td>
          <td>
            Pooja Mandke, Pallavi Kompella, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-09-30</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Head and neck cutaneous squamous cell carcinoma (HNCSCC) remains underexplored compared to oropharyngeal squamous cell carcinoma, particularly in relation to human papillomavirus (HPV) and molecular markers such as p16 and p53. While p16 is a well-established surrogate for HPV in oropharyngeal cancer, our review highlights its unreliable role in HNCSCC, where positivity is instead associated with recurrence and metastasis. Similarly, p53 illustrates a dual role - wild-type as a genomic safeguard, mutated as an oncogenic driver - complicating prognostication. Methodological considerations, including the limitations of immunohistochemistry for HPV detection, underscore the need for multi-method and molecular validation in future studies. Ultraviolet radiation is posited as a key modifier of p16 function, decoupling expression from tumor suppression. To contextualize these findings, we draw parallels to glioblastoma (GBM), where subclonal evolution, p53 dysfunction, and intratumoral heterogeneity drive relapse despite aggressive multimodal therapies. GBM exemplifies how bulk-level biomarker generalizations often obscure dynamic cellular ecosystems, reinforcing the necessity of single-cell and spatial approaches. Multi-omics integration - encompassing genome, transcriptome, proteome, and tumor microenvironment mapping - coupled with single-cell RNA sequencing and spatial transcriptomics, offers a path forward for resolving subclonal dynamics in both HNCSCC and GBM. These technologies provide the resolution needed to track tumor-immune-stromal co-evolution, identify therapy-resistant clones, and anticipate recurrence. We argue for a N-of-1, patient- and cell-centric paradigm that reframes biomarkers not as static surrogates but as dynamic readouts of cancer evolution across time and tissue contexts. Conceptually, we propose kinetic and microenvironmental frameworks (e.g. , “load-and-lock” barriers; dormancy and immune-synapse stabilization) as hypothesis-generating avenues to stall clonal handoffs and improve outcome prediction. Together, these perspectives argue for revised biomarker frameworks in HNCSCC and ethically inclusive, mechanism-anchored studies that bridge discovery with individualized care. By bridging insights from HNCSCC with the lessons of GBM, this review underscores the need for ethically inclusive, mechanistically informed frameworks that integrate subclonal evolution, biomarker re-interpretation, and precision-personalized hybrid models. Such an approach will be essential for advancing from one-size-fits-all strategies to individualized lifetime cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa038c0ad9d152c7382847ca5a53a02fc9fd0417" target='_blank'>
              Rethinking p16, p53, and HPV in HNCSCC through lessons from glioblastoma subclonal evolution toward patient-centric N-of-1 single-cell RNA sequencing paradigm
              </a>
            </td>
          <td>
            Henry Michael Lee, Shengwen Calvin Li
          </td>
          <td>2025-11-16</td>
          <td>World Journal of Clinical Cases</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79383ce31e18bce44c59616029f77e566e53fa0" target='_blank'>
              The Utility of Fibroblast Co-culture for the Maintenance of Episomes in Human Papillomavirus-Associated Cancer Models
              </a>
            </td>
          <td>
            Austin J. Witt, Rachel L. Lewis, Xu Wang, Phoebe Bridy, Jenny D Roe, I. Morgan, C. James, Molly L Bristol
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Aim: The A2UCOE, a well-characterized chromatin-modifying element originating from a constitutively expressed human gene locus, promotes consistent and durable gene activity, even when foreign DNA sequences are inserted into densely packed heterochromatin regions. The aim of this study is to examine the Daedalus fragment in A2UCOE to identify key sub-regions responsible for its prominent chromatin opening activity. This would enable the discovery of a minimal yet fully functional element, allowing for more efficient utilization of vector capacity to accommodate therapeutic transgenes when incorporated into lentiviral vectors (LVs).
Method: To assess the chromatin-opening activity of the Daedalus sub-region within the A2UCOE, lentiviral vectors carrying eGFP reporter constructs were generated and transduced into murine embryonal carcinoma cell lines (P19 and F9). Transgene expression stability was assessed across both pluripotent cells and those differentiated with retinoic acid. HEK293T cells were used for lentiviral packaging. Plasmid propagation was performed in E. coli DH5α, and standard molecular cloning techniques were applied. Flow cytometry was used to measure eGFP expression levels, and significance was assessed via Student’s t-test (p < 0.05).
Results: Flow cytometry and RT-qPCR analyses revealed that the positive control vector (1.5A2UCOE-SEW) maintained stable eGFP expression in both differentiated states of P19 and F9 cells, whereas the UCOE vectors named Daedalus-F and Daedalus-R were unable to prevent transgene silencing. In undifferentiated cultures, eGFP expression from these vectors decreased by 40–50% within two weeks, with a similar decline observed in differentiated cells. The average vector copy numbers remained stable, indicating transcriptional silencing rather than vector loss.
Conclusion: The Daedalus sub-region of A2UCOE alone could not prevent transgene silencing in pluripotent or differentiated cells, indicating additional A2UCOE elements are needed for stable long-term expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b98626fce66cdb8386ed52347362958929e350" target='_blank'>
              Daedalus fragment from the CBX3 ubiquitous chromatin opening element as an optimum UCOE candidate
              </a>
            </td>
          <td>
            Ali Osman Arslan, Ömer Faruk Anakök
          </td>
          <td>2025-10-26</td>
          <td>Northwestern Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a1d7b33f411a72234eadba08fcc3ddda8c8f36" target='_blank'>
              Integrative single-cell and bulk RNA-seq analysis identifies glycosyltransferases-related signature in triple negative breast cancer
              </a>
            </td>
          <td>
            Junyi Hu, Ningning Yuan, Zhenglan Huang, Yancheng Liu, Bohan Tu, Xinyue Yu, Tianli Hui, Guowei Zuo
          </td>
          <td>2025-11-08</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff9dd6e8b4e7b70e0de9451cb1c5dd37501bb1c" target='_blank'>
              A comprehensive genomic framework for identifying genes predisposing to homologous recombination repair deficient breast cancer
              </a>
            </td>
          <td>
            José Camacho-Valenzuela, Thibaut Matis, Carla Roca, Jorge Luis, Cuamatzi Flores, N. Hamel, Barbara Rivera, Simon Gravel, P. Polak, Carla Daniela, Robles-Espinoza, William D. Foulkes
          </td>
          <td>2025-10-23</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Shillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Unlike other cancer types the determinants of anti-PD-1 response in clear cell RCC are not well defined; ccRCC exhibits a low mutational burden and CD8+ T cells infiltration is associated with worse prognosis. Cytomegalovirus (CMV) and human papillomavirus (HPV) infections are postulated to modulate the anti-tumor immune responses within the tumor microenvironment. We sought to investigate the clinical and immunologic impact of CMV and HPV viral transcriptomic signatures, as defined by prior studies, in patients with RCC. Methods RNA-seq data from three RCC prospetive trials were upper-quartile normalized, log2-transformed (TPM + 1), and batch-corrected using ComBat. Transcriptomic scores for CMV (75 genes) and HPV (79 genes) were computed using geometric means (71 and 77 genes were found, respectively). CD8 immunofluorescence (IF) was performed on tumor samples. Overall survival (OS) and progression-free survival (PFS) were assessed using multivariable Cox regression, adjusting for age at therapy start, gender, IMDC risk, sarcomatoid/rhabdoid features, and number of previous lines received. Results A high intratumoral CMV transcriptomic score was significantly correlated with T-effector signature (ρ  =  0.54, p = 4.8 × 10−6), IF tumor core CD8+ (ρ  =  0.43, P < .001), IF tumor margin CD8+ (ρ  =  0.41, P < .001), and myeloid signature (ρ  =  0.40, P < .001). The CMV score was significantly higher in infiltrated versus excluded/desert tumors (P = .0079). The CMV score was associated with worse OS among patients receiving immune checkpoint blockade (ICB) (HR 4.98, P = .039). In contrast, the HPV transcriptomic score showed only a modest correlation with myeloid signature (ρ  =  0.24, P = .051) and no association was found with OS nor PFS. Conclusions In metastatic RCC, a high tumor CMV transcriptomic score was associated with a paradoxical immune landscape, marked by CD8+ infiltration yet co-enriched with suppressive myeloid signatures, that predicts inferior overall survival following ICB. These viral signatures within the complex immune architecture of RCC may help understand immune responsiveness and resistance within distinct genomic and therapeutic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c3c656ee910ac2c2d9287a96ba1a9501c97839" target='_blank'>
              39Intratumoral viral transcriptomic signatures stratify immune phenotypes and clinical outcomes in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            Mustafa Saleh, E. Saad, Pablo Barrios, M. Machaalani, Liliana Ascione, W. Daoud Khatoun, Jad El Masri, Razane ElHajj Chehade, Marc Eid, Rashad Nawfal, Karl Semaan, E. Yekeduz, Clara Steiner, Weiwei Bian, Maxine Sun, S. Signoretti, E. V. Van Allen, D. Braun, Alexander Gusev, Toni Choueri
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Adrenocortical carcinomas (ACC) are aggressive and resistant to medical treatment. This study reports a single-nucleus transcriptome atlas of steroid and microenvironment cells in 38 human normal adrenals and adrenocortical tumors. We identify intermediate-state cells between glomerulosa and fasciculata, a transition state in the centripetal trans-differentiation of normal steroid cells. In tumors, steroid cells show expression programs reflecting this zonation. Although ACC microenvironment is scarce, its signatures combine with those of steroid cells into ecotypes. A first ecotype combines cancer-associated fibroblasts, tumor-associated endothelial cells, with hypoxia and mitosis signatures in steroid cells. Another ecotype combines exhausted T cells, with fasciculata steroid signature. These ecotypes are associated with poor survival. Conversely, a third ecotype combines inflammatory macrophages, with reticularis steroid signature, and better outcome. These steroid/microenvironment cells interplays improve outcome predictions and may open therapeutic options in aggressive ACC, through immune microenvironment activation by modulating glucocorticoids/androgens balance. Adrenocortical carcinomas (ACC) are aggressive and often resistant to therapy. Here, the authors provide a single-nucleus transcriptomic atlas of ACCs and normal adrenal glands, finding ecotypes in steroid and microenvironment cells that are associated with clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9a1421593f90d7b71feb29a6ec33441fea9876" target='_blank'>
              Impact of steroid differentiation on tumor microenvironment revealed by single-nucleus atlas of adrenal tumors
              </a>
            </td>
          <td>
            A. Jouinot, Yoann Martin, Florian Violon, Thomas Foulonneau, Yanis Bendjelal, Philip Calvet, Brigitte Izac, Franck Letourneur, C. Bertholle, Muriel Andrieu, R. Onifarasoaniaina, Maryline Favier, C. de Guitaut, Archibald Fraikin, D. de Murat, Roberta Armignacco, N. Benanteur, M. F. Birtolo, Mathilde Sibony, K. Perlemoine, P. Vaduva, L. Bouys, F. Bonnet-Serrano, Bertrand Dousset, Martin Gaillard, É. Pasmant, Maxime Barat, Anthony Dohan, M. Haissaguerre, Antoine Tabarin, R. Libé, L. Guignat, L. Groussin, A. Berthon, B. Ragazzon, Jérôme Bertherat, G. Assié
          </td>
          <td>2025-10-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Clear cell renal cell carcinoma (ccRCC) represents the most aggressive form of renal cell carcinoma (RCC), distinguished by pronounced intratumoral heterogeneity, extensive metabolic reprogramming, and marked resistance to conventional therapeutic approaches. This study aimed to comprehensively characterize the cellular heterogeneity, epigenetic regulation, and transcription factor (TF) networks in ccRCC by integrating multi-omics data, and to identify functional key genes with prognostic and therapeutic significance. Methods Single-cell RNA sequencing (scRNA-seq), single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq), and spatial transcriptomics (ST) were integrated to comprehensively explore cellular heterogeneity, epigenetic regulation, and TF networks in ccRCC. To uncover dynamic alterations in gene expression during cellular differentiation, single-cell pseudotime analysis and gene set enrichment analysis (GSEA) were performed. Furthermore, the functional significance of Y-box binding protein 3 (YBX3) in ccRCC cells was experimentally validated. Results Single-cell transcriptomic profiling revealed 16 distinct cell populations within the ccRCC tumor microenvironment (TME), including ccRCC tumor cells, exhausted CD8+ T cells (Exhau CD8+ T cells), and macrophages. The scATAC-seq analysis demonstrated cell type-specific chromatin accessibility in immune cells, whereas ccRCC tumor cells exhibited reduced accessibility at immune-related genes, such as cluster of differentiation 2 (CD2). Epigenetic profiling further indicated that differentially accessible chromatin peaks in ccRCC cells were primarily enriched within intronic and exonic regions, implicating key TFs, including hepatocyte nuclear factor 1-beta (HNF1B) and the FOS-JUNB complex. An integrated analysis of scRNA-seq and scATAC-seq datasets identified five critical genes, YBX3, cubilin (CUBN), small nucleolar RNA host gene 8 (SNHG8), acetyl-CoA acyltransferase 2 (ACAA2), and protein kinase AMP-activated catalytic subunit α2 (PRKAA2), that were significantly associated with ccRCC prognosis. Pathway enrichment analysis revealed their involvement in metabolic reprogramming and tumor progression. Functional assays further confirmed that YBX3 knockdown inhibited ccRCC cell proliferation and migration. Conclusions This study elucidates cellular heterogeneity, the epigenetic regulatory landscape, and the key genes driving ccRCC progression. The integration of multi-omics data offers novel insights into precise diagnostic strategies and therapeutic interventions, highlighting the pivotal role of genes such as YBX3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ee8baff74e64150e50eefda9fdb1e11ec3dfcb" target='_blank'>
              Single-cell multi-omics and spatial transcriptomics reveal the transcriptional regulatory landscape of clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Juan Duan, Peifeng Ke, Bangqi Wang, Xiaofu Qiu, Yifeng He, Zongtai Zheng, Zemin Wan, Chuling Wu, Zhuoyun Lv
          </td>
          <td>2025-10-01</td>
          <td>Translational Andrology and Urology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The rate of DNA synthesis is crucial for full DNA duplication. We report a key role of p21 in controlling this rate. During normal replication, p21 promotes nascent DNA synthesis alongside the DNA polymerase iota (Pol ι)/p53 complex. When p21 is down-regulated but detectable, nascent DNA tracks are longer and discontinuous and rely on primase and DNA polymerase (PrimPol). With the complete elimination of p21, nascent DNA tracks become shorter and continuous and depend on Pol kappa (κ). Endogenous p21 levels are critical for genomic stability, as both PrimPol- and Pol κ–mediated syntheses can induce chromosomal instability. The residual expression of p21 in p53-null cells influences the involvement of PrimPol or Pol κ in nascent DNA synthesis and subsequent chromosomal instability. Our results demonstrate that endogenous levels of p21 in cycling cells, insufficient for cyclin-dependent kinase inhibition, prevent genomic instability through proliferating cell nuclear antigen binding (PCNA), limiting PrimPol and Pol κ’s role in nascent DNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31f81e290a07076ab9200968e4922ab77b5675a" target='_blank'>
              Endogenous p21 levels protect genomic stability by suppressing both excess and restrained nascent DNA syntheses
              </a>
            </td>
          <td>
            N. L. Calzetta, S. Mansilla, Candelaria Mares Ahlers, Agostina P. Bertolin, Lilen I. Caimi, Sofía Venerus Arbilla, Jingkun Zeng, Lisa Wiesmüller, V. Gottifredi
          </td>
          <td>2025-10-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 The genomic regions surrounding tumor suppressor genes are often lost as collateral damage during tumorigenesis, providing novel therapeutic vulnerabilities. To systematically identify synthetic lethal pairs of genes within DNA damage repair (DDR) pathways, we conducted a combinatorial genome-wide CRISPR screen. We found that RNASEH2B, a gene frequently co-deleted with RB1 in prostate cancers, emerged as a top vulnerability with knockout of APEX2, which is a nuclease involved in DNA repair. This interaction was validated in RNASEH2B isogenic cell line pairs using whole-genome CRISPR screens and further confirmed across multiple cancer lineages using a focused mini-pool sgRNA library. Single gene knockouts demonstrated that loss of APEX2, specifically its catalytic activity, is sufficient to eliminate RNASEH2B mutant cells both in vitro and in vivo. Importantly, loss of RB1 did not impact the synthetic lethal interaction between RNASEH2B and APEX2. Consistent with prior reports, APEX2 knockout also impairs growth of BRCA1 mutant cells and enhances sensitivity to PARP inhibitors. Overall, these findings position APEX2 as a compelling synthetic lethal target in tumors with RNASEH2B deletion or BRCA1 mutations and support the development of APEX2 inhibitors as a potential monotherapy to improve patient outcome.



 Antoine Simoneau, Steve Lombardo, Ruthie Swain, Serge Gueroussov, Binzhang Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H. Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, Douglas A. Whittington, Yi Yu, Jannik N. Andersen, Teng Teng. APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f840d30e01fa153da53003edd382b7d06deea42" target='_blank'>
              Abstract B039: APEX2 is a synthetic lethal target for RNASEH2B-deleted or BRCA-mutant tumors
              </a>
            </td>
          <td>
            Antoine Simoneau, Steven A. Lombardo, Ruthie Swain, Serge Gueroussov, B. Shen, Shangtao Liu, Samuel R. Meier, Gwen Cusing, Ashley H Choi, Tenzing Khendu, Hannah Stowe, Hsin-Jung Wu, Hongxiang Zhang, D. Whittington, Yi Yu, Jannik N Andersen, T. Teng
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Significance How new genes arise and gain function is a central question in biology. New genes can evolve from nongenic DNA, yet their adaptive potential remains unclear. Here, we use millions of (semi-)random sequences as experimental models of emerging genes and find that thousands confer phage resistance in Escherichia coli. Expressed random sequences can produce both protein- and RNA-based functions that reprogram cellular systems to counter viral infection. Resistance arises through activation of a cell envelope stress response or downregulation of membrane receptor expression. Our results reveal that genetic novelty, in the form of genes appearing for the first time, can shape host–virus interactions, providing insight into microbial evolution and the surprising ease with which functional genes can emerge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c17bf5a40c5c5bfce399a75abe69e8bb312d9" target='_blank'>
              Emergence of antiphage functions from random sequence libraries reveals mechanisms of gene birth
              </a>
            </td>
          <td>
            Idan Frumkin, Christopher N Vassallo, Yi Hua Chen, Michael T. Laub
          </td>
          <td>2025-10-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT We have previously reported that human adenovirus E1A interacts with ARGLU1, a small disordered cellular protein. The consequences of this interaction for virus replication were unclear. E1A is the first protein produced during adenovirus infection. Via protein-protein interactions, E1A modifies the cellular environment to create optimal conditions for viral replication. To better understand the molecular mechanisms driving viral infection, we further investigated the functional consequences of the interaction between ARGLU1 and E1A. ARGLU1 interacts with E1A directly as determined by a GST-pulldown assay with recombinant proteins. Importantly, ARGLU1 was found to act as a transcriptional repressor when it was localized to viral promoters. Repression was driven by enhanced promoter-proximal RNA polymerase II pausing. Significantly, ARGLU1-induced repression of viral promoters is likely an unintended consequence of ARGLU1 binding to E1A, as a mutant of E1A (dl1102), unable to bind to ARGLU1, did not show reduced viral gene expression. Furthermore, ARGLU1 was found to colocalize with E1A in infected cell nuclei. Finally, the binding of E1A to ARGLU1 appears to reduce DNA damage repair in bleomycin-treated and adenovirus-infected cells, suggesting that E1A binding to ARGLU1 is important for DNA damage response inhibition. Overall, this study underscores the complexity of virus-host interactions and reveals a novel role for ARGLU1 during adenoviral infection. IMPORTANCE This study uncovers a novel antiviral role for ARGLU1, known for its involvement in splicing, transcriptional regulation, and DNA damage repair. Our study demonstrates that ARGLU1 functions as a viral restriction factor by reducing virus growth through the inhibition of viral gene expression and enhanced DNA damage response, leading to reduced viral growth. These findings not only highlight an important role for ARGLU1 in host antiviral mechanisms but also emphasize the utility of adenovirus as a tool to uncover critical cellular pathways. This study uncovers a novel antiviral role for ARGLU1, known for its involvement in splicing, transcriptional regulation, and DNA damage repair. Our study demonstrates that ARGLU1 functions as a viral restriction factor by reducing virus growth through the inhibition of viral gene expression and enhanced DNA damage response, leading to reduced viral growth. These findings not only highlight an important role for ARGLU1 in host antiviral mechanisms but also emphasize the utility of adenovirus as a tool to uncover critical cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e29ce875c83fc5e2a9c3669beab1e24607b11d4" target='_blank'>
              ARGLU1 is a negative regulator of the adenoviral replicative cycle
              </a>
            </td>
          <td>
            Amit Koul, Lauren Fulham, N. Akkerman, Drayson Graves, Esha Kaul, Khadija Khadija, Peter Pelka
          </td>
          <td>2025-11-04</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Retinoblastoma (RB) is the most common pediatric eye cancer. Most cases of RB are initiated by bi-allelic mutational inactivation of the RB1 gene, yet most RB tumors harbor additional genomic aberrations that may promote tumor progression. After RB1, the gene that is most commonly mutated gene in RB is BCOR, which is mutated in approximately 20% of RB tumors and is associated with a more aggressive tumor phenotype and worse patient outcomes. Despite its importance, little is known about the role of BCOR in RB. Here, we interrogated BCOR in low passage RB cell lines using mass spectrometry, chromatin immunoprecipitation sequencing, and RNA sequencing. We show that the BCOR protein interacts with members of the ncPRC1.1 Polycomb Repressive Complex and localizes at gene loci with traditionally activating and repressing chromatin markers. Loss of BCOR downregulates the expression of genes associated with cell cycle regulation and upregulates genes associated with hypoxic adaptation. We conclude that BCOR mutations slow cell proliferation and drive hypoxic adaptation in RB via epigenetic mechanisms that may be amenable to targeted therapy. Implications: This study reveals that BCOR may play a noncanonical, multi-faceted role in retinoblastoma with implications in cell cycle, differentiation, and hypoxic adaptation, ultimately shedding insight into its molecular framework for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba892d255c691eee277b20e16deae8de79fd0b76" target='_blank'>
              BCOR mutations deregulate cell cycle and hypoxic adaptation pathways in retinoblastoma.
              </a>
            </td>
          <td>
            Michelle G. Zhang, J. Kuznetsoff, Natalie C Cetta, Sebastian Salazar, Renata L Volonterio, Stefan Kurtenbach, Z. Corrêa, Daniel Pelaez, J. W. Harbour
          </td>
          <td>2025-11-05</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Oligodendrogliomas (OL) and Astrocytomas (AS), typically affect young adults and are characterized by the mutation of the isocitrate dehydrogenase genes (IDH1/2). While IDH-mutant gliomas are of good initial prognosis, they relapse almost systematically in higher-grade gliomas. These tumors display large intra- and inter-patient heterogeneity hallmarked by dynamic adaptation of tumor cell phenotypes and microenvironment across tumors and grades. However, the spatial organization and interactions between cell-types from the brain tumor microenvironment remain elusive and is required to better understand tumor growth and relapse. Here, we profiled in-depth IDH-mutant gliomas with Visium spatial transcriptomics across 32 areas from 20 patients that underwent surgical resection. Our cohort spans tumors of grade II to IV in WHO 2021 classification. We implement state-of-the art bio-informatics methods to decipher spatial heterogeneities such as chromosomal alterations, spatial domains, cell-type distribution and cell-cell interactions. With copy number variation inference, we retrieve stereotypical chromosomal alteration describe in the literature and classify spot according to their malignant content to identify tumor core areas compare to infiltrated cortex. By performing cell-type deconvolution with cell2location, we provide a map of IDH-mutant tumors microenvironment showing tumor cell states and brain tumor microenvironment cell spatial distribution and relationships. We then use machine learning and spatial statistics to identify recurrent patterns across samples and compare their spatial distribution across IDH-mutant subtypes and grades. With matrix factorization technique, we identify cellular niches defined by recurrent cell-type co-localizations found across IDH-mutant gliomas. We also identify clusters enriched in hypoxia programs that are linked to the specific cell-type compositions uncovered with deconvolution. Finally, we provide a spatially resolved landscape of the cell-cell interaction of IDH-mutant gliomas where we identify communication patterns between cellular niches. To summarize, our study provides a novel comprehensive landscape of IDH-mutant glioma spatial organisation across subtypes and grades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575bed115523c0aa5dd8c1a1b4cc0cb22e9ef744" target='_blank'>
              EPCO-18. UNRAVELLING IDH-MUTANT GLIOMAS HETEROGENEITIES WITH SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Lucie C Gaspard-Boulinc, A. Lyne, Coraline Krafft-Coletta, Emma Molière, Océane Saïbou, I. Couderc, D. Meyronet, F. Ducray, F. Cavalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT: Gene editing exploits endogenous DNA repair pathways to introduce precise modifications into the genome. The CRISPR system was first identified in Escherichia coli (1987) and, based on Cas protein architecture, is divided into Class I (types I, III, IV) and Class II (types II, V, VI). The widely adopted CRISPR/Cas9 system comprises a guide RNA (gRNA) and the Cas9 endonuclease, which orchestrate genome editing through a tripartite mechanism: target sequence recognition via gRNA binding to a complementary DNA site adjacent to a protospacer adjacent motif (PAM), double-stranded DNA cleavage by two nuclease domains (RuvC cleaving the non‑target strand and HNH the target strand), and repair of the resulting break by cellular pathways. This approach has been applied to in vitro fertilization-derived embryos and meiotically developing oocytes to disrupt or correct genes, offering potential for eliminating heritable diseases. However, concerns remain regarding off-target effects that introduces unwanted genetic mutations, necessitating improved specificity and ethical scrutiny.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e32b11816cdfe9dbe37c9b7f4fa9f2f1d5d3ed5" target='_blank'>
              CRISPR/CAS9-Mediated Gene Editing in Human Gametes: A Review
              </a>
            </td>
          <td>
            Esha Kumari, Neha Banu, Kathrina Marbaniang, Faridha Jane R.M. Momin, B. C. Hynniewta
          </td>
          <td>2025-09-30</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. Y. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 In frame fusions between Fibroblast Growth Factor Receptor 3 (FGFR3) and Transforming Acidic Coiled-Coil containing protein 3 (TACC3) genes on chromosome 4p16.3 have been reported across diverse cancers, including bladder cancer (∼5%) and IDH-wildtype glioblastoma (4.1%), as well as in small subsets of non-small cell lung (∼0.5%), esophageal squamous cell (1.4–2.1%), cervical (1.9%), head and neck (3.7%), and rare nasopharyngeal (2.5%) carcinomas, and other cancer types. These chromosomal translocations generate FGFR3-TACC3 (F3T3) driver oncofusion proteins which promote tumor cell growth and survival. The F3T3-driven cancer cells are critically dependent on the constitutively active oncofusion kinase activity for survival and proliferation, a phenomenon known as "oncogene addiction". While inhibitors of FGFR kinases, such as erdafitinib and fexagratinib, are the treatment of choice for F3T3-driven cancers, most treated patients do not respond to the drug, suggesting insufficient inhibition of the oncofusion activity. Additionally, most tumor responses to FGFR kinase inhibitors are partial and short-lived, followed by development of resistance and disease progression within few months. Based on the oncogene addiction of cancer cells on the F3T3 activity, we hypothesized that rapid ablation of this oncofusion protein can induce robust apoptotic response, leading to significant tumor regression and sustained remissions. To this end, we explored the use of Proteolysis Targeting Chimeric (PROTAC) bifunctional small molecules which can rapidly degrade F3T3 oncofusion protein by harnessing the cell's own proteasome machinery. We synthesized more than 100 novel PROTAC small molecules targeting F3T3 fusion through combination of previously reported TACC3 ligands with various linkers and ligands of E3 ubiquitin ligases CRBN, VHL or XIAP. We evaluated our new anti-F3T3 PROTAC molecules using F3T3-driven bladder (RT4 and RT112) and breast (SUM185PE) cancer cell lines, which express two of the most common oncofusion variants. Several PROTACs were able to degrade the native F3T3 oncofusion protein and suppressed the growth of F3T3-driven cells. One PROTAC (KHS-C4) degraded F3T3 at low nanomolar concentrations, strongly suppressed the oncogenic pathway downstream F3T3 fusion, had excellent predicted pharmacological properties, acceptable in vitro liver microsome stability (Clint = 59 mL/min/kg for human and 318 mL/min/kg for mouse microsomes) and no in vivo toxicity. Importantly, low concentrations of KHS101-C4 (but not of FGFR3 kinase inhibitors) eradicated F3T3-driven cancer cells (but not cancer cells that do not express the F3T3 oncofusion) in vitro. These data suggest that chemically-induced degradation of the F3T3 fusion protein can lead to oncogenic signal ablation sufficiently potent to induce a lethal oncogenic shock and can thus yield more durable responses than FGFR3 kinase inhibitors. KHS-C4 was selected for further evaluation to establish in vivo proof of concept for the PROTAC-based therapeutic strategy against F3T3-driven cancers.



 Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea. Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5bb4e0b8d5df4124ad0dd08279a3f1803abead8" target='_blank'>
              Abstract B120: Development of PROTACs for targeted degradation of FGFR3-TACC3 oncofusions
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Keita Nakane, Seiya Kitamura, Eugen Dhimolea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="ABSTRACT Centromeres are the chromosomal sites at which the kinetochore forms and attaches to spindle microtubules, directing chromosome segregation. Plasmids based on centromeres can maintain stability and distribute accurately during cell division, supporting them as effective genetic tools for research and biotechnological applications. Here, the centromeric regions on seven chromosomes of Ogataea polymorpha, a methylotrophic non‐conventional yeast with great potential in biotechnology, were located by ChIP‐seq with the native Cse4. The actual centromeric sequences of chromosomes 1, 2 and 5 were characterised, which are very different from those of other eukaryotes and each unique. Although long terminal repeats (LTR) were found in these centromeres, they are ‘solo LTR elements’ and not important for the function of centromeres. Hence, the O. polymorpha centromeres were categorised into small regional centromeres. O. polymorpha centromeric plasmids were constructed for the first time, which exhibit high genetic stability and compatibility. Application potential for multigene or multicopy expression was validated by the production of uricase and triterpene squalene. This research elucidates the structural features of O. polymorpha centromeres and constructs new, stable centromeric plasmids, expanding the phenomenal diversity of centromeres and providing a powerful genetic toolbox for multi‐node and multiple‐layered engineering of cell metabolism and physiological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8595896eccb7f21123d64f19b9f60bcb2fa63adb" target='_blank'>
              Centromeric Sequences in Ogataea polymorpha Genome Enable Development of Stable Multigene Expression Plasmid Tools
              </a>
            </td>
          <td>
            Yanfei Cheng, Yuanyuan Shi, Ning Sun, Xin Zhang, Mengwei Sun, Xiuping He
          </td>
          <td>2025-11-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) remains the most aggressive primary brain tumour in adults, marked by pronounced cellular heterogeneity, diffuse infiltration, and resistance to conventional treatment. In recent years, transcriptomic profiling has provided valuable insights into the molecular mechanisms that govern the progression of glioblastoma. This systematic review aims to synthesise the current literature on dysregulated gene expression in GBM, focusing on gene signatures associated with stemness, immune modulation, extracellular matrix remodelling, metabolic adaptation, and therapeutic resistance. Methods We conducted a systematic search of PubMed, The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and the GlioVis portal for studies published between January 2005 and April 2025, limited to English-language reports. Studies were eligible if they included adult glioblastoma tissue or patient-derived datasets and reported gene-level expression or clinical associations. Reviews, commentaries, and studies on non-GBM gliomas were excluded. Screening followed the PRISMA 2020 checklist, with 410 records initially identified, 90 duplicates removed, and 125 studies retained after full-text review. Data were synthesised descriptively, and findings were validated against TCGA/CGGA expression datasets to ensure consistency across cohorts. Results We categorised recurrently dysregulated genes by their biological function, including transcription factors (SOX2, ZEB2), growth factor receptors (EGFR, PDGFRA), immune-related markers (PD-L1, TAP1, B2M), extracellular matrix regulators (MMP2, LAMC1, HAS2), and metabolic genes (SLC7A11, PRMT5, NRF2). For each group, we examine the functional consequences of transcriptional alterations and their role in driving key glioblastoma phenotypes, including angiogenesis, immunosuppression, invasiveness, and recurrence. Conclusion We further discuss the prognostic implications of these gene signatures and evaluate their potential utility in precision medicine, including current clinical trials that target molecular pathways identified through transcriptomic data. This review highlights the power of gene expression profiling to stratify glioblastoma subtypes and improve personalised therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f178b970f37a5487e44c1e368727e8a6a513f43" target='_blank'>
              RNA Expression Signatures in Glioblastoma: A Systematic Review of Tumour Biology and Therapeutic Targets
              </a>
            </td>
          <td>
            Amber Hassan, Badr Hafiz, Taghreed A. Alsinani, Rakan Bokhari, D. Mirdad, Awab T. Tayyib, Alaa Alkhotani, Ahmad Fallata, Iman Mirza, Eyad Faizo, Saleh Baeesa, Huda Alghefari, Maher Kurdi
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In the initiation and development of cancer, DNA damage plays a very crucial role. In the case of environmental perturbation, mutation events in critical control genes may occur as the integrity of genetic material is lost, either due to replication errors, oxidative stress, etc. And provided that these are proto-oncogenes or tumour suppressor genes, they can interfere with normal control of the cell cycle, programmed cell death (apoptosis) and repair of damaged DNA(Branzei & Foiani, 2023). Consequently, they can start to multiply out of control, which can result in the appearance of growth and malignancy. This abstract discusses the molecular processes that connect the occurrences of DNA damage and mutations with the out-of-control growth of cancer cells, pointing to their importance in the development of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cd781c4b40bc6486e36eec6ae725a99f2c36b2" target='_blank'>
              Exploring How DNA Damage And Mutations Lead To The Uncontrolled Growth Of Cancer Cells?
              </a>
            </td>
          <td>
            Yashvi Chhabra
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Scientific and Research Publications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Chemoradiotherapy (CRT) is a standard-of-care treatment for several solid tumors, including locally advanced rectal cancer (LARC) and head and neck squamous cell carcinoma (HNSCC), yet the absence of predictive biomarkers impedes patient stratification and therapeutic optimization. CRT induces DNA double-strand breaks (DSBs), and repair of these lesions is influenced by chromatin architecture, where histone lysine methylation plays a critical role. We identified a germline missense single nucleotide polymorphism (SNP) in KDM3C, a lysine demethylase, that significantly correlates with CRT outcomes. We hypothesize that this serine-to-threonine SNP modulates CRT response by disrupting protein interactions required for BRCA1 recruitment to DNA damage sites.



 Isogenic SNP and wild type (WT) rectal (RCM-1) and head and neck (Cal-27, SCC-9) cancer cell lines were generated via CRISPR/Cas9. Functional analyses included immunofluorescence staining of DSB repair proteins, bulk RNA sequencing, and colony survival assays. Structural modeling was conducted using AlphaFold-Multimer (v2.3) and AlphaFold3.



 SNP-bearing cells exhibited significantly reduced survival after treatment with irradiation, platinum-based and other chemotherapeutics, and PARP inhibitors (p<0.001, unpaired T-test). Bulk RNA-sequencing revealed downregulation of DNA repair and upregulation in immune response genes in SNP-bearing cells (q<0.05). Upon irradiation, SNP cells showed impaired colocalization of RAP80-BRCA1 foci (p<0.001, Mann-Whitney test), indicating defective homologous recombination. TCGA rectal tumors stratified by KDM3C genotype (A/A, A/T, T/T) were analyzed for COSMIC single-base substitution (SBS) mutational signatures. SBS3, a hallmark of homologous recombination deficiency, was absent in A/A (WT) tumors, modestly present in A/T heterozygotes, and most enriched in T/T homozygotes. AlphaFold2 modeling revealed that KDM3C comprises three structured domains: a N-terminal SH3-like β-barrel, a central zinc-binding module (residues 1694-1938) with a KDM3-specific zinc-binding domain and ZZ domain, and a C-terminal JmjC catalytic domain (residues 2150-2450). The rest of KDM3C including the SNP site, is predicted to be intrinsically disordered. Modeling of RNF8’s FHA domain in complex with KDM3C peptides (Ser/Thr at the SNP site, ±phosphorylation) demonstrated higher predicted binding stability for phosphorylated Thr over Ser (order of stability pT > pS >> T > S), consistent with known FHA domain preferences. Additionally, RNF8 FHA domain is predicted to interact with SH3-like domain of KDM3C, further implicating this polymorphism in modulating DSB repair via BRCA1 recruitment.



 Our findings highlight the functional role of KDM3C SNP in regulating DSB repair and CRT response. This SNP could serve as a stratification marker to enhance CRT efficacy in LARC, HNSCC and potentially other tumors.



 Adria Hasan, Pragya Priyadarshini, Elena V. Demidova, Philip Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya Shah, Gavin Hearne, Mark Andrake, Yan Zhou, Margret B. Einarson, Thomas J. Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, Erica A. Golemis, Johnathan R. Whetstine, Joshua E. Meyer, Sanjeevani Arora. Mechanistic insight into a germline KDM3C polymorphism associated with chemoradiotherapy outcomes in cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c582e66806bb92842892fa0c1e3162cb08cd2c" target='_blank'>
              Abstract A029: Mechanistic insight into a germline
 KDM3C
 polymorphism associated with chemoradiotherapy outcomes in cancer
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Leonny Gathuka, Takahiko Murayama, Shreya M Shah, G. Hearne, Mark Andrake, Yan Zhou, M. Einarson, Thomas J Galloway, Roland Dunbrack, Israel Cañadas, Gail L. Rosen, Barbara Burtness, E. Golemis, Johnathan R. Whetstine, J. E. Meyer, S. Arora
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to establish divergent gene regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under "cellular constraint". We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and noncoding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share signals of positive and negative selection, suggesting a dynamic role in influencing human traits and cellular phenotypes. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd414947d1caaf71463120749f3e2d01e73021fa" target='_blank'>
              Conserved facultative heterochromatin across cell types identify regulatory sequences underpinning cell identity and disease.
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, , Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2025-10-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Epstein-Barr Virus (EBV) establishes life-long latent infection in >90% of adults and is a causal agent for diverse cancers and autoimmune diseases. EBV has a complex life cycle in multiple different tissue types that involve dynamic variations in viral gene expression. These gene expression changes account for the success of the virus in long-term persistence and evading host immune control, as well as its potential for driving cancer evolution and autoimmune disease. Here, we review some of the salient features of EBV gene regulation highlighting the many variations of viral transcription. We review recent advances in our understanding of the factors that bind and regulate EBV gene expression. Based on this diversity of viral transcription patterns, we propose that EBV genome consists of gene modules regulated by local promoter-proximal transcription factor combinations that are further regulated by distal regulatory interactions among the various modules that interact through architectural factors, such as CTCF and cohesion. These modules are likely to represent chromatin architectural domains, and can also interact with host chromosome domains that further regulate viral and host gene expression. We propose that this gene regulatory hierarchy provides EBV with necessary plasticity for viral persistence, as well as a strong potentiator for cancer and autoimmune disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a251ae41cd508722e48d62d5ca0f4e635efbec9f" target='_blank'>
              Transcriptional regulation of gene modules in Epstein-Barr virus.
              </a>
            </td>
          <td>
            Paul M. Lieberman
          </td>
          <td>2025-09-22</td>
          <td>Transcription</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Chromophobe renal cell carcinoma (ChRCC) is the second most common non-clear cell renal cell carcinoma (ncRCC). A series of 496 ChRCC from Memorial Sloan Kettering Cancer Center demonstrated that sarcomatoid features, which were present in about 1.2% of the cases, are strongly correlated with metastatic disease and reduced overall survival. Little is known about the biological and immunological characteristics of the sarcomatoid variant of ChRCC and how it differs from its classic, non-sarcomatoid form. Methods Eight ChRCC tumor samples containing both classic and sarcomatoid components were selected. Xenium Prime 5K assay (5000 curated genes related to oncoimmunology) was used for high-resolution spatial transcriptomic profiling. Data were analyzed using Seurat and custom scripts to annotate cell types and conduct detailed phenotypic analysis. Pathway enrichment and transcription factor regulatory networks were analyzed using the SENIC package. Cell-cell communication was inferred using CellChat, and spatial proximity of immune and tumor cells (“cellular neighborhoods”) was analyzed, which model spatial gene expression patterns and intercellular distances. Results Spatial transcriptomic analyses revealed a significantly more immune-inflamed tumor microenvironment in sarcomatoid regions compared to their classic counterparts. We identified distinct spatial immune “neighborhoods” in sarcomatoid areas that were enriched in CD8+ T cells, polarized macrophages, and dendritic cells. These neighborhoods exhibited organized proximity to tumor cells expressing immune evasion markers such as PD-L1. In contrast, classic regions were relatively immune-deserted. Analysis of transcription factor activity showed upregulation of IRF1 and STAT1-related programs in the immune infiltrates of sarcomatoid regions, consistent with inflammatory activation. Importantly, comparison of matched sarcomatoid and classic regions within the same tumors enabled us to define a “sarcomatoid-enriched gene signature” composed of immune-related, pro-metastatic, and epithelial-to-mesenchymal transition (EMT) genes. This signature was validated in bulk RNA-seq data from The Cancer Genome Atlas (TCGA), where it correlated with poor prognosis in ChRCC patients. Conclusions This study presents the first spatial transcriptomic atlas of sarcomatoid ChRCC in comparison to matched non-sarcomatoid (classic) from the same patient. Profound immune microenvironmental differences were identified between tumor compartments. The sarcomatoid variant exhibits hallmarks of an immune-inflamed but potentially immune-evasive phenotype, including enriched immune infiltration, pro-inflammatory signaling, and EMT-associated transcriptional programs. These findings highlight the biological differences between sarcomatoid and classic ChRCC and uncover specific immune pathways and spatial cell-cell interactions that may be targeted in future immunotherapeutic strategies. Our work also establishes a foundational spatial map for ChRCC that offers new insight into the disease heterogeneity and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aba1786fb42a662d7a501d0a751ee46eaa895c9" target='_blank'>
              52Spatial transcriptomic mapping reveals immune-inflamed niches in sarcomatoid chromophobe renal cell carcinoma
              </a>
            </td>
          <td>
            Yan Tang, Tiegang Han, Hadi Mansour, M. Hirsch, Katrina Collins, Elizabeth P. Henske
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres.
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epithelioid hemangioendothelioma (EHE) is a difficult to treat vascular sarcoma defined by TAZ-CAMTA1 (TC) or YAP-TFE3 (YT) fusion proteins. Human cell lines needed to further understand the pathogenesis of EHE have been lacking. Herein, we describe a method to generate EHE extended primary cell cultures. An integrated multi –omic and functional approach was used to characterize these cultures. The cell cultures, relatively homogenous by single cell RNA-Seq, demonstrated established characteristics of EHE including increased proliferation, anchorage independent growth, as well as the overall gene expression profile and secondary genetic alterations seen in EHE. Whole genome sequencing (WGS) identified links to epigenetic modifying complexes, metabolic processes, and pointed to the importance of the extracellular matrix (ECM) in these tumors. Bulk RNA-Seq demonstrated upregulation of pathways including PI3K-Akt signaling, ECM/ECM receptor interaction, and the Hippo signaling pathway. Development of these extended primary cell cultures allowed for single-cell profiling which demonstrated different cell compartments within the cultures. Furthermore, the cultures served as a therapeutic platform to test the efficacy of TEAD inhibitors in vitro. Overall, the development of primary EHE cell cultures will aid in the mechanistic understanding of this sarcoma and serve as a model system to test new therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4728cebf1ce6a2c3cd2884745cce7881e7bc18e9" target='_blank'>
              EHE cell cultures are a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M Bellizzi, Anthony N. Snow, Patrick Breheny, M. Chimenti, M. Tanas
          </td>
          <td>2025-10-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is the leading cause of death in women under 50. The majority of breast cancers are estrogen receptor α-positive (ER+) and are commonly treated with hormonal therapies such as tamoxifen that inhibit ER activity. The TP53 tumor suppressor gene, encoding the p53 protein, is the most frequently mutated gene in breast cancer, and TP53 mutations are associated with diminished tamoxifen response and worse prognosis for breast cancer patients. Here, we report that in breast cancer cells p53 and ER cooperate to regulate the transcription of a set of genes encoding chromatin modifiers. The net result is a global increase in H3K4me3 and decrease in H3K9me3 chromatin marks. The resultant "open" chromatin is associated with increased transcription of luminal cell identity genes and enhanced tamoxifen sensitivity. Conversely, diminished p53 control of these chromatin modulators is associated with the evolution of tamoxifen resistance and cancer stem cell properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a29e75c12c65ca56f17dc6c777618735a34c52" target='_blank'>
              p53 regulates the expression of histone modifiers to restrict stemness and maintain differentiated luminal identity in breast cancer.
              </a>
            </td>
          <td>
            Yael Aylon, N. Furth, A. Pirona, Adi Lavie, Olga Fedorova, Ori Hassin, Nuno Padrão, Maxime Steinmetz, Avital Sarusi-Potuguez, Liat Fellus-Alyagor, Irit Shimoni, B. Dassa, W. Zwart, E. Shema, Moshe Oren
          </td>
          <td>2025-10-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f13f3a42ad0960ba2f1336cd288a90d6e8ad4e1f" target='_blank'>
              Opposing Copy Number Variation Dynamics Shape Adaptation to Glucose and Galactose in Diploid Yeast
              </a>
            </td>
          <td>
            Prachitha Nagendra, Saket Choudhary, S. Saini
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract INTRODUCTION Faulty DNA repair and epigenetic regulation contribute to neurodegeneration in Alzheimer's disease. Long‐read sequencing enables analysis of “dark regions” that are difficult to study via traditional sequencing methodologies. METHODS Using nanopore whole‐genome DNA sequencing of post mortem brain from early‐ and late‐ stage Alzheimer's disease cases and controls, we analyzed retrotransposition, non‐allelic homologous recombination (NAHR), structural variation, and DNA methylation within repetitive regions. RESULTS Retrotransposon insertions and NAHR were enriched in centromeric/pericentromeric and ribosomal DNA (rDNA) in the aged brain. Putatively somatic AluY retrotransposon insertions trended upward in late‐stage disease. Enrichment of NAHR between repetitive regions and DNA demethylation were detected in early disease. Differential methylation of dark regions within specific Alzheimer's disease risk genes and repetitive elements occurred across disease stage. DISCUSSION This study provides the first long‐read analysis of repetitive elements in the aged human brain and identifies these regions as hotspots for genomic variation in Alzheimer's disease. Highlights Long‐read sequencing enables analysis of “dark” regions Alzheimer's disease brains Somatic AluY retrotransposon insertions may be elevated in late‐stage disease Retrotransposon‐associated non‐allelic homologous recombination (NAHR) is enriched in repetitive regions in early disease DNA demethylation within the centro/pericentromere and ribosomal DNA (rDNA) occur across disease stage Dark regions of risk genes are differentially methylated across disease stage">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f2a36cf2db9b5da1ca7fa6928e8ec70338fa2ed" target='_blank'>
              Long‐read sequencing reveals genomic and epigenomic variation in the dark genome of human Alzheimer's disease
              </a>
            </td>
          <td>
            Paulino Ramirez, Wenyan Sun, Shiva Kazempour Dehkordi, Habil Zare, G. Pascarella, P. Carninci, Bernard Fongang, Kevin F. Bieniek, Bess Frost
          </td>
          <td>2025-11-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3160e385fe96505bcb09a45df54fdae63937a125" target='_blank'>
              NONO as a Sensor of Intracellular Oxidation: Relevance to Neuroblastoma Cell Death
              </a>
            </td>
          <td>
            S. S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BRCA1/2 are crucial in the homologous recombination repair (HRR) pathway, with loss-of-function (LoF) alterations predicting sensitivity to PARP-inhibitors (PARPi). Whether other HRR-gene alterations confer PARPi sensitivity remains unclear. In the Drug Rediscovery Protocol, patients receive off-label drugs matched to their tumor molecular profile. Here, olaparib efficacy and safety were evaluated in adult patients with treatment-refractory, progressive malignancies harboring LoF alterations in ATM (cohort A) or other HRR-genes including CDK12, PPP2R2A, CHEK1/2, and RAD51B (cohort B). Primary endpoints were clinical benefit (CB: confirmed objective response or stable disease ≥16 weeks) and safety. Pre-treatment biopsies were analyzed by whole-genome sequencing (WGS) for target validation. CB was observed in 8/25 patients (32%) in cohort A (prostate cancer: n = 6, adenoid cystic carcinoma: n = 1, endometrial cancer: n = 1). No effectiveness was seen in patients with colorectal cancer (n = 8). Median progression-free survival (PFS) and overall survival (OS) were 3.4 months (95% CI 1.8-5.3) and 9.2 months (95% CI 5.2-21.3), respectively. In cohort B, the CB rate was 41.7% (10/24) with median PFS and OS of 3.5 months (95% CI 3.4-6.6) and 8.1 months (95% CI 6.6-14.2), respectively. CB was observed in CKD12 (n = 7), RAD51B (n = 2), and CHEK2-altered tumors (n = 1), but not in PPP2R2A (n = 6) or CHEK1-altered tumors (n = 1). No unexpected toxicities occurred. WGS confirmed inclusion target in 84% of tested patients. In conclusion, PARPi sensitivity varies across HRR-genes, indicating that relying solely on an altered common mechanistic pathway is insufficient to predict response. Future studies should target specific HRR-genes to assess subgroup-specific benefits and determine proper use of molecular diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/307d50a92ba3f268facbb2ba41b55d6d57fef8c8" target='_blank'>
              Olaparib for patients with tumors harboring alterations in homologous recombination repair genes: Results from the drug rediscovery protocol.
              </a>
            </td>
          <td>
            I. A. C. Spiekman, N. Mehra, L. Zeverijn, B. Geurts, K. Verkerk, S. H. Haj Mohammad, V. van der Noort, P. Roepman, W. D. de Leng, A. Jansen, A. V. D. van de Luijtgaarden, T. van Voorthuizen, T. Buffart, H. Gelderblom, E. Voest, Henk M. Verheul
          </td>
          <td>2025-10-16</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="With the increase in dog life expectancy, canine mammary cancer has become the most critical and frequent neoplasm in dogs. As in humans, this type of cancer has spontaneous development and is influenced by several risk factors, such as age and hormonal exposure in addition to genetic and epigenetic factors. Thus, knowing that epigenetic patterns influence a variety of biological mechanisms, such development cancer, this study aimed to correlate the methylation pattern of LINE-1 elements in healthy and neoplastic canine mammary tissue, as well as in isolated cell lines of carcinoma canine with tumoral aggressiveness degree. To this end, a retrospective study was conducted in which genomic DNA was extracted from histological sections of biopsies and cell lines using the phenol/chloroform method. To determine the percentages, the genomic DNA was pretreated with methylation-sensitive endonucleases and then amplified by qPCR with primers specific for the L1td1 gene. The results revealed that both the tumor cell lines and the canine adenoma and inflammatory carcinoma samples are hypomethylated relative to normal tissue. However, it was not possible to confirm the relationship between the percentage of LINE-1 hypomethylation and tumor aggressiveness in canine breast cancer. Therefore, the combined analysis of our results allows us to conclude that the methylation pattern of LINE-1 elements can be used as a molecular biomarker to differentiate normal canine mammary tissue from neoplastic canine mammary tissue. However, to confirm whether there is a relationship between the degree of methylation of LINE-1 elements and tumor staging, further studies with canine samples are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3e474a09cd5273ac4d9d3cd5402967aa34da4d" target='_blank'>
              ASSOCIATION BETWEEN LINE-1 HYPOMETHYLATION IN CANINE MAMMARY TUMOR TISSUES: IMPLICATIONS FOR DIAGNOSIS AND RESEARCH
              </a>
            </td>
          <td>
            Patrícia A. dos Santos de Carvalho Joaquim, José Guilherme Xavier, M. Lallo, Rodrigo A. Silva
          </td>
          <td>2025-09-24</td>
          <td>Revista CPAQV - Centro de Pesquisas Avançadas em Qualidade de Vida</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reversine is a small-molecule Aurora kinase inhibitor known for its pro-apoptotic effects and potential to remodel chromatin architecture. Although its impact on mitotic regulation is established, its effects on telomere dynamics and nuclear organization in chronic myeloid leukemia (CML) remain unclear. This study aimed to investigate the effects of reversine on telomere architecture, genomic instability, and apoptosis in CML cell lines (K-562 and MEG-01). Reversine was applied at increasing concentrations, and cytotoxicity was assessed using caspase-3/7 activation assays. Quantitative PCR was used to measure AURKA and AURKB mRNA expressions. Three-dimensional telomere architecture was analyzed with TeloView® v1.03 software after Q-FISH labeling to quantify telomere number, signal intensity, aggregation, nuclear volume, and a/c ratio. Reversine induced a dose- and time-dependent apoptotic response in both cell lines and significantly downregulated AURKA and AURKB expressions. Three-dimensional telomere analysis revealed a marked reduction in telomere number and aggregates, signal intensity, and nuclear volume. While reduced signal intensity may indicate telomere shortening, the concurrent decrease in aggregation and altered spatial parameters suggests telomeric reorganization rather than progressive instability. These features reflect structural nuclear remodeling and early apoptotic commitment. Differences between K-562 and MEG-01 responses underscore potential heterogeneity in telomere maintenance mechanisms. Reversine modulates genomic stability in CML cells through dual mechanisms involving Aurora kinase inhibition and telomere architecture remodeling. The integration of 3D telomere profiling highlights reversine’s potential as a therapeutic agent targeting nuclear disorganization and mitotic dysregulation in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e06bedd101bf31c7285fbbc7d8f35b57bf8952e" target='_blank'>
              Reversine-Induced Telomere Architecture Remodeling in Chronic Myeloid Leukemia Cell Lines: Insights from TeloView® Analysis of 3D Nuclear Architecture
              </a>
            </td>
          <td>
            Fábio Morato de Oliveira, Isabela Dias Cruvinel, Bruno Machado Rezende Ferreira, Sabine Mai
          </td>
          <td>2025-10-31</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceefcc8e25ead58e8d1394b9af76c598697ce059" target='_blank'>
              Nuclear Localization Signals Enable the Cellular Delivery of an Anti-CRISPR Protein to Control Genome Editing
              </a>
            </td>
          <td>
            A. Vera, Franklin J. Avilés-Vázquez, T. Ha, Amit Choudhary, Ronald T. Raines
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction DNA double-strand breaks (DSBs) are repaired via homologous recombination (HR) or the more error-prone non-homologous end joining (NHEJ). breast cancer gene 1 (BRCA1) and breast cancer gene 2 (BRCA2) are key genes in HR, and their mutations are associated with aggressive prostate cancer (PCa). While PARP inhibitors (PARPi) improve survival in BRCA-mutated PCa, their efficacy in late-stage disease is limited and often accompanied by serious side effects. This study aims to develop an in vitro model of BRCA-mutated PCa and evaluate the therapeutic potential of DNA-dependent protein kinase (DNA-PK) inhibitors that target the NHEJ pathway. Methods The genes BRCA1 and BRCA2 were targeted for knockout (KO) in lymphnode cancer of the prostate (cell line) [LNCaP] using clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9 technology. The KO were assessed via Western blot analysis. Scramble LNCaP, BRCA1 KO, and BRCA2 KO cells were treated with the PARPi talazoparib, the DNA-PK inhibitor nedisertib and their combination. The impact on cell proliferation was evaluated using the CellTiter-Glo assay and synergy of the treatments was analyzed with SynergyFinder. Cytotoxic effects were measured by flow cytometry using an Annexin V-fluorescein isothiocyanate (FITC) apoptosis detection kit. The presence of DSB was quantified using immunofluorescence. Results BRCA2 and BRCA1 protein expression were successfully downregulated in the knockout (KO) cell lines. After two days of treatment with talazoparib and/or nedisertib, a significant decrease in cell proliferation was observed. Additive effects of the combination treatment were detected exclusively in the BRCA KO cells. These cells also exhibited significantly higher rates of necrosis after treatment compared to scramble cells and, DNA DSB were significantly more prevalent in the BRCA KO cells. Additionally, BRCA1/2 loss is inversely correlated with DNA-PK, with knockout leading to increased DNA-PK expression to support NHEJ. Conclusion BRCA knockout LNCaP models were established, exhibiting increased DNA-PK activity and indicating greater susceptibility to DNA-PK inhibition. Therapeutically targeting NHEJ presents a promising approach in treating BRCA-mutated PCa. Further in vivo investigations are required to assess the tolerability of this drug combination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a995f399e397271dd4a847fb2e46b35d96ad921b" target='_blank'>
              Combined DNA-PK and PARP Inhibition as a Therapeutic Strategy in BRCA-Mutated Prostate Cancer: An in Vitro Pilot Study
              </a>
            </td>
          <td>
            T. Scherer, S. Salemi, Valentin Baumgartner, Dominik Enderlin, Alekhya Mazumdar, Daniel Eberli
          </td>
          <td>2025-11-14</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="As a highly invasive malignant tumor, pancreatic cancer (PC) has an increasing incidence rate and a long-term low 5-year survival rate. At present, CRISPR has been used in the research of PC to analyze the function of oncogenes, explore drug resistance mechanisms and develop treatment strategies. Most of them are in the stage of cell and animal experiments. However, there are still research gaps in tumor microenvironment penetration, low delivery efficiency in vivo, off target effects and treatment limitations caused by tumor heterogeneity. This article systematically reviews the CRISPR foundation, analyzes the pathogenesis of high-frequency mutant genes in PC, elaborates three major treatment strategies based on CRISPR, combs the progress of second-generation CRISPR, clarifies the current technical challenges, and proposes targeted vector development, high fidelity Cas protein application and other corresponding programs. This study provides a comprehensive reference for the clinical transformation of CRISPR in the field of PC, helping to break through the bottleneck of traditional treatment; However, the long-term safety of this technology in the human body still needs to be verified. Future research can rely on the third-generation CRISPR to optimize the targeted delivery system, explore the joint strategy with immunotherapy and chemotherapy, promote the technology to clinical implementation, and provide more accurate treatment schemes for PC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1280741d28338ece3f7cdcb3f5fc9bdf722360" target='_blank'>
              Application of CRISPR in the Treatment of Pancreatic Cancer
              </a>
            </td>
          <td>
            Zixi Tian
          </td>
          <td>2025-10-23</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aging constitutes a major risk factor for pan-cancer development, with epidemiological studies indicating that 60% of new malignancies occur in adults age 65 and older. This review synthesizes cutting-edge insights from single-cell sequencing databases (e.g., TCGA and GEO) that decipher how aging reprograms the tumor microenvironment (TME) to fuel carcinogenesis. Single-cell RNA sequencing (scRNA-seq) has revealed that senescent cell subpopulations (e.g., CDKN2A+/LMNB1- cells) accumulate in aged tissues at frequencies up to 15%, driving genomic instability and secrete pro-tumorigenic senescence-associated secretory phenotype (SASP) factors (IL-6 and TGF-β). These factors remodel the TME by inducing fibroblast activation and extracellular matrix degradation, accelerating metastasis by 40-70% in murine models. Crucially, immunosenescence diminishes anti-tumor immunity, with scRNA-seq profiling showing 40-60% increases in exhausted PD-1+ T cells and immunosuppressive myeloid cells in aged TMEs. Pan-cancer analyses have identified conserved aging gene signatures (e.g., p16INK4a upregulation in 12+ cancer types) that correlate with 30-50% poorer survival. While technical challenges persist - including batch effects in scRNA-seq data and low senescent cell abundance (< 5%) - emerging solutions like deep learning can enhance detection sensitivity. Therapeutically, senolytic strategies deplete senescent cells, improving drug response by 3.5-fold in preclinical trials. Future research must integrate multi-omics and AI to examine aging-related targets, advancing personalized interventions for aging-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e67442df6e2549af855ce3f0cc864e0099eaa2" target='_blank'>
              Deciphering the nexus of aging and pan-cancer: Single-cell sequencing reveals microenvironmental remodeling and cellular drivers.
              </a>
            </td>
          <td>
            Yue Han, Nuo Chen, Ping Wang, Chu Zhou, Kenji Karako, Peipei Song, Wei Tang
          </td>
          <td>2025-10-26</td>
          <td>Bioscience trends</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Radiation therapy is a crucial component of the standard of care for newly diagnosed glioblastoma patients. However, the majority of tumors inevitably recur, most within the prior radiation field. Despite its broad clinical application, the molecular mechanisms that mediate response to radiation therapy in glioblastoma remain poorly understood. To address this knowledge gap, we systematically profiled the radiation sensitivity of four glioblastoma cell lines (T98G, SF295, LN18, DBTRG) and four patient-derived neurospheres (BT112, BT145, BT359, CPDM0095). The neurospheres were characterized at the genomic, transcriptomic and clinical levels. These models exhibited a spectrum of responses to radiation, with LD50s ranging from 1.5 to 8 Gy. To identify molecular factors that sensitize glioblastoma cells to radiation, we conducted a genome-wide CRISPR inactivation screen in T98G cells treated with or without a sub-lethal dose of radiation. Cells were transduced with the Inzolia multiplex CRISPR/Cas12a knockout library, targeting 19,687 single genes, 4,435 paralog pairs, 376 paralog triples, and 100 paralog quads. Cells were then divided into a control condition and an irradiated condition (3 Gy). After 18 days in culture, genomic DNA was harvested, and the relative abundance of each guide RNA array was compared between the two conditions. Targets preferentially depleted in the irradiated condition included known radiation sensitizers such as ATM and PRKDC, validating the screen’s performance. Gene set enrichment analysis (KEGG) of the top 200 hits revealed enrichment of DNA repair pathways, including homologous recombination and non-homologous end joining, as well as pathways related to the spliceosome, cell cycle, endocytosis, ubiquitin-mediated proteolysis, and pyrimidine metabolism. Notably, three components of the translesion synthesis pathway (REV1, REV3L, MAD2L2), not previously implicated in radiation response, also emerged as top candidates. A mini-pool library of the top 300 hits was subsequently generated and is being used for validation in additional glioblastoma cell lines and neurospheres. These findings provide a foundation for uncovering novel mechanisms of radiation response and nominate new targets for potential radiosensitization strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15dea4ab418adfb3586fa12497a41869a83e604c" target='_blank'>
              DNAR-13. Uncovering novel mediators of radiation sensitivity in glioblastoma using functional genomic screening
              </a>
            </td>
          <td>
            E. Aquilanti, Vincent Bozinov, Matthew L. Meyerson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This study proposes the implementation of clustered regularly interspaced short palindromic repeats (CRISPR)–CRISPR-associated protein 9 (Cas9) technology for gene therapy targeting genetic mutations in human lymphocytes affected by chronic lymphocytic leukemia (CLL), offering new opportunities for effective treatment of this heterogeneous disease. It focuses on the application of CRISPR–Cas9-mediated targeted sequencing to systematically characterize the biological effects of monoallelic and biallelic TP53 gene lesions, aiming to replace mutant TP53 genes in CLL cells through this technology. CRISPR–Cas9 technology employs a specific enzyme guided by a designed guide RNA (gRNA) to a DNA target. The enzyme first introduces a cut at the target site, and following this cleavage event, it can further disrupt the TP53 gene. The gRNA plays a crucial role by directing the Cas9 protein to the DNA sequence of interest. The gRNA consists of CRISPR RNA (crRNA) and trans-activating CRISPR RNA (tracrRNA) sequences, responsible for target recognition and Cas9 binding, respectively. Examination of the predicted secondary structure of the tracrRNA–crRNA duplex suggests that the features required for Cas9-catalyzed DNA cleavage at specific sites can be captured within a single chimeric RNA. Although the natural tracrRNA–crRNA mechanism operates efficiently, the use of a single RNA-guided Cas9 system is particularly attractive due to its potential for programmed DNA cleavage and genome editing. Importantly, Cas9 can bind and cleave a target sequence only if it is adjacent to a protospacer adjacent motif. Once the gRNA–Cas9 complex binds to the target DNA, Cas9 induces a double-strand break at the specified site. In conclusion, CRISPR–Cas9 technology represents a powerful genetic engineering tool capable of inserting, deleting, or replacing DNA within an organism’s genome using these “molecular scissors.”">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c50ea712276d8879d783d2204b12ae8d8ebc33" target='_blank'>
              Application of CRISPR–Cas9 technology in the treatment of chronic lymphocytic leukemia with TP53 mutations
              </a>
            </td>
          <td>
            A. Udristioiu, Liviu Martin, Delia Nica-Badea, A. V. Tetileanu, Manole Cojocaru, I. Gheorghe
          </td>
          <td>2025-10-28</td>
          <td>Cancer Plus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Supratentorial ependymomas are often life-threatening brain tumors that are characterized by significant molecular heterogeneity. Hypoxia contributes to this heterogeneity, but the extent and nature of this relationship remain unclear. This poses challenges for effective treatment strategies, demanding a more holistic and detailed understanding of the underlying biology. In this study, we employed single-nucleus (n = 63) and spatial (n = 30) transcriptomics to delineate the cellular and spatial landscape of supratentorial ependymoma. Characterizing the transcriptional consensus programs of this entity revealed two previously undescribed programs that are associated with the remodeling of the extracellular matrix and the ZFTA fusion identity. These programs were related to highly unfavorable outcome and were exclusive to ZFTA fusion-positive tumors. Developing a set of spatial scoring algorithms that are sensitive to mixed spot transcriptomes enabled the identification of the role of hypoxia as the potential driver of spatial organization. Together with recurrent spatial associations inferred between programs, this informed a generalized model of the higher-order architecture. The functionally and spatially defined zones of this model displayed distinct immune presence, with the highest seen in the hypoxic zone. These results provide insights into the molecular heterogeneity and spatial organization of supratentorial ependymomas, which may aid in the development of targeted future treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6575f9d3eac6558b40e6240ea8f2f812824b5235" target='_blank'>
              Integrated Single-Nucleus and Spatial Transcriptomics Elucidate Heterogeneity and Hypoxia-Driven Organization of Supratentorial Ependymoma.
              </a>
            </td>
          <td>
            Erik Schüftan, I-Na Lu, Flavia W de Faria, Clara Inserte, C. Rössig, Julia Bothe, Adrienn Bagosi, S. Tippelt, Tobias Blau, Camelia Monoranu, L. Altendorf, D. Spille, Ann-Katrin Bruns, A. Brentrup, Walter Stummer, D. Gorodezki, Martin Ebinger, J. Schittenhelm, C. Thomas, T. Albert, U. Schüller, K. Kerl
          </td>
          <td>2025-10-28</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Embryonal tumors with multilayered rosettes (ETMR) are rare pediatric brain cancers with a median survival of 14 months despite aggressive treatment. The oncogenic driver in ~90% of cases is an amplification and fusion of the chromosome 19 micro-RNA cluster (C19MC), which is yet undruggable. Due to the tumor’s rarity, comprehensive molecular characterization and the identification of therapeutic targets are challenging, and ETMR remains poorly understood.
 We used single cell RNA sequencing and multiplexed protein profiling to study intratumoral heterogeneity and oncogenic signaling in ETMR (n=11). MiRNA-mRNA interactions were investigated by enhanced crosslinking and immunoprecipitation (eCLIP) followed by next-generation sequencing. ETMR cells were treated with locked nucleic acids (LNAs) and small-molecule inhibitors, and their anti-tumor effects assessed through confluency and death assays. Here, we identified three distinct malignant ETMR cell types resembling differentiation stages during early fetal neocortex development: undifferentiated ETMR cells resembling neural-stem cells (NSC), differentiated cells corresponding to mature neurons, and intermediate cells exerting markers of neuronal lineage commitment. C19MC was overexpressed in mitotically active NSC-like cells. Using eCLIP, we uncovered widespread regulatory roles of individual C19MC members. Markers of neuronal differentiation (NR2F2, SOX4, SOX11) and lineage commitment (C2–H2 ZNF family members) were targets of abundant C19MC members. LNA-mediated blockade of key C19MC members significantly reduced ETMR cell confluency and induced apoptosis. Genome-wide cell-cell communication analysis identified fibroblast growth factor receptors (FGFRs) and NOTCH receptors as key mediators of cellular cooperation. Accordingly, FGFR and NOTCH inhibitors had significant anti-tumor effects in vitro, and erdafitinib-supplemented therapy led to partial response in one therapeutically exhausted ETMR patient. Together, we demonstrated intratumoral heterogeneity in ETMR paralleling fetal neurodevelopment. Targeting C19MC in undifferentiated NSC-like cells significantly impaired cell confluency and survival. Additionally, FGFR emerged as a clinically actionable target, showing promising anti-tumor activity in vitro and in one ETMR patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb99e7f0c8810ec820bdec1d791a67a355e38fb" target='_blank'>
              EXTH-92. Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor-ligand interactions
              </a>
            </td>
          <td>
            L. Gabler-Pamer, Alexander Beck, G. A. V. Cruzeiro, S. Lambo, McKenzie L. Shaw, Olivia A Hack, Andrezza Nascimento, S. Madlener, Daniela Lötsch, I. Slavc, C. Dorfer, C. Haberler, Lissa C. Baird, Susan N. Chi, S. Alexandrescu, J. Gojo, Kool Marcel, V. Hovestadt, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND
The integration of regional hyperthermia into the multimodal treatment of patients with localized high-risk soft tissue sarcomas has been shown to improve overall survival. However, specific effects on the tumors' genetic makeup and biology are largely unknown. Since clonal selection of malignant cells capable of thermoresistance might contribute to disease progression, a better understanding of the induced tumor evolution could inform strategies for improving treatment efficacy.


METHODS
We performed whole exome sequencing on paired sarcoma samples obtained before and after treatment with chemotherapy combined with regional hyperthermia (n = 12 patients; n = 9 paired samples, n = 3 post-treatment only).


RESULTS
Tumor evolution was evident in all paired samples, with shifts in small- and large-scale genomic alterations. Overall, these alterations appeared tumor-specific, but recurrent mutations in histone H3-modifying genes were found exclusively in post-therapeutic samples.


CONCLUSION
Our findings showcase the diversity of genomic evolution patterns in sarcomas emerging under combined thermo- and chemotherapeutic selection pressure, indicating that treatment response may vary according to the specific tumor composition. With previous studies linking histone functions to heat stress response, the alterations found in genes modifying H3 in our post-therapeutic cohort provide biological evidence for synergy of chemotherapy combined with regional hyperthermia in soft tissue sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4f8170d2e2ef05cf601f2f68d874581f9fc221" target='_blank'>
              Genomic evolution of high-risk soft tissue sarcomas under thermo- and chemotherapeutic selection pressure.
              </a>
            </td>
          <td>
            M. Schmiester, Antonia Freytag, , E. Sträng, Raphael Hablesreiter, Antje Richter, J. Striefler, Maximilian von Laffert, Michael Hummel, A. Jarosch, David Kaul, P. Ghadjar, R. Öllinger, Daniel Rau, Sven Märdian, Lars Bullinger, A. Dolnik, A. Flörcken
          </td>
          <td>2025-11-04</td>
          <td>International journal of hyperthermia : the official journal of European Society for Hyperthermic Oncology, North American Hyperthermia Group</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase inhibitors (PARPis) have improved the prognosis of patients with high-grade serous ovarian carcinoma (HGSOC). However, PARPis are not effective for all HGSOC patients. Spatial transcriptomics is a powerful tool for characterizing the tumor microenvironment. We used Visium to analyze eight tumor samples from HGSOC patients with clinical information on PARPi sensitivity. Two complementary analyses were performed: an integrated analysis across all samples, without considering spatial information, and an analysis with spatial information within each sample. Both approaches indicated that midkine (MDK) signaling is involved in PARPi resistance. Furthermore, we identified receptors that enhance MDK signaling in cancer cells. To assess the generalizability of this finding, we deconvolved bulk RNA-sequencing data using single-cell RNA-sequencing data as a reference to examine the relationship between receptor expression levels and overall survival (OS). This analysis revealed that high SDC4 expression in cancer cells is associated with poor OS in HGSOC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6e6aeb1b5b6668556eb27188c5496dc383c124" target='_blank'>
              Exploring drug resistance via intercellular crosstalk using spatial transcriptomics in high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Hironori Suzuki, Kosuke Yoshida, Akira Yokoi, Kazuhiro Suzuki, Yuta Hirano, Masami Kitagawa, Eri Asano-Inami, M. Yoshihara, Yukari Nagao, S. Tamauchi, N. Yoshikawa, Hiroaki Kajiyama, Yusuke Yamamoto
          </td>
          <td>2025-11-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Metazoans suppress transposons via small RNA–mediated silencing, but ciliates physically eliminate transposons from somatic genomes. Here, we report the high-quality germline genome assembly of a marine ciliate Euplotes vannus, where ~80% are transposons and other germline-specific DNA that has to be precisely eliminated during soma development. We demonstrate that a class of soma-derived 30-nucleotide small RNAs precisely targets nontransposon DNA against elimination during this process. Small RNA–mediated targeting remains functional across varying small RNA lengths and is compatible with heterozygous sites. These small RNAs are cleaved by Dicer-like ribonuclease from long noncoding RNAs, which are bidirectionally transcribed of somatic chromosomes, initiated at subtelomeric chromosome breakage sequences (5′-TTGAA-3′). On the basis of these findings and time-course transcriptomic profiling, we propose a model elucidating the molecular mechanism of DNA elimination in Euplotes. These findings provide insights into the role of small RNAs in transmitting genetic information across generations and maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42400a7ce7bd33d02826f168bc703d3c0d0af547" target='_blank'>
              Soma-derived 30-nt small RNAs are coupled with chromosome breakage and precisely target nontransposon DNA against elimination in Euplotes vannus
              </a>
            </td>
          <td>
            Li-Yang Lyu, Bijia Ding, Jinyu Fu, I. Rzeszutek, E. Swart, M. Nowacki, Feng Gao
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="
 Telomerase Reverse Transcriptase promoter (TERTp) mutations enable tumor cell immortality in millions of cancer patients annually. TERTp mutations are the most common non-coding mutations across all cancers, including glioblastoma, oligodendroglioma, medulloblastoma, and high-grade meningioma. A multitude of TERTp mutations, including the two hotspots, create de novo E26 transformation specific transcription factor binding sites. Among the 28 family members, we have shown that only the GA-binding protein (GABP, composed of DNA binding GABPA and transactivating GABPB subunits) activates the mutant TERTp. Prior therapies targeting the telomerase RNA component, TERC, lacked tumor selectivity and were poorly tolerated. The GABP-mediated reactivation of the mutated TERTp presents a unique therapeutic opportunity for tumor specific reversal of cellular immortality. Breakthroughs in the design of biological proteolysis-targeting chimera (bioPROTACs) offer a new approach to selectively degrade previously intractable targets such as transcription factors. We combined in silico protein-protein interaction modeling via AlphaFold and experimental validation to identify a minimal GABPB to bind GABPA and fused it with an E3 ubiquitin ligase. Introduction of this GABPA bioPROTAC into TERTp mutant glioblastoma cells depleted GABPA protein and eliminated GABPA binding to the mutant TERT promoter, reducing transcriptional activating (H3K4me3) and increasing suppressive (H3K27me3) histone marks. Reversion of mutant TERTp to an epigenetically silenced state reduced TERT expression by 73% to 95% and shortened tumor cell telomeres in a promoter mutation-specific manner. The GABP bioPROTAC reduced tumor growth and improved survival of mice bearing an orthotopic xenograft of TERTp mutated glioblastoma cells. The effects of the bioPROTAC were validated in vivo via magnetic resonance imaging of metabolic correlates of TERT expression. These data demonstrate that this GABP bioPROTAC potently degrades GABPA, leading to tumor-specific silencing of telomerase expression, and a reversal of glioblastoma tumor cell immortality. This artificial intelligence-guided approach may be applicable to other intractable cancer specific targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102d24aea88be03925f3847c09d31dfd2d63944f" target='_blank'>
              EXTH-77. An AI-engineered bioPROTAC reverses immortality of
 TERT
 promoter mutant tumors
              </a>
            </td>
          <td>
            Nicholas O. Stevers, Ellen Chen, Joshua S. Wu, T. Patel, Sara Collins, M. Montoya, Samuel H. Wu, G. Batsios, A. Gillespie, Chibo Hong, Noriyuki Kasahara, Pavithra Viswanath, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Colibactin is a small genotoxic molecule of polyketide produced by a subset of enteric bacteria including certain Escherichia coli (E. coli) harbored in the human gut microbiota. Its biosynthesis is governed by a multistep enzymatic process encoded by the polyketide synthase (pks) gene cluster. Colibactin is thought to exert its carcinogenic potential primarily through the induction of DNA interstrand crosslinks (ICLs); however, the precise mechanisms underlying its genotoxicity remain largely unresolved. In this study, we focused on ICL formation and its associated repair pathways to investigate whether colibactin-induced ICLs play a central role in the induction of chromosomal aberrations and inhibition of cell proliferation. HAP1 cells deficient in FANCD2, a gene essential for ICL repair, and their wild-type counterparts were infected with colibactin producing (clb⁺) E. coli strains isolated from a Japanese colorectal cancer (CRC) patient. Following recovery culture, the frequency of micronucleated (MN) cells was assessed. The results showed that FANCD2-deficient cells exhibited a significantly higher frequency of MN cells compared to wild-type cells. Additionally, the cytotoxicity of the clb⁺ strains was evaluated using the XTT assay. FANCD2-deficient cells demonstrated higher sensitivity to the clb⁺ E. coli strains than wild-type cells. These findings suggest that colibactin, produced by clb⁺ E. coli, can play a role in the formation of ICLs, thereby contributing significantly to the induction of chromosomal aberrations and the inhibition of human cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5434e698c009fe5d0b86871ed28e1115dcedc77" target='_blank'>
              DNA cross-link repair deficiency enhances human cell sensitivity to colibactin-induced genotoxicity
              </a>
            </td>
          <td>
            M. Kawanishi, Osamu Tsubohira, Ai Ueshima, Yuuta Hisatomi, Yoshimitsu Oda, Michio Sato, Noriyuki Miyoshi, M. Mutoh, Hideki Ishikawa, Keiji Wakabayashi, Takashi Yagi, Kenji Watanabe
          </td>
          <td>2025-09-22</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 The SJMB12 (NCT01878617) clinical trial introduced a molecular subgroup-based approach to the treatment of newly diagnosed medulloblastoma by stratifying 649 evaluable patients into WNT, SHH, and non-WNT/non-SHH strata. While the genomic landscape of medulloblastoma has been extensively characterized, most prior studies analyzed heterogenous or non-trial-associated cohorts, which may not capture the true distribution of oncogenic drivers within a clinically homogeneous population. To address this limitation, we used 1) Illumina EPIC DNA methylation array for subgroup/subtype classification and detection of copy number alterations; 2) whole exome sequencing for detection of single-/multi-nucleotide variants and small insertions/deletions; 3) RNA sequencing for detection of gene overexpression and fusions. In the WNT-stratum (n=90), 96% of profiled tumors harbored canonical exon 3 hotspot mutations in CTNNB1, while the remaining 4% exhibited inactivation of APC, consistent with aberrant WNT/β-catenin pathway activation. Additionally, we identified mutually exclusive loss-of-function mutations affecting several genes on chromosome 6 - LATS1 (8%), FOXO3 (8%), and CSNK2B (9%), co-occurring exclusively with monosomy 6. In the SHH-stratum (n=107), 51% of tumors exhibited PTCH1 inactivation, which was the most prevalent gene-level alteration. Germline loss-of-function variants in ELP1 or TP53 were observed in ~30% of cases, highlighting the clinical utility of germline testing. Three patients exhibited hypermutator phenotypes, with tumor mutational burdens 100x higher than the cohort median. These individuals carried pathogenic germline variants in mismatch repair genes MSH6 or PMS2, consistent with Lynch/Turcot type I syndromes. Notably, two of the three patients also harbored somatic POLE mutations, likely contributing to replication error–driven mutagenesis. In the nonWNT/nonSHH stratum (n=452), beyond known drivers, we identified a novel mechanism of GFI1B enhancer hijacking resulting from an interchromosomal translocation, as opposed to proximal structural variants. The molecular results presented here will be further integrated with survival outcomes and used to inform the design of future clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a482879e34550b636d8ae270c52f70583f96905e" target='_blank'>
              CTP-19. Molecular characterization of the SJMB12 clinical trial for children and young adults with newly diagnosed medulloblastoma
              </a>
            </td>
          <td>
            David Filipovic, Kyle S Smith, S. Dhanda, Doug Strother, Micheal J Fisher, A. Bendel, Eugene I Hwang, Sibo Zhao, N. Gottardo, Tim Hassall, Jennifer D. Elster, S. Perreault, Vijay Ramaswamy, S. Partap, S. Laughton, Richard Cohn, M. Chintagumpala, G. McCowage, Michael Sullivan, Elias Sayour, Daniel C Bowers, Asher Marks, Avery C Wright, A. Onar-Thomas, Amar Gajjar, G. Robinson, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Abstract Background Unrepaired DNA damage leads to the accumulation of cytosolic DNA and the activation of cGAS-STING-mediated inflammatory response, which modulates the tumor immune microenvironment and immunotherapy response. NPRL2 (NPR2 like, GATOR1 complex subunit) is located on chromosome 3p. Deep deletion of NPRL2 was identified in 10% of ccRCC, and is often mutually exclusive with loss of PBRM1, SETD2, and BAP1. However, it remains unclear whether NPRL2 loss independently influences DNA damage and anti-cancer immunity in ccRCC. Methods Both lentiviral shRNA-mediated knockdown and CRISPR/Cas9-mediated knockout were performed in human and murine RCC cell lines to suppress NPRL2 expression. The immunocompetent Renca-Balb/c murine RCC model was used to evaluate therapy response. The tumor immune microenvironment was analyzed using gene expression-based inference and immunohistochemistry (IHC). Immunofluorescence, western blotting and real-time PCR were used for molecular studies. Results Guided by our reverse phase protein array (RPPA) results, we first found that NPRL2 loss activated the ATM-CHK2 DNA damage response pathway and led to the accumulation of cytosolic DNA. The lysosome is known to play important roles in DNA repair and cytosolic DNA degradation. We further found that NPRL2 loss promoted the phosphorylation of TFEB, the master regulator of lysosome biogenesis, and thus prevented its translocation to the nucleus. Pharmacologic activation of TFEB with ML-SA1 reduced ATM phosphorylation and cytosolic DNA accumulation, and lysosomal inhibition by chloroquine or bafilomycin A1 yielded the opposite results. However, genetic inhibition of autophagy by knocking down ATG5, ATG7, BECN1, or p62 had no apparent impact. The increased cytosolic DNA led us to hypothesize that NPRL2 deficiency would consequently activate the cGAS-STING pathway. Unexpectedly, NPRL2 deficiency suppressed the cGAS-STING-mediated inflammatory response, as evidenced by dramatic decreases in TBK1 and IRF3 phosphorylation, and in downstream gene expression (eg, CCL5, CXCL10 and IFNB1). To investigate the underlying mechanism, we first analyzed the expression of the key molecules that are involved in the cGAS-STING pathway. It turned out that NPRL2 loss reduced the expression of STING at both mRNA and protein levels. By analyzing the KIRC TCGA dataset, we found that low NPRL2 expression was associated with reduced T cell infiltration and immune checkpoint (CTLA4 and PDCD1) expression, reflecting an immunologically cold tumor microenvironment, which may respond poorly to immunotherapy. To test this hypothesis, we knocked down the expression of Nprl2 in Renca cells and found that Nprl2 loss reduced the complete response rate to anti-PD1 therapy in a syngeneic tumor model. However, NPRL2/Nprl2 loss sensitized RCC cells and Renca tumors to the ATM inhibitor KU60019, while the combination of KU60019 and anti-PD1 therapy showed no evidence of synergistic effect. Conclusions These results indicate that NPRL2 loss regulates DNA damage and cytosolic DNA accumulation via lysosomal but not canonical autophagic inhibition, and on the other hand, silences cytosolic DNA response by suppressing STING expression. NPRL2 loss defines a genomically unstable but nonimmunogenic tumor phenotype associated with checkpoint inhibitor resistance in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6b45a93450c0576e0305c391ec88eb6a795e2a" target='_blank'>
              56NPRL2 loss enhances DNA damage while isolating cGAS-STING pathway and conferring immunotherapy resistance
              </a>
            </td>
          <td>
            Xiande Liu, Muhammed Onel, Hongchao He, Xuesong Zhang, Yanting Zhang, Anh Hoang, Guang Peng, E. Jonasch
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3258ce606cc85b0f7068fbf4858f5ecfcb1cac8a" target='_blank'>
              Major alleles of CDCA7 shape CG methylation in Arabidopsis thaliana.
              </a>
            </td>
          <td>
            Pierre Bourguet, Z. Lorković, Darya Kripkiy Casado, Valentin Bapteste, Chung Hyun Cho, Anna A Igolkina, Cheng‐ruei Lee, Magnus Nordborg, Frédéric Berger, Eriko Sasaki
          </td>
          <td>2025-11-07</td>
          <td>Nature plants</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/979fccda6253a03a36cbc921b71b8ecae3f2890f" target='_blank'>
              Chromatin state dynamics of autosomes and the B chromosome during spermatogenesis in Pseudococcus viburni.
              </a>
            </td>
          <td>
            M. Herbette, Laura Ross
          </td>
          <td>2025-10-13</td>
          <td>Heredity</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The adenoma-adenocarcinoma pathway represents a crucial mechanism underlying the development of colorectal precancerous lesions, encompassing approximately 85%-90% of colorectal cancer (CRC). Elucidating the molecular mechanisms underlying colorectal cancer progression is of paramount importance for achieving early and accurate diagnosis as well as effective treatment. We collected peripheral blood mononuclear cells (PBMC) from healthy controls, adenoma patients, and adenocarcinoma patients, and performed transcriptomic profiling to characterize dynamic gene expression during carcinogenesis. Diagnostic potential was assessed using receiver operating characteristic (ROC) analysis, and a random-forest model was trained to classify disease status. The screening identified genes with consistent expression changes as potential early diagnostic markers, and further exploration of their functions and significance in CRC is conducted through analysis of the TCGA database. The findings revealed that the progression of precancerous lesions in the “Normal-Adenoma-Cancer” (N-A-C) sequence was accompanied by a sustained enhancement of the immune response. Notably, HECW2, WARS1, SLC16A3, SECTM1, IFITM3, ADAMTSL4, FCGR1A, F2RL1, OPLAH, SERPINA1, FCGR1CP showed consistent upregulation with promising diagnostic performance. In our PBMC cohort, the random-forest classifier achieved an accuracy of 93.62%, indicating potential for distinguishing cancer from precancerous lesions. The bioinformatics analysis revealed a significant association of these genes with DNA methyltransferase, DNA mismatch repair, m6A regulator, tumor mutational burden (TMB) and microsatellite instability (MSI). Furthermore, a detailed analysis was further performed on WARS1. In the “N-A-C” sequence, WARS1 exhibited a significant upregulation in both blood and tissues, demonstrating a positive correlation with augmented infiltration of immune cells, activation of stromal and immune responses, as well as heightened activity during the cancer immune cycle. However, it demonstrates a declining trend in the progression of CRC from stage I to IV, which may be intricately associated with the metastasis of CRC. The WARS1 can serve as a reliable indicator of the immune response in CRC, thereby demonstrating its potential to impede tumorigenesis or metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa1e73dcc984320ef956ec066c98a00971bc8f16" target='_blank'>
              Progressively altered genes in colorectal carcinogenesis link oncogenesis immune cycle and tumor microenvironment
              </a>
            </td>
          <td>
            Bingwen Zhou, Qingrui Liu, Chuyue Huang, Hao Chen, Pei Wang, Yueyang Lu, Shujun Jiang, Desong Kong, Lu Wang, Zhimin Fan
          </td>
          <td>2025-10-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07b66fdfdb5795523b8bbc3556aa7471128f3b" target='_blank'>
              Epithelial cell fusion is required for tissue repair following UV-A irradiation
              </a>
            </td>
          <td>
            Minqi Shen, Lillie G Mitchell, Lydia W Boer, Vicki P. Losick
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Lung carcinoma cells harboring mutations in STK11 and/or KEAP1 in a KRAS mutant background have intrinsic therapeutic resistance. We found that these cells are sensitive to preclinical stage pharmacological inhibitors of N-myristoyltransferases, which reduce tumor growth in xenograft mouse models. Unexpectedly, the sensitivity to NMT inhibitors correlates with cell’s dependency on the inner mitochondrial protein Translocase of Inner Mitochondrial Membrane 17 A (Protein: TIM17A, Gene: TIMM17A). Leveraging data from The Cancer Genome Atlas (TCGA), we set to further explore the significance of N-myristoyltransferase-1 (NMT1) and TIMM17A expression individually and together in cox-regression models to test their association to clinical endpoints in lung carcinoma. Our results showed that lung adenocarcinoma (LUAD) tissue with high expression of both NMT1 and TIMM17A had worse overall survival (OS), disease-specific survival (DSS), and disease-free survival (DFS) compared to those with low expression of both genes. Moreover, high NMT1 was associated with worse OS, but only in the group with also high TIMM17A. This highlights a novel NMT1/TIMM17A axis as a promising pathway for predicting patient prognosis. Further studies are warranted investigating both genes as targets for novel therapeutic strategies for KRAS mutant non-small cell lung carcinoma with STK11 and/or KEAP1 co-mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c97d4c9a643130433df2e1bae65ee3f96f7f873" target='_blank'>
              Elevated co-expression of TIMM17A and NMT1 is associated with poor survival in non-small cell lung cancer
              </a>
            </td>
          <td>
            Alfred A Chan, K. Sankar, Karen L. Reckamp, Begoña Díaz, Delphine J. Lee
          </td>
          <td>2025-10-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>