<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-11-04 06:33:20 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>26</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>57</td>
          <td>132</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>103</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>293</td>
          <td>132</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>250</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, J. Luebeck, V. Bafna, J. Boeke, P. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is strongly associated with treatment resistance and shorter survival for patients with cancer, including patients with glioblastoma. The non-chromosomal inheritance of ecDNA during cell division is a major contributor to intratumoral genetic heterogeneity. At present, the spatial dynamics of ecDNA, and the impact on tumor evolutionary trajectories, are not well understood. Here, we investigate the spatial-temporal evolution of ecDNA and its clinical impact by analyzing tumor samples from 94 treatment-naive human IDH-wildtype glioblastoma patients. We developed a spatial-temporal computational model of ecDNA positive tumors (‘SPECIES’) that integrates whole-genome sequencing, multi-region DNA FISH, and nascent RNAscope, to provide unique insight into the spatial dynamics of ecDNA evolution. Random segregation in combination with positive selection of ecDNAs induce large, predictable spatial patterns of cell-to-cell ecDNA copy number variation that are highly dependent on the oncogene encoded on the circular DNA. EGFR ecDNAs often reach high mean copy number (mean of 50 copies per tumor cell), are under strong positive selection (mean selection coefficient, s > 2) and do not co-amplify other oncogenes on the same ecDNA particles. In contrast, PDGFRA ecDNAs have lower mean copy number (mean of 15 copies per cell), are under weaker positive selection and frequently co-amplify other oncogenes on the same ecDNA. Evolutionary modeling suggests that EGFR ecDNAs often accumulate prior to clonal expansion. EGFR structural variants, including vIII and c-terminal deletions are under strong positive selection, are found exclusively on ecDNA, and are intermixed with wild-type EGFR ecDNAs. Simulations show EGFRvIII ecDNA likely arises after ecDNA formation in a cell with high wild-type EGFR copy number (> 10) before the onset of the most recent clonal expansion. This remains true even in cases of co-selection and co-amplification of multiple oncogenic ecDNA species in a subset of patients. Overall, our results suggest a potential time window in which early ecDNA detection may provide an opportunity for more effective intervention. Highlights ecDNA is the most common mechanism of focal oncogene amplification in IDHwt glioblastoma. EGFR and its variants on ecDNA are particularly potent, likely arising early in tumor development, providing a strong oncogenic stimulus to drive tumorigenesis. Wild-type and variant EGFR ecDNA heteroplasmy (co-occurrence) is common with EGFRvIII or c-terminal deletions being derived from EGFR wild-type ecDNA prior to the most recent clonal expansion. Tumors with ecDNA amplified EGFR versus PDGFRA exhibit different evolutionary trajectories. SPECIES model can infer spatial evolutionary dynamics of ecDNA in cancer. A delay between ecDNA accumulation and subsequent oncogenic mutation may give a therapeutic window for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e8d31fa2dc1841943bfa95252e831edb576e77" target='_blank'>
              Extrachromosomal DNA driven oncogene spatial heterogeneity and evolution in glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, M. Haughey, J. Luebeck, A. Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Jeanette Kittel, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G Jones, E. Nye, Mary Green, Lucy Meader, E. Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, S. Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf0847e7afd3b17c26839196bb1a05ffa786834" target='_blank'>
              Aneuploidy as a driver of human cancer.
              </a>
            </td>
          <td>
            Eran Sdeor, Hajime Okada, Ron Saad, Tal Ben-Yishay, Uri Ben-David
          </td>
          <td>2024-10-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Translocations have largely been implicated in tumor development. However, beyond the consequences of aberrant gene expression near the breakpoint, their effects remain unexplored. In this work, we characterize the interplay between translocations, chromatin organization and gene expression using mantle cell lymphoma (MCL) as a model. We show that in vitro induced MCL-associated translocations can drive transcriptional changes at entire chromosome arms affecting multiple genes in a regulon-like fashion. Additionally, overexpressed genes in MCL patients are enriched in the exact same genomic regions, further underlining its potential relevance for lymphomagenesis. Moreover, we demonstrate a clear link between the translocation-induced transcriptional alterations and genome organization, with genes most susceptible to change expression residing in pre-existing long-range interacting loops spanning 50 megabases. The translocation places the strong immunoglobulin enhancer into this loop, allowing the spread of its regulatory potential over the entire affected chromosome arm. Finally, we show that translocation-induced effects mainly represent expression enhancement of genes already active prior to translocation formation, highlighting the importance of the epigenetic state of the cell in which this initial hit occurs. In summary, we show that translocations can induce ample, simultaneous gene expression changes affecting entire chromosome arms, representing an important new mechanism for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdbb3468215c566e38e0c9ba35afc86f4ee62c1" target='_blank'>
              Translocations can drive expression changes of multiple genes in regulons covering entire chromosome arms
              </a>
            </td>
          <td>
            Anna Oncins, Roser Zaurin, Houyem Toukabri, Kimberly Quililan, José R Hernández Mora, Magdalena A. Karpinska, Erik Wernersson, Alastair Smith, Agostina Bianchi, Leone Albinati, Andrea Rivero, François Serra, Raúl Gómez, Cristina López, S. Beà, Nadia Halidi, A. Valencia, Magda Bienko, A. M. Oudelaar, Renée Beekman
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4921d9b380eadcd80f19284f6396e453cfba83e" target='_blank'>
              Exploring the potential of extrachromosomal DNA as a novel oncogenic driver.
              </a>
            </td>
          <td>
            Huanbo Zhu, Longtao Huangfu, Junbing Chen, Jiafu Ji, Xiaofang Xing
          </td>
          <td>2024-09-27</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution.
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer Shelton, Minita J. Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) emerges as a crucial promoter of rapid intra-tumour heterogeneity and oncogene amplification, contributing to tumour progression and adaptability to resistance. Multiple types of ecDNA are often observed within the same cancer cell, promoting high levels of complexity. While the study of multiple ecDNA types is still in its early stages with exciting new clinical and experimental studies in human cancers, few theoretical models have been proposed to understand the mechanisms enabling the maintenance of multiple ecDNA types (mix cells). Here, based on our general framework of modeling multiple ecDNA types, we explore the conditions that allow for a stable coexistence of multiple ecDNA types within single cancer cells. We study how genetic mutations or phenotypic switching among different ecDNA types contribute to the maintenance of ecDNA diversity under different fitness regimes. We show that either positive epistasis or switching is required to stably maintain mix cell populations. Counter-intuitively, intermediate rather than high switching rates lead to the highest fraction of mix cells, which is further confirmed by our analysis of sub-population compositions, transition probabilities between pure and mix states as well as single-cell Shannon indices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398d1444c0fac5c6ddf4fe45cb3ef1584d2da2f5" target='_blank'>
              The maintenance of multiple ecDNA types in single cancer cells
              </a>
            </td>
          <td>
            Elisa Scanu, Benjamin Werner, Weini Huang
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal G. Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew D Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K. Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Robertsonian chromosomes are a type of variant chromosome found commonly in nature. Present in one in 800 humans, these chromosomes can underlie infertility, trisomies, and increased cancer incidence. Recognized cytogenetically for more than a century, their origins have remained mysterious. Recent advances in genomics allowed us to assemble three human Robertsonian chromosomes completely. We identify a common breakpoint and epigenetic changes in centromeres that provide insight into the formation and propagation of common Robertsonian translocations. Further investigation of the assembled genomes of chimpanzee and bonobo highlights the structural features of the human genome that uniquely enable the specific crossover event that creates these chromosomes. Resolving the structure and epigenetic features of human Robertsonian chromosomes at a molecular level paves the way to understanding how chromosomal structural variation occurs more generally, and how chromosomes evolve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a20561e10d74149e6076e6c3bf2d3aed806c4" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes
              </a>
            </td>
          <td>
            Leonardo Gomes de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Copy number variations (CNVs) are structural genomic alterations that involve changes in the number of copies of specific DNA regions. These variations can include deletions, duplications, and more complex rearrangements, and play a critical role in cancer progression by amplifying oncogenes, deleting tumor suppressor genes, or altering other key genomic regions. Despite the importance of CNVs in cancer biology, there is a lack of comprehensive resources that aggregate CNV data across multiple cancer types, impeding the exploration of their role in different malignancies. The PanCNV-Explorer was developed to fill this gap by providing a pan-cancer resource that integrates CNV data for over 21 cancer types. Methods: The PanCNV-Explorer was developed, utilizing data from repositories such as COSMIC, DepMap, DGV, and dbVAR. Amplifications and deletions can be visualized, with detailed chromosome-specific plots highlighting CNV patterns. In addition, a genome browser has been implemented to display regions of interest for searching for genes of potential relevance. Results: The database contains over 5 million CNVs from 15,809 samples, spanning 21 primary cancer types. Pathogenic CNVs from ClinVar and DepMap were compared to benign data from DGV and dbVAR, allowing a comprehensive analysis across tissues. CNV patterns were visualized for individual cancers. We demonstrate with use cases, focusing on common oncogenes such as MYC and tumor suppressor genes such as TP53, the ability of the resource to search for cancer driver genes. For example, chromosome 12 in pancreatic cancer showed frequent amplifications in the region containing the KRAS oncogene. In another example, we show how driver gene candidates already described in certain cancer entities can also be found with high frequency in other entities. Conclusion: The PanCNV-Explorer provides a valuable resource for pan-cancer CNV analysis, facilitating the exploration of CNV-driven tumor heterogeneity. The integrated visualization tools and genome browser enable detailed exploration of genomic regions, enhancing cancer research. As a valuable and easy-to-use platform that allows researchers to quantify and compare CNVs across cancer types, the PanCNV-Explorer is an essential tool for advancing cancer genomics research. The main page is available at https://mtb.bioinf.med.uni-goettingen.de/pancnv-explorer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0740ab5d12642c500ab7be4dd721a73d9e32d198" target='_blank'>
              PanCNV Explorer: A pan-cancer database for copy number variations.
              </a>
            </td>
          <td>
            Kevin Kornrumpf, Jürgen Dönitz
          </td>
          <td>2024-09-24</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Traditional drug discovery efforts have largely focused on targeting rapid, reversible protein-mediated adaptations to undermine cancer cells’ resistance to therapy. However, cancer cells also exploit DNA-based strategies, typically viewed as slow, irreversible, and unpredictable changes like point mutations or the selection of drug-resistant clones. Contrary to this perception, extrachromosomal DNA (ecDNA) represents a form of DNA alteration that is rapid, reversible, and predictable, playing a crucial role in cancer’s adaptive response. In this study, we present a novel post-processing pipeline for the automated detection and quantification of ecDNA in Fluorescence in situ Hybridization (FISH) images using the Microscopy Image Analyzer (MIA) tool. Our approach is particularly designed to monitor ecDNA dynamics during drug treatment, providing a quantitative framework to understand how ecDNA enables cancer cells to swiftly and reversibly adapt to therapeutic pressure. This pipeline not only offers a valuable resource for researchers aiming to automate ecDNA detection in FISH images but also sheds light on the adaptive mechanisms of ecDNA in response to epigenetic remodeling agents like JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a33656032abe4b14c89c4da5e33bf1e9ab825af" target='_blank'>
              Leveraging AI to Automate Detection and Quantification of Extrachromosomal DNA (ecDNA) to Decode Drug Responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcfc9fc5c7ce6b28eb00b12e973f760953fb4c62" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-09-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/351d25779682a305652abdf98bc2b4de93e33a64" target='_blank'>
              Evaluation of whole genome sequencing utility in identifying driver alterations in cancer genome
              </a>
            </td>
          <td>
            T. Nagashima, Ken Yamaguchi, K. Urakami, Y. Shimoda, S. Ohnami, K. Ohshima, Tomoe Tanabe, A. Naruoka, Fukumi Kamada, M. Serizawa, K. Hatakeyama, S. Ohnami, Koji Maruyama, Tohru Mochizuki, Maki Mizuguchi, A. Shiomi, Y. Ohde, E. Bando, T. Sugiura, T. Mukaigawa, S. Nishimura, Yasuyuki Hirashima, Koichi Mitsuya, S. Yoshikawa, Y. Kiyohara, Y. Tsubosa, H. Katagiri, M. Niwakawa, Kaoru Takahashi, H. Kashiwagi, Y. Yasunaga, Yuji Ishida, Takashi Sugino, H. Kenmotsu, Masanori Terashima, Mitsuru Takahashi, K. Uesaka, Y. Akiyama
          </td>
          <td>2024-10-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55366bba8897b78ba0d78f6afe4016d1354283cd" target='_blank'>
              Frequent CHD1 deletions in prostate cancers of African American men is associated with rapid disease progression
              </a>
            </td>
          <td>
            M. Dióssy, V. Tisza, Hua Li, P. Sahgal, Jia Zhou, Z. Sztupinszki, D. Young, Darryl Nousome, Claire Kuo, Jiji Jiang, Yongmei Chen, Reinhard Ebner, I. Sesterhenn, J. Moncur, Gregory T Chesnut, G. Petrovics, Gregory T. Klus, Gábor Valcz, P. Nuzzo, D. Ribli, J. Börcsök, Aurél Prósz, M. Krzystanek, Thomas Ried, David Szuts, K. Rizwan, S. Kaochar, S. Pathania, Alan D. D’Andrea, I. Csabai, S. Srivastava, Matthew L Freedman, A. Dobi, S. Spisák, Zoltan Szallasi
          </td>
          <td>2024-09-19</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="It is over twenty years since the publication of the C57BL/6J mouse reference genome, which has been a key catalyst for understanding mammalian disease biology. However, the mouse reference genome still lacks telomeres and centromeres, contains 281 chromosomal sequence gaps, and only partially represents many biomedically relevant loci. We present the first T2T mouse genomes for two key inbred strains, C57BL/6J and CAST/EiJ. These T2T genomes reveal significant variability in telomere and centromere sizes and structural organisation. We add an additional 213 Mbp of novel sequence to the reference genome containing 517 protein-coding genes. We examined two important but incomplete loci in the mouse genome - the pseudoautosomal region (PAR) on the sex chromosomes and KRAB zinc finger proteins (KZFPs) loci. We identified distant locations of the PAR boundary, different copy number and sizes of segmental duplications, and a multitude of amino acid substitution mutations in PAR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30db2388994ae46d83e7501923b0604dde5725a" target='_blank'>
              The structural diversity of telomeres and centromeres across mouse subspecies revealed by complete assemblies
              </a>
            </td>
          <td>
            Bailey Francis, Landen Gozashti, Kevin Costello, Takaoki Kasahara, Olivia S. Harringmeyer, Jingtao Lilue, Mohab Helmy, Tadafumi Kato, A. Czechanski, Michael A. Quail, Iraad Bonner, Emma Dawson, Anne Ferguson-Smith, L. Reinholdt, David J. Adams, Thomas M. Keane
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="As a major type of structural variants, tandem duplication plays a critical role in tumorigenesis by increasing oncogene dosage. Recent work has revealed that noncoding enhancers are also affected by duplications leading to the activation of oncogenes that are inside or outside of the duplicated regions. However, the prevalence of enhancer duplication and the identity of their target genes remains largely unknown in the cancer genome. Here, by analyzing whole-genome sequencing data in a non-gene-centric manner, we identify 881 duplication hotspots in 13 major cancer types, most of which do not contain protein-coding genes. We show that the hotspots are enriched with distal enhancer elements and are highly lineage-specific. We develop a HiChIP-based methodology that navigates enhancer-promoter contact maps to prioritize the target genes for the duplication hotspots harboring enhancer elements. The methodology identifies many novel enhancer duplication events activating oncogenes such as ESR1, FOXA1, GATA3, GATA6, TP63, and VEGFA, as well as potentially novel oncogenes such as GRHL2, IRF2BP2, and CREB3L1 In particular, we identify a duplication hotspot on Chromosome 10p15 harboring a cluster of enhancers, which skips over two genes, through a long-range chromatin interaction, to activate an oncogenic isoform of the NET1 gene to promote migration of gastric cancer cells. Focusing on tandem duplications, our study substantially extends the catalog of noncoding driver alterations in multiple cancer types, revealing attractive targets for functional characterization and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269520ff92f1d6c1a566c78a1f6e2344309fcdcb" target='_blank'>
              Chromatin interaction maps identify oncogenic targets of enhancer duplications in cancer.
              </a>
            </td>
          <td>
            Yueqiang Song, Fuyuan Li, Shangzi Wang, Yuntong Wang, Cong Lai, Lian Chen, Ning Jiang, Jin Li, Xingdong Chen, Swneke D. Bailey, Xiaoyang Zhang
          </td>
          <td>2024-10-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Recent breakthroughs in the genetic manipulation of mitochondrial DNA (mtDNA) have enabled the precise introduction of base substitutions and the effective removal of genomes carrying harmful mutations. However, the reconstitution of mtDNA deletions responsible for severe mitochondrial myopathies and age-related diseases has not yet been achieved in human cells. Here, we developed a method to engineer specific mtDNA deletions in human cells by co-expressing end-joining (EJ) machinery and targeted endonucleases. As a proof-of-concept, we used mito-EJ and mito-ScaI to generate a panel of clonal cell lines harboring a ∼3.5 kb mtDNA deletion with the full spectrum of heteroplasmy. Investigating these isogenic cells revealed a critical threshold of ∼75% deleted genomes, beyond which cells exhibited depletion of OXPHOS proteins, severe metabolic disruption, and impaired growth in galactose-containing media. Single-cell multiomic analysis revealed two distinct patterns of nuclear gene deregulation in response to mtDNA deletion accumulation; one triggered at the deletion threshold and another progressively responding to increasing heteroplasmy. In summary, the co-expression of mito-EJ and programable nucleases provides a powerful tool to model disease-associated mtDNA deletions in different cell types. Establishing a panel of cell lines with a large-scale deletion at varying levels of heteroplasmy is a valuable resource for understanding the impact of mtDNA deletions on diseases and guiding the development of potential therapeutic strategies. Graphical Abstract Highlights Combining prokaryotic end-joining with targeted endonucleases generates specific mtDNA deletions in human cells Engineering a panel of cell lines with a large-scale deletion that spans the full spectrum of heteroplasmy 75% heteroplasmy is the threshold that triggers mitochondrial and cellular dysfunction Two distinct nuclear transcriptional programs in response to mtDNA deletions: threshold-triggered and heteroplasmy-sensing">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c60438e8f46fecefc3ea7cd6e713fab4e42d857" target='_blank'>
              Engineering mtDNA Deletions by Reconstituting End-Joining in Human Mitochondria
              </a>
            </td>
          <td>
            Yinxu Fu, Max Land, Ruobing Cui, Tamar Kavlashvili, Minsoo Kim, Toby Lieber, Keunwoo Ryu, Emily DeBitetto, I. Masilionis, Rahul Saha, Meril Takizawa, Daphne Baker, Marco Tigano, Ed Reznik, Roshan Sharma, R. Chaligné, Craig B. Thompson, D. Pe’er, Agnel Sfeir
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="In Arabidopsis thaliana, the chromosome arms are DNA-hypomethylated and enriched in the euchromatin-specific histone mark H3K4me3. In contrast, pericentromeric regions are DNA-hypermethylated and enriched in H3K9me2. In order to investigate how chromosomal location affects epigenetic stability and gene activity, we induced two differently-sized inversions by CRISPR/Cas and introduced heterochromatic, pericentric sequences into an euchromatic chromosomal arm. The epigenetic status of the lines was investigated by whole genome bisulfite sequencing and chromatin immunoprecipitation. Additionally, we studied the effect of the chromosomal inversions on gene expression. Our analysis revealed that both inversions affected neither the global distribution of eu- and heterochromatin-specific histone marks nor the global DNA methylation landscape. However, minor epigenetic changes were found across the entire genome. Importantly, the inverted chromosome regions and their border regions did not change their epigenetic profile. The transcription analysis of both inversion lines revealed that 0.5 - 1 % of genes were differentially expressed across the entire genome. However, the expression activity of those genes encoded by the inverted segments, and located around the cutting sites of CRISPR/Cas was less affected. Thus, gene expression levels and the epigenetic landscape remain preserved following engineered chromosomal restructuring, at least in the following generations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34939b54562d81f6da0b2b3d50b44429f0417422" target='_blank'>
              Epigenetic state and gene expression remain stable after CRISPR/Cas-mediated chromosomal inversions
              </a>
            </td>
          <td>
            Solmaz Khosravi, Rebecca Hinrichs, Michelle Rönspies, Reza Haghi, Holger Puchta, Andreas Houben
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Allele-specific expression (ASE) is the differential abundance in levels of mRNAs that originated from the paternal and maternal copies of a gene. Such allelic imbalances can contribute to phenotypic variation and influence disease traits, including cancer. There is common ASE in tumors that results from somatic copy-number alterations (CNAs) at the DNA level, but there also exist other causes of ASE: cis-acting genetic or epigenetic variation that can lead to differential expression between the two alleles. However, the latter, non-CNA mechanisms of ASE remain understudied in cancer, as well as their role in tumor evolution and impact on clinical outcomes. By integrating a wide variety of genomic and transcriptomic pan-cancer data from the TCGA project, we show that ASE favoring the preferential expression of the mutant allele in some driver genes is subject to positive selection, and that these events are associated with worse overall survival across all cancer types. We found that the impact of ASE triggered by non-CNA causes is substantial, and we propose that some instances of cis-ASE are explained by the epigenetic changes affecting alleles differently. Furthermore, as a second mechanism, we find that splicing-altering mutations are selected in various cancer genes and result in ASE. We anticipate that the study and understanding of the role of mutant allele imbalances at the mRNA level can help understand epigenetic changes during cancer evolution, as well as identify new prognostic markers and therapeutic approaches that target altered allelic expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a918fe3f2472f99c1403cdc4bbe583961852532" target='_blank'>
              Allele-specific expression is selected in tumorigenesis, results from epigenetic changes and has prognostic relevance
              </a>
            </td>
          <td>
            Guillermo Palou-Márquez, Pere Pericot-Masdevall, F. Supek
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a180cfc530d1d7fe2d98876dc6894d589798a6e8" target='_blank'>
              Mapping extrachromosomal DNA amplifications during cancer progression.
              </a>
            </td>
          <td>
            Hoon Kim, Soyeon Kim, Taylor E. Wade, Eunchae Yeo, A. Lipsa, A. Gołȩbiewska, Kevin C Johnson, Sepil An, Junyong Ko, Yoonjoo Nam, Hwa Yeon Lee, Seunghyun Kang, Heesuk Chung, S. Niclou, H. Moon, S. Paek, V. Bafna, J. Luebeck, R.G.W. Verhaak
          </td>
          <td>2024-10-14</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Baker and colleagues developed a new algorithm called "Gain Route Identification and Timing In Cancer" (GRITIC) to uncover the path of chromosomal evolution in a tumor, particularly in the context of whole-genome duplication. Their approach found that tumors with genome doubling frequently take an indirect path from one copy number state to another. In addition, the timing of genome doubling within a tumor's evolution impacts its consequences on downstream chromosomal instability. See related article by Baker et al., p. 1810.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94d8581b82a4fe4a578bd0700cd9ac63fedd9071" target='_blank'>
              Cancer Genomes Sometimes Take the Longest Evolutionary Road.
              </a>
            </td>
          <td>
            Alison M Taylor
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Harnessing DNA double-strand breaks (DSBs) is a powerful approach for gene editing, but it may provoke loss of heterozygosity (LOH), which predisposes to tumorigenesis. To interrogate this risk, we developed a two- color flow cytometry-based system (Flo-LOH), detecting LOH in ∼5% of cells following a DSB. After this initial increase, cells with LOH decrease due to a competitive disadvantage with parental cells, but if isolated, they stably propagate. Segmental loss from terminal deletions with de novo telomere addition and nonreciprocal translocations is observed as well as whole chromosome loss, especially following a centromeric DSB. LOH spans megabases distal from the DSB, but also frequently tens of megabases centromere-proximal. Inhibition of microhomology-mediated end joining massively increases LOH, which is synergistically increased with concomitant inhibition of canonical nonhomologous end joining. The capacity for large-scale LOH must therefore be considered when using DSB-based gene editing, especially in conjunction with end joining inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d1b33368b2fb7ae9c66926efe7d4fcbd491246" target='_blank'>
              Megabase-scale loss of heterozygosity provoked by CRISPR-Cas9 DNA double-strand breaks
              </a>
            </td>
          <td>
            Samantha B. Regan, Darpan Medhi, Travis B. White, Yi-Zhen Jiang, Su Jia, Qichen Deng, M. Jasin
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Precise transcriptional regulation is critical for cellular function and development, yet the mechanism of this process remains poorly understood for many genes. To gain a deeper understanding of the regulation of neuropsychiatric disease risk genes, we identified a total of 39 functional enhancers for four dosage-sensitive genes, APP, FMR1, MECP2, and SIN3A, using CRISPR tiling deletion screening in human induced pluripotent stem cell (iPSC)-induced excitatory neurons. We found that enhancer annotation provides potential pathological insights into disease-associated copy number variants. More importantly, we discovered that allelic enhancer deletions at SIN3A could be compensated by increased transcriptional activities from the other intact allele. Such allelic compensation effects (ACE) on transcription is stably maintained during differentiation and, once established, cannot be reversed by ectopic SIN3A expression. Further, ACE at SIN3A occurs through dosage sensing by the promoter. Together, our findings unravel a regulatory compensation mechanism that ensures stable and precise transcriptional output for SIN3A, and potentially other dosage-sensitive genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c9cf4cae8ed81690d40ccc0776865b484f3141" target='_blank'>
              CRISPR tiling deletion screens reveal functional enhancers of neuropsychiatric risk genes and allelic compensation effects (ACE) on transcription
              </a>
            </td>
          <td>
            Xingjie Ren, Lina Zheng, Lenka Maliskova, Tsz Wai Tam, Yifan Sun, Hongjiang Liu, Jerry Lee, M. Takagi, Bin Li, Bing Ren, Wei Wang, Yin Shen
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="It is unclear how cells counteract the potentially harmful effects of uncoordinated DNA replication in the context of oncogenic stress. Here, we identify the WRAD (WDR5/RBBP5/ASH2L/DPY30) core as a modulator of DNA replication in pancreatic ductal adenocarcinoma (PDAC) models. Molecular analyses demonstrated that the WRAD core interacts with the replisome complex, with disruption of DPY30 resulting in DNA re-replication, DNA damage, and chromosomal instability (CIN) without affecting cancer cell proliferation. Consequently, in immunocompetent models, DPY30 loss induced T cell infiltration and immune-mediated clearance of highly proliferating cancer cells with complex karyotypes, thus improving anti-tumor efficacy upon anti-PD-1 treatment. In PDAC patients, DPY30 expression was associated with high tumor grade, worse prognosis, and limited response to immune checkpoint blockade. Together, our findings indicate that the WRAD core sustains genome stability and suggest that low intratumor DPY30 levels may identify PDAC patients who will benefit from immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561fe3b30b264bde1623b0000d0a802b77d77a31" target='_blank'>
              WRAD core perturbation impairs DNA replication fidelity promoting immunoediting in pancreatic cancer
              </a>
            </td>
          <td>
            F. Citron, I. Ho, Chiara Balestrieri, Zhaoliang Liu, Er-Yen Yen, Luca Cecchetto, Luigi Perelli, Li Zhang, Luis Castillo Montañez, Nicholas Blazanin, Charles A. Dyke, Rutvi Shah, Sergio Attanasio, Sanjana Srinivasan, Ko-Chien Chen, Ziheng Chen, Iolanda Scognamiglio, Nhung Pham, Hania Khan, Shan Jiang, Jing Pan, B. Vanderkruk, Cecilia S. Leung, Mahinur Mattohti, Kunal Rai, Yanshuo Chu, Linghua Wang, Sisi Gao, Angela Deem, A. Carugo, Huamin Wang, W. Yao, Giovanni Tonon, Yun Xiong, Philip L. Lorenzi, C. Bonini, M. Anna Zal, Bradford G Hoffman, Timmothy Heffernan, Virginia Giuliani, Collene R. Jeter, Yonathan Lissanu, G. Genovese, M. D. Pilato, A. Viale, Giulio F. Draetta
          </td>
          <td>2024-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy nearly half of the human genome and play diverse biological roles. Despite their abundance, the extent to which TEs contribute to three-dimensional (3D) genome structure remains unclear. To investigate this, we developed te_hic, a computational pipeline that integrates genomic data with chromatin interactions. Our analysis reveals that TE sequences are responsible for 3D genome structure in interphase nuclei. This phenomenon is mediated by the recruitment of specific transcription factors to TEs, which enables both the formation and disruption of chromatin contacts. We computationally identified known contact-forming proteins (CTCF, RAD21, SMC3) and breaking proteins (RNF2), as well as novel candidate contact-formers (SMARCA4, MAFK). Knockdown of the predicted contact-formers SMARCA4 and MAFK decreased contacts and loops at and between TEs. Notably, SMARCA4 knockdown selectively reduced short-range contacts, highlighting its role in maintaining 3D genome structure mediated by TE binding. Overall, our findings demonstrate that TEs are crucial determinants of 3D genome organization in mammalian cells. Key findings TEs determine 48% of the 3D genome structure alone, and 70% of heterotypic contacts. A/B compartments, TADs, and loop signatures can be retrieved using TE-mapped reads only. TFs can be divided into contact formers and breakers at TEs. SMARCA4 and MAFK build chromatin contacts between TE sequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d79bb0690215dabbc840c511519efb306a3da77" target='_blank'>
              Dissecting the Contribution of Transposable Elements to Interphase Chromosome Structure
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhouqi Huang, Li Sun, Xuemeng Zhou, Jiangping He, Alexander Strunnikov, Andrew P. Hutchins
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0970058b2af0e5ea082fb819477f1708b4c4db9a" target='_blank'>
              DNA demethylation triggers cell free DNA release in colorectal cancer cells
              </a>
            </td>
          <td>
            V. Pessei, M. Macagno, Elisa Mariella, Noemi Congiusta, V. Battaglieri, Paolo Battuello, Marco Viviani, Giulia Gionfriddo, S. Lamba, Annalisa Lorenzato, D. Oddo, Fariha Idrees, Alessandro Cavaliere, A. Bartolini, S. Guarrera, M. Linnebacher, Laura Monteonofrio, Luca Cardone, Michele Milella, A. Bertotti, Silvia Soddu, Elena Grassi, G. Crisafulli, A. Bardelli, L. Barault, F. Di Nicolantonio
          </td>
          <td>2024-10-09</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2+ tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One Sentence Summary Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6498a7e5198294cbede8c8c23e085245d62203" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2024-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="MYC deregulation occurs in the majority of multiple myeloma (MM) cases and is associated with progression and worse prognosis. Enhanced MYC expression occurs in about 70% of MM patients, but it is known to be driven by translocation or amplification events in only ~40% of myelomas. Here, we used CRISPR interference (CRISPRi) to uncover an epigenetic mechanism of MYC regulation whereby increased accessibility of a plasma cell-type specific enhancer leads to increased MYC expression. This native enhancer activity was not associated with enhancer hijacking events but led to specific binding of c-MAF, IRF4, and SPIB transcription factors that activated MYC expression in the absence of known genetic aberrations. In addition, focal amplification was another mechanism of activation of this enhancer in approximately 3.4% of MM patients. Together, these findings define an epigenetic mechanism of MYC deregulation in MM beyond known translocations or amplifications and point to the importance of non-coding regulatory elements and their associated transcription factor networks as drivers of MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89586e6b56717e928b88a8526a759effc353dc88" target='_blank'>
              Selective Enhancer Gain of Function Deregulates MYC Expression in Multiple Myeloma.
              </a>
            </td>
          <td>
            M. Rahmat, Kendell Clement, Jean-Baptiste Alberge, Romanos Sklavenitis-Pistofidis, R. Kodgule, C. Fulco, Daniel Heilpern-Mallory, Katarina Nilsson, David Dorfman, Jesse M Engreitz, G. Getz, Luca Pinello, R. Ryan, I. M. Ghobrial
          </td>
          <td>2024-09-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Satellite DNA makes up ∼11% of the mouse genome, predominantly located in centromeric and pericentric regions, which are crucial for chromosome segregation. While comprehensive assemblies of these regions have been established in the human genome, they are still lacking in the mouse genome. In this study, we used PacBio long-read sequencing, CUT&RUN sequencing, DNA methylation analysis, and RNA sequencing to generate genomic and epigenomic maps of these regions. We find that centromeric regions are primarily occupied by 120-mer Minor satellites, with other Minor Satellite length variants, 112-mers and 112-64-dimers, localized at centromere-pericentric junctions. Pericentromeric regions are mainly composed of homogeneous Major satellites, while pericentric-chromosomal junctions contain a higher density of divergent satellites. Additionally, the density of non-satellite repeats increases progressively from centromeres to pericentromeres, and further toward chromosomal arm junctions. We found that 120-mer Minor satellites in the core centromere are highly enriched with CENP-A, while the 112-mers and 112-64-dimers show lower CENP-A levels. Homogeneous Major satellites are more enriched with H3K9me3 heterochromatin, whereas divergent Major satellites are preferentially associated with H3K27me3. Furthermore, DNA methylation levels are lower in centromeres compared to pericentric regions. We also observed that only a small subset of satellites is transcribed into RNA, particularly regions exhibiting lower DNA methylation density. Our comprehensive assembly and characterization of the genomic and epigenomic landscape of mouse centromeric and pericentric regions have major implications for satellite biology and ongoing mouse telomere-to-telomere (T2T) assembly efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951d9f9f9b8e262cc6e6811c22c27c69e6bb984f" target='_blank'>
              Genomic and epigenomic maps of mouse centromeres and pericentromeres
              </a>
            </td>
          <td>
            Gitika Chaudhry, Jinguye Chen, Lucy Snipes, Smriti Bahl, Jenika Packiaraj, Xuan Lin, Jitendra Thakur
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios, Lei Peng, Luojia Yang
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Aflatoxin B1 (AFB1), a potent mycotoxin, is one of the environmental risk factors that cause liver cancer. In the liver, the bioactivated AFB1 intercalates into the DNA double helix to form a bulky DNA adduct which will lead to mutation if left unrepaired. Here, we adapted the tXR-seq method to measure the nucleotide excision repair of AFB1-induced DNA adducts at single-nucleotide resolution on a genome-wide scale, and compared it with repair data obtained from conventional UV-damage XR-seq. Our results showed that transcription-coupled repair plays a major role in the damage removal process. We further analyzed the distribution of nucleotide excision repair sites for AFB1-induced DNA adducts within the 3D human genome organization. Our analysis revealed a heterogeneous AFB1–dG repair across four different organization levels, including chromosome territories, A/B compartments, TADs, and chromatin loops. We found that chromosomes positioned closer to the nuclear center and regions within A compartments have higher levels of nucleotide excision repair. Notably, we observed high repair activity around both TAD boundaries and loop anchors. These findings provide insights into the complex interplay between AFB1-induced DNA damage repair, transcription, and 3D genome organization, shedding light on the mechanisms underlying AFB1-induced mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6fed0c3af4e712a403c0af2af9bb447bf31d09" target='_blank'>
              Nucleotide excision repair of aflatoxin-induced DNA damage within the 3D human genome organization
              </a>
            </td>
          <td>
            Yiran Wu, Muhammad Muzammal Adeel, Dian Xia, A. Sancar, Wentao Li
          </td>
          <td>2024-09-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214031bb67ad0add97aa1077e44bc79d2bc07460" target='_blank'>
              Genomic landscape of adult testicular germ cell tumours in the 100,000 Genomes Project
              </a>
            </td>
          <td>
            Máire Ní Leathlobhair, A. Frangou, B. Kinnersley, A. Cornish, D. Chubb, E. Lakatos, Prabhu Arumugam, Andreas J. Gruber, P. Law, Avraam Tapinos, G. M. Jakobsdottir, I. Peneva, A. Sahli, Evie M Smyth, Richard Y Ball, R. Sylva, K. Benes, Dan Stark, Robin J Young, Alexander T J Lee, Vincent Wolverson, R. Houlston, A. Sosinsky, A. Protheroe, Matthew J. Murray, David C Wedge, C. Verrill, 
          </td>
          <td>2024-10-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits widespread epigenetic alterations, yet their impact on cis-regulatory elements (CREs) and retrotransposons remains poorly understood. Here, we present an integrated epigenomic and transcriptomic analysis of HCC tumors and matched tumor-adjacent normal tissues. We identified extensive DNA hypomethylation coupled with changes in histone modifications at partially methylated domains, CREs, and retrotransposons. These epigenetic aberrations were associated with dysregulated expression of genes involved in cell cycle regulation, immune response, and extracellular matrix organization. Notably, our findings revealed a novel mechanism for the transcriptional dysregulation of GPC3, a key HCC biomarker and immunotherapeutic target. We observed that GPC3 upregulation is driven by both the reactivation of a fetal liver super enhancer and hypomethylation of GPC3-associated CpG islands. Moreover, we found that DNA hypomethylation-driven aberrant expression of retrotransposons carries prognostic significance in HCC. Patients with high expression of a long non-coding RNA driven by a HERVE-int element exhibited more aggressive tumors, poorer clinical outcomes, and molecular features associated with favorable immunotherapy response. Together, our study provides a comprehensive resource for understanding the role of epigenetic dysregulation in HCC and identifies retrotransposon-associated transcripts as potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce7deec20d100e57e2cf628c181e276b8e32f748" target='_blank'>
              Epigenomic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potentials
              </a>
            </td>
          <td>
            Clooney C. Y. Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y. J. Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B. S. Lai, K. Yip, Alfred S. Cheng, Danny Leung
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Caspase activated DNase (CAD) induced DNA breaks promote cell differentiation and therapy-induced cancer cell resistance. CAD targeting activity is assumed to be unique to each condition, as differentiation and cancer genesis are divergent cell fates. Here, we made the surprising discovery that a subset of CAD-bound targets in differentiating muscle cells are the same genes involved in the genesis of cancer-causing translocations. In muscle cells, a prominent CAD-bound gene pair is Pax7 and Foxo1a, the mismatched reciprocal loci that give rise to alveolar rhabdomyosarcoma. We show that CAD-targeted breaks in the Pax7 gene are physiologic to reduce Pax7 expression, a prerequisite for muscle cell differentiation. A cohort of these CAD gene targets are also conserved in early differentiating T cells and include genes that spur leukemia/lymphoma translocations. Our results suggest the CAD targeting of translocation prone oncogenic genes is non-pathologic biology and aligns with initiation of cell fate transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95535f8b5c0fda938ec705d36c58cef6067d4ac1" target='_blank'>
              Caspase-Activated DNase localizes to cancer causing translocation breakpoints during cell differentiation
              </a>
            </td>
          <td>
            Dalal Alsowaida, Brian D. Larsen, Sarah Hachmer, M. Azimi, Eric Arezza, Steve Brunette, Steven Tur, C. Palii, Bassam Albraidy, Claus S. Sorensen, Marjorie Brand, F. J. Dilworth, L. Megeney, Senior Scientist
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract N6-Methyladenosine (m6A) is the most abundant internal modification in mRNAs. Despite accumulating evidence for the profound impact of m6A on cancer biology, there are conflicting reports that alterations in genes encoding the m6A machinery proteins can either promote or suppress cancer, even in the same tumor type. Using data from The Cancer Genome Atlas, we performed a pan-cancer investigation of 15 m6A core factors in nearly 10000 samples from 31 tumor types to reveal underlying cross-tumor patterns. Altered expression, largely driven by copy number variations at the chromosome arm level, results in the most common mode of dysregulation of these factors. YTHDF1, YTHDF2, YTHDF3 and VIRMA are the most frequently altered factors and the only ones to be uniquely altered when tumors are grouped according to the expression pattern of the m6A factors. These genes are also the only ones with coherent, pan-cancer predictive power for progression-free survival. On the contrary, METTL3, the most intensively studied m6A factor as a cancer target, shows much lower levels of alteration and no predictive power for patient survival. Therefore, we propose the non-enzymatic YTHDF and VIRMA genes as preferred subjects to dissect the role of m6A in cancer and as priority cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cb34b460873bf086e54bf45d997fe617e03818" target='_blank'>
              The three YTHDF paralogs and VIRMA are strong cross-histotype tumor driver candidates among m6A core genes
              </a>
            </td>
          <td>
            E. Destefanis, D. Sighel, Davide Dalfovo, Riccardo Gilmozzi, F. Broso, Andrea Cappannini, Janusz M. Bujnicki, Alessandro Romanel, E. Dassi, A. Quattrone
          </td>
          <td>2024-10-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="PURPOSE Nuclear protein in testis carcinoma (NC) is an underdiagnosed and aggressive squamous/poorly differentiated cancer characterized by rearrangement of the gene NUTM1 on chromosome 15q14. Co-occurring alternations have not been fully characterized. METHODS We analyzed the genomic and immune landscape of 54 cases of NC that underwent DNA- and RNA-based NGS sequencing (Caris). RESULTS While NC is driven by NUTM1 fusion oncoproteins, co-occurring DNA mutations in epigenetic or cell cycle pathways were observed in 26% of cases. There was no significant difference between the fusion partner of NUTM1 and co-occurring gene mutations. RNA sequencing analysis showed increased MYC pathway activity in NC compared with head and neck squamous cell carcinoma (HNSCC) and lung squamous cell carcinoma (LUSC), which is consistent with the known pathophysiology of NC. Characterization of the NC tumor microenvironment using RNA sequencing revealed significantly lower immune cell infiltration compared with HNSCC and LUSC. NC was 10× higher in patients with HNSCC and LUSC younger than 50 years than in those older than 70 years. CONCLUSION To our knowledge, this is the first series of NC profiled broadly at the DNA and RNA level. We observed fewer intratumoral immune cells by RNA sequencing, which may be associated with anecdotal data of lack of immunotherapy benefit in NC. High MYC pathway activity in NC supports ongoing trials targeting MYC suppression. The incidence of NC among patients younger than 50 years with LUSC/HNSCC supports testing for NC in these patients. The prognosis of NCs remains dismal, and future studies should focus on improving the response to immunotherapy and targeting MYC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c7a9669ff7523a745b866e303487bfb7d01d6d" target='_blank'>
              Multiomic Characterization and Molecular Profiling of Nuclear Protein in Testis Carcinoma
              </a>
            </td>
          <td>
            G. Kroening, Jia Luo, Mark G Evans, T. Adeyelu, S. Ou, Z. Arter, Trisha M Wise-Draper, A. Sukari, Asfar S Azmi, David R Braxton, A. Elliott, David A Bryant, M. Oberley, Chul Kim, Geoffrey I. Shapiro, Christopher A French, M. Nagasaka
          </td>
          <td>2024-10-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Cytogenetic bands reflect genomic organization in large blocks of DNA with similar properties. Because banding patterns are invariant, this organization may often be assumed unimportant for genome regulation. Results here challenge that view. Findings here suggest cytogenetic bands reflect a visible framework upon which regulated genome architecture is built. Given Alu and L1 densities differ in cytogenetic bands, we examined their distribution after X-chromosome inactivation or formation of senescent-associated heterochromatin foci (SAHFs). Alu-rich regions remain outside both SAHFs and the Barr Body (BB), affirming that the BB is not the whole chromosome but a condensed, L1-rich core. Hi-C analysis of senescent cells demonstrates large (~10 Mb) G-bands remodel as a contiguous unit, gaining distal intrachromosomal interactions as syntenic G-bands coalesce into SAHFs. Striking peaks of Alu within R-bands strongly resist condensation. Thus, large-scale segmental genome architectur relates to dark versus light cytogenetic bands and Alu-peaks, implicating both in chromatin regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fa154555819c143e9152f156c516ff87f5d2a" target='_blank'>
              Cytogenetic bands and sharp peaks of Alu underlie large-scale segmental regulation of nuclear genome architecture
              </a>
            </td>
          <td>
            Lisa L. Hall, Kevin M. Creamer, M. Byron, Jeanne B. Lawrence
          </td>
          <td>2024-10-08</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Most phenotype-associated genetic variants map to non-coding regulatory regions of the human genome. Moreover, variants associated with blood cell phenotypes are enriched in regulatory regions active during hematopoiesis. To systematically explore the nature of these regions, we developed a highly efficient strategy, Perturb-multiome, that makes it possible to simultaneously profile both chromatin accessibility and gene expression in single cells with CRISPR-mediated perturbation of a range of master transcription factors (TFs). This approach allowed us to examine the connection between TFs, accessible regions, and gene expression across the genome throughout hematopoietic differentiation. We discovered that variants within the TF-sensitive accessible chromatin regions, while representing less than 0.3% of the genome, show a ∼100-fold enrichment in heritability across certain blood cell phenotypes; this enrichment is strikingly higher than for other accessible chromatin regions. Our approach facilitates large-scale mechanistic understanding of phenotype-associated genetic variants by connecting key cis-regulatory elements and their target genes within gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd75703a3e0196006c63fecef421b6d1dc25cee5" target='_blank'>
              Transcription factor networks disproportionately enrich for heritability of blood cell phenotypes
              </a>
            </td>
          <td>
            Jorge D. Martin-Rufino, Alexis Caulier, Seayoung Lee, Nicole Castano, Emily King, Samantha Joubran, Marcus Jones, Seth R Goldman, U. Arora, Lara Wahlster, Eric S. Lander, V. Sankaran
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Optimizing prevention and early detection of cancer requires understanding the number, types and timing of driver mutations. To quantify this, we exploited the elevated cancer incidence and mutation rates in germline BRCA1 and BRCA2 (gBRCA1/2) carriers. Using novel statistical models, we identify genomic deletions as the likely rate-limiting mutational processes, with 1-3 deletions required to initiate breast and ovarian tumors. gBRCA1/2-driven hereditary and sporadic tumors undergo convergent evolution to develop a similar set of driver deletions, and deletions explain the elevated cancer risk of gBRCA1/2-carriers. Orthogonal mutation timing analysis identifies deletions of chromosome 17 and 13q as early, recurrent events. Single-cell analyses confirmed deletion rate differences in gBRCA1/2 vs. non-carrier tumors as well as cells engineered to harbor gBRCA1/2. The centrality of deletion-associated chromosomal instability to tumorigenesis shapes interpretation of the somatic evolution of non-malignant tissue and guides strategies for precision prevention and early detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/142f79a57f8e5624ca9c9770400c8298f772b241" target='_blank'>
              Deletions Rate-Limit Breast and Ovarian Cancer Initiation
              </a>
            </td>
          <td>
            Kathleen E. Houlahan, Mahad Bihie, Julián Grandvallet Contreras, Daniel J. Fulop, Gonzalo Lopez, Hsin-Hsiung Huang, P. Van Loo, Christina Curtis, P. Boutros, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric malignancy. Based on gene expression profiling, B-ALL can be classified into distinct transcriptional subtypes with differing disease outcomes. Many of these transcriptional subtypes are defined by mutations in transcription factors and chromatin-modifying enzymes, but how such diverse mutations lead to distinct transcriptional subtypes remains unclear. To illuminate the chromatin regulatory landscape in B-ALL, we analyzed 3D genome organization, open chromatin, and gene expression in 53 primary patient samples. At the level of 3D genome organization, we identified chromatin interactions that vary across transcriptional subtypes. These sites of variable 3D chromatin interactions correlate with local gene expression changes and are enriched for core drivers of B-ALL observed in genome-wide CRISPR knock-out screens. Sites of variable 3D genome interactions are frequently shared across multiple transcriptional subtypes and are enriched for open chromatin sites found in normal B-cell development but repressed in mature B-cells. Within an individual patient sample, the chromatin landscape can resemble progenitor chromatin states at some loci and mature B-cell chromatin at others, suggesting that the chromatin in B-ALL patient tumor cells is in a partially arrested immature state. By analyzing transcriptomic data from large cohorts of B-ALL patients, we identify gene expression programs that are shared across transcriptional subtypes, associated with B-cell developmental stages, and predictive of patient survival. In combination, these results show that the 3D genome organization of B-ALL reflects B-cell developmental stages and helps illustrate how B-cell developmental arrest interacts with transcriptional subtypes to drive B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1e7461bf425fea64b8e5097538780b0bffaf" target='_blank'>
              Heterogeneity in chromatin structure drives core regulatory pathways in B-cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Arko Sen, Zhichao Xu, Sélène T. Tyndale, Jean Yasis, Chae Yun Cho, Rosalind Bump, Sahaana Chandran, Linda Luo, Yi Fu, Lillian Kay Petersen, Max Shokhirev, D. J. Kuo, G. McVicker, Jesse R. Dixon
          </td>
          <td>2024-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9a6a91b5f3ac052c57405c773236aad0d8d4b4" target='_blank'>
              Recent insights into the causes and consequences of chromosome mis-segregation.
              </a>
            </td>
          <td>
            Romain Devillers, Alexsandro dos Santos, Quentin Destombes, Mathieu Laplante, S. Elowe
          </td>
          <td>2024-09-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Epigenetic mechanisms often fuel the quick evolution of cancer cells from normal cells. Mutations or aberrant expressions in the enzymes of DNA methylation, histone post-translational modifications, and chromatin remodellers have been extensively investigated in cancer pathogenesis; however, cancer-associated histone mutants have gained momentum in recent decades. Next-generation sequencing of cancer cells has identified somatic recurrent mutations in all the histones (H3, H4, H2A, H2B, and H1) with different frequencies for various tumour types. Importantly, the well-characterised H3K27M, H3G34R/V, and H3K36M mutations are termed as oncohistone mutants because of their wide roles, from defects in cellular differentiation, transcriptional dysregulation, and perturbed epigenomic profiles to genomic instabilities. Mechanistically, these histone mutants impart their effects on histone modifications and/or on irregular distributions of chromatin complexes. Recent studies have identified the crucial roles of the H3K27M and H3G34R/V mutants in the DNA damage response pathway, but their impacts on chemotherapy and tumour progression remain elusive. In this review, we summarise the recent developments in their functions toward genomic instabilities and tumour progression. Finally, we discuss how such a mechanistic understanding can be harnessed toward the potential treatment of tumours harbouring the H3K27M, H3G34R/V, and H3K36M mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d93aa8724101a0a6b04c672de1eafdda67a5e3" target='_blank'>
              Emerging roles of cancer-associated histone mutations in genomic instabilities
              </a>
            </td>
          <td>
            Priyanka Yadav, Ronit Jain, Rajesh Kumar Yadav
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="To achieve replicative immortality, cancer cells must activate telomere maintenance mechanisms. In 10-15% of cancers, this is enabled by recombination-based alternative lengthening of telomeres pathways (ALT). ALT cells display several hallmarks including heterogeneous telomere length, extrachromosomal telomeric repeats and ALT-associated PML bodies. ALT cells also have high telomeric replication stress (RS) enhanced by fork-stalling structures (R-loops, G4s) and altered chromatin states. In ALT cells, telomeric RS promotes telomere elongation but above a certain threshold becomes detrimental to cell survival. Manipulating RS at telomeres has thus been proposed as a therapeutic strategy against ALT cancers. Through analysis of genome-wide CRISPR fitness screens, we identified ALT-specific vulnerabilities and describe here our characterization of the roles of SUB1, a ssDNA-binding protein, as a novel regulator of telomere stability. SUB1 depletion further increases RS at ALT telomeres, profoundly impairing ALT cell growth without impacting telomerase-positive cancer cells. During RS, SUB1 is recruited to stalled forks and ALT telomeres via its ssDNA-binding domain. This recruitment is potentiated by RPA depletion, suggesting that these factors may compete for ssDNA. The viability of ALT cells and their resilience towards RS also requires ssDNA-binding by SUB1. SUB1 depletion accelerates cell death induced by FANCM depletion, triggering unsustainable levels of telomeric damage specifically in ALT cells. Finally, combining SUB1 depletion with RS-inducing drugs rapidly induces replication catastrophe in ALT cells. Altogether, our work identifies SUB1 as a new ALT susceptibility with important roles in the mitigation of RS at ALT telomeres and suggests new therapeutic strategies for a host of still poorly managed cancers. Significance Statement Currently, there are few treatment options for ALT cancers with chemotherapy still occupying center stage despite often limited efficacy. ALT cancer cells experience high levels of replication stress at telomeres and its enhancement (e.g. via ATR inhibition) is a promising therapeutic strategy. Sensitivity to ATR inhibition varies amongst ALT cell lines/tumors warranting the development of additional ways to modulate telomeric replication stress. Here we identify SUB1, a single-stranded DNA-binding protein, as a vulnerability of ALT cells. SUB1 localizes to ALT telomeres and mitigates deleterious replication stress. SUB1 depletion synergizes with ATR inhibition and FANCM downregulation suggesting that co-targeting SUB1 with other regulators of replication stress at telomeres may kill ALT cancer cells more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12dc8d86a154542f040802530b59736f3f3ac6dd" target='_blank'>
              The Single-Stranded DNA-Binding Factor SUB1/PC4 Alleviates Replication Stress at Telomeres and is a Vulnerability of ALT Cancer Cells
              </a>
            </td>
          <td>
            Jean-Christophe Dubois, Erin Bonnell, Julie Frion, Samuel Zimmer, Muhammad Riaz Khan, Gabriela M. Teplitz, Lisa Casimir, Élie Méthot, Amélie Filion, Mouhamed Idrissou, Pierre-Étienne Jacques, R. Wellinger, Alexandre Maréchal
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="During cancer development, the interplay between the nucleus and the cell cycle leads to a state of genomic instability, often accompanied by observable morphological aberrations. These aberrations can be controlled by tumor cells to evade cell death, either by preventing or eliminating genomic instability. In epithelial ovarian cancer (EOC), overexpression of the multifunctional protein claudin-4 is a key contributor to therapy resistance through mechanisms associated with genomic instability. However, the molecular mechanisms underlying claudin-4 overexpression in EOC remain poorly understood. Here, we altered claudin-4 expression and employed a unique claudin-4 targeting peptide (CMP) to manipulate the function of claudin-4. We found that claudin-4 facilitates genome maintenance by linking the nuclear envelope and cytoskeleton dynamics with cell cycle progression. Claudin-4 caused nuclei constriction by excluding lamin B1 and promoting perinuclear F-actin accumulation, associated with remodeling nuclear architecture, thus altering nuclear envelope dynamics. Consequently, cell cycle modifications due to claudin-4 overexpression resulted in fewer cells entering the S-phase and reduced genomic instability. Importantly, disrupting biological interactions of claudin-4 using CMP and forskolin altered oxidative stress cellular response and increased the efficacy of PARP inhibitor treatment. Our data indicate that claudin-4 protects tumor genome integrity by remodeling the crosstalk between the nuclei and the cell cycle, leading to resistance to genomic instability formation and the effects of genomic instability-inducing agents. Graphical abstract Claudin-4 plays a crucial role in remodeling the cytoskeleton, particularly influencing nuclear architecture. This remodeling appears to act as a regulatory mechanism, limiting the progression of ovarian cancer cells into the S-phase and correlating with reduced genomic instability in ovarian tumors. Moreover, since olaparib treatment triggered a cellular oxidative response, it is likely that this claudin-4-mediated remodeling contributes to resistance against the effects of the PARP inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1779163fc52ccc64570aa04b3ca709410f5487db" target='_blank'>
              Claudin-4 remodeling of nucleus-cell cycle crosstalk maintains ovarian tumor genome stability and drives resistance to genomic instability-inducing agents
              </a>
            </td>
          <td>
            Fabian R. Villagomez, Julie Lang, Daniel Nunez-Avellaneda, K. Behbakht, Hannah L. Dimmick, Patricia Webb, K. Nephew, Margaret Neville, Elizabeth R. Woodruff, Benjamin G. Bitler
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Understanding the role of repetitive elements (REs) in cancer development is crucial for identifying novel biomarkers and therapeutic targets. This study investigated the locus-specific dysregulation of REs, including differential expression and methylation of REs, across 12 TCGA cancer types stratified by their genomic context (i.e., genic and intergenic REs). We found uniquely dysregulated genic REs co-regulated with their corresponding transcripts and associated with distinct biological functions in different cancer types. Uniquely dysregulated intergenic REs were identified in each cancer type and used to cluster different sample types. Recurrently dysregulated REs were identified in several cancer types, with genes associated with up-regulated genic REs involved in cell cycle processes and those associated with down-regulated REs involved in the extracellular matrix. Interestingly, 4 out of 5 REs consistently down-regulated in all 12 cancer types were located in the intronic region of the TMEM252, a recently discovered tumor suppressor gene. TMEM252 expression was also down-regulated in 10 of 12 cancer types, suggesting its potential importance across a wide range of cancer types. With the corresponding DNA methylation array data, we found a higher prevalence of hypo-methylated REs in most cancer types (10 out of 12). Despite the slight overlaps between differentially expressed REs and differentially methylated REs, we showed that methylation of locus-specific REs negatively correlates with their expression in some cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58fdef1beb592eec1a6ab24f91cfe621cf7209ac" target='_blank'>
              Dysregulation of locus-specific repetitive elements in TCGA pan-cancers
              </a>
            </td>
          <td>
            Chao Wang, Chun Liang
          </td>
          <td>2024-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107c0d6e681a7d833da603e062a1adf5d877660c" target='_blank'>
              MLL oncoprotein levels influence leukemia lineage identities
              </a>
            </td>
          <td>
            D. Janssens, Melodie Duran, Dominik J. Otto, Weifang Wu, Yiling Xu, Danielle C Kirkey, C. Mullighan, Joanna S Yi, S. Meshinchi, Jay Sarthy, K. Ahmad, S. Henikoff
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Introduction: KRAS and MYC are proto-oncogenes that are strictly regulated in healthy cells that have key roles in several processes such as cell growth, proliferation, differentiation, or apoptosis. These genes are tightly interconnected, and their dysregulation can lead to cancer progression. We previously individually targeted these oncogenes using Polypurine Reverse Hoogsteen (PPRH) hairpins, mostly targeting the complementary strand of G-quadruplex-forming sequences. We validated them in vitro in different cancer cell lines with deregulated KRAS and/or MYC. In this work we focused on our understanding of the cooperative dynamics between these oncogenes, by investigating the combined impact of PPRHs targeting KRAS and MYC in pancreatic and prostate cancer cells. Results: The combinations had a modulatory impact on the expression of both oncogenes, with transcriptional and translational downregulation occurring five days post-treatment. Out of the four tested PPRHs, MYC-targeting PPRHs, especially HpMYC-G4-PR-C directed against the promoter, showed a greater cytotoxic and expression modulation effect. When both KRAS- and MYC-targeting PPRHs were applied in combination, a synergistic reduction in cell viability was observed. Conclusion: The simultaneous targeting of KRAS and MYC demonstrates efficacy in gene modulation, thus in decreasing cell proliferation and viability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b672d2891bef651ec3466a64c2e67fc89cb30cd7" target='_blank'>
              Combinatorial Anti-Cancer Effect of Polypurine Reverse Hoogsteen Hairpins against KRAS and MYC Targeting in Prostate and Pancreatic Cancer Cell Lines
              </a>
            </td>
          <td>
            Simonas Valiuska, Kayla K. Elder, Steven J. McKay, C. Ciudad, V. Noé, Tracy A. Brooks
          </td>
          <td>2024-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Amongst them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8d12d939b21bc8eddc64b81e916d3c7019a271" target='_blank'>
              TopBP1 biomolecular condensates: a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Histones are essential for maintaining chromatin structure and function. Histone mutations lead to changes in chromatin compaction, gene expression, and the recruitment of DNA repair proteins to the DNA lesion. These disruptions can impair critical DNA repair pathways, such as homologous recombination and non-homologous end joining, resulting in increased genomic instability, which promotes an environment favorable to tumor development and progression. Understanding these mechanisms underscores the potential of targeting DNA repair pathways in cancers harboring mutated histones, offering novel therapeutic strategies to exploit their inherent genomic instability for better treatment outcomes. Here, we examine how mutations in histone H3 disrupt normal chromatin function and DNA damage repair processes and how these mechanisms can be exploited for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae6d859b6a16ea85b27994b62cbff421ebf7a45" target='_blank'>
              Histone H3 mutations and their impact on genome stability maintenance.
              </a>
            </td>
          <td>
            Lucas D Caeiro, Ramiro E Verdun, Lluis Morey
          </td>
          <td>2024-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6d816344be553ab4e2f5399f5fa15a18a391b" target='_blank'>
              High prevalence of chromosome 17 in breast cancer micronuclei: a means to get rid of tumor suppressors?
              </a>
            </td>
          <td>
            Laxmi Kumari, S. Sreedharanunni, Divya Dahiya, Pranab Dey, Alka Bhatia
          </td>
          <td>2024-10-22</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3095632bcf16399c0c6ee358ffba9b30df1dc77" target='_blank'>
              Integrative spatial and genomic analysis of tumor heterogeneity with Tumoroscope
              </a>
            </td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Alicja Ra Czkowska, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, Johan Hartman, Dominika Nowis, A. Carbone, Jens Lagergren, E. Szczurek
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer, and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. Through analysis of genome sequencing from primary HGSOC, we observed an association between high levels of APOBEC3 mutagenesis and poor overall survival. We experimentally addressed this correlation by modeling A3A activity in HGSOC cell lines and mouse models which resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal-transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, and providing a mechanism for their increased metastatic potential. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious impact of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b59ee718cc58bb611f3f35bfae513ef8466daf1" target='_blank'>
              APOBEC3A drives metastasis of high-grade serous ovarian cancer by altering epithelial-to-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J. Bednarski, Ronny Drapkin, Mary Mullen, Abby M. Green
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of CTC-IGC was significantly associated with poorer progression-free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13c77b6b25150646e97a913d6046b3f971cd07d" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            James B. Hicks, Michael Schmidt, Amin Nahgdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, Isla Garraway, Michael Lewis, Ana Aparicio, Amado Zurita, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah Amend
          </td>
          <td>2024-10-14</td>
          <td>Research Square</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Group 3 (G3) is one of the most common, aggressive and fatal subtypes of the paediatric cerebellar tumour Medulloblastoma (MB), primarily driven by the MYC oncogene. Targeting MYC has long been challenging and this, combined with our incomplete understanding of G3 MB molecular bases, has hindered the development of effective targeted therapies. Long noncoding RNAs (lncRNAs), with their extensive oncogenic roles, cancer-specific expression, and connection to MYC biology, offer opportunities for unravelling this complexity and providing new insights and therapeutic targets. Methodology Using genome-wide, molecular and cellular assays, we characterised the activity of the MYC-dependent, anti-apoptotic lncRNA lncMB3 in G3 MB cells. Results Through transcriptomic and interactomic analyses, we clarified lncMB3 function and mode-of-action. LncMB3 controls the TGF-β pathway, critically altered in G3 medulloblastomagenesis. This regulation occurs via the direct coding-noncoding RNA interaction between lncMB3 and the mRNA for the epigenetic factor HMGN5, with both sharing targets in the TGF-β cascade. This axis converges on apoptosis through OTX2, another G3 MB driver gene, and photoreceptor lineage genes. Synergistic effects between lncMB3 targeting and cisplatin treatment underscores the relevance of this regulatory network in vitro. Finally, we propose novel ferritin-based nanocarriers as efficient delivery tools for antisense oligonucleotides targeting lncMB3. Conclusions LncMB3 emerges as a central node linking MYC amplification to apoptosis inhibition through a circuit involving RNA-based mechanisms, G3 MB key drivers and underexplored factors. This integrated framework deepens our understanding of G3 MB molecular underpinnings and lay the foundation for translating lncRNA research into potential applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ac8b8204e386a864f9fc23cb6d36ecefc15df24" target='_blank'>
              A circuit involving the lncRNA MB3 and the driver genes MYC and OTX2 inhibits apoptosis in Group 3 Medulloblastoma by regulating the TGF-β pathway via HMGN5
              </a>
            </td>
          <td>
            Alessia Grandioso, Paolo Tollis, Francesca Romana Pellegrini, E. Falvo, Alessandro Palma, Francesco Migliaccio, Alessandro Belvedere, Jessica Rea, Giada Tisci, Annamaria Carissimo, Irene Bozzoni, D. Trisciuoglio, M. Ballarino, P. Ceci, Pietro Laneve
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f93c4dc38f47f9f9a8e9e5ea8975939c269a20" target='_blank'>
              GLTSCR1 deficiency promotes colorectal cancer development through regulating non-homologous end joining.
              </a>
            </td>
          <td>
            Fengyan Han, Xiaoxu Zhou, Lu Liu, Beibei Yang, Pengyuan Liu, Enping Xu, Zhe Tang, Honghe Zhang
          </td>
          <td>2024-10-11</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary When oncogenes, which are genes that can cause cancer, become active, they disrupt normal cell processes, especially DNA replication. This disruption is known as replication stress (RS), in which the DNA copying process is stalled or damaged. To prevent cancer, cells usually have a defense system called the DNA damage response (DDR), which can prevent damaged cells from turning into cancer. However, some cells manage to survive this stress by developing replication stress tolerance (RST). These cells can continue to grow, leading to genomic instability (GIN), which is a key feature of cancer. GIN causes various genetic changes that make cells more likely to become cancerous and more difficult to treat. This review explains how oncogenes cause RS and how cells cope with it, leading to the development of cancer. Understanding these processes can help in developing new cancer treatments. Abstract Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The MYC transcription factor requires MAX for DNA binding and widespread activation of gene expression in both normal and neoplastic cells. Surprisingly, inactivating mutations in MAX are associated with a subset of neuroendocrine cancers including pheochromocytoma, pituitary adenoma and small cell lung cancer. Neither the extent nor the mechanisms of MAX tumor suppression are well understood. Deleting Max across multiple mouse neuroendocrine tissues, we find Max inactivation alone produces pituitary adenomas while Max loss cooperates with Rb1/Trp53 loss to accelerate medullary thyroid C-cell and pituitary adenoma development. In the thyroid tumor cell lines, MAX loss triggers a striking shift in genomic occupancy by other members of the MYC network (MNT, MLX, MondoA) supporting metabolism, survival and proliferation of neoplastic neuroendocrine cells. Our work reveals MAX as a broad suppressor of neuroendocrine tumorigenesis through its ability to maintain a balance of genomic occupancies among the diverse transcription factors in the MYC network. Teaser MAX inactivation deregulates multiple transcription factors to induce neuroendocrine cancers">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6209456a4699162897778d472b3069047b1911" target='_blank'>
              MAX inactivation deregulates the MYC network and induces neuroendocrine neoplasia in multiple tissues
              </a>
            </td>
          <td>
            B. Freie, Ali H. Ibrahim, Patrick A. Carroll, R. T. Bronson, Arnaud Augert, David MacPherson, Robert N. Eisenman
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genome instability relies on preserving the chromatin structure, with any histone imbalances threating DNA integrity. Histone synthesis occurs in the cytoplasm, followed by a maturation process before their nuclear translocation. This maturation involves protein folding and the establishment of post-translational modifications. Disruptions in this pathway hinder chromatin assembly and contribute to genome instability. JMJD1B, a histone demethylase, not only regulates gene expression but also ensures a proper supply of histones H3 and H4 for the chromatin assembly. Reduced JMJD1B levels lead to the cytoplasmic accumulation of histones, causing defects in the chromatin assembly and resulting in DNA damage. To investigate the role of JMJD1B in regulating genome stability and the malignancy of melanoma tumors, we used a JMJD1B/KDM3B knockout in B16F10 mouse melanoma cells to perform tumorigenic and genome instability assays. Additionally, we analyzed the transcriptomic data of human cutaneous melanoma tumors. Our results show the enhanced tumorigenic properties of JMJD1B knockout melanoma cells both in vitro and in vivo. The γH2AX staining, Micrococcal Nuclease sensitivity, and comet assays demonstrated increased DNA damage and genome instability. The JMJD1B expression in human melanoma tumors correlates with a lower mutational burden and fewer oncogenic driver mutations. Our findings highlight JMJD1B’s role in maintaining genome integrity by ensuring a proper histone supply to the nucleus, expanding its function beyond gene expression regulation. JMJD1B emerges as a crucial player in preserving genome stability and the development of melanoma, with a potential role as a safeguard against oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1490db96e0fd741bd69913b1e9fe76e21ec91750" target='_blank'>
              Unraveling the Role of JMJD1B in Genome Stability and the Malignancy of Melanomas
              </a>
            </td>
          <td>
            Perla Cruz, Diego Peña-Lopez, Diego Figueroa, Isidora Riobó, Vincenzo Benedetti, Francisco Saavedra, Claudia Espinoza-Arratia, Thelma M. Escobar, A. Lladser, Alejandra Loyola
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44ac4365f3d537cc727149bd2f0c2b4a261aed5" target='_blank'>
              The neuroendocrine transition in prostate cancer is dynamic and dependent on ASCL1.
              </a>
            </td>
          <td>
            Rodrigo Romero, Tinyi Chu, Tania J González Robles, Perianne Smith, Yubin Xie, Harmanpreet Kaur, Sara Yoder, Huiyong Zhao, Chenyi Mao, Wenfei Kang, Maria V. Pulina, K. Lawrence, Anuradha Gopalan, S. Zaidi, K. Yoo, Jungmin Choi, Ning Fan, Olivia Gerstner, W. Karthaus, E. deStanchina, Kelly V. Ruggles, P. Westcott, R. Chaligné, D. Pe’er, C. Sawyers
          </td>
          <td>2024-10-11</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="Enhancers are noncoding DNA sequences responsible for orchestrating gene expression programs by interacting with transcription factors and chromatin regulators within complex genome structures. However, their fundamental functions can be disrupted by genetic and epigenetic alterations, leading to aberrant enhancer activation or rewiring that contributes to oncogenesis. Analyzing dysregulated enhancer landscapes reveals new subtype-defining genomic features, such as enhancer hijacking, and identifies disease-relevant transcriptional regulators as potential targets for developing enhancer-targeting therapeutic strategies. Here, we delve into evolving concepts and technologies for studying enhancers; discuss how genetic, epigenetic, and topological alterations disrupt enhancer regulation to promote oncogenic gene expression; and underscore the importance of understanding the regulatory principles governing enhancer function to guide therapeutic strategies. Furthermore, we highlight key challenges and emerging opportunities in targeting enhancers and their regulators to improve cancer classification, prognosis, and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34283c5ff3f767bcb89008f6aa1d246ab4bf96db" target='_blank'>
              Genetic and Epigenetic Dysregulation of Transcriptional Enhancers in Cancer
              </a>
            </td>
          <td>
            G. Botten, Jian Xu
          </td>
          <td>2024-10-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) mediated by transposable elements (TEs) is well established in plants. However, the molecular determinants and functional impact of this additional system of inheritance are poorly characterized. Through comprehensive analyses of severe DNA methylation loss over TEs in an experimental population and in >700 natural strains of Arabidopsis thaliana, we uncovered hundreds of TE epivariants in nature, most of which can be inherited independently of DNA sequence changes and thus are true epialleles. Furthermore, natural epivariants are enriched near genes and we uncovered a novel role for the Paf1 complex as a major facilitator of their prevalence. Conversely, related TE copies limit epiallelic variation, by producing the small RNAs that guide DNA methylation in trans. Crucially, natural epivariants mainly affect stress-responsive genes, making them unique targets for selection. Our findings therefore demonstrate that TEs are important mediators of transgenerational epigenetic variation in nature, with singular properties for plant adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6bd4e49538ff89fe2173d3326edef5a9f7c98a" target='_blank'>
              Transposable elements are prevalent vectors of transgenerational epigenetic inheritance in nature
              </a>
            </td>
          <td>
            P. Baduel, Louna De Oliveira, E. Caillieux, Grégoire Bohl-Viallefond, Mounia El Messaoudi, Ciana Xu, Matteo Barois, Vipin Singh, Alexis Sarazin, Martine Boccara, Elodie Gilbault, Antoine de France, L. Quadrana, O. Loudet, V. Colot
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902298b2a705477b62e21297649c84698622422d" target='_blank'>
              Pan-cancer Comprehensive Analysis Identified EGFR as a Potential Biomarker for Multiple Tumor Types.
              </a>
            </td>
          <td>
            Shichao Liu, Muzhi Li, YiTong Liu, RenYi Geng, Jing Ji, Rui Zhang
          </td>
          <td>2024-10-01</td>
          <td>Applied biochemistry and biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a hematological malignancy associated with poor prognosis and relapses. MM has a strong inherited component, and while the mutational and transcriptomic landscapes of MM have been characterized extensively, the molecular consequences of germline genetic variation in MM have not been previously explored. Here, we have leveraged matched genotype and expression data from 607 patients to map the germline and somatic regulatory variation in MM. We find that non-coding variation shapes the transcriptional landscape of MM, contributing to overall survival. These regulatory variants alter the binding of transcription factors with known importance in MM, such as IRF4, providing insights into the underlying mechanisms. These data contribute to the increasing understanding of the genetic, molecular, and cellular forces shaping MM risk and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5c5bba32932ad19d6d4a4784baebd131dde155a" target='_blank'>
              Germline and somatic genetic effects on gene expression and outcome in patients with Multiple Myeloma
              </a>
            </td>
          <td>
            H. Natri, Linh Bui, L. Peter, Bianca Argente, A. Federico, A. Gutierrez, Mei-i Chung, Jonathan Keats, Nicholas E Banovich
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and show that STAG2 is uniquely tumor suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility and 3D genome conformation that impact cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relates STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin mediated mechanisms of lung tumor suppression. SUMMARY STAG2 is a frequently mutated cohesin subunit across several cancers and one of the most important functional suppressors of lung adenocarcinoma. Our findings underscore important roles of STAG2 in suppressing lung tumorigenesis and highlight a STAG2-PAXIP1/PAGR1 tumor-suppressive program that may transcend cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0708e85569c4f6a48ce19a8abe96d4d4b9af8050" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis
              </a>
            </td>
          <td>
            Emily L. Ashkin, Yuning J. Tang, Haiqing Xu, King L. Hung, Julia Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5462de017e3ff2bccdaec319bc740a6c77b29d76" target='_blank'>
              PROTAC-mediated conditional degradation of the WRN helicase as a potential strategy for selective killing of cancer cells with microsatellite instability
              </a>
            </td>
          <td>
            Vikram Tejwani, Thomas Carroll, Thomas Macartney, Susanne Bandau, C. Alabert, Giulia Saredi, R. Toth, John Rouse
          </td>
          <td>2024-09-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7fcfc955df2c529779499d67e1343e62c090d8b" target='_blank'>
              Polyploidy mitigates the impact of DNA damage while simultaneously bearing its burden
              </a>
            </td>
          <td>
            Kazuki Hayashi, Kisara Horisaka, Yoshiyuki Harada, Yuta Ogawa, Takako Yamashita, Taku Kitano, Masahiro Wakita, T. Fukusumi, Hidenori Inohara, Eiji Hara, Tomonori Matsumoto
          </td>
          <td>2024-10-13</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Mammalian genomes are scattered with transposable elements (TEs). TEs are epigenetically silenced to prevent harmful effects caused by either global activation leading to genome instability or insertional mutation disturbing gene transcription. However, whether the activation of a single element can contribute to pathological phenotypes without directly affecting gene expression is largely unknown. Here, we show that tissue-specific expression of a TE in the embryo leads to the production of viral-like particles (VLPs) which can affect organ formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in its co-expression with Fgf8 in the developing embryo. While local gene regulation is unaffected, the LTR retrotransposon participates in chromatin folding at the locus and adopts the expression of the regulatory domain it is located in. This drives the production of VLPs in the Fgf8-expressing cells of the developing limb, triggering apoptotic cell death at the time of digit outgrowth and resulting in a limb malformation resembling human ectrodactyly. This phenotype can be rescued by knock-out or knock-in of the retrotransposon causing mutations preventing its full retroviral cycle. Insertion of the same element at other developmental loci faithfully recapitulates expression according to the neighboring regulatory activity. Our findings provide a mechanism by which TE insertion is incorporated into the local genomic regulatory landscape and show how VLP production in post-implantation embryos can interfere with organ formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6530941b1bbda9df13ec9f7016e41dec0d4f10bf" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, Mai H.Q. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, C. Paliou, Verena Laupert, W. Chan, L. Wittler, Thorsten Mielke, Stefan Mundlos
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Myeloid malignancies carrying somatic DNMT3A mutations (DNMT3Amut) are usually resistant to standard therapy. DNMT3Amut leukemia cells accumulate toxic DNA double strand breaks (DSBs) and collapsed replication forks, rendering them dependent on DNA damage response (DDR). DNA polymerase theta (Polθ), a key element in Polθ-mediated DNA end-joining (TMEJ), is essential for survival and proliferation of DNMT3Amut leukemia cells. Polθ is overexpressed in DNMT3Amut leukemia cells due to abrogation of PARP1 PARylation-dependent UBE2O E3 ligase-mediated ubiquitination and proteasomal degradation of Polθ. In addition, PARP1- mediated recruitment of the SMARCAD1-MSH2/MSH3 repressive complex to DSBs was diminished in DNMT3Amut leukemia cells which facilitated loading of Polθ on DNA damage and promoting TMEJ and replication fork restart. Polθ inhibitors enhanced the anti-leukemic effects of mainstream drugs such as FLT3 kinase inhibitor quizartinib, cytarabine and etoposide in vitro and in mice with FLT3(ITD);DNMT3Amut leukemia. Altogether, Polθ is an attractive target in DNMT3Amut hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a67e19e6f65c3b6d38c0743f352c92d85456a92" target='_blank'>
              DNA polymerase theta-mediated DNA repair is a functional dependency and therapeutic vulnerability in DNMT3A deficient leukemia cells
              </a>
            </td>
          <td>
            Bac Viet Le, Umeshkumar Vekariya, Monika M. Toma, M. Nieborowska-Skorska, Marie-christine Caron, M. Gozdecka, Zayd Haydar, Martin Walsh, Jayashri Ghosh, Elaine Vaughan-Williams, Paulina Podszywalow-Bartnicka, Anna-Mariya Kukuyan, Sylwia Ziolkowska, E. Hadzijusufovic, Gurushankar Chandramouly, K. Piwocka, R. Pomerantz, G. Vassiliou, B. Huntly, P. Valent, Alfonso Bellacosa, Jean-Yves Masson, Gaorav P. Gupta, Grant A. Challen, Tomasz Skorski
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="CRISPR-based genome engineering holds enormous promise for basic science and therapeutic applications. Integrating and editing DNA sequences is still challenging in many cellular contexts, largely due to insufficient control of the repair process. We find that repair at the genome-cargo interface is predictable by deep-learning models and adheres to sequence context specific rules. Based on in silico predictions, we devised a strategy of triplet base-pair repeat repair arms that correspond to microhomologies at double-strand breaks (trimologies), which facilitated integration of large cargo (>2 kb) and protected the targeted locus and transgene from excessive damage. Successful integrations occurred in >30 loci in human cells and in in vivo models. Germline transmissible transgene integration in Xenopus, and endogenous tagging of tubulin in adult mice brains demonstrated integration during early embryonic cleavage and in non-dividing differentiated cells. Further, optimal repair arms for single- or double nucleotide edits were predictable, and facilitated small edits in vitro and in vivo using oligonucleotide templates. We provide a design-tool (Pythia, pythia-editing.org) to optimize custom integration, tagging or editing strategies. Pythia will facilitate genomic integration and editing for experimental and therapeutic purposes for a wider range of target cell types and applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ad57fb5d6450173260d95ec721d1be7ca9efcc" target='_blank'>
              Pythia: Non-random DNA repair allows predictable CRISPR/Cas9 integration and gene editing
              </a>
            </td>
          <td>
            Thomas Naert, Taiyo Yamamoto, Shuting Han, Melanie Horn, Phillip Bethge, Nikita Vladimirov, F. Voigt, Joana Figueiro-Silva, Ruxandra Bachmann-Gagescu, F. Helmchen, S. Lienkamp
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b437992e1cf50ec83b6f6a40806009d50e99add" target='_blank'>
              The genomic and transcriptomic landscape of metastastic urothelial cancer
              </a>
            </td>
          <td>
            Yohann Loriot, M. Kamal, L. Syx, R. Nicolle, C. Dupain, Naoual Mensourri, I. Duquesne, P. Lavaud, C. Nicotra, Maud Ngocamus, Ludovic Lacroix, L. Tselikas, Gilles Crehange, L. Friboulet, Zahra Castel-Ajgal, Y. Neuzillet, E. Borcoman, Philippe Beuzeboc, G. Marret, Tom Gutman, Jennifer Wong, François Radvanyi, S. Dureau, J. Scoazec, N. Servant, Yves Allory, Benjamin Besse, F. André, Christophe Le Tourneau, Christophe Massard, I. Bieche
          </td>
          <td>2024-10-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogen-induced mutations are thought near-random, with rare cancer-driver mutations underlying clonal expansion. Using high-fidelity Duplex Sequencing to reach a mutation frequency sensitivity of 4×10-9 per nt, we report that sun exposure creates pervasive mutations at sites with ∼100-fold UV-sensitivity in RNA-processing gene promoters – cyclobutane pyrimidine dimer (CPD) hyperhotspots – and these mutations have a mini-driver clonal expansion phenotype. Numerically, human skin harbored 10-fold more genuine mutations than previously reported, with neonatal skin containing 90,000 per cell; UV signature mutations increased 8,000-fold in sun-exposed skin, averaging 3×10-5 per nt. Clonal expansion by neutral drift or passenger formation was nil. Tumor suppressor gene hotspots reached variant allele frequency 0.1-10% via 30-3,000 fold clonal expansion, in occasional biopsies. CPD hyperhotspots reached those frequencies in every biopsy, with modest clonal expansion. In vitro, tumor hotspot mutations arose occasionally over weeks of chronic low-dose exposure, whereas CPD hyperhotspot mutations arose in days at 1000-fold higher frequencies, growing exponentially. UV targeted mini-drivers in every skin cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccbc5cb5ec1678ecc35a610c5268f3895b55100" target='_blank'>
              Pervasive Induction of Regulatory Mutation Microclones in Sun-exposed Skin
              </a>
            </td>
          <td>
            Vijay Menon, Alejandro Garcia-Ruiz, Susan Neveu, Brenda Cartmel, Leah M. Ferrucci, Meg Palmatier, Christine Ko, Kenneth Y. Tsai, Mio Nakamura, Sa Rang Kim, Michael Girardi, Karl Kornacker, Douglas E. Brash
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6cd4623814e1323d692f8305bc119537df808a" target='_blank'>
              Role of TRIP13 in human cancer development.
              </a>
            </td>
          <td>
            Chaohu Chen, Pan Li, Guangrui Fan, Enguang Yang, Suoshi Jing, Yibo Shi, Yuwen Gong, Luyang Zhang, Zhiping Wang
          </td>
          <td>2024-10-22</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235fec0818688f0c80daa0063ad7c555bc1856b" target='_blank'>
              Hypoxia-dependent recruitment of error-prone DNA polymerases to genome replication.
              </a>
            </td>
          <td>
            Ran Yehuda, Ido Dromi, Y. Levin, Thomas Carell, Nicholas Geacintov, Z. Livneh
          </td>
          <td>2024-10-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Although generally unknown, the age of a newly diagnosed tumor encodes valuable etiologic and prognostic information. Here, we estimate the age of breast cancers, defined as the time from the start of growth to detection, using a measure of epigenetic entropy derived from genome-wide methylation arrays. Based on an ensemble of neutrally fluctuating CpG (fCpG) sites, this stochastic epigenetic clock differs from conventional clocks that measure age-related increases in methylation. We show that younger tumors exhibit hallmarks of aggressiveness, such as increased proliferation and genomic instability, whereas older tumors are characterized by elevated immune infiltration, indicative of enhanced immune surveillance. These findings suggest that the clock captures a tumor’s effective growth rate resulting from the evolutionary-ecological competition between intrinsic growth potential and external systemic pressures. Because of the clock’s ability to delineate old and stable from young and aggressive tumors, it has potential applications in risk stratification of early-stage breast cancers and guiding early detection efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dff77a79c0945793feb47e32141039b96a0e80" target='_blank'>
              Mapping the Temporal Landscape of Breast Cancer Using Epigenetic Entropy
              </a>
            </td>
          <td>
            Daniel L. Monyak, Shannon T. Holloway, Graham J. Gumbert, Lars J. Grimm, E. S. Hwang, Jeffrey R. Marks, Darryl Shibata, Marc D. Ryser
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metapolycentromeres consist of multiple sequential domains of centromeric chromatin associated with a centromere-specific variant of histone H3 (CENP-A), functioning collectively as a single centromere. To date, they have been revealed in nine flowering plant, five insect and six vertebrate species. In this paper, we focus on their structure and possible mechanisms of emergence and evolution. The metapolycentromeres may vary in the number of centromeric domains and in their genetic content and epigenetic modifications. However, these variations do not seem to affect their function. The emergence of metapolycentromeres has been attributed to multiple Robertsonian translocations and segmental duplications. Conditions of genomic instability, such as interspecific hybridization and malignant neoplasms, are suggested as triggers for the de novo emergence of metapolycentromeres. Addressing the “centromere paradox” – the rapid evolution of centromeric DNA and proteins despite their conserved cellular function – we explore the centromere drive hypothesis as a plausible explanation for the dynamic evolution of centromeres in general, and in particular the emergence of metapolycentromeres and holocentromeres. Apparently, metapolycentromeres are more common across different species than it was believed until recently. Indeed, a systematic review of the available cytogenetic publications allowed us to identify 27 candidate species with metapolycentromeres. Тhe list of the already established and newly revealed candidate species thus spans 27 species of flowering plants and eight species of gymnosperm plants, five species of insects, and seven species of vertebrates. This indicates an erratic phylogenetic distribution of the species with metapolycentromeres and may suggest an independent emergence of the metapolycentromeres in the course of evolution. However, the current catalog of species with identified and likely metapolycentromeres remains too short to draw reliable conclusions about their evolution, particularly in the absence of knowledge about related species without metapolycentromeres for comparative analysis. More studies are necessary to shed light on the mechanisms of metapolycentromere formation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a7898b3ea2f2d63e1ff7e6046657eb47951d50" target='_blank'>
              Structure and evolution of metapolycentromeres
              </a>
            </td>
          <td>
            E. Grishko, P. Borodin
          </td>
          <td>2024-10-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b108c069a149d86683e9a9aa9e66ae175bdebcb9" target='_blank'>
              MECOM Locus classical transcript isoforms affect tumor immune microenvironment and different targets in ovarian cancer
              </a>
            </td>
          <td>
            Ning Lan, Shuheng Bai, Min Chen, Xuan Wang, Zhaode Feng, Ying Gao, B. Hui, Wen Ma, Xiangxiang Zhang, Fengyuan Hu, Wanyi Liu, Wenyang Li, Fang Wu, Juan Ren
          </td>
          <td>2024-10-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pericentromeres are heterochromatic regions adjacent to centromeres that ensure accurate chromosome segregation. Despite their conserved function, they are composed of rapidly evolving A/T-rich satellite DNA. This paradoxical observation is partially resolved by epigenetic mechanisms that maintain heterochromatin, independent of specific DNA sequences. However, it is unclear how satellite DNA sequence variation impacts de novo formation of pericentric heterochromatin, which is initially absent from paternal chromosomes in the zygote. Here we show that satellite variation has functional consequences for zygotic heterochromatin formation, recruitment of the Chromosome Passenger Complex (CPC), and interactions with spindle microtubules. In M. musculus zygotes, Polycomb Repressive Complex 1 (PRC1) is recruited to pericentric satellites by its AT-hook domain, which binds runs of A/T nucleotides, to generate H2AK119ub1 (H2Aub) heterochromatin. By fertilizing M. musculus eggs with sperm from other mouse species, we show that species-specific satellite sequences differ in their ability to recruit PRC1 and form H2Aub. This satellite-DNA mediated increase in PRC1 heterochromatin leads to reduced CPC recruitment and increased microtubule forces on kinetochores. Our results provide a direct link between satellite DNA composition and pericentromere function in the zygote, when epigenetic pathways maintaining pericentromere heterochromatin are absent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6422169819169157447ac8bc6a3224b6f8ab89" target='_blank'>
              Species-specific satellite DNA composition dictates de novo formation of PRC1-mediated pericentric heterochromatin
              </a>
            </td>
          <td>
            Piero Lamelza, Malena Parrado, Emma Hamlin, Micheal A. Lampson
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7ee5fba3b3bfc377211f04436065c1bd9deb16" target='_blank'>
              Elucidating the genotoxicity of Fusobacterium nucleatum-secreted mutagens in colorectal cancer carcinogenesis
              </a>
            </td>
          <td>
            Wenye Xu, Yuchen Zhang, Dongjiao Chen, Dan Huang, Yang Zhao, Wei Hu, Ling Lin, Yingzhi Liu, Shilan Wang, Judeng Zeng, Chuan Xie, H. Chan, Qing Li, Huarong Chen, Xiaodong Liu, Sunny H Wong, Jun Yu, Francis K L Chan, Matthew T V Chan, Siew C Ng, William K. K. Wu, Lin Zhang
          </td>
          <td>2024-09-27</td>
          <td>Gut Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: It is widely accepted that the 3D chromatin organization in human cell nuclei is not random and recent investigations point towards an interactive relation of epigenetic functioning and chromatin (re-)organization. Although chromatin organization seems to be the result of self-organization of the entirety of all molecules available in the cell nucleus, a general question remains open as to what extent chromatin organization might additionally be predetermined by the DNA sequence and, if so, if there are characteristic differences that distinguish typical regions involved in dysfunction-related aberrations from normal ones, since typical DNA breakpoint regions involved in disease-related chromosome aberrations are not randomly distributed along the DNA sequence. Methods: Highly conserved k-mer patterns in intronic and intergenic regions have been reported in eukaryotic genomes. In this article, we search and analyze regions deviating from average spectra (ReDFAS) of k-mer word frequencies in the human genome. This includes all assembled regions, e.g., telomeric, centromeric, genic as well as intergenic regions. Results: A positive correlation between k-mer spectra and 3D contact frequencies, obtained exemplarily from given Hi-C datasets, has been found indicating a relation of ReDFAS to chromatin organization and interactions. We also searched and found correlations of known functional annotations, e.g., genes correlating with ReDFAS. Selected regions known to contain typical breakpoints on chromosomes 9 and 5 that are involved in cancer-related chromosomal aberrations appear to be enriched in ReDFAS. Since transposable elements like ALUs are often assigned as major players in 3D genome organization, we also studied their impact on our examples but could not find a correlation between ALU regions and breakpoints comparable to ReDFAS. Conclusions: Our findings might show that ReDFAS are associated with instable regions of the genome and regions with many chromatin contacts which is in line with current research indicating that chromatin loop anchor points lead to genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3f4228708cdafa938a5ddafca7f4b52c72767a" target='_blank'>
              Peculiar k-mer Spectra Are Correlated with 3D Contact Frequencies and Breakpoint Regions in the Human Genome
              </a>
            </td>
          <td>
            Wisam Mohammed Hikmat, Aaron Sievers, Michael Hausmann, G. Hildenbrand
          </td>
          <td>2024-09-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Mutations in genes involved in DNA damage repair (DDR) often lead to premature aging syndromes. While recent evidence suggests that inflammation, alongside mutation accumulation and cell death, may drive disease phenotypes, its precise contribution to in vivo pathophysiology remains unclear. Here, by modeling Ataxia Telangiectasia (A-T) and Bloom Syndrome in the African turquoise killifish (N. furzeri), we replicate key phenotypes of DDR syndromes, including infertility, cytoplasmic DNA fragments, and reduced lifespan. The link between DDR defects and inflammation is attributed to the activation of the cGAS-STING pathway and interferon signaling by cytoplasmic DNA. Accordingly, mutating cGAS partially rescues germline defects and senescence in A-T fish. Double mutants also display reversal of telomere abnormalities and suppression of transposable elements, underscoring cGAS’s non-canonical role as a DDR inhibitor. Our findings emphasize the role of interferon signaling in A-T pathology and identify the cGAS-STING pathway as a potential therapeutic target for genomic instability syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0762a0a7a477e0941f31815c1aa8f1b5b79a238" target='_blank'>
              The cGAS-STING pathway is an in vivo modifier of genomic instability syndromes
              </a>
            </td>
          <td>
            Marva Bergman, Uri Goshtchevsky, Tehila Atlan, Gwendoline Astre, Ryan Halabi, Hosniyah El, Eitan Moses, A. J. Lemus, BA Benayoun, Y. Tzfati, Ido Ben-Ami, I. Harel
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The zinc finger protein CTCF is ubiquitously expressed and is integral to the regulation of chromatin architecture through its interaction with cohesin. Conversely, CTCFL expression is predominantly restricted to the adult male testis but is aberrantly expressed in certain cancers. Despite their distinct expression patterns, the cooperative and competitive mechanisms by which CTCF and CTCFL regulate target gene expression in spermatocytes and cancer cells remain inadequately understood. In this review, we comprehensively examine the literature on the divergent amino acid sequences, target sites, expression profiles and functions of CTCF and CTCFL in normal tissues and cancers. We further elucidate the mechanisms by which CTCFL competitively or cooperatively binds to CTCF target sites during spermatogenesis and carcinogenesis to modulate chromatin architecture. We mainly focus on the role of CTCFL in testicular and cancer development, highlighting its interaction with CTCF at CTCF binding sites to regulate target genes. In the testis, CTCF and CTCFL cooperate to regulate the expression of testis-specific genes, essential for proper germ cell progression. In cancers, CTCFL overexpression competes with CTCF for DNA binding, leading to aberrant gene expression, a more relaxed chromatin state, and altered chromatin loops. By uncovering the roles of CTCF and CTCFL in spermatogenesis and carcinogenesis, we can better understand the implications of aberrant CTCFL expression in altering chromatin loops and its contribution to disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0158f6b03a46c193781d8f1661a7ed6e9e29" target='_blank'>
              Interaction of CTCF and CTCFL in genome regulation through chromatin architecture during the spermatogenesis and carcinogenesis
              </a>
            </td>
          <td>
            Xin Tong, Yang Gao, Zhongjing Su
          </td>
          <td>2024-10-15</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In this review article we will highlight the evidences that how oncogenes are formed due to the physical genetic variations in proto-oncogenes and tumor suppressor genes and various planned immunotherapies which will include- The immune checkpoint inhibitor-opposing antibodies, adoptive cell treatments, and biologic modifiers (cytokines and vaccines). We will make an effort to provide guidance and potential fixes for these issues, along with pertinent sources for foundational research. For suitable studies, a literature search was undertaken from various database sources such as PubMed, EMBASE, and Google Scholar. One type of gene known as an oncogene-a cellular gene that becomes dysfunctional owing to mutation and overexpression-is the cause of cancer. Certain oncogenes seem to inhibit the homeostatic mechanism by limiting the single cell lineage of leukemia stem cells. According to the clonal theory of oncogenes, tumors are thought to begin in a single cell, Moreover, the growth of tumors is closely linked to the prevention of apoptosis, or programmed cell death. These activities of oncogene can be minimized by some immunological therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77360e601c608aabfa1825689be8b174a0154240" target='_blank'>
              Oncogenetics: Unraveling the Genetic Underpinnings of Cancer for Improved Immunotherapeutic Outcomes.
              </a>
            </td>
          <td>
            Stuti Dwivedi, Praveencumar R, T. Sivakumar, M. K. Posa, R. Dhakar, Ruchi Tiwari
          </td>
          <td>2024-10-29</td>
          <td>Zhongguo ying yong sheng li xue za zhi = Zhongguo yingyong shenglixue zazhi = Chinese journal of applied physiology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Aberrant gene promoter methylation is one of the hallmarks of Acute Myeloid Leukemia (AML). RAD21 is an important gene, implicated in sister chromatids cohesion, DNA repair, the regulation of gene transcription, apoptosis and hematopoiesis. Methods: In this study, we investigate the possible implication of RAD21 promoter methylation in AML pathogenesis using a cohort of AML patients and a cohort of healthy individuals. Results: RAD21 promoter methylation was found in 24% of patients and in none of the controls (p = 0.023), indicating a possible contribution to AML development. Interestingly, a statistically higher frequency of RAD21 methylation was observed in patients with trisomy 8 (9/21, 42.9%, p = 0.021), while none of the patients with aberrations of chromosome 11 had RAD21 gene promoter methylation (0%, 0/11, p = 0.048). Patients with monosomal and complex karyotypes showed low frequencies of RAD21 methylation (7.7% and 15.4%, respectively) without reaching statistical significance. Moreover, ASXL1 mutations were not found to be associated with RAD21 methylation. Conclusions: This is the first study which provides evidence for a possible pathogenetic role of RAD21 promoter methylation in AML development and especially in AML with trisomy 8. Further studies of RAD21 promoter methylation in large series of different AML genetic subgroups may contribute to the elucidation of AML pathogenesis and to the identification of new epigenetic biomarkers with diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c902aea9aee61d29bf4e327d8db03e3f748b0fb1" target='_blank'>
              Cohesin RAD21 Gene Promoter Methylation in Patients with Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Manola, Sophia Zachaki, Katerina Kakosaiou, Agapi Ioannidou, Marina Kalomoiraki, T. Rampias
          </td>
          <td>2024-10-01</td>
          <td>Life</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD. Methods Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration. Results In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment. Conclusion ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4ff74f564f7b2a57419b04c8a2f80af41275c3" target='_blank'>
              Multiregional transcriptomic profiling provides improved prognostic insight in localized non-small cell lung cancer
              </a>
            </td>
          <td>
            Chenyang Li, Thinh T. Nguyen, Jian-Rong Li, Xingzhi Song, J. Fujimoto, L. Little, Curtis Gumb, C. Chow, Ignacio I. Wistuba, Andrew P Futreal, Jianhua Zhang, Shawna M. Hubert, J. Heymach, Jia Wu, Christopher I Amos, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-10-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The field cancerization theory suggests that a group of cells containing oncogenic mutations are predisposed to transformation1, 2. We previously identified single cells in BRAFV600E;p53-/-zebrafish that reactivate an embryonic neural crest state before initiating melanoma3–5. Here we show that single cells reactivate the neural crest fate from within large fields of adjacent abnormal melanocytes, which we term the “cancer precursor zone.” These cancer precursor zone melanocytes have an aberrant morphology, dysplastic nuclei, and altered gene expression. Using single cell RNA-seq and ATAC-seq, we defined a distinct transcriptional cell attractor state for cancer precursor zones and validated the stage-specific gene expression initiation signatures in human melanoma. We identify the cancer precursor zone driver, ID1, which binds to TCF12 and inhibits downstream targets important for the maintenance of melanocyte morphology and cell cycle control. Examination of patient samples revealed precursor melanocytes expressing ID1, often surrounding invasive melanoma, indicating a role for ID1 in early melanomagenesis. This work reveals a surprising field effect of melanoma initiation in vivo in which tumors arise from within a zone of morphologically distinct, but clinically covert, precursors with altered transcriptional fate. Our studies identify novel targets that could improve early diagnosis and prevention of melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96eaad85df9cc9f05779363a3326ab27df67629f" target='_blank'>
              An attractor state zone precedes neural crest fate in melanoma initiation
              </a>
            </td>
          <td>
            Alicia M. McConnell, Maggie H. Chassé, H. Noonan, Jeffrey K. Mito, Julia Barbano, Erika Weiskopf, Irene F. Gosselink, Meera Prasad, Song Yang, P. Abarzua, C. G. Lian, George F. Murphy, C. Trapnell, Leonard I. Zon
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c42616846e496900baa7e96476fa73f42799ad" target='_blank'>
              APC mutations dysregulate alternative polyadenylation in cancer
              </a>
            </td>
          <td>
            Austin M. Gabel, Andrea E. Belleville, James D Thomas, Jose Mario Bello Pineda, Robert K Bradley
          </td>
          <td>2024-10-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endogenous retroelements (EREs), which comprise half of the human genome, play a pivotal role in genome dynamics. Some EREs retained the ability to encode proteins, although most degenerated or served as a source for novel genes and regulatory elements during evolution. Despite ERE repression mechanisms developed to maintain genome stability, widespread pervasive ERE activation is observed in cancer including hematological malignancies. Challenging the perception of noncoding DNA as "junk," EREs are underestimated contributors to cancer driver mechanisms as well as antitumoral immunity by providing innate immune ligands and tumor antigens. This review highlights recent progress in understanding ERE co-option events in cancer and focuses on the controversial debate surrounding their causal role in shaping malignant phenotype. We provide insights into the rapidly evolving landscape of ERE research in hematological malignancies and their clinical implications in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70dce85bfb8495c63680f246f15584cd08410198" target='_blank'>
              Endogenous retroelements in hematological malignancies: From epigenetic dysregulation to therapeutic targeting.
              </a>
            </td>
          <td>
            Mohamed Chour, Françoise Porteu, S. Depil, V. Alcazer
          </td>
          <td>2024-10-10</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 High-grade gliomas are among the deadliest of all childhood cancers and the leading cause of cancer-related mortality in children. The standard treatment is resection followed by chemotherapy, most often temozolomide and radiation therapy. Unfortunately, there is no evidence that temozolomide influences survival in pediatric patients and only a limited number of new drugs have been identified for specific subgroups of tumors. The identification of fusion genes as possible drivers of the disease offers an opportunity for targeted therapies. We present a case of a six-year-old patient with a novel TRIM24::NTRK2 fusion positive epithelioid glioblastoma with subsequent relapses.



 We investigated the ramification of this fusion and evolution of the tumor through whole genome sequencing, RNA-sequencing and DNA methylation profiling, and studied the effect of the patient’s treatment using patient-derived cell cultures.



 Our studies show that the TRIM24::NTRK2 fusion has oncogenic abilities but loses its importance as the tumor evolves. In addition, we demonstrate an increased drug resistance over time, further indicating the diminished driver function of the fusion.



 The use of resequencing revealed parallel development of tumor subclones with different genomic alterations, highlighting the importance of genomic profiling from various parts of the tumor.



 This work was supported by the Swedish Research Council, the Swedish Childhood Cancer Foundation, and the Swedish Cancer Society
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6825c06ca5f91e7d7121415ac1f08b97ce06cc4e" target='_blank'>
              P16.10.A UNRAVELING THE CLINICAL IMPLICATIONS OF TRIM24::NTRK2 FUSION GENE IN PEDIATRIC HIGH-GRADE GLIOMA: A CASE STUDY
              </a>
            </td>
          <td>
            I. Karlsson, S. Keane, L. Deland, A. Lindlöf, T. Kling, T. Olsson Bontell, H. Fagman, J. Nilsson, D. Nilsson, M. Sabel, H. Carén, F. Abel
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The molecular biology of pituitary neuroendocrine tumors (PitNETs) revealed few recurrent mutations and extensive chromosomal alterations, with the latter being the driving force in a subset of these lesions. Addressing the need for an easily applicable diagnostic tool, we conducted a retrospective study of 61 PitNETs operated at a tertiary care center. All cases were subtyped according to the 2022 WHO classification of endocrine tumors. A genome wide NGS panel targeting 1500 single-nucleotide polymorphisms (SNP) was used to classify chromosomal imbalances, loss of heterozygosity and copy number variations in DNA from formalin fixed paraffin embedded tissues. We identified four distinct chromosomal patterns, with varying distribution among different tumor lineages. Forty-two of 61 (69%) PitNETs showed chromosomal alterations. Gonadotroph PitNETs showed mostly quiet genomes. The majority of lactotroph PitNETs (19/20, 95%) were altered, exhibiting a gained genome and a remarkably low recurrence rate. Nine of ten (90%) corticotroph PitNETs harbored chromosomal alterations, of which two aggressive corticotroph tumors and one metastatic corticotroph PitNET showed massive chromosomal losses leading to near haploid/near homozygous genomes. The comparison of the molecular profile of primary and recurrent PitNETs of five patients showed no significant accumulation of alterations over time. A simple genome wide 1500 SNP test can be used in the identification of outspoken aggressive subsets of PitNET by the occurrence of a near haploid/near homozygous genome. Furthermore, the presence of neoplastic tissue in resected material can be potentially confirmed for non-gonadotroph PitNETs in suboptimal histological assessment conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb2fdf64cf9cd448595700bdb95a06c56a9e233" target='_blank'>
              Chromosomal Alteration Patterns in PitNETs: Massive Losses in Aggressive Tumors.
              </a>
            </td>
          <td>
            Maaia-Margo Jentus, L. Bakker, M. Verstegen, Iris Pelsma, T. van Wezel, D. Ruano, E. Kapiteijn, S. Crobach, N. Biermasz, H. Morreau
          </td>
          <td>2024-10-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Chromatin, with its complex spatial and temporal organization, plays a crucial role in regulating gene expression. Recent advancements in super-resolution microscopy have revealed that nanoscale domains of heterochromatin (repressed segments) embedded within a euchromatin (active segments) background are fundamental units of 3D chromatin organization. In tissue-resident cells, the size of these heterochromatin domains varies with the microenvironment, particularly its stiffness, and chromatin organization is also influenced by pharmacological and epigenetic drugs. However, the mechanisms governing heterochromatin domain size under various conditions and their impact on gene expression remain unclear. To address this knowledge gap, we have developed a dynamic, next-generation sequencing informed chromatin copolymer model. Our model simulates the spatiotemporal evolution of chromatin, driven by passive diffusion and active epigenetic reactions, which interconvert euchromatin and heterochromatin. By integrating chromatin-chromatin interaction energetics and diffusion-reaction dynamics, we predict the formation of nanoscale heterochromatin-rich domains and establish a scaling relationship between their size and the modulation of epigenetic reaction rates. Additionally, our model predicts that epigenetic and chromatin compaction changes in response to changes in global reaction rates occur predominantly at domain boundaries. We validated these predictions via Hi-C contact map analysis and super-resolution imaging of hyperacetylated melanoma cells. Subsequent RNA-seq analysis suggested a pivotal role of these epigenetic shifts in influencing the metastatic potential of these cells. We further validated our mesoscale findings against chromatin rearrangement in hMSCs, which exhibit sensitivity of epigenetic reaction rates to changes in microenvironmental stiffness. Finally, we evaluated the effects of cycling of epigenetic reaction rates in silico, mimicking the cellular transition to different extracellular conditions, and back again. This finding reveals a cell-type invariant mechanism driven by domain boundaries, whereby chromatin organization guides epigenetic memory formation. Our findings show that chromatin reorganization in response to changes in epigenetic reaction rates resulting from alterations in the microenvironment, drug exposure and disease progression impacts both immediate cellular responses and long-term epigenetic memory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e55d9db8c8d7ff0628f2ac7f62c8baea4b57b" target='_blank'>
              Polymer Model Integrates Super-Resolution Imaging and Epigenomic Sequencing to Elucidate the Role of Epigenetic Reactions in Shaping 4D Chromatin Organization
              </a>
            </td>
          <td>
            Vinayak Vinayak, Ramin Basir, Rosela Golloshi, Joshua Toth, Lucas E. Sant’Anna, Melike Lakadamyali, R. McCord, Vivek B. Shenoy
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Meiotic recombination is a prominent force shaping genome evolution, and understanding the causes for varying recombination landscapes within and between species has remained a central, though challenging, question. Recombination rates are widely observed to negatively associate with the abundance of transposable elements (TEs), selfish genetic elements that move between genomic locations. While such associations are usually interpreted as recombination influencing the efficacy of selection at removing TEs, accumulating findings suggest that TEs could instead be the cause rather than the consequence. To test this prediction, we formally investigated the influence of polymorphic, putatively active TEs on recombination rates. We developed and benchmarked a novel approach that uses PacBio long-read sequencing to efficiently, accurately, and cost-effectively identify crossovers (COs), a key recombination product, among large numbers of pooled recombinant individuals. By applying this approach to Drosophila strains with distinct TE insertion profiles, we found that polymorphic TEs, especially RNA-based TEs and TEs with local enrichment of repressive marks, reduce the occurrence of COs. Such an effect leads to different CO frequencies between homologous sequences with and without TEs, contributing to varying CO maps between individuals. The suppressive effect of TEs on CO is further supported by two orthogonal approaches–analyzing the distributions of COs in panels of recombinant inbred lines in relation to TE polymorphism and applying marker-assisted estimations of CO frequencies to isogenic strains with and without transgenically inserted TEs. Our investigations reveal how the constantly changing mobilome can actively modify recombination landscapes, shaping genome evolution within and between species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a72fef68d5ba8cf175dc6cdc09958ca9cc7bf824" target='_blank'>
              Varying recombination landscapes between individuals are driven by polymorphic transposable elements
              </a>
            </td>
          <td>
            Yuheng Huang, Yi Gao, Kayla Ly, Leila Lin, Jan-Paul Lambooij, Elizabeth G. King, Aniek Janssen, Kevin H.-C. Wei, Y. C. G. Lee
          </td>
          <td>2024-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The CHAMP1 complex, a little-known but highly conserved protein complex consisting of CHAMP1, POGZ, and HP1α, is enriched in heterochromatin though its cellular function in these regions of the genome remain unknown. Here we show that the CHAMP complex promotes heterochromatin assembly at multiple chromosomal sites, including centromeres and telomeres, and promotes homology-directed repair (HDR) of DNA double strand breaks (DSBs) in these regions. The CHAMP1 complex is also required for heterochromatin assembly and DSB repair in highly-specialized chromosomal regions, such as the highly-compacted telomeres of ALT (Alternative Lengthening of Telomeres) positive tumor cells. Moreover, the CHAMP1 complex binds and recruits the writer methyltransferase SETDB1 to heterochromatin regions of the genome and is required for efficient DSB repair at these sites. Importantly, peripheral blood lymphocytes from individuals with CHAMP1 syndrome, an inherited neurologic disorder resulting from heterozygous mutations in CHAMP1, also exhibit defective heterochromatin clustering and defective repair of local DSBs, suggesting that a defect in DNA repair underlies this syndrome. Taken together, the CHAMP1 complex has a novel role in heterochromatin assembly and the enhancement of HDR in heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454e4622c260f66b26c8cb18854b8ca092214e10" target='_blank'>
              CHAMP1 complex directs heterochromatin assembly and promotes homology-directed DNA repair
              </a>
            </td>
          <td>
            Feng Li, Tianpeng Zhang, Aleem Syed, Amira Elbakry, Noella Holmer, Huy Nguyen, Sirisha Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="T cells recognize an immense spectrum of pathogens to initiate immune responses by means of a large repertoire of T cell receptors (TCRs) that arise from somatic rearrangements of variable, diversity and joining gene segments at the TCR loci. These gene segments have emerged from a limited number of ancestral genes through a series of gene duplication events, resulting in a greatly variable number of such genes across different species. Apart from the complete V(D)J gene annotations in the human and mouse reference assemblies, little is known about the structure of TCR loci in other species. Here, we performed a comprehensive comparison of the TCRα and TCRβ gene segment clusters in mice and three of its closely related sister species. We show that the TCRα variable gene cluster is frequently rearranged, leading to deletions and sequence inversions in this region. The resulting complexity of TCR loci severely complicates the assembly of these loci and the annotation of gene segments. By jointly utilizing genomic and transcriptomic data, we show that in Mus musculus castaneus the variable gene cluster at the α locus has undergone a recent major locus contraction, leading to the loss of 74 variable gene segments. Additionally, we validated the expression of functional variable genes, including atypical ones with inverted orientation relative to other such segments. Disentangling the fine-scale structure of TCR loci in different species can provide valuable insights in the evolution and diversity of TCR repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/050dd2ec4e870352e83af91f6ed80d0e6588c1e2" target='_blank'>
              Distinct evolution at TCRα and TCRβ loci in the genus Mus
              </a>
            </td>
          <td>
            Moritz Peters, Volker Soltys, Dingwen Su, Y. F. Chan
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma and ependymoma are prevalent pediatric central nervous system tumors with significant molecular and clinical heterogeneity. We collected bulk RNA sequencing data from 888 medulloblastoma and 370 ependymoma tumors to establish a comprehensive reference landscape. Following rigorous batch effect correction, normalization, and dimensionality reduction, we constructed a unified landscape to explore gene expression, signaling pathways, gene fusions, and copy number variations. Our analysis revealed distinct clustering patterns, including two primary ependymoma compartments, EPN-E1 and EPN-E2, each with specific gene fusions and molecular signatures. In medulloblastoma, we achieved precise stratification of Group 3/4 tumors by subtype and in SHH tumors by patient age. Our landscape serves as a vital resource for identifying biomarkers, refining diagnoses, and enables the mapping of new patients’ bulk RNA-seq data onto the reference framework to facilitate accurate disease subtype identification. The landscape is accessible via Oncoscape, an interactive platform, empowering global exploration and application. One Sentence Summary A landscape built using only Transcriptomic analysis for medulloblastoma and ependymoma reveals novel insights about subtype specific biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7340d3bdec67105d60c89f8f44d73e7f797360" target='_blank'>
              Transcriptomic landscape identifies two unrecognized ependymoma subtypes and novel pathways in medulloblastoma
              </a>
            </td>
          <td>
            Sonali Arora, Nicholas Nuechterlein, Matt Jensen, Gregory Glatzer, P. Sievers, S. Varadharajan, A. Korshunov, Felix Sahm, Stephen C. Mack, Michael D Taylor, Eric C. Holland
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="The BRCA2 tumor suppressor protein has roles in homologous recombination DNA repair and replication fork protection and carriers of mutation in the BRCA2 gene (BRCA2mut/+) are at high risk for breast cancers as a consequence of genomic instability. Mammary epithelial cells (MECs) are subjected to significant hormone-driven replication. BRCA2mut/+ MECs must therefore be able to mitigate RS-induced chronic DNA damage that could result in cell death or senescence. In response to acute RS in hTERT-MECs, we have found that the EGFR ligand, amphiregulin (AREG) rapidly undergoes retrograde trafficking to the nucleus and nuclear membrane (NM) in association with lamin A resulting in a transient increase in H3K9me3 heterochromatin most markedly in BRCA2mut/+ hTERT-MECs. RS promoted NM incorporation of prelamin A that was dependent on endogenous but not exogenous AREG. Prelamin A is reported to stabilize SUV39h1 and expression of a C-terminal truncated AREG that translocates to the NM increased SUV39h1, HP1α and H3K9me3. AREG knockdown reduced both HP1α and SUV39h1 accompanied by massive decompaction and reduction in global H3K9me3 heterochromatin. Loss of heterochromatin upon AREG depletion increased DNA damage, reduced replication fork speed and resulted in a significant increase in firing of new and multiple origins. Knockdown of endogenous AREG also impacted the NM resulting in activation of a type-1 interferon (IFN)-like response promoting senescence. Chronic RS also stimulated nuclear AREG along with FOXM1 and ESR1 mRNA and protein. Consistent with this, nuclear AREG, FOXM1 and ERα were detected in hyperplastic regions of human tissue. Overall, our work reveals a critical and unanticipated nuclear role for AREG in maintenance and transient induction of heterochromatin in response to RS preventing excessive DNA damage and senescence in BRCA2mut/+ MECs. Since nuclear AREG also promotes ERα expression, this may link RS with aberrant expansion of estrogen receptor-expressing MECs found in BRCA2mut/+ mammary tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7115be9e7c57e4cfb92a7b960c20389981e78db4" target='_blank'>
              Intracellular amphiregulin is critical for heterochromatin maintenance and genomic stability in response to replication stress in BRCA2mut/+mammary epithelial cells
              </a>
            </td>
          <td>
            Tianqi Jiang, A. Zein, M.A. Christine Pratt
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="The Rad18 E3 ubiquitin ligase, a non-essential gene, is a key regulator of DNA damage tolerance that also functions in repair of DNA double strand breaks. Rad18 is overexpressed in the aggressive brain cancer glioblastoma (GBM) and its downregulation sensitizes glioblastoma cells to DNA damaging agents. Here we show that Rad18 has an essential role in GBM cells proliferation in the absence of external damage, surprisingly independent of its catalytic activity. Rad18 downregulation leads to cell cycle arrest in the G1 phase in the absence of apparent DNA damage. We also show that Rad18 sustains GBM stem cells self-renewal and survival, as well as the growth of tumor orthotropic xenografts in mice. We also show that increased Rad18 expression enhances the growth of non-transformed cells and induces features of oncogenic transformation. Mechanistically, we show that Rad18 downregulation negatively regulates the Hippo pathway by interfering with the nuclear retention of the YAP1 transcription factor. Altogether, these data show that Rad18 has an essential, non-catalytic function, in GBM proliferation, and propose Rad18 as a key target to sensitize GBM to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6505ff011c6256d84307c2ba21b28fe90044ba14" target='_blank'>
              A non-catalytic function for Rad18 in sustaining glioblastoma proliferation
              </a>
            </td>
          <td>
            Chames Kermi, Nour Benbahouche, Lenka Stefancikova, A. Siegfried, J. Pascussi, Julie Pannequin, Jérôme Moreaux, Tom Egger, Jean-Philippe Hugnot, Marie-Bernadette Delisle, Elizabeth Moyal, Emmanuelle Uro-Coste, Domenico Maiorano
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Transposable elements (TEs) are key contributors to genetic novelty. Despite increasing evidence of their importance, their roles in shaping the regulatory landscape of diverse immune cell populations remain largely unclear. Using single-cell multiome data from human peripheral blood mononuclear cells, we annotated the cell-specific cis-regulatory elements for major immune cell populations and identified a highly cell-specific signature of the overrepresented TE families. Focusing on monocytes that bear fast-evolving transcriptomes, we found that high proportions of their enhancers are TE-derived and bound by multiple pioneer transcription factors. Among them, we confirmed that the core myeloid regulator SPI1 can bind and regulate hundreds of TE-derived enhancers, which further affect the expression of adjacent immune genes. Additionally, interspecies comparison reveals that non-conserved monocyte enhancers are frequently generated by lineage-specific TE insertions, and correlate with the evolved gene expression between human and mouse. Overall, our study supports the importance of TEs in shaping the regulatory landscape of diverse immune cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec49246bea325aa9a378a750ef1fb9175032369" target='_blank'>
              Unraveling transposable element-mediated regulatory landscape in diverse human immune cells
              </a>
            </td>
          <td>
            Cui-Qiong Du, Hairui Fan, Jiayao Jiang, Juan Yang, Shuai Chen, Nadezhda E. Vorobyeva, Congrui Zhu, Liming Mao, Chenxi Li, Yanhua Li, Wen-hua Bao, Ming-an Sun
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8832647f7b2ef1441658b906011887e6691f5bc" target='_blank'>
              Bioinformatic analysis of molecular characteristics and oncogenic features of CARD14 in human cancer
              </a>
            </td>
          <td>
            Daniil Bespalov, Dayana Pino, Sònia Vidal-Guirao, Júlia Franquesa, Daniel Lopez-Ramajo, Ingrid Filgaira, Li Wan, Paul A O'Sullivan, Steve C Ley, Sonia Forcales, Juan José Rojas, Mercè Izquierdo-Serra, Concepció Soler, Joan Manils
          </td>
          <td>2024-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Ovarian cancer is one of the most lethal gynecological malignancies worldwide, posing immense challenges in early detection, displaying grim prognoses, and having high rates of recurrence and mortality. This malignancy exhibits a range of ethnic diversity-mortality rates, each with distinct sub-categories characterized by variations in both molecular attributes and clinical behavior. The most prevalent and also most aggressive among the various ovarian cancer sub-types is high-grade serous ovarian carcinoma (HGSOC). This carcinoma is marked by genomic instability, resulting in the malfunction of tumor suppressor genes and oncogenes, caused by homologous recombination deficiency, copy number variations, chromosomal abnormalities, and epigenetic changes. The pursuit of understanding and targeting genomic instability has significantly guided approaches to screening, prognosis, and therapeutic strategies in ovarian cancer. Further insights from molecular investigations have unveiled the regulatory role of microRNAs (short non-coding RNAs) in modulating the expression of target tumor suppressors and oncogenes by interacting with messenger RNA (mRNA). Studies have shown that miRNAs are dysregulated in ovarian cancer, however, no data has shown significant relationships between these miRNAs to the patient demographics, including race/ethnicity. miRNAs can alter oncogenes, and tumor suppressor gene expression levels, serving as early biomarkers. We utilized the TCGA database from the University of Alabama at Birmingham (UALCAN) software to analyze the expressions of targeted genes and miRNAs. We determined various oncogenes and tumor suppressor genes with higher statistical significance on survival rates, race, cancer stages, tumor grades, and miRNAs linked to them. Additionally, genomic instability was assessed using aneuploidy score, fraction of genome altered, mutation counts, mutation burden, and prognosis (i.e., overall survival and progression-free survival), downloaded from cBioPortal, and the genomic instability parameters were calculated by Spearman correlation, and represented by a heatmap. Further, DNA and RNA from 22 HGSOC samples of paraffin-embedded tissues of ovarian cancer patients were collected for RT-qPCR assays for several miRNAs. Of the genes analyzed, BRCA1, BRAF, CCNE1, and miRNAs linked to these genes showed statistically significant data on survival rates. BRAF/CCNE1 is upregulated while BRCA 1 is downregulated in ovarian high-grade serous carcinoma. We also found quantitative differential expressions of some miRNAs analyzed in African Americans versus White Americans. Successful identification of early molecular markers in ovarian cancer will potentially allow for early detection and decreased mortality rates with HGSOC. Analysis of the appearance and abundance of miRNAs by qPCR assay in early-stage ovarian cancer can aid as an early diagnostic and prognostic tool. In a further study, we hope to verify our results by testing a larger cohort of samples from African Ancestry.
 Citation Format: Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga. miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f1e0052202e2129f599047f6e77f66413139b6" target='_blank'>
              Abstract C075: miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry
              </a>
            </td>
          <td>
            Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac0738a96662e313aea5fd9200de2f16be79bfc" target='_blank'>
              R-loops’ m6A modification and its roles in cancers
              </a>
            </td>
          <td>
            Yue Qiu, Changfeng Man, Luyu Zhu, Shiqi Zhang, Xiaoyan Wang, Dandan Gong, Yu Fan
          </td>
          <td>2024-10-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0362aabb9265fe87bbca700d69f60f98a5d89595" target='_blank'>
              Steering research on mRNA splicing in cancer towards clinical translation.
              </a>
            </td>
          <td>
            Olga Anczuków, Frédéric H-T Allain, B. Angarola, Douglas L. Black, Angela N. Brooks, Chonghui Cheng, Ana Conesa, Edie I. Crosse, E. Eyras, Ernesto Guccione, Sydney X Lu, Karla M Neugebauer, Priyanka Sehgal, Xiao Song, Zuzana Tothova, Juan Valcárcel, Kevin M. Weeks, Gene W Yeo, A. Thomas-Tikhonenko
          </td>
          <td>2024-10-09</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="C10orf90, a tumor suppressor, can inhibit the occurrence and development of tumors. Therefore, we investigated the gene function of C10orf90 in various tumors using multiple pan-cancer datasets. Pan-cancer analysis results reveal that the expression levels of C10orf90 vary across different tumors and hold significant value in the clinical diagnosis and prognosis of patients with various tumors. In some cancers, the expression level of C10orf90 is correlated with CNV, DNA methylation, immune subtypes, immune cell infiltration, and drug sensitivity in the tumors. In particular, in COAD, the C10orf90 gene is implicated in multiple processes associated with COAD. Cell experiments demonstrate that C10orf90 suppresses the proliferation and migration of colon cancer cells while promoting apoptosis. In summary, C10orf90 plays a role in the onset and progression of various cancers and could potentially serve as an effective diagnostic and prognostic marker for cancer patients. Notably, in COAD, C10orf90 inhibits the proliferation and migration of colon cancer cells, induces apoptosis, and is linked to the advancement of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1559c858c67c63e394ff32e5068255c695dededf" target='_blank'>
              Novel Oncogenic Value of C10orf90 in Colon Cancer Identified as a Clinical Diagnostic and Prognostic Marker
              </a>
            </td>
          <td>
            Chuangdong Ruan, Yuqin Zhang, Daoyang Chen, Mengyi Zhu, Penghui Yang, Rongxin Zhang, Yan Li
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA topoisomerase II (TOP2) is an enzyme that resolves DNA topological problems arising in various nuclear processes, such as transcription. Aclarubicin, a member of the anthracyclines, is known to prevent the association of TOP2 with DNA, inhibiting the early step of TOP2 catalytic reactions. During our research on the subnuclear distribution of human TOP2B, we found that aclarubicin affects the mobility of TOP2B in the nucleus. FRAP analysis demonstrated that aclarubicin decreased the nuclear mobility of EGFP-tagged TOP2B in a concentration-dependent manner. Aclarubicin exerted its inhibitory effects independently of TOP2B enzymatic activities: TOP2B mutants defective for either ATPase or topoisomerase activity also exhibited reduced nuclear mobility in the presence of aclarubicin. Immunofluorescence analysis showed that aclarubicin antagonized the induction of DNA damage by etoposide. Although the prevention of the TOP2-DNA association is generally considered a primary action of aclarubicin in TOP2 inhibition, our findings highlight a previously unanticipated effect of aclarubicin on TOP2B in the cellular environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a0224dd55993d9d812699857cf7d079f645a3e5" target='_blank'>
              Aclarubicin Reduces the Nuclear Mobility of Human DNA Topoisomerase IIβ
              </a>
            </td>
          <td>
            Keiko Morotomi‐Yano, Ken-ichi Yano
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract The heterogeneity of tumor clones drives the selection and evolution of distinct tumor cell populations, resulting in an intricate and dynamic tumor evolution process. While tumor bulk DNA sequencing helps elucidate intratumor heterogeneity, challenges such as the misidentification of mutation multiplicity due to copy number variations and uncertainties in the reconstruction process hinder the accurate inference of tumor evolution. In this study, we introduce a novel approach, REconstructing Tumor Clonal Heterogeneity and Evolutionary Relationships (RETCHER), which characterizes more realistic cancer cell fractions by accurately identifying mutation multiplicity while considering uncertainty during the reconstruction process and the credibility and reasonableness of subclone clustering. This method comprehensively and accurately infers multiple forms of tumor clonal heterogeneity and phylogenetic relationships. RETCHER outperforms existing methods on simulated data and infers clearer subclone structures and evolutionary relationships in real multisample sequencing data from five tumor types. By precisely analysing the complex clonal heterogeneity within tumors, RETCHER provides a new approach to tumor evolution research and offers scientific evidence for developing precise and personalized treatment strategies. This approach is expected to play a significant role in tumor evolution research, clinical diagnosis, and treatment. RETCHER is available for free at https://github.com/zlsys3/RETCHER.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f822108aa29d6372cc1d56eba22e5f8f0a1fffc3" target='_blank'>
              Reconstructing tumor clonal heterogeneity and evolutionary relationships based on tumor DNA sequencing data
              </a>
            </td>
          <td>
            Zhen Wang, Yanhua Fang, Ruoyu Wang, Liwen Kong, Shanshan Liang, Shuai Tao
          </td>
          <td>2024-09-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4200779de8ffeea14186de4448271c5a8d0e9f0" target='_blank'>
              Developmental-status-aware transcriptional decomposition establishes a cell state panorama of human cancers
              </a>
            </td>
          <td>
            Yikai Luo, Han Liang
          </td>
          <td>2024-10-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The genus Ilex belongs to the sole family and is the single genus within the order Aquifoliales, exhibiting significant phenotypic diversity. However, the genetic differences underlying these phenotypic variations have rarely been studied. In this study, collinearity analyses of three Ilex genomes, Ilex latifolia Thunb., Ilex polyneura (Hand.-Mazz.) S. Y. Hu, and Ilex asprella Champ. ex Benth., indicated a recent fusion event contributing to the reduction of chromosomes in I. asprella. Comparative genome analyses showed slight differences in gene annotation among the three species, implying a minimal disruption of genes following chromosomal fusion in I. asprella. Comprehensive annotation of transposable elements (TEs) revealed that TEs constitute a significant portion of the Ilex genomes, with LTR transposons being predominant. TEs exhibited an inverse relationship with gene density, potentially influencing gene regulation and chromosomal architecture. TE insertions were shown to affect the conformation and binding sites of key genes such as 7-deoxyloganetin glucosyltransferase and transmembrane kinase (TMK) genes, highlighting potential functional impacts. The structural variations caused by TE insertions suggest significant roles in the evolutionary dynamics, leading to either loss or gain of gene function. This study underscores the importance of TEs in shaping the genomic landscape and evolutionary trajectories of Ilex species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/603a3cf5c3cd4429400b92b277299162fe5c83b5" target='_blank'>
              Impact of Chromosomal Fusion and Transposable Elements on the Genomic Evolution and Genetic Diversity of Ilex Species
              </a>
            </td>
          <td>
            Zhenxiu Xu, Haikun Wei, Mingyue Li, Yingjie Qiu, Lei Li, Kewang Xu, Zhonglong Guo
          </td>
          <td>2024-09-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Colorectal cancer, a prevalent gastrointestinal carcinoma, has a high risk for recurrence when locally advanced and remains lethal when in an advanced stage. Prognostic biomarkers may help in better delineating the aggressiveness of this disease in individual patients and help to tailor appropriate therapies. CDX2, a transcription factor of gastrointestinal differentiation, has been proposed as a biomarker for good outcomes and could also be a marker of specific sub-types amenable to targeted therapies. Methods: Colorectal cancers from The Cancer Genome Atlas (TCGA) colorectal cohort and colon cancers from the Sidra-LUMC AC-ICAM cohort were categorized according to their expressions of CDX2 mRNA. Groups with CDX2 suppression were compared with cancers showing no suppression regarding their clinical and genomic characteristics. Results: CDX2-suppressed colorectal cancers showed a high prevalence of Microsatellite Instability (MSI) and a lower prevalence of chromosomal Instability (CIN) compared to non-CDX2-suppressed cancers. In addition, CDX2-suppressed cancers had a higher prevalence of mutations in several receptor tyrosine kinase genes, including EGFR, ERBB3, ERBB4, RET, and ROS1. In contrast, CDX2-suppressed cancers displayed lower mutation frequencies than non-CDX2-suppressed cancers in the genes encoding for the two most frequently mutated tumor suppressors, APC and TP53, and the most frequently mutated colorectal cancer oncogene, KRAS. However, CDX2-suppressed colorectal cancers had a higher prevalence of mutations in alternative genes of the WNT/APC/β-catenin and KRAS/BRAF/MEK pathways. In addition, they showed frequent mutations in DNA damage response (DDR) genes, such as BRCA2 and ATM. Conclusion: CDX2-suppressed colorectal cancers constitute a genomically distinct subset of colon and rectal cancers that have a lower prevalence of KRAS, APC, and TP53 mutations, but a high prevalence of mutations in less commonly mutated colorectal cancer genes. These alterations could serve as targets for personalized therapeutics in this subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f07fd50c1ab7d2b069279a733f924c136fa063e" target='_blank'>
              CDX2-Suppressed Colorectal Cancers Possess Potentially Targetable Alterations in Receptor Tyrosine Kinases and Other Colorectal-Cancer-Associated Pathways
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-10-01</td>
          <td>Diseases</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The p53 tumor suppressor is an indispensable regulator of DNA damage responses that accelerates carcinogenesis when mutated. In this report, we uncover a new mechanism by which p53 maintains genomic integrity in the absence of canonical DNA damage response activation. Specifically, loss of p53 dramatically alters chromatin structure at the nuclear periphery, allowing increased transmission of an environmental carcinogen, ultraviolet (UV) radiation, into the nucleus. Genome-wide mapping of UV-induced DNA lesions in p53-deficient primary cells reveals elevated lesion abundance in regions corresponding to locations of high mutation burden in malignant melanomas. These findings uncover a novel role of p53 in the suppression of mutations that contribute to cancer and highlight the critical influence of nuclear architecture in regulating sensitivity to carcinogens. One-Sentence Summary The p53 tumor suppressor reduces carcinogen sensitivity and mutagenic potential by maintaining nuclear architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce95350c5fb375408fd587e0eea1b652005b9ec" target='_blank'>
              p53 Regulates Nuclear Architecture to Reduce Carcinogen Sensitivity and Mutagenic Potential
              </a>
            </td>
          <td>
            Devin A. King, Dakota E. McCoy, Adrian Perdyan, Jakub Mieczkowski, Thierry Douki, Jennifer A. Dionne, Rafael E. Herrera, Ashby J. Morrison
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="MiR-210 is widely recognized as the quintessential hypoxia-responsive miRNA and thought to fine-tune various facets of cellular homeostasis. We hereby present an integrative appraisal of phenotypic and molecular repercussions of disrupting the corresponding locus in human and mouse cells using multiple genetic strategies. Briefly, MIR210 deletion led to decreased cellular fitness and suboptimal responses to several stress types. Transcriptomic comparisons using different profiling platforms, performed independently by members of this collaboration, revealed consistent deregulation of neighboring genes, in locus-disrupted cells. Interestingly, the anticipated enrichment in miR-210 targets failed to materialize in unbiased analyses. Our results point to the biological significance of unrecognized regulatory elements that overlap miRNA genes and should serve as note of caution for studies based for genetic disruption of such loci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/577bbb022ac6c0f286aa80c30f22e14f6aef5719" target='_blank'>
              miR-210 locus deletion disrupts cellular homeostasis; an integrated genetic study
              </a>
            </td>
          <td>
            M. Preda, Evelyn Gabriela Rusu-Nastase, Carmen Alexandra Neculachi, Xiaoling Zhong, C. Voellenkle, Nathalie M. Mazure, O. Bălăcescu, Cristina Ivan, Xiao-Wei Zheng, Mihaela Gherghiceanu, K. Lebrigand, Maya Simionescu, Fabio Martelli, Bernard Mari, S. Catrina, A. Burlacu, Mircea Ivan
          </td>
          <td>2024-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b023a760b40f6707fa684ea79bd76ea869f1a" target='_blank'>
              PER3 promoter hypermethylation correlates to the progression of pan-cancer
              </a>
            </td>
          <td>
            Yaoxu Li, Wenjuan Li, Jinhai Deng, Mingzhu Yin
          </td>
          <td>2024-10-14</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) has different molecular subgroups related to different risks and treatments. Tumor biopsies for BC detection are invasive and may not reflect tumor heterogeneity. Liquid biopsies have become relevant because they might overcome these limitations. We rationalize that liquid cfDNA biopsies through shallow whole genome sequencing (sWGS) could improve the detection of tumor alterations, complementing the genomic profiling. We evaluated the feasibility to detect somatic copy number alterations (SCNAs) in BC using shallow whole genome sequencing (sWGS) in cfDNA from archived samples from National Cancer Institute of Colombia patients. We sequenced tumor tissues from 38 BC patients with different molecular subtypes using a gene panel of 176 genes significantly mutated in cancer, and by liquid biopsies using sWGS on 20 paired samples to detect SCNAs and compare with the tumor samples. We identified an extensive intertumoral heterogeneity between the molecular subtypes of BC, with a mean tumor load of 602 mutations in the gene panel of tumor tissues. There was a 12.3% of concordance in deletions in the cfDNA-tumor pairs considering only the genes covered by the panel encompassing seven genes: BRCA1, CDK12, NF1, MAP2K4, NCOR1, TP53, and KEAP1 in three patients. This study shows the feasibility to complement the genomic analysis of tumor tissue biopsies to detect SCNA in BC using sWGS in cfDNA, providing a wider identification of potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155bdcd081b005cbe1cf2720136c7337af923e73" target='_blank'>
              Mutational spectrum of breast cancer by shallow whole-genome sequencing of cfDNA and tumor gene panel analysis
              </a>
            </td>
          <td>
            Fernando Ambriz-Barrera, E. Rojas-Jiménez, Clara E. Díaz-Velásquez, Aldo Hugo De-La-Cruz-Montoya, Héctor Martínez-Gregorio, Miguel Ruiz-De-La-Cruz, Antonio Huertas, Ana Lorena Montealegre, Carlos Castro-Rojas, Gabriela Acosta, Felipe Vaca-Paniagua, Sandra Perdomo
          </td>
          <td>2024-09-12</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved tool for the removal of DNA double-strand breaks (DSBs) and the preservation of stalled and damaged DNA replication forks. Successful completion of HR requires the tumor suppressor BRCA2. Germline mutations in BRCA2 lead to familial breast, ovarian, and other cancers, underscoring the importance of this protein for maintaining genome stability. BRCA2 harbors two distinct DNA binding domains, one that possesses three oligonucleotide/oligosaccharide binding (OB) folds (known as the OB-DBD), and with the other residing in the C-terminal recombinase binding domain (termed the CTRB-DBD) encoded by the last gene exon. Here, we employ a combination of genetic, biochemical, and cellular approaches to delineate contributions of these two DNA binding domains toward HR and the maintenance of stressed DNA replication forks. We show that OB-DBD and CTRB-DBD confer ssDNA and dsDNA binding capabilities to BRCA2, respectively, and that BRCA2 variants mutated in either DNA binding domain are impaired in the ability to load the recombinase RAD51 onto ssDNA pre-occupied by RPA. While the CTRB-DBD mutant is modestly affected for HR, it exhibits a strong defect in the protection of stressed replication forks. In contrast, the OB-DBD is indispensable for both BRCA2 functions. Our study thus defines the unique contributions of the two BRCA2 DNA binding domains in genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b867a23dce384adad0b091d20596685ac211bb92" target='_blank'>
              Distinct roles of the two BRCA2 DNA binding domains in DNA damage repair and replication fork preservation
              </a>
            </td>
          <td>
            Francisco Neal, Wenjing Li, Mollie E. Uhrig, Neelam Sharma, Shahrez Syed, Sandeep Burma, R. Hromas, Alexander V. Mazin, Eloise Dray, David S. Libich, Shaun K. Olsen, Elizabeth V. Wasmuth, Weixing Zhao, Claus S. Sørensen, C. Wiese, Youngho Kwon, Patrick Sung
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: CD59, a GPI-anchored membrane protein, protects cancer cells from complement-dependent cytotoxicity (CDC) by inhibiting the formation of the membrane attack complex (MAC). It has been demonstrated to be overexpressed in most solid tumors, where it facilitates tumor cell escape from complement surveillance. The role of CD59 in cancer growth and interactions between CD59 and immune cells that modulate immune evasion has not been well explored. Methods: Using cancer patient database from The Cancer Genome Atlas (TCGA) and other public databases, we analyzed CD59 expression, its prognostic significance, and its association with immune cell infiltration in the tumor microenvironment, identifying associated genomic and functional networks and validating findings with invitro cell-line experimental data. Results: This article describes the abundant expression of CD59 in multiple tumors such as cervical squamous cell carcinoma (CESC), kidney renal cell carcinoma (KIRC), glioblastoma multiforme (GBM), head and neck squamous cell carcinoma (HNSC), and stomach adenocarcinoma (STAD), as well as in pan-cancer, using The Cancer Genome Atlas (TCGA) database and confirmed using multiple cancer cell lines. The expression of CD59 significantly alters the overall survival (OS) of patients with multiple malignancies such as CESC, GBM, HNSC, and STAD. Further, the correlation between CD59 and Treg and/or MDSC in the tumor microenvironment (TME) has shown to be strongly associated with poor outcomes in CESC, GBM, HNSC, and STAD as these tumors express high FOXP3 compared to KIRC. Moreover, unfavorable outcomes were strongly associated with the expression of CD59 and M2 tumor-associated macrophage infiltration in the TME via the IL10/pSTAT3 pathway in CESC and GBM but not in KIRC. In addition, TGFβ1-dominant cancers such as CESC, GBM, and HNSC showed a high correlation between CD59 and TGFβ1, leading to suppression of cytotoxic T cell activity. Conclusion: Overall, the correlation between CD59 and immune cells predicts its prognosis as unfavorable in CESC, GBM, HNSC, and STAD while being favorable in KIRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29288f3fe222d9ea1658cdfb0515406b1112eed" target='_blank'>
              Deciphering CD59: Unveiling Its Role in Immune Microenvironment and Prognostic Significance
              </a>
            </td>
          <td>
            Bhaumik Patel, Ashok Silwal, M. Eltokhy, S. Gaikwad, M. Curcic, Jalpa Patel, Sahdeo Prasad
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d92fbde3dae7404c9f30bd22e44407b07e5261a" target='_blank'>
              Self-sustaining long-term 3D epithelioid cultures reveal drivers of clonal expansion in esophageal epithelium
              </a>
            </td>
          <td>
            A. Herms, D. Fernandez-Antoran, M. Alcolea, Argyro Kalogeropoulou, Ujjwal Banerjee, G. Piedrafita, E. Abby, J. A. Valverde-Lopez, Inês S Ferreira, Irene Caseda, M. T. Bejar, S. Dentro, Sara Vidal-Notari, S. Ong, B. Colom, K. Murai, C. King, Krishnaa Mahbubani, K. Saeb‐Parsy, Alan R Lowe, M. Gerstung, P. Jones
          </td>
          <td>2024-09-23</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4788ed4991cb05a8733cec69abe88f6738bf0a69" target='_blank'>
              Differential chromatin accessibility and transcriptional dynamics define breast cancer subtypes and their lineages.
              </a>
            </td>
          <td>
            Michael D. Iglesia, R. Jayasinghe, Siqi Chen, Nadezhda V. Terekhanova, John M Herndon, E. Storrs, A. Karpova, Daniel Cui Zhou, Nataly Naser Al Deen, Andrew T Shinkle, Rita Jui-Hsien Lu, Wagma Caravan, Andrew Houston, Yanyan Zhao, Kazuhito Sato, Preet Lal, Cherease Street, Fernanda Martins Rodrigues, Austin N. Southard-Smith, A. L. N. Targino da Costa, Houxiang Zhu, Chia-Kuei Mo, Lisa Crowson, Robert S Fulton, Matthew A. Wyczalkowski, C. Fronick, L. Fulton, Hua Sun, Sherri R Davies, Elizabeth L. Appelbaum, S. Chasnoff, Madelyn Carmody, Candace Brooks, Ruiyang Liu, M. Wendl, C. Oh, Diane E Bender, C. Cruchaga, O. Harari, Andrea L Bredemeyer, K. Lavine, R. Bose, J. Margenthaler, Jason M. Held, Samuel Achilefu, F. Ademuyiwa, R. Aft, Cynthia X Ma, G. A. Colditz, Tao Ju, Stephen T. Oh, James A J Fitzpatrick, E. S. Hwang, K. Shoghi, M. Chheda, D. Veis, Feng Chen, Ryan C. Fields, William E Gillanders, Li Ding
          </td>
          <td>2024-10-30</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="DNA lies at the heart of the central dogma of life. Altering DNA can modify the flow of information in fundamental cellular processes such as transcription and translation. The ability to precisely manipulate DNA has led to remarkable advances in treating incurable human genetic ailments and has changed the landscape of biological research. Genome editors such as CRISPR-Cas nucleases and TALENs have become ubiquitous tools in basic and applied biological research and have been translated to the clinic to treat patients. The specificity and modularity of these genome editors have made it possible to efficiently engineer genomic DNA; however, underlying principles governing editing outcomes in eukaryotes are still being uncovered. Editing efficiency can vary from cell type to cell type for the same DNA target sequence, necessitating de novo design and validation efforts. Chromatin structure and epigenetic modifications have been shown to affect the activity of genome editors because of the role they play in hierarchical organization of the underlying DNA. Understanding the nuclear search mechanism of genome editors and their molecular interactions with higher order chromatin will lead to improved models for predicting precise genome editing outcomes. Insights from such studies will unlock the entire genome to be engineered for the creation of novel therapies to treat critical illnesses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de15652cba672686fe241fc8c1aaf364c6e3c7f6" target='_blank'>
              Impact of Chromatin Organization and Epigenetics on CRISPR-Cas and TALEN Genome Editing.
              </a>
            </td>
          <td>
            Surbhi Jain, Guanhua Xun, Huimin Zhao
          </td>
          <td>2024-09-24</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer is a complex disease characterized by genetic and molecular diversity, often involving dysregulation of critical cellular pathways. Recent advances in pan-cancer research have highlighted the importance of shared oncogenic mechanisms across different cancer types, providing new avenues for therapeutic exploration. Protein kinases, particularly those involved in phagocytosis, play pivotal roles in cellular homeostasis and immune response. This study systematically examines the genetic alterations and expression profiles of protein kinases associated with phagocytosis across various cancer types, using data from The Cancer Genome Atlas (TCGA) and other publicly available resources. We analyzed single nucleotide variations (SNVs), copy number variations (CNVs), methylation patterns, and mRNA expression to identify recurring alterations and their associations with survival outcomes. Our findings reveal that MET and MERTK are the most frequently mutated genes, with missense mutations dominating across cancers. CNV analysis shows significant correlations with survival in cancers like UCEC, KIRP, and KIRC, while methylation analysis indicates cancer-specific regulatory patterns affecting gene expression. Differential expression analysis highlights distinct cancer-type-specific expression profiles, with genes like MET and BTK displaying significant variation. Crosstalk pathway analysis further reveals the involvement of these kinases in key cancer-related pathways, such as epithelial-mesenchymal transition (EMT) and apoptosis. Drug sensitivity analysis identifies potential therapeutic targets, with gene expression correlating significantly with cancer cell line responsiveness to specific compounds. These findings underscore the importance of the phagocytotic kinome in cancer biology and suggest potential therapeutic strategies targeting protein kinases to enhance immune response and improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec7e3ff62485c6898d190779703562a070a2f0e" target='_blank'>
              Protein Kinases in Phagocytosis: Promising Genetic Biomarkers for Cancer
              </a>
            </td>
          <td>
            Sadhika Arumilli, Hengrui Liu
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf19432cc15369ce28dc15aa8918b2d7b3bd99a" target='_blank'>
              TP53: the unluckiest of genes?
              </a>
            </td>
          <td>
            A. Joerger, Thorsten Stiewe, Thierry Soussi
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1dca550adace3299de2700d468a5c183e07ebc1" target='_blank'>
              Widespread mutagenesis and chromosomal instability shape somatic genomes in systemic sclerosis
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Thomas Blouin, James McCollum, Latarsha Porcher, François Virard, Jiri Zavadil, Carol Feghali-Bostwick, Natalie Saini
          </td>
          <td>2024-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1844470abca3c1fa6252bba317101f9730f2c0f4" target='_blank'>
              5-Formylcytosine: a new epigenetic player
              </a>
            </td>
          <td>
            Dharmendra Kumar, Iqbal Hyder, Wilfried A Kues
          </td>
          <td>2024-10-23</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a29762d25b9a77c8fd5e2a5d7d19911938e8b1" target='_blank'>
              Integrate and conquer: pan-cancer proteogenomics uncovers cancer vulnerabilities and therapeutic opportunities
              </a>
            </td>
          <td>
            Debomita Chakraborty, Rossana Romero, K. Rajalingam
          </td>
          <td>2024-10-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008cfba4377fc6d50c595b7237c526e2cccfb1be" target='_blank'>
              Epigenetic reader ZMYND11 noncanonical function restricts HNRNPA1-mediated stress granule formation and oncogenic activity
              </a>
            </td>
          <td>
            Cheng Lian, Chunyi Zhang, Pan Tian, Qilong Tan, Yu Wei, Zixian Wang, Qin Zhang, Qixiang Zhang, Mengjie Zhong, Li-Quan Zhou, Xisong Ke, Huabing Zhang, Yao Zhu, Zhenfei Li, Jingdong Cheng, Gong-Hong Wei
          </td>
          <td>2024-09-28</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer cells are heterogeneous, each harboring distinct molecular aberrations and are dependent on different genes for their survival and proliferation. While successful targeted therapies have been developed based on driver DNA mutations, many patient tumors lack druggable mutations and have limited treatment options. Here, we hypothesize that new precision oncology targets may be identified through “expression-driven dependency”, whereby cancer cells with high expression of a targeted gene are more vulnerable to the knockout of that gene. We introduce a Bayesian approach, BEACON, to identify such targets by jointly analyzing global transcriptomic and proteomic profiles with genetic dependency data of cancer cell lines across 17 tissue lineages. BEACON identifies known druggable genes, e.g., BCL2, ERBB2, EGFR, ESR1, MYC, while revealing new targets confirmed by both mRNA- and protein-expression driven dependency. Notably, the identified genes show an overall 3.8-fold enrichment for approved drug targets and enrich for druggable oncology targets by 7 to 10-fold. We experimentally validate that the depletion of GRHL2, TP63, and PAX5 effectively reduce tumor cell growth and survival in their dependent cells. Overall, we present the catalog of express-driven dependency targets as a resource for identifying novel therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328c4c5a157ff3ba7e44391b07b799fdfa4b7395" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Medicine
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D. Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2b209080acf772da88d3de252f9c4248e66e84" target='_blank'>
              Hallmarks of a genomically distinct subclass of head and neck cancer
              </a>
            </td>
          <td>
            Tara Muijlwijk, I. Nauta, Anabel van der Lee, Kari J T Grünewald, A. Brink, Sonja H Ganzevles, Robert J. Baatenburg de Jong, L. Atanesyan, S. Savola, M. A. van de Wiel, Laura A N Peferoen, Elisabeth Bloemena, R. van de Ven, C. R. Leemans, J. Poell, R. Brakenhoff
          </td>
          <td>2024-10-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>3</td>
          <td>1</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The repair of DNA double-strand breaks (DSBs) through homologous recombination (HR) is vital for maintaining the stability and integrity of the genome. RNA binding proteins (RBPs) intricately regulate the DNA damage repair process, yet the precise molecular mechanisms underlying their function remain incompletely understood. In this study, we highlight the pivotal role of RPS15, a representative RBP, in homologous recombination repair. Specifically, we demonstrate that RPS15 promotes DNA end resection, a crucial step in homologous recombination. Notably, we identify an interaction between RPS15 and CtIP, a key factor in homologous recombination repair. This interaction is essential for CtIP recruitment to DSB sites, subsequent RPA coating, and RAD51 replacement, all critical steps in efficient homologous recombination repair and conferring resistance to genotoxic treatments. Functionally, suppressing RPS15 expression sensitizes cancer cells to X-ray radiation and enhances the therapeutic synergistic effect of PARP1 inhibitors in breast cancer cells. In summary, our findings reveal that RPS15 promotes DNA end resection to ensure effective homologous recombination repair, suggesting its potential as a therapeutic target in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2aedc66ec12ddcad5319b720ce461d7c9233af5" target='_blank'>
              RPS15 Coordinates with CtIP to Facilitate Homologous Recombination and Confer Therapeutic Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Baohang Lin, Guan Huang, Zishan Yuan, Xun Peng, Chunliang Yu, Jialu Zheng, Zequn Li, Juanyun Li, Jinan Liang, Bo Xu
          </td>
          <td>2024-10-03</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is an immensely complex ecosystem1,2. This complexity underlies difficulties in elucidating principles of spatial organization and using molecular profiling of the TME for clinical use3. Through statistical analysis of 96 spatial transcriptomic (ST-seq) datasets spanning twelve diverse tumor types, we found a conserved distribution of multicellular, transcriptionally covarying units termed ‘Spatial Groups’ (SGs). SGs were either dependent on a hierarchical local spatial context – enriched for cell-extrinsic processes such as immune regulation and signal transduction – or independent from local spatial context – enriched for cell-intrinsic processes such as protein and RNA metabolism, DNA repair, and cell cycle regulation. We used SGs to define a measure of gene spatial heterogeneity – ‘spatial lability’ – and categorized all 96 tumors by their TME spatial lability profiles. The resulting classification captured spatial variation in cell-extrinsic versus cell-intrinsic biology and motivated class-specific strategies for therapeutic intervention. Using this classification to characterize pre-treatment biopsy samples of 16 non-small cell lung cancer (NSCLC) patients outside our database distinguished responders and non-responders to immune checkpoint blockade while programmed death-ligand 1 (PD-L1) status and spatially unaware bulk transcriptional markers did not. Our findings show conserved principles of TME spatial biology that are both biologically and clinically significant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a400ff77ff3e3d5b4307dd66b458357a7b860383" target='_blank'>
              Conserved principles of spatial biology define tumor heterogeneity and response to immunotherapy
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, C. Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BCR–ABL, the pioneering gene fusion resulting from chromosomal translocation, has marked a major milestone in understanding genetic alterations in cancer. Initially, gene fusions were linked solely to chromosomal rearrangements, serving as diagnostic markers and cancer drivers. However, advancements in high-throughput sequencing and bioinformatics have revealed additional mechanisms underlying gene fusion. The term “gene fusion” primarily refers to fusion events at the DNA level, whereas “chimeric RNA” encompasses a wide range of transcripts containing exons from different parental genes, including gene fusion transcripts. Recent developments have identified numerous chimeric RNAs in various cancer types, extending even to non-cancerous tissues. Chimeric RNAs, originating from events such as trans-splicing, read-through, and intergenic splicing, form a complex landscape with varied functions. While some chimeric RNAs have defined roles and therapeutic potential, a comprehensive understanding of their diverse functions remains a priority. Exploring the full spectrum of chimeric RNA activities is crucial for revealing their clinical and therapeutic implications. In addition, chimeric RNAs are key players in tumorigenesis, affecting cellular processes, and driving cancer progression. Understanding their intricate interactions with cellular pathways is essential for developing targeted therapies and precision medicine approaches. The dynamic nature of chimeric RNAs highlights the need for ongoing research to fully harness their diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0ece1a1fed50713b0199fed280d597f09ffa316" target='_blank'>
              Beyond chromosomal rearrangements: The expanding landscape of gene fusions and chimeric RNAs
              </a>
            </td>
          <td>
            Sangeen Khan, Yue Tang, Zhenguo Song, Xi Chen, Lijun Wang, Chengjuan Zhang, Fujun Qin
          </td>
          <td>2024-10-28</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a36577362842a55eb6cd804fed42f385c47b74" target='_blank'>
              Tissue microarray analyses of the essential DNA repair factors ATM, DNA-PKcs and Ku80 in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            H. Zech, Clara Marie von Bargen, A. Oetting, N. Möckelmann, Christina Möller-Koop, M. Witt, N. Struve, Cordula Petersen, Christian Betz, K. Rothkamm, A. Münscher, T. Clauditz, T. Rieckmann
          </td>
          <td>2024-10-30</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Accurate monitoring of base excision repair (BER) activity in cancer cells is critical for advancing the comprehension of DNA repair processes, gaining insights into cancer development, and guiding treatment strategies. However, current assay techniques for assessing BER activity in cancer cells face challenges due to the heterogeneous origins and diversity of BER enzymes. In this work, we present a highly reliable triple loop-interlocked DNA codec (GATED) that enables precise assessment of BER activity in cancer cells through signal amplification mediated by multienzyme orthogonal activation. The GATED device features a dumbbell-shaped DNA probe to encode two BER enzymes for BER-related signal conversion as well as two bound circular DNA to decode the apurinic/apyrimidinic sites for apurinic/apyrimidinic endonuclease 1 (APE1)-mediated signal amplification. Importantly, GATED is orthogonally activated by multiple target BER enzymes (i.e., uracil DNA glycosylase, thymine DNA glycosylase, and APE1), resulting in a unified fluorescent signal that significantly improves the detection specificity and sensitivity to BER enzymes. Additionally, we demonstrate that the GATED has exceptional biostability within complex biological systems, where it was successfully employed to monitor BER activity in cancer cells with high specificity and enabled cell-based high-throughput screening for BER inhibitors. The GATED provides a much-needed tool for the real-time monitoring of BER activity and the screening of BER inhibitors in cancer cells, potentially advancing both the investigation and clinical application of BER biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4455a54c11e6664a89c2b008ca24acc5e5bb02" target='_blank'>
              Multienzymatic Orthogonal Activation of DNA Codec Enables Tumor-Specific Imaging of Base Excision Repair Activity.
              </a>
            </td>
          <td>
            Wei-Jia Zeng, Zhaopeng Chen, Yan-Mei Lei, W. Liang, Yaqin Chai, Ruo Yuan, Ying Zhuo
          </td>
          <td>2024-09-26</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="HAGLR Opposite Strand lncRNA (HAGLROS) is a long non-coding RNA (lncRNA) located on the long arm of human chromosome 2 at locus 2q31.1. Emerging evidence highlights HAGLROS as a pivotal player in human cancers, characterized by its significant upregulation across multiple malignancies where it functions as an oncogenic driver. Its aberrant expression is closely linked to the initiation and progression of 13 distinct cancer types, notably correlating with adverse clinical outcomes and reduced overall survival rates in 9 of these cancer types. Mechanistically, HAGLROS is under the regulatory influence of the transcription factor STAT3, exerts competitive binding to 9 miRNAs, activates 5 signaling pathways pivotal for cancer cell proliferation and metastasis, as well as intricately modulates gene expression profiles. Given its multifaceted roles, HAGLROS emerges as a promising candidate for cancer diagnostics and prognostics. Moreover, its potential as a therapeutic target holds considerable promise for novel treatment strategies in oncology. This review synthesizes current research on HAGLROS, covering its expression patterns, biological roles, and clinical significance in cancer. By shedding light on these aspects, this review aims to contribute new perspectives that advance our understanding of cancer biology, enhance diagnostic accuracy, refine prognostic assessments, and pave the way for targeted therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb39b9ed8d4ba49047e0b81dbd39b3093253cd68" target='_blank'>
              LncRNA HAGLROS: A Vital Oncogenic Propellant in Various Human Cancers.
              </a>
            </td>
          <td>
            Jingjie Yang, Haodong He, Lihan Chen, Yuzhang Wei, Yulong Liu, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2024-10-31</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A. R. Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The interplay between cancer and the immune system has captivated researchers for a long time. Recent developments in cancer immunotherapy have substantiated this interest with a significant benefit to cancer patients. Tumor and immune cells are regulated via a wide range of molecular mechanisms involving intricate transcriptional and epigenetic networks. Epigenetic processes influence chromatin structure and accessibility, thus governing gene expression, replication, and DNA damage repair. However, aberrations within epigenetic signatures are frequently observed in cancer. One of the key epigenetic marks is the trimethylation of histone 3 at lysine 9 (H3K9me3), confined mainly within constitutive heterochromatin to suppress DNA accessibility. It is deposited at repetitive elements, centromeric and telomeric loci, as well as at the promoters of various genes. Dysregulated H3K9me3 deposition disrupts multiple pathways, including immune signaling. Consequently, altered H3K9me3 dynamics may modify the efficacy of immunotherapy. Indeed, growing evidence highlights the pivotal roles of various proteins mediating H3K9me3 deposition (SETDB1/2, SUV39H1/2), erasure (KDM3, KDM4 families, KDM7B, LSD1) and interpretation (HP1 proteins, KAP1, CHD4, CDYL, UHRF1) in modulating immunotherapy effectiveness. Here, we review the existing literature to synthesize the available information on the influence of these H3K9me3 writers, erasers, and readers on the response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/145e5ffc384f02a28629c78f8d74cf791460fc6e" target='_blank'>
              The Roles of H3K9me3 Writers, Readers, and Erasers in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Urszula Oleksiewicz, Monika Kuciak, Anna Jaworska, Dominika Adamczak, Anna Bisok, Julia Mierzejewska, Justyna Sadowska, Patrycja Czerwińska, Andrzej Mackiewicz
          </td>
          <td>2024-10-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The six subunit Origin Recognition Complex (ORC) loads excess MCM2-7 on chromosomes to promote initiation of DNA replication and is believed to be important for origin specification. Mapping of origins in cancer cell lines engineered to delete three of the subunits, ORC1, ORC2 or ORC5 shows that specific origins are still used and are mostly at the same sites in the genome as in wild type cells. The few hundred origins that were up-regulated in the absence of ORC suggest that GC/TA skewness and simple repeat sequences facilitate, but are not essential for, origin selection in the absence of the six-subunit ORC. Despite the lack of ORC, excess MCM2-7 is still loaded at comparable rates in G1 phase to license reserve origins and is also repeatedly loaded in the same S phase to permit re-replication. Thus, origin specification and excess MCM2-7 loading on origins do not require the six-subunit ORC in human cancer cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20fbdd40cdfd2421e119fa832f03205a7825921a" target='_blank'>
              Specific origin selection and excess functional MCM2-7 loading in ORC-deficient cells
              </a>
            </td>
          <td>
            Y. Shibata, Mihaela Peycheva, Etsuko Shibata, Daniel Malzl, Rushad Pavri, A. Dutta
          </td>
          <td>2024-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="In eukaryotes, the cellular processes contributing to gamete formation form the building blocks of genetic inheritance across generations. While traditionally viewed as conserved across model organisms, emerging studies reveal significant variation among eukaryotes in meiotic and post-meiotic processes. Extending our knowledge to non-model organisms is therefore critical to improve our understanding of the evolutionary origin, significance and subsequent modifications associated with gamete formation. We describe the mechanisms underlying recombination, chromosome segregation and meiotic sex chromosome inactivation in the stick insect group Timema. Our results reveal that in males, 1) recombination initiates before synapsis, 2) the mechanisms of X inactivation are variable among species and the X remains silenced despite two waves of transcriptional activation in autosomes during spermatogenesis, 3) chromosome segregation is preceded by the recruitment of centromere proteins that either bind locally or longitudinally along the chromosomes. Together, our observations help understand the evolutionary significance of key cellular processes related to spermatogenesis and shed light on the diversity of their underlying mechanisms among species, including Timema stick insects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8577ddd5aaf9fc201ec5378fe865f42882b1cd" target='_blank'>
              Dynamics of recombination, X inactivation and centromere proteins during stick insect spermatogenesis
              </a>
            </td>
          <td>
            Zoé Dumas, William Toubiana, Marie Delattre, Tanja Schwander
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CRISPR/Cas9 based genome editing has advanced our understanding of a myriad of important biological phenomena. Important challenges to multiplex genome editing in maize include assembly of large complex DNA constructs, few genotypes with efficient transformation systems, and costly/labor-intensive genotyping methods. Here we present a ‘fast-edit’ approach for multiplex CRISPR/Cas9 genome editing system that delivers a single compact DNA construct via biolistics to type I embryogenic calli, followed by a novel efficient genotyping assay to identify desirable editing outcomes. We first demonstrate the creation of heritable mutations at multiple target sites within the same gene. Next, we successfully created individual and stacked mutations for multiple members of a gene family. Genome sequencing found off-target mutations are rare. Multiplex genome editing was achieved for both the highly transformable inbred line H99 and Illinois Low Protein1 (ILP1), a genotype where transformation has not previously been reported. In addition to screening transformation events for deletion alleles by PCR, we also designed PCR assays that selectively amplify deletion or insertion of a single nucleotide, the most common outcome from DNA repair of CRISPR/Cas9 breaks by non-homologous end-joining. The Indel- Selective PCR (IS-PCR) method enabled rapid tracking of multiple edited alleles in progeny populations. The ‘end to end’ pipeline presented here for multiplexed CRISPR/Cas9 mutagenesis can be applied to accelerate maize functional genomics in a broader diversity of genetic backgrounds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5a9d008a68fa3882779d3e691b6cae9cf9fc7" target='_blank'>
              Efficient mutagenesis of maize inbreds using biolistics, multiplex CRISPR/Cas9 editing, and Indel-Selective PCR
              </a>
            </td>
          <td>
            Maruti Nandan Rai, Brian Rhodes, Stephen Jinga, Praveena Kanchupati, Edward Ross, Shawn R. Carlson, Stephen P. Moose
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a10c01fddb459444a48acb4656dc4b4986c01eac" target='_blank'>
              SASS6 promotes tumor proliferation and is associated with TP53 and immune infiltration in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zihao Li, Lingyun He, Jia-Yi Li, Jing Qian, Zuotao Wu, Yongjie Zhu, Ting-ting Zhuo, Ju-Sen Nong, Honghua Liang, Hua Zheng, Weijia Huang, Julu Huang, Jianbin Cao
          </td>
          <td>2024-10-24</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction The mutated in colorectal cancer (MCC) gene was initially identified as a candidate tumor suppressor gene in colorectal cancer, acting as a negative regulator of cell cycle progression. However, its functional roles in brain tumors, particularly glioblastoma, remain largely unexplored. This study reveals a significant association between MCC status and glioblastoma. Methods We explored MCC expression in the glioblastoma database, patient samples, and cell lines. We investigated the proliferation and migration of the cell lines in MCC gene knockdown using small interfering RNA. Results In vitro analyses revealed elevated protein and mRNA levels of MCC in several glioblastoma cell lines (U118MG and T98G). Silencing MCC expression via siRNA-mediated knockdown resulted in increased proliferation and migration of these cell lines. Supporting these findings, analyses of The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and Genotype-Tissue Expression (GTEx) databases confirmed higher MCC expression in glioblastoma tumors than in normal brain tissue. Importantly, we observed that high MCC expression was associated with poor prognosis in glioblastoma patients, highlighting its potential role in disease progression. Additionally, this study identifies a nuclear localization of MCC in the glioblastoma cell line. Discussion These findings indicate that MCC expression is significantly upregulated in glioblastoma and may play a role in its pathophysiology, warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1756f336eb718e69a73fb3b12fe17a3f5407527e" target='_blank'>
              The mutated in colorectal cancer (MCC) gene can serve as a potential biomarker of glioblastoma
              </a>
            </td>
          <td>
            Huonggiang Nguyen, Qingzhi Huang, Uijin Juang, Suhwan Gwon, Woohyeong Jung, Soohyeon Lee, Beomwoo Lee, So-Hee Kwon, In Soo Kim, Jongsun Park, Seon-Hwan Kim
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background/Objectives: Cervical cancer (CC) remains a significant global health challenge, characterized by genetic heterogeneity and a complex molecular landscape, both of which contribute to its pathogenesis. This study aimed to investigate germline variants in proto-oncogenes and tumor suppressor genes in cervical cancer patients, with the objective of clarifying their potential role in disease development. Methods: We utilized a custom next-generation sequencing (NGS) panel targeting 48 genes implicated in oncogenesis. Germline DNA samples from cervical cancer patients were analyzed in order to identify nucleotide sequence alterations. Variants were classified according to pathogenicity and clinical relevance, based on established guidelines. Results: A total of 148 nucleotide variants were detected within the cohort. Of these, 35 variants (23.6%) were classified as benign. In contrast, 105 variants (70.9%) were identified as variants of uncertain significance (VUSs). Moreover, seven pathogenic or likely pathogenic mutations were discovered, along with the polymorphic variant rs1042522 in the TP53 gene, which has been associated with an increased risk of cervical cancer. Conclusions: Our findings contribute to expanding our understanding of the molecular genetic landscape of cervical cancer. They emphasize the potential contribution of rare germline mutations to its development and progression. These results highlight the importance of comprehensive genetic screening in order to improve diagnostic and therapeutic approaches for cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9744546f38ec2bc7fa36858f93c880cbede4f99" target='_blank'>
              Germline Variants in Proto-Oncogenes and Tumor Suppressor Genes in Women with Cervical Cancer
              </a>
            </td>
          <td>
            K. Lenkova, R. Khusainova, R. Minyazeva, A. Zaripova, I. Gilyazova, Natalia Mokrysheva, I. Minniakhmetov
          </td>
          <td>2024-10-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596a54233fb220ea64342c38ad0e59677160ecab" target='_blank'>
              Germline variant affecting p53β isoforms predisposes to familial cancer
              </a>
            </td>
          <td>
            Stephanie A Schubert, D. Ruano, S. Joruiz, Jordy Stroosma, Nikolina Glavak, A. Montali, Lia M Pinto, Mar Rodríguez-Girondo, D. Barge-Schaapveld, Maartje Nielsen, Bernadette P M van Nesselrooij, Arjen R Mensenkamp, M. V. van Leerdam, Thomas H. Sharp, H. Morreau, Jean-Christophe Bourdon, N. D. de Miranda, T. van Wezel
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="High expression of the long non-coding RNA (lncRNA) FAM30A has been previously associated with leukemic stem cell (LSC) activity and poor prognosis in both adult and paediatric acute myeloid leukaemia (AML) patients, yet it has not been functionally studied. This study provides the first cellular characterization of FAM30A focussing on an internal tandemly organised region, referred to as FAM30A repeats. FAM30A levels correlated with canonical AML LSC signatures and FAM30A depletion decreased cell viability as well as increased sensitivity to chemotherapeutics. It also inhibited colony formation, promoted granulocytic differentiation and abrogated leukemic engraftment in murine bone marrow in vivo. Overexpression of FAM30A repeats in this setting enhanced stemness, proliferation, chemoresistance, and engraftment thus highlighting the biological relevance of this region for LSC biology. On the molecular level, FAM30A repeats interact with the pro-LSC regulator Musashi-2 (MSI2), positively influencing expression of its targets including RUNX1 isoforms. We herein uncover that this FAM30A-MSI2-RUNX1 regulatory loop is of potential relevance for LSC maintenance in AML. These findings provide valuable insights into FAM30A’s cellular role and highlight its targeting potential for eliminating LSCs and improving treatment outcomes in AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e944155596f6ba606bda7e6372bd8e29a902812" target='_blank'>
              The long non-coding RNA FAM30A regulates the Musashi2-RUNX1 axis and is required for LSC function in AML cells
              </a>
            </td>
          <td>
            Jaime Calvo Sánchez, Mary T Scott, Athina Varnava, Tala Alakhras, Alice Wedler, Danny Misiak, N. Bley, C. Ihling, A. Sinz, Karen Keeshan, S. Hüttelmaier, David Vetrie, Marcel Köhn
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Extrachromosomal circular DNA (eccDNA) has emerged as a central focus in molecular biology, with various types being found across species through advanced techniques, including high-throughput sequencing. This dynamic molecule exerts significant influence on aging and immune function and plays pivotal roles in autoimmune diseases, type 2 diabetes mellitus, cancer, and genetic disorders.


SUMMARY
This comprehensive review investigates the classification, characteristics, formation processes, and multifaceted functions of eccDNA, providing an in-depth exploration of its mechanisms in diverse diseases.


KEY MESSAGES
The goal of this review is to establish a robust theoretical foundation for a more comprehensive understanding of eccDNA, offering valuable insights for the development of clinical diagnostics and innovative therapeutic strategies in the context of related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f915db54a9ed6fe76aa5229ced4103ebd9df09e" target='_blank'>
              Exploring the Role of Extrachromosomal Circular DNA in Human Diseases.
              </a>
            </td>
          <td>
            Yali Peng, Huihui Tao, Guoying Wang, Mengyao Wu, Tinatin Xu, Chunmei Wen, Xuejia Zheng, Yong Dai
          </td>
          <td>2024-09-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Previous study indicated that CHK1 was important for repairing DNA damage and cell cycle regulation. Aims: To investigate the association of Checkpoint kinase 1 (CHK1) expression with clinicopathological features, prognosis, and immune infiltration in cancer. Methods: Several databases were searched for relevant publications to conduct a meta-analysis to reveal the association between CHK1 and clinicopathological features of cancer. TIMER and GEPIA datasets were utilized to explore the differential expression of CHK1 of tumors. GEPIA and Kaplan-Meier Plotter databases were adopted to detect the prognostic significance of CHK1 in tumor. TIMER and cBioPortal databases were used for the analysis regarding immune infiltration and mutation respectively. Results: We found that CHK1 expression was significantly associated with low differentiation (OR=3.94, 95% CI: 2.73-5.67, P<0.05), advanced stage (OR=3.20, 95% CI: 2.30-4.44, P<0.05), vascular infiltration (OR=3.24, 95% CI: 2.18-4.82, P<0.05) and lymph node metastasis (OR=3.55, 95% CI: 2.62-4.82, P<0.05) of various cancers. CHK1 was highly expressed in multiple cancers and was related to clinical stage, survival, immune infiltration in pan-cancers. Conclusions: Our study found that CHK1 was significantly related to prognosis and immunological status in various cancers, suggesting that CHK1 may serve as a useful biomarker for prognosis and immune infiltration in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48bd08dda4568b1d8c35e755803a8c000bea8fc" target='_blank'>
              Exploration of the carcinogenetic and immune role of CHK1 in human cancer
              </a>
            </td>
          <td>
            Jian Zhou, Ziyi Wu, Dilihumaer Aili, Lu Wang, Tanghai Liu
          </td>
          <td>2024-09-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromatin organization is important for gene transcription in pig genome. However, its three-dimensional (3D) structure and dynamics are much less investigated than those in human. Here we applied the long-reads chromatin interaction analysis by paired-end tag sequencing (ChIA-PET) method to map the whole-genome chromatin interactions mediated by CCCTC-binding factor (CTCF) and RNA polymerase Ⅱ (RNAPⅡ or POLⅡ) in porcine macrophage cells before and after polyinosinic-polycytidylic acid [Poly(I:C)] induction. Our results revealed that Poly(I:C) induction impacts the 3D genome organization in the 3D4/21 cells at the fine-scale chromatin loop level rather than at the large-scale domain level. Furthermore, our findings underscored the pivotal role of CTCF anchored chromatin interactions in reshaping chromatin architecture during immune responses. Knock-out of the CTCF locus further confirmed that the CTCF anchored enhancers are associated with the activation of immune genes via long-range interactions. Notably, ChIA-PET data also supported the spatial relationship between single nucleotide polymorphisms (SNPs) and the related gene transcription in 3D genome aspect. Our findings in this study provide new clues and potential targets to explore key elements related to diseases in swine and are also likely to shed light on elucidating chromatin organization and dynamics underlying the process of mammalian infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97442a763cf352589738944e6bb5a4d860334082" target='_blank'>
              Virus Infection Induces Immune Gene Activation with CTCF Anchored Enhancers and Chromatin Interactions in Pig Genome.
              </a>
            </td>
          <td>
            Jianhua Cao, Ruimin Ren, Xiaolong Li, Xiaoqian Zhang, Yan Sun, Xiaohuan Tian, Ru Liu, Xiangdong Liu, Yijun Ruan, Guoliang Li, Shuhong Zhao
          </td>
          <td>2024-09-23</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="SINE-VNTR-Alu (SVA) retrotransposons can regulate expression quantitative trait loci (eQTL) of coding and noncoding genes including transposable elements (TEs) distributed throughout the human genome. Previously, we reported that expressed SVAs and human leucocyte antigen (HLA) class II genotypes on chromosome 6 were associated significantly with Parkinson’s disease (PD). Here, our aim was to follow-up our previous study and evaluate the SVA associations and their regulatory effects on the transcription of TEs within the HLA class II genomic region. We reanalyzed the transcriptome data of peripheral blood cells from the Parkinson’s Progression Markers Initiative (PPMI) for 1530 subjects for TE and gene RNAs with publicly available computing packages. Four structurally polymorphic SVAs regulate the transcription of 20 distinct clusters of 235 TE loci represented by LINES (37%), SINES (28%), LTR/ERVs (23%), and ancient transposon DNA elements (12%) that are located in close proximity to HLA genes. The transcribed TEs were mostly short length, with an average size of 389 nucleotides. The numbers, types and profiles of positive and negative regulation of TE transcription varied markedly between the four regulatory SVAs. The expressed SVA and TE RNAs in blood cells appear to be enhancer-like elements that are coordinated differentially in the regulation of HLA class II genes. Future work on the mechanisms underlying their regulation and potential impact is essential for elucidating their roles in normal cellular processes and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0524bbc14acbc3e3a48fabf2b87faefb0544899" target='_blank'>
              SVA Regulation of Transposable Element Clustered Transcription within the Major Histocompatibility Complex Genomic Class II Region of the Parkinson’s Progression Markers Initiative
              </a>
            </td>
          <td>
            Jerzy K. Kulski, A. Pfaff, S. Kõks
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Intratumoral heterogeneity is intrinsically comprised of molecular alterations of tumor cells and extrinsically from interconnections with microenvironments. This study explores the spatial heterogeneity of ovarian clear cell carcinoma (OCCC), a rare cancer with significance to East Asian women. We profiled 21 tumors from matched ovarian and metastatic sites with spatial transcriptomic (ST) platforms including GeoMx and Visium CytAssist. Three subclusters of OCCC tumor cells enriched in oxidative phosphorylation, epithelial-mesenchymal transition, and inflammation pathways were identified with preferential geospatial localizations in tumor centers and invasive margins/tumor-stromal interfaces. These ST subclusters correlated with epithelial-mesenchymal gradients and prognostic features in larger cohorts. Single-cell ST in CosMx identified 9 cancer cell populations with organ-specific tropism. Trajectory analysis indicated a NEAT1-high subclone having founder characteristics, progressively dominating the metastatic niche, displaying mesenchymal traits. This founder feature correlated with clinical outcomes in PanCancer datasets. Our findings highlight the adaptive and context-specific interactions between cancer cells and microenvironments within spatial complexities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8349570f5b5a5cc83b0effd91ea241eef3a2ce6c" target='_blank'>
              Spatial Transcriptomic Profiling Of Advanced Ovarian Clear Cell Carcinoma Reveals Intra-Tumor Heterogeneity In Epithelial-Mesenchymal Gradient
              </a>
            </td>
          <td>
            Thang Truong Le, Duncan Yi-Te Wang, Ko-Chen Chen, Sebastian Rui-Gu Yan, Pei-Yu Chu, Tuan Zea Tan, Yi-Chia Chiu, Jia-Yuh Sheu, Chen-Hao Huang, Jieru Ye, Ya-Ting Tai, Hsueh-Fen Juan, Feng-Chiao Tsai, Wei-Chou Lin, Lin-Hung Wei, Ruby Yun-Ju Huang
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background R3HDM1, an RNA binding protein with one R3H domain, remains uncharacterized in terms of its association with tumor progression, malignant cell regulation, and the tumor immune microenvironment. This paper aims to fill this gap by analyzing the potential of R3HDM1 in diagnosis, prognosis, chemotherapy, and immune function across various cancers. Methods Data was collected from the Firehost database (http://gdac.broadinstitute.org) to obtain the TCGA pan-cancer queue containing tumor and normal samples. Additional data on miRNA, TCPA, mutations, and clinical information were gathered from the UCSC Xena database (https://xenabrowser.net/datapages/). The mutation frequency and locus of R3HDM1 in the TCGA database were examined using the cBioPortal. External validation through GEO data was conducted to assess the differential expression of R3HDM1 in different cancers. Protein expression levels were evaluated using the Clinical Proteomics Tumor Analysis Alliance (CPTAC). The differential expression of R3HDM1 was verified in lung adenocarcinoma cell lines and normal lung glandular epithelial cells via RT-qPCR. Cell migration and proliferation experiments were conducted by knocking down the expression of R3HDM1 in two lung adenocarcinoma cell lines using small interfering RNA. The biological role of R3HDM1 in pan-cancer was explored using the GSEA method. Multiple immune infiltration algorithms from the TIMER2.0 database was employed to investigate the correlation between R3HDM1 expression and the tumor immune microenvironment. Validation of transcriptome immune infiltration was based on 140 single-cell datasets from the TISCH database. The study also characterized a pan-cancer survival profile and analyzed the differential expression of R3HDM1 in different molecular subtypes. The relationship between R3HDM1 and drug resistance was investigated using four chemotherapy data sources: CellMiner, GDSC, CTRP and PRISM. The impact of chemicals on the expression of R3HDM1 was explored through the CTD database. Result The study revealed differential expression of R3HDM1 in various tumors, indicating its potential as an early diagnostic marker. Changes in somatic copy number (SCNA) and DNA methylation were identified as factors contributing to abnormal expression levels. Additionally, the study found that R3HDM1 expression is associated with clinical features, metabolic pathways, and important pathways related to metastasis and the immune system. High expression of R3HDM1 was linked to poor prognosis across different tumors and altered drug sensitivity. Furthermore, the expression of R3HDM1 showed significant correlations with immune modulatory molecules and biomarkers of lymphocyte subpopulation infiltration. Finally, the study highlighted four chemicals that could influence the expression of R3HDM1. Conclusion Overall, this study proposes that R3HDM1 expression is a promising biomarker for predicting the prognosis of cancer, especially lung adenocarcinoma, and the efficacy of immunotherapy, demonstrating the rationale for further exploration in the development of anti-tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379d6a67141459985f554b1a56ebc613e9c22b34" target='_blank'>
              Comprehensive pan-cancer analysis and experiments revealed R3HDM1 as a novel predictive biomarker for prognosis and immune therapy response
              </a>
            </td>
          <td>
            Jiawei Liu, Zhitong Bing, Junling Wang
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract DNA damage response (DDR) pathway is the coordinated cellular network dealing with the identification, signaling, and repair of DNA damage. It tightly regulates cell cycle progression and promotes DNA repair to minimize DNA damage to daughter cells. Key proteins involved in DDR are frequently mutated/inactivated in human cancers and promote genomic instability, a recognized hallmark of cancer. Besides being an intrinsic property of tumors, DDR also represents a unique therapeutic opportunity. Indeed, inhibition of DDR is expected to delay repair, causing persistent unrepaired breaks, to interfere with cell cycle progression, and to sensitize cancer cells to several DNA‐damaging agents, such as radiotherapy and chemotherapy. In addition, DDR defects in cancer cells have been shown to render these cells more dependent on the remaining pathways, which could be targeted very specifically (synthetic lethal approach). Research over the past two decades has led to the synthesis and testing of hundreds of small inhibitors against key DDR proteins, some of which have shown antitumor activity in human cancers. In parallel, the search for synthetic lethality interaction is broadening the use of DDR inhibitors. In this review, we discuss the state‐of‐art of ataxia‐telangiectasia mutated, ataxia‐telangiectasia‐and‐Rad3‐related protein, checkpoint kinase 1, Wee1 and Polθ inhibitors, highlighting the results obtained in the ongoing clinical trials both in monotherapy and in combination with chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3effb948dd60b6e28e5a1e98f095e523b94a901" target='_blank'>
              Targeting the DNA damage response in cancer
              </a>
            </td>
          <td>
            Guffanti Federica, Chiappa Michela, Damia Giovanna
          </td>
          <td>2024-10-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A critical area of recent cancer research is the emergence of transition states between normal and cancer that exhibit increased cell plasticity which underlies tumor cell heterogeneity. Pancreatic ductal adenocarcinoma (PDAC) can arise from the combination of a transition state termed acinar-to-ductal metaplasia (ADM) and a gain-of-function mutation in the proto-oncogene KRAS. During ADM, digestive enzyme-producing acinar cells acquire a transient ductal epithelium-like phenotype while maintaining their geographical acinar organization. One route of ADM initiation is the overexpression of the Krüppel-like factor 4 gene (KLF4) in the absence of oncogenic driver mutations. Here, we asked to what extent cells acquire and retain an epigenetic memory of the ADM transition state in the absence of oncogene mutation. We identified differential DNA methylation at Kras-downstream PI3K and Rho/Rac/Cdc42 GTPase pathway genes during ADM, as well as a corresponding gene expression increase in these pathways. Importantly, differential methylation persisted after gene expression returned to normal. Caerulein exposure, which induces widespread digestive system changes in addition to ADM, showed similar changes in DNA methylation in ADM cells. Regions of differential methylation were enriched for motifs of KLF and AP-1 family transcription factors, as were those of human pancreatic intraepithelial neoplasia (PanIN) samples, demonstrating the relevance of this epigenetic transition state memory in human carcinogenesis. Finally, single-cell spatial transcriptomics revealed that these ADM transition cells were enriched for PI3K pathway and AP1 family members, linking epigenetic memory to cancer cell plasticity even in the absence of oncogene mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da20ab316bd30b6e134263f4015da1c852acb904" target='_blank'>
              DNA methylation memory of pancreatic acinar-ductal metaplasia transition state altering Kras-downstream PI3K and Rho GTPase signaling in the absence of Kras mutation
              </a>
            </td>
          <td>
            E. K. Lo, Adrian Idrizi, Rakel Tryggvadottir, Weiqiang Zhou, Wenpin Hou, Hongkai Ji, Patrick Cahan, Andrew P. Feinberg
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb94655719ab703b20f1ab3bcef7376bf12c183f" target='_blank'>
              Genome-wide methylation profiling reveals extracellular vesicle DNA as an ex vivo surrogate of cancer cell-derived DNA
              </a>
            </td>
          <td>
            Kyung-A Kim, Sunmin Kim, I. Wortzel, Suho Lee, Yoon Dae Han, Tae-Min Kim, Han Sang Kim
          </td>
          <td>2024-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a lethal disease with limited therapies. In our screening using patient-derived organoids, we identified cyclin-dependent kinase 12 (CDK12) as a promising target in a subset of PDAC. Both pharmacological and genetic inhibition of CDK12 exerted potent antitumor activity in vitro and in vivo, inducing double-strand DNA breaks and apoptosis. CDK12 is a member of cyclin-dependent kinases that is involved in gene expression regulations at multiple levels. Recently, CDK12 has gained attention as a therapeutic target in several types of cancers including triple-negative breast cancer and Ewing sarcoma. However, little is known about the impact and possibility of CDK12 inhibition in pancreatic cancer treatment. As the lack of specific inhibitors that selectively target CDK12, studying CDK12 has long been challenging. To overcome this, we used CRISPR-Cas9 mediated genome editing to generate FKBP12F36V (dTAG) knock-in PDAC cell lines, enabling rapid elimination of endogenous CDK12 using a degrader against dTAG (dTAG-13). Quantitative-image-based cytometry and time lapse imaging showed that CDK12 ablation induces mitotic arrest, DNA damage and apoptosis (mitotic catastrophe). Specifically, loss of CDK12 kinase activity in the S phase was sufficient to slow DNA replication speed, leading to spindle assembly checkpoint activation and mitotic catastrophe. Importantly, cells that escaped mitotic death accumulate micronuclei, triggering immune response signaling that is mediated by the cGAS-STING-type I interferon pathway. Our results suggest that CDK12 is a promising target in a subset of PDAC patients, where CDK12 inhibition exerts antitumor activity both in cell autonomous and non-cell autonomous ways through inducing mitotic catastrophe as well as triggering immune response.
 Citation Format: Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro. CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A084.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632d48b78e0c9131037bdfa10d93bd71182ca006" target='_blank'>
              Abstract A084: CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer
              </a>
            </td>
          <td>
            Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Long interspersed elements type 1 (LINE-1s or L1s) are highly abundant non-LTR retrotransposons in mammalian genomes. Apart from an evolutionary role in altering the genomic landscape and gene expression, recent discoveries have revealed the scale and impact of active movement by L1, both in the germline and in somatic niches (e.g., the brain and many cancers). As transcriptional activation is a prerequisite for L1 retrotransposition, it is important to understand how L1 transcription is regulated. L1 transcription is controlled by an internal promoter contained in the 5’ untranslated region (5’UTR) of a full-length element. It has been shown that transcription start sites (TSSs) of human L1 are clustered at the beginning of its 5’UTR and the precision of transcription initiation is controlled by transcription factor YY1. The laboratory mouse is an essential model for investigating L1 retrotransposition and the associated consequences in vivo. However, mouse L1 5’UTRs are distinctly different from human L1’s: the former are organized into tandem repeats called monomers and separated from the open reading frames by a nonmonomeric sequence called tether. Unlike the human L1 promoter, transcription initiation and its regulation in mouse L1 subfamilies remain unknown. In this study, we streamlined the cloning of different types of mouse L1 promoters. TSSs were determined using RNA ligation-mediated 5’ rapid amplification of cDNA ends (RLM-5’RACE) incorporating a highly efficient reverse transcriptase enzyme followed by Nanopore sequencing. In the context of two monomer promoter reporter construct, transcription could initiate from either monomer or the tether sequence. While mouse L1 TSSs are distributive, a higher number of transcripts were initiated from the inner monomer. Surprisingly, a significant fraction of transcripts was also initiated from the tether sequence. In addition, we showed that YY1 transcriptionally activates mouse L1 5’UTR and is responsible for the synergy between the monomers. As transcription initiation is a major determinant for the proportion of active L1 loci in the next round of retrotransposition, these findings have important implications on L1 biology in the mouse model.
 Citation Format: Karabi Saha, Wenfeng An. Transcription initiation and its regulation in mouse L1s [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a796751004ae7aa371fe2f625b26733c662788" target='_blank'>
              Abstract A013: Transcription initiation and its regulation in mouse L1s
              </a>
            </td>
          <td>
            Karabi Saha, Wenfeng An
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As we age, many tissues become colonised by microscopic clones carrying somatic driver mutations. Some of these clones represent a first step towards cancer whereas others may contribute to ageing and other diseases. However, our understanding of the clonal landscapes of human tissues, and their impact on cancer risk, ageing and disease, remains limited due to the challenge of detecting somatic mutations present in small numbers of cells. Here, we introduce a new version of nanorate sequencing (NanoSeq), a duplex sequencing method with error rates <5 errors per billion base pairs, which is compatible with whole-exome and targeted gene sequencing. Deep sequencing of polyclonal samples with single-molecule sensitivity enables the simultaneous detection of mutations in large numbers of clones, yielding accurate somatic mutation rates, mutational signatures and driver mutation frequencies in any tissue. Applying targeted NanoSeq to 1,042 non-invasive samples of oral epithelium and 371 samples of blood from a twin cohort, we found an unprecedentedly rich landscape of selection, with 49 genes under positive selection driving clonal expansions in the oral epithelium, over 62,000 driver mutations, and evidence of negative selection in some genes. The high number of positively selected mutations in multiple genes provides high-resolution maps of selection across coding and non-coding sites, a form of in vivo saturation mutagenesis. Multivariate regression models enable mutational epidemiology studies on how carcinogenic exposures and cancer risk factors, such as age, tobacco or alcohol, alter the acquisition and selection of somatic mutations. Accurate single-molecule sequencing has the potential to unveil the polyclonal landscape of any tissue, providing a powerful tool to study early carcinogenesis, cancer prevention and the role of somatic mutations in ageing and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb1ed2352282f2639f8bc59358f511e72d35ab26" target='_blank'>
              Somatic mutation and selection at epidemiological scale
              </a>
            </td>
          <td>
            A. R. Lawson, F. Abascal, P. A. Nicola, S. Lensing, Amy L Roberts, Georgios Kalantzis, A. Baez-Ortega, Natalia Brzozowska, Julia S. El-Sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, Ayrun Nessa, S. Wadge, Anna L. Paterson, Doris M. Rassl, R. E. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, M. Neville, M. Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, Nick Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D. Young, A. KrishnaaT., Mahbubani, K. Saeb-Parsy, Hilary C. Martin, M. R. Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2024-11-01</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Macrophages maintain hematopoietic stem cell (HSC) quality by assessing cell surface Calreticulin (Calr), an “eat-me” signal induced by reactive oxygen species (ROS). Using zebrafish genetics, we identified Beta-2-microglobulin (B2m) as a crucial “don’t eat-me” signal on blood stem cells. A chemical screen revealed inducers of surface Calr that promoted HSC proliferation without triggering ROS or macrophage clearance. Whole-genome CRISPR-Cas9 screening showed that Toll-like receptor 3 (Tlr3) signaling regulated b2m expression. Targeting b2m or tlr3 reduced the HSC clonality. Elevated B2m levels correlated with high expression of repetitive element (RE) transcripts. Overall, our data suggest that RE-associated double-stranded RNA could interact with TLR3 to stimulate surface expression of B2m on hematopoietic stem and progenitor cells. These findings suggest that the balance of Calr and B2m regulates macrophage-HSC interactions and defines hematopoietic clonality. Editor’s summary During embryonic development, macrophages interact with blood stem cells, removing those exhibiting high levels of cellular stress. The surviving hematopoietic stem and progenitor cells (HSPCs) can proliferate and give rise to blood cell linages in adults. Pessoa Rodrigues et al. used zebrafish models and mammalian systems to investigate signals within HSPCs that could prevent macrophages from engulfing them. Molecules associated with innate immune nucleic acid sensing promoted the expression of Beta-2-microglobulin on the surface of HSPCs to increase their survival when they interacted with macrophages. Impairing this pathway decreased the number of distinct stem cell clones in adult zebrafish. —Sarah H. Ross  INTRODUCTION Hematopoietic stem and progenitor cells (HSPCs) produce mature blood cells throughout life. During this process, HSPCs engage in interactions with the variety of cells present in the marrow niche, including macrophages. These macrophages mediate a myriad of processes by releasing cytokines and chemokines and patrolling to remove stressed, dead, or aging cells. In this manner, macrophages contribute to the maintenance of tissue homeostasis. During hematopoiesis, macrophages ensure the quality of normal blood stem cells and determine the number of hematopoietic clones that participate in adult hematopoiesis. They then either engulf a stem cell completely, referred to as “dooming,” or capture portions of the stem cell’s cellular material, referred to as “grooming.” After grooming, the stem cell continues to divide, whereas the doomed HSPC clone is eliminated. RATIONALE Past work has shown that the HSPC-macrophage interaction is mediated by an “eat-me” signal, calreticulin (Calr), on the surface of HSPCs. Surface Calr levels are increased in stem cells with higher levels of reactive oxygen species (ROS), and this triggers macrophages to engulf and kill stem cells that have high amounts of stress-activated proteins. However, the specific molecular cues that regulate the dooming versus grooming behavior are still unknown. RESULTS To investigate the cues mediating grooming versus dooming, we screened a panel of 1200 bioactive small molecules in human cells and identified 93 compounds that robustly increased surface CALR in a dose-dependent manner. Of these compounds, 22 also facilitated interactions between macrophages and stem cells in zebrafish. Compounds that depended on ROS to increase CALR showed higher dooming ratios and, conversely, animals treated with ROS-independent compounds exhibited a higher probability of grooming events despite increased CALR and increased macrophage–stem cell interactions. To investigate the signals involved in interactions under ROS-independent conditions, we undertook a genome-wide screen in human cells. We identified Toll-like receptor 3 (TLR3) as a CALR inducer in the “don’t eat me” context. Next, we found that TLR3 expression was required for the expression of the “don’t eat me” signal Beta-2-microglobulin (B2M). Antibody staining and studies of a b2m knockout zebrafish line showed that b2m on stem cells was required for preventing the macrophage dooming. To determine whether the increased dooming observed in the b2m mutant affected clonal dominance, we generated mosaic deletions using the zebrabow color barcoding system (TWISTR) as a lineage tracer. Mosaic deletion of b2m reduced the number of myeloid clones but increased clonal dominance. The dominant clones in these fish were wild type for b2m that were resistant to dooming and overtook the adult marrow. TLR3 signaling is activated by double-stranded RNA, which can sometimes be found in cells due to the expression of transcripts associated with repetitive elements (REs), including long terminal repeats (LTRs) and endogenous retroviruses. RE expression in HSPCs positively correlated with elevated levels of B2m. Moreover, overexpressing ltr4 in zebrafish led to higher HSPC proliferation, indicative of decreased dooming. Impairing the TLR3-B2M pathway by disrupting the expression of TLR3, IRF3, or B2M expression decreased the number of blood cell clones in zebrafish. Using the DNA methyltransferase inhibitor CM272 to promote up-regulation of REs amplified the proliferation of already established HSPC clones in zebrafish embryos. CONCLUSION Our findings support a model in which endogenous TLR3 ligands lead to expression of b2m, thereby providing a blockade against macrophage-induced dooming and determining hematopoietic stem cell clonality by monitoring stem cell quality with macrophages. The balance of Calr and B2m governs the number of long-lived HSC clones that contribute to the adult blood system through macrophage-mediated quality assurance. This sheds light on the intricate interplay between HSPC clonal expansion and immune-regulated cell clearance. We propose that this protective mechanism may also operate in adulthood in response to environmental stress, such as during infections or in clonal stem cell disorders. Manipulating the levels of “don’t eat-me” and “eat-me” signals may have important therapeutic implications for immune therapy by harnessing the macrophage-selective removal of a mutated stem cell clone. ROS and TLR3 signaling mediates the “eat-me” and “don’t eat me” signals that determine stem cell clonal selection by macrophages. Shown is a schematic overview of the proposed model showing the balance between the Calr-mediated “eat-me” and B2m-mediated “don’t eat-me” signals that influence macrophage behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da253e73cd7f41de67f6f4bba52c4af27467af93" target='_blank'>
              Transcripts of repetitive DNA elements signal to block phagocytosis of hematopoietic stem cells
              </a>
            </td>
          <td>
            Cecilia Pessoa Rodrigues, Joseph M. Collins, Song Yang, Catherine Martinez, Ji Wook Kim, Chhiring Lama, Anna S. Nam, Clemens Alt, Charles Lin, Leonard I. Zon
          </td>
          <td>2024-09-13</td>
          <td>Science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e2a88356e361562e8a5340386c3ce72aae9f6e" target='_blank'>
              m6A-modified circXPO1 accelerates colorectal cancer progression via interaction with FMRP to promote WWC2 mRNA decay
              </a>
            </td>
          <td>
            Xiaowen Zhu, Pengxia Zhang
          </td>
          <td>2024-10-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) have emerged as crucial regulators of cellular processes, overturning their previous classification as “junk” DNA. Their involvement in various cellular functions makes them potential therapeutic targets in a range of diseases, particularly in tumorigenesis and cancer. Among these lncRNAs, MALAT1 (Metastasis-Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear-Enriched Abundant Transcript 1) are prominent oncogenic lncRNAs with complex regulatory roles across multiple cancer types. Although the roles of these lncRNAs in cancer have been extensively studied, the majority of these investigations have been conducted in 2D culture systems. While 2D cell cultures are well-established and widely accepted models in cell biology, they lack the physiological complexity of 3D tumor architecture. In contrast, 3D cultures, where cells grow as three-dimensional clusters, better mimic in vivo conditions, making them essential for understanding tumor microenvironment (TME) dynamics. Despite the importance of 3D models, there is a lack of literature exploring lncRNA expression in 2D monolayers versus 3D cultures across different cancer types. This study examines the expression and function of the lncRNAs NEAT1 and MALAT1 in 3D tumoroids compared to 2D monolayer cultures, aiming to bridge the gap between in vitro models and the complex in vivo tumor microenvironment. We addressed this gap by quantifying the expression differences of NEAT1 and MALAT1 using qRT-PCR in breast cancer, liver cancer, cervical cancer, and glioblastoma (GBM) cells grown in both 2D and 3D cultures. Significant variations in NEAT1 and MALAT1 expression were observed between 2D and 3D cultures across these cancer types, signifying the need to study lncRNAs in 3D micro-environments. Furthermore, we established correlations between NEAT1 expression and cancer traits such as stemness, invasion, glucose transporter expression, and epithelial-mesenchymal transition (EMT) in GBM 3D tumoroids compared to 2D monolayers. Using siRNA to downregulate NEAT1 in GBM tumoroids, we demonstrated that reducing NEAT1 expression to the levels comparable to 2D cultures led to a decrease in the expression of mRNA markers associated with stemness, invasion, glucose transporters, and EMT. Additionally, siRNA-mediated downregulation of NEAT1 in 3D tumoroids directly impacted cancer properties, as validated by phenotypic assays, where reduced proliferation, migration, and invasion were observed in GBM (comparing 3D to 2D models). Therefore, our work provides new insights into the expression of two key oncogenic lncRNAs in 3D microenvironments of various cancers and lays the foundation for integrating lncRNAs as important molecular players within 3D culture systems, offering a better understanding of the in vivo complexity of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4574fbc2dd767027ac41776d25b2fefa2c78aa8" target='_blank'>
              Differential expression of oncogenic lncRNAs NEAT1 and MALAT1 in 2D monolayer vs. 3D tumoroid culture and its implications in cancer progression
              </a>
            </td>
          <td>
            Arpita Ghosh, R. Soundharya, M. Jolly, A. Majumder
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The role of biomarkers in cancer treatment varies significantly depending on the cancer stage. Thus, in clinical practice, tailoring biomarkers to meet the specific needs and challenges of each cancer stage can increase the precision of treatment. Because they reflect underlying genetic alterations that influence cancer progression, copy number variation (CNV) biomarkers can play crucial prognostic roles. In our previous study, we identified potential survival-related genes for colorectal cancer (CRC) by analyzing CNV and gene expression data using a machine-learning approach. To further investigate the biological function of NRXN1, we assessed the use of RNA sequencing, phosphokinase assays, real-time quantitative PCR, and Western blot analysis. We found that NRXN1 copy number deletion was significantly associated with poor overall survival (OS) and recurrence-free survival (RFS), even in patients who received adjuvant chemotherapy. Compared with its expression in normal tissues, NRXN1 expression was lower in tumors, suggesting its potential role as a tumor suppressor. NRXN1 knockdown enhanced CRC cell viability and invasion, and transcriptome analysis indicated that the increased invasion was caused by GSK3β-mediated epithelial–mesenchymal transition. These findings highlight NRXN1 copy number deletion as a novel biomarker for predicting recurrence and survival in patients with resected colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e27ef10cd0612c6dea36dba917116a28dc5525f" target='_blank'>
              NRXN1 as a Prognostic Biomarker: Linking Copy Number Variation to EMT and Survival in Colon Cancer
              </a>
            </td>
          <td>
            H. Bang, H. Shim, M. Park, Sumin Yoon, Kyung Hyun Yoo, Young-Kook Kim, Hyunju Lee, Jeong-Seok Nam, Jun-Eul Hwang, W. Bae, Ik-Joo Chung, Eun-Gene Sun, Sang‐Hee Cho
          </td>
          <td>2024-10-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a26381c0fedff8488ed5a00cbd45a9c5cfc33d" target='_blank'>
              Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer
              </a>
            </td>
          <td>
            Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2024-10-14</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Eukaryotic chromosomes segregate faithfully prior to nuclear division to ensure genome stability. If segregation becomes defective, the chromosome copy number of the cell may alter leading to aneuploidy and/or polyploidy, both common hallmarks of cancers. In eukaryotes, aurora kinases regulate chromosome segregation during mitosis and meiosis, but their functions in the divergent, single-celled eukaryotic pathogen Trypanosoma brucei are less understood. Here, we focused on one of three aurora kinases in these parasites, TbAUK3, a homologue of the human aurora kinase AURKC, whose functions are primarily restricted to meiosis. We show that RNAi targeted depletion of TbAUK3 correlates with nuclear segregation defects, reduced proliferation, and decreased DNA synthesis, suggestive of a role for TbAUK3 during mitotic, not meiotic, chromosome segregation. Moreover, we uncover a putative role for TbAUK3 during the parasite’s response to DNA damage since we show that depletion of TbAUK3 enhances DNA instability and sensitivity to genotoxic agents. HIGHLIGHTS The C-terminus of TbAUK3 is disordered TbAUK3 depletion coincides with nuclear segregation defects Depletion of TbAUK3 enhances DNA instability">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125d96f157b6efd3c7ef3b8d6665537e416170cc" target='_blank'>
              AUK3 is required for faithful nuclear segregation in the bloodstream form of Trypanosoma brucei
              </a>
            </td>
          <td>
            J. A. Black, B. Poulton, B.S. Gonzaga, A. Iskantar, L. Tosi, R. McCulloch
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The transcriptional regulator ATRX, a genetic factor, is associated with a range of disabilities, including intellectual, hematopoietic, skeletal, facial, and urogenital disabilities. ATRX mutations substantially contribute to the pathogenesis of ATRX syndrome and are frequently detected in gliomas and many other cancers. These mutations disrupt the organization, subcellular localization, and transcriptional activity of ATRX, leading to chromosomal instability and affecting interactions with key regulatory proteins such as DAXX, EZH2, and TERRA. ATRX also functions as a transcriptional regulator involved in the pathogenesis of neuronal disorders and various diseases. In conclusion, ATRX is a central protein whose abnormalities lead to multiple diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a69809c785d10b202236aaadc03559763ad0906" target='_blank'>
              Mutant ATRX: pathogenesis of ATRX syndrome and cancer
              </a>
            </td>
          <td>
            Kejia Yuan, Yan Tang, Zexian Ding, Lei Peng, Jinghua Zeng, Huaying Wu, Qi Yi
          </td>
          <td>2024-10-16</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) describes the process where distinct epigenetic states may be transmitted between generations, resulting in stable gene expression and phenotypic differences between individuals that persist independently of DNA sequence variation. Chromatin modifications have been demonstrated as important in TEI, however, the extent to which they require other signals to establish and maintain epigenetic states is still unclear. Here we investigate whether small non-coding RNAs contribute to different epigenetic states of the Fab2L transgene in Drosophila triggered by transient chromatin contacts, which requires Polycomb complex activity to deposit the H3K27me3 modification for long-term TEI. Using mutants deficient in known small non-coding RNAs, high throughput sequencing, investigation of chromatin conformation and gene expression analysis we demonstrate that small non-coding RNAs do not contribute directly to initiation or maintenance of silencing. However, we uncover an indirect role for microRNA expression in transgene silencing through effects on Polycomb complex expression. Additionally, we show that a commonly used marker gene, Stubble (Sb), affects Polycomb complex expression, which may be important in interpreting experiments assaying Polycomb function in Drosophila development. By ruling out a plausible candidate for TEI at the Fab2L transgene our work highlights the variability in different modes of TEI across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be548511a3f566003319963d618b6d4697f22992" target='_blank'>
              Polycomb-mediated transgenerational epigenetic inheritance of Drosophila eye colour is independent of small RNAs
              </a>
            </td>
          <td>
            Maximilian H. Fitz-James, Poppy Sparrow, Christopher Paton, P. Sarkies
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="During meiosis, each chromosome pair experiences at least one crossover (CO), which directs their balanced segregation in addition to shuffling genetic information. COs tend to be away from each other, a phenomenon known as CO interference. The main biochemical pathway for CO formation, which is conserved in distant eukaryotes, involves the ZMM proteins together with the MLH1-MLH3 complex (MutLγ). Here, we aim to clarify the role of MutLγ in CO formation in Arabidopsis thaliana. We show that AtMutLγ is partially dispensable for ZMM-dependant CO formation. HEI10 large foci - that mark CO sites in wild-type-form at a normal level in mlh1 and mlh3 mutants, but are inefficiently maturated into COs. Mutating the MUS81 nuclease in either mlh1 or mlh3 leads to chromosome fragmentation, which is suppressed by further mutating the zmm msh5. This suggests that in the absence of MutLγ, recombination intermediates produced by ZMMs are resolved by MUS81, which does not ensure CO formation. Finally, CO interference is not affected in mlh1, which is compatible with a random sub-sampling of normally patterned CO sites. We conclude that AtMutLγ imposes designated recombination intermediates to be resolved exclusively as COs, supporting the view that MutLγ asymmetrically resolves double-Holliday junctions, yielding COs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/804b8cf2149074d9d9f514d468fba802009e7940" target='_blank'>
              MutLγ enforces meiotic crossovers in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Stéphanie Durand, Qichao Lian, Victor Solier, J. B. Fernandes, Raphael Mercier
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b42acb0d0eb60fb675a97026237e9429fa851a1b" target='_blank'>
              Early-life tobacco exposure is causally implicated in aberrant RAG-mediated recombination in childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tanxin Liu, Keren Xu, Anmol Pardeshi, S. Myint, Alice Y Kang, L. Morimoto, Michael R Lieber, Joseph L Wiemels, Scott Kogan, C. Metayer, A. D. de Smith
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea678de24f2edf0463772717c15047d50861f390" target='_blank'>
              Characterization of mitochondrial DNA mutations in colorectal cancer progression by in silico approach and use as potential biomarkers for diagnosis and prognosis
              </a>
            </td>
          <td>
            Ramya Gadicherla, Niraj Rai, Rajath Othayoth, Srinivasulu Kamma
          </td>
          <td>2024-10-28</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Objective This study aimed to explore the heterogeneity of tumor endothelial cells (TECs) in hepatocellular carcinoma (HCC) and their role in tumor progression, with the goal of identifying new therapeutic targets and strategies to improve patient prognosis. Methods Single-cell RNA sequencing data from nine primary liver cancer samples were analyzed, obtained from the Gene Expression Omnibus (GEO) database. Data preprocessing, normalization, dimensionality reduction, and batch effect correction were performed based on the Seurat package. HCC cell types were identified using uniform manifold approximation and projection (UMAP) and cluster analysis, and the different cell types were annotated using the CellMarker database. Pseudotime trajectory analysis was conducted with Monocle to explore the differentiation trajectory of TECs. MAPK signaling pathway activity and copy number variations (CNV) in TECs were analyzed in conjunction with data from The Cancer Genome Atlas (TCGA), the trans-well and wound healing assay was used for cell invasion and migration activity assessment. Results Two subgroups of TECs (TECs 1 and TECs 2) were identified, exhibiting distinct functional activities and signaling pathways. Specifically, TECs 1 may be involved in tumor cell proliferation and inflammatory responses, whereas TECs 2 is not only involved in cell proliferation pathways, but also enriched in pathways such as metabolic synthesis. Pseudotime analysis revealed dynamic changes in TECs subgroups during HCC progression, correlating specific gene expressions (such as PDGFRB, PGF, JUN, and NR4A1). Subsequently, the JUN gene was predicted by performing binding sites and was shown to act as a transcription factor that may regulate the expression of the PGF gene. CNV analysis highlighted key genes and pathways in TECs that might influence HCC progression, and the PGF as key regulatory factor mediated cell proliferation and migration. Conclusion The study revealed the heterogeneity of TECs in HCC and their potential roles in tumor progression, offering new perspectives and potential therapeutic targets for HCC molecular mechanisms. The findings emphasize the importance of further exploring TECs heterogeneity for understanding HCC pathogenesis and developing personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db89c0c7a388ba83d2034d24b60625ad1b46b81d" target='_blank'>
              Exploring tumor endothelial cells heterogeneity in hepatocellular carcinoma: insights from single-cell sequencing and pseudotime analysis
              </a>
            </td>
          <td>
            Jiachun Sun, Shujun Zhang, Yafeng Liu, Kaijie Liu, Xinyu Gu
          </td>
          <td>2024-10-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e3c1ad704a586aa01f4f3735cd6a0421899426" target='_blank'>
              Rise and SINE: roles of transcription factors and retrotransposons in zygotic genome activation.
              </a>
            </td>
          <td>
            Pavel Kravchenko, Kikuë Tachibana
          </td>
          <td>2024-10-02</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Cancers frequently co-opt lineage-specific transcription factors (TFs) utilized in normal development to sustain proliferation. However, the effects of these TFs on tumor development depend considerably on where in the genome they bind. A new paper by Taylor and colleagues expands on previously developed diamidine compounds that obstruct the DNA binding sites of the pioneer TF PU.1 (SPI1) in acute myeloid leukemia (AML). Immobilization and sequencing of genomic DNA targeted by these compounds revealed that these inhibitors alter the genomic binding patterns of PU.1. The authors report that their strategy constrains the genomic binding preferences of PU.1, leading to redistribution of PU.1 to promoters and other gene-proximal regions with elevated G/C content. Here we discuss recent developments for targeting PU.1 in hematologic malignancies. We also explore the shared functional roles of PU.1 and SWI/SNF ATP-dependent chromatin remodeling complexes, which work together to sustain the enhancer landscape needed for tumor cell proliferation but also have key roles in non-tumor settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241dbf292aad4fcef078a200a7fb9e680896d9df" target='_blank'>
              Pharmacological blockade of a pioneer transcription factor.
              </a>
            </td>
          <td>
            Katerina Cermakova, H. C. Hodges
          </td>
          <td>2024-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Survivin is known for its dual biological role in apoptosis inhibition and mitotic progression. In addition to its being part of the chromosomal passenger complex (CPC), recent findings suggest additional roles for Survivin in the DNA damage response, further contributing to therapy resistance. In this study, we investigated the role of Survivin and the CPC proteins in the cellular response to irradiation with a focus on DNA replication processes. As is known, ionizing radiation leads to an increased expression of Survivin and its accumulation in nuclear foci, which we now know to be specifically localized to centromeric heterochromatin. The depletion of Survivin and Aurora B increases the DNA damage marker γH2AX, indicative of an impaired repair capacity. The presence of Survivin and the CPC in nuclear foci that we already identified during the S phase co-localize with the proliferating cell nuclear antigen (PCNA), further implying a potential role during replication. Indeed, Survivin knockdown reduced replication fork speed as assessed via DNA fiber assays. Mechanistically, we identified a PIP-box motif in INCENP mediating the interaction with PCNA to assist in managing damage-induced replication stress. Survivin depletion forces cells to undergo unphysiological genome replication via mitotic DNA synthesis (MiDAS), resulting in chromosome breaks. Finally, we revealed that Aurora B kinase liberates Pol η by phosphorylating polymerase delta-interacting protein 2 (POLDIP2) to resume the replication of damaged sites via translesion synthesis. In this study, we assigned a direct function to the CPC in the transition from stalled replication forks to translesion synthesis, further emphasizing the ubiquitous overexpression of Survivin particularly in tumors. This study, for the first time, assigns a direct function to the chromosomal passenger complex, CPC, including Survivin, Aurora B kinase, Borealin, and INCENP, in the transition from stalled replication forks (involving PCNA binding) to translesion synthesis (liberating Pol η by phosphorylating POLDIP2), and thus in maintaining genomic integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933e65144a080ac3e72c369a27df2e2dc4c884df" target='_blank'>
              Old Passengers as New Drivers: Chromosomal Passenger Proteins Engage in Translesion Synthesis
              </a>
            </td>
          <td>
            Katharina Falke, E. Schröder, S. Mosel, Cansu N. Yürük, Sophie Feldmann, Désirée Gül, Paul Stahl, Roland H. Stauber, S. Knauer
          </td>
          <td>2024-10-31</td>
          <td>Cells</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The Aurora Kinases (AURKs) are a family of serine-threonine protein kinases critical for cell division. Somatic cells express only AURKA and AURKB. However, mammalian germ cells and some cancer cells express all three isoforms. A major question in the field has been determining the molecular and cellular changes when cells express three instead of two aurora kinases. Using a systematic genetic approach involving different Aurora kinase oocyte-specific knockout combinations, we completed an oocyte-AURK genetic interaction map and show that one genomic copy of Aurka is necessary and sufficient to support female fertility and oocyte meiosis. We further confirm that AURKB and AURKC alone cannot compensate for AURKA. These results highlight the importance of AURKA in mouse oocytes, demonstrating that it is required for spindle formation and proper chromosome segregation. Surprisingly, a percentage of oocytes that lack AURKB can complete meiosis I, but the quality of those eggs is compromised, suggesting a role in AURKB to regulate spindle assembly checkpoint or control the cell cycle. Together with our previous studies, we wholly define the genetic interplay among the Aurora kinases and reinforce the importance of AURKA expression in oocyte meiosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb4d8aa115a4d330a74f0703eb535ccdfa772154" target='_blank'>
              Genetic interaction mapping of Aurora protein kinases in mouse oocytes
              </a>
            </td>
          <td>
            Cecilia S. Blengini, Karen Schindler
          </td>
          <td>2024-09-25</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary In tumorigenesis, aberrant DNA methylation may be an earlier and stronger modifier of gene expression than mutations. Herein, 128 serous ovarian tumors were analyzed, including borderline ovarian tumors (BOTS) with (BOT.V600E) and without (BOT) the BRAF V600E mutation, low-grade (lg), and high-grade (hg) ovarian cancers (OvCa). The methylome of the samples was profiled with Infinium MethylationEPIC microarrays. Global, genome-wide hypomethylation positively correlated with the increasing aggressiveness of tumors, being the strongest in hgOvCa. Remarkably, the ten most significant differentially methylated regions (DMRs) in the genome, discriminating BOT from lgOvCa, encompassed the MHC region on chromosome 6. We also identified hundreds of DMRs potentially useful as predictive biomarkers in BOTS and hgOvCa. DMRs with the best discriminative capabilities overlapped the following genes: BAIAP3, IL34, WNT10A, NEU1, SLC44A4, and HMOX1, TCN2, PES1, RP1-56J10.8, ABR, NCAM1, RP11-629G13.1, AC006372.4, NPTXR in BOTS and hgOvCa, respectively. By identifying potential biomarkers, this study might improve ovarian tumor outcome. Abstract Background: Changes in DNA methylation patterns are a pivotal mechanism of carcinogenesis. In some tumors, aberrant methylation precedes genetic changes, while gene expression may be more frequently modified due to methylation alterations than by mutations. Methods: Herein, 128 serous ovarian tumors were analyzed, including borderline ovarian tumors (BOTS) with (BOT.V600E) and without (BOT) the BRAF V600E mutation, low-grade (lg), and high-grade (hg) ovarian cancers (OvCa). The methylome of the samples was profiled with Infinium MethylationEPIC microarrays. Results: The biggest number of differentially methylated (DM) CpGs and regions (DMRs) was found between lgOvCa and hgOvCa. By contrast, the BOT.V600E tumors had the lowest number of DM CpGs and DMRs compared to all other groups and, in relation to BOT, their genome was strongly downmethylated. Remarkably, the ten most significant DMRs, discriminating BOT from lgOvCa, encompassed the MHC region on chromosome 6. We also identified hundreds of DMRs, being of potential use as predictive biomarkers in BOTS and hgOvCa. DMRs with the best discriminative capabilities overlapped the following genes: BAIAP3, IL34, WNT10A, NEU1, SLC44A4, and HMOX1, TCN2, PES1, RP1-56J10.8, ABR, NCAM1, RP11-629G13.1, AC006372.4, NPTXR in BOTS and hgOvCa, respectively. Conclusions: The global genome-wide hypomethylation positively correlates with the increasing aggressiveness of ovarian tumors. We also assume that the immune system may play a pivotal role in the transition from BOTS to lgOvCa. Given that the BOT.V600E tumors had the lowest number of DM CpGs and DMRs compared to all other groups, when methylome is considered, such tumors might be placed in-between BOT and OvCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c468b8eb0505c51d6a927efad191c6a8ba7d2c09" target='_blank'>
              The Diversity of Methylation Patterns in Serous Borderline Ovarian Tumors and Serous Ovarian Carcinomas
              </a>
            </td>
          <td>
            L. A. Szafron, R. Iwanicka-Nowicka, Piotr Sobiczewski, Marta Koblowska, A. Dansonka-Mieszkowska, J. Kupryjańczyk, L. Szafron
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Dysregulation of DNA methylation is associated with human disease, particularly cancer, and the assessment of aberrant methylation patterns holds great promise for clinical diagnostics. However, DNA polymerases do not effectively discriminate between processing 5-methylcytosine (5 mC) and unmethylated cytosine, resulting in the silencing of methylation information during amplification or sequencing. As a result, current detection methods require multi-step DNA conversion treatments or careful analysis of sequencing data to decipher individual 5 mC bases. To overcome these challenges, we propose a novel DNA polymerase-mediated 5 mC detection approach. Here, we describe the engineering of a thermostable DNA polymerase variant derived from Thermus aquaticus with altered fidelity towards 5 mC. Using a screening-based evolutionary approach, we have identified a DNA polymerase that exhibits increased misincorporation towards 5 mC during DNA synthesis. This DNA polymerase generates mutation signatures at methylated CpG sites, allowing direct detection of 5 mC by reading an increased error rate after sequencing without prior treatment of the sample DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/200dec0a6a354853926cc8c01bdd9f50727f0221" target='_blank'>
              A DNA Polymerase Variant Senses the Epigenetic Marker 5-Methylcytosine by Increased Misincorporation.
              </a>
            </td>
          <td>
            Melanie Henkel, Alexander Fillbrunn, V. Marchand, Govindan Raghunathan, Michael R Berthold, Y. Motorin, Andreas Marx
          </td>
          <td>2024-10-24</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ed2057a00d451d2750eb4c6690ca3020a9e3a6" target='_blank'>
              The prominent pervasive oncogenic role and tissue specific permissiveness of RAS gene mutations
              </a>
            </td>
          <td>
            Ming Yi, Daniel Soppet, Frank McCormick, D. Nissley
          </td>
          <td>2024-10-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Determining a novel etiology and mechanism of human cancer requires extraction of characteristic mutational signatures derived from chemical substances. This study explored the mutational signatures of N-nitroso bile acid conjugates using Salmonella strains. Exposing S. typhimurium TA1535 to N-nitroso-glycine/taurine bile acid conjugates induced a predominance of C:G to T:A transitions. Two mutational signatures, B1 and B2, were extracted. Signature B1 is associated with N-nitroso-glycine bile acid conjugates, while Signature B2 is linked to N-nitroso-taurine bile acid conjugates. Signature B1 revealed a strong transcribed strand bias with GCC and GCT contexts, and the mutation pattern of N-nitroso-glycine bile acid conjugates in YG7108, which lacks O6-methylguanine DNA methyltransferases, matched that of the wild-type strain TA1535, suggesting that O6-methyl-deoxyguanosine contributes to mutations in the relevant regions. COSMIC database-based similarity analysis revealed that Signature B1 closely resembled SBS42, which is associated with occupational cholangiocarcinoma caused by overexposure to 1,2-dichlolopropane (1,2-DCP) and/or dichloromethane (DCM). Moreover, the inflammatory response pathway was induced by 1,2-DCP exposure in a human cholangiocyte cell line, and iNOS expression was positive in occupational cholangiocarcinomas. These results suggest that 1,2-DCP triggers an inflammatory response in biliary epithelial cells by upregulating iNOS and N-nitroso-glycine bile acid conjugate production, resulting in cholangiocarcinoma via DNA adduct formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16ffe28d485b3038d354ba43187bea62b1635ee" target='_blank'>
              Whole Genome Sequencing Analysis of Model Organisms Elucidates the Association Between Environmental Factors and Human Cancer Development
              </a>
            </td>
          <td>
            Shinya Hasegawa, Yutaka Shoji, Mamoru Kato, Asmaa Elzawahry, M. Nagai, Min Gi, Shugo Suzuki, Hideki Wanibuchi, S. Mimaki, K. Tsuchihara, Y. Totsuka
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7228635119ad497118a336ef6dd401a87d976c61" target='_blank'>
              LINC01320 facilitates cell proliferation and migration of ovarian cancer via regulating PURB/DDB2/NEDD4L/TGF-β axis
              </a>
            </td>
          <td>
            Gaigai Wang, Bingya Xu, Xiangling Yu, Meng Liu, Tiantian Wu, Wenxin Gao, Haoyue Hu, Bing Jiang, Yibo Wu, Tao Zhou, Xia Chen, Cong Shen
          </td>
          <td>2024-10-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Differential RNA expression is becoming increasingly valuable in evaluating tumor heterogeneity for a better understanding of malignant tumors and guiding personalized therapy. However, traditional techniques for analyzing cellular RNA are mainly focused on determining the absolute level of RNA, which may lead to inaccuracies in understanding tumor heterogeneity, primarily due to i) the subtle differences in certain RNA types that have similar total concentrations and ii) the existence of variations in RNA expression across different samples. Herein, a detachable DNA assembly module is proposed that is capable not only of quantifying the expression level of target RNA but also of innovatively evaluating its proportion within its RNA family population through a sequential assembly and disassembly route. Using the let-7 family as an experimental model, a significant difference is discovered in let-7a proportion between normal mammary epithelial cells and breast cancer cells, a characteristic that is often missed in bulk analysis of traditional techniques. By combining concentration and proportion information, the detachable DNA assembly module demonstrates markedly higher efficiency in discerning among various types of cells compared to traditional techniques. This innovative assembly module is expected to offer a new perspective to highlight tumor heterogeneity and guide personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b6572cf9fba5fd9ee657a20fd3934f1dd18a4" target='_blank'>
              Detachable DNA Assembly Module to Dissect Tumor Cells Heterogeneity via RNA Pinpoint Screening.
              </a>
            </td>
          <td>
            Wei Liu, Ni Liao, Yanmei Lei, Wenbin Liang, Xia Yang, Ruo Yuan, Chao Yang, Ying Zhuo
          </td>
          <td>2024-10-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for assessing chromatin stability in healthy and sick dogs (with squamous cell carcinoma). The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable a more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. Abstract Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Deleterious germline DDX41 variants constitute the most common inherited predisposition disorder linked to myeloid neoplasms (MNs). The role of DDX41 in hematopoiesis and how its germline and somatic mutations contribute to MNs remain unclear. Here we show that DDX41 is essential for erythropoiesis but dispensable for the development of other hematopoietic lineages. Using stage-specific Cre models for erythropoiesis, we reveal that Ddx41 knockout in early erythropoiesis is embryonically lethal, while knockout in late-stage terminal erythropoiesis allows mice to survive with normal blood counts. DDX41 deficiency induces a significant upregulation of G-quadruplexes (G4), noncanonical DNA structures that tend to accumulate in the early stages of erythroid precursors. We show that DDX41 co-localizes with G4 on the erythroid genome. DDX41 directly binds to and dissolves G4, which is significantly compromised in MN-associated DDX41 mutants. Accumulation of G4 by DDX41 deficiency induces erythroid genome instability, defects in ribosomal biogenesis, and upregulation of p53. However, p53 deficiency does not rescue the embryonic death of Ddx41 hematopoietic-specific knockout mice. In parallel, genome instability also activates the cGas-Sting pathway, which is detrimental to survival since cGas-deficient and hematopoietic-specific Ddx41 knockout mice are viable without detectable hematologic phenotypes, although these mice continue to show erythroid ribosomal defects and upregulation of p53. These findings are further supported by data from a DDX41 mutated MN patient and human iPSC-derived bone marrow organoids. Our study establishes DDX41 as a G4 dissolver, essential for erythroid genome stability and suppressing the cGAS-STING pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5f497f73b0f90438a8c4db6703bd3c7df40581" target='_blank'>
              DDX41 dissolves G-quadruplexes to maintain erythroid genome integrity and prevent cGAS-mediated cell death
              </a>
            </td>
          <td>
            Honghao Bi, Kehan Ren, Pan Wang, Ermin Li, Xu Han, Wen Wang, Jing Yang, Inci Aydemir, Kara Tao, Lucy Godley, Yan Liu, Vipul Shukla, Elizabeth Bartom, Yuefeng Tang, Lionel Blanc, M. Sukhanova, Peng Ji
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e79df5b05ae1c9c7192172e60751801c75f9400" target='_blank'>
              An experimental census of retrons for DNA production and genome editing.
              </a>
            </td>
          <td>
            Asim G Khan, Matías Rojas-Montero, A. González-Delgado, Santiago C. Lopez, Rebecca F Fang, Kate D. Crawford, Seth L. Shipman
          </td>
          <td>2024-09-17</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Homologous recombination deficiency (HRD) represents an impairment in the homologous recombination repair (HRR) pathway, crucial for repairing DNA double‐strand breaks and contributing to genomic instability in cancer. The HRD score may be a more reliable biomarker than HRR‐related gene mutations for identifying patients sensitive to poly(ADP‐ribose) polymerase inhibitors. Despite its relevance in various cancers, the HRD score remains underexplored in esophageal squamous cell carcinoma (ESCC). We retrospectively analyzed HRD scores in 96 ESCC patients, examining correlations with clinical characteristics and survival outcomes, and validated our findings using the TCGA dataset. Genomic sequencing utilized a custom superHRD next‐generation sequencing panel, and HRD scores were calculated from 54,000 single‐nucleotide polymorphisms using Kruskal–Wallis rank‐sum tests and two cut‐off points for analysis. Higher HRD scores correlated with advanced tumor stages, recurrence, and mutations in TP53 and ABCB1, while APC mutations were linked to lower HRD scores. Patients with high HRD scores had significantly shorter disease‐free survival (p = 0.013) and a trend toward shorter overall survival (OS) (p = 0.005), particularly those not receiving adjuvant therapy. Conversely, HRD‐high patients undergoing adjuvant therapy showed a trend toward longer OS (p = 0.015). Multivariate analysis identified HRD as an independent prognostic factor (hazard ratio = 2.814 for recurrence, p = 0.015). Validation with the TCGA dataset supported these findings. This study highlights the associations between HRD scores, clinical characteristics, and genomic mutations in ESCC, suggesting HRD as a potential prognostic biomarker. HRD assessment may aid in patient stratification and personalized treatment strategies, warranting further investigation to validate the therapeutic implications of HRD scores in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d912e5a2dbed0b5c41b52a5becc5d90cac8b304f" target='_blank'>
              Homologous recombination deficiency score is an independent prognostic factor in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Yulu Wang, Bowen Ding, Yunlan Tao, Lingli Huang, Qian Zhu, Chengying Gao, Mingli Feng, Yuchen Han
          </td>
          <td>2024-10-29</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The integrity of the actively transcribed genome against helix-distorting DNA lesions relies on a multilayered cellular response that enhances Transcription-Coupled Nucleotide Excision Repair (TC-NER). When defective, TC-NER is causatively associated with Cockayne-Syndrome (CS), a rare severe human progeroid disorder. Although the presence of unresolved transcription-blocking lesions is considered a driver of the aging process, the molecular features of the transcription-driven response to genotoxic stress in CS-B cells remain largely unknown. Here, an in-depth view of the transcriptional and associated chromatin dynamics that occur in CS-B cells illuminates the role of CSB therein. By employing high-throughput genome-wide approaches, we observed that absence of a functional CSB protein results in a delay in transcription progression, more positioned +1 nucleosomes, and less dynamic chromatin structure, compared to normal cells. We found that early after exposure to UV, CS-B cells released RNA polymerase II (RNAPII) from promoter-proximal pause sites into elongation. However, the magnitude of this response and the progression of RNAPII were reduced compared to normal counterparts. Notably, we detected increased post-UV retainment of unprocessed nascent RNA transcripts and chromatin-associated elongating RNAPII molecules. Contrary to the prevailing models, we found that transcription initiation is operational in CS-B fibroblasts early after UV and that chromatin accessibility showed a marginal increase. Our study provides robust evidence for the role of CSB in shaping the transcription and chromatin landscape both in homeostasis and in response to genotoxic insults, which is independent of its known role in TC-NER, and which may underlie major aspects of the CS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966a15785f4d1763d145786faf0eda45c3c2df7f" target='_blank'>
              Cockayne syndrome B protein is implicated in transcription and associated chromatin dynamics in homeostatic and genotoxic conditions.
              </a>
            </td>
          <td>
            Anastasios Liakos, Katerina Z Ntakou-Zamplara, Nelina Angelova, Dimitris Konstantopoulos, Anna-Chloe Synacheri, Zoi Spyropoulou, Iason A Tsarmaklis, Despoina Korrou-Karava, Georgios Nikolopoulos, Matthieu D. Lavigne, Maria Fousteri
          </td>
          <td>2024-10-06</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The DEK chromatin remodeling protein was previously shown to confer oncogenic phenotypes to human and mouse mammary epithelial cells using in vitro and knockout mouse models. However, its functional role in normal mammary gland epithelium remained unexplored. We developed two novel mouse models to study the role of Dek in normal mammary gland biology in vivo. Mammary gland-specific Dek over-expression in mice resulted in hyperproliferation of cells that visually resembled alveolar cells, and a transcriptional profile that indicated increased expression of cell cycle, mammary stem/progenitor, and lactation-associated genes. Conversely, Dek knockout mice exhibited an alveologenesis or lactation defect, resulting in dramatically reduced pup survival. Analysis of previously published single-cell RNA-sequencing of mouse mammary glands revealed that Dek is most highly expressed in mammary stem cells and alveolar progenitor cells, and to a lesser extent in basal epithelial cells, supporting the observed phenotypes. Mechanistically, we discovered that Dek is a modifier of Ezh2 methyltransferase activity, upregulating the levels of histone H3 trimethylation on lysine 27 (H3K27me3) to control gene transcription. Combined, this work indicates that Dek promotes proliferation of mammary epithelial cells via cell cycle deregulation. Furthermore, we report a novel function for Dek in alveologenesis and histone H3 K27 trimethylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f96378ddd56b750c6cb56cd86a3b8644a37da590" target='_blank'>
              The chromatin remodeler DEK promotes proliferation of mammary epithelium and is associated with H3K27me3 epigenetic modifications
              </a>
            </td>
          <td>
            Megan Johnstone, Ashley Leck, Taylor Lange, Katherine Wilcher, Miranda S. Shephard, Aditi Paranjpe, Sophia Schutte, Susanne Wells, Ferdinand Kappes, Nathan Salomonis, L. P. Privette Vinnedge
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42bb491bf59a8058f50cf3417e3f4f9a441fc86" target='_blank'>
              Cancer treatment monitoring using cell-free DNA fragmentomes
              </a>
            </td>
          <td>
            Iris van 't Erve, B. Alipanahi, K. Lumbard, Z. Skidmore, Lorenzo Rinaldi, Laurel K Millberg, J. Carey, B. Chesnick, S. Cristiano, Carter Portwood, Tony Wu, E. Peters, K. Bolhuis, Cornelis J A Punt, Jennifer Tom, P.B. Bach, N. Dracopoli, Gerrit A Meijer, Robert B Scharpf, V. Velculescu, R. Fijneman, A. Leal
          </td>
          <td>2024-10-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="In order to maintain genomic integrity, DNA replication must be highly coordinated. Disruptions in this process can cause replication stress which is aberrant in many pathologies including cancer. Despite this, little is known about the mechanisms governing the temporal regulation of DNA replication initiation, thought to be related to the limited copy number of firing factors. Here, we present a high (1-kilobase) resolution stochastic model of Saccharomyces cerevisiae whole-genome replication in which origins compete to associate with limited firing factors. After developing an algorithm to fit this model to replication timing data, we validated the model by reproducing experimental inter-origin distances, origin efficiencies, and replication fork directionality. This suggests the model accurately simulates the aspects of DNA replication most important for determining its dynamics. We also use the model to predict measures of DNA replication dynamics which are yet to be determined experimentally and investigate the potential impacts of variations in firing factor concentrations on DNA replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2569c7c42c75809ab1a5196f0e7f32c8168f06e" target='_blank'>
              Regulation of replication timing in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Rosie Berners-Lee, Eamonn Gilmore, Francis X Berkemeier, M. Boemo
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Latinos have a higher risk of developing gastric cancer (GC) than non-Latino Whites (NLW). Multiple studies have shown increased treatment resistance in diffuse-type cancers, which are more prevalent in Latino patients and dominated by the genomically stable (GS) genomic subtype. However, there is little genomic data from Latino gastric tumors due to the few studies targeting this minority population. Here, we attempt to rectify the disparity of genomic data by performing whole exome and/or low-coverage whole genome sequencing on tumors from 195 Latino gastric adenocarcinoma patients, from Colombia and Mexico. We also analyze 42 Latino patients from the US from a previous study. In total, we analyze tumors from 237 patients, the largest cohort of Latino gastric cancer tumors comprehensively sequenced thus far. Molecular subtyping of our cohort shows a higher proportion of tumors with a genomically stable (GS) subtype, 59%, and less with chromosomal instability (CIN), 24%, than reported in non-Latino patients from the Cancer Genome Atlas (20% and 50% respectively). Our earlier study of germline variants from Latinos, indicated a potential enrichment of variant related to homologous recombination deficiency (HRD), an important DNA repair mechanism. Focusing on this pathway, we performed HRD scoring and show that CIN tumors had the highest HRD score compared to all other subtypes. An alternative method analyzing somatic mutational patterns identified an indel mutational signature (ID6), also related to HRD, was enriched in GS tumors. Overall, these data provide a broader picture of the genomic landscape of Hispanic tumors, highlighting HRD targeting drugs as a potential treatment for both GS diffuse-types cancers particularly seen in Latinos and CIN tumors found more generally in gastric cancer.
 Citation Format: Dennis J Montoya, Ana Estrada-Florez, Paul Lott, Katherine Chiu, John Suarez-Olaya, Fabian Castro, Magdalena Echeverry de Polanco, Javier Torres, Mabel Bohorquez, Luis Carvajal-Carmona. Molecular characterization of gastric adenocarcinoma from Latinos shows homologous recombination deficiency [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C095.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b22944f466a6785df0771f5067c90022abd54c7" target='_blank'>
              Abstract C095: Molecular characterization of gastric adenocarcinoma from Latinos shows homologous recombination deficiency
              </a>
            </td>
          <td>
            Dennis J Montoya, Ana P. Estrada-Florez, Paul C. Lott, Katherine Chiu, John J. Suarez-Olaya, Fabian Castro, Magdalena Echeverry de Polanco, Javier Torres, M. Bohórquez, Luis G Carvajal-Carmona
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c35f97c112d09b7e983874a9efe5c2fc2f69d6c" target='_blank'>
              Enhancers in T Cell development and malignant lesions
              </a>
            </td>
          <td>
            Tong Zhang, Lin Zou
          </td>
          <td>2024-09-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: The heterogeneity of breast cancer (BC) subtypes poses a significant challenge, with carcinogenesis involving multiple stages and genes, including proto-oncogenes, tumor suppressor genes, and DNA repair genes. Next-generation sequencing has expanded access to multigene panels, such as RAD51 paralogs, which increase the risk of ovarian cancer and possibly triple-negative (TN) BC.
Case presentation: We present a rare case of a 45-year-old woman with TNBC and a RAD51D gene mutation. Mammography and breast ultrasonography revealed an irregular, 30 mm hypoechoic area and dystrophic calcifications in the right breast. Immunohistochemistry showed a lack of expression of ER, RP, HER-2, and P53, with 50% of neoplastic cell nuclei positive for Ki-67. Next-generation sequencing revealed a mutation in RAD51D and MUYTH genes. The patient underwent partial mastectomy, chemotherapy, and prophylactic mastectomy.
Conclusion: Genetic analysis is crucial for identifying specific mutations contributing to TNBC development. Current preventive interventions primarily address BRCA1 and BRCA2 mutations, following established guidelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad46462fe1e999efcd6af2efedf0788ba216b934" target='_blank'>
              A Rare Case of Triple-Negative Breast Cancer with RAD51D Gene Mutation
              </a>
            </td>
          <td>
            Annie Monica Ch, Rithwik Goud Burri, Sriharsha Lankala, Jun Yi Lim, Altynai Zhumabekova
          </td>
          <td>2024-10-25</td>
          <td>Archives of Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Precise regulation of gene expression is essential for proper development and the maintenance of homeostasis in organisms. Studies have shown that some transcriptional regulatory proteins influence gene expression through the formation of dynamic, locally concentrated assemblies known as condensates, while dysregulation of transcriptional condensates was associated with several cancers, such as Ewing sarcoma and AML [Wang Y et al. (2023) Nat Chem Biol 19, 1223-1234; Chandra B et al. (2022) Cancer Discov 12, 1152-1169]. Mutations in the histone acetylation "reader" eleven-nineteen-leukemia (ENL) have been shown to form discrete condensates at endogenous genomic targets, but it remains unclear how ENL mutations drive tumorigenesis and whether it is correlated with their condensate formation property. Liu et al. now show, using a conditional knock-in mouse model, that ENL YEATS domain mutation is a bona fide oncogenic driver for AML. This mutant ENL forms condensates in hematopoietic stem/progenitor cells at the genomic loci of key leukemogenic genes, including Meis1 and Hoxa cluster genes, and disrupting condensate formation via mutagenesis impairs its chromatin and oncogenic function. Furthermore, they show that small-molecule inhibition of the acetyl-binding activity displaces ENL mutant condensates from oncogenic target loci, and this inhibitor significantly impairs the onset and progression of AML driven by mutant ENL in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9e0f2605685a3812a479056f634ccfed5efb36" target='_blank'>
              ENL mutation and AML: a new model that reveals oncogenic condensate's function in leukemogenesis.
              </a>
            </td>
          <td>
            Zhong Fan, Yanan Jiang, Xiaotian Zhang
          </td>
          <td>2024-09-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb6a4f203d0ffda30fcc3f0dfa56bd3b405a484" target='_blank'>
              Amplified centrosomes—more than just a threat
              </a>
            </td>
          <td>
            Eva Kiermaier, Isabel Stötzel, Marina A Schapfl, A. Villunger
          </td>
          <td>2024-09-16</td>
          <td>EMBO Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a1777e23c2d1626a0cd330b4394f18dc2d3ea7" target='_blank'>
              Comment on, "Expression of decitabine-targeted oncogenes in meningiomas in vivo".
              </a>
            </td>
          <td>
            Hethesh Chellapandian, Sivakamavalli Jeyachandran
          </td>
          <td>2024-09-25</td>
          <td>Neurosurgical review</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Each cancer is the result of an individually unique evolutionary process. Yet, as demonstrated by extensive genetic analyses of human specimens, the genetic endpoints of cancers across different tissues are remarkably specific, with individual cancer driver genes being typically associated with only a limited set of tumor types. Tissue and cellular contexts thus impose strong and genetically predictable evolutionary constraints on carcinogenesis. Lineage-specific transcription factors, central regulators of organismal development, tissue homeostasis, and regeneration, often also support cancer cell fitness in a lineage-specific manner. In this review, we discuss recent results on the interactions between transcriptional lineage factor programs and oncogenic pathways and how such interactions may determine the oncogenic competence of cancer-associated genetic alterations. These developments are starting to shed light on the molecular basis of tissue specificity in carcinogenesis, with relevance for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f236b52bc63786a8f5d7d9e1f3c0d9457cd40be6" target='_blank'>
              Lineage-Specific Transcription Factors in Carcinogenesis
              </a>
            </td>
          <td>
            Charles J. David, Sakari Vanharanta
          </td>
          <td>2024-10-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background Breast cancer, despite significant advancements in treatment, remains a major cause of cancer-related deaths among women. Immunotherapy, an emerging therapeutic strategy, offers promise for better outcomes, particularly through the modulation of immune functions. Glioma-Associated Oncogene Homolog 1 (GLI1), a transcription factor implicated in cancer biology, has shown varying roles in different cancers. However, its immunoregulatory functions in breast invasive carcinoma (BRCA) remain elusive. The current study aimed to unravel the expression patterns and immune-regulatory roles of GLI1 in BRCA. Methods Utilizing multiple bioinformatic platforms (TIMER2.0, GEPIA2, and R packages) based on The Cancer Genome Atlas (TCGA) and/or Genotype-Tissue Expression (GTEx) databases, we analyzed the expression of GLI1 in BRCA and its pan-cancer expression profiles. We further validated these findings by conducting qPCR and immunohistochemical staining on clinical BRCA samples. Kaplan-Meier analysis and Cox proportional hazards regression were performed to assess the prognostic value of GLI1. Additionally, the association between GLI1 expression and immune infiltration within the tumor immune microenvironment (TMIE) was examined. Results The findings reveal dysregulated expression of GLI1 in numerous cancers, with a significant decrease observed in BRCA. High GLI1 expression indicated better survival outcomes and was correlated with the age and stage of BRCA patients. GLI1 was involved in immune status, as evidenced by its strong correlations with immune and stromal scores and the infiltration levels of multiple immune cells. Meanwhile, GLI1 was co-expressed with multiple immune-related genes, and high GLI1 expression was associated with the activation of immune-related pathways, such as binding to proteasome and mismatch repair and retinol metabolism signaling pathways. Additionally, the differential expression of GLI1 may be related to the effect of immunotherapy on CTLA-4, PD-1, and other signals, and can effectively predict the immune efficacy. Conclusion Our study underscores the critical role of GLI1 in BRCA, both as a potential tumor suppressor and an immune regulator. The association between GLI1 expression and favorable prognosis suggests its potential as a prognostic biomarker and immunotherapeutic target in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a81802d7ee86a4f3f182d6937952c4611208b17a" target='_blank'>
              Glioma-associated oncogene homolog 1 in breast invasive carcinoma: a comprehensive bioinformatic analysis and experimental validation
              </a>
            </td>
          <td>
            Teng Qi, Yujie Hu, Junhao Wan, Bo Zhao, Jinsuo Xiao, Jie Liu, Ye Cheng, He Wu, Yonggang Lv, Fuqing Ji
          </td>
          <td>2024-10-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The fragmentomics-based cell-free DNA (cfDNA) assays have recently demonstrated remarkable capabilities in identifying various cancers from non-conditional healthy controls. However, their accuracy in differentiating early-stage cancers from benign lesions, especially with inconclusive imaging results, remains unclear. In breast cancer, imaging-based screening methods often yield high false-positive rates, particularly in women with breast nodules, leading to unnecessary biopsies and increasing patient discomfort and healthcare burden. In this multi-center study, we enrolled 560 female participants and showed that cfDNA fragmentomics, using whole-genome sequencing, is a strong non-invasive biomarker for breast cancer. Among various cfDNA fragmentomics profiles, the fragment size ratio (FSR), fragment size distribution (FSD), and copy number variation (CNV) exhibited superior distinguishing ability compared to Griffin, motif breakpoint (MBP), and neomer features. Our cfDNA fragmentomics (cfFrag) model, which incorporates the three optimal fragmentomics features, successfully discriminated early-stage breast cancers from benign nodules, even at a low sequencing depth (3x). Notably, it demonstrated a specificity of 94.1% in asymptomatic healthy women at a 90% sensitivity for breast cancer. Furthermore, we highlight the clinical utility of the cfFrag model in predicting patient responses to neoadjuvant chemotherapy (NAC), and its integration with multimodal features, including radiological results and cfDNA methylation features, achieved high AUC values of 0.93-0.94 and 0.96, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec1ffa1e5f7be8782a9fb046c2b6f1ed5cb766" target='_blank'>
              Cell-free DNA Fragmentomics Assay to Discriminate the Malignancy of Breast Nodules and Evaluate Treatment Response
              </a>
            </td>
          <td>
            Jiaqi Liu, Yalun Li, W. Tang, Lijun Dai, Ziqi Jia, Heng Cao, Chenghao Li, Yuchen Liu, Yansong Huang, Jiang Wu, Dongxu Ma, Guangdong Qiao, H. Bao, Shuang Chang, Dongqin Zhu, Shanshan Yang, Xuxiaochen Wu, Xue Wu, Hengyi Xu, Hongyan Chen, Yang Shao, Xiang Wang, Zhihua Liu, Jianzhong Su
          </td>
          <td>2024-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA in eukaryotic genomes is under constant assault from both exogenous and endogenous sources, leading to DNA damage, which is considered a major molecular driver of ageing. Fortunately, the genome and the central exome are safeguarded against these attacks by abundant peripheral non-coding DNA. Non-coding DNA codes for small non-coding RNAs that inactivate foreign nucleic acids in the cytoplasm and physically blocks these attacks in the nucleus. Damage to non-coding DNA produced during such blockage is removed in the form of extrachromosomal circular DNA (eccDNA) through nucleic pore complexes. Consequently, non-coding DNA serves as a line of defence for the exome against DNA damage. The total amount of non-coding DNA/heterochromatin declines with age, resulting in a decrease in both physical blockage and eccDNA exclusion, and thus an increase in the accumulation of DNA damage in the nucleus during ageing and in age-related diseases. Here, we summarize recent evidence supporting a protective role of non-coding DNA in healthy and pathological states and argue that DNA damage is the proximate cause of ageing and age-related genetic diseases. Strategies aimed at strengthening the protective role of non-coding DNA/heterochromatin could potentially offer better systematic protection for the dynamic genome and the exome against diverse assaults, reduce the burden of DNA damage to the exome, and thus slow ageing, counteract age-related genetic diseases and promote a healthier life for individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7879deb4075aa031b3e715005da950caf3fffb2" target='_blank'>
              Protection of the genome and the central exome by peripheral non-coding DNA against DNA damage in health, ageing and age-related diseases.
              </a>
            </td>
          <td>
            Guo-Hua Qiu, Mingjun Fu, Xintian Zheng, Cuiqin Huang
          </td>
          <td>2024-09-26</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e2a897c78751fc35fbbb35887e00def7a3bd49" target='_blank'>
              One-carbon-mediated purine synthesis underlies temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Kimia Ghannad-Zadeh, Alyona Ivanova, M. Wu, Taylor M Wilson, Alyssa Lau, Robert Flick, David G. Munoz, Sunit Das
          </td>
          <td>2024-10-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653d7fee74f8332187f2be2503fada0f457b09c5" target='_blank'>
              STAG2 loss in Ewing sarcoma alters enhancer-promoter contacts dependent and independent of EWS::FLI1
              </a>
            </td>
          <td>
            D. Giménez-Llorente, Ana Cuadrado, María José Andreu, Inmaculada Sanclemente-Alamán, Maria Solé-Ferran, M. Rodríguez-Corsino, Ana Losada
          </td>
          <td>2024-11-01</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="We developed a linear amplification-mediated high-throughput genome-wide translocation sequencing method to profile Ig class-switch recombination (CSR) in human B cells in an unbiased and quantitative manner. This enables us to characterize CSR junctions resulting from either deletional recombination or inversion for each Ig class/subclass. Our data showed that more than 90% of CSR junctions detected in peripheral blood in healthy control subjects were due to deletional recombination. We further identified two major CSR junction signatures/patterns in human B cells. Signature 1 consists of recombination junctions resulting from both IgG and IgA switching, with a dominance of Sµ-Sγ junctions (72%) and deletional recombination (87%). Signature 2 is contributed mainly by Sµ-Sα junctions (96%), and these junctions were almost all due to deletional recombination (99%) and were characterized by longer microhomologies. CSR junctions identified in healthy individuals can be assigned to both signatures but with a dominance of signature 1, whereas almost all CSR junctions found in patients with defects in DNA-PKcs or Artemis, two classical nonhomologous end joining (c-NHEJ) factors, align with signature 2. Thus, signature 1 may represent c-NHEJ activity during CSR, whereas signature 2 is associated with microhomology-mediated alternative end joining in the absence of the studied c-NHEJ factors. Our findings suggest that in human B cells, the efficiency of the c-NHEJ machinery and the features of switch regions are crucial for the regulation of CSR orientation. Finally, our high-throughput method can also be applied to study the mechanism of rare types of recombination, such as switching to IgD and locus suicide switching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76f6ad58cd1fa3e33c899adc7067ec2a694fd440" target='_blank'>
              Orientation Regulation of Class-switch Recombination in Human B Cells.
              </a>
            </td>
          <td>
            Likun Du, Valentyn Oksenych, Hui Wan, Xiaofei Ye, Junchao Dong, Adam Yongxin Ye, Hassan Abolhassani, S. Vlachiotis, Xuefei Zhang, K. de la Rosa, L. Hammarström, M. van der Burg, Frederick W. Alt, Q. Pan-Hammarström
          </td>
          <td>2024-09-09</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f851c6a2c2eff349bc11c1d2c38a41832c7746f" target='_blank'>
              The development of the occurrence and metastasis of breast cancer by single-cell sequencing
              </a>
            </td>
          <td>
            Man Chen, Mengya Feng, Hai Lei, Dan Mo, Shengnan Ren, Dechun Yang
          </td>
          <td>2024-10-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281649fc3a01628d835005fce4ae40aa70c5e3f2" target='_blank'>
              Oncometabolites at the crossroads of genetic, epigenetic and ecological alterations in cancer.
              </a>
            </td>
          <td>
            Letizia Lanzetti
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alterations in deoxyribonucleoside triphosphate (dNTP) pools have been linked to increased mutation rates and genome instability in unicellular organisms and cell cultures. However, the role of dNTP pool changes in tumor development in mammals remains unclear. In this study, we present a mouse model with a point mutation at the allosteric specificity site of ribonucleotide reductase, RRM1-Y285A. This mutation reduced ribonucleotide reductase activity, impairing the synthesis of deoxyadenosine triphosphate (dATP) and deoxyguanosine triphosphate (dGTP). Heterozygous Rrm1+/Y285A mice exhibited distinct alterations in dNTP pools across various organs, shorter lifespans and earlier tumor onset compared with wild-type controls. Mutational spectrum analysis of tumors revealed two distinct signatures, one resembling a signature extracted from a human cancer harboring a mutation of the same amino acid residue in ribonucleotide reductase, RRM1Y285C. Our findings suggest that mutations in enzymes involved in dNTP metabolism can serve as drivers of cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe3b3a3839a433e41d5ae4062ca8fa74fde87a0" target='_blank'>
              Altered dNTP pools accelerate tumor formation in mice.
              </a>
            </td>
          <td>
            Phong Tran, Pradeep Mishra, Leonard G. Williams, Roman Moskalenko, Sushma Sharma, A. K. Nilsson, Danielle L. Watt, Pernilla Andersson, Anders Bergh, Z. Pursell, A. Chabes
          </td>
          <td>2024-10-03</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a101d4e936af34d1b466e5552558a397fc61b" target='_blank'>
              Carcinogenic Mechanisms of Hexavalent Chromium: From DNA Breaks to Chromosome Instability and Neoplastic Transformation.
              </a>
            </td>
          <td>
            Idoia Meaza, Aggie R Williams, Sandra S Wise, Haiyan Lu, John W Pierce
          </td>
          <td>2024-10-28</td>
          <td>Current environmental health reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c48dd8dbdc624fab3a017de972a086b160ea4b3" target='_blank'>
              The lncRNA AFAP1-AS1 is upregulated in metastatic triple-negative breast tumors and controls hypoxia-activated vasculogenic mimicry and angiogenesis
              </a>
            </td>
          <td>
            A. P. García-Hernández, David Núñez Corona, Ángeles Carlos-Reyes, Mónica Sierra-Martínez, Gustavo Acosta-Altamirano, M. Cisneros-Villanueva, Yussel Pérez-Navarro, Eloisa Ibarra-Sierra, L. Marchat, C. López-Camarillo
          </td>
          <td>2024-10-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b57b76c24e922e56ff6f3fa5ae5de6f644999f2" target='_blank'>
              Human papillomavirus-encoded circular RNA circE7 promotes immune evasion in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Junshang Ge, Yi Meng, Jiayue Guo, Pan Chen, Jie Wang, Lei Shi, Dan Wang, Hongke Qu, Pan Wu, Chunmei Fan, Shanshan Zhang, Q. Liao, Ming Zhou, Bo Xiang, Fuyan Wang, Ming-Liang Tan, Zhaojian Gong, Wei Xiong, Zhaoyang Zeng
          </td>
          <td>2024-10-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Endometrial carcinoma (EC) is a prevalent gynecological malignancy that poses a significant threat to women’s health worldwide. However, its pathogenesis and underlying mechanisms remains unclear. In this study, expression quantitative trait loci data, Mendelian randomization analysis, and differential expression analysis were performed to identify potential targets. A prognostic risk signature was subsequently constructed for EC patients based on the expression of these genes. Four bioinformatics algorithms, including generalized linear model, extreme gradient boosting, support vector machine, and random forest, were used to identify hub genes in EC. The expression of ring finger protein 144A (RNF144A) was validated using quantitative real-time polymerase chain reaction. Cellular proliferation and migration ability were evaluated using CCK-8 and Transwell assays, respectively. The genes RNF144A, ketohexokinase, and Rab interacting lysosomal protein like 2 were identified as potential targets for EC. Their differential expression was observed in EC patients, and Mendelian randomization analysis revealed a negative correlation between these genes and the development of EC. Mechanistic analyses suggested a strong association between these genes and the tumor immune microenvironment. The constructed risk signature was significantly associated with the prognosis, age, cancer stage, and grade of EC patients. Furthermore, based on interacted model algorithms, RNF144A was identified as a hub gene in EC. It was found to be significantly downregulated in EC samples, and its expression was positively correlated with the stage and grade of EC patients. In vitro experiments showed that overexpression of RNF144A significantly promoted cell growth and migration in EC cells. In conclusion, this study provides insights into the molecular mechanisms underlying EC progression and identifies preliminary candidate biomarkers for the development of EC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30ccab2586145d3cccf21eccbbc93cc5b295b360" target='_blank'>
              RNF144A as a potential risk gene for endometrial carcinoma: Insights from Mendelian randomization, bulk RNA sequencing, single-cell RNA, and experimental analysis
              </a>
            </td>
          <td>
            Xiqin Ruan, Ni Wang, Qingwen Xie, Yi Du
          </td>
          <td>2024-10-04</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842eba76f2be7eab5006f9657126f976601f2a" target='_blank'>
              GUCA2A dysregulation as a promising biomarker for accurate diagnosis and prognosis of colorectal cancer
              </a>
            </td>
          <td>
            Pooya Jalali, Shahram Aliyari, Marziyeh Etesami, Mahsa Saeedi Niasar, Sahar Taher, K. Kavousi, E. Nazemalhosseini Mojarad, Zahra Salehi
          </td>
          <td>2024-11-01</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
N6-methyladenosine (m6A) modification plays a crucial role in the progression of various cancers, including pancreatic cancer, by regulating gene expression. However, the specific mechanisms by which m6A affects pancreatic cancer metastasis remain unclear. This study aims to elucidate the role of METTL16, an m6A writer gene, in regulating core genes such as CAPN2 and MROH8, influencing tumor growth and metastasis.


MATERIALS AND METHODS
Transcriptomic data from pancreatic cancer patients in The Cancer Genome Atlas (TCGA) were analyzed to identify m6A-related genes. We performed correlation and survival analyses to uncover core genes influenced by m6A expression. Functional assays, including METTL16 knockdown and overexpression experiments, were conducted in pancreatic cancer cell lines, patient-derived organoids, and animal models. Immunofluorescence, co-immunoprecipitation (Co-IP), and m6A-specific quantitative PCR were used to validate protein interactions and m6A modifications. Chromatin immunoprecipitation (ChIP) analysis was utilized to investigate transcription factor binding at gene promoter regions.


RESULTS
METTL16 and METTL3 were identified as key m6A regulators associated with improved prognosis in pancreatic cancer patients (P<0.05). CAPN2, CHMP2B, ITGA3, ITGA6, ITPR1, and RAC1 were identified as core genes linked to m6A expression, all significantly correlated with patient prognosis (P<0.05). METTL16 overexpression significantly inhibited tumor growth and metastasis (P<0.001) by downregulating CAPN2 through an indirect mechanism involving the transcription factor TBP and the gene MROH8. MROH8 negatively regulated CAPN2 by promoting TBP degradation, with METTL16 enhancing MROH8 mRNA stability through m6A modifications (P<0.01). Functional assays demonstrated that METTL16 and YTHDC2 (an m6A reader) collaboratively enhanced MROH8 mRNA stability, thereby inhibiting CAPN2 expression and reducing tumor proliferation and metastasis (P<0.001).


CONCLUSION
This study reveals a novel regulatory axis involving METTL16, MROH8, and TBP that modulates CAPN2 expression, contributing to the suppression of pancreatic cancer progression. The METTL16-MROH8-TBP-CAPN2 pathway offers potential therapeutic targets for pancreatic cancer treatment, highlighting the significance of m6A modifications in tumor regulation. Further clinical validation is needed to confirm these findings in human patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600e2f3854319a590eb18af4a1487bf0d3bf1afb" target='_blank'>
              METTL16 inhibits pancreatic cancer proliferation and metastasis by promoting MROH8 RNA stability and inhibiting CAPN2 expression.
              </a>
            </td>
          <td>
            Tingzhuang Yi, Chunming Wang, Xia Ye, Jie Lin, Chen Lin, Fengzhen Qin, Wanlin Yang, Yulu Ye, Dengchong Ning, Jinyan Lan, Huafu Li, Chunying Luo, Jian Ma, Zhongheng Wei
          </td>
          <td>2024-10-22</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="The incidence of colorectal cancer (CRC) among individuals under age 50, or early-onset CRC (EOCRC), has been rising over the past few decades for unclear reasons, and the etiology of the disease remains largely unknown. Known genetic risk factors do not explain this increase, pointing to possible environmental and as-yet unidentified genetic contributors and their interactions. Previous research linked genetic variation on chromosome 6 to increased CRC risk. This region harbors multiple immune genes, including the gene encoding Major Histocompatibility Complex (MHC) class I polypeptide-related sequence A (MICA). MICA is a polygenic ligand for the Natural Killer Group 2D receptor (NKG2D), a receptor expressed on Natural Killer (NK) cells and other lymphocytes. Given that intra-tumoral NK cell infiltration correlates with favorable CRC outcomes, we hypothesized that germline genetic variation in MICA could influence CRC risk. In a discovery set of 40,125 cases and controls, we show that the minor G allele at Chr6:31373718C>G (hg19) is associated with increased risk for CRC (odds ratio (OR) = 1.09, 95% confidence interval (CI) 1.04 - 1.15, p = 0.0009). The effect is stronger in EOCRC (OR = 1.26, 95% CI 1.08 - 1.44, p = 0.0023) than in those 50 and over (OR = 1.07, 95% CI 1.02 - 1.13; p = 0.012) (Ratio of ORs = 1.32, 95% CI 1.14 - 1.52, p = 0.0002). In an independent validation set of 77,983 cases and controls, the adjusted interaction by age -of-onset was significant at OR = 1.15 (95% CI 1.03 - 1.34, p = 0.0150) with a higher risk in EOCRC. Expression quantitative trait locus analysis in normal colonic epithelia showed that MICA RNA expression decreases linearly with each additional copy of the minor G allele (p = 3.345 x 10-18). Bulk RNA analysis of the tumor immune microenvironment revealed that tumors from patients with CG or GG genotypes have lower resting and activated NK cell infiltration as compared to tumors from patients with CC genotype. Multiplex immunofluorescence analysis demonstrated that patients with a G allele (i.e. CG or GG genotype, but not CC genotype) have a statistically significant decrease in the number of NK cells in tumor compared to adjacent normal colonic mucosa. Taken together, population-based epidemiologic, molecular, genetic, cellular and immunologic evidence demonstrate that MICA genotype is associated with increased risk of EOCRC and reduced number of NK cells in colorectal tumors, suggesting that patients with a G allele have altered NK cell-mediated immunosurveillance. These novel findings suggest that EOCRC may have a previously unrecognized innate immune-mediated etiology which merits further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5c740e57eda493d70527270b12cf7926d01624" target='_blank'>
              Genetic Variation and Regulation of MICA Alters Natural Killer Cell-Mediated Immunosurveillance in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            H. McGee, J. Bonner, Colt Egelston, Yubo Fu, Oscar Colunga Flores, S. Lindsey, Lawrence Shaktah, Ferran Moratalla-Navarro, Yasmin Kamal, Kevin K. Tsang, C. P. Walker, G. Idos, K. McDonnell, H. Rennert, Elizabeth L Barry, Hermann Brenner, Daniel D. Buchanan, P. Campbell, A. T. Chan, J. Chang-Claude, Jane C Figueiredo, M. Gago-Domínguez, M. Hoffmeister, Li Hsu, J. Huyghe, M. Jenkins, L. Le Marchand, Heinz-Josef Lenz, Li Li, A. Lindblom, Yun Ru Liu Ruby, B. Lynch, Christina C. Newton, K. Offit, Shuji Ogino, Rebeca Sanz Pamplona, Andrew J. Pellatt, Paul D. P. Pharoah, Amanda I. Phipps, Lorena Reynaga, Allyson S. Templeton, C. Um, Alicja Wolk, M. Woods, A. H. Wu, Yen Yun, Weishan Zheng, Terence M. Williams, David V Conti, Ulrike Peters, F. Lejbkowicz, J. Greenson, S. Schmit, W. Gauderman, Stanley R. Hamilton, Victor Moreno, Gadi Rennert, Stephen B. Gruber
          </td>
          <td>2024-09-26</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="High Grade Serous Ovarian Cancer (HG-SOC), the most prevalent and aggressive gynecological malignancy, is marked by ubiquitous loss of functional p53, largely due to point mutations that arise very early in carcinogenesis. These mutations often lead to p53 protein misfolding and subsequent aggregation, yet the alterations in intracellular p53 dynamics throughout ovarian cancer progression remain poorly understood. HG-SOC originates from the fallopian tube epithelium, with a well-documented stepwise progression beginning with early pre-malignant p53 signatures. These signatures represent largely normal cells that express and accumulate mutant p53, which then transform into benign serous tubal intraepithelial lesions (STIL), progress into late pre-malignant serous tubal intraepithelial carcinoma (STIC), and ultimately lead to HGSOC. Here, we show that the transition from folded, soluble to aggregated mutant p53 occurs during the malignant transformation of benign precursor lesions into HGSOC. We analyzed fallopian tube tissue collected from ten salpingo-oophorectomy cases and determined the proportion of cells carrying soluble versus mis-folded/mutant p53 through conformation-sensitive staining and quantification. Misfolded p53 protein, prone to aggregation, is present in STICs and HG-SOCs, but notably absent from preneoplastic lesions and surrounding healthy tissue. Overall, our results indicate that aggregation of mutant p53 is a structural defect that distinguishes preneoplastic early lesions from late premalignant and malignant ones, offering a potential treatment window for targeting p53 aggregation and halting ovarian cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e004df7ee189f194b9dc43a60b80d81862858bf" target='_blank'>
              Mutant p53 Misfolding and Aggregation Precedes Transformation into High-Grade Serous Ovarian Carcinoma
              </a>
            </td>
          <td>
            S. Sartini, Lexi Omholt, Neda A Moatamed, A. Soragni
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c899c38678aa8e86ec7ec4d5de561bdad4078c7d" target='_blank'>
              CDK4/6 inhibition initiates cell cycle arrest by nuclear translocation of RB and induces a multistep molecular response
              </a>
            </td>
          <td>
            Ting Hong, Anna C Hogger, Dongbiao Wang, Qi Pan, Julie Gansel, T. Engleitner, R. Öllinger, Jürgen E Gschwend, R. Rad, R. Nawroth
          </td>
          <td>2024-10-26</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d829dac90ae6e0bee2354e14ecf9cb6f0b45ddf8" target='_blank'>
              Polyclonal-to-monoclonal transition in colorectal precancerous evolution.
              </a>
            </td>
          <td>
            Zhaolian Lu, Shanlan Mo, Duo Xie, Xiangwei Zhai, Shanjun Deng, Kantian Zhou, Kun Wang, Xueling Kang, Hao Zhang, Juanzhen Tong, Liangzhen Hou, Huijuan Hu, Xuefei Li, Da Zhou, L. T. Lee, Li Liu, Yaxi Zhu, Jing Yu, Ping Lan, Jiguang Wang, Zhen He, Xionglei He, Zheng Hu
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="
 Forkhead box A1 (FOXA1) acts as a pioneer factor for Estrogen Receptor alpha (ERα), playing a crucial role in determining where ERα binds to DNA and regulates gene expression in breast cancer cells. A pioneer factor is a class of transcription factors (TF) that is responsible for opening the chromatin to allow other TFs to attach and activate transcription of that gene. FOXA1 acts as a facilitator for ERα-DNA interactions. It binds to specific DNA regions, opening the chromatin for ERα to bind. It “pre-marks” these sites in the genome even in the absence of estrogen. In fact, FOXA1 is frequently overexpressed in ER+ breast cancers, suggesting it has a crucial role in driving tumor growth. As such, FOXA1 is an attractive therapeutic target that already serves as a great biomarker for many types of breast cancers. Inhibiting FOXA1 can potentially block ER activity and inhibit growth of ER+ breast cancer, especially in those resistant to anti-estrogen therapies, such as tamoxifen. In this project, we identified sites that FOXA1 opens and correlated them with SNP mutations commonly found in the Hispanic population. The presence of these SNPs portends a worse outcome in Hispanic patients. To assess which open chromatin regions are crucial for breast cancer pathogenesis, a ChIP-seq analysis was done on vehicle-treated and estrogen-treated samples to determine FOXA1 binding sites. After identifying gained, lost, and differentially expressed peaks set, these were compared to genome-wide association studies (GWAS) of Hispanic patients to develop an understanding of how much overlap is common between sites of mutations and open sections of chromatin due to FOXA1. Through our analysis, we uncovered a subset of FOXA1 binding sites that overlap with sites of inherited mutation in Hispanic patients. This informs on the subset of genes that are linked to breast cancer and potential therapeutic targets.
 Citation Format: Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil. FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e221203c9b5991c2cb9afa79cdfe37b8baf68cb3" target='_blank'>
              Abstract B011: FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations
              </a>
            </td>
          <td>
            Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Gliomas, the most prevalent type of primary brain tumor, stand out as one of the most aggressive and lethal types of human cancer. Methods & Results: To uncover potential prognostic markers, we employed the weighted correlation network analysis (WGCNA) on the Chinese Glioma Genome Atlas (CGGA) 693 dataset to reveal four modules significantly associated with glioma clinical traits, primarily involved in immune function, cell cycle regulation, and ribosome biogenesis. Using the least absolute shrinkage and selection operator (LASSO) regression algorithm, we identified 11 key genes and developed a prognostic risk score model, which exhibits precise prognostic prediction in the CGGA 325 dataset. More importantly, we also validated the model in 12 glioma patients with overall survival (OS) ranging from 4 to 132 months using mRNA sequencing and immunohistochemical analysis. The analysis of immune infiltration revealed that patients with high-risk scores exhibit a heightened immune infiltration, particularly immune suppression cells, along with increased expression of immune checkpoints. Furthermore, we explored potentially effective drugs targeting 11 key genes for gliomas using the library of integrated network-based cellular signatures (LINCS) L1000 database, identifying that in vitro, both torin-1 and clofarabine exhibit promising anti-glioma activity and inhibitory effect on the cell cycle, a significant pathway enriched in the identified glioma modules. Conclusions: In conclusion, our study provides valuable insights into molecular mechanisms and identifying potential therapeutic targets for gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d22cc28dbea3a944dde294d9243d2a693f09d67" target='_blank'>
              Identification of Key Immune and Cell Cycle Modules and Prognostic Genes for Glioma Patients through Transcriptome Analysis
              </a>
            </td>
          <td>
            Kaimin Guo, Jinna Yang, Ruonan Jiang, Xiaxia Ren, Peng Liu, Wenjia Wang, Shuiping Zhou, Xiaoguang Wang, Li Ma, Yunhui Hu
          </td>
          <td>2024-09-28</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c75249f54c500f783739b3eb3c7b22b00e436d" target='_blank'>
              Azacytidine treatment affects the methylation pattern of genomic and cell-free DNA in uveal melanoma cell lines
              </a>
            </td>
          <td>
            Sarah Tadhg Ferrier, Mingyang Li, J. Burnier
          </td>
          <td>2024-10-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Immunotherapies such as immune checkpoint inhibitors (ICIs) have greatly improved patient outcomes in multiple tumor types, but ICIs have failed to achieve responses in many patients. Meta-analyses of patient cohorts treated with ICIs have observed that worse clinical outcomes are strongly correlated with high levels of intratumoral genetic heterogeneity (ITH). ITH occurs when tumor cells mutate as they divide, typically due to oxidative stress or DNA damage from rapid proliferation and continued carcinogenic insults, generating tumor subclones. However, the mechanisms by which ITH impairs the anti-tumor immune response are not fully understood. We propose a novel mechanism for how ITH in highly T cell-infiltrated inflamed tumors can adversely affect T cell behavior. We hypothesize that strongly pro-inflammatory tumor subclones secreting robust pro-inflammatory chemokine profiles can dominate T cell trafficking within the tumor, resulting in a weakened immune response against other tumor subclones. Using a murine squamous cell skin carcinoma (SCC) model, we have demonstrated that in immune-inflamed tumors, the spatial distribution and behavior of tumor-infiltrating T cells is influenced by ITH. A strongly pro-inflammatory tumor line expressing an endogenous RFP fluorophore was mixed with a weakly pro-inflammatory tumor line expressing an endogenous YFP fluorophore at a 1:1 ratio. The resulting tumors contained distinct RFP+ and YFP+ regions. These were analyzed by immunofluorescence (IF) and by manually dissecting the tumors and analyzing each region by flow cytometry. Flow cytometry analysis determined that the strongly pro-inflammatory RFP+ regions contained significantly more total T cells and CD4+ T cells than YFP+ regions. Furthermore, FoxP3- “conventional” CD4+ T cells were significantly more likely to be “terminal effector-like” CD44+CD62L- in RFP+ regions, and significantly more likely to be “naïve-like” CD62L+CD44- in YFP+ regions. IF analysis confirmed the preferential distribution of CD4+ T cells in RFP+ regions, and observed that this spatial pattern was evident as early as 11 days after subcutaneous injection. Notably, this localization trend did not occur with CD8+ T cells, which were evenly dispersed throughout the tumors. ELISA analysis of lysates of both tumor cell lines determined that the pro-inflammatory RFP+ cell line secretes significantly higher levels of the chemokine CCL5, which has been observed by other studies to selectively mediate CD4+ T cell chemotaxis. These results demonstrate that intratumoral CD4+ trafficking is shaped by the spatial distribution of specific tumor subclones, and suggest tumor cell chemokine signaling may directly mediate T cell spatial localization and behavior within tumors.
 Citation Format: Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Reeves. Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeeff4ff2175551d0a146978acdaeb3ec633ea2b" target='_blank'>
              Abstract A033: Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors
              </a>
            </td>
          <td>
            Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Q Reeves
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic instability is a driver and accelerator of tumorigenesis and influences disease outcomes across cancer types. Although genomic instability has been associated with immune evasion and worsened disease prognosis, emerging evidence shows that genomic instability instigates pro-inflammatory signaling and enhances the immunogenicity of tumor cells, making them more susceptible to immune recognition. While this paradoxical role of genomic instability in cancer is complex and likely context-dependent, understanding it is essential for improving the success rates of cancer immunotherapy. In this review, we provide an overview of the underlying mechanisms that link genomic instability to pro-inflammatory signaling and increased immune surveillance in the context of cancer, as well as discuss how genomically unstable tumors evade the immune system. A better understanding of the molecular crosstalk between genomic instability, inflammatory signaling, and immune surveillance could guide the exploitation of immunotherapeutic vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab73a1200d75ccbafd1672ebbaf3535106c1200" target='_blank'>
              Genomic instability as a driver and suppressor of anti-tumor immunity
              </a>
            </td>
          <td>
            M. Requesens, F. Foijer, Hans W Nijman, M. de Bruyn
          </td>
          <td>2024-10-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is a dynamic nexus where cancer cell metabolism and the immune system intricately converge, with nucleotide metabolism (NM) playing a pivotal role. This review explores the critical function of NM in cancer cell proliferation and its profound influence on the TME and immune landscape. NM is essential for DNA and RNA synthesis and is markedly upregulated in cancer cells to meet the demands of rapid growth. This metabolic rewiring fuels cancer progression, but also shapes the TME, impacting the function and viability of immune cells. The altered nucleotide milieu in the TME can suppress immune response, aiding cancer cell evasion from immune surveillance. Drug discoveries in the field of NM have revealed different therapeutic strategies, including inhibitors of nucleotide synthesis and drugs targeting salvage pathways, which are discussed thoroughly in this review. Furthermore, the emerging strategy of combining NM-targeted therapies with immunotherapies is emphasised, particularly their effect on sensitising tumours to immune checkpoint inhibitors and enhancing overall treatment efficacy. The Human Genome Project paved the way for personalised medicine, countering the established 'one size fits all' approach to cancer treatment. Advances in understanding the TME and NM have spurred interest in personalised therapeutic strategies. This review highlights the potential of leveraging individual tumour metabolic profiles to guide treatment selection, aiming to optimise efficacy and minimise adverse effects. The strategic importance of targeting NM in cancer therapy and its synergistic potential with immunotherapies offers a path towards more effective and personalised cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bc17614b60cc3e87bf59862d1663d401029e4eb" target='_blank'>
              Harnessing nucleotide metabolism and immunity in cancer: a tumour microenvironment perspective.
              </a>
            </td>
          <td>
            Hadil Sulieman, Alexandra J Emerson, Peter M. Wilson, Karl A Mulligan, R. Ladner, Melissa J LaBonte
          </td>
          <td>2024-09-22</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187812e12825019ab421661fdd39125bff16e32a" target='_blank'>
              Potential role of heteroplasmic mitochondrial DNA mutations in modulating the subtype-specific adaptation of oral squamous cell carcinoma to cisplatin therapy
              </a>
            </td>
          <td>
            Amnani Aminuddin, Pei Yuen Ng, Chee Onn Leong, S. Makpol, E. Chua
          </td>
          <td>2024-10-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="INTRODUCTION
In chromatin nucleosomes, the presence-instead of canonical histone H3-of its variant, CENH3 (in plants), is considered the most reliable marker of the location of centromeres. In this study, we investigated effects of distant hybridization and maternal cytoplasm on centromere size in alloploid hybrids between wheat and rye as compared to their parental forms.


METHODS
Centromere sizes were measured using 2D images of CENH3 fluorescent signals on interphase nuclei obtained from parental forms and a triticale hybrid (genomic formula AABBBRR) in which the maternal form is wheat and secalotriticum hybrids (genomic formula RRAABBB) in which the maternal form is rye. For measurements, we selected the largest spherical nuclei with large nucleoli in the late G2 phase, in which most of the loading of CENH3 into centromeric chromatin takes place.


RESULTS
When processing the results of measurement of centromere sizes in the hybrids, the obtained values were compared with those expected for the case of no change in centromere sizes in any of the parental sets of chromosomes. We found no significant differences between expected and measured values.


CONCLUSION
We believe that in the case of allopolyploid hybrids between wheat and rye, centromeres of chromosomes from the parental species retain the sizes formed during evolution. This conservatism may be promoted by the high similarity in the structure of the CENH3 molecules between these species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13c11924647b4d33fd28b8eee261c47cb717952" target='_blank'>
              The parental centromere sizes remain unaltered in allopolyploid wheat-rye hybrids.
              </a>
            </td>
          <td>
            E. V. Evtushenko, S. S. Gatzkaya, Petr I Stepochkin, A. Vershinin
          </td>
          <td>2024-10-01</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Esophageal cancer (EC) is a major health issue, ranking seventh in incidence and sixth in mortality worldwide. Despite advancements in multidisciplinary treatment approaches, the 5-year survival rate for EC remains low at 21%. Challenges in EC treatment arise from late-stage diagnosis, high malignancy, and poor prognosis. Understanding the tumor microenvironment is critical, as it includes various cellular and extracellular components that influence tumor behavior and treatment response. Mast cells (MCs), as tissue-resident immune cells, play dual roles in tumor dynamics. High-throughput single-cell RNA sequencing offers a powerful tool for analyzing tumor heterogeneity and immune interactions, although its application in EC is limited. Methods In this study, we investigated the immune microenvironment of EC using single-cell RNA sequencing and established a comprehensive immune profile. We also performed analysis of upstream transcription factors and downstream pathway enrichment to further comprehensively decipher MCs in EC. Besides, we performed knockdown experiments to explore the role of epidermal growth factor receptor (EGFR) signaling pathway in MCs-tumor cell interactions, highlighting its potential as a prognostic marker. Finally, we constructed a prognostic model for EC, which provided valuable suggestions for the diagnosis and prognosis of EC. Results Our analysis identified 11 major cell types, of which MCs were particularly present in pericarcinoma tissues. Further grouping of the 5,001 MCs identified 8 distinct subtypes, including SRSF7-highly expressed MCs, which showed strong tumor preference and potential tumor-promoting properties. Moreover, we identified the key signaling receptor EGFR and validated it by in vitro knockdown experiments, demonstrating its cancer-promoting effects. In addition, we established an independent prognostic indicator, SRSF7+ MCs risk score (SMRS), which showed a correlation between high SMRS group and poor prognosis. Conclusion These findings illuminate the complex interactions within the tumor microenvironment of EC and suggest that targeting specific MCs subtypes, particularly via the EGFR signaling pathway, may present novel therapeutic strategies. This study establishes a comprehensive immune map of EC, offering insights for improved treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b52619ad1abc5e3af051c9a613c06b47617866c" target='_blank'>
              Unraveling the ecological landscape of mast cells in esophageal cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Shengyi Zhang, Xinyi Zhang, Zhikai Xiahou, Shunqing Zuo, Jialong Xue, Yi Zhang
          </td>
          <td>2024-10-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3aec5d0096d9dcb8611372282f9781c46d3671f" target='_blank'>
              Comprehensive pan-cancer analysis reveals ENC1 as a promising prognostic biomarker for tumor microenvironment and therapeutic responses
              </a>
            </td>
          <td>
            Zhenyu Cao, Jinfeng Zhu, Zicheng Wang, Yuhuai Peng, Liyun Zeng
          </td>
          <td>2024-10-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although the concept of cancer stem cells is still controversial, previous studies have shown that blood cancers, as well as specific types of solid cancers such as colorectal cancer, rely on stem cells during the onset of tumor growth and further tumor development. Moreover, resistance to therapeutic treatment in leukemias such as acute myeloid leukemia and in colorectal cancer can be attributed to a small population of cells with stemness properties known as minimal residual disease. In this review, we look back on the discovery of cancer stem cells and the contribution of the findings in blood cancer to a parallel discovery in solid cancers. We focus on CD44 as a stem cell marker, both in blood cancers and in several types of solid cancers, particularly of the gastro-intestinal tract. This review highlights newly discovered molecular mechanisms of action of CD44 which indicate that CD44 has indeed a function in stemness, stem cell maintenance and drug resistance. We attempt here to make the link between the functions of CD44 isoforms in stemness and their involvement in specific steps of tumor growth and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8580e7867b570560c73223a09cea13edca2587fc" target='_blank'>
              CD44: A Stemness Driver, Regulator and Marker - all in one?
              </a>
            </td>
          <td>
            S. J. Sonnentag, Nagwa S M Ibrahim, V. Orian-Rousseau
          </td>
          <td>2024-10-04</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Radiotherapy remains a cornerstone in cancer treatment, leveraging ionizing radiation to eradicate malignant cells. Its efficacy, however, is frequently challenged by the heterogeneous sensitivity of tumors and surrounding tissues to radiation. Therefore, understanding the molecular mechanisms underlying radiosensitivity is crucial for improving treatment outcomes. Among the myriad of molecular players involved, the tumor suppressor protein p53 stands out as a central regulator with significant implications for radiosensitivity. Known as the “guardian of the genome”, p53 plays a pivotal role in maintaining genomic stability and orchestrating cellular responses such as cell cycle arrest, DNA repair, apoptosis, and senescence in response to various stress signals, including radiation-induced DNA damage. Activation of p53 triggers the transcription of target genes involved in DNA repair pathways, such as p21, MDM2, and GADD45, facilitating the repair of radiation-induced DNA damage or the elimination of irreparably damaged cells. This, in turn, influences the overall radiosensitivity of tissues. Mutations in the TP53 gene, which encodes p53, are among the most frequent genetic alterations in human cancers. Loss or dysfunction of p53 can compromise the cellular response to radiation, leading to increased resistance to therapy and poorer clinical outcomes. Conversely, intact p53 function is associated with enhanced radiosensitivity due to its ability to promote cell cycle arrest and apoptosis in response to radiation-induced DNA damage. In conclusion, elucidating the molecular mechanisms by which p53 influences radiosensitivity is essential for advancing our understanding of the radiation response in cancer cells and developing more effective therapeutic approaches to cancer treatment. This review provides a comprehensive overview of the multifaceted role of p53 in modulating cellular responses to radiation, emphasizing its influence on radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edeb6b0914ac30942be5b714c5e52dd1ba564b4d" target='_blank'>
              Exploring the Role of p53 in Radiosensitivity: A Key Player in Cancer Therapy
              </a>
            </td>
          <td>
            Tusher- Al-Arafat, Aihong Mao, T. Katsube, Bing Wang
          </td>
          <td>2024-10-24</td>
          <td>Radiation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Genome-wide association studies (GWAS) have identified independent signals at 5p15.33 across numerous cancers, with protective alleles for one cancer often conferring risk for another. Many of these associations are thought to act via allele-specific alterations in the cis- regulation of target genes. Transcriptomic analyses in multiple tissue types implicate two plausible target genes: TERT and CLPTM1L. However, the mechanisms linking cancer susceptibility alleles to their presumed target genes, along with the observed antagonistic pleiotropy between different cancers remain elusive. With this in mind, we employed a comprehensive approach, integrating statistical fine-mapping of GWAS summary statistics with massively parallel reporter assays (MPRA) and a densely tiled CRISPRi screen performed across eight cell lines, representative of four distinct cancers associated with 5p15.33 alleles – pancreatic, melanoma, lung, and bladder cancer. Integrative screening identified up to 30 variants across multiple signals in the locus with putative allele-specific transcriptional activity. Furthermore, a variable number tandem repeat (VNTR) in intron 9 of CLPTM1L emerged as a potent transcriptional enhancer in CRISPRi screening. Leveraging PacBio sequencing, VNTR alleles of European ancestry samples from the 1000 Genomes Project were genotyped and found to be highly polymorphic, with longer VNTRs linked to the haplotype tagged by rs31490-A (risk increasing for pancreatic cancer). Subsequent imputation into a pancreatic cancer GWAS of 9,040 cases and 12,496 controls revealed an association between longer VNTRs and risk of pancreatic cancer (OR for VNTR in the 75th percentile = 1.17, 95% CI = 1.13-1.22, P = 2.21x10-12). The significance of the association for the lead SNP rs31490 after conditioning on the VNTR genotype dropped to P= 2.20x10-5, suggesting that VNTR alleles merit consideration as part of the credible variant set for pancreatic cancer. Targeted CRISPRi assays suggested VNTR-mediated cis-regulation of both TERT and CLPTM1L in PANC-1 and MIA PaCa-2 pancreatic cancer cell lines, while dual luciferase reporter assays in the same cell lines revealed that VNTR enhancer activity is mediated by Hippo pathway transcription factors. In ongoing experiments, we aim to expand on luciferase reporter assays and DNA-protein binding assays in relevant cell lines with the aim of not only deciphering how longer VNTRs differentially exert their transcriptional effects compared to shorter alleles but also characterizing the divergent directions of effect between different cancer types. These findings underscore the proposition that pancreatic cancer susceptibility at the 5p15.33 locus may be mediated by both SNPs and an uncharted class of variants, yet to be characterized in the post-GWAS landscape.
 Citation Format: Aidan O'Brien, Hyunkyung Kong, Minal Patel, Katelyn E Connelly, Mai Xu, Irene Collins, Jun Zhong, Jason Hoskins, Michelle Ho, Brenen Papenburg, Mark M Iles, Matthew H Law, Maria-Teresa Landi, Rachael Stolzenber- Solomon, Brian M Wolpin, Alison P Klein, Nick Orr, Stephen J Chanock, Sara Lindström, Marc-Henri Stern, Ludmila Prokunina-Olsson, Jieyon Choi, Kevin M Brown, Laufey T Amundadottir. Unraveling pancreatic cancer susceptibility at 5p15.33: Functional characterization of a novel VNTR element [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0777eadfb43a66af250607da2ed11cfe90c7930b" target='_blank'>
              Abstract C055: Unraveling pancreatic cancer susceptibility at 5p15.33: Functional characterization of a novel VNTR element
              </a>
            </td>
          <td>
            Aidan O'Brien, Hyunkyung Kong, Minal Patel, Katelyn E. Connelly, Mai Xu, I. Collins, Jun Zhong, Jason W. Hoskins, Michelle Ho, Brenen Papenburg, M. Iles, M. Law, Marion Landi, Rachael Stolzenber- Solomon, B. Wolpin, Alison P Klein, N. Orr, S. Chanock, Sara Lindström, Marc-Henri Stern, Ludmila Prokunina-Olsson, Jieyon Choi, Kevin M Brown, L. Amundadottir
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a liver cancer of adolescents and young adults characterized by fusions of the genes encoding the protein kinase A catalytic subunit, PRKACA, and heat shock protein, DNAJB1. The chimeric DNAJB1-PRKACA protein has increased kinase activity and is essential for FLC xenograft growth. Here, we explore the critical oncogenic pathways controlled by DNAJB1-PRKACA using patient-derived FLC models, engineered systems, and patient samples. We show that a core function of DNAJB1-PRKACA is the phosphorylation and inactivation of Salt-inducible kinases (SIKs). This leads to deregulation of the CRTC2 transcriptional co-activator and p300 acetyltransferase, resulting in transcriptional reprogramming and increased global histone acetylation, driving malignant growth. Our studies establish a central oncogenic mechanism of DNAJB1-PRKACA and suggest the potential of targeting CRTC2/p300 in FLC. Notably, these findings link this rare cancer's signature fusion oncoprotein to more common cancer gene alterations involving STK11 and GNAS, which also function via SIK suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77fde5ecc4bc9340e646816c229c18dfe2fdb5e" target='_blank'>
              DNAJB1-PRKACA fusion drives fibrolamellar liver cancer through impaired SIK signaling and CRTC2/p300-mediated transcriptional reprogramming.
              </a>
            </td>
          <td>
            Ilaria Gritti, Jinkai Wan, Vajira Weeresekara, Joel M Vaz, Giuseppe Tarantino, Tenna Holgersen Bryde, Vindhya Vijay, Ashwin V. Kammula, Prabhat Kattel, Songli Zhu, Phuong Vu, Marina Chan, Meng-Ju Wu, John D Gordan, K. Patra, Vanessa S Silveira, R. Manguso, Marc N. Wein, Christopher J Ott, Jun Qi, David Liu, Kei Sakamoto, Taranjit S. Gujral, N. Bardeesy
          </td>
          <td>2024-09-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing.
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Introduction Breast cancer is one of the most prevalent types of cancer and a leading cause of cancer-related death among females worldwide. Anoikis, a specific type of apoptosis that is triggered by the loss of anchoring between cells and the native extracellular matrix, plays a vital role in cancer invasion and metastasis. However, studies that focus on the prognostic values of anoikis-related genes (ARGs) in breast cancer are scarce. Methods Gene expression data were obtained from The Cancer Genome Atlas (TCGA), Gene Expression Omnibus (GEO), and Molecular Taxonomy of Breast Cancer International Consortium (METABRIC) databases. Five anoikis-related signatures (ARS) were selected from ARGs through univariate Cox regression analysis, LASSO regression analysis, and multivariate Cox regression analysis. Subsequently, an ARGs risk score model was established, and breast cancer patients were divided into high and low risk groups. The correlation between risk groups and overall survival (OS), tumor mutation burden (TMB), tumor microenvironment (TME), stemness, and drug sensitivity were analyzed. Moreover, RT-qPCR was performed to verify the gene expression levels of the five ARS in breast cancer tissues. Furthermore, a nomogram model was constructed based on ARGs risk score and clinicopathological factors. Results A novel ARGs risk score model was constructed based on five ARS (CEMIP, LAMB3, CD24, PTK6, and PLK1), and breast cancer patients were divided into high and low risk groups. Correlation analysis showed that the high and low risk groups had different OS, TMB, TME, stemness, and drug sensitivity. Both the ARGs risk score model and the nomogram showed promising prognosis predictive value in breast cancer. Conclusion ARS could be used as promising biomarkers for breast cancer prognosis predication and treatment options selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2304d0d98fc1550ec1a30e39d41f3e284722407" target='_blank'>
              Identification of Novel Anoikis-Related Gene Signatures to Predict the Prognosis, Immune Microenvironment, and Drug Sensitivity of Breast Cancer Patients
              </a>
            </td>
          <td>
            Jiena Liu, Hao Wu, Qin Wang, Shengye Jin, Siyu Hou, Zibo Shen, Liuying Zhao, Shouping Xu, Da Pang
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Homologous recombination (HR) is an important mechanism for repairing DNA double-strand breaks (DSBs) and preserving genome integrity. Pathogenic mutations in the HR proteins BRCA2 and the RAD51 paralogs predispose individuals to breast, ovarian, pancreatic, and prostate cancer. The RAD51 paralogs: RAD51B, RAD51C, RAD51D, XRCC2, and XRCC3 form two complexes RAD51B-RAD51C-RAD51D-XRCC2 (BCDX2) and RAD51C-XRCC3 (CX3). Similar to BRCA2, loss of RAD51 paralog functions in mammalian cells lead to chromosomal abnormalities, growth defects, disrupted RAD51 foci formation, and PARP inhibitor sensitivity. Despite significant e]ort over the past three decades, the specific molecular functions of the human RAD51 paralogs have remained elusive due to technical challenges such as low protein expression in human cell lines and instability of the purified proteins. Recent studies have determined the molecular structures of the BCDX2 and CX3 complexes dramatically enhancing our understanding of these challenging proteins. Using multiple approaches, we demonstrate that the RAD51 paralogs interact with BRCA2 at two distinct interaction hubs located in the BRC repeats and the DNA binding domain. We confirm, using a yeast 3-hybrid approach, that human RAD51 paralogs interact directly with BRC repeats one and two (BRC1-2) of BRCA2. Because of the dynamic nature of the RAD51B C-terminal domain (CTD), identified in the recently solved cryo-EM structures, we focused on elucidating the interaction with RAD51B. We determined that BRCA2 interacts with the CTD of RAD51B and not the N-terminal domain (NTD) that is involved in stacking interactions with RAD51C and RAD51D. Furthermore, the interaction with RAD51B is dependent upon an FxxA motif located on a surface exposed region of the CTD. Our study has identified novel interactions between the RAD51 paralogs and BRCA2 and further demonstrated that a previously unrecognized FxxA motif located within a mobile element of RAD51B is critical for the interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08aaf8658412f259814459f47746d486e139a2ec" target='_blank'>
              RAD51 Paralogs and RAD51 Paralog Complexes BCDX2 and CX3 Interact with BRCA2
              </a>
            </td>
          <td>
            Jacob G. Thrasher, Adeola A. Faunloye, Fabricio S. Justiniano, Kara A. Bernstein, Ryan B. Jensen
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a–RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Asymmetric partition of fate determinants during cell division is a hallmark of cell differentiation. Recent works suggested that such a mechanism is hijacked by cancer cells to increase both their phenotypic heterogeneity and plasticity and in turn their fitness. To quantify fluctuations in the partitioning of cellular elements, imaging-based approaches are used, whose accuracy is limited by the difficulty of detecting cell divisions. Our work addresses this gap proposing a general method based on high-throughput flow cytometry measurements coupled with a theoretical frameworks. We applied our method to a panel of both normal and cancerous human colon cells, showing that different kinds of colon adenocarcinoma cells display very distinct extents of fluctuations in their cytoplasm partition, explained by an asymmetric division of their size. To test the accuracy of our population-level protocol, we directly measure the inherited fractions of cellular elements from extensive time-lapses of live-cell laser scanning microscopy, finding excellent agreement across the cell types. Ultimately, our flow cytometry-based method promise to be accurate and easily applicable to a wide range of biological systems where the quantification of partition fluctuations would help accounting for the observed phenotypic heterogeneity and plasticity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89534ca95bbe7d2c00c135f356aa0991c7b534e4" target='_blank'>
              Robust assessment of asymmetric division in colon cancer cells
              </a>
            </td>
          <td>
            Domenico Caudo, Chiara Giannattasio, Simone Scalise, V. Turris, Fabio Giavazzi, G. Ruocco, Giorgio Gosti, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-09-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Meiotic crossover, i.e., the reciprocal exchange of chromosome fragments during meiosis, is a key driver of genetic diversity. Crossover is initiated by the formation of programmed DNA double-strand breaks (DSBs). While the role of ATAXIA-TELANGIECTASIA AND RAD3-RELATED (ATR) kinase in DNA damage signaling is well-known, its impact on crossover formation remains understudied. Here, using measurements of recombination at chromosomal intervals and genome-wide crossover mapping, we showed that ATR inactivation in Arabidopsis (Arabidopsis thaliana) leads to dramatic crossover redistribution, with an increase in crossover frequency in chromosome arms and a decrease in pericentromeres. These global changes in crossover placement were not caused by alterations in DSB numbers, which we demonstrated by analyzing phosphorylated H2A.X foci in zygonema. Using the seed-typing technique, we found that hotspot usage remains mainly unchanged in atr mutants compared to wild-type individuals. Moreover, atr showed no change in the number of crossovers caused by two independent pathways, which implies no effect on crossover pathway choice. Analyses of genetic interaction indicate that while the effects of atr are independent of MMS AND UV SENSITIVE81 (MUS81), ZIPPER1 (ZYP1), FANCONI ANEMIA COMPLEMENTATION GROUP M (FANCM) and D2 (FANCD2), the underlying mechanism may be similar between ATR and FANCD2. This study extends our understanding of ATR's role in meiosis, uncovering functions in regulating crossover distribution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacde0c3932254585c19d65235879a141c6f4b4" target='_blank'>
              The kinase ATR controls meiotic crossover distribution at the genome scale in Arabidopsis.
              </a>
            </td>
          <td>
            Longfei Zhu, J. Dluzewska, N. Fernández-Jiménez, Rajeev Ranjan, Alexandre Pelé, Wojciech Dzięgielewski, Maja Szymańska-Lejman, Karolina Hus, Julia Górna, M. Pradillo, Piotr A. Ziolkowski
          </td>
          <td>2024-10-29</td>
          <td>The Plant cell</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Breast cancer is associated with high morbidity and mortality, which are closely influenced by protein post-translational modifications (PTMs). Lysine crotonylation (Kcr) serves as a newly identified PTM type that plays a role in various biological processes; however, its involvement in breast cancer progression remains unclear. Minichromosome maintenance 6 (MCM6) is a critical component of DNA replication and has been previous confirmed to exhibit a significant role in tumorigenesis. Despite this, a comprehensive analysis of MCM6, particularly regarding its modifications in breast cancer is lacking. In this study, we found MCM6 is upregulated in breast invasive carcinoma (BRCA) and is associated with poorer overall survival by regulating the DNA damage repair mechanisms. Furthermore, MCM6-knockdown resulted in decreased cell proliferation and inhibited the DNA replication, leading to DNA replication stress and sustained DNA damage, thereby enhancing the chemotherapeutic sensitivity of breast cancer. Additionally, SIRT7-mediated crotonylation of MCM6 at K599 (MCM6-K599cr) was significantly upregulated in response to DNA replication stress, primarily due to the disassemebly of the MCM2-7 complex and regulated by RNF8-mediated ubiquitination. Concurrently, kaempferol, which acts as a regulator of SIRT7, was found to enhance the Kcr level of MCM6, reducing tumour weight, particular when combined with paclitaxel, highlighting its potential chemotherapeutic target for BRCA therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc331419a4f2d077914bda931ebc6dc0480b9e3e" target='_blank'>
              Crotonylation of MCM6 enhances chemotherapeutics sensitivity of breast cancer via inducing DNA replication stress.
              </a>
            </td>
          <td>
            Haoyun Song, Zhao Guo, Kun Xie, Xiangwen Liu, Xu Yang, Rong Shen, Degui Wang
          </td>
          <td>2024-10-30</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic DNA breakages and the subsequent insertion and deletion mutations are important contributors to genome instability and linked diseases. Unlike the research in point mutations, the relationship between DNA sequence context and the propensity for strand breaks remains elusive. Here, by analyzing the differences and commonalities across myriads of genomic breakage datasets, we extract the sequence-linked rules and patterns behind DNA fragility. We show the overall deconvolution of the sequence influence into short-, mid- and long-range effects, and the stressor-dependent differences in defining the range and compositional effects on DNA fragility. We summarize and release our feature compendium as a library that can be seamlessly incorporated into genomic machine learning procedures, where DNA fragility is of concern, and train a generalized DNA fragility model on cancer-associated breakages. Structural variants (SVs) tend to stabilize regions in which they emerge, with the effect most pronounced for pathogenic SVs. In contrast, the effects of chromothripsis are seen across regions less prone to breakages. We find that viral integration may bring genome fragility, particularly for cancer-associated viruses. Overall, this work offers novel insights into the genomic sequence basis of DNA fragility and presents a powerful machine learning resource to further enhance our understanding of genome (in)stability and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fef79035ee077cd9a529c7ecfbf4892bf0585e2" target='_blank'>
              Towards the genomic sequence code of DNA fragility for machine learning.
              </a>
            </td>
          <td>
            Patrick Pflughaupt, Adib A Abdullah, Kairi Masuda, Aleksandr B. Sahakyan
          </td>
          <td>2024-10-23</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The kinetochore complex, an important protein assembly situated on the centromere, plays a pivotal role in chromosome segregation during cell division. Like in animals and fungi, the plant kinetochore complex is important for maintaining chromosome stability, regulating microtubule attachment, executing error correction mechanisms, and participating in signaling pathways to ensure accurate chromosome segregation. This review summarizes the composition, function, and regulation of the plant kinetochore complex, emphasizing the interactions of kinetochore proteins with centromeric DNAs (cenDNAs) and RNAs (cenRNAs). Additionally, the applications of the centromeric histone H3 variant (the core kinetochore protein CENH3, first identified as CENP-A in mammals) in the generation of ploidy-variable plants and synthesis of plant artificial chromosomes (PACs) are discussed. The review serves as a comprehensive roadmap for researchers delving into plant kinetochore exploration, highlighting the potential of kinetochore proteins in driving technological innovations in synthetic genomics and plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f840a6e5c8eb7572f7b8a202d1475f565b0be6f0" target='_blank'>
              Plant kinetochore complex: composition, function, and regulation
              </a>
            </td>
          <td>
            Yuqian Xie, Mingliang Wang, Beixin Mo, Chao Liang
          </td>
          <td>2024-10-10</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Objective High-risk human papillomavirus (HPV) is a carcinogenic virus associated with nearly all cases of cervical cancer, as well as an increasing number of anal and oral cancers. The two carcinogenic proteins of HPV, E6 and E7, can immortalize keratinocytes and are essential for HPV-related cellular transformation. Currently, the global regulatory effects of these oncogenic proteins on the host proteome are not fully understood, and further exploration of the functions and carcinogenic mechanisms of E6 and E7 proteins is needed. Methods We used a previously established platform in our laboratory for constructing recombinant adenoviral plasmids expressing the HPV16 E7 gene to further construct recombinant virus particles expressing HPV16/18 E6, E7, and both E6 and E7 genes. These recombinant viruses were used to infect C33A cells to achieve sustained expression of the HPV16/18 E6/E7 genes. Subsequently, total RNA was extracted and RNA-Seq technology was employed for transcriptome sequencing to identify differentially expressed genes associated with HPV infection in cervical cancer. Results RNA-Seq analysis revealed that overexpression of the HPV16/18 E6/E7 genes upregulated GP6, CD36, HDAC6, ESPL1, and DNMT3B among the differentially expressed genes (DEGs) associated with cervical cancer. Spearman correlation analysis revealed a statistically significant correlation between the HDAC6 and DNMT3B genes and key pathways, including DNA replication, tumor proliferation signature, G2M checkpoint, p53 pathways, and PI3K/AKT/mTOR signaling pathways. Further, qRT-PCR and Western blot analyses indicated that both HPV16/18 E7 can upregulate the expression of HDAC6 and DNMT3B, genes associated with HPV infection-related cervical cancer. Conclusion The successful expression of HPV16/18 E6/E7 in cells indicates that the recombinant viruses retain the replication and infection capabilities of Ad4. Furthermore, the recombinant viruses expressing HPV16/18 E7 can upregulate the HDAC6 and DNMT3B genes involved in cervical cancer pathways, thereby influencing the cell cycle. Additionally, HDAC6 and DNMT3B are emerging as important therapeutic targets for cancer. This study lays the foundation for further exploration of the oncogenic mechanisms of HPV E6/E7 and may provide new directions for the treatment of HPV-related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0d44806ba3671a0acf8b9742e6838903e85bc59" target='_blank'>
              Recombinant adenoviruses expressing HPV16/18 E7 upregulate the HDAC6 and DNMT3B genes in C33A cells
              </a>
            </td>
          <td>
            Yunting Shao, Pir Tariq Shah, Qisheng Su, Shanhu Li, Fang Huang, Jun Wang, Peng Wang, Chengjun Wu
          </td>
          <td>2024-10-01</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The role of Epiplakin1 (EPPK1) in colon adenocarcinoma (COAD) was analyzed through a comprehensive evaluation of its expression, methylation, genetic mutations, and prognostic implications. A significant up-regulation of EPPK1 expression in COAD malignant cells compared to normal control samples was observed using data from the UALCAN database. EPPK1 expression was found to be elevated across different cancer development stages, racial groups, genders, and age groups, emphasizing its crucial role in cancer proliferation. Validation of EPPK1 expression through the GEPIA2.0 dataset further confirmed its overexpression in COAD when compared to normal samples. The analysis revealed dysregulation across all four stages of cancer development, with the highest expression in stage IV and the lowest in stage II. Additionally, promoter methylation analysis demonstrated a fundamental relationship between COAD samples and normal controls, revealing significant methylation patterns across different clinical parameters, including cancer stages, race, gender, and age. Overall survival (OS) and disease-free survival (DFS) analysis using the KM plotter showed a strong association between high EPPK1 expression and worse survival outcomes in COAD patients. Conversely, low EPPK1 expression was linked to better OS and DFS. Genetic mutation analysis using cBioPortal identified minimal EPPK1 mutations in COAD, predominantly truncating and missense mutations, highlighting their relevance to EPPK1 dysregulation in COAD. These findings underscore the important role of EPPK1 in COAD development and proliferation, suggesting its potential as a therapeutic target and prognostic marker. Further exploration of EPPK1’s molecular mechanisms and its involvement in the COAD microenvironment may reveal new pathways for targeted treatments and precision medicine strategies against this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1746eb2037e9f374ca2ed51f59d8d79d309f8805" target='_blank'>
              Deciphering the Role of EPPK1 in the Development of Colon Adenocarcinoma by Integrated Bioinformatics Approach
              </a>
            </td>
          <td>
            Muhammad Akram, Muhammad Umair Abid
          </td>
          <td>2024-10-24</td>
          <td>Journal of Clinical and Nursing Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>