<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-24 09:27:56 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>30</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>66</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>107</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>319</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>275</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/effa3cdf47f124f2fc25cc824cb0bcd26047dfd8" target='_blank'>
              Comprehensive dissection of cis-regulatory elements in a 2.8 Mb topologically associated domain in six human cancers
              </a>
            </td>
          <td>
            Christina M. Caragine, Victoria T Le, Meer Mustafa, Bianca Jay Diaz, John A Morris, Simon Müller, A. Méndez-Mancilla, Evan Geller, Noa Liscovitch-Brauer, Neville E. Sanjana
          </td>
          <td>2025-02-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aneuploidy is prevalent in cancer and associates with fitness advantage and poor patient prognosis. Yet, experimentally induced aneuploidy initially leads to adverse effects and impaired proliferation, suggesting that cancer cells must adapt to aneuploidy. We performed in vitro evolution of cells with extra chromosomes and obtained cell lines with improved proliferation and gene expression changes congruent with changes in aneuploid cancers. Integrated analysis of cancer multi-omics data and model cells revealed increased expression of DNA replicative and repair factors, reduced genomic instability, and reduced lysosomal degradation. We identified E2F4 and FOXM1 as transcription factors required for adaptation to aneuploidy in vitro and in cancers and validated this finding. The adaptation to aneuploidy also coincided with specific copy number aberrations that correlate with poor patient prognosis. Chromosomal engineering mimicking these aberrations improved aneuploid cell proliferation, while loss of previously present extra chromosome impaired it. The identified common adaptation strategies suggest replication stress, genomic instability, and lysosomal stress as common liabilities of aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2024-03-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8275563011d6ab6090a01c6b82a9c990741f5" target='_blank'>
              Z-DNA formation induces the totipotent-like state and primes Zscan4-dependent chromatin compartmentalization
              </a>
            </td>
          <td>
            Shireen Shajahan, Yann Loe-Mie, Marion Salmon-Legagneur, Tatiana Traboulsi, Agnès Thierry, Anne Dejean, J. Cossec
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Transposable elements (TEs) are reactivated in tumors and serve as significant contributors to tumor transcriptomic complexity and heterogeneity. However, their repetitive nature and diverse transcriptional forms have hindered efforts to characterize TE-derived transcripts independent of host gene contexts. Here, using our self-developed splicing-junction analysis tool ASJA, we systematically interrogated TE transcription across 32 cancer types, identifying transcripts autonomously initiated from and exclusively spliced among TEs with high TE content, termed TEtrans. Our analysis revealed 5,361 TEtrans exhibiting pan-cancer prevalence, tumor-specific expression and features of mature transcripts, including canonical splicing, 5’ caps and polyA tails. Mostly unannotated and enriched in intergenic regions, TEtrans are predominantly derived from primate-specific classes with conserved splice pattern. TEtrans burden demonstrates heterogeneous associations with prognosis and immune activity across cancers, while their expression can be epigenetically dual-regulated by stemness- and inflammation-associated transcription factors. Functional studies uncovered that TEtrans could act as oncogenes, exemplified by tsTE1, a HERVH-derived transcript that promotes colorectal cancer proliferation by enhancing TOP1-mediated DNA supercoil relaxation. Remarkably, ∼7.2% of TEtrans are identified to encode tumor neoantigens, including viral proteins and unannotated peptides, which are shared among patients and validated by proteogenomic analysis. These TEtrans-derived neoantigens are immunogenic both in vitro and in vivo, and exceed neoepitopes from genomic alterations in abundance per tumor, particularly in cancers with low mutational burden. Collectively, as a gene-independent form of TE-derived transcripts, TEtrans represents a unique source of oncogenes and tumor neoantigens, expanding the functional repertoire of TEs in cancer biology and offering new avenues for therapeutic targeting. Highlights TEtrans are tumor-specific transcripts with conserved splicing patterns, predominantly derived from primate TEs in intergenic regions. TEtrans expression is epigenetically activated and dually regulated by stemness/inflammation transcription factors. TEtrans function as oncogenes, exemplified by tsTE1 enhancing TOP1-mediated DNA relaxation. TEtrans encode tumor neoantigens that elicit CD8+ T cell responses, providing targets for immunotherapy, especially in low-mutation tumors. Abstract figure">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90153db4362d7e931500fbc74364483bb05472a" target='_blank'>
              Transcriptomic splicing analysis reveals a gene-independent form of TE reactivation in tumors representing unrecognized source of oncogenes and neoantigens
              </a>
            </td>
          <td>
            Ziteng Li, Yichao Bao, Qin Li, Hongwu Yu, He-Yun Zhang, Yifan Wen, Yu Yang, Jingjing Zhao, Peng Lin, Yan Li, Zhixiang Hu, Xichun Hu, Xiaodong Zhu, Shenglin Huang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bef3e779a036b2c2440760328f43291d95c45" target='_blank'>
              Topoisomerase Regulation of Cancer Gene Expression.
              </a>
            </td>
          <td>
            Laura Baranello, Fedor Kouzine, D. Levens
          </td>
          <td>2025-03-18</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Centromeres are chromosomal loci that ensure proper chromosome segregation by providing a platform for kinetochore assembly and spindle force transduction during cell division. Human centromeres are defined primarily by a unique chromatin domain featuring the histone H3 variant, Centromere Protein A (CENP-A), that marks a single active centromere locus per chromosome. CENP-A chromatin typically occupies a small subregion of low DNA methylation within multi-megabase arrays of hypermethylated alpha-satellite repeats and constitutive pericentric heterochromatin. However, the mechanisms defining and maintaining precise centromere position and domain size, and the role of the underlying alpha satellite DNA sequence, are poorly characterised. Using an experimentally-induced neocentromere in RPE1 cells, we discovered that the SUV39H1 and H2 methyltransferases tri-methylate H3K9 at neocentromere boundaries to maintain CENP-A domain size independent of DNA methylation or satellite sequences. Furthermore, we found that the CENP-A domain at canonical alpha-satellite-based centromeres is characterized by local depletion of H3K9me3-mediated heterochromatin, coinciding with the DNA methylation dip region. We identified the SETDB1 methyltransferase as key to maintaining H3K9me3 within flanking active higher order alpha satellite arrays while SUV39s and SUZ12 contribute to globally heterochromatinize both alpha satellites and neighbouring repeats. Loss of this heterochromatin boundary results in the progressive expansion of the primary CENP-A domain, erosion of DNA methylation, and the nucleation of new centromeres across alpha satellite sequences. Our study identifies the functional specialization of different H3K9 methyltransferases across centromeric and pericentric domains, crucial for maintaining centromere domain size and number.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21ac02ba5e831d24deadbae2a16d8fc0cb0af4d" target='_blank'>
              Heterochromatin boundaries maintain centromere position, size and number
              </a>
            </td>
          <td>
            Ben L. Carty, Danilo Dubocanin, Marina Murillo-Pineda, Marie Dumont, Emilia Volpe, Pawel Mikulski, Julia Humes, Oliver Whittingham, D. Fachinetti, Simona Giunta, Nicolas Altemose, L. Jansen
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Summary The epigenetic state of chromatin, gene activity and chromosomal positions are interrelated in plants. In Arabidopsis thaliana, chromosome arms are DNA‐hypomethylated and enriched with the euchromatin‐specific histone mark H3K4me3, while pericentromeric regions are DNA‐hypermethylated and enriched with the heterochromatin‐specific mark H3K9me2. We aimed to investigate how the chromosomal location affects epigenetic stability and gene expression by chromosome engineering. Two chromosomal inversions of different sizes were induced using CRISPR/Cas9 to move heterochromatic, pericentric sequences into euchromatic regions. The epigenetic status of these lines was investigated using whole‐genome bisulfite sequencing and chromatin immunoprecipitation. Gene expression changes following the induction of the chromosomal inversions were studied via transcriptome analysis. Both inversions had a minimal impact on the global distribution of histone marks and DNA methylation patterns, although minor epigenetic changes were observed across the genome. Notably, the inverted chromosomal regions and their borders retained their original epigenetic profiles. Gene expression analysis showed that only 0.5–1% of genes were differentially expressed genome‐wide following the induction of the inversions. CRISPR/Cas‐induced chromosomal inversions minimally affect epigenetic landscape and gene expression, preserving their profiles in subsequent generations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8552d926f37133e02cf503023e99c3ac17866bc9" target='_blank'>
              Epigenetic state and gene expression remain stable after CRISPR/Cas‐mediated chromosomal inversions
              </a>
            </td>
          <td>
            Solmaz Khosravi, Rebecca Hinrichs, Michelle Rönspies, Reza Haghi, Holger Puchta, Andreas Houben
          </td>
          <td>2025-01-29</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6be79817ee79c2803379821d2392c413312af98" target='_blank'>
              Personalized tumor-specific amplified DNA junctions in peripheral blood of patients with glioblastoma.
              </a>
            </td>
          <td>
            Mohamed F Ali, C. Riviere-Cazaux, Sarah H Johnson, Rebecca Salvatori, Alan R Penheiter, James B Smadbeck, Stephen J. Murphy, F. Harris, Lex F McCune, Lucas P Carlstrom, Michael T Barrett, F. Kosari, Leila A Jones, Cristiane M Ida, M. Borad, B. Bendok, Alfredo Quiñones-Hinojosa, Alyx B Porter, Maciej M Mrugala, K. Jaeckle, P. Anastasiadis, S. Kizilbash, John C Cheville, David M Routman, Terry C Burns, G. Vasmatzis
          </td>
          <td>2025-02-28</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="One fundamental yet open question is how eukaryotic chromosomes fold into segregated territories, a process essential for gene transcription and cell fate. Through analyzing Hi-C and chromatin-tracing DNA-FISH data, we identify long-range chromo skeleton loop structures that span over 100 Mb, extending beyond the reach of loop extrusion models. Spatial density analyses point to assembly formation independent of major nuclear structures. A subset of genomic loci serves as nucleation centers, driving loop clustering. These complexes are highly stable, as shown by live-cell imaging with sequence-specific fluorescent labeling, and biophysical model analyses reveal a multivalent binding mechanism. Our findings suggest a redundant, distributed cluster mechanism that ensures robustness across cell types and against mutations, guiding both chromosome compaction and the formation of smaller-scale chromosomal structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04b5ac6a8c671d63e4c9bd85fcca4ea46155b48" target='_blank'>
              Long-range genomic loci stochastically assemble into combinatorial forms of chromosome skeleton
              </a>
            </td>
          <td>
            Jingyu Zhang, Siyuan Wang, Simon C Watkins, Jianhua Xing
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Regulator of telomere length 1 (RTEL1) helicase facilitates telomere replication by disassembling DNA secondary structures, such as G-quadruplexes and telomeric loops (t-loops), at the ends of the chromosomes. The recruitment of RTEL1 to telomeres occurs during the S-phase of the cell cycle, but the dynamics of the process has not been studied. Here, we utilized CRISPR genome editing and single-molecule imaging to monitor RTEL1 movement within human cell nuclei. RTEL1 utilizes rapid three-dimensional diffusion to search for telomeres and other nuclear targets. Only 5% of the chromatin-bound RTEL1 is associated with telomeres at any time in the S-phase, but the telomere-bound RTEL1 has much more extended associations. This binding is enhanced by the interaction between RTEL1 and the telomeric protein TRF2 but is largely independent of RTEL1 ATPase activity. The absence of RTEL1 catalytic activity leads to severe defects in cell proliferation, slow progression out of S-phase, and chromosome end-to-end fusion events. We propose that the rapid diffusion of RTEL1 allows this low-abundance protein to explore the nucleus, bind TRF2, and be recruited to telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d521287870dd985b41001fde2641677325213069" target='_blank'>
              Rapid dynamics allow the low-abundance RTEL1 helicase to promote telomere replication
              </a>
            </td>
          <td>
            Guanhui Wu, Erin Taylor, Daniel T. Youmans, Nausica Arnoult, Thomas R. Cech
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Genome stability is safeguarded by a finely orchestrated cascade of events that collectively represent the DNA damage response (DDR). In eukaryotes, the DDR operates within the dynamic chromatin landscape, where the interplay between DNA repair factors, chromatin remodeling, replication, transcription, spatial genome organization, and cytoskeletal forces is tightly coordinated. High-resolution studies have unveiled chromatin alterations spanning multiple scales, from localized kilobase-level changes to megabase-scale reorganization, which impact chromatin’s physical properties and enhance the mobility of damaged regions. Leveraging this knowledge could pave the way for innovative therapeutic strategies, particularly in targeting chromatin dynamics to destabilize cancer cells selectively. This review, focusing on DNA double-strand breaks (DSBs), sheds light on how chromatin undergoes dynamic modifications in response to damage and how these changes influence the DDR at both local and global levels, offering a glimpse into how nuclear architecture contributes to the delicate balance between genome stability and adaptability and highlighting the importance of exploring these interactions in the context of cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627a3749979e4f865f8823e54ada96686be7874" target='_blank'>
              Interplay and Dynamics of Chromatin Architecture and DNA Damage Response: An Overview
              </a>
            </td>
          <td>
            Susanna Ambrosio, Anna Noviello, Giovanni Di Fusco, Francesca Gorini, Anna Piscone, Stefano Amente, Barbara Majello
          </td>
          <td>2025-03-11</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Abstract The alternative lengthening of telomeres (ALT) pathway is a telomerase-independent mechanism for immortalization in cancer cells and is commonly activated in low-grade and high-grade glioma, as well as osteosarcoma. The ALT pathway can be activated under various conditions and has often been shown to include mutational loss of ATRX. However, this is insufficient in isolation and so other cellular event must also be implicated. It has been shown that excessive accumulation of DNA:RNA hybrid structures (R-loops) and/or formation of DNA–protein crosslinks (DPCs) can be other important driving factors. The underlying cellular events leading to R-loop and DPC formation in ALT cancer cells to date remain unclear. Here, we demonstrate that excessive cellular reactive oxygen species (ROS) is an important causative factor in the evolution of ALT-telomere maintenance in ATRX-deficient glioma. We identified three sources of elevated ROS in ALT-positive gliomas: co-mutation of SETD2, downregulation of DRG2, and hypoxic tumour microenvironment. We demonstrate that elevated ROS leads to accumulation of R-loops and, crucially, resolution of R-loops by the enzyme RNase H1 prevents ALT pathway activity in cells exposed to elevated ROS. Further, we found a possible causal link between the formation of R-loops and the accumulation of DPCs, in particular, formation of TOP1 complexes covalently linked to DNA (Top1cc). We also demonstrate that elevation of ROS can trigger over-activity of the ALT pathway in osteosarcoma and glioma cell lines, resulting in excessive DNA damage and cell death. This work presents important mechanistic insights into the endogenous origin of excessive R-loops and DPCs in ALT-positive cancers, as well as highlighting potential novel therapeutic approaches in these difficult-to-treat cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33ca69e995ecf5806fb82d5a6c050835cc9faa" target='_blank'>
              Elevated reactive oxygen species can drive the alternative lengthening of telomeres pathway in ATRX-null cancers
              </a>
            </td>
          <td>
            Tomas Goncalves, S. Cunniffe, Tiffany S Ma, Natalie Mattis, Andrew W Rose, Thomas B. Kent, David R Mole, Helene E B Geiller, Linda van Bijsterveldt, Timothy C Humphrey, Ester M. Hammond, Richard J Gibbons, David Clynes, Anna M Rose
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Centromeres are an important part of chromosomes which direct chromosome segregation during cell division. Their modifications can therefore explain the unusual mitotic and meiotic behaviour of certain chromosomes, such as the germline-restricted chromosome (GRC) of songbirds. This chromosome is eliminated from somatic cells during early embryogenesis and later also from male germ cells during spermatogenesis. Although the mechanism of elimination is not yet known, it is possible that it involves a modification of the centromeric sequence on the GRC, resulting in problems with the attachment of this chromosome to the mitotic or meiotic spindle and its lagging during anaphase, which eventually leads to its elimination from the nucleus. However, the repetitive nature and rapid evolution of centromeres make their identification and comparative analysis across species and chromosomes challenging. Here, we used a combination of cytogenetic and genomic approaches to identify the centromeric sequence of two closely related songbird species, the common nightingale (Luscinia megarhynchos) and the thrush nightingale (L. luscinia). We found a 436-bp satellite repeat present in the centromeric regions of all regular chromosomes, making it a strong candidate for the centromeric repeat. This centromeric repeat was highly similar between the two nightingale species. Interestingly, hybridization of the probe to this satellite repeat on meiotic spreads suggested that this repeat is missing on the GRC. Our results indicate that the change of the centromeric sequence may underlie the unusual inheritance and programmed DNA elimination of the GRC in songbirds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cc2a0f86985b1b169e70e74228b3c32d8f2ac99" target='_blank'>
              Germline-restricted chromosome of songbirds has different centromere compared to regular chromosomes
              </a>
            </td>
          <td>
            Jakub Rídl, D. Dedukh, Zuzana Halenková, Stephen A. Schlebusch, V. Beneš, Mireia Osuna Lopez, T. Osiejuk, Francisco J Ruiz-Ruano, Alexander Suh, Tomáš Albrecht, Jiří Reif, Radka Reifová
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition.
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab99623f5d226080b2ad6d750b3bbe647805883" target='_blank'>
              The landscape of Helicobacter pylori-mediated DNA breaks links bacterial genotoxicity to its oncogenic potential
              </a>
            </td>
          <td>
            Hadas Sibony-Benyamini, Rose Jbara, Tania Shubash Napso, Layan Abu-Rahmoun, Daniel Vizenblit, Michal Easton-Mor, Shira Perez, Alexander Brandis, Tamar Leshem, Avi Peretz, Yaakov Maman
          </td>
          <td>2025-02-24</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70f85dd49412748b7fb2dbe319b405d34092c20d" target='_blank'>
              Multi-omics analysis reveals the panoramic picture of TOP2A in pan-cancer
              </a>
            </td>
          <td>
            Jin Zhang, Tianxiao Yang, Kenie Wang, Jie Pan, Junjun Qiu, Shengnai Zheng, Xuesong Li, Guanghao Li
          </td>
          <td>2025-02-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa270994f0b80ef6d572c92b9c5ee380872d96e" target='_blank'>
              Increased maize chromosome number by engineered chromosome fission
              </a>
            </td>
          <td>
            Yibing Zeng, Mingyu Wang, J. Gent, R. Dawe
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc713b72314da7faa35c83a1d8de687d85095e55" target='_blank'>
              HRDetect in Ovarian Carcinoma: Stratification and Therapeutic Implications.
              </a>
            </td>
          <td>
            K. Banda, H. R. Davies, Y. Kumar, Y. Memari, A. Degasperi, M. Radke, I. Rodriguez, T. A. Gooley, F. Menghi, T. Harding, K. Lin, E. T. Liu, S. Nik-Zainal, E. M. Swisher
          </td>
          <td>2025-03-06</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728a2708c415752990d708812f36363b77d5df" target='_blank'>
              Epigenetic Control of TERRA by FTSJ3 is Critical for Telomerase-Driven Cancers
              </a>
            </td>
          <td>
            Jared D.W. Price, Frederick S. Vizeacoumar, Omar Abuhussein, Vincent Maranda, Yue Zhang, H. Adachi, Liliia Kyrylenko, A. Rangel-Pozzo, He Dong, Lihui Gong, Prachi Walke, Ashtalakshmi Ganapathysamy, Connor Denomy, T. Freywald, Renuka Dahiya, Hussain Elhasasna, Anjali Saxena, Jeff P. Vizeacoumar, Hardikkumar Patel, Karthic Rajamanickam, Kathryn Nguyen, D. M. de Oliveira, Mary Lazell-Wright, Alain Morejon Morales, Aanchal Aggarwal, Jia Lin Xu, N. Alli, E. P. Munhoz, Peng Gao, Jayme Salsman, Dinesh K Dahiya, Cristina Gonzalez-Lopez, Patricia Thibault, Michael Levin, Graham Dellaire, Nicholas Jette, Gary Groot, Anand Krishnan, Shahid Ahmed, Christopher Eskiw, Khaled H. Barakat, Yuliang Wu, Ronald A. DePinho, Sabine Mai, Yi-Tao Yu, J. Wong, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Large eukaryotic genomes are packaged into the restricted area of the nucleus to protect the genetic code and provide a dedicated environment to read, copy and repair DNA. The physical organisation of the genome into chromatin loops and self-interacting domains provides the basic structural units of genome architecture. These structural arrangements are complex, multi-layered, and highly dynamic and influence how different regions of the genome interact. The role of chromatin structures during transcription via enhancer-promoter interactions is well established. Less understood is how nuclear architecture influences the plethora of chromatin transactions during DNA replication and repair. In this review, we discuss how genome architecture is regulated during the cell cycle to influence the positioning of replication origins and the coordination of DNA double strand break repair. The role of genome architecture in these cellular processes highlights its critical involvement in preserving genome integrity and cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d6e724c4765bfa8612f020a665aed0f98394cb" target='_blank'>
              Roles for the 3D genome in the cell cycle, DNA replication, and double strand break repair
              </a>
            </td>
          <td>
            Katherine A Giles, P. Taberlay, Anthony J. Cesare, Mathew J. K. Jones
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b454878504977a6303670c09e4ebf64ca85ff3" target='_blank'>
              Senataxin prevents replicative stress induced by the Myc oncogene
              </a>
            </td>
          <td>
            Silvia Sberna, Marco Filipuzzi, Nicola Bianchi, Ottavio Croci, Federica Fardella, C. Soriani, S. Rohban, Sara Carnevali, Alessandra Alberta Albertini, Nicola Crosetto, S. Rodighiero, Arianna Chiesa, Laura Curti, S. Campaner
          </td>
          <td>2025-03-19</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Non-canonical roles for growth factors in the nucleus have been previously described, but their mechanism of action and biological roles remain enigmatic. Platelet-derived growth factor B (PDGFB) can drive formation of low-grade glioma and here we show that it localizes to the nucleus of human glioma cells where it binds chromatin to preserve genome stability and cell lineage. Failure of PDGFB to localize to the nucleus leads to chromosomal abnormalities, aberrant heterochromatin architecture and accelerated tumorigenesis. Furthermore, nuclear localization of PDGFB is reliant upon the expression levels and mutation status of isocitrate dehydrogenase (IDH). Unexpectedly, we identified macrophages as the predominant source of PDGFB in human, finding that immune-derived PDGFB can localize to the nucleus of glioma cells. Collectively, these studies show that immune derived PDGFB enters the nucleus of glioma cells to maintain genomic stability, while identifying a new mechanism by which IDH mutations promote gliomagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646b1a4f23d92e4dc1293f0abe42bdd8c61153d0" target='_blank'>
              Mutant IDH impairs chromatin binding by PDGFB to promote chromosome instability
              </a>
            </td>
          <td>
            R. Curry, Malcolm F McDonald, Peihao He, B. Lozzi, Yeunjung Ko, Isabella O'Reilly, Anna Rosenbaum, Wookbong Kwon, Leyla Fahim, Joshua Marcus, Noah Powell, Su Wang, Jin Ma, Asha Multani, Dongjoo Choi, Debo Sardar, Carrie Mohila, Jason Lee, Marco Gallo, Arif O. Harmanci, A. Harmanci, Benjamin Deneen, Ganesh Rao
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37294bb2928f0f4df70a33eed7a5c2638b0c5df2" target='_blank'>
              Epigenetically dynamic human centromeres are maintained within a stable DNA methylation signature
              </a>
            </td>
          <td>
            Megan A. Mahlke, Lior Lumerman, Poulomi Nath, Cy Chittenden, Savannah J. Hoyt, Jonas Koeppel, Yuan Xu, Rebecca Raphael, Kylie Zaffina, P. Hook, W. Timp, K. Miga, Peter J. Campbell, Rachel J. O’Neill, Nicolas Altemose, Yael Nechemia-Arbely
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d0020163f84c6d4b9e21837bb90e991331c10b" target='_blank'>
              Enabling sensitive and precise detection of ctDNA through somatic copy number aberrations in breast cancer
              </a>
            </td>
          <td>
            Riccardo Scandino, A. Nardone, Nicola Casiraghi, F. Galardi, Mattia Genovese, Dario Romagnoli, Marta Paoli, C. Biagioni, Andrea Tonina, I. Migliaccio, M. Pestrin, E. Moretti, L. Malorni, L. Biganzoli, M. Benelli, Alessandro Romanel
          </td>
          <td>2025-03-08</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c637d49373bdbc8ffeb399dfd912513e88f9743f" target='_blank'>
              A discrete region of the D4Z4 is sufficient to initiate epigenetic silencing
              </a>
            </td>
          <td>
            Ellen M. Paatela, Faith G. St. Amant, Danielle C Hamm, Sean R. Bennett, Taranjit S. Gujral, Silvère M. van der Maarel, Stephen J. Tapscott
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Homologous recombination maintains genome stability by repairing double strand breaks and protecting replication fork stability. Defects in homologous recombination results in cancer predisposition but can be exploited due to increased sensitivity to certain chemotherapeutics such as PARP inhibitors. The NEK8 kinase has roles in the replication response and homologous recombination. NEK8 is overexpressed in breast cancer, but the impact of NEK8 overexpression on homologous recombination has not been determined. Here, we demonstrate NEK8 overexpression inhibits RAD51 focus formation resulting in a defect in homologous recombination and degradation of stalled replication forks. Importantly, NEK8 overexpression sensitizes cells to the PARP inhibitor, Olaparib. Together, our results suggest NEK8 overexpressing tumors may be recombination-deficient and respond to chemotherapeutics that target defects in recombination such as Olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8173c08c1d4d30ea1a0d792e303a9db46e92a70" target='_blank'>
              Overexpression of the NEK8 kinase inhibits homologous recombination
              </a>
            </td>
          <td>
            Joshua L. Turner, Georgia Moore, Tyler J. McCraw, Jennifer M. Mason
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, Pascale Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279ce0f97c7d66cc42d068594f84880c4fe69c7b" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2025-02-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background The escalating global cancer burden necessitates the development of biomarkers with enhanced specificity and sensitivity for early diagnosis and therapeutic efficacy monitoring. The CNIH4 gene, an emerging biomarker, is increasingly recognized for its role in the malignant progression across various cancers. Methods We conducted a comprehensive multi-omics analysis of CNIH4, including pan-cancer expression profiles, epigenetic alterations, immune microenvironment characteristics, and therapeutic response patterns. Our focus was on clinical features, molecular underpinnings, and drug sensitivity in breast cancer (BRCA) associated with CNIH4. In vitro studies were also performed to assess the effects of CNIH4 knockdown on cell proliferation and cell cycle in the MDA-MB-231 cell line. Results CNIH4 upregulation was observed in multiple cancers, significantly correlating with genomic instability. High CNIH4 expression levels were linked to poor prognosis across cancers and associated with key cancer-related pathways, particularly those in cell cycle regulation and DNA repair. Correlation analyses suggest a role for CNIH4 in the tumor immune microenvironment, as evidenced by its association with immune subtypes, immune-related genes, and immune cell infiltration. Single-cell and spatial transcriptome analyses confirmed that CNIH4 expression in BRCA predicts tumor malignancy. Drug sensitivity analysis revealed a significant correlation between CNIH4 and responsiveness to various kinase inhibitors and chemotherapeutic agents. In vitro experiments demonstrated that CNIH4 knockdown significantly impacts the proliferation and cell cycle of MDA-MB-231 cells. Conclusion Our study highlights CNIH4 as a promising pan-cancer biomarker with significant implications for tumor progression and a critical role in cell cycle regulation in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7a2eade185851de2705c33f65b5768b5aafc69" target='_blank'>
              Deciphering the role of CNIH4 in pan-cancer landscapes and its significance in breast cancer progression
              </a>
            </td>
          <td>
            Yao Xu, Zengzhen Lai, Chaolin Li
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The genome is dynamically reorganized, partitioned, and divided during mitosis. Despite their role in organizing interphase chromatin, transcription factors were largely believed to be mitotic spectators evicted from chromatin during mitosis, only able to reestablish their position on DNA upon entry into G1. However, a panoply of evidence now contradicts this early belief. Numerous transcription factors are now known to remain active during mitosis to achieve diverse purposes, including chromosome condensation, regulation of the centromere/kinetochore function, and control of centrosome homeostasis. Inactivation of transcription factors during mitosis results in chromosome segregation errors, key features of cancer. Moreover, active transcription and the production of centromere-derived transcripts during mitosis are also known to play key roles in maintaining chromosomal stability. Finally, many transcription factors are associated with chromosomal instability through poorly defined mechanisms. Herein, we will review the emerging roles of transcription factors and transcription during mitosis with a focus on their role in promoting the faithful segregation of sister chromatids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b9bda3d766e3ac7d2259564d99c0e6e216b7356" target='_blank'>
              Emerging Roles for Transcription Factors During Mitosis
              </a>
            </td>
          <td>
            Samuel Flashner, Jane Azizkhan-Clifford
          </td>
          <td>2025-02-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d43986e3ec1bf10fa7949b61cdcc071a4afe56" target='_blank'>
              Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, Masashi Kariya, Ayami Osaka, S. Ishino, Masataka Yamauchi, Tomonori Tamura, I. Hamachi, Shoji Takada, Y. Ishino, H. Atomi
          </td>
          <td>2025-02-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer, characterized by the uncontrolled proliferation of cells, is one of the leading causes of death globally, with approximately one in five people developing the disease in their lifetime. While many driver genes were identified decades ago, and most cancers can be classified based on morphology and progression, there is still a significant gap in knowledge about genetic aberrations and nuclear DNA damage. The study of two critical groups of genes—tumor suppressors, which inhibit proliferation and promote apoptosis, and oncogenes, which regulate proliferation and survival—can help to understand the genomic causes behind tumorigenesis, leading to more personalized approaches to diagnosis and treatment. Aberration of tumor suppressors, which undergo two-hit and loss-of-function mutations, and oncogenes, activated forms of proto-oncogenes that experience one-hit and gain-of-function mutations, are responsible for the dysregulation of key signaling pathways that regulate cell division, such as p53, Rb, Ras/Raf/ERK/MAPK, PI3K/AKT, and Wnt/β-catenin. Modern breakthroughs in genomics research, like next-generation sequencing, have provided efficient strategies for mapping unique genomic changes that contribute to tumor heterogeneity. Novel therapeutic approaches have enabled personalized medicine, helping address genetic variability in tumor suppressors and oncogenes. This comprehensive review examines the molecular mechanisms behind tumor-suppressor genes and oncogenes, the key signaling pathways they regulate, epigenetic modifications, tumor heterogeneity, and the drug resistance mechanisms that drive carcinogenesis. Moreover, the review explores the clinical application of sequencing techniques, multiomics, diagnostic procedures, pharmacogenomics, and personalized treatment and prevention options, discussing future directions for emerging technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4acb0abcdb89e4becb0be9ae1ca6990d3d6cdf49" target='_blank'>
              Translational Advances in Oncogene and Tumor-Suppressor Gene Research
              </a>
            </td>
          <td>
            R. Stojchevski, Edward Agus Sutanto, Rinni Sutanto, N. Hadzi-Petrushev, M. Mladenov, Sajal Raj Singh, J. K. Sinha, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, Rakesh Bhaskar, Dimiter B Avtanski
          </td>
          <td>2025-03-17</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The ability to replicate and pass information to descendants is a fundamental requirement for life. In addition to the DNA-based genetic information, modifications of the DNA or DNA-associated proteins can create patterns of heritable gene regulation. Such epigenetic inheritance allows for adaptation without mutation, but its limits and regulation are incompletely understood. Here we developed a C. elegans system to study the transgenerational epigenetic inheritance of H3K27me3, a conserved histone posttranslational modification associated with gene repression. We find that induced alterations of the genome-wide H3K27me3 landscape and the associated fertility defects persist for many generations in genetically wildtype descendants under selective pressure. We uncover that the inheritance of the altered H3K27me3 landscape is regulated by two chromodomain proteins with antagonizing functions, and provide mechanistic insight into how this molecular memory is initiated and maintained. Our results demonstrate that epigenetic inheritance can act as a mutation-independent, heritable mechanism of adaptation. In Brief Özdemir et al. demonstrate that an altered genomic distribution of the histone modification H3K27me3 can be epigenetically inherited across many generations through the activity of HERI-1/SET-32/MES-4, which is antagonized by CEC-6/PRC2 in C. elegans. Highlights Altered H3K27me3 landscapes can be inherited for at least 15 generations in C. elegans. The chromodomain proteins CEC-6 and HERI-1 have opposite roles in antagonizing or promoting the maintenance of the altered H3K27me3 landscape. H3K23me3 and H3K36me3 replace H3K27me3 to promote the intergenerational and transgenerational inheritance of the altered epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9477951c95e1934dfca6ddb054dc1db83139c5" target='_blank'>
              Transgenerational maintenance of H3K27me3 heterochromatin is balanced by chromodomain proteins in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Isa Özdemir, Anna Höfler, Kamila Delaney, Joanna M. Wenda, Chengyin Li, Arneet L. Saltzman, Andreas Boland, Florian A. Steiner
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genome three-dimensional organization is essential for the coordination of eukaryote gene expression. The chromosomes of the pathogen Trypanosoma brucei contain hundreds of silent variant surface glycoprotein (VSGs) genes in subtelomeric regions. However, T. brucei transcribes a single VSG gene and periodically changes the VSG expressed by transcriptional or recombination mechanisms, altering its surface coat to escape host antibodies by antigenic variation. We show that VSG-rich silent subtelomeric regions form distinct chromosome compartments from transcribed regions, with subtelomeric compartments of different chromosomes co-interacting. We uncovered chromatin-associating factors at the boundaries of transcribed and silent compartments. Among these, repressor activator-protein 1 (RAP1) marks the compartment boundaries and spreads over silent regions. Inactivation of phosphatidylinositol phosphate 5-phosphatase removed RAP1 from compartment boundaries and subtelomeric regions, disrupting compartment assembly and derepressing all VSG genes. The data show spatial segregation of repressed from transcribed chromatin and phosphoinositides regulation of silent compartment assembly and genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a481ddec30044015c744ee5e03cef2d29b31493" target='_blank'>
              Chromosome compartment assembly is essential for subtelomeric gene silencing in trypanosomes
              </a>
            </td>
          <td>
            Luiza Berenguer Antunes, Tony Isebe, O. Kutova, I. Cestari
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, Titus K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Developing cancer therapies that induce robust death of the malignant cell is critical to prevent relapse. Highly effective strategies, such as immunotherapy, exemplify this observation. Here we provide the structural and molecular underpinnings for an approach that leverages chemical induced proximity to produce specific cell killing of diffuse large B cell lymphoma, the most common non-Hodgkin’s lymphoma. We develop KAT-TCIPs (lysine acetyltransferase transcriptional/epigenetic chemical inducers of proximity) that redirect p300 and CBP to activate programmed cell death genes normally repressed by the oncogenic driver, BCL6. Acute treatment rapidly reprograms the epigenome to initiate apoptosis and repress c-MYC. The crystal structure of the chemically induced p300-BCL6 complex reveals how chance interactions between the two proteins can be systematically exploited to produce the exquisite potency and selectivity of KAT-TCIPs. Thus, the malignant function of an oncogenic driver can be co-opted to activate robust cell death, with implications for precision epigenetic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d96f4bdeddc3af87404d0b766ff6c147fee90d7" target='_blank'>
              A Bivalent Molecular Glue Linking Lysine Acetyltransferases to Oncogene-induced Cell Death
              </a>
            </td>
          <td>
            Meredith N. Nix, S. Gourisankar, Roman C. Sarott, Brendan G. Dwyer, Sabin Nettles, Michael M. Martinez, Hind Abuzaid, Haopeng Yang, Yanlan Wang, Juste M. Simanauskaite, Bryan A. Romero, Hannah M. Jones, Andrey Krokhotin, Tara N. Lowensohn, Lei Chen, Cara Low, Mark M. Davis, Daniel Fernandez, Tinghu Zhang, , Stephen M. Hinshaw, N. Gray, G. R. Crabtree
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185bf82df0621d29d2214d3aa74691b2937269d4" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome regulation relies on complex and dynamic interactions between DNA and proteins. Recently, powerful methods have emerged that leverage third-generation sequencing to map protein-DNA interactions genome-wide. For example, Directed Methylation with Long-read sequencing (DiMeLo-seq) enables mapping of protein-DNA interactions along long, single chromatin fibers, including in highly repetitive genomic regions. However, DiMeLo-seq involves lossy centrifugation-based wash steps that limit its applicability to many sample types. To address this, we developed DiMeLo-cito, a single-tube, wash-free protocol that maximizes the yield and quality of genomic DNA obtained for long-read sequencing. This protocol enables the interrogation of genome-wide protein binding with as few as 100,000 cells and without the requirement of a nuclear envelope, enabling confident measurement of protein-DNA interactions during mitosis. Using this protocol, we detected strong binding of CTCF to mitotic chromosomes in diploid human cells, in contrast with earlier studies in karyotypically unstable cancer cell lines, suggesting that CTCF “bookmarks” specific sites critical for maintaining genome architecture across cell divisions. By expanding the capabilities of DiMeLo-seq to a broader range of sample types, DiMeLo-cito can provide new insights into genome regulation and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69924e2da138038fc52ccdade72f6d918fe9612d" target='_blank'>
              DiMeLo-cito: a one-tube protocol for mapping protein-DNA interactions reveals CTCF bookmarking in mitosis
              </a>
            </td>
          <td>
            Nathan Gamarra, Cy Chittenden, Kousik Sundararajan, Jacob P. Schwartz, Sofia Lundqvist, Denise Robles, Oberon Dixon-Luinenburg, Jeremy Marcus, Annie Maslan, J. M. Franklin, Aaron M. Streets, A. Straight, Nicolas Altemose
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells harbor cell-intrinsic programs capable of sculpting immunogenic landscapes, yet molecular brakes that enforce immune evasion remain poorly defined. Here, we identify C9ORF50, a previously uncharacterized intrinsically disordered protein, as a liquid-liquid phase separation-driven regulator of RNA processing and actionable immunotherapy target. Genetic ablation of C9ORF50 induces selective intron retention in spliceosome component transcripts, leading to cytoplasmic accumulation of immunogenic double-stranded RNA (dsRNA). This dsRNA activates type I interferon pathway that in turn orchestrates a chemokine secretory program critical for T cell recruitment. The resultant enhancement of tumor immunogenicity and reprogramming of tumor immune microenvironment drives robust antitumor immunity, effectively converting immunologically “cold” tumors to “hot” phenotypes. Therapeutic targeting of C9ORF50 via RNA interference achieved robust tumor suppression in syngeneic models, suggesting its translational relevance. Notably, C9ORF50 exhibits tumor-predominant expression, with minimal baseline levels detected in healthy tissues or immune cell populations, underscoring its potential safety as a therapeutic target. Our study not only identifies C9ORF50 as a novel LLPS-dependent regulator of RNA splicing but also establishes it as a promising and potentially safe therapeutic target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7433cc0492d9a830d01b0f8ac6ca2662ebe2b369" target='_blank'>
              C9orf50 is a Targetable Spliceosome Regulator for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Fangzhou Liu, Lulu Zhang, Ming Li, Tao Hou, Yanhua Qi, Jingyang Wang, Sujuan Zhang, Yu Liu, Jiali Liu, Yanming Hu, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b8ef7036ff1fb3a03cc5984eb57541db6e9f2c" target='_blank'>
              Functional screen identifies RBM42 as a mediator of oncogenic mRNA translation specificity.
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2025-02-04</td>
          <td>Nature cell biology</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="Background Because transposable elements (TEs) can cause heritable genetic changes, past work on TE mobility in Arabidopsis thaliana has mostly focused on new TE insertions in the germline of hypomethylated plants. It is, however, well-known that TEs can also be active in the soma, although the high-confidence detection of somatic events has been challenging. Here, we leveraged the high accuracy of PacBio HiFi long reads to evaluate the somatic mobility of TEs in individuals of an A. thaliana non-reference strain lacking activity of METHYLTRANSFERASE1 (MET1), a major component of the DNA methylation maintenance machinery. Most somatically mobile families coincided with those found in germline studies of hypomethylated genotypes, although the exact TE copies differed. We also discovered mobile elements that had been missed by standard TE annotation methods. Somatic TE activity was variable among individual plants, but also within TE families. Finally, our approach pointed to the possible involvement of alternative transposition as a cause for somatic hypermutability in a region that contains two closely spaced VANDAL21 elements. We conclude that long-read sequencing can reveal widespread TE transposition in the soma of A. thaliana hypomethylated mutants. Assessing somatic instead of germline mobilization is a fast and reliable method to investigate different aspects of TE mobility at the single plant level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62607e49f83191e3ad9371253c4b7877fe10686a" target='_blank'>
              Long-read detection of transposable element mobilization in the soma of hypomethylated Arabidopsis thaliana individuals
              </a>
            </td>
          <td>
            Andrea Movilli, Svitlana Sushko, Fernando A. Rabanal, Detlef Weigel
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30732cfd43295287b3ea69f83c8a2c64e5ceda15" target='_blank'>
              Pan-cell type continuous chromatin state annotation of all IHEC epigenomes
              </a>
            </td>
          <td>
            Habib Daneshpajouh, Ismail Moghul, Kay C Wiese, Maxwell W. Libbrecht
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89eb232b40b0237c425e76f52fd0c4fca33e33a1" target='_blank'>
              Pan‑cancer analysis of the oncogenic role of telomeric repeat binding factor 2 (TERF2) in human tumors and in vitro validation in gastric cancer by TERF2 knockdown
              </a>
            </td>
          <td>
            Qiang Ma, Wen Xu, Xiaolan Guo
          </td>
          <td>2025-02-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eed7b2fdc5b17bdbf958e634178449e843736cc" target='_blank'>
              Genome-Wide CRISPR Screening Identifies Cellular Factors Controlling Nonviral Genome Editing Efficiency
              </a>
            </td>
          <td>
            Shivani Saxena, Meha Kabra, Amr Abdeen, Divya Sinha, Min Zhu, Ruosen Xie, Giovanni Hanstad, Maria A. Fernandez Zepeda, David M. Gamm, B. Pattnaik, Shaoqin Gong, Krishanu Saha
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20cf50f0eb4725efe513e5848c9f4b814e08742b" target='_blank'>
              Endogenous DNA damage at sites of terminated transcripts.
              </a>
            </td>
          <td>
            Jingjing Liu, Jullian Perren, Cody M Rogers, Sadeieh Nimer, Alice X Wen, J. A. Halliday, Devon M Fitzgerald, Qian Mei, Ralf B Nehring, Mary Crum, Stanislav G Kozmin, Jun Xia, Matthew B Cooke, Yin Zhai, David Bates, Lei Li, P. Hastings, I. Artsimovitch, Christophe Herman, Patrick Sung, Kyle M. Miller, Susan M Rosenberg
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, Lodewyk F. A. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Infections by the major opportunistic pathogen of human Candida albicans are commonly treated with echinocandin (ECN) drugs. However, C. albicans can adapt to grow in the presence of certain amounts of ECNs. Prior studies by several laboratories have defined multiple genes, as well as mechanisms involving induced aneuploidy, that can govern this. Still, the mechanisms of ECN adaptation are not fully understood. Here, we use genome-wide profiling of chromatin accessibility by ATAC-seq to determine if ECN adaptation is reflected in changes in the chromatin landscape in the absence of aneuploidy. We find that drug adaptation is coupled with multiple changes in chromatin accessibility genome-wide, which occur predominantly in gene promoter regions. Areas of increased accessibilities in promoters are enriched with the binding motifs for at least two types of transcription factors: zinc finger and basic leucine zipper. We also find that chromatin changes are often associated with differentially expressed genes including genes with functions relevant to the ECN-adapted phenotype, such as cell wall biosynthesis. Consistent with this, we find that the cell wall is remodeled in ECN-adapted mutants, with chitin up and glucan down and increased cell surface exposure. A full understanding of ECN adaptation processes is of critical importance for the prevention of clinical resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3773ea0cdd0e2f8c5db7467a8fc51861d1557f3" target='_blank'>
              Echinocandin Adaptation in Candida albicans Is Accompanied by Altered Chromatin Accessibility at Gene Promoters and by Cell Wall Remodeling
              </a>
            </td>
          <td>
            S. Sah, Anshuman Yadav, Tyler Stahl, Jeffrey J. Hayes, Michael Bulger, E. Rustchenko
          </td>
          <td>2025-02-01</td>
          <td>Journal of Fungi</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fde8cc47c2c193748a3f06ce03baac9ac12fe7" target='_blank'>
              CTCF-mediated insulation and chromatin environment modulate Car5b escape from X inactivation
              </a>
            </td>
          <td>
            H. Fang, A. Tronco, G. Bonora, Truong Nguyen, Jitendra Thakur, J. B. Berletch, Galina N. Filippova, S. Henikoff, Jay Shendure, W. S. Noble, Zhijun Duan, Christine M. Disteche, Xinxian Deng
          </td>
          <td>2025-03-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew G Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22ab73f4c8cf8c6df493069f963fc3cc43cc9298" target='_blank'>
              The epigenetic basis of hepatocellular carcinoma - mechanisms and potential directions for biomarkers and therapeutics.
              </a>
            </td>
          <td>
            Hong-Yi Lin, Ah-Jung Jeon, Kaina Chen, Chang Jie Mick Lee, Lingyan Wu, S. Chong, C. Anene-Nzelu, R. Foo, P. Chow
          </td>
          <td>2025-03-08</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebeb9fa82784b82f60e0038c9b75808f64425555" target='_blank'>
              Merkel cell polyomavirus infection and persistence modelled in skin organoids
              </a>
            </td>
          <td>
            Silvia Albertini, Manja Czech‐Sioli, Thomas Günther, Sanamjeet Virdi, Patrick Blümke, Lisann Röpke, Veronika Brinschwitz, M. Dandri, L. Allweiss, Rudolf Reimer, Carola Schneider, Arne Hansen, S. Krasemann, E. Wyler, M. Landthaler, A. Grundhoff, Nicole Fischer
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract During X chromosome inactivation (XCI), Xist RNA establishes silencing by coating the chromosome in cis and binding diverse proteins to promote formation of a heterochromatic domain. However, Xist repeat B role beyond initiation of XCI remains unclear. Here, we find that loss of Xist repeat B in female mice allows survival and leads to a small body size persisting throughout life. Epigenetic and transcriptomic analyses reveal low levels of H3K27me3 and H2AK119ub occupancy on the X chromosome, except in certain CpG island regions, and partial reactivation of X-linked genes on the inactive X across multiple tissues. Notably, overdosage of Usp9x promotes centrosome amplification and chromosome instability. We further demonstrate that Usp9x overdosage alters asymmetric cell division, thereby affecting the process of cell differentiation. Thus, Xist repeat B is necessary for gene-specific silencing during XCI maintenance and impacts cell proliferation and differentiation during development. This provides insights into repeat B importance in maintaining XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9e14e58c0496a8253f79c766a6a837dca408f9" target='_blank'>
              Deletion of Xist repeat B disrupts cell cycle and asymmetric cell division through Usp9x hyperactivation in mice
              </a>
            </td>
          <td>
            Mingming Liang, Lichao Zhang, Heng Gong, Li Yang, Haijun Wang, Na Song, Liangxue Lai, Wanhua Xie, Zhanjun Li
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing.
              </a>
            </td>
          <td>
            Qiuhui Li, Ayse G Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The structural integrity of the chromosomes is essential to every functional process within eukaryotic nuclei. Chromosomes are DNA-histone complexes that are essential for the inheritance of genetic information to the offspring, and any defect in them is linked to mitotic errors, cancer growth, and cellular aging. Changes in the mechanical properties of a chromosome could lead to its compromised function and stability, leading to chromosome breaks. Here, we studied the changes in chromosome physical properties using metaphase chromosomes isolated from moderately malignant (MCF7) and highly malignant (MDA-MB-231) human breast cancer cells exposed to valproic acid (VPA), a known epigenetic modifier drug involved in histone hyperacetylation and DNA demethylation. Due to chromosomal structural intricacy and preparative and technical limitations of analytical tools, we employed a label-free atomic force microscopy approach for simultaneously visualizing and mapping single chromosome elasticity and stretching modulus. Additionally, we performed electron transport characteristics through metaphase chromosomes to elucidate the effect of VPA. The chromosomal elasticity and electron transport alterations are manifestations of VPA-mediated chromatin’s epigenetic changes. Our multiparametric strategy, as shown by receiver operating characteristics analyses with the physical properties of chromosomes, offers a new scope in terms of analytical tools for studying chromosomal structural changes/aberrations linked to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4081e3a40a851c1d9b5593b224de0890423bfe7" target='_blank'>
              Epigenetic Modifier Drug Valproic Acid Enhances Cancer Metaphase Chromosome Elasticity and Electron Transport: An Atomic Force Microscopy Approach
              </a>
            </td>
          <td>
            Tanya Agrawal, Debashish Paul, Amita Mishra, Ganesan Arunkumar, T. Rakshit
          </td>
          <td>2025-02-12</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34804a6772ac12af08ec23d6c1744145feb9ff1c" target='_blank'>
              Combined utility of genomic breakpoints and frame is a reliable predictor of ALK transcript function
              </a>
            </td>
          <td>
            Qiong Yang, Yumin Guo, Weihui Guo, Dongdong Liu, Haitao Wang, Xufan Cai
          </td>
          <td>2025-03-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome functions are regulated by the means of chromatin and 3D chromosome organization. Dynamic condensation and relaxation of chromosomes during cell cycle is largely controlled by the Condensin complexes. We mapped mutants in Condensin II subunits SMC2A, CAP-D3 and CAP-H2 as hypersensitive to DNA-protein crosslink (DPC) inducer zebularine. This suggested that Condensin II complex is required for resistance to genotoxic stress in Arabidopsis and prompted us to explore the underlying phenotypes. We show that the Condensin II role in resistance to zebularine is independent of the DNA damage response signaling by SOG1 and the homology-directed repair pathway. Furthermore, we found that Arabidopsis Condensin II mutants have incompletely condensed mitotic chromosomes and show abnormal chromatin connections during anaphase. Based on the structural and temporal analyses, we propose that the connections represent catenated chromatids. The chromosome connections were more frequent when the Condensin II mutants were treated by zebularine or TOP2 crosslinker and inhibitor ICRF-187. Altogether, we demonstrate that the proper large-scale chromatin organization by Condensin II is important for resistance to DNA damage response-inducing agents in Arabidopsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6084d605a281f1ce54dec4a816d64fd93e825187" target='_blank'>
              Condensin II mediates efficient chromatid resolution and resistance to genotoxic stress in Arabidopsis
              </a>
            </td>
          <td>
            Jovanka Vladejić, Klára Procházková, E. Tomaštíková, Zynke de Cock, Jana Zwyrtková, Kateřina Kaduchová, Markéta Pavlíková, A. Pečinka
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1fcc8488427a1fecd1258b2070d8c49656eecc" target='_blank'>
              Long-read RNA sequencing enables full-length chimeric transcript annotation of transposable elements in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yang Li, Yahui Liu, Yingxin Xie, Yaxuan Wang, Jing Wang, Huan Wang, Lin Xia, Dan Xie
          </td>
          <td>2025-03-15</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3e483c9466e24512f997df84699534d6031c04" target='_blank'>
              Super-enhancer-driven EFNA1 fuels tumor progression in cervical cancer via the FOSL2-Src/AKT/STAT3 axis.
              </a>
            </td>
          <td>
            Shu-Qiang Liu, Xi-Xi Cheng, Shuai He, Tao Xia, Yi-Qi Li, Wan Peng, Ya-Qing Zhou, Zi-Hao Xu, Mi-Si He, Yang Liu, P.-P. Wei, Song-Hua Yuan, Chang Liu, Shu-Lan Sun, Dong-Ling Zou, Min Zheng, Chun-Yan Lan, Chun-Ling Luo, Jin-Xin Bei
          </td>
          <td>2025-02-18</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, Surya P. Singh, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab65a9ee1a176544b4d48a367f9b18440b017c" target='_blank'>
              R-loops acted on by RNase H1 influence DNA replication timing and genome stability in Leishmania
              </a>
            </td>
          <td>
            J. Damasceno, E. Briggs, M. Krasiļņikova, Catarina A Marques, C. Lapsley, R. McCulloch
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad4692fdd18b494d4b749d2c1b7ef3cd9972515b" target='_blank'>
              Cholangiocytes’ Primary Cilia Regulate DNA Damage Response and Repair
              </a>
            </td>
          <td>
            E. Peixoto, Kishor Pant, Seth Richard, J. Abrahante, Wioletta Czaja, S. Gradilone
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="All organisms depend on specific proteins to compact and organize their genomes. In eukaryotes, histones fulfil this role, while bacterial chromosomes are shaped by nucleoid-associated proteins (NAPs). Among its pleiotropic functions, the NAP Hfq plays a pivotal role in bacterial genome organization. In this study, we characterized the structure of the C-terminal extension of Hfq, which mediates chromosomal compaction, in its DNA-bound state. Using an integrative approach that combined transmission electron microscopy, neutron scattering, site-directed mutagenesis, and molecular modeling, we identified an amyloid module formed by the C-terminal region of Hfq. This module uniquely bridges and compacts six DNA molecules, marking the first documented instance of an amyloid structure with DNA-bridging properties. Our findings redefine the functional landscape of amyloids, linking them to genome architecture and gene regulation. This result suggests that amyloid-DNA interactions may represent a conserved mechanism across biological systems, with profound implications for understanding genome organization and the regulation of gene expression in both prokaryotes and eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c1a71c24ec17a7b470740db10c59903a0c75c1" target='_blank'>
              Amyloid-like DNA bridging: a new mode of DNA shaping.
              </a>
            </td>
          <td>
            Frank Wien, Marcos Gragera, Tatsuhito Matsuo, Gautier Moroy, M. Bueno-Carrasco, Rocío Arranz, A. Cossa, Anne Martel, Heloisa N Bordallo, S. Rudić, Marisela Vélez, J. V. D. van der Maarel, J. Peters, V. Arluison
          </td>
          <td>2025-02-27</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08496aae0238cce493a4d3baa60b2ce62c4ee152" target='_blank'>
              The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication.
              </a>
            </td>
          <td>
            Oystein Monsen, L. Grønvold, Alex Datsomor, T. Harvey, James Kijas, Alexander Suh, T. R. Hvidsten, S. Sandve
          </td>
          <td>2025-02-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8319ec7e3e2dd161ac113462d4f85e344ef0e334" target='_blank'>
              NCBP2 predicts the prognosis and the immunotherapy response of cancers: a pan-cancer analysis
              </a>
            </td>
          <td>
            Shichao Li, Yulan Wang, Xi Yang, Miao Li, Guoxiang Li, Qiangqiang Song, Junyu Liu
          </td>
          <td>2025-03-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background UCK2 (Uridine-Cytidine Kinase 2) is a promising prognostic marker for malignant tumors, but its association with immune infiltration and cancer stemness in pan-cancer remains to be fully understood. we find that gene UCK2 is closed related to RNA stemness scores (RNAss) and DNA stemness scores (DNAss), which is measured the tumor stemness. We also discover an association between UCK2 expression and immune cells by CIBERSORT algorithm, ESTIMATE algorithm and ssGSEA algorithm, especially, related to T cell, monocytes, mast cells, and macrophages. This study aims to shed light on the role and possible mechanism of UCK2 in pan-cancer. Methods We used the R programming language for pan-cancer bulk sequencing data analysis, which were obtained from the University of California, Santa Cruz (UCSC) datasets. UCSC database is a very useful for explore data from TCGA and other cancer genomics datasets, The data we explored at the UCK2 transcriptome level came from TCGA data in the UCSC database. We explored differential UCK2 expression between tumor and normal samples. Immunohistochemistry (IHC) was utilized to validate the expression of UCK2 in different types cancers using tumor tissue chips. The correlations of UCK2 with prognosis, genetic instability, DNA repair, cancer stem cell characteristics, and immune cell infiltration were investigated. Furthermore, single-cell datasets, acquired from the Gene Expression Omnibus (GEO) database, were used to validate the relationship between UCK2 and immune cells. GEO is a famous public genomics database supporting freely disseminates microarray data. Finally, we analyzed the correlation between UCK2 and drug sensitivity. Results UCK2 expression was observed to be high in most cancers and was remarkably related to the prognosis of pan-cancers. We found that the increased UCK2 expression was associated with higher genetic instability. Additionally, positive relationships were observed between UCK2 expression and mismatch repair genes, homologous recombination repair genes, and cancer stemness across different cancer types. There were significant correlations between UCK2 and T cells, monocytes, mast cells, and macrophages. Moreover, as expected, the immune checkpoint human leucocyte antigen (HLA) was found to be negatively related to UCK2. Similarly, UCK2 was also observed to have a negative association with major histocompatibility complex (MHC) genes. We noted that UCK2 had significant correlations with the sensitivity to various anti-cancer drug. Conclusion We have observed that UCK2 plays pivotal roles in prognosis and tumor immunity, and it is associated with DNA repair and cancer stemness. The UCK2 gene exhibits a strong correlation with the immune checkpoints HLA. This study highlights its potential impact on drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3d615de6acd1c023835e11b2fb04b8e2aecd39" target='_blank'>
              Uridine-cytidine kinase 2 is correlated with immune, DNA damage repair and promotion of cancer stemness in pan-cancer
              </a>
            </td>
          <td>
            Jinlong Tian, Yanlei Li, Yu Tong, Yuan Zhang, Tingxiao Zhao, Yao Kang, Qing Bi
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Motivation Understanding the factors involved in DNA double-strand break (DSB) repair is crucial for the development of targeted anti-cancer therapies, yet the roles of many genes remain unclear. Recent studies show that perturbations of certain genes can alter the distribution of sequence-specific mutations left behind after DSB repair. This suggests that genome-wide screening could reveal novel DSB repair factors by identifying genes whose perturbation causes the mutational distribution spectra observed at a given DSB site to deviate significantly from the wild-type. However, designing proper controls for a genome-wide perturbation screen could be challenging. We explore the idea that a genome-wide screen might allow us to forgo the use of traditional non-targeting controls by reframing the analysis as an outlier detection problem, assuming that most genes have minimal influence on DSB repair. Results We propose MUSICiAn (Mutational Signature Catalogue Analysis), a compositional data analysis method that ranks gene perturbation-specific mutational spectra without controls by measuring deviations from the central tendency in the distributions of all spectra. We show that MUSICiAn can effectively estimate pseudo-controls for the existing Repair-seq dataset, screening 476 genes and 60 non-targeting controls. We further apply MUSICiAn to a genome-wide dataset profiling mutational outcomes induced by CRISPR-Cas9 at three target sites across cells with individual perturbations of 18,406 genes. MUSICiAn successfully recovers known genes, highlights the spliceosome as a lesser-appreciated player in DSB repair, and reveals candidates for further investigation. Availability github.com/joanagoncalveslab/MUSICiAn.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86446b59ac2517d127c22b884f9cd2e873cbddf" target='_blank'>
              MUSICiAn: Genome-wide Identification of Genes Involved in DNA Repair via Control-Free Mutational Spectra Analysis
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d06e293f47afb8c1e77d741db124642957b624" target='_blank'>
              Characterization of RNF144B and PPP2R2A identified by a novel approach using TCGA data in ovarian cancer
              </a>
            </td>
          <td>
            P. Manasa, S. Krishnapriya, C. Sidhanth, S. Vasudevan, Kanchan Murhekar, T. S. Ganesan
          </td>
          <td>2025-02-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2339cd78386f39fd2971aafdfae213be0112c00" target='_blank'>
              Detecting higher-order chromosome interactions by polymer modeling
              </a>
            </td>
          <td>
            Mattia Conte, Simona Bianco, Andrea Esposito, Alex Abraham, Sougata Guha, Francesca Vercellone, Andrea Fontana, Florinda Di Pierno
          </td>
          <td>2025-01-28</td>
          <td>The European Physical Journal Plus</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8283805e8d954027bac5a7fd108f9580c6417e4d" target='_blank'>
              Abstract PR004: Interplay between histone modifications and DNA methylation in cancer
              </a>
            </td>
          <td>
            Chao Lu
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human fungal pathobiont Candida albicans displays extensive genomic plasticity, including large-scale chromosomal changes such as aneuploidy. Chromosome trisomy appears frequently in natural and laboratory strains of C. albicans. Trisomy of specific chromosomes has been linked to large phenotypic effects, such as increased murine gut colonization by strains trisomic for chromosome 7 (Chr7). However, studies of whole-chromosome aneuploidy are generally limited to the SC5314 genome reference strain, making it unclear whether the imparted phenotypes are conserved across C. albicans genetic backgrounds. Here, we report the presence of a Chr7 trisomy in the “commensal-like” oral candidiasis strain, 529L, and dissect the contribution of Chr7 trisomy to colonization and virulence in 529L and SC5314. These experiments show that strain background and homolog identity (i.e., AAB vs ABB) interact with Chr7 trisomy to alter commensal and virulence phenotypes in multiple host niches. In vitro filamentation was the only phenotype altered by Chr7 trisomy in similar ways across the two strain backgrounds. Oral colonization of mice was increased by the presence of a Chr7 trisomy in 529L but not SC5314; conversely, virulence during systemic infection was reduced by Chr7 trisomy in SC5314 but not 529L. Strikingly, the AAB Chr7 trisomy in the SC5314 background rendered this strain avirulent in murine systemic infection. Increased dosage of NRG1 failed to reproduce most of the Chr7 trisomy phenotypes. Our results demonstrate that aneuploidy interacts with background genetic variation to produce complex phenotypic patterns that deviate from our current understanding in the genome reference strain. AUTHOR SUMMARY Candida albicans is a clinically important fungus of humans that is also part of the microbiota that typically colonizes our bodies. Changes in the number of copies of chromosome 7 (Chr7) in the genome of the reference strain of C. albicans enhances host colonization. We identified a third copy of Chr7 in a strain classified as a good colonizer of humans that doesn’t cause disease, mimicking the increased colonization of the reference strain. Here, we demonstrate that the number of copies of Chr7 alters phenotypes differently between these strains, and three copies of Chr7 does not universally produce a strain that is only a better commensal strain. Additionally, which copy of Chr7 in this diploid species exists as the third copy influences commensal and pathogen phenotypes. Therefore, trisomic changes of the same chromosome in different C. albicans strains have substantially different effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2332e9447e18d3247be54ad40b31742ed672d8fa" target='_blank'>
              Strain background interacts with chromosome 7 aneuploidy to determine commensal and virulence phenotypes in Candida albicans
              </a>
            </td>
          <td>
            Abhishek Mishra, N. Solis, Siobhan M Dietz, Audra L Crouch, S. Filler, Matthew Z. Anderson
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f45e736911a5471bab3e71faffe39b31598773a" target='_blank'>
              IER3: exploring its dual function as an oncogene and tumor suppressor.
              </a>
            </td>
          <td>
            M. Kanduri, Santhilal Subhash, Rossana Putino, Sagar Mahale, C. Kanduri
          </td>
          <td>2025-03-16</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d71f4d600cde047c309be7d9f03d4993b2ab30" target='_blank'>
              EXO1's pan-cancer roles: diagnostic, prognostic, and immunological analyses through bioinformatics
              </a>
            </td>
          <td>
            Zheng Liu, Shu Huang, Rui Luo, Xiaomin Shi, Mingzhu Xiu, Yizhou Wang, Ruiyu Wang, Wei Zhang, Muhan Lv, Xiaowei Tang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5624b8ad84bcb73e88ab4dda70b0e6af9169b6fa" target='_blank'>
              Nucleosome Engagement Regulates RORγt Structure and Dynamics
              </a>
            </td>
          <td>
            Timothy S. Strutzenberg, Matthew D. Mann, Xiandu Li, Hyejeong Shin, Jordan Kelsey, S. Aiyer, Jingting Yu, Gennavieve Gray, Zeyuan Zhang, Zelin Shan, Bo Zhou, Ye Zheng, Patrick R. Griffin, Dmitry Lyumkis
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Ovarian cancer (OC) is a highly heterogeneous malignancy, often characterized by complex genomic alterations that drive tumor progression and therapy resistance. In this paper, we report a novel de novo BRCA2 germline variant NM_000059.3:c.(8693_8695delinsGT) associated with early-onset OC that featured two regions with differential MMR (Mismatch Repair) gene expression. To date, only six cases of de novo BRCA2 variants have been reported, none of which were associated with early-onset high-grade serous OC. The immunohistochemical analysis of MMR genes revealed two distinct tumor areas, separated by a clear topographic boundary, with the heterogeneous expression of MLH1 and PMS2 proteins. Seventy-five percent of the tumor tissue showed positivity, while the remaining 25% exhibited a complete absence of expression, underscoring the spatial variability in MMR gene expression within the tumor. Integrated comparative spatial genomic profiling identified several tumor features associated with the genetic variant as regions of loss of heterozygosity (LOH) that involved BRCA2 and MLH1 genes, along with a significantly higher mutational tumor burden in the tumor area that lacked MLH1 and PMS2 expression, indicating its further molecular evolution. The following variants were acquired: c.6572C>T in NOTCH2, c.1852C>T in BCL6, c.191A>T in INHBA, c.749C>T in CUX1, c.898C>A in FANCG, and c.1712G>C in KDM6A. Integrated comparative spatial proteomic profiles revealed defects in the DNA repair pathways, as well as significant alterations in the extracellular matrix (ECM). The differential expression of proteins involved in DNA repair, particularly those associated with MMR and Base Excision Repair (BER), highlights the critical role of defective repair mechanisms in driving genomic instability. Furthermore, ECM components, such as collagen isoforms, Fibrillin-1, EMILIN-1, Prolargin, and Lumican, were found to be highly expressed in the MLH1/PMS2-deficient tumor area, suggesting a connection between DNA repair deficiencies, ECM remodeling, and tumor progression. Thus, the identification of the BRCA2 variant sheds light on the poorly understood interplay between DNA repair deficiencies and ECM remodeling in OC, providing new insights into their dual role in shaping tumor evolution and suggesting potential targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3d22f654834c47296af7f38cdf9042ad40753" target='_blank'>
              Novel De Novo BRCA2 Variant in an Early-Onset Ovarian Cancer Reveals a Unique Tumor Evolution Pathway
              </a>
            </td>
          <td>
            G. Miolo, Giovanni Canil, Maurizio Polano, M. Dal Bo, Alessia Mondello, Antonio Palumbo, F. Puglisi, Giuseppe Corona
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf439585ea0599081bd08309bf4216f355afcfa4" target='_blank'>
              B cell stimulation changes the structure and higher-order organization of the inactive X chromosome
              </a>
            </td>
          <td>
            Isabel Sierra, Natalie E. Toothacre, Robin H. van der Weide, Claudia D. Lovell, Son C. Nguyen, R. Jordan Barnett, Ashley L. Cook, Han-Seul Ryu, Sarah Pyfrom, Harrison Wang, D. Beiting, Jennifer E. Philips-Cremins, Eric F. Joyce, Montserrat C. Anguera
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, and mutation hotspots in human cancer genomes co-localize with alternative DNA structure-forming sequences (e.g. H-DNA), implicating them in cancer etiology. H-DNA has been shown to stimulate genetic instability in mammals. Here, we demonstrate a new paradigm of genetic instability, where a cancer-associated H-DNA-forming sequence accumulates more oxidative lesions than B-DNA under conditions of oxidative stress (OS), often found in tumor microenvironments. We show that OS results in destabilization of the H-DNA structure and attenuates the fold increase in H-DNA-induced mutations over control B-DNA in mammalian cells. Furthermore, the mutation spectra revealed that the damaged H-DNA-containing region was processed differently compared to H-DNA in the absence of oxidative damage in mammalian cells. The oxidatively modified H-DNA elicits differential recruitment of DNA repair proteins from both the base excision repair and nucleotide excision repair mechanisms. Altogether, these results suggest a new model of genetic instability whereby H-DNA-forming regions are hotspots for DNA damage in oxidative microenvironments, resulting in its altered mutagenic processing. Our findings provide valuable insights into the role of OS in DNA structure-induced genetic instability and may establish H-DNA-forming sequences as promising genomic biomarkers and potential therapeutic targets for genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560e3fa66aab00812738293c545080a843274a94" target='_blank'>
              Oxidative damage within alternative DNA structures results in aberrant mutagenic processing
              </a>
            </td>
          <td>
            Maha Zewail-Foote, Imee M A Del Mundo, Alex W. Klattenhoff, Karen M. Vasquez
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The Hippo/YAP1 signaling pathway regulates normal development by controlling contact inhibition of growth. In cancer, YAP1 activation is often dysregulated, leading to excessive tumor growth and metastasis. SRC kinase can cross talk to Hippo signaling by disrupting adherens junctions, repressing the Hippo cascade, or activating YAP1 to promote proliferation. Here, we demonstrate that the IGF2 messenger RNA-binding protein 1 (IGF2BP1) impedes the repression of YAP1 by Hippo signaling in carcinomas. IGF2BP1 stabilizes the YAP1 messenger RNA (mRNA) and enhances YAP1 protein synthesis through an m6A-dependent interaction with the 3′ untranslated region of the YAP1 mRNA, thereby increasing YAP1/TAZ-driven transcription to bypass contact inhibition of tumor cell growth. Inhibiting IGF2BP1–mRNA binding using BTYNB reduces YAP1 levels and transcriptional activity, leading to significant growth inhibition in carcinoma cells and ovarian cancer organoids. In contrast, SRC inhibition with Saracatinib fails to inhibit YAP1/TAZ-driven transcription and cell growth in general. This is particularly significant in de-differentiated, rather mesenchymal carcinoma-derived cells, which exhibit high IGF2BP1 and YAP1 expression, rendering them less reliant on SRC-directed growth stimulation. In such invasive carcinoma models, the combined inhibition of SRC, IGF2BP1, and YAP1/TAZ proved superior over monotherapies. These findings highlight the therapeutic potential of targeting IGF2BP1, a key regulator of oncogenic transcription networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16383314a8de10bc799de0f2ecf7435c95cb185c" target='_blank'>
              The IGF2BP1 oncogene is a druggable m6A-dependent enhancer of YAP1-driven gene expression in ovarian cancer
              </a>
            </td>
          <td>
            Annekatrin Schott, Theresa Simon, Simon Müller, Alexander Rausch, Bianca Busch, Markus Glaß, Danny Misiak, Mohammad Dipto, Hend Elrewany, Lara Meret Peters, Sunita Tripathee, Ehab Ghazy, Florian Müller, R. Rolnik, M. Lederer, Ali Hmedat, Martina Vetter, Markus Wallwiener, W. Sippl, S. Hüttelmaier, N. Bley
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f296dc7b10466d316234339cf9617d0c3278a601" target='_blank'>
              DNA-PKcs, a player winding and dancing with RNA metabolism and diseases
              </a>
            </td>
          <td>
            Jiabao Hou, Mingjun Lu, Jingwei Guo, Jinghong Wu, Chenyang Wang, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2025-03-04</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8599e8314fc9026e2db49b39f76e5203a6d7080" target='_blank'>
              Allele-specific genomics decodes gene targets and mechanisms of the non-coding genome
              </a>
            </td>
          <td>
            Tim P. Hasenbein, Sarah Hoelzl, S. Engelhardt, Daniel Andergassen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The sequence-driven organising principles of the 3D genome are crucial for interpreting the core effects of genomic variation and for understanding the evolution of genome organisation and function. We investigated these by isolating and analysing cell-type-persistent contacts, heavily dependent on the similarly cell-type-persistent genomic sequence. We stratified long-range contacts from a diverse group of human tissues and cell lines based on contact persistence, cp, reflecting their presence across cell or tissue types, presenting them as an atlas of contacts and the cell-type invariant (CETI) hubs they form across human chromosomes. Our survey of more than 300 chromatin and genome features revealed their association with cp, contrasting variable from persistent contacts in terms of co-localisation with genes, 3D architectural domains, epigenetic and sequence elements. We found persistent contacts to be predominantly comprised of AT-rich sequences and related to heterochromatin. A key outcome is finding a link between the experimental genomic contacts and the complementarity between pairs of contacting DNA loci. This work provides evidence for a sequence determinant of genomic contacts contributing to the decoding of the relationship between sequence and structure that is crucial for functional and evolutionary studies concerning the 3D genome organisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9e662847ce0652a3c57525411bd551072e0b44" target='_blank'>
              Analysis of long-range contacts across cell types outlines a core sequence determinant of 3D genome organisation
              </a>
            </td>
          <td>
            Liezel Tamon, Zahra Fahmi, James Ashford, Rosana Collepardo-Guevara, Aleksandr B. Sahakyan
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Garra rufa, commonly known as doctor fish, is a small freshwater cyprinid fish recognized for its high-temperature tolerance. G. rufa is primarily known for its role in ichthyotherapy, in which it helps remove keratinized skin from human hands and feet. However, in recent years, its high-temperature tolerance has attracted attention owing to its potential use as a model for human disease research, including infectious diseases and cancer xenograft models. Despite this interest, the genomic basis underlying high-temperature tolerance remains largely unexplored, primarily because of the limited availability of genomic resources, which hinders its development as an experimental model. In the present study, a high-quality chromosome-level genome assembly of G. rufa was generated using a combination of PacBio HiFi long-read sequencing and Hi-C technology. We generated a chromosome-level genome assembly with 25 chromosomes, 1.38 Gb in total length, and scaffold N50 of 49.3 Mb. Approximately 59% of the genome comprises repetitive elements, with DNA and LTR elements being particularly abundant. In total, 27,352 protein-coding genes were annotated, of which 26,900 genes (98.3%) were functionally annotated. Benchmarking Universal Single-Copy Orthologs (BUSCO) benchmark for genome assembly and gene annotation demonstrated 94.5% and 94.7% of complete BUSCOs, respectively. We identified two heat shock transcription factor (HSF) and 239 heat shock protein (HSP)-related genes. Heat shock elements, which are HSF-binding motifs, were present within 3 kb upstream of 944 genes, with statistically significant enrichment of HSP-related genes in this set. Furthermore, molecular phylogenetic analysis and whole-genome comparisons revealed that G. rufa is evolutionarily closely related to species in the Labeoninae subfamily of the Cyprinidae family. This chromosome-level reference genome provides a valuable resource for future research aimed at elucidating the molecular mechanisms underlying high-temperature tolerance in G. rufa and establishing it as a model organism for human biomedical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba084e3b18995a64de9db1867a75cd406732e5f" target='_blank'>
              Chromosome-level genome assembly of the doctor fish (Garra rufa)
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Kiki Syaputri Handayani, Liqing Zang, Fahrurrozi, Oleg Simakov, Victor David, Nico Gultom, Yasuhito Shimada
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Mitoregulin (MTLN) is a 56-amino-acid mitochondrial microprotein known to modulate mitochondrial energetics. MTLN gene expression is elevated broadly across most cancers and has been proposed as a prognostic biomarker for non-small cell lung cancer (NSCLC). In addition, lower MTLN expression in lung adenocarcinoma (LUAD) correlates with significantly improved patient survival. In our studies, we have found that MTLN silencing in A549 NSCLC cells slowed proliferation and, in accordance with this, we observed the following: (1) increased proportion of cells in the G1 phase of cell cycle; (2) protein changes consistent with G1 arrest (e.g., reduced levels and/or reduced phosphorylation of ERK, MYC, CDK2, and RB, and elevated p27Kip1); (3) reduction in clonogenic cell survival and; (4) lower steady-state cytosolic and mitochondrial H2O2 levels as indicated by use of the roGFP2-Orp1 redox sensor. Conflicting with G1 arrest, we observed a boost in cyclin D1 abundance. We also tested MTLN silencing in combination with buthionine sulfoximine (BSO) and auranofin (AF), drugs that inhibit GSH synthesis and thioredoxin reductase, respectively, to elevate the reactive oxygen species (ROS) amount to a toxic range. Interestingly, clonogenic survival after drug treatment was greater for MTLN-silenced cultures versus the control cultures. Lower H2O2 output and reduced vulnerability to ROS damage due to G1 status may have jointly contributed to the partial BSO + AF resistance. Overall, our results provide evidence that MTLN fosters H2O2 signaling to propel G1/S transition and suggest MTLN silencing as a therapeutic strategy to limit NSCLC growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728830e41932164b648250623dd9f7dc576066d5" target='_blank'>
              Mitoregulin Promotes Cell Cycle Progression in Non-Small Cell Lung Cancer Cells
              </a>
            </td>
          <td>
            Colleen S. Stein, Connor R. Linzer, Collin D. Heer, Nathan H. Witmer, Jesse D. Cochran, D. R. Spitz, Ryan L. Boudreau
          </td>
          <td>2025-02-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, with its incidence and mortality rates projected to rise due to population aging. In this study, we utilized PCa transcriptome data from public databases and applied bioinformatics methods to identify three prognostic genes (CDC20, RAD51, and TTK) related to centrosome duplication in PCa. CDC20 is involved in cell cycle regulation, RAD51 in deoxyribonucleic acid double-strand break repair, and TTK in spindle assembly checkpoint function and cell proliferation. We constructed a risk model and a nomogram model, both demonstrating moderate to good predictive performance with area under the curve values ranging from 0.611 to 0.765 at different time points. Gene set enrichment analysis revealed that these genes were enriched in 64 pathways, including the cell cycle pathway, which is dysregulated in cancer. Furthermore, we analyzed the immune microenvironment and identified 13 differential immune cells and 13 differential immune checkpoints between high- and low-risk groups, providing insights into potential immunotherapy targets for PCa. In conclusion, this study contributes to a deeper understanding of PCa pathogenesis and lays important theoretical and experimental foundations for developing new diagnostic markers and treatment strategies. Future research requires more clinical samples and continued monitoring of the mechanism of these genes in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8581bf4b2f1cfe894fadfd64d88a1382b8cc282" target='_blank'>
              Prognostic Value of Centrosome Replication-Related Genes in Prostate Cancer Based on Transcriptomic and Mendelian Randomization
              </a>
            </td>
          <td>
            Qizhong Lu, Yufan Wu, Qiwei Yu, Jun Ouyang
          </td>
          <td>2025-03-01</td>
          <td>American Journal of Men's Health</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24649fa8cee2fc610da3934361ee2a3da1b227a9" target='_blank'>
              Ku suppresses RNA-mediated innate immune responses in human cells to accommodate primate-specific Alu expansion
              </a>
            </td>
          <td>
            Yimeng Zhu, Angelina Li, Suvrajit Maji, Brian J. Lee, Sophie M. Korn, Jake A. Gertie, Tyler J. Dorrity, Jianhua Wang, Kyle J. Wang, Amandine Pelletier, Daniel F. Moakley, Rachel D. Kelly, Antony B. Holmes, Raul Rabadan, D. Edgell, Caroline Schild Poulter, Mauro Modesti, Anna-Lena Steckelberg, Eric A. Hendrickson, Hachung Chung, Chaolin Zhang, Shan Zha
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2140165b325769cf3794c12431eadebdd635d22" target='_blank'>
              Chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Nina Trautwein, Bodo Laube, Christian Dietz, Robert W. Stark, M. C. Cardoso
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23fd0fe606d16164cbf75d226bfad5eec1b690f" target='_blank'>
              Complex determinants of R-loop formation at transposable elements and major DNA satellites.
              </a>
            </td>
          <td>
            Timothy J. Stanek, Adam Kneebone, Matthew A Lawlor, Weihuan Cao, Christopher E Ellison
          </td>
          <td>2025-02-28</td>
          <td>Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106d9fd456a3ff5a33ecce59e35a5f050b56ab0e" target='_blank'>
              RECQL4 requires PARP1 for recruitment to DNA damage, and PARG dePARylation facilitates its associated role in end joining
              </a>
            </td>
          <td>
            Mansoor Hussain, Prabhat Khadka, Komal Pekhale, Tomasz Kulikowicz, Samuel Gray, A. May, D. Croteau, V. A. Bohr
          </td>
          <td>2025-01-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c7a49a64fc95173d062b597a609f1c34ffa6ed" target='_blank'>
              PIF/harbinger transposon-derived protein promotes 7SL expression to enhance pathogen resistance
              </a>
            </td>
          <td>
            S. Geng, Xing Lv, T. Xu
          </td>
          <td>2025-01-30</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background. Colorectal carcinomas (CRCs) are seldom eradicated by cytotoxic chemotherapy. Cancer cells with stem-like functional properties, often referred to as “cancer stem cells” (CSCs), display preferential resistance to several anti-tumor agents used in cancer chemotherapy, but the molecular mechanisms underpinning their selective survival remain only partially understood. Methods. In this study, we used Transcription Factor Target Genes (TFTG) enrichment analysis to identify transcriptional regulators (activators or repressors) that undergo preferential activation by chemotherapy in CRC cells with a “bottom-of-the-crypt” phenotype (EPCAM+/CD44+/CD166+; CSC-enriched) as compared to CRC cells with a “top-of-the-crypt” phenotype (EPCAM+/CD44neg/CD166neg; CSC-depleted). The two cell populations were purified in parallel by fluorescence-activated cell sorting (FACS) from a patient-derived xenograft (PDX) line representative of a moderately differentiated human CRC, following in vivo chemotherapy with irinotecan (CPT-11). The transcriptional regulators identified as differentially activated were tested for differential expression in normal vs. cancer tissues, and in cell populations enriched in stem/progenitor cell-types as compared to differentiated lineages (goblet cells, enterocytes) in the mouse colon epithelium. Finally, the top candidate was tested for mechanistic contribution to drug-resistance by selective down-regulation using short-hairpin RNAs (shRNAs). Results. Our analysis identified E2F4 and TFDP1, two core components of the DREAM transcriptional repression complex, as transcriptional modulators preferentially activated by irinotecan in EPCAM+/CD44+/CD166+ as compared to EPCAM+/CD44neg/CD166neg cancer cells. The expression levels of both genes (E2F4, TFDP1) were found up-regulated in CRCs as compared to human normal colon tissues, and in a sub-population of mouse colon epithelial cells enriched in stem/progenitor elements (Epcam+/Cd44+/Cd66alow/Kitneg) as compared to other sub-populations enriched in either goblet cells (Epcam+/Cd44+/Cd66alow/Kit+) or enterocytes (Epcam+/Cd44neg/Cd66ahigh). Most importantly, E2F4 down-regulation using shRNAs dramatically enhanced the sensitivity of human CRCs to in vivo treatment with irinotecan, across three independent PDX models. Conclusions. Our data identified E2F4 and the DREAM repressor complex as critical regulators of human CRC resistance to irinotecan, and as candidate targets for the development of chemo-sensitizing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3c8fee4cf42a4e76a7d265977fa7fcfba7541" target='_blank'>
              The E2F4 transcriptional repressor is a key mechanistic regulator of colon cancer resistance to irinotecan (CPT-11)
              </a>
            </td>
          <td>
            Junichi Matsubara, Y. F. Li, Sanjay Koul, Junko Mukohyama, Luis E. Valencia Salazar, Taichi Isobe, D. Qian, Michael F. Clarke, Debashis Sahoo, R. Altman, Piero Dalerba
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Scriptaid is a chemical compound with anti-tumoural effects due to its role as a histone deacetylase inhibitor. Despite sharing part of the chemical structure with other ligands of G-quadruplexes (G4s), the interaction of Scriptaid with G4s has not been explored before. We synthesized Scriptaid and screened its cytotoxic activity in cellular models of colorectal cancer (CRC). We extensively evaluated its biological activity by cell cycle, immunofluorescence, qRT-PCR and Western blot experiments. To identify the G4 targets of Scriptaid, we conducted a panel of binding assays. Here, we show that Scriptaid induced cytotoxicity, cell cycle arrest and nucleolar stress in CRC cells. Moreover, Scriptaid impaired RNA polymerase I (Pol I) transcription, stabilized G4s and caused DNA damage. Finally, we disclose that these effects were attributable to the binding of Scriptaid to G4s in ribosomal DNA. In conclusion, our work reveals that a primary impact of Scriptaid on human cells is the interaction with G4s.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650a5d0a9e8fbb2585c46607a446ad7a0c9f6bc" target='_blank'>
              The histone deacetylase inhibitor Scriptaid targets G-quadruplexes
              </a>
            </td>
          <td>
            Victoria Sánchez-Martín, Dušan Ružić, María J Tello-López, Andrea Ortiz-Morales, Javier Murciano-Calles, M. Soriano, Katarina Nikolić, José Antonio García-Salcedo
          </td>
          <td>2025-02-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Resistance to radiotherapy remains a critical barrier in treating colorectal cancer (CRC), particularly in cases of locally advanced rectal cancer (LARC). To identify key kinases involved in CRC radioresistance, we employed a kinase‐targeted CRISPR‐Cas9 library screen. This approach aimed to identify potential kinase inhibitors as radiosensitizers. Our screening identified salt‐inducible kinase 2 (SIK2) as a critical factor in CRC radioresistance. Increased SIK2 expression correlated with reduced tumor regression and poorer outcomes in LARC patients undergoing neoadjuvant chemoradiotherapy. The depletion of SIK2 significantly enhanced radiation‐induced apoptosis and tumor regression. Mechanistically, SIK2 interacts with valosin‐containing protein (VCP), promoting its hyperphosphorylation. This modification improves VCP's capacity to extract K48‐linked ubiquitin‐conjugated proteins from chromatin, thus aiding the recruitment of RPA and RAD51 to DNA damage sites. This mechanism strengthens homologous recombination–mediated DNA repair, which contributes to radioresistance. Importantly, ARN‐3236, a SIK2 inhibitor, markedly sensitized CRC cells to radiation both in vivo and in vitro, providing a potential strategy to overcome radioresistance. In summary, our findings reveal a novel mechanism by which SIK2 contributes to the radioresistance of CRC, proposing SIK2 as a potential therapeutic target with its inhibitor significantly enhancing CRC radiotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27d0bffcf12a04202c03cb4fe50e95cbe2133af6" target='_blank'>
              Salt‐inducible kinase 2 confers radioresistance in colorectal cancer by facilitating homologous recombination repair
              </a>
            </td>
          <td>
            Yuan Meng, Shuo Li, Da‐Shan Lu, Xue Chen, Lu Li, You‐fa Duan, Gao‐yuan Wang, Wenlin Huang, Ran-Yi Liu
          </td>
          <td>2025-01-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57cf07128fb65aaad9c55c65c3a93161bd10c47" target='_blank'>
              LINE-1 Retrotransposons as Cell-free DNA Biomarkers for Multi-Cancer Early Detection.
              </a>
            </td>
          <td>
            J. Szymanski, Pradeep S Chauhan, A. Chaudhuri
          </td>
          <td>2025-01-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d82ec08c9ad540baf17cea17836bedb8e4f45c6" target='_blank'>
              Chemical inhibition of Exon Junction Complex assembly impairs mRNA localization and neural stem cells ciliogenesis
              </a>
            </td>
          <td>
            Tommaso Villa, Oriane Pourcelot, D. Dierks, Marion Faucourt, Cindy Burel, Floric Slimani, Léa Guyonnet, Nathalie Spassky, Schraga Schwartz, Edouard Bertrand, Olivier Bensaude, H. Le Hir
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653fd5d6406b6894ce87859c219648b7e181e16b" target='_blank'>
              Sequence-dependent activity and compartmentalization of foreign DNA in a eukaryotic nucleus.
              </a>
            </td>
          <td>
            Léa Meneu, Christophe Chapard, Jacques Serizay, Alex Westbrook, Etienne Routhier, Myriam Ruault, Manon Perrot, Alexandros Minakakis, Fabien Girard, Amaury Bignaud, Antoine Even, G. Gourgues, Domenico Libri, C. Lartigue, Aurèle Piazza, Agnès Thierry, Angela Taddei, Frédéric Beckouët, J. Mozziconacci, Romain Koszul
          </td>
          <td>2025-02-07</td>
          <td>Science</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecfa6b604141f203a1f4a0229bf2941af19cc8c" target='_blank'>
              Multiple functions of the ALT favorite helicase, BLM
              </a>
            </td>
          <td>
            Shun Chang, Jiang Tan, Ren Bao, Yanduo Zhang, Jinkai Tong, Tongxin Jia, Jing Liu, Juhua Dan, Shuting Jia
          </td>
          <td>2025-03-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Traditional drug discovery efforts primarily target rapid, reversible protein-mediated adaptations to counteract cancer cell resistance. However, cancer cells also utilize DNA-based strategies, often perceived as slow, irreversible changes like point mutations or drug-resistant clone selection. Extrachromosomal DNA (ecDNA), in contrast, represents a rapid, reversible, and predictable DNA alteration critical for cancer’s adaptive response. Methods In this study, we developed a novel post-processing pipeline for automated detection and quantification of ecDNA in metaphase Fluorescence in situ Hybridization (FISH) images, leveraging the Microscopy Image Analyzer (MIA) tool. This pipeline is tailored to monitor ecDNA dynamics during drug treatment. Results Our approach effectively quantified ecDNA changes, providing a robust framework for analyzing the adaptive responses of cancer cells under therapeutic pressure. Discussion The pipeline not only serves as a valuable resource for automating ecDNA detection in metaphase FISH images but also highlights the role of ecDNA in facilitating swift and reversible adaptation to epigenetic remodeling agents such as JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed078aa348654cdad2d2c2164c322729a574c316" target='_blank'>
              Leveraging AI to automate detection and quantification of extrachromosomal DNA to decode drug responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Christina Ford, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, S. Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Rationale: Tumor cell epigenetics, especially chromosome accessibility, has been reported to be closely related to the tumor immune landscape and immunotherapy. However, the exact mechanism remains unknown. Methods: Whole-exome sequencing was used to analyze 13 colorectal tumor samples treated with PD1 immunotherapy. The assays for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing were used to detect tumor cells' chromosome accessibility status and screen regulatory pathways. Results: Polybromo-1 (PBRM1) was one of the 12 genes with the highest frequency of somatic mutations associated with immunotherapy sensitivity. PBRM1/Pbrm1 deficiency in colorectal cancer promoted PD-1 immunotherapy sensitivity and chemotaxis of CD8+ T and NK cells in the microenvironment in vivo and in vitro. ATAC sequencing revealed that deletion of Pbrm1, a critical component of the SWI/SNF complex, increased chromosomal accessibility in tumor cells and triggered the release of cytokines, such as CCL5 and CXCL10, by activating the NF-κB signaling pathway. Application of ACBL1, a PROC inhibitor of PBRM1, in BALB/C mice or colorectal patient-derived tumor organoids (PDTOs) significantly promoted the sensitivity to PD1 antibody immunotherapy. Conclusions: Our study established that PBRM1/Pbrm1 deficiency was positively correlated with PD1 immunotherapeutic sensitivity in colorectal cancer. The underlying molecular mechanisms involved regulation of chromosome accessibility, activation of the NF-κB signaling pathway, and immune cell infiltration in the microenvironment. These findings identify potential molecular targets for enhancing immunotherapy for colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7444e0c65d77850e20585c8a5ec38a58620ecd" target='_blank'>
              PBRM1 deficiency enhances PD1 immunotherapeutic sensitivity via chromosomal accessibility in colorectal cancer
              </a>
            </td>
          <td>
            Rui Li, Jie He, Chaoqun Liu, Zesheng Jiang, Jiasheng Qin, Kun Liang, Zhuocheng Ji, Liang Zhao
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d94723cd3fd533169f04d9798f438ae76e7b03eb" target='_blank'>
              TRF2 Non-Telomeric Function is Indispensable for Neural stemness
              </a>
            </td>
          <td>
            Soujanya Vinayagamurthy, Amit Kumar Bhatt, Sulochana Bagri, Supratim Ghosh, Arpan Parichha, Arindam Maitra, Shantanu Chowdhury
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERV) represent nearly 8% of the human genome. Of these, HERV-K subtype HML-2 is a transposable element that plays a critical role in embryonic development and in the pathogenesis of several diseases. Quantification and characterization of these multiple HML-2 insertions in the human chromosome has been challenging due to their size, sequence homology with each other, and their repetitive nature. We examined a cohort of 222 individuals for HML-2 proviruses 6q14.1 and 7p22.1a, two loci that are capable of producing full-length viral proteins and have been previously implicated in several cancers, autoimmune disorders and neurodegenerative diseases, using long-read DNA sequencing. While the reference genome for both regions suggests these two loci are structurally dissimilar, we found that for both loci about 5% of individuals have a unique tandem repeat-like sequence (three long terminal repeat sequences sandwiching two internal, potentially protein coding sequences), while most individuals have a standard proviral structure (one internal region sandwiched by two long terminal repeats). Moreover, both proviruses can make full-length, or nearly full-length, HERV-K proteins in multiple transcription orientations. The amino acid sequences from different loci in the same transcriptional orientation share sequence homology with each other. These results demonstrate a clear, previously unreported, relationship between HML-2 loci 6q14.1 and 7p22.1a and highlight the utility of long-read sequencing to study repetitive elements. Future studies need to determine if these polymorphisms determine genetic susceptibility to diseases that are associated with them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd67bb0a75d66d24c8cfbfff72aa27068e59aecc" target='_blank'>
              Characterization of novel human endogenous retrovirus structures on chromosomes 6 and 7
              </a>
            </td>
          <td>
            Nicholas Pasternack, Ole Paulsen, Avindra Nath
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee7c5cbf5b3479ea3f3880a032c93a1d4cc7dfa8" target='_blank'>
              DUBR non-coding RNA regulates gene expression by affecting AP-1 enhancer accessibility.
              </a>
            </td>
          <td>
            Simone D Hall, Khoa Tran, Jonathan Zhu, Tong Su, Colleen A. McHugh
          </td>
          <td>2025-03-21</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f33e9da345df8224f97295c4e7efc05d28c012" target='_blank'>
              Crosstalk Between Plk1 and PTEN in Mitosis Affects Chromosomal Stability.
              </a>
            </td>
          <td>
            Wei Li, Xianning Wang, Jiannan Liu, Bing Liu, Yongjian Hao
          </td>
          <td>2025-03-21</td>
          <td>DNA and cell biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fca76d81501556f6f871b8db3defb5ed37bf0b72" target='_blank'>
              Loss of BCL7A Permits IRF4 Transcriptional Activity and Cellular Growth in Multiple Myeloma.
              </a>
            </td>
          <td>
            C. Chakraborty, S. Talluri, Moritz Binder, E. Morelli, Jessica Encinas Mayoral, Sanika Derebail, Anil Aktas Samur, Charles B Epstein, Kenneth C Anderson, M. Shammas, M. Samur, M. Fulciniti, Nikhil C Munshi
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4b38200a1231a3f3c1164315cc7d10135fa36f" target='_blank'>
              Redirecting resistance evolution in BRAFV600 melanoma by inhibition of the peroxiredoxin-thioredoxin system
              </a>
            </td>
          <td>
            Stefanie Egetemaier, H. Chauvistré, R. Váraljai, Yichao Hua, Smiths S. Lueong, Samira Makhzami, Nalini Srinivas, Jan Forster, V. Ullrich, Simone Stupia, Valéria Schroeder, Sarah Scharfenberg, A. Hoewner, B. Shannan, J. Siveke, M. Baietti, Eleonora Leucci, J. Marine, A. Paschen, Bjoern Scheffler, D. R. Engel, Lisa M. Becker, Felix Nensa, Johannes Koester, Barbara Grünwald, S. Poepsel, Sabrina Ninck, F. Kaschani, Dirk Schadendorf, Juergen C. Becker, A. Tasdogan, Florian Rambow, Alexander Roesch
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, with prostate cancer, lung cancer, colorectal cancer, and breast cancer accounting for nearly half of all diagnoses. Despite advancements in cancer treatment, metastasis to distant organs continues to be the leading cause of cancer-related mortality. The progression of cancer involves the alteration of numerous genes, with dynamic changes in chromatin organization and histone modifications playing a critical role in regulating cancer-associated genes. Special AT-rich sequence-binding protein 1 (SATB1), a critical chromatin organizer, plays a pivotal role in cancer progression by regulating gene expression, chromatin remodeling, and cell signaling pathways. SATB1 binds to AT-rich DNA sequences, acting as a scaffold for chromatin-modifying enzymes and transcription factors, thus coordinating the regulation of extensive gene networks. Its overexpression has been implicated in a wide range of cancers and is associated with poor prognosis, aggressive tumor phenotypes, and enhanced epithelial–mesenchymal transition (EMT). Moreover, SATB1’s activity is modulated by microRNAs (miRNAs) and post-translational modifications, further contributing to its complex regulatory functions. Given its crucial involvement in cancer progression and metastasis, SATB1 has emerged as a promising target for novel therapeutic strategies. This review delves into the molecular mechanisms of SATB1 in cancer and explores potential therapeutic approaches for targeting this key regulator in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b48f08c648e41770b775be56070039b081ae8" target='_blank'>
              SATB1 in cancer progression and metastasis: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Jinping Bai, Gege Yang, Qi Yu, Qianya Chi, Xianlu Zeng, Wenjing Qi
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Topoisomerase III-beta (Top3b) reduces nucleic acid torsional stress and intertwining generated during RNA and DNA metabolism while protecting the genome from pathological R-loops, which otherwise result in DNA breakage and genome instability. By studying Top3b knockout mice (Top3b-KO), we find that the loss of Top3b accelerates the development of spontaneous lymphoid tumors arising in spleens and lymph nodes, the organs with prominent Top3b expression. Aging Top3b-KO mice also display splenomegaly and systemic immune alterations including neutrophilia and lymphopenia suggestive of chronic inflammation. At the molecular level, Top3b deficiency causes genome-wide R-loop accumulation in splenocytes as measured by CUT&Tag sequencing. Increased R-loops is associated with genomic DNA breaks and activation of immune signaling pathways including the IL-6 signaling, interleukin-7 signaling and cGAS-STING. Moreover, knocking-out Top3b promotes the rapid development of syngeneic EL4 T-cell lymphomas. In conclusion, our work implies that, in addition to its role in preserving the nervous system, Top3b protects from tumorigenesis and immune dysregulations. Significance Topoisomerase III-beta (TOP3B) is the only dual RNA-DNA topoisomerase. Its inactivation has previously been shown to be critical for neurodevelopmental disorders in humans and R-loop suppression in cell line models. Here, we demonstrate that loss of Top3b accelerates spontaneous lymphomagenesis and promotes syngeneic lymphoma growth in association with innate immune defects and R-loop accumulation in Top3b-KO mice. These findings provide evidence for a previously unappreciated role of Top3b as a tumor suppressor gene and immune regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670788860a0acb20f381f0101f5e0bb53a8bd0df" target='_blank'>
              Topoisomerase IIIβ protects from tumorigenesis and immune dysregulation
              </a>
            </td>
          <td>
            Rasel Al Mahmud, S. Baechler, A. Dhall, Sourav Saha, Hongliang Zhang, Shuling Zhang, , Nahoko Sato, Shraddha Rastogi, Suresh Kumar, Muhammad S. Alam, L. Saha, Beverly A. Mock, Valentina M Factor, Y. Pommier
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A prominent feature of eukaryotic chromosomes are centromeres, which are specialized regions of repetitive DNA required for faithful chromosome segregation during cell division. In interphase cells centromeres are non-randomly positioned in the three-dimensional space of the nucleus in a cell-type specific manner. The functional relevance and the cellular mechanisms underlying this observation are unknown, and quantitative methods to measure distribution patterns of centromeres in 3D space are needed. Here we have developed an analytical framework that combines robust clustering metrics and advanced modeling techniques for the quantitative analysis of centromere distributions at the single cell level. To identify a robust quantitative measure for centromere clustering, we benchmarked six metrics for their ability to sensitively detect changes in centromere distribution patterns from high-throughput imaging data of human cells, both under normal conditions and upon experimental perturbation of centromere distribution. We find that Ripley’s K Score has the highest accuracy with minimal sensitivity to variations in centromeres number, making it the most suitable metric for measuring centromere distributions. As a complementary approach, we also developed and validated spatial models to replicate centromere distribution patterns, and we show that a radially shifted Gaussian distribution best represents the centromere patterns seen in human cells. Our approach creates tools for the quantitative characterization of spatial centromere distributions with applications in both targeted studies of centromere organization as well as in unbiased screening approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b99c12d3169a7c41eecb80d8015ad55f9b0ac14" target='_blank'>
              Simulation and quantitative analysis of spatial centromere distribution patterns
              </a>
            </td>
          <td>
            Adib Keikhosravi, Krishnendu Guin, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e5a35f02b297af4af1d7c2312aeb3f7e720c6f2" target='_blank'>
              p53-dependent chromatin relaxation is required for DNA double-strand break repair.
              </a>
            </td>
          <td>
            Hongyu Chen, Jin Shan, Wenjing Qi, Lili Chen, Xianlu Zeng
          </td>
          <td>2025-02-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a73bead74e9165b080991fb6c70e4def4cd0e8e6" target='_blank'>
              ABL Kinases Modulate EZH2 Phosphorylation and Signaling in Metastatic Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Ashley Colemon, Tatiana Prioleau, Clay Rouse, A. Pendergast
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9667c9a72868a1c9d7c6fb195bb8c2fd73a80a" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification
              </a>
            </td>
          <td>
            Jane E. Blackmer, Erin A. Jezuit, Archan Chakraborty, Ruth A. Montague, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, Tian-Li Wang, I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb9686b4a983fe402a837ae9d2a07843ebb7fe" target='_blank'>
              Knockdown of DNAJC12 slows tumor progression and affects tumor radiosensitivity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiao Ju, Jianbo Zhang, Linke Yang, Pei Li, Ping Wang
          </td>
          <td>2025-03-06</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Pseudogenes are epigenetic elements whose function is mostly unknown in cancer including head and neck cancers (HNSCCs). In our study we analyzed selected three pseudogenes, aurora kinase A pseudogene 1 (AURKAPS1), hect domain and RLD 2 pseudogene 2 (HERC2P2) and succinate dehydrogenase complex flavoprotein subunit A pseudogene 1 (SDHAP1), in the context of molecular function, biological role and potential utility as a biomarker in HNSCCs. Materials and methods Based on The Cancer Genome Atlas (TCGA) data we checked potential association of pseudogenes with pathological and clinical features, survival, cellular phenotype and involvement in pathways and cellular processes, and association with patients’ response to radiotherapy. Results Only AURKAPS1 pseudogene has significant upregulation in cancer than in normal samples and could be used as a diagnostic biomarker. Expression levels of all pseudogenes are dependent on cancer localization. SDHAP1 are the most differentiated and associated with tumor subtypes, expressions of AURKAPS1 do not depend on this tumor classification. Higher expression levels of AURKAPS1, HERC2P2 and SDHAP1 were associated with more aggressive phenotypes and associated with important cellular pathways and biological processes. Moreover, we observed that the expression of all pseudogenes were higher in human papilloma virus (HPV)(+) than in HPV(–) patients. Only AURKAPS1 was associated with higher genome instability and worse response to radiotherapy. Patients with higher expression levels of AURKAPS1 and HERC2P2 displayed better survival. Conclusions AURKAPS1 is a potential biomarker for HNSCC patients. This pseudogene is associated with changes in DNA repair, which should be more deeply analyzed in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d1a10ebd3706ce6a19d1ac7f97bdb08382ac67" target='_blank'>
              AURKAPS1, HERC2P2 and SDHAP1 pseudogenes: molecular role in development and progression of head and neck squamous cell carcinomas and their diagnostic utility
              </a>
            </td>
          <td>
            Tomasz Kolenda, Joanna Kozłowska-Masłoń, Patrycja Mantaj, Nina Grzejda, Kacper Kamiński, Maria Dziuba, Małgorzata Czarnecka, A. Leszczyńska, Paulina Poter, Kacper Guglas, Klaudia Dudek, Katarzyna Regulska, Paulina Gieremek, Beata J Stanisz, Anna Florczak-Substyk, Marlena Janiczek-Polewska, U. Kazimierczak, K. Lamperska, Zefiryn Cybulski, A. Teresiak
          </td>
          <td>2025-02-19</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine.
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, J. Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Tumour heterogeneity will accumulate and amplify during cell culture and passage, even if derived from the same strain. In the current study, multiple batches of colorectal cancer HCT116 and HT29 cell lines were obtained using different conditions of trypsin digestion and processed for RNA sequencing. CD39 was identified as a biomarker highly expressed in easily trypsin‐digested cells compared to the undigestible ones. Furthermore, CD39 was determined to enhance cell invasion and suppress cell apoptosis but not affect cell proliferation. Moreover, CD39 could activate the TGF‐β/SMAD3 signalling pathway, whereas the expression of CD39 was negatively regulated by SMAD3 via recruitment of SETDB1 and adding H3K9me3 to the CD39 promoter in HCT116 cells. Overall, our study uncovered distinct gene signatures amongst different heterogeneities of colorectal cells and revealed the effect of CD39 on cell invasion and apoptosis, as well as determined the epigenetic role in regulating CD39 transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4bb713ea12ae36c80b493256ba51492ce28889" target='_blank'>
              CD39 Contributes to the Ability of Cell Invasion in Heterogeneity of Colorectal Cancer
              </a>
            </td>
          <td>
            Xiaosong Li, Yifen Shen, Jianhua Lang, Jianzhong Wu, Zhenhai Qian, Genhai Shen, Yihang Shen
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The current treatment of triple-negative breast cancer (TNBC) is still primarily based on platinum-based chemotherapy. However, TNBC cells frequently develop resistance to platinum and experience relapse after drug withdrawal. It is crucial to specifically target and eliminate cisplatin-tolerant cells after platinum administration. Here, it is reported that upregulated N 6-methyladenosine (m6A) modification drives the development of resistance in TNBC cells during cisplatin treatment. Mechanistically, histone deacetylase 2 (HDAC2) mediates delactylation of methyltransferase-like 3 (METTL3), facilitating METTL3 interaction with Wilms'-tumor-1-associated protein and subsequently increasing m6A of transcript-associated DNA damage repair. This ultimately promotes cell survival under cisplatin. Furthermore, pharmacological inhibition of HDAC2 using Tucidinostat can enhance the sensitivity of TNBC cells to cisplatin therapy. This study not only elucidates the biological function of lactylated METTL3 in tumor cells but also highlights its negative regulatory effect on cisplatin resistance. Additionally, it underscores the nonclassical functional mechanism of Tucidinostat as a HDAC inhibitor for improving the efficacy of cisplatin against TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5ff71ca33fdaff595792bb621bbec0ae0038c4" target='_blank'>
              HDAC2-Mediated METTL3 Delactylation Promotes DNA Damage Repair and Chemotherapy Resistance in Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Xiaoniu He, Yuanpei Li, Jian Li, Yu Li, Sijie Chen, Xia Yan, Zhangrong Xie, Jiangfeng Du, Guoan Chen, Jianbo Song, Qi Mei
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0fddfdbd53dd2702eeb20d67caa73e1dfbdec2c" target='_blank'>
              Impact of loss of PRLHR on aneuploidy and initiation of bladder cancer.
              </a>
            </td>
          <td>
            Xi Zheng, Bingjie Yang, Wenya Wang, A. Azoitei, Felix Wezel, Christian Bolenz, Hongqian Guo, Cagatay Günes
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68584e4b61388edd6ff9ff31b6a17585ebe8fc4" target='_blank'>
              FXR1 modulates the expression of oncogenes and tumor suppressor genes associated with poor cancer prognosis
              </a>
            </td>
          <td>
            F. Khan, Xin-Ying Ji, Shaoping Ji, Jingcheng Dong
          </td>
          <td>2025-02-05</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb942377bafe8f7e43907ef0e3522eb17cee58" target='_blank'>
              1333 THY1 Upregulates Glioma Stem Cell Phenotype and Induces Treatment Resistance in Glioblastoma
              </a>
            </td>
          <td>
            Arushi Tripathy, S. Syed, V. Ravikumar, Dah-Luen Huang, A. Rehemtulla, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f17dd1fdac811901b5625401f98738e2abb40163" target='_blank'>
              IP6K2 mutations as a novel mechanism of resistance to oncolytic virus therapy
              </a>
            </td>
          <td>
            Zhijian Huang, Xiangqian Zhao, Zirong Jiang, Xiaoting Qiu, Xinhao Sun, Dawei Wang, Hucheng Zhang, Qi Chen, Ruirong Tan, Yangkun Shen
          </td>
          <td>2025-03-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a859188c2bcbbbd269e7e181ed7e2f2850df16d" target='_blank'>
              Amplified Production of a DNA Decoy Catalyzed by Intracellular MicroRNA.
              </a>
            </td>
          <td>
            Soshu Yasuda, K. Morihiro, Shuichiro Koga, Akimitsu Okamoto
          </td>
          <td>2025-02-03</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Summary Recently formed allopolyploid species offer unprecedented insights into the early stages of polyploid evolution. This review examines seven well‐studied neopolyploids (we use ‘neopolyploid’ to refer to very recently formed polyploids, i.e. during the past 300 years), spanning different angiosperm families, exploring commonalities and differences in their evolutionary trajectories. Each neopolyploid provides a unique case study, demonstrating both shared patterns, such as rapid genomic and phenotypic changes, and unique responses to hybridization and genome doubling. While previous studies of these neopolyploids have improved our understanding of polyploidy, significant knowledge gaps remain, highlighting the need for further research into the varied impacts of whole‐genome duplication on gene expression, epigenetic modifications, and ecological interactions. Notably, all of these neopolyploids have spontaneously arisen due to human activity in natural environments, underscoring the profound consequences of polyploidization in a rapidly changing world. Understanding the immediate effects of polyploidy is crucial not only for evolutionary biology but also for applied practices, as polyploidy can lead to novel traits, as well as stress tolerance and increased crop yields. Future research directions include investigating the genetic and epigenetic mechanisms underlying polyploid evolution, as well as exploring the potential of neopolyploids for crop improvement and environmental adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5a204f000a3e9de1ebd9bce03b0ead5230c52b" target='_blank'>
              Natural neopolyploids: a stimulus for novel research
              </a>
            </td>
          <td>
            P. Edger, D. Soltis, Shunsuke Yoshioka, Mario Vallejo-Marín, Rie Shimizu‐Inatsugi, Kentaro K. Shimizu, A. Salmon, Simon Hiscock, Malika Ainouche, Pamela S Soltis
          </td>
          <td>2025-02-14</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="Abstract Lung cancer sequencing efforts have uncovered mutational signatures that are attributed to exposure to the cigarette smoke carcinogen benzo[a]pyrene. Benzo[a]pyrene metabolizes in cells to benzo[a]pyrene diol epoxide (BPDE) and reacts with guanine nucleotides to form bulky BPDE adducts. These DNA adducts block transcription and replication, compromising cell function and survival, and are repaired in human cells by the nucleotide excision repair pathway. Here, we applied high-resolution genomic assays to measure BPDE-induced damage formation and mutagenesis in human cells. We integrated the new damage and mutagenesis data with previous repair, DNA methylation, RNA expression, DNA replication, and chromatin component measurements in the same cell lines, along with lung cancer mutagenesis data. BPDE damage formation is significantly enhanced by DNA methylation and in accessible chromatin regions, including transcribed and early-replicating regions. Binding of transcription factors is associated primarily with reduced, but also enhanced damage formation, depending on the factor. While DNA methylation does not appear to influence repair efficiency, this repair was significantly elevated in accessible chromatin regions, which accumulated fewer mutations. Thus, when damage and repair drive mutagenesis in opposing directions, the final mutational patterns appear to be dictated by the efficiency of repair rather than the frequency of underlying damages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfd6043f375e2cb6f682b0d7fa5f6366bcb4c10f" target='_blank'>
              The epigenetic landscape shapes smoking-induced mutagenesis by modulating DNA damage susceptibility and repair efficiency
              </a>
            </td>
          <td>
            Elisheva E. Heilbrun, Dana Tseitline, Hana Wasserman, Ayala Kirshenbaum, Yuval Cohen, Raluca Gordân, S. Adar
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ed27bb6383161df0bd1281e88568674842389e" target='_blank'>
              Transposition element MERVL regulates DNA demethylation through TET3 in oxidative-damaged mouse preimplantation embryos
              </a>
            </td>
          <td>
            Lihong Liu, Siyao Ha, Dan Cao, MingQing Li, Zhiling Li
          </td>
          <td>2025-03-12</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc6f21df46de0763db659ebbc92b87b44e41294" target='_blank'>
              Centrophilic Retrotransposons of Plant Genomes.
              </a>
            </td>
          <td>
            Alexandros Bousios, T. Kakutani, Ian R. Henderson
          </td>
          <td>2025-02-14</td>
          <td>Annual review of plant biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d901b9b33dfa1544156c22b0678e85042f152af" target='_blank'>
              Genomic landscaping of receptor tyrosine kinase ALK with highly frequent rearrangements in cancers.
              </a>
            </td>
          <td>
            Wei Ye, Wen-Bin Ou
          </td>
          <td>2025-02-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Rationale: Meiotic homologous recombination is a critical event in gametogenesis, which is tightly regulated to ensure the generation of crossovers on homologous chromosomes. This process is crucial for ensuring the accurate segregation of genetic material and maintaining genetic diversity within species, ultimately contributing to reproductive success. Nevertheless, comprehensive mechanisms of post-translational modification (PTM) regulating homologous recombination during meiosis require further investigation. The aim of this study is to investigate the regulatory mechanisms and physiological functions of NAE1-mediated neddylation during meiosis of mammalian spermatogenesis and its consequential role in infertility. Methods: The dynamic localization of NAE1 at various sub-stages during spermatogenesis was determined using immunofluorescence staining and seminiferous tubule staging. We explore the role of NAE1-mediated neddylation by utilizing germ cell-specific Nae1-knockout mice. The impact on homologous synapsis and recombination during the meiosis prophase I were verified through chromosome spread fluorescence staining. We used 10 × Genomics single cell transcriptomics and ubiquitinomics to analysis the causes of spermatogenesis arrest and spermatogenic apoptosis. Results: NAE1 exhibited high nuclear expression within spermatocytes from the pachytene stage onwards. Nae1-SKO male mice showed a late-pachytene arrest in spermatocytes, resulting in infertility. In NAE1-deficient spermatocytes, there is an increase in apoptosis. Nae1 deletion led to double-strand break (DSB) repair failure with normal autosomes synapsis. From a mechanistic perspective, we verified excessive recombination intermediate stabilization and failed crossover formation, which ultimately resulted in impaired meiotic recombination. Further analysis showed that ubiquitination regulation coordinated with NAE1-mediated neddylation was implicated in meiotic recombination. Conclusion: NAE1-mediated neddylation regulates ubiquitination during meiosis and is involved in the stabilization of recombination proteins related to crossover differentiation. We provide cytological evidence for the neddylation-ubiquitination system (NUS) in mammalian meiotic recombination during spermatogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d73e475fa9d9d51166b51741f2be3758e72a36" target='_blank'>
              NAE1-mediated neddylation coordinates ubiquitination regulation of meiotic recombination during spermatogenesis
              </a>
            </td>
          <td>
            Yu Xi, Chenjia Gong, Zhe Zhang, Feiyin Zhu, Ying Zhang, Yanlin Tang, Li-Hua Yan, Hui Jiang, Jie Qiao, Qiang Liu
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c19557776012bc520b8b5ecf4614f5b4aaa6f641" target='_blank'>
              The transcription factor SRF regulates MERVL retrotransposons and gene expression during zygotic genome activation.
              </a>
            </td>
          <td>
            C. Hermant, Carlos Michel Mourra-Díaz, Marlies E. Oomen, Luis Altamirano-Pacheco, Mrinmoy Pal, Tsunetoshi Nakatani, M. Torres-Padilla
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="p27Kip1 is a key cell cycle gatekeeper governing the timing of Cyclin-dependent kinase (CDK) activation/inactivation and, consequently, cell proliferation. Structurally, the protein is largely unfolded, a feature that strongly increases its plasticity and interactors and enhances the number of regulated cellular processes. p27Kip1, like other intrinsically unstructured proteins, is post-translationally modified on several residues. These modifications affect its cellular localization and address p27Kip1 for specific interactions/functions. Several germline or somatic CDKN1B (the p27Kip1 encoding gene) mutations have been demonstrated to be associated with multiple endocrine neoplasia type 4 (MEN4), hairy cell leukemia, small-intestine neuroendocrine tumors, and breast and prostate cancers. Here, we analyzed the effect of four CDKN1B missense and nonsense mutations found in patients affected by MEN4 or cancers, namely, c.349C>T, p.P117S; c.397C>A, p.P133T; c.487C>T, p.Q163*; and c.511G>T, p.E171*. By transfecting breast cancer cell lines, we observed increased growth and cell motility for all the investigated mutants compared to wild-type p27Kip1 transfected cells. Furthermore, we discovered that the mutant forms exhibited altered phosphorylation on key residues and different localization or degradation mechanisms in comparison to the wild-type protein and suggested a possible region as crucial for the lysosome-dependent degradation of the protein. Finally, the loss of p27Kip1 ability in blocking cell proliferation was in part explained through the different binding efficiency that mutant p27Kip1 forms exhibited with Cyclin/Cyclin-dependent Kinase complexes (or proteins involved indirectly in that binding) with respect to the WT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b858b52e6f58495fd391a4dfcc6f1900f00849e" target='_blank'>
              p27Kip1 and Tumors: Characterization of CDKN1B Variants Identified in MEN4 and Breast Cancer
              </a>
            </td>
          <td>
            Debora Bencivenga, E. Stampone, Jahanzaib Azhar, Daniela Parente, Waqar Ali, Vitale Del Vecchio, Fulvio Della Ragione, A. Borriello
          </td>
          <td>2025-01-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35698f05d68e24632b02180e50c1bfb7426f2b2" target='_blank'>
              The establishment of the 3D genome structure during zygotic genome activation.
              </a>
            </td>
          <td>
            Anastasiia L Sivkina, O. Iarovaia, S. V. Razin, S. Ulianov
          </td>
          <td>2025-03-03</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiaojun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26334a481a2a25fb7be72ec2c09273c472c1ce03" target='_blank'>
              Intrinsic immunity against HAdV is achieved by a novel epigenetic silencing complex
              </a>
            </td>
          <td>
            Julia Mai, Maryam Karimi, A. Kuderna, Anna Heider, Thomas Günther, C. Friedel, A. Grundhoff, Thomas Stamminger, Sabrina Schreiner
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c779b27de9c7947d2736cb472ee433351114fe43" target='_blank'>
              Native stem cell transcriptional circuits define cardinal features of high-risk leukemia.
              </a>
            </td>
          <td>
            Qing Wang, Francesco Boccalatte, Jason Xu, Giovanni Gambi, Bettina Nadorp, Fatema Akter, Carea Mullin, A. Melnick, Elizabeth Choe, Anna C. McCarter, Nicole A Jerome, Siyi Chen, Karena Lin, Sarah Khan, R. Kodgule, Jonathan H Sussman, P. Pölönen, Javier Rodriguez-Hernaez, Sonali Narang, Kleopatra Avrampou, B. King, A. Tsirigos, Russell J H Ryan, C. Mullighan, D. Teachey, Kai Tan, I. Aifantis, M. Chiang
          </td>
          <td>2025-02-19</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromatin is intricately folded into dynamic 3D structures, orchestrating key DNA metabolic processes. DNA replication, a core chromatin metabolic event, is tightly linked to these chromatin architectures. Recently, a replication-associated chromatin interaction structure was discovered as fountains via Replication-associated in situ Hi-C (Repli-HiC), supporting that replication forks remain spatially coupled from initiation to termination. Chromatin fountains are pivotal for understanding DNA replication within the complex chromatin landscape. However, the characteristics of these fountains can vary due to factors such as origin firing efficiency and local chromatin organization. In this study, we introduce a reinforcement learning-based computational framework, integrated with Monte Carlo Tree Search (MCTS) and value gradient optimization, to develop the Fun2 algorithm. Fun2 enables a comprehensive characterization of trajectory-like chromatin architectures, including both chromatin fountains and stripes, with enhanced adaptability and accuracy. This tool facilitates systematic investigation of the spatiotemporal dynamics of DNA replication and extends to other chromatin remodeling processes, such as Cohesin-mediated loop extrusion. The Fun2 algorithm provides a versatile computational tool for deciphering dynamic chromatin architectures, offering new insights into genome organization and its regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15cb3607b3c794a60281b1b32cb6d7e652eebf3a" target='_blank'>
              Characterizing trajectory-like chromatin architectures with Fun2
              </a>
            </td>
          <td>
            Zhengrong Zhangding, Xuhao Liu, Jiazhi Hu
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7451322e1f10fdefd4135068d35cc6376d1125" target='_blank'>
              Exploring the Potential of Optical Genome Mapping in the Diagnosis and Prognosis of Soft Tissue and Bone Tumors
              </a>
            </td>
          <td>
            Alejandro Berenguer-Rubio, Esperanza Such, Neus Torres Hernández, P. González-Rojo, Álvaro Díaz-González, G. Avetisyan, Carolina Gil-Aparicio, Judith González-López, Nicolay Pantoja-Borja, L. Rubio-Martínez, Soraya Hernández-Girón, María Soledad Valera-Cuesta, Cristina Ramírez-Fuentes, María Simonet-Redondo, Roberto Díaz-Beveridge, C. de la Calva, J. Amaya-Valero, Cristina Ballester-Ibáñez, A. Liquori, Francisco Giner, E. Mayordomo-Aranda
          </td>
          <td>2025-03-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e899cf4a3d1394fc8fe742ffd50bbb8e81d067" target='_blank'>
              Extensive mutual influences of SMC complexes shape 3D genome folding.
              </a>
            </td>
          <td>
            Han Zhao, Lirong Shu, Shiyi Qin, Fangxuan Lyu, Fuhai Liu, En Lin, Sijian Xia, Baiyue Wang, Manzhu Wang, Fengnian Shan, Yinzhi Lin, Lin Zhang, Yufei Gu, Gerd A. Blobel, Kai Huang, Haoyue Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982b46ad0f67ebc704b6d142139e5b777f0090b1" target='_blank'>
              Exosomal lncRNA Mir100hg from lung cancer stem cells activates H3K14 lactylation to enhance metastatic activity in non-stem lung cancer cells
              </a>
            </td>
          <td>
            Lei Shi, Bowen Li, Jiyu Tan, Ling Zhu, Sicheng Zhang, Yuhan Zhang, Meng Xiang, Jie Li, Yan Chen, Xue Han, Jia-Cheng Xie, Yao Tang, H. Xing, Jingyu Li, Jianyu Wang, cnJingyu Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dc1a99977067c119e44f09575781707edcc62" target='_blank'>
              Interplay of aurora kinase a functional residues and Epstein-barr Nuclear Antigen 1 in Epstein-barr virus associated Gastric cancer using AGS cells
              </a>
            </td>
          <td>
            N. Varshney, Siddharth Singh, M. Kandpal, Vaishali Saini, Erle S. Roberston, Hem Chandra Jha
          </td>
          <td>2025-01-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="FOXP3+ natural regulatory T cells (nTregs) promote resolution of inflammation and repair of epithelial damage following viral pneumonia-induced lung injury, thus representing a cellular therapy for patients with acute respiratory distress syndrome (ARDS). Whether in vitro induced Tregs (iTregs), which can be rapidly generated in substantial numbers from conventional T cells, also promote lung recovery is unknown. nTregs require specific DNA methylation patterns maintained by the epigenetic regulator, ubiquitin-like with PHD and RING finger domains 1 (UHRF1). Here, we tested whether iTregs promote recovery following viral pneumonia and whether iTregs require UHRF1 for their pro-recovery function. We found that adoptive transfer of iTregs to mice with influenza virus pneumonia promotes lung recovery and that loss of UHRF1-mediated maintenance DNA methylation in iTregs leads to reduced engraftment and a delayed repair response. Transcriptional and DNA methylation profiling of adoptively transferred UHRF1-deficient iTregs that had trafficked to influenza-injured lungs demonstrated transcriptional instability with gain of effector T cell lineage-defining transcription factors. Strategies to promote the stability of iTregs could be leveraged to further augment their pro-recovery function during viral pneumonia and other causes of ARDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8259d3d12170176131ead01b0d7705b88e45403" target='_blank'>
              Maintenance DNA methylation is required for induced regulatory T cell reparative function following viral pneumonia
              </a>
            </td>
          <td>
            A. Joudi, J. Gurkan, Qianli Liu, Manuel A. Torres Acosta, K. Helmin, Luisa Morales-Nebreda, N. Mambetsariev, Carla P. Reyes Flores, H. Abdala-Valencia, Elizabeth M. Steinert, S. Weinberg, Benjamin D. Singer
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acfe5ecebe48c685d0ade29fb13cedf2e576caf" target='_blank'>
              Frequency and impact of a genome-wide homologous recombination deficiency signature (HRDsig+) on the genomic landscape of clinically advanced urothelial bladder carcinoma (CAUBC).
              </a>
            </td>
          <td>
            Michael F. Basin, Andrea Necchi, Roger Li, Philippe E. Spiess, A. Kamat, P. Grivas, Shilpa Gupta, Neeraj Agarwal, Liang Cheng, D. Pavlick, E. Sokol, Ryon P Graf, Douglas I. Lin, J. Ross, A. Schrock, J. Quintanilha, Gerald Li, J. Jacob, A. Basnet, G. Bratslavsky
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aca8c4eb6696cc9c11ef54c9581b2900ee7f04" target='_blank'>
              Active telomere elongation by a subclass of cancer-associated POT1 mutations.
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard J Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Epigenetic regulation of gene expression is essential for cellular development and differentiation processes in higher eukaryotes. Modifications of cytosine, in particular 5-methylcytosine (5mdC), in DNA play a central role through impacting chromatin structure, repressing transposons, and regulating transcription. DNA methylation is actively installed by DNA methyltransferases and reversed through Tet-dioxygenase-mediated oxidation of 5mdC to 5-hydroxylmethylcytosine (5hmdC), 5-formylcytosine (5fdC), and 5-carboxycytosine (5cadC). It is crucial to understand the role of these epigenetic DNA modifications in cellular differentiation and developmental processes, as well as in disease state mapping and tracing of 5mdC and its oxidized forms. In bisulfite sequencing, which has been the benchmark for mapping 5mdC for the last few decades, degradation of the majority of genetic material occurs through harsh chemical treatment. Alternative sequencing methods often utilize Tet-enzyme-mediated oxidation of 5mdC to locate 5mdC and 5hmdC in genomic DNA. Herein, we report the development of novel Tet3-variants for oxidation-based bisulfite-free 5mdC- sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e804da833f12d96ee4c49d305fc6d792d26acb4a" target='_blank'>
              Novel Tet3 enzymes for next-generation epigenetic sequencing.
              </a>
            </td>
          <td>
            Özge Simsir, Tobias Walter, Hanife Sahin, Thomas Carell, Sabine Schneider
          </td>
          <td>2025-03-10</td>
          <td>RSC chemical biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f65c0d127245d474792c410ee7afae8aec904" target='_blank'>
              Complicated crosstalk between HMGA and non-coding RNAs modulates hallmarks of cancer
              </a>
            </td>
          <td>
            Lijie Zhang, Xiaomin Zhao, Xianghong Gao, Hao Qin, Feng Chen, Zhijuan Lin
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff246fd6cb059fea5f5e9204ca830e7b6e029678" target='_blank'>
              Characterizing macrophage diversity in colorectal malignancies through single-cell genomics
              </a>
            </td>
          <td>
            Tingshuo Zhao, Yinyi Luo, Yuanjie Sun, Zhigang Wei
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67046daa5df3cebbea573edba163f96ed0a30fda" target='_blank'>
              Visualization of chromosomal reorganization induced by heterologous fusions in the mammalian nucleus
              </a>
            </td>
          <td>
            Meng Yan, X. Zhang, Zhenhua Yang, Miao Jia, Rongyu Liao, Jinsong Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Synthetic lethality approaches in BRCA1/2-mutated cancers have focused on poly(ADP-ribose) polymerase (PARP) inhibitors, which are subject to high rates of innate or acquired resistance in patients. Here, we used CRISPR/Cas9-based screening to identify DNA Ligase I (LIG1) as a novel target for synthetic lethality in BRCA1-mutated cancers. Publicly available data supported LIG1 hyperdependence of BRCA1-mutant cells across a variety of breast and ovarian cancer cell lines. We used CRISPRn, CRISPRi, RNAi, and protein degradation to confirm the lethal effect of LIG1 inactivation at the DNA, RNA, and protein level in BRCA1-mutant cells in vitro. LIG1 inactivation resulted in viability loss across multiple BRCA1-mutated cell lines, whereas no effect was observed in BRCA1/2 wild-type cell lines, demonstrating target selectivity for the BRCA1-mutant context. On-target nature of the phenotype was demonstrated through rescue of viability with exogenous wild-type LIG1 cDNA. Next, we demonstrated a concentration-dependent relationship of LIG1 protein expression and BRCA1 mutant cell viability using a titratable, degradable LIG1 fusion protein. BRCA1 mutant viability required LIG1 catalytic activity, as catalytically dead mutant LIG1K568A failed to rescue viability loss caused by endogenous LIG1 depletion. LIG1 perturbation produced proportional increases in PAR staining in BRCA1 mutant cells, indicating a mechanism consistent with the function of LIG1 in sealing ssDNA nicks. Finally, we confirmed LIG1 hyperdependence in vivo using a xenograft model in which LIG1 loss resulted in tumor stasis in all mice. Our cumulative findings demonstrate that LIG1 is a promising synthetic lethal target for development in patients with BRCA1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831e285c6dc25f79bcb13593b00dacbe2a024fd9" target='_blank'>
              LIG1 is a synthetic lethal target in BRCA1 mutant cancers.
              </a>
            </td>
          <td>
            Lauren Martires, Leanne G Ahronian, Charlotte B Pratt, Nikitha M Das, Xiaobin Zhang, D. Whittington, Hongxiang Zhang, B. Shen, Jon Come, Patrick McCarren, Mu-Sen Liu, Chengyin Min, Tianshu Feng, Haris Jahic, Janid A Ali, Daniel R Aird, Fang Li, Jannik N Andersen, A. Huang, William D Mallender, Hilary E. Nicholson
          </td>
          <td>2025-01-27</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Disrupted nuclear shape is associated with multiple pathological processes including premature aging disorders, cancer-relevant chromosomal rearrangements and DNA damage. Nuclear blebs (i.e. herniations of the nuclear envelope) can be induced by (1) nuclear compression, (2) nuclear migration (e.g. cancer metastasis), (3) actin contraction, (4) lamin mutation or depletion, and (5) heterochromatin enzyme inhibition. Recent work has shown that chromatin transformation is a hallmark of bleb formation, but the transformation of higher-order structures in blebs is not well understood. As higher-order chromatin has been shown to assemble into nanoscopic packing domains, we investigated whether (1) packing domain organization is altered within nuclear blebs and (2) whether alteration in packing domain structure contributed to bleb formation. Using dual-partial wave spectroscopic microscopy, we show that chromatin-packing domains within blebs are transformed both by B-type lamin depletion and the inhibition of heterochromatin enzymes compared to what is seen in the nuclear body. Pairing these results with single-molecule localization microscopy of constitutive heterochromatin, we show fragmentation of nanoscopic heterochromatin domains within bleb domains. Overall, these findings indicate that chromatin within blebs is associated with a fragmented higher-order chromatin structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691331448a7d0baa47fbb1a0f67d9e5391519de2" target='_blank'>
              Nuclear blebs are associated with destabilized chromatin-packing domains
              </a>
            </td>
          <td>
            Emily M. Pujadas Liwag, Nicolas Acosta, L. Almassalha, Yuanzhe Su, Ruyi Gong, Masato T. Kanemaki, A. Stephens, Vadim Backman
          </td>
          <td>2025-01-29</td>
          <td>Journal of Cell Science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324fb87b02d8939636995cfccebb627de6d9cb0d" target='_blank'>
              Abstract B002: DNA methylation protects cancer cells against senescence
              </a>
            </td>
          <td>
            Xiaoying Chen, Kosuke Yamaguchi, Brianna Rodgers, D. Goehrig, D. Vindrieux, Xavier Lahaye, Matthieu Nolot, Laure Ferry, Nadine Martin, Pierre Dubus, F. Miura, Takashi Ito, Nicolas Manel, Masato Kanemaki, David Bernard, P. Defossez
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Noncoding microRNAs tend to evolve within introns of coding genes that provide them with transcriptional opportunity. As an outcome of natural selection, the intragenic position of microRNAs is crucial for their expression, evolution, and functional cooperation with the host gene. Therefore, understanding the evolution of intragenic microRNA structures may bring novel insights into genetic and phenotypic evolution. However, it remains largely unexplored. Here, by analyzing microRNA genomics in 34 metazoan species, we found that the majority (630/1,154) of microRNA families originated from introns of coding genes that provided them with initial transcriptional capacity. The most rapid expansion of intragenic microRNAs happened at the advent of vertebrates when 21 microRNA families emerged from introns of neural genes and reorganized the gene regulatory network, leading to the rise of vertebrate-specific neural crest cells, which transformed the invertebrate head and enabled the ecological shift from filter feeding to active predation. Intragenic microRNAs gradually gain independence from their host genes, which is accelerated by whole-genome duplications. After a whole-genome duplication, the purging of redundant host genes often set an orphaned microRNA “free” to diversify with the transcriptional elements inherited from the host. Whole-genome duplications facilitated a dramatic microRNA diversification during the initial divergence of vertebrates, as the intragenic status of 12 neural crest-regulating microRNAs was retained in jawed vertebrates but was lost in jawless cyclostomes, which diverged their neural crest development. We propose that coding genes not only facilitate the origination of new microRNAs, but also “sacrifice” themselves to help microRNA diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec14a5673e5811f8ba88226799b41e9414365cb1" target='_blank'>
              Coding Genes Helped the Origination and Diversification of Intragenic MicroRNAs
              </a>
            </td>
          <td>
            Chengzhang Liu, Xiaojun Zhang, Jianbo Yuan, J. Xiang, Fuhua Li
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53dcae9646767b55b828bf57d0d3c3835c11bb8d" target='_blank'>
              Abstract B035: Evaluating drug sensitivity heterogeneity in cancer organoids derived from circulating tumor cells
              </a>
            </td>
          <td>
            Y. Han, David Hsieh, Shian-Jiun Shih
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7749502012ae50efd549b8d9691628c7f5bce04c" target='_blank'>
              Epigenetics of Human Telomeres.
              </a>
            </td>
          <td>
            Nicole Bettin, Mélina Vaurs, Anabelle Decottignies
          </td>
          <td>2025-03-17</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd9f59a39a075b237d1c5fa6c39932e43dffde05" target='_blank'>
              Long term latency of highly mutated cells in normal mouse skin is reversed by exposure to tumor promoters and chronic tissue damage.
              </a>
            </td>
          <td>
            Y. R. Li, E. Kandyba, Kyle Halliwill, R. Delrosario, Quan Tran, Nora Bayani, Di Wu, Olga K. Mirzoeva, Melissa Q Reeves, S. M. A. Islam, Laura Riva, E. Bergstrom, Kavya Achanta, John DiGiovanni, Ludmil B. Alexandrov, Allan Balmain
          </td>
          <td>2025-03-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/590a8f98137453dc965f4fb64612753d1428ec38" target='_blank'>
              The functions and modifications of tRNA-derived small RNAs in cancer biology.
              </a>
            </td>
          <td>
            Abdulaziz Ahmed A Saad, Kun Zhang, Qianqian Deng, Jiawang Zhou, Lichen Ge, Hongsheng Wang
          </td>
          <td>2025-03-01</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, M. K. Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc280a650e26ad4fbfbcabfa8901f734f1fa074d" target='_blank'>
              Chromosomal localization of PHOX2B during M-phase is disrupted in disease-associated mutants.
              </a>
            </td>
          <td>
            Yuki Sato, Shin-ichiro Hayashi, Souichi Oe, T. Koike, Y. Nakano, Ryohei Seki-Omura, Hikaru Iwashita, Y. Hirahara, Masaaki Kitada
          </td>
          <td>2025-02-11</td>
          <td>Development, growth & differentiation</td>
          <td>0</td>
          <td>17</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>